<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/9/3023" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/9/3023.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/9/3023.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/9/3023.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/9/3023.atom" />
<meta name="HW.pisa" content="ggg;9/9/3023" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Transcriptome-Wide Association Supplements Genome-Wide Association in Zea mays" />
<meta name="DC.Identifier" content="10.1534/g3.119.400549" />
<meta name="DC.Date" content="2019-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2019 Kremling et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Modern improvement of complex traits in agricultural species relies on successful associations of heritable molecular variation with observable phenotypes. Historically, this pursuit has primarily been based on easily measurable genetic markers. The recent advent of new technologies allows assaying and quantifying biological intermediates (hereafter endophenotypes) which are now readily measurable at a large scale across diverse individuals. The usefulness of endophenotypes for delineating the regulatory landscape of the genome and genetic dissection of complex trait variation remains underexplored in plants. The work presented here illustrated the utility of a large-scale (299-genotype and seven-tissue) gene expression resource to dissect traits across multiple levels of biological organization. Using single-tissue- and multi-tissue-based transcriptome-wide association studies (TWAS), we revealed that about half of the functional variation acts through altered transcript abundance for maize kernel traits, including 30 grain carotenoid abundance traits, 20 grain tocochromanol abundance traits, and 22 field-measured agronomic traits. Comparing the efficacy of TWAS with genome-wide association studies (GWAS) and an ensemble approach that combines both GWAS and TWAS, we demonstrated that results of TWAS in combination with GWAS increase the power to detect known genes and aid in prioritizing likely causal genes. Using a variance partitioning approach in the largely independent maize Nested Association Mapping (NAM) population, we also showed that the most strongly associated genes identified by combining GWAS and TWAS explain more heritable variance for a majority of traits than the heritability captured by the random genes and the genes identified by GWAS or TWAS alone. This not only improves the ability to link genes to phenotypes, but also highlights the phenotypic consequences of regulatory variation in plants." />
<meta name="DC.Contributor" content="Karl A. G. Kremling" />
<meta name="DC.Contributor" content="Christine H. Diepenbrock" />
<meta name="DC.Contributor" content="Michael A. Gore" />
<meta name="DC.Contributor" content="Edward S. Buckler" />
<meta name="DC.Contributor" content="Nonoy B. Bandillo" />
<meta name="article:published_time" content="2019-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Transcriptome-Wide Association Supplements Genome-Wide Association in Zea mays" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Modern improvement of complex traits in agricultural species relies on successful associations of heritable molecular variation with observable phenotypes. Historically, this pursuit has primarily been based on easily measurable genetic markers. The recent advent of new technologies allows assaying and quantifying biological intermediates (hereafter endophenotypes) which are now readily measurable at a large scale across diverse individuals. The usefulness of endophenotypes for delineating the regulatory landscape of the genome and genetic dissection of complex trait variation remains underexplored in plants. The work presented here illustrated the utility of a large-scale (299-genotype and seven-tissue) gene expression resource to dissect traits across multiple levels of biological organization. Using single-tissue- and multi-tissue-based transcriptome-wide association studies (TWAS), we revealed that about half of the functional variation acts through altered transcript abundance for maize kernel traits, including 30 grain carotenoid abundance traits, 20 grain tocochromanol abundance traits, and 22 field-measured agronomic traits. Comparing the efficacy of TWAS with genome-wide association studies (GWAS) and an ensemble approach that combines both GWAS and TWAS, we demonstrated that results of TWAS in combination with GWAS increase the power to detect known genes and aid in prioritizing likely causal genes. Using a variance partitioning approach in the largely independent maize Nested Association Mapping (NAM) population, we also showed that the most strongly associated genes identified by combining GWAS and TWAS explain more heritable variance for a majority of traits than the heritability captured by the random genes and the genes identified by GWAS or TWAS alone. This not only improves the ability to link genes to phenotypes, but also highlights the phenotypic consequences of regulatory variation in plants.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/09/01" />
<meta name="citation_mjid" content="ggg;9/9/3023" />
<meta name="citation_id" content="9/9/3023" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/9/3023" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/9/3023.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/9/3023.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/9/3023.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400549" />
<meta name="citation_pmid" content="31337639" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3023" />
<meta name="citation_lastpage" content="3033" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Karl A. G. Kremling" />
<meta name="citation_author_institution" content="Institute for Genomic Diversity" />
<meta name="citation_author_institution" content="Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853, and" />
<meta name="citation_author_email" content="kak268@cornell.edu" />
<meta name="citation_author_email" content="nonoy.bandillo@ndsu.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5068-0372" />
<meta name="citation_author" content="Christine H. Diepenbrock" />
<meta name="citation_author_institution" content="Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853, and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8411-0343" />
<meta name="citation_author" content="Michael A. Gore" />
<meta name="citation_author_institution" content="Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853, and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6896-8024" />
<meta name="citation_author" content="Edward S. Buckler" />
<meta name="citation_author_institution" content="Institute for Genomic Diversity" />
<meta name="citation_author_institution" content="Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853, and" />
<meta name="citation_author_institution" content="United States Department of Agriculture-Agricultural Research Service, Robert W. Holley Center for Agriculture and Health, Ithaca, NY 14853" />
<meta name="citation_author" content="Nonoy B. Bandillo" />
<meta name="citation_author_institution" content="Institute for Genomic Diversity" />
<meta name="citation_author_email" content="kak268@cornell.edu" />
<meta name="citation_author_email" content="nonoy.bandillo@ndsu.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5941-9047" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=F. W. Albert;citation_author=L. Kruglyak;citation_title=The role of regulatory variation in complex traits and disease.;citation_pages=197-212;citation_volume=16;citation_year=2015;citation_pmid=25707927;citation_doi=10.1038/nrg3891" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=A. Battle;citation_author=Z. Khan;citation_author=S. H. Wang;citation_author=A. Mitrano;citation_author=M. J. Ford;citation_title=Genomic variation. impact of regulatory variation from RNA to protein.;citation_volume=347;citation_year=2015;citation_pmid=25657249;citation_doi=10.1126/science.1260793" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. J. Bradbury;citation_author=Z. Zhang;citation_author=D. E. Kroon;citation_author=T. M. Casstevens;citation_author=Y. Ramdoss;citation_title=TASSEL: Software for association mapping of complex traits in diverse samples.;citation_volume=23;citation_year=2007;citation_pmid=17586829;citation_doi=10.1093/bioinformatics/btm308" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=R. Bukowski;citation_author=X. Guo;citation_author=Y. Lu;citation_author=C. Zou;citation_author=B. He;citation_title=Construction of the third generation zea mays haplotype map.;citation_pages=1-12;citation_volume=7;citation_year=2018;citation_doi=10.1093/gigascience/gix134" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. B. Carroll;citation_title=Evo-devo and an expanding evolutionary synthesis: a genetic theory of morphological evolution.;citation_pages=25-36;citation_volume=134;citation_year=2008;citation_issue=1;citation_pmid=18614008;citation_doi=10.1016/j.cell.2008.06.030" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. M. Chick;citation_author=S. C. Munger;citation_author=P. Simecek;citation_author=E. L. Huttlin;citation_author=K. Choi;citation_title=Defining the consequences of genetic variation on a proteome-wide scale.;citation_pages=500-505;citation_volume=534;citation_year=2016;citation_pmid=27309819;citation_doi=10.1038/nature18270" />
<meta name="citation_reference" content="citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=S. P. Dickson;citation_author=K. Wang;citation_author=H. Hakonarson;citation_author=D. B. Goldstein;citation_title=Rare variants create synthetic genome-wide associations.;citation_pages=e1000294-e1000294;citation_volume=8;citation_year=2010;citation_issue=1;citation_pmid=20126254;citation_doi=10.1371/journal.pbio.1000294" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=C. H. Diepenbrock;citation_author=C. B. Kandianis;citation_author=A. E. Lipka;citation_author=M. Magallanes-Lundback;citation_author=B. Vaillancourt;citation_title=Novel loci underlie natural variation in vitamin E levels in maize grain.;citation_volume=29;citation_year=2017;citation_pmid=28970338;citation_doi=10.1105/tpc.17.00475" />
<meta name="citation_reference" content="citation_journal_title=TAG;citation_author=P. Dubreuil;citation_author=A. Charcosset;citation_title=Genetic diversity within and among maize populations: A comparison between isozyme and nuclear RFLP loci.;citation_pages=577-587;citation_volume=96;citation_year=1998;citation_doi=10.1007/s001220050776" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=S. A. Flint-Garcia;citation_author=A. C. Thuillet;citation_author=J. Yu;citation_author=G. Pressoir;citation_author=S. M. Romero;citation_title=Maize association population: a high-resolution platform for quantitative trait locus dissection.;citation_pages=1054-1064;citation_volume=44;citation_year=2005;citation_issue=6;citation_pmid=16359397;citation_doi=10.1111/j.1365-313X.2005.02591.x" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=J. Fu;citation_author=Y. Cheng;citation_author=J. Linghu;citation_author=X. Yang;citation_author=L. Kang;citation_title=RNA sequencing reveals the complex regulatory network in the maize kernel.;citation_pages=2832;citation_volume=4;citation_year=2013;citation_pmid=24343161;citation_doi=10.1038/ncomms3832" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=G. Gibson;citation_title=Rare and common variants: Twenty arguments.;citation_pages=135-145;citation_volume=13;citation_year=2012;citation_pmid=22251874;citation_doi=10.1038/nrg3118" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_title=The Genotype-Tissue Expression (GTEx) pilot analysis: Multitissue gene regulation in humans;citation_pages=648-660;citation_volume=348;citation_year=2015;citation_issue=6235;citation_pmid=25954001;citation_doi=10.1126/science.1262110" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=A. Gusev;citation_author=S. H. Lee;citation_author=G. Trynka;citation_author=H. Finucane;citation_author=B. J. Vilhjalmsson;citation_title=Partitioning heritability of regulatory and cell-type-specific variants across 11 common diseases.;citation_pages=535-552;citation_volume=95;citation_year=2014;citation_pmid=25439723;citation_doi=10.1016/j.ajhg.2014.10.004" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=A. Gusev;citation_author=A. Ko;citation_author=H. Shi;citation_author=G. Bhatia;citation_author=W. Chung;citation_title=Integrative approaches for large-scale transcriptome-wide association studies.;citation_pages=245-252;citation_volume=48;citation_year=2016;citation_pmid=26854917;citation_doi=10.1038/ng.3506" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=C. E. Harjes;citation_author=T. R. Rocheford;citation_author=L. Bai;citation_author=T. P. Brutnell;citation_author=C. B. Kandianis;citation_title=Natural Genetic Variation in Lycopene Epsilon Cyclase Tapped for Maize Biofortification;citation_pages=330-333;citation_volume=319;citation_year=2008;citation_issue=5861;citation_pmid=18202289;citation_doi=10.1126/science.1150255" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=B. M. Henn;citation_author=L. R. Botigue;citation_author=C. D. Bustamante;citation_author=A. G. Clark;citation_author=S. Gravel;citation_title=Estimating the mutation load in human genomes.;citation_pages=333-343;citation_volume=16;citation_year=2015;citation_pmid=25963372;citation_doi=10.1038/nrg3931" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=C. N. Hirsch;citation_author=J. M. Foerster;citation_author=J. M. Johnson;citation_author=R. S. Sekhon;citation_author=G. Muttoni;citation_title=Insights into the maize pan-genome and pan-transcriptome.;citation_volume=26;citation_year=2014;citation_pmid=24488960;citation_doi=10.1105/tpc.113.119982" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. B. Hufford;citation_author=X. Xu;citation_author=J. van Heerwaarden;citation_author=T. Pyhajarvi;citation_author=J. M. Chia;citation_title=Comparative population genomics of maize domestication and improvement.;citation_pages=808-811;citation_volume=44;citation_year=2012;citation_issue=7;citation_pmid=22660546;citation_doi=10.1038/ng.2309" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=H. Y. Hung;citation_author=C. Browne;citation_author=K. Guill;citation_author=N. Coles;citation_author=M. Eller;citation_title=The relationship between parental genetic or phenotypic divergence and progeny variation in the maize nested association mapping population.;citation_pages=490-499;citation_volume=108;citation_year=2012;citation_issue=5;citation_pmid=22027895;citation_doi=10.1038/hdy.2011.103" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. Juntawong;citation_author=T. Girke;citation_author=J. Bazin;citation_author=J. Bailey-Serres;citation_title=Translational dynamics revealed by genome-wide profiling of ribosome footprints in arabidopsis.;citation_volume=111;citation_year=2014;citation_pmid=24367078;citation_doi=10.1073/pnas.1317811111" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=K. A. Kremling;citation_author=S. Chen;citation_author=M. Su;citation_author=N. Lepak;citation_author=C. M. Romay;citation_title=Dysregulation of expression correlates with rare allele burden and fitness loss in maize.;citation_pages=520-523;citation_volume=555;citation_year=2018;citation_doi=10.1038/nature25966" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. A. Law;citation_author=S. E. Jacobsen;citation_title=Establishing, maintaining and modifying DNA methylation patterns in plants and animals.;citation_pages=204-220;citation_volume=11;citation_year=2010;citation_issue=3;citation_pmid=20142834;citation_doi=10.1038/nrg2719" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=S. Leiboff;citation_author=X. Li;citation_author=H. C. Hu;citation_author=N. Todt;citation_author=J. Yang;citation_title=Genetic control of morphometric diversity in the maize shoot apical meristem.;citation_pages=8974;citation_volume=6;citation_year=2015;citation_doi=10.1038/ncomms9974" />
<meta name="citation_reference" content="citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=A. E. Lipka;citation_author=M. A. Gore;citation_author=M. Magallanes-Lundback;citation_author=A. Mesberg;citation_author=H. Lin;citation_title=Genome-wide association study and pathway-level analysis of tocochromanol levels in maize grain.;citation_volume=3;citation_year=2013;citation_pmid=23733887;citation_doi=10.1534/g3.113.006148" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=C. Lippert;citation_author=J. Listgarten;citation_author=Y. Liu;citation_author=C. M. Kadie;citation_author=R. I. Davidson;citation_title=FaST linear mixed models for genome-wide association studies.;citation_pages=833-835;citation_volume=8;citation_year=2011;citation_issue=10;citation_pmid=21892150;citation_doi=10.1038/nmeth.1681" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=N. Mancuso;citation_author=H. Shi;citation_author=P. Goddard;citation_author=G. Kichaev;citation_author=A. Gusev;citation_title=Integrating Gene Expression with Summary Association Statistics to Identify Genes Associated with 30 complex traits.;citation_pages=473-487;citation_volume=100;citation_year=2017;citation_doi=10.1016/j.ajhg.2017.01.031" />
<meta name="citation_reference" content="citation_title=Populations, species, and evolution: An abridgment of animal species and evolution;citation_year=1970" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. F. Owens;citation_author=A. E. Lipka;citation_author=M. Magallanes-Lundback;citation_author=T. Tiede;citation_author=C. H. Diepenbrock;citation_title=A foundation for provitamin A biofortification of maize: Genome-wide association and genomic prediction models of carotenoid levels.;citation_volume=198;citation_year=2014;citation_pmid=25258377;citation_doi=10.1534/genetics.114.169979" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=B. Pasaniuc;citation_author=A. L. Price;citation_title=Dissecting the genetics of complex traits using summary association statistics.;citation_pages=117-127;citation_volume=18;citation_year=2017;citation_doi=10.1038/nrg.2016.142" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. Riedelsheimer;citation_author=A. Czedik-Eysenberg;citation_author=C. Grieder;citation_author=J. Lisec;citation_author=F. Technow;citation_title=Genomic and metabolic prediction of complex heterotic traits in hybrid maize.;citation_pages=217-220;citation_volume=44;citation_year=2012;citation_issue=2;citation_pmid=22246502;citation_doi=10.1038/ng.1033" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=E. Rodgers-Melnick;citation_author=D. L. Vera;citation_author=H. W. Bass;citation_author=E. S. Buckler;citation_title=Open chromatin reveals the functional maize genome.;citation_volume=113;citation_year=2016;citation_pmid=27185945;citation_doi=10.1073/pnas.1525244113" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. A. Shabalin;citation_title=Matrix eQTL: Ultra fast eQTL analysis via large matrix operations.;citation_volume=28;citation_year=2012;citation_pmid=22492648;citation_doi=10.1093/bioinformatics/bts163" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=D. Speed;citation_author=N. Cai;citation_author=M. R. Johnson;citation_author=S. Nejentsev;citation_author=D. J. Balding;citation_title=Reevaluation of SNP heritability in complex human traits.;citation_pages=986-992;citation_volume=49;citation_year=2017;citation_doi=10.1038/ng.3865" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. A. Tishkoff;citation_author=F. A. Reed;citation_author=A. Ranciaro;citation_author=B. F. Voight;citation_author=C. C. Babbitt;citation_title=Convergent adaptation of human lactase persistence in Africa and Europe.;citation_pages=31-40;citation_volume=39;citation_year=2007;citation_issue=1;citation_pmid=17159977;citation_doi=10.1038/ng1946" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. G. Wallace;citation_author=P. J. Bradbury;citation_author=N. Zhang;citation_author=Y. Gibon;citation_author=M. Stitt;citation_title=Association mapping across numerous traits reveals patterns of functional variation in maize.;citation_pages=e1004845;citation_volume=10;citation_year=2014;citation_pmid=25474422;citation_doi=10.1371/journal.pgen.1004845" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. Welter;citation_author=J. MacArthur;citation_author=J. Morales;citation_author=T. Burdett;citation_author=P. Hall;citation_title=The NHGRI GWAS catalog, a curated resource of SNP-trait associations.;citation_volume=42;citation_year=2014;citation_pmid=24316577;citation_doi=10.1093/nar/gkt1229" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. J. Wisser;citation_author=J. M. Kolkman;citation_author=M. E. Patzoldt;citation_author=J. B. Holland;citation_author=J. Yu;citation_title=Multivariate analysis of maize disease resistances suggests a pleiotropic genetic basis and implicates a GST gene.;citation_volume=108;citation_year=2011;citation_pmid=21490302;citation_doi=10.1073/pnas.1011739108" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yan;citation_author=C. B. Kandianis;citation_author=C. E. Harjes;citation_author=L. Bai;citation_author=E. H. Kim;citation_title=Rare genetic variation at Zea mays crtRB1 increases beta-carotene in maize grain.;citation_pages=322-327;citation_volume=42;citation_year=2010;citation_issue=4;citation_pmid=20305664;citation_doi=10.1038/ng.551" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Yu;citation_author=J. B. Holland;citation_author=M. D. McMullen;citation_author=E. S. Buckler;citation_title=Genetic design and statistical power of nested association mapping in maize.;citation_volume=178;citation_year=2008;citation_pmid=18202393;citation_doi=10.1534/genetics.107.074245" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Transcriptome-Wide Association Supplements Genome-Wide Association in Zea mays" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
<meta name="twitter:description" content="Modern improvement of complex traits in agricultural species relies on successful associations of heritable molecular variation with observable phenotypes. Historically, this pursuit has primarily been based on easily measurable genetic markers. The recent advent of new technologies allows assaying and quantifying biological intermediates (hereafter endophenotypes) which are now readily measurable at a large scale across diverse individuals. The usefulness of endophenotypes for delineating the regulatory landscape of the genome and genetic dissection of complex trait variation remains underexplored in plants. The work presented here illustrated the utility of a large-scale (299-genotype and seven-tissue) gene expression resource to dissect traits across multiple levels of biological organization. Using single-tissue- and multi-tissue-based transcriptome-wide association studies (TWAS), we revealed that about half of the functional variation acts through altered transcript abundance for maize kernel traits, including 30 grain carotenoid abundance traits, 20 grain tocochromanol abundance traits, and 22 field-measured agronomic traits. Comparing the efficacy of TWAS with genome-wide association studies (GWAS) and an ensemble approach that combines both GWAS and TWAS, we demonstrated that results of TWAS in combination with GWAS increase the power to detect known genes and aid in prioritizing likely causal genes. Using a variance partitioning approach in the largely independent maize Nested Association Mapping (NAM) population, we also showed that the most strongly associated genes identified by combining GWAS and TWAS explain more heritable variance for a majority of traits than the heritability captured by the random genes and the genes identified by GWAS or TWAS alone. This not only improves the ability to link genes to phenotypes, but also highlights the phenotypic consequences of regulatory variation in plants." />
<meta name="og-title" property="og:title" content="Transcriptome-Wide Association Supplements Genome-Wide Association in Zea mays" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/9/3023" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Modern improvement of complex traits in agricultural species relies on successful associations of heritable molecular variation with observable phenotypes. Historically, this pursuit has primarily been based on easily measurable genetic markers. The recent advent of new technologies allows assaying and quantifying biological intermediates (hereafter endophenotypes) which are now readily measurable at a large scale across diverse individuals. The usefulness of endophenotypes for delineating the regulatory landscape of the genome and genetic dissection of complex trait variation remains underexplored in plants. The work presented here illustrated the utility of a large-scale (299-genotype and seven-tissue) gene expression resource to dissect traits across multiple levels of biological organization. Using single-tissue- and multi-tissue-based transcriptome-wide association studies (TWAS), we revealed that about half of the functional variation acts through altered transcript abundance for maize kernel traits, including 30 grain carotenoid abundance traits, 20 grain tocochromanol abundance traits, and 22 field-measured agronomic traits. Comparing the efficacy of TWAS with genome-wide association studies (GWAS) and an ensemble approach that combines both GWAS and TWAS, we demonstrated that results of TWAS in combination with GWAS increase the power to detect known genes and aid in prioritizing likely causal genes. Using a variance partitioning approach in the largely independent maize Nested Association Mapping (NAM) population, we also showed that the most strongly associated genes identified by combining GWAS and TWAS explain more heritable variance for a majority of traits than the heritability captured by the random genes and the genes identified by GWAS or TWAS alone. This not only improves the ability to link genes to phenotypes, but also highlights the phenotypic consequences of regulatory variation in plants." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
    <title>Transcriptome-Wide Association Supplements Genome-Wide Association in Zea mays | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"2mdcP8jU3vdgCzBMhQfNwzKPxtP7SfFszNnC70ECLBs","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494050","apath":"\/ggg\/9\/9\/3023.atom","pisa":"ggg;9\/9\/3023","ac":{"\/ggg\/9\/9\/3023.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/9\/3023.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/9\/3023"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/9\/3023"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/9\/3023":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022sozrfBDokm7YrqDNhP8hsptGXYOCGuMnPgxulJwgmoS2k7vyqD9sVcekqytd03WKxHZX5HY+XKK+Hz0dI4AUSO\\\\\\\/MUayHhVCBxtoCjTaoZPFmVqZi5EhWVzKMRdTKLMTJ23ykYudPlm0tZzU66WCeV+TTrBlCOKaC0qHYjVLJ1oGO2kin2TawGKv2PRteLThhZEj1lvJbv0Tq7TL8jG9jyGs9t7bZJ56kIpjZKaw4a1ux5yMw6HPluLYaEOJpw20LjCYRjrxQxOKQTf0usKm0Mwf3quZvT3TtNtHnnKnDzno8M5wPccew1dKFFSg+Obg2QarbG7a37tf+wbRqALYynJROd8tsXgmIxR8Fph69PPXrzq2btjALsh57Qk+LzDNfBb1GA9X0EqKW9\\\\\\\/eIktsuwLKnLG+CQ9HlshMf2FirnGB8JdF6Nx\\\\\\\/g33RcDAJVLwDKkvlmTw8PNNgx+n4EH8ig3Yk4IKgQAy\\\\\\\/8c5i26uBGwcStcr0OR8EMS+ak\\\\\\\/69teS2K46k529ZItagZNfmqimEnRB351+zYse0KoKy4EuxJ25UV9Lyz3oqufliIi00ihtvzdwyAyOQtMqTiu3\\\\\\\/StupGsAHA6DAAqiX8QYblS+PPLK1DPFMfSPwhOxJUCuqcrzFHM\\\\\\\/Ds62SmOQ1WRXWDNHsTuVqYrx0y9ijH5YJO3tGq+RUCaXJkKUk12zWLa8QuzCS6o2cywDG0SkHMzCRAQ\\\\\\\/K4q5XxfwIWg31KYu18L22VLT9EQM8xa+w9\\\\\\\/PY3CfadNYwakaUGq6\\\\\\\/Ik+cq7cQlkhRmAS8w+UNWosYsJ+MKeWkwPFmV1lMl7PgcKyRXrgqy2IaHEgLXKP9hfPBYnDTkJJD5UHRd8OpqKcIDN6tZeCg+qah+lchQtN9Wta0do3HV5AMWFiw2IBkKrkTdqNjTWMThWf0D608hi0tr45JNP2mcmBPCNYTlCrWSqOQrJs3kV43wm0U2vltw5Khn7sev9i83oj6bDM1C7xATqio7WdBcoPT8wJPzTY6J8fW2CTh8khubMLa5kWbEamDWR9zwJtRXem3\\\\\\\/TKgT6raUPinCXd8mO7S9KEpZ1gYA0rbQ\\\\\\\/\\\\\\\/EcDGMNsp+L8WQ8ztSTSQ\\\\\\\/6iC\\\\\\\/0XiyIWAUUHCfMiEHhsK3s2UcTcv1Y0gWoAJdBZ8NvAp50cvvtVfTG9bQkT1iXeOhY76oF79IQKQeyCBHS\\\\\\\/YtVSQq6NedEGFo6FKHSYqnhg0oJETidY2o2gn2QJ33rh5pA4PehEdVf2Uy2T20AO2Jgtj3lLo4YmNCDW2bcNbZ40f\\\\\\\/\\\\\\\/\\\u0022,\\\u0022iv\\\u0022:\\\u0022Qi4Xg2\\\\\\\/SV15GyU\\\\\\\/N43rxCQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ef568cb26a3cb5aa0909fd93fe4da3370c6fc89e48bd28e00600b1c5fe4046fd\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022rzy+9xznf4C3bV\\\\\\\/LN3XMAVI275FOJmb\\\\\\\/NpoIfg0JoeoGGq3jGx8kgK6H7Rzgvbnq1rKzs9r+xsVZHjHIOUz7KE6\\\\\\\/YkyG42Eq8JriQoQa15P9Zto3ULG4G2mX5BWedksS7klSO3l9MePVsDLOvXkxCc2Fkx5TDrfRfqyA1NmKEvk32gaSySb7oJTcD2CPKuuvrWFdp3Iy1pydurcm5iwk2JqpgR1Nv\\\\\\\/4Sd9gLeoa1c3SX9SK7HhwyarRGoaave+fFSvHpjftOOhjyuhcRCpzDtVQnGLrLL4tzE5S\\\\\\\/bhTTEkRSZJFpIiJPQ5bfXPX0h4lR83HD8xisMpNFi+hRjx4L1TMCPdTiw5uY1l57cV\\\\\\\/0I7fxPaLCinYdzoO0meWRFkrSnXcqZHg78bIDgX6Dt5VbWHpVAmBiJnIhTHDFa0iL1TYn63nb8J9lRjfOVvjnvil+bisG9SAuY9\\\\\\\/KGpKzXgqMu50UzVqZ9jpx+AWbDZD7f8M35epONT10YGYCEFqsad7htrxyVeauC3aieKN60fMV4Co54HPCe7cE8M\\\\\\\/9ycgM6lXydiqgl+0eaHd7hXDP1PwW4v6W49tqrNYmyhU8ufCNL\\\\\\\/Wj8lW7BBzlv3RzjibcystjoImkhS48yIA5tIM3MpFCLPo1HkgMY4NzJzME9Bkqe6t2GVoWoTxE+hcCPT1M5LrVbaKZArIHaY95zV8bm2AXYtq+HnZEYG\\\\\\\/V2a1VdxlEi5tXUpVYmcnOdSDxGtWaUaaTOnMvK0BUG9wfEb+wi8aWtauioZ6PMIn8Xu+qTeXpd8mTdnZ\\\\\\\/noKFxs\\\\\\\/dk3HAvLuPb4GNIpWVIeiRM5V8XE0cthf4RI3bq6rJ9nC4wfR4\\\\\\\/wy2SN22XaiMcCzHgOR3PmfM9kFpd49DZ7+0XhljfqJ8uEFv3vUYuXNSrhCYOgw6rwD4Yx89y84J3j2v\\\\\\\/eCIf32e6VYbc\\\\\\\/nplGIsGk8Rv0+Q8RM5iHs9fF9lmF7WUFNHk+7HepRYXZY+vOQQe9GK2XVfvvBVNKGIT0LKpNhOx5s3bjuj6Cw74Njj+pR0iCa9WW6ixKaSvggVjApAX6PaN0uzEcRUFoy4s0q5YMQlFjrXdtivszSihBtVskMA2w6XV1WNAV\\\\\\\/PXnzstmc1l\\\\\\\/yOkdnyOMjRTnCq+UBr6UednQyxbCQK8epthuyYiINHYJn6wMo50v+TASjtsafxlR0nkWm5GAQqZNKUg6HZBuppLrh8BGi\\\\\\\/vfYrRhmGec2zfMLUur8VBtpy3xHELGlnv3rXfytEliQtquVZc+7FVZpb31xtBPgL6vsB3wNPa7sq7TQnGSBV5tW00yzDr+ivxY7vQsl5tokuR1kzecMg1kO0B6pSbdhMxxqvgW1chhSdRC8GWZ4e5Piuy7An\\\\\\\/DaA\\\\\\\/6t9vEXG2DMPEc+RQDxn7cIkLyeU4WsA1ecKyuIYu4K6iRX52Ap1goqIGc3iv+01AMi18r1fOnfAILX0L6VsjZFImNbo7fHdp\\\\\\\/pG0Nv3j\\\\\\\/1vqZN\\\\\\\/xqnzD9MawmEq+LujM8OnZ1LSVeeg+IVgfCVWFTkaIbKrG5e3mHWDl6E6w3ZyVj4+NMmdo3zuzVTYU5qv+RfkTOKErIeLfYt0pzaR8rFgbYPx3nqOyD0pxzsAygXNZPPmJqbdzOf0L+r2+gpWMkaN\\\\\\\/w\\\\\\\/BdxUxrW0Q6gn0FDqkZQx5m3j5b5d8UG1iQpq8qsjawy\\\\\\\/6GLYAxNBkkRdxhmRRrQSLbfloXt7YxKvsIEuJsvVH0DPI09hjw1bjtHY4lpMI05Mpf5CS3oPakU6Ze0RmC4NVxo36qeY1J1Tg8gkQj0RXJqlbK7Tjgmkzi8fUfzp6Mx1nYXvVCXnugJonaR80IagmhKd0O760R4Rz52N1GuEAPkDSJNt6jJvTBjAmvu\\\\\\\/gpnl1vdwuusqaUbwrSb\\\\\\\/20RzBn7W8sT0b6HH+KgwgaCdDwzn+Oc8trmj6ZQsFEbw8fQ5izsxn5YmGTY1ssROnR\\\\\\\/hZ0biLtmO3HW8V41J+U0F2LNUh9VdHvSfl5UW2802OqNyicaNXCjO70cVYaMxWZr72wUNHrTbsXJkQrMBuS3qVuPiiBtwJ3d9tIKJEu2qK0jdkRRDxlZ+9QHCH3orNM+Oy4nHt6YgBIMewsdu8b8yuEBAABkyc2uYrAJ2B5oE2xaeeTl\\\\\\\/s3pQEk8RYXoliuyL8CJh747Z6bekcrDNkIN\\\\\\\/s+gcpzg6PCBtEdvdCqCS3v6R0ZOtdehvsR4ZL+z+1Hrc7vDAel5PvjXV8ZxNNTjszeYMyDTt0zo9LvRmgfQukT5r8XrenWiZezJFxNRruOJBSxdHLYW4b1k9PJWkrgvunHX0Ii2N\\\\\\\/cZZh1sm5Ojn9nJHyJDeBMXpZxYDxNmj2F8lZ69XuncmKsHIMSQyvDPOeVEbGPDBKFAiVgVjYEqzK4Moq9i422ZWG49W98Iqq6JGWUNNUsqG0MXk3AH2TJNEuVmrbUEEBAk2M49hfRQiZMTUE8Z9jRMh001tTKOi9X0CxC3f81c1aIxDD2zezcJv7ig5tDucyKdMNo35KaiMcPfhNy4PXIL63bOadT\\\\\\\/6R4CtQS2tFG16pxve7Vh4UhroaqpuCHDwO8pLyfrDpk6nRD162z6LKIgVyvDw4bnhc8y9XsP2kDm4hJ7QWU0H91YMGcNoQuUCB2cKv+XjV8a1rsSXdodx4PowtuVd4C00HMY3e9bGJGh0rvlhbbpu+qLYZ2nTTJYRnyJYgaAJWQcAK1GhNoboqdLHFyyaFlO7KfyPKNjBzZg44oWKIhrCUkHvs77XsokOyH4iyIPSTQtIAlaCj+p4sYzkv7Kv5FGEXw3+WX2JHmFjXSLEmnQ2d074g0G0eZ2H4js8SaK5gB0yXkWlptDwIlXb4c3QWI+1+ZvF9xVW2h+mfkxSZPPjsFMfgYi0J\\\\\\\/QnX7O0nsTT6x945lYY5LCSXNjatU1vJ9xzQCFCa0XGxCRdVlLwpzwPz1efd5cFgDhBL26+T0WKJJHgspONghr2eJNmKnX9wta3Lj3Q24iMp\\\\\\\/Dnj9l1+sDtwEWdtltQPHRo6Zdf0HPXcJgoLNgUS+08vsnZ5VNL2IQjx8PxaupqXoLqEp2msNKN\\\\\\\/7y2C+\\\\\\\/iIHDHCsk4jyiSwuxdwCXiBN2R5XqFre8jMb7EQNTxHwwLZ80+qV4DuOmZ1rwaEhCnkH0yuJcc5aFH08zNzxw5Bc6EDHj+tmWQ7bZB1r9XtY9GGh2w+U469jRhw7uWoBORcx2p2j\\\\\\\/1SAaEhK2gUEVK4wUANfva88FoxfYyYHrXmPA1k49x+MX52lZ4LUL96F1c2GzAW8UTDq\\\\\\\/yo973Pu1gFvamOvrScyLA9yg9lW4mI9kITtQBZw0HgcT00Zl7Wgg8vLx+urOM\\\\\\\/s7qoHiEN+ZbnYhWJYsat5uunttopaeZR57j5kRRcmhhDpjvhxp0IRunQcSlDWdVWthQZTXNWLra+nc41gHuDxdjk0qL0E3k\\\\\\\/CIFQezxK3SWvRsm7Er+4EEtmYhce+3Pj7kuta76UZV\\\\\\\/DVmhtmFNMaImIxOW29dfKZgnMsXUQt88TYlnyS6LMy4pn\\\\\\\/xR1e1+VayJu1PLzyf8Y+1LSU4o4oug6innmr6G3ajVuFDedFahfCr\\\\\\\/3Cdm5QZ5qvdvKj4sVFFD30nV\\\\\\\/yqL9AxNnlHaY+QIXw5e9l1lZ5dNKI0W9\\\\\\\/PZUJSydLjAhwshTRP5ZN3M4zYx4HOghX+E9H6+gFrvO29nqNk\\\\\\\/nBzFrM5WCqpU4cHDNsiUpXS8XUmtIFXtRTs8luPeLdQDMtiRz8HL39qqSqa4aDHy66FPSKAK9WKXFPYnr5NGRIuuDl3UcRhAr3A9fQKjjf5uO5kfRpWMKDDMGE7sTH2fiETDCnR02ii8SdkITsn0XjLJNHwhOh\\\\\\\/Jk0+04RgS6Ua8iA5dQIcxpj8v8WhYP4wkYoWmoXYsPxzcwl6cG5eeHp1VaNkZApKpXJu7PKso6iHZg0dqyC9eqLYpMM21f71xW03BZWDeyI7TceFsBAryO46BLtNHqtdnWmB0D8haRsv0iOAmZtR32lXHqjwQBOldDY+nh3FGzTkrL\\\\\\\/UXbBoXyiV40fSISgnI0J5YOfLml4j43oh2LllhAinoOmc+FU8YrIwrlTArsZxweAKTKZlxE5roGm0gkNvj0LrojMvFh7I9q7LJWZ4qeleYbm9yUgcJZDV2hUji71lPKQl61ehbOy+\\\\\\\/gVJjOC\\\\\\\/ysNZJGFBbAtASsa5OrrRHRL4aZLW8ko6ZR3U+xdKclfz9C\\\\\\\/5R3cTRi4vU0tWPuvr7E\\\\\\\/k0ApRQs83AETwORjglHOMoFmG9YI+\\\\\\\/ZTw3fbbPlyLoUnWTMFnX9dMn61N\\\\\\\/S8AWqrgvPfj7J5fv160\\\\\\\/Opw3m4BWzIokAwELzlYAlDrvOjHSlqbgfleqkpVTN5jiCxjnUF8Nt8ftkVkGPfhb6oHinuWkzBhpkk9hmFS+uE7GiMmblmEO2RtkBoWIZB9P8q+2K88bUzJjSixlJC+MKXid7UeVbjA4rFApdzmwvBs\\\\\\\/TX9ZZdmIeQEvjafuBgIwFGhL7QIHISrnMF7ejXna\\\\\\\/GLL3\\\\\\\/dfaV8MI8m8Tic3KdAOSyIdnKYvBngNKf9X3Yn29VPV34qdXCCeuyN1JzcmiWKuv8fB+E44x8HQ+cExt1mIQm7zLlRcxqdMAr+xhFeCXvSEsbi+IqErzYeF5wBuwZLy3PLmxHMRrV+ix3AuEYaQM29VVXWpZbnCTS+IAErid6izA67nkyXd042jDojbHyknrJPzxW1pQG56c5oLV1qXVhh+o21GgVZpao8VJCQ9QHx7R\\\\\\\/XaAPDzBh8RQl6Pvrxm8UXGE+T6bI1Shee5gEYP7AeLo4bqNSbvDRo7R6DVVAKSfCAJoAj05x3XS2pLNNYWlAkNW\\\\\\\/6OxsaGjdOBsmXu3WjN7ISbE\\\\\\\/07RYCbpLIVgc\\\\\\\/DhskFrA38i9G9lrrU+H67MFZ00XFx2YusZNVpMaoRVgs8HPr2W90CsC2rEqfZRri5o51cJh9idm6pMgJ+zufU9SLjW6C1+amvu7hV44wzdzSQV1phnojWy15P3GBERZztcz0dLqswjhp9AkokzOWUE4hWU1ObJAHZV3ff\\\\\\\/incR79qI8qUYDWXuoSyAHb3xs\\\\\\\/bl8PfXTJc6vDsbRCAl7XRCldHFXw8KnaPBmc505K5Vn5wh1pioNpJxjM4kfMjKTXzQw9\\\\\\\/P8ZYfGapKkrAnhTuacLNlqJ0Pg4bDcIHUWBSs2BRaQbGl+QKM+fjemCsnpVbSXxnm3IXi9az\\\\\\\/wtqDSMaSKvJPnqmxmSceQ3R88mYcMQvTOcWvroQho6RJoUOipXZRfw3uu7FaDVA8ci8wV+1PFW1OUZKGGCzlbbrCeLf\\\\\\\/eeevweMqnI7SAvLdFWHatpbae4ME1+j9IIaXTV\\\\\\\/XSEJAnqTyJ2Lvi3NuncLMuPJtcPDHz7P75NktGyOrRpEPTySzlyVTN7O4bA6YyRaPHCG9wmgRdLckl763ZxjYDFvT+VmjOPjmBGygZ\\\\\\\/DYkrW4wlnXn7T9\\\\\\\/J0ZOgcFgfwRDG8WmJPTge35sbWZ3HENYMd8EM1zI7PccDE8knpTGfVjHHx4ZyZkJ5GvrSaFK8KpmS8y+cwlU29tFZqcIW26nr0Rw\\\\\\\/OmnUQ5AQx8oPVknCvCFPZg7Hw0HK3A34qTjswJxjCdssLLrGW0B2EE9jayZ3mvFPydE4M9hOH1wNhqT4bGq7MGHRdqsrbZQYPgj9VcILwRb1cNgYJizk4WtvtZ5SHAIf9OCM5w4D6Wu5fr7UKpIYAWUUgMVz\\\\\\\/QzjJ59p7xv1SDlIQNhLumUOu0PpivAhiEzsiJY+Q70hEvb8Pg\\\\\\\/cWYUAhbQKWkMId5cWhMKxeAdcBjNB1wHrLETPRFXdq0iV2WhWyAerYcNzJ2rYEhOPIrAbXOiLN0E3mV5eiVnoS78j\\\\\\\/HAwGGbTnNQLHNt8HvZw3BlyxsiVtn\\\\\\\/0rlpbrv07MoLBPH0zccDLdWwpyRNkqObJIqvWqHyoPS9nZKBaBqSPpHoAAlV5CcXxab7h6BLQ3Mq1u22IS7d+ufT+yhBB0faGBnTZE+I+mRcxLbd\\\\\\\/u4JvPPcjkJ4UQCsY3i6UQng0WtIQK7Fw3+1D2ZVtZlwHn2LuwOHEMilIzOcqBGimfU9uSex1t22HV+4Pt3BcX0HEYgzK97fSLH\\\\\\\/xkS\\\\\\\/jwffA1yNWV72Os7ojgRG5JK\\\\\\\/fNNxx28y+iOr+\\\\\\\/V6xDW3pIf1XKq2S8BW8dMOn7+0Dt3KeFEQHc0mfiPH6zA+1G5AP3t6z1tu8MwnzEXbMTXMwR19uIXRyzngSFJM67wbEqJoqsMPVYecJ58PZdBJ5NDBxRhrOK5OmjBclAIscXZhLpL3N7hMdT494pQqtM9rSi1HB+irH8vkmCm0m0J1aHsLD05wV6rVh2QdGeY14tR+HGyF8SWlP6U8qqq7CdjkBQNb8N8japJbg8EWh7KyZ3Odp29sSN4zS3Jr9NWEK8DRkJ7VVCxmcv+DBrBKaTonXXuVVzO32KxHEBw55qqWMU\\\\\\\/3I8o73hrQspaCeb4bYiTKKY2VYyaGSCOJsS6oF0GoMv47EIOJ\\\\\\\/37bOPxY8XYaAdN9poUZFoEAoCI4f7Igs0vQLN\\\\\\\/HwU8rsyvAVAkFdpnxul88Hxv06vp2h\\\\\\\/VBqhsLZpY2L6GJXxksx1+E5K65iohPNzJ1Uf\\\\\\\/0qZE5wGvCP15DUXwoBPgLvb6Y5XoQ0+sZuZxoveGP2ET\\\\\\\/vSwVdw1X27eNGlHLG\\\\\\\/8Cs0ia2E\\\\\\\/ss0tIhP7BD9JD4dw7Y7RlLpHJwcUfcKmXLzjgMEq4+KH6+k88iiBhHTkOnN1CjeMI2JfqexCpXZ0Yah0IWcLTpzS\\\\\\\/Tg1ltphuifVkMmhGUihNTuXAEnhsfjO\\\\\\\/gicA69VxPXVWpZcKQoRht6k+8WUDaCqZZsvDVsRuGg1P47wsFG2622fdd8a5FIDOlieWIOMkvJxyhsBPINH0UVkzislWSPmfCSpE+QhI5QQkLX6e82vOy+QoL41vFgjqEOF3zndFWNPPBbFWGSvjbnnY9iqF+xCMPGs4s3ORbuqCl+5Q0AG+zhztqofS3mBjosfkCzV0dWP\\\\\\\/3JndNQ12OgGl3dInvwrp41TWiEVSwN8ofG3MSP1V3zz4F6kIzTon9\\\\\\\/9pY43vuuowgFiWSYxMccdy3wGU2IEMh\\\\\\\/hzZ+JMNE4E4s+p16alCaG4nX3KjZ676BJszM\\\\\\\/LutTJb50wHhxfrhjqhGtcHe5WL3x1oQ3122DBLhISL89oLS3rIaLq4\\\\\\\/gcn0212YTxF+GXSNqpAXbrZT\\\\\\\/nm231MAZ0N3ImiPZKfnvJEqJ71Cq8VUdJz9v7uv\\\\\\\/mG3w6B4oRIzYd91Bu5mEm8kRlwpBNUpWPVh4zXzQ5hpx7Xiip4hX6F53XEiJNxBHtG5TfYDdCp4o5THaq\\\\\\\/HWyg5qkcpa+pLrmFThHRloUN6hHxtrnZK1iOnjkKP+K4DKqAsYcDXgCnsez4IT3kkzMgwxjdXXdJN33CfxP9jgq\\\\\\\/SuzGKk5PaoqUB\\\\\\\/oa2j9PFXU5gZ4lyLy96ayeYzbW4eKHTuhgACyJ6uROGXtXscCWTVP6yUxiBl7W\\\\\\\/h13a2F5v+kFRzJhgZBSVEg96eNxBa6+fx8JBTUEIR56LMpUgGocsY5IhP\\\\\\\/WYME63IUH6T57qSoScaLnc9XqYslsqX3pZvRf9vmtk6QnfTAaAmxrGuALIDSZiCkMrnxN39gHnT0ixCt++OBC+UCRgAZPUl\\\\\\\/FjSyQ32E+SMk4B1BoFiY8zaFopXvE6184Pe1H9Km3RTn+jEdRKoVcsh5A6x71V+O4TzXN52ZGReivQruZp2G+IQYVWTNnAejwfuGaAiq0SqNf8qvppVhBzFViiYonRHMkbAVotHxZ4v\\\\\\\/sdolrQJhwjxwazSbfMYJGqZPs9vwR9t7Bo9j+tPW+00WNMuGJOOFCgv7m5PDRgdjBBWfqEQq9trWDFGM+eAzHqJEX4G2bbfeb4bMRryfEByJPVZt7Tm0nV0VEA1BZvUJ+KcF8DwV2NHv9AvgBWEx2SWQrPWfGyPY57RR9v4X8kwgm4vaJQuV9Z1A3N\\\\\\\/emwz5ghPCAl\\\\\\\/LwMPWF73OINxOWkdQ\\\\\\\/l+YUqkWpsPfh64XgKnICZN3xOtlSWVT+sInD8RvjkZOw44Qd4x\\\\\\\/sVOBWCln0KK9rs9cRi7bqjODp10JwaE6NLhHxZ1TyCGYcvvsBcTWDdToMrSfrSWekRG+5SEwR5jLEOp8+qOFbf74\\\\\\\/4869pXfV5yOZjLo\\\\\\\/C+9aDd\\\\\\\/8lvKjaKqutPS3ZKm2JRLOZAi5y3hWGhixt184ZvWHw+z4lxb366+HrViFut7suqqZY\\\\\\\/ipHEu3jHywl3+bbwxDgnSU6qU3TKLHJpLWAP8yMcq1Bdp8s6qzgJbyMqFBCQFAkYCn1EotDv8sSpmgRFe6ib+SVQeaee0S+9hHBZ2cZv3yY8qvoFcsswxkDHkhS031LNHrqmoCqjRr9iq8sQyKpw7wgSB7uFmefAVk\\\\\\\/H4fDC\\\\\\\/nyDv9vITZqDAGACCOMzUSJMzwqQZMGDJDfokNmR0p2fbkC6FFlShkoQy5UWUYU8q5BZgZrN8w\\\\\\\/jW3xKuCQH7nu0TkMjRmSDc41OISRJbtf6lzCBhADcgQBP8sv5ESb4XSkIBwHVRwLua0GfDgeatsdsaRjgKwDh1wsWYttePp0brbHGdtBcyz93zGRe+m\\\\\\\/y3vK7l9cdPNPV3QRaW92Z8NN3X7BrzBDqKYraWlRA7lnnon6ar03CUHkDE8dNmph\\\\\\\/AV65jRGbRuStzRs5Y2FPWreTQFU3tPIF+QrYRsRuRkNnfoiyArjjH5NMY9bK0PNnkFk5i1rZTTOq+FUHADEpTEk2g6CMOjWmkP2FLu+SNEbM3iA418mahn676UP+d\\\\\\\/N4AaVrvsTkZs78yiWcLjaMJ3fI6dZRAdfmcKvuXZUtnfAKf5cflmgpLnV71k4oHx9BGhiM1cTOWc7lLIiVk4hZ4+s2+zV8e3rWQnOtp1380syENq764+dCJRENpUw6oEDV\\\\\\\/vSH0+fT9GWQ6bIYDtrCzcsqoieinnAMEnvSsRP6YJqDl8CzGQXlhP4k+AkguNo3Y62YfxgB0NbLDO+Dt4sYFQhLmakZt2QUzJBV21HprwTf9j7BZBcJX2OWesvlZFQ75tNXJB55uu6Umw3jl1sFAo+F6ayr4SapLh\\\\\\\/WPoCOCBf3qR8LoUspPUxLq0xPqezd\\\\\\\/1C6SWByrLOgcrzsWz0o5jxwZ2GsOe+Y4Zrlh9gUXt3pf87GFwv9s5eSHRyIIQaIyQ7HTgiWoeraFYb+aUwjP9tqqJkMxXOzt4k8k4s8chsPR3xEWnaJTNB5QoNeU68etwRcmEy+csuIJCs7U5xTrzE37Rymv0orO51rR4DOJmV0F4jMuBxzTMyRWXPLaLjv+PRYG3Mz7MOxV7gxH6Q94VWnnE4vl8aKDvMEJgpX0rpsmLCUdXRBTdCwkZT3dHDeJDMvrjpF0LzU6oPmzKlYdkt7z\\\\\\\/fQrNbSjyVU0\\\\\\\/8WbyPKPvV8PlfOX\\\\\\\/GuT8rseOiUQ7uBvqcOTMZBM5PS6yXBevmXHK5AczRUfTyeHXh\\\\\\\/MgM+tp9BsMPdSqKIBlZ9nN38kll\\\\\\\/JHP0xhS5kAmdBe2iTMnq4RPsIRMSJFqb1hb0Tj\\\\\\\/fWe8Xt\\\\\\\/Pej3y5LU4JXUEndF2GRg6f5+bVBhXohTRiOMqcEfzuP\\\\\\\/oPJ7ceyT\\\\\\\/9hkATsvzgsBE01rkG0+C3NUiw5EcbDO9NQiNcFDsHN2jxbtxe\\\\\\\/HnqYSCgrRbOQwFZkaiQ0LdYLH6h1ioQ8\\\\\\\/0yySmaeKNt7qriIHS\\\\\\\/ADLElWW3rxlNNOda0wLsEEgoT51+vyVBgOWVV73\\\\\\\/J0oM44bkLmuZ7eZ7Fq4QLrLoUKijjUX5a3HkihL7STV2NWprD6sTBrI5qbWoIRgTCVqgj7HwRGPwREZLJxh2di5VW809ZGSIYzV3\\\\\\\/GDqvO\\\\\\\/\\\\\\\/PgKuzc1EC6E0cD28bym0F0Z0eSszburGHuRBwL9CAcwEpb5tzgxMri7JQgz2zdOlYtcWPQ4tFkpRw0QLYrIZjcPMsIsW5td\\\\\\\/aIc9Ue9J9\\\\\\\/4Sfj+PdOhxD5eUaw9Sqre\\\\\\\/mVIy8iZOng7GnNgZ9fpYBCaPRPDqkYlFnurBgvXotfbMbMhtZBeymYuszB4etk+KP1F8a0IlAotPohu7yrRU+b28FC64raXd0\\\\\\\/xdG8w+M7g3j\\\\\\\/KUa35SCL\\\\\\\/4uPdjFG+slhUJELaNfSMXNAJ+zEyKc+mZRuYG8lgQKTDspCOvkGgXed\\\\\\\/86msfXEw87vqNijIZUH0UPPH6vGLjEZIpW7IwSqYWJAjmnF3eOlrBbR4Flq7aQsTH61E+xfN4pMVYhGTzy76MQ0JYxmBHom6WRk9j8cbu34iooTA00NtW5neQNBJ9rzdf8Q7ejxTLpKG1lVlDiyrWjlL2LNN5M+Ke++o5oXIknMuIpBBWpZFH1AiiVQrJWhWfLA9Z1R9btOlzdTWLoE86413e3QqrQZURWm+UFZrRIhOcTi\\\\\\\/Xrx\\\\\\\/30skiRerK9ZdyULAgEdgWcbCd8LUuZmOBJmeuKMqAv3ceoE+S32mI3zkijMaPkPPujRwxTLDb7pBigdobliJrtz0AhW74o5POny18on5BkYQ7CygXOY1L1ESCLr5moQIzqOZ5X\\\\\\\/Sde2vANtl4Vwy1be8JJy79bRHbqYj02a8\\\\\\\/KIGIISDoRJxIScLSuKxC7QvP9QIz4fBT30iM8MumhBdgBlcfViBM8yrthoAMMthlRuNvOiD2O\\\\\\\/+PO91RM8pTMa6u+kPwA0GVZLYOi3CayCZiGHGwFYwZZBYE3+yXennDIZiX87mMd3bM9XPQB9iRmk5MWe+\\\\\\\/Ocd82wrDWkJlouI0pDL7kHIEfNO9PfvK9pRLyxVJov3UqalAWh+tFxmWfCpVoyh2Kl1nZmO8az+onMqHiLFJKZHDCS\\\\\\\/RCHS4CGf4zrtrYTGW9S2EoHZcipekSeyM5cR5\\\\\\\/HjQQb+KJgqg+6h1fi8\\\\\\\/9Nd0wrr3xuO2s2KwngNDoZeUa41QQCntkmLmF9w\\\\\\\/eswqIVa5VkONSDJssxC7+BUME8hYUR0amA7T+l+kwi+h91asaq1mMWZtCUJF16mePQDea4YXwa3udwzEdW9ztVrNlefYjUEKkVsPNaluTsW00l0bA9VA7nmf\\\\\\\/OUyARhWcieOJAgFsT0g9JlQpKVC44pzwfQ+LDtVy8o\\\\\\\/\\\\\\\/Er1b7sJzNiHH2vKTdzHBICPe1SHmFE4elwWTsTcGHaNFBfZb5p+kg7gc+erLCYpmaqBg527cI3zgsGckSP4CIie3QnVYQIgopWG7Jd40Vw0C2mlf6p80LXCPValksoAVCOYJ7Xx2YwSnQw2cpPO+DbCU3BRntnn2n0LYUx6q5YUse3rTLB8txbzMtIBVi1KFjIZRqx4lOY7M5JAP\\\\\\\/989eHMgKutG7clpv\\\\\\\/fCQ08mlyxUPzQdYTlFW\\\\\\\/gGnDHQEoHV4VbAbkano\\\\\\\/LfdlIlYcHBXmVYOsSQVrs++i9JGRNyHXeO1NffNpUEAYLtxcWSD3UCK4FsK+aTcD2YqYxEgqR0kVRhyKekLur5k5O1ITzU7BDnpnz7Pot+iZFUNQ6FXs80t775tqcvkosxJWrz4P1jlBRuf8nmFx8LNzppYRCvuOFVoDUwDkxZ1oSFu\\\\\\\/lCWwMmjrwjnOyaf4IetTOIFPlxGwXbjcIl+urZ5L2jBI0NPDTpGqAGgtOlvM+rqsQK3bJglDS\\\\\\\/7IT0WZDXYLlVCgvCriLw1zXAIEpr8MXofZJCgtM\\\\\\\/b9Jv2f6ojogGE9O03hwPcUFQtqM5Hr+8M1pCs4oATYN0jZdPvv8n5ObInDtNmzIrAMtW1HpJXtS1QalVtnqLxMWxyTSuY1+8\\\\\\\/Mt4PKYtcv86Pxq9MlQ1kCHzhKPpOQKM9utqp9+2par+iPCTb578QyVWU\\\\\\\/CDmkK1zdjt9mUbLbIXnkzcteqkN5BkyYJ3Fdgz9bpsvTr\\\\\\\/d3u3c6sdTpVfDOt1au4n1H4sRr5uF2tw361ZthqcFPsHCbB37GUUdt8oi8UXdYszalaBMsZivBgFJlvUkkQaShGwC\\\\\\\/i2huiZDkOdE5wpeLYDLIrsjC81FYc9+n6fPf1vCusJCRIXsqL1IY9LS+rFXcbJSIoL7WEffAg+gdrspWnW1y6o2Np\\\\\\\/nVma0tIPNUDqK+vTgR3dQIIIgJAAErPo2gOCtYG\\\\\\\/9Vf87QM7rn1R1pROG8QaJIC8ut4cBE6IHSoAMjX8by7jLsiKEe5Et5k+C3TlReRm5PNkj6t4BUPfx4S0DBpPOL2pJu+D8xwlKf5KLC5UL\\\\\\\/OUms7X32YXn396czaOFxur\\\\\\\/\\\\\\\/z9xKf0GbWwlscOyCqRxob5iF3LEgVgdRRYrOufUkEtBFe4\\\\\\\/K8If3ItHF\\\\\\\/+5k5Gro3\\\\\\\/lHP2kDXSUduNN9nZgTrjtREYQbY8oyFWyNcc8h8qhnWXEQBBG03xJAA0oCocs3+3s0iAEbqLY5Jy15n+96vUGDga6bwX3Wk9J+1pJVREdiQKhiahtaRyTOJQ3rqnKFZjHGRFnKcBrpP4\\\\\\\/ZrUlBrvn\\\\\\\/Mu\\\\\\\/TQRYQt9l\\\\\\\/e0nWfACTS9FDh88oxQsGpSFJ7QPgh0nxxbr3c\\\\\\\/X0W52FaSK\\\\\\\/OWqA3Fw315w98TPEnAjj5aYIGs869qEZIC1+rk++M8Jj58q04VI9b9cOOYuQd\\\\\\\/pQ\\\\\\\/AFShfC9wGliQt8o8bo9r7\\\\\\\/Mc1te0+gY2XNBcjJeqjcF0U8on1eHDNy0EWzmBqGV9lXkfTyESihEs0MzigWYtay2ssR6oYuEaQ2lCUteUWfZDeENjHNasebYBSpS2elJf+8xlc\\\\\\\/EItZ3ZsA7dQPykKdWdhSOvfd4aAJzFzblskEN7rGCJlFkDE9hXXpnpchBOlHkPPaIUIueYQbTuwhF0xemYxKxOIexpMd\\\\\\\/XtcCYLm2okOQjcKrF2\\\\\\\/wtWnSTkovTRRFjknuhJUMLq\\\\\\\/3QLJhoKlKPI4221dF0qiZ6Abm7kME\\\\\\\/p\\\\\\\/z4R8CM4x8f\\\\\\\/d9JYWVvuTSEfWf6bgFsR6JdAIqTjReCLmPFxWL2Zk2s+n6O7VmvmjE17Exd2ck1qrzp8j0mPnTPuzZSk2ZHPJaIj2FehMBF+IttvTN7JA9FUxh0AfUqxrHYGBJASY89NwS3kwa1QZd5ReMZRT3fuZXIrcM1pSZlCtZnRutJ9xap8PK7mTC91f0SKHALgWRh3DxpwPWyDEKvwZKPJjIpfs3x4ju3MGc7BHnhp6+cR9V5Pnsq3rB+UsBwYqjIpSFkKIlYmxk+SXi0u1SE7t+Ox1pvR0P3qUEpQru+eWSORHaRBaeIMnSLwQ2Gnri1OJkafArfU1xxtit\\\\\\\/YgnjmhBFQEc6\\\\\\\/A4XDEyUiXbrAXDDkBFuLmkj4kbDD+oX28FKEFvTgye4M1Fer7sH8BzSQX+449KeJE5APmrqZ+S92V8CLvJcgGS6+n0O7UYiE087RvfPJAhVXEvDEZotmR5gexdh41C1iNVMB\\\\\\\/3KzHHer\\\\\\\/\\\\\\\/zYXfUE5MqOXlsqUTRydVFI2ocT+bz\\\\\\\/bmvPbxQ6WUZ+q9mpRfYSJd+pFGKMOg8eGSCWKXVPHqUKqCDWg1dYdcZwGiYhwEBFpW32nd1MtNuOAc1o150Dg\\\\\\\/tZPSymy5XGTFBeW6Xwdv\\\\\\\/JJvgTaH7SX8OwsoOKrotztztqC4UGAHj4Px3X1ydn0s2ZJYqxIObCiz5kjSxUuJQSSZ7O0chx7E8fR4J8vXfs4IrDtydhrY6dfqMxinFQQpOiDo6w8XdHAYXxKcIksVUQZ6JYzA6Eyrqxo9xz3eEcGji5HEkeVtLJgyG4HI\\\\\\\/GW7NtgtldGqm\\\\\\\/lTyd2CKzV2b4oQRmd7BIzJP3xwm5u1tP+MA3CBndAkCI2jNckA5EuQdDE8T+LhekSLCUtsAvXta379PKTzjr71h7BfGONJOy\\\\\\\/zBxGFzK4djmNie8t0bjQPwskqaybIBCAzmRC60G6d1x5XIwHoVcG\\\\\\\/TPF4SU2adA4CWq381buNQk0Mlqd9lyuWYalCIjXmJHxt2PteMDCFa2u2KC1riq6jWrfJChWUWvoFXyFJzxamLM\\\\\\\/imWHFs03kzGpb0+pXQzABD+3UBO2k5XDPBVUSOR2B\\\\\\\/8WYniobOP4M3SNgfXbitOYUb\\\\\\\/JIdNla0m19V1SqW1VyEhptPPi6G0gLdcwC\\\\\\\/aMNp51fzurGj6m44ZAxlXFhW9GMkg315EAc5x3yy3jfoVhu8e6kE3\\\\\\\/mkXMPlNMTi59soMxYOxSKJ7\\\\\\\/IdJNUFmyvZv2VAW41weoZRC3IJSPz0WK1zxzuamrUy1W32yOqabhogpuWISkp3j1FIyXJU3TTv4Nova6756\\\\\\\/GAooXbXEfuGAEpG+dThtclvILcJElCb3kwk\\\\\\\/18Yr5gacW0tSFpVOPRJ5GQ3ZD4iOvPBB8G5nVrYHovY8tMxVteieG9DHRC6IoNE8uiFxydYnsIye9NjYBFgSoLCfnjgSFs+VdDcDlvftFNzbxZk0hegaInHkfA7RRASUarojrCoEpv+NhKTekpyiqD0X0UOELDlB8eUxPgHn9LMzRZHYR7BtPQDvGqR0ox2kcake28jBnTWc3eazjoq9wlVHHtArjEHwfco2rgt7TOwBPtcjI58BQFnwLNt8TYBFNT1oYdjXhP97YPtq1rcdc5UF7gEn1IqHwPBZvdD+b\\\\\\\/xcceW4C1UvfWFOmdXy2inEj\\\\\\\/I4noQldBd0t\\\\\\\/0UFrsGoYHM86rxDMzb+sDOZ52Q8wT8Piuue1oGBcRc1EZAscdg5cBW6KcasZEFOGMcugRGA02c1mb35jVobyOLOPTgfyVD2q2z5DEgM2v8Lml3esRRStks0IiAuCuOIVnnM5VQnqkmwthmaA6Ukl21KJefI58B\\\\\\\/KCc\\\\\\\/6gKMcRwtfYUE+IO+5Ilrvb4JKDkSVCETtbxYF\\\\\\\/2l6x1WYdhDzktNTfdVtkiYXtxNvXmIKBsnznE96eeD8clI91tEu+5JPwS6n7Ewyl4TZ9MfQk7\\\\\\\/5j17OJAqmQDDxKFHvz3lyZUfetYL3u5bRaK4WrgJHs6I8Nrcx0+mI2Iauj4lfERr2hY3hQP9Vg\\\\\\\/zzZaCaFN8Cclnu+BtSfF7oe7Te9kQOjMQBj3CLgW7uhB6AyvxHPBS7zLDvn65c+yVyEtieLjDp3O5NEBYr\\\\\\\/9wDSY1dgI9OnwxO+RL1qV1C\\\\\\\/MUuHRoooH09w2CX8iFXCttoM1d+ZMl4yGVI5fvQ4XUE7Kw3GP65GJVYqOGchmSgytbmZ2jcG+N2MXH7Tw6Q+JRFNVwEMrzEdBvzthAiih1NMe1TOomuSGLNL6oNwi0wi4j5N+CXIAjhYnouFYUbmAtc7lc4vwar8pudWB4RET00D4RLFRrCg1yt91JXlQltPXQwW0KJ+jm1HOtyZsfsv8kph0D1m4M51qBMgcLaOLyvRDP0J0qADYI2nQQyEvhRkPc2ZDdCldZRia9njKbQ9ENsbFPZCQ3L+LXiH0Vs0pNKh8drOLnmzxRxuPsQK78zqOMmx\\\\\\\/fAkxK7lOmFe8r59Taxg5PlQiNTjUJW21qoJb61JB6ODGqlk9GOEjwt1qiLx0nKkY86UwLSD0fnKFPZhKeO71MOaNDKPRTTjEr4dT6xIZMj07IpSydGVdE7KqqoVVCXQ5lFytDx8pJf3KI68Lj+jRnIIW\\\\\\\/V5+AhcRuhuI1deXaKxZcmFbz66Uzi1Nk2aZTPuEomZfD24twyNCjaSSwu4C\\\\\\\/lVDu2jdMj0niSSzKtE5XaNLj6IYx3wLfsxFxkdXvY4ONUcRcjQAKDb0W\\\\\\\/mowJLK658nhB4lQp3OMA3GM5gRrBaDG0JJOIQQug0TeFRFerebi\\\\\\\/2vf\\\\\\\/k69\\\\\\\/4F8Gl+VY7J4AheflydwOGEFdo7FdXmw1CW967uYoBHlBfIShR0iIZnnNL0HJqZ1ojz\\\\\\\/mRvrHqQCwvBBxskTOaz5HjafQuWWDnKSZcqQlj\\\\\\\/SbwwpEbEpy\\\\\\\/I3J8yinkAWQYc7CQsk0OKRPAQg69g5EObg+aetET7mVSx5DgFddW8ZioL0DUd\\\\\\\/9JA50jUSS3a2utTCQoZaYFih4Uzg1rRKeqXJZ1ohu2FpPZUHW+Pt0f7ggeJizdTuwtuASiUWvUoVQPjQuNydg89GgwVzbb6AQPVZiljYRe8qJeOVquMF2uOKecpxMYw+2\\\\\\\/gfD\\\\\\\/CtltppN2wQmNZ8iAvIE7IZ8gnP89sWkYMQehcPbetuU2DoCA7TGwR\\\\\\\/kCoQhoitMfdRAhT+c03dxsezafRRxQouWOej2O+3nkQLbj6zMLaLGTaDv6Tpw\\\\\\\/E2jU4ZdWjahJ4UzM6WymZzJNdD5d4dq8tAwMgzsGLlqOLCotvY7pG0izmQngeUaNtvfUWFTUc0ljWwWFHDgPs3dK3mMtf0YExr72Xz+rgLboUNp4mhiy1XKLuqwknre8ybW7KQrpUsG\\\\\\\/y78O+bFyAtCyV9RCukg1KcMEaQfdM\\\\\\\/hKa9oVs27kNIcNbU3TLGv+GPY1Qpq\\\\\\\/tDokGZ\\\\\\\/l6mHZjx0XDgPzDrAQyBWWlLgt60QGciFOkNmefIYF5vwzIqy\\\\\\\/CqP8OVDrbUF6ayCqEWzU\\\\\\\/g2aNVzBs\\\\\\\/u5Wgmhp+5nK1LSVi14CDimv7Li9usYDCBzhoOxIalYOc7dv5lz7DRo9nVHUdmThHzseAab0DFUZsF0sObm7702liOlhuTR2G9+p\\\\\\\/C59RWhkUFVF8+G2UdPnwb9m99iOGE6Vdn4hdnkvz4vHHCzN2UPS+FoQR2owqBUYnHOlx3Co1mrMHOO36DNSVVV+2PgUD1yLvCJY1L53YveY7PHhILtD52aCgGMdf8\\\\\\\/\\\\\\\/zxb2cpuvy2fF7VY\\\\\\\/NFtYQTKMqLgdXy1PWUIBuWKSuBh+zF+L80pSXw8pv7mQclevbLL\\\\\\\/9Im3a\\\\\\\/6qGO+lbzK0ZF4\\\\\\\/6tFQbVq0D+eVm8aL4u9SvyulUurXxBbRpg2qiwmPUn7h1zDlnseMIG9d+Nh4LUyfcEm0VpGDA1uu4BUXGB0Xjwj7btWBdSdf++RsBo8v\\\\\\\/4R+JtvpQIsFlZ3zXH8Kqp1LXiykSF4+5BYx\\\\\\\/tiY572mzE5s1wFJEBPi7lqmOitNNjt4Re4rrbLntmeMYPFdmkR\\\\\\\/lai5OvNK0FhT+7IE+olO+R7X6MeWUV65C2NzdszS9ClqN+Y2JXDe3oeyPyUJarUcXbpWznlEyXoR4UjFG5n1Xnofest8YWg\\\\\\\/3wXCoHhYnLzgaOf6UoiO9gQye3cGDQeZrg\\\\\\\/qjkosySlvT6\\\\\\\/+dPh7GYBmwc1iBmgupOz9IEpDrZ6VXsEVedEiygCYJidvbaAI6rq+DKBwOT4B7lCsNqw8O2qkf0Mt7cXrsjOjf0bdbCBTEcPsFyM7vwtDPGb6Wwx2lQEqYrUPBpnV0zuPfbyTRmsFZtOUBpNveMoOocW7OfpFU0xwMwjccvvqGw3C6WNqj\\\\\\\/+N0H9xoDGHUmQSQsHiHnefbBH5NwIVFTBYeid9gEgDCWjQDzD2Ppe+rr7a9VAWlhtEjjBGzgun2O82PtT0VmKIXeRuE8Js4MgJZbmbdRGvZUBWwOgn3oZZ6sLpTezs1H53\\\\\\\/wxSYfWPLBk8h41URgG6klLbbeEkZDUgizxO3jf0DIi3f8S1pET1Bf7KayaYJ0RvXskG6YU5T42dQvCsRQ\\\\\\\/f1QWoCin2k6ZlPwgOY+j9SVgtA\\\\\\\/S8POxmFMpMiz+MyZFSs7OgPj9MN01D\\\\\\\/mD\\\\\\\/X2csIg0Ny7d3JW5uKC7Z9l7ncdx4+RyO95a6LFq2cLfV9Ki6nfl8Ah\\\\\\\/bq68qdE\\\\\\\/nm049pQJ5H\\\\\\\/Q5vGBkSHxZMEPvprQ18YniH7mGzVpCZI9XYBSBYW5HR1t+A34BvPomKh21owK92wxIa\\\\\\\/F5M5oyky3Ul+6g9sCwttu+SvlAwJNSHZFgEwX48ujnVlA+z5KCpQX2KiD21yHSE047wBusrH5uCTlMXY7sSam87PDNmUw1DQ+iI6h5Ov6zT\\\\\\\/Z4NViAHKMvzj3cQmBeycGMog6qR9rApvt9vFQ7byjawSX4LfU+hSGQdCFjwDV164pO51uFj7u9GeRtJCHU4RQvdsybQ6DVhiSRKaE30m7iAeVL1pDoKHHZFcASeEnit7CoR7MHcVCJf+lz9X4jZ9hsK61aYDM8YMn9y+NJY7B+Bdnp4c5K+T5KFpL7i+qQYpkkdjhRshTtx+OCyOxij8c1d7TrSvAYg99LuslZrOJ5AOCvf9Mwojk1PtASsKwneB1qm7DbY5x2S919GF\\\\\\\/vx573m89SM5M2thvy1Nl6lHCyLZvs2BhKA9\\\\\\\/ofEzjD4P\\\\\\\/IpHKoN9\\\\\\\/0XBHW6DA5FSoHLJ4Um9RiBoCiViqjB5hjUi9J7FHp3jWA\\\\\\\/ZS7Njm0j5kwWQXalSEWlCGwRpc1kJxQog192z+3Ayxq3QSnihaXybnvq8sqoTnWIZiE3fLIQnJMD7NLkrQL7F2k5BDKm8VCJ6m0+LNrkiJkbbvx7YLOx29EDe50VBzSEX7tYR+viYEXZhYXmkzp+PXOH8QijWe1W64JjGuob2vba8CAfbH6NiSRY8KkReWaUprpF4wcuql4t27E4QshVSiYYUJ07PTs9qMJgTbVcAPyYOtwwbWcbrqXeSAfPHiPJ3KJu+JjcU5pOGmLBsGhYtSuwOCcw0tH3PNT6XdTmBOVtrxve9oKFcc477E7WZWbF8qjZpY\\\\\\\/6LkUFotatHtH30ZPgk6+Xvk0q2mInlenxePJOJo+qwVcBDZ1PLcai49sMn8MPtdna1DGw+vuzW7fhMvenlMqNyJWWuZLF8naC08VE+JuNK10eVXsv30kuNO55brUafmmWshgNGXtPYAjHwhQVpqz2yjsMrDf67B2tBzD6loJd9YUdd6jfNE0ikD7dwa7wxkATpDI1\\\\\\\/BG3OZMeAew+5usbYIPBbCwJVOKpFjLnEJUvU29RE438\\\\\\\/+dNFWrUh5F6GTIMh56J1WkS+ms9Vw4iTf4kpqAjapujQmwFmZjvH\\\\\\\/SMXC8pgwBVdLRw7ZJ8SXX7lXXX87w3XkJQ08myKQxr1C9vb+tjheMNF\\\\\\\/0MbjIIJWkFBNxDmggXSNbjC0LIwMG9o9RRCakJ4hbFAQipOcmz4lHBqtMZkIgK0wTihPL421wFeZO3QNUIHyDJTusooolmI125pTtOuBLsd+0l52nYZPaBwPBqZxKm8T\\\\\\\/d3iCxiQt7QRDsLQXlzwtlgAAQP1qMN8cQ\\\\\\\/B4YrBqObgkTMy7ES4ESr2\\\\\\\/HLx\\\\\\\/q26KF2t8g5oVCD7n6XW6UPwROp\\\\\\\/mRKeVtYYfTZr22EoEnHPiqwiiOAOkucOM+FJBNBG\\\\\\\/DPeDANZb17wpcwX7PrLMOLYIJl2Y3HQBy15PxVl\\\\\\\/K\\\\\\\/K3skBHted39zweEhPV0XycbdhJDLqCDTDIP6V\\\\\\\/JDP2u3\\\\\\\/Pt8LPsFrTlG0VRgPDgcoGj5ZPMpVVqD0ub4earD1FK+ls+wzSM3TIUzoQyCDz2IqqGwml\\\\\\\/r8pTMgiobB9xeJ7UJDXyGeJJIZvcX6e3pFUY5j09mFIKvueZQIDkUNWQvWVV3ZajwA+yEMIv\\\\\\\/mxhN0HAUzLQiiSUJg43iBaIFcuwtKOyKPFBTEgMlG\\\\\\\/pIc\\\\\\\/OFhesxewyG5+6ui3nXSBDASwh\\\\\\\/yaN1AYTZImY9Q1qklVF2V0EDRHCTT9\\\\\\\/O9kPxDegG8F04sR6FsagTlx9K6o6CPf9xQOS1iJXp8ibM7b3\\\\\\\/zj4\\\\\\\/IpDcKiquuTaqrDbll7JSSGJEb8D3GT4lyFjKJyz\\\\\\\/TVTbdsCQXkMBUwt+UaAfeOKOq6Y0KpuiXn93R9XM5hyR1VoDsm1h9M8TzLax6DyV1F9qGwk\\\\\\\/Z359hww2lvNmdQfTGeaAvVs\\\\\\\/S3+gkF1VgpfXhint32IYl526xe9IIZMIn3JN6j8uBHWNTlh72ofv6wxtFHWyAJJPhs7LBqAc7xVVLiG1kQT4Zq+HCdXmGtaqLoc+LdotXrv3ZH8ERRt0wlhBRg71apoLPuEuYLRvQqc2O\\\\\\\/puZ7RDHYzNklAhOD1MIIfxFtz\\\\\\\/juLD4Tijwn9rVEv7ibcHeYTsIe\\\\\\\/eYplayJ\\\\\\\/u\\\\\\\/nCPUGkSOoD4RjurUhcCGn910ORb\\\\\\\/2XVdWIaMWgv8Qd76OkkeSH3WKNpu9WMdmZe\\\\\\\/8OwfsjlMzEedM54kr08xXYMwYwiyFAy6if\\\\\\\/azRxGyKOvXvHGeJWxdUYYXBjTf+Kg2r6N721n8spYuEknTEQFIla9jkd0LREc1H+rpgbHziBHsEzWCRWY6lFAkwmmO5HV+Ef\\\\\\\/Nliq672RUI0MZSvyoD14xQAEBGA82CSRPStHRQpE6qcG2rD2W4SFnXPjCg26YkuKEjMIYBzg9U1EBc9DqkbLUhy4OJVPZByl+01BR3rYtNkVAD3v915u+m0pzFGuf8ETOxzp63yokW31GvNNCWOveEj5Ef6R9y+QIksSGoglZCdJB4huV1+FJHGfJLIpUqGYdUgJq65C1BirMah0G4iNdKEiWkAshAL19JV1Fb8LA3HqpS0\\\\\\\/g1l\\\\\\\/xwIzQUrPngD\\\\\\\/jVC9nJKxZiPkXm4Y8M+1a\\\\\\\/o2A5e9nQINRNH9pcIEaTed\\\\\\\/GSD5QYdpbiMGhBpsvvWJt3+g22fX0Ny+ST6BXvqBUOdWnP2G1TpV3VHcru0TDmujSRHromywxJdhcteI3CIOzpALnKfLRfD4cUf+HeUdlfDX22rEHbHL0VWMCuAX9mQo\\\\\\\/RhJ7MxOSOO9vtJ1R4ZmCBc28RaVTg9LPVkbdmlG4avuJvIxPG3Ilz3hRpl1Hl6uqE378R1eAqprg3iUjFjDkNK1sbxK91IIWJCrmaxzE7psXEhyPPgjPw61QwPdyIiYf39wHJlxRRQ16xIpyABO26264GWQPjAKsp5VmF9aIPlfXpGsSkNtMnPLaCg1dXyvDlCHmQs6O427\\\\\\\/YJ3gMhh7C5JjR7rLRQyoa1gRcfaxVfm1vlsREU9i3yFfPzSDt5HxY3\\\\\\\/FdNWezEg1ap8LU4ksFXTT86pXchQ9dxaE+aQWazt1\\\\\\\/mmIujGUhHRSWf52sutPPon2zxM6OvkY+izSCICiPIjnsrCCd0FKfRyt7b\\\\\\\/mopEdMl3gb0Ks25oYTzATDTATeqwRNqM7o8dVQ+musFcI1mGgSZ0Fureg6gT1f3Y584KFF2gzqPZBhdiBj7pBJ8nmUXcWn6nNKIOLHnHWDwHiwxj8y6wGL0gTo+5r3\\\\\\\/3w8az113Fbbtz0jsyn8bwCghmApd5071NBtk87di2Tt\\\\\\\/VpZhZGF2x+ImTotY\\\\\\\/oLMFKeSenFXPb5DLeMeMGbA7FwLfgb57SUeHiCvmIljv2PJdie4iWzi7CSfwGEJUUqP4vp9mydfiOBXVgbNuC6ahUMLrURMETJpWd2lTrb+t\\\\\\\/F0fXAtg0J89cfowmazTUaDJGwnG97SSbx+LebnkVwVDJqQJoMxMJK1oP5QeUdpzQD+6naXpy1khOpFdzG9OxG4BwlrOvAwBJoqGEfizBbDMc4jVHXrAe6jyxVxTtOyIAioj8Bq9a7JHU1zUImlJISQbB9bCm1qoRTzDEAi+WRLo7aNt9IOR1QTgUTD1CDzQOhEkNapxySm7iF0ViMQQrpXT9RGPkerb4DeuuXLLVatE248mmQ+JF6AykqKhEtTl5pjUUhLG1+AOE4aAiC99AzM8UKz61D+IiLVTTbQNm6uj8RutgWks4iTaMRct\\\\\\\/UY91ZByipr1OEku9wbgSkrBsNbacpIhbTpT4f1o7EtuJxnI\\\\\\\/KtOsYlNRv\\\\\\\/GnGfxDRukZEk0jrQ51ug1EfkeQeP13nlfO1Z7UQbTLp0rT2goqvoHpjLzIZQiVxB\\\\\\\/aMTC3FTNOL9GjDE8CPOQB7bwIlfCedCiXo\\\\\\\/U2H9aoimH6U\\\\\\\/\\\\\\\/KsTitadSPk4ptj6YaXcUIGQxLIOxEddtzVilhrqhogSPxvTuVF\\\\\\\/lf40buupeQ2iif2euEpkZXeEeaAKRHgWfVK5q27lGHcZnHNqqzjvXrAoW7SfKtJ1fK1DNzaRN5ucIDxKJdHdI\\\\\\\/K3by400zQz16CMbatS0qyCkFioMFJ4uFSp7oroAiZj52SEvit4kJ6TR+dKoVbEritiyyyfMSR7Fhe5xG\\\\\\\/kIKkV4vwqKK44sMOX6whxDXvvDOwZXWyZbSodCfxSIhuucgYILkSVbsRifXpd5sAvFAAxAqa7v064\\\\\\\/7Mp5nNPaqSN3y61ATxYiEIzJav2+fPAwLCz\\\\\\\/vzmA9U+91gyyIjWV7RwUsI4NIHieJUbJ8NA0+Z9pfHHKdbaHnDomiBAyI8Bx2XbJkNYzHjdBtMk5V9cfHhmKuDZARQx+vSF9\\\\\\\/44FEE1ppxs\\\\\\\/vnFjEYrcgo0Eb9LEfrk93ggu84imAxd6nVE0+YtZ4EZlQ11G7iJMKk7oG0tve+1wnCs4wXGlaiuOAhf6f1slxOKlkxf5xd7UaT9IJ3E2LCkJ2jNPEe1En0h4VY6JFIoAVE4fWPwO3OYLf3\\\\\\\/2Td1VWHZ9wl6QhmZcjPuNfYncPtU1UjK1Vprvh1wMLU\\\\\\\/\\\\\\\/j6QLUpzODqZ0p7DYCCm3e0PJ72enTbNB+DykQJTihwQnD+TX620TJgw4iPJ6Ad7MtbsCIEVeC+lCLlG6EPjmSwnJjwit3ozjNtrEt5r1pl9i\\\\\\\/oVXkmedDrzn7ngvLp9DHKKE6rFGTkHPtMfRd\\\\\\\/TfkitGnZJEPMnKAZtW80VKAP58WrYbe2qxNb5j71kVAJQ2+221coG8Fcvfg\\\\\\\/BCWEr\\\\\\\/5WGX0jeTz7jRlbcz\\\\\\\/hD1NRLkx0zgr4F5R5ygB3mv7c\\\\\\\/TCj\\\\\\\/pbRUjuVXcGW6JbDep\\\\\\\/GG8d2YkVHFeiNMZuDCP\\\\\\\/oEln96NLZfnp0crvge\\\\\\\/c0W2tMjyvjH4uIjSzAWTa31jRjM9\\\\\\\/orVvXdgsRkRcrMTygdsJNj02T8juxyrCxfbrKZm1LviXs6Fkr5OAPvJv7t7LIDlo5UgDv115x\\\\\\\/b4t0Dn1cvVc+CHCn3n3WlHaARd19NEQS1xiR9B141\\\\\\\/5GuaDd21\\\\\\\/0fh4CCn9lGzegZOfTiqtymJnLcGbSTSRqHRWG8VOZAZJ1\\\\\\\/WFIBvQcjZJnzj1gwLsWYo5qw1+F2vNDvaVnmwy\\\\\\\/VGT5\\\\\\\/FWbbGfyms2Z+DazB0hRM0k86OyA\\\\\\\/jFXlTvT9uwy+M2ioyHIrJ02iYbdGK\\\\\\\/uDIid0tXJu6a+EJ5GfIbtm7TZlWOGV6a8+G1EddujFzchtZbL7zZODhb6TUdTpjFjZiRLNOQybX8vgVbT7CVObiDMQPDBYJ4OMzWfv62oF7vmwaqj64Vo8jRSdFb+7FRpJkxUviRgsDVqKCqEVMqTmO8MWO3D1Q4BB\\\\\\\/LldLLqIdFMtG6zjjdrWmyF1JQTiHoyBGw6iAfFR+7JqjY356LW3g35y4C2NwZJsUOd2V\\\\\\\/tQQEN0Zyf4Crk3uYgGusNjc6QI+ElmurC7aOhBUtbo0xQ\\\\\\\/SXxaeOqI4FZgv4XDQhHt2OIgD1Ro3a7qMno7rb1rD+SGMoovH3o7pp5O6R\\\\\\\/hWuUKaviz9\\\\\\\/LGogDdPxeCnioRh\\\\\\\/STwYNThoWqT7TOu99C4pvQrr1yx0QH944SiQ6pAtX6IzLMA3JNs6GilOMdrJ4+erb+p+OnzW5OXURha9cZmxwNG3D2E0zvyn8IBHMpTXDSgzbAIzLJBl5f7njlgEczthkdpClAyLcJ2jHFrGNJ1zUHbUF+czw\\\\\\\/97IIzOQntCMebk4RfS\\\\\\\/+R4j8x8an+0M41e9jcqNQl6vJaXuE9qtL9YPIsKq3JMwBkvYoZhW8dfyYG14bvYNyJp35hfwF1mvQ9Y5vHOZZJdBF6OGzT5MkfNGxHmbl+cUipmmmJbcSeXX8M2zYOWSOpBfJVINPvJST30R+oy+rmGAq6ORBm6neilg8X2uREyaUA2rB8cQFpGXOzj\\\\\\\/QjV5l56uXN6kpJjPnddknDUOpJ7Qg978oBoyG\\\\\\\/n2aiDjVudnKPz6IiPwsVXydHYshqx9cDRS+fFV+QVL8x9z4e26v3\\\\\\\/ETRlQwswr87RLbF8RBl+0t0ET3LUFJyu0FuioBvbThCv1bHmSrhnaBl5WwMiSLCw\\\\\\\/GtTNtv57YI\\\\\\\/sgXkxglmKxphj60Tk2nMyG7Ib75eRAGSnRrNU+TxO4MatOTVLKZou6MEUp7SCXoiuroqISdIxAtGRFXZZd4pka2eriMRIPYLSdgrBqRZPjXUA5ywnr9slcSK\\\\\\\/ixEciETVwPtZc7PipfoqRTBtbhhoIOAzmlOln4Xa9czaIJhwb28L4wscAx5AOmisA1KEvC1vTcwV5RrCt7\\\\\\\/QEvmuggEyD3321qfp9yumQtu8jKAvfRf5R2sWhFnKhHW2hVvaBdblPRlZv4RZtmauxsiYDpwo9cdvuvLI4CZnG+hgI2UYWnS0vuCw4vsAAFAE9Kwg4KnbGzSGyhzkgmPUArdfqdDU9m6RTW1GaDJwf1iWESxZaLphlEJlodhPg8AgpO3y26fXDBddfl2HIfNTeVvu0U\\\\\\\/8Z+zICyZJyyV9JUtXKP\\\\\\\/aad9aQTCRJW2yRF2N8lPHtL5aFuGGdQYdfDrPOUzu7vjmauhhk+VKJNg8BGmR4KEL+N0rDHbLKm4vEWIJ3Ac21cYadWeMHuNMrYnFiemt2njtvIzfL8NVoG0aNyjo6aqfFpsPb5H8muzdLN1IMcUY9Q6K8C7Da5EGtDT2sZW9hWFFsbPWY1S0D1tzvVWoEtV7Kl6SIpS86ihmSmRL2pxb\\\\\\\/3Lc2QhMlE1HpFt9lfY0hjl1SdvK2\\\\\\\/OkL60\\\\\\\/dGDUiDJMobge3oLf5qHKHBGn0v9L1+25zogxe6zX9j9pdIVRPfpRiZsJ0Ne76SVzijohaIX+t2ejt4QNle2D9Lkm99LZO2rpduJ0i1iEXMQ6WjosWXCHt\\\\\\\/1g4Ss+A7peVBTt0yZvPMu99YUX3mG5juEYNMCkfDbCWalP3VP4hn2AAjIbmLGHTWl2HaMM9jmqSVeB0jxr8EzYwO3gKjMuj4HxSj6My\\\\\\\/9aMcaK8NwcdWWo1re3692Ulc7OON3itEsRo1ern16iqGe8kexwqoeuq7v+h9mchGKBWBLFt9YI\\\\\\\/yhVAK0HTOl3lnDoySjZSRV2sT3CxeNn3DwUmmBC2c2tHrbxM7zhE+md7sLaF+Hgk6rKoNiHnbFswkdLLooM9VvA6XfI2p8qJzQmHop0WTzP2zAX4ppcgYfSn45mDEfs9k4BgucWhWG5d1w4RQAjip5WtIh30Yr8GY23yWJ3lLgTN4725mN2TLxmdAUaqXn0Q55epaa2mB5w0sDFsTG7N+LtnaiJzJY3YTp7xzE6zKznH5WyvvYMrAFjMKOiMI+h6ruo+JIHKoUutnB9hVdoOzOuEq0WMxkhITxfAFMDNf9UMEa4msjbClGVdIpkZ+zX4PVGRPXWMgLRAzXpI2WnVE1oMXi5J01uG+mTA61NdDyk\\\\\\\/kzk3hZuZMM6if6onD4OseyHX2mtZVmT\\\\\\\/n2bKnymJsiBlkASmknpFMUI+2fcS7vP5l70gf++zaKS\\\\\\\/rUvlToCyVZFRO9y4GdGTmSwhcA2IK1T3FBmObgJklO2nghhCYO8\\\\\\\/TjZ4b81s6w7S+T1K3yolXBCxLKQeEC7KBrSdUk3Zz4jd7coThHyKnj4sRUBL3mplzj8895CvZBDWPOvWWTJUkwX3grjqf\\\\\\\/87\\\\\\\/PZVtUe7lmg4TlCTGEGYaEhI3+Q1QHbVaRngNmHkKzXzSvsyxjXrmo4WpeDh2kkSpYvvYlMht6kL5rXV\\\\\\\/uNiLJlCbNhwZV9KV+ihFY0LJ5A1qpgtQremnXsmmax3\\\\\\\/T9CMCVVAbYpA1iI+mwsohnKcU2VZ3F\\\\\\\/pkeN6gniFws0w+InzWaMjlJ7jjGV14Q+mut3WSPVpC7Et0g+igM+MJvoPtgCffBxGEN1X+b+gxGwLUwNzlQN8OCyYQ\\\\\\\/dfqCRv4LjPolHL8pUUga8CVy+c31rlHLjw0mOESw70BkGlFX76gdkCegA\\\\\\\/wlxOvAtspJZzXZN5vRhWDpN9R13wIf+88ErDzs7tK7caPgg2LIrP9m92pVfcSPz0UIbcfMmuISYXEwoFPXcC1rE3C6dOE9iKJRBd+wOwzG9n7dj5j5tFv3LCNToSx9tjDjKhfZRpM5bdWBOZZ7uuwEaJJC\\\\\\\/G4t6jgqFeOd5nrgsQMzRYnEhXG1NPdnKghXYLhQS9qqrUUj4tWjz7So8rcSshGTz5yLzb3h\\\\\\\/4IKu+RemzFiJ4pqr65bWqSoMUv8NgAFWXVpXyiyGRm3lxY\\\\\\\/ib9iP8x8+1uIpIcL3Rxm4Snguo53weUtzveefV8L4D8zSoUDYuFT0vlNJ\\\\\\\/6uYv8CGHi9+PhD\\\\\\\/ho4\\\\\\\/H0PXlDZpzvz\\\\\\\/m1OaSMvqL+maBDTVDOAAV3xo+JkHOSdr1xUoxGGVFVTPzrRJklrfF+SO1PgGnC7pxc4xqb1+yYL95LE4c931nLysoYb05arpX4C\\\\\\\/IU8CQWEDSjzlHkJ9SVa9Ub9nVdgtG0TfHoGhgIwiI4c+nDWFPoh7jW7XGEdieVQ9UYRCLnF0VDd85TZbuhejyLbfiyRrPH+Yk0omJfZ6cPS9yCBsP+WoqS9sEnO1D+5WyYGi\\\\\\\/73xkm0ZKb7Dxp86UmUxvtiTbxNL6jxLYD7GE3wDstCQssg+9Q2rXysM46g5KOU5zxbueOFqQXowC6TNMxNZg4JkW8ArkMpBKkRbwPK\\\\\\\/QTmQ3EqKpM4xL\\\\\\\/YmxfDE5FO3fqzDFUSfXoqcYzC4GZIu9gDYWXzBJ7wydizxxh0wQb8PpL15IIBk36jfiqh\\\\\\\/BJb\\\\\\\/wHPHxMY2QMAXwBUcSzI4hHJmgC\\\\\\\/gYHp0XGejpCtDph2hVHE9UGLWyDo9Zuu6BXcMvyZmuPRg9TyhwVWdFRNSJ2I7FzBlGo5bqZmFPTgQegDokS6C2F0XOLGwlaCKJFJYJ3ewwrmlFRRChIJWMwrV+7esRHzJBc0xLOGdVLmHymEPd+zdN0dd0JouPBIsxnLlEuPPv+ihV9BXPwpEnHUGs7JqtwWP9Tw9MWhb1YBoKkKR3WCv+rKd8xaSETDRfqyejUB\\\\\\\/vLxiOTOKafnvHl2VfDpqW2xWrZ012IEe9rOVReJl7DthA9PDEW+ovOy4d4\\\\\\\/Ck8W7vOi9Om8sCNeJ6NxZMqNba3Qe1uuWmj2n80vSGVoq1Edk1SoIqJ5LK+\\\\\\\/K1wJU0+Zol9ZA4gASV2knneZg8ZnX2H5\\\\\\\/i4NJq5C6aYnbXtMJMxINwT3qwA5+9CAjfZfmYLvYD66w15B5Ocx0gDsIaMWG+5sncMpBz06+PsQIDNWjepALaITTccQDBBqDLawkfIHi2WVSFpY5pExqSuxO3\\\\\\\/Xlgs4Zd1KGbZPZt6dhoV30B8rhXp4t5x3\\\\\\\/7CqzNC1f8VKiDnf7ahUQo19Eg8gsdNJo0jNSezSCnmE7P553S\\\\\\\/bFUsRYGCSd41T78mjNgbaElBBfj6fGA3XiHRydBPcZOR1HndE8TqH+njugHkYPLyd9QvIAgWvj8c3OgNksJ9VqCaY5SqbirGv7FWWwonIxAiu68ayx0u+Cix4Evkqiw548K9jMnwo\\\\\\\/m9jXFoA2QTxSroPYVejcR2Wd5I3wrxQCWYLc5eOd2v7XCvkbBnVi1FVx18gQnS4Y9Glks1xL4Y8V+tjVqBOmiIRE5otjLyMLpkj0scDSrNUJ8WJSJkrHWF5sf5ZiYdr21ANKRs9ffh8yUUd3uYjbE3NmxkkLkeElR\\\\\\\/WvDGLR+ku7L3TF1jF4dSz06I9pl\\\\\\\/5qbc6Q\\\\\\\/NmxhYFeXTWAlmayk\\\\\\\/ogHdLcfV\\\\\\\/qjzgs7SNx4f6p26YW9usIin\\\\\\\/2SiHuQHH8EooyJbAgyZBBM9aIc8h+ZWXqLnlVItfwT01VfbBWNXek7ZWIHwtr4JzYfcK9k7Y2HIfrfyeAsr1\\\\\\\/PV2Lxe55LlHVheB7b5qjuVLHl+f7e634ohcZx19xbu0SIZXiI4sfsu5L7glxv78DaUDpX3ERmMwGddf0Ns3N\\\\\\\/oML5FH+mBFuCZLwtZCWbcXg+6vZpm93aOUoh0aQKCnrBWj2RAX31V6BBYd23pi5tmtuA2ucE423urYa06oTy7XwNWdc06\\\\\\\/8qRKsEFID62zN5T92LsqvsOPPGUKy8wrrnrr\\\\\\\/TwBsbXJfFCDDa66DCiZPcWBdDnwsFKx\\\\\\\/DZ8HB8t4dW7sOMkinqIOiWCAamReAeYWv0PfBnsgPQMR9taE+CheKBbTk3+Ak9AmnzHFNTlFEyo7z7OFG0vWJPNlWUyxcopLX9B\\\\\\\/5U3BKDyMuAGDlh1wWHJsvJwG5KKemtfxdbkEWxL3lrexllhNcf7A8CJjLM01iu3n8RDimZtMdTf6YKnJHU6vJ08E2NYhC5QPKPIw0l5Brx3bvcZIBOx+NSTyOgdBJSprG4QEfHULX+biboVFQHA6hktIY\\\\\\\/r0\\\\\\\/7w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022zfD54sXLVcMY5G7k2oW8eQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d1d2f4ef99c6211441c5d661d85fa8069ed52c738e907ea0d3f395d724f82dbd\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022fgMLHIrGP03gLoVp1KRmM3UTIx0iAXpdrVhG+hd8yd+RUJG7cm82mUNwCpFQJ3xw7Ry1B7mnCJKd0QiqkjWh56DkYAr1A+9OiZUNcidnOmbRBNKSiaqoM5QhrbWX6\\\\\\\/twSP9j+I+qxtqNWjTRj0T9\\\\\\\/8zz62eDT7m7Htmzw1Jh4tgVVnlRrNb2QtZ7kL0SxXe6z+hLhOcJyabNSxZPVBFj1wxWydnm5aTsSRpFkXDt5luvxOogn+dyqJaZQceqz7EizUELDZh0nVQOI90rZ7MlZbYU1NigQy1mtljfSdEclNMs2Ncu9o9fmStgfAFe+A2AqQ9RJJL6MF8KsV6uRqldcAEVQNpuc6Eua7BA6xFw2OyzbM15Fre9j45QT0\\\\\\\/S7s7cIhOHbWCn5e86xZzrpjqVzuT5tLCyC2oxil8L\\\\\\\/gDLSqi5hoH11S8iWeA+4WytYHY0KBwPgmF0Dl7vQomzJld74r1YvIhqOTM39J+CUbziKzttbYAI\\\\\\\/WkvIIYTaTgpZlmNlehZ3psk+DCwmfwyhhGq\\\\\\\/pM6ZKNA9VgB2oYEYNDDZR8vzuaT20VEXBZZP0ekYvP5E1PTxc\\\\\\\/ITprkJ\\\\\\\/ufPuCPa1BXJqHqnBeU7SZ8GIkwO\\\\\\\/HdeyIDZtNxOgSSS\\\\\\\/GnFU032Egf7EYCgxGNkkQO0+KUI7EicDrMFILqGsHsAv3diMGOeU4ZEuZPgyoD9ByPhUyxgC46ijMDS0CPAFQ0cxG+PuWG21fBWafRYrLUOuSIIc5wwoL2AToldoZx0wojxfXHcenwBF2yWK9iGrd2DRGIQ1FGl2txvPe2KF0Pamh6ayT51BLy4zzu8JSE\\\\\\\/PVv0wzGVSvVNrZP9OqmliaSZbNVXwvV0\\\\\\\/uCQijI0iW31X5HWQu8LDuhDz6OluVu2MXFFs0lWI8Kz96X5tiYBEbmcBddZguHL0jz7zPbQh1EBB8O9cFxs5loO3JHMy9\\\\\\\/Ku2wrQtC80hnj1ahNnIf65GZBnTNSSHml9M8INaaehC34HI\\\\\\\/iGbiPiFTXg8bT9tcVa5BxD7hWk271EWm4ytLFkWecluazB7va4x6sJMD3+vmtHeiO5ZYqstAxSdzWfAyr+ZxZHXxD2B5Q60UXOSGpJgxQVg\\\\\\\/zecfdk91yun+Dl2YELh1nmkkx7jEpkVV71lGopwpJiUqxVSNBItl4eWN8by\\\\\\\/MOv6N8DR8c3ZvFlIumheqJqt595R1bsGcBn6cWTY69DQHw7X8GFsGaclZtjwW2NoXz0DYvYqGLQDPPctGJNWl3BOrsajQ\\\\\\\/V5tzWErv9W4Ti4+IVuz27423qsLY8SiGGAmbU1+wedQpzn7QtNxVumQ7G1G947kEh+3rc1OIjoRz3yMTCmmgwBqwEP0rtiSh+AEvIOWE5wwZIgDT2\\\\\\\/C2f1c\\\\\\\/1mIzp6PdPbpTGgSLySjjQenJkk4BYi9OCz\\\\\\\/X9+hF\\\\\\\/hr8\\\\\\\/N0dOL8do44WfRYSBNKmsAjQuPFFgL1O6Md5+3oXELEVG66LMOYEjVGMZoamAJq5HJqPpYFAYYLNFJ1gx0s5ZjIe2GSlgQEKcslQTCswaFleHMwIOc3l7FXHSYaBcKIU8SY2vkntyont2yWurFm796P0sqzV9sGJWJvQUog7GTV5EKi0yh1zyydCI6XvFgrlWHExOn4Zr9UaPpXfgYDFHGu7L29kw1lPfu51aD4t3Fin\\\\\\\/tRF3uP1YBHXOCleK0LKypfYWI\\\\\\\/5fXqj+NmTJlP1WJ8sT\\\\\\\/0L3w8skD8d7VnV7vw+LgLWFbHYJ\\\\\\\/DnTl8seLZWltjRlhNe\\\\\\\/hUt9XS2w9HT5wkDjn2TrtyUR4xJuT6KVVaXAR5CxZB96bXkAk\\\\\\\/806n2KjCfL9kgI3greBkVzKvfuIslNUP7vuu\\\\\\\/s+1SBPeO8Xt\\\\\\\/vYZsQdKz5rxTazhRZbQfPu8QbnUUCaCul02b6Qw+1vWkg0CKley+9kaq7ZeLEJGNOmAJuJuSb+Jxvf6LT7thSFlcwNC3xdgsL7CAkl7\\\\\\\/uKcqY6kycjjyaGiLOPuhqoDv0Jth8fU+SkxnZIAfBBRyuSWw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022avBLO5C7dimpMbKUYsVCvA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d8ea81ac5568b87515cfc9b10286bce6f33c2fe3d36b7631bcb911595afa217c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494050 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/3023" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-TZYeOFAf-ouHertF6H9Kwi0q5LRddTc0asFwiQ1ksQM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/3023" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-hQDG_jy2XOTMVg0tFGPmQT0FhhZx3XZr4r0W9gTG8V0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/9/3009" title="Cyfip1 Haploinsufficiency Increases Compulsive-Like Behavior and Modulates Palatable Food Intake in Mice: Dependence on Cyfip2 Genetic Background, Parent-of Origin, and Sex" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/9/3035" title="Correlating Genotype and Phenotype in the Asexual Yeast Candida orthopsilosis Implicates ZCF29 in Sensitivity to Caffeine" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494050" id="node494050" data-pisa="ggg;9/9/3023" data-pisa-master="ggg;g3.119.400549" data-apath="/ggg/9/9/3023.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Transcriptome-Wide Association Supplements Genome-Wide Association in <em>Zea mays</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5068-0372" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Karl A. G. Kremling</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8411-0343" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christine H. Diepenbrock</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael A. Gore</span>, <span class="highwire-citation-author" data-delta="3">Edward S. Buckler</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-5941-9047" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nonoy B. Bandillo</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3023-3033;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400549 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494050" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Karl A. G. Kremling </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Genomic Diversity</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karl%2BA.%2BG.%2BKremling%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kremling%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKarl%2BA.%2BG.%2BKremling%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5068-0372" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Karl A. G. Kremling</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kak268@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">kak268@cornell.edu</a>
<a href="mailto:nonoy.bandillo@ndsu.edu" class="" data-icon-position="" data-hide-link-title="0">nonoy.bandillo@ndsu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Christine H. Diepenbrock </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christine%2BH.%2BDiepenbrock%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Diepenbrock%20CH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristine%2BH.%2BDiepenbrock%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8411-0343" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christine H. Diepenbrock</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michael A. Gore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BA.%2BGore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gore%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BA.%2BGore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael A. Gore</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Edward S. Buckler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Genomic Diversity</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853, and</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>United States Department of Agriculture-Agricultural Research Service, Robert W. Holley Center for Agriculture and Health, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edward%2BS.%2BBuckler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buckler%20ES&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEdward%2BS.%2BBuckler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Nonoy B. Bandillo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Genomic Diversity</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nonoy%2BB.%2BBandillo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bandillo%20NB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANonoy%2BB.%2BBandillo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5941-9047" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nonoy B. Bandillo</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kak268@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">kak268@cornell.edu</a>
<a href="mailto:nonoy.bandillo@ndsu.edu" class="" data-icon-position="" data-hide-link-title="0">nonoy.bandillo@ndsu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/9/3023" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494050" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494050/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/9/3023.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494050" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494050/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/9/3023.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494050" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494050/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/9/3023.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494050" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494050/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Modern improvement of complex traits in agricultural species relies on successful associations of heritable molecular variation with observable phenotypes. Historically, this pursuit has primarily been based on easily measurable genetic markers. The recent advent of new technologies allows assaying and quantifying biological intermediates (hereafter endophenotypes) which are now readily measurable at a large scale across diverse individuals. The usefulness of endophenotypes for delineating the regulatory landscape of the genome and genetic dissection of complex trait variation remains underexplored in plants. The work presented here illustrated the utility of a large-scale (299-genotype and seven-tissue) gene expression resource to dissect traits across multiple levels of biological organization. Using single-tissue- and multi-tissue-based transcriptome-wide association studies (TWAS), we revealed that about half of the functional variation acts through altered transcript abundance for maize kernel traits, including 30 grain carotenoid abundance traits, 20 grain tocochromanol abundance traits, and 22 field-measured agronomic traits. Comparing the efficacy of TWAS with genome-wide association studies (GWAS) and an ensemble approach that combines both GWAS and TWAS, we demonstrated that results of TWAS in combination with GWAS increase the power to detect known genes and aid in prioritizing likely causal genes. Using a variance partitioning approach in the largely independent maize Nested Association Mapping (NAM) population, we also showed that the most strongly associated genes identified by combining GWAS and TWAS explain more heritable variance for a majority of traits than the heritability captured by the random genes and the genes identified by GWAS or TWAS alone. This not only improves the ability to link genes to phenotypes, but also highlights the phenotypic consequences of regulatory variation in plants.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/endophenotypes" class="hw-term hw-article-keyword hw-article-keyword-endophenotypes" rel="nofollow">endophenotypes</a></li><li class="kwd"><a href="/keyword/fisher%E2%80%99s-combined-test" class="hw-term hw-article-keyword hw-article-keyword-fisher&#x2019;s-combined-test" rel="nofollow">Fisherâs combined test</a></li><li class="kwd"><a href="/keyword/genome-wide-association-studies" class="hw-term hw-article-keyword hw-article-keyword-genome-wide-association-studies" rel="nofollow">genome-wide association studies</a></li><li class="kwd"><a href="/keyword/natural-variation" class="hw-term hw-article-keyword hw-article-keyword-natural-variation" rel="nofollow">natural variation</a></li><li class="kwd"><a href="/keyword/transcriptome-wide-association-studies" class="hw-term hw-article-keyword hw-article-keyword-transcriptome-wide-association-studies" rel="nofollow">transcriptome-wide association studies</a></li><li class="kwd"><a href="/keyword/variance-partitioning" class="hw-term hw-article-keyword hw-article-keyword-variance-partitioning" rel="nofollow">variance partitioning</a></li></ul><p id="p-5">Discovery of variation that underlies quantitative traits remains central to the genetic improvement of agricultural species. Functional variation can alter coding sequence or act to regulate an intermediate phenotype. Regulating the abundance of phenotypic intermediates, such as mRNA expression or protein level, provides a more spatially and temporally subtle target for selection than coding sequence changes, which are more likely to be pleiotropic and therefore maladaptive (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Mayr 1970</a>). Thus, regulatory variation is the frequent target of both natural and artificial selection that shapes genomes across life, including domesticated plants (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Carroll 2008</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hufford <em>et al.</em> 2012</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Mayr 1970</a>). It is likely that about half of functional variation is regulatory (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albert and Kruglyak 2015</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gusev <em>et al.</em> 2014</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Rodgers-Melnick <em>et al.</em> 2016</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Welter <em>et al.</em> 2014</a>). It should also be noted that regulation can take place at any biological level of organization from the epigenetic state (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Law and Jacobsen 2010</a>), to gene expression (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Albert and Kruglyak 2015</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Fu <em>et al.</em> 2013</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">GTEx Consortium 2015</a>), to ribosome occupancy (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Juntawong <em>et al.</em> 2014</a>), to metabolites (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Riedelsheimer <em>et al.</em> 2012</a>), to protein abundance (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Battle <em>et al.</em> 2015</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chick <em>et al.</em> 2016</a>), furnishing multiple levels at which intermediate and terminal phenotypes can be associated.</p><p id="p-6">In standard genetic mapping approaches, like association or linkage mapping, associations between genetic markers and terminal phenotypes of interest are tested for significance (black arrow, <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). However, multiple levels of biological organization exist between the DNA sequence and the terminal observed phenotypic outcomes, enabling trait dissection to be conducted between intermediate levels of biological organization (hereafter endophenotypes, designated by an orange and red arrow in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Associating endophenotypes with terminal phenotypes predates the use of molecular genetic markers for mapping. The use of linked observable traits and isozyme migration patterns are examples of tying markers from biological intermediates to terminal phenotypes of interest. Similarly, just as relationships between individuals can be calculated from molecular genetic markers (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Flint-Garcia <em>et al.</em> 2005</a>), endophenotypic similarity from isozyme markers can also be used to quantify relatedness (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dubreuil and Charcosset 1998</a>). These same principles have recently been extended to phenotypic prediction guided by metabolites (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Riedelsheimer <em>et al.</em> 2012</a>) or by expression dysregulation (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kremling <em>et al.</em> 2018</a>). However, the use of molecular intermediates, which are now readily measurable at large scale across diverse individuals, remains underexplored in plants for the inverse task of causal inference.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/3023/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Levels of biological organization between the ultimate cause of genetics and the terminal phenotypic outcomes can be exploited individually to improve power and inference of biological mechanism. Genotype can be linked to endophenotype as in eQTL or protein QTL (pQTL), or endophenotype can be linked to terminal phenotype by methods like TWAS." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-106138756" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Levels of biological organization between the ultimate cause of genetics and the terminal phenotypic outcomes can be exploited individually to improve power and inference of biological mechanism. Genotype can be linked to endophenotype as in eQTL or protein QTL (pQTL), or endophenotype can be linked to terminal phenotype by methods like TWAS.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/3023/F1.medium.gif" width="440" height="180"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/9/3023/F1.medium.gif" width="440" height="180"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/3023/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/3023/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494153" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-7" class="first-child">Levels of biological organization between the ultimate cause of genetics and the terminal phenotypic outcomes can be exploited individually to improve power and inference of biological mechanism. Genotype can be linked to endophenotype as in eQTL or protein QTL (pQTL), or endophenotype can be linked to terminal phenotype by methods like TWAS.</p><div class="sb-div caption-clear"></div></div></div><p id="p-8">Associating endophenotypes with terminal phenotypes has multiple distinct advantages. First, while genetic mapping is dominated by the covariance structure of neighboring SNPs and complex haplotypes, endophenotypes provide orthogonal information that often permits inference regarding biological mechanism, which may not be possible from genetic variants alone. Second, genetic mapping often points to intergenic (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Wallace <em>et al.</em> 2014</a>) regulatory variants that are not within the coding sequence of the gene that alters the phenotype (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Albert and Kruglyak 2015</a>). Therefore, an association signal cannot directly be tied to a corresponding gene and may even be in the body of a second unrelated gene (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Tishkoff <em>et al.</em> 2007</a>) or in the case of synthetic association, between multiple true causal variants affecting different genes (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dickson <em>et al.</em> 2010</a>). Association tests with intermediate expression phenotypes do not suffer from these limitations. Third, the abundance of endophenotypes is largely independent of linkage disequilibrium (LD), unlike in the case of genetic markers. In other words, even multiple genes that are perfectly linked, and thus not independently observable in separate individuals, can be prioritized for association with a trait because their expression patterns are independent. This is of greatest utility in species where linkage disequilibrium is extensive or where making high-resolution mapping populations is not feasible.</p><p id="p-9">Intermediate phenotypes, such as expression, can also integrate the signal from changes in multiple components of a network, which may not be individually detectable either because their effects are small or changes to the peripheral network components occur at low frequencies. Similarly, intermediate phenotypes can integrate a phenotypic signal from underlying genetic variants for which low frequencies preclude direct detection. The most deleterious of variants are expected to segregate at the lowest frequencies (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gibson 2012</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Henn <em>et al.</em> 2015</a>) and, thus, escape detection by mapping without prohibitively large sample sizes. However, rare deleterious variants can be expected to drive common maladaptive patterns in intermediate phenotypes that are thus more easily detected through endophenotype association tests like transcriptome-wide association studies (TWAS) (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hirsch <em>et al.</em> 2014</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Pasaniuc and Price 2017</a>). Methods for integrating expression association tests with GWAS have also been used extensively in the human context as shown by <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gusev <em>et al.</em> (2016)</a> and <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Mancuso <em>et al.</em> (2017)</a>. However, those methods rely on summary statistics, LD scores, and expression imputation and are computationally more intensive than the more accessible Fisherâs combined test whose utility and improved power over TWAS or GWAS alone we have shown here for the first time and recommend for other researchers in model contexts.</p><p id="p-10">Here, we illustrate the power of using gene expression endophenotypes measured in a large 299-individual, seven-tissue gene expression resource (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Kremling <em>et al.</em> 2018</a>) collected from the Goodman maize diversity panel (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Flint-Garcia <em>et al.</em> 2005</a>). Expression levels are correlated with terminal phenotypes in TWAS (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Hirsch <em>et al.</em> 2014</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Pasaniuc and Price 2017</a>) and then combined with genotype-based associations from GWAS. The method is demonstrated here in a maize inbred diversity panel (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Flint-Garcia <em>et al.</em> 2005</a>), which has been widely used to dissect the architecture of dozens of traits of varying complexity (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Harjes <em>et al.</em> 2008</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lipka <em>et al.</em> 2013</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Owens <em>et al.</em> 2014</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Wisser <em>et al.</em> 2011</a>).</p><p id="p-11">Related work in maize that relies on associating expression differences directly with phenotype using a Bayesian method, called expression read depth GWAS (eRD-GWAS), has been published recently (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lin <em>et al.</em> 2017</a>). This work used 369 maize samples from which shoot apex RNA was collected. Beyond the difference in frequentist <em>vs.</em> Bayesian approaches, our study also exploits expression measurements from seven tissues in a multiple-regression-based TWAS and integrates the signal from TWAS and GWAS into a more powerful combined test which can be readily visualized as a Manhattan plot. We also compare the power of each model based on the ability to detect known genes, and the capacity to explain variance in a separate population, which differs from the approach of the previous study (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Lin <em>et al.</em> 2017</a>). To make this comparison we use the maize NAM population (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Yu <em>et al.</em> 2008</a>), which has the advantage of being largely independent of the diversity panel (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Flint-Garcia <em>et al.</em> 2005</a>) in which detection was performed.</p><p id="p-12">We assess the efficacy of TWAS by quantifying the capacity to identify previously identified genes, and by the fraction of phenotypic variance explained (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Gusev <em>et al.</em> 2014</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Rodgers-Melnick <em>et al.</em> 2016</a>) by the most strongly associated genes, and compared the TWAS results with GWAS and an ensemble approach combining both TWAS and GWAS. We illustrate that the results of TWAS are a valuable supplement to GWAS mapping that aids in prioritizing likely causal genes when both methods are used in a combined test.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Genotypic data</h3><p id="p-13">Genotypes for the Goodman diversity panel (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Flint-Garcia <em>et al.</em> 2005</a>) used in the genome-wide association studies were from the unimputed maize HMP 3.2.1 called against the B73 reference genome (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bukowski <em>et al.</em> 2018</a>). Variants segregating above 5% minor allele frequency (MAF) in the union of all lines were considered for mapping. Variance component estimation in the maize NAM population (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Yu <em>et al.</em> 2008</a>) was performed using imputed HMP 3.2.1 variants [filename: NAM_HM321_KNN.hmp.txt.gz].</p></div><div id="sec-3" class="subsection"><h3>Phenotypic data</h3><p id="p-14">For mapping in the Goodman diversity panel, kernel carotenoid BLUPs from 30 traits were from <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Owens <em>et al.</em> (2014)</a> and the 20 kernel tocochromanol traits BLUPs were from <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Lipka <em>et al.</em> (2013)</a> after additional outliers were removed. The 22 field-based agronomic trait BLUPs were those calculated by <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hung <em>et al.</em> (2012)</a>. Phenotypes used in variance partitioning with the maize NAM population were from <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Diepenbrock <em>et al.</em> (2017)</a> for the tocochromanol traits. Agronomic trait BLUPs were previously calculated by <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Hung <em>et al.</em> (2012)</a>.</p></div><div id="sec-4" class="subsection"><h3>Expression data</h3><p id="p-15">Expression quantifications were those created from seven diverse tissues in maize by aligning 3â² mRNAseq reads against the AGPv3.29 maize genome as described by <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Kremling <em>et al.</em> (2018)</a>.</p></div><div id="sec-5" class="subsection"><h3>Genome-wide association study</h3><p id="p-16">Genome-wide association tests were conducted in the maize Goodman diversity panel (<a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">Flint-Garcia <em>et al.</em> 2005</a>) using a mixed linear model as implemented in FastLMM (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lippert <em>et al.</em> 2011</a>) accounting for kinship and a naive general linear model fit using MatrixEQTL (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Shabalin 2012</a>) as implemented in TASSEL (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bradbury <em>et al.</em> 2007</a>).</p></div><div id="sec-6" class="subsection"><h3>Transcriptome-wide association study</h3><p id="p-17">Transcriptome-wide association tests were conducted in the maize Goodman diversity panel (<a id="xref-ref-11-7" class="xref-bibr" href="#ref-11">Flint-Garcia <em>et al.</em> 2005</a>) for genes that were expressed in at least half of individuals represented in a specific tissue. A linear model was fit individually for each phenotype*expressed gene combination in which the explanatory variable is the expression value of a gene across individuals. TWAS was attempted both without covariates and with five genetic principal coordinates (calculated from maize HMP3.2.1 used in (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Kremling <em>et al.</em> 2018</a>) and 25 probabilistic estimation of expression residuals (PEER) hidden factors (calculated separately for each tissue) as calculated in (<a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">Kremling <em>et al.</em> 2018</a>). Multi-tissue TWAS was also performed. First a model was fit once per trait using the five principal coordinates described above. This model was then compared by analysis of variance (ANOVA) to a model for each gene containing terms for each tissue and the principal coordinates. The p-value resulting from this ANOVA was used to determine whether the multi-tissue model is significantly better than the covariate-only model. This p-value was also used as the p-value in the second of the Fisherâs combined tests below.</p></div><div id="sec-7" class="subsection"><h3>Fisherâs combined tests of TWAS and GWAS</h3><p id="p-18">The GWAS p-value (mixed linear model with kinship as a random effect) of each SNP in the top 10% of most associated SNPs was assigned to nearest gene and then combined with the TWAS p-value (linear model with multi-dimensional scaling (MDS) principal coordinates + PEERs) for that same gene using Fisherâs combined test as implemented in the sumlog method in the <em>metap</em> package (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Dewey 2017</a>) in R. TWAS p-values for genes which were not tested in TWAS (<em>i.e.</em>, their expression was not observed in at least half of individuals) were set to <em>P</em> = 1 prior to combining with GWAS p-values. Fisherâs combined tests were performed in the same way when including the multi-tissue TWAS results instead of the kernel-only results.</p></div><div id="sec-8" class="subsection"><h3>Variance partitioning</h3><p id="p-19">Using the k-Nearest Neighbors (KNN) imputed Nested Association Mapping population HMP3.2.1 genotypes described above, kinship matrices were calculated based on the top ten genes identified by each of the TWAS, GWAS, and combined models described in the Goodman diversity panel (<a id="xref-ref-11-8" class="xref-bibr" href="#ref-11">Flint-Garcia <em>et al.</em> 2005</a>). To independently assess the accuracy of detected genes, the phenotypic variance explained by each kinship matrix was calculated in the Nested Association Mapping population, within each family and across all the NAM families. For TWAS, the top 10 genes were taken and all SNPs within a 0.5 Mb radius of the start and end of the gene (maize annotation AGPv3.29) were used to calculate a single kinship matrix per trait using the Variance Component Annotation Pipeline in TASSEL (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Bradbury <em>et al.</em> 2007</a>). The REML solver in LDAK (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Speed <em>et al.</em> 2017</a>) was used to calculate the variance explained by the single kinship matrix. For GWAS, the SNPs were ordered based on significance and assigned to their nearest gene. The top ten unique genes from this list were taken to calculate kinship matrices using the same 0.5 Mb radius around the gene. To avoid picking multiple genes and redundant variants from the same peak based on the GWAS results, the top most associated gene was used within a peak and all other genes within the 0.5 Mb radius were excluded from selection as top genes.</p></div><div id="sec-9" class="subsection"><h3>Overlap with known kernel metabolite genes</h3><p id="p-20">Fourteen known tocochromanol biosynthetic genes identified in NAM (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Diepenbrock <em>et al.</em> 2017</a>) and 58 <em>a priori</em> candidate genes relevant to the biosynthesis and retention of carotenoids (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Owens <em>et al.</em> 2014</a>) were used as positive controls to test the capacity of our GWAS, TWAS, and combined methods to re-detect known genes. In order to avoid comparison of p-value thresholds across methods, positive detections were counted if a gene was detected among the top 1% of genes associated with a trait.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-21">All data are held in public repository. The SNP data for the Goodman diversity panel (<a id="xref-ref-11-9" class="xref-bibr" href="#ref-11">Flint-Garcia <em>et al.</em> 2005</a>) used in the genome-wide association studies were from the unimputed maize HMP 3.2.1 called against the B73 reference genome (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bukowski <em>et al.</em> 2018</a>). The imputed HMP 3.2.1 variants [filename: NAM_HM321_KNN.hmp.txt.gz] for the maize NAM population (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Yu <em>et al.</em> 2008</a>) was used for the variance component estimation. Expression quantifications were those created from seven diverse tissues in maize by aligning 3â² mRNAseq reads against the AGPv3.29 maize genome as described by <a id="xref-ref-23-6" class="xref-bibr" href="#ref-23">Kremling <em>et al.</em> (2018)</a>. Kernel carotenoid BLUPs from 30 traits were from <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Owens <em>et al.</em> (2014)</a> and the 20 kernel tocochromanol traits BLUPs were from <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Lipka <em>et al.</em> (2013)</a> after additional outliers were removed. The 22 field-based agronomic trait BLUPs were those calculated by <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Hung <em>et al.</em> (2012)</a>. Phenotypes used in variance partitioning with the maize NAM population were from <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Diepenbrock <em>et al.</em> (2017)</a> for the tocochromanol traits. Agronomic trait BLUPs were previously calculated by <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Hung <em>et al.</em> (2012)</a>. Supplemental material available at Figshare: <a href="https://figshare.com/s/ef57544b4d09d5c55131">https://figshare.com/s/ef57544b4d09d5c55131</a>.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><p id="p-22">To test the utility of expression data in dissecting quantitative traits in maize, we performed single-tissue-based and multi-tissue-based TWAS (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Pasaniuc and Price. 2017</a>) and compared these results with GWAS results, and an ensemble approach combining GWAS and TWAS results using the Fisherâs combined test. In TWAS, expression levels across seven tissues from a maize diversity panel (<a id="xref-ref-11-10" class="xref-bibr" href="#ref-11">Flint-Garcia <em>et al.</em> 2005</a>) were used individually and together in a multiple regression as independent variables and correlated with previously measured phenotypes for maize kernel traits, including 30 grain carotenoid abundance traits (<a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Owens <em>et al.</em> 2014</a>), 20 tocochromanol abundance traits (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Lipka <em>et al.</em> 2013</a>), and 22 field-measured agronomic traits (<a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Hung <em>et al.</em> 2012</a>).</p><div id="sec-12" class="subsection"><h3>Integrating TWAS with GWAS improves power for identifying and prioritizing known genes</h3><p id="p-23">To assess the relative power of each method to detect known genes, we counted the number of known genes identified in the top 1% ranked genes (based on p-values) found by each method for each trait. This identification of known genes among the top 1% of hits for each method measures how often known genes appear in the tail of the distribution of detected genes and avoids direct comparisons of p-values between differently powered and structured tests that rely on continuous (TWAS) or discrete (GWAS) independent variables.</p><p id="p-24">As shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Tables 1</a>, <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">2</a>, S1, and S2, the combined test outperforms either the genotype-based or expression-based tests alone for both classes of traits, with 30 total detections of known genes among the top 1% of associations across tocochromanol and 75 detections of putative carotenoid related genes (<a id="xref-ref-31-6" class="xref-bibr" href="#ref-31">Owens <em>et al.</em> 2014</a>) when using the carotenoid traits. Using the tocochromanol and carotenoid lists from (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Diepenbrock <em>et al.</em> 2017</a>; <a id="xref-ref-31-7" class="xref-bibr" href="#ref-31">Owens <em>et al.</em> 2014</a>) genes are detected more often in each of the tocochromanol and carotenoid trait classes when using the combined method. However, the Fisherâs combined test of GWAS results with the multi-tissue TWAS results did not perform better. The detection rate was consistently higher for kernel-based TWAS over the multi-tissue TWAS, most likely because the tocochromanol and carotenoid traits are predominantly controlled by gene expression in the kernel.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494117/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494117/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of total and unique known gene detections in top 1% of results across tocochromanol traits by kernel TWAS with PEERS and PCs, multi-tissue TWAS with PCs, MLM GWAS, Fisherâs combined test of kernel TWAS with PEERS and PCs and MLM GWAS, and Fisherâs combined test of multi-tissue TWAS with PCs and MLM GWAS. There are 14 previously known tocochromanol genes in maize (<a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Diepenbrock <em>et al.</em> 2017</a>). On the left half of the table the number of detections exceeds the number of known genes because a gene is counted as detected each time it is in the top 1% of associations for the 20 tocochromanol component traits</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494151/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494151/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary of total and unique putative carotenoid gene [28] detections in top 1% of results across carotenoid traits by kernel TWAS with PEERS and PCs, multi-tissue TWAS with PCs, MLM GWAS, Fisherâs combined test of kernel TWAS with PEERS and PCs and MLM GWAS, and Fisherâs combined test of multi-tissue TWAS with PCs and MLM GWAS. On the left half of the table the number of detections exceeds the number of known genes because a gene is counted as detected each time it is in the top 1% of associations for the 30 carotenoid component traits</span><div class="sb-div caption-clear"></div></div></div><p id="p-25">We also compared the methods at the level of single traits. To determine how the combined method prioritizes genes that are not detected in the individual TWAS and GWAS methods and aggregates genes that are detected by only one method, we plotted the results across models for each individual trait. In <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> we plotted the signals mapped for the zeaxanthin trait. Note that points representing SNPs from the MLM GWAS model in (c) and (a) are identically placed, but in (c) they are colored by TWAS significance. The top five genes detected by each method are labeled (a is not individually labeled because the points and top five genes are identical to those in plot c) and previously detected genes found by <a id="xref-ref-31-8" class="xref-bibr" href="#ref-31">Owens <em>et al.</em> (2014)</a> are highlighted in red. As shown by the TWAS results plotted in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>, the known expression-regulated gene <em>crtRB1</em> has expression which is most strongly correlated (r = 0.309, <em>P</em> = 2.84e-5) with zeaxanthin abundance in our TWAS model that includes genetic and expression-derived covariates (see methods). <em>crtRB1</em> is not among the top MLM GWAS-detected genes in our study, but the detection of <em>crtRB1</em> by kernel TWAS is consistent with previous results (<a id="xref-ref-31-9" class="xref-bibr" href="#ref-31">Owens <em>et al.</em> 2014</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Yan <em>et al.</em> 2010</a>), highlighting this geneâs role as a principal determinant of grain carotenoids which acts through variable expression.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/3023/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Manhattan plots of zeaxanthin abundance A) mixed linear model GWAS accounting for kinship, B) kernel TWAS with PEER and genetic MDS PC covariates, C) MLM colored by TWAS significance, and D) Fisher&#x2019;s combined model of MLM and TWAS p-values using kernel expression. E) Fisher&#x2019;s combined model of MLM and multi-tissue TWAS p-values. The top five most associated genes are labeled and previously identified genes by Owens et al. (2014) are highlighted in red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-106138756" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Manhattan plots of zeaxanthin abundance A) mixed linear model GWAS accounting for kinship, B) kernel TWAS with PEER and genetic MDS PC covariates, C) MLM colored by TWAS significance, and D) Fisher&#x2019;s combined model of MLM and TWAS p-values using kernel expression. E) Fisher&#x2019;s combined model of MLM and multi-tissue TWAS p-values. The top five most associated genes are labeled and previously identified genes by Owens &lt;em&gt;et al.&lt;/em&gt; (2014) are highlighted in red.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/3023/F2.medium.gif" width="244" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/9/3023/F2.medium.gif" width="244" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/3023/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/3023/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494098" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-26" class="first-child">Manhattan plots of zeaxanthin abundance A) mixed linear model GWAS accounting for kinship, B) kernel TWAS with PEER and genetic MDS PC covariates, C) MLM colored by TWAS significance, and D) Fisherâs combined model of MLM and TWAS p-values using kernel expression. E) Fisherâs combined model of MLM and multi-tissue TWAS p-values. The top five most associated genes are labeled and previously identified genes by <a id="xref-ref-31-10" class="xref-bibr" href="#ref-31">Owens <em>et al.</em> (2014)</a> are highlighted in red.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">As is clear in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A, C</a> another zeaxanthin-implicated gene, zeaxanthin epoxidase, <em>zep1</em>, is detected by GWAS in our study (<a id="xref-ref-31-11" class="xref-bibr" href="#ref-31">Owens <em>et al.</em> 2014</a>). <em>zep1</em> expression is correlated (r= 0.232, <em>P</em> = 0.0014) with zeaxanthin abundance, but it is not among the fifty most significantly associated genes in our TWAS results, and would not be prioritized by TWAS alone. However, within the peak covering <em>zep1</em> in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2 A,C</a> the markers most strongly associated with zeaxanthin from the MLM GWAS results prioritize a different gene first, GRMZM2G127123, which lacks a known function. The linkage-independent kernel TWAS results also show nearly equal support for both genes, providing evidence that GRMZM2G127123 (r= 0.218, <em>P</em> = 0.0025) and <em>zep1</em> (r= 0.232, <em>P</em> = 0.0014) both affect zeaxanthin abundance. Both Fisherâs combined models using the single-tissue and multi-tissue TWAS results also support the importance of both genes.</p><p id="p-28">To test the capacity of the TWAS, GWAS, and combined methods to re-identify genes known to underlie QTL for another trait class, we examined the detected genes for the total tocotrienol trait measured by <a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Lipka <em>et al.</em> (2013)</a>. In <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> the most strongly associated variant identified by GWAS is on chromosome 9 nearest a gene of unknown function, GRMZM2G431524. However, as is illustrated in the MLM GWAS Manhattan plot in which points are colored by TWAS significance (c), the other points in the chromosome 9 peak are near other genes known to underlie QTL whose expression is variably associated with total tocotrienol abundance. These second and third most strongly associated genes based on proximity to the most significant markers identified by GWAS are GRMZM2G345544 (function unknown) and <em>hggt1</em>, which has been previously tied to total tocotrienol content (<a id="xref-ref-27-6" class="xref-bibr" href="#ref-27">Lipka <em>et al.</em> 2013</a>), and is essential for tocotrienol biosythesis. However, because <em>hggt1</em> expression is most strongly correlated with total tocotrienol measurements from among these first three genes in the chromosome 9 peak, the combined test using single tissue and multiple tissues of expression dat<em>a priori</em>tizes the known gene <em>hggt1</em> suggesting it is the functional gene in this region, consistent with previous evidence. This illustrates how the supplementary information from expression associations prioritizes likely causal genes that are not among the top hits of either individual expression or genotype-based methods.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/3023/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Manhattan plots of total tocotrienol abundance A) mixed linear model GWAS accounting for kinship, B) kernel TWAS with PEER and genetic MDS PC covariates, C) MLM colored by TWAS significance, and D) Fisher&#x2019;s combined model of MLM and TWAS p-values using kernel expression. E) Fisher&#x2019;s combined model of MLM and multi-tissue TWAS p-values. The top five most associated genes are labeled and previously identified genes Diepenbrock et al. (2017) are highlighted in red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-106138756" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Manhattan plots of total tocotrienol abundance A) mixed linear model GWAS accounting for kinship, B) kernel TWAS with PEER and genetic MDS PC covariates, C) MLM colored by TWAS significance, and D) Fisher&#x2019;s combined model of MLM and TWAS p-values using kernel expression. E) Fisher&#x2019;s combined model of MLM and multi-tissue TWAS p-values. The top five most associated genes are labeled and previously identified genes Diepenbrock &lt;em&gt;et al.&lt;/em&gt; (2017) are highlighted in red.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/3023/F3.medium.gif" width="244" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/9/3023/F3.medium.gif" width="244" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/3023/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/3023/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494139" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-29" class="first-child">Manhattan plots of total tocotrienol abundance A) mixed linear model GWAS accounting for kinship, B) kernel TWAS with PEER and genetic MDS PC covariates, C) MLM colored by TWAS significance, and D) Fisherâs combined model of MLM and TWAS p-values using kernel expression. E) Fisherâs combined model of MLM and multi-tissue TWAS p-values. The top five most associated genes are labeled and previously identified genes <a id="xref-ref-9-6" class="xref-bibr" href="#ref-9">Diepenbrock <em>et al.</em> (2017)</a> are highlighted in red.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Variance component estimation from TWAS- and GWAS-detected genes</h3><p id="p-30">To further assess the capacity of each method to correctly identify genes affecting each trait, an independent variance partitioning approach (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Gusev <em>et al.</em> 2014</a>; <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Rodgers-Melnick <em>et al.</em> 2016</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Speed <em>et al.</em> 2017</a>) was also performed. Using variants in a 1 Mb window around the ten top ranked genes identified in the Goodman diversity panel (<a id="xref-ref-11-11" class="xref-bibr" href="#ref-11">Flint-Garcia <em>et al.</em> 2005</a>) by GWAS alone, TWAS alone, and the combined method, separate kinship matrices were calculated. These relationship matrices were fit as random effects in separate models of phenotypic variance explained for traits measured in the NAM population, which is largely independent of the Goodman diversity panel in which the various mapping strategies were performed. The additive genetic variance explained by the variants underlying each kinship matrix was calculated providing an estimate of heritability explained by the genes identified by each method.</p><p id="p-31">Using variance partitioning across all NAM families, we found some advantage for including expression data in detecting likely functional regions of the genome (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Among the tocochromanol kernel traits (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>), eight out of ten traits exist in which TWAS or the Fisherâs combined method is superior to GWAS alone (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a>). Heritable variance explained on a per-trait basis by either the TWAS alone or the Fisherâs combined method showed about 25% improvement on average over the MLM GWAS, with notable advantage for alpha-tocotrienol (40%), gamma-tocotrienol (41%) and total tocopherol (43%). For more complex field-based agronomic traits, the multi-tissue TWAS or Fisherâs combined method also showed an advantage over GWAS alone in 16 out of 22 agronomic traits (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). On average, the multi-tissue TWAS had 24% improvement over GWAS alone, while the FisherGWASmultiTWAS had notable advantage for kernel number (24%), leaf width (15%), and node number below ear (19%). Based on mean heritable variance across traits per trait class, the combined Fisherâs test explained the most heritability among the models; it showed 4â8% improvement for the tocochromanol kernel traits (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4A</a> inset). However, little improvement was observed for agronomic traits likely due to trait complexity (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4B</a> inset). Because previously known genes are more often re-identified in the top 1% of hits by combining GWAS and TWAS (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), the variance explained by markers near detected genes also reflect this advantage on heritability with known oligogenic architecture.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/3023/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Variance partitioning of heritable variation using all NAM families for A) tocochromanol traits and B) agronomic traits. Vertical barplots represent the heritability estimated from kinship matrices made from the genetic regions adjacent to the top 10 ranked genes mapped by MLM GWAS, kernel-based TWAS, multi-tissue TWAS, the Fisher&#x2019;s combined test of the MLM GWAS + kernel-based TWAS, and the Fisher&#x2019;s combined test of the MLM GWAS + multi-tissue TWAS. Horizontal barplots compare model based on mean heritability across traits per trait class. Heritability explained by using all SNPs for each trait was put at the top of each grouped barplot." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-106138756" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Variance partitioning of heritable variation using all NAM families for A) tocochromanol traits and B) agronomic traits. Vertical barplots represent the heritability estimated from kinship matrices made from the genetic regions adjacent to the top 10 ranked genes mapped by MLM GWAS, kernel-based TWAS, multi-tissue TWAS, the Fisher&#x2019;s combined test of the MLM GWAS + kernel-based TWAS, and the Fisher&#x2019;s combined test of the MLM GWAS + multi-tissue TWAS. Horizontal barplots compare model based on mean heritability across traits per trait class. Heritability explained by using all SNPs for each trait was put at the top of each grouped barplot.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/3023/F4.medium.gif" width="342" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/9/3023/F4.medium.gif" width="342" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/3023/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/3023/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494207" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-32" class="first-child">Variance partitioning of heritable variation using all NAM families for A) tocochromanol traits and B) agronomic traits. Vertical barplots represent the heritability estimated from kinship matrices made from the genetic regions adjacent to the top 10 ranked genes mapped by MLM GWAS, kernel-based TWAS, multi-tissue TWAS, the Fisherâs combined test of the MLM GWAS + kernel-based TWAS, and the Fisherâs combined test of the MLM GWAS + multi-tissue TWAS. Horizontal barplots compare model based on mean heritability across traits per trait class. Heritability explained by using all SNPs for each trait was put at the top of each grouped barplot.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">We further tested the heritability explained by the top ten ranked genes identified by each method using family-based variance partitioning (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Heritable variance was decomposed for each NAM family, giving 24 independent tests of variance partitioning for each trait tallying a total of 3,840 independent tests (24 families * 5 models * 32 traits). To evaluate the best winning model for each trait, we took the sum of heritable variance across 24 NAM families (hereafter, summed heritability). Based on the same set of genes identified from each model, our results illustrate the differing levels of heritability among families for both tocochromanol (Fig. S1, <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>) and agronomic traits (Fig. S2, and S3; <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>). For Î±-tocotrienol, which is an oligogenic trait, the FisherGWASTWAS method explained the most heritability in 18 out of 24 NAM families (Figure S1a), giving a fourfold advantage on summed heritability over either GWAS or TWAS alone (Figure S1b; <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5A</a>). The FisherGWASTWAS method captured the most summed heritability in 10 tocochromanol traits (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5A</a> inset), consistent with what we found in variance partitioning using all NAM families for tocochromanol traits (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4A</a>). On a per-trait basis, we note that the kernel-based TWAS or the FisherGWASTWAS was the winning method for eight out of 10 tocochromanol traits. We do see a similar pattern in 19 out of 22 field-based complex traits in which either the multi-tissue TWAS or FisherGWASmultiTissueTWAS explained the most heritability (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5B</a>). We see greater advantage of the FisherGWASmultiTissueTWAS over the GWAS MLM for tassel primary branch (54%), cob length (103%), kernel number (112%), ear mass (98%) and total kernel weight (106%) (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5A</a>). For the more complex traits such as plant height, the multi-tissue TWAS was the winning model, which explained about twofold higher heritability than the GWAS alone (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5B</a>, Fig. S3). We found that in 16 NAM families, the multi-tissue TWAS explained the most heritability among other models for plant height. Based on total summed heritability across 22 agronomic traits (<a id="xref-fig-5-9" class="xref-fig" href="#F5">Figure 5B</a> inset), the FisherGWASmultiTissueTWAS and multi-tissue TWAS showed a 15% and 17% improvement in heritability explained over the GWAS MLM alone, respectively.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/3023/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Family-based variance partitioning on individual NAM family. Heritability for each trait was estimated for each of 24 NAM families using kinship matrices made from the genetic regions adjacent to the top 10 ranked genes mapped by MLM GWAS, kernel-based TWAS, multi-tissue TWAS, the Fisher&#x2019;s combined test of the MLM GWAS + kernel-based TWAS, and the Fisher&#x2019;s combined test of the MLM GWAS + multi-tissue TWAS. There were a total of 24 independent tests for each trait-model combination. Heritability estimates were then added together (hereafter, summed heritability) for A) tocochromanol traits and B) agronomic traits. Horizontal barplots compare model based on total summed heritability across traits per trait class." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-106138756" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Family-based variance partitioning on individual NAM family. Heritability for each trait was estimated for each of 24 NAM families using kinship matrices made from the genetic regions adjacent to the top 10 ranked genes mapped by MLM GWAS, kernel-based TWAS, multi-tissue TWAS, the Fisher&#x2019;s combined test of the MLM GWAS + kernel-based TWAS, and the Fisher&#x2019;s combined test of the MLM GWAS + multi-tissue TWAS. There were a total of 24 independent tests for each trait-model combination. Heritability estimates were then added together (hereafter, summed heritability) for A) tocochromanol traits and B) agronomic traits. Horizontal barplots compare model based on total summed heritability across traits per trait class.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/3023/F5.medium.gif" width="352" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/9/3023/F5.medium.gif" width="352" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/3023/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/3023/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494172" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-34" class="first-child">Family-based variance partitioning on individual NAM family. Heritability for each trait was estimated for each of 24 NAM families using kinship matrices made from the genetic regions adjacent to the top 10 ranked genes mapped by MLM GWAS, kernel-based TWAS, multi-tissue TWAS, the Fisherâs combined test of the MLM GWAS + kernel-based TWAS, and the Fisherâs combined test of the MLM GWAS + multi-tissue TWAS. There were a total of 24 independent tests for each trait-model combination. Heritability estimates were then added together (hereafter, summed heritability) for A) tocochromanol traits and B) agronomic traits. Horizontal barplots compare model based on total summed heritability across traits per trait class.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><p id="p-35">By far the majority of efforts to dissect the architecture of terminal phenotypes have relied on associations with genetic variants; this capacity to link genotype to phenotype has recently been accelerated by the plummeting cost of sequencing. The more recent advent of technologies which permit the quantification of endophenotypes like mRNA, metabolite, or protein abundance now enable mapping and trait dissection to be done between intermediate levels of biological organization. Assaying and associating these endophenotypes with traits of interest provides insight on biological mechanisms, serves as an independent source of evidence of associations, and facilitates prioritizing potentially causal variation while linking genes directly to traits in a way that potentially integrates the effects of multiple independent genetic variants. Here, we illustrated the utility of using a large RNA-seq resource in maize (<a id="xref-ref-23-7" class="xref-bibr" href="#ref-23">Kremling <em>et al.</em> 2018</a>) for transcriptome-wide association studies and integrating these results with associations based on genetic variation.</p><p id="p-36">We find evidence supporting the inclusion of transcriptome-wide variation in addition to genetic variation in models seeking to associate traits to underlying and likely causal genes in diverse maize lines, especially when the goal is to infer function of genes underlying oligogenic traits. Across tocochromanol trait classes, the inclusion of TWAS results enables more frequent detection of known causal genes and helps to prioritize novel candidate genes in the profiled panel. Crucially, transcriptional variation alone does not improve over genotype-based associations, but it is in combination with genotypic information that the power of gene detection is increased.</p><p id="p-37">As we demonstrate here, TWAS in combination with GWAS enhances the capacity to prioritize candidate genes over the use of GWAS alone. Given that more than half of detections are supported by TWAS (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>), our results also reveal much of the functional variation for these traits to be regulatory. While not all previously identified genes are detected by TWAS, this is likely a combination of insufficient power compared to the previous association studies in the NAM population with &gt;16x as many observations (<a id="xref-ref-9-7" class="xref-bibr" href="#ref-9">Diepenbrock <em>et al.</em> 2017</a>), the sampling of a single time point per tissue, and the fact that not all functional variation is regulatory. Despite these limitations, TWAS adds value to GWAS mapping alone and increases the power to re-detect known genes. Our finding that TWAS alone is a valid method for finding true gene-trait associations is consistent with the recent findings of <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Lin <em>et al.</em> (2017)</a> and colleagues despite the difference between the eRD-GWAS and TWAS models. However, our results differ in that we demonstrate that a combined test integrating TWAS and GWAS yields a more powerful test than either method individually when it comes to re-identifying known genes underlying oligogenic traits (<a id="xref-ref-9-8" class="xref-bibr" href="#ref-9">Diepenbrock <em>et al.</em> 2017</a>).</p><p id="p-38">We also note that our efforts to validate our TWAS and GWAS detections differ from <a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Lin <em>et al.</em> (2017)</a>. In contrast to comparing the overlap of the detections by GWAS and TWAS in the same study, we compared our detections to previously known genes found in a largely independent set of germplasm, namely the NAM population (<a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Yu <em>et al.</em> 2008</a>), which was used to find tocochromanol associations (<a id="xref-ref-9-9" class="xref-bibr" href="#ref-9">Diepenbrock <em>et al.</em> 2017</a>). Also, in contrast to the previously published study, we did not perform our cross-validation analysis in the same set of germplasm in which discovery was conducted by GWAS and TWAS to assess accuracy. Using variance partitioning in the largely independent NAM population, we found similar levels of variance explained by the genes detected by each method in the Goodman diversity panel (<a id="xref-ref-11-12" class="xref-bibr" href="#ref-11">Flint-Garcia <em>et al.</em> 2005</a>), illustrating that even when the identified genes are tested in an outside population, the detections of the transcriptome-only and combined methods are found to be valid and explain similar amounts of variance to the genotype-based methods (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figures 4</a>, <a id="xref-fig-5-10" class="xref-fig" href="#F5">5</a>). This is roughly consistent with the cross-validation results comparing SNP_BayesB and eRD-GWAS presented in Table S4 by Lin and colleagues (<a id="xref-ref-26-5" class="xref-bibr" href="#ref-26">Lin <em>et al.</em> 2017</a>). However, the previously published results show an advantage for eRD-GWAS for only one of fourteen traits, while on the basis of variance partitioning for kernel traits we find an advantage for the kernel-based TWAS or the Fisherâs combined model for nine of the ten kernel-based traits for which measurements in NAM exist.</p><p id="p-39">In further contrast to the previously published work (<a id="xref-ref-26-6" class="xref-bibr" href="#ref-26">Lin <em>et al.</em> 2017</a>), none of the SNPs used in our GWAS or variance partitioning methods were derived from RNA-seq data, allowing for less bias toward expressed genes and giving the genotype-based tests more independence from the expression-based tests. In the previous work, more than 0.9M of the 1.2 M genetic variants were derived from the alignment of RNA-seq reads (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Leiboff <em>et al.</em> 2015</a>; <a id="xref-ref-26-7" class="xref-bibr" href="#ref-26">Lin <em>et al.</em> 2017</a>), potentially confounding the ability to make associations by GWAS with the presence of an expressed gene, and thus limiting the power of the genotype-based GWAS to make associations which are independent of expression.</p><p id="p-40">It is striking that even in diverse maize lines where linkage decays quickly (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Wallace <em>et al.</em> 2014</a>) and thus the power to resolve mapping peaks to individual genes is high, TWAS provides a valuable supplement to genetic mapping alone. This benefit of TWAS would be compounded in species or populations in which resolution is limited. Additionally, by imputing expression values based on local/<em>cis</em> haplotype, as has been successfully shown in humans (<a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Pasaniuc and Price 2017</a>), the utility of TWAS could potentially be extended further in maize. Imputing expression to a larger panel would permit the exploitation of previously measured phenotypes across a much larger set of individuals which have not been expression profiled. By imputing only the local/<em>cis</em> genetic component of expression, and implicitly averaging over <em>trans</em> and environmental effects, the capacity to attribute field phenotypes to the genetic component of expression would likely be further improved.</p><p id="p-41">The lack of improvement in re-detecting known tocochromanol traits by the multi-tissue TWAS models alone or as part of the Fisherâs combined tests is notable, but unsurprising for these genetically simple and very tissue specific traits. This lack of improvement indicates that kernel-based expression alone is most predictive of the kernel-based metabolites and accuracy is not improved by the incorporation of all other tissues. Rather than comparing the inclusion of all tissues <em>vs.</em> kernels only, in the future a variable (tissue) selection TWAS approach should be used in which can remove uninformative terms from the model rather than including them but giving them a very small coefficient. It is also plausible that for more genetically complex traits which are also affected by expression across tissues, the multi-tissue TWAS results are more likely to be informative.</p><p id="p-42">A further cause of the limited improvement for the kernel TWAS or Fisherâs combined test seen in the variance partitioning results is likely because GWAS identifies genomic regions which, when expanded to a 1 Mb window, could cover the functional variants. Furthermore, while the correct functional gene may not be prioritized by GWAS, if the trait is affected by genetic regulation rather than coding sequence change, the sites near the GWAS hit may in fact be more functional than those near the mechanistically significant gene itself even if they are misattributed to the incorrect proximal gene. Using a large independent diverse panel with very low LD to assess the heritability explained by the SNPs identified by each method may also provide a better estimate as the functional variants are not as easily tagged over long distances.</p><p id="p-43">While the utility of expression endophenotypes in dissecting traits has been demonstrated here, it should be noted that associations made between endophenotypes and terminal phenotypes are inherently more susceptible to environmental effects than genotype-based associations. This susceptibility to environmental effects likely allows us to associate only the environmentally independent heritable fraction of expression with phenotype in our study, especially because expression data were collected from separate plants than those for which terminal phenotypes were measured. Given that in endophenotype-based association studies, like TWAS, environmental variation separately impacts and increases error in both the independent and dependent variables, methods like TWAS alone may plausibly be expected to perform more poorly than genetics-based associations. However, this shortcoming is partially compensated for by the more direct link between endophenotype and terminal phenotype and the potential discovery of mechanism. The collection of expression data from the same plants and conditions in which the phenotypes are collected would likely benefit the dissection of genotype by environment interactions by highlighting the impact of variation in expression for a specific gene within an environment, but cannot be examined here as terminal phenotypes and expression values were calculated from separate environments and years.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-44">We thank Sara Miller for thorough reading of the manuscript and assistance with syntax. This work was supported by the US Department of AgricultureâAgricultural Research Service and the National Science Foundation grants IOS-0922493 and IOS-1238014 to E.S.B. The National Science Foundation Graduate Research Fellowship Program grant DGE-1650441 and the Section of Plant Breeding and Genetics at Cornell University provided support to K.A.G.K. The information, data, or work presented herein was funded in part by the Advanced Research Projects Agency-Energy (ARPA-E), U.S. Department of Energy, under Award Number DE-AR0000598. The views and opinions of authors expressed herein do not necessarily state or reflect those of the United States Government or any agency thereof.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-2"><a class="rev-xref" href="#xref-fn-2-1">âµ</a><span class="fn-label">3</span> Department of Plant Sciences, Plant Reproductive Biology, Room 1109 University of California Davis, CA 95616</p></li><li class="fn" id="fn-3"><p id="p-45">Supplemental material available at Figshare: <a href="https://figshare.com/s/ef57544b4d09d5c55131">https://figshare.com/s/ef57544b4d09d5c55131</a>.</p></li><li class="fn" id="fn-4"><p id="p-46"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-11-26"><span class="received-label">Received </span>November 26, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-07-18"><span class="accepted-label">Accepted </span>July 18, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2019 Kremling <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.1" data-doi="10.1038/nrg3891"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albert</span>,  <span class="cit-name-given-names">F. W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Kruglyak</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">The role of regulatory variation in complex traits and disease.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">197</span>â<span class="cit-lpage">212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3891</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D16%26rft.spage%253D197%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3891%26rft_id%253Dinfo%253Apmid%252F25707927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3891&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25707927&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F3023.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.2" data-doi="10.1126/science.1260793"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Battle</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Khan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. H.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Mitrano</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Ford</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Genomic variation. impact of regulatory variation from RNA to protein.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">347</span>: <span class="cit-fpage">664</span>â<span class="cit-lpage">667</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1260793</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1260793%26rft_id%253Dinfo%253Apmid%252F25657249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDcvNjIyMi82NjQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMzAyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.3" data-doi="10.1093/bioinformatics/btm308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>,  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. E.</span> <span class="cit-name-surname">Kroon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. M.</span> <span class="cit-name-surname">Casstevens</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Ramdoss</span></span> <span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2007</span>â<span class="cit-article-title">TASSEL: Software for association mapping of complex traits in diverse samples.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2633</span>â<span class="cit-lpage">2635</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btm308</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm308%26rft_id%253Dinfo%253Apmid%252F17586829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm308&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17586829&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F3023.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250673800021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.4" data-doi="10.1093/gigascience/gix134"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bukowski</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Guo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Lu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Zou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">He</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span>â<span class="cit-article-title">Construction of the third generation zea mays haplotype map.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gigascience/gix134</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgigascience%252Fgix134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gigascience/gix134&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.5" data-doi="10.1016/j.cell.2008.06.030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carroll</span>,  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Evo-devo and an expanding evolutionary synthesis: A genetic theory of morphological evolution.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">25</span>â<span class="cit-lpage">36</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2008.06.030</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DCarroll%26rft.auinit1%253DS.%2BB.%26rft.volume%253D134%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D36%26rft.atitle%253DEvo-devo%2Band%2Ban%2Bexpanding%2Bevolutionary%2Bsynthesis%253A%2Ba%2Bgenetic%2Btheory%2Bof%2Bmorphological%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.06.030%26rft_id%253Dinfo%253Apmid%252F18614008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.06.030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18614008&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F3023.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257513100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.6" data-doi="10.1038/nature18270"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chick</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. C.</span> <span class="cit-name-surname">Munger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Simecek</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. L.</span> <span class="cit-name-surname">Huttlin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Choi</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Defining the consequences of genetic variation on a proteome-wide scale.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">534</span>: <span class="cit-fpage">500</span>â<span class="cit-lpage">505</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature18270</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D534%26rft.spage%253D500%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature18270%26rft_id%253Dinfo%253Apmid%252F27309819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature18270&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27309819&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F3023.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dewey</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span>âMetap: Meta-analysis of significance values r-package.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.8" data-doi="10.1371/journal.pbio.1000294"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dickson</span>,  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Hakonarson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. B.</span> <span class="cit-name-surname">Goldstein</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Rare variants create synthetic genome-wide associations.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e1000294</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.1000294</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DDickson%26rft.auinit1%253DS.%2BP.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253De1000294%26rft.epage%253De1000294%26rft.atitle%253DRare%2Bvariants%2Bcreate%2Bsynthetic%2Bgenome-wide%2Bassociations.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000294%26rft_id%253Dinfo%253Apmid%252F20126254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000294&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20126254&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F3023.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.9" data-doi="10.1105/tpc.17.00475"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Diepenbrock</span>,  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. B.</span> <span class="cit-name-surname">Kandianis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Lipka</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Magallanes-Lundback</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Vaillancourt</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Novel loci underlie natural variation in vitamin E levels in maize grain.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">2374</span>â<span class="cit-lpage">2392</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.17.00475</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.17.00475%26rft_id%253Dinfo%253Apmid%252F28970338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyOS8xMC8yMzc0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzMwMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.10" data-doi="10.1007/s001220050776"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dubreuil</span>,  <span class="cit-name-given-names">P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Charcosset</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span>â<span class="cit-article-title">Genetic diversity within and among maize populations: A comparison between isozyme and nuclear RFLP loci.</span> <abbr class="cit-jnl-abbrev">TAG</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">577</span>â<span class="cit-lpage">587</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s001220050776</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG%26rft.volume%253D96%26rft.spage%253D577%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050776&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.11" data-doi="10.1111/j.1365-313X.2005.02591.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flint-Garcia</span>,  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Thuillet</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Yu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Pressoir</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Romero</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Maize association population: A high-resolution platform for quantitative trait locus dissection.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1054</span>â<span class="cit-lpage">1064</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-313X.2005.02591.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DFlint-Garcia%26rft.auinit1%253DS.%2BA.%26rft.volume%253D44%26rft.issue%253D6%26rft.spage%253D1054%26rft.epage%253D1064%26rft.atitle%253DMaize%2Bassociation%2Bpopulation%253A%2Ba%2Bhigh-resolution%2Bplatform%2Bfor%2Bquantitative%2Btrait%2Blocus%2Bdissection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2005.02591.x%26rft_id%253Dinfo%253Apmid%252F16359397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2005.02591.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16359397&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F3023.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233776900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.12" data-doi="10.1038/ncomms3832"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Cheng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Linghu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Yang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Kang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">RNA sequencing reveals the complex regulatory network in the maize kernel.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">2832</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms3832</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D4%26rft.spage%253D2832%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms3832%26rft_id%253Dinfo%253Apmid%252F24343161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms3832&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24343161&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F3023.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.13" data-doi="10.1038/nrg3118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>,  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Rare and common variants: Twenty arguments.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">135</span>â<span class="cit-lpage">145</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3118</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D13%26rft.spage%253D135%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3118%26rft_id%253Dinfo%253Apmid%252F22251874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3118&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22251874&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F3023.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.14" data-doi="10.1126/science.1262110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">GTEx Consortium</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Human genomics. the genotype-tissue expression (GTEx) pilot analysis: Multitissue gene regulation in humans.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">348</span>: <span class="cit-fpage">648</span>â<span class="cit-lpage">660</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1262110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DThe%2BGTEx%2BConsortium%26rft.auinit1%253D%2B%26rft.volume%253D348%26rft.issue%253D6235%26rft.spage%253D648%26rft.epage%253D660%26rft.atitle%253DThe%2BGenotype-Tissue%2BExpression%2B%2528GTEx%2529%2Bpilot%2Banalysis%253A%2BMultitissue%2Bgene%2Bregulation%2Bin%2Bhumans%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1262110%26rft_id%253Dinfo%253Apmid%252F25954001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDgvNjIzNS82NDgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMzAyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.15" data-doi="10.1016/j.ajhg.2014.10.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gusev</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. H.</span> <span class="cit-name-surname">Lee</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Trynka</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Finucane</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Vilhjalmsson</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Partitioning heritability of regulatory and cell-type-specific variants across 11 common diseases.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">535</span>â<span class="cit-lpage">552</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2014.10.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D95%26rft.spage%253D535%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2014.10.004%26rft_id%253Dinfo%253Apmid%252F25439723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2014.10.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25439723&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F3023.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.16" data-doi="10.1038/ng.3506"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gusev</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ko</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Shi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Bhatia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Chung</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Integrative approaches for large-scale transcriptome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">245</span>â<span class="cit-lpage">252</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3506</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D48%26rft.spage%253D245%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3506%26rft_id%253Dinfo%253Apmid%252F26854917%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3506&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26854917&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F3023.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.17" data-doi="10.1126/science.1150255"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harjes</span>,  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. R.</span> <span class="cit-name-surname">Rocheford</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Bai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. P.</span> <span class="cit-name-surname">Brutnell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. B.</span> <span class="cit-name-surname">Kandianis</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Natural genetic variation in lycopene epsilon cyclase tapped for maize biofortification.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">319</span>: <span class="cit-fpage">330</span>â<span class="cit-lpage">333</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1150255</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHarjes%26rft.auinit1%253DC.%2BE.%26rft.volume%253D319%26rft.issue%253D5861%26rft.spage%253D330%26rft.epage%253D333%26rft.atitle%253DNatural%2BGenetic%2BVariation%2Bin%2BLycopene%2BEpsilon%2BCyclase%2BTapped%2Bfor%2BMaize%2BBiofortification%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1150255%26rft_id%253Dinfo%253Apmid%252F18202289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTkvNTg2MS8zMzAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMzAyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.18" data-doi="10.1038/nrg3931"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henn</span>,  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. R.</span> <span class="cit-name-surname">Botigue</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. D.</span> <span class="cit-name-surname">Bustamante</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. G.</span> <span class="cit-name-surname">Clark</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Gravel</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Estimating the mutation load in human genomes.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">333</span>â<span class="cit-lpage">343</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3931</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D16%26rft.spage%253D333%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3931%26rft_id%253Dinfo%253Apmid%252F25963372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3931&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25963372&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F3023.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.19" data-doi="10.1105/tpc.113.119982"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>,  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Foerster</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Johnson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. S.</span> <span class="cit-name-surname">Sekhon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Muttoni</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Insights into the maize pan-genome and pan-transcriptome.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">121</span>â<span class="cit-lpage">135</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.113.119982</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.113.119982%26rft_id%253Dinfo%253Apmid%252F24488960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjI2LzEvMTIxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzMwMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.20" data-doi="10.1038/ng.2309"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hufford</span>,  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Xu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">van Heerwaarden</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Pyhajarvi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Chia</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Comparative population genomics of maize domestication and improvement.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">808</span>â<span class="cit-lpage">811</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2309</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHufford%26rft.auinit1%253DM.%2BB.%26rft.volume%253D44%26rft.issue%253D7%26rft.spage%253D808%26rft.epage%253D811%26rft.atitle%253DComparative%2Bpopulation%2Bgenomics%2Bof%2Bmaize%2Bdomestication%2Band%2Bimprovement.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2309%26rft_id%253Dinfo%253Apmid%252F22660546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2309&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22660546&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F3023.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.21" data-doi="10.1038/hdy.2011.103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hung</span>,  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Browne</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Guill</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Coles</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Eller</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">The relationship between parental genetic or phenotypic divergence and progeny variation in the maize nested association mapping population.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">490</span>â<span class="cit-lpage">499</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2011.103</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHung%26rft.auinit1%253DH.%2BY.%26rft.volume%253D108%26rft.issue%253D5%26rft.spage%253D490%26rft.epage%253D499%26rft.atitle%253DThe%2Brelationship%2Bbetween%2Bparental%2Bgenetic%2Bor%2Bphenotypic%2Bdivergence%2Band%2Bprogeny%2Bvariation%2Bin%2Bthe%2Bmaize%2Bnested%2Bassociation%2Bmapping%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2011.103%26rft_id%253Dinfo%253Apmid%252F22027895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2011.103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22027895&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F3023.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.22" data-doi="10.1073/pnas.1317811111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Juntawong</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Girke</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bazin</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bailey-Serres</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Translational dynamics revealed by genome-wide profiling of ribosome footprints in arabidopsis.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">E203</span>â<span class="cit-lpage">E212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1317811111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1317811111%26rft_id%253Dinfo%253Apmid%252F24367078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTExLzEvRTIwMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOS8zMDIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.23" data-doi="10.1038/nature25966"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kremling</span>,  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Chen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Su</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Lepak</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Romay</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span>â<span class="cit-article-title">Dysregulation of expression correlates with rare allele burden and fitness loss in maize.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">555</span>: <span class="cit-fpage">520</span>â<span class="cit-lpage">523</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature25966</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D555%26rft.spage%253D520%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature25966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature25966&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.24" data-doi="10.1038/nrg2719"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Law</span>,  <span class="cit-name-given-names">J. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Jacobsen</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Establishing, maintaining and modifying DNA methylation patterns in plants and animals.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">204</span>â<span class="cit-lpage">220</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2719</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DLaw%26rft.volume%253D11%26rft.issue%253D3%26rft.spage%253D204%26rft.epage%253D220%26rft.atitle%253DEstablishing%252C%2Bmaintaining%2Band%2Bmodifying%2BDNA%2Bmethylation%2Bpatterns%2Bin%2Bplants%2Band%2Banimals.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2719%26rft_id%253Dinfo%253Apmid%252F20142834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2719&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20142834&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F3023.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274649900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.25" data-doi="10.1038/ncomms9974"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leiboff</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. C.</span> <span class="cit-name-surname">Hu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Todt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Yang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Genetic control of morphometric diversity in the maize shoot apical meristem.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">8974</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms9974</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D6%26rft.spage%253D8974%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms9974%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms9974&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Yang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Liu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span>âSubstantial contribution of genetic variation in the expression of transcription factors to phenotypic variation revealed by eRD-GWAS. Genome Biol. 18: 192-017-1328-6.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.27" data-doi="10.1534/g3.113.006148"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>,  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Gore</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Magallanes-Lundback</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Mesberg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Lin</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Genome-wide association study and pathway-level analysis of tocochromanol levels in maize grain.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1287</span>â<span class="cit-lpage">1299</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.113.006148</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.006148%26rft_id%253Dinfo%253Apmid%252F23733887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjMvOC8xMjg3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzMwMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.28" data-doi="10.1038/nmeth.1681"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lippert</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Listgarten</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Kadie</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. I.</span> <span class="cit-name-surname">Davidson</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">FaST linear mixed models for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">833</span>â<span class="cit-lpage">835</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1681</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLippert%26rft.auinit1%253DC.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253D833%26rft.epage%253D835%26rft.atitle%253DFaST%2Blinear%2Bmixed%2Bmodels%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1681%26rft_id%253Dinfo%253Apmid%252F21892150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1681&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21892150&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F3023.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295358000017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.29" data-doi="10.1016/j.ajhg.2017.01.031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mancuso</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Shi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Goddard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Kichaev</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Gusev</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Integrating Gene Expression with Summary Association Statistics to Identify Genes Associated with 30 complex traits.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">473</span>â<span class="cit-lpage">487</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2017.01.031</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D100%26rft.spage%253D473%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2017.01.031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2017.01.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-book" id="cit-9.9.3023.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mayr</span>,  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span>â<span class="cit-source"><em>Populations</em>, <em>species</em>, <em>and evolution: An abridgment of animal species and evolution</em></span>, <span class="cit-publ-name">Harvard University Press, Cambridge, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.31" data-doi="10.1534/genetics.114.169979"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Owens</span>,  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Lipka</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Magallanes-Lundback</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Tiede</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. H.</span> <span class="cit-name-surname">Diepenbrock</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">A foundation for provitamin A biofortification of maize: Genome-wide association and genomic prediction models of carotenoid levels.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">1699</span>â<span class="cit-lpage">1716</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.169979</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.169979%26rft_id%253Dinfo%253Apmid%252F25258377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OC80LzE2OTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMzAyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.32" data-doi="10.1038/nrg.2016.142"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pasaniuc</span>,  <span class="cit-name-given-names">B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Price</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Dissecting the genetics of complex traits using summary association statistics.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">117</span>â<span class="cit-lpage">127</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg.2016.142</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D18%26rft.spage%253D117%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg.2016.142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg.2016.142&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.33" data-doi="10.1038/ng.1033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Czedik-Eysenberg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Grieder</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Lisec</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Technow</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Genomic and metabolic prediction of complex heterotic traits in hybrid maize.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">217</span>â<span class="cit-lpage">220</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.1033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DRiedelsheimer%26rft.auinit1%253DC.%26rft.volume%253D44%26rft.issue%253D2%26rft.spage%253D217%26rft.epage%253D220%26rft.atitle%253DGenomic%2Band%2Bmetabolic%2Bprediction%2Bof%2Bcomplex%2Bheterotic%2Btraits%2Bin%2Bhybrid%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.1033%26rft_id%253Dinfo%253Apmid%252F22246502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.1033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22246502&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F3023.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.34" data-doi="10.1073/pnas.1525244113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rodgers-Melnick</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. L.</span> <span class="cit-name-surname">Vera</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. W.</span> <span class="cit-name-surname">Bass</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. S.</span> <span class="cit-name-surname">Buckler</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Open chromatin reveals the functional maize genome.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">E3177</span>â<span class="cit-lpage">E3184</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1525244113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1525244113%26rft_id%253Dinfo%253Apmid%252F27185945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEzLzIyL0UzMTc3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzMwMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.35" data-doi="10.1093/bioinformatics/bts163"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shabalin</span>,  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Matrix eQTL: Ultra fast eQTL analysis via large matrix operations.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1353</span>â<span class="cit-lpage">1358</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts163</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts163%26rft_id%253Dinfo%253Apmid%252F22492648%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts163&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22492648&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F3023.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304053300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.36" data-doi="10.1038/ng.3865"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Speed</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Cai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Johnson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Nejentsev</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Balding</span></span>;<span class="cit-auth cit-collab"> UCLEB Consortium</span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Reevaluation of SNP heritability in complex human traits.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">986</span>â<span class="cit-lpage">992</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3865</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D49%26rft.spage%253D986%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3865&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.37" data-doi="10.1038/ng1946"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tishkoff</span>,  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. A.</span> <span class="cit-name-surname">Reed</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ranciaro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. F.</span> <span class="cit-name-surname">Voight</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Babbitt</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Convergent adaptation of human lactase persistence in africa and europe.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">31</span>â<span class="cit-lpage">40</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1946</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTishkoff%26rft.auinit1%253DS.%2BA.%26rft.volume%253D39%26rft.issue%253D1%26rft.spage%253D31%26rft.epage%253D40%26rft.atitle%253DConvergent%2Badaptation%2Bof%2Bhuman%2Blactase%2Bpersistence%2Bin%2BAfrica%2Band%2BEurope.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1946%26rft_id%253Dinfo%253Apmid%252F17159977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1946&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17159977&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F3023.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243136500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.38" data-doi="10.1371/journal.pgen.1004845"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wallace</span>,  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Bradbury</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Gibon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Stitt</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Association mapping across numerous traits reveals patterns of functional variation in maize.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-elocation-id">e1004845</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004845</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004845%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004845%26rft_id%253Dinfo%253Apmid%252F25474422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004845&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25474422&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F3023.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.39" data-doi="10.1093/nar/gkt1229"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Welter</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">MacArthur</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Morales</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Burdett</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Hall</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">The NHGRI GWAS catalog, a curated resource of SNP-trait associations.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">D1001</span>â<span class="cit-lpage">D1006</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkt1229</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1229%26rft_id%253Dinfo%253Apmid%252F24316577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt1229&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24316577&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F3023.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331139800147&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.40" data-doi="10.1073/pnas.1011739108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wisser</span>,  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Kolkman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Patzoldt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Holland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Yu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Multivariate analysis of maize disease resistances suggests a pleiotropic genetic basis and implicates a GST gene.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">7339</span>â<span class="cit-lpage">7344</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1011739108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1011739108%26rft_id%253Dinfo%253Apmid%252F21490302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzE4LzczMzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMzAyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.41" data-doi="10.1038/ng.551"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. B.</span> <span class="cit-name-surname">Kandianis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. E.</span> <span class="cit-name-surname">Harjes</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Bai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. H.</span> <span class="cit-name-surname">Kim</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Rare genetic variation at <em>Zea mays</em> crtRB1 increases beta-carotene in maize grain.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">322</span>â<span class="cit-lpage">327</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.551</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYan%26rft.auinit1%253DJ.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D322%26rft.epage%253D327%26rft.atitle%253DRare%2Bgenetic%2Bvariation%2Bat%2BZea%2Bmays%2BcrtRB1%2Bincreases%2Bbeta-carotene%2Bin%2Bmaize%2Bgrain.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.551%26rft_id%253Dinfo%253Apmid%252F20305664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.551&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20305664&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F3023.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276150500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.3023.42" data-doi="10.1534/genetics.107.074245"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Holland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">McMullen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. S.</span> <span class="cit-name-surname">Buckler</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Genetic design and statistical power of nested association mapping in maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">539</span>â<span class="cit-lpage">551</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.074245</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.074245%26rft_id%253Dinfo%253Apmid%252F18202393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc4LzEvNTM5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzMwMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/9/3023.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/9/3009" title="Cyfip1 Haploinsufficiency Increases Compulsive-Like Behavior and Modulates Palatable Food Intake in Mice: Dependence on Cyfip2 Genetic Background, Parent-of Origin, and Sex" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/9/3035" title="Correlating Genotype and Phenotype in the Asexual Yeast Candida orthopsilosis Implicates ZCF29 in Sensitivity to Caffeine" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/9"><span>Volume 9 Issue 9, September 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/9.cover-source.jpg?itok=4kZTDz_V" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/3023" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-wIUKP-ZlgI7IZrhtoH0v02W1ellWFNWQpmK1kqaihM4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/9/3023" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/9/3023" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/9/3023" class="active" data-icon-position="" data-hide-link-title="0">Transcriptome-Wide Association Supplements Genome-Wide Association in Zea mays</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494050" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Hir-_UpLEQraiwJh0f1NYbXqSR5Sk7ovyCDZEDQoGN4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/9/3023.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/9/3023" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-igonOoEgB4B5GkYYn2qHffUBsWNycv1IU0lO68LkLZY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/9/3023" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/9/3023.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494050" id="node494050--2" data-pisa="ggg;9/9/3023" data-pisa-master="ggg;g3.119.400549" data-apath="/ggg/9/9/3023.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Transcriptome-Wide Association Supplements Genome-Wide Association in <em>Zea mays</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5068-0372" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Karl A. G. Kremling</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8411-0343" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christine H. Diepenbrock</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael A. Gore</span>, <span class="highwire-citation-author" data-delta="3">Edward S. Buckler</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-5941-9047" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nonoy B. Bandillo</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3023-3033;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400549 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494050--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Karl A. G. Kremling </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Genomic Diversity</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karl%2BA.%2BG.%2BKremling%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kremling%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKarl%2BA.%2BG.%2BKremling%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5068-0372" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Karl A. G. Kremling</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kak268@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">kak268@cornell.edu</a>
<a href="mailto:nonoy.bandillo@ndsu.edu" class="" data-icon-position="" data-hide-link-title="0">nonoy.bandillo@ndsu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Christine H. Diepenbrock </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christine%2BH.%2BDiepenbrock%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Diepenbrock%20CH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristine%2BH.%2BDiepenbrock%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8411-0343" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christine H. Diepenbrock</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michael A. Gore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BA.%2BGore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gore%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BA.%2BGore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael A. Gore</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Edward S. Buckler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Genomic Diversity</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853, and</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>United States Department of Agriculture-Agricultural Research Service, Robert W. Holley Center for Agriculture and Health, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edward%2BS.%2BBuckler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buckler%20ES&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEdward%2BS.%2BBuckler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Nonoy B. Bandillo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Genomic Diversity</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nonoy%2BB.%2BBandillo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bandillo%20NB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANonoy%2BB.%2BBandillo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5941-9047" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nonoy B. Bandillo</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kak268@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">kak268@cornell.edu</a>
<a href="mailto:nonoy.bandillo@ndsu.edu" class="" data-icon-position="" data-hide-link-title="0">nonoy.bandillo@ndsu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/9/3023" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/9/3023&amp;title=Transcriptome-Wide%20Association%20Supplements%20Genome-Wide%20Association%20in%20Zea%20mays" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/9/3023&amp;title=Transcriptome-Wide%20Association%20Supplements%20Genome-Wide%20Association%20in%20Zea%20mays" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/9/3023&amp;title=Transcriptome-Wide%20Association%20Supplements%20Genome-Wide%20Association%20in%20Zea%20mays" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/9/3023&amp;text=Transcriptome-Wide%20Association%20Supplements%20Genome-Wide%20Association%20in%20Zea%20mays" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/9/3023&amp;title=Transcriptome-Wide%20Association%20Supplements%20Genome-Wide%20Association%20in%20Zea%20mays" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/9/3023&amp;t=Transcriptome-Wide%20Association%20Supplements%20Genome-Wide%20Association%20in%20Zea%20mays" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/9/3023&amp;title=Transcriptome-Wide%20Association%20Supplements%20Genome-Wide%20Association%20in%20Zea%20mays" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/9/3023&amp;title=Transcriptome-Wide%20Association%20Supplements%20Genome-Wide%20Association%20in%20Zea%20mays" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494050/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494050" id="node494050--3" data-pisa="ggg;9/9/3023" data-pisa-master="ggg;g3.119.400549" data-apath="/ggg/9/9/3023.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Transcriptome-Wide Association Supplements Genome-Wide Association in <em>Zea mays</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5068-0372" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Karl A. G. Kremling</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8411-0343" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christine H. Diepenbrock</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael A. Gore</span>, <span class="highwire-citation-author" data-delta="3">Edward S. Buckler</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-5941-9047" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nonoy B. Bandillo</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3023-3033;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400549 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494050--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Karl A. G. Kremling </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Genomic Diversity</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karl%2BA.%2BG.%2BKremling%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kremling%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKarl%2BA.%2BG.%2BKremling%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5068-0372" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Karl A. G. Kremling</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kak268@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">kak268@cornell.edu</a>
<a href="mailto:nonoy.bandillo@ndsu.edu" class="" data-icon-position="" data-hide-link-title="0">nonoy.bandillo@ndsu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Christine H. Diepenbrock </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christine%2BH.%2BDiepenbrock%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Diepenbrock%20CH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristine%2BH.%2BDiepenbrock%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8411-0343" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christine H. Diepenbrock</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michael A. Gore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BA.%2BGore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gore%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BA.%2BGore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael A. Gore</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Edward S. Buckler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Genomic Diversity</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, NY 14853, and</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>United States Department of Agriculture-Agricultural Research Service, Robert W. Holley Center for Agriculture and Health, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edward%2BS.%2BBuckler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buckler%20ES&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEdward%2BS.%2BBuckler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Nonoy B. Bandillo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Genomic Diversity</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nonoy%2BB.%2BBandillo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bandillo%20NB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANonoy%2BB.%2BBandillo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5941-9047" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nonoy B. Bandillo</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kak268@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">kak268@cornell.edu</a>
<a href="mailto:nonoy.bandillo@ndsu.edu" class="" data-icon-position="" data-hide-link-title="0">nonoy.bandillo@ndsu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3023&amp;rft.epage=3033&amp;rft.atitle=Transcriptome-Wide%20Association%20Supplements%20Genome-Wide%20Association%20in%20Zea%20mays&amp;rft.volume=9&amp;rft.issue=9&amp;rft.date=2019-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Kremling%2C+Karl+A.+G.&amp;rft.au=Diepenbrock%2C+Christine+H.&amp;rft.au=Gore%2C+Michael+A.&amp;rft.au=Buckler%2C+Edward+S.&amp;rft.au=Bandillo%2C+Nonoy+B."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494050/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494050/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494050/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494050/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494050/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494050/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494050/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494050/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494050/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494050/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494050/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494050/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/9/3023" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/9/3023.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/9/3023.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/9/3023.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
