<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/9/2823" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/9/2823.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/9/2823.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/9/2823.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/9/2823.atom" />
<meta name="HW.pisa" content="ggg;9/9/2823" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="GWAS for Meat and Carcass Traits Using Imputed Sequence Level Genotypes in Pooled F2-Designs in Pigs" />
<meta name="DC.Identifier" content="10.1534/g3.119.400452" />
<meta name="DC.Date" content="2019-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Falker-Gieske et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="In order to gain insight into the genetic architecture of economically important traits in pigs and to derive suitable genetic markers to improve these traits in breeding programs, many studies have been conducted to map quantitative trait loci. Shortcomings of these studies were low mapping resolution, large confidence intervals for quantitative trait loci-positions and large linkage disequilibrium blocks. Here, we overcome these shortcomings by pooling four large F2 designs to produce smaller linkage disequilibrium blocks and by resequencing the founder generation at high coverage and the F1 generation at low coverage for subsequent imputation of the F2 generation to whole genome sequencing marker density. This lead to the discovery of more than 32 million variants, 8 million of which have not been previously reported. The pooling of the four F2 designs enabled us to perform a joint genome-wide association study, which lead to the identification of numerous significantly associated variant clusters on chromosomes 1, 2, 4, 7, 17 and 18 for the growth and carcass traits average daily gain, back fat thickness, meat fat ratio, and carcass length. We could not only confirm previously reported, but also discovered new quantitative trait loci. As a result, several new candidate genes are discussed, among them BMP2 (bone morphogenetic protein 2), which we recently discovered in a related study. Variant effect prediction revealed that 15 high impact variants for the traits back fat thickness, meat fat ratio and carcass length were among the statistically significantly associated variants." />
<meta name="DC.Contributor" content="Clemens Falker-Gieske" />
<meta name="DC.Contributor" content="Iulia Blaj" />
<meta name="DC.Contributor" content="Siegfried Preuß" />
<meta name="DC.Contributor" content="Jörn Bennewitz" />
<meta name="DC.Contributor" content="Georg Thaller" />
<meta name="DC.Contributor" content="Jens Tetens" />
<meta name="article:published_time" content="2019-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="GWAS for Meat and Carcass Traits Using Imputed Sequence Level Genotypes in Pooled F2-Designs in Pigs" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;In order to gain insight into the genetic architecture of economically important traits in pigs and to derive suitable genetic markers to improve these traits in breeding programs, many studies have been conducted to map quantitative trait loci. Shortcomings of these studies were low mapping resolution, large confidence intervals for quantitative trait loci-positions and large linkage disequilibrium blocks. Here, we overcome these shortcomings by pooling four large F2 designs to produce smaller linkage disequilibrium blocks and by resequencing the founder generation at high coverage and the F1 generation at low coverage for subsequent imputation of the F2 generation to whole genome sequencing marker density. This lead to the discovery of more than 32 million variants, 8 million of which have not been previously reported. The pooling of the four F2 designs enabled us to perform a joint genome-wide association study, which lead to the identification of numerous significantly associated variant clusters on chromosomes 1, 2, 4, 7, 17 and 18 for the growth and carcass traits average daily gain, back fat thickness, meat fat ratio, and carcass length. We could not only confirm previously reported, but also discovered new quantitative trait loci. As a result, several new candidate genes are discussed, among them &lt;i&gt;BMP2&lt;/i&gt; (bone morphogenetic protein 2), which we recently discovered in a related study. Variant effect prediction revealed that 15 high impact variants for the traits back fat thickness, meat fat ratio and carcass length were among the statistically significantly associated variants.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/09/01" />
<meta name="citation_mjid" content="ggg;9/9/2823" />
<meta name="citation_id" content="9/9/2823" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/9/2823" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/9/2823.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/9/2823.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/9/2823.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400452" />
<meta name="citation_pmid" content="31296617" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2823" />
<meta name="citation_lastpage" content="2834" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Clemens Falker-Gieske" />
<meta name="citation_author_institution" content="Department of Animal Sciences, Georg-August-University, 37077 Göttingen, Germany" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9160-1909" />
<meta name="citation_author" content="Iulia Blaj" />
<meta name="citation_author_institution" content="Institute of Animal Breeding and Husbandry, Kiel University, 24118 Kiel, Germany" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8744-3415" />
<meta name="citation_author" content="Siegfried Preuß" />
<meta name="citation_author_institution" content="Institute of Animal Husbandry and Breeding, University of Hohenheim, 70599 Stuttgart, Germany, and" />
<meta name="citation_author" content="Jörn Bennewitz" />
<meta name="citation_author_institution" content="Institute of Animal Husbandry and Breeding, University of Hohenheim, 70599 Stuttgart, Germany, and" />
<meta name="citation_author" content="Georg Thaller" />
<meta name="citation_author_institution" content="Institute of Animal Breeding and Husbandry, Kiel University, 24118 Kiel, Germany" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6782-2039" />
<meta name="citation_author" content="Jens Tetens" />
<meta name="citation_author_institution" content="Department of Animal Sciences, Georg-August-University, 37077 Göttingen, Germany" />
<meta name="citation_author_institution" content="Center for Integrated Breeding Research, Georg-August-University, 37077 Göttingen, Germany" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5352-464X" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=L. Andersson;citation_title=Genome-wide association analysis in domestic animals: a powerful approach for genetic dissection of trait loci.;citation_pages=341-349;citation_volume=136;citation_year=2009;citation_doi=10.1007/s10709-008-9312-4" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=H.-J. Ban;citation_author=J. Y. Heo;citation_author=K.-S. Oh;citation_author=K.-J. Park;citation_title=Identification of type 2 diabetes-associated combination of SNPs using support vector machine.;citation_pages=26-26;citation_volume=11;citation_year=2010;citation_pmid=20416077;citation_doi=10.1186/1471-2156-11-26" />
<meta name="citation_reference" content="citation_journal_title=BMC developmental biology [electronic resource];citation_journal_abbrev=BMC Dev Biol;citation_author=H. E. Barker;citation_author=G. K. Smyth;citation_author=J. Wettenhall;citation_author=T. A. Ward;citation_author=M. L. Bath;citation_title=Deaf-1 regulates epithelial cell proliferation and side-branching in the mammary gland.;citation_pages=94-94;citation_volume=8;citation_year=2008;citation_pmid=18826651;citation_doi=10.1186/1471-213X-8-94" />
<meta name="citation_reference" content="citation_journal_title=Oncogene;citation_author=E. B. Berens;citation_author=G. M. Sharif;citation_author=M. O. Schmidt;citation_author=G. Yan;citation_author=A. Wellstein;citation_title=Keratin-associated protein 5-5 controls cytoskeletal function and cancer cell vascular invasion.;citation_pages=593-605;citation_volume=36;citation_year=2017;citation_doi=10.1038/onc.2016.234" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=I. Blaj;citation_author=J. Tetens;citation_author=S. Preuß;citation_author=J. Bennewitz;citation_author=G. Thaller;citation_title=Genome-wide association studies and meta-analysis uncovers new candidate genes for growth and carcass traits in pigs.;citation_pages=e0205576;citation_volume=13;citation_year=2018;citation_doi=10.1371/journal.pone.0205576" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=N. Borchers;citation_author=N. Reinsch;citation_author=E. Kalm;citation_title=Familial cases of coat colour-change in a Piétrain cross.;citation_pages=285-287;citation_volume=117;citation_year=2000;citation_doi=10.1046/j.1439-0388.2000.00255.x" />
<meta name="citation_reference" content="citation_journal_title=J. Dairy Sci.;citation_author=R. F. Brøndum;citation_author=G. Su;citation_author=L. Janss;citation_author=G. Sahana;citation_author=B. Guldbrandtsen;citation_title=Quantitative trait loci markers derived from whole genome sequence data increases the reliability of genomic prediction.;citation_pages=4107-4116;citation_volume=98;citation_year=2015;citation_doi=10.3168/jds.2014-9005" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=B. L. Browning;citation_author=Y. Zhou;citation_author=S. R. Browning;citation_title=A One-Penny Imputed Genome from Next-Generation Reference Panels.;citation_pages=338-348;citation_volume=103;citation_year=2018;citation_pmid=30100085;citation_doi=10.1016/j.ajhg.2018.07.015" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. R. Browning;citation_author=B. L. Browning;citation_title=Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.;citation_pages=1084-1097;citation_volume=81;citation_year=2007;citation_issue=5;citation_pmid=17924348;citation_doi=10.1086/521987" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=C. C. Chang;citation_author=C. C. Chow;citation_author=L. C. Tellier;citation_author=S. Vattikuti;citation_author=S. M. Purcell;citation_title=Second-generation PLINK: rising to the challenge of larger and richer datasets.;citation_pages=7;citation_volume=4;citation_year=2015;citation_pmid=25722852;citation_doi=10.1186/s13742-015-0047-8" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=M. Cohen-Zinder;citation_author=E. Seroussi;citation_author=D. M. Larkin;citation_author=J. J. Loor;citation_author=A. E. Wind;citation_title=Identification of a missense mutation in the bovine ABCG2 gene with a major effect on the QTL on chromosome 6 affecting milk yield and composition in Holstein cattle;citation_pages=936-944;citation_volume=15;citation_year=2005;citation_issue=7;citation_pmid=15998908;citation_doi=10.1101/gr.3806705" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=A. G. Comuzzie;citation_author=S. A. Cole;citation_author=S. L. Laston;citation_author=V. S. Voruganti;citation_author=K. Haack;citation_title=NOVEL GENETIC LOCI IDENTIFIED FOR THE PATHOPHYSIOLOGY OF CHILDHOOD OBESITY IN THE HISPANIC POPULATION;citation_pages=e51954;citation_volume=7;citation_year=2012;citation_pmid=23251661;citation_doi=10.1371/journal.pone.0051954" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=H. Cui;citation_author=S. Yang;citation_author=M. Zheng;citation_author=R. Liu;citation_author=G. Zhao;citation_title=High-salt intake negatively regulates fat deposition in mouse.;citation_pages=2053;citation_volume=7;citation_year=2017;citation_doi=10.1038/s41598-017-01560-3" />
<meta name="citation_reference" content="citation_journal_title=Med. Sci. Monit.;citation_author=Y. Cui;citation_author=W. Chen;citation_author=J. Chi;citation_author=L. Wang;citation_title=Comparison of Transcriptome Between Type 2 Diabetes Mellitus and Impaired Fasting Glucose.;citation_pages=4699-4706;citation_volume=22;citation_year=2016;citation_doi=10.12659/MSM.896772" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=G. Dalpé;citation_author=M. Mathieu;citation_author=A. Comtois;citation_author=E. Zhu;citation_author=S. Wasiak;citation_title=Dystonin-deficient mice exhibit an intrinsic muscle weakness and an instability of skeletal muscle cytoarchitecture.;citation_pages=367-380;citation_volume=210;citation_year=1999;citation_issue=2;citation_pmid=10357897;citation_doi=10.1006/dbio.1999.9263" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=E. R. Davenport;citation_author=D. A. Cusanovich;citation_author=K. Michelini;citation_author=L. B. Barreiro;citation_author=C. Ober;citation_title=Genome-Wide Association Studies of the Human Gut Microbiota.;citation_pages=e0140301;citation_volume=10;citation_year=2015;citation_pmid=26528553;citation_doi=10.1371/journal.pone.0140301" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Y. Fan;citation_author=Y. Xing;citation_author=Z. Zhang;citation_author=H. Ai;citation_author=Z. Ouyang;citation_title=A further look at porcine chromosome 7 reveals VRTN variants associated with vertebral number in Chinese and Western pigs.;citation_pages=e62534;citation_volume=8;citation_year=2013;citation_doi=10.1371/journal.pone.0062534" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Fujii;citation_author=K. Otsu;citation_author=F. Zorzato;citation_author=S. de Leon;citation_author=V. Khanna;citation_title=Identification of a mutation in porcine ryanodine receptor associated with malignant hyperthermia;citation_pages=448-451;citation_volume=253;citation_year=1991;citation_issue=5018;citation_pmid=1862346;citation_doi=10.1126/science.1862346" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=T. Haller;citation_author=T. Tasa;citation_author=A. Metspalu;citation_title=Manhattan Harvester and Cropper: a system for GWAS peak detection.;citation_pages=22;citation_volume=20;citation_year=2019;citation_doi=10.1186/s12859-019-2600-4" />
<meta name="citation_reference" content="citation_journal_title=Evolution GSE;citation_author=B. Hayes;citation_author=M. E. Goddard;citation_title=The distribution of the effects of genes affecting quantitative traits in livestock. Genetics, Selection;citation_pages=209-229;citation_volume=33;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Z. Hu;citation_author=C. A. Park;citation_author=J. M. Reecy;citation_title=Building a livestock genetic and genomic information knowledgebase through integrative developments of Animal QTLdb and CorrDB.;citation_pages=D701-D710;citation_volume=47;citation_year=2019;citation_doi=10.1093/nar/gky1084" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=W. Huang da;citation_author=B. T. Sherman;citation_author=R. A. Lempicki;citation_title=Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources.;citation_pages=44-57;citation_volume=4;citation_year= 2009;citation_issue=1;citation_pmid=19131956;citation_doi=10.1038/nprot.2008.211" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=D. W. Huang;citation_author=B. T. Sherman;citation_author=Q. Tan;citation_author=J. R. Collins;citation_author=W. G. Alvord;citation_title=The DAVID Gene Functional Classification Tool: a novel biological module-centric algorithm to functionally analyze large gene lists.;citation_pages=R183-R183;citation_volume=8;citation_year=2007;citation_issue=9;citation_pmid=17784955;citation_doi=10.1186/gb-2007-8-9-r183" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=S. A. Knott;citation_title=Regression-based quantitative trait loci mapping: robust, efficient and effective;citation_pages=1435-1442;citation_volume=360;citation_year=2005;citation_issue=1459;citation_pmid=16048786;citation_doi=10.1098/rstb.2005.1671" />
<meta name="citation_reference" content="citation_journal_title=Cancer Research;citation_journal_abbrev=Cancer Res.;citation_author=M. Koslowski;citation_author=U. Sahin;citation_author=R. Mitnacht-Kraus;citation_author=G. Seitz;citation_author=C. Huber;citation_title=A Placenta-Specific Gene Ectopically Activated in Many Human Cancers Is Essentially Involved in Malignant Cell Processes;citation_pages=9528-9534;citation_volume=67;citation_year=2007;citation_issue=19;citation_pmid=17909063;citation_doi=10.1158/0008-5472.CAN-07-1350" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=R. Krishnan;citation_author=N. Boddapati;citation_author=S. Mahalingam;citation_title=Interplay between human nucleolar GNL1 and RPS20 is critical to modulate cell proliferation.;citation_pages=11421;citation_volume=8;citation_year=2018;citation_doi=10.1038/s41598-018-29802-y" />
<meta name="citation_reference" content="citation_journal_title=Eur. Rev. Med. Pharmacol. Sci.;citation_author=G. Li;citation_author=X.-J. Wu;citation_author=X.-Q. Kong;citation_author=L. Wang;citation_author=X. Jin;citation_title=Cytochrome c oxidase subunit VIIb as a potential target in familial hypercholesterolemia by bioinformatical analysis.;citation_pages=4139-4145;citation_volume=19;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=J. Thorac. Dis.;citation_author=Y. Liu;citation_author=P. Zhan;citation_author=Z. Zhou;citation_author=Z. Xing;citation_author=S. Zhu;citation_title=The overexpression of KIFC1 was associated with the proliferation and prognosis of non-small cell lung cancer.;citation_pages=2911-2923;citation_volume=8;citation_year=2016;citation_doi=10.21037/jtd.2016.10.67" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=W. McLaren;citation_author=L. Gil;citation_author=S. E. Hunt;citation_author=H. S. Riat;citation_author=G. R. S. Ritchie;citation_title=The Ensembl Variant Effect Predictor.;citation_pages=122;citation_volume=17;citation_year=2016;citation_pmid=27268795;citation_doi=10.1186/s13059-016-0974-4" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Y. Nagamine;citation_author=C. S. Haley;citation_author=A. Sewalem;citation_author=P. M. Visscher;citation_title=Quantitative Trait Loci Variation for Growth and Obesity Between and Within Lines of Pigs (Sus scrofa);citation_pages=629-635;citation_volume=164;citation_year=2003;citation_issue=2;citation_pmid=12807783" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. Nezer;citation_author=L. Moreau;citation_author=B. Brouwers;citation_author=W. Coppieters;citation_author=J. Detilleux;citation_title=An imprinted QTL with major effect on muscle mass and fat deposition maps to the IGF2 locus in pigs.;citation_pages=155-156;citation_volume=21;citation_year=1999;citation_issue=2;citation_pmid=9988262;citation_doi=10.1038/5935" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. C. Ng;citation_author=S. Henikoff;citation_title=SIFT: predicting amino acid changes that affect protein function;citation_pages=3812-3814;citation_volume=31;citation_year=2003;citation_issue=13;citation_pmid=12824425;citation_doi=10.1093/nar/gkg509" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=Truong. Nguyen D.;citation_author=K. Lee;citation_author=H. Choi;citation_author=M.-k. Choi;citation_author=M. Thong Le;citation_title=The complete swine olfactory subgenome: expansion of the olfactory gene repertoire in the pig genome.;citation_pages=584-584;citation_volume=13;citation_year=2012;citation_pmid=23153364;citation_doi=10.1186/1471-2164-13-584" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=U. Ober;citation_author=W. Huang;citation_author=M. Magwire;citation_author=M. Schlather;citation_author=H. Simianer;citation_title=Accounting for genetic architecture improves sequence based genomic prediction for a Drosophila fitness trait.;citation_pages=e0126880;citation_volume=10;citation_year=2015;citation_pmid=25950439;citation_doi=10.1371/journal.pone.0126880" />
<meta name="citation_reference" content="citation_journal_title=Evolution GSE;citation_author=L. R. Porto-Neto;citation_author=W. Barendse;citation_author=J. M. Henshall;citation_author=S. M. McWilliam;citation_author=S. A. Lehnert;citation_title=Genomic correlation: harnessing the benefit of combining two unrelated populations for genomic selection. Genetics, Selection;citation_pages=84;citation_volume=47;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. M. Ramos;citation_author=R. P. M. A. Crooijmans;citation_author=N. A. Affara;citation_author=A. J. Amaral;citation_author=A. L. Archibald;citation_title=Design of a high density SNP genotyping assay in the pig using SNPs identified and characterized by next generation sequencing technology.;citation_pages=e6524-e6524;citation_volume=4;citation_year=2009;citation_issue=8;citation_pmid=19654876;citation_doi=10.1371/journal.pone.0006524" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=G. A. Rohrer;citation_author=D. J. Nonneman;citation_author=R. T. Wiedmann;citation_author=J. F. Schneider;citation_title=A study of vertebra number in pigs confirms the association of vertnin and reveals additional QTL.;citation_pages=129;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12863-015-0286-9" />
<meta name="citation_reference" content="citation_journal_abbrev=Int J Biol Sci;citation_author=M. F. Rothschild;citation_author=Z. Hu;citation_author=Z. Jiang;citation_title=Advances in QTL mapping in pigs.;citation_pages=192-197;citation_volume=3;citation_year=2007;citation_issue=3;citation_pmid=17384738;citation_doi=10.7150/ijbs.3.192" />
<meta name="citation_reference" content="citation_journal_title=Evolution GSE;citation_author=C. Rückert;citation_author=J. Bennewitz;citation_title=Joint QTL analysis of three connected F2-crosses in pigs. Genetics, Selection;citation_pages=40;citation_volume=42;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Genes Chromosomes Cancer;citation_author=I.-M. Schaefer;citation_author=P. Dal Cin;citation_author=L. M. Landry;citation_author=C. D. M. Fletcher;citation_author=G. J. Hanna;citation_title=CIC-NUTM1 fusion: A case which expands the spectrum of NUT-rearranged epithelioid malignancies.;citation_volume=57;citation_year=2018;citation_doi=10.1002/gcc.3" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=M. Schmid;citation_author=R. Wellmann;citation_author=J. Bennewitz;citation_title=Power and precision of QTL mapping in simulated multiple porcine F2 crosses using whole-genome sequence information.;citation_pages=22;citation_volume=19;citation_year=2018;citation_doi=10.1186/s12863-018-0604-0" />
<meta name="citation_reference" content="citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=American journal of medical genetics;citation_journal_abbrev=Am J Med Genet;citation_author=J. Sphranger;citation_author=J. Gehler;citation_author=M. Cantz;citation_title=Mucolipidosis I--a sialidosis.;citation_pages=21-29;citation_volume=1;citation_year=1977;citation_issue=1;citation_pmid=610423;citation_doi=10.1002/ajmg.1320010104" />
<meta name="citation_reference" content="citation_journal_title=Anim. Genet.;citation_author=P. Stratz;citation_author=M. Schmid;citation_author=R. Wellmann;citation_author=S. Preuß;citation_author=I. Blaj;citation_title=Linkage disequilibrium pattern and genome-wide association mapping for meat traits in multiple porcine F2 crosses.;citation_pages=403-412;citation_volume=49;citation_year=2018;citation_doi=10.1111/age.12684" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=M. Stuible;citation_author=N. Dubé;citation_author=M. L. Tremblay;citation_title=PTP1B Regulates Cortactin Tyrosine Phosphorylation by Targeting Tyr446*S.;citation_volume=283;citation_year=2008;citation_pmid=18387954;citation_doi=10.1074/jbc.M710534200" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Oncol.;citation_author=K. Umene;citation_author=M. Yanokura;citation_author=K. Banno;citation_author=H. Irie;citation_author=M. Adachi;citation_title=Aurora kinase A has a significant role as a therapeutic target and clinical biomarker in endometrial cancer.;citation_pages=1498-1506;citation_volume=46;citation_year=2015;citation_doi=10.3892/ijo.2015.2842" />
<meta name="citation_reference" content="citation_journal_title=Evolution GSE;citation_author=I. van den Berg;citation_author=D. Boichard;citation_author=M. S. Lund;citation_title=Sequence variants selected from a multi-breed GWAS can improve the reliability of genomic predictions in dairy cattle. Genetics, Selection;citation_pages=83;citation_volume=48;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Obes.;citation_author=K. S. Vimaleswaran;citation_author=A. Cavadino;citation_author=D. J. Berry;citation_author=J. C. Whittaker;citation_author=C. Power;citation_title=Genetic association analysis of vitamin D pathway with obesity traits.;citation_pages=1399-1406;citation_volume=37;citation_year=2013;citation_doi=10.1038/ijo.2013.6" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Oncol.;citation_author=L. Wang;citation_author=P. Park;citation_author=F. La Marca;citation_author=K. Than;citation_author=S. Rahman;citation_title=Bone formation induced by BMP-2 in human osteosarcoma cells.;citation_pages=1095-1102;citation_volume=43;citation_year=2013;citation_pmid=23900689;citation_doi=10.3892/ijo.2013.2030" />
<meta name="citation_reference" content="citation_journal_title=Evolution GSE;citation_author=R. Wellmann;citation_author=S. Preuß;citation_author=E. Tholen;citation_author=J. Heinkel;citation_author=K. Wimmers;citation_title=Genomic selection using low density marker panels with application to a sire line in pigs. Genetics, Selection;citation_pages=28;citation_volume=45;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. M. Wozney;citation_author=V. Rosen;citation_author=A. J. Celeste;citation_author=L. M. Mitsock;citation_author=M. J. Whitters;citation_title=Novel regulators of bone formation: molecular clones and activities;citation_pages=1528-1534;citation_volume=242;citation_year=1988;citation_issue=4885;citation_pmid=3201241;citation_doi=10.1126/science.3201241" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=J. Yang;citation_author=S. H. Lee;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_title=GCTA: a tool for genome-wide complex trait analysis.;citation_pages=76-82;citation_volume=88;citation_year=2011;citation_issue=1;citation_pmid=21167468;citation_doi=10.1016/j.ajhg.2010.11.011" />
<meta name="citation_reference" content="citation_journal_title=Diabetes;citation_author=L. Yip;citation_author=R. Fuhlbrigge;citation_author=C. Taylor;citation_author=R. J. Creusot;citation_author=T. Nishikawa-Matsumura;citation_title=Inflammation and hyperglycemia mediate Deaf1 splicing in the pancreatic lymph nodes via distinct pathways during type 1 diabetes.;citation_volume=64;citation_year=2015;citation_pmid=25187368;citation_doi=10.2337/db14-0803" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=S. Yoon;citation_author=M. J. Molloy;citation_author=M. P. Wu;citation_author=D. B. Cowan;citation_author=E. Gussoni;citation_title=C6ORF32 is upregulated during muscle cell differentiation and induces the formation of cellular filopodia.;citation_pages=70-81;citation_volume=301;citation_year=2007;citation_issue=1;citation_pmid=17150207;citation_doi=10.1016/j.ydbio.2006.11.002" />
<meta name="citation_reference" content="citation_journal_title=Genes, chromosomes &amp; cancer;citation_journal_abbrev=Genes Chromosomes Cancer;citation_author=A. Zatkova;citation_author=J.-M. Rouillard;citation_author=W. Hartmann;citation_author=B. J. Lamb;citation_author=R. Kuick;citation_title=Amplification and overexpression of the IGF2 regulator PLAG1 in hepatoblastoma.;citation_pages=126-137;citation_volume=39;citation_year=2004;citation_issue=2;citation_pmid=14695992;citation_doi=10.1002/gcc.10307" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Biol. Sci.;citation_author=L. Zhang;citation_author=X. Zhou;citation_author=J. J. Michal;citation_author=B. Ding;citation_author=R. Li;citation_title=Genome Wide Screening of Candidate Genes for Improving Piglet Birth Weight Using High and Low Estimated Breeding Value Populations.;citation_pages=236-244;citation_volume=10;citation_year=2014;citation_doi=10.7150/ijbs.7744" />
<meta name="citation_reference" content="citation_year=2017" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="GWAS for Meat and Carcass Traits Using Imputed Sequence Level Genotypes in Pooled F2-Designs in Pigs" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
<meta name="twitter:description" content="In order to gain insight into the genetic architecture of economically important traits in pigs and to derive suitable genetic markers to improve these traits in breeding programs, many studies have been conducted to map quantitative trait loci. Shortcomings of these studies were low mapping resolution, large confidence intervals for quantitative trait loci-positions and large linkage disequilibrium blocks. Here, we overcome these shortcomings by pooling four large F2 designs to produce smaller linkage disequilibrium blocks and by resequencing the founder generation at high coverage and the F1 generation at low coverage for subsequent imputation of the F2 generation to whole genome sequencing marker density. This lead to the discovery of more than 32 million variants, 8 million of which have not been previously reported. The pooling of the four F2 designs enabled us to perform a joint genome-wide association study, which lead to the identification of numerous significantly associated variant clusters on chromosomes 1, 2, 4, 7, 17 and 18 for the growth and carcass traits average daily gain, back fat thickness, meat fat ratio, and carcass length. We could not only confirm previously reported, but also discovered new quantitative trait loci. As a result, several new candidate genes are discussed, among them BMP2 (bone morphogenetic protein 2), which we recently discovered in a related study. Variant effect prediction revealed that 15 high impact variants for the traits back fat thickness, meat fat ratio and carcass length were among the statistically significantly associated variants." />
<meta name="og-title" property="og:title" content="GWAS for Meat and Carcass Traits Using Imputed Sequence Level Genotypes in Pooled F2-Designs in Pigs" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/9/2823" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="In order to gain insight into the genetic architecture of economically important traits in pigs and to derive suitable genetic markers to improve these traits in breeding programs, many studies have been conducted to map quantitative trait loci. Shortcomings of these studies were low mapping resolution, large confidence intervals for quantitative trait loci-positions and large linkage disequilibrium blocks. Here, we overcome these shortcomings by pooling four large F2 designs to produce smaller linkage disequilibrium blocks and by resequencing the founder generation at high coverage and the F1 generation at low coverage for subsequent imputation of the F2 generation to whole genome sequencing marker density. This lead to the discovery of more than 32 million variants, 8 million of which have not been previously reported. The pooling of the four F2 designs enabled us to perform a joint genome-wide association study, which lead to the identification of numerous significantly associated variant clusters on chromosomes 1, 2, 4, 7, 17 and 18 for the growth and carcass traits average daily gain, back fat thickness, meat fat ratio, and carcass length. We could not only confirm previously reported, but also discovered new quantitative trait loci. As a result, several new candidate genes are discussed, among them BMP2 (bone morphogenetic protein 2), which we recently discovered in a related study. Variant effect prediction revealed that 15 high impact variants for the traits back fat thickness, meat fat ratio and carcass length were among the statistically significantly associated variants." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
    <title>GWAS for Meat and Carcass Traits Using Imputed Sequence Level Genotypes in Pooled F2-Designs in Pigs | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Z3fbeJi-FQUc0CVyRY15MwonIons2XI2WLxw1ia0-CA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494048","apath":"\/ggg\/9\/9\/2823.atom","pisa":"ggg;9\/9\/2823","ac":{"\/ggg\/9\/9\/2823.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/9\/2823.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/9\/2823"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/9\/2823"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/9\/2823":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022gcu4mSBTI2EyUk+s7Ne1C7Rbpn\\\\\\\/W+t0lw5xb150q++HrDNDR6p5raq4kzSJsNGsdXxb5oesRrlwoVb0u2TG4FywSqMv4qO1TaX4AUizgplspLQ+EDAHFdXO\\\\\\\/WBRQ0xbruLDrWfWnynUuPWxY6qaTJAgEbh6sKwjmBwXs8S6PTUtzcbqpewiU5eKx\\\\\\\/sR4B0EgS5L2LZWy\\\\\\\/PiPq3CrFjFZAyNiWoF9DhTyXyDpr8WBJl7N0bPLwaUq42+NZuNRgWJDwcosryqY1Qra76p6bx54y5cn8jxJi\\\\\\\/x6POClppOrELThBXWdKi8W2kS3Dya0xDn2LS7m\\\\\\\/pX6WIrnupGSCnIAraDSHzMVFK+W3WrXxGaG1o7NT6o2YSubQtb+yrrrALXbcw4HRP412bSXTpSTEmEK5TsFOzp\\\\\\\/dCefDttIwaehPogmlkPQVKKce97ymUwKb8OrpYUHRqLwD00r7Opwq3XI\\\\\\\/kfBh5faE0g8+A7NonpJZ82v4KTnjQPWgKGyt2KNtvogRXmr8mNF8kD25F0nbsGL2XWCpdeWxuCKh5\\\\\\\/uPRWtUqnsInFxseMU6Sida44LDgBRN\\\\\\\/+VHBw7iW+7ZTl4TJPoQrc5MjdZZfA2KuUJ9BgFWIedwyJWYaXe7+WsPCvr9hGWaEQbFUJh\\\\\\\/ApCxCmJFWPQI0JgGvndLzcWr\\\\\\\/BsGUflvBkx5\\\\\\\/cIdWiSseIxvLdpwKP4Qaxb2jNPaESutte9zBTeXOPRXSUZUkLCUkuoXz5apVR0pfMeha7GYgLMDg2N+a8CXLwsIZewI4gS9V82Hrhk9M6+41aGI\\\\\\\/IJyeWXcqtYcnkthyVBCqcn8WbZ72X2IisvYwB4Mjjjdu8YZHuei+tlUkhoLf+DsXCrRzcj8u13Fz7ejrfecibvQIXMarBLhMZk6wqtH1WEBbht39t+pELu4g8g+\\\\\\\/53rHtYGrN489Qc0ovg3PytcrElblxufv6BjNoabFblU0kUjoh2KrEFTr2V3rkFXossE3lfX9NmFK76jdKCcN2IISNX6ihVDIP2iUQLYts4KMxUtWHCjh4TEYQqJYOOMSLuD+4\\\\\\\/55t8HdKaS+3MEZzhXIJY3OOa2etgiX7C+QWMKxFfKi15WZTxPUfyAke0jgoSrTbPbhuvEKf9SSNvNqWhfVPLf6ZsoV3FAIf6Dvt1bPdy0\\\\\\\/8Z8p+7K6wkZ+VkQ790MEv+00G3brK27mOPtQCV68PXiHNoZRR1allkR2ZBaItQS3jAvsCXS9lqMatfrGwnx0ImGlUtvVOAe\\\\\\\/Qij5AZv7mpE4qlVf1ieoln\\\u0022,\\\u0022iv\\\u0022:\\\u002276AgOPdLT7uTPntqVQQ62g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d32396b5f2d726193cabe98e27ee414792e03350c01dc5219b5bf1ac2f2a6a11\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022l4px4kGqdMYq48DaV\\\\\\\/6fMl2r\\\\\\\/xV\\\\\\\/s9ekm1so4huhsAjQqjmx\\\\\\\/NEJbUzWDIpcuuIwvH5tJj2pw9dgX6q3Rba42mfw6OLDcCeiB+VwcTH8Ot85ZD1y1fQZOvGqqPB6eJ4zRO3SAu70ljxNmFdVTQWz74aRImWxrdcULK7M0BEYXzuYA1dtKHzu0Hglp3Nd+06FUyXQLpEHx8PSbrHFapJChogMGdRf+59LzpRNuyJ2PbPiBSUhJIhSLUps2TX4YcXPVf21Jkns1NGXsNKpJ1ulvHoOyQrjBySuF8A4p4s8+wXLVXna95QrkZ6pI4pt\\\\\\\/6THwNlq4ixsg+HWN\\\\\\\/jCymUeR+r7vsdmBA1B6NMRpJwyiNHCAkUKmqCtodNH\\\\\\\/\\\\\\\/2jtjyD6e0G1QyLul2i2zVwmP6eewOG4rN\\\\\\\/239Y2wnVaRvjbp7882rqvvJzKH8BBbvEhvEv2Tz7BSElcgWNomSC4VcY\\\\\\\/8jdkFFDKWpoizbxSHw8CXiw3nvW+huqCE0uLISi+WqLBDyXMwugKFdIMiZZYJ+RKD+OvjBMe1+WhJ6skChG3jJoj3bawKn+O0wrFc7rY\\\\\\\/aMiyu3JFxHTkfci7Unl1D\\\\\\\/Ol8eLZjOb1FgEep8or1oYTJIGS5UuyHPTv6RZnFxL0Qnibeicb1FgOJ33gb0BKCK0L9itSf0RXpBGOk4KCnc\\\\\\\/nC1I+E9vdcd8a4pzMFcr0fFTQDYK8Zg6FLI0SddIfLYZzQe8UkVY37NMnVFlyTbwdppDCp6MDHWrkwUsVZbPRxeRVUZvpdjNwRnitJAfTn9ilb+hgUQJQ1yWlsAAlyAm960ErmzPBS4uLbGbA3b6eymdXZOAD+BVvKclb8spWcDm0Nb\\\\\\\/lNXKMkpcVwru10D3yqUNL\\\\\\\/4kqcX8rS7zl\\\\\\\/jTBBLRnLARPCyMNEYdZsSGoErXlpBac3Kap01XjJV+W6KbrSXjftIuQ8FcIawiG2uZXH1AAshz3VA2DNhPRn+bFO4zm8qj8PGvP5aHKvJPp037cWpJ8ULFiNIEuMJ1MMP9dMX8c7AL1YZ1JfVdo39fDCHIZUcwq+l9nZz3igvTI2uHywgUBOKW\\\\\\\/OXv0D\\\\\\\/2\\\\\\\/VkQHOUT1rmidpUh5BEedAOtSUXfKfuyEV55\\\\\\\/pN2A8CaixAOdXTF3Qs81l6V5v9oCt8+dC4NgLr8TxBRxVvpL1Fc1YLoOhlQ+FlpZuurQahZDMgH8SOBsfN98SfG22SEhABFXH5O9bRDRQFl++2OjLkb4ktfoBbCqdf78gPtMPx\\\\\\\/4K6hXe7HCIzAYaIozoUuK8068Jul4THhFUulVrb0IweNwR8YXcjVeDsVd9EkUbHzrGscSMJNf+ld7fZoyIiSIMXZ3zS+uaogr4o2Cw7UzUBgb0RSKzf6yELa0j8DKYidYAGINRPCkxUDQU8qdyecF9l\\\\\\\/p04vQb1E5tv3NQKvL8P7EpH+M7q59Zv3Oa3bWGfWfQC5gVVuvYvQuNe\\\\\\\/qUflMfuEUFWJ4rH5GQO5\\\\\\\/ckEtp7RS0fygNywPYxbnsjTzBZQE1v7VgPzAQOCI6GSpxyGOULp1sJ1GN5v1ZxKUyBdAI4i3lxGT1hS9+gzcCmpl7jJNiiX5Z8ehcek2Hx88SCoA0bLjhtvCO3Eb6jTkyRJr\\\\\\\/Ju9EdgmfozgpxNJUlfx08X84voQQgTJEzzWxKhtETlyia7HxVMbbPlcc8zPYs4SnEWul1GkIXsbXCXfEpT\\\\\\\/N2FN4QaK9NxxWp2xfEhxqfQQK00AB0W3QuuuDAgnFx6IkAFuK3dSEFx6GrmSDoDszbVaUEE9of4\\\\\\\/K9P\\\\\\\/WNFLlQEgrEOFpuvnsp4zjJZ6GEQlchWgvwvEqRXmET4StzBi6uuiCywkATM1Pg9TFjzqvWiiEWbqM0ca5SHQD7MAiz74V0Y9Axt67ETBbWWnUdm4SG1UZMvT+Jddi+EJmD\\\\\\\/UQzrqiWIJ5tqfeagkUSlbJmnZI6Rczqy\\\\\\\/xrfr6dX517hTWQat5Gqy52btA1XWio4foA0KMdLjtIEDYQsT0mbRm39zVCDAvt+jIi\\\\\\\/ZauCJRMLYUTcph+JCSt7oTmEQNWZtHXRFtRVvzmEv4HcJoCC8awEN3xEhxcsFF7Mk0u8kIW41QNHatVHgxUZDjug0+CEN9Lao5Ev\\\\\\\/fEXkDJsTBDa2ao848vUgD1zPCfN2PMR5lViL7rt7RHKh16EVLdRGzu3DIDpml\\\\\\\/DI0ReSVAIME8poE5k5HM19KjZy9T\\\\\\\/XayTzfKZ+Y4c7Ry6+T1jTZYU4ShfB+UbZga7bMg6X3ue0ON8bYeoTtWZzoEwPpiZfwBewx3MKhpnY65YUhmb9jVMNgNEHSsQH4FAdCMsVMf4+bJRJZScgt2IaWd698j5BH8rXxX5ieu5oKaLlQfz9lxSqAatOtOaZFJ3r\\\\\\\/PYpm+YjtFo4Y4JG+qrr5seF0bi6S074DYjW3NzGehfN9hxy1X8T\\\\\\\/izTYu6M8kmB3PlRpkWkqmpAoRYTR1MCPAIKAtUSop2E9P5VYAlrf80z95qOkuNZ8p+hozy7ouGYL+wkhQDlvD\\\\\\\/OpN3zROy3fVur6VYIuaEi9jKGEErpyPblVgaoC2mc62OeUKuUlu2YykoR6TdxNSgVmG8CIYX\\\\\\\/ZnXwEicWq0lVN69cNC7DSvpup2O1Wlia1cYmYnFr1vWXFp4jL2V48959hYhqwcc4EPRfSaW700ujfWGpfTTMsEohVB6KiXOUC1aFqc89sQNJMVa++LnJRR8bYqU5bHFUvFJM2ztuJRzi5BU4GJkpi+DbUWtmOc4RhtlaLgsCKaKbPp\\\\\\\/VW9Wns60LF6klwS8tzqJYLWw1f9rijSRH2j8Tz57Up4wK+JnGKr26M+O+QdEQdnPhnq2P8LC4twDMlF+m85QA55n7WljCzGfgvabgVbIaHSPy4bErVGlNBgrkIkVmvcbgq1ysAzUs+gNj8EmdXYgszYiDh+Yw3QLSZYQhcXbPsCtvya0kCBzAMKozTAidvgjX1MeZB9pZcgVLQIxACPAwf1GdZMtqU3Vk1AgjkFMRHVk1pKGyfZdY0hiuL9yJHtX4IpJ5i49P9a1UCzRKZiAytU7ABrAy2WeabaVi98NFXbRS7W+rk8p4apF9cDH1n6Pxlyo2jRkYkzrs03wOCB+43n0\\\\\\\/TeAooVZwhXInu4eRSXl23a+gjR0WaYI7rkLBsLr1do0pkLM7gK4bziCsEclSgZqnTHRdPzTvtHHkI8V1H+hbm+N9AS5Qu5ElBV3DGhJZWUqJEcRbrkJZwDhFkR+\\\\\\\/8+Ropn1MykT9wfOObrcHZOpOXCSQOYRvaUQY+r47qpFA5p3HYTx8egtc9s1Mn5XPP4znwFr9MJXQIzsIGC7Wv1PZs8lym14B5MdKXtRYgB7dXsjdeD\\\\\\\/lhsVpPlzproqMLs\\\\\\\/1iIgmVzTI6uYChht46F7cXlUUKEzTqX0Ig\\\\\\\/mi64bHb075CFM21hobZqfxrJBRIX+xeoKhU8clcolaNA1OYL\\\\\\\/NN7jQvHeN08cQrQXf0y3JZS4MxPRS9gKUGH+GZ6IKugjDvIyqlOeZ3ZnmK\\\\\\\/CdK9sfWKGHuZxZQ9qsnsYDRaOPcMzuAzv7\\\\\\\/VgD05EzYLPZ8FDUeahB7AdhvOfMxBUnB2jCxihUBJPRLVmTqTvZEfjGNZBXvMhqe0w6F0PiJoeaIRf0d5AcT8hq7X4WFdRUJhM8lnQbTIEvytD+cN88NRNM+dyYfkRWOKuY3XlOZ9vMMXsawBGnEvwOL+btoitakPY5Gk\\\\\\\/BKlawRBOoEPwO5ZKL3soZMXW1S72WiDVA5BnzJfF\\\\\\\/G6WM8\\\\\\\/dButurtp0W4mCjY91CHLX\\\\\\\/6GytMtcBTmxG7fwgehTFuHh+d\\\\\\\/VG29xJz1QGLSok2IBG1zDbAkuZeFsVd4xTqI6Ddn0Z5I8YOQ2Sv1v+a+XKINtDgMEKlf3rtzkJadHaYGrJ1Phf+Ln7mssTILur057Kq3150NRSU64NWEvolllfqLIztS0ZlYidEzvufzVcySRwSd4mAPxO4xaw67QSmTRZWc3Fez0Dc\\\\\\\/jJzsGJv8zdLFlcZds6ZzU7I826k58Wyt4Tai3qFJ9Jx7vajOKwloGwr+iMmMnPSlvEVemQEgE7GaniJd0dA2D5iLS2wsn38TQSkLpUv1B91ynryKDHR4Jp06rNIqa78Sfu3rtmH7p0rNetYyB5GbEGU64ObThzGNKRzIhAx\\\\\\\/izRH8sQod+NcBCBtFOjopX1sAfAXSEH0ozDlFqnEjDRcfe1TR+BlX9HsMMvpg5Vj3ad7pMkrIhnjJRE9b5yBGUE0Tv9cxhzhoaqYl+y6lEEhgBwzQEtuJb1s6Di1zezXnEtIJuAnv7UE0IQiQTqb5uU80RPXbPy6kD001WlXFDUwbjmFSK3gpobl5Bz4eCCN2CNakffbnZfOSbNrxfL2SKqnZ120OFT45yF2TDLpyLEGbMU6KQWexfIzUROCegU0OB2efur4QYxlFqQ7vQAqos9syt5lIdVCtTyXVL1dZV4dH6tUeCB+4dpYO\\\\\\\/GKfgSSsq\\\\\\\/qCNDJ1908JdsyqaWdtkRXOFFvGRwvPjRVYKbJ4ICPqach9ui3CtfDbDUm+EfyDzRB\\\\\\\/9ft7sGu0bneUaFd41J883GMoXC4E2eR9KPBgA7LX7i0VsNZtm1zJYbK\\\\\\\/c8YbfwWRj0fcRkmWefsGFfCF+sKl0ka26lAa4+VZSnXCwKffVza23HDFUG5Wmuug\\\\\\\/0j9AeFC0AKhEehhcRJMVl56QgvwiXaD5akzUCCqbNpAePHL8KSA9kRCiyhc5Rq\\\\\\\/IzikjsP3CUicZVO2NsXjGVP4yzc6tUO3qhC6KlPWA+oTz8tQp\\\\\\\/XYxw4NNs5zirxP6Uk2DS0UM5OL+fJO5DCZW1yahDKC\\\\\\\/TbLGJA4c6w3vsad5wEQQGXQQW6euxVb40n2mr6QEB14gNXfklPydP1qGrr8FEl4HfWV2SjWtJuBlW7CNZpnXCU9uq1UuIbeFANCchqnk8Gh3QrjHFoEn+hDrkv2sv5uBO6LVDO1xnaiLSzxOe4nCiWAvSwFh69qr\\\\\\\/wt8QYwizJYshIqx7fIeBg2vWal6lGRFTAojKilOpOxhrVAFBhm1goSiESZfoWFSJVqcUvH0sJEDEhVDVFJByekjTeqD+qmyugVHadd0kT6H752v3Q3wXDZx4OcJXIpmfbKzj8azoBhktUlaGRuva5O3iEOI5KZiI2G77dxcMhiB0p1V8uITabobBKsT6bsY1otvlBbsIB7P9j2ICZPt4VL7+Y6owSu+KmnlolJ0ttVkvpwMUOb0FH0aaWgYGojALpU2A3Q0gQgY6Pjqp4d6aXERKLxf5sN8JXK6vv13Vv\\\\\\\/41S96IxeDqj3NfTxUzdMidAa3ZTsOP1c\\\\\\\/fxkCTc8H8Jppjq8n33NCA1BJ+68w56LzilLq3WB6lG9c79s2WfaDntb+sILj+S1SvQCnJicOPEvP3kW29EJJAukncCpJeoBSth+6SUeUl5o35FqPQjpOPHekO1jQ9DgEt8ze\\\\\\\/e4iax\\\\\\\/blGpyddxX47qiPQxpp7YyjLDKS9FEuLGusHVKeY0c08xCXnqAwxZS103wlFqhlgzsOZ7itDijRoW0W2Z1mLV8c+PFhNLl0jrivsFtnSFAIOyYPy8dcKWYlMebTMTlxtJXWAPj+XtmFwqsHlxc\\\\\\\/aGYDf0XonyyadhSGaMwbQosAYF3A3IR\\\\\\\/5+SzBygIqHd4lImuUiTrps4eAvYMqv1dLaE1RGa+r2jBv1tY2giW+xFj0LyKvrnZFFuBY4Suu1zBLQ2GlC6AWBWWpWGJ8N59g4\\\\\\\/6URHpcWmh6Jp5yBO7vBkN5z7W4fd4kTiwJSEQNdFiom3vmUonhSU+RAl0a6gGuk9OBK\\\\\\\/OPz3xFsA9pEbwb+4lI77F3ROEowAPIJgoUrwgpyq4vID+0Me0dczHfAAdC4tV++gF2\\\\\\\/sDDyeyDpKgHZ2LtSh\\\\\\\/uADdBXD0wwor8PA+D\\\\\\\/Q10p7vDdNGxQbzorAZJWwVo1sWIoSylJEGt38iDppn4wy38DMP2qP5adB96gxgqrwpQ1tOxXt94KlKaMCVzYljSlRjlQas2By0T21IV0UPh1pl4DWEZt43tSLkNyo6WwNj51vIdxsHKddAzl66sKYi5PqG3Qd65jqi5OUHcc95+UecW1E+5vYDbL3Pe\\\\\\\/nQ+2rk2jNHnvcNEhNOgfWcAmj3EtqASQW0IOYHg8KrNPorqVd7Bw656wBw4kMsgSxTdN1X0g7QEpBI4ReS2SBW6iQsuCQtXZl+Z8gtklBaq6F2ssOvMNvLQKANTDSCvt0T5crBHtwnDRn4FJ0fAriBGcgj7ZK\\\\\\\/6cBvSeWa\\\\\\\/Eq5UZ2oBtV2G\\\\\\\/qPV5QNOyEw42dQTOVd3zHBmSHLhlSMAhQr1U\\\\\\\/xi2ngVyikWbDsUaVsO8aL2yR9MkL40KnKMd8yPUruNu9bSt37px+XquB6M5nlLAR2FHDVUjlWE2HfL5lI6uZMAr2U2VQ21lhTtbQu4iL97oPzf+cCKfYrv33sNttwDnxZy5AxkSkyIWZkako3DN2Da2oFS8Sl5XAnyo1dDxG+pUu+amj1AQolZKpi3QUfbC8ggMRHMefEKslYY9c6wMMfjF8i49Z6UYi9XMsCm1PVbF6\\\\\\\/Imx1sIpuX1v\\\\\\\/uOO2K2MefO68HvXL8CFW+tcmdevfMZR4LpM5Ap7ZoA0oo9j05V\\\\\\\/\\\\\\\/bSN8fEhapZqB+EAXy3RI3CCahO7dMCuD5yUV1FZM2Tk6INd8P3KagsqFs1r\\\\\\\/1FdT15\\\\\\\/YlWmE750O5GSzdRyv5Ays3Tm+RAWQi1Q1wUBqJvYk7B4sjRIffXz9Oa0L1fi4XEmbg9UsS3BTwjThzMdqpmSLzS3IyhyxoLkhsOk\\\\\\\/yHj8StFTgZ\\\\\\\/oHJYrtQd4T9rMlArX8dSg2kgJxjeltIdvKLWv\\\\\\\/YS7vAQPpOLv1xciftosNpubIVPApcZ+TKo2ikVZUIefMc55Rs4aLBg+Q45C5JLK+22acW5xXO9q9VwJ3w1h5SJ6omLvsRYTd4i0lrZgECEGxevm5iDsQANNsGxqfUZ1zy5qxvuf5tSbIdSF+y4lxg1lssnQOuvjX5EesmpKOGtLsPceCuFUK0y8ZcBGluiX4FIe79ynrFRlNS\\\\\\\/MvZDhQSoRIuITxEj2Km7ruob6EXy39O7MiF4wScioKP+S+rbPgYT6V\\\\\\\/TYH5+NX4Zn0iha4lJoWB6vzR2nw+k\\\\\\\/\\\\\\\/dHQG9oqv54Ri3hoQyItYc73ut3PAWOzIKCZtKssJqPLQa0oNcfJnf4ejKjT5B+txJeM3BmTeSt07MkfgavCIs+Mo6qKhP7IOmAdXj\\\\\\\/IiDoVOrT1G7XbUluQaDDO123NWAUiCrTMZiko1CSpFoAI5jXRcGme5VqeWZbIn+4zsHOzWWCv+wNsxyyoIABFc6q5G\\\\\\\/7HZmvrjRZQsZ5j9F9hrqcxT8nomlIRy6aYLDtIqZpLNP8hke42vhrveNVEeFWpCUrWlkwge5EZPrfQ4kkalxiJv\\\\\\\/nS61F6akPKSThVjIu1Kc2c2+f3+HDbVsjOiaYQuN1eS5qVwrnYjHqCwunrA9Ktm1HOZXZtb4MRXIMttuBwpxkTibCMvktWyCV71jCX8KyugzSOAxx51dk51O64rY\\\\\\\/9Qyn2gMlJVzZXokPIZytIqQ60L4NvnP2V2NCDS8hlPtY2sGatoWBuhhJYEtLX\\\\\\\/OT+ZKuNJJi09MxpJbdJX6GHFCDduqZRkRi9Iw7zVKdpLbPIUxOzPWrfvmLCpj0zSMTlzLWA5t758R\\\\\\\/RFrtcAHTbK5gTLwhCq\\\\\\\/D0PNPwMnS7J7Xdth8vL0D+wgz5k1Uw4DdCfwR\\\\\\\/M1rBes9OrF1\\\\\\\/WZM0F0wrrm4fl8cTMG6NHzCg5QwQflIpmaE6BkXyhBqMsJ\\\\\\\/dJxQaq9NgqtzvF4L3oLsDoqAEX9mYUi7vomi1XfCQmmhg6yq\\\\\\\/NVEHc8owODAwRfOWbKX\\\\\\\/6+xJ+T7gxh5AQfU587C6Z2f0lckqaPEZQ44cAfkt\\\\\\\/rcTz+phjwx22k5FpRljRhvrv10s17b9cAs7IMusCEZqLgRjhPeCu+Q90nRzZRcobtw8GgXZzSfg2TWGL9vVfciBlyTs4EI31rS86p3T7gD6+606FIwrNckVU1646wT\\\\\\\/r3sxa0EhobCMG2ZxknJ0lW\\\\\\\/jhnbaEfJ8Wt0359oV0VBosasQnZiLToXM7\\\\\\\/zpCmxAPh65eBd309CpbNOdJOm8jGl+XH5Gva0EqKBvSo7hhTpuoLz6ITx0b4GFBossAM5P+jw+DL\\\\\\\/Z+q8G9NIF\\\\\\\/ZH+iMtyAthpxSv\\\\\\\/XgA5Q15uQg8ORUUMUJel3BGsFWdOsLwMSvw1XdjG6WVg3a+ZcnHlwsnDcSq+D\\\\\\\/mcCCyiu7sfjqJvx\\\\\\\/AlzzORIoFQ+r3pa\\\\\\\/u6a5Fnrrsd8\\\\\\\/3pot2AnWFMV3Rk\\\\\\\/iNI+rCXuIpgjqUuFF\\\\\\\/DqhjM8MU88JSMxdoADWrHx5OhEPw9lfLWfF6WxuZSYsoOxVcmLgfcH\\\\\\\/MagdX888YzdhISgDoQpRmCSNuoFv\\\\\\\/n06ei5GZ2fJZ\\\\\\\/biw9Dd6Nw\\\\\\\/sLez9sRpPGEvgtPRVH7sehksTpQcmNfohjJWrpl6ThO6zvIylwbsBQvB3uQs5r8jvAm1IY5Jzc0GW\\\\\\\/H6ydnE7SsEtWfqnfamjhH7j3pKQNATA71gKOljdIy5CXhx\\\\\\\/qbxACppRYSXwmeYuUVUH4uWsHocBQsJVVPPYpj9HGBFKOC2WuQ7nnfPCw1LZwtXiFD\\\\\\\/i54Vjs\\\\\\\/NyeU7bxk1Z+CSe\\\\\\\/\\\\\\\/fiy0UIYgLP3bT8DonK26iR+oOD2GEZThXGCZR8AOiXLRhCWGyL0ztXmvzyjUSjbNyd43TRU5yjyGaw0QlZCAYj27CyGEjqe1UhPAKfMhh6yNzC8kEDIolz155QdSkvjaaiYqL3H3YAYU03j0tWMsT\\\\\\\/M2LCvRkrLGfL1M6AUqqJ9zUYr40SB1DlcXBvZ\\\\\\\/Cao2mOYnwfLappS8MUb4ya9ioOVSNq8dV77OHYMRj3Y57fiZeKsJWBLtKrX4+cTM0HSo2FXGbltvVdHLFgPFDXzooR4M\\\\\\\/VsaO2hVF0Ss3zGqPkx5i7ZPe\\\\\\\/4IgBLbXf+ReYCaIWqM6pLaqHe7AoC0xnqUkVJa2qllo0rd\\\\\\\/DxWIZtD9X0WR\\\\\\\/ZwR+Jr+yH98Rl5tdvQzV+PPuBNPt8hTwt8C5XEUSbLXNZn6vIQ\\\\\\\/hElUFjm9hld0mGDQ1b0cUlCYn8C604GdLVcElrdOIsPrHUI8JKCJv4aTVzs0As8lVNAa4bYKVBFI5sYk3LNodL6KNMXJzBNHyA3\\\\\\\/3ziEU1d1s9KkDOBzGuEDY7cOdHUut3iTl5JiAHRJUHO6ev6FPeuAnIoyF38UcBwX0P2pZjyTuSHZFIf19n6KhTOBucy2J8pJ7J6cmteSAxOT0udt7E3C+yQ9qaPuFj6rSWYdR5kjuupfQTHS65nFgdAY7E3RB\\\\\\\/\\\\\\\/yLJBDflFfZHQ7qSrjmQGgDABeyf0yJMDVEndKsAs68uZvgx9teWyLlCTeDLhOCaxHRGG6etJXNGuGgLw\\\\\\\/WihnM2dZaAnlNHWEolfkR6p3hXBRG+bQYxl7ejFiTTOFJjtBCCQvfzH8CLEPMnOU7KXUjiWbc7rGnA2GRg922a8auBUp3hIjaYx1uz+PWYDUaG0Y11qhKh2dYIsG6PN+QdaFUtOBKuYLue6coIaVHbA7NDiH353Yfc9muL5ww3T2At25Ztd+0HGYiKOyhn65E6lZQecJe8eQdOgB+YNL0Cve5X1qPC5QLLehYlHy8RC2E\\\\\\\/IyezP+NFhUuY8IrQqbUwY17zk9R\\\\\\\/cHHGIc4ZOa1vZA0aPS+vlFoiR3qVRN5cmUNFsFVBOjXH2R5zBiYm9vqYjfOEJXSxZowNWLR10Cz6XIzY2pKtDulft9Epb7SiO3dDOio8WbvOoLJoSJRjpA1c3XGSWxxIolBpN0YAbFvA5vyXZrGUsExo8Hv+rtCKFZbGYRs0gVupFeWGYv2PDkdjrwPIuq1d2P+VBtWGEi\\\\\\\/p8thbjGxnjd3GEfYz276gD3GTN2pzKa+DEou61\\\\\\\/Lm6Uru2hZC9w4qdvha+lr\\\\\\\/smlAa0LbYhtPlQpJHYZIUgdrscjSZq+u79nLNBvb3OJlqLsDl6TQrrtI6WO2zJEFVjOW+5i\\\\\\\/z+Xtpgrg3f5Ytaa2r12uMU8\\\\\\\/6qYfqMuaid+Nb8ErbdcxdYdYXsI1B08\\\\\\\/MJGsL5hQUaLJVo4gBdcUwJYBPXlc1nEHtLbFZH+uF9VuDoe3LUCM6QNUd99KWH3i\\\\\\\/h+DYcQaeNnU3tdTrmc84TSKuPcoD7fwfEWlDVL3NHXiXLY6ZSW1mGPQ5o2+5hRRaBkFl\\\\\\\/hDQuIpcrSiKcnvEm6X7jfETFG5N8PIqPb\\\\\\\/X6cQzci6IcDgQpyOoZ2p6enUxK\\\\\\\/5v+fQhe+lZvb7Gp8oCylt60n9FMXnX3LdCIIZC+I\\\\\\\/NvLbK1VtY7BQ\\\\\\\/4Q+NXmpF3D2s4IsAosSo6hdESZ93ATTiT5zrQezorortKfjIEVAKYyKSTVWOCfcPjpkQO4yRWJx\\\\\\\/7hoEU+QbMAwlh1onyPc6O9H2Ga27e4rlv8i+XPJF+KcYTCXi9ouMLloB29S3CUEGJPE5zAFfJxxMYMVUhpG2EOv5YDgB2MreZSim048HnnLkJDfCdrxh6j7eX2PHvOVSaAEWbkmoePkkkpNiAFFGpGdW1kFMUTMa1pPRVLmwXYcCTq0iNcgccwx6\\\\\\\/eOVtV6WZYFTPjcKjLvR+ThruXUof56NIGDP5o\\\\\\\/0gCHwHRnCUi7dOKesJ2idS85BC34HPi7C\\\\\\\/AIfocUE44bJz2Q8u7KHsI3EXNOQQNFW3AWQ17mgQ\\\\\\\/53vpACkxDrFsm6CCzIlNzhhJQ0PZu379OZBrxZAt28vBVIk6GIhHBDcpxhgJ5JTGmYF+1JafixA41cohDsQLh7RXCGsDl+E2oSSGgJhD4UE8W32MavYNjB0TccwVwLuOzQ6PxuZJ8jCLZum02qiPQtRfNhsnFxdDeG8G5Y18m\\\\\\\/DGZGKJ7Ff7nLieOyaeIJz+4tiD1U6qAbkHJAOXmWHqZl\\\\\\\/un9VLTO8IBxSrTOEu9gAJdAbBT1jGqHpSkQtNbjosMBeESGSdxX4mCDZG2EjNDRiOnpejkyyEexnpSm+h\\\\\\\/ZnnKbS8dFVoM7eSa6xDACCXAlwepNoU0+DmbqvKLIcJwkoyXg1MDUeqKDs1ppK23zc574VWnrwSH\\\\\\\/elBYpCJh0QxdK+3usuChcwwebs40LhbF0pAk1N3obr4Wvsktwqp8ma4uZKUraZ0phaZH56NYcNKCyDc3qDHpX0aCwJsd\\\\\\\/C5ehlMAcaIK1xJlv86XJbgO91WDY0HW2fXTKmswvRkkCB\\\\\\\/nzyc00e7+5SL5IAYyjqBxdEuqzfYAKtANHdQ8L0ibVQiqtz1kGbcODTw0Jegp8GDHge5Hp3DmGWe9ENLJXJkwRyM0KjJVhVchWdEK99kzVqeyxqYt\\\\\\\/FT96BrSDj7ttCwjK5gg6jv9p+EGW5DJqx0ZvBSoVUWI5uqK3h8my3k\\\\\\\/xXzGBhku+Ks4ciB5waohH7lpmC4tSOwRtofgFwl35m8e2WZyy7mKASjRj\\\\\\\/0z3JH3xaF0\\\\\\\/VFM9Upq97+vTB2PNaMG4q09fhZpHjfpf4NIJHm4cxXLz8tFQWy4ObrxXjABk1SKa+bjvatca5y\\\\\\\/b2Lz02QNl5woSP+eZKw4xXx6eB3rgpmIZQ4s06MgzGUDiF2MAf1UO5GzIorzOC1mk1qpbEGp0g8oFcococ1ok3fdhIdNs5JYjIPSufOABT8eVdd2DMaxAMhRw0hw+b3lGi8z3y9LJAT0HN1NvbooYIVajFsBTQbYrJiRUiCJKLoVJTazOiu\\\\\\\/utXnqZuxiY4scXxR4LhX\\\\\\\/iD91aWBpfwmeyFSmu69N2cxO2xGx8Ym9lqf1s0I8gOdTRN85jMwAFdek8uXQ+i54wBq2xHez5RA1epjqlBA572Du9ghH48M4ZrVuDO+fJ6Zt6KWVIXP\\\\\\\/91J76M0yjUxvPO1ezHSw+aOD9khUs\\\\\\\/HtPSruYSiMKWYf6EQij36xlJR7UGBi3GLIEY3olTLBejmBMEv\\\\\\\/mrJ+zOboupckdrBI4DkofRjnYExtmVtd3kDXYwkbv+i5HatC4GojorGcrkz2\\\\\\\/z47mrAd4irHHHAyms42wmIJxVGdx6xNh71n3NMd3UJaoRGdAtG0K8VYWyvplx\\\\\\\/+Ud1qDncqt0VKe3mgezoiwtm2lKRc2dopwxcEsNkhB3Rily9Yv+l7MK6e48WycYIkSA2+IsxnsvGqZMaO6O8h4CWdUkgGc7+76aunlK1FfUMHHT+8EpadCHJ8r8yW3sZZeYlmB5cKRqFDF3wtwyW87rvUZX5d0L72a4ft9qkPM5\\\\\\\/G5KK6EKpeoIdbHjANyfhfoXd1kUq304zs9yF47qS6Z4eYl2AfZ1HKu9aARkF6kT3SPEOfDM\\\\\\\/3T6ONYl3V3QgPx6+8RLiugaiHtNbQ27mjQjEO9V9O7put49t2GOkvOte9TKrwyxJq+74XdJWxWDo4ImZ6Sj9WPtAIR4kCHKxmlxgwxa8NiPmATuDx2YH1nYIqfbCXX\\\\\\\/2dJ3H4CVrxRuU4CRx2vcq+VpOLtEvDKTfYwv4VVNZ8JQvb1yqisR6azK3TL5\\\\\\\/g2ENvsrDdRaz9M1raVFSrT7kRlN4G5v0kTKh5PZYCmcEwiWIOz+kUaVRfWq67xuzZCef9wr59KvAeHHM30CAldgJbRY7PqdK870jJ5vURmqWsD\\\\\\\/gY6QQ4AX+jeyJ57kVKbXwKzwChVxqeyqyXkV1IE6vmUrmEaGRLX+jE77HQrOFdEOOHw2\\\\\\\/0HSZz8x3lrXyVDkiCFuwl8dPOsxNn3Oua9Exnzz1wAIQY6J8xwot82w3Lyxfg\\\\\\\/P8yMTIWAu6F00UVDNhUqDw+FVcgyeSkFOdJ+Jxw3xkAscmEPDJVY6KpvQZ+S5z1NH7sHN\\\\\\\/QmPABvWgUfNyhY4gX4uyH8IYNIvhxJ3o+j8xhTqeLBV7dofTZ891GhTEF8EC0MrGbKWEqLSwqVAfSAUNB1+9vmD1c5ZTnmDNmZrtWvE7+u75nySNJD6rKzB2hoRLVEVu4o2px6MPIj3R+7cZJKG+UaV8s+oIlkz97IGNEpLV8lRljwlC8IrTDOXFCwR5qZRT59cE6ffr6AzxgMAdzsMMqTpeUvzY0R0uYxUWFBAI\\\\\\\/a77CbyG0Yxe51ebiajO84QbXCsMlD20KcUpWRgXWMtW0gNX9L0xEkIM7LhgqUt\\\\\\\/mv2SBXezjFgqBtHS8kdMYZH760vC8srMl+AjG0IDpluMCkRCB47WdpN\\\\\\\/D7gcsKX32BGMwARgxeNhhDB9kaZe3CumDQYv0tEyKjxe6NJLQdNM\\\\\\\/0YnqAk7vSRWMOwJM3ELtUZQ35MeDFUccQF9HDOIhD129f3iWf7QiioEOINB1p37ubjVz8vC7vVrp1ezOyv2w+80sRfbsnskAGOw9O9TlrC5RggygQG6f6OHjo2ezAfr8v0B5GhNFG2BVhlWCX4hsgqmiVW+kZFP2GOfJoyUOqjvMw4xuz3kIo\\\\\\\/S9J50ugL7al1cDPrFVxII\\\\\\\/Q87\\\\\\\/rcCsOPkkRS8XqltDLaYJvHxkdQt6mQsmlMcDKiEdzj1I2VotRmMbXXlL9pY17sUCVLLXfKWLPeCt8rceN+ocMTQ85+5k4gYVv4bWZcNRf2EXOH1osxNSKg9gwlOOE4eRCDtODUWCOo6GO8snWziFFcXhbBhlH1Yul9pSVW+GfxpNkHNhWLg0QZby+alYKacKIC3EROJEKVxS75LjcWnI5kZ4fWkvWOJ9HjSHoaIaWK+old+ngB9ZY9PtDYku8K8zaPj4LseX3OwmO1hIv1mLOZcI\\\\\\\/f7ND6dYbtEVsV4EC\\\\\\\/WN0g3D5E\\\\\\\/6nLrzp+nFmc4kNBm4DoJY7GTsCiKDvdwt6C\\\\\\\/phzzZ1A3SPfPFlB\\\\\\\/LLhsH2xQ21THtpPnX10kYyxLNF4wPGEr1Fu9BsQ5a1p9LOaJ67kJ64n8kQByKGaVKKRaaPFtA3JB+uTdHzcPfFk\\\\\\\/VYv1\\\\\\\/lzgXi671W\\\\\\\/l7UsYUTj\\\\\\\/Iimd4vVgIhS+qyrF+ggCwOGEH8TTMVq8FsjjeMjjALSxjyeLyuywGV8UrOYZsqr\\\\\\\/gGooHwCHaSfRc2TPuXqqvWNXL86zq1x58BeE4QoJntcyk7mS8xLiV6KSBvGB9QM2y4ZbOvI55hYO0uklZlI8jo4u1DSRvGn6opgSz9dvaT7pqRDbzyRR6oBDCdu7GzRbwlLZeVm4Fs9tP1+SnELpPZlODr5h6Plwx\\\\\\\/khWqk6XPjvRncr0\\\\\\\/jPS3gPo1e8dhJWH56nZg6nqoXH\\\\\\\/cBALaEqyXkAZCL+Jc2xukHQfrrxQsS4Dl8CwHRKCHVcVUKutzWBMtT5sCs0xPiP9oIiLZd+7otw5kpSyzspR6ekmgwCiy+VK4VwQmb2SA56l3z611XBKWFQWecyqdlRZKdfRy3bfVq59LCQ2Fyn7o3r29mW5fqChjVLWgqCrtP7ULzGMS4vigP7BDqUJoD6tpxUgoQsqAT0RP+QhSTnGRsgx8uODLbUEbgF4gTV+tMssLHIvrddLxIWSYJg1VG87N7+w5sHWj6pSnPuoFuwnF1s7iznOcvnx1rG8s3pOVN6vQ2tjzQ9o+qKL6Nphp4c3eV5XCwcDjQ409tYm8kmrNvn9G0ag+f0lZHeNBz\\\\\\\/e5hh1Tmcema2t6Wci8xDhVdCzeGe2tpX2gvGyf6xCCIxT+q6bL+CAuAYTPQdrfAudH7WmVpjvSbHFlqw7WP347n1pyVdQ49J\\\\\\\/FdQMapYFH5GwCoNQCKXebvxJgQNX52ChHoFKUFU3BryLo3Y0tFCV6scOXW+R8b4tCFu1b5hHMyPzKJzR\\\\\\\/ZkikoqfGGcyIMjfXF0PVjXN+HHOFvbjGw1sJcWBaMmsgsfnLeuSuezoxZ+OGbK1yBf9NXYH2dVQvUTZ43Vy6skPPZ7K2eJlvrxXTcK0dQKnoNVCHE9kQrnZYZDJnatrThjtOlcg7EW68x2btaHI0Zw69t3Eh2JV4E4kMuxFoeEWxgOQfFSm7Jith6FALfBSnSR\\\\\\\/Ycf3KuZpn+zlEH3y76W0ldhUF+xzr0Elwr6NXgbp4nHp0+nXhOoC0WJiZgxyWm2r6V4LIGyIFeQ12K3iTjGpRLWp695xasUq7yj2kLj+QYpfD1lFua2ZbU\\\\\\\/15PrRuGcvH+9MRBjuvd9cY8KGpsQxZePSZHK+EhzoRz+2N83itGpeKfLjwE8inNtt7azXBGl2hKq7chMOAGyEkGIQ61f\\\\\\\/WHHSQ9c99\\\\\\\/qU\\\\\\\/CJs77UI3yIyJ\\\\\\\/nEKy4dqOmnWhIn8VcynVBnN3xI3PjFkGTXZohZP5Keo4QvKpHCTXrqsYBpZBKrrw\\\\\\\/hKHX2oDwPiyNMoU9hWQu9dFWNKO0PLTuBqaqDsMVihvxlueUr4upbaBNULExpfyrmMA3vfy3RCHoHD8Tfoem1f0GQTzFrqF0k2vr9BOqTd8UjRRck6l8UYLhuclpfgP\\\\\\\/5sy+MHXvTzTiBQRWTkIVWaOxUcQPrb8b8mIgyE9Ns64bsGlffAigTZisiVttKmCSbDsdf6oHy+DLyxQEo5BlP3wapeDHK00vlZaxWe737CX7GSPSTaXUi1m7wUaUpsk8fPcSYMm+vyEZtd5f3P4\\\\\\\/uXb57GZvXkgyRT18qyzcmH\\\\\\\/H7A10rWb7tcQUmtBd7XqqMSl24rMFpPE6tJO1dQM5VwpQDCN\\\\\\\/0sc2h8R3TiNxT+oP8PbGX9fu6av0ZPVDwup6qCrIvAf9bVSdnUOB5iO7n6r4BVDWpzS85W6\\\\\\\/FuypmN52RDdAoyEJTPwRWS\\\\\\\/DATss20\\\\\\\/RZKEA1lwji\\\\\\\/CQAcxTRd594jmcKNrT3TP0k7tXZRNgZX+LVw0ittXWt7vgyCkehkOhk7TN+MaWavZ6UxyFPdn7VbL7zZy6RfjFOVXiRoali332b7sbqDQEvc4l6U3\\\\\\\/tftke5Eu+2nVuDTcPeEz08LiS7f5ukGp+NPu\\\\\\\/aOIf140jtYRedvLpMSL0jHpiSK5EGAo21tRVJH+B4K\\\\\\\/ED2QXbX4lfQxMXV4ufLrEoRlv1eKUp7ni+M1Oa51w5EGAQ37Vl61fz3IA32kkwjUPRtPdvsdx6URhZ0NVXMclVOknLBfN1NaSWPSY7tS7XlpzMEDFfYnh5wOlIEY+kTVzk2QqKKG57ewQFLYk9FEeKai4XU+Aj8Mu8o8seW00uUG4BwZO5xUlXjc4Viubs3OUhZgmiw2chmUF\\\\\\\/m7aKBpabOzpNH3ixDKjTQ6Fqbv\\\\\\\/twppCz0EA7agDE+vnKPQMA9uWH9CI0CzhEtC+B\\\\\\\/rKxBZyqioCaNmR9+MvznEfFqTk0EIuEYXL6jewPDJnXpVAnHprVKZt1akggiqLJhd0mGYH77mH+hWL\\\\\\\/QPA+wY9NUoxJ4hkqXjnAnwPO1VT674GdrNJa+hP+1kglZlFbeZQCFajXieEc0I7cjLj5ZwcL61G0ivTbO8tDVbngqyjGjbjfkCioLAt1UtIAR7DyWH92xqY8Y\\\\\\\/aVM3yYp6QM\\\\\\\/Nf+DdgR4N9YrOgj4noNUQl4eDXNchXyJ8ffInC0LvvqqjlnLEhShDXEP1+R4QmARshSzr2RM+Eox2e6\\\\\\\/cUx5C+IFHgUfavSwIkp7O1DSryki9w83bDp3LOVAUITH3EhF6kORZAq+pyM\\\\\\\/5fvC5vMU+f9iNEzthkWIO3VbXYvc6x5qn8OTtt+6d7Hlo6wRgZodP\\\\\\\/Hz8xUChnxPbnXDxavFMMHkNKGGvO92chfxDE5xsHoAflneX+uXk4B0Say6+CTUsKrtwl7S+XBO2Xlr4SJsqW66xW3DTJbY3IqGq2rytoAPfgsZvJkdDGtnCLEV1uwDZCJbvvcDcLWs2SwtP1\\\\\\\/09glFsMQcV3NREqOahE5YlXY2MhKTGUecDlZgJNlTEXfk+iQW\\\\\\\/D5Z0\\\\\\\/veFPcCu7zh76ejGs7EUok66bgYrrOhFE6EAmYSicfWyxIX7k8qdA8LTpgqvWfoJKxZcEZBt4ZZKOfZtGfk1mmaCQIqht0Mkjk6upV8ORcqM+cBy3L0\\\\\\\/KFR0issZJuDCm491uMfsWemicep5Qz9SPijLx98Bx2oz2D0UtUAWTS1Gn\\\\\\\/XsilIl4ZEFqKOKyEiNHaJSCNW4kbw1qjFKIrBSa+TctqixrDnDXt1LGiYagcF32Oeh0j9w+LarV0o4MfzlrfaxcYvUkRDXsh6Yb+cezme1\\\\\\\/4yh\\\\\\\/5A6JbNDClvHJPEk+ZrpmHr4aOLR2CRFeL4NEQcd9rLMffr1tgRIfVPcaQlrImA4WUlrchMKmm568z3VmXmj7XfE+LFdpkOvtK2zvjOtN4iGd4YXh0tiswfri3SP6jRHdtWcE2WDrI6V9eULy+z4UhhldG8UuxSnCvIw6RHzes2\\\\\\\/2TT8q5rh1bbmsSjAZV9+bD2Z1SMbP\\\\\\\/9rk2x7FrIkt8pfhgquf9Qxqs3N56V4YEOU0X94cqq+nh0cvQ5m+cIXdKu3b2gVxkzbF9KYp0wIkyaqQUPvkJpUzkeDDTA\\\\\\\/+jatFATwPeXeTDuJTUbBZ7R+hciNohScdiGlz1uEHFfecnGQmqa8\\\\\\\/GYsP8WU81ddErZfIFnbwCht4sFyPcj+MhyNbomWdo3RA0Y6VhIE1v2itsf8OxSaytqPYq8GwsbiaoIPrYKii5SPtJ160yNCY2pz\\\\\\\/nwBUuepRYquM7TsGS89P6t4EtmKo4Pzej0pUlkt5mEapShCn5BmoxLGJtH33EvmKCH6SdDftfowe4+F40zR5aYwchxunfn9De+huQ8hJWgbf2akm4pBQpnspkM\\\\\\\/yXQXJnivm6L+XDeC86aC3sm5KUmEGp7N0ZYjuFMW0JkhpB\\\\\\\/toSLI7w4ZN8XdNLHt3TEP8HfAMae7HwyQMOZ0J7BunpfHuO9RWDzmzwpKeE+qWlWFGCGuM7wdekD\\\\\\\/u5cjTUW9GAXge+q+PuZt5gnNf6vvrrVe8ZPZqn4BvbnB6bWxzYZeW\\\\\\\/t5rSqGspJFw6kV+u\\\\\\\/MDLBDUp+3ewKVzI+gNS16N7qX51XTHp0VdG+tZ67Bg0d8xGhwXd43XMKz5o8jQ67A3UiXqOFqpcgWVH20m4YwdkfYkDu7k0TundfcZFN9D1XsakQtIlY55AHOM\\\\\\\/j8ptEkDgIgTsRbeYoAfJKiLRZdWTKonX5ZS\\\\\\\/StKm0AMsDVJ\\\\\\\/ei8cGd1Lh4TxxqxXv3NCFimQZiA7CNyCgIo\\\\\\\/JLRE\\\\\\\/i4TbkJTpGDNbbjAT+CkI6NSvQLWd4TcO8org3o6bbUsEZjcRriQmzWl1QR0ARTVDlJZp6v3nPMiCizc92euzXQWOvf8S5nynO1fiv+ROQt+ugLqCI2DGEtv\\\\\\\/NploxCS0GcV+0852zJDbt7fOryWdxOVY9vKx\\\\\\\/\\\\\\\/ZiaCEfDr3ydURpx0oMQVfyQZ6ARVkvvgGKhySOTlflLXpVjR0a+Jd2rS9148DiIEw6ingcEkPHbxO28wqi38A0WXWRSz78KBowsIZuTa6e0+TDT5biroG2iWAdz5EMo\\\\\\\/ORvsu3MVVCWT9rse9MUrEJ6\\\\\\\/bhIWFbWNR5ecFp\\\\\\\/t6Y5CnuHzSD8pT58s1joYl4+rPqs4ypdbEaV97iZXyHjzxDZt3br9GghtNnvakBfy47UvAzXj2JHWWrie4CRPjbTHBv0ZKDP40PZ\\\\\\\/TsX3bIiSQXxpgTz6k5ugLMHE0wu568VradvaNx98O46NsB4ZTdyM1\\\\\\\/VyxSwxG0QW6j\\\\\\\/k5NO3Tmz5PmlMvNq+0+rUlHcRojeWCSMUF96Y2cIAKj0pnYewRgCP5FWO78CoNw35yJHS985oOCWuKfALtDhjeEMb++Q8XslkAp3oU8coyyqTl3uQ6oC6m6s8DcAbkrY5KMnZu6D1locTsxA9J2wz5vu79q95gLokDx7dvcxgwTwjjKkZJD1jtx1nUpyhkkdUMsSnO8ryoKUpujL+8hsKWahO6r+twqzhVfEVpWkDaVrs0XmOzbA+Jn5966GSaAT1RMREe\\\\\\\/z8YqT2Y8\\\\\\\/XyOqxi9EdpklZ6xTlSEstYo8rbWlTazZel9aRZuzYi+VaElUPVVwV3RzwRC5iDyF2gQ1k8OzA0Vfztc+D0odhLlOqBafyLAVujp6IOPf3KDhr0p+FG26PhqoXsNY7\\\\\\\/tUkLxmhZ6dgKb74z9SzSdMOYAtv9NKFMWlMgtjV2oB73HCtIc3298\\\\\\\/XaoomzY7sqPL8cMoIWMuL5tgt469Coi8d8IBTf60RECM4dtnF5F8OlvdlOf45r+Godzfk0Zs5FbS\\\\\\\/fENVnPhLJC\\\\\\\/7Ss4kfw5snH01qssBjdvwz\\\\\\\/X28z7PBlRhwrtd95MKO\\\\\\\/co2UoX5hN3RTlKN1zvz8B\\\\\\\/ejFnjxkubwLpYToj2SW7uX6aFs3w9TgdMEbG0C2phtS95qGRTX\\\\\\\/FLBkHxpV0pTnYmxtJvrHlQZY9Hl6t9iPKt9bR2YLe2Sc0AhlUPe8bZZBLungVUXBISQbMT4XJIL\\\\\\\/E5aHBuCPJb40ygIG9OWbGVQ8nGnia4Z4jOYPu4N9V1SF5lAUwfdocBJzua3cjSLWDeFwKkuWBEr48OSXo7jST9+0XeHB4KAu3xoZM9ir3ZMQ\\\\\\\/aGm5mTS6EbFPY8KbByXLEIghQYW8FPT71XieU5nJaCuanksJiDH1\\\\\\\/E3phjwfO3AF8KdopFfYhE8akW9aaW5uszMKxziBiUq5NMzc8IH7wDZAHZ3++nFSTMwhwTsZxqY8JMFwLkw6P0xZWCtnpVFcSlqjbatD5KcKmjikGMDyzqgGYzwOs\\\\\\\/Dk1CbT7AOOGnYTqBZcqyl1U\\\\\\\/AmnynlxR5yw6SINyQea6\\\\\\\/fkr\\\\\\\/gfHu03l7aft1Zy4TYeQ+qWjqWlzv5rnHHXVUsFBTUxkvoyy1AJ8J+hvdXMdmHWuEnNhLFuqEFAxOb7OKVoTGd4eni\\\\\\\/UwIAI49t086C2hAOtBrcH1NiYBHTFQs311YGmrRoHgleB0a1+xGBtVqrre3\\\\\\\/RPveA\\\\\\\/1yc5XeuH0UTgc4iIY5TExwxA1BcL4GTsv6k7JpcvRAsP5+m2qYADIFRimdaMfqt56Uzpvx1Xpm4fvq5bJGg5XX8I5u8zvhf3IQiWN1DUkBtGFS0bY1X7NDE6w0Cxw7yl01GuR1DVejCurSh4yxcoSrLIalsWHsA0GiRzKs7kZCkZPaVXHAjK9SwGrpB23lQPvXgSUWdfS64He0FMzNHyXMgVFMhXMJffNBD4BprY\\\\\\\/nYwXBktO72c44Eo+MMeHjC1H1sNDQDEogH4AM1ygdvN1pdkKnSeBD6SYsJ6Ob3oJ6b0k7fGAdXqsL6QS7PCU+DqwV63Sa3n5PszMhflLrjpDFDsx67cXkD0DCjSHvUp3NZW9j1IcyZhN\\\\\\\/q1jjbGbGuLDM21oVc2fwsmKBCNsPMqaienMgmvnwsiFzlI9HjzwOfQoYx1btQ+4UzPAr3IDvMJwXFJ7RaaX8cDyzwXk0XFnHJvG2\\\\\\\/eus\\\\\\\/bREz+TN8XLQMiibDL4SMOfVhPLBBFaOCQWF9x5SUf+DnLChP\\\\\\\/aocnpXb\\\\\\\/90vGO3ofo8LoNkqy36rzL1UtNdWmBuoghcJrw+AVE8BzA8wWvzncW6GPbQTvgQ0hEP0o9toGvX9duafZIPYzsyJcQveHsckx59b8TVFCDxowWR5Ofbnnm4iwyAPAcWuzxyMZ7MziHtgwnty+Aj+Ln7VzRn2xEp2\\\\\\\/Tx4N51\\\\\\\/ySQYmgk\\\\\\\/1qsXD0Bx2QvAo5pASBeAjmsesaPR3xDZOWUFuczQUcyFKIC6xqXWaFKKpbJpoLA0rih6SYz29YdWM7DBQ3i7cChNKAQOoowABbrGxEgBdJ5G8hs8zGXgkSgF4jwAckWfYWSb5ZtkHWiiEmjXsX\\\\\\\/SMQHKahyn3anUicK6XXZovNgt4Fpt1aWWBHn65DMs2t4VP5zx2rq6UJPvUlj7H1YXHKPkdY64qY01MyDRkXo+lI2BDI4k2eaIV0LNzVN9oN6KN3zWQJ88marSPomfgLAUeJV1mixtzbTbSBjRlbfMjSyj53EIxUPBLms+BS2pe8f1g+\\\\\\\/8guURwNODwa6ED9LuzJVbtXGqIVrQCQOg+067RvqpLw425XqbLEnEV8GQhG9cAy6s2ohHhL6TmIHsUeiqCkC6iomuSuiJbZ7aWSX3vhsPmtw2o1IAcX9SoYoPZRNzbYhHfJXhmWewfo+80TjaQpAQH3VwDzbD6v9qLp8cTR6QAk2fFnQsL2pYlwrBNXa2+gNSdlRjmY\\\\\\\/WFZh8b2I0QEKtLfI9DZi12h0zwH5nMY7v\\\\\\\/zWqh+K7UtnOzhCTadlmRvHOiURpz5qaeL2VvwH+lvO3z8It74FEAhX65AXHlkalS03HL57jIWLYsjcxIw5h5Yudqej06KkAWKli9ZjDBEEpeOUsyRJghvQWCZTmmLg+MENH9wUVJsCEDHRoKLs36\\\\\\\/YjdY0V\\\\\\\/M2ydotSMVnoOm8z57K+Jkl+XcVD+8UKc2+Puiaizg2eLWXiG7Z+8wTSzy1WyW+F0GotV8552TVCd2OtPYQgWNrY8urOPBQJ8IQU1k7Np\\\\\\\/NmxAz+e2Swr9QjcXfxr6rFpcow0I1SbcfE8VN8C\\\\\\\/FOj+29XzCxdJ0v8iQZXXPOFcB\\\\\\\/MHla7Jw22PuW2nZbb0SR0q70UDtizmdZdyKsNlSSsW6pbMVzutFd3U1uloQ00buHVAm2kmLEfsP\\\\\\\/uNjkbzYIP7WTmk4uS4AeyLYhsEaH3DOn87Bua48bluV\\\\\\\/cC2esSpO9K7dXubvajxoi9yQm124I8Hp7MUzndhkXKs+2BzJXyhbyq8Vod4OLRKACxuP+RldWXV\\\\\\\/vbP2k\\\\\\\/+7ChHBGpmk\\\\\\\/V3gkLUw1CpacRhWKrGsYkhvai\\\\\\\/mbUpuRa7QaE2G9PNth06HU4SMNm1eBCR8pT4cmMOVHSusiKr4XESvt5G9Gzln9HPb8bALf9GRJEsVNYuf6JTExMGwpqznUBa9+DINOD4bprORU9qjcXvBIhd8ecZXzf3aUxzaHJ8f\\\\\\\/ok6dv39JVxndUOtPFL9iIDU9RVaGK9FpVFzyyawI2uroQdGXI76kgofSJ1RG6p4XQ1cqOOTrROzadjuB4mI90TJOxsvoJOLawLdR2lEhYLFSoktOYy7CPlb2iOdT\\\\\\\/5Wa4IDWNH8kzgZe3uI1G8ViwDvX\\\\\\\/amuiyuI8pl218Vi0gDOrA\\\\\\\/+fYkmeJfoCqyu7hOqdylvHQrovB4CtdWY5L\\\\\\\/1kUL3KcH8Udia3bFEjjUVSZqUmbMhIAAWoAEGF3sIDklOGO4ZUVkaMLFKBKUAeeSA3ZqRce+aZKNLxc\\\\\\\/SvJEWOciplvOhRYYAYg5q7MMItJHGrFBAovf4zsVSZ03YDkGmvJyqgQm22aVxtoC4Uzdc1En7ixVhgA5PrFjmmSyloP4z6WCb6tyaAfyHHWIqj\\\\\\\/l5fAI6OLExySl2dMMRh6hQPigLOXjScjZm5+yuX3Wxufaa4BTfSj6Mw3QlMVbAHVJ3X4\\\\\\\/knXgSnl+mL82LlD21W7BUVNRXgGDYktCQMpmpiZNc6TGGdelG5vy1aAU43jwJvoS+z3uv5pMMEEjfuBNPa1D7JO2NFczaM\\\\\\\/x+1rBsLHLG8gGep8+ypVjcvzAAjh7n6jIfHG3mgoxGGxIhCm08smfmTXlrxoaj8JWjD7OuhWWWkCllEZ8NWLK6s6LtyFO+7vIneLVlN1PAWt5Im\\\\\\\/K8cghM1yg+SjoJGVwOK+B3LCIaknXgXZRwE+czIjO+hNBe0CaOXgxtojmrTTtQH3VqTqyetc5N+WJb+PjKfs7UDzCigDFu90bMcmdXqsnxCdaIYK7EmQUvgG5RvzxRfJi2ULJEuelPbz88005UDKuw31wK9i9jyBYZQ6ZthdFJriZFwn2SQ2va4IwY7wclTDwrH6SmbMuVN+D70brhHeYtgHVvRaNPavW00900gSVCB8A4EQuJWfi1B3tl0OIr5ICbhPONj7oqeOv4lLucmUh\\\\\\\/ocVxq3xFRNDvQzUgcXwtEvsMP+zk\\\\\\\/tk5TmIMS6BTbSJjNsX\\\\\\\/zEXt\\\\\\\/3Lzm9JKzFiYt+zdMmkgOR87ftUQ\\\\\\\/SJLKLnTuNOcaLJKwYITjIicYZmeVtC3PAUmo569j5AAeNKbC94l8SX6SC1UYD\\\\\\\/9rpaqDnuSQ4w2+Fyz9czQ+eS8YYYN4PvkNQjGC5DElKORlBW\\\\\\\/AjnpdVkHm1QZUvUb5kH+K5Tw74NnVuy89oyDW36vIamVALMUwpp\\\\\\\/bP+uSqT3tvVF3k+o5EsGeOcUs9quO7Uu9hZ5Gy0z0SG2+5bMmTnw0RK3zl3iA\\\\\\\/JQcApS17mJ3FqOJ9UOy79wxwAsetIJ7IyuTpcwyG9lwVh9VuXUwF6dO5WhWSfKEn9IezH7+KYlhrv3k9JWtgW\\\\\\\/LV+O3pu7bZIQJZB+siwNRo9PEAlpn2Ek7MdgNLcNx9SqSwBoCxM8JWRYuOEDlane4K\\\\\\\/pWSrCb+d5pLw23Ir5RdkVKd3PM3U7\\\\\\\/87xI1aYOJq0sSeCIY99KEwrhyZaxSarNmlqdBJThwrqI6zNlManHmkaS8++lHLnO+IxBlfhXRossBc9szDnIfDLKMh6IiodJmpyRIvZI4nGneDcn30oo8NjqByZBd9g9\\\\\\\/FZK44ehgqsBezjGtjCzTrknLBPtU4PC0eEVlQ7alm7HTpJ2k69+YV8ln+JyoHB6f+Y6ZyTGzbvlJ3jNub3KYZ9ae+xHp8mvVXfyqQvTrCUfClyQUOrLUu1DedkHaPJBrOvd9Kyw\\\\\\\/jFujFGbbiYOzgp9bcw1RVDLo1MUl\\\\\\\/3\\\\\\\/h29aGMIgXlqQKb7\\\\\\\/IabVsAHckyTmhp2ayw7o8TC7Yk436l1AUff4PyFiwBSfo3L6vh0cnmjo6cUtxWROZtIhpHyyJhhRfbSYvcyePrbwFuBFZKRWA\\\\\\\/EdckvJ+Ji4pBmMqjwB5wz2ZMm4g1Ydr4XdcSKKAb6kpOsIbw4tU0AA7yl7W8VXjvCE8JmEHYKJpZM4VHxEiEeunnwulC5HwiC7E4wqHfBcIuUEkxbc9w44cMc+u+NtzQCNk99GvpkUtISlvKmgh0JV\\\\\\\/Dcql7Ts4KS+OxdU8rQcas\\\\\\\/zzp2URWeurUpnLwIa3v2Gi819hkTQMH6eWy70IkNRgwjAx8AhUPxIZf6KIL9P5RHRoSfvg1Jb4vguNw7CfZcybmQiv\\\\\\\/Ad4+J8c7cMbC2qFcY1XBOEmD69hdhKynryYlPILh5q2fNCy3ew2oe39p0bHNT252XZ\\\\\\\/AFLTfFjOJotTzLHDaDvuNsAGMcstFdIqzj\\\\\\\/Fum8WIwp1BimqKDKkJCaOAnd1tDMiJIOwo\\\\\\\/Eo9bP8+prfgQgm325sZPkeF5z5OK3acEgY4kBZrsgiZE4\\\\\\\/5vh\\\\\\\/CeJhw2VlHDYoOz9woQKOnsnluDXGUX7s8EfVFfhsApCV9OO5phJOAeN4xkGqMyqONb18zBhJgsHNogG1R5wKXK7nHOjb5qC56lqfFYivNom1xZBc9\\\\\\\/rdr2XE1n3lWCjSgowJXGVoOOpYSsy3P8qwmI9NrMheD+H2QZvODOkpb4SmjxrEr4Qr6xsWrhi7DEM4rWRvnwxfjPnH3wLn3kut6fAkCsYb5ag7v6keSPlGQa9CJyfn\\\\\\\/QpYqIs2Fc0mb+xB22d\\\\\\\/phlvjEdTRrrarIgXC9jQRcJJUxZxOAUw3AwVUhu5vylzaGCWGfAfttNb69V1joVs4vd\\\\\\\/mh7kbMPzx2zm38Ftiofqrnk39+MG2Qy7bZK61mAs\\\\\\\/p9ufZVZoEBESwbTlhPugBLzng0vOqWTBWo7FGWmh5iOVbxvEb6HFRP+PJt3TAutlJDWlQ0pd7ghl4Hrr3coAOgx7ka8LEx4bFA+2riO7hmQl4Mj+N900v588O1lPnufwvwHviWTy2ENk0m1OWUeQKKzTgujxGxWGsY8\\\\\\\/d7jJLN4WxGSiv\\\\\\\/SU3QLh8gh3Lfx\\\\\\\/KUyRbp4sDfpQL4F\\\\\\\/patOJ9VyE5UmUlbMAXorod0fGWmojoL1QYaUJyf4GARkWWc2f91msa7NnPO\\\\\\\/G1AiXl989d4oR\\\\\\\/dFZsTlyWRmGLLKVs+yFBjOt4bT3RB8T4CkkVuOf0RieRZOY2T5+JXAQ+Fl8wW8GAq6p7x7aywrLH7f29rWCZTreOOFcbtFFaMwLoe6IeXS7OnFAPe7MePy+W1L7oeC5SSGBd7yUoQCI5BhpvMpBMnY4twP+x3TfmdQLTuwOB0q1QC2A6LgveQlvgl6Um9s1dOHAyJ3R889E2TqVS7ssBT5EPRNVHaBvhdu06+BPm7ULLNQ5EL4WmV9U3DcfKNIQh2MxW7CEyx46PgB2Kbnp6fbLshEhGletlR2LO2ESUlKX52mu8jwHSZMuwEuTsAZO1b6RDN7mdjR+M6oD\\\\\\\/GgRg1zGXOJW0L6W+xvqwN1qA0ff\\\\\\\/WDTAR7NVmLM+ZH\\\\\\\/jKlV16EE9tKE8iQyLm4gpvL4e5KI3hT3kaN92vZeO5lfpAEo23BuOPE7w0wIXISdmXvNQZk1RESegmhgoUz86EWyjFlQoqRWvrQZxHJsorzGP6Tx4O+fkZkWoSkBKgj3KMqgAB3jWmSxKt8VXb5Ocae3b+wEzM9aGR\\\\\\\/rTH5WGSRrG2sih7fQ\\\\\\\/S0iSU\\\\\\\/89dx\\\\\\\/7KAG\\\\\\\/HIdcWZ3IHS7aKIN3\\\\\\\/Jq9D6nGTsec\\\\\\\/GeTZL8h+kvKUo47tCQB\\\\\\\/2N45KOGUCfJQ0f204PPckbuzEOLKPJ6qo1ZbX3+L6jC8Vsnpla0DXTrrVE0vLkolQT97CrBISiuzJCrHW8B2FIvQSk4Fc9zl\\\\\\\/iCubweEewX2MyEr8vVXy5P8nkftn+\\\\\\\/oWn1qcVYWbZ3SvvYmfQs+KjoBJmDLqnKiZRd\\\\\\\/7Wduni1PcCjZ8svb9EZY3hEHa3ZxceKGXp8hqic0479RDLZETdSZx6Q7Kk6KLsX+Ouf4e\\\\\\\/gpE3LzuBVlZ2I+9RsNC4Ftj2AAIwXrQrgx3qpipfLyFt42hmQ\\\\\\\/nBotPfYV3sbcHdh29HXCpoEYg\\\\\\\/BlxPZAbLeX+ITnOoMfh51Clvmx6DB1PBm2B43VdGU+IOHBku\\\\\\\/JeSCDsOq2KLLJ5aAHt9O7lbKorgo0+0\\\\\\\/E2vGWMQmwVVJNOvA\\\\\\\/G872VF4jxLpaZuBVIurws1\\\\\\\/FOnn2URYFTP4sueJG+c+V1TxkarVT5VEgGUAIA4XAgMNYCbairwTTlvIJQhq6KA48VFOUqPdKWX6igINhCrxXGPr\\\\\\\/GmF\\\\\\\/9dRDbyHlJIkSl896RbFJDJv8TfdLpIH01WeEb9Rd1DxQfgzfbjujsYol4bpdgPGMs6IrtCPPek+flyi7wtq71jGdRe1nC8drRH3Ooog9CXk53utQrYMRWeTMPWhM0Fj\\\\\\\/5\\\\\\\/mc6Ti3B3wUt9yfo5MQfmD0rtes7ktsIfidRxdUlmx+aF4fJNer6QPqBK2Hi0ixNBkg8jQ5ZW2yrzUngMhpXFnOOEie3tbTJkiv+0pH+3r2d77zbzdVLwVIpVgMoLQn72x4\\\\\\\/gqZJWAlfBtcRQv8XcEfaf9uxdeH14ygJO62MR92LRqJkkD+d3L2CFYhf\\\\\\\/iu3fB6FjcFf+aJ3MQSQ8iKDQNabPlH5MLaqx0tk5tXypJDL6Ob+w\\\\\\\/C37D+W92YpZRHhSAsMoMkI\\\\\\\/In\\\\\\\/SCIaq2BTI135+eOapXbpsw2QkrjtAv4YU0xKzVopadFOezucrgw7Kp+OcBwLyxVCTM1xoMqc2jhJOxdz5C3\\\\\\\/MROdVAXtIpH+uu+uzbz6lUVsAujFtWsYlnLMg9qzEM8qhdZ4ZOzDPFFvvYgM8m0Wpcvu+feVZJc3zo2x78a2fcLaCqp8T1M0GPAtaWO5C4WgU\\\\\\\/0\\\\\\\/kECWu6MQ\\\\\\\/QkpVCv+JGIX1AVYA7VG2q1lvV6KjoEkVe2U6OWqjLz8PYoEgMwNzcqKwPpyAam+ZZwTsIw9o81xE7ZT9vvXqHpJ2q7VwqHxTXmmIoLCn+gWAYWQW9kX3b+OFzOwhHwqIHpXJQIE2UhYhka4Y2kOdy2S\\\\\\\/9wXS3IgkK4cVwhyzdihx8zgn6H2q2U6QdZ4CxwNvDkZzuehLdfRRw38zVd5dLBvOJCoA0UUuhhqzgJ82kwsAMCvhHHp0+NscuPWnsvNtuR4s2d8adm8CPxqjuvv8AcAtLaO8UKTTaUZnymLB+Lp9rPyMVkUjvurltzAnV4mgasXycNdxb0L4Vofwi8Zya3inUW2VfaHpf9oh4TtoFtOGlsqadyf2SIkGpPK\\\\\\\/PfznbVAl4Jq8ZXlJyWyaDq11qN+xAiTobxEfCpaXx3uNiaZRiT\\\\\\\/YE5neUPGUchkc00pPTRK5dv0ir9e8LP4\\\\\\\/5sNkjBsi2boYgpN2Hu4A48FYJ6CP15IIqzhBQr1VOIee4PpnMkRxO1sp14WzGwIrc++U5a9nH375+LhrYtBCBthK6xu1iSobAKShSqNd\\\\\\\/IznQxfCJjUrVn\\\\\\\/a0+c9\\\\\\\/PV1Gqzz0gfq+dz6OQkCE4ZPiH+pO4Sbbn6su8WVO9t73UrFgW3JsFtLuaMDv529GwJO26laIArpZkRg54yh9l\\\\\\\/JE0BJPcTHIRJzIdr9BEzWzgAFc4e92wl1PBSlPiibk0KpBzlq2TTdd\\\\\\\/m+Jlu1AHTK+QidB6y71CX9gs+kUGtRDQrwbZ7xcKLBMbSESQ5ly1u3bnnDgQmTFS5xlJR+R0xHWsV1RmstXalN70kN92xSWEXQpykhtXFNOp4vtG9UVE82Imtp\\\\\\\/nnaOJ7hMWHPz2pSps3WFhYpNPQap7H5jUa46ajOpR8yzlEfy8CbCu5zhuiDDOSj9Gvo0m9XqXCiDj51dG5HRBKVwig6u0UxogP3uiHbmwqcO+UMOqmQybnXjkxFBfA92WngmAxgSj6uEEiqmb5uFgR+T4NRasZ\\\\\\\/pzF0EtqgmYQl3jj0NZU6fRSvTvYwz3H\\\\\\\/9jZ9Td0g2YiKlvMnST2T4BKK2XpHEAfVmdEj4rqcBWbotU2mwgd3w2XP+WyUq\\\\\\\/raEfPmUaHln2UiLNKsD952HfsXHOkHiNnwkmeA52zBWeX77+9TyOZmL8Vs8myBs+93vHH2k3oIS5jirRJkjwKEaK75osJXmLW3MxekhV0YhV0uCXNsWhKhlgBet4j97Y2u5RpHMyjtH\\\\\\\/itEPxqSKqGcBkTy3ftZJMcxPidbhZ9Uon6EK39ZJVs\\\\\\\/wYMzrt\\\\\\\/PW59ukzaPcqnrgxbcMPAFNfwbO0Sl4cbHL7l1FKtmWhcIk9p3fDTL9E+rfdEvXiTNzAI79uBSV9uGPrKEJVBSiamn2SLFdH56CG5JFx25g1K4KR+5PnEfHkCfXKWCnOiiU1cwiYAXJUSXkPuyRL5rQUjmA+ZPbt0Uao\\\\\\\/WkmOInnFFzDR21ekqc24\\\\\\\/2D+MNvXeVL6zE0dGTmVqVFVejAh5cph9nQrZNoFEpMb97LrXUILwYFPw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022PPJpxDXSBcdyuqi93CRzrw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022bd7f2285bf9dc47256de55109d0c8846910f76cacbb9a8a33568c3f6776032c9\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022BpXDP0LIWdUMa1b93+l4sgDNOYxpfeTt4Ak70ylWXBDnE5itK+WRlDXR7EQItvzDsEko9BHVPqDhOg4LlS3ihPQEkwjWIoBwv+ZWwoTXktcBxPXlKx0qd2JJNIjIzSg8zyJTaUdC8EtxPpo+cj24JwzI4ITXW9EMR0CeFlB7ZAtErryA4Ag0lWuNKIS7fKXpfLt1r4t4dGrp0nn16lvwlDHF7KX6Id8iCvXqoN7DsxfkXub20dNcxkMvrF+5LqHt2C0HKF4JqMGHUKpgx62MGHL483lWDpXOKh7yQ1UgQacp9ln6bTdfVVowTs9kalf7EWy2ySKuRDr+mQ4UKQHqP5oWQWGhAnk4Tia6\\\\\\\/bnlPKPckvSVmak2We9EaPf5fzJHZFs+qOG0W\\\\\\\/1Arw2tzr0D0RQ9Tc5EtmpfDOwKkdkEk\\\\\\\/fIdWUM0bAzyCqV3RtWOeWG9p3e+IxM73Ggaa6ienqEcgvCiQdl87obeg9gq8FQpPlKspjR2G0meAEAC8iIs9JQbdCqmIDwmAwZtthknoT5emp6A2ctNaI5qoVa0xEyFUClYAtWgWbc1KlBgv5lb8xImHlc\\\\\\\/4Kmfpw18wS0ctXACSitsDys8W3eSMXVHDqXrwEIEjb7UZYoN8l8K\\\\\\\/t2xft+sIHQH9VCvwqb+xaQb3wa6j6mRk3mb6laxhA1VoxcsFgel37VxY9Qs5anbDdy0EGMA+FponEXrjQynLdF+uAjZjcZFm8viqceRPV2zwuf3Jb315o2ohqED45Exnpv4+8VXcynXSaLjSpVOWIW1Xu8ixk17hgMbwpi\\\\\\\/u+nMmaw6c34\\\\\\\/R7l3TAPHwIHY06WH3HG5sHI87ZpmMJajzItFo4euWmM6\\\\\\\/zD3zcom0bH6EtKIwIe3VplmdYGP\\\\\\\/yWEwpN8LEGfZ+A0\\\\\\\/lD+olxEPM7XeAfrPIK0dgkJh\\\\\\\/s4FxbIcruQtuwOSGDDZlYg6Dv6Fk7lSrjcY\\\\\\\/D6BOX+YMgzjP7MtRpkv6rhEknq\\\\\\\/ZUVl4hC9d5H9hLT3nINUhszS6vdbqvyd\\\\\\\/7GZ80cPZTwnR3L25GOF7Bbe4Db+AiYTygc6H5T5LAqfzrs93ax3228JUyX1kpLw6HXuDyKlned0OO20yPJfZf26Om9x8FmPeIdJkgQDHD7s1QtokYQZMq4qlXjJNqQXqhZudlTHWobCLxJF2FSFQEJu0xwrTUs1HHin\\\\\\\/OHUIlax8syYX5wyvApJ0d8NLtoI1Hd7WAxPU3sZtHHsb39e5lffKZIzRDpMH+4M2GF0A+NMCOwWygJgI2jZ96maymvHsXumssc5m0Z2q4kTX\\\\\\\/rnf2R5jbESt9Ehb5DHphgIErKUGI2SgHZbbLnPf9U2AO1A9zNk0Q2kTmneBg6nUhcBvzzb0hVbDCSzxib3RK8xIg1U58k6blonvE4CxOden+\\\\\\\/OxSx0vkhgfxZsJJKumADmiQH5ZZDI6aACE+59hoJcT9g\\\\\\\/akpjdvdgAoYDvm8daef+qvbdtCxAk2IGOtEdEZEjI0R0nncOb6NGwpWaKP+QkT48rmFWyXrsEU9FWMFvQFxJhS1NOR62INg6hNQlPTXDkD0xAF2mY9q+2uXlUQja56sE+ZvQscPxH6NF5SwTFAYmfQdXoBNcRxJUIJcE5KrEgDFrCz9kIxCyvi2siyk17JoVH02JNb4b1oD+QGI23qD7pm9UdjNuCKrC+ebiMdRteTKrxlP5Dc1MCZVaAB8vnvJJW+VrycIXUZtZ00Wbi\\\\\\\/8NuMXQvUCgsbhWwVzmJu0C7qS\\\\\\\/RYqcCIzkOK6dRpz\\\\\\\/3lPthh6NFY07GExyUVn4Rmm32iQ67FrPb72x2F+PcIRbxFVpqU61IsKE94ewngMpftzLgwPBEgZUYY1wLOsnNARsn+qaBkArLCwr0iPktS5vXdR+D8GrSMtkKsHlvRd5NcBO8zvMGj\\\\\\\/JW0r0nJ3EpZRO6PZJhsE4Oqkq4Qk7ePijjl9dNP1ezT+Jd5da2QJdfLWKUPjCqUlXpKvfzklvgwtlsJt8B2zfhidxxGZTCT5y3gag==\\\u0022,\\\u0022iv\\\u0022:\\\u0022QOi8YCZilwdqExEnsIoW3Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022135fba2445b98f106ed05e5c13668ce11eb911c1c24dbbf2e154eb7ec9152a29\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494048 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/2823" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ocEVUAERSheobxuK9uiSsSVTkDxtoMZhrCa31J4KRCc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/2823" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-M_O4KPbMT6IrG7MEBjToGTF9Ckxox5ODcvo85n3I_zc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/9/2811" title="Partial Selfing Can Reduce Genetic Loads While Maintaining Diversity During Experimental Evolution" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/9/2835" title="Single-Gene Deletions Contributing to Loss of Heterozygosity in Saccharomyces cerevisiae: Genome-Wide Screens and Reproducibility" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494048" id="node494048" data-pisa="ggg;9/9/2823" data-pisa-master="ggg;g3.119.400452" data-apath="/ggg/9/9/2823.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">GWAS for Meat and Carcass Traits Using Imputed Sequence Level Genotypes in Pooled F2-Designs in Pigs</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9160-1909" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Clemens Falker-Gieske</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8744-3415" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Iulia Blaj</span>, <span class="highwire-citation-author" data-delta="2">Siegfried Preuß</span>, <span class="highwire-citation-author" data-delta="3">Jörn Bennewitz</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-6782-2039" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Georg Thaller</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-5352-464X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jens Tetens</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2823-2834;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400452 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494048" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Clemens Falker-Gieske </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Sciences, Georg-August-University, 37077 Göttingen, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clemens%2BFalker-Gieske%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Falker-Gieske%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AClemens%2BFalker-Gieske%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9160-1909" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Clemens Falker-Gieske</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Iulia Blaj </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Animal Breeding and Husbandry, Kiel University, 24118 Kiel, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Iulia%2BBlaj%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blaj%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIulia%2BBlaj%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8744-3415" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Iulia Blaj</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Siegfried Preuß </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Animal Husbandry and Breeding, University of Hohenheim, 70599 Stuttgart, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Siegfried%2BPreu%C3%9F%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Preu%C3%9F%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASiegfried%2BPreu%25C3%259F%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jörn Bennewitz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Animal Husbandry and Breeding, University of Hohenheim, 70599 Stuttgart, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%B6rn%2BBennewitz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bennewitz%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ%25C3%25B6rn%2BBennewitz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Georg Thaller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Animal Breeding and Husbandry, Kiel University, 24118 Kiel, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Georg%2BThaller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thaller%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGeorg%2BThaller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6782-2039" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Georg Thaller</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jens Tetens </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Sciences, Georg-August-University, 37077 Göttingen, Germany</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Center for Integrated Breeding Research, Georg-August-University, 37077 Göttingen, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jens%2BTetens%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tetens%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJens%2BTetens%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5352-464X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jens Tetens</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/9/2823" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494048" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494048/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/9/2823.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494048" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494048/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/9/2823.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494048" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494048/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/9/2823.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494048" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494048/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">In order to gain insight into the genetic architecture of economically important traits in pigs and to derive suitable genetic markers to improve these traits in breeding programs, many studies have been conducted to map quantitative trait loci. Shortcomings of these studies were low mapping resolution, large confidence intervals for quantitative trait loci-positions and large linkage disequilibrium blocks. Here, we overcome these shortcomings by pooling four large F2 designs to produce smaller linkage disequilibrium blocks and by resequencing the founder generation at high coverage and the F1 generation at low coverage for subsequent imputation of the F2 generation to whole genome sequencing marker density. This lead to the discovery of more than 32 million variants, 8 million of which have not been previously reported. The pooling of the four F2 designs enabled us to perform a joint genome-wide association study, which lead to the identification of numerous significantly associated variant clusters on chromosomes 1, 2, 4, 7, 17 and 18 for the growth and carcass traits average daily gain, back fat thickness, meat fat ratio, and carcass length. We could not only confirm previously reported, but also discovered new quantitative trait loci. As a result, several new candidate genes are discussed, among them <em>BMP2</em> (bone morphogenetic protein 2), which we recently discovered in a related study. Variant effect prediction revealed that 15 high impact variants for the traits back fat thickness, meat fat ratio and carcass length were among the statistically significantly associated variants.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genome-wide-association-study-0" class="hw-term hw-article-keyword hw-article-keyword-genome-wide-association-study" rel="nofollow">Genome wide association study</a></li><li class="kwd"><a href="/keyword/whole-genome-sequencing-0" class="hw-term hw-article-keyword hw-article-keyword-whole-genome-sequencing" rel="nofollow">Whole genome sequencing</a></li><li class="kwd"><a href="/keyword/imputation" class="hw-term hw-article-keyword hw-article-keyword-imputation" rel="nofollow">Imputation</a></li><li class="kwd"><a href="/keyword/meat" class="hw-term hw-article-keyword hw-article-keyword-meat" rel="nofollow">Meat</a></li><li class="kwd"><a href="/keyword/carcass" class="hw-term hw-article-keyword hw-article-keyword-carcass" rel="nofollow">carcass</a></li><li class="kwd"><a href="/keyword/and-production-traits" class="hw-term hw-article-keyword hw-article-keyword-and-production-traits" rel="nofollow">and production traits</a></li><li class="kwd"><a href="/keyword/variant-calling" class="hw-term hw-article-keyword hw-article-keyword-variant-calling" rel="nofollow">Variant calling</a></li></ul><p id="p-4">Mapping experiments in livestock generally serve two purposes: The first is to understand the genetic architecture of quantitative traits, and to derive and prove new hypotheses of trait expression. The second is the identification of genetic markers that may be useful for livestock breeding. There have been many quantitative trait loci (QTL) mapping experiments carried out over the last decades (see review article by (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Rothschild <em>et al.</em> 2007</a>)), mainly in experimental F2 crosses established from two outbred founder pig breeds. In early studies, genotyping was mainly achieved using microsatellite markers and mapping was achieved through linkage analysis (see overview in (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Knott 2005</a>)). These designs were set up to enable QTL detection with high power, but they suffered from a low mapping resolution and large confidence intervals for QTL-positions. This was partly due to the limited number of meiosis cycles exploited in these designs in conjunction with typically small numbers of 300 to 500 F2 individuals. Furthermore, this approach assumes the divergent fixation of the QTL alleles in the founder breeds, and highly different gene frequencies and variation within these breeds were not considered (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Nagamine <em>et al.</em> 2003</a>). The breed Piétrain, for instance, has been selected for growth and meat yield for many generations and still exhibits a large genetic variation for these traits (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Wellmann <em>et al.</em> 2013</a>). More recent QTL-mapping experiments utilized genome-wide association studies (GWAS), which in contrast to linkage analyses, exploit historical meiosis and rely on linkage disequilibrium (LD) requiring high marker densities. The precision of GWAS is then a function of LD block lengths and the number of individuals analyzed, which in turn limits the usefulness of its application in F2 designs (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hayes and Goddard 2001</a>). However, enormous efforts have been made in the establishment of these mapping populations, usually including extensive phenotyping far beyond what would be available in field populations. It would thus be desirable to revisit these resources using current genotyping and sequencing technologies, which would require an increase in the number of individuals and a decrease in the LD block lengths. In a recent simulation study, it was shown to be possible by pooling F2 designs, particularly when founder breeds are closely related and QTL are segregating in one founder breed (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Schmid <em>et al.</em> 2018</a>). This approach has already been successfully applied based on medium density SNP chip data (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Blaj <em>et al.</em> 2018</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Stratz <em>et al.</em> 2018</a>).</p><p id="p-5">With the aim to overcome the aforementioned limits in mapping resolution and to fully exploit the potential of the resource populations, we pooled four well-characterized F2 designs (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), three of them having the founder breed Piètrain in common. Twenty four founder animals were genotyped by high coverage whole genome sequencing (WGS) and 91 of the F1 animals were sequenced at a low coverage for subsequent imputation to a high coverage WGS level. A total of 2,657 F2 animals that were genotyped with the 62K Illumina PorcineSNP60 BeadChip (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Ramos <em>et al.</em> 2009</a>) were imputed to WGS levels with pedigree information and analyzed in a joint GWAS (see workflow in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). As a proof of concept four relevant production traits were analyzed: Average daily gain (ADG), back fat thickness (BFT), meat to fat ratio (MFR), and carcass length (CRCL).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494128/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494128/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Per cross information of the sequenced individuals (F0 and F1) and SNP array genotyped individuals (F2). F0 and F1 animals served as the reference panel for the imputation of the F2 generation to sequence level for subsequent genome wide association analyses</span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2823/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genotyping workflow. 24 Founder animals were sequenced with high coverage, variants were called with GATK 4.0 and phased with Beagle 5.0. 91 F1 animals were sequenced with low coverage and variants were called with GATK 3.8 and BCFtools mpileup. The F1 dataset was imputed using Beagle 4.0 and pedigree information with phased Founders as a reference-panel for haplotype structure. The imputed F1 was then merged with the F0 variant call data set and phased with Beagle 5.0. Finally the 2657 chip genotyped F2 individuals were imputed to WGS levels with Beagle 4.0 and pedigree information with the merged and phased Founder/F1-imputed dataset as the reference-panel." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-233535348" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genotyping workflow. 24 Founder animals were sequenced with high coverage, variants were called with GATK 4.0 and phased with Beagle 5.0. 91 F1 animals were sequenced with low coverage and variants were called with GATK 3.8 and BCFtools mpileup. The F1 dataset was imputed using Beagle 4.0 and pedigree information with phased Founders as a reference-panel for haplotype structure. The imputed F1 was then merged with the F0 variant call data set and phased with Beagle 5.0. Finally the 2657 chip genotyped F2 individuals were imputed to WGS levels with Beagle 4.0 and pedigree information with the merged and phased Founder/F1-imputed dataset as the reference-panel.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2823/F1.medium.gif" width="440" height="327"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/9/2823/F1.medium.gif" width="440" height="327"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2823/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2823/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494085" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-7" class="first-child">Genotyping workflow. 24 Founder animals were sequenced with high coverage, variants were called with GATK 4.0 and phased with Beagle 5.0. 91 F1 animals were sequenced with low coverage and variants were called with GATK 3.8 and BCFtools mpileup. The F1 dataset was imputed using Beagle 4.0 and pedigree information with phased Founders as a reference-panel for haplotype structure. The imputed F1 was then merged with the F0 variant call data set and phased with Beagle 5.0. Finally the 2657 chip genotyped F2 individuals were imputed to WGS levels with Beagle 4.0 and pedigree information with the merged and phased Founder/F1-imputed dataset as the reference-panel.</p><div class="sb-div caption-clear"></div></div></div><div class="section materials-methods" id="sec-1"><h2 class="">Material and Methods</h2><div id="sec-2" class="subsection"><h3>Description of resource populations and phenotypes</h3><p id="p-8">Four well characterized experimental populations were pooled for this study. Detailed descriptions of the resource populations were done by Borchers <em>et al.</em> (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Borchers <em>et al.</em> 2000</a>) and Rückert <em>et al.</em> (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Rückert and Bennewitz 2010</a>), hence they will only be described briefly. The largest population was obtained from five purebred Piétrain boars and one Large White and six crossbred sows Landrace x Large White. The other three populations stemmed from a Meishan boar or Wild boar crossed with either Piétrain or Meishan females. The Wild boar and three Piétrain females were common founders in three of the crosses. The F2 generation was the result of repeatedly crossing F1 boars with F1 sows in order to obtain large full-sib families. From the crosses, a total number of 2772 animals were chosen (24 F0-generation pigs, 91 F1-generation pigs, 2657 F2-generation pigs) and blood samples were used to extract genomic DNA for genotyping purposes (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The F0 and F1 animals selected for sequencing were generally chosen according to the number of F2 individuals, <em>i.e.</em>, we prioritized individuals from which large families were derived. Four phenotypic traits were considered: ADG, BFT, MFR, and CRCL. The phenotypes were pre-corrected for systematic effects (<em>e.g.</em>, stable, slaughter month) and for the effect of <em>RYR1</em> gene (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Fujii <em>et al.</em> 1991</a>) using a general linear model. Trait definition, descriptive statistics and information about the pre-adjustment and the fixed effects used per cross can be found in (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Blaj <em>et al.</em> 2018</a>).</p></div><div id="sec-3" class="subsection"><h3>Sequencing</h3><p id="p-9">A total number of twenty four founder animals were sequenced with an average 19x coverage at the sequencing facility University Hohenheim. Out of 17 F1 families, 91 animals were sequenced with an average 0.9x coverage. All paired-end sequencing (read length 2 × 100 bp) was done on an Illumina HiScan SQ using TruSeq SBS v3 Kits. For the library construction, the DNA samples were fragmented on a Covaris S220 ultrasonicator. Parameters were adjusted to yield 350 bp inserts. Fragment length was measured with High Sensitivity DNA Chips on an Agilent Bioanalyzer. Sequencing adapters and indexes were ligated using Illumina’s TruSeq DNA PCR-Free Library Prep Kits. Quantification of libraries was done by qPCR using KAPA Library Quant Kits. Flow cells were prepared using an Illumina cBot and TruSeq PE v3 Cluster kits. Raw sequencing data were demultiplexed and converted into FASTQ files using Illumina’s CASAVA software.</p></div><div id="sec-4" class="subsection"><h3>Mapping and variant detection</h3><p id="p-10">Mapping and variant calling of the F0 generation was performed according to the GATK best practice pipeline using GATK v. 4.0 (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">McKenna <em>et al.</em> 2010</a>) and genome assembly <em>Sus scrofa</em> 11.1 (GCA_000003025.6 provided by Swine Genome Sequencing Consortium on NCBI). Base quality score recalibration was performed with dbSNP build 150 as the knownSites dataset. Truth datasets used for Variant Quality Score Recalibration (VQSR) were as follows. SNPs: Illumina Infinium PorcineSNP60 v2 BeadChip and Affymetrix Axiom PorcineHD. INDELs: High confidence fraction (filter settings: QD 15.0, FS 200.0, ReadPosRankSum 20.0) of the PigVar database. Training dataset for SNP VQSR was also a high confidence fraction of the PigVar database (filter settings: QD 21.5, FS 60.0, MQ 40.0, MQRankSum 12.5, ReadPosRankSum 8.0, SOR 3.0) (Zhou <em>et al.</em> 2017). A truth sensitivity of 99.0 was chosen for SNPs and INDELs. The known dataset for SNP and INDEL VQSR was dbSNP build 150. Since SNPs were filtered with two truth datasets a Ti/Tv free recalibration according to the GATK best practice guidelines was applied to the data. Low coverage sequencing reads of F1 animals were processed according to the GATK best practice guidelines with the following deviations. SNP Calling was performed using GATK HaplotypeCaller v. 3.8 in joint mode with the settings minPruning 1 and minDanglingBranchLength 1 as well as BCFtools mpileup v 1.9 (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Li <em>et al.</em> 2009</a>), respectively. INDELs in the F1 variant call dataset were neglected due to low sequencing depth. An intersection variant call set between HaplotypeCaller, mpileup and the founder SNPs was created and stringently filtered with the following settings: QD 30.0, FS 60.0, MQ 40.0, QUAL 300.0.</p></div><div id="sec-5" class="subsection"><h3>Haplotype construction and imputation</h3><p id="p-11">To make use of the most recent phasing algorithms Beagle 5.0 was used for all phasing operations (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Browning <em>et al.</em> 2018</a>). Beagle 4.0 was applied for genotype imputation since it is the latest version that supports the usage of pedigree information (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Browning and Browning 2007</a>). Haplotype phasing of the F0 generation variant call set was done using Beagle 5.0 and subsequent imputation with pedigree information of the F1 low coverage SNPs was achieved with Beagle 4.0. F0 and imputed F1 variants were merged with GATK CombineVariants and phased with Beagle 5.0. F2 generation 60k SNP chip data were imputed with Beagle 4.0 and pedigree information with merged and phased F0 and F1 WGS level variants as the reference panel. Local drops in imputation accuracy were determined by the construction of 24 F0 reference-panels with one animal left out. Genotype data acquired with the 60k SNP chip from each F0 individual was imputed with a reference-panel where the respective individual was missing utilizing Beagle 4.0. The 24 individual datasets were merged and together with the F0 reference dataset converted to additive coding with Plink 1.9 (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chang <em>et al.</em> 2015</a>). Correlation (coefficient of determination, R<sup>2</sup>) for each variant on QTL harboring chromosomes was calculated with an in house R script.</p></div><div id="sec-6" class="subsection"><h3>Genome wide association studies and cluster assignment</h3><p id="p-12">Single-trait association analyses were performed with GCTA v. 1.92.4 beta 3 on the F2 population only (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Yang <em>et al.</em> 2011</a>). In order to perform a “leave one chromosome out” (LOCO) analysis, multiple genomic relationship matrices (GRMs) were created from the F2 60k SNP chip data by excluding each chromosome once with a minor allele frequency (MAF) cutoff of 1%. Mixed linear model association analyses (MLMAs) were performed with imputed F2 variants for each chromosome separately using the GRM where the respective chromosome was left out and a MAF cutoff of 1%. To account for the pooled population structure, covariates representing the different crosses (4 classes) were included in the MLMA. For further downstream analysis, significance threshold was established by applying Bonferroni correction (<em>i.e.</em>, 0.05/number of independent tests). Manhattan plots were created with R, where variants with p-values &gt; 0.001 were excluded due to software limitations. Clusters incorporating potential genomic regions of interest were defined using the Manhattan Harvester (MH) tool (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Haller <em>et al.</em> 2019</a>). MH provides quality assignment for each peak via a general quality score (GQS) which can be used as the main parameter for peak assessment. The GQS is generated based on a trained mixed-effects proportional odds model using 16 various parameters (<em>e.g.</em>, maximal slope, height to width ratio) and human peak identification data. For this study, the variants with a p-value below 1.0x10<sup>−7</sup> (option <em>-inlimit</em>) were included and further the clusters with a GQS &gt; 3.5 (1 is min and 5 is max) were taken into account. Conditional association analyses were performed by including single highly associated variants as fixed effects in a LOCO analysis.</p></div><div id="sec-7" class="subsection"><h3>Variant effect prediction and gene enrichment analysis</h3><p id="p-13">To predict variant effects the Ensembl Variant Effect Predictor (VEP) release 94 was utilized, which is part of the Ensembl advanced programming interface (API) (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">McLaren <em>et al.</em> 2016</a>). The vep command using the clusters’ statistically significant variants was executed with the following settings: <em>–merged –force_overwrite –variant_class –symbol –nearest gene</em>. To provide further functional interpretation, the Database for Annotation, Visualization and Integrated Discovery (DAVID) (Huang da <em>et al.</em> 2009) was used for a systematic and integrative analysis. The gene list from the VEP output was the input for DAVID (Huang <em>et al.</em> 2007) and <em>Sus scrofa</em> genes were considered as the background. Gene Ontology (GO) terms (<em>i.e.</em>, cellular component, molecular function, and biological process) from the functional annotation chart report which were significantly overrepresented with an EASE Score (<em>i.e.</em>, a modified Fisher Exact P-Value) below 0.05 and with a gene count higher or equal to 5 were retained.</p></div><div id="sec-8" class="subsection"><h3>Statement on data and reagent availability</h3><p id="p-14">Sequencing data, which were used to conduct this study will be made publicly available upon publication of the article in the NCBI Sequence Read Archive (SRA). Supplementary Tables have been uploaded to GSA. Supplementary Table 1 contains the coefficients of determination (R<sup>2</sup>) for each variant on QTL harboring chromosomes where calculation was possible. Supplementary Table 2 containts the complete list of clusters identified in the GWAS with additional supporting information for cluster assignment. GRMs from F2 60k genotypes (File_S1.zip) were created by a “leave one chromsome out” approach using the program “Genome-wide Complex Trait Analysis (GCTA) version 1.91.4 beta3”. GWAS was conducted with imputed sequence level F2 genotypes (Supplementary File_S2.zip) for each chromosome using the GRM where the respective chromosome was left out(GRM command: gcta64–bfile SG_F2_chip_wo_chrNO–autosome–maf 0.01–make-grm–out SG_F2_chip_wo_chrNO–thread-num 10–autosome-num 18 ; GWAS command: gcta64–mlma–covar Kiel_Hoh_cross.covar–bfile F2_beagle4.0_ped_ChrNO–grm SG_F2_chip_wo_chrNO–pheno TRAIT.pheno–out TRAIT_chrNO–maf 0.01–thread-num 10; replace NO with the respective chromosome number and TRAIT with the respective trait to be analyzed) Phenotype files are located in Supplementary File_S2.zip for the traits ADG, BFT, MFR, and CRCL were used in the GWAS. Crosses were used as covariates in the GWAS and provided as a gcta compatible covar file in Supplemental File_S4.zip. SNP locations can be found in the bim files of the genotype data (Supplementary File_S1.zip and Supplementary File_S2.zip). Population structure information is provided in form of a Beagle 4.0 compatible pedigree file (Supplementary File_S3.zip). Raw sequencing data are accessible via the NCBI Sequence Read Archive (SRA) under BioProject ID PRJNA553106. File_S5 contains the 60k chip genotype data in variant call file (VCF) format. Genomic positions have been lifted to genome assembly <em>Sus scrofa</em> 11.1 (GCA_000003025.6) and annotated with dbSNP build 150. gcta compatible covar files for the conditional association analyses with top variants are provided in File_S6. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8287847">https://doi.org/10.25387/g3.8287847</a>.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Whole genome sequencing and variant calling</h3><p id="p-15">An average of 592,788,350 (SD = 38,623,216, MIN = 525,921,083, MAX = 649,442,924) sequencing reads per sample were aligned to the reference genome in the F0 generation with an average mapping efficiency of 99.37%. In the F1 generation an average of 26,562,876 (SD = 6,619,568, MIN = 15,915,385, MAX = 59,300,856) reads were mapped to the reference assembly with an average mapping efficiency of 99.33%.</p><p id="p-16">With respect to the number of SNPs detected in the founder population, 22,671,759 were previously reported and 3,950,955 were novel. Furthermore, 1,482,139 of the INDELs were previously reported and 4,335,345 were novel. Per chromosome, average distances among the variants are summarized in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. The Ti/Tv of SNPs in the founder population was 2.39, whereas known SNPs had a Ti/Tv of 2.44 and novel SNPs had a Ti/Tv of 2.08. Due to the low sequencing coverage (average = 0.96 x, min = 0.58 x, max = 2.14 x) only autosomal SNPs were called in the F1 population. The raw output of Haplotypecaller consisted of 20,055,697 known and 3,529,441 novel SNPs and the raw output of mpileup contained 19,932,201 known and 3,291,758 novel SNPs. The intersection of the two datasets resulted in 19,264,662 known and 2,951,058 novel SNPs whereas removing all SNPs that were not present in the founder variant calling dataset lead to a final number of 19,224,132 known and 2,911,780 novel raw SNPs. After the application of a stringent filtering approach (see Material and Methods) 5,753,444 known and 741,155 novel SNPs remained in the variant calling dataset of the F1 population.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494101/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494101/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Average distance between variants discovered in the founder population. A number of 24 F0 animals were sequenced at high coverage and the average distances between variants (SNPs and INDELs) were calculated per chromosome</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Identification of local drops in imputation accuracy</h3><p id="p-17">To detect local inaccuracies in the imputed data, we imputed chip data from each founder with the remaining 23 founders as a reference panel. The data does not provide information about the imputation accuracy of the experiment since pedigree information could not be used. The coefficient of determination for each variant located on a chromosome harboring relevant QTL was determined where feasible. The average coefficients of determination for each chromosome analyzed are summarized in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> (complete analysis results in Supplementary Table 1).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494090/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494090/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Identification of local imputation inaccuracies. Chip data from each of the 24 founders was imputed using the remaining 23 founder animals as the reference panel. Coefficients of determination (R<sup>2</sup>) were calculated for each variant in order to calculate average R<sup>2</sup> for SSC1, SSC2, SSC4, SSC7, SSC17, and SSC18</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>GWAS results and clusters</h3><p id="p-18">From the genome-wide association study conducted in the pooled F2 population, the following number of variants exceeded the genome-wide significance threshold: 448, 17,105, 6635, and 27,641 for ADG, BFT, MFR, and CRCL, respectively. Manhattan plots of the GWAS for the four phenotypic traits are shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. A total of 120 clusters were designated by the MH tool (<em>i.e.</em>, 4 for ADG, 33 for BFT, 22 for MFR and 61 for CRCL) and they were located on the following <em>Sus Scrofa</em> chromosomes (SSC): 1, 2, 4, 5, 7, 17, and 18. The complete cluster list with additional supporting information for cluster assignment can be found in Supplementary Table 2. From each of the defined clusters, the top 5 variants were retained. The genes incorporating or lying nearby these highly significant associations are presented in <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>. The clusters associated with the traits overlapped on several chromosomes, specifically on SSC2, SSC4, and SSC7. The location and the extent of the overlapping clusters is depicted in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. Particular chromosomes had exclusive clusters assigned, <em>e.g.</em>, SSC17 for CRCL and SSC18 for MFR. To evaluate all possible relations among the variants exceeding the significance threshold for each trait, a Venn diagram was used (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The highest number of common variants (<em>i.e.</em>, 6,859) was between BFT and CRCL and the second highest was between BFT and MFR (<em>i.e.</em>, 2,380). To get an estimate of systemic bias, quantile-quantile plots were generated for all p-values from each GWAS (Supplementary Figure 2). As a measure of association between observed and expected p-values, lambda values were calculated for all four traits: λ<sub>ADG</sub> = 1.282319, λ<sub>BFT</sub> = 1.333425, λ<sub>CRCL</sub> = 1.422044, and λ<sub>MFR</sub> = 1.35587.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2823/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Manhattan plots of the &#x2212;log10 p-values for association of variants with the traits (A) average daily gain (ADG), (B) back fat thickness (BFT), (C) meat to fat ratio (MFR), and (D) carcass length (CRCL). P-values &gt; 0.001 were excluded from the plots." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-233535348" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Manhattan plots of the &#x2212;log&lt;sub&gt;10&lt;/sub&gt; p-values for association of variants with the traits (A) average daily gain (ADG), (B) back fat thickness (BFT), (C) meat to fat ratio (MFR), and (D) carcass length (CRCL). P-values &gt; 0.001 were excluded from the plots.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2823/F2.medium.gif" width="440" height="277"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/9/2823/F2.medium.gif" width="440" height="277"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2823/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2823/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494132" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-19" class="first-child">Manhattan plots of the −log<sub>10</sub> p-values for association of variants with the traits (A) average daily gain (ADG), (B) back fat thickness (BFT), (C) meat to fat ratio (MFR), and (D) carcass length (CRCL). P-values &gt; 0.001 were excluded from the plots.</p><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494181/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494181/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Top associated genes for average daily gain (ADG), back fat thickness (BFT), meat to fat ratio (MFR), and carcass length (CRCL) identified in the GWAS. Genes incorporating or nearby the top 5 variants in the clusters are listed with chromosome and cluster numbers</span><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2823/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Cluster overlap for (A) SSC2, (B) SSC4 and (C) SSC7 for all traits (average daily gain (ADG) &#x2013; red, back fat thickness (BFT) &#x2013; green, meat to fat ratio (MFR) &#x2013; purple, and carcass length (CRCL) - blue). The heights of the clusters are according to the top variant (&#x2013; log10 p-value) within each given cluster." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-233535348" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Cluster overlap for (A) SSC2, (B) SSC4 and (C) SSC7 for all traits (average daily gain (ADG) &#x2013; red, back fat thickness (BFT) &#x2013; green, meat to fat ratio (MFR) &#x2013; purple, and carcass length (CRCL) - blue). The heights of the clusters are according to the top variant (&#x2013; log&lt;sub&gt;10&lt;/sub&gt; p-value) within each given cluster.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2823/F3.medium.gif" width="440" height="215"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/9/2823/F3.medium.gif" width="440" height="215"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2823/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2823/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494191" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-20" class="first-child">Cluster overlap for (A) SSC2, (B) SSC4 and (C) SSC7 for all traits (average daily gain (ADG) – red, back fat thickness (BFT) – green, meat to fat ratio (MFR) – purple, and carcass length (CRCL) - blue). The heights of the clusters are according to the top variant (– log<sub>10</sub> p-value) within each given cluster.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2823/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Variants concordance and discordance between the traits average daily gain (ADG), back fat thickness (BFT), meat to fat ratio (MFR), and carcass length (CRCL). The Venn diagram contains statistically significant variants. Intersections between traits include the number of common variants. Numbers of variants that were exclusively found in the single traits are outside of intersections." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-233535348" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Variants concordance and discordance between the traits average daily gain (ADG), back fat thickness (BFT), meat to fat ratio (MFR), and carcass length (CRCL). The Venn diagram contains statistically significant variants. Intersections between traits include the number of common variants. Numbers of variants that were exclusively found in the single traits are outside of intersections.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2823/F4.medium.gif" width="440" height="246"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/9/2823/F4.medium.gif" width="440" height="246"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2823/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2823/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494202" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-21" class="first-child">Variants concordance and discordance between the traits average daily gain (ADG), back fat thickness (BFT), meat to fat ratio (MFR), and carcass length (CRCL). The Venn diagram contains statistically significant variants. Intersections between traits include the number of common variants. Numbers of variants that were exclusively found in the single traits are outside of intersections.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>VEP and high impact variants</h3><p id="p-22">To predict functional consequences on genes the ensembl VEP tool was employed. Multiple transcripts per gene resulted in larger numbers of annotations that are reflected in the higher number of predicted effects as compared to the actual number of identified variants per trait. All inferred consequences for bonferroni corrected variants per trait and their percentage breakdown are summarized in <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>. The large majority (over 70%) of the consequences were classified as intron variants. According to the severity of the variant consequence, intron variants are assigned to having a modifier impact, which means that predictions are difficult to be made or there is no solid evidence of impact. Variants inferred to have a disruptive impact on the protein, leading to protein truncation, loss of function or causing nonsense-mediated decay were of further interest. These significant high impact variants (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>) were mostly located on SSC7, with the exception of SSC2:rs1110687780 (splice donor variant) affecting <em>TCN1</em> for the trait MFR. For the BFT, the most severe consequences were located in the genes <em>C6orf89</em>, <em>PI16</em>, <em>DST</em>, and <em>PRIM2</em>, while for the CRCL disruptive impact variants were found in <em>NEU1</em>, four novel genes, <em>ABCD4</em>, <em>DST</em>, <em>PRIM2</em>, and <em>LPCAT4</em>. Notably, the same two splice donor variants affect the common genes for BFT and CRCL: <em>DST</em> and <em>PRIM2</em>. Sorting Intolerant From Tolerant (SIFT) scores were determined for all significant missense variants and are summarized in Supplementary Table 3 (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Ng and Henikoff 2003</a>).</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494089/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494089/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Results of variant effect prediction for the production traits average daily gain (ADG), back fat thickness (BFT), meat to fat ratio (MFR), and carcass length (CRCL). Bonferroni-corrected variants were analyzed</span><div class="sb-div caption-clear"></div></div></div><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494084/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494084/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Statistically significant high impact variants that were discovered in the genome wide association studies for the production traits average daily gain (ADG), back fat thickness (BFT), meat to fat ratio (MFR), and carcass length (CRCL)</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Gene set analysis</h3><p id="p-23">GO functional enrichment analysis revealed eleven significantly overrepresented GO terms including molecular functions (MF), biological processes (BP), and cellular components (CC). A list containing the GO terms and the associated list of genes is presented in <a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a>. For BFT a GO-MF term was overrepresented and related to calcium ion binding (GO:0005509). Several olfactory receptor genes were prevalent for the GO terms assigned to MFR (<em>e.g.</em>, GO-BP GO:0007186 G-protein coupled receptor signaling pathway, GO-MF GO:0005549 odorant binding). The gene set for the CRCL trait was associated with two BP terms (GO:0001666 response to hypoxia and GO:0008283 cell proliferation) and two CC terms (GO:0045177 apical part of the cell and GO:0031410 cytoplasmic vesicle).</p><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494193/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494193/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title">Most significant Gene Ontology (GO) terms from DAVID for the top associated genes that were identified in genome wide association studies for the the traits back fat thickness (BFT), meat to fat ratio (MFR), and carcass length (CRCL)</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><div id="sec-16" class="subsection"><h3>Genotyping strategy</h3><p id="p-24">The genotyping strategy that we developed for this study is outlined in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>. Briefly: 24 F0 pigs were subjected to high coverage Illumina short read sequencing and in addition 91 F1 animals were sequenced at low coverage and imputed to high coverage WGS levels in order to allow phasing. 2657 F2 animals were chip genotyped and imputed using a merged dataset of F0 and imputed F1 as reference-panel. All imputation steps involved pedigree information. Opposed to a population-based strategy this approach does not rely on a large reference-panel but on the relatedness of individuals. In general, the genotyping strategy can be considered reliable since the majority of the QTL identified were already described for the four traits analyzed in this study (cross-reference with Pig QTL database (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hu <em>et al.</em> 2019</a>)). Nevertheless, we expected to identify a variant that was associated with muscle mass and fat deposition in exon 2 of <em>IGF2</em>, which has been extensively described to influence muscle development (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Nezer <em>et al.</em> 1999</a>). The absence of <em>IGF2</em> associated variants can be explained by a local drop in coefficients of determination from an average of R<sup>2</sup> = 0.22 to R<sup>2</sup> = 0.03 in the genomic region where <em>IGF2</em> resides (SSC2 1469183 – 1496417 bp, <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). It must be pointed out that those coefficients of determination cannot be used to draw conclusions about the actual accuracy of the imputation. Since no pedigree information was included in the simulation, it can solely be used to identify local inaccuracies, which were most likely due to assembly errors in the reference genome.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2823/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Imputation accuracy on SSC2 between positions 1,250,000 and 2,000,000. IGF2 is located between bp 1,469,183 and 1,496,417." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-233535348" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Imputation accuracy on SSC2 between positions 1,250,000 and 2,000,000. &lt;em&gt;IGF2&lt;/em&gt; is located between bp 1,469,183 and 1,496,417.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2823/F5.medium.gif" width="440" height="264"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/9/2823/F5.medium.gif" width="440" height="264"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2823/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2823/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494204" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-25" class="first-child">Imputation accuracy on SSC2 between positions 1,250,000 and 2,000,000. <em>IGF2</em> is located between bp 1,469,183 and 1,496,417.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">The genotyping approach presented in this study can be considered a reasonable strategy to radically increase the marker density of large F2 populations to WGS levels. By sequencing the founder individuals with high coverage and the F1 with low coverage, which are only a fracture of the number of F2 animals, the approach provides an affordable opportunity to improve the power and potential of otherwise obsolete datasets. Due to the relatedness of the animals deep sequencing of only a few animals is necessary, rendering it economically attractive.</p></div><div id="sec-17" class="subsection"><h3>Cluster identification and exploratory analysis</h3><p id="p-27">To fully exploit the potential of the four resource populations, the crosses were pooled and further used for conducting GWAS. The increased sample size together with the increased marker density ensures a high resolution that might allow the pinpointing of more specific causative genes and mutations. Further experiments, <em>e.g.</em>, Sanger sequencing of promising regions could elaborate on that. Designing F2 populations implies that the LD-blocks are longer, a fact that is counteracted to some extent by jointly analyzing the four designs. Lambda values of 1.282319 to 1.422044 point to a moderate degree of p-value inflation in the GWAS, which is most likely caused by the usage of WGS data and a LOCO GWAS approach. However, to exploit the whole depth and power of the dataset we chose a LOCO analysis approach. To further comprehend the closely linked association signals from GWAS, the following approach was employed: i) clusters incorporating strong evidence for trait-associated chromosomal regions were defined, ii) the effect of the significant variants was predicted, and iii) a gene set analysis was employed to identify sets of genes jointly associated with the traits of interest.</p><p id="p-28">The quantitative traits considered for this study have been investigated in the past and are mostly well represented in the Pig QTL database (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Hu <em>et al.</em> 2019</a>), except for MFR. The clusters assigned to each trait were compared with the QTL regions from the database. For MFR, additional fat-related traits (<em>e.g.</em>, fat percentage in the carcass and fat-cuts percentage) were considered in order to allow an adequate comparison given that the trait has few records in the database and the trait definition can be country dependent. Most of the clusters overlapped or were in the vicinity of the previously reported QTL. This was expected as the database has been recently updated and also includes our previous results (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Blaj <em>et al.</em> 2018</a>) using SNP chip data and three out of the four pig populations which were taken into account here. Some of the earlier reported QTL in the database spread over large genomic regions (<em>e.g.</em>, &gt; 5Mb). It is assumed that many of these large QTL regions might in fact not be due to a single mutation, thus representing haplotype effects caused by several causative variants (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andersson 2009</a>). In the current study, we were able to assign numerous clusters within these regions, which implies that a higher genomic resolution was achieved and that it may be possible to disentangle distinct quantitative trait nucleotides.</p><p id="p-29">Conditional association analyses by including the top variant as a fixed effect in the MLMA were carried out in order to gather statistical evidence for putative causality (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cohen-Zinder <em>et al.</em> 2005</a>) and was specifically applied to CRCL and BFT on SSC7. This chromosome exhibits the highest number of clusters (SM with Clusters) and the highest association signals. By including the top variant (rs81228492) for BFT, only one well-supported peak was above the significance threshold (Supplementary Figure 1) meaning that there is additional genetic variation within this region. Similarly, for CRCL the two top variants (rs333021601 and rs319044994) representing the two different significant genomic regions were included alternatively in the model. After fixing the effect of the latter variant, the surrounding significant region disappeared, pointing to the possibility that there could be only one QTL responsible for CRCL on SSC7 around the 99 Mb region. An alternative or additional explanation could be the presence of long LD blocks, long-range LD and/or various epistatic interactions among the loci. The overlap among the BFT and CRCL significant variants (see <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>) localized mostly in the genomic region 24-32 Mb indicate the existence of pleiotropic loci for the two traits. When conditioned on the top BFT variant (rs81228492) as a fixed effect for a MLMA on CRCL and the top CRCL variant (rs333021601) for MLMA on BFT, the initially associated clusters and those nearby dropped in the intensity of the association signals (Supplementary Figure 1), supporting the presence of pleiotropic loci. It is also noteworthy that CRCL might be influenced by the number of thoracolumbar vertebrae (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Rohrer <em>et al.</em> 2015</a>). Since the variant that has been associated with a higher number of vertebrae is a large Indel in intron 1 of the <em>VRTN</em> gene (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Fan <em>et al.</em> 2013</a>) we were not able to discover this variant since the genotyping pipeline applied in this study does only cover small INDELs.</p><p id="p-30">In order to gain insight into the possible genetic mechanisms that control the traits, an enrichment analysis of the gene function was performed with DAVID, prioritizing on the GO terms. The GO-MF calcium ion binding term found for BFT supports the relationship between the calcium ion, food intake and lipid metabolism previously described in the literature (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cui <em>et al.</em> 2017</a>). Furthermore, one of the genes in this group is <em>DST</em>, a strong candidate gene for which high impact variants were found via VEP, which is discussed in detail below. A GO-BP term related to cell proliferation comprised the <em>FAM83B</em> gene, which is the gene incorporating the top variant found for CRCL. Interestingly, the majority of the genes included in the over-represented terms for MFR were olfactory receptors. This enrichment is a consequence of the MFR-identified clusters overlapping regions that are rich in various olfactory receptor genes. This particular gene family is known to have significant expansion throughout time within the <em>Sus Scrofa</em> genome (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Nguyen <em>et al.</em> 2012</a>).</p></div><div id="sec-18" class="subsection"><h3>Variant effect prediction</h3><div id="sec-19" class="subsection"><h4>ADG:</h4><p id="p-31">A QTL for ADG found on SSC7 comprises 115 statistically significant intron variants and 83 variants upstream (min. p-value 8.71 × 10<sup>−14</sup>) of the <em>HMGCLL1</em> gene, which was shown by Comuzzie <em>et al.</em> to be associated with childhood obesity in the Hispanic population and to influence creatinine levels. Another QTL on SSC2 contains 112 intron variants in <em>SHANK2</em> (min. p-value 1.33 × 10<sup>−12</sup>). <em>SHANK2</em> was also shown to be associated with childhood obesity in the same study and to have an influence on estradiol blood concentrations (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Comuzzie <em>et al.</em> 2012</a>). A third QTL on SSC4 harbors 2 intron and 12 downstream variants (min p-value 1.06 × 10<sup>−13</sup>) affecting <em>LYN</em>, which encodes for the LYN proto-oncogene, which was also identified by Comuzzie <em>et al.</em> and correlated with the amount of fat mass in obese children (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Comuzzie <em>et al.</em> 2012</a>). Six additional variants in the QTL on SSC4 (min. p-value 2.44 × 10<sup>−12</sup>) lie in an intergenic region 13,463 – 14,460 bp downstream of <em>RPS20</em>, a gene which in interplay with <em>GNL1</em> is critical for cell growth (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Krishnan <em>et al.</em> 2018</a>). Another likely candidate SNP to influence ADG is an intron variant in the <em>PLAG1</em> transcription factor (p-value 1.32 × 10<sup>−11</sup>), which is a regulator of <em>IGF2</em> expression (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Zatkova <em>et al.</em> 2004</a>).</p></div><div id="sec-20" class="subsection"><h4>BFT:</h4><p id="p-32">A QTL for BFT with a very prominent peak was detected on SSC7. The SNP with the lowest p-value (6.63 × 10<sup>−54</sup>) is an intron variant in gene <em>C6orf106</em>. <em>C6orf106</em> is a target of the human miRNA has-miR-192, which has been identified to have regulatory functions in type 2 diabetes mellitus (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Cui <em>et al.</em> 2016</a>). The second top scoring SNP is an intron variant in the <em>RIPOR2</em> gene (p-value 4.34 × 10<sup>−50</sup>). <em>RIPOR2</em> expression and protein levels are upregulated during muscle cell differentiation in human fetal muscle cells (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Yoon <em>et al.</em> 2007</a>). Another gene containing top scoring variants on SSC7 is <em>KIFC1</em> (7 intron variants, min p-value 3.12 × 10<sup>−47</sup>). Overexpression of <em>KIFC1</em> promotes cell proliferation in non-small cell lung cancer (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Liu <em>et al.</em> 2016</a>). 21 intron and 8 downstream variants in <em>BMP5</em> (min. p-value 1.91 × 10<sup>−29</sup>), which induces cartilage and bone formation (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Wozney <em>et al.</em> 1988</a>), are also located in a cluster on SSC7. 6 variants downstream of the aforementioned RPS20 (min. p-value 1.90 × 10<sup>−15</sup>) were found in the cluster on SSC4.</p></div><div id="sec-21" class="subsection"><h4>MFR:</h4><p id="p-33">GWAS for the MFR trait revealed a strong QTL on SSC2 with variant rs81327136 upstream of <em>KRTAP5-5-like</em> being the most significant (p-value 1.59 × 10<sup>−23</sup>). Of 72 variants 6 were located in <em>KRTAP5-5-like</em> introns and 66 in the vicinity of the gene. <em>KRTAP5-5</em> was shown to control cytoskeletal function and cancer cell vascular invasion (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Berens <em>et al</em>. 2017</a>). Other variants found in clusters on SSC2 are located in or adjacenct to <em>DEAF1</em> (8 intron variants, min p-value 3.47 × 10<sup>−29</sup>), which is a transcription factor that regulates proliferation of epithelial cells (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barker <em>et al.</em> 2008</a>) and that forms a dominant-negative splice isoform in type 1 diabetes, which correlates with disease severity (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Yip <em>et al.</em> 2015</a>). Clusters on SSC2 also harbor variants associated with <em>SHANK2</em> (1,714 intron variants, 3 5′ UTR variants, min p-value 2.18 × 10<sup>−26</sup>) and <em>CTTN</em> (188 up- and downstream variants, min p-value 1.53 × 10<sup>−25</sup>). <em>CTTN</em>’s protein product Cortactin binds to and is indirectly phosphorylated by obesity factor PTP1B (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Stuible <em>et al.</em> 2008</a>). A noteworthy intron variant is located in the vitamin D pathway gene <em>DHCR7</em> (p-value 3.06 × 10<sup>−25</sup>), which has been associated with obesity traits in humans (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Vimaleswaran <em>et al.</em> 2013</a>). A total of 14 <em>DHCR7</em> intron variants were above the significance threshold. A less prominent QTL on SSC4 harbors variants in or close to the aforementioned genes <em>RPS20</em> (17 downstream variants, min p-value 1.51 × 10<sup>−25</sup>) and in <em>SNTG1</em> (19 intron variants, min p-value 1.48 × 10<sup>−14</sup>), which has been associated with type 2 diabetes (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ban <em>et al.</em> 2010</a>). A third, rather minor QTL on SSC18, contains 21 variants downstream of <em>MDFIC</em> (min p-value 1.97 × 10<sup>−15</sup>), a gene which has been linked to improved piglet birth weight (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Zhang <em>et al.</em> 2014</a>). 25 intron and 42 downstream variants were found for the <em>PPP1R3A</em> gene (min p-value 6.92 × 10<sup>−15</sup>), which in a whole exome sequencing study was found to be associated with type 2 diabetes in a Mayan population (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Sánchez-Pozos <em>et al.</em> 2018</a>).</p></div><div id="sec-22" class="subsection"><h4>CRCL:</h4><p id="p-34">In the GWAS for CRCL 52 clusters were identified on SSC7. Although not located in one of the clusters, the two lowest p-values (min p-value 5.40 × 10<sup>−49</sup>) were found in the intron and coding region (silent mutation) of <em>FAM83B</em> (or <em>C6orf143</em>) respectively. A total of 62 significant variants in FAM83B were discovered comprising of 60 intron variants, 1 silent mutation, and 1 missense mutation. Cipriano <em>et al.</em> demonstrated that overexpression or mutation of FAM83B leads to EGFR hyperactivation by direct interaction and consequent hyperactivation of the EGFR downstream effector phospholipase D1, which was previously associated with BMI in humans (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Davenport <em>et al.</em> 2015</a>). An intron variant in the <em>RIPOR2</em> gene with a p-value of 5.08 × 10<sup>−47</sup> is the same SNP, which was found in the GWAS for BFT. A total of 85 mostly intronic <em>RIPOR2</em> variants were found for the CRCL trait. A second, less prominent QTL on SSC7 harbors 9 intron, 12 downstream and 317 upstream variants (min p-value 3.62 × 10<sup>−31</sup>), which have been assigned to the <em>RSL24D1</em> gene. <em>RSL24D1</em> has been identified as a potential target in familial hypercholesterolemia (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2015</a>). One of the clusters identified for CRCL on SSC17 contains 230 variants 122,416-126,520 bp downstream of BMP2 (min p-value 7.21 × 10<sup>−38</sup>), a bone formation inducing factor (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Wang <em>et al.</em> 2013</a>). In addition, 18 intron and 114 variants upstream of <em>TMX4</em> were discovered. <em>TMX4</em> was associated with feed conversion ratios in chickens (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Shah <em>et al.</em> 2016</a>).</p></div><div id="sec-23" class="subsection"><h4>High impact variants:</h4><p id="p-35">Various high impact variants were discovered by variant effect prediction. A splice donor variant (rs80834233) in <em>DST</em>, the gene encoding Dystonin, is associated with BFT (p-value 1.98 × 10<sup>−19</sup>) and CRCL (p-value 1.25 × 10<sup>−19</sup>). Knockout of DST leads to intrinsic muscle weakness and instability of skeletal muscle cytoarchitecture in mice (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Dalpé <em>et al.</em> 1999</a>). Variant rs793752812 leads to a probable start codon loss in <em>NEU1</em> and is associated with CRCL (p-value 1.49 × 10<sup>−12</sup>). A deficiency of the <em>NEU1</em> gene product Neuraminidase 1 leads to vertebral deformities in humans (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Sphranger <em>et al.</em> 1977</a>), which is reasonable considering CRCL is largely determined by the number of vertebrae. Furthermore one frameshift variant in <em>AURKA</em> (rs693811701, p-value 2.95 × 10<sup>−12</sup>) and one splice donor variant in <em>NUTM1</em> (rs331245426, p-value 1.38 × 10<sup>−9</sup>), both oncogenes (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Umene <em>et al.</em> 2015</a>) (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Schaefer <em>et al.</em> 2018</a>), are associated with CRCL. The splice donor variant rs1110687780, which affects the gene coding for placenta-specific protein 1-like, was detected in the GWAS for MFR. In humans, <em>PLAC1</em> has been found to be highly expressed in various types of tumors (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Koslowski <em>et al.</em> 2007</a>).</p></div></div><div id="sec-24" class="subsection"><h3>Application of results in breeding programs and follow up studies</h3><p id="p-36">Functional validation studies based on appointed candidate genes and genetic variants will be considered in follow-up studies. Besides understanding the underlying molecular mechanisms of ADG, BFT, MFR and CRCL, the results of GWAS can render a substantial increase in the reliability of genomic predictions in breeding programs. This concept was demonstrated in several studies in cattle (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brøndum <em>et al.</em> 2015</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Porto-Neto <em>et al.</em> 2015</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">van den Berg <em>et al.</em> 2016</a>) and in <em>Drosophila melanogaster</em> (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Ober <em>et al.</em> 2015</a>) by including pre-selected variants from GWAS results in the prediction models. Even though implementing genomic selection is becoming a common practice, the usage of marker-assisted selection or genomic screening is not obsolete pointing out that the identification of relevant genetic markers via GWAS and post-GWAS analyses is still of practical importance in pig breeding.</p></div><div id="sec-25" class="subsection"><h3>Conclusion</h3><p id="p-37">Putting the results of previous simulation studies to test, we conducted GWAS in four pooled F2 designs, which have been imputed to sequence level based on high coverage founder and low coverage F1 sequencing. We found that by pooling the designs the sequence level marker density can be exploited efficiently. QTL for four well-characterized traits were identified in agreement with previous mapping studies and candidate genes and pathways were unraveled, that should be subject to further studies. Thus, the approach applied herein is a feasible strategy to efficiently utilize extremely well phenotyped experimental designs that have been established in the past.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-38">We acknowledge support by the German Research Foundation and the Open Access Publication Funds of the Göttingen University. We also express our gratitude to Dr. Alexander Charles Mott for proof reading and language advice.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">2</span> Contributed equally.</p></li><li class="fn" id="fn-3"><p id="p-39">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8287847">https://doi.org/10.25387/g3.8287847</a>.</p></li><li class="fn" id="fn-4"><p id="p-40"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-04-25"><span class="received-label">Received </span>April 25, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-06-28"><span class="accepted-label">Accepted </span>June 28, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Falker-Gieske <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.1" data-doi="10.1007/s10709-008-9312-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>,  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome-wide association analysis in domestic animals: a powerful approach for genetic dissection of trait loci.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">341</span>–<span class="cit-lpage">349</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10709-008-9312-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D136%26rft.spage%253D341%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-008-9312-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-008-9312-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.2" data-doi="10.1186/1471-2156-11-26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ban</span>,  <span class="cit-name-given-names">H.-J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. Y.</span> <span class="cit-name-surname">Heo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.-S.</span> <span class="cit-name-surname">Oh</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.-J.</span> <span class="cit-name-surname">Park</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Identification of Type 2 Diabetes-associated combination of SNPs using Support Vector Machine.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">26</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2156-11-26</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DBan%26rft.auinit1%253DH.%2BJ.%26rft.volume%253D11%26rft.spage%253D26%26rft.epage%253D26%26rft.atitle%253DIdentification%2Bof%2Btype%2B2%2Bdiabetes-associated%2Bcombination%2Bof%2BSNPs%2Busing%2Bsupport%2Bvector%2Bmachine.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-11-26%26rft_id%253Dinfo%253Apmid%252F20416077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-11-26&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20416077&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2823.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.3" data-doi="10.1186/1471-213X-8-94"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>,  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. K.</span> <span class="cit-name-surname">Smyth</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wettenhall</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. A.</span> <span class="cit-name-surname">Ward</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Bath</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Deaf-1 regulates epithelial cell proliferation and side-branching in the mammary gland.</span> <abbr class="cit-jnl-abbrev">BMC Dev. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">94</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-213X-8-94</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bdevelopmental%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BDev%2BBiol%26rft.aulast%253DBarker%26rft.auinit1%253DH.%2BE.%26rft.volume%253D8%26rft.spage%253D94%26rft.epage%253D94%26rft.atitle%253DDeaf-1%2Bregulates%2Bepithelial%2Bcell%2Bproliferation%2Band%2Bside-branching%2Bin%2Bthe%2Bmammary%2Bgland.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-213X-8-94%26rft_id%253Dinfo%253Apmid%252F18826651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-213X-8-94&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18826651&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2823.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.4" data-doi="10.1038/onc.2016.234"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berens</span>,  <span class="cit-name-given-names">E. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. M.</span> <span class="cit-name-surname">Sharif</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. O.</span> <span class="cit-name-surname">Schmidt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Yan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wellstein</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Keratin-associated protein 5-5 controls cytoskeletal function and cancer cell vascular invasion.</span> <abbr class="cit-jnl-abbrev">Oncogene</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">593</span>–<span class="cit-lpage">605</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/onc.2016.234</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOncogene%26rft.volume%253D36%26rft.spage%253D593%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fonc.2016.234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/onc.2016.234&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.5" data-doi="10.1371/journal.pone.0205576"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blaj</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Tetens</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Preuß</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bennewitz</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Thaller</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genome-wide association studies and meta-analysis uncovers new candidate genes for growth and carcass traits in pigs.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">13</span>: <span class="cit-elocation-id">e0205576</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0205576</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D13%26rft.spage%253De0205576%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0205576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0205576&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.6" data-doi="10.1046/j.1439-0388.2000.00255.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borchers</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Reinsch</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Kalm</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Familial cases of coat colour-change in a Piétrain cross.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">287</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1439-0388.2000.00255.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D117%26rft.spage%253D285%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1439-0388.2000.00255.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1439-0388.2000.00255.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.7" data-doi="10.3168/jds.2014-9005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brøndum</span>,  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Su</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Janss</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Sahana</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Guldbrandtsen</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Quantitative trait loci markers derived from whole genome sequence data increases the reliability of genomic prediction.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">4107</span>–<span class="cit-lpage">4116</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2014-9005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BDairy%2BSci.%26rft.volume%253D98%26rft.spage%253D4107%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2014-9005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2014-9005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.8" data-doi="10.1016/j.ajhg.2018.07.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>,  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhou</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Browning</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">A One-Penny Imputed Genome from Next-Generation Reference Panels.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">338</span>–<span class="cit-lpage">348</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2018.07.015</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D103%26rft.spage%253D338%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2018.07.015%26rft_id%253Dinfo%253Apmid%252F30100085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2018.07.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30100085&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2823.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.9" data-doi="10.1086/521987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>,  <span class="cit-name-given-names">S. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. L.</span> <span class="cit-name-surname">Browning</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">1084</span>–<span class="cit-lpage">1097</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/521987</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DS.%2BR.%26rft.volume%253D81%26rft.issue%253D5%26rft.spage%253D1084%26rft.epage%253D1097%26rft.atitle%253DRapid%2Band%2Baccurate%2Bhaplotype%2Bphasing%2Band%2Bmissing-data%2Binference%2Bfor%2Bwhole-genome%2Bassociation%2Bstudies%2Bby%2Buse%2Bof%2Blocalized%2Bhaplotype%2Bclustering.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F521987%26rft_id%253Dinfo%253Apmid%252F17924348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/521987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17924348&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2823.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250480900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.10" data-doi="10.1186/s13742-015-0047-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Chow</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. C.</span> <span class="cit-name-surname">Tellier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Vattikuti</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Purcell</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Second-generation PLINK: rising to the challenge of larger and richer datasets.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13742-015-0047-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D4%26rft.spage%253D7%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13742-015-0047-8%26rft_id%253Dinfo%253Apmid%252F25722852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13742-015-0047-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25722852&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2823.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.11" data-doi="10.1101/gr.3806705"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cohen-Zinder</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Seroussi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Larkin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Loor</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Wind</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Identification of a missense mutation in the bovine ABCG2 gene with a major effect on the QTL on chromosome 6 affecting milk yield and composition in Holstein cattle.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">936</span>–<span class="cit-lpage">944</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.3806705</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DCohen-Zinder%26rft.auinit1%253DM.%26rft.volume%253D15%26rft.issue%253D7%26rft.spage%253D936%26rft.epage%253D944%26rft.atitle%253DIdentification%2Bof%2Ba%2Bmissense%2Bmutation%2Bin%2Bthe%2Bbovine%2BABCG2%2Bgene%2Bwith%2Ba%2Bmajor%2Beffect%2Bon%2Bthe%2BQTL%2Bon%2Bchromosome%2B6%2Baffecting%2Bmilk%2Byield%2Band%2Bcomposition%2Bin%2BHolstein%2Bcattle%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3806705%26rft_id%253Dinfo%253Apmid%252F15998908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE1LzcvOTM2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.12" data-doi="10.1371/journal.pone.0051954"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comuzzie</span>,  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Cole</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Laston</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. S.</span> <span class="cit-name-surname">Voruganti</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Haack</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Novel genetic loci identified for the pathophysiology of childhood obesity in the Hispanic population.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e51954</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0051954</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D7%26rft.spage%253De51954%26rft.atitle%253DNOVEL%2BGENETIC%2BLOCI%2BIDENTIFIED%2BFOR%2BTHE%2BPATHOPHYSIOLOGY%2BOF%2BCHILDHOOD%2BOBESITY%2BIN%2BTHE%2BHISPANIC%2BPOPULATION%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0051954%26rft_id%253Dinfo%253Apmid%252F23251661%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0051954&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23251661&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2823.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.13" data-doi="10.1038/s41598-017-01560-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Yang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Zheng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Zhao</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">High-salt intake negatively regulates fat deposition in mouse.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">2053</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41598-017-01560-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D7%26rft.spage%253D2053%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-017-01560-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-017-01560-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.14" data-doi="10.12659/MSM.896772"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Chen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Chi</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Wang</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Comparison of Transcriptome Between Type 2 Diabetes Mellitus and Impaired Fasting Glucose.</span> <abbr class="cit-jnl-abbrev">Med. Sci. Monit.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">4699</span>–<span class="cit-lpage">4706</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.12659/MSM.896772</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMed.%2BSci.%2BMonit.%26rft.volume%253D22%26rft.spage%253D4699%26rft_id%253Dinfo%253Adoi%252F10.12659%252FMSM.896772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.12659/MSM.896772&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.15" data-doi="10.1006/dbio.1999.9263"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dalpé</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Mathieu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Comtois</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Zhu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Wasiak</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Dystonin-deficient mice exhibit an intrinsic muscle weakness and an instability of skeletal muscle cytoarchitecture.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">210</span>: <span class="cit-fpage">367</span>–<span class="cit-lpage">380</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/dbio.1999.9263</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DDalp%25C3%25A9%26rft.auinit1%253DG.%26rft.volume%253D210%26rft.issue%253D2%26rft.spage%253D367%26rft.epage%253D380%26rft.atitle%253DDystonin-deficient%2Bmice%2Bexhibit%2Ban%2Bintrinsic%2Bmuscle%2Bweakness%2Band%2Ban%2Binstability%2Bof%2Bskeletal%2Bmuscle%2Bcytoarchitecture.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fdbio.1999.9263%26rft_id%253Dinfo%253Apmid%252F10357897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/dbio.1999.9263&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10357897&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2823.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081249400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.16" data-doi="10.1371/journal.pone.0140301"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davenport</span>,  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Cusanovich</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Michelini</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. B.</span> <span class="cit-name-surname">Barreiro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Ober</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-Wide Association Studies of the Human Gut Microbiota.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-elocation-id">e0140301</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0140301</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0140301%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0140301%26rft_id%253Dinfo%253Apmid%252F26528553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0140301&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26528553&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2823.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.17" data-doi="10.1371/journal.pone.0062534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Xing</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Ai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Ouyang</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A further look at porcine chromosome 7 reveals VRTN variants associated with vertebral number in Chinese and Western pigs.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e62534</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0062534</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De62534%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0062534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0062534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.18" data-doi="10.1126/science.1862346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fujii</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Otsu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Zorzato</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">de Leon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Khanna</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Identification of a mutation in porcine ryanodine receptor associated with malignant hyperthermia.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">253</span>: <span class="cit-fpage">448</span>–<span class="cit-lpage">451</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1862346</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DFujii%26rft.auinit1%253DJ%26rft.volume%253D253%26rft.issue%253D5018%26rft.spage%253D448%26rft.epage%253D451%26rft.atitle%253DIdentification%2Bof%2Ba%2Bmutation%2Bin%2Bporcine%2Bryanodine%2Breceptor%2Bassociated%2Bwith%2Bmalignant%2Bhyperthermia%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1862346%26rft_id%253Dinfo%253Apmid%252F1862346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNTMvNTAxOC80NDgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjgyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.19" data-doi="10.1186/s12859-019-2600-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haller</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Tasa</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Metspalu</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Manhattan Harvester and Cropper: a system for GWAS peak detection.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">22</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-019-2600-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D20%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-019-2600-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-019-2600-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>,  <span class="cit-name-given-names">B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The distribution of the effects of genes affecting quantitative traits in livestock. Genetics, Selection</span>. <abbr class="cit-jnl-abbrev">Evolution GSE</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">209</span>–<span class="cit-lpage">229</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2BGSE%26rft.volume%253D33%26rft.spage%253D209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.21" data-doi="10.1093/nar/gky1084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Park</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Reecy</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Building a livestock genetic and genomic information knowledgebase through integrative developments of Animal QTLdb and CorrDB.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">D701</span>–<span class="cit-lpage">D710</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gky1084</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D47%26rft.spage%253DD701%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky1084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky1084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.9.2823.22" data-doi="10.1038/nprot.2008.211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang da</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. T.</span> <span class="cit-name-surname">Sherman</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Lempicki</span></span></li></ol><cite>,<span class="cit-pub-date"> 2009</span> <span class="cit-article-title">Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">44</span>–<span class="cit-lpage">57</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nprot.2008.211</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DHuang%26rft.auinit1%253Dd.%2Ba.%2B%2B.%2BW.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253D44%26rft.epage%253D57%26rft.atitle%253DSystematic%2Band%2Bintegrative%2Banalysis%2Bof%2Blarge%2Bgene%2Blists%2Busing%2BDAVID%2Bbioinformatics%2Bresources.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2008.211%26rft_id%253Dinfo%253Apmid%252F19131956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2008.211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19131956&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2823.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265781800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.9.2823.23" data-doi="10.1186/gb-2007-8-9-r183"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. T.</span> <span class="cit-name-surname">Sherman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Tan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Collins</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. G.</span> <span class="cit-name-surname">Alvord</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The DAVID Gene Functional Classification Tool: a novel biological module-centric algorithm to functionally analyze large gene lists.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">R183</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2007-8-9-r183</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DHuang%26rft.auinit1%253DD.%2BW.%26rft.volume%253D8%26rft.issue%253D9%26rft.spage%253DR183%26rft.epage%253DR183%26rft.atitle%253DThe%2BDAVID%2BGene%2BFunctional%2BClassification%2BTool%253A%2Ba%2Bnovel%2Bbiological%2Bmodule-centric%2Balgorithm%2Bto%2Bfunctionally%2Banalyze%2Blarge%2Bgene%2Blists.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2007-8-9-r183%26rft_id%253Dinfo%253Apmid%252F17784955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2007-8-9-r183&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17784955&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2823.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.24" data-doi="10.1098/rstb.2005.1671"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>,  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Regression-based quantitative trait loci mapping: robust, efficient and effective.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">360</span>: <span class="cit-fpage">1435</span>–<span class="cit-lpage">1442</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rstb.2005.1671</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DKnott%26rft.auinit1%253DS.%2BA.%26rft.volume%253D360%26rft.issue%253D1459%26rft.spage%253D1435%26rft.epage%253D1442%26rft.atitle%253DRegression-based%2Bquantitative%2Btrait%2Bloci%2Bmapping%253A%2Brobust%252C%2Befficient%2Band%2Beffective%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2005.1671%26rft_id%253Dinfo%253Apmid%252F16048786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2005.1671&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16048786&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2823.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.25" data-doi="10.1158/0008-5472.CAN-07-1350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koslowski</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Sahin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Mitnacht-Kraus</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Seitz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Huber</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A placenta-specific gene ectopically activated in many human cancers is essentially involved in malignant cell processes.</span> <abbr class="cit-jnl-abbrev">Cancer Res.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">9528</span>–<span class="cit-lpage">9534</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1158/0008-5472.CAN-07-1350</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BResearch%26rft.stitle%253DCancer%2BRes.%26rft.aulast%253DKoslowski%26rft.auinit1%253DM.%26rft.volume%253D67%26rft.issue%253D19%26rft.spage%253D9528%26rft.epage%253D9534%26rft.atitle%253DA%2BPlacenta-Specific%2BGene%2BEctopically%2BActivated%2Bin%2BMany%2BHuman%2BCancers%2BIs%2BEssentially%2BInvolved%2Bin%2BMalignant%2BCell%2BProcesses%26rft_id%253Dinfo%253Adoi%252F10.1158%252F0008-5472.CAN-07-1350%26rft_id%253Dinfo%253Apmid%252F17909063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjEwOiI2Ny8xOS85NTI4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.26" data-doi="10.1038/s41598-018-29802-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krishnan</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Boddapati</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Mahalingam</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Interplay between human nucleolar GNL1 and RPS20 is critical to modulate cell proliferation.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">11421</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41598-018-29802-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D8%26rft.spage%253D11421%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-018-29802-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-018-29802-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.-J.</span> <span class="cit-name-surname">Wu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.-Q.</span> <span class="cit-name-surname">Kong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Wang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Jin</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Cytochrome c oxidase subunit VIIb as a potential target in familial hypercholesterolemia by bioinformatical analysis.</span> <abbr class="cit-jnl-abbrev">Eur. Rev. Med. Pharmacol. Sci.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">4139</span>–<span class="cit-lpage">4145</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEur.%2BRev.%2BMed.%2BPharmacol.%2BSci.%26rft.volume%253D19%26rft.spage%253D4139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.28" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Handsaker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wysoker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Fennell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ruan</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2823.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.29" data-doi="10.21037/jtd.2016.10.67"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Zhan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Zhou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Xing</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Zhu</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The overexpression of KIFC1 was associated with the proliferation and prognosis of non-small cell lung cancer.</span> <abbr class="cit-jnl-abbrev">J. Thorac. Dis.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">2911</span>–<span class="cit-lpage">2923</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.21037/jtd.2016.10.67</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BThorac.%2BDis.%26rft.volume%253D8%26rft.spage%253D2911%26rft_id%253Dinfo%253Adoi%252F10.21037%252Fjtd.2016.10.67%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.21037/jtd.2016.10.67&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.30" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hanna</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Banks</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sivachenko</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Cibulskis</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.107524.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOS8yODIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.31" data-doi="10.1186/s13059-016-0974-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McLaren</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Gil</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Hunt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. S.</span> <span class="cit-name-surname">Riat</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. R. S.</span> <span class="cit-name-surname">Ritchie</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The Ensembl Variant Effect Predictor.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">122</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-016-0974-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D17%26rft.spage%253D122%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-016-0974-4%26rft_id%253Dinfo%253Apmid%252F27268795%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-016-0974-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27268795&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2823.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagamine</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. S.</span> <span class="cit-name-surname">Haley</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sewalem</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Visscher</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Quantitative trait loci variation for growth and obesity between and within lines of pigs (Sus scrofa).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">629</span>–<span class="cit-lpage">635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNagamine%26rft.auinit1%253DY.%26rft.volume%253D164%26rft.issue%253D2%26rft.spage%253D629%26rft.epage%253D635%26rft.atitle%253DQuantitative%2BTrait%2BLoci%2BVariation%2Bfor%2BGrowth%2Band%2BObesity%2BBetween%2Band%2BWithin%2BLines%2Bof%2BPigs%2B%2528Sus%2Bscrofa%2529%26rft_id%253Dinfo%253Apmid%252F12807783%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY0LzIvNjI5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.33" data-doi="10.1038/5935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nezer</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Moreau</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Brouwers</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Coppieters</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Detilleux</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">An imprinted QTL with major effect on muscle mass and fat deposition maps to the IGF2 locus in pigs.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">156</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/5935</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DNezer%26rft.auinit1%253DC.%26rft.volume%253D21%26rft.issue%253D2%26rft.spage%253D155%26rft.epage%253D156%26rft.atitle%253DAn%2Bimprinted%2BQTL%2Bwith%2Bmajor%2Beffect%2Bon%2Bmuscle%2Bmass%2Band%2Bfat%2Bdeposition%2Bmaps%2Bto%2Bthe%2BIGF2%2Blocus%2Bin%2Bpigs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F5935%26rft_id%253Dinfo%253Apmid%252F9988262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/5935&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9988262&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2823.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000078399500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.34" data-doi="10.1093/nar/gkg509"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>,  <span class="cit-name-given-names">P. C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Henikoff</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">SIFT: Predicting amino acid changes that affect protein function.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3812</span>–<span class="cit-lpage">3814</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkg509</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DNg%26rft.auinit1%253DP.%2BC.%26rft.volume%253D31%26rft.issue%253D13%26rft.spage%253D3812%26rft.epage%253D3814%26rft.atitle%253DSIFT%253A%2Bpredicting%2Bamino%2Bacid%2Bchanges%2Bthat%2Baffect%2Bprotein%2Bfunction%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg509%26rft_id%253Dinfo%253Apmid%252F12824425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkg509&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12824425&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2823.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183832900117&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.35" data-doi="10.1186/1471-2164-13-584"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nguyen D.</span>  <span class="cit-name-given-names">Truong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Lee</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Choi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.-k.</span> <span class="cit-name-surname">Choi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Thong Le</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> The complete swine olfactory subgenome: expansion of the olfactory gene repertoire in the pig genome. <abbr class="cit-jnl-abbrev">BMC genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">584</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-13-584</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DNguyen%26rft.auinit1%253DD.%2BT.%26rft.volume%253D13%26rft.spage%253D584%26rft.epage%253D584%26rft.atitle%253DThe%2Bcomplete%2Bswine%2Bolfactory%2Bsubgenome%253A%2Bexpansion%2Bof%2Bthe%2Bolfactory%2Bgene%2Brepertoire%2Bin%2Bthe%2Bpig%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-584%26rft_id%253Dinfo%253Apmid%252F23153364%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-584&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23153364&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2823.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.36" data-doi="10.1371/journal.pone.0126880"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ober</span>,  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Huang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Magwire</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Schlather</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Simianer</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Accounting for genetic architecture improves sequence based genomic prediction for a Drosophila fitness trait.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-elocation-id">e0126880</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0126880</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0126880%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0126880%26rft_id%253Dinfo%253Apmid%252F25950439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0126880&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25950439&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2823.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Porto-Neto</span>,  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Barendse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Henshall</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">McWilliam</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Lehnert</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic correlation: harnessing the benefit of combining two unrelated populations for genomic selection. Genetics, Selection</span>. <abbr class="cit-jnl-abbrev">Evolution GSE</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2BGSE%26rft.volume%253D47%26rft.spage%253D84%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.38" data-doi="10.1371/journal.pone.0006524"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramos</span>,  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. P. M. A.</span> <span class="cit-name-surname">Crooijmans</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Affara</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Amaral</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Archibald</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Design of a high density SNP genotyping assay in the pig using SNPs identified and characterized by next generation sequencing technology.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>: <span class="cit-elocation-id">e6524</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0006524</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DRamos%26rft.auinit1%253DA.%2BM.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253De6524%26rft.epage%253De6524%26rft.atitle%253DDesign%2Bof%2Ba%2Bhigh%2Bdensity%2BSNP%2Bgenotyping%2Bassay%2Bin%2Bthe%2Bpig%2Busing%2BSNPs%2Bidentified%2Band%2Bcharacterized%2Bby%2Bnext%2Bgeneration%2Bsequencing%2Btechnology.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0006524%26rft_id%253Dinfo%253Apmid%252F19654876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0006524&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19654876&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2823.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.39" data-doi="10.1186/s12863-015-0286-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rohrer</span>,  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Nonneman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. T.</span> <span class="cit-name-surname">Wiedmann</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. F.</span> <span class="cit-name-surname">Schneider</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A study of vertebra number in pigs confirms the association of vertnin and reveals additional QTL.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">129</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12863-015-0286-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D16%26rft.spage%253D129%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12863-015-0286-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12863-015-0286-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.40" data-doi="10.7150/ijbs.3.192"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rothschild</span>,  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Hu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Jiang</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Advances in QTL mapping in pigs.</span> <abbr class="cit-jnl-abbrev">Int. J. Biol. Sci.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">192</span>–<span class="cit-lpage">197</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7150/ijbs.3.192</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DInt%2BJ%2BBiol%2BSci%26rft.aulast%253DRothschild%26rft.auinit1%253DM.%2BF.%26rft.volume%253D3%26rft.issue%253D3%26rft.spage%253D192%26rft.epage%253D197%26rft.atitle%253DAdvances%2Bin%2BQTL%2Bmapping%2Bin%2Bpigs.%26rft_id%253Dinfo%253Adoi%252F10.7150%252Fijbs.3.192%26rft_id%253Dinfo%253Apmid%252F17384738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7150/ijbs.3.192&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17384738&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2823.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246980700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rückert</span>,  <span class="cit-name-given-names">C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bennewitz</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Joint QTL analysis of three connected F2-crosses in pigs. Genetics, Selection</span>. <abbr class="cit-jnl-abbrev">Evolution GSE</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2BGSE%26rft.volume%253D42%26rft.spage%253D40%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sánchez-Pozos</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. G.</span> <span class="cit-name-surname">Ortíz-López</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. I.</span> <span class="cit-name-surname">Peña-Espinoza</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">de Los Ángeles Granados-Silvestre</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Jiménez-Jacinto</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> Whole-exome sequencing in maya indigenous families: variant in PPP1R3A is associated with type 2 diabetes. Molecular genetics and genomics MGG 293: 1205–1216.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00438-018-1453-2</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.43" data-doi="10.1002/gcc.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaefer</span>,  <span class="cit-name-given-names">I.-M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Dal Cin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. M.</span> <span class="cit-name-surname">Landry</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. D. M.</span> <span class="cit-name-surname">Fletcher</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. J.</span> <span class="cit-name-surname">Hanna</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">CIC-NUTM1 fusion: A case which expands the spectrum of NUT-rearranged epithelioid malignancies.</span> <abbr class="cit-jnl-abbrev">Genes Chromosomes Cancer</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">446</span>–<span class="cit-lpage">451</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gcc.3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BChromosomes%2BCancer%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgcc.3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gcc.3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.44" data-doi="10.1186/s12863-018-0604-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmid</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Wellmann</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bennewitz</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Power and precision of QTL mapping in simulated multiple porcine F2 crosses using whole-genome sequence information.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">22</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12863-018-0604-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D19%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12863-018-0604-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12863-018-0604-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. V.</span> <span class="cit-name-surname">Patel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. B.</span> <span class="cit-name-surname">Patel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Upadhyay</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Mohapatra</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> A genome-wide approach to screen for genetic variants in broilers (Gallus gallus) with divergent feed conversion ratio. Molecular genetics and genomics MGG 291: 1715–1725.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.46" data-doi="10.1002/ajmg.1320010104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sphranger</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Gehler</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Cantz</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Mucolipidosis I–a sialidosis.</span> <abbr class="cit-jnl-abbrev">Am. J. Med. Genet.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">29</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/ajmg.1320010104</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bmedical%2Bgenetics%26rft.stitle%253DAm%2BJ%2BMed%2BGenet%26rft.aulast%253DSphranger%26rft.auinit1%253DJ.%26rft.volume%253D1%26rft.issue%253D1%26rft.spage%253D21%26rft.epage%253D29%26rft.atitle%253DMucolipidosis%2BI--a%2Bsialidosis.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fajmg.1320010104%26rft_id%253Dinfo%253Apmid%252F610423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/ajmg.1320010104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=610423&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2823.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1977DY86000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.47" data-doi="10.1111/age.12684"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stratz</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Schmid</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Wellmann</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Preuß</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Blaj</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Linkage disequilibrium pattern and genome-wide association mapping for meat traits in multiple porcine F2 crosses.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">412</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/age.12684</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BGenet.%26rft.volume%253D49%26rft.spage%253D403%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fage.12684%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/age.12684&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.48" data-doi="10.1074/jbc.M710534200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stuible</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Dubé</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Tremblay</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">PTP1B Regulates Cortactin Tyrosine Phosphorylation by Targeting Tyr446<sup>*</sup>S.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">283</span>: <span class="cit-fpage">15740</span>–<span class="cit-lpage">15746</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1074/jbc.M710534200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M710534200%26rft_id%253Dinfo%253Apmid%252F18387954%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODMvMjMvMTU3NDAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjgyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.49" data-doi="10.3892/ijo.2015.2842"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Umene</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Yanokura</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Banno</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Irie</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Adachi</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Aurora kinase A has a significant role as a therapeutic target and clinical biomarker in endometrial cancer.</span> <abbr class="cit-jnl-abbrev">Int. J. Oncol.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1498</span>–<span class="cit-lpage">1506</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3892/ijo.2015.2842</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BOncol.%26rft.volume%253D46%26rft.spage%253D1498%26rft_id%253Dinfo%253Adoi%252F10.3892%252Fijo.2015.2842%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3892/ijo.2015.2842&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van den Berg</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Boichard</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Lund</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Sequence variants selected from a multi-breed GWAS can improve the reliability of genomic predictions in dairy cattle. Genetics, Selection</span>. <abbr class="cit-jnl-abbrev">Evolution GSE</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2BGSE%26rft.volume%253D48%26rft.spage%253D83%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.51" data-doi="10.1038/ijo.2013.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vimaleswaran</span>,  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Cavadino</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Berry</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Whittaker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Power</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic association analysis of vitamin D pathway with obesity traits.</span> <abbr class="cit-jnl-abbrev">Int. J. Obes.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">1399</span>–<span class="cit-lpage">1406</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ijo.2013.6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BObes.%26rft.volume%253D37%26rft.spage%253D1399%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fijo.2013.6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ijo.2013.6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.52" data-doi="10.3892/ijo.2013.2030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Park</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">La Marca</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Than</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Rahman</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Bone formation induced by BMP-2 in human osteosarcoma cells.</span> <abbr class="cit-jnl-abbrev">Int. J. Oncol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1095</span>–<span class="cit-lpage">1102</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3892/ijo.2013.2030</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BOncol.%26rft.volume%253D43%26rft.spage%253D1095%26rft_id%253Dinfo%253Adoi%252F10.3892%252Fijo.2013.2030%26rft_id%253Dinfo%253Apmid%252F23900689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3892/ijo.2013.2030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23900689&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2823.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wellmann</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Preuß</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Tholen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Heinkel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Wimmers</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic selection using low density marker panels with application to a sire line in pigs. Genetics, Selection</span>. <abbr class="cit-jnl-abbrev">Evolution GSE</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2BGSE%26rft.volume%253D45%26rft.spage%253D28%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.54" data-doi="10.1126/science.3201241"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wozney</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Rosen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Celeste</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. M.</span> <span class="cit-name-surname">Mitsock</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Whitters</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Novel regulators of bone formation: molecular clones and activities.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">242</span>: <span class="cit-fpage">1528</span>–<span class="cit-lpage">1534</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.3201241</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWozney%26rft.auinit1%253DJ.%26rft.volume%253D242%26rft.issue%253D4885%26rft.spage%253D1528%26rft.epage%253D1534%26rft.atitle%253DNovel%2Bregulators%2Bof%2Bbone%2Bformation%253A%2Bmolecular%2Bclones%2Band%2Bactivities%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.3201241%26rft_id%253Dinfo%253Apmid%252F3201241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNDIvNDg4NS8xNTI4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.55" data-doi="10.1016/j.ajhg.2010.11.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. H.</span> <span class="cit-name-surname">Lee</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Visscher</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">GCTA: a tool for genome-wide complex trait analysis.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">76</span>–<span class="cit-lpage">82</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2010.11.011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D88%26rft.issue%253D1%26rft.spage%253D76%26rft.epage%253D82%26rft.atitle%253DGCTA%253A%2Ba%2Btool%2Bfor%2Bgenome-wide%2Bcomplex%2Btrait%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2010.11.011%26rft_id%253Dinfo%253Apmid%252F21167468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2010.11.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21167468&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2823.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.56" data-doi="10.2337/db14-0803"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yip</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Fuhlbrigge</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Taylor</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Creusot</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Nishikawa-Matsumura</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Inflammation and hyperglycemia mediate Deaf1 splicing in the pancreatic lymph nodes via distinct pathways during type 1 diabetes.</span> <abbr class="cit-jnl-abbrev">Diabetes</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">604</span>–<span class="cit-lpage">617</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2337/db14-0803</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDiabetes%26rft_id%253Dinfo%253Adoi%252F10.2337%252Fdb14-0803%26rft_id%253Dinfo%253Apmid%252F25187368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZGlhYmV0ZXMiO3M6NToicmVzaWQiO3M6ODoiNjQvMi82MDQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjgyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.57" data-doi="10.1016/j.ydbio.2006.11.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yoon</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Molloy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Wu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. B.</span> <span class="cit-name-surname">Cowan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Gussoni</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">C6ORF32 is upregulated during muscle cell differentiation and induces the formation of cellular filopodia.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">301</span>: <span class="cit-fpage">70</span>–<span class="cit-lpage">81</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ydbio.2006.11.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DYoon%26rft.auinit1%253DS.%26rft.volume%253D301%26rft.issue%253D1%26rft.spage%253D70%26rft.epage%253D81%26rft.atitle%253DC6ORF32%2Bis%2Bupregulated%2Bduring%2Bmuscle%2Bcell%2Bdifferentiation%2Band%2Binduces%2Bthe%2Bformation%2Bof%2Bcellular%2Bfilopodia.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2006.11.002%26rft_id%253Dinfo%253Apmid%252F17150207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2006.11.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17150207&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2823.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243467800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.58" data-doi="10.1002/gcc.10307"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zatkova</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-M.</span> <span class="cit-name-surname">Rouillard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Hartmann</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Lamb</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Kuick</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Amplification and overexpression of the IGF2 regulator PLAG1 in hepatoblastoma.</span> <abbr class="cit-jnl-abbrev">Genes Chromosomes Cancer</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">126</span>–<span class="cit-lpage">137</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gcc.10307</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%252C%2Bchromosomes%2B%2526%2Bcancer%26rft.stitle%253DGenes%2BChromosomes%2BCancer%26rft.aulast%253DZatkova%26rft.auinit1%253DA.%26rft.volume%253D39%26rft.issue%253D2%26rft.spage%253D126%26rft.epage%253D137%26rft.atitle%253DAmplification%2Band%2Boverexpression%2Bof%2Bthe%2BIGF2%2Bregulator%2BPLAG1%2Bin%2Bhepatoblastoma.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgcc.10307%26rft_id%253Dinfo%253Apmid%252F14695992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gcc.10307&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14695992&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2823.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187851500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2823.59" data-doi="10.7150/ijbs.7744"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Michal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Ding</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome Wide Screening of Candidate Genes for Improving Piglet Birth Weight Using High and Low Estimated Breeding Value Populations.</span> <abbr class="cit-jnl-abbrev">Int. J. Biol. Sci.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">236</span>–<span class="cit-lpage">244</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7150/ijbs.7744</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BBiol.%2BSci.%26rft.volume%253D10%26rft.spage%253D236%26rft_id%253Dinfo%253Adoi%252F10.7150%252Fijbs.7744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7150/ijbs.7744&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.9.2823.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Z.-Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. O.</span> <span class="cit-name-surname">Otecko</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.-H.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Irwin</span></span>, <span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2017</span> PigVar: a database of pig variations and positive selection signatures. Database the journal of biological databases and curation.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/9/2823.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/9/2811" title="Partial Selfing Can Reduce Genetic Loads While Maintaining Diversity During Experimental Evolution" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/9/2835" title="Single-Gene Deletions Contributing to Loss of Heterozygosity in Saccharomyces cerevisiae: Genome-Wide Screens and Reproducibility" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/9"><span>Volume 9 Issue 9, September 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/9.cover-source.jpg?itok=4kZTDz_V" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/2823" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-mH9dlKH2VFoJCO8V-eSxpXoZ0kl4eFKAmX7Vp8D9CdM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/9/2823" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/9/2823" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/9/2823" class="active" data-icon-position="" data-hide-link-title="0">GWAS for Meat and Carcass Traits Using Imputed Sequence Level Genotypes in Pooled F2-Designs in Pigs</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494048" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-HDUWnmtw2AstR0ueG1d48E5eZ_lcF2SexuujBYaTnSM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/9/2823.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/9/2823" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-qLddBsXBxow_OLXCIgewCNUPOkiYOxYq_IF0rLYpvJM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/9/2823" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/9/2823.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494048" id="node494048--2" data-pisa="ggg;9/9/2823" data-pisa-master="ggg;g3.119.400452" data-apath="/ggg/9/9/2823.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">GWAS for Meat and Carcass Traits Using Imputed Sequence Level Genotypes in Pooled F2-Designs in Pigs</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9160-1909" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Clemens Falker-Gieske</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8744-3415" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Iulia Blaj</span>, <span class="highwire-citation-author" data-delta="2">Siegfried Preuß</span>, <span class="highwire-citation-author" data-delta="3">Jörn Bennewitz</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-6782-2039" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Georg Thaller</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-5352-464X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jens Tetens</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2823-2834;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400452 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494048--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Clemens Falker-Gieske </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Sciences, Georg-August-University, 37077 Göttingen, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clemens%2BFalker-Gieske%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Falker-Gieske%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AClemens%2BFalker-Gieske%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9160-1909" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Clemens Falker-Gieske</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Iulia Blaj </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Animal Breeding and Husbandry, Kiel University, 24118 Kiel, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Iulia%2BBlaj%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blaj%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIulia%2BBlaj%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8744-3415" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Iulia Blaj</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Siegfried Preuß </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Animal Husbandry and Breeding, University of Hohenheim, 70599 Stuttgart, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Siegfried%2BPreu%C3%9F%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Preu%C3%9F%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASiegfried%2BPreu%25C3%259F%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jörn Bennewitz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Animal Husbandry and Breeding, University of Hohenheim, 70599 Stuttgart, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%B6rn%2BBennewitz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bennewitz%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ%25C3%25B6rn%2BBennewitz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Georg Thaller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Animal Breeding and Husbandry, Kiel University, 24118 Kiel, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Georg%2BThaller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thaller%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGeorg%2BThaller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6782-2039" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Georg Thaller</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jens Tetens </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Sciences, Georg-August-University, 37077 Göttingen, Germany</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Center for Integrated Breeding Research, Georg-August-University, 37077 Göttingen, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jens%2BTetens%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tetens%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJens%2BTetens%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5352-464X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jens Tetens</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/9/2823" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/9/2823&amp;title=GWAS%20for%20Meat%20and%20Carcass%20Traits%20Using%20Imputed%20Sequence%20Level%20Genotypes%20in%20Pooled%20F2-Designs%20in%20Pigs" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/9/2823&amp;title=GWAS%20for%20Meat%20and%20Carcass%20Traits%20Using%20Imputed%20Sequence%20Level%20Genotypes%20in%20Pooled%20F2-Designs%20in%20Pigs" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/9/2823&amp;title=GWAS%20for%20Meat%20and%20Carcass%20Traits%20Using%20Imputed%20Sequence%20Level%20Genotypes%20in%20Pooled%20F2-Designs%20in%20Pigs" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/9/2823&amp;text=GWAS%20for%20Meat%20and%20Carcass%20Traits%20Using%20Imputed%20Sequence%20Level%20Genotypes%20in%20Pooled%20F2-Designs%20in%20Pigs" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/9/2823&amp;title=GWAS%20for%20Meat%20and%20Carcass%20Traits%20Using%20Imputed%20Sequence%20Level%20Genotypes%20in%20Pooled%20F2-Designs%20in%20Pigs" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/9/2823&amp;t=GWAS%20for%20Meat%20and%20Carcass%20Traits%20Using%20Imputed%20Sequence%20Level%20Genotypes%20in%20Pooled%20F2-Designs%20in%20Pigs" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/9/2823&amp;title=GWAS%20for%20Meat%20and%20Carcass%20Traits%20Using%20Imputed%20Sequence%20Level%20Genotypes%20in%20Pooled%20F2-Designs%20in%20Pigs" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/9/2823&amp;title=GWAS%20for%20Meat%20and%20Carcass%20Traits%20Using%20Imputed%20Sequence%20Level%20Genotypes%20in%20Pooled%20F2-Designs%20in%20Pigs" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494048/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494048" id="node494048--3" data-pisa="ggg;9/9/2823" data-pisa-master="ggg;g3.119.400452" data-apath="/ggg/9/9/2823.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">GWAS for Meat and Carcass Traits Using Imputed Sequence Level Genotypes in Pooled F2-Designs in Pigs</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9160-1909" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Clemens Falker-Gieske</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8744-3415" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Iulia Blaj</span>, <span class="highwire-citation-author" data-delta="2">Siegfried Preuß</span>, <span class="highwire-citation-author" data-delta="3">Jörn Bennewitz</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-6782-2039" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Georg Thaller</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-5352-464X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jens Tetens</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2823-2834;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400452 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494048--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Clemens Falker-Gieske </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Sciences, Georg-August-University, 37077 Göttingen, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clemens%2BFalker-Gieske%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Falker-Gieske%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AClemens%2BFalker-Gieske%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9160-1909" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Clemens Falker-Gieske</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Iulia Blaj </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Animal Breeding and Husbandry, Kiel University, 24118 Kiel, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Iulia%2BBlaj%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blaj%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIulia%2BBlaj%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8744-3415" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Iulia Blaj</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Siegfried Preuß </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Animal Husbandry and Breeding, University of Hohenheim, 70599 Stuttgart, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Siegfried%2BPreu%C3%9F%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Preu%C3%9F%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASiegfried%2BPreu%25C3%259F%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jörn Bennewitz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Animal Husbandry and Breeding, University of Hohenheim, 70599 Stuttgart, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%B6rn%2BBennewitz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bennewitz%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ%25C3%25B6rn%2BBennewitz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Georg Thaller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Animal Breeding and Husbandry, Kiel University, 24118 Kiel, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Georg%2BThaller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thaller%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGeorg%2BThaller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6782-2039" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Georg Thaller</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jens Tetens </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Sciences, Georg-August-University, 37077 Göttingen, Germany</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Center for Integrated Breeding Research, Georg-August-University, 37077 Göttingen, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jens%2BTetens%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tetens%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJens%2BTetens%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5352-464X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jens Tetens</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2823&amp;rft.epage=2834&amp;rft.atitle=GWAS%20for%20Meat%20and%20Carcass%20Traits%20Using%20Imputed%20Sequence%20Level%20Genotypes%20in%20Pooled%20F2-Designs%20in%20Pigs&amp;rft.volume=9&amp;rft.issue=9&amp;rft.date=2019-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Falker-Gieske%2C+Clemens&amp;rft.au=Blaj%2C+Iulia&amp;rft.au=Preu%C3%9F%2C+Siegfried&amp;rft.au=Bennewitz%2C+J%C3%B6rn&amp;rft.au=Thaller%2C+Georg&amp;rft.au=Tetens%2C+Jens"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494048/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494048/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494048/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494048/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494048/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494048/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494048/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494048/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494048/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494048/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494048/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494048/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/9/2823" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/9/2823.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/9/2823.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/9/2823.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
