<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/9/2913" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/9/2913.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/9/2913.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/9/2913.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/9/2913/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/9/2913.atom" />
<meta name="HW.pisa" content="ggg;9/9/2913" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Deep Kernel for Genomic and Near Infrared Predictions in Multi-environment Breeding Trials" />
<meta name="DC.Identifier" content="10.1534/g3.119.400493" />
<meta name="DC.Date" content="2019-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Cuevas et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Kernel methods are flexible and easy to interpret and have been successfully used in genomic-enabled prediction of various plant species. Kernel methods used in genomic prediction comprise the linear genomic best linear unbiased predictor (GBLUP or GB) kernel, and the Gaussian kernel (GK). In general, these kernels have been used with two statistical models: single-environment and genomic × environment (GE) models. Recently near infrared spectroscopy (NIR) has been used as an inexpensive and non-destructive high-throughput phenotyping method for predicting unobserved line performance in plant breeding trials. In this study, we used a non-linear arc-cosine kernel (AK) that emulates deep learning artificial neural networks. We compared AK prediction accuracy with the prediction accuracy of GB and GK kernel methods in four genomic data sets, one of which also includes pedigree and NIR information. Results show that for all four data sets, AK and GK kernels achieved higher prediction accuracy than the linear GB kernel for the single-environment and GE multi-environment models. In addition, AK achieved similar or slightly higher prediction accuracy than the GK kernel. For all data sets, the GE model achieved higher prediction accuracy than the single-environment model. For the data set that includes pedigree, markers and NIR, results show that the NIR wavelength alone achieved lower prediction accuracy than the genomic information alone; however, the pedigree plus NIR information achieved only slightly lower prediction accuracy than the marker plus the NIR high-throughput data." />
<meta name="DC.Contributor" content="Jaime Cuevas" />
<meta name="DC.Contributor" content="Osval Montesinos-López" />
<meta name="DC.Contributor" content="Philomin Juliana" />
<meta name="DC.Contributor" content="Carlos Guzmán" />
<meta name="DC.Contributor" content="Paulino Pérez-Rodríguez" />
<meta name="DC.Contributor" content="José González-Bucio" />
<meta name="DC.Contributor" content="Juan Burgueño" />
<meta name="DC.Contributor" content="Abelardo Montesinos-López" />
<meta name="DC.Contributor" content="José Crossa" />
<meta name="article:published_time" content="2019-09-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="Deep Kernel for Genomic and Near Infrared Predictions in Multi-environment Breeding Trials" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Kernel methods are flexible and easy to interpret and have been successfully used in genomic-enabled prediction of various plant species. Kernel methods used in genomic prediction comprise the linear genomic best linear unbiased predictor (GBLUP or GB) kernel, and the Gaussian kernel (GK). In general, these kernels have been used with two statistical models: single-environment and genomic × environment (GE) models. Recently near infrared spectroscopy (NIR) has been used as an inexpensive and non-destructive high-throughput phenotyping method for predicting unobserved line performance in plant breeding trials. In this study, we used a non-linear arc-cosine kernel (AK) that emulates deep learning artificial neural networks. We compared AK prediction accuracy with the prediction accuracy of GB and GK kernel methods in four genomic data sets, one of which also includes pedigree and NIR information. Results show that for all four data sets, AK and GK kernels achieved higher prediction accuracy than the linear GB kernel for the single-environment and GE multi-environment models. In addition, AK achieved similar or slightly higher prediction accuracy than the GK kernel. For all data sets, the GE model achieved higher prediction accuracy than the single-environment model. For the data set that includes pedigree, markers and NIR, results show that the NIR wavelength alone achieved lower prediction accuracy than the genomic information alone; however, the pedigree plus NIR information achieved only slightly lower prediction accuracy than the marker plus the NIR high-throughput data.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/09/01" />
<meta name="citation_mjid" content="ggg;9/9/2913" />
<meta name="citation_id" content="9/9/2913" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/9/2913" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/9/2913.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/9/2913.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/9/2913.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400493" />
<meta name="citation_pmid" content="31289023" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="2913" />
<meta name="citation_lastpage" content="2924" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jaime Cuevas" />
<meta name="citation_author_institution" content="Universidad de Quintana Roo, Chetumal, Quintana Roo, 77019 México" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0685-2867" />
<meta name="citation_author" content="Osval Montesinos-López" />
<meta name="citation_author_institution" content="Facultad de Telemática, Universidad de Colima, C. P. 28040, Edo. de Colima, México" />
<meta name="citation_author" content="Philomin Juliana" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Carretera Mexico- Veracruz Km. 45, El Batán, 56237, Texcoco, Edo. de Mexico, Mexico" />
<meta name="citation_author" content="Carlos Guzmán" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Carretera Mexico- Veracruz Km. 45, El Batán, 56237, Texcoco, Edo. de Mexico, Mexico" />
<meta name="citation_author" content="Paulino Pérez-Rodríguez" />
<meta name="citation_author_institution" content="Colegio de Postgraduados, Montecillos 56230, Edo. de Mexico, Mexico, and" />
<meta name="citation_author" content="José González-Bucio" />
<meta name="citation_author_institution" content="Universidad de Quintana Roo, Chetumal, Quintana Roo, 77019 México" />
<meta name="citation_author" content="Juan Burgueño" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Carretera Mexico- Veracruz Km. 45, El Batán, 56237, Texcoco, Edo. de Mexico, Mexico" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1468-4867" />
<meta name="citation_author" content="Abelardo Montesinos-López" />
<meta name="citation_author_institution" content="Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías, (CUCEI), Universidad de Guadalajara, Guadalajara, Jalisco, 44430" />
<meta name="citation_author" content="José Crossa" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Carretera Mexico- Veracruz Km. 45, El Batán, 56237, Texcoco, Edo. de Mexico, Mexico" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9429-5855" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Akdemir;citation_author=J.-L. Jannink;citation_title=Locally epistatic genomic relationship matrices for genomic association and prediction.;citation_volume=199;citation_year=2015;citation_pmid=25614606;citation_doi=10.1534/genetics.114.173658" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=B. L. Browning;citation_author=S. R. Browning;citation_title=A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals.;citation_pages=210-223;citation_volume=84;citation_year=2009;citation_issue=2;citation_pmid=19200528;citation_doi=10.1016/j.ajhg.2009.01.005" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Burgueño;citation_author=G. de Los Campos;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers.;citation_pages=707-719;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0299" />
<meta name="citation_reference" content="citation_year=2009" />
<meta name="citation_reference" content="citation_author=J. Crossa;citation_author=J. Burgueño;citation_author=S. Dreisigacker;citation_author=M. Vargas;citation_author=S. A. Herrera-Foessel;citation_year=2007;citation_pmid=17947425;citation_doi=10.1534/genetics.107.078659" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=G. de los Campos;citation_author=P. Pérez;citation_author=D. Gianola;citation_author=J. Burgueño;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.;citation_volume=186;citation_year=2010;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=J. Crossa;citation_author=P. Pérez-Rodríguez;citation_author=J. Cuevas;citation_author=O. Montesinos-López;citation_author=D. Jarquín;citation_title=Genomic Selection in Plant Breeding: Methods, Models, and Perspectives.;citation_pages=961-975;citation_volume=22;citation_year=2017;citation_doi=10.1016/j.tplants.2017.08.011" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Cuevas;citation_author=J. Crossa;citation_author=V. Soberanis;citation_author=S. Pérez-Elizalde;citation_author=P. Pérez-Rodríguez;citation_title=Genomic prediction of genotype × environment interaction kernel regression models.;citation_pages=1-20;citation_volume=9;citation_year=2016;citation_doi=10.3835/plantgenome2016.03.0024" />
<meta name="citation_reference" content="citation_year=2017" />
<meta name="citation_reference" content="citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=H. Naya;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=A. Legarra;citation_title=Predicting quantitative traits with regression models for dense molecular markers and pedigree.;citation_volume=182;citation_year=2009;citation_pmid=19293140;citation_doi=10.1534/genetics.109.101501" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=G. de los Campos;citation_author=D. Gianola;citation_author=G. J. Rosa;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.;citation_pages=295-308;citation_volume=92;citation_year=2010;citation_issue=4;citation_pmid=20943010;citation_doi=10.1017/S0016672310000285" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Gianola;citation_author=J. B. C. H. M. Van Kaam;citation_title=Reproducing Kernel Hilbert Spaces Regression Methods for Genomic Assisted Prediction of Quantitative Traits;citation_pages=2289-2303;citation_volume=178;citation_year=2008;citation_issue=4;citation_pmid=18430950;citation_doi=10.1534/genetics.107.084285" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_title=Priors in whole-genome regression: The Bayesian alphabet returns.;citation_volume=194;citation_year=2013;citation_pmid=23636739;citation_doi=10.1534/genetics.113.151753" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. M. González-Camacho;citation_author=G. de los Campos;citation_author=P. Pérez;citation_author=D. Gianola;citation_author=J. E. Cairns;citation_title=Genome-enabled prediction of genetic values using radial basis function neural networks.;citation_pages=759-771;citation_volume=125;citation_year=2012;citation_issue=4;citation_pmid=22566067;citation_doi=10.1007/s00122-012-1868-9" />
<meta name="citation_reference" content="citation_year=2016" />
<meta name="citation_reference" content="citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=B. J. Hayes;citation_author=J. Panozzo;citation_author=C. K. Walker;citation_author=A. L. Choy;citation_author=S. Kant;citation_title=Accelerating wheat breeding for end use quality with multi-trait genomic predictions incorporating near infrared and nuclear magnetic resonance derived phenotypes.;citation_pages=2505-2519;citation_volume=130;citation_year=2017;citation_doi=10.1007/s00122-017-2972-7" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. Jarquín;citation_author=J. Crossa;citation_author=X. Lacaze;citation_author=P. Du Cheyron;citation_author=J. Daucourt;citation_title=A reaction norm model for genomic selection using high-dimensional genomic and environmental data.;citation_pages=595-607;citation_volume=127;citation_year=2014;citation_pmid=24337101;citation_doi=10.1007/s00122-013-2243-1" />
<meta name="citation_reference" content="citation_author=Y. Jiang;citation_author=R.H. Schmidt;citation_author=J.C. Reif;citation_year=2018;citation_pmid=29549092;citation_doi=10.1534/g3.117.300548" />
<meta name="citation_reference" content="citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=Genes Genomes Genetics;citation_author=A. Montesinos-López;citation_author=O. A. Montesinos-López;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=C. M. Hernández-Suárez;citation_title=Multi-environment Genomic Prediction of Plant Traits Using Deep Learners With Dense Architecture. G3;citation_pages=3811-3828;citation_volume=8;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Genes Genomes Genetics;citation_author=O. A. Montesinos-López;citation_author=A. Montesinos-López;citation_author=J. Crossa;citation_author=D. Gianola;citation_author=C. M. Hernández-Suárez;citation_title=Multi-trait, multi-enviroment deep learning modeling for genomic-enabled prediction of plant traits. G3;citation_pages=3829-3840;citation_volume=8;citation_year=2018" />
<meta name="citation_reference" content="citation_year=2019" />
<meta name="citation_reference" content="citation_title=A bench marking between deep learning, support vector machines and Bayesian threshold best linear unbiased prediction for predicting ordinal traits in plant breeding. G3;citation_pages=601-618;citation_volume=9;citation_year=2019" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=G. Morota;citation_author=D. Gianola;citation_title=Kernel-based whole-genome prediction of complex traits: A review.;citation_pages=1-13;citation_volume=5;citation_year=2014;citation_pmid=24567736;citation_doi=10.3389/fgene.2014.00001" />
<meta name="citation_reference" content="citation_title=Bayesian Learning for Neural Networks;citation_author=R. Neal;citation_year=1996;citation_doi=10.1007/978-1-4612-0745-0" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=L. Ornella;citation_author=S. Singh;citation_author=P. Pérez-Rodríguez;citation_author=J. Burgueño;citation_author=R. Singh;citation_title=Genomic prediction of genetic values for resistance to wheat rusts.;citation_pages=136-148;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.07.0017" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=L. Ornella;citation_author=P. Pérez-Rodríguez;citation_author=E. Tapia;citation_author=J. M. González-Camacho;citation_author=J. Burgueño;citation_title=Genomic-enabled prediction with classification algorithms.;citation_pages=616-626;citation_volume=112;citation_year=2014;citation_doi=10.1038/hdy.2013.144" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Biol. Environ. Stat.;citation_author=S. Pérez-Elizalde;citation_author=J. Cuevas;citation_author=P. Pérez-Rodríguez;citation_author=J. Crossa;citation_title=Selection of the Bandwidth Parameter in a Bayesian Kernel Regression Model for Genomic-Enabled Prediction.;citation_pages=512-532;citation_volume=20;citation_year=2015;citation_doi=10.1007/s13253-015-0229-y" />
<meta name="citation_reference" content="citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Genes Genomes Genetics;citation_author=R. Rincent;citation_author=J. P. Charpentier;citation_author=P. Faivre-Rampant;citation_author=E. Paux;citation_author=J. Le Gouis;citation_title=Phenomic Selection Is a Low Cost and High-Throughput Method Based on Indirect Predictions: Proof of Concepts on Wheat and Poplar. G3;citation_pages=3961-3972;citation_volume=8;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=IBM J. Res. Develop.;citation_author=A. L. Samuel;citation_title=Some Studies in Machine Learning Using the Game of Checkers.;citation_pages=210-229;citation_volume=3;citation_year=1959;citation_doi=10.1147/rd.33.0210" />
<meta name="citation_reference" content="citation_journal_title=Anal. Chem.;citation_author=A. Savitzky;citation_author=M. J. E. Golay;citation_title=Smoothing and differentiation of data by simplified least squares procedures.;citation_pages=1627-1639;citation_volume=36;citation_year=1964;citation_doi=10.1021/ac60214a047" />
<meta name="citation_reference" content="citation_year=2017" />
<meta name="citation_reference" content="citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=F. Technow;citation_author=C. Riedelsheimer;citation_author=T. A. Schrag;citation_author=A. E. Melchinger;citation_title=Genomic prediction of hybrid performance in maize with models incorporating dominance and population specific marker effects.;citation_pages=1181-1194;citation_volume=125;citation_year=2012;citation_issue=6;citation_pmid=22733443;citation_doi=10.1007/s00122-012-1905-8" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. Unterseer;citation_author=E. Bauer;citation_author=G. Haberer;citation_author=M. Seidel;citation_author=C. Knaak;citation_title=A powerful tool for genome analysis in maize: development and evaluation of the high density 600 k SNP genotypin array.;citation_pages=823;citation_volume=15;citation_year=2014;citation_pmid=25266061;citation_doi=10.1186/1471-2164-15-823" />
<meta name="citation_reference" content="citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. V. Wimmer;citation_author=T. Albrecht;citation_author=H. J. Auinger;citation_author=C. C. Schön;citation_title=Synbreed framework for analysis of genomic prediction data using R.;citation_pages=2086-2087;citation_volume=28;citation_year=2015;citation_doi=10.1093/bioinformatics/bts335" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Deep Kernel for Genomic and Near Infrared Predictions in Multi-environment Breeding Trials" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
<meta name="twitter:description" content="Kernel methods are flexible and easy to interpret and have been successfully used in genomic-enabled prediction of various plant species. Kernel methods used in genomic prediction comprise the linear genomic best linear unbiased predictor (GBLUP or GB) kernel, and the Gaussian kernel (GK). In general, these kernels have been used with two statistical models: single-environment and genomic × environment (GE) models. Recently near infrared spectroscopy (NIR) has been used as an inexpensive and non-destructive high-throughput phenotyping method for predicting unobserved line performance in plant breeding trials. In this study, we used a non-linear arc-cosine kernel (AK) that emulates deep learning artificial neural networks. We compared AK prediction accuracy with the prediction accuracy of GB and GK kernel methods in four genomic data sets, one of which also includes pedigree and NIR information. Results show that for all four data sets, AK and GK kernels achieved higher prediction accuracy than the linear GB kernel for the single-environment and GE multi-environment models. In addition, AK achieved similar or slightly higher prediction accuracy than the GK kernel. For all data sets, the GE model achieved higher prediction accuracy than the single-environment model. For the data set that includes pedigree, markers and NIR, results show that the NIR wavelength alone achieved lower prediction accuracy than the genomic information alone; however, the pedigree plus NIR information achieved only slightly lower prediction accuracy than the marker plus the NIR high-throughput data." />
<meta name="og-title" property="og:title" content="Deep Kernel for Genomic and Near Infrared Predictions in Multi-environment Breeding Trials" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/9/2913" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Kernel methods are flexible and easy to interpret and have been successfully used in genomic-enabled prediction of various plant species. Kernel methods used in genomic prediction comprise the linear genomic best linear unbiased predictor (GBLUP or GB) kernel, and the Gaussian kernel (GK). In general, these kernels have been used with two statistical models: single-environment and genomic × environment (GE) models. Recently near infrared spectroscopy (NIR) has been used as an inexpensive and non-destructive high-throughput phenotyping method for predicting unobserved line performance in plant breeding trials. In this study, we used a non-linear arc-cosine kernel (AK) that emulates deep learning artificial neural networks. We compared AK prediction accuracy with the prediction accuracy of GB and GK kernel methods in four genomic data sets, one of which also includes pedigree and NIR information. Results show that for all four data sets, AK and GK kernels achieved higher prediction accuracy than the linear GB kernel for the single-environment and GE multi-environment models. In addition, AK achieved similar or slightly higher prediction accuracy than the GK kernel. For all data sets, the GE model achieved higher prediction accuracy than the single-environment model. For the data set that includes pedigree, markers and NIR, results show that the NIR wavelength alone achieved lower prediction accuracy than the genomic information alone; however, the pedigree plus NIR information achieved only slightly lower prediction accuracy than the marker plus the NIR high-throughput data." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
    <title>Deep Kernel for Genomic and Near Infrared Predictions in Multi-environment Breeding Trials | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"nb7g9Q23oIYKd2rCmxzEcv3mjkRBeeLPldxsP7ofCg4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494051","apath":"\/ggg\/9\/9\/2913.atom","pisa":"ggg;9\/9\/2913","ac":{"\/ggg\/9\/9\/2913.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/9\/2913.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/9\/2913"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/9\/2913"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/9\/2913.figures-only":true,"\/content\/9\/9\/2913":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022f5H026jkddrAW8dcfGSa4LYVMVZuqGjW91\\\\\\\/SYewNN9n4XnYVdkggxB4CP2Mw1kmq9DnJjKMGptjgp6UuafHupPBdrRW9ZU8j\\\\\\\/PY9KyrtidaypK+S1dGQO7nmbClIACt\\\\\\\/fbPPwwOIhLfA9PQKRhV1XmlO2pAU\\\\\\\/IxSs5aVEobqR6qJlbRypBDvMWD6YzPGGPsPtnP4RkGCqv2qvw8dCw0\\\\\\\/cqmynuDGUHpWpANznPKriwJmnOJtdav2N+7kD3NVv1GfsjydbGGTnT5suQh\\\\\\\/d5LC2BDetxx65rJ0Ik3r5oyK1sV1FH8xVF1owmZO+6brevnECN3R1kXnE1OCRMKpJf7Ef\\\\\\\/XHhpL+CR8fJH5dWhfg4gMYS3rA+85JHl2jXdB0a0G5KFynrJH35UWzUXVgxH1exLI1Z+4FkurRjtT74gI5fwNdhbFdfpqB3l62QIo2+fzuMl0LXyr5A3\\\\\\\/0QkXhjoDuHOyLjX3E2hYDQGIV5JqUmVZbubOS+uG+IECzEKa3xAsYiz9\\\\\\\/vsa51cGKLk1460is03VGUQ0oBa20JLrtd4TAbdr2FZJq9CCIvgZJ8FKCJTPnTK32CDRdiHdtskCAg+9J4vYnO3xb9ickbqZiEUGQ3xzrss7Jy7NiM0fuc0v7t4YFdPHhmQfXHpy1r1EIe4FRlBJxP89OMDCTaqF+AKVfEDXM1n+LWDsCYbMsGu\\\\\\\/8ZktIQGKKuLX2u0PEnhI8lQZxVtD+ZEOC5zMOrf7Qk3dxouS39TaKIldbsmJXU0sEmhlxLXD8M2lv\\\\\\\/273zRkm2XUd3q3+HK+lckS8GGW7uKNoqzcq3IjsuLtsw2+vx0tnraOTQ2nbvPhESoZ7V\\\\\\\/h9fS+JVoZWa94kyLSFSiIfZaSzbEumzlZd2SMTYCy60mgt29w6eBVbyBKjEQTOzkaHI3Tjo2dUP8ACSRefggFwx80FZFsOEJUv6\\\\\\\/2Ft8Efh34gj1GSH0miEGHfjd0ZTXaYJ+z8bC5hw4YGA5siP3ZsqGmPOxdvTmWs9iwo+ORMj\\\\\\\/CQ2rhlA1gyItrXMExFA7troG4JNSXAFz7FHrR4ivEprXQu12F3NW4DGDNw1x3BTKzg+IW2ZGCw1kutqjhkKuI1n6Yx\\\\\\\/RBPigB3BOEbS66eibg5nPW8HBWihDikruKJ3VtUziJrrqXMojG0sbhlH\\\\\\\/2jOpdvQ2K+n5qR4FmysVBN17WD0sQOpfLVXQ25\\\\\\\/tv8CIGfvxarNzqa\\\\\\\/NBjN7wbpSgev7QBoy7iCS7jqDDCG3YLy5EF5N0nuiiSgWuBFZ1mFMpX0ZvP\\\u0022,\\\u0022iv\\\u0022:\\\u0022OxcuNm\\\\\\\/iWNlu7eyUZccV0Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e9e0ca66ff84b39675fe7165bd70048d170755cf90c6600f615bf789a5307408\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022oRRbG3CSMpJTxLN0CYP4Qw7aH0z4axbPGDIS7bI8FDvcbLd37PRJ0RW1vHTr3uOL3hAwqqSzGd7+qajgLbFsd5Y1WobQlHdwJdYHfn2BkmpQQQiqQotvICmzBAx1OV3zdfPbmX1pL4DjXnb+Au5NevuUcZlTyXlIDb\\\\\\\/OYGwkZ8QajJQCP4TGGQ6emVLRj34P2HBaLliY8mE40sd6smhqrqFlJ4acpt68bNNPK2cCMG9hDiCIaw8sHQDdpTqey0aTtNkcW+ViR2U7tAHId4m064GwbxpsZ+PZpNG3zsU4nEPOc5JHNBq9kKv2znMorNrFdA3OdW6njCqmgJUADckkldoM+YSfYRob3K5fZqwgPpb+2b53yayZg7eakC2\\\\\\\/ehhr0j5xtPXnRPCzkl+ashqHmnxe4RKf4+qAcivY18wOAN975gx6EzM0Lb5ygPWbYqnP3F8ESjSOxFopDYTUDcXaiOCjWBJEX5Bag5jt2PR3tevpx6VWFYTytEqZeFDw6cfrKOEK1eciC08TpYHWYp9vGeQUgksmj3\\\\\\\/FEo\\\\\\\/RG3+wn+BEIdgWC2wx+9mU5rhd8NbwOF+o2WDOmg61yXzQOsDPQ4ReWr5MhEsQyKfgiy5\\\\\\\/OpkzNLJ\\\\\\\/m4XF\\\\\\\/Ot+PIGEPSj2Q2IkM7r+7OwtBHDtOXvkmTXbelZdfvOe7mJECIccTkNRmHruISe1abvUstY0KQYO80I6tbVCb7XLq6zxXW88SnmQNYzLBu2qEK8fN3L10c4L3YVm6odLIJQTBXYLp0tahjJDRezzoRysJ7lL5kq3kOgkDu44OV8RkaQ+HN36nCOCTJzIoVdIRA9UpsCfxGgc44SotnX3Rwi7zFXe+HyhpxOBcM3Sk9CotdnLzlq7By2C+IdhnQ6yKLXyNh2\\\\\\\/8vqi\\\\\\\/U1HjLDXBF7DrzfaTV14hfITmY5z0pH4YWySaSc6X0T1RBNeg8prT4BbFu6kOs9KtaN8nOyinO8CrKEEYgkMdZZg4UK8ZG2\\\\\\\/CMnsq6T4cw5+C4ny+kqpztMu6zSkBOo2+T7J2WUVmNl+8dIPne7tlTE5gbjqMtXOGuKoiscCLRrVpCcFpxDyBQRrbtIZkRLjMhdDojrXcv5T8tycBQfUyboDOb9DvNgAE\\\\\\\/rd1DGv+Ad\\\\\\\/sFznes+NAWfQ9meq+W2FjilA4S7i5\\\\\\\/GAUgnf8YAov0tBWXWgWVMitTstnUf1j81vcwGD9wShdXCZ\\\\\\\/gDZE8gA4kYf+SKpN69+wvMMLhHzobgxgWxvri0S99QaG6wV\\\\\\\/un4QpJZxdRbG+hV2ys5AUXvESWeLYk64AtXYWlDhxtsiIhvoFlU9IeTKQ62QeS\\\\\\\/GwkirP9KyuLQFReTNuRxN5pi1ff+Z1MR6mYx8sdNyfsOaHo0DF7Pu7vyDXl0c6XVyOeJOYQ451kaAmvZ0vC8\\\\\\\/ixBqXH4vNS+P1f6vL3IN9sXUkmVEwx5YLFzZOnbL8Mq9ouMQJG+UvAnAeRXQg4lb6lKQIr+RJJUfvVKJ801v7fVsRbPkR\\\\\\\/zNdBTIrv6gdGhLTepLueiFQEHc2kCrTwVK\\\\\\\/dbiy0WYKeh2Ze4IVsUJ\\\\\\\/gg21TEWzBxu36i2Rg7M3kzxs\\\\\\\/iw+y8zHdnl7cFzBuOnHYB0RvGTfGYyi2kyb4jfJLqe712kGEIQFPvySIIghe1FXTLeIkAY\\\\\\\/DQyvgI7aaMr1Zo0TqtoUMpdZAesSXKjZwivJkhagTwhfgQjQmY3JoJLQJHcQ4OrwO\\\\\\\/CfSLzLWT2MJNZt6eFK8A0l50UF0HN054pT0pv4TqzUiKa+gjW37DdSVze\\\\\\\/HxyMiMKQyztzpKymAH8yXTJ7d0wj8uqzc2WMq9tZgWQSG5AADlGM5da1m+Z47DSYm7VLecnTwweZWZzVlpCiYXCDiPPBZQx\\\\\\\/G0bQy\\\\\\\/GgEQI7H5dXMm38Yj93HbYFSJt9wFDQZm9rwVDMSV09MMn7v5kuMI+FXu6d7YbwpOvGFsyV0p1rOpyYEyif9ZjdYLh1qqUAHtk6IRT4XHdKbxoTvI8FUpx6Ztzpamf75ZDhZ2ChVcMKFSs4i\\\\\\\/JqCaWnyx7AnF38RPrIji0XgCkRZUpJ3DUwT1fwbVR30qt0BfZ2FcMm+3KHanvgcz8hV8uWYlxp9LrgrcMTnca4jVxgSyZliAm3\\\\\\\/3XwSIRMIjZpoi9kx\\\\\\\/uzUE8OzI4aCowzcZdjjqFle26V6db5Tw\\\\\\\/DpNkIJK16J6TVtovqQFdIN5FRPLbdBQCA0PzEsl7nxdrTxvHMNcfurARhc97oJV1gHDdRMco6y669blv\\\\\\\/wk2lAj6YLSkgb0Hrpawc4BaceSAy+TaieUHlpyMmMX9NOuLk3hw4nMNaQ5c7FtpCqakEe5UGKxWX5z288TvWPWirlVQuZGFtImt3FdV8+vAAC5XhlIuDLy+5pAbzQKQMoMGHD9XUcq4BWQo1b2f79s5Oo3AuIltd+qO89yQhRGRfLfjBTGlPUnlWVwKAP8iPvJ8FqtzLXOYIFQ\\\\\\\/OawSgGYHkIx\\\\\\\/CItlWLun0PpOd78TBldt3QD3DZnpZU2J1EA0u9yOCUj1fB8Dhl8IkR05sfEB8xZwmU2jBLCK81i9zcXOb0hSNZvoSXCweBXLHhGrjCcr0CLlBiEsDB8CnhwuJXn3aLD76Ts2gLWvn62E\\\\\\\/iizKO75eFc48u32DUcM65Jevt0Wv3+uO6JMqpFixgTblYhGHoT87Ipp\\\\\\\/DRG5BiC78\\\\\\\/VSNNwgiNFq2b5kG9WXKo0BUrkQBG85xubG0uSvL5HWt9xsLFE2DKhrHDymOUMqKRG\\\\\\\/gjjMzQ4TERDV1NWXyDDhh1LUzhqVIJ72Zb1xzkMYMV9\\\\\\\/6JfHCGQ+TjHLvX5FTowmDCDaNea+t7SSVKSECL8Bp5qtRbOYBbQIlL3\\\\\\\/AKbFLqmHLVbYEPGwOIn31pT+wRodWjFVj1v\\\\\\\/I97skt+KyZsWUtruyRPW9JVhPlRMsXBJJzJ0oOSMuw\\\\\\\/DFOaVNNu9\\\\\\\/t\\\\\\\/hlFdujIGVQkD24N2+aTPwSBwOcjdyQWLjsyzYR2cl3DM+8r3MfaM6mcvPhb9Pb\\\\\\\/PB3B5BqXk6Lcf8eCAEb\\\\\\\/QjkU0lLnrYB9Ew+ygNaDn7R3dS7KxtgiVI0B7UXcX3xnvYV17Uvr+TnTEHaXcdW258z7+u6Q+xQnB+w+Fb+g1YTAsiRzXb35P7e4nO+qjN17qdRXuuAnl9Uw3J0ZdeUVGMdHvOMS3mA9PMJfphFzeFTxQHtfL8VSl\\\\\\\/WBzd2uTKulLYOwYNHk3LZlMtKjx3EmbcTf8ZSuclw2pcjLW9oWUauEb9DQEM4vi8ipkkQ59bOuw3C4XfN2Io8gnvLmA8DQvvNbdFCL5kVGssgkhS6Q84pTP9fqZjwto9YTG+VVmPRfKFCEMCmufTsPHJXXiHLxhWEmT24M201kHp2NjD1BDJUZr8lewJFC53AKqx9JmYLSoS6GF+sJMLnpKxXRB3CJojmGZ9cyaqT4rZ3Y9vIwdt5+L0mN2LjxVcLViwkrXgrSoLWwv61rpZU\\\\\\\/ca+D4QosCNLY6GI4xLryAMAM6LRim9UqDszCewa1zr3FwLdg1Ua8nEv88wEYDcaYJwTNCzvlIg5AdfqBdRUsiXnP901+m8IFDPPYnthOvwAPm\\\\\\\/biGDticDGMJzdZzFsOwmbeMhVdHxBe9\\\\\\\/tT18E0bkjik5+uOgU9RlwPPzNQWm5QHPR5NwBZMMKIS9HJW4XGo+KmXOUZUsKB4EQHfRpq\\\\\\\/FUN2ym0JydwOg+wx7AWe0Vle+1\\\\\\\/4NYaMzFvjut2nTgU+Udfjn5ecNyQQZaow+4c0ir70Z8VPQ\\\\\\\/nreVZnGcBvUO+7\\\\\\\/wxMcMWj+AyZAzYqLGDj230JcFNG4pazGSylOfVdOnbuhtkNUTUXdZo6toZ\\\\\\\/AlFCxq\\\\\\\/9jiWSBpKZ\\\\\\\/JDz\\\\\\\/4MNgj2XoPGKQb90PdP44AgWnxkM0lRntFjpCqjgtimu3FlCeMdQA\\\\\\\/RGHKROEFVYD6MFADeFjoNnDbVoGvLOonQN18dW6YlzVh9qUDZ2yfkN5PHc5Z+FpfPtXAIHUCvXMH+BKkNy7MM1F1k\\\\\\\/yITDIPpTNtM4rcA3knO8sh1S6oDDJkd\\\\\\\/4W4di7KFjhHY5\\\\\\\/sQ2yjropvddEi4Pa4CMNrYsZLQA6wwSuVSo+iTaK7n0RhEU+lFiTAJw884PzIsxcMn\\\\\\\/fX0pgiblYYBbS1JFVYoF7GJwPsaIvWO2dLLZaCn7O0aCsozVmS+p94t\\\\\\\/tibj3i50qgOP1ePKaXN0fumKIXX8dUC8lTUify0vEy01+E6RzWSphIOYtckQmKOt1PpqLMULyNKBfpwwGUBgMiceRYL4R9yOuVJR1E3++VyME6KHeWOcbVOo+sFIO9lyUelgk17pC10BjRV+I6io8cfYG41t6pjCZuM6W54G1F67kwO366VeBvg4omHjx82cF+zW6cbdNwzwKffIFZ46SR0hJXmd\\\\\\\/sC+9eWKwz6F0dZzODekVOvN+fIFe7lIVJfJmiifOTxBz9s89ocpv8a2f6iH9i\\\\\\\/4a7N5jpCQyKxqbioKt21dTlzUJyT90Ez+gHBkjcxGiVJm7HD9GMzYXDJNhNt5oK+AetwCy7Ouyunuu3v+gDlvKFjLt0uRTU24Tn852PRGxI6svuULkCdKdNcbwndN8Yak7CAC9EEEY8zike3rt9G0VNhVqmiSQDSln\\\\\\\/\\\\\\\/aPyDAYSFCfJq+ysfWXm96NNNREiH+AHzqLQc5CLdb+ymdmCRplM1F2q2fs2erLJ8N0kPN1FXSu55chqSLF2QJ7ZoV2sDxQ5FtyC8XTrps43KmXtpfA01I38lYOZVjpaB2566nrXPppYB\\\\\\\/P57\\\\\\\/11EcQqTcVKg6Vkenzhc4rjnpiiTfMmSiLQLoFjh96nSkEjppLoQVoR4t\\\\\\\/dasSaoSdUNcDsUMtkw+W9+qe+FRete8otPIPkWJg2eBqu7h17cRmwYHGPE+bLIzYzmY8EAFKECta3QwRgrBeuVf1f9eyxtlzqEivAGX0l2PKDXudh24XaYJ8QHjwpoprap6YQuu+EndwtbMxkx3K\\\\\\\/OhmPewGtKU01BV63O7YpKde4Vf5xFEB9phqKZOrT3S5Avgdx4d9Sa7xfKUHZiRqd9XpKVtzPSqFaim64MY\\\\\\\/p8kUIqqN2w+pmNjXDpuqDHscSkJ+wRCqKmsvMGyqE0Q8IkglzkT7rLk\\\\\\\/Pt9njto3f6hNDRFvKb17Kh3S2ed2t6zfqEU\\\\\\\/XpxMFPIYBsruWl52B9C+ft8xsnuOmcNEu7NxuchvW5F1BHlwKXWtIgQ5FUnKwozQefiJQW9LzDpOROQu+TJieuOmvt2sANJ8DOyYbDR2E010627U5mPN7DW2uWXxaKey2MNCMtvQUeADdGFT04fT7dXQbsVB0Y9outJESnf4lH9+zV8vqFHrtr0dyW0jFELITmaFVxFnMWa5IvAiTiR\\\\\\\/CdEu8aj6bXaPsHoOyL3maY94PVKx5t9E2S80S7k9AXBsNxAb+l3Tv8g1GoGUp4y2ozLQjsPaG\\\\\\\/\\\\\\\/Yl3C1ZUBOdufSAx7yxNpRwvU8bojdPpdMvDSCA1f5EPNynEBDSuF38P0mNRimFo1pbTbvQIT0qimFxV\\\\\\\/nTWO1UQw0HNmNWBF446DAH\\\\\\\/SxMlO9qGUAfgfhqXOzROXG5Qf3Vck4lC6SvgMvbYXusOxgkOFlJYL\\\\\\\/NWD2aVatLKDaxs7sDSysmCETuEnECCmShS0d3diNlDN+nKjPaptmZBCZcDJI7Po2IOOiginvjLZgxxve3+ypO44ecoenjCur4aLk26JViNKcfTOl7APWhB6Y41eL0HLCuXNqjBGszqDwjjsz8nRBu\\\\\\\/7IEPTWCmDVI+u3dUkl0jD70WctS7rqjot9faTJm21fFvCThiAW1ncLMpmUJm9VDORjurpyvVAuL9klfY0wTyEXItz9ZT0H0nODsg7deIwrgFtNEjS2w0PYdZiidCyqrYvy30kEdifnmDuYO445MhQEUH0su4niWZrmMDcfQ1ZAmZAXyXnQRkTBytxvydByQowD8bJGMlpviv5OiCOEXV9Deocm\\\\\\\/n703x2WfzFzTRIQKfAdtbv2UwwxOPcxwVXkf4X79PIyfcbJYU6+KY3ARQPqZru56Nqx63pzUDhczkbvCKjTB3gXCimacSqBLj6qyz9rhRgx2Uq7T0G9aTizq9sXt0iQd0JHtPo\\\\\\\/7IOr+UryBEMBqB6iAyWpTRXKaVrYgojRWzDUGfYbOZzjDWNkzaMfKomIYcNtGo2X4korv3iECqv+PpfJQjcWxSXn9X3B0y9vC4Etdsl8a+Xc1VzpStUNKuac6W2hChG13BABx3iaTkHcZetFm8cLUEyxto42aMFEgrcf\\\\\\\/vzVjbAypfMNIJrtZuwm0EzSWF3k3bFDwEWvHvze+ZaM0EbyWmVvugTt9xXAO4Dq4KDw3ULF+uBPYhK+B5snaloEsQT42FGU5mu0499qszd97Db8JfRh3uEu8dJ1znHcB9hNH43Lk0Cp0\\\\\\\/9AgKBKLYx9L+B4K8epDxfUFbL\\\\\\\/lOFggbUGSZugILnCrgDZiC2VgifETL60syItoXon5J0iLO+jay+m0KZl415AvDUL6BXl0bJ7n3gRnoXhM3ust9BsRTSL2VwOq7MsJltLMcyqExCwow0qXHW9SqvMBCCwa\\\\\\\/L4EzgORK39eKxA+DoI6zWnMpFm17nKxEkoX1cPhR\\\\\\\/JgEwY79x4V2vsh9MKvZ5VP+9X11gRlqKlQwswiQua2gqe53QTevMmPuNWIQzSayQihqoCT2jggOTwmQrPAVHUCuxmsTfVcFj9MKAHGcUz3XqXrWAEAjqz\\\\\\\/AQN2je9j\\\\\\\/aic597DcqIxbMsbFkOWSjFTETZsVkyvZfmVPlyZv5aSEALnM+Ebfzm10EdiUAuN60V6+B20t5bUmrVzDS4J1Ir+X3OaP2OB+WfgDFBxj89HG6HJdHM7e\\\\\\\/bvcqqBIaNfB4QHKk7uyWbFS9XVEvwaBcss3cuKN6xjxKuTD4zR1tsCm1eX9lFdlkZOomrcqNnnR0uZNJIR\\\\\\\/GuZ9sW25j\\\\\\\/sG4bdNqXaR1IEdiL+tbldhZoEYdt\\\\\\\/AKG\\\\\\\/e6OqZh2Lqylw8aRcuh\\\\\\\/nhVXZIkIaFaHEKM3GApuThx1amM8VakpobeMAOZPEtdNN46b7fWQmc4iqw9opFNFSwJ9S2AZPRhu8UYa0HVULfvP30Pxw8\\\\\\\/v2XHUM+NoflHKt4d2jgWMd\\\\\\\/XWbAY3iSmDD7y1hLFpy1DvZcCEYiwX55McLArCi33WgNgtMngR4IwSt95WJNPxeehxQzM54iDXOxjyW6CFCPBaettEhcBNONAggafuxJB0AbhE4ziqXi1BUhOo5LWR5jH580IjCGVl+ZNLQqoE3JpfPfA1hmVOc0SWFtwcGBXU0usQDBlmo4TS1fp6lCE34crI4Cj25sPjW9gJzcPtvBhuQda1pnrUyeq4DoVNde75dgxVN0MV5RKXLlesOiOLdeMV6QT3dcEodY75DDiT7K3mN2YKVJagcWxXydQcFeKuEJO3gkzyq\\\\\\\/LDTExggGRKHuFLI7l6Q9Fhe7XCkjYu4xd8WexpyPaDjxPZcmTTX2fo4nPu2O37sCCAzMO3BeANcI5BU325b+E3EneR0brBSiOqs5bRCu+sbUUvusM4TRcTP4QA0EXaM3uETmDAvg\\\\\\\/zqHtTn+\\\\\\\/+PGhSmH8rYXE+0OHBUVnMZvtuQAOyo2PrtNCgTyFJRvJGxWydWMBomdByP1jSdmifnCZ2j+7t4PfhSP+G5Zb8xhfb3e9vKExVehoBE2o92kB5mahcUpD8FaEBMDnCf2J\\\\\\\/bF0UC2M\\\\\\\/uMZbJ6OVhVMANFbjBr6W5f7e9lkVCvFicA4\\\\\\\/GcetFyD5hCEbJQOQOwiBFh3Kxmafd9N0BLbErlrPBSTC6fj0qnLXVKl+AGFNvNErighVPrMfEC35Ux9fGGJibRY0tq0nVlQmenyVFzhZN7+bA\\\\\\\/mZ14oPYOPwR9j3LHfpW53c6o1NpNqYmiollcEuKGYfNkVWpA9je+n7PdI\\\\\\\/XYAM4HZscCzO0v1IkXowBt1bTfH08AZrSkm9Zu8cpG4sss+x8G9sNmTWkc3glpXKlwAOAn2s5Yd8rrdvehKwCaOua7Ur5NEGi6cUSF4IHg3tyIYYl07Lj\\\\\\\/\\\\\\\/pd4Qbgg1GVKmIYdDUv4emaHt\\\\\\\/GU4oc1Iyv3D7f70OdKiehmD87+xrXGOAXCDw9B1LnEzl66IJwWpUFqPA\\\\\\\/8Vt4AARy1EKhcD4TfQ+WSa1b6PvjmUeGwmfTeVu\\\\\\\/cSLVZ5q3\\\\\\\/a1c6EGve2kWdZTqwMMPYqKPtyjwn0ExXEUHWTRrHQ+PT92Hq5+IxQhnP4tCeRyWRqQd84KvxhAa\\\\\\\/Zz9\\\\\\\/0gGBvMp\\\\\\\/BfbmCRnm8YwRy+bRVMgqOJoLih2BB6nZligwoLnNliaGEo4xLy6rAEtQzePfI2sYzWjNsvvJ0+NtF+3p3+vBV4fH+d5Udn31ymkhVHe6bu7fFcU8yw7S\\\\\\\/VN4aO7jOCTRhHPyT6Vypc2tVVtEXyy1s8W1ho3m5Cw\\\\\\\/Ld+fYm\\\\\\\/Z6VOE3g6zYwGztjdmnWr2zdUFxNNP\\\\\\\/8ms9VP9bLy\\\\\\\/NjzDHZmBR33HjgrTfBqnivdvcsSF1kX9BeNmGULSn1aYWYZbcvVLt5IXfBAroZZtQOxi1o7u3OgHEkoYGjlQrk8LmcNTUn6IUkytI+KMaegMPiwskJnhdZlWPY0nzBmpfiIlcqp1LlOLjzUXHIPtV0HrWc5tUU1taO1qGRGyxQ97hPYsIHmqzyE\\\\\\\/mLaqPaNGhLPrU2V\\\\\\\/rw7vpMiJk9HSaNg9jk+xHKVwsI77tZgGgBPto8fJ5JAHkfJwBtEUGsAtbWquQHA1RQMNGCzfUIqZloYRd+4QzidoEkfA0TRINrtdcv9IlbVKizEIPv+179Fa5VTb2I1\\\\\\\/Ylhv\\\\\\\/LCe75ESbVPflvBgNpIbqVswU7Rq8wf80LF8WI\\\\\\\/o0oUxAeibE+4sSLlhsFVJpu7FqwUrUIPKmucA7O9FHK3O3J6amSFl5KHWE0NqMHb88Vi7cXsKDp6\\\\\\\/yPFH4dfwy3wc3zKZYmTAaMFFX6naL8dUUaWYYNBBYHQ\\\\\\\/tN+nUjnRq+Oog8BRxRQ6qjNjLmW5sggtmocgvARDBiXRxnhCHJBDg4ut20DVVry14sUGo202A1gfUX+3GpSioall3R1nHCbddTOf4FK2h6mFjl77ZNGu3a\\\\\\\/k3T3x1FXDcu+lUSkcQhex9qIMp76N6CZxddMBBK8PGQbnn8YXmjNfGeUTqJbaN8cpR9sKOizdqSv60vS0SwUNfNXhqcfijaH8LwHtBL530khMmryUv111rVliqHGGDtMmrov64puoXh6qnM735UFU7ywg0YdNI3AdTIQgQgghr7XK86vUJDco0tm92AUC5vSX8eHOz2YPz\\\\\\\/1ltHJ6nneI4Xt31TTxGBOvNEhMlOQjzoEbnL5a076le2jtTtWg9lPu8mFj\\\\\\\/yKtUBuT2L192qJ+VmFEra1KWn0KC3DZ8SBnoeh\\\\\\\/ykaQp26CJpV6iUnjy7HOfaF7kmGdTfvHG+gAuJamcFGXy+r2q09WAGSb4rYL2YEq1lHk1jHW4z67AV1PKoK94peT6N+rX3Fjjhycj0jHfeShIRISuC697LAoggwVa2gN8ImKqLBu53RphXihoNLzBlA4QSHltlg1fNt1tYEWUaq56rWu5fyttRMHqADZ1YQDGgh3jz54g5YKXKs\\\\\\\/2OZGTB77iQYE8m4Mve42ftF2L8SwQGl9ogChX7WwbijgZgBd37n9azKAauubHpTSQHyWtObN8DF0VeAoUdKr\\\\\\\/6+GaiBGa82wgI0WQ6twwMU0VZG+DMJqtkhE40OhrzAVEjIymDwq3cuMsyJrBqhmi17sDynWlSi0OzUI3s2CqPZF+qklwXbnb62\\\\\\\/Q9p5wbt8Ucoi3xsN3BRPGd\\\\\\\/+87j1zMtJS5JLDZkp+CWGn1C1EHHaAUA06jNznZCdJKDNf2JslxFmQZqCHyO0ZoFxJTqtwRjbhhUN41Qxfc+tMHBcMbdmjj0KbZnAfBE\\\\\\\/A1XnYXFXG+J+yvZR7\\\\\\\/xcwv2xt779dRBS+HHFr5KaVlDK1ArIKJFS4TlizkeR6boGVhGwy5S4I8xym2mtffoaSoAPD9OSoP4A4MFSBNVhDQTSNNIwnMw4QDF7s1LwgNb1bAtscOagGX1CvGdkBdjpWaNPpqumSeBCN2z1eSJTIBnQZvnR0+rXIfnx\\\\\\\/px0qM+8jJTrgt\\\\\\\/SLMe5Z+t7j43DFhgiLJku4mqNoQ6GD9LKEQsik\\\\\\\/aZjf6GNNcUt23DmfusAnOgfFs9b+XIJlMB0uayV4rp3s9VCzw3VCSDlMwN6inQpYBVYDw2ZX\\\\\\\/5tJQYze9WM8Fv1k6ttM3Ouvj5rB+T0uGjEPt2epArR6xkNpzQ\\\\\\\/jSn8hNjr9GQb3mpXfDRQ6EaPPQJbQqYBLVsX8gi+sFIr5dSQu\\\\\\\/nKHQL4R2s\\\\\\\/QzOYHUOKObSV7BsvAo47UVSHAIxceLPQdYGpRfWeQcdJ7oFgW3+XXykwJLBJkYvY6SFFfVOW3zMrrCIqpTSg0AeM\\\\\\\/KiB+8IeJmJDY6Ua+2+TKdeyqyVrFLAeHmzzyMY5DJAKTydcGWLQUF5aQ7W3+Ac5PPtrEVhP7qgWzWvFkJUnSauHrfSC8UVwh471gPI\\\\\\\/L7uXeMYYf\\\\\\\/TM1+fsSQIY995ixyBIFpNgN\\\\\\\/4d6vDtU1XBkqJPtzMajCCsJoisLASPChFkebcdF4mPxwaZnhaEj5tWbqrNnAFcebHy5fgAnvSBqyZCNnPOqOWsCKEo8VJ8SQqtyLc6teyrifGUWvTyrnSwYDEchYY5AeWp74UHKjbquRVL97j1bY57UWWnz+WilF8nbsZ590IqL43MzFZxZJEAc2F0gbMW3t4EMWGO20t3CxouKjmdhwcCwxAlmDU2NWQmSSbHB1kItJV8b2RiCkUYZBXkF9VZ2Z4HROiwltwA54peap9eaqohBjjNPlSRin3EAmpHbJbIg0KwbC72Sr19PhiFpGQwWeZq+vT7GSjThgyCvJ0Ed0KId0WFoxmDOq65i44mxydDcZGEkiGDgPCqbKXHZbzOzlhg2hJXXuzu3t0EtWDtEhgqaLVe22dauSopKwzq9oNNHlvuV4bTPqLB+sBnVbfqTlOVAAffeayAdhIxg1vcLaii8keyAVf0+krezZHOOdKt8ztLv8ZUgoJHPRh2LARa67uphfe4HM9gdME2Gor9lztRGornKVZKYV\\\\\\\/lPvZ0odmxFF7\\\\\\\/3oNWkghgyi\\\\\\\/ZVl1j98KsKF4HCzyevuzdyGsjNF32xQwrOiBDg\\\\\\\/o+F9EtFkI6q3pbhn7gQzN49\\\\\\\/9\\\\\\\/VLmJ2MFM300aM\\\\\\\/GD5Ukd5SoM5ACQ6CISRSjpdHxi\\\\\\\/cNTYu2CKhddSAb7WOPDTPjG25o8337HkZV+8nwphqjkAXAl4AQ8Fcn618\\\\\\\/mLnWIWGPgeDE+CTl4zXNuPP0APVlZi4CRbLP6ISSpEsmlmtZkZ309BQ6EYynBQQn+EssdHiHVOQph+dew5fc9EJ+PBYJSx0+HWivWgvRNH3IrN+oVhv8fwo0nZSsOmeH8C6IbOE2MBVm5Y4CzC\\\\\\\/\\\\\\\/hdI6VITUC4axtBiPOLZhgFC4Z+4AR5Efz2LI+USLWUbbJIstvuHcc3kVtCwCmQF8P9i5xFFQ+gE3Z8\\\\\\\/mK65uP2IJmmjynKKG1OdXmpWIacKXztdedb5xE5DsYD0qBmKHDEpPTKc0qNFb5K8XovLDHl3WFZB5ntB22vEUKy6q\\\\\\\/80b8GCZ6xhnH08FwQCoVDAq2AWtKCl+spM5l5\\\\\\\/xQZ1MQ8PuILpCvs9MPO3oaNF6r\\\\\\\/mfXv8P+CoX1G9dUq9GA5UKg9Hk62bGkigSjYN8q\\\\\\\/w3seKi09wsIKFCGyz6utkuB4StxOnG+s7csBLDShoy27BBKoAai2NqO2+SqYIrISYagZizPvUCp+85jfM8ZHobvTisFO640yPsLNmKfwt4Cju9H5B5Cn4gAcu5T+vC1ixVdq9sMWehjWPXeAMxtj5V6N2ccRdhv5jxIpSSBkaBRMjTBWyq7gpNnJmgO5SedKI+djB9VlZFKt\\\\\\\/LCX9zzmNqLtt1GWlSdA1YEyrYGdomo2URt4oVb4T8UGaiBUQAKYv04XxMqH35YazR1SJ8wRVrcUQLSj98wQ8hOuFKEu0ocjUK7gv5WM2x1uMasdbPyrtTlR6IqYnEx9HvuY6nfjklp6koiF8pxGos1uqdamwBm0OA9FZLA9zlGXOhofjbRsVDpk\\\\\\\/bP\\\\\\\/Q8irOPzsoQ362XbhSmy+EdudKgkKqIAcx8UE1j0jvHCZp646LN2dcNYDF3sc+n5FnJunVTd4Qig6z\\\\\\\/IZqlHMqfh6N6o+KtdL9GQbr7OmWIwTs2gJNrKcN48q9BRbDm6kiDgX3SUQrXAkwdQKEFbF2J\\\\\\\/9rUYWHWX6H39L2OnqVIKlnuHc0kaDrH2cUuaCLl9U5qktRxeVKmFqXY7orLxg9LhX8d+cWHDpW65NhaW\\\\\\\/DqpqjP7X1TI\\\\\\\/ghqsOqa1JE+kZOoW+qDW\\\\\\\/OI+tpBI91lCKqeUeOc7wrd6GssaBdbCs1CUlYBAMX1xXS98+6H+s1dKiYfjsj0bsiJkRMnecIQzvnJBa5kVgYtDfypLpgAeAEO+FdJsgdBFovgOVABK2T0TENSQITe0PS2pCLYVXFQBA+6MJzJTzHWwn\\\\\\\/MEEUPxHlUXxY\\\\\\\/zl0w2+9jw9K5NYvXjXk2GjVqfIrfFtaey2xBUPAxyZJSOjb8BK55ytB47PEP82vcMQpEVIaaBakWE4\\\\\\\/IAAfcKRQMgFEfSkAnQ\\\\\\\/N4E5O5KWeF3S+blp8hj\\\\\\\/OwbPe\\\\\\\/kJvUyzVUhlXdBwmxFZkJ97S9ISfLQDrNSzzN4DHOcc1KIZJhC7lxmtU1GCeQKsuGRmtLYungO+XBG\\\\\\\/djnPICq+TFLwR0NMg3OTQyBM4IzM0HwCxgf\\\\\\\/wUb2fGb5PqEXDlUfO8aKXlowHcnrTEY3OT3qgGTrghBlGNJfnZBw35\\\\\\\/OESnXrV6PZJsgTUOwVrrmIUdCwKTSTkRh9I8DC3hZCRnrVWQOgCXkY1gc0V080vKkQPGBOK50HvVE2wuwKSDigqXUIjqiNQVlAnV9hppeqKHB7oh\\\\\\\/hE1\\\\\\\/k0LQuvfelHpeWdPGWVsPtb5Xwgw6k19uzR4566sg1UrMSaOpxNRxFHHtOtpvD3dXJYqxku1q1hGusyIuD9lryLaet4iMkQKYrii9e9EmHpRCbW0LCNncQw7N1ndsTCyp0S3jARCTjGUFDtq5oVVDi9eHPitP+e6ib\\\\\\\/8TdilJjv1MOy45xQA1VhkXp2GC7Ve7BuQOcrqYKt1pBda6ZDsZErWbRuOESYyFmBCEEbg+dnTTJ6bC9dBNUMCju2wg8oQxuw+pnHu0z+V1vgnN9NYluwzLmizFuvvMkmxiPi9+QaG+F6B\\\\\\\/TNOy8VT8bcbDKSU75BmZPMUazDTs3ccdmhlHnLFLkSEK0N7ZSxinwMDSk\\\\\\\/f+nAlNgl9E6YmBbdLfGsj6eX45+FiLlc7vmvDfvtHKfwvWu7z9pJ5485wt30JsCeZzIDYofw5dWwrltHnxnHpQBVrnQZcnyp+CZi9XGnGqbRRAegkqOyyq8mT2UOIVx0ZQUi8JD1ZS6YDP4XoaqlKwAp8iL4k42dmuh8e3sc9Oxe+NCbzHu7CxTb8VZ5krL3GbAiU\\\\\\\/fz6Az4Uv7FdUY+DvN3mgjUtvzWrvgH\\\\\\\/Yh5QHen8WFqwZ7Df0eQacdzLpsuYHKALqaGPLsVSJIWrRiT\\\\\\\/TF2JS1VoICbY5e5onyaYp8ZDQIpT+yFYnHlmU\\\\\\\/BB73yYneUgvcjbpeXdul\\\\\\\/lhGVepQqr\\\\\\\/NF7gH2DeqldLC0cDsvlieYY6zBHRmSpgOjk4pOvzGnRt1PN7K7QWq840bp1be0ymuUR96SKs9XKc2wSn9mP8YZsTcJpRa6wgsfjx5JivTRLszPFkS6d95HdmfcyvqwFbR9JTbt4zUvxIweoecjmvPGrn66v1c8AlPxP+KZIZrEMz6eWVdAGgjaWTaao0uH6Sh9C+NpxbBXH1aUnXwVVSvAB1PZQZ\\\\\\\/UU7hGjnpdhpyYtIAtinmSacMr5X8KVKJ9rcYTwU+H1QviCjjyO+yLSJUDUt+9HRpfGDwyV0rukRvy8otumZj+4rK9+QwQ6PGiJI9pAoj1yarg81IVfuVmos+9B8vNktNIZd0D2K7pnIRWy4Hh5rnUtv98K88EGLBGjvleQ7ArCrGZMQrA7xRMtw+\\\\\\\/toVDvPeqJYdGnjW0VjWYSA\\\\\\\/EBQafbwT06qZ1Ebk0LS4XcErL7LtAL7bL\\\\\\\/EL2bQdLwFl401yqvq6BF\\\\\\\/uRvYyPF4GvCkEikjkx3rTxUEkaD40nAf5R4X3\\\\\\\/cYNGhiHH2vyiZbQ8pgDUkOJJufUWOFFbBJs3Mq9CyDJLaHlDdNocnJKoeHylgZTkJS+tdTcglLi6cYCMeBRGOJ2gjXHVGEOT0UQGiz74v0i2mc8FCxwEGJFscS3hlkzIMtmOoypOUuxy67YEaaHu3mWuGPZD3+MUTonq1KXIq815IzNBHC8ulTgppoT94zLxW1+H8rowac2rbQSJ+Dt+qVUEM+Ips+8yr47jChbVU4cI45GZzYZawzJGM\\\\\\\/1PG4JTC7Yjdz5je0VbvcjRYUKkGys\\\\\\\/eJ6S7B1JSuYj\\\\\\\/7AWFlZ4vexwPTjoyFL\\\\\\\/gOMFkfXiqQjxO4tXzMKTCNiPn4hP4V\\\\\\\/uQ+r53Kswtoln\\\\\\\/9a86RFNw3+vU+Q00FrIDGDPcVp+QQmEslCoSQ5fT5vayB1nWfAiCnKvSAC71PXoFqTYyyM0YDUicc0GinX34etrLAdIBJijIv5Ia6NbB9n4OrEmHqXQF+XCT4Oaj+4YJMgHI8Jarfu7PdZkBtF3TEHoKY37zLlYVv98f6mQsquXRDt2o74XJCXB73ie7mI2o+vDHFhUbLsfhir9VzVqbbaDdXMPDoVayG5SQXazc1VQHZ\\\\\\\/wOy8R5Uawe\\\\\\\/toKKhFDGBExNYQ0Kao3p67CzTBU6xy545xYo4pXtrkasaP302ee65wvoww5h1gSH1G53wAal+oSB4qdbaRzh\\\\\\\/JqLcXCl6147wHmvCIgVodaTOMjp6Zf6g3P9JMTEjaNG1+YFmQ16abCwAvxQoK2hEdK6kGDkHYUG08mL2+Ex6Tu\\\\\\\/iQUSK0f\\\\\\\/QZaKms1c+zTYjxPZjeFnqoGdJjHHXWX6Fqo\\\\\\\/ap8vaH2szEF0U4\\\\\\\/aG5fyaZXuyYoPYo6zcB4niy4GlxJt+LYmpSongTCTxyhVXlxXiy0riW7CXbKNWp+W+ZvgqN4saJM7MxlzjU5zg5pKb9WQXaO8JVS4tRFC8EexEFaqqEkVgKld\\\\\\\/BgedIp86n3446Gko3XJSyn2oBS7rdfLGg8WLOxpsQCW+gT0gx+MUKKKHZiDCekqTG674o4Da9YMP7CPgMpAVnDxEzUvnoFDjGlspazSYxtlFWwF4WBq28sdbHaLmnzi2wqaIsMZ10YHLE50mzmC3IKM9hSjf2bw4TQzl+B\\\\\\\/ajhUWTZ2XmfhavhIuiFolyYpakJ7w+twLwpNetSyLlLjKy26jwa5wQoUby6\\\\\\\/pG9AjHJhzscr5kJhm5HUC0Ff4BJTMDwTpqyuTj2aQqS2dRlR1SdVCEZZOlw6LnqTiN01M\\\\\\\/5x4F7ZXhv9YkvIHVK9iC472CkL\\\\\\\/e95s6AcQKoV2ieXg3wc3WtWBdAtG3CyamAM\\\\\\\/BOGhCMMxT3ni1TmQCgWm\\\\\\\/yxUZkkQ\\\\\\\/NJ64nLMC9NkBlJnmVKcJ1wv2gG2m7iLIQV+i9zBCid2QoV9Dg95FcQl9FzA\\\\\\\/Yqz0ST4AEGtW7fCA3xTSxiu6Jmp9on0E1KoaetPPz+Uzfj1EP6AGef5utiNI+JpF9XmyNmekQ2GlKIAgRimchI3zs3uZDoBXH6qSfbU70z1nWexU\\\\\\\/fXVO\\\\\\\/iY0Qxj1zedD0j7LQT2\\\\\\\/dJKfgrCUKCoJq8lBdDWfEqVUurgwPxakn6wYlS2MYIyQ6+wyGFY446JnmlfSB1c2d0xgLAUnnvY5alVYlyOe+2HyYsHL1m\\\\\\\/S0HiI4l0+D7nYgVY07JU3ND5gcmS2t65mACjNEfeaUr8wSgCkVqWo54k3KNgQegg1s\\\\\\\/OKslkMyoBkxjMxq+S4Fgk9pzE8DkZMv9KqfDp8JDs6v1F+p6ue2vSP\\\\\\\/ZfRdsXh5XFHoqOJQ5ujd1rLiW7eE9NCtTUbjURWv\\\\\\\/\\\\\\\/36lSi3YgJjRAJtP32qLVafTwxaGETgGQ\\\\\\\/poni9gDbiS5El1w76g7lb1dJQ85uxZwBvlddnKCprLU+pIsDluESMUnFaYiM5IS8eMbrSTSifNSUS3Y+2vj3Z8mraXgo8dgqRIfYZJ5DZaROBqtZmM0pUwJn2gCEGs45ATH+N8HPGOr9JLBtB8WO2fuvZUyET++B4Qu1+nfunDEyTmQqd3aa8T0m91jP8kNQnuTq+wh0X6F2sSLV2w0Uovr6UKD2a9UCbZBOpJXnoBfrHCPNSp2kEYZaHm0WRHSrk2n2UGD1820nnCU+ySBGN909HUHYXU571h0hPWunF110lZtqx+IssR4B29efnsyG5Xyf8NdoaIlNL5xboypl1YlmgTMxPWgi73hLgATIZc5nnkCnEBDZyNLnVqey9+YExdqO+birGW9kBRzVfcORro9nfHjMBR44yliM57+Cl6dZ3MtSvs16ZUkuEFw\\\\\\\/mNaLFoTHOUir4bp8RPpIogeicttkibYVi5uRFfs6XAp\\\\\\\/aibMz6uq5pS\\\\\\\/t6nymjPNlQjyY9Vjj02A2kMlTTQxiuM+KHdL+BjRUMXJzgxKE+2YmkI\\\\\\\/Q9vsCBXCuZRhFWQYizkPO3JhSkqL5crLSIBgoKNbpPg1GSrOGenOzp9+vOTCpCWfln99SjqV3foccsAyO6UtkX58yDwnHzSrpleAouZX6nBgyVsfcVecvyERtrc+NGH3KIafYwGaNX8qBvs8HWu7wFEyhPo6z4yoLA9eq28iU9mkRiz71pMifZJjqvu7TriHTycIxujVqB+xO6UvjWGV6tummvAipO0vUHcc9KmbElGXJ3uEn3fkbD\\\\\\\/H4Ou6SbLex97JZdpfama5Mcfde+xOSEuAKK4qSu9vBPv8IQ1SPQT7X6cz\\\\\\\/sX7RbmhLWus5wUHRSGynWmUgwYt5vhT1h08RrQ71NplS9Lci+aEOe8PdfXBBIWpRl\\\\\\\/+RXwDDnmVEygEcyL0yF02w0HhVrBmmnGtWMDNT8UOJSjRh6KbWublSiJ5\\\\\\\/VqO1TI+czVhehtV\\\\\\\/CCmFb6d0edP0xo8OKaf2WBfSjdltWU8Y+z2qaKUvQr0hATRlSUDEDPQoshI2QeA+A1R+mnt\\\\\\\/yH2jKgHXI4\\\\\\\/btUbHTi2lrtu3bOJssXedFBvOiO1e0H3WA\\\\\\\/RyxpfejQmIS8RFSmkd44oJ65c7\\\\\\\/i4DCtDtIzAUrBX3vzkc\\\\\\\/yq1I39KUWCn7w5ZVZTNUhW9vawAZzg9iWfI4kkvPB6pk8nN+uP15xxVrvv9OUL4D9x1M8NK3NQk64PfR0sNZ71Z6AmdQ8v2mB\\\\\\\/7nAammsXakZnLBsr5raDoIQyvVef2qKpVAY1MSG60LXRdkUVJIQx0Bha6svUcp3\\\\\\\/Ip9UWtCAZdYFSwHsNZOu9Hn7u7ZjrU3XG5v9z4TRSqWDaDh2DX5RY46vAoQntom7ys+aco9PZ0UEDPee6JQFYEkMwsmx\\\\\\\/ZDOd2Xh6syvVQgld1n92q5YHqeca47I2PqYUgi5Fl9m69ARAc5PHTeIWRIqKHIqxzpjt+\\\\\\\/GE9LKOJVoNRmcC8RXlxXsJWhFuUHCm3JDIkvSU3CfqmsphoQVkwnWuZm7J9OYggpk6VmEpvQOlS4s3HrCO\\\\\\\/K5jvQyCUTXKpjVZn4n9QA0U0SMdPFRJxEo4ZfGxwuu6OYlhu\\\\\\\/3ZJnoNnepw97d60WgXbC4c2B\\\\\\\/rIsLKduABOIXrdE4nmc9ttVrT+H2AK4pa74Mz9m81TnmscB+DY\\\\\\\/mkA2ZXGllC59N2xlrYjEZ9bSYtoxB\\\\\\\/Qh4p4OeitxrHU2KpjvCOAEgb0yITOgJMua8GEsW7wcy5\\\\\\\/BAfiex0BoPG+s4W\\\\\\\/ZGWd1F7EGzStPx73S3uvJJQkZCGFKRJ+k\\\\\\\/1oiqcstiPEpWFpgYh+ybSIpAeypxd7ipNA1y2ySCaMMpTBVxkJ7AcjbAnq8l1yAbuNowQi6ogsLtO+QGmybNOeO1FEraH\\\\\\\/BC7d0gV2nWg4PNa76qYmF\\\\\\\/Q+PXRpmsa2uXjnHdGGznP\\\\\\\/djYxSRb\\\\\\\/y8xUWYTGjI4XNtjtOBwMlHp9tckeaHF72YUVXi164pV+Xas8VKD+3dg2cczhQX8xrPaLivZaWm0nBcV3hofd26H0v8DdHVhof9YYIiJ3kx3VRk+\\\\\\\/v+b13pPUpmeUIWF6DK83X89JmT7KFpQBQ9bnmI2Lvi\\\\\\\/ERV5bFx0F5Aho1r5U0zkL\\\\\\\/P3+wa63t5v0JPyxo5aDJcOH3reRN1onZ\\\\\\\/XcNCG+qDE1syudxviMAo8qiMEKJT8EdaxM4N4sZKi3LI9p38cXW4SnHbYi7oUVc4Il4P8cToP8Vas+bHA7xmQi0yrwytD7fz60NeETSF9jJzPCj9gCYarnYo57f+IUfY1CqwbkE0GvgY2crY1bkCcGdT1TBhfBBPOLp293EY2GjEyl867E5h0wF3gF6paGTzoT+9fCGJQtNfbsqij8ziQXEPlfrtJ\\\\\\\/fXzsm2GJR\\\\\\\/GJEHdTEn+2S9eGDz5FYMQX3rApDu564I2M2BzeTQnhBh6N9IcwscNmWV8wQoV8ym1pwbsmQaFk0PAVxJJLOvvTqMcfUFzwgVDi2hw4\\\\\\\/JVHapDPyuBDksi7dGeMiXy6NWJxBpQQIVoxH82Z26rVn7jrWF6JNw3MYkOeOQoI+qfyGEKEGt3RHmvQ6HcDvniXJsTgjsPxccG31TOQEjQrDs04QrM+Y5SqFRvmoAPO7MrkCPBsWyWuhe7nZckR9Zc6jSlq8umkRWPkORwxwgXNnhdHsuxcMcR4V2rWlR6x5fNA25BG4NmI\\\\\\\/2gzILtjJP3pzNS8AEwUWgMvYyYA4YjPJDQEGiRiA5LDNq+IXfRAkipuMxuVtSC3bKrNJp5wh2tgjISVqmKG4F6NFTk09mv7XK2OHjqtHvvFAPecdg6aeUmOBy+f+8c++PWhGycidAlqezxcR2aUPnPawe7fSyzUuXV56GtKsnQpSPIcb+s3pymEtaT7U9um71JSqejzelFd3YvpaZKROQwJfSXEoVtTYcimX\\\\\\\/RCso1FKHSzm0BGYL6V7aOVu0AGA4fxpKk5p9qPuKo40oeckhV3nnTtkm180\\\\\\\/QZucyAiyD0R\\\\\\\/eLjazwCSzWZ+OfhUQwCGygIWOUFob7pm0fWQgwBOcPPQl9F793L37DD\\\\\\\/r5Bc\\\\\\\/JXvLIS18bPNUlbUymJxWJcdcbwRnla7g1RVNE+FrpeFwqan+Urj2l+ZBrzO\\\\\\\/exWO+Zsh6PNh\\\\\\\/6DvgIriuu023cHsxIXjr3fpuF1UgDRWfxiSjwXodayHjv0ow2FJPSqC45LMegS0DvdVHDe+isVVJBzzyyJfWnV370aN8z\\\\\\\/D5\\\\\\\/NuYrnq+SomP9yiBd1OE2cnU0pSa4m4FmgY5aFE+X\\\\\\\/afkxPIZNk2LrzfSV6BTqlUfSmlmjCDX4apYUC4Lb6O2aRAgkxT1BquPn0OffllmQR3Iwz9WmeadDR2s3eVh3yEY5TCfYrRUHMgfE5VW+zI3N7roxzTdDij7k4ocUXMTy5nlzQ39DBIdoY3Tgq1\\\\\\\/SUyGdNsdcLiPFjkhqW8TpBoTuqTdys4jKPy5\\\\\\\/HEd0V7u8EDdxcMi0srf6GLXHH6OdQQyAXzBW8KdSu5yhGcCytTKkLAIjhDwOZfRGZxABuyvcGwX5W9x0s0JmZmqsWMsnR8n1GDjEN2saH\\\\\\\/U6waxpKLdAmGY7p9fAhPyEnxhEqVw0vFROLg9PoLQfMxLyLew8Je\\\\\\\/3pJMEt7A+k8RZcK5cpoKjVlYUEvpucCB1shu8cvpViym1Yj3W+RjMC0H4mXVV+q\\\\\\\/mTtxtqxaI\\\\\\\/HlgZa9boGqq3wvC1RXzRpkOcbchfKbe90GNJhYxycfNdQIlI+c1trDBZb4REdBNgEspGut2TWoXh0pKzuc\\\\\\\/dRi9y3RAsKMuJBhUBHN2BPZsAC6qpmDhB80h7nnvKyN1KH\\\\\\\/cKmgfhM26masMW\\\\\\\/A9p7m4wLhVO\\\\\\\/JUl+1VKBQ9ljVgFOlFJx7+Rug9rBTuKVVBIT7MJFxniWEOEqy2LmNNjA1HRJ95QQAtgtze1sf32\\\\\\\/g+qTvvBsxrgIkf6NYvyPmehmE75kTKQliAEgOAQIywCa\\\\\\\/QxtVehOsEPbxETqFqyf9QNQKWp5mD5YFcOnFGtw1U0FcMDbrDov1iuiKZQJ6F5ToNge3tKOdt2gvybuUw+ypruQwpqRvOfbkw5F0jtvw0bqWmsiDElj3DKVdZU+ptIzVTV3YGQAAkYBavBXG8a5j4qz\\\\\\\/gy2f7ya1QYhodScXGRZyb94fcl0TH5gQ6pXTIvlY01eQZfJ+yQ1AwvrUIElwFUGZtvdKFjf13nVQH+KKsQL3WVMWVDiS76m8q9U72pkzFk2dZDnE8oiQLkN2Dx2Nvwv5imlDusYLr7wgCPd5lZ14kQpI8j\\\\\\\/ROQWrnM6zCTEi8CKqrbFBJD62gxnYhqemRu7mxi6\\\\\\\/O17WFgxoBtB\\\\\\\/FX4oWnd7HSyxqnEJnJ+4QSL1NXMluAC8nOS\\\\\\\/wFU+Ulf7lb4d3+tUoBvDJiHCvi0vlzxTr4xENt932hfLV+qX+kaJefF7Oobztum7Elup1xzKP5viT069wc+gm+xvgJtKInJlZQbl1RHI7ZLSPk4NAupCo9N\\\\\\\/dwTo2\\\\\\\/E1RFHQFPOxsmxJz3PruFXXmB+fLtv0Ao+WRWoukp0eeibv6hXDxm\\\\\\\/USsFBV7vTG5ypbKLM144lyEtLapTgCgrbs2MaCbzkIAqfuRklUBkgaoXRzPcv60KMa2LbZPWNJGtnEV1T+WiGE3eMCEzMic+3cnjb1TL0iTzdHvKMK3vobsCBv9B\\\\\\\/l8S62CMpfBRMCljFcGmrRViV66ka2XcURl9tcgHLmtTvCfGR4E5K40XVvRT+0r26cLC6ELgA1\\\\\\\/nCpMmUlh0DBpKl6pC\\\\\\\/4hIeQ6HIuZJy6\\\\\\\/Amtet\\\\\\\/nwHfBClydBg0kA+\\\\\\\/0fZE99TEy8YNa0ZsL+OLTU1\\\\\\\/Uivh8JzOMdMuGi9EpMFwU2V+mj+MBg\\\\\\\/ai9tl9TRFgnhpjtSQNC065HIbT+Ps7frs\\\\\\\/yo2m\\\\\\\/5mjfNovxQBvgQrU13HYrUSqrKT9WJEzmAZWlsW9CMUhJddVkogjxPeE83wUb\\\\\\\/g15DTKCUAUwqROFZTrZG+B+oWTfUjGkM2HXNKJYT+PdRbyvOWhizELSp6yaCx1KGnxT198ntXPFOqRJYg1w0GnGaVnsZeifxD3zuPd84+Ma+tphJJHGgHbyB24RfQBedlQocymbh0eK16yLRtjMqwGJlpSEKFSLwJ0\\\\\\\/5jTiVAwX7EqFytKUsJfj28vSQjmRLYB\\\\\\\/wcC1Yi00ka7uvXlLP+wMC7qCXh6JxNQ9s6xaA5iPdTcUmWx1TXNqWaLFynHFx5IlJmyolBOpuLwbWTU6fDg9Fh68NBYNRLDsR3R0beE7LczVjzqa5vH1IGEQ1kEYCecfc7jGO8RM2ixkadYb6CxPdm2nlyfoK0DAOGLWKs1Jyro9HkZiMDabDXdMBm2B5tUE21az6ezI9Iwm0caW26Ii4hQt4Kq3Xq2JeVKTpTqgnxyGwRs61x\\\\\\\/TFbp85JSwyOvYT7L36nrIvBmBDeQey\\\\\\\/aZ7rlFR0v3UP75VxCPKtEnCWEeZnqRlEyGod5x9idDS22InkpehKawau0adtfessJG8YUUCYXBTYiVhbZ9scFl+yqCvh6YgvoprRV1QCLLmdZaSOQ3o3mDx1IjBAn4Hy7J1kunXHSMAUuV7Ep26WuKGJCaaNF+1kYel+2\\\\\\\/h6Dikvl1Qov2\\\\\\\/0XCIQHB\\\\\\\/Ccwu0a3Em8qpOQC0CwPz6ylitqQPsjopRYYDl5pLhm+Sj\\\\\\\/T5h1xhApJlsL9EN6mB5HZ8W0u3heQsz7FlENsUj9HIHcgk6vxbYRBXOFvNZnNd6R1BmWzL1Am9733\\\\\\\/ov69HOJlwLjTGr+WJIUQXgsvmNSZ\\\\\\\/QHn8obmXxYRI2uKycXdmwLBfSG9QQWBqpEhcrd37TMw4c8e3RsoJbSYiQxZ0OgnbOVjrO9z4DX18FQju+LoAL9\\\\\\\/AsO9WrMYfzlOITKYxhSiTS6foCKkYwZVGe4RwohbpOb9G9Bw6bCu3TibQMDtHD1Z0jhQfZzgG7pDSFNO1\\\\\\\/2ESTmJqnzEMDYShljGvuIlwYqckpbcxswrE6l\\\\\\\/knSiGCVtYKZyzTVcNU5qzEo0TJUv4yIc6OlVY2LyMZxbQ4yYBn\\\\\\\/7SuAFOzSSahx65PXmQq\\\\\\\/Gkf\\\\\\\/Gxe\\\\\\\/04FiJwi50OLHIEJgdf\\\\\\\/60uHPQbW\\\\\\\/D2ONBWWYjximBX\\\\\\\/Co5GDJHWIFBexcJwlf7O0koBa\\\\\\\/3qpqTyPcRkakoXJZAUiXYDXbNbyp1imy7TIny+MHfD00IMp8XeAQxsa8s7Ea\\\\\\\/pIzbxuP281rLUeMl1DKNtSNBim9JzA0lNPqIceMgrCn7oB6K63sym0chtAMYhA6MxRu24yKkCgFHpkBBuKKwxEMdILBp0ZtwZk67VkZHy70xWe8Fx2LkbDKcOFGUj3Z4sHg6H9K+prF1FTRhXc4zRxOjzNRKXy1Kk3boEiSVAcWjpqtAg1C7\\\\\\\/bh7frBy9Wqs2kp5bZi4WC4lduEtupaom5a5nHxXk+I6tFeeDMixrSdv9mqZ4pK2hKTeePrgZAY88KdnfsOnSVeDapDV8AafbUk\\\\\\\/e29A4E8ftJLTYB7JcW1Tta+ZP1zSzSBVhGDLBexcX6sD\\\\\\\/hFlhdcqhTxaXcbzMsDTzPipSF8oXbpXr1KsYnrnoo1t853mYaO7l2VVTeIRKGff9B\\\\\\\/MtfTYLKeWcWXZmhJYgr9PPE7gKf2gw9KQwGmiaITrk20u+0LGHws8zP46THhexqyj\\\\\\\/S3iqK9TJ8P0Ccf5mfT\\\\\\\/cG69azRc6dV0CjwRq4fdW3Be+NQktRv8fD7Y9lqloh5wxVZaAvg7LytTe73b3c66CJ0UNpH88OSEX7fg+6QAAOWfHKdJ6BWUXAPZfLnXhTMj6swUJYHtjDFW36fshVcoG2NHBMLSSAGVWqUCpmJfToP9k5VKb8hB3fMkqj+WPjMAibiH32O6nTWpon18qoub7Fl\\\\\\\/3eoKuTCag3LO5VMoycv1MCbLpvhnssIAhkoyhC2CNgra1m4HOU+\\\\\\\/W04qjyTb8V4uba5EQxU52x1GS794f\\\\\\\/5yhNfHToRz7dTcr4JmQ7y\\\\\\\/q6\\\\\\\/+xuECRN\\\\\\\/J4onjddlsC7w+23Z172nYJ\\\\\\\/l1VE0LnAW7tje\\\\\\\/MkRuLMlSULxtdXd80i3RQYFvfsIKRa\\\\\\\/B8n7YKhjA4dBSCAN\\\\\\\/905Tl19IPvLbmUzOO6FSNKPG54p1SZ8fkHDxH1Jq8OU\\\\\\\/bBm4TnqoPzYuomPM0+aSFgLMHg54d1HmcXWHR0zxFaHrY76kJa9U7n\\\\\\\/WAfk7JeDJipDQjhWUnNGB7hrfzibjOmc3jBl+x9jyIBlLvaeIwUoc8LmQlnLTyEzfczriQl42xf70T4qRbuGJ9R8FqNhIcoUDC0SM2rViQoh404BEfSkpP4WbvX4q99JhZAKNv3\\\\\\\/MBi0xaNH\\\\\\\/rfHgitcTdj85F1Zj0sKZU9vMpxo7c9BohBB1AwIeggKfsokeMgxb57E8pIN8G2OcsMI4GVRlBfyXK5SP01Z9lHyZNeVIufi1nZdfaXj00FQeSwWgawzAbuxIyL0oyLdMtS1Rx22Ht2yq5mLoP7Zc\\\\\\\/qsp6OdjjKr56kPlNHoN2VeDy1Glqwf9rSDIREff\\\\\\\/ViWJ6gbzfKWsXAs5KaUtSq4NJFn6uLKEyiojepBNk03QPR1Yb1QTODr4bikgE8VUlKEGLLD6oVsEexzew7xDZ\\\\\\\/+a31FIoVnEEDgXXwSn6Nmq79YFWbIQEVHzEdJUAyNoqxEqLyS3IXzvELvAzNu1z6UUN8DPCfFUxkZhO4TczQYTkdvYRk12fJ5eqc5G5S0qMJeOoAnfENX2LOLHGIQrUiparVh3wuvRYlb\\\\\\\/2xbqw\\\\\\\/Ck\\\\\\\/4S8AlmhtPY8+Ezfg07tuE4Liay3RdUVk\\\\\\\/FNvEQtKCcUNDEUORvJuoWAZ3f+Lx5\\\\\\\/jFFwW9pzU8xW5NF0SjuwtPVLThEZiHjB7FUtX58Xcj9b+PM+uVrOpKOn\\\\\\\/991E6a4jpuDRlo\\\\\\\/vuXEB9RlVWm4Rs3BoVufoLoCEjQSPLipD5ClvQwvmT7jqsXf0EijVgcn+B6jJR2jMgTXU4EFe9fIHUUDZSY\\\\\\\/JT1jgzoEB3ccyr9Oy2G\\\\\\\/+XbxAhlBwAQcnTE1SAPsYrh\\\\\\\/sOopfaY1FGEMrd896cxwW3FPaw7zF1ePQqgTHtxR\\\\\\\/DMWcU0jqNIxb8xgns0rS\\\\\\\/9gJsHWfLmzcxzXAmc4dlzCoaRSzMxf8+DOfPAcOIs+goaTnX1nfo4Sjh3NDTJwc+8ECn1umseNvE1su3sdSh3KyQSWS++0kSfHBvsTcZLwcggrTVWFYUbRkQKoRZljeTJCszRnpWqMe\\\\\\\/gWDWyNTUK6Fn6vNYQOCF3qja9gu+9JwcdsLGJ3JnIJZJhbZgc2Bg+MjTbyXSXyVhOj2iSjsKFVY5x528AMWHgFVi6dsMKJHNFMF+sP+p6LZO2NWmAu67BcjxCg80EtaY4Dw34KfZaZmm3kqh3f8pUAmPRRH+OHq1oHt0TcfQ0jfsLgkOhAApk5SHEs4skyF9iG\\\\\\\/LM30tfYKoygiI75DMzIWHB5qt3j6q0tJmRkzlvIsg+opP0k0h\\\\\\\/doZfFpcqqJKzL7mPPmzoHECH7pcoAH0YkafCWbKmL+8x3IPQ73x7udMXmXOLA9vCsFrZjAcltUmur\\\\\\\/gIOVcM1kzTog3L\\\\\\\/KoNHmZlxOJ6CIYceWNj\\\\\\\/EsTNPIOU4sExlEwMacEq2PMOkjiHLTTK3tYdGv3GokOLaEcNZrFBGvgg0rnDruscvMRVQn8wQirkvMiR8LL7l90YTal\\\\\\\/DdJCMvdgY\\\\\\\/b6p+l7GRkyGB2QticzNRQ8Lc6NA2fkVioYZlWImV0P\\\\\\\/QwvvkxjWm8iYxvjlNF+Aj0i\\\\\\\/wFCokPflEKYjCQGXW1l+zdk2ShG0SXyADc3bVNxqM9nfijWfRgdozvn9NSxCcBod7wxb3PK3UAdb8R0i5V9ibkIZHmmCB0jUUpu3HqXLLxYZjGFffFRIWIhUPr5Hoyl94ix06joibCPymj+6W74Pl7Nzlbkv2qeveA1G2yINnbs25wNPuRWVjTPgNBrklf4Uy5qu11kErOjsD0ZwHnOYIhOrckztGzHthJ8\\\\\\\/2A\\\\\\\/+SSPezkDLgiDmOfP0XKabPEecN6n52zJjvJ0dedoUv3bCAeQ9MQxWU9Qryxojji3Uo+s5coco\\\\\\\/wVGIm1+WOOyMOVt6P0ebJN9o4Cb+nA6CFYtwVCCksyitpXYSvn59M4siBCfOnZl\\\\\\\/0OsLpKYcd7STug7ibeiNdQtDwtxdntsieoV075IZJjPAxk9T1d80oTA9l8xRwIiyjc\\\\\\\/RzHRuV4p5eXMQ7zfk6MiyxxG0b0tYbN1ricEE\\\\\\\/aiKAjbeUtWNVoF3sTs3AT3r4lg1nx+XGCiwzievmnFUfefOAAJ2VCJeoG+EIu1DEbnyMEZoCREY4h8lQOhgLz2ivOuWCd4SFlhpsDouZZPKsTxzbqQoT2b5VnKZTpZWI4uBktC+\\\\\\\/CEyD634N5Srls8t+juEtAvOU3rr9md5UnF1Op79tipVak9uzNdWAtKicZueF+Erp8wib2SHcS5TU9SNC+ex+NK4rJ9pUS3lBaE9Y6ljybnMpwO+E\\\\\\\/4d9xxJNntMTIM26A+QFluiDeUw4qA2YlYVEMO8UZwV\\\\\\\/djXN7kRnAceaOw1Gquad+jCuwP\\\\\\\/8gfc5i3QLXGk7YqPkh6Bguv2SxvpbMnPMfoFGhboKyM5GymtmywfEPtEA9AqOSe2Ob8aYVwyfPafv5Il+kzhU+rIex6ytmCeEwDH9s30vzJc3Thj0sbUs7DKAx4fcviX8n+ApIgavvDXBQtxZJeWm3UiFoX7EaekC24l8bSgpOX9MNaKhkjU8j4tnYEKGCH+N7tTrZeJxxXSR2buOI7EYl8EybAYTL5Pj4SJ9iuMpYBMjeScb7odYGybuk+hQ0\\\\\\\/XFboFc6bJMZdeYu8pzPxMxawCykxtH+u8nY1IxI4Y+KiL5Bk5FGgtrxQ0C1ILYxL+Tk4\\\\\\\/iT8UDVmp50cxRZyASXK\\\\\\\/1bJ2EY\\\\\\\/tDl587eE2FB1Tear0k2SFWSr0lgJiw34EzqhoeVXbIMHjV+IuCIjls3xsN2wiMAonYrE8cJxn4Ob5qiV7kFf66KFXqT9L3cIBUx5EAPyqiLPnqxVQEBta3ugGqZHqLiuCwl3XdjPNtKND1+xhKa0M85wZXE5v\\\\\\\/fo+3DvGn\\\\\\\/zNHn85TN4qx0\\\\\\\/Enpz7S+soS5Wk4n7yR5JCARwFbQDiE7nPY+b7CnTo9EWzlyJ9\\\\\\\/V7wkrkODUplIAHEc7jf74CXpmkoZEsG8UMjy9HRD9XK3GGSFD12QyJl93JJ4n6Xagi82R55NjV\\\\\\\/sMgzNIVCnYd1ji09\\\\\\\/x3+m3IaeYZMtchED2BYOgQcllhCxrXaTbwSzGpJFL68zKSItdrIFA+0H28yAVcXjbcS8a2FDbXsMAxaFr2TakgfUNC\\\\\\\/N5VQ6lf6z9pQUrjZI2ryuRkFHUY9iz8+P1O+HVHty94rqfTICVgFeZACHKnNAdVPuXn4Id0taZ5AMrn73\\\\\\\/rw+cu+KOyuN1XjNKC4eUDdolgYw8nIdS1XqcTtAwJpxEY0i1S5+jFIAHGqNXhbZYvKgak6hvC9cKSmIyIP8ujHEIJpt8diTp4OGzjJFHwVkKs+zt7bJonwO04dIvmCI9\\\\\\\/KBS1FuDUIelohSj5U5yMsjmkR8jiPOnrPyDtlE5MHlLw\\\\\\\/ktKWm7lS7YvNedBVRfS2C2g2Q2Qay0j4dN+ft2ogM9n33hnl3XHJYo4TwwU\\\\\\\/v0enLTTbsr4mgKnsYCHPx2eKJKFs7vIcoBfsqq\\\\\\\/iKXZOmZB6j4J0mLDFXuTkV\\\\\\\/RihGcNKC8+\\\\\\\/AhmAboY6bFO+JS2rja47nlJMUbdJ3LRhTvJtLCRX4dUARm74jO4YTgUyfBLB1IqV0VcPSzSXdhswdCdVJYGx5\\\\\\\/4K6P01qNRZLibMN5qsy106byiPhz7gfUZT2k+\\\\\\\/tCjnLQs1gCZ3+Dr1abef5LIx3iHFL3bO3J1G8v6hUy\\\\\\\/jWYAHVlTZKbxayXGv7mBzgnYoiuMhQk9r39BLBx2zMHULajaJB9g1UYOvpv6uRBeKyf3pXFoiOK54D62SWD4GjQ75AO5GTGRYp5DHtnUM5JbkL6W\\\\\\\/jPD9EHtgXswCVy6YxsjikqbU9ETGcGKlTwfKtDoXC2AIDYz\\\\\\\/uquo1in0sCSNAk7vLRTAEJ2WvIN9K+rsLZIO+3NdI++COjevZQ98WH7qjWemy+5kRGSMlSNG0fK8u5wi88U5E18itkBApuRdv8iI7o7zmLpbPyIpQve5LKCs3aRp6ZWnOt6aEYTx++w1\\\\\\\/tlz0zZswWGsJx8JfG8hHns4Ojm5GS3msK2NXxHFaCymJ5a7n82xABzlK2YUap7FPJhBffNfLgTTOb+prMDWbfUJtz3fR9Izj4YnErcrmuPYAp2522UptiQ+8WnDARq7P9hlQb\\\\\\\/pu97tIePYn2eKQIsiWB9b4WyjVIiwjYIu4reKizwWsQasZ3QudM8bAD+UnjpEH\\\\\\\/qPh\\\\\\\/Nn7hkXh3PcoQ+3hIaSg4HwnQ74A\\\\\\\/L9VDa9oCouIi0yxkrbHljF+PABAakE6facDtxPNQ6G15P8Uud1h0oXRQndekjAYrpUaVSUluMNnKh7sTYAYKTt6UUu+2CsVhV62f0x+w0Ity0zdimk3rVnzPf8TJNEQZjORSU84bdz03GOHJRWnj\\\\\\\/2\\\\\\\/sJl6+e\\\\\\\/r7Mtxchdvqi5jfpinlur8+Z5f2WzaHSj\\\\\\\/XgutSB1Fjb6+FRVfwJkVpDUel3Llp9yr8Epd19qIoX6\\\\\\\/xrTdEHs5OPyDRsu8eI5z2qluWVCJCgncGU82bdEXr+zPFZ0tVNRxi5QL8xB\\\\\\\/LXBH8PKyjc1HRM5FpHFs+CBAO+Ky6wTfStp9TZNkt5ULmOPLmZmmUMVHxN7LO4p40VU\\\\\\\/Xa9ghgvx2XIf5i3Yp3oQmGCSo72C+f0e0xX6rPHBto2V3TaWKJyTBJBVsbWMQfyRdS63yRdl\\\\\\\/2\\\\\\\/1N0eNSAMBru66KPQI9EMqb1o4sppYd0fKYQfGINCgIsXs\\\\\\\/XpWn6D55FBwyqGFux0hfAbSn8y+ZlXCO1rnemEse092ClGrZSrSkgFgsDwrRpyGJII2esI3aqwgJ8s2OpbCup5+vHvhu\\\u0022,\\\u0022iv\\\u0022:\\\u0022vduq2Bip8GkjxivQbl4GvQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225d3308be3232e501c38a17e798cf202b2e631edad5df8ff30995e42796179ac4\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022n9EtOTTPBuUtZKWeWU+wHAVBUilR688tJwNioUpFoMxKqmflsO5cy7oa2A1KHijtYyE+A+h\\\\\\\/Uu3HUqdjV48pv0N7hr4+9hsPrV3nNeyCgo++ytCNeJ8Uu6Z1NGeHuqTui6zfwpVoxyY6eAzXsRXeTzEbWGklAdOQw1hLfpMCTSbQTP9w39tHTNOV1zW550KYmgvPlmps0QZ+12bw+gAcDaVavDZvFawhcfdsvk92\\\\\\\/ZD\\\\\\\/JW2nKeOCBGZjFJHN5aEo1Gp0ai7iKTUVwecRwl\\\\\\\/\\\\\\\/VFO8\\\\\\\/Ebc5a6LT6AWL6P\\\\\\\/Qg4yTnoXE3eNXy4KpfskR7Y7z9eUf6V4ye4Myi7JjstGy24R\\\\\\\/rQ0xY\\\\\\\/h\\\\\\\/mxEDMhoL3azGuj5y5j8XafEh\\\\\\\/4n8a68bSuo1P\\\\\\\/\\\\\\\/ANOH5ek3ROBXKLLnAxde+4D2q85ktZTtHdKFXVI9g+CzViiPT47uRdo+mjSggNx\\\\\\\/u2MPE6LZrJ4lFU8Efuk1Bq6180O7EvnY58EsC3T6ZWVGPBhzA6Q4waFUcCAjC5NZR5ol7I5\\\\\\\/NKyrM9QVHcsvvPR3mwanwABCdD1TDjst5zuxlKVtSdeJcbcZuDfJ0w5+0JuMP74m3dD444UDOjjq33pQCXSOmyB+vaGdNoSysWm0BhNd\\\\\\\/0E+eYn68iR8XC2VdszMnzmTZWt+h09wdqbIsRgpgtzIBd0Fw+lNrDfo0NIuKzu0l42zePza3Y4fB51nfMsikYbkzTF9Dy3AiUC5t5O7KnKAQrevTU+m6NDavhrdF6x+3YjTs95mzw33Q1x3OvmtiSoCJEBGXdMLoQFrI8n9kmmprevP3z0mA8ESnz\\\\\\\/0EJ7OSTAwAP4MBBrBBTRFv4yTn2utn6Zgjd0OYIGZbTWkqcIfsCM00elmTZJzXmPts0muJcxGNwTMni07O10qWX5htS9Sw11jx+GGCOtW7oVELxQNlwy1JuwVf8ZINwGlnXjCax90BXUqOS298P7xB7TdEh5r2eIVtL4ckqEI+CsSG1GKuHqF+IOO53TZQ2f\\\\\\\/qtPVpajxht5Z3cnCD4UP+Ukglfo0xofIYCmTAYuG1kNmKmpYXaqLqokQ5K\\\\\\\/oJg89Y9Hced9iaqPCRMl7AByVxYmZI3s+8drVigLQcFAxTl4AhuFCKglGt1yKK8rXXlqTWywuwTn4MJTsTr3lgIO7MYc9hPdmz1RM1gjaEsNeERws00UHI8AVLpMeern9COlM08j\\\\\\\/iV2jkL8GdE0zLsMOVa5Mu72aTe2u\\\\\\\/X8ldEN933ArnfxBcf2f4rCSGNQ9gGuDHyyC6+xTrX8uuHx81mGmW+7D4xmYUpd6xnS2bFE1YAX3ZCKAnqADmRvhOzCxmYBvZ+fslcXWC3yCXUJejrfTUIJvYoNPcjNPiTKIHmJNA0gLUfDttuXYEECZ5VgA8yLziIlx4C\\\\\\\/cBt2H\\\\\\\/HStDaNCLkz99Q4GSPvWbXL8lQkC55ftqbtlsofQDU8W4bLvtXI\\\\\\\/8OmC32EJWc0Tjyh5CY2XVVISiVcVnOfd2rjrTUdyMDisirUDQf0o+wrVBvVlw+FP0irZ3iZraZtnMalswIabS9QxDORVDYLQhVo2zNoy0YctrFLK+TkcyQtgRArgVuGT6s\\\\\\\/IE6K0Bjp81NAAqTiD92QFW6WMbG9PxrK6MF6A\\\\\\\/xel5Lvr+TTO8DvyYXkkmW+zPdUALsPPUZoP8sbF+RRnWaQquAWv25YUDFI3FaOgP8lHjTL+APqpwo41Sf0FOE7uEHV8oIf6ihPiz\\\\\\\/2rlmUCGLxh6fp+MMwrK75SfAVtVLffxvHiRo9OMhIv75OoVJ4\\\\\\\/wvXKJ+Hn6dFjc3g6Qqkc7JAfMUxOzFttLX30RPi+m6UjKBivtzc7Md2rUM\\\\\\\/K1NbSGCew8T5NdG+FhVvS696yrBscEOoueVIstzBo7di8\\\\\\\/Tf9IfpTuVb8t8aImu2AC7ziFXVbLINo+d5UpUmrjIrZe+VAFnvaexWVVSa+ntVTaRThkDL2qnFFLYg1YQPECxhkPumYTH0lOTf30gBnlniDmOimMPUkRwJQXIC1WvJkefKhQIWqSBlOhc0UPRnAgEZjmn1oYq5O7cMt\\\u0022,\\\u0022iv\\\u0022:\\\u0022XUWvnc6Nnbqpfq2\\\\\\\/gkLrkw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c9c7465d30247123df9368524d46f7388af6bbf2478b0da83408b937d5e4795a\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494051 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/2913" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-z3j39NVMAlGjWCgDCR3fF3H3o9hugLzam3yvsr5CE44" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/2913" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-h-oHr31_JFBVErm5bWb4TUvjuNO9az1jpzBGlDQu7Nc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/9/2905" title="Validation and Characterization of Maize Multiple Disease Resistance QTL" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/9/2925" title="Joint Use of Genome, Pedigree, and Their Interaction with Environment for Predicting the Performance of Wheat Lines in New Environments" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494051" id="node494051" data-pisa="ggg;9/9/2913" data-pisa-master="ggg;g3.119.400493" data-apath="/ggg/9/9/2913.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Deep Kernel for Genomic and Near Infrared Predictions in Multi-environment Breeding Trials</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0685-2867" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jaime Cuevas</span>, <span class="highwire-citation-author" data-delta="1">Osval Montesinos-López</span>, <span class="highwire-citation-author" data-delta="2">Philomin Juliana</span>, <span class="highwire-citation-author" data-delta="3">Carlos Guzmán</span>, <span class="highwire-citation-author" data-delta="4">Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author" data-delta="5">José González-Bucio</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-1468-4867" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Juan Burgueño</span>, <span class="highwire-citation-author" data-delta="7">Abelardo Montesinos-López</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2913-2924;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400493 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494051" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jaime Cuevas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Universidad de Quintana Roo, Chetumal, Quintana Roo, 77019 México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaime%2BCuevas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cuevas%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJaime%2BCuevas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0685-2867" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jaime Cuevas</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Osval Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Facultad de Telemática, Universidad de Colima, C. P. 28040, Edo. de Colima, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Philomin Juliana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Carretera Mexico- Veracruz Km. 45, El Batán, 56237, Texcoco, Edo. de Mexico, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philomin%2BJuliana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Juliana%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhilomin%2BJuliana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Carlos Guzmán </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Carretera Mexico- Veracruz Km. 45, El Batán, 56237, Texcoco, Edo. de Mexico, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carlos%2BGuzm%C3%A1n%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guzm%C3%A1n%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarlos%2BGuzm%25C3%25A1n%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Colegio de Postgraduados, Montecillos 56230, Edo. de Mexico, Mexico, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">José González-Bucio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Universidad de Quintana Roo, Chetumal, Quintana Roo, 77019 México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BGonz%C3%A1lez-Bucio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gonz%C3%A1lez-Bucio%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BGonz%25C3%25A1lez-Bucio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Carretera Mexico- Veracruz Km. 45, El Batán, 56237, Texcoco, Edo. de Mexico, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1468-4867" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Juan Burgueño</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías, (CUCEI), Universidad de Guadalajara, Guadalajara, Jalisco, 44430</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Carretera Mexico- Veracruz Km. 45, El Batán, 56237, Texcoco, Edo. de Mexico, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/9/2913" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494051" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494051/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/9/2913.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494051" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494051/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/9/2913.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494051" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494051/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/9/2913.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494051" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494051/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Kernel methods are flexible and easy to interpret and have been successfully used in genomic-enabled prediction of various plant species. Kernel methods used in genomic prediction comprise the linear genomic best linear unbiased predictor (GBLUP or GB) kernel, and the Gaussian kernel (GK). In general, these kernels have been used with two statistical models: single-environment and genomic × environment (GE) models. Recently near infrared spectroscopy (NIR) has been used as an inexpensive and non-destructive high-throughput phenotyping method for predicting unobserved line performance in plant breeding trials. In this study, we used a non-linear arc-cosine kernel (AK) that emulates deep learning artificial neural networks. We compared AK prediction accuracy with the prediction accuracy of GB and GK kernel methods in four genomic data sets, one of which also includes pedigree and NIR information. Results show that for all four data sets, AK and GK kernels achieved higher prediction accuracy than the linear GB kernel for the single-environment and GE multi-environment models. In addition, AK achieved similar or slightly higher prediction accuracy than the GK kernel. For all data sets, the GE model achieved higher prediction accuracy than the single-environment model. For the data set that includes pedigree, markers and NIR, results show that the NIR wavelength alone achieved lower prediction accuracy than the genomic information alone; however, the pedigree plus NIR information achieved only slightly lower prediction accuracy than the marker plus the NIR high-throughput data.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-based-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-based-prediction" rel="nofollow">Genomic based prediction</a></li><li class="kwd"><a href="/keyword/genomic-best-unbiased-predictor-gblup-gb-linear-and-non-linear-kernel-methods-0" class="hw-term hw-article-keyword hw-article-keyword-genomic-best-unbiased-predictor-gblup-gb-linear-and-non-linear-kernel-methods" rel="nofollow">Genomic Best Unbiased Predictor (GBLUP, GB linear and non-linear kernel methods)</a></li><li class="kwd"><a href="/keyword/near-infrared-nir-high-throughput-phenotype" class="hw-term hw-article-keyword hw-article-keyword-near-infrared-nir-high-throughput-phenotype" rel="nofollow">near infrared (NIR) high-throughput phenotype</a></li><li class="kwd"><a href="/keyword/single-environment-model-0" class="hw-term hw-article-keyword hw-article-keyword-single-environment-model" rel="nofollow">single-environment model</a></li><li class="kwd"><a href="/keyword/deep-learning" class="hw-term hw-article-keyword hw-article-keyword-deep-learning" rel="nofollow">deep learning</a></li><li class="kwd"><a href="/keyword/genomic-%C3%97-environment-interaction-model" class="hw-term hw-article-keyword hw-article-keyword-genomic-&#xD7;-environment-interaction-model" rel="nofollow">genomic × environment interaction model</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic Prediction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">In genomic selection (GS) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Meuwissen <em>et al.</em> 2001</a>), Bayesian models were introduced in the context of whole-genome regression and since then have become common in genomic prediction (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gianola 2013</a>). Genomic-assisted selection has the advantage over phenotypic selection that it saves time and resources when making selection by adopting predictive methods and models for complex traits, along with information on pedigree, dense molecular markers and, when available, climatic covariates (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">de los Campos <em>et al.</em> 2009</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Crossa <em>et al.</em> 2017</a>).</p><p id="p-4">Using dense molecular markers leads to the problem of <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-1.gif"/></noscript></span></span> (markers), which can be avoided by assuming that the contribution of molecular markers is a random variable with some distributions and with a variance-covariance matrix that consists of a scalar variance component and a known covariance kernel obtained from the markers (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gianola and Van Kaam 2008</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">de los Campos <em>et al.</em> 2010</a>). This covariance kernel can model genetic effects as additive, dominant, and epistatic, as a mixture of these effects or even as genetic and non-genetic remaining effects (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Crossa <em>et al.</em> 2010</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Technow <em>et al.</em> 2012</a>). During the last years, genomic-enabled predictions have usually been made with models and methods that take into account genomic × environment interaction (GE) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Burgueño <em>et al.</em> 2012</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Jarquín <em>et al.</em> 2014</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">López-Cruz <em>et al.</em> 2015</a>).</p><p id="p-5">A variety of multi-environment models and methods can be described based on kernel matrices, and using information on genotypes, molecular markers and environmental covariables (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Burgueño <em>et al.</em> 2012</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Jarquín <em>et al.</em> 2014</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Souza <em>et al.</em> 2017</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cuevas <em>et al.</em> 2016</a>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">2017</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">2018</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Granato <em>et al.</em>, 2018</a>). Multi-environment kernels can be constructed depending on the choice of covariance function and the multi-environment model. Usually two kernel functions have been used for genomic-enabled prediction of multi-environment breeding trials: the Genomic Best Linear Unbiased Predictor (GBLUP or GB) and the Gaussian Kernel (GK) (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Cuevas <em>et al.</em> 2016</a> and <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">2017</a>). The GB is the standard linear kernel derived from the properties of a multivariate normal distribution in linear mixed models and is usually called the genomic relationship matrix obtained from the marker matrix. The GK appeared as a reproducing kernel in the semi-parametric model Reproducing Kernel Hilbert Spaces (RKHS) (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Gianola and Van Kaam 2008</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">de los Campos <em>et al.</em> 2010</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">González-Camacho <em>et al.</em> 2012</a>) with a bandwidth (<em>h</em>) parameter that controls the rate of decay of the covariance between genotypes. In this context, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Pérez-Elizalde <em>et al.</em> (2015)</a> proposed an empirical Bayes method for estimating the bandwidth parameter <em>h</em>. Results have consistently shown for single-environment models as well as for GE models, that GK performs better than GB in terms of genomic-enabled prediction accuracy (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Cuevas <em>et al.</em> 2016</a>, <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">2017</a>, <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">2018</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Souza <em>et al.</em> 2017</a>).</p><p id="p-6">The GK kernel is one of the covariance matrices that attempts to capture complex marker effects and their possible interactions. The GK kernel is constructed based on a squared distance between a pair of genotypes multiplied by an estimated bandwidth parameter, <em>h</em>. <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Wilson (2014)</a> pointed out that the potential of the non-parametric or semi-parametric Bayesian models is better expressed on big data sets. These authors mentioned that the best proposition for the Bayesian non-parametric and/or semi-parametric models is the GK kernel, which in reality does not capture all the available information existing in the data because it uses only one value for the bandwidth parameter, which is globally calculated with the available training data (marker and phenotypic data) using cross-validation methods or the marginal maximum likelihood. The consequence of having models with a tendency toward fitting global effects is that they lose the ability to account for local effects, thus limiting their predictive ability. One way to minimize this problem is to use a matrix of bandwidth values of order equal to the marker matrix; however, its estimation by cross-validation (or by any other method) is not computational feasible due to its potentially extremely large dimensions.</p><p id="p-7">There have been alternatives for increasing the prediction ability of genetic values of these kernel methods; for example, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Jiang <em>et al.</em> (2018)</a> model the local epistatic effects with different kernels constructed by means of haplotypes. A kernel of different characteristics was proposed by <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cho and Saul (2009)</a> that emulates the artificial neural networks. To achieve this, the structure of the covariance matrix is constructed with a similarity matrix between genotypes that takes the Arc-cosine angle between the vector of genotypes. This makes it possible to generate a non-linear Arc-cosine kernel (named AK) that emulates the artificial neural network with one layer (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Neal 1996</a>). <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Cho and Saul (2009)</a> used the theoretical framework of the kernels as the interior product of characteristic functions to emulate the hidden layers of an artificial neural network such that the kernel at one level is a function of the kernel at a previous level. Thus, the result mimics a multiple hidden layer structure (or levels) that can be used with the Bayesian paradigm from a Gaussian process with the machine learning infrastructure.</p><p id="p-8">In general, kernel methods have been perceived as being much less flexible than neural network methods (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Cho and Saul 2009</a>). That is why <a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Cho and Saul (2009)</a> investigated the use of kernel methods in deep learning by proposing a family of kernel functions that mimic the computation of large neural networks and used those kernel functions that exploit the advantages of deep learning. The authors proposed a kernel function based on the magnitude of two observations, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-2.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-3.gif"/></noscript></span></span>, and the angle between them, with the main purpose of emulating the performance of neural networks with one layer. Among the basic functions, <a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Cho and Saul (2009)</a> proposed one function that is not linear with similar properties and functions of other kernels. <a id="xref-ref-4-6" class="xref-bibr" href="#ref-4">Cho and Saul (2009)</a> also proposed a recursive formula to emulate the performance of various layers (<em>l</em>), that is, the covariance matrix (kernel) in one layer (or level) is computed by taking a function of the previous kernel, and applying it to classification problems with a multilayer kernel machine algorithm. The new kernel methods developed by <a id="xref-ref-4-7" class="xref-bibr" href="#ref-4">Cho and Saul (2009)</a> had distinct properties, and the authors evaluated them on a wide range of real data sets and observed important increases in accuracy, computer speed and memory consumption.</p><p id="p-9">Machine learning is a field of computer science that has the ability to improve performance on a specific task from data. In general, machine learning explores algorithms that can learn from current data and make predictions on new data, by building a model from sample inputs (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Samuel 1959</a>). Several machine learning algorithms uses an artificial neural network with multiple layers linked nonlinearly. The “deep” in deep learning refers to the number of layers through which the data are transformed. The layers in these methods consist of multiple stages of nonlinear data transformations, and the features of the data are represented by successively higher layers in the neural network system. In plant breeding, initial applications of artificial neural networks were used for genomic-enabled prediction (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">González-Camacho <em>et al.</em> 2012</a>, 2016; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Ornella <em>et al.</em> 2012</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">2014</a>). The application of new deep learning methods for genomic-enabled prediction was recently studied for extensive maize and wheat data sets including or not including GE, and for continuous as well as mixed binary, ordinal and continuous multi-traits and multi-environments (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Montesinos-López <em>et al.</em> 2018a</a>,<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">b</a> and 2019a,<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">b</a>). Results show that artificial neural networks in the form of one or more hidden layers (deep machine learning) could be an efficient option for handling big and complex data sets that are common in plant breeding.</p><p id="p-10">It has been shown that other sources of molecular variation can be used for the prediction of complex traits; however, these tools have limited applications in large plant breeding trials. Recently, researchers have used low cost non-destructive tools such as near-infrared spectroscopy (NIR) and nuclear magnetic resonance as a high-throughput phenotype for predicting grain yield and other end-quality traits in various crop species (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hayes <em>et al.</em> 2017</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Rincent <em>et al.</em> 2018</a>). Interestingly, <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Rincent <em>et al.</em> (2018)</a> used high-throughput phenotyping for wheat lines and a tree species (poplar) irradiated with infrared measured absorbance from 400 to 2500 nm. <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Rincent <em>et al.</em> (2018)</a> used NIR wavelength data as regressors to calculate the NIR similarity matrix between genotypes (NIR BLUP) in grains and leaves, and found better prediction accuracy than when using molecular markers, GBLUP (genomic similarity matrix). <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Hayes <em>et al.</em> (2017)</a> used a multi-trait approach that incorporates NIR and nuclear magnetic resonance information in the genomic models for end-use quality traits in wheat. The authors found that genomic predictions ranged from 0-0.47, whereas after adding information using NIR and nuclear magnetic resonance, prediction accuracy increased to 0-0.69.</p><p id="p-11">Based on the above statistical and biological considerations and in an attempt to examine the performance of new kernel methods in genomic-enabled prediction, we used the recursive formula of <a id="xref-ref-4-8" class="xref-bibr" href="#ref-4">Cho and Saul (2009)</a> for constructing the arc-cosine kernel (AK), where the kernel in one layer (or level) is a function of the kernel in the previous layer. The genomic-enabled prediction accuracy of AK is compared with the accuracy of GB and GK kernels in a single-environment as well as GE models. The AK is incorporated as a kinship or covariance matrix of the random effects of a genomic mixed model (de los Campos <em>et al.</em> 2010). The number of levels (layers, <em>l</em>) to be used is defined by optimizing the marginal maximum likelihood on the training set, which is similar to what <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Pérez-Elizalde <em>et al.</em> (2015)</a> proposed for computing the bandwidth parameter <em>h</em> of the Gaussian Kernel. We compared the prediction accuracy of the Arc-cosine Kernel (AK) <em>vs.</em> GB and GK using genomic data and for one data set, using genomic, pedigree and NIRs data. The comparisons between genomic-enabled prediction accuracy between the different methods (GB, GK and AK) were performed using random cross-validations that form different random groups of training (TRN) and testing (TST) sets, and computing the correlations between the observed and the predictive values for each random partition. Thus, the average correlations from all partitions and their standard deviations are calculated and reported. To form the TRN and TST sets in models including GE, the random cross-validation CV2 is used where a certain percentage of the lines are unobserved and thus predictive in some environments but observed in other environments (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Burgueño, <em>et al.</em> 2012</a>).</p><p id="p-12">For this study, we used 4 data sets. Three data sets with only genomic data (one wheat, WHEAT1, and two maize, MAIZE2 and MAIZE3 data sets) were already published; we fitted these three data sets with models including single-environment and GE multi-environment models with the three kernels methods: GB, AK, and GK. We also used an unpublished wheat data set from the CIMMYT Global Wheat Program (WHEAT4) comprising elite wheat lines evaluated in one environment and genotyped with dense molecular markers. For data set WHEAT4, the wheat seed of the lines incorporated information on NIR. For these data sets, prediction accuracy combining pedigree, markers, and NIR was studied using only the single-environment model with the GB, GK and AK kernels.</p><div class="section materials-methods" id="sec-1"><h2 class="">Material and Methods</h2><p id="p-13">In this study, we used the same models (single-environment and GE models) and methods (GB and GK) on all four data sets (WHEAT1, MAIZE2, MAIZE3, and WHEAT4) including non-linear kernel AK in all data sets plus pedigree and NIR information in the WHEAT4 data set. We briefly define methods for GB and GK kernels and describe the new AK non-linear kernel in detail.</p><div id="sec-2" class="subsection"><h3>Kernel methods</h3><p id="p-14">The GBLUP (GB) is the standard linear kernel derived from the properties of a multivariate normal distribution in linear mixed models and is usually called the genomic relationship matrix obtained as <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-4.gif"/></noscript></span></span>where <em><strong>X</strong></em> could be the scaled marker matrix (with <em>p</em> molecular markers) (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">López-Cruz <em>et al.</em> 2015</a>) or the kernel matrix formed based on NIRs (NIR BLUP). The GK appeared as a reproducing kernel in the semi-parametric model Reproducing Kernel Hilbert Spaces (RKHS) (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">González-Camacho <em>et al.</em> 2012</a>). It is defined as <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-5.gif"/></noscript></span></span> where <em>h</em> is the bandwidth parameter that controls the rate of decay of the covariance between genotypes, and q is the median of the square of the Euclidean distance, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-6.gif"/></noscript></span></span>, which is a measure of the genetic distance between individuals based on molecular markers. <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Pérez-Elizalde <em>et al.</em> (2015)</a> proposed an empirical Bayes method for estimating the bandwidth parameter <em>h</em>.</p><p id="p-15">Below we describe the Arc-cosine kernel (AK) method proposed by <a id="xref-ref-4-9" class="xref-bibr" href="#ref-4">Cho and Saul (2009)</a> that includes one basic function or level (layer one) and a recursive function for different levels of recursion.</p></div><div id="sec-3" class="subsection"><h3>Arc-cosine Kernel (AK)</h3><p id="p-16">This method considers that the main similarity measure between two individuals is the angle between two vectors that is computed from marker genotypes (or NIRS) <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-7.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-8.gif"/></noscript></span></span>, that is:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-9.gif"/></noscript></span></span>where ⋅ denotes the inner product and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-10.gif"/></noscript></span></span> is the norm of line <em>i</em>. <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Neal (1996)</a> showed that the following kernel is semidefinite positive and related to the neural network with a single layer.<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-11.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where π is the pi constant (3.1415...) and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-12.gif"/></noscript></span></span>. Equation (1) gives a symmetric positive semidefinite matrix (<span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-13.gif"/></noscript></span></span> with entries given in Equation 1) preserving the norm of the entries such that <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-14.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-15.gif"/></noscript></span></span> and models non-linear relationships.</p><p id="p-17">Because the kernel is the inner product in an induced characteristic space <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-16.gif"/></noscript></span></span>, the reasoning of <a id="xref-ref-4-10" class="xref-bibr" href="#ref-4">Cho and Saul (2009)</a> is that repeating this process <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-17.gif"/></noscript></span></span>times <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-18.gif"/></noscript></span></span> will emulate the performance of the neural network with <em>l</em> hidden layers by using the following recursive formula.<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-19.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>Equation (1) considers the case with <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-20.gif"/></noscript></span></span> (AK<sup>1</sup>), from where (2) is derived for <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-21.gif"/></noscript></span></span> (AK<sup>2</sup>, AK<sup>3</sup> ...). Note that (1) does not require continuous bandwidth as in the case of GK (Gaussian Kernel). Also from (2), the <em>l</em> level with the best predictive ability can be selected using the training set and the recursive equation. We selected the <em>l</em> level that has the maximum marginal likelihood in a way that is similar to what <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Pérez-Elizalde <em>et al.</em> (2015)</a> did to select the bandwidth parameter (<em>h</em>) of the GK kernel method.</p></div><div id="sec-4" class="subsection"><h3>Statistical models</h3><p id="p-18">The statistical models employed in this study were defined and described in <a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Cuevas <em>et al.</em> (2016</a>, <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">2017</a>, <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">2018</a>) and used by <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Souza <em>et al.</em> (2017)</a> for the single-environment model and the GE model using the GB, and the GK kernel methods. A brief discussion is given below.</p></div><div id="sec-5" class="subsection"><h3>General linear model</h3><p id="p-19">The following general mixed linear model can cover a great diversity of cases and models such as those described in <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Crossa <em>et al.</em> (2010)</a>, <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Jarquín <em>et al.</em> (2014)</a>, <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">López-Cruz <em>et al.</em> (2015)</a>, and <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Crossa <em>et al.</em> (2017)</a> for relating the phenotypic observations in one environment or in multi-environments, with covariables like dense molecular markers, pedigree, environmental covariables, NIR, and others.<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-22.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where <strong><em>y</em></strong> is the vector of observations of size <em>n</em>, the scalar <em>μ</em> is a general mean, matrix <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-23.gif"/></noscript></span></span> is an incidence matrix for fixed effects <strong><em>β</em></strong>. It is assumed that the random vectors <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-24.gif"/></noscript></span></span>, each of size <em>n</em>, are independent from other random effects and follow a normal distribution <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-25.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-26.gif"/></noscript></span></span> being a variance component parameter estimated by the model, and <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-27.gif"/></noscript></span></span> a known semidefinite positive symmetric matrix of order <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-28.gif"/></noscript></span></span>. The random errors are assumed independent with normal distributions <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-29.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-30.gif"/></noscript></span></span> is the error variance.</p></div><div id="sec-6" class="subsection"><h3>Single-environment model</h3><p id="p-20">For a single environment and only one kernel, equation 3 can be expressed as:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-31.gif"/></noscript></span></span>where vector <strong><em>y</em></strong> represents the phenotypic observations in that environment, <strong><em>u</em></strong> is the genomic random effects assumed normally distributed as <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-32.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-33.gif"/></noscript></span></span> is the genomic variance estimated from the data, matrix <strong><em>k</em></strong> is constructed as <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-34.gif"/></noscript></span></span> where matrix <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-35.gif"/></noscript></span></span>the genotypes with the observations, <strong><em>G</em></strong> models the relationship of the genotypes and could be a pedigree matrix or a genomic matrix or a matrix based on NIR (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Rincent <em>et al.</em> 2018</a>) throughout GB, GK and AK, as previously described. The random errors are assumed independent with normal distributions <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-36.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-37.gif"/></noscript></span></span> is the error variance.</p></div><div id="sec-7" class="subsection"><h3>Multi-environment models</h3><p id="p-21">In the GE multi-environment reaction norm model (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Jarquín <em>et al.</em> 2014</a>; <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">López-Cruz <em>et al.</em> 2015</a>; <a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">Cuevas <em>et al.</em> 2016</a>), equation (3) takes the form<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-38.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-39.gif"/></noscript></span></span> are the observations in each of the <em>m</em> sites (or environments). If environments are considered fixed effects, they are represented in the <strong><em>β</em></strong> effects and included in matrix <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-40.gif"/></noscript></span></span> , which relates the sites (or environments) to the phenotypic observations. For convenience, in the notations we call this matrix <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-41.gif"/></noscript></span></span> (<a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Souza <em>et al.</em> 2017</a>), that is, <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-42.gif"/></noscript></span></span>. In this model, <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-43.gif"/></noscript></span></span> represents the genomic main effects, <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-44.gif"/></noscript></span></span>is the genomic variance component estimated from the data, and <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-45.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-46.gif"/></noscript></span></span> relates the genotypes to the phenotypic observations. The random effects representing the interaction between the genomic effects and their interaction with sites or environments are modeled as <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-47.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-48.gif"/></noscript></span></span>, where ° is the Hadamard (or Schur) product. The Hadamard product takes two matrices of the same order and generates a matrix as a result of multiplying each element <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-49.gif"/></noscript></span></span> from one matrix with the corresponding element <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-50.gif"/></noscript></span></span> of the other matrix (of the same order). As previously mentioned, matrix <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-51.gif"/></noscript></span></span> relates the sites (or environments) to the phenotypic observations.</p></div><div id="sec-8" class="subsection"><h3>Method for selecting the number of layers (l) for the AK kernel</h3><p>The following sequence is used to select the appropriate <em>l</em> level (number of layers) when fitting the AK kernel with the single-environment and the GE multi-environment models.</p><ol class="list-alphalower " id="list-1"><li id="list-item-1"><p id="p-23">The first step is computing the <strong><em>G</em></strong> base matrix for <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-52.gif"/></noscript></span></span> using equation (1) from the markers or the NIR, such that <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-53.gif"/></noscript></span></span> (single-environment and GE multi-environment) (see the auxiliary function in link hdl.handle.net/11529/10548180 where the four data sets and other codes are given).</p></li><li id="list-item-2"><p id="p-24">Compute the maximum marginal likelihood for this first level using the available training data.<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-54.gif"/></noscript></span></span>This is similar to what <a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Pérez-Elizalde <em>et al.</em> (2015)</a> did when estimating the bandwidth parameter <em>h</em> for the GK kernel (see the APPENDIX).</p></li><li id="list-item-3"><p id="p-25">Construct <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-55.gif"/></noscript></span></span> using equation (2).</p></li><li id="list-item-4"><p id="p-26">Compute the maximum (<em>M</em>) marginal (<em>M</em>) likelihood of <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-56.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-57.gif"/></noscript></span></span>.</p></li><li id="list-item-5"><p id="p-27">Compute the maximum marginal likelihood of <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-58.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-59.gif"/></noscript></span></span>; if maximum marginal (MM) <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-60.gif"/></noscript></span></span> &gt; <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-61.gif"/></noscript></span></span>, we increase one level <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-62.gif"/></noscript></span></span> and repeat from c); if <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-63.gif"/></noscript></span></span> &lt;= <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-64.gif"/></noscript></span></span>, we select l.</p></li></ol><p>With the selected level of <em>l</em> we compute the kernel matrices required for the model and thus fit the model. To select the level of <em>l</em> for the GE multi-environment model, we only consider the main effects, which is where there is exchange (borrowing) of information between environments. That is, when modeling GE, we use phenotypic observations from all environments; thus, when deriving the prediction for a given line not observed in one environment, the GE model benefits from records of that line that were collected in other correlated environments. This is the random cross-validation CV2 (<a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">López-Cruz <em>et al.</em> 2015</a>) that considers some lines observed in some environments and predicted in others.</p></div><div id="sec-9" class="subsection"><h3>Phenotypic and genotypic data sets</h3><div id="sec-10" class="subsection"><h4>WHEAT1:</h4><p id="p-28">This data set was first used by <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Crossa <em>et al.</em> (2010)</a> and also by <a id="xref-ref-8-6" class="xref-bibr" href="#ref-8">Cuevas <em>et al.</em> (2016</a>, <a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">2017</a>) and comprises 599 wheat lines from the CIMMYT Global Wheat Program evaluated in four international environments representing four basic agroclimatic regions (mega-environments). The phenotypic trait considered here was grain yield (GY) of the 599 wheat lines evaluated in each of the four mega-environments. The 599 wheat lines were genotyped using 1447 Diversity Array Technology (DArT) markers generated by Triticarte Pty. Ltd. (Canberra, Australia; <a href="http://www.triticarte.com.au">http://www.triticarte.com.au</a>). The markers were filtered and those with a minor allele frequency lower than 0.05 were removed, and missing genotypes were imputed using samples from the marginal distribution of marker genotypes. The number of DArT markers after filtering was 1279.</p></div><div id="sec-11" class="subsection"><h4>MAIZE2:</h4><p id="p-29">This data set was included in <a id="xref-ref-36-5" class="xref-bibr" href="#ref-36">Souza <em>et al.</em> (2017)</a> and is from the Helix Seeds Company (HEL). It consists of 452 maize hybrids obtained by crossing 111 pure lines (inbreds); the hybrids were evaluated in 2015 at five Brazilian sites (E1-E5). The experimental design used in each site was a randomized block with two replicates per hybrid. The parent lines were genotyped with an Affymetrix Axiom Maize Genotyping Array of 616 K SNPs (Single Nucleotide Polymorphism) (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Unterseer <em>et al.</em> 2014</a>). Standard quality controls (QC) were applied to the data, removing markers with a <em>Call Rate</em> ≥ 0.95. The remaining missing data in the lines were imputed with the Synbreed package (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Wimmer <em>et al.</em> 2015</a>) using the algorithms from the Beagle 4.0 software (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Browning and Browning 2009</a>). Markers with Minor Allele Frequency (MAF) of ≤ 0.05 were removed. After applying QC, 52,811 SNPs were available to make the predictions. The phenotypic and genomic data of inbred lines are credited to Helix Seeds Ltda. Company.</p></div><div id="sec-12" class="subsection"><h4>MAIZE3:</h4><p id="p-30">This maize data set was included in <a id="xref-ref-36-6" class="xref-bibr" href="#ref-36">Souza <em>et al.</em> (2017)</a>, comes from USP (Universidad Sao Paulo) and consists of 740 maize hybrids obtained by crossing 49 inbred lines. The hybrids were evaluated in four environments (E1-E4) in Piracicaba and Anhumas, São Paulo, Brazil, in 2016. The hybrids were evaluated using an augmented block design, with two commercial hybrids as checks to correct for micro-environmental variation. At each site, two levels of nitrogen (N) fertilization were used. The experiment conducted under ideal N conditions received 100 kg ha<sup>-1</sup> of N (30 kg ha<sup>-1</sup> at sowing and 70 kg ha<sup>-1</sup> in a coverage application) at the V8 plant stage, while the experiment with low N received 30 kg/ha of N at sowing. As in the MAIZE1 data set, the parent lines were genotyped with an Affymetrix Axiom Maize Genotyping Array of 616 K SNPs. Markers with Minor Allele Frequency (MAF) of ≤ 0.05 were removed. After applying QC, 54,113 SNPs were available to make the predictions.</p></div><div id="sec-13" class="subsection"><h4>WHEAT4:</h4><p id="p-31">A total of 561 elite wheat lines from CIMMYT’s advanced bread wheat breeding program were evaluated in field trials for grain yield and genotyped with 5,500 molecular markers. The collected 1059 NIR wavelengths were normalized by taking the first (NIR1) and second derivative (NIR2) in an attempt to increase the possible differences in sample signals that might be overlooked without this mathematical treatment. The pedigree information was extracted and used for prediction based on pedigree by using the numerical relationship matrix <strong><em>A</em></strong>. The single-environment model was also fitted using the numerical relationship matrix <strong><em>A</em></strong> in combination with the NIR BLUP and the genomic similarity between wheat lines given by the linear <strong><em>G</em></strong> kernel of the GBLUP.</p></div></div><div id="sec-14" class="subsection"><h3>NIR (Near-infrared reflectance spectroscopy) used for the WHEAT4 data set</h3><p id="p-32">Spectroscopy in the near infrared has been used widely for the rapid determination of organic components. The NIR technique requires that the energy absorbed in the near-infrared region by a sample causes covalent bonds of C-H, O-H, and N-H, important components of organic substances, to vibrate in different forms. For this study we used an NIRS 6500 FOSS-NIR System that poses a scanner software ISIScan version 1.6 on the range of wavelength from 400 to 2500 nanometers (nm) of reflectance. The database was managed by the software WinISI and the reflectance data were stored as the log (1/R) (where R is the reflectance) using intervals of 20 nm. We deposited samples consisting of 20-25 grams of clean wheat seed in the cell for the NIR.</p></div><div id="sec-15" class="subsection"><h3>Assessing prediction accuracy by random cross-validation</h3><p id="p-33">In each data set we partitioned the data 50 times, with 70% of the data for training and 30% for testing. The prediction accuracy was measured by Pearson’s correlation between the predicted and observed values (the standard deviation was also computed). For data sets WHEAT1, MAIZE2, and MAIZE3, the predictions were made for each environment for both models (single-environment model and GE multi-environment model) using kernels GB, GK and AK constructed with the molecular markers. For model GE multi-environment, the random cross-validations formed a CV2 cross-validation where some lines had been tested in some environments but not in others (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Burgueño <em>et al.</em> 2012</a>). For data set WHEAT4, we compared results only for the single-environment model for kernels GB, GK and AK from the markers and from the NIRs obtained with the first and second derivatives. We also included the pedigree and fitted different models with different combinations of markers, NIR and pedigree.</p><p id="p-34">In data set WHEAT1, we made predictions using AK1-AK4 with the objective of comparing the prediction accuracy of AK with different numbers of levels (layers, <em>l</em>). In all the data sets we computed the appropriate number <em>l</em> using the marginal maximum likelihood. When fitting each model, we used 30,000 iterations, eliminated the first 5000 and thinned every three iterations.</p></div><div id="sec-16" class="subsection"><h3>Software</h3><p id="p-35">Models were implemented using R codes (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">R Core Team 2018</a>). In particular for the first and second derivatives of the matrix of NIRs of WHEAT4 data set, we used the Savitzky Golay function from the library prospector (Stevens and Ramirez-Lopez 2014). For fitting the models and making predictions, we used the BGGE function from the BGGE library (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Granato <em>et al.</em> 2018</a>). As mentioned in <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Granato <em>et al.</em> (2018)</a>, the BGGE is an algorithm constructed to fit models considering a Bayesian framework that uses the Gibbs sampler for the Montecarlo Markov Chain (MCMC), which allows convergence to a posterior predictive distribution that provides the predictive values. The codes can be found at link hdl.handle.net/11529/10548180.</p><p id="p-36">The link has word files identified as WHEAT1, MAIZE2 MAIZE3, and WHEAT4 data sets explaining where the different data sets are located, where the R codes are located, and the name of the files where the R codes of the methods (GB, GK, AK) are and where the models (single-environment) and GE environment are placed.</p></div><div id="sec-17" class="subsection"><h3>Data availability</h3><p id="p-37">Although WHEAT1, MAIZE2, and MAIZE3 data sets have been published and the data are already available from those articles, we have uploaded them again at the following link, together with WHEAT4 data (pedigree, markers and NIRs). At this link hdl.handle.net/11529/10548180 we have included the phenotypic and genotypic data of each data set so that the readers can download and use it immediately without having to do any further data manipulation, extraction, etc. Furthermore, the folders of each data set contain all the R software used to analyze each data set, as previously described [there are two files with the R codes for computing the various functions, Auxiliar.functions.R for AK and GE, and another one that is used for each data set with all the models named Model.(WHEAT1…)R. In addition, we added a word file explaining all the previous files].</p></div></div><div class="section results" id="sec-18"><h2 class="">Results</h2><p id="p-38">Results are presented by data set for the single-environment and GE multi-environment models and for the three kernel methods, GB, AK and GK. The tables present the average correlation between the observed and predictive values for each model-method. The best predictive correlations are in boldface for the corresponding method (AK; GK; GB). Tables 1-5 have two types of superscripts; (+) indicates the significant <em>t</em>-test of the hypothesis that the average correlation of the AK method is higher at the 5% probability level than the average correlation of the GB method. The superscript (*) denotes the significant <em>t</em>-test of the hypothesis that the average correlation of the AK method is higher at the 5% probability level than the average correlation of the GK method.</p><div id="sec-19" class="subsection"><h3>WHEAT1 data set</h3><p id="p-39">The highest genomic-enabled prediction accuracies for the single-environment model were obtained with the AK method, closely followed by the GK method (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The genomic prediction accuracy of various AK methods (AK<sup>1</sup> – AK<sup>4</sup>) increased as more levels (layers) were added, but their accuracy did not improve after four layers, whereas the AK with the recursive method for increasing the number of layers by examining the marginal likelihood achieved higher prediction accuracy than the AK<sup>1</sup> – AK<sup>4</sup> methods. <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> shows the levels (layers) of the maximum marginal likelihood for each environment; these values are also displayed in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> for each environment (E1-E4). The prediction accuracy of the AK method was significantly higher than that of the GB method but not of the GK method.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494206/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494206/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">WHEAT1 data set. Average Pearson’s correlations between observed and predictive values (and their standard deviation in parentheses) for seven methods for a single-environment model for 50 random partitions with 70% of the lines in the training set and 30% of the lines in the testing set. Methods GB, GK, and AK are the GBLUP, Gaussian Kernel, and Arc-Cosine Kernel, respectively. Methods <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/T1/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/T1/embed/mml-math-65.gif"/></noscript></span></span> - <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/T1/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/T1/embed/mml-math-66.gif"/></noscript></span></span> correspond to the Arc-Cosine kernel model with 1-4 levels (layers). The best predictive model for each environment (E1-E4) is in boldface</span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2913/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="WHEAT1 data set. Marginal likelihood for several levels (layers) for environments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1544100896" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;WHEAT1 data set. Marginal likelihood for several levels (layers) for environments.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2913/F1.medium.gif" width="440" height="316"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/9/2913/F1.medium.gif" width="440" height="316"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2913/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2913/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494163" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-41" class="first-child">WHEAT1 data set. Marginal likelihood for several levels (layers) for environments.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">When the GE multi-environment model was fitted, the method with the highest accuracy was AK, again closely followed by the GK method (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>) (no significant differences). Except for the genomic-enabled prediction of E1 (that has negative correlations with the other 3 environments, <a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Burgueño <em>et al.</em> 2012</a>), where the single-environment model was more accurate than the GE multi-environment, for the other environments (E2-E4), the GE multi-environment had higher prediction accuracy than the single-environment model for all three methods GB, AK and GK. The GB kernel method did not overcome any of the two nonlinear kernel AK and GK in terms of prediction accuracy. For the GE multi-environment model, six levels (layers) were selected using the previously described marginal likelihood.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494134/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494134/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">WHEAT1 data set. Average Pearson’s correlations between observed and predictive values (and their standard deviation in parentheses) for three methods for a GE multi-environment model for 50 random partitions with 70% of the lines in the training set and 30% of the lines in the testing set. Methods GB, GK, and AK are the GBLUP, Gaussian Kernel and Arc-Cosine Kernel, respectively, with six levels (layers). The best predictive model for each environment (E1-E4) is in boldface</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h3>MAIZE2 data set</h3><p id="p-44">In general, the prediction accuracies for the MAIZE2 data set were higher than those found in the WHEAT1 data set. The best predictive method was AK, which had significantly higher prediction accuracies than the GK and GB methods for all single-environment analyses (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). The differences between the AK and GK methods for E1-E5 were substantial and significant in terms of average correlation (<em>e.g.</em>, E1:755 AK <em>vs.</em> 0.733 GK; E2: 0.533 AK <em>vs.</em> 0.488 GK; E3: 0.759 AK <em>vs.</em> 0.718 GK; E4: 0.502 AK <em>vs.</em> 0.469 GK; E5: 0.564 AK <em>vs.</em> 0.522 GK).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494137/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494137/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">MAIZE2 data set. Average Pearson’s correlations between observed and predictive values (and their standard deviation in parentheses) for 3 methods for single-environment and GE multi-environment models for 50 random partitions with 70% of the lines in the training set and 30% of the lines in the testing set. Methods GB, GK, and AK are the GBLUP, Gaussian Kernel and Arc-Cosine Kernel, respectively. The best predictive model for each environment (E1-E5) is in boldface</span><div class="sb-div caption-clear"></div></div></div><p id="p-47">When the GE multi-environment model was fitted, results show that the highest prediction accuracies for grain yield were obtained with AK in environment E1 (0.811); similar predictions were achieved with AK and GK in E2 and E3, and GK had a higher average correlation to AK in environments E4 and E5 (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). In all the environments, the prediction accuracies achieved by the GE multi-environment model were higher than those obtained by the single-environment model with all three methods. Also, the non-linear kernels significantly outperformed the linear kernel GB in all five environments (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>).</p></div><div id="sec-21" class="subsection"><h3>MAIZE3 data set</h3><p id="p-48">This data set has four environments, and for the single-environment model, non-linear kernels AK, GK and linear kernel GB gave very similar (and non-significantly different) prediction accuracies; AK accuracies ranged from 0.271 (E1) to 0.421 (E4) (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>), whereas GK accuracies ranged from 0.273 (E1) to 0.423 (E4).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494198/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494198/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">MAIZE3 data set. Average Pearson’s correlations between observed and predictive values (and their standard deviation in parentheses) for 3 methods for single-environment and GE multi-environment models for 50 random partitions with 70% of the lines in the training set and 30% of the lines in the testing set. Methods GB, GK, and AK are the GBLUP, Gaussian Kernel and Arc-Cosine Kernel, respectively. The best predictive model for each environment is in boldface</span><div class="sb-div caption-clear"></div></div></div><p id="p-51">Regarding the results from the GE multi-environment model, the AK kernel showed a slight increase in prediction accuracy over the GK and much greater and significant prediction accuracy (average correlations) than the linear kernel GB. Also, for all combinations of environments and kernels, the prediction accuracies of the GE multi-environment were higher than those of the single-environment model, thereby reaffirming the advantages of using GE models to improve the prediction accuracy (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). For the GE multi-environment model, the AK method had significantly higher average correlations between the observed and predictive values than the GB linear kernel but no significant differences with the GK non-linear method.</p></div><div id="sec-22" class="subsection"><h3>WHEAT4 data set</h3><p id="p-52">This data set has only one environment, so only the single-environment model can be studied. However, information on several kernel covariance matrices is available (pedigree, marker, and NIR). In general, and across all cases, the average of the predictions of the non-linear kernel GK is similar to that of the AK kernel (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>); although their average correlations were higher than those of the GB methods, the statistical test does not reject the hypothesis of being similar. <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figures 2a-c</a> depict the patterns of variation of the absorbance for raw NIR, first derivative and second derivative, respectively. The first and second derivatives (NIR1 and NIR2, respectively) computed under the <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Savitzky and Golay (1964)</a> method, filtered the noise of the original NIRs and generated the NIR1 and NIR2 with a more symmetric distribution around zero, thus eliminating some abnormal shapes of the original spectra (see Figure 2) that should help to improve the prediction accuracy (<a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Rincent <em>et al.</em> 2018</a>).</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494069/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494069/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">WHEAT4 data set. Average Pearson’s correlations between observed and predictive values (and their standard deviation in parentheses) for 3 methods for a single-environment model for 50 random partitions with 80% of the lines in the training set and 20% of the lines in the testing set. Methods GB, GK, and AK are the GBLUP, Gaussian Kernel and Arc-Cosine Kernel, respectively. The best predictive model for each environment is in boldface. The three GB, AK, and GK were applied to data from NIR1 (first derivative), NIR2 (second derivative), markers, pedigree and some combinations. The best predictive model for each type of data used is in boldface</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2913/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="WHEAT4 data set. All spectra for wheat lines are depicted in the three figures; light colors are for each wheat line, while the strong color line in each graph is the average of all spectra. (a) absorbance raw data (red); (b) normalized absorbance (first derivative) (blue); (c) normalized second derivative (green)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1544100896" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;WHEAT4 data set. All spectra for wheat lines are depicted in the three figures; light colors are for each wheat line, while the strong color line in each graph is the average of all spectra. (a) absorbance raw data (red); (b) normalized absorbance (first derivative) (blue); (c) normalized second derivative (green).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2913/F2.medium.gif" width="440" height="293"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/9/2913/F2.medium.gif" width="440" height="293"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2913/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2913/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494140" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-54" class="first-child">WHEAT4 data set. All spectra for wheat lines are depicted in the three figures; light colors are for each wheat line, while the strong color line in each graph is the average of all spectra. (a) absorbance raw data (red); (b) normalized absorbance (first derivative) (blue); (c) normalized second derivative (green).</p><div class="sb-div caption-clear"></div></div></div><p id="p-55">When using only NIR information (NIR1, NIR2), the prediction accuracy is much lower than the accuracy found when using genomic-enabled for the three kernels: GB = 0.424 (GENOMIC) <em>vs.</em> 0.349 (NIR1), AK = 0.456 (GENOMIC) <em>vs.</em> 0.347 (NIR1), and GK = 0.454 (GENOMIC) <em>vs.</em> 0.354 (NIR1) (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>). For pedigree, the BLUP used the numerical relations matrix (<strong><em>A</em></strong>), which gave prediction accuracies higher than NIR1 and NIR2 but lower than genomic prediction. It is interesting to observe that NIR2 (with a second derivative) improved the prediction accuracy with respect to NIR1 with AK. Results show that combining markers with NIR (either NIR1 or NIR2) increased the prediction accuracy over the accuracy of using markers alone for all the three kernels (GB, GK, and AK). Interestingly, combining pedigree with NIR (either NIR1 or NIR2) gave slightly lower prediction accuracy compared with the prediction using genomic for the three kernels (GB pedigree+NIR1 = 0.420 <em>vs.</em> GENOMIC = 0.424; GK pedigree+NIR1 = 0.425 <em>vs.</em> GENOMIC = 0.454; AK pedigree+NIR1 = 0.413 <em>vs.</em> GENOMIC = 0.456) (<a id="xref-table-wrap-5-3" class="xref-table" href="#T5">Table 5</a>). These results (NIR +pedigree) may be useful as a feasible alternative when markers are not available.</p><p id="p-56">The two combinations with the highest prediction accuracy were kernel AK with GENOMIC + NIR2 (0.466) and kernel AK with GENOMIC+pedigree+NIR2 (0.460). Although the prediction accuracy of NIR2 <em>vs.</em> NIR1 did not change for kernel GK, for kernel AK, NIR2 was better than NIR1. In combination with markers, NIR2 was slightly superior to NIR1 only when used with kernel AK. In combination with pedigree, NIR2 and NIR1 did not change much when used with the different kernels.</p></div></div><div class="section discussion" id="sec-23"><h2 class="">Discussion</h2><p id="p-57">The main objective of this research was to show a kernel method, the Arc-cosine, AK, as an alternative to the Gaussian kernel but avoiding the need to estimate the bandwidth, which requires intense computing resources. The AK kernel also offers an alternative to the use of deep learning that requires the definition of a large number of hyper parameters. However, in this study, we did not compare the genomic-enabled prediction accuracy between AK and deep learning.</p><p id="p-58"><a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Tables 1</a>-<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">4</a> show that the results for WHEAT1, MAIZE2, and MAIZE3 follow the same pattern, that is, non-linear methods (AK, GK) gave significantly higher prediction accuracy than the linear GB kernel, with the exception of the single environment of the MAIZE3 data set. Methods AK are GK gave no significantly different prediction accuracy except for the single-environment model, where AK was significantly better than GK and GB. Nevertheless, in the WHEAT4 data set, the hypothesis that AK is not different from GB and GK is not rejected.</p><p id="p-59">In general, the accuracy of genomic prediction is influenced by: (1) linkage of extensive blocks of chromosomes from parents in the training sets that are passed on to the progeny (for example in F2 bi-parental populations where maximum linkage disequilibrium, LD, is achieved for extensive chromosome blocks), (2) linkage disequilibrium between dense markers and certain QTL that affect the trait in the training and testing sets, and (3) both linkage of extensive blocks and linkage disequilibrium. Usually in self-pollinated crops like wheat, the extent of linkage disequilibrium is important (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Crossa <em>et al.</em> 2007</a>); thus closely linked markers and QTL will remain together through a certain number of generations, and nonlinear models such as GK and AK could be suitable for capturing cryptic locally epistatic effects for prediction accuracy (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akdemir and Jannink 2015</a>). In maize, linkage blocks are shorter than in wheat, so capturing epistatic effects should be more problematic than in wheat. Models that account for these epistatic effects are complex and must have a non-linear component to capture these small hidden relationships between markers. The GK and AK non-linear kernels were much more capable of capturing and exploiting these cryptic and small relationships than the linear kernel GB. For the WHEAT1 and MAIZE2 data sets, kernel AK was slightly but consistently superior in terms of prediction accuracy to GK for both the single-environment and the GE multi-environment models. For the MAIZE3 data set, kernel AK was consistently superior in terms of prediction accuracy to GK for the GE multi-environment models. For data set WHEAT4, there are more cases where GK was superior to the AK kernel.</p><div id="sec-24" class="subsection"><h3>Kernels with genomic information – WHEAT1, MAIZE2, and MAIZE3 data sets</h3><p id="p-60">Parametric genomic models (such as the linear GB) that use kernels constructed with markers allow clear biological and genetic interpretation of the genomic estimates from marker effects and their interactions with the environment accounted for by the reaction norm models (<a id="xref-ref-21-6" class="xref-bibr" href="#ref-21">López-Cruz <em>et al.</em> 2015</a>). These models capture only additive genetic effects and account for additive genetic variance of the measured traits. However, their genomic-enabled prediction is usually lower than that of other non-linear kernels like GK, as demonstrated by <a id="xref-ref-8-7" class="xref-bibr" href="#ref-8">Cuevas <em>et al.</em> (2016</a>, <a id="xref-ref-9-6" class="xref-bibr" href="#ref-9">2017</a>, <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">2018</a>) and <a id="xref-ref-36-7" class="xref-bibr" href="#ref-36">Souza <em>et al.</em> (2017)</a>. Nevertheless, models based on GB can also represent the transformation of one kernel into another kernel. For example, a polynomial kernel of order two of GB represents a non-linear relationship. Methods for transforming one valid kernel into another valid kernel can be found in <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Wilson (2014</a>, p. 40).</p><p id="p-61">The Gaussian kernel (GK) has been used in genomic prediction by <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Crossa <em>et al.</em> (2010)</a>, <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">de los Campos <em>et al.</em> (2010)</a>, and <a id="xref-ref-8-8" class="xref-bibr" href="#ref-8">Cuevas <em>et al.</em> (2016</a>, <a id="xref-ref-9-7" class="xref-bibr" href="#ref-9">2017</a>, <a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">2018</a>) and can model cryptic relationships between markers that usually improve prediction accuracy. This is partly due to the fact that matrix GK can model non-linear relationships between lines because the bandwidth parameter models these relationships as estimated in the training set through cross-validation or by maximizing the marginal likelihood (<a id="xref-ref-31-6" class="xref-bibr" href="#ref-31">Pérez-Elizalde <em>et al.</em> 2015</a>). However, selecting the bandwidth has two inconveniences: it requires intensive computing time and assumes that the training set can determine the true optimum value of the bandwidth. These inconveniences can be solved in part by scaling the Euclidean distances by their median (<a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Crossa <em>et al.</em> 2010</a>); this gives good results particularly when the markers are standardized. The similarity measurement used in GK, the squared Euclidean distance, can be interpreted as the frequency of the differences in the presence of markers. <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Morota and Gianola (2014)</a> mentioned that no other kernel has higher prediction.</p></div><div id="sec-25" class="subsection"><h3>The AK kernel</h3><p id="p-62">Results of this study show that for some data sets, the AK kernel may perform the same as or slightly better than the GK kernel. These results are of interest for exploring hidden and small marker relationships using a similarity measurement different to that employed by the GK. The AK uses the magnitude of the vectors between two lines and the angle induced by the interior product in its characteristic space; this repetition generates a recursive formula for exploring hidden relationships and simulating layers. In this study, we used equations (1) and (2) to construct the AK that maximizes the marginal distribution. Equation (1) generates the basic matrix and in practice, it consumes less computing time than the required for GK when computing the distances between individuals based on marker. The optimal level (number of layers) depends on the training data sets and varies for each data set. For example, <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a> for the WHEAT1 data set shows that the prediction accuracy increases as more hidden layers are included, until reaching a maximum that could correspond to the level that maximizes the marginal likelihood. <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> shows that for WHEAT1, the increase in prediction accuracy is related to the increase in the maximum of the marginal likelihood. Every environment can have a different pattern of marginal likelihood; for example, in E2, the maximum values of the marginal likelihood are around 3 layers, whereas in the other environments, the maximum values of the marginal likelihood are around 10 levels (layers). However, <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> shows that for WHEAT1 and the GE multi-environment model, the level of layers used was 6; this is the level that maximizes the marginal likelihood when considering the main effects (the optimum number of layers can change depending on whether the marker matrix is or not scaled). It should be noted that for random cross-validation CV2, the exchange (borrowing) of information between environments occurs throughout the genomic main effects. <a id="xref-ref-4-11" class="xref-bibr" href="#ref-4">Cho and Saul (2009)</a> proposed an algorithm where the output from one level is the input to another level, mimicking the deep learning neural network algorithm that is explained in <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Montesinos-López <em>et al.</em> (2018a</a>,<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">b</a>; 2019a,<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">b</a>).</p><p id="p-63">As previously mentioned, for selecting level <em>l</em>, we initially computed the marginal likelihood, moved one more level and recalculated the marginal likelihood, and continued the process until there was no further improvement in the likelihood. Thus the data are fitted with this last <em>l</em> level. There are other alternatives that could be studied, for example, considering other basic functions as <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-71.gif"/></noscript></span></span>, as proposed by Cho and Saul (2009). Another alternative would be to fit the data with <em>l</em> = 1 and prepare the output to be used as input for the next level. We have tried this approach with varying results; nevertheless, further research is required. In the future, research is needed to compare the prediction accuracy of AK including different layers with the deep learning artificial neural network method studied by Montesinos-López <em>et al.</em> (2018 a,<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">b</a>); this type of study would require comparing AK (having different numbers of layers) with single-environment and GE multi-environment models with deep learning neural networks (having different numbers of layers) under both models.</p></div><div id="sec-26" class="subsection"><h3>Kernels with genomic and near infrared plus pedigree information WHEAT4 data set</h3><p id="p-64">As proposed by <a id="xref-ref-33-6" class="xref-bibr" href="#ref-33">Rincent <em>et al.</em> (2018)</a>, in this study we used the near infrared (NIR) method as a high-throughput method for constructing relationship matrices based on the reflectance of various wavelengths applied to sample seed of different bread wheat lines. Reflectance should be related to some chemical properties of seed samples for numerous wavelengths. <a id="xref-ref-33-7" class="xref-bibr" href="#ref-33">Rincent <em>et al.</em> (2018)</a> proposed evaluating the efficiency of the NIR for making predictions utilizing the NIRs as markers (NIR BLUP), similar to what the GBLUP does in conventional genomic prediction. Here the similarity between lines is given by the NIR wavelengths. <a id="xref-ref-33-8" class="xref-bibr" href="#ref-33">Rincent <em>et al.</em> (2018)</a> proposed using NIRs for phenomic selection defined as high-throughput phenotyping, thus obtaining a large number of variables (wavelength) as regressors in statistical models similar to those used in GS. The authors noted that as opposed to classical methodology, where NIRS collected on a sample serve to make predictions about that particular sample, their proposition is to make predictions for other samples (<em>e.g.</em>, environments). <a id="xref-ref-33-9" class="xref-bibr" href="#ref-33">Rincent <em>et al.</em> (2018)</a> found for several traits in wheat and a tree species (poplar) that the NIR BLUP prediction accuracies from grains, leaves, and wood are better than genomic prediction accuracy.</p><p id="p-65">Our results using NIR, pedigree, and markers show that using only NIR does give similar prediction accuracy as when using pedigree, but lower prediction accuracy than when using markers. In general, the accuracies of genomic-enabled prediction observed in this study are similar to those previously reported for grain yield of CIMMYT elite wheat lines. Markers gave higher correlations between observed and predicted values than pedigree and NIR. Interestingly, pedigree + NIR gave only slightly lower prediction accuracy (0.413-0.425) than marker + NIR (0.435-0.466) and similar prediction accuracy to prediction accuracies using markers, pedigree and NIR together (0.448-0.462) (<a id="xref-table-wrap-5-4" class="xref-table" href="#T5">Table 5</a>). It is likely that the main reason for this is that the wheat lines used in this study are elite lines with not much grain yield differences between them.</p><p id="p-66">In other work <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Hayes <em>et al.</em> (2017)</a> used genomic, NIR and nuclear magnetic resonance (NMR) for the prediction of grain and end-use traits that usually comprise low samples with small amounts of flour. Under these circumstances, it is important to use additional information to complement genomic-enabled predictions. The authors found that incorporating NIR and NMR into the genomic multi-trait model increased the accuracies from 0 to 0.47 to 0-0.69 for 19 end-use quality traits. In our study, the inclusion of NIR increased the prediction accuracy of the GBLUP genomic model, especially for the AK kernel (0.456 Pearson’s correlation for GBLUP <em>vs.</em> 0.462 and 0.66 for GBLUP+NIR1 and GBLUP+NIR2, respectively; <a id="xref-table-wrap-5-5" class="xref-table" href="#T5">Table 5</a>). Interestingly, for the GK model there was almost no increase in accuracy of GBLUP <em>vs.</em> GBLUP+NIR1 and GBLUP+NIR2.</p><p id="p-67">The results from the WHEAT4 data set are preliminary and used on highly elite wheat lines as compared to those of <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Hayes <em>et al.</em> (2017)</a>, which included highly variable wheat accessions, and the study of <a id="xref-ref-33-10" class="xref-bibr" href="#ref-33">Rincent <em>et al.</em> (2018)</a>, which comprised a highly diverse historical panel of wheat lines from many years. However, our results agree with previous studies where the inclusion of NIR did increase the prediction accuracies of the lines in the testing set when using markers, when using pedigree and when using both. An important result is that the use of pedigree and NIR offers an opportunity to increase prediction accuracy similar to that obtained when using markers plus NIR (at a higher cost).</p></div><div id="sec-27" class="subsection"><h3>Conclusions</h3><p id="p-68">In this study we used three kernel methods for genomic prediction: the linear genomic best linear unbiased predictor (GBLUP) kernel (GB), the non-linear Gaussian kernel (GK) and the non-linear Arc-cosine kernel (AK). The AK emulates deep learning artificial neural networks and is required to define the <em>l</em> levels that define the number of layers. The three data sets included only genomic information, whereas one data set included near infrared spectroscopy (NIR), genomic information, and pedigree information. Our study included two statistical models: the single-environment and the genomic × environment (GE) models. For the four data sets, methods AK and GK achieved higher prediction accuracy than the linear GB kernel for the single-environment model, as well as the GE multi-environment model. Interestingly, AK achieved similar or slightly higher prediction accuracy than the GK kernel. For data set WHEAT4 (with pedigree, genomic information and NIR), NIR wavelength alone achieved less prediction accuracy than the genomic information alone. Interestingly, pedigree + NIR information achieved slightly lower prediction accuracy than genomic information + NIR. We conclude that the AK kernel has the advantage over the GK, which does need to estimate the bandwidth parameter while producing similar prediction accuracies.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-72">We thank all scientists, field workers, and lab assistants from National Programs and CIMMYT who collected the data used in this study. We are also thankful for the financial support provided by CIMMYT CGIAR CRP (maize and wheat), the Bill &amp; Melinda Gates Foundation, as well the USAID projects (Cornell University and Kansas State University) that generated the CIMMYT maize and wheat data analyzed in this study. We acknowledge the financial support provided by the Foundation for Research Levy on Agricultural Products (FFL) and the Agricultural Agreement Research Fund (JA) in Norway through NFR grant 267806. The authors thank Dr. Morten Lillemo for his valuable comments on the content of the manuscript.</p></div><div class="section app" id="app-1"><h2 class="">Appendix</h2><p id="p-69">For simplicity, assume model <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-72.gif"/></noscript></span></span>, where <strong><em>y</em></strong> is centered at zero, <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-73.gif"/></noscript></span></span>) and <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-74.gif"/></noscript></span></span><strong>)</strong>. The marginal distribution is the integral over all the parameters (noise) of the posterior distribution:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-75.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-76.gif"/></noscript></span></span>The first step is to integrate over vector <strong><em>u</em></strong>, after eigen values decomposition of matrix <strong><em>K</em></strong>, such that <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-77.gif"/></noscript></span></span>, where <strong><em>U</em></strong> is an orthogonal matrix of the eigen vectors and <strong><em>S</em></strong> is a diagonal matrix with the eigen values on its diagonal order from the highest to the lowest. Vector <strong><em>b</em></strong> is transformed as <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-78.gif"/></noscript></span></span> and assuming <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-79.gif"/></noscript></span></span> then the distribution of <strong><em>b</em></strong> is <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-80.gif"/></noscript></span></span></p><p id="p-70">To facilitate the computations, the integration is along <strong><em>b</em></strong> instead of <strong><em>u</em></strong><strong>:</strong><span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-81.gif"/></noscript></span></span>Completing the rest of the terms are<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-82.gif"/></noscript></span></span>where<span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-83.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-84.gif"/></noscript></span></span></p><p id="p-71">Now if we consider that the prior distribution of <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-85.gif"/></noscript></span></span> is <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-86.gif"/></noscript></span></span>, then<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-87.gif"/></noscript></span></span><span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-88.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-89.gif"/></noscript></span></span> is the <em>i</em><sup>th</sup> value of <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2913/embed/mml-math-90.gif"/></noscript></span></span>.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-8"><p id="p-73"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-04-28"><span class="received-label">Received </span>April 28, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-07-04"><span class="accepted-label">Accepted </span>July 4, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Cuevas <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.1" data-doi="10.1534/genetics.114.173658"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>,  <span class="cit-name-given-names">D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Locally epistatic genomic relationship matrices for genomic association and prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">857</span>–<span class="cit-lpage">871</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.173658</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.173658%26rft_id%253Dinfo%253Apmid%252F25614606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk5LzMvODU3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI5MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.2" data-doi="10.1016/j.ajhg.2009.01.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>,  <span class="cit-name-given-names">B. L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Browning</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">210</span>–<span class="cit-lpage">223</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2009.01.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DB.%2BL.%26rft.volume%253D84%26rft.issue%253D2%26rft.spage%253D210%26rft.epage%253D223%26rft.atitle%253DA%2Bunified%2Bapproach%2Bto%2Bgenotype%2Bimputation%2Band%2Bhaplotype-phase%2Binference%2Bfor%2Blarge%2Bdata%2Bsets%2Bof%2Btrios%2Band%2Bunrelated%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2009.01.005%26rft_id%253Dinfo%253Apmid%252F19200528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2009.01.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19200528&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263799700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.3" data-doi="10.2135/cropsci2011.06.0299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de Los Campos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Weigel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">719</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2011.06.0299</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D707%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. K.</span> <span class="cit-name-surname">Saul</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> Kernel Methods for Deep Learning. NIPS’09 Proceedings of the 22nd International Conference on Neural Information Processing Systems, 342–350.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.5" data-doi="10.1534/genetics.107.078659"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Burgueño</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Dreisigacker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Vargas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Herrera-Foessel</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> Association analysis of historical bread wheat germplasm using additive genetic covariance of relatives and population structure. Genetics 177: 1889–1913. doi. ISSN: 0016–6731. <span class="cit-pub-id cit-pub-id-doi">10.1534/genetics.107.078659</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.078659%26rft_id%253Dinfo%253Apmid%252F17947425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny8zLzE4ODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjkxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.6" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Burgueño</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.118521</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNzEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI5MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.7" data-doi="10.1016/j.tplants.2017.08.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez-Rodríguez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Cuevas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Montesinos-López</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Jarquín</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic Selection in Plant Breeding: Methods, Models, and Perspectives.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">961</span>–<span class="cit-lpage">975</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2017.08.011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D22%26rft.spage%253D961%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2017.08.011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2017.08.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.8" data-doi="10.3835/plantgenome2016.03.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Soberanis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Pérez-Elizalde</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez-Rodríguez</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction of genotype × environment interaction kernel regression models.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">20</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.03.0024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D9%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.03.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.03.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Montesinos-Lopez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Burgueno</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Perez-Rodriguez</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> Bayesian Genomic Prediction with Genotype × Environment Interaction Kernel Models. G3 (Bethesda) 7: 41–53.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Granato</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Fritsche-Neto</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O. A.</span> <span class="cit-name-surname">Montesinos-Lopez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Burgueño</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> Genomic-enabled prediction Kernel models with random intercepts for multi-environment trials. G3 (Bethesda) 8: 1347–1365.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.11" data-doi="10.1534/genetics.109.101501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Naya</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Legarra</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting quantitative traits with regression models for dense molecular markers and pedigree.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">385</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.101501</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101501%26rft_id%253Dinfo%253Apmid%252F19293140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI5MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.12" data-doi="10.1017/S0016672310000285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. J.</span> <span class="cit-name-surname">Rosa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Weigel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">308</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672310000285</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DDE%2BLos%2BCampos%26rft.auinit1%253DG.%26rft.volume%253D92%26rft.issue%253D4%26rft.spage%253D295%26rft.epage%253D308%26rft.atitle%253DSemi-parametric%2Bgenomic-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Breproducing%2Bkernel%2BHilbert%2Bspaces%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000285%26rft_id%253Dinfo%253Apmid%252F20943010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20943010&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.13" data-doi="10.1534/genetics.107.084285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>,  <span class="cit-name-given-names">D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B. C. H. M.</span> <span class="cit-name-surname">Van Kaam</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Reproducing kernel Hilbert spaces regression methods for genomic assisted prediction of quantitative traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">2289</span>–<span class="cit-lpage">2303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.084285</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGianola%26rft.auinit1%253DD.%26rft.volume%253D178%26rft.issue%253D4%26rft.spage%253D2289%26rft.epage%253D2303%26rft.atitle%253DReproducing%2BKernel%2BHilbert%2BSpaces%2BRegression%2BMethods%2Bfor%2BGenomic%2BAssisted%2BPrediction%2Bof%2BQuantitative%2BTraits%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.084285%26rft_id%253Dinfo%253Apmid%252F18430950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC80LzIyODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjkxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.14" data-doi="10.1534/genetics.113.151753"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>,  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Priors in whole-genome regression: The Bayesian alphabet returns.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">596</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.151753</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.151753%26rft_id%253Dinfo%253Apmid%252F23636739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI5MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.15" data-doi="10.1007/s00122-012-1868-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">González-Camacho</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Cairns</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-enabled prediction of genetic values using radial basis function neural networks.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">759</span>–<span class="cit-lpage">771</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-012-1868-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DGonzalez-Camacho%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D125%26rft.issue%253D4%26rft.spage%253D759%26rft.epage%253D771%26rft.atitle%253DGenome-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Bradial%2Bbasis%2Bfunction%2Bneural%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1868-9%26rft_id%253Dinfo%253Apmid%252F22566067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1868-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22566067&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.9.2913.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">González-Camacho</span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Ornella</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Daniel</span> <span class="cit-name-surname">Gianola</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> Genomic-enabled prediction using probabilistic neural network classifiers. BMC genomic 17(1)208.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Granato</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Cuevas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Luna</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Montesinos</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> BGGE: A new package for genomic-enabled prediction incorporating genotype x environment. G3 (Bethesda) 8: 3039–3047.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.18" data-doi="10.1007/s00122-017-2972-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Panozzo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. K.</span> <span class="cit-name-surname">Walker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Choy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kant</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Accelerating wheat breeding for end use quality with multi-trait genomic predictions incorporating near infrared and nuclear magnetic resonance derived phenotypes.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">2505</span>–<span class="cit-lpage">2519</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-017-2972-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D2505%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-017-2972-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-017-2972-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.19" data-doi="10.1007/s00122-013-2243-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Lacaze</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Du Cheyron</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Daucourt</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A reaction norm model for genomic selection using high-dimensional genomic and environmental data.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">607</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-013-2243-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D595%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2243-1%26rft_id%253Dinfo%253Apmid%252F24337101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2243-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24337101&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.20" data-doi="10.1534/g3.117.300548"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">Y</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.H.</span> <span class="cit-name-surname">Schmidt</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.C.</span> <span class="cit-name-surname">Reif</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> Haplotype-Based Genome-Wide Prediction Models Exploit Local Epistatic Interactions among markers. G3 (Bethesda) 8:1687–1699.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.300548</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.300548%26rft_id%253Dinfo%253Apmid%252F29549092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjgvNS8xNjg3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI5MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">López-Cruz</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Bonnett</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Dreisigacker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Poland</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> Increased Prediction Accuracy in Wheat Breeding Trials Using a Marker × Environment Interaction Genomic Selection Model. G3 (Bethesda) 5: 569–82.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.22" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>,  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Hayes</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjkxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O. A.</span> <span class="cit-name-surname">Montesinos-López</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Hernández-Suárez</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">a</span> <span class="cit-article-title">Multi-environment Genomic Prediction of Plant Traits Using Deep Learners With Dense Architecture. G3</span>. <abbr class="cit-jnl-abbrev">Genes Genomes Genetics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">3811</span>–<span class="cit-lpage">3828</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BGenomes%2BGenetics%26rft.volume%253D8%26rft.spage%253D3811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>,  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Montesinos-López</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Hernández-Suárez</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">b</span> <span class="cit-article-title">Multi-trait, multi-enviroment deep learning modeling for genomic-enabled prediction of plant traits. G3</span>. <abbr class="cit-jnl-abbrev">Genes Genomes Genetics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">3829</span>–<span class="cit-lpage">3840</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BGenomes%2BGenetics%26rft.volume%253D8%26rft.spage%253D3829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.9.2913.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>,  <span class="cit-name-given-names">O.A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Martín-Vallejo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span></li></ol><cite>,Hernández-Suárez, C.M., <em>et al.</em>, 2019a New deep learning genomic-based prediction model for multiple traits with binary, ordinal, and continuous phenotypes. G3 (Bethesda) g3.300585.<span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>,  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Martín-Vallejo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Hernández-Suárez</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span><span class="cit-comment">b</span> <span class="cit-article-title">A bench marking between deep learning, support vector machines and Bayesian threshold best linear unbiased prediction for predicting ordinal traits in plant breeding. G3</span> (Bethesda) <span class="cit-vol">9</span>: <span class="cit-fpage">601</span>–<span class="cit-lpage">618</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.27" data-doi="10.3389/fgene.2014.00001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morota</span>,  <span class="cit-name-given-names">G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Kernel-based whole-genome prediction of complex traits: A review.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2014.00363</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D5%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2014.00001%26rft_id%253Dinfo%253Apmid%252F24567736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2014.00001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24567736&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-book" id="cit-9.9.2913.28" data-doi="10.1007/978-1-4612-0745-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neal</span>,  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source">Bayesian Learning for Neural Networks</span>, <span class="cit-publ-name">Springer Verlag</span>, <span class="cit-publ-loc">New York</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-4612-0745-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBayesian%2BLearning%2Bfor%2BNeural%2BNetworks%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4612-0745-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4612-0745-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.29" data-doi="10.3835/plantgenome2012.07.0017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ornella</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Singh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez-Rodríguez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Burgueño</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Singh</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of genetic values for resistance to wheat rusts.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">136</span>–<span class="cit-lpage">148</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2012.07.0017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D5%26rft.spage%253D136%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.07.0017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.07.0017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.30" data-doi="10.1038/hdy.2013.144"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ornella</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez-Rodríguez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Tapia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">González-Camacho</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Burgueño</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic-enabled prediction with classification algorithms.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">616</span>–<span class="cit-lpage">626</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2013.144</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D112%26rft.spage%253D616%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2013.144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2013.144&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.31" data-doi="10.1007/s13253-015-0229-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Elizalde</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Cuevas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez-Rodríguez</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Selection of the Bandwidth Parameter in a Bayesian Kernel Regression Model for Genomic-Enabled Prediction.</span> <abbr class="cit-jnl-abbrev">J. Agric. Biol. Environ. Stat.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">512</span>–<span class="cit-lpage">532</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s13253-015-0229-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BBiol.%2BEnviron.%2BStat.%26rft.volume%253D20%26rft.spage%253D512%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs13253-015-0229-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s13253-015-0229-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-web" id="cit-9.9.2913.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2018</span> R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. URL <a href="https://www.R-project.org/">https://www.R-project.org/</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rincent</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Charpentier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Faivre-Rampant</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Paux</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Le Gouis</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Phenomic Selection Is a Low Cost and High-Throughput Method Based on Indirect Predictions: Proof of Concepts on Wheat and Poplar. G3</span>. <abbr class="cit-jnl-abbrev">Genes Genomes Genetics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">3961</span>–<span class="cit-lpage">3972</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BGenomes%2BGenetics%26rft.volume%253D8%26rft.spage%253D3961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.34" data-doi="10.1147/rd.33.0210"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Samuel</span>,  <span class="cit-name-given-names">A. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1959</span> <span class="cit-article-title">Some Studies in Machine Learning Using the Game of Checkers.</span> <abbr class="cit-jnl-abbrev">IBM J. Res. Develop.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">210</span>–<span class="cit-lpage">229</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1147/rd.33.0210</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIBM%2BJ.%2BRes.%2BDevelop.%26rft.volume%253D3%26rft.spage%253D210%26rft_id%253Dinfo%253Adoi%252F10.1147%252Frd.33.0210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1147/rd.33.0210&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.35" data-doi="10.1021/ac60214a047"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Savitzky</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J. E.</span> <span class="cit-name-surname">Golay</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">Smoothing and differentiation of data by simplified least squares procedures.</span> <abbr class="cit-jnl-abbrev">Anal. Chem.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1627</span>–<span class="cit-lpage">1639</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1021/ac60214a047</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnal.%2BChem.%26rft.volume%253D36%26rft.spage%253D1627%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fac60214a047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ac60214a047&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Souza</span>,  <span class="cit-name-given-names">M.B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Cuevas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.G.</span> <span class="cit-name-surname">de O. Couto</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez-Rodríguez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Jarquín</span></span>, <span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2017</span> Genomic-Enabled Prediction in Maize Using Kernel Models with Genotype × Environment Interaction. G3 (Bethesda) g3.117.042341.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-9.9.2913.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Ramirez-Lopez</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> An introduction propectr package. <a href="https://cran.r-project.org/web/packages/prospectr/vignettes/prospectr-intro.pdf">https://cran.r-project.org/web/packages/prospectr/vignettes/prospectr-intro.pdf</a>. March, 2019.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.38" data-doi="10.1007/s00122-012-1905-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Riedelsheimer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. A.</span> <span class="cit-name-surname">Schrag</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Melchinger</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of hybrid performance in maize with models incorporating dominance and population specific marker effects.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">1181</span>–<span class="cit-lpage">1194</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-012-1905-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DTechnow%26rft.auinit1%253DF.%26rft.volume%253D125%26rft.issue%253D6%26rft.spage%253D1181%26rft.epage%253D1194%26rft.atitle%253DGenomic%2Bprediction%2Bof%2Bhybrid%2Bperformance%2Bin%2Bmaize%2Bwith%2Bmodels%2Bincorporating%2Bdominance%2Band%2Bpopulation%2Bspecific%2Bmarker%2Beffects.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1905-8%26rft_id%253Dinfo%253Apmid%252F22733443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1905-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22733443&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.39" data-doi="10.1186/1471-2164-15-823"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Unterseer</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Bauer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Haberer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Seidel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Knaak</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A powerful tool for genome analysis in maize: development and evaluation of the high density 600 k SNP genotypin array.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">823</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-823</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D823%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-823%26rft_id%253Dinfo%253Apmid%252F25266061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-823&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25266061&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>,  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> Covariance Kernels for Fast Automatic Pattern Discovery and Extrapolation with Gaussian Processes. PhD thesis. University of Cambridge.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2913.41" data-doi="10.1093/bioinformatics/bts335"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>,  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Albrecht</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. J.</span> <span class="cit-name-surname">Auinger</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Schön</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Synbreed framework for analysis of genomic prediction data using R.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2086</span>–<span class="cit-lpage">2087</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts335</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D28%26rft.spage%253D2086%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts335&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/9/2913.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/9/2905" title="Validation and Characterization of Maize Multiple Disease Resistance QTL" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/9/2925" title="Joint Use of Genome, Pedigree, and Their Interaction with Environment for Predicting the Performance of Wheat Lines in New Environments" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/9"><span>Volume 9 Issue 9, September 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/9.cover-source.jpg?itok=4kZTDz_V" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/2913.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-TfFRYAZYkS_vJIz7cm69852YqyhD18TXvEuwVUJVDcQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/9/2913" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/9/2913" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/9/2913" class="active" data-icon-position="" data-hide-link-title="0">Deep Kernel for Genomic and Near Infrared Predictions in Multi-environment Breeding Trials</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494051" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-dANHNmnrK6tDzu77DAIfCtl_p47qVs4LdNnBIYMvemg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/9/2913.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/9/2913" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-8TEOsl4cBOcPbJUOHovGPKuvvlxJlG6F0mOjx78NKPo" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/9/2913" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/9/2913.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494051" id="node494051--2" data-pisa="ggg;9/9/2913" data-pisa-master="ggg;g3.119.400493" data-apath="/ggg/9/9/2913.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Deep Kernel for Genomic and Near Infrared Predictions in Multi-environment Breeding Trials</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0685-2867" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jaime Cuevas</span>, <span class="highwire-citation-author" data-delta="1">Osval Montesinos-López</span>, <span class="highwire-citation-author" data-delta="2">Philomin Juliana</span>, <span class="highwire-citation-author" data-delta="3">Carlos Guzmán</span>, <span class="highwire-citation-author" data-delta="4">Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author" data-delta="5">José González-Bucio</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-1468-4867" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Juan Burgueño</span>, <span class="highwire-citation-author" data-delta="7">Abelardo Montesinos-López</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2913-2924;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400493 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494051--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jaime Cuevas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Universidad de Quintana Roo, Chetumal, Quintana Roo, 77019 México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaime%2BCuevas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cuevas%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJaime%2BCuevas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0685-2867" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jaime Cuevas</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Osval Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Facultad de Telemática, Universidad de Colima, C. P. 28040, Edo. de Colima, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Philomin Juliana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Carretera Mexico- Veracruz Km. 45, El Batán, 56237, Texcoco, Edo. de Mexico, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philomin%2BJuliana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Juliana%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhilomin%2BJuliana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Carlos Guzmán </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Carretera Mexico- Veracruz Km. 45, El Batán, 56237, Texcoco, Edo. de Mexico, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carlos%2BGuzm%C3%A1n%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guzm%C3%A1n%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarlos%2BGuzm%25C3%25A1n%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Colegio de Postgraduados, Montecillos 56230, Edo. de Mexico, Mexico, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">José González-Bucio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Universidad de Quintana Roo, Chetumal, Quintana Roo, 77019 México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BGonz%C3%A1lez-Bucio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gonz%C3%A1lez-Bucio%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BGonz%25C3%25A1lez-Bucio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Carretera Mexico- Veracruz Km. 45, El Batán, 56237, Texcoco, Edo. de Mexico, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1468-4867" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Juan Burgueño</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías, (CUCEI), Universidad de Guadalajara, Guadalajara, Jalisco, 44430</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Carretera Mexico- Veracruz Km. 45, El Batán, 56237, Texcoco, Edo. de Mexico, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/9/2913" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/9/2913&amp;title=Deep%20Kernel%20for%20Genomic%20and%20Near%20Infrared%20Predictions%20in%20Multi-environment%20Breeding%20Trials" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/9/2913&amp;title=Deep%20Kernel%20for%20Genomic%20and%20Near%20Infrared%20Predictions%20in%20Multi-environment%20Breeding%20Trials" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/9/2913&amp;title=Deep%20Kernel%20for%20Genomic%20and%20Near%20Infrared%20Predictions%20in%20Multi-environment%20Breeding%20Trials" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/9/2913&amp;text=Deep%20Kernel%20for%20Genomic%20and%20Near%20Infrared%20Predictions%20in%20Multi-environment%20Breeding%20Trials" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/9/2913&amp;title=Deep%20Kernel%20for%20Genomic%20and%20Near%20Infrared%20Predictions%20in%20Multi-environment%20Breeding%20Trials" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/9/2913&amp;t=Deep%20Kernel%20for%20Genomic%20and%20Near%20Infrared%20Predictions%20in%20Multi-environment%20Breeding%20Trials" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/9/2913&amp;title=Deep%20Kernel%20for%20Genomic%20and%20Near%20Infrared%20Predictions%20in%20Multi-environment%20Breeding%20Trials" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/9/2913&amp;title=Deep%20Kernel%20for%20Genomic%20and%20Near%20Infrared%20Predictions%20in%20Multi-environment%20Breeding%20Trials" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494051/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494051" id="node494051--3" data-pisa="ggg;9/9/2913" data-pisa-master="ggg;g3.119.400493" data-apath="/ggg/9/9/2913.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Deep Kernel for Genomic and Near Infrared Predictions in Multi-environment Breeding Trials</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0685-2867" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jaime Cuevas</span>, <span class="highwire-citation-author" data-delta="1">Osval Montesinos-López</span>, <span class="highwire-citation-author" data-delta="2">Philomin Juliana</span>, <span class="highwire-citation-author" data-delta="3">Carlos Guzmán</span>, <span class="highwire-citation-author" data-delta="4">Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author" data-delta="5">José González-Bucio</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-1468-4867" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Juan Burgueño</span>, <span class="highwire-citation-author" data-delta="7">Abelardo Montesinos-López</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2913-2924;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400493 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494051--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jaime Cuevas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Universidad de Quintana Roo, Chetumal, Quintana Roo, 77019 México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaime%2BCuevas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cuevas%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJaime%2BCuevas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0685-2867" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jaime Cuevas</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Osval Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Facultad de Telemática, Universidad de Colima, C. P. 28040, Edo. de Colima, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Philomin Juliana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Carretera Mexico- Veracruz Km. 45, El Batán, 56237, Texcoco, Edo. de Mexico, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philomin%2BJuliana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Juliana%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhilomin%2BJuliana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Carlos Guzmán </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Carretera Mexico- Veracruz Km. 45, El Batán, 56237, Texcoco, Edo. de Mexico, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carlos%2BGuzm%C3%A1n%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guzm%C3%A1n%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarlos%2BGuzm%25C3%25A1n%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Colegio de Postgraduados, Montecillos 56230, Edo. de Mexico, Mexico, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">José González-Bucio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Universidad de Quintana Roo, Chetumal, Quintana Roo, 77019 México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BGonz%C3%A1lez-Bucio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gonz%C3%A1lez-Bucio%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BGonz%25C3%25A1lez-Bucio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Carretera Mexico- Veracruz Km. 45, El Batán, 56237, Texcoco, Edo. de Mexico, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1468-4867" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Juan Burgueño</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías, (CUCEI), Universidad de Guadalajara, Guadalajara, Jalisco, 44430</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Carretera Mexico- Veracruz Km. 45, El Batán, 56237, Texcoco, Edo. de Mexico, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2913&amp;rft.epage=2924&amp;rft.atitle=Deep%20Kernel%20for%20Genomic%20and%20Near%20Infrared%20Predictions%20in%20Multi-environment%20Breeding%20Trials&amp;rft.volume=9&amp;rft.issue=9&amp;rft.date=2019-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Cuevas%2C+Jaime&amp;rft.au=Montesinos-L%C3%B3pez%2C+Osval&amp;rft.au=Juliana%2C+Philomin&amp;rft.au=Guzm%C3%A1n%2C+Carlos&amp;rft.au=P%C3%A9rez-Rodr%C3%ADguez%2C+Paulino&amp;rft.au=Gonz%C3%A1lez-Bucio%2C+Jos%C3%A9&amp;rft.au=Burgue%C3%B1o%2C+Juan&amp;rft.au=Montesinos-L%C3%B3pez%2C+Abelardo&amp;rft.au=Crossa%2C+Jos%C3%A9"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494051/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494051/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494051/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494051/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494051/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494051/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494051/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494051/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494051/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494051/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494051/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494051/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/9/2913" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and Methods</a></li><li class="odd"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-23" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/9/2913.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/9/2913.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/9/2913.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
