<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/9/2935" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/9/2935.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/9/2935.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/9/2935.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/9/2935/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/9/2935.atom" />
<meta name="HW.pisa" content="ggg;9/9/2935" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Use of Repeated Group Measurements with Drop Out Animals for Variance Component Estimation and Genetic Evaluation: A Simulation Study" />
<meta name="DC.Identifier" content="10.1534/g3.119.400484" />
<meta name="DC.Date" content="2019-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2019 Gao et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The efficiency of feed utilization plays an important role in animal breeding. However, measuring feed intake (FI) is costly on an individual basis under practical conditions. Using group measurements to model FI could be practically feasible and cost-effective. The objectives of this study were to develop a random regression model based on repeated group measurements with consideration of missing phenotypes caused by drop out animals. Focus is on variance components (VC) estimation and genetic evaluation, and to investigate the effect of group composition on VC estimation and genetic evaluation using simulated datasets. Data were simulated based on individual FI in a pig population. Each individual had measurement on FI at 6 different time points, reflecting 6 different weeks during the test period. The simulated phenotypes consisted of additive genetic, permanent environment, and random residual effects. Additive genetic and permanent environmental effects were both simulated and modeled by first order Legendre polynomials. Three grouping scenarios based on genetic relationships among the group members were investigated: (1) medium within and across pen genetic relationship; (2) high within group relationship; (3) low within group relationship. To investigate the effect of the drop out animals during test period, a proportion (15%) of animals with individual phenotypes was set as the drop out animals, and two drop out scenarios within each grouping scenario were assessed: (1) animals were randomly dropped out; (2) animals with lower phenotypes were dropped out based on the ranking at each time point. The results show that using group measurements yielded similar VCs estimates but with larger SDs compared with the corresponding scenario of using individual measurements. Compared to scenarios without drop out, similar VC estimates were observed when animals were dropped out randomly, whereas reduced VC estimates were observed when animals were dropped out by the ranking of phenotypes. Different grouping scenarios produced similar VC estimates. Compared to scenarios without drop out, there were no loss of accuracies of genetic evaluation for drop out scenarios. However, dropping out animals by the ranking of phenotypes produced larger bias of estimated breeding values compared to the scenario without dropped out animals and scenario of dropping out animals by random. In conclusion, with an optimized group structure, the developed model can properly handle group measurements with drop out animals, and can achieve comparable accuracy of genetic evaluation for traits measured at the group level." />
<meta name="DC.Contributor" content="Hongding Gao" />
<meta name="DC.Contributor" content="Bjarne Nielsen" />
<meta name="DC.Contributor" content="Guosheng Su" />
<meta name="DC.Contributor" content="Per Madsen" />
<meta name="DC.Contributor" content="Just Jensen" />
<meta name="DC.Contributor" content="Ole F. Christensen" />
<meta name="DC.Contributor" content="Tage Ostersen" />
<meta name="DC.Contributor" content="Mahmoud Shirali" />
<meta name="article:published_time" content="2019-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Use of Repeated Group Measurements with Drop Out Animals for Variance Component Estimation and Genetic Evaluation: A Simulation Study" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The efficiency of feed utilization plays an important role in animal breeding. However, measuring feed intake (FI) is costly on an individual basis under practical conditions. Using group measurements to model FI could be practically feasible and cost-effective. The objectives of this study were to develop a random regression model based on repeated group measurements with consideration of missing phenotypes caused by drop out animals. Focus is on variance components (VC) estimation and genetic evaluation, and to investigate the effect of group composition on VC estimation and genetic evaluation using simulated datasets. Data were simulated based on individual FI in a pig population. Each individual had measurement on FI at 6 different time points, reflecting 6 different weeks during the test period. The simulated phenotypes consisted of additive genetic, permanent environment, and random residual effects. Additive genetic and permanent environmental effects were both simulated and modeled by first order Legendre polynomials. Three grouping scenarios based on genetic relationships among the group members were investigated: (1) medium within and across pen genetic relationship; (2) high within group relationship; (3) low within group relationship. To investigate the effect of the drop out animals during test period, a proportion (15%) of animals with individual phenotypes was set as the drop out animals, and two drop out scenarios within each grouping scenario were assessed: (1) animals were randomly dropped out; (2) animals with lower phenotypes were dropped out based on the ranking at each time point. The results show that using group measurements yielded similar VCs estimates but with larger SDs compared with the corresponding scenario of using individual measurements. Compared to scenarios without drop out, similar VC estimates were observed when animals were dropped out randomly, whereas reduced VC estimates were observed when animals were dropped out by the ranking of phenotypes. Different grouping scenarios produced similar VC estimates. Compared to scenarios without drop out, there were no loss of accuracies of genetic evaluation for drop out scenarios. However, dropping out animals by the ranking of phenotypes produced larger bias of estimated breeding values compared to the scenario without dropped out animals and scenario of dropping out animals by random. In conclusion, with an optimized group structure, the developed model can properly handle group measurements with drop out animals, and can achieve comparable accuracy of genetic evaluation for traits measured at the group level.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/09/01" />
<meta name="citation_mjid" content="ggg;9/9/2935" />
<meta name="citation_id" content="9/9/2935" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/9/2935" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/9/2935.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/9/2935.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/9/2935.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400484" />
<meta name="citation_pmid" content="31278176" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="6" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2935" />
<meta name="citation_lastpage" content="2940" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Hongding Gao" />
<meta name="citation_author_institution" content="Center for Quantitative Genetics and Genomics, Department of Molecular Biology and Genetics, Aarhus University, Tjele 8830, Denmark and" />
<meta name="citation_author_email" content="hongding.gao@mbg.au.dk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6018-0766" />
<meta name="citation_author" content="Bjarne Nielsen" />
<meta name="citation_author_institution" content="SEGES, Pig Research Centre, Copenhagen 1609, Denmark" />
<meta name="citation_author" content="Guosheng Su" />
<meta name="citation_author_institution" content="Center for Quantitative Genetics and Genomics, Department of Molecular Biology and Genetics, Aarhus University, Tjele 8830, Denmark and" />
<meta name="citation_author" content="Per Madsen" />
<meta name="citation_author_institution" content="Center for Quantitative Genetics and Genomics, Department of Molecular Biology and Genetics, Aarhus University, Tjele 8830, Denmark and" />
<meta name="citation_author" content="Just Jensen" />
<meta name="citation_author_institution" content="Center for Quantitative Genetics and Genomics, Department of Molecular Biology and Genetics, Aarhus University, Tjele 8830, Denmark and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3291-8468" />
<meta name="citation_author" content="Ole F. Christensen" />
<meta name="citation_author_institution" content="Center for Quantitative Genetics and Genomics, Department of Molecular Biology and Genetics, Aarhus University, Tjele 8830, Denmark and" />
<meta name="citation_author" content="Tage Ostersen" />
<meta name="citation_author_institution" content="SEGES, Pig Research Centre, Copenhagen 1609, Denmark" />
<meta name="citation_author" content="Mahmoud Shirali" />
<meta name="citation_author_institution" content="Center for Quantitative Genetics and Genomics, Department of Molecular Biology and Genetics, Aarhus University, Tjele 8830, Denmark and" />
<meta name="citation_reference" content="citation_journal_title=SIAM Rev.;citation_author=J. Bezanson;citation_author=A. Edelman;citation_author=S. Karpinski;citation_author=V. B. Shah;citation_title=Julia: A fresh approach to numerical computing.;citation_pages=65-98;citation_volume=59;citation_year=2017;citation_doi=10.1137/141000671" />
<meta name="citation_reference" content="citation_journal_title=Poult. Sci.;citation_author=F. Biscarini;citation_author=H. Bovenhuis;citation_author=E. D. Ellen;citation_author=S. Addo;citation_author=J. A. van Arendonk;citation_title=Estimation of heritability and breeding values for early egg production in laying hens from pooled data.;citation_volume=89;citation_year=2010;citation_pmid=20709968;citation_doi=10.3382/ps.2010-00730" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=F. Biscarini;citation_author=H. Bovenhuis;citation_author=J. A. M. van Arendonk;citation_title=Estimation of variance components and prediction of breeding values using pooled data.;citation_volume=86;citation_year=2008;citation_pmid=18539838;citation_doi=10.2527/jas.2007-0757" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=A. R. Gilmour;citation_author=R. Thompson;citation_author=B. R. Cullis;citation_title=Average information REML: An efficient algorithm for variance parameter estimation in linear mixed models.;citation_pages=1440-1450;citation_volume=51;citation_year=1995;citation_doi=10.2307/2533274" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=C. R. Henderson;citation_title=Best Linear Unbiased Prediction of Nonadditive Genetic Merits in Noninbred Populations;citation_pages=111-117;citation_volume=60;citation_year=1985;citation_issue=1;citation_doi=10.2527/jas1985.601111x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=D. L. Johnson;citation_author=R. Thompson;citation_title=Restricted Maximum Likelihood Estimation of Variance Components for Univariate Animal Models Using Sparse Matrix Techniques and Average Information;citation_pages=449-456;citation_volume=78;citation_year=1995;citation_issue=2;citation_doi=10.3168/jds.S0022-0302(95)76654-1" />
<meta name="citation_reference" content="citation_journal_title=J. Dairy Sci.;citation_author=B. Li;citation_author=B. Berglund;citation_author=W. F. Fikse;citation_author=J. Lassen;citation_author=M. H. Lidauer;citation_title=Neglect of lactation stage leads to naive assessment of residual feed intake in dairy cattle.;citation_pages=9076-9084;citation_volume=100;citation_year=2017;citation_doi=10.3168/jds.2017-12775" />
<meta name="citation_reference" content="citation_year=2013" />
<meta name="citation_reference" content="citation_pages=19-22;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=K. M. Olson;citation_author=D. J. Garrick;citation_author=R. M. Enns;citation_title=Predicting breeding values and accuracies from group in comparison to individual observations;citation_pages=88-92;citation_volume=84;citation_year=2006;citation_issue=1;citation_pmid=16361494;citation_doi=10.2527/2006.84188x" />
<meta name="citation_reference" content="citation_journal_title=Biometrika;citation_journal_abbrev=Biometrika;citation_author=H. D. Patterson;citation_author=R. Thompson;citation_title=Recovery of inter-block information when block sizes are unequal;citation_pages=545-554;citation_volume=58;citation_year=1971;citation_issue=3;citation_doi=10.1093/biomet/58.3.545" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=K. Peeters;citation_author=E. D. Ellen;citation_author=P. Bijma;citation_title=Using pooled data to estimate variance components and breeding values for traits affected by social interactions.;citation_pages=27;citation_volume=45;citation_year=2013;citation_doi=10.1186/1297-9686-45-27" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2019" />
<meta name="citation_reference" content="citation_journal_title=Animal;citation_author=M. Shirali;citation_author=V. H. Nielsen;citation_author=S. H. Moller;citation_author=J. Jensen;citation_title=Longitudinal analysis of residual feed intake and BW in mink using random regression with heterogeneous residual variance.;citation_pages=1597-1604;citation_volume=9;citation_year=2015;citation_doi=10.1017/S1751731115000956" />
<meta name="citation_reference" content="citation_journal_title=Livest. Sci.;citation_author=M. Shirali;citation_author=P. F. Varley;citation_author=J. Jensen;citation_title=Longitudinal genetic dissection of feed efficiency and feeding behaviour in MaxGro pigs.;citation_pages=79-85;citation_volume=199;citation_year=2017;citation_doi=10.1016/j.livsci.2017.03.017" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=G. S. Su;citation_author=P. Madsen;citation_author=B. Nielsen;citation_author=T. Ostersen;citation_author=M. Shirali;citation_title=Estimation of variance components and prediction of breeding values based on group records from varying group sizes.;citation_pages=42;citation_volume=50;citation_year=2018;citation_doi=10.1186/s12711-018-0413-y" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Use of Repeated Group Measurements with Drop Out Animals for Variance Component Estimation and Genetic Evaluation: A Simulation Study" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
<meta name="twitter:description" content="The efficiency of feed utilization plays an important role in animal breeding. However, measuring feed intake (FI) is costly on an individual basis under practical conditions. Using group measurements to model FI could be practically feasible and cost-effective. The objectives of this study were to develop a random regression model based on repeated group measurements with consideration of missing phenotypes caused by drop out animals. Focus is on variance components (VC) estimation and genetic evaluation, and to investigate the effect of group composition on VC estimation and genetic evaluation using simulated datasets. Data were simulated based on individual FI in a pig population. Each individual had measurement on FI at 6 different time points, reflecting 6 different weeks during the test period. The simulated phenotypes consisted of additive genetic, permanent environment, and random residual effects. Additive genetic and permanent environmental effects were both simulated and modeled by first order Legendre polynomials. Three grouping scenarios based on genetic relationships among the group members were investigated: (1) medium within and across pen genetic relationship; (2) high within group relationship; (3) low within group relationship. To investigate the effect of the drop out animals during test period, a proportion (15%) of animals with individual phenotypes was set as the drop out animals, and two drop out scenarios within each grouping scenario were assessed: (1) animals were randomly dropped out; (2) animals with lower phenotypes were dropped out based on the ranking at each time point. The results show that using group measurements yielded similar VCs estimates but with larger SDs compared with the corresponding scenario of using individual measurements. Compared to scenarios without drop out, similar VC estimates were observed when animals were dropped out randomly, whereas reduced VC estimates were observed when animals were dropped out by the ranking of phenotypes. Different grouping scenarios produced similar VC estimates. Compared to scenarios without drop out, there were no loss of accuracies of genetic evaluation for drop out scenarios. However, dropping out animals by the ranking of phenotypes produced larger bias of estimated breeding values compared to the scenario without dropped out animals and scenario of dropping out animals by random. In conclusion, with an optimized group structure, the developed model can properly handle group measurements with drop out animals, and can achieve comparable accuracy of genetic evaluation for traits measured at the group level." />
<meta name="og-title" property="og:title" content="Use of Repeated Group Measurements with Drop Out Animals for Variance Component Estimation and Genetic Evaluation: A Simulation Study" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/9/2935" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The efficiency of feed utilization plays an important role in animal breeding. However, measuring feed intake (FI) is costly on an individual basis under practical conditions. Using group measurements to model FI could be practically feasible and cost-effective. The objectives of this study were to develop a random regression model based on repeated group measurements with consideration of missing phenotypes caused by drop out animals. Focus is on variance components (VC) estimation and genetic evaluation, and to investigate the effect of group composition on VC estimation and genetic evaluation using simulated datasets. Data were simulated based on individual FI in a pig population. Each individual had measurement on FI at 6 different time points, reflecting 6 different weeks during the test period. The simulated phenotypes consisted of additive genetic, permanent environment, and random residual effects. Additive genetic and permanent environmental effects were both simulated and modeled by first order Legendre polynomials. Three grouping scenarios based on genetic relationships among the group members were investigated: (1) medium within and across pen genetic relationship; (2) high within group relationship; (3) low within group relationship. To investigate the effect of the drop out animals during test period, a proportion (15%) of animals with individual phenotypes was set as the drop out animals, and two drop out scenarios within each grouping scenario were assessed: (1) animals were randomly dropped out; (2) animals with lower phenotypes were dropped out based on the ranking at each time point. The results show that using group measurements yielded similar VCs estimates but with larger SDs compared with the corresponding scenario of using individual measurements. Compared to scenarios without drop out, similar VC estimates were observed when animals were dropped out randomly, whereas reduced VC estimates were observed when animals were dropped out by the ranking of phenotypes. Different grouping scenarios produced similar VC estimates. Compared to scenarios without drop out, there were no loss of accuracies of genetic evaluation for drop out scenarios. However, dropping out animals by the ranking of phenotypes produced larger bias of estimated breeding values compared to the scenario without dropped out animals and scenario of dropping out animals by random. In conclusion, with an optimized group structure, the developed model can properly handle group measurements with drop out animals, and can achieve comparable accuracy of genetic evaluation for traits measured at the group level." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
    <title>Use of Repeated Group Measurements with Drop Out Animals for Variance Component Estimation and Genetic Evaluation: A Simulation Study | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"-cE7m2CPc63RRC7u7hNrKdMDuB9BsLv7fGkiOc3ZzkY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494046","apath":"\/ggg\/9\/9\/2935.atom","pisa":"ggg;9\/9\/2935","ac":{"\/ggg\/9\/9\/2935.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/9\/2935.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/9\/2935"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/9\/2935"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/9\/2935":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022SCzSJti3s4fOAoNiYLQf6dNuD1mLX67dBVVUgKnXnWPzgBp+6kIhA4wq73hHd05G6M+EcDB\\\\\\\/p+GKOUC4Y\\\\\\\/K3FntrWYMOU+5Yry7sVWkxLSTUDNsT\\\\\\\/rz3W62s68zGUdtXPf7zR0334M9MxpvbmXDGxA8KOujslCAmpYgFvhhMkLxP837cH\\\\\\\/ufJN0Z7Zlfwt4BOe84wTVU3SAOOFoOXAfrDlNPbgAK61twF6g2TC1KMyYeroDyY4L4LAmT6ASYaLzU1y4WZ1cuZ1QI3iJhoCQOtxyLDPZ3qPo4UzM3SDH2cruaucpW1V4PxNo0faVntiHjUWfDoyRz8P9Qm9BDnavwqVVYMtJ0S1o7c6biyr8hw7uKl9VAyRqih6bWvBZn\\\\\\\/x8ImpfRSDaV3tJXi6xXtcIHUcnZgqgZIu3FyD+sVicHPopmNYeb6EbwAgFx88OsUCsA2ygQ0u3dzfWB0tgJpURjEVl1bBuVf3ggtrxWDD9JullrmMX4HBuei+PYbZS0r2cGpRQNE\\\\\\\/L4PbUTYlIGOBnNH0fRHPyhUqKS2jqd8dhECTpe0g5E6IQk4Bub3m3ygffX+dqjqAwROEJtvnDtCjXdlV5buv3CrLi+Aooa4QD9RtO6S+qOV9SVWSctlQA2KjYu90sGR05PDEHhaIcaGmAsaqTWpnv5QBU7BQwDo0J0bt\\\\\\\/ugBky42kDD78cdc\\\\\\\/FZE\\\\\\\/vEh4KnYKvcFY3spWfyf2cQaVnnYhYdQ2OKWayYZtC+2+znDA10FcpDzhb7auuLYTG27wU8NgSOAl997rFUO19cG+fAazOdCFVxuTnLLiSh8P00ApV07i0OigRopHIcWF8BdsR8gdwFxHk\\\\\\\/iVnGx1uUSugkGOYEuVYhJBPtLd0EfDzxAy4QOnhMEfRfQqYaqPb3TMYdVmXDLwjz6X22lG51Dg9wblEa46lPKNhp8bxL4\\\\\\\/xk5Zg9f33bbza7XsRRHml4DiP18yb0dYvsNQlxBKQsYlqRvICsv7kz891ifIjRJXHAC+3xdSLlKH4ZsjMV3d5hq6NmrlyYkYlWhVv9UOKrhdecTWbaI0BiiMz9LofRpmf9Pgu2zYYkNYDp9QDGSAlcX3ZBhf7ZWlKD\\\\\\\/pDMFjfJ0q6zsiNR3n4zNgidjulooVtpNGUttGQMegDkpEsu3F2qLiyaDImrvrRGLNgV+OHyP\\\\\\\/PF8BeeuxN4CY2Oxuo8\\\\\\\/tZRpgxUNjIqADHQ+gCoVFdI+amA8Gf\\\\\\\/C3+pJshODALuMlh+Dt0bQz4hr6B+lXizZpBpcujUf4qQzPDxwklB8rA\\\u0022,\\\u0022iv\\\u0022:\\\u0022ecqYhnz251eR\\\\\\\/1l0IqOKAg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a3c6d0ec563ffd8f3a52b4dc5afb85cc599e6875bbe607ff8c02b97df09df6c3\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022nc1slTPQj+IqyOPvLI9xZG9+T6rapQjf7EW6Bf9AP72xhBHAx7cia51otNARfQBmOQoP9L4LpEIhU2M5ajhRiY4ySJCK\\\\\\\/GVY9ldbeXGwubYLgEvfnhfCpbWVvla4437ZPoPOvfNLeVhomRt6VP6sWIvSK4s0CcqOchzThz9tX5qfWrsGWdNn63daMAFtS1qatVRFp0dTBHVal3sDdbeatVHuF+thH+rSK6Lhn5FTkPAwszgmFwZPHOMkkYcQ1AFA60MwzIa0zBo+oadHjz98kah7FRYw4eKPkgozHg7J28eoDocQd1en\\\\\\\/SsaRZDuBtM+Z+SIeoLkZiRmVqeqQkUtVr10M9l1zaNV7ghwD3IGviEJTrJY+ukNei1pFD2Lc3ocgWxso\\\\\\\/ZmFALzM5+UAHAL62xotrp4nbHF3vkZ7WWJc3pwcx6OupavUhY\\\\\\\/PnYaHHxM5JTyoSgo26qmDGJVUt8LzyxfeZMmWQyPgbXTHPPKcgsNOdvzXSa6UvQnGEozUZxgaz6x1tBg9pSaVB0LGQWmgoWTpI5Fn+VS6YPmyoJS+rSYWWmHYIQ5ejaJiFPXPyY4S8zb8kKPkSAZBU4ZUU4XKJbKU\\\\\\\/Y49XF63DUJnQvFpNnca976jxWM0ZqjjebJ+JNzSwTf9bIa5nc19FfQigZGwUVD6PhJa0wdYhcHJ8yLfefWJwkLjuFFVjyqCpWRmUDKjkoMsZL7q6SbDsEGVDksWdl2stlW20fXvlYIyrPjilKiD41nJR7FhXKCZV+v2E+Jr5tLHEzHs3gYBt0En9JjCyp+bC4GFm+qFp8saU2bwMFYSO9z5D3\\\\\\\/PnyUJ1MlNyBJl1dqk5H+o8S9ZrxgHX4F\\\\\\\/n6Zmu357OR8ZyjqCmtqNlvCy2FbIoc1XOZAxWbN2ZaNdUSm3XRITYNXkx5yeLRdPOtLkU\\\\\\\/Paj0KhI5q0okMZhtWysyt70k9\\\\\\\/92e9nPjPl\\\\\\\/Lgq0fTDj96EBmAntLz0JfEGjSdqptvzN4TLcIPjMBbYKWWxHN2mZ4LjNqQPh9wgczbI6RBH7MBSRC7U\\\\\\\/KYIQsZXoTEpeyWibfEE3gLAZdv2JVxaQW0+DSnUVQybOtpzVRUHYseyRNep1VIuIQeibAeIIoveYWy+KZU\\\\\\\/Gh+veFcpJ7Telj\\\\\\\/1\\\\\\\/VF3OOlAUTSAUrQaa1APqv7rMVsvAelmqi0Cy5x+P8Hci7cg+Q6Zh5kX5mShUSGsxriqIEci4WjPjVcsBR0P8CkPzSLXQdcQXVcmnlPl55yPj5FwofreEWrnEAtAWDKNAbIUTmnTf+P4cJjixzm72Npgb2EYMojP0UBgbtP5a3u\\\\\\\/ecMW+5permOnahOJI2g9dSMm9b27GQPmrQsFG6+aLGVSIxkaozB\\\\\\\/S3yPgy6Frh6VtRbdt3wPLV+IlWT\\\\\\\/YZQTIeTCCrEwYvqDE3EAEAyMBSQho369CoQvvAaKmftAVKHLkYiGqP3189EvUAEW9ZaGIHEOSCA60FK2KBBnFVI1pEnFZAGe\\\\\\\/LpOa91VHDkmJ8B\\\\\\\/rEYgMDWplmdJQOhO15wSStHwOv5uce20kcYT4sj+D+M+tSh7h\\\\\\\/nnqcv3BHWU+bzBlbgaMY5DE4N+6wK3TnUFzrRAeAo4xmLVCce4LsnmcVmulYa7X6bCY+twXoLaR+w002nOuxR16MfD87eOdO7Qb6Lk1Dz431DQ8roDo1ca52OpuOiz06rsEEkFKOk3GcNPLkthqMgfbC4KABDDNEVWFUs5WE5ZKp3z1poSGe5LLDQBJ0FY++1hX8CeGK3fFcBdKY\\\\\\\/CtRlzyxicDhnUIUTi0TJXmr0JclLN5YeyJnc0a6T6cJQ72juZc2ccD\\\\\\\/sEJYbBI3k+rf5pKlTiJfzDXzVcZtLEZgzZeQhLwb++caKwwg25JCPvWTUQXBxwC1VONB+8lf1FH0LDEMzAiZ\\\\\\\/SnHzbJ8mbU1VnyOj0GFNQ9pZFr6njxlxsT1w8c+zVQR6ZxYrmK4e+Lm0R0mVcvoPSqK7AyQyX3VXgX\\\\\\\/6RFlTcQF+HuJNjnBeazWq2ic7qL8wZVd5gl\\\\\\\/pS9Z1V23saft2ph8RF4LUK9\\\\\\\/Go9DQUMiParXd4ds9I4dQvongABLz64XLcrYcGb+qFettOSHGsaQDmx08VrURrmPVj9yzoYNvxWgKEjL\\\\\\\/nFFZ22TkL+eWLPfacs7dp+OBaJliN6d9ccED5MkDV\\\\\\\/T7OZ0+w6F9mjzI9ysmrr5JRfQrKoTECJQbvauxZmrNXHviKvx9+TgJufZsD9Y7OjDwLsFzYtUL5siYT2hEwqDQU4HhWchJhge1Gckt6XsUfyUg7s8WM4m88oL1X0Yu1rAJNteDMY290TbLuIW2pdaYP1z2R8javYo0j8l+GJLlZ7dPhrn9hNxMCuduUtyK\\\\\\\/qsVW836wmboJEle1cmohXM+g\\\\\\\/NBnql6+FEBjyQBjclDCabkWYgFF7pPKJxeCMobWT9TVe7M3IdcJeApEZ7tlMzCZKOib2FKU7ZMvzCxTjhGld7s+ORIxUFmtkK+VVPdkkNmA+QubtPe9jAlxWZL\\\\\\\/Rh6QuglMJys8Sse6SRGNk+a36hnOM36Pnu1k08vUYt+ctYr7+NwQ5NEpzXYt6f8ltJX6E5uaW6SbTXd3fQYkSetZqiPbcBNbStr6HdDhfEVeXIevt9FlybJVP+b3vLN6\\\\\\\/2NJC0i4kdmLdcmmv4O\\\\\\\/PgrTqsP+P+1zoEzN4X2ElL2NGXLxhYe61lg0oEmJJxkhURFj6RNlEl1UPQ8vJnqfowQGXXG8Fk8q4PsxEcpjH3GfGFC6BXCWTEi\\\\\\\/MCQFAg+k3GWMC\\\\\\\/XqfNEgohLlJvQCfnHxIxS52lGQj49kqAGnT6uhjnyvcfX3Z48mUPdqfx++aoKhohVrMQxi5NBSw6GioQ+0czUm8G4haGJRbrDEtMDaO6It4RWlSLvQxrP4s7SC0KcKVsVJkTFUsaX56oAdwYU1pomIpwTbZe\\\\\\\/I\\\\\\\/mAMU8Hqv6f2sq4xRA\\\\\\\/SLZxfKKOfPmh+RYro3PC5ymau2vVitJef2TxdV7VQRcw8b+VSSjot4gbT1gTdV17BKVpe\\\\\\\/B8560Be5UXQlrJ6\\\\\\\/MAL9se7sqj8KL1zY+sF0nAXRgbGbg4EPidjPNI7rpynwQYh9+NXQgzCqUxhkzH2XnDMWjHaXk\\\\\\\/FYyjHhTTWFvdICpkoyEHWgEcxNZxQxpKz\\\\\\\/WCrMTCVszUzBLtA0v8OwoJlN4fbTIRm2UBhlfal+NWKh6uhj1BaJT6hGgdTBmdIrLx4LCZXlHgudVxFmU3jsccuLJUbLq49Yygw0YRp9J8iWQ2lTcF4UjsHJ1gL7IvpmFhKxftgWKf5bIA+cvO0jxrE8ivAaKOfllBKTuw0VjOSt6BJUprJrRwajfUR\\\\\\\/r2Ptfufstmqgw456OaQ2rqxLP2hAqdP5MQZa0zt+EsTU\\\\\\\/NrD51fnNF7z5ZTgGJDTzIe43BflPopk6CK9iIvyd+oqJXMkoKqOvwugH4MNy5oRAgWo0R+Z6QexWVweDmK5v8W\\\\\\\/jx9y\\\\\\\/XlR\\\\\\\/Jgr17N3soPAZSeej5pECd2jX1LqBAirO7wS48lDJGluyoMjkWwJwLrO9MxTT1t4rm7PAspgHgfAxa3kVExB\\\\\\\/brTDUWrCTBzOgbWtuHa3sfDwadmWWyuAYeO8LnDxRtY+h61tVP5m9IHJEwf5fcJfJHRF2D8B3moXQ2P4yVtQVYlr5pTeFC\\\\\\\/53prFEFHosJd4d8WCrOE443Il+OOzbUuGx1lA+gj8U4zRauFTOFf\\\\\\\/ASKV0lq3t+JGZeV0WwLEOorIN0tF4YgPTNQifosTjmghwv0pnPa8IF+BGhC+TBHo\\\\\\\/Oli\\\\\\\/LuJOSrXxMTbP\\\\\\\/KfiKHfY+9KY7AiE9dkXTEL4MuztmzV+Tah8q8pZcVebg85zea8+0wRXPfKI3oMa0iw2yOKjJV0upK4e1\\\\\\\/RtCL2GnzXSMswk0ZDOzt+7S\\\\\\\/1q9s+BLW\\\\\\\/Lh8+GLZGfdSEWeBWk2um4Xjn6kThX2gXP7nuj6O3X9889qx2O6IPhW0P1\\\\\\\/PQ8LwJIWrDsdUIt7kbMimKUa39FYpzrGUZwwL2tK++uF+Ac\\\\\\\/YTC6IcyBlncR46D1P93eRx0ZQFmWgKbqADgVnpnoc8Xks7p\\\\\\\/ecSAKK39YCZY+vVwKDi8yd55c+jJo1acZwfFkNUwW0MjAVc7Pt1YiA1BwHNHMls\\\\\\\/atD9vq7inBpe5BJzf7IusSt2s2qswzJ9e5gJLzQlcj1LZ1QB\\\\\\\/0X5NHA5\\\\\\\/FQclMn0U9fnB6VG3CRVEjnIb8vOt8e1igy7e4k\\\\\\\/vcoMkbJiCyLAv7rZ7kzlrxj\\\\\\\/UUGMQ09PvlCN\\\\\\\/4B0KKSPKSclRZVEHv4WK4TvIA7H5RYr5QL22LPX442Qc1Phr\\\\\\\/KkGVO8oN839UvYfPNHf4B66Pt1m5uuysK7jixE2gFgHJ6QX66N+If6dKafutHtoZwVcsfcn10O1S9ilOfMGJi5ZAIVsoPzIxU4RHB9FWiKHm7YpUgTIK72PmQ2Q20iUy7hJkhlw9TF9lAH99bEK7il\\\\\\\/f8SYCYqsOUDxip6p9046GzcmCPS\\\\\\\/V64OAVXg8PNXkOyySGGXP8Vy94S+B+I+o57bZlqH6g3DkXD6SLZFCg4I5rdL0YZZvt\\\\\\\/uTn7QtJf5ujp3VGEPM1Qyz9\\\\\\\/kjJ\\\\\\\/yNOFJxEr3S63j7WDFYrzHT5EBpBsFU8QaqiRIIkvZcm3fGh68l1w0I8MLTMKf5P82UK++n5ng\\\\\\\/NsfoSuzrKD9f5pdbxFv\\\\\\\/bi9nbtiwa3sI2Hm+9m9CQh2z8Y9+1eKTUKJOLxk5+tDURwUJBxob3+kZVWjkE1bW7DqmkuW3YBxcOlJ41dhzRDo1bfDP1Rym96gQzMRaRehcCmwfw1dUcdUQSDfh04er8re7+woDVUmxmsmjtMS4vYJXfqwvSB0NSTO0Rv3TPh511rw3HQl92yFHeBdWP2PO6hJ6arj4tpBjChMtnpdnPiJu3ixDJ8BCrV2pU05PWNZGSF9QvRT10WnwV+KimPA7Nu0RPjZjRrUfuNd\\\\\\\/xsYtoniHPfFTBzE9Va2kyVXxZlnHnUPZQYk\\\\\\\/tTyeF69EYE9g12BwuOT1TiqWIJGwhPw+vldUNTRkbaGJr1big0KOZXW0K9H12hmzd6R53CEGK9iblRAvm47EjSs54wphwR3Aw7zBHs+k42Ab5ObLSq2zsuukF+EXhuChn4d2tuq7ejZOB9RhittVaEzqyg2LNHE0dXnjFbLczjDu5cBpGxPB9y\\\\\\\/3xKE3AizlrwzW7EuX0ED4V566mGHyCStpMe5X6K\\\\\\\/xwf4Meq6almJ+n7Kq6JVAagyLOC12CiRZUGbMAbPdivpOfNQldnwc3iR5LbhHCxwJnCMDKpKDoo+008NxQMMiWR\\\\\\\/1YJNt44HEsROkBDSTT+C1c43Glr5GJvw5GNj7n0Bv39GXJphxsv2UVjyJpMX0X8+Gx4mncbxFUHR\\\\\\\/V5rQF5etviTzF7vgoH\\\\\\\/c9iLpd2N0ibdpdURsfUQfMW+bD9fVugsfJMadT2mfbe4GXB3sVdOkJIFFNuwYDAAmlWlJGJ3lrwhkgtLR2smjGUseX8hanHyKfTAnao5unDSwZZM0CwKBN\\\\\\\/JzRNk2KrFSneB6M97ecZL9dnojHHjhHWtIiTjzw9pI5EJKU57jF+18UDHstrjQdMLkfRTaUFdk0aUgN7fQygi8+odQRRnT1q8QQFWzTZQicAW8KxxVQvNVyJxi9TIeO2UteJWuX5ZNvf7AdMVN\\\\\\\/7k+muJOaX14mVjxICVz038wP7SqB3fqZxz6JE0+aZEKZ001X9e88NQGcaLYxn6CFxgo6IBSG9uHyTG9qBDL4zHltewhkKrDf66uEbZngD2Mz5oUdWuY4Dz0oUYTPuHxRXt2I+WS4uJFk3swpCER1S\\\\\\\/2SZpFsvikiYwma3Lo6AFJNXXpCoamM1uLfwu28ro0N7suy4XSCtjafC\\\\\\\/mtomM9XLjvIXZg9XDPhjGMJ+lDm5SF6X1jmhvbtCn1DXQta\\\\\\\/FrrKEsurJQIdEUWC\\\\\\\/ZQq+HcxurvVXsdbrQUOgr9Pc7ZsNTTRu\\\\\\\/udE6lZDPUvyqFPzh8nHBcN83fcMN2XdOcSIrTQEuqoWFbalI3V3pRopElse1MgZWXYGyd91tSGapCCl8QcGW38bBwDklzdYqrH4uTsvnl20GSETrE54tNEuEfHALEqhbp4x39SZtwQVDFMgsS+DXSb8Kz3AGD3ujiZ\\\\\\\/3JFHuj2Reb0PH7W4PJfzZqpKE+GaA9bCLy8aB++aSmHX3ErsqN9ldzvO6j8dmQMXjE7j6FTOMV8xRefflZUpqpsaD4leQab2F0ABVtdjYgLpy5GJa\\\\\\\/3a1gyoAJy\\\\\\\/3zxOp7Jk+808YoWUnV30M2krlhTzjUBV14t8b1xpeO+P7iQazRodd\\\\\\\/2DjBEp7Rb5aXims95E6gXB8C2DiroSX82x\\\\\\\/zYmBCZJncf3hibgz\\\\\\\/StudB8gSrKrDjrST74P2R8+9bQITdJRIIV3znHMlQRVLjcPJsZoeMy1rK\\\\\\\/KgFNyGIcAtFDtieMtGne0Rl7EFj\\\\\\\/ggljnDNSS6PfxF6dqbIZQL2dZ7tNCRfr0rmdkgfSef4rSXxqwgucHtXgfKuIFEqW4WyMYd8tNIma498mpI3NUH+OBh\\\\\\\/H05vIBCKlYl1zkbUuVqvkoTueCOvn1vqTCbfGzcLcfaa4sfyece5MSaP+cwk31c11jgxUMbWiI+AbRR3hY+xVlZ5NlJ35t0UockcMJALrkABLSEWwU\\\\\\\/TmFg4Fxz7mUB4iGwuBny6GRLuo79gutYyv5dDjLE3z0y\\\\\\\/47HqQ6EqMw5vW9M2DS0rWbq+szxrYCtjsN14nANanREk2qJOEf2tVZTWaLnBetS0Qwp2w1s0onYZlsdFprFRezNn2+EmyP16MYs+WjGdNAvi6hBW+IL3yzjMhG7QD9PI933KiG4QoOnVmtBfgzoISbs5AbGuSWAcYszKeoXOB9VNzMk3WfePyAWsfyPe4vKbOQORYxzyZMISqsRhQwa\\\\\\\/OLiGx1k3ILsIT5NE62D\\\\\\\/4b75oc9I0oKFY3FgXz2bb+p1KDc7hivC\\\\\\\/UVNmxJD+RI4IF49AmEto4OOkCjBCUVAV0Nk3QzIyqVBbMiyqpS0NSJEmMeS9ghca2kv3iCt288ud3DNYdICnHUI3whHyd6Oxe9hbX4AQo\\\\\\\/Vq+KzA+XlLaFOU\\\\\\\/cUXpnCN5bMnZRKbjBqKsm1RGEJrUms3C\\\\\\\/9eevLlOIONlihWMmz5iNNs0ZbVEVOOLiJ3xGgcQPZ5I7uHvOaJUMWcfn8wSTRd1s5a9U0i7vAT8i8lvv1ycLcBhxVvnNYD0D2xAi930g5q1bbYbI0TrARUnzIji1YjPTE5Sf8zufhiYM0pAzp\\\\\\\/dRAltlQrkdh9MCJwaiYIw3RYB91AmZOHn43Sv1ByLoGVThhM34PLjiLm4exzHCTt\\\\\\\/fkldwuSOcscupNlLTL7ZGIsbXDbaD2Hwiq2J2vKSa+QAREkalnlY4uWnyNO4UJbqG+OADGGh3WkwDtavyjEXYkNr1DrnMkH7WNOn+7pfRdreC1UvNJuWshFLEngHEIa1B40pEMkfxCSmIcdVpDcpjmapvyd5NbOEOZyMvJuEZnXfmkFNqwR7iuG4iaRZ+K4eH+0ATfPwqDVNbXe9ER4gQ6HdY6kADZ6zgSh7l2t2cnQ140\\\\\\\/aTeXZ\\\\\\\/BOHnwEGeT\\\\\\\/gezF4A9aMqAiFsLzk5rJnJkTsAoLabBPis732qHOmr0E60BFB900L6whlrcetRwlVemN4JXY4loBY3xdkhIb+ecrP5bxGhjOGOpPPJNiVGeAYmO3Tgro3a+YHMBLXqtrDwky2VKy1TFgXW2Xn4cTgupVMhZP6rn\\\\\\\/PI6keMdznJmLU8AGqHooRZjTrkpyvFZc14XCnh0PavQIO0As5vTwQTJWd\\\\\\\/kWPtM0B05wi8Cbj9iQE4v3vml6RVeSwUb568Q5FQf8aNoE09O3yFf9lpzQgeRhIuNDH7GprUcgaQdYYfCxKaceO8E0gWkNIXawys1mz00PsUowUnUxEx66NEzY7JNyGOTDRVYxHuGVizteXWQ4Q7EoPLUMwHiUFDB4NCprgQrs4OXRiLbFdDFfzkRMNR2C3xRUNXB\\\\\\\/OXnfEjmWeIJYMTqQMV+WVYVO5xYt22NOibFMesucU0jzoFZ0bMW5dYv6lxekIEOigkoSSSb128lATzHlKgI4dgU09Ay6Yhd+Ptoz4yjixw9p4On2S7lNPOaj7pn2aVtxD6aqq3SE5geUJhg0a9KbWXcVmBMx9ME0e4DZ5jQiNTpUzUylIsTXSgUZ3nzH7RqX86uf1UbTQJurjSaFJY\\\\\\\/6OMCqUGSemdOoArF7y+OH+pGSY3Jvg2mfJDliTxZSEloA0sR3lle7wJ2Yy0EKDZ5a+ZEX6I4li3yROPGXpmCBdimjYIs1v45FYEyBg7AUYGFDUZGWIo3tMxxHLQTRUNW5eG\\\\\\\/ExMpvftnk8vuBbz\\\\\\\/GkpG8+LSrsNHwvp0fqlwgs7dTKs+L0Qf0aHeQyl9KrV\\\\\\\/Mg3czClwcjDl31h5qFoIiiV\\\\\\\/4GbC0\\\\\\\/aBcXsJqlKFwWF4Lr1afaZCq25Zz3nBBcTrGWqJPGKxm1Jet84HKO+S92MrMGthmngodt5GBWQ1XcZD\\\\\\\/7hGG5P3tEiumcp8SAA9MwPvLfuUg38zNOU44DuGOtX6dzqVXhsa53r7\\\\\\\/2\\\\\\\/pP8APS91ACzpEn69H\\\\\\\/H6m\\\\\\\/ADltkkPija7Gol\\\\\\\/LeDpXVHN58sV2RC8Oi\\\\\\\/FJ7AFcHKrCmUUjfunhGa11zfRZ4C2DBiExBaKTGx9NZ2VlDRlMjsVK+NfL4U6cPlulxoib9vQ+k9hVhDI\\\\\\\/+wx8LQbmptRIw4xNy82aGOCgbyVSoWfCwfzLALxZsObVwiClIDRJ6xj4TPrNIdV5E1get9RdkGJZSGxXaa0AYmRtyxC0gW4\\\\\\\/Dz3ndLVbWPbuIM4tuI+t7WgtLmV2EGY1oNFZ3GWLYcq+t+3xAwhTlY0+7RAKad9rOCC4Ocjr5Tj\\\\\\\/7rcW7Da\\\\\\\/U+w8fTh+QfhB0TxAl0eLfcjGnSvTNeSB61oyyFhGwzECTdtJ7\\\\\\\/89MAvAzl5c3DcWYPYvyxWIVj5xir0QfVsij4iJq78conmtPaRnQQ4OZCifj1ia\\\\\\\/NgGq3mcKthUyzPcOQYGoTSKc1gDzizoQzFM0ZvvXwF0vMAMQ+wOlBL+0sean6W5GH6nbK+0VxPHf1kFTrD+k0Hk\\\\\\\/v5nqcSUEgZMvusZvJO9JwLJUDkihXj7Wu6+rARMkLspaiIi4JgurVq4oNbdnlbb6Kn+mVrLAI+zxTI9Hf4N4zlr8+R7cmVbWDx6+FJDIRpBAi4i3bICZ+A4YBxx1MDebuZhL4z2dIVeNDD9JLqmZCntrGeoAvKDW+jpd\\\\\\\/hs0LUD\\\\\\\/3xHAWP2y29rRakK6vU04U4wcSXEN7pVxYpmNYtE9jdXffGWiItD+rEmsu4d8DtGk0f2msTBJBv4qQTNGdGm1V2h5PcocljEm0JXtNEUZ764\\\\\\\/aGcv9CnpCL2oAgxc1xFL8VHug2WasNFbf3UZys8GrxBEzf4UhcAyCUdpq0Dus7iaCeXJynNDdEx+irDLLkSDgyv1lVVRMfsbk7x2XBmMYVtyNT5Uf+dMqR1k+BnJwO\\\\\\\/CUJtuWm\\\\\\\/jqbqjCEZfXKCu8MqLZ3vqjUow7ebjkklSPWKuuzW1slTlqHKvh2Ip9COsZb0b+Wi655VN9bjMtQX2qendrKCKtvAFO6VeqM4ZvkP5Kh3GTK4EUHEPXoQfw+CVd1I8dfhDF6fvjE9MjZeAutd6qAZ3H6Ymoz5Fg8zx+N6OWZPNzzpS6\\\\\\\/KX2Ib4TxPE\\\\\\\/tcz+EiWnPAXB51GiGRsCyCsxeUTjxPHWIrA3NSzmLXazxo3FNBhlLOGkU\\\\\\\/e5Xe6ZUdnIAT4KdOyfVTZuS3n1mKQbDmGfXHPo+5QTq5mI+wbT+6UdJUUHEqTpR0Rw7FU8bQHgEaT3gK+iUtFN7O4Wn3kqN+UPOH8uIFbFf0YACP5RweNxk94CtiDcT3nd69lzrPGFvn7y3Of6PGnIi0Epc7eV4aVfL7FYACWLnS3PhH1UMLn8X+SCG\\\\\\\/RAK5nBmkeN+\\\\\\\/rfX2ME87y+vvqvyA8LoVTIr85ZaGfXML77MkOoq63dDuU2aGWSCuU9BKMMchrt9vCrX\\\\\\\/21N\\\\\\\/C5kVnLwTzjsn9dbG6UHQBzCuDeyKJYTN1sjk5C266VITr9ru3Vv9YZKqjsgP4owmxfVPPJryPPJ3dRHbbqtRJDWSPBBcUJkxf6k7rcvUdViwwX4VjQVjH+1m069mXMvGPTsP1m9964SWCt+KDi4RiV6thSyG1v5Dfz5jfzjREcRTE0ULHQRx+0T02IsKK3wazs7\\\\\\\/SwrKQqK\\\\\\\/NjPRhrLOhFQbZQY\\\\\\\/ff6+RixSN0SWh2QgYdWjB5lxCWVlG7HqW5ATj35qnw90yiCrGAlpprg6I0XUI0pHwdFHwQ5Lj3Kdh52nVe3qxgbEsg1iOSIzXOEWmc452JsaVrmIT+LWrfisMIfuBWrOuGuVVUe6tII1VuDV4kt5wN3Ydvz0KPmptJLvcvfKwy63gXrZCVHzfbwu1sBnJB\\\\\\\/ANTJSfv4NDUYeSkbkD8Dx7xf2rfJSfXdtrvHCoqxtHhsF4SahjNhaZ2ECH5ysr7mjwxU+f65UZ8cdLpAj+V\\\\\\\/TV37fW2edK3gfCf6fi4Qmrbhet7BPGfUcqCuDFY7AFrp9AsUghG01DD6tSpuWHrzBXIMSH6EPxOtvvFkqk+jOX3fCehBSfA7UANMkvcDA0x4cay7V+cSwYcDntOxsi8ae7pIN43BZsltHQ1O2f5sI1coUJcwu4LuNuNOppXdC85jKjeOBHXwGLkdpMJrm1mLQBdx1Ua+pn7UYqIuka6L0x8HMWywfvq+2VGQ2gBIVHvc0w+5omZKnSQ2yuo4o+Xx23m+VLSbkn0G11hiGeWgGz4fAgTB+pedhHc2T+5FFfWbPi2XgoDr\\\\\\\/Cv3CTwM1QFYHkEJLpPps3l40zFf10Z6gi\\\\\\\/9AJKu0vsw0A8AbZrucGZYrvf\\\\\\\/mpGdhJ3LUFmyqI16CwQuEp\\\\\\\/9eqqO0V9ptcLOQF3+hPQuezaO5csY8OROJByNPwB0\\\\\\\/lwnNXLF+CIsRGzTGWikR6iL9\\\\\\\/efEgRCQFFCQXRfuY8f6Sibq66UVVk72QvhTdVqeh0EmxNSstpyJfPS+S6aMW9QMP5u7CeU1gtitirDEbt\\\\\\\/WtXdroz9HsnKITjH25FAiVFzyjkn54WnP9DvGBweqcPWUdh06OXtfnBiQMEBbgM0nJBhArJY8GzueMvx9jfddrLV4p36Li\\\\\\\/grMTrQmGIRwFn\\\\\\\/2BcATQoeBYBo1+Z1WEia7iE3M4KT5xrbkY+UGu1EFt85AVQ+TIduasqyfew3B8S\\\\\\\/Di8KhMWc4giqs+iUxA+o10SYb1bGI7A1wNr9vBbwCawq4Epft5GqLSGEOJqGEBRDj0Ns3OPhMv2VtWjKbWlYfOOnE9\\\\\\\/+yaa6iHpTfGneTqj2MlJ742mWPCLk7nL5oouKoosYX8ikUpNvkHVL2xLM5yiBVbRNY5rcNyUUuwOCxD4OTideG7voIPLTz8tRg5ECblQ1rZzBrk9qoeZvL24YfhXpzPW2CRBfPcpQ6wNsj5axa8\\\\\\\/L2sPK5nkFEk84bJLKJVXereYeGnOfxOVGFFhI1\\\\\\\/qveSdaIHOjK4Gu+izsPPycFTfVRS5P129ZJpL7IaksZSn2M240G1AGYYXXC5CpVfGSJFkbADcR6538h7qP9HLMJSmkpIvUru5BwI2JrPrrin6NFjHFI2B3bu5HYGLDXhXe9zXdEoaWS+Mls10weC3UOWBwnfH7BW92C5cKUvlYXYLeHDk71G0Iz7RHFmsDv6aRKd1iUm7ZXJJpr1ISrTYbbCCvjGpNPrKX7Bczl4pq3+JNBKYQ6w1RT6oQw6MffKW9PkQfGl7AteolLLg3UidVluXn7T2lBJMNNsSvDWBbTucSb56I+2I7GHxFLar9NTJqJzTV4im6N9tROmuQxqcT54o3KxCUC7Z1h+1AmKBu9rdHegdRuOmgoUx6dwzy\\\\\\\/Dtu32hgX2jbsL7SL4DEPiXJJ3pUHwVPC88mC3Lfzu8etH52Q5\\\\\\\/aj8p0\\\\\\\/MXkuYw06xx2wF1YNDU8swHoYhq\\\\\\\/DN8A3Ia1WiZXZgmBxZyFn0JT43vhw8x+BE\\\\\\\/s+ylrGy61eWTFq2vP1Kp9cNqbo9umYDloak9hNG+0kf9AG6d6ckltGpnm5jnSDobonZ1VDy2DXDtxASSQQmUP2tvcCJ\\\\\\\/CD15rsQuYy1RuBd6MFDLwYJv7fqWhy0RKG0WUuKCIqUDNeFwIFfdVU0Qs7EDK0yxJvAlLrCxG2160ThWdzlNCMUn7FcA3FCF4WRyb\\\\\\\/Sse\\\\\\\/GouHlE8Q51Uuk3la\\\\\\\/nF5vuLZJA+7l+xxtjkjwRTPn25HsVk34W9aeejG+gZVdEo1prqHb3Z8Mb69zc92YlR+lvZC9FDkAySqvk3RR6UCNvbRLnd9iqG\\\\\\\/\\\\\\\/BUM81fCu98OcSlCh9t1VuL2bUuvv2RscfWPvlLM3r\\\\\\\/\\\\\\\/WdXJN2A1T+HShmaijS9puVbwrH9QPptv\\\\\\\/CLpZZVbBWJ+YF6VtiC0+rCRW4Jqu\\\\\\\/\\\\\\\/MnvVC5TJ7TiHlxjYuZNhaxmIKckOoxi24gTSXtE8RoY4qWGQXgfKWsHqAM0UA8oGYvlUWSHdQX17F58LkHoqrI25QS1DLtCm31JQeRiIIiT+wU4FVQWbAMOPDvdd6Il5xdKW9gdT7eI5BhsWKpai1+XwaLsHe5E0liebgYrx+cQUt2z1M4mlCAQN1D213Eh1lcmuPbrrRjtdvFZucR\\\\\\\/iSmtVHFSnDBTf0zXkzkbCNyeLczb8k2YdGeMtKErURJZi+gFYrRRqWqNM9I1xbsIKbLsJSwE3NXwAq35run7NXIiAN4E4LVmIocDr+DQIhisao6pa7oZp+B20STTNGfiXgL96q2Xwu+8Z9Af9oeqJGqFu2xqowKDg2kFEKNMv4Bb7QfFPjSkvX+1lVb02MVlaVYCAuZVFEQyIT4QZQWFgnInSCicycLFz2MWMVjan4w2Y5owdkxP9ZfFF5Adw\\\\\\\/Pd+oo2hZEL65hOj5zqJTLPzuFxpdtPuiHizxyrw5qGfTY3KtM+OZL46bAJxpQaXRaEZGslDMBAuzNzFWsQfLrpdn6i3was5jTX3T8Xm6p2j7wZsJ3E9Fms\\\\\\\/Doa2XOChbFBN4i42iCuuX9MRKwVASTYcL3nDtrjvw\\\\\\\/MUwChpnLrXe+FoDU\\\\\\\/WGWMapeLvPK85ksHM9X1PYL5WWi4uTxo1wmI2MkNpNTeHx3HEOJDMKfH3V4ZfHDXHFVEDwLuBOz7QbmbxwPPYPr85IOrhvheNP4IvmlL6J3RVWmYJe\\\\\\\/rqD5i\\\\\\\/bohoJr3C\\\\\\\/a+INhWFMjL2tpnlLKfGjH49Yvm0luPCnaxF1IL+gMDWlHUVJHiNwwXcu6Fj0Pvmm+v5SvJHP49wVR12AZ8h1PW67rQSdl57AVKSx0QlXW1AbpGWG2Ofdd4uuN9ChTmQ4x\\\\\\\/i9Bi4r\\\\\\\/o\\\\\\\/8j0veLqG+ASs85Z0BgoYMEQKj75WP1BzOJtUPBBK43M8HZG8nnTSIZ6d09gc+rVoVW5DDhdBbZNg4OkBcXE6+FAGBmM86YXJZ61t7mXR\\\\\\\/V11se+3HBcK6eIJ3VoctPRfYQZLa+Ln0GtRmKqFM5sUhxyBDI8E9vqwV8yswSjhu8av11GdtzcIlHGztEKom6DHHoiSP8Mr+bQ1q1m8alEsn6RWyNUPYGaa4yb7ElWt324\\\\\\\/4a4PUlclfRJecXlKO3memgvVZrhZcHKE7Gz0Sm0MUgUjkEsn41kUz6i7hPnKzyngbArkaPuXg1Q+sMPgzJmwm\\\\\\\/dqqayE84ZJlzAYRCnkWCKjB79VIzwAwQhRHjhw8wo8F2PZ97DgutnQMZiessusOLImKTGa2MLFL+32l7umRXJB85AjEnZ9tX7L1KAEJHfbalJiRbH5Jl5FohjglSxL\\\\\\\/GCeJWmcAqVcJ4msFIrjo3A1vjqrHmHKmq+J+9bu3m9T1bLMXy8qIcJUIjYnj3iqF3+Rtpt+YkTYThRT4+pzg+CM8mZA48lRwsm7D1ut0qUxGvXj5U5Olt+8AZxM5gWMX+\\\\\\\/tmwoK\\\\\\\/qnEoC5tjMPoovyNYa0VKmPRYkY3ohpLkuqQ27VD6KTyEhv6iGnRjWsoXNuCcZUIdjw9UEbvFTooYM9ZV3mIAq8PprYd2A4J6vUVSeJwKuEiNLr6TJ7O8SXy2OIo2opQsIu3+N0f7gkfQMGLSFUE+FFWwv9umlqaNNW8+cL0Jco3BMC3ivfBP7zXxhWaYK\\\\\\\/itrFCPaAd2xl76sLWG8CgaQY3Z1TNLifSWTZc3fjnFhivhImcVaDBc8gFNRuqa0BiGewBSezHWLSVEFv7K0ON6WRbtWYL5nDBgb4pFlu6YEC00ocmRRSpZfYeWi2\\\\\\\/LKCoF2eY+OlxOkc15lCyYNpKD4BJ6knxQwr0v96R0Lxy5+sXPCNiGFrw\\\\\\\/cjEgarNOM3d2\\\\\\\/BWIc5I0H1blVvux4\\\\\\\/s+EUEyPYh8bs3EQqkqDqNPUP4Qb9fk9IizXpBQ6MgAGGF2Sv6KUroIo+7Px2R7LcGL4CUO78iohGHE4HHd0\\\\\\\/nQnYcE9gY115zjZcU2UJ+17k2xcOSFMKeHD6IqqpYmtVF\\\\\\\/FBiiGAFcQYTE+lO\\\\\\\/of8Pv+xBo4u8mgo5EmxtgZqn6RsnFSpQppAELY0PQtUOEMUM1sJUVgF05wNePnd2ABwaV6KXHmG4aRFl3tCks0u6aKjrsvZtakhWCio\\\\\\\/3uCvQShUZxqFAwK1\\\\\\\/EpJXH12y24Wr\\\\\\\/dlpEVslhk2JEBqzu6tIDHSDe25o1aBQmpelrt4DrSTOyTB5nEEv9ogyx7QoUu1VasNCYq5SxCE4xgknpHt80gC2bKVTeZNZ7yQO7+FF\\\\\\\/oWbXcYN9oVrEMVP2WOHA5b+b9zIjRUVAuxNVssTJaVabWQ5bkzDSTam5UslSHrb30vSJaaefd3gjJiXdbvv2IPuntLc1CLjVinlrAXN4ZJKQru\\\\\\\/YBLnNi98PdaP\\\\\\\/WWwgdXZdyRHvEb6d4YVZo7BF\\\\\\\/\\\\\\\/NQ\\\\\\\/+yDzDXEer323reEz3s2f\\\\\\\/2lDwwH2l1L5ME6hDudq6cYumO\\\\\\\/OvWtNQumLftXSBO3AdbQZtAzOiinpel3D+OH8+gqverFogvnLIZ6h8Nzz6oFPVmLUb5ledz6+zDKRLPXM4r0X5+8NPxmt8AaQ7ea5+222EiQKIgn5gv9\\\\\\\/Uu8tx019u8faqiH+T\\\\\\\/uUoWUlu665AfeC+6vvfeCZp85\\\\\\\/XwFzqP\\\\\\\/LEVa5HIG62EA2V3E+lYucwSAirONtqKfCqaMul8O5yLcWGeQxBhAORWK8rKnQeY9TUgbKKlriQfU\\\\\\\/U2E4OQ9jNgFGIjG4dFvK4iINpLHzWInYPrVZnjeSqBVqsTHadeKSWxiU7YhrPOvf3GyJXn2Nj0tmCD+\\\\\\\/qjP23T4lioTR3BKbf3D1\\\\\\\/0VEbkahXRXbZhuxMheM6igAZ0g9HcL0BXtgnjrwoehLhUSB34f2vEbwjPEAgJy46w\\\\\\\/EUGPg5cGikV9QRjSQAVHnw\\\\\\\/LQv7ukby+Hed9CgIJRNbhSuk7g0v9OZXB5HmnqoAusTj8yOLZB81jz1EfgFiNB1yWh6JpQRx8y46dGwZjW4+riGYr0TS2Er1QYZ1Gg5gogrFkO3CdedCXqvrXsJS3mEVHhFv6HHPTXdLSu0mkVWajQ4n21A95+BK7\\\\\\\/rQ1VCC6kSJzuV5NVy3nhztrrdRvD87YvP8qLVLJIrQs8ujTokPxfOoYkS\\\\\\\/PAVx9qqaU06lORtxCq7ZB1SUAUPrYIyrIplKWH9TnLqFcj8jWJG4CjYakl81eGstKsucnzcBP03U0JXxBsyJYfnv0DmEt\\\\\\\/FOnEVDIJazy+w712\\\\\\\/FbLL1luLS8NiuXWi40YeF5IjAnRKG36URF+NoJ\\\\\\\/aV1TBuUN\\\\\\\/qWQxZ6qn8lEsq9cJOL45f\\\\\\\/fquRO\\\\\\\/7cd7ICmXj2OTj4yvz4PlvzSiv9b1vSJ9Npgy+IAykqTmu8URU17sAMl9DNQ3BOP\\\\\\\/W04oKES1hb8r2U+sAkNvHn0DgYxeNaG+oFc29llT0\\\\\\\/d+kqUvMo8Qd5ZAuWABsfYpLI4oY6emElQXnVhHKw70RCzqtL\\\\\\\/YbATxQWtfqAetAO9woQC45p1Cn\\\\\\\/SAk+Rxq1+qIIqoxIeMbCYZLmytgGoMNSLSQQ3jzwi1xzr4RZSZ5pFzmVIwu52BPUPn5mZomu6gBUZNAoxB+\\\\\\\/UUuqqrKX0wHS\\\\\\\/8TIzcO+BKwM1fPQH7D543x2q4lNMXpAT9Emus5gcTKLiYcregRvhDaCDMC48Yn0glfClTxhoKlr2Ah1eqIuj92W1MK+dKS8JhSvwsIzQvl6zBBJPuQqNA2QPpRezxpZ490aRTIilV7plZ0BBsBHH5JpBl2WPZ5p2ctNRlAyeUYapQBuOgCjSccflvLmlrj7V+b6j5kJQ2N8iftsFvoiwWT2bSX0uklFI2GByfCXYK1VF4kWUqJvdg6vbn1AvEHJztMEQEv61ABmlEOXqHBGxqTr5er96mncpEoe8BIdcref+Bc05Q\\\\\\\/+FKYzc\\\\\\\/+R7B2TgqUMddSuAJYDlQXyb8A5pzo5uKyoOS9ylUqtAufy20MOrQ0JUQpJjTgD0Bh8unXPzg03tKPJ6Dkp1sb5YnKhpPnmpP6sHTevoXZzn691peWv4I+gqiiYWPcH9NzgVgykKOyMuvqlGnJgtTobE7D+cZ6\\\\\\\/Ah0o57qo0iuO9kOazCdpDFLvB+Zf0ia2w4RVU9xbN4yN2tkuvuu42AooPaG+2tV5HpYMkq7T0lguutEUapHnvq5olupK3pAdb7iXZ67WZMVfuFxOyuhZ2KWdLUNe6Xq0u\\\\\\\/nk9jl05hNLf2Oy2rsnu9lBOF42K9P\\\\\\\/iT6gskPhZ9R+dSTpyifLh5b+88wdrTVnYtgJGkYCl0DVImg25mJ6D4HfIf6+2co+3Q2DcsDVlNG76zH2Bs7ju554C64++VLm1eMT9rAZTQ47IhTKBBUeR15\\\\\\\/dEozLkVacLsHZgMbKanPJk4DCb9VLo9yqGlJ7I\\\\\\\/golZ8WDnLxmU7NqDWEj2TCXM3d0fCL15pTMnUwZdcdxlL4Ye30Y+sInQG9PPFDIHuSGp2FZ1jyRTDOm+KRqK7haF01fT20L6t1wwx7wSG1\\\\\\\/rHG3fHTMsl3TqBgLLk5glwPsMm2J1Aez2RUY1XqA3ntc9CeMbmgSYcZrYWGagp1a9HR0Z7gKPPKjvhGSr1nKGqGccqfaL1npsM50AHxY9t59vT7pb\\\\\\\/578xoPyYwPtJueN51EOQaxyShCUQvmnxqgZgsK6BCIWVWBQlNLlDtENoNJiBWwq7QIpxoGYybT4laHe+Y6QLohFgQQOcvgspJwd+VAHmXFm+Mw3+HT58SZ5i87XQz6NYGxAI+NbLMqu+6CkjIM5NgViTeFIfiQDJdTx3XQf4jiQ3VsdnG7ZCVGQZkgTjOHTU4Ok6SmOqtNoznmC+bb+mx9lEHwWpfCxizCvdRkYl2KV3Gk43jURabJVW2Kxgwly6xrb89Astxl8fuRti4NvgVGrEr4+M5Lg284OrQqf+TE7sB+jXPL72HfjkZzlGbGjYqshfvDZYhUDadnLSbvxzEYHpH5g06V2iY9uai6KtMHYzI3J3gohUdlH+5hfrxJvEv\\\\\\\/vG51haa8B0aWDT2mhMGngAigZcBnCJkiwnfTOi5dZyCWevU12bT\\\\\\\/+G\\\\\\\/5jJGiMYG62dgTdVvSt5sj06xrJ1Vu7KQccno69+pM5LxQ8FWEhXiyscZd67aXEUXaVbNLIbxtZnBHBORWrw3nc4qj9Dem2CMtlUiwM8s7obONuv0Vo1QrhfAnpbWc3zHbE8PkV2qKWwEnpi37lxqNZikVlbkMkHFH3fVlzPXrPznN7vKo+AeHTADuj801mas\\\\\\\/1eqmNBN60d0r8iFrXZXt3WoHbDZEkQJA+L5CV4ckVGvhsTJ4TjQK3L5Cadx0qyWd1l03qzBKuPpjGkTUU4jYarKCn3YNvMEhEuTKHC4vanHumIrLa6TERHKIWm\\\\\\\/dX7L\\\\\\\/la9yKfMZmwMXpzBLYSALO3x60IhQq5sf1ce7TmXYALFurQxlLuHjjBp+i2xE27bm1yRmkg0YO3JWkSy7yuoQ0RPRXqPqsPKZjCptniAxDrTseyc3diTcJ4fHn5jU\\\\\\\/ZJWbgWjd6MAOIpkMEPlbuXthp\\\\\\\/GJo4x2j7U5SyOLMUPdHDSHVpUd2ERvGTerJMFVfSoozcjiXzjzjq0KGPqhBE6VcduFhshaukgW0Q2NxqVX1hILjLwyOyjupS192MK9JS719YRxtdj2ImavrAzOAwkSWAROlF7lLfOd4sLImey\\\\\\\/Pyc1tlHUSWRnh3\\\\\\\/Awv6r35y6aOdrszMatEZ4yauAhUfREoH9Ytm8vocq3HKqkc4BpSTm5tzqgV5bJ63ouRiXuXTmPXwFg7D+rcKK6yLH4wmz0qLURHDnA+WiUmgJ219ACFq4wz2Okp4R8w2gcI7FHVgKY8pFjJ1sL213KtuKrYQsCQd0\\\\\\\/IYQG4hxbprMBFyjTxsHpTfneyPC8hFgSVVNWP29+cNKk6fZD0gjpPYWrLOoVWF8U8tHn\\\\\\\/uSwmWwLt9AW9Vr73ESlxyVbst9OmHLqcsdW4xQShZE+obEDA5DLIvujDQc+2bVM198rQVneAX00JayZtPJj0XO\\\\\\\/VMgXORS1kauQrBqe5muPeddUQpV1Q7HXjdXnl6R8VTMfN37QL3uU6YJeF5pGZg+RiUE7rD1sjSgpC3z09jIMjKHE\\\\\\\/eEI2NrKZbbEQrgdSfh8UlSsURxYjJ9Cx7EX8LAO7oEmVcx5FlUcitTWu4njgRJZwYomQt52rYqz1vCKCYX+xzmEEpaP3AbdSSTEpTkG\\\\\\\/MsTrKhF3nbxCwKDRkYvf\\\\\\\/Gi2A6Mp7YIiUcG+JN4x4W9kiAPJ\\\\\\\/IfKhktRQIY42k3nVk5EoAbMyd9\\\\\\\/UX8vxq2MpqBuZxMIivKsE8NkDbr5oWck0CE0sAAc+oUZx9hXBZC+6dOzJDB3zoY6QM5+6jpi8yZcC08INkIkVjIWSLvcXT4JN2VUU5+GvOteiM4PFUAo3S4Tt\\\\\\\/gKTPHPEINWQqgPdJjQ5K9FIvEvjuauvH26RCGRR6esqSXJr5sdsE6w2DnqAaDNzHMAWylfDAj27m\\\\\\\/i0kFYH5f615UlER60Q+Sy1qpdqwtE2ska6GDrMz5rcq9dwM7\\\\\\\/IyO2fQT7y35j6QhjUhbJlBvbT5+hBGwOSWBZlMHoRqW6psGvHt1lGGE2VxnVT4aTj1mZTKOjeYzCyZqtXAJkkxfSzHaEOTghsoVdxgPWdpYBfXoXIojmzoEumPe4qsfpySvUHJKkiOsXx8kCmFbrMpYB2CW2Y7bldM+oMqyuHuSBZiiBclQdfqkZ6oLvKUJ2LM2WO43oXCtxrUU7vMKG+07HHK90fVe\\\\\\\/ahNwWmsm4RmMJU1z+txSc7xci9oYRkGzwvbmFISw+GBfMqPTfEDwx4qSLHVugKsl52fdcVM80fbHca12LeN4obVka7iuKTwgF9KcEVSGBIOTQpppDusT65MGS6QCzGUMNfx8LkrFcPC1Vi\\\\\\\/2PWYNaTo0yLzrksiDlttBJoizG+y0EyNqpjVudN0DakOkx9teWev2PqTliv7pro5Vlcq9XPZY4LcsVFPu6Y\\\\\\\/i7DAH6hAVAZq2xnLzvXjOvnl7VAg63Fke6DSj+q1cJ+MfraXlkpnmLoaxN\\\\\\\/vGN4D8mfvUiJSR6j4WGqV9fNq42SYIXayS+r2IifluLvkybX60E24ARvtNXX8pjwgOs5nMq8RNF65sZXeg7sSRIwz86YjFTWjYa7wZPlgF+BZB3AlNqt7nZCPkETUwHfOJE20k7DKe3ffOb0gePELgInT6En1YjC9QDTR2t2YuoY9Y4r7rS8ZDw66tNmvxITpTrtPbdAlTm36P7tUZp9cVG4TBTIYCK08WdCZwgRyLy+EYkk3toTf8nWIq6JE7D+Js4wwa57jDpr9i1R6tBQjEGpx2yDnR5yGAenz9k4RUiI1dWppqvUp1jRPf8N1PkW9xN1HUN6lNN3g97wtmZAvz\\\\\\\/o3k56bPZtCQJzGJqaAe6NKi+B4B3xFQB9sf0vR9sI3W1cKUJ64YeopmzXpPlRRDToqDgPeXUmjxOrL8oPWRRRAcoKpiKn\\\\\\\/ogBey2axGHrD\\\\\\\/xxUNlaDrtyB8CfCkbdG2NXEOA47daMZFZnk6lZsEVtI0RD3dIu7HLj\\\\\\\/Cgd+Eviug4javkYKg\\\\\\\/Flhj63VKmXx6LX23ti5MHR4HjlseITjnzRlht26HChcnvhH3ZRuLWp67ATl9crwNypBOKUa6lMEfje1wyjGjtY+DXrn33v2TibjR1\\\\\\\/WQXb35Ly+fyJlWHfkttciVIvYlbRCyhDoBCW4BGiyxq+PRKhv4zX2MiVtv2kKDdEC3WBFninbcvxYw9SzAlEbSkY3fFKFF6WlXi+BZbk6YTnKonqOIWBwDpBT4y1jKSH71XUCvMCYkYpkIZH6T\\\\\\\/eVmA\\\\\\\/81sPTbjXuqTTDpn0QlyoiddIeTZsaC1h2vVIkc82JlYKxtceYMz6iFMaOrQ+sX7Qt7CxoFdX3gYdy2HJ\\\\\\\/0Sf9gxeb14WFsiFKIGdyAVT79xwRHBg3ZkCNvddNbIn5NfThR6fsg6i4cFWhOH2b\\\\\\\/gWgePB5yCHwCTZW6jgW5VS35C9y2vplWK992QPLCMBAeCoD7X\\\\\\\/10IBeROgN1GtyZPjbSK\\\\\\\/4bl8Q6CUAYi+EeEEyayp76Ci3MDIao\\\\\\\/F9sdbRy6W6LZ6umyf0tM\\\\\\\/VPNy1QYhFnuvSAcfOf1co7abmL\\\\\\\/gt1MtNRfaXR9CorPeOg1Wix61NLggOt\\\\\\\/9XQMfT+9W0jMd1Xg5S2djPrXGgzhfB1WeDnMr3fmguSlz1s0qBYwXs15MSv1BxBQUFyOlQiBxCL94jJidGk28K1Jn7YGR8OxsycyK30ojKX4rGRs7GtUKgV5DPaFK+tF7sYSwcGyy9f31xajQTNNCoi0tkGjrpiA3nGNF\\\\\\\/SDorekAs+1VGTPqOxPbGW4Fq59YJmq+nXWs2TPi5nod5TcEszD+q5vBN2kzxV4wb\\\\\\\/kl\\\\\\\/1xyEMkhZPcbXhPwGN841xol9MPSSGlJv3EQvHdv4+mPYJTiC0c6bXmOCKsk56s8hrHmSrLxLPUtPes0YKxmCw5+y665wkihy\\\\\\\/zpzldlHtxt9j37vr6ieYBGsBsxRYky+SyelmBj1E7VwS4eiJ2OCupNRlKUvemW2cum6lRtIypGXWdFXHztLsj6X9qZA35heYF61F1KQ4vPhnywc0E8UAi8lqeGVJwkX3d4p036ulmJ+k7vXEP\\\\\\\/EGD2ABW8WoEYxkY70DKgLMCJkSXIY8BOjDUR9e8pB0CfRLI1y97C6ekZwzCQ7W1bDZJuJn25axITy0MFql7LnzuiHOzN8UV0KzftWidJ0rI+9jJt3ZSr5lIPRK7LHLzEnRB8zqvFKwY0\\\\\\\/NRWT3Wnmhw6FqkinCyRKzB0ZvcB+0zY63n7iSKYhP0cF4Q+JHwaeu4pK2u6veklELtJsKSYcQMnwPu51eqAczkBESi8TKFcwnxN+ItN3RHi8q0OXCSHYA\\\\\\\/6ifFOLAG4YNm\\\\\\\/LuJVO6ZSvRDsaNhs7KF+WxfHmgxo\\\\\\\/7F9NvxZ078UAwmiCU135aAGKdUv06YDRO\\\\\\\/nwiDiEaEz0eWsz\\\\\\\/vbldZC81ta+WMgQit26QWBM6IweD0\\\\\\\/WWiUiCWwdief5Vmim6WyddVho1jmCF\\\\\\\/iL0xNjk4jwtS9oAlB2qHiLVJnhGWnq0SPe0xNHJALEAae9aU3cYOBH5O68XcoPAYcZN5o7x9BeAMTTHFvovbtTwfsQ0wPYAXTM2GESxOfL5SykKj5QhGM5ShOgct7qiQvQCExK77Fpxxup4b0Ooj\\\\\\\/7jHPlI21LX2mn+53AqNlDj9IEwu6XuewJpSgnhSjmrCysYyvqCMJJ+8kqsxv+Z3xCS24BNE3k74ixSDJMm8tC5etA2mSeZsMDh5SA+hdN8LvnJYBQRa4BknNbt1hXeDCL5C9Cij4kuMgZkH6sBzn07+KXmou0eEUJJz3sTYCgFXd9SGcbnfZSQFewa1RjYtOU4fjLVvXcEqSDpE1jRSnqqOutQJDtNjY6inPv8DWJCsV3wKWND71HqRA\\\\\\\/fLxSELP8bjwOVdWoSUxWAsT2aoqdpmYwVNpv\\\\\\\/zqdfD8jmUgJd6Oug83oOgYb79ewF1gBGUgU5DgzZmEtnHmYIRshH1wR1OU56RLOl2bV4GPDGYifvg5p+matKWzeBoCj19ViGisXSPU8634nxfdJWwsMcGGk3wNhTeSDLKNwPun31gdFz92TkyiwSAu+DDg8ngfSGh9ED\\\\\\\/aPMDpeNPEQzdw4T\\\\\\\/gc9ZRC1k3RePCuHzl5AJy9bUzM8YMiz6QTp3ZGHhg3RpZ2kvR1hyp+VtYRKESLl2CWhZkqXD5r+B4xeqYlLnWWYFfH6D9GPG+xxCXAdKyGVTgzsaAppqmTWVTSDCkm\\\\\\\/F7U\\\\\\\/TdBdsbgTlXB\\\\\\\/ZSSSp5rgu9UXAQYzLSeeDD1RbsnawDmjOPuM+JmGGy5oUg8Dt6YF5MDxqa6v9kZVf1iuSYTVTKytySM8DwOWO8bfUiT3yVEaDjfhcVsr2CJnT6he41CtVtUd74wxObeZD\\\\\\\/VlrsudKgUWv4nwD5VxkIN8nWIoVsHOE8nnUa4iZc418CWT4u0a1C4zgF5wiasJ\\\\\\\/eI51XbWkZxKLaJzzX5zA\\\\\\\/Sd1C+7SA1QTGF7X3tuBxa\\\\\\\/\\\\\\\/vkCLvPA+GNeu\\\\\\\/Ozizuw7i2TgDc5EaxkEo\\\\\\\/4wMgautE9YW7fyx3xj1iO9WBA08Bu5b1H6\\\\\\\/s3bvWYgdoNMw6cT+TPbV3D85zUHoK5IP0ansh9X86YKZGDnYjQaSEIJZGvhYE6KZrZpHzSByJdN32Tk4glI2W8LYRirX41WrvhATYrk2+LjYYQNuI6UA6jnIMhBLFmJL\\\\\\\/OlFzBGXCVPcRB7cVPaHHud8CbaOzZvM3A2vlm8WTivYHzNwW1L7ICKTpgZpvR94HPNAFYz8GO40x3iIG\\\\\\\/efHUpGCWvOZj2rktEVTfDlJNLKczP7GAwfkLsC+\\\\\\\/Y923pC7imzAL\\\\\\\/tyl+v93HUlF4W9Si21f0ZMWIySYnlhYpuq7GxShW2FItxRXLakaXRr7wxkShRluUEL76svGKX89CsthQQz\\\\\\\/jDTkP1u1ROrWCl2g7EUpEtYdaUAKwTbxujzJudRnitL3SHUDs\\\\\\\/L9BeDRmx4agwVEpHKBojvyLA3Dpf10o6ZI9ZoUomWTAeWT\\\\\\\/wsN2QT6UsVMa50ME8HTrn7zcX7cBnocPyGz84ttd7bX9dkJ1eGC8a4BexhYDYyUXlXNXcRH1kA6sO4PS5uOhXtREoGBJ0eRGYTCb9oZhLe5cHfTWKzML2sS+UBIRZHDv9gEPOKpv0PS3m+9uZmXnQi1IH9WRMticqO8gbSQfgFSE8blPX6qymFpUr2CG8\\\\\\\/MUDt2XqveAJBlI6nSA3ZB8dgWDnNTwYgsgSDqEfdKughmcyL5YY27Q+d3EWsoTMff2YyIa1clNB2UVcD6NyN5RzB7o\\\\\\\/yZECbmAhLc\\\\\\\/6Mv0gCm5CMts5v0Xs\\\\\\\/7+ADEi7x4BPCAQvwLBlqIy6bSjyFWYQHghTBua5RWV34aIOXqqoi+RgSxPH\\\\\\\/yZr3XLKemRpWkHUkx4bBFJoW\\\\\\\/NGvLa\\\\\\\/LVtnyLVYKmrk+xlYb2svZdb2UrU5Qc2pg9S\\\\\\\/nuqlC8FXaqsd0Qg49ZmTnMSrSWiAKgmUpsnAEnf83JToA9Dy0Adn2KkFY+1sHTkeImu0osqDd+aWlS3gLWUsuVyJ9v9UZ4qwK3cB0lvvreyGcOnQfUJUAAeaaFE3EDl1Od39Guo9r6B\\\\\\\/luaifWGjZaCb1ThXHZbRdTDHKLOUc1OSTGSHLnvWRagMeVKwo43WNzlh\\\\\\\/GGiTaeOGh4ToRPcFWvadjF8qDAgeMYQRg32VCWLzh8PW96nTP4u203RUIzkPdK9QThcb\\\\\\\/8mH3xxhXUT7c3JAw8qAtPsarkCcXl5\\\\\\\/LFdYCiFw8rM9QOWmba\\\\\\\/9n1oFVYhYmqOCU1voN\\\\\\\/S8ATlgWZNzNQN4q6ygBRw1PrRlew7QTjmmQwVAqmHnQngWViL0xfKENXawUt8HhUawEdsfHOGL0fdxDYwHnIyD1p7RW\\\\\\\/hW+0\\\\\\\/VKptIqqEVv3UIixgU5\\\\\\\/Qkf0CrpW2tHRN9UlfjbKk11H113gV+y\\\\\\\/gLwos4pP5dMnkQ8gee7KjNp2gWIbXvKHrM3nR081lmAWxOtfYt97qx+k38YCem8dDB5waG8TeGU5LpQUk75dgLnNsftXYbM5cMp8dVrRxfC6R8jvZ5R4HWjOaRRatYok1LTlUhHPlEO1ZhWiGZGoCzQbhBGp\\\\\\\/rBiiA0NZQ3GmcJwmAoTNNousQT9LAi\\\\\\\/e491rgfayeFmsDdO7h4Xmde4n9HpibcXEiHd8yjAeY1LxhvOPoo4d5aMx82ylPNnlo8208CyXoEAWP4FXzYrqT2iVXHTpYF9jot7F9fOVxzt8H6\\\\\\\/hnXG42pum7lwjIWf9k1jT218kz3NQDcevkoM8POwzt3gmbOjl7MwnSWwJdhaON7LKJ41qiV7duAlJNO2L4ZkKEzyBeN1gDWzP47Vs\\\\\\\/pU9etiR\\\\\\\/fzEI5Lums8+6MeIGP6QUJ0R0+Jye7V0QLkAAyJ8e5+sTT4Q\\\\\\\/B8zbluqwitBEKBoerWlIhWHotdAVmRHCc4w1XBylAKG82Ixr121wK0RYkmtXPFnmc32LkwTo17KyDyRPtC9RbtiuaoKeJ7FI6sQwPr9qmmn5SG2W625tpVpFNK+Drx+iVfbDNYMjNCMCJvpmqdiz5XLVcMWQ03S5jgsCWGi5nMJPbF3GqjziKn6lbU1YZyZqcBseGqIJeHqD77MwUe2LveFVkfbigASgRWzMPMB+1SlN5ofVNfPaGt7SCLqJphKxSico04qWJ0BmZwB8Kihj98\\\\\\\/fcoxFU7XfdC8GzuGR1v8V4FYHt3ukTU7QQN9bN7Sqjbm7yjS1jvpGud5cmG8Tn9CGKuqYkiXyBZCmquU2oAsbBC9DRcH5slt3OZFilnDTpA7kAwavRpK4V4M2ZOJ\\\\\\\/K1mA3rMO4XgA+eTlYpwgVO7HdFd2tVgWo598pLfLUtruBm7D2bL4ZHu1rWMLBfStzS+86epLuSwoPdDiApgMPWhA7nVi+ZrNgDS1bHzdjkEa1KbhdsYo9\\\\\\\/+TEJmNnZEA5ll\\\\\\\/gAwv6ncBaipegobCzJz\\\\\\\/Jb\\\\\\\/cnwtE9qzMaDZSzvZou5PHJ468bM7c0eHNTG2UfO5YuBrlNB+gHKFFF0EUdu8t87JRkpf7KVPlnh1oAi4RrntK3gvik3ybd0Eaz4KHqauJlVWpJN\\\\\\\/SgiIYGhomgnQ0Je0rnK29M+ViFk7AQxfxrxlcAePOG0E3+6VBdF0GBK9vEq29aVyH02vUF\\\\\\\/ANeV1AkTFCyOJKJBmAUHXNgyJUQky8DXevcYOq8ShylU\\\\\\\/DSgNskaAjlxyhKDqLlH2jdlEKMhxjJUC5NQdFGVwApK93zq0fEsNsomhX\\\\\\\/E9LN1JlX43UpPC1OQLJR0qvU+mCqjBWoogM0oOUT5rKkDqEAcxrdl1Yc7uTy0BQQ9icCh6vvtZ2LFaJsh6loqQaI8Q6+y6SpwxYLVtAY9s1uNSYGM2XI3+08MVHZwNimhu9aFoRnygK4+0quqMMt7coUfHOvWdDHFBhqBp3iZV1Qn0PodTma9Y8DTwtBkaCh7a3YErEzWsOzgN\\\\\\\/puSMTC0vmiqz641g1zaJ73bEypQUBl9rRBtoAlwu1tC1hb04hyICAUduhDhAJ32oFY6clZP22DJ\\\\\\\/dJDg9r7dV7dyuAVwmoAwqZ3is76qNnMY0Vs26aRszvnaECN7wImWhFlvuEcDIdyjR60+2D0LH9kb36P\\\\\\\/bH2K48+gumNGzVwPuSckphWy9\\\\\\\/\\\\\\\/ltoZFmDIPNny0T6eRS+Yxbc07fAVzLYH9U6HiZqFULWQ9gcUiMi\\\\\\\/Ku+2QM\\\\\\\/VpOsY\\\\\\\/i11VhydPOF4RorR6fJzeCMqW8spBkQcZ3OiebzLucLjRdB6Gv4DpKVB\\\\\\\/WWoc+zxz1HoVpCvEpXnyGMucbStZIHc5acuc+cs0VrtAfCBtbjrw1ccVHaA15g1EnOym3GVvTfP0ybBYN82FoI7e7Pe0svSdQ8eHBx3\\\\\\\/9ERHas3pnOGL7IupLapDcdSt+IjyeIXj7T2yPYPo95ioct3w96dsWMdXpy6AIvtKbgC\\\\\\\/62xjSVQbj4x4UerAqR1z9lKlveRZNuogtmqFnBs6ogLSdH7b\\\\\\\/e1uzjthrlFbHjMRKyS9+DjSP3uj2VYQ2MP9kL6xo6x1Kcd7CqhHiGnMg6FZi5SUTmJUpXe1WRwjCb75XL2fmZccGNdkjmof2+jhSGA65RTYu9WJRFO2V5CPV0EX5Gb+PCmah4MmhsfRfZFrF7DaZzgNKfyXi0CTaiBSLSU1\\\\\\\/IHMP3YZUL154mIVdoNBeuhkw0Pd52jxM7zNztstyYu0TX94pru6Xb+mBtCSF92UD9RSEU2GWh7Vm1zmEAiA2tm4rjGwgC9bInCtHrWDk7siSYF2kxzStOwDXBofV6tm9jqcD1GLpoQaeBuGBc5yuiX+nJlifBCBXKFEzgSqh0xF7DireNz82FKrpOo+l4QYWBEsNYBUr2d5tu+hPDs\\\\\\\/KWg2E4j00Zc+GiVCHH+z0F8kxdckJ4I1Ey8leoxSDfnbKNK1lhkkB5EP\\\\\\\/LKA0kfbZWRjxMmVPzXFbxfxTIt5KFETuyzZ6rZSEadhaaFIZ+Bf8W4hsDQ4E1q88I2atSw7MEsfrEHLHK\\\\\\\/uQTDXNExkjvgMXDeD6AHppvKf4s2\\\\\\\/rzvgnsIyYoSWP29kImzVyQH7p5d5mY0IvDGJj+J29ILrpWB1eaB7ObCSXV10xj35Gr+wOV3jhlqAL\\\\\\\/Y1x5Pu9yZaqTdpWZWdbF5CBCE4bK8+QhImhchohBsFyD9zfAueQaqa+moM35JLaKDSwEvOMUM1MGDutJCdrnF+LmuM0xcvFAp3gmIpIjuyEboFrfxbvKuK40np59IkB+1T+se27SZPEtzojuFtfhCoiBz6HGkWE7jCO\\\\\\\/wb2lXxlvzH5eROI4rPHR\\\\\\\/6UQZ8ZfL+jnEATaIjM6zbsxHbn\\\\\\\/yX5bT1LglA+2ycWJpScSly8qyMMLp+TA+KHF4Yocycm0zYMI2xz67DI\\\\\\\/Wq+CUH+Fyx2KJWEyJ0jimMXZBadiCZkHTb7ZaaT7LWv9fbNrnbIRZJY2YXfoBDR4acP3Y8H8j2AuxRtzx92Vh+Xdy6glzBvrMEBy7B0sDETwxYBty9S6HndItG8XLsJ1r4ZBRj2uFFw8POpuv4hwvQNjwntyej7+O1wZCDXhXDpuowH68Y+SCKe6GgSJHcak3Ers6ZjOXLO5Fo9f35p7qOsGAyB2uW1fYmYBzxs81h9a89ldgeJdTUCKrzkGdPeD18PkNwxe5AFUZyCcSOLxm6jTtrMkD3Z7p7cPPdkqSFC+SXOMDTGjRB6ATZvu1SM4Tmjih4+qmrThW0fzOqg+KsNq7hte+p1t6bVVUHBLbZti8vE1NBNutrbmkXYiUvWiQ0KPSGA1877apanO7N1LOI1d9vfCHp6tfXpAQ0TQx9BslAS6AbHhOalFVvNtAD0hp09PfRvbqgT1WqoS38ZGp18GIL0I9AMM0IA8Ra1cpp5PdoAqOmKReYmya4Neu37Tfyvh3vRrQnH5IUukP6eCj08eoLDOwOVj8W4Kj4MXZUzZLtojdxoYgpBLc8AVndzhGD+ak\\\\\\\/4Blls+zozNdABb2wnXXmoeRDo+0H7DCBCEwsuCpAQgd6R4Mrmq2D7rdBNk5Bo6DdPXilyGG7lDtboWcRUqHXu6tNg8wH1A\\\\\\\/LVSTxemccqVAsGnMUl0Yu\\\\\\\/Vp3aSQYndfMQLO9jwO+tj3qbKuy3j3a4goRHW4jM+53AhXOB97twJF8\\\\\\\/zHNrm3LNLmDOfQamKIj1+W+YMadSv7MHraKiUiQbvsUtohsckfmkovPRl97dCazcNNQPyWbQ8jDMOs8RtnJpXgDrlZtNuYMkgZHfCzJF9PEuDKYtR\\\\\\\/vsXtPp7yvX6iAd9t+J2F09xsNjPnl07xeLpr7U\\\\\\\/CP\\\\\\\/WC3YqUhukcujRvC0Y+b78\\\\\\\/yIwQzryawpj3MZE9HbN8UeaQBuiDXfmMRdwZwqK\\\\\\\/cECEYR6WbYWVxgQSaZ\\\\\\\/EFGPdT6DQGNrWFWjPIUuiY4\\\\\\\/4qawbN9JwbSG3xveIDc0H3qBtJdUQ+y+4tTmskRclT7x4xdKO\\\\\\\/f36dCTbYww==\\\u0022,\\\u0022iv\\\u0022:\\\u0022K3u7v3fh9SejfBRpgn1ZIg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222619686468bfdf9dd92a09e7037f5920c2380d68db03da9461b2911787cbfb9b\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022UtOD2fUUcuEhhunngeZpo9BfJlY9z3vATrhjWjnx9iz9LCcFI0GwEZCR71r7j6Owe1qZpw\\\\\\\/LeGz30wu5fnWr\\\\\\\/XlUImmHO0ogi6VGlZ5CFt2Ube6LBzue43X4jbPTgpGoXlL0YDrH69zAgS9OUm6XY9kvb407xNoCgB2fc832FEnMs+XEKYABWYAMyWKoDosW8J9EGV5gweK59sgvSMnzmcDVW8YLnV87yVFSRV4Gj0NMmLVJZjIoM+Yq4pEHm8m02LMWMYe6Mizj1dBFw7Bhlr+eQ1bEWfsR3GB7YJC7jUoTGyFJxzq6siC4XKN2dBsSixXxjmMnkgA3Sxqao3ZxnBtlbSykVkiG+GbOoYUl\\\\\\\/u8KMXd7kR0nBAVj6xxY5Hk2I61N0LTIBEDJeSTe5IcbhBu91hgYbW6cUO4o5nw0CVQ15BYRbd4I5w9sBaoMW7UCZMqQV9vczeotkYIbbO86otUsBxhnLnIAUqQBP6cyFFiCWgQjvM60RVYody4d5KAVOAHIVabmh8TX4vcvrDIshfJ1nPeuaNbsVbu\\\\\\\/869tL04kJF1jixUebMAH\\\\\\\/jKy674+J5FSPiI6E\\\\\\\/Uj579I+eo5GLBZe3LXGwJw65MqHQlB0O5stY2++7QK+F8bdBoYuYq33lvdYSqzw3DvSIM+0DhoI5CyJ5HZA0IJW5\\\\\\\/ATzqQSCVuw1\\\\\\\/K2\\\\\\\/Pq2qBa67+nCWjztPWhsvqRpQ1pToeAZPOoSXdlZh+ShSUO2K4E\\\\\\\/tvEx4wgn8a95ZfNKpZ0NfwLEIzK5Zqs++nEFLYdM4cFWDSyiC6JTJkAU+JtMLt6DoJ4+2+cESK5qiMRK2bfULlwc\\\\\\\/15s4YAIP\\\\\\\/6zn6Ip5nm9dBo1K3Rhm27mDHbkH5MD6WschFFXmzssQVFnb2PgMHIllyKxKCjQ9m9lc2zIQYHJVLoH5ttTXKkV\\\\\\\/6F44ROQ4p+PqHC30G1ZBsiW+bC0yCKaKksUemyqRdJI6mPCnqP+eGNIo4M\\\\\\\/cpgwTKM\\\\\\\/nOnz+bTSbuB1MvV7HZ3Aggqi0YPEQk1ETSfr4lFi2pp4CivOofreEF3j8NRj9T6U2h3BXz6tJsqZCq+F+e4wmR2icd9fqAPxY3QXaSrwk\\\\\\\/RS7f8xA6IBn08xF3p8XQcUwmP2LMFo9mPgNmNNdlSj6wyVwKFdE7vJubVnSA4qH8vCkHiKIh4gd\\\\\\\/j2Qc4ttWpcKVV43FVB\\\\\\\/IqlOJ45K4QNeR6rW65dwOr4uOSHspHmVjVlWmdEGU+jDbIZmu8SeQQw7N26QEnDHLUJU\\\\\\\/dsBr\\\\\\\/aKUe3daC0Vljr7+xWCc3V\\\\\\\/cqXywbXIeOETbA7QwG4RTi5IyuhfQ2Ru1154QfHw2Y8bO4pLZ2l7NjnCkGb9NXJMENy8welktKPwZ41PAyachV2GAp+CfIdJCY6oOn3+ho4Cuwg7EfKyYbLw7byTs8whtrTfSgDO5YPgYEl3PIeL+yT0zJWc0AYhs99fSJYpn34OExsoWF4SWjlYki7\\\\\\\/T5shhWoUf4NcUkNnpwKmqj\\\\\\\/1gv6b8Gpmpu7uEWvfeZaWTCoMRyJhSm7qbBkBsgMtgqgji90WjbbRWb6INtWcb9F2AxqUaA9hn15b6yom7usqj+xvMKw9JhoFh03gOpne9p04QzH590\\\\\\\/9JFZT7stbOPkzJUlHzDd6IkqFkK5lbqKpu90AvXuuZurAuReUioyxSu5l8a2VETyG9VxiKD2tBjryEL1c0TN\\\\\\\/VvOhlJwyW5nYWWExMdt1qTM2Sjteyj5Zf9ILSaCvmTY9iPDfRdT2+0Ikxxg5LwbNjoShE+n5CaBIpG4FnDQPiBhHGYi3OE1UUOTUCGVL8gJ5OIlHWW6vvBF\\\\\\\/4eK3PAjJnX34llUNWhdAoyjkE3Cx435QjUjdq6Vythq\\\\\\\/hbxM8VjtUvro3+B2Z3oBgJTGdCMYeWHGdpAt9\\\\\\\/G9cHpaQg62BDld0nIaGoqy9bb7qXLfR9K86+LnIniOBlzbX2L88qLsLEjNqkP5Cv2Sbu8ZBpCR7REf0B8YQOl0z0eTxLAsza\\\\\\\/fOLTA==\\\u0022,\\\u0022iv\\\u0022:\\\u002280DylQZIl+dbZ8YgZBRYIw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220573fafa0da50b0f2ffc1608aad2c788252654fc35a4be5cb67fbe4ef3b3220b\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494046 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/2935" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-NuIKhZv94lDQq6Pf2CfM5cM2t2mAAUI8EBTi8r5Ynm4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/2935" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-mcMwo1AOpMG0AWaIvXF-0LJBCXItg7CHoeqBBHeHIak" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/9/2925" title="Joint Use of Genome, Pedigree, and Their Interaction with Environment for Predicting the Performance of Wheat Lines in New Environments" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/9/2941" title="When West Meets East: The Origins and Spread of Weedy Rice Between Continental and Island Southeast Asia" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494046" id="node494046" data-pisa="ggg;9/9/2935" data-pisa-master="ggg;g3.119.400484" data-apath="/ggg/9/9/2935.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Use of Repeated Group Measurements with Drop Out Animals for Variance Component Estimation and Genetic Evaluation: A Simulation Study</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-6018-0766" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Hongding Gao</span>, <span class="highwire-citation-author" data-delta="1">Bjarne Nielsen</span>, <span class="highwire-citation-author" data-delta="2">Guosheng Su</span>, <span class="highwire-citation-author" data-delta="3">Per Madsen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-3291-8468" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Just Jensen</span>, <span class="highwire-citation-author" data-delta="5">Ole F. Christensen</span>, <span class="highwire-citation-author" data-delta="6">Tage Ostersen</span> and <span class="highwire-citation-author" data-delta="7">Mahmoud Shirali</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2935-2940;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400484 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494046" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hongding Gao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Quantitative Genetics and Genomics, Department of Molecular Biology and Genetics, Aarhus University, Tjele 8830, Denmark and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hongding%2BGao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gao%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHongding%2BGao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6018-0766" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hongding Gao</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hongding.gao@mbg.au.dk" class="" data-icon-position="" data-hide-link-title="0">hongding.gao@mbg.au.dk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bjarne Nielsen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>SEGES, Pig Research Centre, Copenhagen 1609, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bjarne%2BNielsen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nielsen%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABjarne%2BNielsen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Guosheng Su </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Quantitative Genetics and Genomics, Department of Molecular Biology and Genetics, Aarhus University, Tjele 8830, Denmark and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guosheng%2BSu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Su%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuosheng%2BSu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Per Madsen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Quantitative Genetics and Genomics, Department of Molecular Biology and Genetics, Aarhus University, Tjele 8830, Denmark and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Per%2BMadsen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Madsen%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APer%2BMadsen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Just Jensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Quantitative Genetics and Genomics, Department of Molecular Biology and Genetics, Aarhus University, Tjele 8830, Denmark and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Just%2BJensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jensen%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJust%2BJensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3291-8468" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Just Jensen</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ole F. Christensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Quantitative Genetics and Genomics, Department of Molecular Biology and Genetics, Aarhus University, Tjele 8830, Denmark and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ole%2BF.%2BChristensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Christensen%20OF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOle%2BF.%2BChristensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Tage Ostersen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>SEGES, Pig Research Centre, Copenhagen 1609, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tage%2BOstersen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ostersen%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATage%2BOstersen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Mahmoud Shirali </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Quantitative Genetics and Genomics, Department of Molecular Biology and Genetics, Aarhus University, Tjele 8830, Denmark and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mahmoud%2BShirali%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shirali%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMahmoud%2BShirali%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/9/2935" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494046" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494046/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/9/2935.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494046" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494046/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/9/2935.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494046" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494046/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/9/2935.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494046" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494046/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The efficiency of feed utilization plays an important role in animal breeding. However, measuring feed intake (FI) is costly on an individual basis under practical conditions. Using group measurements to model FI could be practically feasible and cost-effective. The objectives of this study were to develop a random regression model based on repeated group measurements with consideration of missing phenotypes caused by drop out animals. Focus is on variance components (VC) estimation and genetic evaluation, and to investigate the effect of group composition on VC estimation and genetic evaluation using simulated datasets. Data were simulated based on individual FI in a pig population. Each individual had measurement on FI at 6 different time points, reflecting 6 different weeks during the test period. The simulated phenotypes consisted of additive genetic, permanent environment, and random residual effects. Additive genetic and permanent environmental effects were both simulated and modeled by first order Legendre polynomials. Three grouping scenarios based on genetic relationships among the group members were investigated: (1) medium within and across pen genetic relationship; (2) high within group relationship; (3) low within group relationship. To investigate the effect of the drop out animals during test period, a proportion (15%) of animals with individual phenotypes was set as the drop out animals, and two drop out scenarios within each grouping scenario were assessed: (1) animals were randomly dropped out; (2) animals with lower phenotypes were dropped out based on the ranking at each time point. The results show that using group measurements yielded similar VCs estimates but with larger SDs compared with the corresponding scenario of using individual measurements. Compared to scenarios without drop out, similar VC estimates were observed when animals were dropped out randomly, whereas reduced VC estimates were observed when animals were dropped out by the ranking of phenotypes. Different grouping scenarios produced similar VC estimates. Compared to scenarios without drop out, there were no loss of accuracies of genetic evaluation for drop out scenarios. However, dropping out animals by the ranking of phenotypes produced larger bias of estimated breeding values compared to the scenario without dropped out animals and scenario of dropping out animals by random. In conclusion, with an optimized group structure, the developed model can properly handle group measurements with drop out animals, and can achieve comparable accuracy of genetic evaluation for traits measured at the group level.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/random-regression" class="hw-term hw-article-keyword hw-article-keyword-random-regression" rel="nofollow">random regression</a></li><li class="kwd"><a href="/keyword/longitudinal" class="hw-term hw-article-keyword hw-article-keyword-longitudinal" rel="nofollow">longitudinal</a></li><li class="kwd"><a href="/keyword/group-composition" class="hw-term hw-article-keyword hw-article-keyword-group-composition" rel="nofollow">group composition</a></li><li class="kwd"><a href="/keyword/feed-intake" class="hw-term hw-article-keyword hw-article-keyword-feed-intake" rel="nofollow">feed intake</a></li><li class="kwd"><a href="/keyword/accuracy" class="hw-term hw-article-keyword hw-article-keyword-accuracy" rel="nofollow">accuracy</a></li></ul><p id="p-3">Feed intake (FI) and feed efficiency are commonly measured in repeated periods during productive periods of animals and have been shown to have varying genetic background across the test trajectory in a number of species such as mink (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Shirali <em>et al.</em> 2015</a>), pigs (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Shirali <em>et al.</em> 2017</a>), and dairy cattle (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Li <em>et al.</em> 2017</a>). Under commercial animal breeding practices, for some economically important traits, it may be more cost-effective to measure performance records on group level compared to individual level. Several studies have reflected on the fact that a large number of records are available on group level in many animal breeding programs, for instance FI in cages of mink (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Shirali <em>et al.</em> 2015</a>), egg production and body weight in groups of laying hens (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Peeters <em>et al.</em> 2013</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Biscarini <em>et al.</em> 2010</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Biscarini <em>et al.</em> 2008</a>).</p><p id="p-4">The use of such records for genetic evaluation were first suggested in a simulation study by <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Olson <em>et al.</em> (2006)</a>, who pooled individual records together to represent the joint performance of a group of animals, and demonstrated the efficacy of using group records with the usual mixed model for genetic evaluation. They concluded that group records can be effectively applied in animal genetic evaluation and selection decision despite a loss of evaluation accuracy. Based on real data for body weight from laying hens, <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Biscarini <em>et al.</em> (2008)</a> developed a procedure for estimating genetic parameters and predicting breeding values using group records. Their results demonstrated that, using group records for genetic analyses were both theoretically and practically feasible, provided groups had the same size. Recently, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Su <em>et al.</em> (2018)</a> further developed the approach of <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Olson <em>et al.</em> (2006)</a> in a simulation study. They concluded that it was possible to extend the approach to account for differing group size, and non-genetic random effects such as litter and pen effect for genetic parameter estimation and breeding value prediction. These approaches for genetic analyses of group records have revealed that group composition and group size are the vital factors influencing the accuracy of genetic parameter estimation and genetic evaluation when using group measurements (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Peeters <em>et al.</em> 2013</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Su <em>et al.</em> 2018</a>). However, all these applications of using group measurements were with a single measurement from each group.</p><p id="p-5">In order to effectively use all information in practical breeding programs, it would be attractive to further extend to models for analyzing longitudinal records. A particular concern with group records on longitudinal traits is that animals may drop out from the groups during the test, due to <em>e.g.</em>, sickness or death. Hence, such group records at different time points may not represent the same animals.</p><p id="p-6">The objectives of this study were (1) to develop a random regression model based on repeated group measurements with a consideration of drop out animals for estimation of variance components (VCs) and genetic evaluation; and (2) to examine the effect of group composition on estimates of VC and accuracy of predicting breeding values.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Simulated Data</h3><p id="p-7">A dataset based on individual measurements for FI in a pig population was simulated. The simulated population consisted of 11 generations without overlap. In each generation, 30 sires were selected and each sire was mated to 20 dams with a litter size of 6 individuals. For simplicity, random selection and random mating were performed in the simulated population and sows were assumed to have one parity only. Thus, the possible genetic trend across generations was not accounted for in this study. Each individual had 6 measurements on FI at 6 different time points reflecting 6 different weeks during the test period. In this study, the group measurement was defined as the sum of the individual measurements within a group. Consequently, each group had 6 measurements reflecting 6 different time points. The phenotypes from the last 5 generations were used for VCs estimation and genetic evaluation.</p><p id="p-8">The simulated phenotypic records consisted of additive genetic, permanent environmental, and residual effects. First order Legendre polynomials were used to generate both additive genetic and permanent environmental effects; hence, additive genetic effects for each animal contained two coefficients, intercept (b<sub>0</sub>) and slope (b<sub>1</sub>). Under such a design, additive genetic effects for the founder animals were drawn directly from a multivariate normal distribution <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-1.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-2.gif"/></noscript></span></span> is a 2Ã2 genetic covariance matrix of the intercept and regression coefficient; thus, the breeding values for founder animals were calculated as <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-3.gif"/></noscript></span></span>, where <strong>L</strong> is a 6Ã2 matrix of Legendre polynomials for the 6 test time points; additive genetic effects for each offspring were calculated as <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-4.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-5.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-6.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-7.gif"/></noscript></span></span> are the vector of breeding values for animal <em>i</em>, with sire <em>s</em> and dam <em>d</em>, respectively. The vector <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-8.gif"/></noscript></span></span> is the Mendelian sampling terms for animal <em>i</em> drawn from a multivariate normal distribution <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-9.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-10.gif"/></noscript></span></span>; where <em>F<sub>s</sub></em> and <em>F<sub>d</sub></em> are the inbreeding coefficients of sire <em>s</em> and dam <em>d</em> for animal <em>i</em>, respectively. Permanent environmental effects were drawn from a multivariate normal distribution with mean of zero and variance of <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-11.gif"/></noscript></span></span>. The heritabilities (<span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-12.gif"/></noscript></span></span>) assumed along the testing time trajectory are shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. By assuming a homogeneous residual variance in the simulation, residual effect for each animal was drawn from a normal distribution with mean of zero and variance of <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-13.gif"/></noscript></span></span>. It was assumed that there were no interactions among these effects.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2935/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The trajectory of simulated (true) heritability, and heritability estimates (mean of 10 replicates) as a function of days on test for different drop out scenarios within individual measurement and each grouping strategy. No_Drop: measurements without drop out animals; Dropran: drop out animals were randomly selected and the drop out time of each drop out animal was sampled from a vector of the integers from 1 to 6 representing the six testing time points; Dropphe: six time intervals were defined based on six testing time points, the number of drop out animals at each interval was sampled from a Poisson distribution with a mean equal to the defined proportion of drop out animals divided by six. Animals were ranked based on their phenotypes at each testing time point, then dropped out based on the number sampled from the Poisson distribution; medium within and across pen genetic relationship (Group3&#xD7;4): group was consisted of 4 different families with 3 pigs from each family; high within group relationship (Group6&#xD7;2): all animals from 2 different litters were allocated to a group; low within group relationship (Group1&#xD7;12): all animals were from different litters." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-916533441" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The trajectory of simulated (true) heritability, and heritability estimates (mean of 10 replicates) as a function of days on test for different drop out scenarios within individual measurement and each grouping strategy. No_Drop: measurements without drop out animals; Drop&lt;sub&gt;ran&lt;/sub&gt;: drop out animals were randomly selected and the drop out time of each drop out animal was sampled from a vector of the integers from 1 to 6 representing the six testing time points; Drop&lt;sub&gt;phe&lt;/sub&gt;: six time intervals were defined based on six testing time points, the number of drop out animals at each interval was sampled from a Poisson distribution with a mean equal to the defined proportion of drop out animals divided by six. Animals were ranked based on their phenotypes at each testing time point, then dropped out based on the number sampled from the Poisson distribution; medium within and across pen genetic relationship (Group&lt;sub&gt;3&#xD7;4&lt;/sub&gt;): group was consisted of 4 different families with 3 pigs from each family; high within group relationship (Group&lt;sub&gt;6&#xD7;2&lt;/sub&gt;): all animals from 2 different litters were allocated to a group; low within group relationship (Group&lt;sub&gt;1&#xD7;12&lt;/sub&gt;): all animals were from different litters.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2935/F1.medium.gif" width="440" height="413"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/9/2935/F1.medium.gif" width="440" height="413"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2935/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2935/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494095" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-9" class="first-child">The trajectory of simulated (true) heritability, and heritability estimates (mean of 10 replicates) as a function of days on test for different drop out scenarios within individual measurement and each grouping strategy. No_Drop: measurements without drop out animals; Drop<sub>ran</sub>: drop out animals were randomly selected and the drop out time of each drop out animal was sampled from a vector of the integers from 1 to 6 representing the six testing time points; Drop<sub>phe</sub>: six time intervals were defined based on six testing time points, the number of drop out animals at each interval was sampled from a Poisson distribution with a mean equal to the defined proportion of drop out animals divided by six. Animals were ranked based on their phenotypes at each testing time point, then dropped out based on the number sampled from the Poisson distribution; medium within and across pen genetic relationship (Group<sub>3Ã4</sub>): group was consisted of 4 different families with 3 pigs from each family; high within group relationship (Group<sub>6Ã2</sub>): all animals from 2 different litters were allocated to a group; low within group relationship (Group<sub>1Ã12</sub>): all animals were from different litters.</p><div class="sb-div caption-clear"></div></div></div><p id="p-10">Individual phenotype for each animal at each time point on the test was obtained as <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-14.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-15.gif"/></noscript></span></span> is a 1Ã2 matrix coefficients of Legendre polynomials at time point <em>t</em>; <strong>a</strong><sub>t</sub> and <strong>p</strong><sub>t</sub> are 2Ã1 matrices of sampled intercept and regression coefficient of each animal for additive genetic and permanent environmental effects, respectively; e<sub>t</sub> is a scalar of sampled residual. A pen was constructed by allocating 12 animals to a group unit.</p></div><div id="sec-3" class="subsection"><h3>Group Composition Scenarios</h3><p id="p-11">Three grouping scenarios based on genetic relatedness of the group members were investigated for VCs estimation and genetic evaluation: (1) medium within and across pen genetic relationship (Group<sub>3Ã4</sub>): a litter was randomly split into two sub-litters of size 3, group was consisted of 4 different sub-litters (families); (2) high within group relationship (Group<sub>6Ã2</sub>): all pigs from 2 different litters were allocated to a group; (3) low within group relationship (Group<sub>1Ã12</sub>): group consisted of 12 pigs, which were randomly selected from 12 different litters. The grouping process was conducted within generation.</p><p id="p-12">To investigate the effect of the drop out animals during test period, a proportion of animals (15%) from the last five generations (since only phenotypes of animals from the last five generations were used for analysis) was set as the drop out animals. Once the animal was dropped out, the phenotypes of that animal from the drop out time point and forward were set to missing. In the present study, two drop out strategies within each grouping scenario were assessed: (1) Drop<sub>ran</sub>: drop out animals were randomly selected, and the drop out time of each drop out animal was sampled randomly from a vector of the integers from 1 to 6 representing the six testing time points; (2) Drop<sub>phe</sub>: six time intervals were defined based on six testing time points, the number of drop out animals at each interval was sampled from a Poisson distribution with a mean equal to the total size of drop out animals divided by six. Animals were ranked from low to high level based on their phenotypes at each testing time point, then a number of animals with lowest of phenotypes were dropped out. The number of drop out animals was based on the number sampled from the Poisson distribution. For each scenario, we performed 10 replicates.</p><p id="p-13">The simulation was conducted in R (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">R Core Team 2019</a>) and Julia (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bezanson <em>et al.</em> 2017</a>). The scripts are in supplemental files.</p></div><div id="sec-4" class="subsection"><h3>Statistical models</h3><p id="p-14">Repeated individual measurements were analyzed by random regression model as follows:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-16.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-17.gif"/></noscript></span></span> were the phenotype of animal <em>i</em> measured on time point <em>t</em>; <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-18.gif"/></noscript></span></span> is the <em>k</em><sup>th</sup> fixed regression coefficient; <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-19.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-20.gif"/></noscript></span></span> are the <em>k</em><sup>th</sup> random regression coefficients of animal <em>i</em> for additive genetic and permanent environmental effects, respectively; <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-21.gif"/></noscript></span></span> is the Legendre polynomials as the covariates for the record of animal <em>i</em> measured on time point <em>t</em>; <em>nf</em> is the order of polynomials fitted as fixed regressions; <em>nr</em> is the order of polynomials for additive genetic effects; <em>np</em> is the order of polynomials for permanent environment effect, <em>nf</em>, <em>nr</em>, and <em>np</em> were all first order Legendre polynomials; <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-22.gif"/></noscript></span></span> is the random residual with assumption of homogeneous variance. It was assumed that the additive genetic effects follow <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-23.gif"/></noscript></span></span> the permanent environmental effects follow <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-24.gif"/></noscript></span></span> and the residual effects follow <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-25.gif"/></noscript></span></span> where <strong>A</strong> is the numerator relationship matrix, and <strong>I</strong> is the identity matrix.</p><p id="p-15">Repeated group measurements were analyzed by the following random regression model obtained by summing the terms in the model for individual measurements within each group <em>j</em>, and is as follows:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-26.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-27.gif"/></noscript></span></span> is the phenotype of group <em>j</em> measured at time <em>t</em> with group size (<em>GS</em>) of <em>m</em> (<em>m</em> = 1, 2, â¦... 12); <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-28.gif"/></noscript></span></span> is the fixed effect of group size <em>m</em> for group <em>j</em>; <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-29.gif"/></noscript></span></span> is the <em>k</em><sup>th</sup> fixed regression coefficient for group <em>j</em> nested within group size effect <em>m</em>; <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-30.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-31.gif"/></noscript></span></span> are the <em>k</em><sup>th</sup> random regression for additive genetic and permanent environmental effects, respectively, for animal <em>i</em> in group <em>j</em>; <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-32.gif"/></noscript></span></span> is the covariate coefficient of Legendre polynomials for animal <em>i</em> in group <em>j</em> measured at time <em>t</em>; <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-33.gif"/></noscript></span></span> is the number of animals in group <em>j</em> at time <em>t</em>; <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-34.gif"/></noscript></span></span> is the random residual with the variance of <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/embed/mml-math-35.gif"/></noscript></span></span>, where <strong>D</strong> is a diagonal matrix with elements of group sizes for group <em>j</em>.</p><p id="p-16">Variance components were estimated by restricted maximum likelihood (REML) (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Patterson and Thompson 1971</a>) with the average information algorithm (AI-REML) (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Gilmour <em>et al.</em> 1995</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Johnson and Thompson 1995</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Madsen <em>et al.</em> 1994</a>), breeding values were predicted by best linear unbiased prediction (BLUP) (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Henderson 1985</a>). All analyses were performed using the DMU package (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Madsen and Jensen 2013</a>).</p></div><div id="sec-5" class="subsection"><h3>Validation</h3><p id="p-17">A selection index of FI was defined in this study. Based on the characteristics of longitudinal records, where performance is measured sequentially over time on each animal. The FI measurements for all 6 time points during the test phase can be considered as 6 different and correlated traits. The selection index was then calculated as a weighted sum of the estimated breeding values (EBV) from all the 6 testing time points using weight of 1 for each trait. To compare the efficiency of using group measurements with individual measurements, accuracy of genetic evaluation was computed as the Pearson correlation coefficient between EBVs and simulated breeding values for animals having phenotypic data. Unbiasedness of genetic evaluation was evaluated through the regression of simulated (true) breeding values on the EBVs for animals having phenotypic data.</p></div><div id="sec-6" class="subsection"><h3>Data Availability</h3><p id="p-18">Simulated data available at Figshare: <a href="https://doi.org/10.6084/m9.figshare.8047367">https://doi.org/10.6084/m9.figshare.8047367</a>.</p><p id="p-19">Files at this URL are simulated pedigree and phenotypes based on different grouping scenarios and drop out strategies. Simulation scripts available at Figshare: <a href="https://doi.org/10.6084/m9.figshare.8312555">https://doi.org/10.6084/m9.figshare.8312555</a>.</p><p id="p-20">The authors affirm that all data necessary for confirming the conclusions of the article are present within the article, and tables. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8046995">https://doi.org/10.25387/g3.8046995</a>.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><p id="p-21"><a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> presents the mean and standard deviation (SD) of estimated VCs over 10 replicates using repeated individual and group measurements based on different grouping and drop out scenarios. Unbiased estimates of VCs were obtained when analyzing individual measurements. In general, using group measurements yielded similar VCs estimates but with larger SDs as the corresponding scenario with individual measurements. Similar VC estimates were observed from random drop out scenarios (Drop<sub>ran</sub>) compared with no drop out scenario (No_Drop). Different grouping scenarios based on genetic relatedness of the group members produced similar VCs estimation. The grouping scenario of high within group relationship (Group<sub>6Ã2</sub>) resulted in lowest SDs for genetic (co)variances and residual variance among all the grouping scenarios. However, the SDs for permanent environmental (co)variances were the highest among all the grouping scenarios.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494066/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494066/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Mean and standard deviation (SD) of estimated variance components (VC) of genetic (G<sub>ij</sub>), permanent environmental (P<sub>ij</sub>) and residual effects (<span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/T1/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2935/T1/embed/mml-math-36.gif"/></noscript></span></span>) over 10 replicates from repeated individual measurements and group measurements based on different grouping and drop out scenarios<sup>1</sup></span><div class="sb-div caption-clear"></div></div></div><p id="p-24"><a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> presents a multi-panel plot showing the trajectory of simulated (true) heritability, and heritability estimates (mean of 10 replicates) as a function of days on test for different drop out scenarios within each grouping strategy. Clearly, the heritabilities were not constant throughout the test period with a tendency toward lower heritability estimates in the middle and late stages of test period. This trend was consistent across different grouping strategies. The trajectories of heritability estimates were similar between no drop out scenario (No_Drop) and random drop out scenarios (Drop<sub>ran</sub>). Lowest estimates of heritability were observed for scenario of Drop<sub>phe</sub>.</p><p id="p-25">Mean and SD of evaluation accuracy and bias using repeated individual and group measurements based on different grouping and drop out scenarios are shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. As expected, the highest accuracy was achieved by using individual measurements. As can be seen from the table, compared with no drop out scenarios, drop out scenarios show that there is no loss of evaluation accuracy. Among the grouping scenarios, the high within group relationship (Group<sub>6Ã2</sub>) had highest accuracy, whereas the low within group relationship (Group<sub>1Ã12</sub>) had lowest accuracy. Within drop out scenarios, random drop out scenarios (Drop<sub>ran</sub>) and scenarios of drop out by phenotypes (Drop<sub>phe</sub>) resulted in similar accuracies. Unbiased evaluations reflected by regression of simulated on predicted genetic values, were obtained when using individual measurements and group measurements from no drop out grouping scenarios (No_Drop) and random drop out scenarios (Drop<sub>ran</sub>). Deflation of EBVs from scenarios of drop out by phenotypes (Drop<sub>phe</sub>) were found across all the scenarios.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494194/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494194/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Mean and standard deviation (SD) of prediction and bias of estimated breeding values (EBV) over 10 replicates from repeated individual measurements and group measurements based on different grouping and drop out scenarios<sup>1</sup></span><div class="sb-div caption-clear"></div></div></div></div><div class="section discussion" id="sec-8"><h2 class="">Discussion</h2><p id="p-28">In the present study, a random regression model for analyzing repeated group measurements was developed and investigated on simulated data. The model was able to consider varying group sizes during the test period caused by drop out animals, which addresses an important practical issue. Different grouping strategies in terms of genetic relatedness among the group members were also assessed. The results show that the random regression model for longitudinal group measurements yielded unbiased VCs estimation for scenarios of random drop out, and the high within group relationship (Group<sub>6Ã2</sub>) achieved the highest accuracy among all the grouping scenarios.</p><div id="sec-9" class="subsection"><h3>Group <em>vs.</em> individual measurements</h3><p id="p-29">The SDs of estimated VCs from group measurements were larger compared with those from individual measurements. This indicates less information was available due to merging individual measurements into group measurements. Larger SE of genetic parameter estimates was also reported by (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Biscarini <em>et al.</em> 2008</a>) when group analysis of body weight on cage level was compared to individual body weight recording for laying hens. <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Peeters <em>et al.</em> (2013)</a> used simulated pooled records to estimate VCs for traits affected by social interactions in laying hens and found the VC estimates from group records did not differ significantly from the simulated VCs. The results in the current study is in line with a recent study of <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Su <em>et al.</em> (2018)</a>, where VCs estimated from univariate group records were reported to be consistent with those estimated from univariate individual records but with larger standard errors.</p><p id="p-30">The accuracy of EBV is of great importance for animal improvement programs as animals are ranked based on their breeding values. In the current study, we observed lower accuracies when using group measurements compared with using individual measurements. The reduction of evaluation accuracy is as expected and due to the reduction of information content by merging the animals into groups. <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Olson <em>et al.</em> (2006)</a> used simulated group measurements for genetic evaluation, and reported that accuracies of using group measurements compared to those of using individual measurements decreased from 0.77 to 0.50, from 0.77 to 0.53 for random grouped animals and animals grouped based on sires, respectively; the reduction of accuracy were more profound as the group size increased.</p><p id="p-31"><a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Biscarini <em>et al.</em> (2008)</a> reported that EBV accuracy obtained from pooled records was about 70â80% of the accuracy of EBV predicted from individual records. <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Su <em>et al.</em> (2018)</a> reported 68% for the EBV accuracy achieved from group records to that from individual records. However, the notable cost savings of using group measurements compared with using individual measurements can counteract the reduced prediction accuracy.</p></div><div id="sec-10" class="subsection"><h3>Group Composition</h3><p id="p-32">In the present study, we simulated the trait of FI in a pig population, hence, a pen was defined as a group. In practice, the strategies of allocating pigs into pens varies across herds; therefore, a good grouping strategy can maximize the effective use of group information. In this study, this was demonstrated by the group scenario of Group<sub>6Ã2</sub> in terms of highest accuracies of genetic evaluation among all the grouping scenarios, where the animals in the group had the closest genetic relationships compared with scenarios of Group<sub>3Ã4</sub> and Group<sub>1Ã12</sub>. Our results are in line with the previous simulation study by (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Olson <em>et al.</em> 2006</a>) who found that allocating animals to groups on the basis of their sire information achieved better accuracies of breeding values than allocating animals to groups on the basis of maternal grandsire (MGS) and random allocation. Through theoretical derivations and simulations, <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Peeters <em>et al.</em> (2013)</a> showed that smaller SE on genetic variance estimates were achieved when allocating animals from the same family to the group. Interestingly, when directly using the VCs from simulation, <a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">Su <em>et al.</em> (2018)</a> verified the accuracy of breeding values were increased with increased genetic relationships of the group members. A possible explanation for these results might be that, when we use the group measurements, the proportion of additive genetic to phenotypic variance increases as the genetic relationships among the group members increase (<a id="xref-ref-16-6" class="xref-bibr" href="#ref-16">Su <em>et al.</em> 2018</a>). However, it is important to note that one major difference between their studies and the current study is that we used repeated group measurements whereas they used single group measurement.</p></div><div id="sec-11" class="subsection"><h3>Drop outs</h3><p id="p-33">The method presented in this study allows consideration of missing phenotypes caused by drop out animals in each group. In particular, the scenario of dropping out animals by ranking of phenotypes (Drop<sub>phe</sub>) was designed to reflect the practical situation, where animals with poor phenotypes are more susceptible to diseases and thus, more likely to be culled. We observed reduced VC estimates from the scenario of Drop<sub>phe</sub> for both individual and group measurement (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), leading to larger biases of EBV (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>), these results were mainly influenced by the decreased phenotypic and genetic variances due to reduced number of animals with lower performance records. Therefore, this seems that, it would be preferable to perform genetic evaluation using group measurements with VCs estimated from individual measurements, if feasible.</p></div><div id="sec-12" class="subsection"><h3>Extra random effects</h3><p id="p-34">In the current study, litter and pen effects were not simulated in order to provide a simple platform for explanation of the approach. However, those effects were investigated by</p><p id="p-35"><a id="xref-ref-16-7" class="xref-bibr" href="#ref-16">Su <em>et al.</em> (2018)</a> based on a univariate analysis (single measurement), where group records were modeled with considering of litter and pen effects, and compared with the models ignoring litter and pen effects. Their results showed that ignoring litter and pen effects had no influence on the prediction accuracy. Extending the use of the current model to include extra random effects can refer to the same methodology as presented by <a id="xref-ref-16-8" class="xref-bibr" href="#ref-16">Su <em>et al.</em> (2018)</a>.</p><p id="p-36">The design of the current simulation study was based on a single trait model. Nevertheless, genetic evaluations are mostly carried out using a multi-trait model to take advantage of the genetic correlations among traits and estimate breeding values simultaneously. It can therefore be expected that traits with group measurements, can benefit from genetically correlated traits with individual measurements via multi-trait model (<em>e.g.</em>, FI and average daily gain).</p></div></div><div class="section conclusions" id="sec-13"><h2 class="">Conclusions</h2><p id="p-37">In conclusion, the proposed random regression model is feasible to handle repeated group measurements with consideration of drop out animals. In this study, we observed reduced VC estimates when animals were dropped out by ranking of phenotypes. However, using group measurements with drop out animals by ranking of phenotypes can provide similar accuracy but larger bias of EBV compared to those of using group measurements without dropped out animals. Therefore, it would be preferable to perform genetic evaluation using group measurements with VCs estimated from individual measurements. In addition, the findings clearly indicate that group composition plays a critical role in genetic evaluation when using group measurements. Hence, allocating animals with close genetic relationships to the same group could optimize the use of group measurements. Overall, this model could provide a cost-effective solution in breeding programs, thus it can be applied to the routine genetic evaluations and extended to more species and traits.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-38">The authors acknowledge the funding from the Green Development and Demonstration Program (GUDP Grant No. 34009-14-0849) by the Danish Ministry of Food, Agriculture and Fisheries; SEGES, Pig Research Centre; and Aarhus University.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-39">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8046995">https://doi.org/10.25387/g3.8046995</a>.</p></li><li class="fn" id="fn-6"><p id="p-40"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-04-26"><span class="received-label">Received </span>April 26, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-07-04"><span class="accepted-label">Accepted </span>July 4, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2019 Gao <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.2935.1" data-doi="10.1137/141000671"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bezanson</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Edelman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Karpinski</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. B.</span> <span class="cit-name-surname">Shah</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Julia: A fresh approach to numerical computing.</span> <abbr class="cit-jnl-abbrev">SIAM Rev.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">65</span>â<span class="cit-lpage">98</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1137/141000671</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSIAM%2BRev.%26rft.volume%253D59%26rft.spage%253D65%26rft_id%253Dinfo%253Adoi%252F10.1137%252F141000671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1137/141000671&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.2935.2" data-doi="10.3382/ps.2010-00730"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Biscarini</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Bovenhuis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. D.</span> <span class="cit-name-surname">Ellen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Addo</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">van Arendonk</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Estimation of heritability and breeding values for early egg production in laying hens from pooled data.</span> <abbr class="cit-jnl-abbrev">Poult. Sci.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">1842</span>â<span class="cit-lpage">1849</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3382/ps.2010-00730</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPoult.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.3382%252Fps.2010-00730%26rft_id%253Dinfo%253Apmid%252F20709968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3382/ps.2010-00730&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20709968&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2935.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.2935.3" data-doi="10.2527/jas.2007-0757"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Biscarini</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Bovenhuis</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A. M.</span> <span class="cit-name-surname">van Arendonk</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Estimation of variance components and prediction of breeding values using pooled data.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">2845</span>â<span class="cit-lpage">2852</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas.2007-0757</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2007-0757%26rft_id%253Dinfo%253Apmid%252F18539838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2007-0757&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18539838&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2935.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260425000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.2935.4" data-doi="10.2307/2533274"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>,  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Thompson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. R.</span> <span class="cit-name-surname">Cullis</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span>â<span class="cit-article-title">Average information REML: An efficient algorithm for variance parameter estimation in linear mixed models.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">1440</span>â<span class="cit-lpage">1450</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/2533274</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D51%26rft.spage%253D1440%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2533274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2533274&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995TQ15600022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.2935.5" data-doi="10.2527/jas1985.601111x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>,  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span>â<span class="cit-article-title">Best Linear Unbiased Prediction of Nonadditive Genetic Merits in Noninbred Populations.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">111</span>â<span class="cit-lpage">117</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas1985.601111x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DHenderson%26rft.auinit1%253DC.%2BR.%26rft.volume%253D60%26rft.issue%253D1%26rft.spage%253D111%26rft.epage%253D117%26rft.atitle%253DBest%2BLinear%2BUnbiased%2BPrediction%2Bof%2BNonadditive%2BGenetic%2BMerits%2Bin%2BNoninbred%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas1985.601111x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas1985.601111x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1985AAM9700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.2935.6" data-doi="10.3168/jds.S0022-0302(95)76654-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>,  <span class="cit-name-given-names">D. L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Thompson</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span>â<span class="cit-article-title">Restricted Maximum-Likelihood-Estimation of Variance-Components for Univariate Animal-Models Using Sparse-Matrix Techniques and Average Information.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">449</span>â<span class="cit-lpage">456</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.S0022-0302(95)76654-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DJohnson%26rft.auinit1%253DD.%2BL.%26rft.volume%253D78%26rft.issue%253D2%26rft.spage%253D449%26rft.epage%253D456%26rft.atitle%253DRestricted%2BMaximum%2BLikelihood%2BEstimation%2Bof%2BVariance%2BComponents%2Bfor%2BUnivariate%2BAnimal%2BModels%2BUsing%2BSparse%2BMatrix%2BTechniques%2Band%2BAverage%2BInformation%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.S0022-0302%252895%252976654-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.S0022-0302(95)76654-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995QH36700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.2935.7" data-doi="10.3168/jds.2017-12775"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Berglund</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. F.</span> <span class="cit-name-surname">Fikse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Lassen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. H.</span> <span class="cit-name-surname">Lidauer</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Neglect of lactation stage leads to naive assessment of residual feed intake in dairy cattle.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">9076</span>â<span class="cit-lpage">9084</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2017-12775</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BDairy%2BSci.%26rft.volume%253D100%26rft.spage%253D9076%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2017-12775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2017-12775&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-web" id="cit-9.9.2935.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Madsen</span>,  <span class="cit-name-given-names">P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jensen</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> A Userâs Guide to DMU - A Package for Analysing Multivariate Mixed Models. Version 6, Release 5.2: <a href="http://dmu.agrsci.dk/DMU/Doc/Current/dmuv6_guide.5.2.pdf">http://dmu.agrsci.dk/DMU/Doc/Current/dmuv6_guide.5.2.pdf</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.2935.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Madsen</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jensen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Thompson</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> Estimation of (co)variance components by REML in multivariate mixed linear models using average of observed and expected information, pp. <span class="cit-fpage">19</span>â<span class="cit-lpage">22</span> in <em>5th World Congr. Genet. Appl. Livest. Prod</em>., Guelph, Canada.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.2935.10" data-doi="10.2527/2006.84188x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Olson</span>,  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Garrick</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Enns</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Predicting breeding values and accuracies from group in comparison to individual observations.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">88</span>â<span class="cit-lpage">92</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/2006.84188x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DOlson%26rft.auinit1%253DK.%2BM.%26rft.volume%253D84%26rft.issue%253D1%26rft.spage%253D88%26rft.epage%253D92%26rft.atitle%253DPredicting%2Bbreeding%2Bvalues%2Band%2Baccuracies%2Bfrom%2Bgroup%2Bin%2Bcomparison%2Bto%2Bindividual%2Bobservations%26rft_id%253Dinfo%253Adoi%252F10.2527%252F2006.84188x%26rft_id%253Dinfo%253Apmid%252F16361494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/2006.84188x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16361494&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2935.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.2935.11" data-doi="10.1093/biomet/58.3.545"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>,  <span class="cit-name-given-names">H. D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Thompson</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span>â<span class="cit-article-title">Recovery of Inter-Block Information When Block Sizes Are Unequal.</span> <abbr class="cit-jnl-abbrev">Biometrika</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">545</span>â<span class="cit-lpage">554</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/biomet/58.3.545</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft.stitle%253DBiometrika%26rft.aulast%253DPATTERSON%26rft.auinit1%253DH.%2BD.%26rft.volume%253D58%26rft.issue%253D3%26rft.spage%253D545%26rft.epage%253D554%26rft.atitle%253DRecovery%2Bof%2Binter-block%2Binformation%2Bwhen%2Bblock%2Bsizes%2Bare%2Bunequal%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252F58.3.545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/biomet/58.3.545&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1971L054700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.2935.12" data-doi="10.1186/1297-9686-45-27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peeters</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. D.</span> <span class="cit-name-surname">Ellen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Bijma</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Using pooled data to estimate variance components and breeding values for traits affected by social interactions.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">27</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-45-27</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D45%26rft.spage%253D27%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-45-27%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-45-27&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-book" id="cit-9.9.2935.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2019</span>â<span class="cit-source">R: A Language and Environment for Statistical Computing</span>, <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.2935.14" data-doi="10.1017/S1751731115000956"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shirali</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. H.</span> <span class="cit-name-surname">Nielsen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. H.</span> <span class="cit-name-surname">Moller</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jensen</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Longitudinal analysis of residual feed intake and BW in mink using random regression with heterogeneous residual variance.</span> <abbr class="cit-jnl-abbrev">Animal</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1597</span>â<span class="cit-lpage">1604</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S1751731115000956</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%26rft.volume%253D9%26rft.spage%253D1597%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS1751731115000956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S1751731115000956&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.2935.15" data-doi="10.1016/j.livsci.2017.03.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shirali</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. F.</span> <span class="cit-name-surname">Varley</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jensen</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Longitudinal genetic dissection of feed efficiency and feeding behaviour in MaxGro pigs.</span> <abbr class="cit-jnl-abbrev">Livest. Sci.</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">79</span>â<span class="cit-lpage">85</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.livsci.2017.03.017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BSci.%26rft.volume%253D199%26rft.spage%253D79%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.livsci.2017.03.017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.livsci.2017.03.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.9.2935.16" data-doi="10.1186/s12711-018-0413-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Su</span>,  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Madsen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Nielsen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Ostersen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Shirali</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span>â<span class="cit-article-title">Estimation of variance components and prediction of breeding values based on group records from varying group sizes.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">42</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-018-0413-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D50%26rft.spage%253D42%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-018-0413-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-018-0413-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/9/2935.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/9/2925" title="Joint Use of Genome, Pedigree, and Their Interaction with Environment for Predicting the Performance of Wheat Lines in New Environments" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/9/2941" title="When West Meets East: The Origins and Spread of Weedy Rice Between Continental and Island Southeast Asia" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/9"><span>Volume 9 Issue 9, September 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/9.cover-source.jpg?itok=4kZTDz_V" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/2935" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-bFLmWJFR2Y7ghCDv8pG1VeGzZPRH85sQDYcG8KrIzMs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/9/2935" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/9/2935" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/9/2935" class="active" data-icon-position="" data-hide-link-title="0">Use of Repeated Group Measurements with Drop Out Animals for Variance Component Estimation and Genetic Evaluation: A Simulation Study</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494046" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-cZ0WdwBf73YwUyRddvBMTJSLLJyxl39qIP0oaJVrWOg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/9/2935.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/9/2935" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-VLISUz-aEQ0AmLKr8tWLzWpNZZ3puKh2xx1Ijj1xMOs" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/9/2935" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/9/2935.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494046" id="node494046--2" data-pisa="ggg;9/9/2935" data-pisa-master="ggg;g3.119.400484" data-apath="/ggg/9/9/2935.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Use of Repeated Group Measurements with Drop Out Animals for Variance Component Estimation and Genetic Evaluation: A Simulation Study</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-6018-0766" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Hongding Gao</span>, <span class="highwire-citation-author" data-delta="1">Bjarne Nielsen</span>, <span class="highwire-citation-author" data-delta="2">Guosheng Su</span>, <span class="highwire-citation-author" data-delta="3">Per Madsen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-3291-8468" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Just Jensen</span>, <span class="highwire-citation-author" data-delta="5">Ole F. Christensen</span>, <span class="highwire-citation-author" data-delta="6">Tage Ostersen</span> and <span class="highwire-citation-author" data-delta="7">Mahmoud Shirali</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2935-2940;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400484 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494046--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hongding Gao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Quantitative Genetics and Genomics, Department of Molecular Biology and Genetics, Aarhus University, Tjele 8830, Denmark and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hongding%2BGao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gao%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHongding%2BGao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6018-0766" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hongding Gao</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hongding.gao@mbg.au.dk" class="" data-icon-position="" data-hide-link-title="0">hongding.gao@mbg.au.dk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bjarne Nielsen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>SEGES, Pig Research Centre, Copenhagen 1609, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bjarne%2BNielsen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nielsen%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABjarne%2BNielsen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Guosheng Su </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Quantitative Genetics and Genomics, Department of Molecular Biology and Genetics, Aarhus University, Tjele 8830, Denmark and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guosheng%2BSu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Su%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuosheng%2BSu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Per Madsen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Quantitative Genetics and Genomics, Department of Molecular Biology and Genetics, Aarhus University, Tjele 8830, Denmark and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Per%2BMadsen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Madsen%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APer%2BMadsen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Just Jensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Quantitative Genetics and Genomics, Department of Molecular Biology and Genetics, Aarhus University, Tjele 8830, Denmark and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Just%2BJensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jensen%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJust%2BJensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3291-8468" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Just Jensen</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ole F. Christensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Quantitative Genetics and Genomics, Department of Molecular Biology and Genetics, Aarhus University, Tjele 8830, Denmark and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ole%2BF.%2BChristensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Christensen%20OF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOle%2BF.%2BChristensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Tage Ostersen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>SEGES, Pig Research Centre, Copenhagen 1609, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tage%2BOstersen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ostersen%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATage%2BOstersen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Mahmoud Shirali </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Quantitative Genetics and Genomics, Department of Molecular Biology and Genetics, Aarhus University, Tjele 8830, Denmark and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mahmoud%2BShirali%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shirali%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMahmoud%2BShirali%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/9/2935" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/9/2935&amp;title=Use%20of%20Repeated%20Group%20Measurements%20with%20Drop%20Out%20Animals%20for%20Variance%20Component%20Estimation%20and%20Genetic%20Evaluation%3A%20A%20Simulation%20Study" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/9/2935&amp;title=Use%20of%20Repeated%20Group%20Measurements%20with%20Drop%20Out%20Animals%20for%20Variance%20Component%20Estimation%20and%20Genetic%20Evaluation%3A%20A%20Simulation%20Study" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/9/2935&amp;title=Use%20of%20Repeated%20Group%20Measurements%20with%20Drop%20Out%20Animals%20for%20Variance%20Component%20Estimation%20and%20Genetic%20Evaluation%3A%20A%20Simulation%20Study" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/9/2935&amp;text=Use%20of%20Repeated%20Group%20Measurements%20with%20Drop%20Out%20Animals%20for%20Variance%20Component%20Estimation%20and%20Genetic%20Evaluation%3A%20A%20Simulation%20Study" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/9/2935&amp;title=Use%20of%20Repeated%20Group%20Measurements%20with%20Drop%20Out%20Animals%20for%20Variance%20Component%20Estimation%20and%20Genetic%20Evaluation%3A%20A%20Simulation%20Study" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/9/2935&amp;t=Use%20of%20Repeated%20Group%20Measurements%20with%20Drop%20Out%20Animals%20for%20Variance%20Component%20Estimation%20and%20Genetic%20Evaluation%3A%20A%20Simulation%20Study" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/9/2935&amp;title=Use%20of%20Repeated%20Group%20Measurements%20with%20Drop%20Out%20Animals%20for%20Variance%20Component%20Estimation%20and%20Genetic%20Evaluation%3A%20A%20Simulation%20Study" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/9/2935&amp;title=Use%20of%20Repeated%20Group%20Measurements%20with%20Drop%20Out%20Animals%20for%20Variance%20Component%20Estimation%20and%20Genetic%20Evaluation%3A%20A%20Simulation%20Study" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494046/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494046" id="node494046--3" data-pisa="ggg;9/9/2935" data-pisa-master="ggg;g3.119.400484" data-apath="/ggg/9/9/2935.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Use of Repeated Group Measurements with Drop Out Animals for Variance Component Estimation and Genetic Evaluation: A Simulation Study</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-6018-0766" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Hongding Gao</span>, <span class="highwire-citation-author" data-delta="1">Bjarne Nielsen</span>, <span class="highwire-citation-author" data-delta="2">Guosheng Su</span>, <span class="highwire-citation-author" data-delta="3">Per Madsen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-3291-8468" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Just Jensen</span>, <span class="highwire-citation-author" data-delta="5">Ole F. Christensen</span>, <span class="highwire-citation-author" data-delta="6">Tage Ostersen</span> and <span class="highwire-citation-author" data-delta="7">Mahmoud Shirali</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2935-2940;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400484 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494046--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hongding Gao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Quantitative Genetics and Genomics, Department of Molecular Biology and Genetics, Aarhus University, Tjele 8830, Denmark and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hongding%2BGao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gao%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHongding%2BGao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6018-0766" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hongding Gao</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hongding.gao@mbg.au.dk" class="" data-icon-position="" data-hide-link-title="0">hongding.gao@mbg.au.dk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bjarne Nielsen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>SEGES, Pig Research Centre, Copenhagen 1609, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bjarne%2BNielsen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nielsen%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABjarne%2BNielsen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Guosheng Su </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Quantitative Genetics and Genomics, Department of Molecular Biology and Genetics, Aarhus University, Tjele 8830, Denmark and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guosheng%2BSu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Su%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuosheng%2BSu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Per Madsen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Quantitative Genetics and Genomics, Department of Molecular Biology and Genetics, Aarhus University, Tjele 8830, Denmark and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Per%2BMadsen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Madsen%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APer%2BMadsen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Just Jensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Quantitative Genetics and Genomics, Department of Molecular Biology and Genetics, Aarhus University, Tjele 8830, Denmark and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Just%2BJensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jensen%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJust%2BJensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3291-8468" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Just Jensen</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ole F. Christensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Quantitative Genetics and Genomics, Department of Molecular Biology and Genetics, Aarhus University, Tjele 8830, Denmark and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ole%2BF.%2BChristensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Christensen%20OF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOle%2BF.%2BChristensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Tage Ostersen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>SEGES, Pig Research Centre, Copenhagen 1609, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tage%2BOstersen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ostersen%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATage%2BOstersen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Mahmoud Shirali </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Quantitative Genetics and Genomics, Department of Molecular Biology and Genetics, Aarhus University, Tjele 8830, Denmark and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mahmoud%2BShirali%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shirali%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMahmoud%2BShirali%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2935&amp;rft.epage=2940&amp;rft.atitle=Use%20of%20Repeated%20Group%20Measurements%20with%20Drop%20Out%20Animals%20for%20Variance%20Component%20Estimation%20and%20Genetic%20Evaluation%3A%20A%20Simulation%20Study&amp;rft.volume=9&amp;rft.issue=9&amp;rft.date=2019-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Gao%2C+Hongding&amp;rft.au=Nielsen%2C+Bjarne&amp;rft.au=Su%2C+Guosheng&amp;rft.au=Madsen%2C+Per&amp;rft.au=Jensen%2C+Just&amp;rft.au=Christensen%2C+Ole+F.&amp;rft.au=Ostersen%2C+Tage&amp;rft.au=Shirali%2C+Mahmoud"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494046/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494046/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494046/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494046/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494046/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494046/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494046/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494046/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494046/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494046/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494046/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494046/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/9/2935" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/9/2935.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/9/2935.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/9/2935.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
