<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/9/2851" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/9/2851.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/9/2851.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/9/2851.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/9/2851/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/9/2851.atom" />
<meta name="HW.pisa" content="ggg;9/9/2851" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Developing a High-Quality Linkage Map for the Atlantic Killifish Fundulus heteroclitus" />
<meta name="DC.Identifier" content="10.1534/g3.119.400262" />
<meta name="DC.Date" content="2019-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Miller et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Killifish ( Fundulus heteroclitus ) are widely distributed among different aquatic environments where they demonstrate an impressive range of highly-plastic and locally adaptive phenotypes. High-throughput sequencing has begun to unravel the mechanisms and evolutionary history of these interesting features by establishing relationships in the genotype-phenotype map. However, some genotype-phenotype analyses require a higher order of contiguity than what initial scaffolded (fragmented genome assembly where contigs have been assemble into scaffolds) genome assemblies can provide. Here, we used 5,685 high-quality RAD-Seq markers from a single mapping family to order 84% of the scaffolded genome assembly to 24 chromosomes. This serves to: 1) expand the killifish genomic toolkit, 2) estimate genome-wide recombination rates, and 3) compare genome synteny to humans and other fishes. After initially building our map, we found that the selection of thresholds for sequence data filtration highly impacted scaffold placement in the map. We outline each step of the approach that dramatically improved our map to help guide others toward more effective linkage mapping for genome assembly. Our final map supports strong conservation of genomic synteny among closely related fish species and reveals previously described chromosomal rearrangements between more distantly related clades. However, we also commonly found minor scaffold misorientations in F. heteroclitus and in other assemblies, suggesting that further mapping (such as optical mapping) is necessary for finer scale resolution of genome structure. Lastly, we discuss the problems that would be expected from misoriented/unplaced scaffolds and stress the importance of a quality mapped genome as a key feature for further investigating population and comparative genomic questions with F. heteroclitus and other taxa." />
<meta name="DC.Contributor" content="Jeffrey T. Miller" />
<meta name="DC.Contributor" content="Noah M. Reid" />
<meta name="DC.Contributor" content="Diane E. Nacci" />
<meta name="DC.Contributor" content="Andrew Whitehead" />
<meta name="article:published_time" content="2019-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Developing a High-Quality Linkage Map for the Atlantic Killifish Fundulus heteroclitus" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Killifish (&lt;i&gt;Fundulus heteroclitus&lt;/i&gt;) are widely distributed among different aquatic environments where they demonstrate an impressive range of highly-plastic and locally adaptive phenotypes. High-throughput sequencing has begun to unravel the mechanisms and evolutionary history of these interesting features by establishing relationships in the genotype-phenotype map. However, some genotype-phenotype analyses require a higher order of contiguity than what initial scaffolded (fragmented genome assembly where contigs have been assemble into scaffolds) genome assemblies can provide. Here, we used 5,685 high-quality RAD-Seq markers from a single mapping family to order 84% of the scaffolded genome assembly to 24 chromosomes. This serves to: 1) expand the killifish genomic toolkit, 2) estimate genome-wide recombination rates, and 3) compare genome synteny to humans and other fishes. After initially building our map, we found that the selection of thresholds for sequence data filtration highly impacted scaffold placement in the map. We outline each step of the approach that dramatically improved our map to help guide others toward more effective linkage mapping for genome assembly. Our final map supports strong conservation of genomic synteny among closely related fish species and reveals previously described chromosomal rearrangements between more distantly related clades. However, we also commonly found minor scaffold misorientations in &lt;i&gt;F. heteroclitus&lt;/i&gt; and in other assemblies, suggesting that further mapping (such as optical mapping) is necessary for finer scale resolution of genome structure. Lastly, we discuss the problems that would be expected from misoriented/unplaced scaffolds and stress the importance of a quality mapped genome as a key feature for further investigating population and comparative genomic questions with &lt;i&gt;F. heteroclitus&lt;/i&gt; and other taxa.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/09/01" />
<meta name="citation_mjid" content="ggg;9/9/2851" />
<meta name="citation_id" content="9/9/2851" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/9/2851" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/9/2851.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/9/2851.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/9/2851.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400262" />
<meta name="citation_pmid" content="31289021" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2851" />
<meta name="citation_lastpage" content="2862" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jeffrey T. Miller" />
<meta name="citation_author_institution" content="Department of Environmental Toxicology, Center for Population Biology, Coastal and Marine Sciences Institute, University of California, Davis, CA" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2326-1855" />
<meta name="citation_author" content="Noah M. Reid" />
<meta name="citation_author_institution" content="Department of Molecular &amp; Cell Biology, University of Connecticut, Storrs, CT, and" />
<meta name="citation_author" content="Diane E. Nacci" />
<meta name="citation_author_institution" content="US Environmental Protection Agency, Office of Research and Development, National Health and Environmental Effects Research Laboratory, Atlantic Ecology Division, Narragansett, RI" />
<meta name="citation_author" content="Andrew Whitehead" />
<meta name="citation_author_institution" content="Department of Environmental Toxicology, Center for Population Biology, Coastal and Marine Sciences Institute, University of California, Davis, CA" />
<meta name="citation_author_email" content="awhitehead@ucdavis.edu" />
<meta name="citation_reference" content="citation_journal_title=Biol. Reprod.;citation_author=B. Abraham;citation_title=Species Profiles. Life Histories and Environmental Requirements of Coastal Fishes and Invertebrates (Mid-Atlantic). MUMMICHOG AND STRIPED KILLIFISH.;citation_pages=33;citation_volume=82;citation_year=1985" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Amores;citation_author=J. Catchen;citation_author=I. Nanda;citation_author=W. Warren;citation_author=R. Walter;citation_title=A RAD-tag genetic map for the platyfish (Xiphophorus maculatus) reveals mechanisms of karyotype evolution among teleost fish.;citation_volume=197;citation_year=2014;citation_pmid=24700104;citation_doi=10.1534/genetics.114.164293" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. W. Barnett;citation_author=E. K. Garrison;citation_author=A. R. Quinlan;citation_author=M. P. Str̈mberg;citation_author=G. T. Marth;citation_title=Bamtools: A C++ API and toolkit for analyzing and managing BAM files.;citation_volume=27;citation_year=2011;citation_pmid=21493652;citation_doi=10.1093/bioinformatics/btr174" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. W. Baxter;citation_author=J. W. Davey;citation_author=J. S. Johnston;citation_author=A. M. Shelton;citation_author=D. G. Heckel;citation_title=Linkage mapping and comparative genomics using next-generation RAD sequencing of a non-model organism.;citation_pages=e19315-e19315;citation_volume=6;citation_year=2011;citation_issue=4;citation_pmid=21541297;citation_doi=10.1371/journal.pone.0019315" />
<meta name="citation_reference" content="citation_journal_title=J. Vis. Exp.;citation_author=N. L. van Berkum;citation_author=E. Lieberman-Aiden;citation_author=L. Williams;citation_author=M. Imakaev;citation_author=A. Gnirke;citation_title=Hi-C: A Method to Study the Three-dimensional Architecture of Genomes.;citation_pages=1869;citation_volume=6;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=PLoS Curr.;citation_author=R. Betancur-R;citation_author=R. E. Broughton;citation_author=E. O. Wiley;citation_author=K. Carpenter;citation_author=J. A. López;citation_title=The tree of life and a new classification of bony fishes.;citation_pages=1-45;citation_volume=5;citation_year=2013;citation_pmid=23653398;citation_doi=10.1371/currents.tol.53ba26640df0ccaee75bb165c8c26288" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=I. Braasch;citation_author=A. R. Gehrke;citation_author=J. J. Smith;citation_author=K. Kawasaki;citation_author=T. Manousaki;citation_title=The spotted gar genome illuminates vertebrate evolution and facilitates human-teleost comparisons.;citation_pages=427-437;citation_volume=48;citation_year=2016;citation_pmid=26950095;citation_doi=10.1038/ng.3526" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=K. M. Bradley;citation_author=J. P. Breyer;citation_author=D. B. Melville;citation_author=K. W. Broman;citation_author=E. W. Knapik;citation_title=An SNP-Based Linkage Map for Zebrafish Reveals Sex Determination Loci.;citation_pages=3-9;citation_volume=1;citation_year=2011;citation_doi=10.1534/g3.111.000190" />
<meta name="citation_reference" content="citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Biol.;citation_author=R. S. Brennan;citation_author=F. Galvez;citation_author=A. Whitehead;citation_title=Reciprocal osmotic challenges reveal mechanisms of divergence in phenotypic plasticity in the killifish Fundulus heteroclitus.;citation_volume=218;citation_year=2015;citation_pmid=25722001;citation_doi=10.1242/jeb.110445" />
<meta name="citation_reference" content="citation_journal_abbrev=Comp Biochem Physiol Part D Genomics Proteomics;citation_author=K. G. Burnett;citation_author=L. J. Bain;citation_author=W. S. Baldwin;citation_author=G. V. Callard;citation_author=S. Cohen;citation_title=Fundulus as the premier teleost model in environmental biology: opportunities for new insights using genomics.;citation_pages=257-286;citation_volume=2;citation_year=2007;citation_issue=4;citation_pmid=18071578;citation_doi=10.1016/j.cbd.2007.09.001" />
<meta name="citation_reference" content="citation_title=Bioinformatics;citation_author=G. G. Faust;citation_author=I. M. Hall;citation_title=SAMBLASTER: fast duplicate marking and structural variant read extraction.;citation_volume=30;citation_year=2014;citation_pmid=24812344;citation_doi=10.1093/bioinformatics/btu314" />
<meta name="citation_reference" content="citation_journal_title=J. Fish Biol.;citation_author=P. B. Fisher;citation_author=J. W. Rachlin;citation_title=Karyotypic analysis of 2 sympatric species of fish, Fundulus heteroclitus L. and F. majalis.;citation_pages=67-71;citation_volume=4;citation_year=1972;citation_doi=10.1111/j.1095-8649.1972.tb05653.x" />
<meta name="citation_reference" content="citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. A. Hackett;citation_author=L. B. Broadfoot;citation_title=Effects of genotyping errors, missing values and segregation distortion in molecular marker data on the construction of linkage maps.;citation_pages=33-38;citation_volume=90;citation_year=2003;citation_issue=1;citation_pmid=12522423;citation_doi=10.1038/sj.hdy.6800173" />
<meta name="citation_reference" content="citation_journal_title=Physiol. Genomics;citation_author=T. M. Healy;citation_author=W. E. Tymchuk;citation_author=E. J. Osborne;citation_author=P. M. Schulte;citation_title=Heat shock response of killifish (Fundulus heteroclitus): candidate gene and heterologous microarray approaches.;citation_volume=41;citation_year=2010;citation_pmid=20103695;citation_doi=10.1152/physiolgenomics.00209.2009" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=W. G. Hill;citation_author=A. Robertson;citation_title=The effect of linkage on limits to artificial selection.;citation_pages=269-294;citation_volume=8;citation_year=1966;citation_issue=3;citation_pmid=5980116;citation_doi=10.1017/S0016672300010156" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=A. S. Hinrichs;citation_title=The UCSC Genome Browser Database: update 2006;citation_pages=D590-D598;citation_volume=34;citation_year=2006;citation_issue=suppl_1;citation_pmid=16381938;citation_doi=10.1093/nar/gkj144" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=O. Jaillon;citation_author=J.-M. Aury;citation_author=F. Brunet;citation_author=J.-L. Petit;citation_author=N. Stange-Thomann;citation_title=Genome duplication in the teleost fish Tetraodon nigroviridis reveals the early vertebrate proto-karyotype.;citation_pages=946-957;citation_volume=431;citation_year=2004;citation_issue=7011;citation_pmid=15496914;citation_doi=10.1038/nature03025" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=W. Kai;citation_author=K. Kikuchi;citation_author=S. Tohari;citation_author=A. K. Chew;citation_author=A. Tay;citation_title=Integration of the genetic map and genome assembly of fugu facilitates insights into distinct features of genome evolution in teleosts and mammals.;citation_volume=3;citation_year=2011;citation_pmid=21551351;citation_doi=10.1093/gbe/evr041" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Kasahara;citation_author=K. Naruse;citation_author=S. Sasaki;citation_author=Y. Nakatani;citation_author=W. Qu;citation_title=The medaka draft genome and insights into vertebrate genome evolution.;citation_pages=714-719;citation_volume=447;citation_year=2007;citation_issue=7145;citation_pmid=17554307;citation_doi=10.1038/nature05846" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. Krzywinski;citation_author=J. Schein;citation_author=I. Birol;citation_author=J. Connors;citation_author=R. Gascoyne;citation_title=Circos: An information aesthetic for comparative genomics.;citation_volume=19;citation_year=2009;citation_pmid=19541911;citation_doi=10.1101/gr.092759.109" />
<meta name="citation_reference" content="citation_author=H. A. Lewin;citation_author=D. M. Larkin;citation_author=J. Pontius;citation_author=S. J. O’Brien;citation_year=2009;citation_pmid=19596977;citation_doi=10.1101/gr.094557.109" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=L. Li;citation_author=C. J. Stoeckert;citation_author=D. S. Roos;citation_title=OrthoMCL: Identification of Ortholog Groups for Eukaryotic Genomes;citation_pages=2178-2189;citation_volume=13;citation_year=2003;citation_issue=9;citation_pmid=12952885;citation_doi=10.1101/gr.1224503" />
<meta name="citation_reference" content="citation_journal_title=Biol. J. Linn. Soc. Lond.;citation_author=J. E. Mank;citation_author=D. E. L. Promislow;citation_author=J. C. Avise;citation_title=Evolution of alternative sex-determining mechanisms in teleost fishes.;citation_pages=83-93;citation_volume=87;citation_year=2006;citation_doi=10.1111/j.1095-8312.2006.00558.x" />
<meta name="citation_reference" content="citation_journal_title=Hereditas;citation_journal_abbrev=Hereditas;citation_author=G. R. A. Margarido;citation_author=A. P. Souza;citation_author=A. A. F. Garcia;citation_title=OneMap: software for genetic mapping in outcrossing species.;citation_pages=78-79;citation_volume=144;citation_year=2007;citation_issue=3;citation_pmid=17663699;citation_doi=10.1111/j.2007.0018-0661.02000.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. R. Miller;citation_author=J. P. Dunham;citation_author=A. Amores;citation_author=W. A. Cresko;citation_author=E. A. Johnson;citation_title=Rapid and cost-effective polymorphism identification and genotyping using restriction site associated DNA (RAD) markers.;citation_volume=17;citation_year=2007;citation_pmid=17189378;citation_doi=10.1101/gr.5681207" />
<meta name="citation_reference" content="citation_title=Chromosome rearrangement by ectopic recombination in Drosophila melanogaster: genome structure and evolution. Genetics;citation_pages=1085-1098;citation_volume=129;citation_year=1991" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=D. Nacci;citation_author=D. Proestou;citation_author=D. Champlin;citation_author=J. Martinson;citation_author=E. R. Waits;citation_title=Genetic basis for rapidly evolved tolerance in the wild: adaptation to toxic pollutants by an estuarine fish species.;citation_pages=5467-5482;citation_volume=25;citation_year=2016;citation_pmid=27662639;citation_doi=10.1111/mec.13848" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. Naruse;citation_author=S. Fukamachi;citation_author=H. Mitani;citation_author=M. Kondo;citation_author=T. Matsuoka;citation_title=A Detailed Linkage Map of Medaka, Oryzias latipes: Comparative Genomics and Genome Evolution;citation_pages=1773-1784;citation_volume=154;citation_year=2000;citation_issue=4;citation_pmid=10747068" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. F. Oleksiak;citation_author=S. I. Karchner;citation_author=M. J. Jenny;citation_author=D. G. Franks;citation_author=D. B. M. Welch;citation_title=Transcriptomic assessment of resistance to effects of an aryl hydrocarbon receptor (AHR) agonist in embryos of Atlantic killifish (Fundulus heteroclitus) from a marine Superfund site.;citation_pages=263-263;citation_volume=12;citation_year=2011;citation_pmid=21609454;citation_doi=10.1186/1471-2164-12-263" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. F. Oleksiak;citation_author=J. L. Roach;citation_author=D. L. Crawford;citation_title=Natural variation in cardiac metabolism and gene expression in Fundulus heteroclitus.;citation_pages=67-72;citation_volume=37;citation_year=2005;citation_issue=1;citation_pmid=15568023;citation_doi=10.1038/ng1483" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=N. M. Reid;citation_author=C. E. Jackson;citation_author=D. Gilbert;citation_author=P. Minx;citation_author=M. J. Montague;citation_title=The landscape of extreme genomic variation in the highly adaptable Atlantic killifish.;citation_pages=659-676;citation_volume=9;citation_year=2017;citation_doi=10.1093/gbe/evx023" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=N. M. Reid;citation_author=D. A. Proestou;citation_author=B. W. Clark;citation_author=W. C. Warren;citation_author=J. K. Colbourne;citation_title=The genomic landscape of rapid repeated evolutionary adaptation to toxic pollution in wild fish;citation_pages=1305-1308;citation_volume=354;citation_year=2016;citation_issue=6317;citation_pmid=27940876;citation_doi=10.1126/science.aah4993" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=M. Roesti;citation_author=D. Moser;citation_author=D. Berner;citation_title=Recombination in the threespine stickleback genome—patterns and consequences.;citation_pages=3014-3027;citation_volume=22;citation_year=2013;citation_pmid=23601112.;citation_doi=10.1111/mec.12322" />
<meta name="citation_reference" content="citation_journal_title=Rev. Fish Biol. Fish.;citation_author=G.-E. Sandra;citation_author=M.-M. Norma;citation_title=Sexual determination and differentiation in teleost fish.;citation_pages=101-121;citation_volume=20;citation_year=2010;citation_doi=10.1007/s11160-009-9123-4" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. A. Sawyer;citation_author=J. Parsch;citation_author=Z. Zhang;citation_author=D. L. Hartl;citation_title=Prevalence of positive selection among nearly neutral amino acid replacements in Drosophila.;citation_volume=104;citation_year=2007;citation_pmid=17409186;citation_doi=10.1073/pnas.0701572104" />
<meta name="citation_reference" content="citation_journal_title=Mechanisms of development;citation_journal_abbrev=Mech Dev;citation_author=M. Schartl;citation_title=A comparative view on sex determination in medaka.;citation_pages=639-645;citation_volume=121;citation_year=2004;citation_issue=7-8;citation_pmid=15210173;citation_doi=10.1016/j.mod.2004.03.001" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. Schwartz;citation_author=X. Li;citation_author=L. Hernandez;citation_author=S. Ramnarain;citation_author=E. Huff;citation_title=Ordered restriction maps of Saccharomyces cerevisiae chromosomes constructed by optical mapping;citation_pages=110-114;citation_volume=262;citation_year=1993;citation_issue=5130;citation_pmid=8211116;citation_doi=10.1126/science.8211116" />
<meta name="citation_reference" content="citation_title=Sex-specific recombination rates in zebrafish (Danio rerio). Evolution in Health and Disease;citation_pages=649-657;citation_volume=160;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=K. Song;citation_author=L. Li;citation_author=G. Zhang;citation_title=Coverage recommendation for genotyping analysis of highly heterologous species using next-generation sequencing technology.;citation_pages=35736;citation_volume=6;citation_year=2016;citation_doi=10.1038/srep35736" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=C. G. Sotero-Caio;citation_author=R. N. Platt;citation_author=A. Suh;citation_author=D. A. Ray;citation_title=Evolution and Diversity of Transposable Elements in Vertebrate Genomes.;citation_pages=161-177;citation_volume=9;citation_year=2017;citation_doi=10.1093/gbe/evw264" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=P. Stam;citation_title=Construction of Integrated Genetic-Linkage Maps by Means of a New Computer Package - Joinmap.;citation_pages=739-744;citation_volume=3;citation_year=1993;citation_doi=10.1111/j.1365-313X.1993.00739.x" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Zool.;citation_author=A. H. Sturtevant;citation_title=The linear arrangement of six sex linked factors in Drosophila, as shown by their mode of association.;citation_pages=43-59;citation_volume=14;citation_year=1913;citation_doi=10.1002/jez.1400140104" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=H. Tang;citation_author=X. Zhang;citation_author=C. Miao;citation_author=J. Zhang;citation_author=R. Ming;citation_title=ALLMAPS: Robust scaffold ordering based on multiple maps.;citation_pages=3;citation_volume=16;citation_year=2015;citation_pmid=25583564;citation_doi=10.1186/s13059-014-0573-1" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=T. J. Treangen;citation_author=S. L. Salzberg;citation_title=Repetitive DNA and next-generation sequencing: computational challenges and solutions.;citation_pages=36-46;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=22124482;citation_doi=10.1038/nrg3117" />
<meta name="citation_reference" content="citation_journal_title=Open Journal of Genetics Open Journal of Genetics;citation_author=E. R. Waits;citation_author=J. Martinson;citation_author=B. Rinner;citation_author=S. Morris;citation_author=D. Proestou;citation_title=Genetic Linkage Map and Comparative Genome Analysis for the Atlantic Killifish (Fundulus heteroclitus).;citation_pages=28-38;citation_volume=6;citation_year=2016;citation_doi=10.4236/ojgen.2016.61004" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. B. Walter;citation_author=J. D. Rains;citation_author=J. E. Russell;citation_author=T. M. Guerra;citation_author=C. Daniels;citation_title=A Microsatellite Genetic Linkage Map for Xiphophorus;citation_pages=363-372;citation_volume=168;citation_year=2004;citation_issue=1;citation_pmid=15454549;citation_doi=10.1534/genetics.103.019349" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Whitehead;citation_author=J. L. Roach;citation_author=S. Zhang;citation_author=F. Galvez;citation_title=Genomic mechanisms of evolved physiological plasticity in killifish distributed along an environmental salinity gradient.;citation_volume=108;citation_year=2011;citation_pmid=21444822;citation_doi=10.1073/pnas.1017542108" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=I. G. Woods;citation_author=P. D. Kelly;citation_author=F. Chu;citation_author=P. Ngo-Hazelett;citation_author=Y.-L. Yan;citation_title=A Comparative Map of the Zebrafish Genome;citation_pages=1903-1914;citation_volume=10;citation_year=2000;citation_issue=12;citation_pmid=11116086;citation_doi=10.1101/gr.10.12.1903" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=I. G. Woods;citation_author=C. Wilson;citation_author=B. Friedlander;citation_author=P. Chang;citation_author=D. K. Reyes;citation_title=The zebrafish gene map defines ancestral vertebrate chromosomes.;citation_volume=15;citation_year=2005;citation_pmid=16109975;citation_doi=10.1101/gr.4134305" />
<meta name="citation_reference" content="citation_journal_title=Simultaneous Maximum Likelihood Estimation of Linkage and Linkage Phases in Outcrossing Species.;citation_author=R. Wu;citation_author=C. Ma;citation_pages=349-363;citation_volume=363;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Xu;citation_title=Quantitative Trait Locus Mapping Can Benefit From Segregation Distortion.;citation_volume=180;citation_year=2008;citation_pmid=18957707;citation_doi=10.1534/genetics.108.090688" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Developing a High-Quality Linkage Map for the Atlantic Killifish Fundulus heteroclitus" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
<meta name="twitter:description" content="Killifish ( Fundulus heteroclitus ) are widely distributed among different aquatic environments where they demonstrate an impressive range of highly-plastic and locally adaptive phenotypes. High-throughput sequencing has begun to unravel the mechanisms and evolutionary history of these interesting features by establishing relationships in the genotype-phenotype map. However, some genotype-phenotype analyses require a higher order of contiguity than what initial scaffolded (fragmented genome assembly where contigs have been assemble into scaffolds) genome assemblies can provide. Here, we used 5,685 high-quality RAD-Seq markers from a single mapping family to order 84% of the scaffolded genome assembly to 24 chromosomes. This serves to: 1) expand the killifish genomic toolkit, 2) estimate genome-wide recombination rates, and 3) compare genome synteny to humans and other fishes. After initially building our map, we found that the selection of thresholds for sequence data filtration highly impacted scaffold placement in the map. We outline each step of the approach that dramatically improved our map to help guide others toward more effective linkage mapping for genome assembly. Our final map supports strong conservation of genomic synteny among closely related fish species and reveals previously described chromosomal rearrangements between more distantly related clades. However, we also commonly found minor scaffold misorientations in F. heteroclitus and in other assemblies, suggesting that further mapping (such as optical mapping) is necessary for finer scale resolution of genome structure. Lastly, we discuss the problems that would be expected from misoriented/unplaced scaffolds and stress the importance of a quality mapped genome as a key feature for further investigating population and comparative genomic questions with F. heteroclitus and other taxa." />
<meta name="og-title" property="og:title" content="Developing a High-Quality Linkage Map for the Atlantic Killifish Fundulus heteroclitus" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/9/2851" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Killifish ( Fundulus heteroclitus ) are widely distributed among different aquatic environments where they demonstrate an impressive range of highly-plastic and locally adaptive phenotypes. High-throughput sequencing has begun to unravel the mechanisms and evolutionary history of these interesting features by establishing relationships in the genotype-phenotype map. However, some genotype-phenotype analyses require a higher order of contiguity than what initial scaffolded (fragmented genome assembly where contigs have been assemble into scaffolds) genome assemblies can provide. Here, we used 5,685 high-quality RAD-Seq markers from a single mapping family to order 84% of the scaffolded genome assembly to 24 chromosomes. This serves to: 1) expand the killifish genomic toolkit, 2) estimate genome-wide recombination rates, and 3) compare genome synteny to humans and other fishes. After initially building our map, we found that the selection of thresholds for sequence data filtration highly impacted scaffold placement in the map. We outline each step of the approach that dramatically improved our map to help guide others toward more effective linkage mapping for genome assembly. Our final map supports strong conservation of genomic synteny among closely related fish species and reveals previously described chromosomal rearrangements between more distantly related clades. However, we also commonly found minor scaffold misorientations in F. heteroclitus and in other assemblies, suggesting that further mapping (such as optical mapping) is necessary for finer scale resolution of genome structure. Lastly, we discuss the problems that would be expected from misoriented/unplaced scaffolds and stress the importance of a quality mapped genome as a key feature for further investigating population and comparative genomic questions with F. heteroclitus and other taxa." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
    <title>Developing a High-Quality Linkage Map for the Atlantic Killifish Fundulus heteroclitus | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"VItIqLN3dnDEA7hO2exNo3rU0xZkqfj1P5F1BCcgNEI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494060","apath":"\/ggg\/9\/9\/2851.atom","pisa":"ggg;9\/9\/2851","ac":{"\/ggg\/9\/9\/2851.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/9\/2851.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/9\/2851"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/9\/2851"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/9\/2851":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00224vE99VbxTpDgb4Qv323I2EzZlBmc2amE7un+yDw75sVSqhtb6YYf9sJTaDyV\\\\\\\/v3pNRjQ+SiMzwQ81oFoQNBsuAWU3AdZBoh9bgjhkHmY\\\\\\\/cNxL+BpVml4RWwujrykQpmG77wdmA0LsBlURGU3AKevt1hFqC1VpLi9Ftt7OREZWPvKdszxI\\\\\\\/a+vQo+3c+NruaVXpjJ0ncQg22dX4w0HUpmORMCPvQePnj7j6XsLcMU2qVz9bA7A4XhGUXt29rRwT33p7Od7FATdL5X74AwUeDY+5mRG2WRsXvAduwPidqUnG8RJYWtH0RLiDWt7VHosbQ9VA3AqE1moNUJaMRNIYFF4+YsbyIHVez1eJV1RaE\\\\\\\/rmbfnzQ4M7QF\\\\\\\/nILvid5+IZSw+MhLo5lCq3ER1LIW0nh8sWaAwMBpahRqfmubE1dsDxfEPzPpQWG3qmKQHu+LnSPm1VzPPFLo\\\\\\\/K7oJri1OAVi102aPCkC10DrH5CGhWE9C5CF7pB4yOtP4G5JdPUoCMSIiiQAgyYLlKa6Lc90kSHxYFZQpeHIr7DOExZnVqEMLGPAb08AnZ\\\\\\\/4jQ0wYqktsYYomWVcSfslM6r8u0foWPRMkLeQ6jpigfLq5uYLZO8XxIYO5sxGQWWcdcmx6vTU5tSR6eMgXeNTr58RtTtx2qjgyM7iTKtSx4i31Rk+4zG1Km5bZmzd3HoIADyG2ccihOiqtTfmA35P97O5pcD11w\\\\\\\/FUk73ChvreyMbcuEDW2QXJBTDAnAXYbl52IM\\\\\\\/bPKilrhu3B9F+n9wd5to6rTq26ltJFDC9kGEJD5Wj1kbvL\\\\\\\/2tUaEXkmSchobtUF+dJvFt9IIM43f2lR3L7ihUl6RbYpOnITjx9nwWOsZzWSMghjs9z9fafLvwLbU7bts3irP8wu28abKAM1crducyWH+FVYoI4wr3WzBdQ28ibId8w84FLMUOR8nL7Nh1jwgzObGwayRCF2AhQ1gZMAWQmdUPfuIzqlFNrw\\\\\\\/J9cHVugJMJvc9LlLFDZLzuEK1uwcIRCZ1oAwZAIYTp35O11sAfoD7JUxNK\\\\\\\/z44iF7mI0agwu6mtPPciX\\\\\\\/0qTOzitAweNL92Zn6L3Zv4YhDn\\\\\\\/XLiwwNTK\\\\\\\/NoCYbaO8V5NwGD3p2A2KhVShvjP83wTQqWGoGuSHnas+OOG1\\\\\\\/cAuHCIX8gG+Xs9ZElPgy1o2UfUx8RbaaHdt3UP8uUHx2dXpEdaWqT4qwxHDlNUMSQhIFbyLmKGnPV31GG2E1WbHbQrdH7Btg1nlK+A3sx4TNDhXO97oBw+xSRKF9Z\\\u0022,\\\u0022iv\\\u0022:\\\u0022l+SEqJKw4FXpEcHIa4ZEzw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e26b39509ed664dfba4807199f0f1d8a811e282239f06043dd883247b2e420e2\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022KCMPFCjuLx0\\\\\\\/RDUhFrxD13tKAzII+aw1RbatgNd1W3vnY6xRzkQuLHOq7IAqeP3bQzql8oLfmEUmlaCv84hE5ERlcv0QgemPGmVSaBwKi0nGmzKad6U14c1DqOxI4GbQTKaamPz5vv8LP9wRtQkOXqyPKnDjsyV\\\\\\\/WKZ8vISxAJhqBSHbksgkq8Q9Aeg4x63dPIuIKYF4sCeOwu4LdhwSrkR\\\\\\\/QtezVByVTeoH5cooAFZmar6BbUM6nc\\\\\\\/ndBhmajtGyTQlYIkrAsO4+X+TSplivzGZ8qqf8YQdFhZIDwg\\\\\\\/f5fykNHkDiX47ekyg71tENHDQukVOsvZJr2aSOQDtMH5Mt93sHviWtr4j6t9RgVI9z2OzJ8sGuTEbde+8I8gF2Pt1rZCRimB7ZNJPvp0lEj1yd5MLYd6kfOD5YX+DvIWqfhI4neqmaxDEDlKrqeBMFTWKkhq6X0Ot9o8C5cpWlD1FS9ImQRTPN59BSSp4dvbhXqJkJEpvcigV+MwEbz6RA13AdFxJJOOC5p+h9VDgyff6B8CttqPW3\\\\\\\/BO7JfzQADUh4f50odjPLOT0s0si1LfvBWbMDaQJUpZz72ix9PI552n16Ym4UkBLg8DhYkBpQfrifmO+inP4YzJscP7t9T8HLCksVUQOnNDXAxz8GPS3LFniU+jQrsSlW28xSQm3NCfEOdHQ5i6iwA5mPFUdpZ3Fm2ZlRcly\\\\\\\/1dxdf3e+fWrlteC5b7YGEYOoC9BOBnQESGlgE\\\\\\\/dyeLJK1sejNajNyMASc5OtH5TGKFrJsqk4MRIeTxjt1z4vGhzt3x27kgpkG2fFP09m1rzg+UnVAxAtsfjftSb6x8MmzG+CW0504gaqjEtXlkbwaZYeRSo+l4x+pHGscwxiXf\\\\\\\/zWvvjOnZoOOJ7VmgYcU28VwL3XfEtfTmo\\\\\\\/F7BYyYnp2VpyZ1ozc+OTGRjDgW4Z5veih5syBBZ59pb71FUVBegyext54mtDrRWCuXJF9dmyEVlonXy8ey2rK59zPB5w71NetvjQpjFWISNmgvWx7TRcHJQWF8ZeU3cHJjQH312RFNvHlwgd5ewnbo8KthcXMxntVjRlpRMDrB3LGdw0A28VcF6dPXn8h9kUf\\\\\\\/cTahPbBSlnSfRydAIN6ip+ZO+pbxtmYNuGLTExI0E0KGZ9by+56esddQfHRqi5PZTMJFjvpn\\\\\\\/VRHzZukkZVGxEZzvXMQxD71hxUs6frUSBVIR7n8Ur9z9wHIeyahaCsObWKNKlV7Uz2kUVgH9J+kAJxxfpnaHWuNTfy0mU3tAOhVKGp53eHkS1xHhL+BAUqq2CLBKW1TjwB85zE4n9iBnPgsVUeh4ZxA+QqJ3nSrUZHkOTd5a9tZBodgKc38btTRzvjdWKcLarcXRO7x2FjhexDiday88GlxclzWqg5NsDmiiiuqgC7xKkLAy4FRuXGsRtoCHXI6AJ52DHFv3GgsnqxeU5FWf5LjjBnY8YlsXX7b9bbWA1VHi5TDzwA5VIDy5YwOA\\\\\\\/wmHLUgRkWHTemYMYMrUzu0EKotpOfB\\\\\\\/umZ9GKnHAWCxbSgnJDpKq8Fi8ePoMMockq1WxX5TIA76pYy0DZS0afcowst9ZrirkiYsxmxWDbBdN5EaQXHf2xjnV4sqZNWxgNymfQzWuz3uIypyMxb7yKcRMIRXp\\\\\\\/DR0BDme7HuGzRYOfQUqpS5hsNA2QlVUOvykvqCpHJ2sUwn132Zj2RsZWdwT7mIFk6JICnu6FbJUKe5F0YLaMUui\\\\\\\/TysVsaA9JrDy9jG0yG9dcAXJc+B3NI5gN70gJDOZxCmRsCizz+aAzMXmwcazbdkwsC343KYHo17+ybra+TbdMO\\\\\\\/DIuwLwfnvXC5sw+d3V87Bp1rI38nXuCa4F1sQgo9mA8HnUZyUFsFdMnyXxATc9V7PU048gl6iqbfhQVRD+5jELZscZVdTjpX+rYnZtiFbRpsIIo9eqD1QRqxhcDdEBgpbNDUzZU4oaPEX2kJaWK9qgZy+4UfQVz7WHRGI\\\\\\\/eGX7dzq8rhsiZUV100xZy8pf6tm1WFvnz6gekYpdBjS+8eT6XMlaMHkuj\\\\\\\/9LAQd9fsLcxhQrqHlIOHWH+Kw7XqQkCuRb61ggQOdSOmAiLnEJf+Uaej9dzicb4ZZs+OM89C8I5B3x6A\\\\\\\/5Y+ef6gxkwCJX0akG7iaeL+hl5jcCoXDE\\\\\\\/\\\\\\\/8A3w2FF1Tftm8UeTmjhWiC6w+K2aanAfSNe9Ftnk5FH1jAbpVe5E7J6bwnFeQRHAcDwuCKzXGz7Lo0rLTmJZ5VKtU8K6vXzxwVUtcqXcJEQ9kWTT70kiZY7c3k04JPpfMJtkcWWGQ5sSB1KKSbF9FqPj9ekUffnf+9mh\\\\\\\/AxtAgzV69CL7bycvvRrrZhW\\\\\\\/k63xywGrpxqUoVpP2Pq\\\\\\\/Y0YqfTLL2qKyl+Y8MB6Vw0XMlhdgc4oXZ1hATHKip9J3EqpVbGyWklFzrPJ1pQ7gX6Q3MpisiGagQgOMd9T26G2ISEV0oPsEgLlD+o5p+Rul5crCRIc\\\\\\\/zWvldViULY1Z0GcaaQpmOGHoXL+hpJLXOnjhs7ax4LI1o1DAZrY1ryRMmOYnzSlS3nIMkXWBHNMygNlyvYeKTabMn\\\\\\\/VtoVfIGVW1IdUjnKi0TVa+pA6+Dq0YsZucWPD8wNfByyYFr9ZDC0SH0Iq+s0YEy37krYQIz+2GudV2RyO7ImsGE2mHKjBRvqWVRThswfJvQ9wzZc9cfzBraXtSO62Pe9tCtP1DJRZ9YWAreTt7niAl8C0GV353YPS8hBxLt9pSNjUkdWDt56aqG2C\\\\\\\/u5+k9vLPReaEASrHdcD3Fj1hTvGrGjpkMLMFOyFopVCom6skjsMo5BSRl5a6FhhBDJ0dWtSgvUJf9yvK+CWc1KqS5aeLNKz1vVrxm8pEilm2M9NZmda4rdlRTrq2l5XA1wtWkKaGDJmMn038ejWAr\\\\\\\/V+UzB\\\\\\\/G7RXDrSgzQekXUEft1IQz2cX\\\\\\\/TQwM0vXyNdnCvWpCzM+k2SNVvI4r+9NPHzgnWzQPB5jw+ucEMLBXpi5hWRjzB648Znmd3cOPumWQqcc0NWN1MA9SxncO9ER\\\\\\\/eQNnSUaug1RimyJZKbtdfi++M8lRAUCOF5Fc67M54rTgLIU1Lu1LPyj+cY6jkzmeGk8VD8mB+euM7DIKWjOGoGZya16t9mA1CNFKDx2kljsAzBi53Z4xpbBi03RPR7Q3mllzmcAn80CwkbqSO7WMrp1\\\\\\\/lyKawsHgZt2fhMq0CC7gTDfvhN0zOh13D1n1jE0vD2Ng3swuOl1sEaMppNL2tpcRNTf8KnIs97J2MDQ5ZGyFLtv17fwZuloBc+Po3xrHpd38dDFnRuXC7Cx9EGmOU8bAIl7GBS5z6bw1yXjqyBKtdE+rWfVtfLtX1TlHbOhg79t\\\\\\\/dERI7\\\\\\\/Sb3jz86PQg\\\\\\\/TixjU7lJmVunMyUhYzn6hhKwnX0z4LF\\\\\\\/oh8iJUqC9zel1ChYUTFzv35oPRa9wDifSoAlLxHBNUExuA2+HtosrcuGo4pYRZhP4M8s+vXq47DkuDx4+9M2zosyQhn0g2Bw8K68WWWKEjHMgkk\\\\\\\/JypxiN6D3vnvvMwQSwsuVFkg\\\\\\\/mnWHyBG+lOm7f9W+v+ftt61gK6jyIaW\\\\\\\/YXtHV7eJvODPJbnKAVohXA7jzw7Sqg4US3fAsxD+CD7fS7gIFYHNz04cEI0ltZjWCusMLVh6hj2E3ULPmNdNAY22iYBfcfoN7ZB9yyerqYY4vmtnDG\\\\\\\/p19BXBRUo\\\\\\\/\\\\\\\/p8Ahfzl75CNuTLEWoKV21oBeyvTaFs3IdzGTiy\\\\\\\/qFgtpWC7djQxkR\\\\\\\/4wbunC4u8vrxvhWWxvv1\\\\\\\/OT87WjlkVYPWpdjIPFNXem2kqSSeUlnKdY2+n6vayPRM2\\\\\\\/P+35U8zigqlSlYYUEtkCqeYYdd\\\\\\\/Lm2Bj+HAiMvQLT2ifnoEqt6z4XfakXV4HcsgzHUdzIkqW6UM6Yvt\\\\\\\/xb+1o4fvFsyN4iiuVsmjG5fEhIZnxrpo+jcDBTyWMhjAxsnyLY5AmyVChz4HHgsm\\\\\\\/ST1YhckT1pqQqbxjFJbZ1CgR3+fFDM6reWdWBZW1ysYHmseR1qEf3kP2tywtl6o0Ikf+2gZtcHkaTaoDw7hret8ioln7X8ty515HD7\\\\\\\/pTB7k7Lrj1\\\\\\\/fo8gNw3ZX1sVl\\\\\\\/4ZJ4Gn9\\\\\\\/1JSrTntjVJCfzRLVRkjm4pYcHztdEvkbXFVLXrGs7t1JUhXzGaJikq6sz43vxbrAjPsznq\\\\\\\/v2in3x+y0qLa2kai21cFx1L8ngmlieq1KrMaOkwwOKXidqxJjKtiBH3md46UQJfHl\\\\\\\/KsAcOmfvRM1cqh3bJ7\\\\\\\/OpbTaBG2K3FPhHf4bAucz91HzDtoQ4vgMN2j8HjCvUqnJyobiwt\\\\\\\/9B5O68BwZCqaCgfRfRCSAh3I3hK8tr7BOA77H8mpw48j\\\\\\\/HHvEiv8jISpoQcdTN61jAcUDVyvOmwBRaOaCtEvpawkkpin8z1mGxpWP2oRcPEyIbsWUbqMq5P0oFA7yooRV+JgObfZ6jO5OGEV+Q3zpLhnkikf6WgUMknkEsi\\\\\\\/5CR+unAUow\\\\\\\/E3k\\\\\\\/f9hm67cRHZWsTTjZWhZRnL\\\\\\\/jHkNl7ibzHPPWHNNO9z2GIsr9rLF1RrEzi9rqG2He2WEWQ2sQJ0tFdTDuN5qXNlckhQlPJicflntEPirz8K1HLFluL5JshnJMQsUbCljlC9SYk58ti5qEU3oNkz9NS3vdTpUogcn2th1h90ImQDESY3o2GlIzzthr7tejiH8fij19J9yq9\\\\\\\/BaO1kPreXhuWZxTlBtO4i1mX75jAmiURPJmeji1htxIhFCOaf4gz5+U4CbOYDrLrQ805E01Cqnw9FQqqztZYIV98rriq6ioo5RoVVbbhkmxmQGnXiBpJun9fawvddriQ4qh8F2pw7htB5eiNTZlog6GOGcpm1Tds5ayeNOicH6SUiZdyd35hzir8J+gJC5R9hNbuUfKWpH4bClb8o7j\\\\\\\/NjhuWorS5xeYznJxteXaQr+a8zXCLNFYnerY6BdUuK9Sc\\\\\\\/xvMPbVFlBx3waGBM6yWhPva8XC2oyMZApBC7Af3mTYOBP1ShH65\\\\\\\/8usQRDDkV9qsLuZM1bLyBQDhyMSIi7eSqlRk3BH2RaN6yCn3MYG\\\\\\\/py3DzMUX44pBEyy\\\\\\\/EPVd1bF4LZbNg2+nUX1NuiECGHOFHlkb9GArup3erRm3WrRteQsZzqYQ0fGFjbHRx4QIzCmwq\\\\\\\/e+nXOTwBqYV+IPHcFa1tIPkSypde+pNBZQg+X3MqQnjDSteYhk0zeP4HK5QJl5\\\\\\\/\\\\\\\/compjI7eUbzVO59Q0A\\\\\\\/7yi5u7F8vKvLHAEp3\\\\\\\/K8FfpDN2tLfqpxKSoUGeiJsMs6GkScOizZMYYfZVgh6L8ZeUpHQXyqlygf4u0QpsDqCHBH2Tqt71bcW0v3YVoPv+ceS3I0HhIMQ7dCVRjDcnt2ZSFHRG5t0QPyqJmAdyL7ENxKF+VQEQNrMywaxoQv9Z7Lfx3KZss8IXBX4hQbx121ThEnlp3pLMEr4i6BnxRsejhGyppNm7Mr+32fsm7janRjl3jtR8perTEzb+pSNbgMFi8bumR5pmNnK0X7YlkcL+e5caG0ej2Sx3PMxuWzXqqYI8VIN0hUFTbDx564O0MaVG0LKHULYx8El60eOkQMGFPbS+B00T4emj\\\\\\\/FE2BToheOZFOGK7Ln0zmLg5pdpzngz\\\\\\\/5AtNGnchgAvtOp7qxV6heTE87+MYd3Tb2TGvEk90cKGnyrWhpn4jfEZu0AAU65b6URhpRk7kYX5MqZBgsKXztcfqj8NO2bBLQq7FbYLYgvtaLr+oY7J5lJCwmjJ554F7P\\\\\\\/BJhDBALS8\\\\\\\/tT8Xz+NgrEdwkn2nLOd2MVeGdXccRvauUpJcUSHN2Yn\\\\\\\/wF1C0ksOZszYBCOtODkdIlG0ZSd31WWeA0GzUOWJhnltKusIMNdVQfxlomeikZop3JRU+dQN8Ynr0rv6lP2a+1LfnDW1VlurBW7O3G7TqQR3\\\\\\\/QSlCH3enoP3DOhjId3wOn7y7sC4MOpWis1xhH06vSPlhVm1zR\\\\\\\/oY9wKigNrYovBOwLtaEuWIHS+vt00lnqhHekhxqjst0ZmBSgWPHanM67Q\\\\\\\/v2K6Q6GPG8M+pgQjI3Zvcx2ByBdar94bt220OFLM2Z2rfJ8KGQot+T0agZkrHKuhbREcVSHthE3Xhf4AAHAynD15ifPetCXUC0EEUZvzy8HyLXTqtLOeapiI6ZzrfddCvca3Fc79GGtS2itdhlygoryvzwODhEJIXB1H6i2HFwcaGZ6XRT7tNt2EI6MpsYev\\\\\\\/huPAH163NQDage8FDVYyRjhUJUD9V6G7gJ6HyWnM39UOmwu528HCt\\\\\\\/kKgOeeEHZAK8sXSfnSRh4puRRbsaIXrPMnQfqlI0ixzx62jHcgKKn+McChmVEwN0J9gWEsXCbyOI5o49hMdsUWpbiPwgr1PRIoriZSQhdN6xL+iqxQ62dsg2bM7j+uGT6NrEm9o0VVzMrUTm7CrF0W3HkUHVF\\\\\\\/yCO2X3i0IyB8X3sEk4vK6XDSEkqKz7\\\\\\\/\\\\\\\/kwtGfBYTESFj0BHRbzSI8jiGjhB\\\\\\\/KeUUKWPhEk1\\\\\\\/3iUyU8xhor16J\\\\\\\/83va19BdN18ecES6RSlOAaogLfW7F0XmHfl574WO3v9+CKsC86TD3Eeffj\\\\\\\/+yRTqA9bAJPSOhQpzW4vQZn3nFydF0PmWmbjCki5KwsxHgmwsWZx97M9dQOp6jk4bJJjIFL3l06q99AqGJObJZAvlzVYUKtuKaIWNh6Rq7XLCtMACMHzOSoZLm1itMuKIa3IdN0cp7i+Na2+BuAIq7m4ZcqhMMiWgiwdWElsHnY\\\\\\\/7DxE8EU3Wxw1D2UearCMpd4RiAkgosnRKfOpWpAgAwJps8pWpCPZNAZ4FlHt9aCgXxCzkLcwzXho2UZi4unmUQ2ndcW3k0dLX6RzxBAhmCrx+aaY4XS\\\\\\\/pzXAI8L0k2g+GCe9sNgp1ObTJGH6rJ2dJkh4XEopiznH0GMZqpdpqSO\\\\\\\/WoApkPP4Y7h5NPKk7fUR4do\\\\\\\/BUuxSl7HJPka9XDxHjRrbFTOaJsiV7dMff\\\\\\\/kO8BJxCE5uS5p76OM8RtuNrXiNQvbvh6WAGWsJfeoZWfNEwkbjOJ0dneoS42PQ0buSFH4jJfl2emdgX3wMsLiQ16qgxG6F14H7b7s5c67YrCbzercYwHmmLuqUGcdmMEkkgKvf3002Pc6BJlAyONsdOB7KIrigKD6p6EZ1mssU1qOZ2wZcGn7J49O13XlhZkLIW5V7pCW+lJ96pjMqrm0BmemFPOUj+L16KvA8OQB9dK8Oer1Pyiesc4p3hdK5T046WaphTD+kXLy9LnlfYJ78VQRPFh9CcHFI7SC0Jp5J06P26pRwNgeCLeqe5F0YJnGmPWTizoEw8x9JEmGKVIVQAp68TcZCDPswVebvc1e8499GRXTaNmb50n2yJTP41xYYuNIAqYmg2QkAZ0j\\\\\\\/VC8cSpMAm\\\\\\\/pULFv\\\\\\\/OhD1OubmiajK1n++Kmb4lLD38ZJ+uyHPsG+nWHA2w2uQVmWZsSI8U7TgHUodWI8N1NPLC2h56CJi0uFIlk6rHWjXASbochRet23+4cmEH7NQaBCjBYBSkh88RkOLXtYJI+dNiU1txP1gS9NhPdg7ALAKTHATLXn0YvdQU0qjh7IITCWTNNQPJ1+2dJqWK3xi71h8gVkVS5voTeqz1xGlM0cl7I3fpjFCupiwePAg5UaGwLNRVmicufIBN6\\\\\\\/ZEND2FICfR5O8vQsBfP6nAYCCv9g3fJFh6fNtg9iV9634kt0Jr0pM6NYzLuCV1DN8TnZThaxEu4v1SF66y7NtULiuiJIGH5NZokCTWfxqXpZ2smmnvY317EHzyUOxHqHc69SWR2OTUM\\\\\\\/JCKjs7fuino4PIiwyC33Ef87X7fQwfOPnb+KWLqxNO+lDCtCMAYaHfxfnxpsif+uHenMiHpQBmL0tGWkfSGb21MR4\\\\\\\/odE8n2qGgzhk0vEkzwClPkYbf955s5ud7G36hJGOck\\\\\\\/aDHg4e5Yk0UjWWkyGAoDM5572Aog4V\\\\\\\/s5IwFKHiMAZJxRSnYxH+OWJMlNhMfL2tuUwHgdCnTikdW5Z7eW1vsJcnxvc3J+GMIfHVj6UM4Jwasct8Y2ygJWf\\\\\\\/hvUgoASdUPCYyVFVH4Sa9EfNeY9O0Bc5bTG6z\\\\\\\/iwVCoj3iADGp07txPtBaZGmCscPj+A4H5uBHe\\\\\\\/rKaT4D3Ih+5GClC\\\\\\\/M\\\\\\\/xjlk1UV0KMNibxdt4ww8xL5U+yrpjFhiHUf09nyDUghJ06w\\\\\\\/DHi9iwlTydD5ncYFV5ZYcbyySAdD0o2wI\\\\\\\/cwUpYTW5248O7ipiXwJPLcbjvYVFq8bArPvHmB8D9fsZhxez6hT3m\\\\\\\/IDZE6rvi93yxiBvU7WkT220m5QALLCXbOnavBI0+XvN+PRG42Q3j\\\\\\\/\\\\\\\/DoesrvXRV8Vwqwb7LIR2tzhBPmleiLvfOOGPTzfx8V4FOKgPEWIFJcjcFmjVzgunUvQ+I5aSsvDZhlgMPh762PW6r3aBT4xIPfKLYzq2mccoxjy8OSY3NRp8bzE\\\\\\\/xzWE7z0OitoyevpMrtew8dZnTbRltcQKvgjJHUXjDUDuzPSCe5KzrG7Q0wT3lkHSzlfpJkzDdl6hOi2nmOxrpuUAp+zKkgREuaR9NYGgcQhQ\\\\\\\/F5LtfJi1LZuCxHijx2eXKSx9Y3Iy6ZE2fCI4hWfy2xuEMTTnbe8hrOGT74j7VUmwekQ3Dp6ti2QIKAS03\\\\\\\/OZECJWJ8yYQpvG0+wNtP7kOn4lz6zkm2qrZXmgHLLMP3fz\\\\\\\/UCFXasZWk+lHnljabXY6\\\\\\\/7s5nntcjeS2PQ3YXd\\\\\\\/cEeC8hdxTa27qAzyDsLmFWfEj2TEQZzlTGwzrwaeV7jtn4Gs6oLWA091GQGoesp5WcgKuvCK37oFMWDHYtwmoku82ssotFKUDmN8I\\\\\\\/C5n3U9NNBX\\\\\\\/dNN7YEBQZqNa93iaxOrCKqXZHKE3R+D+HG1FrAOs4VCqwzVV1DODWdvL6IslVDwnyWGfZ8SJBj80VkY4a1WyOQJKI\\\\\\\/pH+7Uv2w20XjILrd2PGiQOSKTBarDZNA5OENilZD5W\\\\\\\/QJ9YWeguS0ADPjEzYIBQ\\\\\\\/TvvyqumwPTfwIE5dBsFx4I7CmQ2pHUL5lWVkYV+25\\\\\\\/gW6oTuTAMECEeGUM\\\\\\\/pxPrQYjUfhVP9FvnHRH\\\\\\\/By5CzQkOJJyVD0KjCNguM716gNKZF5FHwdX0RECxOo8T+nyEE0FlH6nhyagMlIqnZR1q17DfZsfpvlNrJZBGvyyLPS7SAi3S9AMLb80YePhQJ\\\\\\\/qiM05vLWKxI4g1sptJHrjC1XWUALfmFSKL7OhJBk0uz10as2pr2\\\\\\\/MR7SMnDH66pmrkDcmomgJPRZSGhe3r1iraKBc9q\\\\\\\/JKTNyJORYAZ7EBaVueR1T7vBvKk1o0fsuCqHvp0mPbAvS6SwrfK+8tAzvcZeSwASNE2SS+TLqw8bXZxpz8I\\\\\\\/6oh9KjC4nu+tncDJG01+io36mx2RzSjkVr78hns0Z8gC8adohp5YGSgiOpcIvGJjwYD7kgBDlE92bz2zDxZF8ekOrzBYtMDYNGIUbG4tvnWMEbihx1aD\\\\\\\/f1UyQGr5QCOtkd0SmAlfW7dcnvFu6Hd\\\\\\\/zSwABD9zMM9jLx2lLDFi\\\\\\\/430WErk1YsRWd1yz0yxRYzwACqsqrUWyv5uIMFaU6Yaraabrnv1tHIudOi51OSA+Um3CD0k9FcNa8YNCcHj8SGbo74MaIHEBd0n8o+Qd9qjupG\\\\\\\/DwD5Y6zHl7HnLZJzNF9oXJwFHyogUZ\\\\\\\/K9cP+Yi4VfBKssbCJaQJ1rJMdJGK6qe8pGsRWAgVEMC3Neew7vmyEoL3dP4p8nrS8oIedLVZPWjvegm\\\\\\\/vrssUFPMs8EEr2uKMarLHopX1kp5KWeQ7\\\\\\\/qZQFCiyxGZh9RHzZXnFRsHlPDEKYDolgbt1mBGxkIqQS3PEsmEGGWca4d7odgE73urrlHdDGKgkBUJKhHyy+4TECdI5p5wa+mq2JOoFflVw2k+f7bhdoqrFT6\\\\\\\/zSZAyOeATN9wWUmAFBb4xbCSCt550obIf1YVbDGtUBpGyHWqKZlyEkP2QRUBEmBLj5SiC5QiRAJni61IR\\\\\\\/A4BF5cPjyP4LjctoTP3di2hK0emevrOahbUaSXMmkG8Fcig8wcTvXbwu5zutyNHy3ZpRQ7UlMRXXgEtt7targ9pEpZjfaLGRJb+5GyZMqzhmxqOGo9\\\\\\\/06GmQ8JyGcC99Sg5XopZ3zU5KX0dhb3OAIrfMffWOrooju3hoiVVedHGf8lcnSz5yInRcLJ0neQyp5caZ7kvnruxZQw6yatcbfOyhz+OY4IRGHvzFPB+W4C0zvXBSMF0HDohDK3aOWFzspeKip2Taczih7N9bNHDLj\\\\\\\/29PbljTa8PvL2STXb3D+OrT0T0jT1nKdsDreucfOPZBr+66T+7yrWGTREZPr9TfQ0LsCBeQsTeEPtKGYg28wF+gR36SKEkkoBBAapfYQN\\\\\\\/d8uGfsqrHXc33PQOMmUGA3cXZ6LKNwg+B1kZrXc1hA5SqugfDA6\\\\\\\/qLXkqj84CudH8f\\\\\\\/qvIxO72kbssJk3ObeiOseDS9pbDOCsGhrJX6p3cPpzsaPCgWjMAlBBqvmY4yI9VCK1f4nyxW6cbHsuKMunpPp1Aa71KXlFBXFUT1qw4nkSm5csYzzIZ9ahYFzCtm4OkPiStw6hbs6DkLKjl7YehUNdVpeYs7MjSstqJtgMXHUPYkWNp\\\\\\\/1rsnDPGsQYGGUqrBtstSixOzF5RkXAcBMtwujWR+idaRdz26W2HuhdQ68AJj97u+i\\\\\\\/s2GcpkdI7b6WhN0mPhnMz1p5Fzx9Kj5Qbn\\\\\\\/SUbxPxUapAbvImfa1RvI7wfDTsRwvjydaeS3Y8bENmUo74yoRDaPY0qsxajs0chdiBT6D2D+dpkgkMt1xvwLUMLtoLSB4CYYAQHtn\\\\\\\/TrFO6J7A0+p84mqmcXiiFPCfvDY1kBcBRwktb2tQZsheiMejE22FlTmUkViXdiGwezOHg8MaiGleTL\\\\\\\/5iXetaIWyTzIImVU6W7AB9SlJ\\\\\\\/8UMFMoFZkuwCXOVU27lx8JjbNdI2VrFqYfnVUhFiVm3KhmrCy4\\\\\\\/q3PYmxEIJN0QYmnsPPZ6GR8dcnimeOjsFuLEjlyL0T8kE2j\\\\\\\/elUYC4RWBFJQyq7YE7CPJMLdl3YHkCTiz34s8NQ+hjeJxc9DROJeTTFrFTLtoELldyPCwDeJ1\\\\\\\/C2cUM8GRzSa+vJVJ4UtHcs7Ue85nDl\\\\\\\/irFotX\\\\\\\/EeLeyOHkPx2lQL2dcyussknTCfthkOHmzo0CsHEipuF2\\\\\\\/9IdZ+cPdor2LvPP7O46KJ5Tbzw\\\\\\\/MbjSag5jzERH0aqlnADSLzSfsBydw\\\\\\\/t+qYdWfXbeo66wthZR50mBdc\\\\\\\/2t9QbhsxI7XABZiddGSlzNJf8bC5+UjAFvFSSP+N6Li5CwaYtvREkh6+mIrTVy6Hq3zRpVHxU9voidjj6J0ap0rxHdbD0\\\\\\\/jOw1fo8sWFnDC1wyYnkkxvCnShPQHd8La34hcXRed1hJdWgo1qp+Owb+l5El80ckxkW6YkG5+6xzWEhoYqbwtJ5J5JbNZxMFTmUeRjbwE8jq3LRGi04s6yzFUAD4LJtD5\\\\\\\/xYqPtVzeoayVeLZUZiOlqexXT0nhpMkxC09VjaNRC564SAU8P8pVIZYKYHHoGlo8\\\\\\\/d0X78tTvl5A9nV74Ih+nXxMQvkzmc6XhXrM62Q7YvKGkPipcmbguOghuydsUQ49j4MU+ZNbJ+VSxLSnY24seKBayEX3PVF4RMsUxaxYolJ1I6fn0rybmN+mg5FYSZJU\\\\\\\/8mT2E7sn4IttpnhH5l8aT9yCrwGbzEClIzy9GU+oDcjFHHxSbtvvyMjSXLnFljPD0dXFNLB2FKGtIPiH6LW1Ks7go3gGYu0iaoVOb2vXcp\\\\\\\/wkRkO6Sqlk5hzk0VeVoaVkPowF7rFVeW06xWuogaup5A8F48at9N\\\\\\\/UZA0yEuwJVdFbCPXEkwLLxZL52gqUfZsEVYWuS58grAMfIv0tTmyKvyKJr+VWUrqRQr9ebvPTlN8zfGPFWZt+VZ2Hp6d5ijdrMXicOJr2QOL5ljiqcaY5ij0kgc\\\\\\\/hO7pP5aFLV3UPjhGhiWi1+3U0UT3p9MVHCwP1KNRyvJBg5dWVeuhfVEssPERUv6HBbyOdb3rBMOzf+Dsi\\\\\\\/svGGbywbjwxoldJEsO1RCPhIBX7yHXxftPWsmyN5VvSYe3R0ZpdIX3ylZLYcyPCJ+zw90PYTbOn14r6sWHre+djUUfBN0oNoiVbUKbzdscOomJ92pjxgIiM0nMGYs9brwctmEeYnHBB092s819wgisMKCTEDHtwtplRYzevJwr248OM01Gcaf6PRuURxX2IydSuB9ICWBJohshL7lUQYQg3QCdAHDXdD2xKtxWr9SjX7gJl82bnfKQLkPYQLWjtTZfcIpp1Kft+CriN2HCgl0eGo\\\\\\\/+Xi9Srv\\\\\\\/AOxIUcIr0oRtqvtPgXfsIHBz48AQoeQ4kIDzHzus8BXCB0LSZC0JEQ3pHHNumOaCmBSyX7rhOcTabFbP\\\\\\\/XAEZ35bGD4SKXOMfh6ZVOb8NP9h+whWfBYjrG83QQ5De7x8f5Od+LeVH63sdBSjjSccwHE090FdutsZzVk4HEWNTONM+Q7XGmo\\\\\\\/EN0R+06K68HF0\\\\\\\/Ghs5UetDgUZBN1YXdUUBBdXju53+guPl8hccucvm6U+3y\\\\\\\/ior7ViuajiqoCvDr1ZOIl3G5u3rFmj7CTC2XgIS0SAPv+ygOKCuvw2vHY8PhvfN1lWlBAUtUMCrdmZqsWfTAC6j4ZQ1JuZnrmslz\\\\\\\/4+ZC6MsHwhMY\\\\\\\/ILI1\\\\\\\/6XXC9DRnNBqh8yEnEibDm5aDn5sfx84zoG6Dtq2wDEw9M1sGOIJyAfpYVJ9QUHrlGv1lWG8TmDIj4AGqJTBJjOif1NRUCrqis12YK2rnkesOOIqjBwbI7hipp4rxSKdIvgjnktWVUjTvPSqTIv6er+E\\\\\\\/09xK5Pjay8Xk8t\\\\\\\/oWDjkYZMomnBEvZaCz4avlnUPS6OLtHNuotl6FA64gcmqMwTGw4UQ39unL\\\\\\\/Ci6MYRdCfwIkbtF4hEn67lnbMPNKFmvfzKtnbHRXcnPI3oFTEbjlC4KlG\\\\\\\/1xDFI6Kbwx0TQES6ad+QuGs0qItdXuJgRPgL3o0lk5bPwzCOS8Tpc2qES09wwmcczR5y1ZoHWVxcCUSD0CezL7s4iu6e+NjRDfU2KBxPqv2m\\\\\\\/9JtD\\\\\\\/gdZfQhRwAkEHizgY9RYcguu77J2slmsCHG2FqgaAsVxivaOHdDvaHzQq+AtJ9vuxiw\\\\\\\/BQ0k9jGgILTEJ8MnN9zjgvOu2iDo4ba1ohdTEgAz8DKV1pKvh4rGNDmkTWhm8h5PzrMklltiDWiY6sgof1Uv7mmnn6+4HHBPpEiNjE5PgCjXnClVJOXd1C5zl4IuQyObiv9ppmbdOLysb8spPgjo3wqjDft+LT4csLH9DXgy2CF1MRqLzBX\\\\\\\/Fjc4f48T5\\\\\\\/0sjfu5XF3t0erjHyPQa4i4OTFEAlEltKsnXePhrSvGOv8EFTgKPLHJDRqydEyo4p8yx9glUxqe1TnNr7wxM5QmoVkUqx\\\\\\\/L1qgQ8eSL2nvekqsHnXJMKo3IaQqoqDxGUYHkXiS\\\\\\\/25D+O9F0qj9j91ZRU3HY5l+1ZwA293det4AWyn2BRwybXuBYx7cnVSmioFXhbhrgIs0RPo\\\\\\\/oRMAQPO\\\\\\\/avLo04r8HDDC9GWKZC3lQQoghxRz3k52g5KdaJR\\\\\\\/QGn5W3tJmzF2wrWpsucCKIZGbMllc3ZUwUzmmr6mXUGi6+Bj2jOPxW0xCM0xkKZ5ocPqrnFsCVYkFO57OAubBeyldISqvHfaNtMw5LVD8J94GEx+0QiXE8uo05wbJ7u8uA2pwBvbYTchA+cpe7Q0GnLLoNb+5R28n4a\\\\\\\/LRqxkE+D0aQw\\\\\\\/fmrEB39xrW0ei5De6Fmz869ePmjlcf0\\\\\\\/afPuHzHgWgDryqe21V0sKlUnEn+1O3XUndGYoT4ZUuKCgigveoGNH\\\\\\\/m1t4oN67dDB+lscOpQ0vr8bn2O48wR4CNT1wtR\\\\\\\/xkj1zX0Mh5x0aEcTmYJ28WZ4MEBmbTu3GSxnqIjNHYSDK2USlYjv+eskyKCpIqurE4w6tADZi9KVCJ1j9Vhfg9DLqPvfpOZxRgm5Adkt1CifE\\\\\\\/lkA5W1fgBZgwQwuY1lJd+S82sKauT+djnGRGI4g5B4IzqDwxMma9QCfNBTs76\\\\\\\/CvAXofXlry8TvnLjSFxGN20d+Gg7Pyt1vgGRsAgcsf5gHyD5oMrCwxHBJffIRt8iixHatBhDwND9BH5SDX7SbxG3Qb3PYuvBZ\\\\\\\/ZYryJAK+4cI4TxuGlQVWc3ieVFLMArwTeRtoGY3wzSncTzSP8zNhWoaRgItq3hhN+3aYmRRvKv6EEvw7BXdtach7SkgvPeGx8RPp8fi4Zr7p9xb4hoPswwrD3UGr4vAtFYNCTmO+iOAhTaBOHZnEMP4E0OjKM+tJn2sKD\\\\\\\/+1CAM8wXUUBj+CbBAW0cLZ4PS3BjbM945aFW7mmdpzt8NfIMTww2VGzobZP\\\\\\\/Qgpud2W4sZ4LqbVQqDfGujFdOjfDr+HGMKOFlgp5cTMWjoy2hI3QTPsODBN8r6gyENIZleKA01Bl4bHri8hXGM6ka+vCWJvm71o\\\\\\\/k8Mcm44eF4ncDeZcky7djDsZN73WfBZvhudyLgaRRvUTRXyOjA\\\\\\\/ik73gTz7MjZcisn+E15EZewPoOs9FA5yHbXfESwJHS\\\\\\\/u91ZM8VY23GK8c\\\\\\\/WEQJOaHRmjUjEw+WSMGzdOz6TYAEg5E7bJBsYwQxZnVffOz80Z4UnPdA5MiUSISjkv2D1oLVmXxzpiHXikzHc85U9mWykb+SaYEnNxArY0+1DAHM44mGjLXLO3KHwUPGWQ44IOYXMQw0KwnVBBVw6EoJOV1Y\\\\\\\/e89n93HFat112FqJxjhx\\\\\\\/0jIhLL1O1Kg0jAHxGPy+QuYYTbEpBUgBKq5I0JzRE1Numc2mED\\\\\\\/KRr0WiDUdMHLY+ODUXswOE+zLSjpajSTHXj6enRj0JD\\\\\\\/up9FFUz740DVDsBINfayPzMG30gJrybGp\\\\\\\/Sz\\\\\\\/zYPBeU3lOuwVfKUDCQU3rpjIzBP5d399iKpzXIXnpMUDkrWao4BK\\\\\\\/mPmIbWZVqjK0onwSpRCq7IHssC7ulYQ8Jg1oGctx6KnWVd\\\\\\\/imOJxSh21GMHv2yEHVaEM1YUj6NVWnLwQ9gl+WZPB9GsNxKcsckuZUA5OlKxoXtW9q2cA+9YlMFYZxvXHk3Heddoc+UOg\\\\\\\/7WTWQouAYfjk4NU1n7spIC\\\\\\\/GuEJm\\\\\\\/DVM9grKN3Rqy33F8WSYhkzchVHWnQlUIttFS1rH8HJ3\\\\\\\/r1pvYIOKNF1eSwIdepV1l0qFNq8vu5LNajsSqHxsF7+M23ctun7qWqLCW9c9wIQ1XguY5LcL\\\\\\\/F2Mt8I0Yr37OamdeH3C4o0f58oxraE9v\\\\\\\/hT6TNgOF\\\\\\\/XD3cNgT0hxroXYULKJ\\\\\\\/k0CYYALaPa5UV8RtkpDrRC9Nm7D+hITe8uKdbLWXs\\\\\\\/OpgO3Wvtcwe1dxuksUeQDLO\\\\\\\/r+wx73aW9ngoaYlvubrE4QvSMd\\\\\\\/VYqas12j1HBVOrhVKDs8oACBDvNrspW2krbxeSsaQbjjfh2DgLg3jp2+0zzRIZ+gHAPZZAQVMgRWv+TO42xLwK+Y4N98HclZfbBP8ptBGejtq7QkVT98iYUjjCwz82fS\\\\\\\/twFq+Wzu\\\\\\\/hVbZmFmBdEQcNgQI7fM\\\\\\\/IjXrtQHjUxE2Zyb93lvV06Z9CirZl5E1x3HY2oj27R6drr0oPat3Cq3cNcj0oyObhbNuXP13dmCwETc\\\\\\\/LMHnA+zjf1iRFLqWgoaFdLXFVTm07wzvz6rKXRJrlkucf1uVW0U5NVpVVjsrLgtz+so1dJuVtLM4idPeEgu+xcP2QA9MNMFnpRcslNCCJOjGNTKhIBJUFp8FGAhwukaqh0Deb+Blc7YpY7QYdJYFyQq28uVEOYtyEhgbnQswVLS8vVnDgwFrJdMgm3It14ho6D8AhACBvWzUZRVUq6ElSvcuBWMWyBZJnVneMF3qy9YwAYDfX20leQx7+59YlESVCmrRtAJWs6sN2G6LG3sbZPWKLavSbilO4cQ1KEycvR3eguyt4qryRxjjrp3CuQr94P9LRme9ObQ3oRlqMjiG9I0+iADDT1Bj6ogaDdInGgSZ1DmlFN7r4VjcyaYYN247RBiyyP\\\\\\\/rgjEMjT1JL5Zeav\\\\\\\/3rezmAj88YAVTultr86G+Pr+iot+nlS3Hq9fAu69ievV4Lmq8XTlSJH\\\\\\\/JVWRfEXBrWYnnX3th20U1yxJJcHfgxQONDYMoLBEMmmOXr90aR3dMtFuxDOW94d60Nhy7pjLqqjv6t\\\\\\\/cLJXBUtmjdqaHAKa2VEMRV1h0eVR2EpA542b262rsXxJRtek7lSvcf1WtEqZy+Suol+cMXFOO8iHXrH7EXZMIAdbdaWqE1NFb2TSf18V6ZoWPcsBjf2x7HRYhqlp0PRVsacfrIcY2oHQCwR3F7dVgzsMFdCIVl38IqRRdZ74uM6zjUnAkGkFE+k70KBIKVXOVz5MdVnXzTex4jhYmGSc3iFXNYJBT8HQKwL7wiL+OlWKgU8ItCofGvWnUPOw3HNG2hsJnF880Lo5O4H3\\\\\\\/cKpksU4d6xrSidn4ZFQj6nsPhrDIdD4aAiu10BFJ0jfvjbZSQCinIVzorUBeteHw+2aLdPtuGQgHM8n+BjUlnDB4Xa9e2QJ1jsDXdU0SNMy1njsdq+kwZnmyXEzTOwbRMNCGg\\\\\\\/a03FCj6Ht0ssLzDM3tGUsi7QIh1xDhFFTOaGW\\\\\\\/FXAzqipmYTdtedxay787mpFeAmDzcNeqcHxJRqbXrlDAE3Q1J+sF7pEK8xbo3\\\\\\\/kniFvFSuFMfDYZbh1W3PNZ+CFrzNTMembpIdfu05f3d2NwYICBEKPvuMkWADB2uQoN+3Ajh6LEUmQ6D6BUJ7P7\\\\\\\/JTmevysyRxwUKi0\\\\\\\/V+mFYnuQmd654s7SVkV70PbpQG\\\\\\\/Z8rx6TWjJtJvSMvltfJIapdyfGniVWTInZ5uP8gCzmjOlcw1Fo8j7JoKrKQfU9nU+PymHvJ2ccWvE88OuykZXIdztGV3iIkrQvqih2IRIGDpCZFiRgfvgRrxzD9sgFYw7gPHz+MQ6aOvAVWYM0v6wWzCqpbCRC6p07vPuVBvn2\\\\\\\/J0z55Qr8oBYO6FbaeCtncanJOcFAFY0Mz276BXavpgIgjSlaiGJ83VEN9NqDNF2vEwlPu5mrQeHcvIloTNIxV0vuc+SRwHKKBx1c1jCRMmDPuj+gWfbhBM8T1AHIx+qO66zHyYzdVWM03FeqpYzBtZ0quOWYZPXda\\\\\\\/MTHvBf4MfBzT9dLZqgWVtiuXf3op4gj6YD2\\\\\\\/Z9Klw\\\\\\\/OGLl9GV8MooHMGsKjRfFjq2Q7Nla0jDJCC5AnUKHSlIjxwU0ZBoy557yrB\\\\\\\/akFHe5Ym1richovjxHWsqWsZCh2AanUHkMA4dPFi2NSOxHsk3yEYvlpVUsiqSbI8npBddBFM8KAleHFBwNlr1TMwynzMhAhl4vYUgrvhdkQLOidx6fmfS6CHZ0VIIKrFpYvEr79s6RWe8UrV1Hyl0+5wsL7AFUkhqmEHrm5D4pBbwWRb3889iqSADSbUgaDQHt0DNq3vk68cXOEcs\\\\\\\/g6Zy\\\\\\\/XyrY1ja3j9lBTGwWFFTQOSDUTo0ZPYy3lOzJEL3GaFGm\\\\\\\/c8jhuuvzQ\\\\\\\/iYtlyADahfhzjxtDPmC18V6LcAZG1EiXOhYlVgMgmaduh\\\\\\\/lTCeYWMaUJ5FTpn8tImzhe5cdrjc0Swu\\\\\\\/3G2e8hMht534jdIBZ+EUg2Ezjr4vYcYp6cxHqO\\\\\\\/0dQMi3StqS47QCKgVr5EBjzYfORjOW7NwlRdqneNrAa0ZxmIFRrIm4e24qIoGBPfxX8bUJnaNa7SMzyEvqCIMMOJN0gtFsxlPUyo4DbnVY9aBalWxrfrlr+aIjz0RTHBJO26TTqCU9ilLLpyKkU6oSz1Avig+ErPzuEfvX0rrYUOObWGiOau9W7mTX5pKSzvNOzgv7ZhqnKeZPNSq+MEQQvJGjFMjpTdSPNw\\\\\\\/vKTaeXIFkhERX5DoOAiqCh8AC2xDDe\\\\\\\/A7kC7ZshBF0BTbshz4WINZjgRKUmyj2VrKpLJb6IJ78OXF5PK9Q5dBnPsJoYmoqYzYa1kJHJ1vUIcZTFOx+NDeA8d8yzyKpWQFAXFaOYhWwTXEjj5Ok2Jg7Rcmp2KGe997Z6nUkBCQcauqau8v8hNeVZ5UZ6N9DxBkBXPbPTD8k6vqjFyzld8PW4PpUphei\\\\\\\/VUdoSb+ZC\\\\\\\/dsDdufE0EUUTsP0+PDFvYrg4P9sHjyPzCs5ZJwUFB8iDg8JlQs6ARr6zR87uzxB8coIbt+XziforS+ybmQ0pb+Kra\\\\\\\/dftsUAmtFnnobUzqU6orbONZAU4lJJpY\\\\\\\/wQcwgP2CnXK0+wk3pWVJ1UzonAjn7xhMRzJu+heOopGEQnC\\\\\\\/SV3PRBelZDT\\\\\\\/R9cWLEepElYo02nne9qxLLxb2ZN8Ld6kDPBF0QrTHD69h8mcje9qLReyuT+CNp1VF1lmH8i1Q76weqQHh5ciqPY51yj2r5dnS0kmE3lyEh49msfYDsZtz4tOiAPgDWsWr6Z9jekyspLpmY0RuPBxBpswrzlypOlZv8KTLpHmmcmWhk4XWVgdUSTmtQgJwoTynVRTPf1+ETAjuIctp0GnZcpONtZ3V72d80wEqaLgQ0v6y1+rusQ4v29yPGo2ZB1GjNorjuuC3hrd8qgnmWa0jLvERzZRVF3+zoOUi68KBE0QFw7TgqzmT9wlTylad6i6UWWF8jOvRa01NPEJ5NNnBDBPZCevntUhlboCmEBbxfwA04Ksr0drS5tH1dBFH3kNQskHBZcdEzXcAd90H1iOHoX\\\\\\\/g0aAH+uF6vtqEVk4mytWVOex2v7IkB6UViOMibzvP2Wyhgn\\\\\\\/0g+rurYkMyJJ2BT2WBrxDvpJ1\\\\\\\/J80ZOKXATcK7ZAsOBX8r9MMPZZ2Na9APsGlf8J\\\\\\\/5f9mAM\\\\\\\/g6TF3TTaWpggVcQm\\\\\\\/epWWC+EexRrYjpvejP8EbYpZlvWxN1nqd2Xv\\\\\\\/ho0\\\\\\\/sJVrTYG+0znYmQaWLLUwpDvkGZvI8RjNUuSJKnfZtiIgI2JTYZZZZgEBc7RkPBUqK\\\\\\\/eQ4qWBl2jLxQXZmOWx\\\\\\\/qBpgZH\\\\\\\/YEIZgmCV8LG\\\\\\\/RU+P3h3Cw1iD2+k9Swi+\\\\\\\/IdKF+M2LhZ\\\\\\\/cVR7IGp\\\\\\\/kcxIGD5yf08dLKMhgZpP3IPyCXVZnPs2FOhB+Ne4mn4qF\\\\\\\/JPuNWyGOYSziNpYHTOgm02X\\\\\\\/kMh8ogTCLfwtSXJF0l3ZQDakdzsD7xX41m75XdV8+t1az\\\\\\\/exEOBWzZCR0N7m6Mm++2q5FFn6z\\\\\\\/cCyo785Y7vPXitmdBUM7ZAIYIS31+UYPGn4wAYawXd3rK34lCwOyiT0FmpyTJNZzdW+I4DMYkiayTtfZCIiegeO0hIsdCs\\\\\\\/AJpOeUf+7GIIJiE8YlaWDNFQhRLeUdFlgD2YXdW2jyXNhci51vykEYkCrRjl9qVv\\\\\\\/Dsy7\\\\\\\/3FRBtNTVRYQC5aBdZBRyMoOFU6IEEB0lboF6XuIwqf\\\\\\\/HxzWPmAJQeqjF\\\\\\\/mcIm6DgbCN0gTqyvs0tJH2EO8Eas8Zt1TDLBnWfhA3gNy4jSY0iRYtorQszft+Jj9\\\\\\\/HmgdKGzAfwcMWmvh6e6RDabsLFlM44zOwCxINJ7Z1LOUltkFEW1+SAQBnFxVjk8SN28IzH5pbqpZ0CZtB3VXY64UCkwRA9MGr+HLkeBA8uDBUMW0ijDSIJVD+oFwjlmzHdy2mPnWHbCHa5n0KM1gRv1K0dBvNfeOkMZ\\\\\\\/Xj8xBOHkt5zXiAP1zipckfT32xtOa+K0j177EgIChVLbnIAT+ceqvXJEJkpWbf1dpV3tF38g2oGpV3k3rkONw6QmNItLUlEntf8qvVtYKbkj4sMIBD1hqD3X4Or7XZktMCMO\\\\\\\/ZIbl3xPTogf9lo9KXXGiAVig8x9T0oiZW82kypZPNai7i4RfHWolfCYWV1zhbCH1JRAWGvaoXB26EvtEkkS3kDT198oNmGP6Q8NIyyNB\\\\\\\/Fto31zJdPSN+aLTRRXdAFBRIOf12iL2fRnLkC4z+CwrHiRcbxMG5SncmVRoN8JEpQ1jLGp5E0gSGtAcaQOxcP\\\\\\\/a03nYW8kAhIbgghPETfhlMzoNSAmfCJ7wNxz7u3mNFtiI2YhyR3vgl6zrQM1CM4TK\\\\\\\/xmsh2JyZ83LytSKmjNldThQtQz5maYZKJQeROagPxGQLXwXFpkkrezW+\\\\\\\/71kdyZ5S7UAQsXNo\\\\\\\/sSOGoBjn5p5vzPn6XV8h4rSezW+Q4QUjji7CBel1\\\\\\\/mxzbgkgNTjXZM6Gx6SIvvjq5v7hDTIa8bn8C6ZT\\\\\\\/XptVuGVAfvjaT5F4IAde0xWi\\\\\\\/wWhTESrDRQtI2LDUT276E0uvtvSthYW9mqEOrqmDVeSOiN2F4VI7mOBzDPsMTkZ\\\\\\\/bAG5d0GFDZDKh0W3PyXfv30dSKEm32Zgd9aa0i0EnqXxjwVLDLn+NLVY+n45GJgpdqrkbSa9O0JEqCDlWEe0+FO1+uhRSg9wEXBtaIzZ3HxpfSMIV3TRLMSCaw9jBYnTsmCbs0Qus6sdF3W+EcdYsnQ+\\\\\\\/u6bGhK54Cp69RcGnaFVcTv8ACyNJctJKX5oqvkAIlUZV5+rer7hdrfoKMcwpsyFCQsBy7f2ecTAP4iNK7CXm5ot\\\\\\\/IDBoGsZiPK\\\\\\\/HYcUN0QlcZvsFaqJGYlDgGkC55xzu4ryG+XlBYLhcQf56jRnpYyAz9HR1IFPqq3\\\\\\\/n4ujSePCpvtcnVZuM0IaAGkIU5aQKvoUBiWtrRoBonXWaHelaFSIy2JWRqG1eGM7PZpZw11GQ7NSJXd5sf\\\\\\\/mbV5H\\\\\\\/3KPQNKAoKZnAtBJWedKnH9tpUW+z3xOIpqWS+7zOdHTfas22KtR8iAa9lAkZMfIrqW9YRRw0d2IkXBA6pJVA4mhNmUCBEh9wB2Gkcn6OQi0W2zJCxdsWe4kv8jdbemI+KiqfbInpsZmO8qvsq\\\\\\\/dKRY4icyl2et742seqb08dnQY+k8+1h7SA9skXohEm3OWyQK9gcXZHg6sSWtTVjnj3NWARudLU7wky1tVte6H0YU8JRuW8et7bPt8ar\\\\\\\/bqTygD09\\\\\\\/BC\\\\\\\/PQI2h+254ya0z3XBN\\\\\\\/Br3nTDEdejIz8\\\\\\\/KztChKINmzGnBpUet6kyAp79E4u0t5y+\\\\\\\/utemKUhLRVU2b9zBFuI42sZx3Hb8Q5NHJUPo0532\\\\\\\/lr9n6oIgn\\\\\\\/hVKHxBJVNe0a0Tz67b5lmBp20EEvHzl7RKkVtAkdU65UhUNXEgyKxgkorOuJYQlLhDBZ7JLbsR13mx5uke7Jp4PPkQqpej1AAgd1RoSF8qCn2+HxFbqkxs4GRBsWIBJO8XZgS1kNEQ1yQNvhnhCdBUZBn0u7qgDyeCr1cB\\\\\\\/g4dq3dDCT4s+jG\\\\\\\/g+wAD64tWVMUD4iK+di2Agv2O0NKHO2Dv+hdC+GLPq1AcqSaClHacBQghNTa3wW0MbGP5qBqnInVEl5uHKMCZ0OvyeByRAIvIaJA6owTc9KcEyBEu3cnieLTX6YRJEydADYrbNS+prlAfDBHvpX+Kyw81Swtn\\\\\\\/gM9UHwd\\\\\\\/wQEWuWTkQ0spcdt8wu1rTutz\\\\\\\/sBKV+PqZVIFxgtXzdr4zpmlLxYb1Usgwihfc9Ayej6RK+C6aYykzPT4rfKf0d1dt6T\\\\\\\/x+b31KZoMRRFyKIEQoEVM853\\\\\\\/Act08aN\\\\\\\/bGI+g+qIMsIDpaK2N8s82xFfwefoo0zzn4DyeXpfhE50Wx0NE3UO6JrwhrdLGrlvKilABsruA\\\\\\\/LBVi3mLxkI8pbO9SA+RbSz8JC+SRNByXOZR5pr\\\\\\\/rbdUWrby+npqgUGtjdAg9nrq9QBbs6vQJGbGVZWpBFtfdpqbcpW4vI+bHqVo25jsi3dqyos4y+MhPJxwpqJvWZ2W+G4KipqyFu7zobLu6jx9BvG+WhB6Iv5pH0P\\\\\\\/kIPR1OP7netwzCp\\\\\\\/5oZvTsRIU6Z8e+nELqGRBKpf5iV3z\\\\\\\/G63AxK24czjU8A9nGhBWy61rCev0Gdx0g05QLQdXBlCbZ\\\\\\\/EWYeC0Q97S1Vra7BbWdLmV5Z8V7+k8F+8G0kxSDOri4oi34vzUDFKTw3Ku+ttdcHNsM+g6bsnYk60jEMpMO8q7CLsLGy4WmrkiVnNeSH43qm3qzGT4AFTgTx+185k7567CWkjEPVZkb7ZSvVFoZXs\\\\\\\/GSIywFou8SgMDrsDCHLsSNFBBzd\\\\\\\/BERjYOMToVcN9Kmr2FPluM955N71MusWSzwaNstBasmm7DF4pjGf6sFM\\\\\\\/\\\\\\\/XW+7xqnAN3izdWDde\\\\\\\/GXzNYccXwWVOqZcBBPVH5jSfPmCFz5ezY58ANLLcBkvlYUadtvHXTLhfdvxQOUtCBaXJ1KVfC\\\\\\\/ZRIMKKw5glUO\\\\\\\/82eo0KYOEuyCJaDjNySK6aDYKpSnsiPWWMaEqV4BXnR2RtOQRGLWLKPMxMy8N6md\\\\\\\/AO6pcs02JN7jt3zWzTIwWEEHFtBqSVD9D81adJW3B+nOcanKOL0IHHCxa7N8zBgVS\\\\\\\/liBnSEl+u7iFagwEgG3X4xbhxB+LFrd\\\\\\\/AtCjH7Cb1A7PcDOw6pIwnX8lb\\\\\\\/7PNdP3bWO1E\\\\\\\/CvSUzun10vGxSTZFv0zkQ4nTi3pYTlUAH0t+g41lszHebDKdWxpYBliUalQejWwcs4X0nxMjHhP0eityLtea+fBph92CmGTXxeMcEX4tzPShgR0WRcauqZs3UKpQN\\\\\\\/LSqesnsEaAHxfXX6lgyYXx7V0lqFdvtPBrSLPLRDAnwaILgpMYsVtzD\\\\\\\/iGWzyFX73BuCKbL1vTKVAqDctdJwjvJvJslYgYCEZpF5soeNURfXLCHyzNjgNdTCWbNMWBreIGuDoafjw9WUndNkoBPRpSv7g2XwELM10BmscBFC3g9IJbflwOFSR\\\\\\\/K0brkFgInbf2ZYXF+237LX65zTN1qoNSCc+o893ANRKFSNckSLsZ6PbtxH4g08QyHWP8b5CaP6+migOSoSORvGY1vQ3n5p0k7aM4GfKLHaP81W46yqoTnoetzI3nsMFUirPdUD6ZUFXB72PFNU1FZD2DT0TFtXc2HxEioTvfwhHq18Fcv\\\\\\\/+VN9R633PN\\\\\\\/raj0Tl0k9xVSIBDSFkZKRmv6KVUF1G8bYEC8QcVOVl0oq0Hirl1gbsoEDofvU9VxDcd8JP\\\\\\\/sJo4sxIpGK5doJYRYVO+LM\\\\\\\/duaxsizVj9WmXZARDCWVAAjx21JaBbtYVW+ZCnZTD2kWGQ7TMjKzbr7dA4Xu1ImwJfqCf6ouDsYEaUXWLR\\\\\\\/ftrDoDOirsUK5ssr4xm5mBVuMMYATymsoHUy1m2q25NcCva3yjm97Haj6RJXrBTZ8t8hEj3NmuUqowq9Tn646gbCWXAnxZd5L\\\\\\\/V\\\\\\\/AaMDjlQiKwzyxP5iPiSplAJOX36wMTSHt1HlbdV5UWGhk\\\\\\\/cuxMHv3FQontT3VAwA3C6kzH1qnyQirry9HJMgf37pJiZn46HkOnDvTMWBPIePsy6G5QbF7OIdTf5iOKRkJ5y2RfOhApIO6bIbt7PvOxShR9nkrYU5EZxQhngylDW5OKSy++4zl\\\\\\\/pKojC3GPt8Uk2GM\\\\\\\/e1JFml3cVTC1+qQ50Jfm\\\\\\\/TgLQRfRXZvyulJpzGpZeh5KmlEmBCznhOVt++MBTxZ1v4fEwV7AAnjb4W7AScZZJB0GTJKVWcJlhLLRl+6j9iPUpbX9uby\\\\\\\/CrZNqCSe9yPMK6uzMJ9ml\\\\\\\/aTNJjZRuKIm3\\\\\\\/8tin3xL0SSetBGgMCNJdM2VBqhXhVOZGj1U5UD6zigi+XqwwR31cjy\\\\\\\/aQ+oGJHrMqaPauUcgWpS8SXGo\\\\\\\/qcxZmqlgmOVuXlfqht8L7O7a6\\\\\\\/HN8RiOGMAVpak0V7t9ilsXcxGXgTIB68K9uSOnXv1coSvMYeLvCVzGR10Ncj2LN7Qxc1fL04iPkxgfI9IkXWG+RssDxCx33OPCicUZZSGyj7Nl3oR\\\\\\\/pzWrTzywx79nKgxvtkaZR1nS1864N2jN879rXn3hwkLdeA95cCLoeyiwl2+TjngzodL8cxAw9mWWG20vFWzUiPPvWQ+2JXR6ao\\\\\\\/Hxscr9hKPk2DCkaTLdH2wKUvvvI+jOkkYY7DxUi9WClCo2Y6kBDLz5+z+6K8SBdaGLxR1SQWqcSyqQSgkoRQCy5HppGwQRy35U0nn14Qo6pF7BbluOlc0nrpg1U7efWHggqQ1DO3oZDvIADE3PRqd00f6OZ7Tz+YK+ubqBkav2zvmWbVVOKMOyznW7sfrdwE3oy8yZ6YVS3T6Oyr77hTebIN800CcCpbOks0rZJFitA83OB8bt512rN4Mtbs95YiCeqikspnL\\\\\\\/eXJ9fVe2Njw\\\\\\\/uNs8eJzEX3zm4KnQKwCoiRqt+KRX51rQupuh0l36kLVA7+JZCSxq5AkS9vYWLUfxreQxbHACq1N2IIdJ1c1nLnxvTGFqLZRSY58XIoyOschDxxzXT9mEA0+MssSN48TiiL2oqKd43o7TvZinEIVwQQrDEuIJpFKsVxoJ655F30hEfKb\\\\\\\/KIt1qzt6QzD1P+NGRDFdX+ZlQ4RiNJ8zIk7d2nOQ84W5bZUaIawNpCk01ONWgGEPqtdfA7hue+Fgwp0FCw26I9ksckqPIM3LbP8KNGV6jp2+493mPrePi3vVG9Fgi4zmhMFKHF1P2qnWBKObXTEPvspC6JAO0xQk9IgLj6w3gwFjoTi\\\\\\\/+tVoxtT1E7\\\\\\\/5zJ06BeVf\\\\\\\/muTbV7NruSiGmGGNWVBsh80uaMbOeW0ujxe0l3FljLXxynFtgHvIzlI6Fvw9PvkySdC5Wd+APnONdXXmlevI1YS1Uk0ROwjVhkXP6ntYQyQTcCmjdLDTNRHmRgeB8f0dE8x4mzzgX0oLGlj0FqpwLMQ9S+HVOFHGDIkanEDHpOpuDDC5NHnr2nXzNab1MAwvDVIcrweeNxnC9FI7jpEQAvXKTQ0wXCfw0TdKgMVqETF6Tb7NwPJtCjbBX09y738fFMvcBXgUcJnsSvnPL9W7RgvZp2pvBfoUBLoK3UFEDiwP3yigSMUgf683iKUg4RETXKht9kORWXdBsRKab5++5m5NvKzd2WAzOc3LZoL7XOicaVja54mmvrm9eT7taea\\\\\\\/tJRX659kz4TSTQlFvlXVOl2zcdftyH4\\\\\\\/YalcXO4HY\\\\\\\/2DCAFldoC9YH0wdE\\\\\\\/J+BfBFbD9NZzuxHhrsYdl5x1nj2BjBjBrMZEur1dP+i5wnLfLHpsCOoNFBSkJ7wm5PE7xa7\\\\\\\/IerbJbUwQBwwFBdlK4BHeXHc9gMEeF7PknDrs3uZfHnYsngieQcqpd\\\\\\\/V8XIJCpP6uRdyuWQj5s2Z8rtMfTzMzQSXdm7DnoNrslT1BYX0nk62ivCg9v3X3zXvZ16qTiSwEKiDDL80jIwXvfNBB+N7WWPkK6tM2X3j8BF2zVo8ziy\\\\\\\/mVX0T8JJ8GWJLAPKD7pHAVJdxZTxoaNNxoSF348LNas5LuPb7jCyQ1fzL0HjBfHoV5h2H+kpIBdK7Rt6S2iNNZ79KVwx+9OtkDnCIvVBdXv06xq2QfV+i4j2bEVqUl5kqAKvX+ot7cEUtdyBDX1c8hnLYHLUwx0fNZ4UM2tUvviHUKRHhx9pvx9\\\\\\\/ADvNpf7dGvrnVeeO0HftmNQInTdr8eKNoeJaKRIGiQm1gjLaBOvIG2UsAgxgKyyLB63YmcLU6GOdozVetZ6pBKSQbXQ9i1cDYOYtE7e3rjqM+uz4\\\\\\\/GxJGtYIieKs5OBfTKxJRcQfzMgrEk0M4p+aC01+24OdiCottLDj91w2XJfJTdUEFpB7eJO49uCNFDqL1oh2NzCcWvEXEWIQs57PTwt9Pd91dXdv\\\\\\\/NU8TdjqTuqyWmFDPdQFYU5SXDRPepvdanibs4cGrjJyhMFPZGgbgyluV5LcbHKcVuu6e9eXODE+QzP7KuV7Opj9hbm4JuA1+562p4YtDS+uu5ZsKgeXD8hlkMxKE+DSDHLACy6yMM8yrc56sprXXLgvi\\\\\\\/Im5tEUaW7X4NOBlJEJWV8lY61oyJhxvaAdMya\\\\\\\/ymDUNg9SvDtZEkzD7I1PjXRLU6b5SqR7kKOoL89lAQugj9laRpspUChgNWKNEBS\\\\\\\/ndyo\\\\\\\/VQzfLA9ieq6Vw189WmzOmjWZ8exqZE9nYGgy9RbbJEwSI1e6N8\\\\\\\/aQyGMmebMK8aPnGATVFccKOCKeFc7SNYlsJ1vEbzCkJ\\\\\\\/TtcJl5N0KcMI9ATAdS7Ymwhxfslsxrmhkjd0alAylJqa0RItAn1a16Cl3pGtDwnrqJiQC\\\\\\\/1yru64UBxv\\\\\\\/i46UY5jkUcmwNXNvZvjnhniA8sGmOeaQfAaFzAXzSdKuYoTVRp1t7HPeFYqZ4rhxeV5+mDtlT4SB91WtbKS5xr6hsAPpuyJvnDCiqMVGV9k1L8\\\\\\\/l9au1BmSdkzSxPQhPqgUwMpO6JivoLIyOva97QQ5OxL1T\\\\\\\/u5HsvvL1gc7akT2wHiXw7ua2yQlJ0UW3js4p9J+erUJIurAQYAKdGfdY2UGIGrgfXjNkHTkjM5axN8seXecRfVnfPTUb1Swgvfhycv3M2tKfaj\\\\\\\/26pHggBmERn\\\\\\\/2rChDBbYcI2dlz8wdnhed2cr+VIiDyhHVzvgTw8SsbvYydMLFPohyyrBZ7RhIxEhyxYUscsIzTVc7\\\\\\\/7DLBJePVfsycZ\\\\\\\/kHrIUXWYtAsMO7Odxl17+iv5pSTXNbwBz1GISGnbXjNxRhWP9se1oYkxYkfln9Rzh2z5L690jpYYjhf52742mCMlUsizMVy+nKk+O4KodWPuSJQ35Ch5Os32WAtNiX3aqynE75MFCqpcJuU8Z1wad2mlS10XFLEQm2WDi2C6D\\\\\\\/vDnwPN6wagdf38CihcmynMi21ASTipHd3SD15lHNf6zIeTP3YlQ65k0Hc+76yl9c25gZlkWKibt9RJWmkGWEVZGn4\\\\\\\/9PL1hxWuiRXfYUGy1bYXZVSd35UlZOqRalXrzAulkLa4ChbgVLfb5Et9ewYb5ATdy+85aDU69nWu3Qs2N+p2L9LFqsFiwHamORZIo9UH3wmpCN5H4oFPDJj\\\\\\\/YdiT07LD2X+OXcbN2owzAgXVwG\\\\\\\/qH6pwKqCSjI9pift6VxrCaz5vbOnCC1rFXgNap8uslTtRd8mKQ0zpfU0nNM9sTFZYrI3pSLIVQTyl8b+VaOdTq75UDpl3AoahowNcnL6VzApG2ORl95XAhvMvFv8+1hCU6RjXpdAFweQ8XSK9y6dlCaTB0ltUuvQ+lhyK+A95Wtq0IzCRHlidgikZq2acP2y9t7WbQLEZjKz3yf\\\\\\\/dGr5mfT6Cd4GIP6bEiSOb5BtZlhWQQ0vdsw9cB1GhQnOvTY5AKBZFl7hbc91bjsKhZRvFGpO3Wv5GGmW6yiAbwqsSLkMPRlRhFmoyXNFefG1mh+K\\\\\\\/wZvw+62xIyarNgcxUejiaxqmiH1N\\\\\\\/v5UslPN98WorCpDcLBYOVl9NWpEx4Fev6V+pQJU0Y5IEkFKrk6kveZs3G3jO26wRg86k2IfSL48OqfexlICIzku4ydJmXA52W+AnKR2yir0Vfcz0EbWDrqblH+LBWUgXWUUI6DPsaYb5yEh0JWVmVbEZ+E4H0MsTEp0cJWgcQnXVw36XDIKB6hlYte7HZoerqquU+bpy7OMBFThiwhkyD+Ye2FuwyIn+P3HjCiECJa2aEO\\\\\\\/TFBq0CpB95kMvmrFX\\\\\\\/esQjtzBSzqan5lRxUYAc4PD\\\\\\\/Wz+E\\\\\\\/gLKVJmzgrQD+Nm42vILaMlzHjLpBRZtHAbatqeI7qcKfYFAr4DNG7VjC3ZWQyBIaxFFGcTV4kpsVtgQxegpfYNcBPeeRjAIOyp+23q7YP6kr15FspNVIhT0q6Q6ZxxLsQQXTQ8xP4tqfAJQSkGYnHzZgl00Wtk0tlYcNsA7UayhTrXv\\\\\\\/maAhqevFS1Ep9TTxxQkoToxzbheBv5bCo6aFBBmJZIrTiRKqH9uNCa6iqcm684c49g489vQS9C4LljRa4Dvvt7wxbELC7ssiWvwqkO7ZInmravAG21\\\\\\\/5ryhpY49Js3CIN2gJiumoFYXMUbO+eZ+sjj12C8yKn2gIHT2loZrGKKgrXIthW5TVv7ojPYkR7rYMwVh\\\\\\\/7s8LIz2b9j7GJ+pRWhOcD8wByy+TNQiXk3QNvYfDROPSTUdqXzd7z7e78Ug==\\\u0022,\\\u0022iv\\\u0022:\\\u00224+a7OgbBmncG+nU\\\\\\\/wz6rEg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a56951c4848b6b2864de3c6e003f9288e846affc1f95c91287c5b7e190ca6799\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022dCu2Kas\\\\\\\/qsra9hmlpPvfnhLwK2b1bbPGzAIFnPFFz4WyZIvMNfE\\\\\\\/KDoZVvx\\\\\\\/hU8fVd6p7l1Uqn7fJNGTanWY0rbWRLcEkm8g0Sr3C6hZlFGhIr5FdJylxw8aVrPagx8wBgZIZvFatcM\\\\\\\/cKMBonZ0RqXkIQ\\\\\\\/gZIchrMRrDjcF+7b4vFVTBMsyVDSY1FZCY4crorajO4gi0wKEnK4+0d0d3Y0+ijAhiVT85kXckSQKPhZEue4FdU6taVaT590qT2mfyRrvr79IUukcoY1d97qp7Wu1R4ZHFcRLg\\\\\\\/hibu7J25QPR++fzHuV\\\\\\\/mVgYKSLs2IdRnmnuAW\\\\\\\/iFwEBCNVgYDN9KCG2fA9ASRJ+KDm+RFigSBwbAlaqJKAzVFoIHs\\\\\\\/s5O1j4DgXNDXHgZHO\\\\\\\/3T\\\\\\\/scBjd7Fzg\\\\\\\/CiYc8LAN3i8UXKZiLPEp7FCCv1z0eTb9511z506NWXTTQGYLhkhcmA66INe41YHDVsqKnim5XAC5U2stEllRsJ4BJ+CTCsXoxDshyOVLCvEJjf1xz5BOMg3bHebJbk6bqLnqw8GiZANkSPMXrl3eYLsnFjhyovHL1cbBDOjZBhZZF+adnAzoRV91vfpu9eW8jFePBgTA+9pdNz4s2H5p890WU9PAZ85aUh1+XP\\\\\\\/7Qp1g3fNG1vZjnTCjCm1EnXRpWGdqyipwX+V8SrqrRWnCp7jci6Fut42WG2XpOmqi7cHDCF4kg4fITPz7adfwIn9hd\\\\\\\/JANwCgC32jAP1ZJwGpqgjoZqVW0cW62TrAXJ5O1cBQfZblpJuxa2fV0hBAdQvY1mDc3TgebOR3CpG\\\\\\\/5ODZe\\\\\\\/tdVC9f5yfLvp+dX2DSTn2oP8GsUXHzralXaLeO\\\\\\\/FMM0h88Ftmv7V8iqpfjCglSye\\\\\\\/yIgdGvgGmNpyuh6i7lZUq9UyL5cNDA9PgrjgM+70W7GPksqT8jaTzx+9WmELK7MM9uEjlxyiu6Hso5GlYAXbvnRmblg4ByZ7GT\\\\\\\/yAZ8ZhJ5ZHoaNMrTD7aHv+MMaGK1FQ17g1vR81+CC7ci1k+UMuX+\\\\\\\/cKYg1TF4wWMblwnh13+NXpUyU2atkS5vYVuHBz+YC\\\\\\\/16dKiyRaxocWZMACt3pIs+Y71fk35ftBA0v2Reup+\\\\\\\/4ZmdRPBoLqldsJrBKEuulzn5hoAQQoY\\\\\\\/cboBQs4KTrkxzpEZV4Ee8JvX0RxcrxwpRbJkxQgoOATAM1Y1F8P+Q0FjD5jnFtvKLjGTfGtyeldoPhMsPd6E6nJD20lxUF1DBAOGOnz371xmfsW7F3jE3P154EVz38taXrmu6bl3WnSpNt8QPYf86oQLvhUdqBybcw48ESWahCnEZL+aFx87aV2VI5GLZjm9xlF6IfaVCZprqCAH6Nn7SCDhU70oi0ft2UCGrslh09MToomU\\\\\\\/GvpeaeN8jStyRl9zTGv8IG3osxBXwVsMAM7ciWATsCMQolt\\\\\\\/+PKe0bTPCsfZpbl9v1sYkv572zrO07hReu7ANMzHPDY4aCfDGtiuuO\\\\\\\/xX8W1iepdKwAU10JEZ7khaZymKrreh7Z7QxB8eCpnuUZey7AMVrGDCG5yMW1UnIRn28Q\\\\\\\/zqAQnSvtmubzB9nbeRmHKMFobbXDtLOb2pNJu9yJxUaxeZglw\\\\\\\/OQTg6gdUMYxJ2mNGJrjIajAbvq1nzk1qd6uaLX+YNircHg46yon54567ZwhaygflokOe7HH+JgS8QOqcZ5bfGoQytuFNNBuNZHrFkL+CTADK744oF9GE1PHKpuLh4L8ylNpQDA8yGt0dzQHm4Ly3DBHkGAyqtynbYbNwbzPTfovV4t0llTdat+t1Wv\\\\\\\/QLTR6LsWcy2\\\\\\\/CgnL1fU3mRAdcVv0YTV8r6J7ue2jQqdH87MR+ZsKzgCPw4g+1AxMfRyugw32A8zFpSKoJFoPLV\\\\\\\/17dC49PIAioaolMrHdfsdE409dS9W91ve2A8OU9qD0yll0PT3k+OV+Uc6P\\\\\\\/QK7n9D6iStmNhBzQF3pAq0WFOU6qE7IDB3LwmMj53RVUuqvp43mD7kWVz8KQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022QgQ6q+cNa9QWgssCQ2Gzgg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022780256811058e820692798883ab127e1e68c8d43853b97a739472d735d4646ca\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494060 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/2851" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-NNCQSgNUV5TjDbVimnXfcipcFOuffNv4VdPZ4B-REmA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/2851" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ceOHHkmlWkWlNQnrbfrWXgkkZe3K6hn05tWFvOT63Dk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/9/2835" title="Single-Gene Deletions Contributing to Loss of Heterozygosity in Saccharomyces cerevisiae: Genome-Wide Screens and Reproducibility" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/9/2863" title="A Prospective Analysis of Genetic Variants Associated with Human Lifespan" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494060" id="node494060" data-pisa="ggg;9/9/2851" data-pisa-master="ggg;g3.119.400262" data-apath="/ggg/9/9/2851.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Developing a High-Quality Linkage Map for the Atlantic Killifish <em>Fundulus heteroclitus</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-2326-1855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jeffrey T. Miller</span>, <span class="highwire-citation-author" data-delta="1">Noah M. Reid</span>, <span class="highwire-citation-author" data-delta="2">Diane E. Nacci</span> and <span class="highwire-citation-author" data-delta="3">Andrew Whitehead</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2851-2862;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400262 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494060" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jeffrey T. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Environmental Toxicology, Center for Population Biology, Coastal and Marine Sciences Institute, University of California, Davis, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BT.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20JT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeffrey%2BT.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2326-1855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jeffrey T. Miller</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Noah M. Reid </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular & Cell Biology, University of Connecticut, Storrs, CT, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Noah%2BM.%2BReid%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reid%20NM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANoah%2BM.%2BReid%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Diane E. Nacci </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>US Environmental Protection Agency, Office of Research and Development, National Health and Environmental Effects Research Laboratory, Atlantic Ecology Division, Narragansett, RI</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diane%2BE.%2BNacci%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nacci%20DE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiane%2BE.%2BNacci%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Andrew Whitehead </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Environmental Toxicology, Center for Population Biology, Coastal and Marine Sciences Institute, University of California, Davis, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BWhitehead%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whitehead%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BWhitehead%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:awhitehead@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">awhitehead@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/9/2851" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494060" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494060/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/9/2851.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494060" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494060/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/9/2851.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494060" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494060/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/9/2851.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494060" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494060/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Killifish (<em>Fundulus heteroclitus</em>) are widely distributed among different aquatic environments where they demonstrate an impressive range of highly-plastic and locally adaptive phenotypes. High-throughput sequencing has begun to unravel the mechanisms and evolutionary history of these interesting features by establishing relationships in the genotype-phenotype map. However, some genotype-phenotype analyses require a higher order of contiguity than what initial scaffolded (fragmented genome assembly where contigs have been assemble into scaffolds) genome assemblies can provide. Here, we used 5,685 high-quality RAD-Seq markers from a single mapping family to order 84% of the scaffolded genome assembly to 24 chromosomes. This serves to: 1) expand the killifish genomic toolkit, 2) estimate genome-wide recombination rates, and 3) compare genome synteny to humans and other fishes. After initially building our map, we found that the selection of thresholds for sequence data filtration highly impacted scaffold placement in the map. We outline each step of the approach that dramatically improved our map to help guide others toward more effective linkage mapping for genome assembly. Our final map supports strong conservation of genomic synteny among closely related fish species and reveals previously described chromosomal rearrangements between more distantly related clades. However, we also commonly found minor scaffold misorientations in <em>F. heteroclitus</em> and in other assemblies, suggesting that further mapping (such as optical mapping) is necessary for finer scale resolution of genome structure. Lastly, we discuss the problems that would be expected from misoriented/unplaced scaffolds and stress the importance of a quality mapped genome as a key feature for further investigating population and comparative genomic questions with <em>F. heteroclitus</em> and other taxa.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/atlantic-killifish" class="hw-term hw-article-keyword hw-article-keyword-atlantic-killifish" rel="nofollow">Atlantic Killifish</a></li><li class="kwd"><a href="/keyword/meiotic-linkage-map" class="hw-term hw-article-keyword hw-article-keyword-meiotic-linkage-map" rel="nofollow">meiotic linkage map</a></li><li class="kwd"><a href="/keyword/rad-seq" class="hw-term hw-article-keyword hw-article-keyword-rad-seq" rel="nofollow">RAD-Seq</a></li><li class="kwd"><a href="/keyword/genome-synteny" class="hw-term hw-article-keyword hw-article-keyword-genome-synteny" rel="nofollow">genome synteny</a></li><li class="kwd"><a href="/keyword/scaffold-inversions" class="hw-term hw-article-keyword hw-article-keyword-scaffold-inversions" rel="nofollow">scaffold inversions</a></li></ul><p id="p-4">Understanding evolution, and how it generates taxonomic and phenotypic diversity, requires tools that can examine now genetic variation is shaped by the environment. Killifish (<em>Fundulus</em> sp.) form a model study system that is well-suited for such studies because they occupy a wide range of naturally variable and human-altered environments. A significant and growing body of research has documented locally-adaptive and plastic phenotypes that have been shaped by temperature, hypoxia, salinity, and pollutants (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Burnett <em>et al.</em> 2007</a>). Population genetic and gene expression analyses with killifish have revealed some of the genomic features and mechanisms that underlie these phenotypes (<em>e.g.</em>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Brennan <em>et al.</em> 2015</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Healy <em>et al.</em> 2010</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Nacci <em>et al.</em> 2016</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Oleksiak <em>et al.</em> 2005</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">2011</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Whitehead <em>et al.</em> 2011</a>). This work has been facilitated by the development of genomic tools for killifish, which now includes an annotated Atlantic killifish (<em>Fundulus heteroclitus</em>) reference genome assembly (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Reid <em>et al.</em> 2016</a>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">2017</a>). The <em>F. heteroclitus</em> genome has been available as an un-mapped assembly that is sufficient for some analyses, such as a read mapping template for RNA-seq and population genomics (genomic DNA re-sequencing). However, some types of analyses, such as genotype-phenotype mapping or comparative analysis of genome structure, require a greater level of chromosomal contiguity than that which most draft genome assemblies can provide (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lewin <em>et al.</em> 2009</a>). Mapping the killifish genome assembly into chromosomes enables additional research questions to be addressed and furthers the genetic toolkit available for this model species.</p><p id="p-5">Initial <em>de novo</em> genome assemblies are typically composed of many fragments of varying sizes (scaffolds) due to the difficulty of assembling across highly repetitive regions of the genome, which is particularly challenging for assemblies that are derived from short-read sequences (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Treangen and Salzberg 2012</a>). To increase contiguity of a reference genome, scaffolds can be mapped to a position and orientation in a chromosomal-level assembly with physical mapping (such as Hi-C and Optical Mapping) and genetic mapping (Linkage or Recombination Mapping). Physical mapping resolves a large proportion of reference genome scaffolds to their physical position in one or few individuals (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Schwartz <em>et al.</em> 1993</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">van Berkum <em>et al.</em> 2010</a>). In contrast, recombination mapping relies on the statistical detection of linkage between genetic markers among related individuals (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Sturtevant 1913</a>). Inferring a relative position and orientation for scaffolds with recombination mapping inherently includes recombination rate estimates along the genome. Recombination rates are particularly useful for modeling demographic and evolutionary processes that affect the fate of population genetic variation. We chose linkage mapping to 1) increase the contiguity of the <em>F. heteroclitus</em> reference genome and to 2) obtain local recombination rates for the species.</p><p id="p-6">Initially, we applied common filter parameters to our sequence data, which resulted in a low-quality map that commonly split up scaffolds and resulted in unreasonably high recombination rates. To determine if the use of arbitrary parameters had influenced the map quality, we applied a series of custom filters that were optimized for our data. We optimized the threshold parameters of our filters by visualizing the empirical distribution of the sequence data at each filter step. Our data-guided filters and agglomeratively clustering linked scaffolds (merging smaller clusters) built a male and female map that were more congruent with one another than in the initial map, and more congruent with the other mapped fish genomes. We merged these maps to correct and extend the previous <em>F. heteroclitus map</em> and to evaluate the quality of the final map. We predicted that our final map was of high quality if it tended to have conserved gene synteny with closely-related fish genomes (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kai <em>et al.</em> 2011</a>), and if we recovered the well-characterized chromosomal rearrangements that have been observed in more distantly related taxa (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Braasch <em>et al.</em> 2016</a>). We found that the quality of our map improved with our heuristic approach to minimizing the retention of deeply covered markers that contained genotyping errors and by minimizing the influence of spurious linkage caused by missing data.</p><p id="p-7">Much of our approach to improving our map consisted of routine, data-guided techniques that have been applied by others or implemented in software packages. However, it is often unclear how and why important threshold parameters are set. We have outlined our heuristic approach for marker selection to provide a simple guide for using basic visualizations of the sequence data to discover the dataset-specific filter parameters. These parameters are discussed in the results and discussion section. Lastly, we use the killifish map to highlight some interesting variation in synteny, and briefly discuss the limitations of mapped assemblies when inferring fine-scale structural evolution of chromosomes.</p><div class="section materials-methods" id="sec-1"><h2 class="">Material and Methods</h2><div id="sec-2" class="subsection"><h3>Samples, sequencing, alignment and variant calling</h3><p id="p-8">A single mapping family was generated from a mating between an outbred male and female <em>F. heteroclitus</em>. The male and female founders were sexed by the presence of secondary sex characteristics; males exibit a colorful speckled patterns while females are drab and become visibly gravid when reproductively active (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abraham 1985</a>). Parents were collected from Bar Harbor, ME, in 2007, held under clean laboratory conditions and mated in 2012, producing ∼150 larval siblings. After hatching, genomic DNA was isolated from 90 of these and both parents with the Qiagen DNeasy kit and digested with SbfI restriction enzyme. 96 unique custom barcodes were ligated to genomic DNA of the 90 offspring and parents, with parents represented on 3 different barcodes each (File S1). Barcodes were distinguished by at least two mutations to prevent erroneous read assignments. A total of two lanes of 100 nt paired-end Illumina HiSeq 2500 sequences (268,373,087 + 360,771,323 reads) passed quality filters. Reads were de-multiplexed and counted per barcode in an initial lane of sequencing to determine relative concentrations of each individual with the pool. Individual samples were re-pooled with readjusted concetrations in the second lane of sequencing to normalize the number read per barcode. Reads with low quality, without a RAD site, or an un-matched barcode were discarded. Barcode and low-quality sequences (below phred score of 10 per 15 bases) were trimmed from the sequences and sorted with custom perl scripts (File S2). Eleven and 23 million reads were assigned to the parents (female and male respectively), leaving just fewer than 600 million reads distributed among 90 sibling offspring (average of 66.6 thousand reads per offspring). These reads were aligned to the <em>F. heteroclitus</em> reference genome (version 3.0.2, NCBI RefSeq assembly accession: GCF_000826765.1) using BWA (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Li and Durbin 2009</a>). Duplicate reads were marked with SAMBLASTER, and all reads were converted to sorted BAMs with SAMTOOLS (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Li and Durbin 2009</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Faust and Hall 2014</a>). We used SAMTOOLs to calculate the depth of coverage per base for visualization and pre-genotype caller filtration as discussed in the results (custom R scripts, File S3). Alignments (BAM files) that met the pre-filter criteria were passed to FreeBayes, a haplotype-based Bayesian genotype caller, using the four best alleles per locus with the ‘pooled discrete’ flag (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Garrison and Marth 2012</a>). BAMTOOLS marked low mapping quality reads (x &lt; 30, reflects base call quality, depth, and possible number of alignments) and improperly paired reads for removal (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Barnett <em>et al.</em> 2011</a>).</p></div><div id="sec-3" class="subsection"><h3>Filtering variants for quality genotypes</h3><p id="p-9">We read the VCF file from Freebayes into R to extract coverage statistics for bi-allelic single nucleotide polymorphisms (SNPs). As discussed further below, we visualized sequence coverage and set filter thresholds to exclude sites with unusually high coverage, and sites/genotypes with coverage too low to produce an accurate genotype call (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). After an intial filter that removed sites with extremely high coverage, we then filtered sites where the depth of the major and minor alleles were highly imbalanced. We visualized allele balance by plotting the ratio of coverage and setting filters to exclude sites where the ratio highly deviated from a bi-allelic segregating site (1:3, 1:1, or 3:1). Next, we set individual genotypes with very high and low coverage to “missing”, and filtered sites with less than 50 of the 90 possible genotypes. With these markers, we imputed the most likely parental genotypes from the offspring genotype segregation ratios. This step allowed us to filter and correct offspring genotypes according to the observed parent genotypes. For bi-allelic SNPs, there are 4 possible informative genotypic combinations within a cross of two outbred parents (aa x ab, ab x ab, aa x bb, or ab x bb). To call the most likely parental and offspring cross-type for each locus, we used Maximum Likelihood (ML). We assumed the genotypes followed a multinomial distribution with a global heterozygote dropout error rate <em>e</em>. Under this model, the likelihood of an error rate <em>e</em> and parental cross <em>aa x ab</em> given a distribution of genotypes <em>n<sub>aa</sub>,n<sub>ab</sub>,n<sub>bb</sub></em> would be <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2851/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2851/embed/mml-math-1.gif"/></noscript></span></span>. The same logic follows for the other cross-types. We estimated the global ML value of <em>e</em> and the cross-type for all loci simultaneously using the custom R functions in File S3. We determined that roughly 3,000 impossible offspring genotypes should be corrected to heterozygotes. These genotypes were due to allelic dropout, where sequencing had not sampled both alleles of a truly heterozygous individual. An example of this is if parental genotypes are het/homo (aa x ab) cross-type, but one or more offspring were identified as homozygous alternate (bb) at that locus; genotypes of these individuals would have been corrected to ab.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494203/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494203/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Criteria for filtering loci and genotypes. Filter steps that included a vizualiztion for setting thresholds are labled with a letter that corresponds to <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>. Unlabled filter steps did not require a visualiztion</span><div class="sb-div caption-clear"></div></div></div><p id="p-10">After these corrections, we evaluated each locus for segregation distortion from the most likely cross (aa x ab, ab x bb, and ab x ab) with binomial and chi-square tests (for single and double heterozygote cross-types, respectively) and excluded each locus with resulting p-values &lt; 0.05. For double heterozygote cross-type markers (ab x ab), all three genotypes are possible in offspring, which prevents the correction of impossible genotypes and reduces the power for detecting segregation distortion. We excluded these loci from the marker set for this and other reasons discussed below. Finally, sites were excluded when the parent genotypes and ML offspring cross-type did not match. This final set of markers was used for the linkage mapping.</p></div><div id="sec-4" class="subsection"><h3>Assignment to linkage groups and mapping</h3><p id="p-11">To establish the initial linkage groups, we calculated the average pairwise recombination frequencies for all scaffolds with the R ONEMAP package (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Margarido <em>et al.</em> 2007</a>). Each scaffold (maximum size ∼6mb) containing more than one locus of a given cross-type was checked for contiguous regions of unlinked loci to identify potential misassembled scaffolds. Initial agglomerative clustering grouped scaffolds with recombination frequency less than 0.07 with any scaffold in the group. Pairwise recombination frequencies grouped the initial LGs showing evidence of linkage and excluded a small number of individual loci showing strong evidence of linkage to multiple groups (agglomerative clustering).</p><p id="p-12">To further imporve the computational efficieny of the marker ordering step, we downsampled markers and generated a map for each of the parents. The parent-specific maps (called sex-specific maps here) were generated by separating markers by the ML cross-type that we determined for each marker in the previous filtration steps. As indicated above, double heterozygote cross-type markers (ab x ab) cannot be filtered or mapped by the same criteria as the sex-specific cross-type markers. We discarded double heterozygote cross-type markers and retained the two sets of sex-specific cross-type markers (<em>aa x ab</em> and <em>ab x bb</em>). Sex-specific cross-type markers are informative for calculating recombination in each of the parents; recombination rate is calculated from the pairs of markers that are heterozygous in one parent and invariant in the other. We then downsampled these markers to the 5 most complete markers on each scaffold. Markers assigned to each linkage groups were loaded and mapped independently in JOINMAP (maximum likelihood-mapping algorithm and default parameters) (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Stam 1993</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Wu and Ma 2002</a>). ALLMAPs merged the sex-specific maps to form the final consensus map (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Tang <em>et al.</em> 2015</a>). ALLMAPs uses a ‘traveling salesman problem’ solver to compute a consensus scaffold order (rather than marker order) and orientation from the equally weighted sex specific maps based on scaffold ID and SNP position (Figure S1). ALLMAPS then generates updated FASTA, agp, and CHAIN files with the un-mapped genome assembly and the consensus map. This CHAIN file was used with LIFTOVER (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hinrichs 2006</a>) on the NCBI <em>F. heteroclitus</em> gene model set (NCBI gff version 3.0.2) to map genes to their new chromosomal map position for gene synteny analysis. To validate the final order of scaffolds, we evaluated the co-linearity of the sex-specific maps and final concensus map.</p><p id="p-13">To align the consensus map LG labels with a previous <em>F. heteroclitus</em> linkage map, 240 microsatellites and SNP markers from (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Waits <em>et al.</em> 2016</a>) (called ‘microsat map’ hereafter) were blasted to the NCBI reference genome to match the microsat markers to the scaffolds that we mapped. We also blasted the microsat markers to the platyfish genome to find the ‘best-hit’ chromosome and determine if linkage group assignments in both maps were supported by chromosomal homology between platyfish and killifish. We renamed our linkage groups to reflect the microsat map LG names where possible and compared the resulting orders of both maps.</p></div><div id="sec-5" class="subsection"><h3>Orthology and synteny analysis</h3><p id="p-14">Gene positions and karyotypes from each species in the analysis were derived from publicly available genome feature format files (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, File S4). One-to-one orthologous genes were selected for this analysis from an orthology database (File S6) for a number of species generated by a reciprocal blast algorithm (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Li <em>et al.</em> 2003</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Reid <em>et al.</em> 2017</a>). From this list of species, genomes were selected for synteny analysis if the publicly available genome feature format file was mapped to chromosomes. Whole genome ortholog oxford plots were visualized in R. We visualized the order of scaffolds in the male and female generated linkage maps and compared 1-to-1 ortholog synteny between <em>F. heteroclitus</em> and each species with CIRCOS (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Krzywinski <em>et al.</em> 2009</a>). We used these comparisons to designate homologous chromosomes between killifish and other species. For a complete set of figures showing homology, see File S6.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2851/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic relationships among species used in our comparative analyses. Atherinomorphs include Southern platyfish X. maculatus, and Japanese medaka O. latipes. Percomorpharians include green-spotted puffer T. nigroviridis and three-spined stickleback G. aculeatus. Teleostei also includes zebrafish D. rerio. A whole genome duplication (box marked WGD) occurred in the teleost lineage after it diverged from Actinopterygii that includes the spotted gar L. oculatus. Phylogeny is adapted from Betancur-R et al. (2013). The number of 1-to-1 orthologs included in the synteny analysis is denoted in brackets." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-651762063" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phylogenetic relationships among species used in our comparative analyses. Atherinomorphs include Southern platyfish &lt;em&gt;X. maculatus&lt;/em&gt;, and Japanese medaka &lt;em&gt;O. latipes&lt;/em&gt;. Percomorpharians include green-spotted puffer &lt;em&gt;T. nigroviridis&lt;/em&gt; and three-spined stickleback &lt;em&gt;G. aculeatus&lt;/em&gt;. Teleostei also includes zebrafish &lt;em&gt;D. rerio&lt;/em&gt;. A whole genome duplication (box marked WGD) occurred in the teleost lineage after it diverged from Actinopterygii that includes the spotted gar &lt;em&gt;L. oculatus&lt;/em&gt;. Phylogeny is adapted from Betancur-R &lt;em&gt;et al.&lt;/em&gt; (2013). The number of 1-to-1 orthologs included in the synteny analysis is denoted in brackets.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2851/F1.medium.gif" width="440" height="168"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/9/2851/F1.medium.gif" width="440" height="168"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2851/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2851/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494082" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-15" class="first-child">Phylogenetic relationships among species used in our comparative analyses. Atherinomorphs include Southern platyfish <em>X. maculatus</em>, and Japanese medaka <em>O. latipes</em>. Percomorpharians include green-spotted puffer <em>T. nigroviridis</em> and three-spined stickleback <em>G. aculeatus</em>. Teleostei also includes zebrafish <em>D. rerio</em>. A whole genome duplication (box marked WGD) occurred in the teleost lineage after it diverged from Actinopterygii that includes the spotted gar <em>L. oculatus</em>. Phylogeny is adapted from <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Betancur-R <em>et al.</em> (2013)</a>. The number of 1-to-1 orthologs included in the synteny analysis is denoted in brackets.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-16">All data were deposited under NCBI BioProject PRJNA177717 (Fundulus_heteroclitus-3.0.2). Demultiplexed sequence data are submitted at NCBI Sequence Read Archive (SRA, submission: SUB5220906). The map that orders scaffolds in the <em>Fundulus heteroclitus</em> reference genome assembly is deposited at EMBL-EBI Biostudies (accession S-BSST163). Supplemental files and figures are available at FigShare. File S1 is a list of barcodes for sequence libraries. File S2 is the custom perl scripts for demultiplexing sequence libraries. File S4 are custom R scripts for data filtering. File S4 contains mapped genome data and orthology tables. File S6 contains circos images of chromosomal homology for all taxa in this analysis. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.8001572">https://doi.org/10.25387/g3.8001572</a>.</p></div></div><div class="section results-discussion" id="sec-7"><h2 class="">Results &amp; Discussion</h2><p id="p-17">A previous genetic map with 240 markers ordered ∼25% of the genome sequence into 24 linkage groups (LGs) (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Waits <em>et al.</em> 2016</a>). We sought to extend this microsatellite-based map and include more of the genome to improve the killifish genomic toolkit and compare genomic synteny to humans and other fish. We used Restriction Site Associated DNA Sequencing, or ‘RAD-Seq’ (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Miller <em>et al.</em> 2007</a>), markers from a single family including 90 larval F1 offspring and their parents (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baird <em>et al.</em> 2008</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Baxter <em>et al.</em> 2011</a>) to map the genome assembly of <em>Fundulus heteroclitus</em>. Our heuristic approach resulted in a recombination map of 5,685 RAD-Seq markers (mean density of 6.6 markers per Mb) that ordered 1,287 of the largest (216/221 of the scaffolds &gt;N50) of the 10,180 scaffolds. The ordered scaffolds account for ∼84% of the 1.02GBs and gene models in the current reference assembly. In total, this map ordered 854 MB of genomic sequence into chromosomes and provide recombination rate estimates along each chromosome.</p><div id="sec-8" class="subsection"><h3>Strategies for building a good map</h3><p id="p-18">The proportion of quality markers in a given sequence dataset is influenced by many factors, including the quality of the reference genome assembly, the sequencing platform, marker depth and density, and heterozygosity of the sequenced samples (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Song <em>et al.</em> 2016</a>). Therefore, the optimal threshold parameters for marker selection would be expected to vary between datasets. Initially, we designated our depth and segregation distortion filters for our map by surveying the literature for maps with similar taxa and experimental designs. Our sequence data were of relatively low coverage, so applying conservative and inclusive parameters that we surveyed resulted in either a low proportion of genome in the final map, or the appearance of incongruities with other maps. A heuristic, iterative approach guided by a series of data visualizations addressed these issues by optimizing thresholds to minimize the number of genotyping errors (due to artifacts such as repetitive sequence and paralog alignment) and missing data, while retaining the highest number of informative markers. As a result, the correlation with the previous <em>F. heteroclitus</em> map, between our male and female map, and between the <em>F. heteroclitus</em> map and closely related species had improved. We discuss the rationale for each of our steps to guide threshold parameter discovery for others (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2851/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Designation of data filter thresholds guided by visualization of sequence/allele coverage and missing data distributions. Thresholds for our data were determined with simple visualizations of allele and sequence depth for loci and genotypes. Cutoff values at each filter step are indicated on plots with red vertical lines. Filters are for entire loci or individual genotypes (column 1). Each panel (A-F) corresponds to the filter in Table 1. These thresholds are specific to our data and are aimed at reducing genotype errors that are expected in reference genome alignment. Mapping algorithms are sensitive to erroneous genotypes and their removal increased the co-linearity of genomic scaffolds and markers in each of our maps that were estimated from recombination in either parent. We set these thresholds by visualizing: A) The log10 coverage for each base in the reference genome (depth &gt; 75). B) The log2 ratio of read counts for the major and minor alleles for each locus. C) The distribution of total coverage for each locus summed across all individuals. D) The distribution of individual genotype coverage (major + minor allele). E) The log2 ratio of major to minor alleles for individual heterozygote genotypes. F) The distribution of missing data in remaining loci (percent of offspring genotyped)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-651762063" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Designation of data filter thresholds guided by visualization of sequence/allele coverage and missing data distributions. Thresholds for our data were determined with simple visualizations of allele and sequence depth for loci and genotypes. Cutoff values at each filter step are indicated on plots with red vertical lines. Filters are for entire loci or individual genotypes (column 1). Each panel (A-F) corresponds to the filter in Table 1. These thresholds are specific to our data and are aimed at reducing genotype errors that are expected in reference genome alignment. Mapping algorithms are sensitive to erroneous genotypes and their removal increased the co-linearity of genomic scaffolds and markers in each of our maps that were estimated from recombination in either parent. We set these thresholds by visualizing: A) The log10 coverage for each base in the reference genome (depth &gt; 75). B) The log2 ratio of read counts for the major and minor alleles for each locus. C) The distribution of total coverage for each locus summed across all individuals. D) The distribution of individual genotype coverage (major + minor allele). E) The log2 ratio of major to minor alleles for individual heterozygote genotypes. F) The distribution of missing data in remaining loci (percent of offspring genotyped).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2851/F2.medium.gif" width="440" height="315"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/9/2851/F2.medium.gif" width="440" height="315"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2851/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2851/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494115" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-19" class="first-child">Designation of data filter thresholds guided by visualization of sequence/allele coverage and missing data distributions. Thresholds for our data were determined with simple visualizations of allele and sequence depth for loci and genotypes. Cutoff values at each filter step are indicated on plots with red vertical lines. Filters are for entire loci or individual genotypes (column 1). Each panel (A-F) corresponds to the filter in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>. These thresholds are specific to our data and are aimed at reducing genotype errors that are expected in reference genome alignment. Mapping algorithms are sensitive to erroneous genotypes and their removal increased the co-linearity of genomic scaffolds and markers in each of our maps that were estimated from recombination in either parent. We set these thresholds by visualizing: A) The log10 coverage for each base in the reference genome (depth &gt; 75). B) The log2 ratio of read counts for the major and minor alleles for each locus. C) The distribution of total coverage for each locus summed across all individuals. D) The distribution of individual genotype coverage (major + minor allele). E) The log2 ratio of major to minor alleles for individual heterozygote genotypes. F) The distribution of missing data in remaining loci (percent of offspring genotyped).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Marker selection with basic visualizations of the data</h3><p id="p-20">Parameter selection can be cumbersome with a large number of densely spaced markers. We found that filtering the alignment before genotyping reduced the dataset to a more manageable size for faster re-analysis and data visualization. BAMTOOLS can quickly generate a two-column file of depth and position for easy loading and visualization in R. The initial per base alignment depth ranged from 1 to well over 100,000 reads (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2A</a>). Visualizing the depth allowed us to set an upper threshold for coverage just beyond the accumulation of deeply covered positions in the genome, which is consistent with intended targeting of sequencing at RAD tags (centered near log10 = 3.0, or 1000). We suspected that the very widely distributed and more deeply covered sequence was the result of highly-repetitive sequence alignments that are found throughout the genome and are not reliable markers for recombination mapping (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Treangen and Salzberg 2012</a>) (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2A</a>, red line set at log10(depth)=3.2). The distinct accumulation of low coverage positions is a combination of paired-end read alignments and probably some non-RAD tag associated reads (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2A</a>, below red line at log10(depth)=2). Our lower threshold was more inclusive than would be necessary to discover an informative mapping marker (100x across 92 individuals). While this filter step is redundant to a later filtering step aimed at suspicious genotypes and markers, the primary purpose here was to reduce the dataset size. We found that pre-filtering the very high and very low coverage sequence sufficiently reduced the time and computational load of later genotype calling and visualization. This step would be particularly useful for large datasets or genotype callers that require greater computational resources.</p><p id="p-21">After calling genotypes, we filtered the data based on visualizations of allele coverage, balance, and depth distributions of loci and called genotypes. The first filter removed markers with a highly skewed ratio of reference to alternate allele reads (minor/major allele coverage &lt; 0.167). Even though short read sequences have a low sequencing error rate, highly skewed ratios could indicate a small number of sequencing errors at an invariant site. These sites are more likely to be additional alignment errors where multiple loci have mapped to a single locus in the reference genome but had lower coverage than the pre-genotype filter threshold. The alignment error rate is expected to vary between genome assemblies and taxa; alignments to genomes with expanding gene families or genomes that have more recently experienced a whole genome duplication (such as teleosts compared to gar, <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>) would be expected to have an elevated rate of mis-aligned and ambiguously mapped reads. This threshold applied to our data (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2B</a> red line at ∼6:1 and 1:5, ref/alt) eliminated 18,000 of almost 90,000 loci (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). The asymmetry between the peak heights of heterozygotes in this distribution (left and right peaks in <a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2B</a>) is due to the tendency for the alternate allele (relative to the reference genome) to be at lower frequency in the parents (ex: parents are more likely to be ref/ref X ref/alt than ref/alt X alt/alt). The increase in the number of loci in the positive tail (rightmost peak) is due to alignment bias toward reads that contain more reference alleles (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Brandt <em>et al.</em> 2015</a>). We chose a slightly offset threshold in the positive direction (log2(ref/alt) =2.6) to avoid removing a large number of quality markers that would be removed with the symmetrical threshold of log2(ref/alt) =2.3.</p><p id="p-22">The total locus depth distribution (now for genotypes) guided the choice of thresholds to capture what we suspected as the range of coverage for RAD tags (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2C</a>). Loci with coverage less than 320 and higher than 1,400 reads were eliminated from the analysis, removing 32,000 and 116 loci respectively. While this may appear aggressive, loci with 320 reads split between half of the offspring would, at best, provide ∼7 reads for each genotype (3.5/allele). We later determined the lower threshold of 7 reads by comparing the proportion of parent genotypes that matched with the ML parent genotypes as determined from the offspring genotype frequencies (<a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2D</a>; red line at 7). Below 7 reads, the proportion of markers matching expectations fell sharply. We also used the ML genotypes to remove markers that differed from Mendelian expectations (segregation distortion). Genetic mapping tools commonly provide similar filters for markers with genotype frequencies that deviate from segregation expectations in a bi-parental cross. Real segregation distortion is common in mapping populations (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Xu 2008</a>). Our ML approach likely included some distortion and used an allelic-dropout error term (see Methods) in an effort to retain as many markers as possible. The R code for inferring the ML parental genotypes from offspring is relatively straightforward and is included in the supplemental File S4. The upper threshold at 40 reads per individual genotype excluded a very small number of highly covered genotypes (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2D</a>; red line at 40).</p><p id="p-23">Our last set of visualization-guided filters removed individual genotypes that had suspiciously high or skewed coverage. We observed a small number of genotypes with suspiciously high allelic bias (10:1 major/minor allele) among the genotypes that remained after prior filtering. These genotypes (∼1%) were also changed to missing to reduce the possibility of retaining sequencing errors that changed the genotype call (<a id="xref-fig-2-12" class="xref-fig" href="#F2">Figure 2E</a>). Removal of genotypes in these two steps increased the proportion of missing data to ∼45% (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>), indicating that many of the remaining loci had very low coverage (likely falling in the paired-end of reads). As discussed below, markers with a higher proportion of missing data tend to spuriously link to multiple linkage groups. High proportions of missing data can lead to errors in marker order and tend to deflate mapping distance (Hackett and Broadfoot 2003). We filtered out markers with more than 50/90 missing genotypes (&gt;55.55% missing), which resulted in 24,227 higher coverage loci (<a id="xref-fig-2-13" class="xref-fig" href="#F2">Figure 2E</a>). We used parental genotypes to determine that 2,998 individual offspring genotypes were incorrectly called homozygous alternate, presumably due to allelic dropout. Correcting genotypes affected by dropout rescued just over 10,000 loci from the following segregation distortion filter (<em>P</em> &lt; 0.05, exact binomial and chi-square tests). We corrected dropout genotypes to heterozygotes and re-filtered the data for markers where the parent and the ML determined parent genotypes mismatched.This mappable set of 19,928 markers from the heuristic filtering step aligned to 1,287 assembly scaffolds that accounted for ∼85% of the killifish genome sequence (854 Mb of 1,021.88 Mb) and gene models (21,435 of the 25,601 NCBI gene models).</p></div><div id="sec-10" class="subsection"><h3>Assignment to linkage groups</h3><p id="p-24">Markers are typically assigned to linkage groups by varying the recombination frequency and LOD threshold parameters <em>ad hoc</em>, and manually assigning groups of markers with one another. During the assignment step, we found that markers with a greater proportion of missing data were susceptible to spurious linkage with markers on different linkage groups. That is, clustering markers without considering scaffold ID information tended to link scaffolds that otherwise appear unlinked when considering all markers on a scaffold. We cannot identify these markers <em>a priori</em>, and once anchored in their correct LG, spuriously linked markers can still contain information on the order and orientation of scaffolds within LGs, so it may be beneficial to retain them. To keep these markers, we took an iterative approach that clustered entire scaffolds of markers together. We calculated the pairwise recombination fraction for markers to perform the agglomerative grouping and validated our strategy of assigning scaffolds to linkage groups by visualizing the matrix of recombination frequencies for markers along each scaffold (Figure S2).</p><p id="p-25">Our initial filtering focused on removing low quality markers. Even though the remaining markers are of higher quality, many are redundant to one another and do not provide additional mapping information. After scaffolds were clustered into linkage groups, we added another filtering step to reduce these redundant markers to improve mapping computational efficiency. Before assigning markers it linkage groups, the final marker set was pruned to the 5 RAD loci per scaffold. Markers that are in close proximity to one another, such as those in a single RAD-Tag, are extremely unlikely to contain a recombination event. As such, these nearby markers do not add to the resolution of the final map but do significantly increase the computation time for mapping. Rather than drop non-recombinant or random markers, we thinned the dataset to five markers per parental cross-type per scaffold to reduce the mapping computation time without sacrificing resolution. As a final step to conservatively filter out suspicious genotypes, we removed a small number of loci that appeared as short double crossover events within a scaffold (visualized in Joinmap) in one or more individuals, bringing the total number of markers to 5,739 (2,992 and 2,855 male and female specific markers, respectively: <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494187/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494187/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary statistics for the male, female, and consensus map of <em>Fundulus heteroclitus</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Ordering within linkage groups and the final Map</h3><p id="p-26">Once markers are assigned to linkage groups, the outcrossing mapping algorithm typically integrates the two sex specific maps into a single consensus map (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Wu and Ma 2002</a>). We chose a different approach. The outcrossing algorithm relies on the parental double heterozygote markers as landmarks for a map that integrates the sex specific maps. We found that this approach resulted in a final map where the scaffolds were more commonly broken within the group (mapped to different positions along the linkage group rather than clustered), which appeared to increase the uncertainty about the consensus position for each scaffold. These issues could be related to the reduced power to detect segregation distortion in double heterozygote markers, and to the inability to correct allelic dropout and impossible genotypes. To explore this issue further, we re-mapped two sets of markers for each linkage group; we separated markers that were informative for recombination in each parent to generate the sex specific maps independently (see methods for clarification). We found that the sex specific maps were congruent with one another and were far less likely to separate markers on each scaffold. We decided to merge these maps with ALLMAPs (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Tang <em>et al.</em> 2015</a>, Table 2). The ALLMAPs ordering algorithm improved the congruence between the individual maps and merged map by anchoring whole scaffolds from the marker data rather than treating each marker as an independent observation. A drawback to this approach is that scaffolds that have evidence for mis-assembly (chimeric scaffolds that were improperly joined during the primary genome assembly phase) are anchored to a common position. Chimeric scaffolds could be addressed before the mapping stage by evaluating recombination frequencies within scaffolds and splitting the groups of markers that have a discordant ratio between physical distance and recombination rate. ALLMAPs includes a resource for merging maps with scaffolds that were split because they appear linked to multiple linkage groups. We chose to keep all scaffolds intact because low-coverage, correctly-placed scaffolds are difficult to distinguish from chimeric scaffolds. The final map from ALLMAPs indicates that the killifish genetic map had an average of 81 cM and 35 MB per linkage group (similar to <em>D. rerio</em>) for 24 linkage groups (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Woods <em>et al.</em> 2000</a>). We estimate an average, genome wide recombination rate at 2.34 cM/MB (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bradley <em>et al.</em> 2011</a>).</p></div><div id="sec-12" class="subsection"><h3>Map comparisons</h3><p id="p-27">To further examine the quality of the map, we compared the independent male and female genetic maps, and a previously generated microsatellite map (<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Waits <em>et al.</em> 2016</a>). After applying the agglomerative clustering and mapping algorithm, the order of scaffolds appeared highly congruent between sex specific maps (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). However, we did observe differences in the number and order of some scaffolds in two linkage groups (LGs). For example, the marker density and the number of scaffolds with informative markers in linkage group 21 (LG21) was lower in the male map (see asterisk in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). This linkage group contains a region of very low recombination, despite a similar total mapping distance between the homologous chromosomes (88 cM in the Male, 86 cM in the Female, Figure S3A). However, the linkage group appears corrected in the consensus map. After merging the maps with ALLMAPs, the consensus map is congruent with the female map that contained a majority of LG21’s markers. When ortholog order is compared with closely related Atherinomorph species (as discussed below), the consensus map for LG21 showed highly conserved synteny with the homologous platyfish chromosome (XM18), with the exception of two transpositions from either chromosomal arm to the other (Figure S3B). The translocation on the lower arm of LG18 is likely real because the region contains multiple scaffolds that are syntenic in both species and medaka. The homologous chromosome LG13 in medaka (Figure S3B, right) does not share this rearrangement, indicating that this inversion has occurred in the platyfish lineage after divergence from killifish.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2851/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Karyotype and comparison of scaffold ordering and marker density between independent (sex-specific) maps plotted by physical position of scaffolds. Male- (ring of 24 multi-colored LG segments in plot marked by *) and female-specific scaffold orderings (inner ring of 24 multi-colored LG segments). Each linkage group scaffold order is plotted by concatenating the physical size of scaffolds (smaller colored segments in each of the 24 LGs) to emphasize scaffold order rather than the centi-morgan position (cM). The orange bar plots along the scaffolds in each linkage group (left and right of the rings containing scaffold orders) indicate the number of markers (between 1 and 5, fully orange bar is 5 markers used) used for ordering and orienting each scaffold. The outermost track for each linkage group (colored by chromosome) is the position of RAD-Seq markers along each scaffold. Numbers on each group indicate mapping length of each LG in cM. Scaffold orders appear highly congruent (two rows of inner bars colored by scaffold ID) and were independently mapped using pairs of markers that were heterozygous in one sex and homozygous in the other (sex-specific markers). *The differences between the number of scaffolds in the male and female map at LG21 could suggest a heteromorphic chromosome. However, LG21 has a similar total centimorgan length between each map (88 and 86 cM) and share common scaffolds at both ends of the LG, which is typical of homomorphic autosomes (see Figure S3A)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-651762063" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Karyotype and comparison of scaffold ordering and marker density between independent (sex-specific) maps plotted by physical position of scaffolds. Male- (ring of 24 multi-colored LG segments in plot marked by *) and female-specific scaffold orderings (inner ring of 24 multi-colored LG segments). Each linkage group scaffold order is plotted by concatenating the physical size of scaffolds (smaller colored segments in each of the 24 LGs) to emphasize scaffold order rather than the centi-morgan position (cM). The orange bar plots along the scaffolds in each linkage group (left and right of the rings containing scaffold orders) indicate the number of markers (between 1 and 5, fully orange bar is 5 markers used) used for ordering and orienting each scaffold. The outermost track for each linkage group (colored by chromosome) is the position of RAD-Seq markers along each scaffold. Numbers on each group indicate mapping length of each LG in cM. Scaffold orders appear highly congruent (two rows of inner bars colored by scaffold ID) and were independently mapped using pairs of markers that were heterozygous in one sex and homozygous in the other (sex-specific markers). *The differences between the number of scaffolds in the male and female map at LG21 could suggest a heteromorphic chromosome. However, LG21 has a similar total centimorgan length between each map (88 and 86 cM) and share common scaffolds at both ends of the LG, which is typical of homomorphic autosomes (see Figure S3A).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2851/F3.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/9/2851/F3.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2851/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2851/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494104" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-28" class="first-child">Karyotype and comparison of scaffold ordering and marker density between independent (sex-specific) maps plotted by physical position of scaffolds. Male- (ring of 24 multi-colored LG segments in plot marked by *) and female-specific scaffold orderings (inner ring of 24 multi-colored LG segments). Each linkage group scaffold order is plotted by concatenating the physical size of scaffolds (smaller colored segments in each of the 24 LGs) to emphasize scaffold order rather than the centi-morgan position (cM). The orange bar plots along the scaffolds in each linkage group (left and right of the rings containing scaffold orders) indicate the number of markers (between 1 and 5, fully orange bar is 5 markers used) used for ordering and orienting each scaffold. The outermost track for each linkage group (colored by chromosome) is the position of RAD-Seq markers along each scaffold. Numbers on each group indicate mapping length of each LG in cM. Scaffold orders appear highly congruent (two rows of inner bars colored by scaffold ID) and were independently mapped using pairs of markers that were heterozygous in one sex and homozygous in the other (sex-specific markers). *The differences between the number of scaffolds in the male and female map at LG21 could suggest a heteromorphic chromosome. However, LG21 has a similar total centimorgan length between each map (88 and 86 cM) and share common scaffolds at both ends of the LG, which is typical of homomorphic autosomes (see Figure S3A).</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">Sex-specific recombination rates are common in fish and a dramatic reduction in recombination rate accompanied by differences in homologous chromosome structure could indicate a heteromorphic sex determination system (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Singer <em>et al.</em> 2002</a>). Both LG21 and LG5 differed in scaffold order between the sexes and contained some regions of low combination in the male map (Figures S3A, S1B respectively). Whole-genome re-sequencing of hundreds of individual <em>F. heteroclitus</em> (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Reid <em>et al.</em> 2016</a>) revealed markers that significantly associated with sex on scaffolds that mapped to LG5. Teleost fishes possess a variety of sex determination systems (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Schartl 2004</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Mank <em>et al.</em> 2006</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Sandra and Norma 2010</a>) and further study is required to identify candidate mechanisms for sex-determination in <em>F. heteroclitus</em>.</p><p id="p-30">Previously, a genetic map was constructed for <em>F. heteroclitus</em> using 229 microsatellite and SNP markers which ordered approximately 25% of the genome sequence (<a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">Waits <em>et al.</em> 2016</a>). Our RAD-seq map extends the proportion of the reference genome included in the map to 83.7%. We compared the consistency of linkage group assignment between the two maps. Most microsat map loci fell on scaffolds with good RAD-Seq coverage in the male and female maps. Twenty-six of the 240 microsat map markers aligned to one of the 216 scaffolds that were missing RAD loci, and these scaffolds tended to be smaller than 0.3 MB, but did include a 1.1, 3.8, and 5.0 MB scaffold. Half of all RAD-seq LGs corresponded exclusively to only one LG from the microsat map. The LGs that contained markers that were differently assigned between the two maps contained a majority of markers that we assigned to the same LG (homolog), but also grouped 1-2 markers that were from different microsat linkage groups, indicating errors in one of the maps. Our data indicate that 19/216 markers from the microsat map may have been incorrectly assigned to linkage groups, and/or these scaffolds contain assembly errors. Scaffolds that were reassigned in the RAD-Seq map are highlighted in File S5. Our data indicate that the small LG18 in the microsat map is actually part of the larger LG6. What we designate as LG18 in the RAD-seq map does not have a homologous LG in the microsat map, indicating that the microsat map failed to sample markers on this chromosome. The 3 markers making up microsat map LG23 were re-assigned to the RAD-seq map LG17 (supported by 26 markers). The microsat map included markers for LG23, but they were mis-assigned to two other linkage groups. Both of these corrections are supported by more markers in the RAD-seq map and highly conserved synteny of the RAD-seq map with other fish maps (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figures 4</a>, <a id="xref-fig-5-1" class="xref-fig" href="#F5">5</a>; File S5). Other than LGs 18 and 23, all LGs have a clear homolog between the RAD-Seq and microsat maps.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2851/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparisons of conserved synteny within a phylogenetic framework. Phylogenetic relationship of each species (reference genomes that were mapped to chromosomes) to killifish, indicated at center, increases in distance clockwise from Fundulus heteroclitus (top right). Killifish LGs (green segments at right in each Circos plot) are ordered (1-24) and compared to mapped chromosomes in each species. Colored lines (colored by killifish LG number) in each plot connect the positions of one-to-one ortholog genes between the species. Chromosomes around each Circos plot are ordered by the largest number of orthologs relative to each killifish LG. Highly conserved synteny and orthology (nearly all lines on a chromosome connect to a single killifish LG) was used to identify the homologous chromosomes in percomophs. For larger Circos plots that identify homologs in each species, see supplemental File S6. Fish images were reproduced under the Creative Commons License (File S9)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-651762063" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparisons of conserved synteny within a phylogenetic framework. Phylogenetic relationship of each species (reference genomes that were mapped to chromosomes) to killifish, indicated at center, increases in distance clockwise from &lt;em&gt;Fundulus heteroclitus (top right)&lt;/em&gt;. Killifish LGs (green segments at right in each Circos plot) are ordered (1-24) and compared to mapped chromosomes in each species. Colored lines (colored by killifish LG number) in each plot connect the positions of one-to-one ortholog genes between the species. Chromosomes around each Circos plot are ordered by the largest number of orthologs relative to each killifish LG. Highly conserved synteny and orthology (nearly all lines on a chromosome connect to a single killifish LG) was used to identify the homologous chromosomes in percomophs. For larger Circos plots that identify homologs in each species, see supplemental File S6. Fish images were reproduced under the Creative Commons License (File S9).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2851/F4.medium.gif" width="440" height="369"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/9/2851/F4.medium.gif" width="440" height="369"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2851/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2851/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494067" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-31" class="first-child">Comparisons of conserved synteny within a phylogenetic framework. Phylogenetic relationship of each species (reference genomes that were mapped to chromosomes) to killifish, indicated at center, increases in distance clockwise from <em>Fundulus heteroclitus (top right)</em>. Killifish LGs (green segments at right in each Circos plot) are ordered (1-24) and compared to mapped chromosomes in each species. Colored lines (colored by killifish LG number) in each plot connect the positions of one-to-one ortholog genes between the species. Chromosomes around each Circos plot are ordered by the largest number of orthologs relative to each killifish LG. Highly conserved synteny and orthology (nearly all lines on a chromosome connect to a single killifish LG) was used to identify the homologous chromosomes in percomophs. For larger Circos plots that identify homologs in each species, see supplemental File S6. Fish images were reproduced under the Creative Commons License (File S9).</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2851/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Synteny comparisons with platyfish and medaka. A) Oxford plot for one-to-one ortholog genes (colored dots) between killifish and closely related platyfish. Genes are colored by killifish scaffold identity and ordered by both linkage group and physical position in each reference genome (Killifish along Y-axis, platyfish along X-axis). Homology is conserved at the LG (or chromosome) level if few genes are translocated/mis-assigned (lone points that have mapped to a different chromosome) and if physical positions of orthologs correlate between the homologous chromosomes (tendency for genes to fall along the diagonal in each LG). B) Example killifish LGs and the correlation of physical positions of ortholog in each percomorph. Orthologous genes are colored by killifish scaffold ID (dots, non-homologous LGs left out). Scaffolds that we mapped are in the same relative position in the comparison species if orthologs of the same color (scaffold ID) fall on the diagonal. LGs that contain the same order of scaffolds but contained multiple &#x2018;micro-inversions&#x2019; within the same killifish scaffold ID (example in B, LG15 compared to the homologous platyfish chromosome) were considered as possible misoriented scaffolds. C) Killifish chromosome 9 (green) and homologous platyfish chromosome (16, red) and medaka chromosome (8, gray) with orthologous gene physical positions connected by lines that are colored by either platyfish (left Circos plot) or killifish scaffold ID (middle and right Circos plot). Possible mis-orientations were further identified by inversions that appear to be perfectly delimited by scaffold identity in one comparison (ex: large blue platyfish scaffold), but not in the more distantly related comparison (ex the same region of the LG compared to medaka positions) and are not delimited by a single scaffold ID." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-651762063" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Synteny comparisons with platyfish and medaka. A) Oxford plot for one-to-one ortholog genes (colored dots) between killifish and closely related platyfish. Genes are colored by killifish scaffold identity and ordered by both linkage group and physical position in each reference genome (Killifish along Y-axis, platyfish along X-axis). Homology is conserved at the LG (or chromosome) level if few genes are translocated/mis-assigned (lone points that have mapped to a different chromosome) and if physical positions of orthologs correlate between the homologous chromosomes (tendency for genes to fall along the diagonal in each LG). B) Example killifish LGs and the correlation of physical positions of ortholog in each percomorph. Orthologous genes are colored by killifish scaffold ID (dots, non-homologous LGs left out). Scaffolds that we mapped are in the same relative position in the comparison species if orthologs of the same color (scaffold ID) fall on the diagonal. LGs that contain the same order of scaffolds but contained multiple &#x2018;micro-inversions&#x2019; within the same killifish scaffold ID (example in B, LG15 compared to the homologous platyfish chromosome) were considered as possible misoriented scaffolds. C) Killifish chromosome 9 (green) and homologous platyfish chromosome (16, red) and medaka chromosome (8, gray) with orthologous gene physical positions connected by lines that are colored by either platyfish (left Circos plot) or killifish scaffold ID (middle and right Circos plot). Possible mis-orientations were further identified by inversions that appear to be perfectly delimited by scaffold identity in one comparison (ex: large blue platyfish scaffold), but not in the more distantly related comparison (ex the same region of the LG compared to medaka positions) and are not delimited by a single scaffold ID.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2851/F5.medium.gif" width="440" height="394"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/9/2851/F5.medium.gif" width="440" height="394"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2851/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2851/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494158" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-32" class="first-child">Synteny comparisons with platyfish and medaka. A) Oxford plot for one-to-one ortholog genes (colored dots) between killifish and closely related platyfish. Genes are colored by killifish scaffold identity and ordered by both linkage group and physical position in each reference genome (Killifish along Y-axis, platyfish along X-axis). Homology is conserved at the LG (or chromosome) level if few genes are translocated/mis-assigned (lone points that have mapped to a different chromosome) and if physical positions of orthologs correlate between the homologous chromosomes (tendency for genes to fall along the diagonal in each LG). B) Example killifish LGs and the correlation of physical positions of ortholog in each percomorph. Orthologous genes are colored by killifish scaffold ID (dots, non-homologous LGs left out). Scaffolds that we mapped are in the same relative position in the comparison species if orthologs of the same color (scaffold ID) fall on the diagonal. LGs that contain the same order of scaffolds but contained multiple ‘micro-inversions’ within the same killifish scaffold ID (example in B, LG15 compared to the homologous platyfish chromosome) were considered as possible misoriented scaffolds. C) Killifish chromosome 9 (green) and homologous platyfish chromosome (16, red) and medaka chromosome (8, gray) with orthologous gene physical positions connected by lines that are colored by either platyfish (left Circos plot) or killifish scaffold ID (middle and right Circos plot). Possible mis-orientations were further identified by inversions that appear to be perfectly delimited by scaffold identity in one comparison (ex: large blue platyfish scaffold), but not in the more distantly related comparison (ex the same region of the LG compared to medaka positions) and are not delimited by a single scaffold ID.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Orthology and synteny analysis</h3><p id="p-33">To further examine the quality of the RAD-seq map, we evaluated the shared gene order (synteny) of 1-to-1 orthologs between <em>F. heteroclitus</em>, six fish species, and human. We visualized synteny between these genomes within a phylogenetic framework for additional evidence of the order and orientation of scaffolds in the genome assemblies, and to examine patterns of karyotype evolution. Other sequenced Atherinomorphs have 24 linkage groups with clear chromosomal (1-to-1) homology (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fisher and Rachlin 1972</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Naruse <em>et al.</em> 2000</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Walter <em>et al.</em> 2004</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Amores <em>et al.</em> 2014</a>). Our map also grouped killifish scaffolds into 24 linkage groups that are each homologous to an Atherinomorph chromosome, which indicates that our agglomerative clustering approach reliably assigned scaffolds to chromosomes. Similar to the platyfish-to-medaka map comparison (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Amores <em>et al.</em> 2014</a>), we detected very few potential inter-chromosomal rearrangements per linkage group between species in this clade, with the number of putative translocations ranging from zero (LG12 orthologs appear exclusively on platyfish 10 or as unmapped in platyfish, with only one apparent transposition in medaka) to 10 with an average of 3.97 per chromosome (Figure S4). Among Atherinomorphs, an average of 2.2% (143 of nearly 7,000 genes) of all putative one-to-one orthologs are designated as orthologous to a gene on a chromosome other than the homologous chromosome. The translocations that do appear tend to include a single gene model (singletons) rather than major structural rearrangements that carry multiple syntenic genes. Singletons are either genome assembly errors, errors in assigning orthology (older paralogs and true ortholog loss), or small real translocations in each branch. At least 10/265 singleton genes are found to be translocated in all teleosts relative to killifish, which indicates killifish specific error in assigning orthology or translocations. Interestingly, singletons appear at an even lower rate in the more distantly related Percomorpharians (∼3.3 singletons per chromosome) but are elevated in zebrafish (at least 5.16/chromosome), gar and human (Figure S5-6). Transposable element activity and repetitive sequence is related to single gene transpositions (Montgomery <em>et al.</em> 1991). Zebrafish transposable element diversity and abundance is the highest among the teleosts in our analysis (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Sotero-Caio <em>et al.</em> 2017</a>). This may indicate that singletons in more distantly related taxa could be attributed to a combination of errors and real translocations.</p><p id="p-34">We also observed previously described chromosomal rearrangements that are responsible for the differences in karyotype number among some teleost clades (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Jaillon <em>et al.</em> 2004</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Roesti <em>et al.</em> 2013</a>). The karyotype number and genome structure of ancestral Teleostei is likely 23-24 chromosomes and is highly conserved in medaka (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kasahara <em>et al.</em> 2007</a>). The reduction of chromosome number in Percomorpharians (stickleback and pufferfish, n = 21) occurred after the divergence from the Atherinomorph lineage (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Kai <em>et al.</em>, 2011</a>). Our map recovered the independent chromosomal fusions in each lineage and the shared synteny with their ancestral homologous chromosome segments (Figure S7A). Relative to Percomopha, the zebrafish lineage (n = 25) diverged earlier in the Teleost radiation. Relative to killifish and the ancestral 24 teleost chromosomes, the zebrafish genome experienced an increase in independent inter- and intra-chromosomal rearrangements (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Kasahara <em>et al.</em> 2007</a>, Figure S5 A-E). However, as with previous comparisons (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Amores <em>et al.</em> 2014</a>), the accumulation of these complex rearrangements and an increase in singleton translocations makes it difficult to infer the events that shaped the zebrafish karyotype compared to <em>F. heteroclitus</em>.</p><p id="p-35">At the broader phylogenetic scale, we observed expected patterns of highly conserved chromosome homology and synteny between closely related species that deteriorates with phylogenetic distance (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Much structural genome variation has evolved over the more than 300 million years since sharing common ancestry with spotted gar. However, we recovered the characteristics of the whole genome duplication and reciprocal gene loss that occurred in the teleost linage after the divergence from the lineage leading to spotted gar (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Jaillon <em>et al.</em> 2004</a>; <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Amores <em>et al.</em> 2014</a>). For example, dot-plots comparing killifish to human and killifish to spotted gar chromosomes shows that almost all 1-to-1 orthologs for a given region in these taxa are distributed primarily on two <em>F. heteroclitus</em> chromosomes (<em>e.g.</em>, gar LG8 and human LG14, Figure S6). Circos plots of the gar and duplicated killifish chromosomes (ohnologs) show that the 2:1 homology often spans entire chromosomes and does not indicate fission (Figure S7B, chromosome split with syntenic segments of homology to different chromosomes). After duplication, ohnologs tend to evolve such that one of the two duplicate genes are lost (reciprocal gene loss) leading to the overlapping pattern of orthologs that we observe (Figure S7B; (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Woods <em>et al.</em> 2005</a>).</p><p id="p-36">Genome-wide visualizations indicate that orthologous genes appeared ordered the same between platyfish and killifish (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>). However, closer inspection at the chromosome level revealed very short regional inversions (termed micro-inversions here, <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>). In fact, some chromosomes appear to be dominated by micro-inversions (<em>e.g.</em>, LG15, <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a>), while others are almost completely syntenic with killifish (no micro-inversions, <em>e.g.</em>, LG4, <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5B</a>). It is possible that these micro-inversions are mistakes in scaffold orientation within the linear ordering of a linkage group. If micro-inversions are bounded by the ends of a scaffold, then this is suspicious and unlikely to be a real inversion, and more likely to be a mistake in scaffold orientation within the map. To examine this, we generated synteny plots where ortholog connections are colored by scaffold identity (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5C</a>). We discovered that inversions were often delimited by scaffold boundaries in either the platyfish or killifish map, suggesting that they are likely mis-oriented scaffolds. To further test whether micro-inversions are mis-orientations, we compared gene order in a third species. We predicted, for example, that mis-orientations in platyfish would appear as 1) a short region negatively ordered to killifish, but not mis-ordered between killifish and medaka, and 2) delimited by platyfish scaffold ID. Scaffolds that meet this criterion are found in both platyfish and killifish. The scaffolds highlighted in the black box in <a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5C</a> are examples of likely scaffold mis-orientations in platyfish. In some cases, synteny appears much more strongly conserved between more distantly related taxa and killifish (<em>e.g.</em>, LGs 11, 13, 15, <a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5B</a>) than between killifish and platyfish (LG14, <a id="xref-fig-5-9" class="xref-fig" href="#F5">Figure 5B</a>). This indicates either an increase in rearrangements in the platyfish lineage or an elevated rate of scaffold mis-orientation in these regions. We consider the latter explanation more likely. Our comparison suggests mapping data tend to order scaffolds correctly within a chromosome, but that scaffold mis-orientations within that arrangement are common. These observations suggest that caution should be observed when interpreting fine scale structural variation between genomes assembled with recombination mapping alone. Mis-orientations would bias analyses toward higher levels of intra-chromosomal inversions and rearrangements. For chromosomes that have a higher rate of rearrangements such as the putative sex chromosome LG5 (<a id="xref-fig-5-10" class="xref-fig" href="#F5">Figure 5B</a>), it would be impossible to distinguish micro-inversions from intra-chromosomal rearrangements. Longer scaffolds, for example generated from long-read sequencing, are likely less susceptible to mis-orientation with recombination mapping.</p><p id="p-37">A quality mapped reference genome enables comparison of genome structure between species. A quality map is also an important resource in evolutionary and ecological genomics studies, such as those that aim to establish genotype-phenotype relationships and the genetic basis of local adaptation via signatures of selection. Genotype-phenotype association mapping (<em>e.g.</em>, GWAS, QTL mapping, admixture mapping) reveals the identity and number of genomic regions that associate with phenotypic variation among individuals. Adaptation genomics seeks to identify regions of the genome showing signatures of natural selection within and between populations, where regions can be large because of linked selection (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hill and Robertson 1966</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Sawyer <em>et al.</em> 2007</a>). A good reference genome, where scaffolds are correctly ordered and oriented, helps to distinguish the boundaries of genomic regions that are diverging by natural selection. In particular, signatures of very recent or strong selection may be manifest across wide regions of a chromosome that may span multiple scaffolds. For example, some signatures of strong and recent selection during adaptation to extreme pollution in killifish were greater than 1 Mb (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Reid <em>et al.</em> 2016</a>). In a highly fragmented or mis-ordered assembly, this footprint of selection could be expected to span many scaffolds, inflating the estimated number of genomic regions that were targets of selection, and potentially altering conclusions about the polygenic nature of adaptive phenotypes.</p></div></div><div class="section conclusions" id="sec-14"><h2 class="">Conclusions</h2><p id="p-38">Our map improves the reference genome of killifish <em>Fundulus heteroclitus</em> to further its utility for studies in ecological and evolutionary genetics. We resolved a large portion (∼84%) of the reference genome into 24 linkage groups (chromosomes). We describe the rationale for each of our empirical data filtering steps to provide explicit guidance for the selection of study-specific empirical quality filtering thresholds that may be useful for others. This careful filtering of read and genotype data dramatically improved our resulting maps. We evaluated the quality of our map by comparing the synteny of 1-to-1 orthologous genes to other mapped genomes within a phylogenetic framework. We recovered signals of whole genome duplications, fusion/fission and rearrangements that have been described in other comparative analyses, confirming the accuracy of our map. We also found that scaffold misorientations within linkage groups are common and suggest caution when interpreting fine scale variation between closely related species due to these linkage mapping artifacts. This well-assembled genetic map will enable the next generation of quantitative genetic and population genetic studies in <em>F. heteroclitus</em>.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-39">This work was supported by the National Institutes of Environmental Health Sciences [1R01ES021934-01 to A.W.]; and the National Science Foundation [OCE-1314567, DEB-1265282 to A.W.]. This research was also supported in part by the Postdoctoral Research Program at the U.S. Environmental Protection (US EPA) Office of Research and Development administered by the Oak Ridge Institute for Science and Education (ORISE) through Interagency Agreement No. DW92429801 between the U.S. Department of Energy and the US EPA. The views expressed in this article are those of the authors and do not necessarily represent the views or policies of the US EPA.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-40">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.8001572">https://doi.org/10.25387/g3.8001572</a>.</p></li><li class="fn" id="fn-3"><p id="p-41"><em>Communicating editor: T. Matise</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-04-15"><span class="received-label">Received </span>April 15, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-06-28"><span class="accepted-label">Accepted </span>June 28, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Miller <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abraham</span>,  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Species Profiles. Life Histories and Environmental Requirements of Coastal Fishes and Invertebrates (Mid-Atlantic). MUMMICHOG AND STRIPED KILLIFISH.</span> <abbr class="cit-jnl-abbrev">Biol. Reprod.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BReprod.%26rft.volume%253D82%26rft.spage%253D33%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.2" data-doi="10.1534/genetics.114.164293"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Catchen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Nanda</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Warren</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Walter</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A RAD-tag genetic map for the platyfish (Xiphophorus maculatus) reveals mechanisms of karyotype evolution among teleost fish.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">625</span>–<span class="cit-lpage">641</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.164293</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164293%26rft_id%253Dinfo%253Apmid%252F24700104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk3LzIvNjI1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.3" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>,  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. D.</span> <span class="cit-name-surname">Etter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. S.</span> <span class="cit-name-surname">Atwood</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Currey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Shiver</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-elocation-id">e3376</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0003376</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.4" data-doi="10.1093/bioinformatics/btr174"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barnett</span>,  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. K.</span> <span class="cit-name-surname">Garrison</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Quinlan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Str̈mberg</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. T.</span> <span class="cit-name-surname">Marth</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Bamtools: A C++ API and toolkit for analyzing and managing BAM files.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1691</span>–<span class="cit-lpage">1692</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr174</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr174%26rft_id%253Dinfo%253Apmid%252F21493652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr174&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21493652&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291261300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.5" data-doi="10.1371/journal.pone.0019315"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baxter</span>,  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Davey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Johnston</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Shelton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. G.</span> <span class="cit-name-surname">Heckel</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Linkage mapping and comparative genomics using next-generation rad sequencing of a non-model organism.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e19315</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0019315</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaxter%26rft.auinit1%253DS.%2BW.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253De19315%26rft.epage%253De19315%26rft.atitle%253DLinkage%2Bmapping%2Band%2Bcomparative%2Bgenomics%2Busing%2Bnext-generation%2BRAD%2Bsequencing%2Bof%2Ba%2Bnon-model%2Borganism.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019315%26rft_id%253Dinfo%253Apmid%252F21541297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019315&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21541297&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Berkum</span>,  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Lieberman-Aiden</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Williams</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Imakaev</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Gnirke</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Hi-C: A Method to Study the Three-dimensional Architecture of Genomes.</span> <abbr class="cit-jnl-abbrev">J. Vis. Exp.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1869</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVis.%2BExp.%26rft.volume%253D6%26rft.spage%253D1869%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.7" data-doi="10.1371/currents.tol.53ba26640df0ccaee75bb165c8c26288"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Betancur-R</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. E.</span> <span class="cit-name-surname">Broughton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. O.</span> <span class="cit-name-surname">Wiley</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Carpenter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">López</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The tree of life and a new classification of bony fishes.</span> <abbr class="cit-jnl-abbrev">PLoS Curr.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BCurr.%26rft.volume%253D5%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fcurrents.tol.53ba26640df0ccaee75bb165c8c26288%26rft_id%253Dinfo%253Apmid%252F23653398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/currents.tol.53ba26640df0ccaee75bb165c8c26288&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23653398&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.8" data-doi="10.1038/ng.3526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Braasch</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Gehrke</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Smith</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kawasaki</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Manousaki</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The spotted gar genome illuminates vertebrate evolution and facilitates human-teleost comparisons.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">427</span>–<span class="cit-lpage">437</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3526</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D48%26rft.spage%253D427%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3526%26rft_id%253Dinfo%253Apmid%252F26950095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3526&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26950095&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.9" data-doi="10.1534/g3.111.000190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradley</span>,  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Breyer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. B.</span> <span class="cit-name-surname">Melville</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. W.</span> <span class="cit-name-surname">Broman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. W.</span> <span class="cit-name-surname">Knapik</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">An SNP-Based Linkage Map for Zebrafish Reveals Sex Determination Loci.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.111.000190</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D1%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.000190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.111.000190&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brandt</span>,  <span class="cit-name-given-names">D. Y. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. R. C.</span> <span class="cit-name-surname">Aguiar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. D.</span> <span class="cit-name-surname">Bitarello</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Nunes</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Goudet</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> Mapping Bias Overestimates Reference Allele Frequencies at the \textlessi\textgreaterHLA\textless/i\textgreater Genes in the 1000 Genomes Project Phase I Data. G3 (Bethesda) 5: 931–941.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.11" data-doi="10.1242/jeb.110445"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brennan</span>,  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Galvez</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Whitehead</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Reciprocal osmotic challenges reveal mechanisms of divergence in phenotypic plasticity in the killifish Fundulus heteroclitus.</span> <abbr class="cit-jnl-abbrev">J. Exp. Biol.</abbr> <span class="cit-vol">218</span>: <span class="cit-fpage">1212</span>–<span class="cit-lpage">1222</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1242/jeb.110445</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjeb.110445%26rft_id%253Dinfo%253Apmid%252F25722001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4YmlvIjtzOjU6InJlc2lkIjtzOjEwOiIyMTgvOC8xMjEyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.12" data-doi="10.1016/j.cbd.2007.09.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burnett</span>,  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. J.</span> <span class="cit-name-surname">Bain</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. S.</span> <span class="cit-name-surname">Baldwin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. V.</span> <span class="cit-name-surname">Callard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Cohen</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Fundulus as the premier teleost model in environmental biology: Opportunities for new insights using genomics.</span> <abbr class="cit-jnl-abbrev">Comp. Biochem. Physiol. Part D Genomics Proteomics</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">257</span>–<span class="cit-lpage">286</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cbd.2007.09.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DComp%2BBiochem%2BPhysiol%2BPart%2BD%2BGenomics%2BProteomics%26rft.aulast%253DBurnett%26rft.auinit1%253DK.%2BG.%26rft.volume%253D2%26rft.issue%253D4%26rft.spage%253D257%26rft.epage%253D286%26rft.atitle%253DFundulus%2Bas%2Bthe%2Bpremier%2Bteleost%2Bmodel%2Bin%2Benvironmental%2Bbiology%253A%2Bopportunities%2Bfor%2Bnew%2Binsights%2Busing%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cbd.2007.09.001%26rft_id%253Dinfo%253Apmid%252F18071578%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cbd.2007.09.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18071578&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-book" id="cit-9.9.2851.13" data-doi="10.1093/bioinformatics/btu314"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Faust</span>,  <span class="cit-name-given-names">G. G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. M.</span> <span class="cit-name-surname">Hall</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">SAMBLASTER: fast duplicate marking and structural variant read extraction.</span> <span class="cit-source">Bioinformatics</span> <span class="cit-vol">30</span>: <span class="cit-fpage">2503</span>–<span class="cit-lpage">2505</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu314</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu314%26rft_id%253Dinfo%253Apmid%252F24812344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu314&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24812344&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000342912400059&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.14" data-doi="10.1111/j.1095-8649.1972.tb05653.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>,  <span class="cit-name-given-names">P. B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Rachlin</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Karyotypic analysis of 2 sympatric species of fish, Fundulus heteroclitus L. and F. majalis.</span> <abbr class="cit-jnl-abbrev">J. Fish Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">71</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1095-8649.1972.tb05653.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BFish%2BBiol.%26rft.volume%253D4%26rft.spage%253D67%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1095-8649.1972.tb05653.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1095-8649.1972.tb05653.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-web" id="cit-9.9.2851.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrison</span>,  <span class="cit-name-given-names">E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Marth</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-comment">Haplotype-based variant detection from short-read sequencing. arXiv preprint arXiv:1207.3907 9</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.9.2851.16" data-doi="10.1038/sj.hdy.6800173"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>,  <span class="cit-name-given-names">C. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. B.</span> <span class="cit-name-surname">Broadfoot</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Effects of genotyping errors, missing values and segregation distortion in molecular marker data on the construction of linkage maps.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">38</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/sj.hdy.6800173</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHackett%26rft.auinit1%253DC.%2BA.%26rft.volume%253D90%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D38%26rft.atitle%253DEffects%2Bof%2Bgenotyping%2Berrors%252C%2Bmissing%2Bvalues%2Band%2Bsegregation%2Bdistortion%2Bin%2Bmolecular%2Bmarker%2Bdata%2Bon%2Bthe%2Bconstruction%2Bof%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800173%26rft_id%253Dinfo%253Apmid%252F12522423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800173&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12522423&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181165800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.17" data-doi="10.1152/physiolgenomics.00209.2009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Healy</span>,  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. E.</span> <span class="cit-name-surname">Tymchuk</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. J.</span> <span class="cit-name-surname">Osborne</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Schulte</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Heat shock response of killifish (Fundulus heteroclitus): candidate gene and heterologous microarray approaches.</span> <abbr class="cit-jnl-abbrev">Physiol. Genomics</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">184</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1152/physiolgenomics.00209.2009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiol.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fphysiolgenomics.00209.2009%26rft_id%253Dinfo%253Apmid%252F20103695%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1152/physiolgenomics.00209.2009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20103695&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276515200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.18" data-doi="10.1017/S0016672300010156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>,  <span class="cit-name-given-names">W. G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Robertson</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">The effect of linkage on limits to artificial selection.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">269</span>–<span class="cit-lpage">294</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672300010156</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D269%26rft.epage%253D294%26rft.atitle%253DThe%2Beffect%2Bof%2Blinkage%2Bon%2Blimits%2Bto%2Bartificial%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300010156%26rft_id%253Dinfo%253Apmid%252F5980116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300010156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5980116&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A19668750300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.19" data-doi="10.1093/nar/gkj144"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hinrichs</span>,  <span class="cit-name-given-names">A. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The UCSC Genome Browser Database: update 2006.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">D590</span>–<span class="cit-lpage">D598</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkj144</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DHinrichs%26rft.auinit1%253DA.%2BS.%26rft.volume%253D34%26rft.issue%253Dsuppl_1%26rft.spage%253DD590%26rft.epage%253DD598%26rft.atitle%253DThe%2BUCSC%2BGenome%2BBrowser%2BDatabase%253A%2Bupdate%2B2006%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkj144%26rft_id%253Dinfo%253Apmid%252F16381938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkj144&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16381938&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239307700126&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.20" data-doi="10.1038/nature03025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaillon</span>,  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-M.</span> <span class="cit-name-surname">Aury</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Brunet</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Petit</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Stange-Thomann</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genome duplication in the teleost fish Tetraodon nigroviridis reveals the early vertebrate proto-karyotype.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">431</span>: <span class="cit-fpage">946</span>–<span class="cit-lpage">957</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature03025</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJaillon%26rft.auinit1%253DO.%26rft.volume%253D431%26rft.issue%253D7011%26rft.spage%253D946%26rft.epage%253D957%26rft.atitle%253DGenome%2Bduplication%2Bin%2Bthe%2Bteleost%2Bfish%2BTetraodon%2Bnigroviridis%2Breveals%2Bthe%2Bearly%2Bvertebrate%2Bproto-karyotype.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03025%26rft_id%253Dinfo%253Apmid%252F15496914%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15496914&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224585600035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.21" data-doi="10.1093/gbe/evr041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kai</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kikuchi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Tohari</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. K.</span> <span class="cit-name-surname">Chew</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Tay</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Integration of the genetic map and genome assembly of fugu facilitates insights into distinct features of genome evolution in teleosts and mammals.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">424</span>–<span class="cit-lpage">442</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evr041</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevr041%26rft_id%253Dinfo%253Apmid%252F21551351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evr041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21551351&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.22" data-doi="10.1038/nature05846"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kasahara</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Naruse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Sasaki</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Nakatani</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Qu</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The medaka draft genome and insights into vertebrate genome evolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">447</span>: <span class="cit-fpage">714</span>–<span class="cit-lpage">719</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature05846</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKasahara%26rft.auinit1%253DM.%26rft.volume%253D447%26rft.issue%253D7145%26rft.spage%253D714%26rft.epage%253D719%26rft.atitle%253DThe%2Bmedaka%2Bdraft%2Bgenome%2Band%2Binsights%2Binto%2Bvertebrate%2Bgenome%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05846%26rft_id%253Dinfo%253Apmid%252F17554307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05846&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17554307&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247030700044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.23" data-doi="10.1101/gr.092759.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krzywinski</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Schein</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Birol</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Connors</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Gascoyne</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Circos: An information aesthetic for comparative genomics.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1639</span>–<span class="cit-lpage">1645</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.092759.109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.092759.109%26rft_id%253Dinfo%253Apmid%252F19541911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTYzOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOS8yODUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.24" data-doi="10.1101/gr.094557.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lewin</span>,  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Larkin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Pontius</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">O’Brien</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <em>Every genome sequence needs a good map</em>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.094557.109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.094557.109%26rft_id%253Dinfo%253Apmid%252F19596977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMS8xOTI1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.25" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Durbin</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.26" data-doi="10.1101/gr.1224503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Stoeckert</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. S.</span> <span class="cit-name-surname">Roos</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">OrthoMCL: Identification of ortholog groups for eukaryotic genomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2178</span>–<span class="cit-lpage">2189</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.1224503</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DLi%26rft.auinit1%253DL.%26rft.volume%253D13%26rft.issue%253D9%26rft.spage%253D2178%26rft.epage%253D2189%26rft.atitle%253DOrthoMCL%253A%2BIdentification%2Bof%2BOrtholog%2BGroups%2Bfor%2BEukaryotic%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1224503%26rft_id%253Dinfo%253Apmid%252F12952885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjEzLzkvMjE3OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOS8yODUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.27" data-doi="10.1111/j.1095-8312.2006.00558.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mank</span>,  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. E. L.</span> <span class="cit-name-surname">Promislow</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Avise</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Evolution of alternative sex-determining mechanisms in teleost fishes.</span> <abbr class="cit-jnl-abbrev">Biol. J. Linn. Soc. Lond.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">93</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1095-8312.2006.00558.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BJ.%2BLinn.%2BSoc.%2BLond.%26rft.volume%253D87%26rft.spage%253D83%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1095-8312.2006.00558.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1095-8312.2006.00558.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000235281200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.28" data-doi="10.1111/j.2007.0018-0661.02000.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Margarido</span>,  <span class="cit-name-given-names">G. R. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Souza</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A. F.</span> <span class="cit-name-surname">Garcia</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">OneMap: Software for genetic mapping in outcrossing species.</span> <abbr class="cit-jnl-abbrev">Hereditas</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">78</span>–<span class="cit-lpage">79</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.2007.0018-0661.02000.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHereditas%26rft.stitle%253DHereditas%26rft.aulast%253DMargarido%26rft.auinit1%253DG.%2BR.%26rft.volume%253D144%26rft.issue%253D3%26rft.spage%253D78%26rft.epage%253D79%26rft.atitle%253DOneMap%253A%2Bsoftware%2Bfor%2Bgenetic%2Bmapping%2Bin%2Boutcrossing%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.2007.0018-0661.02000.x%26rft_id%253Dinfo%253Apmid%252F17663699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.2007.0018-0661.02000.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17663699&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248257800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.29" data-doi="10.1101/gr.5681207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Dunham</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Amores</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. A.</span> <span class="cit-name-surname">Cresko</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">Johnson</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Rapid and cost-effective polymorphism identification and genotyping using restriction site associated DNA (RAD) markers.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">240</span>–<span class="cit-lpage">248</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.5681207</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5681207%26rft_id%253Dinfo%253Apmid%252F17189378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE3LzIvMjQwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.9.2851.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montgomery</span>,  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Huang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. H.</span> <span class="cit-name-surname">Langley</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. H.</span> <span class="cit-name-surname">Judd</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Chromosome rearrangement by ectopic recombination in Drosophila melanogaster: genome structure and evolution. Genetics</span> <span class="cit-vol">129</span>: <span class="cit-fpage">1085</span>–<span class="cit-lpage">1098</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.31" data-doi="10.1111/mec.13848"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nacci</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Proestou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Champlin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Martinson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. R.</span> <span class="cit-name-surname">Waits</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genetic basis for rapidly evolved tolerance in the wild: adaptation to toxic pollutants by an estuarine fish species.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">5467</span>–<span class="cit-lpage">5482</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13848</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D25%26rft.spage%253D5467%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13848%26rft_id%253Dinfo%253Apmid%252F27662639%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13848&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27662639&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Naruse</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Fukamachi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Mitani</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kondo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Matsuoka</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A detailed linkage map of medaka, Oryzias latipes: comparative genomics and genome evolution.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">1773</span>–<span class="cit-lpage">1784</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNaruse%26rft.auinit1%253DK.%26rft.volume%253D154%26rft.issue%253D4%26rft.spage%253D1773%26rft.epage%253D1784%26rft.atitle%253DA%2BDetailed%2BLinkage%2BMap%2Bof%2BMedaka%252C%2BOryzias%2Blatipes%253A%2BComparative%2BGenomics%2Band%2BGenome%2BEvolution%26rft_id%253Dinfo%253Apmid%252F10747068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NC80LzE3NzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjg1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.33" data-doi="10.1186/1471-2164-12-263"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oleksiak</span>,  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. I.</span> <span class="cit-name-surname">Karchner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Jenny</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. G.</span> <span class="cit-name-surname">Franks</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. B. M.</span> <span class="cit-name-surname">Welch</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Transcriptomic assessment of resistance to effects of an aryl hydrocarbon receptor (AHR) agonist in embryos of Atlantic killifish (Fundulus heteroclitus) from a marine Superfund site.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">263</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-12-263</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DOleksiak%26rft.auinit1%253DM.%2BF.%26rft.volume%253D12%26rft.spage%253D263%26rft.epage%253D263%26rft.atitle%253DTranscriptomic%2Bassessment%2Bof%2Bresistance%2Bto%2Beffects%2Bof%2Ban%2Baryl%2Bhydrocarbon%2Breceptor%2B%2528AHR%2529%2Bagonist%2Bin%2Bembryos%2Bof%2BAtlantic%2Bkillifish%2B%2528Fundulus%2Bheteroclitus%2529%2Bfrom%2Ba%2Bmarine%2BSuperfund%2Bsite.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-263%26rft_id%253Dinfo%253Apmid%252F21609454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-263&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21609454&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.34" data-doi="10.1038/ng1483"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oleksiak</span>,  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Roach</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. L.</span> <span class="cit-name-surname">Crawford</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Natural variation in cardiac metabolism and gene expression in Fundulus heteroclitus.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">72</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1483</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DOleksiak%26rft.auinit1%253DM.%2BF.%26rft.volume%253D37%26rft.issue%253D1%26rft.spage%253D67%26rft.epage%253D72%26rft.atitle%253DNatural%2Bvariation%2Bin%2Bcardiac%2Bmetabolism%2Band%2Bgene%2Bexpression%2Bin%2BFundulus%2Bheteroclitus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1483%26rft_id%253Dinfo%253Apmid%252F15568023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1483&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15568023&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225997500022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.35" data-doi="10.1093/gbe/evx023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reid</span>,  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. E.</span> <span class="cit-name-surname">Jackson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gilbert</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Minx</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Montague</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The landscape of extreme genomic variation in the highly adaptable Atlantic killifish.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">659</span>–<span class="cit-lpage">676</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evx023</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft.volume%253D9%26rft.spage%253D659%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevx023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evx023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.36" data-doi="10.1126/science.aah4993"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reid</span>,  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Proestou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. W.</span> <span class="cit-name-surname">Clark</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. C.</span> <span class="cit-name-surname">Warren</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Colbourne</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The genomic landscape of rapid repeated evolutionary adaptation to toxic pollution in wild fish.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">354</span>: <span class="cit-fpage">1305</span>–<span class="cit-lpage">1308</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.aah4993</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DReid%26rft.auinit1%253DN.%2BM.%26rft.volume%253D354%26rft.issue%253D6317%26rft.spage%253D1305%26rft.epage%253D1308%26rft.atitle%253DThe%2Bgenomic%2Blandscape%2Bof%2Brapid%2Brepeated%2Bevolutionary%2Badaptation%2Bto%2Btoxic%2Bpollution%2Bin%2Bwild%2Bfish%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aah4993%26rft_id%253Dinfo%253Apmid%252F27940876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNTQvNjMxNy8xMzA1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.37" data-doi="10.1111/mec.12322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roesti</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Moser</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Berner</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Recombination in the threespine stickleback genome—patterns and consequences.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3014</span>–<span class="cit-lpage">3027</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12322</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3014%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12322%26rft_id%253Dinfo%253Apmid%252F23601112.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23601112.&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319675300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.38" data-doi="10.1007/s11160-009-9123-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sandra</span>,  <span class="cit-name-given-names">G.-E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.-M.</span> <span class="cit-name-surname">Norma</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Sexual determination and differentiation in teleost fish.</span> <abbr class="cit-jnl-abbrev">Rev. Fish Biol. Fish.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">121</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11160-009-9123-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRev.%2BFish%2BBiol.%2BFish.%26rft.volume%253D20%26rft.spage%253D101%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11160-009-9123-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11160-009-9123-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.39" data-doi="10.1073/pnas.0701572104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sawyer</span>,  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Parsch</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Zhang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. L.</span> <span class="cit-name-surname">Hartl</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Prevalence of positive selection among nearly neutral amino acid replacements in Drosophila.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">6504</span>–<span class="cit-lpage">6510</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0701572104</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0701572104%26rft_id%253Dinfo%253Apmid%252F17409186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA0LzE2LzY1MDQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjg1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.40" data-doi="10.1016/j.mod.2004.03.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schartl</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A comparative view on sex determination in medaka.</span> <abbr class="cit-jnl-abbrev">Mech. Dev.</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">639</span>–<span class="cit-lpage">645</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.mod.2004.03.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMechanisms%2Bof%2Bdevelopment%26rft.stitle%253DMech%2BDev%26rft.aulast%253DSchartl%26rft.auinit1%253DM.%26rft.volume%253D121%26rft.issue%253D7-8%26rft.spage%253D639%26rft.epage%253D645%26rft.atitle%253DA%2Bcomparative%2Bview%2Bon%2Bsex%2Bdetermination%2Bin%2Bmedaka.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mod.2004.03.001%26rft_id%253Dinfo%253Apmid%252F15210173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mod.2004.03.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15210173&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222571200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.41" data-doi="10.1126/science.8211116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Hernandez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Ramnarain</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Huff</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Ordered restriction maps of Saccharomyces cerevisiae chromosomes constructed by optical mapping.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">262</span>: <span class="cit-fpage">110</span>–<span class="cit-lpage">114</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.8211116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSchwartz%26rft.auinit1%253DD.%26rft.volume%253D262%26rft.issue%253D5130%26rft.spage%253D110%26rft.epage%253D114%26rft.atitle%253DOrdered%2Brestriction%2Bmaps%2Bof%2BSaccharomyces%2Bcerevisiae%2Bchromosomes%2Bconstructed%2Bby%2Boptical%2Bmapping%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.8211116%26rft_id%253Dinfo%253Apmid%252F8211116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNjIvNTEzMC8xMTAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjg1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singer</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Perlman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Yan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Walker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Corley-Smith</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Sex-specific recombination rates in zebrafish (Danio rerio). Evolution in Health and Disease</span> <span class="cit-vol">160</span>: <span class="cit-fpage">649</span>–<span class="cit-lpage">657</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.43" data-doi="10.1038/srep35736"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Li</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Zhang</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Coverage recommendation for genotyping analysis of highly heterologous species using next-generation sequencing technology.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">35736</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep35736</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D35736%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep35736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep35736&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.44" data-doi="10.1093/gbe/evw264"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sotero-Caio</span>,  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. N.</span> <span class="cit-name-surname">Platt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Suh</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Ray</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Evolution and Diversity of Transposable Elements in Vertebrate Genomes.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">161</span>–<span class="cit-lpage">177</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evw264</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft.volume%253D9%26rft.spage%253D161%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevw264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evw264&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.45" data-doi="10.1111/j.1365-313X.1993.00739.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stam</span>,  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Construction of Integrated Genetic-Linkage Maps by Means of a New Computer Package - Joinmap.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">739</span>–<span class="cit-lpage">744</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-313X.1993.00739.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D3%26rft.spage%253D739%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.1993.00739.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.1993.00739.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1993LC75800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.46" data-doi="10.1002/jez.1400140104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sturtevant</span>,  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1913</span> <span class="cit-article-title">The linear arrangement of six sex linked factors in Drosophila, as shown by their mode of association.</span> <abbr class="cit-jnl-abbrev">J. Exp. Zool.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">59</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/jez.1400140104</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BZool.%26rft.volume%253D14%26rft.spage%253D43%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjez.1400140104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jez.1400140104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000187376000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.47" data-doi="10.1186/s13059-014-0573-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Miao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Ming</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">ALLMAPS: Robust scaffold ordering based on multiple maps.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">3</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-014-0573-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0573-1%26rft_id%253Dinfo%253Apmid%252F25583564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0573-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25583564&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.48" data-doi="10.1038/nrg3117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Treangen</span>,  <span class="cit-name-given-names">T. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Salzberg</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Repetitive DNA and next-generation sequencing: Computational challenges and solutions.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">36</span>–<span class="cit-lpage">46</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3117</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DTreangen%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D36%26rft.epage%253D46%26rft.atitle%253DRepetitive%2BDNA%2Band%2Bnext-generation%2Bsequencing%253A%2Bcomputational%2Bchallenges%2Band%2Bsolutions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3117%26rft_id%253Dinfo%253Apmid%252F22124482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22124482&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.49" data-doi="10.4236/ojgen.2016.61004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waits</span>,  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Martinson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Rinner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Morris</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Proestou</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genetic Linkage Map and Comparative Genome Analysis for the Atlantic Killifish (Fundulus heteroclitus).</span> <abbr class="cit-jnl-abbrev">Open Journal of Genetics Open Journal of Genetics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">28</span>–<span class="cit-lpage">38</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4236/ojgen.2016.61004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOpen%2BJournal%2Bof%2BGenetics%2BOpen%2BJournal%2Bof%2BGenetics%26rft.volume%253D6%26rft.spage%253D28%26rft_id%253Dinfo%253Adoi%252F10.4236%252Fojgen.2016.61004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4236/ojgen.2016.61004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.50" data-doi="10.1534/genetics.103.019349"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walter</span>,  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Rains</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Russell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. M.</span> <span class="cit-name-surname">Guerra</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Daniels</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A Microsatellite Genetic Linkage Map for Xiphophorus.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">363</span>–<span class="cit-lpage">372</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.103.019349</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWalter%26rft.auinit1%253DR.%2BB.%26rft.volume%253D168%26rft.issue%253D1%26rft.spage%253D363%26rft.epage%253D372%26rft.atitle%253DA%2BMicrosatellite%2BGenetic%2BLinkage%2BMap%2Bfor%2BXiphophorus%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.103.019349%26rft_id%253Dinfo%253Apmid%252F15454549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY4LzEvMzYzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.51" data-doi="10.1073/pnas.1017542108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whitehead</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Roach</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Zhang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Galvez</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic mechanisms of evolved physiological plasticity in killifish distributed along an environmental salinity gradient.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">6193</span>–<span class="cit-lpage">6198</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1017542108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1017542108%26rft_id%253Dinfo%253Apmid%252F21444822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzE1LzYxOTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjg1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.52" data-doi="10.1101/gr.10.12.1903"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woods</span>,  <span class="cit-name-given-names">I. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. D.</span> <span class="cit-name-surname">Kelly</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Chu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Ngo-Hazelett</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.-L.</span> <span class="cit-name-surname">Yan</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A Comparative Map of the Zebrafish Genome.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1903</span>–<span class="cit-lpage">1914</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.10.12.1903</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DWoods%26rft.auinit1%253DI.%2BG.%26rft.volume%253D10%26rft.issue%253D12%26rft.spage%253D1903%26rft.epage%253D1914%26rft.atitle%253DA%2BComparative%2BMap%2Bof%2Bthe%2BZebrafish%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.10.12.1903%26rft_id%253Dinfo%253Apmid%252F11116086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMC8xMi8xOTAzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.53" data-doi="10.1101/gr.4134305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woods</span>,  <span class="cit-name-given-names">I. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Wilson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Friedlander</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Chang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. K.</span> <span class="cit-name-surname">Reyes</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The zebrafish gene map defines ancestral vertebrate chromosomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1307</span>–<span class="cit-lpage">1314</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.4134305</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4134305%26rft_id%253Dinfo%253Apmid%252F16109975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE1LzkvMTMwNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOS8yODUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Ma</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <abbr class="cit-jnl-abbrev">Simultaneous Maximum Likelihood Estimation of Linkage and Linkage Phases in Outcrossing Species.</abbr> <span class="cit-vol">363</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSimultaneous%2BMaximum%2BLikelihood%2BEstimation%2Bof%2BLinkage%2Band%2BLinkage%2BPhases%2Bin%2BOutcrossing%2BSpecies.%26rft.volume%253D363%26rft.spage%253D349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2851.55" data-doi="10.1534/genetics.108.090688"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Quantitative Trait Locus Mapping Can Benefit From Segregation Distortion.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">2201</span>–<span class="cit-lpage">2208</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.108.090688</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.090688%26rft_id%253Dinfo%253Apmid%252F18957707%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MC80LzIyMDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjg1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/9/2851.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/9/2835" title="Single-Gene Deletions Contributing to Loss of Heterozygosity in Saccharomyces cerevisiae: Genome-Wide Screens and Reproducibility" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/9/2863" title="A Prospective Analysis of Genetic Variants Associated with Human Lifespan" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/9"><span>Volume 9 Issue 9, September 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/9.cover-source.jpg?itok=4kZTDz_V" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/2851" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-wsGB8oeTsoLlMmzbbXDYfBuZzu4OT3TKTU9qXrQeEbk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/9/2851" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/9/2851" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/9/2851" class="active" data-icon-position="" data-hide-link-title="0">Developing a High-Quality Linkage Map for the Atlantic Killifish Fundulus heteroclitus</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494060" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-iejl8MJjqBRnGsuyc2cK1FaBG_6dK5vRDkLkJr_hftI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/9/2851.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/9/2851" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-FrHnaCzDIFuSTrtDpKqUT6VKj3Bog32cDVlISAy0c1Y" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/9/2851" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/9/2851.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494060" id="node494060--2" data-pisa="ggg;9/9/2851" data-pisa-master="ggg;g3.119.400262" data-apath="/ggg/9/9/2851.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Developing a High-Quality Linkage Map for the Atlantic Killifish <em>Fundulus heteroclitus</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-2326-1855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jeffrey T. Miller</span>, <span class="highwire-citation-author" data-delta="1">Noah M. Reid</span>, <span class="highwire-citation-author" data-delta="2">Diane E. Nacci</span> and <span class="highwire-citation-author" data-delta="3">Andrew Whitehead</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2851-2862;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400262 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494060--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jeffrey T. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Environmental Toxicology, Center for Population Biology, Coastal and Marine Sciences Institute, University of California, Davis, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BT.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20JT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeffrey%2BT.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2326-1855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jeffrey T. Miller</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Noah M. Reid </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular & Cell Biology, University of Connecticut, Storrs, CT, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Noah%2BM.%2BReid%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reid%20NM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANoah%2BM.%2BReid%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Diane E. Nacci </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>US Environmental Protection Agency, Office of Research and Development, National Health and Environmental Effects Research Laboratory, Atlantic Ecology Division, Narragansett, RI</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diane%2BE.%2BNacci%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nacci%20DE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiane%2BE.%2BNacci%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Andrew Whitehead </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Environmental Toxicology, Center for Population Biology, Coastal and Marine Sciences Institute, University of California, Davis, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BWhitehead%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whitehead%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BWhitehead%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:awhitehead@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">awhitehead@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/9/2851" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/9/2851&amp;title=Developing%20a%20High-Quality%20Linkage%20Map%20for%20the%20Atlantic%20Killifish%20Fundulus%20heteroclitus" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/9/2851&amp;title=Developing%20a%20High-Quality%20Linkage%20Map%20for%20the%20Atlantic%20Killifish%20Fundulus%20heteroclitus" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/9/2851&amp;title=Developing%20a%20High-Quality%20Linkage%20Map%20for%20the%20Atlantic%20Killifish%20Fundulus%20heteroclitus" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/9/2851&amp;text=Developing%20a%20High-Quality%20Linkage%20Map%20for%20the%20Atlantic%20Killifish%20Fundulus%20heteroclitus" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/9/2851&amp;title=Developing%20a%20High-Quality%20Linkage%20Map%20for%20the%20Atlantic%20Killifish%20Fundulus%20heteroclitus" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/9/2851&amp;t=Developing%20a%20High-Quality%20Linkage%20Map%20for%20the%20Atlantic%20Killifish%20Fundulus%20heteroclitus" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/9/2851&amp;title=Developing%20a%20High-Quality%20Linkage%20Map%20for%20the%20Atlantic%20Killifish%20Fundulus%20heteroclitus" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/9/2851&amp;title=Developing%20a%20High-Quality%20Linkage%20Map%20for%20the%20Atlantic%20Killifish%20Fundulus%20heteroclitus" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494060/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494060" id="node494060--3" data-pisa="ggg;9/9/2851" data-pisa-master="ggg;g3.119.400262" data-apath="/ggg/9/9/2851.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Developing a High-Quality Linkage Map for the Atlantic Killifish <em>Fundulus heteroclitus</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-2326-1855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jeffrey T. Miller</span>, <span class="highwire-citation-author" data-delta="1">Noah M. Reid</span>, <span class="highwire-citation-author" data-delta="2">Diane E. Nacci</span> and <span class="highwire-citation-author" data-delta="3">Andrew Whitehead</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2851-2862;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400262 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494060--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jeffrey T. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Environmental Toxicology, Center for Population Biology, Coastal and Marine Sciences Institute, University of California, Davis, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BT.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20JT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeffrey%2BT.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2326-1855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jeffrey T. Miller</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Noah M. Reid </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular & Cell Biology, University of Connecticut, Storrs, CT, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Noah%2BM.%2BReid%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reid%20NM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANoah%2BM.%2BReid%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Diane E. Nacci </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>US Environmental Protection Agency, Office of Research and Development, National Health and Environmental Effects Research Laboratory, Atlantic Ecology Division, Narragansett, RI</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diane%2BE.%2BNacci%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nacci%20DE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiane%2BE.%2BNacci%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Andrew Whitehead </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Environmental Toxicology, Center for Population Biology, Coastal and Marine Sciences Institute, University of California, Davis, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BWhitehead%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whitehead%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BWhitehead%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:awhitehead@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">awhitehead@ucdavis.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2851&amp;rft.epage=2862&amp;rft.atitle=Developing%20a%20High-Quality%20Linkage%20Map%20for%20the%20Atlantic%20Killifish%20Fundulus%20heteroclitus&amp;rft.volume=9&amp;rft.issue=9&amp;rft.date=2019-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Miller%2C+Jeffrey+T.&amp;rft.au=Reid%2C+Noah+M.&amp;rft.au=Nacci%2C+Diane+E.&amp;rft.au=Whitehead%2C+Andrew"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494060/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494060/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494060/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494060/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494060/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494060/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494060/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494060/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494060/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494060/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494060/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494060/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/9/2851" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results &amp; Discussion</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/9/2851.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/9/2851.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/9/2851.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
