<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/9/2925.full.txt" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/9/2925.full.pdf" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/9/2925/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/9/2925.atom" />
<meta name="HW.pisa" content="ggg;9/9/2925" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Joint Use of Genome, Pedigree, and Their Interaction with Environment for Predicting the Performance of Wheat Lines in New Environments" />
<meta name="DC.Identifier" content="10.1534/g3.119.400508" />
<meta name="DC.Date" content="2019-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Howard et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genome-enabled prediction plays an essential role in wheat breeding because it has the potential to increase the rate of genetic gain relative to traditional phenotypic and pedigree-based selection. Since the performance of wheat lines is highly influenced by environmental stimuli, it is important to accurately model the environment and its interaction with genetic factors in prediction models. Arguably, multi-environmental best linear unbiased prediction (BLUP) may deliver better prediction performance than single-environment genomic BLUP. We evaluated pedigree and genome-based prediction using 35,403 wheat lines from the Global Wheat Breeding Program of the International Maize and Wheat Improvement Center (CIMMYT). We implemented eight statistical models that included genome-wide molecular marker and pedigree information as prediction inputs in two different validation schemes. All models included main effects, but some considered interactions between the different types of pedigree and genomic covariates via Hadamard products of similarity kernels. Pedigree models always gave better prediction of new lines in observed environments than genome-based models when only main effects were fitted. However, for all traits, the highest predictive abilities were obtained when interactions between pedigree, genomes, and environments were included. When new lines were predicted in unobserved environments, in almost all trait/year combinations, the marker main-effects model was the best. These results provide strong evidence that the different sources of genetic information (molecular markers and pedigree) are not equally useful at different stages of the breeding pipelines, and can be employed differentially to improve the design and prediction of the outcome of future breeding programs." />
<meta name="DC.Contributor" content="Réka Howard" />
<meta name="DC.Contributor" content="Daniel Gianola" />
<meta name="DC.Contributor" content="Osval Montesinos-López" />
<meta name="DC.Contributor" content="Philomin Juliana" />
<meta name="DC.Contributor" content="Ravi Singh" />
<meta name="DC.Contributor" content="Jesse Poland" />
<meta name="DC.Contributor" content="Sandesh Shrestha" />
<meta name="DC.Contributor" content="Paulino Pérez-Rodríguez" />
<meta name="DC.Contributor" content="José Crossa" />
<meta name="DC.Contributor" content="Diego Jarquín" />
<meta name="article:published_time" content="2019-09-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="Joint Use of Genome, Pedigree, and Their Interaction with Environment for Predicting the Performance of Wheat Lines in New Environments" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genome-enabled prediction plays an essential role in wheat breeding because it has the potential to increase the rate of genetic gain relative to traditional phenotypic and pedigree-based selection. Since the performance of wheat lines is highly influenced by environmental stimuli, it is important to accurately model the environment and its interaction with genetic factors in prediction models. Arguably, multi-environmental best linear unbiased prediction (BLUP) may deliver better prediction performance than single-environment genomic BLUP. We evaluated pedigree and genome-based prediction using 35,403 wheat lines from the Global Wheat Breeding Program of the International Maize and Wheat Improvement Center (CIMMYT). We implemented eight statistical models that included genome-wide molecular marker and pedigree information as prediction inputs in two different validation schemes. All models included main effects, but some considered interactions between the different types of pedigree and genomic covariates via Hadamard products of similarity kernels. Pedigree models always gave better prediction of new lines in observed environments than genome-based models when only main effects were fitted. However, for all traits, the highest predictive abilities were obtained when interactions between pedigree, genomes, and environments were included. When new lines were predicted in unobserved environments, in almost all trait/year combinations, the marker main-effects model was the best. These results provide strong evidence that the different sources of genetic information (molecular markers and pedigree) are not equally useful at different stages of the breeding pipelines, and can be employed differentially to improve the design and prediction of the outcome of future breeding programs.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/09/01" />
<meta name="citation_mjid" content="ggg;9/9/2925" />
<meta name="citation_id" content="9/9/2925" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/9/2925" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/9/2925.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/9/2925.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/9/2925.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400508" />
<meta name="citation_pmid" content="31300481" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="2925" />
<meta name="citation_lastpage" content="2934" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Réka Howard" />
<meta name="citation_author_institution" content="University of Nebraska – Lincoln, Lincoln NE, 68583, USA" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8837-3770" />
<meta name="citation_author" content="Daniel Gianola" />
<meta name="citation_author_institution" content="University of Wisconsin-Madison, Madison, Wisconsin 53706, USA" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8217-2348" />
<meta name="citation_author" content="Osval Montesinos-López" />
<meta name="citation_author_institution" content="Universidad de Colima, CP 28040 Colima, México" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4464-3385" />
<meta name="citation_author" content="Philomin Juliana" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Km. 45, Carretera México-Veracruz, El Batán, Texcoco, Edo. de México, CP 56130, México" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6922-0173" />
<meta name="citation_author" content="Ravi Singh" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Km. 45, Carretera México-Veracruz, El Batán, Texcoco, Edo. de México, CP 56130, México" />
<meta name="citation_author" content="Jesse Poland" />
<meta name="citation_author_institution" content="USDA-ARS and Dep. of Plant Pathology, Kansas State University, 4024 Throckmorton Hall, Manhattan KS, 66506, USA, and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7856-1399" />
<meta name="citation_author" content="Sandesh Shrestha" />
<meta name="citation_author_institution" content="USDA-ARS and Dep. of Plant Pathology, Kansas State University, 4024 Throckmorton Hall, Manhattan KS, 66506, USA, and" />
<meta name="citation_author" content="Paulino Pérez-Rodríguez" />
<meta name="citation_author_institution" content="Colegio de Postgraduados, Montecillos, Edo. de México, CP 56230, México" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3202-1784" />
<meta name="citation_author" content="José Crossa" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Km. 45, Carretera México-Veracruz, El Batán, Texcoco, Edo. de México, CP 56130, México" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author_email" content="diego.jarquin@gmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9429-5855" />
<meta name="citation_author" content="Diego Jarquín" />
<meta name="citation_author_institution" content="University of Nebraska – Lincoln, Lincoln NE, 68583, USA" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author_email" content="diego.jarquin@gmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5098-2060" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=B. R. Basnet;citation_author=J. Crossa;citation_author=S. Dreisigacker;citation_author=P. Pérez-Rodríguez;citation_author=Y. Manes;citation_title=Hybrid Wheat Prediction Using Genomic, Pedigree, and Environmental Covariables Interaction Models.;citation_pages=180051;citation_volume=12;citation_year=2019;citation_doi=10.3835/plantgenome2018.07.0051" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_title=Prediction of Maize Single-Cross Performance Using RFLPs and Information from Related Hybrids;citation_pages=20-25;citation_volume=34;citation_year=1994;citation_issue=1;citation_doi=10.2135/cropsci1994.0011183X003400010003x" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Burgueño;citation_author=G. de los Campos;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers.;citation_pages=707-719;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0299" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=M. P. L. Calus;citation_author=A. F. Groen;citation_author=G. de Jong;citation_title=Genotype x Environment Interaction for Protein Yield in Dutch Dairy Cattle as Quantified by Different Models;citation_pages=3115-3123;citation_volume=85;citation_year=2002;citation_issue=11;citation_pmid=12487479;citation_doi=10.3168/jds.S0022-0302(02)74399-3" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=M. P. L. Calus;citation_author=R. G. Veerkamp;citation_title=Estimation of Environmental Sensitivity of Genetic Merit for Milk Production Traits Using a Random Regression Model;citation_pages=3756-3764;citation_volume=86;citation_year=2003;citation_issue=11;citation_pmid=14672207;citation_doi=10.3168/jds.S0022-0302(03)73982-4" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Crain;citation_author=S. Mondal;citation_author=J. Rutkoski;citation_author=R. P. Singh;citation_author=J. Poland;citation_title=Combining high-throughput phenotyping and genomic information to increase prediction and selection accuracy in wheat breeding.;citation_volume=11;citation_year=2018;citation_doi=10.3835/plantgenome2017.05.0043" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=G. de los Campos;citation_author=P. Pérez-Rodríguez;citation_author=D. Gianola;citation_author=J. Burgueño;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.;citation_volume=186;citation_year=2010;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=J. Crossa;citation_author=D. Jarquín;citation_author=J. Franco;citation_author=P. Pérez-Rodríguez;citation_author=J. Burgueño;citation_title=Genomic prediction of gene bank wheat landraces.;citation_volume=6;citation_year=2016;citation_pmid=27172218;citation_doi=10.1534/g3.116.029637" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=J. Crossa;citation_author=P. Pérez-Rodríguez;citation_author=J. Cuevas;citation_author=O. Montesinos-López;citation_author=D. Jarquín;citation_title=Genomic selection in plant breeding: methods, models, and perspectives.;citation_pages=961-975;citation_volume=22;citation_year=2017;citation_doi=10.1016/j.tplants.2017.08.011" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=H. Naya;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=A. Legarra;citation_title=Predicting quantitative traits with regression models for dense molecular markers and pedigree.;citation_volume=182;citation_year=2009;citation_pmid=19293140;citation_doi=10.1534/genetics.109.101501" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. L. Calus;citation_title=Whole-genome regression and prediction methods applied to plant and animal breeding.;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=D. S. Falconer;citation_title=The problem of environment and selection.;citation_pages=293-298;citation_volume=86;citation_year=1952;citation_doi=10.1086/281736" />
<meta name="citation_reference" content="citation_title=Introduction to quantitative genetics;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_author=R. L. Fernando;citation_author=A. Stella;citation_title=Genomic-assisted prediction of genetic value with semiparametric procedures.;citation_volume=173;citation_year=2006;citation_pmid=16648593;citation_doi=10.1534/genetics.105.049510" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Gianola;citation_author=B. C. H. M. van Kaam;citation_title=Reproducing Kernel Hilbert Spaces Regression Methods for Genomic Assisted Prediction of Quantitative Traits;citation_pages=2289-2303;citation_volume=178;citation_year=2008;citation_issue=4;citation_pmid=18430950;citation_doi=10.1534/genetics.107.084285" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_author=G. de los Campos;citation_author=W. G. Hill;citation_author=E. Manfredi;citation_author=R. Fernando;citation_title=Additive genetic variability and the Bayesian alphabet.;citation_volume=183;citation_year=2009;citation_pmid=19620397;citation_doi=10.1534/genetics.109.103952" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_title=Priors in whole-genome regression: the Bayesian alphabet returns.;citation_volume=194;citation_year=2013;citation_pmid=23636739;citation_doi=10.1534/genetics.113.151753" />
<meta name="citation_reference" content="citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=H.-R. Gregorius;citation_author=G. Namkoong;citation_title=Joint analysis of genotypic and environmental effects.;citation_pages=413-422;citation_volume=72;citation_year=1986;citation_pmid=24247951;citation_doi=10.1007/BF00288581" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. Jarquín;citation_author=J. Crossa;citation_author=X. Lacaze;citation_author=P. D. Cheyron;citation_author=J. Daucourt;citation_title=A reaction norm model for genomic selection using high-dimensional genomic and environmental data.;citation_pages=595-607;citation_volume=127;citation_year=2014;citation_pmid=24337101;citation_doi=10.1007/s00122-013-2243-1" />
<meta name="citation_reference" content="citation_author=D. Jarquín;citation_author=C. Lemes da Silva;citation_author=R. Gaynor;citation_author=J. Poland;citation_author=A. Fritz;citation_year=2017;citation_doi=10.3835/plantgenome2016.12.0130" />
<meta name="citation_reference" content="citation_journal_title=Agronomy (Basel);citation_author=D. Jarquín;citation_author=R. Howard;citation_author=A. Xavier;citation_author=S. Das Choudhury;citation_title=Increasing Predictive Ability by Modeling Interactions between Environments, Genotype and Canopy Coverage Image Data for Soybeans.;citation_pages=51;citation_volume=8;citation_year=2018;citation_doi=10.3390/agronomy8040051" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Jiang;citation_author=J. C. Reif;citation_title=Modeling epistasis in genomic selection.;citation_volume=201;citation_year=2015;citation_pmid=26219298;citation_doi=10.1534/genetics.115.177907" />
<meta name="citation_reference" content="citation_author=P. Juliana;citation_author=R. P. Singh;citation_author=J. A. Poland;citation_author=M. Suchismita;citation_author=J. Crossa;citation_year=2018;citation_doi=10.3835/plantgenome2018.03.0017" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=P. Juliana;citation_author=O. A. Montesinos-López;citation_author=J. Crossa;citation_author=S. Mondal;citation_author=L. G. Pérez;citation_title=Integrating genomic-enabled prediction and high-throughput phenotyping in breeding for climate-resilient bread wheat.;citation_pages=177-194;citation_volume=132;citation_year=2018;citation_doi=10.1007/s00122-018-3206-3" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. W. R. Martini;citation_author=V. Wimmer;citation_author=M. Erbe;citation_author=H. Simianer;citation_title=Epistasis and covariance: how gene interaction translates into genomic relationship.;citation_pages=963-976;citation_volume=129;citation_year=2016;citation_doi=10.1007/s00122-016-2675-5" />
<meta name="citation_reference" content="citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=G. Morota;citation_author=D. Gianola;citation_title=Kernel-based whole-genome prediction of complex traits: a review.;citation_pages=363;citation_volume=5;citation_year=2014;citation_pmid=25360145;citation_doi=10.3389/fgene.2014.00363" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=G. Morota;citation_author=P. Boddhireddy;citation_author=N. Vukasinovic;citation_author=G. Gianola;citation_author=S. DeNise;citation_title=Kernel-based variance component estimation and whole-genome prediction of pre-corrected phenotypes and progeny tests for dairy cow health traits.;citation_pages=56;citation_volume=5;citation_year=2014;citation_pmid=24715901;citation_doi=10.3389/fgene.2014.00056" />
<meta name="citation_reference" content="citation_author=M. Lopez-Cruz;citation_author=J. Crossa;citation_author=D. Bonnett;citation_author=S. Dreisigacker;citation_author=J. Poland;citation_year=2015;citation_pmid=25660166;citation_doi=10.1534/g3.114.016097" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Pérez;citation_author=G. de los Campos;citation_title=Genome-wide regression &amp; prediction with the BGLR statistical package.;citation_volume=198;citation_year=2014;citation_pmid=25009151;citation_doi=10.1534/genetics.114.164442" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=P. Pérez-Rodríguez;citation_author=J. Crossa;citation_author=K. Bondalapati;citation_author=G. De Meyer;citation_author=F. Pita;citation_title=A Pedigree-Based reaction norm model for Prediction of Cotton Yield in Multi Environment Trials.;citation_pages=1143-1151;citation_volume=55;citation_year=2015;citation_doi=10.2135/cropsci2014.08.0577" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=P. Pérez-Rodríguez;citation_author=J. Crossa;citation_author=J. Rutkoski;citation_author=J. Poland;citation_author=R. Singh;citation_title=Single-step genomic and pedigree genotype × environment interaction models for predicting wheat lines in international environments.;citation_pages=2;citation_volume=10;citation_year=2017;citation_doi=10.3835/plantgenome2016.09.0089" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=D. K. Ray;citation_author=N. D. Mueller;citation_author=P. C. West;citation_author=J. A. Foley;citation_title=Yield Trends Are Insufficient to Double Global Crop Production by 2050.;citation_year=2013;citation_doi=10.1371/journal.pone.0066428" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=M. Sousa;citation_author=J. Cuevas;citation_author=E. G. de Oliveira Couto;citation_author=P. Pérez-Rodríguez;citation_author=D. Jarquín;citation_title=Genomic-enabled prediction in maize using kernel models with genotype x environment interaction.;citation_volume=7;citation_year=2017;citation_pmid=28455415;citation_doi=10.1534/g3.117.042341" />
<meta name="citation_reference" content="citation_title=Improving wheat production in the Central and West Asia and North Africa (CWANA) region;citation_author=W. Tadesse;citation_author=A. Amri;citation_author=M. Sanchez-Garcia;citation_author=M. El Bouhssini;citation_author=M. Karrou;citation_year=2017;citation_doi=10.19103/AS.2016.0004.39" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=R. Whitford;citation_author=D. Fleury;citation_author=J. C. Reif;citation_author=M. Garcia;citation_author=T. Okada;citation_title=Hybrid breeding in wheat: technologies to improve hybrid wheat seed production.;citation_volume=64;citation_year=2013;citation_pmid=24179097;citation_doi=10.1093/jxb/ert333" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Joint Use of Genome, Pedigree, and Their Interaction with Environment for Predicting the Performance of Wheat Lines in New Environments" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
<meta name="twitter:description" content="Genome-enabled prediction plays an essential role in wheat breeding because it has the potential to increase the rate of genetic gain relative to traditional phenotypic and pedigree-based selection. Since the performance of wheat lines is highly influenced by environmental stimuli, it is important to accurately model the environment and its interaction with genetic factors in prediction models. Arguably, multi-environmental best linear unbiased prediction (BLUP) may deliver better prediction performance than single-environment genomic BLUP. We evaluated pedigree and genome-based prediction using 35,403 wheat lines from the Global Wheat Breeding Program of the International Maize and Wheat Improvement Center (CIMMYT). We implemented eight statistical models that included genome-wide molecular marker and pedigree information as prediction inputs in two different validation schemes. All models included main effects, but some considered interactions between the different types of pedigree and genomic covariates via Hadamard products of similarity kernels. Pedigree models always gave better prediction of new lines in observed environments than genome-based models when only main effects were fitted. However, for all traits, the highest predictive abilities were obtained when interactions between pedigree, genomes, and environments were included. When new lines were predicted in unobserved environments, in almost all trait/year combinations, the marker main-effects model was the best. These results provide strong evidence that the different sources of genetic information (molecular markers and pedigree) are not equally useful at different stages of the breeding pipelines, and can be employed differentially to improve the design and prediction of the outcome of future breeding programs." />
<meta name="og-title" property="og:title" content="Joint Use of Genome, Pedigree, and Their Interaction with Environment for Predicting the Performance of Wheat Lines in New Environments" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/9/2925" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genome-enabled prediction plays an essential role in wheat breeding because it has the potential to increase the rate of genetic gain relative to traditional phenotypic and pedigree-based selection. Since the performance of wheat lines is highly influenced by environmental stimuli, it is important to accurately model the environment and its interaction with genetic factors in prediction models. Arguably, multi-environmental best linear unbiased prediction (BLUP) may deliver better prediction performance than single-environment genomic BLUP. We evaluated pedigree and genome-based prediction using 35,403 wheat lines from the Global Wheat Breeding Program of the International Maize and Wheat Improvement Center (CIMMYT). We implemented eight statistical models that included genome-wide molecular marker and pedigree information as prediction inputs in two different validation schemes. All models included main effects, but some considered interactions between the different types of pedigree and genomic covariates via Hadamard products of similarity kernels. Pedigree models always gave better prediction of new lines in observed environments than genome-based models when only main effects were fitted. However, for all traits, the highest predictive abilities were obtained when interactions between pedigree, genomes, and environments were included. When new lines were predicted in unobserved environments, in almost all trait/year combinations, the marker main-effects model was the best. These results provide strong evidence that the different sources of genetic information (molecular markers and pedigree) are not equally useful at different stages of the breeding pipelines, and can be employed differentially to improve the design and prediction of the outcome of future breeding programs." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.g3journal.org/content/9/9/2925" />
    <title>Joint Use of Genome, Pedigree, and Their Interaction with Environment for Predicting the Performance of Wheat Lines in New Environments | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"i8GRXsznHYKZHPyKeCosrrHgxSGUnSIYR-QS96QXTZ0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494053","apath":"\/ggg\/9\/9\/2925.atom","pisa":"ggg;9\/9\/2925","ac":{"\/ggg\/9\/9\/2925.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/9\/2925.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/9\/2925"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/9\/2925"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/9\/2925":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022GMY+hw7GJvmuQepCWt4niWRTR0rY6o0gUy0x7urFX0wqnU1wVWdJEgIAZSVUGux7aMoOcXa3S9Xs7rcrK4jPxrGbm\\\\\\\/LKEChNhhryKKht+yMWfJK45ORso6EXCYmUIrV4L23Rmln311zKKhAXQQ1O\\\\\\\/pmrld93ko6CRMfWhmG7JtVdd2EwI9NRTsKdi33ZSeVtmUo94F+81jZtIs\\\\\\\/inEAmPfz7dsm8lm13hj2RVqyabrAp3jCT\\\\\\\/v9CYkvNR2KW2V6sF4jSPQWv9OdoVqHl1PhrpYBUSWRa2UtdUxTMbDDBeFtZnGaFUT4UnLlKmDbXRMVBPcDLq9VoPOw9jFVb5td225POxWA0XzQuf1Ts4Hl2geQC86iXJTlurcMBCVvNnJ\\\\\\\/NuFbrUdF29FXicoBMkwoA748MqIYBkiS+JVj1\\\\\\\/NUY1eXLvCoiLK7jzTl\\\\\\\/m7XVWEcPHEZu3L\\\\\\\/9pRJjnCsZrkXlYq4U9+Ab30MAEByUVBisAwrobjPS6dpXsS+wW6p\\\\\\\/pzk2xeUunMqzR6H7rOh8tN8S9n2LVX6r2brM\\\\\\\/888SURNsRfWTjg7c5GtVTRCz1e0mImiG9PdU2mvz1HqgL5IY3QMY2BVXhGDHGmKDENRto9aW+nUtdJYKO\\\\\\\/CpgjeL+G3gVf41muIye+aTXZzbiPa5erd4SMNkOOvJnPF7\\\\\\\/RjKTaBWEVuMkMMqekUZQulgkPyUyCI1jr6lq8ZRd\\\\\\\/m1u8F2+EXvwiWO9yjC64pqOlzHV6YoCoRGUURvA+V89kYK2bqeJfuHCdpHvNxzUemlXzPl+I4yYZBHHRBhzYkytmUTsCNVB5fXAc0BkywkkDB3f9RJngRt739+Ky9ApUr6SAlwvCwYyb3SB\\\\\\\/pkBTP2WVry1OqEM9\\\\\\\/7IAaHXe8\\\\\\\/qhiKr+KfbiFCFxoRY73DDtkxyG9Stz7e1Ts4l\\\\\\\/Qf9AxyMBzvpCE2j2WmtQEa9ZM9gQr\\\\\\\/kpDJjEJxAOjWCCZPXUEafdjkI35kvN93Wvi4z3cnYonKXVRezcsVjLjEWY6ydMQLEXMseXljAKskon6tqaA2C1z6H+CH\\\\\\\/Xp\\\\\\\/Xostvaol\\\\\\\/vs8XTjCS0oeIB8LlppNjQbVfgEgqqZCK+9WxPD6P\\\\\\\/FmFRyTAYKb3y\\\\\\\/d\\\\\\\/usRPQKNrlKEIu+dxaBLRF5poYu2oAa3KA8LizD4gObU0MEgAeOHzf7lCBazcmAhYRCEB+1UuvLLEXQ2\\\\\\\/0NLOwoYTq0Bu6Bszn4ciBkWyPMrSfFXA077JWWMrcUw7ZoSW1523OU20sW0LS0\\\\\\\/ddNVAEGPWX\\\\\\\/BrZVaw8Uoubu\\\u0022,\\\u0022iv\\\u0022:\\\u0022XktbaYQMUqU2tr\\\\\\\/cHolIsw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f03ae9b862ffcaa2650bb5b09f8228033e5d3b02387b50ba57e49860ec5d1b54\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022m+9KyqOtDgv8kt3F0NnS9j4owlq9qlRXAR27BcnXt\\\\\\\/fS+72FO65XFxMWXt9s3KR\\\\\\\/nBHA56r\\\\\\\/mKJJnkxyoQqiFAXSACRDAR9Lxri8RdF3dppRi+PJsG5EttxNZf1w2LeuQnSKH3Rrk\\\\\\\/9vte8hxgs2CP+e5qiLR\\\\\\\/wAIWPY7llFW8ZbpIfduXkbO8cdvfrgC6KbqWeT+UwgxVyl7YKRyUz0GJbBJTx5msLgC32BRe8KSYa2j3MOMn2kI+xPoFgETfhW7YOn\\\\\\\/5sWhLfwRewiUAJ6jJiwF063gp3dBselFQEP3AMVPLwUWiwDz7OpfO3Ga5wNcesvFlQgb+nY9gcC2GxOPlWEfQM11pA2zvGyiy17XPjhMpSCJQx35fXJnGJI3x5eVRcL1PQ1bcWNKb6b5BYq9qfHBjX\\\\\\\/Eq8rqKNHCZ19\\\\\\\/PrqKAWKfPaYuj42Go1s27sae7xH1fsyLQm6dwnLV\\\\\\\/4aDcvymQzpWX7Wa4WCeuAAhqRa\\\\\\\/qIzs68FqrEpqaN6RgZFXi8RLovdI3\\\\\\\/p0R5EuURuhIrrJNHqvKW2FF0VKeGfpu7a53ky8ZTpHwOuJNapF7Hds+DWOnIBd46oH\\\\\\\/Q8qjdo7FFWKVNwWB3niZfc3cEuL\\\\\\\/b0ElLwuWKysTayod7Htji+zzriVN03\\\\\\\/w6TSIyMsdv5mck2llO8EmgXx\\\\\\\/mUhj66JsmtioHp978BKpKa+nhNBzry2RjJsa9WoRRABdFoykKuy\\\\\\\/giNhUhAAWbDfGPjcNTbrF0Jm1hErPwwOPt6lOmkHotCTWxX3X\\\\\\\/EMDRxD7WFDIQZFjIJn0doeL8ZW0KTNTvqgkB23gy8Q4Dr1PxEsXIQ91EPFUSvywKVH0S24NKh+T7stq3CE2gh\\\\\\\/LyKAkoeU0msKllMx72XECnIf8tcIXBKYuElPfZrlwsJz9C+NfzhuseYDqGvRAySyw0vKcr81lSFipJ9rsROIN2y66BwNnRVBgeJMHLqwJan4tDIm4yWtDjsDfNyA4cf43trWXpGW3fzrP+UwlLA1lu1uEhm\\\\\\\/aTB4weeBG9kE\\\\\\\/3yh6eN4kYRsHiocE0MnG2uuDKUMc55yse3k2A70vCfxIHTkaVmNE00eOsmNSFEbJHpwXGrenjMg+6J\\\\\\\/obUGe3QHNPJluRij5x6fSxOYU3hN+9NtR3lkudKV71+FcJy7PaSyxTyJD1Pms0i7BgP0Kq3PzBpevs8DmKIRNwpMczWT6bnluq\\\\\\\/13ZYnF5mWbl32bfesM24Rq3mVZHLWHGnpiV\\\\\\\/BbJJCEcOx7mZ1XB4+a9on7DTo0P1QpI1r1BqsXVIcccKH0CkUFwj8c7RKblbAQwp0hgWYvPMFRjWTE\\\\\\\/\\\\\\\/wndOkLxqheJc9OPdrSOBTgBPdm7DSbfpedFNFICU7Oo9hqIxNRoFQgiVh10OOurjWgp+7vHdHKyX\\\\\\\/s\\\\\\\/9IzitTbyHVCHU1OPoPU7SU2JeT6CdZxew\\\\\\\/mCC47KW7Sn8qb6eJ0b8cWT6pId+1pFKklpo+UYHSZiaZyI7hJgocwBz2jGbQpcRirRiuncwr4XZKNoUyZt6CgZYn6k0u0VCJZfkKoBrKA7f8oqLjKr\\\\\\\/R1fr3j2ssoR7t4AzbRMKG3OYgRHxr3+A+MxKDSM98Vgn51PTfPdHdjG8TZaCbhfao6+apr2XJSgGJNl7gTKMvpCsH8TtI+FV5OHLykblP+SDrHCK5zfCerxwbJD7JPfZM8VdwJF0+Rg0wBYr4l03QXR+uxRNmiP\\\\\\\/bc+WMv2CC+lUBH8TRY1NYrsrWDhTn3dt4fvRyfvIYAeOXTOVBh1O7iMYUNah0gmhTCH3UBKRvwxGkFS1NNDVoUIlgYvK556gSNWNXC2xKhjcBRfLJwc6NSv\\\\\\\/3ROJ2wBGQO+yYWrJ\\\\\\\/tYjq8Od1DvmuSHhFVoWGHSkl5b6dBn9KmtrGFnda2FnYPSiPx1YeV2aV57QHltY293HL2ZbLEDW+\\\\\\\/E4qGlitagQQ\\\\\\\/1aGmblur7kZfszYEmp30RoJLWVWsTi2jV3u5YFM073a2LJ8NrFUqpAqn9X63yMCaRwLRP\\\\\\\/tS5i2jPFMnkZjA0vUUHUMqlldzIjSm0R2LFxQml38vrGpryimmBGcr9VqcsKaRNecEy+0fSzQqLp6Oa6iV7yl4z3\\\\\\\/Tk0q8DpjrqNXDlbMp3hNhwwe3U2HU9PuOxDbCo05YO2AJjxW3oLX8gRMphq7lza3fAWcVCeqUdG5URHYL6aUOPealvsKyAnFKJD\\\\\\\/MIw\\\\\\\/KiYVjAXRUKnoCL4cW1mq1Lvg\\\\\\\/Cb3YNDJeyxYRz+1\\\\\\\/\\\\\\\/qTkfOV7is7zxcE+tDzscE5OzWyiDQfGhJfFhDv\\\\\\\/qm\\\\\\\/FBpDlAawILB3LMsq0NazwSZARtTwRizBdsv5VScbXVhZfId92MgNyHuJJFk5A4nL1k1z8HI2190UVNn4xsH15sJDeRhP6J09PenJQ0RVbbPe7pHbBvMesQ417iX9g8pHsgfFAvdG6ygxG\\\\\\\/Vl34lUB+zGFgDFsCB38+DINlS86L4lxEjDZf4P8pAOMW7pXUm\\\\\\\/0M0lfYpQ3+cFKg62uaas8bplcF6mjoLLlQKBFddq0Kru6fFEKEi6FvmEuwoDDeywm719n3IvlQQdnuKcEpxEeeX1cnm+iVO3NaQtjAG1+rFuh8LDP8l1Pz84nPIHTolhf8Owg1BdnH5OZg4w3NwOB\\\\\\\/9hCkS0hT1evKVeMDHD4EMwKk5T6f4RLbn87qh0TOWwqrECbnA20FPupwenrPkZvKqWcF3kJfdGmMEsvVFf7EZaT3Gv9xdK6eJLB3qSLoavf3LsfKqR2c3\\\\\\\/XoxiZOf00PJNRvoc\\\\\\\/D\\\\\\\/MBr1a9NTe9P5nEmtO+bAsJP82eYesxp\\\\\\\/5X0B97WoyzpHId1E68f1wzzxML9pjW10\\\\\\\/ulXYLgoaj9h8e5fjvLjTScEAqQOAtgj4XiK4aZZF\\\\\\\/tsQhnbNSNpIHuwK1JEpJDLYVgGn7xSQJA1KdTB7w4FsgccfRJGa81EisDvHnCNfb2lSjOS2a2s\\\\\\\/1bHfBoXcDHTKNlxbTn38lFNhC1a9HbJPI2yHUylVGLuzBAjKvYs2+KMSDruc\\\\\\\/MOeoZ64kUGE7OqVrwolmfIdjRLrOtpldN3csD25CBAvs1dfl+K3RR\\\\\\\/XaojXGDCLJ6OuR9FkXFURJaUBzAEOs3nNrP0+HDGgzFK8L0H1MLpYzJ9T4CBB5wKXm7E+rkYgnQDhhmkEK+naszqGqA8fWCwJb72lBVPHM23xvXb5K9mymHWp5hu3sjEHwrJMRqk85BVPKd2egkq9W27T7R8o+odPPuoblKYehIoHxx0m\\\\\\\/xKEjNQ2Tx4xyL6nwi0UxWE3Hsibt\\\\\\\/3O0XoSFfYR3lhK1tfuTOnlk8Ewequizyk57O8g1y5BN0OFAZYUEnLTrjBBviuIyQg0IQt2S8HZte0ZJr+ABioV\\\\\\\/XwVxZ8nSDGTut93hz9dYaZvndeAOc8R7\\\\\\\/8sqNednCu5w6z\\\\\\\/upqjvPt6HZHpCztX75sr\\\\\\\/JJj\\\\\\\/6tdzzLKTv+584iisUoGqqB+I4CyzRNJ6qmke+WJacMnBZqoAhCyDcKPALughukBDYEhyzkK9dPr6nXnBPTwXfGs1tGDdKJZ6LKPbD\\\\\\\/47UeEz8WRtt2zgnDCJK4tjvVpM8O85D2nKZJf12nwEnFB5rEnXoctmhnM13U969FT4SKVGwYgrGXiis4wnfVUHwHL21NliXWWvRXBR6IeNCnSyWscpqVf8c9bsIyuVjcHnkIjTuTpw8uiLq74IkKpFqZmQbfKTkYkeHLdIafZsZ5QLl\\\\\\\/TGUDrAOSb0Gw5gzmXgvQdpIw10OqCWOkIVueBWb0EBqfiqgOcMave4rHbkn8gfrrXAZvBfaDtlgj4\\\\\\\/ncOiN8RQHy9rA8vtVIOAEU6J0iQW1OPCoOFBIgn\\\\\\\/6LqQwf0PryxIvYdSYS99kEFS\\\\\\\/8w3s3JI6y0\\\\\\\/pJrsKEnnm83A+yoVI2MdFZ6\\\\\\\/pjqZFcBpUv5LMDTkDy8H+8aRmZ29CWD+D\\\\\\\/J\\\\\\\/N6Ar3okOpaIfAV5CoU2Pxt+So3VS97Je9xI0twWamQdl5EKqP3OGfNNpM1apXmKSQAOU2sW+dVN\\\\\\\/HiaSyezSizOwHVbqMfSg\\\\\\\/nVypRRvRe\\\\\\\/etmn0fuw0rc0WQNSPO\\\\\\\/bguudqsaRonsbiPw34PSw2l1YNITt1Su1AuIGX3hve3nB08ulGh6IikhwLKaZ+ze7NKbGhlbMNtD9KvjHnbLGtlqknLTtmIRVHc8Emr1uQdQ\\\\\\\/p7QGF1bNU2\\\\\\\/O0ae4mP8qFbBXHlcDWxY5HpjfAONia9A0iPxL5kEZZAaY5oukIMs1djd4Ee6bpB6s++O13sY37Hj4L0HK+9Q+XmAZ0iPGj387WCM5SG16cezcX9K63ZxgOFbl0i9fCcYx8Asm+1o65m+TguMSm9ZGoWoUnstub51BFiC1HpZiyNMelSD+QB95aGoHMXsN8NAlk0ReOEVO6U2RWOGlUUqmyiTns8YaWuDYwivh6o7JcfUDCQ9oQZXi67et5kJM5X+Y278eAadlGytU4skAYISsGLgckRRxVutwMvzDBrXL2rWoe0iZdYxMSmE+oLe8vftVl1ztVBNeXAcSW3F1hD5bsJYEywGms4EGMszp\\\\\\\/SHI9xHGNXnyZvTvtf10pMcemKi3\\\\\\\/Oaka0lPtFOpIYjJWlF1o4zayQ6VMfEXict3HvLz5Gy9naGuniLrclmwAVPBbJpI022AjQJ0SJ9v30vcJZovKg1HQFRCPtBHNWUl6UoHgYy4rRtV7Qxtlw0g7t16+uU46RWZrH1rC4aB\\\\\\\/D1aYZD8xeX0oMbgSxutQf09xiXbowaHrtnvSagTEArVHgP0PZa\\\\\\\/MRhT7g+ZTAHFfCV2t85GA9nnVo6SrvwsZDvLTDtxj51EMDE6s3gggW7J322Xe4+L03xnbOy3pUxU6xXOraJQBnn\\\\\\\/7s3cm8yiNbhXs4FbhVZpNU1J5z8TrVuxIm2Skr4ezrCxrGlaayPkpDUWLUoreLmV460QtogI\\\\\\\/DAIlvB8bRdkvtLBc\\\\\\\/mNW8+ZOgjJaSM5ZzvwBVu\\\\\\\/\\\\\\\/3Slzd5YRE7AZnVnk+0blIlHYKrobV+RkOGsacblGd0jnvY3dDFaOnmZsdQnTFFSRMCUHYSY99Oo2pctXP+PeBFBKlgh3QWdXnmcP6\\\\\\\/3+b30IefJgQSi8HmftHfj1xzCt3URPt659wYlh5zr2+snFh3vUMcB+8eCQJ5WcqLzbk1znBoi0PRl89x9VxNvNzqJLRyqFZJtFzcp9njgYtwwXVg7BzOHDlEMWil7s0vOw7y0VU5nynYepeMnWUBsntRbSyIMUrRZIh6m7qrWGKT8xSBia1vgC\\\\\\\/HrD6qDfq\\\\\\\/DdMPhHXLitMTkh1SsQBNTLzIb1KGG+9nmGthT3C0\\\\\\\/eWiDSc6piREq6X6D8+nfJfviD6\\\\\\\/1O1HKDo6ZOKOZC0\\\\\\\/j9lLdbGFs9SSloBNiyAsUNgMIQyoKDoejk8l85D8I4NoDaKAOQU0cetvf\\\\\\\/K2ApkvODe9dtWHLHQUH5LU0q0CBfPrLfUNst9xIrV2r+XOhGQBn\\\\\\\/fvbPqdtyJb\\\\\\\/MBb7PSQj7wfPjRMdr3ldzXkucrM5lVuO4WzKsnQsup2EBoaYTVI6jktj5tDMvUV0PraGbhA9K6TrmLwU9UzDvqYzj+Zsi1Ok7lWSjQV2EqeZ4Slz7Z2\\\\\\\/rIsSugXXes4b3HchFOVWM2LU2sXq2BaNDyMcmDm0xBFGsXv9315Q9+mCP1Qm6UaMNDrLM70lx0gnM4g8TqyQUSDw+c8\\\\\\\/Itot8AEW0+EPPvPIqzgXw3TmJJHkSf6d6C6zK8cgkecGBR9DEyxhH5pmt9FE+LVntKw7H7jmAHSm2vmo+E2v2UdQDFNClNCeBnJJJ1br5PIF7\\\\\\\/YrqHyflp+9QYxXU8y1utCXpIUz1JtQVBZE4F5znnQ\\\\\\\/jqEnt3gMPvwSGEfj2k2jUTndGzIzROfZ01Vp94Gcit0cXX+YMXhGTqSihxXknsb9LbNEy+VKxSjoomKvpCMcfrALmySGa+BQv7QucCRWmr533h4ot5K5Y0avQhwTcMH2L7XsAVvfBi0pauuzYiqmm5xrnHbJGeYh0g66+1+Y9LsxA74YGhzXiLQEyzmBlEruIgkw5xZUwfqgiq2D9oxxqkE89tEEComxPIAeS50vQWQJsyblg\\\\\\\/+Vardw7eSexbc8dTVzj2dHlv+nWfI3+2uUZCI2paN0L6oBP0KJnnq8qLGQI3LllCHgA5mUCpCyP7XV8q3h2kMSAKZfpyz+M5XTO31ICf6c6AsSmAVvZBl0yNtwfNDQUg6HnVBNyCSD6yn+ov62h9Y2iYvoPHx9M8PgtTfTPwcTBaM0oKY9qcHpceL37qtg5yn2RoKdnOgGoC4IJ1soRWjyFkGy4mLBhrnngRgA4RqdfN0PeTKFKLzKILS4NoofOyIi5Sh5gsmv9RZUG4piecakOHhuh+ZeKArtx\\\\\\\/V7X2YkjaV6Bq0Ylha93LDYpNObyJsCNL\\\\\\\/vt\\\\\\\/H+O6lekXYYW4bKAhPlp7Jx3cp42BBdpfbiToonHtQ5mE4bpNb0k0iGZbTUnYKeYZe12c4gRWCUF7MDLdR\\\\\\\/RG6zo25NX3mM8VkWYW2VYG3DUHYjUanVZMYa8IYUGJrmooUbpOFlx4189WCAYFpTi\\\\\\\/nn7av0jwunIFVg1hUT9wLXTywLBVST95eaNvQ0R5YcWF7dl5wkNhxv1Om+f7fbry7fao0lQQHfdHJIWR26mbWHOw2sfkaEwYq8VfFJ1giz3lSsBggvo9UQ4V6VfZw7Y1AElOvEAt7Oex9fvKx3QDl5azVQnVur00stypJ+7YMef0j486dIdMuzFi1sMsYA3+1T4hxTcfp2g+JfPcKxHC9i6YQkWsrxpFA+7LvTRplRF9pt8NmETb+Anj4XphNcN1gAQ7llkwMbZZrJqzOFmoVwxIKUyluhomWs9vWdWPjL9NsZOM1FmFwmJ08EiUCPB3yhIYaFh6uQ155o247MBdY3kZJ45k3aETENyhSlTR0Xn+s0rO3BXFHhGz8rEC8D2GLScIypc0DWDXGk9qNd9qIME7rqQOegRpWeDCLiNvhfAD4i6xc5rC5IWFPTjrIPiirgb+wfswSReGptHR4iMLQfJqaCTLfC+iESzLSCwmzwydJZh7JXTtA2MoI0xLd5RMn5pFpf8rK+uew6A01mbQ1106P6O84jw73lxZhXl8tJTEh5drIhfkF5N3AmhPb4+pu18F5NfflhAYd\\\\\\\/U4d2hs9e8DGvNGdFySEYU2Z1HVBDKjGlBsikTEQ3nLoNClTjqITUueXEwzP8eBYZ97+Jzvi1lv2t+cOqAnP6GDriUm06q1BzJ+8EukU1R3USME4YWXCO9npa9P5UX7tusZgDw4r6EsgWO51HlnnzTX07gq8esH45nSaksGTJhkUg+SJAmKpxNJkZ0eu8EGv9fkfr6nvFibMMaFUGUpNmJxnkJnaRr10IS3QJQgzTMgy7fY44TyvHwCxW+5Tr4la2\\\\\\\/GJ8gc9EIbwVGQz1xd31Rg+XCjzNnXgSCLnu5uw\\\\\\\/cqnQqAHZlcZFl2Kh1wENcB2Sge7tK\\\\\\\/\\\\\\\/83Wn5ktwDARlpjI6nJzEW1hocxGzAheLsJIzlVBIWd+FBd\\\\\\\/dj5dOUHYTfZdleyUjC8PaOJ2MMLd6+dEFFq1kf5GXIrpFyYLEq2TaFdzijYumef\\\\\\\/zwjEsmZMQjxeNbfPXxN5APkDsHtMOAlmBUPk8IItXDc0IifUKl53jbnqLdhYwkEYT2OHwW7hnN9nfsP37ceWfw7yMmHv8uRxKYHIywrJlnpIxT2zMYw9sI1AwritMl61MhHVG\\\\\\\/uFn2rQfohAaJPrs5554uUEUoUN+Vb8PW9gG8JYTBmJKj+tDP2Vq5v5dWWRv1o1hRd9QRt5enaiWGq7oOD++\\\\\\\/KOq56Ing4N3aIk368Cd2P8qcnZ3Y8espyko8UgP4Zdu8w0n6PRgyViTJBmzSSGpqumFy8p7P1wV8u7HRCkSwdWxb3e5tmQ4Zu8Ruw\\\\\\\/jWXqvCb+xsz40w5jumk21dYgodacfYs6SbHV1XJKLj45wp\\\\\\\/bbR1MjcC8xIq1JtxPaOhqeJOhRRFpdltmyohQ1BLljNbosOB4PWPYWKbNGKue8ThWpeWVTvFj8VBkWS8V6Mnz7Et22w+RP9rtsHbaGUnovO0\\\\\\\/WbfRM5GDnzkJgoWIWGoECj6PsajnlyDyuYIppv+yFJhIhC0IaaPIB7CDSmRYk3wFifWdrGtRdb6I1xpE+zpLeuc2c7BmRwUlxVUsWOsye0pzF9sG77oXA7gNeapXx4IM4ZUIwhP\\\\\\\/1eymR\\\\\\\/iAFtZOpjn\\\\\\\/+IrUEcHeMxF3L2urhskjgErJP4XzDnC0Bhmzbsj0trjyk2+cp1aOybgf\\\\\\\/3+x8U30a6\\\\\\\/ZhCHo\\\\\\\/5CEgn\\\\\\\/IifmZ41LPyBHN5rBFjQlyEb8X7DMi+wnsl9K5M6O7RJ517jwJihLJU2qy0BOabE+ADYARMUngx37q8iJT150gYfvWaT2vRe1iNINWXrF3ZD1u59xjdH0APjzOuSVNMNOJynRPMD8Kq4hVY5qbbn7HZDXgcPTFvV1BCM4RSSTEjs8+pq5VJhHvDgCFqSXTqWaBfaGA03cU2QUj\\\\\\\/3sUWjKzRnBQTlg0uMAx4HOysRzh8JKQOs6jjgdvJFv4z+YTbB+b2aEecJ\\\\\\\/Tgk86Lt2Swn66vRPrq6ekQC+sjafdG7ON969Qe1G6sk2gkvXHQk+RGGzREnlJYW\\\\\\\/qK8ZKYU8gtiWlXp+ntsTxEJXO+fEM5euwatC+gpddp7iD5iisF6kiio56IzQL\\\\\\\/0mFqD15scuL1DLAxNz2q8kRN6WP\\\\\\\/CuDkZA9ozfy1QX1W7WbGbBZZjGx6MaKa19FNCmsInn+OVubA011mOxWadZ662Ad37QwvKIAvj4zA\\\\\\\/dVPyPLkAdXug662diYXk7cRtN\\\\\\\/cJY9gIrmzM+WYS9GT+Qg7Xe+Q72rNs2Yi+vjeMCLNehlbLrWqbYABeISe5yApnfhQGzpBTg8SRmgS\\\\\\\/wVL1EMYBiGxDCum9+YvSUefqahamOROIbIw+2xyzgy2S8Ldzk7YpTJ99N75DWhYlbvtjDNrBlJEoTfzEoJhu3QpDAM+AB7omOrt63y+pAbLUoXeG83dOQIYDO9\\\\\\\/oylbqFcVupN3vV3RBw5dqzQR\\\\\\\/0tKDgTQMrp2q1HHU5jCMYg1uLGB0yzADOol2etTp2jg1Y\\\\\\\/6cx\\\\\\\/PfbmUj0vyhH509VKtdBKw1OozZeOW6at2JfhUQaAg6y+0oBjVy2D9sIXGoCLGi8SQZ+5g8WkhQs\\\\\\\/J33mP\\\\\\\/sb6Y7edstgP9Mg1L0z0P5JassGmw3srmV9K4kRUpeJ5LJWvp5vqQaWuPP+dBvcIO3YXFR8pemG7IHumZRj6EUSL4KvrbtGoFYBbr3zK+e0as4640vkj7RWEXTKF2Tmx4hulLgd9TeBtlZqi31iUCdRcnLTDcL+5lErSHgLpbSq96rGtFTVEQJHCRV5t+DR3rtabDVvDJr2hnG8LMZCUuXVrfMnVhx6hcKQP\\\\\\\/9uceMjXTBj0bl5NbssSjZUPGXp1KmURS\\\\\\\/laEPidihFA08waLGhMwdX\\\\\\\/cQmAGbWJtjqCmvcJfBv3RExdg0K55uwtoQc12Uip6213KGD6\\\\\\\/6ffkpMgxibcjsqK83BCkj4PIpv0JuLuKEYgNHWU4n6keZ03OtqdNAJ85qwQ7+nKQ5b8NJKqh+zS4C1R8TNKuOLujgcNdkWzj83AO7CBa4HDX7Qs1d\\\\\\\/7CdXoDaYNtvfr7\\\\\\\/fIsYcvjEe4bm\\\\\\\/GY8PnN\\\\\\\/GeDQ\\\\\\\/ZNBZtnY\\\\\\\/eoHKzWXrWetWtIXmbvp1APt1KmUTFgmkZIz0HzgpTqVMTgh8oazz2IaiRYcbB9o6yWbQXuOgwxBRF6r9+wqjmOELjH1wWeiPfsq7VEXnbjI38yqF\\\\\\\/QTtitzY3n9uwtKpLx8FCieTRhF02kE3BVsLb0mIBbRGR+0M3XAlmwrofeN1zuz8cI9xD09J5I061KneMJcoCD0p\\\\\\\/Izj831o2vHp2uGSt8effj5f5e9pZEvwVFja6brxHTDCGzM2Q61fDE68fjn8jV8L70MIQkMU7h7aop3wI\\\\\\\/ydGBh5HJpwC394i6hNHimekz6AQ6pFN32ykr2llBoCHtz4yA7M0h4KIF\\\\\\\/VyOzrHBXl2n8PeqXAj3jSHiFTboqL7spW\\\\\\\/AZtwTKHnwdECRqzUWgtut7p+EkaDG84iq9JGBDeqg1byfl5+dm650WgozWmL9wqziG+piHhJxGwX\\\\\\\/HvAoRNNR0B6+MT+TvJJw1+YnJWU0WlqerPR5Y48hCwwMzehSVx1YlKHd0IX5Owa0vkNZXbGMNoBWDXA64JYQ0gyjTWksSgnHH7Pr\\\\\\\/fDzDHPxLEYXn9qjiUfzEtcBVBM2h81yeLpEl69r+f5pk+eK547rPvF2mzATThYbm7z+00vXOExG0VzPcGgCTjxTrOzl+Per\\\\\\\/VloGJgve0gkOPP7JppK8MDzVBHi0ZNc1OIzyWQWnwi6\\\\\\\/qY\\\\\\\/PHyj8j3ZC+a+\\\\\\\/LJzGSi\\\\\\\/mnY91El7itk0suogQpBJdi2J7FuC7BhJoRxcs1LYAvopvYBxjovoMsplmvLn2Hsc5InHLrxoUBwi+jiTGvG2tJJ9ZDzGQfoJcRdYIVWmXd3PrLqL2x2AUy9f\\\\\\\/zHGvCBMKFkxbWARilPjDAljOUcuWT+dvZtb+h2j3byNpzDFCWu1tSbc84NQSIE7z6rv7OqR2lCT2J9sWWtNxoR\\\\\\\/KGKcBlUW\\\\\\\/jajiVm5l5U7QMKblIuaX24qobqo2lyMz7rBhSRmNMRQwuu32NskJH92M8hh30gF5p05ifdvAkJIg5IItY9A2M\\\\\\\/a85KHHYI7+mFzthj7H3Ic7KFuqjGaVq1231z22ef9d5nD3T9mb25i9GLHIWXGM9v7dQIcnLYw8kMJXtcLqGLimDeoluQkl7yKu2EKshNZmjTusU63nZdDOydIzDocasmgzLDyQRN8LvZUACkJV87HqWXuCTqTanAzA2lomqON+qC2XYiGuwj+7\\\\\\\/nof\\\\\\\/a1mdStQnVe9OIWohM144w70xjRpWgpK1eCGGiXc6GIAJwzJM8g7a8sioebHDb2ecUDt32kjVIAroJw9t1js6Cr0ADOrjMH+YwcY44\\\\\\\/Xsz2FtoTae5prMclkPxBBgj2VzGCJ64kNujOC13ocaoOUShpRCl4uYqS+g38xLFem5zVNQeAabSU8I\\\\\\\/mp4HYy83UJqkixGHSkgzXUZhX2lmb\\\\\\\/M97rnJR6vtubrXmas1HGTzgnkHXuSbrCSJXSUjdYqrRxfdUM4ot21B2qgVTeZucqHGxMNgmvSeXv4Ra2HwEDh9s2JE\\\\\\\/p39OgdGtqQMxKa7XGQ8HBhOO9KPvz7LDX86pjzYTydPTXpXRe5G18EG5F3FedAJ1F\\\\\\\/2yyRYN1Jsm33L3ZslAHIO0kSGXgjUYGCmKjVo5ZMX62WxJNbUU\\\\\\\/xePSeDca3KfjjzuqAX4IF+JW6W3QqbdNMr8vDTKkOA\\\\\\\/IVsQxf2S2xaxMonVKI5QmpA9W5olzl3FaMqu7TQY2lJ1qPDdOKH57V\\\\\\\/Uiuw9NeCsQCNsamzIAzawYYaP4Fknpejv5FcbcG6t21bbtihYCJ97CT05jkT2rcDrStkpCu+bRygwoKb5Fu\\\\\\\/ZCp75FFMshsOLioSEcnWA0Imiv2J9AmF2roHL\\\\\\\/YXZhl8NXRvJD91FultCYubyy1t\\\\\\\/+6XllB10PclOjB7AY12VJ5fNa0l8cg52C2eK0hHQVnyiVD900uk559W5BoPpD8NCwwW\\\\\\\/qssbd1Am4auAX2XrYjRhPlHztaE5HXZP5J5esby8srW\\\\\\\/ls9xNkm+ynx5fTudOdTt0C\\\\\\\/YlVTZosW1sBgkPpiWadVM1oyUZhJnruRigdIQ3rlBjF7n+mbZ9AfoWtjSokE+OxZ+H9tyswLCYwahSwfISzKKewHf7mTRgGJWehsj+fKcPtLGAwSTg+4mI0u5Z3svyddO6aL3w96b46mQebbwxg5nN2I4T877oE7OiJqljGKaRzz1lSsnLFwtWHtiHyBkB+Q+Nbj43noNFUwvWZRvewNxzQI1ydP4o\\\\\\\/BHrPW5w4qtIWxRmdH8HZ3rgHCjFSFXRhapdhqbqRxvqAlQ81A\\\\\\\/57PxKrHCCQpvFyo5Fb1v4c9oTQUoI3MVbeM1IC5k12Pd0\\\\\\\/BHPzaE4RjJkmoDx790kBEVrUL9lLj9uJmQwR05QvO7c5igdX20V0dNabAlbP3LXE8ZP1+8RMHp31QN6E\\\\\\\/KdQZP+SVUJ\\\\\\\/zHEPmKe9+NBR9n0Lz6e\\\\\\\/R7tL51wxQGwmma1ctDRXm42uI8NC+cs76I+vKvmgWflomuRHvtgeOkO\\\\\\\/5Mp7MAecWlWXR\\\\\\\/DyGIbzolw3\\\\\\\/qIfA5cVhx2ErXtxt2B5sDx0Abr88JKDK9V8epf\\\\\\\/eogt+aUh957L0\\\\\\\/PbhN\\\\\\\/DJkGHsy+42FAECs5YcSGgNID3nRKP6C5fp25I41fZMmk5KJ98a9ZG7zJ+jG3r0IaMEeYXV\\\\\\\/fO8idJvxLUN9gZZ6rRjyEf\\\\\\\/5m5aucTKDAj8XZ1tD2n0WUB0DvZ2kG1P3QN5xw+LZpIBmTbvs\\\\\\\/4mgxPGXJefZ6QOQpHqGqWEd9noizCGNkWy77mVztZ83Gv9u0LtCsjhrrOygriOFQO1UsOHUXOLc6K08mRwo3P3fgeczQ6SQdjUGHm22+l+\\\\\\\/wbXDnaHLKELml4d6EmjaGhqZudZ77Bb2TJ2A9VTsTW\\\\\\\/X0QO\\\\\\\/v6g6J1ndQpI5oPJmSRmHH1ooEQ772WMfHRJt47Iz4KudfQv0WcDWzcun9\\\\\\\/E1+EMV6XEY0GalfRxqek\\\\\\\/kT5uS75zQhRgBOv\\\\\\\/ixjp131lC0jY2SSJ4cpwlxDK1PSJ8LTfDqm7ii\\\\\\\/yLnjAWMVk\\\\\\\/XVt+0xPlRryPUZ7e7GTv+Lugw8XuOb5ypuX5NkfDviqVhQIZoJlcHtbg4qMyX4AJLKEcYzWEyoaSehrJJXl5wIhiqpk96BH\\\\\\\/OGLPcelsNJHQkJjmo4irZCL1rCNi3VHR\\\\\\\/t76666p1h8EAunzEKHGiBD11K0kfcuMfK1D6JZ4w0dR+3L1lPsWun0iKXAFGR0\\\\\\\/OzB+ju7xyVEpOzxMpoceJNlreftRSL4TpHN+jyCBkYOeLnoZG8AWrGpFs+7LzVVnGmDk2LHZE2qggxIineoDN3P6OF3x8liZ55qPdVlakiGT0MItCSl9\\\\\\\/9DB1PLgV7oC3rNoFn5W7tfocF2vLoQ4307pzTYGx1QqdqL1Bb7nbqROpKWrML4UMtHEKmwrawThQuSBf+IyQ7AOghiso8D3G1qcFCbv++ZU2AMaIEqthFUgAYT9s8GM0cxoL2rqWhfx5xeDvutwwemJYZKK0TktgWEVTNgmbXQC9SW7vpjYzT2csU4U8oKjautzi1MNl0g3f+Pzo8DM85uzOE6719u3jGn6q7RZQfy3vCDgEPd9rJxBNMw8kOhLU1sDKkZbenGF+jGqSVsUJxoAvXLVBUEPq\\\\\\\/57AceHuqvGdSkqkV+rBMRAZmruLODypdAZn\\\\\\\/qgdf4RUMzBDgXBmsjVk\\\\\\\/maLcPTHXnZXDFpf3Rqxm2Oji0Q3k6mmyUNqSPKtYUsx1BSY7cSoLjhNjTod3kYVseXk58j+vluzDJhKHtCtMqM7Ymi79tKhtqIctwJHKWucYfIPQzCe0sGpnqCMN4eTejwGEUh9bGByq9e7bnNMKh+DS4Q8uSMUMGjRVPUGQgUljOwu6ne+O7BWGQN2Njlrw84z8X34mKAYncv2cgOXyr8tki4jvamdEwK2cc53P+sl\\\\\\\/JB530EtTF+3AaMNWgdgOlLciHXtu3gxaQTVXFJkj1bqZcV7C+CUa0BSJ0mr+Oi6ch04rXbIqZbxl+PfNl15GgNQgGeVdecRvaXbmeIA+TJEMLM1c3zFIPDknMZW1u3worM+Dzfc2C2ddfrikxStn2GshUHFhoOJKbCVekhYm7svX8I+RRnwPwDRkHqTcsY\\\\\\\/Rh6J28awFviLlg9WfIoHGzBTHSiCPcdQL4vU2lG3pfzcVZVhhsm01x\\\\\\\/yn386rZvrK4ypefYN2VrgA0lTiE\\\\\\\/j\\\\\\\/IBAl+MtKOLRutVDITS\\\\\\\/OUqQZ1SgmRKE3bOtBMKW2Qx3A1itZ2nol9BbHi4WCNNtH+GQ486rEDjVS7EoIcCJEcpTND559E+XiHizfjBTeFWFYbiUfCrF2BJEvgFM2NSi1h1mMG+lvspX4vJDkRC5F5WuFr\\\\\\\/kJk4yFqKe4uUSO+UgxUsY0FeRTrw5tmEcVHwTlHOvFDzG0jP2nOeJ1vcaqYD8AQ9Vng0wdOBOtw+O\\\\\\\/Xu+8ViCwDb2dpAtzc8+sHUmc5uLpkEbdU937gs36LsydxiKziQKviGU3CmbP7K61NqTC8pAn\\\\\\\/XesWsvJkwSI8DCCD6o2XJB4QkZl+vhEa3OBPCoRmZyQEjhqB2CynFhOjh4ii\\\\\\\/XQpHb94F4feRmm0I3zia56PHCStajchNKCIeTJ5pI7eXE0UPcpRLGiSyG3gRvXNQjuQqPBhUfDSWLNdDdf9ku+SeO6eTPXOl\\\\\\\/Nt8WxCH7EK2iOfGYjdSgRDjwRxGP9duVe6+21i9jG1+JfyPEXaj294C0rzPdm7VDPS6awWRJfMCbSimSOalizYO4LfE4jG1h8a5oMPL9EZ3QdlVEdqAAuNTx0MjQoDGPtBemqyc5MvAsDJQgOY4q6SXyEGHsK\\\\\\\/ziWndOtsCYS1gWtMle4H72hQZ0EqXHLCUB\\\\\\\/hY8LuBoHKhVH3i36YX6zVujZ9xaJ+Lu8++3SbuncmX4tY1gc6uulFf7LELxlgcqrgFpa3yOQ\\\\\\\/0Tse9t3uiDcPiGQm1v16RKCiIjpFL\\\\\\\/VwEEm\\\\\\\/Oa50soRhPIVT54AO1gF3Ib9ftaF0vnpJNykp1aIiSvNqWyl9W4cLkvAvwoFBmjvcMnfDDAcrscTitRpgqiCJeNUEANJt9ZcKGCmt3JIIeUwxVcROSdJ2WcRVN80B4ZqE\\\\\\\/tqievoyefAYq3mb3j7nX0tIok165YcHkPDFSekJUBFtia\\\\\\\/zTpozr4\\\\\\\/bl9y\\\\\\\/6syhpmlTnW4VLtozjTYtdatW9nDynd1dx8F6zVxXX\\\\\\\/CpN5JR6jp8+uPHB4lIpCyCUe8Q15Utv\\\\\\\/l8W5aNUZRjw+PBTpq2X5U4zK5T6f\\\\\\\/gvZ4Jy51vnzsjvtJ9aaVsn1CB55FpHeBZishMrdFUNwoKXKkDTSHLStZIRz1sr12MZmOdDODcOa8f09Nl3r6WkNM9cfhXxv3tDs9b5rCXpN8i7xLKUh7OqYf6ZeURT9YDy1VedyIuhLBGfktoao24L6rtseiA1w0ZsIoa8fkIj7p7Mpzv\\\\\\\/+hngISDmTWxvTu1SiVP28kBsNB8hNvOfIOiweUOc8NkyHfmwMw7LbM0d7UzGz1TT8GmVgZCBG6fiZH3DMuK9OzVii7pi9gCkR5wuwY7uy61x+i\\\\\\\/R0U1Y8+PbV71y4dtQHTTrSgw1pug+hbxgpJ7qBJgR+eC7Fhb+2CBi25XR7O2wjRefcyl5RW0LA\\\\\\\/x+U20si\\\\\\\/whG6oUn9wDR\\\\\\\/5MHfoUKb44s1dQ4eUGkNfu+HsX\\\\\\\/mbVy6RcVmx4p\\\\\\\/dC9MYkWABhjxf533Z+o9jO8XDS9sr8NzUDi3YUj9u8Q33doHMI5PzkIvRaGGDSaGnefStZSvH+U2w6QeIwAKQFGSsXDuy4cBAqLLsfSxW3uu1g+pCMpmLqoSDjkney7yiB8oDO9rEp8oDAKGRDkKgLMlGU21mwbXhGu8DjHlZiqhz9zk4o6xHFxOnL+E+v5kKcLjeD6y3tN5uXS9rfgPU+vfE+GWEgI37gE4KSV3gh1ij3Hb2WfQOgIVgTh8p4uDUWeSrPO5RpC7Kj\\\\\\\/ZJzMLL\\\\\\\/frQBfRR77zG4p4q\\\\\\\/oXDBVgPpB+X5ipmq5uw7kAB5HxWrcwXwoiE88LknOYeGajclF6tjRCLunQX+7Rw+hF2fPZNGNm6G4OHubGG4ZpaRaxgm8Px4jET3geoVySmPCSsgWlNiX8UwOVBJTlF1CaAlPnxv4f4G+Yr1si2bX\\\\\\\/YRwD667LHVZl4B\\\\\\\/Kh3kK3oJW7oar5j6tTDqbW0OIzMfgr\\\\\\\/I7WP3Y4Abyf+fMsSIClC3RkrIh3L6Yn904fs5yNPY7d3b68DIHHbhpIo0XkjOCULCaN2HB+w5IGmci6R65Hl3JRPA1ujcNBq19cm+5\\\\\\\/csNelKStwZyKVPwNoDlqnGr2SYLzLOc3Wtwn9lAOPZL\\\\\\\/XMQWHm0mwEETLkaH4NyK\\\\\\\/al2OWLEefe5SeJhDiJQoSQN0QL3bqP\\\\\\\/yRHLscqwNWSdBa6IoH6kPLusl8adechjVAh0+VDc\\\\\\\/4MSCaayEahJWeclDdg2g9zRQmGcxSt762HRM40yVR\\\\\\\/kLUNOQxnkvDYtM2JdYCzc6GQAzhTTSAQy8PjjN15JbLb9166JtEQXs2n9TGUP+a25SBbmtY1FUdiOnh9ppJqqznB5FBzixLDMcASr6gmv3ao5huGOYq0oASohoND0BI3XuBpmdZfA0MwMYgFrMK57vLeYx7Ioc+gWE+wa04cwQ\\\\\\\/SPbH\\\\\\\/Uy3JweqBqjP8YSzt1VHzG1q09hmVMdU\\\\\\\/vh+q4xQgrK+J94tkYTmaDMFa6WHOM5oa2pEVc0Wru7puMFmOKMxWnrKjOxt2eRMmpC1v9KrCbHd1Z1slH+hwl0vJrCuRryBynCmj0ZcZOppTeh3YBSWhA70jo2uU9kn0F6dZo0anma0I5n1Xb15kHDugQQsNuRxATDqVXIhSYJr26+qaYu6GEjO+ZQ6u3nlJIX57qXZFzykuzMH6I1ivMx+URKer+85Y5DmiRyE+OA8o+nvGWk0p6K9lwOk4AqQ1jwvoFddVn3+VD+K03uhbnDGBt\\\\\\\/LWc4HDiXp39s8Kj5N1dxiS7lwq+BrTDYL0G0y5dXOmR45lUP9D8OvtjPSuWRpsuSBaJl47vjHRY0b3pUqhz+3PhNzRv9\\\\\\\/RUxkDeHoYtflkIgcB14+IjVKeoTMguZHluirUfLg2ondmXDgxNFLg7RwBWQaEH2BfLUmtXLimeeeaqhcfMF2hDlomJvle6OHVI\\\\\\\/tWx9e6rP8PY7vT8rORZ9sMhbTVifv7HnJ2qijobDoVHYp5HeyxbN7N+X5ud4K4hCbSd3vrovpcHRHSD7TkRvBvhAko1qrEjhOEu1Ms2EAFcqky+I7WY5ptMHmEBv7Z9IhyPpklUJBIKqc0qpiGT964H8lRg9x1c8LaQtwI85GnnOwf\\\\\\\/lDrGKHHGwTPaCik8g3+FGbVFustYtyMNbWDapvjIP5o4g4O3U8ZPsunPKGuA\\\\\\\/V6RZZGLeBnvLBj8Mqz9WawCY8iiXmbZCxsEqHPcP1GgIYr3LZnZyDC2+VnZSJGV5kRqJtoIJshaFdGPM+cEsFNskjBHqvPuai8uuSH83PjYgdhGBa0x8yxgrgfnl1UB9zzD\\\\\\\/JviKuWsE2fXIS8baj5IpGuFu2p2VxcbyiJDd0UtYzNqzJI772BsFNRITc77Blb4UnBhZg5vp5KH2nBvhM2LYpVQLrtUuoCt4pFafKWC6bTk+mTtVQSF9hSAEeEtVPVnDns5jWve+QRbpM1w4WjUMCH5hO80Hw0voTIph6XldOpA2XXYd7LM6FddFC0PzlHpKWFGsbyRVBauh2y7C\\\\\\\/yor2O0Hm6woXnVDqSxhcXTPI0pz7duWq43\\\\\\\/XxKUD\\\\\\\/Ul0q1d1krZCbydcfHRAnYiJUEWjQIRSafKNjr0MA7BCC9gtSscLQ4b5bPG2WCfZXz1NUM1+oKx3MRRqltl\\\\\\\/M6\\\\\\\/XNoHYufgiykmL+gb75EZJQ0Dus4rLvIBbs9jGnktPk+q137k0go7R1hb8fMmCUKKGaFm4yZ4cQs+GoQ5Nk04QD0TQw5xmUnXWXK0su+az7HehMOMIlNyRP8YNcw0MWMa5vvT34cgsB\\\\\\\/YJMfNxvfEzFdzg4qajTkEa1zd3iZCUtqH4VcgD8qBUwPHRz8krZnktwUrZJ5uxC6HMjn29TbGjBDUiZU5SV3dKgzFHn1GWNysWQjgDcbKHfrdVNf3W0WNcS4SCCiPRp9l7+n2DHcHDEW7LG2U9u5vrKXtGsAwZG2nPLCaizUFSdYpgxID7Jg3FkNdOHsWq6+k7FOSYjbuxnOgSvFZbADQYh31FfXq6NQVSLxxYi6hcxkcckG3DvnL+LrF4UCBxd29v+ZyOJJfQbbrs5Axux1wVxi4ms1k\\\\\\\/1mULGrepyEz7HdYNb77I670JpuKD7Gtohe0Jl5uFjDJ8+9jssT9TBr83oOF1WQ5semNHXldVuYen0d7aL5l3KN3odQNNY1lz1HoEvj3KVStunTME8wOL2Ap1\\\\\\\/CRF4LcwkuHi39TKrOEfppNH\\\\\\\/ySKU\\\\\\\/PJFYgQgVLI+\\\\\\\/14Z4UNZb0kbk3mwDIkii9mHbi0ENnxMv3PNDMVtCX0CPUftXeoVB\\\\\\\/tV9k4z62aq\\\\\\\/pqSKILBwD8zkzhzl5rtd+159BevsmjbXOyoS\\\\\\\/qkGJM77D7epzx9AamajYJ4NjcFX1Y01tPBOWfcgnE3V5tie+Gc9jwl90SZhfgm+J9NVxyd0Wxn8ry0BxMiW9EQlmGuRz50kp9TZmPQRZeXtLquQ6zOVRtoB+fcrNGofKsNQ0ZIll8Lo+j+INXQtM+hdp3dAkAWFfkZUnCi3T3JE4QGZNryE8qjJP4vbGL0x86GWh4RzF7jzde+zro89ySiakSk+L6hzYWAl0BjytR5pLE0hA4pbpD6ANZDUPmXkCSktVstiahx1zzaOfATX2Ouws0ju4PeFGr6YiKpZSx1zv\\\\\\\/O2LF\\\\\\\/Dncr8SADALpAmLlzbrwLlYiSuCpjvrdh9hlmEvtx2r0dVsiBDfhqS8u8f5NDqhl6NlUSSVUM3ipLRlBFUWskkm5IsZtsHKDbmIaJSy7CSbyEuk\\\\\\\/j49KyXxMl57auFsoMIebrhUnFZV4M1YjZEyBgPCKog67q4KA2XZtaD2LEeMgc9o7ptM7Vj+UjHsCJ+mgTF6BiDR8cG4YbW2niblE2VKM1v5ESAq5OAuDpOP+PxjTrTqzzPjQBFjVgNpZogbOHMBBNjF2dor+aug+KGKG5kD2BcR0Lf6VzHvIhXzdDyyxwwAETV\\\\\\\/z08mI3abVy0q4yRwE1PE9BJMuzQ\\\\\\\/jpxhWVu6mG0qbrUSnscDq+ZOdywi+kP3TQ+CKwX3JTH+efqnCpBYhbRbp5CDn11p\\\\\\\/aQMF4cnN0KCB9yjjW2RZUW9+eIJnO4KZHDVTZpE+betdY+OcXiIZKJoVgw6M5J3C7WGZ7RNii2GJ3SZlGoKWitaaJO3UeMuHELdCgXsiVkMo7uV1qphVjd3D1UAT7qWZCaTJTkZetZwxUnSWMtlX7KrfCS\\\\\\\/BBvC1mMn8Hpy5ju6HSz8XtS1dgjaDOf4lrBmt\\\\\\\/OybCYoqqoN0yqmEyWOhtiIG0S5yx3YJ5dBlzCHIeWPqXoZPsPzWj9CcyRkG\\\\\\\/lTGBpp9CMsaxSRrjPphYEK8p86JFwWE\\\\\\\/f4sZChoBgKXr0YctQ34XMJaMNfiY5uDIwCgzfo+4M3Z7MWWnTT7YyvuSLXCeThgzmWwzhM+J5fH3sFeYzkbCvS6hxk1EjP89z1G\\\\\\\/RYgT1kDHnAA80DTszx3J0cgaZqKeikYtuU4DlqrnC8rB+C5bGiOJ8cs\\\\\\\/UmFIx2t+fhSbaKUrBRSiIBiNYK+eMcQHeF\\\\\\\/rbCi\\\\\\\/ka5Nw6zmDOUGIHOa9j9oHJauBQhFYDpQ2aQ\\\\\\\/HYBEs\\\\\\\/fTUAIcylKlQHSy1JelAOlD5pd1WW0LcHeLhgPWXQMtRoax8tJbkCsUIqNJfy4hgSJku1m05UuHpCZaRgUY1aPAeFz9MsHk1rwx0GB6b3kRgFkRVInzmDAoCfiEGKMkx1So44y+T0vXotA9UTuDowve+ASn6o32pAJvIiS8sU0xHHXJUTXH2IsjIAKn5Hz8w9hD1Wr3SrAjOTffAqFgQml\\\\\\\/6jS9DnquGndFVTipIkZv+6PnSPCovO0iSZAqpNaPNhMaqeL\\\\\\\/lgfjgcDlDCxyFdk05UaAHjjSN+kOEmR+uUp4sBDbNmeRMbJAgUkpdFHoBCyY+OPA2Gg4dHTLLMCv0OCtTlK1NE6mmvHKL9VkmnAsCXMjvXXOBdHkSD6OjA33cdop4tzgk1GUkYIyFXfUdh9LV\\\\\\\/\\\\\\\/fwy4i85WhcYL0193o5p6GhJcP7z+tiMBajGPsVDDO+5eeSvQCM6kEyDAFpm0Cusb9oYluEAwLAsbq4WzFL2oh46HF2A1\\\\\\\/zonr07XLLnfIqd1UJVgrm\\\\\\\/zyJgW+gOz9Gyme8S5kgJBLEGfLowUerpskFcyeGMK5qyC96bV9nJRguPVG2AP8WJhBJrFihFs5ta\\\\\\\/MajPEwu1NsPHsRKfSipGLe19yKw3XEjUixX7fXkNQkW+v54b+ElEEADkOncUITVf6w1jYfkIdgKgvaYVpX1LIQFzPozePSfS3s1DIjm\\\\\\\/0SgNS6doXfcGD+aGSD1zBD5M5EgIvGSnKOpRRFFdrek6pRdSOiFmvG\\\\\\\/1tpe5CBJcWBvF8vLUnNx1j7lqRgVl54dlB2LrMEeNQPSx1Evoud9J94grKY7+6tMl8comJr56zEzaUDAeET6kM7clxBcKiHH6KJWuPAOUxVG679iF3giRfQUCYZ7dBxOYgZ1pIWHlfawBKevsG7gyJMYQbswvErTkye2w\\\\\\\/hPpxQjkzEUuoZKS\\\\\\\/ft9j046bfyBdUrmfvWRbe3hKDpMemY8CoSHnlwgAWbHkt8\\\\\\\/QzAw8+\\\\\\\/Hphcso+M1T+ey2ckfxkIeSyvjyFYTCEKrGZHcsc6OQ5qFwoO72fP4F\\\\\\\/g0hK6uvsup7CIlauG5v829HINsEaLH4qFmW\\\\\\\/OncXatsYGzwph+LeSpmqSH9+lmdkfVqUcS4qWHA3Mc15oxm1MF+Ivnxw+hcapoLfhzzL4c+dbjTSRn1NB9x9nCBEqihDtpBjor7ImSPJYzcbPNsgj0m2vct0akWXP44wUuey0cOEsQmmSmgInLM7Sh\\\\\\\/bh\\\\\\\/KHbo7p6Ba4aD1DEKqDqtRYFV8r8YnPw\\\\\\\/Yg2q1qJaAPG3xlINgSvDXEXadimyLRUBqYh0UA69FmPz+hxpkS66lp\\\\\\\/c+VD0broXkpi44GcHuKjSGs6HKk5OMz\\\\\\\/N0WAK6hy3OZocqgpElUpL019XG16\\\\\\\/itN3UAHc\\\\\\\/mmVvd7DJo3EgWCvhRxpETvomoZ6M\\\\\\\/uxS\\\\\\\/mYALrvvm5z7IprY97+0pZ4Z84D7eMyVOja8uzwI7nKkwoY8qgun+Eh0NADSdi7fvPHxBSBc2CHOj8+npdcK+iFUiE9UzeaM1yhRqTcG6RjNH+yh9exdcnsa5OTyEtNkSIAM+zZGNVQ0yebe3XS8X9M8bBjPDRHHkQ+m7Xm1gokoHdtxsUvLehcbZTeZCi6VF\\\\\\\/huMhPbFIu1An5RZ1IyGXTx3Js1Fr48UaGnp2gKCZG\\\\\\\/kJV8AfB7K\\\\\\\/IjCEXLUuscK\\\\\\\/kAmLIOT6CosmAEqj2bgbSYZ+7Ij0AHMF\\\\\\\/NC5VAay04P3X\\\\\\\/VWdED2j1v1492KYADK+v8c7vHPsMVfEu1sNnvsFJ00VBWj7YyzQtJ8xeBz2WdyO8xBPE7j3lM3kFiz2\\\\\\\/C7KFWaDE8a229DpRXMTD+CgP424uHvYBu2tsgK+ZEwys2nPybtdqrMhbSJ02Dn2wVn0uFXQdEjhvMqY9uxytY3EPy\\\\\\\/RhRPBzaB4pjpCvZHUh5wVJwEnQg7OSTkgSF4bQB3CfgsRqkiPHX6xOpHF1S4dOhBMBk468SqQUBtt+EHQs82P0rPA91zashLiu1Rfbra9Ku8lNe5Q2uG5yULi6XGjEWepYUNjRj+a3wsJvnsFfK5IzdfLySf1J7lCP3B2F4i4VivcI9SYaDhI7NWQ3QZZA9zN1y6+QaWRMsd2ILAUNv0ZNquKZO4vGBONBxTnvEeInLisWe+4vq3MMKb73TyqgsUfyd3DzvI0YIMlKdmTqi3OUnoT9btaCOq5q0+N5TT64jRBXDfprLUfplPXKVeyeVozfYJkmfw+Gmkl4I4bdK0DrsGNxwuJz27Dq3ie+cGizS19plJBHWA5674rtdsnWWt9LQmVZHauHIr9iIIkOnmSKuJaLxgJDkpIC2WymPFAEBwfOEHB40o+Gk5UFrKiXV4EEoxH39hAceZBtVVumqLAo09tJ02nPToE\\\\\\\/erwiFObJW4Hb6PCDzVcF4w62Ad4PphY6AJhDPoHfDpFH5yhbYXFGEzAyApgg\\\\\\\/9MDzkmz1ZBQKivva9r0JaxXxbCGvRoRxaWhO4UnW8brhaltuDg7XIwimMgS9S6z\\\\\\\/b5NwRZpRMuJe8MBr+e2xlN4mg+ZMUl5aRysTj\\\\\\\/AEgk1bursz8MM4HbnCBL\\\\\\\/Uyu5cwJ3Q956uYQTXsGnNjmd1lBUuJ0mi40RUTrU3+TP59duFMY4m2SSkEbPcScHIg8L101Tgq7BUdID\\\\\\\/XfTjfLT4MtQzVLFMBlwtqFBIJkFDNN+cqNCxmH\\\\\\\/adk3nROs5myhyW50S0RqKDEgv7laIjN1S8HWyobqfa+Y4dweCNGV\\\\\\\/Mf1RZIFtiiixCv\\\\\\\/T7J3ekUWdMtfDz\\\\\\\/2HJPr6LjEVjOycFTory8uKgzbH8G41Jw8dGJ3EbDX5GEmrv9MDBi490nah60Sm5ztYSAZDwBjVQ3t27\\\\\\\/nbCDqNzBwnC6B3ycfQOw9qWkHGizbRvfSjLdnvIK3EYHpgIFMga6YY4pyjhfUOxCg53serNiGX51u6R2xExfjMVdDach07M21QJJVUMap9EjZ4LjW0jL2PkFUzKL4\\\\\\\/1z78ilD7bH1ptpL4u9Y\\\\\\\/nYHXIoxH\\\\\\\/av7o6vfirdz7LqhK\\\\\\\/24+q3Un2DMew0G7xNSJjduVO30ezDATaxuaycl3ORpTP2LRWqqaEnR85A3gI0GFABbdGu6UErHCC7OGD2jMWzIoze9n+OKht9zzphmVkfjfV49HjHLvcQE1d61rJNlq\\\\\\\/GRbq+\\\\\\\/Kfwcpq6epAT\\\\\\\/sthdARSQ3LEXgdAtTFPfADOdYBIEl4uf7Nk3MwMnlggHyRmtPavWRE3LfroU1J9L71QKgo3YcnPsAvuf6vDvxF5HHI5WbG8jzqxoHmaRO2iPjwhxfSZtddn87jFVUAAIUs\\\\\\\/aF9ye4OuinYxf0L+IUNFuR8Ggh+AiaC1UYO+7iq+QF0if0I8MgYA3pPTbYe5NfF+zT+D7QAUjBFCYf68Yp7swfp4+LrimViPFTDnZodKt0u2HJQkJwIJ+1gpC4FAZFIq900ZOThokFbTmNEGKy9csJaorfeDWz7iQGwFtIcMBMWrNr32tX6\\\\\\\/WPMcKVYafW+SBx5qLDaNKvsP3boYdPu\\\\\\\/Jk4SI6Vt16JRwIZtp1C2gdMHyZqsi18R8wmLSSBwEasDq16Wx9yr3fR87iUcjb710UnUQeQGWqhNPMYKEKlli6BElFKIuJI3nA2FPHfA\\\\\\\/RNQetQiKoa+7E6xByLOIh\\\\\\\/+B2\\\\\\\/FLsOAPpMA37bX16aqEc+Nvjo9ukeYvtgN\\\\\\\/8HUoyuX6Pb4XIgrRaPU37XnVYqSHDywL8BijqGEmNYoLN5GAn0yZK\\\\\\\/K0oSZnsiX\\\\\\\/AqTrZKhEO\\\\\\\/ubHKsmEtRtixGIWhOqKMPoAZOHRKfNXKJZWFmIIYRi5g1TXpscPYXqiT\\\\\\\/TYj8oz7Avf8glZzM1rDjo49x6o9GpDhaRVZFoWn7\\\\\\\/AmrpdGs00hxkA5RRWwwcMgbYW8EM7A5PcX6PTg2YL3+cCYJo6GjiSNuBFJYmReBSf+OzxgYY0\\\\\\\/bvmgs6UuJ6DQf3Geksf7JlRCV5XA7jD2cJBe6iBHEgQifiaVjFjTWB8DkH4gq2saAydba2iDXC3g3huaLEwGczBkSqnt1T7VrHTz7ihY0RHYuwCIX+Ys0+qlpV7nNQ2ibgSyX8zuZ19al2bjAhYMKSrj6ZnF4JsZaCGNnVoKr8\\\\\\\/KxMMVeuPZW8Rekke9b8OPPGB5A7aR59RwiEK8B7r7SgmWNE7mNg2ylQI9OWD1LYjxsPxRdB3avphjB3nKejRUkTW8THUTw5HMTpkY8J0V3LMR5uqePoeSdUnlEZ7rDGk9I1TCR65ujzX7tAEwFWByeNYVKTvLp1LAJa1ZVA2SAndeuMQd0uj59DLXDjl2CgPjx\\\\\\\/xpxrsJr0y\\\\\\\/5hsQf5V5gbhsLsTZtkNAFaL+ay4W2fhLz\\\\\\\/1EnbUtuytMsCDPVpQUMRWWdZ+esT287CMpR1kYBST9+drTzrtLwLYNxp7czFoG\\\\\\\/AYoLU4Xyo1YXtPiPPBUojYgyQFhlxRI+WyLt5ZUO8+JlHKA8teo\\\\\\\/s2RGnCpvET2LNuOn0s3OuChbDQc2SoVgm9BLMpl2\\\\\\\/3a9zk4lOkqC0nijvy8p1hVT+PuJycmxOotEtrnk1qfijf9ME13WQQR1msADgUH1BC4vyPIc8zOGGWZOhWgFmWibaAqmtE24\\\\\\\/vLmed\\\\\\\/S3m+Ku66UJLRSZRHE3EIADZqpg2O4MD1YgSUxTH6CzuuOzquTVoLBDjwZOcGbgMwJUySFi96OKMhliLKu4hKYzmBZS8dEOE+l5fZirQhlJB60AEGfVavmwmXe6RGKMxlrwkK\\\\\\\/x+FDFPur\\\\\\\/6K50PpzfT8Byne1CyMtQmur6wXCI\\\\\\\/a0NOUXN0KYGKsu7hSEUqqYGtUz+JxK4nKaQVVUjJywWM+fPjWW4D4L2r3za9xXJmfTvEYv6bvBf2Y7x+cJRTRjJ1gv0Sl9\\\\\\\/JGl\\\\\\\/4dzKC28YCvYdauLrdyUubVZsCmYqOZ42Mvbp8CcIQ1lZ82h7amyDq8L6nHWdzpoSY9eF7kO6sVywsBQPXl9cTodtyGl3JzeTFhV1WnD2Raz2t4uF402UcJREwuRMU4p74qdeHs6p7vBcI4mu3FRwj9eHqVbEaRb91FX2gMsf8Y3Sr5OaCKEaaWa2KkJNz7f1XCCwUkXO89unX0Hq2xXTX0CH3VT6i8SbvibkLfNYl2H6MJlbK0BlcBCQVl4J2w6xHDEp4HEVbAp2SvuQt\\\\\\\/oPA5uK\\\\\\\/sO6K46tNc643\\\\\\\/9ztBVUF3pkO4cQoZLoXZVj\\\\\\\/TBORfrk1Bjy1tTlAf4\\\\\\\/3MDs36k22lpBrq+a6msMdS5LvVSN9FbThfYhHYHKdyz6sl4XwvX9u9TG7bsYl4ZI60LvhrwwwN1pKzZiiPtY982bFI4wBHX9MZm7uwSRm8dex\\\\\\\/yLcuoLZxf6eRp0aynOGxgXkyxNDZQSsyfG4BRSAm1nQ1jq+JTwvHcmw0sDex1EpwKxtHf\\\\\\\/GWIzAEBjm0+4\\\\\\\/8vHCypG4ffkKXwK3+oLLUlNwPMEr0lHpduRQAMR5w5kOHEsHMcliYG9HyvFMcVTdX6oKL\\\\\\\/s13\\\\\\\/JVoEe5UoS7LnVRCOha\\\\\\\/R\\\\\\\/E2vzz6oImSytvt6A+CSi7wTqwRwBQcCAa7ncykAmSVr7eOQ2vpgFIFqnzVx+mYpJ7BQ+88uH1jd9zMLVGAM8bWcpIzVD\\\\\\\/O6lf3JN5YDOP57HsaSIZ5eC6Cf21KmHKD0FBWS0ASfUSnRVP1crccBt45p2PJtgJfrdgacbkLNLZPYdAGaMncTz7noLg0+syLgJEvKfB1nHCPl072Ms\\\\\\\/cJGqeAxd1dxd3lrVgR2s6JnyVfH\\\\\\\/MmAADJ1Z1S9xIbJSyKDtN4k8DqJeEJjakcpmiIkFkrvLa2G7D7Krhw15R3Ttd428TUjDkNM19kWjXMU85xL2qQT7AhLRMte2dm2hpyhieG4Q1nRAwXSqlLR\\\\\\\/+D01WLcSkDs1yCimMJjHwa7eQq0uvN9MPw0YXYMtti7PsiiAD03u9WsgHm\\\\\\\/7Xcar8jSdpAIieGi1A4l5813LjVMqeJUYSezj+BgBL9B8dXgw53xa0Q+Dp+58smJLEsu+ZEjndNmj2mK1nmzqvoZyVZ0n9tsQWhkyQSYoCjx+8pPczxmhZ3oWZrGTkWMPn+alRnFZr7zh330R8nEqNQb9KC\\\\\\\/z3zrtRXv95Xjbzy2itlBfST8xq\\\\\\\/rA\\\\\\\/HxGhI1OnlL\\\\\\\/NDvpUcUSSCSxn82xs1mC0Ab7DzCNiye+PJKBNummRrRiGeuofQQWRH+Mpp7SLxz0RPiTt3K08IDrLaKUd1kzcuzyPiNeexwXizL8zz8tp1kAOGic9OpKNSNjpT5HmLh65YHi\\\\\\\/\\\\\\\/eB+E0pXVBBiSg3MvBAiOkX7EUrZkl8jy7zQc48jEFxm9WTCYZRci+tvsrUWtYZW3evVYJPFi7Ua8Rlg1GsZKRg36uYNAIOJXkGQDfTcPh\\\\\\\/eCZ2nhEtriHZXI4rNfgIEaG8XNyv4grZSCNliC4qYAWhTwoYBQHjbXC5TjpWD7xyH1mWdPNfZAZDwVExC3svEGjrRdUW1jmtU5HRInSPwUr1y1SmAVE1xfnV25aMpCZwiz6GpV2AKRM1M1Cj9rmFxejNAvlDNaxxVx1hnvDP0m768UuMMNG1oFM8KxDvPqJJ8fzcOPqULsHqmbH8bKd\\\\\\\/lGiBAIJ\\\\\\\/mK7eYfjhKL00osoUzFy6NGlOVosrBcUjxRkqv748RFmuFVLzlmfG5vdLTwVViZNtLSeMXD2rb023h6XggkLnMmT39tVR6nV6nIU7S3PK02A8rWcEKFoYhDAlNnwF4OvgbgXfh4CPgcLBxN893NUioMBMFOhDrd1xB\\\\\\\/+4lSlJ8KkEhI3Aoezn8ucOvcUjxaurs1ewjs4pa+kYzGsdtrxDvaaiVQnoGnTDLX1v2kECnKnRt39q5xuHdUxlBAtHrgHcLzacgb\\\\\\\/xy0paO31zulz93T\\\\\\\/A+48z6\\\\\\\/2mL5mtAMwBtYyerNthGq5rspViw\\\\\\\/OyDl7m1Kjo+PP\\\\\\\/TOj0dNoOEjJemN0AdgDB+4ymLlKVb8EuMsaQPd0yeVq7qlDlnw+TKJ\\\\\\\/pQF4PZ2ARwotpkZmlzHwzLxYLDdxy5ccSXJtdBrmCHPOa75FfljsO5dSpGSWJyHqSfwuwncnZ893VbVqggt0rFoKAbWR3Nc8uP6kZPtsnHjTPQRZjKR0r\\\\\\\/3Mg7ltDJHJAP26GodMWogSoCuSbTZVT+CTu8FcdQiAbRNoMUO0aiTj90rZBEAjT93mBZiUKN2eV5YQrRovP45vlPUL2l\\\\\\\/24WoZOF+EzXGrJ1LI7JeMBubUlNZr+3lNF6Rik6qoOWnNO8YTTuRz5rlABgWiWOjwIbnuJFcCBEcr32cU\\\\\\\/ALu933lDu+ZAhjihsDbv82gsOTs+42KkeUBgd75XlvBZQiZ\\\\\\\/A4s31GdfmBIxIHHLInx3k1XV2md9\\\\\\\/mG55X9wVONroRgM4IZC3Itp6J5efrF9mqA2vwwsEiEryZfvrzFW+E17alqTUtS+Ac4kSJNRbSAynVe1C5aifsSx7FhtNvCemZX\\\\\\\/byg8VpsXsWmdYEwr2jJReFaafCUCy\\\\\\\/NwK2Ne28r9dVgZdpSJehk2LACissDsvobzjbk2DQ1EyknHDruT5Dr2mx1+orhRO9cgFVuyq3zE5TSXo7zjNqYdF32o1leaP3qrgX9swMQDZkTWmNNSy3dqYGuVsuDtzUrli7gaLpK5cQw246VWVLIv8KWe4CVEZFd83pslY8jITcXhZq95socQn0xp1GRd5kpCMKY9rxF+pnIQNXGDvtMMfqdyV7Ep\\\\\\\/3yqwFwwEdv8Z+KRcUFrYYWov+x1+FmU9Y5yK6GTp9poZo68BSYLYmhw45QrWn7\\\\\\\/PaJ2GyDqKnM7rW53Gh9YIkk5s9bZP7Y9bAy1hPg6Tsb9ZcNCwullh1yoGKjTfJNvrAPGIxau2wkZyRw5KIyy5frXTsKBMoPkGF7t32yqxS\\\\\\\/ru68L1+cOh\\\\\\\/6JlDhAmmGw9FyG0sTRHrKXWchuiSbAYtkCzE9yTs\\\\\\\/rT++9i7IqYiuGL4grKuqZbJ84b8AE3LH+nwZipMbXNk4tZWfIF0JsDVxLrGwGFX9vcnlvrIVnH3jHo35qupre95vYCiVMvhikVqKfjblp7LP81i6tiA49RCHriWbS1vUzU7qVe1MgieB8nRifjzhjvjS68pk6WqsbnAxEJY5o8XjbIRkYO12h6PuQ1iPfrl7DPlq8EqH4iUlrQHwvA9M1RNEl+vLKf1c6mQQuNhbeQirYgBi01SJLraKv2Wa3sN+j7yrsrF1\\\\\\\/Eeoh9eURLGvJCAN6LId0Q2T9wA31vL8Y7SGNncgb+WjQRLaZoUX8wpSPyrXZjeb0e0Su2ahvOYjLg\\\\\\\/\\\\\\\/2rDVPHvzpsda1WNbOlluq9F104pE2SRIwpWVDTCPFQzqXo4XpZlfZ0DWttxHMbT4lyThhESZ3GTbOxWFPzKTblquENEW+cLouMAtt\\\\\\\/TrLLo8ejUimNK\\\\\\\/knqbOQ7gLhkyr1TrwKbByuaYnNFqGrTwNju4938vqskGAqhw4IPOuQPACGV+R8E58wEZGNWpQAPtJLWMXFkkc8Tf5Gf+GLXWB8gGIF5m9ct4lwEQ3Caudj9lRLGTtmB2eg79AE2hKLcyuiN+XB7MA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022oln2VJsKtf41v0FiDaCowA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222764dddda2d6daa6a6168a1af7f42c974c260c0c2213f415fe968b5e031376f2\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Wx8Kmni9j3HR8dHPo5tMBeDkeAlvrZ7OlnxFKht0feIX6qr8fU1yMrN91XJVZXOs93yXkjh6EdybZtVhU86b4lmSzL0vqlDeNF6iMpzT98XkkWo6FaMtLuHS2neblMFqhCu3g29OVhNQ+vUfbE2lnnRawSTeB1Z1QtPrdL3v10XQC\\\\\\\/xC7Kwg55vDhs6JIHED4qNUvaF8zB1Pb1BADrLMsWAXsVHzs5tKqngAbxjMVKCYUlwrFN\\\\\\\/FMEQPI4dSbc\\\\\\\/a07hYVK0PlRFvX1yRd2tBhVyVFsf\\\\\\\/f+jTN1vT+duBrNGdXxtEY880OAHuTR5OkF74Ewv7rWKbDEDSfhhGmHDk1eDfQkza5HiBNcEEqgyX1yu6Vya3WDOogPH1KSjhNy4A4jqyTv6MupKgAUuGctQgG2FvRSBpml6aWiFTd10VGWUL\\\\\\\/TWELy5e1p59V7pSyqB90PLjFwv73wwwQ0ycf2I+exxgcIyLPzbAmT2dfXMTGJwgkTarisTKYou6mWlJ3Y158rRRZoC6c32xu0WyuZwMW1UlHOJhHSDLULIdpdeWmAjwn+MfvDekl3ZsiRSQzB4pdem+5cYyLdFcy7seJbfRKRUGeN3+\\\\\\\/R3JKWwJGtYinALnyjpu5Ujyfw9P7oZes5GnHgd21OqUCHdW6N54qx4w5TfMS0KwZIFC\\\\\\\/dvr+4XO1dFHn7nb1n44Qi1ysyHhyZG41yFoNCqiwvLAXkFqrOfbLdKSy4plnGGeAeBRBirRY4UB+JBo3E6YwOF\\\\\\\/C+5m8enwZfwTjtsW+BMr4zwDebuHT\\\\\\\/LQKbOBWvUp1520ZC9BzeFykXvXSQVLNUCfKSL\\\\\\\/UPY0Ywod\\\\\\\/dqNK+LnQOfRiKsTpX483V\\\\\\\/Z519a+hfuI0wGck18MazvI0ZO9n\\\\\\\/J3YpfAImBJdVQpOLAhCUfilaRw5icPApNl343wwE1Lzdlc3Hb17wwvBU2Q0mW+XqGjAPXxLNCHf+3AOHcxQT\\\\\\\/wN1RmjYut5phETDXg5u7VBqBiKapQdI6\\\\\\\/auVI2K2EYRUs8b6AuDMymFIZtx8y8YbNc29zAxhViLE+Gj2V+R4a6kMRgJV0CfkzBAulXchWKQHCCyAignP\\\\\\\/J+0ZtJRTkpq+6ulkLpdOC3oewb51tDew8gVEmayiq7ehfZeC8ZPI1RW0kCGDtgQyDkETKH3FRyBPyPt0q5cSqOMzW5sRt5he7PQ0IXZ+WJZC03io9KYImprAuKYApEZpbud4j+ClsrtZS7HJVqr\\\\\\\/PgLbIZnWcsjpZ6QYNNkl+btP5fY+inBGK9WTIxbuHiXqLobSm5jOYJ5IXaD0U4djJPag9T2B5W3XVB\\\\\\\/3Cy8P0QU5O8uLNlq4yWTMvKQfVUahmPNFwDfWfnjPHYYL7PfJG33E1QrjEaBVH2XDh6JIFLf0VwSnOuOhQhoh\\\\\\\/XBmExnGCdmL\\\\\\\/rf2bct7r71ROKXPXfJOUbmnkrMGhH4tiQDGNxnq\\\\\\\/oJAu4lFNLVVWJu4qsqf4+igpzSAqZuz1s1lvFQQoUpmusIs4M7RX6st7iqrCqGrhZhxsfgv1cus\\\\\\\/DEpYN5S6aNYyIul7f0odrCuWV5PXUTGVhK\\\\\\\/KAZeeS9ph\\\\\\\/QLGQzWYx0d2VHZs39wOW8RPNJ6TjyyVFlPi8XzWrPVKoOxzdEcBl4DNxMVFUsS0tjRG0Qpeo1UzqBDZWbYNwt+8jmFURNvDd2sVw5UUhE+tbhVRJaTOt3fCwqfXaVaRdGLc1j2b4QG2speAdc5XZnVnHSkKIpoRMBLkpFpBC9vFBG7k4HkvSt6lST79NwC9UlriH2GsRQRG8d186ZX5bIvaU1GKQrGH9An8rZrXBzjes5jXjh68dFj29x1SVZtXGN2DToXuuQIhF136Wysv3MVEuVyZXrIJDkKzU956rUViFrF6Sc8cwd+qS3jEkK4AMNf2KSgUq1D27BvnjqqoPDQrBWklOXVuuYQYeQCg1oemxRDinLYwpC+ze5EwCmDgrzIM6HwcuYaO0r4oWIHV9dsGO31N9E7eu5YaeTQT1M+LrYPg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022cLAEArMHjrXcpfkVx5x26g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002205fc0f29b2bebc6df06820e2b68db7ac006193f15f1e6129a599a9464e8dd794\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494053 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/2925" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-fhpQYiMrlonLpPohsqzglHVM_EjWIf52kxM5oM_Uz80" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/2925" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3jpq6Yo9XG6LjO-fNGEOtDVSmUTAeDU_ps15rKRL1KM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/9/2913" title="Deep Kernel for Genomic and Near Infrared Predictions in Multi-environment Breeding Trials" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/9/2935" title="Use of Repeated Group Measurements with Drop Out Animals for Variance Component Estimation and Genetic Evaluation: A Simulation Study" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494053" id="node494053" data-pisa="ggg;9/9/2925" data-pisa-master="ggg;g3.119.400508" data-apath="/ggg/9/9/2925.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Joint Use of Genome, Pedigree, and Their Interaction with Environment for Predicting the Performance of Wheat Lines in New Environments</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8837-3770" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Réka Howard</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8217-2348" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel Gianola</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-4464-3385" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Osval Montesinos-López</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-6922-0173" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Philomin Juliana</span>, <span class="highwire-citation-author" data-delta="4">Ravi Singh</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-7856-1399" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jesse Poland</span>, <span class="highwire-citation-author" data-delta="6">Sandesh Shrestha</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-3202-1784" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-5098-2060" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Diego Jarquín</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2925-2934;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400508 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494053" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Réka Howard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Nebraska – Lincoln, Lincoln NE, 68583, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R%C3%A9ka%2BHoward%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Howard%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AR%25C3%25A9ka%2BHoward%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8837-3770" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Réka Howard</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel Gianola </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>University of Wisconsin-Madison, Madison, Wisconsin 53706, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BGianola%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gianola%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BGianola%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8217-2348" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel Gianola</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Osval Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Universidad de Colima, CP 28040 Colima, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOsval%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4464-3385" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Osval Montesinos-López</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Philomin Juliana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>International Maize and Wheat Improvement Center (CIMMYT), Km. 45, Carretera México-Veracruz, El Batán, Texcoco, Edo. de México, CP 56130, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philomin%2BJuliana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Juliana%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APhilomin%2BJuliana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6922-0173" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Philomin Juliana</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ravi Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>International Maize and Wheat Improvement Center (CIMMYT), Km. 45, Carretera México-Veracruz, El Batán, Texcoco, Edo. de México, CP 56130, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ravi%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARavi%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jesse Poland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>USDA-ARS and Dep. of Plant Pathology, Kansas State University, 4024 Throckmorton Hall, Manhattan KS, 66506, USA, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BPoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poland%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJesse%2BPoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7856-1399" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jesse Poland</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Sandesh Shrestha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>USDA-ARS and Dep. of Plant Pathology, Kansas State University, 4024 Throckmorton Hall, Manhattan KS, 66506, USA, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sandesh%2BShrestha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shrestha%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASandesh%2BShrestha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Colegio de Postgraduados, Montecillos, Edo. de México, CP 56230, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3202-1784" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Paulino Pérez-Rodríguez</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>International Maize and Wheat Improvement Center (CIMMYT), Km. 45, Carretera México-Veracruz, El Batán, Texcoco, Edo. de México, CP 56130, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:diego.jarquin@gmail.com" class="" data-icon-position="" data-hide-link-title="0">diego.jarquin@gmail.com</a></span></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Diego Jarquín </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Nebraska – Lincoln, Lincoln NE, 68583, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BJarqu%C3%ADn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarqu%C3%ADn%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADiego%2BJarqu%25C3%25ADn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5098-2060" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Diego Jarquín</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:diego.jarquin@gmail.com" class="" data-icon-position="" data-hide-link-title="0">diego.jarquin@gmail.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/9/2925" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494053" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494053/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/9/2925.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494053" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494053/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/9/2925.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494053" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494053/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/9/2925.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494053" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494053/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genome-enabled prediction plays an essential role in wheat breeding because it has the potential to increase the rate of genetic gain relative to traditional phenotypic and pedigree-based selection. Since the performance of wheat lines is highly influenced by environmental stimuli, it is important to accurately model the environment and its interaction with genetic factors in prediction models. Arguably, multi-environmental best linear unbiased prediction (BLUP) may deliver better prediction performance than single-environment genomic BLUP. We evaluated pedigree and genome-based prediction using 35,403 wheat lines from the Global Wheat Breeding Program of the International Maize and Wheat Improvement Center (CIMMYT). We implemented eight statistical models that included genome-wide molecular marker and pedigree information as prediction inputs in two different validation schemes. All models included main effects, but some considered interactions between the different types of pedigree and genomic covariates via Hadamard products of similarity kernels. Pedigree models always gave better prediction of new lines in observed environments than genome-based models when only main effects were fitted. However, for all traits, the highest predictive abilities were obtained when interactions between pedigree, genomes, and environments were included. When new lines were predicted in unobserved environments, in almost all trait/year combinations, the marker main-effects model was the best. These results provide strong evidence that the different sources of genetic information (molecular markers and pedigree) are not equally useful at different stages of the breeding pipelines, and can be employed differentially to improve the design and prediction of the outcome of future breeding programs.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genome-enabled-prediction" class="hw-term hw-article-keyword hw-article-keyword-genome-enabled-prediction" rel="nofollow">genome-enabled prediction</a></li><li class="kwd"><a href="/keyword/pedigree-enabled-prediction" class="hw-term hw-article-keyword hw-article-keyword-pedigree-enabled-prediction" rel="nofollow">pedigree-enabled prediction</a></li><li class="kwd"><a href="/keyword/genomic-%C3%97-environment-interaction" class="hw-term hw-article-keyword hw-article-keyword-genomic-&#xD7;-environment-interaction" rel="nofollow">genomic × environment interaction</a></li><li class="kwd"><a href="/keyword/pedigree-%C3%97-environment-interaction" class="hw-term hw-article-keyword hw-article-keyword-pedigree-&#xD7;-environment-interaction" rel="nofollow">pedigree × environment interaction</a></li><li class="kwd"><a href="/keyword/genomic-%C3%97-pedigree-%C3%97-environment-interaction" class="hw-term hw-article-keyword hw-article-keyword-genomic-&#xD7;-pedigree-&#xD7;-environment-interaction" rel="nofollow">genomic × pedigree × environment interaction</a></li><li class="kwd"><a href="/keyword/cimmyt-wheat-evaluation-trials" class="hw-term hw-article-keyword hw-article-keyword-cimmyt-wheat-evaluation-trials" rel="nofollow">CIMMYT wheat evaluation trials</a></li><li class="kwd"><a href="/keyword/genomic-prediction-genpred-shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction-genpred-shared-data-resources" rel="nofollow">Genomic Prediction, GenPred, Shared Data Resources</a></li></ul><p id="p-3">It is important to increase food production to meet the needs of an increasing world population (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Whitford <em>et al.</em>, 2013</a>), and satisfy people’s dietary requirements. Wheat is a major cereal grain worldwide and the third most-produced cereal after maize and rice. It is an important source of protein and calories and the dominant staple in North Africa, West and Central Asia, and the European Union (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Tadesse <em>et al.</em>, 2017</a>). The Consultative Group for International Agricultural Research (CGIAR) estimated that, by 2050, wheat production would need to increase by 60% to satisfy consumers’ needs. Such an objective cannot be met with contemporary techniques because wheat production is currently increasing by less than 1% per year (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Ray <em>et al.</em>, 2013</a>). Thus, there is an urgent need to develop methods that would make it possible to increase wheat production to meet the current and future global demand.</p><p id="p-4">In plant and animal breeding, predicting the breeding values of individuals depends, among other factors, on the target population, the genetic complexity of the traits, and the target environments where the traits are to be measured. In particular, efficient plant breeding programs need to address genotype × environment interactions (G×E) that reflect differential reactions of certain genotypes across a set of environmental circumstances. Environmental conditions affecting gene expression induce G×E, and estimates of genetic correlations of line performance across environments summarize the collective action of genes and environmental conditions. In theory, the correlation between performances in different environments may vary across chromosome regions, and standard mixed models for assessing G×E typically do not decompose the total G×E into specific chromosome region × environment interactions.</p><p id="p-5"><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bernardo (1994)</a> was the first to propose using markers in pedigree-related lines for predicting yet-to-be-observed individual crosses. Subsequently, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Meuwissen <em>et al.</em> (2001)</a> proposed whole genome regression models for genomic selection (GS) where hundreds of thousands of markers were fitted jointly as covariates for predictive purposes. When the number of markers (<em>p</em>) is larger than the number of individuals (<em>n</em>), regularization methods are often beneficial for prediction. Proper prior distributions in Bayesian models produce regularization, but can have a marked effect on inference when <em>n</em>&lt;&lt;<em>p</em> (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gianola <em>et al.</em>, 2009</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">de los Campos <em>et al.</em>, 2013</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gianola, 2013</a>). Empirical evidence obtained with plant and animal breeding data has shown that GS often outperforms pedigree-based prediction methods by a sizable amount (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">de los Campos <em>et al.</em>, 2009</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Crossa <em>et al.</em>, 2010</a>). GS has become a useful approach for improving quantitative traits of many crops in plant breeding.</p><p id="p-6">Wheat production and its profitability are highly influenced by environmental factors, especially climate. The interaction between environmental factors and genotypes also affects wheat yield. Hence, environmental factors and G×E interaction must be included in prediction models. The first study using a model including genome × environment interaction was conducted by <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Burgueño <em>et al.</em> (2012)</a>. These authors developed GS models for multi-environmental trials based on the genomic best linear unbiased predictor (GBLUP), using the notion of genetic correlations between environments (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Falconer, 1952</a>). <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Burgueño <em>et al.</em> (2012)</a> found that the multi-environmental GBLUP had higher prediction accuracy than the single-environment GBLUP. Later, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jarquín <em>et al.</em> (2014)</a> proposed random effects models where main effects and interaction effects between markers and environmental covariates were introduced using covariance structures as in reproducing kernel Hilbert spaces (RKHS) regression (<em>e.g.</em>, Gianola <em>et al.</em>, 2006; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gianola and van Kaam, 2008</a>). The approach presented by <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Jarquín <em>et al.</em> (2014)</a> is an extension of GBLUP, and can also be interpreted as a reaction norm model where genetic and environmental gradients are represented by linear functions of markers and environmental covariates (<em>e.g.</em>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gregorius and Namkoong, 1986</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Falconer and Mackay, 1996</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Calus <em>et al.</em>, 2002</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Calus and Veerkamp, 2003</a>). The advantage of the GBLUP model of <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Jarquín <em>et al.</em> (2014)</a> is that it allows using not only large numbers of markers and environmental covariates, but also high-dimensional additive relationship matrices obtained from the pedigree information.</p><p id="p-7">The model of <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Jarquín <em>et al.</em> (2014)</a> has been used successfully for multi-environmental data and for incorporating interactions among multi-type input sources (<em>e.g.</em>, dense molecular markers, pedigree, high-throughput phenotypes) in several crops (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Pérez-Rodríguez <em>et al.</em>, 2015</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Crossa <em>et al.</em>, 2016</a>; Sousa <em>et al.</em>, 2017; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Crain <em>et al.</em>, 2018</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Jarquín <em>et al.,</em> 2018</a>). Many studies using the <a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Jarquín <em>et al.</em> (2014)</a> model indicated that including G×E interaction in the model substantially increased the accuracy of across-environment (locations and/or years) predictions (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Crossa <em>et al.</em>, 2017</a>).</p><p id="p-8">Most studies involving pedigree or genome × environment interaction do not include more than 1000 lines per year in different environments (year by location combinations), with all lines repeated in all environments (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Crossa <em>et al.</em>, 2017</a>). <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Pérez-Rodríguez <em>et al.</em> (2017)</a> was the first study in which GBLUP and pedigree models were used to assess the prediction accuracy of a large number of CIMMYT wheat lines (58,798) evaluated in several site-year combinations in México, and to predict grain yield performance of wheat lines in several sites of South Asia using the GBLUP with G×E model. The predictive correlation of models using only pedigree, only markers, or both pedigree and markers to predict performance in South Asia (India, Pakistan, and Bangladesh) was around 0.25-0.38, which was higher than the correlation attained using phenotypic values only (0.20).</p><p id="p-9">To our knowledge, no studies have addressed the potential interaction between pedigree and genomic information, and between pedigree, genomic, and environmental information for a large number of lines evaluated for several years where lines are tested for only one year. If genome-enabled prediction is more effective when individuals are genetically (pedigree) related than when they are nominally unrelated (<em>e.g.</em>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Habier <em>et al.</em>, 2007</a>), the interaction between pedigree and markers and between marker, pedigree, and environmental information could result in better prediction accuracy. When the interaction with environments is studied, the number of environments plays an important role because the number of correlations that needs to be estimated grows exponentially, and the complexity of the model increases significantly. The marker × pedigree interaction would indicate interactions between QTL and the genetic (pedigree) background, or simply a discrepancy between the pedigree data and the marker data. When the pedigree is deep, the expected additive genetic relationship should be close to the realized estimated relationship obtained by markers. In real plant breeding and animal data, most of the plant lines or animals evaluated in one year are not repeated the following year, and thus interaction terms between genetic (pedigree or marker) information and environments can only be assessed by the link between individuals established by pedigree and/or markers. Thus, it is possible to borrow information for predicting unobserved lines in new environments by using pedigree, marker, and correlated environmental information. However, it is acceptable that some degrees of confounded information’s between pedigree, genomic, and environments would exist in large plant breeding trials when not all lines are tested in all environments and the degree of overlapping of lines across years (or environments) is not high.</p><p id="p-10">To answer some of the questions and considerations described above, our study aimed to assess the impact, over and above the main effects of pedigree and molecular markers, by including several types of interactions in prediction models. In particular, we contrasted models with interactions between pedigree and markers, and between pedigree, markers, and environments, with respect to models studied previously. Eight models that incorporated pedigree and dense molecular markers were tested. Our data included phenotypic information collected by the CIMMYT Global Wheat Program on first-year yield trial panels that evaluated 35,403 lines during 2013-2014 (13-14), 2014-2015 (14-15), 2015-2016 (15-16), and 2016-2017 (16-17), and none of the lines were tested in more than one year. Five traits were considered in the prediction study: grain yield (GY, ton ha<sup>-1</sup>), days to heading (DTH), days to maturity (DTM), plant height (PH, in cm), and lodging score (LD, 0-5). None of the wheat lines evaluated in one year were evaluated the following year; thus any attempt to study genetic (pedigree and markers) × environment interactions relies on the connectivity between lines across years based on pedigree and/or markers. The depth of the pedigree used in this study was 5 generations back, and most family sizes varied a great deal. The fact that none of the lines were repeated the following year does not prevent estimating the pedigree or genomic × environment interaction; the association between lines due to pedigree and/or genomic interaction establishes the link that makes it possible to estimate and account for these interactions.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-11">Phenotypic, pedigree, and molecular marker data for 35,403 wheat genotypes developed as described in <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Juliana <em>et al.</em> (2018a)</a> were used in this study. Briefly, this information corresponds to four observation periods of the breeding pipeline: 7,577 genotypes for 13-14; 8,901 genotypes for 14-15; 9,311 genotypes for 15-16; and 9,614 genotypes for 16-17. Genotypes were derived from 5,369 crosses with family sizes ranging between 1 (1,017 families) and 116 (1 family). For example, there were 1,172, 301, 92, 32, and 10 families with at least 10, 20, 30, 40, and 50 individuals, respectively. The pedigree relationship between members of each family was identical, while marker-based relationships varied because of segregation effects due to recombination and Mendelian sampling. A total of 5,369 families were observed across periods; however, in some cases, individuals of these families were observed in 2 and 3 different periods (not the same genotypes but belonging to the same family). Out of the total number of families, 4,585 were observed exclusively in only one cycle (13-14; 14-15; 15-16, or 16-17); 775 in two cycles, and 9 in 3 cycles. In cycle 13-14, there were 1,382 families with sizes ranging from 1 to 51, in cycle 14-15 there were 1,294 families with sizes ranging from 1 to 48, in cycle 15-16 there were 1,332 families with sizes ranging from 1 to 65, and in cycle, 16-17 there were 2,154 families with sizes ranging from 1 to 116. It should be noted that the number of lines reported by <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Juliana <em>et al.</em> (2018a)</a> corresponds to a subset of lines reported in this study; thus the number of wheat lines reported by <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Juliana <em>et al.</em> (2018a</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">b</a>) is lower than the number of wheat lines reported in this study.</p><div id="sec-2" class="subsection"><h3>Phenotypic data</h3><p id="p-12">Phenotypic data were recorded for the four aforementioned breeding periods/years (13-14, 14-15, 15-16, and 16-17). The traits that we considered were GY (ton ha<sup>-1</sup>), DTH, DTM, PH, and LD. The total number of wheat lines in each year were 7,577 (cycle13-14), 8,901 (cycle 14-15), 9,311 (cycle 15-16), and 9,614 (cycle 16-17). There were 200 to 350 hundred trails in each of the breeding periods. Each trial had 30 wheat lines organized in 6 incomplete blocks of size 5, and analyzed as incomplete block design with recovery of inter-block information, and incomplete block design plus spatial analyses using autoregressive in the row direction × autoregressive in the column direction model to account for soil variability. A final combined analysis was performed across all trails, and Best Linear Unbiased Estimate (BLUE) for all wheat lines was computed. All the 200-350 trails in each of the breeding periods were sown under full irrigation at the wheat experimental stations in Cd. Obregon, Sonora, México.</p></div><div id="sec-3" class="subsection"><h3>Genotyping data</h3><p id="p-13">Genome-wide markers for the 46,089 lines were obtained as described by <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Juliana <em>et al.</em> (2018b)</a>. Initially, 11,293 markers were obtained and, after quality control, 6,978 SNPs remained for analysis. Markers with more than 30% missing values and a minor allele frequency smaller than 0.05 were discarded. After edits, we had pedigree, marker, and phenotypic information for 35,403 wheat lines and 6,978 SNPs.</p></div><div id="sec-4" class="subsection"><h3>Statistical genome-enabled prediction models</h3><p id="p-14">The predictive ability of eight models was evaluated by assessing GY, DTH, DTM, PH, and LD as target traits. These models considered the main effects of pedigree and/or markers, and different types of interactions between pedigree, markers, and environments. The interactions were included by implementing the model of <a id="xref-ref-21-6" class="xref-bibr" href="#ref-21">Jarquín <em>et al.</em> (2014)</a>.</p><p id="p-15">All of the terms were entered as random effects into the model using co-variance structures. Variance components for each of the eight models were obtained under a Bayesian framework using the complete data set (<em>i.e.</em>, no missing values allowed). The eight models are described below.</p></div><div id="sec-5" class="subsection"><h3>M1 - Environment + Pedigree model</h3><p id="p-16">This model is written as<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-1.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-2.gif"/></noscript></span></span> denotes the phenotypic value for the <em>j</em><sup>th</sup> genotype tested in the <em>i</em><sup>th</sup> environment; <em>μ</em> is the overall mean, <em>E<sub>i</sub></em> (<em>i</em> = 1,…, <em>I</em>) is a random environmental effect such that <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-3.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-4.gif"/></noscript></span></span> as the variance component; <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-5.gif"/></noscript></span></span>(<em>j</em> = 1,…, <em>J</em>) is a random additive genetic effect for line <em>j</em> and <strong><em>a</em></strong>={<span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-6.gif"/></noscript></span></span>} is assumed to follow the multivariate normal distribution <em>N</em>(<strong>0</strong>, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-7.gif"/></noscript></span></span>), where <strong>A</strong> is the numerator relationship matrix computed from pedigree information, and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-8.gif"/></noscript></span></span> is the additive genetic variance component; <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-9.gif"/></noscript></span></span> is the random residual term with <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-10.gif"/></noscript></span></span> and IID stands for independent and identically distributed. This model produces the same estimated breeding value for all individuals from the same family.</p></div><div id="sec-6" class="subsection"><h3>M2 - Environment + Pedigree + Pedigree × Environment model</h3><p id="p-17">M2 extends M1 such that an interaction, based on pedigree, between genotypes and environments (G×E) is included in the model. The M2 model is written as<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-11.gif"/></noscript></span></span>where <em>μ</em>, <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-12.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-13.gif"/></noscript></span></span>are defined as before, and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-14.gif"/></noscript></span></span> is the interaction between the additive value of the <em>j</em><sup>th</sup> genotype and the <em>i</em><sup>th</sup> environment. As described by <a id="xref-ref-21-7" class="xref-bibr" href="#ref-21">Jarquín <em>et al.</em> (2014)</a>, this interaction effect can be handled using covariance structures such that <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-15.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-16.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-17.gif"/></noscript></span></span> are incidence matrices for the lines and environments, respectively, ‘°’ denotes the Hadamard (element by element) product between two matrices, and <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-18.gif"/></noscript></span></span> is the corresponding variance component of the random interaction term <strong><em>aE</em></strong>.</p></div><div id="sec-7" class="subsection"><h3>M3 - Environment + Genome model</h3><p id="p-18">M3 is similar to M1 but instead of pedigree-based additive effects, it includes a main effect term for the genome-based (marker) breeding value. The model is<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-19.gif"/></noscript></span></span>where <em>μ</em>, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-20.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-21.gif"/></noscript></span></span> are as for M1. Here, <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-22.gif"/></noscript></span></span>corresponds to the genomic breeding value of the <em>j</em><sup>th</sup> line defined as a linear combination of marker codes and the corresponding marker effects such that <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-23.gif"/></noscript></span></span> where <em>p</em> is the number of markers, <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-24.gif"/></noscript></span></span> is the marker code for the <em>j</em><sup>th</sup> line at the <em>m</em><sup>th</sup> marker position (<em>m</em> = 1,…, <em>p</em>), and <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-25.gif"/></noscript></span></span> is the corresponding marker effect. The marker effects are assumed normally distributed variables such that <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-26.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-27.gif"/></noscript></span></span> being the variance component of the marker effects. The covariance matrix of the vector of genomic values <strong><em>g</em></strong>={<span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-28.gif"/></noscript></span></span>} can be written as <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-29.gif"/></noscript></span></span>, where <strong>G</strong> is the genomic relationship matrix computed as <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-30.gif"/></noscript></span></span>, <strong><em>X</em></strong> is the standardized genotype matrix (by columns), <em>p</em> is the number of markers, and <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-31.gif"/></noscript></span></span> denotes the genomic variance component. Hence, <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-32.gif"/></noscript></span></span>. Since molecular marker information varies across individuals (even within the same family), the estimated breeding values are unique for each genotype.</p></div><div id="sec-8" class="subsection"><h3>M4 - Environment + Genome + Genome ×Environment model</h3><p id="p-19">M4 is the same as M2, but the G×E interaction is based on marker data instead of pedigree information. The model is written as:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-33.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-34.gif"/></noscript></span></span> is the component that collectively represents interactions between molecular markers of the <em>j</em><sup>th</sup> line and the <em>i</em><sup>th</sup> environment. The distributional assumption for this term is similar to the interaction term in M2, but using the <strong>G</strong> matrix instead of the <strong>A</strong> matrix, such that <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-35.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-36.gif"/></noscript></span></span> is the variance component of the random interaction component<span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-37.gif"/></noscript></span></span>. As pointed out by Basnet <em>et al.</em> (2019), if observations are sorted by environment, then the variance-covariance matrix <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-38.gif"/></noscript></span></span> is a block-diagonal matrix whose structure is similar to the Marker × Environment model of Lopez Cruz <em>et al.</em> (2015).</p></div><div id="sec-9" class="subsection"><h3>M5 - Environment + Genome + Pedigree model</h3><p id="p-20">This model combines M1 and M3 since it contains both main effects (pedigree [<span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-39.gif"/></noscript></span></span>] and genomic information [<span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-40.gif"/></noscript></span></span>]); it is written as:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-41.gif"/></noscript></span></span>where all of the terms were previously defined.</p></div><div id="sec-10" class="subsection"><h3>M6 - Environment + Genome + Pedigree + Genome × Environment + Pedigree × Environment model</h3><p id="p-21">This model extends M5 by adding pedigree × environment and genome × environment interactions. M6 can also be viewed as a combination of M2 and M4, and the model is<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-42.gif"/></noscript></span></span>where all model terms were previously defined.</p></div><div id="sec-11" class="subsection"><h3>M7 - Environment + Genome + Pedigree + Genome × Pedigree model</h3><p id="p-22">By introducing pedigree × marker interaction effects via covariance structures, this model extends M5 as<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-43.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-44.gif"/></noscript></span></span> is the interaction between the genomic and pedigree-based breeding values of the <em>j</em><sup>th</sup> genotype. The interaction component is distributed as</p><p id="p-23"><span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-45.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-46.gif"/></noscript></span></span> being the corresponding variance component.</p></div><div id="sec-12" class="subsection"><h3>M8 - Environment + Genome + Pedigree + Genome × Pedigree + Genome × Pedigree × Environment model</h3><p id="p-24">This is the most parameterized model, where M7 is enriched by adding the interaction among pedigree, markers, and environment as:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-47.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-48.gif"/></noscript></span></span> is a three-way interaction term among the genomic breeding values and the additive value of the <em>j</em><sup>th</sup> genotype and the <em>i</em><sup>th</sup> environment. The interaction term is assumed to follow the distribution <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-49.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2925/embed/mml-math-50.gif"/></noscript></span></span> is the corresponding variance component.</p></div><div id="sec-13" class="subsection"><h3>Cross-validation and validation schemes</h3><p id="p-25">The performance of the models for predicting the five traits was evaluated using the weighted average Pearson’s correlation coefficient between observed and predicted values. For this, within each year/period, the Pearson’s correlation coefficient between predicted and observed values was computed first. Then, these values were weighted by the ratio between the sample size for each year divided for the total number of observations across years, and the resulting values were summed up. Thus, the periods with the largest number of observation are given more importance in the weighted average. One random cross-validation scheme (CV1) and a leave-one-out validation method (V00) were implemented, and predicted values <em>vs.</em> observed values were compared within and across time periods. These schemes mimic real plant breeding situations. CV1 refers to the scheme where the performance of 20% of the wheat lines was not observed in any of the years (environments) and the rest of the wheat lines (80%) were already observed in the same target environments. For this scheme, a fivefold random partitioning (80% of data used as the training set, and the remaining 20% used as the testing set) was employed. Four folds were used for training the models for predicting the remaining fold. This procedure was repeated over the five folds, and the predictions from the testing fold (5 in total) were joined in a single vector. Then, Pearson’s correlation between predicted and observed values within the same environment were computed. The partitioning was repeated 20 times at random.</p><p id="p-26">The objective of V00 was to predict the performance of all new lines in a new year. The leave-one-year (environment) out scheme was implemented to define the training and testing sets; since random partition was not possible, it was ran only once. Also, since the lines were observed only once across all periods, no records on the same genotype in different environments were available. A more detailed description of the cross-validation techniques can be found in <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Jarquín <em>et al.</em> (2017)</a>.</p></div><div id="sec-14" class="subsection"><h3>Software and Data availability</h3><p id="p-27">The Bayesian Generalized Linear Regression (BGLR) R-package (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Pérez and de los Campos, 2014</a>) was used for fitting the previously described GS models. Genomic and phenotypic data can be found in hdl.handle.net/11529/10548169. This link contains the genomic relationship matrix (G.RDA), the pedigree matrix (A.RDA), and the G×A matrix (GA.RDA). Entries in the columns and rows of the <strong>A</strong>, <strong>G</strong>, and <strong>GA</strong> matrices align with rows in the phenotypic data (Column 1; GID in Y.csv). In addition, the link hdl.handle.net/11529/10548169 contains Figures S1 and S2 in the Supplementary Material.</p></div></div><div class="section results" id="sec-15"><h2 class="">Results</h2><div id="sec-16" class="subsection"><h3>Variance components from full data</h3><p id="p-28"><a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> presents the estimated percentage of total variance accounted for by A, G, E, interaction, and the residual (Res) variance components for the eight different models (M1-M8) and the five traits. We focus on GY because it is the most important breeding trait and because the general results were similar to other traits. For GY, when comparing M1 and M3, the environmental component E explained around 45% of the variance in both cases, while A captured 38%, a much larger fraction than G (13.9%). When interactions between A or G and E were included through models M2 and M4, the environmental component explained around 40% in both models, while the interaction between pedigree and environments (AE) and between markers and environments (GE) captured 21.7% and 16.0% of the total variance, respectively. The relative amount of variability represented by the interaction components in both models was similar, but the difference in the amount of variance explained by main effects in these models was larger: 17.6% for A in M2 and only 6.7% for G in model M4. In model M2 (pedigree-based), the residual term (Res) represented 21.8%, while it represented 36.1% in M4 (genome based). In M5, where A and G were the main effects, the residual fraction was the same as in M1. Here, the pedigree term (A) captured 36.4% of the phenotypic variability, while the markers (G) captured only 8.4%. In M6, the relative amounts of phenotypic variability explained by the variance components were: E (35.3%), A (12.3%), G (6.6), AE (20.3), and GE (7.1); the remaining 18.5% was accounted for by the residual term. Also in M6, we observed a similar picture as in M2 and M4: the AG and GE components took away a large amount of the phenotypic variability explained by main effects. This was not surprising, as the E term usually explains a large amount of phenotypic variability in multi-environment trials where interaction terms carry sizable information.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494178/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494178/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Estimated percentage of the total variance accounted for by each random effect of the corresponding model for grain yield (GY), days to heading (DTH), plant height (PH), lodging (LD), and days to maturity (DTM)</span><div class="sb-div caption-clear"></div></div></div><p id="p-29">When the interaction between pedigree and markers (GA) was introduced using M7, the fraction of total variability explained by the different components was: E (38.1%), A (26.6%), G (7.6%), GA (19.2), and the residual term (8.5%). Including the GAE interaction in M8 reduced the fraction of residual variance the most (6.0%) compared to M1-M7, where the residual term captured between 40.7% and 8.5% of the phenotypic variability. In M8, the proportion of total variance was as follows: E (29.9%), G (5.7%), A (41.8%), GA (5.7%), and GAE (10.9%). It is clear that when interactions GA and GAE are included in the model, the fractions of variance accounted for by the residual term dropped remarkably in M7 and M8 compared to models that exclude GA. Results for other traits showed similar patterns, as already mentioned.</p></div><div id="sec-17" class="subsection"><h3>Prediction: Random cross-validation CV1</h3><p id="p-30"><a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and Figure S1 (Supplementary Material, in hdl.handle.net/11529/10548169) display the results for each environment (mean and standard deviation across the 20 fivefold random partitions of the CV1 scheme) for grain yield (GY), days to heading (DTH), days to maturity (DTM), plant height (PH), and lodging (LD). For GY the weighted average Pearson’s correlation over the four years using M1 (<em>i.e.</em>, environment + pedigree, E+A) was 0.562. When the interaction between pedigree and environment (AE) was included via M2, this value increased slightly to 0.577 for GY. On the other hand, when only the main effect of markers (G) was included using M3 for GY, the mean correlation decreased to 0.458, but when GE was added via M4, it improved to 0.515.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494079/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494079/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Weighted average (WA) Pearson’s correlation and standard deviation (SD) over 20 replicates of a fivefold (CV1) random cross-validation (prediction of new lines not observed in any year) for eight models (M1: environment + pedigree; M2: environment + pedigree + pedigree × environment; M3: environment + marker; M4: environment + marker + marker × environment; M5: environment + pedigree + marker; M6: environment + pedigree + marker + pedigree × environment + marker × environment; M7: environment + pedigree + marker + pedigree × marker; M8: environment + pedigree + marker + pedigree × marker + pedigree × marker × environment) for grain yield (GY), days to heading (DTH), days to maturity (DTM), plant height (PH), and lodging (LD) for a wheat breeding pipeline comprised of 35,403 lines observed in four periods/years (13-14, 14-15, 15-16 and 16-17). Lines were observed only once across all periods</span><div class="sb-div caption-clear"></div></div></div><p id="p-31">When combining the main effects of environments, markers and pedigree (E+A+G) using M5, for GY the correlation increased to 0.606. Even better results were obtained for GY when both AE and GE were added to the previous model using M6. In this case, the predictive correlation increased to 0.623, so these results were better than when using models M1-M5 for all years/periods for GY. M7 combining A, G and G×A gave a Pearson’s correlation of 0.622. M7 produced better results than M1-M5 for GY for all periods/years and better results than M6 for two out of the four years. M8, an extension of M7 that also considers G×A×E interaction, increased the correlation to 0.630. M8 was the best model for all years.</p><p id="p-32">The most effective model (M8) improved predictive ability for GY by 12.1% and 37.5% over models that included only main effects of pedigree and markers (M1 and M3). The year when prediction accuracy was best was 13-14 for all models, and the year with the worst prediction outcome was 14-15. Days to heading (DTH) showed patterns that were similar to those of GY. The weighted average correlations ranged between 0.532 and 0.675, with M3 and M8 delivering the lowest and highest values, respectively. For DTH, M8 performed better than the other models (M1-M7) in all years. For days to maturity (DTM), the weighted average Pearson’s correlation for all models ranged between 0.528 and 0.678, and as they did for the other traits, M3 and M8 delivered the worst and best results, respectively. The average Pearson’s correlation for plant height (PH) ranged between 0.444 and 0.585. Again, M3 and M8 produced the worst and best results, respectively. The mean Pearson’s correlation for Lodging ranged between 0.436 (M3) and 0.605 (M8).</p></div><div id="sec-18" class="subsection"><h3>Prediction: Leave-one-year-out validation (V00)</h3><p id="p-33">Results for grain yield (GY), days to heading (DTH), days to maturity (DTM), plant height (PH), and lodging (LD) obtained with the V00 scheme are provided in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> and Figure S2 (Supplementary Material in <span class="underline">hdl.handle.net/11529/10548169</span>). For all traits, the model with a main effect of markers (M3) outperformed the other seven models most of the time for all periods/years. There were exceptions for plant height, where M5-M8 performed equally or slightly better than M3. For V00, none of the models including interactions provided better average correlations than M3, as occurred with the CV1 scheme, where accommodating interactions increased the prediction accuracy obtained by models with main effects only. For yield (GY), the mean Pearson’s correlation over all models ranged between 0.072 (M2) and 0.293 (M3). When main effects of markers and pedigree were combined in M5, the predictive ability decreased to 0.250. Pearson’s correlations for the best model (M3) for periods 13-14, 14-15, 15-16, and 16-17 were 0.336, 0.264, 0.350, and 0.231, respectively.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494161/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494161/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Weighted average (WA) Pearson’s correlation for validation V00 (prediction of new lines observed only in one year - leaving one year out at a time as testing set) for eight models (M1: environment + pedigree; M2: environment + pedigree + pedigree × environment; M3: environment + marker; M4: environment + marker + marker × environment; M5: environment + pedigree + marker; M6: environment + pedigree + marker + pedigree × environment + marker × environment; M7: environment + pedigree + marker + pedigree × marker; M8: environment + pedigree + marker + pedigree × marker + pedigree × marker × environment) for grain yield (GY), days to heading (DTH), days to maturity (DTM), plant height (PH), and lodging (LD) for a wheat breeding pipeline comprised of 35,403 lines observed in four periods/years (13-14, 14-15, 15-16 and 16-17). Lines were observed only once across all periods</span><div class="sb-div caption-clear"></div></div></div><p id="p-34">The average Pearson’s correlation for days to heading (DTH) ranged between 0.116 (M2) and 0.332 (M3). The model with A and G produced a mean accuracy of 0.3. For the four periods/years, Pearson’s correlations obtained with M3 were 0.325, 0.281, 0.409, and 0.309. Results for days to maturity (DTM) produced a mean Pearson’s correlation of 0.135 for the worst model (M2), and 0.347 for the best model (M3). Pearson’s correlations obtained with M3 were 0.321, 0.309, 0.416, and 0.338 for periods 13-14, 14-15, 15-16, and 16-17, respectively.</p><p id="p-35">For plant height (PH), the average Pearson’s correlations for the worst and the best models were 0.139 (M1) and 0.295 (M3). Although M3 produced the best results on average, in some years, M5-M8 performed the same (M5) or slightly better (M6-M8) than M3. Results for Lodging were in line with results for the other traits. The mean Pearson’s correlation varied between 0.110 (M2) and 0.288 (M3). For the best model, correlations for the three years 13-14, 14-15, and 15-16 were 0.403, 0.248, and 0.233, respectively.</p></div></div><div class="section discussion" id="sec-19"><h2 class="">Discussion</h2><p id="p-36">Genome-enabled prediction is a tool for plant breeders that has the potential to increase genetic gain for high yielding wheat varieties with agronomically desired traits. Since one of the main objectives of wheat breeding is to develop varieties that are insensitive to environmental changes, it is important to develop prediction models that incorporate information not only on dense molecular markers and pedigree, but also interactions with the environment. We implemented models that use the reaction norm concept and incorporate the interactions among the terms via a covariance (kernel) structure, because this approach enables the inclusion of high dimensional data. We evaluated eight models for predicting five traits using a cross-validation (CV1) and a leave-one-year-out validation scheme (V00).</p><div id="sec-20" class="subsection"><h3>Cross-validation prediction accuracy</h3><p id="p-37">The trend was similar for predicting all traits in terms of model predictive ability under both validation schemes (CV1 and V00). However, the two validation schemes showed differences. For CV1, accounting for the environment via interactions with genomic sources seemed more beneficial than for V00. Perhaps this is due to the fact that CV1 allows borrowing information from other environments to increase predictive ability, whereas in V00 we predicted the performance of unobserved lines in unobserved environments, certainly a very complex task, specially under unpredictable year conditions. One of the main goals of this study was to examine whether including the interaction between marker and pedigree information would enhance prediction accuracy. When we predicted the performance of unobserved lines in unobserved environments (V00), there was no advantage in including GA interaction. However, under CV1, M8 outperformed all models for most traits and periods (years). In addition to the main effects of E, A, and G, M8 included GA and GAE interactions.</p><p id="p-38">A main observation in our study was that including pedigree information in the model instead of G was better for within-year CVs, whereas markers had advantages across years. This is consistent with observations of elite yield trials made by Juliana <em>et al.</em> (2018 a, <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">b</a>). These results utilizing various types of information may enhance prediction problems in the breeding pipeline, as mentioned by <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Morota and Gianola (2014)</a> and <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gianola <em>et al.</em> (2014)</a> when discussing multi-kernel approaches to prediction. Another possible reason why pedigree-based models (M1 and M2) outperformed marker-based models (M3 and M4) for the CV1 scheme is that some families (∼15%) overlapped, meaning that they were observed in more than one year, enabling to borrow information across periods.</p></div><div id="sec-21" class="subsection"><h3>Pedigree × genomic and pedigree × genomic × environment interactions</h3><p id="p-39">It has been argued that genome-enabled prediction should work well when there is genetic relatedness among lines (as indicated by the pedigree), but not otherwise. In other words, the importance of a genome-based model depends on the strength of pedigree relationships (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Habier <em>et al.</em>, 2007</a>). In a model with pedigree, genomic information, and their interaction, as in model M7 (A+G+GA), the expected phenotypic change per unit of genomic relatedness depends on the level of genetic relationship conveyed by the pedigree. We found that the amount of variance explained by GA in model M7 was between 19.2% and 23.8%, a sizable amount.</p><p id="p-40">However, the phenotypic change per unit of genomic relatedness may be influenced not only by pedigree relationships, but also by variation in environment. This three-factor interaction (GAE) explained a sizable proportion of the within environment variance for the different traits. For grain yield, it explained 10.9% of the variability in model M8 (A+G+GA+GAE), and was also the genome-based prediction model for predicting all periods (13-14, 14-15, 15-16, and 16-17) under CV1. Under V00, the impact of the interactions was negligible and the main determining factor was G (marker); thus including the genome-based information produced the best prediction accuracy for all models in the environments.</p><p id="p-41">A wide range of variation in genomic relatedness in a given genetic relationship is expected with large family sizes. In this data set, the family size varies; as already mentioned, family sizes range between 1 (1,017 families) and 116 (1 family). There were 1,172, 301, 92, 32, and 10 families with at least 10, 20, 30, 40, and 50 individuals, respectively. Perhaps with more families with larger family sizes, the effect of variation in the genomic relationship per unit of genetic relatedness (pedigree) could increase, in which case GA and GAE interactions could influence prediction accuracy more strongly. The complex three-factor interaction, GAE, may also capture cryptic, small, epistatic effects, which would change from environment to environment and should be accounted for by the GAE term.</p><p id="p-42">Interestingly, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Morota <em>et al.</em> (2014)</a> observed that when kernels are mutually orthogonal to each other could enhance predictive ability; however, these are not easy to construct. On the other hand, when these are not mutually orthogonal the partition of variance is difficult to interpret mechanistically. Hence, the partitioning of variance into A, G, and GA may reflect “collinearity” among kernels. Therefore, while a multi-kernel approach may be more effective for prediction, the interpretation of variance components should be interpreted with caution.</p></div><div id="sec-22" class="subsection"><h3>What variability is accounted for pedigree, genomic and their interaction?</h3><p id="p-43">It is expected for <strong>A</strong>◦<strong>G</strong> to be similar to the <strong>G</strong>◦<strong>G</strong> matrix, which has been used to model pairwise additive × additive epistasis (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Jiang and Reif, 2015</a>). These models have been shown to be effective in increasing genomic prediction accuracy (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Crossa <em>et al.,</em> 2010</a> and <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Martini <em>et al.,</em> 2016</a> using the 599 CIMMYT wheat lines). It is possible that in this study <strong>A</strong> and <strong>G</strong> would be dissimilar enough to produce a covariance structure that would differ from the expected additive × additive epistasis covariance. Therefore, the covariance <strong>A</strong>◦<strong>G</strong> may simply be picking up epistasis. Biologically it is not easy to explain exactly what variation A◦G captures compared to A◦A and G◦G, but mathematically it explains the interaction among loci (epistasis) that is accounted by the interaction of the marker and pedigree information (instead of accounting for only the marker relatedness). The more complex model, M8, would therefore simply include the epistasis term by environment interaction.</p><p id="p-44">In general, pedigree shrinks the additive covariance within family to 0, emphasizes closer relationships and deemphasizes further relationships (<em>i.e.</em>, “unrelated” lines do share some common ancestor, and is dependent on pedigree depth). Markers may produce unbiased estimates depending on their distribution in the genome, and the filtering methods used (<em>e.g.</em>, redundant markers are often discarded). Therefore, it can be speculated that <strong>G</strong>◦<strong>G</strong> (and similarly <strong>A</strong>◦<strong>G</strong>) would also tend to overemphasize close relationships while deemphasizing further relationships.</p><p id="p-45">Arguably, for a G+A model, matrix A picks up some genetic variance that is not tagged by G (markers). In livestock, markers mainly pick up realized relatedness (better than A) as opposed to QTL effects. On the other hand, in humans with high density SNP markers and weak pedigree relationships, the markers would pick up mainly QTL effects. In wheat, it is possible that a G constructed with high density markers would pick up a combination of line, pedigree, and QTL effects. Unfortunately, since A and G kernels are not mutually orthogonal, the interpretation of variance component estimations is ambiguous.</p><p id="p-46">Several studies indicated that genome-enabled prediction works when training and testing sets are genetically related, but that it does not work when genetic relationships are weak (where the definition of weak and strong relatedness highly depends on the species in question, but mathematically speaking the relatedness can be captured by the correlation of the genetic marker or pedigree information corresponding to the individuals in the testing/training sets). This phenomenon could be interpreted as one in which there is interaction between A and G. In other words, if A encodes strong genetic relatedness, a G model may be useful. Conversely, if A is sparse, a G model may not help prediction. This basic idea could be exploited predictively in a model with A+G+GA kernels. We hypothesized that the GA interaction kernel would capture such a phenomenon within a population and evaluated the conjecture with real data.</p></div><div id="sec-23" class="subsection"><h3>Conclusion</h3><p id="p-47">GS is an important part of a breeding program’s selection and decision making. Increased prediction accuracy can reduce the cost of a breeding pipeline, increase the selection intensity and shorten the selection cycles, thus increasing the genetic gain per cycle. Since the environmental factors are a big part of how a variety performs, it is also important to consider environmental information when building prediction models for GS. It has also been shown that GS models can benefit from the inclusion of pedigree information. Thus, in this study we compared models including marker genotype, pedigree, and environmental information in various combinations. Some of the models included main effects only, and some included interactions. We found that, depending on the validation scheme, including the interaction between the marker genotypes, pedigree, and environmental factors can be beneficial, highlighting the importance of developing future GS models where other sources of information and their interactions are considered, with the potential to increase prediction accuracy.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-48">We thank all scientists, field workers, and lab assistants from National Programs and CIMMYT who collected the data used in this study. We acknowledge the financial support provided by the Foundation for Research Levy on Agricultural Products (FFL) and the Agricultural Agreement Research Fund (JA) in Norway through NFR grant 267806. We are also thankful for the financial support provided by CIMMYT CRP (maize and wheat), the Delivering Genetic Gain in Wheat (DGGW) project (managed by Cornell University and funded by the Bill &amp; Melinda Gates Foundation and the United Kingdom Department for International Development (DFID)); the Feed the Future project of the U.S. Agency for International Development, Contract No. AID-OAA-A-13-0005, and the Innovation lab at Kansas State University for their support in generating the CIMMYT wheat data analyzed in this study. We are also thankful to two anonymous reviewers and the Senior Editor of <em>G3:Genes|Genomes|Genetics</em>, Dr. D. J. de Koning for their valuable comments, suggestions, and positive criticisms to a previous version of the manuscript. We have improved the scientific quality of the article based on their suggestions and added some of their ideas on the revised version.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-49"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-06-03"><span class="received-label">Received </span>June 3, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-07-10"><span class="accepted-label">Accepted </span>July 10, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Howard <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.9.2925.1" data-doi="10.3835/plantgenome2018.07.0051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Basnet</span>,  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Dreisigacker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez-Rodríguez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Manes</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Hybrid Wheat Prediction Using Genomic, Pedigree, and Environmental Covariables Interaction Models.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">180051</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2018.07.0051</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D12%26rft.spage%253D180051%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2018.07.0051%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2018.07.0051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.2" data-doi="10.2135/cropsci1994.0011183X003400010003x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>,  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Prediction of maize single-cross performance using RFLPs and information from related hybrids.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">20</span>–<span class="cit-lpage">25</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1994.0011183X003400010003x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D34%26rft.issue%253D1%26rft.spage%253D20%26rft.epage%253D25%26rft.atitle%253DPrediction%2Bof%2BMaize%2BSingle-Cross%2BPerformance%2BUsing%2BRFLPs%2Band%2BInformation%2Bfrom%2BRelated%2BHybrids%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1994.0011183X003400010003x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1994.0011183X003400010003x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994MV45600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.3" data-doi="10.2135/cropsci2011.06.0299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Weigel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">719</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2011.06.0299</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D707%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.4" data-doi="10.3168/jds.S0022-0302(02)74399-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>,  <span class="cit-name-given-names">M. P. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. F.</span> <span class="cit-name-surname">Groen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de Jong</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genotype × environment interaction for protein yield in Dutch dairy cattle as quantified by different models.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">3115</span>–<span class="cit-lpage">3123</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.S0022-0302(02)74399-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DCalus%26rft.auinit1%253DM.%2BP.%2BL.%26rft.volume%253D85%26rft.issue%253D11%26rft.spage%253D3115%26rft.epage%253D3123%26rft.atitle%253DGenotype%2Bx%2BEnvironment%2BInteraction%2Bfor%2BProtein%2BYield%2Bin%2BDutch%2BDairy%2BCattle%2Bas%2BQuantified%2Bby%2BDifferent%2BModels%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.S0022-0302%252802%252974399-3%26rft_id%253Dinfo%253Apmid%252F12487479%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.S0022-0302(02)74399-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12487479&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2925.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.5" data-doi="10.3168/jds.S0022-0302(03)73982-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>,  <span class="cit-name-given-names">M. P. L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. G.</span> <span class="cit-name-surname">Veerkamp</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Estimation of environmental sensitivity of genetic merit for milk production traits using a random regression model</span>. <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">3756</span>–<span class="cit-lpage">3764</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.S0022-0302(03)73982-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DCalus%26rft.auinit1%253DM.%2BP.%2BL.%26rft.volume%253D86%26rft.issue%253D11%26rft.spage%253D3756%26rft.epage%253D3764%26rft.atitle%253DEstimation%2Bof%2BEnvironmental%2BSensitivity%2Bof%2BGenetic%2BMerit%2Bfor%2BMilk%2BProduction%2BTraits%2BUsing%2Ba%2BRandom%2BRegression%2BModel%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.S0022-0302%252803%252973982-4%26rft_id%253Dinfo%253Apmid%252F14672207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.S0022-0302(03)73982-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14672207&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2925.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186225900040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.6" data-doi="10.3835/plantgenome2017.05.0043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crain</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Mondal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Rutkoski</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. P.</span> <span class="cit-name-surname">Singh</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Poland</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Combining high-throughput phenotyping and genomic information to increase prediction and selection accuracy in wheat breeding.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">170043</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2017.05.0043</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2017.05.0043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2017.05.0043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.7" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez-Rodríguez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Burgueño</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.118521</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNzEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI5MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.8" data-doi="10.1534/g3.116.029637"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Jarquín</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Franco</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez-Rodríguez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Burgueño</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction of gene bank wheat landraces.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1834</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.029637</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.029637%26rft_id%253Dinfo%253Apmid%252F27172218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjYvNy8xODE5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI5MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.9" data-doi="10.1016/j.tplants.2017.08.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez-Rodríguez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Cuevas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Montesinos-López</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Jarquín</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic selection in plant breeding: methods, models, and perspectives.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">961</span>–<span class="cit-lpage">975</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2017.08.011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D22%26rft.spage%253D961%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2017.08.011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2017.08.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.10" data-doi="10.1534/genetics.109.101501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Naya</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Legarra</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting quantitative traits with regression models for dense molecular markers and pedigree.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">385</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.101501</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101501%26rft_id%253Dinfo%253Apmid%252F19293140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI5MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.11" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Hickey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Pong-Wong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. D.</span> <span class="cit-name-surname">Daetwyler</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P. L.</span> <span class="cit-name-surname">Calus</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-genome regression and prediction methods applied to plant and animal breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.143313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI5MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.12" data-doi="10.1086/281736"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>,  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1952</span> <span class="cit-article-title">The problem of environment and selection.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">293</span>–<span class="cit-lpage">298</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/281736</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D86%26rft.spage%253D293%26rft_id%253Dinfo%253Adoi%252F10.1086%252F281736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/281736&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1952UY00500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-book" id="cit-9.9.2925.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>,  <span class="cit-name-given-names">D. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. F. C.</span> <span class="cit-name-surname">Mackay</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source">Introduction to quantitative genetics</span>, <span class="cit-edition">Ed. 4th</span>. <span class="cit-publ-name">Pearson Education Limited</span>, <span class="cit-publ-loc">England</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.9.2925.14" data-doi="10.1534/genetics.105.049510"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Fernando</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Stella</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genomic-assisted prediction of genetic value with semiparametric procedures.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">1761</span>–<span class="cit-lpage">1776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.049510%26rft_id%253Dinfo%253Apmid%252F16648593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3My8zLzE3NjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjkyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.15" data-doi="10.1534/genetics.107.084285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>,  <span class="cit-name-given-names">D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. C. H. M.</span> <span class="cit-name-surname">van Kaam</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Reproducing kernel Hilbert spaces regression methods for genomic prediction of quantitative traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">2289</span>–<span class="cit-lpage">2303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.084285</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGianola%26rft.auinit1%253DD.%26rft.volume%253D178%26rft.issue%253D4%26rft.spage%253D2289%26rft.epage%253D2303%26rft.atitle%253DReproducing%2BKernel%2BHilbert%2BSpaces%2BRegression%2BMethods%2Bfor%2BGenomic%2BAssisted%2BPrediction%2Bof%2BQuantitative%2BTraits%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.084285%26rft_id%253Dinfo%253Apmid%252F18430950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC80LzIyODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjkyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.16" data-doi="10.1534/genetics.109.103952"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. G.</span> <span class="cit-name-surname">Hill</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Manfredi</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Fernando</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Additive genetic variability and the Bayesian alphabet.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">363</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.103952</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.103952%26rft_id%253Dinfo%253Apmid%252F19620397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgzLzEvMzQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI5MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.17" data-doi="10.1534/genetics.113.151753"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>,  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Priors in whole-genome regression: the Bayesian alphabet returns.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">596</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.151753</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.151753%26rft_id%253Dinfo%253Apmid%252F23636739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI5MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-web" id="cit-9.9.2925.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Morota</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> Genome-enabled prediction of complex traits with kernel methods: What have we learned? Proc. 10<sup>th</sup> World Congress on Genetics Applied to Livestock Production, Vancouver (refereed). <a href="https://asas.org/wcgalp-proceedings">https://asas.org/wcgalp-proceedings</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.19" data-doi="10.1007/BF00288581"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gregorius</span>,  <span class="cit-name-given-names">H.-R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Namkoong</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Joint analysis of genotypic and environmental effects.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">413</span>–<span class="cit-lpage">422</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00288581</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D72%26rft.spage%253D413%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00288581%26rft_id%253Dinfo%253Apmid%252F24247951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00288581&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24247951&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2925.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986C864600019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.20" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Fernando</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C. M.</span> <span class="cit-name-surname">Dekkers</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.081190</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjkyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.21" data-doi="10.1007/s00122-013-2243-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Lacaze</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. D.</span> <span class="cit-name-surname">Cheyron</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Daucourt</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A reaction norm model for genomic selection using high-dimensional genomic and environmental data.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">607</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-013-2243-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D595%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2243-1%26rft_id%253Dinfo%253Apmid%252F24337101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2243-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24337101&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2925.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.22" data-doi="10.3835/plantgenome2016.12.0130"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lemes da Silva</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> C. <span class="cit-name-surname">Gaynor</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Poland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Fritz</span></span>, <span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2017</span> Increasing Genomic-Enabled Prediction Accuracy by Modeling Genotype × Environment Interactions in Kansas Wheat. Plant Genome 10:0.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.12.0130</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.12.0130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.12.0130&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.23" data-doi="10.3390/agronomy8040051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Howard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Xavier</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Das Choudhury</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Increasing Predictive Ability by Modeling Interactions between Environments, Genotype and Canopy Coverage Image Data for Soybeans.</span> <abbr class="cit-jnl-abbrev">Agronomy (Basel)</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">51</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3390/agronomy8040051</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgronomy%2B%2528Basel%2529%26rft.volume%253D8%26rft.spage%253D51%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fagronomy8040051%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/agronomy8040051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.24" data-doi="10.1534/genetics.115.177907"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Reif</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Modeling epistasis in genomic selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">759</span>–<span class="cit-lpage">768</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.177907</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.177907%26rft_id%253Dinfo%253Apmid%252F26219298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAxLzIvNzU5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI5MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.25" data-doi="10.3835/plantgenome2018.03.0017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Juliana</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. P.</span> <span class="cit-name-surname">Singh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Poland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Suchismita</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, <span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">2018</span><span class="cit-comment">a</span> Prospects and challenges of applied genomic selection - a new paradigm in breeding for grain yield in bread wheat. Plant Genome<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2018.03.0017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2018.03.0017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2018.03.0017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.26" data-doi="10.1007/s00122-018-3206-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Juliana</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O. A.</span> <span class="cit-name-surname">Montesinos-López</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Mondal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. G.</span> <span class="cit-name-surname">Pérez</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">b</span> <span class="cit-article-title">Integrating genomic-enabled prediction and high-throughput phenotyping in breeding for climate-resilient bread wheat.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">177</span>–<span class="cit-lpage">194</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-018-3206-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D132%26rft.spage%253D177%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-018-3206-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-018-3206-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.27" data-doi="10.1007/s00122-016-2675-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martini</span>,  <span class="cit-name-given-names">J. W. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Wimmer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Erbe</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Simianer</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Epistasis and covariance: how gene interaction translates into genomic relationship.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">963</span>–<span class="cit-lpage">976</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2675-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D963%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2675-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2675-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>,  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Hayes</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> Prediction of total genetic value using genome-wide dense marker maps. Genetics 157: 1819–1829. doi:11290733.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.29" data-doi="10.3389/fgene.2014.00363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morota</span>,  <span class="cit-name-given-names">G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Kernel-based whole-genome prediction of complex traits: a review.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">5</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2014.00363</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D5%26rft.spage%253D363%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2014.00363%26rft_id%253Dinfo%253Apmid%252F25360145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2014.00363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25360145&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2925.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.30" data-doi="10.3389/fgene.2014.00056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morota</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Boddhireddy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Vukasinovic</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Gianola</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">DeNise</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Kernel-based variance component estimation and whole-genome prediction of pre-corrected phenotypes and progeny tests for dairy cow health traits.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">56</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2014.00056</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D5%26rft.spage%253D56%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2014.00056%26rft_id%253Dinfo%253Apmid%252F24715901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2014.00056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24715901&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2925.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.9.2925.31" data-doi="10.1534/g3.114.016097"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lopez-Cruz</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Bonnett</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Dreisigacker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Poland</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> Increased prediction accuracy in wheat breeding trials using a marker × environment interaction genomic selection model. G3:(Bethesda).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.114.016097</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.016097%26rft_id%253Dinfo%253Apmid%252F25660166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjUvNC81NjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjkyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.32" data-doi="10.1534/genetics.114.164442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>,  <span class="cit-name-given-names">P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide regression &amp; prediction with the BGLR statistical package.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">495</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.164442</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164442%26rft_id%253Dinfo%253Apmid%252F25009151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI5MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.33" data-doi="10.2135/cropsci2014.08.0577"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Bondalapati</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">De Meyer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Pita</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A Pedigree-Based reaction norm model for Prediction of Cotton Yield in Multi Environment Trials.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1143</span>–<span class="cit-lpage">1151</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2014.08.0577</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D1143%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.08.0577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.08.0577&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.34" data-doi="10.3835/plantgenome2016.09.0089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Rutkoski</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Poland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Singh</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Single-step genomic and pedigree genotype × environment interaction models for predicting wheat lines in international environments.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">0</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.09.0089</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D10%26rft.spage%253D2%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.09.0089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.09.0089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.35" data-doi="10.1371/journal.pone.0066428"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ray</span>,  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. D.</span> <span class="cit-name-surname">Mueller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. C.</span> <span class="cit-name-surname">West</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Foley</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Yield Trends Are Insufficient to Double Global Crop Production by 2050.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0066428</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0066428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0066428&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.9.2925.36" data-doi="10.1534/g3.117.042341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sousa</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Cuevas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. G.</span> <span class="cit-name-surname">de Oliveira Couto</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez-Rodríguez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Jarquín</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic-enabled prediction in maize using kernel models with genotype x environment interaction.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1995</span>–<span class="cit-lpage">2014</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.042341</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.042341%26rft_id%253Dinfo%253Apmid%252F28455415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvNi8xOTk1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI5MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-book" id="cit-9.9.2925.37" data-doi="10.19103/AS.2016.0004.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tadesse</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Amri</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Sanchez-Garcia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">El Bouhssini</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Karrou</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-source">Improving wheat production in the Central and West Asia and North Africa (CWANA) region</span>, <span class="cit-publ-name">Burleigh Dodds Science Publishing Limited</span>, <span class="cit-publ-loc">Sawston, Cambridge, UK</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.19103/AS.2016.0004.39</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImproving%2Bwheat%2Bproduction%2Bin%2Bthe%2BCentral%2Band%2BWest%2BAsia%2Band%2BNorth%2BAfrica%2B%2528CWANA%2529%2Bregion%26rft_id%253Dinfo%253Adoi%252F10.19103%252FAS.2016.0004.39%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.19103/AS.2016.0004.39&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2925.38" data-doi="10.1093/jxb/ert333"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whitford</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Fleury</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Reif</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Garcia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Okada</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Hybrid breeding in wheat: technologies to improve hybrid wheat seed production.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">5411</span>–<span class="cit-lpage">5428</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jxb/ert333</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Fert333%26rft_id%253Dinfo%253Apmid%252F24179097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/ert333&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24179097&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2925.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000329135600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/9/2925.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/9/2913" title="Deep Kernel for Genomic and Near Infrared Predictions in Multi-environment Breeding Trials" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/9/2935" title="Use of Repeated Group Measurements with Drop Out Animals for Variance Component Estimation and Genetic Evaluation: A Simulation Study" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/9"><span>Volume 9 Issue 9, September 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/9.cover-source.jpg?itok=4kZTDz_V" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/2925" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-A_gOz2J3xJuX06ezvyBucjiQDH_lpJ6FkSwIWmPMsbI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/9/2925" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/9/2925" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/9/2925" class="active" data-icon-position="" data-hide-link-title="0">Joint Use of Genome, Pedigree, and Their Interaction with Environment for Predicting the Performance of Wheat Lines in New Environments</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494053" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-lAqYSl1DPlZdNA58exSkHxIEubS3mzh8hJKTOjcjIW4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/9/2925.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/9/2925" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-7BdLxwt05OTGjb5mN0s4xIyt1yqBBP4B__vxASM9Zbs" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/9/2925" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/9/2925.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494053" id="node494053--2" data-pisa="ggg;9/9/2925" data-pisa-master="ggg;g3.119.400508" data-apath="/ggg/9/9/2925.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Joint Use of Genome, Pedigree, and Their Interaction with Environment for Predicting the Performance of Wheat Lines in New Environments</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8837-3770" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Réka Howard</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8217-2348" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel Gianola</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-4464-3385" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Osval Montesinos-López</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-6922-0173" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Philomin Juliana</span>, <span class="highwire-citation-author" data-delta="4">Ravi Singh</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-7856-1399" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jesse Poland</span>, <span class="highwire-citation-author" data-delta="6">Sandesh Shrestha</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-3202-1784" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-5098-2060" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Diego Jarquín</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2925-2934;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400508 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494053--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Réka Howard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Nebraska – Lincoln, Lincoln NE, 68583, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R%C3%A9ka%2BHoward%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Howard%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AR%25C3%25A9ka%2BHoward%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8837-3770" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Réka Howard</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel Gianola </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>University of Wisconsin-Madison, Madison, Wisconsin 53706, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BGianola%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gianola%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BGianola%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8217-2348" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel Gianola</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Osval Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Universidad de Colima, CP 28040 Colima, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOsval%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4464-3385" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Osval Montesinos-López</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Philomin Juliana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>International Maize and Wheat Improvement Center (CIMMYT), Km. 45, Carretera México-Veracruz, El Batán, Texcoco, Edo. de México, CP 56130, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philomin%2BJuliana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Juliana%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APhilomin%2BJuliana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6922-0173" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Philomin Juliana</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ravi Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>International Maize and Wheat Improvement Center (CIMMYT), Km. 45, Carretera México-Veracruz, El Batán, Texcoco, Edo. de México, CP 56130, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ravi%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARavi%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jesse Poland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>USDA-ARS and Dep. of Plant Pathology, Kansas State University, 4024 Throckmorton Hall, Manhattan KS, 66506, USA, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BPoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poland%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJesse%2BPoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7856-1399" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jesse Poland</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Sandesh Shrestha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>USDA-ARS and Dep. of Plant Pathology, Kansas State University, 4024 Throckmorton Hall, Manhattan KS, 66506, USA, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sandesh%2BShrestha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shrestha%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASandesh%2BShrestha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Colegio de Postgraduados, Montecillos, Edo. de México, CP 56230, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3202-1784" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Paulino Pérez-Rodríguez</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>International Maize and Wheat Improvement Center (CIMMYT), Km. 45, Carretera México-Veracruz, El Batán, Texcoco, Edo. de México, CP 56130, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:diego.jarquin@gmail.com" class="" data-icon-position="" data-hide-link-title="0">diego.jarquin@gmail.com</a></span></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Diego Jarquín </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Nebraska – Lincoln, Lincoln NE, 68583, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BJarqu%C3%ADn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarqu%C3%ADn%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADiego%2BJarqu%25C3%25ADn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5098-2060" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Diego Jarquín</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:diego.jarquin@gmail.com" class="" data-icon-position="" data-hide-link-title="0">diego.jarquin@gmail.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/9/2925" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/9/2925&amp;title=Joint%20Use%20of%20Genome%2C%20Pedigree%2C%20and%20Their%20Interaction%20with%20Environment%20for%20Predicting%20the%20Performance%20of%20Wheat%20Lines%20in%20New%20Environments" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/9/2925&amp;title=Joint%20Use%20of%20Genome%2C%20Pedigree%2C%20and%20Their%20Interaction%20with%20Environment%20for%20Predicting%20the%20Performance%20of%20Wheat%20Lines%20in%20New%20Environments" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/9/2925&amp;title=Joint%20Use%20of%20Genome%2C%20Pedigree%2C%20and%20Their%20Interaction%20with%20Environment%20for%20Predicting%20the%20Performance%20of%20Wheat%20Lines%20in%20New%20Environments" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/9/2925&amp;text=Joint%20Use%20of%20Genome%2C%20Pedigree%2C%20and%20Their%20Interaction%20with%20Environment%20for%20Predicting%20the%20Performance%20of%20Wheat%20Lines%20in%20New%20Environments" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/9/2925&amp;title=Joint%20Use%20of%20Genome%2C%20Pedigree%2C%20and%20Their%20Interaction%20with%20Environment%20for%20Predicting%20the%20Performance%20of%20Wheat%20Lines%20in%20New%20Environments" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/9/2925&amp;t=Joint%20Use%20of%20Genome%2C%20Pedigree%2C%20and%20Their%20Interaction%20with%20Environment%20for%20Predicting%20the%20Performance%20of%20Wheat%20Lines%20in%20New%20Environments" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/9/2925&amp;title=Joint%20Use%20of%20Genome%2C%20Pedigree%2C%20and%20Their%20Interaction%20with%20Environment%20for%20Predicting%20the%20Performance%20of%20Wheat%20Lines%20in%20New%20Environments" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/9/2925&amp;title=Joint%20Use%20of%20Genome%2C%20Pedigree%2C%20and%20Their%20Interaction%20with%20Environment%20for%20Predicting%20the%20Performance%20of%20Wheat%20Lines%20in%20New%20Environments" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494053/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494053" id="node494053--3" data-pisa="ggg;9/9/2925" data-pisa-master="ggg;g3.119.400508" data-apath="/ggg/9/9/2925.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Joint Use of Genome, Pedigree, and Their Interaction with Environment for Predicting the Performance of Wheat Lines in New Environments</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8837-3770" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Réka Howard</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8217-2348" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel Gianola</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-4464-3385" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Osval Montesinos-López</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-6922-0173" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Philomin Juliana</span>, <span class="highwire-citation-author" data-delta="4">Ravi Singh</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-7856-1399" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jesse Poland</span>, <span class="highwire-citation-author" data-delta="6">Sandesh Shrestha</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-3202-1784" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-5098-2060" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Diego Jarquín</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2925-2934;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400508 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494053--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Réka Howard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Nebraska – Lincoln, Lincoln NE, 68583, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R%C3%A9ka%2BHoward%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Howard%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AR%25C3%25A9ka%2BHoward%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8837-3770" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Réka Howard</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel Gianola </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>University of Wisconsin-Madison, Madison, Wisconsin 53706, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BGianola%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gianola%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BGianola%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8217-2348" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel Gianola</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Osval Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Universidad de Colima, CP 28040 Colima, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOsval%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4464-3385" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Osval Montesinos-López</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Philomin Juliana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>International Maize and Wheat Improvement Center (CIMMYT), Km. 45, Carretera México-Veracruz, El Batán, Texcoco, Edo. de México, CP 56130, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philomin%2BJuliana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Juliana%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APhilomin%2BJuliana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6922-0173" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Philomin Juliana</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ravi Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>International Maize and Wheat Improvement Center (CIMMYT), Km. 45, Carretera México-Veracruz, El Batán, Texcoco, Edo. de México, CP 56130, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ravi%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARavi%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jesse Poland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>USDA-ARS and Dep. of Plant Pathology, Kansas State University, 4024 Throckmorton Hall, Manhattan KS, 66506, USA, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BPoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poland%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJesse%2BPoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7856-1399" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jesse Poland</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Sandesh Shrestha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>USDA-ARS and Dep. of Plant Pathology, Kansas State University, 4024 Throckmorton Hall, Manhattan KS, 66506, USA, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sandesh%2BShrestha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shrestha%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASandesh%2BShrestha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Colegio de Postgraduados, Montecillos, Edo. de México, CP 56230, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3202-1784" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Paulino Pérez-Rodríguez</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>International Maize and Wheat Improvement Center (CIMMYT), Km. 45, Carretera México-Veracruz, El Batán, Texcoco, Edo. de México, CP 56130, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:diego.jarquin@gmail.com" class="" data-icon-position="" data-hide-link-title="0">diego.jarquin@gmail.com</a></span></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Diego Jarquín </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Nebraska – Lincoln, Lincoln NE, 68583, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BJarqu%C3%ADn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarqu%C3%ADn%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADiego%2BJarqu%25C3%25ADn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5098-2060" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Diego Jarquín</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:diego.jarquin@gmail.com" class="" data-icon-position="" data-hide-link-title="0">diego.jarquin@gmail.com</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2925&amp;rft.epage=2934&amp;rft.atitle=Joint%20Use%20of%20Genome%2C%20Pedigree%2C%20and%20Their%20Interaction%20with%20Environment%20for%20Predicting%20the%20Performance%20of%20Wheat%20Lines%20in%20New%20Environments&amp;rft.volume=9&amp;rft.issue=9&amp;rft.date=2019-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Howard%2C+R%C3%A9ka&amp;rft.au=Gianola%2C+Daniel&amp;rft.au=Montesinos-L%C3%B3pez%2C+Osval&amp;rft.au=Juliana%2C+Philomin&amp;rft.au=Singh%2C+Ravi&amp;rft.au=Poland%2C+Jesse&amp;rft.au=Shrestha%2C+Sandesh&amp;rft.au=P%C3%A9rez-Rodr%C3%ADguez%2C+Paulino&amp;rft.au=Crossa%2C+Jos%C3%A9&amp;rft.au=Jarqu%C3%ADn%2C+Diego"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494053/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494053/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494053/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494053/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494053/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494053/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494053/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494053/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494053/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494053/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494053/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494053/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/9/2925" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/9/2925.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/9/2925.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/9/2925.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__CK8qNg7zHnCPfipcSU0qjCyIiw2pe-WKGx2V0TgizGw__wprDSLLH9CuacxJkxX2DuWt0EaauZA-f_LGjovfY3aw__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
