<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/9/2977" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/9/2977.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/9/2977.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/9/2977.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/9/2977.atom" />
<meta name="HW.pisa" content="ggg;9/9/2977" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="FERMI: A Novel Method for Sensitive Detection of Rare Mutations in Somatic Tissue" />
<meta name="DC.Identifier" content="10.1534/g3.119.400438" />
<meta name="DC.Date" content="2019-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Liggett et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="With growing interest in monitoring mutational processes in normal tissues, tumor heterogeneity, and cancer evolution under therapy, the ability to accurately and economically detect ultra-rare mutations is becoming increasingly important. However, this capability has often been compromised by significant sequencing, PCR and DNA preparation error rates. Here, we describe FERMI (Fast Extremely Rare Mutation Identification) - a novel method designed to eliminate the majority of these sequencing and library-preparation errors in order to significantly improve rare somatic mutation detection. This method leverages barcoded targeting probes to capture and sequence DNA of interest with single copy resolution. The variant calls from the barcoded sequencing data are then further filtered in a position-dependent fashion against an adaptive, context-aware null model in order to distinguish true variants. As a proof of principle, we employ FERMI to probe bone marrow biopsies from leukemia patients, and show that rare mutations and clonal evolution can be tracked throughout cancer treatment, including during historically intractable periods like minimum residual disease. Importantly, FERMI is able to accurately detect nascent clonal expansions within leukemias in a manner that may facilitate the early detection and characterization of cancer relapse." />
<meta name="DC.Contributor" content="L. Alexander Liggett" />
<meta name="DC.Contributor" content="Anchal Sharma" />
<meta name="DC.Contributor" content="Subhajyoti De" />
<meta name="DC.Contributor" content="James DeGregori" />
<meta name="article:published_time" content="2019-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="FERMI: A Novel Method for Sensitive Detection of Rare Mutations in Somatic Tissue" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;With growing interest in monitoring mutational processes in normal tissues, tumor heterogeneity, and cancer evolution under therapy, the ability to accurately and economically detect ultra-rare mutations is becoming increasingly important. However, this capability has often been compromised by significant sequencing, PCR and DNA preparation error rates. Here, we describe FERMI (Fast Extremely Rare Mutation Identification) - a novel method designed to eliminate the majority of these sequencing and library-preparation errors in order to significantly improve rare somatic mutation detection. This method leverages barcoded targeting probes to capture and sequence DNA of interest with single copy resolution. The variant calls from the barcoded sequencing data are then further filtered in a position-dependent fashion against an adaptive, context-aware null model in order to distinguish true variants. As a proof of principle, we employ FERMI to probe bone marrow biopsies from leukemia patients, and show that rare mutations and clonal evolution can be tracked throughout cancer treatment, including during historically intractable periods like minimum residual disease. Importantly, FERMI is able to accurately detect nascent clonal expansions within leukemias in a manner that may facilitate the early detection and characterization of cancer relapse.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/09/01" />
<meta name="citation_mjid" content="ggg;9/9/2977" />
<meta name="citation_id" content="9/9/2977" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/9/2977" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/9/2977.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/9/2977.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/9/2977.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400438" />
<meta name="citation_pmid" content="31352405" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2977" />
<meta name="citation_lastpage" content="2987" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="L. Alexander Liggett" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO 80045" />
<meta name="citation_author_institution" content="Linda Crnic Institute for Down Syndrome, University of Colorado School of Medicine, Aurora, CO 80045" />
<meta name="citation_author" content="Anchal Sharma" />
<meta name="citation_author_institution" content="Rutgers Cancer Institute, New Brunswick, NJ 08901" />
<meta name="citation_author" content="Subhajyoti De" />
<meta name="citation_author_institution" content="Rutgers Cancer Institute, New Brunswick, NJ 08901" />
<meta name="citation_author" content="James DeGregori" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO 80045" />
<meta name="citation_author_institution" content="Linda Crnic Institute for Down Syndrome, University of Colorado School of Medicine, Aurora, CO 80045" />
<meta name="citation_author_institution" content="Integrated Department of Immunology" />
<meta name="citation_author_institution" content="Department of Pediatrics" />
<meta name="citation_author_institution" content="Department of Medicine, Section of Hematology, University of Colorado School of Medicine, Aurora, CO 80045" />
<meta name="citation_author_email" content="james.degregori@ucdenver.edu" />
<meta name="citation_reference" content="citation_journal_title=J. Vis. Exp.;citation_author=A. Z. Albitar;citation_author=W. Ma;citation_author=M. Albitar;citation_title=Wild-type Blocking PCR Combined with Direct Sequencing as a Highly Sensitive Method for Detection of Low-Frequency Somatic Mutations.;citation_year=2017;citation_issue=121;citation_doi=10.3791/55130" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=L. B. Alexandrov;citation_author=S. Nik-Zainal;citation_author=D. C. Wedge;citation_author=S. A. J. R. Aparicio;citation_author=S. Behjati;citation_title=Signatures of mutational processes in human cancer.;citation_pages=415-421;citation_volume=500;citation_year=2013;citation_pmid=23945592;citation_doi=10.1038/nature12477" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=S. Behjati;citation_author=M. Huch;citation_author=R. van Boxtel;citation_author=W. Karthaus;citation_author=D. C. Wedge;citation_title=Genome sequencing of normal cells reveals developmental lineages and mutational processes.;citation_pages=422-425;citation_volume=513;citation_year=2014;citation_pmid=25043003;citation_doi=10.1038/nature13448" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=S. Benzer;citation_title=ON THE TOPOGRAPHY OF THE GENETIC FINE STRUCTURE;citation_pages=403-415;citation_volume=47;citation_year=1961;citation_issue=3;citation_pmid=16590840;citation_doi=10.1073/pnas.47.3.403" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=F. Blokzijl;citation_author=J. de Ligt;citation_author=M. Jager;citation_author=V. Sasselli;citation_author=S. Roerink;citation_title=Tissue-specific mutation accumulation in human adult stem cells during life.;citation_pages=260-264;citation_volume=538;citation_year=2016;citation_pmid=27698416;citation_doi=10.1038/nature19768" />
<meta name="citation_reference" content="citation_journal_title=Cancer Discov.;citation_author=A. A. Chaudhuri;citation_author=J. J. Chabon;citation_author=A. F. Lovejoy;citation_author=A. M. Newman;citation_author=H. Stehr;citation_title=Early Detection of Molecular Residual Disease in Localized Lung Cancer by Circulating Tumor DNA Profiling.;citation_volume=7;citation_year=2017;citation_pmid=28899864;citation_doi=10.1158/2159-8290.CD-17-0716" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=K. C. Cheng;citation_author=D. S. Cahill;citation_author=H. Kasai;citation_author=S. Nishimura;citation_author=L. A. Loeb;citation_title=8-Hydroxyguanine, an abundant form of oxidative DNA damage, causes G---- T and A----C substitutions;citation_pages=166-172;citation_volume=267;citation_year=1992;citation_issue=1;citation_pmid=1730583" />
<meta name="citation_reference" content="citation_journal_title=Trends Cancer Res.;citation_author=N. Coleman;citation_author=S. De;citation_title=Mutation Signatures Depend on Epigenomic Contexts.;citation_pages=659-661;citation_volume=4;citation_year=2018;citation_doi=10.1016/j.trecan.2018.08.001" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. Dressman;citation_author=H. Yan;citation_author=G. Traverso;citation_author=K. W. Kinzler;citation_author=B. Vogelstein;citation_title=Transforming single DNA molecules into fluorescent magnetic particles for detection and enumeration of genetic variations.;citation_volume=100;citation_year=2003;citation_pmid=12857956;citation_doi=10.1073/pnas.1133470100" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=P. Flaherty;citation_author=G. Natsoulis;citation_author=O. Muralidharan;citation_author=M. Winters;citation_author=J. Buenrostro;citation_title=Ultrasensitive detection of rare mutations using next-generation targeted resequencing.;citation_volume=40;citation_year=2012;citation_pmid=22013163;citation_doi=10.1093/nar/gkr861" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=P. A. Fujita;citation_author=B. Rhead;citation_author=A. S. Zweig;citation_author=A. S. Hinrichs;citation_author=D. Karolchik;citation_title=The UCSC genome browser database: update 2011.;citation_volume=39;citation_year=2010;citation_pmid=20959295;citation_doi=10.1093/nar/gkq963" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. J. Gaffney;citation_author=P. D. Keightley;citation_title=The scale of mutational variation in the murid genome.;citation_volume=15;citation_year=2005;citation_pmid=16024822;citation_doi=10.1101/gr.3895005" />
<meta name="citation_reference" content="Garrison, E., and G. Marth, 2012 Haplotype-based variant detection from short-read sequencing. arXiv [q-bio.GN]." />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Greaves;citation_author=C. C. Maley;citation_title=Clonal evolution in cancer.;citation_pages=306-313;citation_volume=481;citation_year=2012;citation_issue=7381;citation_pmid=22258609;citation_doi=10.1038/nature10762" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. B. Hiatt;citation_author=C. C. Pritchard;citation_author=S. J. Salipante;citation_author=B. J. O’Roak;citation_author=J. Shendure;citation_title=Single molecule molecular inversion probes for targeted, high-accuracy detection of low-frequency variation.;citation_volume=23;citation_year=2013;citation_pmid=23382536;citation_doi=10.1101/gr.147686.112" />
<meta name="citation_reference" content="citation_journal_title=Analytical Chemistry (Washington, DC);citation_journal_abbrev=Analytical Chemistry (Washington, DC);citation_author=B. J. Hindson;citation_author=K. D. Ness;citation_author=D. A. Masquelier;citation_author=P. Belgrader;citation_author=N. J. Heredia;citation_title=High-throughput droplet digital PCR system for absolute quantitation of DNA copy number.;citation_pages=8604-8610;citation_volume=83;citation_year=2011;citation_issue=22;citation_pmid=22035192;citation_doi=10.1021/ac202028g" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. G. Hwang;citation_author=P. Green;citation_title=Bayesian Markov chain Monte Carlo sequence analysis reveals varying neutral substitution patterns in mammalian evolution.;citation_volume=101;citation_year=2004;citation_pmid=15292512;citation_doi=10.1073/pnas.0404142101" />
<meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.;citation_author=A. Ivey;citation_author=R. K. Hills;citation_author=M. A. Simpson;citation_author=J. V. Jovanovic;citation_author=A. Gilkes;citation_title=Assessment of Minimal Residual Disease in Standard-Risk AML.;citation_pages=422-433;citation_volume=374;citation_year=2016;citation_pmid=26789727;citation_doi=10.1056/NEJMoa1507471" />
<meta name="citation_reference" content="citation_journal_title=Nat. Protoc.;citation_author=S. R. Kennedy;citation_author=M. W. Schmitt;citation_author=E. J. Fox;citation_author=B. F. Kohrn;citation_author=J. J. Salk;citation_title=Detecting ultralow-frequency mutations by Duplex Sequencing.;citation_pages=2586-2606;citation_volume=9;citation_year=2014;citation_pmid=25299156;citation_doi=10.1038/nprot.2014.170" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=S. Kim;citation_author=K. Jeong;citation_author=K. Bhutani;citation_author=J. Lee;citation_author=A. Patel;citation_title=Virmid: accurate detection of somatic mutations with sample impurity inference.;citation_pages=R90;citation_volume=14;citation_year=2013;citation_pmid=23987214;citation_doi=10.1186/gb-2013-14-8-r90" />
<meta name="citation_reference" content="citation_journal_title=J. Clin. Oncol.;citation_author=J. Krönke;citation_author=R. F. Schlenk;citation_author=K.-O. Jensen;citation_author=F. Tschürtz;citation_author=A. Corbacioglu;citation_title=Monitoring of minimal residual disease in NPM1-mutated acute myeloid leukemia: a study from the German-Austrian acute myeloid leukemia study group.;citation_volume=29;citation_year=2011;citation_pmid=21555683;citation_doi=10.1200/JCO.2011.35.0371" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. S. Lander;citation_author=L. M. Linton;citation_author=B. Birren;citation_author=C. Nusbaum;citation_author=M. C. Zody;citation_title=Initial sequencing and analysis of the human genome.;citation_pages=860-921;citation_volume=409;citation_year=2001;citation_issue=6822;citation_pmid=11237011;citation_doi=10.1038/35057062" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=M. J. Lercher;citation_author=E. J. B. Williams;citation_author=L. D. Hurst;citation_title=Local Similarity in Evolutionary Rates Extends over Whole Chromosomes in Human-Rodent and Mouse-Rat Comparisons: Implications for Understanding the Mechanistic Basis of the Male Mutation Bias;citation_pages=2032-2039;citation_volume=18;citation_year=2001;citation_issue=11;citation_pmid=11606699;citation_doi=10.1093/oxfordjournals.molbev.a003744" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows–Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Biochemistry;citation_author=T. Lindahl;citation_author=B. Karlstrom;citation_title=Heat-induced depyrimidination of deoxyribonucleic acid in neutral solution.;citation_pages=5151-5154;citation_volume=25;citation_year=1973;citation_doi=10.1021/bi00749a020" />
<meta name="citation_reference" content="citation_journal_title=Biochemistry (Easton);citation_journal_abbrev=Biochemistry (Easton);citation_author=T. Lindahl;citation_author=B. Nyberg;citation_title=Heat-induced deamination of cytosine residues in deoxyribonucleic acid.;citation_pages=3405-3410;citation_volume=13;citation_year=1974;citation_issue=16;citation_pmid=4601435;citation_doi=10.1021/bi00713a035" />
<meta name="citation_reference" content="citation_journal_title=Nature medicine;citation_journal_abbrev=Nat Med;citation_author=J. Li;citation_author=L. Wang;citation_author=H. Mamon;citation_author=M. H. Kulke;citation_author=R. Berbeco;citation_title=Replacing PCR with COLD-PCR enriches variant DNA sequences and redefines the sensitivity of genetic testing.;citation_pages=579-584;citation_volume=14;citation_year=2008;citation_issue=5;citation_pmid=18408729;citation_doi=10.1038/nm1708" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Lynch;citation_title=Mutation and Human Exceptionalism: Our Future Genetic Load;citation_pages=869-875;citation_volume=202;citation_year=2016;citation_issue=3;citation_pmid=26953265;citation_doi=10.1534/genetics.115.180471" />
<meta name="citation_reference" content="citation_journal_title=Clin. Chem.;citation_author=S. Mansukhani;citation_author=L. J. Barber;citation_author=D. Kleftogiannis;citation_author=S. Y. Moorcraft;citation_author=M. Davidson;citation_title=Ultra-Sensitive Mutation Detection and Genome-Wide DNA Copy Number Reconstruction by Error-Corrected Circulating Tumor DNA Sequencing.;citation_volume=64;citation_year=2018;citation_pmid=30150316;citation_doi=10.1373/clinchem.2018.289629" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=I. Martincorena;citation_author=A. Roshan;citation_author=M. Gerstung;citation_author=P. Ellis;citation_author=P. Van Loo;citation_title=High burden and pervasive positive selection of somatic mutations in normal human skin;citation_pages=880-886;citation_volume=348;citation_year=2015;citation_issue=6237;citation_pmid=25999502;citation_doi=10.1126/science.aaa6806" />
<meta name="citation_reference" content="citation_journal_title=Clin. Chem.;citation_author=C. A. Milbury;citation_author=M. Correll;citation_author=J. Quackenbush;citation_author=R. Rubio;citation_author=G. M. Makrigiorgos;citation_title=COLD-PCR enrichment of rare cancer mutations prior to targeted amplicon resequencing.;citation_volume=58;citation_year=2012;citation_pmid=22194627;citation_doi=10.1373/clinchem.2011.176198" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. W. Nachman;citation_author=S. L. Crowell;citation_title=Estimate of the Mutation Rate per Nucleotide in Humans;citation_pages=297-304;citation_volume=156;citation_year=2000;citation_issue=1;citation_pmid=10978293" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=A. M. Newman;citation_author=A. F. Lovejoy;citation_author=D. M. Klass;citation_author=D. M. Kurtz;citation_author=J. J. Chabon;citation_title=Integrated digital error suppression for improved detection of circulating tumor DNA.;citation_pages=547-555;citation_volume=34;citation_year=2016;citation_pmid=27018799;citation_doi=10.1038/nbt.3520" />
<meta name="citation_reference" content="citation_journal_title=Comput. Sci. Eng.;citation_author=T. E. Oliphant;citation_title=Python for Scientific Computing.;citation_pages=10-20;citation_volume=9;citation_year=2007;citation_doi=10.1109/MCSE.2007.58" />
<meta name="citation_reference" content="citation_journal_title=Haematologica;citation_author=E. Onecha;citation_author=M. Linares;citation_author=I. Rapado;citation_author=Y. Ruiz-Heredia;citation_author=P. Martinez-Sanchez;citation_title=A Novel deep targeted sequencing method for minimal residual disease monitoring in acute myeloid leukemia.;citation_volume=104;citation_year=2019;citation_pmid=30093399;citation_doi=10.3324/haematol.2018.194712" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. L. Preston;citation_author=A. E. Royall;citation_author=M. A. Randel;citation_author=K. L. Sikkink;citation_author=P. C. Phillips;citation_title=High-specificity detection of rare alleles with Paired-End Low Error Sequencing (PELE-Seq).;citation_pages=464;citation_volume=17;citation_year=2016;citation_doi=10.1186/s12864-016-2669-3" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=M. W. Schmitt;citation_author=E. J. Fox;citation_author=M. J. Prindle;citation_author=K. S. Reid-Bayliss;citation_author=L. D. True;citation_title=Sequencing small genomic targets with high efficiency and extreme accuracy.;citation_pages=423-425;citation_volume=12;citation_year=2015;citation_pmid=25849638;citation_doi=10.1038/nmeth.3351" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Shibutani;citation_author=M. Takeshita;citation_author=A. P. Grollman;citation_title=Insertion of specific bases during DNA synthesis past the oxidation-damaged base 8-oxodG.;citation_pages=431-434;citation_volume=349;citation_year=1991;citation_issue=6308;citation_pmid=1992344;citation_doi=10.1038/349431a0" />
<meta name="citation_reference" content="citation_journal_title=BioTechniques;citation_journal_abbrev=Biotechniques;citation_author=P. J. Sykes;citation_author=S. H. Neoh;citation_author=M. J. Brisco;citation_author=E. Hughes;citation_author=J. Condon;citation_title=Quantitation of targets for PCR by use of limiting dilution.;citation_pages=444-449;citation_volume=13;citation_year=1992;citation_issue=3;citation_pmid=1389177" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Tan;citation_author=G. R. Abecasis;citation_author=H. M. Kang;citation_title=Unified representation of genetic variants.;citation_volume=31;citation_year=2015;citation_pmid=25701572;citation_doi=10.1093/bioinformatics/btv112" />
<meta name="citation_reference" content="citation_journal_title=J. Clin. Oncol.;citation_author=M. Terwijn;citation_author=W. L. J. van Putten;citation_author=A. Kelder;citation_author=V. H. J. van der Velden;citation_author=R. A. Brooimans;citation_title=High prognostic impact of flow cytometric minimal residual disease detection in acute myeloid leukemia: data from the HOVON/SAKK AML 42A study.;citation_volume=31;citation_year=2013;citation_pmid=24062400;citation_doi=10.1200/JCO.2012.45.9628" />
<meta name="citation_reference" content="citation_journal_title=Blood;citation_author=F. Thol;citation_author=R. Gabdoulline;citation_author=A. Liebich;citation_author=P. Klement;citation_author=J. Schiller;citation_title=Measurable residual disease monitoring by NGS before allogeneic hematopoietic cell transplantation in AML.;citation_volume=132;citation_year=2018;citation_pmid=30190321;citation_doi=10.1182/blood-2018-02-829911" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=B. Vogelstein;citation_author=K. W. Kinzler;citation_title=Digital PCR;citation_pages=9236-9241;citation_volume=96;citation_year=1999;citation_issue=16;citation_pmid=10430926;citation_doi=10.1073/pnas.96.16.9236" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=A. L. Young;citation_author=G. A. Challen;citation_author=B. M. Birmann;citation_author=T. E. Druley;citation_title=Clonal haematopoiesis harbouring AML-associated mutations is ubiquitous in healthy adults.;citation_pages=12484;citation_volume=7;citation_year=2016;citation_doi=10.1038/ncomms12484" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="FERMI: A Novel Method for Sensitive Detection of Rare Mutations in Somatic Tissue" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
<meta name="twitter:description" content="With growing interest in monitoring mutational processes in normal tissues, tumor heterogeneity, and cancer evolution under therapy, the ability to accurately and economically detect ultra-rare mutations is becoming increasingly important. However, this capability has often been compromised by significant sequencing, PCR and DNA preparation error rates. Here, we describe FERMI (Fast Extremely Rare Mutation Identification) - a novel method designed to eliminate the majority of these sequencing and library-preparation errors in order to significantly improve rare somatic mutation detection. This method leverages barcoded targeting probes to capture and sequence DNA of interest with single copy resolution. The variant calls from the barcoded sequencing data are then further filtered in a position-dependent fashion against an adaptive, context-aware null model in order to distinguish true variants. As a proof of principle, we employ FERMI to probe bone marrow biopsies from leukemia patients, and show that rare mutations and clonal evolution can be tracked throughout cancer treatment, including during historically intractable periods like minimum residual disease. Importantly, FERMI is able to accurately detect nascent clonal expansions within leukemias in a manner that may facilitate the early detection and characterization of cancer relapse." />
<meta name="og-title" property="og:title" content="FERMI: A Novel Method for Sensitive Detection of Rare Mutations in Somatic Tissue" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/9/2977" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="With growing interest in monitoring mutational processes in normal tissues, tumor heterogeneity, and cancer evolution under therapy, the ability to accurately and economically detect ultra-rare mutations is becoming increasingly important. However, this capability has often been compromised by significant sequencing, PCR and DNA preparation error rates. Here, we describe FERMI (Fast Extremely Rare Mutation Identification) - a novel method designed to eliminate the majority of these sequencing and library-preparation errors in order to significantly improve rare somatic mutation detection. This method leverages barcoded targeting probes to capture and sequence DNA of interest with single copy resolution. The variant calls from the barcoded sequencing data are then further filtered in a position-dependent fashion against an adaptive, context-aware null model in order to distinguish true variants. As a proof of principle, we employ FERMI to probe bone marrow biopsies from leukemia patients, and show that rare mutations and clonal evolution can be tracked throughout cancer treatment, including during historically intractable periods like minimum residual disease. Importantly, FERMI is able to accurately detect nascent clonal expansions within leukemias in a manner that may facilitate the early detection and characterization of cancer relapse." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
    <title>FERMI: A Novel Method for Sensitive Detection of Rare Mutations in Somatic Tissue | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"uQgQTWGY8ynZMljoW9NJroCkUEnWQgflZ7j08mzbF50","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494055","apath":"\/ggg\/9\/9\/2977.atom","pisa":"ggg;9\/9\/2977","ac":{"\/ggg\/9\/9\/2977.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/9\/2977.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/9\/2977"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/9\/2977"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/9\/2977":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022lwNp2FFY+s1PlCmpVRRdYqk7+S9mLosmaVxMyfavvbigANZyvdyEYjjsPok63dQhAzniyNxlP9SjG9F5wxvuZPkvJnK8SLIrYrbRUJTFcDdzB84c5lWj+YNEnpaIthAuxM0aVn\\\\\\\/fCpSdXpsbVoGZfHzOylYKfq8nRAh\\\\\\\/W3bS7tRHjbcSXuE+I2JuCZi0R8\\\\\\\/FbzYu9jfUliyWo8GwKVQRHTuFrVJmAc8\\\\\\\/bkLm\\\\\\\/8J8CgvTKBe3kcavz\\\\\\\/JNxwLtSi7qpvtjuQZ8XapqFc+1rt6UwCs\\\\\\\/D8bAwMUl9ZArT3XLfL1UxBoP5DoCerkHFbcn9WJyDdWUIOvhzyv5mG3\\\\\\\/d5vmf+XhkatStdDRmEx4AM\\\\\\\/2qUq2d3QZmRr72ak\\\\\\\/nBcwcwhi\\\\\\\/DqvzZZ91dFyTRqNZNHw0B2gC2VAyUk\\\\\\\/7jJ2+0sWBLI3oNYAI8vPSpojIv\\\\\\\/nyUFmzyx7G7eGv+tALx2IXFCMzBkzVkSormZIZya6S2vZOM3VgUpzYyiMsm9XABCLrBY91vifzLaue+cqOdmRfk4MmUt6Vr30WidXK6bCkQBL5GUmGuh7LylCDXxbzk7RxNnnZdkzq2KayUIcNbY1I+KfihcVJV4QhUj+rSpeouzL9WScfTTJcDkAAqSjuCX3Cy0SBNm9cnmaS8xvAYXoyPiRQVhtef6sRWZ9yExl95G0XFuoNrwJ0NB7raL\\\\\\\/+347W5qx6xmkqfpzG1USPHhUvqACXwBKENqnUjpqnPVw+lugmiser\\\\\\\/JauosiFPwnxP7eN47V+OyL6AGhlhA9nXbieKipvC8sqVXZKpjKjK6hYhCVC6Y4g6lo6hQa1QByrLpBSCcvFw3wiLTkhP+b4IFH62\\\\\\\/zu2AHbJ5RAdXqWdKgdIY04TA+GXMrtYp7oUPJa3c+i0rhqjOzrD3OW8EBXR4l8qPSo6jl7YtWRfzbQkGB4TUgsHrR5zNgTzoShEqwyu11xOY7IOVeMsI+GIxcYtKwduq1M8Srunbdr5Rf+\\\\\\\/cv5A6tKfuNlKy7RO11\\\\\\\/whjQXKTchgk8MF8t+8OxtN15U0xYWUe+XY3ZtjO2LTF4gk2in8YiNRnyuv89iEc+nFl8JXGnNrQ3q6TcEEpDduuEcvRAw8TBWklhrZ9\\\\\\\/HYae9CtNfJXcRSIHHBnHINcZXzrVSfKNRLwPm1E\\\\\\\/Pyop1m+2eAhP3wKmrDj4gRSpCoMd8ty8N30dE2lp5EjTRVjjorP595yHpvVJRwdYxm6qrsCE41ydEnXJfdYORjXqpVCFr5O747O8ZBsouOwo+J+mnwAR6mnhCUA\\\u0022,\\\u0022iv\\\u0022:\\\u0022CB8L6p58CBu3WEiJThINEA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223386bad2d7ce0e31fcabe7d75ceee2a385ab789800668f7805700c5be88016cd\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022v3exyDdj1EgE0tSfbKqt7obTr81uUPu4STgcPMWgkyVQGHOs0WSRY6203ePmcdfu2wIFye6XSr8hvAD\\\\\\\/ieRbqHkEw\\\\\\\/bDcYCOvKYZ4NYxwLXLASccu89r5TzdrumpyECG7cL71BnO3X+xRbj3ceo5t7N8sBj6cYvMkhB84zp4gnm1atWwYhnEZZoeEVTPdheW7NHMXr1k6tpxF04zNiGFgOOEz4rihwUnFjq9Gyx95H13uU8eHQZfYE6FKyxW1Dadv7CcVDwH+cUpBogXBdgux4FGBw3J3gFY9ri7ZsHbeNsjuWzNQn\\\\\\\/NemlKpE63e6LD2pAJQi58LR+c8l0SRC2oJHT2JS29V93NroQnQHOP9Fm8nfjAcCulGpREI12nH67hsBgCC0XnOTPRkkhKYxtX8F\\\\\\\/mBIxq2vJqke4ivjGvflgeuk\\\\\\\/oUlcHdff6JjOFmoqcygKZbJVQ4NKps4XveTtxxVNUG4WcFAGoQT1svDVRB1qeHsA5Aludc\\\\\\\/yaByJXKymhhMNwPmTcLnLLSl3wZkbWwp5qx8D01Fr+70jn3hSUxtmTTlCmm6X5LGddjR612+3OhWcUS6RuhNwUsvfi\\\\\\\/pBzCKLT7idZh4D7Dn6t3lcdeW4kJkgsoTsG6MA2Cc3T9SlNh\\\\\\\/zhsBL\\\\\\\/PaT5WCjtFQJ7lK\\\\\\\/wAZq7rNDo9pm6RczBZkYx5Zbu7utgfRdtDqZLv8mdpwN1TiDcANf5sSmtZRIq2CWKyHxv5\\\\\\\/2vhw+IQVE3+wpQ355NGvbSd3waSyb2PpeWPYfdiK0hMfXQfSUFUJJcwuU4IgJ08vJdDuls2RXAkTWKebf7vf1XOUh43Nqku1sawTAsK02hqRZgrwt5pscnDfQQM9UxMYlv7BhPh1chMwRnc23khBpvhjgO6bUQHHxkXxNCHPGFHg+45KXAXBhkCFQd9QIt+YCsf1+AvfGh2oq4\\\\\\\/Fr1opwr2sS+FEJGyhcKqbbwlXnWglpINSN7rw5AqtHn8to4HbBmPJ8xt6RiyqnUQzx82SkijngrcnpBMgltgH0xDHTrfcF0N68rxsy2Emagio6op5rlirca5PZ66qMiGavt3ungEvheipNXqL5U80r6qxKLIMPGJRPwXcrxFZK6gAtaScx8e1frB0Cpf5DIOf0I\\\\\\\/2Yas6IygYpnlUdLXw\\\\\\\/jwp2JzzCRsFnhGyn5m8ZmMVV5afJdKr2nVaSWLsmy8Jvf1XygN8CBpVMaT\\\\\\\/h\\\\\\\/E+WYg\\\\\\\/Ai6NTD9VostvDcP3p7sj7ffaWhbpLo0rx2vv9j4C2zj3dkOjmi9P31d6AHmmy2rowc7Ta4h0iUeOpvKBWrdtGA6Q9+81ylpnslqCdDJCMUmNvXnFp\\\\\\\/XqCZwnBlo1G4s0CRFhdXE9rLT13JQfV0xG\\\\\\\/BbpGmYFrHG1+g+QCKBFbin\\\\\\\/srSiOyrF6gKfux8qdqnaZHzoxpaCsJuHK7DmuWgOe8LOIGmlpf2QaAv\\\\\\\/\\\\\\\/mWY2eT9EbL9\\\\\\\/wjSUwA524\\\\\\\/VvutC3oIanzfdclvTYpRToohnUycUV2UJQAy5mQAp7zUzYsDN67pA2+K6MDsnLozgU284d48zZStidDhGBSqVnI4BeWnXUEX8\\\\\\\/q7vhnbVeglXbt5v7W\\\\\\\/tgIirYnYLcnNEywUfHZCq7mTSVCQRKDYALZo6ib8wt9Op1LOgmu9mpjFLtQT8Id9XgzCqzc0ZVAM2WkhVT3eFB\\\\\\\/SLZz8WMluETT+oAy5J8PRrERlfLw1cTTWHptBVfQzgMYOYXCEgLn65Ovw+3s6fFgyRuciifKjzxYePPIHq3Ahk3c+1FZ+h\\\\\\\/xKpbnSqcWJKuNI7Js9gJcmxXalruRQa6kQ0pgSzyHnqamAJ51CuclqVdigaNt+pcJxrJgzKIFbTC4iq5C55W55GU7Klqkab62psd4CNNd\\\\\\\/dAOEWe2S69tNufHp1k8CWlUBAya+sqwWv4fkbmp1TzhKl7LyDK1hmyvVa7Q\\\\\\\/mSTiP7T9c5T7s4a3E3JXPH8jaK9DyoHIl6oEiVyntF7ZlA944Ufi\\\\\\\/nVhazwZUrJ3bHvT+UYOxBsmb+qYQFos9rJsaYPSXsR8UNQkehRpsqu\\\\\\\/w2ccAZJ+EQgW\\\\\\\/AfGyD4Ob1xwERfVLfQfzlTdsZ9vImaZzTvAHLsWdvQpHrHS+bdMP67lypUu1I+J4pc0TZyxcPxyHwaV6Gw2oYjVjN5Zon1hIG7+zcVJ7eNPxqW8wZcSvbUfEthteOPRSUZI9m9UEGQz9OtjB9gl5f99c4fyJ7OUJep7+gYrg5OdtauXQpvJFNOSuI8KZsbQfirK9NLvn9tc5MGSf5jXBon\\\\\\\/GN1yMufvsbUcUlOVIK2MxA8bsBvgK5rEU5dcLZl6HOxe6dnUAU+crDuy04N07B5iu+d+vA\\\\\\\/CPBdQVtcg+5HpYNuep8cwM3yjnaFAISPlnX4KkJlYyAOtDHIPBRN8nTFkPBVM7W7Gv+OL27XeXPCh6XidCX6aGFb5iA3WCnr5Cf7otx39OE1Tx5vm6FOqWrWromhr2T6twbvtFWoTF1PEF\\\\\\\/F6\\\\\\\/wLAJvNNhyWtZh1VvjkeKNMXeTioDBjwT73Dmtd5VswIeGqBAjuyqJNC+IOr1k2ghRppSmjJLAQCK4BLSKLgCn2yfZJqFXvkSCLsa4m0xj6drQc1mBl3vOlESDgX9gosbHX00Hyh44SePk3skPcjVOv5JD16rqE4IAz+cNSJaKG5t3Qc26KdLHvOvqdhCvoikTm3T39gp2C1AZ4oYWIqRHVxfx61M01BJZyoDC+4GlRr845yDadK1HryKsuLWTKVyi3h9zLsVq9qfRrK7ZDsVaM4njQrG+tayAMbAexpSRh44+\\\\\\\/73urO10dGk3lmDbtydldfaknCYrnPcubZTecFczQU4OLCEpgcciX5WixCIzEUmxuy9iVplLHtmdvbTSVhdfNK7nKPo+M+5YaMNBGnxNKQD2LObK6rWFlRhqtNMEevPXx3TSzGdg07De4EgDOuLn7kkxAEtkS21cr8n86dAxMTraOYeayYrPR\\\\\\\/PPXZHRhTwh2PyRxKwWLkek3fzMzCuY66CCzYW5l4\\\\\\\/vrVnIpoZ1aNgxoYK3iBRPamcQVEvWEtuGh7izh1KeBLiDFLXnGLMxcO\\\\\\\/sEjzEll8F1W41YpxjPz5nOvX0eOYw4AaagcpHdCDkOy3eltF6zufobbRA3ih2PMKfdfw6Pk+Qnx2PZW+lAc6L0IgtI1Vo1+pOPStM9nTKpabJpIlaL4lH5i657HsjbyE5Io7TJ\\\\\\\/sGws+uDSGVboPSl7t80F\\\\\\\/rC6moB2ZzUiNYDH7IeYle\\\\\\\/wjVsDjqIDgXgjwKF5zut1Y1Q\\\\\\\/LptgvJlebWw3O+0xLLjOMsVUjQFGHsawsttvQjm\\\\\\\/WzAAq6Wik96QTg09m1XHemXAjjJbkFWx54ber3qMkW5Cog5vQRjzOGpzPXeChCrjMSNMn8fY+d0nRITILOyVE6njEtUT+kXxeZJy26OpjIPA6yhJ43c6IRw7uoCzmSuwdV389Ha4ofJN4kT\\\\\\\/KqvQKtozvHParjnMrbHjRDmPj0xakh8nz3EpTKJBkBnXBwyM4BVKGfKdvAaYj1\\\\\\\/mVsgk+41ExrSHAADSjcgGU4hQcuNLRqiLB\\\\\\\/azf5zgL\\\\\\\/QjfFEARD5zwOhH4gwgUgiVaeefp8Udl4wMUZ9bA9Mla8Rwldx\\\\\\\/iZcRTDzVMqyj+mviJmDDsIdJna30\\\\\\\/Zn2zLqLyovTX81EPAHRmEmjAElLB+GVl2JRDBGahWFdjO8ZNnk9vgvjdOcnWtBF\\\\\\\/eKAjkKTv5oB6IlimQ4j6bBYDvd4EFhMPLTzVayQNcWcf0l2TWelauGSbfAS\\\\\\\/uh\\\\\\\/R\\\\\\\/ZmkQFVWnMHSapzTETkJMLKcAuxfgaDFy5I7DmhAfMFomevdhW5BCvWHcQNX7l18Bn92\\\\\\\/VX6fPDtYoAVdry+dFSz+zBxPnEUFCUmg\\\\\\\/3fZR2HvH2wgkjrT434Ir2plC404uqtOA977ClWivz688ssBkeWwlJyV1h5sRcDzZxTm7HuuHBmFtb0eVsqOo\\\\\\\/F2P7KW9QxKQOfE\\\\\\\/L1O9YMGrJdjxkfe6Xa0Ch2Gxv8iYrJRd91qQ7mTKfMEilay0NyghFbX94pi5d8b6VV2DnfME++QoLyPX0W0M5xKXNT6gPYMSQgFOOpA7iuj+9R6UubZU3sucwNnDdMjQ\\\\\\\/Ou1RYMzzTxOdj6wqXOgFpcHjD3NEIEVT5Wozr+V\\\\\\\/3udButwCtscaEnWO80h4uhNlbQI6o+Aaz+vIluiyzLjYqczgltlVCNi2aE951oena3z\\\\\\\/5N8eXN6sepxXsC9FxtsIzCh7d1974YylFi5eTvCjMCbDy6U+0w86IYxrPUp9NHeCMR8a1k+B+mchSIlp2sIIubC41Jjv6b+jzbhRr4BXQphVkGa3q96qB4gRUfeFCO4pDOgTy2CqNgBZwKfHPGR7oWLTgoXc13sb4uxI7QQQq1Zq0SZ+cktgoxdFrhVqj06TtDGuXeINxfPg\\\\\\\/4o4n6CIpvp5IqczC1tQOduXv1Bpx8fHmzZLmf9jZjYEEaI+usDT7lR8GpOibCwVxuuKwD0RQCCgu2yhPK0W7rCSzD3v+dWVdZac94+M87ljgL4jh87xMEy1cwNE+9hKoULFo8spWuj09yGhIisULMD+c6IPr+VwMbdFxGMWVuksg3OeVV8cet6rPv\\\\\\\/6AtgShEhhCFtrnZBTwgT1QguBMp1OU95aHJGIv9ynNwVjmaY9G7dRLyinu\\\\\\\/w448Q91scFl6l\\\\\\\/86phy5FwRErA0GFk2QJx6GkbBt5ozRhuKVIkP\\\\\\\/0sfLWvHYZWpe9oKVrXFUPZxTl54sLHifbxLRS3VtdYojWTKFTygSGcjJlbdZf0uhYF3YiVHYizAWRt82X5tU\\\\\\\/6sUWMO\\\\\\\/UKxHZ3kjRkXXh9iVx\\\\\\\/zrppKWeCuXPML1J7S3vUkEjTyJOVuPSXP0KV6rlLkTC\\\\\\\/WFT+efn2q1ljCS1duAxGSRrwI5IjQ\\\\\\\/Y54fcFIip+Z\\\\\\\/Sb4c1PyIwgpoFwbNKWrc27egJPeXKWi2RY6vUEtEPyf6hw3PIyAVaVtVKyoH9nfc38MjKT1hxDZcjBTEfbhM7N8I2oJqOrW68U4fX56bnN48pskD6VRfsXBtIfXJ98tijz4ECvUsVMrqsJfOSg19tAgV0RKs8KGYblLXgqu+zGHUOM\\\\\\\/Qn5Ok8m\\\\\\\/55dRFc7H6+yFgzsoOgji\\\\\\\/wDzS3DGJ6nBjngnk2Xa7Uhn4ud6XUXjXSz4kjQaIHeM67l3u7qCVI9RCo62AosfgNx+nFFXWKDoRemeSukrkzvxTXq6wLp8yuVZF+VA\\\\\\\/Yb19J\\\\\\\/z9m6rPTHi9rRmXafw5v22cSCbr+Omeiy3xZiJlaahhmNL1nX6h\\\\\\\/OHm\\\\\\\/zGUA0t9\\\\\\\/qHtsQplCQfLlD6+o4auYQQgYJb6YJpZzR3xMkYxdI3\\\\\\\/fNH0VF2gDRPd7B574AGu6hwdvsWrssGxYSgoXsgSaAMvdNqpynTC8aNIuRyMAE+rjvzeA6W9mmf1G52BTcqEtEOaU3lRJ9WC6FOQw+gmwEDAmIIajJAEe8ACppiEi23P4ePDmwdSHx4J4uT3HL\\\\\\\/IPG0edVphs0oBlrTOKuTRqXgtBfZt5A5UbNvsX1d2v69Pyial8HJIOqyTJ9dyK\\\\\\\/SXXgplDRA1bCGH7EimdM\\\\\\\/H8JvALyLNBNjN8tFAyXBpFHhMG97PPtPWN8i0NEfu+j5b84VHjm2o2Xydx1LZHngM8CnI2PTP+ELH3A6A9l6Sp3btwosKSqKgLH+E4i4OLPImjX\\\\\\\/fg\\\\\\\/6IXZQrJ83fdfRKUrebFt\\\\\\\/EMx2qrPPgVyH\\\\\\\/B+P8Trek8E3DY+J7eLKHNVJAsRWtTo2zkoU0oX6zIFqawmYVdbMcJaHAwtYNnvr7npF0v6amxOhQ2ND9oxoZYwZEcDliuANDyNOt\\\\\\\/iR0o84dRH8FvwUEGjri5X8wZW2xiyNiu7kQYEu5QtPvqXRDHmsrupvSYVB0gvFxeWmkGUWOMDa5XPdUhJ22zPdH8poUCDnWNhSfcDT\\\\\\\/iwTGYGHQXbpusS7kWD\\\\\\\/M1xLwOoJFfh9KlyKPnix1k87ituV8tXxrn506BoAiD5gLIqmKIzFev8cUmgqG4mvjCEGoEHWcGySr+UFHCz4nuNDQV9x\\\\\\\/oiLTJjdITUN7GMspnvMpwUWD5AfrVeq1yeUsU7E904MXQKYdY4MJPnW0kwl72N3w5pfag8uRZPYLU+iQXyxG\\\\\\\/y\\\\\\\/zdR6XB6x2YUeIFgAXq+6rH+uTatU228FarTvf8PMxeuuE7BBbLFDtTKPCTyzB8YOH2KB7mjPFAC2fVVmhAhzRVm4d0DN75xgHZvJs5Eg8ovg\\\\\\\/Bqf5xQUvJnFFGRw+xfJaBwIi4lscuqoPiFsQfrddlPPIEFudrJXf4UNVeVXpD3GEb3hphzKfmsw\\\\\\\/5\\\\\\\/1xkFO4XhnU2OijfiCwIj7\\\\\\\/5kWRx4+7uovNTwCeYuxmbTz84BsiZwPftCCp6bPZFS7TC8KUoqtMJuoXml85bExnHAShWMXy2CNZu+tBqaHgPLwBJas1XOO2w1zWEQGzmm0ZvVAGHf\\\\\\\/SVYd8goOBvugHr3UO5W9OU12g3ouk4d3rB5F\\\\\\\/DAb5Y\\\\\\\/2gQXi0PtJwHsZWtNOafeqSyP73pfa\\\\\\\/dHBs+O3ewAqfJbzklvHplc2tk5x2nY8LIp4J8iM\\\\\\\/tWen0z+UYDvDeNoZerGox\\\\\\\/K49nxwCzaRmqZjoUBYus66k+xd8iGkpv\\\\\\\/LzOUJh0Cr2LpLBmLBtuCVfMwy\\\\\\\/MaPdxKhb8h9AXTekaXis8OkettmW1K3iDjG78lKGZ2POkSEUBoP5CV271NnkWu5JwvL5jTd5+UA9bCegIXxTyNBn+MpaPHXSiy0ofYSXGRS906KRPH8KTRqIaoaLaLp\\\\\\\/SwTVU1iMHYeWl0ZkbB\\\\\\\/K4tEOIfW1s7eEduBZ0ifyq4JGhjCkzAOjKtD6o7v\\\\\\\/3RVggU0NaWn7v+ZnGasWHbLZROQX8kMamOPb3KW4twkv2ODtOcdhxCcVussLJayiHX8e8+QunOADBDvjP9D2NJsI8\\\\\\\/h7zgQY3jF4yjfBhKc+yDRH93ILaP+jORKJw2dsQzr+JXR0\\\\\\\/WgOGKpF9EPOkESZosquLI8OrA3OBheDYdLTgoc9G0ZuERag\\\\\\\/Sf00IUhbuJOK0xcdIphib9bYCCL5FRwbAMGzk6GR+DvElvq5A8FIiv7iPukTEmr4kAV9CaT+WoDCzzM7xhKV4yZnJ8GCX66q0rvqG19m\\\\\\\/o01PAsAi2k\\\\\\\/LGOsw48gijbVwc\\\\\\\/6pZw+yq4tIGB6JXhLmLrNZW1qubvNoNXLb5Vo0297wMxPauSvAI3Q1QZCnlNl4LHXuXhGNteYOguCo9iIT9RhLPi11LPJZRzD5fPA8qtDaa9nKZZ38Uv7qgXgGWfbep9s2tly2HQSESknvZzA0YwdD1Iu1DUNu3zB7bzQb0N1rsJLBuR0voO\\\\\\\/zzFSpP8aNqUnIWuaNjnnn8eFq9MRWyp+iuSOGV3Wp17MpGQ3ii53UlEabo6GoAbeVaxbula8MhE1mr8ZIk1O5gcm9RjGwVcs3EI2fp5gtArSjmiLpU+y0FZ3THRKzSqeqWw5fyLwPPtKcF3vihdM1sjFqzBBJY+pHEYU5JDGNfzZeL0Q3MSNTnVe0FntPHGprAoPUM9it2rKf\\\\\\\/T+nZF8hPG9rA2+YKRxkF7GKz2Shqs9TQsu+f6poyCvEOJgaQeDJYm1JaQVJRd264e7YsrKDfzXmz5a\\\\\\\/qYhf51sVZEixSM5bmFyV6snmLbCnE9o9HMqGh+vQkUo2GE9R8H2Ghp9zk3UYWu2nwhJkZ7qSxfH1l9uLx6F8WUEnOgh0TbWrZzyoJGfRaN1Ncz04PmHNGvrOKl8qPtxmvBi+c6lTyISfzbt2zCyr7VNu5863T6hKQywTNW4i+bl7S4GXPZeeiL2KYCTtP\\\\\\\/U4NAr22OkQOxEHDvjrfsxIDCVLZYew\\\\\\\/Q2+Y44IK0gE4QHeAXR+HZ9nbo2UY5omARYm5Z2tpqBO9fIf6KqQVWvZiQRq5gYl5GWxUaLrjJ2Ulm2NDEF3\\\\\\\/oFAAKglA+\\\\\\\/9tcLAWrNfIgcCdeHa4BlCEm\\\\\\\/wXNKNNfflaqArBIF2CB4TdgUaciJ6iTlYAYdcL5oNBODdEdUwf68v9QTkwBa8ioVjuTyqpEW7vpi8joZWXDMOrKW1lA9zK+5vLtSwscVP09MydzU7aU2IP8lA9YL8FE4Kzwn2hefkYZuvm25sHlPxV01apmFVE2Y3J4shgBF0dGSZ3bSUusJHSxTG8NJS9NyhtfsGaeBQCj2nW6\\\\\\\/dHoA\\\\\\\/XN7kcWcXIpbj+dLHXK7tX7GoFyloK4Ry1VpGHqQF9pfNCowMCVKwoQwnoLs2DZjSCo0Y2sRJ2xWvpo2ZH6Y0MsUJ46VQs5\\\\\\\/tLk3W5IvERHjdjlSDytuCA1ceHb\\\\\\\/UDBV+8youkdbpActgyYlekE7Acg9PnMVTMgpL\\\\\\\/1VG1VGEqUV5lrIT4DgC6wIgsYZfEWDM76FBjaTr8UwLO4jSPA3AkOHHrvApzSn6wb+gS65ITyvlzLrtoygBqdQzqkwzMMm3AdiNJwqZuWm20ABjghW3XfIElCUKoQ1u4+XqWru9Cb6iW8AEXBqPTmcy2wGC1DSG\\\\\\\/ube4YRd71on2Dnzq6jgYZyxQuhwplz9ckJzjhQSJkysCMApGWl0XS9zc+ORih7IOPwlEP4NinPFJgT0+UPVQvfAveKq3vUi2NY\\\\\\\/vCFYa\\\\\\\/BnVvSef0nxmykjbipGofskfrYyY\\\\\\\/+Ex8bzV6EQsCpvzkV2ZM+C568gHMfTm\\\\\\\/mRa2kMe+7x1zZ9vq4EUkBijioABe+YEqkSlEyv4v86pni0vcHbyjthPwjP0gpnK4sL30nHxnIagjDYVQ1El2cbMAlw7AjbiDXPgKnnoWeYUHgGBmycRJSIa5ah9PqpIX5\\\\\\\/3E8q9Kp10EaD3jPEbBbhl3IwM8oOe2+xKnyVKtgsMxHZ9JTxEiBM1otw3llCFerYg2pwj8i2k4kGbxYNfajWbgUDGpjWblMVMOBdGsEVfDNz+rNj5wdFSchI696Pya9FA5bBKAX64ILl6iCqAAaZkd\\\\\\\/REz1QLouuFG9diJzB5bHgjldGdubelr7aVNr8X4rnJvD9n2pSlBoUrHzHSh95YJwLBxvKFwCYYHfZPrFIoOPhw3c4pLJ9Kk9avsXvU+yz7VO8bzSXeKhU9sitzynERIMA2N3KU6Gqj1\\\\\\\/eWCvRognt2kVJ9VsH7Dl7k+UMSHa1udYNEG9zFKBZrTZ0dKgUUHi4mQZkMUxERYLx0V5tCzTAt5OiR\\\\\\\/oO4bPLBChGQuWlAXIBsFCJdHAENlMvZ0iNXmwJW9nUVJtI3afaCg3oipdHiak83nGw\\\\\\\/L3Sgig6kVVRtIoJxcCzKG6Q9FxKoUCTSiwFOnzYi3jZZPUk1JSh5\\\\\\\/z1H5r0wFs6gb8p9oAuVEXhw8it\\\\\\\/RHoHojvgZ3g5+EcIpfpDSfIoyQa2LeF8COrSioqqe7MUeW1fvqck62CROS1Hx4E+DMVGJL6OYMtZsjysYALAPlqKN9LrWU14stal4dLP6icLDSGEWhIObj4eM6b5qa8OCb0DfviUzG75bvh694hK4HGUIk4giTDPccIxRLGYAEo7IEBgrLClgqaqNnhIVfBAiuF0ZJYbxl\\\\\\\/OdPReIfmill++iiC1iNjufM9M9GnIGeDnYiium2eP4yXRWMKUb3wJBVjw7I63ywj1rrUc66rnsIcMTxvRWjbFEi5PysA8pdDg0Q9SqEDCe13ZgQA7+gmgdtdrfA8D0+DRh7Vz1Ky089VB8Dr+n81IQ69MhRD+Whtv\\\\\\\/dsybUngZST4lc73U65yXpGYA+YrtHa2cHIfeFMEzbHqU2+zhyqqe5h\\\\\\\/7nZHnP8KBMFL87XLrnlMyOQkXe80dSS\\\\\\\/GsM1ylk0DFe6eiTpq9j2k0uHn7EL7HcEod5kticPs8bHwkLNSndHH3+oyvhrKNBeClDDPzRxJyd0M+ba2L133vgvO9kUUlDG5y9E1+3+qOnskrXAASQJ1ZGaXI64j7EkV2ub1M7hn8ZJZld8yRb3PXtfHcxjzQEoSXREloQu9QtYxh33UJTeGZ5ij+FoNLYV+8Ggkv\\\\\\\/JapFTfgrifdk2N0EAl+oWT\\\\\\\/moyOsCvIGvpMz8VKo7Xy7XvRYvoPaZn+VmkttHr8mCH6OvYWGwQVFMAY3rm\\\\\\\/fATScnIvcZdz\\\\\\\/Go2cNYv8eMade2i+R1zppvXQw90TD2v8sbEHD+fFboS5vmog5MkB4aha8hq\\\\\\\/gdkmp9RD5GwrtOCinxgYsqqL8nvHRB2ZaYShKUK8i0cVKVy68HDRezZIcaQ\\\\\\\/Sj5heBOQ4wzncCyPPngbTQbuWymNsWbY0e9djliscehsIueKCft4f8A0xoPnKhRqxsihEqUqroeyEqnav0w8m60Vb6L2UePKSxWd1ewZsHTbzWy2tJTsDM9crTCHkB9c3UT+zyIFNfLs2NDESlfovmbO3Bxvfv4v2\\\\\\\/SqSZzuuu7f4jLfnfTSQ7cbuSKVjJfKV3ZBfFYrwcALD97\\\\\\\/k2IyI8jKoIStCrW579sNukjQ70ECqucll0VSpZRESMcYiD1BUqXp4\\\\\\\/AS3pb2QDPMY6VA7CZ5UewEBeieBSZvBxEkF6joza9Os9iZfeXrMCDDTSPTVXcgEvQG0alMpLU4CFHZ+hEKLylbBcbGNP0riLFDwEMGy6n5InAjzTSOFi4VK4Ei7CtsBMSGJ7gff0GeBZ4VPPSF\\\\\\\/7VwkQ5mRlCK+CB2ZQhb2h3cZVGuHnu7+J9kDJoWdFMwtXGPjRjoo3H6lgLzkEUJetJ2uU0m4P0xnC7HeWbNeNs1RinZc75nplosKX0DWV0MXE\\\\\\\/Owfs7thx6YoR4xUMKo6BOa2nafs+12H3Q3QpZILj\\\\\\\/QGPLjSYop6isYTf0aiPc5MGJPBj6BtXrha96hUkG2rvZVRNtf4yCyJUED3bPAebtl1hhxIZyTHBd1BQcmGg0jC684B4KU0TUmX4vgeqX3abbSA4zik7B5jq1+8ix1Y9wmsJnZRIUzpAyNNQZK8Fx6xfKxK8tvsSY6pz2CeQTjOsucIeYmnzvMHhG3pu83ux9mYylZwoeu8eC33IadWpfdxWX+BmB9JhvHt4pz7Tr\\\\\\\/JO+Uy\\\\\\\/XLjxRTStzUU24NecoqiNTRiyR+1NoS6U0HpLOmu7yk1aQQ9zeKnXg4GBrKiMz13cEG+2pBePgXqtmIivFzNACCQX0rxeEdbPoUeuybRVUxck\\\\\\\/1XFrbuVCmWDT4UAVNUjvfjiadom+ikl9omu2QlBL4ecThZ26xNfp9369\\\\\\\/ShYJrjMhQ\\\\\\\/J1x73HzJ\\\\\\\/SD4JGtmC1apckgLbITvfjqSlvl7pl68sHNV0H8hce2sHKPQxIfqV7wo4j9hO\\\\\\\/bbK63RKLa+vy\\\\\\\/DfhNTL1S2TIwtf7oksuggtzSYjRqAGduvBnuKkHJWQW9EyqBz0oc6LJmcW1XpurVHwnkl9Sas1u8LcvcMxK7xc+zwx6UOnFspzv3epIz4yRfZtmFGT6HYhQjvN0wqNIb9OGl21jU5CA1Z0l+Bme2Grc4pQeUjZBMAyOWjq4dG57agku4BlYgHfNcVsYoZJ4kHHeMmn23IXr7ZfpoodrpkKVtA+ASe5Lbq4F4SCXivK5MXdlNI51wpxwhnELtk9WZM5u\\\\\\\/CwjRtS5HBX6MMn2XiB1bdDLZwoBw7hCr4RhtBDznIEe9T4t7cvstvgdPy0Mm5dtpMxTRJgJ2U7ZHa6hVQpTNWBJnHfHEEuI2WshWKY8+ygN2MowCmhyx+G\\\\\\\/\\\\\\\/4ImPcL0ZtH4kEPS9x0K+z301UgDu+Fw3Clug05Rhw1oLXycJyTmc1sabSMKfoQh8m+cyMYqVzlRG01gm2d4h2rRQKtpM9bpmFXpmHxqML+ZFN7e50mUpjvFNvxfNQk2lpg6\\\\\\\/GWShBvx3keQW8ZAxSkAareenKUSM4gomE6dMz6+53\\\\\\\/+ZtlNYlkOGiTsxp0GT5sqjEusjSRkh5HhrkYcd66MGtllHF083NhoZgF3PFkkGLO7GDhOBjcFIUPhDnpIEfJl9+XYX2IqGUwEeptqQigee96npEx1qGwSVNG\\\\\\\/uzyrLf1kOFaK3nEoCx72Xf\\\\\\\/RcgjqOoqZOjkl1O1AnRd3fGbOytJFlu4lLWqU5+rCwiyUbWuWKE7dyHYu96FCwmNp4qtK2QgrP8\\\\\\\/lmSxj0SNQOk6X6l4TSw8fuvC+wnWh2WR2hnZPjZQnDqB9vEI+sHb+aZ8KLTA1tEDpQ6pqGEwQ5RP0SsB0CY\\\\\\\/uEKwZWF4XBv2tdNaqhv4boSMroYRY\\\\\\\/PKjmLKW88wV5LIgA7xmlroSLRNN\\\\\\\/E+kfqrVx8lI7TJtbR8NjiZO0u2GkF2mCnHmZ8f2m2yPdKdnjDtnoLggdl9\\\\\\\/2ljsX51f8Kokvxtre5kUBJ8eLvj3HVGqmlXD1jbIIAYW3bsqNBKLnrzht1g6ROlem2CM1QOHO38Yr0HabBTxnC9o5ynV3HHNrQ256Q7mQXQxBg2OC484SuEF\\\\\\\/6BIFR+954o0q9imqKZcRTquncZNVH6BU2DhqA6QQRNSJqzbFguCniAT+LHTyverheuKev8NwS5+q1Ynvw\\\\\\\/tD+5Ufb7OTteIetrDoWm51qoLew4gzWVjtOBRt8JSpRk+i8juDHgRbs4naHjLKwUojxNfHfspLGOJvXvsquyXTe9r5jD9HsZTqVYnk46juwwjeV0yjkHRCH+CRcBuGwHVUCSOLqU4QXLFB30suYplw2WgyqdIp9WFPBdELSmrWCKbPErz0f3XYEEUfqS5BgZF5ztRGWiZyHqslfz3cTauVWggxijyTVEhHbXx7UPQuRsWtmLqSDeed9IE51X3ql1St3dOfrfG4mn2B3gjWPUIlRr0\\\\\\\/gkV9Dx2enYfastiWQbN6KmM\\\\\\\/kvtVdSjUj97I3sw\\\\\\\/iIGyr2XuSmewtChmiICANUif4a9lVFmleBZtl3wBQguXzcCefnxCG3nMWNjQFTAnN9sSmhu\\\\\\\/YO2qxQewNNZuY8QsdH4BPG1q3XscXd6GYRqH0FIloZQm7T5z23moV2khsHifBEfWghneBEIeLwmBiqdctY5BIU+CViLNCaxvG9rNZouhf4y3PrRTxA\\\\\\\/vhncN\\\\\\\/KdWNSlXNXn\\\\\\\/xgk+uuqW2U\\\\\\\/AKga+e4HuE4CuyDyx+7rqYW4qc+MOoO4Drm1VR7peHtTYzSKRzCrwit68KIICS8k0z7nl83Rhy+hB41Xgb1cGkQWMNdVTgGYkOQn\\\\\\\/7kIKfROB+\\\\\\\/kEB7gkWRdMElud1RX+ZwUZJMbGUVAhemXqFLw+LNEynydhwzdUGwpbzsAMrwNGi80im9qxgMEgCFoOthHmLOst+jxlDkQSoZP7QSUtWipuYvAlum4vpHXPFD5hiW2gATKG0Cz\\\\\\\/xC1CxpCGAKH1oAlDm9EtRq5QMANsCVKAu+\\\\\\\/eEMDX9tFSEjdI35DT2y4H1QF20UoAcSaB5HI6zd9WhLtvSe6NXrRHSgZeppZna1y6feNvr5P7ClmdKeMcn95RqzmBXv0oHZl14XT77QZR0nVggW+Xdtc8YHpokHL4v692KiC87arOS579aC8SW9YB4C3rRxeG1c\\\\\\\/cihS7TR3fFaOKvCVtXyQjZyBCFiLxjkPiZgw+O+4b9kozOVQLnxO5M3Ydux5Po4oqEL8sm9cNhDamZzhOyhkRO\\\\\\\/7+8lkvRcUxq+z8dWzfIsX9ZFrR88RooW+8LWKOIkkR9+j\\\\\\\/Xj8XSRsPGsXSHKCV+x\\\\\\\/gQyYkquRX\\\\\\\/1+DRSTSWfz307h9JzJmppUWMTEAILo8PWa9nHfTd2HQzZ+e5lfcd1MMYVUy8e7tGU2VdY7KrA4vwS3xueB0u\\\\\\\/1aWZZ5zFCsgweC1ffdRwH67CjGBTX2HAo3odZytMOKcsHxFOtjwm8DVdI2Fqed3Ywi7fXl2l+g6CT7h80Qb5bMrRrYnn1iNmWe49GnESn81XVOyD863Hrj6eaPlhus8J3ntS4DVWM7CEmp+WPk\\\\\\\/OFBbdzihSr2KgAOzCFTzenGyYtzB4XNtBgNa5kWy7Gt7Wj\\\\\\\/YYaLN3mytUyt5MI2BDJOIqVfbUxm+cUq2scqd3AzRucKKip9QtNOKuPH\\\\\\\/DcUw7Z4w6V4W73FZTTG\\\\\\\/GvITQ1ZxmsClgQCNdlxeQaZgXoHmp4vHYzl50WSs54wI0VQLqu9LTt8mX\\\\\\\/nofAMkEm3cg5+9XbGSCoruC5\\\\\\\/p+Dasb8uZgX79eDT+1JvEKkpnH9c8H3f2Z2sR+ysXF3SOV3n\\\\\\\/bS6uNxhkp6mt\\\\\\\/gh9wBvEZlzUxL21Jffi3RrofR3FqODttO92Qtw2\\\\\\\/vhhNQe6rduEDCxZWV0nDdfpBUlRvmu4hLan\\\\\\\/DRbg9Js0lcvgHfXBtq7KgvHTRUsEYCNoZuWEU7xnQtO4Fr8XC+p76+k+iv9gyoVq05t\\\\\\\/zbHV1FWEVXkxJjfi9JJszSOLdfR2+v9iYHx6Ag4jBbKgse2F4cICLKJ51bzVqMoHujSegdv1bT2ciEXr33R3rJULoDDxXn8ThdR3JpNVn\\\\\\\/vhC7ikqoaHerhHBFH5KY84hEpMvIqzYKtcpxIAgRxrgMkomM1BQ6cdM5Lx7Sr8LlFd\\\\\\\/w12wMtSWWNNr0PdEc4spnpVm8Wh2jnJexuxEq2UdDYxU7LEAT8Lbc6Mdp05U95mRKl6c1+X6MK8qRWhgv57lRl5COFNjKkt92GKSTtoQcKRaPr3bvGHXiakiUa05UqBAhz8e5mlg98YPFkgvI2RuvbzsuP+wczrcayTRWE2Zp5mvBvz94gF1hmTbaE9X1qYcjhEDVs1ANKl9BjJRKC7ynPX+y4TcLI2uNHhzmNytDqDFXGo86YBY+zTIs8AsfsHGfF0VInsPkBKtabDELH5YapACSQoQaaaWHlhu8RJMGzeC0z63G4cRRzJLESBWWIHmHhtxD2PvOzOUROYbsoV9\\\\\\\/uzdjTdrO7DqQB4Guu7wJVC+RFjoJlrCc5CpE8InJhJOXhQG1XbC4ZmBr3lf4LhjxH+C6tPVZ7ea4F+pZdsEKH0f4\\\\\\\/NMkjFSFGMFTu\\\\\\\/eH\\\\\\\/QskhRer0ZcGhzRDmyjHlwJ78rnnaKtaHTylVnvzpsEkXzVLh3209917oPyDZan2UU7c0kf6rf0jvZYqXR1jwJc2OAvSmC2+irzeBk4ZMmhNqeEwt3zBitWR+z2o3IojnjqmVgKBWsL1\\\\\\\/4I0BbS3zqyKUwj1b0TW4xQtRYEfcBJfyJ8dOvDsSdGtoO3Zh+Stw829bV\\\\\\\/g0Yd2qnjf\\\\\\\/2slSqhZIGVWVIVU+7mU79bncadlxGIpeVDdt7Jc5zoJ\\\\\\\/N0SpBlvlAf4Gbyd\\\\\\\/QJVoqD0nud4NGPpWQ5KvmFb9NYtKGU5eY2Dxk6IQT02eVdShABkoGS0otmnnoql8rJt1Uo\\\\\\\/NycTQaUv2tDPV8ij640DL3zZ0PwNKk2Py94IihK\\\\\\\/rP1QIoi5w8U6J4HBCf1TyOogy9TdgGPd36HBJ8TL55HhgCzbALUFWiYLzDZg5njoQSidQ5Ndw2qBZxy5lx7+fXri+l+EW2KUwwBVTZva6ZwcdaK\\\\\\\/AD0TDCdA3Jcs9VYChuGyFmsAEq9Gi7ZwNs2V1tLUs5GDg853C\\\\\\\/om0pTvXFhHs9GO+U\\\\\\\/liiTPIk4wiGPVHbslhe6iyNg1EstMXpa7pmgbCnE+x\\\\\\\/5HXqIyYmKIApfZYdYl1+RF\\\\\\\/rl3KongBthRPm8WrenUIs0iw8v4IN3ef8VI8q7iKOwBA7LTH5eFbhmLYtIVC\\\\\\\/9QFDPnT831eYiNphGioOKvHx6RIiPi8bt21qK53fdKMfJ8\\\\\\\/CCx3TV5VQ1U69hMo5d7tHsd6cKUaAzd22Fp1PSnzAeQXhDObD+UPEwqVguPZygo3IzHM+cCOd6zQNTPxHOmbBldDyvhPzMdzw3zsRuv9OsANxKA9ivc0jNSqY6c8o80K95VzSWjwgd8qgTVnY5IAV8hFLKz+7yFYc08wANlIXUMuj9EnYOOUYyd4k58VrR7lLee3KFqerJf+iQo8Hs\\\\\\\/izcBeTgLjkZiYMm\\\\\\\/pDjfZAeOqCJcrxpddKWopofYPg\\\\\\\/8escCnLZOLcNEATObaUI84Yk\\\\\\\/Lqe3lxjZuZT3jG6ZwqJlf5MVps\\\\\\\/8wblVDWt2+N4s1KJIjvFLHqMsqZF+pyBZsKRhmyJEMsH+EuREFzf9vES1IoCPnYfnK8PpGilVZ9R24ggdDpyQultnyHlvqZFb5PmTGZYa3FPv7rI5A1f4wXw3OilcTxv+wI74u+FDzw0iIfCmN4Xz1jEVd+hU+sRVHweptx6aGE7OIaReDlWrnwV5ch8YRsqkFbFyg2GWFK2uwxdWSlBq99lfeCyIhjEvT5KB+MNCwnZlJkGvFdZ6T6W6owrXeXGYekU0gwg9GDhKBSlOHluv2n8aHFfva5sBmhpecC1Vz1MooWDO7Ws6BEug2moOp1XXEo3VI0ZXFl8apiXt+6Wmr\\\\\\\/KL5NPuZTceHRr7t8h\\\\\\\/p3lRjvtGqVxzDyyVBzFVZJ9raxAc\\\\\\\/RzKy+SKMt1Bmv5rAOLaZUK0cioqsXBvx0rxasNceBLbWYTf1\\\\\\\/uy7JJc+VerNI5xxfkak5NSVdxEgUyI6IMaUPsL4LGJ4udZw42jxHZWwCJBNSwq2gsgcqPanAsZqnYe40t9exEw71BVvS7KWjz65r8TE+Kn5G4tiu6izX0XhqjZVfaeiLh\\\\\\\/1fIf\\\\\\\/EXyvwj44o2sd3Z16UZe414xN8jIVScctvgnl9UCnBYoyIBwRPIxw2mjCVjCqK3MuzwyJbr37egoCuahyj3O8pf9+1t5TYFR56AMuHp6E3rs9f1NszH\\\\\\\/StW\\\\\\\/zKILP4nxHRqqLEWVd6UF5u26b6Nhi9d+UmBeS9lGDY2Df+FuXJi13Hoke8slMrq0950D+U0be8b4oZ2My3UpEZU20wz8z\\\\\\\/ZUWlJDhx0YTsrCIUYWCUQFmgOLlAB9OO21vq5h14Zv2HNqLJdZKeUtNT8DlTlXFhhn5rVkkKdL1LrZ96kOeAbEGK20YSSQU7C8+fOzDUAYC+hbKsfPHL9uydajWmr1mFq89gAypflB20FJHcElz16tSBlOZCraWzibv2j2jnCDL0fb1B5EbqDOx9xZz14x\\\\\\\/ZXz9NpxrqoHJPxS9oz2Ugu4xOz0FL8sLHmsAi404CwWkKr+29BwPk+8PVRCRCmiNC7U0zXwHBy1Bg7v15VDBKQFES+rt5ACrq7txwX2fFr7S8VWW5asE2QeDQKoymyCGvCg7eSQNjOTlK3BkZ7nFnu3I5nac35DCvAWVkv\\\\\\\/uDOwjmRF8RIhwlw2ZESpmkLxBUDm0ghQ+kIQtvPyRIqqV2swmOXi7a3+p1G3YHH6TR5OEOnHQUs\\\\\\\/QeDp0TFMNrM4ULl+8QfLwrpPjWQWZzkCmXYBfjxjW9RY3uv0agnLLFFl1re9HPcl6Wguq75kNpaJYS5onhWLZA5X5E81smqjA\\\\\\\/YcDD16AIIfcrTu3nN2u\\\\\\\/cid\\\\\\\/SRAxjzZVKS4IlBuWHDD4NwAhI1ejP3K1rAj19rbGDXv0bzt\\\\\\\/575giBMj1K915pIacwLsFDR9kRcFu\\\\\\\/puxBMEr4mZO+KZFzdTiUDc6qOf9kfeOBIM2H1oNVXYb2UPiYvV880uSB70mRpAFx3ML6NVkEocsflwazzAT7sTmZQLrf3TfmS++R+twnBpD8GyUXgZwdkkwBt\\\\\\\/5q6ZzoXjf+ZSrmye48\\\\\\\/9NIMu0aSt4UmVSr0erdf\\\\\\\/n8sMLEf1Y6gNwY0vYS20CCLGTBlXpTay4ZktBCXm6wI29AEcPnr44ix8hc3KrBZAK2pfqrbiyQF0j6q5\\\\\\\/6C\\\\\\\/gUdsxFAVa2iIyx07t2YSIYo0O8A5Wi\\\\\\\/0EIpTj\\\\\\\/psBoz2PaTn0AAQaii1Y1onCLJ8RRlYSmEdBqnt4lL0OcoUdDLr33r8EFyInOWnUBJjbj4pJNR0K\\\\\\\/brYX4R4DVF7T5lArcJduwO2wWar9urddrNW47xzf3tKkdaQo2chkKRczYFrZUijkjk1DBgEa7x1ku\\\\\\\/dk1zcu\\\\\\\/doN\\\\\\\/wnQht\\\\\\\/y0ySmabf0gSYuA8JQeeQ7WD6n6XJa7T\\\\\\\/5WzL84FQTGSW3A0JLpkAYAUQWkYN1ssjtuMDq1NAjsUagYf4Lre8RahVm1hx36xybN5LFPRKq+BL4fcqsbBY9lY970kjSNqnbeOPLBW1MAAXgWqkmRFMTXgmguePeL2GgyIRfikT0YhVPByItweWaw52J4x78gSW5RW3keik+4NtLrlQSy0FtqRY5nIk1G2zUfS7cVTRIOqHB96SoUEjmQXXuLscaoi2e60eOKmFfHYjqLe636VGyjjw+3Gf3oIysWEPnS466Rx2qnXzz6H+dLdoAl2bA\\\\\\\/63ZZRgk\\\\\\\/hJJSF8q0g8\\\\\\\/1UevLnluZ5nJelyShIs7Rgkuw2fwN8haQT1A8h2H1ikYsMlPFmq2aiBj4a+1X\\\\\\\/it4AVKR6lZ5424XSeY6pZUpkOdZLKqvqjXrFhha8qxF9NVsjjccPQr6jtTPxdRqfbezRf+vNw6RmQpTv3P7YGq6zGlLH3tAb9GXJgMmmuUvxhJiL\\\\\\\/HEif2AfQsoWk0Pa3pRxPstVjwY4++vS7ZZEUtYLiVqmWmRfyaC4gPl4x3rtlcmTTGe4Tx7A6e0IC9bIZHFJTZtEqgQTr++ZwfaAgB25zFM7QrFfSFxfiF7TEtkXAWR5ROL150UE6dlIWZj0z+U\\\\\\\/hPXcfhqReZNSlgPT6\\\\\\\/NXggHS0inmhlDGVuwDIqn1g4PrW6chpcquqkJov+6dmxE5tqBznmjNUGZXV72GshD4vYIIhHN4VZRIJEduLsogMDlPEHIGYx74ptWg84c1qJS\\\\\\\/wlqdGrL7TXF6cAJZpTIKs6NUhs4i7gJ1ExDooRH2MxhGLaNwt6wiPgnFIlXSrvWLuvAQ6RHyMcd54dN2Avs10t1QDxX+zLKCJuxfYR832Es9egG4HKkKxTiMAgor7SlPx73MeUzoa8Ms5jzYUUl4flG+aJpqtVA5ggalr22KKuHvoHppYRcPIH2Ym9O86d5ngx\\\\\\\/WOgSJ4ykHQGHA5rTjhrfHGPJYIw5ejfUQBQxt78wjnnvQaBhfV0ofgY1zJPgv7slcXay29l0km2lH7TTA8YmSQd3ddMgkUPi19NTyDnFqNozElquHSCsljBD3bDxkTa8AJnoLyI\\\\\\\/gwOE1ke+xsZTJfCIPhXHyyOi1v\\\\\\\/M3Ebi\\\\\\\/BtWj1QlryinGaAYz8DgAq\\\\\\\/faTEFS+q8tnagL5iDu7HyMRnuUI9krThgoxcb61LXLKf6J2deVXAMTM2iZMItsXP1tGo7YayQOIS6x5eXIgYFDjc39sJ5WNb2pYOToYHTCklVIrk6Yd873Q4HZCjIDia0OX4q1GZ1HYsfBegbVpKQFxRqyLgY5rvWXcsRv660njd5Gz1ZsdybHc72bCFlnEUBSTzjPmsaPJgrfg8CDurbSXsCtBamLZRSEFyFOrkQj9t7uBjFg5FIrccTLA3d3xkAkU7re4TTLKvsSc+fAvQmKFk2Ytw00dw1s\\\\\\\/UvWHBUJomGSixGrPULvus8owY7ji8MzmTJdkPZ7uYCyjGptyAazQMOOFwZHKTuqvKyHx278dB7\\\\\\\/dhlqhtGWXP898oswlXTqFDYscNPVGTFclls6mOp4GTPO3roQvG9PM\\\\\\\/y2gCeZIX7EwmrhnthWzVzrvEC7XH02FZ2cmKhFlydLUCI0wnbDCqeJ5S9EO7RZo+6xVupnsmDZizDuM8OVbAzPXVTSCSYUFXnQrH0+jxAYkUQc24JLqsR9eEvz3YyWUM3Z6xkWsCz3tLxbsXgKXtEN0yEorxsS0w+\\\\\\\/zmiDIH89qDq60rig7oKE3Akqzd64a5QWqJYR\\\\\\\/Gd9hA5iNW0xRqdhbV+HyrJjA+T\\\\\\\/wpVAyGMPkH1BXju6IAjHypdbMWnNlVVFzdK1e5V9l7uLh\\\\\\\/9cTKN9UjM+onHWckrMrPBanZzW81fLXu+\\\\\\\/w65XbOcgDaGLiJLpb\\\\\\\/lh73albmpL5bBz38Lmhot6YuSG7u17OIPbMRzLzFp74kgEHcaI86M9fZ9O1hdKEG9Y3DgSjLUI+\\\\\\\/n4wZB5AVTuGlOexxw0m01xCoMg1gzeL3dZBSSG6nB\\\\\\\/oCA1yQ\\\\\\\/ABdwkc5PYvOrgbtOsRTZa1zPn+o4xnEhekw5DG+qgy5JkP\\\\\\\/hxiYrP2yvKgvIUSQve2SHoS2ZZT\\\\\\\/4ZRpQkD9j4b4bVzHQaW3uMvA\\\\\\\/THFCT+fOlktGpcJYEzbTGo4RqiWQXb+jG3y0IRn\\\\\\\/J1YAEteVRW5wuAdytekKe+uV6ZyxqXsSAlwHbvqCIMea85RYEStWvhiK9L5VBuzkYYIl9GqQPJ3BCAmx\\\\\\\/eQQYapLWaLAZujuH4LzEMwQ9nMQLGnPBBFqPa+SaO5k+sqkb7IKRM6eiVfbVmqLf4K\\\\\\\/yEanL\\\\\\\/t1pVhscWPJZ6VjFEFZfjy6FqRWXPcRrlH2W+7avuMlM\\\\\\\/E7TET3KT7KPPMfn+eKgBKq6fNVyR3JgFYbqDalbbzOPlPm1LjvF26LxCaXQG2VAI7dWa6nDHKKiSDMs4vb6I256S3CM9Du+52ZQ3qQEC7KewFav0O97SVjax+egD48vvRvo95bsy5kmW\\\\\\\/8VRaQFg1cVRBghOzPT9wiKfKWRxM0KB29mj1kW6TSoaL1DSFJS9aW3LQlWOOpa8bVoeKi6cDygCZege\\\\\\\/MPqjVTtS3sj+y+qnmGwDKwvdAdUOE9usCdLXxDBGJ0XlaL+WydVdqWNVCGKzMJ++miqod58a843YhPbapTMedr7c5KCIr+hZdGMtr2ZdfG49iSpdQHJjcIHVgRtX8TmDq2dtGZC0t4eIZzHGFRz1Hqoc6Jf21a1\\\\\\\/JDZVf4LYCORBaTKfLAC\\\\\\\/OJpDnp26zP6SbOmMm7MV54zvuTQjbjIWdgdlfvfhX\\\\\\\/\\\\\\\/C8qJxD+oKtzJ4mRJTImQXn+gP0W177wy\\\\\\\/PlzkjXpU6tctFFXcEiQ7ENhKdm0FZ18qDTPWrk2yWluH07+S1E9rLf9iGoU4WowtdlbadKScnz7\\\\\\\/LmwYCfE+NhDs8h3NfIohRC6crhltTg9dnBxVuxnk4QZrWqehdcMWlYWCwoHaLlaHqTyts93L1FV90XxwThWNiSJ0JTCPvCH7v422vGtYaSZkxMRmxHrSHYU1l0IKgKiLAGVfPobWwJbQiCPPPHnwIx2\\\\\\\/DSkTkvRmZWAq9NQnhJEcM35vyFq2Cf9KVL9NracYn9H+6T5bedX8joo24coEyE8axTvJ3P5glJ7ejiNpfM5r8yZMEaiML7+eg8bfIXvW5YC7ec3wRPOpbW+Q7Fe5PmT82PKRmLTUlec13C64p91LfGO8JOiMmSNp3l51mA1m0k2xUB23JgzXl\\\\\\\/TmdcRVN4LZ76WKCPBKMmT\\\\\\\/eEQyxHg1RipG0VgjgZ4yVZdkfMUlks0fwWwv3w9Ci3\\\\\\\/wr3nVagwi13LLNELpQC\\\\\\\/M5tZnIW1hEpqxiqIvVL0AeQfMDzm57LtPrQJcSswpCQWxRIzHzJ5zkNE5INe4tfhmhONmcOOUK+ykTE\\\\\\\/ip6VrZbLqMBWW+059vwmlLA1SVXCgmfnvOXZPYPvdsdPSJ29uKMSHtvtqnBTSqDf2R+2sVqYtGljeC8DBtLGgowSU6pUMglxkdC66df5R8mkhpGrAULLqwUQK6aY9uEsb1nkcMVO3cdEuUixieQrdK6RoyTdIXnLyq7gp2fvUCMlkE18fDDPV3\\\\\\\/JcxmmGw38gOJVOIeZMu+lrp9rQw4NE0+Rfs\\\\\\\/+9GPLvIWr7iXUrE6nhPU85iaz4N+NCTi+OA02S8ZPsRX9xbc8rZwJn61PgWlCS+OJTJslq4WtvdQc6SCkEdZHdsNErBwVriFtJFngapCq3DUSLxT+Ng9DqnMQ3sfjVDK0cZfJNkQ+95gJPn+bPSPieXXGrTEgRMXWtgcj5kNlWgOYPHIhbh4IkDQPqwHQHepMENICejRwzLK1iHXrVgE1q7zWc++W8lbiyQh2OcD+Ckf+mIkREJPeThgWD8wJ8oBg8Ww8ruuWotr9VtZr8PSwhYzT2uDt7wMxccJOOtO36ByR+DTDW\\\\\\\/X+O6KgM7taEclLjBoch\\\\\\\/7e\\\\\\\/C4YWLb5kCP606LVwIWyt6A906H1SPLJJvHMIU9VXuIIz01ql6cGe8+psjPF0aHqvDVWXpkuLYHL7RgmKIkVOcJmlnqprDEUNAtizclDyYXVkH2LsuZG3TwpUUxJ3kHTm\\\\\\\/qKlJyoKISpVcuODfzTu26s1cQ0i+5U9F7X8iO4gl23akVS6BEWvzh8mdhRwAXWWIFcQlU7vDIkz35vROVSE8LOLmoYOIFk4Dkb9SuQ1EP3llHGnGsqafexpE\\\\\\\/Mfok+l30MpyzMrvnKcq5NFgK9VLigwgIlkpXhoCt5LahenMWDph\\\\\\\/ZjMLHSPZ8yN2bd2so4d+wwce+Dt1jFuHCFnNTsJ7nzM\\\\\\\/CqnyzBIj7pU\\\\\\\/G2CETV1YSrHb\\\\\\\/IE3ia66R13QLvxDIAz6BA4vut7b92syS7XZLuEsxBrO9flrUD9snisu6oJFL+dwLBHl6szfddKKuc1dyfJPEpOSkMMdVMBFBUz4NeQ4z7xj2Cgo62H4dc5c9jgqHTh5l0I5k9Zz35Oq13\\\\\\\/Gw8F5mcaagYvTTsTyB0VnJxMWHC7CyxPQ6eT3wskGjAmANuC7nW0BcvbgD0fvnmWJQwRotgDyHc0Ng4batUfwtsnZDln3zuz\\\\\\\/cGMU7diqyEFn\\\\\\\/X\\\\\\\/a+OrowBLOY04Lwe5Un0FOaVGGp4\\\\\\\/\\\\\\\/edIKjWywCIuKSVSkk3i5ehaHtMox\\\\\\\/5u9BKf0XhRGZEQuD\\\\\\\/F86ULjxUUNQRx8TmG3O1kzOu8CToNIbAm3DNhVP1L28zQL6HfG2McUmEsl77Tv4c8cZ+SkP0slL6j+HpkYOPaCVqNvkX4aUF8h5crSznJfuXRJYy3qbR+GEzoK+\\\\\\\/ixv9qUoTtPFYR7P+b0GEr6fRg\\\\\\\/Yp0RgS6vBJNugb9UKw4tMNG\\\\\\\/Dt2ohwa3boewop0mopQtai6pPKaZOM3akHpvRBjuYXKXgsy7D9yT4HXbN\\\\\\\/yz8zsmV4ELqyzq4dGDBcfPQAdQe6w2olP00OrzONCYx2KKL3AM2duyK3uuRqkoHj\\\\\\\/vhh78pY+cpaFKRLR0of5w\\\\\\\/cWkyV40nMFb002MqORBhEbxZu8xzuUlYla0V2fRwDusGrRlfc\\\\\\\/U6VfFdwr7J1N2KeOxEtaobKpJEN3VpRso8k8C64xysymh8ezG2abeNV8QAMgk9dgM3ELqJZl\\\\\\\/alv3a4m2FHRIlqWIu4qo4U736wTRZ7R9rWRME6lAq0xV4OMCzSdDXk\\\\\\\/uwQ2rlrOQ91XjJyqt+zumOJgtxlPgXQhqH7tVVUjomZHvcOgzMBbU44oDH3nfDo1j0vW5l6mLvLVK8JvimttTiJPtlMaqumslCnsZwoJCS1H8xHVZrDfGELLLrYYCaeeVoqXh3tv3q+Wt\\\\\\\/U+0dalh5ja6rY9Agwm99nXfSyV\\\\\\\/A5pOdJzXj3Vi\\\\\\\/gCpSmVP5R6oio3JZQtLjYmnh5w3GJmiohJflaGg90GWOKDl3jaq\\\\\\\/63UbSg1ORQIjpJYfFwT5puOVzoWIpS3UTjHkHvW4wHU2pEpvtI5nDsHm\\\\\\\/\\\\\\\/aPfvW\\\\\\\/9epfk9gFSmUWiLlrGzvkrDxiqcWoFJJRK40F0hCXgRiklEMy9IHuA3yWdk2VD2DcJmna0XQ73ytSGGJSSZYNsSZk0kq3Wg\\\\\\\/ILPFzOS1NFzRw\\\\\\\/T6t46S7R+M3lhpkk2rrKvb\\\\\\\/7rPl3wK8f7MyV5j30M9vjDfuLM02\\\\\\\/Z299mO07G6OuscgTqUyASOLQUHmri5hz9MTve0xsNw+E9h1GOerFeFhETOMc7tNdZX2V328YDw3mbQeehYor7cZejFyU3ZqlgmFb8fzqtOsXLe4qUYTyzUp7QLdcfYW0wiw5ywk2h+rPjms65yaBtix2teZzyb6kQUmh6CD8HL+WgA8xYBi8evL1o95CfPRRNg7k\\\\\\\/f8A945UL1eNU8aNL4HlpWuwNkLK7VZkepJboTFl4Q8zU3u3jiNMQb6E6obUIvHXLe7fkxCBbtzRkLG4JmcQskb9bMjtOa09S18hsKW3NIxFfevZVHI3t6WELeabRoyqRKG7WJ2UnkmpH9fk6U68XH50gi9MamKjxOMEAwF7Y2uazJFP\\\\\\\/WfUIpuG302Ra4AszAEfKBi\\\\\\\/Qq19wF\\\\\\\/eX+MeOrT74ZJr5snTp93HbqFIvxPdPWi9zxkhbHyHCOZUyrFl8afDdGF\\\\\\\/HdOU2mJ7lghDL5Def5Q+6KvgU3Tn5LkvYvUpqevw5MyKuusKDtNLb1gEPU9Du0Yx5l3a8fol\\\\\\\/sAno4xfv5Lx4uRV\\\\\\\/o3RUlpDj3JRNigondvKhd0CMV4UGjdXKuv+c68jkRcbRg7ZvQqFoJHoOAGZCNgr4sR7+9QqNl0OsNJ7exS9Mwp5PxvlB\\\\\\\/k1UohB\\\\\\\/PRnpZplZ71mZYAJUtdotzp0ChHYrlzmNPHjyOvNM3Ljfz9g0f1v8BVzSdfNZoXgifzt5mTu+2dRW3HumsaHM9774TYBYsyO7815hX\\\\\\\/Z29DcYwxARs9pu50oEfTe7y6fcs\\\\\\\/v5aFIizQgVNwfzaalvWYzjVTzoVS0dUHdipSzfrtlY4cCZ8dzKjlSjiS\\\\\\\/ImpsweGjZzG5+WABQW8pscNvJ9EFQrHXEvm8NO\\\\\\\/CsjmYQ1Pjd\\\\\\\/5Cq0dbTf65YeBDuEdUBVB725lbds9hYoz8Bsp1C2GhrTjxyDZ1AFPxAm6MTEj+0odN+8WULYHHHK0PvD0N2p5BniAr7VBv2EGKlsuW7\\\\\\\/e6kFoG4iR9\\\\\\\/\\\\\\\/zeIZtSNiyPweBKSeNy+bJQRJptevU0\\\\\\\/05F09Ba3H5pXjvWyzuxSMi7Uglhdb8DMdGayrbd0Xr10uxwt7kjlqWGUKtFVnWZ+TT0RmkaJjOnuqN0VuFAhnCtr2VMjECnGz9IxiJRNAFxGYuXJncY8LMPOegQsDwswjbPnXB0HM7KIkROhcn7v+T8NTVrObNX4C2V3US\\\\\\\/tSwcMIvhYe\\\\\\\/L2jSYUU5C51KrcBoyhQd6ktOLiKDq9Q8OB0icTAu8eZVjLCe0ehvrzRxmQyEY1PQs3keVbxhXbozVuWT9v4kzMSCI0ViqJtDMJLnle8+4NOqr2TDJ7FcUjNxUqsSuA8Bt\\\\\\\/nrh9oat\\\\\\\/66tqrL0oXPepWXAt85yOI0A1stvBryO0qKLobKEo9H99695dAIg4J2ali+CrCfdvVyaMA5B8MTaDIRATCzpheJLKJSawKdFDOQZcQvaEjrx09nF3eFN5nWPkcf8ORsczd71zr9EAvf7vjY4ArCb3HmskebVRdgIVmwBAWBEbbHbPt88D8Xv8FKZfzrGCFlAZi+LZHKGCIgfM9kOwHP0qm7sYyaOgPxBnERuiL6gcGgih383+kQBa0L6Ibu1q0ov2BsmqDDh4FrWzZGXH0R7zjHlo3i53eHiv6eTd8CYmBdOp4YB4QVyOetQDSmzzNR1iGy2JvE8ppTUO1HGm9uiqXCQKw9fNH1x4o15azu60lMKDWQvJHI7KcyvdYz9rzYgO\\\\\\\/r+piYKvc7IpEtL9P\\\\\\\/xCVkZzC3aYcsSkza0XH6ZEtBCtA\\\\\\\/tVuLndHGGGmuXNPynZ53XTIa8mmhWE68CoBuL73m\\\\\\\/2ob2hcN+fH1CWLZym25bHe+0ApD+MuskOnbJiCJhd5gqBjKSrrxlMMpEacjnJ4ZNltq+wPj7QO0\\\\\\\/vyK8G0D5vl+6WDrk1JzPC1S+euZR7QG\\\\\\\/62N\\\\\\\/IRAvsucnPuC3UkjuAMA5octM5WNshfyiiw0NNuwW+IIrkBe2jXvxwFB688kI1h\\\\\\\/NYpoTct4Eh\\\\\\\/ANfbkCAXiKZkpV9qcxGH1DTMlagu4ffu15imwWGop8J0CihIENABaf9sDcYA8YB4x6xwhmDNIjmJVUHQIDtWaTI3EWGwsulcOwFZROKhuAIkUGKI2c1ZRsAps37OVOF80X8sI3BIhcraj8oqH5JxcHYURiHxOtCjr2Sd9y3+uv46aW7k3flXHFFwMQ5jno+M3\\\\\\\/U9qzET30CkTy1AbHhLiShc6sPyaw2TUB2IrP7buxeTkLL49INmBOUDx3w8kGM\\\\\\\/bKYpzNqP1no47viZ+j9uEcms5m9FlGgExtgp71brSzO60rlXFztf1VrwNCv5sT0IdBJZpPrOCFa+\\\\\\\/QMPGxkUfoGvibDoX+qvZxYR62McqUjJViCz4twjPa1UP4Rn+Vpyh28ydr4gvYSSyf6Yjtkg\\\\\\\/Z2VJXI6diOQlmIG+Oy1nTcjfVXzDbMCHVYl3hT3\\\\\\\/fAaykEyZMQTDdhq5k80bzKIOPoXpPLmGUxZO6GABofBbkVcbxdpdmKArd50\\\\\\\/ILTXEsraK915iAcQ3hl\\\\\\\/OuKvxL1DoYIsXd8Gt7zOODijZ5pgaMyr8zSUWmLnAhZpw09iX83+YbSLlk6sX\\\\\\\/WGhlrAJIOhucl9UYLLfZ41W0Fr7s4CK7iZLkhNiSdrnwDbcQgw\\\\\\\/8EA0tN\\\\\\\/RciEB8zkWoy6\\\\\\\/pVxJXprM61fE44RzAUOcwBkZuDRmi6AliWztPqHgTmMfbLUk2yL\\\\\\\/DE\\\\\\\/15i7x1+WFkqhdGDIc0Y52J7pt1JKT\\\\\\\/IY3NWic9S87Ydw8mWhMiOIaKSBiZ3liVkeGdff+CTk7dxYKPRxnJ9CA0e98a6+qqUDxYERu87AMCmplKQgel+dhOS48ecRwCOwiBlVD2LcsjN3lohLC5hnHiegZ5xAKm\\\\\\\/3I83D\\\\\\\/Av18VSo8J44F5KHdq9j0PdOcgTEIeK5QAfmQZshCiZabUfae0yN6p\\\\\\\/jPjbuLylbBFu+2hsw83o1ihB8h9dkwKVXOpoA0BP9ORqYvouLkA\\\\\\\/vSFqlyiVT5d0iSieRTx02Nf4i7PRJJKe\\\\\\\/mYq1NO0RJ32DG2U\\\\\\\/mG82UwT2nEafnEWG8BVPGJpAXIQn0pSFAIeyOiuk84Ng5EEdmGGnkoBPy88Q4yC2dtP791TaVJ1PtkT2XhyKnrbABFOFD\\\\\\\/blBYkZfNigL+SgxafuMV9GNisJkgIACdFbCo68lZuAk9H160O9qy\\\\\\\/aAVPAjHd99N95mRotVxWZy0rCwu\\\\\\\/5sxSVTIIVU4fHlThwI4TwivgpMBBpTKU4ckOPWt8BsrixBl2SSbs4YPeskO\\\\\\\/IakYb+c0CQ3ZV0c5ZUXm6ttW96FykWhrHY6OWEMe66eBU8qts99tj2Ju2W7J38ZErVpXhcW199g9mKi1rrKydaSnuFmLTcRgJIRJi2izmwR5EIShfhcL1xr7gjGW5KLh\\\\\\\/cWXjFwz5qs8YRz+soJhPn0MiTyFCJZxHWdeanyCO0QU3zsMm3LSiuE1QZmdrWGR9tpQ1wCVAM4hF35kriaT1YlhexFagtDDM2JiQR90bcxHnlR2TUF\\\\\\\/c6scMf7Y25oTiZP7wx2HmTqwfsmg3E\\\\\\\/NrD5ZZzNqMX85SBspKIB1WjAFrrU8akmM9J99adNtizCMj3XDopa9bEGGhW6qhyjs8Cl0LjozieVPhSiPcfcE5C9fp9KPLPaoOJCinAeScgX\\\\\\\/cCJw394MxOKiJ72d0Ak8yNsKpqaA\\\\\\\/AUwCx7Hg2VrlG4Bku9K12qPx5YsmrRm+SIGRhzVUYByfuTdrIWDsA\\\\\\\/nwkmo+XVLYVZo23hVIRAa+I02fcaAFlcUGSYwjYgORFzDySxTpiAa2DkGLELgjWt7c3Q9Yvl5r\\\\\\\/OajUthdex\\\\\\\/j+suJIjJLE2dSQmOYgKMy7tTLX+4i+d0eVxPkNZM0qEPCOvtnidh9ye3RDxgkByRQTbc2IZFGPxU7AOEBr\\\\\\\/\\\\\\\/2O1cm8u1EJxzqx6QFM+htCvSOePqRU8k6jDqTxrb3KD9TUiZ9a+NbvaZO0JSNmtCk06zIFF8BmthBBSIG37RH7LsOyRY64Hxij67RAXiilClY7\\\\\\\/eqoqATGsEDue6xzTZYV+S08RLOdYdyCro4tLO6pstvl+jiWYlb1K6VMeWinnDLbXDUjkt21KoWLXAJmj6hm3kiyqttEu4Q\\\\\\\/4axY\\\\\\\/IDm+iQ54ZIz6DNe41xmDzqwcoGsXHdRZ88Pj18QzHENDouVeJwxXWtRMHVPC5aJb\\\\\\\/AU11G7BJgXVnSxRVX12NM8ikloy14g7b\\\\\\\/5vgAD5myaA\\\\\\\/H\\\\\\\/MKqQ77ulg9QXv3wLR0K6vKr4hHRCKpODs15UbFYcrbMObbY0aic4U\\\\\\\/vg2tIaL+YaLQa+t6Nfut6xZy+9lcDC+2iPnXEJFQIumh2l61cIXpw7tREdBPaa25U543YopP3hyJZKbT\\\\\\\/fsI5haRNbFeayUYViOa2Oh31ZIDXiw0kJ8yVqSiLOvhR6REyhgW0Bl40M+Ykc+cDQkvTj51UtO4SMDzfr92aK9Ytq5c6\\\\\\\/qB7mzfxdaVu8y84Tstcj5X0s\\\\\\\/Kg7wqkuT+pYGTxPX1Y5ix1rge6mOlRZuBeZhA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022CTQ\\\\\\\/5mT5koDONQzn9y+Yuw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002281c4044ae48ef3bbf6d02b1034c344f813c6d7d6e2dc0e5127a2bf836814c7f9\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022d40JV8zfwhNfNeDQnxzvi9ghORiJfCPtOVCctMzUca4Y2tSvcw\\\\\\\/lA0kZSvhtvCEQ2tk6CRFZHmPgf8lFS3IMbhT538Kai4gMjRSInUUqnuPNmMdDiwSR+fdnxZa\\\\\\\/TprCAb1c9Hg+MKCdJT0XktMQAxl8M849x0UNhZttgZ57Bio9+Kjgj52VbImfQs0OQwN11slmVWZCAQ\\\\\\\/XUwgq9A95YT3Yfr5htKpxi30DgZ8431G3d4BpxODG18wlcK8pKDFaZjsBNDE\\\\\\\/nZenIaiggIQ9EsN2KWV6M8DLg5JzKYn8RsCO9KrZsHnOnQ41Z+\\\\\\\/XWw8tZmvtxp+pXwrO+SJcl4BdrhBX7sUu3FlHUgONwG0FGOQwroslyZzvjbkR4+8JRKd\\\\\\\/AGIOGn+71XwRVcpG2gFZEsRSqaDRnvOz6ofEK7\\\\\\\/Y\\\\\\\/3ceVGsmyaI78YZFr7xry63OxvInW34V3B0vTfRCaBxbbiFTv77zqsmLFXsZ3auYYywPHzzuSnEOMBNlBlQzYfwKVXTiSZV\\\\\\\/shkMgRG0QoMqhFCpfGFoiyhA1\\\\\\\/V9uZ4hRnL2kkYTsWApNMXtE\\\\\\\/P9MYNMi5pHWIFbE847sYg1ETvSkVWEdbG8QXni\\\\\\\/NR4SDfjlwu8ozHwC+CUIu8F72fUyQLbfIxkjCJb\\\\\\\/+fUo8cypLvIaIF3RJ0RCvOYMujOpPNVP8o4eoX\\\\\\\/Pnf7PtMchvs1ZHEtMzje0Jkfm7eQcJChjzWyCuIF4rJq9rZwF5L67bxabUff3+HGGKWo9JD+qEVdFVQpiDMRlHyf3LIEyvMnRo1WzIm8mnOdHpUCcnA5mQJze42Id6mG+287bQeTlu+FdbiB+WadKyCvnMXGKGkUDtPM8VNSSKdjWmyV3qmdaWdVRGcEHnzkOukUqZBJIu6M8PedtYwhyPAHVk\\\\\\\/0ED\\\\\\\/veiw45F39rmmZcFe\\\\\\\/t0vAe9lcBn94bu9qrKSEMBGb0m3liRz8jpvtafXYMyCJhkuubkTM3aNlGGcyUbfjVoY9xwZxmahrv0ppNxo3J0ycp9AvlwigVcffstd5N\\\\\\\/9I9qAvSRKImvbA2msM0qoN2nBz3MRZiz6+wtPoWG6DRNUyFyNxHeviK2L1WCuNVWf1b7hyUrq+mBLorOGuuz9rwm+pfP1JvC4tYmZ9ldnjnGOLhmmbrRphrFr97pkrmtTfWlGgjDC1B8OK0EgrWscy42iC9aLXWw\\\\\\\/3wL0mw6xfCf69xREywZMb68wmoxM\\\\\\\/cBYR1\\\\\\\/8T3rktO\\\\\\\/QGjozXEFkFvxdwM9GQrMb4E6KNn34CRmRJX\\\\\\\/42pbCxAhIhtjQkMrKRFpBFG2iZxoBR6NgfAY\\\\\\\/AkM+ZvgiRh3GLoK7BGA1t+vHJL1sKFgOYrMJLvmSEoCwqKGPbdxivAuqWaaY+iBm8wMnxuHQ1pjwZHDC+Yi6cDDKDBEsBJChVcMhoofCPyJFhZcMlCEpvYScED7rgrZ2QoITRXtzC6Pfm54PUNxXT4TvwRilZb2afUosPmdwiTc3bUGplhuk8E2b9FbHaLdgjjpGfmFUTjhd\\\\\\\/G98ImDbTTtliDZjtyxzi\\\\\\\/BFcJW\\\\\\\/3su08UHnMPihzhK8td+oVJZZ4C4Ln9Lqc0J\\\\\\\/vzNh3uppK7KVEGW9sXXsp81r5dTFdijRy38F25TvqqxxbaSaj+eDrpI6hJsq+9uVgFbmRUsAsj4os8lrgx7tAuyFg2CLqmNOqMR6\\\\\\\/bZ1QNyX+EX6lE16hlIwIdS1xYmOFqtQX6a05SvpcSWB05ZoVj26v8+MSC2SX4zUwGjha0AhuZVT\\\\\\\/6g+JD+81N2bUgGXxLg6+sOSAFpb7xlOUChwh6gjwFoICijOLJgQ\\\\\\\/rB8Si3qStksLvMws7Q\\\\\\\/6NTiVprln+His1rCa5wpr59maR6jMGIsXRrjb8zG1pQz1H8fyIRUlQrRXNznvSSy45A06fYcv8j62w3H4Is11TMs0mjuD3BMH8vwK06oq24wXOhHYV9A0UOP\\\\\\\/JUc46Cl527hEnr1P20WTZStfyV\\\\\\\/ez1hJy83oQ8i50oikVc+SqmDJWqNQdA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022UbVgphW4Fh7fJHMtJKCh4A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022231c9477ad6348773c71d808e11f22ba76d5e36209694e1fade5b4ed4349f852\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494055 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/2977" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-nDtX9aS-JSxUX_a-hA30iXYi3sAGvbyffapoG-7tp1I" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/2977" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-rVqg4fMEHY8HSsI8HYLaGo2BHxhAAJNTOwYj5gublyM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/9/2963" title="Multivariate Genome-Wide Association Analyses Reveal the Genetic Basis of Seed Fatty Acid Composition in Oat (Avena sativa L.)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/9/2989" title="Genome-Wide Association Analysis of Anoxia Tolerance in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494055" id="node494055" data-pisa="ggg;9/9/2977" data-pisa-master="ggg;g3.119.400438" data-apath="/ggg/9/9/2977.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">FERMI: A Novel Method for Sensitive Detection of Rare Mutations in Somatic Tissue</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">L. Alexander Liggett</span>, <span class="highwire-citation-author" data-delta="1">Anchal Sharma</span>, <span class="highwire-citation-author" data-delta="2">Subhajyoti De</span> and <span class="highwire-citation-author" data-delta="3">James DeGregori</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2977-2987;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400438 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494055" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">L. Alexander Liggett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO 80045</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Linda Crnic Institute for Down Syndrome, University of Colorado School of Medicine, Aurora, CO 80045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=L.%2BAlexander%2BLiggett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liggett%20LA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AL.%2BAlexander%2BLiggett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anchal Sharma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Rutgers Cancer Institute, New Brunswick, NJ 08901</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anchal%2BSharma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sharma%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnchal%2BSharma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Subhajyoti De </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Rutgers Cancer Institute, New Brunswick, NJ 08901</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Subhajyoti%2BDe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=De%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASubhajyoti%2BDe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">James DeGregori </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO 80045</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Linda Crnic Institute for Down Syndrome, University of Colorado School of Medicine, Aurora, CO 80045</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Integrated Department of Immunology</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Pediatrics</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Medicine, Section of Hematology, University of Colorado School of Medicine, Aurora, CO 80045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BDeGregori%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=DeGregori%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJames%2BDeGregori%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:james.degregori@ucdenver.edu" class="" data-icon-position="" data-hide-link-title="0">james.degregori@ucdenver.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/9/2977" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494055" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494055/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/9/2977.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494055" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494055/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/9/2977.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494055" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494055/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/9/2977.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494055" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494055/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">With growing interest in monitoring mutational processes in normal tissues, tumor heterogeneity, and cancer evolution under therapy, the ability to accurately and economically detect ultra-rare mutations is becoming increasingly important. However, this capability has often been compromised by significant sequencing, PCR and DNA preparation error rates. Here, we describe FERMI (Fast Extremely Rare Mutation Identification) - a novel method designed to eliminate the majority of these sequencing and library-preparation errors in order to significantly improve rare somatic mutation detection. This method leverages barcoded targeting probes to capture and sequence DNA of interest with single copy resolution. The variant calls from the barcoded sequencing data are then further filtered in a position-dependent fashion against an adaptive, context-aware null model in order to distinguish true variants. As a proof of principle, we employ FERMI to probe bone marrow biopsies from leukemia patients, and show that rare mutations and clonal evolution can be tracked throughout cancer treatment, including during historically intractable periods like minimum residual disease. Importantly, FERMI is able to accurately detect nascent clonal expansions within leukemias in a manner that may facilitate the early detection and characterization of cancer relapse.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genetics" class="hw-term hw-article-keyword hw-article-keyword-genetics" rel="nofollow">Genetics</a></li><li class="kwd"><a href="/keyword/cancer" class="hw-term hw-article-keyword hw-article-keyword-cancer" rel="nofollow">Cancer</a></li><li class="kwd"><a href="/keyword/sequencing" class="hw-term hw-article-keyword hw-article-keyword-sequencing" rel="nofollow">Sequencing</a></li></ul><p id="p-3">The simultaneous growth in accuracy and reduction in cost of DNA sequencing has encouraged its use throughout many diverse areas of biology. Accompanying this explosion of applications for sequencing has been a natural demand for increasingly sensitive sequencing methods. While the detection of high frequency variants like germline SNPs is not particularly challenging by most sequencing technologies, sequencer and library-preparation error rates are typically high enough to mask most rare or somatic variants. What is perhaps most challenging about library preparation is that the very isolation of DNA exposes it to oxidation that can change base identities (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Shibutani <em>et al.</em> 1991</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cheng <em>et al.</em> 1992</a>), and high-temperature exposure can thermally alter nucleotide identities (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lindahl and Karlstrom 1973</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lindahl and Nyberg 1974</a>).</p><p id="p-4">Because of these sequencing and library-preparation limitations, quantitative PCR (qPCR) and multiparameter flow cytometry (MFC) have remained common methods of rare variant detection (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Terwijn <em>et al.</em> 2013</a>). More recent technologies such as high-throughput digital droplet PCR (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Sykes <em>et al.</em> 1992</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Vogelstein and Kinzler 1999</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hindson <em>et al.</em> 2011</a>), COLD-PCR (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Li <em>et al.</em> 2008</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Milbury <em>et al.</em> 2012</a>), and BEAMing (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dressman <em>et al.</em> 2003</a>) have shown promise for rare mutation detection, but are often limited to variant allele frequencies (VAFs) greater than 1% or are restricted to assaying only a few chosen mutations at a time.</p><p id="p-5">A number of studies have sought improvements in sequencing technology accuracies by targeting and labeling small regions of genomic DNA such as sMIPs (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hiatt <em>et al.</em> 2013</a>), by paired strand collapsing (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kennedy <em>et al.</em> 2014</a>) and through other targeting methods (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Flaherty <em>et al.</em> 2012</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kim <em>et al.</em> 2013</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albitar <em>et al.</em> 2017</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Onecha <em>et al.</em> 2019</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Mansukhani <em>et al.</em> 2018</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Thol <em>et al.</em> 2018</a>). Some groups have also incorporated error-correction methods to eliminate sequencing and PCR errors, like PELE-Seq (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Preston <em>et al.</em> 2016</a>), and error-correcting enrichment processes (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Schmitt <em>et al.</em> 2015</a>). While these targeting and enrichment methods have certainly improved rare variant detection, they are still often limited to detecting variants that exist in at least 1% of a sample, or are limited to simultaneous detection of only a handful of variants.</p><p id="p-6">Here, we describe a novel integrated genomic method that utilizes single molecule tagging and position specific background correction to push the limit of detection to variants existing in as little as 0.01% of a sample. Initial detection improvements come from the quantitative tracking ability of molecular barcodes that facilitate the elimination of the vast majority of sequencer and PCR amplification errors. Combined with paired-end sequence collapsing, consensus reads are produced that contain reduced numbers of false variants.</p><p id="p-7">In a similar manner to previous methods (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Newman <em>et al.</em> 2016</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Young <em>et al.</em> 2016</a>), we then experimentally derive a background of expected errors for each position within the consensus reads. As we know that sequence context impacts nucleotide stability (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Benzer 1961</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Nachman and Crowell 2000</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lercher <em>et al.</em> 2001</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hwang and Green 2004</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gaffney and Keightley 2005</a>), we use this background to correct our consensus reads based on probability density functions created for each assayed nucleotide position. We build on previous methods by then extensively characterizing the background error probabilities that generally occur in our sequencing library preparations. These characterizations were sufficiently comprehensive when applying different degrees of bottlenecks to cell lines, it appeared that no variants were detected when none existed within the 1/10,000 detectable range, supporting the possibility that FERMI often eliminates all background mutations.</p><p id="p-8">One recent application of especially sensitive sequencing technologies is assaying and understanding clonal evolution within cancerous tissues (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Greaves and Maley 2012</a>). The rarity of somatic mutations, even within the clonally expanding pool of cells that exists within a tumor, has limited the observation of changes that can occur. Such an understanding would be valuable, as cancer therapies often leave behind a small number of cells that can frequently lead to relapse. In leukemias, the state during which these small numbers of cells remain after initial treatment is referred to as minimal residual disease (MRD). During this MRD stage, residual leukemia cells continue to evolve, and successful detection of relapsing leukemia at early stages would facilitate improved prognosis and treatment strategies (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Krönke <em>et al.</em> 2011</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Ivey <em>et al.</em> 2016</a>).</p><p id="p-9">As a proof of principle, we directly sample leukocyte genomic DNA and demonstrate the ability of FERMI to detect oncogenic changes during the MRD state, and monitor clonal changes with time. We also show that by concurrently sampling a diverse panel of oncogenic regions, we can detect the expansion of new oncogenic variants during MRD. Such observations could be critically important in predicting relapse in patients.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Amplicon design</h3><p id="p-10">Amplicon probes for targeted annealing regions were created using the Illumina Custom Amplicon DesignStudio (<a href="https://designstudio.illumina.com/">https://designstudio.illumina.com/</a>). UMIs were then added to the designed probe regions and generated by IDT using machine mixing for the randomized DNA. Probes were PAGE purified by IDT. All probes are listed in Table S2 along with binding locations and expected lengths of captured sequence.</p></div><div id="sec-3" class="subsection"><h3>Genomic DNA isolation</h3><p id="p-11">Human blood samples were purchased from the Bonfils Blood Center Headquarters of Denver Colorado. Our use of these deidentified samples was determined to be “Not Human Subjects” by our Institutional Review Board. Biopsies were collected as unfractionated whole blood from apparently healthy donors, though samples were not tested for infection. Samples were approximately 10 mL in volume, and collected in BD Vacutainer spray-coated EDTA tubes. Following collection, samples were stored at 4° until processing, which occurred within 5 hr of donation. To remove plasma from the blood, samples were put in 50 mL conical tubes (Corning #430828) and centrifuged for 10 min at 515 r<em>cf</em>. Following centrifugation, plasma was aspirated and 200 mL of 4° hemolytic buffer (8.3g NH<sub>4</sub>Cl, 1.0g NaHCO<sub>3</sub>, 0.04 Na<sub>2</sub> in 1L ddH<sub>2</sub>O) was added to the samples and incubated at 4° for 10 min. Hemolyzed cells were centrifuged at 515 rcf for 10 min, supernatant was aspirated, and pellet was washed with 200 mL of 4° PBS. Washed cells were centrifuged for at 515rcf for 10 min, from which gDNA was extracted using a DNeasy Blood &amp; Tissue Kit (Qiagen REF 69504).</p></div><div id="sec-4" class="subsection"><h3>Amplicon capture</h3><p id="p-12">For amplicon capture from gDNA, we modified the Illumina protocol called “Preparing Libraries for Sequencing on the MiSeq” (Illumina Part #15039740 Revision D). DNA was quantified with a NanoDrop 2000c (ThermoFisher Catalog #ND-2000C). 500ng of input DNA in 15μl was used for each reaction instead of the recommended quantities. In place of 5μl of Illumina ‘CAT’ amplicons, 5μl of 4500ng/μl of our amplicons were used. During the hybridization reaction, after gDNA and amplicon reaction mixture was prepared, sealed, and centrifuged as instructed, gDNA was melted for 10 min at 95° in a heat block (SciGene Hybex Microsample Incubator Catalog #1057-30-O). Heat block temperature was then set to 60°, allowed to passively cool from 95° and incubated for 24hr. Following incubation, the heat block was set to 40° and allowed to passively cool for 1hr. The extension-ligation reaction was prepared using 90 μl of ELM4 master mix per sample and incubated at 37° for 24hr. PCR amplification was performed at recommended temperatures and times for 29 cycles. Successful amplification was confirmed immediately following PCR amplification using a Bioanalyzer (Agilent Genomics 2200 Tapestation Catalog #G2964-90002, High Sensitivity D1000 ScreenTape Catalog #5067-5584, High Sensitivity D1000 Reagents Catalog #5067-5585). PCR cleanup was then performed as described in Illumina’s protocol using 45 μl of AMPure XP beads. Libraries were then normalized for sequencing using the Illumina KapaBiosystems qPCR kit (KapaBiosystems Reference # 07960336001).</p></div><div id="sec-5" class="subsection"><h3>Sequencing</h3><p id="p-13">Prepared libraries were pooled at a concentration of 5 nM. Libraries were sequenced on the Illumina HiSeq 4000 at a density of 12 samples per lane with 5% PhiX DNA included, or on the Illumina NovaSeq 6000, allocating approximately 30 million reads per sample.</p></div><div id="sec-6" class="subsection"><h3>Bioinformatics</h3><p id="p-14">The analysis pipeline used to process sequencing results can be found under FERMI here: <a href="http://software.laliggett.com/">http://software.laliggett.com/</a> or here: <a href="https://github.com/liggettla/FERMI">https://github.com/liggettla/FERMI</a>. For a detailed understanding of each function provided by the analysis pipeline, please refer directly to the software. The overall goal of the software built for this project is to analyze amplicon captured DNA that is tagged with equal length UMIs on the 5′ and 3′ ends of captures, and has been paired-end sequenced using dual indexes. Input fastq files are either automatically or manually combined with their paired-end sequencing partners into a single fastq file. Paired reads are combined by eliminating any base that does not match between Read1 and Read2, and concatenating this consensus read with the 5′ and 3′ UMIs. A barcode is then created for each consensus read from the 5′ and 3′ UMIs and the first five bases at the 5′ end of the consensus. All consensus sequences are then binned together by their unique barcodes. The threshold for barcode mismatch can be specified when running the software, and for all data shown in this manuscript one mismatched base was allowed for a sequence to still count as the same barcode. Bins are then collapsed into a single consensus read by first removing the 5′ and 3′ UMIs. Following UMI removal, consensus sequences are derived by incorporating the most commonly observed nucleotide at each position, so long as the same nucleotide is observed in at least a specified percent of supporting reads (75% of reads was used for results in this manuscript) and there are least some minimum number of reads supporting a capture (5 supporting reads was used for results in this manuscript). Any nucleotide that does not meet the minimum threshold for read support is not added to the consensus read, and alignment is attempted with an unknown base at that position. From this set of consensus reads, experimental quality measurements are made, such as total captures, total sequencing reads, average capture coverage, and estimated error rates. Typically we required 5 total captures of a variant to be observed for the variant to be counted as real.</p><p id="p-15">Derived consensus reads are then aligned to the specified reference genome using Burrows-Wheeler (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Li and Durbin 2009</a>), and indexed using SAMtools (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Li <em>et al.</em> 2009</a>). For this manuscript consensus reads were aligned to the human reference genome hg19 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lander <em>et al.</em> 2001</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fujita <em>et al.</em> 2010</a>) (though the software should be compatible with other reference genomes). Sequencing alignments are then used to call variants using the Bayesian haplotype-based variant detector, FreeBayes (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Garrison and Marth 2012</a>). Identified variants are then decomposed and block decomposed using the variant toolset vt (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Tan <em>et al.</em> 2015</a>). Variants are then filtered to eliminate any that have been identified outside of probed genomic regions. If necessary, variants can also be eliminated if below certain coverage or observation thresholds such that variants must be independently observed multiple times in different captures to be included.</p><p id="p-16">The final variants called from the consensus sequences were then compared to experimentally derived confidence intervals for each probed position. These confidence intervals were created by using FERMI to sequence control peripheral blood samples from the same experiment as test samples. Following the logic described in Results, it was assumed that low frequency variants that are detected across multiple individuals (including in blood and sperm, where few variants are expected) were not real signal but rather false positive background. All of the variants from these control samples were thus used to construct a standard background. This background was calculated for each position at which a variant was observed within the standard control samples, and was uniquely calculated for each type of change. Often, in the construction of the background, the highest frequency alleles were eliminated in an effort to minimize the effect of true mutations on the background. A student’s <em>t</em> continuous random variable function was used to create a probability density function that describes the background distribution for each substitution type at every probed locus (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Oliphant 2007</a>). By specifying a particular alpha fraction of the distribution, high and low VAF endpoints were derived that were then used to determine if an experimental signal was significantly above background. While the number of samples required to make a useful background will certainly specific to a particular experiment, for the analyses performed in this manuscript and associated work, 5-10 samples seemed to provide sufficient data for the construction of a background. As outlier variants can also be eliminated at each nucleotide position, it is helpful to note that in some cases, samples can be internally controlled without requiring separate samples.</p></div><div id="sec-7" class="subsection"><h3>Elimination of false positive sequencing and library creation artifacts</h3><p id="p-17">A number of steps have been included within sample preparation and bioinformatics analysis specifically to reduce false background signal. Using the dilution series shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figures 1C-D</a>, we can show sufficient sensitivity to identify signal diluted to levels as rare as 10<sup>−4</sup>. While these dilutions show significantly improved sensitivity over many current sequencing methods, background error could still exist. The two largest sources of erroneous mutation when sequencing DNA will typically be from PCR amplification mutations (caused both by polymerase errors and exogenous insults like oxidative damage), and sequencing errors.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2977/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Method overview. A) Schematic representing the steps involved in identifying mutations with FERMI. B) The average number of unique captures varies by probe location. Error is standard deviation across 20 samples." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-678226960" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Method overview. A) Schematic representing the steps involved in identifying mutations with FERMI. B) The average number of unique captures varies by probe location. Error is standard deviation across 20 samples.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2977/F1.medium.gif" width="408" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/9/2977/F1.medium.gif" width="408" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2977/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2977/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494156" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-18" class="first-child">Method overview. A) Schematic representing the steps involved in identifying mutations with FERMI. B) The average number of unique captures varies by probe location. Error is standard deviation across 20 samples.</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">These are the steps taken to eliminate errors before final background derivation:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-20">• Elimination of first round PCR amplification errors</p></li><li id="list-item-2"><p id="p-21">• Elimination of subsequent PCR amplification errors</p></li><li id="list-item-3"><p id="p-22">• Elimination of sequencing errors</p></li></ul></div><div id="sec-8" class="subsection"><h3>Elimination of first round PCR amplification errors in consensus reads</h3><p id="p-23">The first round of PCR amplification performed during library-preparation causes mutations that are challenging to distinguish from those that occurred endogenously. Since there is little difference between those mutations that occur during the first round of PCR amplification and those that occurred endogenously, we rely on probability to eliminate these errors. Since we are performing sequencing of individually captured alleles, we can ask whether requiring that a mutation be observed in multiple captured alleles before it is called as a true positive signal alters the frequency of variants identified. We expect about 400 first round PCR amplification errors, and the probability that the identical mutation will occur in multiple cells becomes exponentially unlikely. By requiring a mutation be observed in just five captures before it is called as real signal, theoretically, none of the first round PCR amplification errors should make it into the final consensus reads.</p></div><div id="sec-9" class="subsection"><h3>Elimination of subsequent PCR amplification errors</h3><p id="p-24">Elimination of PCR amplification errors after the first round of PCR is done using UMI collapsing (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). Each time a strand is amplified, the UMI will keep track of its identity. Any mutations that occur after the first round of PCR will be found on average in 25% of the reads (or fewer for subsequent rounds). This allows us to collapse each unique capture and eliminate any rarely observed variants (&lt;75%) associated with a given UMI. Utilizing the UMI in this way allows us to essentially eliminate any PCR amplification errors that occurred after the first round of PCR. The method should also eliminate most errors resulting from DNA oxidation <em>in vitro</em>.</p></div><div id="sec-10" class="subsection"><h3>Elimination of sequencing errors</h3><p id="p-25">Sequencing errors are eliminated in two ways. This first method is by using paired-end sequencing to read each strand of a DNA fragment (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>). The sequence of these reads (Read1 and Read2) should match if no sequencing errors have been made. For an error to escape elimination it would need to occur at the same position (changing to the same new base) within both Read1 and Read2. Therefore, when the base call differs at a position on Reads 1 and 2, these changes are eliminated from the final sequence. This collapsing should eliminate most sequencing errors, although sequencing errors of the same identity occurring at the same position will escape. These errors should be removed when collapsing into single capture bins (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a>). As with the logic when eliminating subsequent PCR amplification errors, most sequences associated with each UMI pair should be identical. Therefore, sequencing errors passing through Read1 and Read2 will be very unlikely to match other sequenced strands from the same capture event, and are eliminated during consensus sequence derivation.</p></div><div id="sec-11" class="subsection"><h3>Data availability</h3><p id="p-26">The raw sequence data produced for this study are available in the Sequence Read Archive. The data are available as raw fastq files which have been prepared and sequenced as described in this manuscript. These fastq files can be analyzed using the FERMI software provided on <a href="https://github.com/liggettla/FERMI">https://github.com/liggettla/FERMI</a>. The fastq files can be found in the BioProject repository under BioProject ID PRJNA525088. This is the direct link to the hosted files: <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA525088">https://www.ncbi.nlm.nih.gov/bioproject/PRJNA525088</a>. Sequence data are available at BioProject with the accession number: PRJNA525088. The code used to generate and analyze the data can be found at <a href="https://github.com/liggettla/fermi">https://github.com/liggettla/fermi</a> The authors affirm that all other data necessary for confirming the conclusions of the article are present within the article, figures, and tables. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.9037457">https://doi.org/10.25387/g3.9037457</a>.</p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>Method overview</h3><p id="p-27">We devised FERMI as a method to overcome current sequencing challenges facing rare mutation detection. FERMI is based on Illumina’s TrueSeq Custom Amplicon and AmpliSeq Myeloid protocols, which are designed for mutation detection across selected genomic regions. In FERMI, sequences found within human genomic DNA (gDNA) are captured by targeted oligomer probes, which are then sequenced and analyzed for the presence of any existing mutations. We adapted the AmpliSeq process to target a much smaller number of regions of the genome (32 <em>vs.</em> 1500 regions) in order to achieve a greater sequencing depth per location with a reduced sequencing cost. We designed DNA probes to our 32 selected regions, each approximately 150bp in length, that span either AML-associated oncogenic mutations or Tier III (non-conserved, non-protein coding and non-repetitive sequence) regions of the human genome. The exonic regions were selected to efficiently cover loci that are commonly oncogenically mutated by substitutions in leukemias based on COSMIC classifications. The gDNA used for capture and sequencing was purified either from blood, cancer cell line or sperm cells, though most of our work focused on peripheral blood cells. The method should be adaptable to any species.</p></div><div id="sec-14" class="subsection"><h3>Barcode-guided single molecule sequencing</h3><p id="p-28">Capture of gDNA, including any existing variants, begins by incubating double-stranded gDNA together with oligomer probes designed to bind specified regions of the genome (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1A</a>). These probes span regions of approximately 150bp in length and contain two identifying indexes. The first index is a 16-bp sequence specific to each sample being processed, and the second is a 12-bp unique molecular identifier (UMI) of randomized DNA that should be unique to each captured strand of gDNA. Double-stranded gDNA is melted apart to allow these targeting probes to bind the resulting single-stranded DNA. Probe annealing is then achieved by slowly cooling the samples to allow for efficient targeting.</p><p id="p-29">Following hybridization of the probes and gDNA, DNA polymerase is used to copy the template, and DNA ligase joins the strands together into a single contiguous amplicon. Using the sample indexes and the capture-specific UMIs to ensure each capture is tracked, amplicons are amplified by polymerase chain reaction (PCR) and pooled together for sequencing. Samples were sequenced using paired-end 150 bp sequencing, allocating approximately 30 million Illumina HiSeq or NovoSeq reads per sample. This coverage encompasses on average about 1,000,000 capture reactions per sample, resulting in about 30X sequencing coverage for each capture (given an average of 30,000 captures per probed region). Though capture efficiency was not uniform for the different probes, which exhibit a fivefold range in the numbers of successful unique captures, we show sufficient coverage at each probed location to capture mutations at least as rare as 0.01% (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1B</a>).</p></div><div id="sec-15" class="subsection"><h3>Assessment of background error profile</h3><p id="p-30">Following sequencing, reads are distributed into sample-specific bins by their sample index. Within these sample-specific bins, paired-end reads are combined into single consensus reads by marking all mismatched base calls as an unknown identity. This approach yielded better results than elimination of pairs with some threshold of mismatches, as it retained substantially more sequencing information. These paired-end consensus reads are then sorted into capture-specific bins by their UMI sequences. These capture-specific bins are then collapsed into final consensus reads. In order to qualify for this final UMI-based consensus derivation, a UMI-specified capture is required to have at least 5 supporting sequencing reads, and the base at each position is only called if 75% of supporting reads agree with its identity. The final consensus reads are then compared against an experimentally determined background to distinguish true-positive variants from false positive signal, as described below.</p><p id="p-31">Though UMI barcode collapsing of sequencing probes is an important technique by which sequencing sensitivity and accuracy can be increased (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Hiatt <em>et al.</em> 2013</a>), we find that UMI-collapsed data still retains a significant amount of false-positive variant signal. Using leukocyte gDNA purified from putatively healthy blood donors, we find approximately 5000 unique variants within our UMI collapsed consensus sequences in each individual. To estimate how much of this signal might be false-positive background, consensus sequences were computationally binned by the presence or absence of heterozygous SNPs found within our probed individuals. This sorting created bins of sequencing that should have originated from only a single allele. Theoretically, if rare variants were indicative of mutations that existed in-vivo, by their very nature of being rare, the mutations should exhibit an associative bias with only one of the two alleles. When we call variants within these two allele-specific bins however, we find that the variants associate quite uniformly across both alleles, suggesting that much of the variant signal found within our final consensus reads is erroneous (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2977/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The background of false positive variants is similar across individuals. A) Using heterozygous SNPs to identify different alleles in human leukocyte gDNA, rare variants within consensus reads equally associate with each of the alleles suggesting they are occurring randomly ex-vivo. Axes are VAFs of variants found on corresponding alleles. B) Using FERMI to measure somatic mutation loads within two different samples shows that background signal is similar within leukocyte gDNA. Axes are VAFs of variants found within each individual. Points represent specific substitutions at each locus. C) The insertions/deletions found within three different individuals were identified. Indel counts are shown on the y-axis. Black dots represent sample groups, where the indel counts are those found within all samples indicated (either each sample alone, or commonly found across the indicated samples; for example, the next to the last vertical bar reflects indels found in both samples 3 and 2). Horizontal bars quantify the total number of indels found in a sample. D) Relative prevalence of observed substitutions within background signal found in leukocyte gDNA consensus reads. Complementary changes such as C &gt; T and G &gt; A are combined. Error is standard deviation across 20 individuals. E) Relative prevalence of observed substitutions classified by the neighboring upstream and downstream nucleotides (trinucleotide context). Error is standard deviation across 20 individuals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-678226960" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The background of false positive variants is similar across individuals. A) Using heterozygous SNPs to identify different alleles in human leukocyte gDNA, rare variants within consensus reads equally associate with each of the alleles suggesting they are occurring randomly ex-vivo. Axes are VAFs of variants found on corresponding alleles. B) Using FERMI to measure somatic mutation loads within two different samples shows that background signal is similar within leukocyte gDNA. Axes are VAFs of variants found within each individual. Points represent specific substitutions at each locus. C) The insertions/deletions found within three different individuals were identified. Indel counts are shown on the y-axis. Black dots represent sample groups, where the indel counts are those found within all samples indicated (either each sample alone, or commonly found across the indicated samples; for example, the next to the last vertical bar reflects indels found in both samples 3 and 2). Horizontal bars quantify the total number of indels found in a sample. D) Relative prevalence of observed substitutions within background signal found in leukocyte gDNA consensus reads. Complementary changes such as C &gt; T and G &gt; A are combined. Error is standard deviation across 20 individuals. E) Relative prevalence of observed substitutions classified by the neighboring upstream and downstream nucleotides (trinucleotide context). Error is standard deviation across 20 individuals.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2977/F2.medium.gif" width="440" height="278"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/9/2977/F2.medium.gif" width="440" height="278"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2977/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2977/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494185" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-32" class="first-child">The background of false positive variants is similar across individuals. A) Using heterozygous SNPs to identify different alleles in human leukocyte gDNA, rare variants within consensus reads equally associate with each of the alleles suggesting they are occurring randomly ex-vivo. Axes are VAFs of variants found on corresponding alleles. B) Using FERMI to measure somatic mutation loads within two different samples shows that background signal is similar within leukocyte gDNA. Axes are VAFs of variants found within each individual. Points represent specific substitutions at each locus. C) The insertions/deletions found within three different individuals were identified. Indel counts are shown on the y-axis. Black dots represent sample groups, where the indel counts are those found within all samples indicated (either each sample alone, or commonly found across the indicated samples; for example, the next to the last vertical bar reflects indels found in both samples 3 and 2). Horizontal bars quantify the total number of indels found in a sample. D) Relative prevalence of observed substitutions within background signal found in leukocyte gDNA consensus reads. Complementary changes such as C &gt; T and G &gt; A are combined. Error is standard deviation across 20 individuals. E) Relative prevalence of observed substitutions classified by the neighboring upstream and downstream nucleotides (trinucleotide context). Error is standard deviation across 20 individuals.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">Further suggestive of a significant false-positive presence within consensus reads, we show that when the rare variants found within the blood of any two individuals are compared, the same variants are found in each sample at nearly the same allele frequencies (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). This similarity is not limited to inter-blood sample comparisons, as sperm gDNA shows similar patterns (Supplemental Figure 1). Finally, being similar to that in blood, the somatic mutation load we observe in sperm cells is well above previous estimates of less than 100 mutations per genome (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lynch 2016</a>). Furthermore, when blood from healthy individuals was compared, mutations were no more similar in repeats from the same individual than between individuals (Supplemental Figure 4). Combined, these observations suggest that a false-positive background exists relatively uniformly across samples and sample types, and invites the possibility for a correction algorithm to distinguish real from false signal in order to significantly improve sequencing detection limits.</p><p id="p-34">While over 90% of detected background variants were substitutions, occasionally insertions and deletions (indels) were observed. As many of these indels were observed in multiple captures for the same regions, we thought they might represent real mutations. However, as shown in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>, individual samples contain roughly 500 insertions or deletions, and about 250 of these are conserved across all samples. Furthermore, when a group of 20 individuals was pooled, only one insertion was not found at least twice within the pool. As these indels are often found in multiple captures, the repetitive occurrence between individual samples suggests that some mutagenic mechanism during sample processing is responsible for indel occurrence. Because of this recurrent observation, without modification to the protocol it seems detecting indels would be quite challenging.</p><p id="p-35">Within our final consensus reads, single-nucleotide substitutions account for the majority of falsely identified variants, and within this group of variants, there is significant identity bias. We find that C &gt; T substitutions account for nearly 50% of the variants present in our final consensus reads, while other changes like C &gt; G and T &gt; G are far more rare indicative of differences in nucleotide stability or mutability (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2D</a>). Breaking down these substitutions into their trinucleotide contexts by including the bases located 5′ and 3′ of each change, we find that sequence context significantly impacts the probability of a false variant being identified (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2E</a>). Among the trinucleotide contexts, false variants within CpG sites are overrepresented within our final consensus reads.</p><p id="p-36">Importantly, the patterns we identify in the trinucleotide context-independent and context-dependent substitutions mirror those identified in other studies of both normal tissues and cancers (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alexandrov <em>et al.</em> 2013</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Martincorena <em>et al.</em> 2015</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Blokzijl <em>et al.</em> 2016</a>). The similarity of these patterns provides a cautionary note for mutation detection, as obedience to known patterns does not necessarily provide confidence in the accuracy of calls. It is possible that in-vivo mechanisms of mutation generation are similar to those experienced by template DNA ex-vivo, and therefore results in similar patterns within the background.</p></div><div id="sec-16" class="subsection"><h3>Nucleotide context insufficiently explains background signal</h3><p id="p-37">In search of common patterns within our false positive background, we looked for surrounding sequence contexts that play a role in the prevalence of a false variant. While trinucleotide context does impact the probability that a substitution is found within our final consensus read pool, it often incompletely predicts the resulting variant allele frequency (VAF). We observe that many of the background substitutions found within our final consensus reads such as C &gt; A within the contexts of CCA and ACA, exist within two relatively distinct VAF groups (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). This separation indicates that within a given trinucleotide context, a substitution such as C &gt; A will occur with either a high or a low frequency. Alternatively, some substitutions such as C &gt; A within the CCG context largely occur with a low frequency, while other changes such as T &gt; G in the context of CTA almost never exist at a high frequency. Both results suggest that trinucleotide context is not sufficient to predict background substitution rates at a given locus (Supplemental Table 1), consistent with recent reports that broader (epi)genomic contexts play key roles in replication errors, DNA damage, and repair (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Coleman and De 2018</a>). We do find, that regardless of the substitution identity or the trinucleotide context, a substitution defined only by trinucleotide context never exclusively occurs at high frequency.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2977/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Trinucleotide context is insufficient to predict VAF. A) Mean VAF of background variants calculated from two different groups of 10 individuals probed with FERMI. Variants are either classified by substitution identity alone or within a particular trinucleotide context. B) Relative substitution rates for different substitutions classified by triplet context across all probed regions. Error is standard deviation across 20 individuals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-678226960" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Trinucleotide context is insufficient to predict VAF. A) Mean VAF of background variants calculated from two different groups of 10 individuals probed with FERMI. Variants are either classified by substitution identity alone or within a particular trinucleotide context. B) Relative substitution rates for different substitutions classified by triplet context across all probed regions. Error is standard deviation across 20 individuals.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2977/F3.medium.gif" width="440" height="209"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/9/2977/F3.medium.gif" width="440" height="209"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2977/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2977/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494184" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-38" class="first-child">Trinucleotide context is insufficient to predict VAF. A) Mean VAF of background variants calculated from two different groups of 10 individuals probed with FERMI. Variants are either classified by substitution identity alone or within a particular trinucleotide context. B) Relative substitution rates for different substitutions classified by triplet context across all probed regions. Error is standard deviation across 20 individuals.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">If the background variants are separated by their presence in either the upper or the lower VAF population, we find that for some changes such as C &gt; A, both the 5′ and the 3′ nucleotides of the trinucleotide context significantly impact the VAF of the change (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). This impact of the trinucleotide context is however not present for all changes, such as C &gt; T substitutions, which show minimal bias of any of the possible trinucleotide contexts. We further searched for patterns within the 10bp upstream and downstream of a given change and find that only the triplet context showed any meaningful impact on mutation rate (Supplemental Figure 2).</p></div><div id="sec-17" class="subsection"><h3>Algorithmic background subtraction eliminates most false positive signal</h3><p id="p-40">Although the trinucleotide context alone does not provide a sufficient amount of contextual information to determine the frequency with which a background variant is observed, nucleotide position strongly impacts the VAF of a substitution within the final consensus sequences. Throughout the probed regions, each nucleotide locus shows a unique background signal pattern that is relatively conserved across individuals (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>; similar conservation of background signal is observed across all other segments, data not shown). Some of the mutational patterns we observe within our background signal are similar to those found in The Cancer Genome Atlas (TCGA) (Supplemental Figure 3). Notably, enrichment for previously defined signatures are evident in background variants, representing artifacts of damage to isolated gDNA.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2977/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Background confidence intervals eliminate most variants. A) Observed substitution VAFs for a Tier III probe, illustrating the varying mutation presence at each nucleotide locus. Error is standard deviation across 20 individuals. B) Subsets of the IDH2 probe region from two different groups of 10 individuals illustrates the degree of similarity and differences that are commonly observed between samples. Error is standard deviation across the individuals. C) Total number of variants deemed significantly above background when only triplet context was used to generate expected background substitution rates (Generic), or when position-specific substitution rates generated from a different experiment (External) or the same experiment (Internal) are used. D) The total numbers of variants called as significantly above background for 5 individuals (labeled as sample number) at confidence intervals from 0.9 - 0.999999999999999. where the number of trailing 9&#x2019;s is indicated by x-axis value. E) The total numbers of variants called as significantly above background for two leukemic patients using sample 4 from MRD1 and sample 3 from MRD2 (See Figure 6), which are both points that had followed treatment, and at which leukemic burden was low." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-678226960" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Background confidence intervals eliminate most variants. A) Observed substitution VAFs for a Tier III probe, illustrating the varying mutation presence at each nucleotide locus. Error is standard deviation across 20 individuals. B) Subsets of the IDH2 probe region from two different groups of 10 individuals illustrates the degree of similarity and differences that are commonly observed between samples. Error is standard deviation across the individuals. C) Total number of variants deemed significantly above background when only triplet context was used to generate expected background substitution rates (Generic), or when position-specific substitution rates generated from a different experiment (External) or the same experiment (Internal) are used. D) The total numbers of variants called as significantly above background for 5 individuals (labeled as sample number) at confidence intervals from 0.9 - 0.999999999999999. where the number of trailing 9&#x2019;s is indicated by x-axis value. E) The total numbers of variants called as significantly above background for two leukemic patients using sample 4 from MRD1 and sample 3 from MRD2 (See Figure 6), which are both points that had followed treatment, and at which leukemic burden was low.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2977/F4.medium.gif" width="440" height="201"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/9/2977/F4.medium.gif" width="440" height="201"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2977/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2977/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494072" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-41" class="first-child">Background confidence intervals eliminate most variants. A) Observed substitution VAFs for a Tier III probe, illustrating the varying mutation presence at each nucleotide locus. Error is standard deviation across 20 individuals. B) Subsets of the IDH2 probe region from two different groups of 10 individuals illustrates the degree of similarity and differences that are commonly observed between samples. Error is standard deviation across the individuals. C) Total number of variants deemed significantly above background when only triplet context was used to generate expected background substitution rates (Generic), or when position-specific substitution rates generated from a different experiment (External) or the same experiment (Internal) are used. D) The total numbers of variants called as significantly above background for 5 individuals (labeled as sample number) at confidence intervals from 0.9 - 0.999999999999999. where the number of trailing 9’s is indicated by x-axis value. E) The total numbers of variants called as significantly above background for two leukemic patients using sample 4 from MRD1 and sample 3 from MRD2 (See <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>), which are both points that had followed treatment, and at which leukemic burden was low.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">Within our observed backgrounds, some nucleotide loci exhibit a strong bias toward a particular base change, showing only one type of substitution across all tested individuals. This effect is most commonly observed at nucleotides that exhibit a C &gt; T substitution, where it is often the only observed change at that locus. Other nucleotide positions exhibit multiple different background substitutions, some changing to all three possible other bases. We noticed that while the background signal was surprisingly conserved across samples, that variability did exist (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). It is often the case that a particular nucleotide locus will exhibit the same types of variants across different samples, but the allele frequencies vary.</p><p id="p-43">Because each nucleotide locus tends to show a similar background across all tested individuals (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2B</a>), it was possible to derive a governing probability distribution for each observed substitution at every probed position. To create this distribution, a probability density function was created using a student’s <em>t</em> continuous random variable function. This probability density function was then used to calculate the high and low VAF endpoints of a confidence range by using a specified alpha fraction of the distribution.</p><p id="p-44">We compared a number of different types of backgrounds to understand which best allowed us to eliminate false variants (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>). Initially a generic background was created by deriving a probability density function for each type of nucleotide change based on its neighbors (the three nucleotide patterns shown in <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2D</a>), independent of genomic position. This approach was modestly effective at eliminating background variants from samples, as it reduced the total variant calls by about 80% (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4C</a>, “Generic”). By incorporating positional information and deriving a density function for each observed substitution at all nucleotide positions, about 99.9% of variant calls were eliminated, providing very clean sequencing data. Importantly, experimental variability seems to play a significant role in the accuracy of the background. While the same sample sequenced across multiple experiments generally shows a very similar background, experimental variability does appear (Supplemental Figure 4). While a background derived from samples taken from a different experiment (“External”) will result in about 10 variants being called as real in a given peripheral blood sample, a background created from samples run in the same experiment (“Internal”) will result in about 1-3 variants being called as real (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4C</a>). As expected variants are always retained, but the total number of significant variants is minimized when using an internal background created from samples of the same experiment, internal backgrounds are used for all subsequent analyses.</p><p id="p-45">To understand what alpha fraction of the probability density functions should be called as <em>bona fide</em> mutations, we used 10 healthy blood samples to derive a confidence interval range for each observed substitution across all probed nucleotide positions. These confidence intervals were then compiled into a comprehensive false-positive background against which experimental samples were then compared.</p><p id="p-46">For 5 healthy blood samples, variants were called from their derived consensus sequences, and then compared against the comprehensive background. Within these 5 samples, variants were called as confidently above background if their VAFs were high enough to fall within the specific alpha of their governing probability density function. As expected, as the confidence interval alpha fraction was increased from 0.9 (One 9) to 0.999999999999999 (Fifteen 9’s) the number of variants called as confidently above background exponentially decreases (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4D</a>). This method eliminates nearly all background signal by confidence interval alpha fractions in the range of ten 9’s. Furthermore, at higher confidence intervals even germline variants are often eliminated for being too close to background, indicating excessive stringency. Peripheral samples taken from leukemic patients at different points during therapy were also tested, and show similar exponential decreases in confident variant calls, though the overall numbers of variants are higher than in healthy blood (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4E</a>).</p></div><div id="sec-18" class="subsection"><h3>Assessing FERMI sensitivity and specificity</h3><p id="p-47">To help understand the specificity of FERMI in detecting only true mutations, Molm13 acute myeloid leukemia cells were expanded <em>in vitro</em> after passing them through bottlenecks of 1, 100, or 1 million cells. We show that many mutations can be observed within the cell cultures started from 100 cells, given that the 100-cell bottleneck should create clones at approximately 1% frequency each with occasional variants in our probed region (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). Heterozygous mutations are expected at allele frequencies of 0.005 at 2N loci, and at lower VAFs if a variant falls in a region with greater ploidy. Indeed, most variants fall within this range. As expected, very few mutations are detected in gDNA isolated from the cell cultures started from 1 million cells, as most mutations will exist at rare allele frequencies (below our limit of detection). Similarly, we observed no mutations within the cultures initiated with single cells, consistent with the low odds that a mutation would occur in our probed region during the ∼14 cell divisions required to generate the 10,000 cell limit of detection. The absence of mutations detected in the cultures initiated with 1 cell each, but the presence of mutations within the cultures initiated with 100 cells, indicates that we have sufficiently limited false positive variants, but retained true mutations.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2977/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Assessing FERMI specificity and accuracy. A) MOLM-13 cells grown from 1, 100, or 1 million initial cells were expanded to a pool of 1 million cells, and then probed with FERMI. Samples illustrate how clonality impacts mutation detection by FERMI by altering the VAFs of somatic variants. B) Observed frequencies of a serially diluted blood gDNA sample with a heterozygous germline SNP show successful detection at allele frequencies from 1/2 to 1/10,000 (legend indicates the dilutions not the allele frequencies, where a 1/5,000 dilution of a heterozygous mutation should result in a 1/10,000 allele frequency). Background signal mean and standard deviation shown in red and purple respectively, calculated from 12 samples. C) Limit of detection improvements observed when multiple mutations in linkage disequilibrium are leveraged to eliminate erroneous reads. Background signal calculated from 12 samples." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-678226960" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Assessing FERMI specificity and accuracy. A) MOLM-13 cells grown from 1, 100, or 1 million initial cells were expanded to a pool of 1 million cells, and then probed with FERMI. Samples illustrate how clonality impacts mutation detection by FERMI by altering the VAFs of somatic variants. B) Observed frequencies of a serially diluted blood gDNA sample with a heterozygous germline SNP show successful detection at allele frequencies from 1/2 to 1/10,000 (legend indicates the dilutions not the allele frequencies, where a 1/5,000 dilution of a heterozygous mutation should result in a 1/10,000 allele frequency). Background signal mean and standard deviation shown in red and purple respectively, calculated from 12 samples. C) Limit of detection improvements observed when multiple mutations in linkage disequilibrium are leveraged to eliminate erroneous reads. Background signal calculated from 12 samples.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2977/F5.medium.gif" width="440" height="389"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/9/2977/F5.medium.gif" width="440" height="389"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2977/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2977/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494171" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-48" class="first-child">Assessing FERMI specificity and accuracy. A) MOLM-13 cells grown from 1, 100, or 1 million initial cells were expanded to a pool of 1 million cells, and then probed with FERMI. Samples illustrate how clonality impacts mutation detection by FERMI by altering the VAFs of somatic variants. B) Observed frequencies of a serially diluted blood gDNA sample with a heterozygous germline SNP show successful detection at allele frequencies from 1/2 to 1/10,000 (legend indicates the dilutions not the allele frequencies, where a 1/5,000 dilution of a heterozygous mutation should result in a 1/10,000 allele frequency). Background signal mean and standard deviation shown in red and purple respectively, calculated from 12 samples. C) Limit of detection improvements observed when multiple mutations in linkage disequilibrium are leveraged to eliminate erroneous reads. Background signal calculated from 12 samples.</p><div class="sb-div caption-clear"></div></div></div><p id="p-49">To assess the sensitivity and limit of detection of FERMI, gDNA from human blood containing known heterozygous SNPs was serially diluted into blood gDNA lacking these SNPs. We find that the detection limits of tracking single dilutions to be variable as the level of background noise is position specific, but diluted germline variants were detected at frequencies at least as rare as 1:10,000 at the expected VAFs (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). In other positions, where the background can be much higher, a dilution series would not be detected as low as 1:10,000.</p><p id="p-50">One of the samples tested in the dilution series contained three heterozygous SNPs within the same probe region, on the same allele, allowing for an extra level of error correction. Within this sample, it was assumed that only those consensus reads with all three SNPs or those without any of the SNPs were correct, and all other reads were eliminated. This analysis significantly reduced the background error rates at the SNP positions, and allowed detection of diluted mutations at least as low as 1:10,000 (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a>).</p></div><div id="sec-19" class="subsection"><h3>Oncogenic driver detection in leukemias</h3><p id="p-51">To test the ability of FERMI to detect and follow mutations throughout leukemia treatments, patient biopsies were collected at disease inception, throughout treatment, and during relapse when possible. Using FERMI, we are able to detect these mutations when they are present and observe clonal evolution as it occurs. In some cases, we detect the principle oncogenic driver and watch it fluctuate in frequency in response to treatment without ever disappearing below background (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6A</a>). In another case, a JAK2 mutation is initially observed at high frequency, but treatment eliminates the clone. As relapse occurs, blast counts increase (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6, D–F</a>), but the initial JAK2 clone does not increase in frequency, as the genetics of the leukemia has clearly changed with treatment (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6B</a>). In a third sample, we observe a patient relapse with a previously undetected JAK2 mutation (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6C</a>). While this time point was taken at relapse, we detect it at a frequency significantly below that of most sequencing method sensitivities, requiring only 5 ml of peripheral blood. The early detection of such a clone could allow treatment with a kinase inhibitor before overt disease relapse.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2977/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Oncogenic mutation detection during leukemia treatments. A) Oncogenic driver detection using FERMI on bone marrow biopsies taken at 6 different timepoints starting with clinical presentation of the patient and ending with relapse (x-axis is in chronological order of leukemia samplings). Background signal mean and standard deviation shown in red and purple respectively, and is derived from 20 samples. B) Oncogenic driver detection throughout leukemia treatment in a case where relapse was not driven by a mutation within our panel. Background derived from 20 samples. 27. C) Example of a leukemic relapse in which the detected driver was not unknown prior to blood sampling. Background derived from 8 samples. D,E,F) Corresponding blast counts as percentage of bone marrow biopsies." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-678226960" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Oncogenic mutation detection during leukemia treatments. A) Oncogenic driver detection using FERMI on bone marrow biopsies taken at 6 different timepoints starting with clinical presentation of the patient and ending with relapse (x-axis is in chronological order of leukemia samplings). Background signal mean and standard deviation shown in red and purple respectively, and is derived from 20 samples. B) Oncogenic driver detection throughout leukemia treatment in a case where relapse was not driven by a mutation within our panel. Background derived from 20 samples. 27. C) Example of a leukemic relapse in which the detected driver was not unknown prior to blood sampling. Background derived from 8 samples. D,E,F) Corresponding blast counts as percentage of bone marrow biopsies.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2977/F6.medium.gif" width="440" height="236"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/9/2977/F6.medium.gif" width="440" height="236"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2977/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2977/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494091" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-52" class="first-child">Oncogenic mutation detection during leukemia treatments. A) Oncogenic driver detection using FERMI on bone marrow biopsies taken at 6 different timepoints starting with clinical presentation of the patient and ending with relapse (x-axis is in chronological order of leukemia samplings). Background signal mean and standard deviation shown in red and purple respectively, and is derived from 20 samples. B) Oncogenic driver detection throughout leukemia treatment in a case where relapse was not driven by a mutation within our panel. Background derived from 20 samples. 27. C) Example of a leukemic relapse in which the detected driver was not unknown prior to blood sampling. Background derived from 8 samples. D,E,F) Corresponding blast counts as percentage of bone marrow biopsies.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-20"><h2 class="">Discussion</h2><p id="p-53">In this study, we designed a sensitive sequencing method that enables the accurate detection of rare variants and clonal evolution within primary samples. In leveraging the quantitative power of capture-unique UMI barcodes, we achieve single-allele sequencing resolution from gDNA, and by then combining these sequencing results with a comprehensive analysis of expected background signal, we achieve exceptional sequencing fidelity.</p><p id="p-54">While capture-specific barcoding has been effectively used in the past, an inability to achieve sufficient capture numbers and high background have often held the theoretical limit of detection to variants existing at a frequency of &gt;1/1,000. By probing roughly 30,000 different unique captures for each region of interest per sample, we pushed our theoretical limit of detection to at least 1/10,000. This limit is governed by the magnitude of false positive background observed at a particular location, as variants are difficult to identify when they exist at allele frequencies below the background signal.</p><p id="p-55">Paired-end collapsing has been successfully used to reduce the number of sequencing errors within sequencing data. Unfortunately, errors also occur during library-preparation, and while the molecular barcodes assist with the elimination of these library-preparation errors, mistakes made before or during the first round of PCR will typically appear indistinguishable from a heterozygous variant, such that neither paired-end collapsing nor molecular barcode collapsing will be capable of eliminating them. This understanding prompted the development of an expected false positive background that could be used to filter out common mistakes that occur during library preparation.</p><p id="p-56">Our experimentally derived backgrounds proved vitally important in determining whether or not a variant found in a sample was sufficiently elevated in allele frequency that it could be classified as truly existing in the in-vivo gDNA from which it originated. Similar to background subtraction employed by other groups (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Newman <em>et al.</em> 2016</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chaudhuri <em>et al.</em> 2017</a>), our generalized background allowed us to not only detect expected mutations, but also discover new mutations within samples.</p><p id="p-57">It is interesting to note that within our correction background, we observe similar mutation patterns to those observed for other studies, and even similar mutational signatures (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Alexandrov <em>et al.</em> 2013</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Behjati <em>et al.</em> 2014</a>). This reproducibility may indicate a surprising degree of similarity between the intrinsic mutagenic processes in-vivo and error-causing processes involved in sample preparation. It is possible that this similarity is the result of the conserved behavior of error-inducing machinery like DNA polymerase and DNA ligase both in-vivo and in-vitro, or even similar mutagenic exposures such as oxidative damage. These observed similarities suggest caution against using mutation signatures as validation of the accuracy of sequencing data when attempting to identify rare variants.</p><p id="p-58">The early detection of clonal evolution within cancer samples has held the promise of more comprehensive diagnoses and improved treatment strategies for patients. While deep sequencing has been applied to patient leukemias in the past, mutation discovery accuracies have typically limited these approaches to more of a validation role (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Thol <em>et al.</em> 2018</a>). We obtained a number of leukemic patient biopsies, taken at initial clinical presentation, and throughout treatment, and used FERMI to search for somatic mutations. We find that while the detection limit of FERMI is quite low, the greatest improvements are made through its accurate mutation detection ability. Because we eliminate nearly all background variants, we can accurately detect unexpected relapse mutations and drivers of clonal expansions.</p><p id="p-59">In requiring only around 5ml of blood, FERMI could be easily used in a clinical setting to quickly, cheaply and easily identify important driver mutations and clonal evolution within patient’s cancers. If relapse mutations were caught by FERMI when they are still rare, targeted therapies could be used to prevent them from clonally expanding to fixation and driving leukemic relapse. Additional applications of FERMI could include analyses of mutational patterns in normal epithelial tissues, premalignancies and carcinomas obtained through direct biopsies or via detection in blood.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-60">We would like to thank Ruth Hershberg of Technion University and Jay Hesselberth and Robert Sclafani of the University of Colorado School of Medicine for useful suggestions and for review of the manuscript. We would like to thank Craig Jordan and Amanda Winters of the University of Colorado School of Medicine for the primary leukemia samples. We would like to thank the Illumina Concierge team who assisted with the initial probe design. We would also like to thank Joe Hiatt, Beth Martin, and Jay Shendure of the Shendure lab, who assisted in the initial design of the probing technique. These studies were supported by grants from the National Cancer Institute (R01CA180175 to J.D.), NIH/NCATS Colorado CTSI Grant Number UL1TR001082CU (seed grant to J.D.), a SCOR grant from the Leukemia and Lymphoma Society (to Craig Jordan), F31CA196231 (to L.A.L.), the Linda Crnic Institute for Down Syndrome (to J.D. and L.A.L.), P30-CA072720 (to A.S. and S.D.), and R01-GM129066, P30-CA072720, and Robert Wood Johnson Foundation (to A.S. and S.D.). The research utilized services of the Cancer Center Genomics Shared Resource, which is supported in part by NIH grant P30-CA46934. L.A.L. and J.D. developed the concept of this project, planned the experiments, analyzed results, and wrote the manuscript. L.A.L. processed and prepared samples from blood biopsy to sequencing, and wrote the software used for analysis. A.S. and S.D. analyzed results, and contributed to writing of the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-61">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.9037457">https://doi.org/10.25387/g3.9037457</a>.</p></li><li class="fn" id="fn-2"><p id="p-62"><em>Communicating editor: M. Boutros</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-06-12"><span class="received-label">Received </span>June 12, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-07-14"><span class="accepted-label">Accepted </span>July 14, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Liggett <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.1" data-doi="10.3791/55130"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albitar</span>,  <span class="cit-name-given-names">A. Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Ma</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Albitar</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Wild-type Blocking PCR Combined with Direct Sequencing as a Highly Sensitive Method for Detection of Low-Frequency Somatic Mutations.</span> <abbr class="cit-jnl-abbrev">J. Vis. Exp.</abbr> (<span class="cit-issue">121</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3791/55130</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVis.%2BExp.%26rft_id%253Dinfo%253Adoi%252F10.3791%252F55130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3791/55130&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.2" data-doi="10.1038/nature12477"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexandrov</span>,  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Nik-Zainal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. C.</span> <span class="cit-name-surname">Wedge</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A. J. R.</span> <span class="cit-name-surname">Aparicio</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Behjati</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Signatures of mutational processes in human cancer.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">500</span>: <span class="cit-fpage">415</span>–<span class="cit-lpage">421</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature12477</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> (erratum: Nature 502: 258).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D500%26rft.spage%253D415%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12477%26rft_id%253Dinfo%253Apmid%252F23945592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12477&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23945592&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2977.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000323316100026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.3" data-doi="10.1038/nature13448"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Behjati</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Huch</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">van Boxtel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Karthaus</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. C.</span> <span class="cit-name-surname">Wedge</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome sequencing of normal cells reveals developmental lineages and mutational processes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">513</span>: <span class="cit-fpage">422</span>–<span class="cit-lpage">425</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature13448</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D513%26rft.spage%253D422%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13448%26rft_id%253Dinfo%253Apmid%252F25043003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13448&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25043003&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2977.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000341814900061&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.4" data-doi="10.1073/pnas.47.3.403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benzer</span>,  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1961</span> <span class="cit-article-title">on the topography of the genetic fine structure.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">415</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.47.3.403</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DBenzer%26rft.auinit1%253DS.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D415%26rft.atitle%253DON%2BTHE%2BTOPOGRAPHY%2BOF%2BTHE%2BGENETIC%2BFINE%2BSTRUCTURE%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.47.3.403%26rft_id%253Dinfo%253Apmid%252F16590840%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI0Ny8zLzQwMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOS8yOTc3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.5" data-doi="10.1038/nature19768"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blokzijl</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">de Ligt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Jager</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Sasselli</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Roerink</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Tissue-specific mutation accumulation in human adult stem cells during life.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">538</span>: <span class="cit-fpage">260</span>–<span class="cit-lpage">264</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature19768</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D538%26rft.spage%253D260%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature19768%26rft_id%253Dinfo%253Apmid%252F27698416%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature19768&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27698416&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2977.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.6" data-doi="10.1158/2159-8290.CD-17-0716"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chaudhuri</span>,  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Chabon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. F.</span> <span class="cit-name-surname">Lovejoy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Newman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Stehr</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Early Detection of Molecular Residual Disease in Localized Lung Cancer by Circulating Tumor DNA Profiling.</span> <abbr class="cit-jnl-abbrev">Cancer Discov.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1394</span>–<span class="cit-lpage">1403</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1158/2159-8290.CD-17-0716</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BDiscov.%26rft_id%253Dinfo%253Adoi%252F10.1158%252F2159-8290.CD-17-0716%26rft_id%253Dinfo%253Apmid%252F28899864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiY2FuZGlzYyI7czo1OiJyZXNpZCI7czo5OiI3LzEyLzEzOTQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjk3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. S.</span> <span class="cit-name-surname">Cahill</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Kasai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Nishimura</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. A.</span> <span class="cit-name-surname">Loeb</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">8-Hydroxyguanine, an abundant form of oxidative DNA damage, causes G—-T and A—-C substitutions.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">267</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DCheng%26rft.auinit1%253DK.%2BC.%26rft.volume%253D267%26rft.issue%253D1%26rft.spage%253D166%26rft.epage%253D172%26rft.atitle%253D8-Hydroxyguanine%252C%2Ban%2Babundant%2Bform%2Bof%2Boxidative%2BDNA%2Bdamage%252C%2Bcauses%2BG----%2BT%2Band%2BA----C%2Bsubstitutions%26rft_id%253Dinfo%253Apmid%252F1730583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjk6IjI2Ny8xLzE2NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOS8yOTc3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.8" data-doi="10.1016/j.trecan.2018.08.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coleman</span>,  <span class="cit-name-given-names">N.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">De</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Mutation Signatures Depend on Epigenomic Contexts.</span> <abbr class="cit-jnl-abbrev">Trends Cancer Res.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">659</span>–<span class="cit-lpage">661</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.trecan.2018.08.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BCancer%2BRes.%26rft.volume%253D4%26rft.spage%253D659%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.trecan.2018.08.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.trecan.2018.08.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.9" data-doi="10.1073/pnas.1133470100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dressman</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Yan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Traverso</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. W.</span> <span class="cit-name-surname">Kinzler</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Vogelstein</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Transforming single DNA molecules into fluorescent magnetic particles for detection and enumeration of genetic variations.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">8817</span>–<span class="cit-lpage">8822</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1133470100</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1133470100%26rft_id%253Dinfo%253Apmid%252F12857956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAwLzE1Lzg4MTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjk3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.10" data-doi="10.1093/nar/gkr861"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flaherty</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Natsoulis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Muralidharan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Winters</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Buenrostro</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Ultrasensitive detection of rare mutations using next-generation targeted resequencing.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-elocation-id">e2</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkr861</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr861%26rft_id%253Dinfo%253Apmid%252F22013163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr861&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22013163&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2977.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.11" data-doi="10.1093/nar/gkq963"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fujita</span>,  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Rhead</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. S.</span> <span class="cit-name-surname">Zweig</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. S.</span> <span class="cit-name-surname">Hinrichs</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Karolchik</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The UCSC genome browser database: update 2011.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">D876</span>–<span class="cit-lpage">D882</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkq963</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq963%26rft_id%253Dinfo%253Apmid%252F20959295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq963&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20959295&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2977.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285831700137&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.12" data-doi="10.1101/gr.3895005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaffney</span>,  <span class="cit-name-given-names">D. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. D.</span> <span class="cit-name-surname">Keightley</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The scale of mutational variation in the murid genome.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1086</span>–<span class="cit-lpage">1094</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.3895005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3895005%26rft_id%253Dinfo%253Apmid%252F16024822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE1LzgvMTA4NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOS8yOTc3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.13"><div class="cit-metadata unstructured">Garrison, E., and G. Marth, 2012 Haplotype-based variant detection from short-read sequencing. arXiv [q-bio.GN].</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.14" data-doi="10.1038/nature10762"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Greaves</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Maley</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Clonal evolution in cancer.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">481</span>: <span class="cit-fpage">306</span>–<span class="cit-lpage">313</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature10762</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGreaves%26rft.auinit1%253DM.%26rft.volume%253D481%26rft.issue%253D7381%26rft.spage%253D306%26rft.epage%253D313%26rft.atitle%253DClonal%2Bevolution%2Bin%2Bcancer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10762%26rft_id%253Dinfo%253Apmid%252F22258609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10762&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22258609&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2977.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299210600032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.15" data-doi="10.1101/gr.147686.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hiatt</span>,  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Pritchard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Salipante</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">O’Roak</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Shendure</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Single molecule molecular inversion probes for targeted, high-accuracy detection of low-frequency variation.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">843</span>–<span class="cit-lpage">854</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.147686.112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.147686.112%26rft_id%253Dinfo%253Apmid%252F23382536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIzLzUvODQzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI5NzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.16" data-doi="10.1021/ac202028g"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hindson</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. D.</span> <span class="cit-name-surname">Ness</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Masquelier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Belgrader</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. J.</span> <span class="cit-name-surname">Heredia</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">High-throughput droplet digital PCR system for absolute quantitation of DNA copy number.</span> <abbr class="cit-jnl-abbrev">Anal. Chem.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">8604</span>–<span class="cit-lpage">8610</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1021/ac202028g</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnalytical%2BChemistry%2B%2528Washington%252C%2BDC%2529%26rft.stitle%253DAnalytical%2BChemistry%2B%2528Washington%252C%2BDC%2529%26rft.aulast%253DHindson%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D83%26rft.issue%253D22%26rft.spage%253D8604%26rft.epage%253D8610%26rft.atitle%253DHigh-throughput%2Bdroplet%2Bdigital%2BPCR%2Bsystem%2Bfor%2Babsolute%2Bquantitation%2Bof%2BDNA%2Bcopy%2Bnumber.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fac202028g%26rft_id%253Dinfo%253Apmid%252F22035192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ac202028g&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22035192&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2977.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.17" data-doi="10.1073/pnas.0404142101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hwang</span>,  <span class="cit-name-given-names">D. G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Green</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Bayesian Markov chain Monte Carlo sequence analysis reveals varying neutral substitution patterns in mammalian evolution.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">13994</span>–<span class="cit-lpage">14001</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0404142101</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0404142101%26rft_id%253Dinfo%253Apmid%252F15292512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzM5LzEzOTk0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI5NzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.18" data-doi="10.1056/NEJMoa1507471"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ivey</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. K.</span> <span class="cit-name-surname">Hills</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Simpson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. V.</span> <span class="cit-name-surname">Jovanovic</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Gilkes</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Assessment of Minimal Residual Disease in Standard-Risk AML.</span> <abbr class="cit-jnl-abbrev">N. Engl. J. Med.</abbr> <span class="cit-vol">374</span>: <span class="cit-fpage">422</span>–<span class="cit-lpage">433</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1056/NEJMoa1507471</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BEngl.%2BJ.%2BMed.%26rft.volume%253D374%26rft.spage%253D422%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1507471%26rft_id%253Dinfo%253Apmid%252F26789727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa1507471&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26789727&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2977.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.19" data-doi="10.1038/nprot.2014.170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kennedy</span>,  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Schmitt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. J.</span> <span class="cit-name-surname">Fox</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. F.</span> <span class="cit-name-surname">Kohrn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Salk</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Detecting ultralow-frequency mutations by Duplex Sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">2586</span>–<span class="cit-lpage">2606</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nprot.2014.170</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> (erratum: Nat. Protoc. 9: 2903).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BProtoc.%26rft.volume%253D9%26rft.spage%253D2586%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2014.170%26rft_id%253Dinfo%253Apmid%252F25299156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2014.170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25299156&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2977.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.20" data-doi="10.1186/gb-2013-14-8-r90"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Jeong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Bhutani</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Lee</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Patel</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Virmid: accurate detection of somatic mutations with sample impurity inference.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R90</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2013-14-8-r90</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR90%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-8-r90%26rft_id%253Dinfo%253Apmid%252F23987214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-8-r90&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23987214&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2977.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.21" data-doi="10.1200/JCO.2011.35.0371"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krönke</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. F.</span> <span class="cit-name-surname">Schlenk</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.-O.</span> <span class="cit-name-surname">Jensen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Tschürtz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Corbacioglu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Monitoring of minimal residual disease in NPM1-mutated acute myeloid leukemia: a study from the German-Austrian acute myeloid leukemia study group.</span> <abbr class="cit-jnl-abbrev">J. Clin. Oncol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">2709</span>–<span class="cit-lpage">2716</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1200/JCO.2011.35.0371</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BClin.%2BOncol.%26rft_id%253Dinfo%253Adoi%252F10.1200%252FJCO.2011.35.0371%26rft_id%253Dinfo%253Apmid%252F21555683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNvIjtzOjU6InJlc2lkIjtzOjEwOiIyOS8xOS8yNzA5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI5NzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.22" data-doi="10.1038/35057062"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>,  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. M.</span> <span class="cit-name-surname">Linton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Birren</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Nusbaum</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Zody</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Initial sequencing and analysis of the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">409</span>: <span class="cit-fpage">860</span>–<span class="cit-lpage">921</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/35057062</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D409%26rft.issue%253D6822%26rft.spage%253D860%26rft.epage%253D921%26rft.atitle%253DInitial%2Bsequencing%2Band%2Banalysis%2Bof%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35057062%26rft_id%253Dinfo%253Apmid%252F11237011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35057062&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11237011&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2977.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166938800058&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.23" data-doi="10.1093/oxfordjournals.molbev.a003744"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lercher</span>,  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. J. B.</span> <span class="cit-name-surname">Williams</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. D.</span> <span class="cit-name-surname">Hurst</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Local similarity in evolutionary rates extends over whole chromosomes in human-rodent and mouse-rat comparisons: implications for understanding the mechanistic basis of the male mutation bias.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">2032</span>–<span class="cit-lpage">2039</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/oxfordjournals.molbev.a003744</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DLercher%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D18%26rft.issue%253D11%26rft.spage%253D2032%26rft.epage%253D2039%26rft.atitle%253DLocal%2BSimilarity%2Bin%2BEvolutionary%2BRates%2BExtends%2Bover%2BWhole%2BChromosomes%2Bin%2BHuman-Rodent%2Band%2BMouse-Rat%2BComparisons%253A%2BImplications%2Bfor%2BUnderstanding%2Bthe%2BMechanistic%2BBasis%2Bof%2Bthe%2BMale%2BMutation%2BBias%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a003744%26rft_id%253Dinfo%253Apmid%252F11606699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a003744&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11606699&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2977.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171998900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.24" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Durbin</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows–Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2977.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.25" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Handsaker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wysoker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Fennell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ruan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2977.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.26" data-doi="10.1021/bi00749a020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lindahl</span>,  <span class="cit-name-given-names">T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Karlstrom</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Heat-induced depyrimidination of deoxyribonucleic acid in neutral solution.</span> <abbr class="cit-jnl-abbrev">Biochemistry</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">5151</span>–<span class="cit-lpage">5154</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1021/bi00749a020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D25%26rft.spage%253D5151%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi00749a020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi00749a020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.27" data-doi="10.1021/bi00713a035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lindahl</span>,  <span class="cit-name-given-names">T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Nyberg</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">Heat-induced deamination of cytosine residues in deoxyribonucleic acid.</span> <abbr class="cit-jnl-abbrev">Biochemistry</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">3405</span>–<span class="cit-lpage">3410</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1021/bi00713a035</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%2B%2528Easton%2529%26rft.stitle%253DBiochemistry%2B%2528Easton%2529%26rft.aulast%253DLindahl%26rft.auinit1%253DT.%26rft.volume%253D13%26rft.issue%253D16%26rft.spage%253D3405%26rft.epage%253D3410%26rft.atitle%253DHeat-induced%2Bdeamination%2Bof%2Bcytosine%2Bresidues%2Bin%2Bdeoxyribonucleic%2Bacid.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi00713a035%26rft_id%253Dinfo%253Apmid%252F4601435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi00713a035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4601435&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2977.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.28" data-doi="10.1038/nm1708"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Mamon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. H.</span> <span class="cit-name-surname">Kulke</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Berbeco</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Replacing PCR with COLD-PCR enriches variant DNA sequences and redefines the sensitivity of genetic testing.</span> <abbr class="cit-jnl-abbrev">Nat. Med.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">579</span>–<span class="cit-lpage">584</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nm1708</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bmedicine%26rft.stitle%253DNat%2BMed%26rft.aulast%253DLi%26rft.auinit1%253DJ.%26rft.volume%253D14%26rft.issue%253D5%26rft.spage%253D579%26rft.epage%253D584%26rft.atitle%253DReplacing%2BPCR%2Bwith%2BCOLD-PCR%2Benriches%2Bvariant%2BDNA%2Bsequences%2Band%2Bredefines%2Bthe%2Bsensitivity%2Bof%2Bgenetic%2Btesting.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm1708%26rft_id%253Dinfo%253Apmid%252F18408729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nm1708&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18408729&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2977.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255681800040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.29" data-doi="10.1534/genetics.115.180471"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Mutation and Human Exceptionalism: Our Future Genetic Load.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">869</span>–<span class="cit-lpage">875</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.180471</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D202%26rft.issue%253D3%26rft.spage%253D869%26rft.epage%253D875%26rft.atitle%253DMutation%2Band%2BHuman%2BExceptionalism%253A%2BOur%2BFuture%2BGenetic%2BLoad%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.180471%26rft_id%253Dinfo%253Apmid%252F26953265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAyLzMvODY5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI5NzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.30" data-doi="10.1373/clinchem.2018.289629"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mansukhani</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. J.</span> <span class="cit-name-surname">Barber</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Kleftogiannis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. Y.</span> <span class="cit-name-surname">Moorcraft</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Davidson</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Ultra-Sensitive Mutation Detection and Genome-Wide DNA Copy Number Reconstruction by Error-Corrected Circulating Tumor DNA Sequencing.</span> <abbr class="cit-jnl-abbrev">Clin. Chem.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">1626</span>–<span class="cit-lpage">1635</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1373/clinchem.2018.289629</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1373%252Fclinchem.2018.289629%26rft_id%253Dinfo%253Apmid%252F30150316%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiY2xpbmNoZW0iO3M6NToicmVzaWQiO3M6MTA6IjY0LzExLzE2MjYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjk3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.31" data-doi="10.1126/science.aaa6806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martincorena</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Roshan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Gerstung</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Ellis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Van Loo</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Tumor evolution. High burden and pervasive positive selection of somatic mutations in normal human skin.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">348</span>: <span class="cit-fpage">880</span>–<span class="cit-lpage">886</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.aaa6806</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMartincorena%26rft.auinit1%253DI.%26rft.volume%253D348%26rft.issue%253D6237%26rft.spage%253D880%26rft.epage%253D886%26rft.atitle%253DHigh%2Bburden%2Band%2Bpervasive%2Bpositive%2Bselection%2Bof%2Bsomatic%2Bmutations%2Bin%2Bnormal%2Bhuman%2Bskin%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaa6806%26rft_id%253Dinfo%253Apmid%252F25999502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDgvNjIzNy84ODAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjk3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.32" data-doi="10.1373/clinchem.2011.176198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Milbury</span>,  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Correll</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Quackenbush</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Rubio</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. M.</span> <span class="cit-name-surname">Makrigiorgos</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">COLD-PCR enrichment of rare cancer mutations prior to targeted amplicon resequencing.</span> <abbr class="cit-jnl-abbrev">Clin. Chem.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">580</span>–<span class="cit-lpage">589</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1373/clinchem.2011.176198</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1373%252Fclinchem.2011.176198%26rft_id%253Dinfo%253Apmid%252F22194627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiY2xpbmNoZW0iO3M6NToicmVzaWQiO3M6ODoiNTgvMy81ODAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjk3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nachman</span>,  <span class="cit-name-given-names">M. W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Crowell</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Estimate of the mutation rate per nucleotide in humans.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">297</span>–<span class="cit-lpage">304</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNachman%26rft.auinit1%253DM.%2BW.%26rft.volume%253D156%26rft.issue%253D1%26rft.spage%253D297%26rft.epage%253D304%26rft.atitle%253DEstimate%2Bof%2Bthe%2BMutation%2BRate%2Bper%2BNucleotide%2Bin%2BHumans%26rft_id%253Dinfo%253Apmid%252F10978293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU2LzEvMjk3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI5NzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.34" data-doi="10.1038/nbt.3520"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Newman</span>,  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. F.</span> <span class="cit-name-surname">Lovejoy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Klass</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Kurtz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Chabon</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Integrated digital error suppression for improved detection of circulating tumor DNA.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">547</span>–<span class="cit-lpage">555</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.3520</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D34%26rft.spage%253D547%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3520%26rft_id%253Dinfo%253Apmid%252F27018799%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3520&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27018799&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2977.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.35" data-doi="10.1109/MCSE.2007.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oliphant</span>,  <span class="cit-name-given-names">T. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Python for Scientific Computing.</span> <abbr class="cit-jnl-abbrev">Comput. Sci. Eng.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">10</span>–<span class="cit-lpage">20</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1109/MCSE.2007.58</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BSci.%2BEng.%26rft.volume%253D9%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.1109%252FMCSE.2007.58%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/MCSE.2007.58&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.36" data-doi="10.3324/haematol.2018.194712"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Onecha</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Linares</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Rapado</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Ruiz-Heredia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Martinez-Sanchez</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">A Novel deep targeted sequencing method for minimal residual disease monitoring in acute myeloid leukemia.</span> <abbr class="cit-jnl-abbrev">Haematologica</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">288</span>–<span class="cit-lpage">296</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3324/haematol.2018.194712</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHaematologica%26rft_id%253Dinfo%253Adoi%252F10.3324%252Fhaematol.2018.194712%26rft_id%253Dinfo%253Apmid%252F30093399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiaGFlbWF0b2wiO3M6NToicmVzaWQiO3M6OToiMTA0LzIvMjg4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI5NzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.37" data-doi="10.1186/s12864-016-2669-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Preston</span>,  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Royall</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Randel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. L.</span> <span class="cit-name-surname">Sikkink</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. C.</span> <span class="cit-name-surname">Phillips</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">High-specificity detection of rare alleles with Paired-End Low Error Sequencing (PELE-Seq).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">464</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-016-2669-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D464%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-016-2669-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-016-2669-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.38" data-doi="10.1038/nmeth.3351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmitt</span>,  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. J.</span> <span class="cit-name-surname">Fox</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Prindle</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. S.</span> <span class="cit-name-surname">Reid-Bayliss</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. D.</span> <span class="cit-name-surname">True</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Sequencing small genomic targets with high efficiency and extreme accuracy.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">425</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.3351</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D12%26rft.spage%253D423%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3351%26rft_id%253Dinfo%253Apmid%252F25849638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25849638&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2977.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.39" data-doi="10.1038/349431a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shibutani</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Takeshita</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Grollman</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Insertion of specific bases during DNA synthesis past the oxidation-damaged base 8-oxodG.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">349</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">434</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/349431a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DShibutani%26rft.auinit1%253DS.%26rft.volume%253D349%26rft.issue%253D6308%26rft.spage%253D431%26rft.epage%253D434%26rft.atitle%253DInsertion%2Bof%2Bspecific%2Bbases%2Bduring%2BDNA%2Bsynthesis%2Bpast%2Bthe%2Boxidation-damaged%2Bbase%2B8-oxodG.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F349431a0%26rft_id%253Dinfo%253Apmid%252F1992344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/349431a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1992344&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2977.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991EV51400057&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sykes</span>,  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. H.</span> <span class="cit-name-surname">Neoh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Brisco</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Hughes</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Condon</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Quantitation of targets for PCR by use of limiting dilution.</span> <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">444</span>–<span class="cit-lpage">449</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioTechniques%26rft.stitle%253DBiotechniques%26rft.aulast%253DSykes%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D13%26rft.issue%253D3%26rft.spage%253D444%26rft.epage%253D449%26rft.atitle%253DQuantitation%2Bof%2Btargets%2Bfor%2BPCR%2Bby%2Buse%2Bof%2Blimiting%2Bdilution.%26rft_id%253Dinfo%253Apmid%252F1389177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=1389177&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2977.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JM74700022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.41" data-doi="10.1093/bioinformatics/btv112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. R.</span> <span class="cit-name-surname">Abecasis</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. M.</span> <span class="cit-name-surname">Kang</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Unified representation of genetic variants.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">2202</span>–<span class="cit-lpage">2204</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btv112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv112%26rft_id%253Dinfo%253Apmid%252F25701572%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25701572&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2977.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.42" data-doi="10.1200/JCO.2012.45.9628"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Terwijn</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. L. J.</span> <span class="cit-name-surname">van Putten</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kelder</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. H. J.</span> <span class="cit-name-surname">van der Velden</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Brooimans</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High prognostic impact of flow cytometric minimal residual disease detection in acute myeloid leukemia: data from the HOVON/SAKK AML 42A study.</span> <abbr class="cit-jnl-abbrev">J. Clin. Oncol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3889</span>–<span class="cit-lpage">3897</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1200/JCO.2012.45.9628</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BClin.%2BOncol.%26rft_id%253Dinfo%253Adoi%252F10.1200%252FJCO.2012.45.9628%26rft_id%253Dinfo%253Apmid%252F24062400%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNvIjtzOjU6InJlc2lkIjtzOjEwOiIzMS8zMS8zODg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI5NzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.43" data-doi="10.1182/blood-2018-02-829911"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thol</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Gabdoulline</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Liebich</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Klement</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Schiller</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Measurable residual disease monitoring by NGS before allogeneic hematopoietic cell transplantation in AML.</span> <abbr class="cit-jnl-abbrev">Blood</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">1703</span>–<span class="cit-lpage">1713</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1182/blood-2018-02-829911</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBlood%26rft_id%253Dinfo%253Adoi%252F10.1182%252Fblood-2018-02-829911%26rft_id%253Dinfo%253Apmid%252F30190321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6ImJsb29kam91cm5hbCI7czo1OiJyZXNpZCI7czoxMToiMTMyLzE2LzE3MDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjk3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.44" data-doi="10.1073/pnas.96.16.9236"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vogelstein</span>,  <span class="cit-name-given-names">B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. W.</span> <span class="cit-name-surname">Kinzler</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Digital PCR.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">9236</span>–<span class="cit-lpage">9241</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.96.16.9236</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DVogelstein%26rft.auinit1%253DB.%26rft.volume%253D96%26rft.issue%253D16%26rft.spage%253D9236%26rft.epage%253D9241%26rft.atitle%253DDigital%2BPCR%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.96.16.9236%26rft_id%253Dinfo%253Apmid%252F10430926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTYvMTYvOTIzNiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOS8yOTc3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2977.45" data-doi="10.1038/ncomms12484"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Young</span>,  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. A.</span> <span class="cit-name-surname">Challen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. M.</span> <span class="cit-name-surname">Birmann</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. E.</span> <span class="cit-name-surname">Druley</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Clonal haematopoiesis harbouring AML-associated mutations is ubiquitous in healthy adults.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">12484</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms12484</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D7%26rft.spage%253D12484%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms12484%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms12484&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/9/2977.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/9/2963" title="Multivariate Genome-Wide Association Analyses Reveal the Genetic Basis of Seed Fatty Acid Composition in Oat (Avena sativa L.)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/9/2989" title="Genome-Wide Association Analysis of Anoxia Tolerance in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/9"><span>Volume 9 Issue 9, September 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/9.cover-source.jpg?itok=4kZTDz_V" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/2977" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-wtDntKdKc61w-39hvCUt-pFgPmJirah82jcJWsGWr5w" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/9/2977" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/9/2977" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/9/2977" class="active" data-icon-position="" data-hide-link-title="0">FERMI: A Novel Method for Sensitive Detection of Rare Mutations in Somatic Tissue</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494055" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-UCMWARZiWRyDye1z4PDa6eT-Vgmd9cs8W6rVB06ozrc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/9/2977.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/9/2977" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-YGbZCV5j6ZKPLDK1amnCxWXjHfuht0JHRSsezciYnec" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/9/2977" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/9/2977.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494055" id="node494055--2" data-pisa="ggg;9/9/2977" data-pisa-master="ggg;g3.119.400438" data-apath="/ggg/9/9/2977.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">FERMI: A Novel Method for Sensitive Detection of Rare Mutations in Somatic Tissue</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">L. Alexander Liggett</span>, <span class="highwire-citation-author" data-delta="1">Anchal Sharma</span>, <span class="highwire-citation-author" data-delta="2">Subhajyoti De</span> and <span class="highwire-citation-author" data-delta="3">James DeGregori</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2977-2987;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400438 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494055--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">L. Alexander Liggett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO 80045</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Linda Crnic Institute for Down Syndrome, University of Colorado School of Medicine, Aurora, CO 80045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=L.%2BAlexander%2BLiggett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liggett%20LA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AL.%2BAlexander%2BLiggett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anchal Sharma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Rutgers Cancer Institute, New Brunswick, NJ 08901</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anchal%2BSharma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sharma%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnchal%2BSharma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Subhajyoti De </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Rutgers Cancer Institute, New Brunswick, NJ 08901</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Subhajyoti%2BDe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=De%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASubhajyoti%2BDe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">James DeGregori </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO 80045</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Linda Crnic Institute for Down Syndrome, University of Colorado School of Medicine, Aurora, CO 80045</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Integrated Department of Immunology</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Pediatrics</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Medicine, Section of Hematology, University of Colorado School of Medicine, Aurora, CO 80045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BDeGregori%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=DeGregori%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJames%2BDeGregori%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:james.degregori@ucdenver.edu" class="" data-icon-position="" data-hide-link-title="0">james.degregori@ucdenver.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/9/2977" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/9/2977&amp;title=FERMI%3A%20A%20Novel%20Method%20for%20Sensitive%20Detection%20of%20Rare%20Mutations%20in%20Somatic%20Tissue" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/9/2977&amp;title=FERMI%3A%20A%20Novel%20Method%20for%20Sensitive%20Detection%20of%20Rare%20Mutations%20in%20Somatic%20Tissue" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/9/2977&amp;title=FERMI%3A%20A%20Novel%20Method%20for%20Sensitive%20Detection%20of%20Rare%20Mutations%20in%20Somatic%20Tissue" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/9/2977&amp;text=FERMI%3A%20A%20Novel%20Method%20for%20Sensitive%20Detection%20of%20Rare%20Mutations%20in%20Somatic%20Tissue" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/9/2977&amp;title=FERMI%3A%20A%20Novel%20Method%20for%20Sensitive%20Detection%20of%20Rare%20Mutations%20in%20Somatic%20Tissue" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/9/2977&amp;t=FERMI%3A%20A%20Novel%20Method%20for%20Sensitive%20Detection%20of%20Rare%20Mutations%20in%20Somatic%20Tissue" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/9/2977&amp;title=FERMI%3A%20A%20Novel%20Method%20for%20Sensitive%20Detection%20of%20Rare%20Mutations%20in%20Somatic%20Tissue" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/9/2977&amp;title=FERMI%3A%20A%20Novel%20Method%20for%20Sensitive%20Detection%20of%20Rare%20Mutations%20in%20Somatic%20Tissue" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494055/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494055" id="node494055--3" data-pisa="ggg;9/9/2977" data-pisa-master="ggg;g3.119.400438" data-apath="/ggg/9/9/2977.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">FERMI: A Novel Method for Sensitive Detection of Rare Mutations in Somatic Tissue</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">L. Alexander Liggett</span>, <span class="highwire-citation-author" data-delta="1">Anchal Sharma</span>, <span class="highwire-citation-author" data-delta="2">Subhajyoti De</span> and <span class="highwire-citation-author" data-delta="3">James DeGregori</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2977-2987;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400438 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494055--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">L. Alexander Liggett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO 80045</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Linda Crnic Institute for Down Syndrome, University of Colorado School of Medicine, Aurora, CO 80045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=L.%2BAlexander%2BLiggett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liggett%20LA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AL.%2BAlexander%2BLiggett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anchal Sharma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Rutgers Cancer Institute, New Brunswick, NJ 08901</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anchal%2BSharma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sharma%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnchal%2BSharma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Subhajyoti De </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Rutgers Cancer Institute, New Brunswick, NJ 08901</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Subhajyoti%2BDe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=De%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASubhajyoti%2BDe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">James DeGregori </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO 80045</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Linda Crnic Institute for Down Syndrome, University of Colorado School of Medicine, Aurora, CO 80045</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Integrated Department of Immunology</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Pediatrics</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Medicine, Section of Hematology, University of Colorado School of Medicine, Aurora, CO 80045</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BDeGregori%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=DeGregori%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJames%2BDeGregori%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:james.degregori@ucdenver.edu" class="" data-icon-position="" data-hide-link-title="0">james.degregori@ucdenver.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2977&amp;rft.epage=2987&amp;rft.atitle=FERMI%3A%20A%20Novel%20Method%20for%20Sensitive%20Detection%20of%20Rare%20Mutations%20in%20Somatic%20Tissue&amp;rft.volume=9&amp;rft.issue=9&amp;rft.date=2019-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Liggett%2C+L.+Alexander&amp;rft.au=Sharma%2C+Anchal&amp;rft.au=De%2C+Subhajyoti&amp;rft.au=DeGregori%2C+James"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494055/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494055/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494055/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494055/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494055/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494055/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494055/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494055/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494055/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494055/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494055/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494055/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/9/2977" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/9/2977.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/9/2977.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/9/2977.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
