<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/9/2811" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/9/2811.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/9/2811.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/9/2811.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/9/2811/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/9/2811.atom" />
<meta name="HW.pisa" content="ggg;9/9/2811" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Partial Selfing Can Reduce Genetic Loads While Maintaining Diversity During Experimental Evolution" />
<meta name="DC.Identifier" content="10.1534/g3.119.400239" />
<meta name="DC.Date" content="2019-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Chelo et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Partial selfing, whereby self- and cross- fertilization occur in populations at intermediate frequencies, is generally thought to be evolutionarily unstable. Yet, it is found in natural populations. This could be explained if populations with partial selfing are able to reduce genetic loads and the possibility for inbreeding depression while keeping genetic diversity that may be important for future adaptation. To address this hypothesis, we compare the experimental evolution of Caenorhabditis elegans populations under partial selfing, exclusive selfing or predominant outcrossing, while they adapt to osmotically challenging conditions. We find that the ancestral genetic load, as measured by the risk of extinction upon inbreeding by selfing, is maintained as long as outcrossing is the main reproductive mode, but becomes reduced otherwise. Analysis of genome-wide single-nucleotide polymorphisms (SNPs) during experimental evolution and among the inbred lines that survived enforced inbreeding indicates that populations with predominant outcrossing or partial selfing maintained more genetic diversity than expected with neutrality or purifying selection. We discuss the conditions under which this could be explained by the presence of recessive deleterious alleles and/or overdominant loci. Taken together, our observations suggest that populations evolving under partial selfing can gain some of the benefits of eliminating unlinked deleterious recessive alleles and also the benefits of maintaining genetic diversity at partially dominant or overdominant loci that become associated due to variance of inbreeding levels." />
<meta name="DC.Contributor" content="Ivo M. Chelo" />
<meta name="DC.Contributor" content="Bruno Afonso" />
<meta name="DC.Contributor" content="Sara Carvalho" />
<meta name="DC.Contributor" content="Ioannis Theologidis" />
<meta name="DC.Contributor" content="Christine Goy" />
<meta name="DC.Contributor" content="Ania Pino-Querido" />
<meta name="DC.Contributor" content="Stephen R. Proulx" />
<meta name="DC.Contributor" content="Henrique Teotónio" />
<meta name="article:published_time" content="2019-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Partial Selfing Can Reduce Genetic Loads While Maintaining Diversity During Experimental Evolution" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Partial selfing, whereby self- and cross- fertilization occur in populations at intermediate frequencies, is generally thought to be evolutionarily unstable. Yet, it is found in natural populations. This could be explained if populations with partial selfing are able to reduce genetic loads and the possibility for inbreeding depression while keeping genetic diversity that may be important for future adaptation. To address this hypothesis, we compare the experimental evolution of &lt;i&gt;Caenorhabditis elegans&lt;/i&gt; populations under partial selfing, exclusive selfing or predominant outcrossing, while they adapt to osmotically challenging conditions. We find that the ancestral genetic load, as measured by the risk of extinction upon inbreeding by selfing, is maintained as long as outcrossing is the main reproductive mode, but becomes reduced otherwise. Analysis of genome-wide single-nucleotide polymorphisms (SNPs) during experimental evolution and among the inbred lines that survived enforced inbreeding indicates that populations with predominant outcrossing or partial selfing maintained more genetic diversity than expected with neutrality or purifying selection. We discuss the conditions under which this could be explained by the presence of recessive deleterious alleles and/or overdominant loci. Taken together, our observations suggest that populations evolving under partial selfing can gain some of the benefits of eliminating unlinked deleterious recessive alleles and also the benefits of maintaining genetic diversity at partially dominant or overdominant loci that become associated due to variance of inbreeding levels.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/09/01" />
<meta name="citation_mjid" content="ggg;9/9/2811" />
<meta name="citation_id" content="9/9/2811" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/9/2811" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/9/2811.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/9/2811.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/9/2811.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400239" />
<meta name="citation_pmid" content="31278175" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2811" />
<meta name="citation_lastpage" content="2821" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Ivo M. Chelo" />
<meta name="citation_author_institution" content="Instituto Gulbenkian de Ciência, Apartado 14, P-2781-901 Oeiras, Portugal" />
<meta name="citation_author_institution" content="cE3c – Center for Ecology, Evolution and Environmental Changes, Faculdade de Ciências, Universidade de Lisboa, Lisboa, Portugal" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5414-6625" />
<meta name="citation_author" content="Bruno Afonso" />
<meta name="citation_author_institution" content="Instituto Gulbenkian de Ciência, Apartado 14, P-2781-901 Oeiras, Portugal" />
<meta name="citation_author_institution" content="Institut de Biologie de l’École Normale Supérieure (IBENS), Inserm U1024, CNRS UMR 8197, F-75005 Paris, France" />
<meta name="citation_author" content="Sara Carvalho" />
<meta name="citation_author_institution" content="Instituto Gulbenkian de Ciência, Apartado 14, P-2781-901 Oeiras, Portugal" />
<meta name="citation_author" content="Ioannis Theologidis" />
<meta name="citation_author_institution" content="Institute of Molecular Biology and Biotechnology, Foundation for Research and Technology-Hellas, 73100 Heraklion, Greece" />
<meta name="citation_author" content="Christine Goy" />
<meta name="citation_author_institution" content="Leibniz Research Institute for Environmental Medicine, 40225 Düsseldorf, Germany, and" />
<meta name="citation_author" content="Ania Pino-Querido" />
<meta name="citation_author_institution" content="Instituto Gulbenkian de Ciência, Apartado 14, P-2781-901 Oeiras, Portugal" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6830-4148" />
<meta name="citation_author" content="Stephen R. Proulx" />
<meta name="citation_author_institution" content="Department of Ecology, Evolution, and Marine Biology, University of California Santa Barbara, CA 93106" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0307-1907" />
<meta name="citation_author" content="Henrique Teotónio" />
<meta name="citation_author_institution" content="Institut de Biologie de l’École Normale Supérieure (IBENS), Inserm U1024, CNRS UMR 8197, F-75005 Paris, France" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1057-6882" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. F. Agrawal;citation_title=Sexual selection and the maintenance of sexual reproduction.;citation_pages=692-695;citation_volume=411;citation_year=2001;citation_issue=6838;citation_pmid=11395771;citation_doi=10.1038/35079590" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=E. C. Andersen;citation_author=J. P. Gerke;citation_author=J. A. Shapiro;citation_author=J. R. Crissman;citation_author=R. Ghosh;citation_title=Chromosome-scale selective sweeps shape Caenorhabditis elegans genomic diversity.;citation_pages=285-290;citation_volume=44;citation_year=2012;citation_issue=3;citation_pmid=22286215;citation_doi=10.1038/ng.1050" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=J. Anderson;citation_author=L. Morran;citation_author=P. Phillips;citation_title=Outcrossing and the Maintenance of Males within C. elegans Populations.;citation_volume=101;citation_year=2010;citation_pmid=20212008;citation_doi=10.1093/jhered/esq003" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=N. Bierne;citation_author=A. Tsitrone;citation_author=P. David;citation_title=An Inbreeding Model of Associative Overdominance During a Population Bottleneck;citation_pages=1981-1990;citation_volume=155;citation_year=2000;citation_issue=4;citation_pmid=10924490" />
<meta name="citation_reference" content="citation_title=Genotyping with Sequenom;citation_pages=193-210;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=A. Burt;citation_title=Perspective: sex, recombination, and the efficacy of selection--was Weismann right?;citation_pages=337-351;citation_volume=54;citation_year=2000;citation_issue=2;citation_pmid=10937212;citation_doi=10.1554/0014-3820(2000)054[0337:PSRATE]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=BMC Evol. Biol.;citation_author=S. Carvalho;citation_author=I. M. Chelo;citation_author=C. Goy;citation_author=H. Teotónio;citation_title=The role of hermaphrodites in the experimental evolution of increased outcrossing rates in Caenorhabditis elegans.;citation_pages=116;citation_volume=14;citation_year=2014;citation_doi=10.1186/1471-2148-14-116" />
<meta name="citation_reference" content="citation_journal_title=BMC Evol. Biol.;citation_author=S. Carvalho;citation_author=P. C. Phillips;citation_author=H. Teotónio;citation_title=Hermaphrodite life history and the maintenance of partial selfing in experimental populations of Caenorhabditis elegans.;citation_pages=117;citation_volume=14;citation_year=2014;citation_pmid=24891140;citation_doi=10.1186/1471-2148-14-117" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=B. Charlesworth;citation_author=M. Morgan;citation_author=D. Charlesworth;citation_title=Multilocis models of inbreeding depression with synergistic selection and partial self-fertilization.;citation_pages=177-194;citation_volume=57;citation_year=1991;citation_doi=10.1017/S0016672300029256" />
<meta name="citation_reference" content="citation_journal_title=Annual Review of Ecology and Systematics;citation_journal_abbrev=Annu. Rev. Ecol. Syst.;citation_author=D. Charlesworth;citation_author=B. Charlesworth;citation_title=Inbreeding Depression And Its Evolutionary Consequences;citation_pages=237-268;citation_volume=18;citation_year=1987;citation_issue=1;citation_doi=10.1146/annurev.es.18.110187.001321" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=D. Charlesworth;citation_author=B. Charlesworth;citation_title=Inbreeding depression with heterozygote advantage and its effect on selection for modifiers changing the outcrossing rate.;citation_pages=870-888;citation_volume=44;citation_year=1990;citation_doi=10.2307/2409552" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=D. Charlesworth;citation_author=M. T. Morgan;citation_author=B. Charlesworth;citation_title=Inbreeding depression, genetic load, and the evolution of outcrossing rates in a multilocus system with no linkage.;citation_pages=1469-1489;citation_volume=44;citation_year=1990;citation_pmid=28564321;citation_doi=10.2307/2409330" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=I. M. Chelo;citation_author=S. Carvalho;citation_author=D. Manoel;citation_author=S. Proulx;citation_author=H. Teotónio;citation_title=The genetic basis and experimental evolution of inbreeding depression in Caenorhabiditis elegans.;citation_pages=248-254;citation_volume=112;citation_year=2013;citation_doi=10.1038/hdy.2013.100" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=I. M. Chelo;citation_author=H. Teotónio;citation_title=The opportunity for balancing selection in experimental populations of Caenorhabditis elegans.;citation_pages=142-156;citation_volume=67;citation_year=2013;citation_pmid=23289568;citation_doi=10.1111/j.1558-5646.2012.01744.x" />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot.;citation_author=P. O. Cheptou;citation_title=Clarifying BakerÂ’s Law.;citation_volume=109;citation_year=2012;citation_pmid=21685434;citation_doi=10.1093/aob/mcr127" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B;citation_author=D. R. Cox;citation_title=Regression models and life tables.;citation_pages=187-220;citation_volume=34;citation_year=1972" />
<meta name="citation_reference" content="citation_title=An Introduction to Population Genetics Theory;citation_year=1970" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. D. Cutter;citation_title=Nucleotide polymorphism and linkage disequilibrium in wild populations of the partial selfer Caenorhabditis elegans.;citation_volume=172;citation_year=2006;citation_pmid=16272415;citation_doi=10.1534/genetics.105.048207" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=P. David;citation_author=B. Pujol;citation_author=F. Viard;citation_author=V. Castella;citation_author=J. Goudet;citation_title=Reliable selfing rate estimates from imperfect population genetic data.;citation_pages=2474-2487;citation_volume=16;citation_year=2007;citation_issue=12;citation_pmid=17561907;citation_doi=10.1111/j.1365-294X.2007.03330.x" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=E. S. Dolgin;citation_author=B. Charlesworth;citation_author=S. E. Baird;citation_author=A. D. Cutter;citation_title=Inbreeding and outbreeding depression in Caenorhabditis nematodes.;citation_pages=1339-1352;citation_volume=61;citation_year=2007;citation_issue=6;citation_pmid=17542844;citation_doi=10.1111/j.1558-5646.2007.00118.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Estes;citation_author=P. C. Phillips;citation_author=D. R. Denver;citation_author=W. K. Thomas;citation_author=M. Lynch;citation_title=Mutation Accumulation in Populations of Varying Size: The Distribution of Mutational Effects for Fitness Correlates in Caenorhabditis elegans;citation_pages=1269-1279;citation_volume=166;citation_year=2004;citation_issue=3;citation_pmid=15082546;citation_doi=10.1534/genetics.166.3.1269" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. E. Gaertner;citation_author=M. D. Parmenter;citation_author=M. V. Rockman;citation_author=L. Kruglyak;citation_author=P. C. Phillips;citation_title=More than the sum of its parts: a complex epistatic network underlies natural variation in thermal preference behavior in Caenorhabditis elegans.;citation_volume=192;citation_year=2012;citation_pmid=23086219;citation_doi=10.1534/genetics.112.142877" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. E. Goldberg;citation_author=J. R. Kohn;citation_author=R. Lande;citation_author=K. A. Robertson;citation_author=S. A. Smith;citation_title=Species Selection Maintains Self-Incompatibility;citation_pages=493-495;citation_volume=330;citation_year=2010;citation_issue=6003;citation_pmid=20966249;citation_doi=10.1126/science.1194513" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Ecol. Evol. Syst.;citation_author=C. Goodwillie;citation_author=S. Kalisz;citation_author=C. G. Eckert;citation_title=The evolutionary enigma of mixed mating systems in plants: occurrence, theoreticalexplanations, and empirical evidence.;citation_pages=47-79;citation_volume=36;citation_year=2005;citation_doi=10.1146/annurev.ecolsys.36.091704.175539" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=T. S. Guzella;citation_author=S. Dey;citation_author=I. M. Chelo;citation_author=A. Pino-Querido;citation_author=V. F. Pereira;citation_title=Slower environmental change hinders adaptation from standing genetic variation.;citation_pages=e1007731;citation_volume=14;citation_year=2018;citation_doi=10.1371/journal.pgen.1007731" />
<meta name="citation_reference" content="citation_journal_title=Ann. Eugen.;citation_author=J. B. S. Haldane;citation_title=The association of characters as a result of inbreeding and linkage.;citation_pages=15-23;citation_volume=15;citation_year=1949;citation_pmid=15403124;citation_doi=10.1111/j.1469-1809.1949.tb02418.x" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Plant Sci.;citation_author=B. Igic;citation_author=R. Lande;citation_author=J. R. Kohn;citation_title=Loss of Self-Incompatibility and Its Evolutionary Consequences.;citation_pages=93-104;citation_volume=169;citation_year=2008;citation_doi=10.1086/523362" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=P. Jarne;citation_author=J. R. Auld;citation_title=Animals mix it up too: the distribution of self-fertilization among hermaphroditic animals.;citation_pages=1816-1824;citation_volume=60;citation_year=2006;citation_issue=9;citation_pmid=17089966;citation_doi=10.1111/j.0014-3820.2006.tb00525.x" />
<meta name="citation_reference" content="citation_journal_abbrev=Am Nat;citation_author=M. O. Johnston;citation_author=E. Porcher;citation_author=P. O. Cheptou;citation_author=C. G. Eckert;citation_author=E. Elle;citation_title=Correlations among fertility components can maintain mixed mating in plants.;citation_pages=1-11;citation_volume=173;citation_year=2009;citation_issue=1;citation_pmid=19055444;citation_doi=10.1086/593705" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=E. L. Kaplan;citation_author=P. Meier;citation_title=Nonparametric estimation from incomplete observations.;citation_pages=457-481;citation_volume=53;citation_year=1958;citation_doi=10.1080/01621459.1958.10501452" />
<meta name="citation_reference" content="citation_journal_title=Journal of Theoretical Biology;citation_journal_abbrev=Journal of Theoretical Biology;citation_author=J. K. Kelly;citation_title=Mutation-selection balance in mixed mating populations.;citation_pages=355-365;citation_volume=246;citation_year=2007;citation_issue=2;citation_pmid=17291538;citation_doi=10.1016/j.jtbi.2006.12.030" />
<meta name="citation_reference" content="citation_title=The Neutral Theory of Molecular Evolution;citation_author=M. Kimura;citation_year=1983;citation_doi=10.1017/CBO9780511623486" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Lande;citation_author=E. Porcher;citation_title=Maintenance of Quantitative Genetic Variance Under Partial Self-Fertilization, with Implications for Evolution of Selfing.;citation_volume=200;citation_year=2015;citation_pmid=25969460;citation_doi=10.1534/genetics.115.176693" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=R. Lande;citation_author=D. W. Schemske;citation_title=The evolution of self-fertilization and inbreeding depression in plants. I. Genetic models.;citation_pages=24-40;citation_volume=39;citation_year=1985;citation_doi=10.2307/2408514" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=R. Lande;citation_author=D. W. Schemske;citation_author=S. T. Schultz;citation_title=High inbreeding depression, selective interference among loci, and the threshold selfing rate for purging recessive lethal mutations.;citation_pages=965-978;citation_volume=48;citation_year=1994;citation_doi=10.2307/2410359" />
<meta name="citation_reference" content="citation_year=2015" />
<meta name="citation_reference" content="citation_title=Genetics and Analysis of Quantitative Traits;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Ecol. Lett.;citation_author=L. Masri;citation_author=R. D. Schulte;citation_author=N. Timmermeyer;citation_author=S. Thanisch;citation_author=L. L. Crummenerl;citation_title=Sex differences in host defence interfere with parasite-mediated selection for outcrossing during host-parasite coevolution.;citation_pages=461-468;citation_volume=16;citation_year=2013;citation_pmid=23301667;citation_doi=10.1111/ele.12068" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Matuszewski;citation_author=J. Hermisson;citation_author=M. Kopp;citation_title=Catch Me if You Can: Adaptation from Standing Genetic Variation to a Moving Phenotypic Optimum.;citation_volume=200;citation_year=2015;citation_pmid=26038348;citation_doi=10.1534/genetics.115.178574" />
<meta name="citation_reference" content="citation_title=Genetics;citation_author=L. M. Noble;citation_author=I. Chelo;citation_author=T. Guzella;citation_author=B. Afonso;citation_author=D. D. Riccardi;citation_title=Polygenicity and Epistasis Underlie Fitness-Proximal Traits in the Caenorhabditis elegans Multiparental Experimental Evolution (CeMEE) Panel.;citation_volume=207;citation_year=2017;citation_pmid=29066469;citation_doi=10.1534/genetics.117.300406" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=M. Nordborg;citation_author=B. Charlesworth;citation_author=D. Charlesworth;citation_title=Increased levels of polymorphim surrounding selectively maintained sites in highly selfing species.;citation_volume=263;citation_year=1996;citation_doi=10.1098/rspb.1996.0152" />
<meta name="citation_reference" content="citation_journal_title=Evolution Letters;citation_author=E. Noël;citation_author=E. Fruitet;citation_author=D. Lelaurin;citation_author=N. Bonel;citation_author=A. Ségard;citation_title=Sexual selection and inbreeding: Two efficient ways to limit the accumulation of deleterious mutations.;citation_pages=80-92;citation_volume=3;citation_year=2019;citation_doi=10.1002/evl3.93" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=T. Ohta;citation_author=M. Kimura;citation_title=Development of associative overdominance through linkage disequilibrium in finite populations.;citation_pages=165-177;citation_volume=16;citation_year=1970;citation_issue=2;citation_pmid=5516427;citation_doi=10.1017/S0016672300002391" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Palsson;citation_author=P. Pamilo;citation_title=The Effects of Deleterious Mutations on Linked, Neutral Variation in Small Populations;citation_pages=475-483;citation_volume=153;citation_year=1999;citation_issue=1;citation_pmid=10471727" />
<meta name="citation_reference" content="citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=N. Poullet;citation_author=A. Vielle;citation_author=C. Gimond;citation_author=S. Carvalho;citation_author=H. Teotónio;citation_title=Complex heterochrony underlies the evolution of Caenorhabditis elegans hermaphrodite sex allocation.;citation_pages=2357-2369;citation_volume=70;citation_year=2016;citation_doi=10.1111/evo.13032" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=M. Rockman;citation_author=L. Kruglyak;citation_title=Recombinational landscape and population genomics of Caenorhabditis elegans.;citation_pages=e1000419-e1000419;citation_volume=5;citation_year=2009;citation_issue=3;citation_pmid=19283065;citation_doi=10.1371/journal.pgen.1000419" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. V. Rockman;citation_author=S. S. Skrovanek;citation_author=L. Kruglyak;citation_title=Selection at Linked Sites Shapes Heritable Phenotypic Variation in C. elegans;citation_pages=372-376;citation_volume=330;citation_year=2010;citation_issue=6002;citation_pmid=20947766;citation_doi=10.1126/science.1194208" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Roze;citation_title=Effects of Interference Between Selected Loci on the Mutation Load, Inbreeding Depression, and Heterosis.;citation_volume=201;citation_year=2015;citation_pmid=26269503;citation_doi=10.1534/genetics.115.178533" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Roze;citation_title=Background selection in partially selfing populations.;citation_volume=203;citation_year=2016;citation_pmid=27075726;citation_doi=10.1534/genetics.116.187955" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=S. T. Schultz;citation_author=M. Lynch;citation_title=Mutation and extinction: the role of variable mutational effects, synergistic epistasis, beneficial mutations, and degree of outcrossing.;citation_pages=1363-1371;citation_volume=51;citation_year=1997;citation_doi=10.2307/2411188" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=H. S. Seidel;citation_author=M. V. Rockman;citation_author=L. Kruglyak;citation_title=Widespread genetic incompatibility in C. elegans maintained by balancing selection.;citation_volume=319;citation_year=2008;citation_pmid=18187622;citation_doi=10.1126/science.1151107" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Siller;citation_title=Sexual selection and the maintenance of sex.;citation_pages=689-692;citation_volume=411;citation_year=2001;citation_issue=6838;citation_pmid=11395770;citation_doi=10.1038/35079578" />
<meta name="citation_reference" content="citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. D. Stewart;citation_author=P. C. Phillips;citation_title=Selection and Maintenance of Androdioecy in Caenorhabditis elegans;citation_pages=975-982;citation_volume=160;citation_year=2002;citation_issue=3;citation_pmid=11901115" />
<meta name="citation_reference" content="citation_title=Maintenance of C. elegans;citation_year=1999" />
<meta name="citation_reference" content="citation_journal_title=Methods Ecol. Evol.;citation_author=M. A. Stoffel;citation_author=M. Esser;citation_author=M. Kardos;citation_author=E. Humble;citation_author=H. Nichols;citation_title=inbreedr: An r package for the analysis of inbreeding based on genetic markers.;citation_pages=1331-1339;citation_volume=7;citation_year=2016;citation_doi=10.1111/2041-210X.12588" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=J. A. Sved;citation_title=Linkage disequilibrium and homozygosity of chromosome segments in finite populations.;citation_pages=125-141;citation_volume=2;citation_year=1971;citation_issue=2;citation_pmid=5170716;citation_doi=10.1016/0040-5809(71)90011-6" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=M. Szulkin;citation_author=N. Bierne;citation_author=P. David;citation_title=Heterozygosity-fitness correlations: a time for reappraisal.;citation_pages=1202-1217;citation_volume=64;citation_year=2010;citation_issue=5;citation_pmid=20148954;citation_doi=10.1111/j.1558-5646.2010.00966.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=H. Teotónio;citation_author=S. Carvalho;citation_author=D. Manoel;citation_author=M. Roque;citation_author=I. M. Chelo;citation_title=Evolution of outcrossing in experimental populations of Caenorhabditis elegans.;citation_pages=e35811-e35811;citation_volume=7;citation_year=2012;citation_issue=4;citation_pmid=22540006;citation_doi=10.1371/journal.pone.0035811" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Teotónio;citation_author=S. Estes;citation_author=P. Phillips;citation_author=C. F. Baer;citation_title=Experimental evolution with Caenorhabditis nematodes.;citation_volume=206;citation_year=2017;citation_pmid=28592504;citation_doi=10.1534/genetics.115.186288" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=H. Teotónio;citation_author=D. Manoel;citation_author=P. Phillips;citation_title=Genetic variation for outcrossing among Caenorhabditis elegans isolates.;citation_pages=1300-1305;citation_volume=60;citation_year=2006;citation_issue=6;citation_pmid=16892979;citation_doi=10.1111/j.0014-3820.2006.tb01207.x" />
<meta name="citation_reference" content="citation_journal_title=BMC Biol.;citation_author=I. Theologidis;citation_author=I. M. Chelo;citation_author=C. Goy;citation_author=H. Teotónio;citation_title=Reproductive assurance drives transitions to self-fertilization in experimental Caenorhabditis elegans.;citation_pages=93;citation_volume=12;citation_year=2014;citation_pmid=25369737;citation_doi=10.1186/s12915-014-0093-1" />
<meta name="citation_reference" content="citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=L. L. Vassilieva;citation_author=A. M. Hook;citation_author=M. Lynch;citation_title=The fitness effects of spontaneous mutations in Caenorhabditis elegans.;citation_pages=1234-1246;citation_volume=54;citation_year=2000;citation_issue=4;citation_pmid=11005291;citation_doi=10.1554/0014-3820(2000)054[1234:TFEOSM]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. Weir;citation_author=C. C. Cockerham;citation_title=Mixed self and random mating at two loci.;citation_pages=247-262;citation_volume=21;citation_year=1973;citation_issue=3;citation_pmid=4731639;citation_doi=10.1017/S0016672300013446" />
<meta name="citation_reference" content="citation_title=Genetic data analysis II;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Ziehe;citation_author=J. H. Roberds;citation_title=Inbreeding depression due to overdominance in partially self-fertilizing plant populations.;citation_pages=861-868;citation_volume=121;citation_year=1989;citation_issue=4;citation_pmid=17246494" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Partial Selfing Can Reduce Genetic Loads While Maintaining Diversity During Experimental Evolution" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
<meta name="twitter:description" content="Partial selfing, whereby self- and cross- fertilization occur in populations at intermediate frequencies, is generally thought to be evolutionarily unstable. Yet, it is found in natural populations. This could be explained if populations with partial selfing are able to reduce genetic loads and the possibility for inbreeding depression while keeping genetic diversity that may be important for future adaptation. To address this hypothesis, we compare the experimental evolution of Caenorhabditis elegans populations under partial selfing, exclusive selfing or predominant outcrossing, while they adapt to osmotically challenging conditions. We find that the ancestral genetic load, as measured by the risk of extinction upon inbreeding by selfing, is maintained as long as outcrossing is the main reproductive mode, but becomes reduced otherwise. Analysis of genome-wide single-nucleotide polymorphisms (SNPs) during experimental evolution and among the inbred lines that survived enforced inbreeding indicates that populations with predominant outcrossing or partial selfing maintained more genetic diversity than expected with neutrality or purifying selection. We discuss the conditions under which this could be explained by the presence of recessive deleterious alleles and/or overdominant loci. Taken together, our observations suggest that populations evolving under partial selfing can gain some of the benefits of eliminating unlinked deleterious recessive alleles and also the benefits of maintaining genetic diversity at partially dominant or overdominant loci that become associated due to variance of inbreeding levels." />
<meta name="og-title" property="og:title" content="Partial Selfing Can Reduce Genetic Loads While Maintaining Diversity During Experimental Evolution" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/9/2811" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Partial selfing, whereby self- and cross- fertilization occur in populations at intermediate frequencies, is generally thought to be evolutionarily unstable. Yet, it is found in natural populations. This could be explained if populations with partial selfing are able to reduce genetic loads and the possibility for inbreeding depression while keeping genetic diversity that may be important for future adaptation. To address this hypothesis, we compare the experimental evolution of Caenorhabditis elegans populations under partial selfing, exclusive selfing or predominant outcrossing, while they adapt to osmotically challenging conditions. We find that the ancestral genetic load, as measured by the risk of extinction upon inbreeding by selfing, is maintained as long as outcrossing is the main reproductive mode, but becomes reduced otherwise. Analysis of genome-wide single-nucleotide polymorphisms (SNPs) during experimental evolution and among the inbred lines that survived enforced inbreeding indicates that populations with predominant outcrossing or partial selfing maintained more genetic diversity than expected with neutrality or purifying selection. We discuss the conditions under which this could be explained by the presence of recessive deleterious alleles and/or overdominant loci. Taken together, our observations suggest that populations evolving under partial selfing can gain some of the benefits of eliminating unlinked deleterious recessive alleles and also the benefits of maintaining genetic diversity at partially dominant or overdominant loci that become associated due to variance of inbreeding levels." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9.cover-source.jpg" />
    <title>Partial Selfing Can Reduce Genetic Loads While Maintaining Diversity During Experimental Evolution | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"jBAOPDKdQ_jHaR-ACh_uc1qzMsy8agbePd2qwmD3Gjc","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494049","apath":"\/ggg\/9\/9\/2811.atom","pisa":"ggg;9\/9\/2811","ac":{"\/ggg\/9\/9\/2811.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/9\/2811.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/9\/2811"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/9\/2811"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/9\/2811":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022o05f8Afz3ThAqyfpcnZOhZfnEdbtTIOMvlgZxiV23Of2Sf2L3bG\\\\\\\/nw4yVau+dULYhhS\\\\\\\/x3\\\\\\\/nvlTAEBtjsr\\\\\\\/OPB1VR0QwpKDWWcCcsTcCRK0gCZipJg6HASBNBrjDquhej7VtzFw4oR0sneWJZBRfPj47zPIgZuVB5\\\\\\\/OExQuSRrEZj3CZt7b15GA2MfOoRFXKysfRzc2VUAOA7Q0I1ftO\\\\\\\/IQ2pet9s291siiEUkvwcwI5KVsQ+R+Al9UXmWo0F5iXDtYvHTwv7\\\\\\\/EqvDXxSfN9OV5p6JfVby5Q4rJY8vM\\\\\\\/6jSNed3TIeVvOSrmkjdKbAcwmB7Xl4nN2X6IpczJypjUEQYAhmBCCPiypEQh2YBlyqMXP0BO04HeVGQiRm4CwGhMAHo2lHebe4DWxM5atT+1Y5H12NxmLuT2ox0ywhxygLIxZ65h8QpWxNSactF7kIPc6TrqNCSQ3qUU6pusnQGjTdNjQksoEcj5Kiod7uronoeRadigViQ\\\\\\\/oNDAq+dWbUUf8s7wP\\\\\\\/iC8Mze8EJwyZ6CuClyiSmIYemcmTB1ltJKyasxRqpoHi3ghGQoAQmRl5oXvyYd6F39k8dRY0KQdO3x8yo6c4yGCC7k5RdBsLC9exb9YFS\\\\\\\/7hkooNJmUexUkNI3gCtHgVwATDW3hwaDRSpu63ARLfROZ0wJGcDnVaBcOF4gv1D7zeeYW+zTVqRcU5m0DyQAj8ibPZ8W6U8pPq4QjuBULQIQKwNLp2Aq5pejp7Sebsi+brBiZRcgdnNwogo9n4pY3Qo1cj2D42XUUB1WKRze6Tlmibrr+k8kTmxf\\\\\\\/PqYWpqE714eNSSgvOoWJdh8m1QPhh8h9xotQi+bDHAHLeep0n68Ly\\\\\\\/zb3tj3g4F83rSzeOgZC04WLB+UMSYw4u+PzBI3wZ4RX5jA5nn8fR590IlPBnzZELa542ODvcOjZcvSqdY\\\\\\\/EfWXNg\\\\\\\/m3uhQYJUpQiqgfy3wNe6DkS0HjT90jcWv5ZuBkCC4LFJtdsQXO\\\\\\\/d8oDbLsc5h7w+hMXG4u8753\\\\\\\/4xBOfXS93ju19NJCyoG9LvRQ1nJAMu1EbpctMzdWh8ufMinenBuzVAcUPGET99F3W50y38tyIuV3a3NkBFsMmhA6tcPwyiATU1\\\\\\\/1ajPbytva9pMPR1DGQW1a8h3vIjx6ZktpSRerSNoPx7gDZzjt452iQ6bfXrdf\\\\\\\/S9ci\\\\\\\/L5ffjsKT+QAFpHbAxkgoUXmyKnyNV30ma3TXoNp\\\\\\\/AeQlbmhJrJTLS7yMFgCjhFuqD3wbbLW8JHqruLZ4k8THAQk\\\u0022,\\\u0022iv\\\u0022:\\\u0022R\\\\\\\/QvRuQbHwR7P6em8t8bsg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002253a01bf84ad52d92dc9a0ee4fc167edd8ef21ec6404bf9e9411e0a51d6a47c0b\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022TfkHLuq3EcLBcUn\\\\\\\/QLwD9Lc0EikII48xSeWJnnAOtZlMxgC9lD6Skowp9bQ+PKxwxRfjtW5K+mICF8BujCeSjhvHqk6HiIkWrYM3MxKNGeoBQfOgBijhrSA+w8EJDQaiwTOt\\\\\\\/YlrN0Ze2NsxwptQJpWO4cbdDxftjTUtLBoksSfSfY3xKF6tb++thsHxsvsODobNAmEuJz2wWbseKJunNzaSNeZroqKRPIVSdR\\\\\\\/6NXDORgqMVK8jRy7gx8eY\\\\\\\/qqrZuofioLhLOPlWtg3KBp6di6i1MxbDlAbYrwOzfO1YWLd5k6Yi4go0zHb20ltiB+Lt75tUH\\\\\\\/x8tv2mmkvD6iFfZ7764zUh3MhjuSbeHUZX0alhzWHOYaG3sy1pzOTMb8Ve99tx5emirjVlyaKGiUjfgYe6mfFW7664JXL\\\\\\\/8x37HoKngo0y37kuhkvwRbn2vz\\\\\\\/ncCAeThQVOw4Y2cYJIdnQlpJO56Xdtd439rzJqI6wFaAEOHdQT6AlG\\\\\\\/HTqQ0uEMpnA3BASOUBVinAWRoZsNHBg+pBC5M1sIM27qibtJRRy3ZqiW0jtAzsi6IGSbzrVixx9IEjvXwoaNsu7Sou1f95OCiGlNuVmQaSBnHZmhvaDDkE5j4E4FdGoJnNzhFMyq6PY4xcgSAR8fMBud2zd0mvwL4fk\\\\\\\/LvEyxZIltZcbY1ypwIxzPBdgla2A6hlq5tCVyMJ+Epul2kXkFpGdPKcXsJovF33w3CqmeHJmzYGVcBmSSV1+ZbshTLiqYO1IuS4xfv+HAlxQxaHqBa72TDHyNy\\\\\\\/6uBiYyO0jVRDMtMWjL3IjH+7HXv8TUAExhnCXm49b45eFOyJLz\\\\\\\/9BfBUwYwVjKS2YxAkiu8ov6PYw1IYFG3Iw+aYVECsuiph752BnegKRpmPkvcO2uXdTrcN1MTficcsJrl65OBghN\\\\\\\/VlgLLRjmuzq2wDlVhosVuOcCAAPNkNiOw9kzfS6MRnV\\\\\\\/rF4cWdcHdpp3saiEEx1VtpeVDX3KO3q4YsFdLbPyrd1WMqwb+TPkhXWGSP9Ov3Rm\\\\\\\/RFTb2i79YW8LlCoFz97cE+BmFbXB00cvUwxPaufj8rHhKnOwsfwzne7J5SXBfIugnvkjTjpZd+ekXcjdf4cw3CQAdJi0sTFIu4DwSoCmIPLTxJCnGrutF+zmIMlFsLxNHWOhuF\\\\\\\/BQPKONQD4E0qOH1KJfh2UXXIFti79WWASyrVF\\\\\\\/nrihYhOtaklpWb\\\\\\\/XNinYlbix\\\\\\\/idC806Ektdim7Eo1+FikXyRnKq46xw33h9C2ZrcbMGM7QpAcSK4dvkaudFosxuBQ7hhLPOH2m7Kz0+xN4yNc9pVVCGJVZY6DmtLRVzAIRz0D0UoQ6Nhkc9y31jdZ9VsJHYvG8hOILMNlM3Q1iiKMLzBGQy0xqR2pnIgYXcTmJ6kVFJVhEheKKrrwa84NCuDdG8tE88APh3Pd3tbbKqlmSR1Zx9qgzSZ94Zf2h1vSsGJcilT0lDmbJz9U2uQnbUz5ktJAWwH+47i2ckmYDduq8n7QfZsEIQPz8eICW96N32IVqFce6wrSPEegBa3uP5zOrZCMOkGc\\\\\\\/i9XqyRuzujr9PQd6viZ5FAc2whGzbdB0VguqnO7tmJI0ad1Wge8mgRLaMCxIxUpuvOjQsnjJ5S6FkvryLzNkg\\\\\\\/e0IQ8FxQ0ZHQicnGfFIKZ0+lKqfqGvFzuDm1W30rAG4iukVyTxRQe8ZEYRIhA14AmZZYmD4D56453EU084PmqmZQhVO9hQivb\\\\\\\/lC5ZbrmRNBAa2noFAJnhGkkO58Q3PVwiktjC7+CYkk6A+meHtap8NCwACX1qVV5yvNLSXK1HXYIudYbP5S6B202pt3OzyaX+mf1fhvcbqbr3tdx41JDE8qkRQyZXvTmvfDNMn5uYFtpPOy4DCrdxtO+BcY\\\\\\\/uCi2PwKCkZOlsxkCQTjLFQCVLme84f2ABd4zyapHYDSncI3Fz+UXkY6PPL90VRnLpbczop33kyT89H9WWlwyAtskFAkYOGWd6yhvswOOLLaINbhvKe94jOFXexRfa6bffz0Z0QQkK5Rsbqoa62O8KM460LMaQh7o9zxLsn71fi4Q4PC20GZbKfhVtFMPk3yIvBfSIX3NvgCzTek83agidwKfv7oex2JP9jSiubbrXY78dhpa\\\\\\\/tE6wlVuAQDRkEPqPr9INiy6hcQzA\\\\\\\/hHMDN6ak0aTUe5uX8\\\\\\\/ZVEbtUYaBk9O\\\\\\\/IVeQMvvwVd5DKkKsZWtTN84TygYDhMNxSOH\\\\\\\/sNvmksEje1zqChnV8K05J2XDs3qR\\\\\\\/he+6DVVwovvDRBvGTXXyBXJlJdTniktfmI6vQwXD7OEHkHsL1Zn4chyF2D7Jf1APNEbX2VIA4UjyiDxvsQtRNDsJfGMK8vNFQ\\\\\\\/nAjASTmLNFUMtv7bNZK\\\\\\\/5LCL9aeonv4vAUdX2f6IBxCLtORFevOAHXfgnElWVJ4vunuS5RjlgdVL+K5G59jFuRoenLPEVLjGG2Nq+z4Iki7irUjQsKAocu5X4izTLzLnXZG6QIzMLaGI5ssX9q6GzYV03uCyTM5o03sraFHsKhYQa8uFMEh4r9KZPufl8EXmVh6tbwokOSc3J8SPIJT5A0VOsxRs9wPZ0mhvUgfoelbbI+CDuhCLQez5eQB5ogPiwF0+TtcvwMwKZd5gaLv1qJ4xa5p4DPNLAoGFmtdo+4TVovSq4pGYAF4AlVQRQNk7ukUqwXJuq+l0uVIwY9uorvk4YUir1nTDTECm6b4Wt1FKhXXByu3b4oOl1ftNWQ2ehI9Id7I6Abc2jaOrn\\\\\\\/0l75sFaqkPoxNi1qXwhgnt\\\\\\\/D8B2\\\\\\\/nS+Yw9\\\\\\\/8Qn9drnxcZj0G5xr+Z6xKAmJsXYd7h\\\\\\\/yGcZwCu7T1xagzaKrwGfX3RJ0A5GVMYTyUMha38sup71EOY0U2pJ2kE0fo3ybh6l3h\\\\\\\/r4vgUjDku9ngIMeWs7+6uPsvtVLcmuzngWXzVtsQOA7XqrZT1i3x1Csy+86kBp4u1\\\\\\\/glmr+bImQvMR41jG9fXPss\\\\\\\/mtdWQsy5GiSc4FD\\\\\\\/rQgXiX0qd0ECJZTsqKE60kls+FgSYvZysFGBvn5tp5kE5ITALejPYmIJ3KTSdop+aRADEvLM9V3e\\\\\\\/ZNRKMQKJZzHgogAlimD\\\\\\\/kcfyMjtiFZ8OtCxKEUfc\\\\\\\/N4jK9PFB9OcE\\\\\\\/pMUF7EY3Pz6zIcENyFAVr6W9D+RoTX\\\\\\\/pnX+sTSZNDZynTsC9t9zA3CITGvyfqP44eKf0lKKKJGj9xMW7BZauDilTh+h+R\\\\\\\/GakHlnnRHajrFR0FgT15h9bNF0\\\\\\\/8YY01LiPIYFYocEUCIFGi2aJVuROlCzovJhDIwSaevd8KnlQIMC2CH35gIs6m3fZLdgx8zxhVBB09nMi76q+rUtsIyrRZI1g7IVcmTm2gzEYn\\\\\\\/YY7782pYvUWvkZrkylBpGTcfNJ08A+hC78u5az9t+3CwugYGUHxEYoAJCEvxh1ZNVucsrBVIhjyCOFUriDL66CMkAJYncDLmmKTI2sHjTdLfJXGLHYBNDbegMRfEGFf3D1G1NZKxGHln2cDu5ERGXlmcRPrkQCBW7Pwn1w\\\\\\\/0biGAglVU4ezXKY962jd+wFPAEsms\\\\\\\/BAvLXfcWgxekC4x60wTOzM+SJJ9IZh0k1VhjsKFyL2KvC9JeNv571IbkUPQSvYYykzlyVY14rzqyNPOZklPC52Z5fvIolEZvKz7wd\\\\\\\/SzOAwOiIuT3rtqr\\\\\\\/1wfhwKR+VoqlEZS7dUD2zkhLu\\\\\\\/borR4N7SgB6qNN9wwT7p7augwsOCBuVwj19wznSOBdQwhwWbX5kJgRAfFLwmEqKixfDZYkn5kvejAPyPwHq\\\\\\\/O17wCG7rgcNi8I8tquomjr2SBLpYVTKomjT1e0gqIhM0pyrqjbSHHM1V1U\\\\\\\/OnKO9KoguM76+zzU1vTmihWH\\\\\\\/R+mMA1Mr9f6EJbSw2Zbj7sV\\\\\\\/vIIHz9WiM+fVResCITVlmW1HJ4e7WyOFf95YM+BqYMQS2c+LVyUCPvDMxTKEJ6xcWINk6zRZ4lxWQfErGQ\\\\\\\/J3YVgf1Am71hIz\\\\\\\/kiFvbjidaC9N54FXA\\\\\\\/PztZwdW2LTaEUSAJRJ5qYMk7QlUNEV4IeiLwU+QfgJf6Itjmc+Sv5VtGF2ueq3E7JLWUH\\\\\\\/iPy23EtYSsLxg8nW9VfjTtlnAe\\\\\\\/qem7Ba2BoWRYH\\\\\\\/xzlbMBWGyG7\\\\\\\/OwoMAoKukZQGquwAObrHISysxhUAdcisJs3ikQkxsB0ictVVyImf+rPrIBeBVtizakKPM7Qs\\\\\\\/GD6e4KVtjnWSf9bciWMYGjQwp1pq\\\\\\\/fbYgUeAjliPG1YWk1bVeq15TcCYRwQoq+k7CqgOEeRqTyNQAdwFotJuLPyhCF8y7iDIDVJh2dj4\\\\\\\/qtQDy+61\\\\\\\/gLOHk8MxC2yGIuCpFHM1RrtixCp23qKQO9cvJL6XoZDz4vrphGLUc5hWW+SN32+o5kdKnGF4+klntrsJmR7F7JJWFWStJ\\\\\\\/CNoqdghdC7TxN3UMrhddaiVChIOZn9uDpRg+7Qdq29jWShyeBQlMVDjA8GA2kdD\\\\\\\/pe6IK2s0ebNFAwwZBV93bBa174\\\\\\\/cA7rBqeL2l9EyWXkjgRv9p+MjxPijTaQ1trrmbmKg4cWbvND90TVMMCMJ2fzpB5LEwQuRFkvHirVZ2DH1EXcyB+li8grU0I5kZmrbMW+pvr4jC1BUpZu\\\\\\\/tv6WinRBdf1pR30jSC+atzz5wALqoNttY4fTFpMHuO2vCFFzxVkg\\\\\\\/XGSUnSoZhOG2Js7KH9u3gO\\\\\\\/Se+YU0MLEpldkFFnsgJ6CsPHlJdHIfsNtap0Cm6ar4KEZSfD\\\\\\\/naZt+WspCZIzkXkahIUIMpvIAbnsO+cotEi\\\\\\\/7gM1O5RZcaW9jAYRRdAlArgdKu3km0YPItG3iurSyHdxX9n6lgFF0bJ0V6qrtLV\\\\\\\/Q\\\\\\\/uKSF68e2tFS60fk8BMtZBHdRMXMidbwemP745KCCRWtbYGEcF8uwHl1KpKHq6zdXBPa9wegI83ESWxz0nbKPJqOq\\\\\\\/JsJzgDXQB6oswOt3fyIli8zg\\\\\\\/DSvP21pk\\\\\\\/XzoLYS10fOQWENtyARW0oG7nE5cy9Pfej4\\\\\\\/L6V+oR0Wi3G94kguXZX+RMjOjE2D36Xq\\\\\\\/oC1sqGUfQSpg7H6kVble3qqxqBsug6xrdogdLyRdBGzPE\\\\\\\/+rEX+M4iCuTx6rxGuPegOv4Q03zWRBwQIzjqNqaCQ4NRE1XQweMmSW+ZLv7oCTuFlNk+7\\\\\\\/AzmvtcE4hQlZZ5HI0S1Hps1bPaFpVnSWD0w+4FkETlpBuQDJ7kQORz1mLMe8z3sCkIzZkytA3xRKEZUHfcJas1s\\\\\\\/FtsAodfg8+xv+kK\\\\\\\/0oXfBp9SLi5v1cL9yvnLjOTHqdl+DcgA1DfuZN\\\\\\\/tU0WUMclMJ6G2I8la\\\\\\\/wM2XZhELwiE9koA1HWbuhpzQJO3ETk3cI3QeZCv7gib4yLn\\\\\\\/g\\\\\\\/D5XAu3ev6Ex68NUnsqxcrTfTjcwWXAWwapvQ\\\\\\\/OkAv3F9MwZo0Hpv\\\\\\\/rAasw7QnE1fc1ehEfrOIyOUKggRobe7nXgmewP2MMWE7yLXdhDKhSjDDC0vadmHYifLY+e56h479TVWaCFWRMkRejs8uMvGFc1WyyCOpkM2Asuz3V\\\\\\\/GIAC0\\\\\\\/QF3DAl3GIn96C6hFBTgFwfWUvwqClNAZMvUNHGNubCxfLotPhuUGBxKWg3nrZ1NCWvDpf37P92LZ+1CxDoYALNjUIHPrlxGrHGwghf5LtRYEIOAP8hfHjWcqwXurhOG5pG0PtmIedGHogOFfp\\\\\\\/wxzJx7nfrXyoKiEdaSNvQ9rqOcceMKhGqXeDpYi0mzgXpY7cpdfb4Trp4rpNCouNoUXYZ44PhyieG5wpF0NW82GH4ebc6jDfUqd1+HS+772y+BkQX+S99HLCapEZJw12WZOAhk69zx0V5AJp5IxDn\\\\\\\/p2cLJ5qgEM+CuF6Y0rb45S1dIdeC8FpjA38srW8ZYdqz+69z\\\\\\\/0OpcYzOGCE1htfNhoBSnqkUrab35TNfX1g242HClS+kSlwEWb6sUKMIgrlpThwgiX7HYYhbz3TKwwlrhPPwRSAEmz3SN9pbxbwZdfpdzOCvc0DVEk4e55pc3\\\\\\\/8yI16Crrjvo3tzdut9ri0cC2+OSotT8hQ0AbqjKJcqhOOx9XwX8bUG9rjaemWD8GS\\\\\\\/ujOeQNkGFrjloc3lmY9GGfNd+Hnl2KJr22qs3V\\\\\\\/WECPgIdV7UV3w0yLEF+sIHTMj5T88dLIn5xezCP8ama6wEKnJOGBMFZGD18TftZj2V64TU94b5+y3Fwk8VNuQF33PQWkkZnuw+tqUwooK27gOb7AFmRkoaXEdcznO1NXuzLLjUBrjEt9UWTDeZYG\\\\\\\/uVwPI7MxTIXnuk\\\\\\\/ydDFZ6q1dCGP2rbpu86y5+S7k0zBVLlNXhqLrGO1qjWnJWJEEG\\\\\\\/eDv6jGMiAyZyE0MkvZARwa5vG6ujpRoeY7UgzeBfV1yXOOEhcceg4N6tXtQ5LiF3o8554zlkZdSr6WcXmq0O8UPfGHx3kC7x9y\\\\\\\/g97fpZvbWa3eGtMEhbXGCvTIWsJgw5+nPRlcEkm8wxqzFqWrxfwDJgVsOVCxtKN2B2xKt8kWDlZWaCpRbjQnD9eha9EW85Fn3S7FzvwZFTUhnYvmRgklKTSyWULqoTZo7tc+X\\\\\\\/Ssb6mAEzDbFVbYs+t5TWnnIqdNrCVSZrZJ38tsjRt7mwijW69H1w+cqznxg1ZWYNADyD2OUeOEJ6Mq2E\\\\\\\/xLsvgvd24BwdItSqs6pnOLE2vVDtnX26048iP\\\\\\\/s4DmvCRiv6vDVFcR\\\\\\\/IZlAnt8ciy9DvZVJy5gAfKStefOocOB\\\\\\\/DSljy+rFlLKDD6\\\\\\\/oGvdKRcUBrI+GQHRsH0vdj++TJ+Ki2EWM\\\\\\\/ZAg\\\\\\\/7NliF4zfJBU\\\\\\\/ji2mT55oO4jeSsjBmf6EwRS7aXFA56Eh0lmGFt6xZHnQPcHj\\\\\\\/lJtlMNTIddUyU34t5WSQyytlo+OZrq76eOM3LxdK1uKPr8fmRHbktbXttc4zLk+OQZPdv3lm7DeCxEJB8TuPQrMEnkPfHG4qQ5+0xFgvRlFdp52rYwe9bDN5A4rdp4ww0gcvM7G+8L5nFW4JpFCnkI\\\\\\\/HMlFcTlCMF3KIgwAgLUPcpuPXBPmLcoZwzBcUCcSNswWlN+pRdXH2BhLMMi1H53DEAGdGEXYpe+ymJBLZKZE6wrECkFWajmvt+Hd60Atjca1bPuPSNk08\\\\\\\/Q\\\\\\\/CrKcDCyfG6V\\\\\\\/WmV0n\\\\\\\/pTXdeDKjMsWP9Q5aeK3di4yzuBeMpzFYJQJi6GGurpc3BrB6Pkmd+fvzdCcLsMGzpXoGwdjsDimRKKo19RfTUIoApw30aX6x+KIplRECo3WNQpFAGEh0qoa1Q0v6yuNRVZpdwap56k03S4Ua5uQqFc6+Gj6EcWLeFqKSgMe5E5oUzH6Dx3kObZO\\\\\\\/hETsuTlM66zLz\\\\\\\/btGOmrQrDEwzu3Yk94i2FmlQzTz\\\\\\\/NP7HKfXxp2iLFw0NTGBBXHDVdPfx5Cwgvc6gEmJpMfMgTIOQo4IL6aj0nOzN3jXq5q4O1CFSOgG0JbjI9IBl\\\\\\\/jC2KTrn+r3sJQXSN\\\\\\\/XqPr\\\\\\\/4wxZZcChSCbtKBXb2VOS+YWJjwvdGIQWmz3Csd7CZUBnFBM1yKDPbGTrct4mOAIil43ImRcaB1AlqpgeZO4+znnLTIT6ZXjLvS5D5G\\\\\\\/D8A3hQLqJWmIyurVI4ge32XJL7XbSS7q\\\\\\\/4zt8mQ7SHpYA87IlaJOqzLYZBXXQBLs\\\\\\\/JS610dU6MRpWTi\\\\\\\/sG2ESPhUDEsdDH12ifAuPYB2r36WDfkXIX5MFCewYhrQYXi1zUQlHidMtEsG1ZX+3Lfi5nEUFRZg0V4fXt1V0tfzV1A4ss\\\\\\\/brgwcriQl2w6LtO6rwSzix+HvoVoVRnC7Xc0l970mxBP4l7EfqW2G2exeDPwwqhl0tEEqoJmL8OSPWpeoZZazRdxRDEHiKbtCNvLqunF7ISqC0qLm1vCLIrjsM0amBJFdD+sGIFVbF1YNHSuv+bu7rYvqrl+4Qoqurqqq33sNg88G7xfuDZ\\\\\\\/scDAqjbqxnkViAWeaATo3FTZuDI6gR2cF4JKKzW4TaZjOFfpeguHuJQbB7dGDG3QGHsUf7Q76gahlfGO6L62lm2MI4e+Hj+bujyuTmADiyTfn0DgBmP24tC1Gz5ut\\\\\\\/+nZj6ylaaetvjF9h1a3gMNkITaw6XmZjcXdSnPoqdtsgYvE++mjHQ+\\\\\\\/Fc5OEZdigIZozESBbP1wjdQ0ptlo+fPXFjDqOJ6+ZQebiYb2NBzqE0mYsT3FVg0Bw5QdS1v9RyDRZpE5WBMReEen7TJ6MGA3MeOLrG\\\\\\\/l\\\\\\\/\\\\\\\/ADTbNOIZLCtN20JyS3v9U33XJj1iqcXTRN1bt6ASSd+FLFDqjr9vT7EDxJ4RqoW5d6DQoJbGoLXjRWC8HK2BRCAPCdZtcJMkzFqGh3anRHoFHaE5K2Eb3MvFuU\\\\\\\/SJSuRyypZXwJgGI7+sScOlTXh0VqF52++dreii2FritR2IdagYunZt+p\\\\\\\/A\\\\\\\/7fAbEt1wgltDRzxGEwZewCuyBqevc1wZwGl2sYG0PoYFaxmTQY6ZdLfryIDmOEXLwtUeEJM2toPSbS71zBsA4V5rLEdLi43R1irNRn6VcNqrqi4QPjp2S17j5mKl5CzBjtseYtJnG6ilu1rqLDuTonuGfku1r9S4VBg6wr9xyBkAjAX7gIGGb09ZaZugm4Q8PYEEEjxXbaPprR6DQQ3N8Y3qDB5pu9r+HdvgHVgUIcNYsyX2gNVg1Lpx9OLiTu0lW2wgeEkbmyItK57P2iRYQMUNXoKBP28tpWNF89agoFTRlX9XuEdmb9qfywLEvPU40DX74RGMG55qjJM7Uuy+rULVp0K23NlHcPFMjp5xzJDglGtMIsfBpNta4fMFVQeSY2ZtXFUp1iuAU9ACDbiKtiLO4t4YIK0g8\\\\\\\/nd5g8L6o9ZkWfi8PJ2QLVtwq\\\\\\\/T0E3zYLPN+RgRGmaPjAEk\\\\\\\/cVUEFaUYuNmYwiljyvvK6r1mT+JLTh9mM5\\\\\\\/9DmdgbC\\\\\\\/ByJlhCZiI7\\\\\\\/vq3SZTJvHBSHNicmLRc7gIUR2dOhslTjHHAbPZpmK90nwNWZG+oMkb2dgfQaZQ1kttPQ5P+VJcdefhoZa+K94JQdwBmj4WbVtoFaQNoQYdnHkdHLrSv460TBZTdHXHkI7iEa0IHvWFDT0j3pDzsJ8yKBQW10P299dt0gZiq7mT8ecH1unIwrGNZf5YLRkOggfRuDL1F157Epa3c30XmBIt1BD9WITe98yZlXzSPZSy3dr5YLcvQwos5ehZ+TByBY+BLECZQZ1WeDaeo2pxdOBUFAvg6HEqkzo4RVnql688L7nN+07yQNEBTGG99xvBUd6v0dyt131XlPDJd30OQpGCvC+4SN4VE4ik66N\\\\\\\/m1fYcCac1DuqOAT+oVn1Pgsx3qZoAZGE\\\\\\\/4IZaO+uu7FiB1jXGIUaWV6Vqf2OpbF+D5Y8NYDgWKSQ8DuJU7fUXZTyuQs9K2oedH2u9dY5yQzg+39b5r+CGI\\\\\\\/vR68Rk2z2Z7gS+Y9rssYiZR0\\\\\\\/0mp+shF2e4JLMsvyKQlvO9+UEH4Zt9+e0WXmWOcwnY8jlMiF7XCzw1eQlOZ0YpXJZOwE0UUXOtP5se0g5lkhEZnJXPjmNZVRKutcHzJMQZdyd4O9tdZleLMpNPEAeRZCQj1+t8NaihH7XkIzy78e3Oe2NNDkqJA5QAQyNrqDaEwKFux2zzMnXzxyZhkfRHYGYcMp4pUqT6k9\\\\\\\/8xo9wuDZwgThjRrw315dD7VrUXHcEJfinYX22JNt009OOyIMAGb6pNG7JzEtCy7os3VF\\\\\\\/RyIHtzTQes9Mbm0Z4K6LXvgAgsRmFPB+y8Di0cETnyiRnjIoYGFK2nJapjKylaYYAO8Xh\\\\\\\/FEeXwIs2bGOUxFtqheh\\\\\\\/rNsVnCkzMiDJTgYX+BFn3Eq8ilrylLQ1BO+YzUnmzAnDmNxVzmv8ePmVEAiZ6gN4bbraJsj+0llSPx7IqkW+8WfkTl1K3NBXKNyveqNcJVMk+xWjGB4kLlR32Wj\\\\\\\/NymFtBIHaRv+fL8qUnT9HL6SkfHMIu+vhDkDKFep0fbcCMK+XbB16DUW35wBCJyf1SZw35YSrPBBEuyI7j7JSq1o8Nkmam0SPJflkb2KSFXWPDMujtqKXrHITMTQWU735\\\\\\\/f6gMPiP4IhaK5jSR2dRJqd5IWH8F8qa03+UAKAcQgz5rNKrcdZBAooAVGQD9Y4kdB43DaUHy6nB2T2fjVaXwFgWYjSgtRbp9Q6KwstKdUTor42yp67f6Xa0nHGLfHY2PyJ3Gdu3O1np6Qio3MjLpIF5rmve8+AtznD7YohTOyffK\\\\\\\/U9hZ0TXlVSL9aHjX7SB2Qv4fdqZEnODWJAh3krzpeZK1ss9tQ6q+Wlr\\\\\\\/1bf2eAlc9CtguaUCLacqG8Ag5M2tGurJ13ZSOMUEmNHw8CB5jb2FKxZamPEgNOs09gpRREAFFOLmdtZZjT30lBT\\\\\\\/9qyKId+EvvZi4ooi1iuoDw9XnzHUzyD62WzmuYIiFVHKNINpcx\\\\\\\/9YEu7WV73ypeCuYFNyoHMF7haRXvPcnF7v9lZkAUnxkZ4sCV1b8U9GGxuxAeZO\\\\\\\/4OfCNWxlbtrawMpObLl59774VDQ8DifvwNzqGuL5cDbYbZMQLmoSqUlD8HRE1Jabv\\\\\\\/c\\\\\\\/TfGEMnJDzPtQ3KJF\\\\\\\/C61SqKz3r1a2JOkPDS8eF1lH6YIMlnmxMp116A7eHF49sVKwdfG6eyLjO+M1c5LiUzMepksD0tuTlYoMcboxuqDGOCI2ZSPCOKaBr4mqp9lRRFFC2DGIH6OXHWyUh28NOb+7oLKv7c5JbckFveqRQbDGOOcD2dGm5TfwlgX7kpXw4h9AqjXJnGvkOhr9InGq+n4UwGQnoA7EDRGC8pepM72Xe\\\\\\\/x3NM+SFeIImTUfDz8jIVW9OXsFV9SzgCz61xu6AjTw8l4XDt\\\\\\\/t\\\\\\\/C2SSQ68WgrJmKHKsm\\\\\\\/WjjX5wnfOhmT7fKMFqjPN6EvhJ4a4PD06QuHzv2GW+F7YncUOcQagdQpGYpquIm2ZOktr2VsvoSOGS5V9D3WBdz9I9XHnOiFJ3zGhFc7HABTx220Pfwp+7B7LhqrrUw\\\\\\\/KHHI1g1yeG7ZLGZTJThc3U6LN+c8AmdNnIP70lSNRdsMAktoqBB\\\\\\\/oBGYCErtyg5B4QXedYjjoYsuaU33gUGitU4M4vJR0oA7Vo1+MfGTaj5iBlBjQ3grgksJMNHeV6vtCwJbUZy2vd0Q6fSatQW509rZiYC3gcylIGSek4JCcpYyryyOJrmCfv04JhgN3V8vvU8iY3OU0qKMZIs\\\\\\\/gRwA3xRUULYJ+chNucNxv509dyWXTJMuGroDNZN\\\\\\\/kUSGGhEmOXNprNpEcgYJF0fqoQTNprRn1yUbXW1Kqholu4hELyW6xP114FaUZZ3w\\\\\\\/CC3sd1D7i8dGkjJWwcjg7vR2kCGVPdRi3DcLy9PTKhH4pKHLo3Z24Zl5hOrkwGNFENH\\\\\\\/MYpmJl+WFyGUb+XdyDdaSBW892xnP\\\\\\\/XJwlKvL6GehRnxuMwCScscDXuHFSStnVb9xsamgQ56sSF9V1MqrtLRSo9Zro33EfYOQ5eZVp3s15EjaB5ILAKXXOM1LqmccV572a5OGNd19zDrUoAcTctUXJvxD\\\\\\\/35DphYu4fwIkzGlns74aNv7WWExUg3+OYUYq4ftBvEzkhuqRAeyz0TfaaB7DTHY2aF2yN0CPx7PFqb3AnFhymKUdiiQJyE5Fmth6V6LsMftY9XRuKF2qBpE9mhn5yd9IQHPGD1Ip+SY09wvOzlfSi3NJk3KPK7\\\\\\\/leapaoGSJGcBlkVsVGpqujcVXvcuhNnAdrAeoOFOqEs3cUM+v0fyLZiARAUBnI\\\\\\\/dl0qY20VewAR4dVe9PXmhj3iMUdmVpNdr27Yo8NQ9jgaawEr7Mn88ay8TNzdKSMB18Tp03d3atAsPntSyGhV1nGP1hT\\\\\\\/K\\\\\\\/gwNT2vcDNKNnCD4l57Itc5wOlPf5Hwv\\\\\\\/zqy\\\\\\\/4HeeJU1DOj3BLGQVCELEJW04SW74u7fYnoX20ei6P2Zqk9saiZhZnJdsuVbVyAVhlp2aY5Gtxm\\\\\\\/ccAWhDBLmhE0BZeCV308nBqjljSOHsUg3E\\\\\\\/HtG+mwIEgHTKTcahz9UaPNFyNe1wDA3JxqghOeBtQJabzAqkLv2J4+nqZPIkqB03VhkEBpuycOLD2bao2jpyULmMGYgBjViulOLirdh+QxII7f7gQXPtpG7hSbvuwniIL2rdHZwZGFdz38CfUPpBIkY1Px6UE\\\\\\\/pd3KBf9phE5b8\\\\\\\/5iGMvv16zaWrKUWOwihu\\\\\\\/+91W2SCixvLZC1X+KORxU+lkmr1ElZLACh1uxTWqfrLqEIPhup6HFXWWKeaiBdLL86YJseJAR\\\\\\\/clOj+JxvVuYOWJcuRaPpxkziz3F1DRtOY18t4zKQ3ULXg4HmzjYwIfHNHQ2vp0MtFeKo+87u4Z12\\\\\\\/Vcu+13h2Pf8SNbXtaJ6g5KbOsDW5dK2w6b4QOa9tp67YewSV8XicLkgvqoxYVtTidLemAO9ENu1YWOKZVcmN4moHgSx\\\\\\\/fx4b30Yx2fK\\\\\\\/XcfTAI6YpgUjzJFnQq\\\\\\\/85AouF6J2fEj\\\\\\\/qD0+Y8pnf4oEQd7oZEvh\\\\\\\/XmtsZq8fzgMIHVQwPma5fQ7AzFSR6sL2AM8I8Ttdc0DTNDvL15pysyjGjx9SWeeSnLyiRBDqeqQQ7r7HkUVdQaqmSaCtGOZUE7LemFoD2n0cnaq9umQGBJXF6u6CExVL9IwZigUZt\\\\\\\/RaQ\\\\\\\/2Wu3XrOsp5myo2l6PutNNPYKQ0D3PMK\\\\\\\/xQpp6vAO1DkDVwktTyRUDv5GtkPPkplDp\\\\\\\/12+H2b4I5o0qOAiyIVT4MajrR1lJeeZLCSt1J+FX+OJ7fX5bTVNSKMXtRuixseUe7wANbAjpi8+zzVX+bJk90IMbylkicAIzKvjuan6fxqdhuygyU3hpIl2KC0xX34qmqJ9DIh6LnM2Uh3ncMqDtXGOcE\\\\\\\/n3SQ1cLpes8mXNCIun7zpTAVmPA3qYdmw+yZkUONl1w120umH+\\\\\\\/1aeWGINpW9d6sqXMSJl9ZiI1i\\\\\\\/ME2SHYSbWdGBkXHpiQR94vNacAAN1CrrvSBP3vdf4J34mG8FAxQBv6Gcrcng+lzlrHgm9PkJ\\\\\\\/G38S4y2vYYXQh+ZcS31pxFb8ZmtKSjUeeYbnKpsyT1AWqkorQKMNH\\\\\\\/pS8VmaSI63f55A5VmGDg\\\\\\\/gQR0OVOJedFg4JnbeHGcuyQOQ1n4IZ0XH4L7Wy+CD0Qt8Oc6TswHwOqQ0ZxKTAGcK1\\\\\\\/1bul6spdOa4RZXBO30VQoudauUuJEbOeGZmIdanouL\\\\\\\/zo70qMaTwnLkhiqGfIq50i3Xg9Mq99hX942HafpvtPDo6UFqyGUzRhMUEQ4FgUApr3qWHPFCm0tKrMeGrmgmg5Ovz2ilrCaQqVIslEn3sFeFyAJS9\\\\\\\/ABKiAoNUNb3M\\\\\\\/6FEUWrFFXBXxuXEcdJVGE9VHyaS0IwtU1ULwp3e4e0scYEUXI47yXZ7kiuwGprfWC881IV6gAmgEc9+w22KIXQ3CvKOvZR+SeghljcYPIx50ppBljUhpnKuq0g1hQ1A4M9jswH5IWGu8ZEGVFAJ7KyojmbgX9LRKYjWcHYSr9yB+2AoYM4+TmiP+DgNS8N8jMJazKg7k87YqZ9lDaWgzKnZCzCBIaRxi4qz\\\\\\\/PspGfVQU0h0eIONxTxGYQcycUvPCtEPec1wnpxohdipCu207kz5Qn7KDRtIlBFvxOxSLnZKCBpB3BX0bW035078LcbWLMR08Ru4IH9DyVMYeRWU9bcj4MpGj2eTXXWxzB9ukU3yAFSYHT7zK5nypQf88V3mf584X55vGJ2HtFkit1iLCr5Yr\\\\\\\/B01fJwAyZk76IWXkSa1MbbtH44vdzPEr9UyP9RsSxD8PaGitKsoRr13PMDiozCUlrXr\\\\\\\/0GhHVNBw2QHH6o+MBa5pasaRdA2hWzos\\\\\\\/M9SOcmKVBUAYNFi92Y\\\\\\\/TUFqsp4rvlD9tpfCXgiUZJs\\\\\\\/thmJxxgs2rR8SDRwFZu2COA\\\\\\\/KEciD\\\\\\\/fX9xQ5PuBbbrmN1dFDhDuEBg3Nk01NYBXKANfQOqPwupqrgHEDmiNyIbbigeJ6000eycJQxnun+y6qchaZD7vo88AwZ7OTpYBPhpn1jhlKd0ySM55mDdfYhSbWVR888SjUsaKBSOkDQaBfY8CGIIne8vfo3Y4XroMkYCpMNAwy9zk7mnSnPLrHdlnO7emMv+FPHpzqBR5spIFqrNZXEapTi2Z4iOMfXuAsTnYwl320rvGqG4CnpSqoX5wgO1Mt\\\\\\\/iocZ0OTk19bikaVt74PITT4Klp7Uyg1PpfgXmnYXKSiFSKlFynhmnF5TXzKkNCGMKSJBa3O9Zw\\\\\\\/dKWRIxWIMuQ6YU6OATNwPbCLFZ+2gN9yvoGb4a2nDz3yUhatk38WeGsFnlP\\\\\\\/bNq24+J\\\\\\\/sVQAAnOdGIpr3LWHS2nqOGs10BYlF5LXXmM1Z9UAQt6y1l1\\\\\\\/dLqTwiCHTVYGHivSKNRYOk3nbd5rV5qs1kGPRFxwNRjqXlXschbRO+GUe1gNualhvZK5SD28Pf20RSwAZUBvdfM5\\\\\\\/yrKvZaezp4hrE8ZCTspImZB11yaB9itcQo36ADz\\\\\\\/xf+USJbKpIDkZCvTf7seGu+IIvd4s5dqHVqmIXzUoTwkm8x8wy0MbqDcyZE7WIGcltJRPU6ocSu\\\\\\\/xbFqx\\\\\\\/MtNNGznzTMGGEoYAqMgHVzdNaLcIcYbQdla0p5wXcdFU7b6xom1cxj8iOk1IsbKHJw8aOGYoxIIoOGwwtsF9nSB038tqinWqhkFjvL9UWYJzby9DBG4W1Ye+lXuhq55jcuv+KuJFbTOGVYKKtL96wAy+in2QTCFU5Y6ZTzNRJtzJaNoM27BKohbM3RXxxj7BPic5KVImfXbbGTftzTI7moVj7oruq7OwVj9udTfCTyEdaz2ekQ0cRxy18s2ulo7xtcQx4q7ciNQ024PHpi0bDzbAD4CuqyhRiS+numZV7EbtItUyki5aU06Atsw9enmNv8Uwg\\\\\\\/46uyZxoFcEUF0sGEh9YYfUdYCHpImpRedMR4mJmUHOZjtTswUNoJgW55R7\\\\\\\/utHNFapvZ8w+L0XHp0i2NaQiVT2OdnOeCT6CxiiQXOs2OS4iPCDNvogvcx5gLN7Nv2VZBiQvqhr9tlRKNZLlBZtrsmZLLfEwDMf7QqRRydeLFmxSQhf43GDFoQDvl4ZrptW+FMV3I9GYtoP5+IjWRzEgY3MS+yDpQ96m8ey06oUezlshG\\\\\\\/YCOa2GrzR\\\\\\\/h1snIMDv8UydDuolWxnNu1178vsxUvPjgeuBDLR0FyR\\\\\\\/yng18anr0svlkllWinPbgDZLOM0fe7gRdfDAgWvQSuEWlyUMlElv+yGRxMcmzc6hfVzfyOOB2kGaZJCgVOFOjHv7UZU08z08v3wLx4kSQT2Mgl6qTNlOfSZMIb2KA97hXOHY8IjgIpXfmS8MLZ511DDYVEclVnuhj8mjZR51TTMUsJ66+++X7fnVqAV5Ipzd05WVEpF1bMRI1I1FOwsmINAqsR003fqyz2+\\\\\\\/PGMLiHKScYLcfvQYf8ww1WjENaNf0xtNgGmxort8OMKUXBsubUtS9X3mWQqMyAHvGbUxsJ2cpE7nSpvCnKiw4a9nHVY7PRBw1MZktuNvlNdxTJGR\\\\\\\/n+Hxyh7CohmX7u1nJnNUsKSDTjAanTxvBdB4rQSWCPzzxdOsHNMjtySKgrYYd2a1qoP3v3nHA+EHKh+gmIR4UD\\\\\\\/itrplrpZF8SgONNmxV15ZQgmZnROQjQkYhrgzDlZDQ\\\\\\\/uVmR9pY7PggLXj19zvkWROz8o32t3slmwq+ktmGivzDkGZNoV9uGAnKVRhPdNr9yDBFjuFVSnXD7Wp+1VMt34G8S4Nyfbym1kJ5SEENbAlK5lnEk3KMj6BkRhPosorhMVjtTKGjw9m8kX4pQqarWQZvp8piuhmggJKwUajMNWvaKeKAPbcE2sCrDiQAyGW7oofXpLqTtUDxtoLQyQIXqZHxXeaZ\\\\\\\/v1wzYaR4VPSfO2YTXdjlbbzqk67FMskrp3Dd36CoFn1B43sDJZA0GUBtBupM67i6DemcnMF4aPlRJcWhktJtMw8mEqnME3HZzOFtV3avF+jUKeyndxVyF5GBn6pMS1yoDko1rv03TDLUSh3zIVIEPWnrenxS35v6g78L2uaUjEDPg3NildJqnTtmw0ZUZpVx7+p\\\\\\\/CysQ4fEn2N4TGzJVA9EFXRXmfQBU2Mkl3TdgGFdNdPXWJ6\\\\\\\/EgXuD6roUQQIkrAMZ3f8H2u813KBvPzDh5maMDzBWzlTN+Jp340RBYxsSNj8S99h7zIjjHKRsBfsTfvBE1jBX9cTct4ms9zFoHL6Zm36cuSEWYoEAX89gLSqC2JD6MAqgG8LdfuesCuBrJwGOfi0XdUn6JTAAN6Nnr78Wbi8BiSOVBoBdywrQYQX0nSsaHuptDBZL5X57JLqsR6G8EIcgld8q9b\\\\\\\/BafN9IcdpAK0yJVOBActH9+MO+m6OrTuVQ0VS9AqJgLI1Ku3DnSTx7TGifqtiBCeyIuLawlzZ5OjZYmZRyMQKM99WOCvXlF6ZY+nqsZ+zAgxrSdHAgS8uTFtZP8fMagf\\\\\\\/KkjUoLdS1i5D79c9INOJj20TGm891TtNTgLCjNK9ACqvN4gxoY1lYpD6dtNs\\\\\\\/OFkqPDz0AUJvHI15tWjgq7AOPdAgvnu5FZABtm06hAPh6tmXKYx4pmiUUSKK8xNW3yf34Pw\\\\\\\/JX6RZEouZSvbVeoKdcmKcFO\\\\\\\/gutT2tV1yE3385O+p\\\\\\\/OB9CpChajL5F4Sh6i43Dyh4obw\\\\\\\/k2PN+iVMPL4Fj2R5GQfnlgo3fR+bUzJBp7+j8ynWTMu\\\\\\\/Efpkfexc8v4KBxzIBjK9b3M\\\\\\\/PVb6a9Hqc02DZmaSDXJ5AJpCGyqe8Kpg0pO12N2TGq6uU8HRMhaX2zK6hukv182kM1Kumt5px7Rfm4JqgBTHaX6sGs7n63IG5HYN55q\\\\\\\/qLISv5c5gwglEC3rePT6S8h\\\\\\\/gtP1DqgVpSbuohkDvRd7T7ZtiYFJEHDB4xsE2EsEELnLMFplPzyk0ObBEZK\\\\\\\/7jNbZDK9iQbtgsHFC19sf+FlN6xuGS5q\\\\\\\/y1vnyrtgUoT2uBP+VgalaooQJVVbpf9bnYgCoEqgspxq0\\\\\\\/duhtGf1PkckcFsniZNNuulVWHenjmPRrvhXAVTZ1mlH0PnYfQt8mRlc\\\\\\\/ln9wh\\\\\\\/epPbvNJTr8sqPSP5OHxuzHuUlNmaUQfAQ+b3Zi7rd62efAM3i\\\\\\\/N\\\\\\\/5tYN1JY6QrUOu5uqFSVLv7XJEvb3Qrau+cDN1QkLBwuciEf0p\\\\\\\/QE8UpWgJyEjR1HvZif9D0HBQpsUpcIa43MYS1xWFZJU7IphVJQUeeWL0scZ2BLH7mkbYXBU7qVTRRxss19xorx7DnIFAE8\\\\\\\/w0yjQZzUfZDFpyxC0naeVy6xuOwoeJYBzlcGtZD1tbQ+PmvhebVnegB7vK3SJyDbCYuAN9DU0OM9JZJN7Mp4ZRWPPcp6K0xS5BC0PpdLKZU9oWvhNZieVmuWESVtn5Au5\\\\\\\/l4\\\\\\\/I+x1VKJznZiEuUqSzWtiXqO4J9+sVMA9JFplwWRG\\\\\\\/2cMVRxjBzUtXgQheVQRFlwUPS9OAY5Sj4W1mJx6seWLXTDI99RdKBF1nsFwy4dolv7SoJLz6qyAG5EPlHB+UvZhPV7qAvJejWQmlxXQl6CkwL6eA1KUomOHk+PKd85eJlj9ieiyWhBD4VWN0nWYquCnIrBQM\\\\\\\/IzcW8TMN2eEndMSPQXusacfpRsRjyiTlJvmcRYinVHArR9GlezEVKbTOJS7A00sYP0hSHzLDiwDw5nDHY5iIAWe0rs2\\\\\\\/NQqLFnNw4mSo98vVetbve93y4L4zWvWDaQlBpBnzn\\\\\\\/ypyHKqU44SHW8CPCi3hwADZFMb+qQD54ahUgU06Um5nARiIQjCi65OraSWAc4Q2lv126XAbskdLNJwob76fqU4dXOlIweUcqVSQS\\\\\\\/5gHqWiwXFn04Gr2KBoHrqRaV7coTaYDOteYG+hABflFx90H7eUM7aMuwMvODTHrofAthlq7yx\\\\\\\/T7qomTkDkY4T\\\\\\\/Q96mT8aylXyHg\\\\\\\/X7diJxXEtPnan1WP9PWa\\\\\\\/UTA9ivQSvm70+PGNnykTRkOW+ML+S1TRb5+tJVXZjA6iKRnQWjacK24jHIaQQnJisYVFKO4JbZSGAFRt5wTjvxoo6504NUS5zGyYlscvVF+YeeGmyenoc+k1EM+P0xOs6+BV52GzUQkA4NMp345WBODCJoN\\\\\\\/ipAE0nHQ3hBXmVS\\\\\\\/4GO8fYwdSFuhueBcfJH+VvIocLUfPfhfqeFJ92tkLUwfp22bKiE48MHa6p5q+U+CWkyh\\\\\\\/chnkFDS22gx2bWKjbMv7Pcd6Fy5EF8baG4vBeKePbAK9PuodCDEbSsgq3kE6i8THxibUnHVzPx9dGTSCYSAhN1Qd71paEoQSx5xqYqebxGIx+gFzQNLrkTzwbM1QOImgaSNucxQeyMFAaalZI9AMtlhaKkn9HqG+ZhC1s9CoWNYgp1i4OaomKLCeB\\\\\\\/HuSRP34o78i3h2YBU1NFnsLIKyr\\\\\\\/RpQPRY8owWVIdHETXt3qivaaoSFUZkwbZCjqKYtEd7yWmZkaVyP0kAzLbYHGbcBW4nV+MNnRNIaBLZ80hRdaDQogc3nfKUs\\\\\\\/03M0qcjD2rcW6dZFmtaUH4LMNkLU0peqwnWw2O0d\\\\\\\/K6pDbyn6eOJ79gm7g6elPzvz3OB+4PXzZUv98g+dB6NuHhzMbFPUAkBqtIAH+hDIe4ulPGVFY7mDRYVF70I4g9co0\\\\\\\/\\\\\\\/bsISe5SwC6pjSJyqzL1wm9MUt\\\\\\\/bVlDCculLXG+I90mJUuvkQlfeaI7+1m9Cl045HOXRruPLSHsdNC65Y656Sn7X8XMe9RP7LcQP\\\\\\\/26FAWeDUU6F3iFyjUYe3MjxsOOhcg8xRrPtDuOEfX3c9r9pfvL7nYi+10FnU2Qt46ikqbPAATJZky3nyPMrF08h28fsGNtMZzhOIlQdjPBZKsMiFSuU6Q\\\\\\\/vYkPoD1BqNE97dapwpMH\\\\\\\/6h2cLx8NpoM55ZzzYL916vlTHm\\\\\\\/Dt9+Ii\\\\\\\/EgpVdJZvy6YE5Nq9S1mGq1u\\\\\\\/dyj4q1izG4q76uLNDv0zDvTjcL2+CLNPGTt\\\\\\\/7n9fy+fEiBS+RA8j9DTOHrbFvgUy41q9r\\\\\\\/KQU9eMS0dFKyutAKQwbwppnP0sK4yJDKInbNTv4nqGPG\\\\\\\/Drb12rr1lIsibfb1H7QaIMxkjfgDfIWuPJFX+IacW1yFbTXdbgnYOoCrUOYODYKQYayADyuLblFsUd\\\\\\\/syCc+Nwx3sGW9+JUMXUBIij3NhmdGzWrad4AyPN7c1bGhaisVq64Xcl3p3UsDyIa8vA3D7O4Pm5YUh2LUwCEetCJO6HLWf903TzV20cGrXyZ5FQTonjdqPH9zY7NsDeA\\\\\\\/kLYZmWxbP32D6FHXBqc8sw8oQDBgNs32Kot2uMgfm7ePFxgy\\\\\\\/fmE6LSbOCcu+FaMtEBhDI5nEMrgWBnGz0RRZCf+dROMrOqA0cwHulHgFerT2nvud1sOpK71k4eLhLVUUw9zqS1NEfDNrZW2E\\\\\\\/IlcgdpJl0qLSBha4RvBF0gQa03aYNILXZaVNOsXzeQyxMgDtTM0+7GFLMHyZyqO05+ZoiQjPrfjwZuJcMHG033uEoWJIJcNNWAecsu5wl1Dn1Ty2oSAvC6ZcWuKLomVnKUVkMxrngoysVllFNnq3I4C1rMm8Ua5qIydK59nWaSQeSD5FxMpcw7l3g3XjwzIhiHwstETveOarMUYfL1qx1RO6Sz9WWaSNkQHC\\\\\\\/cbigucBy2hA7tkcakXYGYq4IWL4cpC7zlfZwnpJSTXdl\\\\\\\/0cZ3LUXSNIGykeal1TDGHI8kB6j\\\\\\\/j08yPkTrECRYvbXOvK8zkHlMNGYyvH+RdkWm88XkdlLzOOIXfb16OTgf1PRrFRJEAhaw9COhKT+6mUKx1EQUDl\\\\\\\/KKa3e70VkLg063LXKd0vm00pp2YY+ffhRVrjm33LBVBsHVKdVlfaOgN+7FaGlfyicnloRzoJCXlWxWB18O11yfd0aBLqGI8FZFTCXxwGJ2gWkwn+I6uRnXLaXfxbWVQFvEWfs2Ge5NwNXcLQD2ixFkdv1za0txFEy3lyevkoT\\\\\\\/gv77cBWwHylvmYmrgN\\\\\\\/FXiq8glo87Z+tLMrh5rPQ7IYMzrWvmTr9MwvPTn8BC1Fnrvyp3gtIA1kdap3bucOOawY90ka\\\\\\\/uQyQwISywMc5eU3zPQRe2rkVLmghWzDCjMHZx\\\\\\\/iaeIU65pKXLlz+AHWnicjSmQ4el6hgUngUGTtPc2cxuKtqcvsGZ51k4denXwYudiJqTowotHhedL0k117lT5gMtMtbHIWWaVIq6YiMZtyyBsRbIB2JZRV7hPSgD4e4csCEUh6q7bmU7xaxyKpc8BAGyBmvbIE8sYqs+lPCj0mdkf9eNAYMaPZMz6lzBtUdShxRqpcPzQ0hUeWQtzwPpFoABl\\\\\\\/3fwPv++mAKkuJl8chBhjIJK02k6NPVpE6+YtKVZrdzlk6LB6BrmjnCnnAduh0vCo5q3Dn0czqMDS+K4Kn7S+yogbH2P3Cuz4A07DHtkKx42ywXFQH3AcLK07ihaeT\\\\\\\/KpuwLO3AmBZLeeg9ahXD6F60kTCD7zOauH6xxJcYXhtWMFVjkbrSUG\\\\\\\/NP2kN5zdFnhx2F2JXfi6EJS2TXXIGl85jNRZ4OnZztAz\\\\\\\/5nBCNriPniJwu5PwZra\\\\\\\/g2Z3Pv2\\\\\\\/5HuaT3FHzNspOH5dsSicCq7LCzMPfWiQKhXr6+GG5R9KgiLH8Xv+MYfojtDxwGkGDMiOK9Q+sbLlKMz0Mq1XQkItFQu\\\\\\\/PWxvODaDchu8Mpd2PKZC+RGSLIOMoD9tOY6KrlOxdgsTzXZfZ7v2XRHRrIKiHY06f2egYylvzegD1+\\\\\\\/\\\\\\\/gpLPo7kFmDFC28QElGVNI6OO1jTuY63OpzpAp5uKLAigopcuEopziOm3t9oGH0tOBSvj3m12PVZLmsEsodfoJAqVPu\\\\\\\/CJ7NI4WMN15lzIzEJPWHff\\\\\\\/PdGIJvQAVt0b4N8e5LTKaWPpW2m8\\\\\\\/wyKVSYESF1alpUCrUbZ8bfFOUIg+LI06bmdojdtT1bLdj\\\\\\\/v\\\\\\\/KruGol91BkVGROgat6VIVQCOfAYmAIkds+oimJMNlTV1uwjLuncyqqna1hv+GWvazkJODkZQNk64gEbtA3OROeD2vMMLaNwQfTMslDBaltSNY6IlntgrqEmEdFz3HcsG+oy3Fyf5geityqf+J8JOBrfYtHrHimh37HsACc+SzddVVhU47e\\\\\\\/6hCzlKod7qp9jTQQZJ59aohmjYcx4g94nTXJsRb4fFM4BOfGt3bPKE9Y5cXFR98bJPN5UsEUzFe8dk3xHQ0O3uQ2Ymudmpd31BOkSvH12mjRxQfrNmAAV97FvQV1cjWFhWaeV7\\\\\\\/bIwwDJdeS0f9Du4qiIrPH6\\\\\\\/O0TPHytZNqYUA3RQmj89CnqfkaUsgsJelYe4Td42LbM7Nm3MkvNS5pYvvlknmQgSUGVXpEOwC8d5AX8S04+5rMs+JpsEqpIdGWgElucjwVUry15idhduOYqxf5POCkaDVE\\\\\\\/g\\\\\\\/5gO0TxsqaUorqcZn8C3l9gtnZtcVu8fnCQjGK\\\\\\\/RxAtOFDWLyTLPGYLgKzJKbsQfTipCj31X6Jzf0kL+PF8FaGRuzhARyi8iydBy0UCaptnvpEFY09lR+rvHHf2siV18d0Z3CWsjyLMt5hPFMxaoDOjza0u3oMfNPUzNiZ417coe4HbiOYtB+SvJGR\\\\\\\/fq6S4LiPlPBjHmmtF5lx3TwILg3XRwe1lnvLgei\\\\\\\/77M2MF3FpCb3nAS\\\\\\\/Rm9N4WFhLDWRT\\\\\\\/VyE9AJEmSPzyukLf52aPfbJbwjUlSIaJtWVKqwHWEmTSryq4UhxbgHHsXNrUrGI3FOqEefhUhi4\\\\\\\/H63NoNhpDAHC9wYzZvZGX06lxHIc1VfXB4UHFeQeKxT+CjoHqafv96HpMhWeu1t60f+Q8ZsQeATW0JYkuikb7fhoINbGtu67xHhSTPnog3tldvvpbEFtubqT7rX4+fBEMU1YBrvTWws7O0mU2ksNml7MAfet2ffGkalzmDNa1Ksy9RehdMGUWBQxanywmWDJqT9zQTX6xrVwI2QXHvTgIdUgfeDLITUKADO7NIUSYqx9BGwNfGTMT+V8A3QgO4XKE+\\\\\\\/SV5pWlVlcDwPEZnTWt\\\\\\\/LpL5by5ec3b6CEiIxDQhjs48wzUCxjiVDh\\\\\\\/7iFJJs0UBklMcSGpLZYZNw2jXQD+u1GYDSqu3O1JZUbqZDfriWYdvKUGjxtrXvUs6G7UJh7Tf\\\\\\\/uq7sbVrWGWQCLYAMEsJc+BsC\\\\\\\/ze\\\\\\\/MdA7LmJ3GG4gR87Yk9y+p6L2ROi+2jH52YMgbrE8uWeF1U3q2lkuDRutx1MNpQ73IogrcQmjAbLi98hQCdG1l0i60eyF\\\\\\\/AsCjgRuMSJSY3+xkTbJqgJl0jLo2NB24lF\\\\\\\/tTch3sBKN+LjvTdRi6MO3RKCgK5aZL6WWlVQfD+RJ17d7+pK9kkhqayco1LaeCDO3V\\\\\\\/3iDzkvzxUFWdXRU+bPzF229fDIZfs3yVH2dzT\\\\\\\/vXwfN6ElYHuv+Tu4UtCjRF2WqbHFPOvAIF7HCf6IuPDAQiWsTKCInu2TdO+zFlQlwBi6y9CYhX+lGRJ84Vnd1eRzpZon\\\\\\\/hBBc84VTqXpCicq303DHK9+AH1y61BlzHEKjw0XCH6ji681YyPIvuZzGSkuB\\\\\\\/8I+YtgB+81ob0htg7Jd5TG1JCUFmMGOHiAn4bRzYNb9QUSd5mlvXsC4Q3TPaHbq8c7EIYXQ93W\\\\\\\/SsG8o2H7+7EQMrIFR0cYaspHGySFhx+ibE+VI\\\\\\\/zzMC5Vy302c0RzelQ0a\\\\\\\/M1d+k8pVECaJuTvxdnlu5wZl2a\\\\\\\/GIn+CZGrngtgfrVCgOoQNR5\\\\\\\/syIN+ic+rvcGoSfloAHo+by5paoEx56YskdIP\\\\\\\/pqDSAq4QohAyrbf\\\\\\\/o6HP6IybKiVFdKNP\\\\\\\/9Q8rsyEtwSxa\\\\\\\/pO1BmQD8eugOm2jxl6\\\\\\\/7n+Z0IIh+FF1rrJjr1yaGayzI8RdaCrh4P3ewblyAelr+d3qQKb+gObK69J1AxxOLVrNYjpAOlpr90DhMrvIeQCgEAtHXgRtHMSGdAYIahVWGfhTFyQng+2T2fGobaWnVdjEzkDxofMAS3de0WO\\\\\\\/vwSRyGnuvIj4eTEetbKAxst1TAQJHXmpp9s+uVV1Wzfzl+Xo5dfflHt5BmMpmFnupRF0cFXK46WK0q7ywaISjbOIUE53LYEv3MQPiU8iN0qR68P\\\\\\\/7aEsGj5oDZTcLXowlYOTzvrojMOWehAcsde9\\\\\\\/jZMlmyn5v2JRGlCxVHWol5BDgfYG+6Hw8uibT5V93hzbHOtE4vKt0vvv0bPtXRZ2i4YdWGjbY6oo98bJ69Uwd\\\\\\\/yD8X+RWOmZLgEN5GBL3YTP4FK4KyRBggOJNg+ISkHQOkZUdCjxZGES48DEhpxR1cdqxbxYyh4fxk7\\\\\\\/5Ff+OXdEdrlZ1CUAYfdKlaJjKp8hmXvUmiyUpJ77JP9WNGxR62ceRYefMOiAd6qHuMQaRBI39WKyj\\\\\\\/nY6qrXNssBzMgxmRir\\\\\\\/5jX7TowWQbkril36W\\\\\\\/YjT9ymJhG\\\\\\\/nG7WSvF7a4DA2beSHeNCvo3597pTvwHg2NexK2gH5f9BDIuTMsci9x2U5eRFFoHbOAk4nCNngCSZkNA6gt2lMyrbdmxaY+YWu1GXL2SVfoG8ZzLVipooVO6xgTQsRKlfSELPVxthKRIUa4HtpkcmsVrsTcyGgbgwVYuExFk1ABhtE1bxtAw1WO1H3VGfDSgYJVwbo2tAaaKtbwqr8GpAGS3UmRTv1vKUf9cZuSzdrlvKVHv096TzYQJOXEmSRkaQ8SFEIvfNLANM\\\\\\\/nhSsevXVXCJ+BZK0L9pFGpj5wPsVl29rROHppi2CfOEuRpoNdz2d5I0fjDOoA3xYnr9eEOsAuL6BTy5LYu5QqEdP3P4YjC4b5IiKiaNekt8RtSzhxoVC0YY\\\\\\\/+UPIzTwShtlmnUNs8EmN4FT6hUZtSymqlEebdvgggo\\\\\\\/UOp9hTvX6\\\\\\\/aNYHdzsyLK3MHm0STprmMmrTFlx7gfWO0chkthuKNjKbOfCbdLLy6BgtpzLp6ia7GVw87N1SKxqrRKQDYiiCElhXmesZ2JYM6yF\\\\\\\/RzPjQiIeofShDeFvgdAyYncK6fX869pmjdMmOYGVQmmveCWLf\\\\\\\/ZlK0h4V1VCm0mUsUqWwLD+y\\\\\\\/Wu6Tmcqojn4Qi5Gqnbv1mAXQMRPKcE873o3GR\\\\\\\/YM79zYVm0evmq9rOoZfN8F3zk\\\\\\\/+3CcnLnZOr9KdSmuSe3NnTFLOyFRnCDWyzw8\\\\\\\/Xje94y7FouQJ5gu9Ku5i4MQo8BfJTdPn3M67gDCUL\\\\\\\/4Qxm7I9tssu625FsyLxpFyUD4OTlR7SpeFuFpLHQFjdL4ndze8JJGl3aepNwPFsSMWpbhIHt+oc6eLD8KQGpmJRJgDwVj2dFcJwSMhafKYGK7hH7Vaxg40\\\\\\\/IERZQAmMZ0JYsKy93lxkxCp6nVarfMhZoCxeNCgvOzyVayuXfLzMpLIw4jVY\\\\\\\/7nc\\\\\\\/1NpL4iSxTNhmhOhqzR2LjZjKEfEsasaf8zgHe+fOxAenwer1ARP6mEzJbWVUH1RGrKN45R6xg7HHnTw6puRTiAo+14l\\\\\\\/XDIb8unO0HHT9P1X7P0R5f2LgWzFzjNYR5cSAm7hBzC0HGQfrpbWuQGqGmWq6kTckPvM7BCxcTacKhK5kid4+vur98HFI+aTYhY1QyOOtCiKiAq+KmL8aJ00XuGMEL9G6QU0URIkE65lnEydSwhIBPsnvw5qWHnrNsn6xTRKT8OYjlnw6yu+3uoKKjna4G0iZLn4AXIif2+V1sM7HOZHbC+B9dW\\\\\\\/QzD7pxmXvqK5ou\\\\\\\/dHOYru34Z4+JWmOmv+\\\\\\\/LonP38fi8OT7t4nryqXE8qJV21qYsgl1xcNIoZREU5ZkcrpePsoZZ9PQAwnm0FJteIu0uVFRs2OFY\\\\\\\/u40QHPZf+tjveHL357m4jG\\\\\\\/spBs3Jz5cH5DcbkPwZ3+Vyla+pfmcScXa7YYzw2wSxUb9Fio27TAC6mV7QmjV9QFVbpwjTl94ehYxmOnS97hgTYU1nI\\\\\\\/HsRX2iYRqtAab3tJ+sva7qnwBK4SdlJN5kZsqspWeePRrrXabXsvUMxSf16t31CeZu\\\\\\\/o2Ua1SbDuJGl\\\\\\\/9djDq0rmZvaS\\\\\\\/NiufrX24YcTgFyNaeGQlbVgxZ5Fk8iIhugP5Cr6VKiGVmGaxhORK3MWi+6p6bn26kWDc+UjF9LTLDOrgpC4EbR3I3X0UfgGhLADj9DY6qfTbDv9s+0crh7Mg3fcDK8t3mSRRImKQzTYuaRBWH1EVCr8E1LSqyVyxqtIA6T00fnc58KBpuvPFfg9NEMiS7Cprmjn8kbRUQT0wjOlSHNL4oAfuiwIinM56dkxPDTQ0SoXcY3Y7GFx1IrunwVObF0FOhVQusu0PvqeRKoAlNux0J0PTmBSikqicmTBoJvGB0P\\\\\\\/F7Ln6XXPCaXvbdWWZOTS5D6cpHaibbLg5gPXmU+lyiq9cdEdyrjFMVEl3peKNQQ8QEpu2PioC8GfFaX5tsw5e6C7vLcLmB8cDyyIzqAHteoO\\\\\\\/wIC3K3M4StF67YGJFRf4ODQmr449Olsy2T6Ll7NDY35iauzLbrsvI\\\\\\\/5KMiNfAm\\\\\\\/\\\\\\\/Okm85W+4XnbdNtNadgtP4x9BkktQ34C72ZT+OagArSn5JSxuPUIzjRtpRCsAN4quNnk7rQHT0ZwwNy8auftkz87Glw94LR8ksB85k3ZslirGwlSb5Eweafv169uGfn+HHZ4c7VH2FNw1cwn1eURmdbxYHopd7LdBQDKbJ+24+vVAFUBY17wHMM\\\\\\\/ocAjBmwIRqhwf5K2pklNkzARn0WNRVp\\\\\\\/75\\\\\\\/mPNRb3\\\\\\\/KPU6PeCN2s3bncC0JluHYRrLso3GdnIY1+nuCILlArKYr3oMvouvmzpTbygZK8lzLSMr\\\\\\\/pS5SaKzB8jZ8079lrZZQ+VIg8KKrh2ovpGdOP9ACqxDNCN9VBLTqQedIn89Shgu6X56ktaNvPo9xolGMfXfAedq6lgJFaArYv6lIUhhr1RvKvBJpQtDllca+4kE6imnXjdVAJg5+lVq58Q8Yrq62iG2Jy0ugkdANpxQNyUCrtGS8nsPhijQSl1JxzkIEpPOp45udMesYZeIGsLm3CsvSs38\\\\\\\/40\\\\\\\/IiW72PwLJrMHih8tQTkhn8Oj3uZb9TULCseRUsd9zrXHYy\\\\\\\/KzfXLYoRmqqXo6Sh+rIIKvkiL6Jpc4qShs\\\\\\\/wHkU7WIWZbbF4jEgnVTIULhyVO6DdQHq+MCkSM4Q2l3RLBaMeJnsKokMA3wMK1K31+WZvgu9KpxCW0ALDwUJMAJDuHlauipDvnyJHIB80OEBq8kLaAPLRjltVPsbxmK1L3zKcvTfhL7ldDqTKYX0xastUdB2dWqlNkU0SoPhTNLYP6e0U7yGBlfxxXxmndhKtbQEwk8MsYIp\\\\\\\/TgXBMobs95AmGIE4r0u8dhIO1CSF02fdMzUXzsaDXI7x0wnUhTCpgITgYc5POkmi7Az72diHD6cVbaxnJ3hs1B\\\\\\\/mWhisfeko5XEqaxYMACXL\\\\\\\/5e1PUqjJxCQese2+FuCqvkuS5odP+pTdiFWaGrgTEpaRtZ\\\\\\\/AfkhR7zmJUQezOu3N2rzgozKbDoSrSoSjDmgOOrkirDlHJUILkAtak6LxuLNy2uOqAjsn6hJ8Xim1Pxzkq0nZ19LPMX4qtZO3P8ALQIeqqQvJvjwvPWBt8aEjhGi7xRMfIKhFve5p2BiQriZ92I7BXnLl1mbAW8IqFve34QL+KGlwAaqnxVB4h87XBabv3LVSed+IwJxBI5Huu512qxd0THbt29H57Gz+u2+rTq2l8ixLinzxvl\\\\\\\/yFmXIhYe+lt1AbIUCXr6PdtPyoqWifMwET\\\\\\\/lfI0HT3apga+TqIvS0p12mFeoxJ2KPaxo7NNwpWUa2JOtmL0TIQfzUtYnqUJ2OY1W1XHDqPudCD01y5GsrsdZKYdixVPZebcqJDrYfo7tB5j3ikNHSAM2JJoN\\\\\\\/MENk3b3QFGdw8Cy\\\\\\\/SmfVLZRXukhIZ1V68H9zEU9aNDvzPEwrhs8fFcogKDsAUuZZ8Su+TmhxzPtAjIyuaZcX8m0ekJWfAm7z4SJW7ZapiAC6FEIr2KKe\\\\\\\/gjsYsDdunwGtGkhpBNLHRJKOcqbJcJL6mnbLbheWMFBT5eVDyJFwLp3LjtQN3WyDH+LZo9niFRvkxbOHIDaD+BllAh6G7orKNa\\\\\\\/WQznGejdW2a7P+0oO8F9OarFhjgVeh4nmRfOEhDdHSdvPDKBv27ORLygJqiXrTfRFjaiBxJyRyqDPKsPHbMAkt\\\\\\\/pVCEczKmNJb0LeA7vlQPkhvAUzvHg\\\\\\\/axlXupmqjVBHKYa\\\\\\\/A7qW5YAH5Ry3tsR3JlJ1WAWMEhohZqaDhyW+OZO+YkkXG8EUn1eLbOVMGcDmXqmDrB2sSu4ihVXDSm8o4W5yQaQM5OScvXLgHghd\\\\\\\/j0fS+Cl4g6CmVv4YKWQ+QKavHUXMfqkI78aOYRMlFs9xPldeZeJHGkJvLn2IsJM6kLx52iVnkNpX6wGathmvsjjm52YcHWXgSRFf6sQJ01sPQuqjKsXxrwlMrRZuB2rG1axSG2OSGtE8vnEvowcJRU9GYRba4ebPMj\\\\\\\/M1OiD5Xs2KRAViZBhTDy+qKvFVH5ymTLIsnqXXKUQl+hhqsVzA8m1A\\\\\\\/KdvHtleiWkQ18lG9qaMpH0xPAeGgHT4Tkm5D9Yhi43FYk+8AP3TKCjOp+Q6kTWHN4V7dJl0aF5X0IgkSxvoLNxB7Zyx61A8rqKq46Al8PcSKlGVdibgJ+OCT\\\\\\\/zTtplI8KfXtHXa3pjdR7GA\\\\\\\/hIzY\\\\\\\/Aws+1gzUpLnebFfJ5otdy5hiPubTLXQM1C0Azgw8\\\\\\\/otb6ixx6Bh3z25JR\\\\\\\/NjUyuXPu0KehgfEDmYcXWT0fuInZXISAhEq4AENTx3yRjqiEa74LCHirkJ\\\\\\\/1QNyvVJJddjmnpK4T6tHNYfS5UWHuN5IGf0b0PbQhdthGK3EE9tFIsKJzWWJemy6BSAfrqkVzeSTSUpgdsUDBtsAbN\\\\\\\/wwhF+kN5EcloJrpp0fhBLPsqoNcYXPgwj7sptkm+A\\\\\\\/wzFkc+DEq8kbIg8kROOH6Ois8sw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/7dducN8\\\\\\\/\\\\\\\/tr6V31raHFpQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222e97d567a9eda5bfa3a4d0c934a5669c931cb7bd3fe8f1850243b5c1c8c209f4\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022lhUGHXuEqGsAUQVMUXqRbT\\\\\\\/jU1fL2XEezaXF0ALl+I81mhVd5tsyZpRIezm6Pe23jejqNajzOCjgpYxl73y0pMz1\\\\\\\/mbmpJ7n6zvIkQWRoEbXGmZIPp84TndSbRR9Z4XXW3It+i3Oo\\\\\\\/mUpccqQ\\\\\\\/FGyrCCoekDJhe78TmZEd71yy01Vd6HUanh1mPC53MOXiXP298AOk9NF6znXyJA5CfyK5UPTCu5UQpJGCIm3fARoz9AaL+o96Wg62dXtPL2OncXGtfSxbaWJ9g2lTzACWnjK4379x5Y0ZUaEPbIqsyATx8DO4D3GPIqi7YAODbfdiSitX+GDZbui7KbppXjfyUkKxrWAaIUxCIrU+cyj4ic2RihBHvVJKGsmWi7f2Gim\\\\\\\/qwM7UNyQ0QhnbWumxHhT3P3ilJKA5AsLHzYy3g7uV19RwCFCHHd+EB2JrfBnyqHYSkBaJqLBacmD\\\\\\\/VHEx5kKKf93lHw+SYhCZbeQkQtC6azPf9JuCv6ogc+gd+puEHGkOwyECY97MKQ84Ed2PgCVhzW3OOocepd8azuRsCZeAQxUdvF4DF2vT5Ew8\\\\\\\/2Wgg2uvwBSKflCfjrotQ17yWTRQamyFYVewywg2J63rJk41T4mugEUFXHNuZA59K9yo3fx1vriDmJ46iY4mENeChsvLeWQiKTHxuQyUhHqZlobGG4R4PLSwAACcIeuaBkftCbU9N\\\\\\\/hKL1KwPILBDGzKIt3RlUvc6IpvQUt7LfmxSMs+MXSwaAfc+mtRlhF43UsB\\\\\\\/8ELDB\\\\\\\/gi1HCWcX7XGv+SRDLi9oh6a6xMdeUyM3TleJdZuASXw+efJuhXxEZ7gG3FWJQn2OcGr5ENJrGzM95u7fqzCHJuXOE\\\\\\\/lNJepF5TZeH353zmlQH3PEWJyHw48eoW9bd1gd6R93dq84UowKdLfoLXtVTlI+rdyuL+jnkJ13Q+LfOo+2ZHm29NZ2+muyVuQ29VK40zp++9yDmoqrmfxEKMiqsOcrYe0H1RiAozdgr9C0ZFZSl\\\\\\\/mwHbC1yoy7j+z5LZuh4WZdv8UJ58QYPwKDNxSv3XtzEcdCNqOG05CfNrNQrHEs\\\\\\\/UysMGeaAJfHehFc1+uSqfRDv0PZKjl7utvAMxEHEiwBY7WuUPVYstEXo7X+kAD3dzD40PZ21YCGFr7FDt6TnEU9tByhQP50YJDnhPqRjwAXQjmK6A93CoGwP+gWpsW8W1XoefMLHM78EdDr3nsKCdpQTQpDKCSgXP8t9cBQHvgjeZvlz0XNja4i+lu2LcxR9zHBow4tyziWaZp\\\\\\\/DgHd5f3AbIRF1lw7iqn8WkLyW5rLEb+HYEEz3wm0qgNwkCQzrfOsBrAGwXuW45Dy6ihnRM0tzN\\\\\\\/V+dq2Qqy5y58KPovLWNRw6GrB6zHE80ezjr2OfQgsGZQYynyXzAP3PxZxgcbJgx7derMmKcbXnHLDurO30REP0g7pAFdE5KCQCxjTNt2pKhfjNBLku36gBax+73Hx\\\\\\\/aVYbRa6qNW4NWYyLkwx7g92ExQt\\\\\\\/aJM55mg2oSmEsyenagGfLIt0GwjyiDes7Yla5kfWBKKZ5XX4FQlX27LsdMAr9giqyaLAKFSP\\\\\\\/IVbPkpEK44czAFigJuuKNpZSXggu0Q5IP2GB+AirevFJXoU+jJpxs6P3llPHRDmGnZPHzRmJ48IVlQaWNAX+Hgw\\\\\\\/m4T8vk8e\\\\\\\/kDzESBNG+Lj2X3FJ1gTIw7Km4FjR1qdjNyO9ij4c6R0PtSYzeZhUlc+b9O3fX4faaDAD8iP+YUJquAqXPUtwhI42rMZLjVSb3BpkVheG5PpdCStf7+nSx6BlTT2XTcUvROyD4nxNw60WJ7QumsRwiA4MtLyl\\\\\\\/TF7Xq4w+OYD4LzU2yOd26Zf76T76F\\\\\\\/5oZ1au\\\\\\\/CCtK9ImX0mwvF7qd\\\\\\\/LYp5DjkBDX3NNs9op+i8J1fGWOs2Omv1OuO5c9AMd6PnrHnQa5Sqwbaiw3S8DhgLPUTXb2y9uzPv+fRpvzNpjuhyMqBB\\\\\\\/uQyfwd2rKLvlLTxWPMlwNn54g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022sNJ0lhJJS6KHQVkleFFASg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a7749382f68290210bfb96d8884e20475d670846d84777cb0dc1a72e4543ab0e\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494049 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/2811" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-JaukWdkcD3xvJgLUdlVJs40B34eqiiIOhxFeIe7SWnM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/2811" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-zLVmgTvGwkbq0ohr2mUT860VMGhUHG5cVYBgYl9bTsI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/9/2799" title="Genetic Screening for Mutants with Altered Seminal Root Numbers in Hexaploid Wheat Using a High-Throughput Root Phenotyping Platform" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/9/2823" title="GWAS for Meat and Carcass Traits Using Imputed Sequence Level Genotypes in Pooled F2-Designs in Pigs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494049" id="node494049" data-pisa="ggg;9/9/2811" data-pisa-master="ggg;g3.119.400239" data-apath="/ggg/9/9/2811.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Partial Selfing Can Reduce Genetic Loads While Maintaining Diversity During Experimental Evolution</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5414-6625" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ivo M. Chelo</span>, <span class="highwire-citation-author" data-delta="1">Bruno Afonso</span>, <span class="highwire-citation-author" data-delta="2">Sara Carvalho</span>, <span class="highwire-citation-author" data-delta="3">Ioannis Theologidis</span>, <span class="highwire-citation-author" data-delta="4">Christine Goy</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-6830-4148" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ania Pino-Querido</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-0307-1907" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Stephen R. Proulx</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-1057-6882" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Henrique Teotónio</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2811-2821;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400239 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494049" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ivo M. Chelo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Instituto Gulbenkian de Ciência, Apartado 14, P-2781-901 Oeiras, Portugal</div><div class='author-affiliation'><span class='nlm-sup'>†</span>cE3c – Center for Ecology, Evolution and Environmental Changes, Faculdade de Ciências, Universidade de Lisboa, Lisboa, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ivo%2BM.%2BChelo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chelo%20IM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIvo%2BM.%2BChelo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5414-6625" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ivo M. Chelo</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bruno Afonso </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Instituto Gulbenkian de Ciência, Apartado 14, P-2781-901 Oeiras, Portugal</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institut de Biologie de l’École Normale Supérieure (IBENS), Inserm U1024, CNRS UMR 8197, F-75005 Paris, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bruno%2BAfonso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Afonso%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABruno%2BAfonso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sara Carvalho </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Instituto Gulbenkian de Ciência, Apartado 14, P-2781-901 Oeiras, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sara%2BCarvalho%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carvalho%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASara%2BCarvalho%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ioannis Theologidis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Institute of Molecular Biology and Biotechnology, Foundation for Research and Technology-Hellas, 73100 Heraklion, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ioannis%2BTheologidis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Theologidis%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIoannis%2BTheologidis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Christine Goy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Leibniz Research Institute for Environmental Medicine, 40225 Düsseldorf, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christine%2BGoy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goy%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristine%2BGoy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ania Pino-Querido </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Instituto Gulbenkian de Ciência, Apartado 14, P-2781-901 Oeiras, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ania%2BPino-Querido%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pino-Querido%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnia%2BPino-Querido%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6830-4148" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ania Pino-Querido</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Stephen R. Proulx </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Ecology, Evolution, and Marine Biology, University of California Santa Barbara, CA 93106</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BR.%2BProulx%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Proulx%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStephen%2BR.%2BProulx%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0307-1907" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Stephen R. Proulx</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Henrique Teotónio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institut de Biologie de l’École Normale Supérieure (IBENS), Inserm U1024, CNRS UMR 8197, F-75005 Paris, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Henrique%2BTeot%C3%B3nio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Teot%C3%B3nio%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHenrique%2BTeot%25C3%25B3nio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1057-6882" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Henrique Teotónio</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/9/2811" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494049" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494049/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/9/2811.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494049" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494049/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/9/2811.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494049" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494049/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/9/2811.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494049" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494049/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Partial selfing, whereby self- and cross- fertilization occur in populations at intermediate frequencies, is generally thought to be evolutionarily unstable. Yet, it is found in natural populations. This could be explained if populations with partial selfing are able to reduce genetic loads and the possibility for inbreeding depression while keeping genetic diversity that may be important for future adaptation. To address this hypothesis, we compare the experimental evolution of <em>Caenorhabditis elegans</em> populations under partial selfing, exclusive selfing or predominant outcrossing, while they adapt to osmotically challenging conditions. We find that the ancestral genetic load, as measured by the risk of extinction upon inbreeding by selfing, is maintained as long as outcrossing is the main reproductive mode, but becomes reduced otherwise. Analysis of genome-wide single-nucleotide polymorphisms (SNPs) during experimental evolution and among the inbred lines that survived enforced inbreeding indicates that populations with predominant outcrossing or partial selfing maintained more genetic diversity than expected with neutrality or purifying selection. We discuss the conditions under which this could be explained by the presence of recessive deleterious alleles and/or overdominant loci. Taken together, our observations suggest that populations evolving under partial selfing can gain some of the benefits of eliminating unlinked deleterious recessive alleles and also the benefits of maintaining genetic diversity at partially dominant or overdominant loci that become associated due to variance of inbreeding levels.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/self-fertilization" class="hw-term hw-article-keyword hw-article-keyword-self-fertilization" rel="nofollow">Self-fertilization</a></li><li class="kwd"><a href="/keyword/disequilibrium" class="hw-term hw-article-keyword hw-article-keyword-disequilibrium" rel="nofollow">Disequilibrium</a></li><li class="kwd"><a href="/keyword/evolve-resequence" class="hw-term hw-article-keyword hw-article-keyword-evolve--resequence" rel="nofollow">Evolve &amp; Resequence</a></li><li class="kwd"><a href="/keyword/overdominance" class="hw-term hw-article-keyword hw-article-keyword-overdominance" rel="nofollow">Overdominance</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/c-elegans" class="hw-term hw-article-keyword hw-article-keyword-c-elegans" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">C. elegans</em></a></li></ul><p id="p-3">Classic theoretical models posit a key role for deleterious recessive alleles in the evolution of reproductive mode modifiers (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lande and Schemske 1985</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Charlesworth and Charlesworth 1987</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Charlesworth <em>et al.</em> 1990</a>). Under self-fertilization (selfing), because more homozygotes are produced than under cross-fertilization (outcrossing), there are larger fitness differences between individuals and thus more opportunity for purifying selection. If mutation rates are not high enough to result in extinction (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lande <em>et al.</em> 1994</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Schultz and Lynch 1997</a>), then predominant selfing should evolve. In this case, populations will be mostly free of deleterious recessives but will have reduced standing diversity, which may compromise future adaptation. Otherwise, mutational load can become very high and predominant outcrossing should evolve with populations maintaining deleterious recessives that could become beneficial under environmental change.</p><p id="p-4">Although classic models suggest the evolution of predominant selfing or predominant outcrossing should be common, comparative studies in plants and in animals have found that a number of natural populations show mixed selfing and outcrossing at intermediate frequencies (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Goodwillie <em>et al.</em> 2005</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Jarne and Auld 2006</a>). To explain the maintenance of such partial selfing, several models have been developed to deal with the particular way by which partial selfing structures genetic diversity between individuals and determines the maintenance of genetic loads – here defined as the fitness difference between the focal population and genotypes free of deleterious variation. One of the consequences of selfing is that linkage disequilibrium is increased and thus effective recombination reduced (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Nordborg <em>et al.</em> 1996</a>). Partial selfing further creates identity disequilibrium, as individuals vary in their inbreeding coefficients because their immediate ancestors have experienced a variable number of generations under selfing or outcrossing (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Haldane 1949</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Weir and Cockerham 1973</a>). With identity disequilibrium, heterozygotes tend to be associated more often with other heterozygotes, and homozygotes with other homozygotes, than that expected with random mating or exclusive selfing. When highly deleterious alleles are completely recessive, linkage and identity disequilibrium can prevent the selective purging of genetic loads as outcrossed progeny are much more viable than selfed progeny (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Lande <em>et al.</em> 1994</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bierne <em>et al.</em> 2000</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kelly 2007</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Roze 2015</a>). There is usually a sharp threshold in selfing rates below which populations behave as if they were outcrossing and maintain genetic loads, and above which deleterious recessives are selectively purged (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Lande <em>et al.</em> 1994</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Roze 2015</a>), although partial selfing is not found at mutation-selection balance. This leaves more complex scenarios, such as selection on deleterious alleles of variable dominance together with selection on overdominant loci in the presence of significant linkage and/or identity disequilibrium to explain partial selfing, although such scenarios are in need of further investigation, <em>cf</em>. <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Ziehe and Roberds (1989)</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Charlesworth <em>et al.</em> (1990)</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Johnston <em>et al.</em> (2009)</a>; <a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Roze (2015</a>, <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">2016</a>).</p><p id="p-5">During the last 10 years the nematode <em>Caenorhabditis elegans</em> has become a good experimental model to study how standing genetic variation for fitness depends on reproductive mode (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Teotónio <em>et al.</em> 2017</a>). <em>C. elegans</em> is an androdioecious species, where hermaphrodites are capable of autonomous selfing and of outcrossing with males, but incapable of outcrossing with other hermaphrodites. Sex determination is chromosomal with hermaphrodites having two sex chromosomes and males only one (XX and X0). For these reasons, and because sex segregates in a Mendelian fashion (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Teotónio <em>et al.</em> 2012</a>), one minus twice the frequency of males is a good proxy for the selfing rate of a population (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Stewart and Phillips 2002</a>). Wild isolates of <em>C. elegans</em> are usually devoid of males (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Teotónio <em>et al.</em> 2006</a>), are highly inbred and show outbreeding depression when crossed with each other (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Dolgin <em>et al.</em> 2007</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Chelo <em>et al.</em> 2013</a>). Outbreeding depression is due to the fact that a long history of predominant selfing, metapopulation dynamics and linked selection likely led to the evolution of gene complexes that decrease fitness when disrupted (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Cutter 2006</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Seidel <em>et al.</em> 2008</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Rockman <em>et al.</em> 2010</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Andersen <em>et al.</em> 2012</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gaertner <em>et al.</em> 2012</a>).</p><p id="p-6">While in nature selfing is the predominant mode of reproduction, in the laboratory populations can maintain partial selfing. When experimental evolution starts from populations resulting from the hybridization of wild isolates, partial selfing is stably maintained for up to 100 generations (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Anderson <em>et al.</em> 2010</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Teotónio <em>et al.</em> 2012</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Masri <em>et al.</em> 2013</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Carvalho <em>et al.</em> 2014a</a>). Our previous work suggests that partial selfing is maintained because purging of partially recessive deleterious alleles, which had been generated through the disruption of natural gene complexes, co-occurs with selection of overdominant loci, leading first to a reduction in genetic loads and then to the maintenance of excess heterozygosity (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Chelo <em>et al.</em> 2013</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Chelo and Teotónio 2013</a>). However, our conclusions had to be tempered because of problems in experimental design, as we assumed that biparental and uniparental inbreeding were equivalent, and we lacked a control differentiating reduced effective recombination because of increased linkage or identity disequilibrium.</p><p id="p-7">Here, we analyze the <em>C. elegans</em> evolution experiments first reported in <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Theologidis <em>et al.</em> (2014)</a>, where populations with different reproduction systems, obtained by introgression of sex determination mutant alleles into a lab domesticated population (<a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Teotónio <em>et al.</em> 2012</a>), were cultured for 50 generations in challenging NaCl conditions (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). We ask how androdioecious populations with partial selfing cope with extreme inbreeding and the potential for extinction at very small population sizes when compared with monoecious and trioecious populations. Monoecious populations only have hermaphrodites and thus reproduction is exclusively done by selfing, whereas trioecious populations have males, females and a small number of hermaphrodites and thus reproduction is predominantly done by outcrossing.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2811/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Experimental design and origin of populations used in this work. The genetically diverse androdioecious population A6140 (containing males and hermaphrodites) was used as the ancestral population for experimental evolution under increasing NaCl concentrations for 50 generations, reported as the &#x201D;Gradual regime&#x201D; in Theologidis et al. (2014). This ancestral population was subjected to introgression of sex determination mutant alleles to generate populations with different proportions of males, hermaphrodites and females, which evolved under different contributions of selfing and outcrossing (M00 - monoecious; A00 - androdioecious; and, T00 - trioecious populations). Population A00 was derived by sampling of A6140 without any genetic manipulation. Experimental evolution was done with several replicates and, at the end of it, inbred lines were derived by selfing from the resulting populations as had been for the ancestral A6140. Population nomenclature follows Theologidis et al. (2014) and Noble et al. (2017) and font colors are those used throughout this paper." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1721853206" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Experimental design and origin of populations used in this work. The genetically diverse androdioecious population A6140 (containing males and hermaphrodites) was used as the ancestral population for experimental evolution under increasing NaCl concentrations for 50 generations, reported as the &#x201D;Gradual regime&#x201D; in Theologidis &lt;em&gt;et al.&lt;/em&gt; (2014). This ancestral population was subjected to introgression of sex determination mutant alleles to generate populations with different proportions of males, hermaphrodites and females, which evolved under different contributions of selfing and outcrossing (M00 - monoecious; A00 - androdioecious; and, T00 - trioecious populations). Population A00 was derived by sampling of A6140 without any genetic manipulation. Experimental evolution was done with several replicates and, at the end of it, inbred lines were derived by selfing from the resulting populations as had been for the ancestral A6140. Population nomenclature follows Theologidis &lt;em&gt;et al.&lt;/em&gt; (2014) and Noble &lt;em&gt;et al.&lt;/em&gt; (2017) and font colors are those used throughout this paper.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2811/F1.medium.gif" width="440" height="336"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/9/2811/F1.medium.gif" width="440" height="336"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2811/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2811/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494180" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-8" class="first-child">Experimental design and origin of populations used in this work. The genetically diverse androdioecious population A6140 (containing males and hermaphrodites) was used as the ancestral population for experimental evolution under increasing NaCl concentrations for 50 generations, reported as the ”Gradual regime” in <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Theologidis <em>et al.</em> (2014)</a>. This ancestral population was subjected to introgression of sex determination mutant alleles to generate populations with different proportions of males, hermaphrodites and females, which evolved under different contributions of selfing and outcrossing (M00 - monoecious; A00 - androdioecious; and, T00 - trioecious populations). Population A00 was derived by sampling of A6140 without any genetic manipulation. Experimental evolution was done with several replicates and, at the end of it, inbred lines were derived by selfing from the resulting populations as had been for the ancestral A6140. Population nomenclature follows <a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">Theologidis <em>et al.</em> (2014)</a> and <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Noble <em>et al.</em> (2017)</a> and font colors are those used throughout this paper.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">To measure genetic loads, we follow the extinction rates of lineages derived from evolving populations by extreme inbreeding (selfing), and also the fertility of surviving inbred lines (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). The risk of extinction and fertility of monoecious populations is the best estimate of the fitness of “optimal” genotypes, given that exclusive selfing during evolution quickly reduces heterozygosity (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Guzella <em>et al.</em> 2018</a>). On the other hand, the risk of extinction and fertility of trioecious populations benchmark the genetic load of the ancestral population, since at the small effective population sizes (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-1.gif"/></noscript></span></span>) and time frame employed mutation is not expected to contribute much to the observed responses (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Matuszewski <em>et al.</em> 2015</a>). Genome-wide single nucleotide polymorphism (SNP) diversity of the lines surviving inbreeding is further compared with that of the evolving populations. We do this to find how selection on deleterious variation depends on linkage disequilibrium or on the identity disequilibrium generated by partial selfing.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Ancestral population</h3><p id="p-10">All populations in this study are ultimately derived from a hybrid population obtained by a funnel-crossing scheme where 16 parental wild isolates were reciprocally crossed two-by-two, and their hybrid offspring subsequently crossed also in a pairwise fashion, until obtaining the final hybrid population (<a id="xref-ref-60-4" class="xref-bibr" href="#ref-60">Teotónio <em>et al.</em> 2012</a>). The hybrid population was then cultured under 4-day discrete and non-overlapping life-cycles, following the “bleach/hatch-off” protocol of <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Stiernagle (1999)</a>, by using a sodium hypochlorite (bleach) solution to kill all adults and larvae and let only unhatched embryos survive. Census population sizes of <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-2.gif"/></noscript></span></span> between the first larval stage and the time of reproduction were maintained to derive a 140 generation lab-domesticated population (<a id="xref-ref-60-5" class="xref-bibr" href="#ref-60">Teotónio <em>et al.</em> 2012</a>; <a id="xref-ref-63-4" class="xref-bibr" href="#ref-63">Theologidis <em>et al.</em> 2014</a>) and at the effective population sizes of <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-3.gif"/></noscript></span></span> (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Chelo and Teotónio 2013</a>). During lab domestication, male frequencies were stable at about 25%, thus indicating that selfing was maintained at 50% given that hermaphrodites cannot mate with each other (<a id="xref-ref-60-6" class="xref-bibr" href="#ref-60">Teotónio <em>et al.</em> 2012</a>). The 140-generation lab-adapted population (named A6140) is the androdioecious ancestor population for all the experimental populations reported (see <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>).</p></div><div id="sec-3" class="subsection"><h3>Genetic manipulation of sex ratios</h3><p id="p-11">The genetic manipulation of sex ratios in A6140 has been detailed in <a id="xref-ref-63-5" class="xref-bibr" href="#ref-63">Theologidis <em>et al.</em> (2014)</a>. We mass introgressed an X-chromosome sex determination mutant allele [<a href="http://www.wormbase.org/db/get?name=WBGene00006962;class=Gene"><em>xol-1</em></a><em>(</em><a href="http://www.wormbase.org/db/get?name=WBVar00251838;class=Variation"><em>tm3055</em></a><em>)</em>] into A6140, a mutant impairing X-chromosome dosage compensation and leading to the arrest of male development, to derive a monoecious population composed exclusively of hermaphrodites [homozygous for <a href="http://www.wormbase.org/db/get?name=WBGene00006962;class=Gene"><em>xol-1</em></a><em>(</em><a href="http://www.wormbase.org/db/get?name=WBVar00251838;class=Variation"><em>tm3055</em></a><em>)</em>] that only reproduce by selfing (named M00). In parallel, we mass introgressed an autosomal sex determination recessive mutant allele [<a href="http://www.wormbase.org/db/get?name=WBGene00001482;class=Gene"><em>fog-2</em></a><em>(</em><a href="http://www.wormbase.org/db/get?name=WBVar00240995;class=Variation"><em>q71</em></a><em>)</em>] into A6140, a mutant that eliminates self-spermatogenesis in hermaphrodites, to derive a trioecious population composed of males and females [homozygous for <a href="http://www.wormbase.org/db/get?name=WBGene00001482;class=Gene"><em>fog-2</em></a><em>(</em><a href="http://www.wormbase.org/db/get?name=WBVar00240995;class=Variation"><em>q71</em></a><em>)</em>], and hermaphrodites [homozygous for <a href="http://www.wormbase.org/db/get?name=WBGene00001482;class=Gene"><em>fog-2</em></a><em>(wt)</em>] at an initial frequency of <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-4.gif"/></noscript></span></span> (named T00). This population predominantly outcrossed during experimental evolution, despite the effects of high salt in delaying development (<a id="xref-ref-63-6" class="xref-bibr" href="#ref-63">Theologidis <em>et al.</em> 2014</a>). To generate an androdioecious population (named A00), with a comparable initial genetic diversity to M00 and T00, we sampled the same number of families from A6140, as those during the derivation of M00 and T00. As measured with competition assays against a tester strain (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Teotónio <em>et al.</em> 2017</a>), initial population-fitness differences between M00, T00 and A00 are explained by differences in sex ratios, as all individuals in the M00 population are hermaphrodites whereas this number is reduced to half in the A00 population and to negligible numbers in the T00 population (in <a id="xref-ref-63-7" class="xref-bibr" href="#ref-63">Theologidis <em>et al.</em> (2014)</a> see Figure S5 for fitness data in ancestral populations).</p></div><div id="sec-4" class="subsection"><h3>Experimental evolution</h3><p id="p-12">The evolution experiment has also been previously reported in <a id="xref-ref-63-8" class="xref-bibr" href="#ref-63">Theologidis <em>et al.</em> (2014)</a> and the populations here described were those evolving under the conditions defined for the “Gradual regime”. Replicate populations of M00, T00 and A00 were exposed in NGM-lite media to 8 mM/generation increases in NaCl concentration, from the first larval stage until adulthood during 35 generations, after which they were kept at 305 mM NaCl for 15 additional generations. Here, we report results of 2 replicates derived from T00 (GT150, GT250; nomenclature according to <a id="xref-ref-63-9" class="xref-bibr" href="#ref-63">Theologidis <em>et al.</em> (2014)</a>), 2 replicates from M00 (GM150, GM350), and 3 replicates from A00 (GA150, GA250, GA450), all after 50 generations of experimental evolution (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). In <a id="xref-ref-63-10" class="xref-bibr" href="#ref-63">Theologidis <em>et al.</em> (2014)</a> a total of four replicates were used for experimental evolution but, for some, inbred line derivation was not performed or fertility and genotypic data were not obtained. Fitness measurements obtained with competition experiments show that adaptation to the high salt environment was similar under the three reproduction systems by generation 35, and that possible differential sexual selection was not detected during this period [Figure 4D-F in <a id="xref-ref-63-11" class="xref-bibr" href="#ref-63">Theologidis <em>et al.</em> (2014)</a>].</p></div><div id="sec-5" class="subsection"><h3>Genetic load: inbred line derivation and extinction assays</h3><p id="p-13">For line derivation, we followed standard protocols similar to mutation accumulation evolution experiments, where population sizes are kept to a minimum to allow fixation of segregating alleles in the population within each line in mostly a neutral fashion (<a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Teotónio <em>et al.</em> 2017</a>). In our case, population sizes were of <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-5.gif"/></noscript></span></span>, because we derived lines by selfing of hermaphrodites. In this situation, we expect that alleles with an expected average selection coefficient of <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-6.gif"/></noscript></span></span> will be randomly fixed within each line within 3-4 generations, assuming log additive effects and no initial linkage between them (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kimura 1983</a>). Given the small population sizes, and the duration of the assays, very few if any new mutations should appear and get fixed during line derivation (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Vassilieva <em>et al.</em> 2000</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Estes <em>et al.</em> 2004</a>).</p><p id="p-14">In our experiments, we used trioecious populations, instead of exclusively outcrossing (dioecious) populations so that we could derive inbred lines by selfing from all experimental populations and thus not confound potential selection under biparental and uniparental inbreeding as in <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Chelo <em>et al.</em> (2013)</a>. Specifically, A6140, and all generation 50 populations, were revived from frozen stocks, each with &gt;10<sup>3</sup> individuals, and cultured for two generations in a common environment. At the third generation, hermaphrodites were handpicked at the larval immature L3-L4 stage and placed in individual wells of 12-well cell culture plates, previously filled with NGM-lite media and <em>E. coli</em> bacteria as <em>ad libitum</em> food. During inbred line derivation we used 25 mM NaCl, the salt condition of prior lab domestication during 140 generations. Every 4 to 7 days, one L3-L4 offspring hermaphrodite was handpicked and placed into a fresh well, consecutively for 16 generations for A6140 or for 13 generations in all other populations. Resulting lines were expanded in numbers for 2-3 further generations and cryogenically frozen. During inbred line derivation, the 12-well plates were kept at <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-7.gif"/></noscript></span></span> to prevent growth until successful offspring production could be confirmed in the following generation. A line was considered extinct if reproduction from the previous generation failed twice. Because of this recourse to backups the effective population size during line derivation was slightly above <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-8.gif"/></noscript></span></span>. This procedure, of line derivation, was carried out several times (in “blocks”) always starting from cryogenically frozen experimental populations’ stocks until the final number of lines was obtained. For A6140 this was done in two blocks and for generation 50 populations in another six separate blocks, where in each block lines were derived in parallel from each of the evolved populations.</p></div><div id="sec-6" class="subsection"><h3>Genetic load: fertility assays</h3><p id="p-15">Data from the inbred lines in low salt (25 mM NaCl), the environment where previous lab domestication and line derivation was done, was previously reported in <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Noble <em>et al.</em> (2017)</a>. Here, we present new data from the inbred lines in the target high salt environment (305 mM NaCl) and from all populations before line derivation in both high and low salt. Briefly, hermaphrodites were individually placed at the L3-L4 larval stage into wells of a 96-well assay plate with NGM-lite media and bacteria, and 24h later exposed to the bleach solution. Hatched live L1 larvae were counted 24h after bleach exposure. This assay closely mimics the culture protocol used during experimental evolution with the exception of density and opportunity for mating, which, during experimental evolution are those of 1000 individuals growing in 90 mm (diameter) Petri dishes. Individual fertility was measured in all experimental populations (A6140, A00, M00, T00, and all generation 50 derived populations, see <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>) and the inbred lines at 25 mM and 305 mM NaCl (total n = 40800). Manual curation was done during the assay and during image analysis by removing data where individual hermaphrodites could not be found (n = 917) or were dead (n = 1273) at the time bleach exposure (because worms desiccated on the walls of the plate wells, for example). Cases were males had been inadvertently picked instead of hermaphrodites (n = 449) or more than 1 individual was found were also removed (n = 354), as well as whenever bacteria was seen to have not been seeded (n = 807), to prevent results from being affected by lack of food during final stages of development. Data from wells where the adult hermaphrodite was still alive after bleach/hatch-off were also discarded to prevent an upward bias on estimates due to a protocol’s inefficient implementation.</p></div><div id="sec-7" class="subsection"><h3>DNA collection and genotyping</h3><p id="p-16">Genomic DNA from A6140 and each of the G50 populations was extracted from individuals with the ZyGem prepGEM Insect kit and from pools of individuals from each inbred line using the Qiagen Blood and Tissue kit. A total of 830 SNPs distributed evenly across genetic distance, which was estimated by linear interpolation of the expected F2 map distances from <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Rockman and Kruglyak (2009)</a>, were genotyped using Sequenom methods (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bradic <em>et al.</em> 2011</a>). The choice of the SNPs was based on known variation between the founder wild isolates used to create the A6140 population as described in (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Noble <em>et al.</em> 2017</a>). For the ancestral and evolved populations, each individual was genotyped for chromosomes I and II, III and IV, and V and X, respectively. Quality control was done following the criteria of <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Chelo and Teotónio (2013)</a>, with the resulting genotype matrix being composed of 743 SNPs. Sample sizes after quality control can be found in Supplementary Table 1.</p><p id="p-17">All genotype data for the lines has been previously published in <a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Noble <em>et al.</em> (2017)</a> to cross-validate the whole-genome line sequencing done in that study. Data for the monoecious populations at generation 50 has been published in <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Guzella <em>et al.</em> (2018)</a>. New genotype data are presented for the ancestral population (A6140) and the trioecious and androdioecious populations at generation 50.</p></div><div id="sec-8" class="subsection"><h3>Genetic diversity during experimental evolution</h3><p id="p-18">In ancestral and evolved experimental populations, SNP allele diversity was estimated as <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-9.gif"/></noscript></span></span>, and an inbreeding coefficient (fixation index) as <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-10.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-11.gif"/></noscript></span></span> being the observed proportion of heterozygotes (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Crow and Kimura 1970</a>). To avoid sampling biases, SNPs with <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-12.gif"/></noscript></span></span> lower than 0.05 were removed prior to calculation of <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-13.gif"/></noscript></span></span>.</p><p id="p-19">Linkage disequilibrium between pairs of SNPs was estimated as <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-14.gif"/></noscript></span></span>, where D is the difference in genotypic frequencies from that expected given the allele frequencies <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-15.gif"/></noscript></span></span> in SNPs i and j (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Weir 1996</a>). Comparisons between reproductive systems and the ancestral population were done with replicate as a random factor, via nonlinear regression using <em>nlme</em> R package (Pinheiro <em>et al.</em> 2018), with the following input formula for the <em>nlme</em> function:</p><span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-16.gif"/></noscript></span></span><p id="p-20">With this, we model the observed <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-17.gif"/></noscript></span></span> (“R2” in code) for every pair of SNPs as a function of <em>c</em>, which is the genetic distance between SNPs, and <em>x</em>, which determines the rate of LD decay and can differ between populations (“PopType” - ancestral, trioecious, androdioecious and monoecious). The underlying assumption is that the expected decay of linkage disequilibrium with genetic distance follows the equation given by <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Sved (1971)</a> as <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-18.gif"/></noscript></span></span>, where <em>N</em> refers to the effective population size. Only SNPs segregating at frequencies between 0.05 and 0.95 were used.</p><p id="p-21">Identity disequilibrium, <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-19.gif"/></noscript></span></span> (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">David <em>et al.</em> 2007</a>), was calculated using the R package <em>inbreedR</em> (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Stoffel <em>et al.</em> 2016</a>). Identity disequilibrium is only estimated for populations where partial selfing is possible, that is, the androdioecious populations and trioecious populations.</p><p id="p-22">Finally, we also calculated the mean effective number of haplotypes (<span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-20.gif"/></noscript></span></span>) as a measure of multi-allelic diversity: <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-21.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-22.gif"/></noscript></span></span> being the proportion of haplotype <em>i</em> in a given replicate population or group of lines. <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-23.gif"/></noscript></span></span> can be interpreted as the number of haplotypes segregating in the population if they were all present at the same frequency (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Crow and Kimura 1970</a>). For each population, haplotypes are defined in non-overlapping windows of 10 SNPs, as diploid phased genotypes (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Chelo and Teotónio 2013</a>), independently for each population. Varying SNPs per window between 5 and 20 does not affect the conclusions (not shown).</p><p id="p-23">For all diversity metrics, we only show the genome wide average in each population.</p></div><div id="sec-9" class="subsection"><h3>Genetic diversity among inbred lines</h3><p id="p-24"><span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-24.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-25.gif"/></noscript></span></span> metrics were calculated for the surviving inbred lines and used to evaluate how diversity changed after the 13-16 generations of selfing. For that purpose, means and standard deviations based on single individuals from the experimental populations were compared with the means and standard deviations of single lines (obtained from pools of individuals). Each line was considered as a diploid individual. Due to the sampling design, genetic data from 16 individuals was available from the experimental populations, whereas the final number of lines was usually above 50. To prevent biases arising from unequal sample sizes, we calculated the metrics in the inbred lines as the average of subsets with 16 observations, as obtained by Jackknife resampling.</p><p id="p-25">Following <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Chelo <em>et al.</em> (2013)</a>, to provide neutral confidence intervals for the expected genetic diversity metrics after inbreeding, numerical simulations were run with selfing for 13 (for generation 50 populations) or 16 generations (for A6140). Less diversity after inbreeding, among surviving lines, in comparison with the corresponding populations, indicates purifying selection of highly deleterious alleles during the assay. More diversity indicates that either overdominant loci or deleterious alleles from different loci in repulsion linkage disequilibrium (associative overdominance, (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Ohta and Kimura 1970</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Palsson and Pamilo 1999</a>)), are present in the populations from where the lines are derived. Conditional on the observed diversity before line derivation, 1000 simulations were run per population. All chromosomes were separately analyzed and metrics among SNPs averaged across chromosomes. Each run started by randomly sampling phased diploids, with replacement. Meiotic crossover was simulated by exchanging consecutive sets of alleles between the two parental haplotypes, and assuming complete crossover interference and F2 map sizes of 50cM per chromosome. Crossover occurred randomly between two consecutive SNPs depending on the probability given by the genetic distances between them as in <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Rockman and Kruglyak (2009)</a>, and zygotes obtained by joining two independent gametes from the same individual. Similar sampling of lines was done from the simulations as that in the assays, where usually more than 50 lines survived inbreeding.</p></div><div id="sec-10" class="subsection"><h3>Statistical analysis</h3><p id="p-26">Testing for differences in the several genetic diversity metrics was done with ANOVA using reproductive system as the predictor variable. Because we have only one data point for the ancestral population, we fixed the intercept in these models. Two-sided Tukey <em>t</em>-tests were then used for post-hoc comparisons using the <em>lsmeans</em> package in R (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lenth 2015</a>). The exception was in the analysis of identity disequilibrium, where individual <em>t</em>-tests were uncorrected. P-values from significant tests are reported as: * <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-26.gif"/></noscript></span></span> or ** <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-27.gif"/></noscript></span></span>. Shapiro-Wilk and Bartlett’s tests were done to check for departure from normality and homocedasticity of residuals, respectively. Maximum likelihood parameter estimates obtained by nonlinear regression of <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-28.gif"/></noscript></span></span> decay with genetic distance were compared against the ancestral population via a Wald test.</p><p id="p-27">The risk of line extinction per generation was assessed with Kaplan-Meier estimators (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kaplan and Meier 1958</a>), assuming right censored data. Cox proportional hazards regression models (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Cox 1972</a>) were then used to compare the extinction risks of each population type (ancestral, evolved trioecious and evolved androdioecious) against the monoecious evolved populations (as it provides the expected line extinction due to environmental effects, assuming individuals were homozygous prior to line derivation), or between evolved populations and the ancestral population. In these models there is a baseline risk (hazard ratio, HR) for each line undergoing selfing, which can differ between lines of different treatments in a time-independent manner. We employed a mixed effects model for analysis, with reproductive system as the fixed factor and replicate population as a random factor. Z-scores were used for significance testing. The R package <em>coxme</em> was used for computation (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Therneau 2015</a>).</p><p id="p-28">For fertility data, before data analysis, we noticed a high number of infertile hermaphrodites and females. This occurred because most of them were either not mated when individually transferred 24h before reproduction and/or because they had not reached maturity at the time of the assay. For this reason we decided to first model fertility distributions assuming Poisson, negative binomial, or zero-inflated errors for all populations and inbred lines (using R functions <em>glm</em> in the stats package, <em>glm.nb</em> in the MASS package, and <em>zeroinfl</em> in the pscl package). Likelihood ratio tests were employed for model selection. Zero-inflated negative binomial error distributions were chosen for subsequent analyses as they provided the best fit. We then employed generalized mixed effect models separately for 25 mM and 305 mM NaCl conditions, using the function <em>glmmadmb</em> from package glmmADMB (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Skaug <em>et al.</em> 2013</a>). We tested for the effects of experimental evolution (ancestral <em>vs.</em> evolved) and the reproductive system at generation 50, with assay blocks and replicate populations as random factors. Differences between experimental populations and the average of inbred lines derived from them was again carried out with the <em>lsmeans</em> and <em>pairs</em> functions in R (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Lenth 2015</a>).</p></div><div id="sec-11" class="subsection"><h3>Data availability</h3><p id="p-29">Fertility and genotype data for the inbred lines associated with <a id="xref-ref-40-5" class="xref-bibr" href="#ref-40">Noble <em>et al.</em> (2017)</a> has been archived in FigShare 10.6084/m9.figshare.5326777. All data and R code for analysis can be found in FigShare accessions: 10.6084/m9.figshare.5539606 (readme), 10.6084/m9.figshare.5539612 (code), 10.6084/m9.figshare.8665661 (dataset phenotypes and SNP genotypes), 10.6084/m9.figshare.5539618 (dataset haplotypes), and 10.6084/m9.figshare.5539627 (dataset simulated haplotypes). Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.8397128">https://doi.org/10.25387/g3.8397128</a>.</p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>Selfing rates during experimental evolution</h3><p id="p-30">As previously noted (<a id="xref-ref-63-12" class="xref-bibr" href="#ref-63">Theologidis <em>et al.</em> 2014</a>), delayed development at high salt concentrations (above 250 mM NaCl) resulted in diminished opportunity for mating. This led to outcrossing impairment and, consequently, to androdioecious populations showing a decrease in males from initially &gt; 40% until generation 30 to 10% by generation 50 (Supplementary Figure S1). Despite delayed development, trioecious populations maintained above 40% of males and about 40% of hermaphrodites by generation 50. Under trioecy, hermaphrodites provide reproductive assurance by selfing and they replace females at high salt concentrations. Yet, their starting low numbers and the more favorable conditions during the first 30 generations imply that males are nevertheless maintained at high frequencies, and outcrossing (between males and both females and hermaphrodites) is the predominant reproductive mode. In these populations, by the time stressful salt concentrations start to be experienced (from generation 30 onwards), most hermaphrodites carry the <a href="http://www.wormbase.org/db/get?name=WBGene00001482;class=Gene"><em>fog-2</em></a><em>(</em><a href="http://www.wormbase.org/db/get?name=WBVar00240995;class=Variation"><em>q71</em></a><em>)</em> mutant allele in the heterozygous state, meaning they will also produce females through selfing, and male mating ability has had the opportunity to improve through sexual selection (<a id="xref-ref-63-13" class="xref-bibr" href="#ref-63">Theologidis <em>et al.</em> 2014</a>). Both factors contributed to the maintenance of outcrossing and males at higher frequencies than in androdioecious populations despite high salt conditions.</p></div><div id="sec-14" class="subsection"><h3>Evolution under predominant outcrossing or partial selfing maintains standing genetic variation</h3><p id="p-31">When compared with the ancestral population, androdioecious and monoecious populations reveal an increase in inbreeding coefficients (<span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-29.gif"/></noscript></span></span>; from 0.11 to 0.77 +/− 0.110 SD; Tukey <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-30.gif"/></noscript></span></span> = 5.42, p-value = 0.01, for the androdioecious populations; and from 0.11 to 0.988 +/− 0.003 SD for the monoecious populations; <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-31.gif"/></noscript></span></span> = 6.77, p-value = 0.006), unlike the trioecious populations, which maintain ancestral levels (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2a</a>). In monoecious populations, this increase in inbreeding coefficients was accompanied by a marked reduction in SNP allele diversity after 50 generations of experimental evolution (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2b</a>; <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-32.gif"/></noscript></span></span> = -91.45, <em>P</em> &lt; 0.0001), a result that contrasts with values from androdioecious populations which, together with trioecious populations, only show a small decrease in allele diversity (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2b</a>; for trioecious <em>vs.</em> ancestral <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-33.gif"/></noscript></span></span> comparisons: <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-34.gif"/></noscript></span></span> = -8.98, p-value = 0.002; for androdioecious <em>vs.</em> ancestral: <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-35.gif"/></noscript></span></span> = -18.19, p-value = 0.0001).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2811/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic diversity comparison between the ancestral population and populations evolved under the three different reproductive modes. Legend colors apply equally to all panels. (a, b), Higher inbreeding coefficients () are seen in monoecious and androdioecious populations than in the ancestral or trioecious populations, while SNP allele diversity () is almost halved only in monoecious populations. (c), Linkage disequilibrium within chromosomes, shown as the expected decay in  with genetic distance (cM). Data were fitted with , with respect to x and lines represent fitted values for ancestral and derived populations with shaded areas showing 95% confidence intervals (see Materials and Methods). Lines from the ancestral and trioecious populations overlap. Y axis is shown in a logarithmic scale. (d), Expected identity disequilibrium () is shown with exception of monoecious populations that have no heterozygotes. Means and one SD among replicate populations (dots) are shown (see Materials and Methods). Asterisks show significant differences between experimentally evolved populations and the ancestral population for p-values &lt; 0.05 (*) or p-values &lt; 0.005 (**)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1721853206" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Genetic diversity comparison between the ancestral population and populations evolved under the three different reproductive modes. Legend colors apply equally to all panels. (a, b), Higher inbreeding coefficients (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-36&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) are seen in monoecious and androdioecious populations than in the ancestral or trioecious populations, while SNP allele diversity (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-37&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) is almost halved only in monoecious populations. (c), Linkage disequilibrium within chromosomes, shown as the expected decay in &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-38&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; with genetic distance (cM). Data were fitted with &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-39&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, with respect to &lt;em&gt;x&lt;/em&gt; and lines represent fitted values for ancestral and derived populations with shaded areas showing 95% confidence intervals (see Materials and Methods). Lines from the ancestral and trioecious populations overlap. Y axis is shown in a logarithmic scale. (d), Expected identity disequilibrium (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-40&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) is shown with exception of monoecious populations that have no heterozygotes. Means and one SD among replicate populations (dots) are shown (see Materials and Methods). Asterisks show significant differences between experimentally evolved populations and the ancestral population for p-values &lt; 0.05 (*) or p-values &lt; 0.005 (**).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2811/F2.medium.gif" width="440" height="395"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/9/2811/F2.medium.gif" width="440" height="395"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2811/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2811/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494165" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-32" class="first-child">Genetic diversity comparison between the ancestral population and populations evolved under the three different reproductive modes. Legend colors apply equally to all panels. (a, b), Higher inbreeding coefficients (<span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/F2/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/F2/embed/mml-math-36.gif"/></noscript></span></span>) are seen in monoecious and androdioecious populations than in the ancestral or trioecious populations, while SNP allele diversity (<span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/F2/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/F2/embed/mml-math-37.gif"/></noscript></span></span>) is almost halved only in monoecious populations. (c), Linkage disequilibrium within chromosomes, shown as the expected decay in <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/F2/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/F2/embed/mml-math-38.gif"/></noscript></span></span> with genetic distance (cM). Data were fitted with <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/F2/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/F2/embed/mml-math-39.gif"/></noscript></span></span>, with respect to <em>x</em> and lines represent fitted values for ancestral and derived populations with shaded areas showing 95% confidence intervals (see Materials and Methods). Lines from the ancestral and trioecious populations overlap. Y axis is shown in a logarithmic scale. (d), Expected identity disequilibrium (<span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/F2/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/F2/embed/mml-math-40.gif"/></noscript></span></span>) is shown with exception of monoecious populations that have no heterozygotes. Means and one SD among replicate populations (dots) are shown (see Materials and Methods). Asterisks show significant differences between experimentally evolved populations and the ancestral population for p-values &lt; 0.05 (*) or p-values &lt; 0.005 (**).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Evolution under partial selfing maintains ancestral linkage disequilibrium but increases identity disequilibrium</h3><p id="p-33">Linkage disequilibrium decays with genetic distance in a similar fashion in the ancestral and trioecious populations (<span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-41.gif"/></noscript></span></span>; <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2c</a>). In contrast, androdioecious and especially monoecious populations show increased linkage disequilibrium (<span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-42.gif"/></noscript></span></span> = - 4.4, df = 2, p-value = 0.047, for the androdioecious populations; and <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-43.gif"/></noscript></span></span> = -16.3, df = 1, p-value = 0.03, for the monoecious populations). Identity disequilibrium increased in androdioecious populations relative to the ancestral population (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2d</a>; <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-44.gif"/></noscript></span></span> increased from 0.07 to 2.5 +/− 0.8 SD; Student <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-45.gif"/></noscript></span></span> = 5.71, p-value = 0.029). Trioecious populations maintain the same identity disequilibrium as that observed in the ancestral population (Student <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-46.gif"/></noscript></span></span> = 1.15, p-value = 0.454). Identity disequilibrium is undefined in monoecious populations because individuals are homozygous at the measured SNPs (<span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/embed/mml-math-47.gif"/></noscript></span></span>).</p></div><div id="sec-16" class="subsection"><h3>Evolution under predominant or partial selfing reduces genetic loads</h3><p id="p-34">Given the absence of heterozygotes in the monoecious populations, the observed 85% lineage survival during inbreeding provides the benchmark for lack of a genetic load and should reveal the risk of extinction due to environmental factors. Survival of monoecious populations upon inbreeding contrasts with the ancestral population where only 70% of the lines survived after 16 generations of inbreeding (monoecious <em>vs.</em> ancestral hazard ratio HR = 2.0, p-value = 0.014; <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3a</a>). Androdioecious populations likewise evolved a lower risk of extinction upon inbreeding when compared to the ancestral population (HR = 0.53, p-value = 0.002). Both androdioecious and monoecious populations have similar extinction risks upon inbreeding (HR = 1.05, p-value = 0.84), despite the apparent difference in inbreeding coefficients between them (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3b</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2811/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(a), Kaplan-Meier estimates of survival rates (1 minus risk of lineage extinction) are shown for the laboratory adapted ancestral population (gray line) and the experimentally evolved populations, with shaded areas representing 95% confidence intervals. Hermaphrodites were selfed for 13-16 generations and the proportion of surviving lineages recorded. Simulations of the extent to which lineage extinction in the trioecious population is due to picking segregating females or fog-2(q71) heterozygous hermaphrodites is shown in Supplementary Figure S2. Significant differences in the risk of extinction to evolved monoecious populations, which represent the condition of no inbreeding depression, or ancestral population are shown as lines: * p-value &lt; 0.05, ** p-value &lt; 0.005. In (b), the tendency for populations with higher inbreeding after 50 generations of experimental evolution to have a lower risk of extinction is revealed by a negative correlation (p-value &lt; 0.005) between inbreeding coefficients and hazard ratios, in which the monoecious populations were used as reference (shown in blue). Coefficient of determination is shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1721853206" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(a), Kaplan-Meier estimates of survival rates (1 minus risk of lineage extinction) are shown for the laboratory adapted ancestral population (gray line) and the experimentally evolved populations, with shaded areas representing 95% confidence intervals. Hermaphrodites were selfed for 13-16 generations and the proportion of surviving lineages recorded. Simulations of the extent to which lineage extinction in the trioecious population is due to picking segregating females or &lt;em&gt;fog-2(q71)&lt;/em&gt; heterozygous hermaphrodites is shown in Supplementary Figure S2. Significant differences in the risk of extinction to evolved monoecious populations, which represent the condition of no inbreeding depression, or ancestral population are shown as lines: * p-value &lt; 0.05, ** p-value &lt; 0.005. In (b), the tendency for populations with higher inbreeding after 50 generations of experimental evolution to have a lower risk of extinction is revealed by a negative correlation (p-value &lt; 0.005) between inbreeding coefficients and hazard ratios, in which the monoecious populations were used as reference (shown in blue). Coefficient of determination is shown.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2811/F3.medium.gif" width="440" height="203"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/9/2811/F3.medium.gif" width="440" height="203"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2811/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2811/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494116" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-35" class="first-child">(a), Kaplan-Meier estimates of survival rates (1 minus risk of lineage extinction) are shown for the laboratory adapted ancestral population (gray line) and the experimentally evolved populations, with shaded areas representing 95% confidence intervals. Hermaphrodites were selfed for 13-16 generations and the proportion of surviving lineages recorded. Simulations of the extent to which lineage extinction in the trioecious population is due to picking segregating females or <em>fog-2(q71)</em> heterozygous hermaphrodites is shown in Supplementary Figure S2. Significant differences in the risk of extinction to evolved monoecious populations, which represent the condition of no inbreeding depression, or ancestral population are shown as lines: * p-value &lt; 0.05, ** p-value &lt; 0.005. In (b), the tendency for populations with higher inbreeding after 50 generations of experimental evolution to have a lower risk of extinction is revealed by a negative correlation (p-value &lt; 0.005) between inbreeding coefficients and hazard ratios, in which the monoecious populations were used as reference (shown in blue). Coefficient of determination is shown.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">Experimental evolution under predominant outcrossing (trioecy) did not result in a reduction of the ancestral genetic load. Trioecious populations maintained the ancestral risk of extinction (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3a</a>; HR = 1.41, p-value = 0.1), while maintaining the same inbreeding coefficients (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2b</a>). When compared with monoecious populations, trioecious population have an increased risk of line extinction (HR = 2.82, p-value = 0.0002). Because trioecious populations segregate a large proportion of females and derivation of inbred lines was done by selfing of single individuals, extinction could simply be due to their inadvertent hand-picking during initial assay set-up. Numerical simulations show, however, that that is not the case (Supplementary Figure S2).</p><p id="p-37">We also attempted to detect genetic loads by comparing fertility between experimental populations and the inbred lines derived from them, given that a difference between the mean values of fitness components in populations and the mean value among the surviving inbred lines would indicate inbreeding (negative difference) or outbreeding (positive difference) depression (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lynch and Walsh 1998</a>). Moreover in the case of a negative difference between the means, if some lines have fertility values above those shown by the population, one can infer that they have “purged” deleterious recessive alleles during inbreeding by selfing, while lack of lines with fertility values above the average fertility of individuals in the population is consistent with both purging of deleterious recessive alleles and overdominant selection during line derivation (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Lynch and Walsh 1998</a>, p. 283). During experimental evolution fertility increased, particularly for trioecious and monoecious populations in high salt conditions (Supplementary Figure S3). This is a result that is consistent with adaptation to high salt, as in <a id="xref-ref-63-14" class="xref-bibr" href="#ref-63">Theologidis <em>et al.</em> (2014)</a>. There is, however, little evidence that fertility in the inbred lines is different from the populations they were derived from (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2811/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Reproductive output and line derivation. Mean fertility and one SEM among replicate populations are shown for populations from each reproductive mode, in 305 mM NaCl (a) and 25 mM NaCl (b). In each group, from left to right, results are shown for the experimental populations and the inbred lines derived from them. No significant differences between populations and inbred lines were found." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1721853206" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Reproductive output and line derivation. Mean fertility and one SEM among replicate populations are shown for populations from each reproductive mode, in 305 mM NaCl (a) and 25 mM NaCl (b). In each group, from left to right, results are shown for the experimental populations and the inbred lines derived from them. No significant differences between populations and inbred lines were found.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2811/F4.medium.gif" width="440" height="145"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/9/2811/F4.medium.gif" width="440" height="145"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2811/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2811/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494147" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-38" class="first-child">Reproductive output and line derivation. Mean fertility and one SEM among replicate populations are shown for populations from each reproductive mode, in 305 mM NaCl (a) and 25 mM NaCl (b). In each group, from left to right, results are shown for the experimental populations and the inbred lines derived from them. No significant differences between populations and inbred lines were found.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">Given that inbred line derivation was done at low salt, the observed line extinction differences between reproduction systems might not be representative of fitness differences during experimental evolution. This problem does not appear to be of much concern because we detected a positive genetic correlation in fertility across the two salt environments for the androdioecious populations and no obvious trade-off between high and low salt in the other reproduction systems (Supplementary Figure S4).</p></div><div id="sec-17" class="subsection"><h3>Predominant and partial outcrossing populations are more diverse after inbreeding than expected</h3><p id="p-40">SNP and haplotype diversity metrics were calculated among the lines that survived inbreeding by selfing in order to detect selection during inbreeding. In trioecious and androdioecious populations, simulations of genetic drift during inbreeding and sampling predict that SNP diversity among lines is maintained or slightly reduced whereas haplotype diversity is always expected to be reduced (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). However, observed results show that trioecious and androdioecious populations have higher diversity levels than that expected with the simulations, which are in line with observations of the ancestral population. In contrast, monoecious populations show a larger reduction of SNP and haplotype diversity after inbreeding than expected. For monoecious populations, the simulations predict that diversity is maintained or reduced with inbreeding.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/9/2811/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Diversity, shown as the average SNP allele diversity (, panel a) or mean effective haplotype number (, panel b) is plotted for each replicate population (A6140, GT150, GT250, GA150, GA250, GA450, GM150 and GM350 - dark-colored bars) and among the inbred lines derived from them (A6140L, GT150L, GT250L, GA150L, GA250L, GA450L, GM150L and GM350L - light-colored bars). Adjacent to the inbred lines (in white), the expected diversity with no selection during enforced inbreeding obtained with 1,000 numerical simulations is shown with error bars giving 95% credible intervals (see Materials and Methods)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1721853206" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Diversity, shown as the average SNP allele diversity (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-48&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, panel a) or mean effective haplotype number (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-49&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, panel b) is plotted for each replicate population (A6140, GT150, GT250, GA150, GA250, GA450, GM150 and GM350 - dark-colored bars) and among the inbred lines derived from them (A6140L, GT150L, GT250L, GA150L, GA250L, GA450L, GM150L and GM350L - light-colored bars). Adjacent to the inbred lines (in white), the expected diversity with no selection during enforced inbreeding obtained with 1,000 numerical simulations is shown with error bars giving 95% credible intervals (see Materials and Methods).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/9/2811/F5.medium.gif" width="440" height="142"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/9/2811/F5.medium.gif" width="440" height="142"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/9/2811/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/9/2811/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494190" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-41" class="first-child">Diversity, shown as the average SNP allele diversity (<span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/F5/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/F5/embed/mml-math-48.gif"/></noscript></span></span>, panel a) or mean effective haplotype number (<span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/F5/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/9/2811/F5/embed/mml-math-49.gif"/></noscript></span></span>, panel b) is plotted for each replicate population (A6140, GT150, GT250, GA150, GA250, GA450, GM150 and GM350 - dark-colored bars) and among the inbred lines derived from them (A6140L, GT150L, GT250L, GA150L, GA250L, GA450L, GM150L and GM350L - light-colored bars). Adjacent to the inbred lines (in white), the expected diversity with no selection during enforced inbreeding obtained with 1,000 numerical simulations is shown with error bars giving 95% credible intervals (see Materials and Methods).</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">Further evidence for strong selection during the inbreeding assay comes from the SNP frequency changes observed during experimental evolution (Supplementary Figure S5). For trioecious and androdioecious populations, there is a negative correlation in differentiation during experimental evolution and inbreeding, while in monoecious populations only a much weaker relationship could be found.</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-43">In this study, we found that genetically diverse populations of <em>C. elegans</em> reproducing by selfing for 50 generations evolved reduced risk of extinction when maintained at very small sizes and under extreme inbreeding conditions. This result, which is consistent with the reduction of an ancestral genetic load, was likely caused by selection on viability loci given the similar reproductive outputs that were found in populations and among the surviving lines after inbreeding appear to rule out fertility as a major fitness component. Trioecious populations, which reproduced predominantly by outcrossing, had the same risk of extinction upon extreme inbreeding as the ancestral population, suggesting that they maintained the ancestral genetic load. We also found that populations that experienced some degree of outcrossing during experimental evolution (trioecious and androdioecious populations) maintained more genetic diversity than that expected solely under genetic drift, or under genetic drift and directional selection.</p><p id="p-44">Theory on the evolution of selfing rates provide some indication on the possible genetic underpinnings of fitness variation and how partial selfing might have influenced it. If genetic loads – defined as the potential loss of fitness relative to a genotype free of deleterious genetic diversity – are generated by unlinked and partially to fully recessive deleterious alleles, and with selection acting independently at each locus, then one of two stable outcomes is possible (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Charlesworth <em>et al.</em> 1990</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Lande and Schemske 1985</a>). One possibility is that predominant selfing evolves, with populations concomitantly becoming mostly free of deleterious diversity because homozygous individuals are efficiently selected against. The other possibility is that high levels of outcrossing evolve, with populations bearing a load of deleterious recessive alleles sufficient to avoid extinction, as selection is less efficient in purging them. With synergistic selection between deleterious variation, similar outcomes of predominant selfing or predominant outcrossing are expected, although highly selfing populations can maintain some genetic load (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Charlesworth <em>et al.</em> 1991</a>). Other theoretical studies have shown that linkage and identity disequilibria, <em>i.e.</em>, variation in the degree of individual heterozygosity created by variation in past ancestry of selfing and outcrossing, could result in the maintenance of genetic diversity because of selective interference between completely recessive deleterious alleles, or because of the interaction between genetic drift and selection on beneficial alleles (<a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Lande <em>et al.</em> 1994</a>; <a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">Roze 2015</a>).</p><p id="p-45">Our results are difficult to reconcile with these theoretical studies in that more genetic diversity was maintained under trioecy and androdioecy than expected by chance and directional selection. In principle, two mechanisms could lead to this pattern: (1) associative overdominance generated by deleterious recessive alleles at high linkage disequilibrium and in repulsion phase or, (2) heterozygote advantage generated by overdominant loci (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Ohta and Kimura 1970</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Palsson and Pamilo 1999</a>). Linkage disequilibrium was low and similar under trioecy and androdioecy suggesting that the deleterious recessive alleles did not result in excess genetic diversity unless the relevant loci were very closely linked. Identity disequilibrium was on the other hand clearly important under androdioecy. Although identity disequilibrium is usually thought of as revealing homozygous correlations between different loci, it may also reveal heterozygosity correlations (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Szulkin <em>et al.</em> 2010</a>). Under androdioecy, increased identity disequilibrium might have increased fitness variance among individuals and favored multilocus heterozygotes in loci with deleterious recessive alleles or overdominant loci, which unlike with linkage disequilibrium, can be spread across the genome (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Charlesworth and Charlesworth 1990</a>; <a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Chelo and Teotónio 2013</a>).</p><p id="p-46">If excess genetic diversity during experimental evolution resulted from overdominant loci and/or deleterious recessives in identity disequilibrium, then androdioecious populations should have maintained some of the ancestral genetic load (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Charlesworth and Charlesworth 1990</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Charlesworth <em>et al.</em> 1991</a>); but they did not. It may be that the majority of unlinked deleterious recessives were initially purged because of selfing, allowing then for the expression of only a few unlinked overdominant loci that when in the homozygous state resulted in high mortality, as suggested by <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Nordborg <em>et al.</em> (1996)</a>. In the experimental androdioecious populations, these few overdominant loci would not necessarily generate much fitness variance, otherwise populations would have approached extinction. And although present in the trioecious populations, as we assume that all responses are from standing genetic variation, these loci would have had little opportunity to be expressed during experimental evolution. Instead, in trioecious populations, closely linked deleterious recessives, perhaps still remaining from the initial hybridization of wild isolates, could generate associative overdominance responsible for the observed excess diversity (but see <a id="xref-ref-14-6" class="xref-bibr" href="#ref-14">Chelo and Teotónio (2013)</a>).</p><p id="p-47">Under partial selfing (androdioecy), heterozygote advantage could be also generated by recessive lethals in repulsion phase, which are effectively overdominant loci despite the potentially large physical distances between them. Distinguishing these alternatives would require mapping the relevant fitness loci in the genome, an endeavor that is notoriously difficult and beyond the scope of the present study. Recessive sub-lethals in repulsion have been described in natural populations of <em>C. elegans</em>, albeit always in strong linkage disequilibrium and at close physical distance, <em>e.g.</em>, <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Seidel <em>et al.</em> (2008)</a>, and are thus likely to also be present in our populations as they ultimately are derived from a cross between wild isolates (<a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Chelo <em>et al.</em> 2013</a>).</p><p id="p-48">Monoecious populations show an overall reduction in genetic diversity that is in line with neutral expectations under exclusive selfing (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Crow and Kimura 1970</a>). It is possible, however, that some form of frequency- or density-dependent selection acted to maintain diversity in the experimental populations as it appears that there was a reduction genetic diversity during our inbreeding assay (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). In particular, populations were maintained at high densities where different genotypes could interact, while during the inbreeding assay, which was done by individual selfing, no such interactions were possible.</p><p id="p-49">Our experiments involved placing populations in a novel environment in order for substantial genetic variance for fitness to be expressed (when compared to the domestication environment to which populations prior to sex determination manipulation were adapted to). We have previously shown that in the novel salt environment populations from all three reproduction systems had similar adaptive rates [Figure 4 in <a id="xref-ref-63-15" class="xref-bibr" href="#ref-63">Theologidis <em>et al.</em> (2014)</a>]. Yet, we also found that male competitive ability evolved under obligatory outcrossing in male-female (truly dioecious) populations (Figure 6B in <a id="xref-ref-63-16" class="xref-bibr" href="#ref-63">Theologidis <em>et al.</em> (2014)</a>). Sexual selection among males could thus have confounded the effects of selfing. In particular, sexual selection is an alternative mechanism to inbreeding by selfing for the purging of deleterious recessive alleles when these are preferentially expressed in the heterogametic sex (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Agrawal 2001</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Siller 2001</a>). Because we have not measured genetic loads for male fitness components, whether sexual selection or reproductive mode was responsible for the evolution of standing genetic variation and purging of genetic loads is unknown and further investigation will be required. However, a recent experimental evolution study in the hermaphroditic snail <em>Physa acuta</em>, specifically designed to disentangle sexual selection from selfing in the purging of deleterious recessives, has failed to find a significant role for sexual selection (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Noël <em>et al.</em> 2019</a>).</p><p id="p-50">Selfing rates under androdioecy were unstable during experimental evolution and, unless male fitness components were to evolve, exclusive selfing would be achieved in less than 100 generations (<a id="xref-ref-63-17" class="xref-bibr" href="#ref-63">Theologidis <em>et al.</em> 2014</a>). Although partial selfing is found in natural populations it is not particularly common (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Goodwillie <em>et al.</em> 2005</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Jarne and Auld 2006</a>). It is thus reasonable to ask why partial selfing is not more common if it allows the purging of unlinked deleterious recessives while maintaining genetic diversity at a few partially dominant or overdominant loci in identity desiquilibrium, as shown here. Insights into this problem will come from a better understanding of the short-term advantages of selfing in providing reproductive assurance when outcrossing is limited (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Cheptou 2012</a>; <a id="xref-ref-63-18" class="xref-bibr" href="#ref-63">Theologidis <em>et al.</em> 2014</a>) or in creating the opportunity for the evolution of sexual conflicts and allocation of resources to different sexual functions (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Carvalho <em>et al.</em> 2014b</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Poullet <em>et al.</em> 2016</a>), together with the long-term costs of reduced effective population sizes and recombination that may be important during sustained environmental change (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Burt 2000</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Igic <em>et al.</em> 2008</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Goldberg <em>et al.</em> 2010</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lande and Porcher 2015</a>).</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-51">We thank F. Melo and A. Silva for help with the fertility assays; J. Costa, T. Guzella and L. Noble for support with the genotype analysis; and P. David, T. Guzella, L. Noble, M. Rockman and D. Roze for discussion; the editors and reviewers for helping improve the presentation of this work. Financial support was provided by Fundação para a Ciência e Tecnologia through the FCT Investigator Program (IF/00031/2013) and the research project PTDC/BIA-EVF/0009/2014 to I.M.C.; the National Science Foundation (EF-1137835) to S.R.P., and the Human Frontiers Science Program (RGP0045/2010), the European Research Council (FP7/2007-2013/243285) and Agence Nationale de la Recherche (ANR-14-ACHN-0032-01) to H.T. B.A., S.C., I.T. did experimental evolution and derived the inbred lines; C.G., A.P.-Q. did the DNA collection and genotyping; I.M.C. did the fertility assays and analyzed the data; I.M.C., S.R.P and H.T. designed the project and wrote the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-52">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.8397128">https://doi.org/10.25387/g3.8397128</a>.</p></li><li class="fn" id="fn-2"><p id="p-53"><em>Communicating editor: S. Macdonald</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-04-03"><span class="received-label">Received </span>April 3, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-06-17"><span class="accepted-label">Accepted </span>June 17, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Chelo <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.1" data-doi="10.1038/35079590"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Agrawal</span>,  <span class="cit-name-given-names">A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Sexual selection and the maintenance of sexual reproduction.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">411</span>: <span class="cit-fpage">692</span>–<span class="cit-lpage">695</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/35079590</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAgrawal%26rft.auinit1%253DA.%2BF.%26rft.volume%253D411%26rft.issue%253D6838%26rft.spage%253D692%26rft.epage%253D695%26rft.atitle%253DSexual%2Bselection%2Band%2Bthe%2Bmaintenance%2Bof%2Bsexual%2Breproduction.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35079590%26rft_id%253Dinfo%253Apmid%252F11395771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35079590&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11395771&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169112500045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.2" data-doi="10.1038/ng.1050"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>,  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Gerke</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Shapiro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Crissman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Ghosh</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Chromosome-scale selective sweeps shape <em>Caenorhabditis elegans</em> genomic diversity.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">290</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.1050</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAndersen%26rft.auinit1%253DE.%2BC.%26rft.volume%253D44%26rft.issue%253D3%26rft.spage%253D285%26rft.epage%253D290%26rft.atitle%253DChromosome-scale%2Bselective%2Bsweeps%2Bshape%2BCaenorhabditis%2Belegans%2Bgenomic%2Bdiversity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.1050%26rft_id%253Dinfo%253Apmid%252F22286215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.1050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22286215&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.3" data-doi="10.1093/jhered/esq003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Morran</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Phillips</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Outcrossing and the Maintenance of Males within <em>C. elegans</em> Populations.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">S62</span>–<span class="cit-lpage">S74</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jhered/esq003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesq003%26rft_id%253Dinfo%253Apmid%252F20212008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/esq003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20212008&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282070600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bierne</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Tsitrone</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">David</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">An inbreeding model of associative overdominance during a population bottleneck.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">1981</span>–<span class="cit-lpage">1990</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBierne%26rft.auinit1%253DN.%26rft.volume%253D155%26rft.issue%253D4%26rft.spage%253D1981%26rft.epage%253D1990%26rft.atitle%253DAn%2BInbreeding%2BModel%2Bof%2BAssociative%2BOverdominance%2BDuring%2Ba%2BPopulation%2BBottleneck%26rft_id%253Dinfo%253Apmid%252F10924490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NS80LzE5ODEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjgxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-9.9.2811.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradic</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Costa</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. M.</span> <span class="cit-name-surname">Chelo</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>, pp. <span class="cit-fpage">193</span>–<span class="cit-lpage">210</span> in <span class="cit-source">Genotyping with Sequenom</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Orgogozo</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Rockman</span></span>. <span class="cit-publ-name">Humana Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.6" data-doi="10.1554/0014-3820(2000)054[0337:PSRATE]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burt</span>,  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Perspective: Sex, recombination and the efficacy of selection: was Weissman right?</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">351</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DBurt%26rft.auinit1%253DA.%26rft.volume%253D54%26rft.issue%253D2%26rft.spage%253D337%26rft.epage%253D351%26rft.atitle%253DPerspective%253A%2Bsex%252C%2Brecombination%252C%2Band%2Bthe%2Befficacy%2Bof%2Bselection--was%2BWeismann%2Bright%253F%26rft_id%253Dinfo%253Adoi%252F10.1554%252F0014-3820%25282000%2529054%255B0337%253APSRATE%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F10937212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/0014-3820(2000)054[0337:PSRATE]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10937212&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086886500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.7" data-doi="10.1186/1471-2148-14-116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carvalho</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. M.</span> <span class="cit-name-surname">Chelo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Goy</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Teotónio</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">a</span> <span class="cit-article-title">The role of hermaphrodites in the experimental evolution of increased outcrossing rates in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">116</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2148-14-116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol.%2BBiol.%26rft.volume%253D14%26rft.spage%253D116%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-14-116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-14-116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.8" data-doi="10.1186/1471-2148-14-117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carvalho</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. C.</span> <span class="cit-name-surname">Phillips</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Teotónio</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">b</span> <span class="cit-article-title">Hermaphrodite life history and the maintenance of partial selfing in experimental populations of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">117</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2148-14-117</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol.%2BBiol.%26rft.volume%253D14%26rft.spage%253D117%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-14-117%26rft_id%253Dinfo%253Apmid%252F24891140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-14-117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24891140&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.9" data-doi="10.1017/S0016672300029256"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Morgan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Charlesworth</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Multilocis models of inbreeding depression with synergistic selection and partial self-fertilization.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">177</span>–<span class="cit-lpage">194</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672300029256</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D57%26rft.spage%253D177%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300029256%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300029256&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1991FF16300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.10" data-doi="10.1146/annurev.es.18.110187.001321"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>,  <span class="cit-name-given-names">D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Charlesworth</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Inbreeding depression and its evolutionary consequences.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">268</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.es.18.110187.001321</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%2Band%2BSystematics%26rft.stitle%253DAnnu.%2BRev.%2BEcol.%2BSyst.%26rft.aulast%253DCharlesworth%26rft.auinit1%253DD%26rft.volume%253D18%26rft.issue%253D1%26rft.spage%253D237%26rft.epage%253D268%26rft.atitle%253DInbreeding%2BDepression%2BAnd%2BIts%2BEvolutionary%2BConsequences%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.es.18.110187.001321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.es.18.110187.001321&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1987K958800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.11" data-doi="10.2307/2409552"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>,  <span class="cit-name-given-names">D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Charlesworth</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Inbreeding depression with heterozygote advantage and its effect on selection for modifiers changing the outcrossing rate.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">870</span>–<span class="cit-lpage">888</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.1990.tb03811.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D44%26rft.spage%253D870%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2409552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2409552&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1990DV80100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.12" data-doi="10.2307/2409330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. T.</span> <span class="cit-name-surname">Morgan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Charlesworth</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Inbreeding depression, genetic load, and the evolution of outcrossing rates in a multilocus system with no linkage.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1469</span>–<span class="cit-lpage">1489</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.1990.tb03839.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D44%26rft.spage%253D1469%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2409330%26rft_id%253Dinfo%253Apmid%252F28564321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2409330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28564321&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990EM18900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.13" data-doi="10.1038/hdy.2013.100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chelo</span>,  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Carvalho</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Manoel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Proulx</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Teotónio</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The genetic basis and experimental evolution of inbreeding depression in Caenorhabiditis elegans.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">248</span>–<span class="cit-lpage">254</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2013.100</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D112%26rft.spage%253D248%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2013.100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2013.100&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.14" data-doi="10.1111/j.1558-5646.2012.01744.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chelo</span>,  <span class="cit-name-given-names">I. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Teotónio</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The opportunity for balancing selection in experimental populations of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">142</span>–<span class="cit-lpage">156</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.2012.01744.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D67%26rft.spage%253D142%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2012.01744.x%26rft_id%253Dinfo%253Apmid%252F23289568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2012.01744.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23289568&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000313364500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.15" data-doi="10.1093/aob/mcr127"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheptou</span>,  <span class="cit-name-given-names">P. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Clarifying BakerÂ’s Law.</span> <abbr class="cit-jnl-abbrev">Ann. Bot.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">633</span>–<span class="cit-lpage">641</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/aob/mcr127</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcr127%26rft_id%253Dinfo%253Apmid%252F21685434%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/aob/mcr127&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21685434&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>,  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Regression models and life tables.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">187</span>–<span class="cit-lpage">220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB%26rft.volume%253D34%26rft.spage%253D187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-9.9.2811.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crow</span>,  <span class="cit-name-given-names">J. F.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kimura</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-source">An Introduction to Population Genetics Theory</span>, <span class="cit-publ-name">Harper &amp; Row, Publishers</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.18" data-doi="10.1534/genetics.105.048207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cutter</span>,  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Nucleotide polymorphism and linkage disequilibrium in wild populations of the partial selfer <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">184</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.048207</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.048207%26rft_id%253Dinfo%253Apmid%252F16272415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcyLzEvMTcxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.19" data-doi="10.1111/j.1365-294X.2007.03330.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">David</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Pujol</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Viard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Castella</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Goudet</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Reliable selfing rate estimates from imperfect population genetic data.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">2474</span>–<span class="cit-lpage">2487</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2007.03330.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DDavid%26rft.auinit1%253DP.%26rft.volume%253D16%26rft.issue%253D12%26rft.spage%253D2474%26rft.epage%253D2487%26rft.atitle%253DReliable%2Bselfing%2Brate%2Bestimates%2Bfrom%2Bimperfect%2Bpopulation%2Bgenetic%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2007.03330.x%26rft_id%253Dinfo%253Apmid%252F17561907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2007.03330.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17561907&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247176000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.20" data-doi="10.1111/j.1558-5646.2007.00118.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dolgin</span>,  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Charlesworth</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Baird</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Cutter</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Inbreeding and outbreeding depression in Caenorhabditis nematodes.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">1339</span>–<span class="cit-lpage">1352</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.2007.00118.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DDolgin%26rft.auinit1%253DE.%2BS.%26rft.volume%253D61%26rft.issue%253D6%26rft.spage%253D1339%26rft.epage%253D1352%26rft.atitle%253DInbreeding%2Band%2Boutbreeding%2Bdepression%2Bin%2BCaenorhabditis%2Bnematodes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2007.00118.x%26rft_id%253Dinfo%253Apmid%252F17542844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2007.00118.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17542844&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247125100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.21" data-doi="10.1534/genetics.166.3.1269"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Estes</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. C.</span> <span class="cit-name-surname">Phillips</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Denver</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. K.</span> <span class="cit-name-surname">Thomas</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lynch</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Mutation accumulation in populations of varying size: the distribution of mutational effects for fitness correlates in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">1269</span>–<span class="cit-lpage">1279</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.166.3.1269</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DEstes%26rft.auinit1%253DS.%26rft.volume%253D166%26rft.issue%253D3%26rft.spage%253D1269%26rft.epage%253D1279%26rft.atitle%253DMutation%2BAccumulation%2Bin%2BPopulations%2Bof%2BVarying%2BSize%253A%2BThe%2BDistribution%2Bof%2BMutational%2BEffects%2Bfor%2BFitness%2BCorrelates%2Bin%2BCaenorhabditis%2Belegans%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.166.3.1269%26rft_id%253Dinfo%253Apmid%252F15082546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Ni8zLzEyNjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjgxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.22" data-doi="10.1534/genetics.112.142877"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaertner</span>,  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Parmenter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. V.</span> <span class="cit-name-surname">Rockman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Kruglyak</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. C.</span> <span class="cit-name-surname">Phillips</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">More than the sum of its parts: a complex epistatic network underlies natural variation in thermal preference behavior in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1533</span>–<span class="cit-lpage">1542</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.142877</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.142877%26rft_id%253Dinfo%253Apmid%252F23086219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzE1MzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjgxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.23" data-doi="10.1126/science.1194513"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldberg</span>,  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Kohn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Lande</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Robertson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Smith</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Species selection maintains self-incompatibility.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">330</span>: <span class="cit-fpage">493</span>–<span class="cit-lpage">495</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1194513</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGoldberg%26rft.auinit1%253DE.%2BE.%26rft.volume%253D330%26rft.issue%253D6003%26rft.spage%253D493%26rft.epage%253D495%26rft.atitle%253DSpecies%2BSelection%2BMaintains%2BSelf-Incompatibility%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1194513%26rft_id%253Dinfo%253Apmid%252F20966249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzAvNjAwMy80OTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjgxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.24" data-doi="10.1146/annurev.ecolsys.36.091704.175539"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goodwillie</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kalisz</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. G.</span> <span class="cit-name-surname">Eckert</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The evolutionary enigma of mixed mating systems in plants: occurrence, theoreticalexplanations, and empirical evidence.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">79</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.ecolsys.36.091704.175539</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEcol.%2BEvol.%2BSyst.%26rft.volume%253D36%26rft.spage%253D47%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.36.091704.175539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.36.091704.175539&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.25" data-doi="10.1371/journal.pgen.1007731"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guzella</span>,  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Dey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. M.</span> <span class="cit-name-surname">Chelo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Pino-Querido</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. F.</span> <span class="cit-name-surname">Pereira</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Slower environmental change hinders adaptation from standing genetic variation.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-elocation-id">e1007731</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1007731</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D14%26rft.spage%253De1007731%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1007731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1007731&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.26" data-doi="10.1111/j.1469-1809.1949.tb02418.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haldane</span>,  <span class="cit-name-given-names">J. B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1949</span> <span class="cit-article-title">The association of characters as a result of inbreeding and linkage.</span> <abbr class="cit-jnl-abbrev">Ann. Eugen.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">23</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-1809.1949.tb02418.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BEugen.%26rft.volume%253D15%26rft.spage%253D15%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1949.tb02418.x%26rft_id%253Dinfo%253Apmid%252F15403124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1949.tb02418.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15403124&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.27" data-doi="10.1086/523362"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Igic</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Lande</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Kohn</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Loss of Self-Incompatibility and Its Evolutionary Consequences.</span> <abbr class="cit-jnl-abbrev">Int. J. Plant Sci.</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">93</span>–<span class="cit-lpage">104</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/523362</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BPlant%2BSci.%26rft.volume%253D169%26rft.spage%253D93%26rft_id%253Dinfo%253Adoi%252F10.1086%252F523362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/523362&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000252245600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.28" data-doi="10.1111/j.0014-3820.2006.tb00525.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarne</span>,  <span class="cit-name-given-names">P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Auld</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Animals mix it up too: the distribution of self-fertilization among hermaphroditic animals.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">1816</span>–<span class="cit-lpage">1824</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.0014-3820.2006.tb00525.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DJarne%26rft.auinit1%253DP.%26rft.volume%253D60%26rft.issue%253D9%26rft.spage%253D1816%26rft.epage%253D1824%26rft.atitle%253DAnimals%2Bmix%2Bit%2Bup%2Btoo%253A%2Bthe%2Bdistribution%2Bof%2Bself-fertilization%2Bamong%2Bhermaphroditic%2Banimals.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0014-3820.2006.tb00525.x%26rft_id%253Dinfo%253Apmid%252F17089966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0014-3820.2006.tb00525.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17089966&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241226800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.29" data-doi="10.1086/593705"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>,  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Porcher</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. O.</span> <span class="cit-name-surname">Cheptou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. G.</span> <span class="cit-name-surname">Eckert</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Elle</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Correlations among fertility components can maintain mixed mating in plants.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/593705</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAm%2BNat%26rft.aulast%253DJohnston%26rft.auinit1%253DM.%2BO.%26rft.volume%253D173%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D11%26rft.atitle%253DCorrelations%2Bamong%2Bfertility%2Bcomponents%2Bcan%2Bmaintain%2Bmixed%2Bmating%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F593705%26rft_id%253Dinfo%253Apmid%252F19055444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/593705&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19055444&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261785300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.30" data-doi="10.1080/01621459.1958.10501452"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>,  <span class="cit-name-given-names">E. L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Meier</span></span></li></ol><cite>, <span class="cit-pub-date">1958</span> <span class="cit-article-title">Nonparametric estimation from incomplete observations.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">457</span>–<span class="cit-lpage">481</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/01621459.1958.10501452</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D53%26rft.spage%253D457%26rft_id%253Dinfo%253Adoi%252F10.1080%252F01621459.1958.10501452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/01621459.1958.10501452&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.31" data-doi="10.1016/j.jtbi.2006.12.030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>,  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mutation-selection balance in mixed-mating populations.</span> <abbr class="cit-jnl-abbrev">J. Theor. Biol.</abbr> <span class="cit-vol">246</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">365</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jtbi.2006.12.030</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DKelly%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D246%26rft.issue%253D2%26rft.spage%253D355%26rft.epage%253D365%26rft.atitle%253DMutation-selection%2Bbalance%2Bin%2Bmixed%2Bmating%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jtbi.2006.12.030%26rft_id%253Dinfo%253Apmid%252F17291538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jtbi.2006.12.030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17291538&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-book" id="cit-9.9.2811.32" data-doi="10.1017/CBO9780511623486"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-source">The Neutral Theory of Molecular Evolution</span>, <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">Cambridge</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/CBO9780511623486</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BNeutral%2BTheory%2Bof%2BMolecular%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1017%252FCBO9780511623486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/CBO9780511623486&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.33" data-doi="10.1534/genetics.115.176693"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lande</span>,  <span class="cit-name-given-names">R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Porcher</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Maintenance of Quantitative Genetic Variance Under Partial Self-Fertilization, with Implications for Evolution of Selfing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">891</span>–<span class="cit-lpage">906</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.176693</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.176693%26rft_id%253Dinfo%253Apmid%252F25969460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAwLzMvODkxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.34" data-doi="10.2307/2408514"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lande</span>,  <span class="cit-name-given-names">R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. W.</span> <span class="cit-name-surname">Schemske</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">The evolution of self-fertilization and inbreeding depression in plants. I. Genetic models.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">40</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.1985.tb04077.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D39%26rft.spage%253D24%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2408514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2408514&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1985ACA4300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.35" data-doi="10.2307/2410359"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lande</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. W.</span> <span class="cit-name-surname">Schemske</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. T.</span> <span class="cit-name-surname">Schultz</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">High inbreeding depression, selective interference among loci, and the threshold selfing rate for purging recessive lethal mutations.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">965</span>–<span class="cit-lpage">978</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.1994.tb05286.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D48%26rft.spage%253D965%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2410359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2410359&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994QE32300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-web" id="cit-9.9.2811.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lenth</span>,  <span class="cit-name-given-names">R. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-comment">lsmeans: Least-Squares Means. R package version 2.20–23. <a href="http://CRAN.R-project.org/package=lsmeans">http://CRAN.R-project.org/package=lsmeans</a></span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-book" id="cit-9.9.2811.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Walsh</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-source">Genetics and Analysis of Quantitative Traits</span>, <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.38" data-doi="10.1111/ele.12068"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Masri</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. D.</span> <span class="cit-name-surname">Schulte</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Timmermeyer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Thanisch</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. L.</span> <span class="cit-name-surname">Crummenerl</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Sex differences in host defence interfere with parasite-mediated selection for outcrossing during host-parasite coevolution.</span> <abbr class="cit-jnl-abbrev">Ecol. Lett.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">461</span>–<span class="cit-lpage">468</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/ele.12068</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BLett.%26rft.volume%253D16%26rft.spage%253D461%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.12068%26rft_id%253Dinfo%253Apmid%252F23301667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ele.12068&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23301667&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.39" data-doi="10.1534/genetics.115.178574"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matuszewski</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hermisson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kopp</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Catch Me if You Can: Adaptation from Standing Genetic Variation to a Moving Phenotypic Optimum.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">1255</span>–<span class="cit-lpage">1274</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.178574</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.178574%26rft_id%253Dinfo%253Apmid%252F26038348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMC80LzEyNTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjgxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-book" id="cit-9.9.2811.40" data-doi="10.1534/genetics.117.300406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noble</span>,  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Chelo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Guzella</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Afonso</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. D.</span> <span class="cit-name-surname">Riccardi</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Polygenicity and Epistasis Underlie Fitness-Proximal Traits in the Caenorhabditis elegans Multiparental Experimental Evolution (CeMEE) Panel.</span> <span class="cit-source">Genetics</span> <span class="cit-vol">207</span>: <span class="cit-fpage">1663</span>–<span class="cit-lpage">1685</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.117.300406</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300406%26rft_id%253Dinfo%253Apmid%252F29066469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNy80LzE2NjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjgxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.41" data-doi="10.1098/rspb.1996.0152"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Charlesworth</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Charlesworth</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Increased levels of polymorphim surrounding selectively maintained sites in highly selfing species.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">263</span>: <span class="cit-fpage">1033</span>–<span class="cit-lpage">1039</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rspb.1996.0152</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.1996.0152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.1996.0152&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.42" data-doi="10.1002/evl3.93"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noël</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Fruitet</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Lelaurin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Bonel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ségard</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Sexual selection and inbreeding: Two efficient ways to limit the accumulation of deleterious mutations.</span> <abbr class="cit-jnl-abbrev">Evolution Letters</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">80</span>–<span class="cit-lpage">92</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/evl3.93</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2BLetters%26rft.volume%253D3%26rft.spage%253D80%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fevl3.93%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/evl3.93&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.43" data-doi="10.1017/S0016672300002391"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohta</span>,  <span class="cit-name-given-names">T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kimura</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Development of associative overdominance through linkage disequilibrium in finite populations.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">165</span>–<span class="cit-lpage">177</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672300002391</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DOhta%26rft.auinit1%253DT.%26rft.volume%253D16%26rft.issue%253D2%26rft.spage%253D165%26rft.epage%253D177%26rft.atitle%253DDevelopment%2Bof%2Bassociative%2Boverdominance%2Bthrough%2Blinkage%2Bdisequilibrium%2Bin%2Bfinite%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300002391%26rft_id%253Dinfo%253Apmid%252F5516427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300002391&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5516427&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palsson</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pamilo</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The effects of deleterious mutations on linked, neutral variation in small populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">475</span>–<span class="cit-lpage">483</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPalsson%26rft.auinit1%253DS.%26rft.volume%253D153%26rft.issue%253D1%26rft.spage%253D475%26rft.epage%253D483%26rft.atitle%253DThe%2BEffects%2Bof%2BDeleterious%2BMutations%2Bon%2BLinked%252C%2BNeutral%2BVariation%2Bin%2BSmall%2BPopulations%26rft_id%253Dinfo%253Apmid%252F10471727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTUzLzEvNDc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.9.2811.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pinheiro</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Bates</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">DebRoy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Sarkar</span></span>, and <span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-comment"><em>nlme: Linear and Nonlinear Mixed Effects Models</em>. R package version 3.1–137</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.46" data-doi="10.1111/evo.13032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poullet</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Vielle</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Gimond</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Carvalho</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Teotónio</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Complex heterochrony underlies the evolution of <em>Caenorhabditis elegans</em> hermaphrodite sex allocation.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">2357</span>–<span class="cit-lpage">2369</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/evo.13032</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D70%26rft.spage%253D2357%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.13032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.13032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.47" data-doi="10.1371/journal.pgen.1000419"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Kruglyak</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Recombinational Landscape and Population Genomics of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-elocation-id">e1000419</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000419</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DRockman%26rft.auinit1%253DM.%2BV.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253De1000419%26rft.epage%253De1000419%26rft.atitle%253DRecombinational%2Blandscape%2Band%2Bpopulation%2Bgenomics%2Bof%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000419%26rft_id%253Dinfo%253Apmid%252F19283065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000419&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19283065&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.48" data-doi="10.1126/science.1194208"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>,  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. S.</span> <span class="cit-name-surname">Skrovanek</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Kruglyak</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Selection at linked sites shapes heritable phenotypic variation in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">330</span>: <span class="cit-fpage">372</span>–<span class="cit-lpage">376</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1194208</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRockman%26rft.auinit1%253DM.%2BV.%26rft.volume%253D330%26rft.issue%253D6002%26rft.spage%253D372%26rft.epage%253D376%26rft.atitle%253DSelection%2Bat%2BLinked%2BSites%2BShapes%2BHeritable%2BPhenotypic%2BVariation%2Bin%2BC.%2Belegans%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1194208%26rft_id%253Dinfo%253Apmid%252F20947766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzAvNjAwMi8zNzIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjgxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.49" data-doi="10.1534/genetics.115.178533"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roze</span>,  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Effects of Interference Between Selected Loci on the Mutation Load, Inbreeding Depression, and Heterosis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">745</span>–<span class="cit-lpage">757</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.178533</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.178533%26rft_id%253Dinfo%253Apmid%252F26269503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAxLzIvNzQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.50" data-doi="10.1534/genetics.116.187955"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roze</span>,  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Background selection in partially selfing populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">203</span>: <span class="cit-fpage">937</span>–<span class="cit-lpage">957</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.187955</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.187955%26rft_id%253Dinfo%253Apmid%252F27075726%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAzLzIvOTM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.51" data-doi="10.2307/2411188"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schultz</span>,  <span class="cit-name-given-names">S. T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lynch</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Mutation and extinction: the role of variable mutational effects, synergistic epistasis, beneficial mutations, and degree of outcrossing.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">1363</span>–<span class="cit-lpage">1371</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.1997.tb01459.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D51%26rft.spage%253D1363%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2411188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2411188&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997YE98000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.52" data-doi="10.1126/science.1151107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seidel</span>,  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. V.</span> <span class="cit-name-surname">Rockman</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Kruglyak</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Widespread genetic incompatibility in <em>C. elegans</em> maintained by balancing selection.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">319</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">594</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1151107</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1151107%26rft_id%253Dinfo%253Apmid%252F18187622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTkvNTg2My81ODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzkvMjgxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.53" data-doi="10.1038/35079578"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Siller</span>,  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Sexual selection and the maintenance of sex.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">411</span>: <span class="cit-fpage">689</span>–<span class="cit-lpage">692</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/35079578</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSiller%26rft.auinit1%253DS.%26rft.volume%253D411%26rft.issue%253D6838%26rft.spage%253D689%26rft.epage%253D692%26rft.atitle%253DSexual%2Bselection%2Band%2Bthe%2Bmaintenance%2Bof%2Bsex.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35079578%26rft_id%253Dinfo%253Apmid%252F11395770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35079578&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11395770&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169112500044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Skaug</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Fournier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Nielsen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Magnusson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Bolker</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-comment">Generalized lineag mixed models using AD Model Builder. R package version 0.7.5</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>,  <span class="cit-name-given-names">A. D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. C.</span> <span class="cit-name-surname">Phillips</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Selection and maintenance of androdioecy in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">975</span>–<span class="cit-lpage">982</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DStewart%26rft.auinit1%253DA.%2BD.%26rft.volume%253D160%26rft.issue%253D3%26rft.spage%253D975%26rft.epage%253D982%26rft.atitle%253DSelection%2Band%2BMaintenance%2Bof%2BAndrodioecy%2Bin%2BCaenorhabditis%2Belegans%26rft_id%253Dinfo%253Apmid%252F11901115%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYwLzMvOTc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-book" id="cit-9.9.2811.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stiernagle</span>,  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-source">Maintenance of C. elegans</span>, <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">Oxford</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.57" data-doi="10.1111/2041-210X.12588"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stoffel</span>,  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Esser</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kardos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Humble</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Nichols</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">inbreedr: An r package for the analysis of inbreeding based on genetic markers.</span> <abbr class="cit-jnl-abbrev">Methods Ecol. Evol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1331</span>–<span class="cit-lpage">1339</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/2041-210X.12588</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEcol.%2BEvol.%26rft.volume%253D7%26rft.spage%253D1331%26rft_id%253Dinfo%253Adoi%252F10.1111%252F2041-210X.12588%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/2041-210X.12588&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.58" data-doi="10.1016/0040-5809(71)90011-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sved</span>,  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">Linkage disequilibrium and homozygosity of chromosome segments in finite populations.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">125</span>–<span class="cit-lpage">141</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0040-5809(71)90011-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DSved%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D2%26rft.issue%253D2%26rft.spage%253D125%26rft.epage%253D141%26rft.atitle%253DLinkage%2Bdisequilibrium%2Band%2Bhomozygosity%2Bof%2Bchromosome%2Bsegments%2Bin%2Bfinite%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252871%252990011-6%26rft_id%253Dinfo%253Apmid%252F5170716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(71)90011-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5170716&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.59" data-doi="10.1111/j.1558-5646.2010.00966.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szulkin</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Bierne</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">David</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Heterozygosity-fitness correlations: a time for reappraisal.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">1202</span>–<span class="cit-lpage">1217</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.2010.00966.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DSzulkin%26rft.auinit1%253DM.%26rft.volume%253D64%26rft.issue%253D5%26rft.spage%253D1202%26rft.epage%253D1217%26rft.atitle%253DHeterozygosity-fitness%2Bcorrelations%253A%2Ba%2Btime%2Bfor%2Breappraisal.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2010.00966.x%26rft_id%253Dinfo%253Apmid%252F20148954%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2010.00966.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20148954&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277096400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.60" data-doi="10.1371/journal.pone.0035811"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teotónio</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Carvalho</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Manoel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Roque</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. M.</span> <span class="cit-name-surname">Chelo</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evolution of outcrossing in experimental populations of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e35811</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0035811</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DTeotonio%26rft.auinit1%253DH.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253De35811%26rft.epage%253De35811%26rft.atitle%253DEvolution%2Bof%2Boutcrossing%2Bin%2Bexperimental%2Bpopulations%2Bof%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0035811%26rft_id%253Dinfo%253Apmid%252F22540006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0035811&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22540006&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.61" data-doi="10.1534/genetics.115.186288"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teotónio</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Estes</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Phillips</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. F.</span> <span class="cit-name-surname">Baer</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Experimental evolution with Caenorhabditis nematodes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">691</span>–<span class="cit-lpage">716</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.186288</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.186288%26rft_id%253Dinfo%253Apmid%252F28592504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA2LzIvNjkxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.62" data-doi="10.1111/j.0014-3820.2006.tb01207.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teotónio</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Manoel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Phillips</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genetic variation for outcrossing among <em>Caenorhabditis elegans</em> isolates.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">1300</span>–<span class="cit-lpage">1305</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.0014-3820.2006.tb01207.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DTeotonio%26rft.auinit1%253DH.%26rft.volume%253D60%26rft.issue%253D6%26rft.spage%253D1300%26rft.epage%253D1305%26rft.atitle%253DGenetic%2Bvariation%2Bfor%2Boutcrossing%2Bamong%2BCaenorhabditis%2Belegans%2Bisolates.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0014-3820.2006.tb01207.x%26rft_id%253Dinfo%253Apmid%252F16892979%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0014-3820.2006.tb01207.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16892979&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238969900017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.63" data-doi="10.1186/s12915-014-0093-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Theologidis</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. M.</span> <span class="cit-name-surname">Chelo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Goy</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Teotónio</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Reproductive assurance drives transitions to self-fertilization in experimental <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">BMC Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">93</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12915-014-0093-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBiol.%26rft.volume%253D12%26rft.spage%253D93%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12915-014-0093-1%26rft_id%253Dinfo%253Apmid%252F25369737%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12915-014-0093-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25369737&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Therneau</span>,  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> coxme: mixed effects cox models. R package version 2–2.5.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.65" data-doi="10.1554/0014-3820(2000)054[1234:TFEOSM]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vassilieva</span>,  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Hook</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lynch</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The fitness effects of spontaneous mutations in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1234</span>–<span class="cit-lpage">1246</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.0014-3820.2000.tb00557.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DVassilieva%26rft.auinit1%253DL.%2BL.%26rft.volume%253D54%26rft.issue%253D4%26rft.spage%253D1234%26rft.epage%253D1246%26rft.atitle%253DThe%2Bfitness%2Beffects%2Bof%2Bspontaneous%2Bmutations%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1554%252F0014-3820%25282000%2529054%255B1234%253ATFEOSM%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F11005291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/0014-3820(2000)054[1234:TFEOSM]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11005291&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089317900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.66" data-doi="10.1017/S0016672300013446"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>,  <span class="cit-name-given-names">B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Cockerham</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Mixed self and random mating at two loci.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">247</span>–<span class="cit-lpage">262</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672300013446</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DWeir%26rft.auinit1%253DB.%2BS.%26rft.volume%253D21%26rft.issue%253D3%26rft.spage%253D247%26rft.epage%253D262%26rft.atitle%253DMixed%2Bself%2Band%2Brandom%2Bmating%2Bat%2Btwo%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300013446%26rft_id%253Dinfo%253Apmid%252F4731639%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300013446&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4731639&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F9%2F2811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-book" id="cit-9.9.2811.67"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>,  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source">Genetic data analysis II</span>, <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-9.9.2811.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ziehe</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Roberds</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Inbreeding depression due to overdominance in partially self-fertilizing plant populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">861</span>–<span class="cit-lpage">868</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DZiehe%26rft.auinit1%253DM.%26rft.volume%253D121%26rft.issue%253D4%26rft.spage%253D861%26rft.epage%253D868%26rft.atitle%253DInbreeding%2Bdepression%2Bdue%2Bto%2Boverdominance%2Bin%2Bpartially%2Bself-fertilizing%2Bplant%2Bpopulations.%26rft_id%253Dinfo%253Apmid%252F17246494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIxLzQvODYxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS85LzI4MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/9/2811.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/9/2799" title="Genetic Screening for Mutants with Altered Seminal Root Numbers in Hexaploid Wheat Using a High-Throughput Root Phenotyping Platform" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/9/2823" title="GWAS for Meat and Carcass Traits Using Imputed Sequence Level Genotypes in Pooled F2-Designs in Pigs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/9"><span>Volume 9 Issue 9, September 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/9.cover-source.jpg?itok=4kZTDz_V" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/9/2811" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-csX_R-TrZw7nzJZ9mGTA5DN7belO9hk4WBGV2Xk2m7k" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/9/2811" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/9/2811" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/9/2811" class="active" data-icon-position="" data-hide-link-title="0">Partial Selfing Can Reduce Genetic Loads While Maintaining Diversity During Experimental Evolution</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494049" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Uz0LpaJS2vVs6ahGektHw1bIg-C25zCAiilF0waej7U" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/9/2811.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/9/2811" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-3ogI3neK8e66Q1R5kiYZhYezFnXRYWieU0QVBvwYpm4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/9/2811" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/9/2811.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494049" id="node494049--2" data-pisa="ggg;9/9/2811" data-pisa-master="ggg;g3.119.400239" data-apath="/ggg/9/9/2811.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Partial Selfing Can Reduce Genetic Loads While Maintaining Diversity During Experimental Evolution</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5414-6625" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ivo M. Chelo</span>, <span class="highwire-citation-author" data-delta="1">Bruno Afonso</span>, <span class="highwire-citation-author" data-delta="2">Sara Carvalho</span>, <span class="highwire-citation-author" data-delta="3">Ioannis Theologidis</span>, <span class="highwire-citation-author" data-delta="4">Christine Goy</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-6830-4148" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ania Pino-Querido</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-0307-1907" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Stephen R. Proulx</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-1057-6882" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Henrique Teotónio</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2811-2821;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400239 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494049--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ivo M. Chelo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Instituto Gulbenkian de Ciência, Apartado 14, P-2781-901 Oeiras, Portugal</div><div class='author-affiliation'><span class='nlm-sup'>†</span>cE3c – Center for Ecology, Evolution and Environmental Changes, Faculdade de Ciências, Universidade de Lisboa, Lisboa, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ivo%2BM.%2BChelo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chelo%20IM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIvo%2BM.%2BChelo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5414-6625" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ivo M. Chelo</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bruno Afonso </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Instituto Gulbenkian de Ciência, Apartado 14, P-2781-901 Oeiras, Portugal</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institut de Biologie de l’École Normale Supérieure (IBENS), Inserm U1024, CNRS UMR 8197, F-75005 Paris, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bruno%2BAfonso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Afonso%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABruno%2BAfonso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sara Carvalho </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Instituto Gulbenkian de Ciência, Apartado 14, P-2781-901 Oeiras, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sara%2BCarvalho%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carvalho%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASara%2BCarvalho%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ioannis Theologidis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Institute of Molecular Biology and Biotechnology, Foundation for Research and Technology-Hellas, 73100 Heraklion, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ioannis%2BTheologidis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Theologidis%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIoannis%2BTheologidis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Christine Goy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Leibniz Research Institute for Environmental Medicine, 40225 Düsseldorf, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christine%2BGoy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goy%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristine%2BGoy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ania Pino-Querido </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Instituto Gulbenkian de Ciência, Apartado 14, P-2781-901 Oeiras, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ania%2BPino-Querido%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pino-Querido%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnia%2BPino-Querido%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6830-4148" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ania Pino-Querido</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Stephen R. Proulx </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Ecology, Evolution, and Marine Biology, University of California Santa Barbara, CA 93106</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BR.%2BProulx%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Proulx%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStephen%2BR.%2BProulx%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0307-1907" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Stephen R. Proulx</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Henrique Teotónio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institut de Biologie de l’École Normale Supérieure (IBENS), Inserm U1024, CNRS UMR 8197, F-75005 Paris, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Henrique%2BTeot%C3%B3nio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Teot%C3%B3nio%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHenrique%2BTeot%25C3%25B3nio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1057-6882" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Henrique Teotónio</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/9/2811" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/9/2811&amp;title=Partial%20Selfing%20Can%20Reduce%20Genetic%20Loads%20While%20Maintaining%20Diversity%20During%20Experimental%20Evolution" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/9/2811&amp;title=Partial%20Selfing%20Can%20Reduce%20Genetic%20Loads%20While%20Maintaining%20Diversity%20During%20Experimental%20Evolution" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/9/2811&amp;title=Partial%20Selfing%20Can%20Reduce%20Genetic%20Loads%20While%20Maintaining%20Diversity%20During%20Experimental%20Evolution" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/9/2811&amp;text=Partial%20Selfing%20Can%20Reduce%20Genetic%20Loads%20While%20Maintaining%20Diversity%20During%20Experimental%20Evolution" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/9/2811&amp;title=Partial%20Selfing%20Can%20Reduce%20Genetic%20Loads%20While%20Maintaining%20Diversity%20During%20Experimental%20Evolution" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/9/2811&amp;t=Partial%20Selfing%20Can%20Reduce%20Genetic%20Loads%20While%20Maintaining%20Diversity%20During%20Experimental%20Evolution" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/9/2811&amp;title=Partial%20Selfing%20Can%20Reduce%20Genetic%20Loads%20While%20Maintaining%20Diversity%20During%20Experimental%20Evolution" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/9/2811&amp;title=Partial%20Selfing%20Can%20Reduce%20Genetic%20Loads%20While%20Maintaining%20Diversity%20During%20Experimental%20Evolution" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494049/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494049" id="node494049--3" data-pisa="ggg;9/9/2811" data-pisa-master="ggg;g3.119.400239" data-apath="/ggg/9/9/2811.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Partial Selfing Can Reduce Genetic Loads While Maintaining Diversity During Experimental Evolution</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5414-6625" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ivo M. Chelo</span>, <span class="highwire-citation-author" data-delta="1">Bruno Afonso</span>, <span class="highwire-citation-author" data-delta="2">Sara Carvalho</span>, <span class="highwire-citation-author" data-delta="3">Ioannis Theologidis</span>, <span class="highwire-citation-author" data-delta="4">Christine Goy</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-6830-4148" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ania Pino-Querido</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-0307-1907" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Stephen R. Proulx</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-1057-6882" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Henrique Teotónio</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2811-2821;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400239 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494049--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ivo M. Chelo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Instituto Gulbenkian de Ciência, Apartado 14, P-2781-901 Oeiras, Portugal</div><div class='author-affiliation'><span class='nlm-sup'>†</span>cE3c – Center for Ecology, Evolution and Environmental Changes, Faculdade de Ciências, Universidade de Lisboa, Lisboa, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ivo%2BM.%2BChelo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chelo%20IM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIvo%2BM.%2BChelo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5414-6625" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ivo M. Chelo</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Bruno Afonso </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Instituto Gulbenkian de Ciência, Apartado 14, P-2781-901 Oeiras, Portugal</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institut de Biologie de l’École Normale Supérieure (IBENS), Inserm U1024, CNRS UMR 8197, F-75005 Paris, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bruno%2BAfonso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Afonso%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABruno%2BAfonso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sara Carvalho </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Instituto Gulbenkian de Ciência, Apartado 14, P-2781-901 Oeiras, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sara%2BCarvalho%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carvalho%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASara%2BCarvalho%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ioannis Theologidis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Institute of Molecular Biology and Biotechnology, Foundation for Research and Technology-Hellas, 73100 Heraklion, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ioannis%2BTheologidis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Theologidis%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIoannis%2BTheologidis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Christine Goy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Leibniz Research Institute for Environmental Medicine, 40225 Düsseldorf, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christine%2BGoy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goy%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristine%2BGoy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ania Pino-Querido </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Instituto Gulbenkian de Ciência, Apartado 14, P-2781-901 Oeiras, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ania%2BPino-Querido%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pino-Querido%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnia%2BPino-Querido%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6830-4148" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ania Pino-Querido</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Stephen R. Proulx </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Ecology, Evolution, and Marine Biology, University of California Santa Barbara, CA 93106</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BR.%2BProulx%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Proulx%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStephen%2BR.%2BProulx%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0307-1907" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Stephen R. Proulx</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Henrique Teotónio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institut de Biologie de l’École Normale Supérieure (IBENS), Inserm U1024, CNRS UMR 8197, F-75005 Paris, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Henrique%2BTeot%C3%B3nio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Teot%C3%B3nio%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHenrique%2BTeot%25C3%25B3nio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1057-6882" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Henrique Teotónio</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2811&amp;rft.epage=2821&amp;rft.atitle=Partial%20Selfing%20Can%20Reduce%20Genetic%20Loads%20While%20Maintaining%20Diversity%20During%20Experimental%20Evolution&amp;rft.volume=9&amp;rft.issue=9&amp;rft.date=2019-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Chelo%2C+Ivo+M.&amp;rft.au=Afonso%2C+Bruno&amp;rft.au=Carvalho%2C+Sara&amp;rft.au=Theologidis%2C+Ioannis&amp;rft.au=Goy%2C+Christine&amp;rft.au=Pino-Querido%2C+Ania&amp;rft.au=Proulx%2C+Stephen+R.&amp;rft.au=Teot%C3%B3nio%2C+Henrique"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494049/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494049/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494049/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494049/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494049/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494049/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494049/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494049/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494049/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494049/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494049/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494049/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/9/2811" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/9/2811.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/9/2811.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/9/2811.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
