<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/4/805" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/4/805.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/4/805.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/4/805.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/4/805.atom" />
<meta name="HW.pisa" content="ggg;6/4/805" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome-Wide Analysis of Transposon and Retroviral Insertions Reveals Preferential Integrations in Regions of DNA Flexibility" />
<meta name="DC.Identifier" content="10.1534/g3.115.026849" />
<meta name="DC.Date" content="2016-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Vrljicak et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="DNA transposons and retroviruses are important transgenic tools for genome engineering. An important consideration affecting the choice of transgenic vector is their insertion site preferences. Previous large-scale analyses of Ds transposon integration sites in plants were done on the basis of reporter gene expression or germ-line transmission, making it difficult to discern vertebrate integration preferences. Here, we compare over 1300 Ds transposon integration sites in zebrafish with Tol2 transposon and retroviral integration sites. Genome-wide analysis shows that Ds integration sites in the presence or absence of marker selection are remarkably similar and distributed throughout the genome. No strict motif was found, but a preference for structural features in the target DNA associated with DNA flexibility (Twist, Tilt, Rise, Roll, Shift, and Slide) was observed. Remarkably, this feature is also found in transposon and retroviral integrations in maize and mouse cells. Our findings show that structural features influence the integration of heterologous DNA in genomes, and have implications for targeted genome engineering." />
<meta name="DC.Contributor" content="Pavle Vrljicak" />
<meta name="DC.Contributor" content="Shijie Tao" />
<meta name="DC.Contributor" content="Gaurav K. Varshney" />
<meta name="DC.Contributor" content="Helen Ngoc Bao Quach" />
<meta name="DC.Contributor" content="Adita Joshi" />
<meta name="DC.Contributor" content="Matthew C. LaFave" />
<meta name="DC.Contributor" content="Shawn M. Burgess" />
<meta name="DC.Contributor" content="Karuna Sampath" />
<meta name="article:published_time" content="2016-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genome-Wide Analysis of Transposon and Retroviral Insertions Reveals Preferential Integrations in Regions of DNA Flexibility" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;DNA transposons and retroviruses are important transgenic tools for genome engineering. An important consideration affecting the choice of transgenic vector is their insertion site preferences. Previous large-scale analyses of Ds transposon integration sites in plants were done on the basis of reporter gene expression or germ-line transmission, making it difficult to discern vertebrate integration preferences. Here, we compare over 1300 Ds transposon integration sites in zebrafish with Tol2 transposon and retroviral integration sites. Genome-wide analysis shows that Ds integration sites in the presence or absence of marker selection are remarkably similar and distributed throughout the genome. No strict motif was found, but a preference for structural features in the target DNA associated with DNA flexibility (Twist, Tilt, Rise, Roll, Shift, and Slide) was observed. Remarkably, this feature is also found in transposon and retroviral integrations in maize and mouse cells. Our findings show that structural features influence the integration of heterologous DNA in genomes, and have implications for targeted genome engineering.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/04/01" />
<meta name="citation_mjid" content="ggg;6/4/805" />
<meta name="citation_id" content="6/4/805" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/4/805" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/4/805.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/4/805.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/4/805.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.026849" />
<meta name="citation_pmid" content="26818075" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="805" />
<meta name="citation_lastpage" content="817" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Pavle Vrljicak" />
<meta name="citation_author_institution" content="Division of Biomedical Sciences, Warwick Medical School, University of Warwick, Coventry CV4 7AJ, United Kingdom" />
<meta name="citation_author" content="Shijie Tao" />
<meta name="citation_author_institution" content="Temasek Life Sciences Laboratory, National University of Singapore, 117604, Singapore" />
<meta name="citation_author" content="Gaurav K. Varshney" />
<meta name="citation_author_institution" content="National Human Genome Research Institute, Bethesda, Maryland 20892-8004" />
<meta name="citation_author" content="Helen Ngoc Bao Quach" />
<meta name="citation_author_institution" content="Temasek Life Sciences Laboratory, National University of Singapore, 117604, Singapore" />
<meta name="citation_author" content="Adita Joshi" />
<meta name="citation_author_institution" content="Temasek Life Sciences Laboratory, National University of Singapore, 117604, Singapore" />
<meta name="citation_author" content="Matthew C. LaFave" />
<meta name="citation_author_institution" content="National Human Genome Research Institute, Bethesda, Maryland 20892-8004" />
<meta name="citation_author" content="Shawn M. Burgess" />
<meta name="citation_author_institution" content="National Human Genome Research Institute, Bethesda, Maryland 20892-8004" />
<meta name="citation_author_email" content="K.Sampath@warwick.ac.uk" />
<meta name="citation_author" content="Karuna Sampath" />
<meta name="citation_author_institution" content="Division of Biomedical Sciences, Warwick Medical School, University of Warwick, Coventry CV4 7AJ, United Kingdom" />
<meta name="citation_author_email" content="K.Sampath@warwick.ac.uk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0729-1977" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=A. W. Aday;citation_author=L. J. Zhu;citation_author=A. Lakshmanan;citation_author=J. Wang;citation_author=N. D. Lawson;citation_title=Identification of cis regulatory features in the embryonic zebrafish genome through large-scale profiling of H3K4me1 and H3K4me3 binding sites.;citation_pages=450-462;citation_volume=357;citation_year=2011;citation_issue=2;citation_pmid=21435340;citation_doi=10.1016/j.ydbio.2011.03.007" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Aiuti;citation_author=S. Slavin;citation_author=M. Aker;citation_author=F. Ficara;citation_author=S. Deola;citation_title=Correction of ADA-SCID by Stem Cell Gene Therapy Combined with Nonmyeloablative Conditioning;citation_pages=2410-2413;citation_volume=296;citation_year=2002;citation_issue=5577;citation_pmid=12089448;citation_doi=10.1126/science.1070104" />
<meta name="citation_reference" content="citation_journal_title=Methods in cell biology;citation_journal_abbrev=Methods Cell Biol;citation_author=A. Amsterdam;citation_author=G. K. Varshney;citation_author=S. M. Burgess;citation_title=Retroviral-mediated Insertional Mutagenesis in Zebrafish.;citation_pages=59-82;citation_volume=104;citation_year=2011;citation_pmid=21924157;citation_doi=10.1016/B978-0-12-374814-0.00004-5" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=G. Bourque;citation_author=B. Leong;citation_author=V. B. Vega;citation_author=X. Chen;citation_author=Y. L. Lee;citation_title=Evolution of the mammalian transcription factor binding repertoire via transposable elements.;citation_volume=18;citation_year=2008;citation_pmid=18682548;citation_doi=10.1101/gr.080663.108" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=B. R. Calvi;citation_author=T. J. Hong;citation_author=S. D. Findley;citation_author=W. M. Gelbart;citation_title=Evidence for a common evolutionary origin of inverted repeat transposons in Drosophila and plants: hobo, Activator, and Tam3.;citation_pages=465-471;citation_volume=66;citation_year=1991;citation_issue=3;citation_pmid=1651170;citation_doi=10.1016/0092-8674(81)90010-6" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Cavazzana-Calvo;citation_author=S. Hacein-Bey;citation_author=G. de Saint Basile;citation_author=F. Gross;citation_author=E. Yvon;citation_title=Gene Therapy of Human Severe Combined Immunodeficiency (SCID)-X1 Disease;citation_pages=669-672;citation_volume=288;citation_year=2000;citation_issue=5466;citation_pmid=10784449;citation_doi=10.1126/science.288.5466.669" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. P. Chadwick;citation_title=DXZ4 chromatin adopts an opposing conformation to that of the surrounding chromosome and acquires a novel inactive X-specific role involving CTCF and antisense transcripts.;citation_volume=18;citation_year=2008;citation_pmid=18456864;citation_doi=10.1101/gr.075713.107" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_author=L. D. Cheng;citation_author=X. Y. Jiang;citation_author=Y. M. Tian;citation_author=J. Chen;citation_author=S. M. Zou;citation_title=The goldfish hAT-family transposon Tgf2 is capable of autonomous excision in zebrafish embryos.;citation_pages=74-78;citation_volume=536;citation_year=2014;citation_pmid=24321692;citation_doi=10.1016/j.gene.2013.11.084" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=G. E. Crooks;citation_author=G. Hon;citation_author=J. M. Chandonia;citation_author=S. E. Brenner;citation_title=WebLogo: A Sequence Logo Generator;citation_pages=1188-1190;citation_volume=14;citation_year=2004;citation_issue=6;citation_pmid=15173120;citation_doi=10.1101/gr.849004" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=K. Davie;citation_author=J. Jacobs;citation_author=M. Atkins;citation_author=D. Potier;citation_author=V. Christiaens;citation_title=Discovery of transcription factors and regulatory regions driving in vivo tumor development by ATAC-seq and FAIRE-seq open chromatin profiling.;citation_pages=e1004994;citation_volume=11;citation_year=2015;citation_pmid=25679813;citation_doi=10.1371/journal.pgen.1004994" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. de Jong;citation_author=W. Akhtar;citation_author=J. Badhai;citation_author=A. G. Rust;citation_author=R. Rad;citation_title=Chromatin landscapes of retroviral and transposon integration profiles.;citation_pages=e1004250;citation_volume=10;citation_year=2014;citation_pmid=24721906;citation_doi=10.1371/journal.pgen.1004250" />
<meta name="citation_reference" content="citation_journal_title=J. Virol.;citation_author=S. S. De Ravin;citation_author=L. Su;citation_author=N. Theobald;citation_author=U. Choi;citation_author=J. L. Macpherson;citation_title=Enhancers are major targets for murine leukemia virus vector integration.;citation_volume=88;citation_year=2014;citation_pmid=24501411;citation_doi=10.1128/JVI.00011-14" />
<meta name="citation_reference" content="citation_journal_title=Cell Reports;citation_author=J. De Rijck;citation_author=C. de Kogel;citation_author=J. Demeulemeester;citation_author=S. Vets;citation_author=S. El Ashkar;citation_title=The BET family of proteins targets moloney murine leukemia virus integration near transcription start sites.;citation_pages=886-894;citation_volume=5;citation_year=2013;citation_pmid=24183673;citation_doi=10.1016/j.celrep.2013.09.040" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Emelyanov;citation_author=Y. Gao;citation_author=N. I. Naqvi;citation_author=S. Parinov;citation_title=Trans-kingdom transposition of the maize dissociation element.;citation_volume=174;citation_year=2006;citation_pmid=16951067;citation_doi=10.1534/genetics.106.061184" />
<meta name="citation_reference" content="citation_journal_title=J. Virol.;citation_author=A. Faschinger;citation_author=F. Rouault;citation_author=J. Sollner;citation_author=A. Lukas;citation_author=B. Salmons;citation_title=Mouse mammary tumor virus integration site selection in human and mouse genomes.;citation_volume=82;citation_year=2008;citation_pmid=18032509;citation_doi=10.1128/JVI.02098-07" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=G. J. Faulkner;citation_author=Y. Kimura;citation_author=C. O. Daub;citation_author=S. Wani;citation_author=C. Plessy;citation_title=The regulated retrotransposon transcriptome of mammalian cells.;citation_pages=563-571;citation_volume=41;citation_year=2009;citation_issue=5;citation_pmid=19377475;citation_doi=10.1038/ng.368" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=C. Feschotte;citation_title=Transposable elements and the evolution of regulatory networks.;citation_pages=397-405;citation_volume=9;citation_year=2008;citation_issue=5;citation_pmid=18368054;citation_doi=10.1038/nrg2337" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. A. Genovesi;citation_author=C. G. Ng;citation_author=M. J. Davis;citation_author=M. Remke;citation_author=M. D. Taylor;citation_title=Sleeping Beauty mutagenesis in a mouse medulloblastoma model defines networks that discriminate between human molecular subgroups.;citation_volume=110;citation_year=2013;citation_pmid=24167280;citation_doi=10.1073/pnas.1318639110" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. G. Goll;citation_author=R. Anderson;citation_author=D. Y. Stainier;citation_author=A. C. Spradling;citation_author=M. E. Halpern;citation_title=Transcriptional silencing and reactivation in transgenic zebrafish.;citation_volume=182;citation_year=2009;citation_pmid=19433629;citation_doi=10.1534/genetics.109.102079" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Guo;citation_author=J. M. Park;citation_author=B. Cui;citation_author=E. Humes;citation_author=S. Gangadharan;citation_title=Integration profiling of gene function with dense maps of transposon integration.;citation_volume=195;citation_year=2013;citation_pmid=23893486;citation_doi=10.1534/genetics.113.152744" />
<meta name="citation_reference" content="citation_journal_title=J. Virol.;citation_author=S. S. Gupta;citation_author=T. Maetzig;citation_author=G. N. Maertens;citation_author=A. Sharif;citation_author=M. Rothe;citation_title=Bromo- and extraterminal domain chromatin regulators serve as cofactors for murine leukemia virus integration.;citation_volume=87;citation_year=2013;citation_pmid=24049186;citation_doi=10.1128/JVI.01942-13" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_author=S. A. Harvey;citation_author=I. Sealy;citation_author=R. Kettleborough;citation_author=F. Fenyes;citation_author=R. White;citation_title=Identification of the zebrafish maternal and paternal transcriptomes.;citation_volume=140;citation_year=2013;citation_pmid=23720042;citation_doi=10.1242/dev.095091" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=K. Howe;citation_author=M. D. Clark;citation_author=C. F. Torroja;citation_author=J. Torrance;citation_author=C. Berthelot;citation_title=The zebrafish reference genome sequence and its relationship to the human genome.;citation_pages=498-503;citation_volume=496;citation_year=2013;citation_pmid=23594743;citation_doi=10.1038/nature12111" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. W. Huang;citation_author=B. T. Sherman;citation_author=R. A. Lempicki;citation_title=Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists.;citation_volume=37;citation_year=2009;citation_pmid=19033363;citation_doi=10.1093/nar/gkn923" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=D. W. Huang;citation_author=B. T. Sherman;citation_author=R. A. Lempicki;citation_title=Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources.;citation_pages=44-57;citation_volume=4;citation_year=2009;citation_issue=1;citation_pmid=19131956;citation_doi=10.1038/nprot.2008.211" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_author=U. Irion;citation_author=J. Krauss;citation_author=C. Nusslein-Volhard;citation_title=Precise and efficient genome editing in zebrafish using the CRISPR/Cas9 system.;citation_volume=141;citation_year=2014;citation_pmid=25411213;citation_doi=10.1242/dev.115584" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=K. Kawakami;citation_title=Tol2: a versatile gene transfer vector in vertebrates.;citation_pages=S7;citation_volume=8;citation_year=2007;citation_pmid=18047699;citation_doi=10.1186/gb-2007-8-s1-s7" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. Kawakami;citation_author=T. Noda;citation_title=Transposition of the Tol2 Element, an Ac-Like Element From the Japanese Medaka Fish Oryzias latipes, in Mouse Embryonic Stem Cells;citation_pages=895-899;citation_volume=166;citation_year=2004;citation_issue=2;citation_pmid=15020474;citation_doi=10.1534/genetics.166.2.895" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=K. Kawakami;citation_author=H. Takeda;citation_author=N. Kawakami;citation_author=M. Kobayashi;citation_author=N. Matsuda;citation_title=A transposon-mediated gene trap approach identifies developmentally regulated genes in zebrafish.;citation_pages=133-144;citation_volume=7;citation_year=2004;citation_issue=1;citation_pmid=15239961;citation_doi=10.1016/j.devcel.2004.06.005" />
<meta name="citation_reference" content="citation_journal_title=BMC developmental biology [electronic resource];citation_journal_abbrev=BMC Dev Biol;citation_author=K. Kawakami;citation_author=G. Abe;citation_author=T. Asada;citation_author=K. Asakawa;citation_author=R. Fukuda;citation_title=zTrap: zebrafish gene trap and enhancer trap database.;citation_pages=105-105;citation_volume=10;citation_year=2010;citation_pmid=20950494;citation_doi=10.1186/1471-213X-10-105" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. J. Kent;citation_title=BLAT–the BLAST-like alignment tool.;citation_volume=12;citation_year=2002;citation_pmid=11932250;citation_doi=10.1101/gr.229202. Article published online before March 2002" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=M. G. Kidwell;citation_author=D. R. Lisch;citation_title=Transposable elements and host genome evolution.;citation_pages=95-99;citation_volume=15;citation_year=2000;citation_issue=3;citation_pmid=10675923;citation_doi=10.1016/S0169-5347(99)01817-0" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Koga;citation_author=M. Suzuki;citation_author=H. Inagaki;citation_author=Y. Bessho;citation_author=H. Hori;citation_title=Transposable element in fish.;citation_pages=30-30;citation_volume=383;citation_year=1996;citation_issue=6595;citation_pmid=8779712" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=I. Kondrychyn;citation_author=M. Garcia-Lecea;citation_author=A. Emelyanov;citation_author=S. Parinov;citation_author=V. Korzh;citation_title=Genome-wide analysis of Tol2 transposon reintegration in zebrafish.;citation_pages=418-418;citation_volume=10;citation_year=2009;citation_pmid=19737393;citation_doi=10.1186/1471-2164-10-418" />
<meta name="citation_reference" content="citation_journal_abbrev=Zebrafish;citation_author=I. Kondrychyn;citation_author=C. Teh;citation_author=M. Garcia-Lecea;citation_author=Y. Guan;citation_author=A. Kang;citation_title=Zebrafish Enhancer TRAP transgenic line database ZETRAP 2.0.;citation_pages=181-182;citation_volume=8;citation_year=2011;citation_issue=4;citation_pmid=22181660;citation_doi=10.1089/zeb.2011.0718" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. C. LaFave;citation_author=G. K. Varshney;citation_author=D. E. Gildea;citation_author=T. G. Wolfsberg;citation_author=A. D. Baxevanis;citation_title=MLV integration site selection is driven by strong enhancers and active promoters.;citation_volume=42;citation_year=2014;citation_pmid=24464997;citation_doi=10.1093/nar/gkt1399" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. S. Lander;citation_author=L. M. Linton;citation_author=B. Birren;citation_author=C. Nusbaum;citation_author=M. C. Zody;citation_title=Initial sequencing and analysis of the human genome.;citation_pages=860-921;citation_volume=409;citation_year=2001;citation_issue=6822;citation_pmid=11237011;citation_doi=10.1038/35057062" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Langmead;citation_author=C. Trapnell;citation_author=M. Pop;citation_author=S. L. Salzberg;citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.;citation_pages=R25-R25;citation_volume=10;citation_year=2009;citation_issue=3;citation_pmid=19261174;citation_doi=10.1186/gb-2009-10-3-r25" />
<meta name="citation_reference" content="citation_journal_title=Genesis (New York, N.Y. :  2000);citation_journal_abbrev=Genesis;citation_author=Q. Liang;citation_author=J. Kong;citation_author=J. Stalker;citation_author=A. Bradley;citation_title=Chromosomal mobilization and reintegration of Sleeping Beauty and PiggyBac transposons.;citation_pages=404-408;citation_volume=47;citation_year=2009;citation_issue=6;citation_pmid=19391106;citation_doi=10.1002/dvg.20508" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=S. Lim;citation_author=Y. Wang;citation_author=X. Yu;citation_author=Y. Huang;citation_author=M. S. Featherstone;citation_title=A simple strategy for heritable chromosomal deletions in zebrafish via the combinatorial action of targeting nucleases.;citation_pages=R69;citation_volume=14;citation_year=2013;citation_pmid=23815890;citation_doi=10.1186/gb-2013-14-7-r69" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Lin;citation_author=N. Gaiano;citation_author=P. Culp;citation_author=J. C. Burns;citation_author=T. Friedmann;citation_title=Integration and germ-line transmission of a pseudotyped retroviral vector in zebrafish;citation_pages=666-669;citation_volume=265;citation_year=1994;citation_issue=5172;citation_pmid=8036514;citation_doi=10.1126/science.8036514" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. S. Linheiro;citation_author=C. M. Bergman;citation_title=Testing the palindromic target site model for DNA transposon insertion using the Drosophila melanogaster P-element.;citation_volume=36;citation_year=2008;citation_pmid=18829720;citation_doi=10.1093/nar/gkn563" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harbor Symposia on Quantitative Biology;citation_journal_abbrev=Cold Spring Harb Symp Quant Biol;citation_author=B. McClintock;citation_title=CHROMOSOME ORGANIZATION AND GENIC EXPRESSION;citation_pages=13-47;citation_volume=16;citation_year=1951;citation_issue=0;citation_pmid=14942727;citation_doi=10.1101/SQB.1951.016.01.004" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=D. M. McGaughey;citation_author=H. O. Abaan;citation_author=R. M. Miller;citation_author=P. A. Kropp;citation_author=L. C. Brody;citation_title=Genomics of CpG methylation in developing and developed zebrafish.;citation_volume=4;citation_year=2014;citation_pmid=24657902;citation_doi=10.1534/g3.113.009514" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. McGrail;citation_author=J. M. Hatler;citation_author=X. Kuang;citation_author=H. K. Liao;citation_author=K. Nannapaneni;citation_title=Somatic mutagenesis with a Sleeping Beauty transposon system leads to solid tumor formation in zebrafish.;citation_pages=e18826-e18826;citation_volume=6;citation_year=2011;citation_issue=4;citation_pmid=21533036;citation_doi=10.1371/journal.pone.0018826" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=M. Mullins;citation_title=GENETIC NOMENCLATURE GUIDE. ZEBRAFISH.;citation_pages=31-32;citation_volume=Mar;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=W. K. Olson;citation_author=A. A. Gorin;citation_author=X. J. Lu;citation_author=L. M. Hock;citation_author=V. B. Zhurkin;citation_title=DNA sequence-dependent deformability deduced from protein-DNA crystal complexes;citation_pages=11163-11168;citation_volume=95;citation_year=1998;citation_issue=19;citation_pmid=9736707;citation_doi=10.1073/pnas.95.19.11163" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=M. E. Potok;citation_author=D. A. Nix;citation_author=T. J. Parnell;citation_author=B. R. Cairns;citation_title=Reprogramming the maternal zebrafish genome after fertilization to match the paternal methylation pattern.;citation_pages=759-772;citation_volume=153;citation_year=2013;citation_pmid=23663776;citation_doi=10.1016/j.cell.2013.04.030" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=H. N. Quach;citation_author=S. Tao;citation_author=P. Vrljicak;citation_author=A. Joshi;citation_author=H. Ruan;citation_title=A multifunctional mutagenesis system for analysis of gene function in zebrafish.;citation_volume=5;citation_year=2015;citation_pmid=25840430;citation_doi=10.1534/g3.114.015842" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=R. Rad;citation_author=L. Rad;citation_author=W. Wang;citation_author=A. Strong;citation_author=H. Ponstingl;citation_title=A conditional piggyBac transposition system for genetic screening in mice identifies oncogenic networks in pancreatic cancer.;citation_pages=47-56;citation_volume=47;citation_year=2015;citation_pmid=25485836;citation_doi=10.1038/ng.3164" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=J. A. Roberts;citation_author=I. Miguel-Escalada;citation_author=K. J. Slovik;citation_author=K. T. Walsh;citation_author=Y. Hadzhiev;citation_title=Targeted transgene integration overcomes variability of position effects in zebrafish;citation_pages=715-724;citation_volume=141;citation_year=2014;citation_issue=3;citation_pmid=24449846;citation_doi=10.1242/dev.100347" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Sharma;citation_author=R. C. Larue;citation_author=M. R. Plumb;citation_author=N. Malani;citation_author=F. Male;citation_title=BET proteins promote efficient murine leukemia virus integration at transcription start sites.;citation_volume=110;citation_year=2013;citation_pmid=23818621;citation_doi=10.1073/pnas.1307157110" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=S. Shpiz;citation_author=S. Ryazansky;citation_author=I. Olovnikov;citation_author=Y. Abramov;citation_author=A. Kalmykova;citation_title=Euchromatic transposon insertions trigger production of novel Pi- and endo-siRNAs at the target sites in the drosophila germline.;citation_pages=e1004138;citation_volume=10;citation_year=2014;citation_pmid=24516406;citation_doi=10.1371/journal.pgen.1004138" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=I. Simeonova;citation_author=V. Lejour;citation_author=B. Bardot;citation_author=R. Bouarich-Bourimi;citation_author=A. Morin;citation_title=Fuzzy tandem repeats containing p53 response elements may define species-specific p53 target genes.;citation_pages=e1002731-e1002731;citation_volume=8;citation_year=2012;citation_issue=6;citation_pmid=22761580;citation_doi=10.1371/journal.pgen.1002731" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=G. Song;citation_author=Q. Li;citation_author=Y. Long;citation_author=Q. Gu;citation_author=P. B. Hackett;citation_title=Effective gene trapping mediated by Sleeping Beauty transposon.;citation_pages=e44123-e44123;citation_volume=7;citation_year=2012;citation_issue=8;citation_pmid=22952894;citation_doi=10.1371/journal.pone.0044123" />
<meta name="citation_reference" content="citation_journal_abbrev=Retrovirology;citation_author=B. Studamire;citation_author=S. P. Goff;citation_title=Host proteins interacting with the Moloney murine leukemia virus integrase: multiple transcriptional regulators and chromatin binding factors.;citation_pages=48-48;citation_volume=5;citation_year=2008;citation_pmid=18554410;citation_doi=10.1186/1742-4690-5-48" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=H. Takeda;citation_author=Z. Wei;citation_author=H. Koso;citation_author=A. G. Rust;citation_author=C. C. Yew;citation_title=Transposon mutagenesis identifies genes and evolutionary forces driving gastrointestinal tract tumor progression.;citation_pages=142-150;citation_volume=47;citation_year=2015;citation_pmid=25559195;citation_doi=10.1038/ng.3175" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=D. T. Ting;citation_author=D. Lipson;citation_author=S. Paul;citation_author=B. W. Brannigan;citation_author=S. Akhavanfard;citation_title=Aberrant overexpression of satellite repeats in pancreatic and other epithelial cancers.;citation_volume=331;citation_year=2011;citation_pmid=21233348;citation_doi=10.1126/science.1200801" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. Tyekucheva;citation_author=R. H. Yolken;citation_author=W. R. McCombie;citation_author=J. Parla;citation_author=M. Kramer;citation_title=Establishing the baseline level of repetitive element expression in the human cortex.;citation_pages=495-495;citation_volume=12;citation_year=2011;citation_pmid=21985647;citation_doi=10.1186/1471-2164-12-495" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=G. K. Varshney;citation_author=J. Lu;citation_author=D. E. Gildea;citation_author=H. Huang;citation_author=W. Pei;citation_title=A large-scale zebrafish gene knockout resource for the genome-wide study of gene function.;citation_volume=23;citation_year=2013;citation_pmid=23382537;citation_doi=10.1101/gr.151464.112" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=T. J. Vigdal;citation_author=C. D. Kaufman;citation_author=Z. Izsvak;citation_author=D. F. Voytas;citation_author=Z. Ivics;citation_title=Common physical properties of DNA affecting target site selection of sleeping beauty and other Tc1/mariner transposable elements.;citation_pages=441-452;citation_volume=323;citation_year=2002;citation_issue=3;citation_pmid=12381300;citation_doi=10.1016/S0022-2836(02)00991-9" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=E. Vollbrecht;citation_author=J. Duvick;citation_author=J. P. Schares;citation_author=K. R. Ahern;citation_author=P. Deewatthanawong;citation_title=Genome-wide distribution of transposed Dissociation elements in maize.;citation_volume=22;citation_year=2010;citation_pmid=20581308;citation_doi=10.1105/tpc.109.073452" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=G. P. Wang;citation_author=A. Ciuffi;citation_author=J. Leipzig;citation_author=C. C. Berry;citation_author=F. D. Bushman;citation_title=HIV integration site selection: analysis by massively parallel pyrosequencing reveals association with epigenetic modifications.;citation_volume=17;citation_year=2007;citation_pmid=17545577;citation_doi=10.1101/gr.6286907" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. Wang;citation_author=J. Zeng;citation_author=C. B. Lowe;citation_author=R. G. Sellers;citation_author=S. R. Salama;citation_title=Species-specific endogenous retroviruses shape the transcriptional network of the human tumor suppressor protein p53.;citation_volume=104;citation_year=2007;citation_pmid=18003932;citation_doi=10.1073/pnas.0703637104" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. H. Waterston;citation_author=K. Lindblad-Toh;citation_author=E. Birney;citation_author=J. Rogers;citation_author=J. F. Abril;citation_title=Initial sequencing and comparative analysis of the mouse genome.;citation_pages=520-562;citation_volume=420;citation_year=2002;citation_issue=6915;citation_pmid=12466850;citation_doi=10.1038/nature01262" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. F. Weil;citation_author=R. Kunze;citation_title=Transposition of maize Ac/Ds transposable elements in the yeast Saccharomyces cerevisiae.;citation_pages=187-190;citation_volume=26;citation_year=2000;citation_issue=2;citation_pmid=11017074;citation_doi=10.1038/82827" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=X. Wu;citation_author=Y. Li;citation_author=B. Crise;citation_author=S. M. Burgess;citation_title=Transcription Start Regions in the Human Genome Are Favored Targets for MLV Integration;citation_pages=1749-1751;citation_volume=300;citation_year=2003;citation_issue=5626;citation_pmid=12805549;citation_doi=10.1126/science.1083413" />
<meta name="citation_reference" content="citation_journal_title=Journal of Virology;citation_journal_abbrev=J. Virol.;citation_author=X. Wu;citation_author=Y. Li;citation_author=B. Crise;citation_author=S. M. Burgess;citation_author=D. J. Munroe;citation_title=Weak Palindromic Consensus Sequences Are a Common Feature Found at the Integration Target Sites of Many Retroviruses;citation_pages=5211-5214;citation_volume=79;citation_year=2005;citation_issue=8;citation_pmid=15795304;citation_doi=10.1128/JVI.79.8.5211-5214.2005" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Q. Zhu;citation_author=G. M. Pao;citation_author=A. M. Huynh;citation_author=H. Suh;citation_author=N. Tonnu;citation_title=BRCA1 tumour suppression occurs via heterochromatin-mediated silencing.;citation_pages=179-184;citation_volume=477;citation_year=2011;citation_issue=7363;citation_pmid=21901007;citation_doi=10.1038/nature10371" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genome-Wide Analysis of Transposon and Retroviral Insertions Reveals Preferential Integrations in Regions of DNA Flexibility" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
<meta name="twitter:description" content="DNA transposons and retroviruses are important transgenic tools for genome engineering. An important consideration affecting the choice of transgenic vector is their insertion site preferences. Previous large-scale analyses of Ds transposon integration sites in plants were done on the basis of reporter gene expression or germ-line transmission, making it difficult to discern vertebrate integration preferences. Here, we compare over 1300 Ds transposon integration sites in zebrafish with Tol2 transposon and retroviral integration sites. Genome-wide analysis shows that Ds integration sites in the presence or absence of marker selection are remarkably similar and distributed throughout the genome. No strict motif was found, but a preference for structural features in the target DNA associated with DNA flexibility (Twist, Tilt, Rise, Roll, Shift, and Slide) was observed. Remarkably, this feature is also found in transposon and retroviral integrations in maize and mouse cells. Our findings show that structural features influence the integration of heterologous DNA in genomes, and have implications for targeted genome engineering." />
<meta name="og-title" property="og:title" content="Genome-Wide Analysis of Transposon and Retroviral Insertions Reveals Preferential Integrations in Regions of DNA Flexibility" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/4/805" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="DNA transposons and retroviruses are important transgenic tools for genome engineering. An important consideration affecting the choice of transgenic vector is their insertion site preferences. Previous large-scale analyses of Ds transposon integration sites in plants were done on the basis of reporter gene expression or germ-line transmission, making it difficult to discern vertebrate integration preferences. Here, we compare over 1300 Ds transposon integration sites in zebrafish with Tol2 transposon and retroviral integration sites. Genome-wide analysis shows that Ds integration sites in the presence or absence of marker selection are remarkably similar and distributed throughout the genome. No strict motif was found, but a preference for structural features in the target DNA associated with DNA flexibility (Twist, Tilt, Rise, Roll, Shift, and Slide) was observed. Remarkably, this feature is also found in transposon and retroviral integrations in maize and mouse cells. Our findings show that structural features influence the integration of heterologous DNA in genomes, and have implications for targeted genome engineering." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
    <title>Genome-Wide Analysis of Transposon and Retroviral Insertions Reveals Preferential Integrations in Regions of DNA Flexibility | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bed9eK4HgZgeq1SWIHZ3nUSsO_Lg6SY0VT6zRKv2uw4__AOTOtpTA1sp4jw4ldY-8h8LueAv-V2MjVmPlkIZ--AU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"C2oezdMLfVt7um0gv5xCtSvozOEA3FNE4P-90OfWCIM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460125","apath":"\/ggg\/6\/4\/805.atom","pisa":"ggg;6\/4\/805","ac":{"\/ggg\/6\/4\/805.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/4\/805.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/4\/805"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/4\/805"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/4\/805.figures-only":true,"\/content\/6\/4\/805":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022CvlAPx3adp8u17PiI4Sg4nvcADV7IVTFBotRoNDNEWPcb534xRdt+9oXSMLwNkuIGc1AfYL5AKG5n8qF9GFi5L9uFbm4SG7HnD9GezKKGjpMIGy3BKIX1hkCIPFVIfWzvbkmSlzExyMiLFezQQ12vEG69opc4J\\\\\\\/F7rFsgWkRplJMoDPFALQ6SsIpDEoY6PFveQSyqg5WjS+L8sMcGD3ENmx9jRTQxEjloTRJbBVWgQAuOrWqzNb74MM59p1dr8hF\\\\\\\/JWy+2tiqpGs+gPzWEbhs0yXtGH3Mn+u8KPGPgve9CUK5J5Qjf7USZhNMYwMsqqyn2vVRZ9oegXQfdaTX+V8iqEHM68mO\\\\\\\/fLy4eTLgbl7zKSjQrAAEHQ8HOkFSRBgobOLu81fY9NBfrc0WlytXjws+q8IsT3nwckeE0wwEYf\\\\\\\/BmsyEEMNtGouazIVnhSd16Vmbu4e590W05+ZTVqr5eRHk9y90Y+tdkyp2JN27duANi8+Uez\\\\\\\/GdD4W2XsvcS0X5fTgIqi8IUjRE6Ci\\\\\\\/21M7npKP6OM+C7mwlT91UckEdcHGZOyIhRv6bsmw7KcMoIykHGcDX2iDufp3Ds9KXAp2NsmLl1JEnzS9XZlYsxl5+FtWwUo4hPksFymYMWILhaB+qmAgVLxnvFzAYQmoq62QplQf0p\\\\\\\/gk1EkN+nak1eUMmSXJFZbZbCVyzMlmwe77tYaALArXMEj8DHwCRh6K9xunGJ8LgRNIs0aqhEZp7vw1HJPU75MMYZgDrnihrCqi2Pl\\\\\\\/ehI\\\\\\\/R2Wh1nauTw4biu6RnGVPYjFW9ek7xkRML1nufD016sv\\\\\\\/Dax4Yntvq6a1TCEkVPbwdz2ubI\\\\\\\/JbyxqaQEthv3n6kkBpD7zdHcZkI2rIKSNLwCsL0r0EChnHPUF+5zIVOEzrsVnwdKpM+ENf22UMZ8Hu9w4pWCKnbbH8BLqimAzBGuZfq5zem5L1N\\\\\\\/oBQcgtohzMAxrpUX1Eh3BKwWcEAtOJY+88pDRZj5txbMSmmL6FI1Ki3gGJ4d21REc+kvS4O+JX292VWJ1WKKQEhZPXHCNCLhdXl\\\\\\\/XOgXBsasjo0wfpJsqhsK3fLUFMufxkA8NvEdhjM7lfi7Kqv9bwVQpkpM97BFWqLIhs7jnPOjiUs\\\\\\\/qT5C2KkTVUQFxldD6SHPDoOwhnBxm9jPj5SRDgu8Tgnme4KQDx6EJFKmR4yDjLSrQb+sXrAf0bwtM4mZY8HsWQuca+JhGzX7U0Xa9EnwoYk6+afL4uZuGcoso8GGV24+AzYf1WnR6tSBNzDmG0HtD\\\u0022,\\\u0022iv\\\u0022:\\\u0022GOInlk0ku6ozYNfnpnzhFQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002273f65345e394c4fcb85c6e0c0c528fb87a196d582b4ec0df2d7ab1cb59e79475\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00223CKpVbNIvOXvcO8GTUMJDMBho5f9OjWIRaccC0jTlkP7vGeaienU\\\\\\\/GTC8Z759vjUXBjkj9WzGhWOd4MU4b\\\\\\\/25Jw+UepNL40l7NSDzUUAdlV3aviwzlYBmaR2KbVFXgnj1EEA3+7WLCd8YCAJ95LYWmycFoJaiVWPx8mWBPlDzL6gC+tI5jyysvjtDe83Kl1VhAGJqJvdfz9JX00O2tbhYgyQ1+EmjVcgZGvvQID0LPb7y7ZZLsa7zidWbRjU3f\\\\\\\/BmsQ7vILX1\\\\\\\/wDfDUQlpr0XY1u6AWm86WyhaEnqg3cCCDG+o1WVVPGKQffkbN7jdb+nNrPSiOtve6KdGRWYJBLmgDNfGqdjLLj4gXXYHdBTdBZb02JPZ2+oiczEbQkNtIRkOYiCVfA41GO0WmcvIBLD8peDijzmT08CydJGX4ed8k\\\\\\\/HZyc7U\\\\\\\/4C6Y41wu2YqmJiV91iB7G5omdabA7TaGKBKlAsk44yhjbpJwxUKJihUCsBkOvo2OnRHrOcXbfhNKwKq71Oc\\\\\\\/grQY7+5rCv\\\\\\\/QuvLVg8hVl1vqTHDjmMj8GREiw2rosb35gMgWQyNwrlus9JzlUwGGNxkKfb1iDhzbLcrbULmHpqUdXRteyEvCC5Wiz5CW2cPGzbjtnLRF66UERcuQ4cw+\\\\\\\/AmMls7AArK7MKd+RvlDPWVThABzn2vvL9Yzru9QdxcG23g2XKLuMFB5A1uceoyidZjrB8VydhOXBsiVh8B2X3YeGNJa9yeMDm06KrigAXH2rhjCKhdLBHAU82QU\\\\\\\/hdPCNY4KP3Mvq9QKyiTlZbHfJBzISPBSu1EaOCFpahdpUyJ3qcbEHKfoWAS\\\\\\\/oCqneqPDGecK4iRHDaHRQq8jw6JpbqKh\\\\\\\/TQpn0L\\\\\\\/5EhUmkwuJ7PERzVhYxpdnaI8QDSTnA5lH3YPbWNJ7NPp3kpZu7Ru88\\\\\\\/+Y8Ajbxe1Sa1yAVeIZ04h7d6DQa0CddmxOR8s4Zqm55Mrx75hl5t5vnpIFVSk33BxtroVqK35Xt9R259hpPD8Urofi6Qw+A+ztvdTtbEZs07SsfaUECOpHxysWpHI7StMCOJVGE7+0vZKvhYKMxYDkwWy4AZeEMxPTNHsPcfCxIsd1pyG0lTbupMARK8I6p\\\\\\\/H6VDrxJoLisAIpn+qMnPYw2KFvT701\\\\\\\/eZfrX8yQHautQyXsBjyCa+z0WyHF0zS8MjcOCUQ0\\\\\\\/7I7ndrllbUK\\\\\\\/NNCnvYnjwU\\\\\\\/Gy2dR3Ac\\\\\\\/Q58f18HGN+G+tlSr6l07z85a3VoQjjhARe4xV4yrz18cVvVA6H\\\\\\\/KADRBUTomWI69vbzk0N+gfWe4PvqAlWLghkUeEFWysSO0VrzCs84\\\\\\\/UIWgKCvHSTcPH2xP+ZLfnvrdJui4h22kB0XRkv1yTKexoOcWQ5Y0l\\\\\\\/O6LT+To8T\\\\\\\/CD\\\\\\\/ikCzVAO5L9Q35fWC1ML1MJBe1KGyfMW28ax\\\\\\\/ojLLEliX+LI66kVgPlVZLGT7KWdCmz9rwVE+ZAmxoeW7E1X3Y5ulraCbnNnRASP2dkDoHX+khCe7EV50zdAcaeq9ns8oWeg6JSQ1wwaWcPfpQlSB4NFfRWn+4hd\\\\\\\/PKlsWN2GiTbYxJ27Zovbfq2yuCg+izpYq+DMwUVTpgw8B2NEyt3ikCHx3V9yeleOowkyiKg\\\\\\\/ye1pUXQQIlWYW6dvwyF71JwxWmJCVlKFCx6vTVJ32uju945ck+RRCtUMP\\\\\\\/Nmm+Nib0HGKKK2KToNHjknp95THPsDB2BuHqFYjNObFZCdlyXO\\\\\\\/qZsHr1Y5m0yjceQxSI9GPriLdW21fc+h50s2foZCfdHEXPTHLRmzRlKSwlxMw2wx41yohIySrsJWHsMJMCbSzNh4sHgQ\\\\\\\/\\\\\\\/d65OHaS\\\\\\\/1sQZDloE\\\\\\\/sxu3k3OLKBI1PSngUOB5MeOEQnAwKQDaE0k9XqsZxKYKuc5Ctje82mwEDMcNKmPHhUJzjqa3wqxS9kRD3sChiaU+ghHz4sylSIgnYweJCqyS0UacOO3dmvNjxVKlSw+T0c0Au72ldNc6ojkvIPrn76NN93O7vyWzJQJkwum6yhQBCWXSJ3naWT0bJNfCmYltiW0GUEedrqGNZE4ZOHpWrgSYGjlTH3mRcu4uqkHind7vPEb1n1YC0kHs6a\\\\\\\/YBBV37yFbryP6tefNiE8Xw+QwO+TDa38eX69A+509X3iasol425NPq1dKPMCBak\\\\\\\/wt+av7yjMF8pSWQBmFPP2qhs4iJcokZp6rPa0QTz8QcCc3aXkrBUeYHRhkaz3cm7q2Fcags\\\\\\\/WKEngm7aFmbrZwhVNei2WSVJtv8t\\\\\\\/Bla1wtSNdSiHRvD\\\\\\\/3XejR6yq6NmZWFupJFy1mBsKgjhZqBHzYM2Zq4wJ8CzWd87wvogVSCvFQ7l8JjQYiEMpV6zEi2jnEXdqzZS\\\\\\\/V2WPGYMcmEMjpND3z5FnqVERkqQeth4RmoZv6RexOGEsNCQo\\\\\\\/FCWo5mtdDengM5SuTfTDfeHZci3o\\\\\\\/iKlokQyzycpvBHs6UrjDTwDrxVUQJUzFIKAHTSt3QVEboG2FkD1uLRkoLmU7Xi1iqqXLL+HtTkkaGLjFtEnq++EywFPZLMAYvTOypT3jBqoBA7ugjVe58lB5J71tEWJjbVhOGziwwCclRT2kB0BsOHCau2v\\\\\\\/8Zk8r40zP3QIApKYMmKtS60MAf4rLxiQ\\\\\\\/BFtQQ4hbwssSVFN1GdJ9G+MVwo5QBGCcdU334b8U871H0ejuSpYmOOeyxI7s9KuLq5eApTocRtPkcceBLMXxJ53Kpl9LPyomQuMbuVcQEfP\\\\\\\/9AVEn0OBQtDNllvyJ+HBqktH4hpOeDZcDZ862o3YvlIYQSnkPG62Uf7t4qBigvrORc\\\\\\\/Wz7CP3xxmvJhSmBoQQ2fgcbSL5zpu1hFmgWN0+cdKMsbQQyBoihJXFSsEDU1a4czz07W6LaiW5YEv6xeTPGU5Eq7yojQ1r6MiOaJLkS5pe2wXfUT+7uOQ+i01ZcsrCai2v2wE5rcA2NsUITk2Ea0CQRVDmnGDhhacJR7TtRrSl21eVfTk4WHyxM8\\\\\\\/LYZLLKwse9eXWO06EyQMOBtmHg+xNColTVm7IhijTRwdBlJlxoHuEVaOiOf22YQ9VFBab4USWJEzCrZg8hjCNRY1iqJzwuzhwHngjQ7bxPa8nHP\\\\\\\/UweNnITYg4LY4JUc0o9eRz8aRZl63n\\\\\\\/nn3NVGJxEiHFf0n5VL1wHjyhd8DF\\\\\\\/ay5zrQ8XrY4hu1K9anaaJY3Hf8MA0xbqSQLRPx3vY\\\\\\\/vAlIiobHs+JpJv\\\\\\\/DpGVMjJI8qTejmxMfC5LtKuPqxHVZdENlGzdlbOD\\\\\\\/7gwT3HuI7RJHunWacJ29R68KQL3iLVIpNYDQkzZwSoCf9uHgXcT\\\\\\\/kyWNSXCAdzx2Pki3eXWTfv5o+pajMxJiG+Y9twYtLCf41RJeBJ\\\\\\\/VAkeIM6SWnLZOCldWEZWoj89BlhWVg+JulnvjP9SiZZ9Cn5faVy4gOWEVlJ9QFLnh9dSLlrKx9EURODyf7\\\\\\\/0be369QqhMf0d8pOW1Lzf03wqxHSk82yN\\\\\\\/nJUxD9Mv4rqv3A9YEODuBRirU9lm1tn31c2nYF1F03sCu0+1nHA6hbxqx\\\\\\\/QD+9h4u\\\\\\\/4WE2ZOWgZAI6Et8qrBsXuCGI3oI1bwD1bh+00Lc91wjHEsFNypYLQaBaYlkYjIVeVaJqo7Byq0sxt9xizg3dCqNyILleZvGg+xn7HbAwqgvEBWi0S4cIPp+cLS3P9So7FVDHN\\\\\\\/FX27BJzmjfYkJkc5MMXH\\\\\\\/mridcGdX5BdT64z4k07lyVFke1by3i0sgW+hq5cQjj0h+MMrnuplCBOHgRERVnG7VtI5QYI6Ziqn+gA\\\\\\\/Nt1uCwkT2WV9dbwb1OXS8Abm6lE+LsPQ2okN1TSJ+T2voo0wBNCaFJFY0u3oRkfO\\\\\\\/MvWU1sroNWRxYJ9aZ1Y2vs\\\\\\\/LEtZPmd+Nh6173t5UwzGwi1iVWiqCVqGI38mUmC2yTDj93Nfyl4okTGnMcJjZqIU3JNcySMZCoSsofDoXYQP7HrIDMg3SIoN1eV+RFVbK3MGUXcRyso2CUH+ErZHEo69Hn\\\\\\\/SsTxrYW7LcNlyEcjWP9pJgw3E2w3FPOYxDQ\\\\\\\/V\\\\\\\/nifULkFbZAL0bIJ2iYaBUL80DLyIuCpass3WaSjFd3fKNDMgXaZintedBrv6HSQ0XOYwMv2YjxzzEizSB8TeR165tAQxC9XTStkQNJg4DT7Sqbbh5M4NGDJwCQ6v0r3GQc2CfjPSmQ+02MnPUwZYXzwcHN4WJkdSU\\\\\\\/pQHZVEupD6uHvGB7Jvqv45gsj+9tO7SfWsDx5xvmTWEp1WjcGjwVFmYGWixCXpuxIBg6ZXMcVlwwE5yNkHnf+lRTP74gO0wIHRz64AFMwiD5zAzwSyqj9vLtN+quJq5mtJdQuk\\\\\\\/g2OnvePuvL716vqOuSSSV9bGP2CYzcuMqYIiqGkZM0ZVVKMIjqWanvCgbzBWl6LGqlxWSynA9Hz4\\\\\\\/3WOwGmueX3sGIoRcz6yvIyaxIe293tyPppKwgUr8ozaipQq7zhOGJJz2Op+lQZ6K+mx7GP1nnldhwH3MflIimzQPMuTFBTs3v19CBSkRSBBKkjVAGag5gT\\\\\\\/PQKOrCWm5qFh9POu0GYK7k8oReKYmVKKMhbOYzIyoGZKVp+cuRLLzg2LaKS0n0i75iul2G9pRrUCndTlZwntm\\\\\\\/DOhZVjOK3sn5x0h2JUw8diZQwZqSJ6TDKkf1N2jE3HjUHhHYv45bwxCfCkPs5Qy+GOvYagAJzqMG7wZSNK1mFLH051z4z3Uaeel6wI5uN1iOhb49fPeWenmIq67go1Je66XUTvt67Kj1JtNdupqni2uXVpCTx8GTnFDM2NELfVK4DMGm\\\\\\\/vUMEswlSl1Pgv9H5H29cJShe3mC9SR3ENFIMWE4PAZvHPpZpUFvtsURzIlsZlD2aW1tq6AN0XEj4QeJHQ5SdgKguRte3MlTTjpCdTiAEe9GLJBL6rkfSLHbWAbP3lEUVZccDZk+1BDhl2Nq66zBSa0sHn5Q0Xtbe4GDRVTRjLIzl6NwtlFdbMOK8RJfdhwPiTF7nC7ltSWmC0NsrSyhbRiSqKzB6PjKi1HgIwGGNFPXN2lItbUvLrpgvXK4uH15B4dWqIObe4FX9jBJoA9YkM3aBxinLfwy8ed1AYav6QbNqqEzYHJcJja9TQtMwnvujaFFSS1VWJVzQkdmFCdNPmYboLvBnArSRotrz\\\\\\\/D6uw8k+5LykYuCnPIBzvxahsVlHgR0rhmegUuEUc18HonMjCDxP1vMtDXMJ9pZtqn0smW9XqVjbMVPDU7\\\\\\\/CuizxKgdS\\\\\\\/YguBabbB13o\\\\\\\/FgFndQV6dIW6tVoNwWc1EMDr3ZMTq4ojYPx5yW7rC+7LhFttZ0LAL1C8rmqntS6xyVuMj3NYmeh4KTLL2j09DoNZDHE\\\\\\\/OX1HZftP25nMUkUGo\\\\\\\/J0zbbUrKWo6yZACvMzgfQXOniV3VIXe3FQ6xEngKDfjuU6SJFTokCR2fdGYrWcpV3aCS2pzR5yK9RSDTp6rsu44NmHHOEHQP+izq2JIUQBAD4A2Z7m7CvZLf9sczJme6fb3FYI3WHP5vpwf4uJKD3vQ78pqoFZRos9WzBwWTTqISfGUQvAISbCWChcQ4ax5jeFsrrD3hFhM7m60XwCT4soWNibt0bPIXruStjJYWXiJEC14Uceoo9lrVS98z3A2XAGtwyNhiss0BSWNNRRYKT5od68JIDjt\\\\\\\/FiRtTVtSpZ0Db40XVcjGmL5OE5oiViLWUaBrJg3nv75jPbNKyrtlrDexF9jBORAUoYo\\\\\\\/GYVA\\\\\\\/rGaexklRBNCfIlQRx4iSfY8KsukXVhlvHqTiXcaBVaQkA74fgkf3AxVJweHatctsWCWW0XYlFadR\\\\\\\/i8gMdL1XObt9WdEiZWcferAfjQol3eJM\\\\\\\/jJ3Rb6ijMkwrLkwA9tLNTOX1kaUiogZZrdSjCn+oSWTXhopLKsyLp3uyuiFkVPy0KMGuBFtR8Gk5gG8Y0KxexhY\\\\\\\/XwEKqdCZs9+XDte6P31iinTGY3LcNtPAyYkH\\\\\\\/o\\\\\\\/i7RrF+N13TpWbZjJIP8+uCTetdRYTRL9xF8w5lE8aKqzlKN4VMxtV4Kt2FpwWt4rnspxRLQ02UeAKPp0XeBz+BNMFcWn7G9pGvAZjXCQulkV\\\\\\\/ZDVFbE9\\\\\\\/3jsh0IUXZbjdnIGHBZ7VZs7Q5vyq4lAtkiy1BKllgqXpNdAumd4TSJhzBnG0LgjCsx5MwvWmFYr+LhlylQ563eFsRH4L0UYcIxzcqWEI7mx7C0iR5noSGiImjei1tYQWF5ekjLLLAGYmzxme5Sb7fdp9U4Y2D4IlRsVVWCYTqod33Sj1lIgpZQmBLc2ph8kkkd4twJzvkShNy4+Y90gSwIp2\\\\\\\/qEBDSx5\\\\\\\/dUgcgsrORS7m\\\\\\\/ESqivilWVzYf9f+PTXIkBNDzNq+fNRctdZBoNyeG61zLhyFt1dw4XXMcBcmUBoC9gvK3Jh64GBsj8cmij0px7Mp\\\\\\\/AEdKpLTW2BECRfITFHc5QUEO1or5MS3R74N0Fg\\\\\\\/2OD1Lc45PutN1fN\\\\\\\/u7Z\\\\\\\/uYCrdsJoikv2jKVuI3DiFCZubY0yL8LUP2IxHhGh3anphv6V7ctkm7YbNWyQDa3IWnsWGxwcMuaxdZHxF0esXIEPhs+ZnfcHcbQwpxfyWE0ZlT0uL\\\\\\\/fY6m6JyhhRO91eVhTsvvioYRabMRmF\\\\\\\/DsKvpj\\\\\\\/jJYbVuMc1YgCnCuarmaOOAuA2zyYz+fFfTXgOv3D8RLVdqdBhe0pKemwCAmQad1mDsJ2zMaXSfKlUAx2Wo0Yc9ne9xiZwwBAF1JFbcCKciTMlJxSO6wHu0yo2P5N79DgqPa\\\\\\\/YWQ+69ji5XRctGTwiDYKImdbrw8vQxPghilvsLHtiVA3h7RBfD4ktWNbgP\\\\\\\/FObNZEYeELmPVp4tOUbt5wI\\\\\\\/sssmjuKRMUvGXRiFHB+0fGRHF01CjX6J7MSkYp8ANOHS06CoQ8G9yWWDh9e5kSOOMIXC0SC48RkgdptU3aw0XjNkgKpBG986MjPYop2NKn\\\\\\\/MgoYFq\\\\\\\/8urwrA2ryVCo8iSn\\\\\\\/8CEuknyQGypzk1haLVCE+XDhVE3UTKdzS0GjGB7S9\\\\\\\/sYm\\\\\\\/AEYvyn5kUY9OOn26\\\\\\\/8SXvddBgZ4iJ+iHH8WW9p8QZOs249a76G1HbpAtfQ0QjetrfRMwGAi0+PR29UmHR76v\\\\\\\/zUSz6uLsC+wszShKP8l7ASetOuCKTa2tSo\\\\\\\/dudC7yqkLAUsI7h5YvbpTyeQc7IbHR7oENgQemfccNP4qXhrE2GO3xm1cqNy8EQgOcWuoLR\\\\\\\/R4iOBmigxfqKBljTxUnU9MBYX+jWni1k0W1mO7KDkn2KrmH8yBbfWRqcRpKZv0LlQAuaG0fMSqJVpiKkHZ1YSaswkbfj19db9rK+iPN\\\\\\\/IHluW4fk+\\\\\\\/hEey4flb0Z\\\\\\\/HK9dq9dv1cKRTFyOYaGcKCedF4kcnBI3EdbmEI+vJK7WTXGuUbR3U3m6+h9KB2s8n2BW0AVUe\\\\\\\/+IeRzG8C7f7Sw9uLZqdPnMGnevqtCWAqjyfiDEyCvBdXr9DNR3QFXbqU\\\\\\\/+tQpnVlDkCz8aqfH+9a+nvisaHplUGN5OCQLhAS0mDP+yReLi2Hattqy0MQVIjOxrBbdw8\\\\\\\/+O2LNT132HqoodhpAlX2+TunljEf4ysnlvaWPMVVxpETKXrGwyx+wFfCh1gkMgMGoa3PJZcgfmOpGm7rX54bBuhhyoC0G8ef6AJ7WwacX8\\\\\\\/PoptSTDqLW8aP+tsoeDyG8PL8tVtqgkOnYRlNyL9\\\\\\\/uuSk8fnrfswRPpsozpd94BCKe+vvTLBTs+5iz3DYPHIL3R4tTd6nMF56myRI+u4TfzCtsit\\\\\\\/QrMqn7FM63HGeNV59nJqM+O0sOtDMAFMrOzKGUh6Qpxs7UQYR1UeDhMi7cATELgf4d5sG+bjli7edKhP9\\\\\\\/il0aW+nVfZSvsgmD1rkgJljv3z+7dyOeikBFVUNmqppV+IDQgBEYEN+T7NVww8w5ew\\\\\\\/0C4vBEWV8\\\\\\\/51TI2YRv03GO1l7lLARQjS5cLBWiYO40zFiIco0ZgWYVEOYsIMoD5bQMwcsVFJW2GzagR\\\\\\\/B6wBt4DRGuoiBJZhv7UO0Bi\\\\\\\/FZOmIhk7GmlOMzw4ZrJpeNjpZys37bRMP1nrC91zS2sHql3MpHLKwmlKXBPm502NWlvrB6ypTGSYyTSD8nk4vNkRlJIqiBn5tF5VJxUhKLntnRD7+4Mo6wDG+NDzFUGGD5Sqf3e+1S2\\\\\\\/wIASRjQzbiX5g8UUcTgzEvYf6IRuqoCXUx9U+rgDot6Dg+0GAeeQJUDyGJRMOZNyMuuG7Zy54AB9Aq58fNyAGLY5vwxprb+IwAL6LOiWI6tuMibIJv+u9woCaEhYpKLnWce99WPJhgb8t+6kd3WIsVeHT60Htp+IyvyCd8OqwoTMxUwK8aJTM0ncHnwJdFA4H+KtVW5zfBXIyN0JddXH6Tf7y4PyZT0CwPLVCrx58Cs2UeElmPYChXJdhc4fTfsarSE0UPsI4RS8\\\\\\\/Yxlxfn7Yx6sfYpzagCognmX6eCad1F36kDcVOQMAWNnIWZlCSqj8233mAMGGlVfb8\\\\\\\/\\\\\\\/4HuX+Am0pjsEmgtlkfQWzxB6e6ccdybwe\\\\\\\/yCZwV78swEv\\\\\\\/57UiNXodETxm6brE41PwmXXt1WWsrl\\\\\\\/F8jdkoawKJ4mho+CrAfcjizC0EF+QZF\\\\\\\/3EbT766nzDQhY90s8Ki0D4ePDSQUUuT33y5pJ7nWtiGMdCU0qcKqszPksmoLTaIf5wZ4z6oKHxzfdF1jXyzRPZWT0W6Vm5RnusutF0QXys0bh44TK8+CqTzVX+AuLiP8Efg23uxQ9AIw\\\\\\\/bZd9Z6o\\\\\\\/SOeZMhqTG36F0ha9DYg5Do1yirhRJ1hgxrJC4aRHK1YmNCWSDIOoLjTWk7Chp2kKR\\\\\\\/ExAHJNsPnoWdVlabrp8mzuAL\\\\\\\/wUChlpwwAhniQ0PodOQx9qJjpkHgPGNSE44QcTCjexneKXZdBgXitn7HqlMRZMp2EF1nAbKh2cmjvKR2ocFSq8wnf2UyR4AYoKSguoecJSsc50xXpSZYCaRt1DTA+l44Elyl5y1ig+bEaWbv7BOnhkXCpXse1PLDBLBMh+sJNygW3iWzgXN0XIPS7rfg609Xe9VuYxQJrniAjkpCYIu9kPWtOEaCovZiSQbm01Ot3ptRpAZoj9eTnb1B095whDFWB9H88pHW7fG7qvAAyV2R9zdq6YleeT3oFNFbqcBw70Qpfx6rgslrADkjpyyQoirnCcY\\\\\\\/qREuQRqIMoOt7PbDN8QUPEGKE5l48\\\\\\\/Ua9c1KGQYaui+A\\\\\\\/\\\\\\\/YaAQax6JRuNinjqMn95U8fH0voO+gHswlu6msbK+gx1Lk8+KM17GjBJhFNcaqgqFKCrGxBCXJ3C1\\\\\\\/L24FksXtDbFsIV7SLFxLF\\\\\\\/rMavALpw2meySqniMWIxnf4Xfw\\\\\\\/+Uz68bcRVur6nGGrc\\\\\\\/HeCPRVQsM6OQYdSyargnb4A3ZvlH4JxP5sLXgyD1TGEOAMphdvW02gb23XhhDLS6NnFr7RKVtJJq0m9zzP2AjxwrLhowMKs4f2VxHzNjyOeS8rfSz7x4f5QaKDyciRUBkCx710XXjejCwMuendvy7cdq3jJupv+xbeo5Rfjz2+FGS4+zbnKqTUhhD42R\\\\\\\/\\\\\\\/XXBRZNAiMYJeHZIsu33rHFjZ4byxBVyr4RCfunG8KX6q0Vl5BG5buSnXT8sfYk4O96PxK\\\\\\\/1tkYaalcs\\\\\\\/U5sJo6GJI7fOquEY0GRghuleD74ChvmLSBTsJOQMhliib3CkHIOMZB8GQ6KQ0yVPqGJ4kI6ArxS5Eao3x0zKEE5gZY1jCRAC6yRS\\\\\\\/f6XirWcH\\\\\\\/k+pa0ETQPkHlXZWBHNqtq88wRBI8rYYVMX7uUF0KB4NzdsB83MGfvoq+37BLZl2xcdjMA14Imo9T5QQOgljuHKAXBvbgg9EDLi3n+kkTlx7F3hJtLjMZLrh1Rw0H7myS8GlSuFSJqn7QA6cDLyuoT9WOsYes6ZZAqxTdU0WhLXRr9dAx5tXsxiP2a5R65p9dzgt8\\\\\\\/DV1oQ8CNU7fXOtHDve4WJeSYNgJcXJnNPdWGThz3iVgnE0aRWBeMg5PgVqDQ8NNRYEE\\\\\\\/tXhRbk\\\\\\\/TXNfU5NqSQaH0NVEzqD10MUgg\\\\\\\/QI4mB9zNbau0KjA2HOIK9PHJAuneiFsbCZ6VZEAavUHVctTLwrmj7vkeNCwUbLuC4bdvN6A1NWF06jXDOmK289\\\\\\\/BObOIam7AXrLXFK0VcRMEQqWQV8+wyCCwDv99EEEK8hP\\\\\\\/lF1qiO5koeEgqtyabERNIzYsA+n2yIDt4lf5VlHiw2nGaepFZTeR8XKjvyU4V4VzQD0YHcf7MnHvFBuioNmYfVfpbh\\\\\\\/qG+LeL38DaXxIHnCbZq+\\\\\\\/FwGw\\\\\\\/X9TbLD3Q9Xg+flg0Jr3+pcuy5xB84R1q8OJYr6HnlCCmCi1KaThMX9NTarOt\\\\\\\/h46Iq+YN8Sx5WTY4NZ2LgWED4aAoDhQXx0BxTI5uRK59I\\\\\\\/ZvuwDPSpFNwdUgxAXdq2ClYFnlLCTns\\\\\\\/87cNLpzitKeVbtHPfNdIOZs2wC+7A2a1nW5GkZu\\\\\\\/cVCfGIIwxTrcU75jNf2KDP3KUqjZ9MlzAK0Te5o5Bi4Q4U53GaA0JgcRXzNnXmPnXfW97yaPihQ2Phr2iNBuPXWbc6oqqQPFI6lXAIFzGVfyh6CyR+9LLgzuF\\\\\\\/aS6YDlRUIYOJotUezYqFfOWElQBCl40Zr8EunRZbNgbLb7\\\\\\\/bLuT235A8spzEG98Lsy4MOHO1D\\\\\\\/w+g+AkarH8Y4QrmeGs0+qqlWsrNiwkhoBkOLz\\\\\\\/xIMBchmZYWo2VOevu10s5h9lyj9DRXM8RNgKPdndblGg4PMvPsgbH8qWZvlnx4DX\\\\\\\/wDyebr3pR9oYHF9gaPbZVK+3UUMOtgOiFBK8sosVzDd0i3GZzsiLltyfHXr\\\\\\\/rg7jft3WTL\\\\\\\/7VFNAf9ZmNuTsDD5xWDw8qJzk6TRkXejoJe1nKakiBlXYe5BAAJjfAjsIoUBJVTil+aUHs87OtuYbo2Wc\\\\\\\/E0BCJOHOnNqd1QJPBODy7osSlr0YOP6d0f+jFNkM6TYJGN9GrIw8xS4FFBFzTJgvlc7o4Jq1d63NTINVo1f+bTswEwAJ6HbimV9tNd4yRJIqSBYZXmn08ylty0aZG0P6f4BHX05x3hXlV93877PSUhUcViGIjSwUKQRYCKI3QoFYgp2jcDzp7+yDNWIfQysAfdrx9rPogptsUzGLXI55642Csob7\\\\\\\/FGvcu3FRQuYnjitGK4FpTzJ+6s0fMQlIzn6d6qjrKc1ljbOYGYR1FQGnrhBo7VWuwkCNDnOb249I8WSdIunHp1ZfVmEAeO2Z\\\\\\\/+j6YMUQL20mdoeNSAytBauY1mCGsAktPaOP7JNE6mtCTdat+2+zJPnXvSA0Nq1cF2pyxIUGt5k2FHVAkPUKOGen+B2tIUVSLaIHrCWjWR\\\\\\\/zGNyjgdvMhqh3evQ05BQuH0OHXSYLdaAnZdiVkiDfj\\\\\\\/VWQCwJJIOWJRpzUR25xQ1ZXAr4sASe50\\\\\\\/rYQ3gM+Dx02ZAHJDmHaQ0drcCuAOk25uYSAAzEmrUlclYV8d91AjuZyoarYDbf1uVsxnyU1dEubFsX8yI2xWNWeu3aHwv0FDTi9+7eFYBaZbajP+keqrwtAbhl0gY63fXJ3txTwGxu9ZMJ4LYyocBXGCAgasDJj8ybBQ2R9NrWLp32MZ3wdhagWkn84Nk8\\\\\\\/wq2ZUrQnZv1GSm4eDc3mMCuUwt1Tz5U8Xy3UhVhY4DvPXu6xoDxjBslD11H1cE8JGZWFDjQSy\\\\\\\/SNrHrGSlR\\\\\\\/FLWeAWAHbBWRCr1X7JNUesTNCgxO0EhTkT2qfnwE\\\\\\\/BAeNH1kC++ei0yF2cKqs\\\\\\\/CUSiBf01z+aB7ZnCPraIlAQ44DTwdOF5zq+KzJ6PnPVAA5l9kSVdl1hlo57m27r6iLoGdJKIBo8ZPWVuZ7wu2zkq6+U7GZ8QOql\\\\\\\/7f+o2QGaauaUdw2L5qtwV1uvqJJDjbI4uH3V24htHH3j89RD\\\\\\\/n7GVQIVzG97NCHwkNwa8vlh8CjG22Il2HzlzzXMP+6Q5fjzwzt\\\\\\\/TM\\\\\\\/qyj1m3GrNNNk1c7nEw5iYk+rITiZDXF64UkYep+U5YaO4U\\\\\\\/1E29VaQEdXC+LfXiqTbS1eIYekXgajKxGnkzi9N8owpTS+X34m7VOu9Y2gE8C6j9wMjTQBS5JHU9A+zW4h327vU9Qtgu0S\\\\\\\/mNc5lGnkFy+n3Az\\\\\\\/FNJWjg\\\\\\\/CmUMvhQYuUy92HCr71ffe9KGphCIJURA1EswGtJkLUyYQEU8453qYgG2Xv8LGKipWFS9VNKqGtgqMCghKIsV5B3Ob6ydTG\\\\\\\/EUgWDS6qMwp9onBY\\\\\\\/tpLk0PmMEn55+PHwb3liY36EWtNBR2M2QJf7vhf1rSWD1CxBF0DuUO1MtE+ZjmKnQU3nDopJDfW12TzmxTu4kfXUSdXXCnrwN1DFoE2Oxew6m1BbjlZCbEbA29NeH9rqi+0smWmFhjxMFnGVzM\\\\\\\/baiaKJedzbSxZaTR4QkXBkbNsw7nDnrEV4stcKC38s0Hflt1ZEGAsEOp\\\\\\\/jxXIqFW+XDQwSwMFDO0cvpdKP51P20rclo8SkPgLhJlM+ovBMV1SgFszbIpn3VOiLWj7rmrd7yN7ipn\\\\\\\/L2bFk\\\\\\\/K5lGioI5t0\\\\\\\/4qwvoMlY\\\\\\\/9Y99SCeHlkSDxMHSOexQgaKnU7NVTjOGr+1OTvDELiZ4EhxU7GFpeRSCTx\\\\\\\/Gl01ITBm9DpP0VIZ3QPLnOtAIH4suvtG6iCakRmelUF45a0Lx8XrV9e4Ck9Er7SD6qytsdzRkEmnCjBES+3cLBsmeVSRAC65mYWtoKzhRuFJDPtUFZLguzSiXoNJ5HPWemre4oAkjRJjMv7fNRV9qVMfau39P3sK3EhOMmvExrGSmlWX1Btv8MxfJ9JVFAEGLXKiR4lzjltBoQE+\\\\\\\/MZ6AC0SkNtp1Uigb7mz1X8NkYpROoDKsyEOrovewnzCTHRkicvRK8bxJ0uchaMh5gAycMNIfR34zDUUvujW1w5p\\\\\\\/tgaHTlD1epCWEeqgNlDJvxMkL8DjqLXxLlv9vyhtqmjGYf\\\\\\\/aN5hm93t1LsBun0qASfaZCUBHkwKTTp5DZQBkUKIWEP3L4dYNq3Cuo3i6z7KzDI\\\\\\\/\\\\\\\/YggMo4AW+WLlFoNNbiQqcawQ\\\\\\\/zXDIIEB0TLLDHyDKz3vRtnNSdaWchtYAEMAVjx7Mt2m8MeB8JperBkll2cTjLXomA0gChMFjJ07lCeiEPhx5xDWEZ1RJ1+sC8I\\\\\\\/MMwparrK8457mBdDYSIDdJE66sUNsuJCAtwzS9CVb7gvWDcUW0XIREDLr9MU7So9p\\\\\\\/qwqAcVXwt6iJ9CRYagrmsdRy8ar6YGeWhy5V+m2S4w5YbwiFKJKMruB4Hb7N1rYgf2NOHl6kpaSG7EiOyPJHQ2+QrPncUjOEKXABVeLpAwRXafnTWUdXK\\\\\\\/PrND92CQ4NSktfc1GQGqhQSyc9QU2m7MNmgbuWr71N1kYHc\\\\\\\/H6ljbJphO3cmDoj2879WPb8KMsOxaKj3Xwc3qskV4qglo6S3gOjNsIbN\\\\\\\/0ihVTWNlXW9OD9Ff2GcQIlAITiCyago0wt039h8fZh1E17TQUGAINLL\\\\\\\/C\\\\\\\/kjVs98CoMPfV+Kh3X4U5MJkTe29zzft5bjX7Ry\\\\\\\/lJGRaisHPkpKgB1v8oeg28o9Tiku+\\\\\\\/fppPUvw\\\\\\\/rzeKhNKQBiwQm4HxYRtljmcodXiHeTEkKNQxsZQ30fDMJmZAqoZiOHLozF5252ndKC2k3cLfO+3MQ+YJsUvCXoLqSl2lsfi3jDGKIfWuJWPRW7tch9sBTR\\\\\\\/qCKRVWu1qHIIAPg7SSg9MMOlBAGCDZ8d\\\\\\\/DlBMr40DBDwvOEIsdRgji7pGCMpeVh+hmH2acJHZXoEKdyKXowrR3A3XmerJHnj0Ie8s7VBogrLdn3USjRckZbr6h6vY0902yAmREDnkA0tugpuJuzwXny0kENbQjC8H+TQx+FiushUPVcB3YMi2LoVnB2CFwLyd9FzH6nDvfa\\\\\\\/i1+QSv1kVw9nRybQ+SCP2l1Vffq+1yoYwIA5S22yXk6djgGces\\\\\\\/ytTDy7t1G3Xfhs6azTrbDLWTaD\\\\\\\/gGfdK9184c2Bg2WsFIShdgKwwP5usYfT98XNt2YnZTtWRTHYBbBdlXO+NbnTNr1NKmAPKulhCawbH+jwOhNx4cH71r7yr8CMfCoMtMhsdRdHazDSli2yU4Ro8eaWA\\\\\\\/yjmGeyxMRD\\\\\\\/vFIQ\\\\\\\/TCE5r8aQC1xDDDrINHZfAhPTCscVOAuj72k8PQzjQODsishLlEhMqiuVtMtXEzBtHWsvTSCTdWANpQjC+XXTIjjP3k1U6CEfEw1KgfGp5G3t50Vf6CRD7Zxa0uRRCS1AuLUUb5+EvNEl9q0bNl9hpcwPU\\\\\\\/IxGz207NiVkGGHOhh\\\\\\\/kkhaWU0ZuSuAPEeWiw\\\\\\\/RR9GThE+hC5f1C71JOddTD+30iIuE91tiivxJ\\\\\\\/8qSmDCXqIRr7Mix6TO7bPPi4MHQ+Dw\\\\\\\/qB7ER3f3T1Hkb0Hi28sIoIHIFYAZCJOAqmpbokqKLuOXdyp4gegSU2ZQs2R5PEVHRYTl\\\\\\\/UmfTGibKJUyAt+JC3\\\\\\\/DC1o0RBvS06RHdprVJqHi39lWlUTqrgRAvSrFGh9n4FF94Kat3lmyjoh8GcnqFE51V9brnNT+CIHRGCGfH\\\\\\\/7GHBvI43UwapDOWA+PYCbfIA2c1f3O1RRvPXwqhgpUGTUoOhm7XinkDOscj5oxbJuJZHXmFMtq8Soqd5i5OVDR2WWxtwYMwYE4YgnPorMvJCYt\\\\\\\/AKfL9zYjz\\\\\\\/Nfi21bDLjvfyXk4qpFRY4prNvlQTgaV3WISxqxtzjvfT0TIo1WtdufvKO3hVdfvcHBZQE++qYHm99ikZNbgGGTLcADgVwBGdI7\\\\\\\/kPHK7DkTswuTeZsN6ld4ObGnMfYQN4VlvgelfUOuzE2Qfu2F0qPfnUrVKBwUYVKiLTdWLYSH2idcwaJ\\\\\\\/CNZSvadQWqv3bAvYjTMTZOXwAUBXw5miG1kFeUdM3GQyqqRfoH0mq1X+gGHARPddmM4hX3KXVAUl62fbRoVzun8WICOM\\\\\\\/kOiWSk5ClNRooU58jgCzGDsUKnOQXww+S0ForZ7P5ZxptpMEQuAVRq0VZmnTm8j3FDETTzKdMiuv3DSVxrwwzWU311Q6WPR7+4GL\\\\\\\/3iOr4FWR2LOUyfBM3ZdQcNt9btS43hukflYsUQhsHrX\\\\\\\/X1dM81VkMOaGOgI\\\\\\\/+qZnDkWJeYegTKjsaArzmCWpj0VqYwx2yUEiMi0xzcH0GiJrXlMaOVHCKEPU\\\\\\\/tP7pDbO6TRKWn+hF3hfL6nAU4iL6\\\\\\\/7ILCpMglZFidohrxkGTRqopa1Xm63F57lJ8Rl7gxrAVKb0vaeg0bytAwrMkXZnsK\\\\\\\/Gv+qIt22cGD0OfDqBo\\\\\\\/6B36mcv+GUv7gV8z\\\\\\\/Q5RljWiMWM0LDBOsEJmF\\\\\\\/KrocEe2JzlLAS+uR9VUfL6H9FfewqpsUpyWjqIq8ryIJOCn3gUr1q7wNUcxjjX5i8ztBRRgtpD\\\\\\\/F+QthwB72v1I\\\\\\\/21MdR9ep1balmVx+HVcH+\\\\\\\/AzyyPliQZhafmfgi7ozbIwTJ25Cm1Q+qnCJGW246Lvfsyw0TvaMrS8buZCtM3eeWjY6vSPmrYl24wNUzojU1TIeY5xog8YjTT9TKgPTOM5fCH2LS5aQzy9zrU74RK3S1qQ5a6bg6eXhGzDnfxEORgYNFTUa0ORls65oFEtB7jMhycIRkPYzrEpMAxYKv0+QdFiQ+Lf2IewfZH6XHjc2\\\\\\\/PQccdwP+iCsUpZZPJ66xCRT4WgSDwM1c3CtGG\\\\\\\/IP56eYse7XAFPw9HyBiE5DCZ61EvPTgcz3QsNqHkxIy+oeNPSTHUvuA99W1jhvJ007gue5Bp\\\\\\\/mRlYl3usxlm+1dlmCV+nOE2Fj3XE6+Pc9vHTXBlLhm+fxS6SzmZ3mMapnp\\\\\\\/ZuM2DrYdxMQX4svU6gyvixYGDdY\\\\\\\/KQrIlE1gUb\\\\\\\/0p2yihoREYvR74EV7o81jCm\\\\\\\/BehCQXMi6YoFDqPRUXu0WfLjs2pLsUFVuCUcg5Lrr+4zGrXXM3FRoVKFBz3Xzye6Wn+srFZvUrILcpmNLRGaDh5pJ\\\\\\\/sGNCF5lQeywNU2tOs0Yl+GZYYDnsyi8CmrVcQ93+9JCcBRsvXrDjmS\\\\\\\/cDxxOFQVJTZR\\\\\\\/P2S+b3bVzWS7UVqrXcTrk2dfYHMOqIrHTpd98yvTmgLMWjht0B\\\\\\\/nHKCTAcs95p\\\\\\\/YJ7FgmMmKpTl2vZqpmvw4fASjI8pFyazDCCTOzyH0kBgj4m9GPCpXsAomrhKcB3wEvSmMQU+VioelvOlW+TAwcNxkWbC7YYM95dBW6TcNU7pA7FkPbBeOxySLLgakaGKnuBFyQfk2z25fOBN8saAHVbYI1Ay0Hsq5GFp6hLX0481i4EH54lr1C6OXVWSJlUUPbZCI9853IL7x9+ksgi8fDYuhx8OulqyRxEFJiuW+FNIHMYovmSpV+60QmKiQMk169QnEQiA5RE2P\\\\\\\/3owrWUuypcoCnLV7uzyElShrFExOy4rLPLbmvD0TgoST7bXNZqLnzJK2CPaDw26wrqSrJHuAtuyKJehkkMbgB40FONZFmW\\\\\\\/\\\\\\\/gPVztWmR9BW4G1\\\\\\\/v2Ge9NxxJ2UTr+++hwev5cVTt5HDCJ\\\\\\\/fWAJ1dsbx8Nwg+nFyRk9nhbhycBCHbzPni\\\\\\\/+tP2HNt1qyuhMY2gTYwCmmghz0kQ2dQtMCBZiFziu4bTaDF3BeCa20Tpwv0iwSexOm7q1fRgquJAwwj396SI9SUGDUCt3q\\\\\\\/vCz+qFjv6n5jTzn8nZGGmdEsx3EUefRT\\\\\\\/eYc7ggZuZJZSG9WCOoZ9gEflJ6AEuaeGngEA6\\\\\\\/xKeNynY2QmIKkVJEkUWR3XN5DqDOr3t8+KJNUeTVNu5Bgw6jiGXM5szByzFBxL62NyEhEY\\\\\\\/UDyCUdgi5XsD3WdyOrL5SXczC3dPUaw7El8wAR5bI2BxIHbUyKkzYqYDHpujN6Jr1dYSjFIVRC69Qzl\\\\\\\/SoDbq+kNEmMMlfKlsM3Z5Y59cEbqOC0vKxCcMEb3RPnLX+QhE2KQvi8lLCjDJnVeB\\\\\\\/yM3Wox0Te5UzrbuNekMT00NbFb28DttpQI90o7D7P3yqTmmM6cReagvre684GggcREllKv02GOI+lpeBtran4pylvh8lSXvB4c1Jm2nJpwIpRXp+Y+XTGI8ggmArzfdYRYwWZKBl3QxtdrSl5Hq\\\\\\\/b4lXPo61iQ20\\\\\\\/\\\\\\\/VFAGzNW8EWnN2KACYiM9Qq7RGujdHzIKCncHiTO3oTZ0pV0+t6gxJmf9f1U39xQP68Dvl69+Flugt16DmPUDwML4PFFTmkqy5BG6vDQ4AtLWLyGKj2XO\\\\\\\/kOWs72MHvsQuuINFMlqmKL2n6fmkgS1EWoC01a4p7X+hohx7VqyYlU01PmTGWd8ZwZ4qqQ\\\\\\\/LT8U068N2qfa45Mt7nci0+AgD7TYOKp1RMu1NA3jtYsqb75dBhe6qbByfo9V5\\\\\\\/dDOkrbg2eQHOb\\\\\\\/1ZdT3UqY6Xru+F5JbHp7sGA3uIGSBwKCXNE8rP1BnPIzRfgb35GsUjUn1ASh\\\\\\\/1+gHEkD\\\\\\\/P57DSLw61kNNIU7fO+KqSmIo4PljIucRxPtJBvdX+whu4JynzORaLCWB6ZHWw\\\\\\\/qmSVBBasJKHg2mQGK+ZOd7VOsbCDIjSFlBLMZobtlu6Lbv\\\\\\\/QDY3NgzeYQ3Pnz4OFlTjKBWNTSFYfAvfq3iRCwTnoM8oaT0QB6kayUhk3cmMyrEguY1hppgUZGj2UY7DbzmBD3pv85faFojeY8dEDYFNOnU7zstygm+\\\\\\\/LbyaRQVZAQTKNgwtk4jh8SMMTB6a3xXXyBIfE2ZpaFr9s3lRA436NKFQPNvMppbXJvcxwdOQlYMcG4WhdiFq7jMRhkRL659WsRQWSZHfBd97dQS0xjuEx1uA3PCjTCXYn7ABEaxbSPMSAYC245MeaCoxjI1irZnoRToQE6DxUs09mjiFXWABJ5GlX3ePfwLG4lmIXvL2m6u9OiNSAsQ7tW3PXe544lFRpEZ7EI6DesKUUUPcNQfw5\\\\\\\/7acFG4B+w+HjJKbLKyagZJO0g7aB4tafpv8BoQCy5bic4UqPpL5EmqCF1aijZkSEO4SSTYEeQu6qeNks35WcJSeH4j\\\\\\\/jEj8\\\\\\\/SIfsn1ug5QMJbkSPRYeVWf8cudNx3Mk5i+VFQw8ZNGPiGJZo01KrRLEjFV6q2at4SyXT+Beo\\\\\\\/hpliOHdxUm12bcNFL44havs1DMTNT7Mzc5hrV5BPuIdjPbhC0N7hekJXBxSOKuSTyUXuEM0CVI3V9Rw1ep+II3Au9BYHBQusMonquM9KG9Lklyx1nsw09KrZmBLM88Wj5Zsiujhm6h0l1EKcLj8fw4SUnKLJZJIBD9+DEiSghDd27fc0r2z\\\\\\\/O5izPbiRAfRjmrejOIcaYJye2X3hC92PcFxrVnHmE8NrKBSTzI6MCHFQ0D5EYhu4RCHn1W0AUNj2AN++UrDkx5vTrn7mKy9bZqoVmFSODI3cFhJ00RcB4FwnwJ139+nrkxav++hfFa+ZI79iJS0AVbWNzpYrYZ9beZDQNKsA23Ls2Pyfhbrj1pqR65BHhufQf9eAHwY7TRHcBlOq7CiZWLfYKmC4ApqXHNMxAgj05NgRtS07ObeKV9fbJCDtyJDtCPeS\\\\\\\/G210IEHtJxQZN6zwzX+bNuAzeFviSBmt22+LrKSnzgb06sHb9I+RBVJC8QeBUryfDAyKqUiyoAuxwVcOkImHTiQi1JCCtrh4sqEFnuEo079xI6+LG2lLi3ieIGJ3e7MSiSZNenEy83+I7JrCbvf7d3umB9mCf\\\\\\\/QspdOmbBhm0m+LC87bryD5xZXfuCsmGVOePe6p\\\\\\\/2z1CvD+MnDp8gFLWLCvdAjszuqdH3OzAFwfrwymVOAI7ey57pcpaN9AVjCEq0S5UwrAEjkEgP239YGf5qUYtSQ2R2Rak7gWA5Gt2cupkAg7t9+cJpAbLGM1H3KZOVTLm5mJnxSQNfVp66siThuIUQU17HGYxSU7E2Fn4yopKMMffDeJMm0M3tCNGrdNoA34Yr7nU5C5yuTa5jI7F0TrPGxEUtC17IwSm3sW+mWV6HCOUNwpOzL+q9meaQ29Xp\\\\\\\/pb3VQrwLNrXnMfP1\\\\\\\/nblf4LLITWB9bTpgDbS7IZC1qXvGPzx4qDMii7xNcZk01z4Kh975mdp2R8rMNFqKXK+Rg+vtFjKULDqOpSo6UQlZJM2r7V5nohPrxYGJjieMHsigJr4UaHWBVjIkr723us\\\\\\\/ErscJXjkM\\\\\\\/h6HkTCve3kEW6jre8i9YgZ5eIpwBACtXzRHGEQOWtEAoZhHa62EB0ZkmN4kOG1R8hTZsJqe4XXNCC\\\\\\\/g9SMDHDkw1mfJdF2bJgfTZmi8ed4W7BZm0SCMQCjIA2CPYo1HMWXjlE9DiCpu37DpTBLYkHCV5yOoq7uhCkroJuI6egwucgmF2dXo0VSl7BfXG9e9d6+OD1WdSiVrj6DHT4kZiBdodFaRqVm6TvShGb3+T1+kTtjpveXnV3CvmHpSIS47tkv042jnTkuF5UZMpohBPoDan5zTj6jPagQCv0S62iYMzgN6TdSG3sgH8ljSCY4CgBZIsKxRBymbssPY4VeJHVf4AukCGG6t8tSxOytsROJyMbC+4ex9RHSAEfjoDNypnN39LxrKw+qQHYy8nyfFBoMz8HAV1ZA4ANc\\\\\\\/3\\\\\\\/AeTuNMweI7N9gii4TDeZmu9uoWVhaerSbo5YOPtmfuG87Lbpso8lDhaGZOdtf2V70Cjl0Nvibu0WJb4tHsqMW9vSA\\\\\\\/arnju4Edc1K2YXRUij0PBBXg2or9N+otB1+HpJmXRFjWvf5TUN326UPwnQZMME7ZcL+Nc1rl+KXIQLgpwKrIXF\\\\\\\/NAepfEbBOltEuMaEOXnzlLLKizJmDda\\\\\\\/70PIqfwlfrA4uIPkT+gFXmMlA0wQOgEL3QjgwibLh7nnjWvkqlR6R22UCYOqoZWHyBA5R\\\\\\\/JuF\\\\\\\/gI5HHxcErLivVvov6HFlqJiVdjJKV+9xXOcdhhXPXbErd+n5rSQL47timVutYNWy3qmvTp5B6i+xn5xvtSGb7ooyCeH\\\\\\\/tdtA5iZeQXFzkA7SUqHozSdYOhjsQFP2aGC0ruJwE6nPhXeeB+IUGJkBxR9nqtEe7A8rl+WvO+phf+ECx\\\\\\\/KMn8cjC4uj+r8luUjpLQeHCmRcxfcbeNCYxLMgsCvrMEXM0KiAyIYd+bw+nTM2rtM\\\\\\\/bkITyxj5ujHb\\\\\\\/qkegcYaxTLyzw06OJh7yOvnzrcx0JINy0V500d81bjnEgL4vafGhdmU30lZNSgmXXDj\\\\\\\/TfbgoWudgGPsNdxLq2mz3voN2Ef3lr6264jinKCHxUYyEcRjMb2ffnDnw6H8laWeHuKG7UpllPdK8VIoPQ6ItrSboeMDUTINFpa7Qc3grV6FAFbWQ9LHkAiq2D51ezzeyMulS4tNGVoZG8\\\\\\\/1JWSTL6Ku3LJnIsZsjWFyVpYKuOVHp8hzC+FwV8edNOrHsxABBMJAhOezxh1\\\\\\\/1CXiZYlabjjMYZ94JbBAAZlw0Qk8cYYkdDft0o2uFz+z2OcMXEOwEUB1zMEFNDg5wrB+9rzWpQ48gZPiMPAMgjHV9p8UHt0EkK6UzBGRiljBDF+aBxkknam\\\\\\\/FgfzWL3lbeDF8Sucs3\\\\\\\/GaAqle5xFxNWSmfcvvqNSClP4\\\\\\\/smDi+InUzoX9WKv0V4R6mL3yzOKUblxl1wFveADcOSYOIs5NSvSWCqjhGaiDQgDq+muhc30PETovqaZe1SMzxV6jeeD+\\\\\\\/bhNqzBBNKTyriP\\\\\\\/YVMuuHGLeSPBWDvdOaw\\\\\\\/5GyXM\\\\\\\/gEymaXJsMgRokk2qXBG\\\\\\\/AxMHywkRpj1x0+xBT5vuJ004Bm8dPykI9aNHtx4d1lFiNkeo8yTZre9g\\\\\\\/7xPDrKlAtNKivc7FmnSi8GPbHellRV3KpPVSC3TrTax8kxKqqbwofb0zEw8CsaaRr2IGCD5DHRObKj0qtPa86S3F0kJ5\\\\\\\/NoorD0pwo2Qrdxe8L8ZgQTv+X8xvwyJMVIiytKTl1ViPN38gFVqMVrNuOBHmKM0f7Ne27dHIXNPJTg0zItw4f7993crcuEQIwQpelK49dvxDOc430eXIFlgZUUGSzRpcFP+W\\\\\\\/fsBTxmABlYV9qMnmJPw+7Y\\\\\\\/8Ga5QHh53IZp9GSJ46pvOHQUq\\\\\\\/NtmOEZlSOzyDFaIcWAiyiRBd5a12bOo7+htSIDKdMzO+J84C8nuZeVAoN3KC1dtgUyIG4sDV\\\\\\\/hKwQYSmND\\\\\\\/FaL26SWqzyk5Mu246bH5gaUWm+B5ulIg+V8IGJnP2hEsgrHd4MAmdbXPXc393DYTkZC9P9o2GBtcemzgSqLowls71ABvzGIuDeKzC4DSLK6w9X9pznLQgDAJgAPUMeO5QDg1qrUrXE3mHpnhslA8ELgdLNHFJwk4CrDmrNdNVS2x9El6cIkK7LmN\\\\\\\/eyVt32JT+\\\\\\\/O5sIJyZQEMfpeWATWmpO8HzscYsjFee2Kt4LS861Ab\\\\\\\/nJ7Mj\\\\\\\/GBTIpleJkYCPcpozqRF4qXgxxrbSvop+rk2KTAzP28+T\\\\\\\/sVHcfbsYO+ZTlLq7AtKnaT8yrgcT9WopvkVO0ccdnDyQkXYk4sV5afsIop\\\\\\\/xdhdlj\\\\\\\/2b8u8u7TeQ9kC0pzHqwI7DgYdpUZ6im2\\\\\\\/pAwcKnUdwUwaiKEbnEyOynWHEb++udom2LFAh2wzp6tUOFYe581CpKAc3RKVe8nuWj\\\\\\\/0ZWXSLj2yTs4MRalGcVyYIR0vCBqYK1SYgzwFOh71iiAMogTZtqLpRXd+\\\\\\\/l3a20Se9TDWhSHwJzQqZYMAJ5Ym1X1FA6ylaV\\\\\\\/pNvXjGuBbCSVSJOb6TawAbr1\\\\\\\/u6WOhFpHEHpyuTMziVC4DsprhiQqFE4NSYU3BARtjkWxvmckj0l1Dct8xI0p6FAsfmlePKHU5XwCidwnKeqf+IDYebIe\\\\\\\/NlOmac7hBI9k3HIMyAbgfWVMjQBrUlkM5S\\\\\\\/dM9n9BwsO1J2F7i4\\\\\\\/NLUQWGr5JCxRUcktgq0np5rLyEGw8n5fb0AzRpqlrNRNvGnOrTtEyEg2DDOrxa\\\\\\\/DtS11l63IIvnCubbD+G0fLeolplgv6Wzmts+mhtWEbZNGbWcnt7DXVhBDBvssu8RomKDMDsn5GvkUoy4+K0IyNFiwlABWE\\\\\\\/SJsBMNiLUaibAEwtJonaKNurp7oB+7iyEK7WWICEbF1O0Zgk60niEW3LM8gbUOg9a1THPQuKk8ram3SsvkoU4+Rc3elpuujYoC\\\\\\\/0ofaZpIyejRtySGxgVsTh4HHA0Ivc3vI1QHPABgCw8trsgQPfBDmd18qewcQBGAGaixIU591XmKnsH9aiXVpPbDvpdfmea1rl8VUWRS6f1Pu+heEjfBt2rz97CSFqqnirJempMiI8ZKzGns1OyBlsp564rT\\\\\\\/6B6xa4IWy4ow+aH73P1Q\\\\\\\/dTeaJzVzXleZf5DGUbMtBFXK4rQN2J8xTcsQxVzo4ZooKaIKRYpaSeX0Q3FaXt8UAeb+O0v8WD+nH6jqiJG9MJdRGhG9fEwIdOuqxmu8cgPlphpnVvjLdJnLBEJxfBIXy\\\\\\\/9b8tzg1geTBJlIPW9jeMIsHtmKTrx+8sflLEcJY8pObMXu+VmTyfD35wM8usn5Ixip7tEmgW9+CEPt+tMyV+mb1q9d6oDfdt9CtiMj8PODglfU6O1nc2sZxeZfZo4Q9K+tCEYcRIPJZjRZWSNgRCzIGgWxxRDhnkjSpymXnMqXht+\\\\\\\/pWV1pGTlqI1n+4bCBeDFe9ggXC13C270aA12gqgRaE6p4o95NT3nT3fM5TK6cofqCuRRy4icLPAHePVjmjQ0t7O5kT6+hXia9f1DAqrjUWPp7TXMFCVA8ZwFCywe0GaResXnGWBYfAa9CAlriV36j9v6qM9VGoOP2hWalgMqENeHoKsX1wxxN+sbpTmenk6U87D9rKTI9FvGr7iHfqlwUued31+mz9\\\\\\\/RRY9C2Gja6+sMhvVvmzywC7tT5+ExuhcEu47gBMAqGVp9jgv0pAM2ZIrxIv18W21v75y2yd7O1sTUz2lwup9NleJLljmMO5zjcWJ5ZF10OJ8tXOANi8JT4b46VxKcgnVi1f4ishasfFgkNW8W\\\\\\\/ad+V0tPXIk9L4hxoCHNLJh6F8Ib8QhsdfQ\\\\\\\/N6VgnjDCak6\\\\\\\/JzPApTOBK4CGe9wHpVTSYk+lCsUofWg1QnsAantfqr0NLhgScBm5oMdpoaeQ5iVdgwLTXjwfz6RUouaGFhBA+AkCj2S+RXTWQe6U49hiN+R\\\\\\\/UA3pfO4gfGa5XUambyY+M\\\\\\\/RWFSB0adv0yIaKDFLsWrrA1OmtAwYxiSgbAA4xnoXzndKL9gxXxQLegFBaHob3ti1PrsHQ+dP2CB9gz8Q7\\\\\\\/YYlog1pmIYprjY\\\\\\\/LiV2+m1IPckO3xZ9loRrULioeBi+6XIdtLcdtqxuMxDlFTAe\\\\\\\/AaYGyU+wZ2lDAAoXbgILDgQqXAza\\\\\\\/xoNzkaq8UL0pIql0KJqxfjVs0talx6HJ9hJ1iI9zspmUTcjtOg12YMDVKbiAwGJ3MA2VSHxf6EF6cjGOByMtZEM6W6Ary7WC1mz1rMFoiSuel9TedrNe2vnnurOzwDy5diHYV1OKWRbTjTzG8bHRUqV7Ynm37qzPrHrJwxKsE+llJN6jABxZ3AKtRQyCvlQnusbSOy6PwmON\\\\\\\/iELDrGbZaPocyNXb51BXwjoWrjcJ4T2z1xbl94rIgnRAN3gr9jwlDxLAzwg3f+dTUanJ5xvujO9iNfljZJQ8mMw7ggIx96mM8zxq+C6xUmIcsHk1UhE9i8bhZN\\\\\\\/1DhhWQf2Pu2Sk8kArbuiIrpsz+BaMm2bWvCDpogjDDxnWlBYvpmYAC2REU1pP9Zt1hTudUCpt+pfgqfmpYxQJbzY1gXBymWcVDZemjBymvnyvEvhJyArjeger59BWM8a356n\\\\\\\/wKp\\\\\\\/cMo758L7MHDfHOuH3raT367QowkFjsU5hT1U53xgfsnWXQO07LEnetDSJIx6nfWyKg9Hm3QFeFx6Fp3\\\\\\\/Uf5zH5Q2zZ95T5oHuPaJz8hIk0rlzgHaTt9e2m\\\\\\\/\\\\\\\/jdvh5EB1RlCvexcnlDLsyAZNP52jLj3UOpAsFp+lOBun1aO4u4hAJ2hrHwbq+8LAkt1+kSS2ALt0mamcPsA+dZnJAHs4bawhWrK4ZpJoY6jP6gIxHnPRgVFrcS45M71bIqpbdBgNdCVkfKKL\\\\\\\/Up4OfJGBmyPzE5F\\\\\\\/Gz\\\\\\\/vlebDEZ6cfdwmhQzCztwVMoQ7ig\\\\\\\/6fOgb\\\\\\\/jxf4g8LmBytkzOkO1pm715s+7FkOpCmWxz3HUIlZTHkw4hPRxxwKT5T5XTlah9QOoH8sURpaOelM5thmUDxMumbLEsc7FwXDm6BJCKh5OBdNQ0w0\\\\\\\/RsHBAGgvIJNrNHyZfDvSG7mFr+ESnCexVBTe0N+IuhjxqdGDaYYCGihXHmvuDenZkQb2\\\\\\\/13ugDVJgqd0eOwW4RAYk0B4LBZU9jwL\\\\\\\/f8JIahXbBsF9cafA4UYO4ro\\\\\\\/jyjgv9JwaofogYYAxfVr7jP9Q0XDY1Sq8Ok0FIcf3b1lYlhc4dgKLHDHAwWXJThJdR0tqfh9OmuYgVpPyfDOUtQ1KG93gbjRFsMDaZjPtf9NVxGcs0618hgcywN+t7asfqntVzdIXNJmk5UIuP9PzViEA373jQr0+liobXVTX5gn5en\\\\\\\/zX4FVnEKNw0pPdp\\\\\\\/JUqkfeZa1efnZN87YKYxBu1DZCIfxJp7ws7MBnPcvgfcqrQ\\\\\\\/VYHFmC481Ld8WcoYlWSRGlqDbQls4qiDqYGeAnS5prvLUzsPXOIg7Vxm3ACGzI8pThSjsn+4QxHRe3YBCoHNFz2BL88i14GxbjQPWCZcmQIN4zNPnJvQMf0G3iz7OWdG313spCpj+EiucPAHFWIRUNKiCr2aSZB8EI5Cghf4v3V0fpzdUhlJAL3KDjooOMlqycVI\\\\\\\/gZbbyyXqC8L2\\\\\\\/BKxxDLEDytSo1c6O8ubMjxa1e7W9Znok71Phh0M5vDlWGV4gL6g3Cf\\\\\\\/9Ci0oA8Ow3izU\\\\\\\/j06EQhDXJktWLSU1eZO6\\\\\\\/E9J1Rl9rGqNk+aHUvvGAxKU\\\\\\\/J3NdHRSTv5FikbHmyidbIMVP8mOYbE3U0vI3mXNASyzg7qoRSgomPif64sFsCt3qEw0XOgs75YRdXLWIS7qmcsnXnSyDjti+uVZwTgIv28t\\\\\\\/aE5\\\\\\\/PRfH6YZT\\\\\\\/AeX92dMYgCAi8NcRs5tQPcXJg1SbJ\\\\\\\/dTMopqFNUb1n3jgqtgwCwWHRVrjUAQQFfS1mfF5K0l8M1q10BXFW5xKq894bD1QoBsmV5mCPEaMMxjtTkfZ5Td1wmxROy56ZnxYqGckJvfvofPiPvgyIcfrKBil+BoD2aNe130g\\\\\\\/+FZ38LgXf+jyiYNsGWv7wzyYC6FK6A9NlWtOjb4S3KxvknouF950XhRIkJZnDytY5Qw778tPB0YX2yAmy6IwLHf9g+CjaSm+VNsqq25xjKR\\\\\\\/vfHBp0p3De2EDE\\\\\\\/Nanz3JDbDEEnS3ZW2+wQnYmkUroIoEwJrEjdzJ05BSCDqLP8B1nMFVfhsqNq55R4FS9gu5hHt5Xgowh6E50XmvfCvVXQdUa3KeVjJBFPIm7cP7tOU+m1oJuGjkvwlUEzHL+VC+u46vvxA1FZ94HnmRTR3FRHBCIS\\\\\\\/i+sK9fL0o3ZwK5PwliyTcM7Y0DTM6kn\\\\\\\/bIMk9ltrTpQZnuXDie9qqWnWdU6w79mlAFQTtfDyARlgi1rAzymm3JkCGgfDjvpliuCe4Yg8C7xlIUjnpXbVn\\\\\\\/PlCDwbdYXEusCw7odoVvgM4NV\\\\\\\/9SKHstm0kyfQOHn7rTGqG7OpGb4KptZpOnq36D8bKTT7f+1SX88ZekeiOUGT9BBD8DjcPRXkHTeU3LpAdCO5giAWw4026t93dO0CBMSdxbCRgNh+u0TJoF2QMqRInDVPyFLBGc4PIlxaUegOrs6I\\\\\\\/Q3uPHcK7ilvF9moknwIC6doJFkfmKkv\\\\\\\/xxRtj3Ovrj8NqoxPVOnjLtWR6L3BZ1YfrMkIlX7frER3qjQSZzybu37QjqwMt\\\\\\\/5olKTTlbLVE6V7aclGuSFPG1RuWWT1G1SIlH06hN6+gs7DfO9FqfXbxQxDpZf+RiP0qIy+NASj2jwF1weJbqD\\\\\\\/oiQuWVP+dSJmetayjZd5QvRl+wzlPUhFiWfXN0ZpDek3idzXOPlTzbBH9ZXH+F4fI2f+WwxB0NHiGuhaFLnL9csCicl4ofjBwZiEU885XnPhUiVAdLUSm0LdKZ335UYESmSC\\\\\\\/HMmENRcZWplIfW\\\\\\\/3DnjzsPZCxlej7oVgLHOcxpKGrARQBTypWvi8tRJYfNLhTh4gVQGJsTRjJHaxMGvMB\\\\\\\/yoBPNggQZiW3V6CbKYB75TPmFnx6IUFzAkeipucxQ5vg0QgiKF2PK+on0fBN6eqf0EvmR4uGTx04B2dto\\\\\\\/YG\\\\\\\/02JxYIH1eGdzAa\\\\\\\/qEuZTXSGqRJTS7dT8GfQ1i\\\\\\\/Y\\\\\\\/qzqipw3tFdT1Poqm8ByZKS+fBP81CtcdVlLLjSAiXs82nZ2MaoDmyFinpdvlz0Uv\\\\\\\/JMjgPjIBZutCDXA\\\\\\\/2f7dLi\\\\\\\/bjl4Al4xSbCpDCAVyK9Vj9bqBpx2wXWJuslw+9L6e7Vcnf\\\\\\\/CQhNBXTrWVuaAiJqC0kw1iQ5fPuHQeXNTPtktGLPHpEYSkUpSlaKMyuIq\\\\\\\/KWjSd0CwQgIHSNWXS4dpdto+Z7KFxeSi45V8oWEgaDX78k176UO7qY0ERIrI5rvJlrXJjYzeoJ+RK1iBqyu7VPPQqNqRP0Dzw5ZSDOt\\\\\\\/5Qah4cqZmMvZafeLb6AneobN9rE1OTVwu04AmcO6wf9N4WpssSg5WWpL9o88EyE56CU78YNY43qVIrDJzehAFm2awFU1rWYJ7tKDyqmz5kQuwjYN7uyXueVLKdhDXV7XbKqAnYp4O+DrG\\\\\\\/\\\\\\\/CeswIGh\\\\\\\/9u702PGLLwGR+Q+LQWzt9pIlH3fvObQe4JyTGVLU6NZwXylROhGf37HC3zT5slvJTWnCelKY9qsXViAHqn0wvtuk3iWPqlnVGnBFR\\\\\\\/2hdcDx7CgpYwrBYh9iSsdWKwq\\\\\\\/CxJnd8tZ16zEkgjK4oP9WV1AtHPOHVZAYJ\\\\\\\/jt8q\\\\\\\/F5QvJhfnnKay8LqfrLEXRdmL8vn38Nnl9EDuu7ekwjV0HrwJqe1DucARtCnTZCOlRG2gIhS74bFsvVC6Wyp1jkVFV8LSGQzC8p2aQ\\\\\\\/1QJHzfptbuJbmxJRF2MXTsK7QRh7w3YyRvf2k8Fw1sIK1foFbz+yHsWf8uNfOc7kPmo\\\\\\\/ocn89T88EQufCR1xHLPHiRPSX1V4ZdsoaZw1UUQ5c+I1\\\\\\\/4oRp6tZYbHPZnICwDXnZL1qMmi1\\\\\\\/g5fzfWgklo\\\\\\\/J66g3g5endSH7Sm7BBmNqaD2XByj1dy7mGabJtleJqunggGRBm1jjb33oFjaSrUtydDgowXCeWSEo4T8Hu7S0qlquqAMRX0cdT++enxbUiXjBgtXyy4mrzq+ib22nTk1ojmOLcJU8y99S\\\\\\\/9pt4RFt\\\\\\\/+7isPMXaqTyCw\\\\\\\/e4wULxjf7GIVM7AXaQhKVQZNvYFMxBxIbpWRbEZUqEGHEcJjp1Ct6GM1hu3NtcVbucDYTeraPY3Xm3j77f5mIsA+uv75ffEVOz9DuVBISLj4GRfwMGmJyoKg8kDgueg+6LHqNMkL84bZmr6fFirYfKlonlctxbt3DY4HpWZiKEdDkpWU415KZDfaVtn7x1\\\\\\\/XsYoJeIurNYutxxLqeegdged+gfTtt4Qm2V5Hr8CJiVLlq43eiuEPmJ4GED0Bm9RAp2UL6PvFLgvSPyti7YCXXPRIXoYtvlh8tHsIq3qt2O8aBF8HVETLGWhHwD5GSAwK8P\\\\\\\/D4yToZCcw0MmvfSL9bRPBfdcPj8kXSKIOJGhvVXorUYII9VX9hz1b1HS7mctPoi9R9\\\\\\\/egG1uQ03hXmeBJJr0o7d7jNIpDwHVzg2w9o+cvb6xbq3lz0Iz7fCp5UHiOkyAC6mMrzch9KIbimvQiWlxDYapG3XlhlEYXs4o9fzVIb+HTwj\\\\\\\/\\\\\\\/JByg7kvlyKHB+7KljF3MNytJGGUFkuoyJDu13T2m5UlnDTplZW4KcaKfIWthlvZ\\\\\\\/zzixlmATP8UlOirBtsZQygGCw5FLfqiXrFRNiApo5muVU4drYmngUyS0swOW02DN7CUVmCtajg7r7HEWA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022EjShGe5KS84YJqnFXtGHdA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f08f3a1b8124e65dd88147b9c7c76c676063496096dd6183c9da3a7e578816cd\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022dHXc2n7PWkeD\\\\\\\/JBi49OU2bJ7eX+ADt4CGK3U8u1ZH8T0O6kf+\\\\\\\/DZuVsDJGjmGgit6xW43RVVXCDngKxjzYYceMQ062Pcr+FIvzAVcfxmM6ghLkQ4+QJc44u9LR+tlJOqVBeLej7X1ViN8UAm2dBAWKO2qaNtxJhaw73blNkTI85p1xGXSMpCwPFHSF0X\\\\\\\/RWW2bLbCTKB281IE9j0nKKaPbAhWd93WyyTBZbavYBe6gFMoebTHunkpEBnjXIT2VsOQ5pvjgpeRbTWjZjyyu6edwL1jLoZO+C0k01vFEV2z5fvnwHDwDCaRPsf32H1bHoOEXV5kYolRl39XHO8yVtmuJRrk+UfJFop0D8adDhjkvzRqZCiBCzQBxvvHyWJE0LmXHfgVhAI9NcNPnH+BiRyGgNbzqnHGcahuLs\\\\\\\/ONeC+y3YOcQRZZ4MqoHPXBm7cVWwzSPbDqzko9Mvhy3Etqu+perQVH4SBxnDZ7tA\\\\\\\/ZIYV88exk8VUP6RyPh710FRA1LS2GQ\\\\\\\/ZlDb73YIxJuDylblemzS\\\\\\\/AwGQfrvDcCLz3pO9Q68+NTcPpBoikql8q159g7kG6RbLMZaXQJIM7K9yx9bqns8t2q5XHY9hvGxJ8QYxoQx1goTP9jSbxxS50z6xdJDgX5ftDBGQI0Ry5smKAmPun8JHshm8m\\\\\\\/YrHHQTX3o1z66AvqlV6lkdCWHR49T8uHgyy7orYYIGPMjV3KIxBcTSaGh05nDcFtlUScVsgCGqguW+CftAyMW5tO+I+V41z1\\\\\\\/VmRhusaCNIY4l9CyDMEYxSgZAD0zTbPTe7lfxbbY9tE0AvdaKHLQdg3MSEa7WVlnnD29rqznS4bjPZL4LywPl9kbPFJTwW3LpB5ADu9RMvqas\\\\\\\/rix3\\\\\\\/TsYq8rntdf0otYB5zb83cOjiAhDRXOFpm6lUT9z6H93mYsRzNC\\\\\\\/zE0drwsjXLknCqYRxWzhjKJ4+ilY7uK+AKZ4q7uVCmHNSbzUxQ9YGgOpbN\\\\\\\/IuCKHHkTwtfbhXRRIxAU531VBznKf32EQA+KUEtaMPSfao6+JKJU9Az1g4fUYxwpWcFiIXaAVHN3qk3Ri5ooDWm9qJVo5IEBJiTMJYwPQkbTPuaoFTvktJsGUeXqp\\\\\\\/FLl0GuXF++8l1Rxpb+fdSqIdAJFEcKDMyrFmzAJSnwPAUX+ptbRZXhHC8U+kJaPp4dS4NZBPXuHZy3uPepa8FcBc1XHYD4IpuxC+ZhmYkgIiokPy9SBkV\\\\\\\/G\\\\\\\/zn6n8iBVjB594H1OUbBKQ7e\\\\\\\/XkrF9UDWsvKK6KWrw\\\\\\\/Z2L7UCmqsit9Pd126+cxOfY\\\\\\\/DEkaUagk18Tf439N9zbZyjHQukbcn6uodPUeSsgBMeKrVdgYwXINet1P2Adamthbw6sGOyX+v1UsxaTIRpfxr3j8bdcO6osb1Y9UPeA24hL46pfysHGW+xt+k9dV1nAYMujvthXVw30JT1V17acbS9sqzRuoAo5iOWx3OewIYKI3c5Za6MCPRBHJmX0S2gxKegQ\\\\\\\/U6+2vhxiUWFv7XVyQjM8Y6j3C8bYKmIZJhLMHv5rQbK4vGsI0A6RIcDDTmrI6zBlZDpd02nNzea37XSfvoRxTUU7i6morSyur2brqSCnixYPTynJNarTb6RGC1GUWcCGiW3Oq0eSFS5KB1eea+HfrwttBbCaTTvLLQmST+IeYYQR3cVjP+TrUZuKhqfKCwmZrzZvW3e2qqJXCnvR+aEGne+gCNOmDNpTunXRAqAqaAgEFi8nAFyL\\\\\\\/H7IkTHEXWBSF5HRZD2AKXnnAwbPQkiYiXspc+Fh9GBccbk7Vc\\\\\\\/jDqkJnermPc9EJpYMJRVDAMuKucepflnHe0lEidX4xeWtKM2ouT\\\\\\\/p+mOcKBJW3bYc2Q9Y1pw1oCdx4w5CZFZgxclJUQqGmVF92slBU0IZjxJtGxA2H9LJzSmRySjFxowp53e1jE9Wr8OkYxUzTxPeXWpnECjLGH3oFP0gyXDuP9h9cxUuDa0ybojrjYXXYOHafbQshUrXSOV64Cw3A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022CIXlYc7CQxW02sCha9NaNQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c8380f425498bfac9577942ea925f9bbfd57f477062c516d233ff537f44e5d94\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460125 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/805" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YEMfcNVeOyN10nWENEYroi7bKPva5LaeeQtz_rzHOeE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/805" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Kc8sbZqs8QLEN_aPH7XeWBdTYF9qP4JlBwgAZ8OL8V4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/4/793" title="Identification of Polymorphisms Associated with Drought Adaptation QTL in Brassica napus by Resequencing" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/4/819" title="Drak Is Required for Actomyosin Organization During Drosophila Cellularization" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460125" id="node460125" data-pisa="ggg;6/4/805" data-pisa-master="ggg;g3.115.026849" data-apath="/ggg/6/4/805.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Analysis of Transposon and Retroviral Insertions Reveals Preferential Integrations in Regions of DNA Flexibility</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Pavle Vrljicak</span>, <span class="highwire-citation-author" data-delta="1">Shijie Tao</span>, <span class="highwire-citation-author" data-delta="2">Gaurav K. Varshney</span>, <span class="highwire-citation-author" data-delta="3">Helen Ngoc Bao Quach</span>, <span class="highwire-citation-author" data-delta="4">Adita Joshi</span>, <span class="highwire-citation-author" data-delta="5">Matthew C. LaFave</span>, <span class="highwire-citation-author" data-delta="6">Shawn M. Burgess</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-0729-1977" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Karuna Sampath</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">805-817;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.026849 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460125" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Pavle Vrljicak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Biomedical Sciences, Warwick Medical School, University of Warwick, Coventry CV4 7AJ, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pavle%2BVrljicak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vrljicak%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APavle%2BVrljicak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Shijie Tao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Temasek Life Sciences Laboratory, National University of Singapore, 117604, Singapore</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shijie%2BTao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tao%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShijie%2BTao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Gaurav K. Varshney </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>National Human Genome Research Institute, Bethesda, Maryland 20892-8004</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gaurav%2BK.%2BVarshney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Varshney%20GK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGaurav%2BK.%2BVarshney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Helen Ngoc Bao Quach </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Temasek Life Sciences Laboratory, National University of Singapore, 117604, Singapore</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Helen%2BNgoc%2BBao%2BQuach%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Quach%20HN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHelen%2BNgoc%2BBao%2BQuach%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Adita Joshi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Temasek Life Sciences Laboratory, National University of Singapore, 117604, Singapore</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adita%2BJoshi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Joshi%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAdita%2BJoshi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Matthew C. LaFave </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>National Human Genome Research Institute, Bethesda, Maryland 20892-8004</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BC.%2BLaFave%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=LaFave%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BC.%2BLaFave%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Shawn M. Burgess </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>National Human Genome Research Institute, Bethesda, Maryland 20892-8004</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shawn%2BM.%2BBurgess%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgess%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShawn%2BM.%2BBurgess%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:K.Sampath@warwick.ac.uk" class="" data-icon-position="" data-hide-link-title="0">K.Sampath@warwick.ac.uk</a></span></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Karuna Sampath </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Biomedical Sciences, Warwick Medical School, University of Warwick, Coventry CV4 7AJ, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karuna%2BSampath%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sampath%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKaruna%2BSampath%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0729-1977" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Karuna Sampath</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:K.Sampath@warwick.ac.uk" class="" data-icon-position="" data-hide-link-title="0">K.Sampath@warwick.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/4/805" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460125" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460125/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/805.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460125" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460125/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/805.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460125" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460125/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/805.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460125" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460125/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/4/805.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460125" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460125/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">DNA transposons and retroviruses are important transgenic tools for genome engineering. An important consideration affecting the choice of transgenic vector is their insertion site preferences. Previous large-scale analyses of Ds transposon integration sites in plants were done on the basis of reporter gene expression or germ-line transmission, making it difficult to discern vertebrate integration preferences. Here, we compare over 1300 Ds transposon integration sites in zebrafish with Tol2 transposon and retroviral integration sites. Genome-wide analysis shows that Ds integration sites in the presence or absence of marker selection are remarkably similar and distributed throughout the genome. No strict motif was found, but a preference for structural features in the target DNA associated with DNA flexibility (Twist, Tilt, Rise, Roll, Shift, and Slide) was observed. Remarkably, this feature is also found in transposon and retroviral integrations in maize and mouse cells. Our findings show that structural features influence the integration of heterologous DNA in genomes, and have implications for targeted genome engineering.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/transposon" class="hw-term hw-article-keyword hw-article-keyword-transposon" rel="nofollow">transposon</a></li><li class="kwd"><a href="/keyword/acds" class="hw-term hw-article-keyword hw-article-keyword-ac-ds" rel="nofollow">Ac/Ds</a></li><li class="kwd"><a href="/keyword/genome-wide-analysis" class="hw-term hw-article-keyword hw-article-keyword-genome-wide-analysis" rel="nofollow">genome-wide analysis</a></li><li class="kwd"><a href="/keyword/integrations" class="hw-term hw-article-keyword hw-article-keyword-integrations" rel="nofollow">integrations</a></li><li class="kwd"><a href="/keyword/gene-targeting" class="hw-term hw-article-keyword hw-article-keyword-gene-targeting" rel="nofollow">gene targeting</a></li><li class="kwd"><a href="/keyword/genome-engineering" class="hw-term hw-article-keyword hw-article-keyword-genome-engineering" rel="nofollow">genome engineering</a></li><li class="kwd"><a href="/keyword/functional-genomics" class="hw-term hw-article-keyword hw-article-keyword-functional-genomics" rel="nofollow">functional genomics</a></li><li class="kwd"><a href="/keyword/tol2" class="hw-term hw-article-keyword hw-article-keyword-tol2" rel="nofollow">Tol2</a></li><li class="kwd"><a href="/keyword/retrovirus" class="hw-term hw-article-keyword hw-article-keyword-retrovirus" rel="nofollow">retrovirus</a></li><li class="kwd"><a href="/keyword/mmlv" class="hw-term hw-article-keyword hw-article-keyword-mmlv" rel="nofollow">MMLV</a></li><li class="kwd"><a href="/keyword/mouse" class="hw-term hw-article-keyword hw-article-keyword-mouse" rel="nofollow">mouse</a></li><li class="kwd"><a href="/keyword/es-cells" class="hw-term hw-article-keyword hw-article-keyword-es-cells" rel="nofollow">ES cells</a></li><li class="kwd"><a href="/keyword/vertebrate-genomes" class="hw-term hw-article-keyword hw-article-keyword-vertebrate-genomes" rel="nofollow">vertebrate genomes</a></li><li class="kwd"><a href="/keyword/zebrafish" class="hw-term hw-article-keyword hw-article-keyword-zebrafish" rel="nofollow">zebrafish</a></li></ul><p id="p-4">DNA elements capable of genomic integration, such as transposons and retroviruses, are important tools in molecular biology research. From the fission yeast <em>Schizosaccharomyces pombe</em> to humans, these vectors have been used for gene delivery and insertional mutagenesis (<em>e.g.</em>, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cavazzana-Calvo <em>et al.</em> 2000</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aiuti <em>et al.</em> 2002</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kawakami and Noda 2004</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Wang <em>et al.</em> 2007a</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Guo <em>et al.</em> 2013</a>). Significantly, the integration of these elements has revealed features of genes and genomes, such as the function and regulation of genes, and the “open” state of chromatin (<a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Wang <em>et al.</em> 2007a</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Genovesi <em>et al.</em> 2013</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Guo <em>et al.</em> 2013</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">De Ravin <em>et al.</em> 2014</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Davie <em>et al.</em> 2015</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Rad <em>et al.</em> 2015</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Takeda <em>et al.</em> 2015</a>).</p><p id="p-5">Three of the integrating elements currently used in zebrafish are the Tol2 and Activator/Dissociator (Ac/Ds) transposons, and the Moloney Murine Leukemia Virus (MMLV) (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Lin <em>et al.</em> 1994</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Koga <em>et al.</em> 1996</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kawakami <em>et al.</em> 2004</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Emelyanov <em>et al.</em> 2006</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">McGrail <em>et al.</em> 2011</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Song <em>et al.</em> 2012</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cheng <em>et al.</em> 2014</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Quach <em>et al.</em> 2015</a>). Ac/Ds and Tol2 transposable elements are members of the hAT family (named for hobo, Ac and Tam3) (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Calvi <em>et al.</em> 1991</a>). They integrate into the host DNA through a “cut-and-paste” mechanism requiring <em>cis</em>-terminal elements flanking the transgene of interest and the transposase enzyme, which may be encoded in the autonomous elements (<em>e.g.</em>, Ac) or exogenously supplied (<em>e.g.</em>, Ds) (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">McClintock 1951</a>). Some of the features that make hAT transposons particularly amenable for transgenic work in zebrafish are the accurate mechanism of integration (with well-defined integration sequences), the ability to be remobilized if desired, the small size of <em>cis</em>-required sequences (∼600 bp), a reasonably high transposition frequency, and moderate copy numbers (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Emelyanov <em>et al.</em> 2006</a>). Significantly, transposable element vectors have a relatively large insert capacity (&gt;10 kb) and are easy to generate in a standard molecular genetics laboratory.</p><p id="p-6">In contrast to transposable elements, retroviral vectors such as MMLV have a limited insert packaging size (usually &lt;8 kb) and their production requires specialized technical expertise. However, retroviruses are currently the most efficient way to make a large number of insertions in the zebrafish genome, producing a high number of integrations for a given experiment (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Amsterdam <em>et al.</em> 2011</a>). The high copy number potential and high mutation rate were key features in the successful use of the MMLV retroviral vector in an insertional mutagenesis screen that targeted a large number of protein coding genes in the zebrafish genome (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Varshney <em>et al.</em> 2013</a>).</p><p id="p-7">One important consideration affecting the choice of transgenic vector is their insertion site preferences. Integrations occurring in the 5′-end of genes are advantageous for creating insertional mutants. However, vectors that only target actively transcribed gene regions have limited use in capturing genes that are expressed at low levels, or those regulated by alternative promoters and enhancers. Similarly, while targeting enhancer regions might be an advantage for enhancer traps and detecting open chromatin, transgene expression might suffer from the variability imposed by positional effects (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Roberts <em>et al.</em> 2014</a>). Moreover, targeting of specific repetitive elements might lead to transgene inactivation, while targeting of 3′-UTR sequences might lead to changes in posttranscriptional regulation (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Goll <em>et al.</em> 2009</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">McGaughey <em>et al.</em> 2014</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Shpiz <em>et al.</em> 2014</a>).</p><p id="p-8">Integration bias has been reported for transposons and retroviruses in a number of systems (<em>e.g.</em>, <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Vigdal <em>et al.</em> 2002</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Wu <em>et al.</em> 2003</a>, <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">2005</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Faschinger <em>et al.</em> 2008</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Linheiro and Bergman 2008</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Liang <em>et al.</em> 2009</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Vollbrecht <em>et al.</em> 2010</a>). Integration target sites are thought to be relatively random at large genomic scales, although there have been reports of association with genetic elements such as transcriptional start sites, strong enhancers or promoters, UTRs, and CpG islands (<a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Wu <em>et al.</em> 2003</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Kondrychyn <em>et al.</em> 2009</a>; <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Vollbrecht <em>et al.</em> 2010</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">LaFave <em>et al.</em> 2014</a>). At the nucleotide sequence level, different integrating elements have shown various degrees of sequence bias. MMLV shows a weak preference for T/A nucleotides just outside the 4 bp site of integration (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">LaFave <em>et al.</em> 2014</a>). In contrast, Ac/Ds and Tol2 are not reported to show specific integration motifs (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kawakami 2007</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Kondrychyn <em>et al.</em> 2009</a>; <a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">Vollbrecht <em>et al.</em> 2010</a>). However, these analyses have been limited by the number of genomic features analyzed, and the reliance on a small number of sites in some studies. Crucially, the available insert collections generated in zebrafish were either selected based on reporter gene expression or phenotypes, and only represent integrations that have been incorporated in the germ-line (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Kondrychyn <em>et al.</em> 2009</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kawakami <em>et al.</em> 2010</a>; <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">LaFave <em>et al.</em> 2014</a>).</p><p id="p-9">We recently described the use of an Ac/Ds transposon system for a mutagenesis screen in zebrafish (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Quach <em>et al.</em> 2015</a>). We produced a collection of 642 transgenic lines marking distinct cell and tissue types, and mutagenized genes in the zebrafish genome by trapping and prematurely terminating endogenous protein coding sequences. Significantly, our gene/enhancer trap mutagenesis screen provides an unprecedented amount of Ds integration data in zebrafish.</p><p id="p-10">In this study, we set out to analyze the genome-wide integration preferences of Ds in zebrafish. We examined how Ds integrations compare to those of other popular tools for generating transgenic zebrafish (Tol2 transposon and MMLV retrovirus), from the chromosome to the sequence level, and generated an unselected Ds integration set to explore the effect of selection on integration site preferences. We find that Ds integrations are more broadly distributed across gene regions than Tol2 and MMLV, and reporter-based selection does not affect integration site characteristics. Interestingly, while a strict motif at the target site was not found, we observed a preference for structural features correlated with DNA strand flexibility in the target DNA, which we also found in Ds integrations in maize, and in transposon and retroviral integrations in mouse ES cells.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Ds integration lines</h3><p id="p-11">The Ds integration lines were generated as described previously (<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Quach <em>et al.</em> 2015</a>). Briefly, we coinjected one-cell stage zebrafish embryos with pDsDELGT4 (a gene and enhancer trap construct flanked by Ds sequences) together with <em>in vitro</em> transcribed Ac transposase mRNA. The resulting founder lines were then selected based on expression of either EGFP or mCherry indicating successful enhancer and gene trapping, respectively. To study Ds integration preferences in the absence of external selection, pDsDELGT4 plasmid was microinjected into 576 embryos at the one-cell stage as described above. Injected embryos were grown for 3–5 d to obtain sufficient DNA, pooled into groups of 6 and subjected to next-generation sequencing (NGS).</p></div><div id="sec-3" class="subsection"><h3>Identification of Ds integration sites by TAIL-PCR and genomic sequencing</h3><p id="p-12">In order to identify unique Ds insertion sites, we first used thermal asymmetric interlaced PCR (TAIL-PCR) on expression-selected lines as previously described (<a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">Quach <em>et al.</em> 2015</a>). Flanking sequences obtained from TAIL-PCR were analyzed against the zebrafish reference genome (Zv9) using BLAT (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kent 2002</a>). Flanking sequences were considered unambiguously mapped if the entire TAIL-PCR generated sequence matched a single location of the genome assembly with 85% identity or more. This identity cut-off was derived empirically to account for differences between the AB strain used for Ds insertions and the Tubingen reference genome, and sequence quality derived from the TAIL-PCR protocol. Any ambiguously mapping TAIL-PCR-derived sequence was excluded from further analysis.</p><p id="p-13">TAIL-PCR results represented only ∼75% of sites expected by Southern blot analysis. To determine insertion sites for the lines that could not be resolved by TAIL-PCR, NGS was performed on the Illumina MiSeq as described (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Varshney <em>et al.</em> 2013</a>) with the following modifications. About 500 ng of genomic DNA was fragmented using three pairs of restriction enzymes (<em>Mse</em>I/<em>Pst</em>I, <em>Bfa</em>I/<em>Ban</em>II, and <em>Csp</em>6I/<em>Eco</em>24I) in parallel. The digested samples were pooled and ligated with DNA linkers, and amplified by linker-mediated PCR using linker and Ds specific primers to capture the adjacent genomic DNA sequences. The Ds/gDNA/linker amplicons were subsequently ligated to Illumina paired-end adapters and sequenced. The first round of PCR was performed using a 3′-Ds ITR primer and a linker primer (5′-TATGAAAATGAAAACGGTAGAGGTATTTTACCGACCG-3′ and 5′-GTAATACGACTCACTATAGGGCACGCGTG-3′, respectively) and the second round of PCR was performed using nested 3′-Ds ITR and linker primers (5′-TTTACCGACCGTTACCGACCGTTTTCATC-3′ and 5′-GCGTGGTCGACTGCGCAT-3′, respectively). Ds insertion sites were identified using a modified version of the GeIST program previously used to detect MMLV LTR sequences (<a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">LaFave <em>et al.</em> 2014</a>). For NGS analysis of selected gene and enhancer lines, fish were out-crossed and their resulting embryos were placed in individual wells, following which DNA was extracted and sequenced. For unselected fish, DNA from groups of six injected fish was placed in one well and sequenced.</p><p id="p-14">NGS of selected Ds lines produced 5473 putative inserts with fragment counts ranging from 5–10,000. To obtain a high-confidence integration set, we performed PCR validation of a subset of NGS-identified sites. We also examined the sequencing results of single insert lines and obtained NGS fragment counts for TAIL-identified sites. Based on these observations, a putative integration site was deemed high-confidence if it was detected with &gt; 50 counts.</p><p id="p-15">Based on findings with the selected set, we devised similar criteria for the unselected Ds set, although we lowered the general fragment count cut-off to 7 as we expected inserts to be diluted in each sample. We noticed that recognition sequences for restriction enzymes used during the NGS protocol were highly prevalent in single fragment putative inserts. Therefore, we applied a more stringent cut-off criterion of 50 fragment counts for these sites, so that no more than 10% of the total sites contained the restriction enzyme recognition sequences.</p></div><div id="sec-4" class="subsection"><h3>Tol2 and MMLV integration sites and matched controls</h3><p id="p-16">Inverse-PCR and TAIL-PCR results for Tol2 integrations were obtained from published gene and enhancer trap screens (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Kawakami <em>et al.</em> 2010</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kondrychyn <em>et al.</em> 2011</a>; <a href="http://kawakami.lab.nig.ac.jp/ztrap/">http://kawakami.lab.nig.ac.jp/ztrap/</a>; <a href="http://plover.imcb.a-star.edu.sg/">http://plover.imcb.a-star.edu.sg/</a>), and mapped to the Zv9 genome assembly as described above. MMLV retrovirus integration sites generated by NGS were obtained from the <span class="underline">Z</span>ebrafish <span class="underline">in</span>sertion <span class="underline">c</span>ollection (<a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Varshney <em>et al.</em> 2013</a>; <a href="http://research.nhgri.nih.gov/ZInC/">http://research.nhgri.nih.gov/ZInC/</a>). We used various matched control sets for comparison, taking into account the sequencing technique, genome mapping, and size of the different experimental integration sets. In the case of Tol2 and Ds integration sites obtained by inverse-PCR and TAIL-PCR, we generated one million 50 bp random genomic locations using the BEDTools random tool (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Quinlan and Hall 2010</a>), and mapped them back into the Zv9 genome assembly using Bowtie (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Langmead <em>et al.</em> 2009</a>) to remove regions mapping to multiple locations. We then performed 1000 independent random samplings of these regions to produce control sets of the same size as the experimental sets. For NGS-generated insertions, we replicated the conditions of the sequencing protocol and took account of repetitive regions of the genome (<a id="xref-ref-36-5" class="xref-bibr" href="#ref-36">LaFave <em>et al.</em> 2014</a>). Briefly, we identified the location of all <em>Mse</em>I, <em>Bfa</em>I, and <em>Csp</em>6I restriction enzyme sites across the genome. We then calculated the distance from each integration to the nearest of the three restriction sites that could have produced a mapable fragment. We used these distances to generate files containing one matched random integration of the same distance and same restriction site as each experimental integration. The corresponding sequences were then aligned back to the Zv9 assembly with Bowtie using the same settings as in the experimental workflow, repeating this process 1000 times. In this way, the random sites take into account two potential sources of bias: distance from restriction sites and alignability of the read. The selected and unselected Ds, Tol2, and MMLV integration sites used in our analysis are presented in Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026849/-/DC1/TableS1.xlsx">Table S1</a>.</p></div><div id="sec-5" class="subsection"><h3>Bioinformatic analyses</h3><p id="p-17">Integration sites and control sets were compared with the various genomic features using BEDTools intersect (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Quinlan and Hall 2010</a>). Genomic features were considered overlapping if they shared at least 1 bp of the insertion site. Location of CpG islands and repetitive elements were obtained from the UCSC browser track. H3K4me1 and H3K4me3 hotspots were obtained from <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aday <em>et al.</em> (2011)</a>, and CpG DNA methylation from <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">McGaughey <em>et al.</em> (2014)</a>. Gene models were obtained from the Ensembl database. Where appropriate, standard nomenclature was followed (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Mullins 1995</a>). Gene ontology analysis was performed with DAVID (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Huang <em>et al.</em> 2009a</a>,<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">b</a>).</p><p id="p-18">Ensembl gene annotations were used for determining integration site distribution across gene regions, with different features obtained from the UCSC track. Integrations were assigned to a gene when they occurred anywhere between 5 kb beyond the transcription start site (TSS) and the transcription termination site (TTS). All genes were counted when multiple genes overlapped the integration sites. Similarly, all features were counted when multiple gene features overlapped the integration sites. Intergenic regions were defined as lying beyond 5 kb from TSS or TTS. To look for distribution along a gene region, gene size was normalized to 100%. To study the distribution of insertion sites across the TSS and TTS, the distance from site was obtained within a 1 kb window around the gene feature.</p><p id="p-19">Gene expression information was obtained from previously published RNA-seq experiments (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Harvey <em>et al.</em> 2013</a>). To estimate overall gene expression levels, we combined the expression level in FPKM (fragments per kb of transcript per million mapped reads) for each gene across developmental time-points. Sites were assigned to genes within ± 5 kb of the TSS and TTS.</p></div><div id="sec-6" class="subsection"><h3>Statistical analyses</h3><p id="p-20">We carried out genomic feature preference analyses by bootstrapping, searching for values of a given random control data set that differed from the corresponding value in the experimental set. For a given genomic feature, the enrichment value is the ratio of its prevalence in the experimental set over each of the matched control sets averaged over the total (n = 1000). To calculate the <em>P</em>-value, we counted the number of random sets in which a particular feature was enriched or depleted in relation to the experimental set, and divided the total by 1000 (the number of random tests). <em>P</em>-values were calculated for both enrichment and depletion in every category, although we only report the relevant <em>P</em>-values here. The significance threshold is <em>P</em> = 0.05.</p></div><div id="sec-7" class="subsection"><h3>Mouse cell integrations</h3><p id="p-21">Retrovirus and transposon integrations in mouse cells were obtained from published datasets (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">de Jong <em>et al.</em> 2014</a>). Specifically, 131,594 Sleeping Beauty (SB) integrations and 1,222,667 piggyBac (PB) integrations into mouse embryonic stem cells (mESC), together with 180,469 mouse mammary tumor virus (MMTV) integrations into mouse mammary gland cells (NMuMG), were analyzed for sequence and structural motif discovery, using random integration sites as controls.</p></div><div id="sec-8" class="subsection"><h3>Motif and structural feature discovery</h3><p id="p-22">We obtained 48 bases of flanking sequences from the zebrafish Zv9 or mouse MM10 genome assemblies for motif analyses, preserving the orientation of insertion. In the case of Ds and Tol2 integrations we obtained 20 bases before and after the 8 bp duplicated site. For zebrafish MMLV or mouse SB, PB, and MMTV sites, we obtained 20 bases before and 28 bases after the insertion site, so that position 21 was always the first position of the integration site. Consensus motifs were generated with weblogo v3 and displayed as bits or probability (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Crooks <em>et al.</em> 2004</a>).</p><p id="p-23">We used crystallography-derived values to calculate six DNA strand movements (Rise, Roll, Shift, Slide, Tilt, and Twist) around the integration sites. Using a custom Python script, we reduced each 48 bp sequence to its underlying dinucleotides, such that position 1 and 2 represented the first dinucleotide, position 2 and 3 represented the second dinucleotide and so on. Each dinucleotide was then assigned the corresponding movement values deduced from crystallography data (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Olson <em>et al.</em> 1998</a>). For example, a GC dinucleotide would produce an average of 36.1 degrees of Twist, and 0.41Å of Slide, while a GG dinucleotide would correspond to 32.9 degrees of Twist and –0.22Å of Slide. These movement values were then averaged for each dinucleotide position and plotted.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-24">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Generation of high-confidence Ds integration sites</h3><p id="p-25">Based upon the experimental strategy shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, we generated two sets of Ds integration data for our genome-wide analysis, one “selected” and one “unselected.” For the source of “selected” Ds sites, we analyzed zebrafish lines obtained as part of the FISHTRAP mutagenesis screen (<a href="http://fishtrap.warwick.ac.uk">http://fishtrap.warwick.ac.uk</a>, <a id="xref-ref-49-5" class="xref-bibr" href="#ref-49">Quach <em>et al.</em> 2015</a>). These stable transgenic lines were selected on the basis of expression of fluorescent reporters (mCherry and/or GFP) during the first 7 d of development and represent protein/enhancer trap events. To identify the Ds insertion sites, we first performed thermal asymmetric interlaced PCR (TAIL-PCR) on 310 reporter positive fish lines generating 385 unique insertion sites. TAIL-PCR results represented only ∼75% of sites estimated by Southern blot analysis (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026849/-/DC1/FigureS1.pdf">Figure S1</a>), so we performed NGS on 106 of these lines and 114 additional lines, obtaining a total of 1685 unique high-confidence Ds integration sites from 424 zebrafish lines.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/805/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Experimental design. (A) Pipeline for obtaining high-confidence selected and unselected Ds integration sets. Ds integrations were generated by injection of Ds trapping plasmid together with Ac transposase capped-RNA at 1-cell stage. Following injection, larvae were collected at 3&#x2013;4 days post fertilization (dpf) without selection to obtain a set of unbiased Ds integrations, or raised to adulthood, outcrossed, and selected for reporter gene expression. Ds integrations were analyzed by thermal asymmetric interlaced PCR (TAIL-PCR) and genomic sequencing. Numbers in parenthesis (i.e., 310, 220 and 576) represent the number of fish lines or injected embryos from which the Ds integration sites were identified. (B) Data analysis scheme. Selected and unselected Ds integrations were compared with Tol2 and MMLV (Moloney Murine Leukemia Virus) sites. One thousand controls were generated for each integration dataset. RE, restriction endonuclease; Zv9, zebrafish reference genome. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-53474475" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Experimental design. (A) Pipeline for obtaining high-confidence selected and unselected Ds integration sets. Ds integrations were generated by injection of Ds trapping plasmid together with Ac transposase capped-RNA at 1-cell stage. Following injection, larvae were collected at 3&#x2013;4 days post fertilization (dpf) without selection to obtain a set of unbiased Ds integrations, or raised to adulthood, outcrossed, and selected for reporter gene expression. Ds integrations were analyzed by thermal asymmetric interlaced PCR (TAIL-PCR) and genomic sequencing. Numbers in parenthesis (&lt;em&gt;i.e.&lt;/em&gt;, 310, 220 and 576) represent the number of fish lines or injected embryos from which the Ds integration sites were identified. (B) Data analysis scheme. Selected and unselected Ds integrations were compared with Tol2 and MMLV (Moloney Murine Leukemia Virus) sites. One thousand controls were generated for each integration dataset. RE, restriction endonuclease; Zv9, zebrafish reference genome. &lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/805/F1.medium.gif" width="296" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/4/805/F1.medium.gif" width="296" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/805/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/805/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467890" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-26" class="first-child">Experimental design. (A) Pipeline for obtaining high-confidence selected and unselected Ds integration sets. Ds integrations were generated by injection of Ds trapping plasmid together with Ac transposase capped-RNA at 1-cell stage. Following injection, larvae were collected at 3–4 days post fertilization (dpf) without selection to obtain a set of unbiased Ds integrations, or raised to adulthood, outcrossed, and selected for reporter gene expression. Ds integrations were analyzed by thermal asymmetric interlaced PCR (TAIL-PCR) and genomic sequencing. Numbers in parenthesis (<em>i.e.</em>, 310, 220 and 576) represent the number of fish lines or injected embryos from which the Ds integration sites were identified. (B) Data analysis scheme. Selected and unselected Ds integrations were compared with Tol2 and MMLV (Moloney Murine Leukemia Virus) sites. One thousand controls were generated for each integration dataset. RE, restriction endonuclease; Zv9, zebrafish reference genome. </p><div class="sb-div caption-clear"></div></div></div><p id="p-27">Remarkably, only ∼45% of the TAIL-identified flanking sites were also identified by NGS. Both TAIL-PCR and NGS rely on unambiguous mappings to the genome assembly for integration site identification. However, we found that TAIL-PCR produced longer flanking site sequences and was more accommodating of mismatches between the Tubingen-strain reference genome and the AB strain used in the Ac/Ds mutagenesis screen (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Howe <em>et al.</em> 2013</a>; <a id="xref-ref-49-6" class="xref-bibr" href="#ref-49">Quach <em>et al.</em> 2015</a>). In fact, 133 out of the 385 inserts identified by TAIL-PCR differed substantially from the genome assembly and were not mapped by NGS, explaining the limited overlap between the two techniques. In addition, some previously identified TAIL-PCR sites were detected at very low levels in the NGS results (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026849/-/DC1/FigureS2.pdf">Figure S2</a>), suggesting that different flanking sequences might be more efficiently identified by the two techniques.</p><p id="p-28">Since each selected fish line contains on average 3–4 insertions, some of which are likely unrelated to the fluorescent reporter expression pattern used to identify the line, we expect the selection bias to be mitigated in our dataset. However, these selected Ds inserts would still have to be incorporated into the germ-line for stable transmission. Therefore, in order to obtain an unbiased set of Ds integration sites, we performed NGS on 576 embryos that had been injected with the Ds plasmid and Ac transposase mRNA, but not selected on the basis of transgene reporter expression. In total, we obtained a set of 1344 high-confidence integrations which we used as our “unselected” Ds set. These two Ds insertion sets were compared with 15,223 unselected MMLV retroviral insertions (<a id="xref-ref-61-4" class="xref-bibr" href="#ref-61">Varshney <em>et al.</em> 2013</a>) (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), and analyzed together with appropriately matched control sets of the same size and mapping characteristics. Although limited by the smaller sample size and various selection strategies, we also examined the distribution of 379 Tol2 integrations combined from two published enhancer and protein trap datasets (which we henceforth refer to as the “selected Tol2” set) (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Kawakami <em>et al.</em> 2010</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Kondrychyn <em>et al.</em> 2011</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467922/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467922/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Integration datasets analyzed in this work</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Ds integration sites are distributed throughout the zebrafish genome</h3><p id="p-30">To investigate integration site preferences, we first examined their distribution across the zebrafish genome at the chromosome level (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). MMLV, selected Tol2, and selected and unselected Ds integrations were found distributed across all 25 chromosomes. Consistent with some of the Tol2 integration sites having been created by remobilization of existing genomic integration sites, we observed enrichment of Tol2 sites close to donor locations in chromosomes 14 and 24 (<a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Kondrychyn <em>et al.</em> 2011</a>). Although some chromosomal regions appeared to be either over or underrepresented when compared to matched controls, in general, Ds, Tol2, and MMLV integrations were all found widely distributed across the genome, and we did not observe integration rich regions shared across any of the datasets.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/805/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Insert distribution across the zebrafish genome. Ds, Tol2, and MMLV (Moloney Murine Leukemia Virus) integration sites were distributed across all chromosomes, with regions of relative over and underrepresentation. Solid bars represent integration sites. Open bars represent 20,000 matched control sites." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-53474475" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Insert distribution across the zebrafish genome. Ds, Tol2, and MMLV (Moloney Murine Leukemia Virus) integration sites were distributed across all chromosomes, with regions of relative over and underrepresentation. Solid bars represent integration sites. Open bars represent 20,000 matched control sites.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/805/F2.medium.gif" width="440" height="326"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/4/805/F2.medium.gif" width="440" height="326"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/805/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/805/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467896" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-31" class="first-child">Insert distribution across the zebrafish genome. Ds, Tol2, and MMLV (Moloney Murine Leukemia Virus) integration sites were distributed across all chromosomes, with regions of relative over and underrepresentation. Solid bars represent integration sites. Open bars represent 20,000 matched control sites.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">To explore if specific features of the genome were correlated with integration events we analyzed the overlap of 25 genomic features with the integration datasets (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026849/-/DC1/TableS2.xlsx">Table S2</a>). A plot of the statistically significant enriched and depleted genomic features is presented in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> (with the full results shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026849/-/DC1/TableS3.xlsx">Table S3</a>). In the following sections we discuss the main findings.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/805/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary of genomic features analyzed. Genomic features analyzed for (A) Ds selected, (B) Ds unselected, (C) Tol2, and (D) MMLV (Moloney Murine Leukemia Virus). Average fold enrichment values (representing 1000 ratios of experimental over match controls) plotted on the y-axis. P-values plotted on the x-axis. Statistically significant enriched/depleted features with P-value &lt; 0.05 are presented (detailed results are provided in Table S3)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-53474475" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Summary of genomic features analyzed. Genomic features analyzed for (A) Ds selected, (B) Ds unselected, (C) Tol2, and (D) MMLV (Moloney Murine Leukemia Virus). Average fold enrichment values (representing 1000 ratios of experimental over match controls) plotted on the y-axis. &lt;em&gt;P&lt;/em&gt;-values plotted on the x-axis. Statistically significant enriched/depleted features with &lt;em&gt;P&lt;/em&gt;-value &lt; 0.05 are presented (detailed results are provided in Table S3).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/805/F3.medium.gif" width="440" height="421"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/4/805/F3.medium.gif" width="440" height="421"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/805/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/805/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467940" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-33" class="first-child">Summary of genomic features analyzed. Genomic features analyzed for (A) Ds selected, (B) Ds unselected, (C) Tol2, and (D) MMLV (Moloney Murine Leukemia Virus). Average fold enrichment values (representing 1000 ratios of experimental over match controls) plotted on the y-axis. <em>P</em>-values plotted on the x-axis. Statistically significant enriched/depleted features with <em>P</em>-value &lt; 0.05 are presented (detailed results are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026849/-/DC1/TableS3.xlsx">Table S3</a>).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Ds insertions show preference toward gene regions</h3><p id="p-34">We first examined whether Ds, Tol2, and MMLV integrated preferentially in gene regions. We obtained Ensembl gene prediction coordinates and identified gene features such as introns, exons, and UTRs. Any integration occurring beyond 5 kb of the transcriptional start and termination sites was deemed intergenic. We found that Ds, Tol2, and MMLV integrations were enriched around gene regions, but with differential preference for specific gene features (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). Specifically, Ds integrations showed a preference for coding exons, whereas Tol2 and MMLV preferentially integrated in the 5′-UTR and regions up-stream of the TSS. In general, we observed no differences in the preference of selected and unselected Ds integrations for intronic, 5′-UTR, 3′-UTR, and intergenic regions. Detailed examination of sites that overlapped gene regions showed that while MMLV and Tol2 integrations are biased toward the first 10% of the gene area, Ds integration sites are more broadly distributed throughout gene regions (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/805/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of integration sites across gene regions. (A) Fold enrichment values for various gene subregions. Average enrichment &#xB1; standard error (n = 1000). Ds integrations show preference for coding and 5&#x2032; gene regions. Tol2 and MMLV (Moloney Murine Leukemia Virus) integrations show preference for 5&#x2032; regions of genes. (B) Ds integrations are distributed uniformly across the length of gene regions, while Tol2 and MMLV are enriched toward the first 10% of genes. (C) Ds, Tol2, and MMLV show enrichment close to transcription start site (TSS), but are not enriched around the transcription termination site (TTS) (D). Solid blue bars represent integration sites. Open red bars represent average of 1000 matched controls &#xB1; standard deviation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-53474475" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of integration sites across gene regions. (A) Fold enrichment values for various gene subregions. Average enrichment &#xB1; standard error (n = 1000). Ds integrations show preference for coding and 5&#x2032; gene regions. Tol2 and MMLV (Moloney Murine Leukemia Virus) integrations show preference for 5&#x2032; regions of genes. (B) Ds integrations are distributed uniformly across the length of gene regions, while Tol2 and MMLV are enriched toward the first 10% of genes. (C) Ds, Tol2, and MMLV show enrichment close to transcription start site (TSS), but are not enriched around the transcription termination site (TTS) (D). Solid blue bars represent integration sites. Open red bars represent average of 1000 matched controls &#xB1; standard deviation.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/805/F4.medium.gif" width="395" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/4/805/F4.medium.gif" width="395" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/805/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/805/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467791" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-35" class="first-child">Distribution of integration sites across gene regions. (A) Fold enrichment values for various gene subregions. Average enrichment ± standard error (n = 1000). Ds integrations show preference for coding and 5′ gene regions. Tol2 and MMLV (Moloney Murine Leukemia Virus) integrations show preference for 5′ regions of genes. (B) Ds integrations are distributed uniformly across the length of gene regions, while Tol2 and MMLV are enriched toward the first 10% of genes. (C) Ds, Tol2, and MMLV show enrichment close to transcription start site (TSS), but are not enriched around the transcription termination site (TTS) (D). Solid blue bars represent integration sites. Open red bars represent average of 1000 matched controls ± standard deviation.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">Because of the preference of Tol2 and MMLV for the 5′ region of genes, we examined the distribution of integrations around the TSS (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>). MMLV integrations were significantly biased downstream of the TSS, consistent with previous observations in human cells that identified MMLV integrations going into enhancers within the 1<sup>st</sup> intron (<a id="xref-ref-68-3" class="xref-bibr" href="#ref-68">Wu <em>et al.</em> 2003</a>; <a id="xref-ref-36-6" class="xref-bibr" href="#ref-36">LaFave <em>et al.</em> 2014</a>). In contrast, both Tol2 and Ds integrations show a symmetrical distribution around the TSS, although the number of transposon integrations was smaller than those for MMLV. We found no similarity in integration patterns around the TTS for Ds, Tol2, or MMLV. Taken together, we found that Ds, Tol2, and MMLV integrations have a preference for gene regions. Ds integrations were found more broadly distributed along genes, unlike Tol2 and MMLV, which showed a significant bias toward the 5′ region of genes.</p></div><div id="sec-14" class="subsection"><h3>Integrations are correlated with measures of gene and enhancer activity</h3><p id="p-37">Next, we asked whether the genes targeted by transposon and retroviral integrations shared any common characteristics. For this analysis, integrations were assigned to a particular gene if they occurred within 5 kb of their TSS and TTS. Although a few genes were targeted by more than one type of integration, these tended to span large regions of the genome. Gene ontology (GO) analysis on genes targeted by Ds integrations did not reveal any significant enriched categories (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026849/-/DC1/TableS4.xlsx">Table S4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026849/-/DC1/TableS5.xlsx">Table S5</a>). In contrast, a number of Tol2 integrations targeted Hox genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026849/-/DC1/TableS6.xlsx">Table S6</a>), specifically hoxa5a, hoxa3a, hoxd3a, hoxc5a, hoxd4a, and hoxc3a (GO analysis, Benjamini <em>P</em>-value = 0.015). Since these genes are in chromosomes 9, 19, and 23, this preference cannot be accounted for by local hopping from donor sites on chromosomes 14 and 24. Some genes were targeted two or more times by MMLV integrations. These genes were significantly enriched (Benjamini <em>P</em>-value &lt; 0.001) for GO categories representing biological processes occurring during gastrulation, such as cell migration, regulation of transcription, and embryonic morphogenesis (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026849/-/DC1/TableS7.xlsx">Table S7</a>).</p><p id="p-38">To test if integration was correlated with gene activity, we calculated the gene expression levels of the genes targeted by integrations (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026849/-/DC1/FigureS3.pdf">Figure S3</a>). We measured the median expression level of genes across various time-points as well as an overall expression level (the sum of the individual expression) using published RNA-seq data (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Harvey <em>et al.</em> 2013</a>). We noticed that, on average, genes targeted by Tol2 and MMLV tended to have higher expression in the early embryo (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026849/-/DC1/FigureS3.pdf">Figure S3</a>). In contrast, genes targeted by Ds did not show any gene expression difference from matched controls.</p><p id="p-39">We also analyzed whether these genes shared any particular pattern of expression, categorizing the genes as having maternal only expression, zygotic only expression, or both maternal and zygotic expression (maternal-zygotic) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026849/-/DC1/FigureS3.pdf">Figure S3</a>). Ds integrated similarly in genes with each of these expression patterns. In contrast, Tol2 and MMLV integrated preferentially into zygotically expressed genes, and not maternal genes. Since MMLV injections were performed at the 1000 to 2000 cell stage and after zygotic genome activation, our GO and gene expression analyses are consistent with MMLV preferentially integrating into actively transcribed genes from the onset of zygotic genome activation.</p><p id="p-40">Epigenetic marks involved in gene regulation, such as histone modification and differential DNA methylation, might facilitate integration into specific gene regions (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Potok <em>et al.</em> 2013</a>). Although the overall frequency of integrations within CpG islands was low, we found an increased preference of all integrations toward CpG islands both within and outside promoters, suggesting that regulated regions are preferred regardless of the integration vector used (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026849/-/DC1/FigureS4.pdf">Figure S4</a>). Similarly, while DNA methylation status was not a key determinant of integration preference, regions of higher CpG DNA methylation at 1-cell and midblastula transition (MBT) stages of development showed a twofold increase in Ds integrations compared to matched controls (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026849/-/DC1/FigureS4.pdf">Figure S4</a>). This correlation was found for DNA methylation both within and outside gene regions.</p><p id="p-41">Next, we examined whether integrations overlap with chromatin modifications marking active promoter and enhancer elements (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026849/-/DC1/FigureS4.pdf">Figure S4</a>). All insertion sites, including those that do not overlap with gene regions, preferentially targeted regions rich in chromatin modifications associated with active promoters and enhancers (tri and monomethylation of Histone 3 Lysine 4) suggesting that integration sites in intergenic regions might represent unannotated genes or novel enhancer elements.</p></div><div id="sec-15" class="subsection"><h3>Integrations in repetitive sequences</h3><p id="p-42">Repetitive elements, which account for 52.2% of the zebrafish genome (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Howe <em>et al.</em> 2013</a>), have been shown to have roles in chromosome structural organization, gene regulation, genome integrity, and evolution (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kidwell and Lisch 2000</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lander <em>et al.</em> 2001</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Waterston <em>et al.</em> 2002</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Feschotte 2008</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Ting <em>et al.</em> 2011</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Zhu <em>et al.</em> 2011</a>). Therefore, we examined the integration preferences for the various repetitive element families, such as DNA transposons, LINEs, and SINEs, present in the zebrafish genome (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026849/-/DC1/FigureS5.pdf">Figure S5</a>). DNA transposon sequences and low-complexity repeats were consistently underrepresented in sites of integration. However, we observed varying preferences for the other types of repetitive elements. Specifically, we noticed that LTR sequences were overrepresented in MMLV integration sites, while SINE, LINE, and simple repeats were underrepresented. In contrast, we observed a weak overrepresentation of Ds integration sites overlapping SINE and LTR elements, but no significant over or underrepresentation of sites overlapping simple repeats.</p></div><div id="sec-16" class="subsection"><h3>Ds and Tol2 target sites do not show a strict sequence motif</h3><p id="p-43">We then examined whether any features at the sequence level could help predict Ds, Tol2, and MMLV integrations in the zebrafish genome. In contrast to the strong binding site preference for the element ends, no strong target site consensus sequences have been identified for Ds and Tol2. However, a weak preference for specific nucleotides at the target site has been reported. Previous analysis of Ds integrations in maize suggested the presence of a weak palindromic consensus sequence at the target site (<a id="xref-ref-63-4" class="xref-bibr" href="#ref-63">Vollbrecht <em>et al.</em> 2010</a>), while Tol2 integrations in zebrafish suggested the presence of a TNA(C/G)TTATAA(G/C)TNA motif (<a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Kondrychyn <em>et al.</em> 2009</a>). Therefore, we searched for a consensus sequence at the target site in our Ds data and in the MMLV dataset (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/805/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Target sites show structural features even in the absence of a strict motif. (A) Graphical representations of nucleic acid multiple sequence alignment were generated with WebLogo v3.4 (Crooks et al. 2004). Ds and Tol2 integrations show weak preference for specific nucleotides at the integration site. Numbers on top indicate nucleotide position around the integration site shown on the x-axis. Information measured in bits and probability is shown on the y-axis. (B) Average values of protein-DNA movement for each position in the multiple sequence alignment plotted according to their values. Numbers on top indicate position around the integration site shown on the x-axis. Red lines represent integration site data. Blue lines denote the average of matched controls (n = 1000). MMLV, Moloney Murine Leukemia Virus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-53474475" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Target sites show structural features even in the absence of a strict motif. (A) Graphical representations of nucleic acid multiple sequence alignment were generated with WebLogo v3.4 (Crooks &lt;em&gt;et al.&lt;/em&gt; 2004). Ds and Tol2 integrations show weak preference for specific nucleotides at the integration site. Numbers on top indicate nucleotide position around the integration site shown on the x-axis. Information measured in bits and probability is shown on the y-axis. (B) Average values of protein-DNA movement for each position in the multiple sequence alignment plotted according to their values. Numbers on top indicate position around the integration site shown on the x-axis. Red lines represent integration site data. Blue lines denote the average of matched controls (n = 1000). MMLV, Moloney Murine Leukemia Virus.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/805/F5.medium.gif" width="440" height="325"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/4/805/F5.medium.gif" width="440" height="325"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/805/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/805/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467849" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-44" class="first-child">Target sites show structural features even in the absence of a strict motif. (A) Graphical representations of nucleic acid multiple sequence alignment were generated with WebLogo v3.4 (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Crooks <em>et al.</em> 2004</a>). Ds and Tol2 integrations show weak preference for specific nucleotides at the integration site. Numbers on top indicate nucleotide position around the integration site shown on the x-axis. Information measured in bits and probability is shown on the y-axis. (B) Average values of protein-DNA movement for each position in the multiple sequence alignment plotted according to their values. Numbers on top indicate position around the integration site shown on the x-axis. Red lines represent integration site data. Blue lines denote the average of matched controls (n = 1000). MMLV, Moloney Murine Leukemia Virus.</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">Consistent with previous reports, MMLV integrates preferentially in AT-rich regions (<a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Wu <em>et al.</em> 2005</a>; <a id="xref-ref-36-7" class="xref-bibr" href="#ref-36">LaFave <em>et al.</em> 2014</a>), and we also observed a region of relative AT depletion following the integration site. In our analyses, we were able to detect the weak consensus sequence previously reported for Tol2 (<a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">Kondrychyn <em>et al.</em> 2009</a>). We did not observe the weak sequence motif for Ds integration sites previously observed in maize (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). However, when insertion site sequences were aggregated, we noticed a weak palindromic motif spanning 14 bases around the Ds insertion site with consensus similar to Tol2. This sequence is seldom found within the dataset target sequences <em>per se</em>, and appears only when insertion sites are aggregated.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/805/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ds integration site analysis in maize. 1826 Ds integration sites were obtained from published datasets (Vollbrecht et al. 2010) and compared against random integration sites. (A) Sequence logo for Ds integrations. (B) Structural features of DNA at integration sites." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-53474475" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Ds integration site analysis in maize. 1826 Ds integration sites were obtained from published datasets (Vollbrecht &lt;em&gt;et al.&lt;/em&gt; 2010) and compared against random integration sites. (A) Sequence logo for Ds integrations. (B) Structural features of DNA at integration sites.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/805/F6.medium.gif" width="371" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/4/805/F6.medium.gif" width="371" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/805/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/805/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467968" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-46" class="first-child">Ds integration site analysis in maize. 1826 Ds integration sites were obtained from published datasets (<a id="xref-ref-63-5" class="xref-bibr" href="#ref-63">Vollbrecht <em>et al.</em> 2010</a>) and compared against random integration sites. (A) Sequence logo for Ds integrations. (B) Structural features of DNA at integration sites.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Transposon and retroviral integration sites show similar structural features</h3><p id="p-47">Local interactions between adjacent nucleotides can induce distortions in the regular double helix structure (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Olson <em>et al.</em> 1998</a>). To test whether integration sites were more likely to be deformed by protein-DNA interactions, we used data from protein-DNA complexes to calculate six structural features of DNA: Rise, Roll, Shift, Slide, Tilt, and Twist. For example, protein DNA-twist predicts the twist angle torsion between adjacent bases, so that a dinucleotide pair with a high value of protein-DNA twist is more likely to be deformed by protein–DNA interaction than one with a lower value. For transposon integrations, we observed significant changes from normal for these six features arranged in a symmetrical pattern around the target sites (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). For retroviral integrations, the outlying values extended a few bases downstream of the target site. Regardless of the vector used, all integrations appear to fall in regions of higher DNA flexibility.</p><p id="p-48">To determine if this feature is specific to integrations in zebrafish or whether it is found in other vertebrate genomes, we analyzed previously reported integration sites for Sleeping beauty and piggyBac transposons in mouse ESCs, and mouse mammary tumor retrovirus (MMTV) in mouse mammary cells (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">de Jong <em>et al.</em> 2014</a>) (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>). Sleeping beauty was found to target TATA sequences and piggyBac insertions fell in AT-rich regions. In contrast, no consensus sequence motif was observed for MMTV retroviral insertions. Nonetheless, all integrations occurred at regions of high DNA flexibility (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7B</a>). Taken together, our findings suggest that structural features in the target DNA are common in transposon and retroviral integration sites in vertebrate genomes, and can potentially be used to enhance the efficiency of genome engineering by these and other methods.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/805/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Measures of DNA flexibility show similar features in transposon and retroviral integration sites in mouse cells. (A) Sequence logo for various integrations in mouse cells show an obvious common motif. Numbers on top indicate position around the integration site shown on the x-axis. Information measured in bits and probability shown on y-axis. (B) Structural features of DNA at integration sites. Average DNA flexibility values shown in y-axes. Nucleotide position shown on x-axis. MMTV, Mouse Mammary Tumor Virus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-53474475" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Measures of DNA flexibility show similar features in transposon and retroviral integration sites in mouse cells. (A) Sequence logo for various integrations in mouse cells show an obvious common motif. Numbers on top indicate position around the integration site shown on the x-axis. Information measured in bits and probability shown on y-axis. (B) Structural features of DNA at integration sites. Average DNA flexibility values shown in y-axes. Nucleotide position shown on x-axis. MMTV, Mouse Mammary Tumor Virus.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/805/F7.medium.gif" width="440" height="261"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/6/4/805/F7.medium.gif" width="440" height="261"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/805/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/805/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467942" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-49" class="first-child">Measures of DNA flexibility show similar features in transposon and retroviral integration sites in mouse cells. (A) Sequence logo for various integrations in mouse cells show an obvious common motif. Numbers on top indicate position around the integration site shown on the x-axis. Information measured in bits and probability shown on y-axis. (B) Structural features of DNA at integration sites. Average DNA flexibility values shown in y-axes. Nucleotide position shown on x-axis. MMTV, Mouse Mammary Tumor Virus.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-50">Transgenesis is a powerful tool which, coupled to new genome editing techniques, continues to make zebrafish an excellent model organism in which to perform functional genomic studies. In this study, we set out to compare the integration preferences of three popular tools for generating transgenics (MMLV retrovirus, and Ds and Tol2 transposons). We used a combination of TAIL-PCR and NGS to detect Ds integration sites. Only ∼45% of Ds sites identified by TAIL-PCR were captured by NGS. Many of the Ds integration sites that were not detected by genomic sequencing showed variation from the genome assembly sequence, suggesting they were not mapped under the parameters used. Ds sites showed a wide range of detection efficiency as measured by average fragment counts produced. Differences in the distance of the insertion site to the restriction enzyme cut site used during the NGS protocol, or the efficiency of PCR amplification, could potentially explain the different isolation efficiencies observed.</p><p id="p-51">The use of NGS for mapping made it possible to identify integrations in the absence of germ-line transmission or reporter expression. We found selected and unselected Ds sites to have remarkably similar integration characteristics. Because each selected fish line contains ∼3–4 insertions, it is possible that some of the selection bias could have been mitigated in our Ds “selected” dataset. However, we did not observe a significant difference in genomic feature overlap between selected lines harboring one <em>vs.</em> multiple insertions either. Therefore, the similarity between selected and unselected insertions is unlikely to be explained by the number of insertions per line alone (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026849/-/DC1/FigureS6.pdf">Figure S6</a>).</p><p id="p-52">Despite the small sample size of Tol2 integrations and different selection strategies in the various screens, we found that all vectors showed a preference for gene regions. Tol2 and MMLV were largely concentrated around the 5′ regions of highly expressed genes. In contrast, we found that Ds sites were enriched in coding regions, and broadly distributed along gene regions, matching regions of high DNA methylation outside of promoters. Significantly, Ds integration preference for gene regions was found even in the absence of selection.</p><p id="p-53">The zebrafish genome shows an overall repeat content of 52.2%, the highest reported so far in a vertebrate (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Howe <em>et al.</em> 2013</a>). Consistent with this, integrations frequently overlapped repetitive elements. Analysis of repetitive elements in the zebrafish genome revealed that LTRs, low complexity, and simple repeats are more likely found in coding gene regions, while SINE elements are more likely present in 5′-UTRs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026849/-/DC1/FigureS7.pdf">Figure S7</a>). Our matched control sets have similar mapping characteristics to the experimental integration sets, therefore differences in mapping cannot account for the relative depletion of low complexity repeats within integration sites. The preferential integration into specific repetitive elements could represent a preference for their specific underlying sequences or the resulting structural characteristics. Alternatively, other genomic characteristics could be correlated with the different repetitive elements. Repetitive sequences show differential methylation and activity in zebrafish (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">McGaughey <em>et al.</em> 2014</a>). DNA repeats have also been shown to be transcribed, and have been suggested to provide regulatory elements to protein-coding genes (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Wang <em>et al.</em> 2007b</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bourque <em>et al.</em> 2008</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Faulkner <em>et al.</em> 2009</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Tyekucheva <em>et al.</em> 2011</a>). Moreover, binding sites for important regulatory factors such as CTCF or TP53 are often associated with genomic repeats (<a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Wang <em>et al.</em> 2007b</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bourque <em>et al.</em> 2008</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chadwick 2008</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Simeonova <em>et al.</em> 2012</a>).</p><p id="p-54">An important question concerns the presence of particular insertion hotspots, since integration can cause adverse events such as activation of proto-oncogenes or inactivation of essential cellular genes. Both Tol2 and MMLV showed enrichment for specific GO categories. Ds integrations showed no observable correlation with specific gene types. While no single transgenic tool will be equally suited for every experimental inquiry, our analyses should help in the choice of transgenic system for interrogating gene function.</p><p id="p-55">At the sequence level, both Ds and Tol2 target sites shared a similar weak motif that appeared only when sequences were aggregated. The weak motif likely reflects structural features of the target DNA. Current genome editing methods rely exclusively upon nucleotide sequence for selection of targeting sites (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Lim <em>et al.</em> 2013</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Irion <em>et al.</em> 2014</a>). Our analysis of known transposon and retroviral integration sites in mouse ES cells and in zebrafish shows that regions of higher DNA flexibility are preferred for integrations of exogenous sequences. Thus, structural features in DNA influence the site of insertion in vertebrate genomes. This feature can potentially be used in combination with sequence information to enhance the efficiency of genome editing, and to improve precision engineering at desired locations within genomes.</p><p id="p-56">We found the presence of DNA flexibility features to be conserved among different types of integrations and in different species. However, the specific features differ from system to system, likely resulting from differences in their mechanisms of integration, or the presence of different cofactors. The ability of hAT transposons to function in diverse species suggests that they might not require specific cofactors, or rely on very highly conserved cofactors (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Weil and Kunze 2000</a>; <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Emelyanov <em>et al.</em> 2006</a>). In contrast, several groups have identified bromodomain and extraterminal (BET) proteins as the major host factors that specifically interact with MMLV integrase and mediate the preferential integration of MMLV near TSS (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Studamire and Goff 2008</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">De Rijck <em>et al.</em> 2013</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gupta <em>et al.</em> 2013</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Sharma <em>et al.</em> 2013</a>).</p><p id="p-57">In summary, our genome-wide analysis shows that Ds integration sites in the presence or absence of selection are remarkably similar and can be found across the genome. A strict motif associated with target site was not found, but a preference for structural features in the target DNA was observed. Remarkably, this feature is also found in transposon and retroviral integrations in maize and mouse cells. Our findings show that structural features influence the integration of heterologous DNA in vertebrate genomes, and can facilitate efficient targeted genome engineering.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-58">We thank the Sampath laboratory for discussions and suggestions; H. Woodland and L. Sivashanmugam for suggestions to improve the manuscript. G.K.V., M.C.L., and S.M.B. were supported by the Intramural Research Program of the National Human Genome Research Institute, National Institutes of Health. K.S. and P.V. are supported by Warwick Medical School.</p><p id="p-59">Author contributions: DNA sample preparation and sperm cryopreservation was performed by H.N.B.Q., S.T., P.V., and K.S. TAIL sequencing was performed by H.N.B.Q., S.T., and A.J. Next-generation sequencing was performed by G.K.V. and S.M.B. P.V., M.C.L., S.M.B., and K.S. analyzed the data. P.V. and K.S. wrote the manuscript with input from all authors. The authors declare no conflict of interest.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-3"><p id="p-60">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026849/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026849/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-61"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-11-23"><span class="received-label">Received </span>November 23, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-01-19"><span class="accepted-label">Accepted </span>January 19, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Vrljicak <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.1" data-doi="10.1016/j.ydbio.2011.03.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aday</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lakshmanan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawson</span>  <span class="cit-name-given-names">N. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Identification of cis regulatory features in the embryonic zebrafish genome through large-scale profiling of H3K4me1 and H3K4me3 binding sites.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">357</span>: <span class="cit-fpage">450</span>–<span class="cit-lpage">462</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DAday%26rft.auinit1%253DA.%2BW.%26rft.volume%253D357%26rft.issue%253D2%26rft.spage%253D450%26rft.epage%253D462%26rft.atitle%253DIdentification%2Bof%2Bcis%2Bregulatory%2Bfeatures%2Bin%2Bthe%2Bembryonic%2Bzebrafish%2Bgenome%2Bthrough%2Blarge-scale%2Bprofiling%2Bof%2BH3K4me1%2Band%2BH3K4me3%2Bbinding%2Bsites.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2011.03.007%26rft_id%253Dinfo%253Apmid%252F21435340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2011.03.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21435340&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.2" data-doi="10.1126/science.1070104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aiuti</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slavin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aker</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ficara</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deola</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Correction of ADA-SCID by stem cell gene therapy combined with nonmyeloablative conditioning.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">296</span>: <span class="cit-fpage">2410</span>–<span class="cit-lpage">2413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAiuti%26rft.auinit1%253DA.%26rft.volume%253D296%26rft.issue%253D5577%26rft.spage%253D2410%26rft.epage%253D2413%26rft.atitle%253DCorrection%2Bof%2BADA-SCID%2Bby%2BStem%2BCell%2BGene%2BTherapy%2BCombined%2Bwith%2BNonmyeloablative%2BConditioning%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1070104%26rft_id%253Dinfo%253Apmid%252F12089448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTYvNTU3Ny8yNDEwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80LzgwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.3" data-doi="10.1016/B978-0-12-374814-0.00004-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amsterdam</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varshney</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgess</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Retroviral-mediated insertional mutagenesis in zebrafish.</span> <abbr class="cit-jnl-abbrev">Methods Cell Biol.</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bcell%2Bbiology%26rft.stitle%253DMethods%2BCell%2BBiol%26rft.aulast%253DAmsterdam%26rft.auinit1%253DA.%26rft.volume%253D104%26rft.spage%253D59%26rft.epage%253D82%26rft.atitle%253DRetroviral-mediated%2BInsertional%2BMutagenesis%2Bin%2BZebrafish.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-374814-0.00004-5%26rft_id%253Dinfo%253Apmid%252F21924157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-374814-0.00004-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21924157&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295551800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.4" data-doi="10.1101/gr.080663.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bourque</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leong</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vega</span>  <span class="cit-name-given-names">V. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Evolution of the mammalian transcription factor binding repertoire via transposable elements.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1752</span>–<span class="cit-lpage">1762</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.080663.108%26rft_id%253Dinfo%253Apmid%252F18682548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMS8xNzUyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80LzgwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.5" data-doi="10.1016/0092-8674(81)90010-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calvi</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Findley</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gelbart</span>  <span class="cit-name-given-names">W. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Evidence for a common evolutionary origin of inverted repeat transposons in Drosophila and plants: hobo, Activator, and Tam3.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">465</span>–<span class="cit-lpage">471</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DCalvi%26rft.auinit1%253DB.%2BR.%26rft.volume%253D66%26rft.issue%253D3%26rft.spage%253D465%26rft.epage%253D471%26rft.atitle%253DEvidence%2Bfor%2Ba%2Bcommon%2Bevolutionary%2Borigin%2Bof%2Binverted%2Brepeat%2Btransposons%2Bin%2BDrosophila%2Band%2Bplants%253A%2Bhobo%252C%2BActivator%252C%2Band%2BTam3.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252881%252990010-6%26rft_id%253Dinfo%253Apmid%252F1651170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(81)90010-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1651170&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991GA94100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.6" data-doi="10.1126/science.288.5466.669"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cavazzana-Calvo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hacein-Bey</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Saint Basile</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yvon</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Gene therapy of human severe combined immunodeficiency (SCID)-X1 disease.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">288</span>: <span class="cit-fpage">669</span>–<span class="cit-lpage">672</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCavazzana-Calvo%26rft.auinit1%253DM.%26rft.volume%253D288%26rft.issue%253D5466%26rft.spage%253D669%26rft.epage%253D672%26rft.atitle%253DGene%2BTherapy%2Bof%2BHuman%2BSevere%2BCombined%2BImmunodeficiency%2B%2528SCID%2529-X1%2BDisease%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.288.5466.669%26rft_id%253Dinfo%253Apmid%252F10784449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODgvNTQ2Ni82NjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvODA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.7" data-doi="10.1101/gr.075713.107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chadwick</span>  <span class="cit-name-given-names">B. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">DXZ4 chromatin adopts an opposing conformation to that of the surrounding chromosome and acquires a novel inactive X-specific role involving CTCF and antisense transcripts.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1259</span>–<span class="cit-lpage">1269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.075713.107%26rft_id%253Dinfo%253Apmid%252F18456864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE4LzgvMTI1OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC84MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.8" data-doi="10.1016/j.gene.2013.11.084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">X. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">Y. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The goldfish hAT-family transposon Tgf2 is capable of autonomous excision in zebrafish embryos.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">536</span>: <span class="cit-fpage">74</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D536%26rft.spage%253D74%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2013.11.084%26rft_id%253Dinfo%253Apmid%252F24321692%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2013.11.084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24321692&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.9" data-doi="10.1101/gr.849004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crooks</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hon</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chandonia</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brenner</span>  <span class="cit-name-given-names">S. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">WebLogo: a sequence logo generator.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1188</span>–<span class="cit-lpage">1190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DCrooks%26rft.auinit1%253DG.%2BE.%26rft.volume%253D14%26rft.issue%253D6%26rft.spage%253D1188%26rft.epage%253D1190%26rft.atitle%253DWebLogo%253A%2BA%2BSequence%2BLogo%2BGenerator%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.849004%26rft_id%253Dinfo%253Apmid%252F15173120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE0LzYvMTE4OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC84MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.10" data-doi="10.1371/journal.pgen.1004994"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davie</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobs</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atkins</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Potier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christiaens</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Discovery of transcription factors and regulatory regions driving in vivo tumor development by ATAC-seq and FAIRE-seq open chromatin profiling.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1004994</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1004994%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004994%26rft_id%253Dinfo%253Apmid%252F25679813%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004994&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25679813&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.11" data-doi="10.1371/journal.pgen.1004250"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Jong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akhtar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Badhai</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rust</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rad</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Chromatin landscapes of retroviral and transposon integration profiles.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004250%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004250%26rft_id%253Dinfo%253Apmid%252F24721906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004250&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24721906&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.12" data-doi="10.1128/JVI.00011-14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Ravin</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Theobald</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macpherson</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Enhancers are major targets for murine leukemia virus vector integration.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">4504</span>–<span class="cit-lpage">4513</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00011-14%26rft_id%253Dinfo%253Apmid%252F24501411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg4LzgvNDUwNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC84MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.13" data-doi="10.1016/j.celrep.2013.09.040"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Rijck</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Kogel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Demeulemeester</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vets</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">El Ashkar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The BET family of proteins targets moloney murine leukemia virus integration near transcription start sites.</span> <abbr class="cit-jnl-abbrev">Cell Reports</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">886</span>–<span class="cit-lpage">894</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BReports%26rft.volume%253D5%26rft.spage%253D886%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2013.09.040%26rft_id%253Dinfo%253Apmid%252F24183673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2013.09.040&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24183673&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.14" data-doi="10.1534/genetics.106.061184"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Emelyanov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naqvi</span>  <span class="cit-name-given-names">N. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parinov</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Trans-kingdom transposition of the maize dissociation element.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">1095</span>–<span class="cit-lpage">1104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.061184%26rft_id%253Dinfo%253Apmid%252F16951067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NC8zLzEwOTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvODA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.15" data-doi="10.1128/JVI.02098-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Faschinger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rouault</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sollner</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lukas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salmons</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mouse mammary tumor virus integration site selection in human and mouse genomes.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">1360</span>–<span class="cit-lpage">1367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02098-07%26rft_id%253Dinfo%253Apmid%252F18032509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgyLzMvMTM2MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC84MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.16" data-doi="10.1038/ng.368"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Faulkner</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daub</span>  <span class="cit-name-given-names">C. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wani</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plessy</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The regulated retrotransposon transcriptome of mammalian cells.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">563</span>–<span class="cit-lpage">571</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DFaulkner%26rft.auinit1%253DG.%2BJ.%26rft.volume%253D41%26rft.issue%253D5%26rft.spage%253D563%26rft.epage%253D571%26rft.atitle%253DThe%2Bregulated%2Bretrotransposon%2Btranscriptome%2Bof%2Bmammalian%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.368%26rft_id%253Dinfo%253Apmid%252F19377475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.368&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19377475&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265575600017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.17" data-doi="10.1038/nrg2337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feschotte</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Transposable elements and the evolution of regulatory networks.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">397</span>–<span class="cit-lpage">405</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DFeschotte%26rft.auinit1%253DC.%26rft.volume%253D9%26rft.issue%253D5%26rft.spage%253D397%26rft.epage%253D405%26rft.atitle%253DTransposable%2Belements%2Band%2Bthe%2Bevolution%2Bof%2Bregulatory%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2337%26rft_id%253Dinfo%253Apmid%252F18368054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2337&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18368054&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255057300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.18" data-doi="10.1073/pnas.1318639110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Genovesi</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Remke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Sleeping Beauty mutagenesis in a mouse medulloblastoma model defines networks that discriminate between human molecular subgroups.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">E4325</span>–<span class="cit-lpage">E4334</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1318639110%26rft_id%253Dinfo%253Apmid%252F24167280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzQ2L0U0MzI1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80LzgwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.19" data-doi="10.1534/genetics.109.102079"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goll</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stainier</span>  <span class="cit-name-given-names">D. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spradling</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halpern</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Transcriptional silencing and reactivation in transgenic zebrafish.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">747</span>–<span class="cit-lpage">755</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.102079%26rft_id%253Dinfo%253Apmid%252F19433629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzMvNzQ3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80LzgwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.20" data-doi="10.1534/genetics.113.152744"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Humes</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gangadharan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Integration profiling of gene function with dense maps of transposon integration.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">599</span>–<span class="cit-lpage">609</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.152744%26rft_id%253Dinfo%253Apmid%252F23893486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk1LzIvNTk5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80LzgwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.21" data-doi="10.1128/JVI.01942-13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maetzig</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maertens</span>  <span class="cit-name-given-names">G. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharif</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rothe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Bromo- and extraterminal domain chromatin regulators serve as cofactors for murine leukemia virus integration.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">12721</span>–<span class="cit-lpage">12736</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01942-13%26rft_id%253Dinfo%253Apmid%252F24049186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4Ny8yMy8xMjcyMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC84MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.22" data-doi="10.1242/dev.095091"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harvey</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sealy</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kettleborough</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fenyes</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Identification of the zebrafish maternal and paternal transcriptomes.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">2703</span>–<span class="cit-lpage">2710</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.095091%26rft_id%253Dinfo%253Apmid%252F23720042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTQwLzEzLzI3MDMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvODA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.23" data-doi="10.1038/nature12111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howe</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torroja</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torrance</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berthelot</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The zebrafish reference genome sequence and its relationship to the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">496</span>: <span class="cit-fpage">498</span>–<span class="cit-lpage">503</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D496%26rft.spage%253D498%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12111%26rft_id%253Dinfo%253Apmid%252F23594743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23594743&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000317984400040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.24" data-doi="10.1093/nar/gkn923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lempicki</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">a</span> <span class="cit-article-title">Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn923%26rft_id%253Dinfo%253Apmid%252F19033363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjY6IjM3LzEvMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC84MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.25" data-doi="10.1038/nprot.2008.211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lempicki</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">b</span> <span class="cit-article-title">Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">44</span>–<span class="cit-lpage">57</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DHuang%26rft.auinit1%253Dd.%2Ba.%2B%2B.%2BW.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253D44%26rft.epage%253D57%26rft.atitle%253DSystematic%2Band%2Bintegrative%2Banalysis%2Bof%2Blarge%2Bgene%2Blists%2Busing%2BDAVID%2Bbioinformatics%2Bresources.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2008.211%26rft_id%253Dinfo%253Apmid%252F19131956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2008.211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19131956&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265781800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.26" data-doi="10.1242/dev.115584"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Irion</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krauss</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nusslein-Volhard</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Precise and efficient genome editing in zebrafish using the CRISPR/Cas9 system.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">4827</span>–<span class="cit-lpage">4830</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.115584%26rft_id%253Dinfo%253Apmid%252F25411213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTQxLzI0LzQ4MjciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvODA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.27" data-doi="10.1186/gb-2007-8-s1-s7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawakami</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Tol2: a versatile gene transfer vector in vertebrates.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">8</span>(<span class="cit-supplement">Suppl 1</span>): <span class="cit-fpage">S7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D8%26rft.spage%253DS7%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2007-8-s1-s7%26rft_id%253Dinfo%253Apmid%252F18047699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2007-8-s1-s7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18047699&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.28" data-doi="10.1534/genetics.166.2.895"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawakami</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noda</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Transposition of the Tol2 element, an Ac-like element from the Japanese medaka fish Oryzias latipes, in mouse embryonic stem cells.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">895</span>–<span class="cit-lpage">899</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKawakami%26rft.auinit1%253DK.%26rft.volume%253D166%26rft.issue%253D2%26rft.spage%253D895%26rft.epage%253D899%26rft.atitle%253DTransposition%2Bof%2Bthe%2BTol2%2BElement%252C%2Ban%2BAc-Like%2BElement%2BFrom%2Bthe%2BJapanese%2BMedaka%2BFish%2BOryzias%2Blatipes%252C%2Bin%2BMouse%2BEmbryonic%2BStem%2BCells%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.166.2.895%26rft_id%253Dinfo%253Apmid%252F15020474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY2LzIvODk1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80LzgwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.29" data-doi="10.1016/j.devcel.2004.06.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawakami</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takeda</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawakami</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsuda</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A transposon-mediated gene trap approach identifies developmentally regulated genes in zebrafish.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DKawakami%26rft.auinit1%253DK.%26rft.volume%253D7%26rft.issue%253D1%26rft.spage%253D133%26rft.epage%253D144%26rft.atitle%253DA%2Btransposon-mediated%2Bgene%2Btrap%2Bapproach%2Bidentifies%2Bdevelopmentally%2Bregulated%2Bgenes%2Bin%2Bzebrafish.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2004.06.005%26rft_id%253Dinfo%253Apmid%252F15239961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2004.06.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15239961&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222696300017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.30" data-doi="10.1186/1471-213X-10-105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawakami</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abe</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asada</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asakawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fukuda</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">zTrap: zebrafish gene trap and enhancer trap database.</span> <abbr class="cit-jnl-abbrev">BMC Dev. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bdevelopmental%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BDev%2BBiol%26rft.aulast%253DKawakami%26rft.auinit1%253DK.%26rft.volume%253D10%26rft.spage%253D105%26rft.epage%253D105%26rft.atitle%253DzTrap%253A%2Bzebrafish%2Bgene%2Btrap%2Band%2Benhancer%2Btrap%2Bdatabase.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-213X-10-105%26rft_id%253Dinfo%253Apmid%252F20950494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-213X-10-105&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20950494&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.31" data-doi="10.1101/gr.229202. Article published online before March 2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">BLAT–the BLAST-like alignment tool.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">656</span>–<span class="cit-lpage">664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.229202.%2BArticle%2Bpublished%2Bonline%2Bbefore%2BMarch%2B2002%26rft_id%253Dinfo%253Apmid%252F11932250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzQvNjU2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80LzgwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.32" data-doi="10.1016/S0169-5347(99)01817-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kidwell</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisch</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Transposable elements and host genome evolution.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">99</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DKidwell%26rft.auinit1%253DM.%2BG.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D95%26rft.epage%253D99%26rft.atitle%253DTransposable%2Belements%2Band%2Bhost%2Bgenome%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0169-5347%252899%252901817-0%26rft_id%253Dinfo%253Apmid%252F10675923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0169-5347(99)01817-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10675923&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085405300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koga</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inagaki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bessho</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hori</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Transposable element in fish.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">383</span>: <span class="cit-fpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKoga%26rft.auinit1%253DA.%26rft.volume%253D383%26rft.issue%253D6595%26rft.spage%253D30%26rft.epage%253D30%26rft.atitle%253DTransposable%2Belement%2Bin%2Bfish.%26rft_id%253Dinfo%253Apmid%252F8779712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8779712&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.34" data-doi="10.1186/1471-2164-10-418"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kondrychyn</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Lecea</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emelyanov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parinov</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korzh</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome-wide analysis of Tol2 transposon reintegration in zebrafish.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">418</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DKondrychyn%26rft.auinit1%253DI.%26rft.volume%253D10%26rft.spage%253D418%26rft.epage%253D418%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2BTol2%2Btransposon%2Breintegration%2Bin%2Bzebrafish.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-418%26rft_id%253Dinfo%253Apmid%252F19737393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-418&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19737393&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.35" data-doi="10.1089/zeb.2011.0718"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kondrychyn</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teh</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Lecea</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Zebrafish Enhancer TRAP transgenic line database ZETRAP 2.0.</span> <abbr class="cit-jnl-abbrev">Zebrafish</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">181</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DZebrafish%26rft.aulast%253DKondrychyn%26rft.auinit1%253DI.%26rft.volume%253D8%26rft.issue%253D4%26rft.spage%253D181%26rft.epage%253D182%26rft.atitle%253DZebrafish%2BEnhancer%2BTRAP%2Btransgenic%2Bline%2Bdatabase%2BZETRAP%2B2.0.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fzeb.2011.0718%26rft_id%253Dinfo%253Apmid%252F22181660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/zeb.2011.0718&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22181660&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.36" data-doi="10.1093/nar/gkt1399"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">LaFave</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varshney</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gildea</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfsberg</span>  <span class="cit-name-given-names">T. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baxevanis</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">MLV integration site selection is driven by strong enhancers and active promoters.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">4257</span>–<span class="cit-lpage">4269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1399%26rft_id%253Dinfo%253Apmid%252F24464997%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjQyLzcvNDI1NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC84MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.37" data-doi="10.1038/35057062"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linton</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birren</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nusbaum</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zody</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Initial sequencing and analysis of the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">409</span>: <span class="cit-fpage">860</span>–<span class="cit-lpage">921</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D409%26rft.issue%253D6822%26rft.spage%253D860%26rft.epage%253D921%26rft.atitle%253DInitial%2Bsequencing%2Band%2Banalysis%2Bof%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35057062%26rft_id%253Dinfo%253Apmid%252F11237011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35057062&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11237011&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166938800058&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.38" data-doi="10.1186/gb-2009-10-3-r25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253DR25%26rft.epage%253DR25%26rft.atitle%253DUltrafast%2Band%2Bmemory-efficient%2Balignment%2Bof%2Bshort%2BDNA%2Bsequences%2Bto%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-3-r25%26rft_id%253Dinfo%253Apmid%252F19261174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-3-r25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19261174&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.39" data-doi="10.1002/dvg.20508"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stalker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradley</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Chromosomal mobilization and reintegration of Sleeping Beauty and PiggyBac transposons.</span> <abbr class="cit-jnl-abbrev">Genesis</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">404</span>–<span class="cit-lpage">408</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenesis%2B%2528New%2BYork%252C%2BN.Y.%2B%253A%2B%2B2000%2529%26rft.stitle%253DGenesis%26rft.aulast%253DLiang%26rft.auinit1%253DQ.%26rft.volume%253D47%26rft.issue%253D6%26rft.spage%253D404%26rft.epage%253D408%26rft.atitle%253DChromosomal%2Bmobilization%2Band%2Breintegration%2Bof%2BSleeping%2BBeauty%2Band%2BPiggyBac%2Btransposons.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fdvg.20508%26rft_id%253Dinfo%253Apmid%252F19391106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/dvg.20508&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19391106&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267352400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.40" data-doi="10.1186/gb-2013-14-7-r69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Featherstone</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A simple strategy for heritable chromosomal deletions in zebrafish via the combinatorial action of targeting nucleases.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR69%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-7-r69%26rft_id%253Dinfo%253Apmid%252F23815890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-7-r69&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23815890&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.41" data-doi="10.1126/science.8036514"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaiano</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Culp</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burns</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedmann</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Integration and germ-line transmission of a pseudotyped retroviral vector in zebrafish.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">265</span>: <span class="cit-fpage">666</span>–<span class="cit-lpage">669</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLin%26rft.auinit1%253DS%26rft.volume%253D265%26rft.issue%253D5172%26rft.spage%253D666%26rft.epage%253D669%26rft.atitle%253DIntegration%2Band%2Bgerm-line%2Btransmission%2Bof%2Ba%2Bpseudotyped%2Bretroviral%2Bvector%2Bin%2Bzebrafish%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.8036514%26rft_id%253Dinfo%253Apmid%252F8036514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNjUvNTE3Mi82NjYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvODA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.42" data-doi="10.1093/nar/gkn563"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Linheiro</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergman</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Testing the palindromic target site model for DNA transposon insertion using the Drosophila melanogaster P-element.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">6199</span>–<span class="cit-lpage">6208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn563%26rft_id%253Dinfo%253Apmid%252F18829720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzNi8xOS82MTk5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80LzgwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.43" data-doi="10.1101/SQB.1951.016.01.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McClintock</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1951</span> <span class="cit-article-title">Chromosome organization and genic expression.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Symp. Quant. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">13</span>–<span class="cit-lpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BSymposia%2Bon%2BQuantitative%2BBiology%26rft.stitle%253DCold%2BSpring%2BHarb%2BSymp%2BQuant%2BBiol%26rft.aulast%253DMcClintock%26rft.auinit1%253DB.%26rft.volume%253D16%26rft.issue%253D0%26rft.spage%253D13%26rft.epage%253D47%26rft.atitle%253DCHROMOSOME%2BORGANIZATION%2BAND%2BGENIC%2BEXPRESSION%26rft_id%253Dinfo%253Adoi%252F10.1101%252FSQB.1951.016.01.004%26rft_id%253Dinfo%253Apmid%252F14942727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic3FiIjtzOjU6InJlc2lkIjtzOjc6IjE2LzAvMTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvODA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.44" data-doi="10.1534/g3.113.009514"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McGaughey</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abaan</span>  <span class="cit-name-given-names">H. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kropp</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brody</span>  <span class="cit-name-given-names">L. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomics of CpG methylation in developing and developed zebrafish.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">861</span>–<span class="cit-lpage">869</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.009514%26rft_id%253Dinfo%253Apmid%252F24657902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjQvNS84NjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvODA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.45" data-doi="10.1371/journal.pone.0018826"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McGrail</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hatler</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nannapaneni</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Somatic mutagenesis with a Sleeping Beauty transposon system leads to solid tumor formation in zebrafish.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e18826</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMcGrail%26rft.auinit1%253DM.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253De18826%26rft.epage%253De18826%26rft.atitle%253DSomatic%2Bmutagenesis%2Bwith%2Ba%2BSleeping%2BBeauty%2Btransposon%2Bsystem%2Bleads%2Bto%2Bsolid%2Btumor%2Bformation%2Bin%2Bzebrafish.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0018826%26rft_id%253Dinfo%253Apmid%252F21533036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0018826&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21533036&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mullins</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Genetic nomenclature guide. Zebrafish.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">Mar</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253DMar%26rft.spage%253D31%26rft.atitle%253DGENETIC%2BNOMENCLATURE%2BGUIDE.%2BZEBRAFISH.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.47" data-doi="10.1073/pnas.95.19.11163"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">W. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorin</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">X. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hock</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhurkin</span>  <span class="cit-name-given-names">V. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">DNA sequence-dependent deformability deduced from protein-DNA crystal complexes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">11163</span>–<span class="cit-lpage">11168</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DOlson%26rft.auinit1%253DW.%2BK.%26rft.volume%253D95%26rft.issue%253D19%26rft.spage%253D11163%26rft.epage%253D11168%26rft.atitle%253DDNA%2Bsequence-dependent%2Bdeformability%2Bdeduced%2Bfrom%2Bprotein-DNA%2Bcrystal%2Bcomplexes%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.95.19.11163%26rft_id%253Dinfo%253Apmid%252F9736707%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTUvMTkvMTExNjMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvODA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.48" data-doi="10.1016/j.cell.2013.04.030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Potok</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nix</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parnell</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cairns</span>  <span class="cit-name-given-names">B. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Reprogramming the maternal zebrafish genome after fertilization to match the paternal methylation pattern.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">759</span>–<span class="cit-lpage">772</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D153%26rft.spage%253D759%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.04.030%26rft_id%253Dinfo%253Apmid%252F23663776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.04.030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23663776&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318844000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.49" data-doi="10.1534/g3.114.015842"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quach</span>  <span class="cit-name-given-names">H. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vrljicak</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joshi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A multifunctional mutagenesis system for analysis of gene function in zebrafish.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1283</span>–<span class="cit-lpage">1299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.015842%26rft_id%253Dinfo%253Apmid%252F25840430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjUvNi8xMjgzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80LzgwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.50" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi82Lzg0MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC84MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.51" data-doi="10.1038/ng.3164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rad</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rad</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strong</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ponstingl</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A conditional piggyBac transposition system for genetic screening in mice identifies oncogenic networks in pancreatic cancer.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D47%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3164%26rft_id%253Dinfo%253Apmid%252F25485836%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25485836&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.52" data-doi="10.1242/dev.100347"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miguel-Escalada</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slovik</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">K. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hadzhiev</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Targeted transgene integration overcomes variability of position effects in zebrafish.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">715</span>–<span class="cit-lpage">724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DRoberts%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D141%26rft.issue%253D3%26rft.spage%253D715%26rft.epage%253D724%26rft.atitle%253DTargeted%2Btransgene%2Bintegration%2Bovercomes%2Bvariability%2Bof%2Bposition%2Beffects%2Bin%2Bzebrafish%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.100347%26rft_id%253Dinfo%253Apmid%252F24449846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czo5OiIxNDEvMy83MTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvODA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.53" data-doi="10.1073/pnas.1307157110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larue</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plumb</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malani</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Male</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">BET proteins promote efficient murine leukemia virus integration at transcription start sites.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">12036</span>–<span class="cit-lpage">12041</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1307157110%26rft_id%253Dinfo%253Apmid%252F23818621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzI5LzEyMDM2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80LzgwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.54" data-doi="10.1371/journal.pgen.1004138"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shpiz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ryazansky</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olovnikov</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abramov</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalmykova</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Euchromatic transposon insertions trigger production of novel Pi- and endo-siRNAs at the target sites in the drosophila germline.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004138%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004138%26rft_id%253Dinfo%253Apmid%252F24516406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004138&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24516406&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.55" data-doi="10.1371/journal.pgen.1002731"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simeonova</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lejour</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bardot</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouarich-Bourimi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fuzzy tandem repeats containing p53 response elements may define species-specific p53 target genes.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002731</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSimeonova%26rft.auinit1%253DI.%26rft.volume%253D8%26rft.issue%253D6%26rft.spage%253De1002731%26rft.epage%253De1002731%26rft.atitle%253DFuzzy%2Btandem%2Brepeats%2Bcontaining%2Bp53%2Bresponse%2Belements%2Bmay%2Bdefine%2Bspecies-specific%2Bp53%2Btarget%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002731%26rft_id%253Dinfo%253Apmid%252F22761580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002731&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22761580&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.56" data-doi="10.1371/journal.pone.0044123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Effective gene trapping mediated by Sleeping Beauty transposon.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e44123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSong%26rft.auinit1%253DG.%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253De44123%26rft.epage%253De44123%26rft.atitle%253DEffective%2Bgene%2Btrapping%2Bmediated%2Bby%2BSleeping%2BBeauty%2Btransposon.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0044123%26rft_id%253Dinfo%253Apmid%252F22952894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0044123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22952894&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.57" data-doi="10.1186/1742-4690-5-48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Studamire</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goff</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Host proteins interacting with the Moloney murine leukemia virus integrase: multiple transcriptional regulators and chromatin binding factors.</span> <abbr class="cit-jnl-abbrev">Retrovirology</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DRetrovirology%26rft.aulast%253DStudamire%26rft.auinit1%253DB.%26rft.volume%253D5%26rft.spage%253D48%26rft.epage%253D48%26rft.atitle%253DHost%2Bproteins%2Binteracting%2Bwith%2Bthe%2BMoloney%2Bmurine%2Bleukemia%2Bvirus%2Bintegrase%253A%2Bmultiple%2Btranscriptional%2Bregulators%2Band%2Bchromatin%2Bbinding%2Bfactors.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1742-4690-5-48%26rft_id%253Dinfo%253Apmid%252F18554410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1742-4690-5-48&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18554410&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.58" data-doi="10.1038/ng.3175"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takeda</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koso</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rust</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yew</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Transposon mutagenesis identifies genes and evolutionary forces driving gastrointestinal tract tumor progression.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">142</span>–<span class="cit-lpage">150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D142%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3175%26rft_id%253Dinfo%253Apmid%252F25559195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3175&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25559195&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.59" data-doi="10.1126/science.1200801"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ting</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brannigan</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akhavanfard</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Aberrant overexpression of satellite repeats in pancreatic and other epithelial cancers.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">331</span>: <span class="cit-fpage">593</span>–<span class="cit-lpage">596</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1200801%26rft_id%253Dinfo%253Apmid%252F21233348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzEvNjAxNy81OTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvODA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.60" data-doi="10.1186/1471-2164-12-495"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tyekucheva</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yolken</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCombie</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parla</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kramer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Establishing the baseline level of repetitive element expression in the human cortex.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DTyekucheva%26rft.auinit1%253DS.%26rft.volume%253D12%26rft.spage%253D495%26rft.epage%253D495%26rft.atitle%253DEstablishing%2Bthe%2Bbaseline%2Blevel%2Bof%2Brepetitive%2Belement%2Bexpression%2Bin%2Bthe%2Bhuman%2Bcortex.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-495%26rft_id%253Dinfo%253Apmid%252F21985647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-495&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21985647&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.61" data-doi="10.1101/gr.151464.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Varshney</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gildea</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pei</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A large-scale zebrafish gene knockout resource for the genome-wide study of gene function.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">727</span>–<span class="cit-lpage">735</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.151464.112%26rft_id%253Dinfo%253Apmid%252F23382537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIzLzQvNzI3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80LzgwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.62" data-doi="10.1016/S0022-2836(02)00991-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vigdal</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaufman</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Izsvak</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voytas</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ivics</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Common physical properties of DNA affecting target site selection of sleeping beauty and other Tc1/mariner transposable elements.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">323</span>: <span class="cit-fpage">441</span>–<span class="cit-lpage">452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DVigdal%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D323%26rft.issue%253D3%26rft.spage%253D441%26rft.epage%253D452%26rft.atitle%253DCommon%2Bphysical%2Bproperties%2Bof%2BDNA%2Baffecting%2Btarget%2Bsite%2Bselection%2Bof%2Bsleeping%2Bbeauty%2Band%2Bother%2BTc1%252Fmariner%2Btransposable%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-2836%252802%252900991-9%26rft_id%253Dinfo%253Apmid%252F12381300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0022-2836(02)00991-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12381300&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179083800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.63" data-doi="10.1105/tpc.109.073452"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vollbrecht</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duvick</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schares</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahern</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deewatthanawong</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide distribution of transposed <em>Dissociation</em> elements in maize.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1667</span>–<span class="cit-lpage">1685</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.109.073452%26rft_id%253Dinfo%253Apmid%252F20581308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjIyLzYvMTY2NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC84MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.64" data-doi="10.1101/gr.6286907"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ciuffi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leipzig</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berry</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bushman</span>  <span class="cit-name-given-names">F. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span><span class="cit-comment">a</span> <span class="cit-article-title">HIV integration site selection: analysis by massively parallel pyrosequencing reveals association with epigenetic modifications.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1186</span>–<span class="cit-lpage">1194</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6286907%26rft_id%253Dinfo%253Apmid%252F17545577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE3LzgvMTE4NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC84MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.65" data-doi="10.1073/pnas.0703637104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sellers</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salama</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span><span class="cit-comment">b</span> <span class="cit-article-title">Species-specific endogenous retroviruses shape the transcriptional network of the human tumor suppressor protein p53.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">18613</span>–<span class="cit-lpage">18618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0703637104%26rft_id%253Dinfo%253Apmid%252F18003932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzQ3LzE4NjEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80LzgwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.66" data-doi="10.1038/nature01262"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waterston</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindblad-Toh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abril</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Initial sequencing and comparative analysis of the mouse genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">420</span>: <span class="cit-fpage">520</span>–<span class="cit-lpage">562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWaterston%26rft.auinit1%253DR.%2BH.%26rft.volume%253D420%26rft.issue%253D6915%26rft.spage%253D520%26rft.epage%253D562%26rft.atitle%253DInitial%2Bsequencing%2Band%2Bcomparative%2Banalysis%2Bof%2Bthe%2Bmouse%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01262%26rft_id%253Dinfo%253Apmid%252F12466850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01262&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12466850&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179611600053&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.67" data-doi="10.1038/82827"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weil</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunze</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Transposition of maize Ac/Ds transposable elements in the yeast Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">187</span>–<span class="cit-lpage">190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DWeil%26rft.auinit1%253DC.%2BF.%26rft.volume%253D26%26rft.issue%253D2%26rft.spage%253D187%26rft.epage%253D190%26rft.atitle%253DTransposition%2Bof%2Bmaize%2BAc%252FDs%2Btransposable%2Belements%2Bin%2Bthe%2Byeast%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F82827%26rft_id%253Dinfo%253Apmid%252F11017074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/82827&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11017074&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089638800015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.68" data-doi="10.1126/science.1083413"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crise</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgess</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Transcription start regions in the human genome are favored targets for MLV integration.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">300</span>: <span class="cit-fpage">1749</span>–<span class="cit-lpage">1751</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWu%26rft.auinit1%253DX.%26rft.volume%253D300%26rft.issue%253D5626%26rft.spage%253D1749%26rft.epage%253D1751%26rft.atitle%253DTranscription%2BStart%2BRegions%2Bin%2Bthe%2BHuman%2BGenome%2BAre%2BFavored%2BTargets%2Bfor%2BMLV%2BIntegration%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1083413%26rft_id%253Dinfo%253Apmid%252F12805549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDAvNTYyNi8xNzQ5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80LzgwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.69" data-doi="10.1128/JVI.79.8.5211-5214.2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crise</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgess</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munroe</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Weak palindromic consensus sequences are a common feature found at the integration target sites of many retroviruses.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">5211</span>–<span class="cit-lpage">5214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DWu%26rft.auinit1%253DX.%26rft.volume%253D79%26rft.issue%253D8%26rft.spage%253D5211%26rft.epage%253D5214%26rft.atitle%253DWeak%2BPalindromic%2BConsensus%2BSequences%2BAre%2Ba%2BCommon%2BFeature%2BFound%2Bat%2Bthe%2BIntegration%2BTarget%2BSites%2Bof%2BMany%2BRetroviruses%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.79.8.5211-5214.2005%26rft_id%253Dinfo%253Apmid%252F15795304%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijc5LzgvNTIxMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC84MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.805.70" data-doi="10.1038/nature10371"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pao</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huynh</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suh</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tonnu</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">BRCA1 tumour suppression occurs via heterochromatin-mediated silencing.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">477</span>: <span class="cit-fpage">179</span>–<span class="cit-lpage">184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DZhu%26rft.auinit1%253DQ.%26rft.volume%253D477%26rft.issue%253D7363%26rft.spage%253D179%26rft.epage%253D184%26rft.atitle%253DBRCA1%2Btumour%2Bsuppression%2Boccurs%2Bvia%2Bheterochromatin-mediated%2Bsilencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10371%26rft_id%253Dinfo%253Apmid%252F21901007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10371&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21901007&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294603900028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/4/805.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/4/793" title="Identification of Polymorphisms Associated with Drought Adaptation QTL in Brassica napus by Resequencing" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/4/819" title="Drak Is Required for Actomyosin Organization During Drosophila Cellularization" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/4"><span>Volume 6 Issue 4, April 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/4.cover-source.jpg?itok=Lj45nPaf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/805.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-z2ATDDnRNAAPUJX6yoW8tVgpo0i3pce1qMJ98oju6KI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/4/805" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/4/805" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/4/805" class="active" data-icon-position="" data-hide-link-title="0">Genome-Wide Analysis of Transposon and Retroviral Insertions Reveals Preferential Integrations in Regions of DNA Flexibility</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460125" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-wqykzael1JEkNbXLyBdvdvs52kdTiPTMajpC320w5gI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/4/805.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/4/805" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-JRbB6h3bVckbzu7I4K0WdinVYTp4prXB8tT81y1p0Lk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/4/805" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/4/805.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460125" id="node460125--2" data-pisa="ggg;6/4/805" data-pisa-master="ggg;g3.115.026849" data-apath="/ggg/6/4/805.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Analysis of Transposon and Retroviral Insertions Reveals Preferential Integrations in Regions of DNA Flexibility</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Pavle Vrljicak</span>, <span class="highwire-citation-author" data-delta="1">Shijie Tao</span>, <span class="highwire-citation-author" data-delta="2">Gaurav K. Varshney</span>, <span class="highwire-citation-author" data-delta="3">Helen Ngoc Bao Quach</span>, <span class="highwire-citation-author" data-delta="4">Adita Joshi</span>, <span class="highwire-citation-author" data-delta="5">Matthew C. LaFave</span>, <span class="highwire-citation-author" data-delta="6">Shawn M. Burgess</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-0729-1977" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Karuna Sampath</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">805-817;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.026849 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460125--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Pavle Vrljicak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Biomedical Sciences, Warwick Medical School, University of Warwick, Coventry CV4 7AJ, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pavle%2BVrljicak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vrljicak%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APavle%2BVrljicak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Shijie Tao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Temasek Life Sciences Laboratory, National University of Singapore, 117604, Singapore</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shijie%2BTao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tao%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShijie%2BTao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Gaurav K. Varshney </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>National Human Genome Research Institute, Bethesda, Maryland 20892-8004</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gaurav%2BK.%2BVarshney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Varshney%20GK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGaurav%2BK.%2BVarshney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Helen Ngoc Bao Quach </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Temasek Life Sciences Laboratory, National University of Singapore, 117604, Singapore</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Helen%2BNgoc%2BBao%2BQuach%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Quach%20HN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHelen%2BNgoc%2BBao%2BQuach%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Adita Joshi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Temasek Life Sciences Laboratory, National University of Singapore, 117604, Singapore</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adita%2BJoshi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Joshi%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAdita%2BJoshi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Matthew C. LaFave </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>National Human Genome Research Institute, Bethesda, Maryland 20892-8004</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BC.%2BLaFave%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=LaFave%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BC.%2BLaFave%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Shawn M. Burgess </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>National Human Genome Research Institute, Bethesda, Maryland 20892-8004</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shawn%2BM.%2BBurgess%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgess%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShawn%2BM.%2BBurgess%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:K.Sampath@warwick.ac.uk" class="" data-icon-position="" data-hide-link-title="0">K.Sampath@warwick.ac.uk</a></span></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Karuna Sampath </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Biomedical Sciences, Warwick Medical School, University of Warwick, Coventry CV4 7AJ, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karuna%2BSampath%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sampath%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKaruna%2BSampath%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0729-1977" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Karuna Sampath</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:K.Sampath@warwick.ac.uk" class="" data-icon-position="" data-hide-link-title="0">K.Sampath@warwick.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/4/805" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/4/805&amp;title=Genome-Wide%20Analysis%20of%20Transposon%20and%20Retroviral%20Insertions%20Reveals%20Preferential%20Integrations%20in%20Regions%20of%20DNA%20Flexibility" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/4/805&amp;title=Genome-Wide%20Analysis%20of%20Transposon%20and%20Retroviral%20Insertions%20Reveals%20Preferential%20Integrations%20in%20Regions%20of%20DNA%20Flexibility" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/4/805&amp;title=Genome-Wide%20Analysis%20of%20Transposon%20and%20Retroviral%20Insertions%20Reveals%20Preferential%20Integrations%20in%20Regions%20of%20DNA%20Flexibility" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/4/805&amp;text=Genome-Wide%20Analysis%20of%20Transposon%20and%20Retroviral%20Insertions%20Reveals%20Preferential%20Integrations%20in%20Regions%20of%20DNA%20Flexibility" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/4/805&amp;title=Genome-Wide%20Analysis%20of%20Transposon%20and%20Retroviral%20Insertions%20Reveals%20Preferential%20Integrations%20in%20Regions%20of%20DNA%20Flexibility" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/4/805&amp;t=Genome-Wide%20Analysis%20of%20Transposon%20and%20Retroviral%20Insertions%20Reveals%20Preferential%20Integrations%20in%20Regions%20of%20DNA%20Flexibility" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/4/805&amp;title=Genome-Wide%20Analysis%20of%20Transposon%20and%20Retroviral%20Insertions%20Reveals%20Preferential%20Integrations%20in%20Regions%20of%20DNA%20Flexibility" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/4/805&amp;title=Genome-Wide%20Analysis%20of%20Transposon%20and%20Retroviral%20Insertions%20Reveals%20Preferential%20Integrations%20in%20Regions%20of%20DNA%20Flexibility" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460125/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460125" id="node460125--3" data-pisa="ggg;6/4/805" data-pisa-master="ggg;g3.115.026849" data-apath="/ggg/6/4/805.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Analysis of Transposon and Retroviral Insertions Reveals Preferential Integrations in Regions of DNA Flexibility</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Pavle Vrljicak</span>, <span class="highwire-citation-author" data-delta="1">Shijie Tao</span>, <span class="highwire-citation-author" data-delta="2">Gaurav K. Varshney</span>, <span class="highwire-citation-author" data-delta="3">Helen Ngoc Bao Quach</span>, <span class="highwire-citation-author" data-delta="4">Adita Joshi</span>, <span class="highwire-citation-author" data-delta="5">Matthew C. LaFave</span>, <span class="highwire-citation-author" data-delta="6">Shawn M. Burgess</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-0729-1977" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Karuna Sampath</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">805-817;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.026849 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460125--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Pavle Vrljicak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Biomedical Sciences, Warwick Medical School, University of Warwick, Coventry CV4 7AJ, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pavle%2BVrljicak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vrljicak%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APavle%2BVrljicak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Shijie Tao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Temasek Life Sciences Laboratory, National University of Singapore, 117604, Singapore</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shijie%2BTao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tao%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShijie%2BTao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Gaurav K. Varshney </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>National Human Genome Research Institute, Bethesda, Maryland 20892-8004</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gaurav%2BK.%2BVarshney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Varshney%20GK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGaurav%2BK.%2BVarshney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Helen Ngoc Bao Quach </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Temasek Life Sciences Laboratory, National University of Singapore, 117604, Singapore</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Helen%2BNgoc%2BBao%2BQuach%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Quach%20HN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHelen%2BNgoc%2BBao%2BQuach%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Adita Joshi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Temasek Life Sciences Laboratory, National University of Singapore, 117604, Singapore</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adita%2BJoshi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Joshi%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAdita%2BJoshi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Matthew C. LaFave </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>National Human Genome Research Institute, Bethesda, Maryland 20892-8004</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BC.%2BLaFave%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=LaFave%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BC.%2BLaFave%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Shawn M. Burgess </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>National Human Genome Research Institute, Bethesda, Maryland 20892-8004</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shawn%2BM.%2BBurgess%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgess%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShawn%2BM.%2BBurgess%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:K.Sampath@warwick.ac.uk" class="" data-icon-position="" data-hide-link-title="0">K.Sampath@warwick.ac.uk</a></span></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Karuna Sampath </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Biomedical Sciences, Warwick Medical School, University of Warwick, Coventry CV4 7AJ, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karuna%2BSampath%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sampath%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKaruna%2BSampath%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0729-1977" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Karuna Sampath</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:K.Sampath@warwick.ac.uk" class="" data-icon-position="" data-hide-link-title="0">K.Sampath@warwick.ac.uk</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=805&amp;rft.epage=817&amp;rft.atitle=Genome-Wide%20Analysis%20of%20Transposon%20and%20Retroviral%20Insertions%20Reveals%20Preferential%20Integrations%20in%20Regions%20of%20DNA%20Flexibility&amp;rft.volume=6&amp;rft.issue=4&amp;rft.date=2016-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Vrljicak%2C+Pavle&amp;rft.au=Tao%2C+Shijie&amp;rft.au=Varshney%2C+Gaurav+K.&amp;rft.au=Quach%2C+Helen+Ngoc+Bao&amp;rft.au=Joshi%2C+Adita&amp;rft.au=LaFave%2C+Matthew+C.&amp;rft.au=Burgess%2C+Shawn+M.&amp;rft.au=Sampath%2C+Karuna"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460125/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460125/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460125/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460125/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460125/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460125/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460125/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460125/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460125/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460125/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460125/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460125/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/4/805" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/4/805.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/4/805.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/4/805.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/4/805.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
