<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/4/857" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/4/857.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/4/857.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/4/857.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/4/857.atom" />
<meta name="HW.pisa" content="ggg;6/4/857" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Haplotag: Software for Haplotype-Based Genotyping-by-Sequencing Analysis" />
<meta name="DC.Identifier" content="10.1534/g3.115.024596" />
<meta name="DC.Date" content="2016-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Tinker et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genotyping-by-sequencing (GBS), and related methods, are based on high-throughput short-read sequencing of genomic complexity reductions followed by discovery of single nucleotide polymorphisms (SNPs) within sequence tags. This provides a powerful and economical approach to whole-genome genotyping, facilitating applications in genomics, diversity analysis, and molecular breeding. However, due to the complexity of analyzing large data sets, applications of GBS may require substantial time, expertise, and computational resources. Haplotag, the novel GBS software described here, is freely available, and operates with minimal user-investment on widely available computer platforms. Haplotag is unique in fulfilling the following set of criteria: (1) operates without a reference genome; (2) can be used in a polyploid species; (3) provides a discovery mode, and a production mode; (4) discovers polymorphisms based on a model of tag-level haplotypes within sequenced tags; (5) reports SNPs as well as haplotype-based genotypes; and (6) provides an intuitive visual “passport” for each inferred locus. Haplotag is optimized for use in a self-pollinating plant species." />
<meta name="DC.Contributor" content="Nicholas A. Tinker" />
<meta name="DC.Contributor" content="Wubishet A. Bekele" />
<meta name="DC.Contributor" content="Jiro Hattori" />
<meta name="article:published_time" content="2016-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Haplotag: Software for Haplotype-Based Genotyping-by-Sequencing Analysis" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genotyping-by-sequencing (GBS), and related methods, are based on high-throughput short-read sequencing of genomic complexity reductions followed by discovery of single nucleotide polymorphisms (SNPs) within sequence tags. This provides a powerful and economical approach to whole-genome genotyping, facilitating applications in genomics, diversity analysis, and molecular breeding. However, due to the complexity of analyzing large data sets, applications of GBS may require substantial time, expertise, and computational resources. Haplotag, the novel GBS software described here, is freely available, and operates with minimal user-investment on widely available computer platforms. Haplotag is unique in fulfilling the following set of criteria: (1) operates without a reference genome; (2) can be used in a polyploid species; (3) provides a discovery mode, and a production mode; (4) discovers polymorphisms based on a model of tag-level haplotypes within sequenced tags; (5) reports SNPs as well as haplotype-based genotypes; and (6) provides an intuitive visual “passport” for each inferred locus. Haplotag is optimized for use in a self-pollinating plant species.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/04/01" />
<meta name="citation_mjid" content="ggg;6/4/857" />
<meta name="citation_id" content="6/4/857" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/4/857" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/4/857.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/4/857.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/4/857.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.024596" />
<meta name="citation_pmid" content="26818073" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="7" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="857" />
<meta name="citation_lastpage" content="863" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Nicholas A. Tinker" />
<meta name="citation_author_institution" content="Ottawa Research and Development Centre, Agriculture and Agri-Food Canada, Ontario, K1A 0C6, Canada" />
<meta name="citation_author_email" content="nick.tinker@canada.ca" />
<meta name="citation_author" content="Wubishet A. Bekele" />
<meta name="citation_author_institution" content="Ottawa Research and Development Centre, Agriculture and Agri-Food Canada, Ontario, K1A 0C6, Canada" />
<meta name="citation_author" content="Jiro Hattori" />
<meta name="citation_author_institution" content="Ottawa Research and Development Centre, Agriculture and Agri-Food Canada, Ontario, K1A 0C6, Canada" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=J. M. Catchen;citation_author=A. Amores;citation_author=P. Hohenlohe;citation_author=W. Cresko;citation_author=J. H. Postlethwait;citation_title=Stacks: Building and Genotyping Loci De Novo From Short-Read Sequences;citation_pages=171-182;citation_volume=1;citation_year=2011;citation_issue=3;citation_pmid=22384329;citation_doi=10.1534/g3.111.000240" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. C. Glaubitz;citation_author=T. M. Casstevens;citation_author=F. Lu;citation_author=J. Harriman;citation_author=R. J. Elshire;citation_title=TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline.;citation_pages=e90346;citation_volume=9;citation_year=2014;citation_issue=2;citation_pmid=24587335;citation_doi=10.1371/journal.pone.0090346" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Y. F. Huang;citation_author=J. A. Poland;citation_author=C. P. Wight;citation_author=E. W. Jackson;citation_author=N. A. Tinker;citation_title=Using genotyping-by-sequencing (GBS) for genomic discovery in cultivated oat.;citation_pages=e102448;citation_volume=9;citation_year=2014;citation_issue=7;citation_pmid=25047601;citation_doi=10.1371/journal.pone.0102448" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. J. Lorenz;citation_author=M. T. Hamblin;citation_author=J.-L. Jannink;citation_title=Performance of single nucleotide polymorphisms versus haplotypes for genome-wide association analysis in barley.;citation_pages=e14079-e14079;citation_volume=5;citation_year=2010;citation_issue=11;citation_pmid=21124933;citation_doi=10.1371/journal.pone.0014079" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=F. Lu;citation_author=A. E. Lipka;citation_author=J. Glaubitz;citation_author=R. Elshire;citation_author=J. H. Cherney;citation_title=Switchgrass genomic diversity, ploidy, and evolution: novel insights from a network-based SNP discovery protocol.;citation_pages=e1003215;citation_volume=9;citation_year=2013;citation_issue=1;citation_pmid=23349638;citation_doi=10.1371/journal.pgen.1003215" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. R. Miller;citation_author=J. P. Dunham;citation_author=A. Amores;citation_author=W. A. Cresko;citation_author=E. A. Johnson;citation_title=Rapid and cost-effective polymorphism identification and genotyping using restriction site associated DNA (RAD) markers.;citation_volume=17;citation_year=2007;citation_issue=2;citation_pmid=17189378;citation_doi=10.1101/gr.5681207" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. A. Poland;citation_author=P. J. Brown;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.;citation_pages=e32253-e32253;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22389690;citation_doi=10.1371/journal.pone.0032253" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=H. Sonah;citation_author=M. Bastien;citation_author=E. Iquira;citation_author=A. Tardivel;citation_author=G. Légaré;citation_title=An improved genotyping by sequencing (GBS) approach offering increased versatility and efficiency of SNP discovery and genotyping.;citation_pages=e54603;citation_volume=8;citation_year=2013;citation_issue=1;citation_pmid=23372741;citation_doi=10.1371/journal.pone.0054603" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_title=Novel methods to optimize genotypic imputation for low-coverage, next-generation sequence data in crop plants.;citation_author=K. Swarts;citation_author=H. Li;citation_author=J. A. Romero Navarro;citation_author=D. An;citation_author=M. C. Romay;citation_volume=7;citation_year=2014;citation_issue=3" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Haplotag: Software for Haplotype-Based Genotyping-by-Sequencing Analysis" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
<meta name="twitter:description" content="Genotyping-by-sequencing (GBS), and related methods, are based on high-throughput short-read sequencing of genomic complexity reductions followed by discovery of single nucleotide polymorphisms (SNPs) within sequence tags. This provides a powerful and economical approach to whole-genome genotyping, facilitating applications in genomics, diversity analysis, and molecular breeding. However, due to the complexity of analyzing large data sets, applications of GBS may require substantial time, expertise, and computational resources. Haplotag, the novel GBS software described here, is freely available, and operates with minimal user-investment on widely available computer platforms. Haplotag is unique in fulfilling the following set of criteria: (1) operates without a reference genome; (2) can be used in a polyploid species; (3) provides a discovery mode, and a production mode; (4) discovers polymorphisms based on a model of tag-level haplotypes within sequenced tags; (5) reports SNPs as well as haplotype-based genotypes; and (6) provides an intuitive visual “passport” for each inferred locus. Haplotag is optimized for use in a self-pollinating plant species." />
<meta name="og-title" property="og:title" content="Haplotag: Software for Haplotype-Based Genotyping-by-Sequencing Analysis" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/4/857" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genotyping-by-sequencing (GBS), and related methods, are based on high-throughput short-read sequencing of genomic complexity reductions followed by discovery of single nucleotide polymorphisms (SNPs) within sequence tags. This provides a powerful and economical approach to whole-genome genotyping, facilitating applications in genomics, diversity analysis, and molecular breeding. However, due to the complexity of analyzing large data sets, applications of GBS may require substantial time, expertise, and computational resources. Haplotag, the novel GBS software described here, is freely available, and operates with minimal user-investment on widely available computer platforms. Haplotag is unique in fulfilling the following set of criteria: (1) operates without a reference genome; (2) can be used in a polyploid species; (3) provides a discovery mode, and a production mode; (4) discovers polymorphisms based on a model of tag-level haplotypes within sequenced tags; (5) reports SNPs as well as haplotype-based genotypes; and (6) provides an intuitive visual “passport” for each inferred locus. Haplotag is optimized for use in a self-pollinating plant species." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
    <title>Haplotag: Software for Haplotype-Based Genotyping-by-Sequencing Analysis | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Lk10W1-CPPiKQ9RmvvrZvXZz8a3eRV48wGFDpUvvN0g","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460109","apath":"\/ggg\/6\/4\/857.atom","pisa":"ggg;6\/4\/857","ac":{"\/ggg\/6\/4\/857.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/4\/857.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/4\/857"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/4\/857"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/4\/857.supplemental":true,"\/content\/6\/4\/857":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222LOVEtXw1GlI3Na2WOZZk0+aLzoYJXVwfDAakd0cFZIl41+KeNZPpw76l2+eQLLvyGuga55XJHv5n4Xa3O7t0m\\\\\\\/jFe8yWHY3bMA4ERTxxXTWaKwD\\\\\\\/SNp\\\\\\\/8ptG7gqCc5STCeEveBxTvV36dA6tBbpRkfNtyQughm71YEybNrARRAgTE0xKJeq7HwHKrhvzrbxtrSiZWS5zg2kNpAiNYDEcG\\\\\\\/wozdPnw8xNSIXO9KATDCDHSBioYA96GAdpLubw+68THMaXWGp01EqLfacoPv69U263gqpRO0uKw4t7wCfEd\\\\\\\/d+qJx6UNjLJYPXX9aBAI5yLIZKcogfMHdfRPyVXnKPw1J25P3TyVKj2B\\\\\\\/fCGLUfhIaEXPwH\\\\\\\/NV8ATIK5mlydQbyhdoKZhvfIung4Gwi7lA8NrevBlTbjhzo8RTxvFd79Gh4KZnHqu7hkeknV2vS0\\\\\\\/k1DnzeEPc2FNDrvIOAxOUBZd6ZFC1yeMa3h4mDXEIIoAuO7p7RQm55D3j0jeIA33PDU3F+lyV20OEYG94xBDhWquy8GMoBp2fZGJf+uV6vG2W00PC6SEE8mmGtZUTRlftgiR0NtuuwJAj6ZX6d7ID3sQ6XfBmNGrKIZ5mV0QuN7OQI3\\\\\\\/iim83x3iea6y5MaDXVhCZqCtWCDGSdrwUHcNm0nP32DFM9FPRylFwrmitzel17ZrHLesuBkmLrgzNbdCLUsgwbK\\\\\\\/B8431OiV7IB5W\\\\\\\/d7EwNuE5NGZB84TPLuQBcpHsRyC+m8471PZ5QsyzgY5kV53RheMuyU6KAA9vtz1ZBpUnYjl82eKNiUWyjBNSOxLEgb86Uo+Qk4Xbh+MW+BJY5Jwemj0efBK67gH8BWbwDolwqNFHMq2knt3oDhlej01+V0pNnIHLfqlG6Unjb99iERn5l9ItaW+tpI1K16cJSZ+lN\\\\\\\/l1\\\\\\\/jzhjMBr18qWVPDl0yev4EWV03pd5Zd9Nva+MipC+WHsFfTNg1L5+9go4YODelPwFRUKYsOPzMbZnytbZU\\\\\\\/iigurEM1aTyTmn\\\\\\\/yxRvfhntNicQAIdVLuy5s9LwvLSSTk45cMAnoaam9Pr9fs85Ux\\\\\\\/Cg8XEt7F7pO6ioJDKdyGjjfxTOatXhuZL2dudGHEk4iTg6nKIv22t1TBoi\\\\\\\/Eq\\\\\\\/0BFJEaVciPqH\\\\\\\/ftARmg2cLpZUwWmKGE0lkWfdwL1p86xUcYz4y1Nvn6znU8IiscQFFbtx+0Vee3wGfjsBo\\\\\\\/Z0oN8jMvakvt1Slcgowp3KMEBVr7vtvonkjuM3Sx\\\\\\\/iSA4vjE7l+69lUHM6Z8\\\u0022,\\\u0022iv\\\u0022:\\\u0022a8HahPJbPAScp0U4FUSGow==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022fbb5bd84cccd9afd6594297fa3560ea371625b875f7f71397d9f14a039d3f379\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022RnXBK73KhYdHuYHvEP2Q\\\\\\\/by6yBnM2\\\\\\\/e2IzV2i+\\\\\\\/v5Cc+anOZRHknvUAG9ulkxQMCpihdnp86NO\\\\\\\/3q+5qQcyn8h556j13bI1GHkTqLDqpTOlJjJRGRG4gpn2jycQM5XT\\\\\\\/\\\\\\\/SaO\\\\\\\/FsIDMZQAyF0MtfE\\\\\\\/FlxBjHdNe2RWG5fG5TZUSODx2256Ni5agm3Ji59wpSUS49Sv1+FcekDIJPeEIYsS45TGzbVFUxY0rL6jymZ0iCkvV7glPpPPn3W4HsOQwxYmVig63LKiVscZn16\\\\\\\/lldcncdTA4uF3VT\\\\\\\/0cB31BFQzQISrNm6hwiTpGd5iLdeEzmKd+VDfA8HWY1ObpAd9X7H++tvX26UDiBintEQk82pDi1QB3kiwoAGn9I5CQ6Kh1jrLtCvBB2p3cyNuoPReW8e7gBiXMY9FYXLY1BS7ijfK8JkD1YoysAHedqoEvPpzxHGAUvzgwQLlQpGPnABlEl77TNMv9JQa8i8Aw2thv3NM8LmtaPA5MeFw5VzCREimn0OnsU8B2QF31FvUpB82HwK3+bNuh467I3SNpTvXP0UfRmslaC4rHUxfHuue6+xIYmNgfjaIiIH3NE20h9qKfJ78ydex0\\\\\\\/FYTf\\\\\\\/9FGC7plUa5J6HzWjbT8FWWGNsxdY9rZVHsaatwBj22yTq+mCDx8SHplYDctRiR8unz0n1KhXPTdGvqFxI59\\\\\\\/1aAEG0GGDHo4puSllQHJpert+Ioig+sxYBlyNUWi003+Ydj+xd1s1RWEqrkCDn5u1Oxb39c6pPkVIGjLkMZ2gOSqsyrWCHT7sA5cdkbAQrJE6ZNXf\\\\\\\/GnKfDHogjOvrK4VOgWchGMmGFbUkNY8f8tupGA4vUw76Fr6ppS8MlRaq0D6NoKMWjW3rO00h\\\\\\\/nD\\\\\\\/\\\\\\\/JFAT9Q5XBhAPmXG\\\\\\\/HZjkiQxfqn3\\\\\\\/zWtq+mkyZTWA2618sycmqt6PUsn2PHd3OSO8Ij2kPPpWisgoshPTu5Qge5qN6a8ux38PtaVYFeTw8uTH3QRwxqhPTNCh03+C+0cjTlXaurXxkVUYPQQ7CylSWIdbUGBB0Lo3eEK5yvRJyIf\\\\\\\/7yDaJTuUl1WTwFxATQ2nAf+eypisRd7uKQwS6g14o+N\\\\\\\/H3f2RxpHJDYx\\\\\\\/hOSFVXFJKCOyuoFk8hmWpRyn5H6BLFoUVMsWFBS7wmviGNDdpljz7uyN2QflLOjgRjyi\\\\\\\/Ce03YRPO6WRzMnrbNfAM4W22lDjz2UDY7npBVgN\\\\\\\/rIQ0pbbL3tsx\\\\\\\/x3Kyg+3LLp3Wogf8RjCt01K7vS\\\\\\\/hKbcefKw1d0UBL8T1kwjgOJVEwsm\\\\\\\/N2Xj1xkq+kq+iCAeNs61Lva1NNHM8YKapBt0DoGlEIpena20FdJUwTFFuVntbROzJw2II8P6R9SmrJOqGFMURcMMq1PE7vKsamS2DhP\\\\\\\/UCD4EseSc11Oee4UYTUxpgWi0u0SBofYaHadpurSSXdt+OL15sXg\\\\\\\/sul5AXRZKI8wdvlZ+iTk6eHPiGlmsCN4AY+rBIHWdW3puW36lt6qEb3TyRcYs5K0U0+axnXys1866BrkRwqGLw4uYnqlbOVN7Cw6qEkiPYBP\\\\\\\/sS3MejvifrURfL2UbBn1EnWvWhdB9\\\\\\\/Lmh1+wqEFLs6OLo52EXTk0hCfel2Gza1edonvM2gXywc09fjkt73g1CnQCSz1XS4Jrx4oRf4C8wdWSv0QD3oKtvMPswqcEFUrYoMavf+MmUVef4sk0C\\\\\\\/WlSQLUsbskwLZduSAVHnhe4mSbgt7ugDGLZVNiF9vA2w8MuTfgeM6gQ1eQA6+rma\\\\\\\/oeXFjG\\\\\\\/SW3qUFVBFlKiNCL1SowDXcq1WZEYJtUhhgn2uBf8cqwLlPtKs0XB2dxYUCDWRo\\\\\\\/ZcepqVNhP\\\\\\\/cIRcq8IJLaC+XHdlVtOfpaYr+h2IFjvfJOGHdQya+Jzh\\\\\\\/gOxIp3cvWguQ9R04Da9bvg4sjK7l9ZXHi34vgqwIfOPCBDc9zkNq1kwSIg\\\\\\\/7UGBcPxj4f+UkTaj8c6umVbL8gbywO\\\\\\\/vWH\\\\\\\/kQL3RpdZ6DLQFNld60803e1RS5E16LGJtCoFW8ER7\\\\\\\/SbErfbkpioaavVge3o2EEkHg15gdA+4ZWWiSIPauUrhLv2y70j0T+Xb6a1zg+doO9wDFS2ioOZRkUdpD6XJmF9q1\\\\\\\/slLqnw86G9fRJ5yCl3Lb4Fpt8XGdMSeOV7u5NPJxFNZSQj+\\\\\\\/rW\\\\\\\/omTFIAsrx3+PA0HzryMUwHNmRuoHW7Tl\\\\\\\/kAeyFaFU0URz547b+iRWl9XGYL+Lk84gl9r0iYEC8CDJTcf2AosqrdiDT8KahHfi503rEVrMhKaiisu9SpEKmvkHyAh1eu2Xkqnwe4U5fXAm2BLPNslbP5qBqvJIUIKxoDAeOl0wbTknZ0D3\\\\\\\/CB0fg4O+U9FDzKDRKJVClWxPql7TghCjZ2cCpliiNthXYlEnapuxujukdEe3aP6nqSnkxCtqFtL4m2Zg7jfPaGtEs7dTob5JkUGv6nVMe6DnnKV4ZTk1tMxqA2VnpokhRqI+BGyseG9PJZy+eJShEex3mZCltmJJXq46r5oUlQ1DVFRbw93adOpHfLLs9qXp5mB2QzKc+yoaumpqIYqi5gh3Y\\\\\\\/vbwbPBeARCpxhVgzKTxtYCkMmbwjUA+eXKa5RYP4QuC59HSTKeJoja6pImN8g1\\\\\\\/ezpgzZWo3dLYDVWZUimCawHjy2N1gCY044OzM8XfgdZLo0VXyncusCTmxMjm9k5r\\\\\\\/qdbK8Hhwc7B5SxjDdo0qDGdV0R1Fg2W\\\\\\\/+ut39AEncMqyc8IfA0r46JGytyzSs60kdn8QwkzPsYeMoInEUQdlzUCPvD\\\\\\\/yfa9P9awa6lMoJQWpHbTRMABEP22Ko5QwxKH2EqugXHUmQCCcTDfUhvHOq+goXFpCphA7hcy\\\\\\\/xSaoWqocKowjAKK8HrC\\\\\\\/PJYCWsfDbI4mFcprsGuunVmpjZosxNDWBcr+fkKnClqHq2YmWLPAAjx7IEV+W0X6Wq71ONcwWezlo6mqWRsjPNWCjcC3lsGJBWGxmyQEkBW3cmib+bmxMeizrDuweLIrHJYQWptlxP4zk6l7teHzEwe3EjA1G+AksZqFqXlQaC3SpvqhrKeOsUaOWpxgxWUuPi7xaGXVbGN4CrDf+VuPVavgk6PvyhhJRgi9cq4dzQpCDXoJDWflThQIDKG72r2WtGpcCsjE2ziu5fy2hwIxGW+fSCBC2STbCqFmHYPGuqM23xEnqpVhNf+QXdRqTp\\\\\\\/xO4uLy9ijhDOI8F\\\\\\\/1Sp0cb\\\\\\\/g+QZ5oTG7yIE+f\\\\\\\/oJlADHPVgB0mL8NWNb+7+WUpB237Mvc252oh0i1RavmrPoDEno5r7jcpVV2O98ECXNHJSgjrUNEudqgnd5NQ5YBd9jYHDjz1slFBNuanVjEZDcDeXmAtFR2Ydj9Jvc80MUZCga0Itj0G8DcJJDF2GJsMp67cdwxzaDV4+CHCuRXNtyLxPWf4Khflc3DuX47YINB+qz63OF9X83A7m\\\\\\\/QSTgBNycVem1P93k0sMS4EnPJyt8r1CTxjTv3Za2QiTVPnsN\\\\\\\/4ISMbnLkFgEab3SU0v9KlcSKCj7ZKkUK0C5ZTMylIwXiVYukrXDD2TKnfEyiPk70DPekdllNjJSae+2mJmaGWr+FCQ0QZBE6RrSiDtjq7wrHsgYwrvjKVjPt4zpj6bT7R8MI9tZ5nfsosp4iKjczHHFcAU9vznMw4Jw7KtIIvr6gkQFr0SQQF9aLi+CrO8CNZk6bybU\\\\\\\/HAawjEV9rNbadnS7QetW0HrRn1D1Sl4M\\\\\\\/NY57YCgqqx909LkV29E1OG2iyUnxRPoPHliYSTy1w6JXnhgmtlhW+DjcnMtHh14f1mBmTho\\\\\\\/+vMGBkQZ1yEnJAp9ZmBaFmMxsA7liL+wz9JFZhdYA5tZvNe9cKIPVcUPkccqJ\\\\\\\/9u9bg5tJbVK3GR5\\\\\\\/yoR1IWL0dHsatfOSXqWg40m487zJYYV6Ea6+uv70QKHS7THndhoEzxEFO15e5PwPj84VoZPmbZioZgAvF3iQ0753\\\\\\\/\\\\\\\/coXkCeLhA0jJ4BHJMcmUbhMnGSen9g3Qd0h47LNoh6RCpw+uMiT8SqAWaDDsxiZ97iuFRlQD9fhdna1ddlfNJ8\\\\\\\/yj6hj02u1qZEnKtqYBxztjXfavem+DTcDDokIB4x81p7TpO0Z1EYqMa7ISD0007VXm1BfGrrVKUf6sBCzuVMRVaVcaxJdK52\\\\\\\/E4i7yNKkPS+OmhDVY4nZU9NCvRVw3pd2PiXe156HELtuJF2ToCPQ925qiTHL+kO6\\\\\\\/aY6FcOJPpve+GG5eEHIja4brjx7ltKmGqGsipXLpdrMENiQJ5pQnXBxocDRG8E3XAzX+2JYBNF+\\\\\\\/r+cSJmLKd4N9xEj35xkzkTAmqG9WLrus8znaSu5iOP82C7JlahXPwSGZ7DESLM6AFnGo+EyoOGId0CiYu9YizPW+W7x1M9VuXOXc93wEnHjcemdQ0Zn7IpuyUbOX486uQWtn6VvfW9oJG5qlH4y+8rerA4ohBYdnlwZz2gRV66+zR9u7JqGPgOb8vwFG9TVFUsl4R1sus2OWk66HwhRRJGWCmCQqbwmTiPpa8ZSs5ao+W\\\\\\\/N2gXHw0JeDdBiVzybcl1ZtpHvtIWamngI4wNbvKDUEILv12xiDqDnR+UsuH9vN9lIJ\\\\\\\/UJjSAUEiTGkGq66iS2hLJLEpt07BXYoeMc4HQMBmJt64\\\\\\\/v6+UER7Qt\\\\\\\/\\\\\\\/c4HwfxytYd4nIJHV8j99EJOnDUiZ9EITl2T1BOryxbvt\\\\\\\/S9zrdvAhqI1yATclD5mbAx7kjPx4n2mOnAIlg7nsliiFG6AoA8bTGRKhVHyASzae\\\\\\\/07CVIfdwJ4OhPBguZz5mZL+fS0lHPy49wIgGOLCRCFqWHpc6eHNGG8rXH\\\\\\\/fI\\\\\\\/025sgftrSk\\\\\\\/OaP05MZio6PlxPgEhGGcPk0eyLzVTQ5KGVLyFfh4Hez4DVlclaqU9jNh2+OnGXYQKLli05jGbSThXHZBFqsp1cUhd2sYmJzXDwhq8Szqf2WsPr9xy323txGyLWdSjyLYF+uJw3En9+sn6YkBHXakB8qTy\\\\\\\/1JMpGPvmoO1JfCYTwgnJJedqMjJrOZY+qT+71ch2k0WahhoJFtH2WllDVWkmWnDccIRxkljn3cCrvit5o7ihYc1OuQtuCho2lE9ALMdpR2isDmajJEK6H6Fq9qBqqEwcMbuuXjORiA0WPRWH5Vn1L0AxiAKi3Cz4CG44vYCOztmGnW\\\\\\\/s1TGs53K20DwapfEKdzx7V3CkMHXPJtwcKAEQAH7ucRbVxfUkHaIRK6voVzhPrSIrxa9OKoLVANGM8y\\\\\\\/1ZgTnfjcUvo1tqUNgjGiL\\\\\\\/vYHAxGV0KhJtbGxxVGQYkvbSWsY64qNIa\\\\\\\/ga7apbGUvnd\\\\\\\/alxV2sQTsitxG1nF8tGXFl2of\\\\\\\/S5WA7SWGTYLSZL0WW+AdIwe8OlMv8w+kAfly6xI90AKKUAj7KkQquTtMwOmyOr8WJFkPfsp2uyCibp+924BzuFKaeLUsmx2CMzpcDF2pRVFd1Sv3zMC3dDVH9UaPJOgZEAN96LlwMBxGvHJ3dp0lhX8gKYxeE7k\\\\\\\/v8QkOMAYLepj35eC0jbBH3obRSaRtCq57o8wvjpc\\\\\\\/0niKZUQJV1Od0q874IKRtQypa8+UBR87HwLh3VgpRgUKjV\\\\\\\/HVSfsI9A9SYPKZRw4kBf7rnA6LzhAtUjf+H+5jkAHrKcaLacFTgqKZSCs6Rsvuq+y3tRmy5milSwo5lYP4e22okBa3f6u74dxfW6vvzvVzKZSWMg1UfZdOWflT3n9CmakL4tYjbDV2pTz7KzbelteDYi\\\\\\\/jZi+lNIybVr9xcIIqYWk+BQIAQfbaMxSv\\\\\\\/s1+iIa2+Xsy76InCdpMRENzrK\\\\\\\/rWzkVDC8EAn5\\\\\\\/wxGnx8HxWUN0fEOYHvEw2gbyPenGagA6llXHDFYF0G0ZLJ7XsS5Kf7e8QCQ+AVsq7w5qfguO+OK+G68+IBc5TRqyN6Oq5xVmWIDlgePYXBmz0Edrsu8M5pv20LFEDiQAZCsuNBE0\\\\\\\/zf0hkkPIfTyyfYjEcQrfxambzEqwTwDSzSZx0pr3s3PjmW+Z0uyljrXFHGDoQ2sLr71Hxd5vskPsTJ0TwUst7K7bZ3MfGHGud\\\\\\\/F63\\\\\\\/GI0oCbNp4gpCrbWDQsriYbL\\\\\\\/c7BpUK7NauauYL\\\\\\\/BeBpdFXynAGUoa9BmgBAa+RJe851ViXFejnvub\\\\\\\/ohgmDrHx8cRfy79STDqgzK8r9XGBUpHtydJfILu0gkKA9NFn\\\\\\\/mpCynEp2ykxLvQCmF+r+y9lPmNfDjfXYtYJ1kbxlbnVGbJHn+qHVUDcn3Lwl9GNjHELxMs5H6JoA5cdGxYzco5ja1jRo\\\\\\\/6J2jpZPdlkzHD75wvYL\\\\\\\/SnR0CV4iRYx8hejg1kaj7g0Okq2ms2mz7gV6ZzpWiOptIZAOa\\\\\\\/VGIsq8aMQphI2fwoZHzVMmBSdCM8sUZJyF9y72OiNbOPt0zqF9uK7PdOZ5PV7EWQLNNNApDl5HmommNPppVnKmUM\\\\\\\/JNW5Csd2EILqwyrLyYuwluv8caMThaUS8JTIETL8l2NoaXP+FNxODUG8AaA\\\\\\\/YBE3YxqzhDtyGXy7DhJ0dhVQI7qJN1ulA96snctG8wCvkejOqFSMLzR8rbYSYGwGQcEZNfojDoNY7Vd9EAKMqQEKDuoCieSGIqCAw4igXPx6HJJT4EINiA\\\\\\\/5rpI9dnSDZ\\\\\\\/c+xu7nkfiRBFbnNaxBfmKPONBJVvF4St7WFvG5Un0AWyKpurB3ETbpTKblqBCBodecoWGnjGvTgxTEcndd9CZyOk1Ofd0CarfYkHvtGdPpEJ40+wO6tvfNxhrLEgJojIHEN04WVk5IJUdgeXfEDvL3i7wfUzaIXZRJF5iRfw9FwaicArxlzJljqbawokuAinqaleIIX82p8q8PRwP8o9yMWgTFMGr3pzYte5nJSrhigpXY529a9NJle4H1A3wN7eI5xmZkt+NrI+KaFmKweswpwQz+OmNvtUa0Xw+rXhVeFzNrS4e3nrlDEmzHJwbu1sudZ+k49Z\\\\\\\/ce2leQsddKilrCdI7LJzDcnbflexef30NEC\\\\\\\/H06MZkW7K5Kz7BjPvq8f42J\\\\\\\/7DT1HrFGb5ril+lMv8PV+4UGZb6R3Q984xVnAFQXzCFx3Bnjxsqgm+aq85m6digbk\\\\\\\/z+iU5u7w0vu0N8d+pAxNPPkdsdOIfBJEpRJwBbhpkUn7Zm2SL2wM6YXSD0pZ1Mjfg4J65A4C1PH8vUycPkLvx2iv\\\\\\\/zzgjEb8GSzMGITcm+0r8YsWIN\\\\\\\/HsuhXchFeqLW7p3+JLTEoFy\\\\\\\/pxaFWt+nblJpI5i72uDx2b15gh4v\\\\\\\/O68eaNZO+HoNuRUXk3ByQBiiVp7UyjflkPWQ9OPmYpFmAwYM6c\\\\\\\/ZAD3qZuTkzNMDRyyom8C60CHa8vJ2T7UD80dhg1IoYBuAtfDk9xemUoTTpHNb6L71J4Q2GqIV2QFa\\\\\\\/IgqcTrWJE\\\\\\\/2UiM9h7pDvLQLvLxglgkWeZaOi\\\\\\\/OJwQV\\\\\\\/+jIPXWy\\\\\\\/MnqCY64\\\\\\\/cTG5V9TbX\\\\\\\/e7xQweEr+mPQBHd4CtzTNL0s6gYj+a3XSkkPMbg7pRlX5Lb+zl4E\\\\\\\/oREi8MVF5d4gmlFgMcFK3tyhxetWRuP2CHC38AGD4qF5NftoCEZYlC\\\\\\\/nDEn+06rWQNz8yub+hNYach6baNv8UjxFcskUuDob0xbnGK1A\\\\\\\/bd2MMugu9CAbdRv9fjKa5KFShA\\\\\\\/P6UpbQPhv\\\\\\\/KIA2PScLiVr4EhLEH0gpU2UhcBY3qUdMeHLbtx4hxFqg4qzDakkkwrmZXWz6jCTehVoQ\\\\\\\/Z0gTqOg2tnLtZROiEDnzL5WOJ0Z0G01D1YCLs7udbvFlH7Z9r0riYJXxnrweZaapMa8GEhpnVdTWuBNXDAi96y7JQj06ahS9VePWYbjLL9mxYgSVLphnkImutHDEwYTFu890CcJGoWWit\\\\\\\/u10oD1eb6JyZO7fkkKXnLaPw3Zf1yn9P7VnLZwjIGeCdBcOo6sVw2c0NVoBYApji9LcgOe1IJyMhulv9bhv53L2\\\\\\\/lOCJiLIt3wY\\\\\\\/3muGDxx21JZ3SNtiGi0DUbgT4AvM38l821t28lkYPX8Mb8Jj878AfWIFTuVViEYHBJyBnozUGu43MYwzTLY52txrDtPm04hqDNFTkxGtZx\\\\\\\/RD+Nnt4eH43VrQUE3itm6gD6Ux1MqOYPoKcXbSIUIcLGwNcwBNQg8bIz5fBFNDDAbtIfimEOLgdRN6KS2E8FG7R2Aawtsck0MzR7rZzTzaKVijBXeuIo3PGs2CjJAsIq3C9ju5rhkfmymUgEGfz7qwZZxD2TRePlfVhwuz9S\\\\\\\/6oxpVW2uchg1xzH0R7mCIJlvW8dJMNJCKGERReCl9sqCn9WY6rFg+c1ilmKeZQX\\\\\\\/ZR\\\\\\\/5umD0bUMhDr96kv3HJIIbKA3yEXI5ydkq3sbuNDddegaieMMTDO\\\\\\\/UaHpOXbkYP2oxLdJJB4CXzaVfy72ni01RKo5goVRFTSvLx2EpghOwzItPnE93Htc8mK1koE6NgIMaJhnxAUQLF8VIh42TqOZ6v6wX0UaIck3OhxWFATr9gBljWtTJbJhFmIpbyhPPSg5dswnQ0tky5C+ka71m5\\\\\\\/qpxGlnoXm0qNgdk4JjesCebpGWG6J9RvAkW8oMqau6Bmj+HFJd1ncmHpWh0k7ZPYPDBVBawW4\\\\\\\/dnMZ\\\\\\\/qJL2nK0zF9xJER3olGrzf6lA5XSCRRAX1Ng8S5IhA3NSsyaWUpP+ayQYpFCzsKFSSpCr8xFtnM+AKHiP60CkXjWPYjwHdp43cZmVfeoK+mmXK5lEO59vDjmfwgnIFqeg387K4WBevZEqL1tlL2ad77QgftmO4L0iSYRqaHLtY53XJegcRQ1k22PihM0UyXhdyz\\\\\\\/SEkEFJp6ceyyKws46er7wHZw\\\\\\\/B\\\\\\\/A3VGtO4Oab65VolksnKhpR367rYY3DsMjo6KLKs51\\\\\\\/I4o0C0KXk5nXXHRacl8VjSXAlNr3OEVAxo01trf\\\\\\\/e+UX1jIx\\\\\\\/QCkj6mHPB\\\\\\\/oMY76EHPzRdh\\\\\\\/zdeH+6uj3ro19My7iyUm+dnN7opLO3cPQ0faxXwaO8xqSesYX7oYw0QxVSK\\\\\\\/WTFi6iszM3r3\\\\\\\/62l30EVx\\\\\\\/hdTtMNEOLwyL3pHCnMSzTSHTeyDf3ZxbfUh4P3m2KDE3V2p5KkQJgPBQkuzb7PakTJhOas0bN3Kqsgn3nW0uxklBHflWJ3N4P1Fdp6leapGb3uJSm24+BlWW+pk+j\\\\\\\/GL7KOtL\\\\\\\/YSzN9EEyhDa1ES0T1IP0tuEkxjm573abczQdZYKAkGD2ZaRL10PvUJHuehimQDw5i9wAquRkZVIesfTOTm1EufW8GoPUGawRtteyzmvtGR6B+PX18KuatwaQr+KUEvzpyJZvV5uquRPve6dsJMVrJq5+bDBk8FhJ\\\\\\\/x+e4qvgaSWNGMYjUzWSEBrIB0RmZLnfv3bHyjwJ16jVtNSzN9SD9u\\\\\\\/iGGfe3Iv1EMcgn5uHmR4HnyK9\\\\\\\/WfartmDv2hqZCFPMl\\\\\\\/I8EaT4uJ9HXCzNOTnyuTYDbtegAys2rfP8VSjCFEilapKN8SjWfsXXjD1fuRFJEKOcmc1I+nMO7HdjjTNyiVolcJrUokJ7cfJsBLYLvO0V\\\\\\\/g0xsdYFCNJbu6zj\\\\\\\/StVml2TaGfPdVqE1mMP4GoJcMEbSKUkaxYpe1JPsdWM2fBxERJOBAMRvV4kdj2041MECmyh\\\\\\\/8i+8SPwzNPo4KjzqofJPajDBz2OnjC1PaEuVvsyzeZpmEEcH2GJ65wQsGJpG2rD3WjLKbHiz6m3F6LGiRpDoli1Ttc\\\\\\\/uwV7toZTs0rTDWhNTaU8Pqx4R+gYa3FsBAjcLYiPyQtSN0N7kppYtLnmnY4Bxx9NgW+7NGKec036pPjK1anO0crji0Q\\\\\\\/+IAXT0C6vmyA62TudRbVxFpHcR+1c5xhLs+M7JuYXJh39rqlTAGbyymCS8lM\\\\\\\/wfqOu+nbWp25EeEBVRpA3Cl9NzcW3ut2yry5o1s13E8I09dFxC4zZg9qsZEO83nHwqBOgPkP550M+ANnVAQSAgns45UOLDNJQKrI3z+eL\\\\\\\/NUpxNT2vS6kO9Xu\\\\\\\/3jkMaNDsioDydTp4eIem39g4rnkAS8DJkC\\\\\\\/t0rOOVuiv8cFhbeW2O17QazdE1x0hOigbvisnXt6\\\\\\\/TdGZw0OPhyj0S3ruZRNcWpDTBWJmxk1SqMRBw9oGVXhonCCwuN5i0iK4\\\\\\\/VUv+SmNcv6PqkckVF8SAUwcQBtozw1dNiVFe4ZG3OJbO58hW2s6rnXBzIlqcmcg9taOX8dqD8C+RthHMh+ogpIo3Em1p4e3suOzKk5Ym6Xe7wLjqcWVQ180wrtWRl5kXY02KSP31wRcEWMiWcFXmVGZ2X1vxV8UrwVIlKJHHyHSCcgipS0VeGOc9KhlI3266zaeEsB0nb6Xz2KmqBTUjruLH3ckmw0o9hfRk1dW+0KpgBf4A2BIbf4E1WdRK6x92GdkVSskiqN48CXNqyBD41ET53tzTIbkFnxGT77AUPDzv2\\\\\\\/Z3Dh700tLtOLkE0lO6rdnsptotn4\\\\\\\/HYJWPw+0NWDl1LxmzFihF8Xsp+9ZoGX5Qq52+PMVA+w84K8Ox0wFoPggyB8\\\\\\\/3AYgsAClExB\\\\\\\/tNtM\\\\\\\/Xn0zUkdxwLD1yApkvTBHk2IHNrEPMc3xW0W+6QAtfGS6QEZSj7a1Eu2zYtUKh20wJPkU2026xa2QDKw16\\\\\\\/jV69Kc0uYoLnbdJvoWZM8xFEv8jOTCSthYUTLoqzdnd9FtRNGbEjXTx8LCLpiov02YlSWXjz6J\\\\\\\/eaQiXklyIejKnlWsUCcGm3eo7h+LIFi3XRy1jDaWRvx9DjWrvK0AsNJrLOuS6cZ4tSJrKARdacwM8fjGOBvnRJwi8XvoAYkFA+HKlC+oJH1Z+eF6lj9QtMDKefmnox3iDIF2ebSSm\\\\\\\/WjX6U4IaQIcULvpNU4PN8XSvqHHRj3sq\\\\\\\/PUByhmOg\\\\\\\/pAYqtNcpk3vpDJ\\\\\\\/ufGUz\\\\\\\/PNJpM9OiXG6PpncHi67CN8qgSu0XQYinWn7O4vGe5Vpf4KQVRmVWqBQV7Gt5LT\\\\\\\/NyBBBmcmHIEMgphI0MpzRHN76rLma4kLJd\\\\\\\/jS\\\\\\\/IDnZv24NY7yesTyn4j1zj+zZwceQiy4JGDZMCOPJyglSCZVYxzJ\\\\\\\/C+TZrzYr2mHqfz8pA6JOFhTTPXO30+Kp+oiAlxHdJF+VhtOc8Zhn\\\\\\\/w\\\\\\\/TJ7MwuImFn+EUoUEOjNQLkJq6MaG6NHDFetMh1W6HnW\\\\\\\/gSmgGLhSKWFeXSHNXDe9mrIy31QD936flzHgJOey5GUIhaF9vNAkPfwMKnJmDw3yf+4bbHZlGwiSGN1zA1PUYBW6moU+u1eU7Vojcyap07y723WdYlwFrAYDSsd2ECs2D6JFGteLS7Hu3lsQ9\\\\\\\/THdSWKtFF19Eru431cxfrXDOK2LJBLqMEo\\\\\\\/EarqDtn9yszsq10XN7LPph\\\\\\\/dbC56XzydlaMN2IlhkJ99VE923WV8say7SIKp9ddESWC3VvR6EkrKUBNcjwcAkL3tx6kik4DVgDoToEz5T1FRGueHu+aesMXe8bznTOO11WuGjLFSE1d\\\\\\\/kaeBQmkUf2ZnnmZDzx+F3iV\\\\\\\/DTGh7mYL8TZT6mfh+DgGOfcX4W6N8Dp84dWjRHhYfJ92hXad96RpWpnGiwXmDf6yAtn+dyxkzSZdN1cxtcV\\\\\\\/yhXdEk4JlqZ43tL5c\\\\\\\/\\\\\\\/3Ch+HTyNGNIjumCULj+faJ4QuRaqnFDRtzsD26ALsb0qiX\\\\\\\/9I6R4k1xWzNAyTsU4yaTqXGxTPh\\\\\\\/rRsvFRmioo9s+lqRMHsJWj\\\\\\\/biEp53oUcCliY648vdqwUesxkx7q9GDRenZ8GmvUDv4iwla\\\\\\\/fXdB87M2NjW8kgApXGwflRbMVyLLKd4lehyRrjzSClKu1kRJ1DyO2txG94KuoqNrqjWft4d1KWAulRckT5\\\\\\\/Zl06lnF7jvoJUAum1aGZ9WZGOwrpYEdoLMFoxC30FuET4KTt7qdT952TIulRVBuA4n4PpeZBs70k97dBKKGOWxOn06+UBm\\\\\\\/wA4LXY63xD6ft9ZUrbm1RlMzYJHqP+V6YMOTo\\\\\\\/D8AZVPRpKdaUmZ\\\\\\\/Eo8ycIkhY92MZGXq5DsCixOd+Dzhg3YE\\\\\\\/sKO2Rb9EbaW8DaFrg3H8oKPu8XGHhJDWYpDXbq+6wLpM9LdBrcAs25yowJ8JXRxjaC4KOE5+HgiJDAVBwuk3xyAe2WZan6uj36+cCyieJjzkqNAcFzsG2KzSZDjf3IaS37pX0N2hqovv2yB08vQCsA3oDCcAeX9Ijt1PwnO8hT+8fBJOQO8J3z7rjytlYzVP6Gp9NMO6JGiqUZ4\\\\\\\/+1QhgVOZvVX8aT25xqymWkmB0lcGgp2l7YJ3+6aal+22cabGzxvFY0eifhdC7\\\\\\\/gWobqYT6xybEiPEee5+o8loepC+X23zVHns9wKzcqew6Q3VialEYyY9YiQ8Op4oUE2vtnRQPPC0UrGpI\\\\\\\/lO8VGYcSia+W0J1rF0QmzdhD56RI\\\\\\\/fhSXa7HqDICvkXhgdv0WYWcvZAOGLOLJ\\\\\\\/pgwIfM6\\\\\\\/qc01kwIcxaUdDE+QxoYxZRjn1bb8HrhscNky9yQpkDxjzf+4vIH6SwHnRQ4WDhTt5Gm3y\\\\\\\/Kkk6+9XdYYgbZtC8nLHmae1s9TYg4zGENc1e4naqUux7JMfMcbtn0bcVVrEa1F9vJHemxigbLdbv42B+hf6ybfaAO\\\\\\\/sPj8HTIssMRB2hxg6PqAA9jXsTonTbvOVgfdub6R5\\\\\\\/BMCk3k6gZrA9CbYLLr3Ge4C3N3jCvuF7fEmG4qwev9xJMWNOW7eNIlhRLbJRcVgb6d0N+nLdoQ5mS6WB9l83cCfKirhf8wp8pNAUwC9g8z0Ow\\\\\\\/YfQDtXEkeu\\\\\\\/syiRvNwnaliXH8OB3HeBufEXxE4P0KVxVugSLSTGLxMaYBKSkI7Hw3L8wEc7C1AsEnmgPKzA4O9BSyLPIePHCWG9fR9oFsUWoxBHUOUVhWdD56kFVvOOtOYxa0\\\\\\\/BxRovlKm05l37vKI6WBKdedenygWdkzJL\\\\\\\/yKv9mxPUjXcGjZR2SznMhOcdDsIsTo9HUzLcV8MfWU4m3JWhPLgkRPekZkKaBYCC5y+1i99tu8MEloE\\\\\\\/O04Ntf48EquqmrXpxRnRBdoUBEu7Ir9te93G2hLjq3nUSHoiZPX8\\\\\\\/UbJWCBKXNyfyOb3fCrKkXjogp5yhhOavggNbaxp2uvfAfGzEGQLb53AlmKfeT4geHamWBbaWeaH9JdSnoD4cjsa6jzGHGTawkJaCZQBCAUUMXihAT6QBybWeDwF2+DD6nykrIMbN7Bw\\\\\\\/oYwGgelOQdR9zN\\\\\\\/ZpAvmbtHA14k4cp53S1QJ44ipls0\\\\\\\/Eeb+vwfEjbNsTWiiYWKMQ3FWNo8ljTLgX7Uk78algd9Klq0hgRVbkfP1IcBk6WFOmBNGovuUiOJKwF6XPB+U8dFE504Bw4je6LMUIxFYkzh3L8z1qud2DXPZvfoql7Elk1q\\\\\\\/UoMJCu3klX\\\\\\\/b4t53U+f99wuasRi70CcIhMHXwwYCsKjgy7B1y4p0rMLZizHnK9rpLLd9kiF9tIwixWvLSoqhRm9wpUSPelfBw+9Yb+Xl8EHDm74Xs\\\\\\\/kqK\\\\\\\/iul7Y6tWjlMaZEZ8m8D42ad2yygLvLlXtoAn7pwytW12zmbguaKFBMuRCpfI5e4+g2\\\\\\\/aCeWRqF8xkPvzvw2z7QGHhA5iU+oW2Aeku\\\\\\\/bLUBLAfwBodEb2L4C+hnKsVyH1lY\\\\\\\/b9nzChIus8c7+xvODKsAw2y4H2nWH35cUZVWhxgex1JhEg03gYQJ5OJ\\\\\\\/A7MSuby5Y+ovHR2eNMbI4Gc9jnENP5nXqS5\\\\\\\/6ITpvk9phfkmkr6QUjGkD4OxXDH4Z\\\\\\\/D7yfjdSh41RdQ+MXwntRBv6OmNtKR2Ru+RANU0QTWsi5aTJo7BVJSMO1sZFbZnlkxYp1Zhpex83xzfHQTtjuC647V3p39szMGXsreImW1oP4zlVQfBPOSFAjKsHNSHNaGlIFZ4P0YdBuJ3Mpv4JF81iFHVUR5kuSfSxs2Oz3U59\\\\\\\/g5\\\\\\\/cVN1z3QnceW\\\\\\\/0\\\\\\\/luhghD3qd1NgQFvDH5mqZYNY06sG\\\\\\\/bi+W9f8bLluPJxIb5p9Y6IoPpVQcp40cPlKyDC4\\\\\\\/NIytHqtXY1p2W0h3Ed+vOdHm\\\\\\\/1sAIaH36VMHGEgEUBMcPSm5ZlO9ZKg8mFO6QCAoIwdLKYHRJVKHvASTEQcnX1ZdVr\\\\\\\/9QkuaMZYs1daQ4PVhlOHXCvlOFc2lYf6X6foNrWY1i1+JWcND+uT7LeSSioDVsqeP2Tf9CWEV\\\\\\\/gxjtaNXQEisJraoREGmpoDCBu1qsFKYXrc4aJJNdhUd0RTq1wgVEX6jnFnr7crVedCwbZjGZGD2LPdgNbcj9NvMdu6\\\\\\\/E09YB78kcvTeuhhMtEa\\\\\\\/UWshpgmbDkHPRau45JXFpeHn0PyL\\\\\\\/xCJt9U8oDQwp26b5I3DcKkApJQYKmueUnFiG9CPbQlUo9cw5cYb+Ds0H+utvJ6LkMOuSSNPgmzkyJqPCnzKoTZ\\\\\\\/jsDlK57ODtUv8p59GIKDqO3WyEHImd8Al2pMC6RSD84vp0kDpP4adgDROiy5U7MKVUNAwgrRic6wh1o8lj0seUe01VnLqK0knB88Ks43bOlCz2gQ\\\\\\\/SGkUnkCohodY4pqCL7D57LaJgs1O2XSvx8R2J7MFNfSRu+CZd03M7kiv4jyr\\\\\\\/hkmLrCrlxsWPz7599GBPlrNTdwDNNiAjaBuIM8nCP\\\\\\\/6htauApa4MOTPHMyiMab4pUUVn8klA4CiA4uoDBEWFVyiHfhD5ECZkg0tkHHKtuIT51Hf6MPbQW6iw7aEYYBWL\\\\\\\/IUZPu\\\\\\\/F3SrZMeLRXuof9IA3HHBrFHG0fMVhwQwsgo3B\\\\\\\/9Ba80wLnKLAmCC6bijug6+5D6EazeMQWgdC5biycIDH3LFtGvHkDBhH\\\\\\\/coBIt4A\\\\\\\/Jm5TBnxldh6A0JeXRwywdkiylp8o0gOtv8sQQ8BW01UIKB\\\\\\\/Qsx+ze2dj2cpMxZqEDo5eJ7xxJPQSK6NeaGlGz\\\\\\\/XBy9KG4JXz59QVaJn4t331U9w0Iy1ejYYXP9Xd4VMxyBsW0B+pxrby6oeLd4Bm0IHj\\\\\\\/wTa+7ElnV21CMtWVrvInuqOeERi+yDU8p21Bo\\\\\\\/npBiz1meFJnN9C6gk1XVAJRKRFmut4wR1nz+JqiqMKhzqZPiw85KlbMvCe6WVY+OaTiOv+vjsHjdkMCH+7AuIjGM4VnOWrkdeb6JsBb7Tqo6DqKnGR1Wz3Y9IzcJj0bnXtU37DI8yngYmFkuQjrcapeE0IL5SQeEMPqYMsk7P3Vej+dSvSeUcPdHxo\\\\\\\/adrGFCrSsTINMjj8ylUmP8WRzvv32QbQtZUTGIfiI0QnDeib7exkxJ9TW1A5t91ICSUxEzMy5D+VtKQwZEDAmU54ZRsPMbcXbzCdqlLTJEwlkp\\\\\\\/tFr9yROoO3yQvdqs+6zyanXIYmQhjv15zlunumayq9YAwEW2WC5EQPQa31C0sKMqXWJqbaQaj9cFCBhfgwrtK9n7Kh1\\\\\\\/wncVLZffZ0atNLlkAr2vSlyoLyazQlO24WIsw9poyj0KI2bxkAqf5oe+echX2O5w+7Z4tdGyJImwGPMmgd2tpa8G1Jsu8cHve0brrwKEyMldrh2Bw+Ik2OKEeDZQ8YtRbhYuZoeDJpm3XcQ9DeASDaGt6GAqlV7lyc\\\\\\\/HaEPN2mWC1Klw7tgG3JblYNfW1PCFMAHpIlDthYTXgXGOgnOObMSSarZb7VNByi9A1rFVslPNpiniIq96d+9hc99jDbDrAdFCqqj\\\\\\\/2Y48HVTAXL3k2iwMVriDomQjXVb\\\\\\\/6Le55PKsxDfNZuhPzn5Bit7hW+FztZoEPtZkwMhHNR6pt3oxxYNZcZpdA\\\\\\\/\\\\\\\/Xy8dR4nsY2JHCHFalYXOV\\\\\\\/R6Zdk490X0iQ5loaEguwCZ7eiyZh+htbBPlIGW+0zeDVqxnr3\\\\\\\/9IfwC8tuJRrw92taEZz2Or29sW748ySndXB5t6IkWw\\\\\\\/R49q68bPRzE42ZmeY2IBwnxsZ2Wt4BiygtjswJj+0EYXhlBNUpixlGEvTlDX+zgEFHQq2TY1jG9zqjZPUlag6JgwtImcxqckVvd51r\\\\\\\/KQnwwQFbFmcZAMCX6q0slylM3fu5BuGNwTQqTaLdy3J2s+ZtlfnMG5naqdXiIQ94vc7nrXDYo0S04\\\\\\\/6cazSa3a6eHeiqihLHHwUYZJy+BmCYcfIjgq3wdUitPQ8M8rtXxM6hUe2cA8504tV\\\\\\\/gwPGW+TVmlGjazZi2k6rfIFNwbNUrAeGHS68fcY6RXqZ\\\\\\\/flEUGxbX4ZX6vwtvGlDo+BFAhr4QOF892enTA9aMBYkdqrdwgitJmKffQw45p3ET8pMdoQH3X3qirB4cdeAH8oTeHSLYVK1f\\\\\\\/boKLzimlWHmHGD1XX3pCcORLM2If7jvkoR3khV3Rw\\\\\\\/QuyyThr\\\\\\\/2dDfHyNjESSb+6SSvyOwhCHuuDLt0Oxz1w9F0jhNfYUxUKFU2q+N\\\\\\\/UkKu8lBSLlGGJWXTSs86utAH6uIi0DxZXCAfKFtlwy5KpADOpz\\\\\\\/5vFBRKvx1VmaGMPQkMIvci73YDi6S8p3rRluzw\\\\\\\/ppubBgIoOcT6wE295T9y\\\\\\\/+gGX4CElJ\\\\\\\/cuvVeNIJDbxpOGVO1Glcm0G2JcEAjUYEwVkIDxHQp2rEHIQEPKPnP1FZsfFCb3kGD4otXXHGhtI3kEEnldxeuIqm2NauCUYKohUVuST0QbstynNyE8xWXptvFu9Jz3wjhOt2OwDBg9ca2OaIfRtLRDkFhrB3RDAYITFUISnIqpWmGKzi3qWrTQ6mzMFKNZyn8KU81eAqsyUXmt\\\\\\\/KqwPrEIDzoIuqvbyFy8z7Vl9+NAfbJ1pW7FreyMAeV7FoAShG9mnXcU\\\\\\\/HKAFuFwniDeh0lti7JqJmPQKHgKaE0TXsTGUv2QEG9z1hKCm4H8sfobskaPnLLNZGCMb+3o6DyjK4mGA93LT9b4LSTlA+uLVN22EiPHzupF4vSKltTC0vy7uIQ4KBHXHwbpGb14DJd0PfE1fS72gW8ezybi3KbYwy5dn0Hth6+dL6SoqtShV4zvnJLwupslBEZajzKMWqaNgzfKMgcelMn1ctQ\\\\\\\/3cXeSRNMNGzHs3JCR6z1s+3gjmkKzvxtnIv9e8LPaq+PH0dOUwY7\\\\\\\/LDb5zFbx103djckerhLUTZ1\\\\\\\/jkG+dYvK5+C1mSYCa9m0F8bJ9r0l0iQ0D4ALLY0gAFnZuuHvdoeg4G3WXyGjwpjtIMwqv+VV2cpLOmDrQ4vAHNENtWaiw3qYK+wm0hWV9ktpINFgGCDvgVcoNBMbexQ3RDx7b3DoidUcoJTD2GhphJXzY2\\\\\\\/w8fW5AbcIB6BROgjBTw91chahQtoRfpUBmuuZZucLj7nsSvIg7m6d4Wp\\\\\\\/n1glLRK8ahZXb5yo8Mmv6+Q9vkwlsiNCg4u0+n7riWWEiehboVGfxIIC4VWJjs\\\\\\\/ngdhqB6uW6v+BVIoOupV2U6gxHxFhWTn5AbgShtqxWyMsAsJ9oTBMATiTXGbUAbOt6rh7WJzl4XPeHvC4+Om8PtO0j7mCuEJm+Bvxv1EL8bF9FIvT7a0f3ZO51lTmgnU2TBCj71jh4odR13hB+iP3AgcJNjROixEu25HEGx+UzFm20vwgmL\\\\\\\/BuyaJjvp8A7k9KOWtXRFR\\\\\\\/R\\\\\\\/SVLqhmW4vjyuigtXGA81nP7zaKU2DHP\\\\\\\/3\\\\\\\/hCMkTgPsHtpExkVPG8jIHkR1E2kC1GUDskI7ZQvFqFujZ5Zztl8H6ewjs23enJ+e6P5vxZR62fhY6Zh0XRw6Es2Xuw1OSPMIyb9aHFlM64u8dNKZ3wE1ZOJFd9cL134b4jgR0J45yfe8OprlJcXu1F9ejsPOvuhgoYRK+sUR\\\\\\\/fFHVhnR\\\\\\\/\\\\\\\/uQhfdyXNtwtwQAZtrykwqRIrYX+dxld9sUOjvymU09XiCEiFjlqLYol+2pXWy\\\\\\\/2G\\\\\\\/ijbFglydX7GxVaQxm7tPXUiRLWUJ7d+RaefZpMWGnqB9oo5IrefPFQqO43ubJ4zjo6G\\\\\\\/DGLO5vX4jSgTxU330hCFq4bMnCTgHIbDmR2n8f5CFzULRDhTk5YntQ0sw6Cv2y6KKo7fC9S7fFCGa9\\\\\\\/DaAtvdGoDvmqeUL6FueRSZbRZQfstd+9H8rIIuuUUXfRWaJwVG+nvK9X0necb0QZkBvMpm86us0G+J1T5e+kycwJ+itnlBszdaojTOP\\\\\\\/Ja1k1GyOAPysSiqs1HLJ7kurswcZKywhFKJP702YzPIxImCNt+hHwSWbvFjHnZMNFr\\\\\\\/Zd75IbSG0J+WDMBpoh6\\\\\\\/6e4f9enJlnVI1d3IgUyhYzZmsOdRcgFbi\\\\\\\/OJcjf1ZwBGy6xkfhfTEdtl5G4tt\\\\\\\/IGHojB8LNHpNWdlRZtVreg\\\\\\\/Ok7Q52s0dyN1W4iMAGx4UDaArLCan50fEsmYQcyLpdQlcNyEDyPiU0o79OBOJAO2im4Su8Xzr+AWboWXRtEGIWp7o4r6bcskSdyQ2BBNkDdbhvaqmcwCKfCIthYzqndIPkyNR89XEwvVVXEVwyUZhbLpm0EW4zIY\\\\\\\/Xrh4YGocXEUWkDBjZRt4B7y9soSPJ2iIOoAOdE2eFTgghrIgtkhMevGSjEN44VZThC5Y7eOZz26DtK54w\\\\\\\/HEmXQjCf\\\\\\\/\\\\\\\/MttGONR0\\\\\\\/eqbnbbMaCmsz7zEt1OYRs5d9h8lSOUWUldA\\\\\\\/kIehw9dTY0UDaHgow1x\\\\\\\/TOPdSYuoABOVH7GUqzcuM24f+vAMlXnIUw3ffzj8KMX4q2+lRqvpMZEP+9YT9hBAk\\\\\\\/JqYyL4eM30QZpYgLj2kNVTpdyuWGeZymJIf3hWe4jZH6V12KPD8k4+QH1iGhDWbYRBTarGW0rU\\\\\\\/VeL+1hHpwb0XdpnVNM7sBvgPyLNccqxiqj7rro2Sw1mA0Xtl+r\\\\\\\/ZYHJ+u3C82vKBRwtCqrHDROd4LyO8EALOlpLVfTSqVKESLMsK0jPlTnoqjf7MISiOnwzFxBVQCndHI6WUrZQmqZLFSgRqIj9yo2SRQlYwrksm1FlS+F4P+GUV9gGq6wQqL08kNQt+4N77sdAorrJkuJ9lVBaWC8twjNlBtylyw0mpAitB1qGHr+rKg0aPmRbdqw7XerIIJh9UGks4sC72FUxg9YDeqB1qNQ8uGSvcCOqvTv+djm2jEuRPeVEdNETjRnuu1qa8Xxt0AP+Lu3cftm2+iH0HAvUAnWWgOEn6QZRmsUKpf1Qc57BkfiFSgk0PoREKrabnfiVIeydynfxrOAxXScD4oUiDmkXpJ3ADsFkKC\\\\\\\/zA3tMzGca\\\\\\\/Bb4Gf48FKx9Ofhg6WKhAXUqFxz5iozpbCsXJwwpb1SnozW43vmGvYr623C\\\\\\\/0\\\\\\\/niPT6aKl7jzKJ6VUySYXv4jSteC1WhgpYWAMtUXSYRhXgf9RQ7PSwBhhwdd+hYSz4XFkzmGAUVW8YHdjAMQTkfyOyiH29oJl72HFULzaL60rLhH6KRGEo6lG1D2YMRysFZnElQRYITTekJCq4xaPjles7hIwXsZMv\\\\\\\/RrrHBkyf2RX7lxIFkkxrcnORp7pm5pgJM0gs9n5kPKiOKUJe2LK5kQDaC9GhgQDd\\\\\\\/PN1JiE5LmGlpPDsj9wOKC5M\\\\\\\/IiqrwGIkLb\\\\\\\/mNJJi7xT7tRMppZeW2Lpbona7Zt8ul50zSGaVmcPIsIy7DC1VWHPFOwc+KYcM7gmv6A9dXI1xRnRJXsfm7Hl3JNip3iQWDfcoshMD\\\\\\\/0VHVbedjkSBryZa9RhQfLU2MshCggKd6AKPPwze8ckqseAalBPXGU41RQ3mZj3o8qUbS3f\\\\\\\/Zn\\\\\\\/4z+3qOmMFSFpOv66uuLY8RNvivQPEJZrQ0SFIlvfm31oZeGV+v7VH4JMuPX1imTQjyHDZaZ9AhCbHAt9gGdUh+3ya2iJaZMwmREaHXW4nCRD0gYMvYrzFa8LiRwyd1P8E28yHXDTuMKJReK+R\\\\\\\/7YJWSfArRw3oSaCPWQd8qw1hcGo+qZqN4\\\\\\\/aWwwHV6yk0MVWc2bIq4m1sMh0\\\\\\\/DjyJshHZhy+9oPrfBvNlKIJalCsBXiID+pysDlYW\\\\\\\/utfPhSj5K171gBkwJoRSj5dpKAjSYeIR0bKIJzZrUEyseJe9jesfQ40kjhU82O0N9LodmHj2\\\\\\\/QT0n\\\\\\\/SxmQZh6O+Zv4WEB8MKFgExCtQghvZ5pdalKqn195CXeDNm+uh7tPkr6\\\\\\\/GRt2aqr4EkzCy2SVy3Is5bz4DCmQ7iFWkKq5RRBmO16xn+5xwhSOYIbgN76YLVXah+OtEs6jVsc4vLN9GkmYRrp+7hC\\\\\\\/EcnDqnmo4KHLRkqVjD+u5+g\\\\\\\/hVpClqrooQ8\\\\\\\/Ajiqq4C4tc0CoABg6qHYCSGYA4lQuvqrVxyNm\\\\\\\/V+GX3ofNlNat9yeshCTxtThwq9KxGFgVr9MqVHyXMOu9nnmgJWg9eVDY2VQfIwDwS1FQta344EeQEsGN9ydMTu3eYGDURlYwgWNauDRgpVOiXcNzyjFQly+B\\\\\\\/TxovC+7chTX\\\\\\\/9HvBLiOiRXNic81THE50vZ6ljBw5yhbQanM9nLa8rdwAeST8aOHhfL9c+4QAfdyLoVOzfuSwdnAI\\\\\\\/WMQjWcNq30+NN6w9nozMB8MTecQSeNLz5kOojn+SqCr5WpKvIVyOW6RoOU3oFzjwxkrPcXx8pZ8Jx0P55XPYn80c7m0Xn5igCVpsKkIwvqQG\\\\\\\/4uvzBwlCRY5POqAeqD51BbyngeVG1Wkw1nlzaQjeMdCAge4q3E4qvjzh2SacK1BA\\\\\\\/4UjudyvgRWup1iBqhy4UXgUNKansk7dkqkARhIPJGqGg7ovdShaftmkThIU2PktbbKN4sU9KIOHLkHFugNWEeEMljADv0jvB\\\\\\\/iVf\\\\\\\/JMYJpbGG3VXHbJUlrjLS7yG4N8geanLIoXAbJlAK8MG8PS64MStlnxYGe+9bTFD+R7uN+jrUtqFXz3tnDY5lTErnlF7hSNJA1FTLgVT4\\\\\\\/YRgBkPTdSxJ1DKGJkDL3eSfCKNlmM5EG8ax96rkisvsjgm\\\\\\\/ZHeE8YwsZC7SzXyPmXXfkV9dD7PCk0RJGtZMJEqfi+D8GLlgXIW77A83Ipmz2Z7DQXJ8oNNb8skRjecq2B3gCAeRH+8zooqNvEoR5d+f+E0vTNidwLpkXUALNFg63Wk5UvQ4L3WErGoLRRMWfxuJzdQYeLq9ZyF1msd2ZnIgpkvb0X\\\\\\\/Xq+s0lYtUm6DhqPkT7FjHrJJ5xY2oOziYeDyaTzQ0e7eFXBBvBd6xiatTlAsBho1sZEghXEnnYVcNgHEA+q+6fDbTFc5LeBl7RIg4rhRH65JQhQ70c6CivGUMVNbANcoEhPvGXdLrT1QSO8czqUEb4WVP3t3Den\\\\\\\/\\\\\\\/o8BQi+WlNadv7jmu6t6Df7UnVkRDosgOKwv4F7dhUZ5ENJf5paIjov1NDQaVvJaThVoxsWYldY2WUMOz9DPf+zNvYMQJ030X6NXdFs3jixzvqfl9BH8TsTFb9V2ynP8VJp0nFnsyjljDRKu0Fd\\\\\\\/F5viduh0AuecxYaFfmq1gIvqYOv8phXH\\\\\\\/4y90aSusgpWaHpQxtqIZ1Eila7AJewf7XgoOTZjAMuJwjWpDN6XBd\\\\\\\/shmZh5syW0KZrFliXwBPMY7nEFSJFueB9IARXiP6pLaK2P3xo7F\\\\\\\/cIvWimzvx+vFhY2Vdf3rVLuohHZdxargX1otz+hYDsNYkSXPmAXleiz3gPKUx9lAKDndPl44B4ioS0JtKDTy9BIIWalKeoRP+yiwtBzCLZkJvg+f6u7V1wR89XM3Z6K+52TX8V\\\\\\\/FpeoXiYwZwftubfW9pmOV3BnurTMJfWW913QMeyzJTwDZ6QhctzS\\\\\\\/QGi1R2hLi3Ju\\\\\\\/DThcXX9uz1yNKTxTsWDRbUvq3oXJlIdqVeYYDotY2aFaXbVZJd+gmh+MFwEil0Ds92XyoV5coWI\\\\\\\/lQoucnwW30FSwRJuBdtNiZ5w0VNv3jeD9FaqcVfg+t3tw6fhKzUIAj06oW5L9kV3HQYoUdmriLNO8PbiiCgQ11OcPlODdccrUAFjllIfcUXjJur+3EQ4WHSUs8tFpPjcm9xZ\\\\\\\/UggbLxvAr912SjGO47N6HS5L\\\\\\\/jn9uFHLnUzWbtn9LzPFRf7H0AxpW3ve0Jg5dxTY9SqOb8cKQRRAFXTo1Vz43xYEOYucieFUsczcxVh391LggNf2KyGVlB6hNCk4WL6B6OqsUHx1clkCNtwZE6VG\\\\\\\/bXNEUKU\\\\\\\/hdGhrSpktsvmOm7ExPb5ffFRcfP5dQtqWBaJ5Y8KTbmSKMmE0SZ9cKLP7+LpJrbwQAvfgPM0qype25XSax27Fzs5RWfkCZa1bpoQjxmd8esiPplj9oiUCK9AJm4P1xJDyl3woW8AkyLNj4GhqLKr8yaJQn8BpUcu21biqtKjkTMMsDEagv\\\\\\\/tIzCmx2GtgsKlspfNsJfmRGoFaD+he+mL+eNPHC9OG0JVitBPmBCdPpTwYetLfk4NKPt2jNUrniWAnNDA65Ts9DOQhSrKNdbz83vWiTf8DbdHzr4wDPpWo47PX9QJ10UKkLvRDzLE7rl+mkPAcP8\\\\\\\/a6PQUyavJ4\\\\\\\/SrsQTf\\\\\\\/L+IEenSFOj4gGq7xMog+t1N9EzMoMwi5Akn7\\\\\\\/Fn8+lad17oG1dRrQ9Y+ltiSkd\\\\\\\/bSfnjQnZqzxYe7MwTbE0lHcD6eLuYVgeKRi7SGdNf8kYa84\\\\\\\/Zogfhyd+E+uXlq2ISMGlLrhkmiAuAgBYmkLWeaqvETYWcBZDbeC0O6VLRr\\\\\\\/rZoZm0fZ\\\\\\\/pod2IWs6HpxJwlcOfyUR+xPYGn2vxr\\\\\\\/hiUdbPhX+QuPJxqQnDZ\\\\\\\/mN0qkohYd9xVkXHvvbZKWbek9EjY+Q6GHO2el9iHnxNwmcze8ikOmwjGEp5ZfBlZC8yQfohoXYSIoG9zcOsrj5nHj8kVV5aU1T+jGh6QrU4pwrhf1RQE5UEieEB8vZ2IbV9E4bD9mgxagJVgsI7yH+4zVebDMTCVmpEewHG976xLq9GGMzg7STLUN+ISHaqa5475qOQNg6cV\\\\\\\/NUCkuW25LbU5sOqs+AOyM1tx+P+gdDEWK52mZmLbJD\\\\\\\/HLqTYYnVjVbE51dLC4IVNhFthq4R0Dwxuj84uU6q\\\\\\\/kqLHkVarEjIvPn3pqxZZgjNrezo4YO2utBOO9SaccT9hvyxiP5uXoeY5D0jcjal8JgXLJXLhXPlR2Nl9NkFeCohabFT3ylC7xofUbiSw0VphNsliw4eTKkt5sYOYr\\\\\\\/2fVd+1hgFbcvSTwVuS7CXsAYtWIdEgAXdL+1DYAFtRsiX3x83f3FeouRAD2XqGQxPEwqdX58D8eTW4mXTn6CTMvVxu03Kt\\\\\\\/cRqZqlNs9iixfjqmpRzL9RgWLJcBat7OWjvb4ed6dKNcQM8HhM2\\\\\\\/mqXLBc7eyF0AuMcWuqnDfyWC26KZXe5yNGe1RWfqRCkVGi6LoTDTSLjsUHT1eC4fCEnmsh3p0wPAjlivVTGd6nfLXkWmtW4GCM4lelkwCIKSZ7mbi0Cfwv0zPa2TZsR5I6pjm9C+Ins0vE7WOEriJ6uTIcMPXZFH8waRMKPlNubIBrxHpXr2pSt0\\\\\\\/N5\\\\\\\/8YYUDZzMZpaqTFp64UTLYmZk3\\\\\\\/dK+rEmxNjP7J6ucy9lxWiANMLXVn47Ffwy5pfRfEd95j6iPDBBDzKF8E1Am1AHpLcwIUC\\\\\\\/xGpZRtC\\\\\\\/e+pi0phUybXzfK4MAIkx2ApXf4FDygjTQRxNOAqItNQelBtcGYdtVnjyrj+7Os6BZ+ZK0XjwO6M3U1tVSggbRqG0uP+UUW0vpA3zNAERjtpuaNgHgaxnXBALp+MJppmQpuRk09enDGGP27b8kwC1GqhpyH75hZMyvGVp3vP7GkvHtWNNE6qDgsviDUVGeN4habdq+nz7Zr0S73BhjLDbMu\\\\\\\/OQ+XsaPYW7he9zKNzWM\\\\\\\/W53ymWRmxp6q75izDo\\\\\\\/aCG3783s6w6bOf4kXdZyvUlVtHbmKmQ4J9ueL+bkvDXcwxe35WSDEQjCbTa79HQBJuaYpQf96I6LgECkPjddkSk3WsuRYwmEGv5vX95DtH5\\\\\\\/6EXPxPonfrGoAIFgPyTwcczzSkU\\\\\\\/VZbhQNRO6295JYuiBXqm8+ybQ3s1WVZDQGAAp1FWOen\\\\\\\/F2Nbtbc1Ez+1Y48uXWJVtbVzsxBdPkfBPTPL2EJ7d+G3RTXGFB6fW7B2vg6k9KsNP8FGRaCfg9KmOFrtl5UWDTQZh9Vg\\\\\\\/URPzU1slwd5i8yg45zEbBUdl0aI+u3ungH5Nj5KmC2xKbSkxGn5YYjr8lI\\\\\\\/Rh5vwp\\\\\\\/lDxlKGSqnfuDDN4zwYhbf5BSuNkPL\\\\\\\/fAFB6Mnj0Tvww0PWrlmKVAxw\\\\\\\/\\\\\\\/24BZea0oPHpYXBw8YOAGUptZRvccJkaPFqfMlyx7RZgVrtWZ2XnoCvQ+Vv8mN+V7K5jElDxtZbKK6+cXOq5avM2Bmz7+j0MgjSImt4AmlqbKZeFJn4q6mkC\\\\\\\/vAW5pylPxsO2boPiCeTG8e5TrTX3\\\\\\\/\\\\\\\/BSkOAVLhCCINJxJXkkjkqJyg+ZcaXOQ4++1FIcPVbbGBvz2bxIc+vQ9KkW3gr838n9mHyycPfWCOxIGoZZWYzlPONDxYTGwb9Pvs9bsvVKhSUSKjEDp1BX9zbIBAdOKiZ6HMxuCoEC9EqlPogOmwn081lkb1zZapXPn1tdFypYQtd0IKEJm0\\\\\\\/OKFM30Vkb3UFiJGc9UV85TA9mCns0aATQqtk\\\\\\\/U4IS38YpdnfPd6GqYl6evD83ujgL51RYyxs4XOwDzCoXZJIHQkFnVd+TCvqeM6NbhDv8x+0sHBWFB6tILbBBfQBbPsz\\\\\\\/h0j61z7IN8D2GIKkCUnycUxJOR8HHJvyfxtc5l5Q5Z7pHMs+8n5i6YjST2wKvdG+oAV5JvjUxOodiDnpFLrBWxOfUYzSceRPeMNoM+rhHPMFyzE\\\\\\\/\\\\\\\/a4h7a198CE1mFVXdcCerrZZv9nnj8AOVV9gyd+08WbtjZS5RTmUtwykZh69P2Dnh8Vj+KtACUkFZB4lsSFXFWSGTNab4hY4tskCZz8P+0ziUbxJKLBMMqiwWMvWfy+gqklDbyGKVu1q7TFLbtNL9rvGwfaHSRcICbVqUNRDvwSnhNCPCbBpBCmDGIQn\\\\\\\/5QqCeo9H3UTgVHMnplsesHJKuVGE29wVrTt07FJpdptCd3Mi3rK9VCQ\\\\\\\/cMrj1\\\\\\\/I\\\\\\\/sK1010Gqi9Dc721Be8LpNYxof3BNnEPxIhHZ6acYf8OKQGh5chCexVQ9e\\\\\\\/qhLQP54P9qkc8tjMi5CcZ4Kee19OVG9Xo4iaBP6wQtcHLTKFKc0iNnxq59BBWYABqrLFrEbJyMGbnnVcIXYGHqHJO5IsSVqffwy+0FDKMfurnRGsXRDTvp4LOJt+4kKPoz5+IKK7EEdxNtb90Hgz2jzvoAiLFZfCnwt37CTi8Q01iD3czkW2Yg20fdrGzwyVGpkg0TlgOVwgoStrWtNwJfRhV8zAeCVjmJD+vPpllCuq5YD7gVnGcd8OesLciEHK\\\\\\\/WEhUeDZq0qh65bFPsdKk0mI09To2DRUxvCSfaKUv67pTMaxEUo0rVk9ZfSRD57NP\\\\\\\/4RgcZEJS6M2dvdbpctz8aCXa+VMt6p7QN2ZzY6UgXAUogJxWVBTwRa7iJUQgtM4aUVKYfP+HqO0VO8x+3wrWLYPS\\\\\\\/Bk6Gmie4t3WnKXPfhML4O6JEZW6EiWqSw2qI0W+yoy\\\\\\\/xIgtzRVrgyVd\\\\\\\/ezyMjM7PVgr6BTTHLu2JowRxQnorUFR0h56LXVW7CZwajKtqW7NNRGCvPjqCYUD+T39cb3UBfdtNVTjaED6PMWp5WglUJ38xLZRqGLMPpy1ilvijrOUgQ75GSK\\\\\\\/SCeGjHxXkMYnwMTgydqrWgmnDYEc6iHwx9uqS2nBdatDN5TSzrPV4Yk5j\\\\\\\/ly4lSHzjb5J2hLhmlj6k553wwllBCQbnym1eyMDXCGMUNcMsgb0\\\\\\\/prx1A9js8VwrEqggaOaE3EP5pOEKun63h7XiMVrIMvN\\\\\\\/Eat5gIKYHJ8AvMN1fW7mlmMBLCiS5Z1KdJmFiV9iRd9D4sJPSlsTrgqkskNdOvLTYINdldi6HIUK6VG4+wD0KWu6RuA7XckSVPLlc86QcqKoMyUSV7J8VLC3IWedyK1Euki0BKYtGyG70ijnqHHaAvbsKXF+gNhAeCmyKhdPnCVK7KSm\\\\\\\/q263O\\\\\\\/w9BQByr8vLBGP61CP+WozxGkhWDqQzs58skrgbGL7riN5t1BxttWa98Nd++nWirCw4SpuLKQervdMryLnC+BlgtaH9Eo8p8j5akC92PP++LVC6WZu9jZMxEp1oC7z9Vxlz4ks4Ovdn257tSs7Jof3+hynzoIwgJt+b\\\\\\\/GsHXKoe1o0EHIRp5UTEBI5sQkNrO4gFzjp95l29XMr9CQIV5J9glaNMKCMcJgx61DcW0Ye4U3z2iIh8TBpfdaYoItl+rA8JG5KjKdEtybgjgQghP+Akbn8aY37mEPlNoJ9obDKeNENmlJ4Cupxv4\\\\\\\/wGMJc97+toswMnDw9ya7gcZy\\\\\\\/UxkNMDviJdPswOPdCjm7BJE93xl\\\\\\\/lFFlKjzbcg5XOxcoyCCkzlCmkG5t5\\\\\\\/crryQHcFfKWtskfgnECPgaGQZD+FkRgs4wqZ10uPRNWxLa7vQZtg+tbQEsUMtpIHMDi05OByruzDhdODQdgHXuCBDuNWd40PedcPtfoKkYRN\\\\\\\/wBabfYLs4thRLiX5Gy+DUZmIWZM5g71Xrh2vrNWsmDNgYvC\\\\\\\/a0qfX8MATrjVCFdAvwTC5ngcdZS2gYK2miz\\\\\\\/WV9fMLSU1s9Ax7gKKIaqS8Fz5HAz7F1sAgNNmdc0RXsckFDkcy2ZfmoqfTEq0wJmjQBe4DkEpMXgE4T7piZ1Mafnlv3hwMgtuOYAQgwr4V5V0wDETHUrDVb2OcS2yCeNYQqd4jdPOxfPJv6rEZAd8+Y08Ojs9I2aIU2z4Ounco9ZZIw1UUTc6caedIKeDp5J+lGBjvsCiff3xTaqn4h\\\\\\\/7TRzGK8odXkCgfQ1jhCBZOAjolyYwmQlSMQ+SOMfOKK1y\\\\\\\/MdJb8oYN11gSYXQYWkIReCmRbS7uVkiLXNTOVgLgP2eOK4FS8ISA8fin4KO1AK2i9ysVChN2seDThm7MAsbnJWRMoxJ+BkTE0LDJQ0tJxzbBvysYaliqvijtmA8lGEHLytmZpkyLFe51R\\\\\\\/+2LFyO7+yxxryCGI5gB4k\\\\\\\/+wJ7VfEISx1VWEYHCqmhS2t2niNqzJjYf\\\\\\\/x84G0q6DsSGxhMC4RyKKHCh3ocynEUhZUEsXc9gJGiLoWmZizKTzppQ\\\\\\\/1qiY1UoOo8sP2g88erYC0HV2FdkEyLzRajMOhua4Jtm1y1f9avqxPM3grTBG7SAr4yi1SkGsARn3USUZimSE3wq2gfhkii+CjGO6tyHZpUTISq+8xefS96S3wa\\\\\\\/OEiwz5NhSu2LmVriLxXy86bVvNpQMV6mLD5+ocVmX9UUcU6yQ6bH2VVvA1jHdVwclQ99ssTlNQNJPW+jODAYLvWPDBJv29hwBeLx\\\\\\\/9KmOzOP680V0me0Og5alQNiEkGkW3INwCvbNzdTIjpyg68KEplp1K9SXlduV02Pog\\\\\\\/HhEnmQZcPv+eUBXNYqqUImTIIWG1+UXe9qEs5Sf3rVo3aOeqluYUZyGwhv818G7uibm4fA5mb2NqgDLhJj4i4G6O9U\\\\\\\/vGM+4Vwyisjty4DPp0vorLWdDcUiMGn51LqAiT4Rt9sYyykEZw7gp\\\\\\\/FIGyWM4IXMeguGbg+60HkiCW+75bD0v0iSvWo7lkep\\\\\\\/rs563zRXo124W9NtBOXydww3TZolySliXaxJhaNLsxZb0kQT1JgkEBdWdR246F2VQeXhr4aNrZltpaOb7V0Z26s3nS3jUWKP2NM7Z7l7IArxOfcnlYqJFiw1IHQvEOp85J0sPOGaJSo4FQI5z7xbYbItfnpZTibbltG\\\\\\\/42OdK+BOqBkcus+E46V7h2uT6LQm68gKncOQaeWBToyrTfzrK897Na3h6iasGW22aEoyby89d7E1sH9zIPPPvTOIejPFhK7baRPKrcIsnrfcShMvdNQuyrdx3SAfmOUtkKJAjo3LW5MzT7yxFVNkEcG2S7CORwr4v8yQDPdlFGmglkWfc12k3ANR+hTpjcL31DYhYkOi9vSn46+j8UjHqTBmPJuZxutQ+WFed9goFBpiL3xfkTPdzk\\\\\\\/VPi4bydgYW84SQM9h1du63BLDGYktMMxOLqstH9s+\\\\\\\/RJKnrkFv28wCRbxiUoPgWtgv\\\\\\\/1OE97WSxZwmhD3nL3R\\\\\\\/TgYEaQKglBwIOyAZpS2zpVR28bH\\\\\\\/m9QmFt+4ZJrcY\\\u0022,\\\u0022iv\\\u0022:\\\u0022dOlyyKScuPcXByY3N6xMJA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e393a5dc3dfcadb87b218dd554d24ef64cbcc775ae7c35b124b989a5341fa794\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022j1KT1UOJDkz0Z78uJCqoeV55OU6rQ9euI53fSUYt6LUpfn8BXMB7JqF9OY4tap1jwkM0cot2MFlxFCvqjGqK5JFxeA7v8Nn80o4xwBJmyesNXW33ng8LBQ4NM8kVmIxmvvdonFA6xl1kB+LHFv30Ydq3MGNf2Hmf2GgJAoennUwhlkKAXsAwWpNKjWTnZ3xfFEqgMuPfZEm96g69UKCLasz0Dss4Kiscn+tGPE\\\\\\\/7a4nXjRbKfGvAXeDzpTox5gnW0AritYSUdHIGC1mxQ90lxmHwAwYihNsYGl7cSDWR1wn4qP05PaQnf6CndnOnCHR6dMHzAib9efEQdyk\\\\\\\/79EZ+56gb6Mpf8ZnEqeSJbDZ4QZYfb6BwcNaGmKDZ5q2iU6ZaEY1vehPd6t4MkD5VfUQ7Hptq1C2fbv8eCvoDSBzooyuXWg+Lpv24W24eykz2jPL1Y3hn0jyNF8BfRscCFtaI6Txc\\\\\\\/UtJV4n4w6uo14NXLujjeNBXA4RQ53lrrzdw3pM93dPJkijtgep1SuyXUTSwEeG7RsTaejPm68AGEmM6Bw8ymzTcZwtL\\\\\\\/BSeJZkByieiFpU7MoHtITPENQE9I37sZh37hhyiK3wNOo\\\\\\\/Tw+\\\\\\\/uBxfYprqqkgwIPDMGyxwqoCHAZF6IGA9AQwoprENYdchSOZytB9mvX0MJSfuOWjiPBKFIW\\\\\\\/3vVocOuAdecXcXLl8+ibmIKATdk2JZJur2XWqPL30rcmzlkmZNYKNCHUk8wDJXJIjG59sOmcnGCMBJrTeNxcHAOOshsvP3Jci+9YzE3NGT1efoV8QE0nTexCa7YQx5SVX2NG7iCy3fhQ3fqP9\\\\\\\/ctVoW6+urp5s7oXyQVI\\\\\\\/DEZtIvsrcbpRKHLc+0fbmZvXhugxYZH9mhyoEtCiYdbnenYSWSF6z3YRnpTFbwg\\\\\\\/CvkLYluIjPFLzbBrhy4E+H3h1ZEjWFZlogAvnEbn3Fm8NKAEM1j6oVkUq0dL2cL1cGhED1406KJfCxbPwHb3xSbQJuZyDq\\\\\\\/WUj1KVhU8FF3lbltVrjkkk20rSK1K4wi0+TUIgCsCihIPdqVuCFJ8O0u0jpTPPytxMlzZudd8qgQv9pFc4qpvWtwYgHEYo8kCNsG54KbjdRkJxL64xTxJfB1+d6BCUV9\\\\\\\/Z0luK17D0fbySo27n9FZhD6Exv2vESsrVHFbPJJ+6VdY7g5hsocAjYWWSOwLlQahxWEdboILrIlWu644g6aRJ\\\\\\\/L6HqXPkxwVu07hCjoAxG7UsyUL\\\\\\\/N4igxNKwSVA6wx0mPvEgUGO\\\\\\\/zlwtTveYMNJlTugDwYz8fNPem19OSgAU\\\\\\\/+PRSOFTAJl17p8CaJEV6\\\\\\\/4IFFzB7ObAxRcVQwbL+VwiZ7xrGA88Q53OaGLbHOhTiRDvAgtFt6ZSeinlrDZAJVsOzhEWdoN8w5UYCnuIOtugaBzDf7Uj\\\\\\\/sIrcSSKLVXDBmA3E90WVyqhFQiYRe7Q2RlDiIOB0iggUQCu4MRG39jj7zHvUQgRNvHqVYaLPfa+CKQPSUhVL\\\\\\\/AODYKRDsLTOkURZPwb\\\\\\\/NyBqW9DyLBzBrNeyKcCiuAc7cLnIiEO67U0BYFOD9OJkfaG7G5MuvSJm13GsvNjIDK8nYTPSXqewA79Rahrsd7AjyaoiQeZ3+zh6Tv1uswlTWKgcjM\\\\\\\/XJFAkFc+VyNlcNgYmCJB8VSArkdIJOMbgrFTG6aDzXDY4XcpoGUqnwINVwDdr7dDU3W3HLOe38oapm24IVEVPNSXwHzNdjvu1EptI9j1\\\\\\\/0sR8foHMlNi+GsAkx5ZuwzZphq33GzfDHOh1dYFJ3ydRIiabCgxkBlFICU6nLQyx1A0DtZkMBAUPXAVFl7v6tWjqzSEnjM8WZmk94Jp2a7mf+NC23nx5UecrUG8jIstxCiHnoz60U8NIlE2KIoH\\\\\\\/dEX\\\\\\\/YGZmq27ul+26gUVyY05E53aTvU0XmfOf1vNWb94cA6058jfShENfIQwf7Aa4XzZOv+MoRx4vEu2BorCtMJhFIrnT3qzqv4Z8a906R5VwYGW2\\\\\\\/dlwzOd3gQvTOV9w=\\\u0022,\\\u0022iv\\\u0022:\\\u0022pFgUw5NCHnQxJ0WYeeaxJg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228ba4d63fb14f7c266bd12ef636acb470ab773366f4c41ff1bb895bb2c7f068dd\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460109 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/857" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4BUjicTBVSZE9G52CD0K-U4Jj9NCX816xRdgJC3qr9o" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/857" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tME5R-2n5TRSDlYrHR4TwbZr6C43lYvvgWCapguaRMo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/4/845" title="Increasing Genome Sampling and Improving SNP Genotyping for Genotyping-by-Sequencing with New Combinations of Restriction Enzymes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/4/865" title="Field-Based High-Throughput Plant Phenotyping Reveals the Temporal Patterns of Quantitative Trait Loci Associated with Stress-Responsive Traits in Cotton" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460109" id="node460109" data-pisa="ggg;6/4/857" data-pisa-master="ggg;g3.115.024596" data-apath="/ggg/6/4/857.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Haplotag: Software for Haplotype-Based Genotyping-by-Sequencing Analysis</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nicholas A. Tinker</span>, <span class="highwire-citation-author" data-delta="1">Wubishet A. Bekele</span> and <span class="highwire-citation-author" data-delta="2">Jiro Hattori</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">857-863;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024596 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460109" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nicholas A. Tinker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Ottawa Research and Development Centre, Agriculture and Agri-Food Canada, Ontario, K1A 0C6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BA.%2BTinker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tinker%20NA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicholas%2BA.%2BTinker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nick.tinker@canada.ca" class="" data-icon-position="" data-hide-link-title="0">nick.tinker@canada.ca</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Wubishet A. Bekele </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Ottawa Research and Development Centre, Agriculture and Agri-Food Canada, Ontario, K1A 0C6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wubishet%2BA.%2BBekele%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bekele%20WA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWubishet%2BA.%2BBekele%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jiro Hattori </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Ottawa Research and Development Centre, Agriculture and Agri-Food Canada, Ontario, K1A 0C6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiro%2BHattori%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hattori%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiro%2BHattori%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/4/857" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460109" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460109/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/857.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460109" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460109/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/857.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460109" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460109/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/857.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460109" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460109/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/4/857.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460109" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460109/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genotyping-by-sequencing (GBS), and related methods, are based on high-throughput short-read sequencing of genomic complexity reductions followed by discovery of single nucleotide polymorphisms (SNPs) within sequence tags. This provides a powerful and economical approach to whole-genome genotyping, facilitating applications in genomics, diversity analysis, and molecular breeding. However, due to the complexity of analyzing large data sets, applications of GBS may require substantial time, expertise, and computational resources. Haplotag, the novel GBS software described here, is freely available, and operates with minimal user-investment on widely available computer platforms. Haplotag is unique in fulfilling the following set of criteria: (1) operates without a reference genome; (2) can be used in a polyploid species; (3) provides a discovery mode, and a production mode; (4) discovers polymorphisms based on a model of tag-level haplotypes within sequenced tags; (5) reports SNPs as well as haplotype-based genotypes; and (6) provides an intuitive visual “passport” for each inferred locus. Haplotag is optimized for use in a self-pollinating plant species.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genotyping-sequencing-gbs" class="hw-term hw-article-keyword hw-article-keyword-genotyping-by-sequencing-gbs" rel="nofollow">genotyping-by-sequencing (GBS)</a></li><li class="kwd"><a href="/keyword/single-nucleotide-polymorphism-snp" class="hw-term hw-article-keyword hw-article-keyword-single-nucleotide-polymorphism-snp" rel="nofollow">single nucleotide polymorphism (SNP)</a></li><li class="kwd"><a href="/keyword/polyploidy" class="hw-term hw-article-keyword hw-article-keyword-polyploidy" rel="nofollow">polyploidy</a></li><li class="kwd"><a href="/keyword/pipeline" class="hw-term hw-article-keyword hw-article-keyword-pipeline" rel="nofollow">pipeline</a></li><li class="kwd"><a href="/keyword/haplotype" class="hw-term hw-article-keyword hw-article-keyword-haplotype" rel="nofollow">haplotype</a></li></ul><p id="p-3">Genotyping-by-sequencing (GBS: <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Elshire <em>et al.</em> 2011</a>) and similar methods [<em>e.g.</em>, restriction-site associated DNA (RAD): <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Miller <em>et al.</em> 2007</a>] have become important strategies for whole genome genetic diversity analysis and related studies in many plant and animal species. The objective of these strategies is to resequence a representative fraction of the genome of many individuals, and thereby determine the genotypes of those individuals at loci where sequence variants exist. Methods are based on high-throughput short-read sequencing of enzymatically constructed genomic complexity reductions, followed by discovery of SNPs within sequence tags. While GBS is powerful and economical, it is also complex, requiring the barcoding and multiplexing of samples, the deconvolution of large data files, the alignment of short reads (tags), and the discovery and filtering of SNPs. The application of GBS in large and complex genomes is especially challenging because of the confounding presence of multiple paralogous loci (especially in polyploids), and, often, the absence of a complete reference genome.</p><p id="p-4">There are several available bioinformatics pipelines for GBS analysis, including Stacks (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Catchen <em>et al.</em> 2011</a>), TASSEL (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Glaubitz <em>et al.</em> 2014</a>), UNEAK (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Lu <em>et al.</em> 2013</a>), and other custom-designed pipelines (<em>e.g.</em>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Sonah <em>et al.</em> 2013</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Poland <em>et al.</em> 2012</a>). Most pipelines require, or benefit from, a reference genome, while UNEAK is designed specifically to operate independently from a reference genome, and Stacks has the ability to run with or without a reference genome. Stacks is a flexible and integrative set of tools that produce many types of output, and can be customized for many genetic scenarios. Stacks also provides a unique web-based interface for inspection of results and quality control; a feature that is useful in tuning the many parameters of GBS analysis such that they produce results that are appropriate to the genome and the genetic population. However, Stacks requires a Unix-like computer environment, and a significant investment of effort in building and maintaining a pipeline, and the web-based interface requires a relational database and web server. Most other GBS pipelines also require the installation of third-party programs (<em>e.g.</em>, to align sequences), while UNEAK requires only the installation of a JAVA run-time environment.</p><p id="p-5">To our knowledge, UNEAK, and the customized scripts described by <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Poland <em>et al.</em> (2012)</a>, are the only existing pipelines that will handle data from polyploids in the absence of a reference genome. Both pipelines achieve this by using a population filter to reject SNPs that fail to segregate with the expected genetic ratio in the population under analysis. Because UNEAK can be run on any computer platform with adequate resources, it has been popular among researchers studying species where no reference genome is available. However, the UNEAK pipeline excludes all SNPs that belong to multi-locus series, SNPs from tags containing multiple SNPs, or SNPs with more than two alleles. In our experience with GBS in hexaploid oat (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Huang <em>et al.</em> 2014</a>), UNEAK excluded at least 30% of potentially useful SNPs that were discovered by an alternate customized pipeline. Furthermore, the developers of UNEAK (personal communication) have indicated that no further development of UNEAK will be performed.</p><p id="p-6">With high-density genotyping comes the possibility to analyze data based on haplotypes, and the ability to impute missing data (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Swarts <em>et al.</em> 2014</a>), which may be of particular importance in GBS analyses where incomplete data are prevalent. Genome-wide association studies (GWAS) based on haplotypes could also allow the discovery of cryptic quantitative trait loci (QTL) associations that have eluded analysis based on single SNPs (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Lorenz <em>et al.</em> 2010</a>). Because GBS data are acquired from sequenced fragments that often contain multiple SNPs, direct information about localized ‘tag-level’ haplotypes are available within a GBS pipeline. However, to our knowledge, no GBS pipeline is able to examine the segregation of haplotypes in the application of a population filter, nor does any software provide a simple method to access or examine haplotypes directly in an output file. Since accurate haplotype inference normally requires a reference genome, the ability to extract haplotypes directly from within GBS fragments could be of particular interest in a species where no reference genome is available.</p><p id="p-7">Our objective was to develop user-friendly GBS software that operates with minimal user-investment on widely available computer platforms. Additionally, we intended this software to meet the following requirements: (1) to operate without the requirement for a reference genome; (2) to operate in a polyploid or duplicated genome, distinguishing paralogous loci when an appropriate population filter is available; (3) to provide a discovery mode, as well as an efficient production mode for scoring previously discovered loci; (4) to discover polymorphisms based on models of segregating tag-level haplotypes within GBS sequenced tags; (5) to report results in a variety of formats, including SNP- and haplotype-based genotypes; and (6) to provide an intuitive “passport” for each inferred locus, enabling visual inspection and validation of discovered GBS loci.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-8">Software named ‘Haplotag’ was written in the Pascal programming language, implemented as Free Pascal (<a href="http://freepascal.org/">http://freepascal.org/</a>) within the Lazarus programing environment (<a href="http://www.lazarus-ide.org/">http://www.lazarus-ide.org/</a>). Both of these programming packages are open source, available on multiple platforms, and actively supported by developer communities. Most algorithms within Haplotag were written to operate in parallel when executed on a computer with multiple processors. The code was compiled for the Windows 64-bit environment (Microsoft, Redmond WA), and tested with Windows XP, Windows 7, Windows 8, and Windows 10, as well as server 2008. Haplotag was tested on many different computers, but the evaluations reported below were executed on a computer running Windows server 2008 with two Intel (Santa Clara, CA) Xeon X5670 processors running at 2.93 GHz. Each processor had six cores, and each core was divided into 12 threads (total 24 threads). The test machine contained 96 GB RAM, but all reported analyses were confirmed to run within 24 GB RAM. All input and output data resided on a locally attached 500 GB disk, since prior experience indicated reduced performance when reading and writing to a network drive. Small projects, as well as the demonstration files described below, will run on most ordinary desktop computers, but will require a 64-bit operating system.</p><p id="p-9">Haplotag was evaluated using a set of small simulated demonstration files, as well as on the full set of primary GBS reads from oat described by <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Huang <em>et al.</em> (2014)</a>. The latter data contained 894 taxa consisting of 360 diverse oat lines, and 534 mapping progeny from six biparental populations. Both Haplotag and the UNEAK pipeline were run with a minimum merged tag count of 50, which is higher than the threshold used in earlier work due to subsequent optimization. Output from both pipelines was filtered across the full population to maintain markers for which genotype calls were ≥ 50% or ≥ 80% complete, heterozygosity was ≤ 10%, and minor allele frequency was ≥ 5%. The error detection threshold in UNEAK was set to 0.02. Additional filters for Haplotag included a maximum base difference of three for aligning tags, a maximum of nine tags per cluster, a maximum heterozygote frequency on a haplotype basis of 0.25, and a maximum tolerance for tri-zygotes and multi-zygotes of 1% and 0%, respectively.</p><div id="sec-2" class="subsection"><h3>Terminology</h3><p id="p-10">When referring to SNPs, we use the terms ‘SNP locus’ (a specific base pair), and ‘SNP alleles’ (the variant bases found at a SNP locus). We then define a ‘tag-level haplotype’ as the combined set of SNP alleles that must exist on a single chromosome due to their recovery in the sequence of a single GBS tag. Although the term haplotype implies the existence of multiple loci, we essentially treat haplotypes as multiple alleles at a single composite locus, which we refer to as a ‘Haplotag locus’, and inferences are made under the assumption that the recombination rate within a tag is negligible. The term ‘heterozygosity’ is used when applying a filter that rejects an inference that two or more haplotypes exist at the same Haplotag locus if those haplotypes occur together more frequently than they would be expected to based on the assumed heterozygosity in the population.</p></div><div id="sec-3" class="subsection"><h3>Data availability</h3><p id="p-11">Data analyzed in this report were deposited in the NCBI short read archive (<a href="http://www.ncbi.nlm.nih.gov/sra/">http://www.ncbi.nlm.nih.gov/sra/</a>) under project accession number SRP037730, and the GBS key for analysis was presented in Table S4 of <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Huang <em>et al.</em> (2014)</a>. Supplemental files include: the Haplotag manual (Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024596/-/DC1/FileS1.pdf">File S1</a>), and sample output (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024596/-/DC1/FileS2.zip">File S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024596/-/DC1/FileS3.xlsx">File S3</a>). Haplotag is available as an executable distribution for recent versions of Windows 64-bit environments (XP, and versions 7 through 10). The distribution can be obtained from the site <a href="http://haplotag.aowc.ca/,">http://haplotag.aowc.ca/,</a> which provides a download links for a compressed file that contains the Windows executable, a user manual (also in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024596/-/DC1/FileS1.pdf">File S1</a>), and demonstration files. Future updates will be maintained at this site, and a voluntary registration is provided to monitor interest in this software, and to enable announcements regarding major revisions. The Pascal source code was made available to reviewers of this work, and will be provided by request on an as-is basis for any noncommercial use based on an open source license. The source code is expected to be compatible with any operating system where a Free Pascal compiler is available, although minor modifications to the code may be required to adapt it for the file systems of other operating environments.</p></div></div><div class="section results-discussion" id="sec-4"><h2 class="">Results and Discussion</h2><div id="sec-5" class="subsection"><h3>Software execution</h3><p id="p-12">The operation and function of Haplotag is described in the accompanying manual (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024596/-/DC1/FileS1.pdf">File S1</a>), which references a set of small simulated input files for demonstration purposes. The input files are archived within the software distribution. When extracted, the demonstration files fall within three separate subdirectories, each containing a complete self-contained set of demonstration files for one of three primary modes in which Haplotag can operate. Within each subdirectory is a master input file with the default name “<em>HTinput.txt</em>,” which contains all relevant parameter specifications as well as a set of pipeline commands that Haplotag will follow in the order listed. Based on these commands, Haplotag can read and process data from three starting points (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>), representing the three modes of operation.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/857/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Flow chart showing input files (green), output files (blue), and dependencies (connecting lines) associated with &#x2018;Haplotag&#x2019; GBS discovery software. Default file names are shown in yellow, and are normally appended by &#x201C;.txt&#x201D; in the Windows file system. Three alternative pipelines (A, B, and C) are available, with required input labeled for each. The cluster discovery pipeline (A), and the haplotype discovery pipeline (B), start by aligning a complete inventory of tags (A), or a reduced inventory of tags from prior work (B), to produce clusters. In (B), the complete inventory is then realigned against this template to increase the sampling of new haplotypes. A complete tag-by-taxa matrix of tag counts (HTBT) is then formed for all tags belonging to clusters of two or more tags. Other output files are then created based on haplotype model fitting. In the production pipeline, only the files labeled by (C) are required, since genotyping is based on counting copies of haplotype-tags in the output files from previous discovery work." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1067429398" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Flow chart showing input files (green), output files (blue), and dependencies (connecting lines) associated with &#x2018;Haplotag&#x2019; GBS discovery software. Default file names are shown in yellow, and are normally appended by &#x201C;.txt&#x201D; in the Windows file system. Three alternative pipelines (A, B, and C) are available, with required input labeled for each. The cluster discovery pipeline (A), and the haplotype discovery pipeline (B), start by aligning a complete inventory of tags (A), or a reduced inventory of tags from prior work (B), to produce clusters. In (B), the complete inventory is then realigned against this template to increase the sampling of new haplotypes. A complete tag-by-taxa matrix of tag counts (HTBT) is then formed for all tags belonging to clusters of two or more tags. Other output files are then created based on haplotype model fitting. In the production pipeline, only the files labeled by (C) are required, since genotyping is based on counting copies of haplotype-tags in the output files from previous discovery work.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/857/F1.medium.gif" width="440" height="352"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/4/857/F1.medium.gif" width="440" height="352"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/857/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/857/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468063" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-13" class="first-child">Flow chart showing input files (green), output files (blue), and dependencies (connecting lines) associated with ‘Haplotag’ GBS discovery software. Default file names are shown in yellow, and are normally appended by “.txt” in the Windows file system. Three alternative pipelines (A, B, and C) are available, with required input labeled for each. The cluster discovery pipeline (A), and the haplotype discovery pipeline (B), start by aligning a complete inventory of tags (A), or a reduced inventory of tags from prior work (B), to produce clusters. In (B), the complete inventory is then realigned against this template to increase the sampling of new haplotypes. A complete tag-by-taxa matrix of tag counts (HTBT) is then formed for all tags belonging to clusters of two or more tags. Other output files are then created based on haplotype model fitting. In the production pipeline, only the files labeled by (C) are required, since genotyping is based on counting copies of haplotype-tags in the output files from previous discovery work.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">There is currently a requirement to run part of the UNEAK GBS pipeline prior to running Haplotag in order to deconvolute the raw barcoded sequence data, produce a tag count file for each sample, and write a merged tag count file for the entire project. The UNEAK pipeline executes these steps very efficiently, thus the replacement of this functionality was not a priority. The current Haplotag distribution provides a small helper utility to assist users in writing the UNEAK script, and converting binary output to the text files required by Haplotag. A standalone replacement for UNEAK is being developed, which may allow the analysis of tags longer than 64 bp, but this tag length is a current limitation of both UNEAK and the current version of Haplotag. Sequencing data with short reads of 100 bp are ideal for this type of analysis, since the barcode may occupy up to the first 10 bases, and this allows truncation of lower quality bases at the 3′ end of the read. Reads of longer than 100 bp can be analyzed, but the tags will be truncated at 64 bases.</p><p id="p-15">The cluster discovery mode (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>) is designed for applications where complete <em>de novo</em> SNP discovery is required. This <em>de novo</em> clustering step is multi-threaded, but it may still run slowly on very large data sets. The haplotype discovery mode (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>) reduces the scale of analysis by seeding the clusters with a set of predetermined tags. This feature is useful for maintaining the legacy nomenclature of reference sequences from prior GBS analyses. It could also be used to seed the alignment of clusters using predicted fragments from a sequenced genome. Alternatively, this step could incorporate consensus sequences from an alternate or more efficient clustering algorithm. The production mode (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>) is designed for applications where SNPs and haplotypes have already been discovered by Haplotag using a large, diverse, and representative population, and where the objective is to genotype new samples while maintaining exactly the same nomenclature of loci, haplotypes, and SNPs. No new haplotypes will be discovered in production mode, so it is not recommended for an application where the diversity of new taxa falls outside of the diversity where the model was built.</p><p id="p-16">What distinguishes Haplotag from other GBS pipelines is the treatment of the tags as haplotypes, and the development of locus models using a population filter to validate the diploid segregation of these haplotypes. Prior to model discovery, tags are deliberately overaligned into clusters that potentially represent multiple paralogous loci. Then, Haplotag tests every possible combination of haplotypes within each cluster to identify mutually exclusive groups of haplotypes that behave as single Haplotag loci. This model testing is based on a population filter, which specifies threshold parameters for maximum heterozygosity, minimum, and maximum allele frequency, and genotype-completeness (minimum proportion of nonmissing genotypes). The result can be a single Haplotag locus within a single cluster, or multiple Haplotag loci within the same cluster. The latter is common in polyploid or recently duplicated genomes. Results of locus prediction and genotype scoring are summarized within a single passport file for each cluster (see below). Although the model selection within clusters does not incorporate sequence divergence, the population filter invariably identifies Haplotag loci in which haplotypes diverge less within the locus than they do among other loci within the same cluster.</p></div><div id="sec-6" class="subsection"><h3>Software function, as illustrated by passport files</h3><p id="p-17">Another important and unique feature of Haplotag is the automated production of a ‘passport’ file for each cluster. This is illustrated by one passport from the analysis of the included demonstration data (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Passport files are formatted in plain HTML, such that they can be viewed in any web browser. They are indexed in a master HTML file that can also be opened and searched in any browser. While these files can be opened directly from a local disk, they could also be uploaded to a website in order to provide external access to the results of an analysis. Individual passport files can be inspected to determine if program parameters are appropriate, or to explore the metadata and genotypes of specific Haplotag loci. In our experience, these files also serve as intuitive graphical presentations that can assist in explaining the GBS concept and the program function to a lay audience.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/857/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Passport file produced by Haplotag from simulated demonstration files. Here, six tags (potential haplotypes) are identified at the top. After model fitting by population-based filtering, two locus-models are selected. When Haplotag is run in &#x2018;verbose&#x2019; mode, the details of model selection are written in a separate file (see File S2). Locus-1 contains three haplotypes and Locus-2 contains two. SNP positions are identified by color. The table at the bottom of the passport shows the tag counts at the presumed haplotypes within each locus. Counts greater than or equal to one are shaded, indicating that they are scored as &#x201C;present&#x201D;." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1067429398" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Passport file produced by Haplotag from simulated demonstration files. Here, six tags (potential haplotypes) are identified at the top. After model fitting by population-based filtering, two locus-models are selected. When Haplotag is run in &#x2018;verbose&#x2019; mode, the details of model selection are written in a separate file (see File S2). Locus-1 contains three haplotypes and Locus-2 contains two. SNP positions are identified by color. The table at the bottom of the passport shows the tag counts at the presumed haplotypes within each locus. Counts greater than or equal to one are shaded, indicating that they are scored as &#x201C;present&#x201D;.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/857/F2.medium.gif" width="368" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/4/857/F2.medium.gif" width="368" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/857/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/857/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468035" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-18" class="first-child">Passport file produced by Haplotag from simulated demonstration files. Here, six tags (potential haplotypes) are identified at the top. After model fitting by population-based filtering, two locus-models are selected. When Haplotag is run in ‘verbose’ mode, the details of model selection are written in a separate file (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024596/-/DC1/FileS2.zip">File S2</a>). Locus-1 contains three haplotypes and Locus-2 contains two. SNP positions are identified by color. The table at the bottom of the passport shows the tag counts at the presumed haplotypes within each locus. Counts greater than or equal to one are shaded, indicating that they are scored as “present”.</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">For example, in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, we would first explain that the six sequences at the top (TagID 1–6) constitute all of the unique 64-base tags from the experiment that formed a single cluster. Potential SNPs in this cluster are highlighted, and counts of the number of taxa containing each tag are shown at the left. We would then explain that the species from which these tags are generated is polyploid, such that we suspect these tags may come from more than one locus. We might then click on the “details of model” link (which would open <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024596/-/DC1/FileS2.zip">File S2</a>) to illustrate how Haplotag has inspected all 57 possible combinations (“models”) of two or more tags from the available six tags. This step is referred to as a “population filter”, since it allows the exclusion of inappropriate models based on whether the tags in a model segregate in a diploid manner within the tested population. Parameters for population filtering, reported at the bottom of the details page (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024596/-/DC1/FileS2.zip">File S2</a>) include genotype-completeness, allele frequency, and heterozygosity. Here, each model was evaluated based on whether it would pass this filter (yes or no). Next, the acceptable model having complete data for the greatest number of taxa (Model 42 in S2) was assigned as ‘Locus-1’. All models that overlapped with Model 42 were then removed, and remaining acceptable models were inspected. Of these, the next best model was assigned to a Haplotag locus (in this case, Model 48 is assigned as ‘Locus-2’). The above process is iterated indefinitely until no acceptable models remain. We would then point out that ‘Haplotag Locus-2’ contains only one SNP locus, and, thus, two haplotypes, while ‘Locus-1’ contains two SNPs, which could theoretically form four haplotypes, of which three haplotypes were observed. In practice, it is very rare to observe four haplotypes at a single Haplotag locus with two SNP loci, as this would imply two mutation events at the same SNP locus, or a rare recombination event between two SNP loci in the same tag.</p><p id="p-20">We would then draw attention to the inferred genotypes and segregation of these five combined haplotypes at two putative Haplotag loci within the population of taxa, which are shown in the table at the bottom of the passport (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). In this idealized example, the genotypes of all 10 taxa are complete at both accepted Haplotag loci. The numbers in each cell show the total counts of tags observed for each taxon under each haplotype within a selected Haplotag locus. Those with nonzero counts for two (or more) haplotypes (<em>e.g.</em>, Taxa TJ, under Locus 1) are scored as heterozygotes. These inferred genotypes are written to a simple text-based file called “HTgenos.txt”. Since many programs for genetic analysis cannot read haplotypes, an alternate genotype file is written, where genotypes are defined by SNP locus calls from within the Haplotag loci. In the example in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>, three SNP locus calls would be written, with ‘Locus-1’ being converted to two SNP loci, identified by their SNP positions within the Haplotag loci. Nomenclature output files are also written, such that all dependencies are represented in a hierarchical naming system. These files are designed with shared fields such that they could be easily loaded into a relational database designed for this purpose.</p></div><div id="sec-7" class="subsection"><h3>Parameter selection</h3><p id="p-21">It is well known that results of SNP identification, especially in a polyploid without a reference genome, are highly dependent on methods and parameters (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Huang <em>et al.</em> 2014</a>). As with other methods for SNP identification, there is no formal way to optimize the selection of model parameters within Haplotag. However, parameters need to be selected carefully, possibly using iterative testing, in order to obtain good results and avoid artifacts. In our experience, the best results from Haplotag are obtained when it is run across a large composite base population consisting of a mixture of biparental populations and diverse taxa representative of target germplasm. The biparental populations will allow validation of Mendelian segregation and mapping of the polymorphisms, while the diversity samples will ensure discovery of alternate haplotypes. The parameters used for the oat data presented below were based on recursive optimization for this type of experiment. If biparental populations are analyzed, then the minimum allele frequency filter can be raised appropriately. If the analysis is restricted to a single biparental population, then the filter could be set to achieve a specific chi-square cut-off. Setting the maximum heterozygote frequency to a low value is very useful to exclude non-Mendelian models, but this can be applied effectively only within inbred lines where the expected heterozygote frequency is significantly lower than 50%.</p></div><div id="sec-8" class="subsection"><h3>Evaluation of Haplotag using data from hexaploid oat</h3><p id="p-22">Data from 894 taxa reported by <a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Huang <em>et al.</em> (2014)</a> were reanalyzed to compare performance and output of Haplotag to that of the UNEAK pipeline. The first two steps of the UNEAK pipeline (production of tag counts and merged tag counts) were run to produce a common starting point for both pipelines, requiring approximately 6 hr to run on the test environment from the raw sequence files. The UNEAK pipeline is not multi-threaded so the presence of 24 processors on this machine was not relevant. The remaining steps in the UNEAK pipeline took only 5 min. Data from both UNEAK and Haplotag were filtered and formatted using the small helper-program “CbyT” described by <a id="xref-ref-4-6" class="xref-bibr" href="#ref-4">Huang <em>et al.</em> (2014)</a>, which is now updated and provided in the current Haplotag distribution. The use of CbyT allowed parameters in either pipeline to be relaxed, such that data filtering could be tested at different levels from the same output. The total count of SNP loci from the UNEAK pipeline passing the population filter at a genotype-completeness threshold of ≥ 50% was 12,780. At a threshold of ≥ 80%, the count of filtered SNP loci was 4260 (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467996/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467996/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Comparison of GBS data analysis using UNEAK <em>vs.</em> Haplotag software</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">Running on the same machine, but utilizing 23 processors, the full Haplotag pipeline in cluster discovery mode took 6.9 hr in addition to the 6 hr required by UNEAK. The cluster discovery step took most of this execution time. After applying the same population filter, the number of Haplotag loci was 29,421 with a genotype-completeness of ≥ 50%, or 11,950 with a genotype-completeness of ≥ 80%. When translated to SNP loci, the number of calls was 43,378 at ≥ 50% completeness or 17,117 at ≥ 80% completeness. The larger number of SNP loci relative to Haplotag loci is due to the presence of multiple SNP loci within some Haplotag loci.</p><p id="p-27">In comparing the filtered SNP loci called by UNEAK to the SNP loci called from Haplotag, 4204 (99%) of the 4260 UNEAK SNPs filtered at ≥ 80% genotype-completeness were identical to those called by Haplotag at the same filtering level. In contrast, UNEAK identified only 24% of the 17,117 Haplotag SNPs filtered at ≥ 80% genotype-completeness. In general, Haplotag called most of the same SNP loci discovered by UNEAK, because these represented the clusters in Haplotag with exactly two haplotypes having only a single SNP difference. The small number of UNEAK SNPs that were missed by Haplotag are a result of rare haplotypes, and/or sequencing errors that were aligned into a large cluster by Haplotag. In rare cases, this resulted in a complex cluster that was excluded from the Haplotag project because it exceeded the threshold for the maximum number of tags per cluster. The UNEAK pipeline has a different network-based strategy that is intended to exclude rare haplotypes because it is designed to seek models with only two haplotypes and a single SNP. While it is possible to adjust Haplotag parameters to increase the coverage of UNEAK SNPs, this would be at the expense of a greater number of multi-haplotype models that are called by Haplotag.</p><p id="p-28">Haplotag was also tested in production mode, which required only 11 min in our test environment. As shown in <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>, production mode uses loci and haplotypes discovered in a previous analysis to reduce computation time and preserve an established nomenclature. When we used input files from the previously reported cluster-discovery run, we achieved exactly the same results, as expected. Thus, to test a different scenario, we used input files from an alternate analysis (not reported), where Haplotag had been run in haplotype discovery mode. In that analysis, clusters were built from the full set of SNP reference sequences reported by <a id="xref-ref-4-7" class="xref-bibr" href="#ref-4">Huang <em>et al.</em> (2014)</a>, as well as from additional SNP reference sequences from subsequent work, encompassing a total of 3327 taxa. We had used this strategy in order to preserve SNP nomenclature used in prior published and submitted work. Here, we wanted to test whether the haplotypes discovered using this large inventory of reference sequences would provide similar results to those achieved above using Haplotag in cluster discovery mode. The results of this analysis provided genotypes for 24,412 or 7343 Haplotag loci (at ≥ 50% or ≥ 80% genotype-completeness, respectively), which translated to 31,685 and 8872 SNP loci, respectively (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Averaged across filtering levels, this was a 33% reduction in called loci relative to those from Haplotag in full cluster discovery mode. The disadvantage of this strategy, which we have now demonstrated, is that the current production files have not incorporated a large number of high quality “new” SNPs that are discoverable only by Haplotag. This new result will be considered in future GBS work in oat, and will require careful addition of new clusters, loci, and haplotypes to the existing production files, while still preserving the legacy nomenclature.</p><p id="p-29">Each Haplotag run produces a complete index of passport files, linking each Haplotag locus to a passport file for the cluster where that locus was called. While this index is written in HTML format, it can be easily manipulated into a table, which we have demonstrated in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024596/-/DC1/FileS3.xlsx">File S3</a>. This table provides links to the passport files for the 7343 Haplotag loci called in the production mode and filtered at ≥ 80% genotype-completeness. We have chosen this output because it contains legacy SNPs and nomenclature (from <a id="xref-ref-4-8" class="xref-bibr" href="#ref-4">Huang <em>et al.</em> 2014</a>) to which we have added known map positions. By loading all passport files to a web server, they do not need to be downloaded and duplicated by users of this resource. This strategy will be used in future to provide passports and metadata for public GBS data sets loaded into the T3/Oat database (<a href="https://triticeaetoolbox.org/oat/">https://triticeaetoolbox.org/oat/</a>). Since passport files can also be saved and opened without the need for a web server, an individual passport file can be shared easily with a collaborator when there is an interest in inspecting the sequence and genotypes of a specific Haplotag locus.</p></div><div id="sec-9" class="subsection"><h3>Limitations and future development</h3><p id="p-30">Haplotag was developed primarily to solve problems of genotyping in self-pollinating allopolyploid species without a reference genome. It will also function well in a self-pollinating diploid species. When paralogous loci exist, such that they are aligned together within the same cluster, Haplotag depends on a simple heterozygosity filter to build models of Haplotag loci that exclude haplotypes from nonhomologous loci. Typically, this is very effective in self-pollinating populations where heterozygotes are rare and this filter can be set at a low level (typically between 0.05 and 0.12). In populations where high rates of heterozygosity are expected (in F<sub>2</sub> populations, or in populations of outcrossing species), a heterozygosity filter that was set higher (<em>e.g.</em>, 0.65) could still be effective in excluding nonsegregating haplotypes from paralogous locus, but complications could arise if multiple paralogous loci are segregating simultaneously. We initially considered the application of a Fishers’ test of contingency tables, but extending this test to an arbitrary number of haplotypes was beyond our programing skills. In future, we may consider adding additional population filters to expand the genetic scenarios in which Haplotag can be used, and we welcome suggestions in this regard.</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-4"><p id="p-31">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024596/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024596/-/DC1</a></p></li><li class="fn" id="fn-5"><p id="p-32">Primary data analyzed in this report are available from the NCBI short read archive (<a href="http://www.ncbi.nlm.nih.gov/sra/">http://www.ncbi.nlm.nih.gov/sra/</a>) under project accession number SRP037730. The software described in this report is available from <a href="http://haplotag.aowc.ca">http://haplotag.aowc.ca</a>.</p></li><li class="fn" id="fn-6"><p id="p-33"><em>Communicating editor: E. Akhunov</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-11-08"><span class="received-label">Received </span>November 8, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-01-22"><span class="accepted-label">Accepted </span>January 22, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Tinker <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.857.1" data-doi="10.1534/g3.111.000240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Postlethwait</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Stacks: building and genotyping loci de novo from short-read sequences.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">1</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">171</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DCatchen%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D1%26rft.issue%253D3%26rft.spage%253D171%26rft.epage%253D182%26rft.atitle%253DStacks%253A%2BBuilding%2Band%2BGenotyping%2BLoci%2BDe%2BNovo%2BFrom%2BShort-Read%2BSequences%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.000240%26rft_id%253Dinfo%253Apmid%252F22384329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjEvMy8xNzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvODU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.857.2" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F857.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.857.3" data-doi="10.1371/journal.pone.0090346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harriman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">e90346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De90346%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0090346%26rft_id%253Dinfo%253Apmid%252F24587335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0090346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24587335&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F857.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.857.4" data-doi="10.1371/journal.pone.0102448"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wight</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tinker</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Using genotyping-by-sequencing (GBS) for genomic discovery in cultivated oat.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">e102448</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De102448%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0102448%26rft_id%253Dinfo%253Apmid%252F25047601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0102448&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25047601&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F857.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.857.5" data-doi="10.1371/journal.pone.0014079"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamblin</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Performance of single nucleotide polymorphisms <em>vs.</em> haplotypes for genome-wide association analysis in barley.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">e14079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLorenz%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253De14079%26rft.epage%253De14079%26rft.atitle%253DPerformance%2Bof%2Bsingle%2Bnucleotide%2Bpolymorphisms%2Bversus%2Bhaplotypes%2Bfor%2Bgenome-wide%2Bassociation%2Banalysis%2Bin%2Bbarley.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0014079%26rft_id%253Dinfo%253Apmid%252F21124933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0014079&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21124933&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F857.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.857.6" data-doi="10.1371/journal.pgen.1003215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cherney</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Switchgrass genomic diversity, ploidy, and evolution: novel insights from a network-based SNP discovery protocol.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">e1003215</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003215%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003215%26rft_id%253Dinfo%253Apmid%252F23349638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23349638&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F857.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.857.7" data-doi="10.1101/gr.5681207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Rapid and cost-effective polymorphism identification and genotyping using restriction site associated DNA (RAD) markers.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">240</span>–<span class="cit-lpage">248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5681207%26rft_id%253Dinfo%253Apmid%252F17189378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE3LzIvMjQwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzg1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.857.8" data-doi="10.1371/journal.pone.0032253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">e32253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPoland%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De32253%26rft.epage%253De32253%26rft.atitle%253DDevelopment%2Bof%2Bhigh-density%2Bgenetic%2Bmaps%2Bfor%2Bbarley%2Band%2Bwheat%2Busing%2Ba%2Bnovel%2Btwo-enzyme%2Bgenotyping-by-sequencing%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0032253%26rft_id%253Dinfo%253Apmid%252F22389690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0032253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22389690&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F857.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.857.9" data-doi="10.1371/journal.pone.0054603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sonah</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bastien</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iquira</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tardivel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Légaré</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">An improved genotyping by sequencing (GBS) approach offering increased versatility and efficiency of SNP discovery and genotyping.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">e54603</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De54603%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0054603%26rft_id%253Dinfo%253Apmid%252F23372741%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0054603&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23372741&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F857.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.857.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swarts</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romero Navarro</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">An</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romay</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Novel methods to optimize genotypic imputation for low-coverage, next-generation sequence data in crop plants.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">3</span>): 1–12.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/4/857.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/4/845" title="Increasing Genome Sampling and Improving SNP Genotyping for Genotyping-by-Sequencing with New Combinations of Restriction Enzymes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/4/865" title="Field-Based High-Throughput Plant Phenotyping Reveals the Temporal Patterns of Quantitative Trait Loci Associated with Stress-Responsive Traits in Cotton" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/4"><span>Volume 6 Issue 4, April 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/4.cover-source.jpg?itok=Lj45nPaf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/857.supplemental" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-EZwIGJJ5OOElSSzQoynDyOMzHCOAqBLZl-surlZkmWQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/4/857" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/4/857" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/4/857" class="active" data-icon-position="" data-hide-link-title="0">Haplotag: Software for Haplotype-Based Genotyping-by-Sequencing Analysis</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460109" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-1YVaTm4pTwgaXJFaO0JAkDxFLo8iwNsyXydgxHvWUPU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/4/857.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/4/857" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-2ihDR8PVrglLycv8si5lyK5yfentP0d-GlyemNGLhss" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/4/857" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/4/857.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460109" id="node460109--2" data-pisa="ggg;6/4/857" data-pisa-master="ggg;g3.115.024596" data-apath="/ggg/6/4/857.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Haplotag: Software for Haplotype-Based Genotyping-by-Sequencing Analysis</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nicholas A. Tinker</span>, <span class="highwire-citation-author" data-delta="1">Wubishet A. Bekele</span> and <span class="highwire-citation-author" data-delta="2">Jiro Hattori</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">857-863;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024596 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460109--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nicholas A. Tinker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Ottawa Research and Development Centre, Agriculture and Agri-Food Canada, Ontario, K1A 0C6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BA.%2BTinker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tinker%20NA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicholas%2BA.%2BTinker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nick.tinker@canada.ca" class="" data-icon-position="" data-hide-link-title="0">nick.tinker@canada.ca</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Wubishet A. Bekele </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Ottawa Research and Development Centre, Agriculture and Agri-Food Canada, Ontario, K1A 0C6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wubishet%2BA.%2BBekele%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bekele%20WA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWubishet%2BA.%2BBekele%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jiro Hattori </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Ottawa Research and Development Centre, Agriculture and Agri-Food Canada, Ontario, K1A 0C6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiro%2BHattori%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hattori%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiro%2BHattori%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/4/857" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/4/857&amp;title=Haplotag%3A%20Software%20for%20Haplotype-Based%20Genotyping-by-Sequencing%20Analysis" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/4/857&amp;title=Haplotag%3A%20Software%20for%20Haplotype-Based%20Genotyping-by-Sequencing%20Analysis" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/4/857&amp;title=Haplotag%3A%20Software%20for%20Haplotype-Based%20Genotyping-by-Sequencing%20Analysis" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/4/857&amp;text=Haplotag%3A%20Software%20for%20Haplotype-Based%20Genotyping-by-Sequencing%20Analysis" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/4/857&amp;title=Haplotag%3A%20Software%20for%20Haplotype-Based%20Genotyping-by-Sequencing%20Analysis" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/4/857&amp;t=Haplotag%3A%20Software%20for%20Haplotype-Based%20Genotyping-by-Sequencing%20Analysis" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/4/857&amp;title=Haplotag%3A%20Software%20for%20Haplotype-Based%20Genotyping-by-Sequencing%20Analysis" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/4/857&amp;title=Haplotag%3A%20Software%20for%20Haplotype-Based%20Genotyping-by-Sequencing%20Analysis" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460109/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460109" id="node460109--3" data-pisa="ggg;6/4/857" data-pisa-master="ggg;g3.115.024596" data-apath="/ggg/6/4/857.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Haplotag: Software for Haplotype-Based Genotyping-by-Sequencing Analysis</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nicholas A. Tinker</span>, <span class="highwire-citation-author" data-delta="1">Wubishet A. Bekele</span> and <span class="highwire-citation-author" data-delta="2">Jiro Hattori</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">857-863;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024596 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460109--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nicholas A. Tinker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Ottawa Research and Development Centre, Agriculture and Agri-Food Canada, Ontario, K1A 0C6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BA.%2BTinker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tinker%20NA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicholas%2BA.%2BTinker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nick.tinker@canada.ca" class="" data-icon-position="" data-hide-link-title="0">nick.tinker@canada.ca</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Wubishet A. Bekele </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Ottawa Research and Development Centre, Agriculture and Agri-Food Canada, Ontario, K1A 0C6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wubishet%2BA.%2BBekele%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bekele%20WA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWubishet%2BA.%2BBekele%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jiro Hattori </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Ottawa Research and Development Centre, Agriculture and Agri-Food Canada, Ontario, K1A 0C6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiro%2BHattori%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hattori%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiro%2BHattori%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=857&amp;rft.epage=863&amp;rft.atitle=Haplotag%3A%20Software%20for%20Haplotype-Based%20Genotyping-by-Sequencing%20Analysis&amp;rft.volume=6&amp;rft.issue=4&amp;rft.date=2016-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Tinker%2C+Nicholas+A.&amp;rft.au=Bekele%2C+Wubishet+A.&amp;rft.au=Hattori%2C+Jiro"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460109/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460109/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460109/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460109/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460109/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460109/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460109/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460109/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460109/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460109/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460109/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460109/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/4/857" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/4/857.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/4/857.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/4/857.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/4/857.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
