<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/4/779" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/4/779.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/4/779.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/4/779.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/4/779.atom" />
<meta name="HW.pisa" content="ggg;6/4/779" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Utilizing Gene Tree Variation to Identify Candidate Effector Genes in Zymoseptoria tritici" />
<meta name="DC.Identifier" content="10.1534/g3.115.025197" />
<meta name="DC.Date" content="2016-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 McDonald et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Zymoseptoria tritici is a host-specific, necrotrophic pathogen of wheat. Infection by Z. tritici is characterized by its extended latent period, which typically lasts 2 wks, and is followed by extensive host cell death, and rapid proliferation of fungal biomass. This work characterizes the level of genomic variation in 13 isolates, for which we have measured virulence on 11 wheat cultivars with differential resistance genes. Between the reference isolate, IPO323, and the 13 Australian isolates we identified over 800,000 single nucleotide polymorphisms, of which ∼10% had an effect on the coding regions of the genome. Furthermore, we identified over 1700 probable presence/absence polymorphisms in genes across the Australian isolates using de novo assembly. Finally, we developed a gene tree sorting method that quickly identifies groups of isolates within a single gene alignment whose sequence haplotypes correspond with virulence scores on a single wheat cultivar. Using this method, we have identified &lt; 100 candidate effector genes whose gene sequence correlates with virulence toward a wheat cultivar carrying a major resistance gene." />
<meta name="DC.Contributor" content="Megan C. McDonald" />
<meta name="DC.Contributor" content="Lachlan McGinness" />
<meta name="DC.Contributor" content="James K. Hane" />
<meta name="DC.Contributor" content="Angela H. Williams" />
<meta name="DC.Contributor" content="Andrew Milgate" />
<meta name="DC.Contributor" content="Peter S. Solomon" />
<meta name="article:published_time" content="2016-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Utilizing Gene Tree Variation to Identify Candidate Effector Genes in Zymoseptoria tritici" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;&lt;i&gt;Zymoseptoria tritici&lt;/i&gt; is a host-specific, necrotrophic pathogen of wheat. Infection by &lt;i&gt;Z. tritici&lt;/i&gt; is characterized by its extended latent period, which typically lasts 2&amp;nbsp;wks, and is followed by extensive host cell death, and rapid proliferation of fungal biomass. This work characterizes the level of genomic variation in 13 isolates, for which we have measured virulence on 11 wheat cultivars with differential resistance genes. Between the reference isolate, IPO323, and the 13 Australian isolates we identified over 800,000 single nucleotide polymorphisms, of which ∼10% had an effect on the coding regions of the genome. Furthermore, we identified over 1700 probable presence/absence polymorphisms in genes across the Australian isolates using &lt;i&gt;de novo&lt;/i&gt; assembly. Finally, we developed a gene tree sorting method that quickly identifies groups of isolates within a single gene alignment whose sequence haplotypes correspond with virulence scores on a single wheat cultivar. Using this method, we have identified &amp;lt;&amp;nbsp;100 candidate effector genes whose gene sequence correlates with virulence toward a wheat cultivar carrying a major resistance gene.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/04/01" />
<meta name="citation_mjid" content="ggg;6/4/779" />
<meta name="citation_id" content="6/4/779" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/4/779" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/4/779.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/4/779.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/4/779.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.025197" />
<meta name="citation_pmid" content="26837952" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="779" />
<meta name="citation_lastpage" content="791" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Megan C. McDonald" />
<meta name="citation_author_institution" content="Plant Sciences Division, Research School of Biology, The Australian National University, Canberra, ACT, Australia 2601" />
<meta name="citation_author" content="Lachlan McGinness" />
<meta name="citation_author_institution" content="Plant Sciences Division, Research School of Biology, The Australian National University, Canberra, ACT, Australia 2601" />
<meta name="citation_author" content="James K. Hane" />
<meta name="citation_author_institution" content="Black Box Bioinformatics, Perth, WA, Australia 6102" />
<meta name="citation_author_institution" content="CCDM Bioinformatics, Centre for Crop and Disease Management, Department of Environment and Agriculture, Curtin University, Perth, WA, Australia" />
<meta name="citation_author_institution" content="Curtin Institute for Computation, Curtin University, Perth, WA, Australia 6102" />
<meta name="citation_author" content="Angela H. Williams" />
<meta name="citation_author_institution" content="Black Box Bioinformatics, Perth, WA, Australia 6102" />
<meta name="citation_author_institution" content="CCDM Bioinformatics, Centre for Crop and Disease Management, Department of Environment and Agriculture, Curtin University, Perth, WA, Australia" />
<meta name="citation_author" content="Andrew Milgate" />
<meta name="citation_author_institution" content="NSW Department of Primary Industries, Wagga Wagga Agricultural Institute, NSW, Australia 2795" />
<meta name="citation_author" content="Peter S. Solomon" />
<meta name="citation_author_institution" content="Plant Sciences Division, Research School of Biology, The Australian National University, Canberra, ACT, Australia 2601" />
<meta name="citation_author_email" content="peter.solomon@anu.edu.au" />
<meta name="citation_reference" content="citation_journal_title=Plant Pathol.;citation_author=L. S. Arraiano;citation_author=J. K. M. Brown;citation_title=Identification of isolate-specific and partial resistance to Septoria tritici blotch in 238 European wheat cultivars and breeding lines.;citation_pages=726-738;citation_volume=56;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Aust. J. Agric. Res.;citation_author=B. Ballantyne;citation_author=F. Thomson;citation_title=Pathogenic variation in Australian isolates of Mycosphaerella graminicola.;citation_pages=921-934;citation_volume=46;citation_year=1995;citation_doi=10.1071/AR9950921" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=S. Banke;citation_author=B. A. McDonald;citation_title=Migration patterns among global populations of the pathogenic fungus Mycosphaerella graminicola.;citation_pages=1881-1896;citation_volume=14;citation_year=2005;citation_issue=7;citation_pmid=15910313;citation_doi=10.1111/j.1365-294X.2005.02536.x" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=S. Banke;citation_author=A. Peschon;citation_author=B. A. McDonald;citation_title=Phylogenetic analysis of globally distributed Mycosphaerella graminicola populations based on three DNA sequence loci.;citation_pages=226-238;citation_volume=41;citation_year=2004;citation_issue=2;citation_pmid=14732268;citation_doi=10.1016/j.fgb.2003.09.006" />
<meta name="citation_reference" content="citation_journal_title=Journal of computational biology :  a journal of computational molecular cell biology;citation_journal_abbrev=J Comput Biol;citation_author=A. Bankevich;citation_author=S. Nurk;citation_author=D. Antipov;citation_author=A. A. Gurevich;citation_author=M. Dvorkin;citation_title=SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing.;citation_pages=455-477;citation_volume=19;citation_year=2012;citation_issue=5;citation_pmid=22506599;citation_doi=10.1089/cmb.2012.0021" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genet. Biol.;citation_author=S. Ben M’barek;citation_author=J. H. G. Cordewener;citation_author=S. M. Tabib Ghaffary;citation_author=T. A. J. van der Lee;citation_author=Z. Liu;citation_title=FPLC and liquid-chromatography mass spectrometry identify candidate necrosis-inducing proteins from culture filtrates of the fungal wheat pathogen Zymoseptoria tritici.;citation_pages=54-62;citation_volume=79;citation_year=2015;citation_pmid=26092790;citation_doi=10.1016/j.fgb.2015.03.015" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data.;citation_volume=30;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_author=R. Bouckaert;citation_author=J. Heled;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Biol.;citation_author=R. Bouckaert;citation_author=J. Heled;citation_author=D. Kühnert;citation_author=T. Vaughan;citation_author=C.-H. Wu;citation_title=BEAST 2: a software platform for Bayesian evolutionary analysis. PLoS Comp.;citation_pages=e1003537;citation_volume=10;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_journal_abbrev=Phytopathology;citation_author=P. A. Brading;citation_author=E. C. P. Verstappen;citation_author=G. H. J. Kema;citation_author=J. K. M. Brown;citation_title=A Gene-for-Gene Relationship Between Wheat and Mycosphaerella graminicola, the Septoria Tritici Blotch Pathogen.;citation_pages=439-445;citation_volume=92;citation_year=2002;citation_issue=4;citation_pmid=18942957;citation_doi=10.1094/PHYTO.2002.92.4.439" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genet. Biol.;citation_author=J. K. M. Brown;citation_author=L. Chartrain;citation_author=P. Lasserre-Zuber;citation_author=C. Saintenac;citation_title=Genetics of resistance to Zymoseptoria tritici and applications to wheat breeding.;citation_pages=33-41;citation_volume=79;citation_year=2015;citation_pmid=26092788;citation_doi=10.1016/j.fgb.2015.04.017" />
<meta name="citation_reference" content="citation_journal_title=Plant Pathol.;citation_author=L. Chartrain;citation_author=P. A. Brading;citation_author=J. C. Makepeace;citation_title=Sources of resistance to Septoria tritici blotch and implications for wheat breeding.;citation_pages=454-460;citation_volume=53;citation_year=2004;citation_doi=10.1111/j.1365-3059.2004.01052.x" />
<meta name="citation_reference" content="citation_journal_title=Plant Pathol.;citation_author=L. Chartrain;citation_author=P. A. Brading;citation_author=J. K. M. Brown;citation_title=Presence of the Stb6 gene for resistance to Septoria tritici blotch (Mycosphaerella graminicola) in cultivars used in wheat-breeding programmes worldwide.;citation_pages=134-143;citation_volume=54;citation_year=2005;citation_doi=10.1111/j.1365-3059.2005.01164.x" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=C. Chen;citation_author=B. Lian;citation_author=J. Hu;citation_author=H. Zhai;citation_author=X. Wang;citation_title=Genome comparison of two Magnaporthe oryzae field isolates reveals genome variations and potential virulence effectors.;citation_pages=887;citation_volume=14;citation_year=2013;citation_pmid=24341723;citation_doi=10.1186/1471-2164-14-887" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Pathog;citation_author=I. Chuma;citation_author=C. Isobe;citation_author=Y. Hotta;citation_author=K. Ibaragi;citation_author=N. Futamata;citation_title=Multiple translocation of the AVR-Pita effector gene among chromosomes of the rice blast fungus Magnaporthe oryzae and related species.;citation_pages=e1002147-e1002147;citation_volume=7;citation_year=2011;citation_issue=7;citation_pmid=21829350;citation_doi=10.1371/journal.ppat.1002147" />
<meta name="citation_reference" content="citation_journal_title=Fly (Austin);citation_author=P. Cingolani;citation_author=A. Platts;citation_author=L. L. Wang;citation_author=M. Coon;citation_author=T. Nguyen;citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3.;citation_pages=80-92;citation_volume=6;citation_year=2012;citation_pmid=22728672;citation_doi=10.4161/fly.19695" />
<meta name="citation_reference" content="citation_journal_title=Proc. 16th Int. Parallel Dist. Proc. Symp.;citation_title=TCS:estimating gene genealogies.;citation_author=M. Clement;citation_author=Q. Snell;citation_author=P. Walke;citation_author=D. Posada;citation_author=K. Crandall;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Plant Pathol.;citation_author=L. Cohen;citation_author=Z. Eyal;citation_title=The histology of processes associated with the infection of resistant and susceptible wheat cultivars with Septoria tritici.;citation_pages=737-743;citation_volume=42;citation_year=1993;citation_doi=10.1111/j.1365-3059.1993.tb01560.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS Pathog.;citation_author=D. Croll;citation_author=B. A. McDonald;citation_title=The accessory genome as a cradle for adaptive evolution in pathogens.;citation_pages=e1002608;citation_volume=8;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=D. Croll;citation_author=M. Zala;citation_author=B. A. McDonald;citation_title=Breakage-fusion-bridge cycles and large insertions contribute to the rapid evolution of accessory chromosomes in a fungal pathogen.;citation_pages=e1003567;citation_volume=9;citation_year=2013;citation_pmid=23785303;citation_doi=10.1371/journal.pgen.1003567" />
<meta name="citation_reference" content="citation_journal_title=Plant Pathol.;citation_author=R. Dean;citation_author=J. A. Van Kan;citation_author=Z. A. Pretorius;citation_author=K. E. Hammond-Kosack;citation_author=A. Di Pietro;citation_title=The top 10 fungal pathogens in molecular plant pathology.;citation_pages=414-430;citation_volume=13;citation_year=2012;citation_doi=10.1111/j.1364-3703.2011.00783.x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. A. Dean;citation_author=N. J. Talbot;citation_author=D. J. Ebbole;citation_author=M. L. Farman;citation_author=T. K. Mitchell;citation_title=The genome sequence of the rice blast fungus Magnaporthe grisea.;citation_pages=980-986;citation_volume=434;citation_year=2005;citation_issue=7036;citation_pmid=15846337;citation_doi=10.1038/nature03449" />
<meta name="citation_reference" content="citation_journal_title=Plant Pathol.;citation_author=P. J. G. M. de Wit;citation_author=R. Mehrabi;citation_author=H. A. Van den Burg;citation_author=I. Stergiopoulos;citation_title=Fungal effector proteins: past, present and future.;citation_pages=735-747;citation_volume=10;citation_year=2009;citation_doi=10.1111/j.1364-3703.2009.00591.x" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=P. N. Dodds;citation_author=J. P. Rathjen;citation_title=Plant immunity: towards an integrated view of plant-pathogen interactions.;citation_pages=539-548;citation_volume=11;citation_year=2010;citation_issue=8;citation_pmid=20585331;citation_doi=10.1038/nrg2812" />
<meta name="citation_reference" content="citation_journal_title=Molecular plant-microbe interactions :  MPMI;citation_journal_abbrev=Mol Plant Microbe Interact;citation_author=S. Duplessis;citation_author=S. Hacquard;citation_author=C. Delaruelle;citation_author=E. Tisserant;citation_author=P. Frey;citation_title=Melampsora larici-populina transcript profiling during germination and timecourse infection of poplar leaves reveals dynamic expression patterns associated with virulence and biotrophy.;citation_pages=808-818;citation_volume=24;citation_year=2011;citation_issue=7;citation_pmid=21644839;citation_doi=10.1094/MPMI-01-11-0006" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. C. Edgar;citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput;citation_pages=1792-1797;citation_volume=32;citation_year=2004;citation_issue=5;citation_pmid=15034147;citation_doi=10.1093/nar/gkh340" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=O. Emanuelsson;citation_author=S. Brunak;citation_author=G. von Heijne;citation_author=H. Nielsen;citation_title=Locating proteins in the cell using TargetP, SignalP and related tools.;citation_pages=953-971;citation_volume=2;citation_year=2007;citation_issue=4;citation_pmid=17446895;citation_doi=10.1038/nprot.2007.131" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genet. Biol.;citation_author=H. Fones;citation_author=S. Gurr;citation_title=The impact of Septoria tritici blotch disease on wheat: An EU perspective.;citation_pages=3-7;citation_volume=79;citation_year=2015;citation_pmid=26092782;citation_doi=10.1016/j.fgb.2015.04.004" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Pathog;citation_author=D. M. Gardiner;citation_author=M. C. McDonald;citation_author=L. Covarelli;citation_author=P. S. Solomon;citation_author=A. G. Rusu;citation_title=Comparative pathogenomics reveals horizontally acquired novel virulence genes in fungi infecting cereal hosts.;citation_pages=e1002952-e1002952;citation_volume=8;citation_year=2012;citation_issue=9;citation_pmid=23028337;citation_doi=10.1371/journal.ppat.1002952" />
<meta name="citation_reference" content="citation_journal_title=Plant Pathol.;citation_author=S. B. Goodwin;citation_title=Back to basics and beyond: increasing the level of resistance to Septoria tritici blotch in wheat. Austral.;citation_pages=532-538;citation_volume=36;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. B. Goodwin;citation_author=S. B. M’barek;citation_author=B. Dhillon;citation_author=A. H. J. Wittenberg;citation_author=C. F. Crane;citation_title=Finished genome of the fungal wheat pathogen Mycosphaerella graminicola reveals dispensome structure, chromosome plasticity, and stealth pathogenesis.;citation_pages=e1002070-e1002070;citation_volume=7;citation_year=2011;citation_issue=6;citation_pmid=21695235;citation_doi=10.1371/journal.pgen.1002070" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=B. Haubold;citation_author=F. Klotzl;citation_author=P. Pfaffelhuber;citation_title=andi: Fast and accurate estimation of evolutionary distances between closely related genomes.;citation_volume=31;citation_year=2015;citation_pmid=25504847;citation_doi=10.1093/bioinformatics/btu815" />
<meta name="citation_reference" content="citation_journal_title=Molecular plant-microbe interactions :  MPMI;citation_journal_abbrev=Mol Plant Microbe Interact;citation_author=S. A. Hogenhout;citation_author=R. A. L. Van der Hoorn;citation_author=R. Terauchi;citation_author=S. Kamoun;citation_title=Emerging concepts in effector biology of plant-associated organisms.;citation_pages=115-122;citation_volume=22;citation_year=2009;citation_issue=2;citation_pmid=19132864;citation_doi=10.1094/MPMI-22-2-0115" />
<meta name="citation_reference" content="citation_title=Septoria tritici blotch of Wheat. 2014. Information Note (No. AG1336);citation_author=G. Hollaway;citation_year=2014" />
<meta name="citation_reference" content="citation_title=Cereal Disease Guide 2014. 2014. Information Note (No. AG1160);citation_author=G. Hollaway;citation_author=M. McLean;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=J. Plant Pathol.;citation_author=T. Jürgens;citation_author=C. C. Linde;citation_author=B. A. McDonald;citation_title=Genetic Structure of Mycosphaerella graminicola Populations from Iran, Argentina and Australia.;citation_pages=223-233;citation_volume=115;citation_year=2006;citation_doi=10.1007/s10658-006-9000-0" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=R. Kellner;citation_author=A. Bhattacharyya;citation_author=S. Poppe;citation_author=T. Y. Hsu;citation_author=R. B. Brem;citation_title=Expression profiling of the wheat pathogen Zymoseptoria tritici reveals genomic patterns of transcription and host-specific regulatory programs.;citation_volume=6;citation_year=2014;citation_pmid=24920004;citation_doi=10.1093/gbe/evu101" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_author=G. Kema;citation_author=D. Yu;citation_author=F. Rijkenberg;citation_author=M. W. Shaw;citation_author=R. P. Baayen;citation_title=Histology of the pathogenesis of Mycosphaerella graminicola in wheat.;citation_pages=777-786;citation_volume=86;citation_year=1996;citation_doi=10.1094/Phyto-86-777" />
<meta name="citation_reference" content="citation_journal_title=Plant Pathol.;citation_author=J. Keon;citation_author=J. J. Rudd;citation_author=J. Antoniw;citation_author=W. Skinner;citation_author=J. Hargreaves;citation_title=Metabolic and stress adaptation by Mycosphaerella graminicola during sporulation in its host revealed through microarray transcription profiling.;citation_pages=527-540;citation_volume=6;citation_year=2005;citation_doi=10.1111/j.1364-3703.2005.00304.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular plant-microbe interactions :  MPMI;citation_journal_abbrev=Mol Plant Microbe Interact;citation_author=J. Keon;citation_author=J. Antoniw;citation_author=R. Carzaniga;citation_author=S. Deller;citation_author=J. L. Ward;citation_title=Transcriptional adaptation of Mycosphaerella graminicola to programmed cell death (PCD) of its susceptible wheat host.;citation_pages=178-193;citation_volume=20;citation_year=2007;citation_issue=2;citation_pmid=17313169;citation_doi=10.1094/MPMI-20-2-0178" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Lee;citation_author=M. C. Schatz;citation_title=Genomic dark matter: the reliability of short read mapping illustrated by the genome mappability score.;citation_volume=28;citation_year=2012;citation_pmid=22668792;citation_doi=10.1093/bioinformatics/bts330" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Pathog;citation_author=Z. Liu;citation_author=J. D. Faris;citation_author=R. P. Oliver;citation_author=K.-C. Tan;citation_author=P. S. Solomon;citation_title=SnTox3 acts in effector triggered susceptibility to induce disease on wheat carrying the Snn3 gene.;citation_pages=e1000581-e1000581;citation_volume=5;citation_year=2009;citation_issue=9;citation_pmid=19806176;citation_doi=10.1371/journal.ppat.1000581" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Pathog;citation_author=Z. Liu;citation_author=Z. Zhang;citation_author=J. D. Faris;citation_author=R. P. Oliver;citation_author=R. Syme;citation_title=The cysteine rich necrotrophic effector SnTox1 produced by Stagonospora nodorum triggers susceptibility of wheat lines harboring Snn1.;citation_pages=e1002467-e1002467;citation_volume=8;citation_year=2012;citation_issue=1;citation_pmid=22241993;citation_doi=10.1371/journal.ppat.1002467" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=R. Marshall;citation_author=A. Kombrink;citation_author=J. Motteram;citation_author=E. Loza-Reyes;citation_title=Analysis of two in planta expressed LysM effector homologs from the fungus Mycosphaerella graminicola reveals novel functional properties and varying contributions to virulence on wheat.;citation_volume=156;citation_year=2011;citation_pmid=21467214;citation_doi=10.1104/pp.111.176347" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_title=Global diversity and distribution of three necrotrophic effectors in Phaeosphaeria nodorum and related species.;citation_author=M. C. McDonald;citation_author=R. P. Oliver;citation_author=T. L. Friesen;citation_author=P. C. Brunner;citation_author=B. A. McDonald;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genet. Biol.;citation_author=M. C. McDonald;citation_author=A. H. Williams;citation_author=A. Milgate;citation_author=J. A. Pattemore;citation_author=P. S. Solomon;citation_title=Next-generation re-sequencing as a tool for rapid bioinformatic screening of presence and absence of genes and accessory chromosomes across isolates of Zymoseptoria tritici.;citation_pages=71-75;citation_volume=79;citation_year=2015;citation_pmid=26092791;citation_doi=10.1016/j.fgb.2015.04.012" />
<meta name="citation_reference" content="citation_title=Septoria tritici blotch: Fact sheet;citation_author=A. Milgate;citation_author=H. Wallwork;citation_author=T. Geoff;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Plant Dis.;citation_title=First report of resistance to DMI fungicides in Australian populations of the wheat pathogen, Zymoseptoria tritici.;citation_author=A. Milgate;citation_author=D. Adorada;citation_author=B. Orchard;citation_author=J. Pattemore;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant Pathol.;citation_author=A. Mirzadi Gohari;citation_author=S. B. Ware;citation_author=A. H. J. Wittenberg;citation_author=R. Mehrabi;citation_author=S. Ben M’barek;citation_title=Effector discovery in the fungal wheat pathogen Zymoseptoria tritici.;citation_pages=931-945;citation_volume=16;citation_year=2015;citation_pmid=25727413;citation_doi=10.1111/mpp.12251" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=A. Morais do Amaral;citation_author=J. Antoniw;citation_author=J. J. Rudd;citation_author=K. E. Hammond-Kosack;citation_title=Defining the predicted protein secretome of the fungal wheat leaf pathogen Mycosphaerella graminicola.;citation_pages=e49904;citation_volume=7;citation_year=2012;citation_pmid=23236356;citation_doi=10.1371/journal.pone.0049904" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Plant Pathol;citation_author=E. S. Orton;citation_author=S. Deller;citation_author=J. K. M. Brown;citation_title=Mycosphaerella graminicola: from genomics to disease control.;citation_pages=413-424;citation_volume=12;citation_year=2011;citation_issue=5;citation_pmid=21535348;citation_doi=10.1111/j.1364-3703.2010.00688.x" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=A. Persoons;citation_author=E. Morin;citation_author=C. Delaruelle;citation_author=T. Payen;citation_author=F. Halkett;citation_title=Patterns of genomic variation in the poplar rust fungus Melampsora larici-populina identify pathogenesis-related factors.;citation_pages=450;citation_volume=5;citation_year=2014;citation_pmid=25309551" />
<meta name="citation_reference" content="citation_journal_title=PLoS Pathog.;citation_author=S. Poppe;citation_author=L. Dorsheimer;citation_author=P. Happel;citation_author=E. H. Stukenbrock;citation_title=Rapidly evolving genes are key players in host specialization and virulence of the fungal wheat pathogen Zymoseptoria tritici (Mycosphaerella graminicola).;citation_pages=e1005055;citation_volume=11;citation_year=2015;citation_pmid=26225424;citation_doi=10.1371/journal.ppat.1005055" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=R. Raman;citation_author=A. W. Milgate;citation_author=M. Imtiaz;citation_author=M.-K. Tan;citation_author=H. Raman;citation_title=Molecular mapping and physical location of major gene conferring seedling resistance to Septoria tritici blotch in wheat.;citation_pages=153-164;citation_volume=24;citation_year=2009;citation_doi=10.1007/s11032-009-9280-0" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=M. Rodriguez-Carres;citation_author=G. White;citation_author=D. Tsuchiya;citation_author=M. Taga;citation_author=H. D. VanEtten;citation_title=The supernumerary chromosome of Nectria haematococca that carries pea-pathogenicity-related genes also carries a trait for pea rhizosphere competitiveness.;citation_volume=74;citation_year=2008;citation_pmid=18408061;citation_doi=10.1128/AEM.00351-08" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_title=Transcriptome and metabolite profiling the infection cycle of Zymoseptoria tritici on wheat (Triticum aestivum) reveals a biphasic interaction with plant immunity involving differential pathogen chromosomal contributions, and a variation on the hemibiotrophic lifestyle definition.;citation_author=J. Rudd;citation_author=K. Kanyuka;citation_author=K. Hassani-Pak;citation_author=M. Derbyshire;citation_author=A. Andongabo;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genet. Biol.;citation_author=A. Sánchez-Vallet;citation_author=M. C. McDonald;citation_author=P. S. Solomon;citation_author=B. A. McDonald;citation_title=Is Zymoseptoria tritici a hemibiotroph?;citation_pages=29-32;citation_volume=79;citation_year=2015;citation_pmid=26092787;citation_doi=10.1016/j.fgb.2015.04.001" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. M. Schmidt;citation_author=P. M. Houterman;citation_author=I. Schreiver;citation_author=L. Ma;citation_author=S. Amyotte;citation_title=MITEs in the promoters of effector genes allow prediction of novel virulence genes in Fusarium oxysporum.;citation_pages=119-119;citation_volume=14;citation_year=2013;citation_issue=1;citation_pmid=23432788;citation_doi=10.1186/1471-2164-14-119" />
<meta name="citation_reference" content="citation_journal_title=PLoS Pathog.;citation_author=J. Sperschneider;citation_author=P. N. Dodds;citation_author=D. M. Gardiner;citation_author=J. M. Manners;citation_author=K. B. Singh;citation_title=Advances and challenges in computational prediction of effectors from plant pathogenic fungi.;citation_pages=e1004806;citation_volume=11;citation_year=2015;citation_pmid=26020524;citation_doi=10.1371/journal.ppat.1004806" />
<meta name="citation_reference" content="citation_journal_title=Annual review of phytopathology;citation_journal_abbrev=Annu Rev Phytopathol;citation_author=I. Stergiopoulos;citation_author=P. de Wit;citation_title=Fungal effector proteins.;citation_pages=233-263;citation_volume=47;citation_year=2009;citation_pmid=19400631;citation_doi=10.1146/annurev.phyto.112408.132637" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_author=E. L. Stewart;citation_author=B. A. McDonald;citation_title=Measuring quantitative virulence in the wheat pathogen Zymoseptoria tritici using high-throughput automated image analysis.;citation_pages=985-992;citation_volume=104;citation_year=2014;citation_pmid=24624955;citation_doi=10.1094/PHYTO-11-13-0328-R" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Pathog;citation_author=E. H. Stukenbrock;citation_author=T. Bataillon;citation_title=A population genomics perspective on the emergence and adaptation of new plant pathogens in agro-ecosystems.;citation_pages=e1002893-e1002893;citation_volume=8;citation_year=2012;citation_issue=9;citation_pmid=23028308;citation_doi=10.1371/journal.ppat.1002893" />
<meta name="citation_reference" content="citation_journal_title=Annual review of phytopathology;citation_journal_abbrev=Annu Rev Phytopathol;citation_author=E. H. Stukenbrock;citation_author=B. A. McDonald;citation_title=The origins of plant pathogens in agro-ecosystems.;citation_pages=75-100;citation_volume=46;citation_year=2008;citation_pmid=18680424;citation_doi=10.1146/annurev.phyto.010708.154114" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=E. H. Stukenbrock;citation_author=S. Banke;citation_author=M. Javan-Nikkhah;citation_author=B. A. McDonald;citation_title=Origin and domestication of the fungal wheat pathogen Mycosphaerella graminicola via sympatric speciation.;citation_volume=24;citation_year=2007;citation_pmid=17095534;citation_doi=10.1093/molbev/msl169" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=E. H. Stukenbrock;citation_author=T. Bataillon;citation_author=J. Y. Dutheil;citation_author=T. T. Hansen;citation_author=R. Li;citation_title=The making of a new pathogen: insights from comparative population genomics of the domesticated wheat pathogen Mycosphaerella graminicola and its wild sister species.;citation_volume=21;citation_year=2011;citation_pmid=21994252;citation_doi=10.1101/gr.118851.110" />
<meta name="citation_reference" content="citation_author=D.E. te Beest;citation_author=N.D. Paveley;citation_author=M.W. Shaw;citation_author=F. van den Bosch;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_title=Phytopathogen emergence in the genomics era.;citation_author=E. Thynne;citation_author=M. C. McDonald;citation_author=P. S. Solomon;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=S. F. F. Torriani;citation_author=E. H. Stukenbrock;citation_author=P. C. Brunner;citation_author=B. A. McDonald;citation_author=D. Croll;citation_title=Evidence for extensive recent intron transposition in closely related fungi.;citation_pages=2017-2022;citation_volume=21;citation_year=2011;citation_issue=23;citation_pmid=22100062;citation_doi=10.1016/j.cub.2011.10.041" />
<meta name="citation_reference" content="citation_journal_title=Molecular plant-microbe interactions :  MPMI;citation_journal_abbrev=Mol Plant Microbe Interact;citation_author=R. P. Tuori;citation_author=T. J. Wolpert;citation_author=L. M. Ciuffetti;citation_title=Purification and immunological characterization of toxic components from cultures of Pyrenophora tritici-repentis.;citation_pages=41-48;citation_volume=8;citation_year=1995;citation_issue=1;citation_pmid=7772802;citation_doi=10.1094/MPMI-8-0041" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Pathog;citation_author=A. P. Van de Wouw;citation_author=A. J. Cozijnsen;citation_author=J. K. Hane;citation_author=P. C. Brunner;citation_author=B. A. McDonald;citation_title=Evolution of linked avirulence effectors in Leptosphaeria maculans is affected by genomic environment and exposure to resistance genes in host plants.;citation_pages=e1001180-e1001180;citation_volume=6;citation_year=2010;citation_issue=11;citation_pmid=21079787;citation_doi=10.1371/journal.ppat.1001180" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. H. J. Wittenberg;citation_author=T. A. J. van der Lee;citation_author=S. Ben M’barek;citation_author=S. B. Ware;citation_author=S. B. Goodwin;citation_title=Meiosis drives extraordinary genome plasticity in the haploid fungal plant pathogen Mycosphaerella graminicola.;citation_pages=e5863-e5863;citation_volume=4;citation_year=2009;citation_issue=6;citation_pmid=19516898;citation_doi=10.1371/journal.pone.0005863" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=M. Xue;citation_author=J. Yang;citation_author=Z. Li;citation_author=S. Hu;citation_author=N. Yao;citation_title=Comparative analysis of the genomes of two field isolates of the rice blast fungus Magnaporthe oryzae.;citation_pages=e1002869-e1002869;citation_volume=8;citation_year=2012;citation_issue=8;citation_pmid=22876203;citation_doi=10.1371/journal.pgen.1002869" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=R. S. Zwart;citation_author=J. P. Thompson;citation_author=A. W. Milgate;citation_author=U. K. Bansal;citation_author=P. M. Williamson;citation_title=QTL mapping of multiple foliar disease and root-lesion nematode resistances in wheat.;citation_pages=107-124;citation_volume=26;citation_year=2010;citation_doi=10.1007/s11032-009-9381-9" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Utilizing Gene Tree Variation to Identify Candidate Effector Genes in Zymoseptoria tritici" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
<meta name="twitter:description" content="Zymoseptoria tritici is a host-specific, necrotrophic pathogen of wheat. Infection by Z. tritici is characterized by its extended latent period, which typically lasts 2 wks, and is followed by extensive host cell death, and rapid proliferation of fungal biomass. This work characterizes the level of genomic variation in 13 isolates, for which we have measured virulence on 11 wheat cultivars with differential resistance genes. Between the reference isolate, IPO323, and the 13 Australian isolates we identified over 800,000 single nucleotide polymorphisms, of which ∼10% had an effect on the coding regions of the genome. Furthermore, we identified over 1700 probable presence/absence polymorphisms in genes across the Australian isolates using de novo assembly. Finally, we developed a gene tree sorting method that quickly identifies groups of isolates within a single gene alignment whose sequence haplotypes correspond with virulence scores on a single wheat cultivar. Using this method, we have identified &lt; 100 candidate effector genes whose gene sequence correlates with virulence toward a wheat cultivar carrying a major resistance gene." />
<meta name="og-title" property="og:title" content="Utilizing Gene Tree Variation to Identify Candidate Effector Genes in Zymoseptoria tritici" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/4/779" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Zymoseptoria tritici is a host-specific, necrotrophic pathogen of wheat. Infection by Z. tritici is characterized by its extended latent period, which typically lasts 2 wks, and is followed by extensive host cell death, and rapid proliferation of fungal biomass. This work characterizes the level of genomic variation in 13 isolates, for which we have measured virulence on 11 wheat cultivars with differential resistance genes. Between the reference isolate, IPO323, and the 13 Australian isolates we identified over 800,000 single nucleotide polymorphisms, of which ∼10% had an effect on the coding regions of the genome. Furthermore, we identified over 1700 probable presence/absence polymorphisms in genes across the Australian isolates using de novo assembly. Finally, we developed a gene tree sorting method that quickly identifies groups of isolates within a single gene alignment whose sequence haplotypes correspond with virulence scores on a single wheat cultivar. Using this method, we have identified &lt; 100 candidate effector genes whose gene sequence correlates with virulence toward a wheat cultivar carrying a major resistance gene." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
    <title>Utilizing Gene Tree Variation to Identify Candidate Effector Genes in Zymoseptoria tritici | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"fV-GQFH1ZnTaRaSD3NJ0Lsv4MLSQgXOkRM2_KfZ7BYE","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460130","apath":"\/ggg\/6\/4\/779.atom","pisa":"ggg;6\/4\/779","ac":{"\/ggg\/6\/4\/779.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/4\/779.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/4\/779"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/4\/779"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/4\/779":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022m5MG8REN44hcDe2R71GVCG6XhQSJhsCAbec1XrWmi+Be1eBTDHfmA9Wq0OwJfibg8ZK71dDTSkRWuDsSDGbR3kc3KPIFcYCj+A9w4fnfFRIPzspUBj9RqfctDiP7VrwwBivAWK\\\\\\\/XDCP67vZHIn2e2iQR91nyclrKdVuN54p69uNkrurNID9IKvrRJNvh2MUa5Pfkby83BRbBR1Ef66ujPW44xVCC7BoSICv+vYquvJaAJz1tDX\\\\\\\/zZPnN8d0tB6imVqH6bA29n2EuuAZwxrsYLyBZjeBpZwupP+ze+LPhE8ZuPfFBGIYG9twuNbladq7QZmQiQgBsME2FxZLQL1WXyQtHn8xJVWRCSPBHIMJGTIkc4oC1lAJpqaWmwejIakK0pOnsBpg\\\\\\\/HIPweyvKvI3aPVZMve0AhQT80i+eU9BwzTzYvwMPhUDY8S9tmYqemb80rAMsAXO1UhM9VO4zVg47wWLzxBxBTwS6G7czutOP6ZHO2VHPJJccWK4DUGjhrsiNOh4zH91ZMd96dzbztMykE59m5vYm5Ob3fq+t49SdFXwZzDYoNx2nRcR4qrC\\\\\\\/gadNljXWNO9NAB+QuxUspu24Soum\\\\\\\/9DX3GiZvFx+xA4iWe9R2ruLD45VTZnWyonIBasPZUJIIe4Amp6terkojVQg1rNOQho1WsYlpU6QxHpVJ9fFZdHq4UakBB1TsIows3Wk8APLfPqbJrbh1UevS06G8\\\\\\\/oZNVlNrNK0yvqPxtpKNu+X99exedJrKgAA\\\\\\\/CYXWG2EEXsKSGsEEksNhqf6jIVZADbMZuf4Fc95u10jIzFnZ3EVuFk2I6asRk5n5gRWNcJwX32VEDT47+S4Wdu2NfjcpZ8Q0jt+m\\\\\\\/I8F78SPICuXsHZtAvBD0HkZN6lGKNqU4WAQozkwjZwEG44AUiJsznuUDGXx4Hq2NSBHiR2lr7B2ht0v+tWq9kY2ue2YBOhGB5NZhZwQpozTlJK0PTx2f7BeO1mxdX4WnBL48PnhbjfEDJ46SDyuRPOvfqBGqZ9N4ol2hLQoi3DfAVUtnHgeAX\\\\\\\/Hnqvbh7S35+7kGy3RSDz3DkE7WUDnO\\\\\\\/fALV7GYkO9s7u3hlWoWyK44NkU3DsnuC6ZPE2E9aSX8EXbbItTX01fFvLQpV1nHQdLmXRBNVJs2X2cERolf4lUoS\\\\\\\/qXFsekiS3uJtj12Iau+oxYsIzTwPFbOkq+QUeWDIp2cvcWPIN4pUKBdCnS1sRYHwzSCTrKD4v550Am0kCpBiGVjDnmJJE9T61pdyJ4Z1NdEXpLuGrrbH\\\u0022,\\\u0022iv\\\u0022:\\\u0022GwJgXV1PIbfQzNO8up5PkA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002263eea2abcbbc6c3914f5fc5dd77400fadd716d948ab7e1e235e527ca84e99c59\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00226mdWViOku80WYhIrCTA1XEoSHy0M6fm2EZ0IW0o1dQGvBI\\\\\\\/tVJS8Wc1iOiEW4LBlV5ZhLg6QLXhQhpKVUJYZCMDS0xxvA1Uj8DPmMOahmkV8jD7IEcSeFJU8vU\\\\\\\/on5ejeBvd\\\\\\\/vJHbTNPdvP4968Q1M\\\\\\\/p36Eh\\\\\\\/k2kntznAXWB9R1cDI5l1GyUg1A7UAywlX13ESfof7lXhZjPbunY8fQtzOTolAzN4UlUMUuLGVcQ4534Z1aAw9LFQPevVTj8Cr4bVIWURTIqflbV3B9HAj3k5VBYpG9trFvjRNrsR4qT\\\\\\\/\\\\\\\/IEUV85OBxKq3jiMWpkYMhkbQlPUsqwJYwQuZInUCKP1+DYtwqeOxnlDGRgssjpZfUH4hjK1L9CwVtnkokNj6f9WvGKxgjKrleJnwJhj35ygV8o5vVTQTCYRVM6wm7BZqYEchVqYePv27JvSfRyJ2cRpkPYOVMuC7pWPy8k2bYB1ahQBEW82rRCZ\\\\\\\/cKtqmdH84TK4ymu1b5ntuUY3uwcaIou3v7Pv+EtVaDtq+9yP0HH+z1NFJCnQKCaNd1i47UUyBeIApuBKNrkl+Yo6B\\\\\\\/HehB6MY1QPTaYlv\\\\\\\/MCYBGHT+wB9fkcm0VVQ9zwcmNoXQs+v8KotYhiaHUVdwqy5+OWDrn7ecpZjPObrd8+w0+JgRu8dS\\\\\\\/Wh2BSALxfDqUjj+Fc4ibdbsXnwdbTgkwhgqII53mmk16TD2zoecIi4PkwuiFX8o6DxjfA\\\\\\\/5VSsQNWhhpe9vY0t8hYlBCuNRrQrlswOECTrJ3nnPG+UJPtPHQulIJC7Van8qBpQKMjyF9WtLodwGN0jkATt8vkAJMGpJRbHcOSc8PkyPxP6x9L4VQXnZLozFX8lworSezW9qi5\\\\\\\/DMNo3+Jb8QruJlsme\\\\\\\/Mta8FQ0TSlmglvQM9Jkp6HOCS2Frbd2am7TjLGXUja0olOE25nKaEJNG62p+V\\\\\\\/IUiWORBoHj\\\\\\\/X8ME4kOFanxv9STHY8zbe4in+g2EOc6rZ1NBu4\\\\\\\/0uqfOAUQiGZuS0gy\\\\\\\/ug5rUkWROkuvKuLuwuPxgWd0\\\\\\\/pTQoseNeiAN3nG7L3etmGRCBXt5IleudOqFbcefqYNcy2npGY2vm6t0xhfYj0+4EgCU2P7ZnuEg\\\\\\\/MtQHCY4FqjjQG7QFHRL1qoMkbNRtQYWOARjVLKshTQpdhd1\\\\\\\/B\\\\\\\/UEdfZBqqkm1cBn+gSbi8PRjtKfA\\\\\\\/25OoIFSqiTrccBo8JdNe7LbYxdbshOCiMSjkND\\\\\\\/VGmJpbmKPK3hkhz7w99gfk++23DBOz1cUlAE\\\\\\\/SWrqlvB9tf64u1r31ihtwDMMW+eNtLkO69QyC+2oSfSwBs05sIwlRB6KAwCCQrrS+ZaSHIaINFvkycatrTkzlE4fNXA517xeJGxTAOHh\\\\\\\/g8qKmqebEe41yMXr5AnNUC55P4VZW2BS0gxkyLzB1dC7DieK9ady+hSUr\\\\\\\/T5V3X82vcD1bxfYqPPKZlfNnd4teCG1oRGeYHukWSZnhjyS0JLvieyh9PTImX1Ss6UFGaQojhwM1ktjuwVeUsTK6FpNUe1cchSIVGJL6OGb33YutrjS2kBJdV9XmPJ2WxLfTHlBN+YCFRp8l4\\\\\\\/QTJvW3pfIcajqZ1f9mjzT+Hupn9R6GFPs\\\\\\\/Q72Nhug89uKcfqBzyhzmXegyGRLh\\\\\\\/cJHJU7cljc7xFZK+9iIw54iXfIiULQA7TJxybusxl+If95mquCzHldbP+ugGhkN4hnoiDs9J2o0lYulraI5YLOESMpPcuhWwxxUm7Oq+CWJMXHmkij29ifIYUNzsHbOHf9QDLRed1NR1PoRjjITvZisqKwraTBK2Ty4C7aJaEN0K0iKe36SiDKZwFV2jQtbk4vn3k8ii8ilgutBDfGxzOgZW\\\\\\\/djMrlV7WYS3X2nBG5hdG5QuhlEeepmOz2tmSqaOzU9MNTh9f5liycE08IBZ7BmesbYdpNaMpO14T0I1Vf43adGWVJoLC1drbhRjHEk7HwWV2BopuSWaFSXXeSNjxUJ1MNjb98AV7RtJjOCL32\\\\\\\/OQSyoOjdSHoqKh5lCwMepV+oihwsx9e5ipMNMcEqDNqqQKz7mqu+5oEKxOAyInpbLAguSM8zQ31FzCnVQs+9R870gFJ3ubuxg+ErFP7POjJlSPpaHayYKB\\\\\\\/8I82BDycRKthDyN4KLFk+fL2wsScQYFvyXrv2qVN\\\\\\\/BapQjO9OwFlXs9+S4SYWLFmeY3SVTVcskKpx8XvqcSBTt0aZAzWi2pv80e6vOXyqqDPEZJr+58M5fGTI\\\\\\\/JCMwrJElDUN43uo+c27SUVn9dnFxN\\\\\\\/Ih1e5wnbtTBJOjdB3Sh5kpvJwJJ0pmNZu7kTsha8\\\\\\\/x7+kEM5LoJoF63dvMR8hRo3QSoE6Lqx4Bs79rsELnpL0ETss74CO9fsi7PdqpE8IpooMJXdNGMNwESBepfzFpdLl27gkjKQD5Zq2mWebTE8sGvBm6VK1qIIsXEyU6Xlm5Mta\\\\\\\/wHtA1lZm6ljMigBM8O6Qc1kPUCeZL7ZCybmK0Bjmb6WdmeQQAxcgRS9ayHVYisN3xMuGLETYOJDfzZsrF1xY8CkSNOHTD5PbFsx14eCX7bE11snGJFJmcbGSqnK0N0TeNjgIgMUYCW7bKGusqEJYHtJJQTRStSSi8k4DLcoi6051fWfAMCECzv7wetH+I\\\\\\\/WH+Ljj15q5ZJTngJcpSlNA2wyltDI7wFMz6mEvMKqubfbkhfNfrYfb8dLpwv6gt0eWlB+3yAE0BTgMhOJoNYtgIUEQ0eQt2KmS6Huo+klr6KtXEoy9nsqdADYM1oTOzeyKWEPxmV7c\\\\\\\/VZaSu\\\\\\\/P63EfV6xzTd3IjR27yMZkqDVInM3cFBfI8A1QG4qCrU1oc6+tsyCtU8gw3ZXiO7pHehHItPMLwx3txU5G2YgM3ldBm9dYZlQayVvnRAa4Lans6W9PRWy81TSRbEtxrG0Gs5tLViU\\\\\\\/0uoeu22QeGw\\\\\\\/CJsfBNSLKgyDbiDf+eiH\\\\\\\/3xYYGFdtaCItAuinHQvtsqtJIMpmUwuV9Bw0fufuGuTJYigfSLt4ziVS1YeS3+5vpy7S3dAnynwHTTkVAqy463uyTSp6fHUs2p61uztBp1gCtMu9m\\\\\\\/PeryUZu3k8z5ysvGJszpY\\\\\\\/3+g25Cwz8kAhVyi5VhRAk38Z4YBGXVHPGbyqT0YzOeSOOj17v8AL1DfNG3VS5jQdEbIYg\\\\\\\/DVnnxDYVFICsVY1\\\\\\\/D492vkdsa\\\\\\\/f595xZJYBla6lQRxiVzpJ3JnNCTgRL4LJUCK+QW5uSbWCsWSpuoTIxClge6uxAtaQCthYBxr14DssfRPzZlRrMtGZnkZcu3f7ME4evPIAnqouH2hdvnwqDitD\\\\\\\/Xfv4rLeV09744JK69HU36JVSBK\\\\\\\/18yTRhkF7JHdDd+3wew8tumOiRLpn8XrzowJNSzBA+dSlJ+AAPH2Wak2mSfPTwcclYChyo7mL2x2QB9FpAXz6qkdhIPWTWBft\\\\\\\/cRH6Gg558bUpJUvJEYui0AasJt5v+VwA5Pn\\\\\\\/w0NnYKqKacN4FSsg1vKao+eazty\\\\\\\/eyaerZY6QzHfrHoM+UkNvlcZY36eiEW0FlITp5ZNisi6eD\\\\\\\/fLGCZPgnDjM33wNj90S0hQHsksgt2t9NTus9XPZsQ3xMpcw7qXuAuQXPNEskivkMOebpe+\\\\\\\/6e0c\\\\\\\/9u7EUCPeeihrN0Us7FiArPqGvTIHmg5CZBsxeQmCDXtnJ3K1DrbIltHpR+JSQgCjyTlWN41sWD+xwEClt2LEoLbgzYpQGfHC\\\\\\\/vGD2psdeZTL0u1MKsEgS+9pD5yJbhPrlZX7QGwG31\\\\\\\/tFKkYfgUaOv5vbogRozjXec13TQjWMYQqKCFMAZ7yMZLls+OndiWgFY+HpI3Ih86LXxJ74J0eQuZYlnl+1VNq2Ubo+EFtGXJkyVS2Iq96FBqen3+l5YNOLHCibHRHxNxPj35YbUei6Sjhks6ffnCF\\\\\\\/XlU1XffsOxdGibFafrGGQ4awlTG3au31SMfke\\\\\\\/zL9t\\\\\\\/LLmyninrxGyi34vWzysRYlqEGR3xi1XoeWP7NGp2VPocNalfO3g+0ILTCGZGACaNzQsPDqX7BR2Uep2iCaS5WNIbO+1Ih5fVX\\\\\\\/NLd3FYhWwjJx047\\\\\\\/cDZuK1dknbjUuyp52MpRaPccSua3ZVAvnTb7eFLeZdE2YgUXTavDMsRswoUtDJMwQnWEp9jwkazEsvE3movHD8CV3\\\\\\\/hHKyHjiSrXKES7JwSojmQPHw+m1wP5VflARnQh0yULg7qNsoeJWy91aONFDLtxGpHQXnwWAl2DNmeCHDDvZ4yS8gPUl7Jr06NrkcpJYYnw8DGkM6JrdNSrves5Z40VjlJt9a9eO6+NkU+vLEr5mgTUWWQdI\\\\\\\/gvFDgE405K+ReAVancFZ6S6R6WQvw9cfPZUjG66ulFqzcMuu8dHpYWq6ljn9pFKd\\\\\\\/J\\\\\\\/0yZ\\\\\\\/uEIkffPA\\\\\\\/ZK1BMUGM81339iydx\\\\\\\/JOTeSaSyD82QeoEy9owmBsObBxuDvxpe9kW\\\\\\\/Ozifbbl6bLGoomoHZbFviNISax33M3ZHvIKRTzfl\\\\\\\/gU8QoQWJZwVBDyJ343YT+zXciRhVOg61FenVBB81TmVg1SSQ9aHVt2Ufg5L9tvCF8KZUkAsfGvwJhOh2jhKz70fZHNLNeprBDLuUgtO6ViA+8S6HiGEUDdD2u7F4SgFxpxfWKk057A9UnspBhUTjNUVZK5rJxuLNfHUn382SBC9vnZyz6\\\\\\\/x3oMr1kS4b0SwDUEFPE8e0uSDmV5vP0Zle+Hgw6sMqU35AuqxsrkCLvNvZNjE8\\\\\\\/QHeLDmU4rnJGNYAJKM2WJa6GlBtJW+8v6LP1w3irtWhHIFXEsjhu\\\\\\\/a+kVWZEuV+CJrhR10BkzrAp6icYa2hP5qJZZdGIDWRFTBWVuBI5VnWw+EsvN2\\\\\\\/P9HXcBkim2QrYLRr4Fmw9FU+1Dpx9F4CAaCKjLp8XGMKFXr3tdu8nGk95ucokqZutXL14zRXLLDYDDGu3E5yKqzHyb\\\\\\\/cjlzHxP7MpO7dkGGfqAz1jh\\\\\\\/I0p\\\\\\\/btrhmHnP\\\\\\\/FL6buo7Nu5PP+eLV12Q0ewfk0u2PaeV6zd4KSgTPBhJq4ADpZmAFEt7yEKouhYDIbbpfB6JISKyNHaTOo\\\\\\\/a+y5QogMH23jGZLaa6zfpBFaf5RHfInqD4qOaOJeZbb7scB0JYu6x4IFbeZMx2JR5VTSR44oYnm6Ao2toPjUPLDW7IUbAW8KbczuYc0C8hTJkyITjICf8\\\\\\\/Un4Io+c74mX0ygIeDQzSB+\\\\\\\/0iZqFNHV1NrSehj907qrb1S65mi9MASh2EjbC\\\\\\\/QonbvtBBbmQ3ZEV3WGeob60mNokQbNa1eDYnqIrahdJvMnJqiIAIv8ss9anePTNMc8Cf+Nf+zaee+YjE8l7x7CxW1yRFaG5kKXxDx+ozD4IgG0b0sCmBTo7M7xSSBlscfxwHEj2aK4uJ\\\\\\\/Z3Yny\\\\\\\/f9NbbhzxTXhQg+16Tv5AzIKUTWU8qDy4DliQv\\\\\\\/5Uy32z6IaxSrAKvV8gcduDgJy1ZhbiXms2L4+BuKUamRNsdIf77opHMXQoB7NoIwR+RGi+7qwQmXs\\\\\\\/yUNhOSIMzc\\\\\\\/MbV9VwNmQa1GYstoESHFPcBWkldKZAUSzsEE40VTq7\\\\\\\/THqG0uavLrDCy0\\\\\\\/0BiVTRRJ3w4MPaPdAC0k6yVLStOEGrXNqYky844T1v37q\\\\\\\/PHp1kAzicdyuFX9hWyZW7a35Syyv+ZE72sFX2sfBAqbSdoEzmaz1ve9EJHZ5lcG5zxSIyfH4E\\\\\\\/NiiACfiVpYOSBV6UmuE7rROfO1YyyLJuR1aKNP2bM784FNHq6AU\\\\\\\/XHKvPU4+GeuR3w2BfnlYdItXiYVHiA5pxoKYKFlrMZbWl0913W1Lqb556iqB22pfq0zyFh92qjRkdLSl0ruEy+7cf0WN0XdFf3jLjjwhfabLANYjHmCNwhnwvo+uMYaTf0YZluV9Dukl6bmkg0dFGV1QMVeGxQO\\\\\\\/yxYf1FVwXP+p4fziuNi2bK7KEW92Lm40dDUrgQbny6TcFgGrXiaqEx0Z7F8c3tDQ1cHttICeuqtnasga7XVh7Id38SIAUgEiLkmt9BvxFO2uQefc\\\\\\\/Nd\\\\\\\/6G\\\\\\\/s7\\\\\\\/YvIGBWWFPKqOFk5jf6qFYVRt00\\\\\\\/0odRo6fJmgqvfUTHwYvUvge7vk6fAe7BO1MLSpCvDNzSZveNQDmpL528joH7srrWWnUJDKwVXHniS61L0yWlrbiaackH+ohufq2+61twAEMf4MHxQ4uPs7M8C9JjunluGRZMp2u63OI6ppxHb3fa0y+tm6UKyuQoyjC0hdVVIMwqpSC7t73IvQGQnHQvKA1YtJpVXGEWwBHgJ+S6kA61ZgrT9H6JzQZGbQcCq+AR8f4H\\\\\\\/WNIF2jYjEtidrIhpbHZwsO40z2WYPOndb3K5Vl+BuXAHto7lgCHOpYECM9CnPxl30u\\\\\\\/KlOsBS+V1NCdDq2LlHSHNr9Fkc1uJSd7iuD9QQ9axsYpiE\\\\\\\/M\\\\\\\/dmz39POe+NujdsVu+knaDUeuHP8IEjRQVmPN9C0bKDnuPVtr7pifLyKrBnI7tpTUUcRDmQ1M4kT+ilt6yMQ3ETdPfqRHppK47uqmKEFcLRkeyPccZ+Q8wu3luPO5UVlaD45y2bpaspUY3YFEHBWZQ6Hk1WHnS4XTDi38LlgsUwYIX2YOuCZsWZDlEhXIGFpmLJcWQR8PIKg2fYQSc5dicY7upFHMdMzqEef1L4Fy7LfIHwYnEivN6T9HyP8oTrPVLM0YuX\\\\\\\/cHva2kipaYSHliuWXuEzZ\\\\\\\/lBR7v\\\\\\\/9bmKrtqR8UNU\\\\\\\/L+hSNmfI9ishAlM4aOGn\\\\\\\/D\\\\\\\/Z48eNmaQdqTbbf8K6o7IAP7HJPRcumdH3gfu\\\\\\\/qfJ43aG8Ib6NzZaJmgc9PYXk1WOeQm5G\\\\\\\/GCNCfIm3vxOJS6EkzpOmivim460dexYU91w3QGlAuu\\\\\\\/bnOA6aUv+tG\\\\\\\/Wg9mWoYI94JgerU0YiqFHHBoBOvJTvX0rBRM+58dagduc3qKQEIpJvFs9MBXbUiYn7reSc53ZNNdYT6Nuzd+wp4hlaUS60YrNe7bDCrVaNMXuXuBg7EHzay9E0c3o8Q5tCCVugLm7091Yj\\\\\\\/z1swFAwQfssPzeEquoM3NxviJ3DmT1ZaLBfCvB1u3VZTIzjTJKjNh6HsPiYqtPbKJnttye1M5SSVQTyBgiHS6BwYOa7vPFWAPGdCLlBVXTMH8RWtk7b9bZhzo6ks4hoU9LWE\\\\\\\/9YmLjOQON48B8QWRXEQonXtczK96F+i3Ida5X6bDhIlm4eCDp6rlF1spRyuVANEaqTbWTv6+u8EsSqWVjJvUJHb1mmFmadXfrESQdoo0+yZKOfrIMtoAVv0tiYwwAp9E3nFtkQlZltAQ8CvPO\\\\\\\/k7ZyDFqeas73OlsabpaNBiY0cIkR5WiNJpcmWk5p3qssBNBTsRYGFZ1dnVYI+w0MG557X2c+YeFFruASXKSxrjsxhhvxMN+Z2fUqPxk3BBPDcNm2T9Vo8kubkAc7wulMycc4khdb85yQ7iVFuyNNQ+gFihKW7SYc7\\\\\\\/92K\\\\\\\/klsPt8Q8L+JxjQ5PrqpjuCvFsltywl75O+gqHFQiwPxDle41GS1Kv0RSZGq+HGDGVhb3In4PeH1RAH1rLvD1gYV+edDAEnKv\\\\\\\/2Ls2\\\\\\\/L7N9R1k6FyFtPxlDbH2+kinzhB1vRumNaHOj8KtZXbG3lvHW9ncft8BIgtLqFwS\\\\\\\/Sl+ICAPc27bWc9JK15DVJREkaKObaMExPXq9lY9kvIiRinYquwEIVXi99nlTILBWOfLQI6R\\\\\\\/R4RB0a0hx+6mwu\\\\\\\/0oEymO7VO55b38u8qitPZGItA3GsqCiWu978oAMk3GN+mbJ6cxts8g5FQPxRv2zV7dD2jIhAEmFkGjuSOMBU2QiaoAp8ltS8JXucKWLQKLwV1htrJzqLVB3+zdaUD30no0VcJbywVIyM0dXxOyZpZtZNMm6HiONmWH3HYkSrdnLYX5lcGt5YrHen2IuUJUdpXvUagnGeL04xcpSjbS5+yB4e3RXf15PbH2EEZaCE\\\\\\\/t03t31udSN6aZ+ZdwAU5jFuZC7KYhdJkvckG6Pv3Pe+ZceY7T5tM54HY1aJ8SzOQVhnxYIE\\\\\\\/X0ouUh+0GdBZnR0\\\\\\\/G1KXWtmwQnoKBkESXGsxdp1VLKoZpscW7tySeCKuNioqU1jHblePEz5Jl5x4UOTLxoYK5HU7h2RNwhiBcQ1O3Qgmn8QiKrTkeLri0Muxy98PZxda38lUrd2c89hmozDBmHJWm8XLBFDk182w1dDfZW6JX0EypA0WNGtIqqwoZH31WjpCDL1Xv3Tyz0mTOpstgrHhnAOHoyfXoEO2czvU6WVgISAk+ljaVcm4uezumPCXe\\\\\\\/8x9LY8ol68VnBwHYFlE9mEfBHujhNqekIZVuTHfDGe35zcUtCbQhmby41FhJNvns+srH8zhwa+nFB+QUCYt473VeuELhax4b5yqq5E8VMBftqDPjwSEEJxbMGg7ijArrRs31umiHHq6Wk4h+rVbKgL4KgCooXaSbs8qZiMz8V6j\\\\\\\/C29\\\\\\\/udM+DkBF580lxxXQvlfb5+EoP7jBObN7u0pi1ZlVbOR4ufjlZprL4eCwWJOcKTdKbBbT7rSU6nsT0RljNctF\\\\\\\/2LuoiaOaEnY8iIJjjXokguGpORaXNhTVhgl1WMIdDnEUS1ENbtI6591pQBodQqhknwFpA7cy7pZ\\\\\\\/L4bytZmlv8\\\\\\\/XOPksqM3tw3CSn+uYkdTtb5a7ml4xtBaLRAKsWv34idSjJbV8sSWAK1oTGWI\\\\\\\/ZA96e1VNkm3MpSKm5AIWwDQnX8h2XT7YepFEZmLw5xETdwr0Dy1Mu1BJDgO6iEqzmgU\\\\\\\/WZSjn32101GQgo2R3qGkkrV7igrZArmtDF6tgPidKzwtFvvcakqqAbup+izghMo7K0+OUwPN7w7DDZFVUvhC6N4eGMiqxtYiTtFPqIcIswqO1QQuneGF\\\\\\\/w8mEW\\\\\\\/6LEYnAsXEkEC17nZ3iAX2e7Ly7ju8RFpSwKH5VhyJ7Y67znvX1nvdQ0oeou+4t5IOLLxeztMG\\\\\\\/orXi19LjwAfTk6R4VHoJTXCiTJEUfl2y0pXW83rMa85+Ko4gAYy2O719H8uFh\\\\\\\/aHWvJa5B7BIRrThSzbKcsR9B8TyoctS3\\\\\\\/eBFrnGm4P7+zfM4C\\\\\\\/wdJGokzY4zLxcjppDLJtgoVpatTqjpFtLDN6yDcAvBAlicnYRprekFGZgsNQjPHagBPafQ7e07AdpzunVyiX4B8GbTCA6WygZlOI1qQ9U\\\\\\\/yArviUOFMps2IwayvEaGHRpSOzkZIGlZk6ZWK7TsftcVmgtrUMfgjgpJY73k+gLBhD8zmgQdCxuxmrIRSWr\\\\\\\/J1L2CF6g3066QYqroFzdZN0AIezXu0Ad7IhWuvQeqnUBhXkx7f0xXfPERjAmUYxML5GW0b80zORNPRsgvYOY5mGpMWKMc9wAdDKX3O8MPioqIanOunpDo5FfwS8+OG\\\\\\\/E4FT5UG2JnBcOBTEfnygiHo\\\\\\\/\\\\\\\/COP3i3SAeJHUVHg2coBaModQYLoGn5LuaL7VaNdfOSB\\\\\\\/NeNRbPK4Jg5i3HysF+Eukq+hgn\\\\\\\/2qWFUyAI\\\\\\\/SD0UTNaTH2MJVdvu4YtMOmsSs\\\\\\\/ehBTjXLfe20Q+mk9xkof4UalRuKH3Y7vJMGGcJ5Ujwj\\\\\\\/5HinTj9vLRl5ZUGsOXNjbxWUi1gk24qYpPp0GTkeZy8K1d9NyLBAPIjl6eTPFNTHhdhcOd9MQVJUWCQLZl5klXvCyMQ3b\\\\\\\/AmmoVMc89mciBCnEDQ7jMFuVAfQWrdtDrqXIdUaSRBZFFgROWb4oPqRr7HLY4nP8ZqhfBTQH1\\\\\\\/bkiREt\\\\\\\/I2mPjQirtizG9fAGJ1eHh8JZ3oJU\\\\\\\/b7AaHI9MjFC6O+x0Dv9iMrSpq2bM7deMDdGuMYpH98FyQDpc\\\\\\\/oSD6AnpFt9gTo1T6LKPts6sR8M8jWUmbTSEX\\\\\\\/gPuDidyKQ\\\\\\\/cLePRWCi9chzxJnC+vTtbizT3SZl0D08L+iNe8BzG\\\\\\\/swRBoJZkSJO\\\\\\\/6vFSbM1QEFrlFSCHslZFIvLmNrApOAWT0LZ\\\\\\\/I\\\\\\\/B82W2zGv+Cihy9+r6RXjLQ4nHDUR0uUJYBJ35kqyTuEo00RMBtJLCFckx4mA1S5dj0gaX0CqrFaCqyIAp+Yk77R9K4V0aJ9QR2RwSjuUueTuao4Rb4mXhwW\\\\\\\/mMYHtME6+Pe+HDoqZPQGe+q1yfJZ69YEnq4i5qK3FixQotrKpvQPnMJvDug0KhLy8ZUB20ZZH3bEW+yeLuqzKiU+YjJVZ3QpG2ScccWdxvO\\\\\\\/tVUkhjN3wSQx\\\\\\\/OwWXIOZJr7m0cD0fQ1fTzgtd5sbrOFI74MWZ4aXosC\\\\\\\/7yQqCrQ4Tn0zZjD1GJfhi7P9UlqsSxoUjY0QI8+AgGdq2ZSbPLq44hS+RVa4rIiE1TzkG73zdhwRSPUonz9uZcCuAF9n3+TRQZ7Li4eF1S6KZ4gtUPx+1vmPTD23ZTXVukkJbufzIKXNYqdRkcgwNG9ooPaweAq3mec6LRUX9N59eT6QPA9pECSx54cDsVKrs6u9jh6Wh6r8y7PA3GRL+6p0laad3miW4Iam38JyysTS0t5e3TADSSYmzOVUUY9OIsa+Qa1T8IfoH+qK4ETlXp6SuWPkL0N2gydes7fxmychK3uoIT48Ru37bN1G+0I5y6t7HIICuqVvaFXMaRUN3D0IlQxfjai5iGRZxG20q+EDAXmSrkddII0jpssx40jAjOG+8UhxKijDZTSpq9wBzoPJmqobPAKbnGl+9B1kwhzT7bpFyVyIFopvjJ3ap7\\\\\\\/RToK6egRwUFzigrHqS6dcFHH2CWZIH\\\\\\\/68OFFTD6CK73cYc\\\\\\\/dKHWEH+hava6Ri7IJCmsSj\\\\\\\/EAk0WOJ80fA13bCYEQ+TN7cPMV2EcJaUnd\\\\\\\/b2f7L7lfJuQg72Wr8hpDLu7nieAXk7d0wDcGNV4889eVrDB7OYEEwvOh0fWkvnnvzS359++4iofT6T7uYZO4yZGADBOuGIPgXn69zbvNQ6QpSa+ahE8msUQYYMzhunKelaTAEZ9nzKCkFzutbrY\\\\\\\/6Jm3SYzmUUxZjSE7VbI2gZUtiVXmGS\\\\\\\/uDDmpdfxuP3Ot0AfY\\\\\\\/aStFEtwcpzuxiijqEe\\\\\\\/GyAbISyvtyJ+P2ZvpgJ0ROKFdgBc7Iq7s0n1gOuD17TVvwuV6ufaH8CAt1Xwj7tFg3xUI27ZHNczKq+55Hdx4Hn6FENoDlrHan4n+2YyeRHbJHJQVjCRBvH+xL4JfMyFFadL3ZnHPSZ\\\\\\\/7UAE27dQOCnAFvEbq1vDW9gMAHzq8LRO8ml30lhFe8EjvCoMYwEhGIw3gPEViEwYNqE7uAxz+69bDrXML1VM3qEFKszTOFSpu8M0COnAO92edKj9yq1tfZPdHorgUMj\\\\\\\/jTMglGCI4FSuFuwR1kFKxEd\\\\\\\/yB02V6XlldZewPCRZMxWeCdpMrXQ4MAjXZzc89V89UqkPdpmSy88gEk6vySnA\\\\\\\/piBW+bAluFH+PSDR+dX9qzH4ktjlglP+jwP9O7bWXlXXEnQevSU\\\\\\\/iBJvTHnGOLGzz8A6Vdt\\\\\\\/OeYWMclDPnaLlks8gePKnBCUqN7LTg9BcnXPvkFr1LbrsNgSAqIfD1S6od\\\\\\\/eho10S8c9UKBk\\\\\\\/6qWgdYszkhoNk1Ej3GN94YsgWsqbAEaLIbp1RVVHmoTL93y7e3p9RTgWGVP3uwSMh5xPsvvgATKETrHhTNk2muQ8g\\\\\\\/6sZz1kkkVotK1WmASTUjXbeYBRxop5fNJxnTK0PAOJVbez7Capan98oFKJ8ERtrPFZYYi8O11GZ3Uug5PZjHE+KeBbjQunvJLJXTiezcf0YBSXVlvWSQKcQDWUF1v\\\\\\\/29810dzpYxFOuUggAVqG\\\\\\\/BuS8wbrdeyNhR4mZTnXlNGGFS3KaLfGpucS8lKAFjy3TmNp1IyC6oE3eoemD6Ylx8I3vAiXnX+tocMgcs4y+ojACSeKXiUR\\\\\\\/Nqx4Vgw0C+cnChHcX+RLO1gzzAIOGxOuaVqGr9lviPz0zIsVB1NSxVEKd8g1qkb10zSvecgGiqIKs4PDGoutb\\\\\\\/VGcNSqkLfSRHpjuCOsPtkWCdPY5NaLMA3BYSl0eBXZ4jXo9fLv7Fq4aFgf96fwN9aYJSLH1\\\\\\\/TyhX4kQ6GYc0Ex+lpbFgOtFMxap50Fa0hmyDNMpQmoFki0ckSg7k\\\\\\\/0FCfMSzD8MIntEIxcAqVR0E9KSb3p66Z3WfquFaf1mG3Alfd\\\\\\\/MvoE3Hi5W8j2\\\\\\\/jXS8M6uzjptUUQHlAk80AJKdhvDtodTYsrHN9MRc7eGlw1pjd8FUmaoMZTKKUupUXXxJQwvvpIbAO0KJ2hY+zQ6WhMvw88r0PfOPIUNgkIeKv587qnU8S1V\\\\\\\/xRHvrU4AqZQhKfQfhzHIhfOUKJ+YP+Z5AiPF02OURZDFUA4tOptdd5HDadpg1XnmKgcCr96lO3Sr\\\\\\\/ZlPoA8wBWoa2ifKufFXlpN\\\\\\\/nb4CNZArRsky0Q\\\\\\\/oYQwLg4gpng4uH4mQKNPddC7xWZiP\\\\\\\/zAX91lxaa1bVPagCj62l7D\\\\\\\/i00qWfh1r1HKy56f3g2oKnUG4Aen6JlEsrWcbZia8W9ydKLLqKf0woa4wZVvls2+BxiZhRgItvOAviXVd16R4TLYZBVFV3LgbMaxD0kxigO+YXAy6Rn+wTOs6IQ7hKnCP84TsOQJwAiugVQ3CoxaMQ3FRSkc7FserQ55mCWC5HfvfEVqpuFH2ZlB0Ynq3j2luXvS2rbFuaHmUSleWbXkFPp5aeBkRXfSOKR3xAMYNoqHZZf3tormbUeiBk3K1t\\\\\\\/+OVcw7HVCq1wViIqx2uTRpBoIkl2GNu3V8Zw4+FdNv6jNv1gZiyCJbm0T3vkZ2uty11titctf+9GEuo3pnnj8lgS6e6oFDA2OIfPSgHf9oiI2jvQQutduxwnZKyprDcOlnH\\\\\\\/r314tWfMx\\\\\\\/YtjnM1+xNShVoc4w6EChogDQ77R7qILWSfeyJEKLNAmEvYf1xMuwqmSjU00uJlG0WYa3b5m63TS4ZOJ6x9TBBBWm\\\\\\\/iQtmSSu8IN7oOSJvGdY5UhanCp21eW\\\\\\\/d+5ofcOjjwDjU+yckXpQ7NXzBcwX86H6EML3JTfByNX7DDsVQ433SxHMDnvoVp9d1uYi7CORVWvM0Vd7Duu4gqMkYSsS\\\\\\\/3ZLYPHAWUuIw9i9THfvem9+l9+2WjKLHIRFQalJOHVdjci1roH38X\\\\\\\/CjtIdUqImL5v+K\\\\\\\/mbViJDXfWnmfonzn713P4b6iy\\\\\\\/LacWa3oOl9mCPnZt\\\\\\\/KMDymVRj8ROpQ3rLXglE4js6PT3pqVG0UblfvddG+gd3CW45w5TerXhQQ5WZw+\\\\\\\/DqoJf2CHQSKlAJ\\\\\\\/PvD453g+eZLseQ2MYKwFBmvErLK3100DOXVPB3WB17YhEyhFfunOJH9qzRpaZrB6zFKKiVgUlpcNEOt52J7vKQk54Ih+ny67JsBGE87Gy+F0hUMkl06lce8EmEyfmEhvAO421En2+pYVHOK8tK9CDxuqcIhB8lEtWMTApj7nTdsCA29nMHzFsnAw7YPrB5HSoXwfQHkMYXwfQrAgxgGnAokBr5xJOd8gak8NtLHrviwDX6bqMiGxgerDPmLpK0I7NWjJ3T8s372\\\\\\\/Wnvcp6AtkJ7YEe3TENB\\\\\\\/vI2lXCnOFDmzBPGcXhFO83\\\\\\\/F\\\\\\\/89\\\\\\\/kv6yC9Uajk0aw+VJWXIqhEJddgO\\\\\\\/qHcsgN\\\\\\\/KZ6\\\\\\\/qerV7FUnHv9Cpht1ull\\\\\\\/tESkKVuOlmhQLAbJPU1y4PocizGZZ06MVc4ZO6YVzx5vY\\\\\\\/zu07xGKajmIxPSEa\\\\\\\/4sww8YgLY9+V1FHeRjNSvS44QPSGr3CtXVxiDAwNeeWrtPircuMEAe94dO+KfTJbpb2bOHXnp20C7ljJZTnL0oPy8wP985MB0p7mVRNsCLnhB5+9DmVM37i6ZohOevWj6i8bEyMlms1n45BRA7igsDpTo1nomenSuM1ol6DD0vXUGk171eM8HkvXRZ2zCf4AyWQFGyPlji3aLYxd4HfLaG+EI8iqHU03uaohgB4ntl5KRBSn3+vUW4uoObSBMRBzD\\\\\\\/dxX1Xd5zXGJ90EJ+QHNLTZlcUAY4pFwqjxbv4IUUc7qyLHYf3+UP20u6Y6KniVB0thR4dm6mhI4yvvdPoZZk87LKqnqq5sMXVi55slNTKz\\\\\\\/53HON5jeAiFh8WL7ZH393PwvjIWHs+tyFl+TRYFrlEo16rEz+8zDYC91tBPBnHf+Lgv\\\\\\\/pPrxVWyYlyvonc0XAGgoGFPUH4FOUHruSYqpwgWTnIgp01j5NlIYQiN2m+3rYJEsm6VnmQ6IiMdzcMZamY25aVu\\\\\\\/G73gq7X40gW+jWEDgDo4cQA4mddo99uuCVC6dLvg+\\\\\\\/UliXulaGwjvWk3mWjGW70xajf\\\\\\\/1KgsH9djBsxVFc9ZcRiLeP5HORkXdfbqrY4q4x3V6GPMYQbJs+anTURBduuupHqvcMTv5QjnHFZT+h006aSCFs\\\\\\\/nRsLH3hhCuqOIZ4n45\\\\\\\/90u+S850CSyU\\\\\\\/yxsZyOIyA1YWAk+L1GNpBcivWyom67K27DJIksO3jvd2uj6qgMP5ZeofSd08chQSp3m+BX8uUmfGgPJ7\\\\\\\/qcKF1dE0p+BF\\\\\\\/btHge2q5cU00gancDtHJkrcApQK+BoUKBGbwALBwNs584aWyrirITuvcnaQnINEbQ+Ft48a8Ly51\\\\\\\/qg9qjy7mvRsYl2ezf1nEU4IJr+KhxQ7tWhT0LJSBvgh3NNUiIgRa4grToj6pPtAzJih2U7ofB+xUriPD5mHjdAakN+g60KmvSaiPopumiNh8wFp2fgqHYsfcj3w0m+u630A3toHTbF4KEVKbpbhxRnOrS7T8PjoLuLBVTmvAA+TG8H1yjO4sZR+xPdyYP90w1Av+d7cQ84\\\\\\\/8OtjJlchhMzOny9D38sHvB5v7DXr7xzLm9rOwWSyg73nV5sl9jjHkO3wknYr4gM2N3kIg167Fo97lA9Nxx2zvGqczy7rHwCfmSMYa5reen9w32UNVbpSvWAkC\\\\\\\/KUKy7g5pP1ywohhY971ajMGSEq2E7ehu1RzH0keD2jXKMyZQAf46szGFmiwcrs00ZM9ZCt9ETRVz1caU9FBfJTY1Fe5NaZkR8VyvyGFKlJw8fgWALmI7nHS+MUgAb2ibTfeOfJqIa3m\\\\\\\/x8pTo5jgP62Ftvav4FQCfOsAEU2\\\\\\\/yQ0bHV2dyENsakXJyYi9eYtaLsO0vdXxwLoGoH3uHpQlkFz5Xx6LycvQubU9VJQIyxJXB\\\\\\\/+fB3GpXsA\\\\\\\/+ExwPc4lF0R4eXAY1xvk+XTpKktqUTg6UuC31pdy7NJok8wIKi0BlkDovrEvqSh36b+ud2yCvk4TMm0l4VyNyiugYoPIB2d48zg6gbizfMOuCpe+K5ZR6M4igE43W3W8U+iS1VLlnNQeuoeb5hhlEMcQThAgsYWohyCD8B5g2ihrDRKZeH2fA41Aw56hqZYkdEYQJytqkUR6bHwiBm8CB6Snk1J2QBtWC\\\\\\\/f\\\\\\\/RZ000ebD7eVzmMV4fA34A9XRIz8Ln+Bh+KPUferdSatT0eKjxWM5FJWRWZPpIYaObXsjJWkzqjINWJ1JVWzfdm+kXHcLb5D+EbsFQmtRPybTWP23XrbA+B4ydak1v1fnOoTv7dKwTrCqadco1L1pR0uZldit0Dezt3jL1\\\\\\\/91EWkv6LpKWkoKFImDxuV9PHn6mIeXfzH9iHHtJ55xmzHfHc7iq6ma6mJNbTjwIzutWeCk15fIqMvuB8i9rGy1fgrknxAi0Pv6JJy3JIuEKTfpk3LeXTEIHwEl0CURiCrH3nqFujHghyOmOoE9vp1pkhggeij4HgwfVgqm01z6HRuzGfk2bGmXHMG+qOYXa29EMV47kWTm3d8cjqj2eu9bg7inkDO\\\\\\\/Q7upSuSwj7Ld7IGgO509JtqCXDCn2K5OCBZvKKaY\\\\\\\/mvnf1FQVz4Xfv0K1ooZtoWT3YiQHRMOYMKdlHUES1\\\\\\\/RIwu0nAU0gUCMNuOiv1mIfkUNv4fXEdmoTMW5rd8Jag8mSO7uWg+zy2zpQK+f2XM8L1G\\\\\\\/a0oJgDPQIDcvI3tCbcn4FEOh7sHZYNJOTWbm+9ivrqbSu3Pe69gq6Eecj8PgzR4zyOsJbWoz5xA9Ph+NNPWJADmwZW9dO1yN\\\\\\\/gixyK6Bd7oIMt2YGsOu4Y8GiZqSZWrZjMKwFS9JPUPckameE\\\\\\\/GHpre1UcPOpWNxyLg2UFbDz+nI\\\\\\\/if2B92gnI8LNJf+DQc+r3NvP\\\\\\\/9AQXOpN9JKjw+gb6IV+BQUiyS4aN4aSpiMxK267qs8Pub8hIMibGr1vH9462daBCicPt9lMFs2TTz3CVF0sb\\\\\\\/ZPD6T2DwsAsTB0Z+u1sSkKRGskZqpBx+rgH2cpPaezQJMayfkZZGNqPn4AtI55+ppBup\\\\\\\/1y0tvupgVb0qdCjkrzoJv5mORfrEpXTRHicpjvNIFHPcZFJP7nLTc1pT044WODGaFfpCK6rPNE5RM0HN33ytljgmAgb\\\\\\\/QiON95SRka9UXJjzuKZiOsXZyZ1Q2MJOOHjBeduql8XTvdTguIsmEnA\\\\\\\/1y22\\\\\\\/lAwX7dG2bJdJ8p1a6b41bVP8x71S1Xq3EtTKm\\\\\\\/1FK2grPcG2xdwkXGl3ulhB\\\\\\\/Qp3\\\\\\\/Y1NDfpAGr2XKVHvsOTh7SZIodHIyvitSR0fTvAfmYx8AC44VLXMqM4+KgvhwNzX\\\\\\\/tFL1Wd0J8zhSIDd+\\\\\\\/Sl3SKdO9ekDJbPtgfvlW5mDIHPg+ReZ2\\\\\\\/E3Y7CafQHY1s9g8ZHcqFAJ6gcItJAHHx1hny4Csh+FBRZJjoDRaGUl\\\\\\\/v5gN0XB0xm\\\\\\\/dX0bsuH06RQrQozso1t+jbPhrercWBDG5D09wN3jrm2VI7YcXg+fsc8CkhZDJjXe+d6A5fZtb9eIujoVgj\\\\\\\/6Yds43S3eZLE3tj\\\\\\\/2Jbi5FphNz4l+q8PtT3gLb+u2Q5ibFNkvgpM79a+\\\\\\\/6t6sjCwf8Tr3g9YPZibeAMnsdCit2hnWw86A0\\\\\\\/i6\\\\\\\/JoC9R6JJSPUyprWYppQf+BD4SInpM\\\\\\\/h9sKnqiwc1E2WbXKP1tCPJYMDpdg03V7Kp4NY4sQhsiIH5rxTDcuwRoWSO5jTyXWY4XU6lITLWzeai6IvjclZj4BmWy\\\\\\\/qsRul31o1WG2pJB580Y+PAAC2miyzo1RcIvAlE7HfZoYlNWallQlm134xGi+ob+fJD++\\\\\\\/h3tR3k7\\\\\\\/rHvOpblkMZVnQaTS8JLvWhMaAnguJ9scb2YdoiVvVDE4B8CbJ3QQZU1Xyz3XvJqptDALfLROGrjyY0WdREdp37z9yJhZa1x0g524GZcw1LEzauF4oNXeNKeCgu9HjGuN5ezLvLm112459EPdDRZHSt7YMkfG7pERgc6SBh63gpVwNxcQ2S8YvOYGCOF3lPX6e5GA6KWDDxKmCEkQf7AVSoCP\\\\\\\/rLv5ApWa8xdnjjrJkcCpBoyad0PV1C9lx5WlraVWXikcw7nJB\\\\\\\/pdb0C7vyo8uK1H8x0I7heFwmKEc2ju8ScXuVXgCVXuz33AM5pS6UCEd849xkeWINu\\\\\\\/yLX\\\\\\\/+0HqAq12ILmA6LuE71Ngeuo1L6i4sJkDieF0G+ZRaOSczau1XyVDkxCxJHXWo8cNqr+qOpx+PKngPS3ZAMXImPMVfH0dYm2DXr6LGUvk6\\\\\\\/4ehlr+AT8w6Sn5pSwtrjhI5pw4D+tzy5pQLWdaZ1i6os9nwL7BUPS4dfCFxx\\\\\\\/ztHjriYcVRJlop9PIe15ZwIpFfHbdubC7jiMJAq\\\\\\\/x2Ml3NeYvEu5a+ZOz8dxnZs9TQoLU0\\\\\\\/+S\\\\\\\/0chYHcGzEBcl9EJ1khyfN3WmBZJmWXu6ku+cdcIg6cb9ctzO4VLmmxyED8MocmJGUUaSy8GzsWrsDDMaZhm2qgSEDJ2HlX4+3q6HF8n7RmU2Rk2U7E8wRXY5yaig4PMeFfnXzO3gpiOn0JQx0k10g4bhK5qwMWWxqXN6y\\\\\\\/X3n94f6UvRgOdV4MIHByge0qgJTgHBAqN14AO0YkTLvupOvBlxb9haMUxnKlafOQx4PRx6kkV2o+AYOoDO6w0e0RkNGf2s9buIjngimrn90XIeJ3CIjktsuk7hJIFPw1H7Hdsr3PpQY1diw5WeqOl06OzXSMQW66jQUmvJOeM0p5qHLsCV4cg1sleY0hTRKwYtcv3p8lmLIuWw6v2c1ACoEIlNnq6u+1jlQdjqMPRfhhm7PwlQknUm95eMWDwojQkhKBXLKvWwFvVzF3hCMpGbaDOaKUVonVpFZhHooDI1H6AQwRd+smZN6pshocBlpRZUxXUnyakIUtGVZORZWJVBu+IhZK3igz3OW6g3D8YEqzh16lozHPiGq\\\\\\\/4MYJQDi+adhb6NBtctfq+T1kaT78pTXyf9GWcZPtzHgeJav+Hhk0LR3X1BRqgSWwLSPPF2F449ZLEbEv7ggK+hafGsoP1UeSU86vJRUhU12bNzPfUjW4fYhLQvlv078f3HIwhh83n+Zx\\\\\\\/x3BbjW9LE0hcFq+r5xfgxIKafnPuN75yNuNtczpLNxeALBMHMBfJej8qibQrGj8hng5T3Lmjx1IoDjiFok5+H3odZaqFOghbTK\\\\\\\/y7SfnFg4Nv7r7utLjA499p\\\\\\\/15IaHcxA3pd2RXrlj3uHXqrCCAKd1eQLSwas66BDbhnCeu2P4oAMFTpQlK5Cpp0vmdHT42DkfpISJhfbpB0iEx1rQGbDNfw9cA\\\\\\\/VKQ3A4fmJ3s94dn\\\\\\\/iDwOkvAvKE0BLhTGZNnRSK2otq4twcXzTsVIMciSrMpU4m3k9pWqWTzUpUAKfljdKDD5odp925\\\\\\\/HD6+gzI5\\\\\\\/OGrior7NZTV86EhYyRKUSVGtHl9zG313d0vCGM3ccOSO78JjbsOs9452138lw6ZHYH9k61crO8hBpUa0qLR\\\\\\\/1iKpO+0Ebf+svsE5Tx+FQ8LdbnYvOVPxI57L5IfIjmW3SY7cH7taL43lG0nP+5huRayiGjzDDeV15fXACqnWtKwDCoiYyW1lARE3zXHWvtzVlmm43p\\\\\\\/xGGogvtUpxGxAA6ADu84\\\\\\\/sLfoY2VHLCMBF9z3fg11WJD3FCfvW9iJjSkwsASfksEEWiN4\\\\\\\/EOQJLYIRiYTJFbYeuzT2GwxZLTcyVrq9TtSBjfkqvBkJQD2taQLFxHt9Bcc0lVgMN+Bkki74x8IyRYslf28u1+wuXmHN04\\\\\\\/3kPNrmTDIvouOXF8VeG3sgGc5DWZtavVuiy0LHNu++3ywg9QgDDFEO9L195VVdC0dZFZQ4iQrgfK\\\\\\\/8LvLLvCc3LwP5va\\\\\\\/kx1dKMzkLfWxIjts\\\\\\\/wgT5jI\\\\\\\/BynFSU02lNfgkIcYtc6t0byL0v6PeprxpqJNipcG\\\\\\\/fc09gXYogkzxBczyrfdFQm\\\\\\\/NwkN+YX+c4s3Qn8DywUS03Uk207zX73HLlAjbCqCYDAdnSBt2pafGCTASD3+9jD1JSmCBOLiYl1\\\\\\\/tZ3RQUDIl9XIi2c\\\\\\\/a2OmTsPIZz1POAgy4iCr+i4Iv4Yg+6MpLR6NnfYvr\\\\\\\/VofuWuJmBpvzQwOVBa4XENfWw8gWEXQxumwtlyGgk4voTRN7R2F4Sk+TJOejx5Hlmxwdse8dt61NuIp8nlwVjQpSeJHdFL8J5L3P5g0y5ENsoHIc7mAZkEB+U797IApQO+q+toPbyRAFXfvBi75B0PH3iNgqBa16LZrrJ\\\\\\\/6w89opCNdrNy0GrBEM5lFuGfLXHkuFspFsqnQdyXZbRXQZzfT4aNqTahCeTi84VtZARAcDjQSo+oOAr33IpcEdJctMhcX4Opdebc1hTQsGPZ2gThf9Y0as2q9NGZbWGr1Vmuj7LVOswz6D+LK0+KVxYsfgSo5hqS2kSyM+f71Otf3BvQ2qdF7ymN166ucfc6aQFKhfX0ge\\\\\\\/YraHmSeame3cH\\\\\\\/AtqWeg9pUhkmqjZlHCUdbwOrUwWM6rnq4AtfGRFMq0z5cZIVECy11KduNH97EvQNfj9jFPCWaEO4ps27Brm6hfsemvflokpu\\\\\\\/lSNiSXIM0Zz9kIujDLcJ7yM45poqvtCci7zMpShM\\\\\\\/bTlNQOZzhJNqaK+0OYgAQMgBT\\\\\\\/sRSzJn42IzuyxoBjyMrbCnBwxr9vPkB+akW+DYE7eG2CiJNB38VBjjGY5uzs6VwB\\\\\\\/Stst+xk8P3KDQHJbEbwyia+idFam0Eou1Ni6aTUr67P7vPSkb66oJSAcAo2sdfhlx\\\\\\\/dGOQWdIg8TVzQDS9t19Q45mXAYLbLLQH6h\\\\\\\/NVS89JLUooc+bAw0526\\\\\\\/MSkwU5EWHTHTJUpmEkdVflmRc75IEz3psdldJ7cJmA9YsVZntMjaFtn57\\\\\\\/d1Lo5NramrmRPPCeb3w3nk3XpI8pc1m6Ng\\\\\\\/CXWSBm8lNfQ4uKpoELDt\\\\\\\/OhD6K9uFguFH3dJeTsWq4Z\\\\\\\/t0uphpV4RrbpqwFPKGj0D+ejQVxV7vJV3pye4TNAC7rXb\\\\\\\/ZTDzyumen6C1qsHWxB5ZK1LG\\\\\\\/fkBbBd5n+Jx32C3Hhj+f5y3MOf9KnI2YIqW+YNCOJ9Ulve\\\\\\\/hfTBm7NVe39wGDKr7zbSBHJzYa4FTp9lVb37FVJ+nsIUNnAM7CuKEQuMJXTleDZDf8gV3gPLh6RYPBUiQkF6zbinz7J\\\\\\\/ZPg6MzzQRWqZBN24bx1HS4Sct\\\\\\\/Q3FdXtxbK7C6lF\\\\\\\/Btd957\\\\\\\/5Lg15AORGsjLwAhKCEr2fdMJeX56jixciEwQ3Uk8tva5UKKOgAnv4cAn62mdHQfOUN4vawPhm+lFIltmDur1mEZ7c2MrA2+z7vBvRv5ziM\\\\\\\/FwzInw2PMzV+IDtAY3muEpyZglitRJQMuDblN12xH0o7\\\\\\\/wiTNFP2XY2UHvW9yZMHfpPgj8KiepmvNMhMIbiPtIUW5qUIIoH5B8AOTp4INRlKrht3Fbhu37xmmm8JagubqDztF\\\\\\\/4GZspz2Woan2fl9NKhqN850baxgwRIvfiHdSov5ojfhbmUkXaEcYj6cqLPRr9Dg+DFhqgQwNgXTUTascXCRzcr\\\\\\\/flhFh3ocKmiJ2E6mrf3421mL1fDZWcSf6hTOZJSzd6lAs7jC1OvDLjM0La00PCGv1ympPJdvshO3JvTNqNRHLC0YCITvKbVPAJx333xYjPe\\\\\\\/B0ifmkjbb4L0CsncCrgnathHYf6aTY\\\\\\\/KAKif0nGudEhRHqqkTJ44Pus45UU2pbsxwMjUPfMg+dDyvco4Pe5ve6JD7R1g0w1wM+1W\\\\\\\/6+XIzoG\\\\\\\/85o64hmAHe2O3R1Pj5vXKX5qiaRnE3GIAtKBQpXQNMNYFwpwUxsTYLCYcX1pnLDob\\\\\\\/z8NYtrqR5+V8lfTXhJDllKGbUyPRh651ak7VKD0c4b4W8CdYf2ofaBhtVZqvw\\\\\\\/PzRAAYrLFSeMOSw8EhjB6m6rVN7\\\\\\\/U9RtJE8Jz\\\\\\\/boJdGQGyI1EGDBnb6Dn49sLv05sCRME2kItwGPDGN4\\\\\\\/ryM7nRFJXPLHvrELHZQ1If5yateuxDEvU\\\\\\\/SpR76\\\\\\\/lA3i6Ndrgvbi3pPRQHenKL9zJOzSSKkCHGb1Q+aY2n\\\\\\\/yy1FQP8MFYfqLZ\\\\\\\/AR7At2613nP\\\\\\\/ueDYPnVPi9nW671GxPEByIozTccXv7upZDEfGedTE9P6DJqrRV6twKf98rEjXsispCcxXOpWOKnKTUZU2eXLHVYHeJHThF6gzOAyGpbK9wo2BbABh76z9Fe8x8\\\\\\\/3xkGA4om84RdSutUf9\\\\\\\/FNB7yexV24IQWWTJzEKDahrA5bPQcM7FtKlZ3ppcjMtax3Vk+As7Pi3s8mADamRfeKjqBB5vMPsSsip8lHzA1\\\\\\\/Vy2EbW9Zm9N\\\\\\\/W41y0FuJr3xX+lhJAcaoP1LKvM5VbIqCgL8KE9cMrun3DG5Eb+gcvnZgCJDJojfHZyc0JYzApAu4kjLP5apuJzpO6E6NpSPTHBytamDeiJwuLMWbsLFfe6AyjqokvILGUAojLANdtN1gQ\\\\\\\/0UF7fSUDehrY6wavkr9SqaneDkZDy3nk7hLYnwUD+oJq2oxuVWo5UIt6kWLCE60ejT0a9WBbiuqrwlDjtKe3dyqSud0tmBT4xp0bU2rpccTIUrGNbo2sQPE5QHWMsfqeOaZTFvJKK7fYZHgRAbsouImTZUbvkrv9n89zkaa7GL4MqpX7xUIgmrZbUFS3PlWkW10T7lLEBF6lFN+MzlbBvUA5iFodjEhXA0NZEzFvfShuk2r2EI6NM8agjICO2nZrOzSHUHiDJVLhqEMCg2OVlmQawEA5GtSLBQ58ltYF15PmxK8MD2cthdX0ESXn3hxkins955r2GfHs7iQ76aABRRwUly\\\\\\\/CQXQn1X20ojFYrPcu4VENPBGJdq4zWvwtW\\\\\\\/fuJoNPksbHcrAxaMaisLFlqkYlgvF7gPeskgrT\\\\\\\/gGiNlzQDNLLeHKPTRmpwMDW\\\\\\\/An5M\\\\\\\/dK66Hd1wFlsLCCwhhY+2Qus+cq9mkl1DhyVF5ANipP4rX7zgRks0bNOtebhjossVcop\\\\\\\/67J4LA560g3PELfGA+KBHrylaDCN483gnxYz4spn9kkhX6ljLhYoOn8lFh7bL4lpaZQ3FV1uo1P8EatTyFWoJS6NpXU4fjilCjstEtR1Xaz6VT2QJHMlf16uAxdHJt3IikZ0Sdod2UFQvZX5NHQjDuZvSB5WUUof26blnBnDgrLq1MeBgaPETxILlaoRsj638qnKjHoWombMLCK5wmU1sgAu84SdZcCA0K0LXR6rbcWlblIAoYwD+YecJps10XuD32b9CaOc3lMMXXPVa5wwoXhd1ezPMm+KXUh1KKzLEcFWhDC02Q7Fzj9qrEA3mDsyqn3lJ1KHbut9oN0tebkvSH43kEFGF9QrWA1R0Lsk6GpfBoLIM7j8zh65f4Is2gJzfZDkva30UiDG1pELXd7OjDkFj6ldoh8HqANUm2zvpxFKu5tZYVCXU1VXmLq8r5lVqAi9sC4kLBSn4fRvXk3C8RAxGFRbAwLzUOHXFwtfMAp0XPOjNzEHs5OL19cJfkxDfJ3tu9Vym9rPZPER\\\\\\\/LuKzqLMJ1rEHlEjYNhpy8RJJVpvePPWZBV4Ffi7tws2\\\\\\\/GaUlAFvdfofCaFHITFBeubL+npGWoHwaLJb2seNfwVdDBY2LsHbShxBVJ4t9H8gm4IqqbHRLzMwBR7Jgd\\\\\\\/TquScWaiSIasBsJp5DI2lyZ3xy6Bsrl2H+JGLH1po0muQaZlmp\\\\\\\/t6P5Hs+DJ\\\\\\\/S+wdW8xiVhB4XwYmNLDu9jiG6lG0UMto3mxRrpnGud1s50CYVa1MUB5AEPfxczpnTPNgiXqGew2YcVe2xq3XXgcWJ6mQe3xCv0uN5CsYx1IEMhG+bseJaQNp\\\\\\\/BN\\\\\\\/RBn4DXYDhhQUlgcGAqXDDeQUVEsR4XvunkwZT35v013BVbFezPa8Ch7fj9rQGs\\\\\\\/Zj1kLqO6rMr7945jZwfq6nc7bnm0p6GjGRD0vHVhUBkt8qrp+AoHF\\\\\\\/XiPRnNnHrv9XyD0AHJJeQ+DYuCbiQeatpZ\\\\\\\/gsUQp\\\\\\\/vwc41jmhqYB7Fnn7ju\\\\\\\/3XfmgQr+UKg8nZR0D+cFw7fR+u4VzInyn0VnlXspHGw3IckOgejm0erJTHHmt++7ePGOxlZxR\\\\\\\/qjgtgVvcLIwNbvh6Pb2Zesqy8QgpKK4puuMaPR+NFt+4f\\\\\\\/y2Xkzrfrz9T92GYrCq\\\\\\\/UcNSTzoxobaz20wtF2WRvHB1FkOlYRhQvrq7B93ZS3RH\\\\\\\/UW0c1muBwubc6VZpuQ1t+E+KkQcUrnTzbOZNYh48F+FiSwKulvqifE6lTpyo+2nqvTJCap7PK9ggi0DxxsW\\\\\\\/AgijMfmZyWdKRECEbZu2aWSpV4hj2PRVg5DpqnN9AM6wf0\\\\\\\/fNfoVdEXEZEHXcQHdFZtp8Ox6zrA3YenpSv24Fpvy\\\\\\\/vJcyAujfzrG3D6g+Ml+LdrnUWXX0NCGvtzWb+VoGzWaKajF6McRtrMrrzzg9nYuN3hstWjoD7atkYwkynNRwCzKAU3MXZm+JThevY5kbAHrCkBvdinpjSZ9w5KOssPVagymseFLZyDGrLTmtYtB4FOxjG2h2oGR1JPKNd5aM6T6uo\\\\\\\/fMjRGVQ4e1WmF62XtF8hbBaunzS3FYvET4PQ4UM7SlSPNGL+X14W3eIoMRG6cbrQmD7SO8rIv1Wf+S3s0e99HCxG0x6d1HuVBIiMHZOEvvtMrksITjNRcAlzu+wDWsGYRFb5geXJSiOv54o9gFvtcoy\\\\\\\/k8bLkm7sP1pWdbbpOB2ZcG4G7pTUEUtK9f8LkL7poDYlbrOhqEoo1TDiqYRrsWMJc4QiVTvFskazfOQCBi1HwI2Xlq2ckYD6OFNVsBwLC+8HgdoEP4p2s\\\\\\\/4afWVPFDGM\\\\\\\/84EoXSF+sQvprRREQ9gdfvsxXbZRAocASzTec5r7Kr24wixbOY15zBmgtVVpPdeYsbmoph\\\\\\\/E4uFz2hZjyk78nCj9HPkQmIWdtZSWdCfg55H+Ba9TWI9th3QcRJoGx8IXhY9ePXZtEmzJpck3l3XwA9ilSl4Zf3IwFV719JxXs8BPXd2ioZN\\\\\\\/zZX2qjljTfbB0zNlsdsEMDKia2Ce0hyfdbW1YKYZj+WXqxf9yFaBMT8zTzgBvA+SdEuVttGu0dOUgHM128nA+or0kJIyRqETI7Vly3m96KJs0egNiocrZ4pp58bsPKXBKsbaFziFfVKR7K6+gDbmsvuMkKjqPCbU\\\\\\\/EcZ0lMb7Qp+Lc6XEurfYkNF9tfrggcwOPyBHyuMbXUxkNyj8vxzHIE2jR9+xUvTVARWcT8z49Vd8ZFHzJk3EaY6ECD+7SvqNi8CwhnNMTK6acfjdFgF+lAgLcpWccGNViMWOc2vbPCbjotDsZPwix9idcyb9BHT4fn\\\\\\\/1Ilg3P+tVW6wj3vI2XkHnVsWFhnD1SOoxTh5flw1O+aqwAY84BeRWMQmyVGOo3v7ZwAVQuiXk9nVWjjfsg4fQVL\\\\\\\/2Xd1z\\\\\\\/c+YvfUvv8CHbbRIXurMDzqms4EMHrnYfERp9bzySa2hdYNkmQlZ3a4AJnaMqMnRy2W8Z3NS019S3t5YaC2ZVxURSuiD746e9O364ml\\\\\\\/xeoLYBdFQB7w6pib3YTtuHDtMENebNpUMk7o0RniL7daxAEy5CjRi1abQEjvBBsFXswi1UQ4Rzu95EQvUPp1y1J6Pe\\\\\\\/xS73yz81jAneyoQ5j6WSew9p814WaPQq57Dq56bFZYP1UDGPtTnlAXKQIsDc8V27S4ehE9F00LyTVOoDdJ6bxER2LE8B\\\\\\\/aK\\\\\\\/thSCZDcWgWFEtfH8Hwfm61KU\\\\\\\/zBNpLvucynrfgnCBOwglkZcp2dXsgcYNtS5U5L5RpiQIjWhWDJpvYteTJxH4eylV5kAObHmy6uSmJIuCBvhQ7\\\\\\\/HtgarZLYzslrV0jeA9DlUHv77L5+O3\\\\\\\/BGF+BSAiro4GzqAXmKfV\\\\\\\/HD1i1eOSCe6zUJIVwdbwpuJWRUhv\\\\\\\/DvQMX16oBrKbERxKqhG4\\\\\\\/WV9E8j5P6o3phlurpKBKY5BC1SNdiKkkj80HgdbFR4exBLKNTVqciXAiuEuB4QXvkuaDO1uie77kksrtknybbkXYY+9QmNu4nrTvk3gkLYBE2EWutapX925noog6tODmGG2Hz06JGW+vs8zAzt+74wYlscyaDoygegBRraPbglUiF80qc3YRVeYGk6pJp\\\\\\\/TWL3uarGJEvTM8nurcTEt+UhCN2nykCPasnNtWLTFfk4LA+zolv4V8xa2nawr0CldF\\\\\\\/W\\\\\\\/FCMDQA1mL0UrYWDxdAzFBZm0\\\\\\\/pkzS8Dl\\\\\\\/mGu5BPZ\\\\\\\/YArwL8jlQdbyVNkukZ42Jpmz1Ub55HomONK45v9ySZD8xBCcoylsFVLz8pk\\\\\\\/fXmc9iYrF2yziFvrcpLLT627hkIpTcinDVKo6XSmhX6baedDdNs2YDxq9QLTkJwIPaPbWVJ6+RXTqwDxq7r1lC5qkl2P18wixVxw1Qeih8BZK9zTENN2C23f8G5qIvWUDXMXE7PRdH7oCdThB+WeoP4Bvb07jIo6D4jzK0ee1FFNv3DL7kCcWyCzFjeRmyyNtX+TDpi2oqb0XPqqArH9J8fLN2oVQaZ65OittePsRASPXi28Ded\\\\\\\/L0+NRt4vb1x\\\\\\\/gwvuLg0w9RRatk3tImTId6q76dAcG9wayZ69sNeogoHZ681OONM+95AyiG+5lVLOhwXU8uR5GUjPS8hcEQ\\\\\\\/gC3x5WCxf8aig4DXsKUlMQkSM01Cbk\\\\\\\/Cdwa8w5AKZVgdbgS6uMXAMte+0godguHM1dJrpdxBgOPuhzSav+LycWk2giTBxULfUNWJHfNid8yrXOl2go0Vyu3hR\\\\\\\/Au9kF1QFzKKD4DWaUbOTBI2tpomGGsOvqRNjlgOxpKG83aHRBaJY2v1v4AjsvztzlgDxvjxO985MLFwaDfxaxbgp3NNhTuEfjc663o8lLiKNtAQbnUmvg\\\\\\\/EivkVTlPJcnbW5ZhGAhQi0X5NzZv4zRSqgibNQPzxmPteNi7mfPZ6VWs7rhVuBiGquzgUk\\\\\\\/wsPXK32SqdbIvO\\\\\\\/AUGac8jHBblKwHL4OOjlIPsN2cxLM8EVOI+n1bmWOQLIEa9bJtfo\\\\\\\/y7AmfAwEdRBLIvehMeBLhjeuLyKoqFU1UC1RBCVPpjuS2+QDDdBaPt3slsr75iiTnd0Ir63SandPzyk5MNoy\\\\\\\/BJMV7YCMLHKQHfwHK80Kve92Nivq13fAWlcOjxebSbZ7tJnOsxWilYwzsY+pMjki2G3UlECSywl9SZ30D0pq+G1BAbt4Ng\\\\\\\/Bc1CJUEH8+DhV71irp6SrCplkARiBWgHj7TbqCuDmV5uqIOC9YxICpcvLIs1ybkS2PKrw66x0N0zksK7hyZxMMXcp63XDHq0ZehZbjsG0r+A88Vpx0RgtM9rZikGEn2+9EpvjvPEMBo\\\\\\\/fbNZKA8B\\\\\\\/HnhaXKX+0Nl1PCMKekfNqZrbZlFPZHCyF6d9636YVirR6Zqvb3PbjIKDC94Mtm1+uhWQcyuCeujt4PmkWqn9OlQmul73qOy1Ra5em\\\\\\\/5GTsyHD9NYeVXZw3t17v2\\\\\\\/LUjrh7yowltcwggdz\\\\\\\/R21HM411LCXuJSdTop9RFDg2T+HRoiZe982djJCM5OtPAyn6q8IFbrZTYOdR+0hA24CpxScbZuugiKB9gIxmuwphw9P+q7oNgg+T3TyyahIFGsyo2BEEnG1f+YO1E6o\\\\\\\/gdGgEkpQ4GUFSDZWBz3XfW6tog9Enn010VphkzqwcyNePdevcV1OxPt4F+5upbFfjr4XFVpTJKfQ8XMbZ8bmQdvykn4OE\\\\\\\/eT6wCqbL9nay2grroxQHT3g02fLOAIFGTyiIZIfjdiGUe1T9VFECQg\\\\\\\/tvdgOoPgiEKhTo0PmCd3jlPnvMUAZo+2iRMQYKi\\\\\\\/vvRZfHiaWNLD6rE5jSHAsWs5dDMGd2J3JvCbu5tTe8++7s3x8ZSfzmcBKiZ7ByT+2I5vXRZsIkKWZjckH\\\\\\\/elbt9j5JP3tRuh7RY8bIx3\\\\\\\/6RiF8ipKGzqAEf7INlZuvQFUFG2ZogGbuvRB0+PbLtUZ\\\\\\\/PlhoyTkJK6jHT8VOQyjS6QLeJJ\\\\\\\/qdm3B0V1A4GOgZrlXBoTc05IfeIDYEhiQJ5PiZjpXF2fmLBhCmA8fZyGHb5vTJXSJTv0zAquAqKsHxZefmePKViXHTaLTjRVS0WCb6ZdmTKllgJpJxE44eTUAGoGzkEhGtl7\\\\\\\/jVPeE4sERUO4vXa5w9dqF2EDhALu77+p4+TrOT2qkfYkJoTuVpaY4RrClW5+JdnqkG37i1E51WOYbrhWebPP+EbLB\\\\\\\/q8to4ZS97IfxEer0IGTD3UuI3BAIdVEyky5mWY+yUP6804HsFK0Ipr9SEa7L3RY9aly0AsAaLAyoGV8PBtyhwNQQdEDQtnmZHEsSWnRhh6LS8Wxtn0+FRT1\\\\\\\/z6zpf6NMKJtDNLCZ1eMyYen3xKmvCMxzkAPGQa5vyg8aRiKCsCuVTg+J8RdLdBVGQwO2Lp2mx226yMX94rst\\\\\\\/gNJnzKIbGjJbey\\\\\\\/ravc+dNtJJKbOyeSIMg2ofITUv6lNlCLbQmKGOOjiaTfihNu2WZzItUOfOF9NW9RLSsTPNxT04Weerrv3dgCvYKaau0bwNC+o6gM4E2nWYqC\\\\\\\/jpZCWqjmHLAR0hhCiE8c5Qc8D6fMOKAZ9u7UNilHiJStt\\\\\\\/5r+A2YZTYJXsLSEf08pa4COEhM+2\\\\\\\/xIS8CgSkbHuNYbAMGB7jyGilIZJ8In1WX2NheXfvinZWOAbcd3MxuoeRipLgC\\\\\\\/tRzDz70S+WqT54\\\\\\\/ESoEAwfu3jhmD1vDuhC+25iwgWgcc1QBCEONvhy+rnziOqiAb3NncdP5LgPCo6S+Bk\\\\\\\/kSjVYGBQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Tdmjm3ABFd2AKdk\\\\\\\/QhyZEw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e869237484c5f52eb90bdde5783a9f11c4a04f253b5e3b1a3f7dc26cfcd01467\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002233WUS7V7m7jqNC+J+XVLeSiH7M4lMX4CjBdzJ8lNBwPTPYU6QuB\\\\\\\/HBVbDd0NsYNlpmw8Dt+Z4EPnyQ+a5c0v7MUqmKK469oaQDFbUP15an1nP5BLYlS9uWAZNrMEZTULE2514BzepBi9GePIw3\\\\\\\/1fFObyqHtVMSArqHrBuGdP6O0EQBAZNNbggT9QXCzNIyLfutuPN9tC4hnOxB6b7t8GLljoTcXC2C0hklmtIltZH2zmhM0vFZpcKg9VV894nDf0d43QKYvyyztWKa9BUbtKMJF+\\\\\\\/5Bui4pjMHqQyeIW6qHhddm3EFU6KdTR\\\\\\\/l+6DNZNJa8BY57\\\\\\\/NLukYcoE6MK6c6Dcbyl29XGfKKkFDQPl6XJTDItRsoqEUazZ1ei+ScGmYnk+75K8mXGv3xk8YdCI+0wMOQYqft\\\\\\\/p2yocbfFjXESsD874snlNujP5CPA6XLeuGVyWwUtPtXxBtayDnjamBmYmjy8K9DaCyUd+Vtf9OP6GlHi1xUAr5rVMTI4gTGDmkC0Uv2A7FSUHcz8G9Xk1B5iMUN8tQf5YbIUiyRi4YZEo2LIdzQj5sNL2\\\\\\\/fgel+\\\\\\\/PjdOwbXtA9QNgbnCYZdiNJht6mucXNCIc8g3gpvsWC3EFEwZgQ\\\\\\\/KI8dS2msr6hjhmfjUt21AO0TMe0r\\\\\\\/yJWA0pCdug2TvxVKJnoJmBOifS8FgdTLDKcHwz02lyysNp\\\\\\\/kebEAcZDeY9FejkgpEHjAy1ESt564IcXEkXUgAt6qlGMZhjuF6lM\\\\\\\/OfcGVUO2vdDIfAaFuSybLcO4xw2hf35L7mW0TdDdSzrZ1bcvVXOsFdZelY8pm3roUY5XdXPTY0Nk\\\\\\\/ZbxeDFB4CiM8R+zCFLkWr\\\\\\\/5whrMtBE7xIKdLSRpOKHy2PUg9TgkVks+N5ZlYRoDjV11EwboLVK58UUvdg+p6Z4Tgk80ShgYQGax7BbYwv0yXbL9zjMt+3hedzkk8Fx6HqaCA4XuZZdy4x5m6Yutocq8rAO5Kup5vEU7UmmVWapG1nxMvnzAMwErDj42Iy9uuRL6wBIF7fTkOb\\\\\\\/W0GmNXBOdh+dYQj0PQonm\\\\\\\/pPGENRSzacT0DrgUvIWWqbIiOLMiEDRSOLfODHsrCAYUsE\\\\\\\/32t0OvNSmBIp5KDHSO\\\\\\\/7phNJTfX8JYztGUeqXoKeMPLsQPgQRb8WQdSTCakJ9KTjoUlBda4RAMwyaQYqqTPnB6nQaPp4vu0e9xhzJb18mNJg4OxNRXj3dyCkMizF1EyzD0sepqzGuVEPI7l0Dj12Xwv2IqXWfx4dEbtOVtsJ9wN\\\\\\\/a+wUha71ZTRHsKIzQkmo924AQDjd9l7aaV6BJygQrTGeeJTpBCeXxBTkIhZDyiR2CCTRaJsXd0hjojrauMPFHOi+7two6lAupDQSwFr9bsQv+OsxC07a2Qoeumzp+D1FbOB7Yld3r757MFx3YjIw+56ZC0V+RoKGGyCwHStMEpoUpbqcmulFD\\\\\\\/cBroxssNPZBHVUvV2Unsvu3mGZCLvxUiQRafBwvqdNG8vRKaGv+HzEJIS7Mi+yaK9R1vZduePic+dTS6L9Xga2Uy2hYmhSQKskT2xo5zx2K6Kka2RmFScnBqfG5GcnlGARRhxBg9R\\\\\\\/QZMhzYWQbrPRxDlLkc4icCRJkx6Yp20nvnmqbFrHnf5\\\\\\\/sdc3x08jx2sVqnn\\\\\\\/UDTeGB8kqQXA98T7\\\\\\\/VbEad7ricgH7VuWxZeimg73pTRXZ6BxlPbKw93lvDJ3dUuvHNqeWSfR7av9rvbFmJDdtu0nqiZ+uzdVLuzCPtNqTuSqKCuRx6tH7YC7m9Z2KbGlnwfftk1Zca6RmX5FgQ2MvsxR0PtilAL4sICPYmQdO3ZyyeTKu2SP8ZqEZprEb39etT44uhC+VLS\\\\\\\/pnS7J4YKGV7OQZ\\\\\\\/VEf1mWp48JbONvSyW1IWJGiRZ6Dje3DvkBGhB9JnusL1XYCZlqcm\\\\\\\/0n4OnQ98ysREJ08kxVe89xTNNLsSEHQ5aFgbcY1XEAxa6jARD2Oiyd8sFT+JYIUtaXsXuPKv\\\\\\\/g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022p1DEkIY9N1j0nXQRu3O2ZQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220c955e3358e7f3e69b72208e58b8cef46ebfeafdad2cd9cb8f3c8a869ed020e3\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460130 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/779" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-2orEjJzuUEXdMsS64vuaB3QuKq-8rt5iZQVL4hgZmis" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/779" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-zEGMqPxTE_LNQf9181RwG0bCrwvtE8cU5JF8-KO6Yns" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/4/775" title="Evolution of Plant Phenotypes, from Genomes to Traits" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/4/793" title="Identification of Polymorphisms Associated with Drought Adaptation QTL in Brassica napus by Resequencing" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460130" id="node460130" data-pisa="ggg;6/4/779" data-pisa-master="ggg;g3.115.025197" data-apath="/ggg/6/4/779.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Utilizing Gene Tree Variation to Identify Candidate Effector Genes in <em>Zymoseptoria tritici</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Megan C. McDonald</span>, <span class="highwire-citation-author" data-delta="1">Lachlan McGinness</span>, <span class="highwire-citation-author" data-delta="2">James K. Hane</span>, <span class="highwire-citation-author" data-delta="3">Angela H. Williams</span>, <span class="highwire-citation-author" data-delta="4">Andrew Milgate</span> and <span class="highwire-citation-author" data-delta="5">Peter S. Solomon</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">779-791;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.025197 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460130" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Megan C. McDonald </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Sciences Division, Research School of Biology, The Australian National University, Canberra, ACT, Australia 2601</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Megan%2BC.%2BMcDonald%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McDonald%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMegan%2BC.%2BMcDonald%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lachlan McGinness </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Sciences Division, Research School of Biology, The Australian National University, Canberra, ACT, Australia 2601</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lachlan%2BMcGinness%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McGinness%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALachlan%2BMcGinness%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">James K. Hane </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Black Box Bioinformatics, Perth, WA, Australia 6102</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>CCDM Bioinformatics, Centre for Crop and Disease Management, Department of Environment and Agriculture, Curtin University, Perth, WA, Australia</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Curtin Institute for Computation, Curtin University, Perth, WA, Australia 6102</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BK.%2BHane%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hane%20JK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BK.%2BHane%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Angela H. Williams </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Black Box Bioinformatics, Perth, WA, Australia 6102</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>CCDM Bioinformatics, Centre for Crop and Disease Management, Department of Environment and Agriculture, Curtin University, Perth, WA, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Angela%2BH.%2BWilliams%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Williams%20AH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAngela%2BH.%2BWilliams%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Andrew Milgate </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>NSW Department of Primary Industries, Wagga Wagga Agricultural Institute, NSW, Australia 2795</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BMilgate%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Milgate%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BMilgate%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Peter S. Solomon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Sciences Division, Research School of Biology, The Australian National University, Canberra, ACT, Australia 2601</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BS.%2BSolomon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Solomon%20PS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BS.%2BSolomon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:peter.solomon@anu.edu.au" class="" data-icon-position="" data-hide-link-title="0">peter.solomon@anu.edu.au</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/4/779" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460130" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460130/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/779.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460130" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460130/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/779.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460130" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460130/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/779.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460130" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460130/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/4/779.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460130" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460130/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2"><em>Zymoseptoria tritici</em> is a host-specific, necrotrophic pathogen of wheat. Infection by <em>Z. tritici</em> is characterized by its extended latent period, which typically lasts 2 wks, and is followed by extensive host cell death, and rapid proliferation of fungal biomass. This work characterizes the level of genomic variation in 13 isolates, for which we have measured virulence on 11 wheat cultivars with differential resistance genes. Between the reference isolate, IPO323, and the 13 Australian isolates we identified over 800,000 single nucleotide polymorphisms, of which ∼10% had an effect on the coding regions of the genome. Furthermore, we identified over 1700 probable presence/absence polymorphisms in genes across the Australian isolates using <em>de novo</em> assembly. Finally, we developed a gene tree sorting method that quickly identifies groups of isolates within a single gene alignment whose sequence haplotypes correspond with virulence scores on a single wheat cultivar. Using this method, we have identified &lt; 100 candidate effector genes whose gene sequence correlates with virulence toward a wheat cultivar carrying a major resistance gene.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/mycosphaerella-graminicola" class="hw-term hw-article-keyword hw-article-keyword-mycosphaerella-graminicola" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Mycosphaerella graminicola</em></a></li><li class="kwd"><a href="/keyword/comparative-genomics" class="hw-term hw-article-keyword hw-article-keyword-comparative-genomics" rel="nofollow">comparative genomics</a></li><li class="kwd"><a href="/keyword/intraspecific" class="hw-term hw-article-keyword hw-article-keyword-intraspecific" rel="nofollow">intraspecific</a></li><li class="kwd"><a href="/keyword/fungal-effector" class="hw-term hw-article-keyword hw-article-keyword-fungal-effector" rel="nofollow">fungal effector</a></li><li class="kwd"><a href="/keyword/accessory-chromosome" class="hw-term hw-article-keyword hw-article-keyword-accessory-chromosome" rel="nofollow">accessory chromosome</a></li></ul><p id="p-3"><em>Zymoseptoria tritici</em> is a fungal wheat pathogen responsible for the most serious disease of wheat in the United Kingdom, and other parts of Northern Europe (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">te Beest <em>et al.</em> 2013</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Dean <em>et al.</em> 2012</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Fones and Gurr 2015</a>). The disease is characterized by a long latent period, typically lasting 8–11 d before the first appearance of visible necrotic lesions (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Sánchez-Vallet <em>et al.</em> 2015</a>). While historically considered a “hemibiotroph,” there is gathering evidence from metabolomics, microarray, and RNA sequencing studies that <em>Z. tritici</em> does not feed from living host cells, as per a biotroph, at any stage during its lifecycle (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Keon <em>et al.</em> 2007</a>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">2005</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Rudd <em>et al.</em> 2015</a>). In light of this evidence, it has been proposed that <em>Z. tritici</em> be reclassified as a “latent necrotroph” (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Sánchez-Vallet <em>et al.</em> 2015</a>). <em>Z. tritici</em> penetrates its host through open stomata and grows sparsely in the intercellular space for the duration of the latent period. This early growth is followed by rapid host cell death, and proliferation of hyphae throughout the host tissue (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Kema <em>et al.</em> 1996</a>). Asexual sporulation occurs almost exclusively in the substomatal cavities (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Cohen and Eyal 1993</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Kema <em>et al.</em> 1996</a>). The close and prolonged relationship with the plant during early infection suggests that <em>Z. tritici</em> possesses a sophisticated, and strictly regulated, set of proteins or metabolites that are responsible for restricting its growth in the host, suppressing plant recognition/immunity, and ultimately inducing host-cell death. These proteins or metabolites are commonly referred to as effectors (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">de Wit <em>et al.</em> 2009</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Stergiopoulos and de Wit 2009</a>).</p><p id="p-4">Identification of pathogen effectors (and their corresponding host targets) is essential for understanding how these parasites are able to successfully coopt plants into productive food sources. This is especially true in agricultural ecosystems, where host genetic uniformity provides ample opportunity for specialists to emerge (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Stukenbrock and Bataillon 2012</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Stukenbrock and McDonald 2008</a>). Often, it is the effector genes that are responsible for host- or cultivar-driven specificity (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">de Wit <em>et al.</em> 2009</a>; <a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Stergiopoulos and de Wit 2009</a>). Many described effectors interact with specific resistance or susceptibility genes in their hosts. These specific interactions are referred to as “gene-for-gene” relationships (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Dodds and Rathjen 2010</a>). Despite large differences in pathogen lifestyles and host range, several common characteristics are shared between these genes. Generally, proteinaceous fungal effectors are considered to be small (&lt; 300 aa), secreted, cysteine-rich (&gt; 3%, after signal peptide cleavage), and induced during <em>in planta</em> infection (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">de Wit <em>et al.</em> 2009</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Hogenhout <em>et al.</em> 2009</a>). These characteristics are broadly used to prioritize effector gene candidates for functional studies, though recent work has shown that these characteristics do not encompass all known effectors (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Sperschneider <em>et al.</em> 2015</a>).</p><p id="p-5">In filamentous fungi, effector genes are also commonly found in association with rapidly evolving segments of the genome, such as repeat-rich regions, or on accessory chromosomes (ACs). For example, <em>AvrPita</em> in <em>Magnaporthe oryzae</em>, <em>SIX</em> genes in <em>Fusarium oxysporum</em>, and the PEP cluster in <em>Nectria hematococca</em>, are all located on ACs (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Chuma <em>et al.</em> 2011</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Rodriguez-Carres <em>et al.</em> 2008</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Schmidt <em>et al.</em> 2013</a>). <em>Z. tritici</em> has several ACs that are well described, though, unlike other fungal pathogens, they have never been associated with pathogenicity (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Croll and McDonald 2011</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Croll <em>et al.</em> 2013</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Stukenbrock <em>et al.</em> 2007</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Wittenberg <em>et al.</em> 2009</a>). Other well-characterized necrotrophic effectors, such as ToxA in <em>Pyrenophora tritici-repentis</em>, and Tox3 and Tox1 in <em>Parastagonospora nodorum</em>, were successfully identified using culture filtrates that induced necrosis when infiltrated into susceptible wheat varieties (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Liu <em>et al.</em> 2009</a>, <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">2012</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Tuori <em>et al.</em> 1995</a>). This approach has recently identified two necrosis inducing proteins, ZtNIP1 and ZtNIP2, in <em>Z. tritici</em> (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Ben M’barek <em>et al.</em> 2015</a>). Heterologous expression and infiltration of these proteins into wheat also revealed cultivar specificity, though it remains unclear if this is due to a gene-for-gene, or a more general necrosis inducing response (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Ben M’barek <em>et al.</em> 2015</a>). Several in-depth RNA-sequencing (RNA-seq) studies with <em>Z. tritici</em> have identified many highly expressed “effector-like” genes or secondary metabolite clusters; however, no effector genes critical for virulence were identified in these studies (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kellner <em>et al.</em> 2014</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Rudd <em>et al.</em> 2015</a>).</p><p id="p-6">Thus far, the only gene that has been shown to be essential for virulence in <em>Z. tritici</em> is <em>Mg3LysM</em>, which was discovered based on close homology to another previously described effector gene <em>Ecp6</em> (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Marshall <em>et al.</em> 2011</a>). Three additional small secreted proteins (SSPs) that contribute quantitatively to virulence were recently described by <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Poppe <em>et al.</em> (2015)</a>. These genes were selected for functional analysis because they exhibited positive (syn. diversifying) selection (dN/dS &gt; 1), when compared to genomes of nonwheat-infecting relatives <em>Zymoseptoria pseudotritici</em> and <em>Zymoseptoria ardabillae</em> (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Poppe <em>et al.</em> 2015</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Stukenbrock <em>et al.</em> 2011</a>). Here, the authors showed that the <em>Z. tritici</em> orthologs contributed more to virulence on wheat than the corresponding ortholog from the nonwheat-infecting relatives. Despite the emerging consensus that this disease is highly quantitative in nature, at least one gene-for-gene interaction with the qualitative resistance gene <em>Stb6</em> has been demonstrated genetically, and several major resistance genes in wheat have been mapped (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Brading <em>et al.</em> 2002</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Orton <em>et al.</em> 2011</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Stewart and McDonald 2014</a>).</p><p id="p-7">With the advent of next generation sequencing, whole-genome comparisons of fungal pathogens are now routine (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Thynne <em>et al.</em> 2015</a>). Many studies focus on comparative analysis between closely related species with varying host ranges or pathogenic lifestyles (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Gardiner <em>et al.</em> 2012</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Manning <em>et al.</em> 2013</a>; <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Stukenbrock <em>et al.</em> 2011</a>). These studies provide significant insight into how pathogenic fungi are able to occupy similar niches, <em>i.e.</em>, host specialization (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Gardiner <em>et al.</em> 2012</a>). Yet, so far, intraspecific resequencing of multiple isolates has been limited to a small number of key pathogen species (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Chen <em>et al.</em> 2013</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Persoons <em>et al.</em> 2014</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Xue <em>et al.</em> 2012</a>). Comparative genome studies with <em>Z. tritici</em> are aided by the completeness of the reference genome, Dutch isolate IPO323 (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Goodwin <em>et al.</em> 2011</a>). <a id="xref-ref-66-3" class="xref-bibr" href="#ref-66">Stukenbrock <em>et al.</em> (2011)</a> sequenced two additional <em>Z. tritici</em> genomes from Iran, and several genomes of closely related relatives to examine interspecific changes that may lead to specialization on wheat. This analysis showed accelerated evolution specifically in the <em>Z. tritici</em> species lineage when compared to closely related relatives. <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Torriani <em>et al.</em> (2011)</a> added an additional seven <em>Z. tritici</em> genomes from Switzerland to examine intron presence/absence (P/A) polymorphisms, both within and between species. <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Croll <em>et al.</em> (2013)</a> used these same seven Swiss isolates, plus an additional four newly sequenced genomes, to describe the process of extensive genome rearrangement in ACs. Each of these studies reinforces the perception of <em>Z. tritici</em> as a highly dynamic, diverse pathogen, with the potential to rapidly evolve at the gene or genomic level within a single sexual generation (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Croll <em>et al.</em> 2013</a>).</p><p id="p-8">In Australia, the breaking of the Millennium drought (2001–2009) has driven the reemergence of <em>Z. tritici</em> as a damaging pathogen of wheat in the high-rainfall areas of Victoria and South Australia (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Hollaway 2014</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Hollaway and McLean 2014</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Milgate <em>et al.</em> 2015</a>, <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">2014</a>). Little is known about the current state of the Australian <em>Z. tritici</em> populations, particularly in terms of which wheat resistance genes remain effective at controlling the disease. In this study we characterized the virulence of 13 <em>Z. tritici</em> isolates, collected over 30 yr from all major wheat-producing regions of Australia. Virulence scores were measured on a differential set of 11 wheat cultivars, many of which contain known resistance genes. Subsequently, we sequenced the genomes of each <em>Z. tritici</em> strain in order to assess whether genetic variation, particularly within coding regions, could be associated with observed virulence profiles.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Growth of fungal strains and pathogenicity assays</h3><p id="p-9">The protocol for media and inoculum preparation was followed as described previously (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ballantyne and Thomson 1995</a>). Plants were grown, inoculated and assessed as previously described in <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Zwart <em>et al.</em> (2010)</a> at Wagga Wagga, NSW, with the following modifications. Four seedlings of each differential genotype were grown in a pot, and treated as an experimental unit with two replications per experiment. The full list of fungal isolates and cultivars used is provided in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467856/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467856/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Pathogenicity scores on 11 differential wheat cultivars</span><div class="sb-div caption-clear"></div></div></div><p id="p-11">After all disease scores were assigned, the average score was used to group the isolates based on their virulence profile in two independent ways. First, by calculating the pairwise Euclidean distance between each isolate’s virulence profile using the dist() function in R. This distance matrix was then used to cluster isolates using hclust() with Ward’s method, which clusters by minimizing the sums of squares between isolates. The resulting dendrogram was plotted in R using the plot() function. The same data were used to cluster the isolates using a K-means method, also implemented through R. Unlike the clustering method described above, where a dendrogram is generated to visualize any potential clusters, K-means requires selection of the ideal number of clusters <em>a priori</em>. K-means centers were plotted with three and four clusters on top of the raw virulence scores for each isolate on the cultivars Hereward (<em>x</em>-axis) and Summit (<em>y</em>-axis). From this plot, the fit of k-means centers to the data was visually assessed, and the grouping compared to that derived from the distance-based method described above.</p></div><div id="sec-3" class="subsection"><h3>DNA extraction and sequencing</h3><p id="p-12">Fungal yeast-like spores were grown, as described above, from –80 <strong>°</strong>C glycerol stocks. For DNA extraction, harvested spores were concentrated into pellets via centrifugation. These pellets were cooled to –80° and lyophilized for 24 hr. DNA was extracted using the QIAGEN Plant mini-prep kit (QIAGEN, VIC, Australia) following the standard protocol. Extracted DNA was submitted to the Australian Cancer Research Foundation (ACRF) Biomolecular Resource Facility (JCSMR, Australian National University). Here, libraries were constructed using the Illumina TruSeq DNA LT Sample Prep kit v2 (part # FC-121-2001). Samples WAI320–WAI329 were multiplexed and sequenced together at 100 bp paired end using a v3 Illumina TruSeq SBS Kit (HS 200 cycles, part # FC-401-3001) in a single lane on a HiSeq2000 at the ACRF. Samples WAI55, WAI56, WAI147, and WAI332 were sequenced earlier at the Australian Genome Research Facility (Melbourne, VIC). Raw reads were trimmed of any remaining adapters using Trimmomatic v0.27 (specific Trimmomatic options: ILLUMINACLIP: TruSeq2_PE.fa:2:30:10 LEADING:20 TRAILING:20 SLIDINGWINDOW:4:24 MINLEN:90) (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bolger <em>et al.</em> 2014</a>). All sequenced samples were checked for quality using FastQC v0.10.1 (<a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>). Paired reads are deposited in the NCBI short-read archive: Bioproject ID PRJNA299857 and Biosample IDs SAMN04216882-SAMN04216894. Isolates were assembled <em>de novo</em> using the SPAdes Genome Assembler v3.5 as described previously (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bankevich <em>et al.</em> 2012</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">McDonald <em>et al.</em> 2015</a>).</p></div><div id="sec-4" class="subsection"><h3>Mappability of the reference genome IPO323</h3><p id="p-13">Mappability is defined as the inherent ability of regions within a genome to be correctly mapped using short-sequencing read technology. Many areas of the genome, including coding regions, may not be reliably mapped with short length reads. This will influence both the reliability and accuracy of single nucleotide polymorphisms (SNPs) called in low mappability regions. We calculated mappability on the reference genome isolate IPO323 using a modified version of the program gmav0.1.3 (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Lee and Schatz 2012</a>) (<a href="https://github.com/cbergman/gma-0.1.3">https://github.com/cbergman/gma-0.1.3</a>), and excluded regions of low mappability from variant call format (VCF) files after SNP calling (defined here as a mappability score &lt; 50). The mappability score is influenced by the sequencing technology; therefore, for this project, we used the settings (tech–illumina -b 50).</p></div><div id="sec-5" class="subsection"><h3>Sequence variation in resequenced genomes</h3><div id="sec-6" class="subsection"><h4>SNP calling with the genome analysis toolkit:</h4><p id="p-14">Trimmed paired and unpaired reads were mapped to the reference genome IPO323 v2.0 (<a href="http://genome.jgi.doe.gov/Mycgr3/Mycgr3.home.html">http://genome.jgi.doe.gov/Mycgr3/Mycgr3.home.html</a>) using BWAv0.7.3a-r367. Duplicate reads within the bam files were removed with the Picard v1.87 tool Mark Duplicates, and then labeled with the date of the sequencing run with the Genome Analysis Toolkit (GATK) tool AddOrReplaceReadGroups. The bam files were then used as input for the GATK variant call pipeline using the following tools, respectively, RealignerTargetCreator, IndelRealigner and HaplotypeCaller. Final SNP calls were filtered using VariantFiltration with the following options: -R /LargeDataSet/Megan/GATK/Mycosphaerella_graminicola.allmasked.fa-o Ztritici.filteredSNPS.GATK3-30–filterExpression “QD, 2.0 || FS &gt; 50.0 || MQ, 25.0”–filterName “haploidfilter” -mask mappability_51_0.5andless_3col.bed -maskName “lowmappability”. These variants were then removed using an inverse grep bash command (grep –v “low mappability”). The original prefiltered VCF file is available as Supporting Information <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025197/-/DC1/FileS1.zip">File S1</a>. The bed file used to mask the VCF file is provided as <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025197/-/DC1/FileS2.zip">File S2</a>. The filtered VCF file was then further annotated with SNPeffv3.2a (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Cingolani <em>et al.</em> 2012</a>). To limit the assignment of multiple effects to SNPs with close neighboring genes, we reduced the up and downstream region of the genes to 500 bp using the –ud option (default 5 kb). The genome annotation used for SNP annotation is the RNA-seq based annotation released by Rothamsted Research on Ensembl Fungi (<a href="http://fungi.ensembl.org/Zymoseptoria_tritici/Info/Index">http://fungi.ensembl.org/Zymoseptoria_tritici/Info/Index</a>, gff files available on request from Dr. Jason Rudd at Rothamsted Research, UK). Blast2GO v3.1.2 was used to assign gene ontolotgy (GO) terms and ProtIDs to this annotation. Signal peptide annotations were predicted SignalP v4.1 (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Emanuelsson <em>et al.</em> 2007</a>).</p></div><div id="sec-7" class="subsection"><h4>Gene presence/absence:</h4><p id="p-15">BLASTN searches using each <em>de novo</em> assembly of the Australian isolates as the BLAST database, and IPO323 genomic DNA coding regions as the BLAST query were conducted as described by <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">McDonald <em>et al.</em> (2015)</a>. Briefly, results were limited to the top BLAST hit (maximum bit score), with an e-value cut-off of 0.0001. The regions for each BLAST hit were extracted from each of the <em>de novo</em> assemblies, and written to a combined FASTA file. These FASTA files were aligned using MUSCLEv3.8.31 (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Edgar 2004</a>). Gene presence was called if the top BLAST hit length was ≥ 75% of the annotated gene in IPO323.</p></div></div><div id="sec-8" class="subsection"><h3>Phylogenetic analyses</h3><p id="p-16">A phylogenetic tree was constructed using the Bayesian Markov Chain Monte Carlo (MCMC) model implemented in BEASTv2.3.0 (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bouckaert <em>et al.</em> 2014</a>). A total of 905 gene alignments that contained all 13 resequenced isolates were selected at random, and concatenated as input for Beast (Total alignment length 1,534,110 bp). Beauti v2.30 was used to format the XML file that specifies the model parameters to be run in BEAST (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Bouckaert <em>et al.</em> 2014</a>). Runs were conducted assuming the General Time Reversible nucleotide substitution model with no rate variation categories, a constant Population size, and a strict molecular clock. After a burn-in of 100,000 steps, 10,000 posterior trees were sampled. To assess run convergence, the BEAST log file was visualized in Tracer v1.6.0. The estimated sample size for each parameter was well over 100, which indicates that sampling of the posterior space during the run was sufficient. The sampled 10,000 posterior trees were visualized using Densi Tree v2.2.2 (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bouckaert and Heled 2014</a>). This procedure was repeated independently two times with a smaller set of 500 randomly selected genes, to ensure that gene selection did not affect the final tree topology (data not shown). To verify that concatenation did not adversely affect our tree construction, we used the whole genome phylogenetic software andi (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Haubold <em>et al.</em> 2015</a>). The entire <em>de novo</em> assembly FASTA files for each isolate were used as input for this analysis. The resulting distance matrix was plotted as a tree using R. Haplotype networks were drawn with TCS v1.21 implemented in the software popart (<a href="http://popart.otago.ac.nz">http://popart.otago.ac.nz</a>) (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Clement <em>et al.</em> 2002</a>).</p></div><div id="sec-9" class="subsection"><h3>Matching phenotype to individual gene trees</h3><p id="p-17">In order to quickly sort through gene trees, and search for specific grouping of isolates based on phenotype, we created PhyBi, written in python (<a href="https://github.com/SolomonLab/PhyBi">https://github.com/SolomonLab/PhyBi</a>). PhyBi creates a phylogenetic “binary tree” for each gene by grouping isolates based on their sequence similarity. First, a matrix is created by parsing a Newick formatted tree to calculate pairwise distances (created by summing the branch lengths) between any two isolate pairs (referred to as the “distance matrix”). By definition, the distance matrix is symmetric, and all diagonal entries are zero (distance between an isolate and itself). If each of these distances is below a given threshold (default 0.01 substitutions per site), all isolates are grouped together and a “star” result is recorded. This will collapse genes whose sequence is near identical (&lt; 1 bp substitution per 100 bp) into a single group and not consider them further. In cases where isolates differ by more than the first threshold, a second threshold is used to move the isolates into groups. If any two isolates differ by less than the second threshold (default 0.02 substitutions per site), they are designated to the same group. This reduces the number of groups, and therefore simplifies the iterative grouping performed in the following stage. This initial grouping is followed by an iterative step that progressively combines nearest groups until only two groups remain. The iterative step involves two stages: first, the smallest nonzero entry of the distance matrix is identified. Second, the two groups corresponding to this entry are combined. The distances in the columns, and rows of the first group, are averaged with those of the second group (weighted according to number of isolates in each group). The second group’s rows and columns are then deleted from the distance matrix. The program repeats the iterative step until there are only two groups remaining, creating a phylogenetic “bi-tree” (a tree with one long central branch, and, at either end, a polytomy). A schematic overview of the process, full description of input/output files, and additional features not used in this manuscript are given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025197/-/DC1/FileS3.doc">File S3</a>.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-18">Reference Data Accession Numbers: Bioproject ID PRJNA299857; Biosample IDs SAMN04216882-SAMN04216894</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Single nucleotide variation between the 13 Australian strains</h3><p id="p-19">Trimmed reads from each resequenced genome were aligned to the reference genome isolate IPO323 v.2.0 using GATK as described in the section <em>Materials and Methods</em>. Before quality filtering, over 1 million SNPs were identified between the 13 strains and IPO323. Low quality SNPs (“QD &lt; 2.0 || FS &gt; 50.0 || MQ &lt; 25.0”), and SNPs occurring in “low mappable” (&lt; 0.5 mappability score) regions of the genome were flagged and excluded from further analyses. Many of these SNPs were associated with small structural changes in the genomes, likely larger insertions or deletion events that differ from IPO323, which are often surrounded by a high density of SNP calls. This is also similar for areas near annotated repeat regions, where incorrect mapping of reads to repetitive DNA leads to a high number of SNP calls, and an increase in read mapping coverage. Two examples of SNPs near small indels are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025197/-/DC1/FigureS1.pdf">Figure S1</a>. Our sequencing coverage ranged from 15x to 30x on average across the genome for most isolates (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025197/-/DC1/FigureS2.pdf">Figure S2</a>). Due to this limited coverage, and our short size selection (500 bp paired-ends), we did not investigate these potential large rearrangements further.</p><p id="p-20">After filtering, the remaining 858,965 SNPs were fed into SnpEff to assess their effect (whether or not they occurred in coding regions) on the genome. SnpEff automatically assigns each SNP into one of four major “impact” categories depending upon the effect that the SNP has on annotated regions of the genome. These categories are: 1) Modifier, SNPs with little to no effect on coding regions (<em>i.e.</em>, intergenic). 2) Low, SNPs that occur within coding regions but do not affect the encoded amino acid (<em>i.e.</em>, synonymous mutations, introns). 3) Moderate, SNPs that occur within coding regions, and slightly alter the encoded amino acid (<em>i.e.</em>, nonsynonymous mutations. 4) High, SNPs that dramatically alter the encoded amino acid (<em>i.e.</em>, missense, frame-shift mutations). As expected, most SNPs fell into the Modifier (71.7%) category, followed by Low effect SNPs (17.9%), Moderate (9.0%), and finally High (1.4%). By far the most common SNP location was in intergenic regions, representing 40.3% the total SNPs identified. This was followed by exons (27.4%), downstream (12.2%), upstream (11.9%), introns (7.2%), and 1% in other minor categories. A summary of all single effect SNPs, separated by impact category, for all 13 resequenced isolates is summarized in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a> (12,000 pleiotropic SNPS are excluded from this summary, a complete VCF file is available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025197/-/DC1/FileS2.zip">File S2</a>). Within coding regions, the most common variant effect was synonymous change (which includes SNPs within introns). This type of SNP was several orders of magnitude higher than High effect SNPs (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/779/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary of all single effect SNPs identified in the 13 sequenced genomes using IPO323 as a reference sequence. (A) The number of identified SNPs, colored by their effect on the genome and separated into &#x201C;Low&#x201D;, &#x201C;Moderate,&#x201D; and &#x201C;High&#x201D; effect categories, dependent upon their effect on coding regions. (B) The number of low effect SNPs found within nonsecreted and secreted genes as a percentage of all identified SNPs. The light gray bars show that, on average, Low effect SNPs are found at a higher proportion in nonsecreted genes vs. secreted. Black error bars show 95% confidence intervals (CI), from all genes. (C) The number of moderate effect SNPs found within nonsecreted and secreted genes as a percentage of all identified SNPs within a gene. Higher dark gray bars show that, on average, Moderate effect SNPs represent a higher proportion of identified SNPs in secreted genes. (D) There is no observed trend for High effect SNPs in secreted vs. nonsecreted genes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-539057658" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Summary of all single effect SNPs identified in the 13 sequenced genomes using IPO323 as a reference sequence. (A) The number of identified SNPs, colored by their effect on the genome and separated into &#x201C;Low&#x201D;, &#x201C;Moderate,&#x201D; and &#x201C;High&#x201D; effect categories, dependent upon their effect on coding regions. (B) The number of low effect SNPs found within nonsecreted and secreted genes as a percentage of all identified SNPs. The light gray bars show that, on average, Low effect SNPs are found at a higher proportion in nonsecreted genes &lt;em&gt;vs.&lt;/em&gt; secreted. Black error bars show 95% confidence intervals (CI), from all genes. (C) The number of moderate effect SNPs found within nonsecreted and secreted genes as a percentage of all identified SNPs within a gene. Higher dark gray bars show that, on average, Moderate effect SNPs represent a higher proportion of identified SNPs in secreted genes. (D) There is no observed trend for High effect SNPs in secreted &lt;em&gt;vs.&lt;/em&gt; nonsecreted genes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/779/F1.medium.gif" width="440" height="346"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/4/779/F1.medium.gif" width="440" height="346"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/779/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/779/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467887" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-21" class="first-child">Summary of all single effect SNPs identified in the 13 sequenced genomes using IPO323 as a reference sequence. (A) The number of identified SNPs, colored by their effect on the genome and separated into “Low”, “Moderate,” and “High” effect categories, dependent upon their effect on coding regions. (B) The number of low effect SNPs found within nonsecreted and secreted genes as a percentage of all identified SNPs. The light gray bars show that, on average, Low effect SNPs are found at a higher proportion in nonsecreted genes <em>vs.</em> secreted. Black error bars show 95% confidence intervals (CI), from all genes. (C) The number of moderate effect SNPs found within nonsecreted and secreted genes as a percentage of all identified SNPs within a gene. Higher dark gray bars show that, on average, Moderate effect SNPs represent a higher proportion of identified SNPs in secreted genes. (D) There is no observed trend for High effect SNPs in secreted <em>vs.</em> nonsecreted genes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">We next considered the distribution of SNPs with different effects between genes with a predicted secretion signal. SignalP v4.1 identified 981 genes with a predicted signal peptide. <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1, B–D</a> shows a comparison of the percentage of SNPs within secreted and nonsecreted genes. These results show that a higher percentage of SNPs within nonsecreted genes are classified as Low effect SNPs (<em>i.e.</em>, synonymous mutations) (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>). While this trend is consistent for all resequenced isolates, it was significant [nonoverlapping 95% confidence intervals (95% CIs)] for only eight of the 13 isolates. The reverse trend was seen for the average number of Moderate effect SNPs, with every isolate showing a slight increase in Moderate mutations in genes with a predicted secretion signal (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1C</a>). Again, eight of the 13 isolates had nonoverlapping 95% CI, showing a slight elevation of moderate SNPs in secreted genes. There were no significant differences between any of the isolates between secreted and nonsecreted genes for the High effect SNPs (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1D</a>).</p></div><div id="sec-13" class="subsection"><h3>Presence/absence of coding regions on core and accessory chromosomes</h3><p id="p-23">To assess the presence/absence (P/A) of coding genes, each genome was assembled <em>de novo</em> to exclude any chance that a gene is called missing due to small rearrangements from the reference. A summary of the final <em>de novo</em> assembly statistics for each isolate, including total number of contigs, max contig length, N50 value, and total number of bases assembled (excluding contigs &lt; 500 bp), is provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025197/-/DC1/TableS1.xlsx">Table S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025197/-/DC1/TableS2.xlsx">Table S2</a>. These statistics are similar to other resequencing projects conducted with <em>Z. tritici</em> (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Croll <em>et al.</em> 2013</a>).</p><p id="p-24">BLAST searches with the complete coding sequence of each annotated gene were conducted, whereby a gene was called as present if 75% of the gene length was in the top BLAST hit. The complete dataset, which includes a summary by chromosome, is provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025197/-/DC1/FileS4.xlsx">File S4</a>. <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows a distinct difference in the pattern of gene P/A between the 13 core chromosomes (CCs) and eight ACs. The isolates, in columns, are ordered from left to right by year of sampling. Some ACs were completely absent in the earlier samples (<em>e.g.</em>, Chromosome 14 in isolates WAI332, WAI320, WAI324, WAI329, and WAI329), and appear only in the later columns. The ACs are characterized by large segments of sequential gene presence/absence, whereas on the CCs there were very few instances of large sequential gene absence. Despite many ACs being absent, several genes on these chromosomes were still found in the <em>de novo</em> assemblies. Every AC, except for Chromosome 19, was almost completely absent in at least one isolate. While Chromosome 19 was not completely lost in any single isolate, ∼50% of the coding genes were lost in isolate WAI329.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/779/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary of the presence or absence of IPO323 genes across all chromosomes. Gene presence (dark colors) or absence (translucent) for each re-sequenced isolate is shown for all annotated genes from the reference isolate IPO323. Each row in the figure represents one annotated gene, while each column represents one isolate. There are 13 columns in each panel; the number at the top (in the gray bar) indicates the chromosome. Isolates are ordered from left to right, from earliest to latest year of sampling. Isolates in the same color were sampled during the same year. (A) The core chromosomes of Z. tritici possess most of their annotated genes with very few large deletions. (B) The accessory chromosomes possess several large segmental gene deletions, many of which indicate that entire chromosomes are missing or significantly altered in structure from the reference genome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-539057658" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Summary of the presence or absence of IPO323 genes across all chromosomes. Gene presence (dark colors) or absence (translucent) for each re-sequenced isolate is shown for all annotated genes from the reference isolate IPO323. Each row in the figure represents one annotated gene, while each column represents one isolate. There are 13 columns in each panel; the number at the top (in the gray bar) indicates the chromosome. Isolates are ordered from left to right, from earliest to latest year of sampling. Isolates in the same color were sampled during the same year. (A) The core chromosomes of &lt;em&gt;Z. tritici&lt;/em&gt; possess most of their annotated genes with very few large deletions. (B) The accessory chromosomes possess several large segmental gene deletions, many of which indicate that entire chromosomes are missing or significantly altered in structure from the reference genome.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/779/F2.medium.gif" width="440" height="304"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/4/779/F2.medium.gif" width="440" height="304"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/779/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/779/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467912" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-25" class="first-child">Summary of the presence or absence of IPO323 genes across all chromosomes. Gene presence (dark colors) or absence (translucent) for each re-sequenced isolate is shown for all annotated genes from the reference isolate IPO323. Each row in the figure represents one annotated gene, while each column represents one isolate. There are 13 columns in each panel; the number at the top (in the gray bar) indicates the chromosome. Isolates are ordered from left to right, from earliest to latest year of sampling. Isolates in the same color were sampled during the same year. (A) The core chromosomes of <em>Z. tritici</em> possess most of their annotated genes with very few large deletions. (B) The accessory chromosomes possess several large segmental gene deletions, many of which indicate that entire chromosomes are missing or significantly altered in structure from the reference genome.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">The observation of higher rates of moderate SNPs within secreted genes led us to question whether secreted genes were more likely to be found with P/A polymorphisms. These results are summarized in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>; genes that are absent in all Australian isolates are not polymorphic and were not included in this count. Due to the strong difference in gene P/A between core and accessory chromosomes, we summarized the proportion of all or secreted genes for CC and AC chromosomes separately (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Overall, there was little evidence of elevated rates of gene P/A polymorphisms in secreted <em>vs.</em> nonsecreted genes, with the exception of Chromosomes 2 and 10. Large sectional gene absence was far more rare on CCs in comparison to ACs (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467877/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467877/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary of putative presence/absence (P/A) polymorphisms in secreted and nonsecreted genes by chromosome</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Linking genes to virulence</h3><p id="p-28">A Bayesian phylogenetic tree was constructed using 905 aligned genes concatenated into a ∼1.5 Mbp alignment. This analysis was performed three times independently, selecting the genes at random to ensure that gene selection did not dramatically alter the consensus topology from the posterior sample (data not shown). Additionally, the whole genome phylogenetic program andiv0.1 was used to estimate the pairwise distance between isolates based on minimum exact matches throughout the whole genome (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Haubold <em>et al.</em> 2015</a>). The topology of the consensus tree from the posterior sample, and the tree constructed with pairwise distance estimates from andi, were identical, indicating that concatenation did not bias our Bayesian tree topology. The DensiTree rendering of the posterior sample from Beast is shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>, and the andi dendrogram is provided as <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025197/-/DC1/FigureS3.pdf">Figure S3</a>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/779/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Isolate grouping based on phylogenetic or phenotypic signal. (A) The 10,000 posterior sampled trees estimated with BEAST. Dark blue bars and a lack of crossing over events between branches provides strong support for the topology shown (99.87% of posterior sample). The European reference isolate forms a distinct out-group from the Australian isolates. Two major Australian clades are seen; one with isolates WAI321&#x2013;WAI323, and isolate WAI327. (B) A cladogram representing the pairwise distances of the Australian isolates based on their qualitative virulence scores. Three distinct groups were identified, which largely match the neutral phylogeny. Isolate WAI327, however, is separated from its closest relatives, clustering more closely with the less virulent isolates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-539057658" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Isolate grouping based on phylogenetic or phenotypic signal. (A) The 10,000 posterior sampled trees estimated with BEAST. Dark blue bars and a lack of crossing over events between branches provides strong support for the topology shown (99.87% of posterior sample). The European reference isolate forms a distinct out-group from the Australian isolates. Two major Australian clades are seen; one with isolates WAI321&#x2013;WAI323, and isolate WAI327. (B) A cladogram representing the pairwise distances of the Australian isolates based on their qualitative virulence scores. Three distinct groups were identified, which largely match the neutral phylogeny. Isolate WAI327, however, is separated from its closest relatives, clustering more closely with the less virulent isolates.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/779/F3.medium.gif" width="440" height="345"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/4/779/F3.medium.gif" width="440" height="345"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/779/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/779/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467917" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-29" class="first-child">Isolate grouping based on phylogenetic or phenotypic signal. (A) The 10,000 posterior sampled trees estimated with BEAST. Dark blue bars and a lack of crossing over events between branches provides strong support for the topology shown (99.87% of posterior sample). The European reference isolate forms a distinct out-group from the Australian isolates. Two major Australian clades are seen; one with isolates WAI321–WAI323, and isolate WAI327. (B) A cladogram representing the pairwise distances of the Australian isolates based on their qualitative virulence scores. Three distinct groups were identified, which largely match the neutral phylogeny. Isolate WAI327, however, is separated from its closest relatives, clustering more closely with the less virulent isolates.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">To assess whether the phylogenetic relationship between isolates matched the observed virulence profiles, we performed Ward’s pairwise distance and k-means clustering. The results of k-means clustering, including the curve used to select the ideal number of groups, are presented in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025197/-/DC1/FigureS4.pdf">Figure S4</a>. The ideal number of clusters should be at the steepest point of the curve, which corresponded to either three or four clusters in our data. Both clustering methods supported the separation of the isolates into three distinct virulence groups. The dendrogram generated with Wards distance clustering is presented in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>. These three virulence groups can be classified based on their virulence profile into high, medium and low virulence, with the high virulent isolates causing more severe symptoms on a wider range of wheat genotypes (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). The neutral phylogeny revealed two highly supported groups of isolates, with WAI321–WAI323, and WAI327 grouped together, distinct from the other sister clade with the remaining nine Australian isolates. The European isolate, IPO323, formed an outgroup from the Australian isolates (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3A</a>). This largely matches the clusters identified by virulence grouping, with one notable exception; isolate WAI327 grouped more closely with the “less-virulent” cluster away from isolates WAI321–WAI323.</p><p id="p-31">The separation of isolate WAI327 from its closest relatives, WAI321–WAI323, suggested that the genes encoding cultivar specificity may be different between these isolates. To pursue this hypothesis, we developed PhyBi (described in <em>Materials and Methods</em> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025197/-/DC1/FileS3.doc">File S3</a>) to identify individual gene trees that group WAI321–WAI323 together without isolate WAI327. PhyBi identified 71 genes out of 10,117 gene alignments that matched this grouping. Of these 71 genes, only six have a signal peptide for secretion. The six secreted genes, along with their location in the genome annotation, and other annotated traits, are listed in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. While the functionality of PhyBi was extended to identify genes that are present or absent from a given gene tree, these results were not summarized, as the analysis showed several hundreds of gene presences/absences throughout the 13 genomes.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467951/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467951/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">The six secreted genes identified using PhyBi</span><div class="sb-div caption-clear"></div></div></div><p id="p-33">The six effector candidates were inspected manually to ensure that intragenic recombination did not interfere with the PhyBi grouping. Five of the six genes matched a reference annotation, while one gene was unique to the Rothamsted annotation (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). Expression of genes whose annotation matched the original genome annotation were compared to two published RNA-seq studies to check for evidence of expression <em>in planta</em> (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>) (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Kellner <em>et al.</em> 2014</a>; <a id="xref-ref-57-4" class="xref-bibr" href="#ref-57">Rudd <em>et al.</em> 2015</a>). None of the six genes were induced during <em>in planta</em> infection in either study. Many of these genes matched several characteristics used to classify effector genes (&lt; 300 aa; &gt; 3% cysteine) (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). Three of the six genes encode proteins that are &lt; 250 aa before cleavage of the signal peptide. ZtRRes_01714 is extremely small, and is predicted to be only 39 aa after cleavage of the signal peptide. Three of the identified proteins, however, are large (&gt;400 aa), and would be excluded from traditional effector cutoff values based on their size. One of these large proteins contains a CFEM domain, which has been associated with virulence in <em>Magnaporthe oryzae</em> (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Dean <em>et al.</em> 2005</a>). Haplotype networks were generated for each of the six genes to better visualize the extreme variation in gene sequence between the 13 isolates and IPO323 (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). All but one of the identified genes has at least one nonsynonymous substitution that separates isolates WAI321–WAI323 from the other haplotypes. Haplotype networks are colored based on each isolates grouping into low, medium, or high virulence phenotype clusters (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3B</a> and <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/779/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Haplotype networks for six secreted genes identified with PhyBi. Maximum parsimony networks generated with TCS v1.21 and drawn with popart (http://popart.otago.ac.nz). Networks are not drawn to scale and indels were treated as missing data. Single point mutations are shown in parenthesis with the number of nonsynonymous mutations shown with an asterisk below. Haplotype circles are proportional to the number of isolates identified and colored based on their virulence grouping in Figure 3. Black dots are probable unsampled haplotypes as identified by TCS. (A) ZtRRes_00659, (B) ZtRRes_01714, (C) ZtRRes_05182, (D) ZtRRes_06114, (D) ZtRRes_08426, (F) ZtRRes_08778." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-539057658" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Haplotype networks for six secreted genes identified with PhyBi. Maximum parsimony networks generated with TCS v1.21 and drawn with popart (http://popart.otago.ac.nz). Networks are not drawn to scale and indels were treated as missing data. Single point mutations are shown in parenthesis with the number of nonsynonymous mutations shown with an asterisk below. Haplotype circles are proportional to the number of isolates identified and colored based on their virulence grouping in Figure 3. Black dots are probable unsampled haplotypes as identified by TCS. (A) ZtRRes_00659, (B) ZtRRes_01714, (C) ZtRRes_05182, (D) ZtRRes_06114, (D) ZtRRes_08426, (F) ZtRRes_08778.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/779/F4.medium.gif" width="440" height="357"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/4/779/F4.medium.gif" width="440" height="357"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/779/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/779/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467909" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-34" class="first-child">Haplotype networks for six secreted genes identified with PhyBi. Maximum parsimony networks generated with TCS v1.21 and drawn with popart (<a href="http://popart.otago.ac.nz">http://popart.otago.ac.nz</a>). Networks are not drawn to scale and indels were treated as missing data. Single point mutations are shown in parenthesis with the number of nonsynonymous mutations shown with an asterisk below. Haplotype circles are proportional to the number of isolates identified and colored based on their virulence grouping in <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>. Black dots are probable unsampled haplotypes as identified by TCS. (A) ZtRRes_00659, (B) ZtRRes_01714, (C) ZtRRes_05182, (D) ZtRRes_06114, (D) ZtRRes_08426, (F) ZtRRes_08778.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-35">While previous population genetic studies show that the average genetic diversity of <em>Z. tritici</em> in Australia is significantly lower than other populations from around the world, this work demonstrates that, across a wider space and time, Australian <em>Z. tritici</em> isolates are extremely diverse both genetically and phenotypically (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Banke and McDonald 2005</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Banke <em>et al.</em> 2004</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Jürgens <em>et al.</em> 2006</a>). This presents a strong challenge to Australian farmers and breeders looking to avoid the impact of this disease, as it has reemerged subsequent to the Millennium Drought. Overall, we identified over 1700 putative gene absences between the reference strain IPO323 and the 13 Australian genomes, and over 800,000 SNPs. For comparison, a resequencing study using 15 isolates of the poplar rust pathogen <em>Melampsora larici-populina</em>, with a genome size approximately two times that of <em>Z. tritici</em>, identified 611,824 single nucleotide variants (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Persoons <em>et al.</em> 2014</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Duplessis <em>et al.</em> 2011</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Goodwin <em>et al.</em> 2011</a>). These results highlight the extreme diversity that already exists in <em>Z. tritici</em>, as well as the challenge of effector gene identification in such a genetic background. In order to further simplify this search, we utilized variation in virulence phenotypes on a range of differential wheat cultivars to target secreted genes, as discussed below.</p><div id="sec-16" class="subsection"><h3>Extreme sequence diversity in a limited set of isolate pathotypes</h3><p id="p-36">When examining the distribution of Low, Moderate, and High effect SNPs across secreted genes, we noted a significant increase (nonoverlapping 95% confidence intervals) in the number of moderate substitutions (<em>i.e.</em>, nonsynonymous substitutions) found in secreted <em>vs.</em> nonsecreted genes. While this trend was apparent in all 13 resequenced genomes, it was significant for only eight of the 13 isolates (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>). This data suggests that nonsynonymous substitution is an important means of introducing variation in proteins that are predicted to act outside of the fungal cell. This trend is supported by results from <a id="xref-ref-66-4" class="xref-bibr" href="#ref-66">Stukenbrock <em>et al.</em> (2011)</a>, who also found evidence of higher rates of adaptive evolution in genes with a predicted signal peptide in an interspecific comparison of <em>Z. tritici</em>, and its close relatives <em>Z. pseudotritici</em> and <em>Z. ardabiliae</em>. Recently, some of the genes identified by <a id="xref-ref-66-5" class="xref-bibr" href="#ref-66">Stukenbrock <em>et al.</em> (2011)</a> were shown to be important for quantitative virulence in seedling infections and host specificity (<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Poppe <em>et al.</em> 2015</a>). There was no difference in the proportion of High effect SNPs found in secreted <em>vs.</em> nonsecreted genes. This is an interesting observation, as it implies that, at the genomic scale, introduction of nonsense mutations or frame-shifts is not favored as a means to introduce variation in secreted proteins.</p><p id="p-37">The observed elevation in SNP calls near putative small genome rearrangements within or near annotated genes, suggested that calling the presence or absence genes would be more accurate in <em>de novo</em> assemblies (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025197/-/DC1/FigureS1.pdf">Figure S1</a>). Our previous analysis showed that average read mapping to some of ACs was very low when compared to CCs, and, correspondingly, most of the annotated genes on AC with very low mapping coverage were not found in <em>de novo</em> assemblies (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>; <a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">McDonald <em>et al.</em> 2015</a>). Absence of a chromosome, however, does not also imply loss of the gene annotated on that chromosome. For example, up to 40 out of 110 annotated genes on AC18 are present in <em>de novo</em> assemblies, though this chromosome is likely absent, or in a highly divergent form, in these resequenced genomes compared to IPO323. The inheritance of ACs remains a very interesting and perplexing biological phenomenon of <em>Z. tritici</em> and its close relatives (<a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">Croll <em>et al.</em> 2013</a>; <a id="xref-ref-66-6" class="xref-bibr" href="#ref-66">Stukenbrock <em>et al.</em> 2011</a>).</p><p id="p-38">The virulence data indicates that there are distinct differences in an individual isolate’s ability to infect a particular cultivar. This implies that, within the species, there is further cultivar-specific specialization. Many known effectors with cultivar-specific virulence, such as <em>Leptosphaeria maculans AvrLm1</em>, utilize gene deletion as a means of evading host immunity (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">McDonald <em>et al.</em> 2013</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Van de Wouw <em>et al.</em> 2010</a>). In the case of <em>AvrLm1</em>, a simple deletion of the gene renders a plant carrying the resistance gene susceptible to attack. In this study, we postulate that over 1700 genes exist in a presence/absence polymorphism in Australian <em>Z. tritici</em> isolates. While this number is likely an overestimation due to misassembly near repetitive regions or telomeres, 1301 out of 1776 putative P/A polymorphisms were found in more than one isolate. Due to the sheer number of potential P/A polymorphisms, it was not possible to link a single gene absence to virulence on a particular cultivar. Larger sequential deletions (not necessarily polymorphic ones) are rare on CCs, but very common on ACs (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). Importantly, some of the larger segmental deletions, such as the cluster on Chromosome 4 and Chromosome 11, are polymorphic, meaning that sexual recombination can bring this gene deletion in or out of different genetic backgrounds. The isolates selected for this study are too spatially and chronologically diverse to examine the frequency of these deletions in more detail; however, it is tempting to speculate how these larger sequential deletions may change in allelic frequency over time within a field population. Again, these results highlight the endemic genetic diversity of this fungus, and highlight its strong adaptive potential.</p></div><div id="sec-17" class="subsection"><h3>Gene tree-phenotype matching as a novel effector prediction method</h3><p id="p-39">Previous attempts to identify effector candidates based on the known properties of effector genes described in other fungi are limited by the large number of potential proteins that they identify (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Mirzadi Gohari <em>et al.</em> 2015</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Morais do Amaral <em>et al.</em> 2012</a>). Using information from gene-tree alignments, we were able to directly associate gene sequence with virulence phenotypes. This method assumes that differences at the DNA sequence level will influence the function of the proteins they encode and, in turn, influence the observed phenotype. The advantage of such a strategy is that there is no bias on what the effector proteins should “look” like, eliminating the need for arbitrary cutoffs. Using this method, we were able to assess over 10,000 individual gene trees in &lt; 4 min of computational time, resulting in the identification of six candidate genes.</p><p id="p-40">PhyBi was developed after the observation that many of the individual gene alignments possessed highly divergent haplotypes (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). These alignments are difficult to automatically pick out of the ∼10,000 predicted genes, because their pairwise identity remains quite high (large diversity between haplotypes but very little within). The haplotype networks shown in <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4, E and F</a> nicely demonstrate some of the extreme examples of this phenomenon. The maintenance of two highly divergent alleles at the same locus is often an indication of balancing selection. To test this hypothesis, structured populations from a single field would be required to find the allelic frequency of each haplotype. These gene alignments naturally raise the questions why would these divergent forms of genes be maintained (if not pseudo-genes), and why are so few intermediate haplotypes found?</p><p id="p-41">While three out of six candidate genes in Table 4 encode small proteins, none of the six genes fulfill all of the criteria of typical “effectors”. When compared to previously published <em>in planta</em> RNA sequencing data, only one out of the six genes was induced during infection (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Kellner <em>et al.</em> 2014</a>). This fits with the recent observation by <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Mirzadi Gohari <em>et al.</em> (2015)</a> that highly expressed genes do not seem to overlap with virulence quantitative trait loci.</p><p id="p-42">While there are no truly “avirulent” isolates of <em>Z. tritici</em>, isolate-specific interactions with particular wheat cultivars are described extensively in the literature (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Arraiano and Brown 2006</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Brading <em>et al.</em> 2002</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chartrain <em>et al.</em> 2004</a>). Similarly, the pathotypes described here show distinct differences in virulence toward wheat cultivars carrying the same postulated resistance genes (<em>i.e.</em>, Arina and Hereward, <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). These differences in isolate virulence also indicate that there remain uncharacterized resistance genes/loci in these varieties (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). WAI321, WAI322, and WAI323 were selected to group against all other isolates based on the phenotype clustering that did not match the neutral tree (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3</a>). Isolate WAI327, which is genetically similar to these three isolates, is less virulent against four cultivars: Hereward, Tadorna, Veranopolis, and WW2449. These four cultivars are postulated to contain the major resistance genes <em>Stb6</em> and <em>15</em>, <em>Stb4</em>, <em>Stb2</em>, and <em>6</em>, and <em>Stb11</em> and <em>WW2449</em>, respectively (reviewed in <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Brown <em>et al.</em> 2015</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Goodwin 2007</a>). Note, <em>Stb</em>2 and <em>Stb11</em> have been mapped to the same location on the short arm of chromosome 1B (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Brown <em>et al.</em> 2015</a>).</p><p id="p-43">While it is tempting to speculate that the six candidate genes may be an unknown effector virulent against <em>Stb6</em> or <em>Stb2/11</em>, these major genes are present in only two out of the four cultivars. Alternatively, the gene/s responsible for virulence on these four cultivars could be a more general virulence-associated effector. Genetic knockouts of these genes in the isolates used in this study are currently underway to test this hypothesis.</p></div><div id="sec-18" class="subsection"><h3>Concluding remarks</h3><p id="p-44">The lack of understanding of how <em>Z. tritici</em> invades its host limits our ability to identify and pursue durable resistance strategies. This study highlights the high levels of existing genetic diversity in <em>Z. tritici</em> populations within Australia, and the challenge associated with linking virulence to causal genes, where a million SNPs and hundreds of gene P/A polymorphisms obscure meaningful patterns. This work seeks to simplify this search by utilizing established computational tools (BLAST, alignment, and phlyogenetics) to link divergent gene sequences with matching phenotypes. This method is not limited to virulence data alone, and can be applied to a wide variety of phenotypes, or even interesting genes that violate the relationships shown by neutral genetic markers. This method is particularly useful when large numbers of individuals, required by traditional genome-wide association studies, are difficult to obtain.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-45">We would like to acknowledge Cameron Jack of The ANU ABC at JSCMR for constructive coding feedback for PhyBi. M.C.M. acknowledges the Swiss National Science Foundation for postdoctoral support (Project PBEZP3_142919). P.S.S. acknowledges the Australian Research Council (FT110100698). The phenotyping of cultivars was supported by the NSW Department of Primary Industries and Grains Research, and Development Corporation projects DAN00177 and DAN00175. The sequencing of four isolates was funded by Charles Sturt University, and the NSW Department of Primary Industries. The funders had no role in study, design, analyses or preparation of the manuscript for publication.</p><p id="p-46">Author contributions: M.C.M., A.M., and P.S.S. conceived of the study, and coordinated its design and execution. A.M. performed phenotyping. M.C.M., L.M., J.H., and A.W. performed bioinformatics analysis. M.C.M. and L.M. conceived and wrote PhyBi. M.C.M. and P.S.S. drafted the manuscript. All authors contributed to and approved the final manuscript. The authors declare no competing interests.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-4"><p id="p-47">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025197/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025197/-/DC1</a></p></li><li class="fn" id="fn-5"><p id="p-48"><em>Communicating editor: J. D. Faris</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-11-22"><span class="received-label">Received </span>November 22, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-01-15"><span class="accepted-label">Accepted </span>January 15, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 McDonald <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-2" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arraiano</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">J. K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Identification of isolate-specific and partial resistance to Septoria tritici blotch in 238 European wheat cultivars and breeding lines.</span> <abbr class="cit-jnl-abbrev">Plant Pathol.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">726</span>–<span class="cit-lpage">738</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPathol.%26rft.volume%253D56%26rft.spage%253D726%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.2" data-doi="10.1071/AR9950921"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ballantyne</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Pathogenic variation in Australian isolates of <em>Mycosphaerella graminicola</em>.</span> <abbr class="cit-jnl-abbrev">Aust. J. Agric. Res.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">921</span>–<span class="cit-lpage">934</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAust.%2BJ.%2BAgric.%2BRes.%26rft.volume%253D46%26rft.spage%253D921%26rft_id%253Dinfo%253Adoi%252F10.1071%252FAR9950921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/AR9950921&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.3" data-doi="10.1111/j.1365-294X.2005.02536.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Banke</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Migration patterns among global populations of the pathogenic fungus <em>Mycosphaerella graminicola</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1881</span>–<span class="cit-lpage">1896</span>. 10.1111/j.1365–294X.2005.02536.x.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DBanke%26rft.auinit1%253DS.%26rft.volume%253D14%26rft.issue%253D7%26rft.spage%253D1881%26rft.epage%253D1896%26rft.atitle%253DMigration%2Bpatterns%2Bamong%2Bglobal%2Bpopulations%2Bof%2Bthe%2Bpathogenic%2Bfungus%2BMycosphaerella%2Bgraminicola.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2005.02536.x%26rft_id%253Dinfo%253Apmid%252F15910313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2005.02536.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15910313&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229190300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.4" data-doi="10.1016/j.fgb.2003.09.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Banke</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peschon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Phylogenetic analysis of globally distributed <em>Mycosphaerella graminicola</em> populations based on three DNA sequence loci.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">226</span>–<span class="cit-lpage">238</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DBanke%26rft.auinit1%253DS.%26rft.volume%253D41%26rft.issue%253D2%26rft.spage%253D226%26rft.epage%253D238%26rft.atitle%253DPhylogenetic%2Banalysis%2Bof%2Bglobally%2Bdistributed%2BMycosphaerella%2Bgraminicola%2Bpopulations%2Bbased%2Bon%2Bthree%2BDNA%2Bsequence%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2003.09.006%26rft_id%253Dinfo%253Apmid%252F14732268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2003.09.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14732268&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188217300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.5" data-doi="10.1089/cmb.2012.0021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bankevich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nurk</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antipov</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gurevich</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dvorkin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite><span class="cit-pub-date">2012</span> <span class="cit-article-title">SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing.</span> <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">455</span>–<span class="cit-lpage">477</span>. 10.1089/cmb.2012.0021.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DBankevich%26rft.auinit1%253DA.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D455%26rft.epage%253D477%26rft.atitle%253DSPAdes%253A%2Ba%2Bnew%2Bgenome%2Bassembly%2Balgorithm%2Band%2Bits%2Bapplications%2Bto%2Bsingle-cell%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2012.0021%26rft_id%253Dinfo%253Apmid%252F22506599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.2012.0021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22506599&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.6" data-doi="10.1016/j.fgb.2015.03.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ben M’barek</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cordewener</span>  <span class="cit-name-given-names">J. H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tabib Ghaffary</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Lee</span>  <span class="cit-name-given-names">T. A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite><span class="cit-pub-date">2015</span> <span class="cit-article-title">FPLC and liquid-chromatography mass spectrometry identify candidate necrosis-inducing proteins from culture filtrates of the fungal wheat pathogen <em>Zymoseptoria tritici</em>.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">54</span>–<span class="cit-lpage">62</span>. 10.1016/j.fgb.2015.03.015.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BBiol.%26rft.volume%253D79%26rft.spage%253D54%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2015.03.015%26rft_id%253Dinfo%253Apmid%252F26092790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2015.03.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26092790&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.7" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usadel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>. 10.1093/bioinformatics/btu170.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMzAvMTUvMjExNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC83NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bouckaert</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heled</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>. DensiTree 2: Seeing Trees Through the Forest. bioRxiv. Available at: http://dx.doi.org/10.1101/012401. Accessed: October 28, 2015.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bouckaert</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heled</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kühnert</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaughan</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C.-H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite><span class="cit-pub-date">2014</span> <span class="cit-article-title">BEAST 2: a software platform for Bayesian evolutionary analysis. PLoS Comp.</span> <abbr class="cit-jnl-abbrev">Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1003537</span> 10.1371/journal.pcbi.1003537.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%26rft.volume%253D10%26rft.spage%253De1003537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.10" data-doi="10.1094/PHYTO.2002.92.4.439"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brading</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verstappen</span>  <span class="cit-name-given-names">E. C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kema</span>  <span class="cit-name-given-names">G. H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">J. K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A gene-for-gene relationship between wheat and <em>Mycosphaerella graminicola</em>, the Septoria tritici Blotch pathogen.</span> <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">439</span>–<span class="cit-lpage">445</span>. 10.1094/PHYTO.2002.92.4.439.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.stitle%253DPhytopathology%26rft.aulast%253DBrading%26rft.auinit1%253DP.%2BA.%26rft.volume%253D92%26rft.issue%253D4%26rft.spage%253D439%26rft.epage%253D445%26rft.atitle%253DA%2BGene-for-Gene%2BRelationship%2BBetween%2BWheat%2Band%2BMycosphaerella%2Bgraminicola%252C%2Bthe%2BSeptoria%2BTritici%2BBlotch%2BPathogen.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPHYTO.2002.92.4.439%26rft_id%253Dinfo%253Apmid%252F18942957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PHYTO.2002.92.4.439&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18942957&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174530800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-2" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.11" data-doi="10.1016/j.fgb.2015.04.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">J. K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chartrain</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lasserre-Zuber</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saintenac</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetics of resistance to <em>Zymoseptoria tritici</em> and applications to wheat breeding.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">41</span>. 10.1016/j.fgb.2015.04.017.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BBiol.%26rft.volume%253D79%26rft.spage%253D33%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2015.04.017%26rft_id%253Dinfo%253Apmid%252F26092788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2015.04.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26092788&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.12" data-doi="10.1111/j.1365-3059.2004.01052.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chartrain</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brading</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Makepeace</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Sources of resistance to <em>Septoria tritici</em> blotch and implications for wheat breeding.</span> <abbr class="cit-jnl-abbrev">Plant Pathol.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">454</span>–<span class="cit-lpage">460</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPathol.%26rft.volume%253D53%26rft.spage%253D454%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-3059.2004.01052.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-3059.2004.01052.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.4.779.13" data-doi="10.1111/j.1365-3059.2005.01164.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chartrain</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brading</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">J. K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Presence of the Stb6 gene for resistance to Septoria tritici blotch (<em>Mycosphaerella graminicola</em>) in cultivars used in wheat-breeding programmes worldwide.</span> <abbr class="cit-jnl-abbrev">Plant Pathol.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">134</span>–<span class="cit-lpage">143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPathol.%26rft.volume%253D54%26rft.spage%253D134%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-3059.2005.01164.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-3059.2005.01164.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.14" data-doi="10.1186/1471-2164-14-887"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lian</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhai</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite><span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome comparison of two <em>Magnaporthe oryzae</em> field isolates reveals genome variations and potential virulence effectors.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">887</span> 10.1186/1471–2164–14–887.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D887%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-887%26rft_id%253Dinfo%253Apmid%252F24341723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-887&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24341723&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.15" data-doi="10.1371/journal.ppat.1002147"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chuma</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isobe</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hotta</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ibaragi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Futamata</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite><span class="cit-pub-date">2011</span> <span class="cit-article-title">Multiple translocation of the <em>AVR-Pita</em> effector gene among chromosomes of the rice blast fungus <em>Magnaporthe oryzae</em> and related species.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002147</span> 10.1371/journal.ppat.1002147.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DChuma%26rft.auinit1%253DI.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De1002147%26rft.epage%253De1002147%26rft.atitle%253DMultiple%2Btranslocation%2Bof%2Bthe%2BAVR-Pita%2Beffector%2Bgene%2Bamong%2Bchromosomes%2Bof%2Bthe%2Brice%2Bblast%2Bfungus%2BMagnaporthe%2Boryzae%2Band%2Brelated%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002147%26rft_id%253Dinfo%253Apmid%252F21829350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002147&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21829350&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.16" data-doi="10.4161/fly.19695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cingolani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite><span class="cit-pub-date">2012</span> <span class="cit-article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of <em>Drosophila melanogaster</em> strain w1118; iso-2; iso-3.</span> <abbr class="cit-jnl-abbrev">Fly (Austin)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">80</span>–<span class="cit-lpage">92</span>. 10.4161/fly.19695.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFly%2B%2528Austin%2529%26rft.volume%253D6%26rft.spage%253D80%26rft_id%253Dinfo%253Adoi%252F10.4161%252Ffly.19695%26rft_id%253Dinfo%253Apmid%252F22728672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/fly.19695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22728672&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clement</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snell</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walke</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crandall</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">TCS:estimating gene genealogies.</span> <abbr class="cit-jnl-abbrev">Proc. 16th Int. Parallel Dist. Proc. Symp.</abbr> doi: 10.1109/IPDPS.2002.1016.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.18" data-doi="10.1111/j.1365-3059.1993.tb01560.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cohen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyal</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">The histology of processes associated with the infection of resistant and susceptible wheat cultivars with <em>Septoria tritici</em>.</span> <abbr class="cit-jnl-abbrev">Plant Pathol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">743</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPathol.%26rft.volume%253D42%26rft.spage%253D737%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-3059.1993.tb01560.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-3059.1993.tb01560.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Croll</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The accessory genome as a cradle for adaptive evolution in pathogens.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002608</span> 10.1371/journal.ppat.1002608.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D8%26rft.spage%253De1002608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.20" data-doi="10.1371/journal.pgen.1003567"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Croll</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zala</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Breakage-fusion-bridge cycles and large insertions contribute to the rapid evolution of accessory chromosomes in a fungal pathogen.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003567</span> 10.1371/journal.pgen.1003567.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003567%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003567%26rft_id%253Dinfo%253Apmid%252F23785303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003567&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23785303&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.21" data-doi="10.1111/j.1364-3703.2011.00783.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dean</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Kan</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pretorius</span>  <span class="cit-name-given-names">Z. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hammond-Kosack</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Di Pietro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite><span class="cit-pub-date">2012</span> <span class="cit-article-title">The top 10 fungal pathogens in molecular plant pathology.</span> Mol. <abbr class="cit-jnl-abbrev">Plant Pathol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">414</span>–<span class="cit-lpage">430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPathol.%26rft.volume%253D13%26rft.spage%253D414%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1364-3703.2011.00783.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1364-3703.2011.00783.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.22" data-doi="10.1038/nature03449"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dean</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talbot</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebbole</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farman</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">T. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite><span class="cit-pub-date">2005</span> <span class="cit-article-title">The genome sequence of the rice blast fungus <em>Magnaporthe grisea</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">434</span>: <span class="cit-fpage">980</span>–<span class="cit-lpage">986</span>. 10.1038/nature03449.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDean%26rft.auinit1%253DR.%2BA.%26rft.volume%253D434%26rft.issue%253D7036%26rft.spage%253D980%26rft.epage%253D986%26rft.atitle%253DThe%2Bgenome%2Bsequence%2Bof%2Bthe%2Brice%2Bblast%2Bfungus%2BMagnaporthe%2Bgrisea.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03449%26rft_id%253Dinfo%253Apmid%252F15846337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03449&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15846337&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228524600029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.23" data-doi="10.1111/j.1364-3703.2009.00591.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Wit</span>  <span class="cit-name-given-names">P. J. G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mehrabi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van den Burg</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stergiopoulos</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fungal effector proteins: past, present and future.</span> Mol. <abbr class="cit-jnl-abbrev">Plant Pathol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">735</span>–<span class="cit-lpage">747</span>. 10.1111/j.1364–3703.2009.00591.x.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPathol.%26rft.volume%253D10%26rft.spage%253D735%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1364-3703.2009.00591.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1364-3703.2009.00591.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.24" data-doi="10.1038/nrg2812"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dodds</span>  <span class="cit-name-given-names">P. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rathjen</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Plant immunity: towards an integrated view of plant–pathogen interactions.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">539</span>–<span class="cit-lpage">548</span>. 10.1038/nrg2812.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDodds%26rft.auinit1%253DP.%2BN.%26rft.volume%253D11%26rft.issue%253D8%26rft.spage%253D539%26rft.epage%253D548%26rft.atitle%253DPlant%2Bimmunity%253A%2Btowards%2Ban%2Bintegrated%2Bview%2Bof%2Bplant-pathogen%2Binteractions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2812%26rft_id%253Dinfo%253Apmid%252F20585331%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2812&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20585331&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279988800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.25" data-doi="10.1094/MPMI-01-11-0006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duplessis</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hacquard</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delaruelle</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tisserant</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frey</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite><span class="cit-pub-date">2011</span> <span class="cit-article-title"><em>Melampsora larici-populina</em> transcript profiling during germination and timecourse infection of poplar leaves reveals dynamic expression patterns associated with virulence and biotrophy.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Microbe Interact.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">808</span>–<span class="cit-lpage">818</span> 10.1094/MPMI-01–11–0006.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bplant-microbe%2Binteractions%2B%253A%2B%2BMPMI%26rft.stitle%253DMol%2BPlant%2BMicrobe%2BInteract%26rft.aulast%253DDuplessis%26rft.auinit1%253DS.%26rft.volume%253D24%26rft.issue%253D7%26rft.spage%253D808%26rft.epage%253D818%26rft.atitle%253DMelampsora%2Blarici-populina%2Btranscript%2Bprofiling%2Bduring%2Bgermination%2Band%2Btimecourse%2Binfection%2Bof%2Bpoplar%2Bleaves%2Breveals%2Bdynamic%2Bexpression%2Bpatterns%2Bassociated%2Bwith%2Bvirulence%2Band%2Bbiotrophy.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FMPMI-01-11-0006%26rft_id%253Dinfo%253Apmid%252F21644839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/MPMI-01-11-0006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21644839&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291763100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.26" data-doi="10.1093/nar/gkh340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1792</span>–<span class="cit-lpage">1797</span>. 10.1093/nar/gkh340.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjMyLzUvMTc5MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC83NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.27" data-doi="10.1038/nprot.2007.131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Emanuelsson</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunak</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Heijne</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Locating proteins in the cell using TargetP, SignalP and related tools.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">953</span>–<span class="cit-lpage">971</span>. 10.1038/nprot.2007.131.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DEmanuelsson%26rft.auinit1%253DO.%26rft.volume%253D2%26rft.issue%253D4%26rft.spage%253D953%26rft.epage%253D971%26rft.atitle%253DLocating%2Bproteins%2Bin%2Bthe%2Bcell%2Busing%2BTargetP%252C%2BSignalP%2Band%2Brelated%2Btools.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2007.131%26rft_id%253Dinfo%253Apmid%252F17446895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2007.131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17446895&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253138600024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.28" data-doi="10.1016/j.fgb.2015.04.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fones</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gurr</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The impact of <em>Septoria tritici</em> blotch disease on wheat: An EU perspective.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">7</span>. 10.1016/j.fgb.2015.04.004.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BBiol.%26rft.volume%253D79%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2015.04.004%26rft_id%253Dinfo%253Apmid%252F26092782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2015.04.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26092782&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.29" data-doi="10.1371/journal.ppat.1002952"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gardiner</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Covarelli</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solomon</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rusu</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite><span class="cit-pub-date">2012</span> <span class="cit-article-title">Comparative pathogenomics reveals horizontally acquired novel virulence genes in fungi infecting cereal hosts.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002952</span> 10.1371/journal.ppat.1002952.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DGardiner%26rft.auinit1%253DD.%2BM.%26rft.volume%253D8%26rft.issue%253D9%26rft.spage%253De1002952%26rft.epage%253De1002952%26rft.atitle%253DComparative%2Bpathogenomics%2Breveals%2Bhorizontally%2Bacquired%2Bnovel%2Bvirulence%2Bgenes%2Bin%2Bfungi%2Binfecting%2Bcereal%2Bhosts.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002952%26rft_id%253Dinfo%253Apmid%252F23028337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002952&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23028337&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-2" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goodwin</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Back to basics and beyond: increasing the level of resistance to <em>Septoria tritici</em> blotch in wheat. Austral.</span> <abbr class="cit-jnl-abbrev">Plant Pathol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">532</span>–<span class="cit-lpage">538</span>. 10.1071/AP07068.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPathol.%26rft.volume%253D36%26rft.spage%253D532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.31" data-doi="10.1371/journal.pgen.1002070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goodwin</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">M’barek</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhillon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wittenberg</span>  <span class="cit-name-given-names">A. H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crane</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite><span class="cit-pub-date">2011</span> <span class="cit-article-title">Finished genome of the fungal wheat pathogen <em>Mycosphaerella graminicola</em> reveals dispensome structure, chromosome plasticity, and stealth pathogenesis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002070</span> 10.1371/journal.pgen.1002070.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGoodwin%26rft.auinit1%253DS.%2BB.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253De1002070%26rft.epage%253De1002070%26rft.atitle%253DFinished%2Bgenome%2Bof%2Bthe%2Bfungal%2Bwheat%2Bpathogen%2BMycosphaerella%2Bgraminicola%2Breveals%2Bdispensome%2Bstructure%252C%2Bchromosome%2Bplasticity%252C%2Band%2Bstealth%2Bpathogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002070%26rft_id%253Dinfo%253Apmid%252F21695235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21695235&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.32" data-doi="10.1093/bioinformatics/btu815"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haubold</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klotzl</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfaffelhuber</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">andi: Fast and accurate estimation of evolutionary distances between closely related genomes.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1169</span>–<span class="cit-lpage">1175</span>. 10.1093/bioinformatics/btu815.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu815%26rft_id%253Dinfo%253Apmid%252F25504847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIzMS84LzExNjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvNzc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.33" data-doi="10.1094/MPMI-22-2-0115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hogenhout</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van der Hoorn</span>  <span class="cit-name-given-names">R. A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terauchi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kamoun</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Emerging concepts in effector biology of plant-associated organisms.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Microbe Interact.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">122</span>. 10.1094/MPMI-22–2-0115.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bplant-microbe%2Binteractions%2B%253A%2B%2BMPMI%26rft.stitle%253DMol%2BPlant%2BMicrobe%2BInteract%26rft.aulast%253DHogenhout%26rft.auinit1%253DS.%2BA.%26rft.volume%253D22%26rft.issue%253D2%26rft.spage%253D115%26rft.epage%253D122%26rft.atitle%253DEmerging%2Bconcepts%2Bin%2Beffector%2Bbiology%2Bof%2Bplant-associated%2Borganisms.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FMPMI-22-2-0115%26rft_id%253Dinfo%253Apmid%252F19132864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/MPMI-22-2-0115&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19132864&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262441400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.779.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hollaway</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source">Septoria tritici blotch of Wheat. 2014. Information Note (No. AG1336)</span>, <span class="cit-publ-name">Victorian Government Department of Environment and Primary Industries Information Note</span>, <span class="cit-publ-loc">Melbourne, VIC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.779.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hollaway</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLean</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source">Cereal Disease Guide 2014. 2014. Information Note (No. AG1160)</span>, <span class="cit-publ-name">Victorian Government Department of Environment and Primary Industries Information Note</span>, <span class="cit-publ-loc">Melbourne, VIC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.36" data-doi="10.1007/s10658-006-9000-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jürgens</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linde</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genetic Structure of <em>Mycosphaerella graminicola</em> Populations from Iran, Argentina and Australia.</span> Eur. <abbr class="cit-jnl-abbrev">J. Plant Pathol.</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">223</span>–<span class="cit-lpage">233</span>. 10.1007/s10658–006–9000–0.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPlant%2BPathol.%26rft.volume%253D115%26rft.spage%253D223%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10658-006-9000-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10658-006-9000-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.37" data-doi="10.1093/gbe/evu101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kellner</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhattacharyya</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poppe</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsu</span>  <span class="cit-name-given-names">T. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brem</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite><span class="cit-pub-date">2014</span> <span class="cit-article-title">Expression profiling of the wheat pathogen <em>Zymoseptoria tritici</em> reveals genomic patterns of transcription and host-specific regulatory programs.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1353</span>–<span class="cit-lpage">1365</span>. 10.1093/gbe/evu101.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevu101%26rft_id%253Dinfo%253Apmid%252F24920004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjg6IjYvNi8xMzUzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzc3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.38" data-doi="10.1094/Phyto-86-777"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kema</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rijkenberg</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baayen</span>  <span class="cit-name-given-names">R. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Histology of the pathogenesis of <em>Mycosphaerella graminicola</em> in wheat.</span> <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">777</span>–<span class="cit-lpage">786</span>. 10.1094/Phyto-86–777.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.volume%253D86%26rft.spage%253D777%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPhyto-86-777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/Phyto-86-777&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996UV19300017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.39" data-doi="10.1111/j.1364-3703.2005.00304.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rudd</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antoniw</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skinner</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hargreaves</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite><span class="cit-pub-date">2005</span> <span class="cit-article-title">Metabolic and stress adaptation by <em>Mycosphaerella graminicola</em> during sporulation in its host revealed through microarray transcription profiling.</span> Mol. <abbr class="cit-jnl-abbrev">Plant Pathol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">527</span>–<span class="cit-lpage">540</span>. 10.1111/j.1364–3703.2005.00304.x.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPathol.%26rft.volume%253D6%26rft.spage%253D527%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1364-3703.2005.00304.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1364-3703.2005.00304.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.40" data-doi="10.1094/MPMI-20-2-0178"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antoniw</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carzaniga</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deller</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite><span class="cit-pub-date">2007</span> <span class="cit-article-title">Transcriptional adaptation of <em>Mycosphaerella graminicola</em> to programmed cell death (PCD) of its susceptible wheat host.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Microbe Interact.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">193</span>. 10.1094/MPMI-20–2-0178.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bplant-microbe%2Binteractions%2B%253A%2B%2BMPMI%26rft.stitle%253DMol%2BPlant%2BMicrobe%2BInteract%26rft.aulast%253DKeon%26rft.auinit1%253DJ.%26rft.volume%253D20%26rft.issue%253D2%26rft.spage%253D178%26rft.epage%253D193%26rft.atitle%253DTranscriptional%2Badaptation%2Bof%2BMycosphaerella%2Bgraminicola%2Bto%2Bprogrammed%2Bcell%2Bdeath%2B%2528PCD%2529%2Bof%2Bits%2Bsusceptible%2Bwheat%2Bhost.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FMPMI-20-2-0178%26rft_id%253Dinfo%253Apmid%252F17313169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/MPMI-20-2-0178&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17313169&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243542400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.41" data-doi="10.1093/bioinformatics/bts330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schatz</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic dark matter: the reliability of short read mapping illustrated by the genome mappability score.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2097</span>–<span class="cit-lpage">2105</span>. 10.1093/bioinformatics/bts330.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts330%26rft_id%253Dinfo%253Apmid%252F22668792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTYvMjA5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC83NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.42" data-doi="10.1371/journal.ppat.1000581"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faris</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">K.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solomon</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite><span class="cit-pub-date">2009</span> <span class="cit-article-title">SnTox3 acts in effector triggered susceptibility to induce disease on wheat carrying the <em>Snn3</em> gene.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000581</span> 10.1371/journal.ppat.1000581.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DLiu%26rft.auinit1%253DZ.%26rft.volume%253D5%26rft.issue%253D9%26rft.spage%253De1000581%26rft.epage%253De1000581%26rft.atitle%253DSnTox3%2Bacts%2Bin%2Beffector%2Btriggered%2Bsusceptibility%2Bto%2Binduce%2Bdisease%2Bon%2Bwheat%2Bcarrying%2Bthe%2BSnn3%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1000581%26rft_id%253Dinfo%253Apmid%252F19806176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1000581&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19806176&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.43" data-doi="10.1371/journal.ppat.1002467"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faris</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Syme</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite><span class="cit-pub-date">2012</span> <span class="cit-article-title">The cysteine rich necrotrophic effector SnTox1 produced by <em>Stagonospora nodorum</em> triggers susceptibility of wheat lines harboring Snn1.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002467</span> 10.1371/journal.ppat.1002467.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DLiu%26rft.auinit1%253DZ.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253De1002467%26rft.epage%253De1002467%26rft.atitle%253DThe%2Bcysteine%2Brich%2Bnecrotrophic%2Beffector%2BSnTox1%2Bproduced%2Bby%2BStagonospora%2Bnodorum%2Btriggers%2Bsusceptibility%2Bof%2Bwheat%2Blines%2Bharboring%2BSnn1.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002467%26rft_id%253Dinfo%253Apmid%252F22241993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002467&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22241993&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-other" id="cit-6.4.779.44"><div class="cit-metadata unstructured">Manning, V.A., I. Pandelova, B. Dhillon, L. J. Wilhelm, S. B. Goodwin <em>et al.</em>, 2013. Comparative genomics of a plant-pathogenic fungus, <em>Pyrenophora tritici-repentis</em>, reveals transduplication and the impact of repeat elements on pathogenicity and population divergence. G3 (Bethesda) 3: 41–63. doi: 10.1534/g3.112.004044.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.45" data-doi="10.1104/pp.111.176347"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kombrink</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Motteram</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loza-Reyes</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, Lucas, J., <em>et al</em>. <span class="cit-pub-date">2011</span> <span class="cit-article-title">Analysis of two in planta expressed LysM effector homologs from the fungus <em>Mycosphaerella graminicola</em> reveals novel functional properties and varying contributions to virulence on wheat.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">756</span>–<span class="cit-lpage">769</span>. 10.1104/pp.111.176347.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.111.176347%26rft_id%253Dinfo%253Apmid%252F21467214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNTYvMi83NTYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvNzc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friesen</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunner</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Global diversity and distribution of three necrotrophic effectors in <em>Phaeosphaeria nodorum</em> and related species.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> 10.1111/nph.12257.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.47" data-doi="10.1016/j.fgb.2015.04.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milgate</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pattemore</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solomon</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite><span class="cit-pub-date">2015</span> <span class="cit-article-title">Next-generation re-sequencing as a tool for rapid bioinformatic screening of presence and absence of genes and accessory chromosomes across isolates of <em>Zymoseptoria tritici</em>.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">75</span>. 10.1016/j.fgb.2015.04.012.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BBiol.%26rft.volume%253D79%26rft.spage%253D71%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2015.04.012%26rft_id%253Dinfo%253Apmid%252F26092791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2015.04.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26092791&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.779.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Milgate</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wallwork</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geoff</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source">Septoria tritici blotch: Fact sheet</span>, <span class="cit-publ-name">Grains and Research Development Corporation</span>, <span class="cit-publ-loc">Canberra, Australia</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Milgate</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adorada</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orchard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pattemore</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">First report of resistance to DMI fungicides in Australian populations of the wheat pathogen, <em>Zymoseptoria tritici</em>.</span> <abbr class="cit-jnl-abbrev">Plant Dis.</abbr> 10.1094/PDIS-06–15–0704-PDN.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.50" data-doi="10.1111/mpp.12251"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mirzadi Gohari</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ware</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wittenberg</span>  <span class="cit-name-given-names">A. H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mehrabi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ben M’barek</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite><span class="cit-pub-date">2015</span> <span class="cit-article-title">Effector discovery in the fungal wheat pathogen <em>Zymoseptoria tritici</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Pathol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">931</span>–<span class="cit-lpage">945</span>. 10.1111/mpp.12251.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%2BPathol.%26rft.volume%253D16%26rft.spage%253D931%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmpp.12251%26rft_id%253Dinfo%253Apmid%252F25727413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mpp.12251&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25727413&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.51" data-doi="10.1371/journal.pone.0049904"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morais do Amaral</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antoniw</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rudd</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hammond-Kosack</span>  <span class="cit-name-given-names">K. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Defining the predicted protein secretome of the fungal wheat leaf pathogen <em>Mycosphaerella graminicola</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e49904</span> 10.1371/journal.pone.0049904.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D7%26rft.spage%253De49904%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0049904%26rft_id%253Dinfo%253Apmid%252F23236356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0049904&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23236356&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.52" data-doi="10.1111/j.1364-3703.2010.00688.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orton</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deller</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">J. K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title"><em>Mycosphaerella graminicola</em>: from genomics to disease control.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Pathol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">413</span>–<span class="cit-lpage">424</span>. 10.1111/j.1364–3703.2010.00688.x.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BPlant%2BPathol%26rft.aulast%253DOrton%26rft.auinit1%253DE.%2BS.%26rft.volume%253D12%26rft.issue%253D5%26rft.spage%253D413%26rft.epage%253D424%26rft.atitle%253DMycosphaerella%2Bgraminicola%253A%2Bfrom%2Bgenomics%2Bto%2Bdisease%2Bcontrol.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1364-3703.2010.00688.x%26rft_id%253Dinfo%253Apmid%252F21535348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1364-3703.2010.00688.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21535348&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000290169900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Persoons</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delaruelle</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halkett</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite><span class="cit-pub-date">2014</span> <span class="cit-article-title">Patterns of genomic variation in the poplar rust fungus <em>Melampsora larici-populina</em> identify pathogenesis-related factors.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">450</span> 10.3389/fpls.2014.00450.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D5%26rft.spage%253D450%26rft_id%253Dinfo%253Apmid%252F25309551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25309551&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.54" data-doi="10.1371/journal.ppat.1005055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poppe</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorsheimer</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Happel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stukenbrock</span>  <span class="cit-name-given-names">E. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Rapidly evolving genes are key players in host specialization and virulence of the fungal wheat pathogen <em>Zymoseptoria tritici</em> (<em>Mycosphaerella graminicola</em>).</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005055</span> 10.1371/journal.ppat.1005055.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D11%26rft.spage%253De1005055%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1005055%26rft_id%253Dinfo%253Apmid%252F26225424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1005055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26225424&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.4.779.55" data-doi="10.1007/s11032-009-9280-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raman</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milgate</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imtiaz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">M.-K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raman</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite><span class="cit-pub-date">2009</span> <span class="cit-article-title">Molecular mapping and physical location of major gene conferring seedling resistance to <em>Septoria tritici</em> blotch in wheat.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">153</span>–<span class="cit-lpage">164</span>. 10.1007/s11032–009–9280–0.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D24%26rft.spage%253D153%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-009-9280-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-009-9280-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.56" data-doi="10.1128/AEM.00351-08"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez-Carres</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsuchiya</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taga</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">VanEtten</span>  <span class="cit-name-given-names">H. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The supernumerary chromosome of <em>Nectria haematococca</em> that carries pea-pathogenicity-related genes also carries a trait for pea rhizosphere competitiveness.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">3849</span>–<span class="cit-lpage">3856</span>. 10.1128/AEM.00351–08.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.00351-08%26rft_id%253Dinfo%253Apmid%252F18408061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI3NC8xMi8zODQ5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzc3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rudd</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanyuka</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hassani-Pak</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Derbyshire</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andongabo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite><span class="cit-pub-date">2015</span> <span class="cit-article-title">Transcriptome and metabolite profiling the infection cycle of <em>Zymoseptoria tritici</em> on wheat (<em>Triticum aestivum</em>) reveals a biphasic interaction with plant immunity involving differential pathogen chromosomal contributions, and a variation on the hemibiotrophic lifestyle definition.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> 10.1104/pp.114.255927.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.58" data-doi="10.1016/j.fgb.2015.04.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sánchez-Vallet</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solomon</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Is <em>Zymoseptoria tritici</em> a hemibiotroph?</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">32</span>. 10.1016/j.fgb.2015.04.001.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BBiol.%26rft.volume%253D79%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2015.04.001%26rft_id%253Dinfo%253Apmid%252F26092787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2015.04.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26092787&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.59" data-doi="10.1186/1471-2164-14-119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houterman</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schreiver</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amyotte</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite><span class="cit-pub-date">2013</span> <span class="cit-article-title">MITEs in the promoters of effector genes allow prediction of novel virulence genes in <em>Fusarium oxysporum</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">119</span> 10.1186/1471–2164–14–119.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DSchmidt%26rft.auinit1%253DS.%2BM.%26rft.volume%253D14%26rft.issue%253D1%26rft.spage%253D119%26rft.epage%253D119%26rft.atitle%253DMITEs%2Bin%2Bthe%2Bpromoters%2Bof%2Beffector%2Bgenes%2Ballow%2Bprediction%2Bof%2Bnovel%2Bvirulence%2Bgenes%2Bin%2BFusarium%2Boxysporum.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-119%26rft_id%253Dinfo%253Apmid%252F23432788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23432788&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.60" data-doi="10.1371/journal.ppat.1004806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sperschneider</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dodds</span>  <span class="cit-name-given-names">P. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardiner</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manners</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">K. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite><span class="cit-pub-date">2015</span> <span class="cit-article-title">Advances and challenges in computational prediction of effectors from plant pathogenic fungi.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1004806</span> 10.1371/journal.ppat.1004806.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D11%26rft.spage%253De1004806%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1004806%26rft_id%253Dinfo%253Apmid%252F26020524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1004806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26020524&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.61" data-doi="10.1146/annurev.phyto.112408.132637"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stergiopoulos</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Wit</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fungal effector proteins.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Phytopathol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">233</span>–<span class="cit-lpage">263</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bphytopathology%26rft.stitle%253DAnnu%2BRev%2BPhytopathol%26rft.aulast%253Dde%2BWit%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D47%26rft.spage%253D233%26rft.epage%253D263%26rft.atitle%253DFungal%2Beffector%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.phyto.112408.132637%26rft_id%253Dinfo%253Apmid%252F19400631%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.phyto.112408.132637&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19400631&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270591800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.62" data-doi="10.1094/PHYTO-11-13-0328-R"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Measuring quantitative virulence in the wheat pathogen <em>Zymoseptoria tritici</em> using high-throughput automated image analysis.</span> <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">985</span>–<span class="cit-lpage">992</span>. 10.1094/PHYTO-11–13–0328-R.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.volume%253D104%26rft.spage%253D985%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPHYTO-11-13-0328-R%26rft_id%253Dinfo%253Apmid%252F24624955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PHYTO-11-13-0328-R&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24624955&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.63" data-doi="10.1371/journal.ppat.1002893"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stukenbrock</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bataillon</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A population genomics perspective on the emergence and adaptation of new plant pathogens in agro-ecosystems.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002893</span>.10.1371/journal.ppat.1002893</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DStukenbrock%26rft.auinit1%253DE.%2BH.%26rft.volume%253D8%26rft.issue%253D9%26rft.spage%253De1002893%26rft.epage%253De1002893%26rft.atitle%253DA%2Bpopulation%2Bgenomics%2Bperspective%2Bon%2Bthe%2Bemergence%2Band%2Badaptation%2Bof%2Bnew%2Bplant%2Bpathogens%2Bin%2Bagro-ecosystems.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002893%26rft_id%253Dinfo%253Apmid%252F23028308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002893&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23028308&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.64" data-doi="10.1146/annurev.phyto.010708.154114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stukenbrock</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The origins of plant pathogens in agro-ecosystems.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Phytopathol.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">75</span>–<span class="cit-lpage">100</span>. 10.1146/annurev.phyto.010708.154114.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bphytopathology%26rft.stitle%253DAnnu%2BRev%2BPhytopathol%26rft.aulast%253DStukenbrock%26rft.auinit1%253DE.%2BH.%26rft.volume%253D46%26rft.spage%253D75%26rft.epage%253D100%26rft.atitle%253DThe%2Borigins%2Bof%2Bplant%2Bpathogens%2Bin%2Bagro-ecosystems.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.phyto.010708.154114%26rft_id%253Dinfo%253Apmid%252F18680424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.phyto.010708.154114&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18680424&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259199300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.65" data-doi="10.1093/molbev/msl169"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stukenbrock</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banke</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Javan-Nikkhah</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Origin and domestication of the fungal wheat pathogen <em>Mycosphaerella graminicola</em> via sympatric speciation.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">398</span>–<span class="cit-lpage">411</span>. 10.1093/molbev/msl169.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl169%26rft_id%253Dinfo%253Apmid%252F17095534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI0LzIvMzk4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzc3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.66" data-doi="10.1101/gr.118851.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stukenbrock</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bataillon</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dutheil</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite><span class="cit-pub-date">2011</span> <span class="cit-article-title">The making of a new pathogen: insights from comparative population genomics of the domesticated wheat pathogen <em>Mycosphaerella graminicola</em> and its wild sister species.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2157</span>–<span class="cit-lpage">2166</span>. 10.1101/gr.118851.110.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.118851.110%26rft_id%253Dinfo%253Apmid%252F21994252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8xMi8yMTU3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzc3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.67"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">te Beest</span>  <span class="cit-name-given-names">D.E</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paveley</span>  <span class="cit-name-given-names">N.D</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">M.W</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van den Bosch</span>  <span class="cit-name-given-names">F</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> Accounting for the economic risk caused by variation in disease severity in fungicide dose decisions, exemplified for <em>Mycosphaerella graminicola</em> on winter wheat. Phytopathology 103: 666–672. doi: 10.1094/PHYTO-05–12–0119-R.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thynne</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solomon</span>  <span class="cit-name-given-names">P. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Phytopathogen emergence in the genomics era.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> 10.1016/j.tplants.2015.01.009.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.69" data-doi="10.1016/j.cub.2011.10.041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Torriani</span>  <span class="cit-name-given-names">S. F. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stukenbrock</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunner</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Croll</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evidence for extensive recent intron transposition in closely related fungi.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2017</span>–<span class="cit-lpage">2022</span>. doi: 10.1016/j.cub.2011.10.041.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DTorriani%26rft.auinit1%253DS.%2BF.%26rft.volume%253D21%26rft.issue%253D23%26rft.spage%253D2017%26rft.epage%253D2022%26rft.atitle%253DEvidence%2Bfor%2Bextensive%2Brecent%2Bintron%2Btransposition%2Bin%2Bclosely%2Brelated%2Bfungi.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2011.10.041%26rft_id%253Dinfo%253Apmid%252F22100062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2011.10.041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22100062&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.70" data-doi="10.1094/MPMI-8-0041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tuori</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolpert</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ciuffetti</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Purification and immunological characterization of toxic components from cultures of <em>Pyrenophora tritici-repentis</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Microbe Interact.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bplant-microbe%2Binteractions%2B%253A%2B%2BMPMI%26rft.stitle%253DMol%2BPlant%2BMicrobe%2BInteract%26rft.aulast%253DTuori%26rft.auinit1%253DR.%2BP.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253D41%26rft.epage%253D48%26rft.atitle%253DPurification%2Band%2Bimmunological%2Bcharacterization%2Bof%2Btoxic%2Bcomponents%2Bfrom%2Bcultures%2Bof%2BPyrenophora%2Btritici-repentis.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FMPMI-8-0041%26rft_id%253Dinfo%253Apmid%252F7772802%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/MPMI-8-0041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7772802&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QW45000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.71" data-doi="10.1371/journal.ppat.1001180"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van de Wouw</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cozijnsen</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hane</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunner</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite><span class="cit-pub-date">2010</span> <span class="cit-article-title">Evolution of linked avirulence effectors in <em>Leptosphaeria maculans</em> is affected by genomic environment and exposure to resistance genes in host plants.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001180</span> 10.1371/journal.ppat.1001180.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DVan%2Bde%2BWouw%26rft.auinit1%253DA.%2BP.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253De1001180%26rft.epage%253De1001180%26rft.atitle%253DEvolution%2Bof%2Blinked%2Bavirulence%2Beffectors%2Bin%2BLeptosphaeria%2Bmaculans%2Bis%2Baffected%2Bby%2Bgenomic%2Benvironment%2Band%2Bexposure%2Bto%2Bresistance%2Bgenes%2Bin%2Bhost%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1001180%26rft_id%253Dinfo%253Apmid%252F21079787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1001180&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21079787&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.72" data-doi="10.1371/journal.pone.0005863"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wittenberg</span>  <span class="cit-name-given-names">A. H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Lee</span>  <span class="cit-name-given-names">T. A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ben M’barek</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ware</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodwin</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite><span class="cit-pub-date">2009</span> <span class="cit-article-title">Meiosis drives extraordinary genome plasticity in the haploid fungal plant pathogen <em>Mycosphaerella graminicola</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e5863</span> 10.1371/journal.pone.0005863.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DWittenberg%26rft.auinit1%253DA.%2BH.%26rft.volume%253D4%26rft.issue%253D6%26rft.spage%253De5863%26rft.epage%253De5863%26rft.atitle%253DMeiosis%2Bdrives%2Bextraordinary%2Bgenome%2Bplasticity%2Bin%2Bthe%2Bhaploid%2Bfungal%2Bplant%2Bpathogen%2BMycosphaerella%2Bgraminicola.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0005863%26rft_id%253Dinfo%253Apmid%252F19516898%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0005863&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19516898&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.73" data-doi="10.1371/journal.pgen.1002869"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite><span class="cit-pub-date">2012</span> <span class="cit-article-title">Comparative analysis of the genomes of two field isolates of the rice blast fungus <em>Magnaporthe oryzae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002869</span> 10.1371/journal.pgen.1002869.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DXue%26rft.auinit1%253DM.%26rft.volume%253D8%26rft.issue%253D8%26rft.spage%253De1002869%26rft.epage%253De1002869%26rft.atitle%253DComparative%2Banalysis%2Bof%2Bthe%2Bgenomes%2Bof%2Btwo%2Bfield%2Bisolates%2Bof%2Bthe%2Brice%2Bblast%2Bfungus%2BMagnaporthe%2Boryzae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002869%26rft_id%253Dinfo%253Apmid%252F22876203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002869&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22876203&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.779.74" data-doi="10.1007/s11032-009-9381-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zwart</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milgate</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bansal</span>  <span class="cit-name-given-names">U. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite><span class="cit-pub-date">2010</span> <span class="cit-article-title">QTL mapping of multiple foliar disease and root-lesion nematode resistances in wheat.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">107</span>–<span class="cit-lpage">124</span>. 10.1007/s11032–009–9381–9.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D26%26rft.spage%253D107%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-009-9381-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-009-9381-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/4/779.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/4/775" title="Evolution of Plant Phenotypes, from Genomes to Traits" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/4/793" title="Identification of Polymorphisms Associated with Drought Adaptation QTL in Brassica napus by Resequencing" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/4"><span>Volume 6 Issue 4, April 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/4.cover-source.jpg?itok=Lj45nPaf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/779" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-HafwxTIa10TNr3AOy0sWiXHs3aygbLjytMEs0PaqDj8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/4/779" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/4/779" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/4/779" class="active" data-icon-position="" data-hide-link-title="0">Utilizing Gene Tree Variation to Identify Candidate Effector Genes in Zymoseptoria tritici</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460130" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-jWBk39DRdyzx0htFMQFPpo76-wBDVdzWZHlGvvFEqCU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/4/779.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/4/779" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-0DGvgKILvSv8BnekJ5tYPWw2amVLYatQoJOe773_O50" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/4/779" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/4/779.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460130" id="node460130--2" data-pisa="ggg;6/4/779" data-pisa-master="ggg;g3.115.025197" data-apath="/ggg/6/4/779.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Utilizing Gene Tree Variation to Identify Candidate Effector Genes in <em>Zymoseptoria tritici</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Megan C. McDonald</span>, <span class="highwire-citation-author" data-delta="1">Lachlan McGinness</span>, <span class="highwire-citation-author" data-delta="2">James K. Hane</span>, <span class="highwire-citation-author" data-delta="3">Angela H. Williams</span>, <span class="highwire-citation-author" data-delta="4">Andrew Milgate</span> and <span class="highwire-citation-author" data-delta="5">Peter S. Solomon</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">779-791;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.025197 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460130--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Megan C. McDonald </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Sciences Division, Research School of Biology, The Australian National University, Canberra, ACT, Australia 2601</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Megan%2BC.%2BMcDonald%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McDonald%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMegan%2BC.%2BMcDonald%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lachlan McGinness </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Sciences Division, Research School of Biology, The Australian National University, Canberra, ACT, Australia 2601</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lachlan%2BMcGinness%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McGinness%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALachlan%2BMcGinness%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">James K. Hane </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Black Box Bioinformatics, Perth, WA, Australia 6102</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>CCDM Bioinformatics, Centre for Crop and Disease Management, Department of Environment and Agriculture, Curtin University, Perth, WA, Australia</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Curtin Institute for Computation, Curtin University, Perth, WA, Australia 6102</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BK.%2BHane%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hane%20JK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BK.%2BHane%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Angela H. Williams </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Black Box Bioinformatics, Perth, WA, Australia 6102</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>CCDM Bioinformatics, Centre for Crop and Disease Management, Department of Environment and Agriculture, Curtin University, Perth, WA, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Angela%2BH.%2BWilliams%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Williams%20AH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAngela%2BH.%2BWilliams%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Andrew Milgate </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>NSW Department of Primary Industries, Wagga Wagga Agricultural Institute, NSW, Australia 2795</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BMilgate%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Milgate%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BMilgate%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Peter S. Solomon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Sciences Division, Research School of Biology, The Australian National University, Canberra, ACT, Australia 2601</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BS.%2BSolomon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Solomon%20PS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BS.%2BSolomon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:peter.solomon@anu.edu.au" class="" data-icon-position="" data-hide-link-title="0">peter.solomon@anu.edu.au</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/4/779" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/4/779&amp;title=Utilizing%20Gene%20Tree%20Variation%20to%20Identify%20Candidate%20Effector%20Genes%20in%20Zymoseptoria%20tritici" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/4/779&amp;title=Utilizing%20Gene%20Tree%20Variation%20to%20Identify%20Candidate%20Effector%20Genes%20in%20Zymoseptoria%20tritici" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/4/779&amp;title=Utilizing%20Gene%20Tree%20Variation%20to%20Identify%20Candidate%20Effector%20Genes%20in%20Zymoseptoria%20tritici" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/4/779&amp;text=Utilizing%20Gene%20Tree%20Variation%20to%20Identify%20Candidate%20Effector%20Genes%20in%20Zymoseptoria%20tritici" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/4/779&amp;title=Utilizing%20Gene%20Tree%20Variation%20to%20Identify%20Candidate%20Effector%20Genes%20in%20Zymoseptoria%20tritici" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/4/779&amp;t=Utilizing%20Gene%20Tree%20Variation%20to%20Identify%20Candidate%20Effector%20Genes%20in%20Zymoseptoria%20tritici" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/4/779&amp;title=Utilizing%20Gene%20Tree%20Variation%20to%20Identify%20Candidate%20Effector%20Genes%20in%20Zymoseptoria%20tritici" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/4/779&amp;title=Utilizing%20Gene%20Tree%20Variation%20to%20Identify%20Candidate%20Effector%20Genes%20in%20Zymoseptoria%20tritici" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460130/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460130" id="node460130--3" data-pisa="ggg;6/4/779" data-pisa-master="ggg;g3.115.025197" data-apath="/ggg/6/4/779.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Utilizing Gene Tree Variation to Identify Candidate Effector Genes in <em>Zymoseptoria tritici</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Megan C. McDonald</span>, <span class="highwire-citation-author" data-delta="1">Lachlan McGinness</span>, <span class="highwire-citation-author" data-delta="2">James K. Hane</span>, <span class="highwire-citation-author" data-delta="3">Angela H. Williams</span>, <span class="highwire-citation-author" data-delta="4">Andrew Milgate</span> and <span class="highwire-citation-author" data-delta="5">Peter S. Solomon</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">779-791;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.025197 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460130--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Megan C. McDonald </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Sciences Division, Research School of Biology, The Australian National University, Canberra, ACT, Australia 2601</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Megan%2BC.%2BMcDonald%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McDonald%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMegan%2BC.%2BMcDonald%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lachlan McGinness </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Sciences Division, Research School of Biology, The Australian National University, Canberra, ACT, Australia 2601</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lachlan%2BMcGinness%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McGinness%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALachlan%2BMcGinness%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">James K. Hane </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Black Box Bioinformatics, Perth, WA, Australia 6102</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>CCDM Bioinformatics, Centre for Crop and Disease Management, Department of Environment and Agriculture, Curtin University, Perth, WA, Australia</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Curtin Institute for Computation, Curtin University, Perth, WA, Australia 6102</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BK.%2BHane%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hane%20JK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BK.%2BHane%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Angela H. Williams </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Black Box Bioinformatics, Perth, WA, Australia 6102</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>CCDM Bioinformatics, Centre for Crop and Disease Management, Department of Environment and Agriculture, Curtin University, Perth, WA, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Angela%2BH.%2BWilliams%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Williams%20AH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAngela%2BH.%2BWilliams%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Andrew Milgate </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>NSW Department of Primary Industries, Wagga Wagga Agricultural Institute, NSW, Australia 2795</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BMilgate%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Milgate%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BMilgate%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Peter S. Solomon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Sciences Division, Research School of Biology, The Australian National University, Canberra, ACT, Australia 2601</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BS.%2BSolomon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Solomon%20PS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BS.%2BSolomon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:peter.solomon@anu.edu.au" class="" data-icon-position="" data-hide-link-title="0">peter.solomon@anu.edu.au</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=779&amp;rft.epage=791&amp;rft.atitle=Utilizing%20Gene%20Tree%20Variation%20to%20Identify%20Candidate%20Effector%20Genes%20in%20Zymoseptoria%20tritici&amp;rft.volume=6&amp;rft.issue=4&amp;rft.date=2016-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=McDonald%2C+Megan+C.&amp;rft.au=McGinness%2C+Lachlan&amp;rft.au=Hane%2C+James+K.&amp;rft.au=Williams%2C+Angela+H.&amp;rft.au=Milgate%2C+Andrew&amp;rft.au=Solomon%2C+Peter+S."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460130/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460130/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460130/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460130/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460130/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460130/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460130/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460130/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460130/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460130/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460130/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460130/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/4/779" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/4/779.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/4/779.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/4/779.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/4/779.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
