<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/4/973" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/4/973.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/4/973.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/4/973.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/4/973.atom" />
<meta name="HW.pisa" content="ggg;6/4/973" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Mapping of Variable DNA Methylation Across Multiple Cell Types Defines a Dynamic Regulatory Landscape of the Human Genome" />
<meta name="DC.Identifier" content="10.1534/g3.115.025437" />
<meta name="DC.Date" content="2016-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Gu et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="DNA methylation is an important epigenetic modification involved in many biological processes and diseases. Many studies have mapped DNA methylation changes associated with embryogenesis, cell differentiation, and cancer at a genome-wide scale. Our understanding of genome-wide DNA methylation changes in a developmental or disease-related context has been steadily growing. However, the investigation of which CpGs are variably methylated in different normal cell or tissue types is still limited. Here, we present an in-depth analysis of 54 single-CpG-resolution DNA methylomes of normal human cell types by integrating high-throughput sequencing-based methylation data. We found that the ratio of methylated to unmethylated CpGs is relatively constant regardless of cell type. However, which CpGs made up the unmethylated complement was cell-type specific. We categorized the 26,000,000 human autosomal CpGs based on their methylation levels across multiple cell types to identify variably methylated CpGs and found that 22.6% exhibited variable DNA methylation. These variably methylated CpGs formed 660,000 variably methylated regions (VMRs), encompassing 11% of the genome. By integrating a multitude of genomic data, we found that VMRs enrich for histone modifications indicative of enhancers, suggesting their role as regulatory elements marking cell type specificity. VMRs enriched for transcription factor binding sites in a tissue-dependent manner. Importantly, they enriched for GWAS variants, suggesting that VMRs could potentially be implicated in disease and complex traits. Taken together, our results highlight the link between CpG methylation variation, genetic variation, and disease risk for many human cell types." />
<meta name="DC.Contributor" content="Junchen Gu" />
<meta name="DC.Contributor" content="Michael Stevens" />
<meta name="DC.Contributor" content="Xiaoyun Xing" />
<meta name="DC.Contributor" content="Daofeng Li" />
<meta name="DC.Contributor" content="Bo Zhang" />
<meta name="DC.Contributor" content="Jacqueline E. Payton" />
<meta name="DC.Contributor" content="Eugene M. Oltz" />
<meta name="DC.Contributor" content="James N. Jarvis" />
<meta name="DC.Contributor" content="Kaiyu Jiang" />
<meta name="DC.Contributor" content="Theodore Cicero" />
<meta name="DC.Contributor" content="Joseph F. Costello" />
<meta name="DC.Contributor" content="Ting Wang" />
<meta name="article:published_time" content="2016-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Mapping of Variable DNA Methylation Across Multiple Cell Types Defines a Dynamic Regulatory Landscape of the Human Genome" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;DNA methylation is an important epigenetic modification involved in many biological processes and diseases. Many studies have mapped DNA methylation changes associated with embryogenesis, cell differentiation, and cancer at a genome-wide scale. Our understanding of genome-wide DNA methylation changes in a developmental or disease-related context has been steadily growing. However, the investigation of which CpGs are variably methylated in different normal cell or tissue types is still limited. Here, we present an in-depth analysis of 54 single-CpG-resolution DNA methylomes of normal human cell types by integrating high-throughput sequencing-based methylation data. We found that the ratio of methylated to unmethylated CpGs is relatively constant regardless of cell type. However, which CpGs made up the unmethylated complement was cell-type specific. We categorized the 26,000,000 human autosomal CpGs based on their methylation levels across multiple cell types to identify variably methylated CpGs and found that 22.6% exhibited variable DNA methylation. These variably methylated CpGs formed 660,000 variably methylated regions (VMRs), encompassing 11% of the genome. By integrating a multitude of genomic data, we found that VMRs enrich for histone modifications indicative of enhancers, suggesting their role as regulatory elements marking cell type specificity. VMRs enriched for transcription factor binding sites in a tissue-dependent manner. Importantly, they enriched for GWAS variants, suggesting that VMRs could potentially be implicated in disease and complex traits. Taken together, our results highlight the link between CpG methylation variation, genetic variation, and disease risk for many human cell types.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/04/01" />
<meta name="citation_mjid" content="ggg;6/4/973" />
<meta name="citation_id" content="6/4/973" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/4/973" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/4/973.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/4/973.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/4/973.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.025437" />
<meta name="citation_pmid" content="26888867" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="973" />
<meta name="citation_lastpage" content="986" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Junchen Gu" />
<meta name="citation_author_institution" content="Department of Genetics, Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St. Louis, Missouri 63108" />
<meta name="citation_author" content="Michael Stevens" />
<meta name="citation_author_institution" content="Department of Genetics, Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St. Louis, Missouri 63108" />
<meta name="citation_author_institution" content="Department of Computer Science and Engineering, Washington University in St. Louis, Missouri 63130" />
<meta name="citation_author" content="Xiaoyun Xing" />
<meta name="citation_author_institution" content="Department of Genetics, Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St. Louis, Missouri 63108" />
<meta name="citation_author" content="Daofeng Li" />
<meta name="citation_author_institution" content="Department of Genetics, Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St. Louis, Missouri 63108" />
<meta name="citation_author" content="Bo Zhang" />
<meta name="citation_author_institution" content="Department of Developmental Biology, Washington University School of Medicine, St. Louis, Missouri 63108" />
<meta name="citation_author" content="Jacqueline E. Payton" />
<meta name="citation_author_institution" content="Department of Pathology and Immunology, Washington University School of Medicine, St. Louis, Missouri 63110" />
<meta name="citation_author" content="Eugene M. Oltz" />
<meta name="citation_author_institution" content="Department of Pathology and Immunology, Washington University School of Medicine, St. Louis, Missouri 63110" />
<meta name="citation_author" content="James N. Jarvis" />
<meta name="citation_author_institution" content="Department of Pediatrics, University of Buffalo, New York 14203" />
<meta name="citation_author" content="Kaiyu Jiang" />
<meta name="citation_author_institution" content="Department of Pediatrics, University of Buffalo, New York 14203" />
<meta name="citation_author" content="Theodore Cicero" />
<meta name="citation_author_institution" content="Department of Psychiatry, Washington University School of Medicine, St. Louis, Missouri 63110" />
<meta name="citation_author" content="Joseph F. Costello" />
<meta name="citation_author_institution" content="Brain Tumor Research Center, Department of Neurosurgery, Helen Diller Family Comprehensive Cancer Center, University of California, San Francisco, California 94143" />
<meta name="citation_author" content="Ting Wang" />
<meta name="citation_author_institution" content="Department of Genetics, Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St. Louis, Missouri 63108" />
<meta name="citation_author_institution" content="Department of Computer Science and Engineering, Washington University in St. Louis, Missouri 63130" />
<meta name="citation_author_email" content="twang@genetics.wustl.edu" />
<meta name="citation_reference" content="citation_journal_title=Neuron;citation_author=P. Chakrabarty;citation_author=A. Li;citation_author=C. Ceballos-Diaz;citation_author=J. A. Eddy;citation_author=C. C. Funk;citation_title=IL-10 Alters Immunoproteostasis in APP Mice, Increasing Plaque Burden and Worsening Cognitive Behavior.;citation_pages=519-533;citation_volume=85;citation_year=2015;citation_pmid=25619653;citation_doi=10.1016/j.neuron.2014.11.020" />
<meta name="citation_reference" content="citation_journal_title=Experimental cell research;citation_journal_abbrev=Exp Cell Res;citation_author=C. Collin;citation_author=R. Moll;citation_author=S. Kubicka;citation_author=J. P. Ouhayoun;citation_author=W. W. Franke;citation_title=Characterization of human cytokeratin 2, an epidermal cytoskeletal protein synthesized late during differentiation.;citation_pages=132-141;citation_volume=202;citation_year=1992;citation_issue=1;citation_pmid=1380918;citation_doi=10.1016/0014-4827(92)90412-2" />
<meta name="citation_reference" content="citation_journal_title=Cancer Res.;citation_author=G. Deblois;citation_author=G. Chahrour;citation_author=M. C. Perry;citation_author=G. Sylvain-Drolet;citation_author=W. J. Muller;citation_title=Transcriptional Control of the ERBB2 Amplicon by ERR and PGC-1 Promotes Mammary Gland Tumorigenesis.;citation_volume=70;citation_year=2010;citation_pmid=20961995;citation_doi=10.1158/0008-5472.CAN-10-2840" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. Deleidi;citation_author=O. Cooper;citation_author=G. Hargus;citation_author=A. Levy;citation_author=O. Isacson;citation_title=Oct4-induced reprogramming is required for adult brain neural stem cell differentiation into midbrain dopaminergic neurons.;citation_pages=e19926-e19926;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21655272;citation_doi=10.1371/journal.pone.0019926" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. R. Dixon;citation_author=I. Jung;citation_author=S. Selvaraj;citation_author=Y. Shen;citation_author=J. E. Antosiewicz-Bourget;citation_title=Chromatin architecture reorganization during stem cell differentiation.;citation_pages=331-336;citation_volume=518;citation_year=2015;citation_pmid=25693564;citation_doi=10.1038/nature14222" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=G. Elliott;citation_author=C. Hong;citation_author=X. Xing;citation_author=X. Zhou;citation_author=D. Li;citation_title=Intermediate DNA methylation is a conserved signature of genome regulation.;citation_pages=6363;citation_volume=6;citation_year=2015;citation_pmid=25691127;citation_doi=10.1038/ncomms7363" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. J. Fazzari;citation_author=J. M. Greally;citation_title=Epigenomics: beyond CpG islands.;citation_pages=446-455;citation_volume=5;citation_year=2004;citation_issue=6;citation_pmid=15153997;citation_doi=10.1038/nrg1349" />
<meta name="citation_reference" content="citation_journal_title=Archivum immunologiae et therapiae experimentalis;citation_journal_abbrev=Arch Immunol Ther Exp (Warsz);citation_author=S. Gallant;citation_author=G. Gilkeson;citation_title=ETS transcription factors and regulation of immunity.;citation_pages=149-163;citation_volume=54;citation_year=2006;citation_issue=3;citation_pmid=16652219;citation_doi=10.1007/s00005-006-0017-z" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=P. Gascard;citation_author=M. Bilenky;citation_author=M. Sigaroudinia;citation_author=J. Zhao;citation_author=L. Li;citation_title=Epigenetic and transcriptional determinants of the human breast.;citation_pages=1-10;citation_volume=6;citation_year=2015;citation_doi=10.1038/ncomms6945" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=E. Gjoneska;citation_author=A. R. Pfenning;citation_author=H. Mathys;citation_author=G. Quon;citation_author=A. Kundaje;citation_title=Conserved epigenomic signals in mice and humans reveal immune basis of Alzheimer’s disease.;citation_pages=365-369;citation_volume=518;citation_year=2015;citation_pmid=25693568;citation_doi=10.1038/nature14252" />
<meta name="citation_reference" content="citation_journal_title=Neuron;citation_author=M.-V. Guillot-Sestier;citation_author=K. R. Doty;citation_author=D. Gate;citation_author=J. Rodriguez;citation_author=B. P. Leung;citation_title=Il10 Deficiency Rebalances Innate Immunity to Mitigate Alzheimer-Like Pathology.;citation_pages=534-548;citation_volume=85;citation_year=2015;citation_pmid=25619654;citation_doi=10.1016/j.neuron.2014.12.068" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=R. A. Harris;citation_author=T. Wang;citation_author=C. Coarfa;citation_author=R. P. Nagarajan;citation_author=C. Hong;citation_title=Comparison of sequencing-based methods to profile DNA methylation and identification of monoallelic epigenetic modifications.;citation_pages=1097-1105;citation_volume=28;citation_year=2010;citation_issue=10;citation_pmid=20852635;citation_doi=10.1038/nbt.1682" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=S. Heinz;citation_author=C. Benner;citation_author=N. Spann;citation_author=E. Bertolino;citation_author=Y. C. Lin;citation_title=Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities.;citation_pages=576-589;citation_volume=38;citation_year=2010;citation_issue=4;citation_pmid=20513432;citation_doi=10.1016/j.molcel.2010.05.004" />
<meta name="citation_reference" content="citation_journal_title=Nat. Immunol.;citation_author=M. T. Heneka;citation_author=D. T. Golenbock;citation_author=E. Latz;citation_title=Innate immunity in Alzheimer’s disease.;citation_pages=229-236;citation_volume=16;citation_year=2015;citation_pmid=25689443;citation_doi=10.1038/ni.3102" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=L. Hou;citation_author=J. J. Panthier;citation_author=H. Arnheiter;citation_title=Signaling and transcriptional regulation in the neural crest-derived melanocyte lineage: interactions between KIT and MITF;citation_pages=5379-5389;citation_volume=127;citation_year=2000;citation_issue=24;citation_pmid=11076759" />
<meta name="citation_reference" content="citation_journal_title=Arthritis and rheumatism;citation_journal_abbrev=Arthritis Rheum;citation_author=J. N. Jarvis;citation_author=K. Jiang;citation_author=M. B. Frank;citation_author=N. Knowlton;citation_author=A. Aggarwal;citation_title=Gene expression profiling in neutrophils from children with polyarticular juvenile idiopathic arthritis.;citation_pages=1488-1495;citation_volume=60;citation_year=2009;citation_issue=5;citation_pmid=19404961;citation_doi=10.1002/art.24450" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=L. Jiang;citation_author=J. Zhang;citation_author=J.-J. Wang;citation_author=L. Wang;citation_author=L. Zhang;citation_title=Sperm, but not oocyte, DNA methylome is inherited by zebrafish early embryos.;citation_pages=773-784;citation_volume=153;citation_year=2013;citation_pmid=23663777;citation_doi=10.1016/j.cell.2013.04.041" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. J. Kent;citation_author=C. W. Sugnet;citation_author=T. S. Furey;citation_author=K. M. Roskin;citation_author=T. H. Pringle;citation_title=The Human Genome Browser at UCSC.;citation_volume=12;citation_year=2002;citation_pmid=12045153;citation_doi=10.1101/gr.229102. Article published online before print in May 2002" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=H. Kobayashi;citation_author=T. Sakurai;citation_author=F. Miura;citation_author=M. Imai;citation_author=K. Mochiduki;citation_title=High-resolution DNA methylome analysis of primordial germ cells identifies gender-specific reprogramming in mice.;citation_volume=23;citation_year=2013;citation_pmid=23410886;citation_doi=10.1101/gr.148023.112" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=R. Kos;citation_author=M. V. Reedy;citation_author=R. L. Johnson;citation_author=C. A. Erickson;citation_title=The winged-helix transcription factor FoxD3 is important for establishing the neural crest lineage and repressing melanogenesis in avian embryos;citation_pages=1467-1479;citation_volume=128;citation_year=2001;citation_issue=8;citation_pmid=11262245" />
<meta name="citation_reference" content="citation_journal_title=Immunity;citation_author=O. I. Koues;citation_author=R. A. Kowalewski;citation_author=L.-W. Chang;citation_author=S. C. Pyfrom;citation_author=J. A. Schmidt;citation_title=Enhancer Sequence Variants and Transcription- Factor Deregulation Synergize to Construct Pathogenic Regulatory Circuits in B-Cell Lymphoma.;citation_pages=186-198;citation_volume=42;citation_year=2015;citation_pmid=25607463;citation_doi=10.1016/j.immuni.2014.12.021" />
<meta name="citation_reference" content="citation_journal_abbrev=Pigment Cell Melanoma Res;citation_author=J. D. Kubic;citation_author=K. P. Young;citation_author=R. S. Plummer;citation_author=A. E. Ludvik;citation_author=D. Lang;citation_title=Pigmentation PAX-ways: the role of Pax3 in melanogenesis, melanocyte stem cell maintenance, and disease.;citation_pages=627-645;citation_volume=21;citation_year=2008;citation_issue=6;citation_pmid=18983540;citation_doi=10.1111/j.1755-148X.2008.00514.x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=A. Kundaje;citation_author=W. Meuleman;citation_author=J. Ernst;citation_author=M. Bilenky;citation_author=A. Yen;citation_title=Integrative analysis of 111 reference human epigenomes.;citation_pages=317-330;citation_volume=518;citation_year=2015;citation_pmid=25693563;citation_doi=10.1038/nature14248" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=L. Laurent;citation_author=E. Wong;citation_author=G. Li;citation_author=T. Huynh;citation_author=A. Tsirigos;citation_title=Dynamic changes in the human methylome during differentiation.;citation_volume=20;citation_year=2010;citation_pmid=20133333;citation_doi=10.1101/gr.101907.109" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. A. Law;citation_author=S. E. Jacobsen;citation_title=Establishing, maintaining and modifying DNA methylation patterns in plants and animals.;citation_pages=204-220;citation_volume=11;citation_year=2010;citation_issue=3;citation_pmid=20142834;citation_doi=10.1038/nrg2719" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=H. J. Lee;citation_author=R. F. Lowdon;citation_author=B. Maricque;citation_author=B. Zhang;citation_author=M. Stevens;citation_title=Developmental enhancers revealed by extensive DNA methylome maps of zebrafish early embryos.;citation_pages=6315;citation_volume=6;citation_year=2015;citation_pmid=25697895;citation_doi=10.1038/ncomms7315" />
<meta name="citation_reference" content="citation_journal_title=Trends in molecular medicine;citation_journal_abbrev=Trends Mol Med;citation_author=C. Levy;citation_author=M. Khaled;citation_author=D. E. Fisher;citation_title=MITF: master regulator of melanocyte development and melanoma oncogene.;citation_pages=406-414;citation_volume=12;citation_year=2006;citation_issue=9;citation_pmid=16899407;citation_doi=10.1016/j.molmed.2006.07.008" />
<meta name="citation_reference" content="citation_journal_title=Methods;citation_author=D. Li;citation_author=B. Zhang;citation_author=X. Xing;citation_author=T. Wang;citation_title=Combining MeDIP-seq and MRE-seq to investigate genome-wide CpG methylation.;citation_pages=29-40;citation_volume=72;citation_year=2014;citation_pmid=25448294" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Neuron;citation_author=K. L. Ligon;citation_author=E. Huillard;citation_author=S. Mehta;citation_author=S. Kesari;citation_author=H. Liu;citation_title=Olig2-Regulated Lineage-Restricted Pathway Controls Replication Competence in Neural Stem Cells and Malignant Glioma.;citation_pages=503-517;citation_volume=53;citation_year=2007;citation_pmid=17296553;citation_doi=10.1016/j.neuron.2007.01.009" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. Lister;citation_author=M. Pelizzola;citation_author=R. H. Dowen;citation_author=R. D. Hawkins;citation_author=G. Hon;citation_title=Human DNA methylomes at base resolution show widespread epigenomic differences.;citation_pages=315-322;citation_volume=462;citation_year=2009;citation_issue=7271;citation_pmid=19829295;citation_doi=10.1038/nature08514" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=R. Lister;citation_author=E. A. Mukamel;citation_author=J. R. Nery;citation_author=M. Urich;citation_author=C. A. Puddifoot;citation_title=Global epigenomic reconfiguration during mammalian brain development.;citation_volume=341;citation_year=2013;citation_pmid=23828890;citation_doi=10.1126/science.1237905" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=R. F. Lowdon;citation_author=B. Zhang;citation_author=M. Bilenky;citation_author=T. Mauro;citation_author=D. Li;citation_title=Regulatory network decoded from epigenomes of surface ectoderm-derived cell types.;citation_pages=5442;citation_volume=5;citation_year=2014;citation_pmid=25421844;citation_doi=10.1038/ncomms6442" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=T. Matsuda;citation_author=N. Murao;citation_author=Y. Katano;citation_author=B. Juliandi;citation_title=TLR9 signalling in microglia attenuates seizure-induced aberrant neurogenesis in the adult hippocampus.;citation_pages=6514;citation_volume=6;citation_year=2015;citation_pmid=25751136;citation_doi=10.1038/ncomms7514" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. T. Maurano;citation_author=R. Humbert;citation_author=E. Rynes;citation_author=R. E. Thurman;citation_author=E. Haugen;citation_title=Systematic Localization of Common Disease-Associated Variation in Regulatory DNA.;citation_volume=337;citation_year=2012;citation_pmid=22955828;citation_doi=10.1126/science.1222794" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=C. Y. McLean;citation_author=D. Bristor;citation_author=M. Hiller;citation_author=S. L. Clarke;citation_author=B. T. Schaar;citation_title=GREAT improves functional interpretation of cis-regulatory regions.;citation_pages=495-501;citation_volume=28;citation_year=2010;citation_issue=5;citation_pmid=20436461;citation_doi=10.1038/nbt.1630" />
<meta name="citation_reference" content="citation_journal_title=Journal of Investigative Dermatology;citation_journal_abbrev=JID;citation_author=D. Mehic;citation_author=L. Bakiri;citation_author=M. Ghannadan;citation_author=E. F. Wagner;citation_author=E. Tschachler;citation_title=Fos and Jun Proteins Are Specifically Expressed During Differentiation of Human Keratinocytes;citation_pages=212-220;citation_volume=124;citation_year=2005;citation_issue=1;citation_pmid=15654976;citation_doi=10.1111/j.0022-202X.2004.23558.x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Meissner;citation_author=T. S. Mikkelsen;citation_author=H. Gu;citation_author=M. Wernig;citation_title=Genome-scale DNA methylation maps of pluripotent and differentiated cells.;citation_pages=766-770;citation_volume=454;citation_year=2008;citation_issue=7205;citation_pmid=18600261;citation_doi=10.1038/nature07107" />
<meta name="citation_reference" content="citation_journal_title=Development genes and evolution;citation_journal_abbrev=Dev Genes Evol;citation_author=E. M. Mellgren;citation_author=S. L. Johnson;citation_title=A requirement for kit in embryonic zebrafish melanocyte differentiation is revealed by melanoblast delay.;citation_pages=493-502;citation_volume=214;citation_year=2004;citation_issue=10;citation_pmid=15300437" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=D. Meulemans;citation_author=M. Bronner-Fraser;citation_title=Gene-regulatory interactions in neural crest evolution and development.;citation_pages=291-299;citation_volume=7;citation_year=2004;citation_issue=3;citation_pmid=15363405;citation_doi=10.1016/j.devcel.2004.08.007" />
<meta name="citation_reference" content="citation_journal_title=Neuron;citation_journal_abbrev=Neuron;citation_author=L. Moran-Rivard;citation_author=T. Kagawa;citation_author=H. Saueressig;citation_author=M. K. Gross;citation_author=J. Burrill;citation_title=Evx1 is a postmitotic determinant of v0 interneuron identity in the spinal cord.;citation_pages=385-399;citation_volume=29;citation_year=2001;citation_issue=2;citation_pmid=11239430;citation_doi=10.1016/S0896-6273(01)00213-6" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=A. Po;citation_author=E. Ferretti;citation_author=E. Miele;citation_author=E. De Smaele;citation_author=A. Paganelli;citation_title=Hedgehog controls neural stem cells through p53-independent regulation of Nanog;citation_pages=2646-2658;citation_volume=29;citation_year=2010;citation_issue=15;citation_pmid=20581804;citation_doi=10.1038/emboj.2010.131" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=A. Portela;citation_author=M. Esteller;citation_title=Epigenetic modifications and human disease.;citation_pages=1057-1068;citation_volume=28;citation_year=2010;citation_issue=10;citation_pmid=20944598;citation_doi=10.1038/nbt.1685" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=W. Reik;citation_author=A. Lewis;citation_title=Co-evolution of X-chromosome inactivation and imprinting in mammals.;citation_pages=403-410;citation_volume=6;citation_year=2005;citation_issue=5;citation_pmid=15818385;citation_doi=10.1038/nrg1602" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=K. D. Robertson;citation_title=DNA methylation and human disease.;citation_pages=597-610;citation_volume=6;citation_year=2005;citation_issue=8;citation_pmid=16136652;citation_doi=10.1038/nrm1228" />
<meta name="citation_reference" content="citation_journal_title=European journal of cancer (Oxford, England :  1990);citation_journal_abbrev=Eur J Cancer;citation_author=C. Schröder;citation_author=U. Schumacher;citation_author=V. Müller;citation_author=R. M. Wirtz;citation_author=T. Streichert;citation_title=The transcription factor Fra-2 promotes mammary tumour progression by changing the adhesive properties of breast cancer cells.;citation_pages=1650-1660;citation_volume=46;citation_year=2010;citation_issue=9;citation_pmid=20226654;citation_doi=10.1016/j.ejca.2010.02.008" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=M. D. Schultz;citation_author=Y. He;citation_author=J. W. Whitaker;citation_author=M. Hariharan;citation_author=E. A. Mukamel;citation_title=Human body epigenome maps reveal noncanonical DNA methylation variation.;citation_pages=212-216;citation_volume=523;citation_year=2015;citation_pmid=26030523;citation_doi=10.1038/nature14465" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=H. Shen;citation_author=H. He;citation_author=J. Li;citation_author=W. Chen;citation_author=X. Wang;citation_title=Genome-Wide Analysis of DNA Methylation and Gene Expression Changes in Two Arabidopsis Ecotypes and Their Reciprocal Hybrids.;citation_volume=24;citation_year=2012;citation_pmid=22438023;citation_doi=10.1105/tpc.111.094870" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. B. Stadler;citation_author=R. Murr;citation_author=L. Burger;citation_author=R. Ivanek;citation_author=F. Lienert;citation_title=DNA-binding factors shape the mouse methylome at distal regulatory regions.;citation_pages=490-495;citation_volume=480;citation_year=2011;citation_issue=7378;citation_pmid=22170606;citation_doi=10.1038/nature10716" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. Stevens;citation_author=J. B. Cheng;citation_author=D. Li;citation_author=M. Xie;citation_author=C. Hong;citation_title=Estimating absolute methylation levels at single-CpG resolution from methylation enrichment and restriction enzyme sequencing methods.;citation_volume=23;citation_year=2013;citation_pmid=23804401;citation_doi=10.1101/gr.152231.112" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=R. A. Stewart;citation_author=B. L. Arduini;citation_author=S. Berghmans;citation_author=R. E. George;citation_author=J. P. Kanki;citation_title=Zebrafish foxd3 is selectively required for neural crest specification, migration and survival.;citation_pages=174-188;citation_volume=292;citation_year=2006;citation_issue=1;citation_pmid=16499899;citation_doi=10.1016/j.ydbio.2005.12.035" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. Subramanian;citation_author=A. Sarkar;citation_author=A. S. Shetty;citation_author=B. Muralidharan;citation_author=H. Padmanabhan;citation_title=Transcription factor Lhx2 is necessary and sufficient to suppress astrogliogenesis and promote neurogenesis in the developing hippocampus.;citation_volume=108;citation_year=2011;citation_pmid=21690374;citation_doi=10.1073/pnas.1101109108" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=M. L. Suster;citation_author=A. Kania;citation_author=M. Liao;citation_author=K. Asakawa;citation_author=F. Charron;citation_title=A novel conserved evx1 enhancer links spinal interneuron morphology and cis-regulation from fish to mammals.;citation_pages=422-433;citation_volume=325;citation_year=2009;citation_issue=2;citation_pmid=18992237;citation_doi=10.1016/j.ydbio.2008.10.004" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=Quantum nonlinear optics with single photons enabled by strongly interacting atoms.;citation_pages=57-60;citation_volume=488;citation_year=2012;citation_issue=7409;citation_pmid=22832584;citation_doi=10.1038/nature11361" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_author=A. J. Thomas;citation_author=C. A. Erickson;citation_title=FOXD3 regulates the lineage switch between neural crest-derived glial cells and pigment cells by repressing MITF through a non-canonical mechanism.;citation_volume=136;citation_year=2009;citation_pmid=19403660;citation_doi=10.1242/dev.031989" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=A. M. Tsankov;citation_author=H. Gu;citation_author=V. Akopian;citation_author=M. J. Ziller;citation_title=Transcription factor binding dynamics during human ES cell differentiation.;citation_pages=344-349;citation_volume=518;citation_year=2015;citation_pmid=25693565;citation_doi=10.1038/nature14233" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Visel;citation_author=S. Minovitsky;citation_author=I. Dubchak;citation_author=L. A. Pennacchio;citation_title=VISTA Enhancer Browser–a database of tissue-specific human enhancers.;citation_volume=35;citation_year=2007;citation_pmid=17130149;citation_doi=10.1093/nar/gkl822" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. Xu;citation_author=B. Li;citation_author=M. Zhao;citation_author=K. E. Szulwach;citation_author=R. C. Street;citation_title=Base-resolution methylation patterns accurately predict transcription factor bindings in vivo.;citation_volume=43;citation_year=2015;citation_pmid=25722376;citation_doi=10.1093/nar/gkv151" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=G. Yang;citation_author=Y. Li;citation_author=E. K. Nishimura;citation_author=H. Xin;citation_author=A. Zhou;citation_title=Inhibition of PAX3 by TGF-beta modulates melanocyte viability.;citation_pages=554-563;citation_volume=32;citation_year=2008;citation_issue=4;citation_pmid=19026785;citation_doi=10.1016/j.molcel.2008.11.002" />
<meta name="citation_reference" content="citation_journal_title=The journal of investigative dermatology. Symposium proceedings / the Society for Investigative Dermatology, Inc. [and] European Society for Dermatological Research;citation_journal_abbrev=J Investig Dermatol Symp Proc;citation_author=H. Yoshida;citation_author=T. Kunisada;citation_author=T. Grimm;citation_author=E. K. Nishimura;citation_author=E. Nishioka;citation_title=Review: melanocyte migration and survival controlled by SCF/c-kit expression.;citation_pages=1-5;citation_volume=6;citation_year=2001;citation_issue=1;citation_pmid=11764276;citation_doi=10.1046/j.0022-202x.2001.00006.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. Zang;citation_author=D. E. Schones;citation_author=C. Zeng;citation_author=K. Cui;citation_author=K. Zhao;citation_title=A clustering approach for identification of enriched domains from histone modification ChIP-Seq data.;citation_volume=25;citation_year=2009;citation_pmid=19505939;citation_doi=10.1093/bioinformatics/btp340" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=M. Zbinden;citation_author=A. Duquet;citation_author=A. Lorente-Trigos;citation_author=S.-N. Ngwabyt;citation_title=NANOG regulates glioma stem cells and is essential in vivo acting in a cross-functional network with GLI1 and p53;citation_pages=2659-2674;citation_volume=29;citation_year=2010;citation_issue=15;citation_pmid=20581802;citation_doi=10.1038/emboj.2010.137" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. Zhang;citation_author=Y. Zhou;citation_author=N. Lin;citation_author=R. F. Lowdon;citation_author=C. Hong;citation_title=Functional DNA methylation differences between tissues, cell types, and across individuals discovered using the M&amp;M algorithm.;citation_volume=23;citation_year=2013;citation_pmid=23804400;citation_doi=10.1101/gr.156539.113" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=B. Zhang;citation_author=X. Xing;citation_author=J. Li;citation_author=R. F. Lowdon;citation_author=Y. Zhou;citation_title=Comparative DNA methylome analysis of endometrial carcinoma reveals complex and distinct deregulation of cancer promoters and enhancers.;citation_pages=1-21;citation_volume=15;citation_year=2014;citation_pmid=24382143;citation_doi=10.1186/1471-2164-15-349" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=L. Zhang;citation_author=A. Eddy;citation_author=Y. T. Teng;citation_author=M. Fritzler;citation_author=M. Kluppel;citation_title=An immunological renal disease in transgenic mice that overexpress Fli- 1, a member of the ets family of transcription factor genes;citation_pages=6961-6970;citation_volume=15;citation_year=1995;citation_issue=12;citation_pmid=8524263;citation_doi=10.1128/MCB.15.12.6961" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=X. Zhou;citation_author=B. Maricque;citation_author=M. Xie;citation_author=D. Li;citation_author=V. Sundaram;citation_title=The Human Epigenome Browser at Washington University.;citation_pages=989-990;citation_volume=8;citation_year=2011;citation_issue=12;citation_pmid=22127213;citation_doi=10.1038/nmeth.1772" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=X. Zhou;citation_author=D. Li;citation_author=B. Zhang;citation_author=R. F. Lowdon;citation_author=N. B. Rockweiler;citation_title=Epigenomic annotation of genetic variants using the Roadmap Epigenome Browser.;citation_pages=345-346;citation_volume=33;citation_year=2015;citation_pmid=25690851" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=M. J. Ziller;citation_author=H. Gu;citation_author=F. Müller;citation_author=J. Donaghey;citation_author=L. T. Y. Tsai;citation_title=Charting a dynamic DNA methylation landscape of the human genome.;citation_pages=477-481;citation_volume=500;citation_year=2013;citation_pmid=23925113;citation_doi=10.1038/nature12433" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=M. J. Ziller;citation_author=R. Edri;citation_author=Y. Yaffe;citation_author=J. Donaghey;citation_author=R. Pop;citation_title=Dissecting neural differentiation regulatory networks through epigenetic footprinting.;citation_pages=355-359;citation_volume=518;citation_year=2015;citation_pmid=25533951" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Mapping of Variable DNA Methylation Across Multiple Cell Types Defines a Dynamic Regulatory Landscape of the Human Genome" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
<meta name="twitter:description" content="DNA methylation is an important epigenetic modification involved in many biological processes and diseases. Many studies have mapped DNA methylation changes associated with embryogenesis, cell differentiation, and cancer at a genome-wide scale. Our understanding of genome-wide DNA methylation changes in a developmental or disease-related context has been steadily growing. However, the investigation of which CpGs are variably methylated in different normal cell or tissue types is still limited. Here, we present an in-depth analysis of 54 single-CpG-resolution DNA methylomes of normal human cell types by integrating high-throughput sequencing-based methylation data. We found that the ratio of methylated to unmethylated CpGs is relatively constant regardless of cell type. However, which CpGs made up the unmethylated complement was cell-type specific. We categorized the 26,000,000 human autosomal CpGs based on their methylation levels across multiple cell types to identify variably methylated CpGs and found that 22.6% exhibited variable DNA methylation. These variably methylated CpGs formed 660,000 variably methylated regions (VMRs), encompassing 11% of the genome. By integrating a multitude of genomic data, we found that VMRs enrich for histone modifications indicative of enhancers, suggesting their role as regulatory elements marking cell type specificity. VMRs enriched for transcription factor binding sites in a tissue-dependent manner. Importantly, they enriched for GWAS variants, suggesting that VMRs could potentially be implicated in disease and complex traits. Taken together, our results highlight the link between CpG methylation variation, genetic variation, and disease risk for many human cell types." />
<meta name="og-title" property="og:title" content="Mapping of Variable DNA Methylation Across Multiple Cell Types Defines a Dynamic Regulatory Landscape of the Human Genome" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/4/973" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="DNA methylation is an important epigenetic modification involved in many biological processes and diseases. Many studies have mapped DNA methylation changes associated with embryogenesis, cell differentiation, and cancer at a genome-wide scale. Our understanding of genome-wide DNA methylation changes in a developmental or disease-related context has been steadily growing. However, the investigation of which CpGs are variably methylated in different normal cell or tissue types is still limited. Here, we present an in-depth analysis of 54 single-CpG-resolution DNA methylomes of normal human cell types by integrating high-throughput sequencing-based methylation data. We found that the ratio of methylated to unmethylated CpGs is relatively constant regardless of cell type. However, which CpGs made up the unmethylated complement was cell-type specific. We categorized the 26,000,000 human autosomal CpGs based on their methylation levels across multiple cell types to identify variably methylated CpGs and found that 22.6% exhibited variable DNA methylation. These variably methylated CpGs formed 660,000 variably methylated regions (VMRs), encompassing 11% of the genome. By integrating a multitude of genomic data, we found that VMRs enrich for histone modifications indicative of enhancers, suggesting their role as regulatory elements marking cell type specificity. VMRs enriched for transcription factor binding sites in a tissue-dependent manner. Importantly, they enriched for GWAS variants, suggesting that VMRs could potentially be implicated in disease and complex traits. Taken together, our results highlight the link between CpG methylation variation, genetic variation, and disease risk for many human cell types." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
    <title>Mapping of Variable DNA Methylation Across Multiple Cell Types Defines a Dynamic Regulatory Landscape of the Human Genome | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"DXsYHCWSnYXGOt_4OPrMhuT6IvLkTOQbiJe8VhabjyM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460115","apath":"\/ggg\/6\/4\/973.atom","pisa":"ggg;6\/4\/973","ac":{"\/ggg\/6\/4\/973.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/4\/973.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/4\/973"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/4\/973"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/4\/973.figures-only":true,"\/content\/6\/4\/973":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022OzRAwuUHckw+chERU4p1AMCrOWWwQBdhCokuKWndOYEgwPswBufABTe03DVcDnSanSpIXA\\\\\\\/1pQBUftaLXK6DavZgu5GtBK2qGwCknr96sPveMJwwqh8Lqy7a9hc5ZMsg3SCxSEXegBt7Kk6lelQbVeyH2RtLe6N3yf6lgB6Cjy69gxu7HZKTxoNJ7Dy2dnD\\\\\\\/YyNMgncw+p487DGUEurxUcBQlWvSEnAjjkAPO+DjvXv56W4pfvbn+Tm8S1oZA6Y\\\\\\\/2vqbvSjDHhvXofGkfG\\\\\\\/4j0neYzM+nlaxP0\\\\\\\/dtO34+8wpE\\\\\\\/r+5Vzzj+Il0PxVZQNTOk9bP0OeVR20YpGmTD7MVztMtbs5wy54bbTnZ5YNmXyaeYLslMU\\\\\\\/vV3b3lTv+ZvXCT3EJEP75VtKDmoxEX8hxpwH2oJOycE0\\\\\\\/fC4H\\\\\\\/dvSpFidko\\\\\\\/X+uxmMsvCy\\\\\\\/aAXrSReidPmBa3vaPpj3n1O3t1JiEsKAbRxkHhB4V29PAO+v31gt2AOKc5UJnb8WgSawwe+pXdYQatQ4P57S\\\\\\\/2ESTssGKY5+\\\\\\\/gEqu0J+weVV+nvReZ\\\\\\\/+JN3rcE93zDbHztTTaWULGa3Q2C6XuVTp3bJiVLLbbnb1J+bkWyUDKi9Crdzqh3EDV9hjxRzgRr4qX3cClaWsKurnKvBl8CZKUm8BgMSSjx5A5t7ohPfHVQ65IT8Xdx\\\\\\\/j8vnz4a9DmNa+GPhgMbEQfmA1fVsuGUS0\\\\\\\/YEkmwoXaK8LBJImqYN7EEJVQzekv1BtChzdldmPzA3Tno00Ir0vz3WPubHa2Z8KUjuv44qqCouOBiYsxNTMPXDqt\\\\\\\/a\\\\\\\/t0tKoH7NHgTUNJ\\\\\\\/PuVpplppzmBjsrwWwchf6fmy6wRAc0pWlPvbxRyBzludJiAtvKiW1ozNGU+W40\\\\\\\/jdfiV5RqSxTfvxDksnOezyFNYd\\\\\\\/h3uRA0+PpFDhPkm6qAsTq1J5I7cp43sJ7RTrO3sPjFDu9MKKAKVE68WWFX5M9sFBpGlTJPvFpmKn1quuB7o+3zR9HsQUn49Kou5gUOfQ5E5aiuinFpDQEHFahbLe6nAGnKcKBnjnfxvCL9Px5ZoYM8d\\\\\\\/l5zBJubW4CQNeUJkU\\\\\\\/AWhJA9R47pWzQuV9pyh4MSSqlh91Y4FUwC81+W9nt7D+2zpCPJANQomSdjq+OhYC5\\\\\\\/FYy8BAkvrGbk5pbEGYYX3Vp9MczRVfK6sdgZDhoEVnt3nSl3WCT\\\\\\\/+\\\\\\\/+bTbBLA8i9AUzB6Mnqr8br0Zen3uqdwppOkS\\\\\\\/uv96JMeWFGArUx1Y8TVk\\\\\\\/tQqc3tnILGeo+aDG\\\u0022,\\\u0022iv\\\u0022:\\\u0022VY\\\\\\\/LTAyye\\\\\\\/mecNR1ZgOp0g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a17e425c4c529ce3b44eb606f8c5b480d51c2b800f94f2d41f3b5cf9aba1e958\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ltgGsSmpAwzWlFMG4BavK5LLCitwAVLse8JQH1zihFjgr2BQ3T0+pcKYGTqQKk6HQ2l2clzA86U\\\\\\\/jwS0be2rrYu3x158cSMu3HOWpkdqTxFzKhLsTYStsHVICCJgtWQiTnEOJuuCJ2dNa6aXu4baqIoIQ4Jvx8\\\\\\\/lMhuYNsSzv1YTS8uGkB8JewY+\\\\\\\/bEByahMrkQ3gPb1p02TK+3nxXi0gSBaEADii3XFvYceHfzGbvwSVsuPwJ0qtjtqAfzwIKg\\\\\\\/aB80SAkPNa8kvD8SWeCTKIQHEqTQMM1tHJR2u9mwqXNosSPt9VLwATE9kmX3VrmeG29eTazCZzs1BtXYq48kMfI+2Eel28cdFd3x7K7yX6+td5pX0YRmaX2e4Z6KFiUVM9EC\\\\\\\/55WBQ0P\\\\\\\/DMDl2AEN+CWtBzL6Yzvw0jNWjDf5Vdk\\\\\\\/RMaI\\\\\\\/I0vwJk+uTXzQOixzm9Scfsotb2VufBdnL8nWI8NtPI1QFaFvKyCDD0gbdUN9n9VuQBDbzjqAnUh+mNufrJsY9\\\\\\\/srf50qahqRETSzXm5UAwNVEKV8xr1IwnZIE02VhklNgiavdJ04xO5Pi3eoMgKZlPKgI1mjzKT9T6ZUzjVwUGSAWhEizAO7VcRtm3F3kvWU5tSPMUSwbMsFNS3yV0lRysqVJXjxzbMuXGnAv4V9339McLC0D9TI5uRGNTfOBliwFMBPIqvqprP2Gn1u+K9qJb6OaVEPNEAyjsbBlDefrQ5+fCvPbfqMSbzq4RUpsesNPz+DXKRJRu6gzXB1zG93rMG1ieAdIs+8HTrUh7Kn+S0zhoH30R8w9mOux2yxv414WXyjcLHCZT0pqrBlkOBrGqF9krUztWsHI+mEM7X6UPMuqUlRn5D8CyoLAQW9q+DbLNyni3ZGmmgxDlOdWTNRqker6yKTmaQLXalUrpYDntS6GBg39SNgnIFd7omu+m10WjyVOkJu8a\\\\\\\/oIXzQjoOtTP8aZ1gClRNH3et1yAOSwaRr+eDb\\\\\\\/7DmeKhzEzRtPtKonWKGLY75ZrPqgU1rgiC6WGV2lFCkwcHC0ynQsxyXKfhLc9fIS6T+VluCo98Dsc8FOgesSoyTdk4EzQCd1d4inED1aYIecF0N\\\\\\\/h+Firsn2JQ9qsLLIZDhlVUJ5D7s7B+uKz+MSHQ8XEJZET89ddCgosPlY5Lem+OQa70mQUk50oXg8AMHfBZ5ORzPz5nGV7+79LGmb1xwvr1zScylKz5Rlo5RcS48BtL6C01PwuvJSHYLZ\\\\\\\/CiT4ekHTwZWK9+c5iwxpLKI59RB1j96XuLHCsyVJ9K50h5dxQaVoZRXVjzxES8tFtwdk10Kfgmj8NaublRxDbcdSjHln5XNRMJAQ1xOkE74UmCX4aRW9QqmTgZLTEb5aNhYVOoXQjyPePrvBpTPEi1Ai3ZUQL3Nfx6FNIG0T48GDSM7SdQlFtMth6lGoSV4lqpLbMrsbbdPhu2Nfl2ByJX2OUCNylgNkw1Q0\\\\\\\/y5XTaKHmPw1A0mZrQ3w+NC5mSIG\\\\\\\/Nskh4sCARQ59PYM3WUUACh3GtnfdyNh5Be2nDZNIExjKsyzYjBkj\\\\\\\/Oz2owBMum89XJbhQxAbZ1\\\\\\\/QjBcBJT8KaVoAuAWhcASGEO1eKkdX5m2kEXTni39mXyi8tA4U7anoHIkhFDxhrV2CTY7CH5nTP3bfwE1WRmSPJqMtzR+UG2V6XPsSOMxqwaEgub3BsPRFLtD4pNMTGAwWUXPGBVE5GBK0WChzh+\\\\\\\/W62XpCu0YdGDihxAcL1KUm8B0qqYtL6mGbj6sbNjVuhpAPLddP6vuVCA9U7dvSyr6xd\\\\\\\/2NQClsUYIfrgSLJA2MqyP+rzwlLa6CM13OUtDQ7adO\\\\\\\/I2Dk3ADP3H3ViKCk+pFw3m+ca287QL6UH2pY1d+0+zmpRs8g7+W19M0QyV2SPRd\\\\\\\/vXPMCNM6xYpneNTNf4MyVxHrmQBeHjNXHG91aNVJatROqwWuO4\\\\\\\/CZZNHrpJUX3TRbdIeY7p0jGme8GVi4RzhCvmUpQAaqlP6rwmovvzpo5keL9Sl4Cci+Gsrkn6JjHtJdm1TQGm+NTIRs\\\\\\\/2H76sSr61owfucKMPT+VlTmAq+aR3n0T0esntYcm1s2uIXuc2\\\\\\\/m86Z93Tk1SEKpmbS6u1KYVuEciF3+BXFFsfwXCd3SPHv2QrL68MxDdzthSXIUX7XH1rWuUJSuPcQ93nORVGzEgMsOk9iotAYYNZgBpVYaEbYb1Lo94ZUBcwc3kXoZ1LGnqq7\\\\\\\/aHOPyHnfwUkQs71EKX3DtWjfEJaGl1wdCCIvf8JwtYpwWlq\\\\\\\/ElGCig\\\\\\\/NdKGVPQ6Mahu1\\\\\\\/8uX58FRRC0Asm2w5PvHOopMWP7l5Ck2fKQRAwrmSWbvFr6EaugxIt4FybsEgy3dwNf9GaCCqpa9JWo5oKzUZ0s76zIlgIzpg4l4voCXFupb8JJESJ6IQlVEoKdyKYHlcyFKsniWml2uO5PQEwetzdsD55CX2c+zFXCXHVDxA+76WjSJhHCgA+cTj5Qx5261mZ8DSKjvI+R0wBRc9hpAI0QTZImpvEBbZoPVeZZDI810Z4YvvxEV+eet1JitFoyMAIYwA4\\\\\\\/lGVHwooLed2zZr7s+Lc5ybq1qgeJdblWUTnmJh9FUb5HG8FJpKl9r2VHXjMNMepcmSns38Tpz2pT+K8QRoWt9fFIfZIYQd1rj\\\\\\\/\\\\\\\/GtCtEMQdIjQNonR+IsfosY6MxXMB14X6UZ9YFflho4C4OF8TeJmG58+yokSzXoj+0jL8V8eFQ3JeovMJwdksdlkdk5ibu5HKh3ppm7mWehZY8v01y2q7Qrv1pOOSC5IJH5\\\\\\\/uMpdI89pvGsTjn9fSSZHOVMbTQr\\\\\\\/9JkBizPc+182Qe8mOyQYkxZoOYou2dSgJ2u3ock3zIGkQmWi+vckSrSAaTflPxhkXCLueEPtujkFNN4Ce7HrH6KU48kvadeFdf9RI+L3JI9JnqnVlMq04bjjKrvE5dNeKno8d2s2Te9gnqxELHZ93CkvRdSp2WYVsFxN5BSp4PjCV8LA90x5qedncZDFwdGhb4q4SGFFzz7c3q8hHhjOIcwqoLJxnXAWGb64qKHi0KaHTmvmZrKyYGFpUhSvoK1nv+6gccaIPzH09S0V7edVVrM73Vbra5yXCg8jMxzgUWBlW8QrdpNh7hxPAkscAYr0UTSi4lSy+4givqhf9owr7I9cJn4ntPGCoV4uNtC1LJtOzdFTFVGzHUwcEqZBMywTCo8FZnQDRKgpRrvwZiXlIgSFSqC\\\\\\\/aw4F+AYImUrAdGwTbBcJa0EWJ3\\\\\\\/JHUbS4fw6Hw7k2Xu+v4PEVV28GHdkm6SdERYqOysttDDRiuuD9\\\\\\\/RJdw+ZNALPF2xlUDoZvshUwULkm\\\\\\\/UFn6MosVDxbA+a6I4M2SAJAnv4Cmx\\\\\\\/1wrkRO1\\\\\\\/4aZBBBVEibXFjLi230wmkJMMUHb7ABodjpa2AySR3jMGWGf3mL0t+P6lh835inKmD+alFRKWOcQeVgQ1jdHtsoLXLi19hsbqSsJUveze39QPlVfpIZfJYP8vLeXkGFgZAj00ykwm0LO3v0IKSd1N3sn6pr41yjA0m6E1CX6qFeEX6OanjWhuqs+Ols4eQvXizdyrZmx2PDy27Sq28sx\\\\\\\/MtmABMkT4oE8IJSYuoGYttOEWdNwiE5Mmqz8WKHEEly\\\\\\\/qtReAjLUn3478qq0Vd6U9Se7XvcDDZZeAA+EpHm\\\\\\\/Hoc0kp4ePflc4x2Rh8B8i6q8J\\\\\\\/VSGYAMf6nGBdaAg6rQjT34Nt0XqfipG2+rUYcxegwGcMiOBkZobnglIEHqdqkHbQ6Wbxp7e7dT8wEUTVtBJNZt3WVxdd7Y4nD0lTh6ltF+zzH3z2+L7uhFD6+HVjJcD3f8eZoWCzRLAdnEFmL6ZJOunNvut\\\\\\\/vI0ecRZai\\\\\\\/e0eMFuFywoQbA7YPZK5PmrGDZ7ms120EW\\\\\\\/XvnDJZ9wUZFVvvv4\\\\\\\/wMMWvrqzfVcYs9uKOCL2CT6ZYZVSHW+lc7thJ2gEVjG9BdodN31egLFa3kKnpvLiVNjq9c+9GaPhNMjSg7NbAmmRh7DWzh1NodGnxl6TDt8u5i\\\\\\\/tAkpBaNXnvG0Cp0ICqJ8lnOTeT+8Ap9tVy1JNAPoyakj\\\\\\\/vvu+IbM2J787skpKGS38D7Gw6GuRaWea4oDhv3P34QuN1j50ckPmQzeKp0JG6R0d6LIdmPPwIwdLP9H6b\\\\\\\/P8bWwASiNq8KFjfsO7eGfuaiBjgt5iLZQVnCHLezFwPqUMjOL9US72ypku1S8HYAHvGS3tcjhy0B38tYaeynRx+ftyCN3oHwxJBwEvACtTcb4vf66wqL5Lw\\\\\\\/gkh7rFlVEXrTbfgEopT77PA\\\\\\\/6S4g3SosVO0sUqLGqPRKDPE1VWv\\\\\\\/yrhJo1mepYazopWOTcR3+1WkzIdb6AyVVkyHWv4b7HG5\\\\\\\/LG73YiRkMWJCUR9UteoEks4411tzGSHOnTAKK44Q58T6RYvru+s4r8\\\\\\\/DBoe\\\\\\\/DG49Vmkh5K4g\\\\\\\/9NR798YC14Eh0WqBj7JDV+b4HV6KBGseMuagcnxJhLjy2caAkN0hkb82w2bgEUEz7C78XRply0RjID7AT\\\\\\\/eaU7IifZ3S7kP\\\\\\\/Gcg5PAAcRwPnAYWGjJe63PxWusseESTlqLj1Ize98yHBLkZGOUIydEl4Y6exFKztHE3uP6vzcacRqlR4pVf34Btc5rLZq+X9RHJMs\\\\\\\/r\\\\\\\/qy2JO\\\\\\\/mr9u3jBA9VuXRXusC1gefTthu\\\\\\\/LFpuFFBHrb74VMQDoZ0xg7rM\\\\\\\/oLznfRFUaE1Tlv+aLFpL7vZNupDD67CZCHAYKb7SwQyd\\\\\\\/+iDA7wvrZWhymMlOTomLMo9Wa+I+QnnZT0mwMN2Z3N7pqQzhyFfns+OBBvnlORm0HxT5SBPHXdIEDYTMWuGBlCkO\\\\\\\/GkMg88lDmXKIEy1xG0F5NSM8e9hxx8WqKyvub4jZSthYS4Nlp9kBVrcqMCbS5jcff13RB+DOr8mgsdkR2sphB6KWY4OuOtrVNU+XOsfRHajbLLEOG+oQFVk3jgE\\\\\\\/4WFdgwScJH0ESJnQb255QekOoogVt1uSK7mIseTD40kKhfXx2u7xgDUvM+GbtGxkSW3IcVPWLJD6M4PJYACKX7ZH0DZc0FjxIwFhi1WEWrkT3o\\\\\\\/zXhsrOw4OT0feq04XjIdtUAKmezHLwywfwpVjYXj5BRqO\\\\\\\/JKEslyPhtbQqlcgP0D0QsJF\\\\\\\/FgfWKzdf01TFwAt9wf1eTvCYb2kMR2BbYlKUiBa+HQ4jB+quDTTLbbKdrg7qmmfCZfRZkaxHHk8\\\\\\\/7ewWHFP65fKovjvdXP9uD3HwXVih9KkM9dGM2MiqnMOSjVQzf5MR+FaVuq2TUHi5VRsagKzyG+f1ekbBRC1W7EI5N5ZEuK0Fyrs3KYH4GFq5V30PUBOZTPRskox\\\\\\\/Fpe+ahn79pZYcqw\\\\\\\/v19PtGjQLPCBtMr2v4sEs\\\\\\\/Q3d\\\\\\\/dXjLWhIRBQEtDpFQqFZVU+3WqbgHoh\\\\\\\/VMVLxOdg\\\\\\\/n\\\\\\\/NKePtOz9CyMXIf\\\\\\\/RFz6Tm7hXtt96+FRY0PFvFhMud4kRPuufdjupVMZSsf+zYm5Fsl9twvJ6\\\\\\\/cnpuJLjOAbo+eD6qneu3Fow+dH5yT8q3p6ppXc7jJhFL65SF5GuKhdufZYn2JQeWDtNhz8Lfvc8RPjssJ2zFF5ZkcWir+H0qHUxBahM1r9IarKJOsYCFyCFlQuR6qhjzLCmbs\\\\\\\/zAj99N7OAFagH6qRnXR+P3T2r+C\\\\\\\/4sa6lJpthiNHdSG37o6YyBnewtGYKvB1syg9YEuQLL9uYTO9Q+MUd0cuEeIM8Amn1yVPZBV0XHHJN2nuIwuzvSb5Hq+eq5Z8q8nhFrudSnbWBZBj2yqQqlAGAkXqr\\\\\\\/1zms6RQeVWdDGDLGjzrhpuFqWjhHwgV+lUoNRZZXahMRNC4B9vlAKcdIi+SQG5udlJThJeqZBcSjqC6brfoON8WbiZLBUb\\\\\\\/Ew4Uk9dkVInLRKnWGK\\\\\\\/B314xj98hxy4A8eIo9xUSBRMT98yfYaI1cWGCUU5zJAHpU2kvQtfrS9ui\\\\\\\/S95O2FSAZkjHTAZY7deW6DOUJPMhFZwUxgu\\\\\\\/\\\\\\\/IqxYi1edKmyRXq7bQrHXirV7lYpNEEvPtO7Ttw9NY9stTvG7NRQCRY14rshYJnlLdYJiF\\\\\\\/UUk\\\\\\\/FsBrYKf+rUgLXD\\\\\\\/RlcohTmWL5EWT175JrILseDpAmjbRGhaFARp\\\\\\\/u93KLnyIvBETQUXlMYc+i2vjfDHvPhEha8fhUEvjqQtn7HmoetJ78tBMbHo3rgPCEF5xTI3u1XhLYUmxa90hqD42Om9K7sYTxCNDQbDblSYulcghGk9QT+atWWD0ajLY1uGeETunWpPLkdAVbmM8nZJLPjuCqF78qtI3XCysmPaO\\\\\\\/ELxD3wuZYAa0BDEHMkfk23Y7GFJFECB4E7F+Fsyiv2fhF2a5SIpa4JCZDSf7XpV\\\\\\\/R+0TQ6Wi3kCLR+HHzo9mfZJz0ntSTStr3+ze\\\\\\\/qJQIriUJ\\\\\\\/PEuOaIrxei+KPTCP0h6zLqFc8Y97XubyoEq3kRReoBGrwLbtF\\\\\\\/VQuzigWyI\\\\\\\/4q1lv0sviz4RB0ZrvlX3Ds9ga4Ys1PdtPhBvFDIJWhQwMkpfHbbEn0rasIaVA87Ywhw1GVdzbKCxost4D6E8J8Z72lpazSKp\\\\\\\/\\\\\\\/ae3MrE2otoFUjBxpQZLYsVr8VNXrlJ0YIEE5n5eToCoqYb9tQzSFShdTQ44QNMc6TpJHuYDjfZOISlcn5xtDRELfY4MkJ29xqCH\\\\\\\/fChpKWJw3COXJ\\\\\\\/XEZkcilnURVYiK+tQu6oV59dITe4a2bOGk3TB\\\\\\\/PdRLA9qU55c0fpuLw7y+uRBgWMviUGJSZA3Lp5gYGWCk7fft4a1frqknEw0PVfMVihHGM8NwhekFA1O0RaSkKSbZ0Lb594ScmG+olcN41UiCg89tx2wHDsD8UkBmHDUK1k\\\\\\\/5445xlffm0TG8s1V\\\\\\\/shdEBrDDaUGoXdkEKmMzXfnocCqzG6L+oCz2\\\\\\\/r5dAwLRK8L3EcYMvt2+vZiIVORsTLVptNtfPbDj36NIX\\\\\\\/qP7FOjbKCsvcXyUyrbEFakdy2QHDfdOEWLafPdhZhmFwNn9Z3QY9GxdNEQm0dLe7iw+2gGKRQbYIEP23SZTL+rZarCGGEriYJFbIBAY60N8XfI50gMGizqLObSAzI6C\\\\\\\/91kYT6uSlv\\\\\\\/1wPAWMdTZ62E6XyAqifmbhxYfu\\\\\\\/nRaK8wk692nQ8vTzQDbQF9+aLTSIiT9lZRXNEs2UMO9CfcQD70V6gcY9wCmUKnu6Y0EOhNrw7GpDeGT9zXTq\\\\\\\/2zQsd1VU2\\\\\\\/K4CbmfEQ8UUYheStJR8C6lyA5G9oSRmLWFiXW8DXjbSyEC8xusFsTjpsmXb\\\\\\\/UYt8UvyC38gwNe7I7tQn9xlcRROvBF8z8EzhfKSq65UoJzIDbJuCVEtFw8oIciNcIyDsNz+OFTIM+ZTFfuuHZKN\\\\\\\/CMqfQGRvTl2xchoyCoI8NyEoZ3x\\\\\\\/tTElybGYD9udK9GQZHXA3qOJ4yi3CRKbKcQOGSbkSihO0vNx7Tp4nvY7K7lOptoqb0XUnYL8TbZUKuOoBqD4PdP6koOFRjTpvZb9gjFv6w1gR\\\\\\\/QygFkBOuSwLiZr\\\\\\\/HgUijZbkvEdA06\\\\\\\/jsy0OCEBLGMCFwNLMdZHQj9eBxqf+v5mv0oiUQPkIHSbmQuz2L+nVC++Zwiwbc5\\\\\\\/i7ewFz6SQjXvSqAzeriteRQlWyiuhZpNaWZdy982MI5DJEFINIk7wpd40VPrR9qHUxau7IJrRr8rkgn56Lg3L8\\\\\\\/WPaW5LM1k90s04lIUcKtS4f2LbdqWS5zN\\\\\\\/P+n1t2Hffls+JPh79hWWjaILqitHbcet54GSxL\\\\\\\/uX4JFGPBQet9\\\\\\\/eV51\\\\\\\/b9ayb8mTxfRCM2fW8rlJAZlDRIzUvKxN\\\\\\\/8Ddrb4nBk8iizM5hukey9P0rPcGpxgzqM1hjMmMCs0Am9SO6A\\\\\\\/PW84LgkpT7pncJTDg0KLvwUKfdgioGbWP8e\\\\\\\/cYpHBqtO1iM+WEncemG3etgpygn61FZ3MaOjQNiGAkPzzdPhZ0Jj4chPrwXOKfaPtDmh+oEUYWkGQO1+XGDm\\\\\\\/BfiQDPgovCQcvOhh8AeqGGj7HWbeaK9moES\\\\\\\/\\\\\\\/Ft8GAaw\\\\\\\/MwK1jDMhi+KIC+PTHL6yZW1kL31GWFKYAlba4kphCe1YUwz0oJFILmOFW3aBA\\\\\\\/0wZgsKEDNCur\\\\\\\/NqXvkcRoeiFyAygbpeG0oJY21yCceVlheUSH1U24\\\\\\\/7PKHwwBsK3Vhj\\\\\\\/EJMrBFU+7Kl0+LvhiCgAawg4hJd\\\\\\\/+JfoIJ4NkRHzDfWcoPa2ivkqcAZ1u3dJy7PCgH07NP32t5agIizYX6yh0ysFndU3vsQJ087uEozr32PfCL\\\\\\\/Ul1GeISBDJs\\\\\\\/nIhWvq3kMpq\\\\\\\/Pgs1MhbdU7+7zUg9IA08UJglpAywWPiUgT0KOWY\\\\\\\/SVU+Yv2Xlz2xEcXHKV9emvqz10pKhM5\\\\\\\/d5Ts+prvNG\\\\\\\/2WacyHXiu0UggZXjrGf\\\\\\\/qGzqLzB3D7bki4N0toAXgnkGegamAKt2qv1b9bLfK+ZUHIT+dz+V9Hv\\\\\\\/nA1UCngi+cElaUF+QCFAh9qMwldL\\\\\\\/22sk9LFWpCC2uNel85iojbzY04TI4F9jn6Qn4Dzswe\\\\\\\/bKjzBm7PTS9P8\\\\\\\/BSUcKFruLrUiIq4+PFiyMQ8E\\\\\\\/gTaLDjV4qyTmUvKEVk8OruaxXdTKhwehpgBS++qKWxSQvXBlCoe0\\\\\\\/qV9sPF1f8+Ukr90zLa73spRxERHGwK++3LMKSjZoJ9IDJ0Sx8ewDxhfSFbA6HGvRaY08KPgj3c+UWUlF8LqVnNsVaIjpcimI4k1EechK+RQw76gFynmLu+SWXFKpRhkdpaZoqJY9KvHZPRZX7Uwf2GsHUgS2RJqh1oG2zAGSoxkkWF0LWI5fpdwc\\\\\\\/qRh4nKHgKMD2G6DCXB2QOayoUFUKTKCLi5WsDTd7l0a\\\\\\\/cZClBSTmLwRHnKba+V0jfo9NKEb+bqZ2APjnv9dVGladhMYB4G249U5\\\\\\\/44e4OAaL43RfqQ\\\\\\\/u1rvglP504ClZ2xIF784a36DmRhHcYoRUmp0rbn\\\\\\\/Ps8tS1Yrb04Mgl3rtdvYUfZBQV8ar5ttDa0WBLVj4mVXUW8jAcJfSWZmI038FMbevzrHrfr79rIT9oFQJ+mkTcxL0fEEaGd9NdD55BzkK6v9CKiQo\\\\\\\/0fKV08Rz6qmw+VrlmjTEwl42LYL72WS0WMLoQQOPUU\\\\\\\/9s2JhAqDI2bSwoTTpINebEE8tr2Zl68vWoxRIY7bawezJ8kWtJ7qoFxJHZa1325hfekwfuA\\\\\\\/Zq3TrhsWOpVCI+xixnrZh8KUNT\\\\\\\/vl+RvxAaoZh3CGeQxks\\\\\\\/ZuA3p3Z20CBYnjDhGrunxZY7f2o00vf+FfKJu9sL2iH2tDB1ulaeB3+GGrdzxvwKRZ3vUOa0i6zU2tLlxcLLICDpDk6rQlGA2VQDtkmbxF\\\\\\\/FUIxvmYK9ao205Pa8oZqw6hyQywmyhP5mwNge9Mc0NeddCsvWGlZkDmVsTTkBaS8+vvi+8dDFgpT2VUxQZ660f071jPN0pfsgRIDfNYJLJLF8SDPHCqI0FNDH5cbBtHQukdkddHv5Ypl8GKPZJ3bd5bfwPw62y42GoxAdoAT8+QIzjqteCKCuu9zLPj4aD1CmZbcX+6t3KClvi4EWq\\\\\\\/KiZf25WV1Y49E+F4vAQ88ufmnDA+jcMRdopAdIKhIJBRLI2yby0SmBw9JL\\\\\\\/ee1X7\\\\\\\/D8dxDfFaqqDnRNDtJ6BstUiCuBuHZsHrTnglgCJD1QlW2B\\\\\\\/SpTGAZ7JgVWrt55V03veeE\\\\\\\/80\\\\\\\/p1K6aPMQE5UjJ1ZEL8Hvs5S4tNY0OwB6\\\\\\\/Zleyvl\\\\\\\/ds6Qdv6zZEWEzot3LN1DpPVgoTDWxJ8kECt3D8TR3GoxfGVAKgE2oBe0BwhRNqkvsp5+6rCE5+68ZwBG5w8IlGDrBiMU8N9xxYChjA2\\\\\\\/+as1zAZDYXOj4Ia6LqFMYH0XKkBGstpExrLlwJ\\\\\\\/pAqXP1x9tLm0xWD074Lv8KRQQWdiu68jYJMRSjprgjVXPWyK8\\\\\\\/gu+99JjVaw4U2UCuIcZS2LjIfwftJX8CCT686ODwNBGslqGNk57yLjvV1i344yYmCU7sl\\\\\\\/ICqborxnkpvxu2fVxRquQ9C5US2gYSqZhlhROG1kZV0bhcwAVdfoUkvPVeEKHfQU1jjlGhYeveQb60MOOloUzMkHdOjPsk47\\\\\\\/MAppNqPWBo\\\\\\\/zzHdswxPNA1FciyVjjQOf9wQYAqGYdiubSGNujUUDaOFVWOz8F4iIXzxDZfSJsEQQBSQhhmgCYS+sFhFTnMMJc1MCxfeG4sRIMGYV7RbVxUJVCThe07hsP4ksA6jl+WQv3zAQmf0HfUROzXvjs1aFG6+gA0cdX7TsmAugHbx8RMjLULebmeq1npQISME3GcZLUIUtUKE+HeBstKZi6U8odBxGWDHW6budTk1h9BrJxO9pC83Pj2enREkFaYppGewMee9JFwf5970q3EWoYON1veD4BwnNTSC6q8N4ripub49EaF4\\\\\\\/FEfw7\\\\\\\/3dHqTWYdnjF0EZlJoVO9MfmHDYdaUY5WyKl5pQA3ef\\\\\\\/HexPSfbU45mlxa+Zmvat3LM0nzJrN0rgqM4cdO8t4RqeyXM9Ty2S1VUqlwXtZHrckBC\\\\\\\/z5G6ELX+IbydvadSQZrS68va6\\\\\\\/R0dP1XfTjjmcFbmnMQMw+M1YRSIvxMyocVdUN4wnh+VDNlvgy9rEoIeD\\\\\\\/5zMRj1Uz4LAX\\\\\\\/ujOZL+gXQucmHY2IGR2\\\\\\\/wlzYw8znjxPX2K53Yha0LVVwahX9AbDlatpeazcbQN5COwjR74uuJigmnQXTfXxqemnmHQ5jhWStObhlB2z79U7+pxa8ITuNSi6uFje1eSgKUWBIQIxa8Kon1P4zRkxZALgM89rka+IxR9873MsTpHvXCEetCZvRddzaleD9nXHg6mFArRP5z7Ec1Kzpgw1XPVh\\\\\\\/G8QK10HD1wpHNDmp8uRBAqqWJL+pkB8Au78Z9cAI+095KrA1c1JGqZtSJ61i+DURDBR6yUzdGzMpsZK9jkNLmirnHOmMdMSgb5RDRnnsqxm5JiDqkXY+NAeSHitUmPFM9KApnyRcwloLFLbf1vz\\\\\\\/oztjYeOgjtQTZkuMAIb1xjHZdX7P3eAiFXIX5bArTxkowG2wJxxZ8A08mEXJ2JEetDt5CiUGRPNK20W\\\\\\\/yjRMOoUJzIVUBY4XK4sGT\\\\\\\/zZlttbgMqMxDsudWmYCrG9qB4wgCQbNDjfKtsNxkBIwyJAxeK2XCwfEqXpi92ZV+4WnEznX4lrR9rvPgSo5Zigq6c7NsUTIOocR4fQZAyfGSWRX2CDc7obPOdBqfLA+kXoFWpIQd9vDSg1z3kZHnFr+n0UrumNHBughWApn0xPzxrSOTcISLBXmG5OtcpMunlMgKGXilJmSqK+h49RGcloxzsyH9DwfpPW1AXiAs527FULOdzBB0UwiC6QBp8+tSDl17kWnbZHdVAlQCTY4ogaRVOloaWlcpyrJAFxAho9GyjOwzFG9CB3cII9i\\\\\\\/naCDTyQQ7tssd6UTn3t3Chv\\\\\\\/\\\\\\\/d7qLimHcwy533ek16hqnTJVwshyai7wKaIFTz02vaZtBkSZVw80x0yYgJ3OwXH7UclU2iuC5z5I5oOn11ytqT8ElcfjwbQaCdIAplQb2fJ3XQKxBQSe5bdV+2GeDotZE92oUtcXFzrj1OIPAZO4YLu0h0UNcOhrz9KuuAXOgDoGj7PHRtmVztD4k7dJgNMSZgCGueAZ50255dBE82gSFGN3fThd0m1i+nJoC\\\\\\\/nh+P4B6ru8Olm3jsJ+hqjrGIRpAQnAg7fB+S2FPOHYzE11uFfsVEPQHQjdBswW+JvL9E6xLsdEEQtKjtzbHc\\\\\\\/I5F9msN52phWaDN7sUaBouQckbQGL5CKWkEle0jk5SEdz\\\\\\\/8d3zB+1I7tRiyH+mnDNTKQfxRMokz1MOR8FGpljjnvN7wLkU2XStVYHa\\\\\\\/Z6dzEnOMf9AY1\\\\\\\/uAeo4WpowbMAqRoi4qlwhHbbLG6qgbdBR2vqRsdgii5zMif1oVeGo4v1F7h7jfkj2skN0NDnAidabdrr+\\\\\\\/IeaDwjRPfm5olkuUU+g\\\\\\\/XceTe6w59mk5lIdzHEFUHRwHpsYWYAh6CcxuBOo32WSjYaClx4Pzmta0DPgolOMAURIMV+04zN2YbHrSrpEpCAS\\\\\\\/NyXUiZPBFeMFl6Fqhl3ViamSRr1NWpUEbyap9\\\\\\\/1g4\\\\\\\/0FJSJl6\\\\\\\/hQxmNk8vO7eEocvfTkT9Ct79HHhtkfJs0jUSgcTszhgz0e1M6lv6cpEfQ+Lczm\\\\\\\/gKzrWnI2NQVPMRCRURfN9EWYRhU2c5X+6T3DpHvwoCruR66wEIfhK\\\\\\\/Fxq94CPvb5WjlTlbLOU0kflEyLAVvHNR\\\\\\\/PUznr1WJFwL921W1qeUNmKdWihFxaxkSdOKRDtsCcjHonhe2+EYF7CZu5ogNoJBxxnYVpo6GfXlBJ\\\\\\\/uj4\\\\\\\/h1hPY6EWBNxYd\\\\\\\/4AYoq9CbGicVcxhkdNQh67ODG01+D7vA9f30MJnjzrexTqzPjjmZ9hNKdP\\\\\\\/U8KVi3NMlab1ckiWAABIwssa8TdXRCPmlfCai2CCEwxZsE8s42RMY\\\\\\\/XsS765ZGec6AbEkM1A4+l7VxNULeEkPc9e1kpvX697O0E1mC6TkZgwY8yacUeS5IV\\\\\\\/NWy2UDDbkQeUYGSXLI\\\\\\\/v+8Ig0V0y3JlH9Antw27tIqw3n2WjMAJK8BJm4sWaZy\\\\\\\/1I7YZ0XILhK64dfwfCZWXwuHcIRNmhExuDbXQhg02WwG3NUsnis9z4rx9U3gqtQem1StSriRSPAyyg7tV9if0mGMjKWbJDn3QJtKUpe7T\\\\\\\/tjO+yJMjtNILTkDVMoe+fMCrzIDkyj0zkthIJNu6qm1j7fyZ98ipo1vFHblXBdYbRO0r3b28yCM\\\\\\\/QYOBSUyca5zlEqLQO4TrbIXC3MNzee\\\\\\\/EqAyby5kvKSjcDPfK2r1gTy5LIowZb9n3P+IurUWOlkANh8jYFAhDtEaWiiMOc8zDnhe7UA6gyTM0IQaGIKMGgqRgMVBl1R8x2SSAhtE+4HRG\\\\\\\/sw2H75gKYZ3oRVZimDSl4vwNTRBvU82sN30IPogzafKT+IU8EB\\\\\\\/vlRlDZD1V2DQ2nLUFp\\\\\\\/9lM8qrJVMfJDLmZ8fDCUP\\\\\\\/2M8+CyNjEqT\\\\\\\/MZoOc5TNhHWfqcev6yL57QLGWLQw5c9S7FbVZec0u2wywkUl25SwFPrU4mwPP8lXfEq4KF+PBCyXM5Jw1ajCLv5J\\\\\\\/A1TbGpYsSKlANknY9lVar3MsTj+n15+yhdrBBdRynNaKEEMgtxGLn4z0wK0COTm1RpukW5\\\\\\\/+7Hzvnj70eJUI4Jow0E\\\\\\\/eWjSsVF+2WheRSzIuoYXIskQlWrbgZlCWdqCLHl8HxTtadv5dqE60XrWhn8Cx5n4nwDBFKD7Pg8U3IoO7Z2e0zVdRtYHFCrEXPvtPtBp7xbLwYIrDWjPtTC7iSDc9KTlNSGesfw\\\\\\\/tRpvOAvzS+NHr8NgV3T5LwSTWLYqZRVR+wwBKLgn07tAxy8AzSH1ZzALJnIcB2Vz+AzUiUgJcFoRXv7kDr1QEhkM57l56CBVW3qYRqh2mkvm4b3C3rKjELIm8ziu66b3yLHlTVyofY8pvw2OP1abzddDFPSMx3s7rp9UBJac7UZPd89jOK2O478LhSjGtui4m02wqxx3qhE9h8GErO8n22a2x9HKp2zgxMNJPCv75w8NEnqDxnAd4a\\\\\\\/q6E4jUJcyDJkpN+M49IfAP7TjexYQLqz6yZs9lqb8o+KS96OxoI5LFXsPZk+NIYmpY5W5oFmRXJ7l9oql4jRa0p++1E+LCm+cVEJGJXuVVKno2e8Z+1haouEYVG4j\\\\\\\/5Vi1N8oV0NqDSnp2+O6c1R7VbmvhadHoSeaZI7d9hE9Psew\\\\\\\/UKHHQvWV1LuXk1DyzW8eVStusl4+5v9bZ62BWtfwEDVrf9uvToL2vyepqDlZdqF4vKODhzfregrDgE+LOhcbJoO5R0gu12sGY8SOk+OCYUFvcc4Dxzp4uu15iu500EwH6SgeT3+eku5FulNeKX5yNvXoRu\\\\\\\/1rlSihDNA1LvvcvC5XVbQZo6xnLWPE1IPSMQKjNvYpopHGeXkeeh0YDdzRWniQS5Am2yQUrOZu00Nw3SX660l2qXaWvAgM9I5qAuxNacJJFAKhoCUjkWnjzWyhKGdPD7oyT+GinAsMoHTVmde+zOFafP6oVkWl+XEDjrFomPa0JivJTnF3Bep0ti7gwTvr\\\\\\\/ySOazHvLniIBErgBfxy7zl8WDzDLMidthKknpyEG37n\\\\\\\/6Qzwh6PocUAn8RQWeBQrjTSaR146RaQk0Mxyy5Mfkey3z6DE6rQ9fGrlj1gZpv2o2if1Oq3omNoeMVMe9JlWw1NzsEUCYVrdW9zfru+pScLDaKG9d1wIHulFWHLHGYvG7pnUUTsYIpLcATT6\\\\\\\/GIBQ+TrukpZ\\\\\\\/I5Oza9MK358d2LBrROu3Fr53xKbmgvDLGxJj9VnSHEPeLw\\\\\\\/NaF8vugCtkF48SL790lvDDnLPd90xci0kj+Xfhk1Cb8tI1oHDeaDqG6fuFlT0+a1YJXnsktx5xnffV\\\\\\\/mfzh\\\\\\\/Lah7RCg2+xtodwoRzuYHbVQtUTx4cDh8z3QUHN4rxh\\\\\\\/6l9KkzgqiayI1owiBFlrnSrwakMjFnfgoUzvNvA\\\\\\\/kq7OCGzst4o\\\\\\\/3gmWhGLLsnyZ\\\\\\\/uHs05YIAx+eE++tMslAfphsVJTLbZhkGr5bLNNqqD4Icq4jeLpwvPI86q4bloUrzRF5O06KLNwrtAb7j\\\\\\\/RZQivV+a\\\\\\\/SKEBpLXTJp5JNtaiqGBU6ZXpdk6f\\\\\\\/xQ3CNm\\\\\\\/vE8pjJFQ3hALDcVnjAh0U7kWe1gEU\\\\\\\/QxNGIucTYKCTtNuoQR9yDXS9WcGxPmAn+UBg9d3uVDmy0JzNXGGo1+04EEBdr2KIw9Dl867um2M+Hqc4eOKYVU7kE7wRIM7ovaLnMFOhV3G6GvXWrnOG1iRLMnUObBbAKDW53fon+r\\\\\\\/D0Wln63l5bfpvLpNzeJjU9oxFsRD0Ev3PqPODLfe96pXl7J2m9BPwf+Oz9b1Y7tyTAHn\\\\\\\/8kgATSN4u4bRGOYTiIqkcnrKkP\\\\\\\/2GOJwgoml0OB0Jh0YnFZHA0aMbHw19y+OlYkttneUG1crgEm17HxI4jz+uZ0Tw9o5wuo\\\\\\\/H5+lQZk6nGXiMTgRssTnlrYg3i594B5ADSDdfUFmcSPbWYJ0vGxi0zeCCtOxGP7Lp3h09pzWp9oRev071plaMn9NI9jwqHm5hh5Dswy1mG1nd\\\\\\\/w9BTp8DWt0Kh2AsjWfaGJmiCLiQcA65wz9qkoic7sDLBf3q2mGFYOJ0fdJqo3Y0yju\\\\\\\/JG3BLAyovTeRVfjs840SiDaWOVhZpHkKFfkObTzXoIGb2gCg8RnbgaazWz9Kg8kHnySkbaATfOJFvEJU3BEZ5DR4ApJguJiTST8W\\\\\\\/eIgTK15n2OE6Keo1dE\\\\\\\/8ZaV+uEZVRkcSb0dkgyT9eDHJBQQYTiUTXz9HnvpU\\\\\\\/UTj4cQYbIxATIXjT93xLypotDqRxyBtqGsnTlRoz5CmmU6H+2yB4nkgQ1uzJbGNRYU4zLOIfIfVnCW4oKn+WROsRIed6u8ZEmuHsbIJmlbs52pMXTncTTIQDDB+PkjPiS7egbWWrYsZayryqVjtgmVIZwziaRSQ380xXp7GDAL3zzkuOluCnrDoTJ1RvKIQYPxFtwm4ncxMuBrvFSA8vpcxKeqMGgrWOgoCxyMvsZH4O0eNLs71xdkdgxaqt22fgtn4s95tlbDwgQWxMtC3049Xb52iInKH8HFKIGMQh3kXmK6jVc0DP0DoynAcGNz22fx3mEwMMJSJSLJs24j4fEnUDoAmCpaztyNI0f7Cf90EdzzP8odMRTOqVJjo8fTIwQU1mVvD1vOu3x\\\\\\\/loP8NZG6qDcKrAe89s2ZGUMnlN9XCYoP2QEgwbBLrezdAXFwYBzHPOfqwNKZ4rrKJoETOVNGR17UANPuXPVyTJf3fUV0FR5NDdi0LgCM5OFFlvAaTq7EwVp14CBZSeBqhxY7YAEQC4SCE7+y3IGLMAcInGyeCr4JSFOwimnp8\\\\\\\/FNxszQBYnZC03av+iPWFA81e5+3xEN9zfJoglfRVDMXusu1NKi1D73LQQTPFM\\\\\\\/FfOtjQrxK\\\\\\\/05qxuohZUxrOptsXGorP9Y0UPim55kp86PSrJwbcOG0AnS4kKWBnnwRg7hKHYFQMUdhzTvKY+JHBZ4ESJ\\\\\\\/AF0lZ\\\\\\\/l0AyP+TaXk9BUp9yBSifAiBH8u23OTo8ZQKkBsWWXIDUvKbZdSnDHAMTIvIqPck4tZ3MDHKVx3nNyVhDgmOWNChNddPdS6w9QafRLihm0qgbFOI72qXLhaqNq\\\\\\\/uXfDAIm\\\\\\\/+7qdHsT2tzBjkjUWbNYmv0wB15Lp8tqPmBZ+G7kqq\\\\\\\/7j9NVJo3rhfqZWVwsvLs3ToTTvXPlxMRfv9kt\\\\\\\/VX+3PdM73hbBB6koHL\\\\\\\/EDw1EsNHWzC4084XoYTh73QEYLGcLezIqP9lanf4U7zzhdGclzft3BRQL6zkiSvL91FUQaL0JWKHjRibQ7uResOrVrGIhoNMVgRxuKD48TLA9yXgIgaadSGRUEW5cnZky3j0Ya3p4Ip3UGYltZaYKDaWkVg9SzZ9qnvZaEhUpbR0zB7gzTFmwBg7aJWkkUlFqevbE3UgnzDX4KULW4gdBZ9jzDO1I60mzoPCD+zz6kQfiN7UVxQ77ph+zNqtTBe2KO8MgQdDwkfdI4SzcIrFGN4KPC4WsV\\\\\\\/qOpZkXdtMbuPmvCz7\\\\\\\/LIPlp1hzkyZ1ESMLgrzSqDsldlmJfKoJWulHn8FDkaiEoOChu0usnjrAa2g6A\\\\\\\/uncAau0AsMsOXP5cgkyXYHDDq3yronvpJYrbFbygf5Ce7p88\\\\\\\/2lGAfFJS+X9glcE5QT1nK0\\\\\\\/OUZoiPmXGCcBKQv0r5OKET0ID0LIIzjsv69tZUKPeTBxbvIWrR\\\\\\\/xy41wYNm+ncFB9yR8vL\\\\\\\/KNECbabGYzUuOBaFg20\\\\\\\/J94MdZxYofydowCCYBXy6vaOnIiL3XX+HEk9y1zwNbGq1VLtRu9Dd\\\\\\\/UWsiEj5g12jIFOejU3u6U598gFZOz+jDiaFMY9xXOndI6u0KjykLGxeAwun0WK7WRr9eNjXL59b91otl2Hk\\\\\\\/8dach6Q0FXWjTWXpan93D2AfoPRoXwuinN\\\\\\\/x4Rm6k1adVfJx25km9rS8zFUirr5DGtJLwDtc5ALz5VNfNYVAVvvuMxVU\\\\\\\/x11GC25FFO4JTeGYLTKpZ+UpWmJ5IXqpDOypJ3u56fMA7pf6LEOcCTT45USKBTTfHneND6YYFVBglbi1DK7qCqlwGhezSDxTRC4uFXfQkZNDdRDAC7DUOb53QushJMRuGIeZHnwQoO9iQ9k2N6n92j3II4BGGZUY0NJFwuKQjnm94OmbCZ7Qvx70Gl+qfAVfWihAqauBie7PMcQcwpzCs6ASnRdM5ji75iqDVwDMJeu5AhYyRPR07aVYmOKEubpr2cHolMaET1emvhWeVVnvgp9ZJELLg+fMEPCmRB9DHFAEa6nxjRDrflgZhpM0gDNbMtsgCug2IrT5037ggaftrStPuiTa9EtPb5UrycekBJDzzJrbLZNbqj1Q7eEROkxqOFkCBX2FPJFlAzbLp5XmOchjboYD3GJjeww1HEyIqJh1z6khPR0iMnk5dKa1ttv4cLhF6dKTEoE5\\\\\\\/TpER59hb0tzjSq42+mbOyiQ5k1i\\\\\\\/OxCyy25AXLdjAn9DGzFVgjFHPbkTzneGmgYKeU4IHL+NIojD3T1kfzho0RSHUC+7k3N3gybsOqaxBkt6zHO\\\\\\\/m8Hkm7Tg71H0jEwImoh3tzj+tu0dELaTb3abt6yV5FEU8eZB26X9QW3+5wN90sszFVAGKlm175HwdQS3rXIcz+yrTbv2Z2RXN\\\\\\\/vXXRpDMY4Tj797VPMWBSIjAq7DuWsytz15P2yaKCWZdZ18LryYUvDYZGyQAfkjhczmg9ND7bcYbKyOulWbBbJCUy+t9Z1gQqzwBGHcU4on4\\\\\\\/c7aMDflWawpNCIG8tMxer2mD4ERJmeuvnrzS8QCCBmah18SQP+bGAeKamhmn8emX0mV+TeD9IcBb5H6isD7\\\\\\\/Em3lsyf2OS7fChI6yTLrQlfl0KuIu40iHWZOfJiF39wScwIW990GrqpOxOHN\\\\\\\/nvzDdz6uMnvmf6y8VGKS50RONkDQoSUkvkR+OTqAS4nNYwKl5r8mUJDLXB\\\\\\\/U\\\\\\\/XIWtieG2k5AIH98Uwnxxra1ZK3BcyeeNSJS\\\\\\\/3km5t5OU2rVYzD6VgrwgnacJNmj89tKXUUejjOSNum41lnoeoB9OtftgKRXNgAS2gs370om9RJVYgCc+kAIR713WKsVW5zcF1A9cs5UTlm\\\\\\\/4IimzoER7SeCsqOWgpY+cbMwpPGOt\\\\\\\/cEgT70+lDLAKr77VJG4d3ng9rrharkikNVXkJgYCOntSf4O1yuml6lmuIAAPOSu10UjFSiOgNTA9eMnNYqxajII1LIM3P4FSMUkI0zJVwh3f7gTkpbnx9sdtswWTJpemEFuSqTgTOjyGdlO95o9DEhVH0H2h20dZis8QvSdIi5a1+z\\\\\\\/gt6JOF63vxbLzbE06r\\\\\\\/Hp6qXnN7I+QXVtzNh7kqtdCE0ts8kbZLKmXR8K0EcXuRFXkLdWYMRswdwXiHHVD+ZwvxYaXSr2OlK5p+++FI4LOfbXEK5+82UytWrO+bmbOwVXIcY\\\\\\\/LTQuvRhaIysKvJt7eIAqmO5OBTHWbZS7FZi\\\\\\\/obIV5UtXZvzhM103bElX16NkQI3kvRIK0Dkr\\\\\\\/FnIUfKO4jb9ZI4GW2Y+nykJNB5makwzcFUPLc\\\\\\\/bKV96FSEggwaTu7WwikV65nzV6TNUa5tGiinVvaQReWNLJSDg3npBxwjF4gud\\\\\\\/NKJQ2h7QgoLcsMdqEY73gKHHjrpqh+4uDOnfO2YDsXfHAZnDDX\\\\\\\/G1mewRQUq32zPdbQJJUwcSfVk5mRCyDT0N3DVfNENE3GFllblKbUvh1laHiRSr6ReDUf90Pu7yU5fMekh+MkLyFQg4FDfyGFuPKIaQ0jr1C0TH\\\\\\\/dDY0nHU+5\\\\\\\/xwPio7HCCsUR4qvwpFSBPCzbiSonUiFjvm2B5ccyxYtcfe9Y+aq5c4l6\\\\\\\/iCRmj7tywkWGubNW68hPkBlZk9pFXhd8XXtOQilVEQr7bTbZIbZnedmxD1+hv\\\\\\\/eX3kuseBIXEx4VotWLfPIwAQ89e4ACc0Qx1meREj1yu7R+NyJMCPQXzPI6zDS4TVep\\\\\\\/o9AZvBKf9Wn9FiIrDC3CgwAsd+ie9tbBcuVAE+Wj6YL6VFeQpZE2TQ+n85HD7+najB24eio1lshDgrfgAjV8+QXnozoXlXaZKKrQSprLa9FxX8JhbGcUrnxunUznfQVvdKKD1KgQLu6B3xtzVrGj499XLjlXyUSt4VZYkNhSdE5NR4ElnO0l1mqYVVEmCr\\\\\\\/1vx3pCRj67RQOJfCg9+EMlwIU37r1QgRFHdNxfGlKKUeibBNyu7F5lpiKnw8Y3fX\\\\\\\/k8h\\\\\\\/fJfk8dZSK4YVgvhDf5f3ptzsUS3pnpeTKV0AAAQcl\\\\\\\/21HAWg1gY4mncuqvGzbznSBsh1Nmi3UGB+yuFWzA2HZ8y5DT3XVb2x8I75MC38SIQ7EK7mQWWjx3XmyLMKwCx3EpIcwOCz8ON5A9XKiyKRLeNlYHcF2PN2pCnMDvBnVh+FJTCBsTYuM3XAlq71YLgw9JPHpPGCIy1E5QXZ829h7nGOzTVIe+x8do47QsG7L8bxzSExxoFW1R9ID\\\\\\\/pABdF5isLzUYLdCelW+u8065T0Germo0+YsmZAz5+mlkJfwmPJPFcxzjjRHp6fzPnTEoofZpMoYU3kKC8ffq8oI7pkOQ0HvXOpnWHxL0HlREZBnoP50Qks\\\\\\\/Bl3JPAwA2IgMzLL3OQUBPyo5BkJ4J5x0Ln4hsUSUlyJDgsqDjZy+OtmtmuYaK3MeczogDLCEz84q7MHTJPKv\\\\\\\/UrXE1SHTj7dM1pHn1s83Gz2q5dDuxi6UKf+tziDPAbvAS2YusMEGZ+2yZ\\\\\\\/Wb2HxzFisOO6o\\\\\\\/6n9qJr27heU1xoy4TB624c3AFcezHv2s5+QYvyH73U9SzoY+z6PS0+EKeuxQy2ECzKtw4G5K7QUZRGuWspNMPAb\\\\\\\/PgZWbHVLyyXhkxJMmcfmh\\\\\\\/fOiDWaiOmJUdEBh5d+H7uUAn83XOdsAOXUsLQrYvEjIYBmIcoS8r8TT0W8\\\\\\\/G624RW6n2U1nMsoePQz+BF4Y9M1WbSKJ3Xj72u1Ac0s8gGfhoVxciDUaJ8fwM7yIyf9uDmVPyRfVvzYFoKE6FJhJYeFrVv3HvMJ1WPNJaiWPxXUCupCXUgIzGkLH+ohMydYFZKA+oV7th5tkBzxEdffjNyLmoY7ktSrlQfcKRHYdk93K+slF2ucRdHWS2wPFwWWrBEPHUuQSSyESsLYADhRQ6mDuFYD6FIyn8DyIMxKH6Bu3NaoX9oRr7aO5oje1vvmvV8j0V92j2PLS\\\\\\\/ryj1UGR0K6oGJFXmBofeiZRfIbnBe\\\\\\\/QesINDHtSb9Mzxdnj+ZdN\\\\\\\/7H1ahndJkk92Z92p\\\\\\\/Ylcgd6rg9t69hLpvms3yWqi82SewccpRPlSYF\\\\\\\/fX8IICady5BTfQ\\\\\\\/ktrKxsNw82WmMeU8T5TB9y5Ztmr5gTYshi9f2q2H05+rbtO0yJHr6ZYRpZL2FTI44MmQMg+aMZcExF4yMEkkrCP9+rOSZdnzNtkggLRW8Nox1TQVpo8aEj1Rmn8AoX\\\\\\\/LlPKkZGs93QQYm29GxWeSnLywOC9p0K802S4R2IdKo7oAbmcY6KQHzVkXTi8j3VhQQdb7FXa28NnI+rT7bdVDdPUM9NzzYmMOReFOI1DbGqN063qta4SMSF2J0LY3t8afsTxKkx6GqdebjT8amnIgmfOeGx8e\\\\\\\/0S9vQI6nePKRP6yIN\\\\\\\/HIXD48AmaUtEsC1F3bjDBGR+\\\\\\\/\\\\\\\/DAfu\\\\\\\/8U7\\\\\\\/jyQ6ewJXWXvU4gy2lampcbaQ17GYuUFoqMXet7PFHwAsOeGq9449r8dGDsTraA\\\\\\\/2YGrBg6PecwxLhgnOeHISIdAdenW38w72jjFfLhzMzYgrUo1oD750w1afEZ0lH+D9hzfITbifibWs2LKJNsxb3qO6RgF5a2jhUAfIRHY1hzZhvIRj5GAMJRd09tIX630GHZgl5ofJoPJJD7mg9ZOLPMYN6RlWVNBujd9U2Vnk\\\\\\\/jgT41aMnSMVqoHPIs\\\\\\\/w9JIxP8tbLucm1lhO58C3M90737jtxiQRu0LbqU82LsuAVP40HKR9ysg9lR+9gFlFKSKDhPIZXYhjifw325y+iEnXU3RJxo0yFsgdtlHJErWP\\\\\\\/gMh18gre2tikl3tdU7pibfe7xRBDa5ge1wid1F0a8zelX1+tjoDXnEf+v7sRgkij1oS8yLuamyJGxEar3xWz7KKcXNVqM4ok4aT00ppcqjTT3iq5\\\\\\\/BfWpg7DWc+ryKeVPV9PGcH5f0tSas+z88kQ05U3pEMvDqEQsmdz9RkcuIL2k0tkcGheWIRiMKPlGMzfQUIToRvnQ2IVwQ5PNamRI53hGcQG2xd5izhPGRwoNDEUELqp81oER+A3qP3bajySLk1uygbu8SYz5fHL7\\\\\\\/GenjHcMKprDd1GJEeWs+LUby+HePmN\\\\\\\/2Yd2m0kGjYSIrlHpnVOyun5dm0KOvwhWgutZoDqsZefRBG2zv9witAEtwupidkI32UqTApNvogeGFN78cxDsPe7I6lbccUfrCwTLNojnacXN2GoB9hHs7blc1pDoMMMbJBqIt1zOMBpE+Fgz7spVMzn4KArU18u5Imktxr2EMO8aEU92UNjSHybgI\\\\\\\/T7cOdOr4vF19WlMMpjcwO2w1AvvEQdSf6ztLxKidm21Y2R07HVnlo6zlImoY19nANLTHrsrtrDgP9WpptF3uI\\\\\\\/HealS9DFJWRThkF0Zz8QuQ5h7XVL3r3Ld36FFJu\\\\\\\/cFrlNojxoQXuz6dEsH0ZI\\\\\\\/XlSc3caveW1IYZHeG5jc\\\\\\\/c4M1CZP1GafD9qrcDQp0zCW4+L00tlYKvxg+rmLq\\\\\\\/u8mB\\\\\\\/4C62UeaJ91YFJph58+wM0ySV\\\\\\\/EaV6\\\\\\\/fCqFlJuCXftU2YhZ\\\\\\\/rjf43nEkCXFZkkOcPDrVRoSTASKDqd8alKv3n35esjn8nN1eUa4SkkPM+9R82XNUgTKrixVbktog6eTRocSFNxkpf+4iBZrppucKwS8pkbGFYRqtGz3n5BoZcJ1dswNsR1lRtsC8JpvevnqjAqr4EfkXjzEbBaWZcx7heCIWQFy\\\\\\\/A2HX0Q2XfEf9tWuQnCcnCFvu2OJeqhXHyhopqiCriUjI0LLU+Ojw4sVIXXSl544+KcbfomwvjpaxPTyjoCfPiaZ6f+JS1234tuQ82p9yCyOG7NY8svAwu7JtMbkLWXmeXBE+or2Zvq5io3Pc93Xg9LhHqCqLG8ZKsCebQKZITZIvpszIFIttegzP\\\\\\\/jxaWiRtpVRK8UYafHTNT1dV\\\\\\\/qij+5f7Pz7A5oz2WNo1ZqKqFGZ2r4sDiF1++zrqX\\\\\\\/K0yq\\\\\\\/ZxkKg+SGUT4looJNIFhdb5foo59V8oH0dOcXQQOuodq1ldLNOuOOrv7IJkasvcWlox53NBnuPWUxQFWbHxvMJwXeb9nX2xoUlLUrq3Bj2Y5pW1wiA1mLv368h+eGg2fpstUPR8vuHLl0oGfsYqKn0IQfonF7Vd7z+6n783KJn0Ii7myfLtf33YqmqYjZ1V5la8c0KBPE2t9nWxn6xL4J2nux6La1riVPG1otC0XN9sTknyagKu+OjhSTyNbw1uVn9MumYzJyQ4PWG8NtRsltu+3AsrpUE79k32FDh11iwMkk5RgPCFYvYXYgd8WsyoRH20w2NST54b8q6hPSjcW4YPuLdznH0Y3sb8FxD6+8V2\\\\\\\/M1rxaXzsjkZcVxC7FOM8LYDbRWGfXO7YFWcer78rUVYHcR4kv9\\\\\\\/lLW+MQoqtzfIxMKAVocFRQPv4B0rdIxL5ol9uTWaoQxprr9myGSQl3utYXw1B68ansUt\\\\\\\/wPqcNW5g1q68dQegQDYKPUe5bymPHeecuCf7O+IadAwhis5vyxKgYhIht3d60jCoSJyT4utA1tJC4ZlpS2HzZCsLJyDKWgUzd0VR6lwhX\\\\\\\/q+b3LI3FlvGVrUVGhJdD7Pdf3WBcrMjYi\\\\\\\/xqTW3DrqXw\\\\\\\/XyI5+7nxe9wMu04DZESuagnNr05wmdgbibxMq+O3URVhr57T80oPpiC\\\\\\\/+s8CcPn1Z6T36MWt1GYob0sdagP4WgEimd2QARZGPXgC1no9spZkypjC8Uq50dk2VZDshnyDFAWOqCdxhBFGvYMN0Wt4CudpwDWxPo+lwO+KePUGH619AgOJffkDPpviPcDU+F4xbASkyTesaYDaV5YGpHIMTh50EEhi+v8PtMmleEOIbmZq4qNEbTHW6YzP850f1XtXKfybtFVMwWbY7BRQ7o62OVkvEnpU04Li9zV5qgIAzq7ky4PKa45lsizyGUmfPr0vNFZ1tgelYTokPbPvZfyv+qKQOzA6GSkpZKhPDZty8VjoEsK38m4776qXNRQr+yEuBOwZCp3OpCnPR2vXIG5EcLUbVQRsRsOG3C7Jg\\\\\\\/S7dr7s0ete1LPnodsGQZoh+37FC\\\\\\\/gCPD8wF\\\\\\\/9BveY6KUNfPC2wZZm9ndCw6inpdEWOXz5v9LmozGd6Hw\\\\\\\/rtEVg\\\\\\\/ooYerK74xARAlo+qA7kch8h0xUzDHq7esudhj5mTFlE4SqsJBzgCBlwar1xK0nTKP0DnqPeJ4ll1EHBtaUZR+jjNC7abqf0Wt3P581S87HkdSIUw+Bfsd1s8EZ6\\\\\\\/dFrrPnNHNnfaVb9mYgi0rvWX1XndVXRFdSzunBDDlmXBZ0CylDH7RC0f8OaGhGcRkhyJXovIKuX0lwmaYu\\\\\\\/5ENXeupCSmHxYe5ZqEcSUlslKAzV1vhP6y1b2OS0rxLYcLkrgfNzvJm9FkZRrAtHsxSEEJfLhMxqmgzBGRjdTPnpz3NGHEfoKzuI1N3PlAmEQU8bzmaz1Rtl3XgaBL9BTGNz0zItY6Lf9zf94wnDYn\\\\\\\/xsgd00aQhv2f1\\\\\\\/igjFUhkBvOQQ+4vsIh0\\\\\\\/3A5KT2w9cQiIa37+t8l1uBIYsUh6YRDsrKBEpdX6ky9gzqUZlHuyjxMlgUyyp4zwJDREmfwrj\\\\\\\/b7VoSFD99lutTaY+dM20hiEzpuSF0s+nS6X6tQAibwWiOEqVCPcQDuSPotLhJAzBxYmCqFwdJWjEqMaJwnpTVA8cwDTbuWYHNItMl4e\\\\\\\/mH20U1fHVwnwYOjGQCZYGg0QiuDZwQm\\\\\\\/wNluluz4h5Au5fz6neJesBkUG3rX3\\\\\\\/O8Pz7TUReSFB+KRMIfZRADN8016Bfgv+fxZAZFIin7ce6phZGemTO\\\\\\\/QfZmd6OPG0C6KPNozQJstEhBC67u5sb8WKKU6jzy8LXv2+2WWL7rf7FZB3jfubYHGttUgrsVaky2hlAgKYlYWKQTWcFNAhxhhZCNIqeIYON7ERgeNCYALDOflnP72ODiECs4X8mYwo1i7ZGnbAG1PUwY+8afjBF0Ae8HNa5Mm3DI\\\\\\\/DdkvpvfzIqGYCXKCIHJzkofyHxWrFYT9PBN49dez6eNiRIFYHg\\\\\\\/gjzJH42m1sRLSO\\\\\\\/i\\\\\\\/khm7Iu5z5eu7BBO7W1CqxVafdkE5pj6Q57HYGoKmh92mF+e7MWOkc\\\\\\\/GBoywb9unXxoKGQHkt4mg6mV+yA1vmtmN+ylMMyI3k7b450p9qg4OyvSZCE7PopygCVG+rzARtU5zzwMHU5xKrZk3dvReZ2hKces6pPa8S8X4ALRrGuNHIBcdQ5XatsiyjF7ANXd1OKwNeQJxwxuPHoQ5BN1Zl5sJcepM9nr6n1LWisIw87vhUokblpMWF5lFjCi48Vi\\\\\\\/V\\\\\\\/XMV5WxDiZ+2QWAcBEEoaGx0d6ZE8uOq3xXrcrdk+ImqYOUG28tP42eyBA\\\\\\\/NcCJW1JyKIF74tcHgPCCm+WOt0qMa+uLd17OHEd\\\\\\\/QBRlp\\\\\\\/vi4D+u21G\\\\\\\/VJjLBwBPXTqilLfZx0BBb7g2ySYGduifj69oI006RhCpqrXRw8ZcsZqflOOi1WE2OzzWWXMddZIqlIyIOLjtK64uMPMS2pKF6FI39twAV\\\\\\\/jrCeDOzvb+YFi8TSE\\\\\\\/Zft50mFKXnsT536ZFnKd4vEq91LDp+gGWDqB8eFOM3puMTWo9Iq6QvcJ6z44nnIagfD36j2KaAz3gBqdL7ZpzBN1To7+30NZ7uOmQFAMutwavwW5XwbsrS3H4FuEiJYnJn5rw2yJ0kpFURLSCO87qSWCTdXOBAIoHKRTRtpSoMji+Ge3jq67n0DYIGF5NnvoCqQKQ4Nft3U6fKQ8GrHEElYzKJou0FrvoPwklUheucKKaSXnohoYDmFg0nLShJLSjQwmoEwf3qq+rNrf+l+TaOfF0v634NrSsWQ+Ia9WBScwpSwN3jRSs5zzFmKXpUjEFwHUTjiQPs8JkPQbpzmKNZ2U54GbIPWyt\\\\\\\/sGULxUCHPA+WLiZ0X4dgYjfU\\\\\\\/O3V5Hc5FGF0L0ASjape7JMMgE7pcooHjEJM6N0ZJs\\\\\\\/cV2spWbI9baaO5lX56xbR0tqAJ02g2GH0yJXsjKYumQBDYPbwP892XiWWIPGru4sttrJ9s8A2TnKZSnvqZ7xiY\\\\\\\/DvTMXzmy\\\\\\\/aR6Wv6+2utv1HfiCpmZRVIqXAYdyvg2eYOwIHjwA\\\\\\\/b6eRzbhwV6soB9T49f5OVaobaAOAr9XGqdsp\\\\\\\/CSopb9\\\\\\\/KqwU6hYS9Pm4yVVHELhC\\\\\\\/7gPaU5ldyEnFEEPRq+58CvonxdAOsdIdvTJ5sRxe9qS9nYiuN5WNDx0eLY6oxV3XqemfKEIPNsEzfL5yWPUo2zQI5HKQUhGJV4SIpSveAjhjbss76+A4c0fSyGMhlTUOzoVV1eVctRrwqKnHTcBM4vogfGDKDTWR8dSGk2hjc4+rHfBd+lPh4HhHBDRbKWK9uMvHmDWdmZujRD+Jz7vQuHVfMGWUUCMnwBhLi9ZGJMulhR+UcCMpEXgDFhqHUwLHknWnPLjOq72xUX7+x2YpbkCl4HdhdEI\\\\\\\/ghSdfmWFardBoARCcdGmHHR5NCu41AYXr1hBg1CNSCYCfgsdU1n17fieZY+6GdEjhqGntcQixEhuNp5Rgy0Al896zLAu0VAzFUAryAuis6WvzLTgmFnWra2\\\\\\\/Fpjfm\\\\\\\/B+0BBOfeFe\\\\\\\/eQqrOPEJmyKUV3vfac+T+jgFqUPL\\\\\\\/7RXGxWrcAqw0AVzEemOz60AP\\\\\\\/Oj7aFYuFa3+S9n8GI8jB6Hrs4FFSXJTwer11sc23nlutxFamJHps1lQEpdgdR2Qt56ku+gSdh3ry6oW5yAOrkRvT6t8wr3bFVCtfzUUrRvVqNTnGP2r4GOByQgX4nGOQzhGKPDZLwtV2B\\\\\\\/k4SmqxUlFaSHiOkHgn6oYdRxfF5ywI026yqpD+3Ck+Wnze3l7zmtGlYnjR4SpjJP8xmk4pcelNxM0zKqoyUZdmcnkyjJq1l7Xf5Acv4Nz\\\\\\\/lpOWIgh+WUVhj2uaMqmSeUr4r9EvYv9oxy\\\\\\\/kWVYoF34tKVmeHkjPBpHNoNyU6fkpe4SRuH\\\\\\\/hdAWbYyGItQFW4vVJvMzxx4sMk81E4+11J\\\\\\\/WsDSvsTemKiduDjzRn\\\\\\\/CiiNrAkENDz804b4K3NdQ\\\\\\\/1ds0DnCKd+VJbc3tgJP\\\\\\\/VeRxSQxkNef7usKW1795clb2LCWVkhCB6mZVAGZRQDxrO17MmKZHbu\\\\\\\/1R9nlNcA+7ae9lXfLG5h6tF2xbqg2j1S47uaRXwI6E3oH0ACYdKGjg85pTtfOmSwcmiqxYSN2P+GOpF\\\\\\\/yDqF0FIa\\\\\\\/y5ByTlUpLBJ\\\\\\\/O3NLuzn0Pf\\\\\\\/uuICwwafN9Vqy4zZ7rGrHOlE72\\\\\\\/Uj9rAyYAhp0Tzx\\\\\\\/Pjy2pBn2yawFOrLGJgDZGTu1ubiCn6Wbk+ryVHIeEqJgyV0T+rk5kPxyNffbLzY5F1Ucqz2wlq2JhRXkwlJ40Yf\\\\\\\/rknWfjNBI+zxOqf8C1a1Tjhldu8HQ82oJAbu317a5BMvCM\\\\\\\/aL9ria32PQEqUee\\\\\\\/hPnKN2hP4Dnd7T4Igktg8f46o3CvKpXEey23XqHmhJ3Oh7f7jNoACD+WaE13ZDj+KDeuuzjUC1sh\\\\\\\/H\\\\\\\/siy6Ky9B42CcdUAvs8aRSbHU1ZcxiQIDadUXgnzmhPPDMQZyb4hhyqRHL4l20kipP+wEQbCm9ZQz\\\\\\\/WQm4qntDV4DTFodEo8ac6ilO+yaehGhmVMJDPvKpeftScg90vaksfcJai7CUmFDYHfV5yDWOFNsJ\\\\\\\/gs1Ek9ULpYpgD73fYCQv7e+fkjIY2PxNiePznUZtu+fwC693i9i9xEKVIUmhJbAMy7zua6HPyDgtpm7wRHoYvmJmvd0Lz9GQIs6hOZnbPULe8lBlTQ+RXEp80qcWJGCVpTngO9jR1GoiN0xV4smjD1CfQcGqPsnG7dU08bk29JGz2YJFCRyZJuHc52T9ZTIuRhnOENGr9BAN\\\\\\\/8NgsJT5Rt9NrqaG+YXKLeGJ0yASEMdgtAA29q7ewQ7kE1vqYic92r6kJwvL87oGdoU6VAcCplBGVc7rQWhox8iQt7nfoFSLBm5ZAx7ZeajpzMbgABbxonpDXjQDKD4ANbSH0i20wBy3fF3GoJxahjlwptXQqQScAdGwd9pNx3XZlmYABYE+htCVGEe3+qCI1\\\\\\\/MB+9Aa5tTpexdjMUfDaq2V0nX0K2w2jdfKX5A8qxeBrN1Vq8rKxpJ\\\\\\\/zClg3V0Sis0LqfoRicFIKNqYiOec+7+XOb+lteA\\\\\\\/VMhQt\\\\\\\/yQ2xQJvQhe8sChMDn++KghAhJfIDzxk9wiADAYFjO\\\\\\\/kzl8yJiGdsEnqh95opYDcdINF98nauRFMew7bcmkX7YZiXERqRZ4jBJMec7iDzH8ikzfGK+x44RR8ohfFpxYHCG0EK6xCDxZHjPXNmLwBGfa2QJqwqv0o5cX\\\\\\\/9e71dN\\\\\\\/fhDr8hergYzNv8LJcZjwPoHvystGlFYlsMOfwJjCyTFz2qhZlhfxEqtbaBMyWo3w+ZgVRhA4vI8TfbaVdeKPzT\\\\\\\/lwuWMk8LQWfzM97F3LAhNwNasu5LLixmhD73I7JnVoF5KbK4nokccc7aQ3UpCyUV897I0ILvXKJmnUqBZV4T+sdQr3sFmRJdfGCmuxVSh1stSIo281EZ+jVOH86L5JvtuKFHdJCXgUVNiHEJAYxS+uiBB3Gc4f57A2Nc0Pp8rjmzoxBATgZRKgHDIlL4RCAAE8wZrEI5RoDZV5pTR7WbVjdXS8H\\\u0022,\\\u0022iv\\\u0022:\\\u0022c5NUncoCjMHDim1MgQZUGg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f3794bf71345db419b33f5b496d6537a052f0f0ba2e384c02bb18dfe4494cf52\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022cKGZpts0DSPWcBSAxBatygg4RcWUBLbvC\\\\\\\/4BZ4hD9KCyXZ6gyQtkxKLwDIJm6GgK1Dt2rGZyIrs+5kasRzSnbKUoSMf7BzNABvgGczjPOzRFGjtS9RcZF+1s9WbL+vIFDZQzDTkWjj8+BZ\\\\\\\/m2pWpcRUQj7nNKz95kfftXZz12AsQb2TUJz5\\\\\\\/K+sMNZRRXHip\\\\\\\/a\\\\\\\/F3sqq\\\\\\\/5YSgTSq2AhuryHz2r3OjoS\\\\\\\/lpZLZy+WSWZ7tez3CAkNrbK359y\\\\\\\/xSS2Nf4jx5mmxMYYcN3g5y9MOEZhpAxOl+lhZr5YIOmkuwyQ2Kcl0+wEQAU003E9O2eKcvibp\\\\\\\/f4s1KtSR5mwPNPDuufvi9z17eoFRP9p0leuy1tUFmR1aCHhjcZ0ATpcBXuSu8OGgkH8\\\\\\\/IvE4XL3MHPUFfd4qJpHD1NLqo64cm7YglgsMFzNamf7RSV+g0KsWOJcIF5XjlWaURFi6\\\\\\\/K9tYLZQjT6n\\\\\\\/sl2zTLXrqNudCylsiE9sF24h\\\\\\\/v1eWTVLHiftrcGLWBvaepxTjcaatPDaZsfAN27\\\\\\\/CXKJoNtTZtRm6e+3jHHv2xBVCxSdHGBFrcpgWFr2WHN5JKjUt4uRtrVWxd3PBlkTD4bt6lt8dMPNT7fXI+ey7Lau70OfK1PJQz0dxqEpucI0E3UnwbrMXkbOedAZqlliyR34a7YYSHuX8wfuiEZ39DrXeyasqipsb2bJ3NePOqXGCfp2X2HKCcEMlIcFsh7JU9DG6pGLfgM3WyOdwQZSWvgqmH9Xxsj1G37xAuzBCzNo9CgXhv0X5qSRTeVlhQi\\\\\\\/14Zed\\\\\\\/43TGLMbYAwEXDu\\\\\\\/btVUZaYTnc50qCXiiE9ZxOe2TBfzFm34d8AJQOvdL2wnXFWMLJOvKZlkO9mEooyBIuyS6hkBJbGiz2tBMfzoGAC7FocYLLgkq9AuwPPrVm4WCgR6rVlIshgPgARFnfmMyyfWOpOh8P1ZK5u0E07GilNzShKzdlM3ZNk6lHQYsxX0YNPoWb0x2+C6zeIb1mUbduk4FPKMbYG5Juftety+MBfH7RwKHhhMkSLDOkjD1HwixAiVYXLbKsvNysPmdooTaUv5EMjg3SYU3j7hLG4QCMjJ1Ogf\\\\\\\/FKy5\\\\\\\/CvYk7IVau9cemQR0B2Id4iEnUqUP7pzYOuTGvK0f8UZjVWDiQBy89XZ9Ehg5E1OXkG\\\\\\\/EZPdFt9tBIYtO\\\\\\\/YvxW9dKV+\\\\\\\/7Jk6smirSSs3icRgftVjmGVvJ6w172wuOqYal7rFz+wFairujvjDaC2HO3pDnY2YnYNUD6mn21YloiEePPEbrXHMDhhk6RPS\\\\\\\/hMqjQfaqjWUyJ+XANg8jN56aPbuB6DLPTIuBUPwxqLMuIt5b9ZV2aeGox+Sab2IChD1ufpotZuI8PfoIiRMsg4vEUFU0EK3HW+J0KXiGV40fS16V1oFjaWhxLWBssCUyS\\\\\\\/pTWbMgOFYmczFWGuMr6SIMU65m4+oMTYCVSrrRbYNqScqMLYSk7JYdwXjVZ5VwoSgVKYUyV4Uvt6dX8+NQHmq32F+zyqF7R8wt+5xCmN5mxHnj4W95Cl+ucYHvhpij6eB54IUld+8n0bAi\\\\\\\/Nyj6KnguCv7NYdfWuw6cjcjORmuSJVW0tTNDXlBRxzk4EksdAIHI2k1MEX6M0LWcqHKuD9MEgs37y3R29KiMWhZTEJU2\\\\\\\/ZYX4QFzFLprlO8NIyiWE\\\\\\\/Y6R0iNc90IcFhmalAaF3WZeMIY+hGfPUuszaFxG1qjTkaQaLGB7kpnPe+bLWvqkm50cER\\\\\\\/hoaYKXN\\\\\\\/kDJVan0RDG9UW1KWy82zWGv+kRse2L2xVPXESSQtiDvQF4f9JsEJVQlHbz0HFU6UVvwplV9GcxpNEfCgxz3vUi7hmVSepHIjOThzKsq+DDQc96Hlimw\\\\\\\/pL9zm2xmPvd6mnAlBhdljtRbpqG3lcK6aK7vOJRyP4TiPiZULoQ4jAEZYN7JYf6+8E4TTsA8+kFYc0V1NblCRFJ\\\\\\\/yIevqE3KLnd6B47UdURC7m1jDGYAXzxqOgTPdrTVN6zbJptXosZLR67h7vHb7u558kxbxodIUHUil98tdAoGQ\\\u0022,\\\u0022iv\\\u0022:\\\u0022ZR88xjIjDz32GIaA4R5nAA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f97fab4bf9e7aac1a54e3ae48aaae62be33c66867fa64afbeef2f746b3e031dd\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460115 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/973" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-LKYlusiTaS9QVWQkc0tTrqczeSasui9itZSmknrIG_o" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/973" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Kwrt0tCzyJmHtthoJeVeGEUkI8WDYD4VSZ9ILhqDaDk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/4/957" title="Whole Genome Comparison Reveals High Levels of Inbreeding and Strain Redundancy Across the Spectrum of Commercial Wine Strains of Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/4/987" title="A HAND2 Loss-of-Function Mutation Causes Familial Ventricular Septal Defect and Pulmonary Stenosis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460115" id="node460115" data-pisa="ggg;6/4/973" data-pisa-master="ggg;g3.115.025437" data-apath="/ggg/6/4/973.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mapping of Variable DNA Methylation Across Multiple Cell Types Defines a Dynamic Regulatory Landscape of the Human Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Junchen Gu</span>, <span class="highwire-citation-author" data-delta="1">Michael Stevens</span>, <span class="highwire-citation-author" data-delta="2">Xiaoyun Xing</span>, <span class="highwire-citation-author" data-delta="3">Daofeng Li</span>, <span class="highwire-citation-author" data-delta="4">Bo Zhang</span>, <span class="highwire-citation-author" data-delta="5">Jacqueline E. Payton</span>, <span class="highwire-citation-author" data-delta="6">Eugene M. Oltz</span>, <span class="highwire-citation-author" data-delta="7">James N. Jarvis</span>, <span class="highwire-citation-author" data-delta="8">Kaiyu Jiang</span>, <span class="highwire-citation-author" data-delta="9">Theodore Cicero</span>, <span class="highwire-citation-author" data-delta="10">Joseph F. Costello</span> and <span class="highwire-citation-author" data-delta="11">Ting Wang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">973-986;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.025437 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460115" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Junchen Gu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St. Louis, Missouri 63108 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Junchen%2BGu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJunchen%2BGu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michael Stevens </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St. Louis, Missouri 63108 </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Computer Science and Engineering, Washington University in St. Louis, Missouri 63130</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BStevens%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stevens%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BStevens%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Xiaoyun Xing </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St. Louis, Missouri 63108 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaoyun%2BXing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xing%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaoyun%2BXing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Daofeng Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St. Louis, Missouri 63108 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daofeng%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaofeng%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Bo Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Developmental Biology, Washington University School of Medicine, St. Louis, Missouri 63108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bo%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABo%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jacqueline E. Payton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Pathology and Immunology, Washington University School of Medicine, St. Louis, Missouri 63110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacqueline%2BE.%2BPayton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Payton%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJacqueline%2BE.%2BPayton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Eugene M. Oltz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Pathology and Immunology, Washington University School of Medicine, St. Louis, Missouri 63110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eugene%2BM.%2BOltz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oltz%20EM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEugene%2BM.%2BOltz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">James N. Jarvis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Pediatrics, University of Buffalo, New York 14203 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BN.%2BJarvis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarvis%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BN.%2BJarvis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Kaiyu Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Pediatrics, University of Buffalo, New York 14203 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kaiyu%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaiyu%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Theodore Cicero </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Psychiatry, Washington University School of Medicine, St. Louis, Missouri 63110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Theodore%2BCicero%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cicero%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATheodore%2BCicero%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Joseph F. Costello </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Brain Tumor Research Center, Department of Neurosurgery, Helen Diller Family Comprehensive Cancer Center, University of California, San Francisco, California 94143</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BF.%2BCostello%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Costello%20JF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BF.%2BCostello%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Ting Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St. Louis, Missouri 63108 </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Computer Science and Engineering, Washington University in St. Louis, Missouri 63130</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ting%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATing%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:twang@genetics.wustl.edu" class="" data-icon-position="" data-hide-link-title="0">twang@genetics.wustl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/4/973" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460115" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460115/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/973.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460115" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460115/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/973.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460115" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460115/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/973.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460115" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460115/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/4/973.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460115" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460115/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">DNA methylation is an important epigenetic modification involved in many biological processes and diseases. Many studies have mapped DNA methylation changes associated with embryogenesis, cell differentiation, and cancer at a genome-wide scale. Our understanding of genome-wide DNA methylation changes in a developmental or disease-related context has been steadily growing. However, the investigation of which CpGs are variably methylated in different normal cell or tissue types is still limited. Here, we present an in-depth analysis of 54 single-CpG-resolution DNA methylomes of normal human cell types by integrating high-throughput sequencing-based methylation data. We found that the ratio of methylated to unmethylated CpGs is relatively constant regardless of cell type. However, which CpGs made up the unmethylated complement was cell-type specific. We categorized the 26,000,000 human autosomal CpGs based on their methylation levels across multiple cell types to identify variably methylated CpGs and found that 22.6% exhibited variable DNA methylation. These variably methylated CpGs formed 660,000 variably methylated regions (VMRs), encompassing 11% of the genome. By integrating a multitude of genomic data, we found that VMRs enrich for histone modifications indicative of enhancers, suggesting their role as regulatory elements marking cell type specificity. VMRs enriched for transcription factor binding sites in a tissue-dependent manner. Importantly, they enriched for GWAS variants, suggesting that VMRs could potentially be implicated in disease and complex traits. Taken together, our results highlight the link between CpG methylation variation, genetic variation, and disease risk for many human cell types.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/dna-methylomes" class="hw-term hw-article-keyword hw-article-keyword-dna-methylomes" rel="nofollow">DNA methylomes</a></li><li class="kwd"><a href="/keyword/methylcrf" class="hw-term hw-article-keyword hw-article-keyword-methylcrf" rel="nofollow">methylCRF</a></li><li class="kwd"><a href="/keyword/methylation-dynamics" class="hw-term hw-article-keyword hw-article-keyword-methylation-dynamics" rel="nofollow">methylation dynamics</a></li><li class="kwd"><a href="/keyword/regulatory-elements" class="hw-term hw-article-keyword hw-article-keyword-regulatory-elements" rel="nofollow">regulatory elements</a></li></ul><p id="p-4">DNA methylation refers to the addition of a methyl group at the C5 position of cytosine in DNA sequences. Methylation on cytosine can occur in different sequence contexts but largely in a CpG dinucleotide context (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Fazzari and Greally 2004</a>). Proper establishment of DNA methylation early in embryogenesis is vital for normal development in many organisms (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Law and Jacobsen 2010</a>). DNA methylation plays a pivotal role in genomic imprinting and X-chromosome inactivation, where methylation of one parental allele suppresses its expression and leads to monoallelic gene expression (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Reik and Lewis 2005</a>). In addition, epigenetic modifications of the chromatin, including DNA methylation and histone modifications, orchestrate heritable, cell type- and developmental stage-specific gene expression in vertebrates (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Robertson 2005</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Portela and Esteller 2010</a>).</p><p id="p-5">Since the advent and wide adaptation of next-generation sequencing in epigenomic studies (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Harris <em>et al.</em> 2010</a>), many studies have provided insights into the functions of DNA methylation at a genome-wide scale. We now have a catalog of DNA methylomes of many cell types in different organisms (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lister <em>et al.</em> 2009</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">2013</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Shen <em>et al.</em> 2012</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kobayashi <em>et al.</em> 2013</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Zhang <em>et al.</em> 2013</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lee <em>et al.</em> 2015</a>). While the majority of DNA methylation remains stable once a cell is fully differentiated, DNA methylation undergoes changes during embryogenesis, cell differentiation, tissue development, aging, and disease progression (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Jiang <em>et al.</em> 2013</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Meissner <em>et al.</em> 2008</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Laurent <em>et al.</em> 2010</a>). Recently, the Roadmap Epigenomics Consortium generated and analyzed a large number of reference epigenomic profiles including histone modifications, DNA accessibility, DNA methylation, and gene expression across a large number of cell and tissue types. Collectively, the integrated analysis predicted many elements distal to genes that colocalized with various histone modifications, DNA methylation patterns, and relevant transcription factor binding at either open or closed chromatin regions, and many of them could have gene regulatory functions (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kundaje <em>et al.</em> 2015</a>). In particular, loss of DNA methylation accompanied a gain of histone modifications indicative of a poised chromatin state and a gain of lineage-relevant transcription factor binding during differentiation of human embryonic stem (ES) cells into different lineages (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Tsankov <em>et al.</em> 2015</a>). Furthermore, in agreement with a similar finding in mice, transcription factor binding correlated with a decrease in local DNA methylation (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Stadler <em>et al.</em> 2011</a>). In a study on neural differentiation from ES cells, different patterns of DNA methylation changes were observed during consecutive stages of neural progenitors derived from ES cells (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Ziller <em>et al.</em> 2015</a>). Additionally, allelic gene expression was correlated with promoter DNA methylation in differentiated lineages from H1 ES cells and certain allelic enhancers showed differential DNA methylation (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Dixon <em>et al.</em> 2015</a>). By analyzing primary breast cell types, DNA methylation was found to be a stable signature of expressed exons in luminal and myoepithelial cells (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Gascard <em>et al.</em> 2015</a>). During development, it was found that cell origin has a stronger impact than tissue environment in determining genome-wide DNA methylation patterns (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lowdon <em>et al.</em> 2014</a>). Lastly, intermediate methylation represented a conserved signature of gene regulation and exon usage through an integrative analysis of many cell types (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Elliott <em>et al.</em> 2015</a>).</p><p id="p-6">While much effort has been focused on identifying DNA methylation changes associated with specific treatment, environmental stimuli, or with differentiation, we know relatively little about the systematic patterns of DNA methylation across many cell or tissue types under physiologically normal conditions. One study analyzed 42 human Whole Genome Bisulfite Sequencing (WGBS) libraries and identified 21.8% of dynamically regulated autosomal CpGs (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Ziller <em>et al.</em> 2013</a>). Another recent study found that 15.4% of CpGs are strongly differentially methylated among 36 human DNA methylomes (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Schultz <em>et al.</em> 2015</a>). While these studies provided valuable resources and insights, the number of single-CpG-resolution DNA methylomes and the number of cell types profiled are limited. We still do not have a complete picture of which fraction of the methylome is stable and the extent to which variable DNA methylation contributes to genome regulation.</p><p id="p-7">In this study, we set out to define the variably methylated regions of the human DNA methylome. We combined a large collection of genome-wide DNA methylation profiles spanning multiple cell and tissue types using two complementary methods, Methylation Dependent ImmunoPrecipitation followed by sequencing (MeDIP-seq) and Methylation-sensitive Restriction Enzyme digestion followed by sequencing (MRE-seq) (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Li <em>et al.</em> 2014</a>). Using a recently developed conditional random fields-based algorithm, methylCRF (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Stevens <em>et al.</em> 2013</a>), we estimated 54 single-CpG-resolution DNA methylomes. Consistent with previous reports, we demonstrated that our DNA methylomes are equal in quality to those profiled by Whole Genome Bisulfite Sequencing (WGBS) but much more cost-effective (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Stevens <em>et al.</em> 2013</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Lee <em>et al.</em> 2015</a>). In-depth analysis of these high-resolution DNA methylomes revealed that for each cell type, around 11% of autosomal CpGs are unmethylated, of which 7% are constantly unmethylated; overall 22.6% show variable methylation across the cell types we investigated. Combining the variably methylated CpGs into regions, we characterized the features of these regions and their colocalization with various regulatory elements such as enhancer-associated histone modifications, transcription factor binding sites, and disease-associated GWAS (genome-wide association study) variants, and uncovered many important functions that these dynamic regions might possess.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Sample collection</h3><p id="p-8">All samples were collected from normal individuals and prepared in a timely fashion to ensure the isolation of high-quality genomic DNA. Tonsil, endometrium, granulocytes, and monocytes were processed as previously described (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Jarvis <em>et al.</em> 2009</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Zhang <em>et al.</em> 2014</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Koues <em>et al.</em> 2015</a>). Whole blood was pooled venous blood samples from normal male individuals. All other samples were processed in the Roadmap Epigenomics Project and the sample collections have been previously described (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Lowdon <em>et al.</em> 2014</a>; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Zhang <em>et al.</em> 2013</a>).</p></div><div id="sec-3" class="subsection"><h3>Data processing and methylCRF prediction</h3><p id="p-9">MeDIP-seq and MRE-seq data were aligned using BWA (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Li and Durbin 2009</a>) against human genome reference hg19 and then processed with methylCRF to generate single CpG methylation predictions for each of the 54 samples (<a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Stevens <em>et al.</em> 2013</a>). For the rest of the analysis, only autosomal CpGs were considered.</p></div><div id="sec-4" class="subsection"><h3>CpG categorization</h3><p id="p-10">Autosomal CpGs were initially grouped into four biologically motivated categories based on their distribution of DNA methylation values across the 54 samples: constitutively unmethylated (U) if all values were below 30%, intermediately methylated (I) if at least 50 libraries had values between 30–70%, constitutively methylated (M) if all values were above 70%, and variably methylated if there were was a gap of at least 40% between the third highest and the third lowest methylation values. The remaining CpGs were then assigned one of the categories using k-Nearest Neighbors (k-NN) with a k of 4. k-NN was iterated 10 times, once for each category, to allow convergence.</p><p id="p-11">For enrichment analysis, we merged CpGs within 500 bp of each other with the same category label to create regions. However, CpGs crossing blacklisted regions were not merged (The ENCODE Project Consortium 2012).</p><p id="p-12">For visual inspection on a genome browser, we created two tracks by segmenting the genome based on CpG methylation patterns across all 54 samples. For the CpG Regions track, we merged CpGs within 500 bp of each other for each CpG category, in the same way as for enrichment analysis. For the CpG Regions Extended track, we extended the CpG Regions in both directions so that the neighboring regions were next to each other; for regions between CpGs greater than 1000 bp apart, we created a CpG "desert” state “D”. The “D” state started 50 bp away from each CpG. Finally, we designated blacklisted regions with “B.”</p></div><div id="sec-5" class="subsection"><h3>Genomic distribution of VMRs</h3><p id="p-13">Transcription start sites (TSSs) and information about other genomic features were downloaded from the UCSC genome browser (hg19) (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kent <em>et al.</em> 2002</a>). The promoter was defined as 2000 bp regions upstream of the TSS for each gene. Intergenic regions were defined as intervals between the end of gene bodies and the beginning of promoters. For genomic distribution analysis, background regions of matching number and size with VMRs were randomly selected from the human genome.</p></div><div id="sec-6" class="subsection"><h3>Determining cell type-specific hypomethylated VMRs</h3><p id="p-14">To call hypomethylated VMRs for each cell type, we first calculated the average DNA methylation levels of all the CpGs located in a VMR for each sample and then averaged the DNA methylation levels for samples of the same cell type. We then called a VMR hypomethylated in a cell type if the average cell type DNA methylation level was below 30%. To ensure cell type specificity, we further required that a VMR could only be called cell type-specific hypomethylated in no more than three cell types.</p></div><div id="sec-7" class="subsection"><h3>TFBS ChIP-seq peak fold enrichment</h3><p id="p-15">Transcription factor binding site (TFBS) ChIP-seq narrow peaks data were downloaded from ENCODE (The ENCODE Project Consortium 2012) and used in the transcription factor peak fold enrichment analysis. The fold enrichment of TFBS ChIP-seq peaks in VMRs were calculated using the number of VMRs overlapping ChIP-seq peaks divided by the number of size-matched randomly selected regions overlapping the same TFBS ChIP-seq peaks. We used BEDTools utilities to calculate the intersection (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Quinlan and Hall 2010</a>). We then applied ANOVA with FDR correction to identify tissue type-specific TFBS peak fold enrichment for each transcription factor.</p></div><div id="sec-8" class="subsection"><h3>Transcription factor binding motif enrichment analysis</h3><p id="p-16">HOMER was used to search for known transcription factor binding motifs in hypomethylated VMRs according to the instructions provided on the website with default parameters (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Heinz <em>et al.</em> 2010</a>). The top five motifs and corresponding enrichment score p-values were examined for functional relevance. For tissues with multiple cell types, the median value of all the p-values from different cells was used to plot the enrichment (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS5.pdf">Figure S5</a>).</p></div><div id="sec-9" class="subsection"><h3>ChromHMM state enrichment calculation and histone modification ChIP-seq peak calling</h3><p id="p-17">Core15-state ChromHMM annotation files were obtained from the Roadmap Epigenomics Consortium (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Kundaje <em>et al.</em> 2015</a>). The fold enrichment of ChromHMM states in VMRs were calculated using the number of VMRs whose centers overlap with a state divided by the number of size-matched randomly selected regions from the genome overlapping the same chromatin state. The resulting fold enrichment values were log2 transformed and visualized with a heat map. Histone modification ChIP-seq data were downloaded from GEO (GEO ID: GSE16368). Processed BED files were used for each ChIP-seq dataset. Histone peaks were called using SICER with the default parameters against hg19 (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Zang <em>et al.</em> 2009</a>).</p></div><div id="sec-10" class="subsection"><h3>Enhancer analysis and functional enrichment of genes</h3><p id="p-18">VISTA enhancer databases were downloaded from the VISTA Enhancer Browser (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Visel <em>et al.</em> 2007</a>). Human positively validated enhancer regions were used in the analysis and built into a custom browser track. Gene Ontology enrichment analysis of genes near VMRs was performed using the GREAT analysis tool (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">McLean <em>et al.</em> 2010</a>) with the default settings for defining gene regulatory domains to associate genes to VMRs.</p></div><div id="sec-11" class="subsection"><h3>GWAS variants</h3><p id="p-19">GWAS variants data were downloaded from the GWAS Catalog of the National Human Genome Research Institute (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hindorff <em>et al.</em> 2014</a>). The fold enrichment of GWAS variants in hypomethylated VMRs were calculated using the number of GWAS variants overlapping hypomethylated VMRs divided by the number of GWAS variants overlapping size-matched randomly selected regions. For the relevant GWAS traits enrichment, GWAS traits linked to the identified variants were used to calculate fold enrichment in VMRs against all the occurrences in the GWAS catalog by hypergeometric test. Multiple testing corrections on the hypergeometric p-values were done by the FDR method for each cell type.</p></div><div id="sec-12" class="subsection"><h3>Concordance calculation between methylCRF and WGBS</h3><p id="p-20">Concordances of DNA methylation levels between methylCRF and WGBS were calculated by counting the number of CpGs with methylation level differences between methylCRF and WGBS less than 25%. Then, the percentage of CpGs meeting the above criterion was calculated as the concordance. For WGBS, a minimum read coverage of 10 was used to select CpGs.</p></div><div id="sec-13" class="subsection"><h3>Saturation analysis</h3><p id="p-21">We randomly selected n number of samples (n goes from 6–53) and identified variably methylated CpGs with 10 iterations, and then the percentage of variably methylated CpGs were plotted against the number of samples selected.</p></div><div id="sec-14" class="subsection"><h3>Data availability and visualization</h3><p id="p-22">All the methylCRF prediction custom tracks are displayed using the WashU Epigenome Browser (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Zhou <em>et al.</em> 2011</a>). The read alignment and processed data for MeDIP-seq and MRE-seq from this publication have been submitted to the GEO database and assigned the identifier GSE69894. Relevant variably methylated CpGs and regions data and browser visualization can be accessed on the website: <a href="http://epigenome.wustl.edu/methylomes">http://epigenome.wustl.edu/methylomes</a>.</p></div></div><div class="section results" id="sec-15"><h2 class="">Results</h2><div id="sec-16" class="subsection"><h3>Global analysis of 54 methylomes revealed a consistent proportion of unmethylated CpGs across cell types coupled with distinct cell-type specific signatures</h3><p id="p-23">In total, we interrogated the DNA methylomes of 54 normal human primary cell samples (21 cell/tissue types) including, fetal brain, cortex-derived and ganglionic eminence-derived neurosphere cells, fibroblasts, keratinocytes, skin melanocytes, luminal epithelial cells, myoepithelial and stem breast cells, CD4 memory cells, CD4 naïve cells, CD14 monocytes, whole blood, granulocytes, peripheral blood mononuclear cells (PBMC), and endometrium. For a complete list of cell and tissue types, refer to <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/TableS1.xlsx">Table S1</a>.</p><p id="p-24">methylCRF estimates complete DNA methylomes at single-CpG-resolution by combining MeDIP-seq and MRE-seq. For all the following analyses, only autosomal CpGs were considered. We looked at several metrics of the overall state of methylation in all cell types. The average DNA methylation levels of all the autosomal CpGs for each cell type ranged from 77.5–81.7% and displayed small levels of variation among tissue types and similar levels of methylation among different cell types of the same tissue type profiled in this study (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/TableS1.xlsx">Table S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS1.pdf">Figure S1A</a>). For all cell types, the overall distribution of the DNA methylation levels followed an expected pattern where the majority of CpGs were either highly methylated (82.4–87.8% of the total CpGs) or unmethylated (9.7–12.5% of the total CpGs), and a small percentage (2.4–5.6%) of CpGs were intermediately methylated (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS1.pdf">Figure S1B</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/TableS1.xlsx">Table S1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/973/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Characterization of autosomal CpG methylation patterns across cell types and identification of variably methylated CpGs. (A) Density plot of genome-wide DNA methylation levels of all CpGs for each of the 54 methylCRF (conditional random fields-based algorithm) predictions. (B) Principal Component Analysis of average CpG methylation levels for 1 kb genomic windows across 54 methylCRF predictions. Note the clustering of methylomes by tissue type. (C) Percentage of autosomal CpGs in each category after applying a k-NN (k-Nearest Neighbors) algorithm. M: constitutively methylated CpGs; I: constitutively intermediately methylated CpGs; U: constitutively unmethylated CpGs; V: variably methylated CpGs. (D) An example WashU Epigenome Browser view of a genomic region showing different categories of CpGs and the corresponding combined regions. Note the depletion of methylation around the HOXA7 gene specifically in skin keratinocytes. Three tracks each for skin fibroblasts, keratinocytes, and melanocytes; CpG track, CpG categories, and the resulting regions are displayed. In the CpG Categories and CpG Regions tracks, green indicates constitutively unmethylated CpGs and regions and blue indicates variably methylated CpGs and regions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-432559320" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Characterization of autosomal CpG methylation patterns across cell types and identification of variably methylated CpGs. (A) Density plot of genome-wide DNA methylation levels of all CpGs for each of the 54 methylCRF (conditional random fields-based algorithm) predictions. (B) Principal Component Analysis of average CpG methylation levels for 1 kb genomic windows across 54 methylCRF predictions. Note the clustering of methylomes by tissue type. (C) Percentage of autosomal CpGs in each category after applying a k-NN (k-Nearest Neighbors) algorithm. M: constitutively methylated CpGs; I: constitutively intermediately methylated CpGs; U: constitutively unmethylated CpGs; V: variably methylated CpGs. (D) An example WashU Epigenome Browser view of a genomic region showing different categories of CpGs and the corresponding combined regions. Note the depletion of methylation around the HOXA7 gene specifically in skin keratinocytes. Three tracks each for skin fibroblasts, keratinocytes, and melanocytes; CpG track, CpG categories, and the resulting regions are displayed. In the CpG Categories and CpG Regions tracks, green indicates constitutively unmethylated CpGs and regions and blue indicates variably methylated CpGs and regions.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/973/F1.medium.gif" width="440" height="331"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/4/973/F1.medium.gif" width="440" height="331"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/973/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/973/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468068" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-25" class="first-child">Characterization of autosomal CpG methylation patterns across cell types and identification of variably methylated CpGs. (A) Density plot of genome-wide DNA methylation levels of all CpGs for each of the 54 methylCRF (conditional random fields-based algorithm) predictions. (B) Principal Component Analysis of average CpG methylation levels for 1 kb genomic windows across 54 methylCRF predictions. Note the clustering of methylomes by tissue type. (C) Percentage of autosomal CpGs in each category after applying a k-NN (k-Nearest Neighbors) algorithm. M: constitutively methylated CpGs; I: constitutively intermediately methylated CpGs; U: constitutively unmethylated CpGs; V: variably methylated CpGs. (D) An example WashU Epigenome Browser view of a genomic region showing different categories of CpGs and the corresponding combined regions. Note the depletion of methylation around the HOXA7 gene specifically in skin keratinocytes. Three tracks each for skin fibroblasts, keratinocytes, and melanocytes; CpG track, CpG categories, and the resulting regions are displayed. In the CpG Categories and CpG Regions tracks, green indicates constitutively unmethylated CpGs and regions and blue indicates variably methylated CpGs and regions.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">Multiple studies support the hypothesis that low DNA methylation can act as a signature of gene regulation (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Stadler <em>et al.</em> 2011</a>; <a id="xref-ref-65-3" class="xref-bibr" href="#ref-65">Zhang <em>et al.</em> 2013</a>; <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Tsankov <em>et al.</em> 2015</a>). We thus thought to identify the potential of each CpG to have a regulatory role in gene regulation by calculating the lowest possible DNA methylation level for each CpG. We found that 6,769,260 autosomal CpGs (25.4% of total) have their lowest DNA methylation level below 30% (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS1.pdf">Figure S1C</a>). This finding suggests that, although the distribution of DNA methylation maintains the same pattern where roughly 11% of CpGs are unmethylated (average of the percentage of unmethylated CpGs among all cell types) for any given cell type, 25.4% of the CpGs could be unmethylated and potentially have regulatory functions.</p><p id="p-27">Different cell types have distinct methylation signatures. We confirmed that our data supports this notion. By applying the Principle Component Analysis (PCA) and hierarchical clustering analysis on our DNA methylomes at 1 kb resolution, we found that the separation and clustering of samples were defined by tissue types as well as by cell types (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS1.pdf">Figure S1D</a>). In particular, different breast cell type methylomes had notable similarity to each other whereas blood cell types were not as closely clustered, reflecting tissue type-dependent variability among samples in their global DNA methylation patterns. This variability could be due to epigenomic variability within the same cell type, heterogeneity in cell preparation, or their genetic background.</p><p id="p-28">The average methylation levels for different genomic features in all the cell types examined shared the expected pattern of relatively low average methylation levels in promoter regions compared to gene bodies (exons and introns) or intergenic regions. Low methylation levels were particularly evident in high CpG promoters (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS1.pdf">Figure S1, E and F</a>).</p></div><div id="sec-17" class="subsection"><h3>Classification of CpGs based on their methylation patterns across all samples identified 22.6% of CpGs being variably methylated</h3><p id="p-29">Our data revealed that the overall fractions of methylated or unmethylated CpGs across the 54 methylomes were remarkably similar. However, for any given CpG, its methylation level can be either stable or variable across the samples. To analyze this characteristic, we categorized CpGs into biologically informative types. We used simple and conservative notions of constitutively methylated (M), unmethylated (U), and intermediately methylated (I), as well as variably methylated (V), CpGs to classify CpGs using 70% and 30% as cutoffs (See <em>Materials and Methods</em> for details). We were able to assign 21.7 million CpGs based on these criteria. We then used the categorized CpGs as seed sets for a k-NN algorithm to classify the remaining CpGs as well as to refine the category boundaries. This categorization resulted in 22.6% of CpGs being variably methylated along with 70% of CpGs being constitutively methylated. The fact that 70% of the autosomal CpGs are stably methylated (≥ 70% methylation) in all the 54 samples profiled in this study is in line with the notion that DNA methylation is a stable epigenetic mark across different human cell types. Of constitutively methylated CpGs, 61% were located in repeats and 82% of CpGs in repeats were constitutively methylated (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS2.pdf">Figure S2, A and B</a>). On the other hand, the majority (75%) of constitutively unmethylated CpGs were located in CpG islands, and 74% of all the CpGs in CpG islands were constitutively unmethylated CpGs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS2.pdf">Figure S2, C and D</a>). These results agree with known patterns of DNA methylation with respect to repeats and CpG islands. After we categorized each CpG, we segmented the genome into regions of different DNA methylation patterns by combining CpGs of the same category that are within 500 bp of each other into corresponding regions, which can be viewed on the Washington University Epigenome Browser (<em>Materials and Methods</em>, <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1D</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS3.pdf">Figure S3</a>) (<a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Zhou <em>et al.</em> 2011</a>).</p></div><div id="sec-18" class="subsection"><h3>A total of 85% of VMRs were less than 1 kb in length</h3><p id="p-30">Many studies have identified differentially methylated regions and linked the differential methylation to regulation of gene expression and cell type differentiation (<a id="xref-ref-65-4" class="xref-bibr" href="#ref-65">Zhang <em>et al.</em> 2013</a>; <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Lowdon <em>et al.</em> 2014</a>). We thus hypothesized that variably methylated CpGs play a role in regulating gene expression in different cell types and focused our analysis on CpGs exhibiting variable DNA methylation levels across cell types. Overall, we defined 6,014,012 CpGs as variably methylated CpGs, and 663,916 regions as VMRs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/TableS2.xlsx">Table S2</a>).</p><p id="p-31">VMR lengths ranged from 2 bp (solo variably methylated CpG sites) up to 40 kb (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS4.pdf">Figure S4A</a>). The majority of the VMRs were short as 85% and 96% of VMRs were less than 1 kb and 2 kb in length, respectively. On average, each VMR contained nine CpGs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS4.pdf">Figure S4B</a>). The majority of the VMRs were located in introns and intergenic regions with only a small percentage overlapping promoters or exons, although they tended to overlap with promoters and exons more often than expected (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/973/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Majority of variably methylated regions (VMRs) are small and distal to transcription start sites (TSSs). (A) Distribution of VMR lengths at each range. (B) Distribution of VMRs and size matched randomly selected background regions in different genomic features. (C) Distance of VMRs and background regions to their nearest TSSs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-432559320" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Majority of variably methylated regions (VMRs) are small and distal to transcription start sites (TSSs). (A) Distribution of VMR lengths at each range. (B) Distribution of VMRs and size matched randomly selected background regions in different genomic features. (C) Distance of VMRs and background regions to their nearest TSSs.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/973/F2.medium.gif" width="440" height="400"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/4/973/F2.medium.gif" width="440" height="400"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/973/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/973/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468022" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-32" class="first-child">Majority of variably methylated regions (VMRs) are small and distal to transcription start sites (TSSs). (A) Distribution of VMR lengths at each range. (B) Distribution of VMRs and size matched randomly selected background regions in different genomic features. (C) Distance of VMRs and background regions to their nearest TSSs.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>VMRs were distant from TSSs and enriched for TFBSs</h3><p id="p-33">Interestingly, the majority (88%) of VMRs were located more than 5 kb away from annotated transcription start sites (TSSs, <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>), suggesting they might play a role as distal regulatory elements. Loss of DNA methylation in one germ layer was correlated with binding of lineage-relevant transcription factors during human ES cell differentiation (<a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">Tsankov <em>et al.</em> 2015</a>). Indeed, hypomethylated regions in certain cells might give access to regulatory elements such as transcription factors or other DNA-binding proteins to modulate gene expression. To test this hypothesis in the cell types included in our study, we first determined the hypomethylated VMRs in each cell type (see <em>Materials and Methods</em>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS4.pdf">Figure S4C</a>). Then we integrated publicly available datasets from ENCODE to dissect the functional potential of these hypomethylated VMRs.</p><p id="p-34">We first examined the colocalization between VMRs and TFBSs of more than 160 transcription factors defined by ChIP-seq peaks (The ENCODE Project Consortium 2012). A significant proportion of VMRs (36%) overlapped with at least one TFBS peak and this overlap was highly statistically significant (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>) (Fisher’s exact test, p-value &lt; 2.2e-16). This evidence points to possible correlation between low DNA methylation and transcription factor binding that might be important for certain transcription factors to direct cellular transcriptional programs. In addition, 21% of VMRs overlapped with three or more TFBS peaks, suggesting potential interactions among different transcription factors.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/973/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Variably methylated regions (VMRs) enrich for transcription factor binding sites in a tissue-specific manner. (A) Colocalization between all VMRs and transcription factor binding sites. Number of transcription factor binding peaks in each VMR were counted and the percentage of VMRs with 0, 1, 2, or 3 and more transcription factor binding peaks were calculated. Differential fold enrichment of functionally relevant transcription factor binding peaks in hypomethylated regions for (B) blood, (C) skin and breast, (D) breast, and (E) brain." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-432559320" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Variably methylated regions (VMRs) enrich for transcription factor binding sites in a tissue-specific manner. (A) Colocalization between all VMRs and transcription factor binding sites. Number of transcription factor binding peaks in each VMR were counted and the percentage of VMRs with 0, 1, 2, or 3 and more transcription factor binding peaks were calculated. Differential fold enrichment of functionally relevant transcription factor binding peaks in hypomethylated regions for (B) blood, (C) skin and breast, (D) breast, and (E) brain.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/973/F3.medium.gif" width="343" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/4/973/F3.medium.gif" width="343" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/973/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/973/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468033" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-35" class="first-child">Variably methylated regions (VMRs) enrich for transcription factor binding sites in a tissue-specific manner. (A) Colocalization between all VMRs and transcription factor binding sites. Number of transcription factor binding peaks in each VMR were counted and the percentage of VMRs with 0, 1, 2, or 3 and more transcription factor binding peaks were calculated. Differential fold enrichment of functionally relevant transcription factor binding peaks in hypomethylated regions for (B) blood, (C) skin and breast, (D) breast, and (E) brain.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">To further pinpoint tissue type-specific contribution to these observed colocalizations of transcription factor binding and variable DNA methylation, we calculated the fold enrichment of TFBS peaks over size-matched randomly selected genomic regions and compared enrichment of transcription factor peaks among tissue types (See <em>Materials and Methods</em>). We found that the binding of many transcription factors exhibits tissue type-specific enrichment in hypomethylated VMRs (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3, B–E</a>). There are several interesting examples of transcription factors known to play important roles in particular tissue types. For example, FOSL2 is part of Activator protein 1 (AP-1), which regulates the expression of genes important in keratinocyte differentiation (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Mehic <em>et al.</em> 2005</a>). Its binding peaks were highly enriched in hypomethylated VMRs of skin (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>). Moreover, we found high enrichment of FOSL2 binding peaks located in VMRs hypomethylated in breast, which supports its role in breast cell development and carcinogenesis (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Schröder <em>et al.</em> 2010</a>). We also found ESRRA binding peaks highly enriched in hypomethylated VMRs of breast, where ESRRA is implicated in the etiology of breast cancer (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Deblois <em>et al.</em> 2010</a>). In addition, both NANOG and POU5F1 (OCT4) binding peaks were more highly enriched in hypomethylated VMRs of brain (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3E</a>). Both NANOG and POU5F1 are well-known master regulators of pluripotency; specifically, NANOG is implicated in both normal neural stem cell self-renewal and tumorigenesis in glioblastoma multiforme (GBM), while OCT4 functions in the differentiation of neural stem cells into neurons (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Po <em>et al.</em> 2010</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Zbinden <em>et al.</em> 2010</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Deleidi <em>et al.</em> 2011</a>).</p><p id="p-37">In addition to the discovery of many transcription factor binding peaks enriched in hypomethylated VMRs, we also searched for transcription factor binding motifs in hypomethylated VMRs and found enrichment of motifs known to play important roles in each particular cell type (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS5.pdf">Figure S5</a>). For example, LHX2 is a transcription factor important for neurogenesis in hippocampal development and its motif was found to be highly enriched in hypomethylated VMRs of various brain cell types (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Subramanian <em>et al.</em> 2011</a>). In many blood cell types, we found enrichment of motifs of ETS transcription factor family members such as ETS1 and ERG, which are well-known transcription factors involved in hematopoietic development (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Zhang <em>et al.</em> 1995</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Gallant and Gilkeson 2006</a>). All the evidence supports potential interactions between variable DNA methylation and binding of transcription factors that direct cellular transcriptional machinery.</p></div><div id="sec-20" class="subsection"><h3>VMRs colocalized with validated enhancers and enhancer histone modifications, and many possessed enhancer potential</h3><p id="p-38">Given the location of the identified VMRs being distal to TSSs, and the colocalization of transcription factor binding motifs and assay-identified binding sites, we sought to further explore their regulatory potential by examining their relationships with various histone modifications assayed on the same cells. We utilized available ChromHMM annotated chromatin states on each cell type and found enrichment of enhancer or active TSS-associated chromatin states in hypomethylated VMRs of the majority of cell types (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>) (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Kundaje <em>et al.</em> 2015</a>). In contrast, there was a depletion of heterochromatin and quiescent states in hypomethylated VMRs. We called ChIP-seq peaks of histone modifications for each cell type and found that up to 87% of the cell type-specific hypomethylated VMRs overlapped with enhancer histone modification ChIP-seq peaks represented by H3K4me1 in the same cell type (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS6.pdf">Figure S6A</a>); the overlaps are statistically significant (Fisher’s exact test, p-value &lt; 8e-10 for all). We also calculated the average signal density of ChIP-seq data over 10 kb regions centered around VMRs and found, in general, higher levels of enhancer or active transcription histone modification ChIP-seq signals centered in the VMRs compared to their flanking sequences (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4, B–E</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS6.pdf">Figure S6, B–E</a>). These results led us to further hypothesize that many of the hypomethylated VMRs that we identified could possess potential enhancer activities in different cell types.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/973/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Variably methylated regions (VMRs) enrich for enhancer chromatin states and relevant histone modifications. (A) Enrichment of 15-state ChromHMM annotations in cell type-specific hypomethylated VMRs. Log2 transformed fold enrichment values for each ChromHMM state in each cell type were calculated and plotted. Replicate ChromHMM states were used for skin melanocytes, fibroblasts, keratinocytes, CD4 na&#xEF;ve cells, CD4 memory cells, and fetal brain, with replicate number indicated after cell type abbreviation detailed in Table S1. TssA, Active TSS; TssAFlnk, Flanking active TSS; Enh, Enhancers; TxFlnk, Transcribed at gene 5&#x2032; and 3&#x2032;; EnhBiv, Bivalent enhancer; EnhG, Genic enhancers; BivFlnk, Flanking bivalent TSS/Enh; ReprPC, Repressed Polycomb; TssBiv, Bivalent/poised TSS; Tx, Strong transcription; TxWk, Weak transcription; ZNF/Rpts, ZNF genes + repeats; Het, Heterochromatin; ReprPCWk, Weak repressed Polycomb; Quies, Quiescent. ChIP-seq signal density RPKM values over a 10 kb region centered on hypomethylated VMRs for (B) cortex derived neurosphere cells, (C) CD8 na&#xEF;ve T cells, (D) breast myoepithelial cells, and (E) skin keratinocytes. RPKM values at 50 bp resolution were calculated and plotted." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-432559320" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Variably methylated regions (VMRs) enrich for enhancer chromatin states and relevant histone modifications. (A) Enrichment of 15-state ChromHMM annotations in cell type-specific hypomethylated VMRs. Log2 transformed fold enrichment values for each ChromHMM state in each cell type were calculated and plotted. Replicate ChromHMM states were used for skin melanocytes, fibroblasts, keratinocytes, CD4 na&#xEF;ve cells, CD4 memory cells, and fetal brain, with replicate number indicated after cell type abbreviation detailed in Table S1. TssA, Active TSS; TssAFlnk, Flanking active TSS; Enh, Enhancers; TxFlnk, Transcribed at gene 5&#x2032; and 3&#x2032;; EnhBiv, Bivalent enhancer; EnhG, Genic enhancers; BivFlnk, Flanking bivalent TSS/Enh; ReprPC, Repressed Polycomb; TssBiv, Bivalent/poised TSS; Tx, Strong transcription; TxWk, Weak transcription; ZNF/Rpts, ZNF genes + repeats; Het, Heterochromatin; ReprPCWk, Weak repressed Polycomb; Quies, Quiescent. ChIP-seq signal density RPKM values over a 10 kb region centered on hypomethylated VMRs for (B) cortex derived neurosphere cells, (C) CD8 na&#xEF;ve T cells, (D) breast myoepithelial cells, and (E) skin keratinocytes. RPKM values at 50 bp resolution were calculated and plotted.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/973/F4.medium.gif" width="347" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/4/973/F4.medium.gif" width="347" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/973/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/973/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467939" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-39" class="first-child">Variably methylated regions (VMRs) enrich for enhancer chromatin states and relevant histone modifications. (A) Enrichment of 15-state ChromHMM annotations in cell type-specific hypomethylated VMRs. Log2 transformed fold enrichment values for each ChromHMM state in each cell type were calculated and plotted. Replicate ChromHMM states were used for skin melanocytes, fibroblasts, keratinocytes, CD4 naïve cells, CD4 memory cells, and fetal brain, with replicate number indicated after cell type abbreviation detailed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/TableS1.xlsx">Table S1</a>. TssA, Active TSS; TssAFlnk, Flanking active TSS; Enh, Enhancers; TxFlnk, Transcribed at gene 5′ and 3′; EnhBiv, Bivalent enhancer; EnhG, Genic enhancers; BivFlnk, Flanking bivalent TSS/Enh; ReprPC, Repressed Polycomb; TssBiv, Bivalent/poised TSS; Tx, Strong transcription; TxWk, Weak transcription; ZNF/Rpts, ZNF genes + repeats; Het, Heterochromatin; ReprPCWk, Weak repressed Polycomb; Quies, Quiescent. ChIP-seq signal density RPKM values over a 10 kb region centered on hypomethylated VMRs for (B) cortex derived neurosphere cells, (C) CD8 naïve T cells, (D) breast myoepithelial cells, and (E) skin keratinocytes. RPKM values at 50 bp resolution were calculated and plotted.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">To test the enhancer hypothesis, we utilized the VISTA enhancer project’s validated enhancer data and determined whether our candidate VMRs could have enhancer activities (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Visel <em>et al.</em> 2007</a>). Indeed, we found that a significant portion (71%) of positively validated human VISTA enhancers overlapped with VMRs (Fisher’s exact test, p-value &lt; 2.2e-16). As an example, VISTA enhancer hs629 was validated to have enhancer activity in mouse dorsal root ganglion and it overlapped with a VMR hypomethylated in fetal brain samples (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). For a complete list of VMRs that overlap with positively validated VISTA enhancers, refer to <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/TableS3.xlsx">Table S3</a>.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/973/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genes near variably methylated regions (VMRs) enrich for relevant functions and certain VMRs were experimentally validated as enhancers. (A) VISTA enhancer element 629 was validated positive by transgenic mouse assay. (B) Gene Ontology term enrichment of genes near fetal brain hypomethylated VMRs. (C) WashU Epigenome Browser view of the methylation landscape around the EVX1 gene. VISTA enhancer element 629 overlaps with VMRs downstream of EVX1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-432559320" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genes near variably methylated regions (VMRs) enrich for relevant functions and certain VMRs were experimentally validated as enhancers. (A) VISTA enhancer element 629 was validated positive by transgenic mouse assay. (B) Gene Ontology term enrichment of genes near fetal brain hypomethylated VMRs. (C) WashU Epigenome Browser view of the methylation landscape around the EVX1 gene. VISTA enhancer element 629 overlaps with VMRs downstream of EVX1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/973/F5.medium.gif" width="335" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/4/973/F5.medium.gif" width="335" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/973/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/973/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467871" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-41" class="first-child">Genes near variably methylated regions (VMRs) enrich for relevant functions and certain VMRs were experimentally validated as enhancers. (A) VISTA enhancer element 629 was validated positive by transgenic mouse assay. (B) Gene Ontology term enrichment of genes near fetal brain hypomethylated VMRs. (C) WashU Epigenome Browser view of the methylation landscape around the EVX1 gene. VISTA enhancer element 629 overlaps with VMRs downstream of EVX1.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">To identify the genes and functions that could be potentially regulated by the hypomethylated VMRs, we analyzed the functional enrichment for genes near the VMRs in each cell type and found enrichment of genes whose functions were relevant for that particular cell type. For example, we found many genes near fetal brain-specific hypomethylated VMRs that encode functions such as hindbrain development and spinal cord development (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS7.pdf">Figure S7</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS8.pdf">Figure S8</a>). We identified overlaps between several VMRs hypomethylated in brain and a positively validated VISTA enhancer (hs629) near the <em>EVX1</em> gene, which functions as a determinant of interneuron identity in spinal cord (Region 1 in <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a>). Interestingly, enhancer element hs629 contained a conserved noncoding element that was experimentally validated to have enhancer activities in interneurons in zebrafish, chick, and rat (Region 2 in <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5C</a>) (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Moran-Rivard <em>et al.</em> 2001</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Suster <em>et al.</em> 2009</a>). In different brain cell types, the winged-helix transcription factor FOXD3 has its promoter unmethylated in all the samples, but has multiple VMRs surrounding the gene and possibly regulating its expression and function specifically in brain. <em>FOXD3</em> is known to play important roles in establishing the neural crest lineage and in repressing melanogenesis in either mouse, chicken, or zebrafish (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kos <em>et al.</em> 2001</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Stewart <em>et al.</em> 2006</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Thomas and Erickson 2009</a>). We found low methylation levels in brain and high methylation levels in skin melanocytes at these VMRs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS9.pdf">Figure S9</a>). Interestingly, three positively validated VISTA enhancers were located within a 460 kb region upstream of the <em>FOXD3</em> gene. In addition, BMP signaling, <em>PAX3</em>, and <em>PAX7</em> were also found near hypomethylated VMRs, and they are all part of the gene regulatory interaction networks important for neural crest development (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Meulemans and Bronner-Fraser 2004</a>). We found that another important neuronal transcription factor OLIG2 might be epigenetically regulated in brain (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS10.pdf">Figure S10</a>) (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Ligon <em>et al.</em> 2007</a>). Several genes important in keratinocyte development and pathology were found to be linked to hypomethylation specifically in keratinocytes. For example, keratin 2 gene <em>KRT2</em> were found to be potentially regulated by hypomethylation in several VMRs upstream of <em>KRT2</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS11.pdf">Figure S11</a>) (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Collin <em>et al.</em> 1992</a>). In melanocytes, we found that <em>KIT</em>, <em>MITF</em>, <em>PAX3</em>, and <em>TYR</em> genes underlie functional enrichment, such as developmental pigmentation and melanocyte differentiation in melanocyte-specific hypomethylated VMRs. KIT and MITF are known to be important components of the melanocyte differentiation molecular machinery, with their individual activities and also functional interactions. MITF functions as a master regulator in melanocyte development and its regulation is mediated through transcription factor PAX3. One of its target genes is <em>TYR</em>, tyrosinase, which encodes the rate-limiting enzyme in melanin synthesis (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS12.pdf">Figure S12</a>) (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hou <em>et al.</em> 2000</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Levy <em>et al.</em> 2006</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Yoshida <em>et al.</em> 2001</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Mellgren and Johnson 2004</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kubic <em>et al.</em> 2008</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Yang <em>et al.</em> 2008</a>). All the evidence is consistent with the hypothesis that hypomethylated VMRs act as enhancers in specific cells to regulate nearby functionally related genes, possibly through transcriptional regulation mechanisms.</p><p id="p-43">Lastly, to test whether cell type-specific hypomethylated VMRs could act as enhancers to mediate nearby gene expression, we analyzed gene expression data of the same samples to see whether we could observe expression differences correlated with methylation differences in VMRs. We identified genes near these VMRs, calculated their expression levels from RNA-seq data and compared them between two cell types. We found that between two cell types, genes near hypomethylated VMRs tend to have statistically significantly higher expression levels (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS13.pdf">Figure S13</a>). This result supports our hypothesis that hypomethylated VMRs modulate nearby gene expression in specific cell types, further supporting their potential role as enhancers.</p></div><div id="sec-21" class="subsection"><h3>VMRs enriched for GWAS variants</h3><p id="p-44">GWAS have greatly aided our understanding of the genetic basis of many complex human traits and diseases. It was recently shown that variants identified from GWAS are often enriched in regulatory regions (The ENCODE Project Consortium 2012; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Maurano <em>et al.</em> 2012</a>). We hypothesized that VMRs may contain many genomic variants, whose functions could be regulated by epigenetic mechanisms such as DNA methylation. Thus, we examined the enrichment of genomic variants from many trait and disease-related GWAS within all possible hypomethylated VMRs defined in each cell type. We found a moderate but statistically significant association between VMRs and GWAS variants. Overall, VMRs encompassed 2197 published GWAS variants, representing a 1.4-fold enrichment (Fisher’s exact test, p-value &lt; 2.2e-16), and enrichment of GWAS variants was higher in certain cell types (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). 96.4% of the GWAS variants were located in noncoding regions highlighting the regulatory potential of noncoding sequences. Furthermore, 94.3% of GWAS variants located within VMRs were noncoding. By interrogating the relationship between GWAS and VMRs, the DNA methylation levels of VMRs provide an epigenetic annotation of many GWAS variants, which might be useful in understanding the functional relevance of a GWAS variant in a cell type (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Zhou <em>et al.</em> 2015</a>). We calculated the enrichment of GWAS traits linked to variants in hypomethylated VMRs and found enrichment of GWAS traits in relevant cell types (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>). There was significant enrichment of neuronal disease-related traits in different brain cell types. Interestingly, some immunological function-related traits were enriched in VMRs hypomethylated in brain, in agreement with recent findings concerning the link between the dysfunction of the immune system and Alzheimer’s disease (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Heneka <em>et al.</em> 2015</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Matsuda <em>et al.</em> 2015</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Gjoneska <em>et al.</em> 2015</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Chakrabarty <em>et al.</em> 2015</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Guillot-Sestier <em>et al.</em> 2015</a>). When we looked at individual variants, we found variants associated with red blood cell traits, white blood cell count, and IgG glycosylation within blood hypomethylated VMRs; variants associated with Alzheimer’s disease, schizophrenia, and multiple sclerosis within brain-related VMRs; and variants associated with systemic sclerosis and psoriasis within skin-related VMRs. This limited evidence could be taken to suggest that variants not only tag the disease but may also be within regulatory sites through which DNA methylation may influence phenotypes or disease susceptibility. For a complete list of all the GWAS variants located in hypomethylated VMRs, refer to <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/TableS4.xlsx">Table S4</a>.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/973/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="CpG methylation variation is linked to genetic variation and disease risk in different cells. (A) Fold enrichment of GWAS variants in hypomethylated VMRs of different cell types. (B) Examples of overrepresentations of functionally relevant GWAS traits in different cells. Hypergeometric p-values of significant enrichment of cell type-relevant GWAS traits in hypomethylated VMRs are shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-432559320" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;CpG methylation variation is linked to genetic variation and disease risk in different cells. (A) Fold enrichment of GWAS variants in hypomethylated VMRs of different cell types. (B) Examples of overrepresentations of functionally relevant GWAS traits in different cells. Hypergeometric p-values of significant enrichment of cell type-relevant GWAS traits in hypomethylated VMRs are shown.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/973/F6.medium.gif" width="306" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/4/973/F6.medium.gif" width="306" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/973/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/973/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468037" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-45" class="first-child">CpG methylation variation is linked to genetic variation and disease risk in different cells. (A) Fold enrichment of GWAS variants in hypomethylated VMRs of different cell types. (B) Examples of overrepresentations of functionally relevant GWAS traits in different cells. Hypergeometric p-values of significant enrichment of cell type-relevant GWAS traits in hypomethylated VMRs are shown.</p><div class="sb-div caption-clear"></div></div></div><p id="p-46">To further investigate whether DNA methylation could play a role in the regulation of these variants, we focused on variants that colocalized with variably methylated CpGs. Among the 2197 GWAS variants within VMRs, 30 were in a CpG context. Since DNA methylation levels of VMRs can be regulatory, the variants could potentially have a direct impact on gene regulation by influencing DNA methylation of the CpGs underlying the variants, providing a mechanistic link between genetic variants and their associated traits or diseases. For example, rs10499197 was found to be a susceptibility locus for systemic lupus erythematosus and this CpG was hypomethylated in several blood cell types. The methylation level of CpGs and VMRs might thus influence the risk of disease. For a complete list of all the GWAS variants that in a CpG context, refer to <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/TableS5.xlsx">Table S5</a>.</p></div><div id="sec-22" class="subsection"><h3>Comparison with WGBS-based dynamic CpGs and regions</h3><p id="p-47"><a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Ziller <em>et al.</em> (2013</a>) published a seminal paper detailing 42 DNA methylomes using WGBS in normal developmental tissues and cultured cell lines. The major conclusion from this work was that 21.8% of autosomal CpGs showed dynamic methylation levels that colocalized with enhancer and other gene regulatory elements. Similarly, <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Schultz <em>et al.</em> (2015</a>) recently profiled 18 tissue types and found 15.4% of CpGs to be differentially methylated. Since our study employed different technologies and profiled many nonoverlapping sample types, we determined the extent to which their differentially methylated regions (DMRs) overlap with our VMRs.</p><p id="p-48">To test the similarity between the methylation levels of our methylCRF prediction and those of WGBS profiled on the same cell types, we calculated their genome-wide concordance (defined as the percentage of CpGs with a methylation level difference less than 25%) between samples profiled by both <a id="xref-ref-70-3" class="xref-bibr" href="#ref-70">Ziller <em>et al.</em> (2013</a>) and this study (See <em>Materials and Methods</em>). Although the samples were from different sources, the concordance between tested pairs was as high as 85.3%, and methylCRF predictions of CpG methylation levels were comparable to WGBS (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/TableS6.xlsx">Table S6</a>). This is consistent with previously reported results (<a id="xref-ref-52-4" class="xref-bibr" href="#ref-52">Stevens <em>et al.</em> 2013</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Lee <em>et al.</em> 2015</a>).</p><p id="p-49">We compared the overlap of variably methylated CpGs between ours and those defined in <a id="xref-ref-70-4" class="xref-bibr" href="#ref-70">Ziller <em>et al.</em> (2013</a>) and found that, although their numbers are similar, 36.1% of variably methylated CpGs identified by methylCRF data only account for 38.4% of variably methylated CpGs identified by their WGBS data (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS14.pdf">Figure S14A</a>). The number of VMRs was smaller than that of DMRs (663,916 <em>vs.</em> 716,087), and 40.4% of VMRs overlapped with 26.8% of DMRs defined in <a id="xref-ref-70-5" class="xref-bibr" href="#ref-70">Ziller <em>et al.</em> (2013</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS14.pdf">Figure S14B</a>). Similarly, 58.2% of VMRs overlapped with 29.9% of DMRs defined in <a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Schultz <em>et al.</em> (2015</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS14.pdf">Figure S14B</a>). Even two sets of DMRs defined by two WGBS studies showed small overlaps (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS14.pdf">Figure S14B</a>). The WGBS DMRs covered more genome sequences than those of VMRs (315 million bases for VMRs <em>vs.</em> 492 million bases for Ziller DMRs <em>vs.</em> 384 million bases for Schultz DMRs) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS14.pdf">Figure S14C</a>). When we looked at the set of regions defined in all 3 studies, only up to 27.1% of regions were shared among three studies [27.1% of VMRs, 21.3% of <a id="xref-ref-70-6" class="xref-bibr" href="#ref-70">Ziller <em>et al.</em> (2013</a>) DMRs, and 14.3% of <a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">Schultz <em>et al.</em> (2015</a>) DMRs] (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS14.pdf">Figure S14D</a>). The overlap between VMRs and DMRs was small in scale, suggesting that the analysis of dynamic DNA methylation has not yet reached saturation, as also suggested in <a id="xref-ref-49-5" class="xref-bibr" href="#ref-49">Schultz <em>et al.</em> (2015</a>). In addition to profiling different samples, differences in DNA methylation profiling technology, analysis methods, as well as statistical cutoffs employed by different studies (the present study; <a id="xref-ref-49-6" class="xref-bibr" href="#ref-49">Schultz <em>et al.</em> 2015</a>; <a id="xref-ref-70-7" class="xref-bibr" href="#ref-70">Ziller <em>et al.</em> 2013</a>), could have also contributed to our observation that the identification of dynamic DNA methylation has not reached saturation. One reason for the genome coverage difference comes from the fact that <a id="xref-ref-70-8" class="xref-bibr" href="#ref-70">Ziller <em>et al.</em> (2013</a>) extended single CpG DMRs to 100 bp. 36.6% of DMRs were derived from a single CpG whereas 19.2% of VMRs were single CpG VMRs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS14.pdf">Figure S14E</a>). This difference could be due to the fact that WGBS relies on a sufficient number of reads to accurately estimate methylation levels, and some of the single CpG DMRs could have low coverage and thus lead to inaccurate methylation calls. Indeed, we found that single CpG DMRs from <a id="xref-ref-70-9" class="xref-bibr" href="#ref-70">Ziller <em>et al.</em> (2013</a>) had significantly lower read coverage than the rest of the CpGs covered by DMRs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS14.pdf">Figure S14F</a>, Wilcoxon rank-sum test, p-value &lt; 0.05). If we only considered the nonsingle-CpG regions in both sets, we observed almost the same percentage of covered base overlap and a slightly higher overlap of regions between VMRs and DMRs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS14.pdf">Figure S14, G–H</a>). The discrepancy, not explained by single-CpG regions, is probably due to different cell types profiled in the two studies as only a few cell types were profiled in both, and also the inherent difference between WGBS and MeDIP-seq and MRE-seq methods used to define them or both.</p><p id="p-50">By comparing regions identified uniquely in each study, we found that they shared similar size and genomic distribution (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS15.pdf">Figure S15, A–D</a>). VMR-specific regions had overall higher overlap with promoters and intergenic regions, and lower overlap with intronic regions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS15.pdf">Figure S15D</a>). When we examined the colocalization with TFBSs, the distribution was very similar across study-specific regions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS15.pdf">Figure S15, E–G</a>) although they all showed a decrease in the proportion of VMRs harboring potential TFBSs compared to the complete VMR set (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3A</a>).</p></div><div id="sec-23" class="subsection"><h3>Reference methylation dynamics browser track</h3><p id="p-51">Using our segmentation, we created a DNA methylation dynamics track (CpG Regions Extended) for visualization and reference on the Washington University Epigenome Browser (<a href="http://epigenomegateway.wustl.edu/browser/">http://epigenomegateway.wustl.edu/browser/</a> or Visualization on the companion website, see <em>Materials and Methods</em>). It provides a summarized source of insight about the potential activity of genomic regions of interest. This resource should facilitate the interpretation of and hypothesis generation about genomic and epigenomic data.</p></div></div><div class="section discussion" id="sec-24"><h2 class="">Discussion</h2><p id="p-52">In this study, we applied methylCRF to a total of 54 samples spanning multiple cell and tissue types. We categorized CpGs based on their DNA methylation patterns across all the samples examined. A longstanding observation is that the majority of CpGs in the genome are methylated. This was confirmed in our analysis, where ∼70% of the autosomal CpGs were constitutively methylated (≥ 70% CpG methylation) regardless of the cell type. In contrast, 7.5% of the autosomal CpGs were constitutively unmethylated (≤ 30% CpG methylation) and, as we have shown, most of these unmethylated regions are located in CpG islands. In addition, the majority of the constitutively unmethylated regions (UMRs) are less than 2 kb. Compared to VMRs, UMRs are much more enriched in regions close to TSSs, with over a third of them located in promoter regions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS16.pdf">Figure S16</a>).</p><p id="p-53">We identified 22.6% of the genomic CpGs that show variable DNA methylation across 54 samples. These CpGs could be functional in different cell types by varying their methylation levels to influence the transcriptional network in a particular cell type. We combined CpGs into windows and characterized them by looking at their size, distribution, and relationship with various TFBSs, motifs, histone modifications, enhancer potentials, and GWAS variants. We have compelling evidence to show that many of the VMRs are distal to known TSSs and possess enhancer or active transcription histone modifications. They not only harbor many known transcription factor binding peaks but also many functionally relevant transcription factor binding motifs. Many of these regions have been experimentally validated. With the availability of more experimental data, more regions are expected to have enhancer activities in certain cell or tissue types. We found that a considerable number of GWAS variants overlap with variably methylated CpGs, suggesting a possible mechanism for the regulation or dis-regulation of these trait or disease-associated variants.</p><p id="p-54">Our results suggest widespread colocalization between transcription factor binding and identified VMRs, even given the currently limited amount of transcription factor ChIP-seq data. It is tempting to postulate that those VMRs that did not overlap with current transcription factor binding events might also harbor regulatory potentials through the modulation of transcription factor binding.</p><p id="p-55">One striking feature we uncovered is that, regardless of the cell type, roughly the same percentage of CpGs are being methylated and unmethylated, yet exactly which CpGs in a particular cell type are unmethylated is cell type-dependent (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a>). This discovery suggests that a cell is able to maintain its epigenome at a default level. In another words, roughly 11% of the CpGs will be unmethylated in any given cell type and the epigenome somehow directs unmethylation in CpGs that are important in cell type specification. It would be very interesting to understand the molecular basis and mechanisms of this process in detail.</p><p id="p-56">We have previously shown that methylCRF is highly robust in predicting single-CpG methylation levels genome-wide with high accuracy in comparison to WGBS but at a much lower cost (<a id="xref-ref-52-5" class="xref-bibr" href="#ref-52">Stevens <em>et al.</em> 2013</a>). Here, we present the first multiple cell type DNA methylome analysis using methylCRF predictions. Our study again demonstrates the power of combining two complimentary techniques for measuring genome-wide DNA methylation and statistically learning single-CpG-resolution DNA methylation levels in multiple different cell types. In addition, we were able to leverage methylCRF to identify biologically important regions with variable DNA methylation. Based on our analysis, we speculate that more variably methylated CpGs will be identified as more cell types are sequenced and analyzed (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/FigureS17.pdf">Figure S17</a>). We would argue that the overall findings are consistent with those derived from WGBS data and that the results clearly suggest that DNA methylation plays a role in the regulation of cell or tissue type-specific gene functions through distal enhancer sites. They highlight the plausible link between genetic variation and epigenetic regulation. The data generated and results presented here will be of great interest to the research community. Our methylation predictions in many cell types could be utilized to complement other studies in deciphering the epigenomic landscape of cells. The genome segmentation data and track could be very useful in bringing the analytical focus on certain important regions in different studies. The combined use of MeDIP-seq and MRE-seq through methylCRF continues to be an economical yet powerful method to generate complete DNA methylomes. Using the library coverage statistics in the WGBS and our study, we estimate that a WGBS methylome costs more than 10 times that of a combined MeDIP-seq and MRE-seq methylome (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1/TableS7.xlsx">Table S7</a>). Our methods will greatly facilitate the application of methods utilizing DNA methylome data along with other genomic data, such as predicting TFBSs from DNA methylation data (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Xu <em>et al.</em> 2015</a>). With our data and the availability of many more single-CpG-resolution DNA methylome data in the foreseeable future, we expect the field will witness greater insights gained from integrating data from a complete cell type collection.</p></div><div class="section" id="sec-25"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-57">We would like to thank the Roadmap Epigenomics Mapping Consortium (REMC) for generating the majority of the methylation data used in this study. We would also like to thank the Genome Technology Access Center at Washington University for providing sequencing services for the rest of the methylation data. In addition, we would like to thank all members of the Wang lab for constructive discussion and feedback during the preparation of the manuscript. J.G. is supported in part by the McDonnell International Scholars Academy and Lucille P. Markey Special Emphasis Pathway in Human Pathobiology at Washington University. T.W. is supported by the American Cancer Society grant RSG-14-049-01-DMC, and National Institutes of Health grants R01HG007354, R01HG007175, R01ES024992, and U01CA200060. J.E.P. and E.M.O are supported by NIH grant R01- CA156690. The authors declare no conflicts of interest.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">1</span> These authors share first authorship</p></li><li class="fn" id="fn-2"><p id="p-58">Supplemental Material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025437/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-59"><em>Communicating editor: T. R. Hughes</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-12-01"><span class="received-label">Received </span>December 1, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-02-03"><span class="accepted-label">Accepted </span>February 3, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Gu <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.1" data-doi="10.1016/j.neuron.2014.11.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chakrabarty</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ceballos-Diaz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eddy</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Funk</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">IL-10 Alters Immunoproteostasis in APP Mice, Increasing Plaque Burden and Worsening Cognitive Behavior.</span> <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">519</span>–<span class="cit-lpage">533</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D85%26rft.spage%253D519%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2014.11.020%26rft_id%253Dinfo%253Apmid%252F25619653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2014.11.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25619653&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.2" data-doi="10.1016/0014-4827(92)90412-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moll</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kubicka</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ouhayoun</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franke</span>  <span class="cit-name-given-names">W. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Characterization of human cytokeratin 2, an epidermal cytoskeletal protein synthesized late during differentiation.</span> <abbr class="cit-jnl-abbrev">Exp. Cell Res.</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">132</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExperimental%2Bcell%2Bresearch%26rft.stitle%253DExp%2BCell%2BRes%26rft.aulast%253DCollin%26rft.auinit1%253DC.%26rft.volume%253D202%26rft.issue%253D1%26rft.spage%253D132%26rft.epage%253D141%26rft.atitle%253DCharacterization%2Bof%2Bhuman%2Bcytokeratin%2B2%252C%2Ban%2Bepidermal%2Bcytoskeletal%2Bprotein%2Bsynthesized%2Blate%2Bduring%2Bdifferentiation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0014-4827%252892%252990412-2%26rft_id%253Dinfo%253Apmid%252F1380918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0014-4827(92)90412-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1380918&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JL08900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.3" data-doi="10.1158/0008-5472.CAN-10-2840"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deblois</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chahrour</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perry</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sylvain-Drolet</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Transcriptional Control of the ERBB2 Amplicon by ERR and PGC-1 Promotes Mammary Gland Tumorigenesis.</span> <abbr class="cit-jnl-abbrev">Cancer Res.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">10277</span>–<span class="cit-lpage">10287</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1158%252F0008-5472.CAN-10-2840%26rft_id%253Dinfo%253Apmid%252F20961995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjExOiI3MC8yNC8xMDI3NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC85NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.4" data-doi="10.1371/journal.pone.0019926"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deleidi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hargus</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isacson</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Oct4-induced reprogramming is required for adult brain neural stem cell differentiation into midbrain dopaminergic neurons.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19926</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DDeleidi%26rft.auinit1%253DM.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19926%26rft.epage%253De19926%26rft.atitle%253DOct4-induced%2Breprogramming%2Bis%2Brequired%2Bfor%2Badult%2Bbrain%2Bneural%2Bstem%2Bcell%2Bdifferentiation%2Binto%2Bmidbrain%2Bdopaminergic%2Bneurons.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019926%26rft_id%253Dinfo%253Apmid%252F21655272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019926&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21655272&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.5" data-doi="10.1038/nature14222"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dixon</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selvaraj</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antosiewicz-Bourget</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Chromatin architecture reorganization during stem cell differentiation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">518</span>: <span class="cit-fpage">331</span>–<span class="cit-lpage">336</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D518%26rft.spage%253D331%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14222%26rft_id%253Dinfo%253Apmid%252F25693564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14222&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25693564&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.6" data-doi="10.1038/ncomms7363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elliott</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Intermediate DNA methylation is a conserved signature of genome regulation.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">6363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D6%26rft.spage%253D6363%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms7363%26rft_id%253Dinfo%253Apmid%252F25691127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms7363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25691127&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.7" data-doi="10.1038/nrg1349"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fazzari</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greally</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Epigenomics: beyond CpG islands.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">446</span>–<span class="cit-lpage">455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DFazzari%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253D446%26rft.epage%253D455%26rft.atitle%253DEpigenomics%253A%2Bbeyond%2BCpG%2Bislands.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1349%26rft_id%253Dinfo%253Apmid%252F15153997%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1349&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15153997&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221759700015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.8" data-doi="10.1007/s00005-006-0017-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gallant</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilkeson</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">ETS transcription factors and regulation of immunity.</span> <abbr class="cit-jnl-abbrev">Arch. Immunol. Ther. Exp. (Warsz.)</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">149</span>–<span class="cit-lpage">163</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArchivum%2Bimmunologiae%2Bet%2Btherapiae%2Bexperimentalis%26rft.stitle%253DArch%2BImmunol%2BTher%2BExp%2B%2528Warsz%2529%26rft.aulast%253DGallant%26rft.auinit1%253DS.%26rft.volume%253D54%26rft.issue%253D3%26rft.spage%253D149%26rft.epage%253D163%26rft.atitle%253DETS%2Btranscription%2Bfactors%2Band%2Bregulation%2Bof%2Bimmunity.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00005-006-0017-z%26rft_id%253Dinfo%253Apmid%252F16652219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00005-006-0017-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16652219&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.9" data-doi="10.1038/ncomms6945"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gascard</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bilenky</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sigaroudinia</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Epigenetic and transcriptional determinants of the human breast.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D6%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms6945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms6945&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.10" data-doi="10.1038/nature14252"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gjoneska</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfenning</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mathys</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quon</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kundaje</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Conserved epigenomic signals in mice and humans reveal immune basis of Alzheimer’s disease.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">518</span>: <span class="cit-fpage">365</span>–<span class="cit-lpage">369</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D518%26rft.spage%253D365%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14252%26rft_id%253Dinfo%253Apmid%252F25693568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14252&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25693568&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.11" data-doi="10.1016/j.neuron.2014.12.068"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guillot-Sestier</span>  <span class="cit-name-given-names">M.-V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doty</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gate</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">J.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leung</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Il10 Deficiency Rebalances Innate Immunity to Mitigate Alzheimer-Like Pathology.</span> <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">534</span>–<span class="cit-lpage">548</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D85%26rft.spage%253D534%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2014.12.068%26rft_id%253Dinfo%253Apmid%252F25619654%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2014.12.068&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25619654&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.12" data-doi="10.1038/nbt.1682"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coarfa</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagarajan</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Comparison of sequencing-based methods to profile DNA methylation and identification of monoallelic epigenetic modifications.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1097</span>–<span class="cit-lpage">1105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DHarris%26rft.auinit1%253DR.%2BA.%26rft.volume%253D28%26rft.issue%253D10%26rft.spage%253D1097%26rft.epage%253D1105%26rft.atitle%253DComparison%2Bof%2Bsequencing-based%2Bmethods%2Bto%2Bprofile%2BDNA%2Bmethylation%2Band%2Bidentification%2Bof%2Bmonoallelic%2Bepigenetic%2Bmodifications.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1682%26rft_id%253Dinfo%253Apmid%252F20852635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1682&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20852635&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282870500032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.13" data-doi="10.1016/j.molcel.2010.05.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heinz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benner</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spann</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertolino</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Simple Combinations of Lineage-Determining Transcription Factors Prime cis-Regulatory Elements Required for Macrophage and B Cell Identities.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">576</span>–<span class="cit-lpage">589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DHeinz%26rft.auinit1%253DS.%26rft.volume%253D38%26rft.issue%253D4%26rft.spage%253D576%26rft.epage%253D589%26rft.atitle%253DSimple%2Bcombinations%2Bof%2Blineage-determining%2Btranscription%2Bfactors%2Bprime%2Bcis-regulatory%2Belements%2Brequired%2Bfor%2Bmacrophage%2Band%2BB%2Bcell%2Bidentities.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2010.05.004%26rft_id%253Dinfo%253Apmid%252F20513432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2010.05.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20513432&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278448100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.14" data-doi="10.1038/ni.3102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heneka</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Golenbock</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Latz</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Innate immunity in Alzheimer’s disease.</span> <abbr class="cit-jnl-abbrev">Nat. Immunol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">229</span>–<span class="cit-lpage">236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BImmunol.%26rft.volume%253D16%26rft.spage%253D229%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fni.3102%26rft_id%253Dinfo%253Apmid%252F25689443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ni.3102&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25689443&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-web" id="cit-6.4.973.15"><div class="cit-metadata unstructured">Hindorff, L. A., J. MacArthur, J. Morales, H. A. Junkins, P. N. Hall <em>et al.</em>, 2014 A catalog of published genome-wide association studies. Available at: <a href="http://www.genome.gov/gwastudies">www.genome.gov/gwastudies</a>. Accessed: October 23, 2014.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hou</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panthier</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arnheiter</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Signaling and transcriptional regulation in the neural crest-derived melanocyte lineage: interactions between KIT and MITF.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">5379</span>–<span class="cit-lpage">5389</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DHou%26rft.auinit1%253DL%26rft.volume%253D127%26rft.issue%253D24%26rft.spage%253D5379%26rft.epage%253D5389%26rft.atitle%253DSignaling%2Band%2Btranscriptional%2Bregulation%2Bin%2Bthe%2Bneural%2Bcrest-derived%2Bmelanocyte%2Blineage%253A%2Binteractions%2Bbetween%2BKIT%2Band%2BMITF%26rft_id%253Dinfo%253Apmid%252F11076759%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTI3LzI0LzUzNzkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvOTczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.17" data-doi="10.1002/art.24450"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarvis</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frank</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knowlton</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aggarwal</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Gene expression profiling in neutrophils from children with polyarticular juvenile idiopathic arthritis.</span> <abbr class="cit-jnl-abbrev">Arthritis Rheum.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">1488</span>–<span class="cit-lpage">1495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArthritis%2Band%2Brheumatism%26rft.stitle%253DArthritis%2BRheum%26rft.aulast%253DJarvis%26rft.auinit1%253DJ.%2BN.%26rft.volume%253D60%26rft.issue%253D5%26rft.spage%253D1488%26rft.epage%253D1495%26rft.atitle%253DGene%2Bexpression%2Bprofiling%2Bin%2Bneutrophils%2Bfrom%2Bchildren%2Bwith%2Bpolyarticular%2Bjuvenile%2Bidiopathic%2Barthritis.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fart.24450%26rft_id%253Dinfo%253Apmid%252F19404961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/art.24450&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19404961&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266071700035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.18" data-doi="10.1016/j.cell.2013.04.041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Sperm, but not oocyte, DNA methylome is inherited by zebrafish early embryos.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">773</span>–<span class="cit-lpage">784</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D153%26rft.spage%253D773%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.04.041%26rft_id%253Dinfo%253Apmid%252F23663777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.04.041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23663777&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318844000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.19" data-doi="10.1101/gr.229102. Article published online before print in May 2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugnet</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furey</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roskin</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pringle</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The Human Genome Browser at UCSC.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">996</span>–<span class="cit-lpage">1006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.229102.%2BArticle%2Bpublished%2Bonline%2Bbefore%2Bprint%2Bin%2BMay%2B2002%26rft_id%253Dinfo%253Apmid%252F12045153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzYvOTk2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzk3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.20" data-doi="10.1101/gr.148023.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakurai</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miura</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imai</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mochiduki</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-resolution DNA methylome analysis of primordial germ cells identifies gender-specific reprogramming in mice.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">616</span>–<span class="cit-lpage">627</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.148023.112%26rft_id%253Dinfo%253Apmid%252F23410886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIzLzQvNjE2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzk3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kos</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reedy</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erickson</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The winged-helix transcription factor FoxD3 is important for establishing the neural crest lineage and repressing melanogenesis in avian embryos.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">1467</span>–<span class="cit-lpage">1479</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DKos%26rft.auinit1%253DR%26rft.volume%253D128%26rft.issue%253D8%26rft.spage%253D1467%26rft.epage%253D1479%26rft.atitle%253DThe%2Bwinged-helix%2Btranscription%2Bfactor%2BFoxD3%2Bis%2Bimportant%2Bfor%2Bestablishing%2Bthe%2Bneural%2Bcrest%2Blineage%2Band%2Brepressing%2Bmelanogenesis%2Bin%2Bavian%2Bembryos%26rft_id%253Dinfo%253Apmid%252F11262245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMDoiMTI4LzgvMTQ2NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC85NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.22" data-doi="10.1016/j.immuni.2014.12.021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koues</span>  <span class="cit-name-given-names">O. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kowalewski</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">L.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pyfrom</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Enhancer Sequence Variants and Transcription- Factor Deregulation Synergize to Construct Pathogenic Regulatory Circuits in B-Cell Lymphoma.</span> <abbr class="cit-jnl-abbrev">Immunity</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">186</span>–<span class="cit-lpage">198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunity%26rft.volume%253D42%26rft.spage%253D186%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.immuni.2014.12.021%26rft_id%253Dinfo%253Apmid%252F25607463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.immuni.2014.12.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25607463&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.23" data-doi="10.1111/j.1755-148X.2008.00514.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kubic</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plummer</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ludvik</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lang</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Pigmentation PAX-ways: the role of Pax3 in melanogenesis, melanocyte stem cell maintenance, and disease.</span> <abbr class="cit-jnl-abbrev">Pigment Cell Melanoma Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">627</span>–<span class="cit-lpage">645</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPigment%2BCell%2BMelanoma%2BRes%26rft.aulast%253DKubic%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D21%26rft.issue%253D6%26rft.spage%253D627%26rft.epage%253D645%26rft.atitle%253DPigmentation%2BPAX-ways%253A%2Bthe%2Brole%2Bof%2BPax3%2Bin%2Bmelanogenesis%252C%2Bmelanocyte%2Bstem%2Bcell%2Bmaintenance%252C%2Band%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-148X.2008.00514.x%26rft_id%253Dinfo%253Apmid%252F18983540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-148X.2008.00514.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18983540&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261079800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.24" data-doi="10.1038/nature14248"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kundaje</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuleman</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ernst</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bilenky</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Integrative analysis of 111 reference human epigenomes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">518</span>: <span class="cit-fpage">317</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D518%26rft.spage%253D317%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14248%26rft_id%253Dinfo%253Apmid%252F25693563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14248&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25693563&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.25" data-doi="10.1101/gr.101907.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laurent</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huynh</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsirigos</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Dynamic changes in the human methylome during differentiation.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">320</span>–<span class="cit-lpage">331</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.101907.109%26rft_id%253Dinfo%253Apmid%252F20133333%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzMvMzIwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzk3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.26" data-doi="10.1038/nrg2719"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Law</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobsen</span>  <span class="cit-name-given-names">S. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Establishing, maintaining and modifying DNA methylation patterns in plants and animals.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">204</span>–<span class="cit-lpage">220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DLaw%26rft.volume%253D11%26rft.issue%253D3%26rft.spage%253D204%26rft.epage%253D220%26rft.atitle%253DEstablishing%252C%2Bmaintaining%2Band%2Bmodifying%2BDNA%2Bmethylation%2Bpatterns%2Bin%2Bplants%2Band%2Banimals.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2719%26rft_id%253Dinfo%253Apmid%252F20142834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2719&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20142834&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274649900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.27" data-doi="10.1038/ncomms7315"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowdon</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maricque</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Developmental enhancers revealed by extensive DNA methylome maps of zebrafish early embryos.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">6315</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D6%26rft.spage%253D6315%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms7315%26rft_id%253Dinfo%253Apmid%252F25697895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms7315&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25697895&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.28" data-doi="10.1016/j.molmed.2006.07.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khaled</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">MITF: master regulator of melanocyte development and melanoma oncogene.</span> <abbr class="cit-jnl-abbrev">Trends Mol. Med.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">406</span>–<span class="cit-lpage">414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bmolecular%2Bmedicine%26rft.stitle%253DTrends%2BMol%2BMed%26rft.aulast%253DLevy%26rft.auinit1%253DC.%26rft.volume%253D12%26rft.issue%253D9%26rft.spage%253D406%26rft.epage%253D414%26rft.atitle%253DMITF%253A%2Bmaster%2Bregulator%2Bof%2Bmelanocyte%2Bdevelopment%2Band%2Bmelanoma%2Boncogene.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molmed.2006.07.008%26rft_id%253Dinfo%253Apmid%252F16899407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molmed.2006.07.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16899407&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240886400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Combining MeDIP-seq and MRE-seq to investigate genome-wide CpG methylation.</span> <abbr class="cit-jnl-abbrev">Methods</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%26rft.volume%253D72%26rft.spage%253D29%26rft_id%253Dinfo%253Apmid%252F25448294%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25448294&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.30" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC85NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.31" data-doi="10.1016/j.neuron.2007.01.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ligon</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huillard</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mehta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kesari</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Olig2-Regulated Lineage-Restricted Pathway Controls Replication Competence in Neural Stem Cells and Malignant Glioma.</span> <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">503</span>–<span class="cit-lpage">517</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D53%26rft.spage%253D503%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2007.01.009%26rft_id%253Dinfo%253Apmid%252F17296553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2007.01.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17296553&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245126900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.32" data-doi="10.1038/nature08514"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lister</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pelizzola</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dowen</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hawkins</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hon</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Human DNA methylomes at base resolution show widespread epigenomic differences.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">462</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">322</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLister%26rft.auinit1%253DR.%26rft.volume%253D462%26rft.issue%253D7271%26rft.spage%253D315%26rft.epage%253D322%26rft.atitle%253DHuman%2BDNA%2Bmethylomes%2Bat%2Bbase%2Bresolution%2Bshow%2Bwidespread%2Bepigenomic%2Bdifferences.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08514%26rft_id%253Dinfo%253Apmid%252F19829295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08514&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19829295&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271899300037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.33" data-doi="10.1126/science.1237905"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lister</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mukamel</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nery</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Urich</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puddifoot</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Global epigenomic reconfiguration during mammalian brain development.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">341</span>: <span class="cit-fpage">1237905</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1237905%26rft_id%253Dinfo%253Apmid%252F23828890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNDEvNjE0Ni8xMjM3OTA1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzk3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.34" data-doi="10.1038/ncomms6442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lowdon</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bilenky</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mauro</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Regulatory network decoded from epigenomes of surface ectoderm-derived cell types.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">5442</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D5442%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms6442%26rft_id%253Dinfo%253Apmid%252F25421844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms6442&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25421844&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.35" data-doi="10.1038/ncomms7514"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matsuda</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murao</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katano</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Juliandi</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, J. Kohyama <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">TLR9 signalling in microglia attenuates seizure-induced aberrant neurogenesis in the adult hippocampus.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">6514</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D6%26rft.spage%253D6514%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms7514%26rft_id%253Dinfo%253Apmid%252F25751136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms7514&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25751136&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.36" data-doi="10.1126/science.1222794"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maurano</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Humbert</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rynes</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thurman</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haugen</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Systematic Localization of Common Disease-Associated Variation in Regulatory DNA.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">1190</span>–<span class="cit-lpage">1195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1222794%26rft_id%253Dinfo%253Apmid%252F22955828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzcvNjA5OS8xMTkwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzk3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.37" data-doi="10.1038/nbt.1630"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McLean</span>  <span class="cit-name-given-names">C. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bristor</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hiller</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clarke</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaar</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">GREAT improves functional interpretation of cis-regulatory regions.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">495</span>–<span class="cit-lpage">501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DMcLean%26rft.auinit1%253DC.%2BY.%26rft.volume%253D28%26rft.issue%253D5%26rft.spage%253D495%26rft.epage%253D501%26rft.atitle%253DGREAT%2Bimproves%2Bfunctional%2Binterpretation%2Bof%2Bcis-regulatory%2Bregions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1630%26rft_id%253Dinfo%253Apmid%252F20436461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1630&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20436461&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277452700030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.38" data-doi="10.1111/j.0022-202X.2004.23558.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mehic</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bakiri</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghannadan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">E. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tschachler</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Fos and jun proteins are specifically expressed during differentiation of human keratinocytes.</span> <abbr class="cit-jnl-abbrev">J. Invest. Dermatol.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">212</span>–<span class="cit-lpage">220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BInvestigative%2BDermatology%26rft.stitle%253DJID%26rft.aulast%253DMehic%26rft.auinit1%253DD.%26rft.volume%253D124%26rft.issue%253D1%26rft.spage%253D212%26rft.epage%253D220%26rft.atitle%253DFos%2Band%2BJun%2BProteins%2BAre%2BSpecifically%2BExpressed%2BDuring%2BDifferentiation%2Bof%2BHuman%2BKeratinocytes%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0022-202X.2004.23558.x%26rft_id%253Dinfo%253Apmid%252F15654976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0022-202X.2004.23558.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15654976&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225951300034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.39" data-doi="10.1038/nature07107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meissner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mikkelsen</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wernig</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genome-scale DNA methylation maps of pluripotent and differentiated cells.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">454</span>: <span class="cit-fpage">766</span>–<span class="cit-lpage">770</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMeissner%26rft.auinit1%253DA.%26rft.volume%253D454%26rft.issue%253D7205%26rft.spage%253D766%26rft.epage%253D770%26rft.atitle%253DGenome-scale%2BDNA%2Bmethylation%2Bmaps%2Bof%2Bpluripotent%2Band%2Bdifferentiated%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07107%26rft_id%253Dinfo%253Apmid%252F18600261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07107&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18600261&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258228000045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mellgren</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A requirement for kit in embryonic zebrafish melanocyte differentiation is revealed by melanoblast delay.</span> <abbr class="cit-jnl-abbrev">Dev. Genes Evol.</abbr> <span class="cit-vol">214</span>: <span class="cit-fpage">493</span>–<span class="cit-lpage">502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%2Bgenes%2Band%2Bevolution%26rft.stitle%253DDev%2BGenes%2BEvol%26rft.aulast%253DMellgren%26rft.auinit1%253DE.%2BM.%26rft.volume%253D214%26rft.issue%253D10%26rft.spage%253D493%26rft.epage%253D502%26rft.atitle%253DA%2Brequirement%2Bfor%2Bkit%2Bin%2Bembryonic%2Bzebrafish%2Bmelanocyte%2Bdifferentiation%2Bis%2Brevealed%2Bby%2Bmelanoblast%2Bdelay.%26rft_id%253Dinfo%253Apmid%252F15300437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15300437&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.41" data-doi="10.1016/j.devcel.2004.08.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meulemans</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bronner-Fraser</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Gene-regulatory interactions in neural crest evolution and development.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DMeulemans%26rft.auinit1%253DD.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253D291%26rft.epage%253D299%26rft.atitle%253DGene-regulatory%2Binteractions%2Bin%2Bneural%2Bcrest%2Bevolution%2Band%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2004.08.007%26rft_id%253Dinfo%253Apmid%252F15363405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2004.08.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15363405&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224082900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.42" data-doi="10.1016/S0896-6273(01)00213-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moran-Rivard</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kagawa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saueressig</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burrill</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Evx1 is a postmitotic determinant of v0 interneuron identity in the spinal cord.</span> <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">385</span>–<span class="cit-lpage">399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.stitle%253DNeuron%26rft.aulast%253DMoran-Rivard%26rft.auinit1%253DL.%26rft.volume%253D29%26rft.issue%253D2%26rft.spage%253D385%26rft.epage%253D399%26rft.atitle%253DEvx1%2Bis%2Ba%2Bpostmitotic%2Bdeterminant%2Bof%2Bv0%2Binterneuron%2Bidentity%2Bin%2Bthe%2Bspinal%2Bcord.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0896-6273%252801%252900213-6%26rft_id%253Dinfo%253Apmid%252F11239430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0896-6273(01)00213-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11239430&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167141700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.43" data-doi="10.1038/emboj.2010.131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Po</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferretti</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miele</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Smaele</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paganelli</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Hedgehog controls neural stem cells through p53-independent regulation of Nanog.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">2646</span>–<span class="cit-lpage">2658</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DPo%26rft.auinit1%253DA.%26rft.volume%253D29%26rft.issue%253D15%26rft.spage%253D2646%26rft.epage%253D2658%26rft.atitle%253DHedgehog%2Bcontrols%2Bneural%2Bstem%2Bcells%2Bthrough%2Bp53-independent%2Bregulation%2Bof%2BNanog%26rft_id%253Dinfo%253Adoi%252F10.1038%252Femboj.2010.131%26rft_id%253Dinfo%253Apmid%252F20581804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjkvMTUvMjY0NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC85NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.44" data-doi="10.1038/nbt.1685"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Portela</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esteller</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Epigenetic modifications and human disease.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1057</span>–<span class="cit-lpage">1068</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DPortela%26rft.auinit1%253DA.%26rft.volume%253D28%26rft.issue%253D10%26rft.spage%253D1057%26rft.epage%253D1068%26rft.atitle%253DEpigenetic%2Bmodifications%2Band%2Bhuman%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1685%26rft_id%253Dinfo%253Apmid%252F20944598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1685&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20944598&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282870500028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.45" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi82Lzg0MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC85NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.46" data-doi="10.1038/nrg1602"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reik</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Co-evolution of X-chromosome inactivation and imprinting in mammals.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DReik%26rft.auinit1%253DW.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DCo-evolution%2Bof%2BX-chromosome%2Binactivation%2Band%2Bimprinting%2Bin%2Bmammals.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1602%26rft_id%253Dinfo%253Apmid%252F15818385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1602&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15818385&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228862900015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.47" data-doi="10.1038/nrm1228"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">K. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">DNA methylation and human disease.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">597</span>–<span class="cit-lpage">610</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DRobertson%26rft.auinit1%253DK.%2BD.%26rft.volume%253D6%26rft.issue%253D8%26rft.spage%253D597%26rft.epage%253D610%26rft.atitle%253DDNA%2Bmethylation%2Band%2Bhuman%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm1228%26rft_id%253Dinfo%253Apmid%252F16136652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm1228&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16136652&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230880700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.48" data-doi="10.1016/j.ejca.2010.02.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schröder</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schumacher</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wirtz</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Streichert</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The transcription factor Fra-2 promotes mammary tumour progression by changing the adhesive properties of breast cancer cells.</span> <abbr class="cit-jnl-abbrev">Eur. J. Cancer</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1650</span>–<span class="cit-lpage">1660</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuropean%2Bjournal%2Bof%2Bcancer%2B%2528Oxford%252C%2BEngland%2B%253A%2B%2B1990%2529%26rft.stitle%253DEur%2BJ%2BCancer%26rft.aulast%253DSchroder%26rft.auinit1%253DC.%26rft.volume%253D46%26rft.issue%253D9%26rft.spage%253D1650%26rft.epage%253D1660%26rft.atitle%253DThe%2Btranscription%2Bfactor%2BFra-2%2Bpromotes%2Bmammary%2Btumour%2Bprogression%2Bby%2Bchanging%2Bthe%2Badhesive%2Bproperties%2Bof%2Bbreast%2Bcancer%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ejca.2010.02.008%26rft_id%253Dinfo%253Apmid%252F20226654%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ejca.2010.02.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20226654&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.49" data-doi="10.1038/nature14465"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schultz</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitaker</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hariharan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mukamel</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Human body epigenome maps reveal noncanonical DNA methylation variation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">523</span>: <span class="cit-fpage">212</span>–<span class="cit-lpage">216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D523%26rft.spage%253D212%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14465%26rft_id%253Dinfo%253Apmid%252F26030523%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14465&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26030523&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.50" data-doi="10.1105/tpc.111.094870"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-Wide Analysis of DNA Methylation and Gene Expression Changes in Two Arabidopsis Ecotypes and Their Reciprocal Hybrids.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">875</span>–<span class="cit-lpage">892</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.111.094870%26rft_id%253Dinfo%253Apmid%252F22438023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjI0LzMvODc1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzk3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.51" data-doi="10.1038/nature10716"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stadler</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murr</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burger</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ivanek</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lienert</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">DNA-binding factors shape the mouse methylome at distal regulatory regions.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">480</span>: <span class="cit-fpage">490</span>–<span class="cit-lpage">495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DStadler%26rft.auinit1%253DM.%2BB.%26rft.volume%253D480%26rft.issue%253D7378%26rft.spage%253D490%26rft.epage%253D495%26rft.atitle%253DDNA-binding%2Bfactors%2Bshape%2Bthe%2Bmouse%2Bmethylome%2Bat%2Bdistal%2Bregulatory%2Bregions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10716%26rft_id%253Dinfo%253Apmid%252F22170606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10716&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22170606&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298318000053&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.52" data-doi="10.1101/gr.152231.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Estimating absolute methylation levels at single-CpG resolution from methylation enrichment and restriction enzyme sequencing methods.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1541</span>–<span class="cit-lpage">1553</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.152231.112%26rft_id%253Dinfo%253Apmid%252F23804401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIzLzkvMTU0MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC85NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.53" data-doi="10.1016/j.ydbio.2005.12.035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arduini</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berghmans</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">George</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanki</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Zebrafish foxd3 is selectively required for neural crest specification, migration and survival.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">292</span>: <span class="cit-fpage">174</span>–<span class="cit-lpage">188</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DStewart%26rft.auinit1%253DR.%2BA.%26rft.volume%253D292%26rft.issue%253D1%26rft.spage%253D174%26rft.epage%253D188%26rft.atitle%253DZebrafish%2Bfoxd3%2Bis%2Bselectively%2Brequired%2Bfor%2Bneural%2Bcrest%2Bspecification%252C%2Bmigration%2Band%2Bsurvival.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2005.12.035%26rft_id%253Dinfo%253Apmid%252F16499899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2005.12.035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16499899&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236727100016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.54" data-doi="10.1073/pnas.1101109108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Subramanian</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarkar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shetty</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muralidharan</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Padmanabhan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Transcription factor Lhx2 is necessary and sufficient to suppress astrogliogenesis and promote neurogenesis in the developing hippocampus.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">E265</span>–<span class="cit-lpage">E274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1101109108%26rft_id%253Dinfo%253Apmid%252F21690374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzI3L0UyNjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvOTczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.55" data-doi="10.1016/j.ydbio.2008.10.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suster</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kania</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asakawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charron</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A novel conserved evx1 enhancer links spinal interneuron morphology and cis-regulation from fish to mammals.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">325</span>: <span class="cit-fpage">422</span>–<span class="cit-lpage">433</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DSuster%26rft.auinit1%253DM.%2BL.%26rft.volume%253D325%26rft.issue%253D2%26rft.spage%253D422%26rft.epage%253D433%26rft.atitle%253DA%2Bnovel%2Bconserved%2Bevx1%2Benhancer%2Blinks%2Bspinal%2Binterneuron%2Bmorphology%2Band%2Bcis-regulation%2Bfrom%2Bfish%2Bto%2Bmammals.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2008.10.004%26rft_id%253Dinfo%253Apmid%252F18992237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2008.10.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18992237&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262589400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.4.973.56" data-doi="10.1038/nature11361"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">ENCODE Project Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An integrated encyclopedia of DNA elements in the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">488</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPeyronel%26rft.auinit1%253DT.%26rft.volume%253D488%26rft.issue%253D7409%26rft.spage%253D57%26rft.epage%253D60%26rft.atitle%253DQuantum%2Bnonlinear%2Boptics%2Bwith%2Bsingle%2Bphotons%2Benabled%2Bby%2Bstrongly%2Binteracting%2Batoms.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11361%26rft_id%253Dinfo%253Apmid%252F22832584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11361&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22832584&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307010700031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.57" data-doi="10.1242/dev.031989"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erickson</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">FOXD3 regulates the lineage switch between neural crest-derived glial cells and pigment cells by repressing MITF through a non-canonical mechanism.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">1849</span>–<span class="cit-lpage">1858</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.031989%26rft_id%253Dinfo%253Apmid%252F19403660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTM2LzExLzE4NDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvOTczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.58" data-doi="10.1038/nature14233"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsankov</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akopian</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ziller</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Transcription factor binding dynamics during human ES cell differentiation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">518</span>: <span class="cit-fpage">344</span>–<span class="cit-lpage">349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D518%26rft.spage%253D344%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14233%26rft_id%253Dinfo%253Apmid%252F25693565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25693565&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.59" data-doi="10.1093/nar/gkl822"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minovitsky</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubchak</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pennacchio</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">VISTA Enhancer Browser–a database of tissue-specific human enhancers.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">D88</span>–<span class="cit-lpage">D92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl822%26rft_id%253Dinfo%253Apmid%252F17130149%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE0OiIzNS9zdXBwbF8xL0Q4OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC85NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.60" data-doi="10.1093/nar/gkv151"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szulwach</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Street</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Base-resolution methylation patterns accurately predict transcription factor bindings in vivo.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">2757</span>–<span class="cit-lpage">2766</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv151%26rft_id%253Dinfo%253Apmid%252F25722376%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjQzLzUvMjc1NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC85NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.61" data-doi="10.1016/j.molcel.2008.11.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishimura</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Inhibition of PAX3 by TGF-b Modulates Melanocyte Viability.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">554</span>–<span class="cit-lpage">563</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DYang%26rft.auinit1%253DG.%26rft.volume%253D32%26rft.issue%253D4%26rft.spage%253D554%26rft.epage%253D563%26rft.atitle%253DInhibition%2Bof%2BPAX3%2Bby%2BTGF-beta%2Bmodulates%2Bmelanocyte%2Bviability.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2008.11.002%26rft_id%253Dinfo%253Apmid%252F19026785%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2008.11.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19026785&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261290000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.62" data-doi="10.1046/j.0022-202x.2001.00006.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yoshida</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunisada</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grimm</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishimura</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishioka</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Review: melanocyte migration and survival controlled by SCF/c-kit expression.</span> <abbr class="cit-jnl-abbrev">J. Investig. Dermatol. Symp. Proc.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2Bjournal%2Bof%2Binvestigative%2Bdermatology.%2BSymposium%2Bproceedings%2B%252F%2Bthe%2BSociety%2Bfor%2BInvestigative%2BDermatology%252C%2BInc.%2B%255Band%255D%2BEuropean%2BSociety%2Bfor%2BDermatological%2BResearch%26rft.stitle%253DJ%2BInvestig%2BDermatol%2BSymp%2BProc%26rft.aulast%253DYoshida%26rft.auinit1%253DH.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D5%26rft.atitle%253DReview%253A%2Bmelanocyte%2Bmigration%2Band%2Bsurvival%2Bcontrolled%2Bby%2BSCF%252Fc-kit%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.0022-202x.2001.00006.x%26rft_id%253Dinfo%253Apmid%252F11764276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.0022-202x.2001.00006.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11764276&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172851500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.63" data-doi="10.1093/bioinformatics/btp340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schones</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A clustering approach for identification of enriched domains from histone modification ChIP-Seq data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1952</span>–<span class="cit-lpage">1958</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp340%26rft_id%253Dinfo%253Apmid%252F19505939%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTUvMTk1MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC85NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.64" data-doi="10.1038/emboj.2010.137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zbinden</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duquet</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorente-Trigos</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ngwabyt</span>  <span class="cit-name-given-names">S.-N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">NANOG regulates glioma stem cells and is essential in vivo acting in a cross-functional network with GLI1 and p53.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">2659</span>–<span class="cit-lpage">2674</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DZbinden%26rft.auinit1%253DM.%26rft.volume%253D29%26rft.issue%253D15%26rft.spage%253D2659%26rft.epage%253D2674%26rft.atitle%253DNANOG%2Bregulates%2Bglioma%2Bstem%2Bcells%2Band%2Bis%2Bessential%2Bin%2Bvivo%2Bacting%2Bin%2Ba%2Bcross-functional%2Bnetwork%2Bwith%2BGLI1%2Band%2Bp53%26rft_id%253Dinfo%253Adoi%252F10.1038%252Femboj.2010.137%26rft_id%253Dinfo%253Apmid%252F20581802%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjkvMTUvMjY1OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC85NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.65" data-doi="10.1101/gr.156539.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowdon</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Functional DNA methylation differences between tissues, cell types, and across individuals discovered using the M&amp;M algorithm.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1522</span>–<span class="cit-lpage">1540</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.156539.113%26rft_id%253Dinfo%253Apmid%252F23804400%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIzLzkvMTUyMiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC85NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.66" data-doi="10.1186/1471-2164-15-349"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowdon</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comparative DNA methylome analysis of endometrial carcinoma reveals complex and distinct deregulation of cancer promoters and enhancers.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-349%26rft_id%253Dinfo%253Apmid%252F24382143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-349&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24382143&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.67" data-doi="10.1128/MCB.15.12.6961"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eddy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teng</span>  <span class="cit-name-given-names">Y. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritzler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kluppel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">An immunological renal disease in transgenic mice that overexpress Fli-1, a member of the ets family of transcription factor genes.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">6961</span>–<span class="cit-lpage">6970</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DZhang%26rft.auinit1%253DL.%26rft.volume%253D15%26rft.issue%253D12%26rft.spage%253D6961%26rft.epage%253D6970%26rft.atitle%253DAn%2Bimmunological%2Brenal%2Bdisease%2Bin%2Btransgenic%2Bmice%2Bthat%2Boverexpress%2BFli-%2B1%252C%2Ba%2Bmember%2Bof%2Bthe%2Bets%2Bfamily%2Bof%2Btranscription%2Bfactor%2Bgenes%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.15.12.6961%26rft_id%253Dinfo%253Apmid%252F8524263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xMi82OTYxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzk3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.68" data-doi="10.1038/nmeth.1772"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maricque</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sundaram</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The Human Epigenome Browser at Washington University.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">989</span>–<span class="cit-lpage">990</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DZhou%26rft.auinit1%253DX.%26rft.volume%253D8%26rft.issue%253D12%26rft.spage%253D989%26rft.epage%253D990%26rft.atitle%253DThe%2BHuman%2BEpigenome%2BBrowser%2Bat%2BWashington%2BUniversity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1772%26rft_id%253Dinfo%253Apmid%252F22127213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1772&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22127213&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000297931700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowdon</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockweiler</span>  <span class="cit-name-given-names">N. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Epigenomic annotation of genetic variants using the Roadmap Epigenome Browser.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">345</span>–<span class="cit-lpage">346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D345%26rft_id%253Dinfo%253Apmid%252F25690851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25690851&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.70" data-doi="10.1038/nature12433"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ziller</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donaghey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">L. T. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Charting a dynamic DNA methylation landscape of the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">500</span>: <span class="cit-fpage">477</span>–<span class="cit-lpage">481</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D500%26rft.spage%253D477%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12433%26rft_id%253Dinfo%253Apmid%252F23925113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12433&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23925113&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000323316100039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.973.71"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ziller</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edri</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yaffe</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donaghey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Dissecting neural differentiation regulatory networks through epigenetic footprinting.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">518</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D518%26rft.spage%253D355%26rft_id%253Dinfo%253Apmid%252F25533951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25533951&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F973.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/4/973.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/4/957" title="Whole Genome Comparison Reveals High Levels of Inbreeding and Strain Redundancy Across the Spectrum of Commercial Wine Strains of Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/4/987" title="A HAND2 Loss-of-Function Mutation Causes Familial Ventricular Septal Defect and Pulmonary Stenosis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/4"><span>Volume 6 Issue 4, April 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/4.cover-source.jpg?itok=Lj45nPaf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/973.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1QUaStucQNW3OnZXkRiq-VdvApWomyfBoJCr5ojxBaI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/4/973" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/4/973" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/4/973" class="active" data-icon-position="" data-hide-link-title="0">Mapping of Variable DNA Methylation Across Multiple Cell Types Defines a Dynamic Regulatory Landscape of the Human Genome</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460115" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-D-4EBjlNp229dsafk6DuOtOSHgGD3KesF3jCevtFDdM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/4/973.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/4/973" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-4uTAgrkJcEXTnN65h4eku0K-RxaVHxemOwxA3yPjGfA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/4/973" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/4/973.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460115" id="node460115--2" data-pisa="ggg;6/4/973" data-pisa-master="ggg;g3.115.025437" data-apath="/ggg/6/4/973.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mapping of Variable DNA Methylation Across Multiple Cell Types Defines a Dynamic Regulatory Landscape of the Human Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Junchen Gu</span>, <span class="highwire-citation-author" data-delta="1">Michael Stevens</span>, <span class="highwire-citation-author" data-delta="2">Xiaoyun Xing</span>, <span class="highwire-citation-author" data-delta="3">Daofeng Li</span>, <span class="highwire-citation-author" data-delta="4">Bo Zhang</span>, <span class="highwire-citation-author" data-delta="5">Jacqueline E. Payton</span>, <span class="highwire-citation-author" data-delta="6">Eugene M. Oltz</span>, <span class="highwire-citation-author" data-delta="7">James N. Jarvis</span>, <span class="highwire-citation-author" data-delta="8">Kaiyu Jiang</span>, <span class="highwire-citation-author" data-delta="9">Theodore Cicero</span>, <span class="highwire-citation-author" data-delta="10">Joseph F. Costello</span> and <span class="highwire-citation-author" data-delta="11">Ting Wang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">973-986;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.025437 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460115--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Junchen Gu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St. Louis, Missouri 63108 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Junchen%2BGu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJunchen%2BGu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michael Stevens </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St. Louis, Missouri 63108 </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Computer Science and Engineering, Washington University in St. Louis, Missouri 63130</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BStevens%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stevens%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BStevens%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Xiaoyun Xing </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St. Louis, Missouri 63108 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaoyun%2BXing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xing%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaoyun%2BXing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Daofeng Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St. Louis, Missouri 63108 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daofeng%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaofeng%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Bo Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Developmental Biology, Washington University School of Medicine, St. Louis, Missouri 63108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bo%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABo%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jacqueline E. Payton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Pathology and Immunology, Washington University School of Medicine, St. Louis, Missouri 63110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacqueline%2BE.%2BPayton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Payton%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJacqueline%2BE.%2BPayton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Eugene M. Oltz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Pathology and Immunology, Washington University School of Medicine, St. Louis, Missouri 63110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eugene%2BM.%2BOltz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oltz%20EM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEugene%2BM.%2BOltz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">James N. Jarvis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Pediatrics, University of Buffalo, New York 14203 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BN.%2BJarvis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarvis%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BN.%2BJarvis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Kaiyu Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Pediatrics, University of Buffalo, New York 14203 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kaiyu%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaiyu%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Theodore Cicero </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Psychiatry, Washington University School of Medicine, St. Louis, Missouri 63110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Theodore%2BCicero%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cicero%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATheodore%2BCicero%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Joseph F. Costello </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Brain Tumor Research Center, Department of Neurosurgery, Helen Diller Family Comprehensive Cancer Center, University of California, San Francisco, California 94143</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BF.%2BCostello%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Costello%20JF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BF.%2BCostello%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Ting Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St. Louis, Missouri 63108 </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Computer Science and Engineering, Washington University in St. Louis, Missouri 63130</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ting%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATing%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:twang@genetics.wustl.edu" class="" data-icon-position="" data-hide-link-title="0">twang@genetics.wustl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/4/973" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/4/973&amp;title=Mapping%20of%20Variable%20DNA%20Methylation%20Across%20Multiple%20Cell%20Types%20Defines%20a%20Dynamic%20Regulatory%20Landscape%20of%20the%20Human%20Genome" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/4/973&amp;title=Mapping%20of%20Variable%20DNA%20Methylation%20Across%20Multiple%20Cell%20Types%20Defines%20a%20Dynamic%20Regulatory%20Landscape%20of%20the%20Human%20Genome" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/4/973&amp;title=Mapping%20of%20Variable%20DNA%20Methylation%20Across%20Multiple%20Cell%20Types%20Defines%20a%20Dynamic%20Regulatory%20Landscape%20of%20the%20Human%20Genome" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/4/973&amp;text=Mapping%20of%20Variable%20DNA%20Methylation%20Across%20Multiple%20Cell%20Types%20Defines%20a%20Dynamic%20Regulatory%20Landscape%20of%20the%20Human%20Genome" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/4/973&amp;title=Mapping%20of%20Variable%20DNA%20Methylation%20Across%20Multiple%20Cell%20Types%20Defines%20a%20Dynamic%20Regulatory%20Landscape%20of%20the%20Human%20Genome" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/4/973&amp;t=Mapping%20of%20Variable%20DNA%20Methylation%20Across%20Multiple%20Cell%20Types%20Defines%20a%20Dynamic%20Regulatory%20Landscape%20of%20the%20Human%20Genome" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/4/973&amp;title=Mapping%20of%20Variable%20DNA%20Methylation%20Across%20Multiple%20Cell%20Types%20Defines%20a%20Dynamic%20Regulatory%20Landscape%20of%20the%20Human%20Genome" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/4/973&amp;title=Mapping%20of%20Variable%20DNA%20Methylation%20Across%20Multiple%20Cell%20Types%20Defines%20a%20Dynamic%20Regulatory%20Landscape%20of%20the%20Human%20Genome" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460115/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460115" id="node460115--3" data-pisa="ggg;6/4/973" data-pisa-master="ggg;g3.115.025437" data-apath="/ggg/6/4/973.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mapping of Variable DNA Methylation Across Multiple Cell Types Defines a Dynamic Regulatory Landscape of the Human Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Junchen Gu</span>, <span class="highwire-citation-author" data-delta="1">Michael Stevens</span>, <span class="highwire-citation-author" data-delta="2">Xiaoyun Xing</span>, <span class="highwire-citation-author" data-delta="3">Daofeng Li</span>, <span class="highwire-citation-author" data-delta="4">Bo Zhang</span>, <span class="highwire-citation-author" data-delta="5">Jacqueline E. Payton</span>, <span class="highwire-citation-author" data-delta="6">Eugene M. Oltz</span>, <span class="highwire-citation-author" data-delta="7">James N. Jarvis</span>, <span class="highwire-citation-author" data-delta="8">Kaiyu Jiang</span>, <span class="highwire-citation-author" data-delta="9">Theodore Cicero</span>, <span class="highwire-citation-author" data-delta="10">Joseph F. Costello</span> and <span class="highwire-citation-author" data-delta="11">Ting Wang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">973-986;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.025437 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460115--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Junchen Gu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St. Louis, Missouri 63108 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Junchen%2BGu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJunchen%2BGu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michael Stevens </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St. Louis, Missouri 63108 </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Computer Science and Engineering, Washington University in St. Louis, Missouri 63130</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BStevens%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stevens%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BStevens%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Xiaoyun Xing </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St. Louis, Missouri 63108 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaoyun%2BXing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xing%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaoyun%2BXing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Daofeng Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St. Louis, Missouri 63108 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daofeng%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaofeng%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Bo Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Developmental Biology, Washington University School of Medicine, St. Louis, Missouri 63108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bo%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABo%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jacqueline E. Payton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Pathology and Immunology, Washington University School of Medicine, St. Louis, Missouri 63110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacqueline%2BE.%2BPayton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Payton%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJacqueline%2BE.%2BPayton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Eugene M. Oltz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Pathology and Immunology, Washington University School of Medicine, St. Louis, Missouri 63110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eugene%2BM.%2BOltz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oltz%20EM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEugene%2BM.%2BOltz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">James N. Jarvis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Pediatrics, University of Buffalo, New York 14203 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BN.%2BJarvis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarvis%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BN.%2BJarvis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Kaiyu Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Pediatrics, University of Buffalo, New York 14203 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kaiyu%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaiyu%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Theodore Cicero </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Psychiatry, Washington University School of Medicine, St. Louis, Missouri 63110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Theodore%2BCicero%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cicero%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATheodore%2BCicero%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Joseph F. Costello </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Brain Tumor Research Center, Department of Neurosurgery, Helen Diller Family Comprehensive Cancer Center, University of California, San Francisco, California 94143</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BF.%2BCostello%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Costello%20JF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BF.%2BCostello%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Ting Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St. Louis, Missouri 63108 </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Computer Science and Engineering, Washington University in St. Louis, Missouri 63130</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ting%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATing%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:twang@genetics.wustl.edu" class="" data-icon-position="" data-hide-link-title="0">twang@genetics.wustl.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=973&amp;rft.epage=986&amp;rft.atitle=Mapping%20of%20Variable%20DNA%20Methylation%20Across%20Multiple%20Cell%20Types%20Defines%20a%20Dynamic%20Regulatory%20Landscape%20of%20the%20Human%20Genome&amp;rft.volume=6&amp;rft.issue=4&amp;rft.date=2016-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Gu%2C+Junchen&amp;rft.au=Stevens%2C+Michael&amp;rft.au=Xing%2C+Xiaoyun&amp;rft.au=Li%2C+Daofeng&amp;rft.au=Zhang%2C+Bo&amp;rft.au=Payton%2C+Jacqueline+E.&amp;rft.au=Oltz%2C+Eugene+M.&amp;rft.au=Jarvis%2C+James+N.&amp;rft.au=Jiang%2C+Kaiyu&amp;rft.au=Cicero%2C+Theodore&amp;rft.au=Costello%2C+Joseph+F.&amp;rft.au=Wang%2C+Ting"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460115/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460115/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460115/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460115/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460115/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460115/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460115/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460115/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460115/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460115/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460115/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460115/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/4/973" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-24" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/4/973.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/4/973.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/4/973.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/4/973.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
