<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/4/845" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/4/845.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/4/845.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/4/845.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/4/845.atom" />
<meta name="HW.pisa" content="ggg;6/4/845" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Increasing Genome Sampling and Improving SNP Genotyping for Genotyping-by-Sequencing with New Combinations of Restriction Enzymes" />
<meta name="DC.Identifier" content="10.1534/g3.115.025775" />
<meta name="DC.Date" content="2016-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Fu et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genotyping-by-sequencing (GBS) has emerged as a useful genomic approach for exploring genome-wide genetic variation. However, GBS commonly samples a genome unevenly and can generate a substantial amount of missing data. These technical features would limit the power of various GBS-based genetic and genomic analyses. Here we present software called IgCoverage for in silico evaluation of genomic coverage through GBS with an individual or pair of restriction enzymes on one sequenced genome, and report a new set of 21 restriction enzyme combinations that can be applied to enhance GBS applications. These enzyme combinations were developed through an application of IgCoverage on 22 plant, animal, and fungus species with sequenced genomes, and some of them were empirically evaluated with different runs of Illumina MiSeq sequencing in 12 plant species. The in silico analysis of 22 organisms revealed up to eight times more genome coverage for the new combinations consisted of pairing four- or five-cutter restriction enzymes than the commonly used enzyme combination Pst I + Msp I. The empirical evaluation of the new enzyme combination ( Hin fI + Hpy CH4IV) in 12 plant species showed 1.7–6 times more genome coverage than Pst I + Msp I, and 2.3 times more genome coverage in dicots than monocots. Also, the SNP genotyping in 12 Arabidopsis and 12 rice plants revealed that Hin fI + Hpy CH4IV generated 7 and 1.3 times more SNPs (with 0–16.7% missing observations) than Pst I + Msp I, respectively. These findings demonstrate that these novel enzyme combinations can be utilized to increase genome sampling and improve SNP genotyping in various GBS applications." />
<meta name="DC.Contributor" content="Yong-Bi Fu" />
<meta name="DC.Contributor" content="Gregory W. Peterson" />
<meta name="DC.Contributor" content="Yibo Dong" />
<meta name="article:published_time" content="2016-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Increasing Genome Sampling and Improving SNP Genotyping for Genotyping-by-Sequencing with New Combinations of Restriction Enzymes" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genotyping-by-sequencing (GBS) has emerged as a useful genomic approach for exploring genome-wide genetic variation. However, GBS commonly samples a genome unevenly and can generate a substantial amount of missing data. These technical features would limit the power of various GBS-based genetic and genomic analyses. Here we present software called IgCoverage for &lt;i&gt;in silico&lt;/i&gt; evaluation of genomic coverage through GBS with an individual or pair of restriction enzymes on one sequenced genome, and report a new set of 21 restriction enzyme combinations that can be applied to enhance GBS applications. These enzyme combinations were developed through an application of IgCoverage on 22 plant, animal, and fungus species with sequenced genomes, and some of them were empirically evaluated with different runs of Illumina MiSeq sequencing in 12 plant species. The &lt;i&gt;in silico&lt;/i&gt; analysis of 22 organisms revealed up to eight times more genome coverage for the new combinations consisted of pairing four- or five-cutter restriction enzymes than the commonly used enzyme combination &lt;i&gt;Pst&lt;/i&gt;I + &lt;i&gt;Msp&lt;/i&gt;I. The empirical evaluation of the new enzyme combination (&lt;i&gt;Hin&lt;/i&gt;fI + &lt;i&gt;Hpy&lt;/i&gt;CH4IV) in 12 plant species showed 1.7–6 times more genome coverage than &lt;i&gt;Pst&lt;/i&gt;I + &lt;i&gt;Msp&lt;/i&gt;I, and 2.3 times more genome coverage in dicots than monocots. Also, the SNP genotyping in 12 &lt;i&gt;Arabidopsis&lt;/i&gt; and 12 rice plants revealed that &lt;i&gt;Hin&lt;/i&gt;fI + &lt;i&gt;Hpy&lt;/i&gt;CH4IV generated 7 and 1.3 times more SNPs (with 0–16.7% missing observations) than &lt;i&gt;Pst&lt;/i&gt;I + &lt;i&gt;Msp&lt;/i&gt;I, respectively. These findings demonstrate that these novel enzyme combinations can be utilized to increase genome sampling and improve SNP genotyping in various GBS applications.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/04/01" />
<meta name="citation_mjid" content="ggg;6/4/845" />
<meta name="citation_id" content="6/4/845" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/4/845" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/4/845.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/4/845.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/4/845.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.025775" />
<meta name="citation_pmid" content="26818077" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="845" />
<meta name="citation_lastpage" content="856" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Yong-Bi Fu" />
<meta name="citation_author_institution" content="Plant Gene Resources of Canada, Saskatoon Research and Development Centre, Agriculture and Agri-Food Canada, Saskatoon, Saskatchewan S7N 0X2, Canada" />
<meta name="citation_author_email" content="yong-bi.fu@agr.gc.ca" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3106-1247" />
<meta name="citation_author" content="Gregory W. Peterson" />
<meta name="citation_author_institution" content="Plant Gene Resources of Canada, Saskatoon Research and Development Centre, Agriculture and Agri-Food Canada, Saskatoon, Saskatchewan S7N 0X2, Canada" />
<meta name="citation_author" content="Yibo Dong" />
<meta name="citation_author_institution" content="Plant Gene Resources of Canada, Saskatoon Research and Development Centre, Agriculture and Agri-Food Canada, Saskatoon, Saskatchewan S7N 0X2, Canada" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. Altshuler;citation_author=V. J. Pollara;citation_author=C. R. Cowles;citation_author=W. J. van Etten;citation_author=J. Baldwin;citation_title=An SNP map of the human genome generated by reduced representation shotgun sequencing.;citation_pages=513-516;citation_volume=407;citation_year=2000;citation_issue=6803;citation_pmid=11029002;citation_doi=10.1038/35035083" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=K. R. Andrews;citation_author=P. A. Hohenlohe;citation_author=M. R. Miller;citation_author=B. K. Hand;citation_author=J. E. Seeb;citation_title=Trade-offs and utility of alternative RADseq methods.;citation_pages=5943-5946;citation_volume=23;citation_year=2014;citation_doi=10.1111/mec.12964" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. M. Beissinger;citation_author=C. N. Hirsch;citation_author=R. S. Sekhon;citation_author=J. M. Foerster;citation_author=J. M. Johnson;citation_title=Marker density and read depth for genotyping populations using genotyping-by-sequencing.;citation_volume=193;citation_year=2013;citation_pmid=23410831;citation_doi=10.1534/genetics.112.147710" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: A flexible trimmer for Illumina sequence data.;citation_volume=30;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_title=Mach. Learn.;citation_author=L. Breiman;citation_title=Random forests.;citation_pages=5-32;citation_volume=45;citation_year=2001;citation_doi=10.1023/A:1010933404324" />
<meta name="citation_reference" content="citation_title=Multiple imputation of unordered categorical data, in multiple imputation and its application;citation_author=J. R. Carpenter;citation_author=M. G. Kenward;citation_year=2013;citation_doi=10.1002/9781119942283.ch5." />
<meta name="citation_reference" content="citation_journal_abbrev=Front Genet;citation_author=J. E. Crawford;citation_author=B. P. Lazzaro;citation_title=Assessing the accuracy and power of population genetic inference from low-pass next-generation sequencing data.;citation_pages=66-66;citation_volume=3;citation_year=2012;citation_pmid=22536207" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. W. Davey;citation_author=P. A. Hohenlohe;citation_author=P. D. Etter;citation_author=J. Q. Boone;citation_author=D. Thureen;citation_title=Genome-wide genetic marker discovery and genotyping using next-generation sequencing.;citation_pages=499-510;citation_volume=12;citation_year=2011;citation_issue=7;citation_pmid=21681211;citation_doi=10.1038/nrg3012" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. De Donato;citation_author=S. O. Peters;citation_author=S. E. Mitchell;citation_author=T. Hussain;citation_author=I. G. Imumorin;citation_title=Genotyping-by-sequencing (GBS): a novel, efficient and cost-effective genotyping method for cattle using next-generation sequencing.;citation_pages=e62137;citation_volume=8;citation_year=2013;citation_pmid=23690931;citation_doi=10.1371/journal.pone.0062137" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=Y. B. Fu;citation_title=Genetic diversity analysis of highly incomplete SNP genotype data with imputations: an empirical assessment.;citation_volume=4;citation_year=2014;citation_pmid=24626289;citation_doi=10.1534/g3.114.010942" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=Y. B. Fu;citation_author=G. W. Peterson;citation_title=Genetic diversity analysis with 454 pyrosequencing and genomic reduction confirmed the eastern and western division in the cultivated barley gene pool.;citation_pages=226-237;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0022" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=Y. B. Fu;citation_author=G. W. Peterson;citation_title=Developing genomic resources in two Linum species via 454 pyrosequencing and genomic reduction.;citation_pages=492-500;citation_volume=12;citation_year=2012;citation_pmid=22177006;citation_doi=10.1111/j.1755-0998.2011.03100.x" />
<meta name="citation_reference" content="citation_journal_title=Genet. Resour. Crop Evol.;citation_author=Y. B. Fu;citation_author=B. Cheng;citation_author=G. W. Peterson;citation_title=Genetic diversity analysis of yellow mustard (Sinapis alba L.) germplasm based on genotyping by sequencing.;citation_pages=579-594;citation_volume=61;citation_year=2014;citation_doi=10.1007/s10722-013-0058-1" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=G. Haberer;citation_author=S. Young;citation_author=A. K. Bharti;citation_author=H. Gundlach;citation_author=C. Raymond;citation_title=Structure and Architecture of the Maize Genome;citation_pages=1612-1624;citation_volume=139;citation_year=2005;citation_issue=4;citation_pmid=16339807;citation_doi=10.1104/pp.105.068718" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=M. T. Hamblin;citation_author=I. Y. Rabbi;citation_title=The effects of restriction-enzyme choice on properties of genotyping-by-sequencing libraries: a study in cassava (Manihot esculenta).;citation_pages=2603-2608;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2014.02.0160" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=C. Heffelfinger;citation_author=C. A. Fragoso;citation_author=M. A. Moreno;citation_author=J. D. Overton;citation_author=J. P. Mottinger;citation_title=Flexible and scalable genotyping-by-sequencing strategies for population studies.;citation_pages=979;citation_volume=15;citation_year=2014;citation_pmid=25406744;citation_doi=10.1186/1471-2164-15-979" />
<meta name="citation_reference" content="citation_journal_title=The American Statistician;citation_journal_abbrev=The American Statistician;citation_author=N. J. Horton;citation_author=K. P. Kleinman;citation_title=Much ado about nothing: A comparison of missing data methods and software to fit incomplete data regression models.;citation_pages=79-90;citation_volume=61;citation_year=2007;citation_issue=1;citation_pmid=17401454;citation_doi=10.1198/000313007X172556" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. E. Huang;citation_author=C. Raghavan;citation_author=R. Mauleon;citation_author=K. W. Broman;citation_author=H. Leung;citation_title=Efficient imputation of missing markers in low-coverage genotyping-by-sequencing data from multiparental crosses.;citation_volume=197;citation_year=2014;citation_pmid=24583583;citation_doi=10.1534/genetics.113.158014" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=X. Huang;citation_author=Q. Feng;citation_author=Q. Qian;citation_author=Q. Zhao;citation_author=L. Wang;citation_title=High throughput genotyping by whole-genome resequencing.;citation_volume=19;citation_year=2009;citation_pmid=19420380;citation_doi=10.1101/gr.089516.108" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=H. Iwata;citation_author=J.-L. Jannink;citation_title=Marker Genotype Imputation in a Low-Marker-Density Panel with a High-Marker-Density Reference Panel: Accuracy Evaluation in Barley Breeding Lines;citation_pages=1269-1278;citation_volume=50;citation_year=2010;citation_issue=4;citation_doi=10.2135/cropsci2009.08.0434" />
<meta name="citation_reference" content="citation_title=Statistical analysis with missing data;citation_author=R. J. Little;citation_author=D. B. Rubin;citation_year=1987" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. Marchini;citation_author=B. Howie;citation_title=Genotype imputation for genome-wide association studies.;citation_pages=499-511;citation_volume=11;citation_year=2010;citation_issue=7;citation_pmid=20517342;citation_doi=10.1038/nrg2796" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Maughan;citation_author=S. M. Yourstone;citation_author=E. N. Jellen;citation_author=J. A. Udall;citation_title=SNP discovery via genomic reduction, barcoding, and 454-Pyrosequencing in Amaranth.;citation_pages=260-270;citation_volume=2;citation_year=2009;citation_doi=10.3835/plantgenome2009.08.0022" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. L. Metzker;citation_title=Sequencing technologies - the next generation.;citation_pages=31-46;citation_volume=11;citation_year=2010;citation_issue=1;citation_pmid=19997069;citation_doi=10.1038/nrg2626" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. Nielsen;citation_author=J. S. Paul;citation_author=A. Albrechtsen;citation_author=Y. S. Song;citation_title=Genotype and SNP calling from next-generation sequencing data.;citation_pages=443-451;citation_volume=12;citation_year=2011;citation_issue=6;citation_pmid=21587300;citation_doi=10.1038/nrg2986" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=B. Peterson;citation_author=J. N. Weber;citation_author=E. H. Kay;citation_author=H. S. Fisher;citation_author=H. E. Hoekstra;citation_title=Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species.;citation_pages=e37135-e37135;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22675423;citation_doi=10.1371/journal.pone.0037135" />
<meta name="citation_reference" content="citation_journal_title=Diversity (Basel);citation_author=G. W. Peterson;citation_author=Y. Dong;citation_author=C. Horbach;citation_author=Y. B. Fu;citation_title=Genotyping-by-sequencing for plant genetic diversity analysis: A lab guide for SNP genotyping.;citation_pages=665-680;citation_volume=6;citation_year=2014;citation_doi=10.3390/d6040665" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. A. Poland;citation_author=T. W. Rife;citation_title=Genotyping-by-sequencing for plant breeding and genetics.;citation_pages=92-102;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.05.0005" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. A. Poland;citation_author=P. J. Brown;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.;citation_pages=e32253-e32253;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22389690;citation_doi=10.1371/journal.pone.0032253" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Poland;citation_author=J. Endelman;citation_author=J. Dawson;citation_author=J. Rutkoski;citation_author=S. Wu;citation_title=Genomic selection in wheat breeding using genotyping-by-sequencing.;citation_pages=103-113;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.06.0006" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. E. Pool;citation_author=I. Hellmann;citation_author=J. D. Jensen;citation_author=R. Nielsen;citation_title=Population genetic inference from genomic sequence variation.;citation_volume=20;citation_year=2010;citation_pmid=20067940;citation_doi=10.1101/gr.079509.108" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. B. Puritz;citation_author=M. V. Matz;citation_author=R. J. Toonen;citation_author=J. N. Weber;citation_author=D. I. Bolnick;citation_title=Demystifying the RAD fad.;citation_pages=5937-5942;citation_volume=23;citation_year=2014;citation_doi=10.1111/mec.12965" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=J. E. Rutkoski;citation_author=J. Poland;citation_author=J.-L. Jannink;citation_author=M. E. Sorrells;citation_title=Imputation of unordered markers and the impact on genomic selection accuracy.;citation_pages=427-439;citation_volume=3;citation_year=2013" />
<meta name="citation_reference" content="citation_title=Algorithms in Bioinformatics, XIV;citation_title=Using cascading Bloom filters to improve the memory usage for de Brujin graphs, WABI;citation_author=K. Salikhov;citation_author=G. Sacomoto;citation_author=G. Kucherov;citation_pages=364-376;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. P. Schilling;citation_author=P. G. Wolf;citation_author=A. M. Duffy;citation_author=H. S. Rai;citation_author=C. A. Rowe;citation_title=Genotyping-by-sequencing for Populus population genomics: an assessment of genome sampling patterns and filtering approaches.;citation_pages=e95292;citation_volume=9;citation_year=2014;citation_pmid=24748384;citation_doi=10.1371/journal.pone.0095292" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=D. Sims;citation_author=I. Sudbery;citation_author=N. E. Ilott;citation_author=A. Heger;citation_author=C. P. Ponting;citation_title=Sequencing depth and coverage: key considerations in genomic analyses.;citation_pages=121-132;citation_volume=15;citation_year=2014;citation_pmid=24434847;citation_doi=10.1038/nrg3642" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. Šmarda;citation_author=P. Bureš;citation_author=L. Horová;citation_author=I. J. Leitch;citation_author=L. Mucina;citation_title=Ecological and evolutionary significance of genomic GC content diversity in monocots.;citation_pages=e4096-e4102;citation_volume=111;citation_year=2014;citation_pmid=25225383;citation_doi=10.1073/pnas.1321152111" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=H. Sonah;citation_author=M. Bastien;citation_author=E. Iquira;citation_author=A. L. Tardivel;citation_author=G. Légaré;citation_title=An improved genotyping by sequencing (GBS) approach offering increased versatility and efficiency of SNP discovery and genotyping.;citation_pages=e54603;citation_volume=8;citation_year=2013;citation_pmid=23372741;citation_doi=10.1371/journal.pone.0054603" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=W. Stacklies;citation_author=H. Redestig;citation_author=M. Scholz;citation_author=D. Walther;citation_author=J. Selbig;citation_title=pcaMethods—a bioconductor package providing PCA methods for incomplete data.;citation_volume=23;citation_year=2007;citation_pmid=17344241;citation_doi=10.1093/bioinformatics/btm069" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. J. Stekhoven;citation_author=P. Bühlmann;citation_title=MissForest - nonparametric missing value imputation for mixed-type data.;citation_pages=112-118;citation_volume=28;citation_year=2011;citation_pmid=22039212" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=O. Troyanskaya;citation_author=M. Cantor;citation_author=G. Sherlock;citation_author=P. Brown;citation_author=T. Hastie;citation_title=Missing value estimation methods for DNA microarrays;citation_pages=520-525;citation_volume=17;citation_year=2001;citation_issue=6;citation_pmid=11395428;citation_doi=10.1093/bioinformatics/17.6.520" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. J. van Orsouw;citation_author=R. C. J. Hogers;citation_author=A. Janssen;citation_author=F. Yalcin;citation_author=S. Snoeijers;citation_title=Complexity reduction of polymorphic sequences (CRoPS): a novel approach for large-scale polymorphism discovery in complex genomes.;citation_pages=e1172-e1172;citation_volume=2;citation_year=2007;citation_issue=11;citation_pmid=18000544;citation_doi=10.1371/journal.pone.0001172" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. Vos;citation_author=R. Hogers;citation_author=M. Bleeker;citation_author=M. Reijans;citation_author=T. van de Lee;citation_title=AFLP: a new technique for DNA fingerprinting;citation_pages=4407-4414;citation_volume=23;citation_year=1995;citation_issue=21;citation_pmid=7501463;citation_doi=10.1093/nar/23.21.4407" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=K. A. Weigel;citation_author=G. de los Campos;citation_author=A. I. Vazquez;citation_author=G. J. M. Rosa;citation_author=D. Gianola;citation_title=Accuracy of direct genomic values derived from imputed single nucleotide polymorphism genotypes in Jersey cattle.;citation_pages=5423-5435;citation_volume=93;citation_year=2010;citation_issue=11;citation_pmid=20965358;citation_doi=10.3168/jds.2010-3149" />
<meta name="citation_reference" content="citation_journal_title=Bulletin of mathematical biology;citation_journal_abbrev=Bull Math Biol;citation_author=M. C. Wendl;citation_title=Occupancy modeling of coverage distribution for whole genome shotgun DNA sequencing.;citation_pages=179-196;citation_volume=68;citation_year=2006;citation_issue=1;citation_pmid=16794926;citation_doi=10.1007/s11538-005-9021-4" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Increasing Genome Sampling and Improving SNP Genotyping for Genotyping-by-Sequencing with New Combinations of Restriction Enzymes" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
<meta name="twitter:description" content="Genotyping-by-sequencing (GBS) has emerged as a useful genomic approach for exploring genome-wide genetic variation. However, GBS commonly samples a genome unevenly and can generate a substantial amount of missing data. These technical features would limit the power of various GBS-based genetic and genomic analyses. Here we present software called IgCoverage for in silico evaluation of genomic coverage through GBS with an individual or pair of restriction enzymes on one sequenced genome, and report a new set of 21 restriction enzyme combinations that can be applied to enhance GBS applications. These enzyme combinations were developed through an application of IgCoverage on 22 plant, animal, and fungus species with sequenced genomes, and some of them were empirically evaluated with different runs of Illumina MiSeq sequencing in 12 plant species. The in silico analysis of 22 organisms revealed up to eight times more genome coverage for the new combinations consisted of pairing four- or five-cutter restriction enzymes than the commonly used enzyme combination Pst I + Msp I. The empirical evaluation of the new enzyme combination ( Hin fI + Hpy CH4IV) in 12 plant species showed 1.7–6 times more genome coverage than Pst I + Msp I, and 2.3 times more genome coverage in dicots than monocots. Also, the SNP genotyping in 12 Arabidopsis and 12 rice plants revealed that Hin fI + Hpy CH4IV generated 7 and 1.3 times more SNPs (with 0–16.7% missing observations) than Pst I + Msp I, respectively. These findings demonstrate that these novel enzyme combinations can be utilized to increase genome sampling and improve SNP genotyping in various GBS applications." />
<meta name="og-title" property="og:title" content="Increasing Genome Sampling and Improving SNP Genotyping for Genotyping-by-Sequencing with New Combinations of Restriction Enzymes" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/4/845" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genotyping-by-sequencing (GBS) has emerged as a useful genomic approach for exploring genome-wide genetic variation. However, GBS commonly samples a genome unevenly and can generate a substantial amount of missing data. These technical features would limit the power of various GBS-based genetic and genomic analyses. Here we present software called IgCoverage for in silico evaluation of genomic coverage through GBS with an individual or pair of restriction enzymes on one sequenced genome, and report a new set of 21 restriction enzyme combinations that can be applied to enhance GBS applications. These enzyme combinations were developed through an application of IgCoverage on 22 plant, animal, and fungus species with sequenced genomes, and some of them were empirically evaluated with different runs of Illumina MiSeq sequencing in 12 plant species. The in silico analysis of 22 organisms revealed up to eight times more genome coverage for the new combinations consisted of pairing four- or five-cutter restriction enzymes than the commonly used enzyme combination Pst I + Msp I. The empirical evaluation of the new enzyme combination ( Hin fI + Hpy CH4IV) in 12 plant species showed 1.7–6 times more genome coverage than Pst I + Msp I, and 2.3 times more genome coverage in dicots than monocots. Also, the SNP genotyping in 12 Arabidopsis and 12 rice plants revealed that Hin fI + Hpy CH4IV generated 7 and 1.3 times more SNPs (with 0–16.7% missing observations) than Pst I + Msp I, respectively. These findings demonstrate that these novel enzyme combinations can be utilized to increase genome sampling and improve SNP genotyping in various GBS applications." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
    <title>Increasing Genome Sampling and Improving SNP Genotyping for Genotyping-by-Sequencing with New Combinations of Restriction Enzymes | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__xRFCM2RdmYuTElvkVNViIiYmtgne5CdsO2p72JRsBog__5nfCKIBebvFfARteH5Scn-kp3rXvzgexaRVofFNQV8g__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"OL4sJWK2Eea7B7nGaY9AOHP39xlfJBVqu-GOYBe-Ny4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460114","apath":"\/ggg\/6\/4\/845.atom","pisa":"ggg;6\/4\/845","ac":{"\/ggg\/6\/4\/845.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/4\/845.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/4\/845"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/4\/845"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/4\/845.supplemental":true,"\/content\/6\/4\/845":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022TZdApUT6pEuOqJcTZB9MrtUOzlWuBHMdQ0NLJWHiwLoN5EWA2ckqRMBxUq0BM+6+W4\\\\\\\/5tsmbup6nwCpYYO\\\\\\\/b1BLPLKeIPTz1jvYRFvQx1Zd\\\\\\\/35l+GETHMKeNMXI9RAh0r0cabEIM1llzbw561uBDSTkmKmH8h9anx\\\\\\\/NUVDoFr0\\\\\\\/X8PLzYefRQrjCqXHeFvQTX44P615qLlOOX7qP\\\\\\\/opIvee9HRJ7Bfgz7FhcGq5XdAGbD3\\\\\\\/yG5dG\\\\\\\/JQQWVpi4vNVZCl1rxSjZTD3hJKSkbfUZ3unMAF45dqBkE+CkoE6p38mFnaoU54E9zXpE1nEdC7wxODOB2uTLPaQxAU9HIy7fSQg3pbVERiKrAywSavaRQIVBRjooaYGpTTgQOAd9OZe6rLAiWLxTq45I9hNfbtuevxE2Y+tuWyp7eg25GoOVjKLzxMNKeSi\\\\\\\/U6FKxaGwyPkMvHz0VEKXWWi9NULjPkvrWR\\\\\\\/wfrZNQI9nnBQIRCJ0Av\\\\\\\/3zxOGmTI93fne870nxkkEcKhRloo2nG3+Dy3ouMiEJA9NCfMBiPV8fPaBE7SN7SwZeGoTFzHLVyCDVdD4iPD0M2OZXb0VknAyh7zPlq0CiJ4Wz0PUG8eb4RSKHIk26FDto5fqW6MlVM+6EEmS1rcEA0d2cFkK9UZUbZTItlg\\\\\\\/SEiNoJMTqHioifEmAsz46AJ5cmleCer0IaO0tAMw+\\\\\\\/n+V6MgDm2jcWNtBSZ6xXsO2v7xQWdqyjZQH2K705YvvEDKS2U\\\\\\\/jsh0gFFZnsYut7BYu6QINrH\\\\\\\/cUpdr8OB1XuDiUYJ2DO9XidDAQEImFHY1zovBijOJ+Mc7sufHPB2IB\\\\\\\/LCjQxA0bP5isLhrre45h+SPCEEeEt1NouyqCxE2pD4rUcZ4gakfMpQ3PDqplWwm6CHMLFt2huTaVSufrXsGOGpATHbSQeD+eqjNQoOY8pftMrHac44EIrtpMIITF2XanmUnPCCDJupJnh8YmOJp6crukNj49tX8S6zbFI0bBhebjy7c0ARSYE3F\\\\\\\/+X9ZkWUlR1CFf2cRAc029ogSW2y3ZfkZZAmWe+JJqUPul6+7pmVwAcdZP1BdkvG2kz7ja1fjXXw2Gv16x\\\\\\\/IfautY70moo0elYEcW9FPZ+uu9ji5PPlmloyFMfe6MsFxDknGArpXHIzqhW4GJhXP68boHtnx\\\\\\\/BvI03eCE1DL6yONY7n76ILGr4yG0zdciM3f5i2GZkNsSqQq1U3ZjylcGcMcdzYr\\\\\\\/muS6P\\\\\\\/nzO0AKh0hh0nVReDKYJqDqLI\\\\\\\/FN4VLpxP0FmqF\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/e5TC7jrFREJ\\\\\\\/\\\\\\\/AhqfcbgQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002209a0626bc7ad2e52ee243e5835921b0e84293d1c6da012d1e651e18710cf21d5\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022T5Kq9lJZwIcbBlPMGLUJAGXrNxaxGfvuNHO77eO7yp4Y+mTR8F\\\\\\\/JzGFuuZU0TS0jmTmsmyKJOszN\\\\\\\/bCmWtrldjxnOtHz643tWlC7ZbMryxzH\\\\\\\/km2\\\\\\\/AuTREwdFoEhEDhLoV1wCb3QBPFRONCueYqXf410gmTGDALXrWq1uGIm1SlWmhIZF6ODrxaq+Jre2\\\\\\\/1s85gPSsv21AxqNNZRXUtZdytnkuAyhmGnJs2PwxQsMbILNRwL29aJW8X9rG6901lz0xAxgNvnzf4d0TNssVZZfUHgl75kJOf6VGijU06sxSpXTeECfKKDNwlQptgBOBxd7JhdxrqBDqUcRdS2jGlAwxMjqdAWCX3XLb8gXNg5agGPKyT6K2oTGcOSIW9K5uuc0rGjiAiC5Z1F9VX8+xTrcAOHbKOgucIGEzUx0AQrXcUQLCYya2W2ERc+jga50\\\\\\\/rNo2aEKu8xzl\\\\\\\/LyhfS0S3ooZmO3JmRA7JeSXWdoNIv6bVwR\\\\\\\/kvvbgyhX5NwRjBtca0tnIc64f731GcVlJ7f3tYcImxWDuG5x21gIdber6PrQFGNS+VTnNRqjkz4Ytv24OJWiYubtmFseK4Oughb4I90n0eIifdP\\\\\\\/yoo3+jLS4TRoGX9K5LAu4ImOR6iRUS9mbTWHLjMP75s31o1PpX5bVZplsdRY1Tj\\\\\\\/5188TKq1ApBeuUEktAig4xY79qxSRd\\\\\\\/iljaE9BgoUrKYazlJsq8Fh6GFMb82J2rOsPkZBkW\\\\\\\/sYZYJvYV8Mf2TPBEgN1D\\\\\\\/Hn9F+IGKMGTMDmsuTY\\\\\\\/O\\\\\\\/NIQv0tm8U2EdDBy98JiQKhMv0mxh06vFX95VCzL73jtbiEP3TguIJTFtQju+5m1dspM+6xfsb\\\\\\\/p2xO0pCkTMfzVMkytuLCs\\\\\\\/rwIkuhK\\\\\\\/k2YA2k4Eq0r2YC8g37bu0vRmkFa\\\\\\\/6tDwQvAeXJHbtsKfDzyywy\\\\\\\/dgvpVgnCUWyipNclCd5A2gn0NL8cJlcl3wGCIq06RBm4lCPKE0+KsRObyE27P0NU+pZnEDOPrPUwqcIgr5IlHih33lrkB3qBZVRx091sdouscsGyatE+p6OchsJ\\\\\\\/tbto4V2VHIBo9703AXDi3RraFpzk2Tk\\\\\\\/DoJF5bHb1mamOgKXNlx1KMbEEf2pXemcS1PSi7zJ96+B65v9OYny7NypURnkYvnzyRaXvt4y7OtIf7KIwG+vqlSp0iSR3hlH1RWTR9g2fo5+15aHWo537tjwgef+I9u8UpYH0xhrpCWKd577YWdfovVCZAej0WOzkNTdRZ3nzukT5FmkUgSsjqi29ZDaKTSd5FklS42DfyxcS5b0w5BYWwxsI4bdPFWCDvrpZ30ez\\\\\\\/X05gWAz8KJ5449nMmCXZrAkGn8ek6vsZ88248Ie0onDQmw5GQalUL8GWvMrKIZY7peJmM4uJCInTqNpM1MUw0pBvcQHKRSsSymcGCC5NCEzowmUzeWRLPgc57SIlZAhQRYuMvURdF3\\\\\\\/OlyUZ6AS8\\\\\\\/G\\\\\\\/qxqXlXHvjIFMDACJujAH94vbHeleEgKJ0wCDg9n86lQ7xqimJPB9IXP1nmXdCl4vlqUzncqhVkgIT\\\\\\\/U5lh+1betF5e45I+udF45e61eeBXxohhTsQL6Mcu4aRmgyOxF4vF4bTHbnyfWruXiHElL\\\\\\\/M9+dfiWcbjkNa3M66H6Aa2r8IwiUqMcn2oIVsfnZeKfasA+78z4Cpd1nW+bm6zHSyBxPeoOvYxudV+gMyY6bC2i1Ph3YiwYeoQ1We1+nbFMg9W94sx9ogjlC84tX37pXkGaHaVKcNTyDhTq0XvdmVMtKV+9pkzNlOAOtNIf5OD51+HJFihckNhEHoy5UZNc47caTgRQstIF1+5JOI0oxs12tbAMXtpqdTEcSoNZWfFx6bCk1J7aoEeh3LJSI6LyQ72AXp4xg7Nt\\\\\\\/sPMrDXwGRWuyBQUUMhpKUrSgCYsPhTNmpRXuh2i4L2TirEXZHbhyQC7eggeoBtSV5tdgC2rddziAos67HCOmKUMw7c0xXGxpskUKLBx6mbUfcirijzxS4xDFbftlUyiOMly4lgEhGgycdyR4t+wF5v0rBms+UUyyB3f\\\\\\\/uuWegN3bkvLitSEDKkcxmWqjOvNaa2PpCoSx8qny+gUhWORkNVUzCheeL7j2MUuoQ3iiqZ8bwgM7vPd1TDNQO5\\\\\\\/200H6+7ybWD8Eiv7CELHJ7GHAFp7RC1KGJxuEuqaJtCMgn3+pRO8DVowMXsjVN9JL7A4xh9pg+NUn6a19RCAGy\\\\\\\/cQ0vMMLXelcwiB50g2+yUN2HGby3YjFwAS42STiiv0RvoX0cfrRTgrZPHrnr3tIbY2faO5S1B1pvy3iECQiihDu0gtYBK1bQxWl1m7xNxPAhoNoUfyF++sA7UjQJ1LVLn0Ox0XA7\\\\\\\/4bITZRpNG6mNS8RPTNnB8xktszIezY09+N3MzjeHQTADAhvjH6UEbkrXrPqSMu3ppKm9TZXyUJeaslypKSOBjhGZ002K\\\\\\\/PjP9f6+zM6\\\\\\\/Qtvw25ho268f8R0hCO\\\\\\\/JPsY\\\\\\\/KyljBj\\\\\\\/KzBlTLmYGHZLazN82sN6eip\\\\\\\/BFNnJfnWOxMpJ3O+fdkZC7EBusEpUVqvTd6HzCOsPK+Ln5TXbIKq6ru1Zz\\\\\\\/\\\\\\\/0R\\\\\\\/\\\\\\\/z2KURXKeEykIeKy16\\\\\\\/JI82AKDWrEt3SoY22Whin2czdneqf0RVKrs5Q\\\\\\\/OZ5w2Nhldg2sHbx0FyimcdNXZQCMk3VJyn1QIV0EjjchiDJe24QKFt6Awdb6tnDrVvKfisNX4phsYEN3iyY7MmRTTch+32NRcNy7ZpgE4SKuYriLJB\\\\\\\/XeIw3zbc8jCPmd1pWbDHQQ5DyfugG1MB3P6QcMQ5JsRrANxilTfw82flFt6eCCwmlGe0UwavHbIlpVtY6sPH4iAypI7GprLfPQmr0yWRhuQJQwmS9cv7o9YEa3ug26fyGNV\\\\\\\/eJmoCyWwsAVI5iihrJAugi48AUotxYcAsaNMGuZsEGsCIIV+mh\\\\\\\/zQk+X+hKSMnUQhQ\\\\\\\/loUZ0z3Ff2DDpZ\\\\\\\/zMEC41XdPT1stXeQz1Khpok1p2xWyvp5NLnCTC8EZTfiRfY0PEVlcVh32HySyhAVZB2148tZZNzC9hyP85a9DcaI30cjlcpSikZReRR2Dh4ttFsOuUqfayvykFB1+1wF4RTdeBGZijvYMm7aa7klW33gprqMt4FywZ8LnNVr4lp6\\\\\\\/jd62PzHTCzQC6RQLXnpjjThxt\\\\\\\/sAn5AS5GhsUr4wRQ2xClWbaq3Wr2HFz0caDqzQmKxqaphwtcutCygmC\\\\\\\/hhJNzETWjrr6TH\\\\\\\/yeROyCaA4J35mV8dgLotHQCSBsRRPyEAzQuSLfggrtuM4216Wi8J4ULSKAwA\\\\\\\/RrQ1P8MvGZI+DDaDu1Q96578RbnFKzXOtZe0Ffn++HIE+C8PWQu+VzvXydHR0f\\\\\\\/lc4UY57yn8yVj2U51hPIMm7gsU23mBUy3iWiW1KyD9\\\\\\\/2aOuppGnyuQ7HHF8zHgsp+2L4hIhlKwqyJWIECPVwHKHooa6RVgd1MvAtPXxOCR+DyXHJeqDR3UNJX7FQj1FSX4mqk3SjY8Jq7ORTI7JujIMYGz0T6c2K3R7wjrkeW5uisNALjgEqK9qzVB+pRrqJ2hyTjFuzs1mH2j2zlik59LzLznKS3vBKDgU9KlJo6ymzSpod0k5zYnGSBB6zV\\\\\\\/0E5xrtQnRxuEOpDp3ZbkqtnW4WPDAmZLRYuUVUgiMZbkOzpqQTx\\\\\\\/46Zk3INU51IlAVOQiDl3j3+rhfLqQ\\\\\\\/RVN6m2N+J5um4etrSZrZiAp\\\\\\\/28dyba0fqitFPmZfOwR3U+Gy58kKClS\\\\\\\/rrtO1leGYCSX7\\\\\\\/SE4pED8hxTpee+jPttewNiI8vTKmYF7k7hbCkWHVZIanJi1FCC7Nqzi2lfrKeMlmLrIZcsoiTNYQSTqhk3zu9vigL0Yh3qBf3nESQBNrxrMlUcVhJipUSIoOoU4tfqFUjmcDn7zK1tv6UksY+F2P2HmeLhFoVq6CNPYpXGPTlx2rr2Mlul1Rd6BnKhPQwUAn0dylzYOCeXrfwtOHAw9kb3fwhUJyknWvAghqdzRFt4yVDZnXDfwWYo1RPmqQ2skU9ElJjKQzdmq\\\\\\\/tOlfOuRh1lNnulN\\\\\\\/S2\\\\\\\/lz9XP+Ws8MZQUWZnH+Zt3U9h4NTQmJJtJ2jh6Mv2i3w6bk9ae8FBnoVf350DKexI5D3sIxN6w8KS6cdTO6t0N+gmmN08Q2L2B4\\\\\\\/YzUP13ockdYtLlOxDRQAmNXnb5Lbyw3jjWc2wGJZuiscfyiPqb\\\\\\\/3WnEB3NwO8gTQ9US3vn7Sa83JdCfhQWnkSvVxXyBqI1LXw+F0k9GsqBKzEAk5tcCs7L7en3QPGZJlbyKvP5CnMZVNnAskEzVpCXp0N9LRL32ZusVSHDlcOzBZh10\\\\\\\/FQ3t3wW83A6QjFwUbzb71NmOrQegtRo2PujmhKmh69vDEfIfGEZyBEZOm3CxXLx9V3ATn3bdw6UoZFUMwV49hsS4gyQiavV4CXI3ATS8Lk1KLP01PX9LKYargzuJUZzt98\\\\\\\/MPFxdfku9LMYPwAcdIsFhf\\\\\\\/nx6BMLhi7NsNdn8wqkPsLUJ0o0tTd9Q+TY67cLaXF5dgHIhswfmHNix8gyvjAuseu\\\\\\\/kb25QQ1tBikzvgw98knJFPpa9QsbraaUZEcZKywiZk1o++rn6msEgTf3uLZMQJ2otBoNzCneVPrSMmWmfrC2363gYjNgGf8aO1l0sCJQajcrWQSh4MmV7hHbZNNi40IHq2g8ytRj58oC3KEQO95d3e3iUWyiOqayIrSdbxqqL\\\\\\\/\\\\\\\/SUv0jnCVLo\\\\\\\/1evPJyCmsGKntGSxX5BOj5Kck7VhjXEx87w\\\\\\\/4oEx83YR+kTyYcM62dKmM1Yohcr1krTvNFtVWYRjqGhsDGV5uVVkOgvt63QGBHV6SRK+mAwyNv8+Ow1VPMrqrWwjxceM\\\\\\\/bmhaeRyERdKDJhw4P2PbTbds6UQ70A5UdrwZzLHzR+HMUUNwx3lTI+PsuBA+V8Em\\\\\\\/S1Ke3XyFX+pDPuLBe8AH4rv5HLsrSPyXTDTJbmYGI5jX3yyB2MHt++\\\\\\\/GJHfOPIPOo5+kLGspAvLLNp5KoV8eC6UNoi9R\\\\\\\/z+VqiwQCWBmGTHpUyrB6AkIQR+8WMh967v7ORwLA9MaozShCGhRBMxLVOE99H6U61Ry+mPg2ziU1gcgbkw5NjYDOd4efGzr+nNTrajub4AIlKxzkmCbLY1HLU8c+\\\\\\\/UOvM29l1GKyZYLNw8nMuziFBfMIWCFeHx2rr39mVIXCDQ9RbqybyThgNgPLSbUdIAMPFjLHDt86OYwpKRgxIeaW2RTvhUnvPL7quedwS89ZhKMlnsOtMDniSIA9WOSTD4JPL7Nf3IgAl0tUIuEXvnQKNJYTqBsU8uDl+DrAbzfuZ0WCaXwDIPdjrsH098W\\\\\\\/dyFAppgqqS0dW9GyMogYXOFyByfYmOVu7\\\\\\\/wFo9wD0\\\\\\\/KtVlPWgEFm5gwIG8jGVJ6YIuJm7zUdB\\\\\\\/FPpg1PRuQvS112vNSFAKSpLssuN4yuDNwW5HtxecpHIrB0xw3hHl5pzNh8eZ1grmyrKRRUqWosdpbOZN48RBaegOUsvkcepJXJFZypQJ6bpT18gwMyWi52vPfwWsjPWVf9BuwR8\\\\\\\/4z1b0hKBDY7lLNSe1Dyn2FyIgI1rGpaFj0Uos\\\\\\\/jqEHgnNRZvNjmu7uxk0uI+ET9X0QUTEEe1u9vJ+1JRc11mQ9OrHQ0wvWnSChkQ1mjO3+9x2\\\\\\\/TK7gEzsNmnCyPyBPgeKUfzF3y\\\\\\\/gNqzwyA16FCIUHEoDs0lrsTPBdMd\\\\\\\/GwTS1B04YLGWT4Z1+cvic\\\\\\\/bsAdTmCnbyF+\\\\\\\/7baE5G1ReQoPWUMenuY59TV82ukSdXZrjjNwjn7FHpYzZsuWeH2cds7ertGhkLb6JfikvT30sPe75lly6MrlMAu92zFoXdjSyfg9aeLoUCx7m2Id4yLsIRSfjI8deOwa54c4PNM9nRjvjz0cRr+8TC62t2ya5fnjPl1gwpq+CcttzHAc7zXCODcxx5zgTSrTZxp+3QwPkfEbjTCh9uvSM90LJQDMLPBRlo4bozNkpZ9tS73pYamGZv0tKjM\\\\\\\/B44pfVpSNEa5t4Y+uahqybsjaIIJNXCsCpO2ENW1byFC2JCJDSgaID37tj4do5NOPSTHSNDH8lAZaFOHpvmAayj2i9XKQOIRsoifYQXL1lySshAvBgf+k2BxLMnM8NvpIgFM48b+5tZMl59LDRI8\\\\\\\/NzYJ7V46BBsxAr41o5X4WTnA58Iiv\\\\\\\/FXwyIruhxGkkIlSzLw2eu+UQMlNxVstbHM+7JrJ16Y+FhO0s1IP73SihQwntBbBQLtmJvx\\\\\\\/Y4G\\\\\\\/H5gZQWDdMMVPrBsUiv\\\\\\\/uXuIE0L8e\\\\\\\/kj3SIugc0BWR6N67K2HK6SrhFG8KiRl2phjkpyJhwdtHHQWtTYpmS6kEnDPtQdQ6mBreo627w3ashiOjTSmwsJiA+J513WDLCosXWIUHVnb3HhWuz8cu\\\\\\\/B81csoAz8IIfKR5BIElBENBctUy52GxtXqPghHcV2Jp00Dx+H3ryZG0h1WZX8VT6JiLkNdul2rI9BV41ZuHO8LBUJtArdyLx\\\\\\\/H9JNHW2yJx5GvUSWval89+RRIRmyqBtWp9N0DRYrbMRUXwdCWuj9omKU95FflwyGa3Hsxlehp48kZdLmjbMDpKy17ki5Bz9vWx+ZDIzahaG5QtMMqxCg\\\\\\\/UAmp5RRX7uEEvkICDa5Kcww5XFgUVD4FL12Gc6RjnN2PNQf7C33dKKNF+MfjIZfp8zzFqC3Tlh6B57b2KV9E0hXIMXjJ3ciVqssQ0mAwvcHByb3erpkgdkT\\\\\\\/\\\\\\\/LvgyzO66HIxueIauVvmVbN22+hj+WEBTimfS9hCuS9d1yL45KcUbLE2P6uqKzqVlZdAg9b24P2uGbqM3yh3nrg502fv8gnVK6MEkwKPNMA\\\\\\\/nSZtwNSWU+USe1KAS83urixvl2HsiCICN5b0LVKiYnLytJyW8baWf26g6z6R5iz83NnH0NpOrZgtyA42xgdHEkesrzKo4V0UROdtCFHJTk1QEYQxjT\\\\\\\/2P\\\\\\\/ysHAJny\\\\\\\/BkBtcN2LJxxCgV1XY7LHaiLmuSt1rt\\\\\\\/g1bZbla3Wbyy1Z46+NVzKZ9ZD8jewCiNsWAlybD7zGXBCvDhnaoh3rsOza0\\\\\\\/aXF2PHAGV3sdW3WLq5\\\\\\\/QdB51gYFyFVvRFy8a9XzNDJT599gYRrytTi2byuaB1yKGLsekqiqW5rm\\\\\\\/L6o4PgvGrPbzcRSbZzoRvl+R7PntI9+FELKxSBTz6KE3SzgHDhmxAwuXS3+w1ejy\\\\\\\/HV1BbsAmTCg4o3vy\\\\\\\/0fi2O2rQNkskDkOkSxr2DD+1UBD\\\\\\\/b1He1UCt8xtlJBiAELOFnFczDcIl58fIv9s4aVe3UJ+RICuHUX2B49KempB+fHnxtQ\\\\\\\/jKlLuKnFqKd7e7n1+y0CrztO+erKI4FOSI4omYFkeCT2CZ+QCibonr9Dl1ielSP5M4W980NCMKpRWpJjBFRDC5oN2VG5xUYn0lVZzNgju2N+R8EaXkWjBbZBQ87fbZ9vGf4IKAbj4sxbZBtSIHZduq6ZeZDJ\\\\\\\/Iy\\\\\\\/YacGknhwItjRGjKUuhd3TncTqYb07dT13MFB7bH+i68t5od3Dy02H\\\\\\\/WEXgY5EEjeu+VaVV77QEAi+4VHFQB7GHj3WLmZCeswU+21mUQvir12gYrX6APN576gNdWtzZFa+7U5hUmzcTSPW9Sbwc0Hcc\\\\\\\/wDuW8ZDAAu6FCFyHx4dZqqexnMt\\\\\\\/KkA96OR77hmx8Y92Y5JuNs3MwJ8WrqK5X2aq76C8TI\\\\\\\/hbYmuk4thVVdcdNUaLIbLOBK1rZbgGDfpuJpBc01LYdlHC4wH1E1bDb7BE8pehn5yV2XffAtylT4Lc\\\\\\\/XSp2F2VaCZ5cM+cCofoQdGfSsE3F\\\\\\\/9fZh7cP6l5wFKv+Uaxnh4sJ5J9kpAaUmQNyRHUHyuv4PHASn+VPSV4rhV96fpMVoQK9xNr1AaNx5o1cu0jS5ERRnK8cctfjZYrL4YRM96SupBhLdT+TIuUHuVrf6sNRwsf0PyxwZCWKy1xUL6jpAP0AHOjHyxe7LXnDVL9PK4m4+RS1s8jSu+zqsVXRuvN0aep+a4oyx8\\\\\\\/GEbND4FZ9lggepa0HUwcuv16ZbPWvGwiQfP5rTbUxdMY0CaN5m8++VmtV+u0BeQdjUxATCYxJ8wt5QdJp\\\\\\\/mbtP6ahZ7Y5jsB33JNoUMZurMYV+0c5ieJZ9wD0J7YAb89lFT4GSXIhaGNh93IOUwnQl44bcDyV+2dVawCdh2OhAHuPQuI+ZbZKHufgPfW4U3NGXBM8foGS1dQtJ9zZm9NJpspCkR8P+CfJL9EuVnutisPLUfN58ZcM77jmN8k88FuMPSzZ7IPc0\\\\\\\/Jb12bSGb8cZjeXOXwQ5mwcbu1Hg7qjCAT3r5KP0cCVHG8K2QjFmArUwqCqirLh5Bg9zd+N1V4qNukq6Mw9uVbPheCJnm7vLRaLa4MYCgZ8LMD\\\\\\\/WNk5gt0bji1UCMQRLMNvxkZyF+ptVvPyzm1akxl5pzJCT8dnfM+NxRflqwAY03OSeRCYkONRVI1Wm1HIjWN0FzI\\\\\\\/ac0Y4ZEq8SnHfNHrgdmoTW429psqKE71ymQIJvfnkC+Bm7MLTQ7VDFO1EMqDkSBipBPcsMCEYFSy6v5HWDQdyvtfvw1EbmbgAKsRqsmnN5A9n+mzY7vaZCPjAGZkmok40\\\\\\\/YnXuxWTn1fQ4hp7XUDzs4fmbSfCr3TiI9wdWdBb6wv8FTSujkPEuGBfdOLIPi1mtGv4sGyyG98Qkz+9rZ\\\\\\\/PE0fzTenfh2GmZoNV2AsZpIYA7adsD2Wx7D2MLkNRUsgHuV6zrXguFchFXWAbTBvHL+addLbKepbeHR5UFUkPk2SLm2TfzyE3LC8ONLKV4D9GIgBn3pyXoqb4276SfTOG+UMrhYVtbYh\\\\\\\/HVbCfEailGvVBA0Eplba4Q2U2w3lNA3bxZJlZpnfR8IkoxZ9xF0K3oIEYgu0ruMzZXbIDJNDA0UJchsctbBrJAgXQipSCVGqT5APeZ2pqHeeXANlSU9xhOLmA\\\\\\\/UNm9l7SS3P9s99gzRj3RA\\\\\\\/FxXCuhkscro9S3nrpcfaKFkRrgtjIn829+x5+xEE6m3hetQd1ZgGdBRgwQXEolkTbRuJZy9MEa7E7TqACyPDDdAejyNNGi7UKqzAZlHcWrl8sJBqtqR3DwzKOlLJqzWsvw9NiH5BOka7Rbnwq7Kp9XqawDC9J7uPnitIujOpHwxp45ceURmU2Yj91sM+39vqmpGfQLFZjDKOyoYgjUTHkEiVA4vGrQfXE\\\\\\\/ZzQEZs6TUcWMbffLb2F\\\\\\\/BQrpcf9kdAaMageJ3o8njoamK8+1G5O422hFi1Jj\\\\\\\/dkPpMsHJgyua3zGu4q5oKSKJ83aixe\\\\\\\/Zw1\\\\\\\/hO43scv7mKotq\\\\\\\/y9QN\\\\\\\/dFm8TKHvXXtBkIvZ1p21nzFhC9Q2E84YoF6K5mKDDlM8IGmmwH9NOzrGzEtAyfAEIjFVmmuHMQ6uDre2TicUhZgZZYenYHPkWhTqjJ+MeCwRuSY1gOCxljXfGOqB875MGGlRUTxo0waRHBANu3bElOhQWuEe+KxxuPntsH0p+ySqARGtmT5VEf6eHlVBxg1rOlKMOxtk5Ad+vN7tArq0EcEroSEaN\\\\\\\/WMuXNRdV2vwbaR0EoGpwNaQWmOD8v7TwvLv3BM0\\\\\\\/scgF2+IBls4NHToTAhB1MIulHLszSVgvOgxs9Ndd2FR2gpjbU8qEyBptdNguErTFJ5UlRjDbNVSh99BjfJ\\\\\\\/VQ9Zw2IhUjeTYO4zIS7pMczeLQ7fVEH6bst6EsSRjFMqPbzhneaN77OmmPgbVmdTCGv\\\\\\\/oGr7sIrICDQuVXBrDrFhT7ZZIZ6K8gaVIwfqbXVg9JQ6ZKIHDEXLDhs908eoIFgLavqX9x9WdxYnhJHW+1VnK8ht5Qpcpe1B5YmQMoTc9dJFuFiuC\\\\\\\/\\\\\\\/Q1hqmX3TZMXDSji+CLfJVplzUwip1ep+FYvarkmOjoXWnZBOnoeOw4tMWdhYte1lbvBEG8J0BqYrmkhFIlY7+rMDQTJQ1QE9EN+xEGVIKc+ySb4j8ueQ9EyxmZIx5ADntZbmn7sm\\\\\\\/mftQ636vsVm2UiIy5PvVPNITmXm24JhQKoLXjXBOnrwDr33tTSOSImf1IMfbd07Z8eZij6SwaZYO7XAHs14ptRVf7p+ZMy3rhIFVmcyEjne+Ns6EtXirWCVGD8kQiwDpXth8sJ1lbDG4uRfaNoXKxdZ7gSvailclutLEJ0KtuzFsmZorlr\\\\\\\/uNGUod9F9lqAjvgM2xNhG5y2G181OnnlG6sqNM2cMCMiTTgniIESmAf9wRMYmb0cwkSvivoPPMe35WdS04b6CVH7jvPpoegdiodCYhTz0o5aJSQqEtQIsQy6WSR07Qs08LvJjuxEZnxqMGVg7hBRTus5iP5nhUqVL0\\\\\\\/J5E2Jvu4PB2\\\\\\\/FQ7l2vq8Q45u9+XcncktdJ59RRz365Ue9DS0T9SGmqHpvnfopeRPOG1r7NpNGiqtRkS\\\\\\\/t9t2WGexfYRnqKD\\\\\\\/b0x6NVmH6KIN0DjZR2kBbK+RENHMl2mWCG3QmPzluI7zC42LAUj0eT29SOV5HASv85LQ1lgniE\\\\\\\/jlqubAi3mtl3wtPYhWF\\\\\\\/haMfB4YRy7BzB6ui3bpUNVp4DjURdOhFTNeD8DRpOaj9fSO2G+g6e\\\\\\\/uYBGIGrJf1diTvpfZlPG99jfB4qy+M39wppQlndB+bYlu9+rAmAf\\\\\\\/L13bU14CzN2H7fzZbr82WE\\\\\\\/TKnby7RCUMQFr1h\\\\\\\/PTNTcvYXD6ukfqHVSaSdTbqeBbjIwEo2d85SPwEdShVnK9ZHMx9lAl4+Uq2p6qiH5ttUYuN+ZglU+fVDsYl+uYCa2NcMlaxZ5XTOYRR5EAmurMFMgWmOOfNLrLwC91KaIi6Z+PUf+wkvznscDnrKq8OfazcNv6lqKFAArdMOZFAr1e0l6awsmSBCwowjRIh8pwJ8cFdwmwfGWgdlFFIRkIpr1faJwGfi\\\\\\\/193GIYuKI5CiHkt8TaCwsFYH5a\\\\\\\/4dcDwCk1LfpspJ7UeQDKtYbWP9ruGoVFCd9i8XFUO5xqAhmen5hYxcZU8tXC8+LMEH3UMhr1iOGhAWqYDhYxDeaYFLsJBJvm6ef8ricU7idN8EXOz5zKQqlJFckz4BNXHsvlYTTU4paHj6VxLtFQbc3E7qHfWsxiJihPn8RP00YxClK8vZJRNYmHZZ5AbuoC8bM7BuicxN4rlHsaiB1nQy37kjr5neZDHMrfU8h+FuA1Cu1dhwHJbaPVRUiH1XqNyVgcbzo6MN4dVnIjl5Gm1cPJwBrdAk8w2aAnSmb9YvKfiHWqAa0ZjxavEbciiPtvmdAyuSeejJL+8cvj0yr0G\\\\\\\/vvofCsPZ2XkWAiLtZozuWn9Eh93tNfn\\\\\\\/gk4crUu1lzE8Rlszp+XzWPqscvpXwo8UQOhI0WUCpRgh\\\\\\\/+FAwVCZV2Hq8oQXEvyoORk0ROAEqmGkuHk46vZhep5+eqN6U4+LtmMSv\\\\\\\/SZ\\\\\\\/JsBXqy1S6sOiQHljPfuE4OODEd\\\\\\\/S\\\\\\\/kJRb\\\\\\\/KBjYfEaY6bVVrJbjK6RB5ZcONtCmJYowEJgYmyHwRepSMWRD\\\\\\\/TRc6rDkLkYMkzZjSt8anavR\\\\\\\/wuTu89SRcbV4h63EXu\\\\\\\/aADxv2B1tiH\\\\\\\/aEo2tFHFJplelKRGXtdik3kL0Y7Y6ovyRiq3xjR4Ehrl\\\\\\\/FqG44BvLfE6mepZvjCzJ+8ZKkypWVYj+uoWbNouIFV1+h3Hy7xQjX3T+0HgB1JKWUG\\\\\\\/Qdud6cL5oUynSI\\\\\\\/7tkRiJdvunmDwtPWp7j94q4Zw5RU5HGAKL5Kw15WxazvLSv34jhWTFgzLjmY\\\\\\\/9JBucFMxISic0v9b62Ox3I9cDhzlCkiBOrtOwWol25onXCTyQ3k5BdL5PMqtL8kUPaS1ZAStSWKHMhY\\\\\\\/4wVyntPWgvgdy4aQ8SOOFmdyfskxD\\\\\\\/ujeyDYXcvsDSG+qIZ34tV03KSu2XJrBj2QC\\\\\\\/1+VkJp2j061ZHGXSi3Q6qtkoA45myFzgH7rbm\\\\\\\/L4mW+t56wJYf8r1I+tkYQRH4GuaSJr3u\\\\\\\/D0xpcVk6C69aKEYylwAd4iE9oymCH5ENfNx\\\\\\\/tdZW0VcIs18NcvdeStMS6RPveF8f0t0oweZ+SGKCeC12aMg3Y2Z+g9qqIb4WEObcKHLigm3NziN4W3aw6TmOx8JNlzwfLOL36X2gxg+6w+GzbtLHKA2zNBx4w8Vz3YvnQccGG25z2bc7z3GzsixtPY1aOOhGTqPwasTZPylBhAnaq\\\\\\\/SmZE2X+dlLqlzMRqKRiWljELImGg9lzBelRQpz+yWJ+eV0UDiIiLrJqRCAor\\\\\\\/5WZ83y6W74fsWJk1+OoSsqZesAEHQfIj9bznoYKArUWxYeXQP9Mzv5i1IudUMYMkHErwx7gRxOmPubUvBmafNJ32wRSXW417a2eFcMkvZWmqrmUeWwFOQ0FQAmkkJkhNaawv9k8Cccd6HgmBpmzFYR3vaXx5gfJZQBqk0Qn+sUt+ZEsBtaBD\\\\\\\/U2ae4XXMFrx9rHxfft36t+WKH1x20IBSRpsetRlYJplr6gOHK2lUt6WM2i3n2aQdM8i1Trm3lo7jFyx\\\\\\\/2iLiRFJ4yc2ZrLZKWeykqk4KIywrxsc88new3kI95WV2yjfvonk4eJUq0ttr8GTBEgfFmDKYgQw9QVdVXZOLlB8bNEUyBJ8k6Y50Zex2ix+resyxt\\\\\\\/rm4ZeE\\\\\\\/RuibJk9ZSLreVMhR\\\\\\\/mQBhc0Kf4rOH8xWEGVHs73lvvlX2NeFPlJLCESOjTpL0OJsDeOqvhscjYzH53KhnXhRSJtxaiTVVJWqiSqCH8gEjKsEf95swJjWrpkXHj9O4GWsnWbMfXPb4vz\\\\\\\/J0ISTxFVtmQwBTmAxsyDfHzDFQLnpuxCwsYi7FCfWU4vMSqeRPrXkeSszBz9xgDUOSO+e7oeiNDo7f9QsqsN6jfsxShdX8jUGrEcjNZ4vLSo6p7wgzXdQOhh8PkO\\\\\\\/kwpWxu+8b+Q1I0v2C1\\\\\\\/BKALBVKyl8EIi9Inlmd0Dih9xg+SHj2t8loLaG3cyMZC5X1mx1Tj7Ilc+Yno0+jq5ashA8vOU6+KGpL9jgNTP4lgUP\\\\\\\/PyCxPKJ6lhTfUurIy9AKJ266JbAV30crYu3xQ8ZdToR5aBvXZmDtqej7w6WIdNkK42gn9EhArZJeI18p+6K0e1WrZwxh1m4H7V3biXxQNcXz8NL\\\\\\\/TaF7oGQGCgyTFvkXAo5Ga4rEk0EABHAvugf0xiWfpgw6T8vsPDLHS8kLSD1wEQiW2hHvL7C+ZJ5KENHHMI07l8o1c6zNvzU2HKMgVflpiVIGjsLq+uyxob6LYnniVFM\\\\\\\/vWsBdXlhEFz8VgM0XBbcIgIVYKQhkuGl7bA1hz1q\\\\\\\/ztMaRDsyLznJHSfeG0gq6enMiN6XNx0s03oHLfy9ZOOYIF9h1PyP40LEDc64MHmiu0IRnGCE2SQEZRHnmBYrk\\\\\\\/wTkYJ27+9MWGiRNqNqF9hHNwxm4n0AsHie\\\\\\\/kOuQ+DRSKYPLCwOQ6u9iKtucwkunhFHlJh1zH9e\\\\\\\/A7kRpRWyl8tE0jMsVKDqUihqezFakht\\\\\\\/kXl\\\\\\\/zMJq5iEnrCjHtzPr4gOGia6EHxkAKGvfKXo6DxD\\\\\\\/4rQ3bsQEau3iNDCxEI9RASKFKJrS1tYGVckw4TjQwj7mVkJFJljsRw+zO4GD1nLU66mAnVaX\\\\\\\/d7MQ6d7WTEGJFSLzU7Lxt1DL1AiOr+gktVEVL7VdPAPQ84VH7+bqxGO\\\\\\\/6tTTnGw3z1RNo6QWGgjb3WPcGDxni+JlfwvuulEjVN9EVdNXi0IuRjqPhs\\\\\\\/Nm7J78UxIf6950fqbjv46gd96QeGRxYDLKe3aTC\\\\\\\/iZ\\\\\\\/fPxoAznHXFGsZqW1R4WkzPHRp4FwRDW3WNIamjQkD0eZjkbsS20osAJltT5fYuESXPy6a52blfVu7+l3StaDXuJLOTzLYAkFX\\\\\\\/iZjz46E4hmtQmLqvBSmd0DAkqJv\\\\\\\/cvGAtkTKuJiiBjijH3f4TxeheOorR13RSYRfboi7ILjBSiP2b6MLt8MD3I0d0ZV+MIT1jZWuHx3Cdbgdi\\\\\\\/4O3fMouThVSwzpo\\\\\\\/ZqDWaCF8kp20Dtws5LEkeZRQ4+4VZ9CmPe4eJxPcGbfd35Ur1Qa80awhW5Ecp4AZKtClu8DOLOTotsBdj9a3wQfJY\\\\\\\/P0SCaxS8ao8C\\\\\\\/6j3aIuKz6C\\\\\\\/7DUaGmQe\\\\\\\/rE4oyoqgYVtfTwWL4PYu8cIGPTGUgbxLQvmvftK8JpcfORYKQk8WNKRXkHlal5BRLn\\\\\\\/9liZRAA9DdWgZlyp7xgsBVlLaNwnVii9FNq6xD2uanxwktz7wGv1JhwTbUrNEt0XF0iZBl9eCUarbdGub9GNflszyIOLEbFM7mU3qrZRW7juVmm2ShvJCBGctS+ll1NDFLNhprDQulIyasiUFhjet+mig9ZmauqkWht8n2CJLGlCMX4kxZsi+REr89WIhiD9rLUxf2HWHZHNhNKMJ6mJ8JtQn6sTdmOkPET2WYv9yO7yqCJBnnetAMexrZ6ORuKdR4UGB51qOro1dddfnq0ADR5otfDsN2yAMqaRcqgrMo79GZmCe0hAeD56mnkkyvYJPRaZ3zKxuUxdNb4AXbVRFe2pMdSx14J0JWABGy\\\\\\\/KDaD4+MQAUISyqKQIBWW6yckMR1iOR2XGs8o9BNMJQDsiLw5scjCyk7rB9RWRMvp7gwDzJniI27amhIrJhppymLomV1CRpEUTuE04P2hkx1VHIdOGBU+UxLyFHsgME\\\\\\\/Tfm\\\\\\\/dOxmvAFO4JpsvfoYm4WPPAjAuW98qlDDoGGtwkIjJPvUuY48H7OEBQNQCLjQPkuZ9OWdGC3I5g0TEn6yZ7FlcuXWhqK3WuYd9fENP8el6GQIhOD2QFP5JYp0lH11S9tz3V\\\\\\\/uzS0Y9m0I6uDLQKkPackdlCREw0o9ruZ20K92OpzwZASbWCU2jOUGMTSG1t4DKBu+haAFxEiaETuNM8xCkVROqnNZvXrw6NNdcCPu0NUI\\\\\\\/PAEyPCzKsLeI86+74UedMpyDjVHylGGqvloMHT\\\\\\\/YRXUGykKPHN4X8aJ0rQyT2SpPxQdr8GteWG5AsmugKx5CKDrNqc0yi4V8dpErFD8RWewaRhy1Ax1zx5gBnegaSx3IsrNKfKDbGMjC6l9bKINF8imenrEbDWihCey41YHDgK3EMVuakyCjyML0RBFzP18rDr+pMRe7OIPOFNzSJ0HzHZTZicqqymluAovoZBvD8H8oR0Qjxgupb2xZ2FQBOu\\\\\\\/BLZ0YI0AMIDk+O5dgZHA8GW57ywHaWqv3J2VFImMOxsYaJ7MMyaAmRw6cCVjrXYvnq9tFfXOUKR8dLI0nuQ2C9z2Je84xUz7oq5LVOEV7Wx7GOIUpxapuM6jMGVPIpD2UdaFwjpunPAhE5FSpXFRzOyQ3iefjVQOGIXWN6inFOQsA6mSaycd09tjefMln+OBhW7oK3IPQlA2oV6SR5\\\\\\\/KZr1SE6snrN\\\\\\\/id+AIT3i0dKKv9umBVj3Ok\\\\\\\/BqQgJtaUiFfXVfmc2+z9MAhrd5M0rUsoN29Ufu\\\\\\\/yRxmPraWWFblSV1ccLDZCcNRLCcNTtR7S7UBaYpw9TLiDBfFV1ieThRBWCwStr\\\\\\\/ku2MNSRa2UiIVpieFLlaLdHjgXaItA2+14WUaANrC4orPSYaDKxBgF1Uclw0b+ed7LSnByhoB0mULiRxNVXURGnezB7kNgbAPEpf2a+VX5T+yHgqGibsc\\\\\\\/bLNxiIz6NgGgEQDYzndNSyBSgTznmIXBAEsPlfxUechPm+i+TwbqJ9GJ1KSh6z33AuZKKt4yhXDWp0u34Z+ZHUih7VRjSGQjqHHVdwK8\\\\\\\/asQW748eiKOjBBpu9VWkRsvtaTpF\\\\\\\/Z0Qcm515d1bpAKbc7G9xlii9TaiwQ+ao8LnJb1O53dxp8435T3KYXjqFAOLTevXMqnlYx4seEnkMoXbDTswx07W3rByq3cP25C0N9WR+JKZW\\\\\\\/JaumYgqdxzVdE7qEFqwytu+uTuTlZ8K+4Rr8KmPBIsa3dw59gU2keXHi0ABue5WE3eXav6wX\\\\\\\/lsi1LAVE3uWhsZ1MlkVYv4dYqjjUi9Ku2o8HG0qng8kDqwdryTPf3BDythCLJxtkT0NerDu4wLRNzTOoz0MR\\\\\\\/2O5UrEad0YIXXPuTduOUNKv\\\\\\\/4YSCT4Ayn0Lh+idr6hz8HREwK9afCec3X2fwyXu06Ycryt7VnW4Gg9Lncx7DbXS83NI7X+usym3XW4Mhyab7vvkFIH7ke0WnZmxzEmjAQIjldUprXOl7btxkTqOEi34xXtmk22XvlU15XQFumQAKy0svbMnEBrLrwgu4wY2OQ\\\\\\\/iMNmoh9rxoNk7t65HUQEpJFbViB15lzAfN1MbcKjLMQwHz4bDn0Rg5LVpdW+TRTWxikYLandlBBLjlQ1djiecgl9C\\\\\\\/q9WCcu\\\\\\\/TPzaBeiwI3YPuoaI\\\\\\\/uQaAEpa4BuvVIBP09eVE1wjuKVEW4+FAvCG1zQ\\\\\\\/439ofHR3wKUomPXMW0ITldSIl8\\\\\\\/GZ0\\\\\\\/LAywvlWrgUAS6M0gr0ie7vdOTT0tRWGUkMwhM1Szi9Q+JItGP3SGSC1uhUF+cGOY+tDmQJPdxgLxGuLIajUpD48JK9hAoKY7s89vQGkNxD8c4O4pWNM9XkQxGJsS+h50+HlVPmU6v73AE7FkvG0klR4EsM8QHPliIfWWbq0+0\\\\\\\/4cDkwCNTxVX2iyNUO5GLxMT\\\\\\\/84\\\\\\\/g7AmY7ofTjCA69oCXPSNnve\\\\\\\/oiIzfxu+yQ5SVbEZCPcTecOXOrMVztJKeRBYZChfug9Uyzv+Sc4Iiv9kZtQ1z7gkYGjrBGi4SkJooKwRMktjoetQA9+ctsmrjeBaXTn3+BSBznNdfL4kgxEZnsksd\\\\\\\/fn1ZlzEwJ4rL401CZNPyy0MiyhVioIvpfCpCVXeWmfNaQjpPqRJPRaWBetprJDheUOd+PQfyBu4gzn54zhKOvc+b1QcrBySd0eziQHlcODQ\\\\\\\/5UfRIB0NIoSonDe15u6VEZ50NQrw42S8eAUS5Yc+N\\\\\\\/qZ1Ygsoho42tTh4N8gCcjP8koJiprY1F\\\\\\\/4AwvlbDo8umC\\\\\\\/pMVYwRfis0ELRpgLxtwmXTIXn9FWkUiKT\\\\\\\/dkl0fbCUcZRoaFIhSHD8ETXrvuR5prDG3kE29dII4h6NyFm+sWPZbMOt43KAbo3uVC3y4gNWitpMNjV8n4ZL9cZa+PRSs4iRIVxsmCbgUXoghzeBIziidXguZtebp9VYi0H7PaWljtD8+Mt+QLovPibxNQ39WNoU29v0kZop6ES15AlWOPl7mp9tSSI7UokmVhpEH\\\\\\\/EguQT5nsEr+vp288VvaFiAqceiDOpHVJyaZ4VFHrWQzKU77ykkqaWt2W6NyAFCPrW06eLLb2I2Vd3Mfsh4IYrmYblD447TwKGHO1uFtxJemTlDxJlBaRpWVfJ2bo8nwuSjG00anUTT3+Qe8fLelluMZeFIfuva6HqhjPwOJ5+EL4NceNUoHRDIlDJ8T4JytaTZoaGoPchYJXzDpVn+f8xg+7IheF0xHJtM7798s3LtBc0naqvGB\\\\\\\/qS1SyExcQA+xQjw\\\\\\\/DeDxMZyL\\\\\\\/a2GyleUgRTXeePhtPz190npHeocHhKsyUKOIQ7lx4EGYuYhpA6xSsX4LuUB3Txi+VYonMff\\\\\\\/zp8e3ZelcwnxPdYpmHUzflnUQ1mkdE01a4axWLKfno9ANqE6swi7CjkK7YsxRBzW1YZ14Iw2num9gfCqD16nsNC\\\\\\\/D1J4QTFHwrCu9o\\\\\\\/Xw9CXfXwy9Afd707lrkfzP1XH64xC6kriEgZWIsIw1vn\\\\\\\/5aaux2lSno7vIrfoHU93g6Aluug0XcLyrcXmwcjHBoDE2MDjE1kVyy+xMBp9MruUiI3aqqtQKSVspd8AKyN9ddPs38\\\\\\\/N7EaENAD5xiqecz4aaKuTstXLLSUktpwx5uAkDn+q\\\\\\\/0TklgqR7KKMdKDA1BsEgUQLi+UtueGmthaGaXj0K8imLYb1TOoSTVWwDeAPva9I6+DI2GCxNQembKDQMcPu\\\\\\\/LxfoE\\\\\\\/B0Kd9J58qzNu5HXLdwPqbOkXCDYn2Z0ASh8cy75oJILCaI8e8uFPhOB8ESzwjHF6XLWIEOPUKPE\\\\\\\/BV1Y\\\\\\\/6vLBqmCtdlof9cMbkYGPNWrw0sFaIPw6qFcxQhfH3Gge1+L8US8HavE6ZDo9DdtK8UKXqaRG73u578MhNPeFMwgXKQFKhRjWM7aEpq8vc2R2RdnIxnyuPB2v6hrNQsHtrugt8DOmfqRjHD89DgGSzHjPBzrUeLYpRTp5PfNiuJ9b3MBmWVuRrucYqYxQ74m4sDIZQlkkUFAG+gb9eBpvq4uqKUvAst+MKrw84vvmK6I504+jS30XsHhxU6anySk50zajte89YQgYDM5alpGtJ1z\\\\\\\/Ek9jRLgV0hhkSjks094eg\\\\\\\/o7wK3WFNfFPfdcdwrZXlbGoUUVPd8K95JeG1o9IkrKEY\\\\\\\/RQ\\\\\\\/1P6wYFaXzUkbbQhXyUQg+FZRALT7JQWmzEXtRWR8CyAosd4REHeKvC1ewBGaXwzOQjLcx\\\\\\\/BYrMP\\\\\\\/JWXpHyTMiLFlvSZ\\\\\\\/IQQvCWIwCwfhhgqs1YH0Ga3IxXDytuULPmqnwr06k9Dk2d\\\\\\\/5ZhFVWHH\\\\\\\/t2+AC5gEgVEJNMr3xWxRPkv5Xn977RW5xiGAqh0u6lCrwvYQEZODAdLyParpn\\\\\\\/kiWfeAxyu8\\\\\\\/Uca\\\\\\\/4bG365\\\\\\\/3PIiZ0IxriEgQ+K93INXL4aXBYWJMzHLHWDUPJT2Ih1N7AjwdYhuLDHCVwpo4nLoBZopvmn8GhVwurS6TQ4UGmerxWdeA548tVZZfEsDUOPo5GErRdd95GUVMDFr8mo0X8JT7D3kXWTSh3+q5WXbf75Pgxb6za1ywkgcZxsU2GPzWS7xlJNzuhk2hK1dvTEh2AHi0HCFIQuSvTpDj74NR96P+axI5xpgb7upw+5HDPSA83Zi0MW\\\\\\\/PN0kFzSfEytB8pFij69GD9PktFDvbJHxQawM\\\\\\\/26Iqn8kVpLpR1GD83uabaQoFFVArrCup\\\\\\\/4RR8MFNFAC8hgQbdErMqC+A1YcZudfnuvjrBNVprCzICeBdENr9wuxwovWf3iHEk8NOxwjBYPZu0Hl8uuSPkWdA1JNa0\\\\\\\/mVygVsF+lduNHu2IiSrvVvtMwxb4tyi2i\\\\\\\/qmRifetMa990pakHDTtQ4xUQYUm9iRxjoRMdhzT0WqtyX4Y2tRq\\\\\\\/t3UfoQAgQOjaDVD9cio8ynV7xaNzW1m7XOIpb1dImI+3H1rz0Fm8r8qeQ4mkJMPBxGL8Py1teW\\\\\\\/PkraM5B3mzvT9KoBuAssWhoj1TqBO6UgAkaCwRPL558A+pm8Rse0bzdclRfwDqu+v6VCOdKPnqqAhEO7flrHZUz7e9NtmTu2dvQ7UEk1Q7zRdjtDbPtC+8+leglJkx2OKEwvzqMsvodAimz6PA03ZG4Nr+l9j2qO+2duXEE8PQBnNw8MsXAzon2JtnFOIVvuC5wWjoKGrUXJjMlxzCBvoj1Tiuvh0Z4LCOAsy2\\\\\\\/aalktEYcb1KF6lj0MK35AVrrxlBhoK1+sxkFjM0hTe6kzmazJio77\\\\\\\/Gb1gWVN\\\\\\\/WX5nRtDXMNXzYGUKFR1xPSMp6uPdM17Ew+m2Dwzj\\\\\\\/TMrjtpDT7SvhHSrCgHWqFjE0wqMxj1UKyngKfrSUoFw+jdaYvdVkDOzFZCIJGFl3hCS3Hjo8Ti1aV0QNmZ1cnCZiMGqO0sYADmZoBiy00hLOfdQfyIpY35D99+za7yLDIyUeP+Uwx3NnY\\\\\\\/JD9AlqNweVBLsZAOoQt1WZukeOHTUesNyRJjV6fjMr2X+YNjYskMf8UiFeirwAVhRTJuhpMQ72SXS3O3IzWLqAnKTINQKMZtFsq9DScfJWowFm301hvebdTVKWx9Wf3uTKMgNTgAAh24srlu7K5OlvjKu5s6DQctPW9iqSzqmuGidA6EUbzQfQCxAc3KtAODSTN1BEKHA31E99cw1ItDUtWkU09fnJpHvctnaGJ7kOAbRLtp6UfHOG6Jyny3wjohiKo55ZsLIlAyExb1PyLdahtVzQwcpX7Y+97U36j9ueKpzLN2JJimfVoC9HgI0JyeGXxjCUBm7x5lGtj0V74D0\\\\\\\/yH8r548MFdS4MT84AVD8PfdOUvfrbohv2xc1mJw7+LnhWfbaCU0GLOgZDhv8TJd9ZhSqR8KCQDgYO0APUZwdeUbzKMEmWFenpqQprLF3BZIfGIrwoDAy4ra4xDbXs9HZ6EiZvwM61LlCQI3PKLM3l7CA+AzYfNElXqGm5kqO9FcFN40jJWw60tytlzxZuNHndhBfJOhgUhB1ycSFUUGi1VC+Vap3kKI09pyRDIHqjayO0nIoANddKYwEEXVX+ONqe40w\\\\\\\/sK6DfeBu8kS5B7ybTazW5zo7s\\\\\\\/UiE0Jjj+S5aXMGl8oEXooYIMhliVqkLxUzoCawLhogViPb9d7D2rdww+Gbrvp2jMvzlMM87h5F98d7TfNDCLYJ3LfGPHTXN86vWfdZMhebdD4lAycvDetbBnmfqQpnDnM7KVhqibIrNAMNPFYBIRgyMCgv7AR70sXJSA4n\\\\\\\/l5t6pC7DkQhNnNI6V5TLewVm5lo0I8mtpYJQ8CftKf3nKjj7EfFsv5ZHSVPMCPZifqi0xBzS33GegmfK20iVtluSAewYwzUOdzLBHQsL6R7y6SrI2ftNNoRCydGUWEt7VKLmoHCFxXKnD+x93x7RncxcGtXVGD\\\\\\\/DlIFYFLF8pZXBPaUl8OLAaWaQkFTia0ghiJGVmND6bFPhj+QZEmlP0t4OtSEWUpkzWH6YUTlsugFCJtvxZ\\\\\\\/Vcyze\\\\\\\/C6B9gtktEmHQUreJ6vaQUUWTE6EBsJMdk0IiGtgujiPJHM4m8U4G1v6ZsaaNlaMFxW69uU\\\\\\\/mpjcu07p9mNMId4ICL6iJ9pZwzUi\\\\\\\/mTnqk+TCmaMo\\\\\\\/8gZYij8FhEMgqui5yxkJAHG61J4+yq++1vFzws90XdemQ6BuodBBVpOZJ4xuTYcGxkPaQLSyapm+J7HzBV+UZZZHR6rHo3SkL7zmnMYa\\\\\\\/zHL68l3DitJ8eyDVoakdOmIvqYiIPuwKC1s1sIjHvWH3Gs3Ou9qlQSSdMoHPQns5H17J7fnc2aXFEG7vAAPlRwBl2Z6u2\\\\\\\/x2KhPLgag+pNW21Y+UB9SqQqtJYcWou\\\\\\\/mUzFTZsfEDpvCjX0ee4SX63KJ4QkOnNptVGhDMyQHUGqT988WjX5Uo7FWonMOindc712lmrBeXQlTk2i3HgXa4eURxfu3CulflN0CdsqQDAyFzoqAX9HvIOkXiHDDx77OKSeJq65TbZGS2dQalYIpM3b3tkBHcngE9ZA4vNoHdJ6hKIg+66Drh3hEWDCvbK7e2siohmAKhFk2CyuMwwC+b1d6CtNqWjmZYmtXNq+GPPLGDQJnTe+cqILZA3GG2+Cuqi+J603nMsQ1XCEF10AeHYDI9CwqCc7MQ9LXSRxTgAhVfuhsNspl6AGLz99UlSAcV2Sg6zCpPpHm9f0Ck4h51Q2DesEXYxCQuPRz6OmeejHmTvcAQx19\\\\\\\/wp0DG\\\\\\\/xOPUJGkOuzraa8GSXi9ztpKVKQQ3q+7lmwu0vFnI5Ppn3ctrdOgM\\\\\\\/kjjR+FN4VYUktCn9hUkv0afs6GHt3p6au1\\\\\\\/wDGPStmt07XaIYKSxbIa9\\\\\\\/i4MAnme5LBHNix8SgodQvo1XBNfnKy0ICuVxwgW+w7npxyoUOtPlozX8zzc3MrFSuPKmktclaWA6l7RQyexNTxKQNGVbOTim4YKxXzgcLFhlwxMVEMtzqXmrIdOAlSUbRQuNosKPANrVXv3KcQ73PSNvmATcfxuhBimGtt+H5KIJ+oeTEKBDexci6poxq+CmoJ+ExB2KMXNOy3ngLVmfiXpNCDBLlQKIX9F94K9u3MOnol3X6SkhpH6e3RVAERtDg6bSMyoHwt1srZ4tBuBe7KSpAdmOljnlU+X7N8lCYprPKdOwDyu7JdJmiSV4R7REP6N7bDskADlUfWzYqJ0J9lcrfweXjkeH5AWQBHxSJugHaGNeWN0Tsod0XJ2+SvgWbZTsaHkM7HFjDQ8\\\\\\\/wFvZp6fuRaJPyX5bTXzOtppLx+Fwupu8kE\\\\\\\/s7XhlA3J5geqsauWBmFlb44t5Ee7PgjQrxnN2gIH3AY98LsUTnDppqGAoE2SNhsmTs5IDv82LnwzZ9W\\\\\\\/1WPm9WOdu6K4\\\\\\\/aDw7y8fU1DCksiPS5q4U6+TBcoHck5bxDfDukDkjZgC+7D1kwZoVbP\\\\\\\/xeLs5QT9OhTSwSChcacTeYPKeHsjOXk1pryNt9cfdpDt1iVXBQ++dPy+XUj5nv0\\\\\\\/wv4EJ10hDl\\\\\\\/nnulJEevuhfn8AQnS7Zht710ju8DyF+mOgzP1U0pJLg6ba4Tvpa3BEmw8JxXM9N8jm+uto1A\\\\\\\/MrXgIryxoh14RIr0NKNt2PdlJqy6PAkv1Aidy72wRcAMDoIQIoF5ZTDRafCCV7UwSFCSnZGfT2E2EGjEUoKDflt4JlM2v58SYXSAE\\\\\\\/5LemaYTK+1h0gW44dzxvepr5R5Qlc3vbjrz6\\\\\\\/shNld7ig6yxvy+DSjuMUurcWNamWoO5tSLW+Zjrig0hwAENFRHsCviE5A14X3YrTlD2+gWJt8CGeaf8WqLO88ykKtTcJvXp4nxMqwH77upRA29SOLnHaAAgGegyPhoZRiB2FBtjAb7YHtzlkl1f0Wlqae6I38uvxsa\\\\\\\/SLio\\\\\\\/Sb2fXBTtu56mxQtyqiWG6iHmnoGw1RGz9coxLgtLJVOBgaxdZeSKC6KnuTBaSp5QEsZck5kto8ActTp6uM9XYIg21\\\\\\\/43fXKYfpU+d7SeJmvWxIqLhMZRV852+2V3htUzIh9GWlNTpZGUJM+tBem0GpwXqa4MaotX+ATyjA+5tPSAi1on3Sg+U4MoqbNkstKJQRPa4OI8Ntux43h30V1BlrML5Drfjtg6xtWItYsBDwg73lni4Ulmcctgl8+4R0CnBbSR8j050htLPjg0yfM56KwhS5Kx1LC4KsGIMSmJ64haQiydz48zfOtFsKJiyK5+7418MGvqbdYpJCJ+kRE5mApb8t7IpkiKDcNJPhDDcjt5Mk1thd9XO1vywpiUmnuJLYrmWAcdZcCHbsY7oH5p+fmcMLLanGEQO42iJDlCCzfMFdtmHRI9WYw6GXLmOFIk+jScYgdwXV3CU+4m3lboEAx\\\\\\\/0MKBefczGQkY2eflisQpXmaempACsBw1pSZy2rpX+kvJgdMi\\\\\\\/idW8JQmXclj+RJ+GqH3R5TgH8CsSlwG0CIw9bdNRa4zLWQfc\\\\\\\/szpsr5JusKht68cm2AvR+1PGeejGhmrQeYsHkAA4YzbcZGQadJcPo4j\\\\\\\/qRUkVHBJY6mAhpNQhhmnP9zr2288t3mXUJyMtfaNbRj8kI56uxYohuTWx+9Rqx1qS2TYqnCjsDq3ulhka2hKSjqIoyCz0A4gT8QZ8YkOWtH7p3yHoDtcVhqCi81kj69vtScr+l2vYjzjlA4643nBQJ0k6zDpY1cy1\\\\\\\/O98vo\\\\\\\/NQfj86dDBCXu+lI49mg0oCnDkVfQ86PuMDFA4EqZ6SmFWSYtIWwpvqcz8CXEJlIcP160+ZWEBXiyCJTpu1jcbISihu0MUL85kt\\\\\\\/OZkhnLmSGWFyUThzvv+vC8YReAGz\\\\\\\/sDJ0YVZ4Q\\\\\\\/3DYvy8tOxc4NDpvCba3s2aHuPUQ2Mc4p2WvI7oE+FM9+FclEGUJZbLCltj86wqIf7Gb10Y8lxMFAsA4BbGKglKsQPOi66WbOzurYyHUMaRWqFFVjWmUL3+EQ+2H4yMnRwBu6MHlpMqUOVFsXi5gV5pf6zTWSeEv83BxFVQEVBOpW1FhGrntfSLPkfdRem+GdKZehZSZFth42yQ3vJYL2r6mg\\\\\\\/0sR7WyPGwMeVHENCi3zqkyIiZz3UlMVbkmwQCoEhNilNTXtsQT\\\\\\\/pAOeoDhL5xJZR2Sjh\\\\\\\/fW0\\\\\\\/w7BHygNUnONKBU7J04qiw5WurA0y1E\\\\\\\/snfGgiU2KLcawMncQj\\\\\\\/p7bQ9i1X3pVvanJ7\\\\\\\/DUXSpUY\\\\\\\/hm\\\\\\\/TxT1zARCIHgs+32+U2iuHeoOrHEBcEcfUJaD4uGH\\\\\\\/oxCwBLxffAGwnonSYNrlHyJ5+rY9VODNwpV\\\\\\\/V\\\\\\\/WMnD3QM\\\\\\\/FbV6cy4DKr6Rd+Nj46lTsobO3wc65zAipgxEU+HBvOK89p0EUOHHqObnEzjEwyIFK3Lm4\\\\\\\/bGrBLB7Wh2sX80vbqm7zPQYUODp6BKJ\\\\\\\/WkU5lrFDVgd7SPGa+zAuMD7en6FQCexiVlV408LuJOm\\\\\\\/MBALPp4u1Z\\\\\\\/59tUnm5MjytXSUqSrnHK6sguemGy\\\\\\\/M3+ZuCNmFffvh7qhqPaQWbJBxKcrG\\\\\\\/74PxpHSncOMTyPVG8vcMzHmwnavcglAaVi5\\\\\\\/IDiGSogGWjw2ZZ1VmynYNf8w8Wtj1hIIx+G51VZtP\\\\\\\/CinK+sqMtBWnBrFHLo+GzCEWgnJGlJQyryZdNzvoFtJawU8l+vGoTbnABzEu6L5OgjMlcCHf92ui2UbVb728rYjcFpSVTApeFP\\\\\\\/tBeN96Gowq+03MT4PHWLz9HjYPtMkUnrZ7Zr6hgE6ivxE8qW0aBZqHBA69pz00Bja+7mH5r3PFpSzWNBFAx448NIOadLRnui4cg7MC+sZXqnDPAwic0hOkr0J68XzyNIW74CJ9GYUpLeAIB+\\\\\\\/mNE1\\\\\\\/78kDpLAEmzRUzDjC+MnnQRtxIudPXjOD4wjAMy\\\\\\\/4r4c5YvBTRwoD07YncjY8vnB\\\\\\\/q4iIROa+rsre9KPThyCgdYJp8hg+yLdqPjfCMpy1YbeJBQFIK5bNZVQLrFQc7589EnVg1QWWNEy9PDZh+6+uX7BwTPZ2ljS3x+DBYUtRK\\\\\\\/LwvEppA19\\\\\\\/njTtDWbq\\\\\\\/lmVYIybuPCRyHVlORyObmuaHSxaoQUIvwH8PM1ano6RjnZZp1lZDPwC+HyhemVR0hJHBAOlzBYYQlkaKULFq7vzAizG1Iy8JWetPJr0E2CiVdwYy5LzgDjjiC6M3pxdwOwPdAT3MMiR5+oyFMbUpd0Cs4\\\\\\\/r2P10jAPmkZMtrQ\\\\\\\/nFSJEBeeVs5ZQa6VcFzL6+LlNMmLGbqPwHk3py9RFynatW+WJjhAe1Df8cksrfD6o8mGjsFFT8PPOJyoSOh9Ffy4Rv\\\\\\\/S\\\\\\\/q2hHL2flqCS06BKhBS+BqOT7MljURNJoRpVQpkaaua0fAhWxXoyc3aFa4wq\\\\\\\/hBkbuDExe7ANzTbLq4jwZoowZPtnWRDbgwunP5zfuvPi8S2tNGMcrniLOq6mCkPB6NJI6S3iLV0z3h90ZerPzDjEhNNtfgQBtAN80ooNhjOt3nBnh42FoWVjlNNOG6HxuHhg3hi+PxKfgU\\\\\\\/jIooVAxobxQuI53flceS91\\\\\\\/Km3xh0jM7vhW6BFPfZ4dqiqeMkgZ6xwlzEpy0Ng8VmJ1qzd61ZNlPZKFKzzxfyLDADAOduEHOCUKJcrX9CIioUyJRz7EpTTpo24whtFNKtBEVZjIruGUy0BpfHW9JhmZjcZuZrm88WarcYPwypYgg+f56bphQkOXJnsV+9mDMJut8WK5h+lj1MXoz6SDNqrU1TAZiGyouDY5M82bcj1NWgrQqHlmJ8s7QQQGKizU8IhIIrpPzI1hnvdLWbGxT6EJfWLfAQ1mnSU4WU5+kn61gDMoag8j+T0\\\\\\\/Ta3FR1VFlQ9lq4mWjN8dE5XjVQUoGroscfLP9nKCwMwci6RjMT06iUA2zwL1os0SvVoMOqUL\\\\\\\/pS6ZP0RpvwyeeZKjacNxl2Z3a8T\\\\\\\/KUihJR58hIOe6ZgVgIDiPNjnE5xL2XrFprxpPkgzhFx8P+cK8zGfYqSu1l5kXnfX+htKQiAJNysbfFkLvfmvTB6hrKNBWEB4\\\\\\\/2utDJdt9MUD\\\\\\\/sFmi5O\\\\\\\/ofcEkyxFskigHAjijLRhNrk72JEiWC27brDhj96T7md5lO1woPtuXnBHv8m2coC1XqngTxqmfZYlC\\\\\\\/QShJPnrAEdF\\\\\\\/+M76su77T2\\\\\\\/4Lz\\\\\\\/6HQkl5wer9seLVKzcC8qH+gmtVJl7cMrkz\\\\\\\/EHiAGv1G36\\\\\\\/CIrmaKWVwdvQQBZDFnkmtOfyK4SRyPZk0yKzSOJsYQxoPZnL3R6jeCW5i3HpuFcgZvVl9esmHD+yU53mAose6moh0PU5q8LdockkyIknFyl1OUa2aFeXf0FlNUJjbz6pgEiCe2smVAN7qABX5ZTAIIwP\\\\\\\/JKT2slmfVcpMyuSoe0wypOUtV86yoOlvFGousUqsHMI32+HEUy7fg98YEnNS\\\\\\\/LiXfSynHwjl2m1cT9Nz3r0bCrDZqL5UIr3cLTirI1fXNT7fxG\\\\\\\/FNtp46o4YSjsuzfOSUvmAXTq3tWQh8SQ1EjwEVhS615l\\\\\\\/3ZAZkK73VapMYabOuhGxNjBGXe8KJS\\\\\\\/MTLd5T4yd\\\\\\\/CLl7FsUzApUmyg7fO2lUuUVIHkOU5y\\\\\\\/UAmJ7ZgICSRCyTPpmGJNx02Z6Ru2jZ\\\\\\\/Js0L1slr2YShEz3a8EmbFWfzSoOjgTIqyomZ6qfEDj3dgEFPRWi8+dwigGgn+smdD69ABhMfZUzxi3YDn+TGp23lshWy32g9AY6hfKChFYG1VxY0Z9ZHZBVIdtfkhM3LsWj9sY3DOcPS0HNGbe3L5ZkCYkkoQJ0l0oL794crCdjkyqtdsGJrcbRBdIKLl4WLCwlniRRYnQ835N9sRZnEDHWg4YQjdgnEu37SVfaOvb468ovOH34s9AWS+FiFQdkYZNv+zcV2kaG\\\\\\\/Tb6ROOCeSw2POnp4m33E2TZpRmOEZkJSI5Z0xDnb7FoJZSSi66CbZffA1BAMvNSxe9AFcANut8H0VW6hbJujacinQeLl5aeUch+6\\\\\\\/u9FTapyxOgt8RsNE2kFisTJ6oDipqmXWhd8obUu6GsUxn9po4W1h7vFxQY1llYZiOx8vkdd5j9tntQLQx8Il+5++2TuNY9kWeWDJIubFkM0M0dpXNioXMHCM4PxHut6+lSkT6JsDIo7Wr0Fbq70sYQuNE3IUFEpSltyeYElDdoCpdLviwkY\\\\\\\/H\\\\\\\/3t0LoBjsdBE5oKfhXQDumxEea8NP73GPhPtYLB+O6UJYFgpv722kXunIUYWtq1hNt+tVyBeRvg7g4nYZA1X3XZGmkb96PtvHO\\\\\\\/vpdDRI6YWK2Rxq7YTGpcbOL+\\\\\\\/Hxd8o2quGErYRo80i9zfuOvOWto6c1\\\\\\\/QowskQCiHD7nvOFbrhuLbwgiOB8DtDUVdZZNSzv5XYOUQUsMhS7VGykxoj+dy5W8Jyb1lNsa1sOepOarOGtD1DQ2ZsmG4zHNw0O+fNyQLMRf94VrylnDGwzEjQBvITIp8n72eKOL8FZw1P05zrlMgrXXe68HJg\\\\\\\/IGk75uh+L3OhIUs6SBFV0sEEgDh0wpSfeubplmk2izEaZRLsbU6hpdpaU6c+rf+lD7DLsIWn+zpV08869ePiCJIoImNHKra2gi+6BZsASWmbZ3prBAAyxk7nVpfR4EaPiJE7tRyCt5uoXdxDj8P2NIFmrhAHzRJib6Xe3E8fKL27Hyb\\\\\\\/+xYoaK855u5CHKtvQxmQy1Jt1cPfpRaBr55dS8LJCamYCWOwduY1pVq9tubk6Jol7Vr2kT6ke+tcnqtidp8vMy3dDiheKcsYbCPdNYVZ6H09r5olODVXcNBaircKKlbvnYy6zP\\\\\\\/CKHDTbPqpXMRyZy7yM2IKNpZi4N3ISgUl+aNtFwZY9LW8GuKn+Cklkjwttt6Pn8qJvGFqpk1RyFHiyif3F3wpLOD46tymK2oczlvoZj3BWDCEa5FTu7kquFrTZHKMfkHAaRf3F3noZNlPEZykPEowCbIHsbDT18VtjNL1wYznBKu\\\\\\\/z110o3e0nbY5SJTv7EiQ2HWd86SYyugaM55ehAvWsnJYqoEgNUlwFsk\\\\\\\/XDgAN3pvR88wsKCvKC1DxE3nvcd+Ttpo\\\\\\\/+bf0co\\\\\\\/SJjWB+lhTARmyqLPIs\\\\\\\/VRybR9tq4D0aw6Q+IEoKvWHBzyaN9QXLu7+sXAe8+fa4B+rsdV0ZDFEs1i1KE3aPkBpKRRSvRIKOFmrWHdrswE31Z6EBxlbSilXSPD9zwQmgNfZvV3pOH59J855QUJ4HSWEdaZgPIIWJDhpxbPOwVtjeTy\\\\\\\/FBy1teRv15u65x2GNWo6NG95+lIGaAXejR\\\\\\\/5NP9mBAiKuDhWkpdcy0Q1lK7F6j\\\\\\\/3Ni2xR7No9dVsHrYozC+mx2hBUckdFVGejoW5nCzVzQA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022V4Dle\\\\\\\/2WEsg8AH2xLqBRVg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227334cbe3a4ae955ca2dc5238bb6c2265b8d9eca121fe12daceb381a751425760\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022JmR1m7TtbnUj0lPOrBa2bZE2NoEOg0Zefnytv2chYLldQbrxP9yMw1IossPU0ygtgfhz5hdQg79XMAYQ9qN+vZNQi\\\\\\\/Z3N7oiofXOms7vrk6\\\\\\\/cD0ztqM2I5ZJ2AHYXCS5oI6GyoUGNlTXLFvg2g71nA7Mn1Sa3jREaP4udi9zsHhQzck+Ed6G4ua+LfepBrNFsqhLyexBUdznLdacWqAv5r3L0KMwmzltVPlU82U46FMo3E9Rb6iKiN6mqwm15HPak3D1mrCbZxx34BxBMVc1U\\\\\\\/F69uXFH0BRC6+\\\\\\\/vIaSgQrL5\\\\\\\/D4jmM\\\\\\\/QjHEiFTHewZlNegl4uYX47jkuYojWsgNmMV+TI8vAdYysdjdZ5VRFoAMZnz258bDzOYhsiyA8e9BMlOluVcLoqx5cMqnyfzC\\\\\\\/74QxUofJTkXr+\\\\\\\/r9zBIjte5RGuZq0lzXIM6yEY1gVq4zWWQ\\\\\\\/yTxfImLYK7PfgAaXyXzrXuH0erp\\\\\\\/p7nmlcwmibiVWBfM+++L5iByY\\\\\\\/lBN5gJWxSprYW3wkSCaSa7evhO1wQKEa2q0mCJ3y+rY\\\\\\\/x6HQ0rzeTdndFs3c0z9ALX3w3IoE83kU5DxPgA0sxSrxKoJdKW2Wk3ZR7\\\\\\\/Dr2yu8v0bzIfhWh7Er3yLY61kJZUzyvdJZBCaayyOvylFv9w\\\\\\\/Sj2+IdSlOrWlIzLxfNfarOyYj8eGFqGZJhQZdBksd0Y8lXa4KlGniHftPEIaTyqYV0n4Eh\\\\\\\/FDk78N4armbu0dB6pS\\\\\\\/vWbn62bMQD\\\\\\\/6hdquo5MfxCIpW42G\\\\\\\/eTdcEBALKhR+py0kY8OU+3\\\\\\\/PK4zJsKCw\\\\\\\/8LKXEZufykLlAqFBC8AylYN2iqQRTMyU\\\\\\\/6uwyOj9BW6X8bXefoGOjPgCMUPGGJexLWs9\\\\\\\/7wjnORvnR9ak0E4uIOJlGl1PWWISYanydmAU9UUaIbdcr40wc3LTf1tJ0e1ZrXjq8E1D+4uqsP7GkvoMDP0XzPTb6L9Egdv+uhgwCTE+CEKnDVOmhrCcD+0+juEqauqqqH\\\\\\\/gKBLtRhUzqiO+4koLjiHUUwzJjXZrwXo2eTXujM5OS94adj2FD+pgz6s1KXMYJeiN8UqPAURPEmzoSgO179M2iZjhAMTag5U8WHXOhcKHMntMRcqPn2d\\\\\\\/v0I1fyTnAF\\\\\\\/VvIliQWN77aMyD45aCuTcZ564jD5Jq3JwspX4txIZ1f26rKXaXhSJrE8p8utu\\\\\\\/TVmX90bhNqGrU1WsDOlnWl3oRG9WXIZ+886v\\\\\\\/AOTQ2utzC2gyqKrmo4mINTql7NwijqSRifnhhZK\\\\\\\/z6EOmO7NHepkomTOdIHwjLf9XJNpvQvkpBWsnc2FqPoDE8wezeKyR8GN0+mW2kvDBf8u1LsEEtyEo76llIkCj0C2G88Z2WffPRryxfP1PNzJpuZOZZe6YowdkaLCmxpD1zi9F16vXpUb+ztryErEtRhZytSOpY1CgMdd2sxmd7izZkyues6mfGONf3c59cHPbsBSO1fLcro113qYg+INSj5Gp2C9W9\\\\\\\/TAcZP\\\\\\\/IzDp7W\\\\\\\/YODz\\\\\\\/6CnNnpW3J8G5ay6IfaR7h98IBgjyfVTwh+xLs24tUlI\\\\\\\/LM+AP+gv32Ae58agLDUmY8mi8c\\\\\\\/PgwdP4gu2+9ddMQ3dDf9w5ns9puLGaFbrRfE0YLFKTNOIFTvoDCDJ6koQzsDvPYQu88UAb0QuxkbboK3pUnUrgjESDrw9+bP5W\\\\\\\/KANHTYEaccwDmf+DQptKdTIlpqAsgAJz9WCMOz5yXn9H1OSKSY1jw8AxrgvDNUJ72mE\\\\\\\/qfvkxV0kRqhbMnoM\\\\\\\/s3JxFHlvvrMh7qAjE118K8IUI5fbqU6ZgGpz\\\\\\\/4ndKtqNbWKUky3cWdpl8R7vqAO7027nPL\\\\\\\/1ygHhoaxGMKb36U+zhTqmVrjuNKlQvaUPYUDMK7sQe\\\\\\\/s4aE\\\\\\\/xKSKIEJ1HbgL4KBdk\\\\\\\/AYTN4PoMTxXJoqNIvzoHMfD37gsQIMSmjeo63ok9vLMTh88Lnzqx92SSQIvnsxliTqO+gJNRDWB0wHYbWxx1CDEz\\\\\\\/7xWf7Ng==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ovUJFuScTLMybYolItrjvw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002254507e5ac84a1f18951b2a6dc81cc922d54913696450b102548bfd38de44f538\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460114 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/845" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-kdJ7NAJTxm3S7FMELgKrGsmORQhvPCcbVVoj9rhccuE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/845" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-AvmZBOrHsxzdZNcnEyVSWoj4HqTjnlOocyB_UMWeH2M" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/4/835" title="Environmental Association Analyses Identify Candidates for Abiotic Stress Tolerance in Glycine soja, the Wild Progenitor of Cultivated Soybeans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/4/857" title="Haplotag: Software for Haplotype-Based Genotyping-by-Sequencing Analysis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460114" id="node460114" data-pisa="ggg;6/4/845" data-pisa-master="ggg;g3.115.025775" data-apath="/ggg/6/4/845.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Increasing Genome Sampling and Improving SNP Genotyping for Genotyping-by-Sequencing with New Combinations of Restriction Enzymes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-3106-1247" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yong-Bi Fu</span>, <span class="highwire-citation-author" data-delta="1">Gregory W. Peterson</span> and <span class="highwire-citation-author" data-delta="2">Yibo Dong</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">845-856;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.025775 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460114" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yong-Bi Fu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Gene Resources of Canada, Saskatoon Research and Development Centre, Agriculture and Agri-Food Canada, Saskatoon, Saskatchewan S7N 0X2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yong-Bi%2BFu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYong-Bi%2BFu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3106-1247" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yong-Bi Fu</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:yong-bi.fu@agr.gc.ca" class="" data-icon-position="" data-hide-link-title="0">yong-bi.fu@agr.gc.ca</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gregory W. Peterson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Gene Resources of Canada, Saskatoon Research and Development Centre, Agriculture and Agri-Food Canada, Saskatoon, Saskatchewan S7N 0X2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregory%2BW.%2BPeterson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peterson%20GW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGregory%2BW.%2BPeterson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yibo Dong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Gene Resources of Canada, Saskatoon Research and Development Centre, Agriculture and Agri-Food Canada, Saskatoon, Saskatchewan S7N 0X2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yibo%2BDong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dong%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYibo%2BDong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/4/845" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460114" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460114/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/845.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460114" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460114/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/845.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460114" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460114/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/845.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460114" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460114/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/4/845.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460114" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460114/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genotyping-by-sequencing (GBS) has emerged as a useful genomic approach for exploring genome-wide genetic variation. However, GBS commonly samples a genome unevenly and can generate a substantial amount of missing data. These technical features would limit the power of various GBS-based genetic and genomic analyses. Here we present software called IgCoverage for <em>in silico</em> evaluation of genomic coverage through GBS with an individual or pair of restriction enzymes on one sequenced genome, and report a new set of 21 restriction enzyme combinations that can be applied to enhance GBS applications. These enzyme combinations were developed through an application of IgCoverage on 22 plant, animal, and fungus species with sequenced genomes, and some of them were empirically evaluated with different runs of Illumina MiSeq sequencing in 12 plant species. The <em>in silico</em> analysis of 22 organisms revealed up to eight times more genome coverage for the new combinations consisted of pairing four- or five-cutter restriction enzymes than the commonly used enzyme combination <em>Pst</em>I + <em>Msp</em>I. The empirical evaluation of the new enzyme combination (<em>Hin</em>fI + <em>Hpy</em>CH4IV) in 12 plant species showed 1.7–6 times more genome coverage than <em>Pst</em>I + <em>Msp</em>I, and 2.3 times more genome coverage in dicots than monocots. Also, the SNP genotyping in 12 <em>Arabidopsis</em> and 12 rice plants revealed that <em>Hin</em>fI + <em>Hpy</em>CH4IV generated 7 and 1.3 times more SNPs (with 0–16.7% missing observations) than <em>Pst</em>I + <em>Msp</em>I, respectively. These findings demonstrate that these novel enzyme combinations can be utilized to increase genome sampling and improve SNP genotyping in various GBS applications.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genotyping-sequencing" class="hw-term hw-article-keyword hw-article-keyword-genotyping-by-sequencing" rel="nofollow">genotyping-by-sequencing</a></li><li class="kwd"><a href="/keyword/restriction-enzyme-combination" class="hw-term hw-article-keyword hw-article-keyword-restriction-enzyme-combination" rel="nofollow">restriction enzyme combination</a></li><li class="kwd"><a href="/keyword/genome-coverage" class="hw-term hw-article-keyword hw-article-keyword-genome-coverage" rel="nofollow">genome coverage</a></li><li class="kwd"><a href="/keyword/snp-genotyping" class="hw-term hw-article-keyword hw-article-keyword-snp-genotyping" rel="nofollow">SNP genotyping</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/silico-analysis" class="hw-term hw-article-keyword hw-article-keyword-in-silico-analysis" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">in silico</em> analysis</a></li></ul><p id="p-3">Genotyping-by-sequencing (GBS) has emerged as a useful genomic approach for exploring genetic variation and performing association mapping on a genome-wide scale (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Huang <em>et al.</em> 2009</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Fu and Peterson 2011</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Poland and Rife 2012</a>), thanks to the advances in next-generation sequencing technologies (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Metzker 2010</a>). The GBS approach is a combined one-step process of SNP marker discovery and genotyping through genome reduction with restriction enzymes (RE) (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altshuler <em>et al.</em> 2000</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">van Orsouw <em>et al.</em> 2007</a>) and SNP calls with or without a sequenced genome (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Fu and Peterson 2011</a>). This approach has displayed a major advantage of being rapid, high throughput, and cost-effective for genome-wide analysis of genetic variation and association mapping (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Davey <em>et al.</em> 2011</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Poland and Rife 2012</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fu <em>et al.</em> 2014</a>). However, GBS usually samples a genome unevenly (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Beissinger <em>et al.</em> 2013</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Schilling <em>et al.</em> 2014</a>) and can generate SNP data with a large proportion of missing observations across assayed samples (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Marchini and Howie 2010</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Rutkoski <em>et al.</em> 2013</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fu 2014</a>). These technical features would limit the power of various GBS-based genetic and genomic analyses (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Pool <em>et al.</em> 2010</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Nielsen <em>et al.</em> 2011</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Crawford and Lazzaro 2012</a>).</p><p id="p-4">Efforts have been made to recover information from GBS SNP data with missing observations. Several imputation methods based on row averages, row medians, and data correlation (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Little and Rubin 1987</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Horton and Kleinman 2007</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Carpenter and Kenward 2013</a>) have been applied to infer missing genotypes for a genetic analysis (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Troyanskaya <em>et al.</em> 2001</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Iwata and Jannink 2010</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Weigel <em>et al.</em> 2010</a>). Specific efforts to impute unordered SNP genotype data have also been made using regression-based methods such as Random Forest (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Breiman 2001</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Stekhoven and Bühlmann 2011</a>) and principal component analysis-based tools (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Stacklies <em>et al.</em> 2007</a>) with the hope of improving genomic selection (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Poland <em>et al.</em> 2012b</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Rutkoski <em>et al.</em> 2013</a>). These efforts help to regain some missing information, particularly from those datasets with fewer than 30% observations missing, but the information gain is less ideal (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Fu 2014</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Huang <em>et al.</em> 2014</a>) as some GBS applications could generate up to 90% missing data in SNP genotyping (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Fu and Peterson 2011</a>).</p><p id="p-5">Research into efficient genome sampling by various REs for genome reduction has helped to develop different GBS protocols (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Fu and Peterson 2011</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Peterson <em>et al.</em> 2012</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Poland <em>et al.</em> 2012a</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Sonah <em>et al.</em> 2013</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Peterson <em>et al.</em> 2014</a>). Specifically, <a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Elshire <em>et al.</em> (2011)</a> presented a GBS protocol using a methylation-sensitive RE, <em>Ape</em>KI, followed by the release of the two-enzyme (<em>Pst</em>I + <em>Msp</em>I) protocol by <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Poland <em>et al.</em> (2012a)</a> and the double digest RADseq protocol by <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Peterson <em>et al.</em> (2012)</a>. The two-enzyme protocol has good technical merit and has gained popularity in application for generating high-density genetic maps (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Poland <em>et al.</em> 2012a</a>), but alternative protocols also exist (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Andrews <em>et al.</em> 2014</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Puritz <em>et al.</em> 2014</a>). Overall, these efforts have enhanced the genome sampling of the GBS approach, particularly with the double-digest protocols (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Peterson <em>et al.</em> 2012</a>; <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Poland <em>et al.</em> 2012a</a>). However, these GBS protocols may not have adequate genome sampling for high-density mapping, nor sufficiently reduce missing data (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Beissinger <em>et al.</em> 2013</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Schilling <em>et al.</em> 2014</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Sims <em>et al.</em> 2014</a>). Further efforts have been made to improve the GBS efficiency in genome sampling (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">De Donato <em>et al.</em> 2013</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Heffelfinger <em>et al.</em> 2014</a>; <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Schilling <em>et al.</em> 2014</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Peterson <em>et al.</em> 2014</a>), particularly with the use of more effective enzyme combinations (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hamblin and Rabbi 2014</a>).</p><p id="p-6">We conducted a search for better RE combinations for GBS applications with the goal of increasing genome sampling and improving SNP genotyping. Our search focused on the development of software, IgCoverage, for <em>in silico</em> evaluation of genomic coverage through GBS with an individual or a pair of REs on one sequenced genome, and had a larger scope with 70 RE combinations and 22 model organisms than the previous efforts (<a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Poland <em>et al.</em> 2012a</a>; <a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Peterson <em>et al.</em> 2012</a>). The specific objective of this search was to explore a new set of RE combinations for improving GBS applications through <em>in silico</em> analyses of 22 organisms and empirical evaluations in some plant species. It was our hope that this exploration would generate a list of candidate RE combinations for broader GBS application in different species, and provide some useful tools for further searches for specific RE combinations for GBS applications in a species of particular interest.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-7">Our search for new RE combinations with increased genome sampling for a GBS application was done through comparisons of genome coverages of a species obtained by the new RE combinations and the commonly used RE pair <em>Pst</em>I + <em>Msp</em>I, either from an <em>in silico</em> analysis or empirical validation. In this study, we defined the genome coverage as the proportional genome covered by DNA fragments digested by a RE or RE pair. To avoid confusion, we named IgC and EgC as the genome coverages of a species estimated from the <em>in silico</em> analysis and empirical validation, respectively. Specifically, the search had five major interactive components through <em>in silico</em> analysis and empirical evaluation (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). They are (C1) <em>in silico</em> analysis of IgC for single REs, (C2) designing RE pairs, (C3) <em>in silico</em> analysis of IgC for RE pairs, (C4) empirical verification of IgC for some RE pairs on plant species, and (C5) further empirical verification for some RE pairs on individual plants. The details of each major step are described below.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/845/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Procedures used to explore new REs for GBS application. A flow chart for exploring restriction enzyme (RE) pairs for a genome-by-sequencing (GBS) application to increase genome coverage of a species through in silico analysis and empirical validation. The genome coverage is measured by the proportion of the genome covered by a selected set of DNA fragments digested with a RE or RE pair. IgC and EgC are the genome coverages of a species estimated from in silico analysis and empirical validation, respectively. Two shell scripts (IgCoverage1RE.sh and IgCoverage2RE.sh) are part of software IgCoverage developed for this study." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1534031285" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Procedures used to explore new REs for GBS application. A flow chart for exploring restriction enzyme (RE) pairs for a genome-by-sequencing (GBS) application to increase genome coverage of a species through &lt;em&gt;in silico&lt;/em&gt; analysis and empirical validation. The genome coverage is measured by the proportion of the genome covered by a selected set of DNA fragments digested with a RE or RE pair. IgC and EgC are the genome coverages of a species estimated from &lt;em&gt;in silico&lt;/em&gt; analysis and empirical validation, respectively. Two shell scripts (IgCoverage1RE.sh and IgCoverage2RE.sh) are part of software IgCoverage developed for this study.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/845/F1.medium.gif" width="440" height="265"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/4/845/F1.medium.gif" width="440" height="265"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/845/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/845/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467978" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-8" class="first-child">Procedures used to explore new REs for GBS application. A flow chart for exploring restriction enzyme (RE) pairs for a genome-by-sequencing (GBS) application to increase genome coverage of a species through <em>in silico</em> analysis and empirical validation. The genome coverage is measured by the proportion of the genome covered by a selected set of DNA fragments digested with a RE or RE pair. IgC and EgC are the genome coverages of a species estimated from <em>in silico</em> analysis and empirical validation, respectively. Two shell scripts (IgCoverage1RE.sh and IgCoverage2RE.sh) are part of software IgCoverage developed for this study.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-2" class="subsection"><h3>C1: in silico analysis of IgC for individual REs</h3><p id="p-9">The <em>in silico</em> search for informative REs started with the selection of 60 REs (Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025775/-/DC1/TableS1.pdf">Table S1</a>) by considering the enzyme usage, recognition site and length, methylation sensitivity, active temperature, and unit cost. The majority of the selected enzymes were four- to six-cutters (or site-recognition), and more than half were methylation sensitive. For each enzyme, we performed <em>in silico</em> cuts of sequenced genomes of four plant species with sequenced genomes (<em>Arabidopsis thaliana</em>; <em>Oryza sativa</em>, rice; <em>Zea mays</em>, maize; <em>Glycine max</em>, soybean) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025775/-/DC1/TableS2.pdf">Table S2</a>). This was done through the development and application of specific software, IgCoverage (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025775/-/DC1/FileS1.rar">File S1</a>), to calculate the IgC as the total DNA fragments of different lengths (100–600 bp) as the ratio of the total fragment length over the sequenced genome of reported sequence length (as shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025775/-/DC1/TableS2.pdf">Table S2</a>). We considered a wider range of fragment lengths (100–600 bp) than the current GBS protocols (200–400 bp), mainly for relative IgC comparison and possible technical advances in future NGS platforms for GBS applications. The shell script IgCoverage1RE.sh in IgCoverage was run in a combination of 60 enzymes and four plant genomes. These IgC results (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) were used to guide the development of new RE combinations for new two-enzyme GBS protocols as described below.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467931/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467931/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">The <em>in silico</em> genome coverages (IgC; %) of four plant species by DNA fragments of different lengths (100–600 bp) obtained from <em>in silico</em> digestions with 60 individual restriction enzymes</span><div class="sb-div caption-clear"></div></div></div><p id="p-11">The software IgCoverage (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025775/-/DC1/FileS1.rar">File S1</a>) was specifically developed and tested for <em>in silico</em> evaluation of expected genome coverage through GBS with an individual or a pair of REs on one sequenced genome. Two shell scripts, IgCoverage1RE.sh and IgCoverage2RE.sh, were written to calculate IgC values for an individual or a pair of REs, respectively. Both functions carry several major steps. First, the genome-sequence files in FASTA format of a species were downloaded from the NCBI database (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025775/-/DC1/TableS2.pdf">Table S2</a>). These FASTA files were renamed according to their respective chromosomes and used as input files. Second, the recognition sequence and cutting position of each RE was obtained as an input file and used to scan over a chromosome for the recognition site positions. Third, for each chromosome, DNA fragments were defined based on the positions of recognition sites. Four fragment metrics were recorded: the total number of DNA fragments, the total length of all DNA fragments, the number of DNA fragments of lengths 100–600 bp, and the total length of the DNA fragments within 100–600 bp. Fourth, a summation of the four fragment metrics over all the chromosomes was made for the whole genome. IgC was calculated for the species. The four fragment metrics and IgC value were saved as output. Fifth, the second to fourth steps were repeated for other REs, and an output file with four fragment metrics and IgC for each enzyme was generated. This script was rerun for each species.</p></div><div id="sec-3" class="subsection"><h3>C2: development of new RE combinations</h3><p id="p-12">A total of 70 RE combinations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025775/-/DC1/TableS3.pdf">Table S3</a>) were designed for <em>in silico</em> and empirical evaluations based on their genome coverage and/or SNP genotyping. These enzymes were paired or selected with the following considerations. First, we focused on two-enzyme GBS protocols with dual-indexing of sequencing runs, and pairs of enzymes were selected, as opposed to single enzymes. Second, enzymes needed to be readily available from a common supplier, capable of generating sticky (overhanging) ends for efficient ligation of adapters, and, preferably, to have compatible reaction buffers and the same incubation temperature to allow for simultaneous digestion. Third, we reasoned that the enzymes with shorter recognition sites would generate more, shorter fragments and that more fragments would lead to higher genome coverage. Thus, enzyme combinations with shorter recognition sites were favored and the following combinations were chosen: 40 5 + 4 bp, 25 6 + 4 bp, two 4 + 4 bp, and one 5 + 5 bp (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025775/-/DC1/TableS3.pdf">Table S3</a>). In addition, two RE combinations with longer recognition sites were selected: one 6 + 6 bp and one 6 + 8 bp (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025775/-/DC1/TableS3.pdf">Table S3</a>). For comparison, six RE combinations were also selected from previously published works (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Vos <em>et al.</em> 1995</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Maughan <em>et al.</em> 2009</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fu and Peterson 2012</a>; <a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">Peterson <em>et al.</em> 2012</a>; <a id="xref-ref-31-6" class="xref-bibr" href="#ref-31">Poland <em>et al.</em> 2012a</a>), including two long recognition site enzyme pairs, 6 + 6 bp and 8 + 6 bp. Fourth, plant genomes vary in overall GC content (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Šmarda <em>et al.</em> 2014</a>). Methylation of cytosine bases can also inhibit the ability of certain enzymes to cut DNA and could reduce the number of potential cut sites. Thus, some RE combinations were selected to address the variability in the GC content of the enzymes’ recognition sites (low, equal, and high, relative to the AT content). There were 47 enzyme combinations with at least one enzyme being methylation sensitive, as indicated in the supplier’s literature. Fifth, the enzymes were also favored if displaying high <em>in silico</em> IgC values in four model plant species (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), and with even digestion patterns and/or large proportions of fragments less than 600 bp from <em>in vitro</em> tests over five plant species [rice, maize, soybean, wheat (<em>Triticum aestivum</em>), flax (<em>Linum usitatissiumum</em>)]. The <em>in vitro</em> digestions were conducted with 24 single REs and 23 RE combinations at 37° for 3 hr using 10 units of each enzyme and 100 ng of genomic DNA with the optimum buffer. Digests were separated on a 2% agarose gel at 100 V for 2 h and visualized by stain with GelRed (Biotium, Hayward, CA, USA) postrun.</p></div><div id="sec-4" class="subsection"><h3>C3: in silico analysis of IgC for RE pairs</h3><p id="p-13">The <em>in silico</em> search for promising RE pairs had several steps. First, we downloaded 18 additional genome sequences from the NCBI database to enlarge the scope of the species assay (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025775/-/DC1/TableS2.pdf">Table S2</a>). Second, the shell script IgCoverage2RE.sh in IgCoverage (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025775/-/DC1/FileS1.rar">File S1</a>) was applied to perform the <em>in silico</em> analysis of double-enzyme cutting of a sequenced genome for each RE combination. Specifically, IgCoverage2RE.sh simulated the digestion of the genome with two REs, and calculated the IgC value of the generated DNA fragments of different ends and with lengths 100–600 bp. Note that we considered only the DNA fragments of different ends and lengths within 100–600 bp as these fragments are likely sequenced in the current GBS protocols. The IgCoverage2RE.sh script was run for 70 RE combinations in one species and the run was repeated for the other 21 assayed species (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025775/-/DC1/TableS2.pdf">Table S2</a>). Third, we also assessed the distribution of the DNA fragments with different ends and lengths within 100–600 bp generated <em>in silico</em> by certain enzyme combinations on some plant genomes, and compared the abundance of DNA fragments generated by these RE combinations at particular genomic regions. Note that IgCoverage2RE.sh differs from IgCoverage1RE.sh mainly in that the former considers the cutting positions by two enzymes over a chromosome and only the DNA fragments of different ends for the IgC estimation.</p></div><div id="sec-5" class="subsection"><h3>C4: empirical verification on plant species</h3><p id="p-14">The IgC verification was performed for three enzyme combinations (<em>Pst</em>I + <em>Msp</em>I = PM, <em>Ava</em>II + <em>Bfa</em>I = AB, and <em>Hin</em>fI + <em>Hpy</em>CH4IV = HH) on 12 plant species (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). PM was the GBS reference RE pair and used for control, while AB and HH were selected to represent the enzyme combinations with the moderate and the high IgC values, respectively (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025775/-/DC1/TableS3.pdf">Table S3</a>).</p><p id="p-15">The HH pair was selected, as opposed to other RE pairs with higher IgC values, as it seemed to have the best combination of genome coverage, enzyme cost, and practical efficiency for verification on plant species. For example, <em>Hpy</em>CH4IV has a 50% GC-content recognition site (ACGT) compared to the AT-rich <em>Mse</em>I site (TTAA) and thus is more likely to avoid noncoding AT-rich regions (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Haberer <em>et al.</em> 2005</a>), especially in large, AT-rich genomes of cereals (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Šmarda <em>et al.</em> 2014</a>). The other RE pairs have different incubation temperatures (<em>e.g.</em>, <em>Cvi</em>AII at 25°, <em>Hin</em>fI and <em>Dde</em>I 37°, <em>Tfi</em>I 65°, and <em>Ape</em>KI 75°), requiring some modification of the current GBS protocols. Some of these pairs, such as <em>Mlu</em>CI/<em>Hin</em>fI, had higher IgC values for 22 species, but relatively lower IgC values in plant species. Also, <em>Bfa</em>I has a short shelf life and strict storage requirements, increasing its overall cost for a routine GBS use.</p><p id="p-16">Six monocots (maize; rice; <em>Elymus lanceolatus</em>, northern wheatgrass; <em>Aegilops umbellulata</em>, goat-grass; <em>Pseudoroegneria spicata</em>, bluebunch wheatgrass; and <em>Agropyron cristatum</em>, crested wheatgrass) and six dicots (<em>Arabidopsis</em>; soybean; <em>Linum grandiflorum</em>, red flax; <em>Carthamus tinctorius</em>, safflower; <em>Pisum sativum</em>, pea; and <em>Sinapis alba</em>, yellow mustard) were selected to represent both model and nonmodel plant genomes of variable size. An individual plant with young leaf tissue collected from our previous genetic diversity research was selected to represent its species. DNA was extracted from leaf tissue and the same GBS procedures as described in <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Peterson <em>et al.</em> (2014)</a> were used, but with adapters modified to anneal to the specific enzyme pairs. Two MiSeq runs were made, and each run consisted of three monocots and three dicots. The MiSeq Reporter software was set to produce demultiplexed data in both the forward and reverse directions in FASTQ format. The FASTQ files for each species/enzyme combination were processed separately.</p><p id="p-17">The bioinformatics analysis of genome coverage for each enzyme combination in each species was performed in several steps. First, the MiSeq FASTQ data were cleaned to remove low quality sequences using Trimommatic (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bolger <em>et al.</em> 2014</a>) with a five-base sliding window, and with a quality cut-off using a PHRED score of 24 and a minimum sequence length of 75 bases. Second, unique sequences were obtained independently from each enzyme and species combination using FastX_Collapser (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gordon 2010</a>), followed by the <em>de novo</em> assembly of contigs using Minia (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Salikhov <em>et al.</em> 2013</a>). Minia was run with a k-mer size of 31, the minimum k-mer abundance of 2 and the genome size of 100,000,000. Third, the total number of contigs and the total number of bases from all contigs from each enzyme and species combination were counted with a custom Perl script. Empirical genome coverage (EgC) was estimated as the number of bases from all contigs against the estimated genome size of a plant species (listed in <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>) from published flow cytometry values from the Plant DNA C-values Database, Kew Royal Botanical Gardens (<a href="http://data.kew.org/cvalues/">http://data.kew.org/cvalues/</a>). There was no report on the genome size of red flax, and its EgC was estimated using the reported genome size of cultivated flax in the Plant DNA C-values Database. Comparisons of EgC values for AB and HH to PM were also made for understanding genome sampling. Note that such a relative comparison of species-specific EgC or IgC values between two RE pairs should not be affected by the use of either estimated genome size (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>) or published genome sequence length (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025775/-/DC1/TableS2.pdf">Table S2</a>) of a species.</p></div><div id="sec-6" class="subsection"><h3>C5: empirical verification on individual plants</h3><p id="p-18">The verification of IgC and SNP genotyping in 12 <em>Arabidopsis</em> and 12 rice individual samples was conducted with HH and PM. The assayed materials (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>) represented 12 races of <em>Arabidopsis</em> and 12 accessions of rice. Seeds were randomly selected from each race or accession and planted in the greenhouse at the Saskatoon Research and Development Centre. Young leaf tissue was specifically collected from a single plant representing a race or accession, and DNA was extracted. Following the gd-GBS protocol (<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Peterson <em>et al.</em> 2014</a>), with adapters modified to anneal to the specific enzyme pairs, two additional MiSeq sequencing runs were conducted. Each MiSeq run consisted of all 48 indexed accession/enzyme combinations from both species, and generated 96 demultiplexed paired-end FASTQ files of each sample/enzyme combination for the two species. The contig assembly and SNP genotyping were performed using the npGeno pipeline with default settings (<a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">Peterson <em>et al.</em> 2014</a>). The statistics on contigs and resulting SNPs, along with their read statistics, were compiled for each enzyme combination in each species. To assess the dynamics of missing values in SNP genotyping for each enzyme combination, total SNPs were analyzed and plotted with respect to the number of samples present and average number of reads per sample for both plant species.</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-19">Online supplementary information contains <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025775/-/DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025775/-/DC1/FileS1.rar">File S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025775/-/DC1/TableS1.pdf">Table S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025775/-/DC1/TableS2.pdf">Table S2</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025775/-/DC1/TableS3.pdf">Table S3</a>. The genome sequence data for 22 assayed organisms are available in the NCBI database with direct links listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025775/-/DC1/TableS2.pdf">Table S2</a>. The original Illumina MiSeq data (in FASTQ format), with two runs for IgC verification on 12 plant species and with two runs for verification on 12 <em>Arabidopsis</em> and 12 rice samples, are available at the NCBI with Sequence Read Archive Project ID: SRP066269. The developed software, IgCoverage.rar (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025775/-/DC1/FileS1.rar">File S1</a>), is available as an online supplementary file.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Genome coverage from in silico analysis</h3><p id="p-20">The <em>in silico</em> digestions by 60 individual REs of the sequenced genomes of four plant species revealed a huge variation in the counts of DNA fragments of different lengths (100–600 bp) and a large variation in genome coverage by these generated DNA fragments in each species (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). For example, the numbers of such DNA fragments obtained in <em>Arabidopsis</em> ranged from 0 (with the <em>Not</em>I enzyme) to 376,984 (with the <em>Cvi</em>KI-1 enzyme), and IgC values ranged from 0– 67.3% for these 60 enzymes. Such large variation in genome coverage was also observed for rice, maize, and soybean. The commonly used enzyme <em>Ape</em>KI displayed the average genome coverage of 17.5%, but there were 22 four- or five-cutter enzymes displaying more genome coverage than <em>Ape</em>KI in these four plant species (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). Generally, frequent-cutter enzymes generated more DNA fragments and showed higher genome coverages. Overall, there were six four-cutter enzymes and 14 four-cutter or five-cutter enzymes having genomic coverages higher than 50% and 30%, respectively. The top cutter enzymes for these four species were either <em>Cvi</em>AII or <em>Cvi</em>KI-1, with the genome coverages larger than 60%.</p><p id="p-21">The <em>in silico</em> digestions with 70 RE combinations of the sequenced genomes of 22 organisms also revealed a large variation in genome coverage in each species (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025775/-/DC1/TableS3.pdf">Table S3</a>). For example, the IgC values for 70 enzyme combinations ranged from 0.1–35.1% and averaged 14% in <em>Arabidopsis</em>, while they ranged from 0.1–30.1% and averaged 10.8% in fruit fly. For most of the RE pairs, there was also a large variation in IgC across the 22 species. Generally, larger IgC values were observed in plant than animal species. More variation was observed in the species with larger genomes. Overall, across the 22 species, there were 11 enzyme combinations having an average IgC of more than 20%, and 21 combinations with more than 15%. The top three RE pairs, with IgC values of 30% or higher across the 22 species, were <em>Cvi</em>AII + <em>Hin</em>fI, <em>Cvi</em>AII + <em>Dde</em>I, and <em>Bfa</em>I + <em>Hin</em>fI. In contrast, the commonly used enzyme combination <em>Pst</em>I + <em>Msp</em>I (or PM) displayed an averaged 4% IgC across the 22 species. The 21 enzyme combinations with the IgC values of 15% or higher (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>) are recommended for consideration for various GBS applications in diverse species, as they could generate 3.8–8 times more genome coverage than the GBS reference RE pair PM. These 21 RE pairs also displayed no significant associations detected between their IgC values and the genome sizes of the 22 species.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467945/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467945/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">The <em>in silico</em> genome coverages (IgC; %) of 22 species with sequenced genomes (eight plant, 13 animal, and one fungus) by DNA fragments of different ends and lengths (100–600 bp) obtained from <em>in silico</em> digestions with the top 21 restriction enzyme pairs and the GBS reference pair <em>Pst</em>I + <em>Msp</em>I</span><div class="sb-div caption-clear"></div></div></div><p id="p-24">To verify the differences in genome coverage among these enzyme combinations, we also assessed DNA fragment distributions for the <em>in silico</em> digestions by three selected RE combinations (PM, AB, HH) on all the chromosomes of <em>Arabidopsis</em> and rice. <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows the DNA fragment distributions on two chromosomes of <em>Arabidopsis</em> and rice. Clearly, HH generated more different DNA fragments than AB and PM on the assayed chromosomes of each species. For example, HH had an average of 37 DNA fragments on each 100 kb sliding-window of chromosome 1 of <em>Arabidopsis</em>, while AB and PM had only 23 and 7, respectively. Similarly, HH had an average of 36 DNA fragments on each 100 kb sliding-window of chromosome 1 of rice, while AB and PM had only 28 and 8, respectively. Note that the striking drops in fragment counts (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2, C and D</a>) reflected the assembly gaps in the respective rice chromosomes.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/845/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fragment distributions detected in silico on selected chromosomes of Arabidopsis and rice. Distributions of DNA fragments generated by in silico digestions with three restriction enzyme combinations on two chromosomes of Arabidopsis thaliana (A, B) and Oryza sativa (C, D). The number of DNA fragments and the average enzyme-cutting position based on a 100 kb sliding-window of a given chromosome are calculated and shown with a colored line for each enzyme combination. The corresponding horizontal linear line represents the average fragment count for an enzyme combination on the chromosome. More digestions were found for HinfI + HpyCh4IV than the other two enzyme combinations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1534031285" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Fragment distributions detected in &lt;em&gt;silico&lt;/em&gt; on selected chromosomes of &lt;em&gt;Arabidopsis&lt;/em&gt; and rice. Distributions of DNA fragments generated by &lt;em&gt;in silico&lt;/em&gt; digestions with three restriction enzyme combinations on two chromosomes of &lt;em&gt;Arabidopsis thaliana&lt;/em&gt; (A, B) and &lt;em&gt;Oryza sativa&lt;/em&gt; (C, D). The number of DNA fragments and the average enzyme-cutting position based on a 100 kb sliding-window of a given chromosome are calculated and shown with a colored line for each enzyme combination. The corresponding horizontal linear line represents the average fragment count for an enzyme combination on the chromosome. More digestions were found for &lt;em&gt;Hin&lt;/em&gt;fI + &lt;em&gt;Hpy&lt;/em&gt;Ch4IV than the other two enzyme combinations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/845/F2.medium.gif" width="440" height="323"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/4/845/F2.medium.gif" width="440" height="323"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/845/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/845/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467873" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-25" class="first-child">Fragment distributions detected in <em>silico</em> on selected chromosomes of <em>Arabidopsis</em> and rice. Distributions of DNA fragments generated by <em>in silico</em> digestions with three restriction enzyme combinations on two chromosomes of <em>Arabidopsis thaliana</em> (A, B) and <em>Oryza sativa</em> (C, D). The number of DNA fragments and the average enzyme-cutting position based on a 100 kb sliding-window of a given chromosome are calculated and shown with a colored line for each enzyme combination. The corresponding horizontal linear line represents the average fragment count for an enzyme combination on the chromosome. More digestions were found for <em>Hin</em>fI + <em>Hpy</em>Ch4IV than the other two enzyme combinations.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Genome coverage at the species level</h3><p id="p-26">We performed an empirical verification of genome coverage for the three selected RE combinations (PM, AB, HH) with two runs of MiSeq GBS sequencing on six monocot and six dicot plant species. Several interesting results emerged (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). First, the numbers of contigs and the averaged contig lengths generated by each enzyme combination varied substantially, with variable genome sizes of the 12 species. Second, each enzyme combination also displayed a wide range of variation in genome coverage across the 12 species. For example, PM displayed EgC values ranging from 0.1–3.02%, and HH had EgC values ranging from 0.55–7.90%. Third, larger EgC values were observed for AB and HH than for PM in each species. For example, the average EgC across these 12 species was 0.88% for PM, 2.28% for AB, and 2.85% for HH. The same pattern of variation was observed when either monocots or dicots were considered separately. Fourth, AB and HH displayed 3.27 and 3.45 times more genome coverage than PM across the 12 species, respectively. Interestingly, AB and HH displayed larger EgC values in dicots than in monocots. Fifth, the linear regression analyses revealed significant associations of the EgC values obtained by AB and HH, but not by PM, with the genome sizes of the 12 assayed plant species (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025775/-/DC1/FigureS1.pdf">Figure S1</a>). Smaller EgC values for AB and HH were observed in species of larger genome size.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468030/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468030/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">The empirical genomic coverages (EgC) for three restriction enzyme combinations (PM = <em>Pst</em>I + <em>Msp</em>I, AB = <em>Ava</em>II + <em>Bfa</em>I, and HH = <em>Hin</em>fI + <em>Hpy</em>CH4IV) and the ratio of the EgC relative to PM (Ratio to PM) in six dicot and six monocot species</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Genome coverage and SNP discovery at the individual level</h3><p id="p-28">We also performed an empirical verification of genome coverage for two selected RE combinations (PM, HH) with two runs of Miseq GBS sequencing on 12 <em>Arabidopsis</em> and 12 rice plants and found several patterns of variation (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). First, the numbers of contigs and the averaged contig lengths generated by each enzyme combination varied greatly among the 12 samples of each species. Second, each enzyme combination displayed more contigs (on average) in the rice than <em>Arabidopsis</em> samples. Third, HH displayed 2.8 and 2.6 times more genome coverage (average of all the samples) than PM in <em>Arabidopsis</em> and rice samples, respectively. For the sample-wise genome coverage, this pattern was also true, but more variation in sample-wise genome coverage was observed in <em>Arabidopsis</em> than rice samples.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467954/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467954/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Statistics of contig and mean contig length per sample obtained for two restriction enzyme combinations (PM = <em>Pst</em>I + <em>Msp</em>I; HH = <em>Hin</em>fI + <em>Hpy</em>CH4IV) in combined runs of 12 <em>Arabidopsis</em> and 12 rice samples</span><div class="sb-div caption-clear"></div></div></div><p id="p-31">We also conducted SNP genotyping for each of the two selected RE combinations in each species using the npGeno pipeline, and generated some basic statistics of contigs and SNPs obtained from the GBS SNP discovery with respect to sequence read and missing data (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). First, more contigs with fewer sequence reads were obtained for HH than PM for either species, suggesting more genome coverage for HH. Accordingly, more average reads per contig were observed for PM than HH. For example, the average read per contig per sample for PM was 50.3 in <em>Arabidopsis</em> and 16.1 in rice, while for HH they were 10.7 in <em>Arabidopsis</em> and 8.0 in rice. Second, more SNPs with fewer reads per SNP per sample were identified for HH than PM for either species. For example, HH displayed 11,489 and 11,526 SNPs in <em>Arabidopsis</em> and rice, respectively, while PM showed 1343 and 7886 SNPs in <em>Arabidopsis</em> and rice, respectively.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468042/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468042/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Statistics of contigs and single nucleotide polymorphisms obtained for two restriction enzyme combinations (PM = <em>Pst</em>I + <em>Msp</em>I; HH = <em>Hin</em>fI + <em>Hpy</em>CH4IV) in combined runs of 12 <em>Arabidopsis</em> and 12 rice samples</span><div class="sb-div caption-clear"></div></div></div><p id="p-33">To understand if a new enzyme combination can also improve GBS SNP genotyping with more SNPs having fewer missing observations, we performed further SNP analysis to assess the extent of missing data in SNP genotyping for each selected RE combination in either species. It was found that there were many more SNPs with 0–16.7% missing observations [or absent in zero, one, or two (out of 12) samples] for HH than PM in either species (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>). For example, there were 6168 SNPs for HH <em>vs.</em> 884 SNPs for PM in <em>Arabidopsis</em>, and 6261 SNPs for HH <em>vs.</em> 4753 SNPs for PM in rice. Also, there were more contigs detected (with SNPs having up to 16.7% observations missing) for HH than for PM in either species. For example, there were 5405 contigs for HH <em>vs.</em> 672 contigs for PM in <em>Arabidopsis</em>, and 5096 contigs for HH <em>vs.</em> 3687 contigs for PM in rice.</p><p id="p-34">To understand the dynamics of missing data in SNP genotyping under a new enzyme combination, we assessed the distribution of total SNPs detected with respect to read count per sample and to the number of samples present (or those having the SNP) for two enzyme combinations (PM, HH) in <em>Arabidopsis</em> and rice samples (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). It is clear that PM displayed more SNPs without missing observations (or present in all 12 samples) than HH in either species (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3, A and B</a>). This is not surprising, as SNPs from PM had more reads per sample in either species (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3, C and D</a>). For example, 28% (<em>Arabidopsis</em>) and 66% (rice) of the detected SNPs for PM had more than 21 reads per sample, while the majority of the detected SNPs from HH had only 4–7 reads per sample. However, it was also observed that the combined number of SNPs with 0–16.7% missing observations was larger for HH than PM in either species (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3, A and B</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/845/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SNP distributions for two RE pairs in Arabidopsis and rice. Distribution of total single nucleotide polymorphisms detected with respect to the number of samples present, and average number of reads per sample, for two restriction enzyme combinations (PM = PstI + MspI; HH = HinfI + HpyCH4IV) in combined runs of 12 Arabidopsis and 12 rice samples." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1534031285" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;SNP distributions for two RE pairs in &lt;em&gt;Arabidopsis&lt;/em&gt; and rice. Distribution of total single nucleotide polymorphisms detected with respect to the number of samples present, and average number of reads per sample, for two restriction enzyme combinations (PM = &lt;em&gt;Pst&lt;/em&gt;I + &lt;em&gt;Msp&lt;/em&gt;I; HH = &lt;em&gt;Hin&lt;/em&gt;fI + &lt;em&gt;Hpy&lt;/em&gt;CH4IV) in combined runs of 12 &lt;em&gt;Arabidopsis&lt;/em&gt; and 12 rice samples.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/845/F3.medium.gif" width="440" height="351"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/4/845/F3.medium.gif" width="440" height="351"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/845/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/845/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468086" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-35" class="first-child">SNP distributions for two RE pairs in <em>Arabidopsis</em> and rice. Distribution of total single nucleotide polymorphisms detected with respect to the number of samples present, and average number of reads per sample, for two restriction enzyme combinations (PM = <em>Pst</em>I + <em>Msp</em>I; HH = <em>Hin</em>fI + <em>Hpy</em>CH4IV) in combined runs of 12 <em>Arabidopsis</em> and 12 rice samples.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-36">Our search for better RE combinations has produced not only a useful <em>in silico</em> analytical tool, IgCoverage (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025775/-/DC1/FileS1.rar">File S1</a>), for further analysis of an individual or a pair of REs for GBS applications in a species of particular interest, but also revealed a novel set of 21 combinations of four- or five-cutter REs with significant increases in genome coverage for different GBS applications (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). The empirical evaluation of the <em>Hin</em>fI + <em>Hpy</em>CH4IV combination in 12 plant species yielded 1.7–6 times more genome coverage than the commonly used RE pair <em>Pst</em>I + <em>Msp</em>I, and 2.3 times more genome coverage in dicots than monocots (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>). Further SNP genotyping analysis showed that <em>Hin</em>fI + <em>Hpy</em>CH4IV generated 7 and 1.3 times more SNPs with 0–16.7% missing observations than <em>Pst</em>I + <em>Msp</em>I in separate MiSeq sequencing runs of 12 <em>Arabidopsis</em> and 12 rice plants, respectively (<a id="xref-table-wrap-5-3" class="xref-table" href="#T5">Table 5</a>). These findings demonstrated the potential of new enzyme combinations for advancing GBS applications with increased genome sampling and improved SNP genotyping.</p><p id="p-37">Our search focused on the two-enzyme system, but the <em>in silico</em> analysis of genome coverage for a single enzyme system in four plant species is also encouraging. For example, the four-cutter enzyme <em>Cvi</em>AII showed an averaged genome coverage of 64%, while the commonly used enzyme <em>Ape</em>KI displayed an averaged genome coverage of 17.5%. There were 22 four- or five-cutter enzymes displaying more genome coverage than <em>Ape</em>KI in these plant species (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). Thus, it is possible for further improvement of GBS protocols with the selection and evaluation of the single enzyme system of genome reduction. More research is needed in this direction to search for better genome sampling with adequate read depth (<a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Beissinger <em>et al.</em> 2013</a>; <a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Schilling <em>et al.</em> 2014</a>).</p><p id="p-38">The <em>in silico</em> analysis of the two-enzyme system was not exhaustive in either enzyme combination or assayed organism, but our search scope was much larger than those reported so far (<em>e.g.</em>, <a id="xref-ref-31-7" class="xref-bibr" href="#ref-31">Poland <em>et al.</em> 2012a</a>; <a id="xref-ref-28-6" class="xref-bibr" href="#ref-28">Peterson <em>et al.</em> 2012</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Heffelfinger <em>et al.</em> 2014</a>). In spite of this, searching further for better enzyme combinations for increased genome coverage is still encouraged, particularly for a specific species of interest. Also, we focused only on the genome coverage and did not consider the issue of read depth, as the latter can be optimized with a given sequencing effort (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Beissinger <em>et al.</em> 2013</a>). However, our effort generated an encouraging outcome with a novel set of 21 combinations of four- or five-cutter REs with increased genome coverage when compared with the GBS reference pair <em>Pst</em>I + <em>Msp</em>I. The enzyme combinations with large genome coverages appeared to be those four- or five-cutter pairs (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). There was only one pair of five-cutters (<em>Ava</em>II + <em>Ape</em>KI) displaying an IgC value of 22.4%, and one pair of four-cutters (<em>Msp</em>I + <em>Mse</em>I) with a value of 11.7%, across the 22 species. These results suggest that the search for the best two-enzyme GBS protocol should focus on those combinations of four- or five-cutters. Also, the increases in genome coverage for these enzyme combinations vary greatly across the 22 assayed species. This variation may reflect the genome size and structure. For example, the plant species with larger genomes (or abundant repeat sequences and genome duplication) appeared to show higher genome coverages (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). Such variation may also reflect the selection of DNA fragments with different ends and lengths within 100–600 bp, as the other DNA fragments not matching these criteria were excluded from consideration for genome coverage.</p><p id="p-39">One encouraging finding in empirical verifications in plants is that the enzyme combinations with larger EgC values also generated many more SNPs with a mild (0–16.7%) level of missing data than <em>Pst</em>I + <em>Msp</em>I. Also, it was found that the read depths of such SNPs are more shallow (with 4–7 reads on average) in these enzyme combinations, suggesting error rates for these SNPs would be expected to be higher than those using <em>Pst</em>I + <em>Msp</em>I. We focused more on the counts of GBS SNPs with the mild level of missing observations, and less on the accuracy of the SNPs and the optimization of read depth <em>vs.</em> SNP density. All of these issues are dependent on the total sequencing output of a GBS effort (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Wendl 2006</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Sims <em>et al.</em> 2014</a>). Higher accuracy of SNPs requires an increased sequence read output per sample, through either more sequencing runs with higher sequencing cost or decreased multiplexing with fewer samples. More research is needed to optimize the parameters of genome coverage, read depth, SNP accuracy, and sequencing effort for a defined research goal in a species (<a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Beissinger <em>et al.</em> 2013</a>; <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Heffelfinger <em>et al.</em> 2014</a>). This optimization is critical for the application of GBS to species with large genomes. <em>e.g.</em>, grass species. The expected number of DNA fragments generated by an enzyme combination is proportional to the genome size. The more DNA fragments generated for larger genomes, the more sequencing output is needed to identify accurately the fragment with adequate read depth (<a id="xref-ref-3-6" class="xref-bibr" href="#ref-3">Beissinger <em>et al.</em> 2013</a>; <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Sims <em>et al.</em> 2014</a>).</p><p id="p-40">The empirical verification also revealed smaller values of genome coverage for those assayed enzyme combinations than those obtained from the <em>in silico</em> analysis (<a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a> and <a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a>). Such variation should not be surprising for several reasons. First, many factors may have contributed to the selection and identification of the DNA fragments. The <em>in silico</em> analysis considered much wider fragment length (100–600 bp), while the actual GBS runs may consider only those of length 200–400 bp. Bioinformatics analysis may have also presented some bias in excluding some DNA fragments for contig identification for both comparative enzyme combinations. It is also possible that some contigs generated by Minia may not be fully unique and still had overlapping sequences among some contigs, biasing upward the genome coverage calculation. Second, a sequencing flow cell has a finite number of binding sites for fragments and, once saturated, additional fragments cannot be sequenced. This technical feature would greatly affect the efficiency of more frequently cutting RE pairs in read output. This was evident as more frequently cutting RE pairs had a lower read depth compared to less frequently cutting combinations (<a id="xref-table-wrap-5-4" class="xref-table" href="#T5">Table 5</a>). Third, our verification effort focused more on the comparative genome coverage, but not on the absolute extent of gain in genome coverage. It was our goal to explore those candidate RE combinations with possible gain in genome coverage, not to test and develop the best RE combination in a specific species.</p><p id="p-41">Our empirical verifications were performed on plant species with a few selected enzyme combinations to illustrate the potential of improvement in genome coverage, thus biasing against animal species. The <em>in silico</em> analysis revealed 21 enzyme combinations with 15% or higher genome coverage in a wide range of plant, animal, and fungus species, which is much more than <em>Pst</em>I + <em>Msp</em>I (4%). More research is needed to verify some of these enzyme combinations in animal species. This is especially important, given the finding of larger variation in genome coverage in 13 animal species (<a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025775/-/DC1/TableS3.pdf">Table S3</a>). Similarly, our <em>in silico</em> analysis of the single enzyme system was done only with plant species. It is possible to expand the search effort for animal species, along with empirical verification (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">De Donato <em>et al.</em> 2013</a>). Our search for higher genome coverage and the developed associated software were largely aimed at genetic diversity and population genetic studies, but both are equally useful for research into different interests focusing on a few hundred loci with higher read depth. For example, searching with IgCoverage for REs with less genome coverage would allow for an optimized experimental design with more samples to be assayed in a sequencing lane.</p><p id="p-42">We are confident, however, that these novel enzyme combinations (<a id="xref-table-wrap-2-8" class="xref-table" href="#T2">Table 2</a>) will play a role in future GBS applications, as these RE combinations can increase genome sampling and improve SNP genotyping. For optimum use of these RE combinations in GBS protocols, several factors are worth mentioning for consideration. First, when selecting an enzyme combination based on its performance in this study, it should be used with a species closely related to one of the relevant species assayed <em>in silico</em> here. Alternatively, a new <em>in silico</em> analysis using IgCoverage should be pursued to assess specific REs or pairs of REs in the genome sequence of the species of interest, or that of a closely related species if sequence data are not available for the first. Second, the adapters used in the GBS protocol need to be modified to anneal to the fragment ends generated by the selected enzyme combination. Third, a preliminary empirical assessment of genome coverage for the selected RE or RE combination in the species of interest is recommended before pursuing a large scale GBS application. Fourth, some effort may also be needed to optimize the enzyme combination for use in a specific species with respect to sequence output, read depth, SNP accuracy, and the extent of multiplexing (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Hamblin and Rabbi 2014</a>; <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Heffelfinger <em>et al.</em> 2014</a>). Pursuing too high a degree of genome coverage may not always be the best option, as it may generate less accurate SNPs with a given sequencing effort (<a id="xref-ref-3-7" class="xref-bibr" href="#ref-3">Beissinger <em>et al.</em> 2013</a>). Fifth, some attention should be paid to the limitations of our search effort, the desired level of genome coverage and the workable level of missing observations in SNP data.</p><div id="sec-13" class="subsection"><h3>Conclusion</h3><p id="p-43">Our <em>in silico</em> analyses of 22 plant, animal, and fungus species produced a novel set of 21 combinations of four- or five-cutter REs with increased genome coverage for a GBS application. The empirical evaluations of some new enzyme combinations in plant species confirmed the increase in genome coverage and the improvement in SNP genotyping. The developed <em>in silico</em> analytical tool, IgCoverage, should also be useful for further analysis of an individual or a pair of REs for GBS applications in a species of particular interest.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-44">We would like to thank Carolee Horbach for her technical assistance in sequencing with MiSeq; Daoquan Xiang, Raju Datla, Lily Tang, and Elena Beynon for supplying <em>Arabidopsis</em> seed and leaf tissue; and two anonymous journal reviewers for their helpful comments on the early version of the manuscript. This research was funded by an Agriculture and Agri-Food Canada A-Base research project to Y.B.F.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-8"><p id="p-45">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025775/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025775/-/DC1</a></p></li><li class="fn" id="fn-9"><p id="p-46"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-12-07"><span class="received-label">Received </span>December 7, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-01-22"><span class="accepted-label">Accepted </span>January 22, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Fu <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.1" data-doi="10.1038/35035083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altshuler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pollara</span>  <span class="cit-name-given-names">V. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cowles</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Etten</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baldwin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">An SNP map of the human genome generated by reduced representation shotgun sequencing.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">407</span>: <span class="cit-fpage">513</span>–<span class="cit-lpage">516</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAltshuler%26rft.auinit1%253DD.%26rft.volume%253D407%26rft.issue%253D6803%26rft.spage%253D513%26rft.epage%253D516%26rft.atitle%253DAn%2BSNP%2Bmap%2Bof%2Bthe%2Bhuman%2Bgenome%2Bgenerated%2Bby%2Breduced%2Brepresentation%2Bshotgun%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35035083%26rft_id%253Dinfo%253Apmid%252F11029002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35035083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11029002&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F845.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089727400050&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.2" data-doi="10.1111/mec.12964"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andrews</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hand</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trade-offs and utility of alternative RADseq methods.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">5943</span>–<span class="cit-lpage">5946</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D5943%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12964&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000346771900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.3" data-doi="10.1534/genetics.112.147710"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beissinger</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sekhon</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foerster</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Marker density and read depth for genotyping populations using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">1073</span>–<span class="cit-lpage">1081</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.147710%26rft_id%253Dinfo%253Apmid%252F23410831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5My80LzEwNzMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvODQ1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.4" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usadel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: A flexible trimmer for Illumina sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMzAvMTUvMjExNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC84NDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.5" data-doi="10.1023/A:1010933404324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breiman</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Random forests.</span> <abbr class="cit-jnl-abbrev">Mach. Learn.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMach.%2BLearn.%26rft.volume%253D45%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1010933404324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1010933404324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000170489900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.845.6" data-doi="10.1002/9781119942283.ch5."><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carpenter</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenward</span>  <span class="cit-name-given-names">M. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-source"><em>Multiple imputation of unordered categorical data</em>, <em>in multiple imputation and its application</em></span>, <span class="cit-publ-name">John Wiley &amp; Sons, Ltd</span>, <span class="cit-publ-loc">Chichester, UK</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep">, .</span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/9781119942283.ch5.</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMultiple%2Bimputation%2Bof%2Bunordered%2Bcategorical%2Bdata%252C%2Bin%2Bmultiple%2Bimputation%2Band%2Bits%2Bapplication%26rft_id%253Dinfo%253Adoi%252F10.1002%252F9781119942283.ch5.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/9781119942283.ch5.&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crawford</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lazzaro</span>  <span class="cit-name-given-names">B. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Assessing the accuracy and power of population genetic inference from low-pass next-generations sequencing data.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFront%2BGenet%26rft.aulast%253DCrawford%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D3%26rft.spage%253D66%26rft.epage%253D66%26rft.atitle%253DAssessing%2Bthe%2Baccuracy%2Band%2Bpower%2Bof%2Bpopulation%2Bgenetic%2Binference%2Bfrom%2Blow-pass%2Bnext-generation%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Apmid%252F22536207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22536207&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F845.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.8" data-doi="10.1038/nrg3012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">J. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thureen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide genetic marker discovery and genotyping using next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDavey%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D510%26rft.atitle%253DGenome-wide%2Bgenetic%2Bmarker%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3012%26rft_id%253Dinfo%253Apmid%252F21681211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21681211&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F845.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.9" data-doi="10.1371/journal.pone.0062137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Donato</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peters</span>  <span class="cit-name-given-names">S. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hussain</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imumorin</span>  <span class="cit-name-given-names">I. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genotyping-by-sequencing (GBS): a novel, efficient and cost-effective genotyping method for cattle using next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e62137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De62137%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0062137%26rft_id%253Dinfo%253Apmid%252F23690931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0062137&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23690931&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F845.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.10" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F845.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.11" data-doi="10.1534/g3.114.010942"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic diversity analysis of highly incomplete SNP genotype data with imputations: an empirical assessment.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">891</span>–<span class="cit-lpage">900</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.010942%26rft_id%253Dinfo%253Apmid%252F24626289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjQvNS84OTEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvODQ1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.12" data-doi="10.3835/plantgenome2011.08.0022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">G. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic diversity analysis with 454 pyrosequencing and genomic reduction confirmed the eastern and western division in the cultivated barley gene pool.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">226</span>–<span class="cit-lpage">237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D226%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.13" data-doi="10.1111/j.1755-0998.2011.03100.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">G. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Developing genomic resources in two <em>Linum</em> species via 454 pyrosequencing and genomic reduction.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">492</span>–<span class="cit-lpage">500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D12%26rft.spage%253D492%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2011.03100.x%26rft_id%253Dinfo%253Apmid%252F22177006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2011.03100.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22177006&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F845.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.14" data-doi="10.1007/s10722-013-0058-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">G. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic diversity analysis of yellow mustard (<em>Sinapis alba</em> L.) germplasm based on genotyping by sequencing.</span> <abbr class="cit-jnl-abbrev">Genet. Resour. Crop Evol.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">579</span>–<span class="cit-lpage">594</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BResour.%2BCrop%2BEvol.%26rft.volume%253D61%26rft.spage%253D579%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10722-013-0058-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10722-013-0058-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-web" id="cit-6.4.845.15"><div class="cit-metadata unstructured">Gordon, A., 2010 FASTX-toolkit. Available at <a href="http://hannonlab.cshl.edu/fastx_toolkit/index.html">http://hannonlab.cshl.edu/fastx_toolkit/index.html</a>. Accessed January 18, 2016.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.16" data-doi="10.1104/pp.105.068718"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haberer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bharti</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gundlach</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raymond</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Structure and Architecture of the Maize Genome.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">139</span>: <span class="cit-fpage">1612</span>–<span class="cit-lpage">1624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DHaberer%26rft.auinit1%253DG.%26rft.volume%253D139%26rft.issue%253D4%26rft.spage%253D1612%26rft.epage%253D1624%26rft.atitle%253DStructure%2Band%2BArchitecture%2Bof%2Bthe%2BMaize%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.105.068718%26rft_id%253Dinfo%253Apmid%252F16339807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTM5LzQvMTYxMiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC84NDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.17" data-doi="10.2135/cropsci2014.02.0160"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamblin</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabbi</span>  <span class="cit-name-given-names">I. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The effects of restriction-enzyme choice on properties of genotyping-by-sequencing libraries: a study in cassava (<em>Manihot esculenta</em>).</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">2603</span>–<span class="cit-lpage">2608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D2603%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.02.0160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.02.0160&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.18" data-doi="10.1186/1471-2164-15-979"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffelfinger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fragoso</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreno</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Overton</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mottinger</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Flexible and scalable genotyping-by-sequencing strategies for population studies.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">979</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D979%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-979%26rft_id%253Dinfo%253Apmid%252F25406744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-979&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25406744&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F845.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.19" data-doi="10.1198/000313007X172556"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleinman</span>  <span class="cit-name-given-names">K. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Much ado about nothing: A comparison of missing data methods and software to fit incomplete data regression models.</span> <abbr class="cit-jnl-abbrev">Am. Stat.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BStatistician%26rft.stitle%253DThe%2BAmerican%2BStatistician%26rft.aulast%253DHorton%26rft.auinit1%253DN.%2BJ.%26rft.volume%253D61%26rft.issue%253D1%26rft.spage%253D79%26rft.epage%253D90%26rft.atitle%253DMuch%2Bado%2Babout%2Bnothing%253A%2BA%2Bcomparison%2Bof%2Bmissing%2Bdata%2Bmethods%2Band%2Bsoftware%2Bto%2Bfit%2Bincomplete%2Bdata%2Bregression%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1198%252F000313007X172556%26rft_id%253Dinfo%253Apmid%252F17401454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/000313007X172556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17401454&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F845.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243781600013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.20" data-doi="10.1534/genetics.113.158014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghavan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mauleon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leung</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Efficient imputation of missing markers in low-coverage genotyping-by-sequencing data from multiparental crosses.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">401</span>–<span class="cit-lpage">404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.158014%26rft_id%253Dinfo%253Apmid%252F24583583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk3LzEvNDAxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzg0NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.21" data-doi="10.1101/gr.089516.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">High throughput genotyping by whole-genome resequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1068</span>–<span class="cit-lpage">1076</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.089516.108%26rft_id%253Dinfo%253Apmid%252F19420380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzYvMTA2OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC84NDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.22" data-doi="10.2135/cropsci2009.08.0434"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Marker genotype imputation in a low marker-density panel with a high-marker-density reference panel: accuracy evaluation in barley breeding lines.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1269</span>–<span class="cit-lpage">1278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DIwata%26rft.auinit1%253DH.%26rft.volume%253D50%26rft.issue%253D4%26rft.spage%253D1269%26rft.epage%253D1278%26rft.atitle%253DMarker%2BGenotype%2BImputation%2Bin%2Ba%2BLow-Marker-Density%2BPanel%2Bwith%2Ba%2BHigh-Marker-Density%2BReference%2BPanel%253A%2BAccuracy%2BEvaluation%2Bin%2BBarley%2BBreeding%2BLines%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.08.0434%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.08.0434&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000279138200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.845.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Little</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-source">Statistical analysis with missing data</span>, <span class="cit-publ-name">John Wiley Sons</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.24" data-doi="10.1038/nrg2796"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genotype imputation for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMarchini%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D511%26rft.atitle%253DGenotype%2Bimputation%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2796%26rft_id%253Dinfo%253Apmid%252F20517342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2796&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20517342&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F845.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278998500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.25" data-doi="10.3835/plantgenome2009.08.0022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maughan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yourstone</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jellen</span>  <span class="cit-name-given-names">E. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">SNP discovery via genomic reduction, barcoding, and 454-Pyrosequencing in Amaranth.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">260</span>–<span class="cit-lpage">270</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D2%26rft.spage%253D260%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2009.08.0022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2009.08.0022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.26" data-doi="10.1038/nrg2626"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Metzker</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Sequencing technologies – the next generation.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMetzker%26rft.auinit1%253DM.%2BL.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D31%26rft.epage%253D46%26rft.atitle%253DSequencing%2Btechnologies%2B-%2Bthe%2Bnext%2Bgeneration.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2626%26rft_id%253Dinfo%253Apmid%252F19997069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2626&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19997069&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F845.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272867700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.27" data-doi="10.1038/nrg2986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genotype and SNP calling from next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">443</span>–<span class="cit-lpage">445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DNielsen%26rft.auinit1%253DR.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D443%26rft.epage%253D451%26rft.atitle%253DGenotype%2Band%2BSNP%2Bcalling%2Bfrom%2Bnext-generation%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2986%26rft_id%253Dinfo%253Apmid%252F21587300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2986&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21587300&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F845.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.28" data-doi="10.1371/journal.pone.0037135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kay</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Double digest RADseq: An inexpensive method for de novo SNP discovery and genotyping in model and non-model species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e37135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPeterson%26rft.auinit1%253DB.%2BK.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De37135%26rft.epage%253De37135%26rft.atitle%253DDouble%2Bdigest%2BRADseq%253A%2Ban%2Binexpensive%2Bmethod%2Bfor%2Bde%2Bnovo%2BSNP%2Bdiscovery%2Band%2Bgenotyping%2Bin%2Bmodel%2Band%2Bnon-model%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0037135%26rft_id%253Dinfo%253Apmid%252F22675423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0037135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22675423&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F845.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.29" data-doi="10.3390/d6040665"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horbach</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genotyping-by-sequencing for plant genetic diversity analysis: A lab guide for SNP genotyping.</span> <abbr class="cit-jnl-abbrev">Diversity (Basel)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">665</span>–<span class="cit-lpage">680</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDiversity%2B%2528Basel%2529%26rft.volume%253D6%26rft.spage%253D665%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fd6040665%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/d6040665&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.30" data-doi="10.3835/plantgenome2012.05.0005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rife</span>  <span class="cit-name-given-names">T. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genotyping-by-sequencing for plant breeding and genetics.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">92</span>–<span class="cit-lpage">102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D5%26rft.spage%253D92%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.05.0005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.05.0005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000312663100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.31" data-doi="10.1371/journal.pone.0032253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e32253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPoland%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De32253%26rft.epage%253De32253%26rft.atitle%253DDevelopment%2Bof%2Bhigh-density%2Bgenetic%2Bmaps%2Bfor%2Bbarley%2Band%2Bwheat%2Busing%2Ba%2Bnovel%2Btwo-enzyme%2Bgenotyping-by-sequencing%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0032253%26rft_id%253Dinfo%253Apmid%252F22389690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0032253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22389690&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F845.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.32" data-doi="10.3835/plantgenome2012.06.0006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Genomic selection in wheat breeding using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D5%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.06.0006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.06.0006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.33" data-doi="10.1101/gr.079509.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pool</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hellmann</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Population genetic inference from genomic sequence variation.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.079509.108%26rft_id%253Dinfo%253Apmid%252F20067940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzMvMjkxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzg0NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.34" data-doi="10.1111/mec.12965"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Puritz</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matz</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toonen</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolnick</span>  <span class="cit-name-given-names">D. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Demystifying the RAD fad.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">5937</span>–<span class="cit-lpage">5942</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D5937%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12965&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000346771900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Imputation of unordered markers and the impact on genomic selection accuracy.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">427</span>–<span class="cit-lpage">439</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D3%26rft.spage%253D427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.845.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salikhov</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sacomoto</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kucherov</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Using cascading Bloom filters to improve the memory usage for de Brujin graphs, WABI</span>, pp. <span class="cit-fpage">364</span>–<span class="cit-lpage">376</span> in <span class="cit-source"><em>Algorithms in Bioinformatics</em>, <em>XIV</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Darling</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Stoye</span>  <span class="cit-name-given-names">J.</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Heidelberg, Germany</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.37" data-doi="10.1371/journal.pone.0095292"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schilling</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">P. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duffy</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rai</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rowe</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genotyping-by-sequencing for Populus population genomics: an assessment of genome sampling patterns and filtering approaches.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e95292</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De95292%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0095292%26rft_id%253Dinfo%253Apmid%252F24748384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0095292&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24748384&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F845.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.38" data-doi="10.1038/nrg3642"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sims</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sudbery</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ilott</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ponting</span>  <span class="cit-name-given-names">C. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Sequencing depth and coverage: key considerations in genomic analyses.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">121</span>–<span class="cit-lpage">132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D121%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3642%26rft_id%253Dinfo%253Apmid%252F24434847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3642&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24434847&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F845.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.39" data-doi="10.1073/pnas.1321152111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Šmarda</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bureš</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horová</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leitch</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mucina</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Ecological and evolutionary significance of genomic GC content diversity in monocots.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">e4096</span>–<span class="cit-lpage">e4102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.volume%253D111%26rft.spage%253De4096%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1321152111%26rft_id%253Dinfo%253Apmid%252F25225383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1073/pnas.1321152111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25225383&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F845.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.40" data-doi="10.1371/journal.pone.0054603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sonah</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bastien</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iquira</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tardivel</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Légaré</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">An improved genotyping by sequencing (GBS) approach offering increased versatility and efficiency of SNP discovery and genotyping.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e54603</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De54603%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0054603%26rft_id%253Dinfo%253Apmid%252F23372741%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0054603&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23372741&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F845.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.41" data-doi="10.1093/bioinformatics/btm069"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stacklies</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Redestig</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scholz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walther</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selbig</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">pcaMethods—a bioconductor package providing PCA methods for incomplete data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1164</span>–<span class="cit-lpage">1167</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm069%26rft_id%253Dinfo%253Apmid%252F17344241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyMy85LzExNjQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvODQ1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stekhoven</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bühlmann</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">MissForest - nonparametric missing value imputation for mixed-type data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">112</span>–<span class="cit-lpage">118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D28%26rft.spage%253D112%26rft_id%253Dinfo%253Apmid%252F22039212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22039212&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F845.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.43" data-doi="10.1093/bioinformatics/17.6.520"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Troyanskaya</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantor</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherlock</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Missing value estimation methods for DNA microarrays.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">520</span>–<span class="cit-lpage">525</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DTroyanskaya%26rft.auinit1%253DO.%26rft.volume%253D17%26rft.issue%253D6%26rft.spage%253D520%26rft.epage%253D525%26rft.atitle%253DMissing%2Bvalue%2Bestimation%2Bmethods%2Bfor%2BDNA%2Bmicroarrays%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F17.6.520%26rft_id%253Dinfo%253Apmid%252F11395428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxNy82LzUyMCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC84NDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.44" data-doi="10.1371/journal.pone.0001172"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Orsouw</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hogers</span>  <span class="cit-name-given-names">R. C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janssen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yalcin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snoeijers</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Complexity Reduction of Polymorphic Sequences (CRoPS) : a novel approach for large-scale polymorphism discovery in complex genomes.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e1172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253Dvan%2BOrsouw%26rft.auinit1%253DN.%2BJ.%26rft.volume%253D2%26rft.issue%253D11%26rft.spage%253De1172%26rft.epage%253De1172%26rft.atitle%253DComplexity%2Breduction%2Bof%2Bpolymorphic%2Bsequences%2B%2528CRoPS%2529%253A%2Ba%2Bnovel%2Bapproach%2Bfor%2Blarge-scale%2Bpolymorphism%2Bdiscovery%2Bin%2Bcomplex%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001172%26rft_id%253Dinfo%253Apmid%252F18000544%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0001172&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18000544&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F845.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.45" data-doi="10.1093/nar/23.21.4407"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vos</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hogers</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bleeker</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reijans</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van de Lee</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">AFLP: a new technique for DNA fingerprinting.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">4407</span>–<span class="cit-lpage">4414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DVos%26rft.auinit1%253DP.%26rft.volume%253D23%26rft.issue%253D21%26rft.spage%253D4407%26rft.epage%253D4414%26rft.atitle%253DAFLP%253A%2Ba%2Bnew%2Btechnique%2Bfor%2BDNA%2Bfingerprinting%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F23.21.4407%26rft_id%253Dinfo%253Apmid%252F7501463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyMy8yMS80NDA3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzg0NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.46" data-doi="10.3168/jds.2010-3149"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Accuracy of direct genomic values derived from imputed single nucleotide polymorphism genotypes in Jersey cattle.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">5423</span>–<span class="cit-lpage">5435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DWeigel%26rft.auinit1%253DK.%2BA.%26rft.volume%253D93%26rft.issue%253D11%26rft.spage%253D5423%26rft.epage%253D5435%26rft.atitle%253DAccuracy%2Bof%2Bdirect%2Bgenomic%2Bvalues%2Bderived%2Bfrom%2Bimputed%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bgenotypes%2Bin%2BJersey%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2010-3149%26rft_id%253Dinfo%253Apmid%252F20965358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2010-3149&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20965358&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F845.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.845.47" data-doi="10.1007/s11538-005-9021-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wendl</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Occupancy modeling of coverage distribution for whole genome shotgun DNA sequencing.</span> <abbr class="cit-jnl-abbrev">Bull. Math. Biol.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">179</span>–<span class="cit-lpage">196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBulletin%2Bof%2Bmathematical%2Bbiology%26rft.stitle%253DBull%2BMath%2BBiol%26rft.aulast%253DWendl%26rft.auinit1%253DM.%2BC.%26rft.volume%253D68%26rft.issue%253D1%26rft.spage%253D179%26rft.epage%253D196%26rft.atitle%253DOccupancy%2Bmodeling%2Bof%2Bcoverage%2Bdistribution%2Bfor%2Bwhole%2Bgenome%2Bshotgun%2BDNA%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11538-005-9021-4%26rft_id%253Dinfo%253Apmid%252F16794926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11538-005-9021-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16794926&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F845.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237266000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/4/845.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/4/835" title="Environmental Association Analyses Identify Candidates for Abiotic Stress Tolerance in Glycine soja, the Wild Progenitor of Cultivated Soybeans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/4/857" title="Haplotag: Software for Haplotype-Based Genotyping-by-Sequencing Analysis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/4"><span>Volume 6 Issue 4, April 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/4.cover-source.jpg?itok=Lj45nPaf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/845.supplemental" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-s3Dh3Jx8VbES6qSmoyVWuKI9BrGiGmlp-h2PWOYM2vU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/4/845" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/4/845" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/4/845" class="active" data-icon-position="" data-hide-link-title="0">Increasing Genome Sampling and Improving SNP Genotyping for Genotyping-by-Sequencing with New Combinations of Restriction Enzymes</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460114" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-r9ubjrI1j88XFHGI7RRp5JMCXDCeDcIf9bYI5-K8fhA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/4/845.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/4/845" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-SiotD63EsoyyfvFJBX0Ie8oArJnVHvAtxMdkQIzuwKQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/4/845" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/4/845.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460114" id="node460114--2" data-pisa="ggg;6/4/845" data-pisa-master="ggg;g3.115.025775" data-apath="/ggg/6/4/845.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Increasing Genome Sampling and Improving SNP Genotyping for Genotyping-by-Sequencing with New Combinations of Restriction Enzymes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-3106-1247" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yong-Bi Fu</span>, <span class="highwire-citation-author" data-delta="1">Gregory W. Peterson</span> and <span class="highwire-citation-author" data-delta="2">Yibo Dong</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">845-856;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.025775 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460114--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yong-Bi Fu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Gene Resources of Canada, Saskatoon Research and Development Centre, Agriculture and Agri-Food Canada, Saskatoon, Saskatchewan S7N 0X2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yong-Bi%2BFu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYong-Bi%2BFu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3106-1247" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yong-Bi Fu</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:yong-bi.fu@agr.gc.ca" class="" data-icon-position="" data-hide-link-title="0">yong-bi.fu@agr.gc.ca</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gregory W. Peterson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Gene Resources of Canada, Saskatoon Research and Development Centre, Agriculture and Agri-Food Canada, Saskatoon, Saskatchewan S7N 0X2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregory%2BW.%2BPeterson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peterson%20GW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGregory%2BW.%2BPeterson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yibo Dong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Gene Resources of Canada, Saskatoon Research and Development Centre, Agriculture and Agri-Food Canada, Saskatoon, Saskatchewan S7N 0X2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yibo%2BDong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dong%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYibo%2BDong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/4/845" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/4/845&amp;title=Increasing%20Genome%20Sampling%20and%20Improving%20SNP%20Genotyping%20for%20Genotyping-by-Sequencing%20with%20New%20Combinations%20of%20Restriction%20Enzymes" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/4/845&amp;title=Increasing%20Genome%20Sampling%20and%20Improving%20SNP%20Genotyping%20for%20Genotyping-by-Sequencing%20with%20New%20Combinations%20of%20Restriction%20Enzymes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/4/845&amp;title=Increasing%20Genome%20Sampling%20and%20Improving%20SNP%20Genotyping%20for%20Genotyping-by-Sequencing%20with%20New%20Combinations%20of%20Restriction%20Enzymes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/4/845&amp;text=Increasing%20Genome%20Sampling%20and%20Improving%20SNP%20Genotyping%20for%20Genotyping-by-Sequencing%20with%20New%20Combinations%20of%20Restriction%20Enzymes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/4/845&amp;title=Increasing%20Genome%20Sampling%20and%20Improving%20SNP%20Genotyping%20for%20Genotyping-by-Sequencing%20with%20New%20Combinations%20of%20Restriction%20Enzymes" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/4/845&amp;t=Increasing%20Genome%20Sampling%20and%20Improving%20SNP%20Genotyping%20for%20Genotyping-by-Sequencing%20with%20New%20Combinations%20of%20Restriction%20Enzymes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/4/845&amp;title=Increasing%20Genome%20Sampling%20and%20Improving%20SNP%20Genotyping%20for%20Genotyping-by-Sequencing%20with%20New%20Combinations%20of%20Restriction%20Enzymes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/4/845&amp;title=Increasing%20Genome%20Sampling%20and%20Improving%20SNP%20Genotyping%20for%20Genotyping-by-Sequencing%20with%20New%20Combinations%20of%20Restriction%20Enzymes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460114/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460114" id="node460114--3" data-pisa="ggg;6/4/845" data-pisa-master="ggg;g3.115.025775" data-apath="/ggg/6/4/845.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Increasing Genome Sampling and Improving SNP Genotyping for Genotyping-by-Sequencing with New Combinations of Restriction Enzymes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-3106-1247" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yong-Bi Fu</span>, <span class="highwire-citation-author" data-delta="1">Gregory W. Peterson</span> and <span class="highwire-citation-author" data-delta="2">Yibo Dong</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">845-856;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.025775 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460114--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yong-Bi Fu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Gene Resources of Canada, Saskatoon Research and Development Centre, Agriculture and Agri-Food Canada, Saskatoon, Saskatchewan S7N 0X2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yong-Bi%2BFu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYong-Bi%2BFu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3106-1247" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yong-Bi Fu</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:yong-bi.fu@agr.gc.ca" class="" data-icon-position="" data-hide-link-title="0">yong-bi.fu@agr.gc.ca</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gregory W. Peterson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Gene Resources of Canada, Saskatoon Research and Development Centre, Agriculture and Agri-Food Canada, Saskatoon, Saskatchewan S7N 0X2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregory%2BW.%2BPeterson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peterson%20GW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGregory%2BW.%2BPeterson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yibo Dong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Plant Gene Resources of Canada, Saskatoon Research and Development Centre, Agriculture and Agri-Food Canada, Saskatoon, Saskatchewan S7N 0X2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yibo%2BDong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dong%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYibo%2BDong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=845&amp;rft.epage=856&amp;rft.atitle=Increasing%20Genome%20Sampling%20and%20Improving%20SNP%20Genotyping%20for%20Genotyping-by-Sequencing%20with%20New%20Combinations%20of%20Restriction%20Enzymes&amp;rft.volume=6&amp;rft.issue=4&amp;rft.date=2016-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Fu%2C+Yong-Bi&amp;rft.au=Peterson%2C+Gregory+W.&amp;rft.au=Dong%2C+Yibo"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460114/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460114/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460114/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460114/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460114/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460114/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460114/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460114/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460114/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460114/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460114/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460114/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/4/845" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/4/845.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/4/845.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/4/845.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/4/845.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
