<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/4/1049" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/4/1049.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/4/1049.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/4/1049.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/4/1049/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/4/1049.atom" />
<meta name="HW.pisa" content="ggg;6/4/1049" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Accuracy of Genomic Prediction in Switchgrass (Panicum virgatum L.) Improved by Accounting for Linkage Disequilibrium" />
<meta name="DC.Identifier" content="10.1534/g3.115.024950" />
<meta name="DC.Date" content="2016-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Ramstein et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Switchgrass is a relatively high-yielding and environmentally sustainable biomass crop, but further genetic gains in biomass yield must be achieved to make it an economically viable bioenergy feedstock. Genomic selection (GS) is an attractive technology to generate rapid genetic gains in switchgrass, and meet the goals of a substantial displacement of petroleum use with biofuels in the near future. In this study, we empirically assessed prediction procedures for genomic selection in two different populations, consisting of 137 and 110 half-sib families of switchgrass, tested in two locations in the United States for three agronomic traits: dry matter yield, plant height, and heading date. Marker data were produced for the families’ parents by exome capture sequencing, generating up to 141,030 polymorphic markers with available genomic-location and annotation information. We evaluated prediction procedures that varied not only by learning schemes and prediction models, but also by the way the data were preprocessed to account for redundancy in marker information. More complex genomic prediction procedures were generally not significantly more accurate than the simplest procedure, likely due to limited population sizes. Nevertheless, a highly significant gain in prediction accuracy was achieved by transforming the marker data through a marker correlation matrix. Our results suggest that marker-data transformations and, more generally, the account of linkage disequilibrium among markers, offer valuable opportunities for improving prediction procedures in GS. Some of the achieved prediction accuracies should motivate implementation of GS in switchgrass breeding programs." />
<meta name="DC.Contributor" content="Guillaume P. Ramstein" />
<meta name="DC.Contributor" content="Joseph Evans" />
<meta name="DC.Contributor" content="Shawn M. Kaeppler" />
<meta name="DC.Contributor" content="Robert B. Mitchell" />
<meta name="DC.Contributor" content="Kenneth P. Vogel" />
<meta name="DC.Contributor" content="C. Robin Buell" />
<meta name="DC.Contributor" content="Michael D. Casler" />
<meta name="article:published_time" content="2016-04-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Accuracy of Genomic Prediction in Switchgrass (Panicum virgatum L.) Improved by Accounting for Linkage Disequilibrium" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Switchgrass is a relatively high-yielding and environmentally sustainable biomass crop, but further genetic gains in biomass yield must be achieved to make it an economically viable bioenergy feedstock. Genomic selection (GS) is an attractive technology to generate rapid genetic gains in switchgrass, and meet the goals of a substantial displacement of petroleum use with biofuels in the near future. In this study, we empirically assessed prediction procedures for genomic selection in two different populations, consisting of 137 and 110 half-sib families of switchgrass, tested in two locations in the United States for three agronomic traits: dry matter yield, plant height, and heading date. Marker data were produced for the families’ parents by exome capture sequencing, generating up to 141,030 polymorphic markers with available genomic-location and annotation information. We evaluated prediction procedures that varied not only by learning schemes and prediction models, but also by the way the data were preprocessed to account for redundancy in marker information. More complex genomic prediction procedures were generally not significantly more accurate than the simplest procedure, likely due to limited population sizes. Nevertheless, a highly significant gain in prediction accuracy was achieved by transforming the marker data through a marker correlation matrix. Our results suggest that marker-data transformations and, more generally, the account of linkage disequilibrium among markers, offer valuable opportunities for improving prediction procedures in GS. Some of the achieved prediction accuracies should motivate implementation of GS in switchgrass breeding programs.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/04/01" />
<meta name="citation_mjid" content="ggg;6/4/1049" />
<meta name="citation_id" content="6/4/1049" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/4/1049" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/4/1049.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/4/1049.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/4/1049.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.024950" />
<meta name="citation_pmid" content="26869619" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="1049" />
<meta name="citation_lastpage" content="1062" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Guillaume P. Ramstein" />
<meta name="citation_author_institution" content="Department of Agronomy, University of Wisconsin-Madison, WI 53706" />
<meta name="citation_author_email" content="ramstein@wisc.edu" />
<meta name="citation_author" content="Joseph Evans" />
<meta name="citation_author_institution" content="Department of Energy Great Lakes Bioenergy Research Center, Michigan State University, East Lansing, MI 48824" />
<meta name="citation_author_institution" content="Department of Plant Biology, Michigan State University, East Lansing, MI 48824" />
<meta name="citation_author" content="Shawn M. Kaeppler" />
<meta name="citation_author_institution" content="Department of Agronomy, University of Wisconsin-Madison, WI 53706" />
<meta name="citation_author_institution" content="Department of Energy Great Lakes Bioenergy Research Center, University of Wisconsin-Madison, WI 53706" />
<meta name="citation_author" content="Robert B. Mitchell" />
<meta name="citation_author_institution" content="Grain, Forage, and Bioenergy Research Unit, Agricultural Research Service, United States Department of Agriculture, University of Nebraska, Lincoln, NE 68583-0937" />
<meta name="citation_author" content="Kenneth P. Vogel" />
<meta name="citation_author_institution" content="Grain, Forage, and Bioenergy Research Unit, Agricultural Research Service, United States Department of Agriculture, University of Nebraska, Lincoln, NE 68583-0937" />
<meta name="citation_author" content="C. Robin Buell" />
<meta name="citation_author_institution" content="Department of Energy Great Lakes Bioenergy Research Center, Michigan State University, East Lansing, MI 48824" />
<meta name="citation_author_institution" content="Department of Plant Biology, Michigan State University, East Lansing, MI 48824" />
<meta name="citation_author" content="Michael D. Casler" />
<meta name="citation_author_institution" content="Department of Agronomy, University of Wisconsin-Madison, WI 53706" />
<meta name="citation_author_institution" content="Agricultural Research Service, United States Department of Agriculture, Madison, WI 53706" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Akdemir;citation_author=J.-L. Jannink;citation_title=Locally epistatic genomic relationship matrices for genomic association and prediction.;citation_volume=199;citation_year=2015;citation_issue=3;citation_pmid=25614606;citation_doi=10.1534/genetics.114.173658" />
<meta name="citation_reference" content="citation_journal_title=Machine learning;citation_author=L. Breiman;citation_title=Random forests.;citation_pages=5-32;citation_volume=45;citation_year=2001;citation_issue=1;citation_doi=10.1023/A:1010933404324" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Burgueño;citation_author=G. de los Campos;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Genomic prediction of breeding values when modeling genotype× environment interaction using pedigree and dense molecular markers.;citation_pages=707-719;citation_volume=52;citation_year=2012;citation_issue=2;citation_doi=10.2135/cropsci2011.06.0299" />
<meta name="citation_reference" content="citation_title=ASReml-R reference manual;citation_author=D. Butler;citation_author=B. R. Cullis;citation_author=A. Gilmour;citation_author=B. Gogel;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=M. P. Calus;citation_author=R. F. Veerkamp;citation_title=Different genomic relationship matrices for single-step analysis using phenotypic, pedigree and genomic information.;citation_pages=1-1;citation_volume=43;citation_year=2011;citation_issue=1;citation_pmid=21208445;citation_doi=10.1186/1297-9686-43-1" />
<meta name="citation_reference" content="citation_journal_title=BioEnergy Res.;citation_author=M. D. Casler;citation_title=Changes in mean and genetic variance during two cycles of within-family selection in switchgrass.;citation_pages=47-54;citation_volume=3;citation_year=2010;citation_issue=1;citation_doi=10.1007/s12155-009-9071-9" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=M. D. Casler;citation_author=E. C. Brummer;citation_title=Theoretical Expected Genetic Gains for Among-and-Within-Family Selection Methods in Perennial Forage Crops;citation_pages=890-902;citation_volume=48;citation_year=2008;citation_issue=3;citation_doi=10.2135/cropsci2007.09.0499" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=M. D. Casler;citation_author=K. P. Vogel;citation_title=Selection for biomass yield in upland, lowland, and hybrid switchgrass.;citation_pages=626;citation_volume=54;citation_year=2014;citation_issue=2;citation_doi=10.2135/cropsci2013.04.0239" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=M. D. Casler;citation_author=K. P. Vogel;citation_author=A. C. Beal;citation_title=Registration of WS4U and WS8U Switchgrass Germplasms;citation_pages=998-999;citation_volume=46;citation_year=2006;citation_issue=2;citation_doi=10.2135/cropsci2005.04-0020" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=G. de los Campos;citation_author=P. Perez;citation_author=D. Gianola;citation_author=J. Burgueno;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.;citation_volume=186;citation_year=2010;citation_issue=2;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=G. de los Campos;citation_author=A. I. Vazquez;citation_author=R. Fernando;citation_author=Y. C. Klimentidis;citation_author=D. Sorensen;citation_title=Prediction of complex human traits using the genomic best linear unbiased predictor.;citation_pages=E1003608;citation_volume=9;citation_year=2013;citation_issue=7;citation_pmid=23874214;citation_doi=10.1371/journal.pgen.1003608" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=C. W. Dunnett;citation_title=New tables for multiple comparisons with a control.;citation_pages=482-491;citation_volume=20;citation_year=1964;citation_issue=3;citation_doi=10.2307/2528490" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. B. Endelman;citation_title=Ridge regression and other kernels for genomic selection with R package rrBLUP.;citation_pages=250-255;citation_volume=4;citation_year=2011;citation_issue=3;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=M. Erbe;citation_author=B. Hayes;citation_author=L. Matukumalli;citation_author=S. Goswami;citation_author=P. Bowman;citation_title=Improving accuracy of genomic predictions within and between dairy cattle breeds with imputed high-density single nucleotide polymorphism panels.;citation_pages=4114-4129;citation_volume=95;citation_year=2012;citation_issue=7;citation_pmid=22720968;citation_doi=10.3168/jds.2011-5019" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=J. Evans;citation_author=J. Kim;citation_author=K. L. Childs;citation_author=B. Vaillancourt;citation_author=E. Crisovan;citation_title=Nucleotide polymorphism and copy number variant detection using exome capture and next‐generation sequencing in the polyploid grass Panicum virgatum.;citation_pages=993-1008;citation_volume=79;citation_year=2014;citation_issue=6;citation_pmid=24947485;citation_doi=10.1111/tpj.12601" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=J. Evans;citation_author=E. Crisovan;citation_author=K. Barry;citation_author=C. Daum;citation_author=J. Jenkins;citation_title=Diversity and population structure of northern switchgrass as revealed through exome capture sequencing.;citation_pages=800-815;citation_volume=84;citation_year=2015;citation_issue=4;citation_pmid=26426343;citation_doi=10.1111/tpj.13041" />
<meta name="citation_reference" content="citation_journal_title=Biostatistics;citation_author=J. Friedman;citation_author=T. Hastie;citation_author=R. Tibshirani;citation_title=Sparse inverse covariance estimation with the graphical lasso.;citation_volume=9;citation_year=2008;citation_issue=3;citation_pmid=18079126;citation_doi=10.1093/biostatistics/kxm045" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=D. J. Garrick;citation_author=J. F. Taylor;citation_author=R. L. Fernando;citation_title=Deregressing estimated breeding values and weighting information for genomic regression analyses.;citation_pages=55-55;citation_volume=41;citation_year=2009;citation_pmid=20043827;citation_doi=10.1186/1297-9686-41-55" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Gianola;citation_author=J. B. van Kaam;citation_title=Reproducing Kernel Hilbert Spaces Regression Methods for Genomic Assisted Prediction of Quantitative Traits;citation_pages=2289-2303;citation_volume=178;citation_year=2008;citation_issue=4;citation_pmid=18430950;citation_doi=10.1534/genetics.107.084285" />
<meta name="citation_reference" content="citation_journal_title=Technometrics;citation_author=G. H. Golub;citation_author=M. Heath;citation_author=G. Wahba;citation_title=Generalized cross-validation as a method for choosing a good ridge parameter.;citation_pages=215-223;citation_volume=21;citation_year=1979;citation_issue=2;citation_doi=10.2307/1268518" />
<meta name="citation_reference" content="citation_journal_title=Annals of Statistics;citation_title=Optimal smoothing in nonparametric mixed-effect models.;citation_author=C. Gu;citation_author=P. Ma;citation_pages=1357-1379;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=G. Guo;citation_author=M. S. Lund;citation_author=Y. Zhang;citation_author=G. Su;citation_title=Comparison between genomic predictions using daughter yield deviation and conventional estimated breeding value as response variables.;citation_pages=423-432;citation_volume=127;citation_year=2010;citation_issue=6;citation_pmid=21077966;citation_doi=10.1111/j.1439-0388.2010.00878.x" />
<meta name="citation_reference" content="citation_author=Z. Guo;citation_author=D.M. Tucker;citation_author=D. Wang;citation_author=C.J. Basten;citation_author=E. Ersoz;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. Dekkers;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=B. Harris;citation_author=D. Johnson;citation_title=Genomic predictions for New Zealand dairy bulls and integration with national genetic evaluation.;citation_pages=1243-1252;citation_volume=93;citation_year=2010;citation_issue=3;citation_pmid=20172244;citation_doi=10.3168/jds.2009-2619" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. J. Hayes;citation_author=P. M. Visscher;citation_author=M. E. Goddard;citation_title=Increased accuracy of artificial selection by using the realized relationship matrix.;citation_pages=47-60;citation_volume=91;citation_year=2009;citation_issue=01;citation_pmid=19220931;citation_doi=10.1017/S0016672308009981" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=D. He;citation_author=Z. Wang;citation_author=L. Parida;citation_title=Data-driven encoding for quantitative genetic trait prediction.;citation_pages=S10;citation_volume=16;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=H.-P. Yang;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic selection in plant breeding: a comparison of models.;citation_pages=146;citation_volume=52;citation_year=2012;citation_issue=1;citation_doi=10.2135/cropsci2011.06.0297" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=J.-L. Jannink;citation_author=M. E. Sorrells;citation_title=Using genomic prediction to characterize environments and optimize prediction accuracy in applied breeding data.;citation_pages=921;citation_volume=53;citation_year=2013;citation_issue=3;citation_doi=10.2135/cropsci2012.07.0420" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=N. Heslot;citation_author=D. Akdemir;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Integrating environmental covariates and crop modeling into the genomic selection framework to predict genotype by environment interactions.;citation_pages=463-480;citation_volume=127;citation_year=2014;citation_issue=2;citation_pmid=24264761;citation_doi=10.1007/s00122-013-2231-5" />
<meta name="citation_reference" content="citation_journal_title=IMA Journal of Numerical Analysis;citation_journal_abbrev=IMA J Numer Anal;citation_author=N. J. Higham;citation_title=Computing the nearest correlation matrix--a problem from finance;citation_pages=329-343;citation_volume=22;citation_year=2002;citation_issue=3;citation_doi=10.1093/imanum/22.3.329" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. Isidro;citation_author=J.-L. Jannink;citation_author=D. Akdemir;citation_author=J. Poland;citation_author=N. Heslot;citation_title=Training set optimization under population structure in genomic selection.;citation_pages=145-158;citation_volume=128;citation_year=2015;citation_issue=1;citation_pmid=25367380;citation_doi=10.1007/s00122-014-2418-4" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Jia;citation_author=J.-L. Jannink;citation_title=Multiple-trait genomic selection methods increase genetic value prediction accuracy.;citation_volume=192;citation_year=2012;citation_issue=4;citation_pmid=23086217;citation_doi=10.1534/genetics.112.144246" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. M. Kang;citation_author=J. H. Sul;citation_author=S. K. Service;citation_author=N. A. Zaitlen;citation_author=S.-y. Kong;citation_title=Variance component model to account for sample structure in genome-wide association studies.;citation_pages=348-354;citation_volume=42;citation_year=2010;citation_issue=4;citation_pmid=20208533;citation_doi=10.1038/ng.548" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=S. Karoui;citation_author=M. J. Carabaño;citation_author=C. Díaz;citation_author=A. Legarra;citation_title=Joint genomic evaluation of French dairy cattle breeds using multiple-trait models.;citation_pages=39;citation_volume=44;citation_year=2012;citation_pmid=23216664;citation_doi=10.1186/1297-9686-44-39" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Langmead;citation_author=C. Trapnell;citation_author=M. Pop;citation_author=S. L. Salzberg;citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.;citation_pages=R25-R25;citation_volume=10;citation_year=2009;citation_issue=3;citation_pmid=19261174;citation_doi=10.1186/gb-2009-10-3-r25" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_issue=16;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=G3:Genes Genomes Genetics;citation_author=G. Li;citation_author=D. D. Serba;citation_author=M. C. Saha;citation_author=J. H. Bouton;citation_author=C. L. Lanzatella;citation_title=Genetic linkage mapping and transmission ratio distortion in a three-generation four-founder population of Panicum virgatum (L.).;citation_pages=913-923;citation_volume=4;citation_year=2014;citation_issue=5" />
<meta name="citation_reference" content="citation_author=A. Liaw;citation_author=M. Wiener;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=N. Long;citation_author=D. Gianola;citation_author=G. Rosa;citation_author=K. Weigel;citation_title=Dimension reduction and variable selection for genomic selection: application to predicting milk yield in Holsteins.;citation_pages=247-257;citation_volume=128;citation_year=2011;citation_issue=4;citation_pmid=21749471;citation_doi=10.1111/j.1439-0388.2011.00917.x" />
<meta name="citation_reference" content="citation_journal_title=Adv. Agron.;citation_author=A. J. Lorenz;citation_author=S. Chao;citation_author=F. G. Asoro;citation_author=E. L. Heffner;citation_author=T. Hayashi;citation_title=Genomic selection in plant breeding: knowledge and prospects.;citation_pages=77;citation_volume=110;citation_year=2011;citation_doi=10.1016/B978-0-12-385531-2.00002-5" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=M. L. Makgahlela;citation_author=E. A. Mäntysaari;citation_author=I. Strandén;citation_author=M. Koivula;citation_author=U. Nielsen;citation_title=Across breed multi‐trait random regression genomic predictions in the Nordic Red dairy cattle.;citation_pages=10-19;citation_volume=130;citation_year=2013;citation_issue=1;citation_pmid=23317061;citation_doi=10.1111/j.1439-0388.2012.01017.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=E. R. Martin;citation_author=D. Kinnamon;citation_author=M. A. Schmidt;citation_author=E. Powell;citation_author=S. Zuchner;citation_title=SeqEM: an adaptive genotype-calling approach for next-generation sequencing studies.;citation_volume=26;citation_year=2010;citation_issue=22;citation_pmid=20861027;citation_doi=10.1093/bioinformatics/btq526" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Meuwissen;citation_author=B. Hayes;citation_author=M. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=B. G. Milligan;citation_title=Maximum-Likelihood Estimation of Relatedness;citation_pages=1153-1167;citation_volume=163;citation_year=2003;citation_issue=3;citation_pmid=12663552" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=G. Moser;citation_author=B. Tier;citation_author=R. E. Crump;citation_author=M. S. Khatkar;citation_author=H. W. Raadsma;citation_title=A comparison of five methods to predict genomic breeding values of dairy bulls from genome-wide SNP markers.;citation_pages=56-56;citation_volume=41;citation_year=2009;citation_pmid=20043835;citation_doi=10.1186/1297-9686-41-56" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=W. Muir;citation_title=Comparison of genomic and traditional BLUP-estimated breeding value accuracy and selection response under alternative trait and genomic parameters.;citation_pages=342-355;citation_volume=124;citation_year=2007;citation_issue=6;citation_pmid=18076471;citation_doi=10.1111/j.1439-0388.2007.00700.x" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_title=Genomic best linear unbiased prediction method reflecting the degree of linkage disequilibrium.;citation_author=M. Nishio;citation_author=M. Satoh;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=T. Park;citation_author=G. Casella;citation_title=The Bayesian lasso.;citation_pages=681-686;citation_volume=103;citation_year=2008;citation_issue=482;citation_doi=10.1198/016214508000000337" />
<meta name="citation_reference" content="citation_author=P. Pérez;citation_author=G. de los Campos;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome J.;citation_author=J. Poland;citation_author=J. Endelman;citation_author=J. Dawson;citation_author=J. Rutkoski;citation_author=S. Wu;citation_title=Genomic selection in wheat breeding using genotyping-by-sequencing.;citation_pages=103;citation_volume=5;citation_year=2012;citation_issue=3;citation_doi=10.3835/plantgenome2012.06.0006" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=R. M. S. Resende;citation_author=M. D. Casler;citation_author=M. D. Vilela de Resende;citation_title=Genomic selection in forage breeding: accuracy and methods.;citation_pages=143-156;citation_volume=54;citation_year=2014;citation_issue=1;citation_doi=10.2135/cropsci2013.05.0353" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=C. Riedelsheimer;citation_author=A. E. Melchinger;citation_title=Optimizing the allocation of resources for genomic selection in one breeding cycle.;citation_pages=2835-2848;citation_volume=126;citation_year=2013;citation_issue=11;citation_pmid=23982591;citation_doi=10.1007/s00122-013-2175-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Rincent;citation_author=D. Laloë;citation_author=S. Nicolas;citation_author=T. Altmann;citation_author=D. Brunel;citation_title=Maximizing the reliability of genomic selection by optimizing the calibration set of reference individuals: comparison of methods in two diverse groups of maize inbreds (Zea mays L.).;citation_volume=192;citation_year=2012;citation_issue=2;citation_pmid=22865733;citation_doi=10.1534/genetics.112.141473" />
<meta name="citation_reference" content="citation_journal_title=Bioresour. Technol.;citation_author=M. Sanderson;citation_author=R. Reed;citation_author=S. McLaughlin;citation_author=S. Wullschleger;citation_author=B. Conger;citation_title=Switchgrass as a sustainable bioenergy crop.;citation_pages=83-93;citation_volume=56;citation_year=1996;citation_issue=1;citation_doi=10.1016/0960-8524(95)00176-X" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=X. Shen;citation_author=M. Alam;citation_author=F. Fikse;citation_author=L. Rönnegård;citation_title=A novel generalized ridge regression method for quantitative genetics.;citation_volume=193;citation_year=2013;citation_issue=4;citation_pmid=23335338;citation_doi=10.1534/genetics.112.146720" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=D. Speed;citation_author=G. Hemani;citation_author=M. R. Johnson;citation_author=D. J. Balding;citation_title=Improved heritability estimation from genome-wide SNPs.;citation_pages=1011-1021;citation_volume=91;citation_year=2012;citation_issue=6;citation_pmid=23217325;citation_doi=10.1016/j.ajhg.2012.10.010" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. D. Storey;citation_author=R. Tibshirani;citation_title=Statistical significance for genomewide studies.;citation_volume=100;citation_year=2003;citation_issue=16;citation_pmid=12883005;citation_doi=10.1073/pnas.1530509100" />
<meta name="citation_reference" content="citation_journal_title=J. Dairy Sci.;citation_author=G. Su;citation_author=O. F. Christensen;citation_author=L. Janss;citation_author=M. S. Lund;citation_title=Comparison of genomic predictions using genomic relationship matrices built with different weighting factors to account for locus-specific variances.;citation_pages=6547-6559;citation_volume=97;citation_year=2014;citation_issue=10;citation_pmid=25129495;citation_doi=10.3168/jds.2014-8210" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=G. Sun;citation_author=C. Zhu;citation_author=M. H. Kramer;citation_author=S. S. Yang;citation_author=W. Song;citation_title=Variation explained in mixed-model association mapping.;citation_pages=333-340;citation_volume=105;citation_year=2010;citation_issue=4;citation_pmid=20145669;citation_doi=10.1038/hdy.2010.11" />
<meta name="citation_reference" content="citation_journal_title=J. Plant Regist.;citation_author=K. Vogel;citation_author=R. Mitchell;citation_author=M. Casler;citation_author=G. Sarath;citation_title=Registration of ‘Liberty’switchgrass.;citation_pages=242-247;citation_volume=8;citation_year=2014;citation_issue=3;citation_doi=10.3198/jpr2013.12.0076crc" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=B. S. Weir;citation_title=Inferences about linkage disequilibrium.;citation_pages=235-254;citation_volume=35;citation_year=1979;citation_issue=1;citation_pmid=497335;citation_doi=10.2307/2529947" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yang;citation_author=T. A. Manolio;citation_author=L. R. Pasquale;citation_author=E. Boerwinkle;citation_author=N. Caporaso;citation_title=Genome partitioning of genetic variation for complex traits using common SNPs.;citation_pages=519-525;citation_volume=43;citation_year=2011;citation_issue=6;citation_pmid=21552263;citation_doi=10.1038/ng.823" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=C.-H. Zhang;citation_title=Nearly unbiased variable selection under minimax concave penalty.;citation_pages=894-942;citation_volume=38;citation_year=2010;citation_doi=10.1214/09-AOS729" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=Y. Zhang;citation_author=J. E. Zalapa;citation_author=A. R. Jakubowski;citation_author=D. L. Price;citation_author=A. Acharya;citation_title=Post-glacial evolution of Panicum virgatum: centers of diversity and gene pools revealed by SSR markers and cpDNA sequences.;citation_pages=933-948;citation_volume=139;citation_year=2011;citation_issue=7;citation_pmid=21786028;citation_doi=10.1007/s10709-011-9597-6" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=X. Zheng;citation_author=D. Levine;citation_author=J. Shen;citation_author=S. M. Gogarten;citation_author=C. Laurie;citation_title=A high-performance computing toolset for relatedness and principal component analysis of SNP data.;citation_volume=28;citation_year=2012;citation_issue=24;citation_pmid=23060615;citation_doi=10.1093/bioinformatics/bts606" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Accuracy of Genomic Prediction in Switchgrass (Panicum virgatum L.) Improved by Accounting for Linkage Disequilibrium" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
<meta name="twitter:description" content="Switchgrass is a relatively high-yielding and environmentally sustainable biomass crop, but further genetic gains in biomass yield must be achieved to make it an economically viable bioenergy feedstock. Genomic selection (GS) is an attractive technology to generate rapid genetic gains in switchgrass, and meet the goals of a substantial displacement of petroleum use with biofuels in the near future. In this study, we empirically assessed prediction procedures for genomic selection in two different populations, consisting of 137 and 110 half-sib families of switchgrass, tested in two locations in the United States for three agronomic traits: dry matter yield, plant height, and heading date. Marker data were produced for the families’ parents by exome capture sequencing, generating up to 141,030 polymorphic markers with available genomic-location and annotation information. We evaluated prediction procedures that varied not only by learning schemes and prediction models, but also by the way the data were preprocessed to account for redundancy in marker information. More complex genomic prediction procedures were generally not significantly more accurate than the simplest procedure, likely due to limited population sizes. Nevertheless, a highly significant gain in prediction accuracy was achieved by transforming the marker data through a marker correlation matrix. Our results suggest that marker-data transformations and, more generally, the account of linkage disequilibrium among markers, offer valuable opportunities for improving prediction procedures in GS. Some of the achieved prediction accuracies should motivate implementation of GS in switchgrass breeding programs." />
<meta name="og-title" property="og:title" content="Accuracy of Genomic Prediction in Switchgrass (Panicum virgatum L.) Improved by Accounting for Linkage Disequilibrium" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/4/1049" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Switchgrass is a relatively high-yielding and environmentally sustainable biomass crop, but further genetic gains in biomass yield must be achieved to make it an economically viable bioenergy feedstock. Genomic selection (GS) is an attractive technology to generate rapid genetic gains in switchgrass, and meet the goals of a substantial displacement of petroleum use with biofuels in the near future. In this study, we empirically assessed prediction procedures for genomic selection in two different populations, consisting of 137 and 110 half-sib families of switchgrass, tested in two locations in the United States for three agronomic traits: dry matter yield, plant height, and heading date. Marker data were produced for the families’ parents by exome capture sequencing, generating up to 141,030 polymorphic markers with available genomic-location and annotation information. We evaluated prediction procedures that varied not only by learning schemes and prediction models, but also by the way the data were preprocessed to account for redundancy in marker information. More complex genomic prediction procedures were generally not significantly more accurate than the simplest procedure, likely due to limited population sizes. Nevertheless, a highly significant gain in prediction accuracy was achieved by transforming the marker data through a marker correlation matrix. Our results suggest that marker-data transformations and, more generally, the account of linkage disequilibrium among markers, offer valuable opportunities for improving prediction procedures in GS. Some of the achieved prediction accuracies should motivate implementation of GS in switchgrass breeding programs." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
    <title>Accuracy of Genomic Prediction in Switchgrass (Panicum virgatum L.) Improved by Accounting for Linkage Disequilibrium | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Qql826g1Sxy0wVJKQvPv2bfdsZOCoyzByW0OorkvMmo","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460127","apath":"\/ggg\/6\/4\/1049.atom","pisa":"ggg;6\/4\/1049","ac":{"\/ggg\/6\/4\/1049.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/4\/1049.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/4\/1049"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/4\/1049"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/4\/1049.full":true,"\/content\/6\/4\/1049":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022s28EJykAhZhqMoqfdhz0j6NwVKNKrKVnY\\\\\\\/Oo1uM89R7na76SKfdd50JGnT+2Q0KTcznl49HaIjidOOIV+IkM1rcTyLwo9NRwI31GNU+Ru0uhi8k6yBI7vqqjyxlq613M8YO\\\\\\\/HTRVsi8m9IIj08X1hXAhvg65lvxSExmo9Wyeu9kkpSVIxTs9ANYWCZ9PA1Vmg7LzV0\\\\\\\/Qx+tZToDfmgaJHBl6B2norZwjdorXNpOCCak97fziGStdSHBTPJ3i15d0sszk1PZPrvjlLVrZkh1APw1MgX5eopJHfbHjLGjN1vHWUSYBGOHCxnqKDd6J6y01sNlI\\\\\\\/fW6zIUy1hAN5lFaVy0HY2fp\\\\\\\/w4TZg2Nevnbl1Yu+2BGj7FlUHE8M702vP5P5QnkudkYwJON9\\\\\\\/mBsrTeKX16OkzYJy6IOD70EnVdBLDRAQ7uT515HKHCx+7HjLIQjkaHPusul6SPUpbXUYCKbkBT1A4R8IurTjPnaEvKGy9Xo3bHPE2R0jXFIq8RjMnSqlqeJs0TvlpkRfBZ2EPkjr+CvWkINmDmnnmMUYnhzu1W4+m+GcR0dV\\\\\\\/vuABzTcLN7CccYk8iMa0tu\\\\\\\/b6t++weX+VF7eJgQKSoGQ2N0RqUkaEt86rS9P2SMPYnWmfpcIgFZ8bd17SDzJ5YKeuqsYFphdCpQasr5D5TL2tDaNQKktE2reQU1goK2BojSe5McLFFTinXZEjul5trp+8BmQP7FR7hqrHOf95RozW6+iQmsSnpFLHnrpMFMnN7OwBvPej3x0ur9+GgT0FrNVg\\\\\\\/wB2vkJroYWv\\\\\\\/YapbSW19opDsOzwWJtzz9y+RgQ9nZ7PWFYP2Oe\\\\\\\/hnDs8e3IJsv+GKB73VJb+MA4W0H4zGQFYMvqV6fBoKj1ocNsHXjiU\\\\\\\/K7HAAorm3t9qXPkJc1EC9xvjEvY+gdUeVhabZq2SLBXD0oGIoFe0Vi5DVYEMcrJEfp7fjrEbnS8v6Je0wV0trqC4yomDWx+KsLE63XVmmz7rt3UeSjHzgHbLACdsEV6Vcq3t\\\\\\\/TEgpol+imML5nuNjfSAWSskydOaspEPRgbe3WJ2AEYVcgW4WqYlESZdWbsmISETzwb16D+w+1yECPk0v+0B6ZUXyOiinE8ym+kSfU1YASVF97YX7tebHmm2l5NPH0cFmcVE7UCfTCbpReavWfR5FhmWuKZx3qyG0HdGOla\\\\\\\/2DFyQtJQGjtzwdgbvfL5uaf7L9n+yAl4S\\\\\\\/w9oPPPp6GK7RshNbyYP92Y6CM+yCEWn6Ilfj5GN\\\\\\\/cPtaJvbtGWK8IacW\\\u0022,\\\u0022iv\\\u0022:\\\u0022pQGkYQAxWbZ8CLrU5U7lHg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b80e0113f9421aaabbf3b8bc77de1837d3d75992d8d36f721cd88ae3c87ddbbf\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022HUTm\\\\\\\/bzBXOeUPQO7TRt2tZ+uuZqXP1+H3HX\\\\\\\/nHHEwNQhM2j3hx+6+K+7XBvmi\\\\\\\/S675h9d\\\\\\\/NHRhL2LaF037HmDl19471v33jgeoYMRIIOl0gyxNVCxfGuSUaE87ly237gxRk6U358o5bpHOeh1Uqw9qOrequR73tSZ\\\\\\\/ya0aS8CgTvRtHuZW1n17KrfaxtC1eFgezjZDusviNpMFfe\\\\\\\/dEyvZAsO+yhZtsUmOK9DIsHl1kZIMS4CuosP+6xouKTDB9j\\\\\\\/aJTC40fUQeLYPBOy+Fe4fCKWZRKcIxgc5bFv8od+H9yejsIqi1+9Tm6ZvcTPu5d1tP+Vdvl7h3H\\\\\\\/n7bbC7OJSmyOzf\\\\\\\/wemgb27+5zn4O8ExhY6szB3PD5gnED7hniPT3YCodZBOEIkCQNXDsVAK366UYYCjWbyvHzAH1X7+YTJJuNkaW\\\\\\\/KhevzibOdsh8b90e9I69jLbdBTBVGLxTuprDxPxuwqyAr0VgHeR\\\\\\\/L\\\\\\\/OeSF8UqV6POMnyItipLEdUURLlte5VU6JzRs6L\\\\\\\/1jHuS3GeYRT1ClaUxz+TB9yFM1cxmYC95vLummRt48yyhxWEEvJONYIEz6FvQykLg7+IE\\\\\\\/igLjjzth1srwHV3095gjFHRKKUOCG0jXaCluimWnd\\\\\\\/L1DOlfp0Zcidr+q8v5ITA4vKFwsl2rPrF0+EJ7zdQuUEQ+215LxmJadEMXOjdP2aMwU2+pd287vZfcnajfkQ5s4ZwREx3dEOEqOSHlDYM5FDhfMegqypb1IqHDi\\\\\\\/Vac7n4syJiV3dYxVqNl47PxAUn+YzI+baJW\\\\\\\/HXw+7vOmixF2WoTFd13fvqEUysYPi+tdZlKfw2WCO6kwKqtL4GE8oZiBrQNi4QiT0DMFsAfQsgnmBYwsh3qTboU6xKvK81ry3KcXfaXiHFdfJssb4pYkwIVhTu9+mDwXJ2vMU2wmjmLOVzXXFXf0B4CgUpD6OGcuw2oN\\\\\\\/I\\\\\\\/ke0vi5rVdlsHZyyJzJ2\\\\\\\/ahTJtEgh8diwzfTcJ1Zw\\\\\\\/WpQbAziwAe8EVVfsbveC2LF54dwqus\\\\\\\/Nmxe4hr8Wbm4d5Vc6jUCdPkBJVakcL8c9\\\\\\\/616USct8QVCCyzn\\\\\\\/kKTC49XhEQvhIkd2Dea8FoIOl7E9JdH6vK\\\\\\\/4wQywdPvg5NpzHqJBjOk06Cl0p73kWEhfV\\\\\\\/usMkkd6I+CX0+lJLmeilw2Pj+QMmbx\\\\\\\/jzy8hBX+ZegyOqlH28E0ZLVJ6SD8XEwkXrGsSzrVeb4d5nxq\\\\\\\/dckFLo0u5fRUf0YmfvNQ5L1HhimQoeiWFtM0Sb6MzbdWMknyyPKtrRZMtNPNVtsPq2iwQjRw+zHSpMa4q4dWN9o3dEBrilfdDTAS4wPfJzRhU16v5DHaE1fqBxekxw3hkI33dH2e6DPxqV+eCnHJZ3jNVwWyou7EZKSSMoqyVVpAHIuPbybmvzH2Da1By5nnsU3o+fUfiZ6YlSTP+Pvdx4WIqpfTA59ZYTwIuu51lt7o6SVkK05xYokjrCaVbvMukSn+OmlsiDjVQPWb5k8\\\\\\\/so3wBFU0HvQLqyhQvBbLwX0T91i9MOYhdqOwcp1ZyHYfgJ57l73Z40kJ8LVZkY5h+B+cy8U7WKQGYj0JhWkYfoDs0HiKOFqwlTbgrXNE\\\\\\\/1L8kXXAInQlkQe+Lt1VS323JZHCdg2JeHp\\\\\\\/ZkfBNjJAmy7LH9eeZxQvdlPstXBDga\\\\\\\/kRrASa+IL4tggSln1kLydZw\\\\\\\/TeBeVTZznWaDVm1pGA8G7O8d2+0Fn7Cl6PgKg7YU2Ep+JMfACuflCLOHIzFzigUnRE\\\\\\\/dKdN75wkvUQ6bNhwny4\\\\\\\/d42YrIql1s3SjWdASu6ThK0fqUp9F102hJyWVjthGrH9Zu\\\\\\\/hC5oQMchuHWX55azyjoxFZVyIzpJKSSPabI4Qq89ALg2CXjhXInez13MAQoAYjnB32zTsbQUnoGqRWtiEjGR1i2ZJgv+ZBFGHhRQcSn9HBUSRE2NjUKlfbgjLqtYGSptxkULbAWDaEvxmO6tHdOEZQS8gvsFQhFriY7ALmSdwcIID\\\\\\\/YHD4Qo7HVUr2F8qPGmZJVXNh5Jt4zzFWmMgCw8E\\\\\\\/iIzg+tMbGKZ3CMzfaQ8nkKAg5wJblLho0G\\\\\\\/8OmA6YA7tDMZ\\\\\\\/f9lDL5fJkw4qYxyjhaY4NU2CHuOQd0cUSDQ6VghlHNqf983AsTNqQOh6iE6A1JSGExfDQyGyJ4euxEyX55cEfBk9tiHEs1jUhKmKovzS6\\\\\\\/6XRiuOywuvyk+GBKVz9gMfboyknqNrVZnPOGF2yOIpin89jp20wK8wJpuZQpM49OlE2eE4xFQm00t9pNGt\\\\\\\/wTzONkZLFN4baGSfkGPIe8h1Ju2qd5mBRFHmimLi4PvaOnjbRy4tJPjQxvwx3sNQ7fC3Epxx5uBCYHFjjuKOHqATj\\\\\\\/i0lBM91Q6J7ZIvk7rqqJrWrZAz6vTrhyXqcGvM71cru7Y3LK0sorLRCzLJsdVqsFAs2\\\\\\\/ZLEaa6haRmYONvq+AkxzJRFs3rGKV4+tNKkxSWIiqdTqjmBvmn9shbj\\\\\\\/2fJ84XyVK2gXF0cvUEfdECCuGRjG1+pA8W0oZr41T7TV6LV\\\\\\\/9kJd7LXKJDWy0Z7f68noVa4il\\\\\\\/lPJhhYKsOIApMBy9qIAdZZH65Lhv3NdmQwe0WbeoUUYtrY8JjISJgosvLM+gRK14OJlBMUq6qEY8omGVHt2HC1r1IUacVRL3DZ53x7dyVa3vo8hN4PYO3VrCbxnJcPhwQoBaSfYGpOhCNeL8r2SNDU+PHBBHvYzDZZlw6j3DAw9sPLO6KPip51M77OXTj2\\\\\\\/e3MhAZD8+4cegUzFDnmS3kbsUsJGHrUJ7LIyxbgBVRxP19PcL8nf6sLARVJS3So\\\\\\\/Yk97ELDxhh9ZRuiqIrr\\\\\\\/t9nFa\\\\\\\/KRnkq5YjMhtNpXL7UAHnslRBaLb7t3bf\\\\\\\/23vwBAaihmp6V0UHRW0y4Z8Ztzj9GhY73sdxbZCVrN+txHLTYcKdSNzXzuO95ZQwK8t5O1qf+CEVqz\\\\\\\/OHsPlHqVoM+L8vb+EbOR6dLUbFg6Q75QpgFTsTWJST9Rt1drOkdAQwQp6a64cLi5lfcjVrwMDeyUghjIeZvDyHRdOxYwvzOn9\\\\\\\/7fz+FPhaWc+4BWbQPxsmxnxw5HbmzSyRmVlNTRWhLnDniGy\\\\\\\/hkvf1KHCT0DaY\\\\\\\/gZPfjRb7EJEt\\\\\\\/7vnqNOVhPqyYdPTfuQ\\\\\\\/ggOwMSCbLIx\\\\\\\/uAfU6EuK3k7Zq1WvH3ACmBgWMfUhogmEQXed1JGlmtTmZgXFCwN8ywvALQAfWRAIV3h5KpF+lnxvzsYfq38p6KfpyrNa0rfWT843jdP3phu0krarUpY4ozITjzo9VjdNpWRhiCGjpxkcwsgaGmed2s06LMpteIgno1XSEuJdxn7rl7T5Zf0xnJGQlR74sPGu4AsQLfgH\\\\\\\/kP4u2m4iv3\\\\\\\/t8FpxkX34w\\\\\\\/OLlWwTUtmykcTbIy1fwr7i67F3K6sn8J7\\\\\\\/s5Ks37S\\\\\\\/fkQkWOORddI5m6V243434mz03nacW2xMiUObrzIdnrxZsj7LwQZnN44FReRoYz0NvWfiKuZ9r787NFYVYCihq0dZLKTiSdwmovjhNadhExWcsAZDvHWEehJxqSwgINMjyuWBHLXA3WNj\\\\\\\/VThAozS6KvHVA\\\\\\\/JMQXcLBJhf88tfdNVtwSTQJeACiHlp6VnZD9B+yya+Yn0NrR0nkJtMbKTOshbtsHRJMSw6xobwQuLTLLGhg5pKE4xkMY1bfHhGhh2+KpQgXALATpVFS\\\\\\\/Zjvl1PW3Ftwv1K0XORHEpTWSj8exHwUUbdZWjuAa3aAy0UNJ0efs0fmwqE8Y60Yos2\\\\\\\/qnm82ilsivPp0kE9C9Yaj6X6qgmpH9nhCRwnm+mkKqvPovNc5VnL3qsMD26ejBSG35+J329N\\\\\\\/wM9yr50PWP9Rp05KQIGZluouMY5ZpbZ0Gg1JkuBa1pZOCL54Zb9YDX0mYpTqdrjVw1awD\\\\\\\/zxbbLfxTGU4Q0Kcfj+ye8vO+bWsluWDX5Nqr\\\\\\\/Y2yPuocM7mMvj3svNS44V66wQ45sUMtXitP2tyLf3txxe9Ncs9f+hMMPm4iQIUpS0gVLuBIu+eieZm1crIZvurA1AiA0aAYYaoKQNhxULTDuTU2BlhP+glHkTLoveKfevE5zE1k8BVedXVAHw7NMzSGtEUIS3\\\\\\\/g5H7fBGKc8sZirX11fzGt6x4EfzGTAJzdIrVs7EbHt1mxP7XinkkzpiJUzsmXh1qEccNMteq4W8pJpc0mfXKXkNwZ0rLmi+5FkFFzrlz\\\\\\\/I4WERiKVcXTIloHn1Q5oFUD9YkOEDezE\\\\\\\/d6A2VEXuZNHO4MZ2LL59wIprd3L6aKIErmatRBEy0+Mk38vVHleWk13TIRZFn29bKzVhDIrkTCsgV2Tis5ZBL+qLiQTSXSomU2Y2LTtA6ZCMVblcx5eFS8b1fuVuzIlUl1hfW+S3GrP9kp0Eu9A0oQqm45YNGS+g8wpe4rHJ\\\\\\\/LI8vWq1+VcyQCaiqX68zLYstiUFzPfnevc+gcDikDnkAtC\\\\\\\/CceT6FAFiV\\\\\\\/t5UUqMNyWVZ89UXd8Bk7lNPCDtnnIHo1H\\\\\\\/\\\\\\\/XfJaEABTQQbABPQ2Pij0ofmkc6dna\\\\\\\/pgoaywGTE1LU0RZzI69GNWSLvhxdjOkOILw9Koz6yU8vi8CfVUdQ2iTDTPpHSpy\\\\\\\/ssns8MHiWRUlJOdxW0dDp7csQRnf4sCcVcl5nUORou9U5r62d3Apr1CTRcU0v+Gq\\\\\\\/rR5t3Hdum+tQmb+IMXGCQe1iSpT0WpveQf\\\\\\\/ICcWVakGIgs03FVYH6tk1B5SF1PKtbFh3inKof4KeraTCvMw0ZJS9uANIbVhZiaxUFLsaLXfqBqjmTHoiWaDppMZt998n64pJi3LBIOUNDnBGAzij44SWbVaDoak2R9vKC6nAvqffrZnQiG9A+0bRwIv9MoLo7VXtS1j+J7X\\\\\\\/MhrvjGwveSrkMp37NRe+yBcTaHLGFjMmg6+qzS0yjPgshZvQU+pQmz67QVLQWd7giTWAvm+WoXwoCQBc9tayDrWzyv3L54zWuJ7WjuESu0qYZkykBNeosXfG1a7T3Opg68YNg3aIPqV0MYed+XRl6N2eFg8peiFbNGRggwSMa7oEX76ITugAS8pIIN\\\\\\\/89aorbdjjzQfefaOSAxVB51+Umo7CWbWOI8Ijor2megJcHiq4itRC2qtzaC8i7tK5qjX\\\\\\\/OgT9\\\\\\\/q3PhQv29RDxrBScn5DxGWg4iMjL2\\\\\\\/nGvsN5ZUcWArh5R0J9LT5f\\\\\\\/ZuINKvwbToW1J6W1z7ga+WMuTX8uI6+Nx10rq\\\\\\\/xMYNbX6XziR3nSDbCkkrpQT8r6pMGJchpu9CXr+frFYEeRHM2G\\\\\\\/KfHmmTxuszLeZG+vyaoPvmyNaXKVQf911owHyLuv\\\\\\\/Oi\\\\\\\/Mg89HHf40uD72QFD4EzXzLPBvqmJEMcUa9v2ngcGioR2rXiFTnY99xZ+IooSmUO5vmvVwRhoBaEC\\\\\\\/S2kPQkrbSkyNEzgeINuAS9GrfP0lJHnNtMGwO4NZqv0gOa7LY2U+54k7RLI5RnCtDZWycRs69aRKsKFKA97\\\\\\\/T3lBBAjSZOBRyK7EG8wqcXlLFuhPkRIt00YZJRkJnFHF1xaN2gqBjdPTZweR\\\\\\\/LfvUM0969Py0jLpEDk2RREdfFYRvvPlCBoL6Frgt0FmrVvU3VBJRsDcSA2jQRz1YrKmLJB0AjquWmXkR6esuBFNlKhsNcDb+8X1H2oWrgcosf5O1iEop1Rk9gmaYF5Kq0AIaIcJJOrrBXGyoSWdh1O9u803MiPUTNb53oVFCG5\\\\\\\/gNa08R\\\\\\\/fSHXlGc6Di0qOxMxavkrAQl8xd8MlLGkf2UYesLKblGaBGlCbKqmUwb0T1PJ3V4vegKTNweUD1dicC73110X5wec9\\\\\\\/Ul8YC0TqafVw1NFYfGj+sSPSL68bJ1yH2QPJCGfFOEurIbjvDB7Xk++nPz\\\\\\\/vVaLo1J30OKOf9b4ieZt46Oq6VU7BCRRHTj2JY3gWH9ZHo+CC+UsRAfdnFFXP3ENPue4E8DIEu3Gmf6nZyjjQFBf6QHUX7Dc8tS1xs5iRREwPmgAo7BFefQpcmQvxFA4hP6CLyrzs03kKYxaPLhWZINPBmWdTHq6vhofdIbmdoWI4jZxyFtuQlHBC83\\\\\\\/Zguy1EwK1OqBS7dMwfwZu6X6nYom7zhoDCzhIXO6Dl9kM1MsreDQL34j7ew1L9oGV98jxgdIv2AC0TX4xARAp\\\\\\\/kT0qjqoYZ4n8xhtVZ1CUqDA3Rf8+FAT4d3g\\\\\\\/d6VX7PBfd9G9G8jOJ4zm8\\\\\\\/JtR\\\\\\\/BD1M4RwHO8xzGi4oi7d1S\\\\\\\/V3imDkfl8OcCgzjr0re1NG+cLvZBz51Rweq1k6iX+uj8KuBsaIl7dzyJWWSqhKI6J30UZnAIx2dZ9GtVkfUCImRcua2FSonBAEV9CZPc3TkxIQ\\\\\\\/EsNR8y7v6pWq0g8fZ4rfk5LBVArMAEvIim4IkJcE4eoYgMJajHxO3AkbLwhXBkBhcGmzmUG2Nv8aEcqAGn9XAK0VhxEXgdraLVmYQCUpw\\\\\\\/bt\\\\\\\/3IFcBmOGrtXchQre75sOhg8Iaxiy2KZrQwOyjeIx00m5nyTt7ZxOHmhpTaM8aCTETG13yDzrcLoY90Eh2XqqyBO1Iv9eZ7mwffCIfQof22Lo7BH3O1rFAw0d62rYfZwzaXhwTyPiyOk2jMjO0eUVA+nb7lD+uaoo\\\\\\\/amX\\\\\\\/uUemUmBhltpc\\\\\\\/qWB6+rumGQ2y883oO51ObbFPnk6W2rxC0rzydXcgN+dZBllZIZXbvAhgbGrOSZM+r4B8K79\\\\\\\/KNHop42oH\\\\\\\/A7iU0fIAIQe+otqgvZA0\\\\\\\/HqgeuuMwuuTMmFACkRrlbOicWCUrVI4AhhOT0sePJvo2dmEap5ybypg2RnUdzYEQRwyz36v\\\\\\\/lj+SQlAK\\\\\\\/7mDImuMyP8qmpCjys+bVezuWvnaiJ1W\\\\\\\/\\\\\\\/J4DMRd6+k5vU09XaMptZqfe2MWOWQb4n+\\\\\\\/dE+PxLRx7wNVuSuE\\\\\\\/GFWPetZ0erywkjbyV8jTHkKnl7xylwOowkqHZ8t\\\\\\\/fig6sfsIkPDBC43aRImGQu41S25aAESKV5oLNCKyUtq5LtvXXy8Bv2W7yf6TXXWWjJyakw9ysdD2Zvh+UrQReoa8UBk6cuc0HvQhEkdU0ggeEXkSDTTHDdxLVfsDQ+qgpaQlIIHA3BHlUvnmXsTAWKy5B1wAdcB0Xa1GBEfCROP6chlCyKFxb\\\\\\\/2+vxmcwTWU7L44xFHr5qgcEArN4tzwoX1hFxPWLAyY8jihTyVWd5rpKo5nygk55r5h69SaAh3UwK\\\\\\\/hay5VOVJTwcnqi02cmzWxvZ32snBbd\\\\\\\/\\\\\\\/E7B\\\\\\\/hMVy5EKjALVaHJw\\\\\\\/Vl5pVtWCO3bdoVSyn98I1fu8QQcQI+ursZI5fOeSwSmz0XS9uVuFy6+eHtCsJb1S44Z257sjYbSHaX2klCToS+VzuBN4KMFlZZflbs+lw6cQuOlaT1ftq6zS3Q+3cmylYUZE8gV\\\\\\\/svajcdf5u+UBxw98ECMch7hVaVj6E01HPlRGsA8oUJQdwbrAG1CibKu4mdF+Mqqfn\\\\\\\/x6VM+P97MkEpqy2NWGd5BFVZnrbkC639joLga9WHEH1jGZjZSDIv2Jy0Yj9LCm\\\\\\\/OIufIMFqJDpS0OCaBy44M7JBtJZnTi98Gl17o0pHpl0pHKrq0b3jDGEYSeJCT6xZBfpdP+S6\\\\\\\/iBwFI\\\\\\\/zHSRSaBM7pp2ATf8eT1WQA4Ocb1oocg92Rf+easI2KwY\\\\\\\/PoLl6NOeg1+P6EEQpckMlJv2QI+QaAqHWLL5PAV0I3YVkr4+xwP9Aby37fwetKvyFoARAMFGUxQYgdCc3z1JyWoPq8EtzGpAprQRmGPrzS\\\\\\\/ip9i\\\\\\\/EzSHNq4ZJpaEbpl7744vdmaYX4scynxsws9pD8oQ3QE7jd5Q96eycgmWwcMkio1jES2nh5kQLRBiq70aCNy3Sn\\\\\\\/TELLQasGt2MtGbOYm8Bb6YlPOyQ2GSJPh\\\\\\\/NDrlJcpndedzDpMxW\\\\\\\/2bXDlsCe6EVXWmy948dcxYSUogsmvMq7MIScb4ozRlFEkLUn57jKHDTyEQaXN1xeA7bH1ZTyzT\\\\\\\/iuSMfx7TDoyGBZnsZkFfxv7tW0gkSfqIaf0NzcV2j5YfK709hwT9vt0LjTqaCiZ7Gq4SnwbhqxkW2XSQ4+pm4XRUgsy6QN8vzlfPHAf0B8BRO+\\\\\\\/CiAmZWVE+woJaqdajMwgFxIxIiGLF2fbTdpL\\\\\\\/RnC0ZZlarRbWtcRCvn5JJPJv3K8v0vDTbbWmyWyIqsht3fzi7OXfX5DaNm8iykhh0+p3Pdv3h3QEA9wC0BnAOxpyJyLdrU1cGVSxPPd1oz9ygnx+2U65YbD7ZK9hZt2EW3PDApGNS4v3py3gJJNU4eCtdbaq7jA9\\\\\\\/mkoBURs+czJwCWZKIGqQBUYYg+rM02NvZ0b7uJRQ\\\\\\\/pjbX\\\\\\\/lVO6xYVkDrTnXYep\\\\\\\/csEguvHKOSFfBi\\\\\\\/3xMNE53LQjm\\\\\\\/ovJth6oZmbMt+oNSLM4\\\\\\\/AlKTGtfD7nvYD10DLQe+EkLyWK\\\\\\\/zxN5SDa4K338XmlK543B5mxYtQueYMwo+gH4+81KWAlIEL4c0RgDZTFqTi2bBMVFH675jM7Db04wjNXnE79JM7WlUspvDY2i+5LKtLZBnjd6uwdZieF+v8x0getBoubUebvql3Up9rA5OSrcKyRFpBZTWDLxEu+KLLbx\\\\\\\/i7e6+ntLyJAIy0yru7KwRVnAvw84D1vpjaTImfNatMT990SZ4Cas2KgKpjYQa0fvsBXyS+\\\\\\\/8xYcMFFZqMppYrmIpXpiDD1XfAz7k0DSvfEpaF8QjpxgCXVaUOGz9ikGb\\\\\\\/H5fjOdG\\\\\\\/LUim35lVZaDMwvGq0C1RMGMZBK4kjCP6WRg7kJ7LNofEd\\\\\\\/aYgBRP5qsDga3X3uHyYoqz3H3IZsYDCnVaTiYmTtzWZbp7ddMb1Hu3sbSFW4qQR306XryB5NgIeoVmFVma1FDOUSwuV03n7HEoBohbDeanVig0Jil7mvAA2r86ByYAbMoxPljcj4J3QMGGiQlloGUSs9U1ljXL\\\\\\\/r1E6DHx0KWBOo6JBWFu0JCncx1QGPU6ISrBmg5X9quu2lKAE3snV+uzaDlmiN6VGOGi16YiKXilpI+e1bRV\\\\\\\/tcRIBLQzuzUzLlN9gS6hvsRVx3oOb4S3rKDQGDW4p8yKI9zWwAKTHsv6NGaIk8+5iQ9RsxewKZIKLRqTIvpyQMrDfUErhGTKVCV9umZBnO5PVnyTpIORo7ha+GmFCYKgs4Y5KtMMBePx4FdznUqer8t8RRwRckyuOjtzyWpLuOnKkDB\\\\\\\/dYXDgLaqjIowmVrbVUJgkWsXgMwgjBk7WB1f8YbAt8irIffoRWW8rukBT0DglrzP\\\\\\\/MDFNsGInsjZ9q3XlxQQebUF0zh\\\\\\\/dXgXKFJ3PVSEB27xJ\\\\\\\/K9KZaBssDHjaVnjuWdzzDMS4AvlDHFzESpjgfpWljG\\\\\\\/vwad2cPYOQtbEPI0BIal98vgIGA9YEqHTjp0kbpK11ETjizz0PmkWpdmGSGYiIYnWU5O5r5yXclypICqbBygedWmVkSeOjI2DMFCpQVxx0yCpkP2Ej23wJmwG+47+S9YVigfVnnCmC8c6DRinWbQFPZk4aJMvKnJQ0DqlRv0\\\\\\\/cy9L9egnYwF7gxlYin8zh\\\\\\\/490dSpW50LrVv3f37+pB\\\\\\\/IRj9Gb46rl0TRZmvhp57GM8PH4U8Rb\\\\\\\/oi0QAjjVR\\\\\\\/nTAxaRcTNR6MarYIswNqaHTKzOvkYGL2FJxOSjCzrOZ4qBiionKBwDSMDACHOKn7DuifsOjvTWDnA3GxEPjCH\\\\\\\/X7z9nPW4FGYjZOwst5EdpuWqfC24y8u\\\\\\\/s+2VsGn4Ow7O8AArcPVi3u8jk1Qkw7\\\\\\\/3uG1OGh22Fp0\\\\\\\/8fQwKw4lXcYCOmggLMm1q3uFO0DBIpA5MFochtQkeh9hfxnXLU5cJfD1jf9dWKQdd+3ZKTbok90VH1DlL3et3HgE+j6vkb7gY7Z03wCA2RqhiyCFMdySI0xwhRqCAMEIbITNpdqw+8NrWTNK1e3CNddr5LIX9gGbyRLaSqjefuUGuDg75OpveeEamtXxK9vLyIK7rwMfOe1pH\\\\\\\/syuuabfPRy\\\\\\\/Wcbv8Ledl+9CxOXJrKCkESM0FYDysWmT0LeSFdOyQ3zJostx\\\\\\\/rPwRTFz397LIwwKLdUbf1TB8rGXoPrbcvtwY2fakygQhv5ER0wfo3omHR76TJW5IaD8YrqP9149F24eH1a0PfUhgE4BBJRq16YyldlFxkBsY7QCNdUwN4sLhfwoA4jjKGXUr3BX+PQOA7kzgWTBhDe4bgfuRWnkGFfl7VeBfhFZopZBJNqeq1EAKZTskBScIZCdjbhS8vgiCYkYEbTNEL2YJcPeCGCoKlgFszLNjYsvvlKGLpiTvuvaTbU+Ka9EaTeI0aYX1hqVF4u7vFEDlRvXn48498vu2of+mqprBTkTVZL3VQN8w7NUM4W2RGxRRIOoJhm\\\\\\\/KaEx+SNocph2SHYkl4wfQ1jljFKxv8y5xEy1CeysUn3ef\\\\\\\/9k63L6iCNEQ7QLpe6msUKshXiMi\\\\\\\/zSX8bA8v1zKsnHSf8s9sg8I78Fw1J1yzCbW7v2CLq412L4vAKfiqlgvdkOd6sWlNXqxvOezLczNoQsGt0oD1xo0Q2EBvgVv1Tzmp+fE2UrwKQpk067lDJxCc11xhgs5pzWaXe7vUH4rIoRJhmSdDG5zPWA9N8zWYstTAkxEmeqc3u3mDDiQ+z5XBfMjFNZEC64e0zdqE8ALiVyWhU2JYyB6qogcQ+lZyWAsu\\\\\\\/uXlylDHIohIDrYpZNw9NwQ4kUDsdjcPgEcH8G2cziGfYWcg\\\\\\\/48HEvLohzo27y4Sjr4PkPKs8mTFjwynKQO\\\\\\\/OyeE28irAXf1iEOTffe6bzZ2vEf8D9UyTw0sBJF+9e82HAde+KR+jbBW08vBSEoP9aOCIuQGNvwWSf\\\\\\\/DOnI8j4POo+oYwwTdeOMGoyX9An2wImm2yWh2RD\\\\\\\/L4sYdt8RiCGWi95I+ovsLMZVRXknH2PplkONiyN1l5ByL+smQIw1X65+BWjlyRrmCKayB9rMIH5OG778yqJnMFsoke71Q3qxbK1F+YeYrjrRXVsWOqiS960HqxLfAsjcwZ35Uu9H1eW4Nog0bH5aPfGgyB5WIb7pdJqrfMNT2nyAfmKg+hHmPc9PNmuknYtJeweHorXs4QgIOx6RLGIt1Kd9EoVrrjlrodBRNA9purTbkuMNNVAeibcS1\\\\\\\/veCQyewB0anF+\\\\\\\/vN+pD5UesWN79XpLMg5eTFjkQhMscdFIZulD04QLO2M2kLCe2zMWqZWyY9irJQer8QIk6rZF7eZR7pHXH6LtZUJeXZfN7B8L6rLPZtR3noh+ifKfnfEDpw+b2vrfU\\\\\\\/t1pvPFtmw\\\\\\\/RU3YxD2QtLOh2rP07LKu\\\\\\\/G44\\\\\\\/T4wZOCFtVHQQX\\\\\\\/Cq146k3s0Ee4j8+MphR1HW29DxemiUvpWURI7rrJblFcG+KpNuDCOYrEntaJutHKrNbHiOA5iKggTv1ZkfbxnB5eKPPyt4nni5pfiNeyL2n0qp25EjCztxZ6VgOHoOkuKgmxcM7gr9VkyMKlJdICaKZOfRKo80b49BURZUk1xn6RzAB+8NeKetHVYclNVs4rEXxNBWenb9HR4m6072fSjGXCuvxb4AUXJi\\\\\\\/XSvXy52leYwyhHn1ypDidsydi4ccpaD5B5c3DPsFzpiSrS2bf5gA6ASz054V4xUpqIhy9oJUz+SdtfokldQm3\\\\\\\/ivJzit4ufDyXiVxHfKNeMLzhmRzjiO21VAlmthsf4gm1cXk0jwVCQ8Dr+dCc5dE0E9s+CV1ya8duROcPbpMXRNGmYPJxN9oAEBWGApX3VSQuSISK7bBLlH67RyHLzAGT3D9tQ2F96lPUGOIjIA\\\\\\\/gqfYuBKFF1jAqQSZf+2pi4ZudL+hKpmvCuSK1P93YJXSETOqFDpfKh6JLlzd0TXi7KgeAPA52H6AVzNzCGsIW66tzSdl5kPJsSccyVoCVXhQcEswL2zCSV\\\\\\\/7oHUrw0BpEesPHrr\\\\\\\/fVmww6H+F8BkTfLHTLUROpyMdpPoZXUnqSLkwxsOSVaex+Sjk3hBQqJYepvkPdjQD\\\\\\\/MHLy6e5EMDndsnOI4bolZae8ambMifP7nJoMvuwUsEcbHdYHmW0rHm8tjHge1z3ZumzE7pgkQPwhQpKTsA6ofie8zjBBsc9tCXa7YPRcEPJtD+2VnKZYnrDDlRD8S8160nH16TsVyUM8fH+PqZWlDTvIu6ON6D1xUlqspWA74vv107cLpeGjEo0xUqOVlSXVonnSIuvFl4CwE4fXo8b046H7rD1UEaaqds1KahhBF1mdgQF\\\\\\\/X88DnwmVDgMEDwj85v9yqeWN+Dw8l9HjBzPsxKXV0O9oD\\\\\\\/CYGJUvM3E6hdcoSdBcBx5h17YWvfcxYsPibSq8LqbZ7sTvrT1ITR7tqIppHgw3UvSvZcEFrQyUCzRQcpc+cE+6Dhk+KW2n0gdXmKTvE8j4Fjoee0OUUg+iDW8h+82RpfaiBchAVJVuJkne4YQdmTLOso48aOqtlfpU\\\\\\\/vdKner8oDbL1SQs6ZrhtjEdOihWegrXT\\\\\\\/vQrzhM+qmOW14BJsNTyGMECP0TiT6vNmUpHYgf64VjeKnqvWEHFOmESmsV\\\\\\\/38mWdqiRaztHJOTntQvlx7dsg\\\\\\\/WWcyHl5GsJ6YWZHgXzLpPNIC\\\\\\\/wOP9+Md+S4t1m46TQg1TpFIiQ1x6+5hcwrJThAXVp4wELL9uxs5U7ARwjJwvs8WQjhPWAIqvh1CCMBhgmYZMc\\\\\\\/3mpU0J5MK6RYNYKEsFaXElXHRpqjuIRRIlKI7ZS3J8YhIa01nP8w1O91kXwsoZMhkrdM8qxZX2pPukwj7b7TPtr9Gm\\\\\\\/R9594cWf2mkZjHJIUCscrUD796GaaYZVIJfhSKz1kXwJFtKeHS+g62Wtp6vdZo7e3J74gElB36xGN4xbnLxIPsumokDOSREtnLjLvlMjZTjOy7hJpVtfQQLb79BJzpHyXpa2XV1VULlA5f8gMR49I3yH7SInInOgRYxDqkqG672q+6h9L4U1e2JNF7oi2Z8swaByQHGvf1I24gb1\\\\\\\/amjq6ozPdBmd88q6Y0hEJdY6p4gZEminqEVIek6JJmFeIfCluoNirD9a45iRcif9KcV1sRgq3n7ywrO8dvhiL4oogWWOyKqJMjQALme4azVlaDtT+oTF62vk6nG0bMK5+6ANK39jf1y+\\\\\\\/hQw5z+hCZKOK24VTib+DIOyRjGiUscmAsshmzToejbXEE+TjfjlKvICYU0WX3H8x2L\\\\\\\/Hzp+aK74\\\\\\\/TDolStH+bTcqaxlOvZ8g82ADq4SgggVsVukzOTW46Lwqs61UjuFZuxVKqPlv2pJRW+rzE1IqCCqP3owDJ6oa1fp6nuImZFk6CpUETmbBrJijbHXkUsSL5BZ7lVuFD9+pYLhMq70cgGXMwmAFslNDD3BcTAcgSCoxRfon9N4EjMjpBQ4BnumP2C\\\\\\\/9ZZTg3\\\\\\\/UVV4xb6iee3XMKeUsNBNNIOGwvCtyIMD+Wp0PJdVP5Wy2OvDXHyjb0vJ5rpnS1kONmf0D1lJ6NL\\\\\\\/aeYY6fMI8FB7cZVwnjPThYIEVO8v+NemI\\\\\\\/PKg3pCGeWSy6PkHcf8Lep61HCzI9T7A4BGdJR1qGnecNecODTKbMI9r3e+HGgNqM6SHG7Q\\\\\\\/7BnRGCQvL9IokoyjjzkdHGwM6jtauuMGrioGrcav9BOx+888RESdZGfG6iI4aWEeIoB049NU9FnvgsnSJ7eAtLiExuzu9Q0mGr2TkdL1PeInpqjz54feAOxS7ukvIsN4Q35WTpjCpScLZkoKaBVz0sqMv1ClH6ofpaW+yI8nL9KaaNhT\\\\\\\/Gd7o7Z7JOKMCJHc50sgSiXKoH7WVy3vBcXKvSGLmQeqa5MoNiryfJtmKbBAeXy2ll37hjD8tkqNUkPYrmdHfcEeMD13SMHGWy1QcrtUsnxe85Cuf2JmQ2x5HoYVl3war2MBTjE2Ko90Md5\\\\\\\/S8N6sjz8HR\\\\\\\/6svX0L1vEftEE4ZIeCkh3PCHfHAU+CI7gufE8AraPlUojsG71nzeluN8mr6iK5URgzCdL8MYdJFGpFkJvD2LQN\\\\\\\/gspoONYUu5pDPdVzNhRbgEZHEjvX4Vsv8ujc30Cbw\\\\\\\/16gr\\\\\\\/DuhwYf\\\\\\\/xUEUJOlC1Awg5g7BoNjhU0q2AznBBIz6ZZ3X2HJtsF9HEmFyk3HzwKpv6R5LMShb4yHz6Kye39wlEZj2YNdhdBEZJVni4NruYRYmy2PcngvbsbuKrXmLGx1cnFo\\\\\\\/tL+bU0LDFXFIrl44NC9x\\\\\\\/u3qh8LgRElbqeoPzrIM0hU50ONW9oXlye\\\\\\\/QLeZZlxbcgxIh3PEFwDc3pg2FawL2u+CdYg7rPsgOnqYtFHYO6ZKJdcjxbJkgUiflLL21wE017AFjm2fY\\\\\\\/FEl4he+pzXwyBtTbyVf0rxfryf9mQgVvDZgnGXSY\\\\\\\/EbOmnKDsvcq6EsBM57MV9kuoKfpI\\\\\\\/zt0SaWpdQbKmMscZsLPWNdgnTHRCj3LTLa4ctxmBH6vfz+1kZo45WQWID9Ditedb\\\\\\\/rv3KmHxMvWd+uCaZp6rzeEyIm+hImMuC+xCfOVCPys6CMVe9FNr8c1XNbVCozcZk1q07fUGl5Y3VBaZUtO\\\\\\\/Fw+u6XtxeJaSCBdNY1LPU580VirN47Be5Szh72uc2Vk2Wa\\\\\\\/dRRCMQbqaaxDqQJDj220GQBVVF9htWnjAfBEGSXOXLfHp\\\\\\\/HfkdUT0Wxi+YkSt\\\\\\\/LT7HOa0HANO8THRhzhGtt\\\\\\\/L6v3OFmeMojMAYnKMXjofhpMTMg\\\\\\\/A3+tH4E0yiedbuiQOIkY\\\\\\\/56YCVYN1qutSL1BPCA2lwwIZP9b+yQQaQJ9sPZYRcmD2t\\\\\\\/mLKwbWuLJXeBeqhE01xP3FyvLnwHDsoXnYoWCF6D8Ebou4yh5sEQLljGNZR8e3CJGutV2YqHhQwRaiiboAz2CE13Gbuy7LgNdQ2g8NXQTqYIYUzBiE1HSxsMZE\\\\\\\/kqG2bSum7MAKr+zmjXjIFjibUvS2LxTfXhrWXL8dnyUsSAp+YCPiDSJqvuBj0JJjTfDK0gjgVtSiM7UCchHSeyW23oibaHJT+KMoEZabwz3xTHXeG1bBbH8QXhHgahOyOhksWjyMYB7YcT3z8C3fFBT9s3diq0K+aaE5Q6XGkvj41k\\\\\\\/H7\\\\\\\/CArjILEh+Cvf+AYoiDRSX3AW6dO5yoEzY0zPoCi0aiuQQkyJQHR9Ful8KzZX\\\\\\\/Uv70wXPt9k8cqRaFzoka06nIHunbk8DgIEgHqrmwHwgk0gZP5Z09mQhRLVkcOwl\\\\\\\/s+WxV1vu5wKLVqu6tnqZbW8+SNHZ+lRvwRgb+fMTemXe9irN7fy58M+Kf0v1I6Y61CBDKba8pZl+1AHeTkxPxX59x68msev6LrCEJlCKCaI\\\\\\\/avXr9HOx2M94lIz14jCYYsNfK9\\\\\\\/o3xdq9CtE8GlSvW\\\\\\\/W79Bd7Ol0wgJEQKiBDXBbEamqfkZeBqtY7kbM6o8Pwlc7oM0i6GG01WuzDzHa4uOHeFLRk7+imi8\\\\\\\/okdNJun+KvblZgBX1+zo1mLNvf1MG\\\\\\\/cwqyAj+KLLfYF\\\\\\\/irbBFpQRhfWwFYdxxEf5gdc4TnFO4Jgv\\\\\\\/2z7tlj4p7YHvQFHqslIE1bcL5JZ4muJTtirGgJEMYxY4qwJ+YjzlfikPTlD1cmrcA7bjaPLGVrGYjXqE5nLHFHxquEMnTZKj23vXPuLNSCfLE5LGEynlQF2gAj09ltLVkejDapr2h73bbVm2Nm6AU2zXPMYsxBY3NKHr5qOzU3tHuXxVi0jBEMPEhdv0nwb\\\\\\\/qFG\\\\\\\/odgueCNq6Bgok7ja8p\\\\\\\/uQM0k4BDITG8RMbACuBg4emOOoHBvuy8tK1tP6dOib+iX2Iuglqd8NebvSYuPrFo3C+qhqUXiahLgYOgTtiGNbbFtstlszgTDs2I0yPvDlwUZaxjT5Rjp5M15BLO3ltM7EWEXtOJLjigJNyNFdtjp82G3ob18PoYqK8yK5W8+t7Lw4wkg9\\\\\\\/DEq8QpoJLFfIK1kP94l5yeLO8YNnKmrsHSah1JnxuoxNvjxs9Op76KxEIFnOYKHqAARgsnsxKMblPSXALHXud2MZgdmRXv6cXtCNTUEln4IgLkS9En4exRzXwrEa5EWfQLsa3hzSVjvFwKiBK9mMe4SNi9EDZmfO7nppKBbJQydwsx4nWBjWr382H5bHNxCqMxbNLu4GZVCzftkkDYtO3k\\\\\\\/Re7P0i+qLPWZV8+Yhyo9V00Lj2d0q3c0T4CGJ4glv\\\\\\\/VtKoq0\\\\\\\/ApewHEfErwriU7MAgsxJbGszOFj1DVDXUAeJ6fTyRqJPBC9vz4Lh5l1j8KnbnEoFhxw+nZNOB556DdEPa0ye0Kgpna87XH+wkpDAY15BUCBiiKAp4MRpoEyxOev9FMxo49uXyWIBneV0L3F4YK9GMSSQ2dREffJmX\\\\\\\/j06+QJhvnx0rKo3tJZmOZz3f9qRoqM9nGFkRlSdaIe6VxyYn43VJTNKeSFneUgRaYvvq2cj5EzWExkgAwmLNKyuuIk04JZBmbDP4Ya5P5I4WbwHw4Fka6j2fycr8tMoyzzFcwuhqWlKA+V9lkTolTxQc9cCY8zEgwLfEC\\\\\\\/uqiH5\\\\\\\/aqg8aRcQFGZNIQXDEu1SPFZQSwjanU0jQl\\\\\\\/o\\\\\\\/2o2z17FlaIAUzcLt2UMRD7uYgW3lryx5WGS5RPFZXFQls3kq24Wwn3ZsHh0pNqTHqIU+s214yKLX1AmegunyiZUnC0Pl1CM90YF1E8dYKRhi5qgDWc9MLVhgCT883KVyX\\\\\\\/ap2NYwjGbA5jcET5wFG68cjbcJtmsd54DFGHnUPJ7nNVYJyJbGUoC9PYhWCsfbCbY\\\\\\\/gMWVhZinEBVhCD+ZAwqzWiMocJoOSZZ6EOveIGxNLbMczj0keRnHxqug5xGbK1eDaI4UibscyD0+\\\\\\\/hs7ZjGRvm4stGYWW3eGg3SADbkVF7JgQ3qLv2hl4iMQq+L6jJW5kTefxi4ouRaPzSnwTtfacCzS1U0DeaPwLfjh8zMRQK8KBJlTpiQtbCWGCzRbgTXlcNkjWXrsKADSx7P7Knd27F06ye8cJmiSazrBmhspI+RLwaSWsKSFwfm5wlSFOtCuU8WGXwgJfFV+cxOYz5V44uV5X7LmeQZruUBA4Tg0vlpxZMlWpoJqdsaLAgybdzNbKdwqHYC4ltjXtQSzqh2MUPelNXNc+ZbBhoORO93SpLD1W4\\\\\\\/tS0+hZBoie+Tny12EovWsQp6vYVUL0dMpqUkPREV7HGp6JGfXFLm5J601Uv7NmwYxkAslmBcf7nb6RKX9d2M8CY1ByZPAxrJxiUZEP0tmCaYzklLamuJdR2XLnlr8oRoTWviCxLNDP1Sbcm9BEgurtp+5lwCnBjZLO2S9lHBMmHzpda9YpGsukjcVdEMDKnLIoSH0qWr7RccCvoR7wLuVCl4W2OiHOb2fPVkcyy0sgClBtkZAGcjDIhuOyPgvP7w0R3IAe+jxzkgUMn7SFQwdGB9unLw9glJbLl6LNc2EZpVcCrzVToiQp6eGBcCShr35tpv93XV0QvkfJIMQN6fr6DmmekL9jG6sGwMUTYw21eR57ghi2TlYip+jf+496NCWgw75SYlGkCITUdRVanou9p4QVPTKRTBfPsrrOzggPLMnpuzEkaoLIvbtMiZJEygo43T\\\\\\\/3ouzrxre7zQxOETqPHYmLMQUWdi35j+vN6lHQAkvc2rAM0B6Vt6E8wX3ijHHz8Leadw+r03StxBelINtKHvAwupBkuAAgZ0enkS1VQZVZYuW1XZJTdVsU0Zx++3\\\\\\\/wndiywOzv62kBf8eD5CP0gmEBYvMuEmER2S0lNmuWvkHrganwGsXLKh4R85ghU8IDO1P4cyBgJyelIA0RLeNDs0+FNTNr4r\\\\\\\/BBsKMCTMU4o\\\\\\\/041l6j+MiQihCpB0J6z9QrBMiY4Sx9bYuHmC6Da2mhAfdtKv\\\\\\\/HBlVAIi4KX1IPY+24LZ5LR3GgbUh3TF\\\\\\\/AwBVbZGzqnI9UH5vaoGsF6acmrkkjOn+AHzPpABbNS4HXsKgwdeyAWE3HGu8\\\\\\\/lkYhGCMuSigatl5IU9i8ZZQyun2fk2lrG\\\\\\\/UsYyUhwXhnTWeXIb32p4j2kIkTKV++sGinDkWBUyIYelO6YaMwRpe6kPIBSleluQ1PieA9RnH9bYau\\\\\\\/k44V6z\\\\\\\/K9VqbF3cYATlUbzPFlnPLu2+vEALYIp\\\\\\\/7rUbwC0f2KFWq2O8Z9gHrzuvlpdxyIKhw8s\\\\\\\/kwAWOp8bmJ0li\\\\\\\/dQRIQhcgUrY8iTXj9qDGlS+PxkEqaPJbMyvOqmRU5vW6jNxrVLjBAEQo5b\\\\\\\/AgdvKml4lVITTi9YlYvazoTWWmW1FzFHPEjde8AsLHSJa80nZT\\\\\\\/sfSOsWuHur\\\\\\\/lZXrRs1lBC9sURzSjqrsGgR8CqmHTXSUdJoViY\\\\\\\/nxk8AIGj8u\\\\\\\/BRarHWDjsKP4DQ4P3sqsadoKtXftG6P1EU2EoKZGc1mErO6ShU\\\\\\\/e3CysUdGsq9XOfyHVSFHoPMKY53X3pzI+8AxCXLa2XYqIvMo2T+13IkkpkqQWScwPP6lK65B4HYqb07j8fcYwl6P30haYWIhOw2CUW85b53QHmsYPtkmLF7Dfo4h\\\\\\\/rECfs3yTECde+s6R9PZFKELG5fzVLaXscYWSTVDgjzx871k033W7gurtAJPLj90WD4Y4hmBikdjDY4L9HGHKJ3XmreBivI+fM\\\\\\\/luOl\\\\\\\/w2I2Qkp1SfaJ97Wk1lO20Vg6nhrKJ\\\\\\\/j8aRp5y5o12vzs4cPDqCj5djuWR9QpyvsYQ2PByFjGZVgl5ryBXCSUgNFitrVG3XTqqRy+VUqD+qe8vs5y6aTRyuRfUF+rqG4Ddc+0Rxkt7otY98VT4GeJZ6G+QyVovD0etU7QZdgejsMfA+V2cbDNU+N2Y\\\\\\\/f5FtytB2ftAAWORi+PNZRO\\\\\\\/2y6XQOMrD7+LPKfirjYYYwXMg\\\\\\\/bplB4XfEgxjkbErYXR33Y6TtZyiRB33X4FtinBsDve8OYqS9uFebkVaGjdbxyd2PXPQwpLAaAJNEfIfZrY0p4zTHwOarTush7\\\\\\\/69BdY2CjJnPiaMzCbE6SFPr2H9YlqIH0m9R919qp5MyIfdi\\\\\\\/mAAOy1wTS6BIYRL2IaCsP8PFn2FIi3i1yHgd03+OFDrPlZfFZlWrPZ05g3h4jjW1NKpgW17jjRdchFOsAeRUJo6x33F5HUTD1O4mOBquR+p6A6xLF5EuJKXiMK\\\\\\\/ub2OoQinaTDqRnMa+7dOkJ3R1apMrLETgDKrobbQZvNiRhrytXgRZVhL2J7xOG+EFCGoi0hKBFr2M2N\\\\\\\/xGuvQzQa\\\\\\\/i9phk+NWFgBWHVzSsHaK3AA04M6vG9P1Hr+P3RbzE++Ki+EzhZvqZj4\\\\\\\/2gnBzIAH4r0Yj8zz83zpxcC1+YtWZFBmV+yHPF9SNOpoOEupcQLxzWdTiecmnYKW02hQQSQM7pyrgVGDwdSp+6xwPFn1DydB\\\\\\\/dNwVrzzgURa5D8HmCrycJHmqnKE1FwJDvr5VbQWSy+ghBT0aXEO+NukhD1aRB4Y5VtpN8lUQAZlZuvn22PxbFAoqoSl6BVjDQfriZWD+flOyiP29ZD\\\\\\\/5vuPxjn3Qgx1wlmulL9KWp17oa3nyRaVqCw4sYXnvwwaL3tG2jaSwGhGP8Cfy5BpYn9\\\\\\\/pFvmQ09VDsDd2mLtmPSvmNsPtxe55tl3m9TEIIIrYVm\\\\\\\/W6ywMzPWcgy\\\\\\\/zb7K1TwGmbYzrfEC0Vr60ZW0cvBi\\\\\\\/6jeY9GRsO7kPmU0UmQxqPi7WCNObxBupa1wZB8TNLUPZJNsmjgDJROBQHogv46z0NaPUWJVanXi1WmsgYRPj41Hm8x3eBmxJkKTxvv9mV\\\\\\\/1BOtMJUg+GOHOKQie77pW2dPWaQBBBiXwz\\\\\\\/y3UDZVJRY6ELrH+J96UIlO7aecCXlip\\\\\\\/EcFoe0hOTjASjJak6ARwiuWyyotHyJtzMiyaH3iob\\\\\\\/DbmOJcuOE1+YdwPmdIy7LpWZ+cMiHwaM3dx5lEZCM9nLFiyQleDY1AW7mrYGMZHe+pX8V84Ot++mi7fR4vnFGADmHtbkJcm5lIlvVIJjvBWOvBigAzUmFhrSRRzCsIFE4ibR\\\\\\\/DMx+67FrjyMq1p1KXN\\\\\\\/hW9fnDu1DZnzOoaNAri\\\\\\\/qSSkSHpTVtL0HgN1VsWlzJojFCAwp8E3\\\\\\\/dJahwqdhZvrZyb4h8gCmuZ2Yr8rIAmqip554vSk3j02CMo2XGe5WdpIsEs8XepxE61X+zHKTS35xq7RuuHK8P++UDTdJjJKAM1FZj\\\\\\\/gEbsScZndjiHYhaknTxFc7BKhjYZOfb04UyE4kujVav7kg53EJ0qnvF9IrwHbJw1eXnzsxdErH\\\\\\\/kPT5Y2VJUAUsvzNC8PsSint5KPDFjOGRfDqqAtxYmANOoYne9SMJjpDfJKnBvRouhDE87CC+P2PTHTctjB\\\\\\\/P8ZqC1cbqOHyHCFFS3F0eFl6jObW8Z1Hdxa39+VYlWV+pVgoxbzGHQCdIQD0iKAK69xKK7bfZkzohK7h17qE18X1d9N1km43MBjnQBMCsG+yyEtojTWrSd9WAM10Jr34THCBXUTnWHjX0V1pj4JDmIRB8KUUAM7zoagX2N7sK7by\\\\\\\/EJZycKyvq4p+N1I\\\\\\\/uvL3xAxLnSqwe35XNI6ZOJNFMU37SxSsW9oh\\\\\\\/XHIN4XXjk1xRDZVfDh3W2yf7V\\\\\\\/\\\\\\\/4JBLzdjZ1DNguRnNUv6e09ZSYdY9e+WI6CsgXux7WP2246UUiaBD2cE62CqcvDAaH16HWeGuBMFj5Ifc+0MD1yopyczKs7koiqbnM+tUdZccMN+XCo0xMG6\\\\\\\/qIKi\\\\\\\/Zj+kKTjXghn9asNlNDSbQniawlG0ARgXVau+LOtVN6rC8KSVWcQxoNC8qEew9OboVsOIQcd5HSH+2JFHHg3IdyjDrGW4agljSg1NXYIQeoj+iDXRYoabdB+l8d0i7iUC2g2Nc4YZQgNm5VXwlrt\\\\\\\/hbHXveJ5KWhPORb1bHFdOZPhcFcYqB0GzjIg2GTzT9EVYppv0Q2uU5N7Bw4jjCBAPen76NtttJ7aIUYFmuu0yYO+RA0+Pt0opjgo68+Shxtdfm\\\\\\\/izc7HVEiFvOGhBLLauoIYCEOv8OgQ0f7vkMLorIECoNcfSZgOF5nALsfa04rpX16Oja2WgCipC71iNNULCL4p\\\\\\\/mpPRcZoMIQM3J7YZuemnKzPdcoLKASHomYDHUYonTKku9fGJC0pPWj3p\\\\\\\/Q1aBNUZNw4gqQzGUGy8zSVBJTNnHgiVmH+WFSGK8ufxSjQv1w500SYSNUJZbni8XWOEnI1jeJlJu9KSgiHaQW+QVc0qteTOCm++9n3AIs1PosDOzYHWY\\\\\\\/l9IkaqzjYT\\\\\\\/G4T7\\\\\\\/955cIeFUNZxOOQ9zODNpA3dM8rdpclfQzQCkdqZVs\\\\\\\/P87q7W6P2yYETiOcER1JShvXkNIS6x1zkkn5LjcO2nEK3UzFhZreN+G6Ckcc5sRzOWnAolSNyaLXrvXTB7gk8P6gX+NNCY1BIxLWQRIlSFXobRFgBRNqLhcf2YLHDo5l3eiAPmJEqmAh6Rz9XF1bHi8UOcfOpchxzPdZHzXaq+KnKQSMyby0fhgQxBqkKezY5F0Kaw9+4Pf0OnDxkQqrbA2qtePS9PHb690T8VmcAC\\\\\\\/lyVFNG455WV03f5qBFQPvCTVwpLcL7n16RZbk9IS5CgCYW+JXrVp+N3clMorNkPFZpn4zPtDXYJ\\\\\\\/oGGraamH8fmV8emgozQ\\\\\\\/9Qv+7PVwpzlhaxfhcDH05hCkaRNHsWI\\\\\\\/B\\\\\\\/L1ctQO4Y42k40uFOQfDU5UyV7T+XeuhODfWE1KTZzZ+2vXGfdaiDOrcZraA+jVqroZbsiF0fuGCeW4I6adySoMmPXxUAud7pf1FYqZ+MFoPtsIHvv\\\\\\\/7yUcCAsHCSUkzlNe3m\\\\\\\/IeQLsOsRRQYyl83yqKOVKKvltC34u9vjocTdoRNmIYVdb8HIZw+WZYjJ7eHRO6GxD3z6E1Vtny7o+c7cVnnwshatLAufUA\\\\\\\/Ly3tPjtCOLlhIPsteK9y47NMsoJve7cigKD0h+Ev4QtP7jzzbqyQOeeI5hZo\\\\\\\/7l7ghxluy9wXttax18aXwd7O\\\\\\\/QCOFfRxJ2ua55h5yZhjlF7YDl7BtTMyispSxKILH0VYvJa\\\\\\\/H+xB1PHLJ0BwHKCyWtLHeGr3pZVoAtEQJ1vUk05vWwPk0c\\\\\\\/6IN0qob+bjX7jbVAZQe+cCHkXWRpXtJSk1iPKofaHKx1pcVt6H65jhXX4h4bFnathx6djWGyO1RGPjaMWPxVhLjA2XAwMTCAKaGVlDH0yQD6JuxSdHlmrin9tlsiJ5SbB3kVMXWmbpmo3Zzm8uvp9E6jopHg9lsHTCyqcOdN2Ea7qp8lYdclwSqODdgYaGF3MJGzyTIlaE5r8B\\\\\\\/1HvEp3KqBxq1c7rbbUmzJBg05C24jQu3qq1yK3hkH08vPTXOsD2b6xpzC60iFH5qHflk9y0ZIr9VBVdsz7GJXKzylofWW2OYqO8pYi7UOqQf3Hlvh2qooJAUpICDIiv3JwBjIfpdvfQhag\\\\\\\/ouUnm8OGuSBhkbwf2lHwTldlakVAnne2V4Wb8kG8NH2lyOITP9DyGv157Ay+Zx7nBmi4rORd1L82nzCuoPFI57F7SiStE5f3o3uIGCj6pU9PeGXJ+goiCjCT1wfFagjB+POMQwNQxAAIpWCPCr\\\\\\\/hyMRAWWnrjfhNqMu5XbU6mccaYNffQxmUWA5h58UBI7wXHxaOXsEdoVPvj1VaWxkNqXitE6liX4B5h2gfyFjD6pnv5B4T6s2kG7Wv9DKx2H2ffldeqQUyd5LEtbR3bUQhlFL7hWnTaMEj2RHWkAoqytkryD8Cy4jMBfCBU6ycH0sX0h2DzaMSRETHHCLROGVITt7rB3FbSMQILuV49Scd0FhgFXcqlo1wTMdJ561GHfXiNNhYKsKxSSPH6XEbSWQKaTHiDB2\\\\\\\/Du9J0qBXLB+YS3lGZ7wsCp7OM85zs4rJFjl3HiDyZl14pRC\\\\\\\/OvqLHmadBmNhp9cAusavwEcJat+xRl43Z6Ff1TvjzG02jum8rjMf9SgG94sS0twhuwvq1Id3NXt2ooT1ew29E3Y+c9bNzLsxRl19bOKKiHJH1NTwmc1CzuFQOtNSdxxQlQoJ\\\\\\\/IhYqSgKNlAIjTr\\\\\\\/KuVtHPmJJ03\\\\\\\/p+1zaQcEjoCRWbszHOVIz4WVxze4Cmq7CKQndYeTOde56rHkkftPmbRvpJR93516Nmlqtvis61SzQ9Qp9SA84Ioxdee+dQ9GslzagPOF5D6WTv3gx329qR0mLbinCb3LjBTDVzzPLA3GDsPNdUHzNsBReGzcyM0oAYNhXWhafnQyeV2OdbKGlpUmd7ymb3p7RduRiDanTtGTR2qKE\\\\\\\/ML\\\\\\\/vvNE\\\\\\\/Qx9yeCh\\\\\\\/xgZ\\\\\\\/sNWWrhsdaQecOp2DjOtJBhKWgaXWq1jjxPFtBd9QYz41DeWDu\\\\\\\/F2y33HpLHendn3ZGJJUyPeWT6QoyGPV7\\\\\\\/1Z3otDHPfN52XeM5mF1qaZFr1u36i5YJ8UaFKGy7diF5Xmawm78nMgCPM+geUyMlswXw\\\\\\\/I9H4z9AZwKzd4a7vB35z0nB0wSuV\\\\\\\/xN4PJeVtJP3n\\\\\\\/locmdWnG3dv70QdgSOTS2DmbfiSA+x3\\\\\\\/ANA0iF+mEh66ojSifatYTe1W\\\\\\\/rL7YuqV\\\\\\\/gZRJa0ChFoFqcqi4h\\\\\\\/+WPTABeW4anOwFgaFf34pYgB1xmv4hZ\\\\\\\/xMRMdxkmsAXBUTqeayG7VKaHzb6cWSk+ybyV5POGHB93dJRFeVKezMO5fsN304x0NRXP\\\\\\\/kTUq+ZJ2swODC9Dbs9W0uHTBkdR3GIMGfPVirTsBSkGhMUR1SugLSu0GvNhFWe3c0boO8eeETxJ1PlzKPwtPXLy6jJZNCTeuJOyo0Z8KkGghRjeh0SXTAiIZh\\\\\\\/48FIHvik1uxD+j58EcchMaxguJZU8NdfEDjbio46VpUiOjKqOcGD09hMmG1udbDlvNz4GGysIHT3QtSpfOgm24F9CtbF\\\\\\\/IH2RzlWtvjRoG+b5Q3hZMh\\\\\\\/VQxVG8Mn6lHyRrN9iMvHLs+povJ\\\\\\\/Ca8SRDoN4g0FROkuLUBkd9AiFtO3HSJNGs4wu+bIQdMWSk3OveRPP2bDOUr2eLoLJbgD+fDIusa+kY5cY3EtNJIRi5G9SkUmZwnBVD1MbsxUbAbKi6O2+\\\\\\\/UTNmKYu+apT24ElDh54f1oPjsWwLnbTQqpbuBLXvOA7o34g+fVMDjf59usD0N9TEgMyc\\\\\\\/P1BYS7UFfv53oPlQXGWPertYX3fOFVZqW92maGKh6G82muhBK4yMs83xioTWq1V7rcJbbvLQRyzqq7pz\\\\\\\/8IiugVzWz5XjjyCzAqcmA4sMMVyp+SuqYUYG4NpjUdA1gAX7I4i0b8X9SXehtJH5poPnPQiidckUw0iV0O2V\\\\\\\/M59BX+88xR1VhdUpuGrHWum87q7g2PZV8b7hs\\\\\\\/ehLgYUL1v1YJ77Ht7ju5xDWEv\\\\\\\/bE+ZrWg73jBebGnG1jGLFKG468xIcMaU9PIrgX26r80y3WUN1D9ETntc0mopIWIbaxGFs9+\\\\\\\/TM3W2rFX2bqYHRCEYRDEtnlxZZyFq4vriIoWRjitIBmzWvrMValecok\\\\\\\/htizFqZiY3FiH4ZLgFa7c3IWBgM\\\\\\\/BWoc9vry2X7xply3+2\\\\\\\/tqiCFvaFi9X0f7g6tq0iocVxIcxDTKHetxcYgEwHh7rQ3HabdsTLmAcrAujFt840czY2pYryNwzCQSaGSAwEysMiYno07RUomCc9Sm2fU\\\\\\\/yy+Sj72FiRLtdmUUYUJZSmnRO3IjZA3sV8oID+2nmltFHg8IzjlLa19mjsoxpsc7glOzC4RUm1UVC95eT6BApQ\\\\\\\/lucS4CoaIRnTOO1gxtIaQXJxXF49LDZW7gROtlHd0rozVN\\\\\\\/cuZUA2vbz0pmV5bPK\\\\\\\/VU\\\\\\\/SFQIpdxrCpKriasAG2aOAV2U9C6tJyHRhO5u3nhCFjMChFh3x3i2XB32MqghpFiYN\\\\\\\/b5caVDc0MukU6VBbSck9u67XzhTNwS7pyuiokJuYIlvSMomfV+OHWqaY8RY5Qmc32t4PiMSJ0T2hFnLCqxSaJ4keWaYgIBIPFfAUvgskY39VSxPLWRUu1RyyI6kQ7B5WZSeXG\\\\\\\/9BvfTAI1sHHAC5XdzZxNShYOVGvEkbbtAzsS3VMrA6Zbg77WkJsX\\\\\\\/Y8RHmWDUpQs\\\\\\\/SaBI69mO9rD32jEZbsLP8EmzyXaWmpvbrV+rYNshJ0ZJDVW7IvJFo\\\\\\\/oDPm6DG8yVMBCgvMZassjgFs2s9NKhOnTE4\\\\\\\/zecBB8lBq7085jQ262rVxIjOYrTenhvHDt4TFbWsjzUwYZGxp2bR2tv+azJFtcqjVtOyEgVfF7k7hAp2Iph4pH8UGC8HDBmDGahF+j44tsu20Af8+KvcCHaRGxKUm1eUkkGuWq5M3J5esfMTEbv+JmZB90YWvs+EpAPKJ0mWACdVzz8CshooHZ1FfxAOX4JmBTVpr5QUdtfLIcGHCwHlrbDvB4veBd9HOFT91p3pxW9u5HvxQvMbm\\\\\\\/3luyP4jVRwY\\\\\\\/G3qUXRiHTK6IHoWWAQDahY7ODK+s2o7f\\\\\\\/pilHhNDZquEFtznlSB6Bx2yXAu+atwo\\\\\\\/49EyRu3BDhdFVMd4rSlz115kBmJTxewzsxAyHsw4Ne0yFW9vd72uYQLG+8wIYyMrhNk\\\\\\\/Bs2SNJ\\\\\\\/EcqHNMtwsMB3g2TQ3i3da4uDHoGMYv8sI0nX36Q18U\\\\\\\/eXpf4pdB2dqFdKD6QqE7TxSEr6gyoxv0IfMobPTNn1\\\\\\\/WOPslQTNW1Sa4R9FYXP7V+RwwLcMcx5+buG4\\\\\\\/wIZEhMGC31cLz4recjZ9lX\\\\\\\/5Voi2ycMTZ7d+gZy2o+DxtH50KQfpdLw0nPO1pBy3Y5Rcy9NMCF+hybpUFFNerBJ1MhmXrzU53FJYtZiQ0ZOjM+AOTuVLxVrwZcaQWnnqpZ7cWOV3eoKSpcGBUZa7+\\\\\\\/N6uQyInFhZvrBUKhPE73QPEiRecU+NOLFrbhmx0SdJzsBaof7\\\\\\\/VJ0O4v2va82V6\\\\\\\/CS5eQs4qp\\\\\\\/8UtOgv5sjO4Y0zzjhZo39leMzCSmr1yg8Bz6MgVDH\\\\\\\/x8Gacphluz6\\\\\\\/vOIUnn9DiwFmwLIGjU64lAvYn+wiY9+lK0D7pGlByfoXydSF6i8EpvcpBU497ms4+V5RKoaLkGgXyY\\\\\\\/PMI\\\\\\\/nZLwuJZGAzQw9zwvUhKIPdnxyuF+HWEHCxRb6o0yNxPe5eH+yEntK+B+kNGEIXKoUW0arzYNmfetmG6x0lo9jbxZUfWjpWgIIVErPvG7Ch1zuYWYdsu0tDlvmJgijBMy6mzsT+QRuZ0HcPLecBqnlKh4JG34EPUdmjv\\\\\\\/EthfFZ4jcbCfI7411IPx7HRiUznrco6PWEiUdduOe83YVXUgbx4P0MVo5wHdgfR1nPdqMSv2Njq2sFXUvprJCNpaDP49fJ03B09HnCFgSk0k1qgQq+WAuvEtq+uXj0A0J3REcZhqG2BNrfn4HsjI\\\\\\\/+Hkm+w2CcjpLfHxPdyeI5v2NYL3mLmDHPkRn5bUHBHQqoKogqF5soviC6koC+7gmAfH\\\\\\\/lOkA8jdg6qFp4AcFjHT6y7sEc0n0pYbkB\\\\\\\/e9mka7iGV+219iB\\\\\\\/S06kmy3js84BwNGvuj6sOMB9Gqfrwl9omm8LmbsnXLfBl9uaYpYCBovxxqHUSD96HEDskcuESLeJzVx6Nx0YoUb8zy5iD\\\\\\\/f8drRs9lkmR+h5hTKHGfWx9c1IzLyD0K2d\\\\\\\/GrGYLG80ULU\\\\\\\/\\\\\\\/gjV9kc0KYZOpV\\\\\\\/65RWX3HCWo4hGKJbYazvdGfIal\\\\\\\/pcMIcu7NPI\\\\\\\/OR6lI\\\\\\\/BqkckG2yGI5dpkLLem3xixaYPeJ9Nvd2oAqymaYMgPCxgmJ3YlaCPUOyAPWPt6g+paDy90hSbEwroosMQG\\\\\\\/0vHKL1J\\\\\\\/F26uv5\\\\\\\/BMM3pbibX9tKDZ0TIPOWIwmes78hNhFjWlJWFzmB98Fij9ekw17QWjsORq6MngbDdyuXsulReNDpx3sl8F5hZjjNW8y0EcdahBmDAJF3zFZg6QmzUMMGQzLuj65dNMC1eO+qZzFV8d+n3Yn3Cf6nf9sMI8jFDNrBLLuK7SFLeh805ixVEOSCgmgryW4nAc+MKbcnoIEEsneyCZV1Hn0u2wvxJwq5mZAbFtx7mursUVKu2EeiHsnhg4iiUNsRz6007L9J+IeLCwbGUB0wJmIzCrHlK3Lof0rd7rRfv1okCZrnEVLPnI1JIj\\\\\\\/vhmkbRIrRyFVlpF3oyPnYxqAemtpabJYJMmLmXbyxgGWLHfAEjG6mcpoXp1iCz1MLdYkeZ4ZvO0z9w7eCO+FnKBnjzVtLfi4QnERTCOwRAT50Le7pBXe7kKybw45dbxunj75oidVoynB+wChD2g3A+xdhQWHlRTSZ8F41dc3iak4UxN762iH5kY8dZ5XokbVVdzXhdlCArPhcF2HQqyWQZFlbuBK6BpY6hvXDg4PfscLtqX7Nh8kYKRdIRNGjCUbadrwRozbb155i5q\\\\\\\/sA1UhLSdagPyZQJlm4+Ouor56WBd7obTVdYfSlko+t\\\\\\\/ikvCFkuTbR6dm\\\\\\\/ZglzPrWw2Jr+F4f9SnZ0ZfmwBfGCOpJqmolPYC9cjSbyB62VFwn4r4885KUeKxy3JIBICrGmdgoxeN3BVPJWNEVWZLNRMb78XC5jzUjRnYjxFagjBRSmfhRNvcZdOpFH8P6hRlGQZFl\\\\\\\/4MoGOu0dkoWk87LwJPyg8z1gEaZ0NwTHOHsoo1ihTUmtpwZeEduMnWe5K2rNl0WDVtpGybUAfSz8k5FAdbbkFExHjicDRLj+tvRWJdJmd5GrUGY3+jy8PeTxS8e87sHuctYewzkphklgHXiWsZKxnx5z+Z8JZcLs1GhP7NPwygH5LvPg4+atcH0\\\\\\\/7QsYyMd2qvNpnsTCSeOhSzj5clMqLMGYOXL45G96lgmGI8cywxCuLQotdn2WzpAJejsgTd9rPZIMrqOvvr3gUVonlqvq+myXBSsfRu+Lbi3vm+BfzOEtafSqW4id0e2m\\\\\\\/bwOvgouTsv6z33ZUSIKfWWBeYSjFqbgq\\\\\\\/uXwf2gSl+qQivBxHvZL79WfAWGO21O95vLPgQWqJ3UPkP4IPLWlhwHRX3jfvStUisVc2082Z4\\\\\\\/oxvYgRXpb9H6BFo9WP7X+UTssZw7ggkfy2SkVy2QXAFBO5Bn9EmGnUlPiDRI2gaeNWx8qoEktgcppcFdlxcA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022cMG7\\\\\\\/zO3k5souY6QoCsSSA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b82483ba7d11b6f7c8ff6acb02f2eb8f1a9fc0712fe306da54ed8c4819599657\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022cd6qIWg0Gr\\\\\\\/SY1xO0+fiZanL3L9MGqpsAdvDucamU+XNKmGIxTRRtUtKeSyvu6b70uUEl5HylDMbsgBSL1VID0wEU3MAVrEl2r6YybV4CxEb2MsKyTM\\\\\\\/L7nqvY8FFeATX6QteFuQz8ohI5PIysjOHpBHENV1cSRm8OtSVaYuoIYkfTC+X8B\\\\\\\/6z1PCMK3SS6M7cv2\\\\\\\/\\\\\\\/xf5ybxMqL+ubxTRAU8Nb\\\\\\\/+ZwWeaxARaQ0TJxMlMtXP8MlReGCv0xlUDr9n1a\\\\\\\/u0oOKUXlvaxejqjtUAKniGxAY9VcHH1HiOVRSodJSyBVD3aN9\\\\\\\/NaRECyWKJqHTT8T\\\\\\\/6YehO6\\\\\\\/ZrbpmQaE1wWCkCtoUTWr3N8RysCoIjCMoZcyyVg2LGWqHJ94h2eqZl0k\\\\\\\/ImIWPv1qO6gViXWp9YAx0zqdgaJHIDaAGHdqIYISK\\\\\\\/qpJcmCuU4qHgydJOtAG4rOPgs0uuoB+dQZgWTg8xzuojdJ2REKJ2c4O5oD8VaFKRixjjaGWDY+hhVFZBA2ovT2cCBWbHdgx4g9iB9boORybl\\\\\\\/a8ra4UObwdfNW5kbb5ocyF5u2eNZDmoCi6Y398AeKr4QRikRMy8aFVmMDo9R1KQEiJaJPt41VrNMqhoEtbJbMR20sae8D5758lEUUopi7JeTeWUiRcaKvbvAtPHuiIw1CK2Mw0VYk6NPEXWX2UDmsKaquwtYCgzDaawhCeG5VOdTQRBV65WiJ\\\\\\\/RSWJfzwWwridNNdkkk0Oi39BiD9GutFuVd5DIkMIe5orsIMwKqYKz4B8Qm8MmiiE378PLOewvBN0y1sWblYyltA6kzkLKJLosYWYwZdcq2KJLKauYE2O2Kr0W\\\\\\\/i5TSOrLqi1eUUDh+j1vdGVJAcyGdT6gZYYprNyqPxfComLygX5VNIhQKO6Ovv4dvWHmtw7210oUHFK2dkwtbg6IEudgtqqb38ymbOFELkJgf1ltlwRsLOnUoSt8XfpRWM13FYyD1tu2uQLzT7AM72e3tHnOXjYL3xv1ZWLEGtz1ZaasIzzYqLlJrXLBhY4ljpHHXXKHrRPC+cSfhJi15ymB9OAgeTO14tLHuxTXE3FQGSAAXmWj5wBOZWHlPEOEvv5Kx3ZRlAL+xFUEveAWsDHL02Ka5JDug4V3ioS\\\\\\\/TSfHFF7fyVgn7gd+gtPKfzUCv+PYypVKFgeye\\\\\\\/BKMU2jzajPUAPHTHDvCoP9aILTq51jCLMsj++K0jVWUXzO0q8nCi34fc9MVrAMBxK0neeDTjDNleYe8jbekf89AAEYVhZxRuRUYde+0lIjpFZDWku2xgeOlAqegm\\\\\\\/kvz4Pikgimn89KC13uWvHX8J4CaCTGF\\\\\\\/ZVVNGrrRRiVIcE5OUoDZQ7k4eHWl6w5RRPwOqQRW6sBK33aFie67yAnLC1EazW2xnNFb1Dt8i\\\\\\\/pGM69tDb7s4tm5al1DqOnhQFMqHalWk8cELT68cfucaaf5AVJlDEfQeGaaGkpjmO6AwDMpJiZ17y6WfIoS1WfVU06ILedi6tk36kpYDIuCWuoVjNzPybcmBkwW3VymwPQ0b25OEVbTvFvyQ6NxWDDHE8T\\\\\\\/O8hvwF\\\\\\\/lSxCwxJ0q+c6VS2BwhKORNV\\\\\\\/ZOxksKXoElskb1UQFUTF3rlruOB0JWM\\\\\\\/5my53RDkC4srI5WYAzE1LemyKozmnuCTwppgfryhqIXfaT5MkiYxpTqIQBXLYo8dMRpHPoi63uNUd9VJttJL5LHN5p2j7DTnqttMVLSCoxfz4oUlSq+HJ6tQxs0HzYtSJF324Wx1SuRNXNLqmg\\\\\\\/GIJBt6ROLUBVq0MkMnyQigv+4BWjtOAKqxDkDkRgmki5M7NW61SUJAs75rSTf4gBv+SUb8zXq0td\\\\\\\/ve5pyeBCH\\\\\\\/K50dG2oKH9mGx1OrS3gmRBbfzto9Xhpg7px7mbJDNPFYwMdXK8MT7I1AhpFYTEBtgPn33dAfcvC59gkEacWTIQ7KSdvOI\\\\\\\/7dnocfH2J0hGzKb4S1g1bGmLGTJupThvFFJARcSkwJxfTU5Kjdb4A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022NiVn08Ekhz9f1LhXTHkZ4g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221f98b22a606851dc10e61ade2c1e4970f1453b8e9a67a54013627f97d3837b8c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460127 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/1049" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-HJWz3YIG1dJT_KkqVwCwZqpLb8Hiecy9qr18QGUTGxo" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/1049" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vjx5TIRwddlKoAcB8v9YE-1Dh39TLRDSEdIJTZm5UrU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/4/1031" title="The Stress Granule RNA-Binding Protein TIAR-1 Protects Female Germ Cells from Heat Shock in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/4/1063" title="Large-Scale Survey of Intraspecific Fitness and Cell Morphology Variation in a Protoploid Yeast Species" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460127" id="node460127" data-pisa="ggg;6/4/1049" data-pisa-master="ggg;g3.115.024950" data-apath="/ggg/6/4/1049.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accuracy of Genomic Prediction in Switchgrass (<em>Panicum virgatum</em> L.) Improved by Accounting for Linkage Disequilibrium</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Guillaume P. Ramstein</span>, <span class="highwire-citation-author" data-delta="1">Joseph Evans</span>, <span class="highwire-citation-author" data-delta="2">Shawn M. Kaeppler</span>, <span class="highwire-citation-author" data-delta="3">Robert B. Mitchell</span>, <span class="highwire-citation-author" data-delta="4">Kenneth P. Vogel</span>, <span class="highwire-citation-author" data-delta="5">C. Robin Buell</span> and <span class="highwire-citation-author" data-delta="6">Michael D. Casler</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1049-1062;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024950 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460127" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Guillaume P. Ramstein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guillaume%2BP.%2BRamstein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramstein%20GP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGuillaume%2BP.%2BRamstein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ramstein@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">ramstein@wisc.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Joseph Evans </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Energy Great Lakes Bioenergy Research Center, Michigan State University, East Lansing, MI 48824</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, Michigan State University, East Lansing, MI 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BEvans%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Evans%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BEvans%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Shawn M. Kaeppler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, WI 53706</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Energy Great Lakes Bioenergy Research Center, University of Wisconsin-Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shawn%2BM.%2BKaeppler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kaeppler%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShawn%2BM.%2BKaeppler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Robert B. Mitchell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Grain, Forage, and Bioenergy Research Unit, Agricultural Research Service, United States Department of Agriculture, University of Nebraska, Lincoln, NE 68583-0937</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BB.%2BMitchell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mitchell%20RB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BB.%2BMitchell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kenneth P. Vogel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Grain, Forage, and Bioenergy Research Unit, Agricultural Research Service, United States Department of Agriculture, University of Nebraska, Lincoln, NE 68583-0937</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kenneth%2BP.%2BVogel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vogel%20KP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKenneth%2BP.%2BVogel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">C. Robin Buell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Energy Great Lakes Bioenergy Research Center, Michigan State University, East Lansing, MI 48824</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, Michigan State University, East Lansing, MI 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C.%2BRobin%2BBuell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buell%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AC.%2BRobin%2BBuell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Michael D. Casler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, WI 53706</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Agricultural Research Service, United States Department of Agriculture, Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BD.%2BCasler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Casler%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BD.%2BCasler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/4/1049" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460127" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460127/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/1049.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460127" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460127/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/1049.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460127" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460127/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/1049.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460127" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460127/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/4/1049.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460127" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460127/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Switchgrass is a relatively high-yielding and environmentally sustainable biomass crop, but further genetic gains in biomass yield must be achieved to make it an economically viable bioenergy feedstock. Genomic selection (GS) is an attractive technology to generate rapid genetic gains in switchgrass, and meet the goals of a substantial displacement of petroleum use with biofuels in the near future. In this study, we empirically assessed prediction procedures for genomic selection in two different populations, consisting of 137 and 110 half-sib families of switchgrass, tested in two locations in the United States for three agronomic traits: dry matter yield, plant height, and heading date. Marker data were produced for the families’ parents by exome capture sequencing, generating up to 141,030 polymorphic markers with available genomic-location and annotation information. We evaluated prediction procedures that varied not only by learning schemes and prediction models, but also by the way the data were preprocessed to account for redundancy in marker information. More complex genomic prediction procedures were generally not significantly more accurate than the simplest procedure, likely due to limited population sizes. Nevertheless, a highly significant gain in prediction accuracy was achieved by transforming the marker data through a marker correlation matrix. Our results suggest that marker-data transformations and, more generally, the account of linkage disequilibrium among markers, offer valuable opportunities for improving prediction procedures in GS. Some of the achieved prediction accuracies should motivate implementation of GS in switchgrass breeding programs.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/linkage-disequilibrium" class="hw-term hw-article-keyword hw-article-keyword-linkage-disequilibrium" rel="nofollow">linkage disequilibrium</a></li><li class="kwd"><a href="/keyword/exome-capture" class="hw-term hw-article-keyword hw-article-keyword-exome-capture" rel="nofollow">exome capture</a></li><li class="kwd"><a href="/keyword/bioenergy" class="hw-term hw-article-keyword hw-article-keyword-bioenergy" rel="nofollow">bioenergy</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/panicum-virgatum-l" class="hw-term hw-article-keyword hw-article-keyword-panicum-virgatum-l" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Panicum virgatum</em> L</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resource" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resource" rel="nofollow">Shared data resource</a></li></ul><p id="p-3">Genomic selection (GS) is the use of genome-wide marker information to predict genotype performance in breeding programs. Predictions in GS should be sufficiently accurate so that it is economically more viable to select individuals based solely on markers, rather than phenotypic measurements (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Muir 2007</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lorenz <em>et al.</em> 2011</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Riedelsheimer and Melchinger 2013</a>). In switchgrass (<em>Panicum virgatum</em> L.), a perennial forage crop, this potential advantage derives from the fact that phenotypic measurements generally require 2–3 yr of field testing (one establishment year and 1–2 yr of trial; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Casler and Brummer 2008</a>), whereas acquiring genome-wide marker information would require less than a year (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Resende <em>et al.</em> 2014</a>). The US Department of Agriculture and the US Department of Energy intend to make switchgrass a principal source of biofuel in the US by 2030, so as to meet the goal of displacing 30% of petroleum use with biofuel (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Sanderson <em>et al.</em> 1996</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Perlack <em>et al.</em> 2005</a>). However, strong and rapid genetic gains for biomass yield to approximately 20 Mg/ha are required to meet this goal (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Perlack <em>et al.</em> 2005</a>). Therefore, in the case of switchgrass breeding, GS is a technology that is not only economically attractive, but also strategically useful.</p><p id="p-4">The first genomic prediction procedure, introduced by <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Meuwissen <em>et al.</em> (2001)</a>—ridge regression-BLUP (RR-BLUP), equivalent to genomic BLUP (GBLUP) (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hayes <em>et al.</em> 2009</a>)—assumed an infinitesimal genetic model (<em>i.e.</em>, all loci in linkage equilibrium with additive effects of equal variance), and a homogeneous genetic architecture throughout individuals. Since then, improvements in genomic prediction models have been made, most notably through the account of (i) marker-effect heteroscedasticity, <em>i.e.</em>, differential variances of marker effects (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Park and Casella 2008</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Erbe <em>et al.</em> 2012</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Shen <em>et al.</em> 2013</a>); (ii) nonlinear marker effects and interactions between markers (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gianola and van Kaam 2008</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akdemir and Jannink 2015</a>); (iii) genotype-by-environment interactions (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Burgueño <em>et al.</em> 2012</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Guo <em>et al.</em> 2013</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Heslot <em>et al.</em> 2014</a>); (iv) correlation among traits of interest (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Calus and Veerkamp 2011</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Jia and Jannink 2012</a>); and (v) population heterogeneity (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Harris and Johnson 2010</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Makgahlela <em>et al.</em> 2013</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Karoui <em>et al.</em> 2012</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Isidro <em>et al.</em> 2015</a>).</p><p id="p-5">Marker-data transformations have previously been used in GS to account for marker-effect heteroscedasticity, with weights on marker variables reflecting the significance of their relationship with the outcome of interest (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">de los Campos <em>et al.</em> 2013</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Su <em>et al.</em> 2014</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">He <em>et al.</em> 2015</a>). Transformations on marker data have also been used to account for redundancy in marker information due to linkage disequilibrium (LD), through principal component analysis (PCA) (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Long <em>et al.</em> 2011</a>); or weights on marker variables; so as to reflect the degree of tagging of loci by markers (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Speed <em>et al.</em> 2012</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Nishio and Satoh 2015</a>). However, to date, few empirical studies in GS have been conducted to assess the usefulness of preprocessing marker data in order to account for LD.</p><p id="p-6">In this study, we assessed the possibility of producing reliable predictions for GS in switchgrass. Our data consisted of relatively few observations on two distinct populations. A total of 247 individuals were genotyped by exome capture sequencing, and evaluated for important agronomic traits: biomass yield, plant height, and heading date. We assessed various types of prediction procedures, which differed from the standard procedure (GBLUP on nontransformed marker data) not only by the prediction model—which might account for heteroscedasticity and/or nonlinearity of marker effects—but also by the type of marker-data transformation—which might account for LD among markers. We also examined the potential of learning schemes (training-set designs) for improving genomic prediction procedures. While multiple-trait models may be useful to account for genotype-by-environment interactions, correlation among traits and/or population heterogeneity, such models were not found useful here because they either were not statistically efficient enough, or failed to effectively fit the data, certainly as a result of our small sample sizes.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Populations assayed</h3><p id="p-7">Genomic selection (GS) in switchgrass was studied in two tetraploid populations. The first population comprised 137 half-sib (HS) families developed from WS4U, an upland-ecotype germplasm pool of 162 plants (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Casler <em>et al.</em> 2006</a>). The HS families were the progeny of genotypes produced in cycle 2 (C2) of selection on WS4U for high biomass yield (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Casler 2010</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Casler and Vogel 2014</a>). The second population comprised 110 HS families developed from the cultivar Liberty, which is a stabilized lowland-upland hybrid cultivar (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Vogel <em>et al.</em> 2014</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Casler and Vogel 2014</a>). The HS families were developed from the Liberty population by an additional breeding generation. They were the progeny of the genotypes selected in C2 for high biomass yield, excellent winter survival, excellent spring greenup, and no apparent diseases using the among and within family breeding method. In both populations, the selected C2 genotypes were polycrossed in isolation. The two populations, hereafter referred to as WS4U-C2 and Liberty-C2, were tested in two locations in the United States: Arlington (WI) and Mead (NE) in 2012, 2013, and 2014. Families were assayed in a row-plot trial and replicated in a randomized complete block design (RCBD), with four family replicates for WS4U-C2, and three family replicates for Liberty-C2. There were up to five HS in each family replicate, with different HS between replicates (there was no vegetative propagation of individual plants). Rows were spaced 0.9 m apart, and plants were spaced 0.45 m apart within rows. Plots were established from greenhouse-grown seedlings in May 2011, and fertilized with 110 kg N ha<sup>–1</sup> in May of 2012 and 2013.</p></div><div id="sec-3" class="subsection"><h3>Phenotypic data and mixed-model analyses</h3><p id="p-8">In this study, trait measurements at different locations were considered different outcomes. There were six outcomes: dry matter yield (DMY), plant height (PH), and heading date (HD) in WI or NE. HD was scored on each individual plant as day-of-year when half of the panicles of a plant had fully emerged from the boot. PH was measured on each individual plant from the ground to the top of the tallest tiller after growth had ceased in late September. DMY was determined by harvesting each row plot with a flail harvester at a 10-cm cutting height, adjusted for moisture concentration using a 400-g sample of harvested tissue dried for 7 d at 60°. Outcomes were measured in 2012 and 2013, with the exception of HD in WI, which was scored in 2013 and 2014 for WS4U-C2 and in 2012, 2013, and 2014 for Liberty-C2 (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468090/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468090/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Description of trait measurements for WS4U-C2 and Liberty-C2 in WI and NE</span><div class="sb-div caption-clear"></div></div></div><p id="p-10">For PH and HD outcomes, measured on an individual-plant basis, the following linear mixed model was fitted:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-1.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-2.gif"/></noscript></span></span> is the grand mean; <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-3.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-4.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-5.gif"/></noscript></span></span> are the random effects of HS family <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-6.gif"/></noscript></span></span>, block <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-7.gif"/></noscript></span></span>, and year <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-8.gif"/></noscript></span></span>, respectively; <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-9.gif"/></noscript></span></span> indicates interactions; <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-10.gif"/></noscript></span></span> are residuals. For each term, the corresponding effects were modeled as independent and identically normally distributed. For PH (in both locations), an additional term <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-11.gif"/></noscript></span></span> was included in the model (on the basis of a lower Bayesian information criterion), where <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-12.gif"/></noscript></span></span> is the Kronecker product of the first-order autoregressive covariance matrices on rows and on columns, respectively.</p><p id="p-11">For DMY outcomes, measured on a plot basis, the following linear mixed model was fitted:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-13.gif"/></noscript></span></span>where effects are as described above, except for <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-14.gif"/></noscript></span></span>, which is the pooled error of plot <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-15.gif"/></noscript></span></span> in year <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-16.gif"/></noscript></span></span>. The linear mixed models described here were all fitted using ASREML-R (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Butler <em>et al.</em> 2007</a>).</p><p id="p-12">The predicted HS-family effects are best linear unbiased predictions (BLUPs) of the transmitting abilities of maternal parents. On the one hand, BLUPs have the property of being shrunk toward their mean (zero) differentially, depending on the relative amount of information available for their computation. As a result, estimates of marker effects based on BLUPs tend to be distorted compared to the estimates based directly on phenotypes (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Garrick <em>et al.</em> 2009</a>), which can be problematic in inferential studies such as QTL analyses, especially if reliabilities of BLUPs are highly variable among genotypes. So, in quantitative genetic analyses, it has been recommended to deregress BLUPs for subsequent use in weighted regression models, accounting for differential levels of uncertainty in the deregressed-BLUP estimates, rather than using nonweighted regression models on BLUPs directly. On the other hand, BLUPs are generally more accurate estimates of the true values, and the approach based on BLUPs does not rely on (possibly suboptimal) weights in regression. Importantly, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Guo <em>et al.</em> (2010)</a> showed, in simulation studies, that GS models based on BLUPs predicted true performance of genotypes as, or more, accurately than weighted GS models based on “daughter yield deviations”, equivalent to deregressed BLUPs (even when strong differences in available information were simulated), which suggests that BLUPs are acceptable alternatives to their deregressed counterpart as response variables in predictive studies. Consequently, here we chose not to deregress HS-family BLUPs, and use them directly as response variables for training and validating GS models. Nonetheless, all methods used in this study can be adapted to accommodate deregressed BLUPs with differential weights on observations, if needed.</p><p id="p-13">The raw phenotypic data and the matrix of HS-family BLUPs are available online as Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024950/-/DC1/FileS1.xls">File S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024950/-/DC1/FileS2.xls">File S2</a>, respectively, and from <a href="http://dfrc.wisc.edu/sniper/">http://dfrc.wisc.edu/sniper/</a>.</p></div><div id="sec-4" class="subsection"><h3>Marker data and quality control</h3><p id="p-14">Exome capture sequencing of HS-family maternal parents was performed using the Roche-Nimblegen protocol for preparation of SeqCap EZ Developer libraries using the Roche-Nimblegen probeset ‘120911_Switchrass_GLBRC_R_EZ_HX1’ as described previously (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Evans <em>et al.</em> 2014</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">2015</a>). Capture was performed on the 247 individuals from WS4U-C2 and Liberty-C2, and sequencing was performed on the Illumina HiSequation 2000 platform, generating 150-nt paired-end reads. Initial quality control was performed using FastQC (v0.10.0; <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>). PCR primers, adapter sequences, and bases with quality scores below 20 were trimmed using Cutadapt (v1.1; <a href="https://code.google.com/p/cutadapt/">https://code.google.com/p/cutadapt/</a>). Reads with lengths shorter than 35 nt were discarded. Cleaned reads were aligned to the hardmasked <em>P. virgatum</em> v1.1 reference genome (<a href="http://phytozome.jgi.doe.gov/pz/portal.html#!info?alias=Org_Pvirgatum">http://phytozome.jgi.doe.gov/pz/portal.html#!info?alias=Org_Pvirgatum</a>) using BowTie v0.12.7 (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Langmead <em>et al</em>. 2009</a>). Unanchored contigs were assigned to scaffolds (ChrUn1-ChrUn15) for more efficient alignment (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Evans <em>et al.</em> 2015</a>). Unique alignments were required, and only a single mismatched nucleotide was permitted in the first 35 bases of the read. Read alignments meeting the alignment criteria were processed using the index, sort, merge (default parameters), and mpileup (-BD –C 0 options) functions of the SAMTools package v0.1.18 (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Li <em>et al.</em> 2009</a>). Counts of reads corresponding to reference and alternate alleles were generated for each individual in the WS4U-C2 and Liberty-C2 populations at a subset of sites (2,179,164 loci; HapMap v2) previously determined to be polymorphic using exome capture sequencing data from two diversity panels, the Northern Switchgrass Panel (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Evans <em>et al.</em> 2015</a>) and a southern switchgrass panel (C. Brummer, unpublished data). Then, marker genotypes were called, assuming disomic inheritance of tetraploid switchgrass based on previous genetic-mapping studies (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Okada <em>et al.</em> 2010</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Li <em>et al.</em> 2014</a>). To infer marker genotypes while effectively accounting for genotype-calling uncertainty, read counts were converted to expected allelic dosages (values between 0 and 2 for the number of copies of the alternate allele); using the algorithm of <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Martin <em>et al.</em> (2010)</a>, fitted on each population separately. The algorithm of <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Martin <em>et al.</em> (2010)</a> estimates the sequencing/alignment error rate and the population allele frequency for each marker separately, using an Expectation-Maximization (EM) algorithm. Then, for each combination of marker and individual, the posterior probability of each allelic dosage given the read-count data is obtained by Bayes’ rule, assuming Hardy-Weinberg equilibrium (HWE) to derive the prior probability of each allelic dosage, and a binomial distribution of read type (reference/alternate) to derive the likelihood of each allelic dosage. Expected allelic dosages were computed as the sum of possible allelic dosages weighted by their posterior probability for each combination of individual and marker.</p><p id="p-15">In the resulting matrix of expected allelic dosages, marker variables were then filtered for (i) proportion of missing values (strictly lower than 5%); (ii) polymorphism (minor allele frequency across populations strictly greater than <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-17.gif"/></noscript></span></span>, and variance higher than <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-18.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-19.gif"/></noscript></span></span> the total number of genotypes across populations); (iii) HWE within each population (p-value for HWE, based on a χ<sup>2</sup>-test, strictly higher than 10<sup>−4</sup> for each population considered individually); and (iv) availability of genomic-location information (available information on chromosome and position from the reference genome sequence, and annotation of <em>P. virgatum</em> v1.1; DOE-JGI, <a href="http://phytozome.jgi.doe.gov/">http://phytozome.jgi.doe.gov/</a>). The resulting matrix <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-20.gif"/></noscript></span></span> contained expected allelic dosages at <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-21.gif"/></noscript></span></span> selected markers across populations, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-22.gif"/></noscript></span></span> in WS4U-C2 only and <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-23.gif"/></noscript></span></span> in Liberty-C2 only.</p><p id="p-16">To characterize LD between markers, we used the correlation matrix <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-24.gif"/></noscript></span></span> consisting of Pearson correlation coefficients between allelic dosages: <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-25.gif"/></noscript></span></span>, where indexing on <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-26.gif"/></noscript></span></span> refers to columns. For matters of efficiency, the <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-27.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-28.gif"/></noscript></span></span> matrix was made block-diagonal, with blocks corresponding to chromosomes (i.e., only local LD was accounted for, through <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-29.gif"/></noscript></span></span>). This assumed sparsity in marker correlations allowed us to compute <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-30.gif"/></noscript></span></span> with reasonable costs in time and memory, while potentially reducing noise in the estimations by assuming zero correlation between markers from different chromosomes. Positive-definiteness of <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-31.gif"/></noscript></span></span> (which implies invertibility) was ensured using the modified projection algorithm of <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Higham (2002)</a> from the nearPD function of the R package Matrix.</p><p id="p-17">Following <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Speed <em>et al.</em> (2012)</a>, for any marker <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-32.gif"/></noscript></span></span> we define the degree of (local) tagging as the sum of squared correlations involving <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-33.gif"/></noscript></span></span> and any other marker within the same chromosome, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-34.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-35.gif"/></noscript></span></span> indexing markers on the same chromosome as <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-36.gif"/></noscript></span></span>’s. This metric is supposed to depict the redundancy in information at <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-37.gif"/></noscript></span></span>, as reflected by <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-38.gif"/></noscript></span></span>.</p><p id="p-18">The matrix <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-39.gif"/></noscript></span></span> for markers selected across both populations is available online as a supplementary file (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024950/-/DC1/FileS3.rar">File S3</a>, with values rounded to the fifth decimal digit) and from <a href="http://dfrc.wisc.edu/sniper/">http://dfrc.wisc.edu/sniper/</a> (with nonrounded values).</p></div><div id="sec-5" class="subsection"><h3>Prediction procedures</h3><p id="p-19">For each possible combination of population and outcome, we evaluated prediction procedures with respect to four components: (i) population learning scheme—set of parent genotypes to include for training; (ii) environment learning scheme—set of locations to include for training; (iii) marker-data transformation—type of transformation on the marker data used to produce marker features; and (iv) prediction model—method used to generate predictions on the outcome based on marker features. In this study, emphasis was placed on the last two components.</p><div id="sec-6" class="subsection"><h4>Prediction models:</h4><p id="p-20">The standard statistical model for prediction was genomic BLUP (GBLUP; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Habier <em>et al.</em> 2007</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Hayes <em>et al.</em> 2009</a>). For a sample of <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-40.gif"/></noscript></span></span> instances and <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-41.gif"/></noscript></span></span> marker features, we define GBLUP as follows:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-42.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-43.gif"/></noscript></span></span> is the <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-44.gif"/></noscript></span></span>-vector of HS-family BLUPs; <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-45.gif"/></noscript></span></span> is the <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-46.gif"/></noscript></span></span>-vector of grand mean; <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-47.gif"/></noscript></span></span> is the <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-48.gif"/></noscript></span></span> design matrix attributing the <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-49.gif"/></noscript></span></span> observations to <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-50.gif"/></noscript></span></span> parent genotypes; <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-51.gif"/></noscript></span></span>), <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-52.gif"/></noscript></span></span> being the <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-53.gif"/></noscript></span></span> genomic relationship matrix derived from marker features as <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-54.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-55.gif"/></noscript></span></span> the <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-56.gif"/></noscript></span></span> matrix of marker features; <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-57.gif"/></noscript></span></span>), with <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-58.gif"/></noscript></span></span> the identity matrix. As explained in the next subsection, the marker features in <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-59.gif"/></noscript></span></span> were not the expected allelic dosages, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-60.gif"/></noscript></span></span>. The normalizing factor in <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-61.gif"/></noscript></span></span> was the sum of sample variances over marker features.</p><p id="p-21">The GBLUP model is equivalent to the RR-BLUP model, where the assumptions of an infinitesimal genetic model are made: effects of marker features are assumed to be additive, linear, homoscedastic (<em>i.e.</em>, having equal variance), and independent (which implies no LD between markers). To accommodate genetic architectures that strongly deviate from the infinitesimal model, we considered eight additional models that were heteroscedastic and/or nonlinear.</p><p id="p-22">Heteroscedastic models were GBLUP-wG, GBLUP-sG, BayesA, and BayesB. The GBLUP-wG model, first used by de <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">los Campos <em>et al.</em> (2013)</a>, consisted in weighting marker features by <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-62.gif"/></noscript></span></span>, where <em>p</em> is the p-value for the effect of a marker feature on the outcome of interest. GBLUP-sG is a variation from GBLUP-wG, where are included in the model only the marker features with a false discovery rate (FDR) for their correlation with the outcome lower than some threshold, determined by tuning; the FDR was calculated using the qvalue package in R (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Storey and Tibshirani 2003</a>). BayesA and BayesB are Bayesian linear regression models, introduced by <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Meuwissen <em>et al.</em> (2001)</a>, which have the following specification:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-63.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-64.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-65.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-66.gif"/></noscript></span></span> are as described above; <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-67.gif"/></noscript></span></span>). In BayesA, <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-68.gif"/></noscript></span></span>. In BayesB, <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-69.gif"/></noscript></span></span> with probability <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-70.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-71.gif"/></noscript></span></span> with probability <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-72.gif"/></noscript></span></span>; <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-73.gif"/></noscript></span></span> was chosen to follow a Beta(0.2, 1.8) in order to reflect relatively sparse distributions of causal variants across the genome while allowing uncertainty about <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-74.gif"/></noscript></span></span>. In both BayesA and BayesB, <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-75.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-76.gif"/></noscript></span></span>), with <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-77.gif"/></noscript></span></span>. The hyperparameters <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-78.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-79.gif"/></noscript></span></span>, s_b<sub></sub>, <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-80.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-81.gif"/></noscript></span></span> were set through the heuristics described in <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Pérez and de los Campos (2014</a>), based on a prior estimation of the proportion of variance explained by the model, which was here chosen to be <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-82.gif"/></noscript></span></span> from a GBLUP model with an update on marker effects from the heteroscedastic effects model (HEM) of <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Shen <em>et al.</em> (2013)</a>. BayesA and BayesB were fitted by a Gibbs sampling algorithm with 5000 burn-in iterations, then 15,000 iterations for actual sampling of parameter values.</p><p id="p-23">The one nonlinear model that we assayed was the reproducing kernel Hilbert space (RKHS) model described by <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Gianola and van Kaam (2008)</a>. In the implementation recommended by these authors, the RKHS model is made equivalent to the GBLUP model, where pairwise relationship coefficients in <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-83.gif"/></noscript></span></span> are replaced by an appropriate nonlinear function of pairwise distances. The pairwise distances were Euclidean distances based on marker features, scaled by the maximum distance over pairs of individuals; the nonlinear function was the Gaussian kernel, with its scale parameter determined by tuning.</p><p id="p-24">To account for both heteroscedasticity and nonlinearity, we extended the RKHS model to RKHS-wG and RKHS-sG, where marker features were weighted, as described above for GBLUP-wG and GBLUP-sG. One last heteroscedastic and nonlinear model that we considered was Random Forest (RF), which is a machine-learning method that combines results from several regression (or classification) trees, fitted to different variations of the data—bootstrap samples of instances and random subsets of features (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Breiman 2001</a>). The RF model was fitted with 200 trees, bootstrap samples of size <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-84.gif"/></noscript></span></span>, and subsets of <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-85.gif"/></noscript></span></span> features.</p><p id="p-25">Tuning for the scale parameter in RKHS (and its extensions -wG and -sG), and the FDR threshold in GBLUP-sG or RKHS-sG, was performed through minimization of the generalized-cross-validation criterion (GCV; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Golub <em>et al.</em> 1979</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gu and Ma 2005</a>) over a grid of values (strictly greater than 0 and lower than 1, with steps of 0.025 for the scale parameter in RKHS, and steps of 0.05 for the FDR threshold in GBLUP/RKHS-sG). The GCV criterion approximates the leave-one-out cross-validation mean squared error, based on one model fit to the whole training set; it is defined as:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-86.gif"/></noscript></span> , </span>where <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-87.gif"/></noscript></span></span> refers to the trace (the sum of diagonal elements of a matrix); <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-88.gif"/></noscript></span></span>, the linear prediction of <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-89.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-90.gif"/></noscript></span></span>, the “hat” (smoothing) matrix, such that <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-91.gif"/></noscript></span></span>, depend on the parameter under tuning.</p><p id="p-26">The GBLUP and RKHS models, as well as their extensions -wG and -sG, were fitted using the R package rrBLUP (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Endelman 2011</a>); the BayesA and BayesB models were fitted using the R package BGLR (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Pérez and de los Campos 2014</a>); the HEM of <a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Shen <em>et al.</em> (2013)</a> was fitted using the R package bigRR (<a id="xref-ref-59-4" class="xref-bibr" href="#ref-59">Shen <em>et al.</em> 2013</a>), and the RF model was fitted using the R package randomForest (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Liaw and Wiener 2002</a>).</p></div><div id="sec-7" class="subsection"><h4>Marker-data transformations:</h4><p id="p-27">As mentioned above, the input <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-92.gif"/></noscript></span></span> to prediction models were transformations of the marker-data matrix <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-93.gif"/></noscript></span></span>. For a given set of individuals, consisting of either WS4U-C2, Liberty-C2, or both populations combined, the following transformations of <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-94.gif"/></noscript></span></span> were made: (i) <em>Base</em>, where features are centered allelic dosages, and correspond to the typical input to GS models: <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-95.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-96.gif"/></noscript></span></span> the <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-97.gif"/></noscript></span></span> matrix with uniform columns containing the mean allelic dosages within the population, <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-98.gif"/></noscript></span></span>; (ii) <em>PCA</em>, where features are uncorrelated synthetic variables each contributed differentially by marker variables: <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-99.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-100.gif"/></noscript></span></span> the <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-101.gif"/></noscript></span></span> rotation matrix in the singular value decomposition of <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-102.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-103.gif"/></noscript></span></span>; <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-104.gif"/></noscript></span></span> is the number of principal components here equal to <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-105.gif"/></noscript></span></span> for all sets of individuals), <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-106.gif"/></noscript></span></span>; (iii) <em>Cor</em>, where features are marker variables scaled through a correlation matrix: <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-107.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-108.gif"/></noscript></span></span> the <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-109.gif"/></noscript></span></span> lower-triangular matrix from the Cholesky decomposition of <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-110.gif"/></noscript></span></span>, such that <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-111.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-112.gif"/></noscript></span></span> being the matrix of correlation between marker variables as previously described, <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-113.gif"/></noscript></span></span>; (iv) <em>LD</em>, where features are marker variables weighted based on their relative degree of tagging (the more redundant information at a given marker, the lower its weight): <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-114.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-115.gif"/></noscript></span></span> the diagonal matrix of weights supposed to adjust for redundancy in marker information due to LD; <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-116.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-117.gif"/></noscript></span></span> was the least-absolute-error solution to <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-118.gif"/></noscript></span></span> subject to <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-119.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-120.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-121.gif"/></noscript></span></span> the matrix of squared correlation between marker variables and <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-122.gif"/></noscript></span></span> the <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-123.gif"/></noscript></span></span>-vector of one values, <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-124.gif"/></noscript></span></span> in WS4U-C2, and <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-125.gif"/></noscript></span></span> in Liberty-C2 (<span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-126.gif"/></noscript></span></span>, as a result of some weights being exactly zero). The publicly available LDAK software was used to calculate <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-127.gif"/></noscript></span></span> (<a href="http://dougspeed.com/ldak/">http://dougspeed.com/ldak/</a>; <a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Speed <em>et al.</em> 2012</a>).</p><p id="p-28">Marker-data transformations were chosen so that they have well-defined equivalencies in the GBLUP and RKHS models. In a GBLUP model, <em>PCA</em> is exactly equivalent to <em>Base</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024950/-/DC1/FigureS1.pdf">Figure S1</a>): <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-128.gif"/></noscript></span></span>, since <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-129.gif"/></noscript></span></span>, and<span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-130.gif"/></noscript></span></span> is the eigendecomposition of <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-131.gif"/></noscript></span></span>; <em>Cor</em> is equivalent to <em>Base</em> when local LD is accounted for through <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-132.gif"/></noscript></span></span>: <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-133.gif"/></noscript></span></span>; <em>LD</em> is equivalent to <em>Base</em> when local LD is accounted for by weights on marker features as in <a id="xref-ref-60-4" class="xref-bibr" href="#ref-60">Speed <em>et al.</em> (2012)</a>: <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-134.gif"/></noscript></span></span>. The equivalencies mentioned for cross-products in GBLUP also apply to Euclidean distances in RKHS. Interestingly, in a RR-BLUP model (equivalent to GBLUP), <em>Cor</em> and <em>LD</em> correspond to <em>Base</em> when marker-feature effects are assumed to follow a <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-135.gif"/></noscript></span></span>, and a <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-136.gif"/></noscript></span></span>, respectively, instead of a <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-137.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-138.gif"/></noscript></span></span> is the variance of marker effects). For the more complex models accounting for heteroscedasticity (BayesA, BayesB, GBLUP-wG, GBLUP-sG, RKHS-wG, RKHS-sG, and RF), the transformed marker variables were considered features in their own right, whose contribution to a given outcome of interest may be weighted similarly to features in <em>Base</em>.</p><p id="p-29">Throughout the article, prediction procedures are referred to by a combination of marker-data transformation and prediction model (<em>e.g.</em>, <em>Cor</em> – RKHS-sG) for a given learning scheme (involving the grouping of populations and environments in a training set; see next subsections).</p><p id="p-30">Matrices <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-139.gif"/></noscript></span></span> for WS4U-C2 and Liberty-C2 (with transformations <em>Base</em>, <em>PCA</em>, <em>Cor</em>, and <em>LD</em>), and the corresponding relationship matrices and Euclidean distance matrices, are available online from <a href="http://dfrc.wisc.edu/sniper/">http://dfrc.wisc.edu/sniper/</a>, in .rds format readable in R.</p></div><div id="sec-8" class="subsection"><h4>Population learning schemes:</h4><p id="p-31">Given a target population, we considered two types of training sets with regard to parent genotypes. The HS-family BLUPs used for training prediction models could be either from the target population only (within-population learning), or from both populations pooled together (across-population learning).</p></div><div id="sec-9" class="subsection"><h4>Environment learning schemes:</h4><p id="p-32">Given a target outcome, we considered two types of training sets with regard to observations at each parent genotype. For example, with DMY in WI as the target outcome, the HS-family BLUPs used for training the prediction model could be either those from the target location only (within-environment learning; <em>e.g.</em>, data on DMY in WI only), <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-140.gif"/></noscript></span></span>, or from both locations considered jointly for the same trait (across-environment learning; <em>e.g.</em>, data on DMY in WI and NE), <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-141.gif"/></noscript></span></span>. In across-environment learning, whenever leaving out HS families from the dataset for validation (see next section), the data on the same HS families in both environments were used for testing.</p></div></div><div id="sec-10" class="subsection"><h3>Validation of prediction procedures</h3><p id="p-33">Prediction procedures were evaluated using prediction accuracy estimated in five-fold cross-validation. Given a random partition of instances in five subsets of similar size, four subsets were used for training, and the remaining subset was used for testing. For each of the five subsets used sequentially for testing, prediction accuracy was computed as the Pearson coefficient of correlation between “observed” and predicted HS-family BLUPs. The significance of the difference in prediction accuracy between a given procedure and a standard procedure was assessed in replicated cross-validation by two-sided paired Dunnett tests, which are <em>t</em>-tests modified to account for multiple comparisons to a single control (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dunnett 1964</a>). For each outcome and population, the standard (control) procedure was chosen to be <em>Base</em> − GBLUP with within-population and within-environment learning. In cross-validation, the overlap between training sets results in lower variability among estimates of prediction accuracy, compared to the hypothetical case where training sets are generated independently. So, in paired Dunnett tests, the <em>t</em>-statistic <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-142.gif"/></noscript></span></span> was adjusted to account for correlation among computed prediction accuracies, as described in <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bouckaert and Frank (2004)</a>: <span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-143.gif"/></noscript></span>, </span>where <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-144.gif"/></noscript></span></span> is the number of “folds” in cross-validation, and <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-145.gif"/></noscript></span></span> is the number of replications (five-fold cross-validation was repeated 10 times); <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-146.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-147.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-148.gif"/></noscript></span></span> the <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-149.gif"/></noscript></span></span>-vectors of prediction accuracies from the test procedure and the standard procedure, respectively, and <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-150.gif"/></noscript></span></span> the Fisher transformation (for normality of prediction accuracies); <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-151.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-152.gif"/></noscript></span></span> are the mean and SD of <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-153.gif"/></noscript></span></span>, respectively. The R package nCDunnett was used to obtain adjusted p-values for <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-154.gif"/></noscript></span></span>.</p><p id="p-34">In order to limit the number of possible combinations to assess, prediction procedures were first optimized with respect to learning schemes only, based on replicated cross-validation, and then optimized with respect to marker-data transformation and prediction model in a two-step process (intended to reduce computational burden in the optimization): for each outcome and population, the combination of marker-data transformation and prediction model with the highest prediction accuracy based on nonreplicated cross-validation was selected; then the selected procedure was compared to the standard procedure (<em>Base</em> − GBLUP) in replicated cross-validation. If the selected procedure differed from <em>Base</em> − GBLUP by both an alternate marker-data transformation and an alternate prediction model (<em>e.g.</em>, <em>Cor</em> − BayesA), the alternate transformation was first compared to <em>Base</em> in a GBLUP model to assess the usefulness of transforming the marker data (<em>e.g.</em>, <em>Cor</em> − GBLUP <em>vs.</em> <em>Base</em> − GBLUP). Then the alternate model was compared to GBLUP using the alternate transformation for both models to assess the benefit from a more complex prediction model (<em>e.g.</em>, <em>Cor</em> − BayesA <em>vs.</em> <em>Cor</em> − GBLUP). Dunnett tests, with adjustments for the number of marker-data transformations or prediction models, were used in these comparisons in order to account for selection bias, <em>i.e.</em>, the fact that the same data were used for both choosing the selected procedure and then comparing it with <em>Base</em> − GBLUP.</p></div><div id="sec-11" class="subsection"><h3>Genetic analysis of phenotypic traits</h3><div id="sec-12" class="subsection"><h4>Genomic correlation:</h4><p id="p-35">To characterize the correlation between genomic effects (<span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-155.gif"/></noscript></span></span>) at different outcomes, a multivariate GBLUP model was fitted on any pair of outcomes as in <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Burgueño <em>et al.</em> (2012)</a>: errors <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-156.gif"/></noscript></span></span> were assumed independent within outcomes; the genomic relationship matrix was <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-157.gif"/></noscript></span></span>; the covariances by outcome of genomic effects and errors were estimated by REML. The R package ASREML-R was used to fit the multivariate GBLUP models.</p></div><div id="sec-13" class="subsection"><h4>Association mapping:</h4><p id="p-36">In order to assess the plausibility of heteroscedastic models, we conducted genome-wide association studies (GWAS) on all outcomes for both populations combined. For each marker in <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-158.gif"/></noscript></span></span> having a minor allele frequency (MAF) higher than 0.05, the EMMAX linear mixed model of <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kang <em>et al.</em> (2010)</a>, in which relatedness was accounted for through <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-159.gif"/></noscript></span></span>, was fitted using the R package rrBLUP. The threshold used to declare significance of associations was a FDR (as from <a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Storey and Tibshirani 2003</a>) lower than 0.05. Significant markers were then selected altogether in one linear mixed model, with relatedness accounted for through <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-160.gif"/></noscript></span></span> and fixed effects for markers, using a forward stepwise selection procedure based on the Bayesian information criterion. No covariate for population structure was included in the GWAS models, since the incentive for performing GWAS here was to investigate whether the ability of GBLUP to capture variation at outcomes could be significantly improved by including fixed marker effects. Even though between-population variability was adjusted for—because HS-family BLUPs were computed within each population separately—and genetic relatedness was captured through <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-161.gif"/></noscript></span></span> in GWAS models, it cannot be ruled out that significant markers actually reflected some population structure.</p></div><div id="sec-14" class="subsection"><h4>Partition of genomic heritability:</h4><p id="p-37">The genomic heritability is defined here as the proportion of variance explained by a GBLUP model, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-162.gif"/></noscript></span></span>. To assess the relative contribution of markers with different degrees of tagging to the genomic heritability, a multiple-component GBLUP model was fitted on each outcome, as in <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Yang <em>et al.</em> (2011)</a>, with three nonoverlapping marker classes: genomic relationship matrices <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-163.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-164.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-165.gif"/></noscript></span></span> were calculated from the distinct marker classes in <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-166.gif"/></noscript></span></span>, and the associated variances <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-167.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-168.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-169.gif"/></noscript></span></span> were estimated by REML. The contribution of class <strong><em>j</em></strong> to the genomic heritability was defined as <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-170.gif"/></noscript></span></span>. Marker classes were determined from tertiles on the markers’ degree of tagging. The R package ASREML-R was used to fit the multiple-component GBLUP models.</p></div></div><div id="sec-15" class="subsection"><h3>Data availability</h3><p id="p-38">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article.</p></div></div><div class="section results" id="sec-16"><h2 class="">Results</h2><div id="sec-17" class="subsection"><h3>Genomic structure and relatedness in populations</h3><p id="p-39">The two populations considered, WS4U-C2 and Liberty-C2, had strong differences in their patterns of decay in local (within-chromosome) LD. WS4U-C2 had a rapid decline in local LD, with the squared correlation between expected gametic phases (<em>r</em><sup>2</sup>) being essentially zero for physical distances between markers above 1 Mb (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). Conversely, in Liberty-C2, values of <em>r</em><sup>2</sup> decayed more slowly (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>), in accordance with the fact that effective population size in this population, derived from a cross between two cultivars, is certainly lower than in WS4U-C2, derived from a diverse collection of 162 upland-ecotype plants. The concordance in LD from WS4U-C2 to Liberty-C2 was very low, as reflected by the low adjusted <em>R</em><sup>2</sup> in a nonlinear regression for (y = r in Liberty-C2; x = r in WS4U-C2) on <em>r</em> in WS4U-C2. This low concordance seemed to be due to the rapid LD decay in WS4U-C2, with many values of <em>r</em> being close to zero in WS4U-C2 only (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>). However, when values of <em>r</em> departed from zero in both populations, there seemed to be some consistency in LD phase, as reflected by the sign of <em>r</em> being relatively consistent from one population to another.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/1049/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Patterns of decay in linkage disequilibrium (LD), represented by the squared correlation between expected gametic phases (r2) in (A) WS4U-C2 and (B) Liberty-C2; the blue curve corresponds to the mean value from a cubic-regression spline model assuming a Gamma distribution for r2. (C) Concordance, from WS4U-C2 to Liberty-C2, in LD as represented by the correlation between expected gametic phases (r); the blue curve corresponds to the mean value (and its 95%-confidence interval) from a cubic-regression spline model assuming a Normal distribution for r in Liberty-C2. Values of r were inferred as described in Weir (1979) using the R package SNPRelate (Zheng et al. 2012). Cubic-regression spline models were fitted using the R package mgcv (Wood 2006). The values of r and r2 shown here are based on random pairs of markers polymorphic in both WS4U-C2 and Liberty-C2, with each marker represented only once across all pairs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1845776361" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Patterns of decay in linkage disequilibrium (LD), represented by the squared correlation between expected gametic phases (&lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt;) in (A) WS4U-C2 and (B) Liberty-C2; the blue curve corresponds to the mean value from a cubic-regression spline model assuming a Gamma distribution for &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt;. (C) Concordance, from WS4U-C2 to Liberty-C2, in LD as represented by the correlation between expected gametic phases (&lt;em&gt;r&lt;/em&gt;); the blue curve corresponds to the mean value (and its 95%-confidence interval) from a cubic-regression spline model assuming a Normal distribution for &lt;em&gt;r&lt;/em&gt; in Liberty-C2. Values of &lt;em&gt;r&lt;/em&gt; were inferred as described in Weir (1979) using the R package SNPRelate (Zheng &lt;em&gt;et al.&lt;/em&gt; 2012). Cubic-regression spline models were fitted using the R package mgcv (Wood 2006). The values of &lt;em&gt;r&lt;/em&gt; and &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; shown here are based on random pairs of markers polymorphic in both WS4U-C2 and Liberty-C2, with each marker represented only once across all pairs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/1049/F1.medium.gif" width="440" height="144"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/4/1049/F1.medium.gif" width="440" height="144"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/1049/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/1049/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467990" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-40" class="first-child">Patterns of decay in linkage disequilibrium (LD), represented by the squared correlation between expected gametic phases (<em>r</em><sup>2</sup>) in (A) WS4U-C2 and (B) Liberty-C2; the blue curve corresponds to the mean value from a cubic-regression spline model assuming a Gamma distribution for <em>r</em><sup>2</sup>. (C) Concordance, from WS4U-C2 to Liberty-C2, in LD as represented by the correlation between expected gametic phases (<em>r</em>); the blue curve corresponds to the mean value (and its 95%-confidence interval) from a cubic-regression spline model assuming a Normal distribution for <em>r</em> in Liberty-C2. Values of <em>r</em> were inferred as described in <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Weir (1979)</a> using the R package SNPRelate (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Zheng <em>et al.</em> 2012</a>). Cubic-regression spline models were fitted using the R package mgcv (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Wood 2006</a>). The values of <em>r</em> and <em>r</em><sup>2</sup> shown here are based on random pairs of markers polymorphic in both WS4U-C2 and Liberty-C2, with each marker represented only once across all pairs.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">The distribution of minor allele frequency (MAF), over all 141,030 <sub></sub> markers selected, differed by population. In WS4U-C2, as MAF decreased, the cognate number of markers increased exponentially (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024950/-/DC1/FigureS2.pdf">Figure S2A</a>). In Liberty-C2, while a high proportion of markers appeared to be fixed, or singletons (likely due to <em>de novo</em> mutations, or possibly genotype miscalls), the remaining markers showed a relatively uniform distribution in their MAF (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024950/-/DC1/FigureS2.pdf">Figure S2B</a>). The concordance in MAF from WS4U-C2 to Liberty-C2 was low (adjusted <em>R</em><sup>2</sup> = 0.24) but significant (p &lt; 0.0001), with many markers being fixed in one population but not the other (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024950/-/DC1/FigureS2.pdf">Figure S2C</a>).</p><p id="p-42"><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2, A and B</a>, shows the distribution in the degree of tagging, reflecting the duplication of marker signals along a given chromosome, in each population. The relative variability and skewness in degree of tagging were larger in WS4U-C2 than in Liberty-C2 (respectively: coefficient of variation of 0.876 and 0.591; skewness of 3.99 and 1.31). In Liberty-C2, the relationship between MAF and degree of tagging was typical, with rare variants generally having a low degree of tagging (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2D</a>). Conversely, in WS4U-C2, degree of tagging was, on average, much higher for rare variants (MAF &lt; 0.10) than for more common variants (MAF &gt; 0.10) (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>). The high degree of tagging for rare variants in WS4U-C2 was due to extended local LD, observed in the LD-decay plots as lines of points for WS4U-C2 (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a>) but not for Liberty-C2 (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1B</a>). We hypothesize that such regions of extended LD in WS4U-C2 are due to the presence, in WS4U (the original collection of 162 plants), of haplotypes consisting of rare alleles, and limited recombination during the two cycles of selection leading to WS4U-C2.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/1049/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of degree of tagging in (A) WS4U-C2 and (B) Liberty-C2. Relationship between minor allele frequency (MAF) and degree of tagging in (C) WS4U-C2 and (D) Liberty-C2; the blue curve corresponds to the mean value (and its 95%-confidence interval) from a cubic-regression spline model assuming a Normal distribution for the degree of tagging. Cubic-regression spline models were fitted using the R package mgcv (Wood 2006)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1845776361" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of degree of tagging in (A) WS4U-C2 and (B) Liberty-C2. Relationship between minor allele frequency (MAF) and degree of tagging in (C) WS4U-C2 and (D) Liberty-C2; the blue curve corresponds to the mean value (and its 95%-confidence interval) from a cubic-regression spline model assuming a Normal distribution for the degree of tagging. Cubic-regression spline models were fitted using the R package mgcv (Wood 2006).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/1049/F2.medium.gif" width="439" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/4/1049/F2.medium.gif" width="439" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/1049/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/1049/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468065" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-43" class="first-child">Distribution of degree of tagging in (A) WS4U-C2 and (B) Liberty-C2. Relationship between minor allele frequency (MAF) and degree of tagging in (C) WS4U-C2 and (D) Liberty-C2; the blue curve corresponds to the mean value (and its 95%-confidence interval) from a cubic-regression spline model assuming a Normal distribution for the degree of tagging. Cubic-regression spline models were fitted using the R package mgcv (<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Wood 2006</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">Genetic relationship, defined here as twice the coefficient of identity by descent (IBD) between genotypes, was generally lower within WS4U-C2 than within Liberty-C2, in accordance with the presumably higher effective population size in WS4U-C2 (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Interestingly, the distribution in genetic relationships within Liberty-C2 was bimodal, with the first peak close to a HS relationship of 1/4, and the second peak close to a full-sib relationship of 1/2. This genetic structure within Liberty-C2 could have been generated by preferential mating of plants from the same ecotype (having similar flowering times), <em>i.e.</em>, assortative mating during crosses between the upland and lowland cultivars. Relationships across populations, inferred to be exactly zero, indicated strong genetic dissemblance between WS4U-C2 and Liberty-C2. This dissemblance, along with the low consistency in LD and MAF across populations, suggests little benefit from pooling populations into one single training set for genomic prediction, as was observed in previous studies (<em>e.g.</em>, <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Karoui <em>et al.</em> 2012</a>; <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">de los Campos <em>et al.</em> 2013</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/1049/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of genetic relationship coefficients based on identity by descent (IBD) within WS4U-C2, within Liberty-C2, and across populations. Coefficients of IBD were inferred from the EM algorithm described in Milligan (2003) using the R package SNPRelate (Zheng et al. 2012); IBD genetic relationship coefficients are equal to twice the IBD coefficients." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1845776361" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of genetic relationship coefficients based on identity by descent (IBD) within WS4U-C2, within Liberty-C2, and across populations. Coefficients of IBD were inferred from the EM algorithm described in Milligan (2003) using the R package SNPRelate (Zheng &lt;em&gt;et al.&lt;/em&gt; 2012); IBD genetic relationship coefficients are equal to twice the IBD coefficients.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/1049/F3.medium.gif" width="440" height="279"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/4/1049/F3.medium.gif" width="440" height="279"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/1049/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/1049/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467892" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-45" class="first-child">Distribution of genetic relationship coefficients based on identity by descent (IBD) within WS4U-C2, within Liberty-C2, and across populations. Coefficients of IBD were inferred from the EM algorithm described in <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Milligan (2003)</a> using the R package SNPRelate (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Zheng <em>et al.</em> 2012</a>); IBD genetic relationship coefficients are equal to twice the IBD coefficients.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>Genetic variability and architecture of phenotypic traits</h3><p id="p-46">The outcomes considered for genomic prediction, described in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, consisted of combinations of location and phenotypic trait, measured in 2 or 3 yr. Mean reliabilities (<em>i.e.</em>, the average inferred squared correlation between true HS-family effects <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/embed/mml-math-171.gif"/></noscript></span></span>’s and their BLUPs) were high (≥ 0.61) for PH and HD in both locations. Reliabilities for DMY were relatively high in NE (0.45 in WS4U-C2, and 0.53 in Liberty-C2), but low in WI (0.10 in WS4U-C2, and 0.21 in Liberty-C2).</p><p id="p-47">Genomic correlations between outcomes of the same trait in different environments (based on multivariate GBLUP models) were positive, strong (≥ 0.81), and significant (p ≤ 0.05), with PH and HD (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024950/-/DC1/FigureS3.pdf">Figure S3</a>), suggesting some benefit from pooling data from different locations into one single training set, for genomic prediction on these two traits. Conversely, genomic correlations between outcomes were not significant (p &gt; 0.05) with DMY, for both populations. Importantly, there were significant genomic correlations between different traits, such as between DMY-NE and PH-NE in WS4U-C2, and between DMY-NE and PH-WI or HD-NE in Liberty-C2. Unfortunately, due to the failure of multiple-trait models to effectively fit the data in cross-validation (likely due to our relatively small sample sizes), no multivariate prediction procedures were considered in this study.</p><p id="p-48">Association analyses were performed on each of the 12 outcomes, for both populations combined. With DMY in WI, seven markers were significantly associated to the outcome, and were selected together in one linear mixed model that explained 42% of the variance after accounting for genetic relatedness, as was reflected by a likelihood-ratio based <em>R</em><sup>2</sup> (<em>R</em><sup>2</sup><sub>LR</sub>) of 0.42 (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Sun <em>et al.</em> 2010</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024950/-/DC1/TableS1.pdf">Table S1</a>). The relatively high <em>R</em><sup>2</sup><sub>LR</sub> suggests that DMY in WI may be explained by few markers with strong effects; heteroscedastic models may then be beneficial in genomic prediction with this particular outcome. Conversely, with all other outcomes, no marker was deemed significant, and therefore the corresponding genetic architecture presumably consisted of rare and/or small-effect causal variants.</p><p id="p-49">For a given population, the relative contribution of different marker classes to the genomic variance at a given outcome was characterized through the partition of genomic heritability, from a multiple-component GBLUP model. The classes of markers were determined based on the degree of tagging. They consisted of markers with a degree of tagging below the 1/3-quantile, between the 1/3- and 2/3-quantiles, and above the 2/3-quantile, respectively. Quite remarkably, strongly tagged markers seemed to capture a large proportion of the genomic heritability in Liberty-C2 for all outcomes (≥ 85% of the total genomic heritability explained) except DMY in WI, for which the total genomic heritability was low, and the contribution from strongly tagged markers was estimated to be null (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). In WS4U-C2, the estimated contribution of strongly tagged markers to the total genomic heritability was moderate-to-large (≥ 54%) for DMY in WI, PH in WI, and HD in both locations, but low for PH in NE (11%), and null for DMY in NE (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). With HD in WI for WS4U-C2 and with DMY in WI for Liberty-C2, weakly tagged markers seemed to capture about half of the total genomic heritability, but, for all other outcomes across populations, weakly tagged markers seemed to capture very little of the genomic heritability.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/1049/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Partition of genomic heritability by marker class based on degree of tagging. &#x201C;Low&#x201D;: markers with a degree of tagging lower than the 1/3-quantile; &#x201C;Intermediate&#x201D;: markers with a degree of tagging between the 1/3- and 2/3-quantiles; &#x201C;High&#x201D;: markers with a degree of tagging higher than the 2/3-quantile. For a given outcome and a given population, the height of the bar corresponding to class j indicates its contribution to the total genomic heritability, i.e., , estimated in a multiple-component GBLUP model, with  as input, using ASREML-R (Butler et al. 2007)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1845776361" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Partition of genomic heritability by marker class based on degree of tagging. &#x201C;Low&#x201D;: markers with a degree of tagging lower than the 1/3-quantile; &#x201C;Intermediate&#x201D;: markers with a degree of tagging between the 1/3- and 2/3-quantiles; &#x201C;High&#x201D;: markers with a degree of tagging higher than the 2/3-quantile. For a given outcome and a given population, the height of the bar corresponding to class &lt;em&gt;j&lt;/em&gt; indicates its contribution to the total genomic heritability, &lt;em&gt;i.e.&lt;/em&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-172&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, estimated in a multiple-component GBLUP model, with &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-173&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; as input, using ASREML-R (Butler &lt;em&gt;et al.&lt;/em&gt; 2007).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/1049/F4.medium.gif" width="440" height="282"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/4/1049/F4.medium.gif" width="440" height="282"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/1049/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/1049/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467804" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-50" class="first-child">Partition of genomic heritability by marker class based on degree of tagging. “Low”: markers with a degree of tagging lower than the 1/3-quantile; “Intermediate”: markers with a degree of tagging between the 1/3- and 2/3-quantiles; “High”: markers with a degree of tagging higher than the 2/3-quantile. For a given outcome and a given population, the height of the bar corresponding to class <em>j</em> indicates its contribution to the total genomic heritability, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/F4/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/F4/embed/mml-math-172.gif"/></noscript></span></span>, estimated in a multiple-component GBLUP model, with <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/F4/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/F4/embed/mml-math-173.gif"/></noscript></span></span> as input, using ASREML-R (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Butler <em>et al.</em> 2007</a>).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>Selection and validation of prediction procedures</h3><p id="p-51">Here, four components of prediction procedures for GS, on only one outcome at a time for each population, were assayed: (i) population learning scheme—training set consisting of data on either the target population only, or both populations; (ii) environment learning scheme—training set consisting of data on either the target trait-location only, or the same trait in both locations; (iii) marker-data transformation—type of marker features used as input to prediction models; and (iv) prediction model—assumed relationship between the marker features and the outcome.</p><div id="sec-20" class="subsection"><h4>Learning schemes:</h4><p id="p-52">The importance of the first two components of prediction procedures (learning schemes) was assessed while using a GBLUP model with <em>Base</em> marker-data transformation (<em>Base</em> − GBLUP), <em>i.e.</em>, using centered expected allelic dosages as input to the standard GBLUP model (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> for DMY and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024950/-/DC1/TableS2.pdf">Table S2</a> for other traits). Across all outcomes, there was no consistent superiority of one particular learning scheme, based on mean prediction accuracies in five-fold cross-validation replicated 10 times. Also, none of the observed increases in prediction accuracy, compared to the within-population/within-environment scheme, were deemed significant (p &gt; 0.10). Pooling data across populations was not beneficial, as could be expected from the low concordance in LD patterns (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1C</a>) and MAF (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024950/-/DC1/FigureS2.pdf">Figure S2C</a>) between WS4U-C2 and Liberty-C2, as well as from the genetic dissemblance between the two populations (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). However, no significant increase in prediction accuracy was obtained with PH and HD from pooling data across environments in spite of the strong positive genomic correlations inferred for these two traits (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024950/-/DC1/FigureS3.pdf">Figure S3</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467830/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467830/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Mean prediction accuracy across population and environment learning schemes for DMY in WI and NE</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-21" class="subsection"><h4>Marker-data transformations and prediction models:</h4><p id="p-54">As described above, pooling data across populations and/or environments did not offer strong opportunities for improving prediction procedures. Thus, the importance of marker-data transformations and prediction models was investigated while using a simple within-population/within-environment learning scheme. Assessment of prediction procedures was performed in two steps. In step 1, a candidate prediction procedure (a combination of marker-data transformation and prediction model) was selected based on mean prediction accuracy in nonreplicated five-fold cross-validation (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> for DMY, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024950/-/DC1/TableS3.dfp">Table S3</a> for other traits, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024950/-/DC1/FigureS4.pdf">Figure S4</a>). In step 2, the selected candidate procedure was then compared to the standard—a GBLUP model on centered expected allelic dosages (<em>Base</em> − GBLUP)—in five-fold cross-validation replicated 10 times (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> for DMY, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024950/-/DC1/FigureS5.pdf">Figure S5</a> for other traits) so as to assess the benefit from the alternate marker-data transformation, and/or the more complex prediction model selected.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467884/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467884/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Mean prediction accuracy across marker-data transformations and prediction models for DMY in WI and NE</span><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/1049/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Validation of selected prediction procedures for DMY in WI and NE. Prediction accuracies () were estimated with a within-population/within-environment learning scheme in five-fold cross-validation, replicated 10 times. In each boxplot, up to two comparisons are made: (i) the candidate-transformation procedure (selected marker-data transformation according to nonreplicated five-fold cross-validation in a GBLUP model; Table 3) is compared to the standard procedure (Base &#x2212; GBLUP), if relevant; and (ii) the candidate procedure (selected prediction procedure according to nonreplicated five-fold cross-validation; Table 3) is compared to the candidate-transformation procedure. The significance of differences in prediction accuracies was assessed by two-sided paired Dunnett tests, which accounted for multiple testing of data transformations, in (i), and of prediction models, in (ii). The t-statistics in Dunnett tests were adjusted to account for correlation among training sets in cross-validation, as described in Bouckaert and Frank (2004)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1845776361" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Validation of selected prediction procedures for DMY in WI and NE. Prediction accuracies (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-174&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) were estimated with a within-population/within-environment learning scheme in five-fold cross-validation, replicated 10 times. In each boxplot, up to two comparisons are made: (i) the candidate-transformation procedure (selected marker-data transformation according to nonreplicated five-fold cross-validation in a GBLUP model; Table 3) is compared to the standard procedure (Base &#x2212; GBLUP), if relevant; and (ii) the candidate procedure (selected prediction procedure according to nonreplicated five-fold cross-validation; Table 3) is compared to the candidate-transformation procedure. The significance of differences in prediction accuracies was assessed by two-sided paired Dunnett tests, which accounted for multiple testing of data transformations, in (i), and of prediction models, in (ii). The &lt;em&gt;t&lt;/em&gt;-statistics in Dunnett tests were adjusted to account for correlation among training sets in cross-validation, as described in Bouckaert and Frank (2004).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/1049/F5.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/4/1049/F5.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/1049/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/1049/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468098" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-56" class="first-child">Validation of selected prediction procedures for DMY in WI and NE. Prediction accuracies (<span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/F5/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/1049/F5/embed/mml-math-174.gif"/></noscript></span></span>) were estimated with a within-population/within-environment learning scheme in five-fold cross-validation, replicated 10 times. In each boxplot, up to two comparisons are made: (i) the candidate-transformation procedure (selected marker-data transformation according to nonreplicated five-fold cross-validation in a GBLUP model; <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>) is compared to the standard procedure (Base − GBLUP), if relevant; and (ii) the candidate procedure (selected prediction procedure according to nonreplicated five-fold cross-validation; <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>) is compared to the candidate-transformation procedure. The significance of differences in prediction accuracies was assessed by two-sided paired Dunnett tests, which accounted for multiple testing of data transformations, in (i), and of prediction models, in (ii). The <em>t</em>-statistics in Dunnett tests were adjusted to account for correlation among training sets in cross-validation, as described in <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Bouckaert and Frank (2004)</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-57">Over cases (combinations of outcome and population), average prediction accuracies across prediction procedures in nonreplicated cross-validation ranged from –0.012 (DMY in WI for Liberty-C2) to 0.545 (HD in WI for Liberty-C2), with generally higher accuracies with PH and HD than with DMY (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024950/-/DC1/TableS3.dfp">Table S3</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024950/-/DC1/FigureS4.pdf">Figure S4</a>). However, high accuracies were achieved with DMY in NE for Liberty-C2 (0.455 on average). With DMY in WI (for both populations), DMY in NE (for WS4U-C2), and PH in NE (for WS4U-C2), marker-data transformation seemed to offer more opportunities for improvement than prediction models. In other cases, prediction accuracies were generally not sensitive to marker-data transformations, except for <em>PCA</em>, with which heteroscedastic models often performed poorly. In general, across cases, the <em>Base</em> marker-data transformation was selected in only three out of 12 cases, which again indicates some potential benefit from accounting for LD through marker-data transformation. Heteroscedastic models were selected nine times out of 12, with strong increases observed with DMY in WI, as could be expected from the many GWAS signals detected with that trait (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024950/-/DC1/TableS1.pdf">Table S1</a>).</p><p id="p-58">The statistical significance of the difference in prediction accuracies between <em>Base</em> − GBLUP and the candidate prediction procedure, selected based on nonreplicated cross-validation, was assessed in replicated cross-validation by paired Dunnett tests. Dunnett tests account for biases on significance due to multiple testing, which arose here from deriving a hypothesis from the same dataset as that used for testing (selection bias). Dunnett tests were further adjusted to account for the overlap between training sets in cross-validation, which caused the SD in prediction accuracy among “folds” to be an underestimate of the general SD (among hypothetical datasets). With DMY and PH in NE for WS4U-C2, and with DMY (in both locations) for Liberty-C2, the selected prediction procedures yielded higher mean prediction accuracies in replicated cross-validation. However, the observed differences were generally not deemed significant according to our tests, with the exception of DMY in NE for WS4U-C2 (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024950/-/DC1/FigureS5.pdf">Figure S5</a>). With DMY and PH in NE for WS4U-C2, the marker-data transformation apparently contributed more than the prediction model to the increase in prediction accuracy. Interestingly, for those cases, markers with a high degree of tagging tended to capture very little of the genomic heritability (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). The most notable case was DMY in NE for WS4U-C2, where a highly significant increase in accuracy (p &lt; 0.01) was observed when comparing <em>Base</em> − GBLUP to <em>Cor</em> − GBLUP, but using a BayesA model (the selected alternate model) rather than a GBLUP model (the standard model) did not yield a significant increase in prediction accuracy (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). This one strong increase indicates that, with our data, some substantial and reliable increase in prediction accuracy could be achieved through marker-data transformation, and not so much by prediction models more complex than GBLUP. Accordingly, when going through the same process of selection and validation of prediction procedures while considering only <em>Base</em> for input to the various prediction models (which is a traditional protocol for prediction-procedure optimization in GS), no significant increase in prediction accuracy could be obtained in any case (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024950/-/DC1/FigureS6.pdf">Figure S6</a>).</p></div></div></div><div class="section discussion" id="sec-22"><h2 class="">Discussion</h2><p id="p-59">This study dealt with the optimization of prediction procedures in GS through four components: population learning scheme, environment learning scheme, marker-data transformation, and prediction model, with emphasis placed on marker-data transformations and prediction models. The distinction between marker-data transformations and prediction models may seem arbitrary, especially for procedures like <em>Cor</em> − GBLUP or <em>Cor</em> − RKHS, which are equivalent to well-defined statistical models (see <em>Material and Methods</em>). However, “marker-data transformations” and “prediction models” account for different characteristics of marker loci, with marker-data transformations accounting for redundancy in marker information, and prediction models all assuming independence of marker-feature effects, and possibly accounting for heteroscedasticity and/or nonlinearity of marker-feature effects.</p><p id="p-60">In GS studies, the type of prediction models has been by far the factor upon which optimization of genomic prediction procedures has most often acted (<em>e.g.</em>, <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Moser <em>et al.</em> 2009</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Crossa <em>et al.</em> 2010</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Heslot <em>et al.</em> 2012</a>). In this study, we demonstrate that applying linear transformations to the marker data to account for local LD among markers may be useful to achieve gains in prediction accuracy that are reliable based on replicated cross-validation and honest significance tests. In fact, the only highly significant increase in prediction accuracy that we achieved was due to <em>Cor</em> (the account of local LD among marker loci through a block-diagonal correlation matrix), with DMY in NE for WS4U-C2 (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>). Some substantial increase in prediction accuracy could also be achieved by <em>Cor</em> with PH in NE for WS4U-C2, and <em>PCA</em> with DMY in WI for Liberty-C2, though the differences observed were not deemed significant (p = 0.37 and p = 0.96, respectively; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024950/-/DC1/FigureS5.pdf">Figure S5</a> and <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>). While optimizing prediction procedures through learning schemes has been useful in some studies (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Rincent <em>et al.</em> 2012</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Heslot <em>et al.</em> 2013</a>), here they did not offer strong opportunities for improving prediction procedures. With PH and HD, for which genomic correlations between environments were high (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024950/-/DC1/FigureS3.pdf">Figure S3</a>), the absence of consistent and significant increase in prediction accuracy from pooling environments may have been due to the high reliability of HS-family BLUPs for those two traits, causing additional measurements on one given genotype to contribute little to the quality of the signal in the data. Pooling data from different populations was not useful due to the strong genetic dissimilarities between WS4U-C2 and Liberty-C2 (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024950/-/DC1/FigureS2.pdf">Figure S2</a>, and <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>), certainly owing to the differences in effective population sizes, as well as to local adaptation of switchgrass populations and the ancient divergence between upland and lowland ecotypes (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Zhang <em>et al.</em> 2011</a>).</p><p id="p-61">In our study, both populations showed large variability in the degree of (local) tagging at marker loci (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2, A and B</a>), probably (to some extent) because markers were derived from exome capture, which target specific regions in the genome for sequencing. In WS4U-C2 specifically, there was also rapid LD decay along chromosomes with nevertheless extended LD (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1A</a>), presumably caused by sampling artifacts in WS4U, resulting in outstandingly high values of degree of tagging (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2A</a>). Also, variants with extreme degrees of tagging in that population tended to be rare (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2C</a>). Conversely, in Liberty-C2, LD decay within a chromosome was slower, and there was no extended LD (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1B</a>), resulting in relatively little skewness in the distribution of degree of tagging in that population (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2B</a>). Also, the relationship between MAF and degree of tagging was quite typical (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2D</a>), similar to that reported in human by <a id="xref-ref-60-5" class="xref-bibr" href="#ref-60">Speed <em>et al.</em> (2012)</a>. Probably as a consequence of the differences in genomic structure across populations, the contribution of strongly tagged variants to the genomic heritability of outcomes was generally lower in WS4U-C2 than in Liberty-C2. The only cases in which there were substantial increases in prediction accuracy from marker-data transformations and prediction models, relatively to <em>Base</em> − GBLUP, were DMY and PH in NE for WS4U-C2, and DMY in WI for Liberty-C2: +75% from <em>Cor</em>-BayesA, +20% from <em>Cor</em>-BayesB, and +75% from <em>PCA</em>-RF, respectively (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5</a>). Quite remarkably, these were also the only cases in which proportions of genomic heritability explained by strongly tagged variants were very low (0%, 11%, and 0% of genomic heritability explained, in these three cases, respectively) (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>). Importantly, previous simulation studies have investigated the effect of marker-data transformation <em>LD</em> on the accuracy of estimations (<a id="xref-ref-60-6" class="xref-bibr" href="#ref-60">Speed <em>et al.</em> 2012</a>) and predictions (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Nishio and Satoh 2015</a>) in genomic studies, and they have shown that the lower the relative degree of tagging at the causal variants for a given outcome, the more beneficial <em>LD</em> tends to be, compared to <em>Base</em>. Therefore, here we argue that cases where marker-data transformations might be useful are those where a small proportion of genomic heritability is captured by strongly tagged markers, which presumably derives from the fact that the causal variants are not strongly tagged. However, we are assuming here that <em>Cor</em> and <em>PCA</em> respond to features of genomic structure similarly to <em>LD</em>; simulation studies would be necessary to support such an assumption.</p><p id="p-62">In Liberty-C2, there seemed to be some population structure, presumably caused by assortative mating, while there was little evidence for such structure in WS4U-C2 (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). Population structure may cause marker loci from different chromosomes to be correlated. Therefore, in Liberty-C2 particularly, with marker-data transformations <em>Cor</em> and <em>LD</em>, it might have been useful to account for global LD, <em>i.e.</em>, correlations among marker loci within and across chromosomes. Interestingly, <em>PCA</em> does account for global LD in heteroscedastic models (in GBLUP and RKHS, the homoscedastic models, <em>PCA</em> is equivalent to <em>Base</em>), and this transformation was useful only in Liberty-C2, in one case (DMY in WI). However, despite being substantial, the increase in mean prediction accuracy in this case was not deemed significant in paired comparisons (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5</a>). Here, we did not account for global LD in <em>Cor</em> and <em>LD</em>, since the transformations would then have involved correlation matrices that are too large to process, or even store. Given the genomic features of Liberty-C2, pruning markers—which should not be too detrimental to prediction accuracy given the relatively large LD extent (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1B</a>)—and then accounting for global LD, might have proved useful with <em>Cor</em> and <em>LD</em>.</p><p id="p-63">In general, the account of LD through marker-data transformations based on correlation matrices (<em>Cor</em> and <em>LD</em>) could be further improved by reducing the level of noise in estimates of marker correlations. Preprocessing correlation matrices for shrinkage and/or sparsity could then prove beneficial, with Gaussian graphical models (<em>e.g.</em>, the graphical LASSO; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Friedman <em>et al.</em> 2008</a>) or generalized thresholding methods (<em>e.g.</em>, the MCP method; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Zhang 2010</a>) being potentially useful tools. Furthermore, basing correlations on expected gametic phases (<em>i.e.</em>, using haplotypic rather than genotypic correlations) would allow a more appropriate account of LD. However, in <em>Cor</em> (whenever preprocessing correlations or using expected gametic phases), one would then face the very serious computational challenge of ensuring that the resulting correlation matrix is positive definite (<em>i.e.</em>, being a proper and invertible correlation matrix). Filtering out markers based on relatively stringent MAF thresholds could result in higher prediction accuracies if the outcome is affected mostly by common causal variants, but it could also increase the benefit from marker-data transformations such as <em>Cor</em> or <em>LD</em>, because correlations would probably be less prone to error if only common markers are considered—some correlations involving rare variants, estimated at low but nonzero values, might simply be spurious rather than effectively due to LD, and would then contribute to overall noise in the marker correlation matrix. However, preselection based on MAF would remove potentially useful information whereas the linear mixed models assayed here (<em>i.e.</em>, all prediction models except RF) have the (supposedly desirable) property of downweighing the effect of markers with low variance (and therefore low MAF) as long as marker variables are not standardized (<em>i.e.</em>, scaled through a covariance matrix). Besides, optimizing prediction procedures with respect to a MAF threshold would result in more intricate studies: if optimization for MAF threshold were to be integrated into a prediction procedure (<em>i.e.</em>, tuning for MAF threshold within the procedure), the resulting computational complexity would greatly increase, since tuning would then have to be performed within each “fold” during cross-validation for each outcome and population; if different MAF thresholds were to be considered for different procedures (<em>i.e.</em>, the MAF threshold would be an additional component of prediction procedures), then the computational burden in the study would increase, but, more importantly, multiple testing would become a greater issue (accounting for the high number of candidates in comparisons of prediction procedures would cause significance tests to be highly conservative). In this study, we considered marker-data transformations that were relatively simple, but some of the treatments suggested above (regularization on correlation matrices, use of expected gametic phases for estimating correlations, and/or preselection based on MAF) may prove useful to better account for LD in GS.</p><p id="p-64">Our results are based on two populations of switchgrass with three traits. The conclusions that we drew about the relative importance of prediction-procedure components are, of course, not generalizable to all GS contexts and genetic architectures. The limited number of genotypes (137 and 110 in WS4U-C2 and Liberty-C2, respectively) likely favored the most parsimonious (statistically efficient) models, regardless of traits’ genetic architecture. With larger sample sizes, the higher flexibility of more complex models may have been more beneficial with traits whose genetic architecture substantially deviates from the infinitesimal model. Also, not all genetic architectures were represented in our data. In particular, there was little apparent benefit from RKHS (the only nonlinear homoscedastic model) on prediction accuracy. This may indicate that the traits considered in our data are influenced mostly by additive effects, but the small sample sizes arguably limit such conclusions. Finally, assessments of GS through cross-validations is limited in that it does not test prediction procedures for persistency of accuracy over generations, a criterion by which prediction models can differ significantly, as was shown through simulations by <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Habier <em>et al.</em> (2007)</a>. It would be important to formally study the repercussion of accounting for LD in GS on the persistence of accuracy over generations, using simulation and/or empirical studies. Quite interestingly, <a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Nishio and Satoh (2015)</a> suggested that accounting for LD through the <em>LD</em> marker-data transformation would be beneficial in long-term GS when causal variants are unevenly tagged, because it would prevent strongly tagged causal variants from being quickly fixed relatively to weakly tagged ones, and therefore would result in genomic predictions from <em>LD</em> − GBLUP having accuracies that are more slowly deteriorated over generations, compared to predictions from <em>Base</em> − GBLUP.</p><p id="p-65">We believe the relatively high prediction accuracies, particularly with DMY in NE, should motivate the implementation of GS breeding programs in switchgrass. Nevertheless, we may conduct future studies to compare GS with phenotypic selection for realized genetic gains, in programs that are run in similar conditions, so as to bring further evidence for the usefulness of GS technologies in perennial grass breeding.</p></div><div class="section" id="sec-23"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-66">The authors thank two anonymous reviewers for remarks and suggestions that greatly helped with improving the manuscript. We are grateful to Jeremy Schmutz of the Department of Energy Joint Genome Institute and Hudson Alpha for his work on the switchgrass genome, and to Nick Baker and Joseph Halinar, USDA-ARS, Madison, WI, and Steve Masterson, USDA-ARS, Lincoln, NE, for assistance with field operations and data collection. This research was funded in part by the following agencies and organizations: the US Department of Energy Great Lakes Bioenergy Research Center, Department of the Environment (DOE) Office of Science BER DE-FC02- 07ER64494 (laboratory operations, genotyping, and bioinformatics), the US Department of Energy Joint Genome Institute, supported by the Office of Science of the US Department of Energy under Contract No. DE-AC02-05CH11231 (sequencing), Agriculture and Food Research Initiative Competitive Grant No. 2011-68005-30411 from the Unites States Department of Agriculture (USDA) National Institute of Food and Agriculture (CenUSA; field operations and phenotypic measurements), USDA-ARS Congressionally allocated funds (field operations, technical support, and logistics), and the University of Wisconsin Agricultural Research Stations (field operations). Mention of commercial products and organizations in this manuscript is solely to provide specific information. The USDA is an equal opportunity provider and employer. G.P.R. was supported by the Gabelman-Shippo Wisconsin Distinguished Graduate Fellowship at the University of Wisconsin–Madison. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-67">Supplemental Material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024950/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024950/-/DC1</a></p></li><li class="fn" id="fn-5"><p id="p-68"><em>Communicating editor: E. Huang</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-11-17"><span class="received-label">Received </span>November 17, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-02-08"><span class="accepted-label">Accepted </span>February 8, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Ramstein <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.1" data-doi="10.1534/genetics.114.173658"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Locally epistatic genomic relationship matrices for genomic association and prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">199</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">857</span>–<span class="cit-lpage">871</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.173658%26rft_id%253Dinfo%253Apmid%252F25614606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk5LzMvODU3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi80LzEwNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.1049.2"><div class="cit-metadata unstructured">Bouckaert, R. R., and E. Frank, 2004 Evaluating the replicability of significance tests for comparing learning algorithms, pp. 3–12 in <em>Advances in knowledge discovery and data mining</em>, edited by H. Dai, R. Srikant, and C. Zhang. Springer, Heidelberg.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.3" data-doi="10.1023/A:1010933404324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breiman</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Random forests.</span> <abbr class="cit-jnl-abbrev">Machine learning</abbr> <span class="cit-vol">45</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">5</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMachine%2Blearning%26rft.volume%253D45%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1010933404324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1010933404324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000170489900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.4" data-doi="10.2135/cropsci2011.06.0299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of breeding values when modeling genotype× environment interaction using pedigree and dense molecular markers.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">707</span>–<span class="cit-lpage">719</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D707%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.1049.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-source">ASReml-R reference manual</span>, <span class="cit-publ-name">Queensland Department of Primary Industries and Fisheries</span>, <span class="cit-publ-loc">Brisbane</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.6" data-doi="10.1186/1297-9686-43-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accuracy of multi-trait genomic selection using different methods.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DForni%26rft.auinit1%253DS.%26rft.volume%253D43%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DDifferent%2Bgenomic%2Brelationship%2Bmatrices%2Bfor%2Bsingle-step%2Banalysis%2Busing%2Bphenotypic%252C%2Bpedigree%2Band%2Bgenomic%2Binformation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-1%26rft_id%253Dinfo%253Apmid%252F21208445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21208445&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.7" data-doi="10.1007/s12155-009-9071-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Changes in mean and genetic variance during two cycles of within-family selection in switchgrass.</span> <abbr class="cit-jnl-abbrev">BioEnergy Res.</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">47</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEnergy%2BRes.%26rft.volume%253D3%26rft.spage%253D47%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12155-009-9071-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12155-009-9071-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.8" data-doi="10.2135/cropsci2007.09.0499"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brummer</span>  <span class="cit-name-given-names">E. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Theoretical expected genetic gains for among-and-within-family selection methods in perennial forage crops.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">48</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">890</span>–<span class="cit-lpage">902</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DCasler%26rft.auinit1%253DM.%2BD.%26rft.volume%253D48%26rft.issue%253D3%26rft.spage%253D890%26rft.epage%253D902%26rft.atitle%253DTheoretical%2BExpected%2BGenetic%2BGains%2Bfor%2BAmong-and-Within-Family%2BSelection%2BMethods%2Bin%2BPerennial%2BForage%2BCrops%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2007.09.0499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2007.09.0499&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.9" data-doi="10.2135/cropsci2013.04.0239"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">K. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Selection for biomass yield in upland, lowland, and hybrid switchgrass.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">626</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D626%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.04.0239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.04.0239&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.10" data-doi="10.2135/cropsci2005.04-0020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beal</span>  <span class="cit-name-given-names">A. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Registration of WS4U and WS8U switchgrass germplasms.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">46</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">998</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DCasler%26rft.auinit1%253DM.D.%26rft.volume%253D46%26rft.issue%253D2%26rft.spage%253D998%26rft.epage%253D999%26rft.atitle%253DRegistration%2Bof%2BWS4U%2Band%2BWS8U%2BSwitchgrass%2BGermplasms%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2005.04-0020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2005.04-0020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.11" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueno</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNzEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi80LzEwNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.12" data-doi="10.1371/journal.pgen.1003608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klimentidis</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Prediction of complex human traits using the genomic best linear unbiased predictor.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">E1003608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253DE1003608%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003608%26rft_id%253Dinfo%253Apmid%252F23874214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003608&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23874214&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.13" data-doi="10.2307/2528490"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dunnett</span>  <span class="cit-name-given-names">C. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">New tables for multiple comparisons with a control.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">20</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">482</span>–<span class="cit-lpage">491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D20%26rft.spage%253D482%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2528490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2528490&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A19647271A00018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.14" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge regression and other kernels for genomic selection with R package rrBLUP.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.15" data-doi="10.3168/jds.2011-5019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Erbe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matukumalli</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goswami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Improving accuracy of genomic predictions within and between dairy cattle breeds with imputed high-density single nucleotide polymorphism panels.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">95</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">4114</span>–<span class="cit-lpage">4129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DErbe%26rft.auinit1%253DM.%26rft.volume%253D95%26rft.issue%253D7%26rft.spage%253D4114%26rft.epage%253D4129%26rft.atitle%253DImproving%2Baccuracy%2Bof%2Bgenomic%2Bpredictions%2Bwithin%2Band%2Bbetween%2Bdairy%2Bcattle%2Bbreeds%2Bwith%2Bimputed%2Bhigh-density%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bpanels.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2011-5019%26rft_id%253Dinfo%253Apmid%252F22720968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2011-5019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22720968&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305363100062&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.16" data-doi="10.1111/tpj.12601"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Childs</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaillancourt</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crisovan</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Nucleotide polymorphism and copy number variant detection using exome capture and next‐generation sequencing in the polyploid grass <em>Panicum virgatum</em>.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">79</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">993</span>–<span class="cit-lpage">1008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D79%26rft.spage%253D993%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.12601%26rft_id%253Dinfo%253Apmid%252F24947485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.12601&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24947485&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.17" data-doi="10.1111/tpj.13041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crisovan</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barry</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daum</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Diversity and population structure of northern switchgrass as revealed through exome capture sequencing.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">84</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">800</span>–<span class="cit-lpage">815</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D84%26rft.spage%253D800%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13041%26rft_id%253Dinfo%253Apmid%252F26426343%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26426343&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.18" data-doi="10.1093/biostatistics/kxm045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Sparse inverse covariance estimation with the graphical lasso.</span> <abbr class="cit-jnl-abbrev">Biostatistics</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">432</span>–<span class="cit-lpage">441</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiostatistics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiostatistics%252Fkxm045%26rft_id%253Dinfo%253Apmid%252F18079126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYmlvc3RzIjtzOjU6InJlc2lkIjtzOjc6IjkvMy80MzIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzQvMTA0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.19" data-doi="10.1186/1297-9686-41-55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Deregressing estimated breeding values and weighting information for genomic regression analyses.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DGarrick%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D41%26rft.spage%253D55%26rft.epage%253D55%26rft.atitle%253DDeregressing%2Bestimated%2Bbreeding%2Bvalues%2Band%2Bweighting%2Binformation%2Bfor%2Bgenomic%2Bregression%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-55%26rft_id%253Dinfo%253Apmid%252F20043827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-55&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20043827&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.20" data-doi="10.1534/genetics.107.084285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Kaam</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Reproducing kernel Hilbert spaces regression methods for genomic assisted prediction of quantitative traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">2289</span>–<span class="cit-lpage">2303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGianola%26rft.auinit1%253DD.%26rft.volume%253D178%26rft.issue%253D4%26rft.spage%253D2289%26rft.epage%253D2303%26rft.atitle%253DReproducing%2BKernel%2BHilbert%2BSpaces%2BRegression%2BMethods%2Bfor%2BGenomic%2BAssisted%2BPrediction%2Bof%2BQuantitative%2BTraits%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.084285%26rft_id%253Dinfo%253Apmid%252F18430950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC80LzIyODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzQvMTA0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.21" data-doi="10.2307/1268518"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Golub</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heath</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wahba</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Generalized cross-validation as a method for choosing a good ridge parameter.</span> <abbr class="cit-jnl-abbrev">Technometrics</abbr> <span class="cit-vol">21</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">215</span>–<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTechnometrics%26rft.volume%253D21%26rft.spage%253D215%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1268518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1268518&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1979GW16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Optimal smoothing in nonparametric mixed-effect models.</span> <abbr class="cit-jnl-abbrev">Annals of Statistics</abbr>: <span class="cit-fpage">1357</span>–<span class="cit-lpage">1379</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.23" data-doi="10.1111/j.1439-0388.2010.00878.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Comparison between genomic predictions using daughter yield deviation and conventional estimated breeding value as response variables.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">127</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">423</span>–<span class="cit-lpage">432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DGuo%26rft.auinit1%253DG.%26rft.volume%253D127%26rft.issue%253D6%26rft.spage%253D423%26rft.epage%253D432%26rft.atitle%253DComparison%2Bbetween%2Bgenomic%2Bpredictions%2Busing%2Bdaughter%2Byield%2Bdeviation%2Band%2Bconventional%2Bestimated%2Bbreeding%2Bvalue%2Bas%2Bresponse%2Bvariables.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2010.00878.x%26rft_id%253Dinfo%253Apmid%252F21077966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2010.00878.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21077966&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">D.M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basten</span>  <span class="cit-name-given-names">C.J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> Accuracy of across-environment genome-wide prediction in maize nested association mapping populations. G3 Bethesda 3: 263–272.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.25" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzQvMTA0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.26" data-doi="10.3168/jds.2009-2619"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic predictions for New Zealand dairy bulls and integration with national genetic evaluation.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">93</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1243</span>–<span class="cit-lpage">1252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DHarris%26rft.auinit1%253DB.%2BL.%26rft.volume%253D93%26rft.issue%253D3%26rft.spage%253D1243%26rft.epage%253D1252%26rft.atitle%253DGenomic%2Bpredictions%2Bfor%2BNew%2BZealand%2Bdairy%2Bbulls%2Band%2Bintegration%2Bwith%2Bnational%2Bgenetic%2Bevaluation.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2009-2619%26rft_id%253Dinfo%253Apmid%252F20172244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2009-2619&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20172244&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275056100042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.27" data-doi="10.1017/S0016672308009981"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Increased accuracy of artificial selection by using the realized relationship matrix.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">91</span>(<span class="cit-issue">01</span>): <span class="cit-fpage">47</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D91%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D60%26rft.atitle%253DIncreased%2Baccuracy%2Bof%2Bartificial%2Bselection%2Bby%2Busing%2Bthe%2Brealized%2Brelationship%2Bmatrix.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672308009981%26rft_id%253Dinfo%253Apmid%252F19220931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672308009981&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19220931&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264226300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parida</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Data-driven encoding for quantitative genetic trait prediction.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">16</span>(<span class="cit-supplement">Suppl 1</span>): <span class="cit-fpage">S10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D16%26rft.spage%253DS10%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.29" data-doi="10.2135/cropsci2011.06.0297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in plant breeding: a comparison of models.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0297&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.30" data-doi="10.2135/cropsci2012.07.0420"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Using genomic prediction to characterize environments and optimize prediction accuracy in applied breeding data.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">53</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">921</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D53%26rft.spage%253D921%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2012.07.0420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2012.07.0420&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.31" data-doi="10.1007/s00122-013-2231-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Integrating environmental covariates and crop modeling into the genomic selection framework to predict genotype by environment interactions.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">463</span>–<span class="cit-lpage">480</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D463%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2231-5%26rft_id%253Dinfo%253Apmid%252F24264761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2231-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24264761&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.32" data-doi="10.1093/imanum/22.3.329"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Higham</span>  <span class="cit-name-given-names">N. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Computing the nearest correlation matrix—a problem from finance.</span> <abbr class="cit-jnl-abbrev">IMA J. Numer. Anal.</abbr> <span class="cit-vol">22</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">329</span>–<span class="cit-lpage">343</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIMA%2BJournal%2Bof%2BNumerical%2BAnalysis%26rft.stitle%253DIMA%2BJ%2BNumer%2BAnal%26rft.aulast%253DHigham%26rft.auinit1%253DN.%2BJ.%26rft.volume%253D22%26rft.issue%253D3%26rft.spage%253D329%26rft.epage%253D343%26rft.atitle%253DComputing%2Bthe%2Bnearest%2Bcorrelation%2Bmatrix--a%2Bproblem%2Bfrom%2Bfinance%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fimanum%252F22.3.329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiaW1hbnVtIjtzOjU6InJlc2lkIjtzOjg6IjIyLzMvMzI5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi80LzEwNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.33" data-doi="10.1007/s00122-014-2418-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Isidro</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Training set optimization under population structure in genomic selection.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">145</span>–<span class="cit-lpage">158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D145%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2418-4%26rft_id%253Dinfo%253Apmid%252F25367380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2418-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25367380&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.34" data-doi="10.1534/genetics.112.144246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Multiple-trait genomic selection methods increase genetic value prediction accuracy.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1513</span>–<span class="cit-lpage">1522</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.144246%26rft_id%253Dinfo%253Apmid%252F23086217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzE1MTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzQvMTA0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.35" data-doi="10.1038/ng.548"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sul</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Service</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">S.-y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Variance component model to account for sample structure in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">348</span>–<span class="cit-lpage">354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D348%26rft.epage%253D354%26rft.atitle%253DVariance%2Bcomponent%2Bmodel%2Bto%2Baccount%2Bfor%2Bsample%2Bstructure%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.548%26rft_id%253Dinfo%253Apmid%252F20208533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.548&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20208533&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276150500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.36" data-doi="10.1186/1297-9686-44-39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karoui</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carabaño</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Díaz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Joint genomic evaluation of French dairy cattle breeds using multiple-trait models.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D44%26rft.spage%253D39%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-44-39%26rft_id%253Dinfo%253Apmid%252F23216664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-44-39&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23216664&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.37" data-doi="10.1186/gb-2009-10-3-r25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.</span> <abbr class="cit-jnl-abbrev">Genome biol</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">R25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253DR25%26rft.epage%253DR25%26rft.atitle%253DUltrafast%2Band%2Bmemory-efficient%2Balignment%2Bof%2Bshort%2BDNA%2Bsequences%2Bto%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-3-r25%26rft_id%253Dinfo%253Apmid%252F19261174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-3-r25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19261174&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.38" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">16</span>): <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNC8xMDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serba</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouton</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanzatella</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic linkage mapping and transmission ratio distortion in a three-generation four-founder population of <em>Panicum virgatum</em> (L.).</span> <abbr class="cit-jnl-abbrev">G3:Genes Genomes Genetics</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">913</span>–<span class="cit-lpage">923</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%253AGenes%2BGenomes%2BGenetics%26rft.volume%253D4%26rft.spage%253D913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liaw</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiener</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> Classification and regression by randomForest. R news 2: 18–22.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.41" data-doi="10.1111/j.1439-0388.2011.00917.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Dimension reduction and variable selection for genomic selection: application to predicting milk yield in Holsteins.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">128</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">247</span>–<span class="cit-lpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DLong%26rft.auinit1%253DN.%26rft.volume%253D128%26rft.issue%253D4%26rft.spage%253D247%26rft.epage%253D257%26rft.atitle%253DDimension%2Breduction%2Band%2Bvariable%2Bselection%2Bfor%2Bgenomic%2Bselection%253A%2Bapplication%2Bto%2Bpredicting%2Bmilk%2Byield%2Bin%2BHolsteins.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2011.00917.x%26rft_id%253Dinfo%253Apmid%252F21749471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2011.00917.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21749471&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.42" data-doi="10.1016/B978-0-12-385531-2.00002-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asoro</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection in plant breeding: knowledge and prospects.</span> <abbr class="cit-jnl-abbrev">Adv. Agron.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">77</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BAgron.%26rft.volume%253D110%26rft.spage%253D77%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-385531-2.00002-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-385531-2.00002-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.43" data-doi="10.1111/j.1439-0388.2012.01017.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Makgahlela</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mäntysaari</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strandén</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koivula</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Across breed multi‐trait random regression genomic predictions in the Nordic Red dairy cattle.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">130</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">10</span>–<span class="cit-lpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D130%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2012.01017.x%26rft_id%253Dinfo%253Apmid%252F23317061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2012.01017.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23317061&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.44" data-doi="10.1093/bioinformatics/btq526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kinnamon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zuchner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">SeqEM: an adaptive genotype-calling approach for next-generation sequencing studies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>(<span class="cit-issue">22</span>): <span class="cit-fpage">2803</span>–<span class="cit-lpage">2810</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq526%26rft_id%253Dinfo%253Apmid%252F20861027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMjIvMjgwMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNC8xMDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzQvMTA0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Milligan</span>  <span class="cit-name-given-names">B. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Maximum-likelihood estimation of relatedness.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1153</span>–<span class="cit-lpage">1167</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMilligan%26rft.auinit1%253DB.%2BG.%26rft.volume%253D163%26rft.issue%253D3%26rft.spage%253D1153%26rft.epage%253D1167%26rft.atitle%253DMaximum-Likelihood%2BEstimation%2Bof%2BRelatedness%26rft_id%253Dinfo%253Apmid%252F12663552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2My8zLzExNTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzQvMTA0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.47" data-doi="10.1186/1297-9686-41-56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moser</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tier</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crump</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khatkar</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raadsma</span>  <span class="cit-name-given-names">H. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A comparison of five methods to predict genomic breeding values of dairy bulls from genome-wide SNP markers.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DMoser%26rft.auinit1%253DG.%26rft.volume%253D41%26rft.spage%253D56%26rft.epage%253D56%26rft.atitle%253DA%2Bcomparison%2Bof%2Bfive%2Bmethods%2Bto%2Bpredict%2Bgenomic%2Bbreeding%2Bvalues%2Bof%2Bdairy%2Bbulls%2Bfrom%2Bgenome-wide%2BSNP%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-56%26rft_id%253Dinfo%253Apmid%252F20043835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-56&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20043835&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.48" data-doi="10.1111/j.1439-0388.2007.00700.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muir</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Comparison of genomic and traditional BLUP‐estimated breeding value accuracy and selection response under alternative trait and genomic parameters.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">124</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">342</span>–<span class="cit-lpage">355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DMuir%26rft.auinit1%253DW.%2BM.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D342%26rft.epage%253D355%26rft.atitle%253DComparison%2Bof%2Bgenomic%2Band%2Btraditional%2BBLUP-estimated%2Bbreeding%2Bvalue%2Baccuracy%2Band%2Bselection%2Bresponse%2Bunder%2Balternative%2Btrait%2Band%2Bgenomic%2Bparameters.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2007.00700.x%26rft_id%253Dinfo%253Apmid%252F18076471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2007.00700.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18076471&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251432800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nishio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Satoh</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic best linear unbiased prediction method reflecting the degree of linkage disequilibrium.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> 132: 357–365.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.50"><div class="cit-metadata unstructured">Okada, M., C. Lanzatella, M. C. Saha, J. Bouton, R. Wu <em>et al.</em>, 2010 Complete switchgrass genetic maps reveal subgenome collinearity, preferential pairing and multilocus interactions. Genetics 185(3): 745–760.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.51" data-doi="10.1198/016214508000000337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casella</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The Bayesian lasso.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">103</span>(<span class="cit-issue">482</span>): <span class="cit-fpage">681</span>–<span class="cit-lpage">686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D103%26rft.spage%253D681%26rft_id%253Dinfo%253Adoi%252F10.1198%252F016214508000000337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/016214508000000337&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000257897500025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> Genome-wide regression and prediction with the BGLR statistical package. Genetics 198: 483–495.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.1049.53"><div class="cit-metadata unstructured">Perlack, R. D., L. L. Wright, A. F. Turhollow, R. L. Graham, B. J. Stokes <em>et al.</em>, 2005 <em>Biomass as feedstock for a bioenergy and bioproducts industry: the technical feasibility of a billion-ton annual supply</em>, Oak Ridge National Laboratory, Oak Ridge, TN.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.4.1049.54" data-doi="10.3835/plantgenome2012.06.0006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in wheat breeding using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">Plant Genome J.</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%2BJ.%26rft.volume%253D5%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.06.0006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.06.0006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.55" data-doi="10.2135/cropsci2013.05.0353"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">R. M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilela de Resende</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic selection in forage breeding: accuracy and methods.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">143</span>–<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D143%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.05.0353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.05.0353&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.56" data-doi="10.1007/s00122-013-2175-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Optimizing the allocation of resources for genomic selection in one breeding cycle.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">2835</span>–<span class="cit-lpage">2848</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D2835%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2175-9%26rft_id%253Dinfo%253Apmid%252F23982591%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2175-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23982591&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.57" data-doi="10.1534/genetics.112.141473"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rincent</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laloë</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altmann</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunel</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Maximizing the reliability of genomic selection by optimizing the calibration set of reference individuals: comparison of methods in two diverse groups of maize inbreds (<em>Zea mays</em> L.).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">715</span>–<span class="cit-lpage">728</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.141473%26rft_id%253Dinfo%253Apmid%252F22865733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzIvNzE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi80LzEwNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.58" data-doi="10.1016/0960-8524(95)00176-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanderson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reed</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLaughlin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wullschleger</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conger</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Switchgrass as a sustainable bioenergy crop.</span> <abbr class="cit-jnl-abbrev">Bioresour. Technol.</abbr> <span class="cit-vol">56</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">83</span>–<span class="cit-lpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioresour.%2BTechnol.%26rft.volume%253D56%26rft.spage%253D83%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0960-8524%252895%252900176-X%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0960-8524(95)00176-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996VG11600013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.59" data-doi="10.1534/genetics.112.146720"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alam</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fikse</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rönnegård</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A novel generalized ridge regression method for quantitative genetics.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1255</span>–<span class="cit-lpage">1268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.146720%26rft_id%253Dinfo%253Apmid%252F23335338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5My80LzEyNTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzQvMTA0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.60" data-doi="10.1016/j.ajhg.2012.10.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Speed</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hemani</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balding</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Improved heritability estimation from genome-wide SNPs.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">91</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">1011</span>–<span class="cit-lpage">1021</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D91%26rft.spage%253D1011%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2012.10.010%26rft_id%253Dinfo%253Apmid%252F23217325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2012.10.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23217325&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.61" data-doi="10.1073/pnas.1530509100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Statistical significance for genomewide studies.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>(<span class="cit-issue">16</span>): <span class="cit-fpage">9440</span>–<span class="cit-lpage">9445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1530509100%26rft_id%253Dinfo%253Apmid%252F12883005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAwLzE2Lzk0NDAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzQvMTA0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.62" data-doi="10.3168/jds.2014-8210"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janss</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comparison of genomic predictions using genomic relationship matrices built with different weighting factors to account for locus-specific variances.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">97</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">6547</span>–<span class="cit-lpage">6559</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BDairy%2BSci.%26rft.volume%253D97%26rft.spage%253D6547%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2014-8210%26rft_id%253Dinfo%253Apmid%252F25129495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2014-8210&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25129495&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.63" data-doi="10.1038/hdy.2010.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kramer</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Variation explained in mixed-model association mapping.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">105</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">333</span>–<span class="cit-lpage">340</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DSun%26rft.auinit1%253DG.%26rft.volume%253D105%26rft.issue%253D4%26rft.spage%253D333%26rft.epage%253D340%26rft.atitle%253DVariation%2Bexplained%2Bin%2Bmixed-model%2Bassociation%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2010.11%26rft_id%253Dinfo%253Apmid%252F20145669%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2010.11&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20145669&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.64" data-doi="10.3198/jpr2013.12.0076crc"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarath</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Registration of ‘Liberty’switchgrass.</span> <abbr class="cit-jnl-abbrev">J. Plant Regist.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">242</span>–<span class="cit-lpage">247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPlant%2BRegist.%26rft.volume%253D8%26rft.spage%253D242%26rft_id%253Dinfo%253Adoi%252F10.3198%252Fjpr2013.12.0076crc%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3198/jpr2013.12.0076crc&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.65" data-doi="10.2307/2529947"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Inferences about linkage disequilibrium.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">235</span>–<span class="cit-lpage">254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DWeir%26rft.auinit1%253DB.%2BS.%26rft.volume%253D35%26rft.issue%253D1%26rft.spage%253D235%26rft.epage%253D254%26rft.atitle%253DInferences%2Babout%2Blinkage%2Bdisequilibrium.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2529947%26rft_id%253Dinfo%253Apmid%252F497335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2529947&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=497335&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1979GN09700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.1049.66"><div class="cit-metadata unstructured">Wood, S., 2006 <em>Generalized additive models: an introduction with R</em>, CRC Press, Boca Raton.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.67" data-doi="10.1038/ng.823"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manolio</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pasquale</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boerwinkle</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caporaso</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome partitioning of genetic variation for complex traits using common SNPs.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">519</span>–<span class="cit-lpage">525</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D43%26rft.issue%253D6%26rft.spage%253D519%26rft.epage%253D525%26rft.atitle%253DGenome%2Bpartitioning%2Bof%2Bgenetic%2Bvariation%2Bfor%2Bcomplex%2Btraits%2Busing%2Bcommon%2BSNPs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.823%26rft_id%253Dinfo%253Apmid%252F21552263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.823&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21552263&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.68" data-doi="10.1214/09-AOS729"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C.-H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Nearly unbiased variable selection under minimax concave penalty.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">894</span>–<span class="cit-lpage">942</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D38%26rft.spage%253D894%26rft_id%253Dinfo%253Adoi%252F10.1214%252F09-AOS729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/09-AOS729&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.69" data-doi="10.1007/s10709-011-9597-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zalapa</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jakubowski</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Post-glacial evolution of <em>Panicum virgatum</em>: centers of diversity and gene pools revealed by SSR markers and cpDNA sequences.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">139</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">933</span>–<span class="cit-lpage">948</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D139%26rft.spage%253D933%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-011-9597-6%26rft_id%253Dinfo%253Apmid%252F21786028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-011-9597-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21786028&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293244900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1049.70" data-doi="10.1093/bioinformatics/bts606"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levine</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogarten</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laurie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A high-performance computing toolset for relatedness and principal component analysis of SNP data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>(<span class="cit-issue">24</span>): <span class="cit-fpage">3326</span>–<span class="cit-lpage">3328</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts606%26rft_id%253Dinfo%253Apmid%252F23060615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMjQvMzMyNiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNC8xMDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/4/1049.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/4/1031" title="The Stress Granule RNA-Binding Protein TIAR-1 Protects Female Germ Cells from Heat Shock in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/4/1063" title="Large-Scale Survey of Intraspecific Fitness and Cell Morphology Variation in a Protoploid Yeast Species" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/4"><span>Volume 6 Issue 4, April 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/4.cover-source.jpg?itok=Lj45nPaf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/1049.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-XbmMxujzHy7_Z6UuNUeFH2NykvMl-7-BviQJOqlW9HQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/4/1049" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/4/1049" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/4/1049" class="active" data-icon-position="" data-hide-link-title="0">Accuracy of Genomic Prediction in Switchgrass (Panicum virgatum L.) Improved by Accounting for Linkage Disequilibrium</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460127" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-AKbnibmR4iN7V58oJYOOIBk91H0RkjPl5FdV83wtyVY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/4/1049.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/4/1049" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ZHADe2FVrYVzOpZVDiYpxYx3O8sK8vmYndRdJobHKgg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/4/1049" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/4/1049.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460127" id="node460127--2" data-pisa="ggg;6/4/1049" data-pisa-master="ggg;g3.115.024950" data-apath="/ggg/6/4/1049.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accuracy of Genomic Prediction in Switchgrass (<em>Panicum virgatum</em> L.) Improved by Accounting for Linkage Disequilibrium</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Guillaume P. Ramstein</span>, <span class="highwire-citation-author" data-delta="1">Joseph Evans</span>, <span class="highwire-citation-author" data-delta="2">Shawn M. Kaeppler</span>, <span class="highwire-citation-author" data-delta="3">Robert B. Mitchell</span>, <span class="highwire-citation-author" data-delta="4">Kenneth P. Vogel</span>, <span class="highwire-citation-author" data-delta="5">C. Robin Buell</span> and <span class="highwire-citation-author" data-delta="6">Michael D. Casler</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1049-1062;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024950 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460127--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Guillaume P. Ramstein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guillaume%2BP.%2BRamstein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramstein%20GP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGuillaume%2BP.%2BRamstein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ramstein@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">ramstein@wisc.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Joseph Evans </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Energy Great Lakes Bioenergy Research Center, Michigan State University, East Lansing, MI 48824</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, Michigan State University, East Lansing, MI 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BEvans%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Evans%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BEvans%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Shawn M. Kaeppler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, WI 53706</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Energy Great Lakes Bioenergy Research Center, University of Wisconsin-Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shawn%2BM.%2BKaeppler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kaeppler%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShawn%2BM.%2BKaeppler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Robert B. Mitchell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Grain, Forage, and Bioenergy Research Unit, Agricultural Research Service, United States Department of Agriculture, University of Nebraska, Lincoln, NE 68583-0937</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BB.%2BMitchell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mitchell%20RB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BB.%2BMitchell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kenneth P. Vogel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Grain, Forage, and Bioenergy Research Unit, Agricultural Research Service, United States Department of Agriculture, University of Nebraska, Lincoln, NE 68583-0937</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kenneth%2BP.%2BVogel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vogel%20KP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKenneth%2BP.%2BVogel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">C. Robin Buell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Energy Great Lakes Bioenergy Research Center, Michigan State University, East Lansing, MI 48824</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, Michigan State University, East Lansing, MI 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C.%2BRobin%2BBuell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buell%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AC.%2BRobin%2BBuell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Michael D. Casler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, WI 53706</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Agricultural Research Service, United States Department of Agriculture, Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BD.%2BCasler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Casler%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BD.%2BCasler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/4/1049" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/4/1049&amp;title=Accuracy%20of%20Genomic%20Prediction%20in%20Switchgrass%20%28Panicum%20virgatum%20L.%29%20Improved%20by%20Accounting%20for%20Linkage%20Disequilibrium" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/4/1049&amp;title=Accuracy%20of%20Genomic%20Prediction%20in%20Switchgrass%20%28Panicum%20virgatum%20L.%29%20Improved%20by%20Accounting%20for%20Linkage%20Disequilibrium" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/4/1049&amp;title=Accuracy%20of%20Genomic%20Prediction%20in%20Switchgrass%20%28Panicum%20virgatum%20L.%29%20Improved%20by%20Accounting%20for%20Linkage%20Disequilibrium" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/4/1049&amp;text=Accuracy%20of%20Genomic%20Prediction%20in%20Switchgrass%20%28Panicum%20virgatum%20L.%29%20Improved%20by%20Accounting%20for%20Linkage%20Disequilibrium" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/4/1049&amp;title=Accuracy%20of%20Genomic%20Prediction%20in%20Switchgrass%20%28Panicum%20virgatum%20L.%29%20Improved%20by%20Accounting%20for%20Linkage%20Disequilibrium" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/4/1049&amp;t=Accuracy%20of%20Genomic%20Prediction%20in%20Switchgrass%20%28Panicum%20virgatum%20L.%29%20Improved%20by%20Accounting%20for%20Linkage%20Disequilibrium" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/4/1049&amp;title=Accuracy%20of%20Genomic%20Prediction%20in%20Switchgrass%20%28Panicum%20virgatum%20L.%29%20Improved%20by%20Accounting%20for%20Linkage%20Disequilibrium" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/4/1049&amp;title=Accuracy%20of%20Genomic%20Prediction%20in%20Switchgrass%20%28Panicum%20virgatum%20L.%29%20Improved%20by%20Accounting%20for%20Linkage%20Disequilibrium" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460127/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460127" id="node460127--3" data-pisa="ggg;6/4/1049" data-pisa-master="ggg;g3.115.024950" data-apath="/ggg/6/4/1049.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accuracy of Genomic Prediction in Switchgrass (<em>Panicum virgatum</em> L.) Improved by Accounting for Linkage Disequilibrium</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Guillaume P. Ramstein</span>, <span class="highwire-citation-author" data-delta="1">Joseph Evans</span>, <span class="highwire-citation-author" data-delta="2">Shawn M. Kaeppler</span>, <span class="highwire-citation-author" data-delta="3">Robert B. Mitchell</span>, <span class="highwire-citation-author" data-delta="4">Kenneth P. Vogel</span>, <span class="highwire-citation-author" data-delta="5">C. Robin Buell</span> and <span class="highwire-citation-author" data-delta="6">Michael D. Casler</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1049-1062;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024950 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460127--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Guillaume P. Ramstein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guillaume%2BP.%2BRamstein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramstein%20GP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGuillaume%2BP.%2BRamstein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ramstein@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">ramstein@wisc.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Joseph Evans </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Energy Great Lakes Bioenergy Research Center, Michigan State University, East Lansing, MI 48824</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, Michigan State University, East Lansing, MI 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BEvans%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Evans%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BEvans%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Shawn M. Kaeppler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, WI 53706</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Energy Great Lakes Bioenergy Research Center, University of Wisconsin-Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shawn%2BM.%2BKaeppler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kaeppler%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShawn%2BM.%2BKaeppler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Robert B. Mitchell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Grain, Forage, and Bioenergy Research Unit, Agricultural Research Service, United States Department of Agriculture, University of Nebraska, Lincoln, NE 68583-0937</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BB.%2BMitchell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mitchell%20RB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BB.%2BMitchell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kenneth P. Vogel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Grain, Forage, and Bioenergy Research Unit, Agricultural Research Service, United States Department of Agriculture, University of Nebraska, Lincoln, NE 68583-0937</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kenneth%2BP.%2BVogel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vogel%20KP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKenneth%2BP.%2BVogel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">C. Robin Buell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Energy Great Lakes Bioenergy Research Center, Michigan State University, East Lansing, MI 48824</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, Michigan State University, East Lansing, MI 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C.%2BRobin%2BBuell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buell%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AC.%2BRobin%2BBuell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Michael D. Casler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, WI 53706</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Agricultural Research Service, United States Department of Agriculture, Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BD.%2BCasler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Casler%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BD.%2BCasler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1049&amp;rft.epage=1062&amp;rft.atitle=Accuracy%20of%20Genomic%20Prediction%20in%20Switchgrass%20%28Panicum%20virgatum%20L.%29%20Improved%20by%20Accounting%20for%20Linkage%20Disequilibrium&amp;rft.volume=6&amp;rft.issue=4&amp;rft.date=2016-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Ramstein%2C+Guillaume+P.&amp;rft.au=Evans%2C+Joseph&amp;rft.au=Kaeppler%2C+Shawn+M.&amp;rft.au=Mitchell%2C+Robert+B.&amp;rft.au=Vogel%2C+Kenneth+P.&amp;rft.au=Buell%2C+C.+Robin&amp;rft.au=Casler%2C+Michael+D."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460127/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460127/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460127/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460127/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460127/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460127/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460127/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460127/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460127/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460127/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460127/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460127/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/4/1049" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-22" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/4/1049.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/4/1049.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/4/1049.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/4/1049.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
