<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/4/893" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/4/893.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/4/893.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/4/893.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/4/893/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/4/893.atom" />
<meta name="HW.pisa" content="ggg;6/4/893" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Detecting and Quantifying Changing Selection Intensities from Time-Sampled Polymorphism Data" />
<meta name="DC.Identifier" content="10.1534/g3.115.023200" />
<meta name="DC.Date" content="2016-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Shim et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="During his well-known debate with Fisher regarding the phenotypic dataset of Panaxia dominula , Wright suggested fluctuating selection as a potential explanation for the observed change in allele frequencies. This model has since been invoked in a number of analyses, with the focus of discussion centering mainly on random or oscillatory fluctuations of selection intensities. Here, we present a novel method to consider nonrandom changes in selection intensities using Wright-Fisher approximate Bayesian (ABC)-based approaches, in order to detect and evaluate a change in selection strength from time-sampled data. This novel method jointly estimates the position of a change point as well as the strength of both corresponding selection coefficients (and dominance for diploid cases) from the allele trajectory. The simulation studies of this method reveal the combinations of parameter ranges and input values that optimize performance, thus indicating optimal experimental design strategies. We apply this approach to both the historical dataset of P. dominula in order to shed light on this historical debate, as well as to whole-genome time-serial data from influenza virus in order to identify sites with changing selection intensities in response to drug treatment." />
<meta name="DC.Contributor" content="Hyunjin Shim" />
<meta name="DC.Contributor" content="Stefan Laurent" />
<meta name="DC.Contributor" content="Sebastian Matuszewski" />
<meta name="DC.Contributor" content="Matthieu Foll" />
<meta name="DC.Contributor" content="Jeffrey D. Jensen" />
<meta name="article:published_time" content="2016-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Detecting and Quantifying Changing Selection Intensities from Time-Sampled Polymorphism Data" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;During his well-known debate with Fisher regarding the phenotypic dataset of &lt;i&gt;Panaxia dominula&lt;/i&gt;, Wright suggested fluctuating selection as a potential explanation for the observed change in allele frequencies. This model has since been invoked in a number of analyses, with the focus of discussion centering mainly on random or oscillatory fluctuations of selection intensities. Here, we present a novel method to consider nonrandom changes in selection intensities using Wright-Fisher approximate Bayesian (ABC)-based approaches, in order to detect and evaluate a change in selection strength from time-sampled data. This novel method jointly estimates the position of a change point as well as the strength of both corresponding selection coefficients (and dominance for diploid cases) from the allele trajectory. The simulation studies of this method reveal the combinations of parameter ranges and input values that optimize performance, thus indicating optimal experimental design strategies. We apply this approach to both the historical dataset of &lt;i&gt;P. dominula&lt;/i&gt; in order to shed light on this historical debate, as well as to whole-genome time-serial data from influenza virus in order to identify sites with changing selection intensities in response to drug treatment.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/04/01" />
<meta name="citation_mjid" content="ggg;6/4/893" />
<meta name="citation_id" content="6/4/893" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/4/893" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/4/893.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/4/893.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/4/893.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.023200" />
<meta name="citation_pmid" content="26869618" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="893" />
<meta name="citation_lastpage" content="904" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Hyunjin Shim" />
<meta name="citation_author_institution" content="School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne, Switzerland" />
<meta name="citation_author_institution" content="Swiss Institute of Bioinformatics, Lausanne, Switzerland" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7052-0971" />
<meta name="citation_author" content="Stefan Laurent" />
<meta name="citation_author_institution" content="School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne, Switzerland" />
<meta name="citation_author_institution" content="Swiss Institute of Bioinformatics, Lausanne, Switzerland" />
<meta name="citation_author" content="Sebastian Matuszewski" />
<meta name="citation_author_institution" content="School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne, Switzerland" />
<meta name="citation_author_institution" content="Swiss Institute of Bioinformatics, Lausanne, Switzerland" />
<meta name="citation_author" content="Matthieu Foll" />
<meta name="citation_author_institution" content="School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne, Switzerland" />
<meta name="citation_author_institution" content="Swiss Institute of Bioinformatics, Lausanne, Switzerland" />
<meta name="citation_author_institution" content="International Agency for Research on Cancer, Lyon, France" />
<meta name="citation_author" content="Jeffrey D. Jensen" />
<meta name="citation_author_institution" content="School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne, Switzerland" />
<meta name="citation_author_institution" content="Swiss Institute of Bioinformatics, Lausanne, Switzerland" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=C. Bank;citation_author=G. Ewing;citation_author=A. Ferrer-Admettla;citation_author=M. Foll;citation_author=J. D. Jensen;citation_title=Thinking Too Positive? Revisiting Current Methods of Population Genetic Selection Inference.;citation_pages=540-546;citation_volume=30;citation_year=2014;citation_pmid=25438719;citation_doi=10.1016/j.tig.2014.09.010" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Ecol. Evol. Syst.;citation_author=M. A. Beaumont;citation_title=Approximate Bayesian Computation in Evolution and Ecology.;citation_pages=379-406;citation_volume=41;citation_year=2010;citation_doi=10.1146/annurev-ecolsys-102209-144621" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. A. Beaumont;citation_author=W. Zhang;citation_author=J. D. Balding;citation_title=Approximate Bayesian Computation in Population Genetics;citation_pages=2025-2035;citation_volume=162;citation_year=2002;citation_issue=4;citation_pmid=12524368" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=G. Bell;citation_title=Fluctuating selection: the perpetual renewal of adaptation in variable environments;citation_pages=87-97;citation_volume=365;citation_year=2010;citation_issue=1537;citation_pmid=20008388;citation_doi=10.1098/rstb.2009.0150" />
<meta name="citation_reference" content="citation_title=Parametric Statistical Change Point Analysis;citation_author=J. Chen;citation_author=A. K. Gupta;citation_year=2001" />
<meta name="citation_reference" content="citation_title=Evolution;citation_author=L. M. Chevin;citation_pages=708-721;citation_volume=67;citation_year=2013;citation_pmid=23461322;citation_doi=10.1111/j.1558-5646.2012.01809.x" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=L. M. Cook;citation_author=D. A. Jones;citation_title=The medionigra Gene in the Moth Panaxia dominula: The Case for Selection;citation_pages=1623-1634;citation_volume=351;citation_year=1996;citation_issue=1347;citation_doi=10.1098/rstb.1996.0146" />
<meta name="citation_reference" content="citation_title=Mathematical Population Genetics 1 - Theoretical Introduction;citation_author=W. J. Ewens;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=R. A. Fisher;citation_author=E. B. Ford;citation_title=The Spread of a Gene in Natural Conditions in a Colony of the Moth PANAXIA DOMINULA L.;citation_pages=143-174;citation_volume=1;citation_year=1947;citation_doi=10.1038/hdy.1947.11" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=M. Foll;citation_author=Y.-P. Poh;citation_author=N. Renzette;citation_author=A. Ferrer-Admetlla;citation_author=C. Bank;citation_title=Influenza Virus Drug Resistance: A Time-Sampled Population Genetics Perspective.;citation_pages=e1004185;citation_volume=10;citation_year=2014;citation_pmid=24586206;citation_doi=10.1371/journal.pgen.1004185" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=M. Foll;citation_author=H. Shim;citation_author=J. D. Jensen;citation_title=WFABC: A Wright-Fisher ABC-Based Approach for Inferring Effective Population Sizes and Selection Coefficients from Time-Sampled Data.;citation_pages=87-98;citation_volume=15;citation_year=2015;citation_pmid=24834845;citation_doi=10.1111/1755-0998.12280" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_title=Bayesian Inference of Selection in a Heterogeneous Environment from Genetic Time-Series Data.;citation_author=Z. Gompert;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=I. J. Good;citation_title=The Bayes/Non-Bayes Compromise: A Brief Review.;citation_pages=597-606;citation_volume=87;citation_year=1992;citation_doi=10.2307/2290192" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=T. I. Gossmann;citation_author=D. Waxman;citation_author=A. Eyre-Walker;citation_title=Fluctuating Selection Models and Mcdonald-Kreitman Type Analyses.;citation_pages=1-5;citation_volume=9;citation_year=2014;citation_pmid=24551193;citation_doi=10.1371/journal.pone.0089680" />
<meta name="citation_reference" content="citation_journal_title=Math. Proc. Camb. Philos. Soc.;citation_author=J. B. S. Haldane;citation_title=A Mathematical Theory of Natural and Artificial Selection. Part IX. Rapid Selection.;citation_pages=224-248;citation_volume=28;citation_year=1932" />
<meta name="citation_reference" content="citation_journal_abbrev=Ecol Lett;citation_author=A. R. Hall;citation_author=P. D. Scanlan;citation_author=A. D. Morgan;citation_author=A. Buckling;citation_title=Host-parasite coevolutionary arms races give way to fluctuating selection.;citation_pages=635-642;citation_volume=14;citation_year=2011;citation_issue=7;citation_pmid=21521436;citation_doi=10.1111/j.1461-0248.2011.01624.x" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=D. A. Jones;citation_title=Temperatures in the Cothill Habitat of Panaxia (Callimorpha) Dominula L. (the Scarlet Tiger Moth).;citation_pages=578-586;citation_volume=84;citation_year=2000;citation_pmid=10849083;citation_doi=10.1046/j.1365-2540.2000.00700.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. E. Jorde;citation_author=N. Ryman;citation_title=Unbiased Estimator for Genetic Drift and Effective Population Size.;citation_volume=177;citation_year=2007;citation_pmid=17720927;citation_doi=10.1534/genetics.107.075481" />
<meta name="citation_reference" content="citation_journal_title=Theor. Popul. Biol.;citation_author=S. Karlin;citation_author=B. Levikson;citation_title=Temporal Fluctuations in Selection Intensities: Case of Small Population Size.;citation_pages=383-412;citation_volume=6;citation_year=1974;citation_doi=10.1016/0040-5809(74)90017-3" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=S. Karlin;citation_author=U. Lieberman;citation_title=Random temporal variation in selection intensities: case of large population size.;citation_pages=355-382;citation_volume=6;citation_year=1974;citation_issue=3;citation_pmid=4460262;citation_doi=10.1016/0040-5809(74)90016-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Kimura;citation_title=PROCESS LEADING TO QUASI-FIXATION OF GENES IN NATURAL POPULATIONS DUE TO RANDOM FLUCTUATION OF SELECTION INTENSITIES;citation_pages=280-295;citation_volume=39;citation_year=1954;citation_issue=3;citation_pmid=17247483" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Lacerda;citation_author=C. Seoighe;citation_title=Population Genetics Inference for Longitudinally-Sampled Mutants under Strong Selection.;citation_volume=198;citation_year=2014;citation_pmid=25213172;citation_doi=10.1534/genetics.114.167957" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A.-S. Malaspinas;citation_author=O. Malaspinas;citation_author=S. N. Evans;citation_author=M. Slatkin;citation_title=Estimating Allele Age and Selection Coefficient from Time-Serial Data.;citation_volume=192;citation_year=2012;citation_pmid=22851647;citation_doi=10.1534/genetics.112.140939" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=I. Mathieson;citation_author=G. McVean;citation_title=Estimating Selection Coefficients in Spatially Structured Populations from Time Series Data of Allele Frequencies.;citation_volume=193;citation_year=2013;citation_pmid=23307902;citation_doi=10.1534/genetics.112.147611" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=R. B. O’Hara;citation_title=Comparing the effects of genetic drift and fluctuating selection on genotype frequency changes in the scarlet tiger moth;citation_pages=211-217;citation_volume=272;citation_year=2005;citation_issue=1559;citation_pmid=15695213;citation_doi=10.1098/rspb.2004.2929" />
<meta name="citation_reference" content="citation_journal_title=Biometrika;citation_journal_abbrev=Biometrika;citation_author=B. Y. E. S. Page;citation_title=CONTINUOUS INSPECTION SCHEMES;citation_pages=100-115;citation_volume=41;citation_year=1954;citation_issue=1-2;citation_doi=10.1093/biomet/41.1-2.100" />
<meta name="citation_reference" content="citation_journal_title=J. Virol.;citation_author=N. Renzette;citation_author=D. R. Caffrey;citation_author=K. B. Zeldovich;citation_author=P. Liu;citation_author=G. R. Gallagher;citation_title=Evolution of the Influenza A Virus Genome during Development of Oseltamivir Resistance in Vitro.;citation_volume=88;citation_year=2014;citation_pmid=24155392;citation_doi=10.1128/JVI.01067-13" />
<meta name="citation_reference" content="citation_title=Statistical Methods for Quality Improvement;citation_author=T. P. Ryan;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=M. Sunnåker;citation_author=A. G. Busetto;citation_author=E. Numminen;citation_author=J. Corander;citation_author=M. Foll;citation_title=Approximate Bayesian Computation.;citation_pages=e1002803;citation_volume=9;citation_year=2013;citation_pmid=23341757;citation_doi=10.1371/journal.pcbi.1002803" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=M. M. Tanaka;citation_author=F. Valckenborgh;citation_title=Escaping an evolutionary lobster trap: drug resistance and compensatory mutation in a fluctuating environment.;citation_pages=1376-1387;citation_volume=65;citation_year=2011;citation_issue=5;citation_pmid=21521192;citation_doi=10.1111/j.1558-5646.2011.01223.x" />
<meta name="citation_reference" content="citation_journal_title=Virology;citation_journal_abbrev=Virology;citation_author=S. Thoennes;citation_author=Z.-N. Li;citation_author=B.-J. Lee;citation_author=W. A. Langley;citation_author=W. A. Skehel;citation_title=Analysis of residues near the fusion peptide in the influenza hemagglutinin structure for roles in triggering membrane fusion.;citation_pages=403-414;citation_volume=370;citation_year=2008;citation_issue=2;citation_pmid=17936324;citation_doi=10.1016/j.virol.2007.08.035" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=S. Wright;citation_title=On the roles of directed and random changes in gene frequency in the genetics of populations.;citation_pages=279-294;citation_volume=2;citation_year=1948;citation_issue=4;citation_pmid=18104586;citation_doi=10.2307/2405519" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=R. Xavier;citation_author=N. Turck;citation_author=A. Hainard;citation_author=N. Tiberti;citation_author=F. Lisacek;citation_title=pROC: an open-source package for R and S+ to analyze and compare ROC curves.;citation_pages=77-77;citation_volume=12;citation_year=2011;citation_pmid=21414208;citation_doi=10.1186/1471-2105-12-77" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Detecting and Quantifying Changing Selection Intensities from Time-Sampled Polymorphism Data" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
<meta name="twitter:description" content="During his well-known debate with Fisher regarding the phenotypic dataset of Panaxia dominula , Wright suggested fluctuating selection as a potential explanation for the observed change in allele frequencies. This model has since been invoked in a number of analyses, with the focus of discussion centering mainly on random or oscillatory fluctuations of selection intensities. Here, we present a novel method to consider nonrandom changes in selection intensities using Wright-Fisher approximate Bayesian (ABC)-based approaches, in order to detect and evaluate a change in selection strength from time-sampled data. This novel method jointly estimates the position of a change point as well as the strength of both corresponding selection coefficients (and dominance for diploid cases) from the allele trajectory. The simulation studies of this method reveal the combinations of parameter ranges and input values that optimize performance, thus indicating optimal experimental design strategies. We apply this approach to both the historical dataset of P. dominula in order to shed light on this historical debate, as well as to whole-genome time-serial data from influenza virus in order to identify sites with changing selection intensities in response to drug treatment." />
<meta name="og-title" property="og:title" content="Detecting and Quantifying Changing Selection Intensities from Time-Sampled Polymorphism Data" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/4/893" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="During his well-known debate with Fisher regarding the phenotypic dataset of Panaxia dominula , Wright suggested fluctuating selection as a potential explanation for the observed change in allele frequencies. This model has since been invoked in a number of analyses, with the focus of discussion centering mainly on random or oscillatory fluctuations of selection intensities. Here, we present a novel method to consider nonrandom changes in selection intensities using Wright-Fisher approximate Bayesian (ABC)-based approaches, in order to detect and evaluate a change in selection strength from time-sampled data. This novel method jointly estimates the position of a change point as well as the strength of both corresponding selection coefficients (and dominance for diploid cases) from the allele trajectory. The simulation studies of this method reveal the combinations of parameter ranges and input values that optimize performance, thus indicating optimal experimental design strategies. We apply this approach to both the historical dataset of P. dominula in order to shed light on this historical debate, as well as to whole-genome time-serial data from influenza virus in order to identify sites with changing selection intensities in response to drug treatment." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
    <title>Detecting and Quantifying Changing Selection Intensities from Time-Sampled Polymorphism Data | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__DbqSGiVjr90FgBIuPlGEAiimgNNAH8S6qbiYFsi8JyQ__E2bjoC6LAI1j30ywBL_JgYRwIhvNl2n0yRyt_snD9IM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"0fVyao1h8wZ4jQKmKlYQQRpxPTW9PQHoa0356YtRsxM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460106","apath":"\/ggg\/6\/4\/893.atom","pisa":"ggg;6\/4\/893","ac":{"\/ggg\/6\/4\/893.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/4\/893.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/4\/893"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/4\/893"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/4\/893":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022DBKIGXEUgBb3tzsmLPP+MJUR\\\\\\\/y6Vw0NSx12f\\\\\\\/5lcwsioUCg6RiWWdNdBuhAlzcQYkM6KnscvLtxOvU\\\\\\\/LqQWNCGio1Cs37RTWcy\\\\\\\/J3kvbjpU0cJfdaiiJHd+C1aWuzUqAFM37hMpWUGT2J2xlF+ZuCcjrBHbN97ynEu4GbdhHHkIwSfh5PAdIwVTpnrAXNE\\\\\\\/M+8G7nXbSn07XHOFxxiPtqzRVebYP6LUV8hiEV4\\\\\\\/LInBdmh6b+TSA\\\\\\\/nWLFRvzG3GV4LdlHiVdffbDdAr7BtzfXYjSHuPz0X4+M2EqQbPwkNmFMx+FJWQBAMJPj\\\\\\\/ibLWYCmtqn3EuSnIIAd6WSj2QA91irlUqySlDflVQISLzy\\\\\\\/529L52YHicZJpVIQDnLVqgxJq4qq5WRHYy\\\\\\\/iogvvBI4w0+qybV\\\\\\\/ADlWC77ERmNRaDB9pZIuioOksnvFg4sifNVWF2a+GucgVl4loeAFzsTZaqsJeLIVWSEK8pG1BrYfmSak5jODr+DmxySCzyYFstvCcDRz4XCDJM8B7cwu4ZtDDqiqHyIKRsHDii7V8VGhqz+4JiFBT0kDumjtxqRsvVPNesUja\\\\\\\/I3\\\\\\\/z0qrgjhBFxqtbYQK+\\\\\\\/mgxbYs5An4nFnsVZwzAetErnbWp7HUtxccyF6wT49rRBeBuEs058RqY+27a+YE+t9XAhIgepQp2zUT2CnC2rwl1Q98yT9MZPQSIf6YeuleSI8lftb2drHUv3ucPSoiDzYarwH\\\\\\\/Gk6OXfcH+EjghWOUE9KMnqOGCRFILr+Tx92dUh9JF9leo+8KjTavihgHH+9OOGT5Ftl991isKh6O9Tp851YFAWRiFfwch8L42ZNXyCqcVJQf77oIan9glUu6hNaolAQzAVwp8+c3rwlaKKErhh2vpcwJ6xSU95XjIRrLee8kW6hz9MpJ7mxXmTJIQZ7p6kdtX68+0uOckWySmaXgo78IHh5myeOh3hskVLuSTqAJR+tFAV3qe1WC012B2zjHs4Pzrgj0rhi\\\\\\\/JRZlyDMbJ7ShJTu+zmFYxG1+0LhFKgyGfmGBFHnuGpb2q\\\\\\\/pRyTGPHYdU7celT6U8PgfHlZ5SrsU5hevd2JcKNPMUGDrWxbQ4IRoXLOvpy9aX5sHPQTREZUHReP9q0OBupuq5OnCUf8fbsopjMV\\\\\\\/pxHyI13DiRxKUNs2GBVaUghk+5GKt66sygq\\\\\\\/b8JilEPUDf8tQrgatlc3DHtv\\\\\\\/wmm4SLLet3KheT3bmbZulCfc5sEM26yltAxSavvlcNqTcEH8giiVPPx2Fh3YETZ2\\\\\\\/2dKRZq\\\u0022,\\\u0022iv\\\u0022:\\\u0022ncLMeSza4RMMeT7jlE0jOg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222476f2a265da8cea12d2c02da3b212608f5bf5a35b310744f2ba2ad49f2ba1bc\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ul9dgITgn29JEYgjth3SErIbmM9LfixFEYRk1\\\\\\\/kaHvUpjrlQxHF6EhATJ8T8yLEL+GOMhcjojIToDxM46OCQ\\\\\\\/aEIZQwsUIKCrW+aXu5VTmM40a0AO+OejK+NIWVwZ6WJBS1DDZJLDLxZe0l1zSsulQebVvlLTDCT4jrWko990wy5oygQYNFZiIjOZ7EKva\\\\\\\/Foh+IXZcnsKDMkjhG2iwGwxjJRcTKB56pTc\\\\\\\/Khn5C0w2xlZ\\\\\\\/D35VJqy+IvTooN9gFe\\\\\\\/l4GAG6Cb7bRARXxjbtAT6cu2EmzL7MUne3nNHhYkPtDl0Nt0O7rz06cYBRePsm0q8mtWlDSxPARPwQ7gboUu35bGXBuC7orSt3JyzuexM+3Q7cdrtG9QMjl3BAjfXS2l58HGFw8HpgKnQXbEy2UElmNaIqj5u4MWirdokAIHDd8IZ1CQFlx5ywXfXE8oHRjlZZa\\\\\\\/NbwYFK6sr509tp3rnn21Q5kNk5xlnnDXaJ7wDgrwaw1JFaON5KvRDagF4JRcyeZWbk6Vxzm\\\\\\\/dhyGimym\\\\\\\/GkPQLE4X5czOuWRVb11VpZv0XPoDmr\\\\\\\/OsKJ6L7fFMUwEpf8wC4EAAOKhexymVLfms0\\\\\\\/ha0c++WWc1Gdvv59yoMi1pTAuWynZJYxg1QmNsZxzMLUjSqdhKZ3LIx3L2WXzpWhG3qmyRSAecJZWXD7rvfK\\\\\\\/mEqANmwxYB3lbSD7hauxmuA6a9dcyWUulCPRfr3noUpfeIViJQkhLl6a2JtG8rvCO5D4CNQ6nJcyGh\\\\\\\/fy6oOLuwC28Nm5k0yUsiEI2joKVcJ83b3XyRU6qqUsogMUyJd2RymauUDPO+Y+kwFnM4IyS8haPLual8tZjnl3k8tzA\\\\\\\/TujkPM2U+i9JTqVNdOKWKVME\\\\\\\/ENYMrCO8cqTCjJJMOOsq92Mg5bM7au51TVhr9gIecZLX6Ay5gzxUt8CjwwEXeI7qjiV6QR1MN4VE0ibNMXN2BBLMfCs5Uubp3Z4+M\\\\\\\/Akmy\\\\\\\/hofgdXD0mZQ6yMrEGpg0ULF7PDjGm1tm+f6ea1AX9djIc0rUw2Ey413b5ooyuP+EE7v7AMkSuQ1JEAf7BdaTsbhznJNH95Tv+0qFDy84IVEsLilsVr3OcA55yWfC0qfxfI45JEznG65faVEhvXu2EO3q68QjLpamddLVTvGtVKhX\\\\\\\/+JbGe9+H5tpwgksIsyPaRl0Mm4fxGleW0DSDvTmtpoDLrdjuEu5mIqQInhPPhJVmeYaQiFpD\\\\\\\/+l8R2dCVoZvPQLUxBDgZBe3L5yF9gZFuDLSMVO36Ib30FUS2O+iJeUzpljYv\\\\\\\/TVz\\\\\\\/4L90n5QDU8qO9m+N9cRguNfTqH0vVhdC6peWWRXSd7CB0joLdOdaLeVlVZn6XCqKchjLzOSBkWWD1+bxhF87CtxN3DCLqylLSUA8SUKAAEK6uY6RGodvtbOghNhc\\\\\\\/qY0YodqccNL4+sYIoHZ59PaZ+iZ59RmqkRKaFVAgHsLytJXsorsErThdzlCfFSXHsyXxcpM\\\\\\\/mYmR9yz+KHEN88TekF86CJqR9WtHUvny6b1XLL\\\\\\\/\\\\\\\/Z+zvCaASwMhxeFJAEhEhZ1jErWZm1pNbrb4pkD\\\\\\\/QZE2hjxX+d6IU8zF7daN83CFWgrjR4J17bYseFgfNcIXZ1mGi3xTt5wUXzoLVP1C2PEpJ7Jb3F44\\\\\\\/+jf2Qc1fS34fSOT2FlhIE0guD\\\\\\\/rNSjh3wRMLtc8\\\\\\\/Jx5KPX+GAyz8tEppiY\\\\\\\/BzMmdKL5x98e9RzuNnVIWDDa4nrK09t388EvErdXkOLy8wQn\\\\\\\/rm88p5QLRWR7t59gwcXgqL4CKt90xg92BK9nCt8M+xo0ImpUsoT094UuJRZzFO63zQZ99Yb6rlRPwFKRmrYgRHvUE+pGDG+Qcv5thuH6mxsAkZ3SYmqTpqz1UwhwHnWcMGQGFFwQ47GCUReexorFB4XI14uSLWlQACTJgze7YfMezKm2pZU2eFbD1WCthZ\\\\\\\/C2MHCSdvjxLCUPKL5msLWk5Sl+a6ivRGk6RuUWhq3SiY3Hf+LWj1i1Pc2aO4F1uPf+e3IQvCcYiAibKnuNd5Qf5sUbdI+hiyxxt5ByE8NzUno\\\\\\\/vL00yFDfrVf7vblU6Um+316sAH5dNZkb2DoWxlXg\\\\\\\/dTFlXI+zw2dhHDOVR58X+o\\\\\\\/ticKt9GYTBHJG1sCh4gVmNfHugHSVRb7XowzxwHfmqh8SngNWwvCw6QmR6CODPnqmGqrZNE7GRvNPNI3qasb\\\\\\\/lQO9Clw9ue\\\\\\\/NOwzb1glphu0JZglHk67Cqz3Z3YvSBiqTr4Ap+Z7jk9jiNad0jGhjMsH2ggqF5IJq9PQ1+y5lgRWD79xHQYbbRvmanc1sUtAUs2jAr5CQDbViLp4bICzTXtFvpSTXMflOl2BqsxZFHH4nJgyXUxYeJSABIOuIpHPwCOGq4eZWsdB3qjrAXuRKg4cxya9ri3r9hAcT0nmQtGC7KcSeA3Mydia5wrXrGl5JDyQsF0heELyQ\\\\\\\/VcJYuwe1woH2AYtEVLJx0XdUJvpqCTJeq1t6HPAAHPdZy+fheXOcOIu7CWnvDagyWDVfeyAhFfUza8rEgM77O+Mp6ltyzYVt4lTqoHGlCvLPgZhYlxyGuvOkuR1i\\\\\\\/lnSGOMglpuSJom1osN\\\\\\\/0u2GXjMTGQrxXnY2A5vdmu\\\\\\\/MIgQRkeT+btcjckyxntaQrwMYGKxiv9j1XVcwHtC+n+yO1Ya7914bpWq9VjZoR549DgjWw\\\\\\\/GIVSo+vxvg\\\\\\\/V2p74g+nKphw4vhYiT3WC0mNfRYEKRtM2iZ5vTIJUk1\\\\\\\/hRE2VOm1Y+Vy9zwLp3aMDcNdzyLC0pZOj++o3GmkPVipaU9s6NF8BQ8epWI+qh8D6BDDTrz6ocaqe\\\\\\\/GVqZFPsBd1OfQsUS\\\\\\\/Dk6TJvaES48aRM\\\\\\\/vqdF+YNQT8L72yrN17RlNgo7hz0dBhUf13spEa+X7Dr9inhwUYy160Vg58uSYlOCqs42YPqPJ6iYIQyvKq\\\\\\\/Z8F9nbP4VfX6AsdZbXNaHHMddSn3+LLppeekjbpLKv6cZr4gr2SUKSdDn8vDE+jDF0rfCYCCAi2Aq6rLZNXKGj9mv\\\\\\\/QEk1LBvJOqJDWvkcgXpeRvHa3XiNQCvQ6oW8Mt+i5VWpuoIBVws552LpQZU0WssEEepghcr9L9nP7miFIvpmpe9MWwt09PbnNjT1IaRGvTIy3zZUxQFNdcDuHJHskA6msP331JDFjWQxiefGacwNIzgj5uHhY+AxQpjq0s4VWPrBP21DREr6qhuj6n54QSG2YMxsxaj2ndVWL\\\\\\\/3bzlsTM5ZDHAn6ip6M4tNwjIQWR0FWQ+rQ1lIhyjJcm0CX3vIFZyquxMPyXVsYTfiytUFwXNxzIT\\\\\\\/ucgYW6bryGmMayJLC2Z7h9vDLM4jMRwLcLUHj1Bg8piiiXI9UBcIPBT6a5JCKNBmf8xQVSHLMiDdWvkYQsUTOpRBOoU+8XgrsmJHTNrgH\\\\\\\/IMQPK6nbGkXL7FJl9NbWDT7fDhrroUlC65yTleACjmkgUY2yPIqRlci9c29O8VwpHhBLH9CvMyOjcTTBD33EV+fmL2M3RLzJlaDWcBf8FeY6RHX9sErh2OxAyCHdaaLMEl1Vjh5lDUYjkZ0DPuTEwlyUmkK0gicA84Bc339LY9C6kbBn+yA8BWc3omn8zihxgiVskQvAzW\\\\\\\/ai+TlZ4goS4fhf1haklCcw20E+Burhx8WiA6B51XOC3Zs29GaMQFf1FZ\\\\\\\/ffa1mLR1TCvx51HAUrfhvmFk2OG9wr0BKstXrlgOAWZLxgZ0a1eiDO7pCkgSPGVSLjgLnqJR5f4kQe+p44obKjTJiIj1jffAcOgsX\\\\\\\/ZN0DBP1NWt37YrYM4YMuUKSS7rk5bbvA6X4H8j2i1FEIAkPDhRblDFpHbPTUk49+4+\\\\\\\/xARSLizIqQEm6woiNG9HH5PrOrU0RVcswp2s7r7iq3+2sbb5M0SxgNacljPwxfGqgzsq+7gLVTwkxi+JM9ghfM0GeNPVxKuNEsyGr\\\\\\\/zsDNmKBQ\\\\\\\/TEP89j6s2IOMzvSCFv+EvJitSA0Jcv8BNfrbY9Ryl9Aug9Nob1tLgPjJl9U7HBKtUJ4frz0jcEyvwHH2TmozHq7VAw9nTphoocvrJO88IZrno6ItOYaXslF6q4FuQSku3EN191ICY5LhSAUKmSaKJnAJKZYsP256Y2ojyxCJaQX46v39zJnQ2TIMXwuShWGvY8BdUtRmOM+\\\\\\\/iwlAVbRxPKHoe4imTN\\\\\\\/PfjkzxeVb4W43Q5SEO1rUQdqCjmRkuwDhdqKMYdA3T9h6bipglqeF\\\\\\\/dnSeVGM9VlEkQFaK+\\\\\\\/BJo7KqMBzANrvQ+FVA1mN6kimi6dcD\\\\\\\/lIl7QwyL1JoPiQBIvEX9bqYDxPlrj7\\\\\\\/\\\\\\\/v1TdsX0RrVAULwszGKI7ksOVi9FhXCRFcs7ooQX3h7m0YRC2Bt6Vm8IAslcbrLv97tTweI4YkTAPC227UXvYIvZZw+wV3qFvJY5oI6wqAT3NbgrwQlHB\\\\\\\/ahaeU7Dyq6dd+04eACXouAwXL1ec16SOapNrS3kTW\\\\\\\/UFfMtLCJl8AGVwFs3QT7oenHvUS76yXh16sGIo3bSVeS8KB32UbWeS8gJiofFZPeVr+Jp0hERLW1i7\\\\\\\/nZQ4OUi3AZTDKziVRHRnWmvhEGOP2EgqWhRsQAC766uAi+8PdO4xScJ+cYWKQR6jcMzWdwgBPsnSaCFd9jppRzFs77hnSjLY6F8VsMlVjgghi+BPDQWF9zhV4cMNaSmZVAES84TweRUDLkPf\\\\\\\/1kG1aKLf1SUnvouBn8QLxZdbGV623l+BxQ7DHjinyrjdWcarybuvyrM+1zRyoyKrC2K1Hjfpkkg84OHV61GdJzJjBaP9xQ7\\\\\\\/IE3eo\\\\\\\/+9c1wq5G9h1\\\\\\\/M4XHsGuXzBPPqUUdfnr4gmyIU\\\\\\\/cbnbrQLHqp76j4KhrstKXTPBBpeXaI2fJmx54HpQW0KpEcXBPqYY7fmgw6z\\\\\\\/5rMMQoRSuY5hzRAUdZKAIPi+P86NGnDnHiW9hu9BzMuLdG3Buo9ID62RrCXcbh+led8OmdS\\\\\\\/ekXUe0ubh+H5\\\\\\\/z8Wu7gtBpHvysN0qk2AOCjvljWJgNVQFFiv1653n0o3BL69rPTOGMe7efpHQ+WzkVrY2ICquJ\\\\\\\/GssD18DE\\\\\\\/\\\\\\\/wyj1JiPMHc2wHWGPuimsm6NsL7ojqvmkByIhrfx1YVAbpHalK6UoQCjVliXKqfc21Ei9gmZzMZt+CyNkUist+vzhvW8GYY9IXUeo8jIVY7CL2UU44+ygdHjUdA5BiY6KU0qGSnok12KOpziTO2r5Neh288YcLNzERm6JoxxP663UPLxv66Ldpq2yhozt+qi0JoIaxMlxZEGHXQHM8tP6D0M+NwVDbAY9v4w1R+FRNzPborHfiG2OSSX+3fF07rADd\\\\\\\/XuI2RfwB4kxCw7Ojv\\\\\\\/YhdlwUE87C2tm9hj+Wh2nQBkZcwaT2x7gzx1RjTax+d0WOvVnQFZmoJAg+8\\\\\\\/2CwBJVO+Flx5RDFK1nWnaDUGjx9s+7cu9gz\\\\\\\/xOAJoMplkUwiFsptgjl7HGyPRZNjpkmZE\\\\\\\/YMeerutvM06MGhW2z436W7tr\\\\\\\/cUXFfIbNtId9zICWIOKinmcvZ7lHFEibrWb6oEH\\\\\\\/M0PfjKSgrJjVCYIQbfHW5aHqzm4WD03LHMqkucmC74JKtr61bAN04j2F\\\\\\\/RiEQwOu1fLQz2dkAqEuH58ve6y\\\\\\\/O5aQbXETv8ALZXEmvQHihdIEO8TinuGP7C9URsW+px2HHlzn7ITFqVKUBndwP08YxmpDwNaGc2hGjWi3sKVQWf7hbiKdiZwIEbDAhXqZXSJMHw+xDUQdxOzSgYfqYG5Odeeh3Z5cgRERP27GLshmjqhg2xEN5QCiGTsQGOjZCFfhUvGTVlokJ+3DB+m9zGNpgfy57fwtQ4uwb8jBUmduWOXqTUBso2NqJmPy0CWMH3vFgXtaFSoixqr8shiTitSJdHMiRlxo+0htjHFbwID7xaTWrO+XVZtFEc+Cy23j+EUmXUJh6hktK0lo1wm7CvAXTaN\\\\\\\/y0QJnB6g0LxsTBshuqlmfInu1\\\\\\\/Sp2Eepdr8KBsZpn0GcUamcwwktYuxamIuy44nthzA8L03in10AdefjooITO2ygdO3+ZP3Y1Ta4IJbUFSlnNWwC2mlG5ZSuV\\\\\\\/vwUieAqzfyn0xTw3gliglX6y\\\\\\\/plpB2Ula1cYKMD6UfD7sbO8\\\\\\\/iX8ApZ1lUS\\\\\\\/4g6gnqrPApTm7UPOHC1rD+2gQPnHkyJyla83SRRasyQ9A2d+BeOtV9CmQT8UfJ6mLU7RRpyoHKdeYypjBp8OSAqov5tYkWR8nEgZ8M1j8lZSbspsZkLbOppG443jqyMDg8i+mgVc3gwBzs1Yy2VfyR6Ai3mOdq98isZpx9jh+EeExi7QuDr46fpnYfWO3\\\\\\\/T1k3Ack3qhZZ2JoNbXHgnCBGERr1GRyq6JOm0uAIYVrDpg5V0ZGAGYiCuPguRO76ndwS0gEuqobnRM4MHhllWBkXph\\\\\\\/A1CAbvmbF\\\\\\\/NU+\\\\\\\/6h8n9qttGJh+Qkk9zYEzHGqqgNOuVD\\\\\\\/ofbHixyDB7eNdhpmZdSJk3TT9aGSZYAHBXFoxkx0kjMr\\\\\\\/PMHqsmENjzr6G8dOwAUS46\\\\\\\/B5FBHYAUo+CBKS32\\\\\\\/IRjneK3QF+dPX0IiOjgGOD4DbF6PgOgZsb2nXGrnJgcGEwB01LiU9OLf+0zNlNyOjq6QcAs2khgKhd1XqH6+yi5aFplHIXW0y+a7J2h0WwmnYzzUh6n885C65N6u4qVsfyxQmnkpiQwYiKEwdr++rRbbHjb370ESw29yv2rHQa9J5exhTVDMITaLT\\\\\\\/47SpLMBwAeo1czTTK3+e91zcIHcukURLFNe5sCbRXxOvDt1aqbPpVqXVpVYvT73Fj5SLU9RZpRSur4BjueUCNv92tyqABvWxmM53fHvSBH7Vqo\\\\\\\/9WRt4iEeFJHzyOMNhvAWceOmUrbLKKlXJHyV9bfWLTjMAC5FOK8uwuzXtUQL2G4lYOXUcBVrsKSbfUN4+84dNKTU8ktrlt5HWxA8PHm71OdtnLerbiCwsXV57SX7rpT6evZZqpkGBx955kaJq91Pjjdgecnq3goaXu9T89zziYvYPhyXiBzBYzyt9e8VOzi7HEqSQThzcSCKf2HD2abkIEI4OcbS7uvZpZ50RDvrEgNlkQFKa71+01XCISj\\\\\\\/U7rW3RQbjIqQS\\\\\\\/KIaypMsONuwgVgz\\\\\\\/9iicIr3jXlzau0m7RSNDbYjxcZ4WJPK8VJBKKqNQl8De0zBJYfIONC4RpO4roq72Joo5UJH+e8fm6vX9rpm09lgABQQX8gfpQTJksVg51GQtoqKrTmp2eRBr7NwLGiToWZxrZY+xVu7nKAVsiBJoMEs4yjsQGBwv7NFESz3XK0H9kNNEkZ0\\\\\\\/e31CjW6cJgg\\\\\\\/5US5ZLtgWdCh3Emi7XpI95JTpDIE5PhqyIzEXeM\\\\\\\/StL+Ck\\\\\\\/o9L9NgNMIQIFj8vXt+bxEA4j4WCrLDzXKJ4UQ1AOuFsXcYtpYg8dGz6bSjXwDolJJGlD414U54tICqY20nzV8P9buAR11Jkq+oONiZBKZmn\\\\\\\/dDncX7ce1XWLeuyH+YKI7Bvd16N8Gk7FCf8FB3yvtth2u55VdKr4VrhWVWDuk9E5+yxtMs4iSsHIba3+spIx4X\\\\\\\/NtnAo3QiPtfKUJiKCmobtSWxok0YwFXFXiqP\\\\\\\/jHxTVd3PO93sCFA1kA8s5Ea0bZ9yAb\\\\\\\/CQLq4msqj8DHBLWhi9b\\\\\\\/+RV3AmtV+njEfkiHiNzDrssl10ch4rbWZsCoBbK3QN9lUDDwNHVo1L7KW2ewk1FSg9fGsQT75swgPW+pCaLoKYTdHJgBisXo6p1hW3tmyZb0+xjSFynoX4UlEN6YlnbdjcWzjSJWAUo\\\\\\\/rOPf6s1vb3XlfQnc5zERLUam7ttp20oT5hTlqnHl+6cOZX0mLjA1alYhNGoUg2HtC3jZr9sxXFogc\\\\\\\/Wc\\\\\\\/jxYvXP75r70ylLcNnq0+SumaLQOfTc331YUSWtlvjtktq\\\\\\\/VTwix00wLGcydDRrEGcy5h7o7\\\\\\\/mvIpAGKAFty\\\\\\\/MerNVhF\\\\\\\/3VQ1okUaJemWZK3AvE7zDuB+nU3YFg4TyMD8mMx5N\\\\\\\/WgQGHnzkTTkqnbYlFXPq4QC8uDh6qmYcie6zxRzBsCklPmBRHjj5IeUIW57i7XNkTDU\\\\\\\/32SctTTlOBuK0EiA16TJv3KFUrjs3\\\\\\\/n4RPC5i4h+yCE0OM2jwInyn92vywN64vnMwojFF4Ep9BHQCm8UlhXKmHOM6prxNZV9zGL4dSTMTs9EGIguFeE4Sw1N8PVs6FGU9jU+BjuLdlXBkEKige\\\\\\\/U4QXsrFhpfI904HRiwsLsqncSkhYIprrkdsEDh1z4LoDSQm0oIDes2EsCiwpjVFwug2nL4SHyOs37uUYY7m1dgUVu6nx2SZ8Hk3E5W0rbl3oAX8BkW7XaJQdsi4eQkumHePy9L6GsXG6OwQ2XLt1u9dpMg8+zGQaWmE3+iu+3sHvHy04eNeEUkY3FkurYeCNaenmvlOR160SQrU8ooWE+ot8PdeZovI0Zu0aIHdzvnMTaXPrOcmf2jtXvkAg9TzqsyZBg6uCvW0yZ8S8pFIrp2z4ZpFk62swfdK\\\\\\\/VWJMdTd1HMnd8hxWwNXCfFNrgrAxS+lVF9qI2\\\\\\\/W07DwCcPYw9x7pE9wZeJmfR6AleVEtG92mfXbJ\\\\\\\/UCK+v\\\\\\\/6o8AngM0qroVPCq5rw9Jd15+XIkY0xmbiyOgYy+LVFTjmkP6faWx4FDCk+evLZx94CM4univvmNUx7gw+dSBUd6RxglDFzElSOk5AUsSm98GYY\\\\\\\/GN32SFBLohta2XVbXc1mhnP7UBiodv83sNKlMD6RwA2AqHXBJGC8qNNABvxkzvwge9R7YXqsnDMpcEmx18H7gbI5ylHbmgh4RqA2CRYg6smaAw9NwecMbtd3FfMTbljyl0rcjjCB8E1QpcpkZ5kCjVA28Uljgi9xVegS9xGzD37ZxNwvdlv\\\\\\\/jxP+rJbHGt5AchegMyv3D74vakdkvZOQvCVjNG0DWQW4E8xUH\\\\\\\/vYixj87lUK5LV++P54pnVosK71Eagc7m501FQr8VUBJkRqWiLvjksUHkauYXqf2nTVh6byZJ2wZQvzkzmZqy8DaJo6F\\\\\\\/8K2\\\\\\\/t7H5PspYrf\\\\\\\/xjx+eEHRpSNBrh6HwHwttXhINu5DdfULYO8CgL2lcgvN\\\\\\\/nI7GTfbL6D2UUYgjJC3i1SZVJJrOMMI1OUxSqnkks9PDTUkx10u6t\\\\\\\/xGMT56rSqoLmwL9wf6snSsD\\\\\\\/7\\\\\\\/py3qEQOk+YHGO8zJmIPlQFfVdrJDdsk5h\\\\\\\/GL2FPAnxzAyHRRmO3uh9j82n27QaYTirFwQJ1Dxw+OQveXuQnt2mqY46CGbx2Qz\\\\\\\/t5YqItgipGV3seO1RC4mmQYXRp2iMw5Wct0iejZpOmkUfT03XSohfvZgzL0LFMz2jujwrTCb3qhfKjeXyv7Sbi24vukmvTRURDQ57pEv2skXzet2SeWtPJDq953e0gpNT50WLGIRF7z6MNI71+tPAWiPN9mQg7XkUjd0fbOfPp4jOveqJNzTHrZUqaiv410+Hk0PzipRHy08alpX3SggvFInEEyx4KNzwBcrwFeONBtobNEkrh2S5yRFIsq3eBRfHn0WPUA\\\\\\\/cvFosvnfbx5Q5x0BYwZQcGw4nYkK9g3xO27aLFxuab2S76nN1Vmg2mWitaz1qCQzHCi6RvkzUoQtcG3D7NPtQg+B1Fw5ehkJtxdgfOFjOfewFRvgot\\\\\\\/DXRNjnn+SG5WHPsSYwNX40P91C5P7qz4LCUqGePRkePmD+oeqb5FpCHvotqGYcZLptkgwsXMvk9mjbogMggm4k2yJFvOdhIjhoO2B7KN\\\\\\\/XN1IKPx0iH33+63oDGjlf5t+WVlaAToxxBy1YASGyCl4v6C5TE+TIRFPYxbXVO21xJY4GWt2UzZbk8JrCxbZ5CTEgp0Y9JUR5P\\\\\\\/PD8J2hyE2oqs+21Ot5iOcP2n7KSBDhXhmayJnhgdGMVJ\\\\\\\/Q5fJSblukfvCYEwjikR+8zh49bcwP\\\\\\\/saxRjCSHddDvX8wq1FjktL7RKo7C08Rb7eV0jfthaHhxRQNzjfjchi1ng1lMb0uBWYjhGWzKsic7\\\\\\\/7iTVnyTVN3K2FBQaPnuhHyC1hIEIWM9AMS90AWSFiTsztMFfAGVJ\\\\\\\/wV\\\\\\\/JWJBlfuaMw\\\\\\\/xJWsTQTI2ZLFhvGyg18YsGsA9y9Whe6BpMF6nMsn2VVdzu73rh+BsyZjPgweQk36Rb\\\\\\\/9SEbRM1zbFT9ow9vPEiU1e1XPWfhuV14nJGNzIxFKqtaL41ty7Y0RZPNMyyo6iSxE3B9Xc5+XtRt0oBMZDLX2ERUuiXN0PXMEyd2yz4THqjiKuiyiknczfZkViXG5Z6ukZLb5F3BmcdwRt98LfH2WpOcUALYwUiaDOEMC3efGQlDzretMzafSUD3WABAEDM5uyJNtAplvjhCgiqQoLs+uPbsNX3GQRZJfJBNxVD4hpdxNolL0Y9rT2UnYfCgWHil1LhGx+8TETGjdV1ehMuA+asUNktbNTxitTfdbMrot81D+vWCav1iaz+LGSyZQnOoNNvhl7ZoYdC8bU5AlXRc567\\\\\\\/8FBDUlSttDOvE1L75Y3zlXUP4eo5nDGeDlL8Q1Ha5qs0Lt8Ry1VZ4IUecIw2t9xIs7wg9HlfJP1NXd4Kj+voJdaS2qSkvKoehY09y3xPwUcehObrMbamVFNT+8BuG73tB6kBLRtnMsnRYgrxZDmvcztasP3+tDyUpe5mO3s327FQ6FGpxWG\\\\\\\/SQAnSx0t7V8b3ktTfxKcD7dO5Mg+sGte6a4VYDV2wpF4i89CSyI3E0aDWL0+2fNaHHwK3FJ052ylwJF9MHO6TdeXzSx3l6E3t9in9yP773ngE5YZOvl\\\\\\\/dQ3GJ3Bjp5JDSZOwdJTT5P5G6s1VntileyMV4ywcj0kj0pz9h7XQBpWl\\\\\\\/gZ3B3KpBpzIB+\\\\\\\/qI7qJa2ewwFYrNQpGyMbdLXGcmqVcVgXwytUzmSZHjmOtV8zlIP0ANh4RnhQPihIb0fOPzMdoF2rNDOcz1W4F6diHH0bASUrTyH74+2YVJadoYPRrHFVWRl9hG1E6dFpJzs77nfG91T2G2Q7ISfywz0vge7pfHjG+Z9y2mM6WPPIyYOwcNBsn085dxEf9fJNKXHmeqcmHmjBsTbFyrEevyoulUjnaomqrW\\\\\\\/n4MhCFhSLDMAd4nwRdKFB6GC8AYL6eruySqemRHAKLZdkTe9jjng3Guo+W7yHrLl1W5Nl97T0wtNuWfcqVkGsGOg\\\\\\\/oq9\\\\\\\/8GRtQFolmDic\\\\\\\/qPsF01\\\\\\\/9T45zTpt4uITfd4vKGNsF+t+UGwN19BxslPzLwcdiGDYSXjMk2AuDLnopqFdhUW8kyzWYdax23PzrDEAYmH4VRRZ4bf91Iy1Tw5PpaVOSky3jwxm6M85GdDpHsOzblq233CwuHQktc+lHGU9PEFFTpXD+5D6gFSV4OhXRbB4oQ2cNXUDFHa9\\\\\\\/e37InSP0DUU92jaPLrYsdtG5IPYUa35v9XbxPNJDuh9OtZ3ZQAYPJDQPQWIwlCpRcidOCUjdZ6sTHlCEwDc7FTS1aSgvj+fyB0OUthG4YBNqmke00xxPzXm1Gm+S6EoAMW71xmPWW4LzZH6ruWQUlm+efKdGnRhW1YQy5b+0Pu2ccRhSZgjiE8wAlkUlGesDn2r\\\\\\\/E3WgNR8if6tQ4bWelduOs5k7eQNKk2tX3FOq97BVQTSmxCHS0JADfeT7g\\\\\\\/AcY55oM5WktzSdv7Drbq66lGxI6s8I5d+itDQL0RsZzr7wkzvX7VJJrJEcVxH7MJ6YbWIBLJZvVPMEuPXq7sIbcSyRcaQ2hXjsiuM1hjv6kY7YvasXWeRvy+hXLeX4Ei6brT18HGLUlU2+RN09JDRW6dUyjNVfTcGVwZNPfxPFzSqjgoWyhRpw4WGhO06hu5TwcPCAHDj290xARyNokgTwcAi+BSsH59L8JxULrHv8bAwsjbVLS8\\\\\\\/kqqcblRkz+IyvF1ZtyZGSpgmXPvQ8Pe3XhbsC964TV+TsfyV6kXcjd3SNVqXZhEtUYb03Uan8IwSKBa4xcyPSxzlpqfq1QHmeICBd47QLQjFL2FQCOef7m5ejDmwdSvTBLN0bnUz2KSU2RLhCubptQNs4MzZ5LkFUFbsMSp8c\\\\\\\/gP+B6bWMZEgBX8b\\\\\\\/smlWsZ01AkyJtLj7bmjrVoffuZ9hRj+2pMBPUAkqKGvx05X2P4GVtCVcZzp+9kSAe1\\\\\\\/OZ++y5G3lYhSAoX+8mUhfI4IlHV3aDXeEd0X1iAFXUxoITss+ws166TPd+2H8kmmM4FBLKT35qaVvWI76tgxHpBGGhCMZp49tcaAigegfcpwOXfo1JC1PgvdKhyRJ+Sj+XyfA2cYmVjQhYwsoUMmi8dup6vv1cZMxA00fdpfDLAg79sVHmm5Dp+0HdHIrCEXOgb0EBHt6ZFmqJvjwa\\\\\\\/xdXnxBIGo4EMAhqk6g6vgf1cGlHac5MNYTxTEMRSweWrtTv\\\\\\\/ZMCrGHGmX6NzU6fVti23UMOps9ARW0vE1zIog75iRVp2595hg2ruwTfm1PhvF+0NpuISSCVoBmr0kCg3JXerIxjLVW\\\\\\\/F0KAjvfmKyREWL8oOsR3mFHJY6gocWT9HsC5e9er2McZDjgPzzG0Ja6Csy3goSe6+W2abLzfqKNanxv0xYCf9VxOWi263dbhi5t59gN\\\\\\\/bmk\\\\\\\/6s0Gos577ut818cmrD2ZssA5bSeuOTmP\\\\\\\/Iq8fCKAuRjTASDIbw7P+v\\\\\\\/NQEYRbIdpxcCWkZbuW4vkG2ONM\\\\\\\/GSGARmlKRHCBllsC6ltVZ3NqhUka25HNvgh80SZnBnTOJ951+enEEgTyhx7UYvPaCgVIGcVSZKSDdqMdoJTFAMCDs3RF3\\\\\\\/p7T3BfhVnUGe1YGI4DkjTRiiCPGlg79TgBBgnkgPIeJCHiZbKshaLzDGKXra1yrma15tMifZmMJsufByJ4DRMnf1L3JsKiprUbJ4rag9QXCNOp6JUcdfUsyGgO0DRvR37OjdCdr6esV24Cr+C5Fo6itFMSWQ+n7pErJqEA\\\\\\\/Z3aWj2eYDMlKMeNfuS7o5lqqwY\\\\\\\/f1qSQGcRuSL8MwBBvcsV9KsSFQW3\\\\\\\/SyZdnvr2hiclWpsoesUPZFw7cj0UyqSSTV03A0GS1pH50MOk38aX8x55KSnakM35bDs2fEsD+YyFrKklgXMT\\\\\\\/\\\\\\\/kiphtTq0QlYGgu\\\\\\\/+FChyerCKYbfVoCf362nW+vTnUO3YgEFoufm3uaIyB\\\\\\\/ixBImD6QxCIf0s1QnlQMrzeLL7Oq4Hu06m4aTp1lAC7R6REZ4KdxjXTxk5I4+Tf5msvrFGX+5KGOfi0HbtpsxvkDpQtqCTT9Y4oljTxRpHj+gkDNO1Aw7aV\\\\\\\/EdIjy\\\\\\\/UPSw3Y0TrEfNwgcSNMlE9YbFc2xBaVFbwAn1Y+2YL5WpwB9qeE7mgwvvfyE3a2NcdD8c4A4lc8IUJi7r6kfSmMzSijjPRqdpTSM039z6LM5xJnTzEqLKmUNnO7oZ0PnphC4hBCs6xrbLorxP88B8HJHlR4VmdW4xhHeZA\\\\\\\/hriay3QCbR0Vg8xgDRo+ruDqsuhlXpWlrVZK0UhH+4LSZPN1kFmn2r5OBzh6ye5r\\\\\\\/RBWWW\\\\\\\/Nmaeok1yhpg\\\\\\\/R0DQYd0sY31JOcfnP8ZN6N\\\\\\\/QOuLTlOmE9BjJ5JgPpvrTTK8gNaMcEnz5T3Ghl7XJjjoBpv7K8ClmNFrARhaV3GGq8TE64E43UGvPTYw5o\\\\\\\/0mW+Izu2ilOoF0mNX5de5PE9LYvpMt+mlE57NJ7tqTz9s8NpFYrO6BUnSNeSfaRe9gm\\\\\\\/r1myGgxX2h5JyoOlMNfMa08XQBl5XPwV8mO8mJLiSQPN4+RNNfzcda3hMy0I6q15eS83osuvMuXP3fm9uk5bExVKREqb5CqxqLs7GLwCm8b3r0H5y4oHNfC4jouLOduINP4cvPPQTvLTbXPhUXGKcW+KZhiFLmmzluAmmdaoYZ3W\\\\\\\/LuSrX8dmIkR7zhJpZ2gqq+BcUS3To\\\\\\\/C6QDcdnUE+YM\\\\\\\/\\\\\\\/6ZEVfP9+qmSpp1EANjDqIGpX1J1CT0XDx40eep3QjfC7XnfP24tp6xkCSCXdOsqZTAIDEc7lyREnPr9rLLi8dqzl78tGKantePxwbz25920Up9pPM\\\\\\\/JFWqo+iiFyMnGjM0IGYdntSNHCXG+5G5d7m6qKDescUcac\\\\\\\/FIn4ErUq3mNOMR8Nyajnzu4REjwZT6gbQNtis2GzdaiSIKkGqDd469m5i3\\\\\\\/d1v0B0NQJjhriINwS7YTgMF9qYtUBgRrKO\\\\\\\/P7vlgPYJ1vYjKe9brkUzDgyYNkwaMzAiuVXMGh2nE8sUU9K0oLcohrRiSAACRIJ9W1UiwpzJI4AHQ+grxwTdAh+Vyvw4MOYSDsvNmCKilyqK6EgavVEZVQqFZO3FU7wUdKljyo56DHrlN+rc+xiSt0ET6fwuUiZZuN0QRlDtAjyokHNQfVPBYA7uSKT2Zadj3vcOKvo4+BUAqBoTx+ejGcd8rI9+fOYwkTBMa27aEwRo3LB22yYZDXjDSAZPkR9D7WCk0pjnXq6On5EhPHxu8AFqDcUWSF4Zoi1d81Q899RrXC+pWO\\\\\\\/Ryg6mLJUTeknHl6FUaFkSd\\\\\\\/zwO70nlsrG2TPGLPy4Nqo6v7glcu7lNMcF2tlP9\\\\\\\/EzHQiyVfnepzedNoQnhg0wYIK2kY4A8vfwryfCwYjwOYE4QhBhjf9DzdNo07Rj0Z8g+lr7Z+SyiXHdaJik9MwscuYR3jfunUw\\\\\\\/1PzwlH3kqfb4q\\\\\\\/7QOb6kRQE7awcmE23stJoMKlZqb+b2GNfDPlnDWxXf5tqWUwVqQFbfqctcvM8F4NL9Y7F31tvELMBqrJEMNwduKqPimNj\\\\\\\/Cc0A+mgp2UAmSEjUAj+hotAFU\\\\\\\/Za0cCdU\\\\\\\/Yd3UFETIw2cUKVfUz8v1uKEFSbFMM73rWhwhp60DjsgMHEfqQletpxlOYyORfXmxPGVEmHQ1wLi703HYEUBaO2Hy22gLv9LAFD6EMEw5CvOrF6j2zmatu4tQLk7MDW\\\\\\\/C+eEHWMtwQXQwcXgwmgWsqtIYvyUIDyzW5MVoYO2ggqavgSLhOZAkMLYrg6uWN91l2jZoZ9NXe\\\\\\\/aiAciilGctesBVmkXJ+qyj55+IvGF8qOvn4nrkJNATh2RwN7MPU+DNQOyqNGlIMJIK4Y+Qyfw2MjRf1+S1KrRgVk0gYhopU91LEiAlMFzvdgGIptR9RnvQhk1biarcl3U0zZg3Q78FLVKDvHZ61M2blWYQ\\\\\\\/\\\\\\\/VJLpTNNHjqzQ2fJQmCgN03HctvhMasNDB19TTHr\\\\\\\/8vbWGj\\\\\\\/XbcjkwvmcOop5fjKuOnFxSq1YJ4pstO14DN572J47nX91fyN7HXF5U2P8Veyp9IsrVFvF4vklSJj7a4pJO9cWEsYEOgMvW8uJYxtW78pKdogijXDdo8OYIPzI2EBqlyS3fBXXB0CM\\\\\\\/n9AkJRGdySGoPqRTsbLX424WRe\\\\\\\/lF52D4MfVqd0n0w6nRNipRc4ZWf6Tmz5t9bxNqqsuNKbegbrnuC\\\\\\\/Sxq1BI+BLwJz+Ckd4HieeOGlpPETPYJ4oBcRwMOYBExd3vp3G8VjBkAWBZRnGUiDrFdd1saBbPt6Gi+cRYAb5Z6AcLzaILIXKzKR9ku7V5SsszCWWm6HmUMoCu9MoKluh3J3PM845bT5fs3srOPN8xV7QNt4m2fgG7kNfVO\\\\\\\/jR1rGmq5tnZZMlI7uiHKT66K6kHrHmT\\\\\\\/yXQaxJOP03MzrviGRUXPH\\\\\\\/sgvdlrRnyvusTAMNA3qqUZaESqNETs+ziZkc2kBOoeIT5H\\\\\\\/egZtof+CvREkMNf8\\\\\\\/f6EWra8yC+iJ57zFxYUKYC5bQQI4KT1ktVFThhetOg8g4qQgLnNJ9akI2yG4ceGV+6WatSNi3wsmNH7KtcWWYzp+BnRDBZLnXTcEZnxQLld\\\\\\\/oNtkpMJYeajJqFSUZ8oevey4E0uAK3lWu5N6Q5i1VMhg0TnOpEmE1EcK\\\\\\\/Vkvlq8tmgWE9Tsv+xqHHoTuu3u\\\\\\\/3Gtl5B8OCNkzCaehCbGM8M60g9\\\\\\\/HlPbM4KBVQFtXG2\\\\\\\/SmHqgc5SoY4GTFePnNY4tR0n9Y8ui2jGplQLctsF5nlq4fUUDg4g4x\\\\\\\/GhU2hpbsSpBEjpeVoCWi+oOy3bePN4eCcltcR6ZSmpSKlcAXhUWsDrepnZpfTW81WStVJdc3LmhcKcX55EC0QzK7yGUbeQFH9lgLawTqVntCY1VGt5QlFFPLcA0pQLVyt4gU3SmI1TgnQOj6oW1+WCBX2P6HlCbE0TcsH2PEylqOPa7krtacgs\\\\\\\/lYN8cM3e7byWlpEldycdo2v2s7QYzrl1NcvUytWJbggOZCUW2evTNEIsharAjEuI5ANueivThBMQ15GiJ3WgJVmsv9N\\\\\\\/z3krNBLUArZvSrtsoQDnS1YdjvFZk65GvEGmgfqvFJHd2ED0wtWIir5A4wPkFu3qJobAb4OTuuCZGmFHdLqwu25ztja7PeOc8TLE562LgU\\\\\\\/zE0+7vYzUV+lLCJ5qtaEvNVaGS0+MIUpha3\\\\\\\/lZgVhiWZbHwbuVHV9zb9dnQlPFLXD3WQZ3f1pwlXWH5x2WZm2TUDKx38UuqFR9qiYnDjdrsIE8\\\\\\\/\\\\\\\/KY5\\\\\\\/EkXS\\\\\\\/6XDC0DAgA9+VMl+jRulCFHag4TZHVxyhIgraIc5h2fzEMqM\\\\\\\/V2mjAGI5NIl9ULdRHsSmwn8VaehoecFvPIDChcjPx0BAu3O4T40v6Rexz\\\\\\\/K9CmHofS00+z6zFGzaOsOT7W2Wh+k3WljQYJbzpqPBw+zxQ+V3zc4WFJwVpZgqN8dJpJ+WEVn+mHh+9TurpCSS416xJLw2gqMJRisCNfP+gzAtyiySWH55bDQOZEfQ+AM\\\\\\\/p2bU4zm5w\\\\\\\/Ph5JTjK+jZRwyNysoSJKbwCwaspjuftif7o8IRp8\\\\\\\/85CWFWBHWSi12v2ltH5fiQc9HslcYpz+jQkVHAeyd9Ac8ks2U7Z5NL14JttpnTK7V6R5kYnNklrXxRDMsIQsSZ6zc+wnNYJ3S8tBDC5\\\\\\\/we7fyL+mQe1RGjFHrvGkUPRod9GFcG0yqI0tFgWCL0GDwEuowzrTcid15Ev2GxAwfpRp5Q4tTZEOkLzHRLK6gHcbtirh2M27rvCZ2BtnxlGxtivNOGlZeRQehua1Cm1GiGtIOWN\\\\\\\/veD8wVqZJReujfuPS0E+s8bNjmJeR6gacriMd1gaWTJGPHPq4dhrQg\\\\\\\/4AtvFkJSQPLThHmEmhqqeLnBQNpbt5KOQIQDd4PYDPccHCeRRK0Ag+YyRHbUEtgDAYAkysI5XU\\\\\\\/Yd8wUKj+sq6yL48w2Nwg5NIBNqkqcR79E9k31\\\\\\\/pmDYxdmD759BEoJDseIGcSOMPPuuT\\\\\\\/QsecZ0AWxJGSBTClEB3GGXb8XdvHrEFPh+IJXDlbj\\\\\\\/DSdfBuHdS\\\\\\\/ZsNcyegXvC\\\\\\\/r1L4iZo1CQHkSpctG7tV0dcxxDl6IdFxnl0BYyd8JVFb0LcignXDUZCwP2sqkDnTDRqq4epNyI0nC+YKsMYn4RY4N8n73gCMIR197abC3jVXe5Tjo0GqPUQ5\\\\\\\/ezcn+xeftZRGk6oR4NuYh1iS2nJAPoQ9SD\\\\\\\/GXApF3OPec3Qkt76AxN0l4hn0k+RMDa4nPl6RoyIZCKwzE9\\\\\\\/PeO0JHKZLWeNTGB2f05VSKftvMUXrlW3jXh0Vt5tjqK5Pxi661eECa4fa+xrFu2PSC2aRP6HIpFLTOQunQy9wpj42fXchMazkd6wWRAjBSTYE8bFLK32DM\\\\\\\/1Wr4okn9YcQHNrNdjx35urEG2HvPpVeQsLxJ\\\\\\\/csQuhr5yE5pqH7lAPETVVK5DzY+DfWt5i\\\\\\\/4hk\\\\\\\/ks2ueZYveLkFZEalETvW0m+F0I7L3b\\\\\\\/GWtWhAkI8rTaLEC3N45N743RgYVOz+kHdMmb1EpaCXLwe46pwgf+R5u8KOMzKM9FptLH5GaxF7Q5Pq9JmFg5JQN3cxu7pD\\\\\\\/ZT0ktRD1WqF+4f+PSlRAfqk6YVIApjzBU9G\\\\\\\/OTFMwidKCJKIqFMUEwU3vNz6xNWntlobNhmcUB\\\\\\\/w53ThpITUH+EACEQFkIY8uajjZZqm8MfAi5mK6EPORMI+wi6rNVN3N7RzTlRnPFputLaVlKwNVXsAftRtIDvO6GjVnlEb2+lNeEjWoD57x\\\\\\\/0Yzr9FkKx+nzvZUCKQFC+V5DQH6rtDBEPU8G4\\\\\\\/6gKobMUpxM6sItHSKKVavuAsGj+1nPnE19KgQsMz35Kb0dNPKr34WY+2k2iLG1bIedisBfsR\\\\\\\/tzNzN6m8rOtDx7e0zB0lzJrBYAfKrFsfbTjrhVr547195RN12HxOvg1ZmUhGuoFPEK9FFvrVASVAn7bqnLgcMghu1Dgxaf0vOeQUonI02iHVeqi2spdlj6dnnJpSplfmWueKuCvnjjJZJ9W\\\\\\\/ymbA0XhNH1hPAWfRv9FgUh8N+6P7pqLuLYuy3GRlCCeJjWV6FSxjWeGn57XCJs\\\\\\\/ZRYf1F1p3iQUHYsA1ulynheEvmgFPoAXuE2iicps6\\\\\\\/FU0Drt8SCBuw1KgZqJBy78MxaZZFzOdSgTHXWqTN8xXvO5ZQoyiAI6RDYMu80Vu6aLXOujM9izQb0lnfkMKG3H5BTQFA13ZkR3ijg6K1kBhSPk+tFxxJHGUROfr5nJsS0RWYi4uE\\\\\\\/yCDnp5HphzVdb3F9\\\\\\\/E8Ac5qpfR7FXDy12OAVfMtBKo+xz7QO8\\\\\\\/71zLaZDDJBaDG6UceDY9TTasMxUa0hoHNj2XZ2Fok\\\\\\\/S0j\\\\\\\/\\\\\\\/yvY6KxNPZBcJ\\\\\\\/1d8Jr+6cSmEXfIhMIjuWcohWQiGslmZbFy4\\\\\\\/tS2DA5juuvuV+cWYjsUOf0E2QCCWFlbWhxjUumQD73pRkDI4a+5bwLb7e1irD7seTTFxYsvllkx\\\\\\\/6YIIZl4jqFq3H5ceICYoTQgEtBda8FvTrXqkyMEW0V0N1IEy3REdxXXCQoO5oNI2of\\\\\\\/U1kCiZPxog\\\\\\\/UFlTP59\\\\\\\/vbQrK+ev2eov7vaEnirlA84wNQsgQobzCqC3Bk4wx6mslWGMS439QuCuk8+ZrdD8zjQnzo7ekm9qLX3Yj25UcbYAO0q9MGztlCIuTmpMCEy4hS3ch+q+xpoWNMJ+a0g9XXBXFlF7HyHGZ94FEVnvimnkntX7bZEFaMJfeZlGKk0EC9nYAGMqtJkpL5apI+cj1jeqhXGplexUjlNiUuScH9k0cOU9p+JjTmjtOeAaTADDSESPbYk7mkorxTWTONWN0JGu8f5yx8DOinvNjHXc4fY7UOMa3EG7XBAFPhOFSiNoANVm5SQyLeq7BiPi4lESD0hI7mMIibPjBrY+bCJcB\\\\\\\/1ZwJuAKlaCIXOrr9M0S3Stjxvz+kTFkXteyKgjRjpEKTI3fib9E\\\\\\\/\\\\\\\/zOE+IULq1JR133gzeBjZ9DUhVARxHSChcsf4uaC3G\\\\\\\/C2gYBZZUQ6g0mL0hININyWQL7bJrgVn4YNvb6UhaEQoco5F7wfobh\\\\\\\/V3xICCl2RqePwVS9LqrvBge\\\\\\\/oIQpEjcKdDCLX+Uu299Bj7nQD4uzzeORKTZEz3mGcqwaEXkYobZah+Rj\\\\\\\/weiqKNHJwK3Rl0mKbpvIsTxbxKLuk+nl7Jn\\\\\\\/MVJcUQiwVO6qbkwv1FT251WqDkVsjeqGXU1vzqeo93FDCy67MqpiQ52Eoss51WIuTc\\\\\\\/5gonFMddPr1\\\\\\\/oqMpBZ6ie6EnRZ5mhzhq5WA5dLSrmhE144GfU8665UxbpIPSwf9Fllho44K3lOYzKMBW69Q0lKqUfDVJR9G3qAi0pAtOvPn920cprTBA08ixTBkEO0Mzj0XevJSPFui2ZJJJBs4vMZel0kxtOKPon85njCrBy69NtMbD+uWlpqsyvwGth0Td5drGo7blDmQsaZB\\\\\\\/kEzxLKQQbG2TPKCr6+mA0H9ANo3+nrEQuSJU26H8TCoyXBYo1rfVnhkRkXE9aIl2sw1TzoC4EcC2wH5inN+tHbye634sIGEKAxR\\\\\\\/1FGzP6iAfEEfUcMplaAY\\\\\\\/jMfCXnfHeVIX5Aoz6nuvqkavuoSw2G3X8xmjUHq5d\\\\\\\/cUmfyhx8TBzaGOxzqCyJfTjkk+9Xub4fWR+JSmZTvJ3kozg49+V5Li99sROvYh1tu0nQ6GNeotMeaKXtowMDzfCmcJhJwVUdR1NQKThv1rBtHZ\\\\\\\/B852+RjnLZ61iqg6E421C1eE4S3SrCxW2nfapwsgdPhBtx2Kl3fpZCny2kTPj0LKOzEkFdi6ACAgkV\\\\\\\/Qgu5vXDuL53VYT2ZFnI7zKMSi5cqfZDMEszp6wK+lFPcuHk3c2CAVl\\\\\\\/fn3+7fKvCGMZLAGTBhD2\\\\\\\/Ib80hd0Zg8m1kfPuEH8b1Wy5AtDz+bcZLDCMg1nfjFllD7VAwu6sQgrfcnLVEeHUY9K6YY+hexYA++PZ4C6qgnCz8T6c3salfF9YqhEqoTywFlgcvk3Vu1PpCCXlwoXPq5ku+yWhp2NAa3lNq8b2d8E0WXXSTlnZipsGATn94pSFm1VuNHZu4cLi0wS6ZQqOVVOSK\\\\\\\/M3BH3V3vnWVlzew7uY4Y+x6ZlsurEj30anz+Rb034ukUdSXSPZK6ADdihMnHk0z8lsstEXkxr9uKF5LBCxR8SZzVL+4BiFo5l62aHsnEXGNKgpOJO1pWGCvH1qMN44LSG\\\\\\\/oinhA\\\\\\\/5ZJA0++Nx1P+iwaArkEduEMGkGkcA+OVKrHVyGK5yyGqm9+efqmV1T\\\\\\\/o8NOmVcWCl7mtAK0Q\\\\\\\/GiQ78KLUfidKOlMSqrThMgq5erKKgcqnYYpsucW2U5qxlROGCHFXM+BkvLRz+lRuYZIa5nD0eDaZpYPUkQX1naAunX0Ewnot8l2bhBpxrQCTuSdr8RDPkrkUFnxHuMyPhvpvOKL6rcI+Fv1Z+lxchznr0SSvAgGLuUsv2EiwLo3UQl2BltxbCpVSWQIbjUUNH+YyV69VDjC3dbUAIBtaHrdO3fnQTIVprtej1AnreNvSkRkTNSyAxsDR7muwsm\\\\\\\/VXHXg4gvana7EACrd86orcC+izMtxvmAFJF4PvBeZZCAAaGv\\\\\\\/xI43e8qbaBSFDiqZP0NQSeIig6nL0WQeQQcm0Vd08BVJq4GcULxvM07r233YaDGqSebyBlmhZRsdgxbecVmaTdIoyLWUkz3YqYk+\\\\\\\/olk\\\\\\\/IiVWvSgDcOBf2WNq0jya2ddbNvx7wOcRNS13yUwWSE\\\\\\\/L1Cr+iXbE4\\\\\\\/z6l3prlrkAAFYkPOsTYgyJVTi8t4RaMQ0e2oMgphxKF95o1AnsdKMI\\\\\\\/+zbWI2gOtJnY774P8BBVM8mNT5IeYwPh5l4G78+2UcfeHPd\\\\\\\/wEUHrIt0KEqs5joAGIxFibH4ffsRNoywRqcWfUxvDggPT2bjo1gT1gBnhz76VuCr+0WjSeNT1xAnvcGxIFiKi\\\\\\\/+mI4yNUw2MWBu7Xk9MuOs9nW9bzNh3QtrlQUpydIc2wTTW5+340QCsTXf8DqE5yng0sPqleQcsbwwKVPKF\\\\\\\/oSddp0\\\\\\\/echN3jJTQNT7p+53ebRcFKxhx+\\\\\\\/fgHdJ6CoLxucdl7CUPiLBFoCAp30ucDqeZQT\\\\\\\/h1JwsFpRLCrqmvCqfxqVCYmRjwEypH5iFyCu0yikDGJtFfeqapkLC8wBPJzJs1xwq\\\\\\\/zkQcPFRWxAUn7J5CtEOnkhrSu4iDeiAtXEz715a4SvRzxmsPBfa8graFTYxlSadPySXyFosx\\\\\\\/RxuBDLEB0T6A0WF61gl6zCemgUxtQNGezjoOtxzVNfGxWRawYCry8dfJYVsseOQmPQohvFE8io8jF0DdFbMX\\\\\\\/X54zquw0WHjr+9DfWOArkWE330hNRZMVmBi6RAVm2Hs0gpxbEeUBHKcqkVVhd0fs15DVBZtOA\\\\\\\/2g\\\\\\\/rZTEl\\\\\\\/2+q6ubnFgi\\\\\\\/p8v0XrbevFwiEfW4y7vedI9XbC6oXPJv9wyA8Jw+OfLJSVONfnA1FG8Jevto4EHf9tDQ4xv2vR66lTXNgprJZyVJDYWAiSu7X2I2iILBPlNMdwtzaQ2pKWdHbLcWPCYiGKysNMfh2LU9TzgnTgMpTHXyQOeMZTkLRQM84dch33\\\\\\\/xdhh2Ty8uW\\\\\\\/k9dprlVaAO9HrS9wQ3AZEY8Jp2cudmXLxOfkFAcS\\\\\\\/YWIAF\\\\\\\/meVtEBHPZZnxlspSGkwNQbkmCV4BmjZEN5CS4La5IvukezKSf7rNSAYVUtRB9R3K2RZe0693bjP2PzatH5hHW8FQ5\\\\\\\/Jau3yERh2rThykOZx04KBNnu5nha\\\\\\\/NOVU4EXIvydNyA7+BargyBtiTiEBQbeFcLP5pc9d30zwQFUswS8893YVn\\\\\\\/hfAtssH4yt4yi4hXamcLXBjbVICXpocAeFME2NBp\\\\\\\/IfhSdTCtH+dCD0SD1a6zNXg9QAN\\\\\\\/RBS8RmaC3oUQ28oAx8Ww4omVz3wKk3O9xRHg9SFDNggmIkpwCHjveAnXCT090b7KQUSXtqc9Y0Uuh5AL69N2n9bkwPwFjKYI2tdAPPiUSt55L+scsB7\\\\\\\/GpTkvyg8\\\\\\\/Tm\\\\\\\/x77IbRiumFHRiO+X661f9amqCKW5pbIXpnjVf37DojWbsWiHtfWpEBDONOfJ9661kuBxrGdJhBX1275sKQliK5MEmxo3LYjcaf00ezwUHmJLhOwwI5+OTskMNrMj3CVuW0ZDkkWSSgr2P2LVzSSeGNohOToBChEU\\\\\\\/QSame7ViBpg+2rY56Tz2EzzPYU5FzvnJxzawwU4GtmhNykPVVgW\\\\\\\/Zs7C\\\\\\\/wamcaTfhWy2khOWYvYFfe8ixpF0\\\\\\\/rF81FByAyEhKMTCPipWqYOaWYOZOPby0uXv5efMgbwqnbVaIYO+Hap6M8YHKnTSJEk9rli27ALurhYFKKsg1iGS+PyJD4TZwvqYFe9WME3U6N4ZWT8YNeFlyzIfI\\\\\\\/2emZF76OFtz1nwWm\\\\\\\/haG9L2+xpGj1uuXO+NxLJVyxRK0qLlmvn6yB8NhtlZSdsIWnsDSw3iZNbcFb9ELiU20ftIAk67tHXk4XtvOMXsNKVVsxnWnudmXCfinn2RzC9NpLXzkoSO9zYGGteImBjwr9HJM27PdSrHTjeqssawNDBOfLxgQpc6qy+6YEQKhP3669drWVpzMQVHPTs4XrR6TX5JBkwEs0XIcDv7Fbe9+b+QJoYdOdP7onhf12dX1ifw29cQBTvBdLteiXRRFQ0qS9ikSDyq+oC\\\\\\\/2zK8rfA9T2Nk6wfFHQ5WzlGJJkRKnwkBxYFj4Ng50V1i7clDe8qXXiQzsfP0gdoTYqp08jkDUdWBlMwTIsUNN7KR0jeG9KF2EmsPmH0m2bi7b500gSClzcP7d\\\\\\\/kDrCYD4MuEuAyRfCVz0xD0Fgf1mjpC7BQYIRhhpnGeCxd0tLirnTTA6IHaMWQrJiTL18n9HEJORhLLURzLFPPR4uGNAAghWl8ui0+6ugdz8BMY\\\\\\\/5L+nEmL8Ie8kcDCztGj5qAZPl51vodeirBmOrHWi3palQ1LxiLtPGJ0JFK0I52PmqoKCF40UkoK5d77cWHMYpje7MCtqel1jeLf\\\\\\\/NT2cvjRbnEhrgRoWsnWaKAVZ5OPlvcx1pxF1CIJ66TTCtrQFRQSra6uUviNU9dwECM94Y6\\\\\\\/wlrexWHAt0yW87lbl7VJvPK7Py5Db\\\\\\\/myclNcYQhSZz5bA\\\\\\\/HgBksRWoHGC0O\\\\\\\/+QMyoYKGidIEVf6GRcQL\\\\\\\/OB0PyLuisJpCPWtMJ4+ZzjtbZQGcJQe\\\\\\\/adpdUCt0LjfTE\\\\\\\/RkTmJ3h4y5\\\\\\\/xcSTggnDIt1DsOrK0+h\\\\\\\/zpt5W3IOM4NPVR56Ao6b0Y2AnyHgxt6MIOYTWAd7EbJ\\\\\\\/mGXcuPEfhcB2BJy0f5lEK3NU\\\\\\\/c9fv563v5oxmsbI1pCP58nB8iBsNN\\\\\\\/8Wiwqf3OOXrz4iHwWja9713zl3496JWtRLs4h9b+7Iy17Lo3cjVjYIys9eKGrSIuvNuqFMVbYDOwk2mTrYjUi3PXmnoCqPDk67zvtQlE3Go9TVoGNMyxnKqJ3Xb1iJJmj6umYsaUhNHIAG3XGPjVonetVK0fa6uswx7NZMBKFGqe8zpBlriOrfU9oE3R3ucRQloq3OuZn+aQHAEsxKMmZgoRpXvOP938TPtux8HttkoO1LK\\\\\\\/kpIB2Q+BJ0BHLOvb0Ye2I39fLHgu8unA0IHuk9TxqnQnDjsk7iKSIXdnBt6njIPVdKR7aXbr\\\\\\\/CIp+CTVBG8ynguy4L\\\\\\\/fmQeuaXn7gurf0QYxAAESMBax8tukRpzsWjJ+w3hzerz+jVAVrYr5eX6QO4pDKg\\\\\\\/fkohR2pM3sGVO8ln6haKAKIodqHsETOejInvsCfHPJ6Qz5NEbmpTZzz3NvWD72SoTfuQQRHVTXhHLRm06T\\\\\\\/upVvfushZ70NCC1\\\\\\\/Oa3SKSxSewwB+mzW8ecSNHORbGQYFNWij9Ts15QBmVYR1ChGVZsBb3A67G2NYnbg+PNXW5s09fEKH0ReuaBlKbY48GAAE361wkzLsX7gIkERHB8rjSLdrM5gxjnsMiZEoTAf7hPStdovAndCY5N8IAZK1iBwDR3U7XKY3H3e2hT\\\\\\\/9ZsdskOvTnc7YK4q9zNevIjqHVjghmv+AwpoJRey6sNZQdMX6QUGcOxTgBirBHCWWpRyLh+wlUR9sWE\\\\\\\/6C3TgMK43njGmvcSh9EyhRrar939cWp4v9JIQbQwxgjtQT1LM999\\\\\\\/4T70Ys\\\\\\\/jX5Vm31QNjL3O62JQL\\\\\\\/3g06VoArpfrrSNqoNvtu9vhzoqJIYJ4fUequHLPQTo6wYnj76wJiak9cGx797sYGQosm1p0LC+jVeXnyo2zKFRcMA0iRVIB0W9hdZmvJgeLpf0kAXWnrKT+QCl5GDDARp+ghToNnw0iDMMjXP1\\\\\\\/YjuSOKSVCTbFhF0B2Z6Snn2jfJacN1XqgTTS\\\\\\\/2Zb1GH8UL4BsaurhuqRlI\\\\\\\/ox4kH9HrH2JKk7kh27tbfzhVlUbi1GLzfZBc0OSQZhoS59lDNEJ5+yk9gGJMf241yZbsbrcT9P\\\\\\\/uUSodOua1Z8CR1B8hmPTDyP\\\\\\\/vnMpH\\\\\\\/P3\\\\\\\/rlujQ\\\\\\\/WApqccgtiF8LTAXPH1gE8LH\\\\\\\/g++cI7\\\\\\\/coJ5RrMmQLatca\\\\\\\/YXhOwszWkiEjkrCwm8xejIEXB\\\\\\\/YlIMiaPzFsX96cYe5BoTU4LV6VX3C2Ch9CaLrjcdMf+jiZgSDVeUwH8IZhhVDWVEMCLMMqm4b5+h4oo6t9pLKkuEgmJMoB9I5i8G5vfK0xi7k3Z\\\\\\\/iCg734WF4sw+7Wd7XgdhohXdE9gX05uixAHZc07A8XF+OreX\\\\\\\/EFPt99wPm0XHYRdUyd4e+7ujAQHxsSrCybOOOgsn4vWFrghMXGfAUcRN5wCjVlpCmyxIQhEOmu0OcsshciGhy+YKpDLEEDx64ag8HhVwxoxl\\\\\\\/\\\\\\\/5iRWvuaRd8LmY477ML\\\\\\\/7PiwQyHRC0G4mrLfGMz\\\\\\\/7STd9uILfIIRNDwspwGSaF7n+SADgtsqadCtot+IhY7PyPh\\\\\\\/jK+wmr35RBzlU5W\\\\\\\/b9UWYLHHx18fzyzz56lvZnG2fAX0W2wMPuG+UY8v0v3h7x5DtizJRyjKK2Q4piBbd7y697yb8ZSbEkIZW8b76Iy0BdrygqbkHfbs3qmpIqbWwtGirEjlLTNCPaPD3+xYG9fJIPs0GH1C2HvG16xAbBeMjAIMD\\\\\\\/9teIsOKKozioPnxxpHkOi81KNCyjHsG1Gh8CPCukbWmVM2lIVoycI2SGu5oY7TNTQ3suUoUBfSy5HhfJj\\\\\\\/4A5kpJFjKW0KHOjh16K8CMbfrV6m6grQWXCo\\\\\\\/h13Ig84qbRdT+O4wH2P2rWBkWvRRT+\\\\\\\/wOS\\\\\\\/eNaB7bZt9vnmeq8EQITRzo24gYcwSl2Aw1TIm0T+2S05d8D4BbOVDfbQouxHNpXs7SnyNwSSMkM5fAZJu\\\\\\\/PJJXg+bDNq5RMkxhKqCrvKrKbWojK52ZD7+c6BRTDjCtlNCR2zYAIPuxD\\\\\\\/I6vg4cNozidsLO3D21tCGK4UXhjCeMledjPqouGc4yRxUUau9eoAJOXJXhC5PUtP8Fdeip2ExoZABH59VWqbbN71Go1pAPa36IpiPxY6EHLkdwq4v8Yq+1Wyr460jGRJ+id7o7yQP7K8O+BESq0hcnuXbq7RVdZFw+\\\\\\\/6\\\\\\\/LUIqCiyCBti1RdH0HJYKJXS8wGYDhiJsOA3QKF0VDvp9Xjcjk0e9U0GBzIJGqV1VJCL4A2jCxmhUzHtByZGf1d1PDBvdILpFgcrtcDSjcN88Ct4Pv\\\\\\\/gPQoSuNSb5oFEdznDzxsrZZviVz4yoCXlJ0beOGRkSM8L4EzB\\\\\\\/seEZddhhZ0ZegY\\\\\\\/3rKOqj7FJHwbYo01\\\\\\\/tbdDeaYqKOjmr8jaqLfxOqCJQDv+\\\\\\\/wQmruv4wuU\\\\\\\/uHnKSJ5rR7It7b5N0My4bl4hW7SDalwYwiV0EMM\\\\\\\/tHcdl4TGuF4ejYXqGZNuSte4U1\\\\\\\/2jmN336cIr8Wa+pJHr+0IHPkWG3m\\\\\\\/bxeNtT90MsIhkvLtg1rknMr2AOI\\\\\\\/F\\\\\\\/PgvdbzW0oe8yRfrrSqtPQkzP50ij1F3bjLK9H8jY4Qh8D+gGrA5SUToRbDr\\\\\\\/C3C0GY8rATONgzoajmKKlfr+Pimht\\\\\\\/0eqEcGpi2NWyWv2AZxiE0rgtzj2O4SARohuwFKBPHSXt\\\\\\\/MjEMS9YF2RewX63p8v4hO4z7gBtQ4wBi1oxzWqFyNbldFH+f8\\\\\\\/h8kHPz3kSlEay+VhgVu6KGRoT8wOE0pCASvZYl7HECYTp1MFAkLHrhSNJhGQaNxkkXZem2RQ\\\\\\\/q3uiXuF8c1ekgeiuaYHvgm5qTk6PP9rdNFic3CRIRG\\\\\\\/0x\\\\\\\/jm7XTnL2pD5HjIt6ZDJ0lFcVT2aiXVZicP\\\\\\\/YhzXkruNgc3vnicoiBTy6QKum6rbJiczumt5lzTEghBHnEwt+nZtcCCrXec4k0yLcMqOtgzUIYrV9TDWevA5gK9sTepnp1JEOcWRCS+pyYLQZ90nmjlUajim0E2FD8Rc6p1n05QMI1P8faF7b4rrdQngdWMkyKShFb2Cvv\\\\\\\/7ni7Xl5EUxUTM6E16qcAGsdOQruPE5eN+c0HJpFdmxgRTbBo0P6K\\\\\\\/SXQvOO9Gy+uE9HTPsVD8Tcw+mDlZjSzMSR4\\\\\\\/JuO57GebnUZyjNdPjbzkU89cf5HfTP6QfKnNnROvqBGBP26XpX0Ha\\\\\\\/QKvtQ8myS0mHjWC6zac1DY21+pnPYQdwSWVnMlDAPuJUU7vEb+C1TB32Lf1pP77tsD8usDjriBQMwlbEkTIn9\\\\\\\/0fLaEyg0nnPABDVQjRFmhIe\\\\\\\/1K\\\\\\\/2p2OgjAI4JSYktwryIZTB3iykvevlAH\\\\\\\/\\\\\\\/S12xXVLu8H5GcwNUb+Ob6bCBYesQViQLZYIXIAQgTzCyzgxvOZigRRnESv4cmwtAIfq7p0GfsnMKfgysBwotZu32eYZxvKMCGKt1+vf\\\\\\\/jx3BtDrDsZFrZY3wLr3lv5IVEU+WVAxXXHv9HQ7Cj2d5msPpx4ZOHfVEvbANao1TC9GgqDwNYxNFpWNVlMEsIgYdzgYy2Ph8XWvkwed2IPy4bedMyBqAos+BAmF1eeNbeFQzr2RnXs6zEgGCUiLXDDcEcuZnZahzFUTwQD8ThreDAdd9KRgzuIagbe59tjP9+oUOcWV2pd\\\\\\\/\\\\\\\/bbhIax3J+BuETceTjGzlXzUNiwwaddQin470bF1Gae6OB4wfChLr0kUETwNFtNYdVTqXH3j7Vq26ulLmWK3TNcVYxxk6gUpTwXOUKNis9bUCPtoTKlA66ZXiP\\\\\\\/XmfHG+12aiq300gQxmJY6delVQnaehYJhEkyECwoQCPhzpRsyFIDB1GeUGyziio1yDqNwIO5I7CzdtLHdtobashsKK\\\\\\\/VkIopfL2uiltnAfYqkIVm9Dqec5KRExwScnHw5bAPwW+NUJYGiJT4WOHlFLsbTxZhCQjJTppYVHu\\\\\\\/rX88Z5GaaRTU6IxBa6Wef\\\\\\\/\\\\\\\/MbheFfD2g2+0w6HAzx8ZHwIqnKmr\\\\\\\/AYJXPpYUS3um8CA2ctZuXHm7tCe3plS+etY\\\\\\\/\\\\\\\/h7Efj4pnEuqO6udIPHJDZQ5mx\\\\\\\/32\\\\\\\/b9xriM8uj6pc5v1kmnphXxPCz23v3Zmf9\\\\\\\/a9Vrw8\\\\\\\/p23juJD3vteshrdSnKeM7hHnN29xyWjsgfoSza1NZfi3SZELhlyW0monOqCxQn+40\\\\\\\/vWmEdbhaxVA2ZfYaiLxhHpZi9a0ZnnyHoJdVo8MnvZXotclxR8\\\\\\\/N\\\\\\\/hg7k7nm64tZTLFYkjje5aha36HzhyJgnrOAZ\\\\\\\/cQLZBamIMnfKr2akjcY1x4rDMPYeFw6tb8MkXQxicmvL6uMYkrkRt6dfq+y6lQuV3714Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022g9L4ijSHZI6Vbpg7SPUPyw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022074fe0032297c5f76aa79df3669c2c1bb797a4ec8ca819c6d258d8880245169b\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022yjDv1l+eDbn8vxqWL0JJU4R5A2RJrBsfsY4Rz\\\\\\\/fj7H9J5R4YE4pmFx7zkEIXDog6uRPxa\\\\\\\/SPzMg47m46L+JH7uam6vQ+AN0qi16skm3PDGfUR1AbuATgnEhCyJY4v4rwjtdO+ccJxIbxnHpgmFbSORc\\\\\\\/pEVcgFIoIKxMkmVQEpuUx\\\\\\\/FZdL+R+9lE4RQaJuKCvLXY4J0yopVKlY\\\\\\\/Hu2ibYLeeQOXvqFxNtI+YyJ296ZIhmKXCdBHtdQy\\\\\\\/xQt\\\\\\\/Tw40g8IGUQHWr0Nhwwp8k9Qiturdy3F2tvbDtUlLxB008UQLOWIrB+gpo6IX6R1SKQRghnvv1pyyhlyPZDcyQHbtlk7jZa1L0p\\\\\\\/tQL9HcYXOG+fl67KgKhjmRds5gmC\\\\\\\/hePp5zvmxDAVAX29JzMmvsEhtP69Tzp3QIau630wVynbZk+T2Ekyyys4b6xxygbV4+SVCMV1BKlGapKB3W092eRNoFeaLTbrs2QXXfOFZhJPoP3f8BEBDxFXvJd5ka7xf6jeUER+2jpFWqbreCfkbhoS8Wd2UNauhgQsShYUNAL9zt6LtCR78qD5hjyWxPHL06TGzfh34WWlqTsiIwnTwTsIj4uqsQ9Zj2j6Po\\\\\\\/\\\\\\\/yeRPcq5j3n2nv0RqhAIgcZEhqeF8OVW+vV2+rH+DaQT9NyfuIT0qzt7OR\\\\\\\/twG5gkkY4JxlmkhzaJjMBiqXB3U4ZrCvFBgg3jGrQ6qCsgFCLOi5j2jBkErX6zCYIMpzNl1RomNhmzm1fZvNLa\\\\\\\/EKqbt9PZm8LGpFDWI3Kh49tw6OMPhq9vYqVcNeESD0zd13V6RWhg2AML7KuOtjMT4S6ZJPTR2sg1hfSHs4GsEnENbcEuF2OugNs92eVyXjVe\\\\\\\/ZezbXWhHe\\\\\\\/mKWYMYQDlFzDLJpZnVXHLZ1yIMs90uIhe5piCar8bBhSAVMW3Mmyq4xQ4SlJSHrKHJvjyLjM+gW47pdODYPvEWpkufZgzf4j\\\\\\\/YAOEd090yEXRqT4tG9DK5UevbsTHZCSp9jG466jzWU7xzoqaKFHYQfHjqxV\\\\\\\/f\\\\\\\/E6pTlrfO1Bq2x05xHa8OaCdXgDxiSWIkrn4QzFU48yrk1YAO+czO8c2uHO\\\\\\\/LJPF7gb3cDm9U0kpwwcwfO5fRSvToUCZuoGRGiGYkJwG\\\\\\\/S9bIsntkP67xzpioBlXQGbraTvplQLG8ghpI\\\\\\\/PdK05AoUnIkHx3fh52igzoHBthRxUSC08QFilogH3yfZhQ8iXCON79i7qQiaS0btWHgpIVdo6IGhjyQWEZ55Vg97GFCJiZhyEcazIi4aucFExTqLNlt3Fdn4AVXMAEZqm4uDkwLgwcu0k\\\\\\\/9Pjd85Wf\\\\\\\/Sd9dkrey9ah4heGl54khGJlh8\\\\\\\/DfDZZm0S9TxLhQlKj0LENz822Bl5VdVtQsyKA0KuIunzkvn2Cgz+B9SKB+G+CE+Ef4I8ifLVUE1LG8DNY8Es7jj4MoJ+rWVj94n8JM6Z\\\\\\\/NCB9lGlbQc3hjXgrPKHFV8x9yIpGMWRhIS++vPygp\\\\\\\/NpS\\\\\\\/+Nx\\\\\\\/PUwWzmP9hZ3YQHF43eni3RW7djKPlg2+gX1LapMa7KxKmk09YN\\\\\\\/I98lLbZHKzDzkyo8zNMB4N4igs6HZ18gFMaiV+5aPMtttfkTWQpHMZoTGvgqvC8Jf9T\\\\\\\/2elFzkvqZxSpDFLjKQ8vx\\\\\\\/3d+LYCF0VRXR3u92GlcQs3bL\\\\\\\/zgJrct+9l2XF61eIAamRmntLVEa1UscUa2k3Rsze6oy9mU67wkLC\\\\\\\/r7IGWfUxoNLsN5tIWa9cchRxEJ7p4vke7EpE9WuqAUPXAI9Ai4JRG+FCJExTlmMW80pTxbTqrrPIKK6jyAWjFBBIKD8YJsSxN3j8BjQGkXqg4mgio77eF5vj2hXqHlJvgfQNvHPBsVHTvqKs3pEWRjQYRYkBsUjloGlGXBOOeJh1W5m1Usi8GvVFFRQd65KetXomZRAjVsUSpERyNExKzSgNj2WxGMJqCjrgYf2YkAZJTROTaAXtgyILanK2sfUkgTV9WqM5t0ZAPMvr9PFwpMw==\\\u0022,\\\u0022iv\\\u0022:\\\u00226wh\\\\\\\/S5ysRHrdNKVQtcShHQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022423d4a00ac8f45afb1f96af39962c39a7e6efbf4c21284460cf5d0b082ce1eb4\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460106 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/893" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-daMa6yOUsxETnza40_rbZ-_Flr6uY2rvej4097zMMJk" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/893" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YvXw41_UsUBfVjKWq4sFkZukZG7vTD8hICHxbCSR9qc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/4/881" title="Barcode Sequencing Screen Identifies SUB1 as a Regulator of Yeast Pheromone Inducible Genes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/4/905" title="Genomic Access to Monarch Migration Using TALEN and CRISPR/Cas9-Mediated Targeted Mutagenesis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460106" id="node460106" data-pisa="ggg;6/4/893" data-pisa-master="ggg;g3.115.023200" data-apath="/ggg/6/4/893.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Detecting and Quantifying Changing Selection Intensities from Time-Sampled Polymorphism Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7052-0971" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Hyunjin Shim</span>, <span class="highwire-citation-author" data-delta="1">Stefan Laurent</span>, <span class="highwire-citation-author" data-delta="2">Sebastian Matuszewski</span>, <span class="highwire-citation-author" data-delta="3">Matthieu Foll</span> and <span class="highwire-citation-author" data-delta="4">Jeffrey D. Jensen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">893-904;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.023200 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460106" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hyunjin Shim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Swiss Institute of Bioinformatics, Lausanne, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hyunjin%2BShim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shim%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHyunjin%2BShim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7052-0971" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hyunjin Shim</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Stefan Laurent </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Swiss Institute of Bioinformatics, Lausanne, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BLaurent%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Laurent%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStefan%2BLaurent%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sebastian Matuszewski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Swiss Institute of Bioinformatics, Lausanne, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebastian%2BMatuszewski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Matuszewski%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebastian%2BMatuszewski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Matthieu Foll </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Swiss Institute of Bioinformatics, Lausanne, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Agency for Research on Cancer, Lyon, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthieu%2BFoll%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Foll%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthieu%2BFoll%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jeffrey D. Jensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Swiss Institute of Bioinformatics, Lausanne, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BD.%2BJensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jensen%20JD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeffrey%2BD.%2BJensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/4/893" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460106" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460106/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/893.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460106" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460106/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/893.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460106" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460106/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/893.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460106" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460106/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/4/893.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460106" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460106/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">During his well-known debate with Fisher regarding the phenotypic dataset of <em>Panaxia dominula</em>, Wright suggested fluctuating selection as a potential explanation for the observed change in allele frequencies. This model has since been invoked in a number of analyses, with the focus of discussion centering mainly on random or oscillatory fluctuations of selection intensities. Here, we present a novel method to consider nonrandom changes in selection intensities using Wright-Fisher approximate Bayesian (ABC)-based approaches, in order to detect and evaluate a change in selection strength from time-sampled data. This novel method jointly estimates the position of a change point as well as the strength of both corresponding selection coefficients (and dominance for diploid cases) from the allele trajectory. The simulation studies of this method reveal the combinations of parameter ranges and input values that optimize performance, thus indicating optimal experimental design strategies. We apply this approach to both the historical dataset of <em>P. dominula</em> in order to shed light on this historical debate, as well as to whole-genome time-serial data from influenza virus in order to identify sites with changing selection intensities in response to drug treatment.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/fluctuating-selection" class="hw-term hw-article-keyword hw-article-keyword-fluctuating-selection" rel="nofollow">fluctuating selection</a></li><li class="kwd"><a href="/keyword/change-point-analysis" class="hw-term hw-article-keyword hw-article-keyword-change-point-analysis" rel="nofollow">change point analysis</a></li><li class="kwd"><a href="/keyword/time-sampled-data" class="hw-term hw-article-keyword hw-article-keyword-time-sampled-data" rel="nofollow">time-sampled data</a></li><li class="kwd"><a href="/keyword/approximate-bayesian-computation" class="hw-term hw-article-keyword hw-article-keyword-approximate-bayesian-computation" rel="nofollow">approximate Bayesian computation</a></li><li class="kwd"><a href="/keyword/wright-fisher-model" class="hw-term hw-article-keyword hw-article-keyword-wright-fisher-model" rel="nofollow">Wright-Fisher model</a></li></ul><p id="p-3">The common assumption of constant selection intensity through time utilized in many tests of selection is often criticized as unrealistic in natural and experimental populations, both owing to environmental changes (<em>e.g.</em>, fluctuations in climate, predator density, or nutrient availability) as well as to genetic changes (<em>e.g.</em>, epistasis, clonal interference). Despite this, such considerations are not accounted for in most population genetic models, since inferring changing selection coefficients (<em>s</em>) from single-time point polymorphism data are difficult. However, owing to recent technological advances, time-sampled polymorphism data are increasingly available, and time-serial analytical methods are expanding (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Malaspinas <em>et al.</em> 2012</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Mathieson and McVean 2013</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Foll <em>et al.</em> 2014</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lacerda and Seoighe 2014</a>; and see review of <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bank <em>et al.</em> 2014</a>), allowing for an empirical evaluation of the importance of changing <em>s</em> models.</p><p id="p-4">Fluctuating selection in natural populations was suggested by <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Wright (1948)</a> with regards to the phenotypic time-serial data of <em>Panaxia dominula</em> (scarlet tiger moth) to account for its observed annual fluctuations in coloring polymorphism (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Fisher and Ford 1947</a>). Since, there have been several theoretical considerations of fluctuating selection (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kimura 1954</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Karlin and Levikson 1974</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Karlin and Lieberman 1974</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chevin 2013</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gossmann <em>et al.</em> 2014</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gompert 2015</a>), as well as many observations of fluctuating selection in natural populations (for a review, see <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bell 2010</a>). Nonetheless, until recently, analyses of fluctuating selection centered on <em>random</em> or <em>seasonal</em> oscillations of selection strength through time, as the mathematical complexity of analytical methods only allowed the simplest cases to be considered.</p><p id="p-5">Approximate Bayesian Computation (ABC) has the advantage of being flexible in integrating complex models due to computational efficiency and the lack of likelihood computation (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Beaumont 2010</a>). Recently, a hierarchical ABC-based method based on the Wright-Fisher model was developed in order to infer genome-wide effective population size and per-site selection coefficients from whole-genome multiple-time point datasets (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Foll <em>et al.</em> 2014</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">2015</a>). While the initial approach performs well overall, the authors noted the possibility for observations inconsistent with a single-<em>s</em> Wright-Fisher model; this was indeed observed at certain sites in their analysis of the influenza virus genome, which were simply excluded from consideration. Thus, as a natural extension, we here investigate the presence of changing selection intensities in these outlier single nucleotide polymorphisms (SNPs); in doing so, we also develop an extended Wright-Fisher ABC-based method capable of detecting and quantifying changing selection intensities through time (termed CP-WFABC).</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Wright-Fisher ABC-based method</h3><p id="p-6">This approach first relies on a previously developed Wright-Fisher ABC method (WFABC - <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Foll <em>et al.</em> 2014</a>, <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">2015</a>) in order to estimate a constant effective population size (<em>N<sub>e</sub></em>). WFABC infers an effective population size from time-serial datasets through the average fluctuation of neutral trajectories over time (following <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Wright 1948</a>). If whole-genome multi-time point datasets are available, the estimated value is referred to as the genome-wide effective population size. The posterior of the <em>N<sub>e</sub></em> estimated from WFABC is used as a prior for the following extended method. The trajectory <em>X</em> of a given allele with a known <em>N<sub>e</sub></em> consists of time-serial allele frequencies f<em><sub>t</sub></em> (<em>t</em> = 1,…,<em>T</em>) where <em>T</em> is the total number of generations (with <em>T</em> &gt; 4 to allow for a change point to be realizable with the Wright-Fisher model), from which a sample <em>n<sub>m</sub></em> is taken at sampling time points <em>m</em> = 1,…,<em>M</em> (with <em>M</em> &gt; 4 to allow for a change point to be detectable). Parameters to be inferred include the selection coefficient prior to the change in selection intensity (<em>s<sub>1</sub></em>), the selection coefficient subsequent to the change in selection intensity (<em>s<sub>2</sub></em>), the change point in selection (<em>CP</em>), and the dominance coefficient (<em>h</em>) for diploid models. The joint posterior distribution of these parameters can be estimated by<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-2.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>for the diploid model and the haploid model, respectively. The ABC approach allows these parameters to be inferred using Wright-Fisher model simulations without calculating the likelihood <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-3.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-4.gif"/></noscript></span></span><sub>.</sub></p><p id="p-7">The Wright-Fisher model simulator with a change point in selection strength is used to simulate the data <em>X</em>, with relative fitnesses <em>w<sub>AA</sub> =</em> 1 <em>+ s</em>, <em>w<sub>Aa</sub> =</em> 1 <em>+ sh</em> and <em>w<sub>aa</sub> =</em> 1 for the diploid model, and <em>w<sub>A</sub> = 1 + s</em> and <em>w<sub>a</sub> = 1</em> for the haploid model (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Ewens 2004</a>). Initially, the random sampling of an allele from generation <em>1</em> to generation <em>CP-1</em> is simulated using <em>s<sub>1</sub></em>, and onwards from the change point (<em>CP</em>) using <em>s<sub>2</sub></em>. In order to simulate realistic allele trajectories with changing selection coefficients, the allele needs to be segregating at the time of the change point. This condition is necessary since the change in selection coefficient cannot occur if the allele is either lost or fixed beforehand, assuming the infinite-site model with no back mutations. Thus, only alleles segregating at the change point are accepted as simulated datasets.</p><p id="p-8">The associated summary statistic for these time-serial data are <em>Fs</em>’, an unbiased estimator of <em>N<sub>e</sub></em> based on the allele frequency change between two sampling time points that remains unbiased even in the presence of highly skewed allele frequencies and small sample size (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Jorde and Ryman 2007</a>), an important property for inferring change points along the allele trajectory. It is given as:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-5.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-6.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where <em>f<sub>m</sub></em> and <em>f<sub>m+1</sub></em> are the allele frequencies at two consecutive sampling time points, <em>m</em> and <em>m</em>+1 (separated by <em>t<sub>i</sub></em> generations), <em>f<sub>i</sub> = (f<sub>m</sub> + f<sub>m+1</sub>) / 2</em>, and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-7.gif"/></noscript></span></span> is the harmonic mean of the chromosome sample sizes <em>n<sub>m</sub></em> and <em>n<sub>m+1</sub></em> at two consecutive time points. Unlike the WFABC approach that summarizes time-serial trajectories into only two summary statistics (increasing and decreasing <em>Fs</em>’; <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Foll <em>et al.</em> 2014</a>), here <em>Fs</em>’ is summarized at every pair of consecutive time points as <em>Fs’<sub>i</sub> with i = 1,…,M−1</em>, where <em>M</em> is the number of sampling time points. This modification allows additional information, such as the timing of increase or decrease in allele frequency, to be captured. In order to retain information about directionality that is lost by using <em>Fs</em>’ as a summary statistic in the original WFABC approach, increasing allele frequencies are made positive and decreasing allele frequencies are made negative with regards to the absolute value.</p><p>The joint posterior distribution of the parameters of interest is obtained using the algorithm described in <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Beaumont <em>et al.</em> (2002)</a>. The approximate posterior density<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-8.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>with U(X) as data-specific summary statistics, θ = (<em>s<sub>1</sub></em>, <em>s<sub>2</sub></em>, <em>CP</em>, <em>h</em>) for the diploid model and θ = (<em>s<sub>1</sub></em>, <em>s<sub>2</sub></em>, <em>CP</em>) for the haploid model, is obtained using an ABC algorithm as follows:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-10">i. Simulate <em>K</em> trajectories from the Wright-Fisher model with a change in selection intensity, with θ randomly sampled from its prior <em>P(</em>θ<em>)</em>, conditional on the allele segregating at the change point.</p></li><li id="list-item-2"><p id="p-11">ii. Compute U<em>(x<sub>k</sub>)</em> for each trajectory using the <em>Fs’<sub>i</sub></em> summary statistic between all consecutive sampling time points <em>i</em>: U<em>(x<sub>k,i</sub>)</em> where <em>i</em> = 1,…,<em>M−1</em> where <em>M</em> is the last sampling time point.</p></li><li id="list-item-3"><p id="p-12">iii. Retain the 1000 best simulations with the smallest Euclidian distance between U<em>(x<sub>k,i</sub>)</em> (from the simulated) and U<em>(X<sub>i</sub>)</em> (from the observed) to obtain an approximate posterior density of P<em>(</em>θ<em>|X)</em>.</p></li></ul><p>For the first step, simulations are performed with the same initial conditions as the observed data, including effective population size, initial allele frequency, and the sampling points and sizes. In addition, a minimum allele frequency in one of the sampling time points is imposed on simulated trajectories as is done in observed data. This ascertainment scheme takes into account the nonrandom criterion of considering only the trajectories reaching values above the sequencing error threshold in the observed data (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Foll <em>et al.</em> 2015</a>).</p><p id="p-13">For the second step, it is important to note that the <em>Fs</em>’ summary statistic is calculated between every pair of consecutive sampling time points (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). This construction of the summary statistic enables information on both the timing and strength of the allele frequency change to be captured, as the timing of the change is essential in detecting the change point and the strength of the change is essential in estimating the corresponding selection coefficients. For the diploid model, an additional parameter <em>h</em> is inferred jointly with the other three parameters, as its value is one of the determining factors in the timing of allele frequency change (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Haldane 1932</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/893/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Illustration of Fs&#x2019; calculated between every pair of consecutive sampling time points with 10 generations and the maximal CUSUM (cumulative sum control chart) value SCP as summary statistics, using a haploid population of Ne = 1000 with a de novo mutation and the sample size as 100. (A) Allele frequency trajectory (B) Fs&#x2019; values along the trajectory (C) CUSUM values along the trajectory" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2009676308" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Illustration of &lt;em&gt;Fs&lt;/em&gt;&#x2019; calculated between every pair of consecutive sampling time points with 10 generations and the maximal CUSUM (cumulative sum control chart) value &lt;em&gt;S&lt;sub&gt;CP&lt;/sub&gt;&lt;/em&gt; as summary statistics, using a haploid population of &lt;em&gt;N&lt;sub&gt;e&lt;/sub&gt;&lt;/em&gt; = 1000 with a &lt;em&gt;de novo&lt;/em&gt; mutation and the sample size as 100. (A) Allele frequency trajectory (B) &lt;em&gt;Fs&lt;/em&gt;&#x2019; values along the trajectory (C) CUSUM values along the trajectory&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/893/F1.medium.gif" width="363" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/4/893/F1.medium.gif" width="363" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/893/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/893/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467935" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-14" class="first-child">Illustration of <em>Fs</em>’ calculated between every pair of consecutive sampling time points with 10 generations and the maximal CUSUM (cumulative sum control chart) value <em>S<sub>CP</sub></em> as summary statistics, using a haploid population of <em>N<sub>e</sub></em> = 1000 with a <em>de novo</em> mutation and the sample size as 100. (A) Allele frequency trajectory (B) <em>Fs</em>’ values along the trajectory (C) CUSUM values along the trajectory</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">For the third step, the simulated <em>Fs</em>’ summary statistics U<em>(x<sub>k,i</sub>)</em> between every pair of consecutive sampling time points are compared with the corresponding observed <em>Fs</em>’ summary statistics U<em>(X<sub>i</sub>)</em> – allowing a small fraction of the simulated trajectories with allele frequency changes that best match the observed trajectory (in terms of both timing and strength) to be retained.</p></div><div id="sec-3" class="subsection"><h3>Wright-Fisher ABC-based method with change point analysis</h3><p id="p-16">In order to increase computational efficiency and sensitivity in change point detection, an additional summary statistic is integrated into the Wright-Fisher ABC-based method. This novel summary statistic is derived from change point analysis – statistical techniques developed and used in many disciplines ranging from finance to quality control in order to detect and estimate change (<em>e.g.</em>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chen and Gupta 2001</a>). Among the techniques available, the cumulative sum control chart (CUSUM) developed by <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Page (1954)</a> is able to detect small and sustained shifts in the statistics β obtained from a sample (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Ryan 2011</a>). Instead of using the entire CUSUM procedure as a separate method for detecting change, the CUSUM value is integrated into the Wright-Fisher ABC-based method as an additional summary statistic that characterizes the time-sampled trajectory of an allele:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-9.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-10.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-11.gif"/></noscript></span></span>. The CUSUM value <em>S</em> is accumulated only when the statistic β is different from its average value in the dataset.</p><p id="p-17">The change point <em>S<sub>CP</sub></em> is the sampling time point with the maximal absolute value of <em>S<sub>i</sub></em>, which is the furthest point from the initial value zero attaining the maximal accumulation of difference from the average value:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-12.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>Here, we take <em>Fs</em>’ at each pair of consecutive sampling time points as the statistic β, since it is a time-serial measure of the allele frequency change, which is indicative of the selection strength change. Thus, when <em>Fs</em>’ is used as the statistic β in the CUSUM, the maximal CUSUM value <em>S<sub>CP</sub></em> is the potential change point of the allele trajectory, as illustrated with an example in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>.</p><p id="p-18">In the change point Wright-Fisher ABC (CP-WFABC), an additional summary statistic <em>S<sub>CP</sub></em> is used to characterize observed and simulated allele frequency trajectories for detecting a change point. In the third step of the ABC algorithm, the Euclidean distance between U<em>(x<sub>k,i</sub>)</em> and U<em>(X<sub>i</sub>)</em> is calculated only if the maximal CUSUM value <em>S<sub>CP,k</sub></em> of the simulated data matches the maximal CUSUM value <em>S<sub>CP</sub></em> of the observed data:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-13.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>This additional step allows the computation to be more efficient – especially when there is a large number of time points sampled – as the Euclidean distance is calculated for a fraction of simulated trajectories whose maximal CUSUM value is equal to that of the observed (<em>i.e.</em>, with the same time-sampled characteristic). Furthermore, as the CUSUM is sensitive to small and sustained changes, integrating the CUSUM into the Wright-Fisher ABC method increases its sensitivity for detecting small and sustained changes in selection strength. The potential bias in the calculation of the maximal CUSUM value is counteracted by the fact that the bias would be present in both the observed and the simulated trajectories.</p></div><div id="sec-4" class="subsection"><h3>Simulated data with constant selection and with changing selection</h3><p id="p-19">We generated simulated datasets of different effective population sizes using the Wright-Fisher model for two scenarios: (1) trajectories of constant selection with only <em>s</em> and <em>h</em> (for diploid models) as parameters, and (2) trajectories of changing selection with <em>s1</em>, <em>s2</em>, <em>CP</em>, and <em>h</em> (for diploid models) as parameters. For selection coefficients, uniform priors of (−1, 1) were used. The uniform prior of <em>CP</em> was set to occur between the second generation and the second-to-last generation (2, <em>T−</em>1), where <em>T</em> is the number of generations of the population in the time-serial data. The dominance coefficient <em>h</em> for the diploid model was randomly drawn from one of three values: complete recessiveness, codominance, or complete dominance [0, 0.5, 1]. Although these prior ranges are uninformative, the constraint on the trajectories to be segregating at the change point shapes the distribution of the prior ranges according to the input parameters such as ploidy, effective population size, initial allele frequency, and number of generations; the updated priors for the haploid population of <em>N<sub>e</sub></em> = 100 and the diploid population of <em>N<sub>e</sub></em> = 50 are shown as examples (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS1.pdf">Figure S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS2.pdf">Figure S2</a>).</p><p id="p-20">The other input values – such as the number of generations (<em>T</em> = 100), the sampling time points (<em>M</em> = 10), the sample size (<em>n</em> = 100), the initial allele arising as a new mutation, and the ascertainment of observing a minimum frequency at 2% – were kept constant for the two scenarios. We retained the best 0.1% of 1,000,000 simulations for each pseudoobservable trajectory using the rejection algorithm based on the Euclidean distance as described above. The mode of the posterior distribution from the best simulations (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Sunnåker <em>et al.</em> 2013</a>) was used to evaluate the estimated parameter value against the true parameter value.</p></div><div id="sec-5" class="subsection"><h3>Data availability</h3><p id="p-21">The R package of CP-WFABC is available on jensenlab.epfl.ch.</p></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><div id="sec-7" class="subsection"><h3>Model choice in the change point Wright-Fisher ABC method</h3><p id="p-22">The first step of CP-WFABC is to be able to distinguish changing selection trajectories from constant selection trajectories. ABC model choice was constructed to choose between two models: M<sub>0</sub> with a single selection coefficient, and M<sub>1</sub> with two selection coefficients and a change point. The relative probability of M<sub>1</sub> over M<sub>0</sub> can be computed through the model posterior ratio as the Bayes factor <em>B<sub>1,0</sub></em> (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Sunnåker <em>et al.</em> 2013</a>):<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-14.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span>when the model prior <em>p</em>(M<sub>0</sub>) is equal to <em>p</em>(M<sub>1</sub>). In practice, the model priors are made equal by producing the same number of simulations for each model and retaining the best simulations from the lot. The posterior ratio is computed as the number of accepted simulations from M<sub>1</sub> over those of M<sub>0</sub>, giving the Bayes factor <em>B<sub>1,0</sub></em>,which is an indicator of the support for a specific model. The performance study was conducted with a haploid population of <em>N<sub>e</sub></em> = (100, 1000, or 10,000) and a diploid population with <em>N<sub>e</sub></em> = (50, 500, or 5000) using the simulated datasets of the two scenarios described in the previous section as M<sub>0</sub> and M<sub>1</sub>, respectively.</p><p id="p-23">We considered two cases for the pseudoobservables to test the sensitivity and specificity of the ABC model choice: the first case when the pseudoobserved trajectories have a single selection coefficient, and the second case when they have changing selection coefficients. One thousand pseudoobservable trajectories were generated for each case with the data ascertainment minimum frequency set to 2% for at least one of the sampling time points. Additionally, for the second case, pseudoobservable trajectories were accepted only when the allele was segregating at the time of the change point – a constraint for realistic combinations of selection coefficients, change points, and dominance (for diploids) – in order to reproduce changing selection trajectories in real datasets. All other input values were kept constant as in the simulated datasets described in the previous section.</p><p id="p-24">The results of the ABC model choice from a haploid population with <em>N<sub>e</sub></em> = 100, and a diploid population with <em>N<sub>e</sub></em> = 500, are represented as ROC curves (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Xavier <em>et al.</em> 2011</a>) in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. Specificity is given on the <em>x</em>-axis showing the true negative rate, while sensitivity is given on the <em>y</em>-axis showing the true positive rate of the Bayes factor <em>B<sub>1,0</sub></em> calculated from 1000 pseudoobservables of changing selection (where <em>B<sub>1,0</sub></em> should be large) and 1000 pseudoobservables of constant selection (where <em>B<sub>1,0</sub></em> should be small). The overall ROC curves in black (all trajectories) show that when the specificity threshold is most conservative in detecting no false positives (<em>i.e.</em>, <em>B<sub>1,0</sub></em> = infinite), the Bayes factor <em>B<sub>1,0</sub></em> has a sensitivity of ∼30% for all populations. Considering that the pseudoobservable trajectories were simulated randomly from a wide range of prior values, the Bayes factor <em>B<sub>1,0</sub></em> from CP-WFABC is, in general, sensitive and specific. The ROC curves in black for the other haploid and diploid populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS3.pdf">Figure S3</a>) also indicate that the Bayes factor <em>B<sub>1,0</sub></em> is sensitive and specific as they are above the diagonal line of no-discrimination. The area under the ROC curve (AUC) is used to assess how reflective the Bayes factor <em>B<sub>1,0</sub></em> is of the true model, as summarized for all pseudoobservable populations in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. The AUC values show that the Bayes factor <em>B<sub>1,0</sub></em> is ∼80% more probable to rank a randomly chosen changing selection case above a randomly chosen constant selection case. Additionally, the distribution of Bayes factors <em>B<sub>1,0</sub></em> under the null model M<sub>0</sub> (<em>i.e.</em>, case 1) was used to compute the significance level α at 1% (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Good 1992</a>). For both diploids and haploids, the significance threshold is higher for smaller population sizes (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>), and the calculation of these thresholds will be important in any given data application.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/893/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="16. ROC (Receiver operating characteristic) curve of the Bayes factor B1,0 from the ABC (approximate Bayesian computation) model choice (A) a haploid population with Ne = 100, and (B) a diploid population with Ne = 500." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2009676308" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;16. ROC (Receiver operating characteristic) curve of the Bayes factor B&lt;sub&gt;&lt;em&gt;1&lt;/em&gt;,&lt;em&gt;0&lt;/em&gt;&lt;/sub&gt; from the ABC (approximate Bayesian computation) model choice (A) a haploid population with N&lt;sub&gt;&lt;em&gt;e&lt;/em&gt;&lt;/sub&gt; = 100, and (B) a diploid population with N&lt;sub&gt;&lt;em&gt;e&lt;/em&gt;&lt;/sub&gt; = 500.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/893/F2.medium.gif" width="440" height="223"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/4/893/F2.medium.gif" width="440" height="223"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/893/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/893/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467823" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-25" class="first-child">16. ROC (Receiver operating characteristic) curve of the Bayes factor B<sub><em>1</em>,<em>0</em></sub> from the ABC (approximate Bayesian computation) model choice (A) a haploid population with N<sub><em>e</em></sub> = 100, and (B) a diploid population with N<sub><em>e</em></sub> = 500.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467934/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467934/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">AUC values and confidence intervals for ROC curves</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467955/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467955/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">The Bayes factor (BF) thresholds for the significance level α of 1% computed using the distribution of Bayes factors under the null model M<sub>0</sub></span><div class="sb-div caption-clear"></div></div></div><p id="p-27">Following the detection of changing selection trajectories using ABC model choice, the quality of parameter estimation by the model chosen was evaluated. The cross-validation results from the haploid population of <em>N<sub>e</sub></em> = 100 are shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> and those from the diploid population of <em>N<sub>e</sub></em> = 500 in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS4.pdf">Figure S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS5.pdf">Figure S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS6.pdf">Figure S6</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS7.pdf">Figure S7</a> for additional results). For the case where the pseudoobservables were of constant selection, the estimation for a single <em>s</em> (and the dominance <em>h</em> for diploid) using CP-WFABC is very accurate, as the mode of the best simulations from the M<sub>0</sub> model for each pseudoobservable lies along the red diagonal line. Exceptions include uninformative trajectories where the allele surpasses the minimum frequency of 2% in the first sampling and is lost immediately due to genetic drift or negative selection and therefore not observed in subsequent samplings. Such uninformative trajectories will always keep the same set of best simulations from the M<sub>0</sub> model since their selection strength is indistinguishable, and they result in horizontal lines along the estimated negative value. This phenomenon is particularly pronounced when population size is small as shown in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS6.pdf">Figure S6</a>, since the role of genetic drift is more significant.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/893/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="ABC (approximate Bayesian computation) model choice parameter estimations for 1000 pseudoobservables with a haploid population of Ne = 100. Each circle is the mode of the posterior distribution from the 0.1% best simulations. Case 1: Pseudoobservables with constant selection (A) M0 estimates of s. Case 2: Pseudoobservables with changing selection (B) M1 estimates of s1, (C) M1 estimates of s2, and (D) M1 estimates of CP (green: positive s1 to positive s2, blue: positive s1 to negative s2, magenta: other cases)" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2009676308" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;ABC (approximate Bayesian computation) model choice parameter estimations for 1000 pseudoobservables with a haploid population of &lt;em&gt;N&lt;sub&gt;e&lt;/sub&gt;&lt;/em&gt; = 100. Each circle is the mode of the posterior distribution from the 0.1% best simulations. Case 1: Pseudoobservables with constant selection (A) M&lt;sub&gt;0&lt;/sub&gt; estimates of &lt;em&gt;s&lt;/em&gt;. Case 2: Pseudoobservables with changing selection (B) M&lt;sub&gt;1&lt;/sub&gt; estimates of &lt;em&gt;s&lt;sub&gt;1&lt;/sub&gt;&lt;/em&gt;, (C) M&lt;sub&gt;1&lt;/sub&gt; estimates of &lt;em&gt;s&lt;sub&gt;2&lt;/sub&gt;&lt;/em&gt;, and (D) M&lt;sub&gt;1&lt;/sub&gt; estimates of &lt;em&gt;CP&lt;/em&gt; (green: positive &lt;em&gt;s&lt;sub&gt;1&lt;/sub&gt;&lt;/em&gt; to positive &lt;em&gt;s&lt;sub&gt;2&lt;/sub&gt;&lt;/em&gt;, blue: positive &lt;em&gt;s&lt;sub&gt;1&lt;/sub&gt;&lt;/em&gt; to negative &lt;em&gt;s&lt;sub&gt;2&lt;/sub&gt;&lt;/em&gt;, magenta: other cases)&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/893/F3.medium.gif" width="440" height="412"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/4/893/F3.medium.gif" width="440" height="412"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/893/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/893/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467878" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-28" class="first-child">ABC (approximate Bayesian computation) model choice parameter estimations for 1000 pseudoobservables with a haploid population of <em>N<sub>e</sub></em> = 100. Each circle is the mode of the posterior distribution from the 0.1% best simulations. Case 1: Pseudoobservables with constant selection (A) M<sub>0</sub> estimates of <em>s</em>. Case 2: Pseudoobservables with changing selection (B) M<sub>1</sub> estimates of <em>s<sub>1</sub></em>, (C) M<sub>1</sub> estimates of <em>s<sub>2</sub></em>, and (D) M<sub>1</sub> estimates of <em>CP</em> (green: positive <em>s<sub>1</sub></em> to positive <em>s<sub>2</sub></em>, blue: positive <em>s<sub>1</sub></em> to negative <em>s<sub>2</sub></em>, magenta: other cases)</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/893/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="ABC (approximate Bayesian computation) model choice parameter estimations for 1000 pseudoobservables with a diploid population of Ne = 500. For cross-validation graphs, each circle is the mode of the posterior distribution from the 0.1% best simulations. For boxplots, red dots are true values and blue dots are average estimated values. Case 1: pseudoobservables with constant selection (A) M0 estimates of s and (B) M0 estimates of h. Case 2: pseudoobservables with changing selection (C) M1 estimates of s1, (D) M1 estimates of s2, (E) M1 estimates of CP, and (F) M1 estimates of h (green: positive s1 to positive s2, blue: positive s1 to negative s2, magenta: other cases)" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2009676308" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;ABC (approximate Bayesian computation) model choice parameter estimations for 1000 pseudoobservables with a diploid population of &lt;em&gt;N&lt;sub&gt;e&lt;/sub&gt;&lt;/em&gt; = 500. For cross-validation graphs, each circle is the mode of the posterior distribution from the 0.1% best simulations. For boxplots, red dots are true values and blue dots are average estimated values. Case 1: pseudoobservables with constant selection (A) M&lt;sub&gt;0&lt;/sub&gt; estimates of &lt;em&gt;s&lt;/em&gt; and (B) M&lt;sub&gt;0&lt;/sub&gt; estimates of &lt;em&gt;h&lt;/em&gt;. Case 2: pseudoobservables with changing selection (C) M&lt;sub&gt;1&lt;/sub&gt; estimates of &lt;em&gt;s&lt;sub&gt;1&lt;/sub&gt;&lt;/em&gt;, (D) M&lt;sub&gt;1&lt;/sub&gt; estimates of &lt;em&gt;s&lt;sub&gt;2&lt;/sub&gt;&lt;/em&gt;, (E) M&lt;sub&gt;1&lt;/sub&gt; estimates of &lt;em&gt;CP&lt;/em&gt;, and (F) M&lt;sub&gt;1&lt;/sub&gt; estimates of &lt;em&gt;h&lt;/em&gt; (green: positive &lt;em&gt;s&lt;sub&gt;1&lt;/sub&gt;&lt;/em&gt; to positive &lt;em&gt;s&lt;sub&gt;2&lt;/sub&gt;&lt;/em&gt;, blue: positive &lt;em&gt;s&lt;sub&gt;1&lt;/sub&gt;&lt;/em&gt; to negative &lt;em&gt;s&lt;sub&gt;2&lt;/sub&gt;&lt;/em&gt;, magenta: other cases)&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/893/F4.medium.gif" width="440" height="282"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/4/893/F4.medium.gif" width="440" height="282"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/893/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/893/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467918" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-29" class="first-child">ABC (approximate Bayesian computation) model choice parameter estimations for 1000 pseudoobservables with a diploid population of <em>N<sub>e</sub></em> = 500. For cross-validation graphs, each circle is the mode of the posterior distribution from the 0.1% best simulations. For boxplots, red dots are true values and blue dots are average estimated values. Case 1: pseudoobservables with constant selection (A) M<sub>0</sub> estimates of <em>s</em> and (B) M<sub>0</sub> estimates of <em>h</em>. Case 2: pseudoobservables with changing selection (C) M<sub>1</sub> estimates of <em>s<sub>1</sub></em>, (D) M<sub>1</sub> estimates of <em>s<sub>2</sub></em>, (E) M<sub>1</sub> estimates of <em>CP</em>, and (F) M<sub>1</sub> estimates of <em>h</em> (green: positive <em>s<sub>1</sub></em> to positive <em>s<sub>2</sub></em>, blue: positive <em>s<sub>1</sub></em> to negative <em>s<sub>2</sub></em>, magenta: other cases)</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">For the second case, when the pseudoobservables are of changing selection intensity, the joint estimation of the parameters is also effective for a restricted range of values. In <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS4.pdf">Figure S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS5.pdf">Figure S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS6.pdf">Figure S6</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS7.pdf">Figure S7</a>, the mode estimation of each pseudoobservable is color-coded according to the three categories of trajectory shape. The green dots are pseudoobservable trajectories that change from positive <em>s<sub>1</sub></em> to positive <em>s<sub>2</sub></em>. The blue dots are those that change from positive <em>s<sub>1</sub></em> to negative <em>s<sub>2</sub></em>, while the magenta colors include all other cases (<em>e.g.</em>, neutral or negative <em>s<sub>1</sub></em> to any value of <em>s<sub>2</sub></em>). There is a clear clustering by category – with the best estimation being of positive values of <em>s<sub>1</sub></em> below 0.5, moderate values of <em>s<sub>2</sub></em> between −0.5 and 0.5, and CP values for the blue category of positive <em>s<sub>1</sub></em> to negative <em>s<sub>2</sub></em>. In trajectories other than those with positive <em>s<sub>1</sub></em> to negative <em>s<sub>2</sub></em>, the change point is difficult to detect, particularly for diploid populations where the additional dominance parameter <em>h</em> was estimated (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS6.pdf">Figure S6</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS7.pdf">Figure S7</a>). This trend is also observed when the ROC curves are generated according to these three categories (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS3.pdf">Figure S3</a>). For all populations, the Bayes factor <em>B<sub>1,0</sub></em> is more sensitive and specific for trajectories changing from positive <em>s<sub>1</sub></em> to negative <em>s<sub>2</sub></em> (ROC curves in blue), reaching above 60% of the true positive rate when there are no false positives. Despite the restricted range of good parameter estimation in <em>s<sub>1</sub></em>, <em>s<sub>2</sub></em>, and <em>CP</em>, the estimation of dominance is robust for both cases of constant and changing selection (except for the small population size of <em>N<sub>e</sub></em> = 50; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS6.pdf">Figure S6</a>).</p><p id="p-31">In order to evaluate the performance of the joint parameter estimation, the coefficient of determination <em>R<sup>2</sup></em> is used to assess the cross-validation between the estimated values (<em>y<sub>m</sub></em>) and the true values (<em>f<sub>m</sub></em>), compared with the simple average of the estimated values (<span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-15.gif"/></noscript></span></span>). The closer the <em>R<sup>2</sup></em> value is to 1, the better the parameter estimation as shown:<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/893/embed/mml-math-16.gif"/></noscript></span><span class="disp-formula-label">(10)</span></span><a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> and <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> summarize the performance of the joint parameter estimation for all cases as the <em>R<sup>2</sup></em> values for the haploid and diploid populations, respectively. The first case is when the pseudoobservables are of constant selection intensity, in which case the true model (M<sub>0</sub>) performs only slightly better than the false model (M<sub>1</sub>) for estimating <em>s</em>. This discrepancy in parameter estimation of M<sub>0</sub> is mainly owing to uninformative pseudoobservable trajectories with constant selection (which have been lost or fixed) being associated with the true model (M<sub>0</sub>) of constant selection; this is due to the constraint for the allele to be segregating at the change point in the (false) model M<sub>1</sub> of changing selection. In the cross-validation of the constant selection case, the parameters estimated form horizontal lines at negative estimated values for those trajectories that are lost, and cluster at the top right corner for those trajectories that are fixed (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS4.pdf">Figure S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS5.pdf">Figure S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS6.pdf">Figure S6</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS7.pdf">Figure S7</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467925/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467925/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">R<sup>2</sup> values of the parameter estimation with the ABC (approximate Bayesian computation) model choice for haploid populations</span><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467926/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467926/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">R<sup>2</sup> values of the parameter estimation with the ABC (approximate Bayesian computation) model choice for diploid populations</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">For the second case in which the pseudoobservables have changing selection coefficients, parameter estimation from the true model (M<sub>1</sub>) performs better than that from the false model (M<sub>0</sub>) for all parameters – particularly when population sizes are large. As expected, there is a trend of better parameter estimation as population size increases (though see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS4.pdf">Figure S4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS5.pdf">Figure S5</a>). Additionally, it has been shown that the value of the Bayes factor <em>B<sub>1,0</sub></em> is a good indicator of the parameter estimation performance (results not shown).</p></div></div><div class="section" id="sec-8"><h2 class="">Data application</h2><div id="sec-9" class="subsection"><h3>Historical dataset of P. dominula</h3><p id="p-33">A long-running dataset based on the <em>medionigra</em> morph responsible for darker wing color in wild populations of <em>P. dominula</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS8.pdf">Figure S8</a>) began in 1939 with collections by <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Fisher and Ford (1947)</a> and continued through to 1999 (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cook and Jones 1996</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Jones 2000</a>). Despite this phenotypic time-serial data having been analyzed previously from various angles (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">O’Hara 2005</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Mathieson and McVean 2013</a>; <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Foll <em>et al.</em> 2015</a>), it is still relevant to consider a model of changing selection in time, as <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Wright (1948)</a> originally suggested.</p><p id="p-34">The recent reconsiderations of the dataset tend to favor a lethal-recessive model with an effective population of <em>N<sub>e</sub></em> = 500 (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Mathieson and McVean 2013</a>; <a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Foll <em>et al.</em> 2015</a>) – however, the biological question of how the <em>medionigra</em> morph could have reached the initial frequency of 11% in the dataset remains unanswered with this conclusion of constant strong negative selection. Wright asserted that the trajectory of the <em>medionigra</em> morph during this period could be explained by fluctuating selection with “no net selective advantage or disadvantage.” Although this alternative hypothesis has been considered as a random fluctuation of selection by estimating selection coefficients between every sampling time point (see <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">O’Hara 2005</a>), the quantitative plausibility of a directional change-in-<em>s</em> model over a single-<em>s</em> model lacks thorough investigation. Thus, we reanalyzed this dataset using the CP-WFABC method in order to investigate the possibility of changing selection in the <em>medionigra</em> morph during the 60-yr data collection.</p><p id="p-35">Using the ABC model choice introduced here as a test for a change in selection strength, and to estimate the parameters of interest for the chosen model, we assume the <em>medionigra</em> allele is a single codominant locus responsible for the homozygous and heterozygous expressions of the phenotypic forms <em>bimacula</em> and <em>medionigra</em>, respectively (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Cook and Jones 1996</a>). The model M<sub>0</sub> assumes a single selection coefficient, thus the only parameter to estimate is <em>s</em>. The M<sub>1</sub> model assumes a change in selection strength, thus the parameters of interest are <em>s1</em>, <em>s2</em>, and <em>CP</em>. Both M<sub>0</sub> and M<sub>1</sub> take the prior range of (−1, 1) for the selection coefficients and the prior range of (2, 59) for the change point in the M<sub>1</sub> model. For the M<sub>1</sub> model, these uninformative priors are updated with the constraint that the allele must be segregating at the time of change point. Here, we create 10<sup>7</sup> simulated datasets for each M<sub>0</sub> and M<sub>1</sub>, and apply the rejection algorithm of the CP-WFABC method to retain the best 1000 simulations compared with the observed trajectory. The effective population size is assumed to be <em>N<sub>e</sub></em> = 500 as in previous studies (<a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Wright 1948</a>; <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Cook and Jones 1996</a>; <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">O’Hara 2005</a>), with an initial allele frequency of 11% and a minimum frequency ascertainment of 2%.</p><p id="p-36">The Bayes factor for M<sub>1</sub> over M<sub>0</sub> is calculated as 0.952, indicating that the single coefficient M<sub>0</sub> model cannot be rejected in favor of the changing selection M<sub>1</sub> model (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). From the parameter estimation of the model M<sub>0</sub> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS9.pdf">Figure S9</a>), the mode of the posterior distribution for <em>s</em> is given as −0.15 as asserted by <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Fisher and Ford (1947)</a>. When the ABC model choice was repeated with a smaller population size of <em>N<sub>e</sub></em> = 50 as suggested by <a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">Wright (1948)</a> and <a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">O’Hara (2005)</a>, the Bayes factor increases to 1.87 (<em>i.e.</em>, the changing selection model is twice as likely as the constant selection) – however, this value is not large enough to be significant for a diploid population of <em>N<sub>e</sub></em> = 50 (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>).</p></div><div id="sec-10" class="subsection"><h3>Experimental evolution of influenza virus with drug treatment</h3><p id="p-37">The evolution of pathogens within a host is one of the most important cases in which the possibility of fluctuating selection must be considered – as they may experience drastically changing selective pressures due to host immune response, specific drug treatments, and/or pathogenic cooperation or competition (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Tanaka and Valckenborgh 2011</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hall <em>et al.</em> 2011</a>). Thus, how these pathogens adapt to these rapid external and internal changes is of major concern to the biomedical community.</p><p id="p-38">The time-serial experimental dataset of influenza A conducted by <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Renzette <em>et al.</em> (2014)</a> and <a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Foll <em>et al.</em> (2014)</a> is an interesting case study on the impact of drug treatment on influenza virus evolution. The dataset consists of 13 sampling points from which population-level whole-genome data were collected. Drug treatment with a commonly used neuraminidase inhibitor (oseltamivir) began after the collection of the third sample and continued, at increasing concentrations, until the final passage. Using WFABC, the genome-wide effective population size across the sampling time points was estimated (<em>N<sub>e</sub></em> = 176) and the SNPs under selection were identified.</p><p id="p-39">Here, we apply the CP-WFABC method on two cases of interest from this study to consider a possible change in selection strength under drug treatment: the first case includes trajectories identified as being driven by positive selection, while the second includes outlier trajectories (<em>i.e.</em>, trajectories not fitting a single <em>s</em> Wright-Fisher model). For all cases, we test the model M<sub>0</sub> (<em>i.e.</em>, a single selection coefficient) and M<sub>1</sub> (<em>i.e.</em>, a changing selection coefficient), with parameters of interest (<em>s</em>) and (<em>s1</em>, <em>s2</em>, <em>CP</em>), respectively. The number of generations per passage is assumed to be 13 (following <a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">Foll <em>et al.</em> 2014</a>), and the minimum frequency of 2% is set as an ascertainment for observing the minor allele in the data. <em>De novo</em> mutations are assumed to occur at the sampling time point where the SNP was first sampled (except for trajectories whose initial sample frequency was first sampled above 1/<em>N<sub>e</sub></em>, which are assumed to be standing variation). This assumption is based on the high mutation rate, large population bottlenecks associated with passaging, and large census population size between passages. A total of 10<sup>7</sup> datasets were simulated for each M<sub>0</sub> and M<sub>1</sub>; the best 1000 trajectories from the lot were retained using the rejection algorithm described in the <em>Methods</em> section. The uniform prior ranges for the selection coefficients were set as (−1, 1), for the change point as (<em>t<sub>0</sub></em>, 157) where <em>t<sub>0</sub></em> is the generation when the SNP was first sampled, and with the constraint of segregating alleles at the change point for M<sub>1</sub>.</p><p id="p-40">The results for the Bayes factors and the parameter estimates are summarized in <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a> for all trajectories of interest. The Bayes factors of most trajectories show strong support for the changing selection model: the stronger the selection strength change, the larger the Bayes factor. Using the Bayes factors from the simulation studies as guidance (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>), the significance threshold to reject M<sub>0</sub> is computed as 3.7 for a small haploid population. As expected, the trajectories identified as outliers of the single <em>s</em> Wright-Fisher model (NP 159, PB1 33) all reject the constant selection model M<sub>0</sub> with a large Bayes factor. We also note that the Bayes factor for the drug-resistant mutation H275Y (NA 823) does not support the changing selection model strongly, confirming that the experimental evolution procedure kept the selective pressure of the drug constant by adjusting the drug concentration to reduce viral plaque numbers to 50% at each passage. As shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, the change points are estimated to be mostly between the seventh and eighth passages, a notable result since three of these trajectories (HA 48, HA 1395, NA 582) are increasing rapidly after the drug-resistant mutation H275Y appears, whereas one trajectory (NP 159) from a different segment decreases rapidly. This result may indicate that positive selective for the three SNPs (including HA 1395; a known compensatory mutation encoded also as D112N) increased along with the drug-resistant mutation H275Y, potentially due to epistatic interactions, whereas another SNP (NP 159) decreased at that time, potentially owing to clonal interference. The single selection estimates from WFABC (<a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">Foll <em>et al.</em> 2015</a>) are similar to the estimates of CP-WFABC only when the Bayes factor does not reject M<sub>0</sub> – strong evidence that an alternative model of changing selection must be considered for some trajectories in order to correctly estimate selection coefficients.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467874/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467874/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Bayes factors and parameters estimated for the influenza trajectories in the presence and absence of drug</span><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/893/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Change points indicated with solid stars for the trajectories of interest. (A) Increasing SNP trajectories in the presence of drug. The red vertical line indicates the sampling time of drug administration. (B) Increasing SNP trajectories in the absence of drug." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2009676308" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Change points indicated with solid stars for the trajectories of interest. (A) Increasing SNP trajectories in the presence of drug. The red vertical line indicates the sampling time of drug administration. (B) Increasing SNP trajectories in the absence of drug.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/893/F5.medium.gif" width="337" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/4/893/F5.medium.gif" width="337" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/893/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/893/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467867" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-44" class="first-child">Change points indicated with solid stars for the trajectories of interest. (A) Increasing SNP trajectories in the presence of drug. The red vertical line indicates the sampling time of drug administration. (B) Increasing SNP trajectories in the absence of drug.</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">We also applied CP-WFABC to the control case of SNPs increasing in frequency without drug as a comparison to the case with drug. The effective population sizes of the viral populations were averaged to be 226 in the absence of drug from the previous study (<a id="xref-ref-10-7" class="xref-bibr" href="#ref-10">Foll <em>et al.</em> 2014</a>). All of the other inputs and the ABC model choice conditions were kept the same as in the drug case. The Bayes factor results summarized in <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a> demonstrate that none of the SNP trajectories under selection in the control experiment can reject M<sub>0</sub> (<em>i.e.</em>, constant selection).</p></div></div><div class="section discussion" id="sec-11"><h2 class="">Discussion</h2><p id="p-46">These simulations demonstrate that the novel CP-WFABC approach presented here is able to detect changing selection trajectories via ABC model choice, and also to estimate a wide range of parameters of interest. Performance was analyzed separately for three categories of allele trajectories according to the nature of the change in selection strength: (1) a change from positive <em>s<sub>1</sub></em> to positive <em>s<sub>2</sub></em>, (2) a change from positive <em>s<sub>1</sub></em> to negative <em>s<sub>2</sub></em>, and (3) all other changes. The datasets for each possible combination were generated using the Wright-Fisher model with a change in selection strength, using the most general prior ranges for all parameters <em>s<sub>1</sub></em>, <em>s<sub>2</sub></em>, <em>CP</em>, and <em>h</em> for diploids, with the only constraint being segregation of the allele at the change point. For both the detection and parameter estimation, CP-WFABC performs best when the change is large, particularly for the second category of change (positive <em>s<sub>1</sub></em> to negative <em>s<sub>2</sub></em>), as shown in the ROC curves (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS3.pdf">Figure S3</a>) and the cross-validation graphs (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS4.pdf">Figure S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS5.pdf">Figure S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS6.pdf">Figure S6</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS7.pdf">Figure S7</a>). For the first category (positive <em>s<sub>1</sub></em> to positive <em>s<sub>2</sub></em>) and the third category (any other changes), the change point is difficult to estimate, particularly for diploids where the additional parameter <em>h</em> is also estimated. The ABC model choice of CP-WFABC has the best sensitivity for full specificity, for larger population sizes (<em>N<sub>e</sub></em> &gt; 500 for diploids), and for haploid populations.</p><p id="p-47">The parameter estimates of <em>s<sub>1</sub></em> and <em>s<sub>2</sub></em> perform best when the values are moderate – given as the intervals (0, 0.5) and (−0.5, 0.5) respectively. For <em>s<sub>1</sub></em>, the optimal parameter range for estimation is (0, 0.5), where a <em>de novo</em> mutation that survives negative selection and segregates until the change point is indistinguishable from other drifting mutations with similar trajectories with uninformative low allele frequency. These trajectories naturally arise more frequently when population size is small and in diploids where the dominance effect plays a role, as shown in the third category of change (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS4.pdf">Figure S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS5.pdf">Figure S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS6.pdf">Figure S6</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS7.pdf">Figure S7</a>; magenta points). When an initial frequency of 10% is used instead of a <em>de novo</em> mutation, the advantage of having a more informative trajectory at the beginning is counteracted by the effect of more cases dominated by negative selection or genetic drift segregating until the change point. Thus, the performance of CP-WFABC for standing variation is similar to that of <em>de novo</em> mutation. For <em>s<sub>2</sub></em>, the optimal parameter range for estimation is (−0.5, 0.5), as trajectories with extreme values are less informative since they are lost or fixed directly after the change point (explaining the clustering of the change points at earlier times). For diploid populations, estimates of <em>h</em> are accurate to the level of determining dominance from codominance or recessiveness, particularly for large population sizes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS7.pdf">Figure S7</a>), given the difficulty of joint estimation with the three other parameters. Indeed, estimation of this additional parameter comes at the cost of worse performance for the other parameters, as can be seen in the ROC curves and cross-validation graphs: the detection and parameter estimation of changing selection cases is always better for haploids. Thus, in diploid cases, we recommend fixing the dominance parameter if known, in order to improve the performance of CP-WFABC.</p><p id="p-48">Although CP-WFABC is intended to detect and evaluate changing selection intensities, the simulation studies demonstrate that the method also performs well in estimating parameters for cases of constant selection (as has been demonstrated by <a id="xref-ref-10-8" class="xref-bibr" href="#ref-10">Foll <em>et al.</em> 2014</a>). For haploids with large population sizes, in particular, the estimated values of the single parameter <em>s</em> correlate almost perfectly with the true values (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS5.pdf">Figure S5</a>). However, when the population size is small for both diploids and haploids, some trajectories that are lost by genetic drift are difficult to estimate, as shown in <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS6.pdf">Figure S6</a> as horizontal lines along negative estimated values. This limitation of constant selection coefficients, however, is due to the simulation conditions of the <em>de novo</em> mutation at the first generation and the minimal ascertainment scheme (minimum frequency of 2% at one of the sampling time points). For real datasets, the conditions are likely to be less stringent, since such uninformative trajectories will not be considered for parameter estimation.</p><p id="p-49">Finally, we utilized this approach to make inference in two very different time-sampled datasets: <em>P. dominula</em> (diploid) and Influenza A virus (haploid). The time-serial <em>medionigra</em> trajectory of <em>P. dominula</em> was reanalyzed to test for a change in selection strength and/or direction during 60 yr of data collection. By assuming <em>h</em> as codominant, the results for <em>N<sub>e</sub></em> = 500 indicate that the model M<sub>0</sub> of constant selection cannot be rejected according to the Bayes factor from the ABC model choice algorithm. The selection coefficient from this model is estimated as −0.15, corresponding with that calculated by <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Fisher and Ford (1947)</a>. However, when the population size is assumed to be smaller (<em>N<sub>e</sub></em> = 50), the Bayes factor result supports M<sub>1</sub> (changing selection) twice as strongly as M<sub>0</sub> (constant selection), but not strong enough to reject M<sub>0</sub> according to the significance level test computed with the distribution of Bayes factors under the null model M<sub>0</sub>. This dataset of the <em>medionigra</em> morph thus demonstrates the difficulty of detecting and evaluating a change in selection when the assumption of a known and constant effective population size is not fulfilled. Depending on the assumed population sizes, the conclusions may indeed change, as this study demonstrates. Furthermore, if the population size fluctuates, this method has the caveat of detecting a change in <em>N<sub>e</sub></em>*<em>s</em> rather than only in selection strength.</p><p id="p-50">Next, CP-WFABC was applied to SNP trajectories of interest from an experimental dataset of influenza A virus. The ABC model choice test was conducted on the trajectories identified as 1) being positively selected and 2) as outliers from the single-<em>s</em> WFABC method. For the SNPs in the presence of drug, the Bayes factor for six out of nine trajectories favored the changing selection model M<sub>1</sub>. As shown in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>, the change points for four out of these six trajectories occurred between passages seven and eight – the interval during which three trajectories from the Hemagglutinin and Neuraminidase segments (HA and NA, respectively) increased rapidly while one trajectory from the segment NP decreased rapidly along with the known drug-resistant mutation NA 823 (H275Y). These results appear to support the presence of epistasis and/or clonal interference, where the selection strength of the other SNPs is influenced by the appearance of a drug-resistant mutation under drug pressure. In fact, a known compensatory mutation HA 1395 (D112N) for infectivity (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Thoennes <em>et al.</em> 2008</a>) was among the three trajectories increasing rapidly, further confirming that increased infectivity might contribute to tissue culture adaptation, thus reinforcing the use of the method to evaluate biological hypotheses. Moreover, the estimated values of selection coefficients differed greatly between the constant-selection method (WFABC) and CP-WFABC. In particular, the estimates of <em>s</em> for PA 2194 and NP 159 from the Wright-Fisher model were inferred by WFABC as being near zero (<em>i.e.</em>, neutral), though the more robust CP-WFABC estimates here indicate fluctuation of <em>s</em> from negative to positive values. Thus, these fluctuations cannot be explained by genetic drift alone, as previously speculated. We have therefore identified some cases where an alternative model of changing selection is essential for correctly estimating selection parameters and identifying change points. In the absence of drug treatment, the Bayes factor test could not reject the constant selection model M<sub>0</sub> for any of the identified SNPs suggesting that in, the absence of the drug, the selective pressures on the population are largely constant as expected.</p><p id="p-51">The simulation studies of changing selection reveal some important points to consider from the standpoint of experimental design. For CP-WFABC, the parameter estimation of a single selection coefficient between two sampling time points performs reasonably well for haploid population sizes above <em>N<sub>e</sub></em> = 1000. However, it is advisable to have three sampling time points to enhance the parameter estimation by maximizing the information contained in the allele trajectory for both selection periods, particularly for diploids and in smaller population sizes (<em>N<sub>e</sub></em> &lt; 1000). Thus, in order for a change to be detectable, it is required to have at least four sampling time points where the change must occur between the second and third sampling time points. The simulation studies of CP-WFABC confirm that the estimation of <em>CP</em> performs best at the intermediate range of time-sampled data, as any change happening before the second and after the second-to-last sampling time point is impossible to detect (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS4.pdf">Figure S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS5.pdf">Figure S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS6.pdf">Figure S6</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1/FigureS7.pdf">Figure S7</a>). Finally, it remains a future challenge to expand this method to consider more than one change point in selection strength, as some of the trajectories in the influenza A application (such as PA 2194 and PB1 33) suggest the presence of several change points along the trajectory, and to resolve the problem arising from the assumption of constant population size.</p></div><div class="section" id="sec-12"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-52">We thank Kristen Irwin for helpful comments. The computations were performed at Vital-IT (<a href="http://www.vital-it.ch">http://www.vital-it.ch</a>) in the Swiss Institute of Bioinformatics. This work was funded by grants from the Swiss National Science Foundation and a European Research Council starting grant to J.D.J.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-53">Supplemental Material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023200/-/DC1</a></p></li><li class="fn" id="fn-6"><p id="p-54"><em>Communicating editor: Y. Kim</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-09-29"><span class="received-label">Received </span>September 29, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-01-28"><span class="accepted-label">Accepted </span>January 28, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Shim <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.1" data-doi="10.1016/j.tig.2014.09.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bank</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ewing</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrer-Admettla</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foll</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Thinking Too Positive? Revisiting Current Methods of Population Genetic Selection Inference.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">540</span>–<span class="cit-lpage">546</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D30%26rft.spage%253D540%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2014.09.010%26rft_id%253Dinfo%253Apmid%252F25438719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2014.09.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25438719&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.2" data-doi="10.1146/annurev-ecolsys-102209-144621"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beaumont</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Approximate Bayesian Computation in Evolution and Ecology.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">379</span>–<span class="cit-lpage">406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEcol.%2BEvol.%2BSyst.%26rft.volume%253D41%26rft.spage%253D379%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-ecolsys-102209-144621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-ecolsys-102209-144621&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000284978200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beaumont</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balding</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Approximate Bayesian Computation in Population Genetics.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">2025</span>–<span class="cit-lpage">2035</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBeaumont%26rft.auinit1%253DM.%2BA.%26rft.volume%253D162%26rft.issue%253D4%26rft.spage%253D2025%26rft.epage%253D2035%26rft.atitle%253DApproximate%2BBayesian%2BComputation%2Bin%2BPopulation%2BGenetics%26rft_id%253Dinfo%253Apmid%252F12524368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Mi80LzIwMjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvODkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.4" data-doi="10.1098/rstb.2009.0150"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fluctuating Selection: The Perpetual Renewal of Adaptation in Variable Environments.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">365</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">97</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DBell%26rft.auinit1%253DG.%26rft.volume%253D365%26rft.issue%253D1537%26rft.spage%253D87%26rft.epage%253D97%26rft.atitle%253DFluctuating%2Bselection%253A%2Bthe%2Bperpetual%2Brenewal%2Bof%2Badaptation%2Bin%2Bvariable%2Benvironments%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2009.0150%26rft_id%253Dinfo%253Apmid%252F20008388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjExOiIzNjUvMTUzNy84NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC84OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.893.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">A. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-source"><em>Parametric Statistical Change Point Analysis</em></span>, <span class="cit-edition">Ed. 2nd</span>. <span class="cit-publ-name">Birkhauser</span>, <span class="cit-publ-loc">Berlin</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-other" id="cit-6.4.893.6" data-doi="10.1111/j.1558-5646.2012.01809.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chevin</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> Genetic constraints on adaptation to a changing environment. <span class="cit-source">Evolution</span> <span class="cit-vol">67</span>: <span class="cit-fpage">708</span>–<span class="cit-lpage">721</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D67%26rft.spage%253D708%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2012.01809.x%26rft_id%253Dinfo%253Apmid%252F23461322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2012.01809.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23461322&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.7" data-doi="10.1098/rstb.1996.0146"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The Medionigra Gene in the Moth Panaxia Dominula: The Case for Selection.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">351</span>: <span class="cit-fpage">1623</span>–<span class="cit-lpage">1634</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DCook%26rft.auinit1%253DL.%2BM.%26rft.volume%253D351%26rft.issue%253D1347%26rft.spage%253D1623%26rft.epage%253D1634%26rft.atitle%253DThe%2Bmedionigra%2BGene%2Bin%2Bthe%2BMoth%2BPanaxia%2Bdominula%253A%2BThe%2BCase%2Bfor%2BSelection%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.1996.0146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEzOiIzNTEvMTM0Ny8xNjIzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzg5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.893.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ewens</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-source"><em>Mathematical Population Genetics 1 - Theoretical Introduction</em></span>, <span class="cit-edition">Ed. 2nd</span>. <span class="cit-publ-name">Springer-Velag</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.9" data-doi="10.1038/hdy.1947.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ford</span>  <span class="cit-name-given-names">E. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1947</span> <span class="cit-article-title">The Spread of a Gene in Natural Conditions in a Colony of the Moth PANAXIA DOMINULA L.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">143</span>–<span class="cit-lpage">174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D1%26rft.spage%253D143%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1947.11%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1947.11&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1947YB41500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.10" data-doi="10.1371/journal.pgen.1004185"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foll</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poh</span>  <span class="cit-name-given-names">Y.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Renzette</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrer-Admetlla</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bank</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Influenza Virus Drug Resistance: A Time-Sampled Population Genetics Perspective.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004185</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004185%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004185%26rft_id%253Dinfo%253Apmid%252F24586206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004185&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24586206&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.11" data-doi="10.1111/1755-0998.12280"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foll</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shim</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">WFABC: A Wright-Fisher ABC-Based Approach for Inferring Effective Population Sizes and Selection Coefficients from Time-Sampled Data.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">98</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D15%26rft.spage%253D87%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12280%26rft_id%253Dinfo%253Apmid%252F24834845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12280&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24834845&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gompert</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Bayesian Inference of Selection in a Heterogeneous Environment from Genetic Time-Series Data.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> (<span class="cit-comment">in press</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.13" data-doi="10.2307/2290192"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Good</span>  <span class="cit-name-given-names">I. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">The Bayes/Non-Bayes Compromise: A Brief Review.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">597</span>–<span class="cit-lpage">606</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D87%26rft.spage%253D597%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2290192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2290192&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.14" data-doi="10.1371/journal.pone.0089680"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gossmann</span>  <span class="cit-name-given-names">T. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waxman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Fluctuating Selection Models and Mcdonald-Kreitman Type Analyses.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0089680%26rft_id%253Dinfo%253Apmid%252F24551193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0089680&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24551193&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haldane</span>  <span class="cit-name-given-names">J. B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1932</span> <span class="cit-article-title">A Mathematical Theory of Natural and Artificial Selection. Part IX. Rapid Selection.</span> <abbr class="cit-jnl-abbrev">Math. Proc. Camb. Philos. Soc.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">224</span>–<span class="cit-lpage">248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMath.%2BProc.%2BCamb.%2BPhilos.%2BSoc.%26rft.volume%253D28%26rft.spage%253D224%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.16" data-doi="10.1111/j.1461-0248.2011.01624.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scanlan</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckling</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Host-Parasite Coevolutionary Arms Races Give Way to Fluctuating Selection.</span> <abbr class="cit-jnl-abbrev">Ecol. Lett.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">635</span>–<span class="cit-lpage">642</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DHall%26rft.auinit1%253DA.%2BR.%26rft.volume%253D14%26rft.issue%253D7%26rft.spage%253D635%26rft.epage%253D642%26rft.atitle%253DHost-parasite%2Bcoevolutionary%2Barms%2Braces%2Bgive%2Bway%2Bto%2Bfluctuating%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2011.01624.x%26rft_id%253Dinfo%253Apmid%252F21521436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2011.01624.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21521436&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.17" data-doi="10.1046/j.1365-2540.2000.00700.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Temperatures in the Cothill Habitat of Panaxia (Callimorpha) Dominula L. (the Scarlet Tiger Moth).</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">578</span>–<span class="cit-lpage">586</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D84%26rft.spage%253D578%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2540.2000.00700.x%26rft_id%253Dinfo%253Apmid%252F10849083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2540.2000.00700.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10849083&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.18" data-doi="10.1534/genetics.107.075481"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jorde</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ryman</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Unbiased Estimator for Genetic Drift and Effective Population Size.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">927</span>–<span class="cit-lpage">935</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.075481%26rft_id%253Dinfo%253Apmid%252F17720927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc3LzIvOTI3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzg5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.19" data-doi="10.1016/0040-5809(74)90017-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karlin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levikson</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">Temporal Fluctuations in Selection Intensities: Case of Small Population Size.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">383</span>–<span class="cit-lpage">412</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BPopul.%2BBiol.%26rft.volume%253D6%26rft.spage%253D383%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252874%252990017-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(74)90017-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.20" data-doi="10.1016/0040-5809(74)90016-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karlin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lieberman</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">Random Temporal Variation in Selection Intensities: Case of Large Population Size.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">382</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DKarlin%26rft.auinit1%253DS.%26rft.volume%253D6%26rft.issue%253D3%26rft.spage%253D355%26rft.epage%253D382%26rft.atitle%253DRandom%2Btemporal%2Bvariation%2Bin%2Bselection%2Bintensities%253A%2Bcase%2Bof%2Blarge%2Bpopulation%2Bsize.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252874%252990016-1%26rft_id%253Dinfo%253Apmid%252F4460262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(74)90016-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4460262&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1974V164400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1954</span> <span class="cit-article-title">Process Leading to Quasi-Fixation of Genes in Natural Populations Due to Random Fluctuation of Selection Intensities.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">280</span>–<span class="cit-lpage">295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKimura%26rft.auinit1%253DM.%26rft.volume%253D39%26rft.issue%253D3%26rft.spage%253D280%26rft.epage%253D295%26rft.atitle%253DPROCESS%2BLEADING%2BTO%2BQUASI-FIXATION%2BOF%2BGENES%2BIN%2BNATURAL%2BPOPULATIONS%2BDUE%2BTO%2BRANDOM%2BFLUCTUATION%2BOF%2BSELECTION%2BINTENSITIES%26rft_id%253Dinfo%253Apmid%252F17247483%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIzOS8zLzI4MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC84OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.22" data-doi="10.1534/genetics.114.167957"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lacerda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seoighe</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Population Genetics Inference for Longitudinally-Sampled Mutants under Strong Selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">1237</span>–<span class="cit-lpage">1250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.167957%26rft_id%253Dinfo%253Apmid%252F25213172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OC8zLzEyMzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvODkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.23" data-doi="10.1534/genetics.112.140939"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malaspinas</span>  <span class="cit-name-given-names">A.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malaspinas</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Estimating Allele Age and Selection Coefficient from Time-Serial Data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">599</span>–<span class="cit-lpage">607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.140939%26rft_id%253Dinfo%253Apmid%252F22851647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzIvNTk5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzg5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.24" data-doi="10.1534/genetics.112.147611"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mathieson</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Estimating Selection Coefficients in Spatially Structured Populations from Time Series Data of Allele Frequencies.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">973</span>–<span class="cit-lpage">984</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.147611%26rft_id%253Dinfo%253Apmid%252F23307902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzMvOTczIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzg5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.25" data-doi="10.1098/rspb.2004.2929"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Hara</span>  <span class="cit-name-given-names">R. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Comparing the Effects of Genetic Drift and Fluctuating Selection on Genotype Frequency Changes in the Scarlet Tiger Moth.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">272</span>: <span class="cit-fpage">211</span>–<span class="cit-lpage">217</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DO%2527Hara%26rft.auinit1%253DR.%2BB.%26rft.volume%253D272%26rft.issue%253D1559%26rft.spage%253D211%26rft.epage%253D217%26rft.atitle%253DComparing%2Bthe%2Beffects%2Bof%2Bgenetic%2Bdrift%2Band%2Bfluctuating%2Bselection%2Bon%2Bgenotype%2Bfrequency%2Bchanges%2Bin%2Bthe%2Bscarlet%2Btiger%2Bmoth%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2004.2929%26rft_id%253Dinfo%253Apmid%252F15695213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMjoiMjcyLzE1NTkvMjExIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzg5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.26" data-doi="10.1093/biomet/41.1-2.100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Page</span>  <span class="cit-name-given-names">B. Y. E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1954</span> <span class="cit-article-title">Continuous Inspection Schemes.</span> <abbr class="cit-jnl-abbrev">Biometrika</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">100</span>–<span class="cit-lpage">115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft.stitle%253DBiometrika%26rft.aulast%253DPAGE%26rft.auinit1%253DE.%2BS.%26rft.volume%253D41%26rft.issue%253D1-2%26rft.spage%253D100%26rft.epage%253D115%26rft.atitle%253DCONTINUOUS%2BINSPECTION%2BSCHEMES%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252F41.1-2.100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo2OiJiaW9tZXQiO3M6NToicmVzaWQiO3M6MTA6IjQxLzEtMi8xMDAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvODkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.27" data-doi="10.1128/JVI.01067-13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Renzette</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caffrey</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeldovich</span>  <span class="cit-name-given-names">K. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallagher</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Evolution of the Influenza A Virus Genome during Development of Oseltamivir Resistance in Vitro.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">272</span>–<span class="cit-lpage">281</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01067-13%26rft_id%253Dinfo%253Apmid%252F24155392%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6Ijg4LzEvMjcyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzg5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.893.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ryan</span>  <span class="cit-name-given-names">T. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-source"><em>Statistical Methods for Quality Improvement</em></span>, <span class="cit-edition">Ed. 3rd</span>. <span class="cit-publ-name">Wiley, New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.29" data-doi="10.1371/journal.pcbi.1002803"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sunnåker</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Busetto</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Numminen</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corander</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foll</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Approximate Bayesian Computation.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1002803</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D9%26rft.spage%253De1002803%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002803%26rft_id%253Dinfo%253Apmid%252F23341757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002803&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23341757&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.30" data-doi="10.1111/j.1558-5646.2011.01223.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valckenborgh</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Escaping an Evolutionary Lobster Trap: Drug Resistance and Compensatory Mutation in a Fluctuating Environment.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">1376</span>–<span class="cit-lpage">1387</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DTanaka%26rft.auinit1%253DM.%2BM.%26rft.volume%253D65%26rft.issue%253D5%26rft.spage%253D1376%26rft.epage%253D1387%26rft.atitle%253DEscaping%2Ban%2Bevolutionary%2Blobster%2Btrap%253A%2Bdrug%2Bresistance%2Band%2Bcompensatory%2Bmutation%2Bin%2Ba%2Bfluctuating%2Benvironment.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2011.01223.x%26rft_id%253Dinfo%253Apmid%252F21521192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2011.01223.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21521192&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289893000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.31" data-doi="10.1016/j.virol.2007.08.035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thoennes</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.-N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">B.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langley</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skehel</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Analysis of Residues near the Fusion Peptide in the Influenza Hemagglutinin Structure for Roles in Triggering Membrane Fusion.</span> <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">370</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DThoennes%26rft.auinit1%253DS.%26rft.volume%253D370%26rft.issue%253D2%26rft.spage%253D403%26rft.epage%253D414%26rft.atitle%253DAnalysis%2Bof%2Bresidues%2Bnear%2Bthe%2Bfusion%2Bpeptide%2Bin%2Bthe%2Binfluenza%2Bhemagglutinin%2Bstructure%2Bfor%2Broles%2Bin%2Btriggering%2Bmembrane%2Bfusion.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2007.08.035%26rft_id%253Dinfo%253Apmid%252F17936324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2007.08.035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17936324&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251698800019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.32" data-doi="10.2307/2405519"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1948</span> <span class="cit-article-title">On the Roles of Directed and Random Changes in Gene Frequency in the Genetics of Populations.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">279</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DWright%26rft.auinit1%253DS.%26rft.volume%253D2%26rft.issue%253D4%26rft.spage%253D279%26rft.epage%253D294%26rft.atitle%253DOn%2Bthe%2Broles%2Bof%2Bdirected%2Band%2Brandom%2Bchanges%2Bin%2Bgene%2Bfrequency%2Bin%2Bthe%2Bgenetics%2Bof%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2405519%26rft_id%253Dinfo%253Apmid%252F18104586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2405519&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18104586&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1948XW56800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.893.33" data-doi="10.1186/1471-2105-12-77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xavier</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turck</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hainard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiberti</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisacek</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">pROC: An Open-Source Package for R and S+ to Analyze and Compare ROC Curves.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">77</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DRobin%26rft.auinit1%253DX.%26rft.volume%253D12%26rft.spage%253D77%26rft.epage%253D77%26rft.atitle%253DpROC%253A%2Ban%2Bopen-source%2Bpackage%2Bfor%2BR%2Band%2BS%252B%2Bto%2Banalyze%2Band%2Bcompare%2BROC%2Bcurves.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-77%26rft_id%253Dinfo%253Apmid%252F21414208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-77&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21414208&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/4/893.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/4/881" title="Barcode Sequencing Screen Identifies SUB1 as a Regulator of Yeast Pheromone Inducible Genes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/4/905" title="Genomic Access to Monarch Migration Using TALEN and CRISPR/Cas9-Mediated Targeted Mutagenesis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/4"><span>Volume 6 Issue 4, April 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/4.cover-source.jpg?itok=Lj45nPaf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/893" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-EgjOP0euGNPJjgoHnhfX92mAJqSQIzRZ8eNQHnJd_lY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/4/893" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/4/893" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/4/893" class="active" data-icon-position="" data-hide-link-title="0">Detecting and Quantifying Changing Selection Intensities from Time-Sampled Polymorphism Data</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460106" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-sPNAdq4vfk39K3xGNEpu2Z7HQd9xw1TGbYomuRP31f4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/4/893.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/4/893" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-RET0sardnCDPa8ZkiBD9qjKD5pihGxFZi6u03Tgu7io" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/4/893" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/4/893.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460106" id="node460106--2" data-pisa="ggg;6/4/893" data-pisa-master="ggg;g3.115.023200" data-apath="/ggg/6/4/893.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Detecting and Quantifying Changing Selection Intensities from Time-Sampled Polymorphism Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7052-0971" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Hyunjin Shim</span>, <span class="highwire-citation-author" data-delta="1">Stefan Laurent</span>, <span class="highwire-citation-author" data-delta="2">Sebastian Matuszewski</span>, <span class="highwire-citation-author" data-delta="3">Matthieu Foll</span> and <span class="highwire-citation-author" data-delta="4">Jeffrey D. Jensen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">893-904;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.023200 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460106--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hyunjin Shim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Swiss Institute of Bioinformatics, Lausanne, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hyunjin%2BShim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shim%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHyunjin%2BShim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7052-0971" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hyunjin Shim</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Stefan Laurent </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Swiss Institute of Bioinformatics, Lausanne, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BLaurent%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Laurent%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStefan%2BLaurent%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sebastian Matuszewski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Swiss Institute of Bioinformatics, Lausanne, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebastian%2BMatuszewski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Matuszewski%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebastian%2BMatuszewski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Matthieu Foll </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Swiss Institute of Bioinformatics, Lausanne, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Agency for Research on Cancer, Lyon, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthieu%2BFoll%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Foll%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthieu%2BFoll%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jeffrey D. Jensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Swiss Institute of Bioinformatics, Lausanne, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BD.%2BJensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jensen%20JD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeffrey%2BD.%2BJensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/4/893" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/4/893&amp;title=Detecting%20and%20Quantifying%20Changing%20Selection%20Intensities%20from%20Time-Sampled%20Polymorphism%20Data" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/4/893&amp;title=Detecting%20and%20Quantifying%20Changing%20Selection%20Intensities%20from%20Time-Sampled%20Polymorphism%20Data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/4/893&amp;title=Detecting%20and%20Quantifying%20Changing%20Selection%20Intensities%20from%20Time-Sampled%20Polymorphism%20Data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/4/893&amp;text=Detecting%20and%20Quantifying%20Changing%20Selection%20Intensities%20from%20Time-Sampled%20Polymorphism%20Data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/4/893&amp;title=Detecting%20and%20Quantifying%20Changing%20Selection%20Intensities%20from%20Time-Sampled%20Polymorphism%20Data" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/4/893&amp;t=Detecting%20and%20Quantifying%20Changing%20Selection%20Intensities%20from%20Time-Sampled%20Polymorphism%20Data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/4/893&amp;title=Detecting%20and%20Quantifying%20Changing%20Selection%20Intensities%20from%20Time-Sampled%20Polymorphism%20Data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/4/893&amp;title=Detecting%20and%20Quantifying%20Changing%20Selection%20Intensities%20from%20Time-Sampled%20Polymorphism%20Data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460106/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460106" id="node460106--3" data-pisa="ggg;6/4/893" data-pisa-master="ggg;g3.115.023200" data-apath="/ggg/6/4/893.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Detecting and Quantifying Changing Selection Intensities from Time-Sampled Polymorphism Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7052-0971" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Hyunjin Shim</span>, <span class="highwire-citation-author" data-delta="1">Stefan Laurent</span>, <span class="highwire-citation-author" data-delta="2">Sebastian Matuszewski</span>, <span class="highwire-citation-author" data-delta="3">Matthieu Foll</span> and <span class="highwire-citation-author" data-delta="4">Jeffrey D. Jensen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">893-904;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.023200 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460106--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hyunjin Shim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Swiss Institute of Bioinformatics, Lausanne, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hyunjin%2BShim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shim%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHyunjin%2BShim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7052-0971" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hyunjin Shim</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Stefan Laurent </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Swiss Institute of Bioinformatics, Lausanne, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BLaurent%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Laurent%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStefan%2BLaurent%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sebastian Matuszewski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Swiss Institute of Bioinformatics, Lausanne, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebastian%2BMatuszewski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Matuszewski%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebastian%2BMatuszewski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Matthieu Foll </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Swiss Institute of Bioinformatics, Lausanne, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Agency for Research on Cancer, Lyon, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthieu%2BFoll%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Foll%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthieu%2BFoll%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jeffrey D. Jensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Swiss Institute of Bioinformatics, Lausanne, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BD.%2BJensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jensen%20JD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeffrey%2BD.%2BJensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=893&amp;rft.epage=904&amp;rft.atitle=Detecting%20and%20Quantifying%20Changing%20Selection%20Intensities%20from%20Time-Sampled%20Polymorphism%20Data&amp;rft.volume=6&amp;rft.issue=4&amp;rft.date=2016-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Shim%2C+Hyunjin&amp;rft.au=Laurent%2C+Stefan&amp;rft.au=Matuszewski%2C+Sebastian&amp;rft.au=Foll%2C+Matthieu&amp;rft.au=Jensen%2C+Jeffrey+D."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460106/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460106/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460106/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460106/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460106/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460106/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460106/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460106/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460106/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460106/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460106/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460106/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/4/893" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Data application</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/4/893.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/4/893.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/4/893.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/4/893.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
