<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/4/993" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/4/993.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/4/993.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/4/993.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/4/993.atom" />
<meta name="HW.pisa" content="ggg;6/4/993" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="High Throughput Analyses of Budding Yeast ARSs Reveal New DNA Elements Capable of Conferring Centromere-Independent Plasmid Propagation" />
<meta name="DC.Identifier" content="10.1534/g3.116.027904" />
<meta name="DC.Date" content="2016-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Hoggard et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The ability of plasmids to propagate in Saccharomyces cerevisiae has been instrumental in defining eukaryotic chromosomal control elements. Stable propagation demands both plasmid replication, which requires a chromosomal replication origin ( i.e. , an ARS), and plasmid distribution to dividing cells, which requires either a chromosomal centromere for segregation or a plasmid-partitioning element. While our knowledge of yeast ARSs and centromeres is relatively advanced, we know less about chromosomal regions that can function as plasmid partitioning elements. The [Rap1][1] protein-binding site ([RAP1][1]) present in transcriptional silencers and telomeres of budding yeast is a known plasmid-partitioning element that functions to anchor a plasmid to the inner nuclear membrane (INM), which in turn facilitates plasmid distribution to daughter cells. This [Rap1][1]-dependent INM-anchoring also has an important chromosomal role in higher-order chromosomal structures that enhance transcriptional silencing and telomere stability. Thus, plasmid partitioning can reflect fundamental features of chromosome structure and biology, yet a systematic screen for plasmid partitioning elements has not been reported. Here, we couple deep sequencing with competitive growth experiments of a plasmid library containing thousands of short ARS fragments to identify new plasmid partitioning elements. Competitive growth experiments were performed with libraries that differed only in terms of the presence or absence of a centromere. Comparisons of the behavior of ARS fragments in the two experiments allowed us to identify sequences that were likely to drive plasmid partitioning. In addition to the silencer [RAP1][1] site, we identified 74 new putative plasmid-partitioning motifs predicted to act as binding sites for DNA binding proteins enriched for roles in negative regulation of gene expression and G2/M-phase associated biology. These data expand our knowledge of chromosomal elements that may function in plasmid partitioning and suggest underlying biological roles shared by such elements.

 [1]: http://www.yeastgenome.org/locus/S000005160/overview" />
<meta name="DC.Contributor" content="Timothy Hoggard" />
<meta name="DC.Contributor" content="Ivan Liachko" />
<meta name="DC.Contributor" content="Cassaundra Burt" />
<meta name="DC.Contributor" content="Troy Meikle" />
<meta name="DC.Contributor" content="Katherine Jiang" />
<meta name="DC.Contributor" content="Gheorghe Craciun" />
<meta name="DC.Contributor" content="Maitreya J. Dunham" />
<meta name="DC.Contributor" content="Catherine A. Fox" />
<meta name="article:published_time" content="2016-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="High Throughput Analyses of Budding Yeast ARSs Reveal New DNA Elements Capable of Conferring Centromere-Independent Plasmid Propagation" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The ability of plasmids to propagate in &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; has been instrumental in defining eukaryotic chromosomal control elements. Stable propagation demands both plasmid replication, which requires a chromosomal replication origin (&lt;i&gt;i.e.&lt;/i&gt;, an ARS), and plasmid distribution to dividing cells, which requires either a chromosomal centromere for segregation or a plasmid-partitioning element. While our knowledge of yeast ARSs and centromeres is relatively advanced, we know less about chromosomal regions that can function as plasmid partitioning elements. The Rap1 protein-binding site (RAP1) present in transcriptional silencers and telomeres of budding yeast is a known plasmid-partitioning element that functions to anchor a plasmid to the inner nuclear membrane (INM), which in turn facilitates plasmid distribution to daughter cells. This Rap1-dependent INM-anchoring also has an important chromosomal role in higher-order chromosomal structures that enhance transcriptional silencing and telomere stability. Thus, plasmid partitioning can reflect fundamental features of chromosome structure and biology, yet a systematic screen for plasmid partitioning elements has not been reported. Here, we couple deep sequencing with competitive growth experiments of a plasmid library containing thousands of short ARS fragments to identify new plasmid partitioning elements. Competitive growth experiments were performed with libraries that differed only in terms of the presence or absence of a centromere. Comparisons of the behavior of ARS fragments in the two experiments allowed us to identify sequences that were likely to drive plasmid partitioning. In addition to the silencer RAP1 site, we identified 74 new putative plasmid-partitioning motifs predicted to act as binding sites for DNA binding proteins enriched for roles in negative regulation of gene expression and G2/M-phase associated biology. These data expand our knowledge of chromosomal elements that may function in plasmid partitioning and suggest underlying biological roles shared by such elements.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/04/01" />
<meta name="citation_mjid" content="ggg;6/4/993" />
<meta name="citation_id" content="6/4/993" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/4/993" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/4/993.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/4/993.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/4/993.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.027904" />
<meta name="citation_pmid" content="26865697" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="20" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="993" />
<meta name="citation_lastpage" content="1012" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Timothy Hoggard" />
<meta name="citation_author_institution" content="Department of Biomolecular Chemistry, School of Medicine and Public Health, University of Wisconsin, Madison, Wisconsin 53706" />
<meta name="citation_author" content="Ivan Liachko" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98105" />
<meta name="citation_author" content="Cassaundra Burt" />
<meta name="citation_author_institution" content="Department of Biomolecular Chemistry, School of Medicine and Public Health, University of Wisconsin, Madison, Wisconsin 53706" />
<meta name="citation_author" content="Troy Meikle" />
<meta name="citation_author_institution" content="Department of Biomolecular Chemistry, School of Medicine and Public Health, University of Wisconsin, Madison, Wisconsin 53706" />
<meta name="citation_author" content="Katherine Jiang" />
<meta name="citation_author_institution" content="Department of Biomolecular Chemistry, School of Medicine and Public Health, University of Wisconsin, Madison, Wisconsin 53706" />
<meta name="citation_author" content="Gheorghe Craciun" />
<meta name="citation_author_institution" content="Department of Biomolecular Chemistry, School of Medicine and Public Health, University of Wisconsin, Madison, Wisconsin 53706" />
<meta name="citation_author_institution" content="Department of Mathematics, College of Letters and Science, University of Wisconsin, Madison Wisconsin 53706" />
<meta name="citation_author" content="Maitreya J. Dunham" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98105" />
<meta name="citation_author" content="Catherine A. Fox" />
<meta name="citation_author_institution" content="Department of Biomolecular Chemistry, School of Medicine and Public Health, University of Wisconsin, Madison, Wisconsin 53706" />
<meta name="citation_author_email" content="cfox@wisc.edu" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. Akiyoshi;citation_author=K. K. Sarangapani;citation_author=A. F. Powers;citation_author=C. R. Nelson;citation_author=S. L. Reichow;citation_title=Tension directly stabilizes reconstituted kinetochore-microtubule attachments.;citation_pages=576-579;citation_volume=468;citation_year=2010;citation_issue=7323;citation_pmid=21107429;citation_doi=10.1038/nature09594" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. D. Andrulis;citation_author=A. M. Neiman;citation_author=D. C. Zappulla;citation_author=R. Sternglanz;citation_title=Perinuclear localization of chromatin facilitates transcriptional silencing.;citation_pages=592-595;citation_volume=394;citation_year=1998;citation_issue=6693;citation_pmid=9707122;citation_doi=10.1038/29100" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=E. D. Andrulis;citation_author=D. C. Zappulla;citation_author=A. Ansari;citation_author=S. Perrod;citation_author=C. V. Laiosa;citation_title=Esc1, a Nuclear Periphery Protein Required for Sir4-Based Plasmid Anchoring and Partitioning;citation_pages=8292-8301;citation_volume=22;citation_year=2002;citation_issue=23;citation_pmid=12417731;citation_doi=10.1128/MCB.22.23.8292-8301.2002" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=A. Ansari;citation_author=M. R. Gartenberg;citation_title=The yeast silent information regulator Sir4p anchors and partitions plasmids;citation_pages=7061-7068;citation_volume=17;citation_year=1997;citation_issue=12;citation_pmid=9372937;citation_doi=10.1128/MCB.17.12.7061" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. P. Bell;citation_author=B. Stillman;citation_title=ATP-dependent recognition of eukaryotic origins of DNA replication by a multiprotein complex.;citation_pages=128-134;citation_volume=357;citation_year=1992;citation_issue=6374;citation_pmid=1579162;citation_doi=10.1038/357128a0" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Genet.;citation_author=K. S. Bloom;citation_title=Centromeric Heterochromatin: The Primordial Segregation Machine.;citation_pages=457-484;citation_volume=48;citation_year=2014;citation_pmid=25251850;citation_doi=10.1146/annurev-genet-120213-092033" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. Botchan;citation_title=Hitchhiking without covalent integration.;citation_pages=280-281;citation_volume=117;citation_year=2004;citation_issue=3;citation_pmid=15109488;citation_doi=10.1016/S0092-8674(04)00410-6" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=L. L. Breeden;citation_title=Cyclin transcription: Timing is everything.;citation_pages=R586-R588;citation_volume=10;citation_year=2000;citation_issue=16;citation_pmid=10985373;citation_doi=10.1016/S0960-9822(00)00634-5" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=F. Chang;citation_author=J. F. Theis;citation_author=J. Miller;citation_author=C. A. Nieduszynski;citation_author=C. S. Newlon;citation_title=Analysis of Chromosome III Replicators Reveals an Unusual Structure for the ARS318 Silencer Origin and a Conserved WTW Sequence within the Origin Recognition Complex Binding Site.;citation_volume=28;citation_year=2008;citation_pmid=18573888;citation_doi=10.1128/MCB.00206-08" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=F. Chang;citation_author=C. D. May;citation_author=T. Hoggard;citation_author=J. Miller;citation_author=C. A. Fox;citation_title=High-resolution analysis of four efficient yeast replication origins reveals new insights into the ORC and putative MCM binding elements.;citation_volume=39;citation_year=2011;citation_pmid=21558171;citation_doi=10.1093/nar/gkr301" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=J. G. Cook;citation_author=L. Bardwell;citation_author=S. J. Kron;citation_author=J. Thorner;citation_title=Two novel targets of the MAP kinase Kss1 are negative regulators of invasive growth in the yeast Saccharomyces cerevisiae.;citation_pages=2831-2848;citation_volume=10;citation_year=1996;citation_issue=22;citation_pmid=8918885;citation_doi=10.1101/gad.10.22.2831" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=A. Crampton;citation_author=F. Chang;citation_author=D. L. J. Pappas;citation_author=R. L. Frisch;citation_author=M. Weinreich;citation_title=An ARS element inhibits DNA replication through a SIR2-dependent mechanism.;citation_pages=156-166;citation_volume=30;citation_year=2008;citation_issue=2;citation_pmid=18439895;citation_doi=10.1016/j.molcel.2008.02.019" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=A. Denoth-Lippuner;citation_author=M. K. Krzyzanowski;citation_author=C. Stober;citation_author=Y. Barral;citation_title=Role of SAGA in the asymmetric segregation of DNA circles during yeast ageing.;citation_volume=3;citation_year=2014;citation_doi=10.7554/eLife.03790" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=E. Dodou;citation_author=R. Treisman;citation_title=The Saccharomyces cerevisiae MADS-box transcription factor Rlm1 is a target for the Mpk1 mitogen-activated protein kinase pathway;citation_pages=1848-1859;citation_volume=17;citation_year=1997;citation_issue=4;citation_pmid=9121433;citation_doi=10.1128/MCB.17.4.1848" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. J. Donato;citation_author=S. C. Chung;citation_author=B. K. Tye;citation_title=Genome-wide hierarchy of replication origin usage in Saccharomyces cerevisiae.;citation_pages=e141-e141;citation_volume=2;citation_year=2006;citation_issue=9;citation_pmid=16965179;citation_doi=10.1371/journal.pgen.0020141" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=M. L. Eaton;citation_author=K. Galani;citation_author=S. Kang;citation_author=S. P. Bell;citation_author=D. M. MacAlpine;citation_title=Conserved nucleosome positioning defines replication origins.;citation_volume=24;citation_year=2010;citation_pmid=20351051;citation_doi=10.1101/gad.1913210" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Enomoto;citation_author=M. S. Longtine;citation_author=J. Berman;citation_title=Enhancement of telomere-plasmid segregation by the X-telomere associated sequence in Saccharomyces cerevisiae involves SIR2, SIR3, SIR4 and ABF1.;citation_pages=757-767;citation_volume=136;citation_year=1994;citation_issue=3;citation_pmid=8005431" />
<meta name="citation_reference" content="citation_journal_title=Nature Cell Biology;citation_journal_abbrev=Nature Cell Biology;citation_author=W. Feng;citation_author=D. Collingwood;citation_author=M. E. Boeck;citation_author=L. A. Fox;citation_author=G. M. Alvino;citation_title=Genomic mapping of single-stranded DNA in hydroxyurea-challenged yeasts identifies origins of replication.;citation_pages=148-155;citation_volume=8;citation_year=2006;citation_issue=2;citation_pmid=16429127;citation_doi=10.1038/ncb1358" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A.-C. Gavin;citation_author=M. Bösche;citation_author=R. Krause;citation_author=P. Grandi;citation_author=M. Marzioch;citation_title=Functional organization of the yeast proteome by systematic analysis of protein complexes.;citation_pages=141-147;citation_volume=415;citation_year=2002;citation_issue=6868;citation_pmid=11805826;citation_doi=10.1038/415141a" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=L. R. Gehlen;citation_author=S. Nagai;citation_author=K. Shimada;citation_author=P. Meister;citation_author=A. Taddei;citation_title=Nuclear geometry and rapid mitosis ensure asymmetric episome segregation in yeast.;citation_pages=25-33;citation_volume=21;citation_year=2011;citation_issue=1;citation_pmid=21194950;citation_doi=10.1016/j.cub.2010.12.016" />
<meta name="citation_reference" content="citation_journal_title=The Journal of Cell Biology;citation_journal_abbrev=JCB;citation_author=M. Gotta;citation_author=T. Laroche;citation_author=A. Formenton;citation_author=L. Maillet;citation_author=H. Scherthan;citation_title=The clustering of telomeres and colocalization with Rap1, Sir3, and Sir4 proteins in wild-type Saccharomyces cerevisiae;citation_pages=1349-1363;citation_volume=134;citation_year=1996;citation_issue=6;citation_pmid=8830766;citation_doi=10.1083/jcb.134.6.1349" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=B. M. Green;citation_author=J. J. Li;citation_title=Loss of rereplication control in Saccharomyces cerevisiae results in extensive DNA damage.;citation_volume=16;citation_year=2005;citation_pmid=15537702;citation_doi=10.1091/mbc.E04-09-0833" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=T. Hoggard;citation_author=E. Shor;citation_author=C. A. Muller;citation_author=C. A. Nieduszynski;citation_author=C. A. Fox;citation_title=A Link between ORC-origin binding mechanisms and origin activation time revealed in budding yeast.;citation_pages=e1003798;citation_volume=9;citation_year=2013;citation_pmid=24068963;citation_doi=10.1371/journal.pgen.1003798" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=R. Y. Huang;citation_author=D. Kowalski;citation_title=A DNA unwinding element and an ARS consensus comprise a replication origin within a yeast chromosome.;citation_pages=4521-4531;citation_volume=12;citation_year=1993;citation_pmid=8223462" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. Y. Huang;citation_author=D. Kowalski;citation_title=Multiple DNA elements in ARS305 determine replication origin activity in a yeast chromosome.;citation_volume=24;citation_year=1996;citation_pmid=8600446;citation_doi=10.1093/nar/24.5.816" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. Ito;citation_author=T. Chiba;citation_author=R. Ozawa;citation_author=M. Yoshida;citation_author=M. Hattori;citation_title=A comprehensive two-hybrid analysis to explore the yeast protein interactome.;citation_volume=98;citation_year=2001;citation_pmid=11283351;citation_doi=10.1073/pnas.061034498" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=W. Kimmerly;citation_author=A. Buchman;citation_author=R. Kornberg;citation_author=J. Rine;citation_title=Roles of two DNA-binding factors in replication, segregation and transcriptional repression mediated by a yeast silencer.;citation_pages=2241-2253;citation_volume=7;citation_year=1988;citation_pmid=3046937" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=S. R. Knott;citation_author=C. J. Viggiani;citation_author=S. Tavare;citation_author=O. M. Aparicio;citation_title=Genome-wide replication profiles indicate an expansive role for Rpd3L in regulating replication initiation timing or efficiency, and reveal genomic loci of Rpd3 function in Saccharomyces cerevisiae;citation_pages=1077-1090;citation_volume=23;citation_year=2009;citation_issue=9;citation_pmid=19417103;citation_doi=10.1101/gad.1784309" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. R. Knott;citation_author=J. M. Peace;citation_author=A. Z. Ostrow;citation_author=Y. Gan;citation_author=A. E. Rex;citation_title=Forkhead transcription factors establish origin timing and long-range clustering in S. cerevisiae.;citation_pages=99-111;citation_volume=148;citation_year=2012;citation_issue=1-2;citation_pmid=22265405;citation_doi=10.1016/j.cell.2011.12.012" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=N. J. Krogan;citation_author=G. Cagney;citation_author=H. Yu;citation_author=G. Zhong;citation_author=X. Guo;citation_title=Global landscape of protein complexes in the yeast Saccharomyces cerevisiae.;citation_pages=637-643;citation_volume=440;citation_year=2006;citation_issue=7084;citation_pmid=16554755;citation_doi=10.1038/nature04670" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=I. Liachko;citation_author=R. A. Youngblood;citation_author=U. Keich;citation_author=M. J. Dunham;citation_title=High-resolution mapping, characterization, and optimization of autonomously replicating sequences in yeast.;citation_volume=23;citation_year=2013;citation_pmid=23241746;citation_doi=10.1101/gr.144659.112" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=J. R. Lipford;citation_author=S. P. Bell;citation_title=Nucleosomes positioned by ORC facilitate the initiation of DNA replication.;citation_pages=21-30;citation_volume=7;citation_year=2001;citation_issue=1;citation_pmid=11172708;citation_doi=10.1016/S1097-2765(01)00151-4" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=M. S. Longtine;citation_author=S. Enomoto;citation_author=S. L. Finstad;citation_author=J. Berman;citation_title=Yeast telomere repeat sequence (TRS) improves circular plasmid segregation, and TRS plasmid segregation involves the RAP1 gene product.;citation_pages=1997-2009;citation_volume=12;citation_year=1992;citation_issue=5;citation_pmid=1569937;citation_doi=10.1128/MCB.12.5.1997" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. S. Longtine;citation_author=S. Enomoto;citation_author=S. L. Finstad;citation_author=J. Berman;citation_title=Telomere-mediated plasmid segregation in Saccharomyces cerevisiae involves gene products required for transcriptional repression at silencers and telomeres.;citation_pages=171-182;citation_volume=133;citation_year=1993;citation_issue=2;citation_pmid=8436267" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=Y. Marahrens;citation_author=B. Stillman;citation_title=Replicator dominance in a eukaryotic chromosome.;citation_pages=3395-3400;citation_volume=13;citation_year=1994;citation_pmid=8045266" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=F. J. McNally;citation_author=J. Rine;citation_title=A synthetic silencer mediates SIR-dependent functions in Saccharomyces cerevisiae.;citation_pages=5648-5659;citation_volume=11;citation_year=1991;citation_issue=11;citation_pmid=1922068;citation_doi=10.1128/MCB.11.11.5648" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=P. Moretti;citation_author=D. Shore;citation_title=Multiple Interactions in Sir Protein Recruitment by Rap1p at Silencers and Telomeres in Yeast;citation_pages=8082-8094;citation_volume=21;citation_year=2001;citation_issue=23;citation_pmid=11689698;citation_doi=10.1128/MCB.21.23.8082-8094.2001" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=C. A. Muller;citation_author=C. A. Nieduszynski;citation_title=Conservation of replication timing reveals global and local regulation of replication origin activity.;citation_volume=22;citation_year=2012;citation_pmid=22767388;citation_doi=10.1101/gr.139477.112" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=P. Muller;citation_author=S. Park;citation_author=E. Shor;citation_author=D. J. Huebert;citation_author=C. L. Warren;citation_title=The conserved bromo-adjacent homology domain of yeast Orc1 functions in the selection of DNA replication origins within chromatin;citation_pages=1418-1433;citation_volume=24;citation_year=2010;citation_issue=13;citation_pmid=20595233;citation_doi=10.1101/gad.1906410" />
<meta name="citation_reference" content="citation_journal_abbrev=Epigenomics;citation_author=S. Nagai;citation_author=P. Heun;citation_author=S. M. Gasser;citation_title=Roles for nuclear organization in the maintenance of genome stability.;citation_pages=289-305;citation_volume=2;citation_year=2010;citation_issue=2;citation_pmid=22121875;citation_doi=10.2217/epi.09.49" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=T. Natsume;citation_author=C. A. Muller;citation_author=Y. Katou;citation_author=R. Retkute;citation_author=M. Gierlinski;citation_title=Kinetochores coordinate pericentromeric cohesion and early DNA replication by Cdc7-Dbf4 kinase recruitment.;citation_pages=661-674;citation_volume=50;citation_year=2013;citation_pmid=23746350;citation_doi=10.1016/j.molcel.2013.05.011" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=C. S. Newlon;citation_author=J. F. Theis;citation_title=The structure and function of yeast ARS elements.;citation_pages=752-758;citation_volume=3;citation_year=1993;citation_issue=5;citation_pmid=8274858;citation_doi=10.1016/S0959-437X(05)80094-2" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=C. A. Nieduszynski;citation_author=J. J. Blow;citation_author=A. D. Donaldson;citation_title=The requirement of yeast replication origins for pre-replication complex proteins is modulated by transcription;citation_pages=2410-2420;citation_volume=33;citation_year=2005;citation_issue=8;citation_pmid=15860777;citation_doi=10.1093/nar/gki539" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=T. J. Pohl;citation_author=B. J. Brewer;citation_author=M. K. Raghuraman;citation_title=Functional centromeres determine the activation time of pericentric origins of DNA replication in Saccharomyces cerevisiae.;citation_pages=e1002677-e1002677;citation_volume=8;citation_year=2012;citation_issue=5;citation_pmid=22589733;citation_doi=10.1371/journal.pgen.1002677" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=T. J. Pohl;citation_author=K. Kolor;citation_author=W. L. Fangman;citation_author=B. J. Brewer;citation_author=M. K. Raghuraman;citation_title=A DNA sequence element that advances replication origin activation time in Saccharomyces cerevisiae.;citation_volume=3;citation_year=2013;citation_pmid=24022751;citation_doi=10.1534/g3.113.008250" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Ptacek;citation_author=G. Devgan;citation_author=G. Michaud;citation_author=H. Zhu;citation_author=X. Zhu;citation_title=Global analysis of protein phosphorylation in yeast.;citation_pages=679-684;citation_volume=438;citation_year=2005;citation_issue=7068;citation_pmid=16319894;citation_doi=10.1038/nature04187" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=H. Rao;citation_author=B. Stillman;citation_title=The origin recognition complex interacts with a bipartite DNA binding site within yeast replicators.;citation_pages=2224-2228;citation_volume=92;citation_year=1995;citation_issue=6;citation_pmid=7892251;citation_doi=10.1073/pnas.92.6.2224" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=H. Rao;citation_author=Y. Marahrens;citation_author=B. Stillman;citation_title=Functional conservation of multiple elements in yeast chromosomal replicators.;citation_pages=7643-7651;citation_volume=14;citation_year=1994;citation_issue=11;citation_pmid=7935478;citation_doi=10.1128/MCB.14.11.7643" />
<meta name="citation_reference" content="citation_journal_title=Annual review of biochemistry;citation_journal_abbrev=Annu Rev Biochem;citation_author=L. N. Rusche;citation_author=A. L. Kirchmaier;citation_author=J. Rine;citation_title=The establishment, inheritance, and function of silenced chromatin in Saccharomyces cerevisiae.;citation_pages=481-516;citation_volume=72;citation_year=2003;citation_pmid=12676793;citation_doi=10.1146/annurev.biochem.72.121801.161547" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. T. Simpson;citation_title=Nucleosome positioning can affect the function of a cis-acting DNA element in vivo.;citation_pages=387-389;citation_volume=343;citation_year=1990;citation_issue=6256;citation_pmid=2405281;citation_doi=10.1038/343387a0" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=C. C. Siow;citation_author=S. R. Nieduszynska;citation_author=C. A. Muller;citation_author=C. A. Nieduszynski;citation_title=OriDB, the DNA replication origin database updated and extended.;citation_volume=40;citation_year=2012;citation_pmid=22121216;citation_doi=10.1093/nar/gkr1091" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Taddei;citation_author=S. M. Gasser;citation_title=Structure and Function in the Budding Yeast Nucleus;citation_pages=107-129;citation_volume=192;citation_year=2012;citation_issue=1;citation_pmid=22964839;citation_doi=10.1534/genetics.112.140608" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=J. F. Theis;citation_author=C. S. Newlon;citation_title=Domain B of ARS307 contains two functional elements and contributes to chromosomal replication origin function.;citation_pages=7652-7659;citation_volume=14;citation_year=1994;citation_issue=11;citation_pmid=7935479;citation_doi=10.1128/MCB.14.11.7652" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. F. Theis;citation_author=C. S. Newlon;citation_title=The ARS309 chromosomal replicator of Saccharomyces cerevisiae depends on an exceptional ARS consensus sequence;citation_pages=10786-10791;citation_volume=94;citation_year=1997;citation_issue=20;citation_pmid=9380711;citation_doi=10.1073/pnas.94.20.10786" />
<meta name="citation_reference" content="citation_journal_title=Trends Biochem. Sci.;citation_author=B. D. Towbin;citation_author=A. Gonzalez-Sandoval;citation_author=S. M. Gasser;citation_title=Mechanisms of heterochromatin subnuclear localization.;citation_pages=356-363;citation_volume=38;citation_year=2013;citation_pmid=23746617;citation_doi=10.1016/j.tibs.2013.04.004" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=W. P. Voth;citation_author=Y. Yu;citation_author=S. Takahata;citation_author=K. L. Kretschmann;citation_author=J. D. Lieb;citation_title=Forkhead proteins control the outcome of transcription factor binding by antiactivation;citation_pages=4324-4334;citation_volume=26;citation_year=2007;citation_issue=20;citation_pmid=17898805;citation_doi=10.1038/sj.emboj.7601859" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=Y. Watanabe;citation_author=K. Irie;citation_author=K. Matsumoto;citation_title=Yeast RLM1 encodes a serum response factor-like protein that may function downstream of the Mpk1 (Slt2) mitogen-activated protein kinase pathway;citation_pages=5740-5749;citation_volume=15;citation_year=1995;citation_issue=10;citation_pmid=7565726;citation_doi=10.1128/MCB.15.10.5740" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=Y. Watanabe;citation_author=G. Takaesu;citation_author=M. Hagiwara;citation_author=K. Irie;citation_author=K. Matsumoto;citation_title=Characterization of a serum response factor-like protein in Saccharomyces cerevisiae, Rlm1, which has transcriptional activity regulated by the Mpk1 (Slt2) mitogen-activated protein kinase pathway;citation_pages=2615-2623;citation_volume=17;citation_year=1997;citation_issue=5;citation_pmid=9111331;citation_doi=10.1128/MCB.17.5.2615" />
<meta name="citation_reference" content="citation_journal_title=Yeast;citation_author=A. Z. Welch;citation_author=D. E. Koshland;citation_title=A simple colony-formation assay in liquid medium, termed “tadpoling,” provides a sensitive measure of Saccharomyces cerevisiae culture viability.;citation_pages=501-509;citation_volume=30;citation_year=2013;citation_pmid=24185677;citation_doi=10.1002/yea.2989" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=G. M. Wilmes;citation_author=S. P. Bell;citation_title=The B2 element of the Saccharomyces cerevisiae ARS1 origin of replication requires specific sequences to facilitate pre-RC formation.;citation_volume=99;citation_year=2002;citation_pmid=11756674;citation_doi=10.1073/pnas.012578499" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=K. Yoshida;citation_author=J. Bacal;citation_author=D. Desmarais;citation_author=I. Padioleau;citation_author=O. Tsaponina;citation_title=The Histone Deacetylases Sir2 and Rpd3 Act on Ribosomal DNA to Control the Replication Program in Budding Yeast.;citation_pages=691-697;citation_volume=54;citation_year=2014;citation_pmid=24856221;citation_doi=10.1016/j.molcel.2014.04.032" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. You;citation_author=J. L. Croyle;citation_author=A. Nishimura;citation_author=K. Ozato;citation_author=P. M. Howley;citation_title=Interaction of the bovine papillomavirus E2 protein with Brd4 tethers the viral DNA to host mitotic chromosomes.;citation_pages=349-360;citation_volume=117;citation_year=2004;citation_issue=3;citation_pmid=15109495;citation_doi=10.1016/S0092-8674(04)00402-7" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=R. Zhao;citation_author=M. S. Bodnar;citation_author=D. L. Spector;citation_title=Nuclear neighborhoods and gene expression.;citation_pages=172-179;citation_volume=19;citation_year=2009;citation_issue=2;citation_pmid=19339170;citation_doi=10.1016/j.gde.2009.02.007" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="High Throughput Analyses of Budding Yeast ARSs Reveal New DNA Elements Capable of Conferring Centromere-Independent Plasmid Propagation" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
<meta name="twitter:description" content="The ability of plasmids to propagate in Saccharomyces cerevisiae has been instrumental in defining eukaryotic chromosomal control elements. Stable propagation demands both plasmid replication, which requires a chromosomal replication origin ( i.e. , an ARS), and plasmid distribution to dividing cells, which requires either a chromosomal centromere for segregation or a plasmid-partitioning element. While our knowledge of yeast ARSs and centromeres is relatively advanced, we know less about chromosomal regions that can function as plasmid partitioning elements. The [Rap1][1] protein-binding site ([RAP1][1]) present in transcriptional silencers and telomeres of budding yeast is a known plasmid-partitioning element that functions to anchor a plasmid to the inner nuclear membrane (INM), which in turn facilitates plasmid distribution to daughter cells. This [Rap1][1]-dependent INM-anchoring also has an important chromosomal role in higher-order chromosomal structures that enhance transcriptional silencing and telomere stability. Thus, plasmid partitioning can reflect fundamental features of chromosome structure and biology, yet a systematic screen for plasmid partitioning elements has not been reported. Here, we couple deep sequencing with competitive growth experiments of a plasmid library containing thousands of short ARS fragments to identify new plasmid partitioning elements. Competitive growth experiments were performed with libraries that differed only in terms of the presence or absence of a centromere. Comparisons of the behavior of ARS fragments in the two experiments allowed us to identify sequences that were likely to drive plasmid partitioning. In addition to the silencer [RAP1][1] site, we identified 74 new putative plasmid-partitioning motifs predicted to act as binding sites for DNA binding proteins enriched for roles in negative regulation of gene expression and G2/M-phase associated biology. These data expand our knowledge of chromosomal elements that may function in plasmid partitioning and suggest underlying biological roles shared by such elements.

 [1]: http://www.yeastgenome.org/locus/S000005160/overview" />
<meta name="og-title" property="og:title" content="High Throughput Analyses of Budding Yeast ARSs Reveal New DNA Elements Capable of Conferring Centromere-Independent Plasmid Propagation" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/4/993" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The ability of plasmids to propagate in Saccharomyces cerevisiae has been instrumental in defining eukaryotic chromosomal control elements. Stable propagation demands both plasmid replication, which requires a chromosomal replication origin ( i.e. , an ARS), and plasmid distribution to dividing cells, which requires either a chromosomal centromere for segregation or a plasmid-partitioning element. While our knowledge of yeast ARSs and centromeres is relatively advanced, we know less about chromosomal regions that can function as plasmid partitioning elements. The [Rap1][1] protein-binding site ([RAP1][1]) present in transcriptional silencers and telomeres of budding yeast is a known plasmid-partitioning element that functions to anchor a plasmid to the inner nuclear membrane (INM), which in turn facilitates plasmid distribution to daughter cells. This [Rap1][1]-dependent INM-anchoring also has an important chromosomal role in higher-order chromosomal structures that enhance transcriptional silencing and telomere stability. Thus, plasmid partitioning can reflect fundamental features of chromosome structure and biology, yet a systematic screen for plasmid partitioning elements has not been reported. Here, we couple deep sequencing with competitive growth experiments of a plasmid library containing thousands of short ARS fragments to identify new plasmid partitioning elements. Competitive growth experiments were performed with libraries that differed only in terms of the presence or absence of a centromere. Comparisons of the behavior of ARS fragments in the two experiments allowed us to identify sequences that were likely to drive plasmid partitioning. In addition to the silencer [RAP1][1] site, we identified 74 new putative plasmid-partitioning motifs predicted to act as binding sites for DNA binding proteins enriched for roles in negative regulation of gene expression and G2/M-phase associated biology. These data expand our knowledge of chromosomal elements that may function in plasmid partitioning and suggest underlying biological roles shared by such elements.

 [1]: http://www.yeastgenome.org/locus/S000005160/overview" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
    <title>High Throughput Analyses of Budding Yeast ARSs Reveal New DNA Elements Capable of Conferring Centromere-Independent Plasmid Propagation | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"S5kq8koPNFLs7qHm8uUIHsQZeVcesZTbuzHVDh2pQfI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460126","apath":"\/ggg\/6\/4\/993.atom","pisa":"ggg;6\/4\/993","ac":{"\/ggg\/6\/4\/993.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/4\/993.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/4\/993"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/4\/993"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/4\/993":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Er5Mqh4RlM98YtdAsxvnRu+Hl2X8SWRTPG\\\\\\\/pwrvAoqsNi3P4NuK2wfoNeyg6zgfWNShoTmLgOtUo2CWD85rCiFZnOVriEJQwhxIL8ZQHolFfAwdfMgPAoh9d8uD+hFkJBhif38xh1Ab5bjrZd1VTAKe9fEY5TS1Pw2PFP6axCpq+jKiSc2esjp+6GT56lIvqBLijI9vRS1zi1mlkgOxfmmaUF1zTEN9dU+6RstIBIVvEhnDktEv\\\\\\\/nemgXVf5T3y0xOyQlHH3dGwwyo4CMlvaq+G74yvFPvke5OUxVeDFe64qQoP70ltKzcNn8khkv+nHD1ZcBTsXtICNNdkPHHAtUEAq1RO3WUxwIOLRuNFwb+umI36MTZE2X3nRD4cTEKfPu9WQMGdz+hEFf7nWP3FbSxMlk+Z\\\\\\\/PJlL0kmb99d8FUFEuPb1KreUcWvEAVNAYKPcUIxbn8XFO4BGhn24A+5eCP2Td1521uQlQWogJHYa6BMRivNbTRkPK+1DyQRu6CXAGJ1zjPZNCXD\\\\\\\/hjJzu9FsH8GrARw5IeEvJt+su1gBMxFP9jc3XGkV1pad4V53bO+ZM9Wu5LecjTdIKc+6MJiboEDo5EXm+K5SIs\\\\\\\/cYQr7xM6MTXkhSkteV0kqINyUbGOBjo3u7quRnZHirsaG1f7tuJaGLO+JgC6foVAczEJiieN83PvOLr2X1xdBhMk1Ywu5HoRTIa6v2jfuFVCVzQ+0hQKMJrnf9mz5Xz8Ub5fU0GHtJNXyly+X6zBnaW70p3\\\\\\\/RbqhkslPPUS4XG5mi+fGQN+w6N63ylE30cauoxR+F71jPm3\\\\\\\/S5gSCg2BfjpaNLcMrTlHz23hq7gki844lLWtTxYqN67UyjN3UXjWi22Orq3VF7dq+IBpKlbx8UdWN\\\\\\\/TndW+mdW0ujA8nYg1jFHdZs8AmfRRW0b4u+\\\\\\\/NPzvAipTTPEDv3sQWQdgvLu9BnboZ1dzNTb2V0zqJn\\\\\\\/9LaqkSP3KtFlQmm2hZoA6M9AEQ7I0jnG7vCVudHrHy1SfVuoQd+RM5GNIXu6ZwrTv6is+0liQ7A5YoZwfkhplcZsoUiyMGrfe3ycU0toqt0wmpdlMw0DbTKrjRsVGWRKJLeVjJdAfvA2zemzZs6wUUv3Mmm8jq9V4Zy\\\\\\\/SZF2CaLo\\\\\\\/3Pd8vmk95dRf7uXjSdBZzJGqWoB12egtoRQ82celcMhmYPflQtM85r1hhUkJApXxCoS67jvLycM74VCioy3jjpxDlFdg6GGrWu2Zh3swAfWvhhEVOZ3WTHKv3ZJya1dKP1ZquJy\\\u0022,\\\u0022iv\\\u0022:\\\u0022ekkdcmTa04UttfI5kCY\\\\\\\/0w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e6faadf4084e12f543c838fc4ddb3a006bf3a84c9705e56994ac2ed6f53dfa86\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022m280Buz8qn183XcK6bckwbiKM8DIAnqYI37ZCdpXf1xGOdoMwCz\\\\\\\/Cr5NVfMaf2PFvaUMb6V2CmESTkmmb\\\\\\\/kgv0UPpAI6D5bvyavSCNk63xM2Rs4ETGVd3DYHBjlBH0R7\\\\\\\/IPbbHGY57mjO6ta+tZB9IiRIwxxBc4gw48R6ol1w06+gszyFknS3v+Ae4Jv8Pg1tlVlzRqeVRC05rUA6l4V0NVM8GrK9ArN04Y6vCzMDzIj3wigp3eU1SflV8WunGl5mzrFBkwLAZUq7w+kuT5vpjPOoUBkubC2TCmaD0kUjMvPcOtHc7NO1lygAdQXEAIvqerSJVQEWOoddZ1o4u2Tm3sEgpTAueyIqJ7uToeDBNyHRP2yjuF4YJ+qlgLrJk+U1I02KeZMyuioMNwI2tuNAQ6V7sS7AO3oa4QZrk30roJpyFCbjIp05eCWwt+zzhm3RLc1j\\\\\\\/d0UF3SOAENbx9KjxSOYLBF7\\\\\\\/EV4z4Xv+d7YNz7JDlqRMpionWdz6NkbH+vztBr9ddG9jKSxcIggPM21OhZHWGZz5g16505BeqtlhiE61HH88R8leByKOOI\\\\\\\/LD8m2w8KZhchjAF3p\\\\\\\/DOB5TxXEg0ep96oe3OR+IbVs1Zmsx5A1iSJG5FwuaWj8mqHxZI2c8mxug0ZFKIR6Vy00B5DlDNCsxTwtueGaC8iFML2HxcUwSWn\\\\\\\/ugGgNePp6g4V1ilDyHYV+fgB9bnw8MlZBOhtp3jn4wqpuMhRhE6cxYDg6AQeAzfVmaMJl0h0jRGXz1EjWmYXUcYnEtmmftVk2Q6DvAdXQZwkhmwBwf0TyRG3ufRIOIaHXh55ji6Cmo6KhiY3FkjY4PtmVuOSKxofi7W3+KGa9EXtyYXbCffkeW3dDrrhWXJ82nG9q+s4DzzIH5SqDHlgpDSxW9qUiqvlg32lVsParcdRIFdYJC5sE+NsVwuEK4GUVoqpt32SGMmkSp6Gib\\\\\\\/COf\\\\\\\/768R32L8jsrWPZQG6EaIuzkoONnBthTn+B8KuF3duNLjOVCDDm4RbIqIBrVMtTvyZXpWL8DCUAleT7NAsan51i63kCbOMQrubrbRaXJsdda1jyitpxIMLXbHgCDL7rR1e6+4dGv86qCPazC\\\\\\\/o1oHQB2tA8SEOSqhJe28Hg7jv7byga4VsVWKPdIZCPnsYXxwg7dqhJ2\\\\\\\/0uZOb+9O5kYcTvOxbJZx1joZnS4NRrPAYqXRA+OJgzCHXa5UWUZHky2BPauUlhRs4cJHvGzJzei75lurn0THi\\\\\\\/wi0IqlpY9ZKbIkftXNS3J40ckMBik033s6sYukl5sILppe9fhzG9IW5Y9qrO12gcXEeU3K8foliZ2TOiNiskLY8HrA+xbt9OB4+R1Viwxt6+axV+ZjQt8EWofuCpJvkIoR+VvV2Dp3IVtD34P4duE+iaRGqJaXwLkWi9fGKFL4dTbaBcVV5pa5KuuHQdIkpENaEqTH3ASfsqegrABI6gYnmcxnwlWunF5XsMcL85pCufui24zRsmgx9n2A0VjInqjA30sM8tQisEslA008KNaV8ZMWR3NTWb4QxjfV\\\\\\\/BqDNyNi8DZdmS1z9JFtbvLBW8jLrkYtK10J+aPmGNA08RWSSjuAtcf\\\\\\\/vG3NYxnzHk48NQdoIFu4BT0yBlanGfGMW3\\\\\\\/3z4WhxITcV9K7YvTVGRm8SkS74GqDh1JXkNS53IHETQLADOUrhtqF4+jFM84gg9c27pQo71khzx80QtZ0ur+ZGA2h0AenwmiT0wjtSbLHiUXDO40hIoW9UMPDhBYf1BtiCg6s5dlKegtQSUazayDGlfkBg6NyTuGhCcqT54wOBSV5KKqf03d7W8phUU6pFo58DaO\\\\\\\/yiKlqxBOFq+H0440MpbGJbDgBTXJKOVmr0xBXzelT2OKIBj2vp036OrMoX6TSRxCbYY9JLKOj0\\\\\\\/cupEDBMBB52P3ge+vRwUhACCw0DKhW2RNmjFUZkx9BMboTWy1hDMurw9ZOddymjaCNapZ418uePIxaPoy4BKZvHgBSh7Ye9e1bm3+F+FWWmJ7WbxDOm1gotPOkv\\\\\\\/q1+0cbVfRrCWYbsDxW6fmXVpsALwmz+7LUtWdmhknsEFps9KsdW36F8vDbAzz7i7kZDcuAS+D2c79a9nqIpvycCwJCklAvJ\\\\\\\/U\\\\\\\/MAhZOVKB8wQscd1k4NrQ+Ta5U46btbpqRgL4xN2Pmbnt0QF\\\\\\\/PhGT1bliAn2NwU1zA3nDRrggtbPUD5YpKUEEsSdxW76wq2O8ujrZOHM6CJrV7vGaj2bxA\\\\\\\/UMVnEN5El7eYcpgBsrIsm6bl7h6T+vDQ19VW5q3Za44EgX87BIWA+gXi+yvvWdG71CnXk611q4M9wBX0af4HZ1J8rxPEqe0o+AqWkjOs+xX9zm0g1i34VgDGSzNQJDsQ\\\\\\\/GRbyp19CCiKwgL4MDtM8yThGF3UD6v3FZBPXXUcU8mdnPB\\\\\\\/pvxuValIv1BG\\\\\\\/bozRg\\\\\\\/KXkTWvF604T9IpORJZkJtrb28c8itOaSXxjs5sQDUD\\\\\\\/3RfY4a5IFrDEwFbAuD8NoFHCQz6hI6GlWeLIMi0LTFFNYChlhVutB\\\\\\\/fd6js2qSxOZRJRL1cCzo\\\\\\\/dhw3M4gIvFBs3bwPwEIwF5XHbCgN1dbiJQCqR6M0PexiCBxdbVfFR7bnJE8FwPNXfj89rKoCAp9rb3QFtXmrR6duidRihk88uWdigwa3ZrQgoW5vK4ASfq8eWbCSlShxVl8G4AL3nClTucTrd\\\\\\\/HXuA9JCBPL0AY95v2vw66aUQeZt0Szt\\\\\\\/6wa+a3G1mgOgFRgkl7w4KNhspemd+Q8vF80Ul4xa0xEiIL4cKBglRDbYngLFCoGJtEtJgGY5O2jJ2dFBlqieZ7nNgRsBYNhcujXIpaqdULlRtMK4+2Hvu+JO9Bx7dBBWRS2psmUYq2lcCYA9C9s9nhSTsUMU+1fTKWNk5ZQCeHMZH+8VACF1oDrOU\\\\\\\/if6asHMw3+wFQepsADK6ZTZQZZsYDlUJA2rmktYXPuTsMSk6GNmxtZ3q3MtRDzdzWzRFoMSiSkIl7oXf8N4I2LVIj9TFBqlX8Oy4w1cmaGI\\\\\\\/a8s0wR3K3aYoDsnWTFjhQo2m6Atu5s9svBZk7lvxKPgWWCbBUB6fYJC89Z1LG2fHawQ5RaEQSQZsJVjpGTU2JbfltcEXpPWrTovOF1ZHvFSblokQW73e\\\\\\\/82bdXO0gn1fpGJPtN1te\\\\\\\/H4mWnHo581le8PDvqLYsNysIfCVyY+ikjKuFAzH8zBZA7cGn1ajkaq8qfHgqBJn\\\\\\\/GgRnbFEyU7oDxA3TGzO8tA\\\\\\\/FNupUwHnpSP538pNYyFxmeJVeecTfSluWdPJJW9vpp1uSxd9ezTONfIASdyrdIPcD7pUK7M9FAvhI59szaprQdU+Gle5vLLLBgeMprZuKTjyPJzxKhh36bmsu\\\\\\\/I1f9007Cedfbac3LHGKdR6RSpdundWkSJrO4ljVOLQqnrB4\\\\\\\/WIH5XjT2xcML7arLgRsJNq2fYvAdAgTL4I\\\\\\\/6UY2W3TYx7oICbc2Hdm+E1MatnF7UWpQeQxqSCiIhMu52TeXZrznigj7k6goFE8QfRN2L012duZhOLxNAiknvl8FlXKF1jwKdpfCAwerAPVStX\\\\\\\/4b9yKlqopZgguEcSaIgRcjrrLm5\\\\\\\/+bgHKkZH+AZdeD\\\\\\\/\\\\\\\/IpPTJPyeLOIlXbXEPxS6h3ovS+poiR9\\\\\\\/ZhckJGB0uVYnvgeBW+r7VsO1nSxq\\\\\\\/OJXZTYqTTkxYdZauLtClqIBdNLJUs93JhWV\\\\\\\/b0jaGlL9klBX3XO2IpXXLfDPrrYQg8910QUOpKyX2azp7c8YfYViAjhQ6QmtVxo43eD3YA0s8wOglS4emFXKSnbgiCBxDmm7a1IM3+UGi7RGXiW8nv0M3x4gENLpPZjI72TPsKmUbOcofCFXb+yNPOqOcZiFo6pLouuaqkBKgxPU09CT2AcupQQTij2IIfajhKXzKkMZw01z19rIYDZ5b\\\\\\\/ruwUalXiCNsegxV\\\\\\\/BwdHNv9m3g3N0da7o1WGsvYbN9tkJf20S3V\\\\\\\/nX9DfT4IgjcXPouK0zPgVz+ODIbVU2DDKm1KMr0xBXlwKqc8c4WXfW5YJ\\\\\\\/zzCJQZYcYjZVyQfm92IaZyp1hZBKC9WittsqGBTUC6DJ3UCgehZZULuOgp00BUbn8Mm8VOybAI85X4s680Ut5t\\\\\\\/rwygdEQDXsZ89nfMQmTb4AXH0WkySz9byy+ZzOwW2LQM5q4Yek\\\\\\\/pkqUeIdF1cP46XX5DSLijd4XAS0ywnCfPWVEoHGikJJSUhkyIehrjr\\\\\\\/6mEbnwXzB+295TRkehggUPf7bozf1cDgpKLvSQUT1z8qTyjJrdEwwqhj06XkBjif3LHAd0ekMmwXaAG0xlOn0Lynr8hEt75YDpQ46PnUgVBCbdXJebI0YkSA6ZIp6sHOKE9Nxb4Vs4QAUkHp8mp0qhjpBrqxBFgQq\\\\\\\/wO5ZxZeZZreFpLW5g36iaqDQedWS3u+Fnr8MjwUO3Rttf4Q3oKyGU2r5JR4R8H2jgvGWtyWu9Q+oM5AYoqHa1KvESAQ6970Hoc7UTufy4LXIPRcnUNUY0NX3DXFKXr1iPwcK29Ni19v94yGURR4GDLYMjxDIX+yGO97HIUgcV5\\\\\\\/pN1hMMkz39wHQR5WbslU42VYgyGjqNlK9XIrHkVhCwvyxO\\\\\\\/1hyuznyQvZZNu0wgxYfjPsrRnqYrk3L0fq9kYeRR3NXLIf2ZGx3\\\\\\\/pz30ewJdTwgb\\\\\\\/GVPUkmnN0wY61A1nYjhSJX2OwRXTOqpPDC6oCs\\\\\\\/ii\\\\\\\/1UEyN8Y8qbDTdpUOzXRO4aFb6N09aePJgiByNq\\\\\\\/tamawyXf5DDvpq883r7hKNpqKxi7SgjwXiYXhYuZSYiA\\\\\\\/9rdg\\\\\\\/HP4N\\\\\\\/J\\\\\\\/RvOtgpjZH1Ss5YrSeX\\\\\\\/EEb3dPl\\\\\\\/7fUVrJ9AnIv0Yl1WDPDBNJa2SzoxvwLwP8PE+A0NXGVkcXAsjBfjo17sY80CNxg4M61S83AgDlDLwmM8muzuEz72+DSdl\\\\\\\/HGX2vUQFTdbfkiY2xbXSuPaa4AdL2baB6iLxTxx5N4adnkppjLZPwPK1W2tG40zQZu4dYs9WGNigts\\\\\\\/EL65+lC4ct7\\\\\\\/f+viQlMhIWgE6SkE2gCso2cra4TFR5fgk8moYiGaxJAqtdjYAoNt63zydmMZ9riKTC0XEbHCFHtjrV1N242hH0rS1XBwPzboKkYazQWQ03Wm3z0TH1\\\\\\\/RZpFJHLic6n6p36hSNTa03wqGRG8ZLRV+XRST8G8OfP0WbthIGaDs4OBrxjtqfbG+28dVoUcktgJ5crm6DHskqBT2DjAKoYoPWvEtYK+kZcg5\\\\\\\/njMlGSTeCab41\\\\\\\/A903neptEYHrWKFy5OB9CW3XDYvLLhUEprDDhVgBt0\\\\\\\/A\\\\\\\/tj3\\\\\\\/QnllT3KlRWGzEtgEZxShNjEWxrDTey2mpc5puwHwLIBOe\\\\\\\/1FBjGC8\\\\\\\/rBeLZkEtWlD3OhW+j\\\\\\\/UNwTU2FsHEBoo8+65rJsJ2A7ggekWteZCz9kgdhhK2BFDqULRGWiwQmiGS2\\\\\\\/Cfkoi2MR43J2wPAbhVWK4v\\\\\\\/58Vi7bhZLl3nAwNh5moxT225tmR+jbI9YR+vEcfYgnDwA4SrcA4Zg25ZC9uTRW\\\\\\\/rAl+5okvYZYtinCC4fScAy1vGCsvz6M2OKxKKUJLRxj+Cfr8gy5mdoJZxMGeO\\\\\\\/g3XwZAWiW4s9UNheLRMOeWMNinoR8wPPyvq5ZMLqlrN5p7bEJPA5duRhK8tpUAybesC+mtpVJhfchi4YMu0Hi7FtW\\\\\\\/KIf7E2UwFkg9n+ZCAAlz4ny942cKk5vVqXLIiJ+OMQLI51V+SPbrn\\\\\\\/RRjod4GPUmaSEJlbYEuxHGwxmtktQeReYjzHR1IZBrL1n4guH0360JZBJTUF4bMWsZmNA3rRFoqgn+B\\\\\\\/Zb9UZ01TeUNldPwqWyzBhtzNx1mcTjcs8bHXxE430oFtXVQbkUMFd2X4BQric0m2gnbTZv5JnNhT+JkyvX\\\\\\\/9MVv+\\\\\\\/NlOubx1qC9lApThWC0tgHxRq0vi3SCu89pwroSU7HUtkMgbuCcSCqsksML6YYvemJTCcQijW+VuzbHiqe2XYu3J+ZpGU561KHK4eVWVK00Tzo3hpwN0SxZBGrXp+JCdOmk5TiV5nwT3Gif+lg4x70G27rl8ZD9MjY++hvMHw3WfSECBUH7p2dZD2mGDjLKsmQkFjNuBBOhYxStYttNdwIGuvJRzNTjoousMuZJcAaE6BJh7KDbrD5\\\\\\\/\\\\\\\/ga6e\\\\\\\/zQ+ypfVDxLBGs8\\\\\\\/dlyj48QXAsVYn1VQIDQY0Mk9xh6YFQ1F2OGR7WTpjxsZ6o4cBIyw6eYSlOmyDgV1qk7LY5dSPDZ7NyOMEL5xa9vciQQLqIfbMZD1lloKlfstWB6x8TfE2HB4ZYmVDsd4ao2PWXtAIv9AbENbdq47CQlisQEab4p+rSztqJYm+z9RazUtoYnsUAA8BItzKtoxOzJZI\\\\\\\/dobtFfvl+w9\\\\\\\/v02jMuK90HndNoeiR2nVHksBI44Uh\\\\\\\/47ukHvkfY2qznDmypSvM2b15qYYcZKYu\\\\\\\/eZFnMXIkBRpBPePlye2epmoOUy1JGUNSDr7CBNSjGGVgma2J5e5F\\\\\\\/OJ3nf81fS5Gn6\\\\\\\/T8eEh8qP\\\\\\\/ldrZxBULilPjG20y0L+11vcKPuNDXIRGkxPoSqPfvRZMdxHdJXOid2jpq5PMGPwochFVweWj06F3GUksRkV6A1UxejldAWlK\\\\\\\/SIqW2q+kgUCNmkHUJeiJ3gUuy7tnROJsOukKLdR4BLCeuIHB8GeWbmWkv19jC8+VZLeVwndWVRiwnhfl0+cqlC9jg9FIauTotuFqTVHppAeJknwsiOCNEbC7cCrzy9EtpulKqlQR+0sTaEQSW9XRVxM7\\\\\\\/ZhRQ\\\\\\\/B7kMnuMbjguN1FypcZiApPk9Wkz4adz+L2PMsIqY5sKeWwTa2wHzkiWntJQzNLYKwVWK1s2xVy2HdsXcXmIcdV4CwG5\\\\\\\/1nyskhEyqxdnfkkbB8nrr76KwyRJx7SK1PtPGqyHi5xLJ\\\\\\\/RV8YH1qXY2TlqWXId96tNNPjwymfLRNEcUiOODhCamMrXpfFNxSopkv7UvJbUNygXMA4NrFM4R9uA3fRt78BxS8M1CmuJkyqMI28AlqtpCTpsP7jzKbAk0OftmyKbYFu+5DrMdOmDda\\\\\\\/BCpEfcV32H9Zv3vxgm0iS2\\\\\\\/4kvwWKSQHTSWGEFKLSUiXNJfDXjgfSMaasl3BGymmNwiITH57TWm5aFMtuhBgjzMAYQiuyCYZpCvXhZtb0W6+ETxiMgbOHwzTzejiyH2s7qi2Gd0\\\\\\\/\\\\\\\/z4RGGP3MuUF4N6\\\\\\\/wR0nt2MvvqkEPwUwGXdTmERAODq1ZJvMizr0hLIOk\\\\\\\/IH0l5p2bRlgS2FuyJMjyACQFrbPizvSoGIzUI27ntCEI7WlSyo+eEYcDSTc+tuRCUjmMcHabM+wrSHtaWfgwpsWLXGbvS\\\\\\\/PfcVmY\\\\\\\/qmCwF\\\\\\\/RoJTayqhYKNAO8yFEqDzuu1nOAgp6T553ZWdJna1hVdEf6olFPcO1No3YzHvXr5ImZuG7kpjCr2aEWisqraIZlFpoxcLbcg5HM4AXyfbUKzQXqx16auGORMyRFe9r13Y3i7oX3aji7UmcWOwKC+fqsI\\\\\\\/QnhT5Nfg+i\\\\\\\/j\\\\\\\/U8kHjv+R4t\\\\\\\/\\\\\\\/WawDwZIsz0FiyIB79WhG\\\\\\\/ZbH7zcFCv80CpptpBZmh5hwe4LGVExi4OfVN9fOYqZMSmCUWNATvr7rJjlXwF555tSakaH9+ZvmpjmarR1OvfnDVSlN4Uj4z8LtKd7kRR\\\\\\\/8jvd62EWrefXNLDpyHGMf5uVxIFb2wLX1qafM5nnGS3MCM7PbrTikSV1n8lXOBsUlKUjWaLCoq+1qCY2SitwWepV7uEalg5iAOkrWrr\\\\\\\/xkwZZRfghfr3usycQurGePBcF5s+qMnvTeUN5G8I9RH+62UsRljHQ+ZCIY2x8T2mZRf1z+7OpRak4enLvV9Plgj4oleRBFK2LGpZ4tHtxMQH4R\\\\\\\/KCMlFFWfg28eHgteVP5FV\\\\\\\/UNioBaTIr+rwlcJqiH7IJuouJk7huaRW4ufV2MGc8zNbK9DmwzpWBrkYfFwPRC4Lvx2z4nbuJoSAwTGDy6p4bI3zcRcpT4nRsFUPwuZGoJV9dENJ3+MMevi0JN5cIlu4qyMuahyooi+55rgbn9Oa6jMj7G6ZW5okuBFn6RuY5TkIWnt\\\\\\\/be9RfZtsdJ140+tjPwCJsB96\\\\\\\/hj2PFhSxdcYUFZMsZdo7areKaDZJR8NI7J4ARcBEhnFjVnEaM\\\\\\\/LKwVtKq9V1aimkVN33lA5R\\\\\\\/yVjPdgxb9mEgRqgbdxbakv3eKzKqf\\\\\\\/ha+nwCUUQyqAdsFmiHsgkD97Y3n6Xr3ciYWMtGMAj9s6N5Cg1g0Amaie8y+fyMo0KgsFPLHVYJZCxDBTpw95sy9JCSG\\\\\\\/1KtZMMejeaE1jMZHlMpQBqSgOWskMQlqZL+fihS0Ql1VQ9tAAW5MLVPlY1WDfOOuRYC4s6FCirp3IAKXjjG6RnO96vQS1ocSedGohKlBT9QlqoOXZ9UXGjWoEBhBMECHI8\\\\\\\/7zC07Ta9+CPgcyRrNizI5LdRKPcGcDXB2I73zSrfrJK\\\\\\\/M8uxLLYE5u\\\\\\\/OYGMidgrEOP9XxqF\\\\\\\/G35yny\\\\\\\/BD5NLd7HblUEjy36j8+DzBVFWg4JHcRpuHrvS1PbGUDoWe3TptIW14KfHjzV5n64eCffSAT\\\\\\\/uPTzOrj23hccNXiudPD+rLP14LV0\\\\\\\/iOZKt\\\\\\\/6gSQBN4TVz4Pd09ag1TA+G2GtnIUx0ml5AJAtU17J22DmQh5XQlifPXy6XhZVqCCgYuhCIePn7iuzYU0RbBgA01zATG7oGDNWgjwV8Yxb3JPpJWAH9gFVxV9rTl9PdGBP+UoejnhoYf9bpkdyKVi12MnFAlA+RKGoA9+Zs3\\\\\\\/ksYgKHtIQjiCqWlB976wElpPVPc54MC4qi0ZCMdM4wFACjdfgc\\\\\\\/T+Kr52U5DmmyTAF9507lW453i\\\\\\\/kX65HwYfDA6oYDdNP3sczjW5jk7ZIw7FvQTC5My+klp2K\\\\\\\/VOfyfpxnsDOK+CB6CfA4nhZt1L4c0\\\\\\\/\\\\\\\/9oPrIV0JSkYQT6n6Ws2a\\\\\\\/w1H8nfPdyTKaPBWXZiz5iTSIUR2CRtNT9RWxr8QLnMkF\\\\\\\/PKKUWwAYnTbKYUwyfMz+Zyz+4OeCzbzs3G1dgkKsmkd+biBiZbPjiruNPR5572EJOZtKEiLIEtQzklE1LNGbTOeYDXwiKrWqiLkPC7ZmT4dYSsx8sxUw8WDCBtAkXlgJP1tNhDn9JeyQSIf8h0Y8oIW3NyDEwaTPlFexp66sIoPnCtvWUlHODw\\\\\\\/dILJLf47Fwd91cvTXYztSBh8D2+doTUdZHYkmPW+NrCDWlGtD3HvG2Li6N1VF90ekr0wANrId+b4DBwzEAcFra0PH9HbYOSYQeTNVGFqezp0C7znUfdbdgjht3k5Imeg9m84eRFd2g2xBzuPwN\\\\\\\/NZnWl8\\\\\\\/rAgekDSs64pBHiKr36Mqr6N67oo\\\\\\\/emixQFPvxuK652Z5zMhOCKJCmEb+sppJfUSjU+DU4bC9Io5XqxwQDc9tUun81w+wZRhUXcfPFaX6sW1niOzOtnIHOUtj6v3DUo6J+LLFFrd+rR0w9EW5hI4oQxdYSFQayTE0Ei1+BJpQh1z8FGOip\\\\\\\/JNG74Fth6ENstOQinEOu8aotJPYb1pk2RTOv5eMIXvZsIzV+qSV5P6OI3WFsz1DrTzVydxpNckuRd+CZg5WxSEYKymNHouS4YhJSbcUtEIMzGKvLKAh6IxO76BS9wncVFRSrl0UeZbd30q8Ne1DBiADzNIcpAgnu5zdzOxQMGnUmgrBK7JYZti3CE\\\\\\\/aQfutV6HuEhLCKg66wifj934yjj27x7lwX5Fo0CVwWif9iAmL4OivgYqC88pQ9xs3\\\\\\\/nCU3zjai9PFdGbtXsuuNFlpDnhT65NNjV\\\\\\\/4vEpuMLVaC7OjA1EEnfIqtlE1B3wWJW66YQN87U2scVAVQykL8fD6dQWcg3WkeGd\\\\\\\/5C3xNwRT8lWm6p3ss8PrcbI9IzvjRd2jmwuBzv4hgLDApykm5J5q+oju+kdzyr2OStjwXlc6DBcFWN0Vn4htSpR7bOgWsrMNN5rjkFNxph2Ar6rgV6NqsnZhmPXhlHq1byU4EYwgMI6a2cHA1shXinn+dvjsDgKnYpVJCEEcBzK1ve6Q4UL1iA80G\\\\\\\/FoDU93PZfCn2349wATj5oxOwH48OmazQvelSQHCO0tg97EEggLnu+lIo1pQZXkPNYgFAJ+XTzJgfRtLo7fbWs5Ug\\\\\\\/+J529E9502wxLg3zZC6Q1IpFhnIknZhiJVVeN2g0qFi1ApILwpmJ6WYEwyznh9jwZdPgyfnMaPK7MSmyCTjHi6yXxS+P4LlOvVBJFiHIJyMgHbyQXeWHT1SXjlP4p94IHIvPw\\\\\\\/bneL4kSZszuKxOGUq4fbu+fW3ffdpVvrQ7wdkGTq8\\\\\\\/Dx5qAoBzihV2ShBMh96CCIpkbZEk0x15K8L6fX\\\\\\\/yahI7tDyAPws\\\\\\\/SAdnM4ZBeUEZ4fw0n6Aj5Oq+4SHoxsL\\\\\\\/dZobdO9bhUjUEL9xaI1DlIZDn8hPEu8KOwprXedGrPcbYiH\\\\\\\/xAeDBxftcibKnWF6wzryHkbhcJPoB1HxLgYkXQpRIdx8fFCu3pT+9WvsxssZKzBsn20\\\\\\\/kYz97K8EbaCWOffVU8T9d5COJ83W0sitWDfUs87CxIQbhDgGbdROqgJqtl9aLkysLwCze3MKPCjEbQouaDAqwaftTiufO19iWU0rcXi1JOmF\\\\\\\/RtBAMooQ224QMRyaWhP4s\\\\\\\/qJ3xgMQVALmBDj97z\\\\\\\/6xHglCPOm9yEfG3yNNy8WqH8hxoJIgsUfebOJDxSf0lH8SR4J9pblRn7e0caPYS69twAorSz7pbxu7\\\\\\\/Fgi+YXHfOhgJwQy9VpLOOMv1WAPoK2AXYnlVaTU2d2rOqIdtLGUgVzBl2\\\\\\\/6NX1n0inF\\\\\\\/d40VY86Mv40Qw193UUhQY0EM0OgtQQ9eKnrX6sqzfD0tFwLPTOFpNApiQEA7XCqY4UIStd9YLhjfBPPdN81Uati6jthpfUvXYUsfRqhkIbEDrlLR2XW1MiaFHmY7jDVspsVPeeFQhyGuF6qZDyLoZtv00fpk\\\\\\\/9+3t5soBoHRvf6cFMI4I\\\\\\\/haOHSD87tk7gC7v6A7n9JOEAxow++vmxfeb3iywAF6rf+GlZXfOepsQoIXvsLXeI8a6A2n\\\\\\\/CxOd1f4tPgiiywicW4H\\\\\\\/tyA6a2PKCRPiQHDJE6nlPreto9pKPLsWUFODJpUlRca01yS4TFbu6JgBgDsZqz1F\\\\\\\/5ugEyfNbRTATKfjZBqzH4olDXz4ERI4CbHkJZfBelKNX1hjoTuRG0fEZ79Bd9Jc6aqOlIIN+NE0R0xqyz3\\\\\\\/jivYhp2bWETN+b2Syb66ySEkEe74fwHI0x6gWYrcEplSj0IXIFIAGGuMkH19n6GQXy1KnSOxRD5NCRPm\\\\\\\/S\\\\\\\/3+sGKAfx71sqx+977bU2ej8cyi5TTR9Vtp4iPXXS+gPW8qvKwFrJEvNxIqme7tSNPimZbTahwOzxFOHHyjQ3kzxh4gmS06ODzwBmjCo0Js1VqfODTbgMjWkTUUFD7oaLz6RfSMObHRynTQ+lhso+VMSaRpSkWs3Q\\\\\\\/ZGG7bDNctw4L+GX8mloSSyYMCeWoPmDSUGg2XcWWNJ7qLTw7fA+zKOIBKf3Ii+RsLRzdITGkovFze4QZi+ALFbZYUzFymXhNBOA5Jc5UmHVB7gXl8JwQk9IryeiWSp+io5RruwKTmr\\\\\\\/gohH7uZ18y9AkR2H+fcWVQ18LNCVvXusNF\\\\\\\/C6iR55gfackz9e8ajEO4ZE2bwxhenVB\\\\\\\/rQc+JXu6nRto03N4IkiA9kXhk17fYF6eEygsbbml+VMvaLB89JbMzeJVkj2r2OvPD+iQ7iwHm+HrUeCfxt0q+WI6miH6Q3WRkvsOTgkhOgPvujh6VzW2onbqmctkuT\\\\\\\/ZEzSFcJYe\\\\\\\/WbRmLNvD0zzgyKj\\\\\\\/wQ1YypoL6GIYKXc\\\\\\\/qi3JXfB2Jc2QTwdn45vJGkHR2iq2MXil\\\\\\\/+HLpCLKyRzXOHOCdOcJKykNgKzXk2N36gkJMQ7ZRoCd6qyIEKyjdAGU+9cDF97zrvCDXwRqo9C044XP66n7zq8phLn+I49nPUcNGS8QS7P+yBSHNxYWbEPV2wYygLo8pL5U6hBfRcvUqyqeCI0SG5wUKIf0uf7FAhtvXiuTxtZh\\\\\\\/rbijBVp567C08JO\\\\\\\/ID1bd4s\\\\\\\/Is8JDPaO9s7QneEHG0idiSrEw12KwwpLqA7xMKmsU8XW30ayNntGd\\\\\\\/phgI2J3Li6wIq25fB0Iv6U+vl2PsO\\\\\\\/EwamBOCcxYafVxoVi+126VOQgYdsDaTMiigoV+QmdyNJ0heLcdjD1J0eQJ2860DI7Vw1Bu44vDncl5sCQieGNGuMCRasxopa5ZIaKocgPWl8\\\\\\\/71av33\\\\\\\/BkPT1Zh6h\\\\\\\/hW9BDl5hoUNCTKmnXovh3dltNrjyne\\\\\\\/kiXw4UJGAs7+vQmlExGyt2JZOXtKIPpaR867M3H8nHKmFZoH2zKw4HzA20gN64r+NIZQDa6Cu6LbVF9cjmMC5WTOxf3Dpca0LayWRoS+x5Ao7CtLXzaPBslMLBJsQjm0lZMxKabPfvhyDPJ\\\\\\\/5+GUue\\\\\\\/TzeuvI0KXBW+fkTggkxH00E3xtaQ\\\\\\\/ACqyXoNoLX\\\\\\\/zxGSfs+grPNzwHCRB0L2qxeR3HfhsXdXh3WMgdOFKPSq9\\\\\\\/e34ssSJRApIcXtPLC7CdUNk+KV\\\\\\\/IYOFRBXT0yvK4v8Wei+NhN6RdagaxBtxk06V5tWB34CL6aEDMuFzQTGSYm8bXiXvWRiWEHzw5XCNW1GppE1T4Z6xTbwn4A\\\\\\\/DIF4lUUurZp6+Ki\\\\\\\/5FwB9hesYRI0\\\\\\\/oVJfcGVH1rczahL2cNWDeX3QSXVpWR71p6vmVedA52sYbRVMY9KDMTIewEN7AY1rEDbD23KS1tUtzQnl6WlX23F1BomOKzodJrVE7z7I5XvqrzmbaCHnPW4Clo2BgbuAi9CTeILKKhttskkGNm6rZn4QAUwzJMfMFr7JmgOEhscu8B5NchUTR\\\\\\\/HnUXUmvjemC7zKcbwXLIWiRDvnLqZx6JxHYxUIdD6pSe6rQa9+QncmwU2C+Pgrxjqqkwj9IgPNOvH3DM\\\\\\\/sVjn7zbMh6m\\\\\\\/Jyth85HfoE6KzHKuXWFHe\\\\\\\/7yqH2Wpku\\\\\\\/Kf79xc6PSrZ7\\\\\\\/Ljt3Y9NiPd8ftZwzQc35F4oZZnqWOaVlvfnbgTkIHlUNL2Xk1KrijPm0781bMZRv9RDhL5Oil9VOFYyyUNMRN0\\\\\\\/Xr3RtEnXjsubHdEke8GvRUIxhUkyWt7ZFqiEjRYh07Qmp5n+TI5IfOoDszQqjLjIpNTmblJ3OB+YnmleSR8Zdlr3Brc8FxKesh4gZrpxaecye0JT+yBVQtw2rTiORSYm0f5e2XNC0sC5wHz+sRejRTxMViqyY2yXfQrc0WkMD5IOllq9dm4HxIQF7zyNi3UnExWZTBQ00ctLG0\\\\\\\/4H5lqN82EWbuE+Hf7NYQMfjAo60m1YbJnTjLdwazaCUb2OI3lgq4cJ3FZ+Kuh6L4ivyKZ+yV6A15lh6dXeq1oMQHs8cF9ummxfuB0E86Nw\\\\\\\/PnIkeL98ZVkhl0D96LBVr41fnaW2JlvCHR6MA7OOjjS9LOJJVDpnyOqZqR031dy4wgotH7HZwzo7yDnrh+JBb7rGqBdIszodRxq8JPayytG4ZdQ6Q+W9EjHkgzqnJHGnY0ToTh7Jxs\\\\\\\/YaFiK4GzGdRFbx0sH9szfMVrLbT0f+oJhjk2rFMft0ZgpKlHGRcJD27qy7zlU4+Tj1wuIpyv9JNj9S9MyEF4poFI6z7ujKrxcy\\\\\\\/rIW9NkmhtOCFV3CmnjCIPQAJs5UrX9bn9Yj3DvmwdvlGG2uy0xoIlMJlXzxzv9D8OROZsAWhsGJqeaYTxjihnGnauFEnm60qTuD+yFMEa+0FaE3f7tBImaE5QzgoGa3R8Tb2Tcbs\\\\\\\/Y+P02TIRAQXiCSHY+aSbGqr3XHCAQjQEf40JJ1\\\\\\\/kS5xwnE\\\\\\\/egD25s+\\\\\\\/IXs1ley1s1DMqtn1LFQXO4j\\\\\\\/8o+LlRKCgbcMeFgkCeW11v4WeRWIIAArALlcLiTvPQPiC6jT3Ow8qRrZY+qB8In\\\\\\\/VRRiiUAuJdGKCV+TeTi7qoyygBePAFOG3PCcEFS9yQNDjRpwotXAbLrmOwncc6owiHFGShY5zuiSsn+tLzFLeUGoeyPJr\\\\\\\/jPLwO8DghmTFPjMPPqMkBAWvlZm\\\\\\\/w7V60kehz0Z0mEdScInp+B6sHFhBtmUZaBqFH+BrvwXk5pwmL4xDZ1QZFhroMULgJ+N+\\\\\\\/bs+uLnZfIruDo9a8tUY9DmZNMiQX59uaT32WMhCiabukS3IDCpTx8f2AMfkJHPcAZ1D4awTOmiXwutMZkOP00yLm\\\\\\\/R2BXTNRMQE71LAgzSFD2\\\\\\\/NWD1nHeKeAziWxUanRXYdmbLeU+h9sQLzNkSPJeR9DlbJ2fPsEZWJQHn\\\\\\\/8njEmc3v8iolcRXqH3LSjEnQ\\\\\\\/4xhx8h0+yHNhoohgr1YdcIACaVUE9DG0kz\\\\\\\/HXOJ3yS4hZxjwsvCvOyhNzDvOkvKo3YUir4FX1OYgqTiE2fm92p4nVs2ZcPaXeV\\\\\\\/H7y65OA1jAVBzis1ieRUf1KjpsaCJdYDk61NLTNyo8VBBIfS2In1LLp7RrODjqdbvKY3Ddv1vcKb+IjzMRctACGRjcz\\\\\\\/cLjEx15Z7h+lb7TpfPrd+1fgKmz0k3kgUs2mG+XMqBjvqcUISPsX2mBrz\\\\\\\/Slqgxdlyppz++qCJh8d97VHvMsvv8ufenavO7d3qRQUGL3Wj9GLf3qKlPpzFDd3BuBZ8g2WeNBx3G4b+43Ik8E9\\\\\\\/wsu6ASFxUPAvg9kirT62BdwKXeQrJrqnRvyuiEkl8PmKA1Fv5ULHK2KoLHi\\\\\\\/kPvN\\\\\\\/Onz3aBp7pf+KPhyBzMWNCaR8MeNicXxzojtaVxZoAfcbzIDkf0U6Yymg7K2ITN4juj9SNJmVuOdehMx6jOYEkhlW7I+EiuvOq1ziTszyWDXtkq0npE07TgIxx5EQYsMIvvkOWKRuiHNlcbWKXou5JyfsXZaI+Z6JZZ6cOMrBKjHrTdZ5ytVObO3UzuD1TxdEBlp3bYs3IcMCAmVXPc7\\\\\\\/QOKXOFzxRaJHVRq9h4x1KnWbOOSrwtK0Sgne4UmOx5OgU\\\\\\\/MVgnPSiTZLY\\\\\\\/GoNJbKIPA+YpBdRWl8XfIfjP3H18SapvyVQZOOYcDkC4a5Xk+8vUZVNccKI069sumfEbboQGaQuVuV20mrRMMaSuj8EKjrJyqehRz0TPKXyFGIJWvflRrX8AyjRpE9MWyCjltUuNT7uwWUE4tDbGaalefPtiaRqPkoEHSgsJJ31grCGqTImQGbBXTYqmaa0Bo2CfgGIGGEEF6r85qsyD5XgCdA0EjE2fXbprt8rEXn\\\\\\\/AmIXgJYTskd3L9YkSp3Ymcjv5dVeWgRgysIuRLzZMvoOld4LBewc7RTAKRE1yL41tzMe+StwW3QRAkBvtZ\\\\\\\/kELu9vuk\\\\\\\/fs0hjtO5wF\\\\\\\/gZb5JnInXRCxwFj\\\\\\\/yhImcrzyPuMPoCZ9JHZUZ7HyrRX9TCA+J+ux5uDZxj5q+Hg8ySDOI4JZOTk66s7tpdENciF3iW64i4MFGvX721NtIF\\\\\\\/cJ8PmopGVpBFOJ2EDYHX6uDlfU+0SBtfVQqj+xjMgm8HLlPJeurDQ3McqoY3wFFymVDh3oRBcffS8Is+MbfeP9E73+XCiWxo\\\\\\\/EIc\\\\\\\/fXphtmQ+PNzL\\\\\\\/7r7NC9GPtVFypiQcemGLV08Fkk+LiXuHiIke2fCVBed+ViW1xWYwiYFvcWQDAaOfAYDbuqgUZmcHPEhGkN93W8JjQAGMz5JO3r5ZmIhIt6coTCfMiKXOD9nn\\\\\\\/CMB4venwktj7uNYSnqK3qFYDCTG\\\\\\\/9SPPgZOlml4vec0ySINsgxp6rWufX5rGMh4YGUAhX96jG3YCdXJrb8e2f3iw7dRtuMNQeIrn6JyhHNMq+6gDD2ouidlPsL58ZgZiSUKc0AT8NrKSQ1ngfKsx8ZtmYBgEXKACIdiPTh0uL1XshtT4gFb63T9y7MAA3YhEnAamT8mdDQXZ2uXQCbvI\\\\\\\/sReRSewerdHdTbv6ATNykeyGZd+cMDyUixNs\\\\\\\/wDHRSg8vr9vi9NJ16lh9by8AbloTpxBUJ1aqYJiTTDatb0BHmL8NGrtVxjYjSQ65WpI89D9EXogclnVD3G9PLV7ee6aI5QDlqgMO27hRmGjmLo2TreOlib2jxd2WSO+xDLmzCsARmBFIBHJir0ykYdhpfKIiSDHR2DqRAklbC5it1Nvya2IEZJj0CktFwS4f+rf9O9JeVPRIFLQ2XN8c3KzWLkaaQSWzPEmjoOfCvalK30S3zlKOfPkRbwGJhYffknE4m9i\\\\\\\/um5YR+4VGRP\\\\\\\/hDvGtOANlj6V7IA0RyAm2vYXBfpzOJybRlvLu6tpBhLV5mV78FdbFfzsC7oWrNlMsChWgqo9acyfgnV6i9ndPRdD2ZvIGrievPDM3GD6qKsKNmUwKmHk547XSUNgIm5Xuvw5C++zN3oUbrlrJ0R+G+TfEDESKCdjRU6X92YhcSPbKFKvWFBx3JTmN0V1U8gJuzDDgGb6PdAN92Y3H8Wzx7MfEE8zAJyw+qVUh1ixF9FoOREk6HonE63XA2\\\\\\\/Qj7b08jrCGENsY5u\\\\\\\/7\\\\\\\/IdG29vY0Mmtt\\\\\\\/VWB8NQ8sXQduRUp4tpfqi4k5WSPuvu9iHRFZcW+ammdZyFQ7LepwcVAj6YlIGxJ7yqJYPa3eicRHSfUQZ7FcY1bq2JtfnUQwKSOgQDQFCGLS50LulGrqMi29vnJSpJzC97JfFG80W4\\\\\\\/19kBlVCto+ma9wa0pMj+tZe6Oq8kwPu\\\\\\\/2gEQG89MuWW0Ko9XWQUa8L1sqbHnN78jTivZ4tptdtcdeIdXzFrK+a84MvLmUMpqvWNak8iL3Cr7uQPKPeTRi51PYS\\\\\\\/\\\\\\\/oe1ZTdMSLxU7oBxIMEOHWLRd9Tjcv8N\\\\\\\/OdHmSD6fXsBTwnKTm7cgzj1ZsIIoJtD0cfzzKpYEWmp3Q+slib3Z25WfXQwZARXTypdrTM48nFE8kjWbktg31lOV5zF+y\\\\\\\/LNNlxnXKthiJKpL30RRx4DXQephYXmf+kF5o7Qr9vyVmnxMk8eBJdBCEdxXsxLt+VCRw0OaOfjcNujRlNFibwhXZ8\\\\\\\/Bb7HJYuzOZodkMtwo9KBtSlYTb+UV5rxE698ODLgqu7R+Z2LaI4ZnC6LAGo1EjOzDtCRg3OMfQFLuKESARTF1w1OMiPEbFiO76Yt6KrL93T7\\\\\\\/v01znPPasZTZ9whOxXRvjefEps7K8J66VZX6ZDDULzNL+DHMHut0m6yJDRmKVP6aV2kc99EXuL010i\\\\\\\/cUbH9DqHE+CM3TjatjvrybpFZ\\\\\\\/qqmTFmpZVGcz19gBI5dQZ6SEPT+0fO8At2K4FZSrn2dxD51L9e5n9KqCUBq13QjHnPZiPctk6NII6OYoiBQUWVICSpI2NcueQ1SftgP+iyF5VQB8wkVpAfrQ70wzwRaqrdj+RdUGhmukSq1+HlDnxNtsHfxgx7K99dfLjzBmVGIzKo14PpnXky7GVe2rQEaUDZC6w62AHSCBwksX7azXZt3CoR2zSJFUoD9Ag871sE4WnCrIUx9ACzxueMM9Iz9Jy2tzFKUMX5Pr\\\\\\\/3ydtECVTpBQygbqZ1xp9031nQNPGDHgrs8PDmHrXoFrKohM44fhKWLq3Y5fYIlMl+UjmRjhKH+JYBxdm0+dmUWnJlGLzw4Hxsks9y0Um+D9kV8TQgDBm4rTSZ0EnkD4\\\\\\\/P\\\\\\\/UYP5z9G\\\\\\\/y6NERsouhRI\\\\\\\/OV5H4eoYs8ABRdF7nxVPNX39FOuZVN3MubDsf08Ocmkn\\\\\\\/V\\\\\\\/LIhGYmS\\\\\\\/UxhYR5HLsQpN1PSgbTuPB1qNsCXshilv6bthHUlTv8nsUodfomff32EqCrZiSZ7Xi+cw\\\\\\\/OY5jevEcLOgfdBU4mDXb+UJ33myGoXsF3hS9aPgMfFf+vZs6cgA9J4DnoQti\\\\\\\/7M3ednAN7sC5nm\\\\\\\/klIgRkqrR24jHyB55uEfUhOJs9n078iglqPnBdSRAd7ttaAYm6fjnXE78iyHI00izPRR9pjOuwo\\\\\\\/t3yTfFLRLCPnrhZR1NA1Bs04kIOlDkIWffWDeuIXqOm4S+XEQw0dICFx3w3t2aLxgd9aIBRJZQ0po6yjpIAet2YdXSALpIAPfRTEFEjUHZu5l1Ysl\\\\\\\/kZ457hBfh2GiRbfuryl+UxT2AbNDfE+tjUxUqailxdJPkvREGr52Ch0VE20e+\\\\\\\/j8IQBQLkTSIPWY1gLlUEy3int21A5tGzq9JNlrnNTw\\\\\\\/mv6TBf\\\\\\\/G\\\\\\\/qQ\\\\\\\/R7NzeDEkKY784tUONFn1oWxlErGtI6H\\\\\\\/qafWr4shp6knDTYL3MMgtuoXMS\\\\\\\/6LxCe9MN\\\\\\\/vod3z9OG382C89vW5vj9olYdGPkvHEvBR7zGitVmHUI7ddlj2DedevibGOUXgA9UfIYLeOwpD8N7AnFeilwBaRt8xFq0K1zY4kORFzrFyC0gSlUiLIgOs2DXgKH7Uygo2otdQr9DhWt7uKZQ4fKnIuZaaOfc0Hn4J6N9fbj4bkChyYeMxlERbLq5DSzgx\\\\\\\/3FLcY3Wf05OFj9mfjpW6HShd+awV\\\\\\\/M1p+Zs9nAHoC2nm8KgjKylTuL0UI24bLrmf85Lzp4kcY1QHtJRvqaYBd2otuDr2EfJ\\\\\\\/yw+vsK\\\\\\\/SWdP4vxVgCJVLGYuSNQfWBNtzMTHWizd1GuRBMClJdGknS+1c88nUhCg9zTB2l+EDmfDAKrMCueLqw\\\\\\\/K5nvFiymIvdnb1Q14iEZ8gfpnrqDAwuqfubPv0epmByR+YJXi+cdANQwfERJKxpJ+g8siV+d8Uh4QASLuXhsj5QKOgvsh5tS3xxzI9mbDCXpnu7RHn+sP0a0474d+edEiqxeSBH7LL4hS7aa5HcpRQMZ1xgmUbi7VVOm0zGjcDRr8QaZGRiwY2am8zLRBH6sUXEHE\\\\\\\/JmekQHE9FMbS6jPhbl932U6EMVW6WMen0gWfLL2Oav+KNWxn1U1hoSJlvufdREnpE42swe5P2ihKKuwsXzTkIyGT7qXFaC\\\\\\\/P6oKbZzDdasBjjWcrMMnnUPvmJgxWKiAgwjUrlK8cs4HL5H0AN8KXfQ6AZ0gGq+vc6DPj4VEwLVKxkia4ETBFqSk4crejM1r2wZ1IX2\\\\\\\/H30QaJxMR6c71eZGlAQivuAFi+zz7Rpjg35lovpfA6d8p\\\\\\\/6j9LQTImA7+EJMfGo9uPuz0cX6NjanKNwhWE6HN\\\\\\\/5FIXpcw78LtXYPsc9JxMm8E++V6uA0Tct+Qahvc2laolwPhC510J11SBuCcHBcwnyefFhd1nxUfbs6UEX0dze1saxp7iQAXccaEB2hiWsacBVFBh4orPZnL+zj41DJ9kMB3xELf77qqRDQ85VqKxCWHzvCy0scqVtuO51Xib5LLPIE8GbWEuI1eu+3sogVWMFC\\\\\\\/MtpJ9vo+L79oolRbiPRqBKCeHEP5j6bsyeIV0Wu7NjYDPG2I+zFfNveXM8gEMPmgbTh4EH0vpkbqEKyeSssmZZmFJT88QPTiNV9PrV1mL4qM9L1hOfH3dz1LYtMCqmrxEMScyaxo6yi9AzLE9DKDsC0P2rCAQEDOcriAqpCR\\\\\\\/oJ+JwFRNQUOgwISlM4kJEhP0frtFpJaf4wm51aI2t978wxDfnSvG9jf0EIskAIyQsLW5KL7G13zx0ly8amAM0f0tDscoH91FRaC5+U+rcDB8umcBJPYtomM2SD9EeWI5Ka6SNeMdzdnputDewlPZYsv4aeuFDmDKLzgq49l0Wz88o6AnwOnHd6uFanAy5pcTBf1suEdjsBEBzWXaLyCuHtL32K5IC2Hab94MgaZTxUAsBdFrGlotI\\\\\\\/1hyxEQhKilFXgDKn6Q2nifJgPt2SA8cDDwQp0LLqQcuMevffJxl5J1YaxXGjKE\\\\\\\/pZMRRAuAgqM9aK0lsz34lKA7PTON7ax87l1xfEklSIkmJoxbV1uas9k1g94fYSceShtv1a9aJHgSfTh2Ry6cZ9HMuHAvshZDg+QP0J0sQjM65VhjBgU3V7Hgi6QqiofIxQPqDKMhw\\\\\\\/jBw070q2UqBSrCVfb1u+cdWZw214Sn6w9cJL6JsAaIoZcCmqpIbArL3EHQQZFKksvVHrskDfSc9QpB4ObNGSdZQeW7wa23+tEH5aXzLJ4CLSqEGGt6gKow2cmM2hzCglJiKj4Trd\\\\\\\/FpKaVRbas9LdJJS6li1az88SaYWhFyfg34P2ULUjR5p1126rOZFuvtAFnCk5NhSTqp\\\\\\\/wdUoFDCoYIV0RYVTG7Iv5qHjYzpXNZsk9S8+t\\\\\\\/LtUM3HsqX6JgzjxlpzhtRlchIWfw5dINN7\\\\\\\/0gJ2NHSiWOmbW2hYhPhhY7g9CXDS0VHkrRumEG0XkZlGuqDPwvct\\\\\\\/FjK7bPJjFXsL9Mrvn4lSij0wP2RqyxEQbnNqNdU1raaoUiqB8husvoWhylZayt6243Nd9dQo4i4FCfloPuLEScqi3W5Y17I1liV+3iMAuyVS6nOPvhL5lKujiGuRRXeh\\\\\\\/Y6lPdb6dZQqfPSa0eg05n2hHCp23uTUlmZf1hmVCxk3SxBgBUZmV4OsZkzIOrhkwp\\\\\\\/l6eP1YH3IV4pMUfmBqPAfc22T49PD4rOgY3uXkWGs7ehDi++NHV1lyaXNMK9ZKxalM0rztMSuuTRkU36xxcUWHS1AKDGCl7RMwHvQZcjur0rUdbWQeyw7\\\\\\\/SA+nbQBBnkHQc79BwHCcD9j3mKwx125hk3CMP1hM3TBD\\\\\\\/giDm+gkRdW\\\\\\\/km2RhMxmNs9Iw9Q+MVCQRhVHKHanQ62puAuNfe3o7vz7iluf6drEYmltSgZhTLUymNE3GG5Q\\\\\\\/K42CjO8qMTnrDsv6uFq638rNJA\\\\\\\/s360hNVCldI4EbTZEAK\\\\\\\/E8I62zUejWnkOv3pDBWlrT3Zd2LmWIzlfRezLa0XcobGdprvkZdpH7d0XBDT51WFkXgdlSOz04Q0fBiZcIb3mPCUVEyHN\\\\\\\/w7fEfdbVbDDNuopSpi3LqVQPlc2XU5yinIDVdGdwk4v3qhXJB7eKFQ5g86KJgdpsH3gOy0\\\\\\\/MErRVi4WwmoK1ot33G9uHL0P4SACLL76Vaahk2EcItKYs9n0ullhbQ7w8Fuw8lYOZgX\\\\\\\/XF\\\\\\\/DQAD5o1DCcK\\\\\\\/C4eyMcH12wqWOdAZ7uEojebPXqFhem5hTr4jX4uqOhymWpr1gf0aTTvPX2tWEzaIVaRIKB9zLH8c2oOYNce9rSyVMTqTf+IE0\\\\\\\/Qgu5P2dqRaO00vte7dipuGPUZBwM62U19sHcbUNMX+Wgbx07He\\\\\\\/5afiB3D\\\\\\\/pcZQOAzfkgQ3J7yEcxri3UBcT9zzSkXYdyrGpOsrdsXpuvBx9yMjOrzDP5Y1sEavJTWXSdNwA6tM8n1QkTJE1b1Gf\\\\\\\/wMFsx6PwsZ1AMVdNH8nQ6Z3BJ9RfLnKtKRNRKACMXeHWO5twGl2LV8IlgTW5aEi0z2WznqotAayh8TLk80qo3hTkkuNppgW3\\\\\\\/XPozz\\\\\\\/5BEj1tZsVg5kwwNUemHuiZ\\\\\\\/IjXmKLyzCDoi91sjGE+JWBi0zGrR50895MULE9Bzexqxp5zesCHP13+eIDwM2RGqzFzpBtFcJ+hN3q4MNCQFEeIFuioSEDw6oI2iTSCyci2TY3pm6YR6o0eSFkvLreenU7RfX6uk\\\\\\\/j5jJUwDONpeV+K+7kkGK\\\\\\\/aDX83m9kVh3f7jpuWuk9rzMGrjxfEV2Gzq+jLwq3csVPWEKaU2ermhwFY1NFbiYIUZ\\\\\\\/2c66lS35Z3WIPeBdP9JmSKV2ZRHJ+OielVEAsg2D57RWmOB5FPUGpyO8iyn73lEsY\\\\\\\/gHwtrmOJKJdcAVMNCertj+kOtsNLiCApBhl89YkB6tB65CUhgIIgjcOXAInedTg\\\\\\\/yCR314Zz7GMqJHx6BDLQaWmz\\\\\\\/j6CcKf2jDjONjSyLvviaZ4\\\\\\\/IGHf3WQYgeZxDFupMXkoTPQZuRFkZEO\\\\\\\/bvOFWIhhxap0j0N9KtLmY18GKHl5HyTX9Zj4hd\\\\\\\/Unt31X6A+64bsitWaOPph2fiM9SYI1F5AWwymyPd1tVcIUUPS\\\\\\\/0NynOxVbiQZKnWQD73kf2jv+dN2CMSMYfJazePFOUAfjd1Jl8GgrO2\\\\\\\/rGV0LqEG0QohW8Qo1+cDzu9Q\\\\\\\/B6Af0\\\\\\\/NPAle5NU9ILDOktZ6ppzUc2HUcJpslpKfb6Ygi9bTBTBMcQ9r9Giqh6SAqA5uJUfZxuK90e+3tAa8z2PRBdWR4XEJq1jxqT0xEqyM7cQoxk1Ni8UtI8Apk6ZjDkJqPfhiKZSNwcBohUJaDbVvwksxWz9G+J7i1dtr8ZSDmi0+l43Y5GIRkltgpPkOdu0CwlVYW6IWNEMueGLchUCKvewNSFWVsgVQa8C8jLW7awuvWqarrpDk1\\\\\\\/0MMBRV6FhgTsDbFfYjiz6yyroBQOW7qupZDcRJ56m63+kEMjfO9JxrxMx0E5ZP5weenUqtd7LRoO5A9CaX1cp2hLFSEtq0vUOuxa\\\\\\\/Nwc1zqbaIkFyIWieHJQojP+oaPWmkt0FNbKzMYUxmRxyQyH84+WQaYHhNOqcw2QLSPvwsA9s1gMg\\\\\\\/3sHls9v5Iv8saFpTSPMI1LU6+2Xyadv66JFvkwO86QXQYcO68Axk5evC+fo5yh3R4nb8aIbjE6IKGcLcv1fbZS8X2473zjo3o6q7Q8XtiyNbMXPMYZGUu56EfpF8s72YmXe6RGUfT3QgC\\\\\\\/4qBXsh8IR7jKKryrIY4nD8FBuNVhgT00iUJI8TWSImp1ZaSbcjS6E7N879Hj86FqphWQyv1awCEa7FH\\\\\\\/vxjb19H5sBM0mlEuJq4fvv8AW6vkCpKb3VrEtYfGoMlUcwldZxZl8nQzp0Qt2vXw48gZWHekUfKNZYOz9TSsBVwcZKakmYeolTcApXFpgKmhw35DWv1AbNVqgmolWw3vnzGAJmEGfvupEpRq2VrWSVb5lw3boUBCRb9ePKQ+BJ710siVc4tpP0o\\\\\\\/M78c4BPKsgqraLYFddDW9ZVL6a+etBlKuIN8NnBL0k5dBTVZbb2JkrgVSU0quGpAdZ9KDUUSEubmL\\\\\\\/FPtXHlDOFexanuNHRv\\\\\\\/uyEAmpduo\\\\\\\/AgIjPRR+0m7QLQGA5P9pz9g3i6DCbPOGk++nT9543z3XoC7PlwJqz\\\\\\\/vdy5krKrTk9pTdkLnzwoJIGfgb\\\\\\\/InKJZl6c+8zfjWW7\\\\\\\/brGf25NJKnDHDpHK+NuoA8NQsRT3FxKYbgrNk92ukAHCJx0fnN\\\\\\\/lCicsnRq9OgikGJohJko0d\\\\\\\/z2A5oum1kyyhebXM4eyL0w0SZdAWAa0Xdr6jjO3fex3lE95ZbfACnUXNU8T+UD6WtEPPvycKxrzns5NuuxbqFW2TgU0p3wqfvPJcBjXvnzg35w9itBbaz0uo1NGkLjzHJtCZk81ZTFuCqEHwFcflxK4PmLluROKvgeJFuIhHqlmeIw0sXFrBpUZSmxWYM6fwK7PxZpWoIj8wr1p9iItegwequwCDZVrra+9js45DFqDYHI9v2tCLTki44MSloRpDEvM7yhn9RZabdxH6VSzqmDmnAo\\\\\\\/Mbt+O0476qMg4a6CFEPNNaiO4\\\\\\\/GESiQf\\\\\\\/nQvlrzL2Ndt03\\\\\\\/d9hQecpMzx0zYYEDEaG5pYQ15fGyiBBkp7LuQM1G7eJ9Mule51PqvFETbszdUG5U0j7TJB6400pNrAsYWnS6KLR8x8MWJbxoBfPNTqg+6\\\\\\\/HWwXTIzZcfmMshJUGgErDqzsPPsYetA3R2XTSHycQ1RhmW5KE4STcCDlJ+ywwLW53G2rm\\\\\\\/KJilMe4O0zr6ukeOLdvC5xUDasKOCuBVK9KZGgIHOFwbx3bNxtTnhcZkHQRca8SNKLO0FQPo6pWbD8KUOE8aMCdlQOYlQIVokJ0Gz0HkLU82+\\\\\\\/Hg91e3fA8P5PnmZElyz1H6wYRpkA7WpEMYMdBe24Zje+gbzFZbVUk+fYFOsmFnsqDs94Mgrxkws2uj6vjVHmuvumXnuZkpfF2yohe70XR0Khj\\\\\\\/tu8qyX0sgJB5PjUNMSqCVT\\\\\\\/BKnt5kpFXBfZ4VXV9hYmSP0sbGi4nvh6F984Z0dIPpmKOIC4WdMGfvcTZyDYJ5BNItlsE9ZMMPlmtpRHM2XegNyeyYmoRMZvLTZsl6hMaQ70TFTKmmlUY8OyrnYtlaI8o9GvcLXg4Fo1NUy9gnMPdKQOO2pMGaIc7dG5Nj3o3a5OQag36Wd\\\\\\\/13H8+s8JKUsZ8wpWTy5sQZX3qOgfZEwHiydWBlzR\\\\\\\/2wCVj0amDqPqD+3FLZGXCuL09arcxgjSVnBRd3jox7g0l+7ZVFFK8Pf0+2lBNGvk\\\\\\\/sEGt9NHtFQeyGgbZ05\\\\\\\/IAZUPJCndWDRa5wegNxXocL7AgC93H9ubXF0KdNdtxsiAKJj64QmhqHJ8lhgoc5n9sfqCFW4B\\\\\\\/gS8HtrHyE+T8IoDvsmncgqb2\\\\\\\/I9Wx6q5PYrY0Dazk+Hvd4wH3Lf0s45ZpKwLl8wgGF5eY6IM3oUxrg7pDyfL+WemiokGLnwX7j6mFUU8nt1wW9ppLe+5BFuuuP+YsSBnXXuXwfypifaXvmYMgO8BUFgSdDoFfWFSPvWfI66iyWgdgDf15gDDMzH308+QA\\\\\\\/GKtbCxZm+VgVkILNklQRRRbqX2P6LkCAo\\\\\\\/8d6GsctNvjRy7IBeubYwMxYatbycHuKeAXRDQKn66p+\\\\\\\/zl25Fjhg+mlBBD\\\\\\\/vHhuQra0WB056ZAJS5FB\\\\\\\/Aew5YFj5o\\\\\\\/sg3CYGclKDgImoWjJCaF55lyv4JPjiEPDKrHW0GrL5+BXOjea5mIa3NCkXgWGhYujX7tEX6v6oGdzZtp2YODtsLCI8YD\\\\\\\/LqELkMhZKugOg5uSs79m24TIpMt+e0FZwHQna9pQpstC5o0xgIIqbt8uUlf3liXgDSwaVa05ckwYtuvIyX+rX\\\\\\\/NqJs1+PzEgvW6MPJd5PiBjIpZy9Jr5jBnHITM95nJIiG2r\\\\\\\/PexC5TUMrga\\\\\\\/caT7rMsgpVFI658YtSHTrCx0vw8VBmi6QHrAVSRn7mV8Es+HhI5+Ivpfy0gnBAf6\\\\\\\/1BiQcZJQanH8QMA5G+vwNOQJhdGSIfCPKAthfm\\\\\\\/Sbrx0QHJ\\\\\\\/6i35cERhk7HLo5\\\\\\\/xZA2UDFVLhu5snq5cG0jwe5aJ5ruS+VvnexsMVasADMkqKpUd+QZM7Wqq4ir1DWnbUTfdxsAq93svsUAgnbXjBvNhty2O8ZfZm+VZ2W3Qex9oP2iJHw9Lfe+NyKz5IuN5HxxPOHRrqi0s0BXpGZxiTPD9\\\\\\\/AHbhyrZxIDjDG6vnt\\\\\\\/lhDHuo8ZDan5g1EW7BDl1hv5CEcJ1rbeZuxdUpd1wh1DILaYuUMMI5UtwlVxgBV1C0nI4vjGPcTUvIiUnifJMC2x8kSg9dr24NceOeZLZk4bwqWSvmS9iQRKMfY014ZvpY8w0h4kfYmxTVJI23IqaA9WZJXGxPIjFBT83oLgbEPI6J3QRx\\\\\\\/dyyoDZga\\\\\\\/iKK9BOoBkjrcHtUCZj3d2658IkF\\\\\\\/DFbz4Y1EwrKRg1t1kG6qrrKcxVhNZTL5PoLaeDekw0W5OcGZBO1AAoHGZtg5jbIpu41J2h9Zij691ENt\\\\\\\/vUzf2a+PL1OkPRBLYwdyg3HNHCXzLE7sc3z3vuuJ4yEwpHn2hV\\\\\\\/UjxrCRMygeFeD1fNCXGl7pzR4gdFRWkjIqJ2nLiyRbLxcn2Es1VCxOpOIOtNRandrex5d+YVuo2e1OLUBt+GCsbHZI9iMFm4P6fHCMHhajE1f9pcGhRH7rzTDR4Tw9BWFj5vSIYJ5lynOrD\\\\\\\/8I4+5Wi6zc3UxYycPDAh9dqrmLpvAljQ9TPX2NZW65SN4804B1S8phelx9v4tjvXDaZby07WopXr9TVZstNv2V2buYXEcrnY+bIP9opPEOaT9aacTQDMqYoemZkd2JruNk8oq2GD9F7bxxCaYtSdd4JDeFiDDk0AbyLMXsh3eT1tXGoFtlwtpMIW41s\\\\\\\/UO6JewISWdSEYHB2YWBC+O3wCgXbGDVURLqJbdkUQoU+wO0JQ0+fIH25vU+\\\\\\\/Sgptp7MAJrb6lwlmhBXBLUDYFW2pznwYRy0mQzC9klzXxWkACMkZlwwHsgwf63zdLdZ08AvCPJHOe42+VZPawT7p+rXGXAH+r6+KpqjVPEWX0yeepwVnNCrNeJiG1NyZTM4\\\\\\\/L45GHT\\\\\\\/7DgRLd6zZCF+UFlw1IDo5IlM34CsFig+utYX3l9FepUUHH9+ULuUYcG1diwRxj6sN4oDviTkUCuxpMmxAIxtmQMB4n5c08Gf6m8hGS3JLwfox7rZM5XYBSlaFe8buw0QYfK0N1f9cCkL2cJQ9AlsrcYNAH6mhpXEJKisAMYHhu2KnqCyik8ajrVHGQRRDmBzCVNya2pnLnzpNUpmbmSrfm3ZG9b7WCUuKFPIBQ3Xb1ioXeoiChVrf5XGF+jyWQRCunB7jdmd9KpO\\\\\\\/JlKmC5UQEy6uM9eqJ+GFWLo4bT4992GY29b7pPrSRSfukl3peQPJJ3fvKFE6eO6P\\\\\\\/Yd2RDr5slooSiKW82QMw21a\\\\\\\/1rOmlAz3WMCTozue8oCFgovKyrMJ47cmd2ZXKG\\\\\\\/W\\\\\\\/dbzRUFuDHIU0NKkNKZ6C7cSnE\\\\\\\/mS1Mmqi4vaOyW\\\\\\\/qeIPcxC5pRGxRROd4ld8UMrn9XAph73dlaCr9qtcdNB\\\\\\\/DqkAqyXuPbJ7GwTEx1X2lGlolAityt17GWWFaTfyCVGsKVuv8jeX3M5Z3TuJ\\\\\\\/CqOoFEhDVeUx0RY7CkXcQHa8Wb3wpbYyJjBPoxBQ4s3m\\\\\\\/rqsF\\\\\\\/MwH5HCTZgnMrOBZR1I+vy9kBAUEVBVa39dpCtbiOCJ3cDkuh7\\\\\\\/xvhnD2PksRIqtrPSwV7Lqtbjj8M3MWsyDZlMYzgF1iqh50jpbKqtGd4i7TbzLR5vT8PNLcMrCujw9gnyTKposAnZAjDQxk9d+FSeKr93HWj\\\\\\\/B2ATuBWQFA94b02iDm8ncGX8fhaw2swZ1W05OnouKhiHdQg0uq7\\\\\\\/38I4HB4LSROOBjbASKP2KhshVOw2gUrAMx4J6SmbMKtRVd+6QkypyaaBahDZBF1btGiq57Yiy\\\\\\\/GKYlN+O3oSMwhIfkIhuy89h88TZr6I3TSuBd6UGZklS+wJZBrFHu67wArBISSypiovMcPg6X5bguwuISmSkBlyXlVwWKe3BkyjibzQ0FnciLahKmlqpJAFaRfyusysek2+8Qb8sR\\\\\\\/P7I0yplH5i3U\\\\\\\/VkB33vjUu6z63pqYXGHa6KsKkxzF6tsA4yIjQzMQBFuPnB16znxjuWKE34uqgd\\\\\\\/ze5fOwA5E9SM9yUGW1\\\\\\\/TsG1A\\\\\\\/YglIW1UM7+80soFT0WHs51Ggcb5xX5nqIEUlg7Wk\\\\\\\/7w+9S1Lcz\\\\\\\/4U3ejS4+ZoGYNvtAy4F6kk5ltsPQ8fit+TVSVWKBp6nDFmK8adP7rfGkZ++juS0W8r9x4wMyjeexiDixyiNSWsCM4V2ybV3Kqj6omvy0bgxx2O1MfYXRY6Xex67p2Ec0PMsCXsX+psmuBMZ1cUM4ms35nxN3Q3wbdPdpYFGB32tlMH\\\\\\\/EHPVXkzXMP8aHkNYWzfbycCMWHEZW56r3opYCsV64Bfb+ky+iX05xHyW85ofS69VtZC2ia5IMcQkUzTHDT3\\\\\\\/A326I1n1ca8aqCdbm954OGESdDulPLZOHofgzJaRAIhCWQbbbXD041p+WqxIPMgncCrV9YthB\\\\\\\/iHvxs6m\\\\\\\/YO+aO\\\\\\\/5XRlT\\\\\\\/vIoGbvb8nkyI4dg2KlOH7aZILX3ZEhv+74d2PumyGTh6E79sWEdAjBmmvVrYQO\\\\\\\/v6ylwAjTewOsYiMJIqIzuv8BAH7oikxV6N0BCAixoEpZD6NJ2PlsZYBCD3BEZlYLZ\\\\\\\/oS+fAj6BQHpfoj5d0XgObRhFKFCGAci2+tQTDKZ10prA0CxOZBd6RoFkugcfs1fYLCQghepQJxJvsrjOl+T6RunUIviegEKu1luZ+X1kOz1KyoVMvSDIMm4a7L7HhwSSuzSviSF6elsA3tldaKh3+9CVSLAQdmTdESd6Cno9Bd\\\\\\\/M9lYk3\\\\\\\/8B6ME5++e+wmAAhs64oCaVFoEAY8nKf4zE+pZk3gIHEyLCz9yK+je8mV+0fblnpRYvsmjWL7zzVdWEYe\\\\\\\/qogFTmiI\\\\\\\/+2soPFBd+IqOzTORyYCnvAgF6uFNaKiiG9pE5o+q+5sBm1XPdjy7aLILT6yUnfWJcYx2wYFIZC1l7JBiLQlmKE1aimuOFfzWONfzqny3xJtLxO80I0gUOuot78gB\\\u0022,\\\u0022iv\\\u0022:\\\u0022Zfl4Md\\\\\\\/qG\\\\\\\/JUCGXa4IVp0g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d483111fa5aa7bf13bd9d32ded99b956df4ec2adf02f19d6ddde34cf67772843\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00221LHA\\\\\\\/Yz0HkdvuYjkADz5rKGtXyyVBg2gwKwx\\\\\\\/7U66jFtn5wsctfcxVQ8d+MhrGuyHU1uyXGw2OR3swK1cLg6ztVwW3lUjA\\\\\\\/ffBJRfIk94kyTHEX7y9fcAeePo3Njyu1G\\\\\\\/aaGdG1epUiFfTLaN+3x6NWEwfGZ65qQKuT8oVZ3xnME+On\\\\\\\/2pL240lhGPPBU0nrTLZUzJ8p7frGyFbnNrOL\\\\\\\/P7NXPxn6QX\\\\\\\/GKAy7qOBZAPXiLuf+NHAUUlcRpiqLjvJOoUj+lWYrSho\\\\\\\/JQSvxHUK2GAtMCjFZkooZYU3N9WJomC8nwiGCf\\\\\\\/KXBOc9kr2irRCiBxizqdn+rL\\\\\\\/IMA33kEl70+Z8w3YBiV6vDQyQMtv3Om7WDH3C2KGR18dBN0gNxZCKtCMy2SKjzTkOvTS9nOz4bELLm7QbymK8+pBX4nNKslyZAMzmXKB8\\\\\\\/R58EqHyDabOYZVOspfeHMQWSomnG2Ydz14W1J\\\\\\\/h3AFTP44Ksxgjtu1q3sB\\\\\\\/\\\\\\\/GvqXDx6B5hpeFFYt2bWQi0SewfYeAcR3YwOi9DriBA8O6ZU2rbATEEWGQXYm7uoQa0LLKmn7+Ts++9oRhqwJIWT2xPgxX+QQLnS6fAknE8HyquG0s1IjnoIh1\\\\\\\/ZonK2xLJyudP0Qb+hd\\\\\\\/1GlkJHzh31U\\\\\\\/VOkja4RqDOLxB5b9Ae+c4bGpeOlDeC5t+6QAVhEanTnJ47hiR0EbBfLD04ITK+bnMAJe+kgN180qZccHiy0IUXWuPAV4dQbYxknwhelu4y\\\\\\\/7z7RuCxHK9cRlk7FJPH\\\\\\\/CKo8H4nt10H52XKORtSIu04W1EOuLSd2mOJBf89kw8aC609y\\\\\\\/4rNAThF5SPZvn3IxjWT3Qn9IYG2CSAG1WPeT\\\\\\\/Fq4jrAHwc6iy8VyeaHazeopQkaMHZGYB5lVuuzlvYrtaBVKPF2KWfnjNrwFIbsyqgs3vig6Jwoqjhs4RxTB3yvQmlauz7upNNUuLp5kPCWvoCsoBDMV70CjUJqSPj12tSrAzrjawurQkS6vzKQ2PID3x1sE79\\\\\\\/F043y4DIL4yq4FQalxOgaLFFWOMaMACuK78rfH\\\\\\\/HFk3FmAnbEUP04u3FYztSDfvb7OFyCEvXxZly+HK44aDAuBIqaM8KMNZtGjjpdTahbDVOjVXf\\\\\\\/Bb8p0GGG+9GIVqavdB5m1UG0sqeLfM96\\\\\\\/2pPDGCKRpeXbcCVFjWoxbVKlMZNJjgpJwOKr4HT3ebT9xkjibSH6YZYpYKwGHF4ELUaw\\\\\\\/7siQXyuOom\\\\\\\/qQSSn5DWsACmdN7hGMJDjMPMjD2mKzyGDDo0kb6o80XK9CnNKSJ6OpE6TSs0TO1GBw\\\\\\\/8\\\\\\\/Im1v1LuMZ\\\\\\\/JWfK1Yrxxq\\\\\\\/KOyVwCciuzvY5eYKoTeFOK\\\\\\\/My\\\\\\\/prGvWiJwn\\\\\\\/S0psBsazsRa1WFFwklcjOhoCjhDgv2pszdcd2sf1R8O20E1+\\\\\\\/FR+qn+UrNfqcD3AllSzMQbVAIabkmUnSh0jRHLzgGLqGp6OlVS3Z6uEYUZIwqgcKoJh4X+Iu5WeJgxoV0xgo1B1Jvf4trW58RPnyYeJr2w1hyv4lNBIrYneleSp+IHbgUTfivo3VwYh8F9nlm78fCaAO3e4q1drh+nkx37WPwWFPeAAxqDFisAAq7p8+b096Z44bw3Xoi2KGcVBD2La6QaTBLAgSZNpnXxwxATBUgbMhaJn5wXM+X5kkLtUNbZEvejVGnvyeFKWvqr2onkwfM974Eb72KXbBTVgjmhUe2nURvOnypcxU1gWv5fM4PQLP7n2mxRh75HI13m3piyuxdt1F0807OrYQR2ZpCl4nHQ7cBJ1booR1jziLHiXbbv6z\\\\\\\/rEWJtp15FodwZKjwdFnZ+mvUTDhdpUjXt74vEpnsqvM+Fd04XogaIlxSGDYlERpXSSS5BYGAP4mcVNwgNUXSPY09sSXWJdu2dZY\\\\\\\/ijKbMbXqqp+NGxzBJr1oCsxqz5aaespHTczwzbzR9AztFPLJk0aA1xiDSZl6HvPIlD+mPsS8LrRmoFjpLTzFLzn6EGdq2IE3i1X\\\\\\\/UaQ4vFt5UA1acFXAvU=\\\u0022,\\\u0022iv\\\u0022:\\\u0022qSpHiUBr4ytJ\\\\\\\/qbPahAX4w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222c221180b40aa88c4661b1c2e16798050fcc48c908b477d45f4186ca4b49a8e6\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460126 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/993" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vaHMHvDwR6BJcaDcXHORaiFkTth7wfTubXEIrWE1XEE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/993" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-gqd7IfBMeDv2jY73VkjdMbMkENpXnB3t4f7hpgP7Aqs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/4/987" title="A HAND2 Loss-of-Function Mutation Causes Familial Ventricular Septal Defect and Pulmonary Stenosis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/4/1013" title="Fluorescence In Situ Hybridization (FISH)-Based Karyotyping Reveals Rapid Evolution of Centromeric and Subtelomeric Repeats in Common Bean (Phaseolus vulgaris) and Relatives" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460126" id="node460126" data-pisa="ggg;6/4/993" data-pisa-master="ggg;g3.116.027904" data-apath="/ggg/6/4/993.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">High Throughput Analyses of Budding Yeast ARSs Reveal New DNA Elements Capable of Conferring Centromere-Independent Plasmid Propagation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Timothy Hoggard</span>, <span class="highwire-citation-author" data-delta="1">Ivan Liachko</span>, <span class="highwire-citation-author" data-delta="2">Cassaundra Burt</span>, <span class="highwire-citation-author" data-delta="3">Troy Meikle</span>, <span class="highwire-citation-author" data-delta="4">Katherine Jiang</span>, <span class="highwire-citation-author" data-delta="5">Gheorghe Craciun</span>, <span class="highwire-citation-author" data-delta="6">Maitreya J. Dunham</span> and <span class="highwire-citation-author" data-delta="7">Catherine A. Fox</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">993-1012;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.027904 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460126" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Timothy Hoggard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biomolecular Chemistry, School of Medicine and Public Health, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BHoggard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hoggard%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATimothy%2BHoggard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ivan Liachko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genome Sciences, University of Washington, Seattle, Washington 98105</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ivan%2BLiachko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liachko%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIvan%2BLiachko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Cassaundra Burt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biomolecular Chemistry, School of Medicine and Public Health, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cassaundra%2BBurt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burt%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACassaundra%2BBurt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Troy Meikle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biomolecular Chemistry, School of Medicine and Public Health, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Troy%2BMeikle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meikle%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATroy%2BMeikle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Katherine Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biomolecular Chemistry, School of Medicine and Public Health, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katherine%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatherine%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gheorghe Craciun </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biomolecular Chemistry, School of Medicine and Public Health, University of Wisconsin, Madison, Wisconsin 53706</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Mathematics, College of Letters and Science, University of Wisconsin, Madison Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gheorghe%2BCraciun%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Craciun%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGheorghe%2BCraciun%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Maitreya J. Dunham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genome Sciences, University of Washington, Seattle, Washington 98105</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maitreya%2BJ.%2BDunham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunham%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaitreya%2BJ.%2BDunham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Catherine A. Fox </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biomolecular Chemistry, School of Medicine and Public Health, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Catherine%2BA.%2BFox%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fox%20CA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACatherine%2BA.%2BFox%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cfox@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">cfox@wisc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/4/993" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460126" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460126/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/993.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460126" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460126/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/993.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460126" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460126/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/993.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460126" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460126/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/4/993.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460126" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460126/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">The ability of plasmids to propagate in <em>Saccharomyces cerevisiae</em> has been instrumental in defining eukaryotic chromosomal control elements. Stable propagation demands both plasmid replication, which requires a chromosomal replication origin (<em>i.e.</em>, an ARS), and plasmid distribution to dividing cells, which requires either a chromosomal centromere for segregation or a plasmid-partitioning element. While our knowledge of yeast ARSs and centromeres is relatively advanced, we know less about chromosomal regions that can function as plasmid partitioning elements. The <a href="http://www.yeastgenome.org/locus/S000005160/overview">Rap1</a> protein-binding site (<a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a>) present in transcriptional silencers and telomeres of budding yeast is a known plasmid-partitioning element that functions to anchor a plasmid to the inner nuclear membrane (INM), which in turn facilitates plasmid distribution to daughter cells. This <a href="http://www.yeastgenome.org/locus/S000005160/overview">Rap1</a>-dependent INM-anchoring also has an important chromosomal role in higher-order chromosomal structures that enhance transcriptional silencing and telomere stability. Thus, plasmid partitioning can reflect fundamental features of chromosome structure and biology, yet a systematic screen for plasmid partitioning elements has not been reported. Here, we couple deep sequencing with competitive growth experiments of a plasmid library containing thousands of short ARS fragments to identify new plasmid partitioning elements. Competitive growth experiments were performed with libraries that differed only in terms of the presence or absence of a centromere. Comparisons of the behavior of ARS fragments in the two experiments allowed us to identify sequences that were likely to drive plasmid partitioning. In addition to the silencer <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site, we identified 74 new putative plasmid-partitioning motifs predicted to act as binding sites for DNA binding proteins enriched for roles in negative regulation of gene expression and G2/M-phase associated biology. These data expand our knowledge of chromosomal elements that may function in plasmid partitioning and suggest underlying biological roles shared by such elements.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/silencers" class="hw-term hw-article-keyword hw-article-keyword-silencers" rel="nofollow">silencers</a></li><li class="kwd"><a href="/keyword/plasmid-partitioning" class="hw-term hw-article-keyword hw-article-keyword-plasmid-partitioning" rel="nofollow">plasmid partitioning</a></li><li class="kwd"><a href="/keyword/dna-replication-origins" class="hw-term hw-article-keyword hw-article-keyword-dna-replication-origins" rel="nofollow">DNA replication origins</a></li><li class="kwd"><a href="/keyword/deep-mutational-scanning" class="hw-term hw-article-keyword hw-article-keyword-deep-mutational-scanning" rel="nofollow">deep mutational scanning</a></li></ul><p id="p-4">Eukaryotic chromosomal DNA encodes the information necessary for establishing and maintaining proper chromosome structure and expression within the nucleus, as well as chromosome duplication and distribution to daughter cells during cell division. However, our understanding of the DNA sequences that ensure appropriate chromosome structure and inheritance is incomplete. In the model eukaryotic microbe <em>Saccharomyces cerevisiae</em>, episomal plasmids are powerful tools for defining DNA sequences that govern chromosomal functions. Notably, plasmid-based assays have helped identify and characterize <em>S. cerevisiae</em> DNA replication origins and centromeres, the DNA regions essential for chromosome duplication, and segregation during cell division, respectively (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Newlon and Theis 1993</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bloom 2014</a>)</p><p id="p-5">Plasmid-based assays of telomeres and transcriptional silencers, DNA sequences that direct the formation of silent chromatin, the budding yeast version of heterochromatin, have defined a DNA element, the <a href="http://www.yeastgenome.org/locus/S000005160/overview">Rap1</a> protein-binding site (<a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site), capable of linking chromosomal regions to the inner nuclear membrane (INM) via its interactions with the silencing protein <a href="http://www.yeastgenome.org/locus/S000002635/overview">Sir4</a> (Kimmerly and Rine 1987; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kimmerly <em>et al.</em> 1988</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Longtine <em>et al.</em> 1992</a>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">1993</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Ansari and Gartenberg 1997</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Andrulis <em>et al.</em> 2002</a>). In the chromosomal context, this juxtaposition to the INM enhances the stability of silent chromatin required for yeast mating-type identity, as well as the stability and structure of telomeres (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gotta <em>et al.</em> 1996</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Andrulis <em>et al.</em> 1998</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Nagai <em>et al.</em> 2010</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Taddei and Gasser 2012</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Towbin <em>et al.</em> 2013</a>). On plasmids, this mechanism promotes robust centromere-independent plasmid distribution, or partitioning, to daughter cells during cell division. Thus plasmid-based studies provided the first clues about the chromosomal architectural function of <a href="http://www.yeastgenome.org/locus/S000005160/overview">Rap1</a>. Despite this evidence that plasmid-partitioning elements have the potential to reveal fundamental features of chromosome biology, our knowledge about such elements in the genome is extremely limited due to the lack of a facile systematic assay for identifying them. In this study, we present a high throughput plasmid-based approach that exploits a recently developed suite of methods for high throughput mapping of yeast DNA replication origins (ARSs) to identify dozens of putative plasmid partitioning elements in the yeast genome (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Liachko <em>et al.</em> 2013</a>).</p><p id="p-6">Plasmid replication requires that the plasmid contains a DNA replication origin, which is a specific chromosomal region where DNA replication initiates. The yeast genome contains ∼400 confirmed origins, and each of these contains a specific DNA binding site for the origin recognition complex (ORC), the protein complex that selects origins in all eukaryotes (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Siow <em>et al.</em> 2012</a>). A yeast origin can confer replication to plasmids, and this ability led to the plasmid-based ARS (Autonomously Replicating Sequence) assay (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Newlon and Theis 1993</a>). Thus, yeast chromosomal origins are named ARSs. In combination with traditional molecular cloning-based mutagenesis, the ARS assay has allowed the core DNA sequence elements required for origin function and their relative organization to be defined for several origins (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Marahrens and Stillman 1992</a>; <a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Newlon and Theis 1993</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Huang and Kowalski 1993</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">1996</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Rao <em>et al.</em> 1994</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Theis and Newlon 1994</a>, <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">1997</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Crampton <em>et al.</em> 2008</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chang <em>et al.</em> 2008</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">2011</a>). A classic study used comprehensive linker-scanning mutagenesis to dissect a ∼150 bp region containing the <em><a href="http://www.yeastgenome.org/locus/S000029652/overview">ARS1</a></em> (a.k.a. <em><a href="http://www.yeastgenome.org/locus/S000029652/overview">ARS416</a></em>) origin and revealed four distinct modular elements, a conserved A-element and poorly conserved B-elements, B1, B2, and B3, and helped establish <em><a href="http://www.yeastgenome.org/locus/S000029652/overview">ARS1</a></em> as the paradigm ARS used in many subsequent biochemical studies (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Marahrens and Stillman 1992</a>). Biochemical studies have established that the A and B1 elements together comprise the ORC binding site (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bell and Stillman 1992</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Rao and Stillman 1995</a>). The B2 element helps load the eukaryotic replicative helicase, while the B3 element binds a transcription factor that excludes nucleosomes from <em><a href="http://www.yeastgenome.org/locus/S000029652/overview">ARS1</a></em> (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Simpson 1990</a>; <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Marahrens and Stillman 1992</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lipford and Bell 2001</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Wilmes and Bell 2002</a>). The conclusions about <em><a href="http://www.yeastgenome.org/locus/S000029652/overview">ARS1</a></em> elements have been recapitulated in studies of chromosomal origin function (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Marahrens and Stillman 1994</a>). However, because of the laborious nature of the ARS assay and traditional mutagenesis methods, less than a dozen of the ∼400 confirmed yeast origins have been characterized in any detail. Thus, it remains unclear how representative an example <em><a href="http://www.yeastgenome.org/locus/S000029652/overview">ARS1</a></em> is for yeast origins. This issue is important because individual origins differ considerably from one another in terms of their functional behavior in S-phase, as well as their sensitivity to transcription interference and dependence on origin binding factors or chromatin regulators (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Raghuraman <em>et al.</em> 2001</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Nieduszynski <em>et al.</em> 2005</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Feng <em>et al.</em> 2006</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Donato <em>et al.</em> 2006</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Knott <em>et al.</em> 2009</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">2012</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Muller <em>et al.</em> 2010</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Pohl <em>et al.</em> 2012</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Muller and Nieduszynski 2012</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hoggard <em>et al.</em> 2013</a>).</p><p id="p-7">Recently, a suite of high throughput methods was developed to map and delineate ARSs on a genome-wide scale (“ARSseq”) and to perform deep-scanning mutagenesis of ARSs, starting with <em><a href="http://www.yeastgenome.org/locus/S000029652/overview">ARS1</a></em> (“mut-ARSseq”) (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Liachko <em>et al.</em> 2013</a>). Specifically, a fragment of <em><a href="http://www.yeastgenome.org/locus/S000029652/overview">ARS1</a></em> was subjected to saturation mutagenesis, cloned into a plasmid without a native ARS, and the resulting library was used to transform yeast. The yeast cells were grown competitively in liquid culture, such that the <em><a href="http://www.yeastgenome.org/locus/S000029652/overview">ARS1</a></em> variants better at propagating in the population over time were enriched relative to other fragments, while weaker fragments were depleted. Deep sequencing of the entire population of plasmids during the competition revealed the changing abundance of the plasmids; multiple individual nucleotides relevant for <em><a href="http://www.yeastgenome.org/locus/S000029652/overview">ARS1</a></em> function were identified in a single experiment. <em><a href="http://www.yeastgenome.org/locus/S000029652/overview">ARS1</a></em> provided a convenient platform for the initial development of this method because its core sequence had already been mapped to a short fragment suitable for deep-scanning mutagenesis. To map small core sequences for many more ARSs, we developed a second method (“miniARSseq”). Briefly, sheared DNA from a comprehensive library of functional ARSs was subcloned and screened for ARS function by a similar library sequencing method as described above. This experiment delineated the A element and flanking sequences important for propagation in yeast for &gt; 100 ARSs, providing the raw material for further dissection of these elements.</p><p id="p-8">While a plasmid’s propagation in this assay will vary with how efficiently the plasmid replicates, plasmid distribution between mother and daughter cells during yeast cell division might also be expected to contribute to variant success in this assay. Partitioning of plasmids to daughter cells is aided by “active” distribution mechanisms because the geometric features of the dividing nucleus and the rapidity of mitosis favor retention of plasmids by the mother cell (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gehlen <em>et al.</em> 2011</a>). In addition to this morpho-kinetic barrier to plasmid diffusion, an additional SAGA-mediated tethering of DNAs to the nuclear pore complex also promotes retention of centromere-lacking plasmids by the mother cell (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Denoth-Lippuner <em>et al.</em> 2014</a>). Thus, to focus on elements critical to origin function and bypass these mechanisms that favor plasmid retention in the mother cell, the traditional ARS assay uses plasmids that include a centromere. However, centromere-independent mechanisms can also contribute to efficient plasmid distribution, including centromere-like elements (CLEs) (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lefrançois <em>et al.</em> 2013</a>). And, as mentioned above, transcriptional silencers, DNA elements that direct the assembly of silenced chromatin at the cryptic mating-type loci <em><a href="http://www.yeastgenome.org/locus/S000029655/overview">HMR</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000029214/overview">HML</a></em>, contain ARS elements closely associated with a plasmid partitioning function mediated by the silencer <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site. These and other studies provide strong evidence that an association between plasmids and the INM is a mechanism for enhancing centromere-independent plasmid distribution during cell division, but additional mechanisms may exist that reflect distinct features of chromosome biology, including mechanisms that promote intra- and interchromosome interactions (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Gehlen <em>et al.</em> 2011</a>). The hypothesis, supported by both mathematical modeling and direct experiments, is that these types of interactions help a plasmid avoid the blockade of segregating chromosomes within the dividing nucleus at the bud neck, either by traveling around it (<em>e.g.</em>, plasmid interactions with the INM) or by “hitching a ride” on a chromosome (<em>e.g.</em>, plasmid-chromosome interactions) (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Gehlen <em>et al.</em> 2011</a>). Obviously, these basic mechanisms, and possibly others as yet undefined, reflect highly relevant aspects of chromosome architecture. However, because no genome-scale examination of partitioning elements has been reported, the breadth of such potential functions across the genome is unknown.</p><p id="p-9">In this study, we adapted the ARS assay to examine both replication and potential partitioning elements in budding yeast at a genomic scale (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Liachko <em>et al.</em> 2013</a>). The miniARS library is comprised of thousands of distinct 100–200 bp chromosomal fragments (<em>i.e.</em>, miniARSs), representing over 100 yeast origins. While the initial study assayed the ability to transform yeast (<em>i.e.</em>, minimal ARS function), here we generated quantitative information about plasmid propagation efficiency by subjecting the pool of transformed yeast to a competitive growth experiment coupled to deep sequencing. The data generated by this experiment defined the chromosomal regions required for maximal plasmid propagation for 112 individual ARSs in a single experiment. They also allowed us to compare the efficiency between different ARSs. Strikingly, the <em><a href="http://www.yeastgenome.org/locus/S000029655/overview">HMR</a></em>-E (mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em>) and <em><a href="http://www.yeastgenome.org/locus/S000029214/overview">HML</a></em>-E (mini<em><a href="http://www.yeastgenome.org/locus/S000028477/overview">ARS301</a></em>) transcriptional silencers were among the most effective ARSs in the library. We performed deep mutational scanning of mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em> and miniA<em>RS301</em>, direct ARS assays and numerical simulations that all supported the conclusion that the <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site-mediated partitioning intrinsic to these ARSs was important for their high efficiency in the miniARS experiment. Significantly, additional ARS assays of several nonsilencer origins revealed previously undefined putative plasmid-partitioning elements. To define such elements more precisely and address their relative frequency among yeast origins, we transferred the miniARS library to a centromere-containing plasmid and performed a new competition experiment. Computational approaches to compare the data generated from the two different miniARS experiments allowed the parsing of the replication and putative partitioning functions on multiple ARS loci. A substantial fraction of the origins represented in these miniARS libraries, ∼25%, contained regions important for plasmid propagation in the absence of a centromere. However, in the presence of a centromere, these regions are dispensable for efficient plasmid propagation. We propose that these regions enhance plasmid partitioning to the daughter cell and therefore called them “putative partitioning regions.” Motif analyses identified 74 new putative partitioning elements within these regions in 26 nonsilencer origins out of a total of 112 origins examined. Many of these motifs were predicted to bind proteins involved in negative regulation of a number of cellular processes, including filamentous growth and gene expression, suggesting underlying linkages among chromosomal elements capable of substituting for centromere function on plasmids.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Cloning of selected fragments into Acen- and Cen-plasmids and ARS assays</h3><p id="p-10">Selected fragments showing either high or low competitive fitness in the Acen-miniARS competition experiment (as diagrammed in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>) were subcloned into comparable plasmids either containing or lacking a centromere [pIL13 (pCF2962) and pIL22 (pCF2963), respectively], as indicated in the figures. Three independent transformed yeast colonies for each plasmid construct were assessed using standard ARS assay methods (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Chang <em>et al.</em> 2011</a>), except that instead of using agar medium in petri plates for colony growth, the wells of 96-well plates (Falcon Tissue Culture Plate, #353705) were used. This method was an adaption of “tadpoling” (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Welch and Koshland 2013</a>). For each biological replicate, at least three independent ARS assays were performed. All replicates were combined to generate the data (PLR data presented as mean of all replicates ± standard error) shown in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>.</p></div><div id="sec-3" class="subsection"><h3>Construction of the Acen- and Cen-miniARS libraries</h3><p id="p-11">The generation of the Acen-miniARS library was presented previously (<a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Liachko <em>et al.</em> 2013</a>). Briefly, a collection of 1–∼2 kb ARS fragments recovered from a yeast ARS library representing 75% of confirmed ARSs were further digested with DNaseI and size-selected for ∼100–200 bp fragments by gel purification. These fragments were subsequently cloned into a pRS406-derived vector (pIL22). To generate a comparable Cen-miniARS library, the original Acen-miniARS library extracted from bacteria was used as a template in multiple high fidelity 15-cycle PCR using primers oCF6560 and oCF6561, which flank the insert-cloning site in pIL22. The PCR fragments were purified, pooled, and cloned into a pRS406-derived vector containing a centromere (pIL13) using the Gibson Assembly method (Clontech In-Fusion HD Cloning Kit #639649). The derivative Cen-plasmid library DNA was used to transform bacteria. For library DNA preparation that was used to transform yeast, &gt;2 × 10<sup>5</sup> <em>Escherichia coli</em> clones were directly scraped from agar plates. The Cen-miniARS and the original Acen-miniARS libraries shared 2295 distinct fragments representing 112 origins with high confidence ORC binding sites (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Eaton <em>et al.</em> 2010</a>), out of a total of 2779 fragments representing 119 origins, indicating efficient transfer of the original library into a centromere-containing plasmid.</p></div><div id="sec-4" class="subsection"><h3>MiniARS and miniARSmut competition experiments</h3><p id="p-12">The appropriate library DNA purified from <em>E. coli</em> was transformed into yeast (MATa W303) and individual yeast transformants were pooled to generate the initial miniARS library for a competitive growth experiment. All competitions were performed in shaking liquid cultures at 30° in growth medium lacking uracil, such that the uracil selectable marker on the plasmid was demanded throughout the experiment. Approximately 10 OD of cells were used to extract DNA at the start of the competition and at set time intervals throughout the competition. The DNA was extracted and purified and used as a source of total plasmid DNA for deep sequencing as described (<a id="xref-ref-36-5" class="xref-bibr" href="#ref-36">Liachko <em>et al.</em> 2013</a>). For the Acen-miniARS competition, the competitive fitness of individual fragments were determined by calculating the slope that described their change in frequency within the population using 0, 3, 6, 9, 12, 15, and 25-hr time points. The Cen-miniARS library was similarly competed, except the final time point was 30 hr.</p><p id="p-13">The mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em>-mut and mini<em><a href="http://www.yeastgenome.org/locus/S000028477/overview">ARS301</a></em>-mut libraries were constructed using synthetic oligos from TriLink BioTechnologies. Randomly mutagenized oligos spanning mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em>-full (153 bp) or mini<em><a href="http://www.yeastgenome.org/locus/S000028477/overview">ARS301</a></em>-full (135 bp) were synthesized with a mutation frequency of 1.2% and 1.8% at each position for mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em>-mut and mini<em><a href="http://www.yeastgenome.org/locus/S000028477/overview">ARS301</a></em>-mut, respectively. The mutagenized oligos contained invariant 5′ and 3′ elements for subsequent primer annealing and PCR amplification. PCR products were cloned using Gibson assembly into the Acen-vector pIL22, and <em>E. coli</em> clones were pooled and used to prepare the miniARSmut libraries. The plasmid libraries were transformed into W303 yeast, and yeast clones were pooled to generate the starting miniARSmut library in yeast. Each miniARS-mut library was grown competitively in liquid batch culture as described for the Acen- and Cen-miniARS libraries above. For these ARSmut experiments, competitive fitness was defined as the ratio of a given fragments’ frequency at the end of the competition to its frequency at the start of the competition.</p><p id="p-14">The plasmid DNA extracted from the yeast population were subjected to analyses by deep sequencing as described (<a id="xref-ref-36-6" class="xref-bibr" href="#ref-36">Liachko <em>et al.</em> 2013</a>). Sequencing data for this project are available at the NCBI Sequence Read Archive BioProject Accession #SRP065331. Fragment coordinates were compared to chromosomal coordinates (sacCer1) of confirmed yeast DNA replication origins annotated in the yeast Origin Database (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Siow <em>et al.</em> 2012</a>; <a href="http://cerevisiae.oridb.org">http://cerevisiae.oridb.org</a>). For all analyses in this study, only fragments from confirmed yeast origins that contained a high confidence ORC binding site (<em>i.e.</em>, an ORCACS) annotated in a previous study were included (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Eaton <em>et al.</em>, 2010</a>). The first nucleotide of the T-rich strand of the ORC binding site corresponded to the first nucleotide of the Extended ACS (EACS) consensus as previously defined (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Hoggard <em>et al.</em> 2013</a>).</p></div><div id="sec-5" class="subsection"><h3>Numerical modeling</h3><p id="p-15">Numerical simulations were done using MATLAB (MATLAB 8.3, The MathWorks Inc., Natick, MA, 2014). The code is provided as a text file in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/FileS1.pdf">File S1</a>. To account for natural variations among the cells in a population, we introduced Gaussian noise for the two main parameters in the model, replication fitness, and partitioning fitness. The results of the simulations are robust with respect to different noise levels.</p></div><div id="sec-6" class="subsection"><h3>Computational analyses</h3><p id="p-16">A consensus <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site was derived in Weblogo from the RAP1 Position Specific Frequency Matrix (PFSM). A consensus sequence for the ORC site was created in Weblogo using the sequences of the 232 ORC sites annotated previously (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Eaton <em>et al.</em> 2010</a>). For the motif scanning of putative partitioning ARSs, 50 nucleotide regions, tiled every 25 nucleotides, from −50 to +175 (skipping the ORC site itself) were analyzed for matches to annotated motifs (JASPAR CORE 2014 fungi) using the Tomtom motif comparison tool in the MEME suite (<a href="http://meme-suite.org">http://meme-suite.org</a>) with a P-value cut-off of 0.05. MiniARS fragments were mapped, aligned and analyzed using standard functions in LibreOffice Calc spreadsheets. All data except for numerical simulations were graphed in R. Supplemental Material <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/TableS1.xlsx">Table S1</a> contains the complete list of motifs identified within the 26 partitioning ARSs (MacIsaac <em>et al</em> BMC Bioinformatics 2006) <a href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-7-113">http://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-7-113</a>. Genotype and Phenotype Ontology: A Gene Ontology term finder algorithm was performed in the <em>Saccharomyces</em> Genome Database (SGD, <a href="http://www.yeastgenome.org">http://www.yeastgenome.org</a>) using GO process terms, demanding a P-value cutoff of 0.01. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/TableS2.xlsx">Table S2</a> includes the complete GO analyses. 84 Phenotype Ontologies (PO), lists of genes associated with a particular phenotype, were downloaded from SGD. The fraction of genes identified by various motif collections that belonged to the PO terms was determined and normalized to the fraction of sequence specific DNA binders (GO: 0043565). <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/TableS3.xlsx">Table S3</a> includes the complete PO analyses.</p></div><div id="sec-7" class="subsection"><h3>Data and reagent availability</h3><p id="p-17">Plasmids are available upon request. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/TableS4.xlsx">Table S4</a> contains the names and sequences of all plasmids. Sequencing data for this project are available at the NCBI Sequence Read Archive BioProject Accession #SRP065331.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>The HMR-E and HML-E transcriptional silencers generated many of the most competitive miniARS fragments</h3><p id="p-18">Yeast transformed with the previously described miniARS library were grown competitively under conditions that selected for the <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em> gene on the plasmid (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>) (<a id="xref-ref-36-7" class="xref-bibr" href="#ref-36">Liachko <em>et al.</em> 2013</a>). Deep sequencing of the miniARSs within the cell population at each time point revealed the composition of the miniARS library over the time course of competitive growth. Sequencing allowed the change in frequency of each individual miniARS during the competition to be determined. Each miniARS fragment could be assigned a competitive fitness value, defined as the slope of the regression line plotted through the changing fragment frequency in the population over time. Fragments enriched over time were assigned high (+) competitive fitness values, while fragments that were depleted were assigned low (−) competitive fitness values.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/993/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The ARSs associated with the HMR-E and HML-E transcriptional silencers generated many of the most competitive miniARS fragments in the miniARS library. (A) A pool of transformed yeast was grown competitively under conditions that selected for the plasmid. Deep sequencing measured the frequency of the miniARS fragments within the population throughout the course of the experiment. In this simple example, miniARS fragment #5 is enriched during the course of the experiment and is assigned a positive competitive fitness value. In contrast, miniARS fragment #2 is depleted during the course of the experiment and is assigned a negative competitive fitness value. If these fragments were from a single ARS (autonomously replicating sequence), ranking them based on their competitive fitness values would help indicate the minimal chromosomal region necessary for maximizing the function of this ARS. In addition, competitive fitness between origins can be compared to allow identification of the most competitive ARSs in the population. (B) The slope of the log2 ratios of the frequency of each miniARS fragment in the yeast cell population through the time course was used to measure each fragment&#x2019;s competitive fitness (CF). This value is plotted against the length of the fragment in bp (x-axis). A gray circle designates each fragment that was assessed in this miniARS experiment. In the left panel, the HMR-E silencer-associated fragments (miniARS317) present in the experiment are filled in black, while in the right panel the HML-E silencer-associated fragments (miniARS301) are filled in black. (C) All of the miniARS317-fragments (n = 70, left panel) and all of the miniARS301-fragments (n = 18, right panel) were ranked based on their CF values and then divided into three distinct bins based on this value. These bins were then ranked based on their average competitive fitness (CF) values from lowest (+) to highest CF (+++). To better visualize nucleotides that were enriched in the most competitive miniARS for each silencer, an arbitrary fragment was selected encompassing each silencer and numbered relative to the first nucleotide of the T-rich strand of the ORC (origin recognition complex) binding site that was given the value &#x201C;0.&#x201D; Each bin was color coded as indicated to visualize the fraction of fragments within that bin that contained a given nucleotide. Thus, the most competitive bin of miniARS317 fragments indicated that greater than 80% of the fragments contained intact RAP1 (Rap1 protein binding site), ORC (origin recognition complex), and FKH (forkhead) sites. The most competitive miniARS silencers (miniARS317-full and miniARS301-full) used for deep mutational scanning in Figure 2 are indicated below the most competitive tercile with a thick black line. Their numbering is relative to the ORC binding site, as above. The FKH*, YHP1* (Yeast Homeo-Protein), and SPT2 (SuPpressor of Ty&#x2019;s)* sites are starred to indicate that their identification is based solely on a strong motif match. In contrast, the silencer ABF1 (ARS-Binding Factor 1), RAP1, and ORC sites have been verified as such by numerous genetic and biochemical experiments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-865468913" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The ARSs associated with the &lt;em&gt;HMR&lt;/em&gt;-E and &lt;em&gt;HML&lt;/em&gt;-E transcriptional silencers generated many of the most competitive miniARS fragments in the miniARS library. (A) A pool of transformed yeast was grown competitively under conditions that selected for the plasmid. Deep sequencing measured the frequency of the miniARS fragments within the population throughout the course of the experiment. In this simple example, miniARS fragment #5 is enriched during the course of the experiment and is assigned a positive competitive fitness value. In contrast, miniARS fragment #2 is depleted during the course of the experiment and is assigned a negative competitive fitness value. If these fragments were from a single ARS (autonomously replicating sequence), ranking them based on their competitive fitness values would help indicate the minimal chromosomal region necessary for maximizing the function of this ARS. In addition, competitive fitness between origins can be compared to allow identification of the most competitive ARSs in the population. (B) The slope of the log&lt;sub&gt;2&lt;/sub&gt; ratios of the frequency of each miniARS fragment in the yeast cell population through the time course was used to measure each fragment&#x2019;s competitive fitness (CF). This value is plotted against the length of the fragment in bp (x-axis). A gray circle designates each fragment that was assessed in this miniARS experiment. In the left panel, the &lt;em&gt;HMR&lt;/em&gt;-E silencer-associated fragments (mini&lt;em&gt;ARS317&lt;/em&gt;) present in the experiment are filled in black, while in the right panel the &lt;em&gt;HML&lt;/em&gt;-E silencer-associated fragments (mini&lt;em&gt;ARS301&lt;/em&gt;) are filled in black. (C) All of the miniARS317-fragments (n = 70, left panel) and all of the miniARS301-fragments (n = 18, right panel) were ranked based on their CF values and then divided into three distinct bins based on this value. These bins were then ranked based on their average competitive fitness (CF) values from lowest (+) to highest CF (+++). To better visualize nucleotides that were enriched in the most competitive miniARS for each silencer, an arbitrary fragment was selected encompassing each silencer and numbered relative to the first nucleotide of the T-rich strand of the ORC (origin recognition complex) binding site that was given the value &#x201C;0.&#x201D; Each bin was color coded as indicated to visualize the fraction of fragments within that bin that contained a given nucleotide. Thus, the most competitive bin of mini&lt;em&gt;ARS317&lt;/em&gt; fragments indicated that greater than 80% of the fragments contained intact RAP1 (Rap1 protein binding site), ORC (origin recognition complex), and FKH (forkhead) sites. The most competitive miniARS silencers (miniARS317-full and miniARS301-full) used for deep mutational scanning in Figure 2 are indicated below the most competitive tercile with a thick black line. Their numbering is relative to the ORC binding site, as above. The FKH*, YHP1* (Yeast Homeo-Protein), and SPT2 (SuPpressor of Ty&#x2019;s)* sites are starred to indicate that their identification is based solely on a strong motif match. In contrast, the silencer ABF1 (ARS-Binding Factor 1), RAP1, and ORC sites have been verified as such by numerous genetic and biochemical experiments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/993/F1.medium.gif" width="431" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/4/993/F1.medium.gif" width="431" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/993/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/993/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467938" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-19" class="first-child">The ARSs associated with the <em>HMR</em>-E and <em>HML</em>-E transcriptional silencers generated many of the most competitive miniARS fragments in the miniARS library. (A) A pool of transformed yeast was grown competitively under conditions that selected for the plasmid. Deep sequencing measured the frequency of the miniARS fragments within the population throughout the course of the experiment. In this simple example, miniARS fragment #5 is enriched during the course of the experiment and is assigned a positive competitive fitness value. In contrast, miniARS fragment #2 is depleted during the course of the experiment and is assigned a negative competitive fitness value. If these fragments were from a single ARS (autonomously replicating sequence), ranking them based on their competitive fitness values would help indicate the minimal chromosomal region necessary for maximizing the function of this ARS. In addition, competitive fitness between origins can be compared to allow identification of the most competitive ARSs in the population. (B) The slope of the log<sub>2</sub> ratios of the frequency of each miniARS fragment in the yeast cell population through the time course was used to measure each fragment’s competitive fitness (CF). This value is plotted against the length of the fragment in bp (x-axis). A gray circle designates each fragment that was assessed in this miniARS experiment. In the left panel, the <em>HMR</em>-E silencer-associated fragments (mini<em>ARS317</em>) present in the experiment are filled in black, while in the right panel the <em>HML</em>-E silencer-associated fragments (mini<em>ARS301</em>) are filled in black. (C) All of the miniARS317-fragments (n = 70, left panel) and all of the miniARS301-fragments (n = 18, right panel) were ranked based on their CF values and then divided into three distinct bins based on this value. These bins were then ranked based on their average competitive fitness (CF) values from lowest (+) to highest CF (+++). To better visualize nucleotides that were enriched in the most competitive miniARS for each silencer, an arbitrary fragment was selected encompassing each silencer and numbered relative to the first nucleotide of the T-rich strand of the ORC (origin recognition complex) binding site that was given the value “0.” Each bin was color coded as indicated to visualize the fraction of fragments within that bin that contained a given nucleotide. Thus, the most competitive bin of mini<em>ARS317</em> fragments indicated that greater than 80% of the fragments contained intact RAP1 (Rap1 protein binding site), ORC (origin recognition complex), and FKH (forkhead) sites. The most competitive miniARS silencers (miniARS317-full and miniARS301-full) used for deep mutational scanning in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> are indicated below the most competitive tercile with a thick black line. Their numbering is relative to the ORC binding site, as above. The FKH*, YHP1* (Yeast Homeo-Protein), and SPT2 (SuPpressor of Ty’s)* sites are starred to indicate that their identification is based solely on a strong motif match. In contrast, the silencer ABF1 (ARS-Binding Factor 1), RAP1, and ORC sites have been verified as such by numerous genetic and biochemical experiments.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">In <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>, the competitive fitness value for each individual fragment in the miniARS experiment (<em>y</em>-axis) was plotted against the fragment’s length in base pairs (<em>x</em>-axis). Fragments overlapping selected origins were then examined for their behavior in the population (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/FigureS1.pdf">Figure S1</a>A). From these analyses, it was clear that the transcriptional silencer <em><a href="http://www.yeastgenome.org/locus/S000029655/overview">HMR</a></em>-E (mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em>) generated a large number of highly competitive miniARS fragments (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a> left and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/FigureS1.pdf">Figure S1</a>D). <em><a href="http://www.yeastgenome.org/locus/S000029214/overview">HML</a></em>-E associated fragments (mini<em><a href="http://www.yeastgenome.org/locus/S000028477/overview">ARS301</a></em>) were also enriched among the most competitive miniARSs, though to a lesser extent than miniA<em>RS317</em> (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1B</a>, right and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/FigureS1.pdf">Figure S1</a>D). <em>HMR-I</em> (mini<em><a href="http://www.yeastgenome.org/locus/S000028493/overview">ARS318</a></em>), a weaker silencer than either of the E-silencers, was also enriched among competitive miniARSs, although to a lesser extent than either of the E-silencers (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/FigureS1.pdf">Figure S1</a>, B–D). Thus, silencer-associated fragments were among the most competitive miniARSs and their overall competitiveness correlated with their relative strength as silencers (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/FigureS1.pdf">Figure S1</a>D).</p><p id="p-21">To define the features of <em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em> associated with the most competitive miniARSs, the <em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em>-associated fragments were binned into three groups based on their competitive fitness values, and the terciles were ranked from lowest (+) to highest (+++) based on their average competitive fitness value (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1C</a>). By focusing on those base pairs that were present in 80% or more of the fragments within the most competitive tercile (+++), a minimal fragment essential for maximal competitive fitness of mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em> was determined. This mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em> fragment contained base pairs −56 to +96 relative to position 0, defined as the first nucleotide of the T-rich strand of the ORC binding site (ORC site) (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1C</a>, left). This fragment will be referred to as mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em>-full. The same approach was used to map the region of mini<em><a href="http://www.yeastgenome.org/locus/S000028477/overview">ARS301</a></em> required for its maximal competitive fitness (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1C</a>, right). Using the same numbering system with respect to the ORC site, this mini<em><a href="http://www.yeastgenome.org/locus/S000028477/overview">ARS301</a></em> fragment contained base pairs −42 to +101 and will be referred to as mini<em><a href="http://www.yeastgenome.org/locus/S000028477/overview">ARS301</a></em>-full.</p><p id="p-22">This mapping revealed obvious similarities between the sequence organizations of the silencer miniARSs that generated the highest competitive fitness. In particular, both silencer miniARS-max fragments contained the silencer <a href="http://www.yeastgenome.org/locus/S000005160/overview">Rap1</a> binding site (<a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site) 5′ of the silencer ORC site. In addition, both silencer miniARSs contained sequences 3′ of this site that are not part of the defined transcriptional silencers and have no known role in silencing (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">McNally and Rine 1991</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Rusche <em>et al.</em> 2003</a>). Thus, both known silencer sites (<a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a>, ORC) and previously uncharacterized nonsilencer sequences [<em>i.e.</em>, including putative binding sites for <a href="http://www.yeastgenome.org/locus/S000001393/overview">Fkh1</a>/2, <a href="http://www.yeastgenome.org/locus/S000002859/overview">Yhp1</a>, and <a href="http://www.yeastgenome.org/locus/S000000963/overview">Spt2</a> proteins, based on motif matches (Tomtom motif comparison tool)] were associated with mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em>-full and mini<em><a href="http://www.yeastgenome.org/locus/S000028477/overview">ARS301</a></em>-full. Statistical analyses of the motifs (<em>e.g.</em>, <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> and FKH) tied to the function of mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em>-full are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/FigureS2.pdf">Figure S2</a>.</p><p id="p-23">We performed a similar analysis with all of the fragments present in the miniARS data set combined to define an “average” ARS (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/FigureS3.pdf">Figure S3</a>). In terms of fragment distance 3′ of the ORC site, the silencer ARSs were, in fact, quite similar to this average ARS structure, and to the structure of the paradigm ARS, <em><a href="http://www.yeastgenome.org/locus/S000029652/overview">ARS1</a></em>. However, the most competitive silencer miniARSs were distinct because they contained a <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site positioned 5′ of their ORC site. Given the known roles of <a href="http://www.yeastgenome.org/locus/S000005160/overview">Rap1</a> on silencer-containing plasmids, these data suggested that <a href="http://www.yeastgenome.org/locus/S000005160/overview">Rap1</a>-mediated partitioning was relevant to the high competitive fitness of the silencer miniARSs in the miniARS competition experiment.</p></div><div id="sec-10" class="subsection"><h3>Deep mutational scanning of miniARS317 and miniARS301</h3><p id="p-24">In a previous study, we used deep mutational scanning of a mini<em><a href="http://www.yeastgenome.org/locus/S000029652/overview">ARS1</a></em> fragment to query the effect of every possible single nucleotide substitution on this fragment’s competitive fitness in a single competitive growth experiment (<a id="xref-ref-36-8" class="xref-bibr" href="#ref-36">Liachko <em>et al.</em> 2013</a>). This strategy was applied to mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em>-full and mini<em><a href="http://www.yeastgenome.org/locus/S000028477/overview">ARS301</a></em>-full to define the nucleotides most critical to their competitive fitness. Mutagenized mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em>-full (mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a>-mut</em>) and mini<em><a href="http://www.yeastgenome.org/locus/S000028477/overview">ARS301</a>-mut</em> libraries were cloned into the same plasmid backbone used for the miniARS library and used to transform yeast. The yeast population was then subjected to competitive growth, and deep sequencing of the plasmid population was performed at the start and finish of the competition. The single nucleotide changes and their effects on miniARS competitive fitness for the two silencer miniARSs are summarized in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/993/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Deep mutational scanning of the HMR-E and HML-E silencer miniARSs-defined nucleotides required for their competitive fitness. (A) The data for a library of miniARS317-mut fragments was graphed to indicate the individual nucleotide substitutions that affected the competitive fitness of miniARS317-full. (B) As in 2A except data are from a miniARS301-mut experiment. (C) Detailed analyses of the mutational profile of the RAP1 (Rap1 protein binding site) sites. A consensus RAP1 site derived from genome-wide chromatin immunoprecipitation data are shown above the profiles. FKH, forkhead; ORC, origin recognition complex; YHP1, Yeast Homeo-Protein; SPT2, SuPpressor of Ty&#x2019;s." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-865468913" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Deep mutational scanning of the &lt;em&gt;HMR&lt;/em&gt;-E and &lt;em&gt;HML&lt;/em&gt;-E silencer miniARSs-defined nucleotides required for their competitive fitness. (A) The data for a library of mini&lt;em&gt;ARS317&lt;/em&gt;-mut fragments was graphed to indicate the individual nucleotide substitutions that affected the competitive fitness of mini&lt;em&gt;ARS317&lt;/em&gt;-full. (B) As in 2A except data are from a mini&lt;em&gt;ARS301&lt;/em&gt;-mut experiment. (C) Detailed analyses of the mutational profile of the RAP1 (Rap1 protein binding site) sites. A consensus RAP1 site derived from genome-wide chromatin immunoprecipitation data are shown above the profiles. FKH, forkhead; ORC, origin recognition complex; YHP1, Yeast Homeo-Protein; SPT2, SuPpressor of Ty&#x2019;s.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/993/F2.medium.gif" width="314" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/4/993/F2.medium.gif" width="314" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/993/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/993/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468014" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-25" class="first-child">Deep mutational scanning of the <em>HMR</em>-E and <em>HML</em>-E silencer miniARSs-defined nucleotides required for their competitive fitness. (A) The data for a library of mini<em>ARS317</em>-mut fragments was graphed to indicate the individual nucleotide substitutions that affected the competitive fitness of mini<em>ARS317</em>-full. (B) As in 2A except data are from a mini<em>ARS301</em>-mut experiment. (C) Detailed analyses of the mutational profile of the RAP1 (Rap1 protein binding site) sites. A consensus RAP1 site derived from genome-wide chromatin immunoprecipitation data are shown above the profiles. FKH, forkhead; ORC, origin recognition complex; YHP1, Yeast Homeo-Protein; SPT2, SuPpressor of Ty’s.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">Several points are worth noting. First, both known (<em>e.g.</em>, ORC and <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a>) and putative (<em>e.g.</em>, FKH) sites present in the maximally competitive silencer miniARSs contained nucleotides that contributed to competitive fitness. A previous study provided evidence that the region containing the putative FKH site contains a second weaker ORC binding site (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Chang <em>et al.</em> 2008</a>). Therefore. it is possible that this region in <em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em>max is not acting as a B3 element but instead is actually providing for additional cryptic origin activity, although based on the functional effects of this site on the competitive fitness of <em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em>max it is much less important for <em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em> propagation compared the silencer ORC site. Second, multiple single nucleotide changes in the known silencer <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> and ORC sites within both silencer miniARSs substantially reduced their competitive fitness. Third, while a small number of single nucleotide changes in the putative B2 and B3 elements of these miniARS fragments reduced their competitive fitness somewhat, the levels of reduction were less than those caused by single nucleotide mutations in either the <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> or ORC sites. It is possible that single nucleotide mutations were insufficient to inactivate these elements. Indeed, a B2 element consensus has been difficult to define, perhaps because it is complex and/or flexible in terms of sequence and/or position (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Chang <em>et al.</em> 2011</a>). However, it is notable that the same A–C change in their putative B2 elements reduced the competitive fitness levels of the two silencer miniARSs to similar extents.</p><p id="p-27">While the importance of the ORC site to silencer miniARS competitive fitness was predicted, the distinct mutational profiles of the mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em>-full and miniARS301-full ORC sites were not. In particular, compared to the ORC site in mini<em><a href="http://www.yeastgenome.org/locus/S000028477/overview">ARS301</a></em>, the ORC site in mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em> was resistant to several single nucleotide substitutions in key regions of this site, particularly in regions outside of the highly conserved A-element of the ORC site (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/FigureS4.pdf">Figure S4</a>). In <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/FigureS3.pdf">Figure S3</a>, the mutational profile of the two different ORC binding sites is presented in an expanded form to allow more detailed examination. For mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em>-full, several nucleotide positions within the conserved core A-element were unexpectedly tolerant of nucleotide substitutions that deviated from the consensus. In addition, mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em>-full was tolerant of mutations within the WTW motif, the conserved nucleotides within the B1 element of yeast origins (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Chang <em>et al.</em> 2008</a>). In contrast, for mini<em><a href="http://www.yeastgenome.org/locus/S000028477/overview">ARS301</a></em>-full [and mini<em><a href="http://www.yeastgenome.org/locus/S000029652/overview">ARS1</a></em> (<a id="xref-ref-36-9" class="xref-bibr" href="#ref-36">Liachko <em>et al.</em> 2013</a>)], the mutational profile of the ORC site followed predictions of the consensus site well. Single nucleotide substitutions throughout the core A-element and within the conserved WTW motif of the mini<em><a href="http://www.yeastgenome.org/locus/S000028477/overview">ARS301</a></em> ORC site substantially reduced its competitive fitness.</p><p id="p-28">In contrast to their differing ORC site mutational profiles, the silencer miniARSs’ <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> sites’ mutational profiles were strikingly similar (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>), suggesting that the same type of sequence-specific binding by <a href="http://www.yeastgenome.org/locus/S000005160/overview">Rap1</a> protein was central to maximizing the competitive fitness of both mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em>-full and mini<em><a href="http://www.yeastgenome.org/locus/S000028477/overview">ARS301</a></em>-full. Thus, this high throughput miniARS-mut approach identified a known plasmid-partitioning element, the <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site, as critical to the competitive fitness of both silencer miniARSs.</p></div><div id="sec-11" class="subsection"><h3>Direct ARS assays showed that the silencer RAP1 site modulated plasmid partitioning without affecting intrinsic replication efficiency of miniARS317 fragments</h3><p id="p-29">While the ORC site is essential for both chromosomal and plasmid origin function of silencers, there are no data to support a similar role for the silencer <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site. However, several reports provide evidence that the <a href="http://www.yeastgenome.org/locus/S000005160/overview">Rap1</a> protein, bound to silencers or telomeres via its silencing function, contributes to plasmid partitioning (Kimmerly and Rine 1987; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Kimmerly <em>et al.</em> 1988</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Longtine <em>et al.</em> 1992</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">1993</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Ansari and Gartenberg 1997</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Andrulis <em>et al.</em> 2002</a>). These studies used standard ARS assays in which individual cells must receive at least one plasmid for a propagation event to be counted. Obviously efficient plasmid distribution is critical in such an assay. However, it was unclear how relevant partitioning was in a population-based miniARS competition experiment that assesses plasmid frequency within the entire cell population as a measure of efficiency. Furthermore, the specific silencer ARS-associated fragments described above were not directly examined in previous studies. Therefore, we directly assessed replicative <em>vs.</em> partitioning functions for several distinct mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em> fragments from the miniARS competition using standard ARS assays in plasmids that either lacked (Acen) or contained (Cen) a centromere (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>). Five of these fragments had low competitive fitness (Low #1–5), while three had high competitive fitness (High #6–8). Fragments 4 and 5 lacked the region upstream of the ORC site that contained the <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> binding site. If these sequences were critical for plasmid partitioning but not replication, then fragments 4 and 5 should provide for efficient ARS activity on a Cen plasmid, even though they showed low competitive fitness in the original miniARS experiment.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/993/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The RAP1 site in miniARS317 contributed to plasmid distribution but not intrinsic replication efficiency. (A) The organization and positions of key elements within silencer ARS317 are indicated at the top by black lines. The eight thicker black lines immediately below these represent the miniARS317 fragment tested in direct ARS (autonomously replicating sequence) assays. Low #1&#x2013;5 indicate fragments with low competitive fitness in the miniARS experiment described in Figure 1. Thus, fragment #3 lacked a portion of the putative FKH* (forkhead) site (putative B3 element), while fragment #4 contained this region of miniARS317 but lacked the RAP1 (Rap1 protein binding site) site. Fragments #6&#x2013;8 showed high competitive fitness in the miniARS experiment. Only fragment #6 contains the silencer ABF1 (ARS-binding Factor 1) site (B) Each of the fragments represented in (A) was tested for ARS function in a centromere-containing plasmid (+centromere, black) or in an analogous plasmid lacking a centromere (&#x2013;centromere, gray). Standard ARS assays were performed as described and the mean plasmid loss rate per generation (PLR) and associated standard error is indicated in the bar graph for a minimum of three independent experiments. ORC, origin recognition complex." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-865468913" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The RAP1 site in mini&lt;em&gt;ARS317&lt;/em&gt; contributed to plasmid distribution but not intrinsic replication efficiency. (A) The organization and positions of key elements within silencer &lt;em&gt;ARS317&lt;/em&gt; are indicated at the top by black lines. The eight thicker black lines immediately below these represent the mini&lt;em&gt;ARS317&lt;/em&gt; fragment tested in direct ARS (autonomously replicating sequence) assays. Low #1&#x2013;5 indicate fragments with low competitive fitness in the miniARS experiment described in Figure 1. Thus, fragment #3 lacked a portion of the putative FKH* (forkhead) site (putative B3 element), while fragment #4 contained this region of mini&lt;em&gt;ARS317&lt;/em&gt; but lacked the RAP1 (Rap1 protein binding site) site. Fragments #6&#x2013;8 showed high competitive fitness in the miniARS experiment. Only fragment #6 contains the silencer ABF1 (ARS-binding Factor 1) site (B) Each of the fragments represented in (A) was tested for ARS function in a centromere-containing plasmid (+centromere, black) or in an analogous plasmid lacking a centromere (&#x2013;centromere, gray). Standard ARS assays were performed as described and the mean plasmid loss rate per generation (PLR) and associated standard error is indicated in the bar graph for a minimum of three independent experiments. ORC, origin recognition complex.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/993/F3.medium.gif" width="440" height="360"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/4/993/F3.medium.gif" width="440" height="360"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/993/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/993/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467891" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-30" class="first-child">The RAP1 site in mini<em>ARS317</em> contributed to plasmid distribution but not intrinsic replication efficiency. (A) The organization and positions of key elements within silencer <em>ARS317</em> are indicated at the top by black lines. The eight thicker black lines immediately below these represent the mini<em>ARS317</em> fragment tested in direct ARS (autonomously replicating sequence) assays. Low #1–5 indicate fragments with low competitive fitness in the miniARS experiment described in <a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1</a>. Thus, fragment #3 lacked a portion of the putative FKH* (forkhead) site (putative B3 element), while fragment #4 contained this region of mini<em>ARS317</em> but lacked the RAP1 (Rap1 protein binding site) site. Fragments #6–8 showed high competitive fitness in the miniARS experiment. Only fragment #6 contains the silencer ABF1 (ARS-binding Factor 1) site (B) Each of the fragments represented in (A) was tested for ARS function in a centromere-containing plasmid (+centromere, black) or in an analogous plasmid lacking a centromere (–centromere, gray). Standard ARS assays were performed as described and the mean plasmid loss rate per generation (PLR) and associated standard error is indicated in the bar graph for a minimum of three independent experiments. ORC, origin recognition complex.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">The ARS assays performed in the Acen plasmid used for the miniARS library revealed that the five distinct mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em> fragments with low competitive fitness values generated high plasmid loss rates (PLRs) (average PLR = 16% ± 3%), whereas the three distinct fragments with high competitive fitness values generated lower PLRs (average PLR = 6% ± 2%) (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>). These results revealed that differences in competitive fitness between these fragments determined from their behavior in the miniARS population experiment were recapitulated by a standard ARS assay in an Acen plasmid. However, ARS assays performed on the same fragments in a Cen plasmid provided evidence that the <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> containing region was important for plasmid partitioning but not the intrinsic replication function of mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em> fragments, as expected based on past studies. In particular, the presence of a centromere affected PLRs substantially for only the mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em> fragments that lacked the silencer <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site (#4 and #5), reducing their PLRs by ∼sixfold and converting them into efficient ARSs. Analyses of variants of <em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em>-full containing single nucleotide substitutions in the <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site confirmed these results, and suggested that the <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site was the only site on this fragment that was essential for Cen-independent partitioning (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/FigureS5.pdf">Figure S5</a>). These data provided evidence that the silencer <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site could at least partially substitute for the function of a centromere in an ARS assay. These data provided evidence that competitive fitness in the miniARS experiment could be modulated substantially by a fragment’s intrinsic partitioning ability.</p><p id="p-32">Early studies documented that the silencers can negatively compete with centromere function when present as the sole replicators on Cen plasmids, a phenomenon named Cen-antagonism (Kimmerly and Rine 1987). If a fragment with replicator function exhibits Cen-antagonism, then it should reduce propagation of a Cen plasmid but not an Acen plasmid. While we did not rigorously test for Cen-antagonism, we note that fragment 6 in <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a> was ∼two-fold less stable in a Cen plasmid compared to an Acen plasmid. This particular fragment contained the entire silencer, including both the <a href="http://www.yeastgenome.org/locus/S000001595/overview">ABF1</a> and <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> sites. Hence, we suspect that this fragment associated more strongly with the Sir proteins and thus promoted a level of <a href="http://www.yeastgenome.org/locus/S000002635/overview">Sir4</a>-<a href="http://www.yeastgenome.org/locus/S000004832/overview">Esc1</a>-mediated partitioning that is capable of competing with (antagonizing) centromere function (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Andrulis <em>et al.</em> 2002</a>). We suspect that weaker associations of Sir proteins may be capable of promoting Cen-independent plasmid propagation without being strong enough to antagonize centromere function, such as the smaller silencer fragments we tested in <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a> that contained only the <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site 5′ of the silencer ORC site.</p></div><div id="sec-12" class="subsection"><h3>Numerical simulations revealed how the effects of cellular plasmid accumulation on replication efficiency could create a strong competitive advantage for fragments with partitioning ability in the miniARS experiment</h3><p id="p-33">The <a href="http://www.yeastgenome.org/locus/S000005160/overview">Rap1</a>-mediated mechanism that links silenced regions to the INM is considered robust. Consistent with this observation, silencer <a href="http://www.yeastgenome.org/locus/S000005160/overview">Rap1</a>-mediated plasmid partitioning is so robust that under some circumstances it can compete against a centromere (referred to as centromere antagonism) (Kimmerly and Rine 1987). Although the data described above supported the idea that silencer <a href="http://www.yeastgenome.org/locus/S000005160/overview">Rap1</a>-mediated partitioning could have a profound consequence on plasmid abundance in the competitive miniARS experiment, it remained unclear how plasmid partitioning might affect any particular miniARS fragment’s competitive fitness in a miniARS experiment. To address this issue, numerical simulations were performed (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Three relevant variables were defined for these exercises as diagrammed in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4, A and B</a>: replication fitness, partitioning fitness, and crowding penalty.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/993/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Numerical simulations revealed how differences in partitioning abilities could affect the competitive fitness of miniARS fragments. (A) Three variables were defined for these numerical simulations, as diagrammed here and discussed in the text. The Crowding Penalty was treated as a variable constant in these simulations, with all plasmids in a given simulation conferring the same Crowding Penalty value. (B) Effect of two different Crowding Penalty values on Replication Fitness and fragment accumulation per cell (C) Results of three different simulations performed under different Crowding Penalty values, as shown, depicted as heat maps. As Partitioning Fitness decreased (i.e., increased probability that fragments remained in the mother cell after cell division), reduced competitive fitness resulted even if the fragment had high Replication Fitness (RF)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-865468913" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Numerical simulations revealed how differences in partitioning abilities could affect the competitive fitness of miniARS fragments. (A) Three variables were defined for these numerical simulations, as diagrammed here and discussed in the text. The Crowding Penalty was treated as a variable constant in these simulations, with all plasmids in a given simulation conferring the same Crowding Penalty value. (B) Effect of two different Crowding Penalty values on Replication Fitness and fragment accumulation per cell (C) Results of three different simulations performed under different Crowding Penalty values, as shown, depicted as heat maps. As Partitioning Fitness decreased (&lt;em&gt;i.e.&lt;/em&gt;, increased probability that fragments remained in the mother cell after cell division), reduced competitive fitness resulted even if the fragment had high Replication Fitness (RF).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/993/F4.medium.gif" width="440" height="405"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/4/993/F4.medium.gif" width="440" height="405"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/993/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/993/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467904" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-34" class="first-child">Numerical simulations revealed how differences in partitioning abilities could affect the competitive fitness of miniARS fragments. (A) Three variables were defined for these numerical simulations, as diagrammed here and discussed in the text. The Crowding Penalty was treated as a variable constant in these simulations, with all plasmids in a given simulation conferring the same Crowding Penalty value. (B) Effect of two different Crowding Penalty values on Replication Fitness and fragment accumulation per cell (C) Results of three different simulations performed under different Crowding Penalty values, as shown, depicted as heat maps. As Partitioning Fitness decreased (<em>i.e.</em>, increased probability that fragments remained in the mother cell after cell division), reduced competitive fitness resulted even if the fragment had high Replication Fitness (RF).</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">Replication fitness was defined as the probability that all copies of a particular miniARS fragment present in a cell would be duplicated in a given S-phase. In the example shown, the miniARS is present in four copies in the starting cell. If the replication fitness of this fragment were 1.0, then the probability would be high that the four fragments would produce eight fragments after S-phase. A lower replication fitness of 0.5 would mean that the probability would be low that all four fragments would be copied. Partitioning fitness was defined as the probability that miniARS fragments in a cell would distribute equitably to mother and daughter cells after cell division. In the example shown, if the two fragments have a high partitioning fitness (<em>e.g.</em>, 1.0), then the probability would be high that each of the mother and daughter cells will receive a fragment after cell division. In contrast, a fragment with low partitioning fitness (<em>e.g.</em>, 0.5) would have a high probability of being retained by the mother cell after cell division. Crowding penalty was described as the probability that the replication fitness of a miniARS fragment, regardless of the intrinsic origin efficiency of the element, will decrease as the numbers of the miniARS fragments increase in a given cell. In this simulation, a crowding penalty of one means that, once a given cell has acquired 100 miniARS fragments, the replication fitness of each of those fragments is reduced by 50%. The crowding penalty was treated as a constant within each individual simulation.</p><p id="p-36">In <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a> the results from three different numerical simulations are depicted, each under a distinct crowding penalty as indicated. In each simulation, 100 hypothetical cells each harboring a different miniARS with a distinct intrinsic replication fitness (<em>x</em>-axis) and partitioning fitness (<em>y</em>-axis) were allowed to divide for twelve generations. To account for natural variations among the cells in a population, Gaussian noise was introduced for these two parameters in the model. Importantly, the model was robust, producing similar results with different levels of Guassian noise. The predicted frequency (copy number, the measure of competitive fitness) of each miniARS fragment at the end of simulation was depicted in a heat map. In the absence of a crowding penalty (Crowding Penalty = 0), only replication fitness had a large impact on the competitive fitness of a miniARS fragment (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4C</a>). However, if the crowding penalty was increased to 1.0, differences in partitioning fitness began to substantially impact the competitive fitness of a miniARS fragment. The effect of partitioning fitness on competitive fitness becomes stronger as the crowding penalty increases. Regardless, even modest assumptions about the magnitude of a crowding penalty meant that both replication fitness and partitioning fitness could contribute substantially to the competitive fitness of a miniARS fragment. These exercises, together with data from the silencer miniARSs described above, suggested that partitioning effects in the miniARS experiment could be significant and were unlikely to be restricted to the three silencer fragments.</p></div><div id="sec-13" class="subsection"><h3>Some nonsilencer miniARSs contained regions that contribute to plasmid partitioning</h3><p id="p-37">Based on the data discussed thus far, the miniARS experiment was sensitive to both replication and partitioning abilities. Natural, chromosomal-based plasmid partitioning elements other than the <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site have not been described. Therefore, the miniARS data set provided an opportunity to test whether other as yet undefined partitioning elements were closely associated with nonsilencer ARSs represented in the library. To test whether other ARSs might be associated with putative partitioning element(s) (<em>i.e.</em>, regions that behaved like the silencer <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site), we examined fragments of low and high competitive fitness for six different nonsilencer miniARSs by standard ARS assays, as described for the mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em> experiments in <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a> (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5A</a>). These miniARSs were selected for further analysis because they each contained at least one fragment representative in the least and most competitive deciles within the miniARS competition, and in this way behaved similarly to mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em>.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/993/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Some nonsilencer miniARSs contained regions that contributed to plasmid partitioning. (A) Diagram of low competitive (white bars) and high competitive (black bars) miniARS fragments examined by standard ARS (autonomously replicating sequence) assays in part (B). The line above each pair of fragments indicates the ORC (origin recognition complex) binding site in the standard orientation used throughout this study&#x2014;the T-rich strand of the ORC site is on the top strand. (B) ARS assays were performed and presented as described for Figure 3B. Acen, lacking a centromere; Cen, containing a centromere." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-865468913" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Some nonsilencer miniARSs contained regions that contributed to plasmid partitioning. (A) Diagram of low competitive (white bars) and high competitive (black bars) miniARS fragments examined by standard ARS (autonomously replicating sequence) assays in part (B). The line above each pair of fragments indicates the ORC (origin recognition complex) binding site in the standard orientation used throughout this study&#x2014;the T-rich strand of the ORC site is on the top strand. (B) ARS assays were performed and presented as described for Figure 3B. Acen, lacking a centromere; Cen, containing a centromere.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/993/F5.medium.gif" width="418" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/4/993/F5.medium.gif" width="418" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/993/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/993/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467906" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-38" class="first-child">Some nonsilencer miniARSs contained regions that contributed to plasmid partitioning. (A) Diagram of low competitive (white bars) and high competitive (black bars) miniARS fragments examined by standard ARS (autonomously replicating sequence) assays in part (B). The line above each pair of fragments indicates the ORC (origin recognition complex) binding site in the standard orientation used throughout this study—the T-rich strand of the ORC site is on the top strand. (B) ARS assays were performed and presented as described for <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3B</a>. Acen, lacking a centromere; Cen, containing a centromere.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">As expected, the six distinct miniARS fragments with high competitive fitness generated relatively low plasmid loss rates (PLRs) compared to the corresponding fragments with low competitive fitness (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a>). For three of these ARSs— <em><a href="http://www.yeastgenome.org/locus/S000118440/overview">ARS428</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000118431/overview">ARS410</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000077378/overview">ARS503</a></em>—the PLRs were similar for their fragments regardless of whether the plasmid contained a centromere. Thus providing a robust plasmid distribution mechanism via a centromere had no effect on ARS stability for these miniARS fragments, suggesting that their competitive fitness differences were due primarily to replication fitness differences. However, for the three remaining ARSs in this group— <em><a href="http://www.yeastgenome.org/locus/S000077395/overview">ARS523</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000028444/overview">ARS1007</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000028445/overview">ARS1018</a></em>—the fragments that produced low competitive fitness in the miniARS experiment (and high PLRs on an Acen plasmid) generated low PLRs when present on a Cen plasmid. These results suggested that the <em><a href="http://www.yeastgenome.org/locus/S000077395/overview">ARS523</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000028444/overview">ARS1007</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000028445/overview">ARS1018</a></em> fragments with low competitive fitness actually possessed intrinsically high core replication abilities, but lacked efficient partitioning abilities, behaving similarly to mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em> fragments that lacked a <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site.</p></div><div id="sec-14" class="subsection"><h3>Using the miniARS experiment to disambiguate partitioning and replication functions</h3><p id="p-40">The data described above revealed that as yet undefined partitioning elements could influence the competitive fitness of miniARSs. To address this issue using the high throughput power of the miniARS experiment described, we transferred the miniARS library to a plasmid containing a centromere. The resulting Cen-miniARS library was used to transform yeast and a competitive growth experiment was performed as for the Acen-miniARS library. The Acen-miniARS and Cen-miniARS data sets from these competitive growth experiments were then compared (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/993/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparing the miniARS experiments to disambiguate partitioning and replication functions associated with miniARS chromosomal fragments. (A) MiniARS317 fragments depleted in the Cen- (centromere containing) miniARS or the Acen- (without a centromere) miniARS experiments were compared to fragments enriched in these experiments. (B) The fragments within the shared miniARS data set (i.e., fragments present in both the Acen-miniARS library and the Cen-miniARS library) that ranked at a given competitive fitness decile are indicated in this heat map. This heat map graph is divided into 100 sections, each representing a distinct combination of competitive fitness deciles within the Acen-miniARS and Cen-miniARS experiments as indicated in the figure. The bottom left corner contained fragments that performed with a low competitive fitness in both the Acen- and Cen-miniARS competitions and were therefore determined to have both weak replication and partitioning abilities relative to other fragments in the population (Partition &#x201C;&#x2212;&#x201D;; Replication &#x201C;&#x2212;&#x201D;). There were 288 fragments representing 62 origins in this portion of the heat map graph. The bottom right corner contained fragments that performed with a high competitive fitness in the Acen-miniARS library but a low competitive fitness in the Cen-miniARS library, and were therefore determined to have relatively strong partitioning abilities but weak replication abilities (Partition &#x201C;+&#x201D;; Replication &#x201C;&#x2212;&#x201D;). There were 103 fragments representing 32 origins in this portion of the heat map graph. The top left corner contained fragments that performed with a low competitive fitness in the Acen-miniARS library but a high competitive fitness in the Cen-miniARS library, and were therefore determined to have relatively weak partitioning abilities but strong replication abilities (Partition &#x201C;&#x2212;&#x201D;; Replication &#x201C;+&#x201D;). There were 105 fragments representing 33 origins in this portion of the heat map graph. (C) Combined analyses of fragments in B designed to define regions critical for maximizing partitioning ability [first (from the top) and fourth pair of heat maps] vs. those more critical for maximizing replication ability (second and third heat map pairs). (D) Heat map graphs as in B, except they contained only miniARS fragments from the indicated ARSs. Please see text for more details. ARS, autonomously replicating sequence; FKH, forkhead; ORC, origin recognition complex; RAP1, Rap1 protein binding site." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-865468913" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparing the miniARS experiments to disambiguate partitioning and replication functions associated with miniARS chromosomal fragments. (A) Mini&lt;em&gt;ARS317&lt;/em&gt; fragments depleted in the Cen- (centromere containing) miniARS or the Acen- (without a centromere) miniARS experiments were compared to fragments enriched in these experiments. (B) The fragments within the shared miniARS data set (&lt;em&gt;i.e.&lt;/em&gt;, fragments present in both the Acen-miniARS library and the Cen-miniARS library) that ranked at a given competitive fitness decile are indicated in this heat map. This heat map graph is divided into 100 sections, each representing a distinct combination of competitive fitness deciles within the Acen-miniARS and Cen-miniARS experiments as indicated in the figure. The bottom left corner contained fragments that performed with a low competitive fitness in both the Acen- and Cen-miniARS competitions and were therefore determined to have both weak replication and partitioning abilities relative to other fragments in the population (Partition &#x201C;&#x2212;&#x201D;; Replication &#x201C;&#x2212;&#x201D;). There were 288 fragments representing 62 origins in this portion of the heat map graph. The bottom right corner contained fragments that performed with a high competitive fitness in the Acen-miniARS library but a low competitive fitness in the Cen-miniARS library, and were therefore determined to have relatively strong partitioning abilities but weak replication abilities (Partition &#x201C;+&#x201D;; Replication &#x201C;&#x2212;&#x201D;). There were 103 fragments representing 32 origins in this portion of the heat map graph. The top left corner contained fragments that performed with a low competitive fitness in the Acen-miniARS library but a high competitive fitness in the Cen-miniARS library, and were therefore determined to have relatively weak partitioning abilities but strong replication abilities (Partition &#x201C;&#x2212;&#x201D;; Replication &#x201C;+&#x201D;). There were 105 fragments representing 33 origins in this portion of the heat map graph. (C) Combined analyses of fragments in B designed to define regions critical for maximizing partitioning ability [first (from the top) and fourth pair of heat maps] &lt;em&gt;vs.&lt;/em&gt; those more critical for maximizing replication ability (second and third heat map pairs). (D) Heat map graphs as in B, except they contained only miniARS fragments from the indicated ARSs. Please see text for more details. ARS, autonomously replicating sequence; FKH, forkhead; ORC, origin recognition complex; RAP1, Rap1 protein binding site.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/993/F6.medium.gif" width="402" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/4/993/F6.medium.gif" width="402" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/993/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/993/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467863" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-41" class="first-child">Comparing the miniARS experiments to disambiguate partitioning and replication functions associated with miniARS chromosomal fragments. (A) Mini<em>ARS317</em> fragments depleted in the Cen- (centromere containing) miniARS or the Acen- (without a centromere) miniARS experiments were compared to fragments enriched in these experiments. (B) The fragments within the shared miniARS data set (<em>i.e.</em>, fragments present in both the Acen-miniARS library and the Cen-miniARS library) that ranked at a given competitive fitness decile are indicated in this heat map. This heat map graph is divided into 100 sections, each representing a distinct combination of competitive fitness deciles within the Acen-miniARS and Cen-miniARS experiments as indicated in the figure. The bottom left corner contained fragments that performed with a low competitive fitness in both the Acen- and Cen-miniARS competitions and were therefore determined to have both weak replication and partitioning abilities relative to other fragments in the population (Partition “−”; Replication “−”). There were 288 fragments representing 62 origins in this portion of the heat map graph. The bottom right corner contained fragments that performed with a high competitive fitness in the Acen-miniARS library but a low competitive fitness in the Cen-miniARS library, and were therefore determined to have relatively strong partitioning abilities but weak replication abilities (Partition “+”; Replication “−”). There were 103 fragments representing 32 origins in this portion of the heat map graph. The top left corner contained fragments that performed with a low competitive fitness in the Acen-miniARS library but a high competitive fitness in the Cen-miniARS library, and were therefore determined to have relatively weak partitioning abilities but strong replication abilities (Partition “−”; Replication “+”). There were 105 fragments representing 33 origins in this portion of the heat map graph. (C) Combined analyses of fragments in B designed to define regions critical for maximizing partitioning ability [first (from the top) and fourth pair of heat maps] <em>vs.</em> those more critical for maximizing replication ability (second and third heat map pairs). (D) Heat map graphs as in B, except they contained only miniARS fragments from the indicated ARSs. Please see text for more details. ARS, autonomously replicating sequence; FKH, forkhead; ORC, origin recognition complex; RAP1, Rap1 protein binding site.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">If a fragment showed high competitive fitness in the Acen-miniARS experiment primarily because it contained an element(s) that performed a partitioning function, such as a <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site, then it should exhibit a comparatively lower competitive fitness in the Cen-miniARS experiment because partitioning ability would no longer be an advantage. Thus, in theory, simply comparing the most competitive fragments between the two different experiments should reveal chromosomal regions that contain plasmid-partitioning element(s). For example, for mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em>, the prediction was that the <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site would not be required for high competitive fitness of mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em> fragments in the Cen-miniARS experiment. To test this prediction, all of the mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em> fragments with low competitive fitness (<em>i.e.</em>, depleted over the course of competitive growth) in a given miniARS experiment were grouped and compared to all of the mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em> fragments with high competitive fitness (<em>i.e.</em>, enriched over the course of competitive growth) in the same experiment (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6A</a>). For the Cen-miniARS competition, the <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site was underrepresented in the group of high competitive fitness fragments compared to the group of low competitive fitness. We interpret this to mean that the <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site was not selected for in the CEN-miniARS experiment because a partitioning function provided no advantage. In contrast, and as expected from analyses in <a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1</a>, for the Acen-miniARS competition, the <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site was overrepresented in the group of high competitive fitness compared to the group of low competitive fitness fragments. Thus, the <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site was selected for in the Acen-miniARS experiment because it provided partitioning function. These data demonstrated that comparative analysis of the Acen- and Cen-miniARS competitions could be used to help identify a known partitioning element.</p><p id="p-43">The comparative approach was used in a combined analysis of all miniARS fragments to determine how many miniARSs were affected substantially by an associated partitioning function in the Acen-miniARS experiment. In <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a>, the miniARS fragments were distributed over a heat map divided into 100 sections, each representing a distinct competitive decile in the Acen-miniARS (<em>x</em>-axis) and Cen-miniARS (<em>y</em>-axis) experiments. A perfect correlation between competitive fitness in the two different miniARS competition experiments would be expected to produce fragments only within the central diagonal portion of the graph. Such an outcome would occur if differences in partitioning bias between fragments played no role in competitive fitness, which we know from the experiments described above is not the case. Of the fragments present in the experiments, 863 (38%) fell near this diagonal line, but many other fragments fell in regions suggesting that partitioning might affect their competitive fitness. For example, 103 fragments fell within the bottom right corner of the graph (grouped in the box labeled Partition: “+”; Replication: “−”) (Note: “Replication: “−”” means only that these fragments are weak replicators relative to others in the population; any fragment that made it into the population at all had to possess some basal replicative function). These fragments showed high competitive fitness in the Acen-miniARS competition but low competitive fitness in the Cen-miniARS competition and represented 32 distinct ARSs, 28% of the ARSs analyzed in this experiment. Fragments in this region of the heat map likely contained (a) DNA sequence element(s) that contributed to partitioning fitness, giving them a high competitive fitness value in the Acen-miniARS experiment even though their intrinsic replication fitness was relatively low. Conversely, the 172 miniARS-fragments, representing 24 ARSs, within the top left corner (Partition: “−”; Replication: “+”) likely contained intrinsically strong replication ability that was masked in the Acen-miniARS experiment by being out-competed by miniARS fragments with robust partitioning ability.</p><p id="p-44">To better understand the sequences that might drive partitioning <em>vs.</em> replication, we initially focused on regions that we interpreted to <em>enhance partitioning</em>. For this purpose, fragments in the bottom left corner of the graph (Partition: “−”; Replication: “−”) were grouped together and compared to fragments in the bottom right corner (Partition: “+”; Replication: “−”). These fragments should differ from each other at the level of partitioning ability. As seen in the top pair of heat maps in <a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6C</a>, acquisition of enhanced partitioning ability was associated with acquisition of sequences 3’ of the ORC site. Conversely, to focus primarily on regions that enhanced replication, fragments in the bottom left corner (Partition: “−”; Replication: “−”) were compared to fragments in the top left corner (Partition: “−”; Replication: “+”). These fragments should differ from each other at the level of replication ability. As seen in the second pair of heat maps in <a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6C</a>, acquisition of enhanced replication ability was also associated with acquisition of sequences 3’ of the ORC site, though to a lesser degree than for partitioning ability. Regardless, these analyses revealed that acquisition of either replication or partitioning ability was associated with acquisition of sequences 3’ of the ORC site, corresponding in terms of position to the B3 element in <em><a href="http://www.yeastgenome.org/locus/S000029652/overview">ARS1</a></em>. Thus, at this level of combined analyses, selecting for either enhanced partitioning or replication abilities led to similar, though not identical, average fragments. A similar outcome was obtained from reverse analyses, that is examining loss of replication abilities or partitioning abilities rather than acquisition of these abilities (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6C</a>, bottom pair of heat maps). Again, both abilities tracked with sequences 3’ to the ORC binding site, but partitioning ability was more sensitive to these regions than replication. These analyses suggested that 5’ positioning of a motif relative to the ORC site, as is observed for the <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site partitioning element of mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em>, is not a general prerequisite for a partitioning function. Instead, the sequence identity of particular elements is likely the larger determinant.</p><p id="p-45">Heat maps analogous to that in <a id="xref-fig-6-7" class="xref-fig" href="#F6">Figure 6B</a> were generated for individual miniARSs, including the silencer miniARSs (<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000028477/overview">ARS301</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000028493/overview">ARS318</a></em>) (<a id="xref-fig-6-8" class="xref-fig" href="#F6">Figure 6D</a>). Each of these miniARSs, with the exception of <em><a href="http://www.yeastgenome.org/locus/S000114834/overview">ARS422</a></em>, which served as an example of a “nonpartitioning” origin, generated, by definition, at least one fragment in the bottom right corner of the heat map in <a id="xref-fig-6-9" class="xref-fig" href="#F6">Figure 6B</a> and thus were considered “partitioning” ARSs. The graphs for both mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em> and mini<em><a href="http://www.yeastgenome.org/locus/S000028477/overview">ARS301</a></em> fragments were fairly distinct examples, in that the majority of fragments skewed toward the far right side of the graph, presumably indicative of strong partitioning capacity relative to replication capacity. In contrast, mini<em><a href="http://www.yeastgenome.org/locus/S000028493/overview">ARS318</a></em>, a considerably weaker silencer than either <em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em> or <em><a href="http://www.yeastgenome.org/locus/S000028477/overview">ARS301</a></em>, generated a distinct pattern of fragments, with many falling in the upper left region of the graph, indicative of relatively strong replication capacity relative to partitioning.</p><p id="p-46">Analogous heat maps for four nonsilencer miniARSs are also shown in <a id="xref-fig-6-10" class="xref-fig" href="#F6">Figure 6D</a>. Three of these ARSs contained regions required for partitioning that could be separated from regions intrinsic to replication ability based on the ARS assays shown in <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a> (<em><a href="http://www.yeastgenome.org/locus/S000077395/overview">ARS523</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000028444/overview">ARS1007</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000028445/overview">ARS1018</a></em>). Each of these miniARSs generated several fragments that fell on the far right side of the graph, including some in the bottom right corner, indicative of fragments containing relatively strong partitioning capacity relative to replication. In contrast, the example “nonpartitioner” mini<em><a href="http://www.yeastgenome.org/locus/S000114834/overview">ARS422</a></em> did not generate any fragments that fell within the far right bottom corner of the graph. In fact, most mini<em><a href="http://www.yeastgenome.org/locus/S000114834/overview">ARS422</a></em> fragments fell on the left side of the diagonal line, suggesting that many <em><a href="http://www.yeastgenome.org/locus/S000114834/overview">ARS422</a></em> fragments had high intrinsic replication ability compared to other fragments in the data set, but relatively weak partitioning ability.</p></div><div id="sec-15" class="subsection"><h3>Mapping regions required for plasmid partitioning</h3><p id="p-47">Because mini<em><a href="http://www.yeastgenome.org/locus/S000028493/overview">ARS318</a></em> generated many fragments distributed throughout the heat map in <a id="xref-fig-6-11" class="xref-fig" href="#F6">Figure 6D</a>, it was used as a test case for comparing a fragment maximally fit for replication (<em>i.e.</em>, high competitive fitness in the Cen-miniARS experiment) to one maximally fit for partitioning (<em>i.e.</em>, high competitive fitness in the Acen-miniARS experiment). To map nucleotides important for the partitioning ability of mini<em><a href="http://www.yeastgenome.org/locus/S000028493/overview">ARS318</a></em>, fragments from the lower left corner of the <em><a href="http://www.yeastgenome.org/locus/S000028493/overview">ARS318</a></em> graph in <a id="xref-fig-6-12" class="xref-fig" href="#F6">Figure 6D</a> were compared to fragments in the lower right corner (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>, top heat maps). The chromosomal region most enriched as mini<em><a href="http://www.yeastgenome.org/locus/S000028493/overview">ARS318</a></em> acquired partitioning ability was on the 5′ side of the ORC binding site and included the silencer <a href="http://www.yeastgenome.org/locus/S000001595/overview">ABF1</a> site, which is essential for its silencer function. These data indicate a role for another silencer binding protein in partitioning, and are consistent with the current model that links intrinsic aspects of silencer function to plasmid partitioning (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Enomoto <em>et al.</em> 1994</a>). To map nucleotides important for maximizing the replication function of mini<em><a href="http://www.yeastgenome.org/locus/S000028493/overview">ARS318</a></em>, fragments from the lower left corner of the <em><a href="http://www.yeastgenome.org/locus/S000028493/overview">ARS318</a></em> graph in <a id="xref-fig-6-13" class="xref-fig" href="#F6">Figure 6D</a> were grouped and compared to fragments in the upper left corner (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7A</a>, bottom heat maps). A distinct fragment was mapped for maximal replication efficiency that showed relatively little enrichment for nucleotides within the <a href="http://www.yeastgenome.org/locus/S000001595/overview">ABF1</a> site, consistent with the relatively minor role for this site in the origin function of <em><a href="http://www.yeastgenome.org/locus/S000028493/overview">ARS318</a></em> compared to the region 3′ of the ORC binding site including the putative B2 element (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Chang <em>et al.</em> 2008</a>). Thus, exploiting the differential fragment distribution in the two miniARS data sets revealed distinct regions of mini<em><a href="http://www.yeastgenome.org/locus/S000028493/overview">ARS318</a></em> important for replication <em>vs.</em> partitioning ability.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/993/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mapping partitioning regions of miniARSs by using heat maps. (A) To map the region that provided for maximal replication fitness of miniARS318, fragments that fell within the bottom left corner of the heat map graph in Figure 6D (Partition &#x201C;&#x2212;&#x201D;; Replication &#x201C;&#x2212;&#x201D;) were grouped and compared to the fragments that fell within the top left corner (Partition &#x201C;&#x2212;&#x201D;; Replication &#x201C;+&#x201D;). To map the region that provided for partitioning of miniARS318, fragments that fell within the bottom left corner of the heat map graph in Figure 6D (Partition &#x201C;&#x2212;&#x201D;; Replication &#x201C;&#x2212;&#x201D;) were grouped and compared to the fragments that fell within the bottom right corner (Partition &#x201C;+&#x201D;; Replication &#x201C;&#x2212;&#x201D;). The color-coding for the heat maps to represent fraction of fragments with a given nucleotide is same as that used in previous figures. (B) Mapping partitioning regions of the three nonsilencer origins in Figure 5 using the same approach as for miniARS318. The white and black fragments above the colored heat maps are the same low and high competitive fitness fragments from the original Acen-miniARS experiment that were assessed for ARS activity directly in Figure 5. Partitioning &#x201C;&#x2212;&#x201D; and Replication &#x201C;+&#x201D; indicates the fragment that had low competitive fitness in the Acen-miniARS experiment, and thus showed a high plasmid loss rate in an Acen plasmid but a low plasmid loss rate in a Cen plasmid (white bar). Partitioning &#x201C;+&#x201D; and Replication &#x201C;+&#x201D; indicates the fragment that showed high competitive fitness in both the Acen- and Cen-miniARS experiments and thus low plasmid loss rates in both an Acen and Cen plasmid. Below these black and white fragments, the heat map graphs generated by a comparison of fragment distributions in the two different miniARS experiments (Figure 6D) for the indicated ARSs were used to map the regions acquired when the miniARS fragments gained partitioning ability, as described above and in the text. The shaded boxes indicate regions that contained putative partitioning elements based on these analyses. Motif matches (Tomtom motif comparison tool) identified within the mapped partitioning regions and their significance (P-value) are indicated. The consensus motif is shown about the actual sequence identified. The position of the ORC site, with the T-rich strand 5&#x2032; to 3&#x2032; is indicated. The binding sites for motifs are starred* to indicate that they were identified only by a motif match. Acen, without a centromere; ARS, autonomously replicating sequence; Cen, with a centromere; FKH, forkhead; ORC, origin recognition complex; RAP1, Rap1 protein binding site; ABF1, (ARS-Binding Factor 1); GTS1, Glycine Threonine Serine repeat protein; RIM101, Regulator of IME2 (Inducer of Meiosis 2); RSC1, Remodel the Structure of Chromatin; CIN5, Chromosome INstability factor 5." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-865468913" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Mapping partitioning regions of miniARSs by using heat maps. (A) To map the region that provided for maximal replication fitness of mini&lt;em&gt;ARS318&lt;/em&gt;, fragments that fell within the bottom left corner of the heat map graph in Figure 6D (Partition &#x201C;&#x2212;&#x201D;; Replication &#x201C;&#x2212;&#x201D;) were grouped and compared to the fragments that fell within the top left corner (Partition &#x201C;&#x2212;&#x201D;; Replication &#x201C;+&#x201D;). To map the region that provided for partitioning of mini&lt;em&gt;ARS318&lt;/em&gt;, fragments that fell within the bottom left corner of the heat map graph in Figure 6D (Partition &#x201C;&#x2212;&#x201D;; Replication &#x201C;&#x2212;&#x201D;) were grouped and compared to the fragments that fell within the bottom right corner (Partition &#x201C;+&#x201D;; Replication &#x201C;&#x2212;&#x201D;). The color-coding for the heat maps to represent fraction of fragments with a given nucleotide is same as that used in previous figures. (B) Mapping partitioning regions of the three nonsilencer origins in Figure 5 using the same approach as for mini&lt;em&gt;ARS318&lt;/em&gt;. The white and black fragments above the colored heat maps are the same low and high competitive fitness fragments from the original Acen-miniARS experiment that were assessed for ARS activity directly in Figure 5. Partitioning &#x201C;&#x2212;&#x201D; and Replication &#x201C;+&#x201D; indicates the fragment that had low competitive fitness in the Acen-miniARS experiment, and thus showed a high plasmid loss rate in an Acen plasmid but a low plasmid loss rate in a Cen plasmid (white bar). Partitioning &#x201C;+&#x201D; and Replication &#x201C;+&#x201D; indicates the fragment that showed high competitive fitness in both the Acen- and Cen-miniARS experiments and thus low plasmid loss rates in both an Acen and Cen plasmid. Below these black and white fragments, the heat map graphs generated by a comparison of fragment distributions in the two different miniARS experiments (Figure 6D) for the indicated ARSs were used to map the regions acquired when the miniARS fragments gained partitioning ability, as described above and in the text. The shaded boxes indicate regions that contained putative partitioning elements based on these analyses. Motif matches (Tomtom motif comparison tool) identified within the mapped partitioning regions and their significance (P-value) are indicated. The consensus motif is shown about the actual sequence identified. The position of the ORC site, with the T-rich strand 5&#x2032; to 3&#x2032; is indicated. The binding sites for motifs are starred* to indicate that they were identified only by a motif match. Acen, without a centromere; ARS, autonomously replicating sequence; Cen, with a centromere; FKH, forkhead; ORC, origin recognition complex; RAP1, Rap1 protein binding site; ABF1, (ARS-Binding Factor 1); GTS1, Glycine Threonine Serine repeat protein; RIM101, Regulator of IME2 (Inducer of Meiosis 2); RSC1, Remodel the Structure of Chromatin; CIN5, Chromosome INstability factor 5.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/993/F7.medium.gif" width="440" height="372"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/6/4/993/F7.medium.gif" width="440" height="372"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/993/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/993/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467936" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-48" class="first-child">Mapping partitioning regions of miniARSs by using heat maps. (A) To map the region that provided for maximal replication fitness of mini<em>ARS318</em>, fragments that fell within the bottom left corner of the heat map graph in <a id="xref-fig-6-14" class="xref-fig" href="#F6">Figure 6D</a> (Partition “−”; Replication “−”) were grouped and compared to the fragments that fell within the top left corner (Partition “−”; Replication “+”). To map the region that provided for partitioning of mini<em>ARS318</em>, fragments that fell within the bottom left corner of the heat map graph in <a id="xref-fig-6-15" class="xref-fig" href="#F6">Figure 6D</a> (Partition “−”; Replication “−”) were grouped and compared to the fragments that fell within the bottom right corner (Partition “+”; Replication “−”). The color-coding for the heat maps to represent fraction of fragments with a given nucleotide is same as that used in previous figures. (B) Mapping partitioning regions of the three nonsilencer origins in <a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5</a> using the same approach as for mini<em>ARS318</em>. The white and black fragments above the colored heat maps are the same low and high competitive fitness fragments from the original Acen-miniARS experiment that were assessed for ARS activity directly in <a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5</a>. Partitioning “−” and Replication “+” indicates the fragment that had low competitive fitness in the Acen-miniARS experiment, and thus showed a high plasmid loss rate in an Acen plasmid but a low plasmid loss rate in a Cen plasmid (white bar). Partitioning “+” and Replication “+” indicates the fragment that showed high competitive fitness in both the Acen- and Cen-miniARS experiments and thus low plasmid loss rates in both an Acen and Cen plasmid. Below these black and white fragments, the heat map graphs generated by a comparison of fragment distributions in the two different miniARS experiments (<a id="xref-fig-6-16" class="xref-fig" href="#F6">Figure 6D</a>) for the indicated ARSs were used to map the regions acquired when the miniARS fragments gained partitioning ability, as described above and in the text. The shaded boxes indicate regions that contained putative partitioning elements based on these analyses. Motif matches (Tomtom motif comparison tool) identified within the mapped partitioning regions and their significance (P-value) are indicated. The consensus motif is shown about the actual sequence identified. The position of the ORC site, with the T-rich strand 5′ to 3′ is indicated. The binding sites for motifs are starred* to indicate that they were identified only by a motif match. Acen, without a centromere; ARS, autonomously replicating sequence; Cen, with a centromere; FKH, forkhead; ORC, origin recognition complex; RAP1, Rap1 protein binding site; ABF1, (ARS-Binding Factor 1); GTS1, Glycine Threonine Serine repeat protein; RIM101, Regulator of IME2 (Inducer of Meiosis 2); RSC1, Remodel the Structure of Chromatin; CIN5, Chromosome INstability factor 5.</p><div class="sb-div caption-clear"></div></div></div><p id="p-49">This mapping approach was extended to nonsilencer partitioning origins (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/FigureS6.pdf">Figure S6</a>) which had been shown by direct ARS assays to contain a region(s) required for partitioning (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5</a>). For each ARS, the region(s) predicted to contain a partitioning function were mapped as described for mini<em><a href="http://www.yeastgenome.org/locus/S000028493/overview">ARS318</a></em> in <a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7A</a>. The fragments used in the ARS assays (<a id="xref-fig-5-9" class="xref-fig" href="#F5">Figure 5</a>) are shown in this figure above the heat maps used to map the partitioning regions of these ARSs. From these analyses, <em><a href="http://www.yeastgenome.org/locus/S000077395/overview">ARS523</a></em> contains a region 5′ of its ORC binding site required for partitioning, while <em><a href="http://www.yeastgenome.org/locus/S000028444/overview">ARS1007</a></em> contains a region 3′ of its ORC binding site required for partitioning. <em><a href="http://www.yeastgenome.org/locus/S000028445/overview">ARS1018</a></em> contains two different regions on either side of the ORC binding site that contribute to its partitioning ability.</p><p id="p-50">We extended this mapping approach to the 23 additional nonsilencer partitioning origins that were defined as such because they contained at least one fragment within the bottom right corner of the heat map in <a id="xref-fig-6-17" class="xref-fig" href="#F6">Figure 6B</a> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/FigureS4.pdf">Figure S4</a>). In the total set of 26 nonsilencer ARSs, including the three shown in <a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7B</a>, 15 contain regions 3′ of the ORC site predicted to contribute to partitioning, while two contain only regions 5′ of the ORC site, and nine contain regions both 5′ and 3′ of the ORC binding site. In summary, this matching approach allowed us to map new partitioning regions in 26 nonsilencer ARSs and indicated that acquisition of partitioning ability was associated with elements both 3′ and 5′ of the ORC binding site.</p></div><div id="sec-16" class="subsection"><h3>Motifs within the partitioning regions defined sequence-specific DNA binding proteins enriched for particular cellular functions</h3><p id="p-51">Comprehensive motif analyses of the 26 nonsilencer partitioning ARSs identified in <a id="xref-fig-7-6" class="xref-fig" href="#F7">Figure 7C</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/FigureS4.pdf">Figure S4</a> revealed 74 discrete motifs within partitioning regions, and 26 discrete motifs within nonpartitioning regions of these ARSs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/TableS1.xlsx">Table S1</a>). Of the 74 putative partitioning motifs, 10 were identified in ≥ 10 of the 26 partitioning ARSs and were therefore called “high frequency” partitioning motifs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/TableS1.xlsx">Table S1</a>). Interestingly, while the <a href="http://www.yeastgenome.org/locus/S000001595/overview">ABF1</a> site was a high frequency partitioning motif, a <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site was not identified in these analyses, consistent with the observation that <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> and ORC site juxtaposition is a defining feature of the E-silencer ARSs. Regardless, these analyses suggested that multiple different sequence-specific DNA binding proteins might influence plasmid partitioning, perhaps some as strongly as the silencer <a href="http://www.yeastgenome.org/locus/S000005160/overview">Rap1</a>-based mechanism (<em>e.g.</em>, compare ARS assays in <a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-fig-5-10" class="xref-fig" href="#F5">Figure 5</a>).</p><p id="p-52">Because a large number of putative partitioning motifs were identified (n = 74) within the partitioning regions queried, we looked for shared features among the DNA binding proteins predicted to bind these motifs by using annotated data sets available on SGD. Gene Ontology (GO) analyses were performed using four different groups of sequence-specific DNA binding proteins. The first group was the entire set of sequence-specific DNA binding proteins defined on SGD (GO: 0043565, n = 256). All results from analyses of the next three selected groups were normalized to this group. The first selected group contained the DNA binding proteins predicted to bind the partitioning motifs we identified in the 26 nonsilencer origins (n = 74; “All” in <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8, B and C</a>). The second selected group contained only those DNA binding proteins predicted by the high frequency partitioning motifs (n = 10; “High frequency” in <a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8, B and C</a>), as described above. The third selected group contained the sequence-specific DNA binding proteins, except ORC, predicted to bind motifs found within the nonpartitioning regions of the 26 partitioning ARSs, as described above. Out of the 27,522 possible GO function categories, 24 were enriched at least twofold in either the high frequency partitioning motifs (n = 10) or all partitioning motif (n = 74) groups when normalized to all sequence-specific DNA binding proteins (n = 256) (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8B</a> shows eight selected GO functions and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/TableS2.xlsx">Table S2</a> includes the complete analyses). Notably, the 26 nonpartitioning motifs were not enriched in any GO category relative to the 256 sequence-specific DNA binding proteins and thus are not shown in <a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure 8B</a> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/TableS2.xlsx">Table S2</a>). Thus, for example, DNA binding proteins defined by the high frequency partitioning motifs were enriched 12-fold over all sequence-specific DNA binding proteins in the GO category “negative regulation of filamentous growth,” while the larger group of all partitioning motifs were enriched ∼threefold in this category. All together, these analyses indicated that the putative partitioning motifs were not a random group of sequence-specific DNA binding motifs, but potentially linked to negative regulation of RNA metabolism and particular growth strategies.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/993/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The sequence-specific DNA binding proteins identified by motifs within partitioning regions were enriched relative to all sequence-specific DNA binding proteins for particular functions, including negative regulation of RNA metabolism and growth-control strategies. (A) Depiction of mechanisms proposed to enhance Cen- (centromere) independent plasmid partitioning adapted from (Gehlen et al. 2011). Based on both mathematical modeling and direct experiments, a plasmid&#x2019;s association with the inner nuclear membrane can enhance Cen-independent plasmid partitioning between mother and daughter cells. The silencer Rap1-mediated mechanism that tethers silencers to the inner nuclear membrane is constitutive, but cell-cycle regulated (transient) association between a plasmid and the inner nuclear membrane could also enhance plasmid partitioning, assuming that the association occurred most efficiently in M-phase as cells divide. Physical association between a plasmid and telomeres (chromosomal/chromatin interactions) also enhances Cen-independent plasmid partitioning. (B) Selected Gene Ontology (GO) term analysis (Saccharomyces Genome Database) of high frequency partitioning motifs [i.e., motifs that were identified within at least 10 partitioning origins, n = 10 (see Table S1 and Table S2)] and all partitioning motifs (n = 74 motifs) identified within the partitioning regions of the 26 nonsilencer ARSs (autonomously replicating sequences, Figure S4). The indicated GO terms shown were enriched at least twofold and with a P-value confidence cut off of 0.01 for at least one of the collections of partitioning motifs [high frequency (n = 10) or all (n = 74)] relative to all motifs annotated as sites for sequence-specific binding DNA binding proteins (GO: 0043565, n = 256). As another control, motifs found within the partitioning ARSs but not within the mapped partitioning regions (n = 26, see Table S1, rows 3-28) were also examined. Because these motifs showed no enrichment in any GO category relative to sequence-specific DNA binding motifs (n = 256), they are not shown in the Figure. While only the GO terms associated with partitioning motifs enriched at least twofold at a P value &#x2265; 0.01 relative to all sequence-specific DNA binders are shown here, Table S2 lists the complete analyses of GO terms queried in these analyses. (C) A similar analysis was performed with selected Phenotypic Ontology (PO) terms. The following phenotypic categories were used in these analyses: 1, Colony appearance; 2, Chromosome and plasmid maintenance; 3, Intracellular transport; 4, Mitotic cell cycle; 5, Morphology; and 6, Development. The phenotype collections were used to annotate the motifs [high frequency partitioning motifs (n = 10), all partitioning motifs (n = 74), nonpartitioning motifs (n = 26), and all sequence-specific DNA binding proteins (n = 256; GO:0043565.)]. As in B, the enrichments for the categories are normalized to the fraction of all sequence-specific DNA binders (n = 256) with that given phenotype category. The nonpartitioning motifs showed no enrichment relative to all sequence-specific DNA binding motifs. Only PO terms associated with partitioning motifs enriched at least twofold at a P value &#x2265; 0.01 relative to all sequence-specific binders are shown. Table S3 lists the complete analyses of PO terms used in these analyses. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-865468913" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The sequence-specific DNA binding proteins identified by motifs within partitioning regions were enriched relative to all sequence-specific DNA binding proteins for particular functions, including negative regulation of RNA metabolism and growth-control strategies. (A) Depiction of mechanisms proposed to enhance Cen- (centromere) independent plasmid partitioning adapted from (Gehlen &lt;em&gt;et al.&lt;/em&gt; 2011). Based on both mathematical modeling and direct experiments, a plasmid&#x2019;s association with the inner nuclear membrane can enhance Cen-independent plasmid partitioning between mother and daughter cells. The silencer Rap1-mediated mechanism that tethers silencers to the inner nuclear membrane is constitutive, but cell-cycle regulated (transient) association between a plasmid and the inner nuclear membrane could also enhance plasmid partitioning, assuming that the association occurred most efficiently in M-phase as cells divide. Physical association between a plasmid and telomeres (chromosomal/chromatin interactions) also enhances Cen-independent plasmid partitioning. (B) Selected Gene Ontology (GO) term analysis (&lt;em&gt;Saccharomyces&lt;/em&gt; Genome Database) of high frequency partitioning motifs [&lt;em&gt;i.e.&lt;/em&gt;, motifs that were identified within at least 10 partitioning origins, n = 10 (see Table S1 and Table S2)] and all partitioning motifs (n = 74 motifs) identified within the partitioning regions of the 26 nonsilencer ARSs (autonomously replicating sequences, Figure S4). The indicated GO terms shown were enriched at least twofold and with a P-value confidence cut off of 0.01 for at least one of the collections of partitioning motifs [high frequency (n = 10) or all (n = 74)] relative to all motifs annotated as sites for sequence-specific binding DNA binding proteins (GO: 0043565, n = 256). As another control, motifs found within the partitioning ARSs but not within the mapped partitioning regions (n = 26, see Table S1, rows 3-28) were also examined. Because these motifs showed no enrichment in any GO category relative to sequence-specific DNA binding motifs (n = 256), they are not shown in the Figure. While only the GO terms associated with partitioning motifs enriched at least twofold at a P value &#x2265; 0.01 relative to all sequence-specific DNA binders are shown here, Table S2 lists the complete analyses of GO terms queried in these analyses. (C) A similar analysis was performed with selected Phenotypic Ontology (PO) terms. The following phenotypic categories were used in these analyses: 1, Colony appearance; 2, Chromosome and plasmid maintenance; 3, Intracellular transport; 4, Mitotic cell cycle; 5, Morphology; and 6, Development. The phenotype collections were used to annotate the motifs [high frequency partitioning motifs (n = 10), all partitioning motifs (n = 74), nonpartitioning motifs (n = 26), and all sequence-specific DNA binding proteins (n = 256; GO:0043565.)]. As in B, the enrichments for the categories are normalized to the fraction of all sequence-specific DNA binders (n = 256) with that given phenotype category. The nonpartitioning motifs showed no enrichment relative to all sequence-specific DNA binding motifs. Only PO terms associated with partitioning motifs enriched at least twofold at a P value &#x2265; 0.01 relative to all sequence-specific binders are shown. Table S3 lists the complete analyses of PO terms used in these analyses. &lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/993/F8.medium.gif" width="440" height="432"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/6/4/993/F8.medium.gif" width="440" height="432"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/993/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/993/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467779" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-53" class="first-child">The sequence-specific DNA binding proteins identified by motifs within partitioning regions were enriched relative to all sequence-specific DNA binding proteins for particular functions, including negative regulation of RNA metabolism and growth-control strategies. (A) Depiction of mechanisms proposed to enhance Cen- (centromere) independent plasmid partitioning adapted from (<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Gehlen <em>et al.</em> 2011</a>). Based on both mathematical modeling and direct experiments, a plasmid’s association with the inner nuclear membrane can enhance Cen-independent plasmid partitioning between mother and daughter cells. The silencer Rap1-mediated mechanism that tethers silencers to the inner nuclear membrane is constitutive, but cell-cycle regulated (transient) association between a plasmid and the inner nuclear membrane could also enhance plasmid partitioning, assuming that the association occurred most efficiently in M-phase as cells divide. Physical association between a plasmid and telomeres (chromosomal/chromatin interactions) also enhances Cen-independent plasmid partitioning. (B) Selected Gene Ontology (GO) term analysis (<em>Saccharomyces</em> Genome Database) of high frequency partitioning motifs [<em>i.e.</em>, motifs that were identified within at least 10 partitioning origins, n = 10 (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/TableS1.xlsx">Table S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/TableS2.xlsx">Table S2</a>)] and all partitioning motifs (n = 74 motifs) identified within the partitioning regions of the 26 nonsilencer ARSs (autonomously replicating sequences, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/FigureS4.pdf">Figure S4</a>). The indicated GO terms shown were enriched at least twofold and with a P-value confidence cut off of 0.01 for at least one of the collections of partitioning motifs [high frequency (n = 10) or all (n = 74)] relative to all motifs annotated as sites for sequence-specific binding DNA binding proteins (GO: 0043565, n = 256). As another control, motifs found within the partitioning ARSs but not within the mapped partitioning regions (n = 26, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/TableS1.xlsx">Table S1</a>, rows 3-28) were also examined. Because these motifs showed no enrichment in any GO category relative to sequence-specific DNA binding motifs (n = 256), they are not shown in the Figure. While only the GO terms associated with partitioning motifs enriched at least twofold at a P value ≥ 0.01 relative to all sequence-specific DNA binders are shown here, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/TableS2.xlsx">Table S2</a> lists the complete analyses of GO terms queried in these analyses. (C) A similar analysis was performed with selected Phenotypic Ontology (PO) terms. The following phenotypic categories were used in these analyses: 1, Colony appearance; 2, Chromosome and plasmid maintenance; 3, Intracellular transport; 4, Mitotic cell cycle; 5, Morphology; and 6, Development. The phenotype collections were used to annotate the motifs [high frequency partitioning motifs (n = 10), all partitioning motifs (n = 74), nonpartitioning motifs (n = 26), and all sequence-specific DNA binding proteins (n = 256; GO:0043565.)]. As in B, the enrichments for the categories are normalized to the fraction of all sequence-specific DNA binders (n = 256) with that given phenotype category. The nonpartitioning motifs showed no enrichment relative to all sequence-specific DNA binding motifs. Only PO terms associated with partitioning motifs enriched at least twofold at a P value ≥ 0.01 relative to all sequence-specific binders are shown. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/TableS3.xlsx">Table S3</a> lists the complete analyses of PO terms used in these analyses. </p><div class="sb-div caption-clear"></div></div></div><p id="p-54">We used the same strategy to assess Phenotypic Ontology terms (PO) as annotated in SGD except that, in contrast to the GO analysis, only selected phenotypes (84 in total) PO terms were queried within six general categories: mitotic cell cycle, intracellular transport, development, morphology, colony appearance, and chromosome/plasmid maintenance (see <a id="xref-fig-8-5" class="xref-fig" href="#F8">Figure 8C</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1/TableS3.xlsx">Table S3</a> for full analysis). If a particular sequence-specific DNA binding protein is associated with a specific phenotypic category, it means that experimental data provide evidence that the wild-type function of this protein contributes to the wild-type version of this phenotype. For example, <a href="http://www.yeastgenome.org/locus/S000001393/overview">Fkh1</a> and <a href="http://www.yeastgenome.org/locus/S000005012/overview">Fkh2</a> binding sites were among the collection of high frequency partitioning motifs (n = 10). Deletions of <em><a href="http://www.yeastgenome.org/locus/S000001393/overview">FKH1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000005012/overview">FKH2</a></em>, while viable, alter cell cycle progression, normal cell shape and colony morphology, and other phenotypes particularly sensitive to normal G2/M regulation, and therefore cause pseudohyphal-like growth (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Breeden 2000</a>). Thus, the PO mapping provides clues about the normal biological functions controlled by the sequence-specific DNA binding proteins identified based on motif analyses. As with the GO analyses, these PO analyses indicated that the partitioning motifs identified sequence-specific DNA binding proteins enriched for particular cellular functions, suggesting connections between these functions and the ability to partition plasmids.</p></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><p id="p-55">The ability of small plasmids to propagate efficiently in the model eukaryote <em>S. cerevisiae</em> has been instrumental in defining chromosomal control elements including gene promoters, DNA replication origins (a.k.a. ARSs), and centromeres. Adaptation of next-generation sequencing strategies to plasmid-based assays provides the opportunity to obtain information about these and other elements at levels of depth and breadth not previously possible. In this study, we used a competitive growth strategy coupled to high throughput sequencing to assay the competitive fitness of thousands of individual miniARS fragments simultaneously. We used this approach to try and disambiguate two fundamental contributors to efficient plasmid propagation: replication fitness and partitioning fitness. By comparing the competitive fitness behavior of fragments from a miniARS library cloned in two different plasmid backbones, one that contained a centromere (Cen) and one that lacked a centromere (Acen), we demonstrated that 26 of 112 yeast miniARS fragments contained previously undocumented putative plasmid-partitioning elements. We mapped chromosomal regions responsible for this ability and found that they contained 74 distinct motifs. Prior to this study, only one partitioning element had been defined—specifically the <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site present in transcriptional silencers and telomeres. Thus, this study describes an effective high throughput approach that can help distinguish between partitioning and replication functions that influence plasmid propagation, reveals that a large fraction of ARS fragments likely contain uncharacterized mechanisms for partitioning, and identifies a large number of candidate factors that might be capable of regulating partitioning.</p><div id="sec-18" class="subsection"><h3>A miniARS competition experiment identified silencers, ARSs known to contain a centromere-independent plasmid partitioning ability</h3><p id="p-56">The silencer-RAP1 site has a role in plasmid partitioning that reflects (<em>i.e.</em>, is essentially a by-product of) its chromosomal role in transcriptional silencing (Kimmerly and Rine 1987; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Kimmerly <em>et al.</em> 1988</a>; <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Longtine <em>et al.</em> 1992</a>, <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">1993</a>; <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Ansari and Gartenberg 1997</a>; <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Andrulis <em>et al.</em> 2002</a>). Specifically, data support a model where <a href="http://www.yeastgenome.org/locus/S000005160/overview">Rap1</a> bound to the silencer DNA interacts with <a href="http://www.yeastgenome.org/locus/S000002635/overview">Sir4</a>, a structural component of silent chromatin itself (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Moretti and Shore 2001</a>). <a href="http://www.yeastgenome.org/locus/S000002635/overview">Sir4</a> in turn interacts with <a href="http://www.yeastgenome.org/locus/S000004832/overview">Esc1</a>, a nuclear protein tightly associated with the INM (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Ansari and Gartenberg 1997</a>; <a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Andrulis <em>et al.</em> 2002</a>). While this series of protein–DNA and protein–protein interactions contribute to chromosomal-based transcriptional silencing, in a plasmid context it can tether a plasmid to the INM and, in doing so, provide for robust, centromere-independent plasmid partitioning.</p><p id="p-57">In the Acen miniARS competition experiment, many of the most competitive miniARS fragments mapped to the <span class="underline">E</span>ssential (E) silencers <em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000029655/overview">HMR</a></em>-E) and <em><a href="http://www.yeastgenome.org/locus/S000028477/overview">ARS301</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000029214/overview">HML</a></em>-E). While it is possible that the reported “rereplication” ability of <em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em> might contribute to this element’s exceptional behavior in the Acen miniARS competition, we propose that plasmid partitioning was likely a larger, more general factor for two reasons (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Green and Li 2005</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Richardson and Li 2014</a>). First, rereplication of <em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em> has only been observed in certain mutants, and our miniARS competitions were performed in wild type yeast. Second, silencer-mediated plasmid partitioning is well documented and could also explain the behavior of <em><a href="http://www.yeastgenome.org/locus/S000028477/overview">ARS301</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000028493/overview">ARS318</a></em> in our studies. The silencer miniARS-mut experiments supported this conclusion by showing that the silencer’s <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> sites, an element known to contribute to robust plasmid partitioning, and the ORC site, an element known to be essential for plasmid replication, contributed equally to the propagation of the E-silencer ARS fragments, even though the <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site is not essential for silencer-origin function. Finally, direct ARS assays confirmed that two different fragments with low competitive fitness in the Acen miniARS experiment actually possessed strong intrinsic replication efficiencies (based on their ARS activity on a Cen-containing plasmid), indicating that they lost competitive fitness in the Acen miniARS experiment when they lost <a href="http://www.yeastgenome.org/locus/S000005160/overview">Rap1</a>-mediated plasmid partitioning ability. Therefore, the Acen mini ARS competition selected for retention of a chromosomal element known to influence plasmid partitioning.</p><p id="p-58">The <em><a href="http://www.yeastgenome.org/locus/S000029655/overview">HMR</a></em>-I (<em><a href="http://www.yeastgenome.org/locus/S000028493/overview">ARS318</a></em>) silencer was also identified as a partitioning ARS in this study, with the partitioning ability mapping to the <em><a href="http://www.yeastgenome.org/locus/S000029655/overview">HMR</a></em>-I silencer <a href="http://www.yeastgenome.org/locus/S000001595/overview">ABF1</a> site. Though the silencer-ABF1 site is not as recognized as the <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site for its silencer-related partitioning ability, early studies of <em><a href="http://www.yeastgenome.org/locus/S000029655/overview">HMR</a></em>-E indicated it could influence Cen-independent inheritance and a later study also connected it to partitioning (Kimmerly and Rine 1987; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Enomoto <em>et al.</em> 1994</a>). <em><a href="http://www.yeastgenome.org/locus/S000029655/overview">HMR</a></em>-I (<em><a href="http://www.yeastgenome.org/locus/S000028493/overview">ARS318</a></em>) is a weaker silencer than either <em><a href="http://www.yeastgenome.org/locus/S000029655/overview">HMR</a></em>-E (<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em>) or <em><a href="http://www.yeastgenome.org/locus/S000029214/overview">HML</a></em>-E (<em><a href="http://www.yeastgenome.org/locus/S000028477/overview">ARS301</a></em>), and in this study the mini<em><a href="http://www.yeastgenome.org/locus/S000028493/overview">ARS318</a></em> fragments with the highest competitive fitness value in the Acen-miniARS still showed less competitive fitness than the most competitive mini<em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em> or mini<em><a href="http://www.yeastgenome.org/locus/S000028477/overview">ARS301</a></em> fragments. The mini<em><a href="http://www.yeastgenome.org/locus/S000028493/overview">ARS318</a></em> partitioning probably works through the same <a href="http://www.yeastgenome.org/locus/S000002635/overview">Sir4</a>-<a href="http://www.yeastgenome.org/locus/S000004832/overview">Esc1</a>-mediated mechanism, but <em><a href="http://www.yeastgenome.org/locus/S000029655/overview">HMR</a></em>-I is simply less effective at binding <a href="http://www.yeastgenome.org/locus/S000002635/overview">Sir4</a>, which would also explain why it functions as a weaker silencer.</p><p id="p-59">The silencer data raised the possibility that the Acen miniARS experiment had the potential to identify other as yet unknown partitioning elements in the yeast genome and, indeed, additional direct ARS assays showed that several other ARSs represented in the library likely contained as yet uncharacterized partitioning ability. In particular, <em><a href="http://www.yeastgenome.org/locus/S000077395/overview">ARS523</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000028444/overview">ARS1007</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000028445/overview">ARS1018</a></em> each contained partitioning region(s) based on the regions’ importance in an Acen plasmid but their complete dispensability in a Cen plasmid. That is, these regions of these nonsilencer ARSs behaved like the <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> site within <em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em>, in that they were not important for the intrinsic replication function of their associated ARSs. Motif analyses identified several distinct sequence motifs in the partitioning regions in these ARSs but none of these matched a <a href="http://www.yeastgenome.org/locus/S000005160/overview">RAP1</a> or <a href="http://www.yeastgenome.org/locus/S000001595/overview">ABF1</a> site, suggesting that silencer independent mechanisms for plasmid partitioning exist. Importantly, these ARSs’ partitioning regions functioned as well as the silencer <em><a href="http://www.yeastgenome.org/locus/S000028492/overview">ARS317</a></em> <a href="http://www.yeastgenome.org/locus/S000005160/overview">Rap1</a>-mediated partitioning ability, based on the PLRs determined from standard ARS assays. These locus-specific experiments, together with the silencer studies, formed the strong proof-of-principle foundation for the identification of 74 new candidate-partitioning motifs by comparing fragment competitive fitness in the Acen- and Cen-miniARS competition experiments.</p><p id="p-60">Numerical simulations also supported the conclusion that partitioning ability is an important element in competitive fitness (<em>i.e.</em>, ARS efficiency) in a Acen miniARS experiment, primarily because individual yeast cells can only tolerate so many copies of a given plasmid before the replication fitness of the plasmid is reduced (expressed as a “crowding penalty”). The assumption of such a penalty is supported by recent studies that show that the large fraction of origins present in the rDNA repeats (representing up to a third of all chromosomal origins in the yeast genome, depending on rDNA copy number) functionally compete with nonrDNA chromosomal origins for limiting origin activation factors (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Kwan <em>et al.</em> 2013</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Yoshida <em>et al.</em> 2014</a>). In addition, retention of plasmids in mother cells contributes to mother cell aging, another phenomenon that would contribute to the crowding penalty parameter and suppress plasmid propagation in the cell population, as measured in the miniARS experiments described here (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Denoth-Lippuner <em>et al.</em> 2014</a>). Thus, it is easy to imagine that a plasmid that fails to partition efficiently will ultimately exhibit a lower average replication fitness in an Acen miniARS competition experiment, even if it contains an intrinsically efficient replication element.</p><p id="p-61">Our data provide evidence that some ARSs must have stronger partitioning abilities compared to others and that this can be highly influential to competitive fitness in the Acen miniARS experiment. Our general approach to identify nonsilencer ARSs with as yet uncharacterized plasmid partitioning functions was to compare miniARS fragment behavior in Acen miniARS and Cen miniARS experiments. Thus, we exploited the well-established function of the centromere in providing for distribution of plasmids between mother and daughter cells. However, it is important to note that recent studies show that chromosomal centromeres possess an additional function—the ability to enhance replication timing of neighboring origins—because a kinetochore protein recruits a regulatory subunit of S-phase kinase, <a href="http://www.yeastgenome.org/locus/S000002459/overview">Dbf4</a>, thus enhancing the concentration of this kinase near centromeric origins (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Pohl <em>et al.</em> 2012</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Natsume <em>et al.</em> 2013</a>). Thus, while the 26 partitioning origins were equally represented by both later and early activating chromosomal origins, suggesting no obvious bias for either type of origin in our approach, it must be acknowledged that some of the partitioning regions we mapped in our high throughput approach may also possess elements that modulate replication fitness. For example, even though partitioning ability is important for competitive fitness in the Acen miniARS experiment, any given “partitioning ARS” we have defined could, in theory, promote its partitioning via distributed elements along the fragment, and the “partitioning regions” we map may actually enhance the replication fitness in the Acen library. In this scenario, they become dispensable to high competitive fitness in the Cen library because the centromere now provides for enhanced replication fitness. This scenario is more complicated because it assumes that whatever replication enhancement is provided for by the putative partitioning region is completely redundant with the centromere’s specific mechanism. That is, the replication enhancement provided by centromeres could reasonably be predicted to “lift all boats” similarly, and therefore have no effect on relative replicative fitness differences between ARSs. Nevertheless, the recent discovery that centromeres possess a mechanism for enhancing origin activation underscores the complexity of elements and mechanisms that might modulate competitive fitness in a population-based experiment like the Acen miniARS competition. Thus, while the identification of the silencers and several lines of other published observations discussed in this study support the conclusion that plasmid partitioning regions can be identified in our high throughput approach, additional studies are essential to determine how to disambiguate “replication enhancers” (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Pohl <em>et al.</em> 2013</a>) from “core” replicative function via a high throughput approach.</p></div><div id="sec-19" class="subsection"><h3>Association with the INM: a general mechanism for plasmid partitioning</h3><p id="p-62">The influence of INM association on plasmid behavior reflects a fundamental feature of yeast cell division (<a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Gehlen <em>et al.</em> 2011</a>). Because yeast cells divide by budding, the dividing nucleus and its contents are constricted at the bud neck. Thus, segregating chromosomes and their attached microtubules create a block to the free diffusion of nucleoplasmic contents, such that retention of a freely diffusible plasmid in the mother cell is a default state favored by a 9:1 ratio. This mechanism is referred to as the morpho-kinetic contribution to Acen plasmid retention in mother cells. This mechanism accounts for the majority of Acen plasmid retention, although a more active SAGA-dependent mechanism has recently been described that is substantial enough to contribute substantially to mother cell aging (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Denoth-Lippuner <em>et al.</em> 2014</a>), which would enhance the effect of the crowding penalty for any poorly partitioning Acen plasmid in the Acen miniARS experiment. Regardless, artificially anchoring a plasmid to the INM enhances plasmid inheritance substantially without affecting plasmid copy number (<em>i.e.</em>, replication efficiency) (<a id="xref-ref-21-6" class="xref-bibr" href="#ref-21">Gehlen <em>et al.</em> 2011</a>). Mathematical modeling explains this phenomenon by showing that movement along the outer edge of the dividing nucleus (<em>i.e.</em>, through the lipid bilayer that constitutes the INM) bypasses the blockade at the constricted bud neck and enhances a plasmid’s ability to get into the daughter cell. Thus, anchoring a plasmid to the INM provides an active mechanism to promote plasmid distribution to the daughter cell.</p><p id="p-63">While the silencer <a href="http://www.yeastgenome.org/locus/S000005160/overview">Rap1</a>-INM tethering mechanism is part of a distinct, well-studied mechanism associated with a specific type of transcriptional repression, many links between transcriptional repression and the INM have been observed in multiple model organisms (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Zhao <em>et al.</em> 2009</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Taddei and Gasser 2012</a>; <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Towbin <em>et al.</em> 2013</a>). Thus, it was notable that we observed an enrichment of sequence specific DNA binding proteins predicted to bind motifs within partitioning regions in the GO category “negative regulation of transcription” and related GO categories. INM association may also be a general mechanism to optimize the conditional regulation of transcription—facilitating efficient conversion from the repressed to the activated state (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Green <em>et al.</em> 2012</a>). In this regard, among the high frequency partitioning motifs identified here, several were predicted to bind proteins required for conditional transcription, including <a href="http://www.yeastgenome.org/locus/S000001393/overview">Fkh1</a>, <a href="http://www.yeastgenome.org/locus/S000005012/overview">Fkh2</a>, <a href="http://www.yeastgenome.org/locus/S000006010/overview">Rlm1</a>, <a href="http://www.yeastgenome.org/locus/S000005970/overview">Dig1</a>, and <a href="http://www.yeastgenome.org/locus/S000002667/overview">Yap6</a> (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Watanabe <em>et al.</em> 1995</a>, <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">1997</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cook <em>et al.</em> 1996</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Dodou and Treisman 1997</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Breeden 2000</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Voth <em>et al.</em> 2007</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hanlon <em>et al.</em> 2011</a>). <a href="http://www.yeastgenome.org/locus/S000001393/overview">Fkh1</a> and <a href="http://www.yeastgenome.org/locus/S000005012/overview">Fkh2</a> proteins activate transcription of <em><a href="http://www.yeastgenome.org/locus/S000006323/overview">CLB2</a></em>-cluster genes in mid to late S-phase, but at other stages may repress or “antiactivate” target genes (<a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Voth <em>et al.</em> 2007</a>). Thus, perhaps some <a href="http://www.yeastgenome.org/locus/S000001393/overview">Fkh1</a>- and <a href="http://www.yeastgenome.org/locus/S000005012/overview">Fkh2</a>- bound chromosomal regions become associated with the INM during G2- M-phase as part of the transcriptionally repressed state. INM-anchoring-mediated plasmid partitioning would be most critical during M-phase, when the nucleus is actually dividing (<a id="xref-ref-21-7" class="xref-bibr" href="#ref-21">Gehlen <em>et al.</em> 2011</a>). Such a Fkh-dependent INM association may also relate to the origin-clustering function reported for the Fkh proteins (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Knott <em>et al.</em> 2012</a>). Regardless of the specific mechanism, enrichment of putative partitioning factors in the GO category “negative regulation of transcription” and PO categories related to G2- M-phase regulation may reflect links between these factors and the INM. In addition, several putative partitioning factors identified by high frequency partitioning motifs, including <a href="http://www.yeastgenome.org/locus/S000005639/overview">Azf1</a>, <a href="http://www.yeastgenome.org/locus/S000001019/overview">Rim1</a>, <a href="http://www.yeastgenome.org/locus/S000005970/overview">Dig1</a>, and <a href="http://www.yeastgenome.org/locus/S000002667/overview">Yap6</a>, physically interact with karyopherins and/or membrane proteins, suggesting links between these gene-regulatory proteins and the INM (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Ito <em>et al.</em> 2001</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gavin <em>et al.</em> 2002</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Ptacek <em>et al.</em> 2005</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Krogan <em>et al.</em> 2006</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Yu <em>et al.</em> 2008</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Tonikian <em>et al.</em> 2009</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Breitkreutz <em>et al.</em> 2010</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Wang <em>et al.</em> 2012</a>).</p></div><div id="sec-20" class="subsection"><h3>Sequence-specific DNA binding proteins that normally enable chromosome–chromosome- or chromosome–kintechore/spindle-interactions could also contribute to plasmid partitioning</h3><p id="p-64">Linking a plasmid to the INM is only one general mechanism capable of enhancing plasmid partitioning. For example, artificially establishing an association between a plasmid and telomeres via a fusion protein also enhances plasmid partitioning (<a id="xref-ref-21-8" class="xref-bibr" href="#ref-21">Gehlen <em>et al.</em> 2011</a>). These and other experiments support a model in which tethering a plasmid to a segregating chromosome helps it to “hitch a ride” to the daughter cell. While most if not all sequence-specific DNA binding proteins may facilitate some level of chromosome–chromosome interactions, it is intriguing that the <a href="http://www.yeastgenome.org/locus/S000001393/overview">Fkh1</a> and <a href="http://www.yeastgenome.org/locus/S000005012/overview">Fkh2</a> proteins, identified here as high frequency partitioning factors, were recently implicated in establishing chromosome–chromosome interactions via 3D chromosome-capture assays (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Knott <em>et al.</em> 2012</a>). If such a chromosomal activity could be captured on a plasmid, it would be expected to enhance plasmid partitioning to daughter cells. In addition, chromosome-independent components of the segregation machinery at the nuclear bud neck could also be relevant to plasmid partitioning. In this regard, it is notable that the predicted high frequency putative partitioning factors <a href="http://www.yeastgenome.org/locus/S000005970/overview">Dig1</a>, <a href="http://www.yeastgenome.org/locus/S000001393/overview">Fkh1</a>, and <a href="http://www.yeastgenome.org/locus/S000002667/overview">Yap6</a> physically associate with components of the kinetochore and or/spindle machinery (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Wong <em>et al.</em> 2007</a>; <a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Yu <em>et al.</em> 2008</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akiyoshi <em>et al.</em> 2010</a>; <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Wang <em>et al.</em> 2012</a>).</p><p id="p-65">A deeper understanding of the mechanisms that modulate plasmid partitioning in yeast may provide novel insights into the cellular activities they reflect. Indeed, silencer-mediated partitioning has provided important tools and insights into the relationships between heterochromatin, telomere structure, and chromosomal architecture in the nucleus. An intriguing hypothesis concerning plasmid cell retention in mother cells is that mechanisms that favor this retention may act as a form of “genomic immunity,” which discourages the propagation of foreign DNA in the cell population (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Denoth-Lippuner <em>et al.</em> 2014</a>). The cost to the mother cell may be enhanced replicative aging and thus irrelevance in the population—the mother’s kill for her brood. Nonchromosomal DNA—foreign or otherwise—capable of enhancing its partitioning to daughter cells, exploiting any one of the possible mechanisms discussed above, can effectively compete for genetic influence on the cell population. While the focus in this study is yeast, the concepts are relevant well beyond this popular model organism. For example, many latent mammalian viruses exploit interactions with cellular host proteins to effectively propagate (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Botchan 2004</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">You <em>et al.</em> 2004</a>). In terms of designing synthetic chromosomes, it will important to learn what effects chromosomal tethering to the INM or other chromosomes might have on chromosome stability. In summary, this study presents a robust methodological framework and new data for future examination of the DNA elements and cognate binding proteins that modulate plasmid inheritance, and ultimately the chromosomal roles this function reflects.</p></div></div><div class="section" id="sec-21"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-66">Special gratitude to S. Fields (Department of Genome Sciences, University of Washington, Seattle, and Howard Hughes Medical Institute) for the initial inspiration for the project, as well as B. Brewer, M. Raghuraman, and D. Fowler (Department of Genome Sciences, University of Washington, Seattle) for helpful discussions. We also thank M. Harrison (Department of Biomolecular Chemistry, University of Wisconsin-Madison) for carefully reading, discussing, and editing the manuscript, and C. Hull, M. Sheets and A. Dummer (Department of Biomolecular Chemistry, University of Wisconsin-Madison) for helpful discussions. This study was supported by National Institutes of Health funding (GM056890 to C.A.F.) and (P41 GM103533 to M.J.D.), and the National Science Foundation (NSF) (1243710 to M.J.D. and I.L.). M.J.D. is a Rita Allen Foundation Scholar and a Senior Fellow in the Genetic Networks program at the Canadian Institute for Advanced Research. C.A.F. acknowledges an Education and Innovation Award from the University of Wisconsin-Madison for the Team Origin Undergraduate Project. G.C. was funded by NSF grant DMS-1412643. </p><p id="p-67">Author contributions: T.H. designed and executed most of the experiments, performed data analyses, designed and generated all figures, trained and directed undergraduate members of Team Origin, and edited the manuscript. I.L. generated the original Acen and the ARSmut libraries, performed the Acen and ARSmut competitive growth experiments and deep sequencing analyses, advised on various experimental designs, and edited the manuscript. G.C. wrote the code for <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a> and performed the numerical simulations. C.B. and T.M. cloned relevant fragments and performed and analyzed the ARS assays in <a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-fig-5-11" class="xref-fig" href="#F5">Figure 5</a>. C.B. and K.J. generated the CEN-miniARS library, and K.J. performed data analyses. C.A.F. and M.J.D. designed and guided the study and wrote the manuscript. All authors have read and approved the manuscript and report no conflicting interests.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-68">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027904/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-69"><a class="rev-xref" href="#xref-fn-3-1">↵</a><span class="fn-label">2</span> These authors are joint senior authors.</p></li><li class="fn" id="fn-4"><p id="p-70"><em>Communicating editor: J. Rine</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-11-18"><span class="received-label">Received </span>November 18, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-02-05"><span class="accepted-label">Accepted </span>February 5, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Hoggard <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.1" data-doi="10.1038/nature09594"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akiyoshi</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarangapani</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powers</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reichow</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Tension directly stabilizes reconstituted kinetochore-microtubule attachments.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">468</span>: <span class="cit-fpage">576</span>–<span class="cit-lpage">579</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAkiyoshi%26rft.auinit1%253DB.%26rft.volume%253D468%26rft.issue%253D7323%26rft.spage%253D576%26rft.epage%253D579%26rft.atitle%253DTension%2Bdirectly%2Bstabilizes%2Breconstituted%2Bkinetochore-microtubule%2Battachments.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09594%26rft_id%253Dinfo%253Apmid%252F21107429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09594&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21107429&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284584200045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.2" data-doi="10.1038/29100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andrulis</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neiman</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zappulla</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sternglanz</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Perinuclear localization of chromatin facilitates transcriptional silencing.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">394</span>: <span class="cit-fpage">592</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAndrulis%26rft.auinit1%253DE.%2BD.%26rft.volume%253D394%26rft.issue%253D6693%26rft.spage%253D592%26rft.epage%253D595%26rft.atitle%253DPerinuclear%2Blocalization%2Bof%2Bchromatin%2Bfacilitates%2Btranscriptional%2Bsilencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F29100%26rft_id%253Dinfo%253Apmid%252F9707122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/29100&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9707122&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000075238700052&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.3" data-doi="10.1128/MCB.22.23.8292-8301.2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andrulis</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zappulla</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ansari</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perrod</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laiosa</span>  <span class="cit-name-given-names">C. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Esc1, a nuclear periphery protein required for Sir4-based plasmid anchoring and partitioning.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">8292</span>–<span class="cit-lpage">8301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DAndrulis%26rft.auinit1%253DE.%2BD.%26rft.volume%253D22%26rft.issue%253D23%26rft.spage%253D8292%26rft.epage%253D8301%26rft.atitle%253DEsc1%252C%2Ba%2BNuclear%2BPeriphery%2BProtein%2BRequired%2Bfor%2BSir4-Based%2BPlasmid%2BAnchoring%2Band%2BPartitioning%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.22.23.8292-8301.2002%26rft_id%253Dinfo%253Apmid%252F12417731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8yMy84MjkyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.4" data-doi="10.1128/MCB.17.12.7061"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ansari</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gartenberg</span>  <span class="cit-name-given-names">M. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The yeast silent information regulator Sir4p anchors and partitions plasmids.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">7061</span>–<span class="cit-lpage">7068</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DAnsari%26rft.auinit1%253DA.%26rft.volume%253D17%26rft.issue%253D12%26rft.spage%253D7061%26rft.epage%253D7068%26rft.atitle%253DThe%2Byeast%2Bsilent%2Binformation%2Bregulator%2BSir4p%2Banchors%2Band%2Bpartitions%2Bplasmids%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.17.12.7061%26rft_id%253Dinfo%253Apmid%252F9372937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxNy8xMi83MDYxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.5" data-doi="10.1038/357128a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stillman</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">ATP-dependent recognition of eukaryotic origins of DNA replication by a multiprotein complex.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">357</span>: <span class="cit-fpage">128</span>–<span class="cit-lpage">134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBell%26rft.auinit1%253DS.%2BP.%26rft.volume%253D357%26rft.issue%253D6374%26rft.spage%253D128%26rft.epage%253D134%26rft.atitle%253DATP-dependent%2Brecognition%2Bof%2Beukaryotic%2Borigins%2Bof%2BDNA%2Breplication%2Bby%2Ba%2Bmultiprotein%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F357128a0%26rft_id%253Dinfo%253Apmid%252F1579162%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/357128a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1579162&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HU12200048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.6" data-doi="10.1146/annurev-genet-120213-092033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">K. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Centromeric Heterochromatin: The Primordial Segregation Machine.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">457</span>–<span class="cit-lpage">484</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BGenet.%26rft.volume%253D48%26rft.spage%253D457%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genet-120213-092033%26rft_id%253Dinfo%253Apmid%252F25251850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genet-120213-092033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25251850&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.7" data-doi="10.1016/S0092-8674(04)00410-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Botchan</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Hitchhiking without Covalent Integration.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">280</span>–<span class="cit-lpage">281</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBotchan%26rft.auinit1%253DM.%26rft.volume%253D117%26rft.issue%253D3%26rft.spage%253D280%26rft.epage%253D281%26rft.atitle%253DHitchhiking%2Bwithout%2Bcovalent%2Bintegration.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252804%252900410-6%26rft_id%253Dinfo%253Apmid%252F15109488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(04)00410-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15109488&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221176400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.8" data-doi="10.1016/S0960-9822(00)00634-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breeden</span>  <span class="cit-name-given-names">L. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Cyclin transcription: Timing is everything.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R586</span>–<span class="cit-lpage">R588</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DBreeden%26rft.auinit1%253DL.%2BL.%26rft.volume%253D10%26rft.issue%253D16%26rft.spage%253DR586%26rft.epage%253DR588%26rft.atitle%253DCyclin%2Btranscription%253A%2BTiming%2Bis%2Beverything.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0960-9822%252800%252900634-5%26rft_id%253Dinfo%253Apmid%252F10985373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0960-9822(00)00634-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10985373&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089121000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.993.9"><div class="cit-metadata unstructured">Breitkreutz, A., H. Choi, J. R. Sharom, L. Boucher, V. Neduva <em>et al.</em>, 2010 A Global Protein Kinase and Phosphatase Interaction Network in Yeast. Science 328: 1043–1046.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.10" data-doi="10.1128/MCB.00206-08"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Theis</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nieduszynski</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newlon</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Analysis of Chromosome III Replicators Reveals an Unusual Structure for the ARS318 Silencer Origin and a Conserved WTW Sequence within the Origin Recognition Complex Binding Site.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">5071</span>–<span class="cit-lpage">5081</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.00206-08%26rft_id%253Dinfo%253Apmid%252F18573888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyOC8xNi81MDcxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.11" data-doi="10.1093/nar/gkr301"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoggard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fox</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">High-resolution analysis of four efficient yeast replication origins reveals new insights into the ORC and putative MCM binding elements.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">6523</span>–<span class="cit-lpage">6535</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr301%26rft_id%253Dinfo%253Apmid%252F21558171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzOS8xNS82NTIzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.12" data-doi="10.1101/gad.10.22.2831"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bardwell</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kron</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorner</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Two novel targets of the MAP kinase Kss1 are negative regulators of invasive growth in the yeast Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">2831</span>–<span class="cit-lpage">2848</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DCook%26rft.auinit1%253DJ%2BG%26rft.volume%253D10%26rft.issue%253D22%26rft.spage%253D2831%26rft.epage%253D2848%26rft.atitle%253DTwo%2Bnovel%2Btargets%2Bof%2Bthe%2BMAP%2Bkinase%2BKss1%2Bare%2Bnegative%2Bregulators%2Bof%2Binvasive%2Bgrowth%2Bin%2Bthe%2Byeast%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.10.22.2831%26rft_id%253Dinfo%253Apmid%252F8918885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjEwLzIyLzI4MzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvOTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.13" data-doi="10.1016/j.molcel.2008.02.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crampton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pappas</span>  <span class="cit-name-given-names">D. L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frisch</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinreich</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">An ARS element inhibits DNA replication through a SIR2-dependent mechanism.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">156</span>–<span class="cit-lpage">166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DCrampton%26rft.auinit1%253DA.%26rft.volume%253D30%26rft.issue%253D2%26rft.spage%253D156%26rft.epage%253D166%26rft.atitle%253DAn%2BARS%2Belement%2Binhibits%2BDNA%2Breplication%2Bthrough%2Ba%2BSIR2-dependent%2Bmechanism.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2008.02.019%26rft_id%253Dinfo%253Apmid%252F18439895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2008.02.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18439895&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255410200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.14" data-doi="10.7554/eLife.03790"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Denoth-Lippuner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krzyzanowski</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stober</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barral</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Role of SAGA in the asymmetric segregation of DNA circles during yeast ageing.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e03790</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.03790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZWxpZmUiO3M6NToicmVzaWQiO3M6MTA6IjMvMC9lMDM3OTAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvOTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.15" data-doi="10.1128/MCB.17.4.1848"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dodou</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Treisman</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The Saccharomyces cerevisiae MADS-box transcription factor Rlm1 is a target for the Mpk1 mitogen-activated protein kinase pathway.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1848</span>–<span class="cit-lpage">1859</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DDodou%26rft.auinit1%253DE.%26rft.volume%253D17%26rft.issue%253D4%26rft.spage%253D1848%26rft.epage%253D1859%26rft.atitle%253DThe%2BSaccharomyces%2Bcerevisiae%2BMADS-box%2Btranscription%2Bfactor%2BRlm1%2Bis%2Ba%2Btarget%2Bfor%2Bthe%2BMpk1%2Bmitogen-activated%2Bprotein%2Bkinase%2Bpathway%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.17.4.1848%26rft_id%253Dinfo%253Apmid%252F9121433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE3LzQvMTg0OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC85OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.16" data-doi="10.1371/journal.pgen.0020141"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Donato</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chung</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tye</span>  <span class="cit-name-given-names">B. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genome-wide hierarchy of replication origin usage in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DDonato%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D2%26rft.issue%253D9%26rft.spage%253De141%26rft.epage%253De141%26rft.atitle%253DGenome-wide%2Bhierarchy%2Bof%2Breplication%2Borigin%2Busage%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020141%26rft_id%253Dinfo%253Apmid%252F16965179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020141&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16965179&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.17" data-doi="10.1101/gad.1913210"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eaton</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galani</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacAlpine</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Conserved nucleosome positioning defines replication origins.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">748</span>–<span class="cit-lpage">753</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1913210%26rft_id%253Dinfo%253Apmid%252F20351051%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMjQvOC83NDgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvOTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Enomoto</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Longtine</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Enhancement of Telomere-Plasmid Segregation by the X-Telomere Associated Sequence in Saccharomyces Cerevisiae Involves Sir2, Sir3, Sir4 and Abf1.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">757</span>–<span class="cit-lpage">767</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DEnomoto%26rft.auinit1%253DS.%26rft.volume%253D136%26rft.issue%253D3%26rft.spage%253D757%26rft.epage%253D767%26rft.atitle%253DEnhancement%2Bof%2Btelomere-plasmid%2Bsegregation%2Bby%2Bthe%2BX-telomere%2Bassociated%2Bsequence%2Bin%2BSaccharomyces%2Bcerevisiae%2Binvolves%2BSIR2%252C%2BSIR3%252C%2BSIR4%2Band%2BABF1.%26rft_id%253Dinfo%253Apmid%252F8005431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM2LzMvNzU3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.19" data-doi="10.1038/ncb1358"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collingwood</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boeck</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fox</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alvino</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genomic mapping of single-stranded DNA in hydroxyurea-challenged yeasts identifies origins of replication.</span> <abbr class="cit-jnl-abbrev">Nat. Cell Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">148</span>–<span class="cit-lpage">155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCell%2BBiology%26rft.stitle%253DNature%2BCell%2BBiology%26rft.aulast%253DFeng%26rft.auinit1%253DW.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D148%26rft.epage%253D155%26rft.atitle%253DGenomic%2Bmapping%2Bof%2Bsingle-stranded%2BDNA%2Bin%2Bhydroxyurea-challenged%2Byeasts%2Bidentifies%2Borigins%2Bof%2Breplication.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb1358%26rft_id%253Dinfo%253Apmid%252F16429127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncb1358&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16429127&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235059800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.20" data-doi="10.1038/415141a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gavin</span>  <span class="cit-name-given-names">A.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bösche</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krause</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grandi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marzioch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Functional organization of the yeast proteome by systematic analysis of protein complexes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">415</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGavin%26rft.auinit1%253DA.%2BC.%26rft.volume%253D415%26rft.issue%253D6868%26rft.spage%253D141%26rft.epage%253D147%26rft.atitle%253DFunctional%2Borganization%2Bof%2Bthe%2Byeast%2Bproteome%2Bby%2Bsystematic%2Banalysis%2Bof%2Bprotein%2Bcomplexes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F415141a%26rft_id%253Dinfo%253Apmid%252F11805826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/415141a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11805826&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173159300033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.21" data-doi="10.1016/j.cub.2010.12.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gehlen</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagai</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimada</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meister</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taddei</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Nuclear Geometry and Rapid Mitosis Ensure Asymmetric Episome Segregation in Yeast.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DGehlen%26rft.auinit1%253DL.%2BR.%26rft.volume%253D21%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D33%26rft.atitle%253DNuclear%2Bgeometry%2Band%2Brapid%2Bmitosis%2Bensure%2Basymmetric%2Bepisome%2Bsegregation%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2010.12.016%26rft_id%253Dinfo%253Apmid%252F21194950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2010.12.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21194950&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.22" data-doi="10.1083/jcb.134.6.1349"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gotta</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laroche</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Formenton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maillet</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scherthan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The clustering of telomeres and colocalization with Rap1, Sir3, and Sir4 proteins in wild-type Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">1349</span>–<span class="cit-lpage">1363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BCell%2BBiology%26rft.stitle%253DJCB%26rft.aulast%253DGotta%26rft.auinit1%253DM.%26rft.volume%253D134%26rft.issue%253D6%26rft.spage%253D1349%26rft.epage%253D1363%26rft.atitle%253DThe%2Bclustering%2Bof%2Btelomeres%2Band%2Bcolocalization%2Bwith%2BRap1%252C%2BSir3%252C%2Band%2BSir4%2Bproteins%2Bin%2Bwild-type%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.134.6.1349%26rft_id%253Dinfo%253Apmid%252F8830766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjEwOiIxMzQvNi8xMzQ5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.23" data-doi="10.1091/mbc.E04-09-0833"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Loss of rereplication control in Saccharomyces cerevisiae results in extensive DNA damage.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">421</span>–<span class="cit-lpage">432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E04-09-0833%26rft_id%253Dinfo%253Apmid%252F15537702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjE2LzEvNDIxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.993.24"><div class="cit-metadata unstructured">Green, E. M., Y. Jiang, R. Joyner, and K. Weis, 2012 A negative feedback loop at the nuclear periphery regulates GAL gene expression. Mol. Biol. Cell 23: 1367–1375.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.993.25"><div class="cit-metadata unstructured">Hanlon, S. E., J. M. Rizzo, D. C. Tatomer, J. D. Lieb, and M. J. Buck, 2011 The Stress Response Factors Yap6, Cin5, Phd1, and Skn7 Direct Targeting of the Conserved Co-Repressor Tup1-Ssn6 in S. cerevisiae. PLoS One 6: e19060.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.26" data-doi="10.1371/journal.pgen.1003798"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoggard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shor</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nieduszynski</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fox</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A Link between ORC-origin binding mechanisms and origin activation time revealed in budding yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003798</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003798%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003798%26rft_id%253Dinfo%253Apmid%252F24068963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003798&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24068963&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">R. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kowalski</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">A DNA unwinding element and an ARS consensus comprise a replication origin within a yeast chromosome.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">4521</span>–<span class="cit-lpage">4531</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft.volume%253D12%26rft.spage%253D4521%26rft_id%253Dinfo%253Apmid%252F8223462%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8223462&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.28" data-doi="10.1093/nar/24.5.816"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">R. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kowalski</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Multiple DNA elements in ARS305 determine replication origin activity in a yeast chromosome.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">816</span>–<span class="cit-lpage">823</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F24.5.816%26rft_id%253Dinfo%253Apmid%252F8600446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjI0LzUvODE2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.29" data-doi="10.1073/pnas.061034498"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ito</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiba</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozawa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshida</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hattori</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A comprehensive two-hybrid analysis to explore the yeast protein interactome.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">4569</span>–<span class="cit-lpage">4574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.061034498%26rft_id%253Dinfo%253Apmid%252F11283351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5OC84LzQ1NjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvOTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimmerly</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buchman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kornberg</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rine</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Roles of two DNA-binding factors in replication, segregation and transcriptional repression mediated by a yeast silencer.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">2241</span>–<span class="cit-lpage">2253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft.volume%253D7%26rft.spage%253D2241%26rft_id%253Dinfo%253Apmid%252F3046937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=3046937&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.31" data-doi="10.1101/gad.1784309"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viggiani</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tavare</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aparicio</span>  <span class="cit-name-given-names">O. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome-wide replication profiles indicate an expansive role for Rpd3L in regulating replication initiation timing or efficiency, and reveal genomic loci of Rpd3 function in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1077</span>–<span class="cit-lpage">1090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DKnott%26rft.auinit1%253DS.%2BR.V.%26rft.volume%253D23%26rft.issue%253D9%26rft.spage%253D1077%26rft.epage%253D1090%26rft.atitle%253DGenome-wide%2Breplication%2Bprofiles%2Bindicate%2Ban%2Bexpansive%2Brole%2Bfor%2BRpd3L%2Bin%2Bregulating%2Breplication%2Binitiation%2Btiming%2Bor%2Befficiency%252C%2Band%2Breveal%2Bgenomic%2Bloci%2Bof%2BRpd3%2Bfunction%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1784309%26rft_id%253Dinfo%253Apmid%252F19417103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6OToiMjMvOS8xMDc3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.32" data-doi="10.1016/j.cell.2011.12.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peace</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ostrow</span>  <span class="cit-name-given-names">A. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rex</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Forkhead transcription factors establish origin timing and long-range clustering in S. cerevisiae.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">99</span>–<span class="cit-lpage">111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DKnott%26rft.auinit1%253DS.%2BR.%26rft.volume%253D148%26rft.issue%253D1-2%26rft.spage%253D99%26rft.epage%253D111%26rft.atitle%253DForkhead%2Btranscription%2Bfactors%2Bestablish%2Borigin%2Btiming%2Band%2Blong-range%2Bclustering%2Bin%2BS.%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.12.012%26rft_id%253Dinfo%253Apmid%252F22265405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.12.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22265405&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299540700017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.33" data-doi="10.1038/nature04670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krogan</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cagney</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Global landscape of protein complexes in the yeast Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">440</span>: <span class="cit-fpage">637</span>–<span class="cit-lpage">643</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKrogan%26rft.auinit1%253DN.%2BJ.%26rft.volume%253D440%26rft.issue%253D7084%26rft.spage%253D637%26rft.epage%253D643%26rft.atitle%253DGlobal%2Blandscape%2Bof%2Bprotein%2Bcomplexes%2Bin%2Bthe%2Byeast%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04670%26rft_id%253Dinfo%253Apmid%252F16554755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16554755&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236350400036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.993.34"><div class="cit-metadata unstructured">Kwan, E. X., E. J. Foss, S. Tsuchiyama, G. M. Alvino, L. Kruglyak <em>et al.</em>, 2013 A natural polymorphism in rDNA replication origins links origin activation with calorie restriction and lifespan. PLoS Genet. 9: e1003329.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.993.35"><div class="cit-metadata unstructured">Lefrançois, P., R. K. Auerbach, C. M. Yellman, G. S. Roeder, and M. Snyder, 2013 Centromere-Like Regions in the Budding Yeast Genome. PLoS Genet. 9: e1003209.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.36" data-doi="10.1101/gr.144659.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liachko</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Youngblood</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keich</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-resolution mapping, characterization, and optimization of autonomously replicating sequences in yeast.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">698</span>–<span class="cit-lpage">704</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.144659.112%26rft_id%253Dinfo%253Apmid%252F23241746%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIzLzQvNjk4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.37" data-doi="10.1016/S1097-2765(01)00151-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lipford</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Nucleosomes positioned by ORC facilitate the initiation of DNA replication.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DLipford%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D7%26rft.issue%253D1%26rft.spage%253D21%26rft.epage%253D30%26rft.atitle%253DNucleosomes%2Bpositioned%2Bby%2BORC%2Bfacilitate%2Bthe%2Binitiation%2Bof%2BDNA%2Breplication.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252801%252900151-4%26rft_id%253Dinfo%253Apmid%252F11172708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(01)00151-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11172708&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166601300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.38" data-doi="10.1128/MCB.12.5.1997"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Longtine</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Enomoto</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finstad</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Yeast telomere repeat sequence (TRS) improves circular plasmid segregation, and TRS plasmid segregation involves the RAP1 gene product.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1997</span>–<span class="cit-lpage">2009</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DLongtine%26rft.auinit1%253DM%2BS%26rft.volume%253D12%26rft.issue%253D5%26rft.spage%253D1997%26rft.epage%253D2009%26rft.atitle%253DYeast%2Btelomere%2Brepeat%2Bsequence%2B%2528TRS%2529%2Bimproves%2Bcircular%2Bplasmid%2Bsegregation%252C%2Band%2BTRS%2Bplasmid%2Bsegregation%2Binvolves%2Bthe%2BRAP1%2Bgene%2Bproduct.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.12.5.1997%26rft_id%253Dinfo%253Apmid%252F1569937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjEyLzUvMTk5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC85OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Longtine</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Enomoto</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finstad</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Telomere-Mediated Plasmid Segregation in Saccharomyces Cerevisiae Involves Gene Products Required for Transcriptional Repression at Silencers and Telomeres.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLongtine%26rft.auinit1%253DM.%2BS.%26rft.volume%253D133%26rft.issue%253D2%26rft.spage%253D171%26rft.epage%253D182%26rft.atitle%253DTelomere-mediated%2Bplasmid%2Bsegregation%2Bin%2BSaccharomyces%2Bcerevisiae%2Binvolves%2Bgene%2Bproducts%2Brequired%2Bfor%2Btranscriptional%2Brepression%2Bat%2Bsilencers%2Band%2Btelomeres.%26rft_id%253Dinfo%253Apmid%252F8436267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTMzLzIvMTcxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.993.40"><div class="cit-metadata unstructured">Marahrens, Y., and B. Stillman, 1992 A yeast chromosomal origin of DNA replication defined by multiple functional elements. Science 255: 817–823.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marahrens</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stillman</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Replicator dominance in a eukaryotic chromosome.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">3395</span>–<span class="cit-lpage">3400</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft.volume%253D13%26rft.spage%253D3395%26rft_id%253Dinfo%253Apmid%252F8045266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8045266&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.42" data-doi="10.1128/MCB.11.11.5648"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McNally</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rine</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">A synthetic silencer mediates SIR-dependent functions in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">5648</span>–<span class="cit-lpage">5659</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DMcNally%26rft.auinit1%253DF%2BJ%26rft.volume%253D11%26rft.issue%253D11%26rft.spage%253D5648%26rft.epage%253D5659%26rft.atitle%253DA%2Bsynthetic%2Bsilencer%2Bmediates%2BSIR-dependent%2Bfunctions%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.11.11.5648%26rft_id%253Dinfo%253Apmid%252F1922068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxMS8xMS81NjQ4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.43" data-doi="10.1128/MCB.21.23.8082-8094.2001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moretti</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shore</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Multiple interactions in Sir protein recruitment by Rap1p at silencers and telomeres in yeast.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">8082</span>–<span class="cit-lpage">8094</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DMoretti%26rft.auinit1%253DP.%26rft.volume%253D21%26rft.issue%253D23%26rft.spage%253D8082%26rft.epage%253D8094%26rft.atitle%253DMultiple%2BInteractions%2Bin%2BSir%2BProtein%2BRecruitment%2Bby%2BRap1p%2Bat%2BSilencers%2Band%2BTelomeres%2Bin%2BYeast%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.21.23.8082-8094.2001%26rft_id%253Dinfo%253Apmid%252F11689698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8yMy84MDgyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.44" data-doi="10.1101/gr.139477.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nieduszynski</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Conservation of replication timing reveals global and local regulation of replication origin activity.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1953</span>–<span class="cit-lpage">1962</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.139477.112%26rft_id%253Dinfo%253Apmid%252F22767388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8xMC8xOTUzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.45" data-doi="10.1101/gad.1906410"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shor</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huebert</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warren</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The conserved bromo-adjacent homology domain of yeast Orc1 functions in the selection of DNA replication origins within chromatin.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1418</span>–<span class="cit-lpage">1433</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DMuller%26rft.auinit1%253DP.%26rft.volume%253D24%26rft.issue%253D13%26rft.spage%253D1418%26rft.epage%253D1433%26rft.atitle%253DThe%2Bconserved%2Bbromo-adjacent%2Bhomology%2Bdomain%2Bof%2Byeast%2BOrc1%2Bfunctions%2Bin%2Bthe%2Bselection%2Bof%2BDNA%2Breplication%2Borigins%2Bwithin%2Bchromatin%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1906410%26rft_id%253Dinfo%253Apmid%252F20595233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjI0LzEzLzE0MTgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvOTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.46" data-doi="10.2217/epi.09.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagai</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heun</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gasser</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Roles for nuclear organization in the maintenance of genome stability.</span> <abbr class="cit-jnl-abbrev">Epigenomics</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEpigenomics%26rft.aulast%253DNagai%26rft.auinit1%253DS.%26rft.volume%253D2%26rft.issue%253D2%26rft.spage%253D289%26rft.epage%253D305%26rft.atitle%253DRoles%2Bfor%2Bnuclear%2Borganization%2Bin%2Bthe%2Bmaintenance%2Bof%2Bgenome%2Bstability.%26rft_id%253Dinfo%253Adoi%252F10.2217%252Fepi.09.49%26rft_id%253Dinfo%253Apmid%252F22121875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2217/epi.09.49&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22121875&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278299100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.47" data-doi="10.1016/j.molcel.2013.05.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Natsume</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Retkute</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gierlinski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Kinetochores coordinate pericentromeric cohesion and early DNA replication by Cdc7-Dbf4 kinase recruitment.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">661</span>–<span class="cit-lpage">674</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D50%26rft.spage%253D661%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2013.05.011%26rft_id%253Dinfo%253Apmid%252F23746350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2013.05.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23746350&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000320678500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.48" data-doi="10.1016/S0959-437X(05)80094-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Newlon</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Theis</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">The structure and function of yeast ARS elements.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">752</span>–<span class="cit-lpage">758</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DNewlon%26rft.auinit1%253DC.%2BS.%26rft.volume%253D3%26rft.issue%253D5%26rft.spage%253D752%26rft.epage%253D758%26rft.atitle%253DThe%2Bstructure%2Band%2Bfunction%2Bof%2Byeast%2BARS%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0959-437X%252805%252980094-2%26rft_id%253Dinfo%253Apmid%252F8274858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0959-437X(05)80094-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8274858&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.49" data-doi="10.1093/nar/gki539"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nieduszynski</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blow</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donaldson</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The requirement of yeast replication origins for pre-replication complex proteins is modulated by transcription.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">2410</span>–<span class="cit-lpage">2420</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DNieduszynski%26rft.auinit1%253DC.%2BA.%26rft.volume%253D33%26rft.issue%253D8%26rft.spage%253D2410%26rft.epage%253D2420%26rft.atitle%253DThe%2Brequirement%2Bof%2Byeast%2Breplication%2Borigins%2Bfor%2Bpre-replication%2Bcomplex%2Bproteins%2Bis%2Bmodulated%2Bby%2Btranscription%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki539%26rft_id%253Dinfo%253Apmid%252F15860777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjMzLzgvMjQxMCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC85OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.50" data-doi="10.1371/journal.pgen.1002677"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pohl</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brewer</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghuraman</span>  <span class="cit-name-given-names">M. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Functional centromeres determine the activation time of pericentric origins of DNA replication in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002677</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPohl%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253De1002677%26rft.epage%253De1002677%26rft.atitle%253DFunctional%2Bcentromeres%2Bdetermine%2Bthe%2Bactivation%2Btime%2Bof%2Bpericentric%2Borigins%2Bof%2BDNA%2Breplication%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002677%26rft_id%253Dinfo%253Apmid%252F22589733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002677&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22589733&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.51" data-doi="10.1534/g3.113.008250"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pohl</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolor</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fangman</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brewer</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghuraman</span>  <span class="cit-name-given-names">M. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A DNA sequence element that advances replication origin activation time in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1955</span>–<span class="cit-lpage">1963</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.008250%26rft_id%253Dinfo%253Apmid%252F24022751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjMvMTEvMTk1NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC85OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.52" data-doi="10.1038/nature04187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ptacek</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devgan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michaud</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Global analysis of protein phosphorylation in yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">438</span>: <span class="cit-fpage">679</span>–<span class="cit-lpage">684</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPtacek%26rft.auinit1%253DJ.%26rft.volume%253D438%26rft.issue%253D7068%26rft.spage%253D679%26rft.epage%253D684%26rft.atitle%253DGlobal%2Banalysis%2Bof%2Bprotein%2Bphosphorylation%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04187%26rft_id%253Dinfo%253Apmid%252F16319894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04187&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16319894&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233593100053&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.993.53"><div class="cit-metadata unstructured">Raghuraman, M. K., E. A. Winzeler, D. Collingwood, S. Hunt, L. Wodicka <em>et al.</em>, 2001 Replication dynamics of the yeast genome. Science 294: 115–121.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.54" data-doi="10.1073/pnas.92.6.2224"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stillman</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">The origin recognition complex interacts with a bipartite DNA binding site within yeast replicators.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">2224</span>–<span class="cit-lpage">2228</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DRao%26rft.auinit1%253DH.%26rft.volume%253D92%26rft.issue%253D6%26rft.spage%253D2224%26rft.epage%253D2228%26rft.atitle%253DThe%2Borigin%2Brecognition%2Bcomplex%2Binteracts%2Bwith%2Ba%2Bbipartite%2BDNA%2Bbinding%2Bsite%2Bwithin%2Byeast%2Breplicators.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.92.6.2224%26rft_id%253Dinfo%253Apmid%252F7892251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5Mi82LzIyMjQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvOTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.55" data-doi="10.1128/MCB.14.11.7643"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marahrens</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stillman</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Functional conservation of multiple elements in yeast chromosomal replicators.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">7643</span>–<span class="cit-lpage">7651</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DRao%26rft.auinit1%253DH%26rft.volume%253D14%26rft.issue%253D11%26rft.spage%253D7643%26rft.epage%253D7651%26rft.atitle%253DFunctional%2Bconservation%2Bof%2Bmultiple%2Belements%2Bin%2Byeast%2Bchromosomal%2Breplicators.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.14.11.7643%26rft_id%253Dinfo%253Apmid%252F7935478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxNC8xMS83NjQzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.993.56"><div class="cit-metadata unstructured">Richardson, C. D., and J. J. Li, 2014 Regulatory Mechanisms That Prevent Re-initiation of DNA Replication Can Be Locally Modulated at Origins by Nearby Sequence Elements. PLoS Genet. 10: e1004358.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.57" data-doi="10.1146/annurev.biochem.72.121801.161547"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rusche</span>  <span class="cit-name-given-names">L. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirchmaier</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rine</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The establishment, inheritance, and function of silenced chromatin in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">481</span>–<span class="cit-lpage">516</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.volume%253D72%26rft.spage%253D481%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.biochem.72.121801.161547%26rft_id%253Dinfo%253Apmid%252F12676793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.biochem.72.121801.161547&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12676793&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185092500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.58" data-doi="10.1038/343387a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">R. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Nucleosome positioning can affect the function of a cis-acting DNA element in vivo.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">343</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">389</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSimpson%26rft.auinit1%253DR.%2BT.%26rft.volume%253D343%26rft.issue%253D6256%26rft.spage%253D387%26rft.epage%253D389%26rft.atitle%253DNucleosome%2Bpositioning%2Bcan%2Baffect%2Bthe%2Bfunction%2Bof%2Ba%2Bcis-acting%2BDNA%2Belement%2Bin%2Bvivo.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F343387a0%26rft_id%253Dinfo%253Apmid%252F2405281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/343387a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2405281&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990CK91800068&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.59" data-doi="10.1093/nar/gkr1091"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Siow</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nieduszynska</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nieduszynski</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">OriDB, the DNA replication origin database updated and extended.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D682</span>–<span class="cit-lpage">D686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr1091%26rft_id%253Dinfo%253Apmid%252F22121216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MC9EMS9ENjgyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.60" data-doi="10.1534/genetics.112.140608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taddei</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gasser</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Structure and Function in the Budding Yeast Nucleus.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">107</span>–<span class="cit-lpage">129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTaddei%26rft.auinit1%253DA.%26rft.volume%253D192%26rft.issue%253D1%26rft.spage%253D107%26rft.epage%253D129%26rft.atitle%253DStructure%2Band%2BFunction%2Bin%2Bthe%2BBudding%2BYeast%2BNucleus%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.140608%26rft_id%253Dinfo%253Apmid%252F22964839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzEvMTA3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.61" data-doi="10.1128/MCB.14.11.7652"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Theis</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newlon</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Domain B of ARS307 contains two functional elements and contributes to chromosomal replication origin function.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">7652</span>–<span class="cit-lpage">7659</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DTheis%26rft.auinit1%253DJ%2BF%26rft.volume%253D14%26rft.issue%253D11%26rft.spage%253D7652%26rft.epage%253D7659%26rft.atitle%253DDomain%2BB%2Bof%2BARS307%2Bcontains%2Btwo%2Bfunctional%2Belements%2Band%2Bcontributes%2Bto%2Bchromosomal%2Breplication%2Borigin%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.14.11.7652%26rft_id%253Dinfo%253Apmid%252F7935479%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxNC8xMS83NjUyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.62" data-doi="10.1073/pnas.94.20.10786"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Theis</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newlon</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The ARS309 chromosomal replicator of Saccharomyces cerevisiae depends on an exceptional ARS consensus sequence.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">10786</span>–<span class="cit-lpage">10791</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DTheis%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D94%26rft.issue%253D20%26rft.spage%253D10786%26rft.epage%253D10791%26rft.atitle%253DThe%2BARS309%2Bchromosomal%2Breplicator%2Bof%2BSaccharomyces%2Bcerevisiae%2Bdepends%2Bon%2Ban%2Bexceptional%2BARS%2Bconsensus%2Bsequence%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.94.20.10786%26rft_id%253Dinfo%253Apmid%252F9380711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTQvMjAvMTA3ODYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvOTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.993.63"><div class="cit-metadata unstructured">Tonikian, R., X. Xin, C. P. Toret, D. Gfeller, C. Landgraf <em>et al.</em>, 2009 Bayesian Modeling of the Yeast SH3 Domain Interactome Predicts Spatiotemporal Dynamics of Endocytosis Proteins. PLoS Biol. 7: e1000218.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.64" data-doi="10.1016/j.tibs.2013.04.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Towbin</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez-Sandoval</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gasser</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mechanisms of heterochromatin subnuclear localization.</span> <abbr class="cit-jnl-abbrev">Trends Biochem. Sci.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">356</span>–<span class="cit-lpage">363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BBiochem.%2BSci.%26rft.volume%253D38%26rft.spage%253D356%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibs.2013.04.004%26rft_id%253Dinfo%253Apmid%252F23746617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibs.2013.04.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23746617&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.65" data-doi="10.1038/sj.emboj.7601859"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voth</span>  <span class="cit-name-given-names">W. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahata</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kretschmann</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lieb</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Forkhead proteins control the outcome of transcription factor binding by antiactivation.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">4324</span>–<span class="cit-lpage">4334</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DVoth%26rft.auinit1%253DW.%2BP.%26rft.volume%253D26%26rft.issue%253D20%26rft.spage%253D4324%26rft.epage%253D4334%26rft.atitle%253DForkhead%2Bproteins%2Bcontrol%2Bthe%2Boutcome%2Bof%2Btranscription%2Bfactor%2Bbinding%2Bby%2Bantiactivation%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7601859%26rft_id%253Dinfo%253Apmid%252F17898805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjYvMjAvNDMyNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC85OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.993.66"><div class="cit-metadata unstructured">Wang, Y., X. Zhang, H. Zhang, Y. Lu, H. Huang <em>et al.</em>, 2012 Coiled-coil networking shapes cell molecular machinery. Mol. Biol. Cell 23: 3911–3922.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.67" data-doi="10.1128/MCB.15.10.5740"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Irie</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsumoto</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Yeast RLM1 encodes a serum response factor-like protein that may function downstream of the Mpk1 (Slt2) mitogen-activated protein kinase pathway.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">5740</span>–<span class="cit-lpage">5749</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DWatanabe%26rft.auinit1%253DY.%26rft.volume%253D15%26rft.issue%253D10%26rft.spage%253D5740%26rft.epage%253D5749%26rft.atitle%253DYeast%2BRLM1%2Bencodes%2Ba%2Bserum%2Bresponse%2Bfactor-like%2Bprotein%2Bthat%2Bmay%2Bfunction%2Bdownstream%2Bof%2Bthe%2BMpk1%2B%2528Slt2%2529%2Bmitogen-activated%2Bprotein%2Bkinase%2Bpathway%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.15.10.5740%26rft_id%253Dinfo%253Apmid%252F7565726%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xMC81NzQwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.68" data-doi="10.1128/MCB.17.5.2615"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takaesu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hagiwara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Irie</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsumoto</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Characterization of a serum response factor-like protein in Saccharomyces cerevisiae, Rlm1, which has transcriptional activity regulated by the Mpk1 (Slt2) mitogen-activated protein kinase pathway.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">2615</span>–<span class="cit-lpage">2623</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DWatanabe%26rft.auinit1%253DY.%26rft.volume%253D17%26rft.issue%253D5%26rft.spage%253D2615%26rft.epage%253D2623%26rft.atitle%253DCharacterization%2Bof%2Ba%2Bserum%2Bresponse%2Bfactor-like%2Bprotein%2Bin%2BSaccharomyces%2Bcerevisiae%252C%2BRlm1%252C%2Bwhich%2Bhas%2Btranscriptional%2Bactivity%2Bregulated%2Bby%2Bthe%2BMpk1%2B%2528Slt2%2529%2Bmitogen-activated%2Bprotein%2Bkinase%2Bpathway%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.17.5.2615%26rft_id%253Dinfo%253Apmid%252F9111331%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE3LzUvMjYxNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC85OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.69" data-doi="10.1002/yea.2989"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Welch</span>  <span class="cit-name-given-names">A. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koshland</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A simple colony-formation assay in liquid medium, termed “tadpoling,” provides a sensitive measure of Saccharomyces cerevisiae culture viability.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">501</span>–<span class="cit-lpage">509</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%26rft.volume%253D30%26rft.spage%253D501%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.2989%26rft_id%253Dinfo%253Apmid%252F24185677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.2989&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24185677&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.70" data-doi="10.1073/pnas.012578499"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilmes</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The B2 element of the Saccharomyces cerevisiae ARS1 origin of replication requires specific sequences to facilitate pre-RC formation.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.012578499%26rft_id%253Dinfo%253Apmid%252F11756674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI5OS8xLzEwMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC85OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.993.71"><div class="cit-metadata unstructured">Wong, J., Y. Nakajima, S. Westermann, C. Shang, J.-S. Kang <em>et al.</em>, 2007 A Protein Interaction Map of the Mitotic Spindle. Mol. Biol. Cell 18: 3800–3809.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.72" data-doi="10.1016/j.molcel.2014.04.032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yoshida</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bacal</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desmarais</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Padioleau</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsaponina</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The Histone Deacetylases Sir2 and Rpd3 Act on Ribosomal DNA to Control the Replication Program in Budding Yeast.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">691</span>–<span class="cit-lpage">697</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D54%26rft.spage%253D691%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2014.04.032%26rft_id%253Dinfo%253Apmid%252F24856221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2014.04.032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24856221&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.73" data-doi="10.1016/S0092-8674(04)00402-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">You</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Croyle</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishimura</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozato</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howley</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Interaction of the Bovine Papillomavirus E2 Protein with Brd4 Tethers the Viral DNA to Host Mitotic Chromosomes.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DYou%26rft.auinit1%253DJ.%26rft.volume%253D117%26rft.issue%253D3%26rft.spage%253D349%26rft.epage%253D360%26rft.atitle%253DInteraction%2Bof%2Bthe%2Bbovine%2Bpapillomavirus%2BE2%2Bprotein%2Bwith%2BBrd4%2Btethers%2Bthe%2Bviral%2BDNA%2Bto%2Bhost%2Bmitotic%2Bchromosomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252804%252900402-7%26rft_id%253Dinfo%253Apmid%252F15109495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(04)00402-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15109495&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221176400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.993.74"><div class="cit-metadata unstructured">Yu, H., P. Braun, M. A. Yildirim, I. Lemmens, K. Venkatesan <em>et al.</em>, 2008 High Quality Binary Protein Interaction Map of the Yeast Interactome Network. Science 322: 104–110.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.993.75" data-doi="10.1016/j.gde.2009.02.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bodnar</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spector</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Nuclear Neighborhoods and Gene Expression.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">172</span>–<span class="cit-lpage">179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DZhao%26rft.auinit1%253DR.%26rft.volume%253D19%26rft.issue%253D2%26rft.spage%253D172%26rft.epage%253D179%26rft.atitle%253DNuclear%2Bneighborhoods%2Band%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2009.02.007%26rft_id%253Dinfo%253Apmid%252F19339170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2009.02.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19339170&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266346900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/4/993.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/4/987" title="A HAND2 Loss-of-Function Mutation Causes Familial Ventricular Septal Defect and Pulmonary Stenosis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/4/1013" title="Fluorescence In Situ Hybridization (FISH)-Based Karyotyping Reveals Rapid Evolution of Centromeric and Subtelomeric Repeats in Common Bean (Phaseolus vulgaris) and Relatives" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/4"><span>Volume 6 Issue 4, April 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/4.cover-source.jpg?itok=Lj45nPaf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/993" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-pGBMgIfzYhmtKAIouwqQu4MFbY-yzK2PeLMU1DvveDM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/4/993" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/4/993" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/4/993" class="active" data-icon-position="" data-hide-link-title="0">High Throughput Analyses of Budding Yeast ARSs Reveal New DNA Elements Capable of Conferring Centromere-Independent Plasmid Propagation</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460126" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-XDfXdXQrDbySlB9jkJ3EEiBIqfP55JNeuk_ey2NcReQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/4/993.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/4/993" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-titM8ktBQakyTOQw3P09R3zILbFk89qwM_rv3D5FxIA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/4/993" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/4/993.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460126" id="node460126--2" data-pisa="ggg;6/4/993" data-pisa-master="ggg;g3.116.027904" data-apath="/ggg/6/4/993.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">High Throughput Analyses of Budding Yeast ARSs Reveal New DNA Elements Capable of Conferring Centromere-Independent Plasmid Propagation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Timothy Hoggard</span>, <span class="highwire-citation-author" data-delta="1">Ivan Liachko</span>, <span class="highwire-citation-author" data-delta="2">Cassaundra Burt</span>, <span class="highwire-citation-author" data-delta="3">Troy Meikle</span>, <span class="highwire-citation-author" data-delta="4">Katherine Jiang</span>, <span class="highwire-citation-author" data-delta="5">Gheorghe Craciun</span>, <span class="highwire-citation-author" data-delta="6">Maitreya J. Dunham</span> and <span class="highwire-citation-author" data-delta="7">Catherine A. Fox</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">993-1012;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.027904 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460126--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Timothy Hoggard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biomolecular Chemistry, School of Medicine and Public Health, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BHoggard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hoggard%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATimothy%2BHoggard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ivan Liachko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genome Sciences, University of Washington, Seattle, Washington 98105</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ivan%2BLiachko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liachko%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIvan%2BLiachko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Cassaundra Burt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biomolecular Chemistry, School of Medicine and Public Health, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cassaundra%2BBurt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burt%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACassaundra%2BBurt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Troy Meikle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biomolecular Chemistry, School of Medicine and Public Health, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Troy%2BMeikle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meikle%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATroy%2BMeikle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Katherine Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biomolecular Chemistry, School of Medicine and Public Health, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katherine%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatherine%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gheorghe Craciun </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biomolecular Chemistry, School of Medicine and Public Health, University of Wisconsin, Madison, Wisconsin 53706</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Mathematics, College of Letters and Science, University of Wisconsin, Madison Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gheorghe%2BCraciun%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Craciun%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGheorghe%2BCraciun%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Maitreya J. Dunham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genome Sciences, University of Washington, Seattle, Washington 98105</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maitreya%2BJ.%2BDunham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunham%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaitreya%2BJ.%2BDunham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Catherine A. Fox </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biomolecular Chemistry, School of Medicine and Public Health, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Catherine%2BA.%2BFox%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fox%20CA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACatherine%2BA.%2BFox%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cfox@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">cfox@wisc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/4/993" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/4/993&amp;title=High%20Throughput%20Analyses%20of%20Budding%20Yeast%20ARSs%20Reveal%20New%20DNA%20Elements%20Capable%20of%20Conferring%20Centromere-Independent%20Plasmid%20Propagation" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/4/993&amp;title=High%20Throughput%20Analyses%20of%20Budding%20Yeast%20ARSs%20Reveal%20New%20DNA%20Elements%20Capable%20of%20Conferring%20Centromere-Independent%20Plasmid%20Propagation" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/4/993&amp;title=High%20Throughput%20Analyses%20of%20Budding%20Yeast%20ARSs%20Reveal%20New%20DNA%20Elements%20Capable%20of%20Conferring%20Centromere-Independent%20Plasmid%20Propagation" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/4/993&amp;text=High%20Throughput%20Analyses%20of%20Budding%20Yeast%20ARSs%20Reveal%20New%20DNA%20Elements%20Capable%20of%20Conferring%20Centromere-Independent%20Plasmid%20Propagation" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/4/993&amp;title=High%20Throughput%20Analyses%20of%20Budding%20Yeast%20ARSs%20Reveal%20New%20DNA%20Elements%20Capable%20of%20Conferring%20Centromere-Independent%20Plasmid%20Propagation" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/4/993&amp;t=High%20Throughput%20Analyses%20of%20Budding%20Yeast%20ARSs%20Reveal%20New%20DNA%20Elements%20Capable%20of%20Conferring%20Centromere-Independent%20Plasmid%20Propagation" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/4/993&amp;title=High%20Throughput%20Analyses%20of%20Budding%20Yeast%20ARSs%20Reveal%20New%20DNA%20Elements%20Capable%20of%20Conferring%20Centromere-Independent%20Plasmid%20Propagation" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/4/993&amp;title=High%20Throughput%20Analyses%20of%20Budding%20Yeast%20ARSs%20Reveal%20New%20DNA%20Elements%20Capable%20of%20Conferring%20Centromere-Independent%20Plasmid%20Propagation" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460126/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460126" id="node460126--3" data-pisa="ggg;6/4/993" data-pisa-master="ggg;g3.116.027904" data-apath="/ggg/6/4/993.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">High Throughput Analyses of Budding Yeast ARSs Reveal New DNA Elements Capable of Conferring Centromere-Independent Plasmid Propagation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Timothy Hoggard</span>, <span class="highwire-citation-author" data-delta="1">Ivan Liachko</span>, <span class="highwire-citation-author" data-delta="2">Cassaundra Burt</span>, <span class="highwire-citation-author" data-delta="3">Troy Meikle</span>, <span class="highwire-citation-author" data-delta="4">Katherine Jiang</span>, <span class="highwire-citation-author" data-delta="5">Gheorghe Craciun</span>, <span class="highwire-citation-author" data-delta="6">Maitreya J. Dunham</span> and <span class="highwire-citation-author" data-delta="7">Catherine A. Fox</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">993-1012;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.027904 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460126--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Timothy Hoggard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biomolecular Chemistry, School of Medicine and Public Health, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BHoggard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hoggard%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATimothy%2BHoggard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ivan Liachko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genome Sciences, University of Washington, Seattle, Washington 98105</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ivan%2BLiachko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liachko%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIvan%2BLiachko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Cassaundra Burt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biomolecular Chemistry, School of Medicine and Public Health, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cassaundra%2BBurt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burt%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACassaundra%2BBurt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Troy Meikle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biomolecular Chemistry, School of Medicine and Public Health, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Troy%2BMeikle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meikle%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATroy%2BMeikle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Katherine Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biomolecular Chemistry, School of Medicine and Public Health, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katherine%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatherine%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gheorghe Craciun </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biomolecular Chemistry, School of Medicine and Public Health, University of Wisconsin, Madison, Wisconsin 53706</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Mathematics, College of Letters and Science, University of Wisconsin, Madison Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gheorghe%2BCraciun%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Craciun%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGheorghe%2BCraciun%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Maitreya J. Dunham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genome Sciences, University of Washington, Seattle, Washington 98105</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maitreya%2BJ.%2BDunham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunham%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaitreya%2BJ.%2BDunham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Catherine A. Fox </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biomolecular Chemistry, School of Medicine and Public Health, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Catherine%2BA.%2BFox%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fox%20CA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACatherine%2BA.%2BFox%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cfox@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">cfox@wisc.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=993&amp;rft.epage=1012&amp;rft.atitle=High%20Throughput%20Analyses%20of%20Budding%20Yeast%20ARSs%20Reveal%20New%20DNA%20Elements%20Capable%20of%20Conferring%20Centromere-Independent%20Plasmid%20Propagation&amp;rft.volume=6&amp;rft.issue=4&amp;rft.date=2016-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Hoggard%2C+Timothy&amp;rft.au=Liachko%2C+Ivan&amp;rft.au=Burt%2C+Cassaundra&amp;rft.au=Meikle%2C+Troy&amp;rft.au=Jiang%2C+Katherine&amp;rft.au=Craciun%2C+Gheorghe&amp;rft.au=Dunham%2C+Maitreya+J.&amp;rft.au=Fox%2C+Catherine+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460126/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460126/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460126/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460126/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460126/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460126/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460126/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460126/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460126/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460126/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460126/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460126/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/4/993" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/4/993.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/4/993.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/4/993.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/4/993.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
