<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/4/939" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/4/939.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/4/939.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/4/939.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/4/939/embed/mml-math-3.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/4/939.atom" />
<meta name="HW.pisa" content="ggg;6/4/939" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genetically Determined Variation in Lysis Time Variance in the Bacteriophage φX174" />
<meta name="DC.Identifier" content="10.1534/g3.115.024075" />
<meta name="DC.Date" content="2016-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Baker et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Researchers in evolutionary genetics recently have recognized an exciting opportunity in decomposing beneficial mutations into their proximal, mechanistic determinants. The application of methods and concepts from molecular biology and life history theory to studies of lytic bacteriophages (phages) has allowed them to understand how natural selection sees mutations influencing life history. This work motivated the research presented here, in which we explored whether, under consistent experimental conditions, small differences in the genome of bacteriophage φX174 could lead to altered life history phenotypes among a panel of eight genetically distinct clones. We assessed the clones’ phenotypes by applying a novel statistical framework to the results of a serially sampled parallel infection assay, in which we simultaneously inoculated each of a large number of replicate host volumes with ∼1 phage particle. We sequentially plated the volumes over the course of infection and counted the plaques that formed after incubation. These counts served as a proxy for the number of phage particles in a single volume as a function of time. From repeated assays, we inferred significant, genetically determined heterogeneity in lysis time and burst size, including lysis time variance. These findings are interesting in light of the genetic and phenotypic constraints on the single-protein lysis mechanism of φX174. We speculate briefly on the mechanisms underlying our results, and we discuss the potential importance of lysis time variance in viral evolution." />
<meta name="DC.Contributor" content="Christopher W. Baker" />
<meta name="DC.Contributor" content="Craig R. Miller" />
<meta name="DC.Contributor" content="Tanayott Thaweethai" />
<meta name="DC.Contributor" content="Jeffrey Yuan" />
<meta name="DC.Contributor" content="Meghan Hollibaugh Baker" />
<meta name="DC.Contributor" content="Paul Joyce" />
<meta name="DC.Contributor" content="Daniel M. Weinreich" />
<meta name="article:published_time" content="2016-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genetically Determined Variation in Lysis Time Variance in the Bacteriophage φX174" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Researchers in evolutionary genetics recently have recognized an exciting opportunity in decomposing beneficial mutations into their proximal, mechanistic determinants. The application of methods and concepts from molecular biology and life history theory to studies of lytic bacteriophages (phages) has allowed them to understand how natural selection sees mutations influencing life history. This work motivated the research presented here, in which we explored whether, under consistent experimental conditions, small differences in the genome of bacteriophage φX174 could lead to altered life history phenotypes among a panel of eight genetically distinct clones. We assessed the clones’ phenotypes by applying a novel statistical framework to the results of a serially sampled parallel infection assay, in which we simultaneously inoculated each of a large number of replicate host volumes with ∼1 phage particle. We sequentially plated the volumes over the course of infection and counted the plaques that formed after incubation. These counts served as a proxy for the number of phage particles in a single volume as a function of time. From repeated assays, we inferred significant, genetically determined heterogeneity in lysis time and burst size, including lysis time variance. These findings are interesting in light of the genetic and phenotypic constraints on the single-protein lysis mechanism of φX174. We speculate briefly on the mechanisms underlying our results, and we discuss the potential importance of lysis time variance in viral evolution.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/04/01" />
<meta name="citation_mjid" content="ggg;6/4/939" />
<meta name="citation_id" content="6/4/939" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/4/939" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/4/939.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/4/939.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/4/939.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.024075" />
<meta name="citation_pmid" content="26921293" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="17" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="939" />
<meta name="citation_lastpage" content="955" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Christopher W. Baker" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, Brown University, Providence, Rhode Island 02912" />
<meta name="citation_author_email" content="baker03@g.harvard.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0819-882X" />
<meta name="citation_author" content="Craig R. Miller" />
<meta name="citation_author_institution" content="Department of Mathematics, University of Idaho, Moscow, Idaho 83844" />
<meta name="citation_author_institution" content="Department of Biological Sciences, University of Idaho, Moscow, Idaho 83844" />
<meta name="citation_author_institution" content="Center for Modeling Complex Interactions, University of Idaho, Moscow, Idaho 83844" />
<meta name="citation_author" content="Tanayott Thaweethai" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, Brown University, Providence, Rhode Island 02912" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0613-4176" />
<meta name="citation_author" content="Jeffrey Yuan" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, Brown University, Providence, Rhode Island 02912" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3855-1994" />
<meta name="citation_author" content="Meghan Hollibaugh Baker" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, Brown University, Providence, Rhode Island 02912" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3226-0811" />
<meta name="citation_author" content="Paul Joyce" />
<meta name="citation_author_institution" content="Department of Mathematics, University of Idaho, Moscow, Idaho 83844" />
<meta name="citation_author" content="Daniel M. Weinreich" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, Brown University, Providence, Rhode Island 02912" />
<meta name="citation_author_institution" content="Center for Computational Molecular Biology, Brown University, Providence, Rhode Island 02912" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1424-7541" />
<meta name="citation_reference" content="citation_journal_title=Microb. Ecol.;citation_author=S. T. Abedon;citation_title=Selection for bacteriophage latent period length by bacterial density: a theoretical examination.;citation_pages=79-88;citation_volume=18;citation_year=1989;citation_pmid=24196124;citation_doi=10.1007/BF02030117" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=S. T. Abedon;citation_title=Lysis of lysis-inhibited bacteriophage T4-infected cells.;citation_pages=8073-8080;citation_volume=174;citation_year=1992;citation_issue=24;citation_pmid=1459956;citation_doi=10.1128/jb.174.24.8073-8080.1992" />
<meta name="citation_reference" content="citation_journal_title=Applied and Environmental Microbiology;citation_journal_abbrev=Appl. Environ. Microbiol.;citation_author=S. T. Abedon;citation_author=T. D. Herschler;citation_author=D. Stopar;citation_title=Bacteriophage Latent-Period Evolution as a Response to Resource Availability;citation_pages=4233-4241;citation_volume=67;citation_year=2001;citation_issue=9;citation_pmid=11526028;citation_doi=10.1128/AEM.67.9.4233-4241.2001" />
<meta name="citation_reference" content="citation_journal_title=Cell Syst.;citation_author=E. Afshinnekoo;citation_author=C. Meydan;citation_author=S. Chowdhury;citation_author=D. Jaroudi;citation_author=C. Boyer;citation_title=Geospatial resolution of human and bacterial diversity with city-scale metagenomics.;citation_pages=1-15;citation_volume=1;citation_year=2015;citation_doi=10.1016/j.cels.2015.07.009" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. G. Barrell;citation_author=G. M. Air;citation_author=C. A. Hutchison;citation_title=Overlapping genes in bacteriophage phiX174.;citation_pages=34-41;citation_volume=264;citation_year=1976;citation_issue=5581;citation_pmid=1004533;citation_doi=10.1038/264034a0" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=R. A. Bernal;citation_author=S. Hafenstein;citation_author=R. Esmeralda;citation_author=B. A. Fane;citation_author=M. G. Rossmann;citation_title=The phiX174 protein J mediates DNA packaging and viral attachment to host cells.;citation_pages=1109-1122;citation_volume=337;citation_year=2004;citation_issue=5;citation_pmid=15046981;citation_doi=10.1016/j.jmb.2004.02.033" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=T. G. Bernhardt;citation_author=W. D. Roof;citation_author=R. Young;citation_title=The Escherichia coli FKBP-type PPIase SlyD is required for the stabilization of the E lysis protein of bacteriophage phi X174.;citation_pages=99-108;citation_volume=45;citation_year=2002;citation_issue=1;citation_pmid=12100551;citation_doi=10.1046/j.1365-2958.2002.02984.x" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=C. J. Brown;citation_author=L. Zhao;citation_author=K. J. Evans;citation_author=D. Ally;citation_author=A. D. Stancik;citation_title=Positive selection at high temperature reduces gene transcription in the bacteriophage ÏX174.;citation_pages=378-378;citation_volume=10;citation_year=2010;citation_pmid=21129199;citation_doi=10.1186/1471-2148-10-378" />
<meta name="citation_reference" content="citation_journal_title=BMC Evol. Biol.;citation_author=C. J. Brown;citation_author=A. D. Stancik;citation_author=P. Roychoudhury;citation_author=S. M. Krone;citation_title=Adaptive regulatory substitutions affect multiple stages in the life cycle of the bacteriophage φX174.;citation_pages=1-12;citation_volume=13;citation_year=2013;citation_pmid=23279962;citation_doi=10.1186/1471-2148-13-1" />
<meta name="citation_reference" content="citation_journal_title=Journal of Theoretical Biology;citation_journal_abbrev=Journal of Theoretical Biology;citation_author=J. J. Bull;citation_title=Optimality models of phage life history and parallels in disease evolution.;citation_pages=928-938;citation_volume=241;citation_year=2006;citation_issue=4;citation_pmid=16616205;citation_doi=10.1016/j.jtbi.2006.01.027" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=J. J. Bull;citation_author=D. W. Pfennig;citation_author=I.-N. Wang;citation_title=Genetic details, optimization and phage life histories.;citation_pages=76-82;citation_volume=19;citation_year=2004;citation_issue=2;citation_pmid=16701232;citation_doi=10.1016/j.tree.2003.10.008" />
<meta name="citation_reference" content="citation_journal_abbrev=Am Nat;citation_author=J. J. Bull;citation_author=J. Millstein;citation_author=J. Orcutt;citation_author=H. Wichman;citation_title=Evolutionary feedback mediated through population density, illustrated with viruses in chemostats.;citation_pages=E39-E51;citation_volume=167;citation_year=2006;citation_issue=2;citation_pmid=16670974;citation_doi=10.1086/499374" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. J. Bull;citation_author=R. H. Heineman;citation_author=C. O. Wilke;citation_title=The phenotype-fitness map in experimental evolution of phages.;citation_pages=e27796-e27796;citation_volume=6;citation_year=2011;citation_issue=11;citation_pmid=22132144;citation_doi=10.1371/journal.pone.0027796" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. J. Bull;citation_author=C. S. Vegge;citation_author=M. Schmerer;citation_author=W. N. Chaudhry;citation_author=B. R. Levin;citation_title=Phenotypic resistance and the dynamics of bacterial escape from phage control.;citation_pages=e94690;citation_volume=9;citation_year=2014;citation_pmid=24743264;citation_doi=10.1371/journal.pone.0094690" />
<meta name="citation_reference" content="citation_journal_title=Br. J. Exp. Pathol.;citation_author=F. Burnet;citation_title=A method for the study of bacteriophage multiplication in broth.;citation_pages=109-115;citation_volume=10;citation_year=1929" />
<meta name="citation_reference" content="citation_journal_title=Journal of Virology;citation_journal_abbrev=J. Virol.;citation_author=L. Chao;citation_author=C. U. Rang;citation_author=L. E. Wong;citation_title=Distribution of Spontaneous Mutants and Inferences about the Replication Mode of the RNA Bacteriophage {phi}6;citation_pages=3276-3281;citation_volume=76;citation_year=2002;citation_issue=7;citation_pmid=11884552;citation_doi=10.1128/JVI.76.7.3276-3281.2002" />
<meta name="citation_reference" content="citation_journal_title=J. Virol.;citation_author=J. E. Cherwa;citation_author=P. Sanchez-Soria;citation_author=H. A. Wichman;citation_author=B. A. Fane;citation_title=Viral adaptation to an antiviral protein enhances the fitness level to above that of the uninhibited wild type.;citation_volume=83;citation_year=2009;citation_pmid=19726521;citation_doi=10.1128/JVI.01297-09" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=A. M. Dean;citation_author=J. W. Thornton;citation_title=Mechanistic approaches to the study of evolution: the functional synthesis.;citation_pages=675-688;citation_volume=8;citation_year=2007;citation_issue=9;citation_pmid=17703238;citation_doi=10.1038/nrg2160" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=M. Delbrück;citation_title=The Burst Size Distribution in the Growth of Bacterial Viruses (Bacteriophages);citation_pages=131-135;citation_volume=50;citation_year=1945;citation_issue=2;citation_pmid=20989330" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=D. T. Denhardt;citation_author=R. L. Sinsheimer;citation_title=The process of infection with bacteriophage phi-X174. 3. Phage maturation and lysis after synchronized infection.;citation_pages=641-646;citation_volume=12;citation_year=1965;citation_issue=3;citation_pmid=5323480;citation_doi=10.1016/S0022-2836(65)80318-7" />
<meta name="citation_reference" content="citation_journal_title=BMC Microbiology;citation_journal_abbrev=BMC Microbiology;citation_author=J. J. Dennehy;citation_author=I.-N. Wang;citation_title=Factors influencing lysis time stochasticity in bacteriophage Î».;citation_pages=174-174;citation_volume=11;citation_year=2011;citation_pmid=21810267;citation_doi=10.1186/1471-2180-11-174" />
<meta name="citation_reference" content="citation_title=The Bacteriophages;citation_title=X174 et al., the Microviridae;citation_author=B. A. Fane;citation_author=K. L. Brentlinger;citation_author=A. D. Burch;citation_author=M. Chen;citation_author=S. Hafenstein;citation_pages=129-145;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_journal_abbrev=Microbiology;citation_author=H. Hadas;citation_author=M. Einav;citation_author=I. Fishov;citation_author=A. Zaritsky;citation_title=Bacteriophage T4 Development Depends on the Physiology of its Host Escherichia Coli;citation_pages=179-185;citation_volume=143;citation_year=1997;citation_issue=1;citation_pmid=9025292;citation_doi=10.1099/00221287-143-1-179" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=M. J. Harms;citation_author=J. W. Thornton;citation_title=Evolutionary biochemistry: revealing the historical and physical causes of protein properties.;citation_pages=559-571;citation_volume=14;citation_year=2013;citation_pmid=23864121;citation_doi=10.1038/nrg3540" />
<meta name="citation_reference" content="citation_title=The Bacteriophages;citation_author=M. Hayashi;citation_author=A. Aoyama;citation_author=D. L. Richardson;citation_author=M. N. Hayashi;citation_title=Biology of the bacteriophage φX174;citation_pages=1-71;citation_volume=Vol. 2;citation_year=1988" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. E. Hubbarde;citation_author=G. Wild;citation_author=L. M. Wahl;citation_title=Fixation probabilities when generation times are variable: the burst-death model.;citation_volume=176;citation_year=2007;citation_pmid=17483420;citation_doi=10.1534/genetics.107.072009" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=C. A. Hutchison;citation_author=R. L. Sinsheimer;citation_title=KINETICS OF BACTERIOPHAGE RELEASE BY SINGLE CELLS OF PHI X174-INFECTED E. COLI.;citation_pages=206-208;citation_volume=7;citation_year=1963;citation_pmid=14062652;citation_doi=10.1016/S0022-2836(63)80046-7" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=C. A. Hutchison;citation_author=R. L. Sinsheimer;citation_title=The process of infection with bacteriophage phi-X174. X. Mutations in a phi-X Lysis gene.;citation_pages=429-447;citation_volume=18;citation_year=1966;citation_issue=3;citation_pmid=5968177;citation_doi=10.1016/S0022-2836(66)80035-9" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=S. J. Labrie;citation_author=M.-E. Dupuis;citation_author=D. M. Tremblay;citation_author=P.-L. Plante;citation_author=J. Corbeil;citation_title=A new microviridae phage isolated from a failed biotechnological process driven by Escherichia coli.;citation_volume=80;citation_year=2014;citation_pmid=25192988;citation_doi=10.1128/AEM.01365-14" />
<meta name="citation_reference" content="citation_title=Handbook of Biological Statistics;citation_author=J. McDonald;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_journal_abbrev=Microbiology;citation_author=S. Mendel;citation_author=J. M. Holburn;citation_author=J. A. Schouten;citation_author=T. D. H. Bugg;citation_title=Interaction of the transmembrane domain of lysis protein E from bacteriophage {phi}X174 with bacterial translocase MraY and peptidyl-prolyl isomerase SlyD.;citation_pages=2959-2967;citation_volume=152;citation_year=2006;citation_issue=Pt 10;citation_pmid=17005977;citation_doi=10.1099/mic.0.28776-0" />
<meta name="citation_reference" content="citation_journal_title=Biometrika;citation_journal_abbrev=Biometrika;citation_author=X.-L. Meng;citation_author=D. B. Rubin;citation_title=Maximum likelihood estimation via the ECM algorithm: A general framework;citation_pages=267-278;citation_volume=80;citation_year=1993;citation_issue=2;citation_doi=10.1093/biomet/80.2.267" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=A. Michel;citation_author=O. Clermont;citation_author=E. Denamur;citation_author=O. Tenaillon;citation_title=Bacteriophage φX174’s ecological niche and the flexibility of its Escherichia coli lipopolysaccharide receptor.;citation_volume=76;citation_year=2010;citation_pmid=20833781;citation_doi=10.1128/AEM.02721-09" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=J. E. Newbold;citation_author=R. L. Sinsheimer;citation_title=The process of infection with bacteriophage phiX174. XXXII. Early steps in the infection process: attachment, eclipse and DNA penetration.;citation_pages=49-66;citation_volume=49;citation_year=1970;citation_issue=1;citation_pmid=5450517;citation_doi=10.1016/0022-2836(70)90375-X" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. Patwa;citation_author=L. M. Wahl;citation_title=Fixation probability for lytic viruses: the attachment-lysis model.;citation_volume=180;citation_year=2008;citation_pmid=18757918;citation_doi=10.1534/genetics.108.090555" />
<meta name="citation_reference" content="citation_journal_title=Journal of Theoretical Biology;citation_journal_abbrev=Journal of Theoretical Biology;citation_author=Z. Patwa;citation_author=L. M. Wahl;citation_title=The impact of host-cell dynamics on the fixation probability for lytic viruses.;citation_pages=799-810;citation_volume=259;citation_year=2009;citation_issue=4;citation_pmid=19464301;citation_doi=10.1016/j.jtbi.2009.05.008" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=W. Pearson;citation_author=T. Wood;citation_author=Z. Zhang;citation_author=W. Miller;citation_title=Comparison of DNA sequences with protein sequences.;citation_pages=24-36;citation_volume=46;citation_year=1997;citation_issue=1;citation_pmid=9403055;citation_doi=10.1006/geno.1997.4995" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. M. Pepin;citation_author=M. A. Samuel;citation_author=H. A. Wichman;citation_title=Variable pleiotropic effects from mutations at the same locus hamper prediction of fitness from a fitness component.;citation_volume=172;citation_year=2006;citation_pmid=16361237;citation_doi=10.1534/genetics.105.049817" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=A. Rabinovitch;citation_author=H. Hadas;citation_author=M. Einav;citation_author=Z. Melamed;citation_author=A. Zaritsky;citation_title=Model for Bacteriophage T4 Development in Escherichia coli;citation_pages=1677-1683;citation_volume=181;citation_year=1999;citation_issue=5;citation_pmid=10049403" />
<meta name="citation_reference" content="citation_journal_title=Journal of Theoretical Biology;citation_journal_abbrev=Journal of Theoretical Biology;citation_author=A. Rabinovitch;citation_author=I. Fishov;citation_author=H. Hadas;citation_author=M. Einav;citation_author=A. Zaritsky;citation_title=Bacteriophage T4 development in Escherichia coli is growth rate dependent.;citation_pages=1-4;citation_volume=216;citation_year=2002;citation_issue=1;citation_pmid=12076123;citation_doi=10.1006/jtbi.2002.2543" />
<meta name="citation_reference" content="citation_title=Life History Evolution;citation_author=D. Roff;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=W. Roof;citation_author=H. Fang;citation_author=K. Young;citation_author=J. Sun;citation_author=R. Young;citation_title=Mutational analysis of slyD, an Escherichia coli gene encoding a protein of the FKBP immunophilin family.;citation_pages=1031-1046;citation_volume=25;citation_year=1997;citation_issue=6;citation_pmid=9350861;citation_doi=10.1046/j.1365-2958.1997.5201884.x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=F. Sanger;citation_author=G. M. Air;citation_author=B. G. Barrell;citation_author=N. L. Brown;citation_author=A. R. Coulson;citation_title=Nucleotide sequence of bacteriophage phi X174 DNA.;citation_pages=687-695;citation_volume=265;citation_year=1977;citation_issue=5596;citation_pmid=870828;citation_doi=10.1038/265687a0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Shao;citation_author=I.-N. Wang;citation_title=Bacteriophage adsorption rate and optimal lysis time.;citation_volume=180;citation_year=2008;citation_pmid=18757924;citation_doi=10.1534/genetics.108.090100" />
<meta name="citation_reference" content="citation_journal_title=J. R. Soc. Interface;citation_title=Stochastic holin expression can account for lysis time variation in the bacteriophage λ.;citation_author=A. Singh;citation_author=J. J. Dennehy;citation_volume=11;citation_year=2014" />
<meta name="citation_reference" content="citation_title=The Evolution of Life Histories;citation_author=S. Stearns;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=S. Stearns;citation_author=M. Ackermann;citation_author=M. Doebeli;citation_author=M. Kaiser;citation_title=Experimental evolution of aging, growth, and reproduction in fruitflies;citation_pages=3309-3313;citation_volume=97;citation_year=2000;citation_issue=7;citation_pmid=10716732;citation_doi=10.1073/pnas.97.7.3309" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Z. J. Storms;citation_author=D. Sauvageau;citation_title=Evidence that the heterogeneity of a T4 population is the result of heritable traits.;citation_pages=e116235;citation_volume=9;citation_year=2014;citation_pmid=25551763;citation_doi=10.1371/journal.pone.0116235" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Lett.;citation_author=Z. J. Storms;citation_author=T. Brown;citation_author=D. G. Cooper;citation_author=D. Sauvageau;citation_author=R. L. Leask;citation_title=Impact of the cell life-cycle on bacteriophage T4 infection.;citation_volume=353;citation_year=2014;citation_pmid=24822278;citation_doi=10.1111/1574-6968.12402" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=S. Tanaka;citation_author=W. M. Clemons;citation_title=Minimal requirements for inhibition of MraY by lysis protein E from bacteriophage Î¦X174.;citation_pages=975-985;citation_volume=85;citation_year=2012;citation_issue=5;citation_pmid=22742425;citation_doi=10.1111/j.1365-2958.2012.08153.x" />
<meta name="citation_reference" content="citation_journal_abbrev=ISME J;citation_author=J. D. van Elsas;citation_author=A. V. Semenov;citation_author=R. Costa;citation_author=J. T. Trevors;citation_title=Survival of Escherichia coli in the environment: fundamental and public health aspects.;citation_pages=173-183;citation_volume=5;citation_year=2011;citation_issue=2;citation_pmid=20574458;citation_doi=10.1038/ismej.2010.80" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. M. Wahl;citation_author=C. S. DeHaan;citation_title=Fixation Probability Favors Increased Fecundity Over Reduced Generation Time;citation_pages=1009-1018;citation_volume=168;citation_year=2004;citation_issue=2;citation_pmid=15514071;citation_doi=10.1534/genetics.104.029199" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=I.-N. Wang;citation_title=Lysis timing and bacteriophage fitness.;citation_volume=172;citation_year=2006;citation_pmid=16219778;citation_doi=10.1534/genetics.105.045922" />
<meta name="citation_reference" content="citation_journal_title=Evol. Ecol.;citation_author=I.-N. Wang;citation_author=D. E. Dykhuizen;citation_author=L. B. Slobodkin;citation_title=The evolution of phage lysis timing.;citation_pages=545-558;citation_volume=10;citation_year=1996;citation_doi=10.1007/BF01237884" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=H. A. Wichman;citation_author=C. J. Brown;citation_title=Experimental evolution of viruses: Microviridae as a model system;citation_pages=2495-2501;citation_volume=365;citation_year=2010;citation_issue=1552;citation_pmid=20643739;citation_doi=10.1098/rstb.2010.0053" />
<meta name="citation_reference" content="citation_journal_title=Archives of Microbiology;citation_journal_abbrev=Archives of Microbiology;citation_author=A. Witte;citation_author=E. Brand;citation_author=P. Mayrhofer;citation_author=F. Narendja;citation_author=L. Werner;citation_title=Mutations in cell division proteins FtsZ and FtsA inhibit phiX174 protein-E-mediated lysis of Escherichia coli.;citation_pages=259-268;citation_volume=170;citation_year=1998;citation_issue=4;citation_pmid=9732440;citation_doi=10.1007/s002030050641" />
<meta name="citation_reference" content="citation_journal_title=Current Opinion in Biotechnology;citation_journal_abbrev=Current Opinion in Biotechnology;citation_author=H. Yin;citation_author=D. Marshall;citation_title=Microfluidics for single cell analysis.;citation_pages=110-119;citation_volume=23;citation_year=2012;citation_issue=1;citation_pmid=22133547;citation_doi=10.1016/j.copbio.2011.11.002" />
<meta name="citation_reference" content="citation_title=The Bacteriophages;citation_title=Phage lysis;citation_author=R. Young;citation_author=I.-N. Wang;citation_pages=104-125;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Trends in microbiology;citation_journal_abbrev=Trends Microbiol;citation_author=R. Young;citation_author=I.-N. Wang;citation_author=W. D. Roof;citation_title=Phages will out: strategies of host cell lysis.;citation_pages=120-128;citation_volume=8;citation_year=2000;citation_issue=3;citation_pmid=10707065;citation_doi=10.1016/S0966-842X(00)01705-4" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_journal_abbrev=Microbiology;citation_author=Y. Zheng;citation_author=D. K. Struck;citation_author=C. A. Dankenbring;citation_author=R. Young;citation_title=Evolutionary dominance of holin lysis systems derives from superior genetic malleability;citation_pages=1710-1718;citation_volume=154;citation_year=2008;citation_issue=6;citation_pmid=18524925;citation_doi=10.1099/mic.0.2008/016956-0" />
<meta name="citation_reference" content="citation_journal_abbrev=Biochemistry;citation_author=Y. Zheng;citation_author=D. K. Struck;citation_author=R. Young;citation_title=Purification and functional characterization of phiX174 lysis protein E.;citation_pages=4999-5006;citation_volume=48;citation_year=2009;citation_issue=22;citation_pmid=19379010;citation_doi=10.1021/bi900469g" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genetically Determined Variation in Lysis Time Variance in the Bacteriophage φX174" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
<meta name="twitter:description" content="Researchers in evolutionary genetics recently have recognized an exciting opportunity in decomposing beneficial mutations into their proximal, mechanistic determinants. The application of methods and concepts from molecular biology and life history theory to studies of lytic bacteriophages (phages) has allowed them to understand how natural selection sees mutations influencing life history. This work motivated the research presented here, in which we explored whether, under consistent experimental conditions, small differences in the genome of bacteriophage φX174 could lead to altered life history phenotypes among a panel of eight genetically distinct clones. We assessed the clones’ phenotypes by applying a novel statistical framework to the results of a serially sampled parallel infection assay, in which we simultaneously inoculated each of a large number of replicate host volumes with ∼1 phage particle. We sequentially plated the volumes over the course of infection and counted the plaques that formed after incubation. These counts served as a proxy for the number of phage particles in a single volume as a function of time. From repeated assays, we inferred significant, genetically determined heterogeneity in lysis time and burst size, including lysis time variance. These findings are interesting in light of the genetic and phenotypic constraints on the single-protein lysis mechanism of φX174. We speculate briefly on the mechanisms underlying our results, and we discuss the potential importance of lysis time variance in viral evolution." />
<meta name="og-title" property="og:title" content="Genetically Determined Variation in Lysis Time Variance in the Bacteriophage φX174" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/4/939" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Researchers in evolutionary genetics recently have recognized an exciting opportunity in decomposing beneficial mutations into their proximal, mechanistic determinants. The application of methods and concepts from molecular biology and life history theory to studies of lytic bacteriophages (phages) has allowed them to understand how natural selection sees mutations influencing life history. This work motivated the research presented here, in which we explored whether, under consistent experimental conditions, small differences in the genome of bacteriophage φX174 could lead to altered life history phenotypes among a panel of eight genetically distinct clones. We assessed the clones’ phenotypes by applying a novel statistical framework to the results of a serially sampled parallel infection assay, in which we simultaneously inoculated each of a large number of replicate host volumes with ∼1 phage particle. We sequentially plated the volumes over the course of infection and counted the plaques that formed after incubation. These counts served as a proxy for the number of phage particles in a single volume as a function of time. From repeated assays, we inferred significant, genetically determined heterogeneity in lysis time and burst size, including lysis time variance. These findings are interesting in light of the genetic and phenotypic constraints on the single-protein lysis mechanism of φX174. We speculate briefly on the mechanisms underlying our results, and we discuss the potential importance of lysis time variance in viral evolution." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
    <title>Genetically Determined Variation in Lysis Time Variance in the Bacteriophage φX174 | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ho-9PvNuoaJxWP8paXjex5tRxt24T8bOvRetIvANPTA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460131","apath":"\/ggg\/6\/4\/939.atom","pisa":"ggg;6\/4\/939","ac":{"\/ggg\/6\/4\/939.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/4\/939.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/4\/939"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/4\/939"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/4\/939":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/anGZjEiPt74ZC\\\\\\\/sZ8kahFCmytbdeWZsut8vNRo6MPDQrfwkHJqc70UWkpw\\\\\\\/zObZCrJ6vdr3QBlgTA5LU3Crfq0Iupj9U8xAD1UhBG\\\\\\\/dzOX72WzTXyv6R24U35chfb5WSieHx\\\\\\\/Zu1BsYDD+ivMWKBs\\\\\\\/ndyqC0XPGEyEeyKk603q30Qt5caMXprUMnMbqexqHUqLjlefi3PaKGdd6sVVau0C45VUcNxhGn4qhwynhEJ6J9FvdkMixN+Udd8L5TPBgyjNzf230mkdBtZn9VWEDk8lzmVKOwxVDENfGX++hacPNLuyYfthw5WmEHmipdK0ffM1MEXBX2pL1bU\\\\\\\/B4thwyza2OHEt\\\\\\\/XAe54jCTeuC+c1T961IPGwIzZFVDT5wqxbGRD1wdtz9DQlWe1TKXfmNlQ07XWxa2UxiK46Hchut0DaKBkhhEEYhoyNYGQi\\\\\\\/cu7zl\\\\\\\/WXGdCr8RJ1X1on23Pd5pEyr8Mn4FvZrZ3OLC6OUVubDCUDGKJnoEQIQ6i4UpI8lN1rbzQD1\\\\\\\/Sy9y\\\\\\\/\\\\\\\/4wzYUjHKOuUdZLEzyqXYH3WHaUSV2z0BzP98UyiicpuGdNuS3QwJh023DZFNyfBFl+vqrnmDZJG8GW6A0ihBsIixogPlwgegPuclj7Id6a0kT+4XOMJuafcK8lFJNCmAcxZtbW\\\\\\\/yyu0fJpiLuUatd+R\\\\\\\/zPpENIh7aUG5l5fGfmPkGPh2hhQ71ZpHN7ZEeTqV6Ql8M1h0NH1BkRj78ZIYvmBb1t4VIaBJDXgG5Q\\\\\\\/ThmNGRiM\\\\\\\/03wirLa+Quhqkq5XgxHSRyjyKdxrqFTQ0iL+Zr50piY6llRpiaiF\\\\\\\/FjzqE1N3+0C\\\\\\\/2\\\\\\\/ouP\\\\\\\/mBvXi0qOzr\\\\\\\/iodxz0ZrtQ8H5h6oWC0sH\\\\\\\/0SZG3iE2c0k6hDf0xTFbWFoJRqtBoRfWLplvA7Pgm\\\\\\\/rcn5pq3s+9oTrk65aafJUtWi\\\\\\\/dG1L+soIlbAwvHhtSJqpBwpBnoVZ8au9GMBfuzndBVoRdbyL1FXPQIhjZwkPLcintNNjHN0Xh3SHcmczD6NLNz\\\\\\\/cA0+oiCVoK9xRzEikOLXMzVGDi2ejdiMLkX818P5niRorcoz6cZDonqtAJo6XBdIaIdECa6HqCmMbtTCwmzScQLzkugzqB0Mej51wmgYmIyL0s4d7Wx5Ea1N3Lj5WxF6bbRAsxzGD+RWo5558fiijVqIttG\\\\\\\/FFvGH8nBhp0JqoN9mYZ5Ll5UW1lS7YB1g04PXKMIoyyrtLdW8Rre1sB\\\\\\\/EPlI7DaLku7JnQftsBLn3gNMa+yJFNC2dMOsZMRFLI\\\u0022,\\\u0022iv\\\u0022:\\\u00225zvMVcF9xhOJqQcDguygQA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222bd0844f49cc0bd12e88fc1b4dba286320bc866d94434108b2ba501a8915efd0\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022lRTU3KM2LVmg5UK\\\\\\\/wnfa9SFXqvm+fOuNvJgzwMLn9L6q7GmMwXTgwY2XQY+Yd13WhWu7TIhsKieiqCwxscyiNoRUp2aCPgPUtt+3UtyV3WRM7WrueHQjlbrRoMUwpJ8LaCF99iRvt5XF1TYVZtpsMt+qJ\\\\\\\/AcwCJFq530HGXTgSpMqjRJxnKxmEMEIJIplWCi50QbX6yUya988jcgMTfahgdjdVeYYB3Us4Sm9+V2hQUpcrUUBQDfITMLnN61gkEPbbT7\\\\\\\/xqJvkMfEgnk2tvMP4sgLjPstkiYxrIrt3h3mZPYUZKj191EsU4PVr8Orszr5Gp\\\\\\\/ue743LhwLqbaDlyRUsuhVeXo7Vym8c9kQ+VyToVFUCV1iyddMaElb1yZShStMwYu1uxrTmMukbEah5CaSyWzC419wbjISP4YeAB4Z3kRTJf9YMlU8xl+B35\\\\\\\/EICsl16efroT9gHBGHb6mZi7qKfopV+jOTllvQGBccviMD+Bbd4Imo+loCzP4yzPBpPgvZjnqpWi0DsVcn61myYsit\\\\\\\/DfPMBNScggXOrCLw4UPRC13NzqMifi7+omH\\\\\\\/nJhp1ZmJ7x33su7RguXOVgqckuSENkCp1R\\\\\\\/k1v\\\\\\\/FKeXAnmhEdVJgLtheXf+\\\\\\\/RoyyO31wEtZXF9YXll8PxOy7ZmwZRWGZCz9+C3KpQ74UD2fHCQkaxt2xfwmajtPvNn+Qdn8BwLeU8lLaByDpEf47APTMxYrMBlnr4hpzjdnFG\\\\\\\/VGpmIjYNl+\\\\\\\/xMJ7z5omPk80aHTUXbn11UWZYI8eMRiGw4zKGlrqhoNlFqSHHnHC9aNZMKmsAbtYHN2Y9W2NXxAN+0qhDs4Kvd8Rwxq+q\\\\\\\/+M8SbTRMOMoeuzHdUmkaGeiQry0e6oB4K9NupzCItCAxzqONbOshOD4wuDlvoO5MFyPmMP1pkc6HWNiDJ5F4B1VrJptiZa0KDR3IhMsnfWKC1XVVYEx2zrlTICcnXzKNo\\\\\\\/ekceXMtxj31qWngeR\\\\\\\/zujy1xBaWIMfqfDTKhXNRJPRljnh\\\\\\\/iR8DYT98QvVDo8cvReXL7\\\\\\\/r1jHd375v29VrPCHSpkNL++5PcVSqtWqaBExctsXENMIGHtjgv8+4q4y3FR35BmJZE5zsMnguHPliOuV3aRrBh09ETP8nzRTjw7xuC6M4ltLONFKHVV0+Zjz\\\\\\\/UGS9jzOws0v18v5XwtvyotYUGwYoFmDf1GEkWmGElvOb86xowcpJpFVMXU+Des23tT3IUY4rFaz+PYzMP7ZP6jSUKd8diPbiyay5zpJNutXYGSJQKX+3y9V1\\\\\\\/WLxsduijsUR+9rmaizc08u1au4wmkIsK5UeVrBG6uwtdJpvUfJYStr65oxTd4+euMBVJZaJAJxZnX9PHdVSlZ26ZdekSwL7uUDZ+9GpVlRjsjbOIwxHPTC3tyWVCEvSdLbCZR1l90Z6O+Dyvtc8sV+uTJFj0vsIrU0BoH3qpEbzQjL4HKa\\\\\\\/NOJhkJpI6DayV4FLYj\\\\\\\/ypFk2ceE295raSdj9aelalOZgu3Vusd5JhyxPPRdIzUzX5SlIdKuWv\\\\\\\/L0PKLwPEDgKuDEs2meiD37nAXMGQkkjyaovSCD7QQs5J5WGgSp2O9jtwW024ASfuktp5tSPNslHRgc\\\\\\\/JhRzhLhD3fNLZX3uU9duqW0sGXE1euAP5MLb2Y\\\\\\\/KcLBTrZoy7Yf+zlfXnIvwTIE+rB\\\\\\\/oHs9RY99SxiMtlioLmVSZL91R7H4mlu\\\\\\\/2EyWSViNcea2xip+BiIceAvNJh\\\\\\\/FmYt8OeTGEOGGQsbbPd+z+t7tgckdWZazxuyDO5B4JwIPG5HmjH2ygCXSv2zeGeis6DJQrFJnrijuNgNZ7XJv9FXKvnUGo79IXaFtY854riiWSGzTqk+IH7ULkqyajDSijKreHCnrA081I5D2wYVgiPcyNa1G6AFqnsfW+C1imtu0nSBaNr1MduR4LuTONMOfi\\\\\\\/C1sI8NP3K8kgv9JpHArQLEk0YPeitod2oQs+i\\\\\\\/\\\\\\\/GGqATdl2fw5NeIi5jodMiVm1dXKfomjiWiC8sPuIz+TQmbDliiXu8Cz3ZMGZ4DfHiZGjzOIuxFQmeQCCFm47JqzmLOzkR8brG587IpWVtL+BlTe8cBV6RKvKmac4lclmMMz1WuVug3KDLQ0KgYbHDn6U66HwN+2DoAwE82t0RbYR7o5VX8b+WvYznp5iSRtKCx+OtX0sC0VvlhH1SfbZeTd8bEfoS7d1bjaV2HBkjh6nHZvb\\\\\\\/RzYMRbos\\\\\\\/PmX5fjLC3pFgTSj9LeivWGl93w55KpAYvzGmtMG7aGeCl4vmjJ3Q2HSsRuQ6pyUDwNwscTJxNppga9dC01UE7nq7zsVIM\\\\\\\/jWLoAHra+U1n94BCpiJCV4120hKQ4XRK3mRin1oze3L8UBnBajPy8iKMcd20LIKgI8lkIDznjncGA9rW\\\\\\\/PzeZIRwuUUhdZmNUeOWaQxlfypRJq5BwvdKkA7BHMMvnNzWpqRKA7mNSJDSQ42kofOgEfL9T3lqs0Os6pkOtqNqNmuCQkRL0bPTmlSHn8aMFcJ7ooBeOyvXn5Ty7cBA393AmQaacFL\\\\\\\/RkYJFm0lUsU8fR3fklLXHhzROjdiAvNb9MrjlwoiPCkhhhw2QyMAu44urX8X6hEn73M6tXmeutTi5HPHRIX9JMZWgcyYVaQ0pXkQUQbozbKUOb+P9LwN\\\\\\\/oEaiRUlmpbrulupLf\\\\\\\/NQor5qsf2WyWMeb6Lz6DanKDLNT4JtYv1cU6lJf3DacQP2xSvwIvEIO9ELDYoDgI\\\\\\\/117CY+Goohrpjua1GKQGfgLDwCDuqz4iXQVenJSiGtbdVE8peVYkMfaZdQmUZt+8RTB\\\\\\\/+OB6YoCt8V3EGq9sDemKl+lloWO+ykBVvr7KJ7sGR20dlQaNXzbnbiflULxhFLUXmTsGQEDU\\\\\\\/fURru\\\\\\\/KiD8aEOwRd4U84blz2jJClq1263BO0XSJsT5SN2YRQZIzFPr\\\\\\\/pfwqVsm9zSfcnFivCMUFBiyZ3ekiueJH1ik0WnS6okPAfdWq1JxI7Mji0YYDPjNCNUzRmHnwDHkRn7XUC5IZ2BDAzmj1qzsuATl\\\\\\\/jSnU7kJ32GzqVpzM3c0SDh3TZQtXlH7crGp3Qrxf5wJNi0QYExq6Sd5f6\\\\\\\/Rmh8FdjyQLVe9R3idFTlx9SkI0Z7g\\\\\\\/mQuKzpXQ4DHvflgVB7330T3U7AL0uo3Z9aW3P4ulC8l0dOrjb+Fw8wFupofjAQ6vS5uWC8ncuhff0OKibb+wBcNM5MAWkGufxiNUtJ4xnzI+fkfwAIIoGgOlXy+KWJfhjule+iL2asxG3VAVhq58f9uDxnP6VvNhLrid73BELOpYhSRg\\\\\\\/NVTUFJ05nmuH1roOINhvQH3nN6fQXTojklZQ0gpS2fP8p\\\\\\\/SUgG0AomJi+QIZffAB5vejYyAhM3ds33Z\\\\\\\/CiF3j4ehCXHgz8vuZhjNVZhZ6lv9fuXPONVEtHQwMqjftjiVZpKTW7Kv5r0GQpQQUN\\\\\\\/G0xU46P\\\\\\\/pd3Nz9B9uAu5jeykYPAxQdcVoqu7DbK3eCp4uuV2mZI4mh+Oi2Ju+xG3JOuiJJ\\\\\\\/Jb3xMqdLxPyuRtv+AjICGy0Tb+SGvVFdM1B43yATZ8tBWUoRwt1APHythI14OW2wrK5yjYRwpybiqXJGRXMLja5BBboIfNFdymNFJb40Y0rk2ha8zylCB8akJZsYCFUgXV4XsiElJY5Tlqp7nwsC1Hfutt12rjcY9kIRkfTXVmaaQsaeANmYg3bbir6P0U\\\\\\\/YGjA6iXOReH8\\\\\\\/OIm47lb88OtAf0azEzWmSxbeSFkQEbjxmYN8sa7gicma1PZugF9wsG0prgUbjqJlU43EIzLfHY6jHM+RDqSXyzGfxthlmxG5tlWw9mb+TheCUsFHRhgNE40DV04IFkYnP7fxFoW+mFhzh5vWFSmhcWxZYbli7mzFPJq\\\\\\\/AT9RhRdAWEl3wkcnijiV7ZRQrvxqZjq4O8zOqIJmJIUA0f1uDjMlyAUDKoTAq8ZGBj6pi0lcy5bjQTy15NZDK0ZmA3uHDlgDn8anbLsk4hq4ZFKLnReEOC624nlVDD9kenAfqrrm5VyK24+lINQ6m6R72jPQsP6hzDLhU7PcInAKDKUaAjMIsNSJ9ABekA15EGfW9EsEUB2x5A1jd68BtEkGwUW4WqThXWUyrQ7H4JgBz2Fw76x7XbsoNMwJsL9mLk\\\\\\\/rpkSlOxPbHURfpsWWcKSIq0d\\\\\\\/hhUl1OEKDvuCj9AUyB6m9tz8pTdeeJ\\\\\\\/fTSFRCUmzL5iTvCzec2ruwpUWuShEIzO7EXayWCc2yGlqX4vsF9yxIwfQcK59Rbmvz6y5RqCtiki\\\\\\\/MX3agxfiizaqOVcbWxor98Ni5YE0qJhizSAGdHuwQ+juSwkwtfXtN\\\\\\\/+JAhdWxEGAXqyxyIfn0yiU+cVMoJFsIG+vX8eLoc1a\\\\\\\/rud4vzTMc2f1yX56lt9EM1BLptAge+kOt\\\\\\\/4TGS2yU3mKLgFsZyE7ejc089Hj6LXB8\\\\\\\/uQSG28MZpBMDfROx\\\\\\\/DVTlSBvm2aDJlj+VGhejgKyUA741uGYJA80e+FH831V+Dy+irhmwW07Lb3X\\\\\\\/ryV2sQAt8LtEz6aCQ\\\\\\\/Q9kyNQYxxL258fM2IkVuIAyHnlapgBig8Hma52rEVU7S5nTJK6uGvgobP0bS+iMznzIbIqS+Z6BpDNoIOOL1c+80MKiuQVEi5u8ZTYXmv4nmCmK3LiHUCfm1oEVXJqPooqctjSSlcaHxMB61DNOydk\\\\\\\/xBBVC1Dn8MDewXBHo9tzcJjR5OM8SrXy3ldaFOuSY0V75UFwzkklo\\\\\\\/OwmK\\\\\\\/Uft9tvRnElSMxt5pQuyABQGvmUDRG9PL9Mm39p0WKeWDZQga75Sp\\\\\\\/x9SAmjYtmAf6Sj5gDql6eB7QXcZJUTyYcLn0QAcL3+cVGMmsQ2CHYzX3Tt6RAP+1aKkAkp\\\\\\\/5X3hTeiLE1I81TI5ecYAROStFZZFFwLxCUqDDfE+46AmgNTEYFGmzF2+XHGG7K5HGgiRfDdgIJ1gg8Xnyg8uaQEIr8pONKIB5fdNmLDtXK8zow0dYcmzBT8NkoKjw8ANfQBDgoVcRUqa+XEWJwA00xMBiOBpLaQVc0uRQp3H9MpaIH7gqvrZkuk\\\\\\\/irCTrNZ14bOScRakDFUeCvfdYoBYUd3n7ZFeGAP6TZOcVt1fx+Fwx+L82m1B\\\\\\\/0N0yPI9l7gvV8pdfGMLaOjET94VyXZ8iHAC02ogTmldamg1c9+N3eV++NE8DYj7+uHhOfR5aQjSSUBl2F2iESSEfdMJKJD7lT0\\\\\\\/1DHib3\\\\\\\/iCqMOZUd4xJ5AF7hvjz\\\\\\\/XlwV3ieeXwyGSk0aTVfD4yJZ0GE6zxsTzSsDlMJiOsbyjBNAzeiE+jYuAOEwMP2heuHAWlasBePnDj7nHQcZt0HC4C7xfGWpStpLNvkFz8dJZov22H1OeFNoBvx6ruUSshU3AZ\\\\\\\/ZUCTTghgxDjucaKklTVvtienhwnh\\\\\\\/s86O9rT1jzT6MTi2MW6HtAfX7jj5V77xUxSRpibyN8FRFEWzjm0NQ7fKWldCpxv1sWrAbtCEOjYufpJRop34fVvfEl2W5xWvjxxBZVjZynnL46EqA6JYGO2wtB7X+P+el\\\\\\\/RD7BB\\\\\\\/kW7TJGN\\\\\\\/eTqPHz9MFgy5390mI4nggGpn4Up6JJcWm5TB89eBYML4DylfC517bWxLluxKe6cZBxPH5413VQ\\\\\\\/D0X9dC9r1WKcu8\\\\\\\/9UBPlVO2AkYEpxk3edhRykmb7335KdKI7eN8sdbJZnyz8ZcG9fEvxBSLCuxUzAhQKKbok4SWO8Zy96d9T+boherS2ODt+8Zd6M\\\\\\\/11sQn0YV0x1RYjeTLlUSDPF+TCAu+i16IM5bYkpZpD3g8gLPcaowI+DmxNJaxq7u\\\\\\\/iWRd3HjkZJ1asknJHWTMoOdO7rOJ0V4rVtzKyXEO3ISHu5f+Ak+Lm1sqjyvTTxwYaRARZHIzr4\\\\\\\/og\\\\\\\/0Ok7XpsEHZbqkbwVwq\\\\\\\/3iJ6cZbiVDa8lMKTwY5WKebO+Kvvx9ybBZSkeQFDbnoScAZRDmyabq8Rq03t6INmBNNvKp8uAHPNL4EEB1s+cHhiuqdEDUODqY+hj\\\\\\\/RuZLGglDv8YdPF7sd+ciZmZcM+8l7m+\\\\\\\/NDBzNVNQ+fNVuahjpwCaqcXh7qXM5Yx1Tta0enLGDND3mF0ED\\\\\\\/fbiJiosQMm3zj012ywboXqc\\\\\\\/hsvfYU7SyC3csuf9Pi8QSVDW8C2+Aa9RPx\\\\\\\/xLsSERHcjYOxAbg7vOE4p\\\\\\\/pZpHYmQEOu5Z7wMvTclOuHatoFrIv7DfQAlkY1AkHWhgC3buNuiDnAvAlBzAOTWJ3wUKPzIHzxyZ\\\\\\\/3EYCJsprXVIiSPd09cTjbiTfdknlZnUT1lhwmQDxERVNNq5fCMqwk+khomSvg0SakMOinhs\\\\\\\/L4QNh1cJHh4IbUe3+hgGGzfvSTMr4j+6K8++rrGy+T1UtbVEWqf\\\\\\\/Tg4D0ZetP7C2CKAKBDVhkDxoaO67gOYsnoY5LwxXlKphpoOWZdA3cjLGtRP+k4++ratOYA3ikl7HGHVxFAkgDwETOUojgu3NUH\\\\\\\/PfZ6ZBs4dHs1njM7mWttNhMv+Dd+3g\\\\\\\/INo1a0msxu\\\\\\\/iwD1iyXllhkgoDJkGlX0k+dE6VUwuB\\\\\\\/VoclT+x3HZsF+cDF8SLg0mhu4R5bHW1kON\\\\\\\/dMAwawA5NLAgdcJ7AACoP1JMuc6You978svHzrQuohAIxQle66sWkxyDYxvt3dW8ETiB\\\\\\\/1q2lSeIV7WjXHRVscfZoT01O1\\\\\\\/I+NXWEnL1M4crzq7zE77WUEdMsNMGzugkwBjnSzcwv4yqbHN4+b1hKonvggNKfgtSjOKJVT7hySn6\\\\\\\/MVBFAEEUpioqAM57nJe1LDnQZhjUULT1Qah8dBF0toSalzHObfAJgHpJfnmB7bkhnI2AWvrx5VqAJPPHQtPAlwL5WaWwpI+SmrYATggWPMk7qo4Qnd4WrRWjUZumOl9iMrzetFQFP3W7fkK36hb9ORxucdJLqnWkCXlzK1L1YBOShYjTMKjCRAk65wa8cmvu9NYI4JZewp\\\\\\\/uovquqU2v0CAvl2+TZVcjc5vXS\\\\\\\/oYNtGFkGK0HuwitSgsMH0mq1Fe57f3hBVCfX4q8PDiYsZnin6Syn3pZUzKWhmlqqQc2a+a+Ip4IynC\\\\\\\/A0k24xHz9PHWoTv9As7MyQmSxC9YNE9Y1PdlsjoP7DKSVQCV2G+YkszYIyLfr1l3vhEU9RDdUWgZh1iYfu7sonz86FCOTw7qPWXA0GVH\\\\\\\/4Mb7Xdbz8h9FIU0e0dv0wvi8VkUz76+IRQhcTI8++GrCrDrDYwBQbRbzBOiqDsXfamCllxWsdwiN9niRflfAs1GjxayP11YC92GoPfbsbMpPPpfBYErzXYZKl+7qgSKiozFlr0PU\\\\\\\/sjjAP3Ly3LYQ83eN1LO2KtDoC\\\\\\\/yn2yfnBxlHYoxt\\\\\\\/8uIvu\\\\\\\/MIITr6d2cfAi6nYILmOYu5HYRGdUbEOax3a1FEvHtaIxliU\\\\\\\/2pmKVajYnsI9a2XM8iaxSBNzORMH9e8ndUbNLcLhfUjmwcxHHItzDxbfs0YtLuTC32aN9wNUaOg1gb3CE99XUvuxe0ZrXq0F\\\\\\\/iiwcXZF01mJnJzAyH7AisnN3LLAgjlyYeMihNoGmBnCaNwlrUvjF4u90IZ9558qUigIxBqKWhh8jRU\\\\\\\/qtOA4lO2nOM0Tx6zsbY5llkP5jCiS59eiAfxvKsdbUT\\\\\\\/o6LYq1rzWKV8VToZY16eYb8z9nLe4pXv6KB5w066Y7aIyxjBPlonL4BxJPWIJFUmYzeaJz7xHpRU5T7CCmS2FckcYfj4QyC\\\\\\\/kkl1IHudjSdl4JfwMsAxMsSVBR\\\\\\\/yf9T7KqQrolaacTDxRLk\\\\\\\/\\\\\\\/Fdl+SRTz\\\\\\\/mVc6Kh0IE3GIGgqbL9Vv8\\\\\\\/16EQrVTgNorF\\\\\\\/ZvdrPh8ygS8CArbeqZ\\\\\\\/aCahB4BblE4g1hCumjivWFpkoqJnS1FFw0hBMK6XGkUzFD3oxyL3xTmUpsCzl8Urz\\\\\\\/dCszMdwQusQ1MTnNvKL\\\\\\\/4ohugoqqgbwxwRoMtd6NMLpacHgAfiey8xVW2FbEs2GNfT24H+ebWcmPKl70+2QOiH06o9eXd9e6U2FV4I9WD4MqsPomvpHGYzHbyKBTFWIhAj0XtUzO1PFAHFGTDf4MhF+rU9KDg3WUHfRYjCD1KiS8Ok2GcMN2bC+JbadcD8188dpqUVFEbbEbEp+0e4GfA5tD\\\\\\\/be5w6Ry1eUgXP519dVY18IZ+9J0Bcc4192Xg4nrU2MH\\\\\\\/OKJoxNH4TO8JEE6GQMhKzUVsmftqqgUGukHJWnVmaBjIPMtU2L22HHhYisTyahnfkL7+TKt3JFlsz7aFvWUda2HXTP0AszhO3fyt2URAJMzxS5YLK\\\\\\\/7esCuSbZgumcdElNH37HMqNjznyfH7Bu6wVpRiFNa+ql2fQqowhk8cHvsPOeA16lDAg7QlmMOQK9HVHzLClaw4ZU9aFXqslRQznmTLMLOe94+PQBz2VgGcGv3xCwopodK+DmkY2a7AVdcH32ZxpR0zWiltGasIJHGkseUUSoeG9hxMSz07V+T1Eh5FYRobPEcE0+aZLe9WuLjFNkjTQ7AQhiETidNGfdIP0FbPsyKPrHOe4SX5P\\\\\\\/Yp9iYNb7AKnYOMO7+KdA4jj3LxdWl0ZQm7196OQBKSIoK8mqkpXZZ9YfIE\\\\\\\/7fVS43Umsw762Yc2WE6ZFlbuNdV3RYwn2AWFA5+xvVeC6SVmwz3Jimhur3K\\\\\\\/bbLwLbjE+vgvwpFPldhbJwBLC0u7uKLCC0lVRebRVLzKGo6rQkXjfFWnVUYSLASGupWbM2ohWr1gVqnFG2qlDEuSVMic5kPMuWBETy032XT3KHFd6\\\\\\\/iL0MUvcWkmMnk+LbssjGhBU87EqDTclFfVkszSIIL5DVOVKX975SBaRj\\\\\\\/YmBRQNUiC87ajOMw4O5+tcb3A\\\\\\\/KIvUhnnKVA8LEZg+HOi\\\\\\\/AFqHnEBLnR9qPmD3\\\\\\\/QGKKwxLDDpBp+V93vF9Dj7txf1hy1t7i4VRKNLqkBVZM6m02MJjFmoSK7utWK4zsPEcPIQ6ccI76q1zXUJRhyTn2gNRJonEZAQ0Id8NlQmGD0iitVIq+59\\\\\\\/6w1+na1LDHOJeIotHOLUBqRn4eRZMm\\\\\\\/Tk9FHiN686PU4+t8BAbD6YT5CjQPvCnQduodS0Ur4DniC59nZKgkH8J9pXuL3GYOdAhjv7V3YfnhyFweQSs5MaUHqxWen3st1WtfbGvhKEnAcAzBoec2NMqsT6AaO7qHxlzSB8CLD2mKzGTmFYfUr\\\\\\\/\\\\\\\/PUl+olPZ1F85ko2bzMKK81+79tHlSVh5dM5lHiSBMe5x3jjxshn+VVe4AwTfvpwx9TsPjjNU3JECOG\\\\\\\/h0zI5TfzKTxjUniClRC16mhG2Rb6cPlVGEQtaFooy5bNHj6I0zrpkN9yuN2F0kX+YBMPY5GRdQzj1DQ9ieWAlV6GwbtJ9TlOmhE1x5Nc7NaZUlzXYhnqDginarGMvSiwOCWm7buBT3MjJGNPwnFKf83KNoEQ87WdeGKNo8xr4ot\\\\\\\/Cd8Hm2+WkMp5MdW6GEpg\\\\\\\/Ef3OHhPg\\\\\\\/2VcI1GcxMPDZ\\\\\\\/euPeBM7tbchMUy3AhMPbyfCvfMv4yxdP6+dpjy29SHmwTzVig2xx6qwV7l1cWkmnMB2yq9NVGiSQnw1sASFhJ1vyyx2syA10wajnMhzNpIL4hxHhwZERkrl6v7QEd02cZUUBz0IbVzB7pVaSNJexdvSLiQ18wn1bjJY49vj0BS5SW2uqozDHTfAsUzONEidRkzJiltNskMxaHsowoGhMYRNgQQ3A6o\\\\\\\/8ndg0vOfSaeGxVVfbMPRbHEsDwUeEoD23d2yN\\\\\\\/n+lr2T4Js8EWzpuvzH2wAeH8kkOvm7U4jss13+K1ajUNnQiJaWDJXKbupaFM8fMVr2Cg+1CgKd2hHgEBilm3DHIjz9crGqyouBLpMuOUC4\\\\\\\/wIRdPRBsB4BtITrwwFq1mY6Of2H5RNaZgaj3kVZ8unyspEFSMjbggWGXvUKvCxEpVOEMR\\\\\\\/VBL\\\\\\\/AB1OT41H4dAIb1ev4fewLsqZVQiz7DBa8VDdkH5RgOQdgQuc41Xj4KvwoAB9rBWz+eKMKV7U6qokUPqigmCuuZZEKCMaXAd9szMFRyFrhUFCqOhitbb0Wxs37O72jbu6JjJ8aZ6mqbM4QZamVni9bHZ470elLhNRiP+hrWuwNCVmToqBCxFbLrQ05DFHecOlpEDwCJVxNQT1T7nmfAFueZUvMxnHlYs88q5IaWVeUtQhep4TpIv8I+MTq9qNqtdQQn9ibAL6WmLm8755cVG3HfV9ZO8KcG0vcHc9Wi5dKfEmmVmPLnPA+ZxqAsHSkifxfqARQ9S2NpfRrqcv0+A5scmCgwN1v6yuGVExsgKbhBlyPJbKcfAPB+41EAQp\\\\\\\/3eDnsM9cTHdrgv9wRU\\\\\\\/RNeq+26HJP9iJv4SYM9Aqqc589hrB2Of6XmnT7NoGsQkY8OwH64VnexrhvwuE+ajFnb\\\\\\\/riquhf91LFrKHcJrjboMtdN1C+3GL6mBdRz5WawwpIRWTwyubzMacxdh4KNMczvVT3WaHRLlqm46YESDGHW3hMyxRVvDF+qKyrExxYd5CPaZIiouzCwWz+5jFcZcb93vRX5oU+u5Uxt0So4ZMj4bnMqs1gSp6WKFvDleqz9nXxgVuBi2px2lbskZtmgH76A+hmymmmjpe\\\\\\\/Yy7GSxOFtq1jOPXQvigFkk6psXJ1Csrc8viXDP9AdFq6\\\\\\\/akXnRQhKapO1pPTwwSIjKXqbRQtfz5iyWTfawkDuXMqQQvWRZ3kooRJqK2pJWm9j6WMT5bB1HVtQSdgT\\\\\\\/p2yg\\\\\\\/suN0FyuI9wIgddU+s79gkDXcG11U0ikFkNmgNoi88rjvf+iNKfyGrjnNlLXHvw18ad9kLvLts+6nB5VMnsxnrPIyKrzALWGjKkb+lZktzzpsEOoCHAtkEd19xnHAdEDXxhrt2KUQchKmS+CHd21vyg0zswxYe8LQoQLqSYrzewjcedRacsR2pBGVfgMQM49U5gjXkS4DrVHX9AcYmyNyIFKc254EWYMevXTlyyYdjBrG7d8nDmXCVYgIf0m0cho5PVHs2SsjGToSlDvV6D6EYH+HPqt5DkUjKnlwdIv0INXK2Id8Ne8VGXgdNS2Y5RwlAqr2aQrhnv6RiDXIgLNusL6UfnEwhKET10srl4foJ3m3\\\\\\\/Y4STFVEjMYOPXPbSIbsygUy6CjryxId+PGcF5X2uY83be2lDQPNJJUdL38vuWMmatErbXV8GyIrXbTDTxadVN6GBgtSxRQ7FP6HbAuyOiQSDX2NG8LoYHHj8nYAuVkEvJprksHD7JIP5mnliVhvLe6JtlUwF4saoaWFb5n0cy\\\\\\\/AZIG\\\\\\\/DXnaC5JyAMvVqiIY1EMxTxaF4cKsYA\\\\\\\/bwYxeVekVN1o0kgiwT9pWsMV4IR4Zpy8pCA1flcw5mAScCwbhtFWWnCIDg86v47ri68ThV\\\\\\\/BjCQaGicH7++iJVyNXlK073S2Aa9\\\\\\\/lQBdWhR29hXgVil4FHIgJUW+c7n6vDoWNSYUeDXX2aeh+3dZeB1fyZ7bRcZJTNDRcRTOJNTEuxgP2qaQr4hwQrzm8LdeqMMaQ2WcvSwQjOSje7FkMyjAht6eNp8hd4ZckFhx5DUsxctucmMvp2soQzg\\\\\\\/ROl9KweNaNi13WHq\\\\\\\/NP0Kop1NuOmzsGEhYlvbfCTlzq4BkWqQo6goiAofVtGwVR4zl3UJGX8URBIgKRL2K6xZswfBhbY1yk60i29UapRMxzi5aX5rVBSZ9QovYm1jCiKYXhYdbuLG5bmNFhjV7H6Iu4tL2CrgafsPNWNRL0S\\\\\\\/tVz6rBpPp1dcVht68aPMWKumKDT\\\\\\\/QSAQudY6Q\\\\\\\/O9M5f3PS16FslrkkzZ6nVV2RV0ZgSwndEaazoVkKfpOAYo7392g5Am8+2KVcaLxCnq6TXaxoLl\\\\\\\/VuVaGgVOVua7RGa\\\\\\\/yxdxUgo0Tb\\\\\\\/RP+gkK9\\\\\\\/5Ks6CiiEDJeXC5I2FvMYexNSGaZD4\\\\\\\/wz1KVlXCxqelzPsBmCVFudFYN9gIm5i32nZ3M9mtS4YD0J9Epi+M69JPyQ34VV037EJ8X2YkcVM5XN+ts\\\\\\\/SwcZAiX1kC\\\\\\\/xbxtZm2n7pHD9s0rNmsJMyL8JvWfMkraNVRBw1i04Uac5f3QTOymlggpavmR6r8\\\\\\\/mme+0kQlurbRXMlobjOFC40ZQtycyn\\\\\\\/vnl0nzLR\\\\\\\/y2cZ0FDVxcAoJwJL0kJs+u\\\\\\\/PAW3J6HrrqjcKQP2\\\\\\\/x9ghUJEtk4Ha9yQixQQxmRUCEfihJbzcPJVXTHlA5Q8yVB5QYTuwBNT\\\\\\\/Nx2r6ooa7RTeYf79EyqruR1NY\\\\\\\/lXS\\\\\\\/HHRA0HTZ5CTXvSFhavnvs+vF1NnVnsGDMiPerHPzj9KYokVIu\\\\\\\/vAIJVtbhlm8KedbSgExO358a7\\\\\\\/X85Niw+WcvzvMASjNg\\\\\\\/ywxAbwdYJYYxuUYyULFgsJw8+PLG6Sk4l+as6AMdMq6G7yFl7X3qckJDLDk7hXke8cX+FnD+nu\\\\\\\/ABe5ar6UfUZlEeuDMVQMmKOcj9S6QKZrb71T4VpHhnPEIvQi2uCWJm3p5KbhVNu71vYxylIKG2kZFM3rrCWuWWP8fFenMwp6wpGuqvX\\\\\\\/no6BBNhU2EShx\\\\\\\/xvyhhpT+imGMVhEzktSVlpllYNtHiEXWl5UyaE4wejLlfIrc7\\\\\\\/dYpLEj59+80QTKMLGyPqUFVWGH2wpB7qIFBdUXOHOXs22guzwwQVgB\\\\\\\/Rh1mq5MPI0aqJOiHnFcV160ZI2qbanpjggmXniqV72OXbVrAayAPGDIpYzraaQhh7OR\\\\\\\/zcoPI8V+FoKpVxm3mQv8cHCCNPPpWARbuLp+iQX9ASNFNQGVsSMwv\\\\\\\/9PZrsYZoGW6BmfCijG86ZhtAGHKFpQFJxwhcbcdCt4Q40LCX5GcWVN5gK\\\\\\\/ytdUEYG1CQ0g0zXkcPvVQkmDzqvzMk88tA+FDclm+UOwRAcWqsvY0J0F8Fmq706C7iEdm0QbetGG8UAdYMlj0Y\\\\\\\/Ohkf+SXGtnaVOUtd2gil7WYpwQlE2jsW9jIrDLhP+v+B\\\\\\\/nhTEkzDwmHQhvHGbtzgJ7ftBOvEIfM2YPjjTSY5yCLb+PS7w3T+mli5zgJn1Q5QXZ9nn63OtqzbGcsHnqZP12QOVg\\\\\\\/KHi8qNG8fMupsuPJM8ld0cCDGV1IVGf6DY9O6vlAxdiq0h5DHaaegMwEvA4WgB1CZ6XZo1ErQlSx6UeDfVZkQk+r1asWFJxgl1m2+e6jBL6mmBc401FWBbvMTGUy02aXkRlt2r0lAEIx2yeTB6wNAa6jvb3kUSkpmPo32EWTyZyGSDwJCj41Sqdpr735KAKig47Bx5q6pr8pgRID4JZ6VpbmhUMYJrpiRLoUk2LM6qa4+I2qsB3KkULxBGD4W15DQTdYI7USwiY6+ZMws+yoVpxWpn8YVj0NicIA9daMxYQSWIei1Nbgiq8K8\\\\\\\/cdgUNScNb89CxqLzciDsfdrNOCnOH2LkyC6D7tD9F1OZUdoxh6FgqwuNM+nuSD7zolg4rLbIo4kVNqKMynaHgqRLgC6NBvPBOTCT7Tfi\\\\\\\/+UndEZJCp0nGzfTLZKYGlxqzQPQxhl+KOiYX88qBOskwSZGQ9BmkeWR5ydoq5SFIS7mZZDNNTS6v2ONCVrIG4eCGF+Zy\\\\\\\/l+OavgDioqqvp76MIKHpMU\\\\\\\/HAY7HP98VeIBO4uuwpMP1T7o8OVYwqVEdZLESGQQBeDDw8ygtOov6YhvNvJzBuUPcOJAl4NwbiQUlKe55ZBTNLtoSHMb03KieYMyBjGPo0sgmVpbQtj8s2w5xXRTRhfJfGAsEEC4QpLXS7BNy1DrDu+ZbfobgWdBEshOK7ChFG1oRog4noa90DfPDUy23DdnaD99PWDS4AkCFaysIrMxpStGnKkFdcSTIPgFS9yahIsbSypqJXnAsp4WLKPL7\\\\\\\/zHpdmIrEAc2Xavw5AG\\\\\\\/k+ZlHtBkj2ya5a+\\\\\\\/WvYT9wreS4R\\\\\\\/AjavdMT6WuLOeflkyvDVoQbzP4+tmK2ef504OterFcMEYKwaYcMgOdVoj7QVXcET22HwVJO7AluRkmoiylWgSPoCHx4Wd7lJl\\\\\\\/coLRKaXtN\\\\\\\/\\\\\\\/a1U\\\\\\\/MGU2Q8M3e7fz3\\\\\\\/bkIJh+ZPStvUMTzt9CcoPJLaEEet\\\\\\\/knqun9y9W4h2Tk3KBLvnm39+Ze0XZgtCKthgEmR0XecEeGNZUhMf0Wim9qy9UyPgplzN5KgElbJSHhCqdB6SSTkHJ3SenTASn8qa5pjICufSKt1NcT7ObAcSrFEhUTcpTbvjmSA9nQyyk8nexgl2pvqRs0\\\\\\\/bQhQIh7vnfUPF8D\\\\\\\/zbpJ\\\\\\\/QPdsPGIf0KnccblfTUMVkzXmNNlEJrUPUBzkW4tPcBqrgDSuY46J3mKhqlfyIBuiQF2+7fXBjZVgg2fW234XBcgE4iiYdrqJP\\\\\\\/eafjO7lwsOLGELXz+CrfgGlIpMTAmcEE\\\\\\\/oPRMdgvE+VrqX3S35z4VJqt5zcD+2sg5eEGRXyfWSMOf1XSq7JOj2i9amW+Su3EhAu4ThHEdBJcpXAtksOaQqxfnQf5ZC1vD7efLhYge18scdiQN3LKuoFRU78R6IOX644TTBAN3WCNMOcxSc88sI3by53yUkhL8eDpumwkOt\\\\\\\/9OyKByNOzOEiAH+K\\\\\\\/+VXrHuWJvSrLirO1RMhyjVdtEvDjIsxTQxgUef5oY3TbYpJIbDio\\\\\\\/Md5MWINwA3gGNh6aFgJn8af0YJIhTQYkiCwHQSrXsPbYXOgqTXdQbo7cTgiFfLEQpnPo7udr9zkdqvcFGVihd3MlPVd6uXKt4H8OKjwrrqBbiOm\\\\\\\/9J5mGEs63bm9wfskhllnHwLGl1SMqZRR2uIBALn189nN+EINgbM+bffe0Ff\\\\\\\/BI4A+8BFcannGPQkczhQ5aZzWLj0RSqw044dITnKjyEk1EvtJ2q7oYxFTi7ph4zDr7dmHvd\\\\\\\/9HaHW9SH6jTv4F09Kipmt4Yu+KRPnJrVTWo91OUt56pVZcVnCd3bJqbFvOqHLF23S2v3g1zaXpGl1B9N3XNXYaDRwsjCiWJUolNV3AefSsth0hhlqNLuXPYgkISyomdneLJoxbtehyu\\\\\\\/y9Nap1aGyVdzYbAOIeY3swPWc+0PqmkKKTmubiHR849tFRUjScVu5psgDySn0eJi+TEAaqYpS4oDHAKNAZQdE3sLd4mk\\\\\\\/t39E6O1OAevY5xScM+hK9T86II36S4wAdHkAdbIQr4qhIodubGodnjA66FIO17pmoB0dGd8pnVzLagyC70+6Ra\\\\\\\/cvf8QxXArYx31Vf7hlMX428PmkF\\\\\\\/De2G2TsHP9TEyLrGTEbZCNb+xEhkL5cCUwHT5pXSEQkJAkKPzs5CDEcWjDx55gdw978KVfibYNKCmp4dU0l4ukHPBZZKMgLtENGZQ7uA\\\\\\\/uQ4pGn8BbelLayd3DXEwqECMtuz9OGf0SCC4JrDWP5QbjV7b0CdEAFsJlLXASewBhP1karM9IyAo3mAa14014i9dVjd68ZnGGNUieRGA1nj9ov6e1Vm+vxbgTelfFd49a9d+b3uM9flkfM6Vb5wE92Z8kG2jkPPSyma\\\\\\\/J889BYHeHzt20v2m1pc+qaciuOe7BdX0xuWjXfDr0z83+dE97FLVt7LoziADldBWGzAp7a\\\\\\\/zbLR75pEBvxJwzhNu5RFLPxBv1fxzlu98JZzHTPijGa9SCiNPE4FpOfyMacrKdpj3rXrAoUKC8qcFntyQQYbTGQRee84VIy1HzKAO6mjukF35iOA0trAk0M3EFtviBYi5THOCinkBYlD4wPnQjKst+LSTGhHy8MA96umaVtOjYhAJIx5i1J\\\\\\\/gPcoTYw2VZICHUfm9N+Q4zGC4GU17aH72jgXwaV7Zrk3dFaG7OJViiFK\\\\\\\/1W2uRUPOKtjGuOFBdK1w6INb4EopWEPyxcl9RE\\\\\\\/FbRCcMRWCM\\\\\\\/tlX+vqKk+92ZepbFvhf+PNbmog+9wyFrFrmDOfYLPDs19K4VmVzUnWK\\\\\\\/7ccVYYEfbdxk11jPJVNc9wx4lrZwYT5O0iR+xyrYfKuEMKE3UNzEP62t2tJcN2R2VWxNU0c6GHihusoStLO6Yh9cJrEcT4kkN0cd0zWebnq4sLaJOTDDFBXGagqQMg1uWOnJbKAbEeP5WAWvFHahScUC8olr4fTxdHomZHU8CPJ4uf2p5zCChpqQxpkJFLq6szTbx+bSqdQOATl2iJKyFK63znkaAtWSN8W6kUXTF2TQaMP9jAGa04zCUS6v9IB2kr9uffJrIsobmnKxhjBJ0n6nt0OTNZAYzZgg5QRqSqqtImTtdTlNbPeQXfH3ALa5Oiqc08Pe1keOEVBXF4epif+P3PM1C3fwi9QVKmqRiFHE+Ie1fyhO2lQ99fKVIHeyvviF9GHlTtV0lyyxScr7Vwr1hJeG7um5FFPsyOF+Fqku4\\\\\\\/Ewy\\\\\\\/A690AIOx8TrbuVoqPOPm8MChIGCEvvq1V24CTf4tA\\\\\\\/64uPmoPdsO+pEMhm9cc\\\\\\\/oIyR59\\\\\\\/lQ4pf9+RK1e7BislyRCf3H4EbSnNPLmmEWu2\\\\\\\/I\\\\\\\/pWcs3zzvBv3sZMRZllAQPSmmPyDKKokpP2wCBaNTXumF0iUD1fPUxJqZvPr37pQNSAuVZOIbgEt2idJ7e1Ay5YlkHHLtFpvJxvJvLxzNDPsT5NluD0L48X7NRGq2KZ4B86OhDO5+QvQDoqnwouohRW35hSiMScEfl1QqPDI+nHIPPGPdeS6Xjr+J9xGWE5cr1X+YB\\\\\\\/tSry6fa2\\\\\\\/sGEgZa\\\\\\\/rKodicJU78XFy+vMlAcu9SMwAngJ0Mfe+1L55yBI\\\\\\\/jxCppPFpsxfcuZNMYx15tY4FELuJ+o3P3TC5KXc7FPr70Xd3\\\\\\\/jsQZ1adYYJIjVDdxlCYC8EdWM3tz\\\\\\\/SRO1c6nYov1f583jliH6\\\\\\\/TiFwIS6NWe3F6jnPpFU9KtJTZNJTHC38jL54bbmuPdeQoaqzKoGbt0V7lt4zYL+vz7SAzq0vr38VZW1UClyQ54Q9BBdSHL9Y4Sz+i9UQVksg3QiTCrPfimiq+vEYYZK0L51nnADzk3E9GJBdiUN9YzpKhoLucorPYnLYYKtsVlzUF8aTKIYWDJy\\\\\\\/EiB8Dj3WCx2usw1nxsskQ7QCCZW0ZJ8l9WS2lhukuP+CuobUMj1YW8XUGT1uf5FJ4enMSKHxsrgm6p4zMd2Pf9OyLbg7bmyJCIDFLnIte4LXmZF3OwXOgZk0ob7NYjqAvgqQSobun6M9Y6WiHciS69eqeaHu5tWpCJFHe+WmgqeYSZs5wERGpDEPjU6+Qp567F9upSDBNW4Z9i\\\\\\\/o2FylQEr6mf\\\\\\\/YgSeroOvCvA8d0fUcbu3bTjdKbBLoxnaXrpthbU3ryMnITzilAhmH5ti0dyEjzOM5wiAy8k8\\\\\\\/HD9v1j8kk0\\\\\\\/OkRLqJlO5lXVjjew0psuYdmHXnzm0seymADV6bWN3+L0y3Ki6kV2O6+rol25VdyBWELmX0p7RKeecvw7oy8SkmAiGoFDh1rPLxJmce7MRWqoW+vaKbqmEa3qdg\\\\\\\/F7FsLdJMCJVIlYuGWweVZ2+zcPF5BJqMo6CYg\\\\\\\/Z8M2zFO2Zdl\\\\\\\/EpRXi1ZM\\\\\\\/6iXNDA6ekxKRt0hFcCi1IOzI7sjU5E4Xduaky\\\\\\\/UekH3t9rQyyDqj3\\\\\\\/bUPCJnCjRu+8SRD6q\\\\\\\/bMBtvr9DpaX5QByrGvaCDFpOgLtFIz5iHit\\\\\\\/6fjQ\\\\\\\/U4NkfIMlVCW2jvTkekyQgDn58RNSkMfWLU0uz31WR6FJEQOatOdIMSlhauC9g2yp61VU60gF7CUJ9nnZh7Wz5EOif8Lf50ObhypN3qJabg8nfZhGmGMB44l135mP1eYSIlBlG3dS0rnysIOu23m+gwaIAIhcB0FGKhLyirw5Nzfo6\\\\\\\/WtsB4sSg\\\\\\\/eJFwqrNxBRKayLr6lHqmu+Nfta2BvGN0tBXR+\\\\\\\/IxN3b0qANcrAdCRYFDJxzcXSDRqDeW7UHHaewTeZCLKUhubX8wRXfm1b0JnPX6t6v+y1CvTLf1QrTqjdS31BV1Eqb8lC8p1VHXJGOGk5Tu\\\\\\\/nnpztZ1qVjVdE2HSF4ElQcbClSvZZndYOMWGYXkGMqgF3qKu41JmDEdUtVoJfTxQbuKJ6VG\\\\\\\/5AszJvYCQu6RuHO8p9UQWQC5ispOdo2hs0hZYYoR6n9NA19Ak6HhnYQb+j0KVF\\\\\\\/5OpqRU8xVWlfO8ARhSRYmA6E2EHlkxNTQ0NKITA6mvQ+YutjRQyAhog1azBG\\\\\\\/6c4N7wscpjXg0vS7eYRcZt18nbrBeju8Y1IUP1M8lW5cV6tPZM5biv6ecH2inwZBz1hJ5ieADcsiKk5zW8MJtJvILxW\\\\\\\/WKisfHHXlEg4saG1JG2MejMz7oIp3Q8I\\\\\\\/d\\\\\\\/rQwK+U82fW92j7QcJfhJMuT7D82EraGnFEWeZvzQtKgt4UooZctGvVTOFimkFhd9Ww9Xal8hSmsieaMWnQRQGFKO4RoN9LvyfURcU\\\\\\\/VVq1hUoHCKABOEM8ehPaFUsHgZ\\\\\\\/xMVO65i5L8yLGMRQg6HbuXcXpFgWW49sODvQKUF4VAtL8b16YQiC41hRX0onAFWMj0VkmP4yw2f0E0ivobQNwPZsssW+dKlTWepUAo5LGbeyN\\\\\\\/vvNazLzIQSTzHqGUuQ3PAItiHBCAzY3b9hGrZfaGgGwsQmZNV0tC9zLHBAvxSavqNwKY1rTJpgwmO1IaVRMZbS+AbQXNmliirYGgvdraPZDq4Qw37tkNPOyzWNtWfa7cVqIHmHGotc33HVW0NkW2i9hT+MepGaLhtdCA0rJ2pgP9fS+ZvybvBhQcleQXk4ik7MVuEe0A5G3FLth4gZcsfeWVgsCUeF4sM2o2\\\\\\\/R0kBmYDAyWNFEGL0z21v+HLJRuadrN90tv64cz4mJRnYZamLpN21o8UyAPi3vE8Qh9izBwb8ph9ojq25vgI1azDQNQtaIs1+aGT7pfLmeOtSXES7g9IX2LqlCT2ipLGbMwsgD+bUt0rjAXMJ01SdZDU5ddbGVpS8+SQhDIyacI9LN1TvYJ8VNLriJJ+ViTVK81l8GZmSUN+PtGuxoBy3KehOLm+Xyy4irxeVM5YcAgQfIqSjSgEfA3xdAfqtOZaAVD2enmzJrYcWkoMEqhhFJG6EVxtNrzsRGfT7NqaLKM8J5OD1BroY7Rn86V0bfE6KXNfdmAgqP2SMtEsW5FV3urIS04VTFbiowWBtRcxmrIJEwmYyvKS3w5JcblIUArXAjWVcEEeQ4gTDEGvtGUDo43GGEHHWyPmcPiFHfXsy6wBV1ndk91cdGalH0I5IzEmZJQZHCSIy1qXw1pSCtUDgyjuUxAei9o8EStJq310QpGrCvklLLya+8eGfqGROANGaD3d+u08+Xl\\\\\\\/XGiAAdfD65Ot7KvRr6uDe+xq8DDdqw90HKweOUKddx1oP3k\\\\\\\/ka3nzxzclbjHBDTOg7xoytNJgi5xiyXrKbK\\\\\\\/mPO22WVp3lk537m2+7C4QgrxI3nDcX8zxgmVNQ3j7webH3aiwV0uwAALkP6qLur7Cawzd9XYnq8EcKBgipOmnDDyrax6+pUkmAKuZMkQRd78TXqRkfhqkO\\\\\\\/uZp8kjg5d8yEzuT9Vcvy8a1+AZ36ob13qbmulL6L6bJCv9VxeDOyduRy80jBctLX68PrSggUMdp0nOCdSrUsFTshQSSanflJn5H1gNrqifuiMtNsfzXuWAf5v+RtoVicihYisyewlKO3pXy3gYsu3d03+i234E\\\\\\\/0Qet5xM0UmQSWgVDaFZN+SJAfF4onUBW8DDmXBMaaxLpjKT2w3lhkrfeeu9ngFYF6ne1slWqxm7xvAkDNWFw2JbLj1HwaEIx2cqy7TGnlhuP2v5f3FBo5zPxNz+7yZtvz0vKRIARkH0kfECR2E03fNuRvPytSE8yoGLUIdK1OuTWQbORU7qlhDeA63HxTTRK\\\\\\\/FXFhhbrDF2R\\\\\\\/zGTiRura4tvtv3Q5hmTJY1N7U2AJ5z1JYAGbg7jH5VECquvNC1II8Xvgd3DfqppPf1a1pgIVO7AbcRylTpgrmMyBB+KKXbjr622dpBVhwnqBUIHfGLQ+e6STLjF3XXTBk6tgplVNvFlsUOk+LH8OTfJ5i\\\\\\\/lk5mQmugUzrbTASOEuV1CRCr5\\\\\\\/PM1xqDHgla4mCJB\\\\\\\/LzdavwbhvmRwv3dMmeyizermz+Z8Efg5vdalXiRzsi1kRGvsHT1QVK7faLnm408GX0UA\\\\\\\/t2VOGDGaESGfcErWvXfBihP+ie2DsJevX3E1WldaoM9iiRabyuFJJ6sRMt2vhCqe4kSRdu4BOCfvmTskg+c\\\\\\\/7y8bjSeoR+iGBVHdurljlp2okR2I6kHpwdMu2cBKao+vWmxZq\\\\\\\/qBQ+30iNaXPM4WMex6o72Rx09ybJDmpvJut3dctfFItgqiwJovKEyTylbvuzAskT95TjYDmcNzg8CeuDKypdHEaNg4shstZN5foKqgeof6jB3wLSdSW9Wq0DElAiA+b9bdkf6Z7kYexJ2L6F0i49J+lk3sNVlt7x3mW6yQZJbv5XUDL1Hubi4R6MOEW\\\\\\\/t+ynIchU9xIt85n4FFNKHhMAfIdZIqQ3\\\\\\\/r6JBb7BLRvvaqBRaIg9eKK+xDwuzR20NjT39ILyjEcuNmoaPQIge1fa2nzMFzy3lArH9o5j+ghvOoTU1cxCbceGTNv8vit0cXmnkib6fcXaTk6NVt+xzBqGYFgdTSnpM6NFrB+QYMxHBKX7D6Zm\\\\\\\/ly92TsJD3fUjbVpCgEqxAV03KLr23QSut0drx4VTUu9faZqPLLYepUh+u2uVCYWTwQjIwcODPeFmI2u1U1xkoh4oE6nUw0WIMAPtvuTAkYTovme1xd24hYpfrAAGoszKjUhKQW0AbOtL3aie1KSe8zIwxgGh78awrLiu1QVpZLXJqwj23qalPu+rbuKyqXCd5mD6XwTWQernMUWQFxhMCy3688IQOcrxBR+Aj05AFP3Tt+dIaXNgz0FOkR1KnqU6T5yXkKu6VOFP7h1C4TgnX\\\\\\\/M6yFG5h2y9xGFbBK3o7FCMDMYWVmTrzoy\\\\\\\/O8UtkEu3DdgmUMc5PptOozwtO9EJFMED4YMj3sCNaru8hyS19ESX6qUnGNk\\\\\\\/aH4\\\\\\\/O+t8sV6quFhlqmKvxwnYTKvdlXJGmXPtTyciAxIVWA1AacIIf4xl34\\\\\\\/YbMM0p9ySCAyCDU6JW+tF+NrOWdqHCb8bp2bH3W10hm1mRJxEZSa5yUGQLBy3FLbbDXklMEnfaG4EDgLjn+SefwJgxTcfwNST+dJIQNCemSxhy0AemEX\\\\\\\/q0rSgtdGOgD7UhtBFn1i718AQdpVSCjscOJhJ+ZQZ5tmMDwY3af26KQWxYcA6bs9npsgstf5s9ShXbqzAVm1KC4YG2SsNz3agWz0hw4+eQW\\\\\\\/llV9CiwKNIjpqZLbD8ydKU1pFnVmVC7n\\\\\\\/nSstTO6IuTSf2ld3t7iMlP43MzyEALfLuT79GVZpAdu5jzGwLu91ZkbIO9VNP4tO3OBu7F5nuom3nPnYOwcAoJiFuRul9Xktp\\\\\\\/CY8ShyRb\\\\\\\/xv2wvi36CNaxEN1gaGkH0VlS2ag4r0P2vB2stivDDFEYjc\\\\\\\/ZN0h7WtlkVw7n2uFpzPM6cY52NWsPvhwQQR+QzfQLhG5HsFOTO+XqkMyAZtyDHtUoOokVeDWkEXmqgn5VUz55+BRd5LhRxbrWbtm+F\\\\\\\/cqJ0tjVMLs9h8k0A+9ziaP2at7JtjT3CxVs5u+E7Orw6n76EWDXs1wGvkBDcRjvsXrxiKDZ2mmxJliq25VNZiSx0w+DFELxBDK+uoQ1XCWGKCdVf0hHgYPh6IiRsztu8ijo4RnZJl3hg4Ir9dy1uxI3c\\\\\\\/0S9PifNo4ScYXzL52SMrSzrD7sfPs7l3lV13Ao1Ic8R7P6ek8R8M0XeEATTl71ZA5HFE9ILXEwzAiQCW32ZwZxAofj8N+LgRul1r8ECcxlI39np3NKZLlCtWY4cY+qZOEqH8\\\\\\\/EQU9gAJ2Pl0butIfHEG7PnZlifp75wOegU8TpFNWfY9k6EFs0tG7CYOj2PaY90mEzKOm4+LxyyBRKIi\\\\\\\/B6XQZuxeJM5JdBPE3u\\\\\\\/DogdPBd\\\\\\\/Mnmjf\\\\\\\/YAKt\\\\\\\/9dNNY1X8iy2IAqO7sT8D1maC6uQ6N845NVbqacKJYVKdIabuB7f7v1oRmMFRU9yHQLpOvUUF3TCRIp1vlN7qVBdd9UzW1Tn0vyhfmB8rpvGcdap0xNtmhXPslaL+UV7RX5WrQzx6YB5e95vZFEOB06oZqgud6qnD1faxW5Ez5CQuIEgbFkZroD2\\\\\\\/KW9PdKd2IL27PoddWkiMQY8Bz0txjsw5E47xO0ZtZIIOIi2\\\\\\\/Xdkbp8JR0DfeH\\\\\\\/6EXm81m\\\\\\\/9V+OI7WNA9aL+pM9kftT7C+Y7UnoC5hCL8+28wVgMXxjcDbvySNnd+13T5Cqmh6voGncKBI2sUnT9JMPWp34xVszPb+njiCOLpss7hjoH+tOsmRfK+26sGpcZl3Z545g8qBK\\\\\\\/WwjwnMrf80Nc6OIWoO\\\\\\\/JFK3jr\\\\\\\/YFI\\\\\\\/SkaJF8ofQi06JL+iTap8GpLJF5cJaF2lz7Uve8TZcRaNjMYuzqoII43DLjW\\\\\\\/gLPZ+QnR42gtopwBnibbvTxmWEb0rZB\\\\\\\/GFALXskSFsIhDAOJSfO3LYdCNztJHH51In9WGE2Q2osjnMdGklM6s032aoT9hG5yMKjcxYc2M8BRP46ZD0FDIAqzknt6ewTBqKIvxoYYkQfB6rPmKeLoVlXALGRr+tlBHIl29eOj\\\\\\\/U7kipKvqtgh2cdavXb9LF3\\\\\\\/7fOl+9CJ47kGkS9qQmlwX1fLh2jsoIZi\\\\\\\/oFmFO7uq39te74MSofv31vLffhuI9W8rV4Tll\\\\\\\/0pCuUKOVI1s3K7RHbvkso7l+qWetgsIPM0j9x0Cs8uA3AR1WPjfH10RPzeJ5jnohhTCqlewCzLV8EKUstWrrNQUjX\\\\\\\/iSSZlWaqgn55G2agQ\\\\\\\/iQi71fZyVHuciI3Ek8cCI085B60HyTs4LhllGkJU7X5OlLHlRLiYUVlRMynfNYmEJ989KFvJqU1WKr6BSE+xH61J2jANHfpsfUCRWuCXX5+Nv\\\\\\\/7LUdn3rzS8\\\\\\\/RqueRwq0V1f\\\\\\\/mWNhZ6d+YC9RKVkWu8KruNsx88PalaI+itr7I1blayajdPeyUOg0+lDVqzU5oLuJXBa3iqkVT\\\\\\\/3jc2RJPQG3wKHYj1uggSwr0bXwTBDHwy5aLqegnF7R1\\\\\\\/xZzTjS41hQsprRt457OenfIIiw5Ro5pTD+TDzfGW\\\\\\\/qUm\\\\\\\/FrN899ZsEPOyjZ2SFZqjCnLZziSWrX3vKOL2GavnjGa1yKXyhhCwd2iSUsb03E3byjvT4YCM3Fl0CZRxpSGoqQ742YseLB\\\\\\\/lpoVCqbCCO4Pw21l0ju4K+XxxOiXb\\\\\\\/gA+UAAi+2n6Yuq7pMOWXW\\\\\\\/EV1yo5s94yPB0IaeOm4PHGoH1+H9iN6a7C8isvxrfe8\\\\\\\/pj+3Bm2u2ZyqW9zCcoIdxcmRM1++iVcUkgnLKjd+iU7cLaa5Co4ClqVDGsuLXiOma02vWIfgWJDyu0kREeXRfChbbpdKUCnQFOM82xqh+VYTzeHZdeFapB19N1USBNXP3\\\\\\\/JtqE1qBBFjvx9Jve7+XURUN1jxT3KVRtXoTh61XHrhLsYZplNJ2\\\\\\\/6bUFkxu4N9iHemqXpiF\\\\\\\/elqEk6fYI8rYgsvYHA1awlBDyJe\\\\\\\/31wwxIF2IT2D7dD4d211TlBhQ1bTtvcEAvOSoNi+m7eINjqbXz84ZyReh\\\\\\\/aD6P\\\\\\\/oYjCXcOvXROmVELebak6DIYFAO5hDAm+BV7fH1ouE6sat6\\\\\\\/6blQjzrNC8hxPRcgou30ix+kQJ6A4VvKTQ+bcNlDxBOQG7\\\\\\\/GW014L9TeB0oPD\\\\\\\/JeG6sDiDwB+SNuELsjdiP0xtqPpzZ+LWRxx7oNXqzFvsAdBbdn+rqNpVO15yGUIGkDH97iEODzOidXCSaZa8b\\\\\\\/H4kPrVyyaCbObaQqcRQXS2VUWOcy5XfuuQV7jap9mqdyvc6Qfekel3EvgxdipjDsRKjoSL9arLxzh9JAhY2U4EV6dRqDxGqKMUZlUHdISLWTqBwowmWQZq4xkB30vcpWIIWv7X+mibvcDpmyTnjHWccWmB+EojiXBseOPDxEBDiJ614Gg0R92MysRu7eqEb\\\\\\\/7fI\\\\\\\/2hRgYbKpxeY6ejnp9+Vkxw\\\\\\\/VXEk+ogOEC8opmWrXVBnBLGo4eLCZt6zdSBA5U6rZK6O+W0W4S7kT8F1nGAF9mie4Zwosg1\\\\\\\/dNAaBv9lzPn32YldpNBuTOyHB8yi7dynpXiR6yoP5ILm5yBS9yvUFBHICbqzkyzgg0z0YMbOYytu4a+l5Hxrg+aOTA6GCSZTw9JPoJVd8tk9mh3mZr+KJFmNWXR40MWc9npo2IzFlnNY\\\\\\\/I5XDHJvHLAriC43ggPlbmh7sg1npuu+x0\\\\\\\/eh8bZvpN3toopTruIpFtTPqbtULD+NrUCRWO+t7tPSpG1dRP++2eGf04KXgv+ODV6nXCG6wRPF5dIDIh1jovhpS1u6OyMM9I+gY97DHWZR\\\\\\\/qtZtedPp\\\\\\\/yD2Fpe928iMZyMM3fAsXwJf5wE2\\\\\\\/6VXxGezlYJT9Z3\\\\\\\/9mjOLL6fyDlchZgN5s4bWvhOMLpAc96\\\\\\\/sPoa\\\\\\\/Yfq1WpmZ3UJ7vqn\\\\\\\/yKI44GiMPL0UsEmbFCwCx5C8zMAU274xKeDU6V25BFt4CNkCC3qfIdvtozfVbxssYHkG+WuPxAbT12yXXBW5YIJB06b8t9SLzg+ICnFFnutNrTtr6RqA7hWY2u4RCZDc21uNSFq1QbiC8+R+qO6xTsGYBKpyHXvaRy44lbVjjSxrVS3N4DkrBwJPk6qL0IK4klDx6DsClMr6pjuk6FsGEv8odiltle6DwewGZADp\\\\\\\/lbRunCDXGLTv55tcWCvmM8onZxdUnFgePI0+LHWxOvLC96+e+sy3yCncI\\\\\\\/dVoQCUEFobR4Sh1pJ4aNHmXPRT4y1gQwDjgECVn3CqfYCC\\\\\\\/n9jGtJeZ68+DQyJGpxqAyE+rZW4p7XkwvGx0OqUI8TddrCeTb8yRW3i7NWbJI92CNQKqsMO1QL2zsM+6+FXF+2B+s92vMXYYOH3SA7RZIADXO9KOm4ZYsfGOUyetDq0R+h8kadI30+qUmSQizKgY1JBg\\\\\\\/PIMm5ckHkWpptM5aT00FOJYEpSjqtJKikXn7foXNxmg6CDepHY9N6QyhkH0zI6HzFA4WwSwqBf5rtDVpq6xsPm0jVPpfILwsFQZfbnmy8urO4YlxqedSAtHy7HooVfDorLvVtdUD3gsydlNkcz8qXkR8HcTbN9vORGtH3nU\\\\\\\/wHRHi+31bZQEDw5EAr2GHmQd3TSosLSUkNSTPqsAUYeyH6+P0GL+i2LRlSTxLPnSsLNgs1H+BxMTXWRuIIvIogmjfmw5xAekB1+QotYHcKFYu4NhKSnDJWiFkD35DTGkcB3Nw3hSpqvsC7RD+5YkulhixMF9gKrMH7j5RHwo9JSA+BGb+jY5nP49tUxQR6CfpPqHTfUIqmCjTta8bgxCId48gqnLefJm9z12Ph7UJdIhM1EciLs+saDYaaal9yMinvItdOdINqEkd2T+dobAMRqkZmPkkrJsJzIkFm5m98mC3nCD+LmL4HTaVmNZXhf1qJI883zmGLYV8aIUqPdo1S1zp0iHRSuT8Lt2\\\\\\\/5AQsZdrCcf0d6wIOjT2dhHrUftCmNzWX0bKJ0ik1smtTDvH8qE\\\\\\\/12qMICEqP1rEpJQBlm8xbccGu8wNsF7MO1K4YeUdtd0DfFwA8sVeo4NoRwyHrSAEjZjdrO6wz2zKs\\\\\\\/uCo5ai5Z56dYc3AT4HvfDRv9lkyP1lpNEyGtB5rofCjuS2zdNJFWz54lp0OyxzYrUKEBOywT0FzTFaWUsDY5ifAwrcYFt+Aqom0l5O3gzKSGMo\\\\\\\/096TJvfP3ZEoz+JbdVDZkKAmPmvfUeUfdf6k4B9WMmCXbYPfZcEhimtD7LScpp0l9XA0k2tyKpN9WJkizLiyu9x0KjIkhrb7sgpj9pEiBjM+tdOqj7PnkmCsT5zoX8RwgrRcPGjKBMpcOklq3aQrjSrfhk0KgL2d5+vOwW0ScMexhT5+IH5besfqokGc42Rx95wNw3unIYy9LJgkqqajZhtJyIouNf6beKvM2f7ZhsqIjXgCt09gWoytYejKSZNys5EfUvE7qQQJX7KRCVmSgrB8lTIiR1mnuTXCtRDKyV5Zeviya4VD67QaawrglU+4VcJDHm3ZvrodWOOMsMofkjQrBSX1\\\\\\\/AlKhzY3SlCJeZBIel03COulPbUyXibYqIolrMQcFaB8uLlMF+krw0wvs0V2wNt4JnMWOmdPWjs9EcxIJDsyhPLBnu0wuOhOs+1TRr05VcqXh488GAirGx+5SJJMIeqGwOhVKELtjpnDPw98KP7f0mojXV2KzXqDGQExyT9YpAP2ZVEEhs1Znr9heRhTzfFR8bE4Yr21MI0zWDPuG1h+\\\\\\\/Q2GJPluegE1lRsdUhEWs4Sf0Evo8pl2Ced9G3gR+drOu0qr8EIZFMrO2dMgEf15USaArA7omJdpE53a7QQLXkKONDy\\\\\\\/9LpnjYVumiHZ8MwPyC7CBploJ19LeCNRPShPrSG21g4HzZoH4lg53XkA16E52gBo\\\\\\\/KN8JsKIZQ+mOT5lD6pyzM4w0itD260iVqQVWfZ6rE3amAT+sjyAjHHcjvsub+75CdpKx62006\\\\\\\/ZhmGrdP9bah30DMF6FLvztoMW0YH5oYbdZysE+Sy8FufM1ekQ6DTYQJLzDqXGCffAiRbRRe5WTZfK7O2+v0w1SZ8WbpblI+0X88gS5X5aWR26iCeYH7ri2VR6qxVvX9yJ\\\\\\\/5EZpy6+jfn4j39ILqvxP9QHTaElIUOIhS\\\\\\\/Ei2hUaSMyruXOuB2pb3m1v1LFHOrt6ssfil6SDJqLEsiDlO+Eju1U8iI1ZZjfeTqxewsT4OUjow+Igfs\\\\\\\/DytZzJV9QLEBGEu4SSoFzqzJAadugEcJo1xFrFkWV2aFfjladyza5\\\\\\\/ah\\\\\\\/yhvJnNA\\\\\\\/h3\\\\\\\/G3w2lwrjmtwwGvOuHGsxaYbcpOYVDw0fVLJsCsbSAiEz3HFMPhohL\\\\\\\/dFLFszD50LBR7On5nTT\\\\\\\/LI0Txib4G9e1ozF9OfwMyD2LgfQHAwzSJEGMQKxfiTgh9qr8+b4U0aN\\\\\\\/MV9PyiT8sYVlT50dZjpbiLvVVhA8HlrWnNxToUrNHEX547SMZgrostvCf0t6LdsxP7XgccVpTJcz6vW4HruiwU6UtcK0zp7gFeSvNvKgI3NVNuNwJosbTpLxy1bLbhjo93iYXf9tbH+wOfo9AzgFJY2CyCt+t9IC1kdQvpGF1uSOPyYE3cFOI41zil1HgghJMvR7DxvS1GEB5OlNKSZNdu3+1fHUbD3yRUOzfVYtY\\\\\\\/cLWuKDR\\\\\\\/TQlnN4DQCnQXQeTHDx9LcXHzsz1gbZcuXrIyM9L1vu7wOuGFHwucXlEtHmjjIn9Tao\\\\\\\/cEk8oGtL2NwmXlhgyosSL1MqD1\\\\\\\/r1gPJCjAUHuySs3vAKEoBwj\\\\\\\/CstkBu9b1ayeGCxWYujWX7gBti2Ufng\\\\\\\/paqJ05Kbb3R7TmXLDnv9xsh1DcSQv3p+HMGUS4prn94ZQ\\\\\\\/dsQlfmebKhz6U0zt57yMtw4+dhbUA7DuixnGTXUKilBz3wSIkY1iQSWkmaD\\\\\\\/3XGyG1cznrNx9\\\\\\\/9xPhoCvQoGx9upzb+7rvymmjAn5puM25dF275WZddFO+lGvUR6\\\\\\\/VU\\\\\\\/Mqg6ExKFh53TiAXyJEtN0IcCEFVCp3o575x9ZI7UF4Kx8Tfy3Pt4T49nu27\\\\\\\/GyAZawhrE2rnWCOPyCAUhHHIg+brkq+ZlRgzge3PcIs39avGnZzO4W0wUdtNVChCmO6x0bH6OYNzwBClCAPNUx0mDbuoMz0FwOUqEnR7PAgh3e4MCo9CMl5eJH8cpIMN3RTzkFSeWavkt0COqe54Kx\\\\\\\/YzOUa6Q+hMRjf2AJroZVk67JuCkMs0J37qoSGJr1qTu1o3PiHaQVWuYmzvQK4UmMuxL43BlPTOhHspIB\\\\\\\/+9edP9qOqjPa8lfGmLX\\\\\\\/WdWvL+YLwTCqYhlyR+ulb9T2IwEIeoZxOIbLw\\\\\\\/7HtBzejCDHATzLcafM2QaOJRh7Y5Lri80rTzeq7MKGsyDhFP901poE9Loh76LmnwI5MxRwkNGji1p\\\\\\\/LrhNkutnRR1GGMsx\\\\\\\/hDYhh6c29HlBUGHhFr5BlxELDMPZNj67pxqnGBIEwph4CYU3QY3Og==\\\u0022,\\\u0022iv\\\u0022:\\\u0022manYi\\\\\\\/34Sx7gBxOMizB\\\\\\\/qg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228af6a879e7de01a029915526d7ecdbb4258b39cb054fe9a00a0c9e87e4a0270b\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022XTm+oV7lfTT7C6X6xuYhWvsXrpgKrDXtJDhQl8FPP5owmZ9sqw3FjPVXF0srLsX\\\\\\\/uHFiroeMTkbw93N+jliz7u1VHF\\\\\\\/RlWyK8dv31lfdyyqRP1lpe4m+zO4OuQhJm8eqObzzISh3FjUE97bd2f2piIZgt7G5ZJaHxxb5nJHdxmfjZs1haT8y1hzy6ygr6EGWjDiYc7HjJynl50KLL15RhkV98eT9pNimYhEZ4CATiTAmUEfG2NeC2oVW1altL3uwuoNbZgtQiIoYGYB++NF2fzomKdtEXRodSpu3QSNblUYbHfoHAt1gv07WGmh+7EOacMs0QHBN9fLIxPENI7rl70r1lTR7vnsF2HHZGnjBTUvAy26aVammmFRyo0H16tWZGr7prsBpThCJD\\\\\\\/5urmiIeecbfubaEB0p3hQm44Zy9jX96zrSgsm5x+jcFMifQD2laHsRr8dhaOAoWhqGklF4JnFehQYg41aIwQhndjcZ3LbKIv4vZq74T0bIt5YCMywAWIuR2\\\\\\\/7OtPZ\\\\\\\/tukrQH\\\\\\\/RYPpGxughs1NBHHXIDOf2ub1Qpm32ZG1+R9SvaFnW20o4SSnBQUNJbxyVZnGhy1+w7RIabsYiAeiizHwZ5NBEcW5v68PRJBs6fJgP3eV88R9RcqSSbypZQoZ8KgxKCNySOrp4dGL2KHq0PZdrLLB60GQJej+Kzd5dPlroinyAInvaMa9bzwDMtvxBUajSXnlvwxixh4I6jNkyoH0xPys2WN7R0l0CYYeUEuLgj+lQrHVyTe0uJpKDaAehT5cDUKS01XLYsjWDnOSILl51rHzt7MzZvxT6HgVZfUm3PIO6gvmd0RvYXMoCojm0KD3\\\\\\\/Bcf45k\\\\\\\/AygGN9D2rafnNYHwtaVxE6yE2FrfeimLQymDB+ErKiZsKTFQLd2hhk97JhJI2WSuKGxgSfqp+pJq8Vhm7dPss5ao0vz+\\\\\\\/NEp\\\\\\\/uL+TpKlVPhvJsi7YyCunvvU1thhxJqKxV5tT0jYM7HnpxgE6Bq5flC9kOWWVFgqV88kPVZSxg9+0wK4hLNSDDcwRq+byEB1FTZi0oKGXl+\\\\\\\/O8Eo4mpLgPVz+O\\\\\\\/+gS17dY\\\\\\\/q4JC9jDXlbAMkSnQMrS3ZiHiF1BxF6PvejMowKlHANgJw8hXqk8E83rQ4uRqXTYJYt3qUahEzjAJE6X\\\\\\\/4B40LKrrB2+wGYYWSp2IJri9\\\\\\\/Dz95HODiXEHnfBfsm5hPOoI4dO3LkLLQvwvID0MyBXfz7oY6cSy\\\\\\\/0sciiEmGNFIYihmEcRx7RHIzxPrgWwgMsjVEuxkmTWNH3UcSTvSPu5gcWlsFqlI\\\\\\\/QA3predxrxGML7snJyTMowHct5ReEn1cXR1vw78txeBR3gOi2fU42qan6V8rym9CRtegmRog4SbmwZ9P7wkio8Vi9R4gHR12wFf7vGatEtnylj7nPfLCgzvT7oIl+jZ7zx6C9P76enUhSwC2TPdwUsd7dr8KEqOuqQA2x\\\\\\\/O59kozWYxyU\\\\\\\/63vv\\\\\\\/dUe0o2QA2RLoDFsHvuC\\\\\\\/odEdDVCD4Tk27Etf2+zUqS97ucUdfSp7HdYY2N2iZH5Q+KW+T0zJj9N9pxFqPm64zxPHR994m7eB7Kt+EDdJ0\\\\\\\/q7uI93qut7qDJYglK969zOGk6ywMWIEuY3w069n9I0tzrvOFilKLarhYMNwwbOzvgEDtJ9sdIzFYqLdXLXA54iCW8D8t7EeGOeg4LIC7fteoEQThEl1Z0Npe3dseRjEw5fSs4500Bb5O3Rn3gh9sWM+zovGy4rO4Mw+8CyQi1TThsCqeYPbRUlOyTzeCadWnqpQO+yy47SZav3kRLJ1znPhI7U92VNh9oF4LMepx+ybMzdwddu3MWaSpw6g8TAB4TOkPNhXB7yjwQV3yhJkCkZw0AgqAZxlM1s0PBgNie9Qyz6FokQXA\\\\\\\/8q3FY3NJfseqbLGBpOT9T30pxjrjZ2OWbMzMi3h7IkktN2HXm1l7FxI3LB0b3XdHEY2TRMG258bsGs6+wYaLJEsHUesGAnHoJSERC+Hdg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022jOd1fqyijEmjibZTBQafdA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224d167650b7dad229e9e72006059069810ab1e50081ef056924874b412ec4e40f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460131 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/939" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-RTTT-FXVPqZCDSnrjwqMh3og17r2Qvo1ubPP8IFh560" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/939" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-LHbQHDNdfySxXbzsbkvSAhNLn4jR0YpTsmy-V-PkzzI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/4/925" title="Sexual Polyploidization in Medicago sativa L.: Impact on the Phenotype, Gene Transcription, and Genome Methylation" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/4/957" title="Whole Genome Comparison Reveals High Levels of Inbreeding and Strain Redundancy Across the Spectrum of Commercial Wine Strains of Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460131" id="node460131" data-pisa="ggg;6/4/939" data-pisa-master="ggg;g3.115.024075" data-apath="/ggg/6/4/939.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetically Determined Variation in Lysis Time Variance in the Bacteriophage φX174</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0819-882X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christopher W. Baker</span>, <span class="highwire-citation-author" data-delta="1">Craig R. Miller</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-0613-4176" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Tanayott Thaweethai</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-3855-1994" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jeffrey Yuan</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-3226-0811" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Meghan Hollibaugh Baker</span>, <span class="highwire-citation-author" data-delta="5">Paul Joyce</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-1424-7541" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel M. Weinreich</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">939-955;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024075 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460131" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Christopher W. Baker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, Brown University, Providence, Rhode Island 02912</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BW.%2BBaker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baker%20CW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristopher%2BW.%2BBaker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0819-882X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christopher W. Baker</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:baker03@g.harvard.edu" class="" data-icon-position="" data-hide-link-title="0">baker03@g.harvard.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Craig R. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematics, University of Idaho, Moscow, Idaho 83844</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biological Sciences, University of Idaho, Moscow, Idaho 83844</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Center for Modeling Complex Interactions, University of Idaho, Moscow, Idaho 83844</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Craig%2BR.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACraig%2BR.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Tanayott Thaweethai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, Brown University, Providence, Rhode Island 02912</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tanayott%2BThaweethai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thaweethai%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATanayott%2BThaweethai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0613-4176" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Tanayott Thaweethai</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jeffrey Yuan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, Brown University, Providence, Rhode Island 02912</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BYuan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yuan%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeffrey%2BYuan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3855-1994" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jeffrey Yuan</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Meghan Hollibaugh Baker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, Brown University, Providence, Rhode Island 02912</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Meghan%2BHollibaugh%2BBaker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baker%20MH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMeghan%2BHollibaugh%2BBaker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3226-0811" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Meghan Hollibaugh Baker</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Paul Joyce </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematics, University of Idaho, Moscow, Idaho 83844</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BJoyce%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Joyce%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaul%2BJoyce%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Daniel M. Weinreich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, Brown University, Providence, Rhode Island 02912</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Center for Computational Molecular Biology, Brown University, Providence, Rhode Island 02912</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BM.%2BWeinreich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weinreich%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BM.%2BWeinreich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1424-7541" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel M. Weinreich</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/4/939" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460131" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460131/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/939.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460131" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460131/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/939.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460131" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460131/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/939.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460131" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460131/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/4/939.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460131" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460131/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-7">Researchers in evolutionary genetics recently have recognized an exciting opportunity in decomposing beneficial mutations into their proximal, mechanistic determinants. The application of methods and concepts from molecular biology and life history theory to studies of lytic bacteriophages (phages) has allowed them to understand how natural selection sees mutations influencing life history. This work motivated the research presented here, in which we explored whether, under consistent experimental conditions, small differences in the genome of bacteriophage φX174 could lead to altered life history phenotypes among a panel of eight genetically distinct clones. We assessed the clones’ phenotypes by applying a novel statistical framework to the results of a serially sampled parallel infection assay, in which we simultaneously inoculated each of a large number of replicate host volumes with ∼1 phage particle. We sequentially plated the volumes over the course of infection and counted the plaques that formed after incubation. These counts served as a proxy for the number of phage particles in a single volume as a function of time. From repeated assays, we inferred significant, genetically determined heterogeneity in lysis time and burst size, including lysis time variance. These findings are interesting in light of the genetic and phenotypic constraints on the single-protein lysis mechanism of φX174. We speculate briefly on the mechanisms underlying our results, and we discuss the potential importance of lysis time variance in viral evolution.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/lysis-time" class="hw-term hw-article-keyword hw-article-keyword-lysis-time" rel="nofollow">lysis time</a></li><li class="kwd"><a href="/keyword/variance" class="hw-term hw-article-keyword hw-article-keyword-variance" rel="nofollow">variance</a></li><li class="kwd"><a href="/keyword/genetics-adaptation" class="hw-term hw-article-keyword hw-article-keyword-genetics-of-adaptation" rel="nofollow">genetics of adaptation</a></li><li class="kwd"><a href="/keyword/evolutionary-theory" class="hw-term hw-article-keyword hw-article-keyword-evolutionary-theory" rel="nofollow">evolutionary theory</a></li><li class="kwd"><a href="/keyword/life-history" class="hw-term hw-article-keyword hw-article-keyword-life-history" rel="nofollow">life history</a></li></ul><p id="p-8">A key opportunity in evolutionary genetics is the decomposition of beneficial mutations into their proximal, mechanistic determinants. To date, much of this work has applied methods from molecular biology to dissect the basis of protein evolution (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Harms and Thornton 2013</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Dean and Thornton 2007</a>), but life history theory (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Stearns 1992</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Roff 2002</a>) similarly decomposes population growth rate into constituent life history traits. It can thus predict which mutations influencing life history will be favored by natural selection (<em>i.e.</em>, increase population growth rate) (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Stearns <em>et al.</em> 2000</a>). The simple life history of the lytic bacteriophage (phage) makes it particularly well suited to this sort of work. Each generation begins with a free phage adsorbing to a host cell and injecting its genome. Transcription and translation facilitate the assembly of progeny phages within the host, and, finally, host cell lysis releases the next generation of free phages. Assuming the density of host cells is not limiting, <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wang <em>et al.</em> 1996</a> provided an expression for phage growth rate in terms of mean burst size and mean time to lysis. Later, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Bull <em>et al.</em> 2006</a> extended this treatment by incorporating adsorption and phage death rates. Other studies have further developed this formalism (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Rabinovitch <em>et al.</em> 1999</a>, <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">2002</a>), with many applying it to explore the mechanistic basis of beneficial mutations in lytic phages (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Pepin <em>et al.</em> 2006</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Shao and Wang 2008</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Bull <em>et al.</em> 2011</a>). Several have demonstrated that a mutation’s fixation probability depends critically on which component of life history it affects (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Wahl and DeHaan 2004</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hubbarde <em>et al.</em> 2007</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Patwa and Wahl 2008</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">2009</a>).</p><p id="p-9">At steady state (<em>i.e.</em>, when host cell infections are at their stable age-of-infection distribution), population growth rate is entirely determined by mean life history traits (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Bull <em>et al.</em> 2011</a>). But the moment a novel mutation appears, it establishes a lineage that, by definition, is far from this equilibrium. In populations deviating from a stable age distribution, variance in both lysis time and adsorption rate may play a significant role in determining growth rate (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Storms and Sauvageau 2014</a>), since it may affect how long the deviation from stable age distribution persists (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Bull <em>et al.</em> 2011</a>).</p><p id="p-10">These findings motivate interest in the possibility of genetic control of higher moments in phage life history traits. Previously, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Dennehy and Wang 2011</a> used a microscopy-based approach to demonstrate heritable variation in lysis time mean and variance among a panel of phage λ holin protein mutants.</p><p id="p-11">To explore heritable variation in phage life history traits, including lysis time variance, we studied a panel of eight genetically distinct clones of the lytic bacteriophage φX174. φX174 is an ubiquitous (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Afshinnekoo <em>et al.</em> 2015</a>) and industrially important phage (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Labrie <em>et al.</em> 2014</a>). It was chosen for this research in large part due to its small, well-characterized genome (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hayashi <em>et al.</em> 1988</a>), the first of its kind ever sequenced (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Sanger <em>et al.</em> 1977</a>). In addition, it infects a well-studied host (<em>Escherichia coli</em> C), has a short generation time, and is robust to a wide range of temperatures, making it amenable to evolution experiments and other lab manipulations (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Wichman and Brown 2010</a>).</p><p id="p-12">The gold standard for measuring phage lysis time and burst size is the one-step phage growth assay (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hyman and Abedon 2009</a>). However, this method affords only limited access to information about lysis time variance and the relationship between burst size and time for a given phage. Therefore, the panel of eight clones was assessed using an updated version of a venerable experimental method first proposed by <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Burnet 1929</a>, and later extended by <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Delbrück 1945</a>. We simultaneously inoculated each of a large number of replicate host cell volumes with, on average, less than one phage particle of identical haplotype. These volumes were then plated at 15-sec intervals on lawns of host cells, and the resulting plaques were counted after incubation.</p><p id="p-13">From this assay, we obtained data that could help determine mean and variance in both lysis time and burst size (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). However, the information we wanted was obscured by a suite of hidden variables. In wells showing lysis events, we did not know when lysis <em>actually</em> occurred. We also did not know the number of phage particles originally in these wells and, if that number was greater than one, how many had lysed their hosts. To infer lysis time and burst size from the two observed metrics (time of sampling and plaque count), we somehow had to circumvent these hidden variables. We overcame this challenge by developing a novel inferential apparatus that complemented the experimental framework with a maximum likelihood-based approach.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/939/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Typical serially sampled parallel infection assay data (pos5,  = 0.71). Here, sampling was performed at 15-sec intervals between 5&#xA0;min and 28.75&#xA0;min. At each time point, three outcomes were possible: zero plaques, a &#x201C;few&#x201D; plaques (here, fewer than eight; see section Statistical methods), and more than a few. Assuming 100% plating efficiency, these outcomes implied respectively that no phage particles were added to the corresponding host cell volume; that the visible number of phage particles was added, but none lysed its host by the time of sampling; and that at least one infected cell had been lysed by the time of sampling. In these data, the first sample with more than a few plaques occurred at 15.5&#xA0;min, giving an upper bound for the time to lysis (). Inset illustrates plaque counts prior to the first observed lysis event. The absence of corresponding stratification among plaque counts after 15.5&#xA0;min implied high variance in burst size. Similarly, the two plaques observed at 23.5&#xA0;min (arrow) corresponded to two phage particles that had not yet lysed their hosts by that time, implying high variance in lysis time. Thus, these data contain information about higher moments in both burst size and lysis time." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1291633927" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Typical serially sampled parallel infection assay data (&lt;em&gt;pos5&lt;/em&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-1&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; = 0.71). Here, sampling was performed at 15-sec intervals between 5&#xA0;min and 28.75&#xA0;min. At each time point, three outcomes were possible: zero plaques, a &#x201C;few&#x201D; plaques (here, fewer than eight; see section &lt;em&gt;Statistical methods&lt;/em&gt;), and more than a few. Assuming 100% plating efficiency, these outcomes implied respectively that no phage particles were added to the corresponding host cell volume; that the visible number of phage particles was added, but none lysed its host by the time of sampling; and that at least one infected cell had been lysed by the time of sampling. In these data, the first sample with more than a few plaques occurred at 15.5&#xA0;min, giving an upper bound for the time to lysis (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-2&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). Inset illustrates plaque counts prior to the first observed lysis event. The absence of corresponding stratification among plaque counts after 15.5&#xA0;min implied high variance in burst size. Similarly, the two plaques observed at 23.5&#xA0;min (arrow) corresponded to two phage particles that had not yet lysed their hosts by that time, implying high variance in lysis time. Thus, these data contain information about higher moments in both burst size and lysis time.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/939/F1.medium.gif" width="440" height="205"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/4/939/F1.medium.gif" width="440" height="205"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/939/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/939/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467787" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-14" class="first-child">Typical serially sampled parallel infection assay data (<em>pos5</em>, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/F1/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/F1/embed/mml-math-1.gif"/></noscript></span></span> = 0.71). Here, sampling was performed at 15-sec intervals between 5 min and 28.75 min. At each time point, three outcomes were possible: zero plaques, a “few” plaques (here, fewer than eight; see section <em>Statistical methods</em>), and more than a few. Assuming 100% plating efficiency, these outcomes implied respectively that no phage particles were added to the corresponding host cell volume; that the visible number of phage particles was added, but none lysed its host by the time of sampling; and that at least one infected cell had been lysed by the time of sampling. In these data, the first sample with more than a few plaques occurred at 15.5 min, giving an upper bound for the time to lysis (<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/F1/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/F1/embed/mml-math-2.gif"/></noscript></span></span>). Inset illustrates plaque counts prior to the first observed lysis event. The absence of corresponding stratification among plaque counts after 15.5 min implied high variance in burst size. Similarly, the two plaques observed at 23.5 min (arrow) corresponded to two phage particles that had not yet lysed their hosts by that time, implying high variance in lysis time. Thus, these data contain information about higher moments in both burst size and lysis time.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">In this article, we describe our finding that, under the tested conditions, genetic heterogeneity could give rise to significant heterogeneity in φX174 lysis time and burst size, including lysis time variance. These findings are interesting in light of the genetic and phenotypic constraints on the single-protein lysis mechanism of φX174, and they point to promising avenues for future theoretical and experimental work.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Media, host cells, and bacteriophages</h3><p id="p-16">Difco LB Lenox Broth (Detroit, MI), pH 7.50, and supplemented with 2 mM CaCl<sub>2</sub> was used throughout. We studied the performance of φX174 on <em>E. coli</em> C [DSMZ 13127, kindly provided by Olivier Tenaillion, Institut National de la Santé et de la Recherche Médicale (INSERM), University Denis Diderot Paris 7], the phage’s natural host. A stock of <em>E. coli</em> C was stored in 40% glycerol at –80°. We used this primary stock to seed bacterial colonies on agar plates stored at 4°. Colony plates were created anew every 2–4 wk. To generate experimental bacterial cultures, 10 mL LB was inoculated with a single colony and shaken in a 50-mL flask at 200 rpm and 37° for about 16 hr. The resulting stationary-phase cells were then diluted somewhere between 33- and 80-fold into 40 mL LB in a 250-mL flask. The diluted culture was shaken at 37° for about 3 hr until it reached an optical density <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-3.gif"/></noscript></span></span> of 0.6. Exponential-phase cells prepared in this way were used for each assay.</p><p id="p-17">To synchronize phage infections, we treated cells using a starvation protocol adapted from <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Denhardt and Sinsheimer 1965</a> and kindly provided by Darin Rokyta, Florida State University. The treatment starved cells so that they “shut down,” allowing for phage adsorption but not entry. (It is likely that the treatment also had the fortunate side effect of reducing variance in time to cell division compared to an exponential-phase culture, since it was bound to cause a lag in the cell cycle. See <em>Discussion</em> for the cell cycle’s relevance.) First, we transferred 1 mL cells to a 1.5-mL centrifuge tube and pelleted them by spinning in an Eppendorf centrifuge (5417R) for 2 min at 20,800 rcf. After aspirating the supernatant, we resuspended the cells in 1 mL HFB-1 solution [0.06 M NH<sub>4</sub>Cl/0.09 M NaCl/0.1 M KCl/0.1 M Tris-HCl (pH 7.4) /1.0 mM MgSO<sub>4</sub>/1.0 mM CaCl<sub>2</sub>] by vortexing for 8 sec. Then, we centrifuged the solution, removed the supernatant, and resuspended in HFB-1 twice more. After removing the supernatant from the third and final HFB-1 rinse, we resuspended cells in 1 ml HFB-2 solution (HFB-1/10 mM MgCl<sub>2</sub>/5 mM CaCl<sub>2</sub>). This served as our culture of starved bacterial cells, ready for phage adsorption.</p><p id="p-18">We studied a panel of eight genetically distinct clones, chosen based on reports of lysis profiles different from the wild type. These included three <em>Epos</em> mutants carrying point mutations in the <em>E</em> gene, whose product is the chief player in φX174 lysis (<em>pos4b</em>, <em>pos5</em>, and <em>pos6</em>, first described in <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bernhardt <em>et al.</em> 2002</a>; kindly provided by Ry Young and Rohit Kongari, Texas A&amp;M University); four carrying point mutations in the <em>D</em>-promoter region, which exerts regulatory control over transcription of the <em>E</em> gene (<em>mut319</em>, <em>mut321</em>, <em>mut323</em>, and <em>mut324</em>, previously described in <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Brown <em>et al.</em> 2010</a> and <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Brown <em>et al.</em> 2013</a>; kindly provided by Celeste Brown and Amber Stancik, University of Idaho); and a wild type (also provided by Ry Young and Rohit Kongari) (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Clones are available from D.M.W. on request. Presence of the mutations of interest was verified by Sanger sequencing of the 65–730 base region of each clone’s genome (nucleotides numbered as in <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Sanger <em>et al.</em> 1977</a>). The clones’ sequences in this region (GenBank accession <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KU646482&amp;atom=%2Fggg%2F6%2F4%2F939.atom">KU646482</a>–<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KU646588&amp;atom=%2Fggg%2F6%2F4%2F939.atom">KU646588</a>)  were identical to the wild-type sequence (GenBank accession <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=J02482.1&amp;atom=%2Fggg%2F6%2F4%2F939.atom">J02482.1</a>) apart from the noted mutations. The wild type employed was the ancestor of the <em>Epos</em> mutants, which differed from the wild-type ancestor of the <em>D</em>-promoter mutants (GenBank accession <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=AF176034&amp;atom=%2Fggg%2F6%2F4%2F939.atom">AF176034</a>) at five sites: G833A (nonsynonymous), G1650A (nonsynonymous), T2811C (synonymous), A4518G (nonsynonymous), and T4784C (synonymous) (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Pearson <em>et al.</em> 1997</a>). The G833A difference entailed an amino acid change in the <em>E</em> (but not the <em>D</em>) gene, though it fell well outside the region coding for the E protein’s essential transmembrane region (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>) (Tanaka and Clemons Jr 2012).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467974/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467974/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Characteristics of <strong>φ</strong>X174 <em>D</em>-promoter and <em>E<span class="sc">pos</span></em> mutants</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/939/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Positions of mutations in &#x3C6;X174 genome. The figure shows a 832-bp region of interest, including genes C, D, E, and J. Gene C is involved in DNA replication, gene D is an external scaffolding protein required for procapsid morphogenesis, gene E is responsible for lysis of the host, and gene J is required for DNA packaging (Fane et al. 2006). Gene E is shown in three sections, corresponding to three domains of the protein. The transmembrane domain binds the E protein&#x2019;s substrate, MraY (Mendel et al. 2006; Zheng et al. 2009). Mutations are indicated with pins at the position, and the site and nucleotide change are indicated to the right of the pin head. The clone(s) carrying each mutation is indicated in parentheses. Note that one of the five background nucleotide differences between the Epos and D-promoter mutants (G833A) is included in this figure." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1291633927" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Positions of mutations in &#x3C6;X174 genome. The figure shows a 832-bp region of interest, including genes &lt;em&gt;C&lt;/em&gt;, &lt;em&gt;D&lt;/em&gt;, &lt;em&gt;E&lt;/em&gt;, and &lt;em&gt;J&lt;/em&gt;. Gene &lt;em&gt;C&lt;/em&gt; is involved in DNA replication, gene &lt;em&gt;D&lt;/em&gt; is an external scaffolding protein required for procapsid morphogenesis, gene &lt;em&gt;E&lt;/em&gt; is responsible for lysis of the host, and gene &lt;em&gt;J&lt;/em&gt; is required for DNA packaging (Fane &lt;em&gt;et al.&lt;/em&gt; 2006). Gene &lt;em&gt;E&lt;/em&gt; is shown in three sections, corresponding to three domains of the protein. The transmembrane domain binds the E protein&#x2019;s substrate, MraY (Mendel &lt;em&gt;et al.&lt;/em&gt; 2006; Zheng &lt;em&gt;et al.&lt;/em&gt; 2009). Mutations are indicated with pins at the position, and the site and nucleotide change are indicated to the right of the pin head. The clone(s) carrying each mutation is indicated in parentheses. Note that one of the five background nucleotide differences between the &lt;em&gt;Epos&lt;/em&gt; and &lt;em&gt;D&lt;/em&gt;-promoter mutants (G833A) is included in this figure.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/939/F2.medium.gif" width="440" height="145"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/4/939/F2.medium.gif" width="440" height="145"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/939/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/939/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467984" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-21" class="first-child">Positions of mutations in φX174 genome. The figure shows a 832-bp region of interest, including genes <em>C</em>, <em>D</em>, <em>E</em>, and <em>J</em>. Gene <em>C</em> is involved in DNA replication, gene <em>D</em> is an external scaffolding protein required for procapsid morphogenesis, gene <em>E</em> is responsible for lysis of the host, and gene <em>J</em> is required for DNA packaging (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Fane <em>et al.</em> 2006</a>). Gene <em>E</em> is shown in three sections, corresponding to three domains of the protein. The transmembrane domain binds the E protein’s substrate, MraY (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Mendel <em>et al.</em> 2006</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Zheng <em>et al.</em> 2009</a>). Mutations are indicated with pins at the position, and the site and nucleotide change are indicated to the right of the pin head. The clone(s) carrying each mutation is indicated in parentheses. Note that one of the five background nucleotide differences between the <em>Epos</em> and <em>D</em>-promoter mutants (G833A) is included in this figure.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">Phage titers were determined by combining a known volume of phage stock with 200 μL exponentially growing host cells (described above), and plating in top agar held at 42°, consisting of 7.0 g/L Difco BactoAgar (Detroit, MI) added to LB medium (as described above). After several minutes on the bench at room temperature, we incubated plates agar-side-up at 37°. Preliminary results (not shown) demonstrated that accurate plaque counts required at least 4 hr of incubation. Occasionally, for logistical reasons, we interrupted incubation with overnight refrigeration at 4°.</p></div><div id="sec-3" class="subsection"><h3>Serially sampled parallel infection assay</h3><p id="p-23">We began by adding 90 μL starved cells to each well of a TempPlate nonskirted 96-well PCR plate (USA Scientific item no. 1402-9598) at 15°. The PCR plate was mounted on a thermocycler to control its temperature precisely. Using a 12-channel pipettor, we then added 10 μl 30-PFUs/mL phage dilution to each well in each row of 12 wells, allowing for a 30-sec interval between rows. PCR plates were held at 15° for at least 20 min, during which time phages could adsorb to the starved cells. We estimate that 10–20% of phage particles adsorbed to hosts during this incubation period (Celeste Brown, personal communication; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Newbold and Sinsheimer 1970</a>).</p><p id="p-24">Then, infections were initiated by raising the PCR plate temperature to 37° and adding 100 μL warm LB (incubated at 37° for at least 45 min) to each well in a row with a 12-channel pipettor, again allowing 30-sec intervals between rows. Consequently, the infective process in each row was staggered by half a minute, allowing ample time for the sampling procedure that followed. Note that the temperature increase alone did not trigger infection. The addition of rich medium, such as LB, was required for infection to advance (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Newbold and Sinsheimer 1970</a>). Based on past reports of φX174 attachment kinetics under conditions similar to our assay (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Brown <em>et al.</em> 2013</a>), unattached phage particles likely adsorbed to hosts very quickly at this time, with an average adsorption time of less than 30 sec [assuming adsorption followed an exponential decay process (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Abedon <em>et al.</em> 2001</a>)].</p><p id="p-25">It is important to note that, based on our knowledge of the mutated genes’ functions (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Fane <em>et al.</em> 2006</a>) and past results (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Brown <em>et al.</em> 2013</a>), the studied mutations likely had little or no effect on phage adsorption rate. This includes the nonsynonymous difference (G1650A) between the <em>D</em>-promoter mutants and other clones in coat protein F. While a change in the F protein could theoretically have affected adsorption (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bernal <em>et al.</em> 2004</a>), it is unlikely to have done so here, because the difference was merely between one basic polar residue (His) and another (Arg).</p><p id="p-26">Five minutes after infection was initiated, we began sampling wells at 15-sec intervals. Sampling was performed by transferring the entire 200 μL contents of a well to a sterile 96-well PCR plate, held at 4° to halt the infective process. We assume that this sampling procedure did not artificially induce lysis in sampled infections. We staggered sampling between rows such that, factoring in the 30-sec gaps in infection between each row, we sampled one well every 15 sec between <em>t</em> = 5 min and <em>t</em> = 28.75 min. The endpoint of 28.75 min was chosen on the basis of preliminary experiments that showed most infections ending in lysis by that time. Even though not all phages would lyse their hosts by 28.75 min, we found no evidence that φX174 undergoes T4-like lysis inhibition (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Abedon 1992</a>), so we believed that samples taken between 5 min and 28.75 min would give us a reasonably complete picture of the clones’ lysis phenotypes.</p><p id="p-27">At the conclusion of this process, the contents of each sampled well—now at 4°—were plated and incubated, as described, and then counted. Typical results are shown in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>. Four to six replicate assays were performed for each clone.</p></div><div id="sec-4" class="subsection"><h3>Statistical methods</h3><div id="sec-5" class="subsection"><h4>The model:</h4><p id="p-28">We began with an overview of the model that we assumed generated the data. We then developed methods of estimation from the model and applied them to real experimental data.</p><p id="p-29">Each time an assay was performed, we assumed a small, Poisson-distributed number of phage particles had been added to each well. The mean of the Poisson was allowed to vary between assays, which is to say, we assumed that phage concentrations in the stocks differed by clone and could change over time.</p><p id="p-30">Within the wells, we assumed phage particles adsorbed instantaneously to starved cells upon the initiation of infection. (We discuss the rationale underlying this assumption in the <em>Results</em> section). We assumed that each phage particle infected and then lysed a cell <em>T</em> minutes after the addition of warm media, where <em>T</em> could not be less than a biologically imposed minimum (the latent period), and lysis was equally probable every moment thereafter (though we also considered a model in which lysis probability changed with time, as described below). We assumed that burst size grew linearly with lysis time, a relationship generally assumed and supported by past work (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hutchison and Sinsheimer 1966</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abedon 1989</a>; <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Wang <em>et al.</em> 1996</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Abedon <em>et al.</em> 2001</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Chao <em>et al.</em> 2002</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Bull <em>et al.</em> 2004</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Wang 2006</a>; but see <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hutchison and Sinsheimer 1963</a> and <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Storms <em>et al.</em> 2014</a>). The fate of each infecting phage particle within a well was assumed to be independent of other particles in that well. Wells were sampled and plated at 15-sec intervals from <em>t</em> = 5 min through <em>t</em> = 28.75 min, and we assumed 100% plating efficiency, implying that the plaque count on a plate revealed the true number of phage particles in a well at the time of sampling. A plaque count of zero meant no particles were in the well, a count of one to a few implied that as many particles as visible plaques were added but that no host cells had been lysed yet, and a count of more than a few represented the sum of the burst sizes of lysis events prior to the sample time, plus any infected yet unlysed cells. Note that, in the third case, we did not know how many phage particles were in the well to begin with, we did not know how many lysis events had occurred by the time of sampling, and, of the lysis event(s) that had occurred, we did not know when they occurred, except that they must have occurred prior to the moment of sampling. From a statistical perspective, this suite of hidden variables made the problem both interesting and challenging.</p></div><div id="sec-6" class="subsection"><h4>Notation:</h4><p id="p-31">Developing estimators under this model required substantial notation. <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> summarizes this notation for the reader’s reference.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467833/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467833/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary of notation used in statistical modeling</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h4>Number of phage particles per well:</h4><p id="p-32">Our first task was to estimate <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-37.gif"/></noscript></span></span>, the Poisson parameter that governs the probability of having <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-38.gif"/></noscript></span></span> phage particles in each well for assay <em>a</em>. This was somewhat complicated by the fact that, before the latent period ended and lysis began, we observed the actual Poisson counts, but after this period, we only observed the true Poisson counts for “zero” wells and wells that do not show lysis events. In wells showing lysis events, we knew only that the Poisson count must have been greater than zero. To derive an estimator tailored to the situation, we ignored the time element of the data and defined <em>p</em> as the probability that a randomly selected phage particle from a randomly selected well would not have lysed its host. We let <em>n</em> be the Poisson number of phage particles initially in a well (<em>i.e.</em>, before lysis occurs). Then, we noted that each sampled well fell into one of three categories: samples with zero phage particles, samples with phage particles but no lysis events, and samples showing lysis events. The probability of a well belonging to each category was, respectively, <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-39.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-40.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-41.gif"/></noscript></span></span>.</p><p>We defined wells not showing lysis events as those having plaque counts greater than zero, and less than or equal to seven. When the Poisson parameter <em>β</em> is less than or equal to one, counts greater than seven are highly improbable in wells where lysis has not occurred (<span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-42.gif"/></noscript></span></span>). Of course, it is possible that wells showing fewer than eight plaques could in fact be the results of infections. But, ultimately, we had to set the threshold for distinguishing lysis events from nonlysis events <em>somewhere</em>. This threshold-setting exercise was not straightforward. Complicating factors included the following:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-34">Poisson average numbers of particles per well varied for each assay, so setting the threshold based purely on the Poisson probabilities was not trivial.</p></li><li id="list-item-2"><p id="p-35">There was a nonzero chance of slight inaccuracies in some plaque counts. Rare errors could have occurred in the counting of plaques that would have increased counts on average (<em>e.g.</em>, plaque-like air bubbles). In addition, certain experimental manipulations (<em>e.g.</em>, mechanical stress from pipetting) could feasibly have induced very premature lysis of some cells, resulting in plates with extremely low plaque counts that otherwise would have had just one plaque.</p></li><li id="list-item-3"><p id="p-36">The few plaque counts of six and seven, a couple of which were sampled at unusually early time points, could have exerted disproportionate influence on estimates of clones’ lysis phenotypes if included as lysis events. This is because, as discussed below, an artificially early lysis event would force the value of the <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-43.gif"/></noscript></span></span> parameter to shift to some time before that event.</p></li></ol><p>Therefore, given these intermediate plaque counts’ rarity, their potential origins in experimental error, and their potential to distort our estimates of clones’ lysis phenotypes, we chose to set the threshold for lysis events at greater than or equal to eight. It is worth noting, however, that when our data instead were analyzed using a threshold of greater than or equal to five, our results did not change qualitatively (analysis not shown).</p><p id="p-37">We let the number of wells in assay <em>a</em> be <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-44.gif"/></noscript></span></span> and the number of wells that showed lysis events be <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-45.gif"/></noscript></span></span>. The likelihood of the data, then, was<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-46.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>We then took the log of the likelihood, the partial derivatives with respect to <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-47.gif"/></noscript></span></span> and <em>p</em>, set each equal to zero, and did a considerable amount of algebra (see Appendix 1 for details). This resulted in the following estimator:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-48.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>We validated this estimator by simulating data under <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-49.gif"/></noscript></span></span> and 2, and we compared it to two reasonable alternatives: the “zero-class” estimator [<span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-50.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-51.gif"/></noscript></span></span> is the proportion of sampled wells that had plaque counts of zero], and the “mean-count” estimator (<span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-52.gif"/></noscript></span></span>, where <em>X</em> is the mean among observed plaque counts, applied only to wells sampled before we witnessed any lysis events). The results (which we will not cover in detail) revealed that Equation 2 had a very slightly positive bias (<span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-53.gif"/></noscript></span></span> when <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-54.gif"/></noscript></span></span>, falling to <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-55.gif"/></noscript></span></span> at <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-56.gif"/></noscript></span></span>) but a lower mean squared error than the other estimators across conditions. Valuing the nearness of the estimate to the truth above all else, we adopted the new estimator of <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-57.gif"/></noscript></span></span> throughout.</p></div><div id="sec-8" class="subsection"><h4>Time to lysis:</h4><p id="p-38">We considered two nested models describing time to lysis. Both assumed that phages do not lyse cells until the latent period of <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-58.gif"/></noscript></span></span> ends. The exponential model assumed that lysis occurred with equal probability each moment after <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-59.gif"/></noscript></span></span>, while the Weibull model allowed the probability of lysis to change with time. Under the Weibull, the time to lysis, <em>T</em>, had the following distribution:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-60.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where α was a hazard parameter that dictated how lysis probability changed with time. When <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-61.gif"/></noscript></span></span>, the Weibull model simplified to the exponential model. Because they are nested, we employed a likelihood ratio test to determine if the improved fit under the Weibull justified the additional parameter. To do so, we analyzed each genetically distinct clone separately. We fitted the data to each model using maximum likelihood (as described below), estimated parameters, obtained the log-likelihood, and took the difference: <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-62.gif"/></noscript></span></span>. We then simulated 100 datasets under the null (exponential) parameter estimates. For each simulated dataset, we fitted the data to both models and calculated the difference in their log-likelihoods: <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-63.gif"/></noscript></span></span>. We then approximated the <em>P</em>-value by the proportion of <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-64.gif"/></noscript></span></span> greater than or equal to <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-65.gif"/></noscript></span></span>. Large <em>P</em>-values (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-66.gif"/></noscript></span></span>) would indicate the simpler exponential model described the data sufficiently well. This test returned <em>P</em>-values greater than 0.3 for all clones, and we therefore assumed the exponential model for the duration of the study. One important implication of this assumption was that, under the exponential model, the lysis time variance was equivalent to <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-67.gif"/></noscript></span></span>.</p><p id="p-39">Defining <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-68.gif"/></noscript></span></span> as the number of phages in well <em>r</em>, assay <em>a</em> that had lysed their hosts by time <em>t</em>, <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-69.gif"/></noscript></span></span> followed the Poisson distribution with rate <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-70.gif"/></noscript></span></span>. If we let <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-71.gif"/></noscript></span></span> be an indicator function that is one when <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-72.gif"/></noscript></span></span> is greater than or equal to one, and zero when <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-73.gif"/></noscript></span></span> is zero, and we let <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-74.gif"/></noscript></span></span> be the number of phages that had not lysed their hosts in well <em>r</em>, assay <em>a</em> at time <em>t</em>, we show in Appendix 2 that the log-likelihood of the data were proportional to<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-75.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>Note that <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-76.gif"/></noscript></span></span> directed us to sum over all assays of a particular clone. We analyzed each clone separately. We have omitted a clone subscript throughout to avoid further complicating notation. In practice, we obtained joint maximum likelihood estimates of λ and <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-77.gif"/></noscript></span></span> by establishing a grid of joint values at narrow intervals, calculating the log-likelihood of each using Equation 3, and selecting those values that together give the best likelihood. Confidence intervals on λ and <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-78.gif"/></noscript></span></span> were obtained by parametric bootstrap. To do this, we used <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-79.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-80.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-81.gif"/></noscript></span></span> to simulate one suite of datasets to match (in number and well count) the real datasets for the clone under analysis. We analyzed the simulated suite of datasets using Equation 4 and a grid of potential values to obtain <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-82.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-83.gif"/></noscript></span></span>. The one condition we imposed in this estimation step was that <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-84.gif"/></noscript></span></span> must be less than or equal to the minimum observed lysis time from the real data. If we did not do this, the bootstrap would have returned values and hence confidence bounds for <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-85.gif"/></noscript></span></span> greater than this minimum lysis time, and we knew such values would not be valid confidence bounds when the probability of the real data under them is zero. We repeated 500 bootstrap replicates. We then sorted the <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-86.gif"/></noscript></span></span> values, and defined the 95% confidence interval as the central 95% of the values. We independently sorted the <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-87.gif"/></noscript></span></span> values and used the central 95% of these. Thus, the confidence intervals on <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-88.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-89.gif"/></noscript></span></span> were not joint. We did not validate that these intervals capture the truth greater than or equal to 95% of the time, but we did confirm that the means of <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-90.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-91.gif"/></noscript></span></span> were close to <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-92.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-93.gif"/></noscript></span></span>, respectively, suggesting that the parametric bootstrap was a valid technique.</p><p id="p-40">We also were interested in inferring mean lysis times from our data. Given the unique properties of exponential distributions, we could derive each clone’s mean lysis time simply by summing its estimated <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-94.gif"/></noscript></span></span> and λ. A confidence interval for this estimate could be obtained by repeating this procedure using the upper and lower bounds on the individual parameter estimates.</p><p id="p-41">We now turn our attention to how to determine whether lysis time parameters λ and <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-95.gif"/></noscript></span></span> are different for different clones. We conducted several formal tests for differences between clones using a likelihood ratio framework. To test whether both λ and <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-96.gif"/></noscript></span></span> clones differ, we note that, under the null hypothesis, a single shifted exponential function would describe the data from both clones being compared; the alternative hypothesis asserted the data for each clone came from a distinct function. We calculated the log-likelihood of the data under the null (<span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-97.gif"/></noscript></span></span>) by pooling both clones’ data, fitting as just described to obtain <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-98.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-99.gif"/></noscript></span></span>, and then using Equation 4. We calculated the log-likelihood under the alternative (<span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-100.gif"/></noscript></span></span>) by doing the same thing, except we fitted each clone’s datasets separately, calculated their log-likelihoods separately, and then summed. We define <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-101.gif"/></noscript></span></span>. To determine whether <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-102.gif"/></noscript></span></span> is significantly large, we needed the distribution of <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-103.gif"/></noscript></span></span> under the null. We obtained this by simulation. We simulated data analogous in size and sample times to the pooled data using <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-104.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-105.gif"/></noscript></span></span>, repeated the fitting process, and calculated log-likelihoods and their difference to yield <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-106.gif"/></noscript></span></span>. We repeated this process 500 times, and took the approximate <em>P</em>-value of <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-107.gif"/></noscript></span></span> as the proportion of bootstrap simulations where <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-108.gif"/></noscript></span></span> was greater than <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-109.gif"/></noscript></span></span>.</p><p id="p-42">To test whether just λ differed between pairs of clones, we used a similar likelihood ratio test. Under the null, both clones shared a common value of λ but could have different values of <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-110.gif"/></noscript></span></span>; under the alternative, both λ and <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-111.gif"/></noscript></span></span> could differ between clones. Note that the null did not fix λ at a single specific value; rather, it merely stipulated that λ, whatever value that may be, be shared by the clones being compared. We fitted the data to the null by finding the maximum likelihood estimates for each clone under the constraint of a shared value of λ and unconstrained values of <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-112.gif"/></noscript></span></span> and then summing their log likelihoods (<span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-113.gif"/></noscript></span></span>). We fitted the alternative by imposing no constraints, and summing the two log-likelihoods (<span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-114.gif"/></noscript></span></span>). We defined <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-115.gif"/></noscript></span></span>. To obtain the distribution of <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-116.gif"/></noscript></span></span> under the null, we simulated 500 datasets using the null parameter estimates, repeated the model-fitting exercise for each, and took the difference in the log-likelihood: <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-117.gif"/></noscript></span></span>. The proportion of the bootstrap replicates where <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-118.gif"/></noscript></span></span> was greater than <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-119.gif"/></noscript></span></span> was the approximate <em>P</em>-value for this comparison. To test for significant differences in <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-120.gif"/></noscript></span></span>, we repeated this procedure, except <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-121.gif"/></noscript></span></span> was constrained and λ was allowed to vary.</p><p id="p-43">To test whether the distribution of <em>P</em>-values calculated for each set of pairwise parameter comparisons diverged from the null expectation (<em>i.e.</em>, a uniform distribution), we binned the <em>P</em>-values into four categories [0.00,0.25), [0.25,0.50), [0.50,0.75), and [0.75,1.00] and conducted an exact multinomial test against expected frequencies in each bin under the null. The null bin frequencies were equal across the four categories (<em>i.e.</em>, 25% in each bin).</p></div><div id="sec-9" class="subsection"><h4>Burst size as a function of time:</h4><p id="p-44">We modeled the potential for burst size to change with time as a linear function, <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-122.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-123.gif"/></noscript></span></span> was the expected burst size when <em>T</em> minutes had elapsed since <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-124.gif"/></noscript></span></span>, μ was the burst size at <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-125.gif"/></noscript></span></span>, α was the rate of increase in burst size (<em>i.e.</em>, the slope parameter), and ε captured the normally distributed variation from expectation (mean 0, variance <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-126.gif"/></noscript></span></span>). Note that none of the terms in this model was observable. If a well sampled at time <em>t</em> showed a lysis event, we knew that lysis occurred before <em>t</em>, but we did not know exactly when. In addition, we did not know how many phage particles initially were in the well. Thus, both lysis time and burst size were hidden variables. In Appendix 3, we show that the observed count (rather than the burst size) could, like burst size, be expressed as a linear function:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-127.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-128.gif"/></noscript></span></span> was the count observed in well <em>r</em> sampled at time <em>t</em> from assay <em>a</em>, <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-129.gif"/></noscript></span></span> was the number of phage particles in well <em>r</em> (assay <em>a</em>) that had lysed their hosts by time <em>t</em>, <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-130.gif"/></noscript></span></span> was the sum of their lysis times, and δ was normally distributed with mean zero and variance <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-131.gif"/></noscript></span></span>. Our goal was to estimate the parameters μ, α, and σ, but because Equation 5 depended on the unobservable <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-132.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-133.gif"/></noscript></span></span>, we could not use standard regression. Instead, we employed an expectation maximization (EM) algorithm, through which the problem was circumvented by replacing <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-134.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-135.gif"/></noscript></span></span> with their expected values before doing regression, and then iterating (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Meng and Rubin 1993</a>). More specifically, our EM algorithm involved the following steps: (1) guess values for <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-136.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-137.gif"/></noscript></span></span>; (2) estimate the parameters μ, α, and σ by least-squares regression (equivalent to maximum likelihood) assuming the guesses of <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-138.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-139.gif"/></noscript></span></span> are correct; (3) assuming the parameter estimates are correct, calculate the expected values of <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-140.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-141.gif"/></noscript></span></span>, and update the guesses to these values; and (4) repeat steps 2 and 3 until convergence occurs. It turns out that both parameter estimation (step 2) and calculating expectations (step 3) involved a fair amount of math; we present this in Appendix 3.</p><p id="p-45">To obtain confidence intervals and test for differences between clones in the regression parameters, we used a parametric bootstrap approach. For each genetically distinct clone, we simulated 1000 datasets using the estimated parameters and, each time, estimated α, μ and σ. We then sorted the bootstrap set of parameter values individually, using the 25th and the 975th values (<em>i.e.</em>, the central 95%) to define the confidence intervals. To keep our analysis simpler, we did not conduct formal likelihood ratio tests for significant differences between clones for these parameters. Instead, we compared confidence intervals and applied the conservative criteria that when confidence intervals were nonoverlapping, the two clones were significantly different.</p><p id="p-46">Data and code required to repeat this analysis can be found in the Supplemental Material <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024075/-/DC1/FileS1.zip">File S1</a>.</p></div></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-47">Strains are available from D.M.W. upon request. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024075/-/DC1/FileS1.zip">File S1</a> contains a description of all supplemental material, including raw experimental data and the R code that was used to analyze them. Sequences of the WT (<em>Epos</em> mutant ancestor) and <em>D</em>-promoter ancestor are given by GenBank accession numbers <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=J02482.1&amp;atom=%2Fggg%2F6%2F4%2F939.atom">J02482.1</a> and AF176034, respectively. Regions of the mutants' genomes were sequenced, and these sequences also were archived in GenBank (accession numbers <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KU646482&amp;atom=%2Fggg%2F6%2F4%2F939.atom">KU646482</a> to <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KU646588&amp;atom=%2Fggg%2F6%2F4%2F939.atom">KU646588</a>).</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><p id="p-48">Thirty-seven assays were conducted for the eight clones (four to six each). (Raw data from the assays are available in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024075/-/DC1/FileS1.zip">File S1</a>.) The estimated Poisson parameter <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-142.gif"/></noscript></span></span> ranged from 0.20 to 0.91, with a mean of 0.46. It follows that approximately 63% of all wells had no phage particles, 29% had one particle, 6.5% had two particles, 1% had three, and 0.1% had four. Observed plaque counts ranged from zero to 1270, with eight being the smallest plaque count classified as representing a lysis event (see section <em>Statistical methods</em>).</p><p id="p-49">Across the clones, the earliest times at which lysis was observed ranged from 11.0 min (<em>pos6</em>) to 14.5 min (<em>mut321</em>, <em>mut323</em>). Estimates of <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-143.gif"/></noscript></span></span> are shown in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. This range of latent periods was consistent with past results, as reported in <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Bernhardt <em>et al.</em> 2002</a> (one-step growth assays in <a id="xref-fig-2-3" class="xref-fig" href="#F2">figure 2A</a> in <a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Bernhardt <em>et al.</em> 2002</a>, about 12 min for wild type, <em>pos6</em>, and <em>pos4B</em> infections, though about 22min for <em>pos5</em>) and <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Brown <em>et al.</em> 2010</a> (about 12–14 min for the <em>D</em>-promoter mutants and wild type).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467911/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467911/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Estimates of λ and <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/T3/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/T3/embed/mml-math-144.gif"/></noscript></span></span> for each clone</span><div class="sb-div caption-clear"></div></div></div><p id="p-51">The smallest observed value of λ among the clones was 4.8 min (<em>mut321</em>), meaning that the cumulative probability of lysis after <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-147.gif"/></noscript></span></span> rose most rapidly for this clone under the experimental conditions. The wild type was found to have the largest λ (11.2 min). Values of λ for other clones, along with confidence intervals, can be found in <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>.</p><p id="p-52"><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> illustrates how the observed increase over time in the proportion of wells showing lysis events was used to estimate the cumulative lysis probability function for each clone. The figure also includes pairwise comparisons of each clone with the wild type, demonstrating clear differences between them. Note that <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> does not visualize the β parameter, which, in addition to the lysis events shown, was important in the estimation of lysis time phenotypes. Taken together, the curves form three rough clusters: (1) <em>pos6</em>, (2) wild type and <em>pos4B</em>, and (3) the remaining clones. This clustering can be observed clearly when the parameters composing those functions, λ and <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-148.gif"/></noscript></span></span>, are plotted against each other (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/939/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimated cumulative lysis probability by time functions, pairwise comparisons of each clone with the wild type. The wild type panel (top left) provides an example of fitting a cumulative lysis probability by time function to wild-type data. The gray line is the best-fit curve. Dotted lines represent 100 of the 500 bootstrap replicates. Each bubble&#x2019;s position shows the proportion of wells sampled at a given time in which we observe a lysis event; the size of each bubble shows the number of samples on which the proportion is based. In all other panels, the wild type is represented with a gray line and gray bubbles." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1291633927" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimated cumulative lysis probability by time functions, pairwise comparisons of each clone with the wild type. The wild type panel (top left) provides an example of fitting a cumulative lysis probability by time function to wild-type data. The gray line is the best-fit curve. Dotted lines represent 100 of the 500 bootstrap replicates. Each bubble&#x2019;s position shows the proportion of wells sampled at a given time in which we observe a lysis event; the size of each bubble shows the number of samples on which the proportion is based. In all other panels, the wild type is represented with a gray line and gray bubbles.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/939/F3.medium.gif" width="331" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/4/939/F3.medium.gif" width="331" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/939/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/939/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467948" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-53" class="first-child">Estimated cumulative lysis probability by time functions, pairwise comparisons of each clone with the wild type. The wild type panel (top left) provides an example of fitting a cumulative lysis probability by time function to wild-type data. The gray line is the best-fit curve. Dotted lines represent 100 of the 500 bootstrap replicates. Each bubble’s position shows the proportion of wells sampled at a given time in which we observe a lysis event; the size of each bubble shows the number of samples on which the proportion is based. In all other panels, the wild type is represented with a gray line and gray bubbles.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/939/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimates of parameters &#x3BB; and  that define the lysis probability function for wildtype and each mutant. Bars represent the approximate 95% confidence intervals for each parameter independent of the other (based on 500 bootstrap replicates)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1291633927" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimates of parameters &#x3BB; and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-149&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; that define the lysis probability function for wildtype and each mutant. Bars represent the approximate 95% confidence intervals for each parameter independent of the other (based on 500 bootstrap replicates).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/939/F4.medium.gif" width="440" height="405"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/4/939/F4.medium.gif" width="440" height="405"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/939/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/939/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468057" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-54" class="first-child">Estimates of parameters λ and <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/F4/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/F4/embed/mml-math-149.gif"/></noscript></span></span> that define the lysis probability function for wildtype and each mutant. Bars represent the approximate 95% confidence intervals for each parameter independent of the other (based on 500 bootstrap replicates).</p><div class="sb-div caption-clear"></div></div></div><p id="p-55">These visual comparisons were formalized into pairwise statistical tests. Along with parameter estimates, <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a> shows the pairwise <em>P</em>-values associated with the null hypothesis that the clones being compared came from the same function (<em>i.e.</em>, that jointly, both λ and <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-150.gif"/></noscript></span></span> are the same for the clones). If either parameter differed, this test would reject the null. By contrast, <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> shows the results of two different sets of tests: the set of pairwise tests of the null in which λ was held to be the same between clones, but <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-151.gif"/></noscript></span></span> was allowed to differ, and a similar test in which <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-152.gif"/></noscript></span></span> was held to be the same, but λ was allowed to differ. In other words, the first of these two sets of tests gauged differences between clones’ values of λ, whereas the second set of tests gauged differences between clones’ values of <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-153.gif"/></noscript></span></span>.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467941/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467941/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Approximate <em>P</em>-values of pairwise comparisons of individual parameters in lysis probability by time functions</span><div class="sb-div caption-clear"></div></div></div><p id="p-57">Notice that we have not corrected for multiple tests. Our <em>P</em>-values are based on 500–1000 bootstrap replicates, whereas the amount of replication needed to achieve significance under a Bonferroni correction would be in the 5000–10,000 range and computationally infeasible. We note, however, that if the joint null hypothesis were universally true (<em>i.e.</em>, all clones followed the same function), we would expect the <em>P</em>-values of the 28 pairwise tests to follow a uniform distribution, and we would expect just one or two false positives (28 <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-157.gif"/></noscript></span></span> 0.05 = 1.4). Indeed, we would be unlikely to have more than three false positives, since the binomial probability of greater than three false rejections is 0.0491. Instead, the distribution of <em>P</em>-values for the joint λ-<span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-158.gif"/></noscript></span></span> comparisons skewed strongly from a uniform distribution [exact multinomial test, <em>P</em> &lt; 0.0001 (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">McDonald 2014</a>)], and we observed 13 pairwise results less than or equal to 0.05 (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). [The binomial probability of 13 or more false rejections is <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-159.gif"/></noscript></span></span>.] We suggest, therefore, that most, though perhaps not all, of the differences that appeared as significant were truly significant. Similarly, the tests for pairwise differences in only λ and only <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-160.gif"/></noscript></span></span> yielded 10 and 13 values less than or equal to 0.05, respectively, which is far more than expected under the null of a uniform distribution (exact multinomial test, <em>P</em> = 0.0068 and <em>P</em> &lt; 0.0001, respectively). (The probability of 10 or more false rejections is <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-161.gif"/></noscript></span></span>, and the probability of 13 or more is <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-162.gif"/></noscript></span></span>; <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>.)</p><p id="p-58">We also note that our omission of adsorption time from our statistical model (see <em>Materials and Methods</em>) had little effect on these findings. By ignoring adsorption time, we treated the time to lysis as a single exponential process, when in reality it involves the sum of two exponential waiting times (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Abedon <em>et al.</em> 2001</a>). However, additional analysis (not shown) demonstrated that, because adsorption happens at least an order of magnitude faster than lysis (<a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Brown <em>et al.</em> 2013</a>), the distribution of lysis time under the one-exponential model is very similar to that generated by a more complex two-exponential model. Furthermore, the variance of our one-exponential model is slightly larger than that of the two-exponential model, which implies that tests underlying the pairwise phenotypic comparisons were conservative. To see why, recall that the variance of an exponential with mean λ is <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-163.gif"/></noscript></span></span>. The variance of the sum of two independent exponentials with means <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-164.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-165.gif"/></noscript></span></span> is the sum of their individual variances: <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-166.gif"/></noscript></span></span>. But the variance of a single exponential with mean <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-167.gif"/></noscript></span></span> is <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-168.gif"/></noscript></span></span>. Therefore, although we might have gained marginally more power in our phenotypic estimates by adopting a more complex two-exponential statistical model (supplemented with data on adsorption) or following the preattachment protocols described in <a id="xref-ref-9-6" class="xref-bibr" href="#ref-9">Brown <em>et al.</em> 2013</a> and Cherwa Jr <em>et al.</em> 2009, the methods we employed were adequate.</p><p id="p-59">The regression analysis of burst size by time revealed that burst size increased with time for all clones (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). The parameter estimates for each clone are provided in <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a> along with confidence intervals. <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> illustrates these regression parameter estimates graphically, showing that clones fell into four clusters of decreasing slope: (1) <em>pos6</em>, (2) <em>pos4B</em> and <em>pos5</em>, (3) wild type and <em>mut319</em>, and (4) <em>mut321</em>, <em>mut323</em>, and <em>mut324</em>. Formal testing of parameter differences was not conducted. However, where confidence intervals were nonoverlapping, there was good evidence for a significant difference. For slope, the confidence intervals within the four clusters were generally overlapping while intervals between clusters were not.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/939/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimated burst size by time relationships for wild type and each mutant." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1291633927" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimated burst size by time relationships for wild type and each mutant.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/939/F5.medium.gif" width="440" height="411"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/4/939/F5.medium.gif" width="440" height="411"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/939/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/939/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468028" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-60" class="first-child">Estimated burst size by time relationships for wild type and each mutant.</p><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467928/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467928/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Parameter estimates and confidence intervals for burst size by time regression analysis</span><div class="sb-div caption-clear"></div></div></div><p id="p-62"><a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a> summarizes the calculated mean and variance in lysis time for each clone.</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467957/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467957/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Estimates of lysis time mean and variance of <strong>φ</strong>X174 clones</span><div class="sb-div caption-clear"></div></div></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-64">In recent years, it has been found that the probability that a beneficial mutation will fix in a lytic phage population depends sensitively on the life history parameter it affects (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Wahl and DeHaan 2004</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Hubbarde <em>et al.</em> 2007</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Patwa and Wahl 2008</a>, <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">2009</a>). Furthermore, analytical work has demonstrated that variance in life history traits, not just the trait means, can influence growth rate predictions, particularly in iteroparous populations far from a stable age distribution (<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Bull <em>et al.</em> 2011</a>). Inspired by this body of work, we wished to learn more about whether bacteriophage life history traits, including higher moments (<em>e.g.</em>, lysis time variance), could be subject to the phage’s genetic control.</p><p id="p-65">To this end, we used a serially sampled parallel infection assay to interrogate the lysis time and burst size phenotypes of a panel of eight genetically distinct φX174 clones. Among these clones were four with mutations in the <em>D</em> promoter that have been reported to downregulate the transcription of several genes, including the lysis gene (<em>E</em>); three <em>Epos</em> clones with mutations in <em>E</em> that have been found to upregulate its expression; and the wild-type ancestor of the <em>Epos</em> mutants. (As noted in the <em>Materials and Methods</em>, these mutations were introduced on slightly different genetic backgrounds.)</p><p id="p-66">Using a novel statistical approach that allowed us to overcome the challenge of hidden variables, we found significant variation in both lysis time and burst size among these clones, including evidence that φX174 exerts genetic control over lysis time variance under our experimental conditions.</p><p id="p-67">The inferred differences among the clones’ burst size phenotypes are generally consistent with our limited understanding of the mutations’ biological effects (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a> and <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). In accordance with their depressed rates of transcription at the <em>D</em> promoter (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Brown <em>et al.</em> 2010</a>), burst size increased more slowly than the wild type in three of the four <em>D</em>-promoter mutants. (Burst size increased with lysis time at about the same rate as the wild type in the fourth <em>D</em>-promoter mutant, <em>mut319</em>.) By contrast, burst size increased more quickly than the wild type in the three <em>Epos</em> mutants. This may be related to the findings of <a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Bernhardt <em>et al.</em> 2002</a> that the R3H and L19F mutations lead to markedly faster E protein synthesis than the wild type, perhaps by increasing mRNA translatability. Given the compact genome of φX174 and the location of <em>E</em> within the <em>D</em> gene (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>), it is possible that these mutations somehow upregulate gene expression more globally, thereby precipitating more rapid accumulation of progeny virions and larger burst sizes.</p><p id="p-68">As with burst size, we inferred significant variation in lysis time phenotypes among the eight clones under the tested experimental conditions. The clones seem to differ in their overall cumulative lysis probability functions (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>) and in both <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-174.gif"/></noscript></span></span> and λ individually (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>).</p><p id="p-69">Within these differences, we observe an inverse relationship between <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-175.gif"/></noscript></span></span> and λ (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>): under the tested conditions, the mutations appear to delay the onset of lysis while causing the cumulative lysis probability to rise more quickly compared to the wild type (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). (The most noteworthy exception is <em>pos6</em>, the only clone with a value of <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-176.gif"/></noscript></span></span> less than that of the wild type.) A consequence of this trend is that, despite substantial differences in <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-177.gif"/></noscript></span></span> and λ, the clones’ mean lysis times (computed as the sum of <span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-178.gif"/></noscript></span></span> and λ) are rather similar (<a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a>). A mechanistic investigation of the apparent maintenance of mean lysis time is beyond the scope of this study, but we speculate that it may reflect constraints on lysis timing imposed by the host’s cell cycle (for φX174: <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Witte <em>et al.</em> 1998</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Young <em>et al.</em> 2000</a>; and other phages as well: <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hadas <em>et al.</em> 1997</a>; <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Storms <em>et al.</em> 2014</a>). While mutations in <em>E</em> and related genes could potentially bias lysis timing earlier or later, or make the process more or less noisy, the host’s physiology and cell cycle may prevent extreme divergence from a common mean.</p><p id="p-70">In fact, given the genetic and phenotypic constraints on φX174 life history, it is notable that burst size and lysis time phenotypes appear to diverge as much as they do. A primary genetic constraint is the phage’s compact genomic architecture. The <em>E</em> gene is entirely embedded within the structurally essential <em>D</em> gene (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Barrell <em>et al.</em> 1976</a>; <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>), and the <em>D</em> promoter simultaneously regulates the transcription of <em>E</em>, <em>D</em>, and four other structural genes downstream (<em>i.e.</em>, <em>J</em>, <em>F</em>, <em>G</em>, and <em>H</em>) (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Fane <em>et al.</em> 2006</a>). The work presented here stands with and expands upon previous studies (<em>e.g.</em>, <a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">Brown <em>et al.</em> 2010</a>, <a id="xref-ref-9-7" class="xref-bibr" href="#ref-9">2013</a>; <a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">Bernhardt <em>et al.</em> 2002</a>) in demonstrating that, despite the relatively limited genotype space accessible to the organism, the genomic architecture of φX174 permits a significant degree of flexibility in lysis time and burst size phenotypes. Still, the genetic constraints on these phenotypes are clearly visible in the form of pleiotropic effects on burst size and lysis time in our data.</p><p id="p-71">The inferred disparity among our clones’ lysis time parameters is also interesting given the aforementioned phenotypic constraints imposed by the host’s cell cycle and physiology. Whereas λ and many other larger DNA phages effect lysis via a tightly regulated, two-protein lysis strategy, φX174 and other <em>Microviridae</em> lyse their hosts by way of a single protein (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Young <em>et al.</em> 2000</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Young and Wang 2006</a>). The lysis timing of φX174 is thought to be dictated largely by the timing of host cell division and, therefore, compared with phages like λ, even less subject to the phage’s genetic control (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Witte <em>et al.</em> 1998</a>; <a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Young <em>et al.</em> 2000</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Zheng <em>et al.</em> 2008</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Dennehy and Wang 2011</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Singh and Dennehy 2014</a>; <a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Storms <em>et al.</em> 2014</a>). But our demonstration of genetic control of both lysis time mean and variance hints at the possibility of as-yet-unknown mechanisms exerting some control over phage lysis timing. Indeed, researchers have long speculated that protein E may mediate lysis timing not just through its interactions with MraY, but also through interactions with the host protein SlyD (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Young and Wang 2006</a>). SlyD plays an important role in cell division and cell cycle regulation (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Roof <em>et al.</em> 1997</a>) and is required for stability of wild-type E protein (<a id="xref-ref-7-7" class="xref-bibr" href="#ref-7">Bernhardt <em>et al.</em> 2002</a>).</p><div id="sec-13" class="subsection"><h3>Strengths and limitations of our experimental and statistical approach</h3><p id="p-72">The strength of the experimental and statistical approach taken in this study lies in its power and applicability. It is powerful in that, more effectively than most other methods, it simultaneously captures information related to lysis time variance and burst size as a function of time. The approach also has potential application to a range of microbiological questions. The principles and tools underlying the assay and inferential apparatus could be applied usefully to other cases in which one seeks to make inferences about dynamic, Poisson-distributed processes that are sampled destructively over a time course (<em>e.g.</em>, tracking the fates of individual tagged mutants in a dynamic population of cells or viruses).</p><p id="p-73">However, we concede our approach has limitations. There is increasing evidence that host cells’ physiology plays a vital role in determining phage infection dynamics (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Hadas <em>et al.</em> 1997</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Rabinovitch <em>et al.</em> 2002</a>; <a id="xref-ref-50-4" class="xref-bibr" href="#ref-50">Storms <em>et al.</em> 2014</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Bull <em>et al.</em> 2014</a>). Yet, in synchronizing and rapidly sampling infections, our assay exposed the cells to temperature changes and nutritional regimes on a short timescale, which may have altered cells’ physiology in ways important for the measured phenotypes (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">van Elsas <em>et al.</em> 2011</a>). While these sorts of manipulations are not uncommon in the literature (<em>e.g.</em>, <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Wang 2006</a>; Cherwa Jr <em>et al.</em> 2009), they limit the generality of some conclusions drawn from this work. The unique conditions of the assay complicate comparisons to previous estimates of the clones’ burst size and lysis time phenotypes, and they make it impractical to validate our methods against traditional single-step growth experiments with much confidence.</p><p id="p-74">By the same token, the lysis times and burst sizes inferred in this work may differ from those φX174 exhibits in nature. Environmental differences in factors like host density and diversity, as well as temperature changes and nutritional regimes, could possibly lead to differences between phenotypes measured in the assay and in the wild. That said, it is unclear how well any particular assay approximates the phage’s natural environment, which remains largely uncharacterized (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Michel <em>et al.</em> 2010</a>). Further study of the ecological niche of φX174 would be necessary to clarify which environmental differences between our assay and the wild are most relevant to the phage’s life history phenotypes.</p><p id="p-75">Finally, our assumption of a linear increase in burst size with time, though substantiated by significant past work with φX174 and other phages (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Hutchison and Sinsheimer 1966</a>; <a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Wang <em>et al.</em> 1996</a>; <a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Wang 2006</a>), may be subject to debate (<a id="xref-ref-50-5" class="xref-bibr" href="#ref-50">Storms <em>et al.</em> 2014</a>). We hope further research will help better evaluate the appropriateness of this assumption.</p><p id="p-76">When it comes to achieving the core aims of this work, we believe the strengths of our experimental and statistical approach outweigh its limitations. It allowed us to demonstrate that, under at least one set of conditions held constant for eight similar but distinct φX174 clones, burst size and lysis time phenotypes (including lysis time variance) differed significantly. Furthermore, it is reasonable to assume that the degree of genetic variability among the clones is comparable to that which is the substrate for natural selection acting in the wild.</p></div><div id="sec-14" class="subsection"><h3>Evolutionary implications of heritable variation in lysis time variance</h3><p id="p-77">Our finding that variance in the lysis time of φX174 can be subject to at least some genetic control inspired us to ask, “What, then, could be the evolutionary implications of mutations affecting variance in phage lysis time?” Trivially, if burst size grows more slowly than exponentially once a cell has been infected, then a mutation increasing lysis time variance will be selectively favored, because early lysis events contribute more to population growth than late lysis events detract (<a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Bull <em>et al.</em> 2011</a>; but see <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Singh and Dennehy 2014</a>). Empirically, phage burst size is thought to increase linearly (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Hutchison and Sinsheimer 1966</a>; <a id="xref-ref-55-4" class="xref-bibr" href="#ref-55">Wang <em>et al.</em> 1996</a>; <a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">Wang 2006</a>; but see <a id="xref-ref-50-6" class="xref-bibr" href="#ref-50">Storms <em>et al.</em> 2014</a>), immediately suggesting one mode of selection. Mutations increasing lysis time variance in a given environment likely would fix quickly under this mode of selection.</p><p id="p-78">In order to control for the influence of this mode of selection, we recently have explored a model in which burst size increases exponentially in time since infection (D. M. Weinreich, C. Brown, and L. M. Wahl, unpublished data). This has allowed us to infer the intrinsic selective consequences of mutations that influence lysis time variance. As expected, in phage populations growing at their stable age-of-infection distribution in unlimited host cells (<a id="xref-ref-13-6" class="xref-bibr" href="#ref-13">Bull <em>et al.</em> 2011</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bull 2006</a>), we found such mutations to be selectively neutral, since, by construction, the contribution of early bursts to population growth is now exactly balanced by (exponentially larger) late bursts.</p><p id="p-79">However, the moment a mutation appears, the mutant subpopulation is by definition not at stable age distribution, and indeed we have found in simulations that mutations increasing variance in lysis time are transiently enriched in the population (D. M. Weinreich and C. Brown, unpublished data). This effect follows from the fact that the frequency of phages with high lysis time variance exceeds (sometimes considerably) the frequency of their low-variance brethren early in each lysis cycle. While the reverse is true late in the cycle, the effect is of smaller magnitude and much shorter lived. Finally, overlaying periodic demographic bottlenecks (<em>e.g.</em>, as employed in lab conditions, but also surely in nature), we observed that mutations increasing lysis time variance can enjoy a dramatic increase in population frequency, despite their neutrality when they finally reach their stable age-of-infection distribution (D. M. Weinreich and C. Brown, unpublished). We therefore suggest a second mode of selection acting on mutations affecting variance in lysis time.</p></div><div id="sec-15" class="subsection"><h3>Future directions</h3><p id="p-80">The findings reported here suggest two complementary avenues for future investigation. One is microbiological, and it involves exploring φX174 lysis in relation to its genetic determinants and its host’s biology. It would be interesting to explore the mutants’ performance in a range of assays, such as those involving thermally regulated microscopy (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Dennehy and Wang 2011</a>), single-cell microfluidics-based techniques (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Yin and Marshall 2012</a>), or periodic filtration and sampling (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Hutchison and Sinsheimer 1963</a>). (The last and oldest of these techniques could prove especially fruitful, since it would allow direct access to both lysis time and burst size.) In addition, it would be highly informative to use methods like those designed by <a id="xref-ref-50-7" class="xref-bibr" href="#ref-50">Storms <em>et al.</em> 2014</a> to study how φX174 mutants’ burst sizes and lysis times vary as a direct function of the host’s cell cycle. These types of experiments could yield valuable insight into the regulation and host-dependence of φX174 life history phenotypes.</p><p id="p-81">Theoretical evolutionary problems present another opportunity for future research. Does natural selection act on lysis time variance in nature? What is the fixation probability of a mutation that affects lysis time variance, and how do populations carrying such mutations evolve in the face of periodic bottlenecks? Existing computational models could be strengthened by more thoroughly exploring the models’ parameter sensitivity and investigating how mutations that pleiotropically influence adsorption, lysis time, and burst size affect predicted population dynamics. In addition, the models must be integrated with other theoretical work exploring the fixation probabilities of mechanistically distinct beneficial mutations in lytic phages (<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Wahl and DeHaan 2004</a>; <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Hubbarde <em>et al.</em> 2007</a>; <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Patwa and Wahl 2008</a>, <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">2009</a>). Once elaborated and refined, models could be tested empirically through both “single-sampling” assays and longer-term evolution experiments. The former approach would assess the probability of loss of lysis time variants introduced at very low frequency in replicate phage populations, while the latter would explore whether real evolutionary outcomes match our theoretical predictions.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-97">We thank Olivier Tenaillion, Amber Stancik, Celeste Brown, Rohit Kongari, and Ry Young for providing the bacterial and phage strains used in this study. Additionally, we thank Darin Rokyta for providing us with the <em>E. coli</em> starvation protocol and Sohini Ramachandran for offering feedback on an early draft of this manuscript. We also are grateful for support from the Brown Division of Biology and Medicine to D.M.W., from the Brown Dean of the College to J.Y., and from the National Science Foundation (NSF) (G-K12 Award 0638688 to T. Herbert) to support M.H.B. C.R.M. and P.J. were supported by grant number R01-GM076040 from the National Institutes of Health. This research is based in part upon work conducted using the Rhode Island Genomics and Sequencing Center, which is supported in part by the NSF under EPSCoR grants nos. 0554548 and EPS-1004057.</p></div><div class="section app" id="app-1"><h2 class="">Appendix 1: Number of Phages per Well Poisson Estimator</h2><p id="p-82">The counts from an assay place each well into one of three categories: empty, not showing lysis events but nonzero, and showing lysis events. The probability of a well being empty is <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-179.gif"/></noscript></span></span>. The probability that a well has <em>n</em> phages is <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-180.gif"/></noscript></span></span>. The probability that all <em>n</em> phages fail to lyse their host is <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-181.gif"/></noscript></span></span>, leading the probability of <em>n</em> in a well showing no lysis events to be <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-182.gif"/></noscript></span></span>. Finally, the probability of a well showing one or more lysis events is one minus the probability of a well being empty, plus the sum of all the probabilities associated with wells that do not show lysis events, which turns out to be <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-183.gif"/></noscript></span></span>. The categorization of wells into three types leads to the following likelihood.<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-184.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>Therefore,<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-185.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>The maximum likelihood estimator is obtained by first calculating two partial derivatives,<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-186.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-187.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span>and then setting both <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-188.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-189.gif"/></noscript></span></span>. Denote the solution by <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-190.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-191.gif"/></noscript></span></span>, respectively. The solution associated with Equation 8 leads to<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-192.gif"/></noscript></span><span class="disp-formula-label">(10)</span></span>and the solution associated with Equation 9 gives<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-193.gif"/></noscript></span><span class="disp-formula-label">(11)</span></span>Now multiply both sides of the above equation by <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-194.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-195.gif"/></noscript></span><span class="disp-formula-label">(12)</span></span>Note that second term on the right side of Equation 10 is equivalent to the left side of Equation 12. This leads to the following simple relationship between <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-196.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-197.gif"/></noscript></span></span>:<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-198.gif"/></noscript></span><span class="disp-formula-label">(13)</span></span>Thus <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-199.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-200.gif"/></noscript></span></span>. We can eliminate <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-201.gif"/></noscript></span></span> from Equation 10 by substituting <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-202.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-203.gif"/></noscript></span></span> to get<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-204.gif"/></noscript></span><span class="disp-formula-label">(14)</span></span>Solving Equation 14 for <span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-205.gif"/></noscript></span></span> gives the estimator,</p><span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-206.gif"/></noscript></span><span class="disp-formula-label">(15)</span></span></div><div class="section app" id="app-2"><h2 class="">Appendix 2: Time to Lysis</h2><p id="p-83">Recall that we model the probability of a phage lysing its host by time <em>t</em> with a shifted exponential, <span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-207.gif"/></noscript></span></span>. Define <span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-208.gif"/></noscript></span></span> as the number of phages in well <em>r</em>, assay <em>a</em>, that have lysed their hosts by time <em>t</em> (when sampling occurs). <span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-209.gif"/></noscript></span></span> is Poisson with mean <span class="inline-formula" id="inline-formula-195"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-210.gif"/></noscript></span></span>. Let <span class="inline-formula" id="inline-formula-196"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-211.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-211.gif"/></noscript></span></span> be an indicator with value 1 if <span class="inline-formula" id="inline-formula-197"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-212.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-212.gif"/></noscript></span></span> and 0 otherwise. Let <span class="inline-formula" id="inline-formula-198"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-213.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-213.gif"/></noscript></span></span> be the number of phages that have not lysed their hosts in well <em>r</em>, assay <em>a</em>, at time <em>t</em> (<span class="inline-formula" id="inline-formula-199"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-214.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-214.gif"/></noscript></span></span>). Based on standard theory <span class="inline-formula" id="inline-formula-200"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-215.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-215.gif"/></noscript></span></span> is independent of <span class="inline-formula" id="inline-formula-201"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-216.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-216.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-202"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-217.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-217.gif"/></noscript></span></span> is also Poisson distributed but with mean <span class="inline-formula" id="inline-formula-203"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-218.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-218.gif"/></noscript></span></span>. However, <span class="inline-formula" id="inline-formula-204"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-219.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-219.gif"/></noscript></span></span> is only directly observable when no phages in the well have lysed their hosts (<em>i.e.</em>, when <span class="inline-formula" id="inline-formula-205"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-220.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-220.gif"/></noscript></span></span>). We now show how to use the binary <span class="inline-formula" id="inline-formula-206"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-221.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-221.gif"/></noscript></span></span> together with Poisson distributed <span class="inline-formula" id="inline-formula-207"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-222.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-222.gif"/></noscript></span></span> to estimate λ and <span class="inline-formula" id="inline-formula-208"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-223.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-223.gif"/></noscript></span></span>. First, the likelihood of the lysis data are given by,<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-224.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-224.gif"/></noscript></span><span class="disp-formula-label">(16)</span></span>The likelihood for the number of phages in wells not showing lysis events is given by<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-225.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-225.gif"/></noscript></span><span class="disp-formula-label">(17)</span></span>However, to account for the fact that <span class="inline-formula" id="inline-formula-209"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-226.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-226.gif"/></noscript></span></span> is only observable when no lysis occurs (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-210"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-227.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-227.gif"/></noscript></span></span>), we must adjust the likelihood for wells showing plaques but not showing lysis events as follows.<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-228.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-228.gif"/></noscript></span><span class="disp-formula-label">(18)</span></span>The combined likelihood is,<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-229.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-229.gif"/></noscript></span><span class="disp-formula-label">(19)</span></span>Dropping factorial terms that do not involve parameters and taking the log, the log-likelihood is proportional to<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-230.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-230.gif"/></noscript></span><span class="disp-formula-label">(20)</span></span>As discussed in the main text, we estimate λ and <span class="inline-formula" id="inline-formula-211"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-231.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-231.gif"/></noscript></span></span> by brute force, where we calculate the likelihood using Equations 3 and 20 for a large array of joint values, taking the largest as the (approximate) maximum likelihood estimates.</p></div><div class="section app" id="app-3"><h2 class="">Appendix 3: Burst Size as a Function of Size</h2><p id="p-84">The count observable in well <em>r</em> from assay <em>a</em> taken at time <em>t</em>, <span class="inline-formula" id="inline-formula-212"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-232.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-232.gif"/></noscript></span></span>, can be expressed as,<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-233.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-233.gif"/></noscript></span><span class="disp-formula-label">(21)</span></span>where <span class="inline-formula" id="inline-formula-213"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-234.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-234.gif"/></noscript></span></span> is the lysis time of phage <em>i</em> in well <em>r</em> of assay <em>a</em>, and <em>I</em> represents an indicator function that takes value 1 when the condition that follows is true, and 0 otherwise. This simplifies to, <span class="inline-formula" id="inline-formula-214"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-235.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-235.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-215"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-236.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-236.gif"/></noscript></span></span> is the number of phages initially in well <em>r</em> of assay <em>a</em> sampled at time <em>t</em>, <span class="inline-formula" id="inline-formula-216"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-237.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-237.gif"/></noscript></span></span> is the sum of their lysis times, and δ is normally distributed with mean zero and variance <span class="inline-formula" id="inline-formula-217"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-238.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-238.gif"/></noscript></span></span>. Our goal is to then estimate μ, α, and σ. Because <span class="inline-formula" id="inline-formula-218"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-239.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-239.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-219"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-240.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-240.gif"/></noscript></span></span> are unobservable but are needed to estimate the parameters, we employ the following EM algorithm which replaces them with their expected value. Note that these steps summarize the process; we provide derivations and details in subsections below.</p><div class="section" id="sec-16"><div id="sec-17" class="subsection"><h4><em>EM algorithm</em></h4><ol class="list-ord " id="list-2"><li id="list-item-4"><p id="p-85">Begin with initial guesses for <span class="inline-formula" id="inline-formula-220"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-241.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-241.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-221"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-242.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-242.gif"/></noscript></span></span> for each well. Let <span class="inline-formula" id="inline-formula-222"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-243.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-243.gif"/></noscript></span></span>.</p></li><li id="list-item-5"><p id="p-86">Given these values, estimate α and μ using least-squares.<span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-244.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-244.gif"/></noscript></span><span class="disp-formula-label">(22)</span></span></p></li><li id="list-item-6"><p id="p-87">Use the least-squares estimates <span class="inline-formula" id="inline-formula-223"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-245.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-245.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-224"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-246.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-246.gif"/></noscript></span></span> to estimate <span class="inline-formula" id="inline-formula-225"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-247.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-247.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-248.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-248.gif"/></noscript></span><span class="disp-formula-label">(23)</span></span></p></li><li id="list-item-7"><p id="p-88">Impute the values <span class="inline-formula" id="inline-formula-226"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-249.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-249.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-227"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-250.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-250.gif"/></noscript></span></span> using their posterior expected values. Details below.<span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-251.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-251.gif"/></noscript></span><span class="disp-formula-label">(24)</span></span><span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-252.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-252.gif"/></noscript></span><span class="disp-formula-label">(25)</span></span></p></li><li id="list-item-8"><p id="p-89">Return to step 2, replacing <em>j</em> with <span class="inline-formula" id="inline-formula-228"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-253.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-253.gif"/></noscript></span></span>, until convergence occurs.</p></li></ol></div><div id="sec-18" class="subsection"><h4><em>Least squares estimates for α and μ (step 2 in EM algorithm)</em></h4><p id="p-90">Because the error is normal, values of μ and α that minimize the sum of squared differences between observed and expected counts provide the maximum likelihood estimates. The function that expresses this sum of squares is,<span class="disp-formula" id="disp-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-254.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-254.gif"/></noscript></span><span class="disp-formula-label">(26)</span></span>Then<span class="disp-formula" id="disp-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-255.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-255.gif"/></noscript></span><span class="disp-formula-label">(27)</span></span>Set <span class="inline-formula" id="inline-formula-229"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-256.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-256.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-230"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-257.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-257.gif"/></noscript></span></span> solve. The solutions are the least squares estimates denoted by <span class="inline-formula" id="inline-formula-231"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-258.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-258.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-232"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-259.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-259.gif"/></noscript></span></span> satisfy the following set of linear equations.<span class="disp-formula" id="disp-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-260.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-260.gif"/></noscript></span><span class="disp-formula-label">(28)</span></span>For notational convenience let<span class="disp-formula" id="disp-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-261.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-261.gif"/></noscript></span><span class="disp-formula-label">(29)</span></span>The estimates are then<span class="disp-formula" id="disp-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-262.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-262.gif"/></noscript></span><span class="disp-formula-label">(30)</span></span>and</p><span class="disp-formula" id="disp-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-263.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-263.gif"/></noscript></span><span class="disp-formula-label">(31)</span></span></div><div id="sec-19" class="subsection"><h4><em>Conditional expectations (step 4 in EM algorithm)</em></h4><p>Now we will calculate the expected number phages conditional on the size of the burst <span class="inline-formula" id="inline-formula-233"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-264.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-264.gif"/></noscript></span></span> as well as the expected sum of lysis times given the burst size <span class="inline-formula" id="inline-formula-234"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-265.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-265.gif"/></noscript></span></span>. Throughout this section, assay <em>a</em> and well <em>r</em> will be fixed; for notational convenience we will drop those subscripts and just denote <span class="inline-formula" id="inline-formula-235"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-266.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-266.gif"/></noscript></span></span> by <span class="inline-formula" id="inline-formula-236"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-267.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-267.gif"/></noscript></span></span> respectively. Because we will be simulating the joint distribution of <span class="inline-formula" id="inline-formula-237"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-268.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-268.gif"/></noscript></span></span> conditional on <span class="inline-formula" id="inline-formula-238"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-269.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-269.gif"/></noscript></span></span> we will sometimes use a second subscript. In this case <span class="inline-formula" id="inline-formula-239"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-270.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-270.gif"/></noscript></span></span> will represent the <em>j</em>th simulated value. To do this, we will first need the joint distribution of <span class="inline-formula" id="inline-formula-240"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-271.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-271.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-241"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-272.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-272.gif"/></noscript></span></span>, which we obtain through the following simulation algorithm.</p><ol class="list-ord " id="list-3"><li id="list-item-9"><p id="p-92">Simulate <span class="inline-formula" id="inline-formula-242"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-273.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-273.gif"/></noscript></span></span> from a Poisson distribution with mean <span class="inline-formula" id="inline-formula-243"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-274.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-274.gif"/></noscript></span></span>. If <span class="inline-formula" id="inline-formula-244"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-275.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-275.gif"/></noscript></span></span> repeat, otherwise proceed.</p></li><li id="list-item-10"><p id="p-93">Simulate <span class="inline-formula" id="inline-formula-245"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-276.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-276.gif"/></noscript></span></span> from a shifted exponential with mean <span class="inline-formula" id="inline-formula-246"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-277.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-277.gif"/></noscript></span></span>, and shift <span class="inline-formula" id="inline-formula-247"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-278.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-278.gif"/></noscript></span></span> estimated using the likelihood equation (4).</p></li><li id="list-item-11"><p id="p-94">Calculate <span class="inline-formula" id="inline-formula-248"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-279.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-279.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-249"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-280.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-280.gif"/></noscript></span></span> for each value of <em>t</em> in the assay. Round values of <span class="inline-formula" id="inline-formula-250"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-281.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-281.gif"/></noscript></span></span> to the nearest tenth of a minute (for values &lt; 1 min), the nearest quarter minute for values between 1 min and 5 min, and the nearest minute for values between 5 min and 100 min. Store.</p></li><li id="list-item-12"><p id="p-95">Repeat 1 through 3 for <span class="inline-formula" id="inline-formula-251"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-282.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-282.gif"/></noscript></span></span> where we set <span class="inline-formula" id="inline-formula-252"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-283.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-283.gif"/></noscript></span></span>.</p></li><li id="list-item-13"><p id="p-96">For each value of <em>t</em>, convert the observed joint values of <span class="inline-formula" id="inline-formula-253"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-284.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-284.gif"/></noscript></span></span> to proportions.</p></li></ol><p>The above algorithm generates the empirical distribution <span class="inline-formula" id="inline-formula-254"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-285.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-285.gif"/></noscript></span></span>. We note that the distribution for <span class="inline-formula" id="inline-formula-255"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-286.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-286.gif"/></noscript></span></span> conditional on <span class="inline-formula" id="inline-formula-256"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-287.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-287.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-257"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-288.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-288.gif"/></noscript></span></span> is normally distributed with mean <span class="inline-formula" id="inline-formula-258"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-289.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-289.gif"/></noscript></span></span> and variance <span class="inline-formula" id="inline-formula-259"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-290.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-290.gif"/></noscript></span></span>. Denote this distribution by<span class="disp-formula" id="disp-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-291.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-291.gif"/></noscript></span><span class="disp-formula-label">(32)</span></span>then<span class="disp-formula" id="disp-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-292.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-292.gif"/></noscript></span><span class="disp-formula-label">(33)</span></span>and<span class="disp-formula" id="disp-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-293.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-293.gif"/></noscript></span><span class="disp-formula-label">(34)</span></span>where<span class="disp-formula" id="disp-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-294.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-294.gif"/></noscript></span><span class="disp-formula-label">(35)</span></span>Using Equations 33 and 34 we can calculate the expected values of <span class="inline-formula" id="inline-formula-260"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-295.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-295.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-261"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-296.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4/939/embed/mml-math-296.gif"/></noscript></span></span> for use in the EM algorithm.</p></div></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-12"><p id="p-98">Supplemental Material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024075/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024075/-/DC1</a></p></li><li class="fn" id="fn-13"><p id="p-99"><em>Communicating Editor: P. C. Phillips</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-10-24"><span class="received-label">Received </span>October 24, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-02-02"><span class="accepted-label">Accepted </span>February 2, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Baker <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-6">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.1" data-doi="10.1007/BF02030117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abedon</span>  <span class="cit-name-given-names">S. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Selection for bacteriophage latent period length by bacterial density: a theoretical examination.</span> <abbr class="cit-jnl-abbrev">Microb. Ecol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrob.%2BEcol.%26rft.volume%253D18%26rft.spage%253D79%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02030117%26rft_id%253Dinfo%253Apmid%252F24196124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02030117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24196124&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989AT46400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.2" data-doi="10.1128/jb.174.24.8073-8080.1992"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abedon</span>  <span class="cit-name-given-names">S. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Lysis of lysis-inhibited bacteriophage T4-infected cells.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">8073</span>–<span class="cit-lpage">8080</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DAbedon%26rft.auinit1%253DS.%2BT.%26rft.volume%253D174%26rft.issue%253D24%26rft.spage%253D8073%26rft.epage%253D8080%26rft.atitle%253DLysis%2Bof%2Blysis-inhibited%2Bbacteriophage%2BT4-infected%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.174.24.8073-8080.1992%26rft_id%253Dinfo%253Apmid%252F1459956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE3NC8yNC84MDczIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80LzkzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.3" data-doi="10.1128/AEM.67.9.4233-4241.2001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abedon</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herschler</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stopar</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Bacteriophage latent-period evolution as a response to resource availability.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">4233</span>–<span class="cit-lpage">4241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft.stitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft.aulast%253DAbedon%26rft.auinit1%253DS.%2BT.%26rft.volume%253D67%26rft.issue%253D9%26rft.spage%253D4233%26rft.epage%253D4241%26rft.atitle%253DBacteriophage%2BLatent-Period%2BEvolution%2Bas%2Ba%2BResponse%2Bto%2BResource%2BAvailability%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.67.9.4233-4241.2001%26rft_id%253Dinfo%253Apmid%252F11526028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjk6IjY3LzkvNDIzMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC85MzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.4" data-doi="10.1016/j.cels.2015.07.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Afshinnekoo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meydan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chowdhury</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaroudi</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Geospatial resolution of human and bacterial diversity with city-scale metagenomics.</span> <abbr class="cit-jnl-abbrev">Cell Syst.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BSyst.%26rft.volume%253D1%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cels.2015.07.009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cels.2015.07.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.5" data-doi="10.1038/264034a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrell</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Air</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hutchison</span>  <span class="cit-name-given-names">C. A.</span>  <span class="cit-name-suffix">III</span>.</span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">Overlapping genes in bacteriophage φX174.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">264</span>: <span class="cit-fpage">34</span>–<span class="cit-lpage">41</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBarrell%26rft.auinit1%253DB.%2BG.%26rft.volume%253D264%26rft.issue%253D5581%26rft.spage%253D34%26rft.epage%253D41%26rft.atitle%253DOverlapping%2Bgenes%2Bin%2Bbacteriophage%2BphiX174.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F264034a0%26rft_id%253Dinfo%253Apmid%252F1004533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/264034a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1004533&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1976CJ56500018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.6" data-doi="10.1016/j.jmb.2004.02.033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernal</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hafenstein</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esmeralda</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fane</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rossmann</span>  <span class="cit-name-given-names">M. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The φX174 protein J mediates DNA packaging and viral attachment to host cells.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">1109</span>–<span class="cit-lpage">1122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DBernal%26rft.auinit1%253DR.%2BA.%26rft.volume%253D337%26rft.issue%253D5%26rft.spage%253D1109%26rft.epage%253D1122%26rft.atitle%253DThe%2BphiX174%2Bprotein%2BJ%2Bmediates%2BDNA%2Bpackaging%2Band%2Bviral%2Battachment%2Bto%2Bhost%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2004.02.033%26rft_id%253Dinfo%253Apmid%252F15046981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2004.02.033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15046981&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.7" data-doi="10.1046/j.1365-2958.2002.02984.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernhardt</span>  <span class="cit-name-given-names">T. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roof</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The <em>Escherichia coli</em> FKBP-type PPIase SlyD is required for the stabilization of the E lysis protein of bacteriophage φX174.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">99</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DBernhardt%26rft.auinit1%253DT.%2BG.%26rft.volume%253D45%26rft.issue%253D1%26rft.spage%253D99%26rft.epage%253D108%26rft.atitle%253DThe%2BEscherichia%2Bcoli%2BFKBP-type%2BPPIase%2BSlyD%2Bis%2Brequired%2Bfor%2Bthe%2Bstabilization%2Bof%2Bthe%2BE%2Blysis%2Bprotein%2Bof%2Bbacteriophage%2Bphi%2BX174.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.2002.02984.x%26rft_id%253Dinfo%253Apmid%252F12100551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.2002.02984.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12100551&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.8" data-doi="10.1186/1471-2148-10-378"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ally</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stancik</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Positive selection at high temperature reduces gene transcription in the bacteriophage φX174.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">378</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DBrown%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D10%26rft.spage%253D378%26rft.epage%253D378%26rft.atitle%253DPositive%2Bselection%2Bat%2Bhigh%2Btemperature%2Breduces%2Bgene%2Btranscription%2Bin%2Bthe%2Bbacteriophage%2B%25CF%2586X174.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-10-378%26rft_id%253Dinfo%253Apmid%252F21129199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-10-378&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21129199&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.9" data-doi="10.1186/1471-2148-13-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stancik</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roychoudhury</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krone</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Adaptive regulatory substitutions affect multiple stages in the life cycle of the bacteriophage φX174.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol.%2BBiol.%26rft.volume%253D13%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-13-1%26rft_id%253Dinfo%253Apmid%252F23279962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-13-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23279962&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.10" data-doi="10.1016/j.jtbi.2006.01.027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bull</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Optimality models of phage life history and parallels in disease evolution.</span> <abbr class="cit-jnl-abbrev">J. Theor. Biol.</abbr> <span class="cit-vol">241</span>: <span class="cit-fpage">928</span>–<span class="cit-lpage">938</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DBull%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D241%26rft.issue%253D4%26rft.spage%253D928%26rft.epage%253D938%26rft.atitle%253DOptimality%2Bmodels%2Bof%2Bphage%2Blife%2Bhistory%2Band%2Bparallels%2Bin%2Bdisease%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jtbi.2006.01.027%26rft_id%253Dinfo%253Apmid%252F16616205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jtbi.2006.01.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16616205&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239842600020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.11" data-doi="10.1016/j.tree.2003.10.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bull</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfennig</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">I.-N.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genetic details, optimization and phage life histories.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">76</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DBull%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D19%26rft.issue%253D2%26rft.spage%253D76%26rft.epage%253D82%26rft.atitle%253DGenetic%2Bdetails%252C%2Boptimization%2Band%2Bphage%2Blife%2Bhistories.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2003.10.008%26rft_id%253Dinfo%253Apmid%252F16701232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2003.10.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16701232&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.12" data-doi="10.1086/499374"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bull</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Millstein</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orcutt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wichman</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Evolutionary feedback mediated through population density, illustrated with viruses in chemostats.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">167</span>: <span class="cit-fpage">E39</span>–<span class="cit-lpage">E51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAm%2BNat%26rft.aulast%253DBull%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D167%26rft.issue%253D2%26rft.spage%253DE39%26rft.epage%253DE51%26rft.atitle%253DEvolutionary%2Bfeedback%2Bmediated%2Bthrough%2Bpopulation%2Bdensity%252C%2Billustrated%2Bwith%2Bviruses%2Bin%2Bchemostats.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F499374%26rft_id%253Dinfo%253Apmid%252F16670974%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/499374&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16670974&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235214200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.13" data-doi="10.1371/journal.pone.0027796"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bull</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heineman</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilke</span>  <span class="cit-name-given-names">C. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The phenotype-fitness map in experimental evolution of phage.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e27796</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBull%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253De27796%26rft.epage%253De27796%26rft.atitle%253DThe%2Bphenotype-fitness%2Bmap%2Bin%2Bexperimental%2Bevolution%2Bof%2Bphages.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0027796%26rft_id%253Dinfo%253Apmid%252F22132144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0027796&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22132144&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.14" data-doi="10.1371/journal.pone.0094690"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bull</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vegge</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmerer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaudhry</span>  <span class="cit-name-given-names">W. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">B. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Phenotypic resistance and the dynamics of bacterial escape from phage control.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e94690</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De94690%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0094690%26rft_id%253Dinfo%253Apmid%252F24743264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0094690&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24743264&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burnet</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1929</span> <span class="cit-article-title">A method for the study of bacteriophage multiplication in broth.</span> <abbr class="cit-jnl-abbrev">Br. J. Exp. Pathol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">109</span>–<span class="cit-lpage">115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBr.%2BJ.%2BExp.%2BPathol.%26rft.volume%253D10%26rft.spage%253D109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.16" data-doi="10.1128/JVI.76.7.3276-3281.2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rang</span>  <span class="cit-name-given-names">C. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">L. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Distribution of spontaneous mutants and inferences about the replication mode of the RNA bacteriophage φ6.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">3276</span>–<span class="cit-lpage">3281</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DChao%26rft.auinit1%253DL.%26rft.volume%253D76%26rft.issue%253D7%26rft.spage%253D3276%26rft.epage%253D3281%26rft.atitle%253DDistribution%2Bof%2BSpontaneous%2BMutants%2Band%2BInferences%2Babout%2Bthe%2BReplication%2BMode%2Bof%2Bthe%2BRNA%2BBacteriophage%2B%257Bphi%257D6%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.76.7.3276-3281.2002%26rft_id%253Dinfo%253Apmid%252F11884552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijc2LzcvMzI3NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC85MzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.4.939.17" data-doi="10.1128/JVI.01297-09"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cherwa</span>  <span class="cit-name-given-names">J. E.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez-Soria</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth cit-collab">Members of the University of Arizona Virology Laboratory Course 2007</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wichman</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fane</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Viral adaptation to an antiviral protein enhances the fitness level to above that of the uninhibited wild type.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">11746</span>–<span class="cit-lpage">11750</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01297-09%26rft_id%253Dinfo%253Apmid%252F19726521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4My8yMi8xMTc0NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC85MzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.18" data-doi="10.1038/nrg2160"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dean</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mechanistic approaches to the study of evolution: the functional synthesis.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">675</span>–<span class="cit-lpage">688</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDean%26rft.auinit1%253DA.%2BM.%26rft.volume%253D8%26rft.issue%253D9%26rft.spage%253D675%26rft.epage%253D688%26rft.atitle%253DMechanistic%2Bapproaches%2Bto%2Bthe%2Bstudy%2Bof%2Bevolution%253A%2Bthe%2Bfunctional%2Bsynthesis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2160%26rft_id%253Dinfo%253Apmid%252F17703238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2160&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17703238&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248882400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Delbrück</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1945</span> <span class="cit-article-title">The burst size distribution in the growth of bacterial viruses (bacteriophages).</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DDelbruck%26rft.auinit1%253DM.%26rft.volume%253D50%26rft.issue%253D2%26rft.spage%253D131%26rft.epage%253D135%26rft.atitle%253DThe%2BBurst%2BSize%2BDistribution%2Bin%2Bthe%2BGrowth%2Bof%2BBacterial%2BViruses%2B%2528Bacteriophages%2529%26rft_id%253Dinfo%253Apmid%252F20989330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czoyOiJqYiI7czo1OiJyZXNpZCI7czo4OiI1MC8yLzEzMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvNC85MzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.20" data-doi="10.1016/S0022-2836(65)80318-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Denhardt</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinsheimer</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1965</span> <span class="cit-article-title">The process of infection with bacteriophage φX174. III. Phage maturation and lysis after synchronized infection.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">641</span>–<span class="cit-lpage">646</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DDenhardt%26rft.auinit1%253DD.%2BT.%26rft.volume%253D12%26rft.issue%253D3%26rft.spage%253D641%26rft.epage%253D646%26rft.atitle%253DThe%2Bprocess%2Bof%2Binfection%2Bwith%2Bbacteriophage%2Bphi-X174.%2B3.%2BPhage%2Bmaturation%2Band%2Blysis%2Bafter%2Bsynchronized%2Binfection.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-2836%252865%252980318-7%26rft_id%253Dinfo%253Apmid%252F5323480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0022-2836(65)80318-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5323480&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.21" data-doi="10.1186/1471-2180-11-174"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dennehy</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">I.-N.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Factors influencing lysis time stochasticity in bacteriophage <em>λ</em>.</span> <abbr class="cit-jnl-abbrev">BMC Microbiol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BMicrobiology%26rft.stitle%253DBMC%2BMicrobiology%26rft.aulast%253DDennehy%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D11%26rft.spage%253D174%26rft.epage%253D174%26rft.atitle%253DFactors%2Binfluencing%2Blysis%2Btime%2Bstochasticity%2Bin%2Bbacteriophage%2B%25CE%25BB.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2180-11-174%26rft_id%253Dinfo%253Apmid%252F21810267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2180-11-174&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21810267&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.939.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fane</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brentlinger</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burch</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hafenstein</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">X174 <em>et al.</em>, the <em>Microviridae</em></span>, pp. <span class="cit-fpage">129</span>–<span class="cit-lpage">145</span> in <span class="cit-source">The Bacteriophages</span>, <span class="cit-edition">Ed. 2</span>, Chap. 11., edited by <span class="cit-ed"><span class="cit-name-surname">Calendar</span>  <span class="cit-name-given-names">R.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Abedon</span>  <span class="cit-name-given-names">S. T.</span></span>. <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">Oxford</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.23" data-doi="10.1099/00221287-143-1-179"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hadas</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Einav</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fishov</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaritsky</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Bacteriophage T4 development depends on the physiology of its host <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Microbiology</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">179</span>–<span class="cit-lpage">185</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft.stitle%253DMicrobiology%26rft.aulast%253DHadas%26rft.auinit1%253DH.%26rft.volume%253D143%26rft.issue%253D1%26rft.spage%253D179%26rft.epage%253D185%26rft.atitle%253DBacteriophage%2BT4%2BDevelopment%2BDepends%2Bon%2Bthe%2BPhysiology%2Bof%2Bits%2BHost%2BEscherichia%2BColi%26rft_id%253Dinfo%253Adoi%252F10.1099%252F00221287-143-1-179%26rft_id%253Dinfo%253Apmid%252F9025292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/00221287-143-1-179&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9025292&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WE41100022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.24" data-doi="10.1038/nrg3540"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harms</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Evolutionary biochemistry: revealing the historical and physical causes of protein properties.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">571</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D14%26rft.spage%253D559%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3540%26rft_id%253Dinfo%253Apmid%252F23864121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3540&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23864121&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.939.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aoyama</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richardson</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">M. N.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Biology of the bacteriophage φX174</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">71</span> in <span class="cit-source">The Bacteriophages</span>, <span class="cit-vol">Vol. 2</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Calendar</span>  <span class="cit-name-given-names">R.</span></span>, <span class="cit-publ-name">Plenum Press</span>, <span class="cit-publ-loc">New York, NY.</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BBacteriophages%26rft.volume%253D2%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.26" data-doi="10.1534/genetics.107.072009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hubbarde</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wild</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wahl</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Fixation probabilities when generation times are variable: the burst-death model.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">1703</span>–<span class="cit-lpage">1712</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.072009%26rft_id%253Dinfo%253Apmid%252F17483420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni8zLzE3MDMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvOTM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.27" data-doi="10.1016/S0022-2836(63)80046-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hutchison</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinsheimer</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1963</span> <span class="cit-article-title">Kinetics of bacteriophage release by single cells of φX174-infected <em>E. coli</em>.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">206</span>–<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DHutchison%26rft.auinit1%253DC.%2BA.%26rft.volume%253D7%26rft.spage%253D206%26rft.epage%253D208%26rft.atitle%253DKINETICS%2BOF%2BBACTERIOPHAGE%2BRELEASE%2BBY%2BSINGLE%2BCELLS%2BOF%2BPHI%2BX174-INFECTED%2BE.%2BCOLI.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-2836%252863%252980046-7%26rft_id%253Dinfo%253Apmid%252F14062652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0022-2836(63)80046-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14062652&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.28" data-doi="10.1016/S0022-2836(66)80035-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hutchison</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinsheimer</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">The process of infection with bacteriophage φX174: X. Mutations in a φX lysis gene.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">429</span>–<span class="cit-lpage">447</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DHutchison%26rft.auinit1%253DC.%2BA.%26rft.volume%253D18%26rft.issue%253D3%26rft.spage%253D429%26rft.epage%253D447%26rft.atitle%253DThe%2Bprocess%2Bof%2Binfection%2Bwith%2Bbacteriophage%2Bphi-X174.%2BX.%2BMutations%2Bin%2Ba%2Bphi-X%2BLysis%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-2836%252866%252980035-9%26rft_id%253Dinfo%253Apmid%252F5968177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0022-2836(66)80035-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5968177&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A19668106300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.939.29"><div class="cit-metadata unstructured">Hyman, P., and S. T. Abedon, 2009 Practical methods for determining phage growth parameters, pp. 175–202 in <em>Bacteriophages: Methods and Protocols</em>, <em>Vol. 1: Isolation</em>, <em>Characterization</em>, <em>and Interactions</em>, Vol. 501 of <em>Methods in Molecular Biology</em>, edited by M. R. J. Clokie, and A. M. Kropinski. Humana Press, Clifton, New Jersey.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.30" data-doi="10.1128/AEM.01365-14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Labrie</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dupuis</span>  <span class="cit-name-given-names">M.-E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tremblay</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plante</span>  <span class="cit-name-given-names">P.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corbeil</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A new microviridae phage isolated from a failed biotechnological process driven by <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">6992</span>–<span class="cit-lpage">7000</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.01365-14%26rft_id%253Dinfo%253Apmid%252F25192988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI4MC8yMi82OTkyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80LzkzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.939.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source">Handbook of Biological Statistics</span>, <span class="cit-edition">Ed. 3</span>. <span class="cit-publ-name">Sparky House Publishing</span>, <span class="cit-publ-loc">Baltimore, MD</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.32" data-doi="10.1099/mic.0.28776-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mendel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holburn</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schouten</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bugg</span>  <span class="cit-name-given-names">T. D. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Interaction of the transmembrane domain of lysis protein E from bacteriophage φX174 with bacterial translocase MraY and peptidyl-prolyl isomerase SlyD.</span> <abbr class="cit-jnl-abbrev">Microbiology</abbr> <span class="cit-vol">152</span>: <span class="cit-fpage">2959</span>–<span class="cit-lpage">2967</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft.stitle%253DMicrobiology%26rft.aulast%253DMendel%26rft.auinit1%253DS.%26rft.volume%253D152%26rft.issue%253DPt%2B10%26rft.spage%253D2959%26rft.epage%253D2967%26rft.atitle%253DInteraction%2Bof%2Bthe%2Btransmembrane%2Bdomain%2Bof%2Blysis%2Bprotein%2BE%2Bfrom%2Bbacteriophage%2B%257Bphi%257DX174%2Bwith%2Bbacterial%2Btranslocase%2BMraY%2Band%2Bpeptidyl-prolyl%2Bisomerase%2BSlyD.%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fmic.0.28776-0%26rft_id%253Dinfo%253Apmid%252F17005977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/mic.0.28776-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17005977&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241234800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.33" data-doi="10.1093/biomet/80.2.267"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">X.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Maximum likelihood estimation via the ECM algorithm: a general framework.</span> <abbr class="cit-jnl-abbrev">Biometrika</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft.stitle%253DBiometrika%26rft.aulast%253DMENG%26rft.auinit1%253DX.-L.%26rft.volume%253D80%26rft.issue%253D2%26rft.spage%253D267%26rft.epage%253D278%26rft.atitle%253DMaximum%2Blikelihood%2Bestimation%2Bvia%2Bthe%2BECM%2Balgorithm%253A%2BA%2Bgeneral%2Bframework%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252F80.2.267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYmlvbWV0IjtzOjU6InJlc2lkIjtzOjg6IjgwLzIvMjY3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80LzkzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.34" data-doi="10.1128/AEM.02721-09"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clermont</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denamur</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tenaillon</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Bacteriophage φX174’s ecological niche and the flexibility of its <em>Escherichia coli</em> lipopolysaccharide receptor.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">7310</span>–<span class="cit-lpage">7313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.02721-09%26rft_id%253Dinfo%253Apmid%252F20833781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI3Ni8yMS83MzEwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80LzkzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.35" data-doi="10.1016/0022-2836(70)90375-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Newbold</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinsheimer</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Process of infection with bacteriophage φX174, XXXII. Early steps in the infection process: attachment, eclipse, and DNA penetration.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DNewbold%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D49%26rft.epage%253D66%26rft.atitle%253DThe%2Bprocess%2Bof%2Binfection%2Bwith%2Bbacteriophage%2BphiX174.%2BXXXII.%2BEarly%2Bsteps%2Bin%2Bthe%2Binfection%2Bprocess%253A%2Battachment%252C%2Beclipse%2Band%2BDNA%2Bpenetration.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252870%252990375-X%26rft_id%253Dinfo%253Apmid%252F5450517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(70)90375-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5450517&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1970G054500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.36" data-doi="10.1534/genetics.108.090555"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patwa</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wahl</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Fixation probability for lytic viruses: the attachment-lysis model.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">459</span>–<span class="cit-lpage">470</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.090555%26rft_id%253Dinfo%253Apmid%252F18757918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzEvNDU5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80LzkzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.37" data-doi="10.1016/j.jtbi.2009.05.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patwa</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wahl</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The impact of host-cell dynamics on the fixation probability for lytic viruses.</span> <abbr class="cit-jnl-abbrev">J. Theor. Biol.</abbr> <span class="cit-vol">259</span>: <span class="cit-fpage">799</span>–<span class="cit-lpage">810</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DPatwa%26rft.auinit1%253DZ.%26rft.volume%253D259%26rft.issue%253D4%26rft.spage%253D799%26rft.epage%253D810%26rft.atitle%253DThe%2Bimpact%2Bof%2Bhost-cell%2Bdynamics%2Bon%2Bthe%2Bfixation%2Bprobability%2Bfor%2Blytic%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jtbi.2009.05.008%26rft_id%253Dinfo%253Apmid%252F19464301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jtbi.2009.05.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19464301&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.38" data-doi="10.1006/geno.1997.4995"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pearson</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Comparison of DNA sequences with protein sequences.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DPearson%26rft.auinit1%253DW.%2BR.%26rft.volume%253D46%26rft.issue%253D1%26rft.spage%253D24%26rft.epage%253D36%26rft.atitle%253DComparison%2Bof%2BDNA%2Bsequences%2Bwith%2Bprotein%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fgeno.1997.4995%26rft_id%253Dinfo%253Apmid%252F9403055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/geno.1997.4995&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9403055&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997YK55200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.39" data-doi="10.1534/genetics.105.049817"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pepin</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samuel</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wichman</span>  <span class="cit-name-given-names">H. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Variable pleiotropic effects from mutations at the same locus hamper prediction of fitness from a fitness component.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">2047</span>–<span class="cit-lpage">2056</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.049817%26rft_id%253Dinfo%253Apmid%252F16361237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi80LzIwNDciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvOTM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rabinovitch</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hadas</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Einav</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melamed</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaritsky</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Model for bacteriophage T4 development in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">181</span>: <span class="cit-fpage">1677</span>–<span class="cit-lpage">1683</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DRabinovitch%26rft.auinit1%253DA.%26rft.volume%253D181%26rft.issue%253D5%26rft.spage%253D1677%26rft.epage%253D1683%26rft.atitle%253DModel%2Bfor%2BBacteriophage%2BT4%2BDevelopment%2Bin%2BEscherichia%2Bcoli%26rft_id%253Dinfo%253Apmid%252F10049403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE4MS81LzE2NzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvOTM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.41" data-doi="10.1006/jtbi.2002.2543"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rabinovitch</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fishov</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hadas</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Einav</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaritsky</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Bacteriophage T4 development in <em>Escherichia coli</em> is growth rate dependent.</span> <abbr class="cit-jnl-abbrev">J. Theor. Biol.</abbr> <span class="cit-vol">216</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DRabinovitch%26rft.auinit1%253DA.%26rft.volume%253D216%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D4%26rft.atitle%253DBacteriophage%2BT4%2Bdevelopment%2Bin%2BEscherichia%2Bcoli%2Bis%2Bgrowth%2Brate%2Bdependent.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjtbi.2002.2543%26rft_id%253Dinfo%253Apmid%252F12076123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jtbi.2002.2543&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12076123&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176634500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.939.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roff</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-source">Life History Evolution</span>, <span class="cit-publ-name">Sinauer</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.43" data-doi="10.1046/j.1365-2958.1997.5201884.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roof</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Mutational analysis of <em>SlyD</em>, an <em>Escherichia coli</em> gene encoding a protein of the FKBP immunophilin family.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1031</span>–<span class="cit-lpage">1046</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DRoof%26rft.auinit1%253DW.%2BD.%26rft.volume%253D25%26rft.issue%253D6%26rft.spage%253D1031%26rft.epage%253D1046%26rft.atitle%253DMutational%2Banalysis%2Bof%2BslyD%252C%2Ban%2BEscherichia%2Bcoli%2Bgene%2Bencoding%2Ba%2Bprotein%2Bof%2Bthe%2BFKBP%2Bimmunophilin%2Bfamily.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.1997.5201884.x%26rft_id%253Dinfo%253Apmid%252F9350861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.1997.5201884.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9350861&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997YA90700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.44" data-doi="10.1038/265687a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanger</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Air</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrell</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulson</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Nucleotide sequence of bacteriophage φX174 DNA.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">265</span>: <span class="cit-fpage">687</span>–<span class="cit-lpage">695</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSanger%26rft.auinit1%253DF.%26rft.volume%253D265%26rft.issue%253D5596%26rft.spage%253D687%26rft.epage%253D695%26rft.atitle%253DNucleotide%2Bsequence%2Bof%2Bbacteriophage%2Bphi%2BX174%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F265687a0%26rft_id%253Dinfo%253Apmid%252F870828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/265687a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=870828&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1977CW61200017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.45" data-doi="10.1534/genetics.108.090100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">I.-N.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Bacteriophage adsorption rate and optimal lysis time.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">471</span>–<span class="cit-lpage">482</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.090100%26rft_id%253Dinfo%253Apmid%252F18757924%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzEvNDcxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80LzkzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dennehy</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Stochastic holin expression can account for lysis time variation in the bacteriophage <em>λ</em>.</span> <abbr class="cit-jnl-abbrev">J. R. Soc. Interface</abbr> <span class="cit-vol">11</span>: 20140140.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.939.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stearns</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-source">The Evolution of Life Histories</span>, <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">Oxford</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.48" data-doi="10.1073/pnas.97.7.3309"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stearns</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ackermann</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doebeli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaiser</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Experimental evolution of aging, growth, and reproduction in fruitflies.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">3309</span>–<span class="cit-lpage">3313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DStearns%26rft.auinit1%253DS.%2BC.%26rft.volume%253D97%26rft.issue%253D7%26rft.spage%253D3309%26rft.epage%253D3313%26rft.atitle%253DExperimental%2Bevolution%2Bof%2Baging%252C%2Bgrowth%252C%2Band%2Breproduction%2Bin%2Bfruitflies%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.97.7.3309%26rft_id%253Dinfo%253Apmid%252F10716732%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5Ny83LzMzMDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvOTM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.49" data-doi="10.1371/journal.pone.0116235"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storms</span>  <span class="cit-name-given-names">Z. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sauvageau</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Evidence that the heterogeneity of a T4 population is the result of heritable traits.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e116235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De116235%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0116235%26rft_id%253Dinfo%253Apmid%252F25551763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0116235&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25551763&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.50" data-doi="10.1111/1574-6968.12402"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storms</span>  <span class="cit-name-given-names">Z. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sauvageau</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leask</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Impact of the cell life-cycle on bacteriophage T4 infection.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Lett.</abbr> <span class="cit-vol">353</span>: <span class="cit-fpage">63</span>–<span class="cit-lpage">68</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BLett.%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1574-6968.12402%26rft_id%253Dinfo%253Apmid%252F24822278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc2xlIjtzOjU6InJlc2lkIjtzOjg6IjM1My8xLzYzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80LzkzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.4.939.51" data-doi="10.1111/j.1365-2958.2012.08153.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clemons</span>  <span class="cit-name-given-names">W. M.</span>  <span class="cit-name-suffix">Jr</span>.</span></li></ol><cite>., <span class="cit-pub-date">2012</span> <span class="cit-article-title">Minimal requirements for inhibition of MraY by lysis protein E from bacteriophage φX174.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">975</span>–<span class="cit-lpage">985</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DTanaka%26rft.auinit1%253DS.%26rft.volume%253D85%26rft.issue%253D5%26rft.spage%253D975%26rft.epage%253D985%26rft.atitle%253DMinimal%2Brequirements%2Bfor%2Binhibition%2Bof%2BMraY%2Bby%2Blysis%2Bprotein%2BE%2Bfrom%2Bbacteriophage%2B%25CE%25A6X174.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2012.08153.x%26rft_id%253Dinfo%253Apmid%252F22742425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2012.08153.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22742425&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.52" data-doi="10.1038/ismej.2010.80"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Elsas</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Semenov</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trevors</span>  <span class="cit-name-given-names">J. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Survival of <em>Escherichia coli</em> in the environment: fundamental and public health aspects.</span> <abbr class="cit-jnl-abbrev">ISME J.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">173</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DISME%2BJ%26rft.aulast%253Dvan%2BElsas%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253D173%26rft.epage%253D183%26rft.atitle%253DSurvival%2Bof%2BEscherichia%2Bcoli%2Bin%2Bthe%2Benvironment%253A%2Bfundamental%2Band%2Bpublic%2Bhealth%2Baspects.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fismej.2010.80%26rft_id%253Dinfo%253Apmid%252F20574458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ismej.2010.80&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20574458&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000290020000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.53" data-doi="10.1534/genetics.104.029199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wahl</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeHaan</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Fixation probability favors increased fecundity over reduced generation time.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">1009</span>–<span class="cit-lpage">1018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWahl%26rft.auinit1%253DL.%2BM.%26rft.volume%253D168%26rft.issue%253D2%26rft.spage%253D1009%26rft.epage%253D1018%26rft.atitle%253DFixation%2BProbability%2BFavors%2BIncreased%2BFecundity%2BOver%2BReduced%2BGeneration%2BTime%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.029199%26rft_id%253Dinfo%253Apmid%252F15514071%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OC8yLzEwMDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvOTM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.54" data-doi="10.1534/genetics.105.045922"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">I.-N.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Lysis timing and bacteriophage fitness.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">17</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.045922%26rft_id%253Dinfo%253Apmid%252F16219778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTcyLzEvMTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzQvOTM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.55" data-doi="10.1007/BF01237884"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">I.-N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dykhuizen</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slobodkin</span>  <span class="cit-name-given-names">L. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The evolution of phage lysis timing.</span> <abbr class="cit-jnl-abbrev">Evol. Ecol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">545</span>–<span class="cit-lpage">558</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BEcol.%26rft.volume%253D10%26rft.spage%253D545%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01237884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01237884&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996VJ62700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.56" data-doi="10.1098/rstb.2010.0053"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wichman</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Experimental evolution of viruses: <em>Microviridae</em> as a model system.</span> <abbr class="cit-jnl-abbrev">Phil. Trans. R. Soc. B</abbr> <span class="cit-vol">365</span>: <span class="cit-fpage">2495</span>–<span class="cit-lpage">2501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DWichman%26rft.auinit1%253DH.%2BA.%26rft.volume%253D365%26rft.issue%253D1552%26rft.spage%253D2495%26rft.epage%253D2501%26rft.atitle%253DExperimental%2Bevolution%2Bof%2Bviruses%253A%2BMicroviridae%2Bas%2Ba%2Bmodel%2Bsystem%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2010.0053%26rft_id%253Dinfo%253Apmid%252F20643739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEzOiIzNjUvMTU1Mi8yNDk1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi80LzkzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.57" data-doi="10.1007/s002030050641"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Witte</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brand</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mayrhofer</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Narendja</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Werner</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Mutations in cell division proteins FtsZ and FtsA inhibit φX174 protein-E-mediated lysis of <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Arch. Microbiol.</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">259</span>–<span class="cit-lpage">268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArchives%2Bof%2BMicrobiology%26rft.stitle%253DArchives%2Bof%2BMicrobiology%26rft.aulast%253DWitte%26rft.auinit1%253DA.%26rft.volume%253D170%26rft.issue%253D4%26rft.spage%253D259%26rft.epage%253D268%26rft.atitle%253DMutations%2Bin%2Bcell%2Bdivision%2Bproteins%2BFtsZ%2Band%2BFtsA%2Binhibit%2BphiX174%2Bprotein-E-mediated%2Blysis%2Bof%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs002030050641%26rft_id%253Dinfo%253Apmid%252F9732440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s002030050641&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9732440&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000076123800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.58" data-doi="10.1016/j.copbio.2011.11.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Microfluidics for single cell analysis.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Biotechnol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">110</span>–<span class="cit-lpage">119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2BOpinion%2Bin%2BBiotechnology%26rft.stitle%253DCurrent%2BOpinion%2Bin%2BBiotechnology%26rft.aulast%253DYin%26rft.auinit1%253DH.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D110%26rft.epage%253D119%26rft.atitle%253DMicrofluidics%2Bfor%2Bsingle%2Bcell%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.copbio.2011.11.002%26rft_id%253Dinfo%253Apmid%252F22133547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.copbio.2011.11.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22133547&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-book" id="cit-6.4.939.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">I.-N.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Phage lysis</span>, pp. <span class="cit-fpage">104</span>–<span class="cit-lpage">125</span> in <span class="cit-source">The Bacteriophages</span>, <span class="cit-edition">Ed. 2</span>. chap 10., edited by <span class="cit-ed"><span class="cit-name-surname">Calendar</span>  <span class="cit-name-given-names">R.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Abedon</span>  <span class="cit-name-given-names">S. T.</span></span>. <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">Oxford</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.60" data-doi="10.1016/S0966-842X(00)01705-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">I.-N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roof</span>  <span class="cit-name-given-names">W. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Phages will out: strategies of host cell lysis.</span> <abbr class="cit-jnl-abbrev">Trends Microbiol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">120</span>–<span class="cit-lpage">128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bmicrobiology%26rft.stitle%253DTrends%2BMicrobiol%26rft.aulast%253DYoung%26rft.auinit1%253DI.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D120%26rft.epage%253D128%26rft.atitle%253DPhages%2Bwill%2Bout%253A%2Bstrategies%2Bof%2Bhost%2Bcell%2Blysis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0966-842X%252800%252901705-4%26rft_id%253Dinfo%253Apmid%252F10707065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0966-842X(00)01705-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10707065&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085789800020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.61" data-doi="10.1099/mic.0.2008/016956-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Struck</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dankenbring</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Evolutionary dominance of holin lysis systems derives from superior genetic malleability.</span> <abbr class="cit-jnl-abbrev">Microbiology</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">1710</span>–<span class="cit-lpage">1718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft.stitle%253DMicrobiology%26rft.aulast%253DZheng%26rft.auinit1%253DY.%26rft.volume%253D154%26rft.issue%253D6%26rft.spage%253D1710%26rft.epage%253D1718%26rft.atitle%253DEvolutionary%2Bdominance%2Bof%2Bholin%2Blysis%2Bsystems%2Bderives%2Bfrom%2Bsuperior%2Bgenetic%2Bmalleability%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fmic.0.2008%252F016956-0%26rft_id%253Dinfo%253Apmid%252F18524925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/mic.0.2008/016956-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18524925&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257171000015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.939.62" data-doi="10.1021/bi900469g"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Struck</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Purification and functional characterization of φX174 lysis protein E.</span> <abbr class="cit-jnl-abbrev">Biochemistry</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">4999</span>–<span class="cit-lpage">5006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBiochemistry%26rft.aulast%253DZheng%26rft.auinit1%253DY.%26rft.volume%253D48%26rft.issue%253D22%26rft.spage%253D4999%26rft.epage%253D5006%26rft.atitle%253DPurification%2Band%2Bfunctional%2Bcharacterization%2Bof%2BphiX174%2Blysis%2Bprotein%2BE.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi900469g%26rft_id%253Dinfo%253Apmid%252F19379010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi900469g&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19379010&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F939.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/4/939.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/4/925" title="Sexual Polyploidization in Medicago sativa L.: Impact on the Phenotype, Gene Transcription, and Genome Methylation" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/4/957" title="Whole Genome Comparison Reveals High Levels of Inbreeding and Strain Redundancy Across the Spectrum of Commercial Wine Strains of Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/4"><span>Volume 6 Issue 4, April 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/4.cover-source.jpg?itok=Lj45nPaf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/939" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CJ5jCKOlwoYaAZE5XvH6JtZsgVLFcWHhdotIiROT54w" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/4/939" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/4/939" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/4/939" class="active" data-icon-position="" data-hide-link-title="0">Genetically Determined Variation in Lysis Time Variance in the Bacteriophage φX174</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460131" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-MsuxUeLLHkQYGka8cfiPr_-lePxzogFwVRb7HUPOHbY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/4/939.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/4/939" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ktgvLcloM6F9Zb4-6YzcVWRdsWOABjUpod8dLIgE-ik" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/4/939" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/4/939.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460131" id="node460131--2" data-pisa="ggg;6/4/939" data-pisa-master="ggg;g3.115.024075" data-apath="/ggg/6/4/939.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetically Determined Variation in Lysis Time Variance in the Bacteriophage φX174</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0819-882X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christopher W. Baker</span>, <span class="highwire-citation-author" data-delta="1">Craig R. Miller</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-0613-4176" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Tanayott Thaweethai</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-3855-1994" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jeffrey Yuan</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-3226-0811" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Meghan Hollibaugh Baker</span>, <span class="highwire-citation-author" data-delta="5">Paul Joyce</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-1424-7541" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel M. Weinreich</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">939-955;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024075 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460131--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Christopher W. Baker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, Brown University, Providence, Rhode Island 02912</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BW.%2BBaker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baker%20CW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristopher%2BW.%2BBaker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0819-882X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christopher W. Baker</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:baker03@g.harvard.edu" class="" data-icon-position="" data-hide-link-title="0">baker03@g.harvard.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Craig R. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematics, University of Idaho, Moscow, Idaho 83844</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biological Sciences, University of Idaho, Moscow, Idaho 83844</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Center for Modeling Complex Interactions, University of Idaho, Moscow, Idaho 83844</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Craig%2BR.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACraig%2BR.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Tanayott Thaweethai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, Brown University, Providence, Rhode Island 02912</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tanayott%2BThaweethai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thaweethai%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATanayott%2BThaweethai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0613-4176" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Tanayott Thaweethai</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jeffrey Yuan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, Brown University, Providence, Rhode Island 02912</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BYuan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yuan%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeffrey%2BYuan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3855-1994" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jeffrey Yuan</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Meghan Hollibaugh Baker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, Brown University, Providence, Rhode Island 02912</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Meghan%2BHollibaugh%2BBaker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baker%20MH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMeghan%2BHollibaugh%2BBaker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3226-0811" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Meghan Hollibaugh Baker</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Paul Joyce </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematics, University of Idaho, Moscow, Idaho 83844</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BJoyce%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Joyce%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaul%2BJoyce%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Daniel M. Weinreich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, Brown University, Providence, Rhode Island 02912</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Center for Computational Molecular Biology, Brown University, Providence, Rhode Island 02912</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BM.%2BWeinreich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weinreich%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BM.%2BWeinreich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1424-7541" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel M. Weinreich</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/4/939" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/4/939&amp;title=Genetically%20Determined%20Variation%20in%20Lysis%20Time%20Variance%20in%20the%20Bacteriophage%20%CF%86X174" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/4/939&amp;title=Genetically%20Determined%20Variation%20in%20Lysis%20Time%20Variance%20in%20the%20Bacteriophage%20%CF%86X174" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/4/939&amp;title=Genetically%20Determined%20Variation%20in%20Lysis%20Time%20Variance%20in%20the%20Bacteriophage%20%CF%86X174" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/4/939&amp;text=Genetically%20Determined%20Variation%20in%20Lysis%20Time%20Variance%20in%20the%20Bacteriophage%20%CF%86X174" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/4/939&amp;title=Genetically%20Determined%20Variation%20in%20Lysis%20Time%20Variance%20in%20the%20Bacteriophage%20%CF%86X174" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/4/939&amp;t=Genetically%20Determined%20Variation%20in%20Lysis%20Time%20Variance%20in%20the%20Bacteriophage%20%CF%86X174" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/4/939&amp;title=Genetically%20Determined%20Variation%20in%20Lysis%20Time%20Variance%20in%20the%20Bacteriophage%20%CF%86X174" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/4/939&amp;title=Genetically%20Determined%20Variation%20in%20Lysis%20Time%20Variance%20in%20the%20Bacteriophage%20%CF%86X174" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460131/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460131" id="node460131--3" data-pisa="ggg;6/4/939" data-pisa-master="ggg;g3.115.024075" data-apath="/ggg/6/4/939.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetically Determined Variation in Lysis Time Variance in the Bacteriophage φX174</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0819-882X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christopher W. Baker</span>, <span class="highwire-citation-author" data-delta="1">Craig R. Miller</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-0613-4176" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Tanayott Thaweethai</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-3855-1994" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jeffrey Yuan</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-3226-0811" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Meghan Hollibaugh Baker</span>, <span class="highwire-citation-author" data-delta="5">Paul Joyce</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-1424-7541" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel M. Weinreich</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">939-955;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024075 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460131--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Christopher W. Baker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, Brown University, Providence, Rhode Island 02912</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BW.%2BBaker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baker%20CW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristopher%2BW.%2BBaker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0819-882X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christopher W. Baker</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:baker03@g.harvard.edu" class="" data-icon-position="" data-hide-link-title="0">baker03@g.harvard.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Craig R. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematics, University of Idaho, Moscow, Idaho 83844</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biological Sciences, University of Idaho, Moscow, Idaho 83844</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Center for Modeling Complex Interactions, University of Idaho, Moscow, Idaho 83844</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Craig%2BR.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACraig%2BR.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Tanayott Thaweethai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, Brown University, Providence, Rhode Island 02912</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tanayott%2BThaweethai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thaweethai%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATanayott%2BThaweethai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0613-4176" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Tanayott Thaweethai</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jeffrey Yuan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, Brown University, Providence, Rhode Island 02912</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BYuan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yuan%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeffrey%2BYuan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3855-1994" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jeffrey Yuan</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Meghan Hollibaugh Baker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, Brown University, Providence, Rhode Island 02912</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Meghan%2BHollibaugh%2BBaker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baker%20MH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMeghan%2BHollibaugh%2BBaker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3226-0811" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Meghan Hollibaugh Baker</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Paul Joyce </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematics, University of Idaho, Moscow, Idaho 83844</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BJoyce%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Joyce%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaul%2BJoyce%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Daniel M. Weinreich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, Brown University, Providence, Rhode Island 02912</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Center for Computational Molecular Biology, Brown University, Providence, Rhode Island 02912</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BM.%2BWeinreich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weinreich%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BM.%2BWeinreich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1424-7541" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel M. Weinreich</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=939&amp;rft.epage=955&amp;rft.atitle=Genetically%20Determined%20Variation%20in%20Lysis%20Time%20Variance%20in%20the%20Bacteriophage%20%CF%86X174&amp;rft.volume=6&amp;rft.issue=4&amp;rft.date=2016-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Baker%2C+Christopher+W.&amp;rft.au=Miller%2C+Craig+R.&amp;rft.au=Thaweethai%2C+Tanayott&amp;rft.au=Yuan%2C+Jeffrey&amp;rft.au=Baker%2C+Meghan+Hollibaugh&amp;rft.au=Joyce%2C+Paul&amp;rft.au=Weinreich%2C+Daniel+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460131/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460131/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460131/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460131/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460131/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460131/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460131/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460131/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460131/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460131/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460131/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460131/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/4/939" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix 1: Number of Phages per Well Poisson Estimator</a></li><li class="odd"><a href="#app-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix 2: Time to Lysis</a></li><li class="even"><a href="#app-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix 3: Burst Size as a Function of Size</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/4/939.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/4/939.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/4/939.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/4/939.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
