<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/4/1063" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/4/1063.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/4/1063.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/4/1063.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/4/1063.atom" />
<meta name="HW.pisa" content="ggg;6/4/1063" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Large-Scale Survey of Intraspecific Fitness and Cell Morphology Variation in a Protoploid Yeast Species" />
<meta name="DC.Identifier" content="10.1534/g3.115.026682" />
<meta name="DC.Date" content="2016-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Jung et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="It is now clear that the exploration of the genetic and phenotypic diversity of nonmodel species greatly improves our knowledge in biology. In this context, we recently launched a population genomic analysis of the protoploid yeast Lachancea kluyveri (formerly Saccharomyces kluyveri ), highlighting a broad genetic diversity (π = 17 × 10−3) compared to the yeast model organism, S. cerevisiae (π = 4 × 10−3). Here, we sought to generate a comprehensive view of the phenotypic diversity in this species. In total, 27 natural L. kluyveri isolates were subjected to trait profiling using the following independent approaches: (i) analyzing growth in 55 growth conditions and (ii) investigating 501 morphological changes at the cellular level. Despite higher genetic diversity, the fitness variance observed in L. kluyveri is lower than that in S. cerevisiae . However, morphological features show an opposite trend. In addition, there is no correlation between the origins (ecological or geographical) of the isolate and the phenotypic patterns, demonstrating that trait variation follows neither population history nor source environment in L. kluyveri . Finally, pairwise comparisons between growth rate correlation and genetic diversity show a clear decrease in phenotypic variability linked to genome variation increase, whereas no such a trend was identified for morphological changes. Overall, this study reveals for the first time the phenotypic diversity of a distantly related species to S. cerevisiae . Given its genetic properties, L. kluyveri might be useful in further linkage mapping analyses of complex traits, and could ultimately provide a better insight into the evolution of the genotype–phenotype relationship across yeast species." />
<meta name="DC.Contributor" content="Paul P. Jung" />
<meta name="DC.Contributor" content="Anastasie Sigwalt" />
<meta name="DC.Contributor" content="Shinsuke Ohnuki" />
<meta name="DC.Contributor" content="Jacky de Montigny" />
<meta name="DC.Contributor" content="Yoshikazu Ohya" />
<meta name="DC.Contributor" content="Joseph Schacherer" />
<meta name="article:published_time" content="2016-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Large-Scale Survey of Intraspecific Fitness and Cell Morphology Variation in a Protoploid Yeast Species" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;It is now clear that the exploration of the genetic and phenotypic diversity of nonmodel species greatly improves our knowledge in biology. In this context, we recently launched a population genomic analysis of the protoploid yeast &lt;i&gt;Lachancea kluyveri&lt;/i&gt; (formerly &lt;i&gt;Saccharomyces kluyveri&lt;/i&gt;), highlighting a broad genetic diversity (π = 17 × 10&lt;sup&gt;−3&lt;/sup&gt;) compared to the yeast model organism, &lt;i&gt;S. cerevisiae&lt;/i&gt; (π = 4 × 10&lt;sup&gt;−3&lt;/sup&gt;). Here, we sought to generate a comprehensive view of the phenotypic diversity in this species. In total, 27 natural &lt;i&gt;L. kluyveri&lt;/i&gt; isolates were subjected to trait profiling using the following independent approaches: (i) analyzing growth in 55 growth conditions and (ii) investigating 501 morphological changes at the cellular level. Despite higher genetic diversity, the fitness variance observed in &lt;i&gt;L. kluyveri&lt;/i&gt; is lower than that in &lt;i&gt;S. cerevisiae&lt;/i&gt;. However, morphological features show an opposite trend. In addition, there is no correlation between the origins (ecological or geographical) of the isolate and the phenotypic patterns, demonstrating that trait variation follows neither population history nor source environment in &lt;i&gt;L. kluyveri&lt;/i&gt;. Finally, pairwise comparisons between growth rate correlation and genetic diversity show a clear decrease in phenotypic variability linked to genome variation increase, whereas no such a trend was identified for morphological changes. Overall, this study reveals for the first time the phenotypic diversity of a distantly related species to &lt;i&gt;S. cerevisiae&lt;/i&gt;. Given its genetic properties, &lt;i&gt;L. kluyveri&lt;/i&gt; might be useful in further linkage mapping analyses of complex traits, and could ultimately provide a better insight into the evolution of the genotype–phenotype relationship across yeast species.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/04/01" />
<meta name="citation_mjid" content="ggg;6/4/1063" />
<meta name="citation_id" content="6/4/1063" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/4/1063" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/4/1063.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/4/1063.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/4/1063.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.026682" />
<meta name="citation_pmid" content="26888866" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1063" />
<meta name="citation_lastpage" content="1071" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Paul P. Jung" />
<meta name="citation_author_institution" content="Department of Genetics, Genomics and Microbiology, University of Strasbourg - Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, 67083 Strasbourg, France" />
<meta name="citation_author" content="Anastasie Sigwalt" />
<meta name="citation_author_institution" content="Department of Genetics, Genomics and Microbiology, University of Strasbourg - Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, 67083 Strasbourg, France" />
<meta name="citation_author" content="Shinsuke Ohnuki" />
<meta name="citation_author_institution" content="Department of Integrated Biosciences, Graduate School of Frontier Sciences, University of Tokyo, Kashiwa 277-8562, Chiba, Japan" />
<meta name="citation_author" content="Jacky de Montigny" />
<meta name="citation_author_institution" content="Department of Genetics, Genomics and Microbiology, University of Strasbourg - Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, 67083 Strasbourg, France" />
<meta name="citation_author" content="Yoshikazu Ohya" />
<meta name="citation_author_institution" content="Department of Integrated Biosciences, Graduate School of Frontier Sciences, University of Tokyo, Kashiwa 277-8562, Chiba, Japan" />
<meta name="citation_author_email" content="schacherer@unistra.fr" />
<meta name="citation_author_email" content="ohya@k.u-tokyo.ac.jp" />
<meta name="citation_author" content="Joseph Schacherer" />
<meta name="citation_author_institution" content="Department of Genetics, Genomics and Microbiology, University of Strasbourg - Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, 67083 Strasbourg, France" />
<meta name="citation_author_email" content="schacherer@unistra.fr" />
<meta name="citation_author_email" content="ohya@k.u-tokyo.ac.jp" />
<meta name="citation_reference" content="citation_journal_title=Nucleosides, nucleotides &amp; nucleic acids;citation_journal_abbrev=Nucleosides Nucleotides Nucleic Acids;citation_author=G. Andersen;citation_author=A. Merico;citation_author=O. Bjornberg;citation_author=B. Andersen;citation_author=K. D. Schnackerz;citation_title=Catabolism of pyrimidines in yeast: a tool to understand degradation of anticancer drugs.;citation_pages=991-996;citation_volume=25;citation_year=2006;citation_issue=9–11;citation_pmid=17065052;citation_doi=10.1080/15257770600889386" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=H. Beck;citation_author=D. Dobritzsch;citation_author=J. Piskur;citation_title=Saccharomyces kluyveri as a model organism to study pyrimidine degradation.;citation_volume=8;citation_year=2008;citation_issue=8;citation_pmid=18823354;citation_doi=10.1111/j.1567-1364.2008.00442.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Bergstrom;citation_author=J. T. Simpson;citation_author=F. Salinas;citation_author=B. Barre;citation_author=L. Parts;citation_title=A high-definition view of functional genetic variation from natural yeast genomes.;citation_volume=31;citation_year=2014;citation_issue=4;citation_pmid=24425782;citation_doi=10.1093/molbev/msu037" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=C. Boone;citation_author=H. Bussey;citation_author=B. J. Andrews;citation_title=Exploring genetic interactions and networks with yeast.;citation_pages=437-449;citation_volume=8;citation_year=2007;citation_issue=6;citation_pmid=17510664;citation_doi=10.1038/nrg2085" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=C. Brion;citation_author=D. Pflieger;citation_author=A. Friedrich;citation_author=J. Schacherer;citation_title=Evolution of intraspecific transcriptomic landscapes in yeasts.;citation_volume=43;citation_year=2015;citation_issue=9;citation_pmid=25897111;citation_doi=10.1093/nar/gkv363" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=W. R. Brown;citation_author=G. Liti;citation_author=C. Rosa;citation_author=S. James;citation_author=I. Roberts;citation_title=A Geographically Diverse Collection of Schizosaccharomyces pombe Isolates Shows Limited Phenotypic Variation but Extensive Karyotypic Diversity;citation_pages=615-626;citation_volume=1;citation_year=2011;citation_issue=7;citation_pmid=22384373;citation_doi=10.1534/g3.111.001123" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=G. A. Cromie;citation_author=K. E. Hyma;citation_author=C. L. Ludlow;citation_author=C. Garmendia-Torres;citation_author=T. L. Gilbert;citation_title=Genomic sequence diversity and population structure of Saccharomyces cerevisiae assessed by RAD-seq.;citation_volume=3;citation_year=2013;citation_issue=12;citation_pmid=24122055;citation_doi=10.1534/g3.113.007492" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=J. C. Dittmar;citation_author=R. J. Reid;citation_author=R. Rothstein;citation_title=ScreenMill: a freely available software suite for growth measurement, analysis and visualization of high-throughput screen data.;citation_pages=353-353;citation_volume=11;citation_year=2010;citation_pmid=20584323;citation_doi=10.1186/1471-2105-11-353" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=K. C. Freel;citation_author=A. Friedrich;citation_author=J. Hou;citation_author=J. Schacherer;citation_title=Population genomic analysis reveals highly conserved mitochondrial genomes in the yeast species Lachancea thermotolerans.;citation_volume=6;citation_year=2014;citation_issue=10;citation_pmid=25212859;citation_doi=10.1093/gbe/evu203" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_title=Mitochondrial genome evolution in yeasts: an all-encompassing view.;citation_author=K. C. Freel;citation_author=A. Friedrich;citation_author=J. Schacherer;citation_volume=15;citation_year=2015;citation_issue=4" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_title=Whole-genome sequencing and intraspecific analysis of the yeast species Lachancea quebecensis.;citation_author=K. C. Freel;citation_author=A. Friedrich;citation_author=V. Sarilar;citation_author=H. Devillers;citation_author=C. Neuveglise;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Friedrich;citation_author=P. Jung;citation_author=C. Reisser;citation_author=G. Fischer;citation_author=J. Schacherer;citation_title=Population genomics reveals chromosome-scale heterogeneous evolution in a protoploid yeast.;citation_volume=32;citation_year=2015;citation_issue=1;citation_pmid=25349286;citation_doi=10.1093/molbev/msu295" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=C. Genolevures;citation_author=J. L. Souciet;citation_author=B. Dujon;citation_author=C. Gaillardin;citation_author=M. Johnston;citation_title=Comparative genomics of protoploid Saccharomycetaceae.;citation_volume=19;citation_year=2009;citation_issue=10;citation_pmid=19525356;citation_doi=10.1101/gr.091546.109" />
<meta name="citation_reference" content="citation_journal_title=Advances in experimental medicine and biology;citation_journal_abbrev=Adv Exp Med Biol;citation_author=Z. Gojkovic;citation_author=S. Paracchini;citation_author=J. Piskur;citation_title=A new model organism for studying the catabolism of pyrimidines and purines.;citation_pages=475-479;citation_volume=431;citation_year=1998;citation_pmid=9598114;citation_doi=10.1007/978-1-4615-5381-6_94" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. Halfmann;citation_author=D. F. Jarosz;citation_author=S. K. Jones;citation_author=A. Chang;citation_author=A. K. Lancaster;citation_title=Prions are a common mechanism for phenotypic inheritance in wild yeasts.;citation_pages=363-368;citation_volume=482;citation_year=2012;citation_issue=7385;citation_pmid=22337056;citation_doi=10.1038/nature10875" />
<meta name="citation_reference" content="citation_journal_title=Molecular &amp; general genetics :  MGG;citation_journal_abbrev=Mol Gen Genet;citation_author=L. Jaquet;citation_author=M. Lollier;citation_author=J. L. Souciet;citation_author=S. Potier;citation_title=Genetic analysis of yeast strains lacking negative feedback control: a one-step method for positive selection and cloning of carbamoylphosphate synthetase-aspartate transcarbamoylase mutants unable to respond to UTP.;citation_pages=81-88;citation_volume=241;citation_year=1993;citation_issue=1–2;citation_pmid=8232215;citation_doi=10.1007/BF00280204" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=P. P. Jung;citation_author=A. Friedrich;citation_author=C. Reisser;citation_author=J. Hou;citation_author=J. Schacherer;citation_title=Mitochondrial Genome Evolution in a Single Protoploid Yeast Species;citation_pages=1103-1111;citation_volume=2;citation_year=2012;citation_issue=9;citation_pmid=22973548;citation_doi=10.1534/g3.112.003152" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=P. P. Jung;citation_author=N. Christian;citation_author=D. P. Kay;citation_author=A. Skupin;citation_author=C. L. Linster;citation_title=Protocols and programs for high-throughput growth and aging phenotyping in yeast.;citation_pages=e0119807;citation_volume=10;citation_year=2015;citation_issue=3;citation_pmid=25822370;citation_doi=10.1371/journal.pone.0119807" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Kellis;citation_author=B. W. Birren;citation_author=E. S. Lander;citation_title=Proof and evolutionary analysis of ancient genome duplication in the yeast Saccharomyces cerevisiae.;citation_pages=617-624;citation_volume=428;citation_year=2004;citation_issue=6983;citation_pmid=15004568;citation_doi=10.1038/nature02424" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=D. J. Kvitek;citation_author=J. L. Will;citation_author=A. P. Gasch;citation_title=Variations in stress sensitivity and genomic expression in diverse S. cerevisiae isolates.;citation_pages=e1000223-e1000223;citation_volume=4;citation_year=2008;citation_issue=10;citation_pmid=18927628;citation_doi=10.1371/journal.pgen.1000223" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Liti;citation_author=D. M. Carter;citation_author=A. M. Moses;citation_author=J. Warringer;citation_author=L. Parts;citation_title=Population genomics of domestic and wild yeasts.;citation_pages=337-341;citation_volume=458;citation_year=2009;citation_issue=7236;citation_pmid=19212322;citation_doi=10.1038/nature07743" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=J. McCullough;citation_author=I. Herskowitz;citation_title=Mating Pheromones of Saccharomyces kluyveri: Pheromone Interactions Between Saccharomyces kluyveri and Saccharomyces cerevisiae;citation_pages=146-154;citation_volume=138;citation_year=1979;citation_issue=1;citation_pmid=374360" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=K. Moller;citation_author=R. B. Langkjaer;citation_author=J. Nielsen;citation_author=J. Piskur;citation_author=L. Olsson;citation_title=Pyruvate decarboxylases from the petite-negative yeast Saccharomyces kluyveri.;citation_pages=558-568;citation_volume=270;citation_year=2004;citation_issue=6;citation_pmid=14648197;citation_doi=10.1007/s00438-003-0950-z" />
<meta name="citation_reference" content="citation_journal_title=J Biosci Bioeng;citation_author=S. Ohnuki;citation_author=K. Enomoto;citation_author=H. Yoshimoto;citation_author=Y. Ohya;citation_title=Dynamic changes in brewing yeast cells in culture revealed by statistical analyses of yeast morphological data.;citation_pages=278-284;citation_volume=117;citation_year=2014;citation_issue=3;citation_pmid=24012106;citation_doi=10.1016/j.jbiosc.2013.08.005" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. Ohya;citation_author=J. Sese;citation_author=M. Yukawa;citation_author=F. Sano;citation_author=Y. Nakatani;citation_title=High-dimensional and large-scale phenotyping of yeast mutants.;citation_volume=102;citation_year=2005;citation_issue=52;citation_pmid=16365294;citation_doi=10.1073/pnas.0509436102" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology of the Cell;citation_journal_abbrev=Mol. Biol. Cell;citation_author=Y. Ohya;citation_author=Y. Kimori;citation_author=H. Okada;citation_author=S. Ohnuki;citation_title=Single-cell phenomics in budding yeast;citation_pages=3920-3925;citation_volume=26;citation_year=2015;citation_issue=22;citation_pmid=26543200;citation_doi=10.1091/mbc.E15-07-0466" />
<meta name="citation_reference" content="citation_journal_title=Yeast;citation_author=J. Peter;citation_author=J. Schacherer;citation_title=Population genomics of yeasts: Towards a comprehensive view across a broad evolutionary scale.;citation_pages=73-81;citation_volume=233;citation_year=2016;citation_issue=3" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Schacherer;citation_author=J. A. Shapiro;citation_author=D. M. Ruderfer;citation_author=L. Kruglyak;citation_title=Comprehensive polymorphism survey elucidates population structure of Saccharomyces cerevisiae.;citation_pages=342-345;citation_volume=458;citation_year=2009;citation_issue=7236;citation_pmid=19212320;citation_doi=10.1038/nature07670" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. M. Selmecki;citation_author=K. Dulmage;citation_author=L. E. Cowen;citation_author=J. B. Anderson;citation_author=J. Berman;citation_title=Acquisition of aneuploidy provides increased fitness during the evolution of antifungal drug resistance.;citation_pages=e1000705-e1000705;citation_volume=5;citation_year=2009;citation_issue=10;citation_pmid=19876375;citation_doi=10.1371/journal.pgen.1000705" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. A. Skelly;citation_author=G. E. Merrihew;citation_author=M. Riffle;citation_author=C. F. Connelly;citation_author=E. O. Kerr;citation_title=Integrative phenomics reveals insight into the structure of phenotypic diversity in budding yeast.;citation_volume=23;citation_year=2013;citation_issue=9;citation_pmid=23720455;citation_doi=10.1101/gr.155762.113" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=J. Warringer;citation_author=A. Blomberg;citation_title=Automated screening in environmental arrays allows analysis of quantitative phenotypic profiles in Saccharomyces cerevisiae.;citation_pages=53-67;citation_volume=20;citation_year=2003;citation_issue=1;citation_pmid=12489126;citation_doi=10.1002/yea.931" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. Warringer;citation_author=E. Zorgo;citation_author=F. A. Cubillos;citation_author=A. Zia;citation_author=A. Gjuvsland;citation_title=Trait variation in yeast is defined by population history.;citation_pages=e1002111-e1002111;citation_volume=7;citation_year=2011;citation_issue=6;citation_pmid=21698134;citation_doi=10.1371/journal.pgen.1002111" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=M. Yang;citation_author=S. Ohnuki;citation_author=Y. Ohya;citation_title=Unveiling nonessential gene deletions that confer significant morphological phenotypes beyond natural yeast strains.;citation_pages=932;citation_volume=15;citation_year=2014;citation_pmid=25344683;citation_doi=10.1186/1471-2164-15-932" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. H. Yona;citation_author=Y. S. Manor;citation_author=R. H. Herbst;citation_author=G. H. Romano;citation_author=A. Mitchell;citation_title=Chromosomal duplication is a transient evolutionary solution to stress.;citation_volume=109;citation_year=2012;citation_issue=51;citation_pmid=23197825;citation_doi=10.1073/pnas.1211150109" />
<meta name="citation_reference" content="citation_journal_title=BMC Syst. Biol.;citation_author=G. Yvert;citation_author=S. Ohnuki;citation_author=S. Nogami;citation_author=Y. Imanaga;citation_author=S. Fehrmann;citation_title=Single-cell phenomics reveals intra-species variation of phenotypic noise in yeast.;citation_pages=54;citation_volume=7;citation_year=2013;citation_pmid=23822767;citation_doi=10.1186/1752-0509-7-54" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Large-Scale Survey of Intraspecific Fitness and Cell Morphology Variation in a Protoploid Yeast Species" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
<meta name="twitter:description" content="It is now clear that the exploration of the genetic and phenotypic diversity of nonmodel species greatly improves our knowledge in biology. In this context, we recently launched a population genomic analysis of the protoploid yeast Lachancea kluyveri (formerly Saccharomyces kluyveri ), highlighting a broad genetic diversity (π = 17 × 10−3) compared to the yeast model organism, S. cerevisiae (π = 4 × 10−3). Here, we sought to generate a comprehensive view of the phenotypic diversity in this species. In total, 27 natural L. kluyveri isolates were subjected to trait profiling using the following independent approaches: (i) analyzing growth in 55 growth conditions and (ii) investigating 501 morphological changes at the cellular level. Despite higher genetic diversity, the fitness variance observed in L. kluyveri is lower than that in S. cerevisiae . However, morphological features show an opposite trend. In addition, there is no correlation between the origins (ecological or geographical) of the isolate and the phenotypic patterns, demonstrating that trait variation follows neither population history nor source environment in L. kluyveri . Finally, pairwise comparisons between growth rate correlation and genetic diversity show a clear decrease in phenotypic variability linked to genome variation increase, whereas no such a trend was identified for morphological changes. Overall, this study reveals for the first time the phenotypic diversity of a distantly related species to S. cerevisiae . Given its genetic properties, L. kluyveri might be useful in further linkage mapping analyses of complex traits, and could ultimately provide a better insight into the evolution of the genotype–phenotype relationship across yeast species." />
<meta name="og-title" property="og:title" content="Large-Scale Survey of Intraspecific Fitness and Cell Morphology Variation in a Protoploid Yeast Species" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/4/1063" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="It is now clear that the exploration of the genetic and phenotypic diversity of nonmodel species greatly improves our knowledge in biology. In this context, we recently launched a population genomic analysis of the protoploid yeast Lachancea kluyveri (formerly Saccharomyces kluyveri ), highlighting a broad genetic diversity (π = 17 × 10−3) compared to the yeast model organism, S. cerevisiae (π = 4 × 10−3). Here, we sought to generate a comprehensive view of the phenotypic diversity in this species. In total, 27 natural L. kluyveri isolates were subjected to trait profiling using the following independent approaches: (i) analyzing growth in 55 growth conditions and (ii) investigating 501 morphological changes at the cellular level. Despite higher genetic diversity, the fitness variance observed in L. kluyveri is lower than that in S. cerevisiae . However, morphological features show an opposite trend. In addition, there is no correlation between the origins (ecological or geographical) of the isolate and the phenotypic patterns, demonstrating that trait variation follows neither population history nor source environment in L. kluyveri . Finally, pairwise comparisons between growth rate correlation and genetic diversity show a clear decrease in phenotypic variability linked to genome variation increase, whereas no such a trend was identified for morphological changes. Overall, this study reveals for the first time the phenotypic diversity of a distantly related species to S. cerevisiae . Given its genetic properties, L. kluyveri might be useful in further linkage mapping analyses of complex traits, and could ultimately provide a better insight into the evolution of the genotype–phenotype relationship across yeast species." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/4.cover-source.jpg" />
    <title>Large-Scale Survey of Intraspecific Fitness and Cell Morphology Variation in a Protoploid Yeast Species | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"OmZdxNX2Vvgw6K_v7pghAKgko_ev3XXqvJd3d960pRU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460124","apath":"\/ggg\/6\/4\/1063.atom","pisa":"ggg;6\/4\/1063","ac":{"\/ggg\/6\/4\/1063.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/4\/1063.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/4\/1063"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/4\/1063"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/4\/1063":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LeFMgXOsR\\\\\\\/DYaR58y\\\\\\\/ij+vEKWU77rT9ZtdAiSbLb3eHg6MvKF3H\\\\\\\/dpdwZV5wkw+fJf6IDPTN1MVnTwBXz0w1tCe2KCuMedC+NOJoQeZaLFYKWJ99YiJDfY9PlBO4TxcdGgGiPsaWQnG4FRc88J3uprWwLUytXUvtMOfxR2F5FxrtNhmL4jPixnpIqiG5Rax3ATToTYp0cpPqxUaEme3csuFEDgwoRsZYjIbvUeiFJARVyM8pDzJ2FRTipnYo30GzfZDYtV38ojj9i7ZSr47UU7Xk2QH83wsaf0wmUjsKJ1L7wWDdCosxy2K+NrkyTxw6SEVDvUpTTBT3NdXlZwOZn6b2nqVHjnO3OY+S5H9VsibFlHZMJUmsb5zDITyTubfhan73q7H0Tvtz3M08i9x40OvVc1TrapO1MdqdOctHHLZMqzpIWBvOuDQbtbdWyypv5KxTFL3bx7Mvyp\\\\\\\/EF6nB4rv3EX2O8Y40\\\\\\\/iptEE7rhbjMsgJtFhxYq6Do\\\\\\\/FXFJ4wUeKtWxP4ltVrAehJxTLV9+wKgZ1Hhwo3dX\\\\\\\/PwzigM1Pnf46mTSkJy55nnv9IlJ4PnOstrpwKxh0uv7upY7qBc2wppGtoToY1H70B5Sk+CQAA39njfIbTaJYlPfnUaBq1q2hrsrc4Uh9U7WP9g33SrjjrAMk23rrpC4e2BXTSxK+fkU0weMY3yjVqZBvPj2if6V8XqnN3bKAp75O9Aqq10J4jx89ICZLb87Iatnm\\\\\\\/83IsWJuiIWSJ0YfHJvymRnjdU18SupPUhRqvVy9er4vgx\\\\\\\/wEhOx\\\\\\\/kfjXf3V863vcxBfxtxMzphx3H6vljCHNzGPMkooCYpHkv6u3taXPZHecphRcnE7jKdWQUe5PPQyI4Bh9ZaMW9eojkC1fdt7E5H\\\\\\\/V8ksfitG79LYMr5D6DFqqpA7rHEn53fYKTZRkb3Y5KFec5GYsYEFsPx0PJnpQPgzREq6ghD1Okg9x16HVRTWA8t2vqnwJ3qaANQefI\\\\\\\/LK\\\\\\\/gXnRO10Q7ybCuJ6\\\\\\\/ptS0sFuAE3\\\\\\\/72P55flVJUNJqUFQxLOxVOeGdN\\\\\\\/MV0OG1ntIMppST5Qjwsg2ELQfLulazczvDYth3FyA0cCj7FJJQ9heugRpYT2qY4MEFUOVS0rIOv5Rem+d55rP4ZgeraOuI3kcp9G2\\\\\\\/XLX41Cg3M6ng3gd5XmA4SCFvN88Rt\\\\\\\/D77edRTnCN\\\\\\\/+h9UIhKROhHLPGyFa+ZoSNgmoETGQRfiHfwQ599EouckWLXp6L23+ve0HCIF2\\\\\\\/TCPMUINCj1lrX\\\\\\\/o49nOmu\\\u0022,\\\u0022iv\\\u0022:\\\u0022fAZWmEaSPEtKV8T4OZvNiQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002257e9cc5e054fc28686216aa3b8b0f8c9122464b0feea1cdea0a1fcd2a74586a7\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022iM0pxijbpUPnvGIb1e7vnEd4EM7NX6KkzkC56YkIuhAa8QBibEVpG9LzbQ\\\\\\\/QfIETFWVSovd6rt8V4FElye7bcvKDpS226LqXAe4hLlU2k694a\\\\\\\/Gv4yu9ee43g7n5rC91Z+QWVFKZJoFJVdsg8BEiSuIGUTdRcLAkwIHQl7A47GYeyyPG5r+OJHdfS0G8dSVqM8HHBu6n7O4UxL8M3py8DHdfvNrkoIyExMizXMX6AWJfGIBgmZmfMfQ6FDnXaqlPVG40KkAQRXmojzKKuXJRMQyZ945p4H\\\\\\\/U5IOHYkTy+yjzKUELMxsVbVHxCcGtJHnLH9EiMdo7YNEWwYTRgyNsxuJACSPeGBu9WVTHxFhgzrXye8x\\\\\\\/+7O9Ij+nP61kshKoPqT8GtOAg0WjFm9M2Y3Z2929zF9NUfc57RrqTsv6QFPElzmDsiKQtEp\\\\\\\/2ggGMitnKPn1TjPTPKnMAxCuhXAoQK96A\\\\\\\/wE4oI70yjstfgpWr30FVhajcMn2Wso3JWOz0ln+n46Fx1hyZogtgqpjWcTX7uW8DZBTA488mG\\\\\\\/00IqeVfTdjYXzFZroUhQXjNplPvPQRf1QXUep1M4jRdZaqRZ+Ljyrrbs1T4M\\\\\\\/D98uS8zUl5bynpjKAJ+e6N7jRlwwtXrzFqkmDPlKdp1LjLYoMt\\\\\\\/c1A0XZ1ce4VLOpKKlxC9kBdwN3tZpIQaWlMYjjVE\\\\\\\/SlV55nl25P5kIkJxSvgzewvBnQHbunxHIIdfHyFxqto+wakqs9mgQZWduJTh7LwoT9ArMWh6bQSTbdNXk+EwUKlN5dIvmjg9ofUPjxpmuH4OpcPqd7oAq+GKTsP5xaTNJGJ7W00lS8+bOBmhL+1yjp1l6DCugM3yEnQIkQokhRerHWjgy1vZmhugy09S4vGts61teC8mvTP8VCnRP8kntbVrPQBzerZztNHH3BRqnjOOqX8D+wKE2miI69ON03lFlKQYFKUWBLdlIGGuPYCjihgJRSvJcZbcRbEUGu1Fk2QDQoQDvQnp0\\\\\\\/xQZJQrIfG5rr9wn5ywRsld3nCLrunQaWEvyKq4Hb5o5R3t14v7NAY+35GVfHOYUs+1kpcKy6Eaxq+11fQbwe4+1u7h9r2QZF3VUNpykJ1pE33Smf05\\\\\\\/GiJfDZC1zS3i2Zk6ogvP7YJmMG608JPXfvKLY2PGjQStgaWduoGltRzEuTCHdnIbB1QCH3xC+DuwePc74jEkr3OGFQ+HkU3ANrEze6RjyXrGiSutHvaUWS4djWFgC54lfoekANDE3J0vP+hnFey2YD6VFeFQkRgk+Z8rnwnDAp6CSBSQ5R4XOFsC70X2ZgbqjANTULILPgMRMzQ49LHyduVgnmNAPWnO5RZCgGaJDtXTZSyXsQxuW+lDWF8gMMDisBHHF\\\\\\\/TeFd0JVdqAKCQ6zvNeheiWxGAX\\\\\\\/ZxghGpuzkPROMI1tWQNCZe2qMJDatFMpsaoAdXJDiX5rzkELyGOiHHpWBMNIkEMR\\\\\\\/oFhN3TvlXHYVQbPWSwxM3uzRKcieQdPeaDiQ4Wr6Q+vaxau93AkxvU8PXdVdVTHo6LXMPewSYTbaMpvH0ntQla27agynmLwqf2C49BjV+rDFp\\\\\\\/LytOk0objggZ59VEx8CYxQcFjozYsTM\\\\\\\/Y8n\\\\\\\/ASLNLITQwavWiZhM61s8kigH6fGHbJN0ZDD71NyExAizdOUXgQONFriTViLsXZqzdnE\\\\\\\/ufwpLHlDAnu\\\\\\\/XLCx1sWjVSsI7XmuJKALiTx7kddu\\\\\\\/BbV701joeHcT\\\\\\\/8uWUFiQ2RGUoJqcZ\\\\\\\/k6MJJKIdOfgAOERAzLxNelaxXz8xPOdx\\\\\\\/wV4EJhzx5HccSviXw3y2N7LBCNYZIxmHbWmyUmeegJP6lg3Bd+fq843fm3pDpIc4l60E5mTbw5AOG8gQaClrd0crGvERg7y7SK2gyXAcCel\\\\\\\/DNr4yieDpC2nw6WqKHzLOBrGRbINnfAIxRBujrIHAAsx7fzczGxxjTzunhb8Z\\\\\\\/NjrZJ8HP4yTWokcpRpZwqAUw02AjUgugZMSBsu0fJ6whSnAzwHJTcHwX+cRqu+nuhdw5GrcoXoBo6FNmDMvybA1\\\\\\\/YkYkpwgVDkGpow4V7E\\\\\\\/fJ1Vo+619Sz0yFgb579asWOh9OaoNMCA6BIWlMz1xGMcXk\\\\\\\/pFU0aBJU4LRZrm\\\\\\\/LWoBIFaR2knW+n9Cjzoaf84g7i3btYTDkq8cQZ4854oZK82sOixcKV2eYJ1TFCTDnkEQVPNa4maWfB9SAyKeLG8TJGH6FPn4KX3elpJnjN2dYxXEEUmNzg0v9dPCN+u20r0R9GGv8VnUcrWtiFGzesN9MP8gft2wZpsjY9p\\\\\\\/Yu554WxmRZbMDy7t3VZBY69JO8iq1BcvWuwBaS\\\\\\\/UVDVCgrZNKfkbXHss9L1r50\\\\\\\/rEfZr9JzZyzNwIyAJGikEHNzM8TJ0ksuxCFxg8iwkXO47WAnnGrTnh6uXY79sCgyDDpJq4JBnlwWrthjdedMzoPdVTygWXMoh3x92kJor9cRmwh90zMhVRnYMYQeQBf7PV76Prh3WU1XEaxJQTUO\\\\\\\/SZgvc2DSyNH1o8AVIuAHBOule\\\\\\\/3VvaXsGJ3UhW84PWGoR5xI4sOGX7Q\\\\\\\/aqdAuNC9W3rRocR2o+GMDMYPdOSEb39PnqoDfUOHux1e4g67uuJzrNWWtetDEaORe1o0s+q1tJXIN7VL7nwkDR2pYPxvgSDkEjgP92ilR7+gJxumPDnYiJXQiE1\\\\\\\/rMQn0E4VQMWCkVXcTQZsh0w7JMiIdfu2wLtYFIxaIQD0hrGCUwUEwAO8c9PHUuzOrZ0S3UMUFqWkaiBVC2nS4GSgwBHcNOF9Xw+Rfpv++qaOc5FyazXz8UYVFXkPOvo7\\\\\\\/y0WlSBDUURvXr39SZpZJ7ELHgWxoqq7Anz\\\\\\\/ylCyUVNtORZXV\\\\\\\/p4j+hunKn\\\\\\\/UaERTxBCQ4dPTjniGDWT0FRy5245qZgwArkQ+GRXcQHjrzOpbGVKJ\\\\\\\/odbtq02FQMl4Xv9wbiFt8DhlXJJ2fNPgV8aHPm2535M2H1bTrbQXo960E2XV2fN\\\\\\\/GyaBglPW+ueEchHXnNwE6mZ\\\\\\\/bu9YwUo2ofj1mwhIzq1l6OHH\\\\\\\/RSqcOT7+ryo8Q4artEoVhTRfhjIL9DgZu0\\\\\\\/FUqVpvaWoN8yOd1tSvjwQqJb+6NTt7Zi3eVL52P1HVm4mlwcQPY654QYgW\\\\\\\/ithMjVJ6k7LDIvMdJzU\\\\\\\/ZvL\\\\\\\/c6smIHcy\\\\\\\/TxbgZmAwOZ6k28chR0uYeS2p6HyXCPR8H9E6fTD+PmTVgCDAE3sEMB5CA+gaODtZFQHJNuFUfO4V3dhQSh+JsCPI7OvjxYfhvuPRe9as3pk4xzvKpW1n2KkKYlWTpEf+y2LTbG1Bv4fhSLspvgVAStXafmukbODpw6pGGgbMqFlS3n5wfF6jpiRxjPF8BDgyt0lgDWjt8xboR5IqBOpu1X\\\\\\\/yFowA4g8I2GSbcwCwRlDgULDmSjvYgV6nPFisDU61VY\\\\\\\/JOBa2j0HEQ\\\\\\\/zP0tIN2f48yV1SUGHc7BsUBsEqc0jnTB1OOGCuYp70lGNkehf0nsdxuRugeMitB49lqjv\\\\\\\/4pb\\\\\\\/HEDe9tIzROzvAl0zG1+GOuvarhVUSH8isusgV8g6F2N5A3HJxQRuyLqD1NxKr2YyJv3TTMK6Abxvp8TGo4CRnakhqZ3Ny9ZMloxI2f9aQbTgmSmj\\\\\\\/csz\\\\\\\/PsttcwjAt\\\\\\\/onO0izbXIHPAsyzfWwwqbNUUPfbmMnI4GC5OWr3vYaHKbHi6lySDxiAUSk4BxOftLnrS7HBvk4VRxclz7tK1ahAN3cu3Zm6qzDKhy+1\\\\\\\/O4YEIMUnM3oIOIEJwYeztStD0sARzee\\\\\\\/oemLfcBeKErJNKWpnicOoGM\\\\\\\/wTgD6Rq7eIhoiRqerXXDcgf5c6EcwE9NOyCIdkU9QGhJ3822Rzi10GezCUmwkgKxd0fGX7N5pGTD6sKI6t42XPdWpIrX\\\\\\\/ICQnjGhE4n1UKPs1imfggJZLmYPRyq6wz8nldBt0n5fhdc+MtlGmaprxgKEoaIyxwucPIPl+wZvIEvfLTpReziyFweYaRnZ0xydXHIvHsMTVkGuDhgWUeYAgwQqWp2zzuxS4QehffhjKNX3RjahchmewLQbeono3rGFnpJ9BcVB8viZSzuTJg\\\\\\\/jUxukEJwvOYdzlFLxZxlNW9bFoHndeX8XAiR6BwG4+HmaZNDzJDZsgVgS6VQerodNTkSpuc9l4tyWTlWl1afST5xaTVSGw9o2thn5obR8IU0wotKLckE2EpzHrEhr+EZjLS\\\\\\\/y3NgVFy1oR2\\\\\\\/NcR8N4d\\\\\\\/oPF5RggFf70NRT8fCWVXc2OqWPlHCh2iefeZT8v+N0CAVG5I48gAm49js5YoPLMdrUi6DgI8SeKyub9\\\\\\\/gVqexdDT4qOX4ovIKz1tpQ1G9xdv4j7EnYqTpUmBDVHEI0q3cggrFfY7b7HBY0tNhnmJS2McywP4ieoqIDJaCgoX6EDi1PpFBOw079r172SlO6xEO8q0W09it7Z2smJqkNc29IOCVZM4EMcPLK8ClWPZS+oTcLlUnBbH7J+DmhWR8kV54bZuUWtsfPVtzT9n\\\\\\\/YiTw\\\\\\\/\\\\\\\/LduZOcCo6SsrNCYa9108krIxb8uLbP7wnrn0etlSDdKAZB8hSG89xvHp\\\\\\\/VOCyvVlp620co8AxLEoU+914hqlALCJMSSC3iL2YiwvZt7wPBcIPTR5BcG9RV0gxK\\\\\\\/OkPDzwbCJtoQR83ASRzRRjwcokAA3fSsgRtkjAmfhU1sBJpZO6M36QxGR\\\\\\\/uR9SUFFBxTK7lQnYunZBAC93sBUoN7yjJ0p9z8L2qZtmhzqnriEo5ymPVlZDPlbeuYUKeFcvF60ULKpk9Mg+DgCz5PgJYlErAmjiIAwZa12M2v02aUjw4gySLkIwgwmrh5Dx\\\\\\\/riSxUZvsV8hcMtUvyXb3eZWxPNSLtuQmv6Lvh1THtrt4o4GGaVTFgd6NHhOH\\\\\\\/zBxC1AKSOHN1O5EoFPQ+DHckilbmXHkBrhehlBUQ\\\\\\\/VCNkjEpsn0q152YoLkIJurIa3f5GAobLrI36aBxOSqffluYpkvam6wdaqs1zUWgtTZrPDRpyeAH2nzY6\\\\\\\/i78B4GNczTL6z06hGks9KN5Nri4OhpPW23rQV2jaHOcbRHmMINJ1qrFHsnifzrZGoYc4FGPG5G3ZOJUyT7AVrDODRwqHZ4le49n4ApAmiq6wOvTDjcF4bpXrUXImpPCpn7+HfMazfj6Rc+gso\\\\\\\/0YEwK4+2j2LJLrTQQlhDGZQDVClARWHaEy7cFWq1uFGh7jYr\\\\\\\/JBUTN8tCYp1rkIuMSQD1gCDYbRZPnvgHA8AZvReDUhyCqBPsD\\\\\\\/RVzD1KqxvNB34PnvF1b7lJ8hYSUM2beFn\\\\\\\/AXTNg9hAflFLRVNo7amOXEVMFb3JZTNMRo\\\\\\\/zy9gtfyrBIfjqadAyco+lJbqLJ3a4usmhz\\\\\\\/aWn8wBY25Y7cjnHxPDJ4u1l8V+XGYgLRB5vvvUds82z6EXMIi9IkIKrnXluScgL18VNz696lUE7Ju7h16RmjgZefDzONVjs6PjfHx\\\\\\\/XhkMICr1kCzaEscPm59BI5KnXqyXSA6vGcTUKW0uTm4FUeeHR52by0o2IoJ3kZ8TgaJh+gNRh8S\\\\\\\/rtZnubm06bHhFvZ\\\\\\\/V4oADhLUmQz8IzBngrfveOVJ8qANaZLA8\\\\\\\/+Jt3jKHzYpJqM6C4\\\\\\\/AQWPw\\\\\\\/pwGJvqWwJgqZ8QvtP\\\\\\\/408jD5HZ47YLQz6MQUFdugsHHotTAeqxub0NL+Chl07QXAXbNCaZ6sjqSgRwohLBlYPVWJO7qvxd+HP0klWWa6jE1lybULbxEZAlDT72f4NpPZ+QJGXJTili7DHI7nPtI\\\\\\\/ytXkrpmluYps07F7LsRD7iPSihoxogEvXGwMbJaVi+vqVkHeQAum6UpxsIGtY01vCy\\\\\\\/3EzxOJIl2tT+29j6lh4FM9P17n8zSrbFCWtLNMkMJm5TjREY9l\\\\\\\/\\\\\\\/mumt7mQ62rSkKhKZNNbAyE3E7OV70SovB+bS4tjf2h4xVLUbK\\\\\\\/+j5Ild9ByQQoe+2Q7syKl2dFBWBWs1WhB3vZmxlTCF8K3S063ZZjy1ZUUMDf\\\\\\\/jwBWvO+ksS4TwV3g8LMdIO\\\\\\\/MBUDlYCLp0RbUR2GhY+7f0WvWIJoYzckm0N5CLU\\\\\\\/n1bulONxXrsYAMM\\\\\\\/4Gbze\\\\\\\/9EFH1Bl0P0oJpKIF6Z+Yk65elAO9oj1p3yNGgIuH03JfHZexbOeIns2Om5wE2feoY73PBAeYHVp3cXrNXU2U+iQXp9b6NUIdzbLIxBYKUT4aOsIEP2wFDQ5y0\\\\\\\/yBKPjhBQbqklmtcx7EV5VtVfRVe0ZJ6PG\\\\\\\/H7xaWlzwdT9yvHYwYCfsN9iGFvsvqacyor\\\\\\\/ApivTJCdR3sbOJRhY28ZHJ7KxjN2KUnghLrjK7EKJaVQeUMIHJHe3zhpXZGKwk\\\\\\\/g3jE8oD3C2HtmfrUk9PBmv7uabdj8jJkbBJA3o3sdk7bbyMG6uCuSb0T5hD2Mg4gIG2aSJKa\\\\\\\/cVRyyDqlN\\\\\\\/+sQwCIElMmoiPWzfDu3nuJ0R0GEMCGGH5rxG13p4QO+3GhH5MCn4CIcWvxLhRzYO2HFHYvkBOVnWFvFizj9y2FAbbc1w+FXv\\\\\\\/ozsHNwcdft3osP2rVa7MibL6g3iDCo+Te64NTFVGOLcmZ6xEtJm2lBdkhnfzVoqLxEpEpilaKBRhdufdHODS83FDnQCLztHRQKv33CAkUBeghmunVUU0PV+KxW1vA1XWf6kdqlAoRYQFp9JQGWu1q3R\\\\\\\/u0HrYkbG438XYDgMBnYe+hkbWNtQIDOobMY\\\\\\\/FNE1H4popX4s0cMcxBHxNUAt7zmjiSQl7pHRu5zaiQxFpXrP6ioX5+4E4JoFlLfJ8ab5q00P5FKREhu1cTiHBIO\\\\\\\/f\\\\\\\/D9Ug\\\\\\\/YFqKLzJEAfS7RX4qV\\\\\\\/4Mh8Sw9ItNobdFilBlbDXuK04kFefMkK1gioI0UHxaKZDwXCrPj7aYnv3A9N1LUEqtCF30XlTVoOEUB98ImvoLI04qxVB\\\\\\\/cEqovMGk\\\\\\\/b42nQ+KZtAsVJOB67vuLC5u0Yt7w4rdQZmyQgRHrPDawr8HHJFhpzAcuJkphGU2giPruavzEMuDabB59+gcGI+o4pO7E+fduS80DA+I+fhFaQhMsl6TGNVQZGjk2k1DE7QGC8BTaXUAySJDg+21rc8EOr+HnE9Whf05hUMKQUhf+IAQ5BOX74Yf5RBtoAafMolaW+i\\\\\\\/mvlNODmKFzNkcUdZfZ2RxlmCvpdeGC6\\\\\\\/tPoR9SutiqAwLXx\\\\\\\/Y\\\\\\\/cR8D6lnV50wmbuBpJKDIRms+yAdhY2FVShJwg2l1qSSp8E6ner81xPF1BbYsVVEzHFrFeggRTlY\\\\\\\/eeV+jw4NhHV5t\\\\\\\/z3RXFCsbotQjHKUykg3TJ\\\\\\\/OacYkjCBmif51mDtQToUxD6jLSYsDSWZLShY72aSabBF54Pm3Deh8pFkb3q45uTp07OVJhejFbmWt\\\\\\\/fW+4QL4C0u+\\\\\\\/Zz1hwD93syrd1C6xoWc1uO7sSZOQeY7DnMe7VQPRuR0KbZ7JNi\\\\\\\/PnlbQSKZ+gFH0xhb6WLOtfYGRTCBI+xFAtmV0uOpFX1ORHOiNHPK3Ox7oka7gUIyk5JUBEx4\\\\\\\/5I48cACcieUgJZi8632DdFAY\\\\\\\/fErFIkuKtTisUUoP9M7vu7+\\\\\\\/Lt1QuP6L+jm\\\\\\\/hQcIIQORAf2a4gSzO3cJ6iPG86VILMpEffKZMRv3uLmXfxrCm6f3VYXWf0T90+6ngO59oEwchwXQNHWvd56968GNafSRv85q3hpH7znNhDzZsONvu6bjHbpZomGfCE9\\\\\\\/kYEfKGMaXiwfgrogDNtskddc9wAt0H+H2\\\\\\\/yM6KyOKoaa+H9sPqLDbg52wMuFkvDCxCUMfPfMS8RTcqDFO3kyfT\\\\\\\/PURyfAH63InhAUk7HGWeHYHqeMM4EbULcgBwaMj936HbE3JaezpdErAKDY9Zukkoo+V027mEdvztxJBZ5Qu0WxpmSXIhgEy9+SuhJiRYzHcWg7DBn0GTEi+HEWc4fD8hAm6idd5GqRkc+c9g6kA5wzaWJv7QMFDdxZmr5UZgz9P51vWveBZ7yraUt8YrZ3k5Qe78mI48mme8hC3VB6X+ER4CBfXMcg4PI9eUCtR\\\\\\\/XLVOYvQSumX97j6RvYhgRaLH8zmzwbAAg9qGv5Jdg6kjrbrVbdOM5OOkeZrG2uv02XefeXb\\\\\\\/fhAFa9v4uXsSXGmWQoBOnBhWVvBeB+KWn9obruaJ76nZW77+TPNpMZUmlTTjNQNpTASdlE1ddk0GOsy5C9bnWCneK+TxfMC3CF3WH3ixQEQ6sI0FQMRP4c+1haIbr1PUgzGWj6NjgAvr4JF+Kv4EO3Gzp+YIX2mtMDIFBcEKe0IZhhI8a79MOVFqYcmbrDpemG6hrAxKRfypS53MYCJx\\\\\\\/jBE8tL8MMygfzFbj3ufANYm+AlLsTU6XFNB24fiaF5D5s\\\\\\\/LClZsku5ravc+D56I9RidK79\\\\\\\/6cMZdXDg86ilwLBFEE0lu6pBp6XOmxn03zOCI0IESNZW+dVIBbRYN+HB8Qhi5LYu07SP8qEQOxeJWfZtveu3uiYrLTlqRm5i4XIdLuUyea3UehoRFKhtMSDzAdb96FY\\\\\\\/8o7Ccg6ZToN0j4tiYpV+D7qvZMeNMmzbDvnJpr7ExeiPJPaw6GL1\\\\\\\/CAvWfDK29GQB4BXqCd03uCz+o2wvFfUh\\\\\\\/okH7rxTqD1Qh3G1+iwL9PpdfmCGFVj6yfSyzmhbqKY7jL7V8zklkWYh\\\\\\\/j8EnEYbD\\\\\\\/6MKhiOWWESHRobL2XHsNSxNw8E+gnILFcu+NJJvyAAV7DTxw1BBsKQMAY0Zw8tRe7yNgcXKJZcpNDrznKhKcX6c\\\\\\\/i3FHa7\\\\\\\/6SFxd+JM9qLkHNsQ6eLPQdgS3cvAOHpMlWuHv1rhF8qqy1ZJWoweNRsiUb\\\\\\\/sUJ6IQLwMPasYVDYys5wo1aS+qp0FzY6LX4jGoy\\\\\\\/ZlFEgrCT24pocxCz\\\\\\\/mTCjkrlclvKS5y13XhBkyRNrYdHK4+cHjYfHTRbmzYQIhH8R9yL\\\\\\\/+kJWXThA720ZSFqdB4pvd9AUau9wDPsHbfohXbKkltqHQyPTWLfzruiEuQDS9yey6GH+07Ir5waqyiI1W1JNcvqUdgGvYnq9IgzvpfQ\\\\\\\/QJe7vqqnJDKKlMDv9e7ADjheNsq4XzNpet86boHocjcyJZMhtkEjA4vo2WAGf2hgbMgzdXkvO0gDiBLW8ZdUGrHY7rQ3LkNOdEzUS4IyKylwAujjzMuu3aGRFIgWu4pPJRbrWP+1SGAwxcUXw4hHU41Ta8rBdYqxvLll\\\\\\\/G4nDIjftoly54rxNdDJFkAKTA3elesHR6fhCIlujvRp913gE80ykHThQEZqIzflvfiUSO1dYkLMa3eTSY1o7VMgk9ZgBNlHg1ZTmIbzjlTMaXYmjRlLThacrL66aKhtSg5yhaW\\\\\\\/N4IqIrMC2VhfsFKknqbSTvCGezzcYgByR6Rg5pKLGMmj4ozqMyKqAX6r7XTwdLjQ+KN55m8aCB8ZNLjXRANmGzsULPQRBo3cmNcmos4vpVGc\\\\\\\/yarQJJHmTGRXKMrh8tzmH5aSR3+PwwEe8HUL844bliKiPebHv+uu9mgWdDZSmkIwJagGVzFaQmJc4ktZ2Ep1JtreP2KgNZcHekLT\\\\\\\/0pnoI46rrS1JKq7YiThb4MtN8VdqQjH+AR7xdEV8KsW9piqCO0NmIbizKMb6jEVQqA+OWhvFxVKkIVJfNlEYB4eQI3KxLGUv4UyNFg4ff62VVDLOGz9jykFj0bjHjo1EKw1HUPml8S\\\\\\\/szD1A7DQdghScw33q4REEXK112uCIId37MIpUiGu6ETsFXAIZcCQlVrifXl0uy4S8t5vNjiHD3bui9U4lvbtz53yxeeoBFC2tLOanLXd2DKbZYXUmxuW00rJ4x\\\\\\\/VqfrBCSCJoPVM9XXu3qOMF9h2YpVTIEb23M88Ngt9LoAahmep66OzZhuBAL3sr\\\\\\\/ZiHSgGa5jI+G007gM1RCZJ+Jb1vRbBqJBnLP9+ATx5LJXYsQ6yX9J9yPssUxGIxYB7VG2lB1DgDdcixGBDBogO+FyPw8Br\\\\\\\/AZsKoWuNCtEl0GZdJL04tRmJYeucJLLtyDULwR5HYLw5Qfwf78OPQFKvxAMnsSWazx3M9kQTvMkVT2SSE4gy0ktZOwTw2fxsdet6NP5Gf5Ft6ktoJkZBqis9qp1G3Cncu0Q\\\\\\\/jE1mYxv0DF1yHnMK0N70Rk5idGFkDfLTuxZDlykJtOGFylbW4eZ8lLlME5HE6D\\\\\\\/uqQo6mKf58IX1pRMiWE7Ao\\\\\\\/GW0SZ78HEKpaeBRWNcjHT1XVzW8neTcHfSp2i8ubH05ogAWGrH5DQmUpdf28HKJJxnigIs1jHfzlynxULJXyhA4UPpMKuKQkMKnPaVRMrNTV3qZ2qQWdB1drYy2E1zx80jXOOpapS6VSWMGpuCiIlU8fG+20MtNqVbTSKsu+D7pfEBqc+fwrj5rprt13Q9D15WXHbSrGON2rwDZg1YK8sA8NpankoofqEySei7VcsiQQIWLwVQo1PC1vEVG70VpoEjjWt\\\\\\\/kQuQKOa6PQ1R1KGXwaalWymUUfdQKP8xeulgBQ89dPA63FIyu2s4LwnnY4oMVkmENFiZxVH9dtiD3Kcb1\\\\\\\/ObGZFJWtHiVbEbDx5hUl6zuvEYi\\\\\\\/lZ2hdrNAujv+1smDzPdKbB1NjoZFm\\\\\\\/YEHPx7VjSwwGns744f3Fvqr0JyNKOlu2Qt8lLczlSjwh0WgDOo4yZ2KiO0qXw6Pui\\\\\\\/L\\\\\\\/Wf5W7vbBMu5ISAOPvlNlUCk2AXsaMifdI4gKY6hIrckABAz9F7cjPIoQjaiK71xs\\\\\\\/plEEdqv+jgkDnfbnquLAxbij7UbzYmRrZYCConK3R6z8zTsz7\\\\\\\/v6MY+JOy+Wnnq\\\\\\\/YMxg2\\\\\\\/epGgal\\\\\\\/W0XUexGmoZDqHjRT2Qc3JbJ4XYpERacC9uyFm7EvujRmHAlJwDZaO2WN9wO9hKbq3ntxyAA7yt8SFJhyyErvXUqPVGrRcHLitwpM5k2NAJP5NtWrQsoHcmlFSWjrN2y1Y1bPEfKil4J8\\\\\\\/IQK7jPjaodV33e4sHYlowlCDT23pwrtvQg0ufiw55\\\\\\\/q4sO2u\\\\\\\/NwwRn7lqY9SXZygxzQdRPm0QWRBxhzHUoEAeTHGU+OeEGwbbWg5tTlZG5H1z4Ne0RBIoNzAck3a554qM0OHow4j1g+LnZkSGYm6XF6BvDboRrJwFtsNLOB8Vg6qU6QzKdGK\\\\\\\/tmMjg5O+R5Zn01pvYtrsjP0cvb+roSTjdI\\\\\\\/nsqnjGWM46JOKcKFQ61FSivKiYR3RKsK8JRgq5QR+DMKAZ+jTzPox63VDbBMmd7z5yf4iQniYtPOPTeLz\\\\\\\/x5rMIvQsoN5qoaZWL3xyJ9XzvI2t4gaZ5BFvqpkvOg8YDf5jBKdA9\\\\\\\/BIfeBAScbwC0p6jVPH\\\\\\\/F7BNFHrUDbCahBIzQ7O\\\\\\\/5rvix232mFs+6jWCEv8O32o9ZSYaiadB6Uu06nBZSq+2lgkb0xBYUDN9rTFX6YMTE52e1kCfSubnk5xEviRaKhnd7XuQL2VRg9qbt2uvo9FGQU2YsfNLS6EONwsbnH\\\\\\\/j9x\\\\\\\/HTv1RHu5pbFu9D3DguPdUzVzPNzrWHcL0Y6c1DJ7cEeJcMxpMvWaaFsFKf8Hl7v1HnN3eDZzAolAAcdAZ\\\\\\\/Y8hpbr97wSBn2T89ap08oERsgMhSauRQg2TZRDqSCJ0\\\\\\\/AudRT0fwzuw\\\\\\\/ajEEWgg3\\\\\\\/KEZBAWZfCSPq60q\\\\\\\/pT\\\\\\\/3kW1jikSQ1xTCo0rhc+rQj2oQ7421u7O3FXUcytMEhqhRn1+9uRUAMEje+kLTllHw\\\\\\\/UWhIAoTTBJm7snJHGRpXQAJi3EuxYR9jsNDLjI4zarnJ9MzVGo3xcnAe2BjQRUVVSe1ZJ8zfGA2EAnwbN6FTP7tCtzDMCzt4+BdLwoGCUOx1R+PopzhsT6qzgwEbCGN7PNM2l3pHcWgbEwBusBzOHysj+8M2CSDerT\\\\\\\/XpTTrJfCMrAT1Sda1AvX5H\\\\\\\/l5hH7JiJfSQUuhZcZN6RleIT6smlrsyQ8IeqIo\\\\\\\/5nPssZc7DtLPabg+AEDfh8k3kQlnN6Bjbz0e8HQtDygUdG30E6gIQ54w1Lm8xEGcxQLsTuN5x4UvyQQW\\\\\\\/D+\\\\\\\/nl234LIvJfkwwMZtrBul0v673U+k6Cwt5jxnYwBAsncbE3jWAUWo85C\\\\\\\/IHqFuWjwz4SOOCcshmIO9RlodqURxodBpzWIFAK2ROITp4vHTIiO+ykOdxcefVyqQZLQbddHZJmjaBFyFqZaQTPCVatcpZ\\\\\\\/p2x0S42+Kpy8N33M37GWkSjFb9ZRibd\\\\\\\/yZy1cOBGrefvLADBAG1TZ3tDpjSIVyW42jRPidZnpwstsMRgd8pBTdJ+f3eWk8cMaREZUAUgIuVGkm4ic35xTYhgpc3AA4\\\\\\\/B8CwezF\\\\\\\/4t52EWMUezLwQbRRlTYPq1hcSRhnQ8EcRijXdGdVWvv8rTCkJ6UoYmX55QAJK2EkLJcH0ccED4\\\\\\\/wRGPPpuO\\\\\\\/Cf4gTJYm0fiZs84bS\\\\\\\/Q6yfRUisvshkeOXNu64kjKGEebsGa24g73fYdd7GPgaPvN3itqe1avGCh1NxpxC64kNdXZZONw6jStQVd9QBvc\\\\\\\/Ip3xwB09IPZYp6UXzz+FpiqxiXOmqOgeTBkF956JJQrMVS6KWQX9YN4igAua9DecmCAygHmymbF\\\\\\\/HUY7OWD5ieGZ1QIbln6hQUU8XBMlBjL9sBgCaM\\\\\\\/vBwI+oHFBgt71iWbabJgO2zr92AMjFR8YB+WbZXzyf7ll20a41PDeLsEgfK4uLRmI0QZVjFIlqh\\\\\\\/l6j5PJB5KPh46c0fo\\\\\\\/3q80Sc9sQHtWFIPn\\\\\\\/u4o9DKn1WtE\\\\\\\/5RwQAxAKb4RJk+ORxpCNqBQSX5+YS+HdX80h2wnKcmxqlBYB5h3RP\\\\\\\/DPljxBlHeXQRQcezNCIRkczeV4D0bLCfWk9U23x8\\\\\\\/9MiJNmfe3AshjkyjK6xsdBrfQzRXJbe5B87dXJtzR1wou9ukVveR8hnYrK0fojgvoDaXKjbZKmR3O7SPvoEgkU\\\\\\\/4Da2OhGrX9+cXUrSqeUmaRmbe8hK9wQYGvHRoQuMmAyv0zmCybxbSlFb7YMHEbTObK8Z9pk2KOTrzpeAbId1Uag1ebSV\\\\\\\/ixWS8+AJtB+hTdw399K9Qmk8i5ehkRQH6BKRfDS0LftvQWfAfr+NXOCHWNVtif0i6LM7QNSwy98SwXHwxXwQjddPXE8DrjysHF82yZ6nBqwAhkSbOwU+s\\\\\\\/Esp3m3aIVm\\\\\\\/DxejZSaKouGlKqa7SbFOWEC4LdZQPvo0R5u0mIYXAHauozpMGtuqC9o2abDMboBXshy3V2lBzqBcypc7f+YzGUa+1ls3fYNxmdAf6kLX09l07zXMkIbthJ4Zfpv\\\\\\\/\\\\\\\/tYlE9WAUakYV6TNrD4WEIDSSJtP3w0b60raXhLcDA0sqe1hXAdJnH5UEcZZrnGhHfpAvXqM8iKXBtIk+gRdUiSJTahbd4dQI8xraEyjY5UOSZCufO07O6LePMzHLfdt64tsPeeG9NaEgA6Uddoi2qNjCL2STh7o2c+vM57nl+Eq8pisX1WwqspkfvrzffbDH2AFd6sdY6J4LPmFOW6\\\\\\\/aRwcmOwaCIOYvnCUJt8Mp35P0Z3f1rwV3yhlIQMyoWfrQnQtxERY02EViY\\\\\\\/vSvQ1RI7k98KlnxBTIjeueIxU8ltaRSIywUC10ZN4qDlnTwfMOHTUJmViyevp9BTRjTdiq2fiuNPvdPpAfwE6oQtPNXbICDixF4erLNv3\\\\\\\/iNAeoG0d7fLcQQW3uqt0M9Uw2HzMSY0FETJQmAAkP+l4DKQINyEnCWt\\\\\\\/eiaBRw6TfGUhzIszmph1GxMjzDWy6wqjHMabQ9uvMb\\\\\\\/qDfrbxGk2skKHmOgrWF9\\\\\\\/2P902+1Uj+37O6y\\\\\\\/HE96NjhvQ2MdVPNSnEH8aaHICH19Dzk406udrL3HmSXtlQzgc2jhlXHyxF0Q6fS9jr\\\\\\\/NKvZLTV4gmzDmAdUiFz+c5UejwFXo7+F9R70+R4P2JNilmQeE1bMiZSKIOS8AKDQ7qrtEHlTZKJeV+iOQ3QielA\\\\\\\/HUr4eee5hbHllFyuopYkex2rekwqTWEdCyd3bOrWZ9QZYxh1EqmeoEHIlI7nZwNpE6Q8uDB+bLYCcDVVDMTygnHVHe+8a2JBecSrzQfALBhb1i6ooonD1DoOdevF5Wa5u5TuyquubJxYJH4IPKC5WNfsWInnT7K4moV+a3NbTIKF7jF+lGl8fANnIGqkgIovfUbtUqOWz8bWBoKn22wblDX99vOz5DvgBB8L0vCHHio3l2YWhHQba7FcKc1rvanRsGYvLVFdvipAuLpJZDrsJ+iV7HUnvEdaQlAMcB3MuDDLZjnw74wgIOul4X7+xrFANDzeC0XJAA4B\\\\\\\/ZQRlFEErKPjW37v5HCuLO114Dp2Xi8uReRnZTTBrzHUzh05iNwJb9FpG60asblfIjGF3nf7VZJiU7dMhuEtfQFsLQ0HkWfDCFCszFtF4X8YaTEOhEHjYYrrFpRme7GJL6vwy9gqxqmXfPnhcntJI5QbRceU2nin1nd2dR4v8O\\\\\\\/fbXlmbBECfVp+YoLpxY1Zj+fyAA+XT9tGEq3BSs0Gp1gAyEN851x7CesPTLbmOI1MGA8iE\\\\\\\/j8QnTiV5mDttIN\\\\\\\/mitRvUd+1uFq9MBSRhJbX6w+Q+gpckq\\\\\\\/veqyAvWAwg8o0XB0oieWf5TUpCs7\\\\\\\/MvyodsYo8XGJ05J9MmvD0lcK2eoetrji26czMZNuux2aJpjyU5yu988ebLFyQzkRBAZqVOXnyDq5lXvNttnGb79qVX5DN+pbYQxi8+oHPAM5QTZVot5atbxglcFxi5PV\\\\\\\/pnm3tgpe8sD7XuGooOKDTGpahdHcel9lIf9ksYIE9KuS6mXzn\\\\\\\/Sdwv5OtpXugkqtZ2EAjoEWBdniMnrA1NSF8Xx53QAcwh54Y+eu3Jdo5lvMMChtW92pcyiNzzt6j6FpXg1COjZkcT3ma+7Vd799BS1Ms\\\\\\\/oX03F9U3H0D6IAQ+hxAYCVdL4VMOq6v45o9sAcHdqX6N\\\\\\\/r7SUijNGx76cMMn7kmSYuXctWWqFjFblyhvrt8wqS6T7Qj7Z0UI9tx42EcYSFgeOiCEqIfTMJcq64KRUz59oxfYCZm17kFWciU8jgKVoy4VbbjzBDmnvrgzrb61MFBuqPlhWNl+p4Ny\\\\\\\/VEwYBsfwQ4J5DGuCb\\\\\\\/wKB+gKkHPDBVGofDNqpxJZUPtsh7yjS2vai+P+FBnG5XzCs+EBXt2GGUiOHmnXkaphe9R6Bal4vhGFNEflAgo\\\\\\\/29x3iyRTyjOjNSxk9Y2ZOXG26Lxep7I9Zikv+5CgjKEDE2wocEm7mSIStSbzKLZGFMWDbAfNg7MJBWSSFQveZztyofh7JbQ7+QfINOdHSzPL5VQEkcBvIRXUjOtXz4n8HcTROLYJHLZbJwgI+HxpKbBlDSjCFFJXZvIyn5cljBcW0w7ocQKUpgOOaPiXPFjQCyjqjuDWJ4da3YLG0luGxBeyCePaby1qPWHkxL3OgjIwBpOHoVuJFHUMolilO5zfNQfj4HsJbRWZV1d0JdPzK4sKshVcn3884Fotp6s01fUjFs8damkMlSEooG4dcxg3n7oMqm\\\\\\\/HkD1Se\\\\\\\/D+OIdX9RnrGe7U374\\\\\\\/\\\\\\\/yzVBu7VbAND+Ia5OnvG4VozUCcksZaQGo1Kq\\\\\\\/uP33k+DCXwH7hD8KFCQBqXFvjO9wDZ+2KA7wXmD9vwdEFFeC4xxzNad3r0ng2NYGwEvo9hOhI2TEShFxITQiUoiLrdpJyhx\\\\\\\/z7NyafGwO67C0683fftqrXGbNEAl\\\\\\\/erCbiNp6YSDvRa\\\\\\\/N3zKNT1ggTXQ\\\\\\\/Dq3AEq\\\\\\\/pHxFu1\\\\\\\/0GJUrSY\\\\\\\/4r1HjW2l9AzEWTHnfH6Q+gM3rOZ\\\\\\\/Yypm4a9zOMSkuVaDwwQ2E7uDTE\\\\\\\/FquEnQesXgPntZGe03h17laqFkuuon4eCcKGeH0S5QfYLpGU84jwLMBx+9uwau8XsCe5KFG0KcEvl44cH3dB56jnqdODX5BD7z2AdpQGepgTpXa4UmWUdgGL2S9cVsOK1p1\\\\\\\/u3fbCPeX0QkTo\\\\\\\/s0ByXklgJvInNhoBvCtTSdUiCm08Mve2h+sRsXwo8NP+WF1QFGPehbEXUgZ456ovGUqqPlrOxsKVDwsfGia+Uuo0LakmndAfjsAB23j\\\\\\\/cqyeVdZGJjS+PQwX19ZdarTVdQQJe2Px7yo6wDXJ2FTRypBlUrGQWcm+17YV3ZAJ8faii+AAjLsGkJw5KytrFseFmIZDh7demJzzrCCplrJtD1c7nD7PWpgF5Ho\\\\\\\/rg0tDdqKz1nGi+rPtlQnMWFWN2G08mK0U\\\\\\\/vlduIKZiNSDvGm+i395+fvhGuOdXPtUj0xlf4UCf2Rrq9Dk5+KY91qrCwF3v5lp\\\\\\\/YeB72NKSZPBAH8TpuTgoVFya+Y9kDWO\\\\\\\/2KzqwZ+zSS4pQtDYyMpC+6iReurYDR6Acvs7mCWwtQyC1uX9iZme+nSjKFXLEuVoS8iZmakx+VBESgS7ghRcyHjkmy0jWh2SURLEBZfcmpUIOwlySD69INPyfuL72ibQwmTRufozK0zX1a4VE6rGyANizB5GxbB\\\\\\\/a6wGoM0ZPzPdR1lMTmN\\\\\\\/NmcJnNq3f6yEEn4ADGcqWFum2ZuAqI1ayvOvLd0WVemeLiMk1+MhzlxURVACBnUwFZDbE6fW8Kh4W8Y6ls63Nlh8atSgJJnVDjtw7K4Wu+dJv8YyPIETiEY5N7W4AxYevPeQXq3ztYp3jeKoWJNfR2vTZ9gHD2q4ibSPwaKW3IDeegIKZeuMSW5QqoTm7UHEAh89DhILpg0hfiwdGFrJl974LBS7Y85ju8u2RE6C7ABJh+AYy2Gpa6K5EFA9YMnDlCHIAh5yXUflGxEbJ+cSgZiNb\\\\\\\/zJiFUcNnc+XX3eAzrsD2JS0ImAAfdQtCEbaqZBWiv5o6fbK3TRAYd0a7SGXqGtEZVPhZPTQWfm8J17ITCZkSyGYgtm9R+TNOMaq17XabUOOpfWdZ2f3i+vimuJf5gbqgy0\\\\\\\/XUzSlD6v0LIIn8H6jeogshX75jSiC1Lcj8qj5LdumaZUgxq3rPFpZenuRBZoOY9LyiRzQbaX1\\\\\\\/VJtQh31R9pEaHXSTQAio1KGRwJxxft8V2R8+ktREz4J+4OUJV7M0f7ZLl1gCn66h8Q0xVES951B2gEaPK2YSHZ7+1GzIA82M7pertV2Sb+mI+S1gpgK2+tjjhk+eZa+tVAlCDTJE58NQ\\\\\\\/rOk9eKo8oHrfFofPOuq4XB6IDrV56o0ccVOAuSwT5\\\\\\\/gwn2M3xDE36regE1sccartqU\\\\\\\/y4GnrJ7DWBN1jiIYWaLODEfd4BepfOjL\\\\\\\/1i2AXkChye6js\\\\\\\/qQsX+EKDVLdnbgVZiyLGZX9CvY4r06XGlhsXjwnNfQ9VP9TyV36bCClDanjJ3dHLz\\\\\\\/Hb1uUv7ZD6xR6+EY1ABBK3c\\\\\\\/QgquHGvqoP2HYnMgC8gE2ds1eXdAZUOzBVR737emQ71ldKicSAnw1r+s0v45EdnxaoKh\\\\\\\/RLfZ6CCbqJoa5rAAoMNy7g0i0qP+F8aFpqapqdlh\\\\\\\/fv3S\\\\\\\/g0kuIX6gWQJfLvG0z530glka2tFTlZdQIMvLdOrok0ohIZHa5BtipEZu2rwRepZFe+K5VxOv4Eluhk4M4Gq2SF7DK4ciWa2cWc6LJY53ojiVDqCmMFCXSjMul8\\\\\\\/vopujx6s6FZjDalQyI0Y3DbF8uvFEZpMUseLLFNfQF7LQnJOE5OXFaWqiPzVG0S0YIF2l\\\\\\\/oGjlxJbP4wYw8V4Y70bnXOo2fpi4ZL3qSZ4xQ9smv+bJ3Ja7OAcBAs0PDVKJvzDszQIT36DSaZKCjB51874f0By9uzI\\\\\\\/5JZX5qWNat0l1XTzBNw1nAsUoadFgJHS4aqQeph30ndmE0UOlI2tw\\\\\\\/5wkvwoXsIJkIHFA9gK\\\\\\\/5YVfwDcMzFZ\\\\\\\/64n9srQd171kOlJI886oLbzUZxvxmKLZ6DRLX2qEDlK+ChAXiAuXAuB4tQLSVlWHsecTpB45x797AihWSxnX8LaJF2pthIhh6hbcZ6MNDhc2ORn\\\\\\\/x2SoVh3pJ0dID5jCMbA9\\\\\\\/ko\\\\\\\/WQmTybmKUPnGysKDWucblsFtcoKVCOTb0wLOXd8vp6tQdEAr8\\\\\\\/788DYdzfqSYwJ6BZAk+isriLjd3E6PjES1tfmdlJ+BkZGSp9hX30QJMonw+z6IEPAwrGXe7jxe58IsuRhlDHViRP4eJrP16g8XgUGfo2G0OzTmyuABpPn+gYfgGxdr9Hy2MrSaOzDiOoYILy1GJXWLdUU1D2kJE7rLro8JkJd6EYek84XK8CJX0bM5WTyE1TjqoYNJn\\\\\\\/izDvvN07qfT6cn88P84fPJp0vufKdUZB8n8+8G36KvMgSh5Z5Nlg9wlYveNqQKEfa7m8RypI0wenmv+Ep0iQd6UScG\\\\\\\/9IlcQqizwIu3DVULLrsbTk8hJrttJ\\\\\\\/61dob0uN8g4koCNC4OyP4Mwwjoh9Ib1sCls9gLuGPhMRZ1TO2wAbEPxz0ClkY9IhhyQ2Bco0XuX\\\\\\\/InKK82W4iQWm9JdaIFR60qUkW9hcHeHjT+aYR8qupebNEAKvcW3tICVh25DL1DLsQhU9mhrvMlNzPS+mHxlYgG2HrUIN9aTag6b\\\\\\\/nb3YyFN+QAknpGBTRc8KxacdR4r47ORty7tHMYgqGloxtQ7r9QHNu4wgTSzm8aVQWGOHKnl6e1jUVONwV9ZpxohYfpix2t6CHsO0A79AkywXdPGi+2fSCt6xlveE0IUFQueM4vsCVEeA4v4ah4Gc0\\\\\\\/S24LoWAgqhER6Ti2c4Ju3Caefj0cGnGo3Zh7\\\\\\\/0AzISRFNXDcD8FgUcqGUM07O2lyAHOWkUBPiJgLFhMEyHgdY69UEDO3TED1TeuLB+PcLjSmjclGvBt3uNnVFTVvwOovDvPwzzinG8v+CkNJD5p5CDIILKasCm8pxfBWU1IfNc8k29IcOlyRcr794\\\\\\\/eaZm7\\\\\\\/EbNfe31RavYZ63FSF+sv+UuCOXjJ+o9J1qLEljsRnG9UBv34irPi\\\\\\\/qK0vkNdtuRkWr1G26V9CFU6TBywnQTQpaiJyEQ6vjA1deeXelFOBTWqkm5Nh6ZplIkbklH+8P3fkCLDw2hhtsWmG+KIs41Mh7i1Rfn4MlrNR13l12EAtfjuc4T\\\\\\\/BiCVarWJk3MGwFpKzDfeKrJunwXSENmmS2XRnnbgnqFrFn81vZL428s03k+PLOyzOXNC\\\\\\\/LoJ9gaGHLWPASTAzo2Lclmsnb+J9flq1Ugbcp6yYEUpgAfTUe7Zp8tRA2kZea9OoidSmNQNE3x0b3lf+QeBl\\\\\\\/S1faGbcRe7ytzUXoT19tg8n2yAmVBkoV5Mq9kOzXChOxTtnVvmwWGOKG6+VpOwKdhCgy7fLfLYL6Nn\\\\\\\/Q3MHZ7c6CvWfSuKpZWEdhsEVH+eriQ8dMCubAFe5eTsYli6PY\\\\\\\/qSmDGFDba+F6PCe8xnQzZTWMtbJKkv7Y6AOLP\\\\\\\/aPtC8Eyw90IWyPIAoKgPZ4psFy9LAs+UnudEQ7QcCq4Vdtmg4dRRyxrCiDfjujwOqBeYmoyc\\\\\\\/YT9QlLwagRL940WFkLaAG3MEFy88L6jOuBnGLxtx4xUSV30ygrfly4eaPwyw31eqhYvMAPiyf4XEifzk260oUVwRIqDUYwbVjQxR33z887aVKOmTSo99f21W8NI7gyM8fUA4V+gIp\\\\\\\/eyvtFT6TkQjwexHxK+Zw5uvBjxuPCt44wKo2NQEwbcwkBLbdPSwu6jPUqqwfsmnFlUa7dHD3tm00jROeXxq8FYCaKgQiVgPGAnxrX0P\\\\\\\/1fXCHL9ze\\\\\\\/uJ+KGvenBTAfX13+YnPwPwC4nlTi8IY\\\\\\\/A4O1IBtTBpWhUhwexYYcqaFf0pTIlYSQz135R9bc0JeeDg0JH9Fv7EeHeAi8NAF\\\\\\\/UfWz\\\\\\\/NFagy1yw91tyAOxKJLy2ESWED8JRjPvoTJrkHfPSQIr3I50VkVNX\\\\\\\/CcGPAizbteLvBdpvww6hX04EiLNzUTNpecON6Jg4Bo3tMsQfbbq5JcFKfoTdyjNUJNa6lKzLq3bgaNe6e4ojuMxAfPPRG0Oc7vjmjR\\\\\\\/nqenht2IWQFnVl4TJa3AeOZENI+G2Yc7EK0RC+8Jsx4bJNnvskg+iPmAt30h5x5\\\\\\\/uHWJBCqBrQ1jhwiSia38jBXbiPpGJEjFBRx7VbUfE3jCMlEuJz0UdLpYJWSR0\\\\\\\/mibLYGUVETdu\\\\\\\/0jGKE\\\\\\\/HQaDzg3Weg0pLwg\\\\\\\/CVs7iMxsyrmrHz\\\\\\\/J1IZZUzAiOhqu+L4D6vId3QtQyArQosyy6tSHnLFTgnv1FF5e9KntJ0uGWRdrHvUy5IBi5pSR9YsG1GmU8Au783FncOvwO0EJJ6yhemmG60ZVdqpGRSiV56010UzecuV9eE7zN48kO4c0Kpc6qoxd1MmwYLyB\\\\\\\/FErlGb4E67Cx5uzfeAjfUp9OCN9XNn5ZuF63avgiaRtX1zWhcVaWPogfCaSJRKkMKYEyzmUFC3atNF3eBbZZ4OMT7uzjoi0Lh7c8NV0JhQgLHoaJ3enivmbzvrgEhC84w0aBDRw+9ewVcNu8GzPcYECHwBG\\\\\\\/Jrv+PUHgpFBnWOd4HmmCBMoNezEs3HXU7LLGSgCmAh6ZWlhGXURXVBjziMFYR1v\\\\\\\/O2P8tKrE+HPIRoi5+RgsxDuTKQjHI1tDgWa5rwZ0Wg10SGtDqdFSQ9fB1wp9G9B1EoPekmAqOVLK\\\\\\\/ckQrSBUAAv3P6uIRnM5PBwhY67vi9QFT9KNboqacxxnBR1M2RWBeISphSOavMn3bLa85YNzHmsl+miW5pchCpjlikuqX4s1UrF+yhzxAbmPxFFXZ+0ndU8CD7PhVogkuSAiEfuGUGXofrSFNz2WWPvVXM84XNFE17Yzg+WICpURtAsEhvpfjk+Z9eKOunlKOp363saHYBxKcVDZXRR9L66sUFJVFpNOBQE+VTXjFpuW0dsws57sliNISSuWAxV6LYsKo6vHrPYG3VqSOV5i6crzx6myqaXqiIkErvR1Cc+ASnpOWsRFEbJ7\\\\\\\/YzDMuinib0lQYAwVTsK7eVa4demv85foap0FCwUWJCvnsAldE9IwrOBVtxfcbDDsvze24tc93pfJTgunCAZfON8lpS8riTDvRMXieJg43+NcIJ1OMCNqp9\\\\\\\/xdEAizdHsxLS9NOM0DOuusoVGO74kmGqNJsVX4F2OGoVIdWiaZ9caV8+Wn0xgRr7dZN\\\\\\\/ivqbDaEUlJlcdi3rnNnV3UJIKDqXUAaIrq7JNMH4bbxU5XFyqqWxvSElfpC3kUHViQzYqkDsH+HujvZMQ7H2FGnPtd0lgpOKug1CyhEZctKFCJ8tRMNE3Q\\\\\\\/1U2yv6LU8Zv5rr+9phzQG8NUWViKgQV1OKN4qQjnVsx8PBVAi6rLQAMF9XrtJGS8vDvpUKi050auKJ+dn4nYQLK4f\\\\\\\/IBbDqA0SkGspT\\\\\\\/ck0Nr08HkFQPw\\\\\\\/X7e4lPigpiSQVWYDqNvU3QDT+yA2B2NojycSR4+38bmHuHBxsZmyRESzhwFdKvy0Y2cogKwV6RBFSXQtd9iXM4xiGzwk0+er9wqd1zbuiD7MOkEy+B50Qyhp66rS5wnDZolVkCFJuubouoj08QoILuLRw1iaSF\\\\\\\/TbS0e0uy0uHXJjaNPktMiwiG4lCDXJJBWrSHx5ab8U2JKrrcLTUqmvkEIc6hrBkSV04\\\\\\\/LJ0bAhktWN+gSwDdOfM0r2nTc7ZYKbf38+WpF2CId1\\\\\\\/C6rRtUYGUVIWrSJffhRMDA3wVFbkFvG6w4mEtjv6SyjBR8QWhrLmuj3zlf3y33ZiVOQ\\\\\\\/fGMkc9ikYLD6Rtf+UeCVY1XgYVSwyU3jgNnjBjMlEbxNiJ4qkvvOHboECuKm+PZ0n8zhAZneGr0gELzzOnYBpZXZDJXtPyV7QXfeoBxLkDGOciBFUirEry15xLlimv4dPdaHWx2AnSWEynIhx52UQJbhk8t1iCjA3zyH+UinJhwy9cbUUxzvTq2WyRKiay9aHUs1Xk3mXYg73m7pObwr4E8\\\\\\\/f0q21pufqlk2sO2794VOVAQdIT290zZ0qsB4hqeYveZ1eo5Y86HvCpUahsrLX1OMyEWGRcptK9lZcGknFRffOOKdb15BTMDGQKoq60UulVClx2E23pMsPDOI1\\\\\\\/a5HCv5LhDuk3hXcEHYiUV1S8UrtgBHkbrHWMvW8xLFvZrlchwcgreYmpli9ufMgNsnVNx37Kft1BzsZZkZArSc0JYzS96s67OI+YPEjxtcoU2YFcvbKLoAEEU0PYYiAVr+bEfI8GkU6jDP\\\\\\\/DFRDLXxakcUxqrwyKBSEEymMzjZc3aRyq46a5\\\\\\\/knxi5MUVto5rRi9ZE4oeRvQBSxBYTUsxiqPOx0sj7Cx6p6NIPyU0Ku8Li8Vv26dcTJFF07JteaDoBJrtWUjTQ1pY+75AHFGClroV9+JgvtqKpCrmzK6BFTZX+urn1mQ1FdCgP2qthbeqN3Lt6IzFVzX\\\\\\\/W9+\\\\\\\/7d7+w1\\\\\\\/Mef\\\\\\\/RzjKgMrOHgY5wfesUFBeWmLPKWWODQOhprllJagT+5vbXh9TetjY3wguhFXSId7\\\\\\\/yS59xf09uHjqF83Qi0J+AGsU6I7sFtViA5JARdoeTpLbqIljBMWItoka8At7mKt+zuaLz8Og5asSg3NWoq89MSpn8m87YaCxCsrurwsGoZaVOOPXXkpiAxiPbr0ECDxCikbj2ss\\\\\\\/w+2PTqt8CW1oQGaYrNt5RZkmTH1AQB86SMqsZUHAz1Rc\\\\\\\/HWABGKykLTDK4RmpnO56KcmfHyydpF4KYhqUUvIyvRg56Ec7tbylmBLnpv5XPyCJSbGjbKGE7uG+ps7oKY3OgKdlsiaNuktK68lijcVo+j4Ge9RURrtIDnAJlGLYIjAwf+IHxuluFQi14cUuLdNqBJaIAO16nGr4vkF\\\\\\\/OUz68W5+5M7MOTOOAIMG6Bhggd7PLGnYPL2D3Nt3v6Br9tCVZeay7DYN5yvfRlJgf+NonAN2nXLllOENVnIYKmOiHuRLYwOxWUmn6J2L7OrltA\\\\\\\/lPnJIwHa4ll5UQTwbT8aIep8fcS5y5XEGk1kjWHHKzC\\\\\\\/jUBjH+CNVsHqOwNNJyIfYo1rIquDzVZkyz+usQcKiWxkSwCFCHzPpWJ3JgyYzbqndRKGnv4RADv3gemNy+HZr7mqJW2VcwdlZG2dA08OjDBn6P7Gs5dyEL9zhdthBBUdMgSu3G0hHbEJhEvecWJI2qTBV84e7Ysj+PWjYg1EejoNgN2OPfCWT3Pg+oGwj\\\\\\\/cOsvxT3KQNsf1L79gjXItzGgbGxPXZ0TKvQCMiViuNcoSnwaq0NvDgI0tTA\\\\\\\/XjFJePdVFD4uz4gFi\\\\\\\/dp04UhWH0NlnoUrOa9c00RpCMMMxJj2hyjJAaShTASZ2j\\\\\\\/6uO9M1EJWqzr1P9hlQgdfESw5snrE9gnHXLGAhAsiWEOSTFt+eIwsEnX8MEWo+DqUlz7zSTXkxwVa2iWD7gQhSe+gvFs5RPHS+2IhrKupp1QHFbEbp2\\\\\\\/l6kwNC5rOqN0RgZv2OfCfeGPMOmr+xBgOk1DvWdKIVmbnOEahmYOL5qzfqRn++S+U5ioa0DUqt4qZh51lXgyEkTZIrC9K8C48SpIs1UB9Co6wh8BORn3ZuKBa+cEGkUq6wRS3oC23Qe2liaWjmzlS7KHX6kZzKwsbGGU\\\\\\\/TF5e1z+0H3AQD30CJhqWub1HuZXrxA8HEVBe0\\\\\\\/j7QdwjzTwNVRV1W4rEHe3NAgY0o8e6Rystyi\\\\\\\/goNrVESBjOkV3hTvPpSAJlMgtCSrGkX7oqP8Gsa2sHx0Ltmy1eV2W5LIVSrvQ\\\\\\\/FSlcdAR0Ya2p4VFZAaELIBOYdr\\\\\\\/ZscUGLqNKadiZr6tq4TzGAQgAVDVTPM0FThTDy\\\\\\\/RnIKTfoIqyuQEKZhZvwRrh+RD5LoMS6T3dp4G17HD92xzDIQW1IVUHka2+1QXbMHMj1gm4jeKG3J2xZjo+nNsYbqfYv1LPjCkMvUxYR87iOURHBm1viNG3hZJeHqGiGMwr0qmraJAxXgda1f6RpVQ0Ln33DERwEtnjHOdzfMfQpZnaaKQYAtx4jzjXfrgfkcAGO7zVTcPAf0Aq8KEm4EqpJPaE5GqfRGdNkCXOvGymMjf62U7GmHDheL80rMQzssso1xxBbXa2OmBuCt87k+mfWo5l1QglRrwa8PMek3pn7F9OxbIhWSDijJ6fiY4iofbSMSRkp4MkSkH5no1jeYUHo9wIvo49xE2NvzzRPNgw4MlkrtJPxEuQMhJvYOvFQtvhRZ3u6eLm\\\\\\\/hohJdqjdnWYBnoliymZVFcmINMbSik11wlVeDq3MtoKngh3E+lCWYxvdAyHX7JWmNKj+Mw0i8fySikirCm0gNqVRG0Qg5p6c8ptrn651fozYoqmcpRJ92dqfC6oH9LvRDKY7gs6cATm7UJb9R0qTnAYMUQZgRJexYEf1b0FuMAPcxE9+uVQsq9hs2gPbc58llSn1exOZ3n076Xrpr4ySCAq4pECg2v4ZSZpjkyvsSnW51bOqAsfAnWa89X8WvgdoQr5Z+qTMyVrvn3KCWC3qVdEG1pNbyMc6ECgq+Weqi7SShMr3q0+dan2jmtepsgvpjvxvS1R7GQ9YSZ3xQDcW04KbWnfhhV7vLsziL0hnvhUmp0tqLXzRBGdi\\\\\\\/LFv4xSSwi8AKZ4o\\\\\\\/TPfuPFAK2YpjiO9WB9qSxoP2q9z9mhEwmO\\\\\\\/5a7eg+bnW38dJFjSo4OsUPgtP3R40X19HXFW2+iqTHVJQY+n2dNT9rJjuW35hAvQCFi27jgBaNNVRJ47+huHcS\\\\\\\/YF4Pngf9HSlS1OWsBmO2Yz3mK2lKJNwHQWn1mmRdV50BcARVeCY7gK8slzwCaL+JvbbOqZ9AvmGkTeiXjnRN465q2xH2riEPXWW4GE7IId8IfpqdcUmDmaV1cpcmMM\\\\\\\/43EoBfg8opWmWaD549y6s9qSpzH9YHe9Z7VYZ9zy8AwvcrL7dl6BKtBLxU1Su5vCyrsxZt+xEieG0yrM5UDK5aCU4YNFC3lVWUaONnNzwfOc\\\\\\\/qf9I5+kVqkMEhsUisWrE7Gw1ISKT41f2YtfepA\\\\\\\/5wUolZY8a0CR4nNTZKCZHd\\\\\\\/0djiSRZ0wt7FyiVph5TQYaeY9JuBEVkRA3WblDw1PlT14wSk6amlpJSHAZ2Ojl4wvDXdbcWsTdQxFNoTXDCX+XiIH5rAXVJ1DEBkSYZS7rE6hVj9Lvr7WSf38oxPGza7ViTdov0pMi\\\\\\\/+Tb13Ln9InXoNoVXpln5J2Jk1sLZec\\\\\\\/QfYAJ2RyWPFgXIDenkEl3eEquTV8oJiNF176fnS0l1C1BvRn\\\\\\\/Y5rF\\\\\\\/tqNjpCjstEl65UJ+6g54NTF1PWGm7qRZFe\\\\\\\/p7eMgXDlCOazJnFXnLY8Rm8bwxIS0fQQr5eKH3eV1ATCDuk05GDHVK9gylXQkbsv96C+dO5cvSapD5S7qcXaZuJv7yEjwsrypHHDixBoPpRFKXl8H85eUP7qIpj\\\\\\\/aAr8pWIdlM9+yR\\\\\\\/R+qughzBEt5jzUTZhUn4ttWlf\\\\\\\/+8+a8U38WJSZUHYB8t2qtC7pm1\\\\\\\/MUZf534hKgtH4gigmtE1bp3S9YkvUYgmOZde7LIndNt7I1HxHqDjTarkZr0jKJNo8DEXBxE\\\\\\\/C8orpwowcR262ld67kITeDtfC+IWJF980Ggl9sVpVxZdp27ukQM+BewWaRiCKi1l7SiVR2Da+G5ejwhF1bwS95jvLUS42Jpa3AJp6oVkboXAndKbLxL1Se5btbZGazhSMtENvO7JU8PsukAy7lM04OM+nuTxKMOUjI+HntNRX7bRkZbLCkKZojfln9xps41v6jK9y9eSHuA6DzoBNlwkXhpUcRCv8qn3uilDkaJVAlAJydDkTbEM3CjsZypTLEwNY0C50UuhakCEJS29O76cPscskjca\\\\\\\/FBpSwpoo1eQtWYqBzFjiWF5Ha+2jDvChweEvbvW0LSXp\\\\\\\/WnrvJR+CV4AtQuaRPxU\\\\\\\/qrPmZqk4\\\\\\\/peh3NPDlKCzJFUJ67iwS8HoCENpmt+muHXhpHEVlb+VK\\\\\\\/0k3uZv2h3oCdBSkEt3yDESffEFlxJo0YmH6wqJwNx1XV3H839AinLpkDbHSx6u8FwdQFNHg2I3hZn7xJDp\\\\\\\/Qo6OX7ZlSFduzx\\\\\\\/wPUokjf9mLhEY+t7TwjTy2pJuU8FpG4BP+o6AOj\\\\\\\/Rs0DvXh65xYj9UaN33CwhmgPPTZLkrYrklDwToO9z0wSBgIStwRWUzjoS++K619mCMwkw0jUIJEG8wGnKatvXfX8hzwTs5am9Z1olCEkjY0HvTvUtGYt95\\\\\\\/im02ir3wpicQLePsTR6OS0Wps5fEbFJVqwiW0UElVuUqejN8oWixiJZ1ZVI3OnJ4NAGoYCkFVnfwHlufxGRaPKfM\\\\\\\/MibzprurbybhdIq\\\\\\\/6NySRi80unSqGeXKoKtPfetaMLgTUwflYofPlWqwoxvYdsGTa+7nPicgGzAcsNoJEX9osWzjkEWE7Lhkx2WD4f\\\\\\\/TTSe1ogMf3fbHy0MT\\\\\\\/cqnTVm7mm1ZHh6h45RCyKJBFX+Bw3EWJ6pe2kBCycRElmmYBd7YecRplQ0RpQB3jvbe8\\\\\\\/4IIJbXnmqpyKHN7FIHFe9hQLEun7oNHiyEXJ12bshfk5feqMxHORK4d7HB0ovkxFRAlWOROIAtq0uOulgnBpKrH09jz6JMBh3d0+BE+JIc+kel8z9FFj+uOnxxd0QEvwYrv9Dsb\\\\\\\/Apj5eMOi+baW60AaSy8V6JcPqBOk7v3Qzz7LwhCGafNB8oSQx3zFdM20+92lDoMHOUemN\\\\\\\/3eZ4xtGfhcDEzdcvrWrI7zmKXJ7SXQqbIhnROX4rtNi826FHwYy\\\\\\\/TRR6cZ\\\\\\\/XXfw01ogVJVJy1KW+72Rbur\\\\\\\/9jK5HN8LlKGW0uuKAfsXPhM55WZ\\\\\\\/KTkPwDlSuJ34hV\\\\\\\/fopD1c\\\\\\\/4KT3nAGY1guSPODygz65Jfz3UHqgJysW0xKnxzC9Mg9Ji6CbyTn7iO09SQz8R6TpM2IgN51WLSiyVNB92lOxbcqlDUDIkfe07u3ZDVeGkpQw06In1ck9TnjJzJ2WmKq09EE9FS1nbHALYpA+t2ERutw6OIkezQM\\\\\\\/bN4jY5mgk37hV4VmX8YCeBq+spnT12eafocoBeZF7vFb3PQM5+DYv9Rwnb10qAu9i0LupAGph8gnLMLyaf8WmECrBq4RLPrBs\\\\\\\/ZNWBsl2fnhHE52QCl8p5rtvqBq9l\\\\\\\/tn3orUdl9iuy4sbGnfg9TNIVfLkn\\\\\\\/8WtEzrtHWUigN59yX+6sW+fi1RTkb2+t210tk2k7reLrNkVIlfdEzElS483KFyFqYznJnC\\\\\\\/j6X28+1M\\\\\\\/umUGD0z47NzSPlGWB\\\\\\\/+WTUgLoKnLN1VCrgV9kWpw\\\\\\\/bFAsT7u4I7qhA98VlNTU3j6BqoD0B1IUN06EGavq3q6GtMheUWmHprT5z4rOkuN1TCj+V0L1mFq1x\\\\\\\/9\\\\\\\/Q4f2fWisUZDutdLaiTLHstkyGqz\\\\\\\/dzvcM0FtXM9o6IAXrByaiIHNIlDS5GLyIV1dUs7Ya\\\\\\\/YZzMo1AeKRDbDSux615NG+v+6EHoGB84YTg1gpEoT5NC\\\\\\\/RmuO4axRVN2+wPAt+73Ty2ChTXJMg0AIkTe5GOVQ5oYrRL5avwoqR8G+g0DHCAVSfearVqX50veRy+NVX1onMiUdgp8ENvEn\\\\\\\/zZB2mfr2OyoucV3sJ9sRj0CtdpDfetHMCy9DEBUR6YxAlZ7WzTCdWahWaWqGqsrKPirVvxN\\\\\\\/\\\\\\\/AOOt+CDtaKx6juiy9YtiN27knApme\\\\\\\/cVwpGgmG0QWjCYnWr8B6vC9ypq3lCy2Eus9xQ87cNy9NLdsH1b6woh8qjDeguU0iAY43ll+gunXmDqL1CoVbn\\\\\\\/171QduduRRdYLPcau77fXprjWgoc802R+h34uzebQtC\\\\\\\/oFjHP4f3UABrRZB\\\\\\\/bLGs8kcY09sS+bRSN6izfFdjvx2GUmlJRJtVPhsQoDcie3GeqW3LOit7NGToXqXLxXm9R1DOwd0ufvgi+2wVDe6FOgippf0vSS8VRYauZQW3AgfCEyUZCQydOVbL0YW\\\\\\\/RYgyBEaR55fFOV8sqHe+1ftAcI+3dDRt9B4gBukJ97nJw7gbnv4pTJJ0uJ9TC5shRd67mLMgTepK4P98BRGwhkaGTqbZj73CjB8H8ZeOV1IgMAbA\\\\\\\/i8ZdGNjuHd5rCXCTMlPajJI8F5GqDf2IWpnI09io9cRSfGAbRBNB4R7nQVhhg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/d96CXXaPtfHXiDLop9Dng==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e294f7c54c74cd2f94bcb031397035c92d852077aaf0e76a205bdae8be837280\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mrE2dUYofCjofFdp1V6v\\\\\\\/pHmrJ6TV\\\\\\\/IUOj4wB1denKlZfAo3Xloh9KemZw0zaF6GJ5RjiS2OkHV0iLHk+fJfD\\\\\\\/fzbWZdVN6YKN7hg4vbhkjKTnFE0pLlxSQTWeD176l4lrkBD4\\\\\\\/WKZiwzuFgYQyRAOLMAsP\\\\\\\/Rlk+MfdHYyaKXqhxjFEPNL\\\\\\\/f55tLYRIZ4knA2F7dOYEAaATWaDzTHgP9r9C09UaZth8IZwwCys4x7DeDXiOp\\\\\\\/FrBcqjMdXCI9hVzGMCI8XJhUNW87WCK9vnlEulZOoMT6AAJvyJQz\\\\\\\/ryMjSglqpRc\\\\\\\/VkR1QzEfVutRxXf6F+ZudailFDB\\\\\\\/LuvD0QxfzJiaOPlf6jn1wlEzPwnQIomI9V0hnrQqgwmx5FMY2U+RdaVRskcKSG9w0Tsckz5772Gj4\\\\\\\/BkOk2zyHXeM3Z56NMVItRTirfhjJ3q2bi5G6\\\\\\\/frl9hG0epw8zzpiW9uZGDEg2TaDtGmahZY4JGSn2y4lDhaaiWMT+xOY2GryXAiLfyBnIkerhJItK+euT+NeEO9gEEdE0P+iMT46RwaUUQd6gFfwNlxX1kyrVMEM8Nl8\\\\\\\/u4q7Iuj3c+4Yo4JVYNJ7zGDxUHEA1zVn6UC2eNuMlrA8bklKcfksYseMCkZ\\\\\\\/zbIlBazlorQ+HX58LER4vkgEth6g5bKsojvYgCbLKwgjO+wd7tOEfA6BCbjCUR1+SDgyJsbz1qrBZ+\\\\\\\/plRXsvtAkFojiQvD+EIJn\\\\\\\/AuGVKDF7vljzwMHtMdkSSSC7uStVfVBGd+4PmNbn3rfssiQymsqIYQ2M9t0j9PekHu70Vl17s6B6DTRwgoqQkme6Hi1dL\\\\\\\/RbjQgAa\\\\\\\/K6fHiNJzbc5zPZLIZ+2pV7gfFHZAK11z3PXd62yL8cyk04AuTNtl6NavsUaBlLNKvddB0j5h4qrEhRDbRktZ60L0GeX04c2caoihidI\\\\\\\/jKTFMwnyrDY3bGZSs+i09\\\\\\\/NBflWMLAfV2TnMn2mADm\\\\\\\/9dAUqqqrmkW1EYQbaADLk0NOm2eniXlrNpZvQl3ec4HZUC9RlxNqKyae1qRJciEh5Y9GJHwaweuilfUrv1ASn20szFUR3POzF7lxJk5KFm0QU03+QlHogpczcsnCBlAS7Mvd6dK8SUD5E5jwSJkxUcDocmMcuiuaVLyEuJK1HJxEbmcMc\\\\\\\/BhwpGycBaqDXrSLSjGA3Nz8VBoWTDNy673qZKkkT3cvGT9ay3bbXf5FD\\\\\\\/YAs9fr4PcCufU3ZMgKoLTkBgesMu8OQXPOnStj+QI9rJ+ivTRq9el4XDrkxBICVLtiHL+n7hC\\\\\\\/nryUSkN6832f+k+yP3II9O9\\\\\\\/cgassgZrGNm4tJtMGK+QSXZz6V+\\\\\\\/g3j5iBGBNquWIb7A+OYLsnOQ0tWOAwD1m7P0I9AP0OMt3I\\\\\\\/3qJ9omcXsqmh7VBp6b3IZKN08n7HOeu+cSP9Koxo2D32PuY\\\\\\\/L10LRYUPbA0UyXSHxshp7g+f2JVP4wQUcdtQv55eOtkg6F3eDAOUtjPbERsj0xMDnmRp3a\\\\\\\/N2v2YvKbosGZQSf0fBSsmG4au6FGxeD+75CgrIH2dW6gqbQwupXEWnaipbwTg1gudwNKCK4r9XBpdLG1qfsVVAy8ETc9B0qKpXd3ZU2cG\\\\\\\/OC5QTS9fd1ZrYvjADedibClh6pBez36m2T6\\\\\\\/acB+\\\\\\\/2YKRy8+X2A4GuLYw1Y6NLn1N2kPojkWXdusUWpUg2nChgIUGSSz\\\\\\\/0a4E9JVvasfKzfUG3AFkX4TBXqO\\\\\\\/e6IPtwz\\\\\\\/g1O\\\\\\\/4DQHyZVy1wRdlYRR0\\\\\\\/jqUriTvxfgbGIxf6gVxbqJ85uTjOrQ7vd+dxqsEH2lGhg6opc0zuok9DNNE2rmiaFwVsjxGiy4dWQle\\\\\\\/tkkW+KPOsmG2zpD6aaY0z7igCDeLUdl04vmlKQGx2FZyYdFGjEWbDUxx93utrNVS7DKujBh\\\\\\\/KyFBD9\\\\\\\/EUJL70befQu3\\\\\\\/sfrfp4V8Qmns6\\\\\\\/\\\\\\\/jblU+g1SGTcG6DBGIyXv8YfBXOFvIHAB0g9AvVQHc+Fvfwhw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022p+tAFy0FiLhBQRUvxE+O6w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022eaef104738059d528b6a0b17f90db308f87e41638ee4629091ba8d1ee31ac5fe\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460124 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/1063" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3g7JvPqHTPyzD6sZrJx48sSSpDuUH-VYxo7zpjLXadE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/1063" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-G-9P1jQID0b84rULIX65SPB03dGXxch9bBiOIeVc6Ow" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/4/1049" title="Accuracy of Genomic Prediction in Switchgrass (Panicum virgatum L.) Improved by Accounting for Linkage Disequilibrium" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/4/1073" title="Embryonal Control of Yellow Seed Coat Locus ECY1 Is Related to Alanine and Phenylalanine Metabolism in the Seed Embryo of Brassica napus" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460124" id="node460124" data-pisa="ggg;6/4/1063" data-pisa-master="ggg;g3.115.026682" data-apath="/ggg/6/4/1063.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Large-Scale Survey of Intraspecific Fitness and Cell Morphology Variation in a Protoploid Yeast Species</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Paul P. Jung</span>, <span class="highwire-citation-author" data-delta="1">Anastasie Sigwalt</span>, <span class="highwire-citation-author" data-delta="2">Shinsuke Ohnuki</span>, <span class="highwire-citation-author" data-delta="3">Jacky de Montigny</span>, <span class="highwire-citation-author" data-delta="4">Yoshikazu Ohya</span> and <span class="highwire-citation-author" data-delta="5">Joseph Schacherer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1063-1071;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.026682 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460124" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Paul P. Jung </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Genomics and Microbiology, University of Strasbourg - Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, 67083 Strasbourg, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BP.%2BJung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jung%20PP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaul%2BP.%2BJung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anastasie Sigwalt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Genomics and Microbiology, University of Strasbourg - Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, 67083 Strasbourg, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anastasie%2BSigwalt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sigwalt%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnastasie%2BSigwalt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Shinsuke Ohnuki </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Integrated Biosciences, Graduate School of Frontier Sciences, University of Tokyo, Kashiwa 277-8562, Chiba, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shinsuke%2BOhnuki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ohnuki%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShinsuke%2BOhnuki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jacky de Montigny </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Genomics and Microbiology, University of Strasbourg - Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, 67083 Strasbourg, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacky%2Bde%2BMontigny%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Montigny%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJacky%2Bde%2BMontigny%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Yoshikazu Ohya </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Integrated Biosciences, Graduate School of Frontier Sciences, University of Tokyo, Kashiwa 277-8562, Chiba, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yoshikazu%2BOhya%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ohya%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYoshikazu%2BOhya%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:schacherer@unistra.fr" class="" data-icon-position="" data-hide-link-title="0">schacherer@unistra.fr</a>
<a href="mailto:ohya@k.u-tokyo.ac.jp" class="" data-icon-position="" data-hide-link-title="0">ohya@k.u-tokyo.ac.jp</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Joseph Schacherer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Genomics and Microbiology, University of Strasbourg - Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, 67083 Strasbourg, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BSchacherer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schacherer%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BSchacherer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:schacherer@unistra.fr" class="" data-icon-position="" data-hide-link-title="0">schacherer@unistra.fr</a>
<a href="mailto:ohya@k.u-tokyo.ac.jp" class="" data-icon-position="" data-hide-link-title="0">ohya@k.u-tokyo.ac.jp</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/4/1063" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460124" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460124/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/1063.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460124" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460124/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/1063.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460124" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460124/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/4/1063.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460124" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460124/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/4/1063.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460124" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460124/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">It is now clear that the exploration of the genetic and phenotypic diversity of nonmodel species greatly improves our knowledge in biology. In this context, we recently launched a population genomic analysis of the protoploid yeast <em>Lachancea kluyveri</em> (formerly <em>Saccharomyces kluyveri</em>), highlighting a broad genetic diversity (π = 17 × 10<sup>−3</sup>) compared to the yeast model organism, <em>S. cerevisiae</em> (π = 4 × 10<sup>−3</sup>). Here, we sought to generate a comprehensive view of the phenotypic diversity in this species. In total, 27 natural <em>L. kluyveri</em> isolates were subjected to trait profiling using the following independent approaches: (i) analyzing growth in 55 growth conditions and (ii) investigating 501 morphological changes at the cellular level. Despite higher genetic diversity, the fitness variance observed in <em>L. kluyveri</em> is lower than that in <em>S. cerevisiae</em>. However, morphological features show an opposite trend. In addition, there is no correlation between the origins (ecological or geographical) of the isolate and the phenotypic patterns, demonstrating that trait variation follows neither population history nor source environment in <em>L. kluyveri</em>. Finally, pairwise comparisons between growth rate correlation and genetic diversity show a clear decrease in phenotypic variability linked to genome variation increase, whereas no such a trend was identified for morphological changes. Overall, this study reveals for the first time the phenotypic diversity of a distantly related species to <em>S. cerevisiae</em>. Given its genetic properties, <em>L. kluyveri</em> might be useful in further linkage mapping analyses of complex traits, and could ultimately provide a better insight into the evolution of the genotype–phenotype relationship across yeast species.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/phenotypes" class="hw-term hw-article-keyword hw-article-keyword-phenotypes" rel="nofollow">phenotypes</a></li><li class="kwd"><a href="/keyword/fitness" class="hw-term hw-article-keyword hw-article-keyword-fitness" rel="nofollow">fitness</a></li><li class="kwd"><a href="/keyword/cell-morphology" class="hw-term hw-article-keyword hw-article-keyword-cell-morphology" rel="nofollow">cell morphology</a></li><li class="kwd"><a href="/keyword/genetic-diversity" class="hw-term hw-article-keyword hw-article-keyword-genetic-diversity" rel="nofollow">genetic diversity</a></li><li class="kwd"><a href="/keyword/yeast" class="hw-term hw-article-keyword hw-article-keyword-yeast" rel="nofollow">yeast</a></li></ul><p id="p-4">Population genomic studies aim to explore both genetic and phenotypic diversity between different individuals belonging to the same species to obtain a better understanding of the mechanisms governing trait variation. Although genome diversity relies on different DNA variants that either affect only one DNA base pair or extend over many, phenotypic diversity may be investigated at various levels including the molecular (<em>e.g.</em>, transcriptome, proteome, metabolome), cellular (<em>e.g.</em>, cell morphology), fitness (<em>e.g.</em>, growth rate, yield of biomass, colony size), or visible features (<em>e.g.</em>, colony morphology, invasive growth, biofilm formation). The budding yeast <em>Saccharomyces cerevisiae</em> and other closely related species, including <em>S. paradoxus</em>, are the most often used for both genetic and phenotypic variation surveys (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Schacherer <em>et al.</em> 2009</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Liti <em>et al.</em> 2009</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Warringer <em>et al.</em> 2011</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cromie <em>et al.</em> 2013</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bergstrom <em>et al.</em> 2014</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Peter and Schacherer 2016</a>). Whereas genomic data have exploded with the emergence of new generation sequencing technologies, phenotypic variation encounters the problem of throughput to investigate variation in a large number of traits across populations.</p><p id="p-5">Significant progress has been made in recent years regarding the analysis of yeast growth, with the development of high-throughput phenotyping strategies based on semiautomated procedures such as the microcultivation approach (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Warringer and Blomberg 2003</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Jung <em>et al.</em> 2015</a>) or colony size variation on solid media (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Boone <em>et al.</em> 2007</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dittmar <em>et al.</em> 2010</a>). Such advances have made possible the comparison of a large number of natural isolates in a reproducible and efficient manner. Large-scale studies have focused on various phenotypic traits including growth in response to different stress in both <em>S. cerevisiae</em> and <em>S. paradoxus</em>, as well as the fission yeast <em>Schizosaccharomyces pombe</em> (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kvitek <em>et al.</em> 2008</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Warringer <em>et al.</em> 2011</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Brown <em>et al.</em> 2011</a>). Interestingly, it was shown that despite higher genetic diversity, phenotypic variation is much lower in <em>S. paradoxus</em> and <em>S. pombe</em> in comparison to <em>S. cerevisiae</em>, maybe because of the domestication of the latter, as fitness variation better reflects the population genetic history of a species than the adaptation to a specific ecological niche (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Warringer <em>et al.</em> 2011</a>).</p><p id="p-6">Besides growth phenotypes, systematic and high-throughput exploration of quantitative morphological traits of yeast cells is also possible using an image-processing system, which automatically processes digital cell images (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Ohya <em>et al.</em> 2015</a>). Through deep investigation of deletion collection, almost half of the nonessential genes for growth have been shown to affect morphological traits in <em>S. cerevisiae</em> (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Ohya <em>et al.</em> 2005</a>). In addition, cell morphology variation across natural isolates of <em>S. cerevisiae</em> was also studied (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Yvert <em>et al.</em> 2013</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Skelly <em>et al.</em> 2013</a>). In contrast to growth phenotypes, no impact of ecological or population genetic history on cell morphology variation was observed in <em>S. cerevisiae</em> (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Yvert <em>et al.</em> 2013</a>). Interestingly, the study of cell morphology provides a direct observation of individual cell behaviors in comparison to fitness, which reflects the contribution of cells living in a community (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Ohya <em>et al.</em> 2015</a>).</p><p id="p-7">As mentioned above, the degree to which phenotypes vary across a species has mainly been investigated in both <em>S. cerevisiae</em> and <em>S. paradoxus</em> to date (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Liti <em>et al.</em> 2009</a>; <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Warringer <em>et al.</em> 2011</a>). In order to establish major parallels between yeast and to observe behaviors specific to species, it is essential to explore the phenotypic variability in diverse genetic landscapes. In this context, we decided to explore the phenotypic diversity within the unexplored yeast species <em>Lachancea kluyveri</em> (formerly <em>S. kluyveri</em>). This latter is a heterothallic yeast having the same life cycle as that described for <em>S. cerevisiae</em>. It can live at both haploid and diploid levels; the two different mating types (<em>MAT</em>a and <em>MAT</em>α) can be crossed to form a stable diploid that can undergo meiosis under unfavorable conditions (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">McCullough and Herskowitz 1979</a>). Thus, <em>L. kluyveri</em> can be suitable for classic as well as quantitative genetic studies.</p><p id="p-8">This species is classified as protoploid, namely a species that did not undergo a Whole-Genome Duplication (WGD) event, in contrast to post-WGD species (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kellis <em>et al.</em> 2004</a>). Genome sequencing and analysis revealed a 11.3 Mb genome spread on eight chromosomes (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Genolevures <em>et al.</em> 2009</a>). Recently, we sought to conduct a comprehensive polymorphism survey by sequencing both the mitochondrial and nuclear genomes of a large set of natural isolates (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Jung <em>et al.</em> 2012</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Friedrich <em>et al.</em> 2015</a>). Nuclear genome evolution within this species showed a broad genetic diversity that can reach up to 2.5% and result in a nonstructured population. Moreover, this population genomic survey clearly demonstrated that distinct recombination and substitution regimes can coexist within a species and lead to different evolutionary patterns (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Friedrich <em>et al.</em> 2015</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brion <em>et al.</em> 2015</a>).</p><p id="p-9">However, little is known about the phenotypic variation spectrum in <em>L. kluyveri</em>, despite some known differences when compared to <em>S. cerevisiae</em>, such as the inability to ferment sugars in the presence of oxygen (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Moller <em>et al.</em> 2004</a>) or the use of pyrimidines and its derivatives as a unique nitrogen source (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gojkovic <em>et al.</em> 1998</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Beck <em>et al.</em> 2008</a>). To gain a better overview of the phenotypic diversity within this species, we used high-throughput analyses based on growth fitness and cellular morphology. Fitness investigations relied on a microcultivation approach, where the growth of each strain was recorded under a large panel of 55 various growth conditions. In addition, intraspecific cellular morphology characterization was carried out using single-cell high-dimensional phenotyping based on microscopic images, where 501 morphological parameters were determined (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Ohya <em>et al.</em> 2005</a>). This large-scale analysis provides a first estimation of the phenotypic variation within a non-<em>Saccharomyces</em> species. Against all expectations, the growth variation was lower in <em>L. kluyveri</em> than in <em>S. cerevisiae</em> despite a higher genetic diversity, whereas an opposite trend was found for cellular morphology. Comparison of phenotypic patterns between isolates showed that trait variations (for fitness or morphological traits) follow neither the population history nor the source environment. However, fitness profiles are more similar between closely related strains than distant isolates. Altogether, our study provides multiple insights into the phenotypic diversity of <em>L. kluyveri</em>, which will be useful to future studies of natural genetic variation in this organism.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strains and media</h3><p id="p-10">The strains used in this study were stable haploids or obtained from random spore analysis of natural diploid isolates. These strains were isolated from different geographical origins and ecological niches (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026682/-/DC1/TableS1.pdf">Table S1</a>). The media used to determine the phenotypes of each strain was the YPD medium (yeast extract 10 g/L, bactopeptone 20 g/L, dextrose 20 g/L) supplemented with various compounds (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026682/-/DC1/TableS2.pdf">Table S2</a>). In the “Carbon Utilization” class, dextrose was substituted by alternative carbon sources (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026682/-/DC1/TableS2.pdf">Table S2</a>). The minimal medium YNB (yeast nitrogen base 6.7 g/L, dextrose 20 g/L) was also used during this analysis.</p></div><div id="sec-3" class="subsection"><h3>Morphological trait analysis</h3><p id="p-11">Cell morphological data of 27 <em>L. kluyveri</em> strains were acquired according to the methods described previously (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Yvert <em>et al.</em> 2013</a>). Briefly, yeast cells were grown in synthetic growth medium (SD; yeast nitrogen base without amino acids 6.7 g/L, glucose 20 g/L). Cells were cultured in 20 ml of liquid SD medium at 30°C to logarithmic-phase. Cell fixation, staining, and image acquisition were performed as described previously (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Ohya <em>et al.</em> 2005</a>). At least 200 cells were captured in a set of acquired images from an independent cell culture. A total of 135 sets of images were acquired from five replicate experiments on each of the 27 strains. The image sets were processed with CalMorph software (version 1.3) as described previously (<a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Yvert <em>et al.</em> 2013</a>).</p></div><div id="sec-4" class="subsection"><h3>High-throughput phenotyping using a microcultivation approach</h3><p id="p-12">Strains were subjected to high-throughput phenotyping by a microcultivation strategy in biological duplicate (n = 2). Briefly, strains were inoculated in 150 μl of YPD medium and incubated overnight at 30°. Precultures were picked in 96-well microplates containing 150 μl of different media (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026682/-/DC1/TableS2.pdf">Table S2</a>) and cultivated for 48 hr in a microplate reader (Tecan Infinite F200) at 30°C unless otherwise specified. These plates were stirred for 8 min with orbital shaking (120 rpm) followed by 2 min with linear shaking (160 rpm) over the course of 48 hr. After each stirring cycle, absorbance was recorded using a narrow spectrum (595–605 nm). In each tested condition, we determined the specific growth rate (population doubling time), the lag time (population adaptation time), and the efficiency (biomass produced during the cultivation) using a “home-made” program. Briefly, after background correction, the specific growth rate was determined given the maximum linear regression line calculated over 15 consecutive absorbance records with a sliding window of 10 points. Lag time was estimated as the intersection between this regression line and the baseline absorbance level. The yield of biomass was determined as the maximal OD reached during the cultivation process. Each value was normalized using results obtained with standard YPD medium, representing 1485 data points for each of the three parameters.</p></div><div id="sec-5" class="subsection"><h3>Data analysis</h3><p id="p-13">All statistical analyses of the cell morphology were performed using the R programming language (<a href="http://www.r-project.org/">http://www.r-project.org/</a>). The Kruskal-Wallis test, a principal component analysis (PCA), and a hierarchical cluster analysis (HCA) were performed as described previously (<a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">Yvert <em>et al.</em> 2013</a>). First, 15 PCs accounting for 90% of the cumulative contribution ratio were used for HCA (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026682/-/DC1/FigureS1.pdf">Figure S1</a>). Each PC was characterized in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026682/-/DC1/TableS3.xlsx">Table S3</a> using the morphological features by grouping the morphological traits as described previously (Ohnuki <em>et al.</em> 2014). Phenotypic variance of the cell morphology among 27 strains was calculated from the Z values of 501 traits, which were estimated by the generalized linear model (GLM) as described previously (Yang <em>et al.</em> 2014). Phenotypic variance among growth rates was estimated as the average of the variance estimated in each environmental condition.</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-14">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Fitness variation within a natural population of L. kluyveri</h3><p id="p-15">The ability to grow and the estimation of doubling time in specific environmental conditions (<em>e.g.</em>, different carbon sources or temperatures) are common parameters allowing for the characterization of the variation between and within species, particularly in yeasts. To obtain insight concerning intraspecific variation within the <em>L. kluyveri</em> species, we determined the specific growth rate, lag time, and the yield of biomass produced during the cultivation process of 27 <em>L. kluyveri</em> strains (listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026682/-/DC1/TableS1.pdf">Table S1</a>) grown in 55 environmental conditions, which were classified into three main groups: carbon utilization, toxins, and environments and metabolites (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026682/-/DC1/TableS2.pdf">Table S2</a>). With the exception of the strain 55-86.1, which was a strain prone to the flocculation phenotype, the same set of strains as those used for unraveling the genetic architecture within <em>L. kluyveri</em> were used (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Friedrich <em>et al.</em> 2015</a>). Overall the reproducibility between biological duplicate is high (Pearson correlation R = 0.95) suggesting low experimental variation. In YPD conditions, the specific growth rates ranged from 0.18 hr<sup>−1</sup> for the strain 68917.2 to 0.72 hr<sup>−1</sup> for the strain NRBC1811. For the reference strain CBS3082a, the doubling time is ∼1.15 hr, which is similar to that of <em>S. cerevisiae</em>.</p><p id="p-16">As described in both <em>S. cerevisiae</em> and <em>S. paradoxus</em>, a strong correlation was found when comparing the growth rate and the biomass yield (Pearson correlation R = 0.78) in most of the experimental conditions tested (with the exception of the substitution of glucose by mannitol or ethanol showing a null correlation). In contrast, the lag phase was independent from these fitness features (R=−0.11 and −0.08 for growth rate and yield of biomass, respectively) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026682/-/DC1/FigureS2.pdf">Figure S2</a>) (<a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">Warringer <em>et al.</em> 2011</a>). Together, these data suggest that a common genetic architecture governs both growth rate and biomass formation. To gain a better overview of the impact of the different environmental conditions on fitness, the growth of every strain in each condition was characterized and normalized to the standard YPD medium (see <em>Materials and Methods</em>).</p><p id="p-17">Barring xylose, growth on fermentable sugars generated little difference in growth compared to glucose. In contrast, nonfermentable sources led to a decrease in the specific growth rates, perhaps due to different uptake mechanisms and metabolization of these sugars (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Similarly, three subgroups of isolates were found for the environments and metabolites and toxins classes: one corresponded to very low growth reflecting high sensitivity, as it is the case for conditions that alter osmolarity; one corresponded to weak growth such as in the case where temperature was altered; and the last subgroup corresponded to growth similar to that observed in YPD (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Strikingly, the 5-FU metabolite generated an enhanced growth rate at concentrations that <em>S. cerevisiae</em> strains are sensitive to (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Jaquet <em>et al.</em> 1993</a>), probably due to the ability of <em>L. kluyveri</em> to use pyrimidine nucleotides and derivatives as a nitrogen source (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andersen <em>et al.</em> 2006</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/1063/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Intraspecific variance of fitness. Growth rates were determined in each condition and normalized to YPD (yeast extract 10 g/L, bactopeptone 20 g/L, dextrose 20 g/L) for the 27 strains. A ratio of 1 (dotted line) represents no variation in comparison to YPD. Growth conditions were classified as carbon sources (green), environment and metabolites (ivory), or toxins (blue). The variance among all strains in each condition is depicted by red dots corresponding to the secondary axis. Conditions were ranked according to their median calculated for the 27 strains. Three groups of strains were determined depending on the resistance: low (median &lt; 0.3, dark gray), mid (median 0.3&#x2013;0.8, intermediate gray), and high resistance (median &gt; 0.8, light gray)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-386382420" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Intraspecific variance of fitness. Growth rates were determined in each condition and normalized to YPD (yeast extract 10 g/L, bactopeptone 20 g/L, dextrose 20 g/L) for the 27 strains. A ratio of 1 (dotted line) represents no variation in comparison to YPD. Growth conditions were classified as carbon sources (green), environment and metabolites (ivory), or toxins (blue). The variance among all strains in each condition is depicted by red dots corresponding to the secondary axis. Conditions were ranked according to their median calculated for the 27 strains. Three groups of strains were determined depending on the resistance: low (median &lt; 0.3, dark gray), mid (median 0.3&#x2013;0.8, intermediate gray), and high resistance (median &gt; 0.8, light gray).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/1063/F1.medium.gif" width="193" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/4/1063/F1.medium.gif" width="193" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/1063/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/1063/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467981" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-18" class="first-child">Intraspecific variance of fitness. Growth rates were determined in each condition and normalized to YPD (yeast extract 10 g/L, bactopeptone 20 g/L, dextrose 20 g/L) for the 27 strains. A ratio of 1 (dotted line) represents no variation in comparison to YPD. Growth conditions were classified as carbon sources (green), environment and metabolites (ivory), or toxins (blue). The variance among all strains in each condition is depicted by red dots corresponding to the secondary axis. Conditions were ranked according to their median calculated for the 27 strains. Three groups of strains were determined depending on the resistance: low (median &lt; 0.3, dark gray), mid (median 0.3–0.8, intermediate gray), and high resistance (median &gt; 0.8, light gray).</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">The estimated variance for both of the 5-FU concentrations was at least twice as high as in the other tested conditions, suggesting that these isolates are characterized by a very broad resistance to this metabolite. Generally, the variance within the different conditions was low but some growth environments led to higher variation, such as succinate or the presence of metal ions (<em>e.g.</em>, NiSO<sub>4</sub> or CuSO<sub>4</sub>) (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). This suggests that mechanisms of resistance to specific substrates are variable between different strains. The low variance estimated in the standard YPD condition (0.013) may reflect a very similar growth profile among all isolates in this condition. However, strain 68.917.2 is a clear outlier as it explains almost 50% of the total variance. Indeed, by removing the growth rate data of 68.917.2, the variance decreased from 0.013 to 0.008. Over all, these data suggest that intraspecies evolution allows for differential resistance to various stresses among some strains.</p></div><div id="sec-9" class="subsection"><h3>Cell morphology variation within a natural population of L. kluyveri</h3><p id="p-20">Analysis of morphology includes a set of phenotypic traits among which variation was previously characterized in the budding <em>S. cerevisiae</em> yeast (<a id="xref-ref-35-6" class="xref-bibr" href="#ref-35">Yvert <em>et al.</em> 2013</a>). We performed high-dimensional morphological phenotyping of the same 27 <em>L. kluyveri</em> isolates after cultivation in complete medium at 30°. These strains predominantly contained unattached individual cells rather than aggregates of unseparated cells, making it possible to perform semiautomated image analysis with CalMorph (<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Ohya <em>et al.</em> 2005</a>). Briefly, each strain was cultivated in five biological replicates. Cells were then fixed with formaldehyde and their cell wall, nuclear DNA, and actin were stained with various specific fluorescent dyes. Images of at least 200 cells per culture were acquired via fluorescent microscopy and analyzed with CalMorph software to quantify 501 traits reflecting the size, shape, orientation, and intracellular organization of the cells. Despite a low variance of the growth rate determined in complete medium, 384 morphological traits revealed significant intraspecies variability out of the total 501 traits tested (<em>P</em> &lt; 0.05 after Bonferroni correction by Kruskal-Wallis test, <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). Detecting a great number of differences (77%) across the 27 strains suggested that the majority of the morphological organization in <em>L. kluyveri</em> is subjected to intraspecies quantitative variation.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/1063/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Morphological parameter variation within the L. kluyveri species. (A) Intra- vs. interstrain variability of morphological traits. Each dot represents one of 501 measured parameters. Orange and gray distinguish the traits that were called significant and nonsignificant by the Kruskal-Wallis test at P &lt; 0.05 after Bonferroni correction, respectively. For the purpose of visual clarity, each parameter was transformed by f(x) = (x &#x2212; &#x3BC;CBS10367) / &#x3C3;CBS10367, where &#x3BC;CBS10367 and &#x3C3;CBS10367 are the mean and standard deviation of the parameter across five replicates of the CBS10367 strain. Note that significance inference was determined from ranks of raw values and was therefore not affected by this transformation. The sum of squares across replicates (x-axis) and across strains (y-axis) was then computed. The three parameters highlighted in red reflect distinct cellular properties: whole cell size of the G1 cell, nuclear fitness for ellipse of the S/G2 cell, and long over short axis ratio of the ellipse fitted to the G1 cell. (B) Boxplot representation of these morphological parameters calculated for the 27 L. kluyveri strains. Box colors represent geographical origins with red for America, blue for Europe, and yellow for Asia. (C) Principal component (PC) analysis of L. kluyveri morphological variation. Strains are represented by their coordinates along the first two principal components, using the same colors as in Figure 2B. (D) Representative cells illustrating the traits contributing to the first two principal components. Bar: 5 &#x3BC;m." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-386382420" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Morphological parameter variation within the &lt;em&gt;L. kluyveri&lt;/em&gt; species. (A) Intra- &lt;em&gt;vs.&lt;/em&gt; interstrain variability of morphological traits. Each dot represents one of 501 measured parameters. Orange and gray distinguish the traits that were called significant and nonsignificant by the Kruskal-Wallis test at &lt;em&gt;P&lt;/em&gt; &lt; 0.05 after Bonferroni correction, respectively. For the purpose of visual clarity, each parameter was transformed by f(x) = (x &#x2212; &#x3BC;CBS10367) / &#x3C3;CBS10367, where &#x3BC;CBS10367 and &#x3C3;CBS10367 are the mean and standard deviation of the parameter across five replicates of the CBS10367 strain. Note that significance inference was determined from ranks of raw values and was therefore not affected by this transformation. The sum of squares across replicates (x-axis) and across strains (y-axis) was then computed. The three parameters highlighted in red reflect distinct cellular properties: whole cell size of the G1 cell, nuclear fitness for ellipse of the S/G2 cell, and long over short axis ratio of the ellipse fitted to the G1 cell. (B) Boxplot representation of these morphological parameters calculated for the 27 &lt;em&gt;L. kluyveri&lt;/em&gt; strains. Box colors represent geographical origins with red for America, blue for Europe, and yellow for Asia. (C) Principal component (PC) analysis of &lt;em&gt;L. kluyveri&lt;/em&gt; morphological variation. Strains are represented by their coordinates along the first two principal components, using the same colors as in Figure 2B. (D) Representative cells illustrating the traits contributing to the first two principal components. Bar: 5 &#x3BC;m.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/1063/F2.medium.gif" width="440" height="424"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/4/1063/F2.medium.gif" width="440" height="424"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/1063/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/1063/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467832" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-21" class="first-child">Morphological parameter variation within the <em>L. kluyveri</em> species. (A) Intra- <em>vs.</em> interstrain variability of morphological traits. Each dot represents one of 501 measured parameters. Orange and gray distinguish the traits that were called significant and nonsignificant by the Kruskal-Wallis test at <em>P</em> &lt; 0.05 after Bonferroni correction, respectively. For the purpose of visual clarity, each parameter was transformed by f(x) = (x − μCBS10367) / σCBS10367, where μCBS10367 and σCBS10367 are the mean and standard deviation of the parameter across five replicates of the CBS10367 strain. Note that significance inference was determined from ranks of raw values and was therefore not affected by this transformation. The sum of squares across replicates (x-axis) and across strains (y-axis) was then computed. The three parameters highlighted in red reflect distinct cellular properties: whole cell size of the G1 cell, nuclear fitness for ellipse of the S/G2 cell, and long over short axis ratio of the ellipse fitted to the G1 cell. (B) Boxplot representation of these morphological parameters calculated for the 27 <em>L. kluyveri</em> strains. Box colors represent geographical origins with red for America, blue for Europe, and yellow for Asia. (C) Principal component (PC) analysis of <em>L. kluyveri</em> morphological variation. Strains are represented by their coordinates along the first two principal components, using the same colors as in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>. (D) Representative cells illustrating the traits contributing to the first two principal components. Bar: 5 μm.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">One of the most striking variations in morphology concerned whole cell size (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>). In <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>, the <em>L. kluyveri</em> strains are aligned in descending order of cell size with CBS10368 characterized by the largest cell size, approximately 1.7 times larger than that of the smallest strain, CBS3082a. Two additional traits that dramatically varied across strains were related to the nuclear shape (nuclear fitness for ellipse) and the elongated cell shape (mother axis ratio). The values of these three traits ranked strains in three different orders, indicating that there was no correlation among cell size, nuclear shape, and elongated cell shape. Thus, the variation of <em>L. kluyveri</em> cellular morphology represents a set of multiple independent traits with different sources of variability.</p></div><div id="sec-10" class="subsection"><h3>Trait variation and population history</h3><p id="p-23">The genetic divergence within the <em>L. kluyveri</em> species ranges up to 2.5% (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Friedrich <em>et al.</em> 2015</a>), whereas trait profiling and morphological feature characterization show little variation. Hierarchical clustering of fitness data obtained for the 27 natural isolates of <em>L. kluyveri</em> reveal two main clusters that are not correlated with either their ecological or geographical origins (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). Except for the strains isolated from Europe that are mainly found in group 2 (7 out of 9 strains), the strains are evenly distributed between clusters. It is notable that strain 68.917.2 does not belong to either of the two groups, and appears to be an outlier within the <em>L. kluyveri</em> species, probably due to its slow growth on YPD. Hierarchical clustering carried out with the different classes of environmental conditions analyzed here identified similar groups clearly, suggesting that trait evolution is independent of strain niche according to the conditions tested (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026682/-/DC1/FigureS3.pdf">Figure S3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/1063/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Trait variation within L. kluyveri is not related to geographical or ecological niches. Hierarchical clustering of trait profiles was performed using a centered Pearson correlation metric and average linkage mapping for growth (A) as well as morphological parameter (B) analyses. In both heatmaps, rows and columns correspond to conditions (or morphological parameters) and strains, respectively. Growth rates were normalized for each condition to YPD (yeast extract 10 g/L, bactopeptone 20 g/L, dextrose 20 g/L); the high growth rates are depicted in red and slow growth rates in blue in the color scale. Similarly, in the heatmap corresponding to morphological data, low to high morphological score values are depicted by a gradient from blue to red. Colored circles represent geographical origins of each strain: Europe (blue), America (red), and Asia (yellow). Similarly, stars depict ecological origins of each strain: tree exudate and leaf (green), soil (brown), insect gut (purple), and unknown (gray)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-386382420" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Trait variation within &lt;em&gt;L. kluyveri&lt;/em&gt; is not related to geographical or ecological niches. Hierarchical clustering of trait profiles was performed using a centered Pearson correlation metric and average linkage mapping for growth (A) as well as morphological parameter (B) analyses. In both heatmaps, rows and columns correspond to conditions (or morphological parameters) and strains, respectively. Growth rates were normalized for each condition to YPD (yeast extract 10 g/L, bactopeptone 20 g/L, dextrose 20 g/L); the high growth rates are depicted in red and slow growth rates in blue in the color scale. Similarly, in the heatmap corresponding to morphological data, low to high morphological score values are depicted by a gradient from blue to red. Colored circles represent geographical origins of each strain: Europe (blue), America (red), and Asia (yellow). Similarly, stars depict ecological origins of each strain: tree exudate and leaf (green), soil (brown), insect gut (purple), and unknown (gray).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/1063/F3.medium.gif" width="440" height="218"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/4/1063/F3.medium.gif" width="440" height="218"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/1063/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/1063/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467861" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-24" class="first-child">Trait variation within <em>L. kluyveri</em> is not related to geographical or ecological niches. Hierarchical clustering of trait profiles was performed using a centered Pearson correlation metric and average linkage mapping for growth (A) as well as morphological parameter (B) analyses. In both heatmaps, rows and columns correspond to conditions (or morphological parameters) and strains, respectively. Growth rates were normalized for each condition to YPD (yeast extract 10 g/L, bactopeptone 20 g/L, dextrose 20 g/L); the high growth rates are depicted in red and slow growth rates in blue in the color scale. Similarly, in the heatmap corresponding to morphological data, low to high morphological score values are depicted by a gradient from blue to red. Colored circles represent geographical origins of each strain: Europe (blue), America (red), and Asia (yellow). Similarly, stars depict ecological origins of each strain: tree exudate and leaf (green), soil (brown), insect gut (purple), and unknown (gray).</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">Similarly, hierarchical clustering of the 501 morphological parameters revealed two clusters, each divided into two subgroups where no correlation with strain origin was distinguishable (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). We then investigated the properties of the variation in morphological features variation using principal component analysis (PCA). The first two components were correlated with cell size and nuclear fitness for ellipse, respectively (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026682/-/DC1/TableS3.xlsx">Table S3</a>). We found that the strains were nearly evenly spaced with no particular subgroups apparent. Analysis of all spaces represented by the first five components (63% of the cumulative contribution ratio, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026682/-/DC1/FigureS1.pdf">Figure S1</a>) led to similar results (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026682/-/DC1/FigureS4.pdf">Figure S4</a>), suggesting that <em>L. kluyveri</em> is characterized by a continuum of morphological features rather than discrete classes of distinct morphologies. In accordance with hierarchical clustering, strains from common ecological origins did not group together, indicating that differences in the morphological traits do not simply reflect adaptation to the simulated environments.</p></div><div id="sec-11" class="subsection"><h3>Relationship between genetic and phenotypic diversity</h3><p id="p-26">According to 881,427 polymorphic sites determined previously within the <em>L. kluyveri</em> species (<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Friedrich <em>et al.</em> 2015</a>), genetic architecture led to the definition of six subgroups, containing the 27 strains used in this study (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). Among these subgroups, there were two clearly defined populations: one corresponded to the strains that had originated from America (group 1) and the second to those isolated from Europe (group 6) (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>) (<a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">Friedrich <em>et al.</em> 2015</a>). The average pairwise comparison of trait profiles within the <em>L. kluyveri</em> species gave a correlation of 0.8. This parameter appears to be variable within and between the different groups. Indeed, the estimated correlation in the group 6 is approximately 0.85 whereas it decreases to 0.7 in group 2 (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>). In contrast, comparison of morphological parameters revealed an average correlation of nearly 0 (−0.034), due to the significant differences among the 384 cellular features. In spite of this almost null correlation, we were also able to determine variation between and within the different groups, for example a weak anticorrelation was estimated (R= −0.2) between groups 3 and 4 (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4C</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/1063/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlation between and within specific groups of L. kluyveri. (A) Genetic diversity within the L. kluyveri based on more than 881,427 segregating sites leads to the distinction of 6 different subgroups. Colored circles represent geographical origins of each strain: Europe (blue), America (red), and Asia (yellow). Pairwise comparison for growth rate (B) and morphological parameters (C) revealed high correlation to fitness independent of the subgroups, whereas no clear correlation was found for morphological data. The color scale represents levels of correlation where high correlation is depicted in red and low correlation in yellow." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-386382420" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Correlation between and within specific groups of &lt;em&gt;L. kluyveri&lt;/em&gt;. (A) Genetic diversity within the &lt;em&gt;L. kluyveri&lt;/em&gt; based on more than 881,427 segregating sites leads to the distinction of 6 different subgroups. Colored circles represent geographical origins of each strain: Europe (blue), America (red), and Asia (yellow). Pairwise comparison for growth rate (B) and morphological parameters (C) revealed high correlation to fitness independent of the subgroups, whereas no clear correlation was found for morphological data. The color scale represents levels of correlation where high correlation is depicted in red and low correlation in yellow.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/1063/F4.medium.gif" width="440" height="314"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/4/1063/F4.medium.gif" width="440" height="314"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/1063/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/1063/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468001" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-27" class="first-child">Correlation between and within specific groups of <em>L. kluyveri</em>. (A) Genetic diversity within the <em>L. kluyveri</em> based on more than 881,427 segregating sites leads to the distinction of 6 different subgroups. Colored circles represent geographical origins of each strain: Europe (blue), America (red), and Asia (yellow). Pairwise comparison for growth rate (B) and morphological parameters (C) revealed high correlation to fitness independent of the subgroups, whereas no clear correlation was found for morphological data. The color scale represents levels of correlation where high correlation is depicted in red and low correlation in yellow.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">To better understand how genetic variation drives phenotypic variability, we determined the relationship between genetic diversity and trait profiles. Strains belonging to group 1 were characterized by less similar trait profiles (average correlation of 0.8) despite their high genetic conservation. However, a significant anticorrelation (Kendall’s test, <em>P</em>-value = 3.94 × 10<sup>−9</sup>) was found when comparing fitness data among all the 27 strains, indicating that trait variation increases with the enhancement of genetic diversity (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). In contrast, no such correlation was determined between morphological features and genetic variation (Kendall’s test, <em>P</em>-value = 0.296) (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). These results suggest that genetic diversity within <em>L. kluyveri</em> influences trait profiles, but cannot explain all of the phenotypic variability given the lower correlation among North American strains.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/1063/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlation between genetic diversity and trait profile variation. Pairwise genetic diversity is plotted against pairwise Pearson correlation calculated for both fitness (A) and morphological parameters (B). The blue line corresponds to linear regressions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-386382420" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Correlation between genetic diversity and trait profile variation. Pairwise genetic diversity is plotted against pairwise Pearson correlation calculated for both fitness (A) and morphological parameters (B). The blue line corresponds to linear regressions.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/1063/F5.medium.gif" width="440" height="150"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/4/1063/F5.medium.gif" width="440" height="150"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/1063/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/1063/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468067" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-29" class="first-child">Correlation between genetic diversity and trait profile variation. Pairwise genetic diversity is plotted against pairwise Pearson correlation calculated for both fitness (A) and morphological parameters (B). The blue line corresponds to linear regressions.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Comparison of trait variation across different yeast species</h3><p id="p-30">Trait variation analysis within <em>S. cerevisiae</em> revealed a correlation with their genomic structure rather than to adaptation associated with their ecological niche. For example, <em>S. cerevisiae</em> strains isolated from Europe are more prone to be resistant to Na<sup>+</sup> and Li<sup>+</sup> ions given the pleiotropic roles of causal loci. In addition, clean lineages within the <em>S. paradoxus</em> species show similar trait profiles (<a id="xref-ref-32-6" class="xref-bibr" href="#ref-32">Warringer <em>et al.</em> 2011</a>). As mentioned above, we previously identified two clean lineages within the <em>L. kluyveri</em> species: one corresponding to strains originated from North America (group 1), the second comprised of strains isolated from Europe (group 6) (<a id="xref-ref-12-7" class="xref-bibr" href="#ref-12">Friedrich <em>et al.</em> 2015</a>). In contrast to <em>S. cerevisiae</em>, trait profile and morphological feature analyses revealed variation independently from the genetic structure of the <em>L. kluyveri</em> species. Moreover, the evolution of resistance to various stresses including heavy metals or high osmolarity is not related to strain niche. The lack of correlation between growth phenotype or cellular parameters on one hand, and the population structure or the origins of the strains on the other, could be related to the small number of isolates available from <em>L. kluyveri</em>. Despite this low number of strains, the overall genetic diversity is greater than 2.5%, whereas it is ∼0.5% for <em>S. cerevisiae</em>. In sharp contrast, the phenotypic variance is lower in <em>L. kluyveri</em> than in <em>S. cerevisiae</em> (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>), which has also been described for both <em>S. paradoxus</em> and <em>S. pombe</em> species (<a id="xref-ref-32-7" class="xref-bibr" href="#ref-32">Warringer <em>et al.</em> 2011</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Brown <em>et al.</em> 2011</a>). However, morphological variance determined in this study for <em>L. kluyveri</em> shows a clear trend where it is much higher than in <em>S. cerevisiae</em> (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>) (<a id="xref-ref-35-7" class="xref-bibr" href="#ref-35">Yvert <em>et al.</em> 2013</a>). Altogether, these data suggest that cellular morphology and growth are clearly two independent features given the impact of genetic diversity on their variation. Nevertheless, the small number of strains analyzed could lead to biased results and a larger collection would be required to definitively characterize morphological variation.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/4/1063/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fitness and morphological parameters variance. Comparison of phenotypic variance within L. kluyveri (blue) and S. cerevisiae (green) for growth data and morphological parameters." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-386382420" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Fitness and morphological parameters variance. Comparison of phenotypic variance within &lt;em&gt;L. kluyveri&lt;/em&gt; (blue) and &lt;em&gt;S. cerevisiae&lt;/em&gt; (green) for growth data and morphological parameters.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/4/1063/F6.medium.gif" width="440" height="344"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/4/1063/F6.medium.gif" width="440" height="344"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/4/1063/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/4/1063/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467886" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-31" class="first-child">Fitness and morphological parameters variance. Comparison of phenotypic variance within <em>L. kluyveri</em> (blue) and <em>S. cerevisiae</em> (green) for growth data and morphological parameters.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-32">Assessing the diversity among individuals within a species is now feasible for nonmodel organisms and essential for understanding population history, as well as obtaining a better insight into the genotype–phenotype relationship (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Peter and Schacherer 2016</a>). The Saccharomycotina subphylum (budding yeasts), which includes the baker’s yeast <em>S. cerevisiae</em>, includes ideal nonmodel organisms with which to explore diversity in terms of genetic and phenotypic variation (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Freel <em>et al.</em> 2014</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">2015</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">2016</a>). In this study, we investigated phenotypic variation within the protoploid <em>L. kluyveri</em> species by analyzing variation in both growth and morphological parameters, allowing for a comparison with previous studies performed in <em>S. cerevisiae</em> (<a id="xref-ref-35-8" class="xref-bibr" href="#ref-35">Yvert <em>et al.</em> 2013</a>; <a id="xref-ref-32-8" class="xref-bibr" href="#ref-32">Warringer <em>et al.</em> 2011</a>). Interestingly, the general behavior of these two categories of phenotype is very different within as well as between species.</p><p id="p-33">Among the 501 morphological features investigated in logarithmically growing cells, 384 showed significant variation among the strains studied, suggesting that these traits are subjected to large intraspecific variation within <em>L. kluyveri</em>. Similarly, 440 morphological parameters were previously described as significantly variable among 37 strains of <em>S. cerevisiae</em> (<a id="xref-ref-35-9" class="xref-bibr" href="#ref-35">Yvert <em>et al.</em> 2013</a>). Comparison of cell morphology between <em>L. kluyveri</em> and <em>S. cerevisiae</em> allowed us to clearly distinguish these two species according to the size of the nuclei or the buds, for instance. <em>L. kluyveri</em> has an extremely high variance in cell morphology of 1.1 compared to 0.62 determined with previously published data generated in <em>S. cerevisiae</em> (<a id="xref-ref-35-10" class="xref-bibr" href="#ref-35">Yvert <em>et al.</em> 2013</a>). In contrast, growth analysis of the same set of <em>L. kluyveri</em> isolates in 55 different environmental conditions showed a very low variance (0.06), demonstrating that growth profiles are more similar than that for <em>S. cerevisiae</em> species, where the variance was estimated to be approximately 0.75 (<a id="xref-ref-32-9" class="xref-bibr" href="#ref-32">Warringer <em>et al.</em> 2011</a>).</p><p id="p-34">In contrast to fitness patterns in <em>S. cerevisiae</em>, trait variation (for fitness and cell morphology) is not defined by population history in <em>L. kluyveri</em>. Using PCA analysis and hierarchical clustering, we did not observe any correlation between the origin (ecological or geographical) and the phenotypic patterns, showing that trait variations are not linked to either the population structure or the source environment. However, the number of <em>L. kluyveri</em> strains used in this study is relatively low and their ecological origins are somehow similar. Increasing the number of strains would probably reveal more structured trait profiles.</p><p id="p-35">Finally, we determined the correlation between pairwise trait variation and genetic diversity in order to have a better view of how genomes and phenotypes are related. Interestingly, there is strong evidence that genetic diversity is anticorrelated with fitness variation. However, this is not the case for differences among morphological parameters, showing a different global behavior of the two types of traits. We are here considering genetic diversity as SNPs but other genome variation can lead to trait profile differences. Genomes of the 27 strains of <em>L. kluyveri</em> were obtained through mapping against a reference genome (<a id="xref-ref-12-8" class="xref-bibr" href="#ref-12">Friedrich <em>et al.</em> 2015</a>). Although this method gives deep information on genome variation, important features are still missing such as transposon content, structural variants, or the presence of specific genes for a distinct group of strains (pan genome). Moreover, prion proteins can also serve as a motor for phenotypic inheritance in <em>S. cerevisiae</em> but are still not well known in <em>L. kluyveri</em> (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Halfmann <em>et al.</em> 2012</a>). A better knowledge of these features would help to gain a better understanding of phenotypic diversity.</p><p id="p-36">Given the genetic characteristics of <em>L. kluyveri</em> and its trait variability, quantitative genetics is definitively applicable to this species and would lead to a better understanding of the genotype–phenotype relationship across diverse yeasts. Moreover, laboratory evolution experiments carried out for both <em>S. cerevisiae</em> and the pathogenic <em>Candida albicans</em> species have shown that the presence of an additional chromosome is linked to stress resistance acquisition (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Selmecki <em>et al.</em> 2009</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Yona <em>et al.</em> 2012</a>). Similar experiments in <em>L. kluyveri</em> would reveal if the acquisition of aneuploidy is a common mechanism across yeast species to improve fitness in stress conditions. Finally, further analysis will, without a doubt, provide deeper insight into functional diversity in this nonmodel organism.</p></div><div class="section" id="sec-14"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-37">We thank Kelle Freel and Christian Brion for comments on the manuscript. We are also thankful to B. Kammerer for the development of a program to extract growth parameters and S. Morinaga for her technical assistance. This work was supported by an Agence Nationale de la Recherche (ANR) grant (2010-BLAN-1606-05) (to J.S.), an ANR Young Investigator grant (2011-JSV6-004-01) (to J.S.), and grants-in-aid for Scientific Research from the Ministry of Education, Culture, Sports, Science and Technology, Japan (24370002 and 15H04402) (to Y.O.). We also thank the Université de Strasbourg (IdEx 2012 Attractivité) for their financial support. A.S. is supported by a grant from the Région Alsace.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-38">Supplemental Material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026682/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026682/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-39"><em>Communicating editor: A. Rokas</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-12-23"><span class="received-label">Received </span>December 23, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-02-11"><span class="accepted-label">Accepted </span>February 11, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Jung <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.1" data-doi="10.1080/15257770600889386"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merico</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bjornberg</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnackerz</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Catabolism of pyrimidines in yeast: a tool to understand degradation of anticancer drugs.</span> <abbr class="cit-jnl-abbrev">Nucleosides Nucleotides Nucleic Acids</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">9–11</span>): <span class="cit-fpage">991</span>–<span class="cit-lpage">996</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleosides%252C%2Bnucleotides%2B%2526%2Bnucleic%2Bacids%26rft.stitle%253DNucleosides%2BNucleotides%2BNucleic%2BAcids%26rft.aulast%253DAndersen%26rft.auinit1%253DG.%26rft.volume%253D25%26rft.issue%253D9-11%26rft.spage%253D991%26rft.epage%253D996%26rft.atitle%253DCatabolism%2Bof%2Bpyrimidines%2Bin%2Byeast%253A%2Ba%2Btool%2Bto%2Bunderstand%2Bdegradation%2Bof%2Banticancer%2Bdrugs.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F15257770600889386%26rft_id%253Dinfo%253Apmid%252F17065052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/15257770600889386&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17065052&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.2" data-doi="10.1111/j.1567-1364.2008.00442.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beck</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dobritzsch</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piskur</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title"><em>Saccharomyces kluyveri</em> as a model organism to study pyrimidine degradation.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">1209</span>–<span class="cit-lpage">1213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2008.00442.x%26rft_id%253Dinfo%253Apmid%252F18823354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjg6IjgvOC8xMjA5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi80LzEwNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.3" data-doi="10.1093/molbev/msu037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bergstrom</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salinas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barre</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A high-definition view of functional genetic variation from natural yeast genomes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">872</span>–<span class="cit-lpage">888</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu037%26rft_id%253Dinfo%253Apmid%252F24425782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMxLzQvODcyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi80LzEwNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.4" data-doi="10.1038/nrg2085"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bussey</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andrews</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Exploring genetic interactions and networks with yeast.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">437</span>–<span class="cit-lpage">449</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DBoone%26rft.auinit1%253DC.%26rft.volume%253D8%26rft.issue%253D6%26rft.spage%253D437%26rft.epage%253D449%26rft.atitle%253DExploring%2Bgenetic%2Binteractions%2Band%2Bnetworks%2Bwith%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2085%26rft_id%253Dinfo%253Apmid%252F17510664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2085&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17510664&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.5" data-doi="10.1093/nar/gkv363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brion</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pflieger</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedrich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schacherer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Evolution of intraspecific transcriptomic landscapes in yeasts.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">4558</span>–<span class="cit-lpage">4568</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv363%26rft_id%253Dinfo%253Apmid%252F25897111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjQzLzkvNDU1OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNC8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.6" data-doi="10.1534/g3.111.001123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A geographically diverse collection of <em>Schizosaccharomyces pombe</em> isolates shows limited phenotypic variation but extensive karyotypic diversity.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">1</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">615</span>–<span class="cit-lpage">626</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DBrown%26rft.auinit1%253DW.%2BR.%2BA.%26rft.volume%253D1%26rft.issue%253D7%26rft.spage%253D615%26rft.epage%253D626%26rft.atitle%253DA%2BGeographically%2BDiverse%2BCollection%2Bof%2BSchizosaccharomyces%2Bpombe%2BIsolates%2BShows%2BLimited%2BPhenotypic%2BVariation%2Bbut%2BExtensive%2BKaryotypic%2BDiversity%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.001123%26rft_id%253Dinfo%253Apmid%252F22384373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjEvNy82MTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzQvMTA2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.7" data-doi="10.1534/g3.113.007492"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cromie</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyma</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ludlow</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garmendia-Torres</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic sequence diversity and population structure of <em>Saccharomyces cerevisiae</em> assessed by RAD-seq.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">2163</span>–<span class="cit-lpage">2171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.007492%26rft_id%253Dinfo%253Apmid%252F24122055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjMvMTIvMjE2MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNC8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.8" data-doi="10.1186/1471-2105-11-353"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dittmar</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reid</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rothstein</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">ScreenMill: a freely available software suite for growth measurement, analysis and visualization of high-throughput screen data.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">353</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DDittmar%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D11%26rft.spage%253D353%26rft.epage%253D353%26rft.atitle%253DScreenMill%253A%2Ba%2Bfreely%2Bavailable%2Bsoftware%2Bsuite%2Bfor%2Bgrowth%2Bmeasurement%252C%2Banalysis%2Band%2Bvisualization%2Bof%2Bhigh-throughput%2Bscreen%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-353%26rft_id%253Dinfo%253Apmid%252F20584323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-353&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20584323&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.9" data-doi="10.1093/gbe/evu203"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Freel</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedrich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schacherer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Population genomic analysis reveals highly conserved mitochondrial genomes in the yeast species <em>Lachancea thermotolerans</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">2586</span>–<span class="cit-lpage">2594</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevu203%26rft_id%253Dinfo%253Apmid%252F25212859%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjk6IjYvMTAvMjU4NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNC8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Freel</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedrich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schacherer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Mitochondrial genome evolution in yeasts: an all-encompassing view.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">4</span>). Available at: <a href="http://dx.doi.org/10.1093/femsyr/fov023">http://dx.doi.org/10.1093/femsyr/fov023</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Freel</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedrich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarilar</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devillers</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neuveglise</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Whole-genome sequencing and intraspecific analysis of the yeast species <em>Lachancea quebecensis</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> DOI:10.1093/gbe/evv262.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.12" data-doi="10.1093/molbev/msu295"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friedrich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reisser</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schacherer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Population genomics reveals chromosome-scale heterogeneous evolution in a protoploid yeast.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">32</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">184</span>–<span class="cit-lpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu295%26rft_id%253Dinfo%253Apmid%252F25349286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMyLzEvMTg0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi80LzEwNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.13" data-doi="10.1101/gr.091546.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Genolevures</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souciet</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaillardin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Comparative genomics of protoploid <em>Saccharomycetaceae</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">1696</span>–<span class="cit-lpage">1709</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.091546.109%26rft_id%253Dinfo%253Apmid%252F19525356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMC8xNjk2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi80LzEwNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.14" data-doi="10.1007/978-1-4615-5381-6_94"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gojkovic</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paracchini</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piskur</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">A new model organism for studying the catabolism of pyrimidines and purines.</span> <abbr class="cit-jnl-abbrev">Adv. Exp. Med. Biol.</abbr> <span class="cit-vol">431</span>: <span class="cit-fpage">475</span>–<span class="cit-lpage">479</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bexperimental%2Bmedicine%2Band%2Bbiology%26rft.stitle%253DAdv%2BExp%2BMed%2BBiol%26rft.aulast%253DGojkovic%26rft.auinit1%253DZ.%26rft.volume%253D431%26rft.spage%253D475%26rft.epage%253D479%26rft.atitle%253DA%2Bnew%2Bmodel%2Borganism%2Bfor%2Bstudying%2Bthe%2Bcatabolism%2Bof%2Bpyrimidines%2Band%2Bpurines.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4615-5381-6_94%26rft_id%253Dinfo%253Apmid%252F9598114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4615-5381-6_94&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9598114&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000074086600094&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.15" data-doi="10.1038/nature10875"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Halfmann</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarosz</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lancaster</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Prions are a common mechanism for phenotypic inheritance in wild yeasts.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">482</span>(<span class="cit-issue">7385</span>): <span class="cit-fpage">363</span>–<span class="cit-lpage">368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHalfmann%26rft.auinit1%253DR.%26rft.volume%253D482%26rft.issue%253D7385%26rft.spage%253D363%26rft.epage%253D368%26rft.atitle%253DPrions%2Bare%2Ba%2Bcommon%2Bmechanism%2Bfor%2Bphenotypic%2Binheritance%2Bin%2Bwild%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10875%26rft_id%253Dinfo%253Apmid%252F22337056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10875&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22337056&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300287100038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.16" data-doi="10.1007/BF00280204"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaquet</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lollier</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souciet</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Potier</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Genetic analysis of yeast strains lacking negative feedback control: a one-step method for positive selection and cloning of carbamoylphosphate synthetase-aspartate transcarbamoylase mutants unable to respond to UTP.</span> <abbr class="cit-jnl-abbrev">Mol. Gen. Genet.</abbr> <span class="cit-vol">241</span>(<span class="cit-issue">1–2</span>): <span class="cit-fpage">81</span>–<span class="cit-lpage">88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bgeneral%2Bgenetics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGen%2BGenet%26rft.aulast%253DJaquet%26rft.auinit1%253DL.%26rft.volume%253D241%26rft.issue%253D1-2%26rft.spage%253D81%26rft.epage%253D88%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Byeast%2Bstrains%2Blacking%2Bnegative%2Bfeedback%2Bcontrol%253A%2Ba%2Bone-step%2Bmethod%2Bfor%2Bpositive%2Bselection%2Band%2Bcloning%2Bof%2Bcarbamoylphosphate%2Bsynthetase-aspartate%2Btranscarbamoylase%2Bmutants%2Bunable%2Bto%2Brespond%2Bto%2BUTP.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00280204%26rft_id%253Dinfo%253Apmid%252F8232215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00280204&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8232215&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.17" data-doi="10.1534/g3.112.003152"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">P. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedrich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reisser</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schacherer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mitochondrial genome evolution in a single protoploid yeast species.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">1103</span>–<span class="cit-lpage">1111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DJung%26rft.auinit1%253DP.%2BP.%26rft.volume%253D2%26rft.issue%253D9%26rft.spage%253D1103%26rft.epage%253D1111%26rft.atitle%253DMitochondrial%2BGenome%2BEvolution%2Bin%2Ba%2BSingle%2BProtoploid%2BYeast%2BSpecies%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.003152%26rft_id%253Dinfo%253Apmid%252F22973548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjIvOS8xMTAzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi80LzEwNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.18" data-doi="10.1371/journal.pone.0119807"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">P. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christian</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kay</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skupin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linster</span>  <span class="cit-name-given-names">C. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Protocols and programs for high-throughput growth and aging phenotyping in yeast.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">e0119807</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0119807%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0119807%26rft_id%253Dinfo%253Apmid%252F25822370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0119807&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25822370&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.19" data-doi="10.1038/nature02424"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kellis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birren</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Proof and evolutionary analysis of ancient genome duplication in the yeast <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">428</span>(<span class="cit-issue">6983</span>): <span class="cit-fpage">617</span>–<span class="cit-lpage">624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKellis%26rft.auinit1%253DM.%26rft.volume%253D428%26rft.issue%253D6983%26rft.spage%253D617%26rft.epage%253D624%26rft.atitle%253DProof%2Band%2Bevolutionary%2Banalysis%2Bof%2Bancient%2Bgenome%2Bduplication%2Bin%2Bthe%2Byeast%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02424%26rft_id%253Dinfo%253Apmid%252F15004568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02424&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15004568&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220697200034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.20" data-doi="10.1371/journal.pgen.1000223"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kvitek</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Will</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gasch</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Variations in stress sensitivity and genomic expression in diverse <em>S. cerevisiae</em> isolates.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">e1000223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKvitek%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253De1000223%26rft.epage%253De1000223%26rft.atitle%253DVariations%2Bin%2Bstress%2Bsensitivity%2Band%2Bgenomic%2Bexpression%2Bin%2Bdiverse%2BS.%2Bcerevisiae%2Bisolates.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000223%26rft_id%253Dinfo%253Apmid%252F18927628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000223&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18927628&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.21" data-doi="10.1038/nature07743"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moses</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Population genomics of domestic and wild yeasts.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>(<span class="cit-issue">7236</span>): <span class="cit-fpage">337</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLiti%26rft.auinit1%253DG.%26rft.volume%253D458%26rft.issue%253D7236%26rft.spage%253D337%26rft.epage%253D341%26rft.atitle%253DPopulation%2Bgenomics%2Bof%2Bdomestic%2Band%2Bwild%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07743%26rft_id%253Dinfo%253Apmid%252F19212322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07743&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212322&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264285600041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCullough</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herskowitz</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Mating pheromones of <em>Saccharomyces kluyveri</em>: pheromone interactions between <em>Saccharomyces kluyveri</em> and <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">138</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">146</span>–<span class="cit-lpage">154</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DMcCullough%26rft.auinit1%253DJ.%26rft.volume%253D138%26rft.issue%253D1%26rft.spage%253D146%26rft.epage%253D154%26rft.atitle%253DMating%2BPheromones%2Bof%2BSaccharomyces%2Bkluyveri%253A%2BPheromone%2BInteractions%2BBetween%2BSaccharomyces%2Bkluyveri%2Band%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Apmid%252F374360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTM4LzEvMTQ2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi80LzEwNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.23" data-doi="10.1007/s00438-003-0950-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moller</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langkjaer</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piskur</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsson</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Pyruvate decarboxylases from the petite-negative yeast <em>Saccharomyces kluyveri</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">270</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">558</span>–<span class="cit-lpage">568</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DM%25C3%25B8ller%26rft.auinit1%253DK.%26rft.volume%253D270%26rft.issue%253D6%26rft.spage%253D558%26rft.epage%253D568%26rft.atitle%253DPyruvate%2Bdecarboxylases%2Bfrom%2Bthe%2Bpetite-negative%2Byeast%2BSaccharomyces%2Bkluyveri.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-003-0950-z%26rft_id%253Dinfo%253Apmid%252F14648197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-003-0950-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14648197&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.4.1063.24" data-doi="10.1016/j.jbiosc.2013.08.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohnuki</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Enomoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshimoto</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohya</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Dynamic changes in brewing yeast cells in culture revealed by statistical analyses of yeast morphological data.</span> <abbr class="cit-jnl-abbrev">J Biosci Bioeng</abbr> <span class="cit-vol">117</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">278</span>–<span class="cit-lpage">284</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiosci%2BBioeng%26rft.volume%253D117%26rft.spage%253D278%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jbiosc.2013.08.005%26rft_id%253Dinfo%253Apmid%252F24012106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jbiosc.2013.08.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24012106&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.25" data-doi="10.1073/pnas.0509436102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohya</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sese</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yukawa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sano</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakatani</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">High-dimensional and large-scale phenotyping of yeast mutants.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>(<span class="cit-issue">52</span>): <span class="cit-fpage">19015</span>–<span class="cit-lpage">19020</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0509436102%26rft_id%253Dinfo%253Apmid%252F16365294%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzUyLzE5MDE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi80LzEwNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.26" data-doi="10.1091/mbc.E15-07-0466"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohya</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimori</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okada</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohnuki</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Single-cell phenomics in budding yeast.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">26</span>(<span class="cit-issue">22</span>): <span class="cit-fpage">3920</span>–<span class="cit-lpage">3925</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Bof%2Bthe%2BCell%26rft.stitle%253DMol.%2BBiol.%2BCell%26rft.aulast%253DOhya%26rft.auinit1%253DY.%26rft.volume%253D26%26rft.issue%253D22%26rft.spage%253D3920%26rft.epage%253D3925%26rft.atitle%253DSingle-cell%2Bphenomics%2Bin%2Bbudding%2Byeast%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E15-07-0466%26rft_id%253Dinfo%253Apmid%252F26543200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8yMi8zOTIwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi80LzEwNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peter</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schacherer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Population genomics of yeasts: Towards a comprehensive view across a broad evolutionary scale.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">233</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">73</span>–<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%26rft.volume%253D233%26rft.spage%253D73%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.28" data-doi="10.1038/nature07670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schacherer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruderfer</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Comprehensive polymorphism survey elucidates population structure of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>(<span class="cit-issue">7236</span>): <span class="cit-fpage">342</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSchacherer%26rft.auinit1%253DJ.%26rft.volume%253D458%26rft.issue%253D7236%26rft.spage%253D342%26rft.epage%253D345%26rft.atitle%253DComprehensive%2Bpolymorphism%2Bsurvey%2Belucidates%2Bpopulation%2Bstructure%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07670%26rft_id%253Dinfo%253Apmid%252F19212320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212320&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264285600042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.29" data-doi="10.1371/journal.pgen.1000705"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Selmecki</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dulmage</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cowen</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Acquisition of aneuploidy provides increased fitness during the evolution of antifungal drug resistance.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">e1000705</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSelmecki%26rft.auinit1%253DA.%2BM.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253De1000705%26rft.epage%253De1000705%26rft.atitle%253DAcquisition%2Bof%2Baneuploidy%2Bprovides%2Bincreased%2Bfitness%2Bduring%2Bthe%2Bevolution%2Bof%2Bantifungal%2Bdrug%2Bresistance.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000705%26rft_id%253Dinfo%253Apmid%252F19876375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000705&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19876375&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.30" data-doi="10.1101/gr.155762.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Skelly</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merrihew</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riffle</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connelly</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerr</span>  <span class="cit-name-given-names">E. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Integrative phenomics reveals insight into the structure of phenotypic diversity in budding yeast.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">1496</span>–<span class="cit-lpage">1504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.155762.113%26rft_id%253Dinfo%253Apmid%252F23720455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIzLzkvMTQ5NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNC8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.31" data-doi="10.1002/yea.931"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blomberg</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Automated screening in environmental arrays allows analysis of quantitative phenotypic profiles in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">20</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">53</span>–<span class="cit-lpage">67</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DWarringer%26rft.auinit1%253DJ.%26rft.volume%253D20%26rft.issue%253D1%26rft.spage%253D53%26rft.epage%253D67%26rft.atitle%253DAutomated%2Bscreening%2Bin%2Benvironmental%2Barrays%2Ballows%2Banalysis%2Bof%2Bquantitative%2Bphenotypic%2Bprofiles%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.931%26rft_id%253Dinfo%253Apmid%252F12489126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.931&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12489126&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180219900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.32" data-doi="10.1371/journal.pgen.1002111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zorgo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zia</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gjuvsland</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Trait variation in yeast is defined by population history.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">e1002111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWarringer%26rft.auinit1%253DJ.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253De1002111%26rft.epage%253De1002111%26rft.atitle%253DTrait%2Bvariation%2Bin%2Byeast%2Bis%2Bdefined%2Bby%2Bpopulation%2Bhistory.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002111%26rft_id%253Dinfo%253Apmid%252F21698134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21698134&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.4.1063.33" data-doi="10.1186/1471-2164-15-932"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohnuki</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohya</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Unveiling nonessential gene deletions that confer significant morphological phenotypes beyond natural yeast strains.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">932</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D932%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-932%26rft_id%253Dinfo%253Apmid%252F25344683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-932&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25344683&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.34" data-doi="10.1073/pnas.1211150109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yona</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manor</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herbst</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romano</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Chromosomal duplication is a transient evolutionary solution to stress.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>(<span class="cit-issue">51</span>): <span class="cit-fpage">21010</span>–<span class="cit-lpage">21015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1211150109%26rft_id%253Dinfo%253Apmid%252F23197825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzUxLzIxMDEwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi80LzEwNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.4.1063.35" data-doi="10.1186/1752-0509-7-54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yvert</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohnuki</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nogami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imanaga</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fehrmann</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Single-cell phenomics reveals intra-species variation of phenotypic noise in yeast.</span> <abbr class="cit-jnl-abbrev">BMC Syst. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BSyst.%2BBiol.%26rft.volume%253D7%26rft.spage%253D54%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1752-0509-7-54%26rft_id%253Dinfo%253Apmid%252F23822767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1752-0509-7-54&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23822767&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F4%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/4/1063.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/4/1049" title="Accuracy of Genomic Prediction in Switchgrass (Panicum virgatum L.) Improved by Accounting for Linkage Disequilibrium" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/4/1073" title="Embryonal Control of Yellow Seed Coat Locus ECY1 Is Related to Alanine and Phenylalanine Metabolism in the Seed Embryo of Brassica napus" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/4"><span>Volume 6 Issue 4, April 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/4.cover-source.jpg?itok=Lj45nPaf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/4/1063" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-VMHX4ZXNVhyqtl3Z99eTgf3sWdl6o5Ue4TiKVDzdtkE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/4/1063" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/4/1063" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/4/1063" class="active" data-icon-position="" data-hide-link-title="0">Large-Scale Survey of Intraspecific Fitness and Cell Morphology Variation in a Protoploid Yeast Species</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460124" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ywBh9gSLAvJofwm-JJf79VHSybeYkn18SsKlR9EoK0k" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/4/1063.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/4/1063" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Uoa7Z9DHEcLCE74uK9OhxCyzP6nK8usQnXSYYWlVuj0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/4/1063" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/4/1063.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460124" id="node460124--2" data-pisa="ggg;6/4/1063" data-pisa-master="ggg;g3.115.026682" data-apath="/ggg/6/4/1063.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Large-Scale Survey of Intraspecific Fitness and Cell Morphology Variation in a Protoploid Yeast Species</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Paul P. Jung</span>, <span class="highwire-citation-author" data-delta="1">Anastasie Sigwalt</span>, <span class="highwire-citation-author" data-delta="2">Shinsuke Ohnuki</span>, <span class="highwire-citation-author" data-delta="3">Jacky de Montigny</span>, <span class="highwire-citation-author" data-delta="4">Yoshikazu Ohya</span> and <span class="highwire-citation-author" data-delta="5">Joseph Schacherer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1063-1071;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.026682 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460124--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Paul P. Jung </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Genomics and Microbiology, University of Strasbourg - Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, 67083 Strasbourg, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BP.%2BJung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jung%20PP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaul%2BP.%2BJung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anastasie Sigwalt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Genomics and Microbiology, University of Strasbourg - Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, 67083 Strasbourg, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anastasie%2BSigwalt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sigwalt%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnastasie%2BSigwalt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Shinsuke Ohnuki </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Integrated Biosciences, Graduate School of Frontier Sciences, University of Tokyo, Kashiwa 277-8562, Chiba, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shinsuke%2BOhnuki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ohnuki%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShinsuke%2BOhnuki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jacky de Montigny </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Genomics and Microbiology, University of Strasbourg - Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, 67083 Strasbourg, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacky%2Bde%2BMontigny%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Montigny%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJacky%2Bde%2BMontigny%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Yoshikazu Ohya </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Integrated Biosciences, Graduate School of Frontier Sciences, University of Tokyo, Kashiwa 277-8562, Chiba, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yoshikazu%2BOhya%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ohya%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYoshikazu%2BOhya%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:schacherer@unistra.fr" class="" data-icon-position="" data-hide-link-title="0">schacherer@unistra.fr</a>
<a href="mailto:ohya@k.u-tokyo.ac.jp" class="" data-icon-position="" data-hide-link-title="0">ohya@k.u-tokyo.ac.jp</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Joseph Schacherer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Genomics and Microbiology, University of Strasbourg - Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, 67083 Strasbourg, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BSchacherer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schacherer%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BSchacherer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:schacherer@unistra.fr" class="" data-icon-position="" data-hide-link-title="0">schacherer@unistra.fr</a>
<a href="mailto:ohya@k.u-tokyo.ac.jp" class="" data-icon-position="" data-hide-link-title="0">ohya@k.u-tokyo.ac.jp</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/4/1063" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/4/1063&amp;title=Large-Scale%20Survey%20of%20Intraspecific%20Fitness%20and%20Cell%20Morphology%20Variation%20in%20a%20Protoploid%20Yeast%20Species" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/4/1063&amp;title=Large-Scale%20Survey%20of%20Intraspecific%20Fitness%20and%20Cell%20Morphology%20Variation%20in%20a%20Protoploid%20Yeast%20Species" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/4/1063&amp;title=Large-Scale%20Survey%20of%20Intraspecific%20Fitness%20and%20Cell%20Morphology%20Variation%20in%20a%20Protoploid%20Yeast%20Species" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/4/1063&amp;text=Large-Scale%20Survey%20of%20Intraspecific%20Fitness%20and%20Cell%20Morphology%20Variation%20in%20a%20Protoploid%20Yeast%20Species" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/4/1063&amp;title=Large-Scale%20Survey%20of%20Intraspecific%20Fitness%20and%20Cell%20Morphology%20Variation%20in%20a%20Protoploid%20Yeast%20Species" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/4/1063&amp;t=Large-Scale%20Survey%20of%20Intraspecific%20Fitness%20and%20Cell%20Morphology%20Variation%20in%20a%20Protoploid%20Yeast%20Species" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/4/1063&amp;title=Large-Scale%20Survey%20of%20Intraspecific%20Fitness%20and%20Cell%20Morphology%20Variation%20in%20a%20Protoploid%20Yeast%20Species" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/4/1063&amp;title=Large-Scale%20Survey%20of%20Intraspecific%20Fitness%20and%20Cell%20Morphology%20Variation%20in%20a%20Protoploid%20Yeast%20Species" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460124/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460124" id="node460124--3" data-pisa="ggg;6/4/1063" data-pisa-master="ggg;g3.115.026682" data-apath="/ggg/6/4/1063.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Large-Scale Survey of Intraspecific Fitness and Cell Morphology Variation in a Protoploid Yeast Species</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Paul P. Jung</span>, <span class="highwire-citation-author" data-delta="1">Anastasie Sigwalt</span>, <span class="highwire-citation-author" data-delta="2">Shinsuke Ohnuki</span>, <span class="highwire-citation-author" data-delta="3">Jacky de Montigny</span>, <span class="highwire-citation-author" data-delta="4">Yoshikazu Ohya</span> and <span class="highwire-citation-author" data-delta="5">Joseph Schacherer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1063-1071;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.026682 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460124--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Paul P. Jung </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Genomics and Microbiology, University of Strasbourg - Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, 67083 Strasbourg, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BP.%2BJung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jung%20PP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaul%2BP.%2BJung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anastasie Sigwalt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Genomics and Microbiology, University of Strasbourg - Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, 67083 Strasbourg, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anastasie%2BSigwalt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sigwalt%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnastasie%2BSigwalt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Shinsuke Ohnuki </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Integrated Biosciences, Graduate School of Frontier Sciences, University of Tokyo, Kashiwa 277-8562, Chiba, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shinsuke%2BOhnuki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ohnuki%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShinsuke%2BOhnuki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jacky de Montigny </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Genomics and Microbiology, University of Strasbourg - Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, 67083 Strasbourg, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacky%2Bde%2BMontigny%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Montigny%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJacky%2Bde%2BMontigny%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Yoshikazu Ohya </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Integrated Biosciences, Graduate School of Frontier Sciences, University of Tokyo, Kashiwa 277-8562, Chiba, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yoshikazu%2BOhya%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ohya%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYoshikazu%2BOhya%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:schacherer@unistra.fr" class="" data-icon-position="" data-hide-link-title="0">schacherer@unistra.fr</a>
<a href="mailto:ohya@k.u-tokyo.ac.jp" class="" data-icon-position="" data-hide-link-title="0">ohya@k.u-tokyo.ac.jp</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Joseph Schacherer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Genomics and Microbiology, University of Strasbourg - Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, 67083 Strasbourg, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BSchacherer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schacherer%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BSchacherer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:schacherer@unistra.fr" class="" data-icon-position="" data-hide-link-title="0">schacherer@unistra.fr</a>
<a href="mailto:ohya@k.u-tokyo.ac.jp" class="" data-icon-position="" data-hide-link-title="0">ohya@k.u-tokyo.ac.jp</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1063&amp;rft.epage=1071&amp;rft.atitle=Large-Scale%20Survey%20of%20Intraspecific%20Fitness%20and%20Cell%20Morphology%20Variation%20in%20a%20Protoploid%20Yeast%20Species&amp;rft.volume=6&amp;rft.issue=4&amp;rft.date=2016-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Jung%2C+Paul+P.&amp;rft.au=Sigwalt%2C+Anastasie&amp;rft.au=Ohnuki%2C+Shinsuke&amp;rft.au=de+Montigny%2C+Jacky&amp;rft.au=Ohya%2C+Yoshikazu&amp;rft.au=Schacherer%2C+Joseph"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460124/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460124/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460124/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460124/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460124/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460124/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460124/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460124/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460124/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460124/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460124/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460124/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/4/1063" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/4/1063.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/4/1063.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/4/1063.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/4/1063.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
