<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/9/3049" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/9/3049.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/9/3049.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/9/3049.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/9/3049/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/9/3049.atom" />
<meta name="HW.pisa" content="ggg;8/9/3049" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Tandem Duplicate Genes in Maize Are Abundant and Date to Two Distinct Periods of Time" />
<meta name="DC.Identifier" content="10.1534/g3.118.200580" />
<meta name="DC.Date" content="2018-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Kono et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.6062972" />
<meta name="DC.Description" content="Tandem duplicate genes are proximally duplicated and as such occur in similar genomic neighborhoods. Using the maize B73 and PH207 de novo genome assemblies, we identified thousands of tandem gene duplicates that account for ∼10% of the annotated genes. These tandem duplicates have a bimodal distribution of ages, which coincide with ancient allopolyploidization and more recent domestication. Tandem duplicates are smaller on average and have a higher probability of containing LTR elements than other genes, suggesting origins in nonhomologous recombination. Within relatively recent tandem duplicate genes, ∼26% appear to be undergoing degeneration or divergence in function from the ancestral copy. Our results show that tandem duplicates are abundant in maize, arose in bursts throughout maize evolutionary history under multiple potential mechanisms, and may provide a substrate for novel phenotypic variation." />
<meta name="DC.Contributor" content="Thomas J.Y. Kono" />
<meta name="DC.Contributor" content="Alex B. Brohammer" />
<meta name="DC.Contributor" content="Suzanne E. McGaugh" />
<meta name="DC.Contributor" content="Candice N. Hirsch" />
<meta name="article:published_time" content="2018-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Tandem Duplicate Genes in Maize Are Abundant and Date to Two Distinct Periods of Time" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Tandem duplicate genes are proximally duplicated and as such occur in similar genomic neighborhoods. Using the maize B73 and PH207 &lt;i&gt;de novo&lt;/i&gt; genome assemblies, we identified thousands of tandem gene duplicates that account for ∼10% of the annotated genes. These tandem duplicates have a bimodal distribution of ages, which coincide with ancient allopolyploidization and more recent domestication. Tandem duplicates are smaller on average and have a higher probability of containing LTR elements than other genes, suggesting origins in nonhomologous recombination. Within relatively recent tandem duplicate genes, ∼26% appear to be undergoing degeneration or divergence in function from the ancestral copy. Our results show that tandem duplicates are abundant in maize, arose in bursts throughout maize evolutionary history under multiple potential mechanisms, and may provide a substrate for novel phenotypic variation.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/09/01" />
<meta name="citation_mjid" content="ggg;8/9/3049" />
<meta name="citation_id" content="8/9/3049" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/9/3049" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/9/3049.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/9/3049.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/9/3049.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200580" />
<meta name="citation_pmid" content="30030405" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3049" />
<meta name="citation_lastpage" content="3058" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Thomas J.Y. Kono" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1388-3194" />
<meta name="citation_author" content="Alex B. Brohammer" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4639-4119" />
<meta name="citation_author" content="Suzanne E. McGaugh" />
<meta name="citation_author_institution" content="Department of Ecology, Evolution, and Behavior, University of Minnesota, Saint Paul, MN 55108" />
<meta name="citation_author" content="Candice N. Hirsch" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics" />
<meta name="citation_author_email" content="cnhirsch@umn.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8833-3023" />
<meta name="citation_reference" content="citation_author=P. Anderson;citation_author=J. Roth;citation_year=1981" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. Athma;citation_author=T. Peterson;citation_title=Ac induces homologous recombination at the maize P locus.;citation_pages=163-173;citation_volume=128;citation_year=1991;citation_issue=1;citation_pmid=1648001" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=G. Blanc;citation_author=K. H. Wolfe;citation_title=Widespread paleopolyploidy in model plant species inferred from age distributions of duplicate genes.;citation_volume=16;citation_year=2004;citation_pmid=15208399;citation_doi=10.1105/tpc.021345" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=R. Bouckaert;citation_author=J. Heled;citation_author=D. Kühnert;citation_author=T. Vaughan;citation_author=C. H. Wu;citation_title=BEAST 2: a software platform for Bayesian evolutionary analysis.;citation_pages=1003537e;citation_volume=10;citation_year=2014;citation_pmid=24722319;citation_doi=10.1371/journal.pcbi.1003537" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=A. B. Brohammer;citation_author=T. J. Y. Kono;citation_author=N. M. Springer;citation_author=S. E. McGaugh;citation_author=C. N. Hirsch;citation_title=The limited role of differential fractionation in genome content variation and function in maize (Zea mays L.) inbred lines.;citation_pages=131-141;citation_volume=93;citation_year=2018;citation_doi=10.1111/tpj.13765" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=S. B. Cannon;citation_author=A. Mitra;citation_author=A. Baumgarten;citation_author=N. D. Young;citation_author=G. May;citation_title=The roles of segmental and tandem gene duplication in the evolution of large gene families in Arabidopsis thaliana.;citation_pages=10-10;citation_volume=4;citation_year=2004;citation_pmid=15171794;citation_doi=10.1186/1471-2229-4-10" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Chia;citation_author=C. Song;citation_author=P. Bradbury;citation_author=D. Costich;citation_author=N. de Leon;citation_title=Maize HapMap2 identifies extant variation from a genome in flux.;citation_pages=803-807;citation_volume=44;citation_year=2012;citation_issue=7;citation_pmid=22660545;citation_doi=10.1038/ng.2313" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=D. E. Cook;citation_author=T. G. Lee;citation_author=X. Guo;citation_author=S. Melito;citation_author=K. Wang;citation_title=Copy number variation of multiple genes at Rhg1 mediates nematode resistance in soybean.;citation_volume=338;citation_year=2012;citation_pmid=23065905;citation_doi=10.1126/science.1228746" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Dong;citation_author=Y. Feng;citation_author=D. Kumar;citation_author=W. Zhang;citation_author=T. Zhu;citation_title=Analysis of tandem gene copies in maize chromosomal regions reconstructed from long sequence reads.;citation_volume=113;citation_year=2016;citation_pmid=27354512;citation_doi=10.1073/pnas.1608775113" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. K. Dooner;citation_author=J. L. Kermicle;citation_title=STRUCTURE OF THE Rr TANDEM DUPLICATION IN MAIZE;citation_pages=427-436;citation_volume=67;citation_year=1971;citation_issue=3;citation_pmid=17248520" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=D. M. Emms;citation_author=S. Kelly;citation_title=OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy.;citation_pages=157;citation_volume=16;citation_year=2015;citation_pmid=26243257;citation_doi=10.1186/s13059-015-0721-2" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=L. E. Flagel;citation_author=J. F. Wendel;citation_title=Gene duplication and evolutionary novelty in plants.;citation_pages=557-564;citation_volume=183;citation_year=2009;citation_issue=3;citation_pmid=19555435;citation_doi=10.1111/j.1469-8137.2009.02923.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Force;citation_author=M. Lynch;citation_author=F. B. Pickett;citation_author=A. Amores;citation_author=Y. L. Yan;citation_title=Preservation of Duplicate Genes by Complementary, Degenerative Mutations;citation_pages=1531-1545;citation_volume=151;citation_year=1999;citation_issue=4;citation_pmid=10101175" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=J. J. Han;citation_author=D. Jackson;citation_author=R. Martienssen;citation_title=Pod corn is caused by rearrangement at the Tunicate1 locus.;citation_volume=24;citation_year=2012;citation_pmid=22829149;citation_doi=10.1105/tpc.112.100537" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=C. W. Hill;citation_author=R. H. Grafstrom;citation_author=B. W. Harnish;citation_author=B. S. Hillman;citation_title=Tandem duplications resulting from recombination between ribosomal RNA genes in Escherichia coli.;citation_pages=407-428;citation_volume=116;citation_year=1977;citation_issue=3;citation_pmid=338920;citation_doi=10.1016/0022-2836(77)90077-8" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=C. N. Hirsch;citation_author=C. D. Hirsch;citation_author=A. B. Brohammer;citation_author=M. J. Bowman;citation_author=I. Soifer;citation_title=Draft assembly of elite inbred line PH207 provides insights into genomic and transcriptome diversity in maize.;citation_volume=28;citation_year=2016;citation_pmid=27803309;citation_doi=10.1105/tpc.16.00353" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=J. D. Hollister;citation_title=Polyploidy: adaptation to the genomic environment.;citation_pages=1034-1039;citation_volume=205;citation_year=2015;citation_pmid=25729801;citation_doi=10.1111/nph.12939" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. R. Hudson;citation_author=M. Kreitman;citation_author=M. Aguadé;citation_title=A Test of Neutral Molecular Evolution Based on Nucleotide Data;citation_pages=153-159;citation_volume=116;citation_year=1987;citation_issue=1;citation_pmid=3110004" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=H. Innan;citation_author=F. Kondrashov;citation_title=The evolution of gene duplications: classifying and distinguishing between models.;citation_pages=97-108;citation_volume=11;citation_year=2010;citation_issue=2;citation_pmid=20051986;citation_doi=10.1038/nrg2689" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=Y. Jiao;citation_author=P. Peluso;citation_author=J. Shi;citation_author=T. Liang;citation_author=M. C. Stitzer;citation_title=Improved maize reference genome with single-molecule technologies.;citation_pages=524-527;citation_volume=546;citation_year=2017;citation_pmid=28605751;citation_doi=10.1038/nature22971" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Kimura;citation_title=THE NUMBER OF HETEROZYGOUS NUCLEOTIDE SITES MAINTAINED IN A FINITE POPULATION DUE TO STEADY FLUX OF MUTATIONS;citation_pages=893-903;citation_volume=61;citation_year=1969;citation_issue=4;citation_pmid=5364968" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. R. Laughnan;citation_title=THE ACTION OF ALLELIC FORMS OF THE GENE A IN MAIZE. IV. ON THE COMPOUND NATURE OF Ab AND THE OCCURRENCE AND ACTION OF ITS Ad DERIVATIVES;citation_pages=375-395;citation_volume=37;citation_year=1952;citation_issue=4;citation_pmid=17247400" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=D. Leister;citation_title=Tandem and segmental gene duplication and recombination in the evolution of plant disease resistance gene.;citation_pages=116-122;citation_volume=20;citation_year=2004;citation_issue=3;citation_pmid=15049302;citation_doi=10.1016/j.tig.2004.01.007" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=E. Lyons;citation_author=B. Pedersen;citation_author=J. Kane;citation_author=M. Alam;citation_author=R. Ming;citation_title=Finding and comparing syntenic regions among Arabidopsis and the outgroups papaya, poplar, and grape: CoGe with rosids.;citation_volume=148;citation_year=2008;citation_pmid=18952863;citation_doi=10.1104/pp.108.124867" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. C. Mak;citation_author=Y. Y. Lai;citation_author=E. T. Lam;citation_author=T. P. Kwok;citation_author=A. K. Leung;citation_title=Genome-wide structural variation detection by genome mapping on nanochannel arrays.;citation_volume=202;citation_year=2016;citation_pmid=26510793;citation_doi=10.1534/genetics.115.183483" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. G. Maron;citation_author=C. T. Guimarães;citation_author=M. Kirst;citation_author=P. S. Albert;citation_author=J. A. Birchler;citation_title=Aluminum tolerance in maize is associated with higher MATE1 gene copy number.;citation_volume=110;citation_year=2013;citation_pmid=23479633;citation_doi=10.1073/pnas.1220766110" />
<meta name="citation_reference" content="citation_title=Evolution by Gene Duplication;citation_title=Tandem duplication involving part of one linkage group at a time;citation_author=S. Ohno;citation_pages=89-97;citation_year=1970" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=E. Pessia;citation_author=A. Popa;citation_author=S. Mousset;citation_author=C. Rezvoy;citation_author=L. Duret;citation_title=Evidence for widespread GC-biased gene conversion in eukaryotes.;citation_volume=4;citation_year=2012;citation_pmid=22628461;citation_doi=10.1093/gbe/evs052" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="R Core Team 2018 R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. URL https://www.R-project.org/." />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=R. L. Rogers;citation_author=L. Shao;citation_author=K. R. Thornton;citation_title=Tandem duplications lead to novel expression patterns through exon shuffling in Drosophila yakuba.;citation_pages=e1006795;citation_volume=13;citation_year=2017;citation_doi=10.1371/journal.pgen.1006795" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=G. M. Rubin;citation_title=Comparative Genomics of the Eukaryotes;citation_pages=2204-2215;citation_volume=287;citation_year=2000;citation_issue=5461;citation_pmid=10731134;citation_doi=10.1126/science.287.5461.2204" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=J. C. Schnable;citation_author=M. Freeling;citation_author=E. Lyons;citation_title=Genome-wide analysis of syntenic gene deletion in the grasses.;citation_volume=4;citation_year=2012;citation_pmid=22275519;citation_doi=10.1093/gbe/evs009" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. C. Schnable;citation_author=N. M. Springer;citation_author=M. Freeling;citation_title=Differentiation of the maize subgenomes by genome dominance and both ancient and ongoing gene loss.;citation_volume=108;citation_year=2011;citation_pmid=21368132;citation_doi=10.1073/pnas.1101368108" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. S. Schnable;citation_author=D. Ware;citation_author=R. S. Fulton;citation_author=J. C. Stein;citation_author=F. Wei;citation_title=The B73 Maize Genome: Complexity, Diversity, and Dynamics;citation_pages=1112-1115;citation_volume=326;citation_year=2009;citation_issue=5956;citation_pmid=19965430;citation_doi=10.1126/science.1178534" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=F. Sievers;citation_author=A. Wilm;citation_author=D. Dineen;citation_author=T. J. Gibson;citation_author=K. Karplus;citation_title=Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega;citation_pages=539-539;citation_volume=7;citation_year=2011;citation_issue=1;citation_pmid=21988835;citation_doi=10.1038/msb.2011.75" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=G. P. Smith;citation_title=Evolution of repeated DNA sequences by unequal crossover;citation_pages=528-535;citation_volume=191;citation_year=1976;citation_issue=4227;citation_pmid=1251186;citation_doi=10.1126/science.1251186" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=N. M. Springer;citation_author=K. Ying;citation_author=Y. Fu;citation_author=T. Ji;citation_author=C. T. Yeh;citation_title=Maize inbreds exhibit high levels of copy number variation (CNV) and presence/absence variation (PAV) in genome content.;citation_pages=e1000734-e1000734;citation_volume=5;citation_year=2009;citation_issue=11;citation_pmid=19956538;citation_doi=10.1371/journal.pgen.1000734" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=R. A. Swanson-Wagner;citation_author=S. R. Eichten;citation_author=S. Kumari;citation_author=P. Tiffin;citation_author=J. C. Stein;citation_title=Pervasive gene content variation and copy number variation in maize and its undomesticated progenitor.;citation_volume=20;citation_year=2010;citation_pmid=21036921;citation_doi=10.1101/gr.109165.110" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=Z. Swigonová;citation_author=J. Lai;citation_author=J. Ma;citation_author=W. Ramakrishna;citation_author=V. Llaca;citation_title=Close Split of Sorghum and Maize Genome Progenitors;citation_pages=1916-1923;citation_volume=14;citation_year=2004;citation_issue=10a;citation_pmid=15466289;citation_doi=10.1101/gr.2332504" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. C. Tan;citation_author=J. C. Guan;citation_author=S. Ding;citation_author=S. Wu;citation_author=J. W. Saunders;citation_title=Structure and origin of the White Cap locus and its role in evolution of grain color in maize.;citation_volume=206;citation_year=2017;citation_pmid=28159756;citation_doi=10.1534/genetics.116.198911" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. Thornton;citation_title=libsequence: a C++ class library for evolutionary genetic analysis;citation_pages=2325-2327;citation_volume=19;citation_year=2003;citation_issue=17;citation_pmid=14630667;citation_doi=10.1093/bioinformatics/btg316" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=L. Wang;citation_author=T. M. Beissinger;citation_author=A. Lorant;citation_author=C. Ross-Ibarra;citation_author=J. Ross-Ibarra;citation_title=The interplay of demography and selection during maize domestication and expansion.;citation_pages=215;citation_volume=18;citation_year=2017;citation_doi=10.1186/s13059-017-1346-4" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=C. J. Weadick;citation_author=B. S. Chang;citation_title=An improved likelihood ratio test for detecting site-specific functional divergence among clades of protein-coding genes.;citation_volume=29;citation_year=2012;citation_pmid=22319160;citation_doi=10.1093/molbev/msr311" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. U. Wingen;citation_author=T. Münster;citation_author=W. Faigl;citation_author=W. Deleu;citation_author=H. Sommer;citation_title=Molecular genetic basis of pod corn (Tunicate maize).;citation_volume=109;citation_year=2012;citation_pmid=22517751;citation_doi=10.1073/pnas.1111670109" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. I. Wright;citation_author=B. Charlesworth;citation_title=The HKA Test Revisited: A Maximum-Likelihood-Ratio Test of the Standard Neutral Model;citation_pages=1071-1076;citation_volume=168;citation_year=2004;citation_issue=2;citation_pmid=15514076;citation_doi=10.1534/genetics.104.026500" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. I. Wright;citation_author=I. V. Bi;citation_author=S. G. Schroeder;citation_author=M. Yamasaki;citation_author=J. F. Doebley;citation_author=M. D. McMullen;citation_author=B. S. Gaut;citation_title=The Effects of Artificial Selection on the Maize Genome;citation_pages=1310-1314;citation_volume=308;citation_year=2005;citation_issue=5726;citation_pmid=15919994;citation_doi=10.1126/science.1107891" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. D. Yandeau-Nelson;citation_author=Y. Xia;citation_author=J. Li;citation_author=M. G. Neuffer;citation_author=P. S. Schnable;citation_title=Unequal sister chromatid and homolog recombination at a tandem duplication of the a1 locus in maize.;citation_volume=173;citation_year=2006;citation_pmid=16751673;citation_doi=10.1534/genetics.105.052712" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Z. Yang;citation_title=PAML 4: phylogenetic analysis by maximum likelihood.;citation_volume=24;citation_year=2007;citation_pmid=17483113;citation_doi=10.1093/molbev/msm088" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Tandem Duplicate Genes in Maize Are Abundant and Date to Two Distinct Periods of Time" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9.cover-source.jpg" />
<meta name="twitter:description" content="Tandem duplicate genes are proximally duplicated and as such occur in similar genomic neighborhoods. Using the maize B73 and PH207 de novo genome assemblies, we identified thousands of tandem gene duplicates that account for ∼10% of the annotated genes. These tandem duplicates have a bimodal distribution of ages, which coincide with ancient allopolyploidization and more recent domestication. Tandem duplicates are smaller on average and have a higher probability of containing LTR elements than other genes, suggesting origins in nonhomologous recombination. Within relatively recent tandem duplicate genes, ∼26% appear to be undergoing degeneration or divergence in function from the ancestral copy. Our results show that tandem duplicates are abundant in maize, arose in bursts throughout maize evolutionary history under multiple potential mechanisms, and may provide a substrate for novel phenotypic variation." />
<meta name="og-title" property="og:title" content="Tandem Duplicate Genes in Maize Are Abundant and Date to Two Distinct Periods of Time" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/9/3049" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Tandem duplicate genes are proximally duplicated and as such occur in similar genomic neighborhoods. Using the maize B73 and PH207 de novo genome assemblies, we identified thousands of tandem gene duplicates that account for ∼10% of the annotated genes. These tandem duplicates have a bimodal distribution of ages, which coincide with ancient allopolyploidization and more recent domestication. Tandem duplicates are smaller on average and have a higher probability of containing LTR elements than other genes, suggesting origins in nonhomologous recombination. Within relatively recent tandem duplicate genes, ∼26% appear to be undergoing degeneration or divergence in function from the ancestral copy. Our results show that tandem duplicates are abundant in maize, arose in bursts throughout maize evolutionary history under multiple potential mechanisms, and may provide a substrate for novel phenotypic variation." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9.cover-source.jpg" />
    <title>Tandem Duplicate Genes in Maize Are Abundant and Date to Two Distinct Periods of Time | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"05GkhlMvYYkX8puVvyI1A3cdi87B7WQ6ylyr6mj8Bmg","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490736","apath":"\/ggg\/8\/9\/3049.atom","pisa":"ggg;8\/9\/3049","ac":{"\/ggg\/8\/9\/3049.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/9\/3049.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/9\/3049"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/9\/3049"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/9\/3049":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022WqhDoYyA1gFh7CYAbvsrEkHhdx5Oh1d8azdC\\\\\\\/0A0EjEKCmpvwO4e0bpAdbfbLWkLjOKSWVXJTwZoOi32DUTZM7QsVGTZ7EQl0iNXUgqawLPXDIW6FmRG3tSXKkcPqISRXaHD7kB5L15njVCe7\\\\\\\/3sU8b4LdLrAWXUz6+b9DKU27ZYGv5AkudSNs6yXlrvgNy4pYXKOlBXce3clh04fxm5qAgSJf2Xp6PL\\\\\\\/ayT57ZFm8sKFkzMDjPmNsdeh1Tc6doej+KyxZV3Y1TGL5nSz2d3Gwy\\\\\\\/LE5G\\\\\\\/Yfr0NsLKwbFdW1BsqEgyBaNOOW2DP4a8q79Kmoo5aWLMHp\\\\\\\/FjmEMVmxTiHhnyMjRyaWlzzyF4w0+jSR0sbOSrjTfIhzy86t7jOhJOx6DXbbA1hrRVmpiILhdikZXQW2Mkgvna9v9pfLuB0ATbJw3epDG3e72muOo3oDXYaFkhgNLW2NF4h9rwl8+CXHaJ+MQzFgat8zyteFuJtXiwdRU+B+nUomlYQX+seNRm5McZmD4hxmd0pIsASmL3ZLrbgirDiugJwTkjwK1WBuMsuY1+uW1iG86lLqihf8kVTiZHvgYJa0t7\\\\\\\/XklvWei5bgsLsZKDbIm54nuhgWRLvpZge9JR2XOjAvBhaGadyzPeNbsOjGftvlew4dFmsr+JIylUzVDhOJPNbjYIPRvCWx\\\\\\\/6wCh+ret3lLr7GIsL4isbdYfKwaUrjqNODti0YP4GbCck83zucz3LrbP0ee6y3E1uaiQ72exQNI+Bu2HElRf69sGxDKGeq90vBgnAmAwKj2L8HVrTv+u\\\\\\\/fjQ4XYtT1S3Iftv20csXoy8FQoIVQVJS2Z\\\\\\\/P1H0DElBuzVfKtR9Z55KRi7qGUyr+3vea1nJIVixjEXz7m5UeE\\\\\\\/YP+dA6cxzBSIy1pjBGs6+G24eL4u9+YLdBDPg9O5nKovRrvjjBnmuPYKIum1oYxPxZDB2941fGkj4kyJfJvo\\\\\\\/WS+K1JQMU+qvivalRxrPy+\\\\\\\/wxscqmGmlLqZcErNoXFnbPYnQm\\\\\\\/LMS4yqRQExdKKqsVvITLTg6NXNGoyLw+jvAlWuYBNOmLWVZkqzQ0Mu9Zkf5q59DPsoHphHR0uKzpUd3CorAb+tLQXUmaMax9I8zJroJJayk+Qk436yyQDQ+8LCOaQrdDeN7WBr1AstUBhnzET2EeS7HTeFfDL0F5x6R6zeHOncFRTFpyuxhN6FP7QBaOBEcz4VBpGvLhBtDsOqYwov1KXqHkD+CYXiQo3Vi\\\\\\\/5zE0qOetUyyukSDinbFXxzwIaOHq\\\u0022,\\\u0022iv\\\u0022:\\\u0022sNuIIlB22YKnNyTbRqHUQg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c7d8af053b116feed1181e188313b4e297142f5c0c71e6e9641ee3e770693755\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Z2Jstr4aOl+97\\\\\\\/XAO5vWLgn73eNb2jPdhE8Z3TEr\\\\\\\/QL7tJFXpk3Y67x14DVixYliifeMz5Rfv+2GsVHZjLLHMN9S3VbICdPZBzT6dOMVcAENjXGE4L7LtlXSSMcMBysYkNVUhzwUMa8mx9tBA6\\\\\\\/OxkUGuPThDYYUJVEPWI6rW9ELrUtVDiz9GBCmbfJwJu1++SrDHYP6rMtGlTvElYxgcNdK+AzkSh6skc4Iml1YXc3rPXdZPUbKQlGB9N\\\\\\\/CCLgOGQ+g7qQ\\\\\\\/X9LGl+GdXyHBzH\\\\\\\/GQbh6jQeukchSfqIE5001xhqnjT6GXiLyUNvyfgJ5m3ycs+CBPycsAgpJSWRwtyJgmQnVqXTi0zgVzCepppCGSYIZU+k8Fbger9VU0nvrl2Md5AdPhGs3bNCuF4W56O27YcARYIDuM4q7XlwdEkHRnO0amSw81fKk\\\\\\\/2vBUBRNpX\\\\\\\/hI5tZQgWvVVnP1e5kGodjnHFs1XNVShzRBiJjx1x3pME1BRdoMdkaHFaN5W48QvDgqMNkqgpcumF2D2IorehbbqjVX9Bu7ji8NLdnslYbddua5kIHF7dmsIV9IwtVFhMS+LjYg7ihyDd5djX0Oi96IOCZCa0+ggZxD\\\\\\\/1vTBKd6yqJadW3X05KNXcpIMr9Wf8t12VVuWYWckApdOB\\\\\\\/7obpEIeBaitntf9HyzybOgZ\\\\\\\/dXWcDeh0VHKnOVx5XHEjXMTf4eIDq3+NA36tPaw5Roh+W7kQE0C0SzluQF\\\\\\\/GGIrpZrFSoViV\\\\\\\/bcr1J3qhS3sIKiPsoLIiFrREkYy6MD0EvuMdp9MVF02UT8sdGYt\\\\\\\/0l\\\\\\\/JsrZnW2Y6pMdxquwdT9BBEcLMLye42vhSY8ICYOIK9ogRm10RmT4gGIHZ2dCCK+csqJebtxWEXJYeY2+\\\\\\\/yIxfIpPaCUizPjnTbyjRAgT3dg2EMEB3JAAgeCpgLzZCh65xokSTSYVj3hUBc5BIUeFS+ERJoIj4VHgpK3BajPMi3Pa5JehFgkvaSmJgbKSWmf+QcAEvO7NsuTcV2TwNCtU3jBjTAR7Nx4NvPkCrc\\\\\\\/KsoMDayVa2Rm8ZG5vIAyqN6Nc6DfbntKkA2FMoi0HlWvH09Q66lYkrPXK8dqH\\\\\\\/BQ8zFdAIs+o2caFpruofKxyjogq07VcstmYTxLEnuPDjLeyJxEblzRxm+TrvtZWyxINDskqDgqXgh0QSmnNGhHTLg25vEc7oVeUxOfx83xSCzi6zpsXmoGOU6lfDttK7PBNih+nzG98M+lZyGYgYhS2rx2SdWs7I65XSkJePzfPe1vViHLOlEbTwgm4LSO5NHPOKQn89daEB4AkC4l9tWEIO2weBF80osBn0RjU91S4PZd9N2Nn9yXr8veLGjBG9OpYyMq5v+srrKUCljtYk7sRrz880mg8LAdTW8U4wm9pMvxT9l\\\\\\\/lQcjjiTiM9My0cw\\\\\\\/9u4lHXk+PVjwMhTT17d1zUvrUgkz30yhamFcVmDgUOOo7HEufd44y7vx6bCxcbWzvOdJyMKgwUtzMZsvw2YQp9RqVcb\\\\\\\/sTmb9geWMiuDo7C8kA9B\\\\\\\/4kRZGUMcD8KGPPUz6xQNL0MK9bEpyyO3W25CQRTMKRf6uurVTxg5eA4M0NevvkBgdaetGQ3FdvDFbAaO8CJPYeRp3\\\\\\\/S+vn7yWEuWBVnOY8BCb\\\\\\\/q+y24rQlp3U1X3KrARXghTkS6GnRW\\\\\\\/tQDKU28gL4SOxOyqfJDlOFaGKL\\\\\\\/PnD72fjlvlCKl2p1caz90ByvVT9WmJ2J+Ya5f4p4TN9GgCvtW7ECaVZu\\\\\\\/rsXwSTrpA7+aD6H4BJkPWe2hl27N2ict3a60g449eZyZCMoPCPOUh5zBZY4vI3FjFSW5201QbvVFtlgvj2tvs8sqS6+g7PMvVk08TApTLhC1QQMvpIXHLW3DThIGLmjxPsIG6EOO1lBaHvko74z6xgkHSkBdCelwjHvskuk\\\\\\\/eNMJkKgEpIu4Xga7nChsF\\\\\\\/7rc+ng2R4Nk8qJqXvcCmQvd8voqhNgnZQI2DCFWsUqJCv6dvqvK1xVHn1MYxiF6VRq\\\\\\\/nqjw\\\\\\\/BsEXATzT3PHYFgkwi313q8nYeKPAGFXZ81jcdF0v+hAjX29dNYjzsOpVkNfFfA4CRROzEP6SS3IwEE\\\\\\\/XSsLN4cfwjF5IRYOpWL2b6+i6lwU5Izvquvam8M6sSUgzkKMiMfLAv2j21de99VKlgDJwJiV31BdI3+EDC+54h1iXsRAYptSdMIuvT6v4Ld\\\\\\\/NtTMWuqD8MC6bDFOwU7Dt0SVI5CU5IsgFoGrCFS5FzO66lITtD9Nu\\\\\\\/La69xR0efA2D5mT92eIztQrulWuCOX\\\\\\\/s4Begg36m54wA+oIKQaJWgpDkWeROSrXFMMNypGASlal4hb2RlxkjUMzQfPqIWgWtNtLXV8TBaayyrzUGz950ReirpPaDVJDDuYoiF2LfSmIignqL4aMly9B\\\\\\\/EPDnwrCAe3efDFc06MQY9FNiTRYOPuiCToxV6e+uezkD1Dxw3BqAzbrrCUvmuIRNe7vH3NmgWu9ergWDapHvBdjbtuy8AWVa9iXpLCKIQshGwmedT3p3nJenOhGvGIKIF\\\\\\\/7+j8Yzppzuv1P7doeRjepeZELygNe7\\\\\\\/8Mo+Djz4etLIoLedHYMdGX1cI0j0jcpdfHy96o1cXvt6m1mAERluP6D6d41ZzEgqERcF+RapkTUBrsJkVEmPVhbdMFahrbfiYUaG0kXluAqe6iiBb\\\\\\\/5PmaO1BdBquS46a5JeB5AXfODX0i\\\\\\\/iq5D97+n32bBR9+KKnHnAheJ7\\\\\\\/V5ot5b6s3xYQj6yy1Hf1uIJmpAKiP9sXx4mQsk1p3uJwjWFAeUops5KH53rMLMpGFTKxeWVHyqzfU05qMivyf3CslPugDvuDd2bjLwZIJrVMffvFuMsYGKCPbzx9PfmAZY2mXNLADXrx81d3pw20kQF0AAMNIuj9wyq8mq2V3fnulU+CBFsocn8Dg7vSvZljU0tC67KSd7OFMC9\\\\\\\/EEZ6Q9knvfRukzH\\\\\\\/DyDsExzFeqVYY+msnSRsnB+NWpuc67qqaxDoxkEORZZnQqh4K\\\\\\\/lD6FGzMTTEqx2QfHeg1PgvP5DE6Dbw0jQlKWa8PCA+WSmJlVUGU+zHo32w9dY23AaJycrjuicVu1Lh8W3n7vBbCSQS36XDAPSOsCMPN6PY6NIDOuXRKsXKd2Fxe6GHwt1IFi4FWaiPHG36hITv0JBGCmFgo0Dxg435HpZ1VX2kg3\\\\\\\/e10ZvgqD7QWJ1WA0IwCqkHXfNLJzFgxeEIZdgSoi+kNmuhqxYslHceYHZGDnQtOn8NRQ0k1IGiAEDX2K3JKn\\\\\\\/CHwEWR4hYnTjY31eD7bvEirJQ+ehRslksoHDtoLhT+3xbe0UnIo+iqHBWyhWdb\\\\\\\/NhpoR2YOq5MuOwkRWqHmSByFv\\\\\\\/yyrMKrYL67J+sOcya0onJn8OAbKVYft2QoGEViPzmTgDEtBJeIiJ+qexpAp7xoPLGXYFHgNK\\\\\\\/nr2qjvYbmyMCnP8z0SG4Y6fxDYeHkYoHeww9v+UcxG9HNUK0Xr5uvYJ4vnd30KVDlfVpMOl52XMOrhoAJcsA0ww7HBNCeoH2Rv2LZX5se4MnGHTWMjY4VFS6FYBYEOL5Wx4NWnYqaGvEC23fQ\\\\\\\/0afEBzxamFDnTbKzBBRGkr1hBbgJTrUYFLUwP1gpHEuYdVUr0t96oyiniBDcauZGkYrxrhjY48YVXH8LvGiM0zgBnR5bBsUGUBJUIRTYZ8SFjcYraQIQBH7y0HUsvokhVJN23NIn\\\\\\\/fn5UoUCFJX0hFi40C7YKstpYCt2C+elpXvPQ8ipCmqHcowM7r9DlX5HTmMMx+FUjiPQKCN8y8O2uhH65L+AWqWjkuKSQqiN9QRLMVrDqTzAXCniZDtQN5U51QcRYinPZtFdEm\\\\\\\/F0K15HI9Db8a7OM7BUs4hmxPYZSMBzM6Ozg9bZYCxOp6hG9tK2haQHfWcS+DlvO8KSc2GTznBMQu+dKFWxlg+7Uz\\\\\\\/YiL+MjKqSCdfcxYlwmpe0sVf0GSAUK1jH\\\\\\\/GsONImdkmet0\\\\\\\/0wyF2Dmp7\\\\\\\/pLCxWiZi2\\\\\\\/ReczcLpAkf5r5seP4HGICqPjxX2K6EuCYCU8WJF\\\\\\\/xFBfscF0MMbiQXsPkvMIqIK7HtoV9EYOHwxrhEhiUAwPfMw8iYEVn9XxH7av5\\\\\\\/7dXa3vG32PQgYCv+uC+Uwj6CzT594OqvSBco\\\\\\\/1rH1UxwyCa\\\\\\\/4ntlIdid1uOY6snwXLRyHKuldaLpoqapNCB61s5fjFOeDwsqPtSCspWoVPRDAI\\\\\\\/te5aPWttCa6C+tXMSQuzUQ8s\\\\\\\/wRSV3BGeKr3xYP0lY5b4zt4bmlp1tTrvcKxe1yAySLKqWf7x3JJUA2t6H33JK+WNQBZ3PN3bgIH+9WsAyAMoUXKYP9af4XBeQV7pjTIsgCANlpllQFbrNOxqAPFO0QN+RyKCo6TOwTFW\\\\\\\/bvY4f+RvqpgZb07i8s4LHbM3CWAe2kIBjPnY9UeE32K8WR\\\\\\\/7x5azTyhQY0gUw7p\\\\\\\/9HB\\\\\\\/0F01uFlUaYnwgUquRmDaQU58pBS4cCbrlnlu7vvjrWvYaTq7W0O0wlkvq6r6M+I\\\\\\\/xoFcwnzddxOYx7gtLNxCZfXdVXNX3r9TPRblPgb6DcptsJuSg6jG2tmaEM2WI18lm8RNqNr5REVBT6rpT1Fka8M3RmcjKKxFJ8WypvUlYPbMjOkvtSgqkOKiylwn1uCC4Ah4b1ZTkG3W6eDnKXb4yMw0IOu4SGNFWJL1\\\\\\\/z5BnIATrVOOgUvEBwhkM8kmdPhnfA12Hpidm7Du13lPMwQ8uj5SKEwiNd\\\\\\\/M0IOSwA2G6c1Wlp+DqxRMAiTpTkJE7oG26\\\\\\\/WDyigGkAJnQE67FOe5lVfOfct7mI9BoYS4sgu5kwWzrVFqhL0Q96ArPurw0mCgzonFOmExCDiLGr8YMas7ybsM3xEDdm44rH2uNi7ndKRmbAkO8SdUltb8YXdOHd2y6mrP0W5skze6m0eYfEBbx4EkxIaIhPj7hB3543j8iVzej61ms433gOatgOqtL1FTpr+xFDOTpqZrIVNYyzHQIx6vY7hvidvTs2t+ZJ48X15+ZhhFVQ4UgnTE7qlF2IduH7r9T+u0m4ISRypDZaQHA4mQIeNQsRuBpp5ZiFPlvaqGkYVwRIoy+M81Iz60jFDmmiYp67GyglzPeDieZzKr0JDlvYBQpD1K6mPkPNGvF7BvVaVNYLb9GOMdcRqrOgNUialGIe6XBbPt74Bm5OfM9WDqDZ7gRv4NNQGOeY2sIIshMteSKfjk1jck9b8VaXHd8dRkscvA9g5ki6lB7ENfr5LGDMoE1dnmHfEz4zi82+b5h04T2dzXpdnHMrOO5hHxTx0KIK96Yp8DQsw+WmLAB5B9+LDXQLRHeCkQbWmF98qCphsQaWzuTqN\\\\\\\/iis7R+7vqyiTWBEdfyfMW0NM2gMmpu09nage4uUQu0gXBXSHE+\\\\\\\/FUrcwr877J6N6Y8MPtTK5w8Z3OM569VzLlakBG6iCyOYK0nxvZ0iznjxtq5MhbiXd+7NWJi348JU5BpPJEAH9E4BU+L+6cQPxwUinl79UFbYmsFDJ+O91esSB3l9MMg6ZGMiH6rNR0+SH9Zj4ZD6Tl6hPZZNsC3GqtHEHgH9sVLB+D+0m0gEZn8riWck2tkNvyDxgkR0WChtmQIHPoXDiesUPfvxk1pA5H65jZmRVk5lxkzEsG2ecbTljivvCpAYaNVDjR0kJzClbZc74JC+DbgILwdsaWTqi\\\\\\\/3AFNS3TSoSjGydhic+iBJBHKlMVOD1zIDOIDmDEwTmc8GfyHPLEht+6MAnfR2LIhxTBooUxN\\\\\\\/gIH7MkMQbIN2h7dRuH80KYOLRSAx+JG\\\\\\\/7mMRXQjJw9urtX9zSC9\\\\\\\/vmkumdBWoFB3cNMx4lVshakxhJ7G2Du4d1B0g2b\\\\\\\/DRsf9SKlyPi7GiWAbSetecXMWYJ4NyAIF+IuZ4XwI8WnVlpI0MomXIwsIjyfSBwYyZTGtsVtAfdG\\\\\\\/TawWY\\\\\\\/ezbhfdC05QDAAcM5K1IF16SGRRWKGRlGeM1vWSHrB+YuCXB5DzgSjOivZj6z9o4IpzIICrSidZANt2mx+KsjGLxgliaTYnXwjHSmY1CRHrztaYOS6gQTGFexJU0xYqnox3Wjg+Tc3yZ9Cq5GZVBkhmVItw6UXOcde8PLHPK22mMDUmZKhtkhAyV0HUHgFsyHBzYsL26rH6SzaZge8ZlYk2NzHk3FgM3uxpoEF4sv7d7ZmZz19sd3q+A\\\\\\\/iqLe\\\\\\\/Ojv6beIoYrGpLEV2k5D0WxIfJEhmM2ZRYIvk4Nv0KdoXSh4Ka67Nt3y9epAARBR588p50bYDvU5w2kTV0Lc\\\\\\\/33uRMi80z4bQXmY7LWi008+VsNAc7yjHj3s2QA7WJbDbt5Xir2LSreG8AgxseyXHoi\\\\\\\/uI7ccdVpT2vvFt4en+m0Z+YJ22dQciYerjAz0F8NmDBjSgk3vG1NzwKtBB+s2mJa6tAcgB9FeSl+tKmybG2qVs5xhATobu0gD02w8S+04+dyK7ihhY\\\\\\\/uRAsRQybeouJYZgw5rHSJ2+H1a65GdwBwmFtlf\\\\\\\/4\\\\\\\/99f7pt4h7UoMANju2z1NmHVxN9B8pbfJGQQA4NTXln62+ooJWU+xdid6RFCNo\\\\\\\/ypfIbxr51q\\\\\\\/YS3\\\\\\\/LDxWbsYguNbHxJMyeA9Tb0wEJIRXO44zRqevf6CPsSYpgCSf4KFA5V5YZ\\\\\\\/kvJWBm4MHnBxitvE8cHOJGohnxVC0EFI+k7SHsp5R844lokQguukS6xey5Uv+4NYy7WvTM\\\\\\\/A2RhJCdPObIwRxPg77sqN6s5LRRJeP8pyo+bVEN7PWmzYCvGbgurm47axSgtc4Fel3iNcOcMob+UklK\\\\\\\/fD7pVTIGKoaI0pvR65eZgUa62HSYcNaCHGXsh3dHAeDQkahNEqQqCmZ1HFXL0XNeEkWxIQHlWj4JyvS0MnVGGgdRUab5SApLGHOtUIAXQ+9\\\\\\\/O3pE08w3Vh8jE\\\\\\\/VA0odDmHihJa4DeHCQV+u15CsJau3vDt+TgDd9dXVqCb62lLGXEmW9ewGxTMcKIxGG10z5rray\\\\\\\/+8DRYFNNtm\\\\\\\/eD8G0vmhJKqVdfYhgOkCpjCm9epi8QDEwcpSJ4FktyaYHkTFArdctXnmsy5ZAcmV20z4zrgqmlTUYOgPAS7A+oVI\\\\\\\/s6jpva2Vg4Y1CWRTAbMfhf29JBYG4qx0eucnDvNdatdqmCIN8bHRwRYdCbXS3YQMD1mKI1BtfRdzwiku\\\\\\\/byjdyjc0pg1SWpSUa5So1tdyBOF1PnECR81+hhyvPLh34+fVrEgrkCRAEVt8oIc+5MykL0dhXqayvlpKJr\\\\\\\/ZUJB6Me0h1LIdVHplhIez+sQsjYCIE7lY3PnfeS5s9sUFbGRuYdlaxmRnqiqaF7dHy2OyfSTJEUsCuRTfX7HgqRIhZpIS3r7RajfEyibsz0KGA1jiZSbQosVAovYGirFW4rOYBE7ZNxSfhT+Jva3fCAC67nMw31bTUOXygHOWQLegBgt6iyZ9gm64tkJycmPEiVDLF4qgkz0YncBfd\\\\\\\/pc1JV5APKMK4mIjlDl+Txsa38p8becD9L79rLlTf2QIHexTbCra+Vc64FuUUtrH4pnNm4ijaBPZKHsaykQJD19MD8APBMK813vRgCY2nKEMSiSS+mA+XWCmO9Fo9FziHTQ7ILe\\\\\\\/K5x19HLMGbHuwXqPuAa899fJu1BWGmT0XBmHaBD8Vj7LVa\\\\\\\/aob\\\\\\\/7nOSiCoImwYi9HbfZhhQcUk61uuv6ycqRF1Shj1eVlqoIyeI3X+agxW197z6TFMzjf8N7tDhvPeZIUJlaE6AODeJ5ZWVqBTJKkwxbIMoOiZDDi1gi7xQB1GL8kyEoxJDvDIRUXTm6cKpalJuk3VMAgs56yqHXhetCCBc56P+Z7paE+g+z1vOYghRGpl8H+iOLnBO3BR51D7anfwe0M0x9RNvUHFbk2PipA3\\\\\\\/j7PMm25JA3dqexdGy9olwe4Ar9AEW3e0F0yYBjJstSgyTZYdtWiXtBeU4g2zJR+rXGfSEQJ31jqZ9o7O278K7izpgpTO2ZuRPR74fxUUMRAtyTp1ES844EWE4BRYsNmqmX0heoAAx8NviLIfZYpJMzmMZgD1VRZp4KPEMHI8HoVFaOOmb2T9OU8\\\\\\\/l88XyOj48zPAXBSRpzqYg0nsGUEC\\\\\\\/H\\\\\\\/sjnbipHjy1NHdf99GS5KhLFxttuPTE2lzVe+ogh2g9fwo8agk1z+9DwEcBM4jxycjSpLqI1mWXYyyodZyW84E5zH6YO3bH0UehFuFCmU4mBx3TuEP6t6xYGXcYlGjmltb3h46ewHHe42+KLfOConG6G0ZvX+yCII2gjH+cMBq6qQlJKYBf7MCopudvbqi8PmYIppJKWK4XewHVzYA382Ars2vMjeo5ilqrRPEttnzF4Htz2FexCcAtu4WMeYCjX8jSxW0aM+BfRhtwjemhIybi9Lkeor9fcJgQy2mve10a0QxOvzjh4670ZKizAE8DaFe3f5MnIsllje2FeovkAMV\\\\\\\/twCtAKxzZBWVolM2QFkyjit8ek0MK3hcKdll1C3nk6DGfbQz5HEzDUZizVZg5Rq7sQfIbNdtfKEJEnq00vtQGIc7gqpKhgt0kakRcvKUyJ6IghiaH6K\\\\\\\/NkDwcHW2l9E04kItyjLdri5Ybv5TN8Dz0YfvHtO7zMLIjdNYkNc\\\\\\\/OuL9VX0\\\\\\\/3x65Em+wlpdp+2mnMcTWaWEfB2i\\\\\\\/Ve0Mecf+vfB7gVM\\\\\\\/RNkx5QF6sJsR0RmqH5p5hQcVQ0XsZagKMH8T0BmU+egfUF92qtpt3vxWln\\\\\\\/u85zSRqHWRO8vIFRZRIvyMcu1nQL281YkpYHjCwApty\\\\\\\/ue2ZsDLK35N\\\\\\\/UoIrj0o8VN3m6WEcNrxM6hnop\\\\\\\/eJ5aTsgRz499mMv7ZhUU1v\\\\\\\/arh+K3oDqCZuZlbLjOmsxmjFeNow317cNjvEnz3e4DpQ\\\\\\\/z2sDFM9TTXmCP7OdU\\\\\\\/BTqP1fssPKkmL2NpmI0hnjc26yTNhfRJtBxws9WSemmm+7dUBg6Gq3zrfpUrrl\\\\\\\/QDgu37\\\\\\\/EzhqXcVgGjcCYHX\\\\\\\/q0OhjkBpiG9ZF60EHoIjK493ERt5C6+a0FRZkJEibEhTbT6TJIzQzP4gguLbdZeAEJ+WopDlSs4Ri2\\\\\\\/lHvvdig9cGq0Z2D2MxDLqbTY5gm5oiR37XJCJcLgHDqcJloxwXaN4yiaiCC3tcFLkVjAJzKyxmI\\\\\\\/zlrb1mrIiMWThBOjOJYwdRuz35x9G0p3UcxkCFGW4jWEF2IbjYXI7NsikRNev7XKWy2shPDobDhUK70d+WO1Ioe0Tf9i9ze7QZOhnupTRlSelRYS+mqs22vQFp64ug6f2E8Xq1xwwqUvFkNx5gx2RxZS8qWuYvPmQpGNOsSukozCamb4z1EAJ10ZE7kXZ6+bVSVHlaOQNHDz6\\\\\\\/W0iAeho9\\\\\\\/j9uDY0+0pX22NlXgT3guG4JhYONuQL3TcDYzN\\\\\\\/l2ai+l5hKYvZMULLv\\\\\\\/EePJWi8mFjbCKwqO3yRIaav\\\\\\\/sRpgeXEXf5NDEwz1FJUN+aumZp1YZAVTzrFnZwEsHegBlSeWOt7w7hAAKqxVeY2QNktVtVuWN1CDHZDZiGyr928ZxCP+1QERILiNRy8FBRPzm\\\\\\\/Ki2jOKy82ynrSop+PctYaKSa\\\\\\\/6f5vvKcTKrtzDMluG31Im6J31CUwWkGnExD0OLdoIvUkWcgU+h68BThspXjozmwbBlEb2bMC5i6jsMVZjwySzNxwlT79\\\\\\\/rOgdf66G86Tj36rMfAcf8IQkI\\\\\\\/b5yFC3VTwFjH0uuyPvQKKwoUhGdShx4zVmkHJ\\\\\\\/JPaz6jVdIjggdze8VmPbM4I4aARuitadF6ti8LW2JBUDa7G+VOZ0zOG6vdwOcJGjGRfXyYcJhEBBTLo0mjRqBchk2J7EzD9AGeFbsvdjZirQCXAGBAh1mu7f2teJsXkccVye+r8SCovpj5JKMpGN2IN8QUKDMOv60Q8xnyabVudXnfWy80HeWp0h1FtoBmKnEjIvsCVhzseXRLWYC3Q\\\\\\\/GPq8RtDLsYSsc+27maGPYQqjWp1qvGYQTqe0ql5JKGrfRFFd\\\\\\\/cuY992GWIPGNd4j8NE0S\\\\\\\/jshSh2ebxA5dt3AEgrh\\\\\\\/mGa2\\\\\\\/7k8d1Btm1LI2PtjoH5br9PfGb56uri7CJB4YJKIDnY7eE9dXTfOUZ4U14b\\\\\\\/HbEfRNQMZ4cpfhI3bFMYybcr6c7811TTHQQs+\\\\\\\/n1ik0wjhP49UsZUrsOI545Nj1DZBd03vyRVmigCNEZh4PT0wgoewm6Z4va3L1qvgHxH2HftblVAlp4tYyo9xHbeTaCQYobKZP6+FwplP4GgcPhKPaoZPyT6tEGbmCGsVSAk\\\\\\\/2iJBzjl31hyeJweC0xxr8vAJwjFE38sPzYnRNHXfIroI4Jt0z+f3Dn7A2RDHG7FPUxxnX4whAP0IY\\\\\\\/HbIraGf3Cl2M6BMj3O8e2hb5KR9\\\\\\\/Qc02b++ycShkG8EacfeugTxWYFPXPh+asLI2qenBwNkPRMsRu4AMpwa1nGDiSoHOyj6yRCSIBGSM+Y3GjOVDRSNoNYghx3i9UwH+dOQSGBpGkkRIG0TuwXe8p3bm8qfD5Pa3eCekoqKy9gAuet94HGM+AzcjdY8ZQf7Np5iba+ahXZCJODkQfcDKQVkZcMOk97I6M7bJVIYShOAW5hoPbCwrmOMouGCHIf9brzgpzQrrGhY0c8vfr7MZ+ENCY3UI\\\\\\\/WWrTxF3BeQKt17t0zknixBWtYIz1fbDkj7chSCBL7TG8Hprs9lrCKDiiKpb4pUPajH3zlGeC6O3U7Q8DasOi6NfB2w0aVgyeaRQ+7dxTv1IoSEkzGyIunTu3sTLfmX6gS+rf1S9xA\\\\\\\/\\\\\\\/NcwoR8VEiF4kuYE9eExLG39Fqftcm48u1pNC5+OWWmsMOgCU\\\\\\\/zdE4QaXPivWGcNqpa55i8L61J\\\\\\\/DIDlVatkbhIXVT+Qft3JQm9deo8j9bryv+Shxiik5Qr9ntGS+KPQfisQn2gZfOIw\\\\\\\/UpEcXYCXc2btcXVqgrvos2zq8qm5qLutJ0itbZ79zWjWZTLKMyH5vcD1gjhvkC1LjUSnPFG0MegoGx9NI\\\\\\\/9r7f6UO4qlBQoe4IIkb2BENt0n6S8iwSih3g\\\\\\\/90gPGjYWUWxFvZoojeUHthrJ5n7NrPnuRpoKaE9FIwd7xkSRkpzIDMTDBWGF\\\\\\\/tIZkOX23gfasL8B6rj2NS6LV2aCLiW6i0syJV\\\\\\\/xV0+fSqHU0xlySCNvRTGMYUmkxS4LYieSZOq47W2TeQ\\\\\\\/iTXZfQQkVa6FrjlNWTDriGM2npNOhorWxmF8mVjNI1Dvot8Fd1RnzsWEgBmkinE1Wi+cnTDrbFFCgeo5dFYMQIUabBp3PefcO+A4wHgON3gRXHvXrjsIDZkgjMVdNRV\\\\\\\/ebMJWTJyUv\\\\\\\/X9yThC5iat1DRuISeBayDSfam52Zdf7FAbmGVlmLggfWCqYEpygKPsexjdHLmbp\\\\\\\/5AlPobYn6U35InJcv7ZJHgasEzHo2X6xDvm37kZshhFblkigkyH363GGSneQF44nTp9VcCjYdCwgdji4FNQEd1Dc18JVgINQQwzdPBLv0hvrUFBxO+wZyUKM7jEBRDCxglqpx\\\\\\\/5FzD675USPAPs4hn61y1LRS\\\\\\\/2YJdgzAZ\\\\\\\/GD4BsXMiafXScfy0y+\\\\\\\/YS0TJb0JRhUMz42c4hFtZJ+n5V8lV1yxcE6Qxn9jEfACNd8dDzk2DO3mWhZCwCtutgbkAroiZinXtUIhJt7ZVtRaQvLgIjAHxNPtX4csbkpQtP31SmLaetE8Tf6pn8B\\\\\\\/IM5NUKuzVdtH0pmFkDMqKPhoCYmy+8tKvCIiG5SUJSyrXG\\\\\\\/zBPcSwwY7Z\\\\\\\/fZJBcw+Y31XG2G9k8zONNEEVE14MkCVVVP8DHVuPYoRLL9X8jJdnGYHinoeaqwVh+VqSieL6moCsrYl9\\\\\\\/TaVoLEnMjFp4BgNEBwto0gO6GI0uOqHAdyj2L1s8g7UN9mugkJrymDL+yVKjhQkKthRsTqdnosauTzfFUTPzkcYgTrkx2l8qJs+IEoV\\\\\\\/Q1WXMMHaonlY+RabI0WDXMGRm68aVUTQm05ZeHJy\\\\\\\/tGdvieNHhR7ebgXEUYM4tqXsp4tqrmfG5cFMZ6Ja57VxTgQoZzxV0pt80bF\\\\\\\/UPS98m7M9HOznCw3OYmKr8d9jL9ATjhhQHjtogCXjeOKulxPWqYvGqB4FD40WCneyqfj0FJNKN2ASey9XzqdswQp9TqesB4UtM1cFmYkBEBbRL2vM+rXsJUJHxBWEe65oArPtD0oUqOk4oOumXDr1CfRLN9uXUtwRP9j+gaNmnF0oW5J7JbrlvqzX9y+\\\\\\\/erNhPOz4jIRctqjEvYHA7vEE6RyFHNxaG+2uDusKUmvxaaWd4tzFnFsaaqSB3ILipuYM6OhVwvze63q13PmJGk11uo4UTvPFBmC55SMC71H9iXDDonDahNif8yisC75MatqAhySgyLyU0pne6GgSlMBrf79V5omeFx6HHNPfJ4bvitTfLLl3zFdPtquv75AmnwUqsOvgKTH7Fpss1vY2HWVP\\\\\\\/v2ZeS8C2Bb3LyiRwR28seDbetPU+cqcb4a5\\\\\\\/m7i4ir7XXEMj40kAR5faGGlXtqgUttHCh3yY1XWs3TWh3KbgdsFhzMSUp0I6wRuwAjFt3r+AWEOSgB3KVJ9Vaf6zIxT1toG3N\\\\\\\/8MX2Uh7JVSP\\\\\\\/gUyjTlm97irclJWOMi\\\\\\\/+etmh6dhSjurAT6tQELvaXPFnb\\\\\\\/fGNJOxQKqZVByvCffvyTdO2S+AdII\\\\\\\/NDfFkhGuVPu2VXqYlk9MpXCyl9GT\\\\\\\/jQdXWZNukHa2iRJcd7fwvTYPhEaBsfLk+fAPOQpW6G8aOb4pSn8mztdW0OYrNlbq5p0W0szV05GuTM1CWnERmdb3rWSBxUOZ3BFrjlelYkuyuiBG4ZN7efDdeDeFs07S3yziMTk\\\\\\\/2uws4tNosFhG8nb8SZOqVgS8PSG8LZFfyMDuglsXIZW1YJz6SREbCS36Mw+y1ilCfnK1Fsr9eBMBTG5DHr0fmV\\\\\\\/oaNsAXGyG4X1nAIfzvkira8z\\\\\\\/29U+0SFA6v5rUVtAeb40hAKqPpUYh1pWREn7OCBJ\\\\\\\/uvUWDrIs9SrTu1O5Bn9GGft2X1mmoDoJyCwUFBdOIZkR8Kt4OGrprjuN3CBBXo7V\\\\\\\/lrEmrjYZsW\\\\\\\/9lMiw68HxityK1zgep+Eu6hRnPEykKIPDz8XVN2OZS8hD+RG0hC6ajcwyxLfJmV9QVnIoVqlqT9ueGDK7QBV4v\\\\\\\/GI1ToIByjJ6ZZF2Xb3M4G7dEecpv5KGSCq7ruALRpkV9GNPEGcMBqKORTkT4aHsL1GBuNO6\\\\\\\/nJv7Cfl0EuzUI+J9wUGUav4rBdH+4NqTe\\\\\\\/A3pWFtYDOMQz6nAFR79Ovt825cUkBurEhmUHTYaAy8o2SmR3TAw78p3MLoAw681MgmPVDIWoHh1Oubzbiyy46peqgH\\\\\\\/IpYPIPw06y8SZsuRNwVrmITi0\\\\\\\/y+xdVDJglLaA06yTQg1\\\\\\\/RL75ZXLIkJ9YmH8YgsbXjcse1gqGR9mylymgp8IJ\\\\\\\/S7NvBgaFwQV24EaRgHVc6JSiQOT0CXMpOQJBvLjJGl34d05MSngfn0ccnmtfPNiyr7IcYAyJR\\\\\\\/wAqWkEIOfDANXUx0gG2pfkKeOYdLVztF446IJlmLdGyz82HOEsLGXGQ57ff34B85NEipD8QN\\\\\\\/NX+EQP1+RFu4+papZxDScyQdve3tmJw3oiiQBizzBRjx6slZcmExzHJ3hfZSwJL+e3r2KtF4H3Vs+fgytFYaSnzdYGHuwrqAgNb7PlxGdnmeh+3KYETCokqTxcfSt9katEoPn6ntgwNdzkvQOfwaBrUMBz6CUmZMvigZcDervbNLmdUL4fAkV2IzvUdhSGuVXvt9kXTEd9kEddfvkAXe84UdHq+L+AMbKGLhd1TMbr9rZColKHaXjqcKofsgrzfYYYwF3rU8ZLwaDqwYDR3B82B4gNVYIf6mzvLdPAGl8ZkhpJ5DC4tcXG4E8tZsNPWegkiKhexgOXdGwODWa2U36fCnCQUx\\\\\\\/GIziaPmsCXVXnCtBWM4fV\\\\\\\/vSiLEbHn1baifyN\\\\\\\/S696kF6jnTDVeiV\\\\\\\/tBnF0DnVtTaXnklA4dKsV0DdwTLsTXZDpGXY+avd7Rkd2Gc+4Fkc14+5u7AYIqdGMlCagg20cJKmMAr1\\\\\\\/JK3nQaEyOgblw1XzJXsg8scMXDgq+RQb4SpUzs\\\\\\\/o7pZSjCn\\\\\\\/Em9TzgZ0wIpR+Q6ZfdETBufH6Z7NCksxfpRn5HO3RuOKowB8FUSgDWcyIncxBvIjn8moDuvu3eZWJMftk68m+QYDBKItB93CCu1gt3UfcyLSbokrSDr7nHeALaMFd6va7wRHiFrOLNCvLyLzm9X5YZYu0FVWqSdo15PqgoGVNVZWWyV1SfhHgL4CG64JxW1DGt8T4\\\\\\\/qAXEih8KAWJHR3hNuc1BR74ZsvosZa+3owGtrJLLmAPZLQyRQt6Kjs058ViNavIjP4rEu7Y4h5Svf\\\\\\\/v3QfXxRCnH0OHNLXEorL2nC42gats+E6Qc8CoxJ1dG8cW\\\\\\\/Ha\\\\\\\/DvL\\\\\\\/gXw188\\\\\\\/KgHVedgQNaCI1\\\\\\\/pfbYjqdw3fDJrE3C+FgbOr8lkkC9OtO68hF7nwJf1oECRXW2MMjKd\\\\\\\/j4ZltFX1zEszIEE8jrxR74R1lQdC7eJ22GdEyvImAjvTVf5\\\\\\\/yfhhp727lgvjUoBQ1Lrf0DTQwLY0Phc601QiLRFsebqYbdEpx7BbL9fo9rtFn363waP5NjLEH3mnH8uhzysAGzw\\\\\\\/Nge4AUIgN4qwaVc\\\\\\\/dk7TCR01AGWnLLgBsNc7OMjQKToD0BHe0pABsUHdbe4YbC+EfIyj668ZsPuaW4vkGaiF6NvnMRRc2\\\\\\\/CWJgCLfUrOEyPLXzJhn\\\\\\\/FoBXGSH8BW2XHNuJ+9oqkEsbJRUEaImIecVnftM00PJ7HOzj6za8OuDPapvRY8QBw6M01EFRZmWZd3+7bfTlNd5g2pR3HLe6NEjk96RXTBW79o3MqjEiFWYDu1yhNbJba\\\\\\\/TPWATgjBzHMjv43eEXN4pcW7X4mvpDRqqLPojk6VT943J+J3WKvVNRpiVgIehIIYgHgpysZ2vtJDyxL9ynzvomT08WE7QnbEPcrT1bAdiq6QLPCswDlY3hNIn3FL2LBKuR7jmPPfCNDPJRXZslt6NogUu31QdqCWtKrDAn4fRXcr4VPDRiD4+TrKdGExzj7RmfqoFVyFVRJZNI3sLx8Wl5e0lSddjemhqpia6KYGWPYM4m\\\\\\\/Hu0nNgKlE8MohcC9MuXiKeP9VOhHPLY4jxoCHfpdH2VSOq6h4K1SERh58TmyNKGxB2AgrO6VT+dXRC5ZHEekvmW5uYrmU85\\\\\\\/jTU6kIQmuLzx66m1p4H2DADxs63vryKYlCbLdgFEJQinzUx3UDZ+CfxBmRf4fHHYNF89A+DAruw6SLQpQBe53Kj9C\\\\\\\/rcUni6m0bvzHwSpbjN9roYXXUfJiItl\\\\\\\/yhu3oCjDP02IF1RAR2VuOw3vRpn24uMS7vK+47KluCdp3V7aq6uxtfK2kKxcvvrC15X2PTTRtryoxZ4bUTjTrglGBM41Iz1wUE9DawPvXA1OyjKonveEMVR\\\\\\\/Xp0b0aQaz1kHjBUbFuWUznfK3nXIWTcvUpuC7+0yWvNGZ6gyth6nVw9NvECAL2lQZnGlh+IGHVVmHLuPkpvvF8r9UttuLjR+pGNkre8zpHhzXryZ0CLnQN\\\\\\\/ncFQX6A6BiS2by\\\\\\\/Z\\\\\\\/Wa7IYvgHKO8hLFbCPWeyKmrEXkcOveedlmmfRff3\\\\\\\/GBJcdvXBfl63\\\\\\\/QrBv\\\\\\\/VigqkRYpmVzxkUlCrX1zQt9zKGUgB9uPT+zGWdqVL093m\\\\\\\/cHj0xp9YruytXCn8N95EdC5Md\\\\\\\/e\\\\\\\/rty1GEsZkI9ju2rG61dnyE8px1YBzEpkNM+VkYhLCa49zvII6ae8ggRggvyaX2xuOKjm\\\\\\\/bNUDRxczJjsWsoQ7Bh41fp5+w961lJyO1P0m350B5GDi+VsLCfGXDSmG+A3jOvUeHTS1ywsfrbhXKQAWzsmACngwUhNzzz2i11erZovpRva1uc+x2rBQhofylqPv+HbUEpIe3vIINIZL2PjxQpkUOKOESk3y8AvSILpy3gfkUL6rG7UkdYUeNOvEFNSLxDg+tVRppH77j+RFUkKyXDEOF0gA+Yvs3aeLzf6pDgCy\\\\\\\/bYjPYgjRN399gb63q2+EKwM7wB2xMQlqdJWDAU8N295dcx5fbDZfSJv0FgTZLhr43QHYA1E7HZMs1xffM\\\\\\\/iN5uArm\\\\\\\/nS06eVlSYtzjjtdsd7wY8MC+VBW7WoMs210RFVTGYBoq4cduHTmN2jylUteiXYWct31ux0rtz+O4mPjbWEy8DYYeYjNpHSKca3uyz6GXapEx5i1zLJZm0rqYl4GG4fuy147PfNrOUGzZ5suTEz0gUYPIIA6vkjHwoXbTuG7I9kKRz5bNPXdjOkUtDFdxXA9zRZixzAt3fT203l5Dww11Soc6qVTOU4snCPj7RvZdHiPdD1b8QOKSZAor9hq9DoEeRB1DnQDlHqhdoxI3MD6mLrV2De57bYFrMY01bGF+aQvTMPk3SGvAB2vAoSM7iKtGNbjkKWQvLtJBW7WMO\\\\\\\/N7IVi24DRI2sbUTQpKoRiff+J7kgo\\\\\\\/GUeRJFgSDUpiwkZp7nv9B9+1ZKQHkOqsuCgX8zzhVIa90Y1VkfSil499fS2iaFtHcrXLMMxAp6BsabxRHhG732HkabtR3sc9PS3CMTgCQNlKOC6KYObRCyfVj4DvONvjBpJr9QmcUS8d3xTojvGO1zUsv4fcV4Z\\\\\\\/2koQR62+q7RepcJ9oEt+Jk2agIuqCPi17l8yX44WqOIauoWMGIist4V\\\\\\\/PdA6FZCUqwGOX0AQEaaZ0KTTVVOu2lmWz6qEWkuaYP2lHDUbVQaUhth4V9ag\\\\\\\/Ci2a13dGPKUKzhJWFlRfJq\\\\\\\/nRVvGr7DojjW4w1cxG\\\\\\\/1mdrrMHMmeDQJjslznX92KunFpvmjayaWMOQ\\\\\\\/5fnZx+Q8qmqG1XyE5FxgXEBggHV1fizyDemsmoL60IslzXyqjyc2d5JVIAz6l1q0DBbtok26RVXZGbs+9+hHq3bruTpzmV+3Afq44DVUXp7GnzQ0nUUoO0t0lFgUt5YTaU8T\\\\\\\/1scfLg1TIlKmLHNN++SGkVYhpJM35wrsnKtsmvtCN0H1nQzloNiMmzpeXf9oQllk0wmAk8StE6ax63eeWn8gRcmIGViOJyswHOGI9nAkZUodd3GovX3jJDMSHVk1SBGFhn6sX9ICvmK\\\\\\\/Oz\\\\\\\/R4dVJgOXqD95O\\\\\\\/4l2Ypo9\\\\\\\/iLQxzFQ4wvXR9\\\\\\\/pK3pOAhWkj5a5n2GAomZNNZCwdzykoUK1EN\\\\\\\/5oh2bpHOOMmG7xJ5jxQOt2WPEEqeRG6D0wZPuhDXqN\\\\\\\/7XlT1k+aSamYOpFzgaBxgJXLmrVHOz15w2VWevSfloiIv4jc93tyTg138DMm64bi7XjYEvZW5QMf+axqh\\\\\\\/28PP0LvTUHKBQ1NmGNBPy13Ztl0YilDnzdQiZ+7+kNv+bzqLc0WcqZtYI0f2oP0AW8nax+stxPBQdMJ41Eo9tpMnPNLlH7MCzoZft0ffRZdcJxqclFJHnNt0RUhtlrs1TOuqAxUaO9D31VkzYBW2tkqs31r7vn71o4tLlnDNgHTfSHyQL9GF2Yd\\\\\\\/hSpFAnPu2X4ukVsix\\\\\\\/g+Tsash1SqyziLD\\\\\\\/4a0P0Rlc+qorxP740rWYfW1NRR+ZcTGIvLb7O1vw5dQKHl4ObJ2FOViUZ8w8BSKMUzEHPvg\\\\\\\/uIoK3T5WU7A3LSquvfDhKBvxDB9iX5GFXvfrks2YVLyZTVqPyhhm67K4dUgtxWa3D9MVayVvdcRlPN5ktcM\\\\\\\/ZaVcHcxTBtiRSi\\\\\\\/8E29F7+ufijdRCbFk7003gOw58FhqkKlBR3kfrdBWq6X+qRchdiuZULbsWQyslzxk\\\\\\\/JgAEpdbjzkGisQHJBU9KNMiEcVjLQAXyjGR\\\\\\\/59uVYtxhE1NLNsFbOkN1+UOviQeXprGdahJhYAynOLuosfu6f3pqRfaJO0O8O2Nz0+DjoYmpo4KW0cnOveSy7J+nQ\\\\\\\/rxxTAHxq33of1thuDggoceryIMuN487JGoKB\\\\\\\/pYgow6Wr4FIXf960bKWAT9zOaYW1XSfQrclwfPF+rD8yzO8mz+d\\\\\\\/RFkCkIV\\\\\\\/lXJHec2EPtlzKau9S95ue9kLoOcHGJl23FQzXa1fZl+mMdzdeWg2Mgh5zB+j6uYR3vwCnCaxFVopR4jLW5JNvJZwAxi7N0ty8Sw4NrZjdG8oy6lO8hZqsY4Y5HeuqXDOBO6AqroHE2rp10WHUB9rGyYkmkqHYY2+yEeOFFeF0W7My0igMxIDBadMHZ9cibpKWXPEoDf9kb4NknDYkuEmi9qJWB0aZR5VP5JWmd0MYiLP5sI\\\\\\\/onNwnsEJswkGPfbevxhb\\\\\\\/h3BV32zqcgNfKcoXRK+Hv6V5oCAx7khOs4b5UtGprRy0RRthDVQ7ZmhG+ZKcO1OOnwd3fsjDcbrK3PfRZphCjKvGB\\\\\\\/441cjGcXJ\\\\\\\/QofMgHgozP\\\\\\\/QjcQ9HLjD07oYkyguAh69+kPr4xIAzIaIJ1b0N5M0KRFqNGEtRVLPaWMN\\\\\\\/58SbDrCdQofXL+ZU8NCh5JMG9064aVWcAtoeCc87TF4PxcslVmofeB2rFtOU54+VuOe7WtHG0+ZjoOU3JMHE2zZJSgE5nvuOq7qnJEo0MkUgc3VfW2AkNPRiP4T2rcVuBAcM9y0+mOSMVZtzO8uC9jxzSIhDitC\\\\\\\/RP2uigspG7Z3JHn0pe031e8royuwzGzz0b8To3wi6PPsiM9mZljwlMQmnx0UPrG2Yy2HUEcUD7R5HxuYvPx6Inie8+ZOXLa8Eo7pvLNxYSf9qMutfzmkIbRCXEWXsAFdBiGIB4gpN+baKukc3h\\\\\\\/FgDp6KvEmA5D8IUgZ7CaJO1RjKPBX01ndRRvRHRRunun7Im5qaUn94erTCu91B\\\\\\\/3DDXy9rSD98p5gr0aPblkkz7XbIslat5XgCn0oCS\\\\\\\/Y3\\\\\\\/ll2OqgGJW4z+8Xmc36gvEPKqy3vySR6xCZQBzF2yza2onolljvqfpeuoLvyEoztAoiFv7pFsPfm8VWdVmUWVJXTXX2upi04vWO1ssD4opPIOm+buARd13zQDG3YzI2Vko6KTA7GTYXBO6RTKfRp3Hql7s70K6XpbAl4mXN9Gr4Sy1Ar28QgyXJzYueMnLpZE0UIi8TUR8Ymn6FDVvHkHQ0c7Rc2lf+vGuNUC0Sr60QaRXtsmHXJEskmKWC4BWkETC+V6vCuWrxNG3CUAO0bk3M7PEYl5YapevX\\\\\\\/a07HKSEC1JWt4rj97KQhJPbdZ42gHvpK+6+rhEiEwR+5uhkz4G6BcmEzj3fP+09puHVR+hhMfEk5U8GKlSvDftH7XNVfogLqTQ1cfwy2tEGsJGa944xtBoFcZhG7glOcLsjUbeKklB28CxL2TrDQYrPqHPq7+FithEXa1DmSNjm9PpSKUb93pIwWA\\\\\\\/Obm9ILjKVC9sfIYKXs11ZDm06SCMb1PBs8WhVz2eO4YMnDTNEHsDMNElc+ixZBufd57thAe3rmfUGE9E358L31bNDOZAcd0Mak459uyiz\\\\\\\/B7co6ddvWGuuRyj23U8ydW5WOt1T\\\\\\\/C10Hjcf3UnC+VXnRZVSpQmiVbch7ejrgJhrvkc7B2jcGWUh8EF3WIZsSNNNsKtBkGjWTnrdD82Bib+9WfbIX+gcIkR0JPJl7JEk31GaPKlb7k8w1nMpGpu+SMEEf3W8CHqTT4KJ7mW4\\\\\\\/+B5+MHPJUTTEmqJszK28DCd5OJeenA5u92XXiBQ6XIJSI6BdNdRKzK67YOZAWDKz0rNYq1PzSm5ZX4WHZe36QBJShpuYdK81C7dN\\\\\\\/ra7OF1EpeQE5YFFZ\\\\\\\/9BNVPhU+p7VlnY0utpmzC5TONAXAGjzfs5Vn1mMh+ZSJPKwLOssRnVqqIkxM4b1hiTpxmnR69VyvECKebgn5HFFuvg47WD4T4X85RJsXuQjCHEqvRRL7tNVvj6MoI2qtrDWOC4YFEskjbyDzuRtTucR5gP8PbHwPS29xmNEU5+SwSL\\\\\\\/5zAlSvb55Q8g9+silG9C0d16KCQmB5hSbrLgvkVFby+2MG1dzyJ+D+arLAX1cs3i16GftDwMpMmr2XOyP4XmABc4LHoOe4v1eMtvlNrI0HCB3TraIqE3vV4Zs4N5oQSDIRpBEp2cPqE\\\\\\\/5EFsvT5toNw\\\\\\\/Og9RxOePBdCjDsBHqZfn9hBmh4u9h8ogsc\\\\\\\/QaUAjqAkK8JkOeGUGt9w7Zy\\\\\\\/a855efKCQG1KKjF8\\\\\\\/ntPpvdiTgJyOl36nOFKy5iUFsAtisESOJhdgVFocqVGiCfWxZJQ9efvKqpC29YCfqPBfWQtr4P2XlfsloUK07KLucqbsDkMyupC6fmc9Pj67ShHWyXO+\\\\\\\/7PQNx4YiWjKEOuCUvhr0e\\\\\\\/A0vkYF\\\\\\\/OeVC2zwbiGIoRuDSPXxpHQ5ftGoSn8VNDy3DYzsF7j7Nx1idPb8e3U2XtsBFQ4Mm8tnkkG3fVtEyFTrkV73I1KXdegyX6thW7DnNpht\\\\\\\/hiFibvZ1JYoQghVQRhpr005f6xUW3usJcMMzHedfSWBmolHsBaz6UbV3Cu+EpYUqj\\\\\\\/aMYEfrCJswQ8gGGrIGujArrz78hIS9ow4+C+eFN8EEg6Kr0+zgoOstBL5Vjm4ofi6+yFmrmdmfOyzAhgylE3i07hYfb1jM9menSWS+pfR4djNi\\\\\\\/0eofIMobfot6Flx6Qjnv+vKd+3wid5Q1igCiWv23DvKNO0kIk69ZSrNh0mAr1VHcBrr6YaU79Sysy5qC93GNd0FPqtXSFLNl6qXK4beNUW9ZBUe6gwCB0HNHfKrpU0p\\\\\\\/XiKEGdCkCExo\\\\\\\/\\\\\\\/V0WHFSUXuRnQbfNR3UirZBs7w\\\\\\\/\\\\\\\/yU5zCOTQjAwH5MRnf3F46BmklXR8m3d+qXrO9FoDNZzy7tv0S9hw9\\\\\\\/8tlwC0cz4lG+SOQBYDv2xNtYBKNctK3zqDwgi8RqnfUIUlYNQfyimuSv2GJbVzcQEnqFVMvVrjzKQYgzbHdTTeIY8atAOrjQCdgQ11JaBn\\\\\\\/KNXUKtM3CXy6KHxJDw+LGEN1QvVaN9zMsBS\\\\\\\/eLsPjL3BmamDujTo77HcUOhnEcSBDEPsu5JDYrMIbNXfiHLsKtC4c\\\\\\\/wgiw2viEikOkubiEtLjW5x2Os00O8S8AQmapwPU7A0USIns9RxpJnTYaF1Lle6j0OeKZ0bMa4E41oPDiiW9XrORJc9E2y+QhGh6MSos+FZdQ27gfrL0r8wrlKEP+IOnmbUNcuYJNcRkvsPtaCLbPcYbGPbQHq3xPx6DRvJ5ksgLCG6mRF6bbBBmwS7SppnZI5sW3b8Gtz4w+e3YPuZJc2r\\\\\\\/KxiEeLX6OjkAe562Da1YpsCDIqklnf8EKH2MrIzfa5+Sd2MbzCkBm1hYiyKESPV2fGHd6cp\\\\\\\/nVjMaATfmX1A2\\\\\\\/ihAgC\\\\\\\/7p+U1qrv+KETEY6d0zfgjVesHtpA2d9dGnaA77RH4DRulutN+u1xGEGv62Y+vWr9BjqE9ClUnPSThy3I2DNjHC0sWwURnUC0yykeefFTm4m\\\\\\\/GfxBDiKLjwdHjkwGfZ79443D\\\\\\\/4bws5EptXyFxR\\\\\\\/7s90DYhO\\\\\\\/0y\\\\\\\/+YZtYeHBWrts7ArebZkZrLsKAv8ZMvjJkzQ5KO5SY4yd5yP4HvDyRcbbkenP0tSKN12a7Ha9hIUyLBPQg1nsJ3\\\\\\\/thWuE4+dfagLFmZFfp2ppKelB61jTh9t2DUJyTHpdI6KmUW9+gFeyj6xs4wjMNfl3JJe0yWpfUPsOYYZS9SvjgjH1laZNKV26UOOPUFxxgz3OD\\\\\\\/pJl607kzaCMt2iZBfLNQEb4Ofuh\\\\\\\/OEtxj3\\\\\\\/mZ6vTGqOTvxSHF0jt0+YGhI2lrTgO3fnKHqx5zImMJI+wwzJvr0TU3EeWrBgZFAJf9MMM+JSzckxu7NFG\\\\\\\/LUKNlHfggnYIudsIhmC9BeBZSpek9VuJTHkIuaAVy7jUnLxY6\\\\\\\/fL8cIjK64cG6gVIl+Z2UNw\\\\\\\/FDVFolWBtB1aWKnY56feZlTR1xOCST9BwbCEcIb0WMdrG5yBpJNclROCxFMwlcgyBU41\\\\\\\/Yf\\\\\\\/+teivd0qqvWITbffaGcd3TG+0p6p5YL1CaFUjGV31kwm1OfFDUz+tkF1RwtHXH3uvQe8EuoTpQ4gsuMEkjKsXvIlTLoPu3E7Meauv6PAgRpTdsBQ3LVd9\\\\\\\/w3J8L75p\\\\\\\/WiqEcp1WQxkiDNx0bIZ6SRLfQkA1\\\\\\\/1OK3cP66x6e8aGKK9PclCwE+oUUq1FG+\\\\\\\/F3yTNWfSLc0kzKIoJieF4D2n+XcHBRgKG1TODUNb3Smppbh3HdII1oOHLlnLCRRGPnWbT0luJJz0C97OaRnaut\\\\\\\/HQsE9QUjF\\\\\\\/LCx752Q5kC9GtTdKpoG1Nt3CKXCzig3Nno2pyJfkdV0reYYokBci16CNLtMd9hrIYzw83ZFWfqf93GK\\\\\\\/7XjwAyzsw2qqu7xCDqtRnn4ZRqBtQKcWZI3j+zRvR8FQrTX+ghBr5B2t9F\\\\\\\/Q\\\\\\\/1\\\\\\\/ACKiLAzKHhwtEUN1Idp++em9yaRqlHA6F\\\\\\\/vrwY+vi08gvenQFHMzDMmdS8nBwQ2R5WBoYSovnXvKygXkSUruRpkVekT3fJPvt8XGiC01hrloWj8fBO2eyoNcigSjommW\\\\\\\/6x3byPRFaPiBT45gdLxOfR3ibY4CRrTwZx3NvsMy8TfK7x1DbTDRN8Qsa5yX2+VOfFGb4BP4ubYERHz0s+NNES1UTKN9E+K\\\\\\\/IvRAOf20MEgJ6uP2JV33Jq4LXnBemGxeQjK163+DD8YZMpCuMTPeBrm2\\\\\\\/IYFmuLa8iCWqyrqkiDe15Gpb4P8Sk+o4SWT+SkCJ0ly3F9nibiSLKqKbOQnVs+FTRNh6Y3lF7boj0sfKlwOezU2ZCKkDnjd4p3sVZPkwY91jCSIrIX20ZtSKp\\\\\\\/hkCe3Y0AMaLyZOL7I0q0vyKxjOMm7sNRDtx4eH0BowaL1MQSeqedYNzZVoOrtYMKrd6RmnMZhrARk7sZH4G5BeeC+9s2y8lPuv4nLm\\\\\\\/qHSYtX7yEaz4CFSaaHVQWMxP\\\\\\\/jpoe8xao6\\\\\\\/4JECrhGBdeL8mpCpUjg98vwsMTpSWw3kbOd2XlQkIc1+mbb2I7c13FEW4S8\\\\\\\/9cWReI7pdcrSt92fsglqS8w5F5pS\\\\\\\/oD4Wjydln4SfvyUnVMrcmRZ74\\\\\\\/e4EeBsayg04ysof0S1sDR4DoxRA6UjVxd38F6QHh3yrEhkxNX4YmtmMS873nUSVwEv+UnoPCm3SH4FzNLeFpM7m1n1yIkOsDAfmXV5VfCQdTdcmkSbbtMymxcWdoJyqM2FLM3pqGvv2uXUzNpXBQwWsbRAwmhcYrqVG\\\\\\\/tGZuXXMe1e1pFOzhkOzLQL2W5QWT6gcft\\\\\\\/vI\\\\\\\/swZ8QXudyqk4I2OV\\\\\\\/4+WntZkS5KXv90i+7A7Jk75Nza2VYExW2zYjbLUlCfHEx25SSrwJ7K7JH0UuMiAi7Z3qi5eMqUJ9THRR4c0pVMV3q8REeVwwi0uZXtQH\\\\\\\/z++v1b7HNSOtoI9Hd5bRzCHNE4O8j0Mj4mSvhQdDwgSOw7xsE9MlX5O8FYSO\\\\\\\/R5aIbQyMJId7i1tnn\\\\\\\/aEJYU+N\\\\\\\/RpTHAScheN60RbKkdDEPKm07cYCNUQ+xFtrx4LU97L1ZuSod9EmlI4RgwvKDJeF9vinCTXNriIt0Iy7B5b132JMIQNcIHEcZ2cWtgXioTyNB473LoODMZlXl4RnUFE9zOEZ1Sdyo9wT+lPc2wJQyzRN7t9MhG75G2i0JF88QjOesPIwHFZQt84vzgylvI1VXpz4dAZkrHlr3mnv5hTCpXW5Qlf3uqMSD6UXhVgiRTStCcm4Uh\\\\\\\/zIU4PGwX6aXCy5DCYSPSe4fHK6xHnrWf+mdilXLEtL5nszFhqcKwM0OnuZTYuF4hmZGgARU0eVi8MQdaXZ8zA++En2kjxfIBYDcaXiZ\\\\\\\/M\\\\\\\/CH1khI9LDVoQlsdH5kHSOPT0m8XPn2TIAIK\\\\\\\/+J+mTSMtO6ooI0K8rqV4+Dd61qmO6vrPBEhC0zg3Tx6aRuc+u6nGfygaI0nwzYSvlpzQa6W3BVtUoo9lB2RovVK+6fRHcGNes5klODxZdiijkRGet+c5\\\\\\\/m0uNI9dZGfNVywNQ4O\\\\\\\/cE0UKF88V2qi7YusZCqXPxQnjyXph4NHCMMRLx7NVcMNl385bxcGa79KG6REb4qQL4PGs\\\\\\\/8s36pCQdNXbDsfgzdUjo9Iqu8NH4mZt4HM6XmPwCeTfxpDxDOHBkeEOFaf55tPHEAa225uPS5ISlKT0M5cw5zZcqeB0a2zG8gJGfNuxF7nfNKKnYjfdhvi08CINNifgzaX72S+OsBYBtGQhY4FjtBfrJw42zH8YFGFO6Nl8KcncpGIRXn3h+AsBwgVqBIt\\\\\\\/LMax2yKDAAk6wB946WGdyPxaKIsj02O+cr3FadF3\\\\\\\/yfhBAUPnN4cI8CiihcVOE+ysDDPAsdkGpoAMVLE1kr6EaQuIBffB+GSs7NhKABzlnk0i0ax9AA\\\\\\\/xrfZpx+h9dAwRYNL1CqhLDotnLImEv9biy6i9VbhwdHW+ESdKZP3nhAW3c8XNaARgvz1qdcBldW9nNnZfV9aE578JZC\\\\\\\/rcxdbS\\\\\\\/jSIhp7LhOpvj+beB+oJn3gTn6eBAw07DvmCMcLnzHWXfbmliGG046BzHs2vBG\\\\\\\/YE2TgRB5DIO3McwjGITtSFsEdZXDdwIRYx2JjBfzd\\\\\\\/p3QAQaKXxnyX66nRtfvmgrdV\\\\\\\/qbYltD5xPgVeYkMwHrVSP9VwVMEGNmcmuXsaD2P5IRi7i6eoCkcjPuq1f1moViisbhWWAjyfvcrKAYbk9KzM+BfXOd1\\\\\\\/UZfDlM2yuqe92jdElm8dZWFztKyfNt5HS9llZ2m\\\\\\\/68Mgx2\\\\\\\/sksUBlUWWmdLld0qVDd8qp3nhXqoN5PGipGhAGKHJWiiBBtO2HLcOoN\\\\\\\/Evk1zaKQUGPrJl8bMwRMekZnu0ZaLPxjcCfqJ\\\\\\\/PgcxzePoBcz5HqS4WLqICdETVL0kVzZ4PAPdt95c72GIa0eqMxKW47wI82A2OdsChQS5ZsWUMV6FjTZVeEbf2kMxGw4KRVIGNEqOVLXBxXOpdz5Kmo04Cg+jegHggUHfXjCwtnBLwQ3lq+DyAJs45zsBkQX6tUVgLMye16mcULl5Zx\\\\\\\/JN6m1j9jsyGf818pDl5dqs3feJCjf6Qc2gURTaMSVx7bqjE+N3qa1meL1gizEMxEqMsNB+RAPXwpv8qj1kMJJLGiyFoB634cH1ncKPPHeKUfOcewz5WLFgVvuSLG869JvpwrIBFTrRn3YkUAHGHu1rWJl2\\\\\\\/CLfFnCfMq53CviBHjMWrPzqGM5VzgBP5vrfucCfUR6VwbzLulJSHTW6Tx7pFJ90IfkAGmvWbbI2un8FKWO7Ax9WiZvRCKXe0+oxPCUZO4z9JKpQYQ8oYkao2md6i07F0F0WiJszOJ\\\\\\\/DxvYkBYk4d9hziCiaeghyKB3jaWYZyfSAzybRZMCPFUvju20BpDY2rMgbw2GvZyagE\\\\\\\/1n+R\\\\\\\/t+NkeOc5IKRK4OHNMim2mX3WDJbC8zI8rQQkrHDIKvo4b+0bMb+VvkYuD6BFJinToVztNXoli8p8BjhXzDEn0ypcaLojHTW8xGmLn49sqN5baRy8Aixaa3N109qQ4dyZLnqBnHGPDEyS+KUS2mq+lvkZIyePabwDj1zkjueLIPp9V5CCrHS7rBEkEckKbpZY2btOxQ330ZM3rRVrS5Vw1qtiG7oBXBDtk38Nt+mC5WOjtpK3xGjIxb2EH+j9pJv+mvAglJRRnuKipGet5ZelyPgVCJibMo9ex8TSWYvaPptXkI+e+GnXWPw3OkLMgg+qksNumwjJKT\\\\\\\/+gvKAYoGLNfVeMaJpJThUO7HE4ZwbCybECVg8ogrATxp\\\\\\\/PrfqD9BgURTYdQH61cYGsujY9\\\\\\\/i49uVrluYOB5fM3LRb9BhXd96P1XWsvAV56X9p5j4Jj9iJt0fGBC\\\\\\\/8tbn3mJ73C+NM6zZQhB\\\\\\\/87fpGI1uWA7klt65p\\\\\\\/mI1YJ80XK6vkaiO2jhO\\\\\\\/c4A3dgIZQBSp\\\\\\\/Jp2c3JR44FmP2yj95rYtIJnu02HqfgG6BCSHv+hngwoWZB1fKhB+GV7h8KrVuDtMCHC4NuiXf5IQBo7XmdwTJGXXQ6K4Lli63Onzlzrmp1NaSlNfKNsEdYhiZPRWSo3SNKpVKFMb7fo8JTB4yV9Pm4IAG62QPob7tiRhBUfo3YKbl+ROr98UcSD7zZasogvhWIVmgFp21yLJgFWqZLFVuS9ovyZYrd2yLFXr+mQbBqwR4Elh\\\\\\\/1OpQKTtSdWLFmzDuuZKpTOFvqvQsi4+6u9ERVemdotmtN4rXtiFM4sIkHjRrR8k3k4q3\\\\\\\/v3td5Y5kYGt0SvaATPr07KjrBPjUDX6ZRJBapejWrPRPkD9z1I1K60ALrNvEUV5m24jxwzBmB8reXtVmAOro00ZQS9CUmuojs4WuCgOYJOTJqVDFEQoRp7rkNhJ79yCsa5jc86DHtm++sbMUFdeNpot9H8tl8sa9KZrVbUYK2MrmsfQg0ODSm0xsQ2T7rsBVVoy8Z25f\\\\\\\/xFRCSHwcwm0l9lqfzOct99boRPCCqjBNuc3XcOeZtknOnfU5LGSwokuoaaGDz0wae4n06CDvrPXIdBq+q2073GKXvMdkf8b8f6JuAPxUOMWRs6OKulEVvO3f4aWyePXtLoYiwkSZAuBoUMlOKoX+fUXdZR2Cl\\\\\\\/yneXKZxLT0b\\\\\\\/OTZs9zDTVwXT6g+naMxTORnCKliKbBVsLudMu6xQGbr+aSb\\\\\\\/Va3MyTP6rziRzSF4hA9mhku+W7Xdf+zcBOKDpOfmJOzo4CEkcV2WVFsctIcMBK2l7MEFIPC0BPJKsdqUaV0T\\\\\\\/C0alSFZ2E0svbpzM3hzkmeNVmJmAtZzVgZh2RxPI31MQ5xLn2wkIVGfvKy0uwMGrTATXutunX7clVhNSaU10HEG4VLGbC6dS5\\\\\\\/3AB\\\\\\\/e1TaHcyjI4KbLFwECvjypnLbCTwEDijgcOnxtFhZlRzTcVKqKKYOYzOOXHjEUOr3rAp0xZiSP8kLwJkBqMMLVT1m8VWgKu+w1ghH2ODsdp8V7azPuMaSQJSPSXkhnbaj4FwIED8qsHz75mngSy33TR4RSUXz74Gyv+OefoQZk59QPkyjCFkdaDUGHFUfQAyE\\\\\\\/d5psr8pa3\\\\\\\/rrlzuoFmcylVa1MJ3MuwuBjT\\\\\\\/C3YYFQljFypJ05Mnl4ctLLgCj6ilSLhtwp4nY1MS79Dn3I9dYmaJHPQfskGrVXX\\\\\\\/xzfM5eH\\\\\\\/bjraCWvhs23DvG0fYumYyVV8DAxG9sa1xCym9tom2Hb4uEmfJkIVKh1onrFd919ZRVLnuE4n0HpKpGX37b0o9hWq7rIgQsZ2hBajH\\\\\\\/TAR7BFIb6uihHQf2qoOcyfD7H6PQzlKsgZVCBDx72NXynDuy+pTwQCO7b7ZTnY52ksCAHRLNqTj1DcuvtBnqF+ynXHgECZkv4DFswI6jMbSRkJZNhCHu9yvIV2yV7CLfEyQ7Cg3\\\\\\\/zymqVCqdarCVBshiOGaubEcAOarRC6GnqRNwfZifAZXwyqJzbtD2FXbGdeA+C3vdQUi2Wlx9FPT1dYdC694jhxOBGaC5Hk5TDVgoLcHvvY8h079hDLV3QyX9E0fHrcEDQ+2RWMG4UJJTCHy5+Z5Gl1dALEngFBELmYCHSOs\\\\\\\/F4+pHjxXQUVasgngBSPENBruCU0uHh8LVuBfqkX+DCeWZGcH1LRa7w8hX5ty2GccE1mP1J9rS2MBbk52dtAEZXtLv+sjXxxVzBlIxTwKcE2CAexwTkysCaMbp\\\\\\\/inf8BapUYXJ9N7IPNqa+xfx8LGYakPRabwGLZuGFD+p\\\\\\\/bOsGnQtZG3rYJnXlaCUMNxMzGX3\\\\\\\/E53JXYfS95SktiL2U+yf1uQtFt\\\\\\\/9RTUjXegS32eyecJLApRQzFD2w6Qu7i\\\\\\\/KSlsJBhC7yLob3s\\\\\\\/o5UMFNnYH+MsmDpXMn+gQfQymYRB5hGPw3Yu9sHMG4MhdD6KBivQeLuegdIz\\\\\\\/rbg17hW4cbS6l292VWfR4qW+VvXc5guTIg\\\\\\\/mIxQNZ5YCyehOD2q0O5mPq9F2TgoR8lJZbHkyFMue2t4+4KeB0CqLmm\\\\\\\/BbneuwXrqDZ8SEGHsLi76e9Bp2r\\\\\\\/XPCzHLQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022NGZ6C\\\\\\\/0KSj90O6LEQlYnaw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022afbf57139b1fb01297e20370412b923c03cb41f76f815c77a7af0a9399f85086\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00221kGsZMVJMAPpe0BzCPd7XL4jrrUo+upFz8xYoMc3kK3sC6WE2YSCmLHQEkZpZC43\\\\\\\/UW16D5f\\\\\\\/KxP3oM02YoNCPkrLDGJpC\\\\\\\/zcABSVC6LKPRi3tUV1DUmDcgB6s3YkWsBtLUIymm4nULPsmQxxsJM5luOj9Y4Si1taeHbchgUd7Va4A6Y7Xda5po5kUOyRKcO3k\\\\\\\/EYCE3+MSFxyLTR5fAW1yykpKXMYRJGB5kQLWw77gLuR238POlMpYq1gLrDo\\\\\\\/BUqeW\\\\\\\/x7UNY2VrAXxq\\\\\\\/LMUorCIl2g5x5ykxrGgOjTmhfSzXEMxd+lYW+iJGt9Le3dRnKH5SAUO4zxiK0ytIkYeoORhPDPrOZ5L33yPxCcX5ahU+mAgtMBVT7HqfUK2rBniuMzKdEr+EeVHJsU5oxX+u6uFxWhqOyScT3pw+rpXrgKZpe6eMSsQmfUajyi++cjZr68LXfJwBd0uBpNfxRh5QMbTsdOu693GvHguqZxHZfFxVDvqPihMhBdP4kRFpL4vKF\\\\\\\/MqQTuq4UhMf8UC042MGNriPT8w+uTkKyMNFPqGq31nwd01lzA7ruqug4XcAGrcROi9kcXmaq17MvaX5zZiC6X0+alUiMddeLBeorT0t5zWeSaWpwfalUbWpNslfiEyyLyBmlUZeFjjFzmLd3tKCzGJl6rT\\\\\\\/WA\\\\\\\/YFEia+XugxGXhblQtvjU4++w1TV0RNaApdb8YMdEb2HnGD7+9z0L7H4\\\\\\\/Eqr9twL274i+i8tkUsZcgfH0fmeSLJQUOB+z0q\\\\\\\/FzclrKc6FMF02WnAiviUPxzFkf4TxV0+XB3bLQjE9zQstP1uopcUjgeErVbEL3MsUn6\\\\\\\/GhsOe27EdWxiaPmXutKrGjLstrcIgnxPvTj+O+aY90aV5Js9ECfCdUKnWUiYCINAJbo0VBxNJe+gd+w09W93i\\\\\\\/GKzCOMCZt6VZb\\\\\\\/+oiHuVsS0p6JPLTDBIdsuQtMTn7OjWpM8s2kdpUZpjiSMDpd4cDbHzFhNp+29AVrECSY8fQTRMzCppYYWYeMptE21OoBnLb+9QnT1rIij5AOl1ExTUMQKqBAINznJmknc9V81k3K69gP8AAfDnkksJfPwznBrrvJHULXACaS8FQqb4CmfCvsM5cx3g694Laj8HUrqDCPH+seAQeBNV1fiMVkUlHY2FLRnrRHiogvq28xV3gLge6kylQF2oHaRiCb0yPj3iiiRHk96Z+cPHzTkxjTFuaLi8+EVbvAXvNddakgmjWXxjzzuaUIqT4Q9vTNcqPo6gaSHNRu\\\\\\\/cVC9gNSxV9U3swVuWGjXxEwB27pvgXrI0YBCZINwrG9jKSK0OBCK5SXT7f8ttipA+3K8wjhcjXhtX9mEvkLdEgoeUMm0tt5bcDfZixDy+A63c1YHx7U\\\\\\\/w0kvOFr7f4kX5GQBfSrU\\\\\\\/NutBYsg4KqXaIV80Ct49sQyvF4YmgZaCUb3liPzb2Vce97YzbMIBCDusIWJeH5v55+MrKWvaGb5kxnUpj4GJEGXPn6ij8Fuj1lsClPM1EP0RjJ4IFXaxn5IFkjYkV7mmBYRElWokclWan34y4+u5OcGR5RPy\\\\\\\/maFp0oJazXJ7TUpa+631hsmp3IyXnoKw9qiNHaWp0+rvBybWHltOpnQVQSXYTTpEMFW8i+adCDLKTTc659YD98KE2njCu1XsZ8D7ItQxFMkUVrbk3SIpRxGOAw1b1Bk3rpoThhA4AFIYlNK\\\\\\\/PHMJBF+z6NUfmdpSl\\\\\\\/VbR4KHPyApOsiBtBfeLLAVrR6NolK1x9pELq8HXUknLWhmVP7E4B9iDpCOE6evLrk3dt750ddrYF4E1fvRFL8JueE7o2\\\\\\\/TauTUt0Bd5KLTy7P8mLfeLEdJPLT5dSP82GxiIfgzpNrzfTkg7y2rY9MnbE6t0H\\\\\\\/i5VeE6Z9w\\\\\\\/ampo9YIPzY5TWY57MEpamJfcfCgpSImDQl4MZNLdba0TzLcCD+fD\\\\\\\/7\\\\\\\/ZRzqztqA8rJcjm5KC6cKxUcWqg\\\\\\\/2eLXlyuKHSopGbz4QLIQnUZtjA1WCZg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Ypb2UfvaI7zRbIQCJczNoA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225ad3ad45647d79546af09535d4da7b260a4345bd8fc4a3b5d82e660556f8c4b9\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-490736 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/9/3049" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Y80MkRQiBK9Zjtr95R7pB1hFZXqykzIzs_YSUk-svBI" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/9/3049" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-BnQHTFEMWK5h-mAx30VvRv0OCBvekrHDwVLJ6z0IMxo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/9/3039" title="BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/9/3059" title="GWAS with Heterogeneous Data: Estimating the Fraction of Phenotypic Variation Mediated by Gene Expression Data" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490736" id="node490736" data-pisa="ggg;8/9/3049" data-pisa-master="ggg;g3.118.200580" data-apath="/ggg/8/9/3049.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Tandem Duplicate Genes in Maize Are Abundant and Date to Two Distinct Periods of Time</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1388-3194" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Thomas J.Y. Kono</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-4639-4119" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alex B. Brohammer</span>, <span class="highwire-citation-author" data-delta="2">Suzanne E. McGaugh</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-8833-3023" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Candice N. Hirsch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3049-3058;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200580 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490736" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Thomas J.Y. Kono </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BJ.Y.%2BKono%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kono%20TJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThomas%2BJ.Y.%2BKono%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1388-3194" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Thomas J.Y. Kono</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alex B. Brohammer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alex%2BB.%2BBrohammer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brohammer%20AB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlex%2BB.%2BBrohammer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4639-4119" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alex B. Brohammer</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Suzanne E. McGaugh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology, Evolution, and Behavior, University of Minnesota, Saint Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Suzanne%2BE.%2BMcGaugh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McGaugh%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASuzanne%2BE.%2BMcGaugh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Candice N. Hirsch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Candice%2BN.%2BHirsch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hirsch%20CN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACandice%2BN.%2BHirsch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8833-3023" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Candice N. Hirsch</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cnhirsch@umn.edu" class="" data-icon-position="" data-hide-link-title="0">cnhirsch@umn.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/9/3049" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490736" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:490736/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/9/3049.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490736" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:490736/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/9/3049.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:490736" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:490736/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/9/3049.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490736" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:490736/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/9/3049.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490736" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:490736/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Tandem duplicate genes are proximally duplicated and as such occur in similar genomic neighborhoods. Using the maize B73 and PH207 <em>de novo</em> genome assemblies, we identified thousands of tandem gene duplicates that account for ∼10% of the annotated genes. These tandem duplicates have a bimodal distribution of ages, which coincide with ancient allopolyploidization and more recent domestication. Tandem duplicates are smaller on average and have a higher probability of containing LTR elements than other genes, suggesting origins in nonhomologous recombination. Within relatively recent tandem duplicate genes, ∼26% appear to be undergoing degeneration or divergence in function from the ancestral copy. Our results show that tandem duplicates are abundant in maize, arose in bursts throughout maize evolutionary history under multiple potential mechanisms, and may provide a substrate for novel phenotypic variation.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/maize" class="hw-term hw-article-keyword hw-article-keyword-maize" rel="nofollow">maize</a></li><li class="kwd"><a href="/keyword/tandem-duplicate" class="hw-term hw-article-keyword hw-article-keyword-tandem-duplicate" rel="nofollow">tandem duplicate</a></li><li class="kwd"><a href="/keyword/genome-evolution" class="hw-term hw-article-keyword hw-article-keyword-genome-evolution" rel="nofollow">genome evolution</a></li><li class="kwd"><a href="/keyword/copy-number-variation" class="hw-term hw-article-keyword hw-article-keyword-copy-number-variation" rel="nofollow">copy number variation</a></li><li class="kwd"><a href="/keyword/transposable-element" class="hw-term hw-article-keyword hw-article-keyword-transposable-element" rel="nofollow">transposable element</a></li></ul><p id="p-4">Gene duplications provide a mechanism through which functional novelty may arise. Many protein-coding genes in eukaryotes are part of large families of genes with related function and are consistent with origins in gene duplication (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Rubin 2000</a>). The pattern of duplicate gene distribution across the genome can have important consequences for the evolutionary fate of duplicate genes. For example, genes that are duplicated in tandem (proximal in the genome) are in the same genomic neighborhoods and potentially have shared regulatory elements, and thus may diverge differently than dispersed duplicates.</p><p id="p-5">The initial impact of gene duplication on phenotypes likely occurs via gene dosage effects. In many cases, the sudden change of gene product concentration has deleterious effects on the physiology of the organism and will be selected against (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Innan and Kondrashov 2010</a>). In some cases, however, the increased gene expression may be beneficial, and there will be selection to maintain the duplication (<em>e.g.</em>, tandem duplications conferring soybean cyst nematode resistance at <em>Rhg1</em> (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cook <em>et al.</em> 2012</a>)). Tandem duplicates can segregate within a species due to either spontaneous birth of a duplicate within an individual in the species or loss of a progenitor duplicate copy in some individuals within the species. Either of these fates can lead to phenotypic variation within the species.</p><p id="p-6">Over evolutionary time, the fate of tandem duplicate genes is less straightforward than simply either retention or purging. Mutations in the regulatory regions or mutations in the coding sequence, may cause the duplicates to be expressed in different tissues or may engender non-redundant functional roles (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Flagel and Wendel 2009</a>). These random mutations that accumulate among duplicate copies of a gene may slowly erode their functions. One general outcome of this process is called subfunctionalization, where each copy is retained, but each may perform a subset of the functions of the ancestral copy (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Force <em>et al.</em>, 1999</a>). Several models such as the “duplication-degeneration-complementation” model or the “escape from adaptive conflict” model have been used to describe these scenarios as possible subfunctionalization outcomes for tandem duplicates (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Innan and Kondrashov 2010</a>).</p><p id="p-7">Many studies of tandem duplicates have focused on specific gene families, such as resistance gene clusters and ribosomal gene clusters (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hill <em>et al.</em> 1977</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Anderson and Roth 1981</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Leister 2004</a>). In addition, a number of classical tandem duplicates have been identified in mapping studies, owing to a large phenotypic impact. For example, the <em>R</em> locus in maize was determined to be tandem duplicated genes by crossing and observation of recombination frequency (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dooner and Kermicle 1971</a>). Other examples of classical tandem duplicates that have been discovered in maize and contribute to phenotypic variation include the <em>White Cap</em> locus (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Tan <em>et al.</em> 2017</a>), the <em>anthocyaninless1</em> locus (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Laughnan 1952</a>), and the <em>P</em> locus (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Athma and Peterson 1991</a>) which all influence grain color, the <em>MATE1</em> locus that contributes to aluminum tolerance (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Maron <em>et al.</em> 2013</a>), and the <em>Tunicate1</em> locus that results in the characteristic phenotype of pod corn in which the glum covers the kernel (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Han <em>et al.</em> 2012</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Wingen <em>et al.</em> 2012</a>).</p><p id="p-8">Identification of tandem duplicates through phenotypic analysis can bias the understanding of genome-wide rates, evolutionary impacts, and potential phenotypic impacts of tandem duplicates within the genome. In contrast, “bottom-up” approaches can identify duplicates in a way that does not condition on the duplicate visibly altering a phenotype. On a genome-wide basis, tandem duplicates may be identified from sequence similarity in long reads (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dong <em>et al.</em> 2016</a>), optical maps (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Mak <em>et al.</em> 2016</a>), or by orthologous searches of all genes within an assembled reference genome (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cannon <em>et al.</em> 2004</a>). Alternatively, <em>de novo</em> assemblies of multiple individuals within a same species would provide an ideal setting for high-resolution identification and analysis of variance for tandem duplicates within a species. There are few plant species with multiple <em>de novo</em> genome assemblies, and as such, genome-wide studies of tandem duplicate gene variation across multiple individuals within a species have been limited to date in plants.</p><p id="p-9">While a number of tandem duplicates have been deeply characterized for their phenotypic effect and there are descriptive studies on tandem duplicate identification in plant species, there still remain a number of important questions surrounding the role of tandem gene duplication in genome evolution. Here we use the genomic resources available in maize including multiple annotated whole genome <em>de novo</em> assemblies to determine the rate of tandem gene duplication on a genome-wide scale and the extent to which these tandem duplicates are shared among individuals within the species. A number of previous studies have shown pervasive copy number variation within maize (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Springer <em>et al.</em> 2009</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chia <em>et al.</em> 2012</a>), but the nature of the methods used in these studies resulted in ambiguity as to if these copy number variants resulting from tandem duplication or duplicate copies dispersed throughout the genome.</p><p id="p-10">Additionally, we seek to test the following hypothesis regarding tandem duplicates on a genome-wide scale. We hypothesize that tandem duplicates will arise largely during periods of genome instability, for example following the allopolyploid event in the history of the maize lineage (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Cannon <em>et al.</em> 2004</a>). We hypothesize that tandem duplicates will have unique genomic features relative to genes that do not have tandem duplicates, such as gene size and exon number. The nature of these unique features can provide some mechanistic insights into the origin of tandem duplicates. Finally, we hypothesize that tandem duplicates will show different substitution rates relative to other maize genes based on the relaxed purifying selection that can lead to subfunctionalization and neofunctionalization.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Tandem Duplicate Identification</h3><p id="p-11">Putative tandem duplicate clusters were identified by comparing the B73 version 4 (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Jiao <em>et al.</em> 2017</a>) and PH207 version 1 (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hirsch <em>et al.</em> 2016</a>) maize genome assemblies each to the <em>Sorghum bicolor</em> v3.1 genome assembly (DOE-JGI, <a href="http://phytozome.jgi.doe.gov/">http://phytozome.jgi.doe.gov/</a>) with SynMap v2 (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lyons <em>et al.</em> 2008</a>) as described in (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brohammer <em>et al.</em> 2018</a>). Up to 15 intervening genes were allowed between potential tandem duplicates. To remove any false positive assignments from SynMap and identify any false negative assignments, the longest transcripts from adjacent maize genes were translated to amino acids, aligned (10 interactions of refinement) with Clustal-omega (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Sievers <em>et al.</em> 2011</a>), and back-translated to nucleotides. Pairwise similarity was calculated with the “compute” program from the “analysis” package (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Thornton 2003</a>). The alignment similarity was down-weighted for the proportion of alignment gaps by calculating the similarity in aligned regions multiplied by the proportion of the total alignment that was not gapped. Pairwise similarity was down-weighted to account for the pairwise similarity metrics being undefined in gapped regions; two genes may have high similarity in a short conserved region, but un-alignable sequence for most of their other sequence. A distribution of adjusted pairwise similarities for adjacent genes in the B73 and PH207 assemblies is shown in Figure S1. Adjacent genes with at least 0.3 adjusted pairwise similarity and genes within SynMap tandem duplicate clusters with at least 0.3 adjusted pairwise similarity were retained for analysis.</p></div><div id="sec-3" class="subsection"><h3>Tandem Duplicate Gene Cassette Identification</h3><p id="p-12">We defined tandem duplicate cassette as a group of at least two interleaved tandem duplicate clusters. Tandem duplicate cassettes were identified by comparing annotated gene coordinates among individual tandem duplicate clusters. A schematic of the procedure for identifying cassette duplications is shown in Figure S2. Genes within each tandem duplicate cluster were sorted from lowest coordinate to highest coordinate. Tandem duplicate clusters on the same chromosome were sorted by the start coordinate of the first gene. Tandem duplicate clusters that overlapped each other were identified. Clusters that were fully nested within another cluster were not considered as cassettes. Tandem duplicate clusters that were interleaved within each other were retained as putative cassette duplications (Figure S2).</p></div><div id="sec-4" class="subsection"><h3>General Linear Model Analysis of Factors Explaining Tandem Duplicate Gene Density</h3><p id="p-13">A general linear model was fit to explain variation in tandem duplicate gene density using various genomic characteristics with the R version 3.4.1 computing environment (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Team 2017</a>). The model regressed tandem duplicate gene density against all annotated gene density, RNA transposable element density, DNA transposable element density, and subgenome assignment:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3049/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3049/embed/mml-math-1.gif"/></noscript></span></span>where <em>g</em> is gene density, <em>r</em> is RNA TE density, <em>d</em> is DNA TE density, and <em>s</em> is subgenome assignment. Assessment of model fit was done by examining the variance explained and the deviance for each explanatory variable. Significance of a variable in the model was tested with analysis of variance (ANOVA) of nested models.</p><p id="p-14">All density calculations were performed in 1Mb windows across the genome. Density was defined as the proportion of bases in each window that are within a feature of a given type. Gene and transposable element annotations were obtained from Gramene (<a href="ftp://ftp.gramene.org/pub/gramene/release-57/gff3/zea_mays/repeat_annotation/B73v4.TE.filtered.gff3.gz">ftp://ftp.gramene.org/pub/gramene/release-57/gff3/zea_mays/repeat_annotation/B73v4.TE.filtered.gff3.gz</a>). Subgenome assignments were from previously reported syntenic block assignments (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Brohammer <em>et al.</em> 2018</a>). Windows were classified into “maize1,” “maize2,” or “nonsyntenic” based on majority assignment. Because annotated gene density is correlated with subgenome assignment (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Schnable <em>et al.</em> 2012</a>), we tested models with and without an interaction between subgenome and gene density. Models with an interaction between subgenome and gene density did not significantly improve model fit (ANOVA of nested models, <em>P</em> &gt; 0.2).</p></div><div id="sec-5" class="subsection"><h3>Duplication Date Estimation</h3><p id="p-15">The dates of tandem duplications were estimated with BEAST version 2.4.7 (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bouckaert <em>et al.</em> 2014</a>). Amino acid alignments of the subgenome homeologues, tandem duplicates, and putative <em>Sorghum</em> ancestral genes that were previously determined (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Brohammer <em>et al.</em> 2018</a>) were generated with Clustal-omega version 1.2.1 (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Sievers <em>et al.</em> 2011</a>). The alignments were back-translated to nucleotides. Each gene alignment was analyzed with BEAST with the following parameters: a GTR+Gamma nucleotide substitution model, estimated transition probabilities and equilibrium base frequencies, a random local clock to allow for branch-specific rate variation, and a monophyletic divergence between the maize subgenomes with a prior of ∼N(11.9, 1) on the divergence date (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Swigonová <em>et al.</em> 2004</a>). The MCMC was run for 10,000,000 steps.</p><p id="p-16">Resulting trees from the BEAST analysis were parsed to obtain the time to most recent common ancestor (TMRCA) between tandem duplicate genes. Homologous genes between B73 and PH207 that share a duplication state were counted as a single duplication event. Gene duplications likely do not follow the infinite sites mutational model (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kimura 1969</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Ohno 1970</a>), and identity by state does not necessarily imply identity by descent. This analysis assumed that tandem duplicates are evolving along truly separate trajectories, and that gene conversion among tandem duplicates is negligible.</p></div><div id="sec-6" class="subsection"><h3>Intersection of Tandem Duplicates and Transposable Elements</h3><p id="p-17">Structural annotation of transposable elements in B73 was obtained from Gramene (<a href="ftp://ftp.gramene.org/pub/gramene/release-57/gff3/zea_mays/repeat_annotation/B73v4.TE.filtered.gff3.gz">ftp://ftp.gramene.org/pub/gramene/release-57/gff3/zea_mays/repeat_annotation/B73v4.TE.filtered.gff3.gz</a>). Genes containing transposable element insertions and genes captured by transposable elements were identified using ‘bedtools intersect’ (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Quinlan and Hall 2010</a>) requiring an overlap fraction of 1.0.</p></div><div id="sec-7" class="subsection"><h3>Relative Rates Calculations</h3><p id="p-18">Relative rates of sequence evolution of maize tandem duplicates were compared with other grass genes by performing Clade model C (CMC) tests (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Weadick and Chang 2012</a>) in PAML v4.9e (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Yang 2007</a>) on tandem duplicates in orthologous gene groups. CMC tests compare dN/dS ratios among a subset of the branches in a gene tree (“foreground”) to the dN/dS ratios on the rest of the tree (“background”). Similar to the BEAST analysis described in the previous section, CMC tests assume that the genes in the tree are independent and non-recombining.</p><p id="p-19">To build alignments and gene trees for each tandem duplicate cluster, orthologous gene groups were identified among publicly available grass genomes from Phytozome V12 and Ensembl Plants V34 using OrthoFinder v1.14 (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Emms and Kelly 2015</a>). The species, sources, and versions of the genomes used as OrthoFinder input are shown in Table S1. OrthoFinder was run with the “dendroblast” ortholog search method, and default parameters for MCL clustering. Amino acid sequences from B73 and PH207 were kept in separate files to allow them to be compared to each other as well as to other grasses. Only orthologous gene groups between 10 and 75 genes were retained for analysis because larger orthologous groups are likely to be gene families that have already diverged in function, and smaller groups do not have enough branches to test relative rates. A distribution of orthologous gene group sizes is shown in Figure S3. Orthologous gene groups that contained between 10 and 75 genes, contained maize tandem duplicates, and contained complete tandem duplicate clusters (<em>i.e.</em>, tandem duplicate clusters that were not split among multiple orthologous groups) were retained for downstream analysis.</p><p id="p-20">Within each of the orthologous groups that passed the above filtering criteria, the amino acid sequences were aligned with clustal-omega and then back-translated to nucleotides using the CDS sequences provided with the genome assemblies. Alignments were filtered to contain only sites with at most 50% gaps (at least eight species) across all sequences, because gaps greatly increase computation time and are not informative for calculating substitution rates. While it is possible that we are excluding a set of rapidly evolving sites with this filter, models of evolution under selective constraint do not yet incorporate insertion and deletion events, and alignments are likely to be misleading in these regions. Maximum likelihood trees were estimated from the filtered alignments with RAxML, using the default rapid hill-climbing search algorithm and a GTR+Gamma nucleotide substitution model.</p><p id="p-21">Four models were fit to the filtered alignments and trees with the ‘codeml’ program in PAML (Figure S4). We tested whether certain foreground branches of the tree exhibited significantly divergent evolutionary rates relative to the remainder of the tree. Model 1 marks all genes as evolving at the same rate (null model) (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Weadick and Chang 2012</a>). Model 2 marks maize genes (including tandem duplicates) and common ancestors of maize genes as foreground and other grass genes as background. Model 3 marks maize tandem duplicates and common ancestors of maize tandem duplicates as different from all other genes. Model 4 distinguishes maize tandem duplicates and common ancestors of maize tandem duplicates from maize genes (Figure S4). The best-fitting model for each orthologous group was identified via a likelihood ratio test against the null model. In orthologous groups where maize genes were evolving at a different rate than tandem duplicate genes (models 3 and 4), omega was compared between tandem and non-tandem maize genes to determine relative constraint. If omega in tandem duplicates was higher than non-tandem duplicates, the tandem duplicate was classified as under weaker constraint, and if it was lower than the non-tandem duplicate it was classified as under stronger constraint. If omega was larger than 10 no test was done as dS was considered too small to test.</p><p id="p-22">We also applied the Hudson-Kreitman-Aguadé (HKA test; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hudson <em>et al.</em> 1987</a>) to test whether tandem duplicates are evolving at different rates from non-tandem duplicates, as implemented in the MLHKA program (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Wright and Charlesworth 2004</a>). High-quality SNP calls from 62 diverse inbred lines (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Brohammer <em>et al.</em>, 2018</a>) were used for diversity data. The number of nucleotide differences between the B73 v4 genome assembly and the <em>Sorghum bicolor</em> v. 3.1 genome assembly was used for divergence data. Diversity and divergence values were treated on a gene-by-gene basis. The list of annotated genes was randomly shuffled and separated into 100-gene windows. For each window, we fit two models: one model treated the tandem duplicates as having a different substitution rate than non-tandem duplicates, and the other model treated all genes as having the same substitution rate. Each window was run with 200,000 MCMC steps. Significance was assessed by likelihood ratio tests (LRT) for each window, with the number of tandem duplicate genes as the degrees of freedom for the chi-squared distribution for the LRT test statistic. If the absolute value of the log(likelihood) was higher for the model with tandems under selection, the window was considered to not have enough power to test the relative rates hypothesis.</p></div><div id="sec-8" class="subsection"><h3>Data Availability</h3><p id="p-23">All supplementary files are available at FigShare. Scripts to perform tandem duplicate identification, sequence alignment and back-translation, orthologous gene group identification, and relative rates comparisons are available at <a href="https://github.com/TomJKono/Maize_Tandem_Evolution">https://github.com/TomJKono/Maize_Tandem_Evolution</a>. Sequence alignments will be made available through the corresponding author upon request. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6062972">https://doi.org/10.25387/g3.6062972</a>.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Tandem duplicate gene clusters are prevalent in maize genomes</h3><p id="p-24">To identify tandem duplicate gene clusters (<em>i.e.</em>, proximally duplicated groups of genes) we utilized the B73 <em>de novo</em> genome assembly generated with single-molecule technology (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Jiao <em>et al.</em> 2017</a>) and the PH207 <em>de novo</em> short-read genome assembly (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Hirsch <em>et al.</em> 2016</a>). The single-molecule technology used for the B73 genome assembly provides a high confidence assembly for evaluating tandem duplicate clusters. The PH207 genome assembly was generated with permissive parameters to avoid collapsing tandem duplicate gene clusters (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Hirsch <em>et al.</em> 2016</a>). However, by the nature of a short read assembly, the PH207 <em>de novo</em> assembly will likely have an underrepresentation of the total tandem duplicate clusters.</p><p id="p-25">Putative tandem duplicates were identified and curated based on a weighted similarity metric that allowed for some interspersed genes. In total, 1,758 tandem duplicate clusters were identified in B73 and 1,467 tandem duplicate clusters were identified in PH207 (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and Table S2). The total number of annotated genes in tandem duplicate clusters was 4,448 (11.3% of the total genes) in B73, and 3,788 (9.3% of the total genes) in PH207. As expected, the number of tandem duplicate clusters and the number of genes within tandem duplicate clusters was slightly lower in PH207. The B73 abundances are likely a more accurate representation of the number of tandem duplicate genes within the maize genome.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490834/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490834/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Tandem duplicate gene cluster and gene counts by subgenome in B73 and PH207. Numbers of clusters are given outside of the parenthesis and number of genes in clusters are given in parentheses</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Similar numbers of shared and private tandem duplicates are observed within species</h3><p id="p-26">Having access to multiple <em>de novo</em> genome assemblies within maize allowed us to determine the consistency of tandem duplicate gene cluster characteristics within the species. A similar distribution was observed between B73 and PH207 for the number of genes per cluster, and the majority of tandem duplicate genes clusters contained only two genes (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). Within clusters, most genes were directly adjacent with no intervening genes in both B73 and PH207 (Figure S5), even though intervening genes were permitted during tandem duplicate identification. Intervening genes were permitted to account for mechanisms that would not cause a duplicate to be directly adjacent but in the same genomic neighborhood, to allow for instances in which a gene is inserted after the duplicate event, and for possible misassembly and misannotation of genes. Only 17% of the duplicate gene clusters that were identified had an interval of greater than two intervening genes between members of the cluster. To determine if specific tandem duplicates were shared between the assemblies, we used homology information that linked the B73 and PH207 gene models (<a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Brohammer <em>et al.</em> 2018</a>). Only 50.3% of the B73 tandem duplicate gene clusters and 60% of the PH207 tandem duplicate clusters were shared between the two genomes (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/3049/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Maize tandem duplicate cluster summary. A) The distribution of cluster sizes in B73 v4 and PH207 v1 genome assemblies. B) Euler diagram of shared and private non-cassette tandem duplicate gene clusters for B73 and PH207. C) Heatmap of log of the number of instances of tandem duplicate gene cluster size relationships between B73 and PH207 (N = 4,393 clusters). Color scale ranges from cream equals zero to red equals 1,257 genes in a cluster. D) Euler diagram of shared and private cassette duplicate gene clusters for B73 and PH207." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1935868967" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Maize tandem duplicate cluster summary. A) The distribution of cluster sizes in B73 v4 and PH207 v1 genome assemblies. B) Euler diagram of shared and private non-cassette tandem duplicate gene clusters for B73 and PH207. C) Heatmap of log of the number of instances of tandem duplicate gene cluster size relationships between B73 and PH207 (N = 4,393 clusters). Color scale ranges from cream equals zero to red equals 1,257 genes in a cluster. D) Euler diagram of shared and private cassette duplicate gene clusters for B73 and PH207.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/3049/F1.medium.gif" width="440" height="430"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/9/3049/F1.medium.gif" width="440" height="430"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/3049/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/3049/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490752" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-27" class="first-child">Maize tandem duplicate cluster summary. A) The distribution of cluster sizes in B73 v4 and PH207 v1 genome assemblies. B) Euler diagram of shared and private non-cassette tandem duplicate gene clusters for B73 and PH207. C) Heatmap of log of the number of instances of tandem duplicate gene cluster size relationships between B73 and PH207 (N = 4,393 clusters). Color scale ranges from cream equals zero to red equals 1,257 genes in a cluster. D) Euler diagram of shared and private cassette duplicate gene clusters for B73 and PH207.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">Another way that individuals can differ with regards to tandem duplicates is the number of duplicate copies within a shared tandem duplicate cluster. To evaluate differences in tandem duplicate copy number between the two genotypes, we compared the 885 tandem duplicate clusters that were shared between the B73 and PH207 genome assembly. Tandem duplicate genes shared between B73 and PH207 exhibited similar cluster sizes as shown by strong heat along the diagonal in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>. That is, when homologous genes were part of tandem clusters in B73 and PH207, these clusters often contained similar numbers of genes. However, there was variation in tandem duplicate cluster size with a difference of up to 16 more copies in one of the genotypes compared to the other (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>).</p></div><div id="sec-12" class="subsection"><h3>Cassette tandem duplication events are rare and often private</h3><p id="p-29">Tandem gene duplication events can occur as a single gene duplication event or in sets of genes that duplicated as a tandem cluster (<em>i.e.</em>, Gene A-1 Gene B-1 followed by Gene A-2 and Gene B-2, see Figure S2). Cassette tandem duplicate gene clusters likely arise from a single event in which a set of genes is duplicated simultaneously. However, it is possible for tandem duplicate cassettes to be generated from multiple duplication events. Candidate tandem duplicate cassettes were identified from interlaced tandem duplicate gene clusters (Figure S2). Cassette duplications were rare in both of the inbred lines with only 58 and 60 cassette duplications in B73 and PH207, respectively (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1D</a>). A higher frequency of private cassette tandem duplicates was observed relative to non-cassette tandem duplicates (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figures 1B</a> and <a id="xref-fig-1-7" class="xref-fig" href="#F1">1D</a>). Only 14 cassettes were shared across genotypes, which equates to approximately one-quarter of the tandem duplicate cluster cassettes in either genome. There was variation for the composition of the cassette between the two genotypes for 10 of the 14 shared cassettes. These differences may be the result of multiple duplication events in one genotype that did not occur in the other, but they are more likely the product of differential loss of a common complete duplication event.</p></div><div id="sec-13" class="subsection"><h3>Tandem duplicate clusters are dispersed throughout the genome and correlate with gene density</h3><p id="p-30">Tandem duplicate genes were identified relatively homogenously throughout the genomes of B73 and PH207 (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and Figure S6). A lower density of tandem duplicates was observed around the centromere where the density of genes is generally lower (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Schnable <em>et al.</em> 2009</a>). To test what variables most explained the distribution of tandem duplicates in the genome, a general linear model was fit with density of tandem duplicates per 1 Mb window regressed against gene density, RNA TE density, DNA TE density, and subgenome assignment within each window. With regards to subgenome, maize is a paleopolyploid that has returned to a diploid state. Two subgenomes remain in the diploid from the most recent allopolyploid event and have been previously characterized based on the number of retained co-orthologous genes to other grass species including <em>Sorghum</em> and rice (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Schnable <em>et al.</em> 2011</a>; <a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">Brohammer <em>et al.</em> 2018</a>). A number of differences are present between the subgenomes such as expression level (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Schnable <em>et al.</em> 2011</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/3049/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic locations of maize tandem duplicates. Purple ticks show tandem duplications. Black line shows gene density, dark gray line shows RNA transposable element density, light gray line shows DNA transposable elements per Mb. Subgenome 1 is shown in green shading and subgenome 2 is shown in blue shading. The top panel shows B73 chromosome 2, and the bottom panel shows PH207 chromosome 2. All chromosomes can be found in Figure S6." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1935868967" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genomic locations of maize tandem duplicates. Purple ticks show tandem duplications. Black line shows gene density, dark gray line shows RNA transposable element density, light gray line shows DNA transposable elements per Mb. Subgenome 1 is shown in green shading and subgenome 2 is shown in blue shading. The top panel shows B73 chromosome 2, and the bottom panel shows PH207 chromosome 2. All chromosomes can be found in Figure S6.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/3049/F2.medium.gif" width="440" height="424"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/9/3049/F2.medium.gif" width="440" height="424"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/3049/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/3049/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490823" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-31" class="first-child">Genomic locations of maize tandem duplicates. Purple ticks show tandem duplications. Black line shows gene density, dark gray line shows RNA transposable element density, light gray line shows DNA transposable elements per Mb. Subgenome 1 is shown in green shading and subgenome 2 is shown in blue shading. The top panel shows B73 chromosome 2, and the bottom panel shows PH207 chromosome 2. All chromosomes can be found in Figure S6.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">As expected, gene density explained the most variance in tandem duplicate density per window (<em>t</em>-test of regression coefficient, <em>P</em> &lt; 0.001), and only 1% more variance was explained by a model containing all of these factors than a model with only gene density. When comparing the proportion of bases within a window in the genome <em>vs.</em> the proportion of genic bases in tandem duplicates we do not see any significant relationship (<em>R</em>=-0.037; <em>P</em> = 0.092). This indicates that the high proportion of variation explained by gene density is a product of the fact that there are more genes in these regions to be potentially duplicated rather than there being a mechanistic impact of having higher gene density on creating more or less than expected numbers of tandem duplicates relative to the number of genes in a window.</p><p id="p-33">When testing the effects of transposable elements, both class 1 and class 2 transposable element density were significant at the <em>P</em> &lt; 0.05 threshold, and higher transposable element density was associated with higher tandem duplicate density. Maize subgenome 2 had lower tandem duplicate density than subgenome 1 (<em>P</em> &lt; 0.001). On average, 49.5% of tandem duplicate clusters were in subgenome 1 (37.7% genome-wide), 21.6% were in subgenome 2 (24.0% genome-wide), and 28.9% were in non-syntenic regions (38.3% genome-wide) of the genome. This is consistent with the results of the general linear model where subgenome 1 has proportionally more tandem duplicates relative to the number of genes in the subgenome (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>).</p></div><div id="sec-14" class="subsection"><h3>Extant tandem duplicates date to two distinct periods</h3><p id="p-34">Phylogenetic analyses were used to estimate the date of origin of tandem duplicates. For each tandem duplicate cluster, all maize B73 and PH207 homeologs (<em>i.e.</em>, subgenome 1 and subgenome 2 copies) and the corresponding <em>Sorghum</em> gene for each of the tandem duplicate clusters were included. Phylogenetic trees were calibrated based on the estimated divergence time of maize and <em>Sorghum</em> at approximately 12 million years ago (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Swigonová <em>et al.</em> 2004</a>). Our hypothesis was that tandem duplicate gene clusters that were shared between B73 and PH207 would be older than those that are unique to either one of the genomes. We see examples of tandem duplicates that were shared and date near the divergence time of maize and <em>Sorghum</em> at approximately 11.7 million years ago (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). Additionally, we observe private tandem duplicates that were estimated to have arisen relatively recently (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/3049/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Date estimates of maize tandem duplications. A) Example of a BEAST tree that dated a tandem duplicate gene cluster as an ancient duplication. B) Example of a BEAST tree that dated a tandem duplicate gene cluster as a relatively recent duplication. In both trees red shows the consensus topology and alternate shading color indicates alternate topologies. The trees in A and B are calibrated based on an approximate divergence date for maize and Sorghum of 12 million years ago. C) Distribution of estimated syntenic and non-syntenic duplication ages. Estimates dates are based on substitution rates relative to the outgroup Sorghum for each tandem duplicate gene based on branch length determined from the BEAST trees. The black triangle shows the estimated divergence date of maize and Sorghum. Shared tandem duplicate clusters are contained in both B73 and PH207 and private are only duplicated in one of the two genomes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1935868967" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Date estimates of maize tandem duplications. A) Example of a BEAST tree that dated a tandem duplicate gene cluster as an ancient duplication. B) Example of a BEAST tree that dated a tandem duplicate gene cluster as a relatively recent duplication. In both trees red shows the consensus topology and alternate shading color indicates alternate topologies. The trees in A and B are calibrated based on an approximate divergence date for maize and&lt;em&gt; Sorghum&lt;/em&gt; of 12 million years ago. C) Distribution of estimated syntenic and non-syntenic duplication ages. Estimates dates are based on substitution rates relative to the outgroup &lt;em&gt;Sorghum&lt;/em&gt; for each tandem duplicate gene based on branch length determined from the BEAST trees. The black triangle shows the estimated divergence date of maize and &lt;em&gt;Sorghum&lt;/em&gt;. Shared tandem duplicate clusters are contained in both B73 and PH207 and private are only duplicated in one of the two genomes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/3049/F3.medium.gif" width="184" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/9/3049/F3.medium.gif" width="184" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/3049/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/3049/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490882" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-35" class="first-child">Date estimates of maize tandem duplications. A) Example of a BEAST tree that dated a tandem duplicate gene cluster as an ancient duplication. B) Example of a BEAST tree that dated a tandem duplicate gene cluster as a relatively recent duplication. In both trees red shows the consensus topology and alternate shading color indicates alternate topologies. The trees in A and B are calibrated based on an approximate divergence date for maize and<em> Sorghum</em> of 12 million years ago. C) Distribution of estimated syntenic and non-syntenic duplication ages. Estimates dates are based on substitution rates relative to the outgroup <em>Sorghum</em> for each tandem duplicate gene based on branch length determined from the BEAST trees. The black triangle shows the estimated divergence date of maize and <em>Sorghum</em>. Shared tandem duplicate clusters are contained in both B73 and PH207 and private are only duplicated in one of the two genomes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">Across all the tandem duplicate gene clusters a bimodal age distribution was observed, with most tandem duplicates either dating to approximately the time of maize and <em>Sorghum</em> divergence or dating quite recently in evolutionary time (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>). Consistent with our hypothesis, tandem duplicate gene clusters that were shared between the two genomes were mostly inferred to be ancient for both subgenome 1 and subgenome 2 gene clusters. Private syntenic duplicates had both ancient and recent inferred ages. Of the 1,044 private clusters, 628 had an estimated date in the ancient peak and likely represent a gene loss event in one genotype and not the other. A comparable number of non-syntenic tandem duplicate gene clusters arose during both of the bimodal age peaks, similar to what was observed for private syntenic tandem duplicate gene clusters.</p><p id="p-37">One explanation for the large proportion of inferred recent duplications is the action of gene conversion. Gene conversion would cause tandem duplicate genes to have higher sequence similarity than non-recombining duplicates of the same age, and thus would bias estimates toward recent events. Gene conversion is often associated with increased GC content due to GC-biased gene conversion (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Pessia <em>et al.</em> 2012</a>). Indeed, tandem duplicate genes showed a higher GC content on average than the GC content that was observed for all maize genes whether in the syntenic or non-syntenic portion of the genome (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). If GC biased gene conversion were contributing to the recent date estimates of tandem duplicates (less than 2 million years ago), we would expected recent tandem duplicates to have higher GC content than ancient tandem duplicates. However, the opposite was observed. Tandem duplicates that were inferred to be ancient had consistently higher GC content, and those that were inferred to be recent exhibited a high proportion of low GC content genes (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). Thus, GC-biased gene conversion was likely not artificially deflating age estimates between duplicates to a substantial degree. However, this is based on using GC content as a proxy for biased gene conversion, and we are not estimating the rate of gene conversion among tandem duplicates.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/3049/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="GC content of tandem duplicate gene clusters. A) GC content of genome wide genes (gray), syntenic tandem duplicates (blue), and non-syntenic tandem duplicates (black). B) GC content of ancient (&#x2265; 10MYA, purple solid) and recent (&#x2264; 2MYA, purple dotted) tandem duplications. Distributions contain both B73 and PH207 tandem duplicate gene clusters." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1935868967" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;GC content of tandem duplicate gene clusters. A) GC content of genome wide genes (gray), syntenic tandem duplicates (blue), and non-syntenic tandem duplicates (black). B) GC content of ancient (&#x2265; 10MYA, purple solid) and recent (&#x2264; 2MYA, purple dotted) tandem duplications. Distributions contain both B73 and PH207 tandem duplicate gene clusters.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/3049/F4.medium.gif" width="440" height="194"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/9/3049/F4.medium.gif" width="440" height="194"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/3049/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/3049/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490861" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-38" class="first-child">GC content of tandem duplicate gene clusters. A) GC content of genome wide genes (gray), syntenic tandem duplicates (blue), and non-syntenic tandem duplicates (black). B) GC content of ancient (≥ 10MYA, purple solid) and recent (≤ 2MYA, purple dotted) tandem duplications. Distributions contain both B73 and PH207 tandem duplicate gene clusters.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Tandem duplicated genes are shorter and more likely to contain LTR transposable elements</h3><p id="p-39">One possible mechanism through which tandem duplicates could arise is through transposable elements. Some TIR elements are enriched for local movement and could contribute to tandem duplication of entire genes that are captured and moved locally. In contrast, LTR elements do not typically move locally, but may also generate tandem duplicates. The mechanism of LTR movement is expected to generate single-exon tandem duplicate genes upon local transposition. In both B73 and PH207, a higher proportion of single exon genes in tandem duplicate gene clusters than was observed genome-wide in non-tandem duplicates (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>, <a id="xref-fig-5-2" class="xref-fig" href="#F5">5B</a>). Instances in which a tandem duplicate cluster comprised a gene with multiple exons and its tandem duplicate contained only a single exon would further support the mechanism that the gene was duplicated through an RNA intermediate. Of the clusters that had a single exon gene (24% of total clusters), only one-third in B73 and two-thirds in PH207 also had a gene with multiple exons. However, 835 (B73) and 815 (PH207) clusters had multiple exons in both genes in the tandem duplicate gene cluster. These results indicate that while some tandem duplicate genes may have arisen through an RNA intermediate, this was not the predominant mechanism.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/3049/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Size distribution of tandem duplicate gene clusters. A) Number of exons in B73 tandem duplicates relative to all other genes genome-wide. B) Number of exons in PH207 tandem duplicates relative to all other genes genome-wide. C) Gene length distribution for single-exon genes and multi-exon genes in B73 and PH207." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1935868967" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Size distribution of tandem duplicate gene clusters. A) Number of exons in B73 tandem duplicates relative to all other genes genome-wide. B) Number of exons in PH207 tandem duplicates relative to all other genes genome-wide. C) Gene length distribution for single-exon genes and multi-exon genes in B73 and PH207.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/3049/F5.medium.gif" width="440" height="415"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/9/3049/F5.medium.gif" width="440" height="415"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/3049/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/3049/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490743" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-40" class="first-child">Size distribution of tandem duplicate gene clusters. A) Number of exons in B73 tandem duplicates relative to all other genes genome-wide. B) Number of exons in PH207 tandem duplicates relative to all other genes genome-wide. C) Gene length distribution for single-exon genes and multi-exon genes in B73 and PH207.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">Another explanation for having a higher than expected proportion of single exon genes in tandem duplicate gene clusters is that these genes were generally shorter and therefore easier to duplicate. For both single exon genes as well as genes with multiple exons in both B73 and PH207, genes that were in tandem duplicate gene clusters were shorter than the genome-wide distribution of gene sizes for single and multi exon gene models based on full gene model length (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a>).</p><p id="p-42">In addition to physically copying genes, transposable elements can also contribute to generating tandem duplicate gene clusters by providing microhomology for nonhomologous recombination. We investigated the relative proximity of tandem duplicate gene clusters to LTR, LINE, SINE, and TIR elements and found no difference in distance to nearest transposable element for tandem duplicate genes <em>vs.</em> the genome-wide distribution of non-tandem duplicate genes (Figure S7). However, there was a substantial enrichment of LTRs inserted into tandem duplicate genes (20.0% of tandem duplicates contained LTRs <em>vs.</em> 8.2% of non-tandem duplicates) and a de-enrichment of tandem duplicate genes that were captured (the entire genic sequence being nested within an element) by LTRs relative to the rate in non-tandem duplicates (2.6% of tandem duplicates <em>vs.</em> 4.6% of non-tandem duplicates; <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490817/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490817/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Counts of tandem duplicate genes and non tandem duplicate genes that contain a transposable element insertion or are nested within a transposable element in the B73 genome</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Recent tandem duplicates evolve at different rates than other maize genes</h3><p id="p-43">We were interested in examining the relative substitution rates of recent tandem duplicates to infer potential evolutionary trajectories of newly arisen tandem duplications. Substitution rates of recent tandem duplicate genes present in syntenic regions were compared to non-tandem duplicate maize genes and grass orthologs with clade models in PAML (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Yang 2007</a>; <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Weadick and Chang 2012</a>). Only duplications that were private to a subgenome and private to either B73 or PH207 were analyzed, as a proxy for recent gene duplication events. A total of 120 grass orthologous groups with maize tandem duplicates met our filtering criteria (see Methods). Four competing models were tested that included testing 1) evolutionary rates of grass genes equal maize genes and equal tandem duplicate genes, 2) evolutionary rates of grass genes do not equal maize genes but equal tandem duplicate genes, 3) evolutionary rates of grass genes equal maize genes but do not equal tandem duplicate genes, and 4) evolutionary rates of grass genes do not equal maize genes and do not equal tandem duplicate genes (Figure S4). The majority (74.2%) of the tandem duplications did not show evidence of evolving at a different rate from their grass orthologs (models 1 and 2). Of the 31 (25.8%) tandem duplicate clusters that have evolved at a different rate than the remainder of the tree (models 3 and 4), 10 tandem duplicates showed lower dN/dS than the remainder of the tree, seven showed higher dN/dS than non tandem duplicates, and 14 did not have enough dS to compare substitution rates (Figure S4). To validate these results we also applied the Hudson-Kreitman-Aguade (HKA) test, which tests for independence of nucleotide diversity from divergence from the outgroup. The majority of genes (94%) did not have sufficient power to test for selection. Of those that could be tested, approximately 13% of tandem duplicates showed evidence of evolving at a different rate than non-tandem duplicates. From the HKA test there are less tandem duplicates showing evidence of evolving at a different rate than non-tandem duplicates. However, only a small portion of tandem duplicates could be tested. Additionally, the HKA test makes a number of assumptions about demographic history that are violated for tandem duplicates. It should also be noted that both of these tests are tests of coding sequences and do not assay the noncoding sequences which could alter gene functionality. Additionally, specific gene conversion events could potentially alter dN and/or dS which could impact omega estimates in the PAML analysis.</p></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><p id="p-44">Tandem duplicate gene studies in plants have primarily focused on single loci and questions about evolutionary mechanism and functional impacts on a genome-wide scale have been limited by available genomic resources. Maize offers a unique opportunity to address questions regarding tandem duplicate origin and evolution given its large, highly repetitive genome and the availability of multiple high quality and well-annotated <em>de novo</em> genome assemblies. Using these resources, we showed that tandem duplicate gene clusters are prevalent in maize, and there is variation within maize lines for tandem duplicate content in the genome. While tandem duplicate clusters are dispersed in genome space, standing variation in tandem duplicates date to two distinct times. A variety of features in the genome were evaluated for association with tandem duplicates, and gene density was the most informative factor for explaining variation in tandem duplicate location across the genome. Tandem duplicate genes are shorter than non-tandem duplicate genes and are more likely to contain LTR transposable elements, which may speak to their origin. For a subset of the tandem duplicates that could be tested, approximately one-quarter were evolving at different rates than other maize genes. These duplicates, along with others that are likely evolving in regulatory control, have potential to generate new functional variation.</p><p id="p-45">While tandem duplicate gene clusters are abundant and generally dispersed throughout the genome at a density similar to the genome-wide gene density, we observed some bias in the location of tandem duplicate gene clusters. Specifically, maize subgenome 2 has proportionally fewer tandem duplicates than either subgenome 1 or non-syntenic regions, accounting for their relative gene densities. There are several explanations for this. One is that duplications may not arise in maize subgenome 2 as readily as other genomic regions. This is unlikely, however, because the patterns of transposable elements and recombination dynamics of subgenome 2 are similar to subgenome 1 (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Schnable <em>et al.</em> 2011</a>). Additionally, subgenome 2 is under weaker purifying selection than subgenome 1 (<a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Schnable <em>et al.</em> 2011</a>), meaning that gene duplications should be more prevalent in subgenome 2 but this was not observed. Another explanation is that genes in subgenome 2 “degenerate” more rapidly than in subgenome 1, and that genes that are truly tandem duplicates are too divergent at the sequence level at this point in time to be identified as duplicates by our methods. It has also been shown that subgenome 2 has a higher ongoing deletion rate than subgenome 1 (<a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">Schnable <em>et al.</em> 2011</a>), and as such may have proportionally fewer tandems due to a faster rate of deletion from subgenome 2.</p><p id="p-46">While dispersed throughout the genome, the estimated ages of standing tandem duplicates is not dispersed throughout evolutionary time. In fact, tandem duplicates have a bimodal distribution of estimated ages (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>). The older peak in the estimated age distribution coincides with the divergence of maize from <em>Sorghum</em> (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Swigonová <em>et al.</em> 2004</a>). This is not unexpected, because the genomic instability and rearrangements caused by an allopolyploidy event can result in many tandem duplicates (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Blanc and Wolfe 2004</a>). The more recent peak coincides with the expansion of maize as an agricultural plant (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wang <em>et al.</em> 2017</a>). The demographic effects and linked selection associated with domestication of maize (Wright <em>et al.</em> 2005) would have multiple effects that affect tandem duplicates: both genetic hitchhiking near “domestication loci” and reduction in the efficacy of purifying selection may increase the frequency of gene duplications in the genome.</p><p id="p-47">There are two processes that can potentially contribute to the observed pattern of bimodally-distributed tandem duplicate dates. One is that the allopolyploid event that occurred near the divergence time of the maize progenitor from the <em>Sorghum bicolor</em> progenitor led to an increased rate of tandem duplication. Genome instability through increased probability of meiotic errors has been documented during both genome duplication events and polyploidization events (reviewed by <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hollister (2015)</a>). However, duplicated genes also exhibit a propensity for loss or functional divergence, processes which erode evidence of recent duplication. These mechanisms are not independent. It is possible that both of these processes are acting in maize to contribute to the standing variation of tandem duplicates. However, a detailed study of many independent maize genome assemblies in a population with known genetic structure would be required to assess the relative contributions of these two mechanisms.</p><p id="p-48">In addition to determining when in evolutionary history tandem duplicates arose, we also tried to determine their mechanistic origin. Our results suggest that tandem duplicates have a close association with transposable elements. This is evident at both the genomic distribution level (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>), and direct comparison of gene model annotations and transposable element annotations, where there is an enrichment of LTRs that are nested within tandem duplicates relative to non-tandem duplicates (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). This suggests that tandem duplicates may arise through a mechanism that preferentially operates on highly repetitive sequence such as transposable elements, such as unequal crossing over (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Smith 1976</a>). Errors in meiotic chromosome pairing are often the result of repetitive elements like tandemly arrayed genes (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Yandeau-Nelson <em>et al.</em> 2006</a>), and may contribute to the substantial level of gene copy number variants observed in maize (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Springer <em>et al.</em> 2009</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Swanson-Wagner <em>et al.</em> 2010</a>). Tandem duplicates may also be generated via transcription-mediated mechanisms associated with RNA transposable elements. We observed a higher proportion of single exon genes in tandem duplicate gene clusters relative to non-duplicated genes. A strong signature of an RNA-intermediate in a tandem duplication would be if one gene within the tandem duplicate contained multiple exons and the other gene within the tandem duplicate contained only a single exon that is the product of a spliced mature mRNA. We did not see this pattern in many of tandem duplicate gene clusters. Tandem duplicate genes, whether single exon or multi exon, however, were generally shorter than non-tandem duplicates. This points to an explanation that single exon genes, which are generally shorter, are easier to copy intact through mechanisms such as non-homologous recombination (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Smith 1976</a>), rather than a RNA intermediate.</p><p id="p-49">In this study, we constrained our analyses to annotated genes. However, tandem duplication can affect regulatory elements or gene fragments (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Rogers <em>et al.</em> 2017</a>). Duplication of functional elements that are not full-length protein coding genes likely has an impact on phenotypic variation, and therefore, evolution of genome structure. Our work presents a special case of tandem duplications, in which entire genes are duplicated. However, we show that even this special case of tandem duplication can affect thousands of genes genome-wide and has the potential for functional outcomes.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-50">This work was supported by the National Science Foundation [IOS-1546727] to CNH and SEM. ABB was supported by a DuPont Pioneer Bill Kuhn Honorary Fellowship and a MnDRIVE Global Food Ventures fellowship. The authors acknowledge the Minnesota Supercomputing Institute (MSI) at the University of Minnesota for providing resources that contributed to the research results reported within this paper <a href="http://www.msi.umn.edu">http://www.msi.umn.edu</a>.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-51">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6062972">https://doi.org/10.25387/g3.6062972</a>.</p></li><li class="fn" id="fn-3"><p id="p-52"><em>Communicating editor: D. Macqueen</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-03-29"><span class="received-label">Received </span>March 29, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-07-17"><span class="accepted-label">Accepted </span>July 17, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Kono <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roth</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> Spontaneous tandem genetic duplications in <em>Salmonella typhimurium</em> arise by unequal recombination between rRNA (rrn) cistrons. Proc. Natl. Acad. Sci. 78: 3113–3117.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Athma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Ac induces homologous recombination at the maize P locus.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">173</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAthma%26rft.auinit1%253DP.%26rft.volume%253D128%26rft.issue%253D1%26rft.spage%253D163%26rft.epage%253D173%26rft.atitle%253DAc%2Binduces%2Bhomologous%2Brecombination%2Bat%2Bthe%2Bmaize%2BP%2Blocus.%26rft_id%253Dinfo%253Apmid%252F1648001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI4LzEvMTYzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzMwNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.3" data-doi="10.1105/tpc.021345"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Widespread paleopolyploidy in model plant species inferred from age distributions of duplicate genes.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1667</span>–<span class="cit-lpage">1678</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.021345</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.021345%26rft_id%253Dinfo%253Apmid%252F15208399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjE2LzcvMTY2NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOS8zMDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.4" data-doi="10.1371/journal.pcbi.1003537"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bouckaert</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heled</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kühnert</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaughan</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">BEAST 2: a software platform for Bayesian evolutionary analysis.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1003537</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1003537</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D10%26rft.spage%253D1003537e%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1003537%26rft_id%253Dinfo%253Apmid%252F24722319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1003537&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24722319&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.5" data-doi="10.1111/tpj.13765"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brohammer</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kono</span>  <span class="cit-name-given-names">T. J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Springer</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGaugh</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">C. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The limited role of differential fractionation in genome content variation and function in maize (<em>Zea mays</em> L.) inbred lines.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">141</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.13765</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D93%26rft.spage%253D131%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13765&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.6" data-doi="10.1186/1471-2229-4-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cannon</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baumgarten</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The roles of segmental and tandem gene duplication in the evolution of large gene families in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2229-4-10</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DCannon%26rft.auinit1%253DS.%2BB.%26rft.volume%253D4%26rft.spage%253D10%26rft.epage%253D10%26rft.atitle%253DThe%2Broles%2Bof%2Bsegmental%2Band%2Btandem%2Bgene%2Bduplication%2Bin%2Bthe%2Bevolution%2Bof%2Blarge%2Bgene%2Bfamilies%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-4-10%26rft_id%253Dinfo%253Apmid%252F15171794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-4-10&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15171794&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.7" data-doi="10.1038/ng.2313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chia</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costich</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Leon</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Maize HapMap2 identifies extant variation from a genome in flux.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">803</span>–<span class="cit-lpage">807</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DChia%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D44%26rft.issue%253D7%26rft.spage%253D803%26rft.epage%253D807%26rft.atitle%253DMaize%2BHapMap2%2Bidentifies%2Bextant%2Bvariation%2Bfrom%2Ba%2Bgenome%2Bin%2Bflux.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2313%26rft_id%253Dinfo%253Apmid%252F22660545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2313&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22660545&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.8" data-doi="10.1126/science.1228746"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">T. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melito</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Copy number variation of multiple genes at <em>Rhg1</em> mediates nematode resistance in soybean.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">338</span>: <span class="cit-fpage">1206</span>–<span class="cit-lpage">1209</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1228746</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1228746%26rft_id%253Dinfo%253Apmid%252F23065905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzgvNjExMS8xMjA2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzMwNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.9" data-doi="10.1073/pnas.1608775113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Analysis of tandem gene copies in maize chromosomal regions reconstructed from long sequence reads.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">7949</span>–<span class="cit-lpage">7956</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1608775113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1608775113%26rft_id%253Dinfo%253Apmid%252F27354512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEzLzI5Lzc5NDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMzA0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dooner</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kermicle</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">Structure of the <em>Rr</em> tandem duplication in maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">427</span>–<span class="cit-lpage">436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDooner%26rft.auinit1%253DH.%2BK.%26rft.volume%253D67%26rft.issue%253D3%26rft.spage%253D427%26rft.epage%253D436%26rft.atitle%253DSTRUCTURE%2BOF%2BTHE%2BRr%2BTANDEM%2BDUPLICATION%2BIN%2BMAIZE%26rft_id%253Dinfo%253Apmid%252F17248520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiI2Ny8zLzQyNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOS8zMDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.11" data-doi="10.1186/s13059-015-0721-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Emms</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">157</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-015-0721-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D157%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0721-2%26rft_id%253Dinfo%253Apmid%252F26243257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0721-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26243257&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.12" data-doi="10.1111/j.1469-8137.2009.02923.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flagel</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Gene duplication and evolutionary novelty in plants.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">557</span>–<span class="cit-lpage">564</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-8137.2009.02923.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DFlagel%26rft.auinit1%253DL.%2BE.%26rft.volume%253D183%26rft.issue%253D3%26rft.spage%253D557%26rft.epage%253D564%26rft.atitle%253DGene%2Bduplication%2Band%2Bevolutionary%2Bnovelty%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2009.02923.x%26rft_id%253Dinfo%253Apmid%252F19555435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2009.02923.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19555435&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268100800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Force</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pickett</span>  <span class="cit-name-given-names">F. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Preservation of duplicate genes by complementary, degenerative mutations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">1531</span>–<span class="cit-lpage">1545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DForce%26rft.auinit1%253DA.%26rft.volume%253D151%26rft.issue%253D4%26rft.spage%253D1531%26rft.epage%253D1545%26rft.atitle%253DPreservation%2Bof%2BDuplicate%2BGenes%2Bby%2BComplementary%252C%2BDegenerative%2BMutations%26rft_id%253Dinfo%253Apmid%252F10101175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1MS80LzE1MzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMzA0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.14" data-doi="10.1105/tpc.112.100537"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martienssen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Pod corn is caused by rearrangement at the <em>Tunicate1</em> locus.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2733</span>–<span class="cit-lpage">2744</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.112.100537</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.112.100537%26rft_id%253Dinfo%253Apmid%252F22829149%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI0LzcvMjczMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOS8zMDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.15" data-doi="10.1016/0022-2836(77)90077-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grafstrom</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harnish</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hillman</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Tandem duplications resulting from recombination between ribosomal RNA genes in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">407</span>–<span class="cit-lpage">428</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0022-2836(77)90077-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DHill%26rft.auinit1%253DC.%2BW.%26rft.volume%253D116%26rft.issue%253D3%26rft.spage%253D407%26rft.epage%253D428%26rft.atitle%253DTandem%2Bduplications%2Bresulting%2Bfrom%2Brecombination%2Bbetween%2Bribosomal%2BRNA%2Bgenes%2Bin%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252877%252990077-8%26rft_id%253Dinfo%253Apmid%252F338920%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(77)90077-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=338920&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1977ED58800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.16" data-doi="10.1105/tpc.16.00353"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brohammer</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soifer</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Draft assembly of elite inbred line PH207 provides insights into genomic and transcriptome diversity in maize.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2700</span>–<span class="cit-lpage">2714</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.16.00353</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.16.00353%26rft_id%253Dinfo%253Apmid%252F27803309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyOC8xMS8yNzAwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzMwNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.17" data-doi="10.1111/nph.12939"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hollister</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Polyploidy: adaptation to the genomic environment.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">205</span>: <span class="cit-fpage">1034</span>–<span class="cit-lpage">1039</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.12939</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D205%26rft.spage%253D1034%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.12939%26rft_id%253Dinfo%253Apmid%252F25729801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.12939&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25729801&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreitman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguadé</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">A test of neutral molecular evolution based on nucleotide data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">153</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHudson%26rft.auinit1%253DR.%2BR.%26rft.volume%253D116%26rft.issue%253D1%26rft.spage%253D153%26rft.epage%253D159%26rft.atitle%253DA%2BTest%2Bof%2BNeutral%2BMolecular%2BEvolution%2BBased%2Bon%2BNucleotide%2BData%26rft_id%253Dinfo%253Apmid%252F3110004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTE2LzEvMTUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzMwNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.19" data-doi="10.1038/nrg2689"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Innan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kondrashov</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The evolution of gene duplications: classifying and distinguishing between models.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">108</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2689</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DInnan%26rft.auinit1%253DH.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D97%26rft.epage%253D108%26rft.atitle%253DThe%2Bevolution%2Bof%2Bgene%2Bduplications%253A%2Bclassifying%2Band%2Bdistinguishing%2Bbetween%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2689%26rft_id%253Dinfo%253Apmid%252F20051986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2689&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20051986&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273688200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.20" data-doi="10.1038/nature22971"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peluso</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stitzer</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Improved maize reference genome with single-molecule technologies.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">546</span>: <span class="cit-fpage">524</span>–<span class="cit-lpage">527</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature22971</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D546%26rft.spage%253D524%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature22971%26rft_id%253Dinfo%253Apmid%252F28605751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature22971&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28605751&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1969</span> <span class="cit-article-title">The number of heterozygous nucleotide sites maintained in a finite population due to steady flux of mutations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">893</span>–<span class="cit-lpage">903</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKimura%26rft.auinit1%253DM.%26rft.volume%253D61%26rft.issue%253D4%26rft.spage%253D893%26rft.epage%253D903%26rft.atitle%253DTHE%2BNUMBER%2BOF%2BHETEROZYGOUS%2BNUCLEOTIDE%2BSITES%2BMAINTAINED%2BIN%2BA%2BFINITE%2BPOPULATION%2BDUE%2BTO%2BSTEADY%2BFLUX%2BOF%2BMUTATIONS%26rft_id%253Dinfo%253Apmid%252F5364968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiI2MS80Lzg5MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOS8zMDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laughnan</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1952</span> <span class="cit-article-title">The action of allelic forms of the gene <em>A</em> in maize. IV. on the compound nature of <em>A</em> and the occurrence and action of its <em>A</em> derivatives.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLaughnan%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D37%26rft.issue%253D4%26rft.spage%253D375%26rft.epage%253D395%26rft.atitle%253DTHE%2BACTION%2BOF%2BALLELIC%2BFORMS%2BOF%2BTHE%2BGENE%2BA%2BIN%2BMAIZE.%2BIV.%2BON%2BTHE%2BCOMPOUND%2BNATURE%2BOF%2BAb%2BAND%2BTHE%2BOCCURRENCE%2BAND%2BACTION%2BOF%2BITS%2BAd%2BDERIVATIVES%26rft_id%253Dinfo%253Apmid%252F17247400%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIzNy80LzM3NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOS8zMDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.23" data-doi="10.1016/j.tig.2004.01.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leister</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Tandem and segmental gene duplication and recombination in the evolution of plant disease resistance gene.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">116</span>–<span class="cit-lpage">122</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2004.01.007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DLeister%26rft.auinit1%253DD.%26rft.volume%253D20%26rft.issue%253D3%26rft.spage%253D116%26rft.epage%253D122%26rft.atitle%253DTandem%2Band%2Bsegmental%2Bgene%2Bduplication%2Band%2Brecombination%2Bin%2Bthe%2Bevolution%2Bof%2Bplant%2Bdisease%2Bresistance%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2004.01.007%26rft_id%253Dinfo%253Apmid%252F15049302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2004.01.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15049302&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220327300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.24" data-doi="10.1104/pp.108.124867"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lyons</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pedersen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kane</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alam</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ming</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Finding and comparing syntenic regions among <em>Arabidopsis</em> and the outgroups papaya, poplar, and grape: CoGe with rosids.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">1772</span>–<span class="cit-lpage">1781</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.108.124867</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.108.124867%26rft_id%253Dinfo%253Apmid%252F18952863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTQ4LzQvMTc3MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOS8zMDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.25" data-doi="10.1534/genetics.115.183483"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mak</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">Y. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">E. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwok</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leung</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-wide structural variation detection by genome mapping on nanochannel arrays.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">351</span>–<span class="cit-lpage">362</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.183483</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.183483%26rft_id%253Dinfo%253Apmid%252F26510793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAyLzEvMzUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzMwNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.26" data-doi="10.1073/pnas.1220766110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maron</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guimarães</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirst</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birchler</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Aluminum tolerance in maize is associated with higher <em>MATE1</em> gene copy number.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">5241</span>–<span class="cit-lpage">5246</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1220766110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1220766110%26rft_id%253Dinfo%253Apmid%252F23479633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzEzLzUyNDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMzA0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-book" id="cit-8.9.3049.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohno</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Tandem duplication involving part of one linkage group at a time</span>, pp. <span class="cit-fpage">89</span>–<span class="cit-lpage">97</span> in <span class="cit-source">Evolution by Gene Duplication</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.28" data-doi="10.1093/gbe/evs052"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pessia</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Popa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mousset</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rezvoy</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duret</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evidence for widespread GC-biased gene conversion in eukaryotes.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">675</span>–<span class="cit-lpage">682</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evs052</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevs052%26rft_id%253Dinfo%253Apmid%252F22628461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evs052&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22628461&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.29" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btq033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20110278&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275243500019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-web" id="cit-8.9.3049.30"><div class="cit-metadata unstructured">R Core Team 2018 R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. URL <a href="https://www.R-project.org/">https://www.R-project.org/</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.31" data-doi="10.1371/journal.pgen.1006795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shao</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Tandem duplications lead to novel expression patterns through exon shuffling in <em>Drosophila yakuba</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">e1006795</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1006795</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D13%26rft.spage%253De1006795%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006795%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.32" data-doi="10.1126/science.287.5461.2204"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">G. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Comparative genomics of the eukaryotes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">287</span>: <span class="cit-fpage">2204</span>–<span class="cit-lpage">2215</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.287.5461.2204</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRubin%26rft.auinit1%253DG.%2BM.%26rft.volume%253D287%26rft.issue%253D5461%26rft.spage%253D2204%26rft.epage%253D2215%26rft.atitle%253DComparative%2BGenomics%2Bof%2Bthe%2BEukaryotes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.287.5461.2204%26rft_id%253Dinfo%253Apmid%252F10731134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODcvNTQ2MS8yMjA0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzMwNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.33" data-doi="10.1093/gbe/evs009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schnable</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freeling</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyons</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide analysis of syntenic gene deletion in the grasses.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">265</span>–<span class="cit-lpage">277</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evs009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevs009%26rft_id%253Dinfo%253Apmid%252F22275519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evs009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22275519&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.34" data-doi="10.1073/pnas.1101368108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schnable</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Springer</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freeling</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Differentiation of the maize subgenomes by genome dominance and both ancient and ongoing gene loss.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">4069</span>–<span class="cit-lpage">4074</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1101368108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1101368108%26rft_id%253Dinfo%253Apmid%252F21368132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzEwLzQwNjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMzA0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.35" data-doi="10.1126/science.1178534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schnable</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ware</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fulton</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The B73 maize genome: complexity, diversity, and dynamics.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">326</span>: <span class="cit-fpage">1112</span>–<span class="cit-lpage">1115</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1178534</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSchnable%26rft.auinit1%253DP.%2BS.%26rft.volume%253D326%26rft.issue%253D5956%26rft.spage%253D1112%26rft.epage%253D1115%26rft.atitle%253DThe%2BB73%2BMaize%2BGenome%253A%2BComplexity%252C%2BDiversity%252C%2Band%2BDynamics%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1178534%26rft_id%253Dinfo%253Apmid%252F19965430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjYvNTk1Ni8xMTEyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzMwNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.36" data-doi="10.1038/msb.2011.75"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sievers</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilm</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dineen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karplus</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">539</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/msb.2011.75</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DSievers%26rft.auinit1%253DF.%26rft.volume%253D7%26rft.issue%253D1%26rft.spage%253D539%26rft.epage%253D539%26rft.atitle%253DFast%252C%2Bscalable%2Bgeneration%2Bof%2Bhigh-quality%2Bprotein%2Bmultiple%2Bsequence%2Balignments%2Busing%2BClustal%2BOmega%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2011.75%26rft_id%253Dinfo%253Apmid%252F21988835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjc6IjcvMS81MzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMzA0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.37" data-doi="10.1126/science.1251186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">G. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">Evolution of repeated DNA sequences by unequal crossover.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">528</span>–<span class="cit-lpage">535</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1251186</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSmith%26rft.auinit1%253DG.%26rft.volume%253D191%26rft.issue%253D4227%26rft.spage%253D528%26rft.epage%253D535%26rft.atitle%253DEvolution%2Bof%2Brepeated%2BDNA%2Bsequences%2Bby%2Bunequal%2Bcrossover%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1251186%26rft_id%253Dinfo%253Apmid%252F1251186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIxOTEvNDIyNy81MjgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMzA0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.38" data-doi="10.1371/journal.pgen.1000734"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Springer</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ying</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeh</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Maize inbreds exhibit high levels of copy number variation (CNV) and presence/absence variation (PAV) in genome content.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000734</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000734</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSpringer%26rft.auinit1%253DN.%2BM.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253De1000734%26rft.epage%253De1000734%26rft.atitle%253DMaize%2Binbreds%2Bexhibit%2Bhigh%2Blevels%2Bof%2Bcopy%2Bnumber%2Bvariation%2B%2528CNV%2529%2Band%2Bpresence%252Fabsence%2Bvariation%2B%2528PAV%2529%2Bin%2Bgenome%2Bcontent.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000734%26rft_id%253Dinfo%253Apmid%252F19956538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000734&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19956538&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.39" data-doi="10.1101/gr.109165.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swanson-Wagner</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eichten</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumari</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiffin</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Pervasive gene content variation and copy number variation in maize and its undomesticated progenitor.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1689</span>–<span class="cit-lpage">1699</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.109165.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.109165.110%26rft_id%253Dinfo%253Apmid%252F21036921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8xMi8xNjg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzMwNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.40" data-doi="10.1101/gr.2332504"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swigonová</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramakrishna</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Llaca</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Close split of sorghum and maize genome progenitors.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1916</span>–<span class="cit-lpage">1923</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.2332504</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DSwigonova%26rft.auinit1%253DZ.%26rft.volume%253D14%26rft.issue%253D10a%26rft.spage%253D1916%26rft.epage%253D1923%26rft.atitle%253DClose%2BSplit%2Bof%2BSorghum%2Band%2BMaize%2BGenome%2BProgenitors%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.2332504%26rft_id%253Dinfo%253Apmid%252F15466289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjExOiIxNC8xMGEvMTkxNiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOS8zMDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.41" data-doi="10.1534/genetics.116.198911"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saunders</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Structure and origin of the <em>White Cap</em> locus and its role in evolution of grain color in maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">135</span>–<span class="cit-lpage">150</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.198911</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.198911%26rft_id%253Dinfo%253Apmid%252F28159756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA2LzEvMTM1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzMwNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.42" data-doi="10.1093/bioinformatics/btg316"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">libsequence: a C++ class library for evolutionary genetic analysis.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">2325</span>–<span class="cit-lpage">2327</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btg316</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DThornton%26rft.auinit1%253DK.%26rft.volume%253D19%26rft.issue%253D17%26rft.spage%253D2325%26rft.epage%253D2327%26rft.atitle%253Dlibsequence%253A%2Ba%2BC%252B%252B%2Bclass%2Blibrary%2Bfor%2Bevolutionary%2Bgenetic%2Banalysis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg316%26rft_id%253Dinfo%253Apmid%252F14630667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg316&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14630667&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186919200024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.43" data-doi="10.1186/s13059-017-1346-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beissinger</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorant</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The interplay of demography and selection during maize domestication and expansion.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">215</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-017-1346-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D18%26rft.spage%253D215%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-017-1346-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-017-1346-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.44" data-doi="10.1093/molbev/msr311"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weadick</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An improved likelihood ratio test for detecting site-specific functional divergence among clades of protein-coding genes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1300</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msr311</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr311%26rft_id%253Dinfo%253Apmid%252F22319160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msr311&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22319160&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303603300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.45" data-doi="10.1073/pnas.1111670109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wingen</span>  <span class="cit-name-given-names">L. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Münster</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faigl</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deleu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sommer</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Molecular genetic basis of pod corn (<em>Tunicate</em> maize).</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">7115</span>–<span class="cit-lpage">7120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1111670109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1111670109%26rft_id%253Dinfo%253Apmid%252F22517751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA5LzE4LzcxMTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMzA0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.46" data-doi="10.1534/genetics.104.026500"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The HKA test revisited: a maximum likelihood ratio test of the standard neutral model.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">1071</span>–<span class="cit-lpage">1076</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.104.026500</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWright%26rft.auinit1%253DS.%2BI.%26rft.volume%253D168%26rft.issue%253D2%26rft.spage%253D1071%26rft.epage%253D1076%26rft.atitle%253DThe%2BHKA%2BTest%2BRevisited%253A%2BA%2BMaximum-Likelihood-Ratio%2BTest%2Bof%2Bthe%2BStandard%2BNeutral%2BModel%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.026500%26rft_id%253Dinfo%253Apmid%252F15514076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OC8yLzEwNzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMzA0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.9.3049.47" data-doi="10.1126/science.1107891"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bi</span>  <span class="cit-name-given-names">I. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schroeder</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamasaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doebley</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The effects of artificial selection on the maize genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">308</span>: <span class="cit-fpage">1310</span>–<span class="cit-lpage">1314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWright%26rft.auinit1%253DS.%2BI.%26rft.volume%253D308%26rft.issue%253D5726%26rft.spage%253D1310%26rft.epage%253D1314%26rft.atitle%253DThe%2BEffects%2Bof%2BArtificial%2BSelection%2Bon%2Bthe%2BMaize%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1107891%26rft_id%253Dinfo%253Apmid%252F15919994%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDgvNTcyNi8xMzEwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzMwNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.48" data-doi="10.1534/genetics.105.052712"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yandeau-Nelson</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neuffer</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnable</span>  <span class="cit-name-given-names">P. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Unequal sister chromatid and homolog recombination at a tandem duplication of the <em>a1</em> locus in maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">2211</span>–<span class="cit-lpage">2226</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.052712</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.052712%26rft_id%253Dinfo%253Apmid%252F16751673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3My80LzIyMTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMzA0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3049.49" data-doi="10.1093/molbev/msm088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PAML 4: phylogenetic analysis by maximum likelihood.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1586</span>–<span class="cit-lpage">1591</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msm088</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm088%26rft_id%253Dinfo%253Apmid%252F17483113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msm088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17483113&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248848400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/9/3049.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/9/3039" title="BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/9/3059" title="GWAS with Heterogeneous Data: Estimating the Fraction of Phenotypic Variation Mediated by Gene Expression Data" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/9"><span>Volume 8 Issue 9, September 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/9.cover-source.jpg?itok=77yoxn8o" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/9/3049" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jN0Y7ZkNRM0eLFES2qKz0QJpTE3Q-MT8OVDU0155a8U" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/9/3049" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/9/3049" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/9/3049" class="active" data-icon-position="" data-hide-link-title="0">Tandem Duplicate Genes in Maize Are Abundant and Date to Two Distinct Periods of Time</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/490736" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-YYK4p-s0nA5WEI_22Jpcr7Fy6BGbkSP2TlfXqTNZj58" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/9/3049.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/9/3049" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-6BuN-l5vgLPDsqOvBeSc9OVT4OIxG8gQ0ySxhzEyor4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/9/3049" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/9/3049.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490736" id="node490736--2" data-pisa="ggg;8/9/3049" data-pisa-master="ggg;g3.118.200580" data-apath="/ggg/8/9/3049.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Tandem Duplicate Genes in Maize Are Abundant and Date to Two Distinct Periods of Time</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1388-3194" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Thomas J.Y. Kono</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-4639-4119" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alex B. Brohammer</span>, <span class="highwire-citation-author" data-delta="2">Suzanne E. McGaugh</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-8833-3023" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Candice N. Hirsch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3049-3058;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200580 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490736--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Thomas J.Y. Kono </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BJ.Y.%2BKono%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kono%20TJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThomas%2BJ.Y.%2BKono%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1388-3194" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Thomas J.Y. Kono</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alex B. Brohammer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alex%2BB.%2BBrohammer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brohammer%20AB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlex%2BB.%2BBrohammer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4639-4119" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alex B. Brohammer</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Suzanne E. McGaugh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology, Evolution, and Behavior, University of Minnesota, Saint Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Suzanne%2BE.%2BMcGaugh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McGaugh%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASuzanne%2BE.%2BMcGaugh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Candice N. Hirsch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Candice%2BN.%2BHirsch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hirsch%20CN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACandice%2BN.%2BHirsch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8833-3023" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Candice N. Hirsch</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cnhirsch@umn.edu" class="" data-icon-position="" data-hide-link-title="0">cnhirsch@umn.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/9/3049" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/9/3049&amp;title=Tandem%20Duplicate%20Genes%20in%20Maize%20Are%20Abundant%20and%20Date%20to%20Two%20Distinct%20Periods%20of%20Time" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/9/3049&amp;title=Tandem%20Duplicate%20Genes%20in%20Maize%20Are%20Abundant%20and%20Date%20to%20Two%20Distinct%20Periods%20of%20Time" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/9/3049&amp;title=Tandem%20Duplicate%20Genes%20in%20Maize%20Are%20Abundant%20and%20Date%20to%20Two%20Distinct%20Periods%20of%20Time" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/9/3049&amp;text=Tandem%20Duplicate%20Genes%20in%20Maize%20Are%20Abundant%20and%20Date%20to%20Two%20Distinct%20Periods%20of%20Time" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/9/3049&amp;title=Tandem%20Duplicate%20Genes%20in%20Maize%20Are%20Abundant%20and%20Date%20to%20Two%20Distinct%20Periods%20of%20Time" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/9/3049&amp;t=Tandem%20Duplicate%20Genes%20in%20Maize%20Are%20Abundant%20and%20Date%20to%20Two%20Distinct%20Periods%20of%20Time" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/9/3049&amp;title=Tandem%20Duplicate%20Genes%20in%20Maize%20Are%20Abundant%20and%20Date%20to%20Two%20Distinct%20Periods%20of%20Time" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/9/3049&amp;title=Tandem%20Duplicate%20Genes%20in%20Maize%20Are%20Abundant%20and%20Date%20to%20Two%20Distinct%20Periods%20of%20Time" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/490736/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490736" id="node490736--3" data-pisa="ggg;8/9/3049" data-pisa-master="ggg;g3.118.200580" data-apath="/ggg/8/9/3049.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Tandem Duplicate Genes in Maize Are Abundant and Date to Two Distinct Periods of Time</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1388-3194" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Thomas J.Y. Kono</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-4639-4119" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alex B. Brohammer</span>, <span class="highwire-citation-author" data-delta="2">Suzanne E. McGaugh</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-8833-3023" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Candice N. Hirsch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3049-3058;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200580 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490736--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Thomas J.Y. Kono </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BJ.Y.%2BKono%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kono%20TJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThomas%2BJ.Y.%2BKono%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1388-3194" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Thomas J.Y. Kono</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alex B. Brohammer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alex%2BB.%2BBrohammer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brohammer%20AB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlex%2BB.%2BBrohammer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4639-4119" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alex B. Brohammer</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Suzanne E. McGaugh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology, Evolution, and Behavior, University of Minnesota, Saint Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Suzanne%2BE.%2BMcGaugh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McGaugh%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASuzanne%2BE.%2BMcGaugh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Candice N. Hirsch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Candice%2BN.%2BHirsch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hirsch%20CN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACandice%2BN.%2BHirsch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8833-3023" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Candice N. Hirsch</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cnhirsch@umn.edu" class="" data-icon-position="" data-hide-link-title="0">cnhirsch@umn.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3049&amp;rft.epage=3058&amp;rft.atitle=Tandem%20Duplicate%20Genes%20in%20Maize%20Are%20Abundant%20and%20Date%20to%20Two%20Distinct%20Periods%20of%20Time&amp;rft.volume=8&amp;rft.issue=9&amp;rft.date=2018-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Kono%2C+Thomas+J.Y.&amp;rft.au=Brohammer%2C+Alex+B.&amp;rft.au=McGaugh%2C+Suzanne+E.&amp;rft.au=Hirsch%2C+Candice+N."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/490736/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/490736/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/490736/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/490736/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/490736/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/490736/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/490736/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/490736/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/490736/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/490736/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/490736/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/490736/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/9/3049" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/9/3049.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/9/3049.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/9/3049.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/9/3049.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
