<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/9/3019" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/9/3019.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/9/3019.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/9/3019.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/9/3019/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/9/3019.atom" />
<meta name="HW.pisa" content="ggg;8/9/3019" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Bayesian Decision Theory Approach for Genomic Selection" />
<meta name="DC.Identifier" content="10.1534/g3.118.200430" />
<meta name="DC.Date" content="2018-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Villar-Hernández et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Plant and animal breeders are interested in selecting the best individuals from a candidate set for the next breeding cycle. In this paper, we propose a formal method under the Bayesian decision theory framework to tackle the selection problem based on genomic selection (GS) in single- and multi-trait settings. We proposed and tested three univariate loss functions (Kullback-Leibler, KL; Continuous Ranked Probability Score, CRPS; Linear-Linear loss, LinLin) and their corresponding multivariate generalizations (Kullback-Leibler, KL; Energy Score, EnergyS; and the Multivariate Asymmetric Loss Function, MALF). We derived and expressed all the loss functions in terms of heritability and tested them on a real wheat dataset for one cycle of selection and in a simulated selection program. The performance of each univariate loss function was compared with the standard method of selection (Std) that does not use loss functions. We compared the performance in terms of the selection response and the decrease in the population’s genetic variance during recurrent breeding cycles. Results suggest that it is possible to obtain better performance in a long-term breeding program using the single-trait scheme by selecting 30% of the best individuals in each cycle but not by selecting 10% of the best individuals. For the multi-trait approach, results show that the population mean for all traits under consideration had positive gains, even though two of the traits were negatively correlated. The corresponding population variances were not statistically different from the different loss function during the 10th selection cycle. Using the loss function should be a useful criterion when selecting the candidates for selection for the next breeding cycle." />
<meta name="DC.Contributor" content="Bartolo de Jesús Villar-Hernández" />
<meta name="DC.Contributor" content="Sergio Pérez-Elizalde" />
<meta name="DC.Contributor" content="José Crossa" />
<meta name="DC.Contributor" content="Paulino Pérez-Rodríguez" />
<meta name="DC.Contributor" content="Fernando H. Toledo" />
<meta name="DC.Contributor" content="Juan Burgueño" />
<meta name="article:published_time" content="2018-09-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="A Bayesian Decision Theory Approach for Genomic Selection" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Plant and animal breeders are interested in selecting the best individuals from a candidate set for the next breeding cycle. In this paper, we propose a formal method under the Bayesian decision theory framework to tackle the selection problem based on genomic selection (GS) in single- and multi-trait settings. We proposed and tested three univariate loss functions (Kullback-Leibler, KL; Continuous Ranked Probability Score, CRPS; Linear-Linear loss, LinLin) and their corresponding multivariate generalizations (Kullback-Leibler, KL; Energy Score, EnergyS; and the Multivariate Asymmetric Loss Function, MALF). We derived and expressed all the loss functions in terms of heritability and tested them on a real wheat dataset for one cycle of selection and in a simulated selection program. The performance of each univariate loss function was compared with the standard method of selection (Std) that does not use loss functions. We compared the performance in terms of the selection response and the decrease in the population’s genetic variance during recurrent breeding cycles. Results suggest that it is possible to obtain better performance in a long-term breeding program using the single-trait scheme by selecting 30% of the best individuals in each cycle but not by selecting 10% of the best individuals. For the multi-trait approach, results show that the population mean for all traits under consideration had positive gains, even though two of the traits were negatively correlated. The corresponding population variances were not statistically different from the different loss function during the 10&lt;sup&gt;th&lt;/sup&gt; selection cycle. Using the loss function should be a useful criterion when selecting the candidates for selection for the next breeding cycle.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/09/01" />
<meta name="citation_mjid" content="ggg;8/9/3019" />
<meta name="citation_id" content="8/9/3019" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/9/3019" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/9/3019.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/9/3019.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/9/3019.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200430" />
<meta name="citation_pmid" content="30021830" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="19" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="3019" />
<meta name="citation_lastpage" content="3037" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Bartolo de Jesús Villar-Hernández" />
<meta name="citation_author_institution" content="Colegio de Postgraduados, Montecillos, Edo. de México, México" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5862-0056" />
<meta name="citation_author" content="Sergio Pérez-Elizalde" />
<meta name="citation_author_institution" content="Colegio de Postgraduados, Montecillos, Edo. de México, México" />
<meta name="citation_author_email" content="sergiop@colpos.mx" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1605-0817" />
<meta name="citation_author" content="José Crossa" />
<meta name="citation_author_institution" content="Colegio de Postgraduados, Montecillos, Edo. de México, México" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit, Genetic Resources Program, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, México, D.F, México" />
<meta name="citation_author_email" content="sergiop@colpos.mx" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9429-5855" />
<meta name="citation_author" content="Paulino Pérez-Rodríguez" />
<meta name="citation_author_institution" content="Colegio de Postgraduados, Montecillos, Edo. de México, México" />
<meta name="citation_author" content="Fernando H. Toledo" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit, Genetic Resources Program, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, México, D.F, México" />
<meta name="citation_author" content="Juan Burgueño" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit, Genetic Resources Program, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, México, D.F, México" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=D. Akdemir;citation_author=J. I. Sánchez;citation_title=Efficient Breeding by Genomic Mating.;citation_volume=7;citation_year=2016;citation_doi=10.3389/fgene.2016.00210" />
<meta name="citation_reference" content="citation_journal_title=J. Quant. Criminol.;citation_author=R. Berk;citation_title=Asymmetric Loss Functions for Forecasting in Criminal Justice Settings.;citation_pages=107-123;citation_volume=27;citation_year=2011;citation_doi=10.1007/s10940-010-9098-2" />
<meta name="citation_reference" content="citation_title=Selection Methods in Plant Breeding;citation_author=I. Bos;citation_author=P. Caligari;citation_year=2008;citation_doi=10.1007/978-1-4020-6370-1" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. Blondel;citation_author=A. Onogi;citation_author=H. Iwata;citation_author=N. Ueda;citation_title=A Ranking Approach to Genomic Selection.;citation_pages=e0128570;citation_volume=10;citation_year=2018;citation_doi=10.1371/journal.pone.0128570" />
<meta name="citation_reference" content="citation_journal_title=TAG Theor. Appl. Genet. Theor. Angew. Genet.;citation_author=J. R. Brisbane;citation_author=J. P. Gibson;citation_title=Balancing selection response and rate of inbreeding by including genetic relationships in selection decisions.;citation_pages=421-431;citation_volume=91;citation_year=1995;citation_doi=10.1007/BF00222969" />
<meta name="citation_reference" content="citation_journal_title=Crop Breed. Appl. Biotechnol.;citation_author=M. M. Costa;citation_author=A. O. D. Mauro;citation_author=S. H. Unêda-Trevisoli;citation_author=N. H. C. Arriel;citation_author=I. M. Bárbaro;citation_title=Analysis of direct and indirect selection and indices in soybean segregating populations.;citation_pages=47-55;citation_volume=8;citation_year=2008;citation_doi=10.12702/1984-7033.v08n01a07" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=J. Crossa;citation_author=P. Pérez-Rodríguez;citation_author=J. Cuevas;citation_author=O. Montesinos-López;citation_author=D. Jarquín;citation_title=Genomic Selection in Plant Breeding: Methods, Models, and Perspectives.;citation_pages=961-975;citation_volume=22;citation_year=2017;citation_doi=10.1016/j.tplants.2017.08.011" />
<meta name="citation_reference" content="citation_journal_title=Ann. Inst. Stat. Math.;citation_author=A. P. Dawid;citation_title=The geometry of proper scoring rules.;citation_pages=77-93;citation_volume=59;citation_year=2007;citation_doi=10.1007/s10463-006-0099-8" />
<meta name="citation_reference" content="citation_author=G. de los Campos;citation_author=A. Grüneberg;citation_year=2016" />
<meta name="citation_reference" content="de los Campos, G., and Pérez Rodríguez, P., 2015. BGLR: Bayesian Generalized Linear Regression. R package version 1.0.5." />
<meta name="citation_reference" content="citation_journal_abbrev=J Stat Softw;citation_author=D. Eddelbuettel;citation_author=R. Francois;citation_title=Using the R Package crlmm for Genotyping and Copy Number Estimation.;citation_pages=1-32;citation_volume=40;citation_year=2011;citation_issue=12;citation_pmid=22523482;citation_doi=10.18637/jss.v040.i01" />
<meta name="citation_reference" content="citation_title=Introduction to Quantitative Genetics;citation_author=D. S. Falconer;citation_author=T. F. C. Mackay;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=T. Gneiting;citation_author=A. E. Raftery;citation_title=Strictly Proper Scoring Rules, Prediction, and Estimation.;citation_pages=359-378;citation_volume=102;citation_year=2007;citation_doi=10.1198/016214506000001437" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Han;citation_author=J. N. Cameron;citation_author=L. Wang;citation_author=W. D. Beavis;citation_title=The Predicted Cross Value for Genetic Introgression of Multiple Alleles.;citation_volume=205;citation_year=2017;citation_pmid=28122824;citation_doi=10.1534/genetics.116.197095" />
<meta name="citation_reference" content="citation_journal_title=Weather Forecast.;citation_author=H. Hersbach;citation_title=Decomposition of the Continuous Ranked Probability Score for Ensemble Prediction Systems.;citation_pages=559-570;citation_volume=15;citation_year=2000;citation_doi=10.1175/1520-0434(2000)015&amp;lt;0559:DOTCRP&amp;gt;2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Rev. Econ. Stat.;citation_author=I. Komunjer;citation_author=M. T. Owyang;citation_title=Multivariate Forecast Evaluation and Rationality Testing.;citation_pages=1066-1080;citation_volume=94;citation_year=2011;citation_doi=10.1162/REST_a_00215" />
<meta name="citation_reference" content="citation_journal_title=Ann. Math. Stat.;citation_author=S. Kullback;citation_author=R. A. Leibler;citation_title=On Information and Sufficiency.;citation_pages=79-86;citation_volume=22;citation_year=1951;citation_doi=10.1214/aoms/1177729694" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_title=Decision Theory: Principles and Approaches;citation_author=G. Parmigiani;citation_author=L. Inoue;citation_year=2009;citation_doi=10.1002/9780470746684" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2017" />
<meta name="citation_reference" content="citation_author=R. K. Shepherd;citation_author=B. P. Kinghorn;citation_pages=431-438;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Plan. Inference;citation_author=G. J. Székely;citation_author=M. L. Rizzo;citation_title=Energy statistics: A class of statistics based on distances.;citation_pages=1249-1272;citation_volume=143;citation_year=2013;citation_doi=10.1016/j.jspi.2013.03.018" />
<meta name="citation_reference" content="citation_journal_title=Mathematics;citation_author=M. Tsagris;citation_author=C. Beneki;citation_author=H. Hassani;citation_title=On the folded normal distribution.;citation_pages=12-28;citation_volume=2;citation_year=2014;citation_doi=10.3390/math2010012" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and Applied Genetics;citation_author=G. Velu;citation_author=J. Crossa;citation_author=R. P. Singh;citation_author=Y. Hao;citation_author=S. Dreisigacker;citation_title=Genomic prediction for grain zinc and iron concentrations in spring wheat.;citation_pages=1595-1605;citation_volume=129;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=N. Wray;citation_author=M. Goddard;citation_title=Increasing long-term response to selection.;citation_pages=431;citation_volume=26;citation_year=1994;citation_doi=10.1051/gse:19940504" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Bayesian Decision Theory Approach for Genomic Selection" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9.cover-source.jpg" />
<meta name="twitter:description" content="Plant and animal breeders are interested in selecting the best individuals from a candidate set for the next breeding cycle. In this paper, we propose a formal method under the Bayesian decision theory framework to tackle the selection problem based on genomic selection (GS) in single- and multi-trait settings. We proposed and tested three univariate loss functions (Kullback-Leibler, KL; Continuous Ranked Probability Score, CRPS; Linear-Linear loss, LinLin) and their corresponding multivariate generalizations (Kullback-Leibler, KL; Energy Score, EnergyS; and the Multivariate Asymmetric Loss Function, MALF). We derived and expressed all the loss functions in terms of heritability and tested them on a real wheat dataset for one cycle of selection and in a simulated selection program. The performance of each univariate loss function was compared with the standard method of selection (Std) that does not use loss functions. We compared the performance in terms of the selection response and the decrease in the population’s genetic variance during recurrent breeding cycles. Results suggest that it is possible to obtain better performance in a long-term breeding program using the single-trait scheme by selecting 30% of the best individuals in each cycle but not by selecting 10% of the best individuals. For the multi-trait approach, results show that the population mean for all traits under consideration had positive gains, even though two of the traits were negatively correlated. The corresponding population variances were not statistically different from the different loss function during the 10th selection cycle. Using the loss function should be a useful criterion when selecting the candidates for selection for the next breeding cycle." />
<meta name="og-title" property="og:title" content="A Bayesian Decision Theory Approach for Genomic Selection" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/9/3019" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Plant and animal breeders are interested in selecting the best individuals from a candidate set for the next breeding cycle. In this paper, we propose a formal method under the Bayesian decision theory framework to tackle the selection problem based on genomic selection (GS) in single- and multi-trait settings. We proposed and tested three univariate loss functions (Kullback-Leibler, KL; Continuous Ranked Probability Score, CRPS; Linear-Linear loss, LinLin) and their corresponding multivariate generalizations (Kullback-Leibler, KL; Energy Score, EnergyS; and the Multivariate Asymmetric Loss Function, MALF). We derived and expressed all the loss functions in terms of heritability and tested them on a real wheat dataset for one cycle of selection and in a simulated selection program. The performance of each univariate loss function was compared with the standard method of selection (Std) that does not use loss functions. We compared the performance in terms of the selection response and the decrease in the population’s genetic variance during recurrent breeding cycles. Results suggest that it is possible to obtain better performance in a long-term breeding program using the single-trait scheme by selecting 30% of the best individuals in each cycle but not by selecting 10% of the best individuals. For the multi-trait approach, results show that the population mean for all traits under consideration had positive gains, even though two of the traits were negatively correlated. The corresponding population variances were not statistically different from the different loss function during the 10th selection cycle. Using the loss function should be a useful criterion when selecting the candidates for selection for the next breeding cycle." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9.cover-source.jpg" />
    <title>A Bayesian Decision Theory Approach for Genomic Selection | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__L27CoAClcEsxY9a29Y0lWctM-B9DY7Wom4B41-wuQsk__hwC62gz5iT0HOH5FHQ1euI6uc2_KNqFWQY8gJMTjS4M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Y3_-O034l512G4fRXwAd_vx490-2qwxEWkEe1GI5q4M","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490720","apath":"\/ggg\/8\/9\/3019.atom","pisa":"ggg;8\/9\/3019","ac":{"\/ggg\/8\/9\/3019.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/9\/3019.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/9\/3019"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/9\/3019"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/9\/3019":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022XWCSuZri5JkPxgyUzxWNgpSUslk7b3O150S\\\\\\\/sTzpx7K\\\\\\\/\\\\\\\/MppOQMy2lKEhT2tHHlWJPmVBLMD2fOad\\\\\\\/fMYmIbE1RV1CtPxnom9PZKvk6DrNlFCQz5oF9uFaQOtBfn+xWttYqaFJmzhZ815zi1mj9EK2hcRJXQghTpHSwNj2exB7qYGHSxYZsEgozU8DtiUFXNsRT+fUAi9n54KavM\\\\\\\/V4USCePI2uLZ2Xrt1h4jl7G4P3cY+yvfIN1AyJrnmCR4\\\\\\\/FVQdBuLe7iUpPh67\\\\\\\/s\\\\\\\/8jjmPLpnKHWowegNLFvQztkWG2J6+au0xV3aP5DlsrQ\\\\\\\/pcJFu3s4ccRZ6MvEk1GoevP0fxtv8Uxw1tmt\\\\\\\/tVK5R+6e7wYuDl3cIRoQcFrFmMAgX4BhzvfX9zgIZgwA4ycgrKOXHfxKSriSJ\\\\\\\/gseTFl1QmUc8aFdlDNIhX\\\\\\\/g5cmoUsitOdN19EEFeVqk1EA5ERnmk90t10rwOHNaE\\\\\\\/7eGr25+N2jLi2nPJoQqc\\\\\\\/X3qi8RbsST2P7T\\\\\\\/2YgBAAbL3uQo3x\\\\\\\/PSDFCY1e2QEaPQvz1VAuFapbE7YD7HdeSyH64RUIdWQjZDTVsaL\\\\\\\/WPJgBBkPjeunV90zuBChqt19jMzQ+W2o2w\\\\\\\/7z1KAJcuMMFj5n7Y7HZW8RlOu90h0YEeOJCA\\\\\\\/YWh7JIYJSq76awAekf79V7bDcI2o9ZTvHhpfxXwBDTE5BNJDLdls1DzlUqJYeJOpXtchOhr4rtmLOgAGeyhumY2lEtfFtmIOVRcVXMzfldxGVsnfCVs0jWkbZUIxCs97zGqsLQ+ppb9JYQxCnTBpH0vX81DM6Jo++gyKuaVqz6wRjv6ySqkzkbIGAMDlmSOwA3HaC6fckZGYj6ZbYNAPVeLrhiMx6g\\\\\\\/sf8f+8glmiXNt+K8a0W6dKD4neJjXql5KBgLSaQ6UZq5+BBTyiGiK6yqPrUUTY3HptH3A\\\\\\\/KTY5L8oXdYXPH+wkgb0X+0DUxJuAU2+Qg3C2GpOn6\\\\\\\/wOFnMbNJORmyJY0nGlakDIowUClZxB2zxCybQQ0wjew8stKjafQx1Vobem3iAFN6sb4StaSUmDTir4wq+KnMemBqPafWkoPunddg3FJVG3ObtUD3GEexmzu3IToyd6YDljNovY4mWWaod9VR8IFEdWwxJ\\\\\\\/A81bpoe5j5UKgu1aKQur\\\\\\\/GMnucjcRAXAEFaT\\\\\\\/O1O8ZxpuNDhSnYx7dBOqXywjoUEt5djhvGT4fI2SZevEnyZCnM5oPMehJUBTj0gNiiwXTpGLsjz0P9W1f4BzHOdUu8+TACi7jD\\\u0022,\\\u0022iv\\\u0022:\\\u0022PR9+JMpTGQTw+BxvWjYC+g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228e04466e55356aa0c503553a67dd1455271b3783d0955c932f0970799cd26082\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022pZZePtthJrcLR1dDESZGD\\\\\\\/45EDWcvb90mTWoSiK7C2OYRaC2UefEBcq84A3hIp8YClscL70i4s+mm3LB\\\\\\\/snX8OUbzj4NMhZhDGurGH8VnVLQaqwD3P35SSEPn40fy2jS+BzNtd2+XoPWVaQQzH0AxijhrtYAzUDC5GWJZiuGIfERtzQYub7Zum3dDJdkqsZFZc8mVrsQ1swpD0QESyHeiH1LWe\\\\\\\/txd\\\\\\\/B043aDtL5U4hJfcrcJZkmRmJnBGTxUrRRTiiKmdzJVeEYQ6cQc7UrhwyWxTLLuH\\\\\\\/DuCGePirhxO5Kb6ZnM4\\\\\\\/8nvKUq7Qt69iMpKhcigRXOTb+imYqC68RzXNaLeXH17091cW6aTF\\\\\\\/XRBssE37W\\\\\\\/6x9WAsp7xY609+qSoXYhCHC5+4L3QSKeV7sn9asHF6C\\\\\\\/+M8cRtwX3x8msHZX0X\\\\\\\/QkYyKo1+\\\\\\\/R\\\\\\\/upAi9K6xNmTwCpWL3RuJNiYceyTiORRFozjaU7QxubKYW9GkwcoQkYKD5LxibBUiAJmWxRkGYEAJiwO+OfByH9wEW0IPpKkAeCm27nVq4DrBUwYjk1rGoij4jM1IA\\\\\\\/fY+LceUiMLTCsgOewSfTKCMcgPlgd0k\\\\\\\/Za1uoGUeHhgC6gnuN0wG9a3UBZ6jO8gN5\\\\\\\/0U0OwprTDFv5M5MkR1hApOYKSmIjB7d\\\\\\\/\\\\\\\/ImqriCSMEHqvD2vYM7QaZf\\\\\\\/06N12NrP9AYO85SYvnhB2BfD3w2gO9M7Ggmang1Cfi8gm4du2xbp7ZGuljVWKzwf2p9ZBB7x0BfLwENjJDoRekGLgOb1RYwL8pLJtpotYPKTqcrouB\\\\\\\/Kuj5Xb5KwRp6098bgWVYMyXQ0QLRs8dLU2tHT9DzD7f9zBznIiLcHX59PgUN9iIBSThG\\\\\\\/kEYwA\\\\\\\/OyzGTVaUU7rlVHatRhzAeBHhiNlqUtsoUN0uiRPiuyUe88y18UGYbYGADrNao4GCKia6EUWVDXr\\\\\\\/EsRCrRx5gqhOAYfmL54xcmwRCDLb3mzRmeZKS0CzBiNg0H1GJGr\\\\\\\/fotkYbeOapyX6nUn1SmzIcHfQGFhkk0zh0QRB7UodQ+ULGqTKavLD4i7NPpxwQwEtGADW760PgaJWOSyXDcPbvNniO+LhyVs7Jd3FQ8qHXehW4EkE+jDO5NbMGAmCU5tYZw8mroUNnXxHh4EK\\\\\\\/al4vKHYgN7WyD58jflDIaMgiHmNe\\\\\\\/\\\\\\\/0K8+Gx\\\\\\\/zKRUzck8JvDfJph3lzOBiA6eWFstFL53dLQ6Mj1aMUEQapB5owRwC0oB7U6UZf0q8oF7bQnSA6O5gi9Qq9VqH8myXdrPgD06YI0maTds5U77HsDTyhZkcnOT2bAMSh5i+AZsgLzVKqdJKJO\\\\\\\/AxOleMqegXzOUjyrGKNLdPLijXWxzAC3\\\\\\\/PVolX\\\\\\\/nyqm9H6L1SM2lVrcURPtliWnJRYCOpX+\\\\\\\/IRVbl8Pbz\\\\\\\/Dy7LwifuSeBFrj49UeiaXmlng2BINS\\\\\\\/8VB7tGRhUmlxP395eZczLH21+ZV\\\\\\\/ViXZA6nTLHKvPpn0c6khXvTL4eomjc25FitzmhEpuRAXj8gj5+J4An+FZKL1jjanbh8xZr9qD3eXspQRwIUUlCL2wXKXlpad7o1TKuVdz7I1viiPAQIEcHiHYIoEi4LVxaTfn7GtDFGyAKUj5npwcMyHjRUJHHmwjwq4crJO4WLPb8sOvy\\\\\\\/RMJNZGKDhp8F59HHofiidKWoMyWjaOCs+YiEl7eCERAcSPEl31K\\\\\\\/uRFWfaT+29g6qgHoILsJXpCYg9uUGlHi+ZlzvkCTwgCk7pP6VvwRf7f0ZUZSDuDzik4diusdCoyHikTn46BcnqmV5virqYwzqWI1brXYnhPcc7xqhfiPxQLJ0RFVZIlC5\\\\\\\/WZfqLu1qEKtluuGJtS7A2eQdUwHK6KocJKvp\\\\\\\/BTZTPMcAcPdhCMfBjcwjubQ+rEBIpia8SWw9tNQQYmAod8DN2hQywJLJljtAHLHDkG0Hz4+HWszrChbt7tjAn2cxdaki1TBTk2HzIW9rnvZcpIzvwtv+LYT6hZAUPalQGb1fCbP\\\\\\\/Ru\\\\\\\/QdHtup368cEkdylu6YrFP2CdrFjbtPW8PCnqOFJc8qB0joaydzHWb\\\\\\\/Qr12jmGtkqmWBBQ2UvADme+i5M6frdtnRC38vf40hfm9EmPfDU\\\\\\\/hCYE3NH\\\\\\\/F9igaJd4wERxgGwpvIBPw5l1YTQy5FBTf9y0liUGfz3bdQmPJP8n6tGFusWhaur8yjnciTxaIptYoD2igIn2ePR0raw7o\\\\\\\/BMYTu9wHK8oxoUmOL09SRv+kfS1vllRviCsX918RXmDpiU7ZrYdTeqgd2SnwYmCaDlnYSf4yWk\\\\\\\/9aagLGbggVxmqrpsB9WxhupzzoLRoxHJlC0Kd7Z6RmLcEnkLznrNomJJYfzarq3\\\\\\\/sCjPv4vtejoXCWklclCsVQJifmtdtsY7TOv2AQVooycEtQPAUgYJqtTGayelJyu9nF2TqaSPuAeIRqis4cmJRoD8hKK6wP9qSuTP3jkYibp1djcE7dPrn\\\\\\\/y0vVQXk+EgDqqNoNp1GmNGTHMNztMltcBLT4iSC2YbwoR11gsjt8IptWiBiqp6JsKRpd9YsDPO245hlxl6Mn5vGYV6mXxKiTYmEevZ9Ii9BnV+Ajf4\\\\\\\/exS9drUHdOhLxdXY3d8O052c2iUIFldo4WECzPagRVhWLsuprU+4Cr8a19FF1b453OZIsvcC\\\\\\\/E7sS2bhgKbYnM3t7f6dilR6NZOoRv0Zi1IqiUQbN28U1Tcg5OVmNp2gUJ\\\\\\\/0xIoP46NKm3wcz7+QQvbjCcwP0CkzAaQsI0vpsUQ6npwx\\\\\\\/D6Ei5cdbtGv6UJyQpPXGJE4XSrgul9XyjZ5xyzHMY46y28zb7hXdk7s3lki9zqeKg40H9eaiE52tGgzQvfk7qmcRf7GKjhBjdiplNp3yYnrqaMOim5IpQyo+VftOEXEsR0BezyAFr6At\\\\\\\/GE72MvrLRj3qfurNuzz+R2YA42vgrkPeYHUU\\\\\\\/wt5uWZePzXZoaAqZA16N7eZ0HuTaWZWH9UwLy2cqNmMDUep5iUe0qOEftJTU\\\\\\\/4RBJAMh\\\\\\\/iOIScl7xZ51OcNxGNsKjeStFsRqLQQivF5zHGV6B7oUoXmT2Q6JSjS6SuYrWTSOPueOz992iHe1Gii\\\\\\\/43IgzdeZml+wsUMXbq3pyvn2RuIgRfFCMIM5h64Ilm27bBvOMgRFtELFMEduLM\\\\\\\/f7FVR5ulVMzjVjuahGN+YUpWDj\\\\\\\/zV75ALprFoDAD3wlPnDcJZHrO9bPZ6SSScWx5RtMCd+LsjmDx+GALXpHyQtqS4mj1Y4\\\\\\\/MxZhbJ+8GA5tP82C49HDV6ZbRkKLuU\\\\\\\/b801aqGbHJcTYps4la8Je0dXLfFGuNHMqEi\\\\\\\/TVmDAmbLfkP9lwYOt4Oxoc9kLrK4a46PJ+ctYG4iww5ADjmkKboNSMyu\\\\\\\/o5LU5drXenhEcL3gDmGYGc+snjISfOs2fCOMRJrYc7BGDr3cj1QaRPkcqcwQSv\\\\\\\/IResSAmwoj0f14CpSzuq3UBF7ASN8GX2LuVJSoFHkuu3Jnv7hPurC4i2M5fA8AoRXGx9r7GMM5d5s8w2pVBYbKnQSMkJ9ttWQFa\\\\\\\/htikaBsztXJ7zF\\\\\\\/L\\\\\\\/4dsOR\\\\\\\/+uBJ+uN5mUeFKBg6AOwM9u0KAktfhNs9oCR0mqtaNTZThKAsLPZgtcV07O\\\\\\\/AmGCqN4\\\\\\\/IAnTQCDGORn2z\\\\\\\/ETY7SD9EQNDQx5YzKeF0hN9VmHgIGOBxC4yCFIZXVOPPc\\\\\\\/gRNLN4+SnlVtnuJiKmQUGwXk5\\\\\\\/Gep7bobC4p0VytKoBs5N076tNXpmZRp2gE68+yFHVxhZNg1L9wHAy3iiHha9i3qRf4NItOvG4kJ0tk\\\\\\\/p2EUCutsGiK20iQIv++NahyTD0+ctpYKq8DyG6EvmocSo8+MFbPAdfz+dBMANR4h1Pa5RH2q5BFUA7gnVKuT+dS6clR3elDmW8kQBWQDT6g76QEMrtGaSBGRjZKObO8dTOmQA6JyB9DbGSgYcOkl3s0LIoZfxQInF2GBMnkBL0bNSEmrewvRc405D5YTqIk2yHDdXyUSu49MTudC1P3YZuYLgouqxM12cbtkkeNbgeyAA59Z+\\\\\\\/rTu73E0fjJ3VC6+rKBSGPCjW+OJpnVmcBSIvmoxSjvRWVTobuaeeCMLDKWfQYlyHsu9xf5gtK4D4p1XyWe2\\\\\\\/+A\\\\\\\/AnWo4UjQHqXOemEGAvImfzai6a8a7gJa3R1jMHq3kfyt7\\\\\\\/042LDfErGZUTKCWfT9r47ZsbTCLRvb7oil8Fw5NTeG12CMs0jTFTcBUQzlTaEh7BR5HwiRIyX9xzGqHfP8\\\\\\\/+j3WmS4P0euqXEtSt2j0n3Eef+n+AdBi\\\\\\\/ooIkwGsWdJ7BfgMvWNePusvsMOnIlaPRU\\\\\\\/+JtOzQdFFfJOtcrww7vlx3eQS24+ScbseM\\\\\\\/BZtt9MhJi8+ZwNUV7y0OityV+WkSbZ9k4OgaVWCdyQ7Fiw2KpPWPebRtzxthCcFYM2w\\\\\\\/2BziqvS4928d\\\\\\\/kVJeHcgeeiKPY56nOZyAswac8bAbRg\\\\\\\/k3DTHlZHospalo8W5\\\\\\\/3SxKVWunknlj\\\\\\\/zq7Psskj10ldkPHMW3GQOi2Cl6tn0GOqSn1d9v8Q8GGhAnNS2CBea\\\\\\\/Vw6Elw77e3IzbCN95Z1qnsbxO0v6xfo+1sB2qPufI0g2jmt0YyoQRM\\\\\\\/nixqQdK8F6yX6ZNFCpBH3dOr6fc\\\\\\\/DHsXDMbwk5amzeWh4fpu4\\\\\\\/JcUf41nmNijun+OGwSiIa20u7XZ+EAynHshaJIPWyIiMZL0spo7POABUMQycCuOG2fW+AfW185Lrhty73regwr3OPUS2H9P7Oe\\\\\\\/6FXm8wa7TdBKtTqu0OCL67W4xYp+q4aAQsqmW7Y7W45+7s3WLlewxxo8htoOv9FMqelSWBJaNX50W5EOUXnPMhCl2Vr3BeVPJUnSdHyFc+z9RL9+if0Z0mM3WUTgmWR\\\\\\\/rpXRLLg+Gc6UKUmTdOMeoA+WE4GRxPRWkvdmsoylbfdfgsa13tOrvk2r3AafvkLpax+taFXBYJUruTfJia\\\\\\\/KjiVfMoD7glhYSlaUOMMOh3yuLvKDCgM\\\\\\\/vF1CfC9psQzaa\\\\\\\/flNp3Dbs8cLcytwaZyk\\\\\\\/cZa20E4rxJ+yiLu++Orv9\\\\\\\/tE1Ll8yISH4OyiH6LWeaIjsCBgAWOqvRmCiCJhgqADYUvGywKKt0yroaJc\\\\\\\/hGmF\\\\\\\/D3v1x+bIDpXrzoG4JDrv6pSataHpy8QZ4tCcomSW3S666s2Qjqw0RCCIpJprqUKVarQFSatm5p5p0gAZSUMsdAsMZaUBZ2ZOrcFVt73K8pDIsqT+FKCxm8inbuUS+DSSurrJuHgtJVtGrCcCn0+YbMtepuNo660jPbQFbdJawWb6DIJbhMEJcYFgwBKbroM4NaQMXL+4+HLGHfG9B8kiXVuCkadUwCnPnJE44i+2nkmBPk5lIkTpCLTzqjafyJSqzYEzMD0N\\\\\\\/oVqvbKfUanqN4u\\\\\\\/OWP8fqL\\\\\\\/nEwk7pqdJEI46QM6ygXnee3R2\\\\\\\/XIEZFoppGWNQzqYCDxOqMu19iaT8OT\\\\\\\/dj0PXMcofOyljEF7fpUSEfiPWUiYx2BeFe7tdTUqiIlwW\\\\\\\/rwNmpS9k6zrLFs9PEshV1tSVdxyaQt4rG9aqJUgz2LCuLJ4MtPu5+OxWm30koXeByeDOeY\\\\\\\/yTiYmNnjm9QLcC0mpahQNNPMgeqtCgSjEsB0Tp5E3+9ilJYqJPPiutVE19Kc9tmXBGTKxVGFQaVghZ\\\\\\\/hAZ0cHPXQPsJNKWVf2OqFSR+2CrARmhZ\\\\\\\/ntJI9CBTZjfVFgTZmc59FB4WqgCfsTvrYgIHzquFMMzhzlmezeQi8fwWuUCfgsITiYKNg\\\\\\\/i+tLEdBVpP19nle+UZexEVPfWJdGqzRm+40EYFCa445r6EAOST7jOLTQ0D8JNumgQK7\\\\\\\/narKDaqSG3cvi8IwG6qwGu+2dJLVI12R98\\\\\\\/VC4y2hoaSkjVLRR1qY42625dQfleWRHelTcx+JA55TKtkh4dBVISpRzRBEKh4o2zHUu2WJlTqCC3ucfedYTKmmzJzucYh+JPPccMJqdWv1RzhUOBOC6bI6gQFoU+nKjPX95W0WIHgcIs17XvC6mhERDQ4vnrL\\\\\\\/9+0D0dMewIHNkSa\\\\\\\/EIhID2Kb1OaMtivyQ5RwxSMN8FRQQM5qiMmsU1VdqZNQNRfHVAF41XonKl+DXytlNOxRhg05T9ZX32Z7FZceIGFEvquG3MTKaKx99s8uZppTtEuUwJFbnCrGtofHsbKIJd6wc5DKLKRM6hcSFGIFAib8fbkLuaAz0TKXMsCM\\\\\\\/+mby86XfUmUuIr32rurWYdpStozjzn5yPyrrmaYq19w5MrgOUi1UII6fk8Eg9O2Iwp7jjODZow9WfjJIMrgXc6yv0U2ZctCk5DRl6hd2mGmE3N6SliCuAhM8sQur5kbr7JpfSZJJUI+c1ViJc034uNV5UQ9SQE7gBZ6hOxYTLawpWK3uwI5RuDJHEcswWkcqZAGEBIk\\\\\\\/7SXmQgxL0sVCv8jR13TXAJWM38B8TQzSoW3Jda6iG3gL8QO+uH7r4d3v5IqK3BLdd71E9UPAArlkG9LrCOjEWZwHAjq8x0BsOrp4t+SFfwrhkHtSFlXCd5bJW3AlorSaaWUmhcjj\\\\\\\/adI1j7Cm3+8V2zWNKTFOzmW9CrUJn4iDUqJxoXsDPxfnzGY1mNpTinM1vWwOY09lFYIoO3CHrzmr5dI2buTI3wpvk3KkWkN37jzfD80t92rZD9Tfhx4jostPHp\\\\\\\/vgEv5jWBJiBUdgjBZ6Qlc\\\\\\\/TR\\\\\\\/c0oQOtlOnp6KXa2jMnwyMuctezNwn19rCRDeMjXwPzRqS+RXWDeCduE+DnLK99D+nUvuiaOLtBJIlcy\\\\\\\/s76SMXSUTvjxQ8jAcYL5FlnPxK2qMU1aXThahbObf8rT4qNb8K3gcRPmkFOxcDZpeTxeec1RU6kVGaJCrmJ9RPrlTP\\\\\\\/BrFPxwL0Y8Rk5bBQAY2FtcfWcEO61J2Ro7AoBlwMuEEeZd69KtawCTlNdSukEb6yByFEY48JwoxtQdn11CzmOQR1tFjtaagUvUyURRxotzJAPAaKgZ21OYXLW42i8Qt8YRV1kzM\\\\\\\/5VDic4OqlboU1jZtupxpGnPI\\\\\\\/8NpGnBAaW4IdBsjTRoKX34g5Y\\\\\\\/nhFuY8Lr610lITtBEI5YCG998APX6ZctQ0L\\\\\\\/tffP7mNMTdhZ0+cyt9xapjkDJAOlzwrRVsF\\\\\\\/3iTYRMzTsijoHVHflNU9oVXcrEjMrpYzXq\\\\\\\/zhxY5fOaBcAMv7AOx8psTz1XQ3umIfW8jFC3YpaDcwJ\\\\\\\/7GRMi+lmIPZDsrWAg4K9nOtl9QsEioQeMxqTe23mZ1kmvYLtRG0TrCeNLYIuV9D0VcmJsB1rbub9qaGRFnnlWNJuhKOffpEpIAW35Jb1enHU\\\\\\\/nzxt4yy\\\\\\\/5UCsfr7SFjfJUMdqqnk7hxjueSt4+2fVaXJCwEemHeuuhvhz\\\\\\\/VrO3Qi6khJoZ686E6GvYxKA2\\\\\\\/Ee8GwBpfc5PEd22dx8n9o8OzOYBhmjS\\\\\\\/aQP0XiHClAyGG5\\\\\\\/UA2CvxzcYJtkw8S5OSTE\\\\\\\/gD00\\\\\\\/Y1EajqY7RSlknPzU\\\\\\\/RXkvt8LXfXJWWF2ga6rgTIqT2I0BH6E3ExrWhpOedU5AxyItrS8Hv8WKTEeKg77w7rbnyg5RoizLrUAczrLxsO5nXtqzeMmhmPPxuFVoP3iNPjUFTZaKM3yBK5D+IX21I3F9QK09JnMbWEAuiuUVgtj17sWHo21MyCe0lwRGA+jan+vswFuaABO3gTg1xMT7bAgo\\\\\\\/TNXXiA2\\\\\\\/93snIdabtMLThFPV+dLnJfvwpM4iWy\\\\\\\/7wTwhE0\\\\\\\/dc3NLwJGABAHP4XQxG+tfmdSM19xW3jDW8Fzh2JDOBVD2N2dGLD0gAcQYQm8X8dm5gTsiKjVU\\\\\\\/pe9oSxdBovFy1epMnZgtZEDC+OvorEdc77gPKXhEA\\\\\\\/4GdmWMnzD7YUW4hcn9G7oYKY9NeGa9gx\\\\\\\/D\\\\\\\/TU5ABP9k9vV3\\\\\\\/aELsKIbCyxcaWV26velbAzrhv41vfHAs7y+2lSww4lbryf7jtkchpYwCV0vQOyo9M3QJcVmpjSQaV\\\\\\\/TChhDyReEe96TW1YM28CEvSfZgf76cW9lh2C1BD3V1S1zf7Jd8qmouRU4meD8Qe3yXtL0CTwSvIg3lwIOC8\\\\\\\/oWTnyqjz8Z4\\\\\\\/7\\\\\\\/4HMn7FIThvP0xo\\\\\\\/GhVtewLnvz0pYVKKD0SbNDoMwm1bIa9N0qfwCE41HTTffR2bMabOG8Ih\\\\\\\/0N9ikLhKDd9+5h5mmJ1USW2T9L3z7Hj11A5mdvFgiT+qGFwdgoFw+u+XMxlPQOeY1O0TJ+iCUV6jB+Vj7FwXz8MPdbFrXpDkVp2nr21pOjrKY1HpeQMFFUb2\\\\\\\/EJHNG24n0HmAm7LlXL4SRGoLO8hcx0obrwEksgZsLi18QDI\\\\\\\/1mDH5T9sxDvDIOdYx2p6ozHG7ceF6PpftjNyF5h1NDoET+2Ty827VFzVmpmuN2PmmZm1RB+OMPBJymqCOBYnXHjfD07LBSTEeJaQyiNRdY\\\\\\\/F7Ro09lX0+QRXM4JyDPxtmFrdVAbOHVTfFZ85J6Pe45+saisIXmB0+5mh\\\\\\\/WTr4wvAy\\\\\\\/vLqoUdmTRt8+VsrNuiMyA9ph7wEISXulr+klRk7TzppLLeYj9JYMBJHX2dEwYq9qO2Az8ifYIarL80SjSIXQXnYS4EzDM\\\\\\\/0T4WTeeu2rACe38coYPe2qvydcLHiQAAqFSahh8pnDLlcP1OscnEYg32atm1K+P2Moix6KiV8avLRFxzfF4Pu058HzQvHdjOwI8OpMNHD8IdxqoxVCJFTyJrEvLuWASmXLIa9MsoA\\\\\\\/Q8a4YAJIhAjMXY9UN0KjCypC+x\\\\\\\/kcy1upiUbrSLU1MCCqbXYlbgi4eX8M64oVX2ZZ2Bq95KRTXKXKqgrtiQmHXdY148iYZyYVn4LQrO3eWXr+huMIU7tTuxyjDyetNxeKugW8ZwhuiAIifd1P0UDDXEOR99XdwZDAJxQPV1\\\\\\\/71OV863kAWca6GR9zjKmceCF07q9ADQwZZPeLiJpKZj08wMT+3U+8vn7yg6ocCLSe6slbiNZg3v0iNWA5LCt9BSPy8WPiU1VYJWSOraaue\\\\\\\/Dox5kc2lBvRnYdtae5vsWmEI+zmGXxJ8OsNha8l\\\\\\\/XIGOBafyhY1q3qAWTcQlQDxOi8oBNGsXv72P3elrELpY7mvTPjNFNyfLdNUM1S+8a4cVbMYJC0Swr77v5gtEm3B0zNEJny0iwAB1C61rzzTr690JUBolXv6PxgvNS3mHNqE7KOXwvtAB+s6gXjjCaWyXiq+FOrne3SjI1H8JVWIECeTOeF7pUxrM8UUtZM3tLlDH5wRupWly+wITexsmoDp5IYxCnSboPKMNsIBJ84S2D0P3jL7Qgzl1\\\\\\\/J571Jd0RwZrsum71B2WgEhyAOE5eq9wwvZf93zgL6CZw3YUbPJlygsJCUx6NwKF0O5cv+nRq9dJZGG9bNJ0NeVo5JaN2WoNm4z+UZeI2s05SAVjMueLBG0W27bqGGw0V71rfO0kaP7IRoRJzSQlz16GfLLt4aOI45eBKb6o54GFSTU680nLgLsKqSe6ZOe0UZRJjAS1ykrOPPlU2JlDzJpMbkobZxjQiNGvICJDd1W2ystjE0VwBafFkCycLQDpGUtjfmDCDN2S\\\\\\\/TRpRfy2BxqOhWOn2Q3yCt0j1LmlJWmnW1\\\\\\\/FWyN\\\\\\\/+pXceTk9Hq6fKp3wgp+Si1g9r2jUELGCVAxto1Irs\\\\\\\/RPgiSJ8ba3p+cIpSYP2CTVO4\\\\\\\/QO\\\\\\\/LWeJx+IBg+p9FW+Bcgn83Oxe6EcAyJcZqRKM8EAH0AqbZyL0yAotmrx7hwKshE8g0lVgqmWwuRMQ3y3pM+RhyPRakgZPVjNIZP9IE9RCbR\\\\\\\/s0VnWsOWEbFL3IGFVkalSGx9qH4VCBCbzc\\\\\\\/cxGAn9RFlZjLga5\\\\\\\/6t5cD4r8ZhRlLud\\\\\\\/TXTqbW6q7PwQ3lVYT\\\\\\\/V57ztiIwAy22DIv4jA3MGfuuFUNZ79ITKt\\\\\\\/V73HWtMGEBVtvCITopxAzqyl6W9N2vkkehL0gWBKYzC\\\\\\\/xrdYyTt6yXYYI7QMQ+4TvLIbYIXWklP5ApTHIbIavpx9NQSyidi2S7roKyLR\\\\\\\/3uGc0UvfiLndZZBdJm17+cEYQbxBgURRqtn29UKBCqNdCZDv9PfSRTumRtgWi8vMH7YBd3Zy5IAC3cpokM0J\\\\\\\/pySxBNYXPbnzPlfsiImI0IlvpujoACjRStzLOVJKCFeOm18irYextWHOTBowDlP8tHffTM9FdqOX7um5gMfo0syI3YLmhspigiqcIjMLxrV2mlfBFazcY1OP9uUhUVo3xMLpPCzv0H9Lq0CAn\\\\\\\/5\\\\\\\/QQEvHgKbIyE1Z9YBrApFAQ2kpB1bZG6cw+YKiUken4LhwrPcXfReWJ92bvI98wjoEegvzsXFLp2i8D0I6QQXgAcqVajkp31H\\\\\\\/8xgXuSdhgvktnsFM1DnythwNZQw0ZyfMsGGfrtgDHPZi4wls\\\\\\\/gaUHur03+VZjGD8RKLBDp9hpVGFfwb3Qh71j2jb2giL02Dxl5qjPNBaWSTugZwt9w8xvCppkUQuV9ONuQ9qpWg16xzEbr\\\\\\\/WOcVq6g9mhi4e\\\\\\\/t4ZLdvF+HUSofAAVO5kZMFDeh2\\\\\\\/yFW\\\\\\\/kXthg7n8oufBIUH5Gu9pPmyOwnrSDcOxKVWWBCNzOl53WdisRGUxsvxbshS3051ZuoeqUlzPi7Hpt1JXwIV1rV8uE3eW9S2XnskCG4uKrjZ1nHp5ERVShbEwTfOvyfIx5Ayc4UXkHbzATb0Hjpd35cR07OxxSLLZUYod\\\\\\\/HICNG4qe50YFgpipUiEgfcU3p4ZwFAflg9SUVvm4bwfjl+ZveG6905l5QATkGTMaJQtaihNGmjjLhykyphk0y4NN1NNwXqnfHeK9eAkzJiIEpguolgpQPbikeR6z7gnqVpOGUd3eLAI6qPK05PLhMqKusrk\\\\\\\/HKgnJUYp8lr7HFJRz5F1C21b5XsCimiAPyYFukbiRpZzmhpa2kEXlp3YdwH3+IiZA0u7I7PcZ4OSvVp\\\\\\\/sVhe2kX1ps\\\\\\\/EcqIP57t03MY\\\\\\\/0EghgXqQT6bh43wxXXL4H64HF7XE1W3bHbR3gFFz1AbM86SZztpBjenPw6bI6Pk75xDN9ZzH\\\\\\\/SKqX3gTIYTdZzhM4MvwbZZKyUjtop\\\\\\\/x969LbO3m2fIjVikPwyUiMEUZD6mw42z3EFPxvloxpu4rcBtPlyDYm\\\\\\\/N3ryXxcQ0+5lQscZDhUQYbB1LntgVAHnRZwVSyDSRBd0lOP2oAC7uot\\\\\\\/nvzQMF3o+8i8cyAdkA\\\\\\\/YVYjqMVEJ\\\\\\\/GbtUic+mVNybjlMnW2Sfi832ltEuyyol1UylYAspT+kil6qQ6wb3pJLdivVG9i3nQGq3VPkcfZVNunnyn8qvYk+hb0I408ET8igb9pRxJazrW5Qx4Efj182ONQa6WjVT2+2hON4vRtjliwOMVsnxa91Z9md4EmirDRBvg2T2m\\\\\\\/+nsOj4VrkmwtI4H2j1KmqqXLvB1EXp37NPDCRdrZ\\\\\\\/UApPpJoBuepTtpA364e8LV0Dsr97bqpn0BG7kIYhTVf0zUClo1+Uv\\\\\\\/Z77T4CtUd0R+Cmo15zuJmXvh\\\\\\\/KN3Iq6n75K6bju49UiI\\\\\\\/lGyw6ciYo7Kc7ppepCsk4urIO5\\\\\\\/\\\\\\\/2dVaJDwV3UidbASU9aEpaer9nyJSa\\\\\\\/tGev7bWKXGtrNvI8c2nF5OiDHu2y17CXsguYvGAkmVe1CWEWutUuEHQcf8TMvTUaSs2hkHXD4OnNfXLNaOeedzp7mayiVw0KnFTwDYRHx4F849yJkvkftEdJHeQaL1dPg0bmGylM7njxQ7obkKz4kG49jqCOQR8a1HKm\\\\\\\/KUGvZ1KBVjz8HFoXtoUl7lQbsI0HGtu6\\\\\\\/1lBXAGlpnI+WxrJzW9BREIiJE+PEXG92TeUUCAk17sWzoiZM0NbhL80Qa+TQ0NUTiy1j2gk8B8ZyDGt+nJB3\\\\\\\/e7lKa6GFVS0mMdo1J0XEwz4nD5qlnQbJxmrwf5YTLoh5d4Pp9\\\\\\\/PFRvuzW7\\\\\\\/HGpYcR\\\\\\\/oLT7gChPwfYWKs5hP3h9qU4ww5XiBa5sL9oa8R9sVJRC2hQUdjlAX7bAM7REn+ZVVXXXjNRumFM8dfs33+YFBRFoeTkcIqaIXee7iIFdIWcXuJ4oaPCGbXanLZLsRQk8dDB2tamRNt4Oos4\\\\\\\/Ujolg0JHtYWnE\\\\\\\/S4z6Keig7BOxyng5TxbrpB5jlHtmQw+40tuiRlHOPeEM9hMR7YIAyoeYajKUYWmKcT0nCeb9nfX97yqOy0qigeOwfpHW1YCgshiEwCA\\\\\\\/7phNRyL+bRTgJNcRAj35mju0WtKf8FjxY3gT8K6p2RalXiuhK5gV4TffgQK4Em9tYh27J0i4WAy\\\\\\\/MaWv8qclZgQoxA25CIDco\\\\\\\/BAPsBAWoOyHSjlaYffUvGTgmcVbNBHau6k76bgS2gKrLfxkfBGqFqOXO3bC3+AeaVgdQcZaiNXsJ9y8P2L+vYbutiNtnkVTU5TXwiIF3aHqpU7oze8Kbid7WfX\\\\\\\/becUQIRgP8DjOgzWyUocNbwcU4PJTbQuCMhMosdsXHZipx59F6p+qhmF9+JmnF3NNZnoSMK018JytXsnqHZPdsW7sbKgmzO\\\\\\\/zvw8BJMq7scHLL4broDQDIIPX5fEh8aBog91yqdWD4eNR39y8ZBNjVGMGx77wXgIkch2fP9rzNDk96wo4l8xozl5Q8UrRUPI4GbY0DI\\\\\\\/ockeauKlYmITO7B7qzgkLkccjGytijmSgQ76\\\\\\\/uAWA3Qw+xLd6OgvCmqFhiIheLIVWoE\\\\\\\/w4xq3qnEU6uX7qqC7meDW+Geu\\\\\\\/3Y08tg\\\\\\\/F\\\\\\\/lFHpSVOeWqAqyeIuUmtMCYpQMPzGvciPdgDmxtwwcC3P7IOhINiXywhRGohcPBsyn0p831pPJgaxMZ4XbovBGSa+WkNg2VITmM5muSLtgNPC5BVXValjkR8JxHi3g24Kj2YY3pndChTGMJOkzXNl+VgXEF9mLKPWJMvTcNkBa1hU+uWJEa64yiudP5dqtbRDaSW2Q0v+gmWIz2jVjyUSefA\\\\\\\/NP\\\\\\\/Rv73PsqxAf0Hj3Id+iaFzs2RAHEI+zBKxv5QC72y7RYjdgnT6R1n281SFqOa\\\\\\\/xgNpVcb4TRffYbd4EsFBW5vbdWWm2E8oMq2RYPvemX2oAB4yR1DT9m0opiqSMPp8r+A+pi3ZCb2yiGqy9WzXhwd0bEmOwO4QEkMsLDsjdqPlRd1iXY6b6Bj7sXdV+xQ7JwvX8ohiwDsaeP5ZFXwVo\\\\\\\/pbILEXD1\\\\\\\/SeoSXNMX8HcPuVyZ5fzQ16Y3kJsRNnyVyBYzwO01zmxOLtAEnZtl5aY7U1HcW6lcbJIO1prq8CTubG1F8OBflaVXqQH9zkGkeytc7fChJxTXTMgeYuWe6xIZ8tWRZSRiNnCLCPtY6fyrtqqcomxQJITFbHjCz4oYR5MdyHIfhdyWNSPsacDGNitQswzN+2BNkdeqhfomEccdTkaMfNuFh3h+rMl2+4BKPunINFJy9xiolKBCQgB5Lds\\\\\\\/qxdZAr8Kl6BB7K5jnBIclmA183cUyeZI8eV9UPPUsrIs2q6Wy11H13LfMNdg46f90S2KtbxdbJfn9eAYubIT+XBLaE74nnRmnN1n17\\\\\\\/bjmtkasvxN6PwDbbaaYk1oDG7wk+dfNRrRqFSkdcU8UgBQtXvluVGWucxtbmiqPxoo7Q78tzledK3thSNJ\\\\\\\/4cWzEyuZDVx7b7x4Sh9IlAncAkRKuOTV5+wPeb\\\\\\\/xAdYu8VdfGYAnyFb8Uob8iDAntmnXHz7tbAeosz1YjTKHyj+5+enKX9UAifdz214QyF0U5pGAH3g76s4OcOYrMQ0krTiHq2S8SUlduhK4esPDSxm1d0UEhGcykICEbjfiTSDdHY2qWP+LWAjgB0ydeWB0aASZsYxcETJmIB\\\\\\\/bgPjWNnzvrvhSHAiBdp2hyEeEKdeARWK6VmeSYxva2Vx9CadseSr\\\\\\\/J8WhBUdSIIYULvIBTpwIq3c58pPc+UJ8NklF5jtgwdHb4SjidH\\\\\\\/vQtfyEL4GPqVSpSG7S8HLX3ad3YzJd1QRW48if7Mv27SnRYvxw3mrMp2fupt8Q+WZXmTSKQqtLcjvsZeUY10N5uucUoA3YWHSZPt6834SCQ6t7Y1lFlbF6cy68\\\\\\\/ChbQ9NS9D7ZaH3S49lJT8fODI3Kl9rTxcCVmZ793lksCcJh44KMY1lHB2JCPT+Ly6Ho5tPjPffzOeQnlSrHoLaj\\\\\\\/QFR6hs\\\\\\\/8ykPWzrNX9NMfIIRwdZ2RRmd\\\\\\\/rR3gtQO1MF7nPXB95LP6qjy\\\\\\\/So3EV5LNMS4o519oJYyhIRcbpPzD5fvCS8vWTg285vR\\\\\\\/hyRPEzZZ8hDuIJl7kzf4l4PEBWc0TtSFMWESf9iqzaTBy7hDlr9LMhlDBiPD3JqGbdN+fc1eZWRuxJ+xlAqHSRxTqjqD\\\\\\\/RXvzOnNrEB6jQ0jie5XQflU9VNrywjek8ehd8wG7naRYfmCMbZ2FpHrLRq+DCMFUKpB7LJ+3Y3V+S+HCVqIhzx\\\\\\\/TQcMFO071mVmRYnv7mn5OAhbbg5zF\\\\\\\/RIr6CQ8OP2jOSpoD8YDLn2752DsWn7QTu0M263ZoQP6G\\\\\\\/Txv\\\\\\\/LwC0OWN+9qV+YWYQwwHZUMTwfbqkHTdmVrQcHdoITjI5A31\\\\\\\/Y8wyR\\\\\\\/qUg0By\\\\\\\/oH2aMr8kuy2O2N3vC1ju8pdSoKcbMDin7ZVFwcGA6U6lZ\\\\\\\/e51YB+CYiHs3PPjpoHrwrr8dlTdgJRJMBoa0ZL2gIz2vu+pErkW4YY0H+dIfuN9GbUmpJkSEWsfa9jRLlZPG3LnI87izOgx6TrjupDx3FK6T\\\\\\\/IieEe1+YO7d1lyXbGFySm82S3kRPXPL28VyiQs0wlsehuCkcjI+Kbu9sSAeBlrosgwFxtrJzskfkF6psPRNRh93xeWN3mUbsy1VNSUfYtaGWgMzlKsomy3yoeLOVvlJxcYDO6aLG62y9\\\\\\\/OZyDMsEZHOLIMq3gL4Rv2V5piZptoqBzBL264v8yRcQxpw2\\\\\\\/Ett0BG1hYp2PbXpCH3rDIV0pzScBiyuMxpqH0I7KfF9LScE+QH8CPlI5xC6BuZdJYdGjqD4JN+GVFKTHkLMaC95g\\\\\\\/Gs\\\\\\\/yJ62a7DqpDhPoaO2qaQqNVOLU1E2V73lWFfsJJEk6HpA9BEChhRoYiy8girli5H89fE7MnmJkQ8F19a4X3ivivh8H0ysp3lD4AAdf+ahn2cduY5wxa707u7UwqyIA1t7udycZpQS132hxkSQLtt\\\\\\\/po6eo\\\\\\\/efQuC\\\\\\\/gIBkPv\\\\\\\/i29uR6L7YlKqJ+LbdILaF\\\\\\\/gnQERvhDX2wxI5KEWQZBdMfybhMt+wyM9JALp9aFKubt+5Efthm1wsIppBV\\\\\\\/h36Qir4l1c+h98XkuPaEl82PVdy1fnMh3BP2JYB2uI90tKixtS1uwb7\\\\\\\/2e\\\\\\\/xKBqYbIUkQtHnh\\\\\\\/fpNXRKODkjSHyj2By\\\\\\\/DX8XGvSd\\\\\\\/yo2wioM1ut4sRuVsBGLFd\\\\\\\/oKM8n4mnfbkOuq9n\\\\\\\/vJ3zI1BAyPW8ScZEDdMT5WJvokJRkjq3Xby\\\\\\\/ZUZgZTacifrWXiZbJnKk1tYIFPKYgLOcyP+AtPfehiY4kU64bTIoAQ4e8D4\\\\\\\/Df8YHQc1HuXk9uIs4LRG+FIyu1FcWFGaTAxPyZe8WFc0iLXAOyE\\\\\\\/JBCUWlS7Qvw84Eynx2UrAVIC9us8tRnRpElOO6DRr4eGZkvh+ddkpA6Ry7woqvujPajItiauQD1q4buUAoM6N7J1Yj+zi+kuvsDrSr3tbWjBmG0nt\\\\\\\/RATm7obXM1hF3RvnYMfqy4frX2DSuxC9xP6ITqH\\\\\\\/pPeD5ByfJfSCDp+1\\\\\\\/NCAKsRYN+3Yq5DZWSBRJ\\\\\\\/RZQaC1liUvHj9hbS6pDXkn\\\\\\\/uQgiwmUgpa1NvJ5bdrEJEAqAhsU4vcDdbcd3V91piIenAF5xrUo\\\\\\\/K++oN50T8vv04cYIx8qXwyWFIZtPfuPWpHuu8xWxLM0J2ojrPR7cpvFZ4TB\\\\\\\/qBz4j8FFQ+K\\\\\\\/1BhQUb5XM3qCwrFYPc6BXO+JbAttOey+FQ5\\\\\\\/5lGVc481HJxCGsStTSjOkFcAzg0dADm+v+22XYtPk5WRrjrL185+xrz9ZLGBc21LQUi2E7y8K8IZAQRpTJMaiF3XOWwzcQbLuppCCqRfq2ek9lJ+pjSyfSO2mjeu2ykMegbKvF3FI6otJo3bAi0EgTze4AXFX8DM++MFpAOSPNoZmLS9l14xuNmoTG+aKAKbKxmOC9l\\\\\\\/N8cFfZR1\\\\\\\/uMV2zFxFjoctyWVRhvhWThnJGTA190rpyv8NQcBQ02GGm2ziV2UrGWktAHZeUXVl2BY+ff5OY90LyBw4BL0Y9Tjg2S3X0fc4nMxZ1FNd8A7UYDjlp6hXlQ9CN4TTKqXq\\\\\\\/BE9gnmIWPqrijFqVwZIDdNkih9nV4aiB8o9+hW9LfuD1hyHegFOSNwjlCLfyCQ5vnkPs1WNrIm6G+fABhwv8GvglCujjObuouovHC0XJUsW3XAwDBmwX53l9Fy9dgE7Npvl11goVkIkC2vCDfYO9mWhqBrSSb3TXB008AyWZ2iS3B9R\\\\\\\/R70Scg6uyelVK9e3\\\\\\\/n\\\\\\\/0+\\\\\\\/edRLLMADDCNMihZrXucclE+ouVchUV7KoGDUdOGaBEPbitmgQrYdqgLgX66ji+aULbwmNiy8Bb9VzYCGSewPCvg2UAXNjQRD7zsHgPX3neAaBmAWBc9snBDXBGEg5Va5klAc6inDNUDzqVuMcCzrOzhfvvOak8Tx+rVUojp062FcW7jSjj5rQEnjtHdATm95t7qHotR2qLDis8caiWKhjKH3K3e\\\\\\\/fJf7OCI17lp8V2SjASqZ4q0gcXxKrNqV1VF9aTfaragnsG0MZk8com0xswKazIsKRO7NWPze2JUdcM376aYO9PJHc6jq9O4g++Mc1E7FT75vEHjWApAHApavl3llQSlBiiuVoqm3plTLS37GR3h+daHUTD85Zua8DFB2LU1wjj5joeNTfkgFim8OQglxPqCkw8o2dcYrcyPejjJKq54GJRdgjvQY9Y44bwiIdW5eYgOxLRgCMh45istRKX4oBm0dwqMDG0b+C8AJyhPtUPCL08+guCCV7fBSPFigAYkHxpOevKPyCZxSnGanGfkob9crBC8fjSM1BjeZ51htiynJbrWVTIW9QHyVdFx7IHZxwYC1b64VLUUZy17cODkYwuoTrdqEcTgFZAzxzWcKii1tYnjXjW+DYWmgLq2+\\\\\\\/YHboYO6JuzJktomoCbR7vJ1\\\\\\\/oruBVPeWfcYpGS7tOnUJJEwPGcimC9Mtv4qpwMK+ghf9KleAQbr28eDcJYHejH2uM2HEjeI6V9UIyI1ALlT\\\\\\\/NnTmNaL4C5nhhRcdmj00MOJOjTGNXSKkKX6IrWibH6cnhcE3V11CaPfx2LNbEMEY6QAOVtowIbLnnJNKIfUnnopcD\\\\\\\/MhGrxLWltvpbkp1XqWs9HJiomyVKzLx2mImZMQn5ZAOpiO2fPbpFpYDgJPwrl6UJVljK38qlc0LkgYAJfYjQIlpu8bXVS0AWKQJ9mdJyKMbxvxFLWRHCXU84tZtJQRTVDk1\\\\\\\/Hynx4SUuDyjqM+LtSmpIL5LriZP+LpVvgFNVqTB9Gq926oh2X+q476tlUTDLhqqOzxGvqURIgMLfSfz2x5O\\\\\\\/7vCB+YdnGWscq0GAwuZcYFhT8KdpS6PXH9QJZT+P335YWSrp9e5uPhDncagQp4wCUY2qyVHmU9vmZIQJqs849qRLXHBkh0eOBSliuU\\\\\\\/dDPyXkgyAkKd2aGIZF0q5mb7VNvAa8N\\\\\\\/IdSGp56APLs5UskM4Lw4S+SA5XBskQRbbbQv4hp4GJIihgzLUHcQVAIYYXXa5Y4Tz6aU+JLO03NCeftgpYzWf\\\\\\\/9Qda4uaZ2vRX2Rxoa9eX5WCq+PsAsaYIIULU\\\\\\\/LO0VGoawA6kUhPab3zlkMsR1iuTf+Mg74PSBt7tubpvkNv8Or+fTaHgJztGO9nq49ngs2ux11SdFVyMA6NpaoQGeThukpOh\\\\\\\/lllBfaQp3FrXc41wK6DIVde8D2MAeORZ3hL1Qx8XtFvJsNvgs5gQQzFbp0tKbAYmZwY9BI9rcHGtElnDF\\\\\\\/eQXQRxI4XpMGl9QTIHYzet0YGnbbiMZkHnNfD0tdt1akOmtNUAwL4nIZqDzUbHyZ717ryF\\\\\\\/MVPVaG+o8+dTg9Cc9qta6sA3rFV6yyofMgqeXhQkoa11YbOwSqAm8bzLHxDY\\\\\\\/j4EYii70nULLnOYveEIQ6kHw0H3HCG1ZZzl+xGuBzljOImj6xC3CzTJn5cfUN8KL0aOHhJ\\\\\\\/LfgWGUwEXb2v1AcHEKdjSYpHvaohfOcRNoW8IyVCSmwFwZMuXS7zA7LSt47J1jK5i\\\\\\\/RRPVSI\\\\\\\/AOCTRC8Bc862u22CNfokvIaC4330Mq+1QJJVFV0q85TgrskQgWrSDaegi9RyXmK4wjIIdYxeYJx3mwv3zJrYXd4Ch67cfmnmZBWGmD1Tpk6\\\\\\\/AB173NSZU+fBubLVccg73JfVmL3ZexzSah79GNV9KF0HX09s8WEEgv8f9a2LYsYSFhZMpWqSlzhZn5yDmptRwqShTVHMR1Y0vSM9+QXHEqGjtm5fCF19rIVKVHIB0lrHbqzfX2ZtB5tcbarIqcMQc608\\\\\\\/Df\\\\\\\/heefIm+hwP8UQAjMH3+if7z7ObCsSBk85yIuCsAaM\\\\\\\/BJkplaR5OyAN\\\\\\\/f6M0yMwtz\\\\\\\/9c1M1Y3LyQ\\\\\\\/RtL7kosY\\\\\\\/ptSxOSICFjJ4zzgLiafHu+k1\\\\\\\/eP3fzzG38Di0aG0hdcFU1ITqsRdPLG4L+HGIP00Vmjb+VZsrZBM25gt6ODNti8YfkEFkWQBqIMotJwI05oj95Ht3tWmF43M24pN+nLlmxMjs14h3KiwwlTLF9WnSWo9e3SP\\\\\\\/PGJzW1cVxicpwjB6HnYm8V4a3eblO71QKssP9\\\\\\\/w4vYvFUYjUAiUng5qXYSsTYwVvyYh5MAw5trKVap2S9A\\\\\\\/JlavzEYJDnX9eNB0i+FMa+QLVBqBL30sbTQdt80Q2QTaVm4cnC12GP6AxQ95S85T+oC6jatUUI25Ap+Qc2rLHodd+JJwbKb5WnYpWp4uOPkD\\\\\\\/qVbIgIv0E3281ku7gOlvh8+TXUjHBa2+1PXumSzMkbVv49eksnMrC15pv3NhWCbKsefwwVkEPOh7udPUhSGsmrvg\\\\\\\/SO90uhUI5CINYLXMTEH2vcbH70ylxmHZi9g3kY3cRpm2QyzBg4qRU8hAUyxPB5c4VhqCvf5\\\\\\\/xte97RWEbmLKaEsnM+ZDyL5cYRD67PDBoIaEbhPSizUVglLjAhSY7wtdWNAr9HWR8Cdf\\\\\\\/zHjq5xNc87wNvdPanHxw3iDFOoDMsUrOmO59F62No3JnwfldV4E08+ThaOSMF8ZziUuYU2nS9IUakwIzA5sMmwbUHLYZzzDexJZGQxwhq8nEa3mGns+ZHpCHF0JUdcdvUTCZh02uEvJq\\\\\\\/6Ct9Giyl63FYdEsviES9GlDnlVHIhR4nAjL43vDp2cDpsMjHhyVjyoFhdagwoJm2yk6ZFLzeu88mOcGmYm6ItBtjXAKQz0Ehq8F5\\\\\\\/\\\\\\\/2RdU8lD\\\\\\\/Ka1kIjFEdAQK3uh1INenz1jmSR8ZqZAY4aEj+agoN+aeaVdC9zi3pDO1cr5tSMDecDUmir8SV6D7R1UxQEYgw\\\\\\\/62AWS7Rnz5oFYAOZrGY2GFrIHOST\\\\\\\/oBjdGzRUq318s\\\\\\\/t5a8dYxaGpVXQLfS+HW0x\\\\\\\/UVBQpU62OQf3SyMlJxOMyQsByD6o3J7xf4EwbIxg4dxfe\\\\\\\/JVDlV6TTz8PGe9JkxU0SMNrK90EqoASBuVFPEOYYqFEFkJ\\\\\\\/a7kHlXiTWe2PkgALfE2CQNbJnHoe+Up4mP1qNL0fxDihiUOp890ZfUwYCQJtkeZlRAG0\\\\\\\/EhlvupS8upVPry4nsrl+LGfTaLOXbEML4jAuHgkOXNdKmDmpozr1JrVoDbzmkD8ImpZG5awGtQ+sCbpwQltQESAmFunbSXUbSUmX8y2jEiXOB49tGJ7btsQIEJtvLV4NDHXrVKFFPEd+MqLijnXqxlMbFNtUMRjqpMQGG4Pcrl8tlnZyFFCJyor1uMEHFRQ6TYhroZG51eiM8SxkSs3CSTZCdx7vRdSiM1k7eXh3zdO5\\\\\\\/IDXHAJF10aysnAD+AvO5Jqhn1m+PLtyc8HJnsRsMUkvjMubQMqnnWqBD5o\\\\\\\/rPnDdmp+9vd5KNWQ9U0qZoPZRWJC1wt4nNuPynvdZIE533WQCN3VsWBAlA2Qj\\\\\\\/6t+M4cOxnhK9BUXUUTH2lSZgoQiKgRI+8hf76QZ0wud0URvzEx3fPnMCoZn5RZ4VM5VX+gXrKyvXicY9fB4NHcr5DsIjeINeBPsGRti8uaPxvzhD7OZ2IgnbC82VridckZUkMhMHjfI8bI0S0YJAI03UHzN3oXw6vjIjaYspARlWFntDJnJlrB1IXg8C6Mu4np6Li2jvl54SGaH1vC2cKUI0w0MQ\\\\\\\/3+kfq9XHlsxJn44xkFS29lWAkinAoqAnWl70hkaUy6uaOfKRNkvaxGCqpGthSWohDbKca7GaGY6M+y6acE1OhfHddnPsv1S3aNl3GOuw6LQAD0\\\\\\\/ushNjuylnj+GAuzI13rExQWJDrk3Q3xbUsqqbs+FX0ekISC0H28\\\\\\\/29K+0FQfnPqcsi38ABXH22P0LiS5AeMZeOBxD0va3Qm\\\\\\\/uUPZ1XjallbXT+hOmfua8U8uBMRjpQAxcqKvW5TFWZ3X0OuSomQ3\\\\\\\/FN0gwCl4zjPur80Gxqrqb5MIN2YMOMwwdmwLEH+UzzTrmZ2bvvc2NbN8icfhWrwoPJ9wSpLQFfrw6pXfjx1ZbbZLt6s69XNz297HIceAp4V5nJBtHqRUpLUV0MXKsalJQvmheoZZXVjG6SAPP+pEerNAK0KenOE3Eso9GTMmfj70CS3A8+ftdyo6iVjeDHvJ5NaJtbVmU4QCKvag806LdynxkoVKt7TbGnOcTfnJFUiOTETG+aoq3ccU3sNF8JkjXW1zSLV84rSRXNGiFaQ679mkq50eGGyBB6H1JjIitU0IFQCQgnOQgJj944JnNTAFt3wQmoW2ubIcIVo9aroygv7RKaLVSoWWGgFAj6CSc\\\\\\\/Ye\\\\\\\/hAr3kkx70G+Ud6pFwWrkQNRspxrtVYxb70bcRMy2yMaBepPYdE6tdEsbNxO4+vj\\\\\\\/0GXm6wEouqmGNe5ETpJVNFDQXOnspbRk7CiWx8VjLCPWrt87z3HJToULKlNkivFMdZpE73Z+OSsUSmao1Ey1kRNOs2IcjhCo2ay+Vq7z94Xj+fvmZ5SHBn4dgO60Y+WWCdqPI0jSmyoM04JcnEuexWiAs5USONWGrUe\\\\\\\/mcDrQGtnNQrPGnVwa6k\\\\\\\/Qe7zXwcNClHr0383XKd4Yt8Mn5CXdTruPwV2jutm51QmdjKEUPc9vRiCbHeTc++MIl5nS1UUFGYB6VolvLXmofskWgg9VCEpwJBa+jgKtzAhNWBvLunNccglAV9j+QhF5QfI6cz0e5GCAsvr\\\\\\\/e7x1bzFk9IG9iDyvb8I8fJ8MAjs7SudkBz99dQ5QVgQvCpur8FqIskWCE58QXd9OsoLVUoyNbSK3QV0Z3f1c79ooVeaqoi\\\\\\\/u26WCU1eS5Jdvt7Hl8efJdKEcCnNtFRjmtOIOtIUiecuJLOkgOHO08\\\\\\\/heJTew+iS1bMG5bUsSS6bHqx+yd5W7F4rhLijdv2T6GxpTk15jwzUhRK+CwB\\\\\\\/xhxPIgsfpmvnoFQd2nHEcRiMFwS6meN15JVei6V9NT1vpA6stC5+eyxHwxdjlQFePMbi2Lvgqd4l9LHAmcm7xs5nncNinqwgmo+u\\\\\\\/DHoQVROiMBl5dBz3i1VnULEpcO9v6C2uJnIP7EieEhPB6tjpeNUhQB9M9tR6jVtFycP7ulQEGZqjnuZwhGT\\\\\\\/EhhMin94MT+luwM7gitWX8nxyaFaUs8GH0DkTv2R0dEwfSuWPC1gRrwx+mxjroqjteQLnHyuszIVkIr5eCg1QSXv4Fak9MFd3oP0rojvZezp35o6E+BrB9gKPGM\\\\\\\/4n5vKJznePu6kwIUl8wQ8hfFKfvkd1YxRpIhygAShSGGfE7l5jOO22Q9V2AWtMTeY\\\\\\\/MlokqiuGiX6QTQunlmcD+RiT+MXxeZ3sW+xF5QHSu7v29cG0aMyJfMbdVRWiFtCfKFPJooipIvmvcRsfrcS8UPg7DRKcLmhYUUvnd+7AhZrO4lAoVtTTEaQ9HimpT16Tu9QVObZIP3D\\\\\\\/lOjOKCzc6HvkHp54iXqyPjMCAmR9pYycx7AYjuKzSVkd\\\\\\\/6j340KSjZ15LTHojwpvAVOSKEK\\\\\\\/ri2RgtrO5GdeUKIJvY0CDau4mK7N6pq4Nq5H2X6sEa8xEZhaUg1elitloebFQiKgOM6utTO6WYEVPcY+B1I9Tzu1Ag0mQaTg++9PfSnfxrIWA7IoXqQeafl1xg9sinkQpWQGv0u0d70v5UYxFXh8NEDYalaukfrsvXRbED69pOBsSjZCoNARf6l9Gynu94tP2cNvqvtoNRBDSwIbYoXrKd65Ff+ftnePbO7gXJaZ0rA8tXBAY\\\\\\\/oxRMYq\\\\\\\/Po2a6i7oh8H5Yvwr7deSoST\\\\\\\/fQas8uLfVFpX1uHvDubfcXLWq7UNMyWBbRNb93Hi6QHal42+ZW9iAiVfCo2BF77yLj9c7zQWskNqagP\\\\\\\/+HSyaFFD5ucHDoqpPZppzykOcFqB8YQlvWljjoqxmiPQgysLzHnBYWuYTTXWdvXim7IpRYSdB0nzoh7ArKdWmVrtYgMK6FXOt78G6G8tAKKYkvZAoUN\\\\\\\/8qih9ob96eRGWh4ZdlmNy8bX89HqRANKMsa4\\\\\\\/yAgU6V\\\\\\\/eal0IoSgVF8ntX0A0cUQlv3+RYCX2PNzaVNjIFkwp9DUc1okuvBmrzwrJivdupJXo9AJp2de+rkswwF+SbU9q7cRQQ3kX6muR8Zzo7cy017n8wqaOsL+SfpCLmqh\\\\\\\/HW00X7CgKYtD29hKpg4HLhR7kZVnM2r\\\\\\\/OfIJzVdrilgIjw8+LFImfbKL7+IUSEsVlYYrsNwIuulNOHce77xmS9A7oFE1L6nTRvGwfKav\\\\\\\/JbiJkkbGrBUdGExa\\\\\\\/5Gr1ZYraO7HbA6jCxqTBuQP6+6iWujItAfdMxLMm\\\\\\\/dTA38HveY4tzWy\\\\\\\/Km3bf1mcrrmM30Btj\\\\\\\/SGtbA1Wn8tvnJgxLh6tS8HNka6muiGT1+sLcdvQ5fobHspWwJVrAA8avyHVcpasIEZ5XYy4YTVzBR0Ac38ipUas4aJd0sJzwuizLaocUKuJnLPg892EEP7YOges8v0MaGDvUpXheiOgw0lAUe1dW17L1JXiqCmJexzq2C9hp\\\\\\\/quszezLQ32X9obsGwxXne01NHFbUF8xpI\\\\\\\/MgAGkUlw0O+m4ryUToi0y+ridtnxy5knDUxZ21rYEuCVjWV+4BauSUHEF7xGrBiVpo6O+ri2r7Ehe0OYjbstb4fvF8soY9LvdkpwxtXi5nJ1XCO3vnDjggGZJNrsH6CmuuwWmnVGdoavFyIP0\\\\\\\/uBC65LATlP2XC5XFtIdD1K9l8XCxZsNk6chTa3P3o7zWMnQGOF1c1J7hpxZvEeHhDflb+oooOaRGn3J13Ej6p8o7FIQbW7RyYicYCyJPIF+EfjrpF9BlEzxpNID52lE6A7XZlIFCgdMiZB4KOd+82UPkexNVhfxmucTh0aIJtJ71gsoV6Wm\\\\\\\/bYGu6Uyc49Q\\\\\\\/7JwQ6FBaDdKMIEGD9TcmlWSIPiyZoTA5UP\\\\\\\/1JZkXVSvStv844wi4FMPlH5XzzJh83s+FiGl93+4+nNsU\\\\\\\/6nfxoDiIdmZ1HLDxq1a99NMJIBGAxOwuoM0INOXjzkH13NndWdcfGpXEW7+V8JHyTkE+IMgYSBonFVLi8hkbW3ncOkf3SebTpwJAEph9LdUfCAXP6WNR6+10dI6D3ZFu6tF4RFiUNz8q4LCJkSi600orej+dRoGb0ZDnF6WE3FJumO67VI6oMI\\\\\\\/xOxrxfGiCXlF8fSHxr9i1KOmeKb3ZF2ytP96HjrnrPjKhzJdkl+l3yjLkIAuI8PLS2tId00AdtiDg9Rm\\\\\\\/Do5SVL9uF8cvgrr\\\\\\\/BR45ZLCo049Xsf923IEe6Re8v65iS4KP8atqIz4DQaCwZKHujhvzyu3Qpb1g6yjWsMhpfSFUQiS2MreAVJ3RfhecXBMNWrQJvFDv93RsQCpArXVzcmfdk2fLuH6C5H0QaVIIzMCK0nFjUWFuTtklfVziFDMyvuB3ClxGyJjKq\\\\\\\/5x8EP5PfqYHkFsgJAbW3ZI6XFXYMijaKjEXhkFmizsYFEZ7DEyVQPuxbyG\\\\\\\/3CTf2X2AOt4YvvCMigHs\\\\\\\/ddbp2TCFoWJUytRVUXrPjZL32g5PWgXWQZlwqU3SZhfu2qqOmoJbHEiwEkKFGYOw5gyxaAFbdfbxmDpgDE73nRRiyDgghNftNYvn3jn0kyk\\\\\\\/davssk3j5xnB3dLh+viDiD4mfb39y8RHVM0cUtFkIm3dy\\\\\\\/LeBimEwm1+l103HeY6pno6PCTqirb5kxOO\\\\\\\/Dxr3JrCFvwieJYq6bwouCh0SHfxGx0waO6tC\\\\\\\/e2EtrhRqURhS8+GjdGR+c3DdpS3XEWLFs7YLfLr2d2\\\\\\\/ucshsk2CYXDXts13ILVhcNdInlcb91RzJsGkUHkQctESMA3SxWcCr6naynXjDbz9GL+Ry05PBzeuLmuUbWSZ\\\\\\\/fP4lqx9Pl+tSYgDFhC0jyv2S6bX7Qh\\\\\\\/jbzB08tDXS4mtJThF0RljbAcb\\\\\\\/xgSBMiauUGonjNLBKyUSLxGYC+KTcXvrCTQu1Zc1MY60N1x4vRV4iXnp1iuyLnZHTsFJ4+62pzpH5oDETQtTTvtZg+oBeHXOVI8mM3MSmTYu6eD+odLbimWkk1c0Jgyi0rPPBQcCkvsMdiT3dBLc+QKMAb9lwvX5N7BRT\\\\\\\/UaYoaugYgQr205aG2vjMKK++gFwjcOXJcZaRb+K8wIcZcdfn8Yshr8q5VBL458LtgyZFbcRc3sHuKTs18f4FVRjSYVdtE1Z0WtJcd8SrYmkEcKQuiz95lyrw1jg+k5VtKUpY9eVKFX8+z+pUt+z4BpPASUgGjDrXFHSDLaO99uuPVao6AzUH0FcMDEwUX80N\\\\\\\/IgARinhpiNx\\\\\\\/58YWELFArQXyfF3ex0cO3o08lwtXZLuBo2oIwOo1Dkmn9mEvnGwdMsuf1OIFKOkMHvirhzTL732hHdJaJvv7l\\\\\\\/uhWh9iP1ksfy3O6wpREZT+v6HwoySalUFWgV5rSt5RQucsVVF5mOWzqFKUpMlYx6LfHkPw6QIz8zDxSAqeLpa5TKH6yAG4oyw7XyRPPe0WNZw+CMYJxrXlxUO6WzbCn32l3uVmdFQS392ws9jNH2XZysdRumQMNGd4b8YgH\\\\\\\/BHMkEUo9lviQHWIdNyIu+LCNtCSf0030amOfFPCSYl5AzUsvV6jrkF1m9cSsqTWwOWW6O\\\\\\\/OLSoQFUvhgTYI+sdtq6YmJGRr5aO6fx27Lz9fNDcLeMa7P2XvIJtZnbddDzm1Utf2Fo0kM9z8LoSSe2RaXmxmnU493kalkPo6cPeKy5N+uNp2vxKMt\\\\\\\/8+pqClNowa2vnGljSOevSRItPQGMy8lbWB8VBYd7ZjuKGTJzX4CQ5xlxfKkDemXQK07cuiq0Czm7NA3OiPs+2xtIawIyJbfji1JRhdbnKVXszJbVqeRjz31InuVl5XgDDKLJk98p3pOmhJnDjaj4cRMGr5hElRQaBqTYB6yE2tbjBTN0XmHUuAz6tMvmcna4wdxNP7kfzNfiXxrMmKC6rXz4xfIlGFfF5AUl2y4LNVwa\\\\\\\/ETv9m\\\\\\\/+klxySJUNhJgXZrdWZ4ppIxcT4sedvynedp40YdltRxbZN7qyUZdkx6oAyRk60ZFwepDNCaCRUKfxKwOIyNr4ibb8iM5E3ioF\\\\\\\/P1XzXZqLVEDrBnqWqx6gOPU\\\\\\\/6maUGvmiDui5i3EHssTtXPQHpWv+bXnPNzyF+iPyGMqnOCAzPG90d1YVppvVP2tPfCvrATH04\\\\\\\/Q28OzzMXrA\\\\\\\/VtG9aVDbmsxzd8NYLF\\\\\\\/sHEz68558heGOKd34FQ1+IE8THCNjWAKNFE4igd1+EimixGdeNceSIAVW93NLK6O2sbwduYCZ3k\\\\\\\/xJOyRReOCodwUj1u3H3QjLz5p6xx7wd7uS8uj83gqNZLgXLXw8yMFvao3xqYESwt84zIagus59u4jWUjZKro58kKsR6sWVBqFTAnyAWxIJQpBwHD46DsOhUsTsPljYh+BJd21To+Qypo65j\\\\\\\/q\\\\\\\/ExW+LKfC5q02TECyE7b1RB4Pc1zj\\\\\\\/nnixMkVfmlunHE+dSnmvrpQO9wVe0NzbD9whe2XwFD2PLCKq0Jslw8ezLyBz1k0LvfF9J8NGNEE70PGPnaIFyx2aFf1m5pj0mmxq69lf3tEwkGKT1\\\\\\\/8wcCnHJyLF3UGMGVozCYb0jOSohzuHttnhZEx4zpfM9pAYqZghU\\\\\\\/61X33IUzEv35NWJn89isJzq6tqO6BtRjQfqveleO79+q1P3odEyexc3OKSQyX1Gsyvvt+ZRT5dvkFTfot7CFSSoWBekVXFyh+KH5tkfM9kN9UHsQWX0syMCKSudmEMnVXQfrrvOj4R\\\\\\\/ZoOCMk84D3ebGNglHFbql3L9dqlVqrNpkWYBI7bXt67KiTiIb\\\\\\\/ZA4ZlD27OeaKjjmz4NXfefCZlFEmdJlLGAj1XZvYCSDZ1\\\\\\\/zUXyiUOiIpUUXazWH\\\\\\\/I\\\\\\\/zLvTcBBIV4iLxsnlA\\\\\\\/zaUwwIuD6rigMSF4rBTIZppu78LwdjmcAAw2skM6gNt8ZajwHDzdwwut8GPh6n213gMajIiQM3\\\\\\\/xDaQ9htTFtaCN7ql6X4Am2VRCBNmjdj8Pbl+CjPOk37AZH2TTzL9J9wDwgX45hWJBJl1BHAjSNTXUldhdROud7BfWUSvEf62IiMpDrG1U2xHheRVEJcyeLEwpETJ7ai1D1a31YbYXyGlck+Wsj6vRfUTbp5MBYcgWHHnLvSGaeZdOo7GZSxgdbiaRJNqaitGDnzQdziNCPc\\\\\\\/bL70lhvSNG6ahHbRWKlXHCXaOjLT9+uySLz7RGef2j71SHjyolpAp9LQN8eVZFQU3\\\\\\\/2LaN8Smh0VDWq7kknXSmP8LHVDwO+1zz3UHwm0\\\\\\\/qu5JmIBJbMla36Vg\\\\\\\/EWzis\\\\\\\/DrOl8\\\\\\\/l6jlurk6INv8zLxYz6RIS0+S+L+Oc\\\\\\\/kyfp5wFG\\\\\\\/XibQeygK7SiKscz7W\\\\\\\/5JMEGHjaJZM0CKygUXqQ5yMNKK9pp6xdy9akkC7Av84CZMtnPMUk\\\\\\\/6GH82VglLTR7lCubDTkslppX5TinZ4m0LXeps7JuZU2xAyfGBapFfz00ZvcbNhQhYShYk42UTn2vaaFjRwqGT9RcoQw6i6As8WYvluBQwTqFLOxzONFjAp\\\\\\\/uVBOcuYEUQ21UGt+zmw1kMZ\\\\\\\/iKDFxVxgc45HA8w6Qb1zCLPbZXaV3VQ6To3NbufndXZkwTm\\\\\\\/ryQkJ7UjdlCMc8w1t1ANP0hhD+pAklzVhgH88hiqJ1tzNTgdEGQvXGBLkWgIZf9N0jq20TN7w7v\\\\\\\/tPwuFzam2NzyAXwJ45yQS4OiGvdchHihKUZEQsV4WEuqm+gvLFpxyWVTUTj8NYSUYtp+Fpju8aFqsuris1SQYk5jsbgbu0JVmkv9dWG09Tux+4Y0LP9EqLTnNKpMBnUGj3cSLGek+ND1O+JTrPdaaQt6MHjKEyHK2qNV38O0gERwAvOFheUacBNZKXDOfQk25CM6+AK+EvpVMLCgGi\\\\\\\/U2XMTiT1ERK2+5g0wn0lEtp6IxSE8suo52IGOXiFjuxO1PHW8oiI7ttDW6WKuwS1VRWaKJNSVr84bRyHNd6YvAk\\\\\\\/BGoX3BP1QlyncBCiH++iV9WVjiI\\\\\\\/325thpjRCuyBAGIZ2DTieHsdVyL0JOH\\\\\\\/89Bzl\\\\\\\/qWvNAa\\\\\\\/DmJyFviEksIr808earXOTvuVPLHu4q7mUa08Hq8vY5fWwZa+A1Bu3ftWyshuYNlr\\\\\\\/SdWcTfZX\\\\\\\/xINDHBMi3TD9u45MumjBWpzFk3kfHI20yTVTZCY+hgPjq+6rWlrS3I4I9pcN2fT9E3GLPQOMoHrFdRLuoxZpUDxfnzuUpyGc64HdYGMfQjoRBp8HS55mEVEeJtlMdI+iE+zrwQm5r58rDfxFeVfJkexD6Ltplehp4lK6yE1zYloy2Ox1nrdJ0LRAeq2RIsQkZvzziayp6ZVNOOTFNBvR32qdpg4V+FlAWiw==\\\u0022,\\\u0022iv\\\u0022:\\\u00221ECXdsNmAK9QDLKhLeWNpQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223a944af2d5c88bb8ece5d41785828664f021ce18c08599ebe0365873e39a7372\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022tawxaKr\\\\\\\/7emUyv2Jq7e3aOZ5BikOR41nzY9fEjCdjljODw8LrB+C30gmYZJa+TwjvP2pM+XpJ74zaqMNR3gDPurss3NzDWi\\\\\\\/SNrLqafoUdfG\\\\\\\/i2AxT4ETvF36XURtZafP5RlEKfsTzopvhIsMhdaCJvTYolq6GA3hfjfEM521y0ZiQvy2IUhMKna\\\\\\\/MRq86bow7yvzERPAV2qbOSCHEagWnTSWxfxsqZvGZLU7nklBRVeumKZTu2dbKE9AIwitz9eQPfr1GNkQgbXyr1q0CCfDrfDjORFUZuXmywgYtAaKdRmZI1oFGUOzP8tt3ceLBdCoMJEsnBXSckvbUJ9QwftNT0wXjyFNgl\\\\\\\/vqO7bzoqMk4xMCkxRa+fenrM4qsUF4UMvUpqf16rw3ugcXnqKfTbY5VT4T36jxtLOka+rSMGSEr8l9uHJAfBtHhvXbU0npE9QfhA474OkO+rlwPYPjMhRnwajbUBcYKp\\\\\\\/DxqbGdnKMGxJW6oPDzLxTn+PE80jww9g9VPjaGhX3XtMjMicfnR0DIBlRt1YGDektIWr4AwG7Ct8Bt8C04cZv+Tzj74JvFlBbjiTEy4lkaZo8A3UARL+8NgGIf2Dg\\\\\\\/3q4av8Q8RnCDWzns33fNkLuHtzeWBk89jnn6FxLvHjJwPkYYt2osXShSxLoYyreKvrFVM4E0fYjQdY1tH1TdfK1QM9unyZHM1rohvymR3MiwkW8SP8DSDoj8TE6Mu9StCsePiZBMlW2EQurebxain1vw1DCPSCWGVFoCCUvvghUR9bltlNTdz0MTo6V6elifxbn6+i1QzVbx3bwsmK\\\\\\\/KGN7HgU6gnDuwrp+FBvcq50rpXkNQoeox\\\\\\\/i43oDEsT8W258RBtj\\\\\\\/LzpprFOQ21RtGYfIKthI9KIcbj3YzwR39Zjila4dvORkqSjBtup+5qP4oLC1hojoE7ojl9cMBgs0Qmhan8rbZU6M4KaetvwqC8+2fKrzjF39ughnxNj02lXCaeodFjuNnR6Dk6YxTw8WQCTMFqBjfFLnyskZaC1vMeQKB7bC9ooTCuIK\\\\\\\/tgppCzoH+woEl0la44psSgW1SqXtyZFRX0l\\\\\\\/pvcjjt+CTQDVR9Ma0MHgSTcWSM1a08U9z03DAfrHlzcwy5sFmG7wk1LKiCVrGMENwijBv9KkpvY89CzILjBQyCZmJmoP0cvgVMcOTR76G\\\\\\\/HGPRhyIEBPl1Fv43IJjRAEWANuDDzNEjb7JWnaA\\\\\\\/FfkVA1yzeTNFXvWfjEQIVnGs8P1S97Kjpu2IzBJW0eEEx9PJ8X1Ud7N0k+PMksfEd8TRsANEcucsN\\\\\\\/X+MuxzY39u9\\\\\\\/ZJicPTpL\\\\\\\/EYrEnuPOtGNBilQzDsvaqR829opETzSuSUjSE25quM9t6lF59h6QHAHGkBEPRGt\\\\\\\/JcdjBxaevikE4dAFXFsgO18O8sh7fr8o6ByBGXZ\\\\\\\/MRw1kKNIxf3ZmY5HGTz2uC+7yKNFj1t9rNrdHqEyJBSedjxQhWqCawqoYTv2VLUNCWBZnK8J3WvULHGCDJGXLUVHAIJ6Q7olXEjnXr59K4RisnlfVxkLspIk2\\\\\\\/iG0aDnZI2fQVnHJSOk4kVwLpyzve8BoKA1ZSeJiTQ53URdOhC95mGMkJD5GljtyoaUhjGu60m60J00AohTqwVfG+6ObpTe1ywg9iuUYFBJs8\\\\\\\/hoyZ0SWr3v7rrcgkzpGIAFJOw8rvDJyIN8yCPx680PLcskwQEg0A4J3fhdZLh9d5AlLmLfWtQRF4VMvNOodptzUhJufXIyY5RwHRcVsiIHDnF+EO7LgnG7ZgjbcpD9wxWaoRP+sIagOUoUPu9GZJBuiMK\\\\\\\/oICudOgdDgECPKiuexi31M\\\\\\\/lnjZntlHSyDgiVyZ6HZwC9lUWuT8Bt6U\\\\\\\/5Ixo14hnhsGGKeMpMvWaOnKoZrrOfehQ5LcIqYFoHqMbbmiEAJjEAnUy4NODb\\\\\\\/2VfCjwqhVhrtTHFi1xpcgE5MEA+p43WlbrsBIp2YnI52TaLvJNV0BMQGizlq1\\\\\\\/OjnZf3psg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022FOkNrBIYF\\\\\\\/zAcWf2S9f3VQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e2355a145f27033741ec22c369f5201890c237e9b49e2c8a972135a453d816d3\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-490720 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/9/3019" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-j4eHdkjaajlQYkAQtzaGzX9B15EwLlFWAyIR195O178" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/9/3019" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Uedu56sMo2vCOYjtO5cDDALBKFx5zA8fCyYas1k4FFQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/9/3005" title="Maternal Proteins That Are Phosphoregulated upon Egg Activation Include Crucial Factors for Oogenesis, Egg Activation and Embryogenesis in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/9/3039" title="BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Bayesian Decision Theory Approach for Genomic Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5862-0056" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bartolo de Jesús Villar-Hernández</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-1605-0817" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sergio Pérez-Elizalde</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="3">Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author" data-delta="4">Fernando H. Toledo</span> and <span class="highwire-citation-author" data-delta="5">Juan Burgueño</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3019-3037;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200430 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490720" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bartolo de Jesús Villar-Hernández </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Colegio de Postgraduados, Montecillos, Edo. de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bartolo%2Bde%2BJes%C3%BAs%2BVillar-Hern%C3%A1ndez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Villar-Hern%C3%A1ndez%20BD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABartolo%2Bde%2BJes%25C3%25BAs%2BVillar-Hern%25C3%25A1ndez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5862-0056" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bartolo de Jesús Villar-Hernández</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sergio Pérez-Elizalde </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Colegio de Postgraduados, Montecillos, Edo. de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sergio%2BP%C3%A9rez-Elizalde%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Elizalde%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASergio%2BP%25C3%25A9rez-Elizalde%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1605-0817" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sergio Pérez-Elizalde</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sergiop@colpos.mx" class="" data-icon-position="" data-hide-link-title="0">sergiop@colpos.mx</a>
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Colegio de Postgraduados, Montecillos, Edo. de México, México</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Biometrics and Statistics Unit, Genetic Resources Program, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, México, D.F, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sergiop@colpos.mx" class="" data-icon-position="" data-hide-link-title="0">sergiop@colpos.mx</a>
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Colegio de Postgraduados, Montecillos, Edo. de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Fernando H. Toledo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Biometrics and Statistics Unit, Genetic Resources Program, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, México, D.F, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BH.%2BToledo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Toledo%20FH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFernando%2BH.%2BToledo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Biometrics and Statistics Unit, Genetic Resources Program, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, México, D.F, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/9/3019" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490720" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:490720/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/9/3019.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490720" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:490720/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/9/3019.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490720" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:490720/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/9/3019.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490720" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:490720/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Plant and animal breeders are interested in selecting the best individuals from a candidate set for the next breeding cycle. In this paper, we propose a formal method under the Bayesian decision theory framework to tackle the selection problem based on genomic selection (GS) in single- and multi-trait settings. We proposed and tested three univariate loss functions (Kullback-Leibler, KL; Continuous Ranked Probability Score, CRPS; Linear-Linear loss, LinLin) and their corresponding multivariate generalizations (Kullback-Leibler, KL; Energy Score, EnergyS; and the Multivariate Asymmetric Loss Function, MALF). We derived and expressed all the loss functions in terms of heritability and tested them on a real wheat dataset for one cycle of selection and in a simulated selection program. The performance of each univariate loss function was compared with the standard method of selection (Std) that does not use loss functions. We compared the performance in terms of the selection response and the decrease in the population’s genetic variance during recurrent breeding cycles. Results suggest that it is possible to obtain better performance in a long-term breeding program using the single-trait scheme by selecting 30% of the best individuals in each cycle but not by selecting 10% of the best individuals. For the multi-trait approach, results show that the population mean for all traits under consideration had positive gains, even though two of the traits were negatively correlated. The corresponding population variances were not statistically different from the different loss function during the 10<sup>th</sup> selection cycle. Using the loss function should be a useful criterion when selecting the candidates for selection for the next breeding cycle.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/bayesian-decision-theory" class="hw-term hw-article-keyword hw-article-keyword-bayesian-decision-theory" rel="nofollow">Bayesian Decision Theory</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic Selection</a></li><li class="kwd"><a href="/keyword/loss-function-1" class="hw-term hw-article-keyword hw-article-keyword-loss-function" rel="nofollow">Loss Function</a></li><li class="kwd"><a href="/keyword/simulation-scenarios" class="hw-term hw-article-keyword hw-article-keyword-simulation-scenarios" rel="nofollow">Simulation Scenarios</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">The breeding process consists of selecting individuals for crossing, each of which has specific traits of interest. Crossing allows alleles to be exchanged between the parents so that diverse individuals are observed in the progeny of future generations. Conventional phenotypic and pedigree breeding is based on truncated selection of the best performing parents, which are intermated to form the next improved population. Candidates are selected based on the breeding value (BV) and genetic merits of single or multiple traits, and decisions are made based on their phenotypic performance in progeny field trials, greenhouse trials, laboratories, etc. Meanwhile, in GS-assisted breeding, all molecular markers are used to predict the BV of the candidates for selection in a population that has been genotyped but not phenotyped (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Meuwissen <em>et al.</em>, 2001</a>). The main advantages of GS over phenotypic and pedigree-based selection methods are that GS reduces the cost per cycle and increases time efficiency for variety development by shortening the breeding cycle (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Crossa <em>et al.</em>, 2017</a>).</p><p id="p-4">In breeding, decisions on which individuals to select and intermate to form the improved population are crucial. For a single trait, the selection differential (S) is the difference between the mean of the selected parents (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-1.gif"/></noscript></span></span>) and the mean of the base population (<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-2.gif"/></noscript></span></span>), whereas the selection response (<em>R</em>) is the difference between the mean of the offspring (<span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-3.gif"/></noscript></span></span>) of the selected parents and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-4.gif"/></noscript></span></span>Therefore, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-5.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-6.gif"/></noscript></span></span> is the narrow-sense heritability of the trait of interest (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bos and Caligari 2008</a>). Under <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-7.gif"/></noscript></span></span> the mean of the offspring of the selected parents tends to the mean of the selected parents and, therefore, <em>R</em>∼<em>S</em>, whereas for small <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-8.gif"/></noscript></span></span>, the mean of the offspring of the selected parents tends to the mean of the base population and thus <em>R</em>&lt;&lt;<em>S</em>. Therefore, the selection response enables breeders to estimate the expected selection progress before carrying it out (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Costa <em>et al.</em>, 2008</a>). When selection is applied to improve the economic value and genetic merits of a crop, plant breeding programs are applied to several traits simultaneously and not just to one trait (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Falconer and Mackay 1996</a>).</p><p id="p-5">Plant breeding decisions regarding which individuals to select for the next cycle have associated risks. The decisions will be associated with the losses (or gains) in the response to selection (genetic gains); ideally, we want to maximize the genetic gains with minimum risk. Minimizing the risk and maximizing the genetic gains when selecting the parents may be achieved by different methods. Most of the work for selecting the best parents to cross is by truncation. As a result, the subset of individuals from the base population that will be parents for the next selection cycle follows a truncated distribution. However, another way to select parents for crossing is to optimize a subset of parents using pedigree based on co-ancestry information and additive genetic values (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Shepherd and Kinghorn 1998</a>). An approach that seeks to minimize inbreeding and co-ancestry in the context of a quadratic optimization problem was proposed by <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Wray and Goddard (1994)</a> and <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brisbane and Gibson (1995)</a>.</p><p id="p-6">Recently, <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akdemir and Sánchez (2016)</a> developed a genetic algorithm that optimizes genomic mating between parents under GS; they developed a measure called the risk of a mating plan and used it to minimize a function that combines measures of inbreeding with the risk function. Another decision approach applied to optimize the selection of a set of donor parents for the introgression of alleles to recipient individuals was presented by <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Han <em>et al.</em> (2017)</a>; the authors proposed framing this introgression process as an algorithm that can be mathematically formulated and optimized.</p><p id="p-7">As mentioned, truncated selection and optimizing genetic mating systems are decision problems that are made under uncertainty. Another approach for selecting the candidates in a single-trait or multi-trait setting is to assess the cost of that decision using a loss function (LF), which ensures the highest profitability given the user’s preferences, that is, by maximizing R. The LF may take into account, among other factors, the correlation between traits, and the mean and variance of those traits, or it may focus on minimizing the distance between the mean of the (theoretical) parental distribution and the mean of the candidates (that is, maximizing <em>R</em>). Decisions made during GS are particularly important because, for a few selection cycles, the only tool available to make selections is based only on the predicted BV of the candidates for selection without observing them in field trials, etc. Genomic prediction computes the BV of the unobserved (genotyped) individuals (testing population) using the phenotypic and genotypic data of their parents, ranks the best predicted BV and selects the top, say, 10%. In a recent study, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Blondel <em>et al.</em> (2018)</a> proposed ranking lines according to their BV by giving the preference to regression models that assign a high rank to lines with high BV.</p><p id="p-8">In a Bayesian framework, the problem of selecting the best parents is concerned with minimizing the posterior expected loss. Suppose that some unknown parameter indexes the statistical model of the observable trait of interest, and the uncertainty about the parameter is represented by a prior distribution. After the data are observed, the uncertainty about model parameters is formally quantified in the posterior predictive distribution of the BV. However, in phenotypic and genomic selection, decisions are made by regarding just the ranking of the candidates predicted for each trait and disregarding the uncertainty attached to the prediction. Here is where LFs play a role in conventional phenotypic or GS, because the breeder’s decisions should be based on both criteria, the whole posterior predictive distributions of BV and the loss attached to the selection of a parent or a set of parents for upcoming selection cycles.</p><p id="p-9">Several divergences between the distributions of candidates and the parents’ may be used as LF. As previously pointed out, those candidates whose distributions are closer to the theoretical parental distribution will have the lowest loss (higher <em>R</em>), and the decision is to advance those lines in a breeding program because they reach the desired mean and keep the genetic variance (high <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-9.gif"/></noscript></span></span>). For example, the Kullback-Leibler (KL) (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kullback and Leibler 1951</a>) LF measures how two probability distributions diverge, and when the two distributions match, the KL is zero. Thus, given the available phenotypic and genomic information, selecting candidates using this approach should be better than using solely the point estimates of BV because it takes into account all the associated uncertainty, namely, the uncertainty due to the estimation of model parameters and the prediction of the BV. Another LF that could be used when selecting candidates in conventional and GS breeding is the Continuous Ranked Probability Score (CRPS) (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gneiting and Raftery 2007</a>) that reflects the distance between the cumulative distributions of two random variables. LFs, KL and CRPS are measures of the divergence between the distributions of the parents (theoretical) and the candidates, and could be used in both single-trait and multi-trait settings; for instance, the Energy distance or Energy Score (EnergyS) is the multivariate CRPS defined in terms of distributions of random vectors (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Székely and Rizzo 2013</a>).</p><p id="p-10">The KL and EnergyS methods are symmetric LFs, given that the penalties due to selection of equidistant distributions located on the right and on the left of the target theoretical distribution are equal. However, in plant breeding, distributions with high density on the right (or left, depending on the trait of interest) of the target distribution [individuals with superior values of the trait(s)] should be less penalized because they are the individuals of interest to the breeder (in case the trait to be selected should increase). Thus, asymmetric LFs should play an important role in plant breeding because those individuals with BV distributed on the right and close to the parents’ distribution should be less penalized. A simple asymmetric LF is the linear-linear (LinLin) function (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Berk, 2011</a>), which is linear on both sides of the target distribution but with different slopes. A generalization of the LinLin function in a multivariate setting is the Multivariate Asymmetric LF (MALF) (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Komunjer and Owyang 2011</a>), which is a function of the distance between predicted BV of individuals’ and parents’ means and a parameter that controls the degree of asymmetry.</p><p id="p-11">Based on the previous considerations, in this study we introduce uncertainty elements related to the decisions made when selecting parents for GS-assisted breeding by using LFs where the space of decisions (<em>i.e.</em>, candidates for selection) is the subset of individuals whose BVs exceed some level of truncation on the base population. The main objective of this study was to show how Bayesian decision theory can be used to select parents for the next breeding cycle by minimizing the expected posterior divergence between the distribution of the candidates and the parental distribution and therefore maximizing the expected response to selection (<em>R</em>) given the phenotypic, genotypic and genomic information at hand. The specific objectives of this study are: (1) to present a summary of the Bayesian decision theory framework adapted to the problem of selecting the parents for the next GS selection cycle; (2) to describe various univariate and their multivariate generalizations LF; and (3) to compare the ranking of the top 10% and 30% of the candidate lines selected using single-trait and multi-trait LF <em>vs.</em> not using LFs from simulated and real wheat data. The connection between the LFs and the genetic gains is shown when deriving the LFs as functions of response to selection (<em>R</em>), selection differential (<em>S</em>), intensity of selection (<em>i</em>), and heritability (<span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-10.gif"/></noscript></span></span>).</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>A decision problem</h3><p id="p-12">A decision problem is defined in terms of an outcome space, an action space, and an LF (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dawid 2007</a>). Broadly defined, the goal of decision theory is to help choose among actions whose consequences cannot be completely anticipated, typically because they depend on some future or unknown state of the world; in our framework, the actions are the choices of the offspring that will be the parents in the next breeding cycle, which depend on the unknown BVs to be predicted. Expected loss theory handles this choice by assigning a quantitative loss to each decision, a probability to each state of the world, and then selecting an action that minimizes the expected value of the resulting loss. This idea has proven to be a widely applicable description of rational behavior (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Parmigiani and Inoue 2009</a>).</p><p id="p-13">Let A be the action space, and <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-11.gif"/></noscript></span></span> be an action. No action can be taken without potential losses. The LF is denoted by <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-12.gif"/></noscript></span></span> and represents the associated penalty when a decision maker takes action <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-13.gif"/></noscript></span></span>, and the real state of nature is <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-14.gif"/></noscript></span></span>. In a Bayesian statistical framework, the expected loss is the expectation of the LF with respect to the posterior measure, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-15.gif"/></noscript></span></span> When comparing two actions, <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-16.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-17.gif"/></noscript></span></span>, after data X have been observed, the preferred action is the one for which the posterior expected loss is smaller. An <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-18.gif"/></noscript></span></span>the action that minimizes the posterior expected loss is called a Bayes action.</p></div><div id="sec-3" class="subsection"><h3>Loss functions in GS</h3><p id="p-14">In our particular case, the action space comprises the set of candidate lines for selection; we expect to assign a loss given our preferences, and then select the best individuals based on the expected loss. To explain the idea of selection under the decision theory approach, we first focused on a single quantitative trait where three LFs are proposed. In addition, we focused on traits whose phenotypic values we wanted to increase in successive selection cycles. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> illustrates the idea of selection under LFs for a univariate trait; <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1a</a> depicts the distribution of the base population that is truncated at <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-19.gif"/></noscript></span></span>, meaning that the selected individuals will have mean <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-20.gif"/></noscript></span></span>. Parameters <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-21.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-22.gif"/></noscript></span></span> are essential for defining the LF. <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1b</a> depicts the three LFs we will describe later; these three functions are minimized when the means of selected candidates is close to <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-23.gif"/></noscript></span></span>, which reflects breeders’ preferences for those individuals with higher response to selection. The gray lines show the theoretical distribution of the trait of interest for individuals according to the breeder’s preference, as well as three possible distributions of candidates. The distribution of the selected candidate is in red because its mean is the closest to <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-24.gif"/></noscript></span></span> (<em>i.e.</em>, higher response to selection) with almost the same variance of the base population; that is, under normality, the divergence between the truncated distribution and the distribution of the selected candidate is the lowest. This idea can be generalized to a multi-trait setting. Below we describe three LFs that we used when selecting candidates to be parents of the next generation.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A) Classic idea of selection by truncation at . Loss functions need to be minimized at  in order to favor lines with high response to selection. Losses were standardized by subtracting the minimum value for representation. Loss functions Kullback-Leibler (KL) and Continuous Ranked Probability Score (CRPS) are symmetric on both sides of target , while LinLin loss is asymmetric. B) The solid line in black represents the base population, the solid gray line corresponds to the truncated distribution after censoring at  representing the breeder&#x2019;s preferences. Dashed lines are theoretical distributions of three possible candidates. The candidates&#x2019; distribution with mean close to the theoretical  (the greater ) and variance similar to that of the parent distribution has the minimum loss." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-388110402" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A) Classic idea of selection by truncation at &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-25&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. Loss functions need to be minimized at &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-26&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; in order to favor lines with high response to selection. Losses were standardized by subtracting the minimum value for representation. Loss functions Kullback-Leibler (KL) and Continuous Ranked Probability Score (CRPS) are symmetric on both sides of target &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-27&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, while LinLin loss is asymmetric. B) The solid line in black represents the base population, the solid gray line corresponds to the truncated distribution after censoring at &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-28&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; representing the breeder&#x2019;s preferences. Dashed lines are theoretical distributions of three possible candidates. The candidates&#x2019; distribution with mean close to the theoretical &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-29&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (the greater &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-30&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) and variance similar to that of the parent distribution has the minimum loss.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/3019/F1.medium.gif" width="440" height="183"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/9/3019/F1.medium.gif" width="440" height="183"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490757" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-15" class="first-child">A) Classic idea of selection by truncation at <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F1/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F1/embed/mml-math-25.gif"/></noscript></span></span>. Loss functions need to be minimized at <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F1/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F1/embed/mml-math-26.gif"/></noscript></span></span> in order to favor lines with high response to selection. Losses were standardized by subtracting the minimum value for representation. Loss functions Kullback-Leibler (KL) and Continuous Ranked Probability Score (CRPS) are symmetric on both sides of target <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F1/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F1/embed/mml-math-27.gif"/></noscript></span></span>, while LinLin loss is asymmetric. B) The solid line in black represents the base population, the solid gray line corresponds to the truncated distribution after censoring at <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F1/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F1/embed/mml-math-28.gif"/></noscript></span></span> representing the breeder’s preferences. Dashed lines are theoretical distributions of three possible candidates. The candidates’ distribution with mean close to the theoretical <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F1/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F1/embed/mml-math-29.gif"/></noscript></span></span> (the greater <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F1/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F1/embed/mml-math-30.gif"/></noscript></span></span>) and variance similar to that of the parent distribution has the minimum loss.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Univariate Kullback-Leibler (KL) loss function</h3><p id="p-16">The KL divergence reflects how different two probability distributions are. The expected loss of information will be minimal if the two distributions approach each other, and it will be zero if they are identical. Therefore, we can measure how close the distribution of the candidates’ BVs is to the hypothetical parental distribution.</p><p id="p-17">Let <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-31.gif"/></noscript></span></span>be a random variable that represents the phenotypic value of a trait of interest in the base population and <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-32.gif"/></noscript></span></span>; if selection is by truncation (<em>i.e.</em>, those realizations of Y falling above a threshold value <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-33.gif"/></noscript></span></span> will be the parents of the next breeding cycle), then <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-34.gif"/></noscript></span></span> is the truncated random variable resulting from left truncation at <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-35.gif"/></noscript></span></span>, which represent the phenotypic values of the selected parents. From properties of normal distribution, <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-36.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-37.gif"/></noscript></span></span> denotes a truncated normal distribution with parameters <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-38.gif"/></noscript></span></span> and b; but for simplicity, let’s denote <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-39.gif"/></noscript></span></span>. The probability density function of <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-40.gif"/></noscript></span></span> is <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-41.gif"/></noscript></span></span>, for <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-42.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-43.gif"/></noscript></span></span>. Let <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-44.gif"/></noscript></span></span> be a random variable that represents the candidates’ phenotypic values. If we assume normality, then <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-45.gif"/></noscript></span></span>. Note that, in order to keep the heritability of the trait, we are assuming that Y and <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-46.gif"/></noscript></span></span> have different means but the same variance. A measure of the divergence between the distributions of <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-47.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-48.gif"/></noscript></span></span> is given by the Kullback-Leibler divergence,<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-49.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>which we want to minimize in the breeding selection framework.</p><p id="p-18">After evaluating the integral in (1) and simplifying (see Appendix A), the KL loss is given by:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-50.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-51.gif"/></noscript></span></span> is the mean of the truncated normal distribution and equal to <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-52.gif"/></noscript></span></span> and φ and Φ denote the <em>pdf (probability density function)</em> and <em>cdf (cumulative distribution function)</em> of a standard Gaussian random variable, respectively. By reordering some terms, <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">Equation (2)</a> can be alternatively formulated as:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-53.gif"/></noscript></span><span class="disp-formula-label">(3a)</span></span><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-54.gif"/></noscript></span><span class="disp-formula-label">(3b)</span></span>where <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-55.gif"/></noscript></span></span> is the selection differential, <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-56.gif"/></noscript></span></span> is the selection response, and <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-57.gif"/></noscript></span></span> is the selection intensity. The second term on the right-hand side of <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">Equation (3a)</a> implies that when <em>R</em> approaches <em>S</em> (increased genetic gains) and the selection intensity increases, the divergence between the truncated distribution and the candidate’s distribution decreases. That is, <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-58.gif"/></noscript></span></span> depends on the intensity of selection and is a decreasing function of <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-59.gif"/></noscript></span></span> (3b).</p></div><div id="sec-5" class="subsection"><h3>Multivariate Kullback-Leibler (KL) loss function</h3><p id="p-19">In GS there are usually many traits of interest for which multivariate normality is assumed. As in the single-trait case, we can compute the divergence between two multivariate distributions and capture all the uncertainty in multivariate BVs. Then, assuming multivariate normality for phenotypic values of the traits under consideration, the KL divergence accounts for the association between traits, which improves and simplifies the decision procedure, given that all traits share some degree of association and are rarely independent.</p><p id="p-20">Let <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-60.gif"/></noscript></span></span> be the random vector of the phenotypic values of t traits of interest in the base population. We assume that <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-61.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-62.gif"/></noscript></span></span> is the vector of means in the parental population and P is a positive definitive variance-covariance matrix that captures the association between traits. Now, let <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-63.gif"/></noscript></span></span> be the random vector resulting from left truncation at <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-64.gif"/></noscript></span></span>. Then <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-65.gif"/></noscript></span></span>, where <em>TMVN</em> denotes the Truncated Multivariate Normal distribution. The <em>pdf</em> of <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-66.gif"/></noscript></span></span> is<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-67.gif"/></noscript></span></span><span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-68.gif"/></noscript></span></span> where the normalization factor is <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-69.gif"/></noscript></span></span>. Now, let <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-70.gif"/></noscript></span></span> be the random vector of the phenotypic values of a candidate, and assume that <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-71.gif"/></noscript></span></span>. Let’s define <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-72.gif"/></noscript></span></span> as the differential selection vector and <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-73.gif"/></noscript></span></span> as the vector of the selection response, where G is the genotypic covariance matrix. As in the univariate approach, the multivariate LF based on the multivariate KL divergence is given by (see Appendix A):<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-74.gif"/></noscript></span><span class="disp-formula-label">(4a)</span></span><span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-75.gif"/></noscript></span><span class="disp-formula-label">(4b)</span></span>Thus, as the phenotypic and genotypic covariance matrices tend to explain the same amount of variation and association between traits, <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-76.gif"/></noscript></span></span>, the divergence between the parents’ distribution and the candidates’ distribution tends to decrease. Note that the expression <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-77.gif"/></noscript></span></span>is the matrix equivalent of multi-trait heritability (the ratio of the genetic variance in the numerator and the phenotypic variance in the denominator); thus, when <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-78.gif"/></noscript></span></span>, the heritability of each trait is 1 and <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-79.gif"/></noscript></span></span> and <strong><span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-80.gif"/></noscript></span></span></strong>.</p><p id="p-21">As in the univariate approach, the multivariate KL loss contains the term <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-81.gif"/></noscript></span></span> that indicates that the joint probability of <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-82.gif"/></noscript></span></span> falling above <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-83.gif"/></noscript></span></span> has less penalty.</p><p id="p-22">Note that P is expressed in terms of its inverse; therefore, those traits with high variance induce a lower penalty, and there is a compromise between the gain from selection and keeping the variability. KL divergence is an appropriate way of selecting because it not only captures the association between traits, but also assigns weights to each trait automatically, thus reducing subjectivism in GS. In addition, <a id="xref-disp-formula-7-1" class="xref-disp-formula" href="#disp-formula-7">Equation (4b)</a> implies that traits with higher variance will have priority in a GS program and will directly preserve the genetic variance as much as possible. Univariate and multivariate KL LFs are represented in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1a</a> (red lines) and Fig. B1a (Appendix B), respectively.</p></div><div id="sec-6" class="subsection"><h3>The Continuous Ranked Probability Score and its generalization, the Energy Score</h3><p id="p-23">The Continuous Ranked Probability Score (CRPS) function is discussed in detail in <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Gneiting and Raftery (2007)</a> and <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hersbach (2000)</a>. The CRPS is a metric that reflects the distance between two random variables in terms of their cumulative distributions. The CRPS in GS acts by choosing candidate lines whose (cumulative) BV distributions are as close as possible to the hypothetical parental distribution that reflects the breeder’s preferences. Nevertheless, the main difference between KL divergence and CRPS is that the latter is less restrictive with lines whose distributions do not perfectly match the distribution that reflects our preferences, as can be seen in <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1a</a> (green line).</p><p id="p-24">Keeping the normality assumption, let <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-84.gif"/></noscript></span></span> be a normal random variable with mean <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-85.gif"/></noscript></span></span> and variance <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-86.gif"/></noscript></span></span>; there is a closed form of this LF (Appendix A), which in our context is defined as<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-87.gif"/></noscript></span><span class="disp-formula-label">(5a)</span></span><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-88.gif"/></noscript></span><span class="disp-formula-label">(5b)</span></span><span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-89.gif"/></noscript></span><span class="disp-formula-label">(5c)</span></span>where φ and Φ denote the <em>pdf</em> and <em>cdf</em> of a standard Gaussian variable, respectively. Again, CRPS increases with the intensity of selection and is a decreasing function of heritability.</p><p id="p-25">When the interest is in multiple traits, we can use the Energy Score (EnergyS), which generalizes the univariate CRPS (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Gneiting and Raftery 2007</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Székely and Rizzo 2013</a>) and allows a direct comparison between multi-traits. The Energy Score in terms of a genomic selection goal is expressed as<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-90.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-91.gif"/></noscript></span></span> denote the Euclidean norm, <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-92.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-93.gif"/></noscript></span></span> were previously defined, and <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-94.gif"/></noscript></span></span> denotes an independent random vector with the same distribution as <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-95.gif"/></noscript></span></span>, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-96.gif"/></noscript></span></span>. In contrast to KL divergence, with the CRPS and the Energy Score we can avoid the normality assumption while holding the assumption that traits are on an interval scale. This is a big advantage given that frequently the traits of interest in GS are not Gaussian. The EnergyS is depicted in Fig. B1b (Appendix B).</p></div><div id="sec-7" class="subsection"><h3>An asymmetric loss function in univariate and multivariate settings</h3><p id="p-26">At this point, the previous LFs that we adapted to the GS problem are all symmetric. Therefore, the penalties on the right- and left-hand side of the target at the same distance are equal. Nevertheless, it is natural to think that in conventional and GS plant breeding, those values of <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-97.gif"/></noscript></span></span> greater than <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-98.gif"/></noscript></span></span> might be less penalized if the goal is to increase phenotypic values. Here is where an asymmetric LF plays a key role because it assigns very small penalties to those individuals whose BV distributions are to the right of the hypothetical parental distribution. One simple asymmetric LF is the linear-linear loss (LinLin), whose behavior is linear on both sides of the target; it has the <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-99.gif"/></noscript></span></span> term that induces different penalties (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Berk 2011</a>) and is the loss of the quantile regression. The LinLin LF is defined as<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-100.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>where <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-101.gif"/></noscript></span></span>; then this LF is also a decreasing function of <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-102.gif"/></noscript></span></span>. LinLin loss is depicted in <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1a</a> (blue line). Note that when <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-103.gif"/></noscript></span></span>, the LinLin loss reduces to a symmetric linear LF.</p><p id="p-27">In the multi-trait setting, we define <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-104.gif"/></noscript></span></span> as the vector of deviations of <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-105.gif"/></noscript></span></span> from <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-106.gif"/></noscript></span></span> and the multivariate LinLin function is 0 when phenotypic and genotypic matrices are identical, <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-107.gif"/></noscript></span></span>. The <em>Multivariate Asymmetric Loss Function</em> (MALF) (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Komunjer and Owyang 2011</a>) is the generalization of LinLin LF. For GS purposes, it is expressed as<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-108.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>where <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-109.gif"/></noscript></span></span> is the Euclidean norm and τ controls the degree of asymmetry. In a simplified version with the <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-110.gif"/></noscript></span></span>, <a id="xref-disp-formula-13-1" class="xref-disp-formula" href="#disp-formula-13">Equation (8)</a> is expressed as<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-111.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span>In the univariate case, letting <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-112.gif"/></noscript></span></span>, <a id="xref-disp-formula-14-1" class="xref-disp-formula" href="#disp-formula-14">Equation (9)</a> reduces to twice the LinLin LF, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-113.gif"/></noscript></span></span>. Both univariate and multivariate LinLin losses favor selecting those lines with distributions similar to the hypothetical parental distribution (which reflects the breeder’s preferences) as much as possible. The MALF is depicted in Fig. B1c (Appendix B).</p></div><div id="sec-8" class="subsection"><h3>Posterior expected loss and their approximations</h3><p id="p-28">Each of the LFs proposed above can be used in conventional selection and/or GS to select the best lines according to our preferences. Below we briefly explain the formal approach used to evaluate posterior mean of the LFs proposed above. Given a prediction model (for example, Multi Trait Model (MTM) or Bayesian Ridge Regression (BRR) for a single trait), we have the joint posterior distribution<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-114.gif"/></noscript></span>,<span class="disp-formula-label">(10)</span></span>where <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-115.gif"/></noscript></span></span> is the likelihood function, and <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-116.gif"/></noscript></span></span> is the prior distribution for θ, and elements of θ are an overall mean (μ), marker effects or any other effects (β), and the variance-covariance matrix (P) captures the association between traits (or <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-117.gif"/></noscript></span></span>, the phenotypic variance in a univariate framework), y is a matrix (or vector in a single-trait setting) containing phenotypic records, and X represents the incidence matrix according to the multi-trait or single-trait model.</p><p id="p-29">Let <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-118.gif"/></noscript></span></span> be the random vector of the phenotypic values of a candidate with a covariate vector of markers <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-119.gif"/></noscript></span></span> and distribution <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-120.gif"/></noscript></span></span>. The posterior predictive density corresponding to <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-121.gif"/></noscript></span></span> is given by<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-122.gif"/></noscript></span><span class="disp-formula-label">(11)</span></span>Hence, given the observed phenotypic and genomic data (<span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-123.gif"/></noscript></span></span>), whatever we can predict about the candidate is given by (11) and any choice of the candidates for selection must take into account the uncertainty described by <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-124.gif"/></noscript></span></span>.</p><p id="p-30">According to the Bayesian decision theory, the optimal choice is one that minimizes the posterior expected loss. In this case, once the phenotypical values, y, and the matrix of covariables, X, are observed, we have to average the LF over all the unknowns in the model, that is, θ and the observable but unknown phenotypic value of the <em>o</em>-th candidate. Given that <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-125.gif"/></noscript></span></span> is computed as a function of μ, <strong>β</strong> and P <strong>(</strong>or <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-126.gif"/></noscript></span></span><strong>)</strong>, hereinafter we will denote all LFs described above as <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-127.gif"/></noscript></span></span>. Thus, the posterior expected value of the LF for candidate <em>o</em> is given by<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-128.gif"/></noscript></span><span class="disp-formula-label">(12)</span></span>Then, for each <em>o</em> line, we have a posterior expected loss <em><span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-129.gif"/></noscript></span></span></em> and we will select those lines with the lowest posterior expected losses.</p><p id="p-31">The expected value of the LFs given in (2), (5a) and (7), as well as their generalizations in the multivariate context (4b), (6), and (9) are approximated by Markov Chain Monte Carlo integration by taking m samples from the joint posterior distributions for <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-130.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-131.gif"/></noscript></span></span> (in the univariate context) or for vectors <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-132.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-133.gif"/></noscript></span></span> (in the multivariate context).</p></div><div id="sec-9" class="subsection"><h3>Application of univariate and multivariate loss functions in a wheat dataset</h3><p id="p-32">To illustrate the application of univariate and multivariate LFs in GS, we used 320 spring wheat lines with records on four traits: grain yield (GY), thousand-kernel weight (TKW), Zn and Fe concentrations in the grain (GZnC and GFeC, respectively) from CIMMYT’s biofortification breeding program and genotyped with DaRT markers (Velu <em>et al.</em> 2016). All traits are positively but low correlated; for example, GY had a correlation of 0.21 with TKW but zero with the other traits, whereas GZnC and GFeC had a 0.26 correlation.</p><p id="p-33">For univariate LFs, we used Bayesian Ridge Regression (BRR) and the BGLR R package (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">de los Campos and Pérez Rodríguez 2015</a>) in order to obtain posterior distributions of parameters and quantities of interest. We selected the top 10% (32 lines) of the wheat lines whose posterior expected losses were the minimum using the three univariate LFs discussed above (KL, CRPS, and LinLin). Results were contrasted with the top 10% based solely on punctual predicted BVs (Std), <em>i.e.</em>, by selecting individuals with the highest punctual predicted BVs. For multivariate LFs (KL, Energy Score and MALF), we fitted the MTM model (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">de los Campos and Grüneberg 2016</a>). The aim here was to select the “best” lines whose performance across all traits is high. For asymmetric LFs (LinLin and MALF), we fixed the value of α (or τ) equal to 0.9 (according to selection pressure) in order to impose big penalties on those individuals far to the left of the target point; on the other hand, lower penalties were given to those that were equal or greater.</p></div><div id="sec-10" class="subsection"><h3>Application of univariate and multivariate loss functions in a simulation scheme</h3><p id="p-34">Simulations were used to evaluate the three univariate and multivariate LFs discussed above. In the univariate scenario, we compared results obtained with the selection based on the standard method (Std). In the multivariate scenario, we contrasted results of different LFs. A recurrent selection scheme was used in which the best individuals were selected based on their merit regarding a single quantitative trait or three correlated quantitative traits, and measured through the LFs.</p></div><div id="sec-11" class="subsection"><h3>Traits and heritabilities</h3><p id="p-35">In the univariate simulation, one trait was genetically simulated, where the gene effects were sampled from a gamma distribution with shape and scale parameters equal to two. Phenotypes were simulated by summing up all true genotypic values and adding a residual effect consistent with the expected heritability fixed at 0.5. That is, <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-134.gif"/></noscript></span></span>, where p is the number of genes, and <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-135.gif"/></noscript></span></span> is the genotype for the j-th gene of the i-line (coded as -1 and 1 for the two homozygotes, respectively). The environmental noise was sampled as <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-136.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-137.gif"/></noscript></span></span> is the genetic variance.</p><p id="p-36">In the multi-trait simulation, three correlated quantitative traits were genetically simulated assuming a full pleiotropic model. This was carried out by randomly sampling gene effects for all segregating sites from a multivariate normal distribution with zero mean and a previously stated variance-covariance, to ensure a genetic correlation at first generation of -0.37 between trait 1 (T1) and trait 2 (T2); a genetic correlation of 0.34 between traits 2 (T2) and 3 (T3); and a genetic correlation of -0.02 between T1 and T3. To mimic complex and simple quantitative traits, narrow-sense heritabilities of 0.3 and 0.6 were assumed for all traits. Hereinafter, we will always refer to the traits in terms of narrow-sense heritability (<span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-138.gif"/></noscript></span></span>), given that in the simulation scheme we simulated a purely additive model and did not include dominance effects.</p></div><div id="sec-12" class="subsection"><h3>Selection cycles</h3><p id="p-37">The forward-in-time component represented ten cycles of a classic recurrent selection scheme in which the breeder has the ability to select based on the genomic predictions of the breeding values.</p><p id="p-38">In the multivariate LFs framework, genomic predictions of the evaluated lines were carried out using the Multi-Trait Model (MTM) (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">de los Campos and Grüneberg 2016</a>), where 70% and 30% of the lines were used as the training and testing sets (candidates), respectively. For single-trait simulation, genomic predictions were made using Bayesian Ridge Regression (BRR).</p><p id="p-39">For multi-trait simulation, the simulated phenotypic values were modeled assuming an intercept for each trait as a fixed effect and the predicted breeding value for each line in each trait as random. As in the wheat dataset, we fitted the same MTM model for the multi-trait simulation. In both the univariate and multi-trait simulation, posterior distributions of the predicted breeding values of the candidate lines were ranked according to each selection process (the univariate and multivariate LFs described above). For all LFs described above, approximations to the posterior expected loss were obtained considering 10,000 MCMC samples after a burn-in period of 30,000 samples. For univariate simulation, the top 10% and the top 30% of the lines in the candidate set were selected via their minimum posterior expected losses, while in the multi-trait simulation scheme, the top 10% of the lines in the candidate set were selected with minimum posterior expected losses.</p><p id="p-40">In the LinLin LF and its generalization MALF, we fixed the value of α or τ equal to 0.9 or 0.7 (depending on whether the selection pressure was 10% or 30%) in order to impose big penalties on those individuals in the candidate set that were on the far left-hand side of the target point, while lower penalties were given to those that were equal or greater.</p><p id="p-41">Results are presented as summaries of 20 replicates for each heritability, selection pressure and LF scenario. Individuals selected as parents will have the highest possible BV and the lowest possible value of the corresponding LFs. The simulations were implemented in a C++ program that was compiled, linked and executed within the R version 3.3.3 (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">R Development Core Team 2017</a>) through the facilities provided by the Rcpp package (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Eddelbuettel and Francois 2011</a>).</p></div><div id="sec-13" class="subsection"><h3>Data Availability</h3><p id="p-42">The durum wheat data including clean imputed markers and phenotypic information for the traits can be found in the following link: <a href="http://genomics.cimmyt.org/Decision_theory_GS/">http://genomics.cimmyt.org/Decision_theory_GS/</a></p></div></div><div class="section results" id="sec-14"><h2 class="">Results</h2><div id="sec-15" class="subsection"><h3>Univariate loss functions for wheat data</h3><p id="p-43">The boxplots in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2a-d</a> correspond to the predictive BVs of the top 10% spring wheat lines using univariate LFs (KL, CRPS, and LinLin) and the Std method (not using LFs) for the four traits. In general, no changes in the mean of selected candidates of the traits are found in the top 10% of the wheat lines because only a few of them (numbers in parentheses) changed for each LF, as compared with the Std method. Note that we only evaluated the selection differential because we could not cross the selected lines to compute the selection response as we did in the simulation study. In fact, KL was the only LF that had four different selected lines for traits GY, TKW and GFeC and only one line for trait GZnC. In contrast, LinLin loss selected almost the same lines as the Std method, given that traits GY and GFeC only diverged in one line, trait TKW diverged in three individuals, and in trait GZnC no different individuals were selected. CRPS LF performed between KL and LinLin by selecting 3, 1, 1, and 2 lines for traits GY, TKW, GZnC and GFeC, respectively, which were different than the lines selected by Std. Despite the fact that only a few lines with BV ranked differently under the LFs <em>vs.</em> the Std method, the overall effect of those different selected lines can change over subsequent selection cycles.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Univariate real data. Boxplots of estimated breeding values for a real wheat dataset (with four traits) of the top 10% of selected lines with three univariate loss functions Kullback-Leibler (KL), Continuous Ranked Probability Score (CRPS) and Linear-Linear loss (LinLin), and breeding values of the lines selected under the standard method (Std) for A) Grain Yield (GY), B) thousand-kernel weight (TKW), C) Zn concentration in grain (GZnC), and D) Fe concentration in grain (GFeC). Values in parentheses are the lines that the loss functions selected but the Std did not." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-388110402" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Univariate real data. Boxplots of estimated breeding values for a real wheat dataset (with four traits) of the top 10% of selected lines with three univariate loss functions Kullback-Leibler (KL), Continuous Ranked Probability Score (CRPS) and Linear-Linear loss (LinLin), and breeding values of the lines selected under the standard method (Std) for A) Grain Yield (GY), B) thousand-kernel weight (TKW), C) Zn concentration in grain (GZnC), and D) Fe concentration in grain (GFeC). Values in parentheses are the lines that the loss functions selected but the Std did not.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/3019/F2.medium.gif" width="440" height="302"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/9/3019/F2.medium.gif" width="440" height="302"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490867" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-44" class="first-child">Univariate real data. Boxplots of estimated breeding values for a real wheat dataset (with four traits) of the top 10% of selected lines with three univariate loss functions Kullback-Leibler (KL), Continuous Ranked Probability Score (CRPS) and Linear-Linear loss (LinLin), and breeding values of the lines selected under the standard method (Std) for A) Grain Yield (GY), B) thousand-kernel weight (TKW), C) Zn concentration in grain (GZnC), and D) Fe concentration in grain (GFeC). Values in parentheses are the lines that the loss functions selected but the Std did not.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Multivariate loss functions for wheat data</h3><p id="p-45"><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3a-d</a> depicts the boxplot of predictive BVs of selected individuals using multivariate LFs, EnergyS, KL and MALF. The selected candidates are the top 10% of lines selected for the four traits.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Multivariate real data. Boxplots of estimated breeding values for a real spring wheat dataset of the top 10% of selected lines for four traits according to the multivariate loss functions Kullback-Leibler (KL), Energy Score (EnergyS) and Multivariate Asymmetric Loss Function (MALF) for A) grain yield (GY), B) thousand-kernel weight (TKW), C) Zn concentration in grain (GZnC), and D) Fe concentration in grain (GFeC). Brown dots indicate the mean of all lines." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-388110402" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Multivariate real data. Boxplots of estimated breeding values for a real spring wheat dataset of the top 10% of selected lines for four traits according to the multivariate loss functions Kullback-Leibler (KL), Energy Score (EnergyS) and Multivariate Asymmetric Loss Function (MALF) for A) grain yield (GY), B) thousand-kernel weight (TKW), C) Zn concentration in grain (GZnC), and D) Fe concentration in grain (GFeC). Brown dots indicate the mean of all lines.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/3019/F3.medium.gif" width="440" height="353"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/9/3019/F3.medium.gif" width="440" height="353"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490775" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-46" class="first-child">Multivariate real data. Boxplots of estimated breeding values for a real spring wheat dataset of the top 10% of selected lines for four traits according to the multivariate loss functions Kullback-Leibler (KL), Energy Score (EnergyS) and Multivariate Asymmetric Loss Function (MALF) for A) grain yield (GY), B) thousand-kernel weight (TKW), C) Zn concentration in grain (GZnC), and D) Fe concentration in grain (GFeC). Brown dots indicate the mean of all lines.</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">For the complex trait GY, which has a 0.21 sample phenotypic correlation with trait TKW and zero correlation with the other traits, the KL was the best LF and the mean of the selected lines was superior to the population mean. The mean GY of the top 10% of the lines developed using MALF ranked second and the top 10% of the lines selected by EnergyS multivariate LF were the worst of the three LFs that were not better than the mean of the base population. For trait TKW, the top 10% of the best lines in the three LFs were similar and superior to the mean of the original population. For trait GZnC, EnergyS and MALF LFs produced the best lines, which were slightly better than those produced by the KL LF. Finally, for trait GFeC, the three LFs produced the top 10% of lines with similar Fe concentration in the grain.</p><p id="p-48">The best LF was the KL loss, given that it selected individuals that, on average, performed better than the estimated population mean in all traits. Results of Hotelling’s two sample T2-tests indicate that the mean of the selected individuals between KL and EnergyS was statistically different at a 0.05 confidence level for error type I. The same test suggested no differences when contrasting KL <em>vs.</em> MALF and MALF <em>vs.</em> Energy. On the other hand, we computed the percentage of lines that do not intersect in each combination of LFs. For example, between KL and EnergyS LFs, 38% (12 of 32) of the lines are different between both LFs. Between KL and MALF, 34% (11 of 32) of the lines are different. In contrast, EnergyS <em>vs.</em> MALF had only 3% (1 of 32) of different selected lines between both LFs. By crossing the selected candidates from each of the LFs and repeating the procedure over selection cycles, we would expect a gain in the mean of all traits, with better performance, on average, from the lines selected under the KL LF.</p></div><div id="sec-17" class="subsection"><h3>Univariate loss function for simulation data</h3><p id="p-49">In the following results of the univariate and multivariate simulation framework, the population mean for each cycle was standardized according to <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-139.gif"/></noscript></span></span>. This is a no-dimensional quantity where <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-140.gif"/></noscript></span></span> represents the population mean in cycle i, <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-141.gif"/></noscript></span></span> is the population mean of the first cycle, <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-142.gif"/></noscript></span></span> is the selection response in cycle i with respect to the first, and <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-143.gif"/></noscript></span></span> is the population standard deviation of cycle 1. For the variance population, we scaled the variances of cycle i by the variances of the first cycle, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-144.gif"/></noscript></span></span>.</p><p id="p-50"><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figures 4a-b</a> show the trends of the average standardized population mean for the univariate simulation scheme separating cycles 1-5 and cycles 5-10 for better visualization. The scale population variance is depicted in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figures 5a-b</a>. These figures summarize the results when selecting the top 10% of candidates with minimum posterior expected losses. There were no significant changes in the population means and population variances when comparing LFs <em>vs.</em> the conventional (Std) selection, <em>i.e.</em>, LFs performed similarly to Std. In contrast, when the selected proportion was 30% of the candidates, there was a slight improvement using LFs as the selection program advanced, as depicted in Figs. B2a-b (Appendix B) for the standardized population mean and in Figs. B3a-b (Appendix B) for the scaled population variance.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results of the univariate simulation study. Standardized selection response for breeding cycles 1 to 5 are illustrated in A), while cycles 5 to 10 are in B). In each selection cycle, the top 10% of lines with minimum posterior expected losses were selected using the Kullback-Leibler loss function (KL), the Continuous Ranked Probability Score (CRPS), the Linear-Linear loss function (LinLin), and the standard method (Std). Selected lines were crossed in each cycle to recover the population size for upcoming selection cycles.  and  represent the population mean and the selection response, respectively, in cycle i;  and  are the population mean and the population standard deviation, respectively, in cycle 1. The black vertical lines indicate the standard error of  under 20 replications of the simulation study." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-388110402" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Results of the univariate simulation study. Standardized selection response &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-145&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;for breeding cycles 1 to 5 are illustrated in A), while cycles 5 to 10 are in B). In each selection cycle, the top 10% of lines with minimum posterior expected losses were selected using the Kullback-Leibler loss function (KL), the Continuous Ranked Probability Score (CRPS), the Linear-Linear loss function (LinLin), and the standard method (Std). Selected lines were crossed in each cycle to recover the population size for upcoming selection cycles. &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-146&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-147&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; represent the population mean and the selection response, respectively, in cycle i; &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-148&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-149&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; are the population mean and the population standard deviation, respectively, in cycle 1. The black vertical lines indicate the standard error of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-150&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; under 20 replications of the simulation study.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/3019/F4.medium.gif" width="440" height="159"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/9/3019/F4.medium.gif" width="440" height="159"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490740" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-51" class="first-child">Results of the univariate simulation study. Standardized selection response <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F4/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F4/embed/mml-math-145.gif"/></noscript></span></span>for breeding cycles 1 to 5 are illustrated in A), while cycles 5 to 10 are in B). In each selection cycle, the top 10% of lines with minimum posterior expected losses were selected using the Kullback-Leibler loss function (KL), the Continuous Ranked Probability Score (CRPS), the Linear-Linear loss function (LinLin), and the standard method (Std). Selected lines were crossed in each cycle to recover the population size for upcoming selection cycles. <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F4/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F4/embed/mml-math-146.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F4/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F4/embed/mml-math-147.gif"/></noscript></span></span> represent the population mean and the selection response, respectively, in cycle i; <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F4/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F4/embed/mml-math-148.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F4/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F4/embed/mml-math-149.gif"/></noscript></span></span> are the population mean and the population standard deviation, respectively, in cycle 1. The black vertical lines indicate the standard error of <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F4/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F4/embed/mml-math-150.gif"/></noscript></span></span> under 20 replications of the simulation study.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results of the univariate simulation study. Scaled population variance  for breeding cycles 1 to 5 are illustrated in A), while cycles 5 to 10 are in B). In each selection cycle, the top 10% were selected using the Kullback-Leibler (KL), the Continuous Ranked Probability Score (CRPS), and the Linear-Linear (LinLin) loss functions, and lines selected under the standard method (Std). Selected lines were crossed at each cycle to recover the population size for upcoming selection cycles.  and  are the population variance in cycle i and cycle 1, respectively. The black vertical lines indicate the standard error of  under 20 replications of the simulation study." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-388110402" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Results of the univariate simulation study. Scaled population variance &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-151&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for breeding cycles 1 to 5 are illustrated in A), while cycles 5 to 10 are in B). In each selection cycle, the top 10% were selected using the Kullback-Leibler (KL), the Continuous Ranked Probability Score (CRPS), and the Linear-Linear (LinLin) loss functions, and lines selected under the standard method (Std). Selected lines were crossed at each cycle to recover the population size for upcoming selection cycles. &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-152&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-153&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; are the population variance in cycle i and cycle 1, respectively. The black vertical lines indicate the standard error of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-154&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; under 20 replications of the simulation study.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/3019/F5.medium.gif" width="440" height="160"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/9/3019/F5.medium.gif" width="440" height="160"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490811" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-52" class="first-child">Results of the univariate simulation study. Scaled population variance <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F5/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F5/embed/mml-math-151.gif"/></noscript></span></span> for breeding cycles 1 to 5 are illustrated in A), while cycles 5 to 10 are in B). In each selection cycle, the top 10% were selected using the Kullback-Leibler (KL), the Continuous Ranked Probability Score (CRPS), and the Linear-Linear (LinLin) loss functions, and lines selected under the standard method (Std). Selected lines were crossed at each cycle to recover the population size for upcoming selection cycles. <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F5/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F5/embed/mml-math-152.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F5/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F5/embed/mml-math-153.gif"/></noscript></span></span> are the population variance in cycle i and cycle 1, respectively. The black vertical lines indicate the standard error of <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F5/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F5/embed/mml-math-154.gif"/></noscript></span></span> under 20 replications of the simulation study.</p><div class="sb-div caption-clear"></div></div></div><p id="p-53">The boxplots in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6a</a> and Figure B4a (Appendix B) depict the mean of the 10-th selection cycle when the proportions of selected individuals were 10% and 30%, respectively. When 10% were selected, no differences were found in terms of the standardized response to selection between LFs and the standard method. In contrast, when 30% were selected, the LFs performed better than the standard method in terms of the average response to selection in the 10-th cycle. <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>a gives the <em>t</em>-test for comparing differences in the mean when selecting 10% of the lines under the univariate LFs <em>vs.</em> the standard method (Std), whereas <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>b shows the same results but for the 30%. There were significant differences among the population means of lines selected under LF <em>vs.</em> those selected without employing any LF for the 30%.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results of the univariate simulation study at the 10th selection cycle for 10% of the selected lines. A) boxplots of the standardized selection response; B) boxplots of the scale population variance using the Kullback-Leibler (KL), the Continuous Ranked Probability Score (CRPS), the Linear-Linear (LinLin) loss functions, and lines selected under the standard method (Std). The boxplots illustrate the mean (white dots) and median (black middle line) of 20 replications of the simulation study.  and  were defined in Figs. 4 and 5. Sub-indices refer to the 10th selection cycle." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-388110402" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Results of the univariate simulation study at the 10&lt;sup&gt;th&lt;/sup&gt; selection cycle for 10% of the selected lines. A) boxplots of the standardized selection response; B) boxplots of the scale population variance using the Kullback-Leibler (KL), the Continuous Ranked Probability Score (CRPS), the Linear-Linear (LinLin) loss functions, and lines selected under the standard method (Std). The boxplots illustrate the mean (white dots) and median (black middle line) of 20 replications of the simulation study. &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-155&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-156&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; were defined in Figs. 4 and 5. Sub-indices refer to the 10&lt;sup&gt;th&lt;/sup&gt; selection cycle.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/3019/F6.medium.gif" width="440" height="216"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/9/3019/F6.medium.gif" width="440" height="216"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490781" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-54" class="first-child">Results of the univariate simulation study at the 10<sup>th</sup> selection cycle for 10% of the selected lines. A) boxplots of the standardized selection response; B) boxplots of the scale population variance using the Kullback-Leibler (KL), the Continuous Ranked Probability Score (CRPS), the Linear-Linear (LinLin) loss functions, and lines selected under the standard method (Std). The boxplots illustrate the mean (white dots) and median (black middle line) of 20 replications of the simulation study. <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F6/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F6/embed/mml-math-155.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F6/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F6/embed/mml-math-156.gif"/></noscript></span></span> were defined in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figs. 4</a> and <a id="xref-fig-5-2" class="xref-fig" href="#F5">5</a>. Sub-indices refer to the 10<sup>th</sup> selection cycle.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490747/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490747/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <p id="p-55" class="first-child">Simulation univariate study. Student t-test of mean and variance differences between the lines selected by the univariate Kullback-Leibler (KL), Continuous Ranked Probability Score (CRPS) and Linear-Linear (LinLin) loss functions <em>vs.</em> lines selected under the standard selection method (Std), after 20 replications of the simulated breeding program. The selected proportions were the top 10% and top 30% of the candidates, and the means and variances were compared at the 10<sup>th</sup> selection cycle</p><div class="sb-div caption-clear"></div></div></div><p id="p-56">The results of the variance of the final populations in cycle 10<sup>th</sup> are shown in <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6b</a> and Figure B4b (Appendix B) for 10% and 30% of the selected individuals, respectively. When 10% were selected, the boxplot in <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6b</a> does not show any difference in the variance obtained under the LFs <em>vs.</em> the variance of the population using the standard method. However, for the 30%, there were substantial differences between variances of populations selected under the LFs compared with those selected based solely on the mean of the breeding values (Figure B4b, Appendix B). <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Tables 1</a>c and 1d show the <em>t</em>-test for comparing the mean of the population variance at the 10-th selection cycle when selecting 10% and 30% of the individuals. The only case where the mean of the population variance of the lines selected under the LF was statistically higher than the mean of the population variance of the lines obtained using the standard method was for KL and LinLin LFs when 30% of the individuals were selected.</p></div><div id="sec-18" class="subsection"><h3>Multivariate loss functions for simulated data</h3><div id="sec-19" class="subsection"><h4>The population mean:</h4><p id="p-57"><a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a> depicts the changes in the standardized population mean for each trait when the heritabilities for all traits were fixed at 0.3 (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7a</a>) and 0.6 (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7b</a>). For T1 with a negative correlation with T2 and negligible association with T3, the KL and MAFL LFs performed similarly with respect to the population mean throughout the breeding cycles, and their performance was better than the performance of the lines selected under EnergyS. The mean percentage differences computed for the 10<sup>th</sup> cycle with respect the first cycle for both heritability cases are shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>a. For trait T1, KL gained 1.583% <em>vs.</em> 1.211% of the MALF LF for a heritability of 0.3. On the other hand, EnergyS had a conservative gain of 0.462%. For the second trait (T2), the three LFs had positive performances. EnergyS had the best performance, with a final gain of 8.224% in the 10<sup>th</sup> cycle for a heritability of 0.3. In second and third place were MALF and KL with gains of 6.493% and 6.441%, respectively. Although the KL and MALF LFs had the smallest gains, the sign of the gains was positive for all selection cycles. It is important to note that the correlation between T1 and T2 was -0.37 and negligible between T1 and T3. Finally, for trait T3 with a heritability of 0.3, the performance of the population mean throughout the breeding cycles was similar for all three LFs. The percentages of gain with respect to the first cycle confirm these results. In descending order, the gains were EnergyS (6.14%), KL (5.762%) and MALF (5.499%) for a heritability of 0.3 (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>a).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results of the multivariate simulation study. A) Standardized population mean  for the breeding cycles when heritability for all traits was fixed at 0.3, and B) standardized population mean when heritability was 0.6 for all traits. In each selection cycle, the top 10% of candidates were selected using the multivariate loss functions: Kullback-Leibler (KL), Energy Score (EnergyS) and Multivariate Asymmetric Loss Function (MALF) to recover the population size for the upcoming breeding cycles.  and  correspond to the population mean and the selection response, respectively, in cycle i;  and  are the population mean and the population standard deviation, respectively, in cycle 1. The black vertical lines indicate the standard error of  under 20 replications of the simulation study." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-388110402" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Results of the multivariate simulation study. A) Standardized population mean &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-157&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for the breeding cycles when heritability for all traits was fixed at 0.3, and B) standardized population mean when heritability was 0.6 for all traits. In each selection cycle, the top 10% of candidates were selected using the multivariate loss functions: Kullback-Leibler (KL), Energy Score (EnergyS) and Multivariate Asymmetric Loss Function (MALF) to recover the population size for the upcoming breeding cycles. &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-158&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-159&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; correspond to the population mean and the selection response, respectively, in cycle i; &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-160&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-161&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; are the population mean and the population standard deviation, respectively, in cycle 1. The black vertical lines indicate the standard error of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-162&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; under 20 replications of the simulation study.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/3019/F7.medium.gif" width="440" height="243"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/8/9/3019/F7.medium.gif" width="440" height="243"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490756" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-58" class="first-child">Results of the multivariate simulation study. A) Standardized population mean <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F7/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F7/embed/mml-math-157.gif"/></noscript></span></span> for the breeding cycles when heritability for all traits was fixed at 0.3, and B) standardized population mean when heritability was 0.6 for all traits. In each selection cycle, the top 10% of candidates were selected using the multivariate loss functions: Kullback-Leibler (KL), Energy Score (EnergyS) and Multivariate Asymmetric Loss Function (MALF) to recover the population size for the upcoming breeding cycles. <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F7/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F7/embed/mml-math-158.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F7/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F7/embed/mml-math-159.gif"/></noscript></span></span> correspond to the population mean and the selection response, respectively, in cycle i; <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F7/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F7/embed/mml-math-160.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F7/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F7/embed/mml-math-161.gif"/></noscript></span></span> are the population mean and the population standard deviation, respectively, in cycle 1. The black vertical lines indicate the standard error of <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F7/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F7/embed/mml-math-162.gif"/></noscript></span></span> under 20 replications of the simulation study.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490753/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490753/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Simulation multivariate study. Means of percentage differences of population means a) and for population variance b) in the 10<sup>th</sup> breeding cycle with respect to the first cycle for trait 1 (T1), trait 2 (T2) and trait 3 (T3) for lines selected under three multivariate loss functions Kullback-Leibler (KL), Energy Score (EnergyS) and Multivariate Asymmetric Loss Function (MALF) (standard errors are in parentheses). Heritability of 0.3 and 0.6 for all traits</span><div class="sb-div caption-clear"></div></div></div><p id="p-59">When we fixed the heritabilities for all traits at 0.6, the results were similar to those described above. <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>a shows the mean of the percent differences computed for the population mean in the 10<sup>th</sup> cycle, taking the mean in the first cycle as a reference. For T1, those differences were, in descending order, 3.318% (KL), 2.347% (MALF) and 0.506% (EnergyS). For T2, EnergyS (9.819%) achieved the best performance for the population mean, followed by the KL LF (6.206%) and finally, MALF (5.721%). All LFs performed similarly for T3, although the EnergyS and KL LFs had the highest gains: 7.291% and 6.935%, respectively. They were followed very closely by MALF, with 5.842% of the cumulative gains at the end of the simulated selection program.</p></div><div id="sec-20" class="subsection"><h4>The population variance:</h4><p id="p-60"><a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a> shows the averages of the scaled population variance for all traits in each selection cycle when the heritabilities for all traits were set at 0.3 (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8a</a>) and 0.6 (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8b</a>). Note that the decrease in the population variance was similar in all cases (trait-LFs combinations). <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>b shows the mean differences in the percentages of population variance (not scaled) for each trait, computed for the 10<sup>th</sup> breeding cycle taking the value of the first cycle as a reference. For T1 and a heritability of 0.3, these differences were, in descending order, -47.437% (MALF), -47.533% (EnergyS), and -47.886% (KL) (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>b). For T2 and 0.3 heritability, the decreases were -47.584% (EnergyS), -47.928% (MALF), and -48.128% (KL). Finally, the decreases in genetic variance for T3 and a 0.3 heritability for the LFs were -47.820% (MALF), -50.287% (EnergyS), and -51.181% (KL) (<a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>b). In general, these results show that the selection of parents does not kill more variance between LFs, although LFs reported different positive gains for the population mean of all traits.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results of the multivariate simulation study. A) Scale population variance  for the breeding cycles when heritability for all traits was fixed at 0.3, and B) scale population variance when heritability was 0.6 for all traits. In each selection cycle, the top 10% of candidates were selected using the multivariate loss functions: Kullback-Leibler (KL), Energy Score (EnergyS) and Multivariate Asymmetric Loss Function (MALF) to recover the population size for the upcoming breeding cycles.  and  are the population variances for cycle i and cycle 1, respectively. The black vertical lines indicate the standard error of  under 20 replications of the simulation study." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-388110402" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Results of the multivariate simulation study. A) Scale population variance &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-165&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for the breeding cycles when heritability for all traits was fixed at 0.3, and B) scale population variance when heritability was 0.6 for all traits. In each selection cycle, the top 10% of candidates were selected using the multivariate loss functions: Kullback-Leibler (KL), Energy Score (EnergyS) and Multivariate Asymmetric Loss Function (MALF) to recover the population size for the upcoming breeding cycles. &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-166&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-167&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; are the population variances for cycle i and cycle 1, respectively. The black vertical lines indicate the standard error of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-168&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; under 20 replications of the simulation study.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/3019/F8.medium.gif" width="440" height="243"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/8/9/3019/F8.medium.gif" width="440" height="243"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490885" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-61" class="first-child">Results of the multivariate simulation study. A) Scale population variance <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F8/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F8/embed/mml-math-165.gif"/></noscript></span></span> for the breeding cycles when heritability for all traits was fixed at 0.3, and B) scale population variance when heritability was 0.6 for all traits. In each selection cycle, the top 10% of candidates were selected using the multivariate loss functions: Kullback-Leibler (KL), Energy Score (EnergyS) and Multivariate Asymmetric Loss Function (MALF) to recover the population size for the upcoming breeding cycles. <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F8/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F8/embed/mml-math-166.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F8/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F8/embed/mml-math-167.gif"/></noscript></span></span> are the population variances for cycle i and cycle 1, respectively. The black vertical lines indicate the standard error of <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F8/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F8/embed/mml-math-168.gif"/></noscript></span></span> under 20 replications of the simulation study.</p><div class="sb-div caption-clear"></div></div></div><p id="p-62">When we fixed the heritability at 0.6 for all traits, the mean of population variances performed similarly for all traits, reinforcing the idea that the LFs proposed in this research do not kill more variance between them, as shown in <a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure 8b</a>. The losses in variance percentages (not scaled) computed in the last cycle with respect to the first selection cycle show that, for T1, the reductions in genetic variance were -44.532% (KL), -44.977% (MALF), and -45,256% (EnergyS) (<a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a>b). For T2, the reductions in genetic variance were -43.975% (KL), -45.451% (MALF), and -46.618% (EnergyS). Finally, for T3, the reductions in genetic variance were -44.910% (MALF), -49.588% (KL), and -48.025% (EnergyS). In summary, all multivariate LFs described in this study performed similarly in reducing the genetic variance as the selection program advanced.</p></div></div></div><div class="section discussion" id="sec-21"><h2 class="">Discussion</h2><p id="p-63">The objective in this study was to propose a formal methodology to select the best performing parents for future selection cycles in a genomic selection program. In a decision theory problem, LFs are the mechanism to penalize our potential decisions. In GS, LFs reflect our preferences for individuals with the desired characteristics to ensure the highest genetic progress. This approach is advantageous mainly when selection is conducted in multi-trait settings, where some of the traits are either positively or negatively correlated. Therefore, losses may be interpreted as deviations from our goal of improving genetic progress. Lines with posterior predictive distributions that are closer to the theoretical distribution will have minimum losses.</p><p id="p-64">For this reason, we proposed three univariate LFs (Kullback-Leibler, CRPS and LinLin) and their multivariate generalizations (Kullback-Leibler, Energy Score, and MALF) to assist plant breeders when selecting the best performing parents for future selection cycles from a candidate set used in GS. The univariate and multivariate, symmetric and asymmetric LFs were presented as decreasing functions of the heritability of the traits(s). From this perspective, deviations between the distributions of candidate parents and the theoretical distribution that reflects the plant breeder’s preferences are seen as divergences. The previous sections presented the results obtained using a real dataset (wheat lines) during one selection cycle, and in a simulation study scheme under the univariate and multivariate LFs (single-trait and multi-trait). When selecting a single trait, we compared the performance of the LFs with the performance of the standard method of selection. The standard selection method ranks individuals based on punctual predictions of BVs. In the multi-trait simulation, we compared the performance of multivariate LFs.</p><p id="p-65">The results of the univariate simulation were encouraging given that LFs performed better than the conventional method when the selection pressure was not too restrictive (30%). In the less favorable scenario, the LFs performed as well as the GS method that does not employ LFs (Std). These univariate results allow us to generalize the use of LFs to the multi-trait framework. In the multi-trait problem, the LFs can be considered as methods that balance the gains simultaneously. However, this does not mean that LFs can attain the maximum possible gains for each trait, such as when selection operates on only one trait at a time.</p><p id="p-66">Addressing selection as a decision theory problem faces the restriction of multi-trait <em>vs.</em> single-trait selection; when some traits are negatively correlated, the selected trait might reach its optimum value while the other traits stay the same or move away from their optimum values. It is important to note that our methodology allows researchers to control parental distribution and thus induce selection that favors some traits over others and restricts the selection of individuals for particular traits, as selection indices do when using restrictive selection indices. For instance, suppose that for T1, we do not desire gains between selection cycles; it is enough to leave without truncation the distribution for such a trait (<span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-169.gif"/></noscript></span></span>).</p><p id="p-67">As previously stated, in the simulation study we induced T1 and T2 with a negative correlation; this means that as the phenotypic values of T1 increase, the phenotypic values of T2 decrease whenever the selection works in favor of T1, and vice versa. In the case of T2 and T3, the induced correlation was positive, whereas T1 and T3 were independent of each other (no correlation). This setting was purposely chosen to show that our methodology works even in the worst scenario. Results indicate that LFs worked in favor of all traits, in spite of their negative correlations; the EnergyS worked better for T2 and T3 than for T1, and in that sense, it is the least promising of the three multivariate LFs. Further research on comparing the performance of LFs <em>vs.</em> selection indices (SI) will be useful. Using SI to calibrate economic weights and obtain simultaneous gains in (negatively) correlated traits is cumbersome and economic analyses should be done. However, in multivariate LF, this is done automatically.</p><p id="p-68">On the other hand, the population means for T2 obtained with KL and MALF showed a lower performance than those obtained with EnergyS, which is understandable given the good performance in T1 under both LFs. The KL and MALF chose those candidates whose BVs ensured gains in all traits. Therefore, for T2, EnergyS had a small but significant gain <em>vs.</em> MALF and KL loss, which is no surprise since it was at the expense of sacrificing the gains in T1. For T3, the LFs had similar performances, with small but significant differences between them. This is illustrated in <a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7a-b</a>. It should be pointed out that although the LF reported increases in the population means of the three traits (for the two heritability values), the population genetic variance decreased at the same rate for all LFs, as depicted in <a id="xref-fig-8-5" class="xref-fig" href="#F8">Figure 8a-b</a>.</p><p id="p-69">In summary, we propose a formal approach for selecting the best performing parents for upcoming selection cycles. Our approach is based on Bayesian decision theory to construct divergences (LFs) that score candidates. Expected posterior losses take into account uncertainties about parameters and predictions involved in regression models. Our results show that using the proposed LFs when selecting a single trait can improve the response to selection while preserving the population variance. In the multi-trait framework, the advantages of LFs are evident. The population means of all traits under consideration showed positive gains, even though two of them were negatively correlated. We believe that selection based on LFs is more convenient than selection based solely on BVs, given that in LFs the weights for each trait are calculated automatically. It is easier to fix a truncation point than to calibrate the economic weights used in selection indices. These results were analogous for both complex and simple traits.</p><p id="p-70">Finally, the Bayesian decision theory using several loss functions studied in this research and applied to GS-assisted breeding can be extended and used in conventional plant and animal breeding methods. Loss functions described in this study can be employed in data from progeny trials in plant and animal breeding to rank the selected individuals based not only on the adjusted means, but also on the loss function for a single trait or multiple traits. As shown in the long-term simulation study, changes in the rank of a few individuals can change the final response to selection after several selection cycles.</p></div><div class="section conclusions" id="sec-22"><h2 class="">Conclusions</h2><p id="p-71">We proposed applying a formal methodology within the decision theory framework, to the problem of selecting in the single- and multi-trait context, when applying genomic selection in plant breeding. Therefore, we discussed a theoretical justification and then proposed three univariate LFs (Kullback-Leibler, CRPS, and LinLin) and their corresponding generalizations in a multivariate setting (Multivariate Kullback-Leibler, Energy Score and MALF) with a theoretical derivation that expresses the LFs in terms of the heritability of the trait(s). We performed an example on a real dataset during one selection cycle for univariate and multivariate LFs, and in a simulation study of a genomic selection program in order to compute the population mean and variance throughout the breeding cycles. We contrasted our results with those obtained using the standard selection method in a single-trait scenario (selection based on predictions of punctual breeding values). Our results suggest that it is possible to obtain better performance in a long-term breeding program in the single-trait context by selecting 30% of the best individuals in each cycle. For the multi-trait approach, our results show that the population mean for all traits under consideration had positive gains, even though two of the traits were negatively correlated. Although each multivariate LF performed very well in the population means, the corresponding population variances were not statistically different.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-100">The authors are thankful to all field and lab assistances of the Global Wheat Program of CIMMYT that generate the data analyzed in this study.</p></div><div class="section app" id="app-1"><h2 class="">Appendix A</h2><div class="section" id="sec-23"><h3>Univariate Kullback-Leibler loss derivation</h3><span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-170.gif"/></noscript></span></span><p id="p-72">where <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-171.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-172.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-173.gif"/></noscript></span></span> denote the mean and the variance, respectively, when truncation at <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-174.gif"/></noscript></span></span> occurs.</p></div><div class="section" id="sec-24"><h3>Multivariate Kullback-Leibler loss derivation</h3><span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-175.gif"/></noscript></span></span><p id="p-73">with <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-176.gif"/></noscript></span></span>.</p></div><div class="section" id="sec-25"><h3>Univariate CRPS derivation</h3><p id="p-74">The CRPS function for genomic selection can be expressed as <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-177.gif"/></noscript></span></span>. If we assume that <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-178.gif"/></noscript></span></span> and use some known results of the folded normal distribution, we can derive an analytic expression of the CRPS function.</p><p id="p-75">Assume that <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-179.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-180.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-181.gif"/></noscript></span></span>. Let <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-182.gif"/></noscript></span></span>, then <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-183.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-184.gif"/></noscript></span></span> means a folded normal distribution. Its probability density function is<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-185.gif"/></noscript></span></span>The expectation of W is<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-186.gif"/></noscript></span></span>Details of folded normal distribution can be found in <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Tsagris <em>et al.</em> (2014)</a>. Now, we will proceed to evaluate <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-187.gif"/></noscript></span></span> as follows. Note that <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-188.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-189.gif"/></noscript></span></span>. Then, using the previous results, we have that<span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-190.gif"/></noscript></span></span>Simplifying,</p><span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-191.gif"/></noscript></span></span></div><div class="section" id="sec-26"><h3>R Code for Univariate Loss Functions</h3><p id="p-76">The proposed univariate loss functions use MCMC samples from posterior distributions after a burn-in time period of parameters described in the paper obtained from the BGLR R package.</p><p id="p-77">Inputs of loss function in R language:</p><p id="p-78">• Xb: matrix of MCMC samples of the mean of the candidates of the unobserved phenotype values. Rows for lines, columns for MCMC samples (<span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-192.gif"/></noscript></span></span>).</p><p id="p-79">• MU: vector of MCMC samples of the population mean (<span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-193.gif"/></noscript></span></span>).</p><p id="p-80">• sigma: vector of MCMC samples of the population standard deviation (<span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-194.gif"/></noscript></span></span>).</p><p id="p-81">• y_c: truncation point (scalar).</p><p id="p-82">• Nsel: number of the candidates to be selected.</p><p id="p-83">• alpha: value indicating the proportion of selected individuals from the candidate set.</p><p id="p-84">Output:</p><p id="p-85">• id of selected lines with the minimum posterior expected losses.</p><div id="T3" class="table pos-anchor"><div class="table-inline"><table frame="border" id="table-4"><col width="100%" span="1" id="column-9"/><thead id="thead-4"><tr id="tr-19"><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-30" class="table-center table-vtop">Selection through Standard Method (Std)</th></tr></thead><tbody id="tbody-4"><tr id="tr-20"><td valign="top" align="left" rowspan="1" colspan="1" id="td-91" class="table-left table-vtop">selStd &lt;- function(Xb, y_c, Nsel, MU, sigma) {</td></tr><tr id="tr-21"><td valign="top" align="left" rowspan="1" colspan="1" id="td-92" class="table-left table-vtop">yHat &lt;- apply(Xb, 1, mean, na.rm = TRUE)</td></tr><tr id="tr-22"><td valign="top" align="left" rowspan="1" colspan="1" id="td-93" class="table-left table-vtop">selected &lt;- order(yHat, decreasing = TRUE)[1:Nsel]</td></tr><tr id="tr-23"><td valign="top" align="left" rowspan="1" colspan="1" id="td-94" class="table-left table-vtop">return(selected)</td></tr><tr id="tr-24"><td valign="top" align="left" rowspan="1" colspan="1" id="td-95" class="table-left table-vtop">}</td></tr></tbody></table><table frame="border" id="table-5"><col width="100%" span="1" id="column-10"/><thead id="thead-5"><tr id="tr-25"><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-31" class="table-center table-vtop">Selection through Kullback-Leibler (KL)</th></tr></thead><tbody id="tbody-5"><tr id="tr-26"><td valign="top" align="left" rowspan="1" colspan="1" id="td-96" class="table-left table-vtop">selKL &lt;- function(Xb, y_c, Nsel, MU, sigma) {</td></tr><tr id="tr-27"><td valign="top" align="left" rowspan="1" colspan="1" id="td-97" class="table-left table-vtop">n.mcmc &lt;- ncol(Xb)</td></tr><tr id="tr-28"><td valign="top" align="left" rowspan="1" colspan="1" id="td-98" class="table-left table-vtop">n.lines &lt;- nrow(Xb)</td></tr><tr id="tr-29"><td valign="top" align="left" rowspan="1" colspan="1" id="td-99" class="table-left table-vtop">losses &lt;- matrix(0, n.lines, n.mcmc)</td></tr><tr id="tr-30"><td valign="top" align="left" rowspan="1" colspan="1" id="td-100" class="table-left table-vtop">for(i in 1:n.mcmc) {</td></tr><tr id="tr-31"><td valign="top" align="left" rowspan="1" colspan="1" id="td-101" class="table-left table-vtop">mu1 &lt;- MU[i]</td></tr><tr id="tr-32"><td valign="top" align="left" rowspan="1" colspan="1" id="td-102" class="table-left table-vtop">mu2 &lt;- Xb[,i]</td></tr><tr id="tr-33"><td valign="top" align="left" rowspan="1" colspan="1" id="td-103" class="table-left table-vtop">sd &lt;- sigma[i]</td></tr><tr id="tr-34"><td valign="top" align="left" rowspan="1" colspan="1" id="td-104" class="table-left table-vtop">yz &lt;- (y_c - mu1) / sd # standardizing</td></tr><tr id="tr-35"><td valign="top" align="left" rowspan="1" colspan="1" id="td-105" class="table-left table-vtop">Z &lt;- 1 - pnorm(yz) # normalization factor</td></tr><tr id="tr-36"><td valign="top" align="left" rowspan="1" colspan="1" id="td-106" class="table-left table-vtop">dens &lt;- dnorm(yz, mean = 0, sd = 1) # density</td></tr><tr id="tr-37"><td valign="top" align="left" rowspan="1" colspan="1" id="td-107" class="table-left table-vtop">prob &lt;- pnorm(yz, mean = 0, sd = 1) # probability</td></tr><tr id="tr-38"><td valign="top" align="left" rowspan="1" colspan="1" id="td-108" class="table-left table-vtop">muT &lt;- mu1 + sd * (dens / (1 - prob))</td></tr><tr id="tr-39"><td valign="top" align="left" rowspan="1" colspan="1" id="td-109" class="table-left table-vtop"># posterior predictive function at each iter.</td></tr><tr id="tr-40"><td valign="top" align="left" rowspan="1" colspan="1" id="td-110" class="table-left table-vtop">yppdf &lt;- sapply(mu2, function(ypred) {</td></tr><tr id="tr-41"><td valign="top" align="left" rowspan="1" colspan="1" id="td-111" class="table-left table-vtop">rnorm(1, mean = ypred, sd = sd)})</td></tr><tr id="tr-42"><td valign="top" align="left" rowspan="1" colspan="1" id="td-112" class="table-left table-vtop">losses[,i] &lt;- -log(Z) + ((muT - yppdf)^2 –</td></tr><tr id="tr-43"><td valign="top" align="left" rowspan="1" colspan="1" id="td-113" class="table-left table-vtop">(muT - mu1)^2) / (2 * sd^2)</td></tr><tr id="tr-44"><td valign="top" align="left" rowspan="1" colspan="1" id="td-114" class="table-left table-vtop">}</td></tr><tr id="tr-45"><td valign="top" align="left" rowspan="1" colspan="1" id="td-115" class="table-left table-vtop">e.loss &lt;- apply(losses, 1, mean, na.rm = TRUE)</td></tr><tr id="tr-46"><td valign="top" align="left" rowspan="1" colspan="1" id="td-116" class="table-left table-vtop">selected &lt;- order(e.loss, decreasing = FALSE)[1:Nsel]</td></tr><tr id="tr-47"><td valign="top" align="left" rowspan="1" colspan="1" id="td-117" class="table-left table-vtop">return(selected)</td></tr><tr id="tr-48"><td valign="top" align="left" rowspan="1" colspan="1" id="td-118" class="table-left table-vtop">}</td></tr></tbody></table><table frame="border" id="table-6"><col width="100%" span="1" id="column-11"/><thead id="thead-6"><tr id="tr-49"><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-32" class="table-center table-vtop">Selection through Continuous Ranked Probability Score (CRPS)</th></tr></thead><tbody id="tbody-6"><tr id="tr-50"><td valign="top" align="left" rowspan="1" colspan="1" id="td-119" class="table-left table-vtop">selCRPS &lt;- function(Xb, y_c, Nsel, MU, sigma) {</td></tr><tr id="tr-51"><td valign="top" align="left" rowspan="1" colspan="1" id="td-120" class="table-left table-vtop">n.mcmc &lt;- ncol(Xb)</td></tr><tr id="tr-52"><td valign="top" align="left" rowspan="1" colspan="1" id="td-121" class="table-left table-vtop">n.lines &lt;- nrow(Xb)</td></tr><tr id="tr-53"><td valign="top" align="left" rowspan="1" colspan="1" id="td-122" class="table-left table-vtop">losses &lt;- matrix(0, n.lines, n.mcmc)</td></tr><tr id="tr-54"><td valign="top" align="left" rowspan="1" colspan="1" id="td-123" class="table-left table-vtop">for(i in 1:n.mcmc){</td></tr><tr id="tr-55"><td valign="top" align="left" rowspan="1" colspan="1" id="td-124" class="table-left table-vtop">mu1 &lt;- MU[i]</td></tr><tr id="tr-56"><td valign="top" align="left" rowspan="1" colspan="1" id="td-125" class="table-left table-vtop">mu2 &lt;- Xb[,i]</td></tr><tr id="tr-57"><td valign="top" align="left" rowspan="1" colspan="1" id="td-126" class="table-left table-vtop">sd &lt;- sigma[i]</td></tr><tr id="tr-58"><td valign="top" align="left" rowspan="1" colspan="1" id="td-127" class="table-left table-vtop">zz &lt;- (y_c-mu1)/sd</td></tr><tr id="tr-59"><td valign="top" align="left" rowspan="1" colspan="1" id="td-128" class="table-left table-vtop">muT &lt;- mu1 + sd*(dnorm(zz)/(1-pnorm(zz)))</td></tr><tr id="tr-60"><td valign="top" align="left" rowspan="1" colspan="1" id="td-129" class="table-left table-vtop"># posterior predictive function at each iter.</td></tr><tr id="tr-61"><td valign="top" align="left" rowspan="1" colspan="1" id="td-130" class="table-left table-vtop">yppdf &lt;- sapply(mu2, function(ypred) {</td></tr><tr id="tr-62"><td valign="top" align="left" rowspan="1" colspan="1" id="td-131" class="table-left table-vtop">rnorm(1, mean = ypred, sd = sd)})</td></tr><tr id="tr-63"><td valign="top" align="left" rowspan="1" colspan="1" id="td-132" class="table-left table-vtop">yz &lt;- (muT-yppdf) / sd # standardizing</td></tr><tr id="tr-64"><td valign="top" align="left" rowspan="1" colspan="1" id="td-133" class="table-left table-vtop">dens &lt;- dnorm(yz, mean = 0, sd = 1) # density</td></tr><tr id="tr-65"><td valign="top" align="left" rowspan="1" colspan="1" id="td-134" class="table-left table-vtop">prob &lt;- pnorm(yz, mean = 0, sd = 1) # probability</td></tr><tr id="tr-66"><td valign="top" align="left" rowspan="1" colspan="1" id="td-135" class="table-left table-vtop">losses[,i] &lt;- -1 * (sd * (1 / sqrt(pi) –</td></tr><tr id="tr-67"><td valign="top" align="left" rowspan="1" colspan="1" id="td-136" class="table-left table-vtop">2 * dens - yz * (2 * prob - 1)))</td></tr><tr id="tr-68"><td valign="top" align="left" rowspan="1" colspan="1" id="td-137" class="table-left table-vtop">}</td></tr><tr id="tr-69"><td valign="top" align="left" rowspan="1" colspan="1" id="td-138" class="table-left table-vtop">e.loss &lt;- apply(losses, 1, mean, na.rm = TRUE)</td></tr><tr id="tr-70"><td valign="top" align="left" rowspan="1" colspan="1" id="td-139" class="table-left table-vtop">selected &lt;- order(e.loss, decreasing = FALSE)[1:Nsel]</td></tr><tr id="tr-71"><td valign="top" align="left" rowspan="1" colspan="1" id="td-140" class="table-left table-vtop">return(selected)</td></tr><tr id="tr-72"><td valign="top" align="left" rowspan="1" colspan="1" id="td-141" class="table-left table-vtop">}</td></tr></tbody></table><table frame="border" id="table-7"><col width="100%" span="1" id="column-12"/><thead id="thead-7"><tr id="tr-73"><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-33" class="table-center table-vtop">Selection through Linear-Linear (Lin-Lin)</th></tr></thead><tbody id="tbody-7"><tr id="tr-74"><td valign="top" align="left" rowspan="1" colspan="1" id="td-142" class="table-left table-vtop">selLin &lt;- function(Xb, y_c, Nsel, MU, sigma, alpha) {</td></tr><tr id="tr-75"><td valign="top" align="left" rowspan="1" colspan="1" id="td-143" class="table-left table-vtop">n.mcmc &lt;- ncol(Xb)</td></tr><tr id="tr-76"><td valign="top" align="left" rowspan="1" colspan="1" id="td-144" class="table-left table-vtop">n.lines &lt;- nrow(Xb)</td></tr><tr id="tr-77"><td valign="top" align="left" rowspan="1" colspan="1" id="td-145" class="table-left table-vtop">losses &lt;- matrix(0, n.lines, n.mcmc)</td></tr><tr id="tr-78"><td valign="top" align="left" rowspan="1" colspan="1" id="td-146" class="table-left table-vtop">for(i in 1:n.mcmc){</td></tr><tr id="tr-79"><td valign="top" align="left" rowspan="1" colspan="1" id="td-147" class="table-left table-vtop">mu1 &lt;- MU[i]</td></tr><tr id="tr-80"><td valign="top" align="left" rowspan="1" colspan="1" id="td-148" class="table-left table-vtop">mu2 &lt;- Xb[,i]</td></tr><tr id="tr-81"><td valign="top" align="left" rowspan="1" colspan="1" id="td-149" class="table-left table-vtop">sd &lt;- sigma[i]</td></tr><tr id="tr-82"><td valign="top" align="left" rowspan="1" colspan="1" id="td-150" class="table-left table-vtop">z = (y_c-mu1)/sd</td></tr><tr id="tr-83"><td valign="top" align="left" rowspan="1" colspan="1" id="td-151" class="table-left table-vtop">prob &lt;- 1-pnorm(z)</td></tr><tr id="tr-84"><td valign="top" align="left" rowspan="1" colspan="1" id="td-152" class="table-left table-vtop">dens &lt;- dnorm(z)</td></tr><tr id="tr-85"><td valign="top" align="left" rowspan="1" colspan="1" id="td-153" class="table-left table-vtop">bias &lt;- sd*dens/prob</td></tr><tr id="tr-86"><td valign="top" align="left" rowspan="1" colspan="1" id="td-154" class="table-left table-vtop">muT &lt;- mu1 + bias</td></tr><tr id="tr-87"><td valign="top" align="left" rowspan="1" colspan="1" id="td-155" class="table-left table-vtop"># posterior predictive function at each iter.</td></tr><tr id="tr-88"><td valign="top" align="left" rowspan="1" colspan="1" id="td-156" class="table-left table-vtop">yppdf &lt;- sapply(mu2, function(ypred) {</td></tr><tr id="tr-89"><td valign="top" align="left" rowspan="1" colspan="1" id="td-157" class="table-left table-vtop">rnorm(1, mean = ypred, sd = sd)})</td></tr><tr id="tr-90"><td valign="top" align="left" rowspan="1" colspan="1" id="td-158" class="table-left table-vtop">losses[,i] &lt;- ((alpha-ifelse(yppdf &lt; muT, 1, 0))*(yppdf-muT))</td></tr><tr id="tr-91"><td valign="top" align="left" rowspan="1" colspan="1" id="td-159" class="table-left table-vtop">}</td></tr><tr id="tr-92"><td valign="top" align="left" rowspan="1" colspan="1" id="td-160" class="table-left table-vtop">e.loss &lt;- apply(losses, 1, mean, na.rm = TRUE)</td></tr><tr id="tr-93"><td valign="top" align="left" rowspan="1" colspan="1" id="td-161" class="table-left table-vtop">selected &lt;- order(e.loss, decreasing = FALSE)[1:Nsel]</td></tr><tr id="tr-94"><td valign="top" align="left" rowspan="1" colspan="1" id="td-162" class="table-left table-vtop">return(selected)</td></tr><tr id="tr-95"><td valign="top" align="left" rowspan="1" colspan="1" id="td-163" class="table-left table-vtop">}</td></tr></tbody></table><div class="table-foot"></div></div></div></div><div class="section" id="sec-27"><h3>R code for Multivariate Loss Function</h3><p id="p-86">The multivariate loss functions proposed were adapted to use MCMC samples from posterior distributions of parameters after a burn-in time period obtained from the MTM R package.</p><p id="p-87">Inputs of loss function in R language:</p><p id="p-88">• Xb: matrix of MCMC samples of the mean of the candidates of unobserved phenotype values. In columns are the lines, in rows are M MCMC samples (<span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-195.gif"/></noscript></span></span>).</p><p id="p-89">• MU: matrix of MCMC samples of population mean, in columns are the means for each trait, in rows MCMC samples (<span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/embed/mml-math-196.gif"/></noscript></span></span>).</p><p id="p-90">• K: matrix of MCMC samples of population var-covariance, in columns each component of covariance matrix, in rows MCMC samples (K).</p><p id="p-91">• y_c: vector of truncation points for each trait, length equal to the number of traits.</p><p id="p-92">• Nsel: number of the candidates to be selected.</p><p id="p-93">• tau: vector of length equal to number of traits. Each value of the vector indicates the proportion of selected individuals from the candidate set.</p><p id="p-94">Output:</p><p id="p-95">• id of selected lines with the minimums posterior expected losses.</p><div id="T4" class="table pos-anchor"><div class="table-inline"><table frame="border" id="table-8"><col width="100%" span="1" id="column-13"/><thead id="thead-8"><tr id="tr-96"><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-34" class="table-center table-vtop">Selection through Multivariate Kullback-Leibler (KL)</th></tr></thead><tbody id="tbody-8"><tr id="tr-97"><td valign="top" align="left" rowspan="1" colspan="1" id="td-164" class="table-left table-vtop">selKL &lt;- function(Xb, MU, K, y_c, Nsel) {</td></tr><tr id="tr-98"><td valign="top" align="left" rowspan="1" colspan="1" id="td-165" class="table-left table-vtop">Xb &lt;- as.matrix(Xb); MU &lt;- as.matrix(MU); K &lt;- as.matrix(K);</td></tr><tr id="tr-99"><td valign="top" align="left" rowspan="1" colspan="1" id="td-166" class="table-left table-vtop">n.traits &lt;- ncol(MU)</td></tr><tr id="tr-100"><td valign="top" align="left" rowspan="1" colspan="1" id="td-167" class="table-left table-vtop">n.lines &lt;- ncol(Xb)/n.traits</td></tr><tr id="tr-101"><td valign="top" align="left" rowspan="1" colspan="1" id="td-168" class="table-left table-vtop">n.mcmc &lt;- nrow(Xb)</td></tr><tr id="tr-102"><td valign="top" align="left" rowspan="1" colspan="1" id="td-169" class="table-left table-vtop">losses &lt;- matrix(0, nrow = n.lines, ncol = n.mcmc)</td></tr><tr id="tr-103"><td valign="top" align="left" rowspan="1" colspan="1" id="td-170" class="table-left table-vtop">Kall &lt;- alply(K, 1, function(V) as.matrix(nearPD(xpnd(unlist(V)))$mat))</td></tr><tr id="tr-104"><td valign="top" align="left" rowspan="1" colspan="1" id="td-171" class="table-left table-vtop">Kallinv &lt;- llply(Kall, solve)</td></tr><tr id="tr-105"><td valign="top" align="left" rowspan="1" colspan="1" id="td-172" class="table-left table-vtop">Xb &lt;- apply(Xb, 1, split, f = rep(1:n.traits, each = n.lines))</td></tr><tr id="tr-106"><td valign="top" align="left" rowspan="1" colspan="1" id="td-173" class="table-left table-vtop">for(i in 1:n.mcmc) {</td></tr><tr id="tr-107"><td valign="top" align="left" rowspan="1" colspan="1" id="td-174" class="table-left table-vtop">mu1 &lt;- as.vector(MU[i,])</td></tr><tr id="tr-108"><td valign="top" align="left" rowspan="1" colspan="1" id="td-175" class="table-left table-vtop">mu2 &lt;- do.call(cbind, Xb[[i]])</td></tr><tr id="tr-109"><td valign="top" align="left" rowspan="1" colspan="1" id="td-176" class="table-left table-vtop">K &lt;- Kall[[i]]</td></tr><tr id="tr-110"><td valign="top" align="left" rowspan="1" colspan="1" id="td-177" class="table-left table-vtop">Kinv &lt;- Kallinv[[i]]</td></tr><tr id="tr-111"><td valign="top" align="left" rowspan="1" colspan="1" id="td-178" class="table-left table-vtop">muS &lt;- as.vector(mtmvnorm(mean = mu1, sigma = K, lower = y_c,</td></tr><tr id="tr-112"><td valign="top" align="left" rowspan="1" colspan="1" id="td-179" class="table-left table-vtop">upper = rep(Inf, length(y_c)),</td></tr><tr id="tr-113"><td valign="top" align="left" rowspan="1" colspan="1" id="td-180" class="table-left table-vtop">doComputeVariance = FALSE)$tmean)</td></tr><tr id="tr-114"><td valign="top" align="left" rowspan="1" colspan="1" id="td-181" class="table-left table-vtop">Z = pmvnorm(lower = y_c, upper = Inf, mean = mu1, sigma = K)</td></tr><tr id="tr-115"><td valign="top" align="left" rowspan="1" colspan="1" id="td-182" class="table-left table-vtop"># posterior predictive distribution at each iteration.</td></tr><tr id="tr-116"><td valign="top" align="left" rowspan="1" colspan="1" id="td-183" class="table-left table-vtop">yppdf &lt;- as.matrix(t(apply(mu2, 1, function(ypred) {</td></tr><tr id="tr-117"><td valign="top" align="left" rowspan="1" colspan="1" id="td-184" class="table-left table-vtop">rmvnorm(1, mean = ypred, sigma = K)})))</td></tr><tr id="tr-118"><td valign="top" align="left" rowspan="1" colspan="1" id="td-185" class="table-left table-vtop">S &lt;- muS-mu1 # selection differential</td></tr><tr id="tr-119"><td valign="top" align="left" rowspan="1" colspan="1" id="td-186" class="table-left table-vtop">UKU &lt;- as.numeric(t(S)%*%Kinv%*%S)</td></tr><tr id="tr-120"><td valign="top" align="left" rowspan="1" colspan="1" id="td-187" class="table-left table-vtop">muSmu2 &lt;- sweep(yppdf,2,muS,’-’) # muS - mu2</td></tr><tr id="tr-121"><td valign="top" align="left" rowspan="1" colspan="1" id="td-188" class="table-left table-vtop">losses[,i] &lt;- as.vector(apply(muSmu2, 1, function(x) {</td></tr><tr id="tr-122"><td valign="top" align="left" rowspan="1" colspan="1" id="td-189" class="table-left table-vtop">0.5*(t(x)%*%Kinv%*%x-UKU)-log(Z) }))</td></tr><tr id="tr-123"><td valign="top" align="left" rowspan="1" colspan="1" id="td-190" class="table-left table-vtop">}</td></tr><tr id="tr-124"><td valign="top" align="left" rowspan="1" colspan="1" id="td-191" class="table-left table-vtop">e.loss &lt;- apply(losses, 1, mean, na.rm = TRUE)</td></tr><tr id="tr-125"><td valign="top" align="left" rowspan="1" colspan="1" id="td-192" class="table-left table-vtop">selected &lt;- order(e.loss, decreasing = FALSE)[1:Nsel]</td></tr><tr id="tr-126"><td valign="top" align="left" rowspan="1" colspan="1" id="td-193" class="table-left table-vtop">return(selected)</td></tr><tr id="tr-127"><td valign="top" align="left" rowspan="1" colspan="1" id="td-194" class="table-left table-vtop">}</td></tr></tbody></table><table frame="border" id="table-9"><col width="100%" span="1" id="column-14"/><thead id="thead-9"><tr id="tr-128"><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-35" class="table-center table-vtop">Selection through Energy Score (EnergyS)</th></tr></thead><tbody id="tbody-9"><tr id="tr-129"><td valign="top" align="left" rowspan="1" colspan="1" id="td-195" class="table-left table-vtop">selEnergyS &lt;- function(Xb, MU, K, y_c, Nsel) {</td></tr><tr id="tr-130"><td valign="top" align="left" rowspan="1" colspan="1" id="td-196" class="table-left table-vtop">Xb &lt;- as.matrix(Xb); MU &lt;- as.matrix(MU); K &lt;- as.matrix(K);</td></tr><tr id="tr-131"><td valign="top" align="left" rowspan="1" colspan="1" id="td-197" class="table-left table-vtop">n.traits &lt;- ncol(MU)</td></tr><tr id="tr-132"><td valign="top" align="left" rowspan="1" colspan="1" id="td-198" class="table-left table-vtop">n.lines &lt;- ncol(Xb)/n.traits</td></tr><tr id="tr-133"><td valign="top" align="left" rowspan="1" colspan="1" id="td-199" class="table-left table-vtop">n.mcmc &lt;- nrow(Xb)</td></tr><tr id="tr-134"><td valign="top" align="left" rowspan="1" colspan="1" id="td-200" class="table-left table-vtop">losses &lt;- matrix(0, nrow = n.lines, ncol = n.mcmc)</td></tr><tr id="tr-135"><td valign="top" align="left" rowspan="1" colspan="1" id="td-201" class="table-left table-vtop">Kall &lt;- alply(K, 1, function(V) as.matrix(nearPD(xpnd(unlist(V)))$mat))</td></tr><tr id="tr-136"><td valign="top" align="left" rowspan="1" colspan="1" id="td-202" class="table-left table-vtop">Xb &lt;- apply(Xb, 1, split, f = rep(1:n.traits, each = n.lines))</td></tr><tr id="tr-137"><td valign="top" align="left" rowspan="1" colspan="1" id="td-203" class="table-left table-vtop">for(i in 1:n.mcmc) {</td></tr><tr id="tr-138"><td valign="top" align="left" rowspan="1" colspan="1" id="td-204" class="table-left table-vtop">mu1 &lt;- as.vector(MU[i,])</td></tr><tr id="tr-139"><td valign="top" align="left" rowspan="1" colspan="1" id="td-205" class="table-left table-vtop">mu2 &lt;- do.call(cbind, Xb[[i]])</td></tr><tr id="tr-140"><td valign="top" align="left" rowspan="1" colspan="1" id="td-206" class="table-left table-vtop">K &lt;- Kall[[i]]</td></tr><tr id="tr-141"><td valign="top" align="left" rowspan="1" colspan="1" id="td-207" class="table-left table-vtop">muS &lt;- as.vector(mtmvnorm(mean = mu1, sigma = K, lower = y_c,</td></tr><tr id="tr-142"><td valign="top" align="left" rowspan="1" colspan="1" id="td-208" class="table-left table-vtop">upper = rep(Inf, length(y_c)),</td></tr><tr id="tr-143"><td valign="top" align="left" rowspan="1" colspan="1" id="td-209" class="table-left table-vtop">doComputeVariance = FALSE)$tmean)</td></tr><tr id="tr-144"><td valign="top" align="left" rowspan="1" colspan="1" id="td-210" class="table-left table-vtop"># posterior predictive distribution at each iteration.</td></tr><tr id="tr-145"><td valign="top" align="left" rowspan="1" colspan="1" id="td-211" class="table-left table-vtop">yppdf &lt;- as.matrix(t(apply(mu2, 1, function(ypred) {</td></tr><tr id="tr-146"><td valign="top" align="left" rowspan="1" colspan="1" id="td-212" class="table-left table-vtop">rmvnorm(1, mean = ypred, sigma = K)})))</td></tr><tr id="tr-147"><td valign="top" align="left" rowspan="1" colspan="1" id="td-213" class="table-left table-vtop">yppdf_<em>P</em> &lt; - as.matrix(t(apply(mu2, 1, function(ypred) {</td></tr><tr id="tr-148"><td valign="top" align="left" rowspan="1" colspan="1" id="td-214" class="table-left table-vtop">rmvnorm(1, mean = ypred, sigma = K)})))</td></tr><tr id="tr-149"><td valign="top" align="left" rowspan="1" colspan="1" id="td-215" class="table-left table-vtop">muSmu2 &lt;- sweep(yppdf,2,muS,’-’) # X-y</td></tr><tr id="tr-150"><td valign="top" align="left" rowspan="1" colspan="1" id="td-216" class="table-left table-vtop">mu2mu2 &lt;- yppdf-yppdf_p # X-X’</td></tr><tr id="tr-151"><td valign="top" align="left" rowspan="1" colspan="1" id="td-217" class="table-left table-vtop">xj_y &lt;- as.vector(apply(muSmu2, 1, function(x) sqrt(sum(x^2))))</td></tr><tr id="tr-152"><td valign="top" align="left" rowspan="1" colspan="1" id="td-218" class="table-left table-vtop">xj_xjp &lt;- as.vector(apply(mu2mu2,1,function(x) 0.5*sqrt(sum(x^2))))</td></tr><tr id="tr-153"><td valign="top" align="left" rowspan="1" colspan="1" id="td-219" class="table-left table-vtop">losses[,i] &lt;- xj_y - xj_xjp</td></tr><tr id="tr-154"><td valign="top" align="left" rowspan="1" colspan="1" id="td-220" class="table-left table-vtop">}</td></tr><tr id="tr-155"><td valign="top" align="left" rowspan="1" colspan="1" id="td-221" class="table-left table-vtop">e.loss &lt;- apply(losses, 1, mean, na.rm = TRUE)</td></tr><tr id="tr-156"><td valign="top" align="left" rowspan="1" colspan="1" id="td-222" class="table-left table-vtop">selected &lt;- order(e.loss, decreasing = FALSE)[1:Nsel]</td></tr><tr id="tr-157"><td valign="top" align="left" rowspan="1" colspan="1" id="td-223" class="table-left table-vtop">return(selected)</td></tr><tr id="tr-158"><td valign="top" align="left" rowspan="1" colspan="1" id="td-224" class="table-left table-vtop">}</td></tr></tbody></table><table frame="border" id="table-10"><col width="100%" span="1" id="column-15"/><thead id="thead-10"><tr id="tr-159"><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-36" class="table-center table-vtop">Selection through Multivariate Asymmetric Loss Function (MALF)</th></tr></thead><tbody id="tbody-10"><tr id="tr-160"><td valign="top" align="left" rowspan="1" colspan="1" id="td-225" class="table-left table-vtop">selMALF &lt;- function(Xb, MU, K, y_c, Nsel, tau) {</td></tr><tr id="tr-161"><td valign="top" align="left" rowspan="1" colspan="1" id="td-226" class="table-left table-vtop">Xb &lt;- as.matrix(Xb); MU &lt;- as.matrix(MU); K &lt;- as.matrix(K);</td></tr><tr id="tr-162"><td valign="top" align="left" rowspan="1" colspan="1" id="td-227" class="table-left table-vtop">n.traits &lt;- ncol(MU)</td></tr><tr id="tr-163"><td valign="top" align="left" rowspan="1" colspan="1" id="td-228" class="table-left table-vtop">n.lines &lt;- ncol(Xb)/n.traits</td></tr><tr id="tr-164"><td valign="top" align="left" rowspan="1" colspan="1" id="td-229" class="table-left table-vtop">n.mcmc &lt;- nrow(Xb)</td></tr><tr id="tr-165"><td valign="top" align="left" rowspan="1" colspan="1" id="td-230" class="table-left table-vtop">losses &lt;- matrix(0, nrow = n.lines, ncol = n.mcmc)</td></tr><tr id="tr-166"><td valign="top" align="left" rowspan="1" colspan="1" id="td-231" class="table-left table-vtop">Kall &lt;- alply(K, 1, function(V) as.matrix(nearPD(xpnd(unlist(V)))$mat))</td></tr><tr id="tr-167"><td valign="top" align="left" rowspan="1" colspan="1" id="td-232" class="table-left table-vtop">Xb &lt;- apply(Xb, 1, split, f = rep(1:n.traits, each = n.lines))</td></tr><tr id="tr-168"><td valign="top" align="left" rowspan="1" colspan="1" id="td-233" class="table-left table-vtop">for(i in 1:n.mcmc) {</td></tr><tr id="tr-169"><td valign="top" align="left" rowspan="1" colspan="1" id="td-234" class="table-left table-vtop">mu1 &lt;- as.vector(MU[i,])</td></tr><tr id="tr-170"><td valign="top" align="left" rowspan="1" colspan="1" id="td-235" class="table-left table-vtop">mu2 &lt;- do.call(cbind, Xb[[i]])</td></tr><tr id="tr-171"><td valign="top" align="left" rowspan="1" colspan="1" id="td-236" class="table-left table-vtop">K &lt;- Kall[[i]]</td></tr><tr id="tr-172"><td valign="top" align="left" rowspan="1" colspan="1" id="td-237" class="table-left table-vtop">muS &lt;- as.vector(mtmvnorm(mean = mu1, sigma = K, lower = y_c,</td></tr><tr id="tr-173"><td valign="top" align="left" rowspan="1" colspan="1" id="td-238" class="table-left table-vtop">upper = rep(Inf, length(y_c)),</td></tr><tr id="tr-174"><td valign="top" align="left" rowspan="1" colspan="1" id="td-239" class="table-left table-vtop">doComputeVariance = FALSE)$tmean)</td></tr><tr id="tr-175"><td valign="top" align="left" rowspan="1" colspan="1" id="td-240" class="table-left table-vtop"># posterior predictive distribution at each iteration.</td></tr><tr id="tr-176"><td valign="top" align="left" rowspan="1" colspan="1" id="td-241" class="table-left table-vtop">yppdf &lt;- as.matrix(t(apply(mu2, 1, function(ypred) {</td></tr><tr id="tr-177"><td valign="top" align="left" rowspan="1" colspan="1" id="td-242" class="table-left table-vtop">rmvnorm(1, mean = ypred, sigma = K)})))</td></tr><tr id="tr-178"><td valign="top" align="left" rowspan="1" colspan="1" id="td-243" class="table-left table-vtop">error &lt;- -1*sweep(yppdf,2,muS,’-’) # mu2 - muS</td></tr><tr id="tr-179"><td valign="top" align="left" rowspan="1" colspan="1" id="td-244" class="table-left table-vtop">abs.error &lt;- abs(error)</td></tr><tr id="tr-180"><td valign="top" align="left" rowspan="1" colspan="1" id="td-245" class="table-left table-vtop">sum.e &lt;- rowSums(abs.error)</td></tr><tr id="tr-181"><td valign="top" align="left" rowspan="1" colspan="1" id="td-246" class="table-left table-vtop">score &lt;- sum.e + t(error %*% tau)</td></tr><tr id="tr-182"><td valign="top" align="left" rowspan="1" colspan="1" id="td-247" class="table-left table-vtop">losses[,i] &lt;- score</td></tr><tr id="tr-183"><td valign="top" align="left" rowspan="1" colspan="1" id="td-248" class="table-left table-vtop">}</td></tr><tr id="tr-184"><td valign="top" align="left" rowspan="1" colspan="1" id="td-249" class="table-left table-vtop">e.loss &lt;- apply(losses, 1, mean, na.rm = TRUE)</td></tr><tr id="tr-185"><td valign="top" align="left" rowspan="1" colspan="1" id="td-250" class="table-left table-vtop">selected &lt;- order(e.loss, decreasing = FALSE)[1:Nsel]</td></tr><tr id="tr-186"><td valign="top" align="left" rowspan="1" colspan="1" id="td-251" class="table-left table-vtop">return(selected)</td></tr><tr id="tr-187"><td valign="top" align="left" rowspan="1" colspan="1" id="td-252" class="table-left table-vtop">}</td></tr></tbody></table><div class="table-foot"></div></div></div></div></div><div class="section app" id="app-2"><h2 class="">Appendix B</h2><div class="section" id="sec-28"><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bivariate representation of multivariate loss functions. A) Kullback-Leibler (KL), B) Energy Score (EnergyS), and C) Multivariate Asymmetric Loss Function (MALF). The base population is , where  (black lines) and . After truncation at  (yellow lines), the theoretical or desired mean of selected candidates is  (red lines). The candidates close to the region where each loss function is minimized will have minimum loss under the breeder&#x2019;s preferences; those lines are the ones to advance in a selection program." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-388110402" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Bivariate representation of multivariate loss functions. A) Kullback-Leibler (KL), B) Energy Score (EnergyS), and C) Multivariate Asymmetric Loss Function (MALF). The base population is &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-197&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, where &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-198&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (black lines) and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-199&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. After truncation at &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-200&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (yellow lines), the theoretical or desired mean of selected candidates is &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-201&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (red lines). The candidates close to the region where each loss function is minimized will have minimum loss under the breeder&#x2019;s preferences; those lines are the ones to advance in a selection program.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure B1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/3019/F9.medium.gif" width="440" height="146"/><noscript><img class="highwire-fragment fragment-image" alt="Figure B1." src="https://www.g3journal.org/content/ggg/8/9/3019/F9.medium.gif" width="440" height="146"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure B1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490907" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure B1.</span> <p id="p-96" class="first-child">Bivariate representation of multivariate loss functions. A) Kullback-Leibler (KL), B) Energy Score (EnergyS), and C) Multivariate Asymmetric Loss Function (MALF). The base population is <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F9/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F9/embed/mml-math-197.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F9/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F9/embed/mml-math-198.gif"/></noscript></span></span> (black lines) and <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F9/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F9/embed/mml-math-199.gif"/></noscript></span></span>. After truncation at <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F9/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F9/embed/mml-math-200.gif"/></noscript></span></span> (yellow lines), the theoretical or desired mean of selected candidates is <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F9/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F9/embed/mml-math-201.gif"/></noscript></span></span> (red lines). The candidates close to the region where each loss function is minimized will have minimum loss under the breeder’s preferences; those lines are the ones to advance in a selection program.</p><div class="sb-div caption-clear"></div></div></div><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results of the univariate simulation study. Standardized selection responses for breeding cycles 1 to 5 are illustrated in A), while in B) are cycles 5 to 10. In each selection cycle, the top 30% with minimum posterior expected losses were selected using the Kullback-Leibler loss function (KL), the Continuous Ranked Probability Score (CRPS), the Linear-Linear loss function (LinLin), and the standard method (Std). Selected lines were crossed at each cycle to recover the population size for upcoming selection cycles.  and  correspond to the population mean and the selection response, respectively, in cycle i;  and  are the population mean and the population standard deviation, respectively, in cycle 1. The black vertical lines indicate the standard error of  under 20 replications of the simulation study." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-388110402" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Results of the univariate simulation study. Standardized selection responses &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-202&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;for breeding cycles 1 to 5 are illustrated in A), while in B) are cycles 5 to 10. In each selection cycle, the top 30% with minimum posterior expected losses were selected using the Kullback-Leibler loss function (KL), the Continuous Ranked Probability Score (CRPS), the Linear-Linear loss function (LinLin), and the standard method (Std). Selected lines were crossed at each cycle to recover the population size for upcoming selection cycles. &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-203&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-204&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; correspond to the population mean and the selection response, respectively, in cycle i; &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-205&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-206&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; are the population mean and the population standard deviation, respectively, in cycle 1. The black vertical lines indicate the standard error of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-207&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; under 20 replications of the simulation study.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure B2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/3019/F10.medium.gif" width="440" height="159"/><noscript><img class="highwire-fragment fragment-image" alt="Figure B2." src="https://www.g3journal.org/content/ggg/8/9/3019/F10.medium.gif" width="440" height="159"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure B2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490808" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure B2.</span> <p id="p-97" class="first-child">Results of the univariate simulation study. Standardized selection responses <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F10/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F10/embed/mml-math-202.gif"/></noscript></span></span>for breeding cycles 1 to 5 are illustrated in A), while in B) are cycles 5 to 10. In each selection cycle, the top 30% with minimum posterior expected losses were selected using the Kullback-Leibler loss function (KL), the Continuous Ranked Probability Score (CRPS), the Linear-Linear loss function (LinLin), and the standard method (Std). Selected lines were crossed at each cycle to recover the population size for upcoming selection cycles. <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F10/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F10/embed/mml-math-203.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F10/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F10/embed/mml-math-204.gif"/></noscript></span></span> correspond to the population mean and the selection response, respectively, in cycle i; <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F10/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F10/embed/mml-math-205.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F10/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F10/embed/mml-math-206.gif"/></noscript></span></span> are the population mean and the population standard deviation, respectively, in cycle 1. The black vertical lines indicate the standard error of <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F10/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F10/embed/mml-math-207.gif"/></noscript></span></span> under 20 replications of the simulation study.</p><div class="sb-div caption-clear"></div></div></div><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results of the univariate simulation study. Scaled population variance  for breeding cycles 1 to 5 are illustrated in A), while in B) are cycles 5 to 10. In each selection cycle, the top 30% were selected using the Kullback-Leibler loss function (KL), the Continuous Ranked Probability Score (CRPS), the Linear-Linear loss function (LinLin), and lines selected under the standard method (Std). Selected lines were crossed at each cycle to recover the population size for the upcoming selection cycles.  and  are the population variance in cycle i and cycle 1, respectively. The black vertical lines indicate the standard error of  under 20 replications of the simulation study." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-388110402" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Results of the univariate simulation study. Scaled population variance &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-208&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for breeding cycles 1 to 5 are illustrated in A), while in B) are cycles 5 to 10. In each selection cycle, the top 30% were selected using the Kullback-Leibler loss function (KL), the Continuous Ranked Probability Score (CRPS), the Linear-Linear loss function (LinLin), and lines selected under the standard method (Std). Selected lines were crossed at each cycle to recover the population size for the upcoming selection cycles. &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-209&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-210&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; are the population variance in cycle i and cycle 1, respectively. The black vertical lines indicate the standard error of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-211&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; under 20 replications of the simulation study.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure B3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/3019/F11.medium.gif" width="440" height="160"/><noscript><img class="highwire-fragment fragment-image" alt="Figure B3." src="https://www.g3journal.org/content/ggg/8/9/3019/F11.medium.gif" width="440" height="160"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure B3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490761" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure B3.</span> <p id="p-98" class="first-child">Results of the univariate simulation study. Scaled population variance <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F11/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F11/embed/mml-math-208.gif"/></noscript></span></span> for breeding cycles 1 to 5 are illustrated in A), while in B) are cycles 5 to 10. In each selection cycle, the top 30% were selected using the Kullback-Leibler loss function (KL), the Continuous Ranked Probability Score (CRPS), the Linear-Linear loss function (LinLin), and lines selected under the standard method (Std). Selected lines were crossed at each cycle to recover the population size for the upcoming selection cycles. <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F11/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F11/embed/mml-math-209.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F11/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F11/embed/mml-math-210.gif"/></noscript></span></span> are the population variance in cycle i and cycle 1, respectively. The black vertical lines indicate the standard error of <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F11/embed/mml-math-211.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F11/embed/mml-math-211.gif"/></noscript></span></span> under 20 replications of the simulation study.</p><div class="sb-div caption-clear"></div></div></div><div id="F12" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results of the univariate simulation study at the 10th selection cycle for 30% of selected lines. A) Boxplots of the standardized selection response; B) boxplots of the scale population variance. The boxplots illustrate the mean (white dots) and median (black middle line) of 20 replications of the simulation study.  and  were defined in Figs. B2 and B3. Sub-indices refer to the 10th selection cycle." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-388110402" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Results of the univariate simulation study at the 10&lt;sup&gt;th&lt;/sup&gt; selection cycle for 30% of selected lines. A) Boxplots of the standardized selection response; B) boxplots of the scale population variance. The boxplots illustrate the mean (white dots) and median (black middle line) of 20 replications of the simulation study. &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-212&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-213&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; were defined in Figs. B2 and B3. Sub-indices refer to the 10&lt;sup&gt;th&lt;/sup&gt; selection cycle.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure B4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/3019/F12.medium.gif" width="440" height="216"/><noscript><img class="highwire-fragment fragment-image" alt="Figure B4." src="https://www.g3journal.org/content/ggg/8/9/3019/F12.medium.gif" width="440" height="216"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure B4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/3019/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490886" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure B4.</span> <p id="p-99" class="first-child">Results of the univariate simulation study at the 10<sup>th</sup> selection cycle for 30% of selected lines. A) Boxplots of the standardized selection response; B) boxplots of the scale population variance. The boxplots illustrate the mean (white dots) and median (black middle line) of 20 replications of the simulation study. <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F12/embed/mml-math-212.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F12/embed/mml-math-212.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F12/embed/mml-math-213.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3019/F12/embed/mml-math-213.gif"/></noscript></span></span> were defined in Figs. B2 and B3. Sub-indices refer to the 10<sup>th</sup> selection cycle.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-101"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-05-16"><span class="received-label">Received </span>May 16, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-07-16"><span class="accepted-label">Accepted </span>July 16, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Villar-Hernández <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3019.1" data-doi="10.3389/fgene.2016.00210"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sánchez</span>  <span class="cit-name-given-names">J. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Efficient Breeding by Genomic Mating.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2016.00210</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2016.00210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2016.00210&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3019.2" data-doi="10.1007/s10940-010-9098-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berk</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Asymmetric Loss Functions for Forecasting in Criminal Justice Settings.</span> <abbr class="cit-jnl-abbrev">J. Quant. Criminol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">107</span>–<span class="cit-lpage">123</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10940-010-9098-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BQuant.%2BCriminol.%26rft.volume%253D27%26rft.spage%253D107%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10940-010-9098-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10940-010-9098-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-8.9.3019.3" data-doi="10.1007/978-1-4020-6370-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bos</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caligari</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-source">Selection Methods in Plant Breeding</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin,</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-4020-6370-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSelection%2BMethods%2Bin%2BPlant%2BBreeding%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4020-6370-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4020-6370-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3019.4" data-doi="10.1371/journal.pone.0128570"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blondel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Onogi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ueda</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">A Ranking Approach to Genomic Selection.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0128570</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0128570</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0128570%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0128570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0128570&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3019.5" data-doi="10.1007/BF00222969"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brisbane</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Balancing selection response and rate of inbreeding by including genetic relationships in selection decisions.</span> <abbr class="cit-jnl-abbrev">TAG Theor. Appl. Genet. Theor. Angew. Genet.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">421</span>–<span class="cit-lpage">431</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00222969</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG%2BTheor.%2BAppl.%2BGenet.%2BTheor.%2BAngew.%2BGenet.%26rft.volume%253D91%26rft.spage%253D421%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00222969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00222969&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3019.6" data-doi="10.12702/1984-7033.v08n01a07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Costa</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mauro</span>  <span class="cit-name-given-names">A. O. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Unêda-Trevisoli</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arriel</span>  <span class="cit-name-given-names">N. H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bárbaro</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Analysis of direct and indirect selection and indices in soybean segregating populations.</span> <abbr class="cit-jnl-abbrev">Crop Breed. Appl. Biotechnol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">55</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.12702/1984-7033.v08n01a07</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BBreed.%2BAppl.%2BBiotechnol.%26rft.volume%253D8%26rft.spage%253D47%26rft_id%253Dinfo%253Adoi%252F10.12702%252F1984-7033.v08n01a07%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.12702/1984-7033.v08n01a07&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3019.7" data-doi="10.1016/j.tplants.2017.08.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic Selection in Plant Breeding: Methods, Models, and Perspectives.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">961</span>–<span class="cit-lpage">975</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2017.08.011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D22%26rft.spage%253D961%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2017.08.011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2017.08.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3019.8" data-doi="10.1007/s10463-006-0099-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dawid</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The geometry of proper scoring rules.</span> <abbr class="cit-jnl-abbrev">Ann. Inst. Stat. Math.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">93</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10463-006-0099-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BInst.%2BStat.%2BMath.%26rft.volume%253D59%26rft.spage%253D77%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10463-006-0099-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10463-006-0099-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-web" id="cit-8.9.3019.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grüneberg</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>. MTM (Multiple-Trait Model) package [WWW Document]. URL <a href="http://quantgen.github.io/MTM/vignette.html">http://quantgen.github.io/MTM/vignette.html</a> (accessed 10.25.17).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-other" id="cit-8.9.3019.10"><div class="cit-metadata unstructured">de los Campos, G., and Pérez Rodríguez, P., 2015. BGLR: Bayesian Generalized Linear Regression. R package version 1.0.5.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3019.11" data-doi="10.18637/jss.v040.i01"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eddelbuettel</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Francois</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Rcpp: Seamless R and C++ Integration.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.18637/jss.v040.i08</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BStat%2BSoftw%26rft.aulast%253DJara%26rft.auinit1%253DA.%26rft.volume%253D40%26rft.issue%253D12%26rft.spage%253D1%26rft.epage%253D32%26rft.atitle%253DUsing%2Bthe%2BR%2BPackage%2Bcrlmm%2Bfor%2BGenotyping%2Band%2BCopy%2BNumber%2BEstimation.%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v040.i01%26rft_id%253Dinfo%253Apmid%252F22523482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18637/jss.v040.i01&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22523482&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3019.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-book" id="cit-8.9.3019.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source">Introduction to Quantitative Genetics</span>, <span class="cit-edition">Ed. 4th</span>. <span class="cit-publ-name">Pearson</span>, <span class="cit-publ-loc">Harlow</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3019.13" data-doi="10.1198/016214506000001437"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gneiting</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raftery</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Strictly Proper Scoring Rules, Prediction, and Estimation.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">359</span>–<span class="cit-lpage">378</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1198/016214506000001437</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D102%26rft.spage%253D359%26rft_id%253Dinfo%253Adoi%252F10.1198%252F016214506000001437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/016214506000001437&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000244361000032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3019.14" data-doi="10.1534/genetics.116.197095"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The Predicted Cross Value for Genetic Introgression of Multiple Alleles.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">205</span>: <span class="cit-fpage">1409</span>–<span class="cit-lpage">1423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.197095</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.197095%26rft_id%253Dinfo%253Apmid%252F28122824%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNS80LzE0MDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMzAxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3019.15" data-doi="10.1175/1520-0434(2000)015&amp;lt;0559:DOTCRP&amp;gt;2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hersbach</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Decomposition of the Continuous Ranked Probability Score for Ensemble Prediction Systems.</span> <abbr class="cit-jnl-abbrev">Weather Forecast.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">570</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1175/1520-0434(2000)015&lt;0559:DOTCRP&gt;2.0.CO;2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWeather%2BForecast.%26rft.volume%253D15%26rft.spage%253D559%26rft_id%253Dinfo%253Adoi%252F10.1175%252F1520-0434%25282000%2529015%2526lt%253B0559%253ADOTCRP%2526gt%253B2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1175/1520-0434(2000)015&amp;lt;0559:DOTCRP&amp;gt;2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3019.16" data-doi="10.1162/REST_a_00215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Komunjer</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Owyang</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Multivariate Forecast Evaluation and Rationality Testing.</span> <abbr class="cit-jnl-abbrev">Rev. Econ. Stat.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">1066</span>–<span class="cit-lpage">1080</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1162/REST_a_00215</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRev.%2BEcon.%2BStat.%26rft.volume%253D94%26rft.spage%253D1066%26rft_id%253Dinfo%253Adoi%252F10.1162%252FREST_a_00215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1162/REST_a_00215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3019.17" data-doi="10.1214/aoms/1177729694"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kullback</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leibler</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1951</span> <span class="cit-article-title">On Information and Sufficiency.</span> <abbr class="cit-jnl-abbrev">Ann. Math. Stat.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">86</span> <a href="https://projecteuclid.org/euclid.aoms/1177729694">https://projecteuclid.org/euclid.aoms/1177729694</a>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1214/aoms/1177729694</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BMath.%2BStat.%26rft.volume%253D22%26rft.spage%253D79%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faoms%252F1177729694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aoms/1177729694&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3019.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMzAxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-book" id="cit-8.9.3019.19" data-doi="10.1002/9780470746684"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parmigiani</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inoue</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">Decision Theory: Principles and Approaches</span>, <span class="cit-publ-name">Wiley Series in Probability and Statistics</span> <span class="cit-comment">The Atrium, Southern Gate, Chichester, West Sussex, PO19 8SQ, United Kingdom</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/9780470746684</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDecision%2BTheory%253A%2BPrinciples%2Band%2BApproaches%26rft_id%253Dinfo%253Adoi%252F10.1002%252F9780470746684%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/9780470746684&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-8.9.3019.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Development Core Team</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>, <span class="cit-publ-name">R Found. Stat. Comput</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-confproc" id="cit-8.9.3019.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shepherd</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kinghorn</span>  <span class="cit-name-given-names">B. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span>. A tactical approach to the design of crossbreeding programs. Presented at the 6th World Congress on Genetics Applied to Livestock Production, Armidale, pp. <span class="cit-fpage">431</span>–<span class="cit-lpage">438</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3019.22" data-doi="10.1016/j.jspi.2013.03.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Székely</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rizzo</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Energy statistics: A class of statistics based on distances.</span> <abbr class="cit-jnl-abbrev">J. Stat. Plan. Inference</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">1249</span>–<span class="cit-lpage">1272</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jspi.2013.03.018</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BPlan.%2BInference%26rft.volume%253D143%26rft.spage%253D1249%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jspi.2013.03.018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jspi.2013.03.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3019.23" data-doi="10.3390/math2010012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsagris</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beneki</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hassani</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">On the folded normal distribution.</span> <abbr class="cit-jnl-abbrev">Mathematics</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">12</span>–<span class="cit-lpage">28</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3390/math2010012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMathematics%26rft.volume%253D2%26rft.spage%253D12%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fmath2010012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/math2010012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.9.3019.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Velu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction for grain zinc and iron concentrations in spring wheat.</span> <abbr class="cit-jnl-abbrev">TAG. Theoretical and Applied Genetics</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">1595</span>–<span class="cit-lpage">1605</span>. <span class="cit-comment">ISSN 1432–2242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2BApplied%2BGenetics%26rft.volume%253D129%26rft.spage%253D1595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3019.25" data-doi="10.1051/gse:19940504"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wray</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Increasing long-term response to selection.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">431</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-26-5-431</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D26%26rft.spage%253D431%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A19940504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:19940504&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994PL67000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/9/3019.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/9/3005" title="Maternal Proteins That Are Phosphoregulated upon Egg Activation Include Crucial Factors for Oogenesis, Egg Activation and Embryogenesis in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/9/3039" title="BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/9"><span>Volume 8 Issue 9, September 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/9.cover-source.jpg?itok=77yoxn8o" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/9/3019" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-BW9SsaQj_1CEmKbN0C--BX3PbOScWQV7y30o7eeZr6o" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/9/3019" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/9/3019" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/9/3019" class="active" data-icon-position="" data-hide-link-title="0">A Bayesian Decision Theory Approach for Genomic Selection</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/490720" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Yr4bEi5DWBN16FooBtJ5-aYRLuo10LJVY-oTG8HrVLI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/9/3019.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/9/3019" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-CyPnc7jVgaFyJCKA2rb7sfmp3PfsfgZbM3a6SiEDsCI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/9/3019" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/9/3019.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720--2" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Bayesian Decision Theory Approach for Genomic Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5862-0056" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bartolo de Jesús Villar-Hernández</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-1605-0817" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sergio Pérez-Elizalde</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="3">Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author" data-delta="4">Fernando H. Toledo</span> and <span class="highwire-citation-author" data-delta="5">Juan Burgueño</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3019-3037;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200430 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490720--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bartolo de Jesús Villar-Hernández </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Colegio de Postgraduados, Montecillos, Edo. de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bartolo%2Bde%2BJes%C3%BAs%2BVillar-Hern%C3%A1ndez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Villar-Hern%C3%A1ndez%20BD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABartolo%2Bde%2BJes%25C3%25BAs%2BVillar-Hern%25C3%25A1ndez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5862-0056" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bartolo de Jesús Villar-Hernández</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sergio Pérez-Elizalde </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Colegio de Postgraduados, Montecillos, Edo. de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sergio%2BP%C3%A9rez-Elizalde%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Elizalde%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASergio%2BP%25C3%25A9rez-Elizalde%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1605-0817" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sergio Pérez-Elizalde</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sergiop@colpos.mx" class="" data-icon-position="" data-hide-link-title="0">sergiop@colpos.mx</a>
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Colegio de Postgraduados, Montecillos, Edo. de México, México</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Biometrics and Statistics Unit, Genetic Resources Program, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, México, D.F, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sergiop@colpos.mx" class="" data-icon-position="" data-hide-link-title="0">sergiop@colpos.mx</a>
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Colegio de Postgraduados, Montecillos, Edo. de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Fernando H. Toledo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Biometrics and Statistics Unit, Genetic Resources Program, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, México, D.F, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BH.%2BToledo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Toledo%20FH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFernando%2BH.%2BToledo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Biometrics and Statistics Unit, Genetic Resources Program, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, México, D.F, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/9/3019" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/9/3019&amp;title=A%20Bayesian%20Decision%20Theory%20Approach%20for%20Genomic%20Selection" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/9/3019&amp;title=A%20Bayesian%20Decision%20Theory%20Approach%20for%20Genomic%20Selection" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/9/3019&amp;title=A%20Bayesian%20Decision%20Theory%20Approach%20for%20Genomic%20Selection" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/9/3019&amp;text=A%20Bayesian%20Decision%20Theory%20Approach%20for%20Genomic%20Selection" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/9/3019&amp;title=A%20Bayesian%20Decision%20Theory%20Approach%20for%20Genomic%20Selection" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/9/3019&amp;t=A%20Bayesian%20Decision%20Theory%20Approach%20for%20Genomic%20Selection" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/9/3019&amp;title=A%20Bayesian%20Decision%20Theory%20Approach%20for%20Genomic%20Selection" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/9/3019&amp;title=A%20Bayesian%20Decision%20Theory%20Approach%20for%20Genomic%20Selection" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/490720/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720--3" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Bayesian Decision Theory Approach for Genomic Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5862-0056" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bartolo de Jesús Villar-Hernández</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-1605-0817" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sergio Pérez-Elizalde</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="3">Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author" data-delta="4">Fernando H. Toledo</span> and <span class="highwire-citation-author" data-delta="5">Juan Burgueño</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3019-3037;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200430 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490720--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bartolo de Jesús Villar-Hernández </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Colegio de Postgraduados, Montecillos, Edo. de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bartolo%2Bde%2BJes%C3%BAs%2BVillar-Hern%C3%A1ndez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Villar-Hern%C3%A1ndez%20BD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABartolo%2Bde%2BJes%25C3%25BAs%2BVillar-Hern%25C3%25A1ndez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5862-0056" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bartolo de Jesús Villar-Hernández</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sergio Pérez-Elizalde </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Colegio de Postgraduados, Montecillos, Edo. de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sergio%2BP%C3%A9rez-Elizalde%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Elizalde%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASergio%2BP%25C3%25A9rez-Elizalde%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1605-0817" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sergio Pérez-Elizalde</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sergiop@colpos.mx" class="" data-icon-position="" data-hide-link-title="0">sergiop@colpos.mx</a>
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Colegio de Postgraduados, Montecillos, Edo. de México, México</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Biometrics and Statistics Unit, Genetic Resources Program, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, México, D.F, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sergiop@colpos.mx" class="" data-icon-position="" data-hide-link-title="0">sergiop@colpos.mx</a>
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Colegio de Postgraduados, Montecillos, Edo. de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Fernando H. Toledo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Biometrics and Statistics Unit, Genetic Resources Program, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, México, D.F, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BH.%2BToledo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Toledo%20FH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFernando%2BH.%2BToledo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Biometrics and Statistics Unit, Genetic Resources Program, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, México, D.F, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3019&amp;rft.epage=3037&amp;rft.atitle=A%20Bayesian%20Decision%20Theory%20Approach%20for%20Genomic%20Selection&amp;rft.volume=8&amp;rft.issue=9&amp;rft.date=2018-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Villar-Hern%C3%A1ndez%2C+Bartolo+de+Jes%C3%BAs&amp;rft.au=P%C3%A9rez-Elizalde%2C+Sergio&amp;rft.au=Crossa%2C+Jos%C3%A9&amp;rft.au=P%C3%A9rez-Rodr%C3%ADguez%2C+Paulino&amp;rft.au=Toledo%2C+Fernando+H.&amp;rft.au=Burgue%C3%B1o%2C+Juan"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/490720/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/490720/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/490720/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/490720/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/490720/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/490720/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/490720/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/490720/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/490720/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/490720/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/490720/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/490720/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/9/3019" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-22" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgements</a></li><li class="odd"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix A</a></li><li class="even"><a href="#app-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix B</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/9/3019.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/9/3019.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/8/9/3019.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
