<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/9/2889" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/9/2889.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/9/2889.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/9/2889.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/9/2889/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/9/2889.atom" />
<meta name="HW.pisa" content="ggg;8/9/2889" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy" />
<meta name="DC.Identifier" content="10.1534/g3.118.200311" />
<meta name="DC.Date" content="2018-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Norman et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://figshare.com/s/287c2c7f1623008487a5" />
<meta name="DC.Description" content="Genomic selection applied to plant breeding enables earlier estimates of a line’s performance and significant reductions in generation interval. Several factors affecting prediction accuracy should be well understood if breeders are to harness genomic selection to its full potential. We used a panel of 10,375 bread wheat ( Triticum aestivum ) lines genotyped with 18,101 SNP markers to investigate the effect and interaction of training set size, population structure and marker density on genomic prediction accuracy. Through assessing the effect of training set size we showed the rate at which prediction accuracy increases is slower beyond approximately 2,000 lines. The structure of the panel was assessed via principal component analysis and K-means clustering, and its effect on prediction accuracy was examined through a novel cross-validation analysis according to the K-means clusters and breeding cohorts. Here we showed that accuracy can be improved by increasing the diversity within the training set, particularly when relatedness between training and validation sets is low. The breeding cohort analysis revealed that traits with higher selection pressure (lower allelic diversity) can be more accurately predicted by including several previous cohorts in the training set. The effect of marker density and its interaction with population structure was assessed for marker subsets containing between 100 and 17,181 markers. This analysis showed that response to increased marker density is largest when using a diverse training set to predict between poorly related material. These findings represent a significant resource for plant breeders and contribute to the collective knowledge on the optimal structure of calibration panels for genomic prediction." />
<meta name="DC.Contributor" content="Adam Norman" />
<meta name="DC.Contributor" content="Julian Taylor" />
<meta name="DC.Contributor" content="James Edwards" />
<meta name="DC.Contributor" content="Haydn Kuchel" />
<meta name="article:published_time" content="2018-09-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genomic selection applied to plant breeding enables earlier estimates of a line’s performance and significant reductions in generation interval. Several factors affecting prediction accuracy should be well understood if breeders are to harness genomic selection to its full potential. We used a panel of 10,375 bread wheat (&lt;i&gt;Triticum aestivum&lt;/i&gt;) lines genotyped with 18,101 SNP markers to investigate the effect and interaction of training set size, population structure and marker density on genomic prediction accuracy. Through assessing the effect of training set size we showed the rate at which prediction accuracy increases is slower beyond approximately 2,000 lines. The structure of the panel was assessed via principal component analysis and K-means clustering, and its effect on prediction accuracy was examined through a novel cross-validation analysis according to the K-means clusters and breeding cohorts. Here we showed that accuracy can be improved by increasing the diversity within the training set, particularly when relatedness between training and validation sets is low. The breeding cohort analysis revealed that traits with higher selection pressure (lower allelic diversity) can be more accurately predicted by including several previous cohorts in the training set. The effect of marker density and its interaction with population structure was assessed for marker subsets containing between 100 and 17,181 markers. This analysis showed that response to increased marker density is largest when using a diverse training set to predict between poorly related material. These findings represent a significant resource for plant breeders and contribute to the collective knowledge on the optimal structure of calibration panels for genomic prediction.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/09/01" />
<meta name="citation_mjid" content="ggg;8/9/2889" />
<meta name="citation_id" content="8/9/2889" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/9/2889" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/9/2889.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/9/2889.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/9/2889.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200311" />
<meta name="citation_pmid" content="29970398" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="2889" />
<meta name="citation_lastpage" content="2899" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Adam Norman" />
<meta name="citation_author_institution" content="School of Agriculture, Food &amp; Wine, University of Adelaide" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0794-4907" />
<meta name="citation_author" content="Julian Taylor" />
<meta name="citation_author_institution" content="School of Agriculture, Food &amp; Wine, University of Adelaide" />
<meta name="citation_author" content="James Edwards" />
<meta name="citation_author_institution" content="School of Agriculture, Food &amp; Wine, University of Adelaide" />
<meta name="citation_author" content="Haydn Kuchel" />
<meta name="citation_author_institution" content="School of Agriculture, Food &amp; Wine, University of Adelaide" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=H. Auinger;citation_author=M. Schӧnleben;citation_author=C. Lehermeier;citation_author=M. Schmidt;citation_author=V. Korzun;citation_title=Model training across multiple breeding cycles significantly improves genomic prediction accuracy in rye (Secale cereale L.).;citation_pages=2043-2053;citation_volume=129;citation_year=2016;citation_doi=10.1007/s00122-016-2756-5" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. Ben Hassen;citation_author=T. Cao;citation_author=J. Bartholomé;citation_author=G. Orasen;citation_author=C. Colombi;citation_title=Rice diversity panel provides accurate genomic predictions for complex traits in the progenies of biparental crosses involving members of the panel.;citation_pages=417-435;citation_volume=131;citation_year=2018;citation_doi=10.1007/s00122-017-3011-4" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=D. Bennett;citation_author=A. Izanloo;citation_author=J. Edwards;citation_author=H. Kuchel;citation_author=K. Chalmers;citation_title=Identification of novel quantitative trait loci for days to ear emergence and flag leaf glaucousness in a bread wheat (Triticum aestivum L.) population adapted to southern Australian conditions.;citation_pages=697-711;citation_volume=124;citation_year=2012;citation_issue=4;citation_pmid=22045047;citation_doi=10.1007/s00122-011-1740-3" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=D. Bennett;citation_author=A. Izanloo;citation_author=M. Reynolds;citation_author=H. Kuchel;citation_author=P. Langridge;citation_title=Genetic dissection of grain yield and physical grain quality in bread wheat (Triticum aestivum L.) under water-limited environments.;citation_pages=255-271;citation_volume=125;citation_year=2012;citation_issue=2;citation_pmid=22374139;citation_doi=10.1007/s00122-012-1831-9" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. Bentley;citation_author=M. Scutari;citation_author=N. Gosman;citation_author=S. Faure;citation_author=F. Bedford;citation_title=Applying association mapping and genomic selection to the dissection of key traits in elite European wheat.;citation_pages=2619-2633;citation_volume=127;citation_year=2014;citation_doi=10.1007/s00122-014-2403-y" />
<meta name="citation_reference" content="citation_title=ASReml-R reference manual;citation_author=D. Butler;citation_author=B. Cullis;citation_author=A. Gilmour;citation_author=B. Gogel;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Crop Pasture Sci.;citation_author=K. Cane;citation_author=H. Eagles;citation_author=D. Laurie;citation_author=B. Trevaskis;citation_author=N. Vallance;citation_title=Ppd-B1 and Ppd-D1 and their effects in southern Australian wheat.;citation_pages=100-114;citation_volume=64;citation_year=2013;citation_doi=10.1071/CP13086" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=D. Jarquín;citation_author=J. Franco;citation_author=P. Pérez-Rodríguez;citation_author=J. Burgueño;citation_title=Genomic prediction of gene bank wheat landraces. G3: Genes, Genomes;citation_pages=1819-1834;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Daetwyler;citation_author=M. Calus;citation_author=R. Pong-Wong;citation_author=G. de los Campos;citation_author=J. Hickey;citation_title=Genomic prediction in animals and plants: simulation of data, validation, reporting, and benchmarking.;citation_volume=193;citation_year=2013;citation_pmid=23222650;citation_doi=10.1534/genetics.112.147983" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=H. Daetwyler;citation_author=U. Bansal;citation_author=H. Bariana;citation_author=M. Hayden;citation_author=B. Hayes;citation_title=Genomic prediction for rust resistance in diverse wheat landraces.;citation_pages=1795-1803;citation_volume=127;citation_year=2014;citation_pmid=24965887;citation_doi=10.1007/s00122-014-2341-8" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. Hickey;citation_author=R. Pong-Wong;citation_author=H. Daetwyler;citation_author=M. Calus;citation_title=Whole-genome regression and prediction methods applied to plant and animal breeding.;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=Z. Desta;citation_author=R. Ortiz;citation_title=Genomic selection: genome-wide prediction in plant improvement.;citation_pages=592-601;citation_volume=19;citation_year=2014;citation_pmid=24970707;citation_doi=10.1016/j.tplants.2014.05.006" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=S. Forni;citation_author=I. Aguilar;citation_author=I. Misztal;citation_title=Different genomic relationship matrices for single-step analysis using phenotypic, pedigree and genomic information.;citation_pages=1-1;citation_volume=43;citation_year=2011;citation_pmid=21208445;citation_doi=10.1186/1297-9686-43-1" />
<meta name="citation_reference" content="citation_journal_title=Comput. Stat. Data Anal.;citation_author=A. Gilmour;citation_title=Mixed model regression mapping for QTL detection in experimental crosses.;citation_pages=3749-3764;citation_volume=51;citation_year=2007;citation_doi=10.1016/j.csda.2006.12.031" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=A. Gilmour;citation_author=R. Thompson;citation_author=B. Cullis;citation_title=Average information REML: an efficient algorithm for variance parameter estimation in linear mixed models.;citation_pages=1440-1450;citation_volume=51;citation_year=1995;citation_doi=10.2307/2533274" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Biol. Environ. Stat.;citation_author=A. Gilmour;citation_author=B. Cullis;citation_author=A. Verbyla;citation_title=Accounting for natural and extraneous variation in the analysis of field experiments.;citation_pages=269-293;citation_volume=2;citation_year=1997;citation_doi=10.2307/1400446" />
<meta name="citation_reference" content="citation_title=Theor. Appl. Genet;citation_author=G. Gorjanc;citation_author=R. C. Gaynor;citation_author=J. M. Hickey;citation_title=Optimal cross selection for long-term genetic gain in two-part programs with rapid recurrent genomic selection.;citation_year=2018;citation_doi=10.1007/s00122&amp;ndash;018&amp;ndash;3125&amp;ndash;3" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=S. He;citation_author=J. Reif;citation_author=V. Korzun;citation_author=R. Bothe;citation_author=E. Ebmeyer;citation_title=Genome-wide mapping and prediction suggests presence of local epistasis in a vast elite winter wheat populations adapted to central europe.;citation_pages=635-647;citation_volume=130;citation_year=2017;citation_doi=10.1007/s00122-016-2840-x" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. Heffner;citation_author=M. Sorrells;citation_author=J. Jannink;citation_title=Genomic Selection for Crop Improvement;citation_pages=1-12;citation_volume=49;citation_year=2009;citation_issue=1;citation_doi=10.2135/cropsci2008.08.0512" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=E. Heffner;citation_author=J. Jannink;citation_author=H. Iwata;citation_author=E. Souza;citation_author=M. Sorrells;citation_title=Genomic selection accuracy for grain quality traits in biparental wheat populations.;citation_pages=2597-2606;citation_volume=51;citation_year=2011;citation_doi=10.2135/cropsci2011.05.0253" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=E. Heffner;citation_author=J. Jannink;citation_author=M. Sorrells;citation_title=Genomic selection accuracy using multifamily prediction models in a wheat breeding program.;citation_pages=65-75;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2010.12.0029" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=C. R. Henderson;citation_title=Estimation of variance and covariance components.;citation_pages=226-252;citation_volume=9;citation_year=1953;citation_doi=10.2307/3001853" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=H. Yang;citation_author=M. Sorrells;citation_author=J. Jannink;citation_title=Genomic selection in plant breeding: a comparison of models.;citation_pages=146-160;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0297" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Hickey;citation_author=S. Dreisigacker;citation_author=J. Crossa;citation_author=S. Hearne;citation_author=R. Babu;citation_title=Evaluation of genomic selection training population designs and genotyping strategies in plant breeding programs using simulation.;citation_pages=1476-1488;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2013.03.0195" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Biotechnol J;citation_author=B. Huang;citation_author=A. George;citation_author=K. Forrest;citation_author=A. Kilian;citation_author=M. Hayden;citation_title=A multiparent advanced generation inter-cross population for genetic analysis in wheat.;citation_pages=826-839;citation_volume=10;citation_year=2012;citation_issue=7;citation_pmid=22594629;citation_doi=10.1111/j.1467-7652.2012.00702.x" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=X. Huang;citation_author=S. Cloutier;citation_author=L. Lycar;citation_author=N. Radovanovic;citation_author=D. Humphreys;citation_title=Molecular detection of QTLs for agronomic and quality traits in a doubled haploid population derived from two Canadian wheats (Triticum aestivum L.).;citation_pages=753-766;citation_volume=113;citation_year=2006;citation_issue=4;citation_pmid=16838135;citation_doi=10.1007/s00122-006-0346-7" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. Isidro;citation_author=J. Jannink;citation_author=D. Akdemir;citation_author=J. Poland;citation_author=N. Heslot;citation_title=Training set optimization under population structure in genomic selection.;citation_pages=145-158;citation_volume=128;citation_year=2015;citation_pmid=25367380;citation_doi=10.1007/s00122-014-2418-4" />
<meta name="citation_reference" content="citation_journal_title=Brief. Funct. Genomics;citation_author=J. Jannink;citation_author=A. Lorenz;citation_author=H. Iwata;citation_title=Genomic selection in plant breeding: from theory to practice.;citation_volume=9;citation_year=2010;citation_pmid=20156985;citation_doi=10.1093/bfgp/elq001" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=H. Kuchel;citation_author=K. Williams;citation_author=P. Langridge;citation_author=H. Eagles;citation_author=S. Jefferies;citation_title=Genetic dissection of grain yield in bread wheat. I. QTL analysis.;citation_pages=1029-1041;citation_volume=115;citation_year=2007;citation_issue=8;citation_pmid=17713755;citation_doi=10.1007/s00122-007-0629-7" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=Z. Liu;citation_author=F. Seefried;citation_author=F. Reinhardt;citation_author=S. Rensing;citation_author=G. Thaller;citation_title=Impacts of both reference population size and inclusion of a residual polygenic effect on the accuracy of genomic prediction.;citation_pages=19-19;citation_volume=43;citation_year=2011;citation_pmid=21586131;citation_doi=10.1186/1297-9686-43-19" />
<meta name="citation_reference" content="Lund M, Van den Berg I, Ma P, Brøndum R, Su G (2016) How to improve genomic predictions in small dairy cattle populations. Animal 10(6):1042–1049" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=L. Maphosa;citation_author=P. Langridge;citation_author=H. Taylor;citation_author=B. Parent;citation_author=L. Emebiri;citation_title=Genetic control of grain yield and grain physical characteristics in a bread wheat population grown under a range of environmental conditions.;citation_pages=1607-1624;citation_volume=127;citation_year=2014;citation_pmid=24865506;citation_doi=10.1007/s00122-014-2322-y" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Meuwissen;citation_author=B. Hayes;citation_author=M. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=S. Michel;citation_author=C. Ametz;citation_author=H. Gungor;citation_author=B. Akgöl;citation_author=D. Epure;citation_title=Genomic assisted selection for enhancing line breeding: merging genomic and phenotypic selection in winter wheat breeding programs with preliminary yield trials.;citation_pages=363-376;citation_volume=130;citation_year=2017;citation_doi=10.1007/s00122-016-2818-8" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=W. Muir;citation_title=Comparison of genomic and traditional BLUP-estimated breeding value accuracy and selection response under alternative trait and genomic parameters.;citation_pages=342-355;citation_volume=124;citation_year=2007;citation_issue=6;citation_pmid=18076471;citation_doi=10.1111/j.1439-0388.2007.00700.x" />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot. (Lond.);citation_author=A. Nakaya;citation_author=S. Isobe;citation_title=Will genomic selection be a practical method for plant breeding?;citation_volume=110;citation_year=2012;citation_pmid=22645117;citation_doi=10.1093/aob/mcs109" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. Norman;citation_author=J. Taylor;citation_author=E. Tanaka;citation_author=P. Telfer;citation_author=J. Edwards;citation_title=Increased genomic prediction accuracy in wheat breeding using a large Australian panel.;citation_pages=2543-2555;citation_volume=130;citation_year=2017;citation_doi=10.1007/s00122-017-2975-4" />
<meta name="citation_reference" content="citation_journal_title=Biometrika;citation_journal_abbrev=Biometrika;citation_author=H. Patterson;citation_author=R. Thompson;citation_title=Recovery of inter-block information when block sizes are unequal;citation_pages=545-554;citation_volume=58;citation_year=1971;citation_issue=3;citation_doi=10.1093/biomet/58.3.545" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=N. Patterson;citation_author=A. Price;citation_author=D. Reich;citation_title=Population structure and eigenanalysis.;citation_pages=e190-e190;citation_volume=2;citation_year=2006;citation_issue=12;citation_pmid=17194218;citation_doi=10.1371/journal.pgen.0020190" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Poland;citation_author=J. Endelman;citation_author=J. Dawson;citation_author=J. Rutkoski;citation_author=S. Wu;citation_title=Genomic selection in wheat breeding using genotyping-by-sequencing.;citation_pages=103-113;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.06.0006" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=A. Price;citation_author=N. Zaitlen;citation_author=D. Reich;citation_author=N. Patterson;citation_title=New approaches to population stratification in genome-wide association studies.;citation_pages=459-463;citation_volume=11;citation_year=2010;citation_issue=7;citation_pmid=20548291;citation_doi=10.1038/nrg2813" />
<meta name="citation_reference" content="R Core Team, (2017) R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna, Austria, URL https://www.R-project.org/" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=J. Snape;citation_author=R. Sarma;citation_author=S. Quarrie;citation_author=L. Fish;citation_author=G. Galiba;citation_title=Mapping genes for flowering time and frost tolerance in cereals using precise genetic stocks.;citation_pages=309-315;citation_volume=120;citation_year=2001;citation_doi=10.1023/A:1017541505152" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=T. R. Solberg;citation_author=A. K. Sonesson;citation_author=J. A. Woolliams;citation_author=T. H. Meuwissen;citation_title=Genomic selection using different marker types and densities.;citation_volume=86;citation_year=2008;citation_pmid=18407980;citation_doi=10.2527/jas.2007-0010" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=X. Sun;citation_author=K. Wu;citation_author=Y. Zhao;citation_author=F. Kong;citation_author=G. Han;citation_title=QTL analysis of kernel shape and weight using recombinant inbred lines in wheat.;citation_pages=615-624;citation_volume=165;citation_year=2009;citation_doi=10.1007/s10681-008-9794-2" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Weed Res.;citation_author=J. Zadoks;citation_author=T. Chang;citation_author=C. Konzak;citation_title=A decimal code for the growth stages of cereals.;citation_pages=415-421;citation_volume=14;citation_year=1974;citation_doi=10.1111/j.1365-3180.1974.tb01084.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. Zhang;citation_author=M. Erbe;citation_author=J. He;citation_author=U. Ober;citation_author=N. Gao;citation_title=Accuracy of whole-genome prediction using a genetic architecture-enhanced variance-covariance matrix. G3: Genes, Genomes;citation_pages=615-627;citation_volume=5;citation_year=2015" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9.cover-source.jpg" />
<meta name="twitter:description" content="Genomic selection applied to plant breeding enables earlier estimates of a line’s performance and significant reductions in generation interval. Several factors affecting prediction accuracy should be well understood if breeders are to harness genomic selection to its full potential. We used a panel of 10,375 bread wheat ( Triticum aestivum ) lines genotyped with 18,101 SNP markers to investigate the effect and interaction of training set size, population structure and marker density on genomic prediction accuracy. Through assessing the effect of training set size we showed the rate at which prediction accuracy increases is slower beyond approximately 2,000 lines. The structure of the panel was assessed via principal component analysis and K-means clustering, and its effect on prediction accuracy was examined through a novel cross-validation analysis according to the K-means clusters and breeding cohorts. Here we showed that accuracy can be improved by increasing the diversity within the training set, particularly when relatedness between training and validation sets is low. The breeding cohort analysis revealed that traits with higher selection pressure (lower allelic diversity) can be more accurately predicted by including several previous cohorts in the training set. The effect of marker density and its interaction with population structure was assessed for marker subsets containing between 100 and 17,181 markers. This analysis showed that response to increased marker density is largest when using a diverse training set to predict between poorly related material. These findings represent a significant resource for plant breeders and contribute to the collective knowledge on the optimal structure of calibration panels for genomic prediction." />
<meta name="og-title" property="og:title" content="Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/9/2889" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genomic selection applied to plant breeding enables earlier estimates of a line’s performance and significant reductions in generation interval. Several factors affecting prediction accuracy should be well understood if breeders are to harness genomic selection to its full potential. We used a panel of 10,375 bread wheat ( Triticum aestivum ) lines genotyped with 18,101 SNP markers to investigate the effect and interaction of training set size, population structure and marker density on genomic prediction accuracy. Through assessing the effect of training set size we showed the rate at which prediction accuracy increases is slower beyond approximately 2,000 lines. The structure of the panel was assessed via principal component analysis and K-means clustering, and its effect on prediction accuracy was examined through a novel cross-validation analysis according to the K-means clusters and breeding cohorts. Here we showed that accuracy can be improved by increasing the diversity within the training set, particularly when relatedness between training and validation sets is low. The breeding cohort analysis revealed that traits with higher selection pressure (lower allelic diversity) can be more accurately predicted by including several previous cohorts in the training set. The effect of marker density and its interaction with population structure was assessed for marker subsets containing between 100 and 17,181 markers. This analysis showed that response to increased marker density is largest when using a diverse training set to predict between poorly related material. These findings represent a significant resource for plant breeders and contribute to the collective knowledge on the optimal structure of calibration panels for genomic prediction." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9.cover-source.jpg" />
    <title>Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"QJryeaJE4WiuOCwRthXBWSpNuawij7HCQ3MiF5t0UPQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490724","apath":"\/ggg\/8\/9\/2889.atom","pisa":"ggg;8\/9\/2889","ac":{"\/ggg\/8\/9\/2889.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/9\/2889.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/9\/2889"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/9\/2889"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/9\/2889":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229RKwDdNqjrIQirqkC18Wo9OfV+hOoRR1LlCXZpilkvrGESock9igcGUCKUmStpngh0zyoEorj6uMVNYnFP8duFMCxBTEahC5+zu2KI2e0zNOSS4BNN0FxISXGll9xiPUM6GYD9goFq+nBwlwuNg\\\\\\\/ceoOfZkxx0+ILllZXVujMYY1pVKunyX49+ejIkbq+S8xy5aIobdfgAOv5vFFDVSiE3PrHeG11E0q+q844IeTlu0oFkcrdEX0kSn8nWyIO7O6uZaIcmLU\\\\\\\/t9avA+HgGmnEM122Fatq6u2YnnZk4L8Kda\\\\\\\/V6EutRi+qQ5DMu7mRiLRJD3+VraXuP9S0\\\\\\\/GRiXrO50y3RXnSnj8rQ5FVWTrfURXr7mL7mncavV5v8\\\\\\\/lgreK80F6v5dQ0NODpOmealfuE1CdpFHvw9OxxJmtUUWKE8pKcA3u0kqUlKJTYBoU75C2Akdzwn9KzZskRKr2RRqJyPWOlEOeQG9Pa16tTZY4bOZRkAi+oOhWlsLjHx2y345g69BbAYUcGDa6wzF2solzPuwJ84JyP5OaOMpmAHe+FJAapnvmIWwv2CunogVnriZXn+WAL794jKvY++xr3fKN3kT0rUq9mzBg2OvJhh0Nprv8Ah6GSszKiIOuIeayn8H4mufR64bUlxD7mnJ9hTa+B\\\\\\\/G641ZDjUBZZmZGnATU2XUmeXUdRK38Yj6YRA1YmnJyAzKVR4UmQPCviWz+gMrYAtf+M1vuZA+MgRRe2kUE1LiW+S3RHiuOlxyBEG9JkscKJcPdbvAs4Q6Agn8GuNlYjXR33FP4gFg55+\\\\\\\/k8WD4rLw0o6v\\\\\\\/CDwDQ0KgxUQr3xtqGVDYS\\\\\\\/68XmPtVLIBr2RORKsRM\\\\\\\/AL3MTLxVFmAiXzPgPQgQYg2cJXCJYvKa5s4WayKakk\\\\\\\/nrA+HZARqAQw+rRNjVmrctIn9RIK3YI2O38vIrJrVPCQvZ7vcw9\\\\\\\/Fssozcfugl5SEpmIKmnDzviYNxJMRyphRiZW02xWOJU95EKUFnMHRW4+JgxslEbe68DLz57CAx7t25UXY9lu9KA1UlajjMtDqPV7Rw3CyMc3Zf2sOBTlqhS036tczOL71Ny3msEfjH+FMbrfQyLYgGJW2I2RsZ0236aYaUC245xBpqK0xo0nJ7HDqW7WkI0ussgVRu0QremP0f5BIYhY9oor4d7Nv\\\\\\\/m8Cy6UelOEnbuiKC7RayAtjJHpRKyf3U2SbDmIMyXaK3DQkelfQVmIuStio2x5mFSx\\\\\\\/MQayO\\\\\\\/sYpLstyBAt3\\\\\\\/Lk99+ZTyp+BUoQDh+mwNd\\\u0022,\\\u0022iv\\\u0022:\\\u00222hHoI3v1Ifkqm0MFYwT7+w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223bde7a79bbe34939e618d79c1951e50df247b5a9bb648596ef95c0078e45b9c0\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022hKuMei+4DbGV87EofMyJipG8YvhPdl7njCPWO1F2JnUf6umr+hhp4uZh9K3iu2rG+rrmbhH5Hv9bexQJj1JeP0fjNmk9ecbX\\\\\\\/xew9vjqYGTXtI2piy52mCjA\\\\\\\/BZQV4h\\\\\\\/Y9KmdYFug8Z8tEzmp\\\\\\\/uE2B18ApImqKp2aO+freswQ7XNqV54b+aTeYr3FRObq5x7Bs5UrC7zS6hGy3R\\\\\\\/UahEOH1MuWx5uprN1MvAVTLOztYicqnQYf5z88tuxrGVSbr\\\\\\\/D9PKNqSm7i+9YlcKpN8EoLITgpI2RKsG20EKbUDGLHH3mQ78fwsrO1mcM9scVlLyk4cvBERFrxuW8MGItngqZkXYjAfTiaOfwmAVhaJ6jbS8zZuWrKtlX7qaetZYvE4HT1ygY3W5oCDfg2p1aSqyHbosQCrQrXUdp0meMDCipDNPOu5f+02i95aDwEnNo23xvO+MIhfkU8EVqz\\\\\\\/\\\\\\\/Yc99eeHnkOyv3dbhstL9XdG9jq1aWgaasm5t1bpJ86Z\\\\\\\/sEouKRa7V86xV+eqAF3PC1kN\\\\\\\/DGtlUJX274WRaQZL18VGiGgydRerv2MuAVYRBVHBPX+Gs4gtdt6+iUaF8vihYvEJhtXoE\\\\\\\/WgZoHwHTfoFNjMyuB+\\\\\\\/ebIJJ\\\\\\\/ZBfcusYG3HW9Lq6cAGn\\\\\\\/xSxzyKtpYk+fb1towDLoM8\\\\\\\/KGIkI0BL0pdMX1RbKpmiwN6dpErs7VQGymREr+7pjFXSX7jBV77P4vZtgkuDzFMuOegpGN51kpaVyz\\\\\\\/jLEetIObbt+Fvn8Jcq8G0K8ayYLEze+Dk2Y\\\\\\\/1HHTXe9HFb9rGm1afsJt6c+II9ib7an8gw+TPPLSdMTdZdOi1dh49Qtf7zKeNS8XUswswTA9Y166tzxnfzYBrJTNtjbUor4MeQBMck4IjQ\\\\\\\/qs6HVyr7kOzvruSqec2VbBsUEjshLd75TVJiuuEoNQ+TyV0HO4lUke1RXTDzGt+nySrTir68lgGxMsi3iRMLKu2NDi\\\\\\\/g+Omrl7nH366Ris00VB0aR6cyENtVEkCfCqiMm+LZr7WB1ocG0SVRR7uBuWZyz3W7lfnlAFvvrypOLgs0RX6EONlEa\\\\\\\/bTyaB2tU7SSkOGW8oGLe75L+7RFMw0gpHh6SpJ0kIWOU2je5GhzfRdg0cqBpTdgQ2RlVtNKAxChYR3YJaiIGp1resftgftDfcLlXnjbGFQHpGJ1N7EugdDeIXJ1AbgkyjwIeBuwDtSlsnFNRzULTQQhF6tcwvtkFrx6vsQehBXYfv8z2D\\\\\\\/I1f6+kwlat4nCJXJPpSQmVY6Fle4jLGTQwKLVwOlAx5VftAZivKc+Nkgq6wk2uw\\\\\\\/LXrl8Z60Ayo5TkGppm0iBc8O0o\\\\\\\/Pkv5AhmDuIZtjf15ktcg6LppoejCLsaf\\\\\\\/cB+6agAT\\\\\\\/HIPaJeKmQ8KlaO4zBhVHNRhz6v8ZiyPA4p0AMzIKLn1+OUVfI2V0cvezBhtt5BbKZKSWzirCk4Z4\\\\\\\/1CKe2YArXVgtjkIfHraxWXiF1E4UJd3ISkQCXUi67mqCuZjcmi3h8fio3fIQH7jDMhtHBjUC4G+BsInyWinAtpSJ\\\\\\\/oklWNPyTvjwBdl49nW0OkeiPT0CdCcDcNHf3tgAXZGwUuTsMew6n+lY+ftr4o\\\\\\\/GKcWdPZELbgKLuLHm5SS55Sko7jpZB5+EoBSnDRCNTc2mYZMZnU6QVKOVMK4Yyq66PXCmJmyOD7gmTqNKuZJwBT7eBmeZceyDOgQi7YK68vXpYFZ3Y\\\\\\\/oPYqBGedEr1CJuT09ClLfbXuNNG\\\\\\\/\\\\\\\/gjrG\\\\\\\/kT98f5WvdUDyA4P9cJZBAp0hhV5KvWovl7k1y1sd9OJQN2IK2vIE0PT5FvWa+z8Awy+RWFNkUvH66+BmrAW2XQiRfjgZURCTykFT5Dzc7O2AKJNUSvBOEApDaNtupPnTq\\\\\\\/15a\\\\\\\/ZzRhicVr0oFGuK4r+0WvF84+PtJ2qXssRkyM3jQ2b+CSMixNecsGwCiROyIzWHkVTAhUkLCJZ1JvnzAtFVzvkwDf44ttypBSdbS5pYu1HHnCywK9wpQ1BjjCOB7ujmNla158+FtXF8\\\\\\\/q5lPe3gbZrZadMaPSy8tk5iYkrtWRSbRsYdIfJmZkru7XC2yQeE0NjGrhO9j4r7epQGlykE\\\\\\\/FuBrHkMiAmyF6Lgal8k9K78JbTBrHcL1fAlbNqio1vJ8yE+NDSsNK7oiujKugIqF+zawAI5\\\\\\\/k+TLrZzKZZO6B2o2H8261h0UoDCxVyjBAULOcj6f0LlY0PFU5SDleD5gJ\\\\\\\/OdBzwkMv70NHK0IPRWObporWKtYWcsWEPAWYZVHcEGgEYJlQPm\\\\\\\/NkHqO3ipCNN9YoVYIWjljzcUlQ8EepxVa9ioAgntq\\\\\\\/MJHnkw8TdQcTiuYw87gkTD\\\\\\\/Ni5rkIBKeGQIaK02+3vmwqIbC7H+xt1RMoEn7+B2vQNyB7hoJoQJgV8bCof204WqSXkWBarGq25eKhB33DraAl\\\\\\\/T+HyDj18gEohsMcjNOdsyw\\\\\\\/i3\\\\\\\/wRBS99cojUVpd6Pa7xjmrhfzeGoHHuY4idHHFmMNaHNVP3TJeAtm7jVa7Z9Tpl5RREJCUjSUv66+MQBjc6l1IVt3syzpi4ALQ59N9COWkq3HVm3oSnTgAY9e0xopyGgUDoFQ3f9X\\\\\\\/CVUfTBoA\\\\\\\/O04VPA7li\\\\\\\/dHUhmeHq1m5ZZ4l+AkMd5Dd3aqeYiMkz4gvHaBWuz2G0A\\\\\\\/b2T0rM5bP8WGbq0O46XYk+CBbQahCbI0hK+RvsXJFVN+yM1JZJFVvCtP0rFPzaSjAxSt5spmn6XgZZowqOYi+RZU\\\\\\\/8GLhXFijqkxCeGa8uV6h030GVUTDzZgZkVjR\\\\\\\/VkZ3CTENGMhryJfMMiQ4L7y2PP7uxxjrQerJB9qEyqFyb7N1LbrA7s+2fdFng4Y0byBu0rycQ2K2wAcCSvaRbloftTXMRsgEd+uEZzT2a\\\\\\\/ZzFnEZsuUVPmFcZ1zpHQ6ONKjnSglMZ1TagKCwGydJfP\\\\\\\/Oob1cSXd\\\\\\\/6+tXMxjJfA2+\\\\\\\/LaVkfPblzs5s42apBbRbjVFIi79I72+AX2z5Hg+CczZt0EnDR+a3VIOLhpw9Ehsoez9xXecYfLjtEnfJsx3vGqU0MjO9D\\\\\\\/M5WgLWEDezI4vXrcS\\\\\\\/hNpBJhDzMv\\\\\\\/tZwnV+rZCQ+nIzKlPUYh2e5NayHjV7ourxAgFLVGkg0Urm+q5+\\\\\\\/x5kXuQSjVhdGi28ifLY\\\\\\\/zwFsy\\\\\\\/V4yJrvPnx45q+FlWDsDpbQmsB1rzOwoXyCX7T7mfv+z3fU1Taby\\\\\\\/FY412wEWEFAJhGCH\\\\\\\/a6I1bzWunDVi84we0RFVt7UHcLXN7paoqPKF9Y+CD9TAEXUF3+ObYXX8bRIwyF+eokBBV5OOWIad4dd69rsduqfJdCynTf7T7aTUVrLuoNN+L\\\\\\\/w+vCILXkRKNGBknRB9LFiUDTNMvkYX4EwQ7eOpv8nLm8w+XL7id9MTR1OWbjYlDRdQ2gVhg3LqDDDHQwVhpsXeI7+44C\\\\\\\/+QooWjMIqScViiWTKWO3xMXu5KgtjFsV7vOeDMywTpKqqtFXJ5+in2TMWIBrU3kOCetAykWGHm8eF\\\\\\\/Xnc8fZ23IRqwbD44U2nkkjCNpuINGSV8UY+VCPBz+cRapaoVhu2K7fw6s8HX0Ung33BcQiaZf8\\\\\\\/6TPJNaaq1hD+7kIH6c3rVQ4TPKz1Wqv\\\\\\\/3tX\\\\\\\/OkiZwqKL+LGbuTkEndixPUvPYp2VkNRfRjsDehki4kwW4OEuj8tIqqqCxHU5uYrqYi9ZZdMHBmHTgxDJpA1cF6OpBqbuqkQjM9LnAEh7CkuMTLgvl9+voTwUTPlM3wjkI4f5plK2adaFVjnI6bIIPuHKjXCo8KiTzol1K+RhV5FMaCnQnVe7NJedcvh0YF9l8fy8opTHI1G7sjAclmL+G1Kis\\\\\\\/Cmq9eAcy3qVyb+3hIfTtRJD\\\\\\\/fPoQoxYRQP+oqUI4owMH9ps51JBt8kHSb75O7J78ZJyrEHXP+wm8N9Q2jDeJJg1O7ombGgJnAqfXqDu4ur+bG8Ej2TSWFDWHvohb2hq0kZa1JJ2TysZNA+dv6aVsXoqs9YQHg0BrX2oHly7AxojC\\\\\\\/4ntC1BFzdp7JwJnfGNxVG5UUyFRHf07Nh75ZpRFGSdhGdr7A0pWwnUNdok4ZhZRBXasKpk9T9hcVRdGIASo4oaJ1vXyFl0AsYSiH4Sqxm\\\\\\\/pDoBby26BRtVSOvPkWs1uhiSR3LzTB5OIi\\\\\\\/FPwvGyGjK0Ai4Us17i3381c3at4XknFL3krnq00fmb2UdGAFzSc9vJZHirVgxhA0ReSCzai\\\\\\\/h5aq+DWUY8\\\\\\\/l9cVE\\\\\\\/5aT\\\\\\\/\\\\\\\/79PXKVvyk3CGoJDX73B02ZYwhkkExgMe17+eJCSGL44y4VqP0fYJ\\\\\\\/eg8gHq5U6Y1GmMukRh6zI2HuIXpddFnRxl5bz2vxD451r+jixH44ZdTC\\\\\\\/iWATZSwgP4WxwKN9UCgPWRjDSspddSaVnnEC3EJ1uR0EGmReKVqHumxqdLepf8Tq7TJgNsGN+Iy+NVXq9NbRbHztev0lKw6ViCq3QZtdCDtyMdso8dLvpQLsFi6LjEw624MR5XkaBnX9RcDO2vZsmJfGL7363LAFkIbsMRR25tJ5P4EqPS6rUdAGymfH3mdK2IXTouurgBDt2dH9k61ZgQlx495X6XtLnuW2BZ0vlVYQrPCkDoFg\\\\\\\/wJbsnCDVmGo3M2xpn3XexroMxnVuirWNVBIWTRYKfkEk5+QkWYajnheXUwD0BtLk\\\\\\\/AVCAacc6zUMMUWh0ag5P56fK4wp4J3og+8XWeME03H0I0a+mSQBTSCukPfWmBEeptKZ6+R6FlMCbnzNjwKRAdCa7X32Dys\\\\\\\/3b4F0DYE5ukhka8CYIk7VE5+SKUC7jQW4rxQ57sFZpsC5la1I3smkjAdR8a8v3t4cA34gLPerTkDTy7nOKBATAsDq+CVdDtQMl9W6O09wOwoqrGWHA4dhyu\\\\\\\/LbYeb3qyZkMw8bM6siVcxV4SxU3B3hs53RTGTg3etqwGFS2Z7XvIbwD14vfVwo6KbVfvQHIbj8NzBq40Bnmfl1kMmms6H\\\\\\\/tzaqxwovL\\\\\\\/AIVmi0vaprwPuKmW2i\\\\\\\/QX5iLwWFSu9wTyG+rr4WBx+\\\\\\\/c5lqJGwlU5p1Q9pthyFWf0sMVOSDTIbuYrjhCUAkq08dgz4R\\\\\\\/G\\\\\\\/qJy\\\\\\\/5cFfIbC6W8W8MewSsNTyT+PGme7YFDTAjOxHOnWK\\\\\\\/VpgHW8E8MCFq4BCpU5CQNzxjEGgsRh54RnS4lc80OlWQ8dvWFd6rKSdC2SqKpSuFoNeYSRgiN+iLpjYIJ7m\\\\\\\/eOEd3ffoP0tolcfSf8QRjTIFoXOOKm+LDy40ATUOEU3v190m2GdksnyCpMTRi\\\\\\\/A48hO+qAVpdjTGQHZsHuTGsayjfBSkgZf9Flw85Kl82z\\\\\\\/lg55+\\\\\\\/3E0xXFMkAbZEVpqojLKsTCi0mq1l0iKYo819jBWDkbNbSH+q1yF91jYIKwlW353Oy3q7YvGARcuLewegwBQYBhkqZ4Ouf7KbS0SOxhp9DT9C328bkyy5864zIEJIOQzbyb00aMvCrzITQCQ2+mc9uX1KC1iFdMtu7ykpiy9HZ575xQyUWv7BhtHfGR+09daDxv01WN21Md9JGOySDpgOobBkjtGkSSYYYBTE1EmexC46kXH\\\\\\\/iG3HCzmYo\\\\\\\/pEK8IxYq9LcR5HGQhwdILklGvMNNZi4dB\\\\\\\/9sPGhOjENEyU7zdgyg1ALEyNkBb2bRttnF9DVhZtbBheURF7ORoS8EwQ7IOPkwqOLYzCl49VSLmv5hGAv7+qqdu0KKOSyeOpjIi+jU70jQIM+lHoNNpcG2U9dIuKrfaBVYyPGyX4ByXuSGhZF2+TmHcSvfKENXwnCfOIYgmlwatpaTwbe\\\\\\\/iFrLVTOpeFutIfjIsG08j\\\\\\\/m4SDb80N59YFJc51Xv4iFg\\\\\\\/ReklVieUmnODZlEauTrjRAL0p526QcjS+v9M+E42nmZom0QcOedqygyHb8lpSZszbkmyrVtfUTelsVgqhxT6A772RmjLMglE\\\\\\\/aXtKcqthIionQiuis\\\\\\\/I42rJmW5EBJDrleWSyGb8f3Y6NTd9CxlzOZzs5kwVpvoUNmN+RhGOduANw8DmEvx6\\\\\\\/ZK4YPlfPd0xd14vzWJRKCsIo3sLB+AR6z4H060sYLCSubveWTqUbZ7e3\\\\\\\/TiCY3j7rJQiXJRx4Okvwgz8TZ67rwhmurB09vtXueGSvLnmUAeF+ZSpwmkyveiXihJ9NVJTKDdgt5AuRPCsORboYlOhZ60nY0sDp79GfO0kcTYmksXw4ZGsRBJfdAdY6EWPALU2NLzmzG16LNHfnbMnzfd3EOwJ+tACfaocUvbz9gG\\\\\\\/Tu46FSH95eCh\\\\\\\/8E8sF0r28gNIIXsNaTu+XBXxHsvadCxH8WDWD8QiUq4KCGSQ24f7O3mcsW4PTy0tEKDc9Cixo\\\\\\\/ptAaasWumnQcO9WPweJDq+jS9XNrjpN6e7qD4bdGzak2DPZGWUzJd\\\\\\\/0xdS2SCIPOqlDo086Md4ZsOmQY46CCs\\\\\\\/+SyVMMA0Zd+kR4A\\\\\\\/HYOdA5XTHGA+8mUtypu4LKe05T++JIYIRv9wLhTXi\\\\\\\/Tjf1CM7lxAMNMnqSWdbiOUsFuWf3SZjEBw1iDJTJCXD3LC54Pnjhj9qkfH+WbN+lavnRsltqmg7tx4WOGApUGy3jdjZ93jRTx0+ishOMqdnKIxJcltdANuPWass5oCcF8G9tM7kYFinOsy8zf1yb1riMk0c+sLcPAyQLyKwwMgny5Rbnu9H5oyNPrPX1iPc90F0SpQ2DIikr2TaC1Nl4gwm7+ZZliwyW94MtstGebyd8Y0iK7VBpDgcFVC\\\\\\\/4aIVMKlTUPe\\\\\\\/diJBmzysa2YSFyq8V59v5Gmh91FC\\\\\\\/6jBeo4UDcwoBU\\\\\\\/2QoAKPBL\\\\\\\/xjkEXOEXkAo92dPbq1BFaevgXwdfjxwVpQ09lIUi1i0n+U7VdPYbXOwHRXNf+vJ34QpeT\\\\\\\/ZsVuWR1sqLasiiRUavg17CbF93r1Zz2M0mHDsPG9l5pm8lAgVeCP1jdSYf89+Au3uh9Hr3rJBlblB3KvIJSthfxA7lEhNtXkIUDuICR4B0Y5Uekm0tE1HEVyJsYHWhjNye3+lWmGb8M\\\\\\\/B\\\\\\\/u8x9ZzYb+WwMLXuRhKKbg\\\\\\\/xL2Ze8zTvMtE+Vz2dqGK\\\\\\\/uW1vceY7480VyROaRZKgyCC1e00mul8\\\\\\\/7aFALH+yxICeT07LLh0KxqFbXTuYeeXnHIdg3rkP\\\\\\\/pqhEmWCgJu+2qP6NpjIzwVczM+ShRMvIQf4UGLMY2B2Ku+LK8hy1dkKBoJyyuvqJHBs25LPYckrKUET3GoolAZyqyPJnHMn0bBJ+frFaEUE63CYOBWVvcrijTMJN9VhCOB5JUxMfREJInIshFzBk6dAj3cpZSsBc9kl1sFc2A6ztG7KfHdYhI2X7wWv9uYU6J\\\\\\\/KMmMcboNmqdglAUGbD0m++iFjmnKw\\\\\\\/XBgUuVxQlZJn9\\\\\\\/wF0mFXaS7wMC3EqqEe5Msk4Ko7wbP7h3uEGm+IvkbyrKt6qVZBL04WqQTNXS5EmAq78rShAmtpZAxjs4+zLoKzkiTd0d+b2O3fPdjQouOHpm7IXp5MGlSnKwifR0VIYGZIaC2+bntcudILOPBB758kswyXjsT87UnVZdq4Af+Grb3SmS7JYDKVT1zc0g3M0hzqm+mWvGvf6itgyjNdAMxNNH8kh+zK1rM\\\\\\\/Y16AbisHxLwcETFOUKdJaWv5jd2k4n8bpbPZat0RLLarA\\\\\\\/sfSZRppxDZxVrVefSYjtI5\\\\\\\/3LBthnXmBRrQYjJBt\\\\\\\/rU1idMDSIIJr8roPWwNDyaDR\\\\\\\/nwG6iwhhojzjHPCB1vegdnO4dI1IyTLuRsUxrknJHwW\\\\\\\/vT4PFpdC3I8Y\\\\\\\/4YzDmct\\\\\\\/VxOCUFowA\\\\\\\/u4jbwzjIzxu23ItDIs7eND0r4PHLPQqI7nmwucyjCDYrB3J0WKKlQcdkJtkrTukFvG0YKXWbYTYQJho9KtxRq7Ml4Hga0jEy7QHaVpt0mA7YNXpf85Ek9YbVk9yQYxzCEmyxiN1neD4Ic6Lk59t1sK84UKMOKC\\\\\\\/tHD3XvrEi8Ffi4KybQhOTG8kb2lbqWp\\\\\\\/Q74ZTa6Y43MiLj2XjFM4OzG9aLZL48zZE2veCiWqze5x\\\\\\\/LZRyArgLJH5qablYZexa90k\\\\\\\/sCyEUs\\\\\\\/KfaMX+0BqsGTtgDZnf0gb+4Dp4j1R2z5r6iKVMxDpHCrdLC3PmMNKKeKekQcX6q4YT5nE3Mf+cx8ZScQs78KD3f5OQ4rC2r9zZ0h0VV\\\\\\\/hNwqlJzQi2QSO5qFJiuFmGWBZyV5UMUo8NitVLMekS7G1iHx9AZrm+PWCbKo8AA35I4B3OQ0UnURQ+cMP8kfjoT0EPEQ1+uiXH3hcorvRk7aOw6hYl+xNBNRX1Zs1K9\\\\\\\/85COXcExk8zEM4EAb76iejVmU8lSdEr+OCstBSk19FUy2tFNEZh+UZXWT9bIVpBNwdzOcuQOE+lCbWd\\\\\\\/szEnfSkCkat6oyevBq7jEgIbHS22AGeSU\\\\\\\/iHpIE7cJeM4WsV+Vl\\\\\\\/+kwt0By2RnBzQI+LlIcjOsabA9rFVwnIMr3yDiQ6TmiQ8mNZbjilxnn3T0ElDY8PUywCaLu9s5VVu5NRc4DvgpJwxqbL8xuXVqh12jbiFsCVVtgncBxd3RPxCbVcapbzPhsevlImlTVKctXqFjPMmlRUU48MQ0Z6+iQmQ87UrFEvXEowDC2b0mJm4dSN0tDnHaVAQC6mqRzBmCRvykrxsJR3Q16WqjNzer+PzEPJqkaVDB05UyALul084NqEGNQe4OePOfMK2GOS3Kt7iHcxEwQBBi+PYk6sH6F8fESqpPrUCqTBYrJJnz9h3v+P4JXcj8YEDnyaK2NfjwmUgeAKHsuvJXZLtwDm14f5y9hodPe8kjifRtOqI40xp8y+J8RG0tg\\\\\\\/rErZeNif0rHzfkjto\\\\\\\/xS\\\\\\\/TRuL+\\\\\\\/Ce2o\\\\\\\/1wGaqwEib9MD\\\\\\\/eJ6pxRUk7j1pAS3eQgaxzzAXgZOsJSbqfEerEZGlZBhrUz4PqOy6na6c6lSFRuFRShNc4Ef4hUvdDKpeyIYsphuyzozhr03fcC6OWuhUr03seEJbTg9CQFGbWMyR7hQrMKNJ9xdrfHDsRXeKlPivybuoPMM+joDsKBsFeP8S0x2S7ygGjE5QaLVYTqi6FQCS4zhIou1rerFy8kOaJET3UmKg34SO7bBHZE87ao+fI9hsttjarvBLg2wkdd6d266bk9jtod3k2ccXoBReR7CB7lu1U3m3wceZ22xwxeXB5eGwWTGMM5yABd5L8g5flIqkzDrdfrRBGyPCWxjkpOYygo6KI3dwQrVCuRGpumqLkAyOckYE2hlASZ0ZEl15EZABzRuJcajFYg7zk4fOoSMYmEtp4OScH7VIT91H4+EY9puBAIUDe9K0L3EXfa32EW9cLlA+zzmp46pKqBBph2Fw3peoKsFVpAcSNpeCvSvy1OGx3mVPTYDOBhZbnFZcUMyfAAv872yEQoyyBfPGfnrPuRM+J0YZp2v6\\\\\\\/wJSVayCYqE67cQFI3poc9iLg9khR\\\\\\\/cSgofI1Ke24A3eE7uDvwTZbOBAhKxgHeOzqqlbRwaRqOX+5mr5i7KVb8I2iHl6Oj8F+Y7DYFeEBinGa7la\\\\\\\/PkDRaA6YfKDR8SSNilzVfxcU2x\\\\\\\/Ioyr0P6xcP4kjiEu0zKz7ZvPx2tlu63f7SA323ijAlLAwN5nmGAABTVf6E4\\\\\\\/uLTsHzL\\\\\\\/E0eK4octDWEeNgPnm3C2PfimXPq59X1jRT9y43YG\\\\\\\/qezPaUQM1rtrcbAmLn7iWsMLU5IDxO209Lpd+v+Q78x3XRDR81QtDTg1+2LHi+IG+BuZsFM24WNEmCU+VjHiCBrSzNuT9OHG0AloYiOtytb6dRnrB6P9P\\\\\\\/GxU6otBWolg4oVAh+xyBeiYbFtW14GU4hNH3JTAYk91l7cVo+mvglsY\\\\\\\/BBbhIHXNxhE1g0HdYwoTOlPBRvlqlYpXgKCi0J4LqbRPeLzWlhWuWhHKBxLRoIw8F59u4cviYjL\\\\\\\/l0AL95WPEoAzJSNmEaFTigaHon1cCf5cf1sgOETwDvbEFBV6SPh+NfChGAwr3Dj39gw8S4rwZFto\\\\\\\/Ku99uN8oM085dpSWHyXiCF8Ww5zsv+WFbZJ5nttpJTWxbjtZ+RRXRs74tcJ0d7UwivMnHCwh1q0HLunLpAVIskYioSXrL1Zvm6zXXEuK0In8Bx7oNVzSB7KA3JihiEwg1dICvXgWa7upVDrzQNgwTdNrWQWGQtOkmR98qaGY7paRRWSPpMiB6f8HEfxHqyjNW5L2B2Uvg4DS\\\\\\\/pE6C\\\\\\\/rCNzDtM0HOakzIh9PFggsYZXQJhvmwrn3h7QhKkolYbkUT877cllzAr87fa+LTOoX1PxtVYhCpFB51ktaJ3FzLJUDiJg3TymbQtVwMjnQZPc+6aMBECXSYksJrEafKOm6iVVB0ZJLHlHUH\\\\\\\/axi72ifdZMldGHCEDsYRl6iiH0htYHwrXjSXNAgT+kBkVmMPWqc+3brle+ozkf7cPFii1XY7VHXjN14Ysw18vtiEkIfWJ45Lb3IpPj8+gPbUxV32YgAy\\\\\\\/PXW9jIAcpJQ2wpRl14mT949fH5H+P1GcF8KJup4GgTswLEsKAGyTaAGyL\\\\\\\/3yXjegVd2yQsVaH0dzOHMMmP1x0TwAzaZYdfDe2ts2CNc31ZgqRVEWybJx0906G+NWg6+lDeP\\\\\\\/Eujki4NDeq0aEs8JIFqzh1DCX7Tig2bPkQqshQTO3pAYoHBlE\\\\\\\/8kFUbOxaIhfocZ7bRLpz6gc0s3YenuEBMjm2imKZQNcrUDqYxEXzDARr+UNHI0h6UHgcCyWCt1PmpN9342b0DEJ3yM9CQgPIBKm90N5zuExi5pXRjUc7Fq7kaet3C8ASWZj5vbwIBswdIqVIhzcOtGYID7QoOkf3jaLcYza\\\\\\\/OA1jOfHZ8noL+uJTCW2zqGtqgOJ5OC4x+kQakzN6G4d+on2NebyL0hqQ33CesLIGPaLDgJm1u+pgBf5McRfV0VPjer5v5S8OhklO7EXFF699jWfm5V+n6DAIzNDI2znoFQHvp0kpzjXP1e9bbwMHdGYqVyikHWoL+RwDaFOezLv\\\\\\\/zD1cszAadvlQFlX9DiltgwP23TLEva1vUa28AzKLuGn05k5qhiXKH2Dh2\\\\\\\/zRJkKxbkb1dbO1kLTy8blcZM60+jBbUwSjLpjiRLlI79QAL03XNl\\\\\\\/Zh5uQmAebPD4ugD5hkQ\\\\\\\/ELvKrPtSNu1tQ5W00O5adeoHPjCTmXyNq5I\\\\\\\/th9dSW573vfPPMQLVpNM0wG+DAx\\\\\\\/tSc0znuqbQpH0guJdlDYJLPLKaplPGH+NUc\\\\\\\/zsCzZoevZgHpi69Aj\\\\\\\/c9m7qgexFHweN2MHLCEEuBVovUndWreXTW0vVDSQ\\\\\\\/AvEEKI3au6mQGZXt18j7vUcPAaAbbjAA9y97Rb+XwkpRa6ZDCSOdV6V7KtN6u5siSWmuPEId+w9UCxqFOPqqEALi4olZIVrVT3ww6OVL8Qec44eCdClde+TxbHa2SEFiTE3G7Rw2JTnEOE4AvZt1dN2wmmS\\\\\\\/QOsdV3FVoGVzKbwZ13jlczQEF8nTrVw6FJX3ltkSnRqn7ZXisO6biCXfHuIY9XfdS2sKJ2nhq26MZiDblmtq7\\\\\\\/cFZjSUMFgYvJmru\\\\\\\/CY+gCtsICSaIrGfd+c9rDEPERcE7ahT1dLtAxevjV9sn\\\\\\\/VsMnSE+LjQwCL+aEU6Hw4u2Zmwta1SaTtmftME9jLOH2DZJVgKFUr7y05HN3uQhLI0nB4OHsnfo1EtKxxvom7+CwxLzd6lAOpAZfmGlXGA28R+vL8jMCz+DdcMHZqvBjZHGvjtCbxiBDbPQ92SkhJ+5fjl5vDtXehifvjt81A4YHWrdEl2pcn9IYfRL\\\\\\\/m0llj8k\\\\\\\/Bw\\\\\\\/GIclIL7THVP0eqQfcYf45V6ta92FqOomB8A0M2Ml44y0Br90AVri48rq4JW1CjqhwDnecedVOfOim\\\\\\\/W7e3\\\\\\\/inpvOD+FVVr4J3cDCsnAejNU8UkslkfXGuzhDBpnHcfEzmsnYu8yITMYoxT0ugzVwNlQVaV7Bq46AJlbzus09Tg9utufKujKAGiQvdPwgNCWTQaItvLM0twl+YwVic1EmM7R5njLcukBkGPzdMVPsKW2E9JVl9BSIw+ArrlPCQ+HmZE9uZf0EXMaOfQxj4fFYkeW+o7PKEJ\\\\\\\/81DTMWG85iyENFRYnhITyydYkxonInRboxgBZ2vOQDfduY1GryoX2psPMfmycWZQiNn1GsJrA3Z5CJN8Bfr8vhU2uGDyFb34N71SXMvT10IOs2DMBGCfaU+HCK30kx8+cmVjBrv1zyexSkcK6XeEjVoOoMS7klLTPYgzM5nNAwVb47GONxO3MfeG5VU5CRjn9ZK60T4btiNtEErzKmd7rN+0WsR6BHG0dkTPimWrJL3Q+fOQpJ3910SQkPtDJovsrEvdan+PjkNUDZ8Sn0v9hORGTh0bDnGYwb580TXssx2xQUlU8\\\\\\\/DFSdQiu414oxHItLSZ6Dbyh80WIXRkWzeB7HFYDqEKlNC6zq9UcrF\\\\\\\/cVwk3oEZHuMlK4c\\\\\\\/p3\\\\\\\/VwTBWxOgCyyG88Ikie8HprZPPVhO4aPIV6C4N7PQzuAMptLPPK36S1IJGsIh+lULJrjJhNo5bNB2J\\\\\\\/aFs9t7OzR3Jt6eZH2nF5ZSk7+w2bHqkEom6axCKQu18Knrcq25Tw80iESjT5BcSgTWwlkOWJqFoO+qST4k5KQjjDu4Z4dC\\\\\\\/ei2fGY+iIFHd9bcehCWDUj2QEsqlCpxYw6ONlvHHVuuWbeUrlNgWY1KSEbwRF\\\\\\\/vkvag1xRj9SPVbhmY+ZLrgmpI6pECWLQUNudrpxwo8LoCfByfaQB4ITNlxE0yVainKkCQNHjCN+Rdd40ecUTlLmuFXv+H2eA+ZmvWkiVWUWS3d93hWmZroli2B9sTETg53pVsatwX+bJsV9r1WkWRMX60dEbnfirchyRY66SI4MvIFlZl8TXR3Jba5fDbTdZihsCs\\\\\\\/DOi\\\\\\\/vgGLhMbekUw2JCXipqmL3u0YA3dJUuUPdnFQQoJS9DlLCMcK9kwGv4cz4PWPk8WzoxBWSjI3EISOTVbussWxYFnUqXNxX6Po3Pmli3VPqodQTCj\\\\\\\/vUt8EFOWkIFTvjjS3+DYhbQyS3R9rGWclSY32QTTam4\\\\\\\/9nOHYX6boXbFLxJFdJ26uAcBz4\\\\\\\/ltn73moCnhG+KmK2Q6jFOfZz7YlVatltsTnvDLxV2\\\\\\\/rp7OOa233m2svFPYWfZju7raiByIqYjqE7B71IcXjL4+IPx+I6zibKit2bgE+1mX9TL1MTmj78Lc4PUO2NRyXJDmaKGSidnNKxoOZ60Ae\\\\\\\/01\\\\\\\/QwZFfEDt2V7HGWP4KRra2\\\\\\\/Qw+qrGbsesB7GyWkt8KGpHYMK1WDQJdwWGa12IfVBF9Ttl23i\\\\\\\/rAAyYWJAawsdPS9NPonTI2S1TZThX8xGlcac1+Fa67PVJ3d8srcRbxsqjTpcXd5zxlhShdvmwttcWdjrnHMpKpxTybk+Nrsvj2jY9PyU9gzIKNwNnMbZFogWeg6EoNNkVImIXYxfBtUJXVfcZa78ksAvx7tiv\\\\\\\/Aze1SzSFrIUPTchbbZM1NErq\\\\\\\/RkzfEtBC805YAT7PauMHkuthxX2D1aTAxsWxADn\\\\\\\/AdVlaLTsSWAa+3RjQOzWZ53sbCK4BOHuNmWtZ7JZRaZGTHW5R4blrCUo2QdKDk3CroRoeSbCoVcfE9f7cMgiBxCzhIRi+ujQKPssHaPMsfZG8mx0n3eA+Wjsq3USnQEdqLOJxvBjuacEoVX0ffcVglzCNY\\\\\\\/u33lbDK0AGtdK+Oj+uM6IisiApQWEhdQtS7bKDhcxeNHuHBo4O02ivuY2nFJ6\\\\\\\/5aaMG7UiyLv6xotVujmAETYmgX\\\\\\\/dIMigZ1rW9YLVx9COpSMX7J30VBTNGdRWcH8gkcOQ56tbH1MbKc6b4JusieVVe47M2rzeq4GnfdLiDZoJiR\\\\\\\/3Ay9AlgGBMwe0kilzGh7GEw93iCmnUGENdyUV8hr\\\\\\\/JNcyTyUoY9oELHujeYAQrikV3gvD6uJhSzMCyYlTjEJBm53GqYZoeNvr239YZkr3h9mbxv2o7tsv1nR1ZYgcRIDbCKe3T8vBWm6IICJcPNZYL0ULzyVYNwr3eALDR2q7px3gzWX\\\\\\\/X1QBMHnuHJQ6vrDhrmarhJZWdDgZ3Wvgc3TJUT\\\\\\\/ZCzItfTU5IhLUVmsSd5BCdoj1oNuj7z+8clc9otzdZq4oR6gAKKMF4RWpNGy7HEAtF8hU1m0MRozx6al5CTVIz0MHBKF36xQgIo2oYMfpdI7AsQMAEt7uw9K6nW0fSStZRSvGsXswrCpMcf9d5aK6fvbfeHxdT+oR4LuNtNOicmsCogPc3cwOGEFPPNtfgThlbbxhYJQbtMRzW24frWcbUWCHWI7TLes0NtMKnJATVqAHBnJ7KR8syFLf5tCKoAKYyjhGfX3lbXAbboQPeCtoivLStljH3wBDIXFGVrrAHA3t4hLZq2EIvhAjtDZttP\\\\\\\/ITXX1r0HghMMY21cPpES4aZu+mfUHD7ggbgWllFoRPHbJIES+PyDJWyxrBcHpgEAcpz3d3FG\\\\\\\/VKMTHjYDb\\\\\\\/\\\\\\\/MKXlmeabHihozQ3JDqHX9kgRKpLIjLbFZ\\\\\\\/pZmlUbcJq9hcpah+Z+EnccZiahwQJzBoykyADCzinAWp5SMYO8tmomwcId2cM6WHxFP1tsfjfBOJ6z\\\\\\\/VFDbWJok\\\\\\\/ycrP1XvuKA0Qd8Xgv1gITMqc+ECgxketmsUI0xwsiNKd8lJzghYI\\\\\\\/kvvME3l67yYQvWMM6IArUIunpEd2PmCs0CEvy+PwMP9N7bT8r7lW4sKMPtSyErR1BBMOFQT4Mhe9p5frqGHBxRNYSZZroQFAVTRh\\\\\\\/F+5ckESaw49Q7NYUeDCXAmMiq4ar4CdcP8WnivSvcbNTuWk9b5Y\\\\\\\/agBYXISYZdxODR1QTU62vZhpUIBoSyVUtsal2j4ef7Udz4wxR6WfbM6cToqo27Kog6p6WA2HNLR5PUBPfqhLUUX3m9NvTK8GK1UEFwrfJoqCC9BtLoHywKQgQ9KKdfzdEYeN3roulEmHXazC9CgKPbFIOc338XXWF01lVhaj869Nu9hf83HGLdpua6GBWw83WrveojeK56qlerUwk7nAkKPevCqyDIIAUU6pKYUTF1SaLNBsI34QCJMbSaEXobsxX+mS+xeJGUbtPozwNJDpTuZrd6\\\\\\\/Ae5FMCOkvWYA61NgRLSo6iliZnU81isyr1H+8nZlQmZhnxaTS5HRCdm5xL3xmZPtJZ74jOKNh8SJAKPRmozTIKBNQIAoya5LC53gIEznAsBV8xZ9L5pbbdWeMtDO\\\\\\\/NTDSUEB37ZyxS\\\\\\\/d5dqqEWT4I+8MlfBIvVsNlg74XQncNgFvHX\\\\\\\/jSKzqptjhys4CCC8RrnNmLtWlyLq7BnaJymVxAiFbqWmyv0GVmrTOrBANVTzLKEpmDlngbAgv5JS8jpP53V9+3n6za1oXuvIGl9u\\\\\\\/Y82mXsXQ987a5Q1HbZVaqslw3lDTGuxYVPKhrpEUcCcbme3QXqHWJR7Rd+EXcz5pO0BK7RuIpFevk2i4gnjV1nyQMSnoFT0cqquHoSCNXeWOdUIJxuSZUvrU71CADl1UlK6fs0yEyl\\\\\\\/XLsr561wVZu5so2vryQvZCxbg\\\\\\\/409aIa5DLc5a4ng9u9BYycony\\\\\\\/AMtzISrA4PEoj+XVvvDM3LzrmMlgm23QE6ok2T8eviJkaL57Yjf7xv5DmPTsIJvQ264PscYpdEIXCSrtEf8RNHC7FDKUd18Fgu9Tp2oggeClSVrRdLZq6KJ1lZ\\\\\\\/FGdK7poYzBGE209cCs0gac44C4WlXBUQptN7SGRzEVPxUYhwPE4n90v+BBNhrr\\\\\\\/EU14g3KWEVIoAAyn0MIAlv2c\\\\\\\/x4S24tnH1vFBjVDaY3usY1R6D5Jw3HjbTXcEP1uDZWEyrrm+VzZBtgLn+4CQe\\\\\\\/sZ49AnSyBWk03Kwz0b0QtDHZoodaC7+dLp6EzrMdM6oyx0cNOFVaOwXC51RV1WF1cxSjLSJoyO3oscwFNLYQflVKl5zBYrGG0SRUGurqXeokdubJelJOy+fN40KjZKu6QypRW4PCT+Ow\\\\\\\/OUoy4+kM\\\\\\\/lATiXH8QasKcXmqRnk+qd3tYFSG5dVTouxbQvGCQA2qgR\\\\\\\/\\\\\\\/ShZtGgClGud5oz8zrPOe9x+oDNR+vhsPlbjRBZPCNxOBk7aGInxmpjUtIXLbePoYMgZAe3BvXkyAa1Fu4lAC4NScxzYxJezjvR18M9nQMBzUk3eqmJ4LNA+5YMnmAJgtyGPGSFSJibBbePoyU8FCMQNXivVAhY6L2O8YEZB1dm8TDLLi\\\\\\\/0itGSu\\\\\\\/paJD+8s1dN7xBA80wmKMn95npZxiNb9TUWUagEf54UzoPsyh8BqMPR37HJleSp65fEg+Y8pJFYYp\\\\\\\/lrqrZpC41LM1iyou0lRkn9AoKoMrKCQKuTmQTiGAf4QxQDNWXJ7k0h4xqWP8mxq0i78cS2x9n++gmpKpCO6Z7xzK6lGpcFisWYtEq3DN1ZB4aQwv5SN5A60sAvP3dWi1sCUbaxx6JEfA2rJp0A6AEsszFgTYqaIrs+CbYzLO4mGmhW2QB3KKW76+w3JbDnTEBsTVPIhhmZsEu95mNB+3XCkYqmUuCJBzZQk+ND+r0Cq153nFzQt6Cnx4fJkLYeB4zboIYy5gp8KH8vXYG6pgGgqwsZnmp45QPpYrmcp3844uXWaazaC5PmUWSzGrAdYkmjskm0Rz0KB8b8WEHm\\\\\\\/TinvgaUiIFnz6AUjsGWLGcPdkIbqhlKVMt\\\\\\\/hRo1kcbb0CmpYNjy1LGSlaL9+uCgJOiIvUZ\\\\\\\/Xwc6Bqtri5xp\\\\\\\/VNPKBWDpyWBGIhCi7982AATC6a2jrmZSOJjSyG4auHjXIhVZoOyCYte61tfs\\\\\\\/Hza108AJuwr2oKEzw0BhFfJiPAtHDabwORb1at1AvtIHZhaiIuFu0lb\\\\\\\/cdDs0TlZud78pSwnJ24w7usWjQYPYbh7bV8fHq8FWsXIqlhk6sKWxYgtI\\\\\\\/0KR3+pnS+bSyfXtKEfEozfLHsX4a+TNaSBPcRUcPGeplX1613JlOiYq8ffU364qq62OrqYENxioCX\\\\\\\/8PX9J3g5LQ7bjtRS\\\\\\\/laY8eWgBO2uhYlgiElyji5AIeBidQs8OdkSlAt8RwP1Vrw0OJIb9vn5zlDxMYRNQMge+86maPAwxqjb4Ikb1cTA9IFXm6v1VjnzuMPzLn6GfAmprGnfYqzP8LHSFzkLQmqcBItNKgq\\\\\\\/UkeoJxbYPaQMD6vGHu1tFxvVWcHOckHt04ueSmCkxmD2Uya8djW9CFkNA84HSe0jH7udue6vTHhoF7hAkQDQ0LnpYT3ayWbSBaD\\\\\\\/vsnCSO2EgiqHLNRpdS9B26Jj1Y4kHkHcrktz0hVNspuj5PmSDDWp81g1emWzfSvzOOKavfuMJrSnzq3Ov4WlX69xCVyn4IF8WmnLSFzg9GSwvVssS26pr4Nd2nveGPA4rqpWh+FqmEMQGJ\\\\\\\/Lox2rDaWSNj0wFx9kxbsw3BqpwQaOhc0dtlFqH2nxbGAvaANYwVAOPrutZYZ534DGQZuspyGW4Y39SbFbwRdplelykCQulsUexMA6kdHMXToJGrvb1wZPlV0CdwPMUohmSfmGOYwfNfaz\\\\\\\/1JOzmw9VL3bv05usJi62AdNcEnH3vrFWSHMzzVwH9FDnAyz\\\\\\\/+X36w1kAzONuotYdJpQmvd7j4NrKdTd0yDAmGMkms8+LzI++m+DE+GGTa5VBbTPmZ6GjCR9OYRXjHHknqzAKDxbbmhcExOJ+D\\\\\\\/HUR0r2PupoX2PEhDk8MrPf1nsAEHQ+5MVUwystwsd3Iiu3p0r4PNoHC+Zq\\\\\\\/QEWtoF5GMpXG5g4Yhn3MjDkOg2gKF1jRlPU1JrQPCgKrgnN7kF\\\\\\\/kacF\\\\\\\/QoYb0WdQhjMc2DpvbW0Y7\\\\\\\/74j2zIUAv0COjyN+CGBRPLoE+TbpY8kYuKYS1c99sYTUNcvovioZbiDe66cob8ZN5ZkRFZE5ZvBnNilQN5ZVPSmkxWQGwYr8IaVlnsNI5udBhvM76nitbs+z238U4aqywCr2UCSyLG5U3uB6ZOEZ3p3FlSbI5Bevmhtt7EENxnCiluxYv2AOmhVuqQtkCE2r\\\\\\\/PlMHeunqyL6Yg9okzHg6vzAiY\\\\\\\/FLWobW7BVZ\\\\\\\/dNc9ff65hVD7xepVJ0x8ge077MrxDno5mtALVAerSYB1sa6RdyEq7J\\\\\\\/Cs4k+XSMd9iBszGSkP7pkA6D+bwNs4bQWIvFu7+DpSgBDrZGNkOPGQTzLhR7\\\\\\\/5Ofqgl4i+krdB5qBU2dvhFrvjF9JHfnrLgo3jQawPAw6sKLNuKrzU5sPq7miQO+Rt+MLXdZ3z04qexYpoWhkBL9oEDc\\\\\\\/LwWlhV\\\\\\\/qqxxiQh5x6AKUAxj+Cd89QpDGKrXIMJmamdPsHZB1IEOqnwAw3jZt75HhzAk3NLfvH1YA6EkTERaPDlMjPgmB\\\\\\\/TvLprt1\\\\\\\/uVA+CbgQ6O4NjSNnnT3D09+xey8n2m8guCG3QTaCtDn6piISDex+HQ6JROrafmfOwy+h7X4u8hnuNEXSCn37lf4lZUbgnuKSkyv+lK5gKj1Pf+hlNp+Li8HwzEwtlWuEEKgww3DZloERzZ\\\\\\\/zQzW9qeus9bt31CU6EHzr41FobOi78JEtZQQS0oZEHuO7Sz4E9LmjyczPAZpfDBD20EDZXZ+x9uUtVcZJ2ylnkYcrWQV+n7GF\\\\\\\/M++BiB11JUXuuCxhSPaNPgufliZWRZwrWP8YMEMZD4l79urivd\\\\\\\/NjDhnkNhpJkWveKh1dl59C3YjY7QZYQoUxa6sPTpgQcEOFLjqRcBmP96udsXCXNpKARUwVwluc0MKhWqkZyDyylxk8+TGGGeeRr70ocF3DpWKfo1ZE5TsegowJXir0SMV5QhGpvIhAR3b37DuNjKH53ti1zlcqgzv1viZ4+LyoUNNr8yLQW8nf3hIUMioyrKSwVqFiS89wQCR\\\\\\\/sGskD2P+EOQcmbgROUwUnHpCShWbcERvRLG4iCrOnwQtLnyw9zAcr2nXLTRCl\\\\\\\/TC6\\\\\\\/2W4GMz7q7\\\\\\\/UbIK0lKMKXoW8TyDcu70WgCjiKRRzv\\\\\\\/p5A5bIBvG+8lgxbxwC\\\\\\\/YQMmDx9SmSNzqTxf2HebW6f8gd154Q9u8jNJ6lvYU2A4V2\\\\\\\/joZ2aIcw2YiM\\\\\\\/Bq1Lf5ux25u4IicFf8QApP5o9JnaMahOTya9JvDoKMrk6MNWLUGQSD0IueFCP2PnwAZ9PwuyEg4Oed526neI4PlYISwJTxwWmWWHrwkCb6v\\\\\\\/SZDREXCZz4tp88V+hfoX6MF2P8eln6bEBH56yoejYzl7YWPNy1\\\\\\\/RFRJ0ex98qLaMkO3FZKTZlTPecSPo9k64zdkPRx0HqXP+e\\\\\\\/iUDkbT1BuILXv9w3d6rToPz+RcCt1hWXQFToz7N6AeRNF5gZ+GwXKhTemP4zH4WDyiiEjzD4cosAxXnRMiOhfRKlFPEZtuyJqvg6gyGEKP9AJBHMzHonOV\\\\\\\/nGe8\\\\\\\/0fBq0RQsMfbztck8mA3i9H8X9lPH9sp25aJkub4fYqPDSBNYzHAg6Cobrli9o0SCRenH0Zsp+Orm+0e\\\\\\\/v06FiC5a0WhbUD5cF2UpHvdh122M4T12SH4tx94K73o8kRwi13BFlZJ2X2l+OStqb5zVZb+GzLlM16WPjcIzoRqABfUgKCqZ0kHQ4Gnp47VFBU2Km\\\\\\\/S6K5nJd7k3rMLK9wGewMvh2WHyLQjcATt0HQ2bTuJ0vWA5foWtIRF20s38MTxMvRBDKcirIr5shXfgZhw0SgKyAi6BfEr6+G4RrrzmzpOO2i5Dyie094gxD1tJCwgHB6xKxH6vBx7T+wD92gG0k55rvwFea9ejuIiXHarQKDIXesgkwTWkFs31NNuk7cbYD3TWoWAlQ3d7YqSFvlHWUnaqj6LdzkM5+gisVyV6h1NbU6cPLaeab07ofP7X5V4AbTe15Rp8jWlritpCm9iGBPgvfct8bBD2IoWfMs\\\\\\\/hAoKCFQbxZb6jkpTiuboomjTeKRMpizn6Nd\\\\\\\/TLf+RmCYO2HG+sKRnOjAbnCQxv5AX39gTjoQWawp1a+eqvPDOVyuwyjH0ssiqz24lUkpMNytGvwQae6Pi6tL5\\\\\\\/K+\\\\\\\/zGaDTJuQCNJTuf3pgyRsNy\\\\\\\/G7FCBhmwlJq8AwJMZvSUuqGPz\\\\\\\/t5p466dCBEC\\\\\\\/D3kCsxYYpKXMUOGt2jh7ZOcLITsxrsg0DZP66lk77R5AKkK9sZIWAeceQAlSHe5SMZrHnJA\\\\\\\/n1gUlMtwbOf70TVCNQb0EEblizDwannB0AOa1x2lFbj0XL4s7ax\\\\\\\/tbLefOkhSa1CM2REpw6XiRkfWGqwN7Obd9PR47Wp5GhgMpDtX5Q2qyDTV8IWRkX8eDNpwB6wq9WEBjTyVKdwXtchMhtBemtRa64DuLymr6PkzorLjC1t8xI3\\\\\\\/ttuhFNUy9sFjbVS8LE4\\\\\\\/WjWoXTkAWf5V7cn6Za\\\\\\\/HG2j0b53WXv7LIZcEfzCregrI5qG\\\\\\\/pO9apEfh9TbrE771IGz31uSN6OMYBgokmMIBL+mjMJ+AhiRajaWS4Yr589qVc4AdjwpcGVFR4wrP8bp8IjAXkr+ZbwPeDE7U2WIS3gFMJR5WDbjraBzS5xUdDfbGGx8cvQMSGJHTb5Vy5zX9Zi77tokiT+mjIDLL3fx0m33+\\\\\\\/xy0c9QG0XNTmWcL+b51CwoHfA1y2oCzD1HfUzYF9B2pxeO6OUIuW4JPyqju9WDk2XvntTpKDcvswXgHPGCJVJZrkmdR2i3s3zEZafNJPE3h9CPSRt8KskC5x0ADKmquayGutsetOTLgAvV9pFlUP9QxuPUSPsGM63jQPJGaaEnVyMw4u+Q5G9cgXGFXymowDl6BSJIAnjwsUo98gFS\\\\\\\/ciIgQ7jBFdm\\\\\\\/2sh4phWm5Dtj08xf\\\\\\\/Ais4ObXnQwPAP6kcJpfxNfOoDM50OuTD1IsDvDHEGU1bHFrT0xNW98i52NTkwu3JtvZ90ZrLNolyHmhXDQGpRHjp5pinMgvcTOuGa8S9qbMNytsGDBC7w3+c2+Sc5MM2rft4lhmctXtdy\\\\\\\/RH\\\\\\\/oLVBAuc4JN7kUxgaruqtxYA1hjSoYY2cDq5KCOYUUt\\\\\\\/T9YaU3nuG3Hh9ScdjMCYp5wANR+79taLOzg6cGEhRzI3OC9HU3dXmyX+U09UF0XBpjPSkpXQ+s2eyuVg3mLwKXQCvoZD5JF13vxQSlNhfJcjXflejIlHdMwqVobB9lJ8tduq\\\\\\\/LTzkYF1lno1IHc\\\\\\\/NB+ieKLh2IQUeAZ78VmQg1qmBpw1pVhcZNmzxOLGp1RR6LLejYFzc1PimtGShdZdbuq5v+1nviKNy0UwmaCHVk9rRLG+gdFT749srE4y7Ly7k1qXZvzX+SI1iDdhd8AvGxGofXZZRwBsObO9nosWZ9YXJHYCebqbh9Lkia1JerI1VfTVau31zbC6ChA+YRVQZSJq1DU0uzutcf1gyhj6MEPK1r4QDvdRY7AwDdkEIt1i\\\\\\\/a65c4fL54vnx3fSlpvcjKckJzEKF\\\\\\\/38YhA1VrTzLZ8plY4WsCNyTSkh497yCnT1zmF02khKw+\\\\\\\/7M\\\\\\\/FGsBBdIYZjOCvlVlOgFL7HmG1yvi0CwPHBkLSeqNgmZyd0vFZDgVvvlPaY6uTofnnRT4ggyt4rCy6uQKY1LYcdOFNGqhHL1UnEjBhyyS2ljr4bgIgkwTVcxsSxAn3YscANSUMzK4M572r5M1YJzF0bkj\\\\\\\/OstMNSzUmdsseYZl+GSAOXk\\\\\\\/M6D\\\\\\\/lp2ub97HCQa\\\\\\\/4Xbn11zzjSyFolZ4vpYj5goBDkVw2SwyvcJfA7RT+vxtcagyo8CRTtj582WlvF7SBgjXmVzCLdZcnSOfBlQjxVmzKcijkgv1LdqQ1F0I6cKujk2FFUJc7kQ\\\\\\\/udHBIF8L6b22uNq1b8qnJnn9lwOCsBm\\\\\\\/nMfjyDetCaXc9QnIGC8gdVE6c75z7hPF8fe4jbGeQ0DodjbIcmRbn\\\\\\\/Rl7StHpUBdYNU+9eiOj96Auuqe+15iA8fZmaC4Mwxe6XtnnKfHcukHEjto4tnoFl3ibebPc0rRhdrhv0OY3rERjYICwkmIcTiO+BUbi7R6uBrMaZICuOYlJKdn3ZMyf1zZTchxzOVO\\\\\\\/H\\\\\\\/6GBwIabJWvvIm9UpBxVosn8ZXjZ7CtxRKmoSYZEry3iue1crlQG28Mex6UA+STIw8OBD0+4kVvWCMBB\\\\\\\/u+oZFwovA1Fy5IplDPQn\\\\\\\/1ZjfyT5hSSqiupwlc7HVmqRuwyv+39soIcPuJDbTZ7HRHQq1PpqvtJDU7jsEAiBCmwK5NArjlpqS\\\\\\\/Ssw0SdkYTw6wiMMrzKy0EUAnpxv6qB3bhtEB0utoFg8wKFReGhX7BwiyfG1mn+3MB\\\\\\\/NZv6623YG+AjnpDCNwrDCgU+uBnVsaEQ1SZNWADF3Xn89J1tTt1OEDrh3SCIsw9OofqjlWpj3uEcL7FU3ipq\\\\\\\/ClSNZzs2y78sSiVKquj\\\\\\\/eWs55jphV8R2zz\\\\\\\/yDP8VngQ1IxE0JEW1gZcCz6eRYtDDcqzCghr01KWI2O3j8QO9HE+u5t6SRXBnvSUnpzDd01P7CotCmSGQ7GQeXNMvpTLvnF9EenBelvbmmJiJ932btoU4cgCPrYLGHFHugxTli13YRSZb1nZgSK9fOqw7HUCzoQkDb4okbs9r5PfAhqnJeDd9adjmLp1Q8QNCLI+sD2HFrC2IXmw7260iGPUlcReRW55XGFK22HDqSnqwinzGqAQXaa+Z\\\\\\\/pT56OjzgcgwSnij9U+xWE5z12Z1ApV7AG5RN1\\\\\\\/xAbzMybjuYn4qPnM2hwkoUblVhpeQ0seCqbZZ\\\\\\\/0i2lwmlAFBih9gu+f\\\\\\\/qgGuPdkS\\\\\\\/MBP7\\\\\\\/AfdnexXw2EcXL0q9INMz\\\\\\\/G2DBMADtsMsIqU\\\\\\\/dI\\\\\\\/7kqSrRVJ70K19iTkyeJwd1zWY1bRefXW0Z7FASUQbgPEJDE8zjbxBVzHhYpFNWZSkSOhosAO9hZSAmEbTzB2fe2hIIiUNB4KmGUH422H6mhFRqFgHAHdNR6Ab3s34JE7kdjI+LSx61XHRxGydnlcISIYeRs6q2k4sGxoYawSWdCMHWuohElKVkED0lydLF9MYWKw0+1lMkkqrlyye2lsV+K9OntNtpIcrF469y5vnVHwyRmgjfRiWZpQCb8XOTDQVcO5XcrAUUj6LedpYfre5pvGrB\\\\\\\/NJYZuPHlc+yvmAe+R53i5k4DOmsXSoDxE1RLAx0NoYPSadU4lhsjuMxoOoN\\\\\\\/TcSYSDrajz6zRwjIFI6NqWmjWZa\\\\\\\/eDwVaxVr64WSq5QTXwVBop1YpQ\\\\\\\/LkXVYp9YcvCxC5FEz1xSsvFu\\\\\\\/2wcPjAmz\\\\\\\/RCgOByB436v67vPuVR8lboQFvFXFIIOoTF5TiBQ267TYzbENSaZIwqaD5wU6AJ7s5y2FIbTlwqkJha4aHtHU\\\\\\\/9UftxdEnVDDoUf9ghA\\\\\\\/b7zWNKPEpFnahBhsOTpA8xXmvEx0mgtEK8NT22qAtYf9kHu+oQU4jYM3t9vlGWeklTLbMAI07mF4xa5LzyB50HFy+t4l69Eorzf1oyXUNAHiLqYqYSssyKLV2PrPsubm0c957AlsAm7mbxphmQp5LtAs8bI5ZNIdJKL7TJ6h3dNVz9uLpx7u6nTKCu7sh7oC62+WQkY7Xs9F3RXFlYvYGfVm3e4jwyskvC5\\\\\\\/cVptup0xKgME4S69IQaOScu2\\\\\\\/NVgJleE9r0\\\\\\\/auI0aJawp6jDUFAcEJyVaRoUjiUgYo5ReaC4sTRqW7PcAJjT5l1ggdJ7hUf18\\\\\\\/urseTx1SCZ7TFHMdOuTn9Rzc99M3NtYgoHCSLVrtLDK1jcV7zIL8A7X3hU6oQPfDNKHogEAl23PO9jfRpD9s756AaO5DWg1p4TE6iZmc0JFOvbJhgMXtYI8hCvO4Cd2ebhOhgMdO1iQWk19i3cZtV4qfw2e1ZujeAJLkpaYC6ojLZ0rDYFGY7M8v2VJQ07yWeNp1mlJ9rJSqxnFtpQBjg+9QU7cHNVCoe5Y\\\\\\\/gfueSRnrXooH3edvYxBUF3bbVPGm6o59vcqADjEmQ8kev1R1X1NSgB6xMJJNlxsN0\\\\\\\/8EXSGhuS0zaRKMkr7OASlRtunhXQeWrX98G32MZwwsiQN6V71\\\\\\\/BLLhef4hf4jOz1y4lx1RhZg7xW1JSFsWGjzma903NfiEjsWt78iWOHFzCs96Pt8Ht1nlYdind7FNsWdqo5KrNv6MTXOwpkoMitO6mFs+hKQitj8+U5HwN2DS06AkAv0vxJwWUUxTloeXznnybQb6k8QZN9vaDqLtYmsLCq100mWDLwyl6MnwnZz9p48qdOBOYT\\\\\\\/hxJRYrPXDPARvB2NvtC\\\\\\\/UoVSD5FyUrlQRr9VnnqTa9b\\\\\\\/f4EW5xf0VhllVYmbAr41cmGc25z3robljVhn73jVAjzxldbMOUtBZmZa7v4XJqjCtfH+aiYH7hc1u\\\\\\\/5xlkcazVOAKDwYq55+rgaipyXPpodj2jFN0a8iNSiMrCu4t47RIlu1JXQlbPDYTRuuxO\\\\\\\/Msc7l6TVvuVqFR1n8JPfFHWItI4phzBHfdM5Su0cl1W+cvs\\\\\\\/QjToNKydsaAhuxjoVZA8LvSm4iBca80xoNYqx6sfYRhqlrPV3BzIIUrz+JG5yT2w24JPduPb77Frnkh\\\\\\\/z6iQCEOBNNwkP2Nzv+Y173IlfF4tMGyTH6zqjaOK57y69zjbry9++rUmD1Ti5rJ6o7pK5TQq3LNQ1fi0pxKFUYl6UiIwvipJsADoaZTSIb7rjUeij5\\\\\\\/N29gy31lRAY1xE90S5zBZauv\\\\\\\/bmi3muw26hmRT3xKMJohhWQAmIyoHIHWgwrVi+KZ0SCjSQ7uvrZpB1bbFTqAxGITAGY9D\\\\\\\/JX4Rp5xrLPuA98fGke6oHmrv9cMlemWSg4AJ5Y76ib7E3iyTmVAy5yWq2U4HI+jAqbmTgYdzEduiD+TlU30D2sJMhLUsns7P51T\\\\\\\/hqO5zjWMF\\\\\\\/2cG9qluvkdF68juB\\\\\\\/KwD0KpKsgrqljMA4yFokmjb\\\\\\\/HdNZm7k7nSWgBSZNtp15SzjsL0kmcNyQxfDxVAiqDVPiCryl14o\\\\\\\/mskZmGYhOH3uQthN7iDo47sO42rxw0BR+\\\\\\\/ehNLIkCYB1YqY7sBm5nRw0wwFHJE0Z\\\\\\\/9EqfjlL9C5M25UrWBPepd64lLLgf7KbI6gnpD+0fJnrF5bA8HGyHqu4woEsx84WEMTBqgfWy02aKI6aazFe6aMlawWIRJ0M4NhZe8qTxl0g\\\\\\\/osB3MR0YLZ5pyvqA3S8wDiVvlEG0fCe3Y6zbYtR3s5hHAlTMwi6iKTi16TqRN7uF4R\\\\\\\/i8F99hn62OIJepZlgKkDsr6Ruq0KtfkZAfXgnkfWlWdnZ+NucDePy5WzChJaiP2PkAG6sgUCQegMtMnurha1UeXby6jkd845gB0UpJWw81DQNSTC0tZohD6Enj8\\\\\\\/bnZg9sDI4M8bvzZxrJqxe+XZiXAxsuWyKv67qbx4ZQxLxvOxld8D5L+RKB2uO1Y8jsFKrl0WzaD6z4C\\\\\\\/MOo7QKfH\\\\\\\/gazZPKH8Upb01lfvFHC7uQd3Vxk5Dvk9AewSDx8ZaHyDbFCxMwhN554h29cbI1aiWzwLKbv2KXcoSSg3m0DiU5S5RWONvZB\\\\\\\/\\\\\\\/nK7yQdPGH6pc8sxFl\\\\\\\/V4bZY2zDzgRceX0NW5fxYm4YcIcfIUWdqkerlUXiaW6EnrxnX0EqbYpUHG4O5OlRZLtmNPnLiHpdJJiE8OxbZjrV6uo5+e7o8rYZ96gL4CyHVG3KdM27AiAx7iE0qJquCubjKjyPbL1W8DqN5otA6lfqB\\\\\\\/BimOCwUXIKprtMj21AVeJ3OhTkhk\\\\\\\/EvOrprQj5X7DpWjEUA3MvFG0OZ7o6EDcHjnNGoipvkGyM+Wxk38R10+JnSZVFoJSM1GsYtlzw9MIaVvTGnydTwhGFqOWelnDm\\\\\\\/qxUctiHcPvX8cWtkbCa8UFjd+wq6Aq87wcKe9daDiNbBJUsmsru2AL72FNUI3PQDrd0wn148yHtVWCJ2vX2TmCpaw4izS2QWv+\\\\\\\/MMxKWarnqkY\\\\\\\/M45Np7gbNizC1wOspX6zxtoWeEYJsrMjcj0wPw0qQRsK0aI2V0ImMKYWpMtaglFN8A2QhvGErMRefhcF+zezn5bwPAtJOk9fXRmyfsArUpGWQa\\\\\\\/Wll6H58FbcJjjXL4oEUjjNNFh+ItVt2VescnAVKmkO1PaFpvOhSyLvi\\\\\\\/HWevM4dei5whZc0EzhzqGnHDaKgjYsYSqGywn\\\\\\\/iJPI1S15gJ8qYGpFGaftXMvnYUwi2VsXMZtItvgQ6f+LMUOcIZ8PTVWgMXPQkjej9oi\\\\\\\/XEhYJthEzzmaAvBuaYazedLx7tkQ1ASlE+Jg+HK20M9xocBHxSbgKg730c22NZH4rpEEFR3DfgfAkS3T97rs0qO+pghHi2jLD04HrV0GwooEOU1nKaw9yyxWLs4eEYcSLScGD\\\\\\\/l+lT4vueEN74\\\\\\\/b0w+SZBIvQxd61W9AVFThmS4B5ldgPlkQBGAhjuLaxXOAa9OZilAhB9FZXFf0i8Z7i2tODDfm\\\\\\\/24G+KF\\\\\\\/JyBtXCylOPUGKcF68MslBSSso\\\\\\\/31xtkV8NTI9CAO0mLSwZTnYU+TdfsBdQ9wiYuBbcw4um28t19Fu3RnackGBbXx+2kjzY\\\\\\\/btTELyEdrUQSTR26mDhTYV8rOBp5HrAfqRBPs4hdpRP4ZWFcHZR0p7YIcnUzBAg7t1h4y\\\\\\\/TNDWxaw+A7ncEa0ow1i\\\\\\\/5mHiYwsVKFgtR+HqwNH\\\\\\\/ZF93z7Z6CZxstAMGhRPmjKyWGaIrgsraajh1E\\\\\\\/Fwq5JTN2koQRlJ\\\\\\\/BXPldWWzuyCC6kdzXey\\\\\\\/d80+MrLVBt1tCptHl7ajSbofQYpEOkUtOX7HMw8DU5LlVtfo8HlK2SMjxo1GAt+GaYcaYKoorc15pB5PSJ6Va+MOYbI6Q3TLVJ6DBBf15xGypjV0R0FxCqTb+8sIVkEsClSPM1XJTXuKP9I\\\\\\\/RKzs6wf66o2WX4SMNptdJEOB2Uqnm8LsdxF2AwVfuKv9XT6teRcxpbAFZUij1lltdaYxcYgScQJ9r75yNr+rgXGW2xSZ2jX\\\\\\\/+CsLiYdqMB8IUMYMx7Oi9nTPtslmgpDEJjXsyayGrhfDWjqSg+QvSunBC4\\\\\\\/mgASCuPI8MB1pbL044c7L85L051xMpr64k5zMARpm66tLp\\\\\\\/p9n7IMvBHfVUOzfi76pZW6GgYGY\\\\\\\/vR1RieySQPy6NWydTaCJbkVW9cnQnNASUdgu9ToSlEJeTAaMW8AqdRQfPRc2ggVgG9W0Ox5KzrMp4iDTkE\\\\\\\/Fng+5VQnqJ64UU50NpuyG6IJXMapHJI5lzn0gqaqIxsTZ4TvSfkbjpOmobCqPkuFlMiqz7V\\\\\\\/nh63gIrm8cn9il134tccten\\\\\\\/l1rpVxX6xbE03ADBRwpzi7fKimdpR2nQi+IxZ889Qv\\\\\\\/E4PpxQ0pnn3SM1jbTjLm5AwMVwnWTZBnxG69\\\\\\\/3vQ8SIiyPwaUAUCusuHOGSe1bGbwx55TIK27u\\\\\\\/EZY2AeypG5Lr4xKrKW1rMQmmOZbdNeJe4l2aZHAK+f8cHd\\\\\\\/XsutF179c1bFy5u\\\\\\\/WrjFnV7mpZmPjMRQwykMtdaBJcWpyZtKruo3MRIrviuuvAIi2IPr9xNyMN7KyzaxOaLyLuEycSxRPw+mZAYZ11OxTr0yepRfcrojLRnSDVl+bpQU3GMMEXYA3NmUSTVmOEZu2e76xRJihfz3mDyvFUf88BwPW\\\\\\\/MCE3A5iyFKHfzXT6fm3eWMZbr4INvlanLIVFuCWQqS7F1yd0Dddy3bCeFRjMCSG\\\\\\\/ggSrgKS+jdjKT0zipGPfSOSeN2W0twVo6dR0LJP+CT4kWJe4dTWLsa9Rzr7Pz0PflBr4bt44IsAqwQ7scwcxs7Kwadyato0zISk5RhneKFUmPVquvZyuILSm3m0J0T+issfCRZ4ETIExkfovqwI5IoXWa9YHpCt64DnXXW3ydsA3HYGKnI0JTp1tcANPIfFIZ6GZ7YGY7p3RWAPFTgSEcvDUDpzPFOBqxbqGRV5QAJ\\\\\\\/teLjBvaSVdMZ1Epha4uVxWuJn8K\\\\\\\/VDnKebJ4uAHBdlrn6S9KY2VVXUT6AJa5Dl1NNt9e4nDF4IzWWDmbU2+hiN\\\\\\\/dXyNmLGC+UrAq+8GbWBk+CrvVTZLmEaYsJn5rB0b5J+pab8BKWIFleS\\\\\\\/GXLZFHP33RobJBB9gwtoIhWlQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022lZKbIYsSNdLL1a9uWKgqfg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002239b5308d440037b1d69867517a7a07c60eb10f6cc8a0a77ba5874d8318a970bf\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022JfQVUnOSy3Y3R1QlsCfDaD1hqyCkW8lZgQ9vcJA+6Sytu1SftnUE+S\\\\\\\/jjv+2atQeXMG7N0jbQhUpC\\\\\\\/YD81M2JBzFVssfhDiEwKGozs5Ewj\\\\\\\/6tPNTC7ybO0LLQ82TQfwEJCZ4R+wGdRRTAk3XXpMd0KvhHHKcVF5zHRbiH9OUW5DhcvuE7tHhLOK\\\\\\\/\\\\\\\/NNojywpS7iPUylWDKZuAj+kqO+gjNb0axZkJfQValJNW8Ih7JU5aMIQ4gXxeghN0TZ9gnJvXpzEFxloRlVHN\\\\\\\/KlxyEz2020loA5yFsNYv4yUF1tis5iXXHJ8zQs9qKpej8CgA32S1Mnx4vopxWh7eaA0zqXQpQmieleYml4iQQ0lSuNr+c4r1myGyvDABEitIoDZfDt8BHAfi5mhSZ8OBidBOxjIZrw4jY4In4c3wJe7Y+9n5SfgYmirLDCZkUKK+SG6RLLztv4Dpy0MG3limROdExsNAAKrCMIMvnPAOmaooCeUNVm9BN4u\\\\\\\/OAO0NTEDch8Ke4RSdHWsY2W3ldPNz1zGsZ3RK3jB30pVA4xOKbDfSCYlga5SYpJgUojMmJKqK7ANi6TVN8O2x3AP2szLzfmKS3k\\\\\\\/kDKpU0eKtoZSQrrDlaT4Rh\\\\\\\/7kfsudle0oACLpPlD\\\\\\\/us6p4BzRvLt4YT\\\\\\\/4H4vk6yLuzI1yzRfbB3J+WTU72Vu2cEkrYEbGPgP0SWaLxRQnmsosz3CR2zOTGAhkrLYRHzwWxgWU0sCWdtSMQmCuF4fjDL8\\\\\\\/EhcVCdZMvd\\\\\\\/r0XDxI0rzP59B0yYhuD84pSPOBJU3+tGtiAzGb3C7gPBcdM9UNn7PuLDGkWCe1d\\\\\\\/4Fxgs9aTqiSuhEhhRYTjTkgVIV8yWq6LH0fSX9AzjpnUEkofni9s1A\\\\\\\/TC99hSoQozXpY8AcMaTw3niFdiXXuQ+xGF\\\\\\\/Up4VM9QO0BflaUzbIe9jMq8gr\\\\\\\/mgU2+G7WxkzjkIh7Wc8SevsMkppD+Z4ru5hkQDJNncR0odfy4Y2waPcf2ZQWazaSPUVLM2VzPfbMk8yGbW0v6N8lDA12A2XZfDfllEJNpRuIJKW8JHV\\\\\\\/pAphd\\\\\\\/Hh8zZpJAwP6gbKHdw673NEOXDLLW2V\\\\\\\/OXef4q3NYpLMN8kFAWHpqBZ+ab94enygG7cDKYrVDXIQq+iYooKiWyR6qmm\\\\\\\/VZfpF+8+FY\\\\\\\/b11Fus6S+XYhNWWZHREhT693hGbG2V\\\\\\\/vvDel2s01hJOSdLD81SslyTqHMV3v\\\\\\\/\\\\\\\/wNV6Fo6qq2yb57PCoqOthBXd1u0j7IjLxh1lhFlgSvH\\\\\\\/9KAxZmQG53mF9uAToqv+dX7dhjwR8bvDLCE3p5oe4YgcDY8Np\\\\\\\/pSanSAqrpH1+VUbNv4NQct9PpZKvH9Zff5Zw9LnVMS5DXCcnavKez+fRfRnvlPaIT+0BJ7J6\\\\\\\/NwFaQsBHH7vJrcnsOUev6Xhnpwt+3LDnmEWnlYK+gL2+fvuoUMRGIPJ2wmIKlnHlh+oj1j5boire0heqsFHZeaAOpPFcMkMUwRatDdV1yAFDPU0Gfe1BjRMRDZXJjC3mNpYpphSJ11fxJHMsifYyNo6jOCc5nk4Dy4qWRN99SaENhIUBa+EtHSeZGKc7omAAvV\\\\\\\/go9pkGImAnSd197R\\\\\\\/Yfyv2MPJ4KL0U8bWVB\\\\\\\/oP1Uk9r+b0pL4TEW1Hc3dNDnyhbqVosIsoVuoViQkPe3r9jaGIwAtzXbjHvAQgoPkuJye7JKvs015p\\\\\\\/CjlQwkQubgeqw1xEnTTaazqe\\\\\\\/anoLLCNSftAoDz5njbzbNnX0BFAKR0hyDFLRY5eDAGpt62rfO0Bbtn\\\\\\\/NMfs81ZbmnN1ldHq+QoYMTZK3DILg2l4+LySKl\\\\\\\/ASeo\\\\\\\/gLLt6sEMxqFqRSiUs06gE2qH8MUb8UR0ixI060Y9R96P8Rn+z7HeMtApzjEbL3Myv0YbsTGKmLKnnvsPQ58IMQgZy0RDjFYf3INsiTNCBoHFSUKgiGQnCUlr0iCFbrQEMbtQnzt1zq0gP3PpOwuDTyNanhopljj4CljPTvUMNEGCX9xKg==\\\u0022,\\\u0022iv\\\u0022:\\\u002227+XqCkF19HCvMVTXioXgw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225f206e3ba84f3f9d4760b42934695a35e60da7dc703448db3c617efb32217c5c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-490724 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/9/2889" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-n_9c9WWbeQ4PL0t5oF9Zdb_MDSgcir0VV8tupP0zBiY" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/9/2889" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-TBHzIye4ZNC4VEMZy9-VtdTYF8OIj7RHJhnX6XVmT-M" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/9/2881" title="Targeted Gene Sequencing in Children with Crohn’s Disease and Their Parents: Implications for Missing Heritability" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/9/2901" title="Tripartite Chromatin Localization of Budding Yeast Shugoshin Involves Higher-Ordered Architecture of Mitotic Chromosomes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0794-4907" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Adam Norman</span>, <span class="highwire-citation-author" data-delta="1">Julian Taylor</span>, <span class="highwire-citation-author" data-delta="2">James Edwards</span> and <span class="highwire-citation-author" data-delta="3">Haydn Kuchel</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2889-2899;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200311 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490724" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Adam Norman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Agriculture, Food & Wine, University of Adelaide</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adam%2BNorman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Norman%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAdam%2BNorman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0794-4907" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Adam Norman</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Julian Taylor </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Agriculture, Food & Wine, University of Adelaide</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julian%2BTaylor%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Taylor%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJulian%2BTaylor%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">James Edwards </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Agriculture, Food & Wine, University of Adelaide</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BEdwards%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Edwards%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BEdwards%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Haydn Kuchel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Agriculture, Food & Wine, University of Adelaide</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haydn%2BKuchel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kuchel%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHaydn%2BKuchel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/9/2889" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490724" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:490724/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/9/2889.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490724" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:490724/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/9/2889.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:490724" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:490724/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/9/2889.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490724" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:490724/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/9/2889.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490724" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:490724/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genomic selection applied to plant breeding enables earlier estimates of a line’s performance and significant reductions in generation interval. Several factors affecting prediction accuracy should be well understood if breeders are to harness genomic selection to its full potential. We used a panel of 10,375 bread wheat (<em>Triticum aestivum</em>) lines genotyped with 18,101 SNP markers to investigate the effect and interaction of training set size, population structure and marker density on genomic prediction accuracy. Through assessing the effect of training set size we showed the rate at which prediction accuracy increases is slower beyond approximately 2,000 lines. The structure of the panel was assessed via principal component analysis and K-means clustering, and its effect on prediction accuracy was examined through a novel cross-validation analysis according to the K-means clusters and breeding cohorts. Here we showed that accuracy can be improved by increasing the diversity within the training set, particularly when relatedness between training and validation sets is low. The breeding cohort analysis revealed that traits with higher selection pressure (lower allelic diversity) can be more accurately predicted by including several previous cohorts in the training set. The effect of marker density and its interaction with population structure was assessed for marker subsets containing between 100 and 17,181 markers. This analysis showed that response to increased marker density is largest when using a diverse training set to predict between poorly related material. These findings represent a significant resource for plant breeders and contribute to the collective knowledge on the optimal structure of calibration panels for genomic prediction.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic prediction</a></li><li class="kwd"><a href="/keyword/wheat-breeding-marker-density" class="hw-term hw-article-keyword hw-article-keyword-wheat-breeding-marker-density" rel="nofollow">Wheat breeding Marker density</a></li><li class="kwd"><a href="/keyword/training-set-design" class="hw-term hw-article-keyword hw-article-keyword-training-set-design" rel="nofollow">Training set design</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic Selection</a></li></ul><p id="p-3">For breeders to make the best use of genomic selection, several factors influencing genomic prediction accuracy should be well understood from empirical breeding germplasm datasets in order to optimize rates of genetic gain. Also, before breeding programs divert finite resources toward the implementation of genomic selection, a number of potentially derailing features of diversity based genetic analysis deserve further attention.</p><p id="p-4">Genomic selection involves estimating a large number of marker effects using a set of training lines, and then using these to predict the value of a separate set of lines (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Meuwissen <em>et al.</em>, 2001</a>). Three major factors that affect the accuracy at which lines can be predicted are training set size, marker density, and population structure, which have been studied previously in wheat populations up to 8,416 lines in size (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Nakaya and Isobe 2012</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Crossa <em>et al.</em>, 2016</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">He <em>et al.</em>, 2017</a>). Larger training sets were shown to increase prediction accuracy within bi-parental populations by <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Heffner <em>et al.</em> (2011a)</a>, where training sets consisted of 24 to 96 lines, and also in multifamily populations by <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Heffner <em>et al.</em> (2011b)</a>, where training sets ranged from 96 to 288 lines in size. This result was corroborated by <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Isidro <em>et al.</em> (2015)</a> and <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Michel <em>et al.</em> (2017)</a> where training sets up to 300 lines in size were tested. Training sets consisting of up to 3,052 lines have been used in other studies, but not to directly investigate the effect of training set size. Larger training sets give higher prediction accuracy as increased sample size reduces bias and decreases the variance of marker effect estimates (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Liu <em>et al.</em>, 2011</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">de los Campos <em>et al.</em>, 2013</a>). Of the studies investigating the effect of training set size, none reached the point where further increases in size would not continue to increase prediction accuracy. Here we address this question using uniquely large training sets (n = 8,300). This research therefore provides the most relevant results to large scale breeding programs which typically work with tens of thousands of lines.</p><p id="p-5">Another factor for breeding programs to consider is the required marker density. Prediction accuracy increases with marker density due to more quantitative trait loci (QTL) being in LD with a marker (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Heffner <em>et al.</em>, 2009</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Desta and Ortiz 2014</a>). <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Solberg <em>et al.</em> (2008)</a> showed using simulated data that increasing single nucleotide polymorphism (SNP) density from one to eight SNP per cM resulted in a 25% increase in prediction accuracy. <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Heffner <em>et al.</em> (2011b)</a> used a multifamily wheat dataset to show a 10% increase in prediction accuracy was achieved when moving from 192 to 1,158 markers. However, most of this increase occurred from 192 to 384 markers, indicating that the response to increased marker density would eventually reach a plateau (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">de los Campos <em>et al.</em>, 2013</a>). The point at which this plateau occurs is determined by the genetic diversity within the population, and the relatedness between the training and prediction sets. <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hickey <em>et al.</em> (2014)</a> showed in a maize simulation study that fewer markers are required when there is high relatedness between training and prediction sets, as they share long haplotype effects and large linkage blocks. The study also found that increasing the size and diversity of the training set was only beneficial when using a large number of markers. <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Heffner <em>et al.</em> (2011a)</a> investigated the response of prediction accuracy to marker density using bi-parental wheat populations, and found a positive response up to 256 markers but a decrease when increasing to 384. As explained by <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Hickey <em>et al.</em> (2014)</a>, large numbers of markers can result in the model being overfitted, where non-genetic effects are attributed to the markers. While this improves the model fit, it decreases the accuracy of predicting independent data sets which do not share the non-genetic effects (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Jannink <em>et al.</em>, 2010</a>). Previous studies have investigated the required marker density in wheat using small empirical datasets of up to 1,158 markers, while other species have been studied using simulated datasets. The current study uses a much larger empirical dataset to extend previous findings into the range where responses can plateau.</p><p id="p-6">Discrete groups of lines with contrasting origin often have differences in allele frequency (population structure) due to selection or parentage (founder effects) (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Isidro <em>et al.</em>, 2015</a>). This can be problematic as differences in observed phenotypic performance between the two groups may be associated with the markers differing in allele frequency, regardless of whether they are linked to the QTL responsible for the trait variance (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Price <em>et al.</em>, 2010</a>). The underlying structure of a population is commonly assessed and accounted for using principal component analysis (PCA) of the complete genetic marker set (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Patterson <em>et al.</em>, 2006</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bentley <em>et al.</em>, 2014</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Daetwyler <em>et al.</em>, 2014</a>). This is an effective method for identifying and visualizing the genetic structure of diverse germplasm panels.</p><p id="p-7">The extent and nature of genetic structure within and across training and validation sets influences the achievable prediction accuracy, and is therefore of interest to breeders when designing training sets. When the training set contains lines closely related to those being predicted, accuracy is higher due to shared long haplotype effects (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Daetwyler <em>et al.</em>, 2013</a>). <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ben Hassen <em>et al.</em> (2018)</a> recently observed this relationship in a small rice germplasm set. However, these large linkage blocks are quickly broken up by recombination events, and so crossing cycles can rapidly decrease prediction accuracy (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Hickey <em>et al.</em>, 2014</a>). If marker density is adequate, increased diversity in the training set will lead to calibration by linkage disequilibrium where short haplotype effects are exploited; this is more stable over multiple generations of crossing (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Hickey <em>et al.</em>, 2014</a>). However, distant relationships increase noise and bias in the genomic relationship matrix, which in turn reduces the power of prediction (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lund <em>et al.</em>, 2016</a>). This study uses multiple breeding cohorts from a commercial breeding program in a unique cross-validation design to investigate the interaction of these opposing effects in an applied scenario, which will inform breeders on optimal training set design.</p><p id="p-8">In this research we study the optimal design of genomic selection training sets by using a panel of 10,375 wheat lines to investigate the effect that training set size, marker density, and genetic structure have on genomic prediction accuracy. We also examine the interaction between marker density and population structure.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant material and associated data</h3><p id="p-9">This study utilizes an association panel of 10,375 bread wheat lines, sourced from preliminary and advanced yield testing programs of Australian Grain Technologies Pty Ltd (AGT). The panel was phenotyped in a dedicated field trial at Roseworthy, South Australia (-34.52, 138.69) in the 2014 growing season. We studied data from a single site in order to remove the potentially confounding effect of genotype by environment interaction (GxE). As described in <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Norman <em>et al.</em> (2017)</a>, the trial was sown as a non-replicated randomized design with repeated grid checks (1 check per 11 plots), as the large number of lines made a replicated trial logistically infeasible. Dimensions of the trial were 476 rows by 24 ranges, and plot size was 3m<sup>2</sup>. The trial was managed according to best local practice which included fertilizer applications to maximize grain yield and grain quality, and fungicide applications to control disease. Grain yield was measured with a machine harvester and thousand kernel weight (TKW) through image analysis. Both glaucousness and relative maturity were assessed visually, glaucousness on a 1-9 scale (1 = low expression) and relative maturity using the Zadoks scale (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Zadoks <em>et al.</em>, 1974</a>). These four traits were selected for the current study as they display sufficient phenotypic variation, represent varying levels of genetic control, and experience different selection pressure in a breeding program. Glaucousness has simple genetic control (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bennett <em>et al.</em>, 2012a</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Norman <em>et al.</em>, 2017</a>) and was not actively selected for in this breeding program. Maturity is predominantly controlled by several large effect genes (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Snape <em>et al.</em>, 2001</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cane <em>et al.</em>, 2013</a>) and is selected for mid range performance suitable for the Australian environment. TKW is quantitative (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Huang <em>et al.</em>, 2006</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Sun <em>et al.</em>, 2009</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bennett <em>et al.</em>, 2012b</a>), and lines are heavily selected to perform above a threshold. Grain yield is a highly complex trait (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kuchel <em>et al.</em>, 2007</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bennett <em>et al.</em>, 2012b</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Maphosa <em>et al.</em>, 2014</a>) and lines are strongly selected to yield as high as possible.</p><p id="p-10">Marker genotyping was performed using a custom Axiom™ Affymetrix array containing 18,101 single nucleotide polymorphism (SNP) markers. Markers with minor allele frequency (MAF) lower than 0.01 were removed. Further details on the development of the genotyping platform and preparation of the marker data are provided in <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Norman <em>et al.</em> (2017)</a>.</p></div><div id="sec-3" class="subsection"><h3>Statistical modeling</h3><div id="sec-4" class="subsection"><h4>One step genomic prediction model:</h4><p id="p-11">In this research we followed the statistical modeling approach similar to <a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Norman <em>et al.</em> (2017)</a>. Initially, the phenotypic data from the full Roseworthy trial as well as the complete genotypic marker data was used to form a one-step genomic prediction linear mixed model. Let <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-1.gif"/></noscript></span></span> be a vector of trait observations then the linear mixed model had the form<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-2.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where τ was a vector of fixed effects with associated design matrix X, and contains an intercept and coefficients for covariates in X explaining potential trends or known environmental anomalies across the layout of the trial. Extraneous non-genetic variation due to the experimental design such as blocks were captured using random effects u with design matrix Z where the effects were assumed to be distributed <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-3.gif"/></noscript></span></span> To ensure dependence between trait observations was appropriately modeled, the residual error, <strong>e,</strong> was assumed to be distributed <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-4.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-5.gif"/></noscript></span></span> was parameterised as a separable AR1 ⊗ AR1 (AR1 = auto-regressive of order 1) correlation structure in the row and column dimensions of the experimental layout (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gilmour <em>et al.</em>, 1997</a>). In (1) the <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-6.gif"/></noscript></span></span> length vector of total genetic effects g were defined by the genetic model<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-7.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where a and p were the additive and residual genetic effects respectively with joint distribution<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-8.gif"/></noscript></span></span>Here, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-9.gif"/></noscript></span></span> where M is the complete marker matrix and <em>s</em> is a scaling constant defined by <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-10.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-11.gif"/></noscript></span></span> is the <em>j</em>th diagonal element of <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-12.gif"/></noscript></span></span> (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Forni <em>et al.</em>, 2011</a>). The matrix K is known as the additive relationship or kinship matrix (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">VanRaden 2008</a>) and can be viewed as a full rank variance matrix detailing the additive connectivity between the genotyped lines. The constant <em>s</em> ensures the genetic variance parameters <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-13.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-14.gif"/></noscript></span></span> are numerically comparable and interpretable.</p><p id="p-12">Parameter estimation in the one-step genomic prediction linear mixed model (1) was achieved through an iterative algorithm. Best linear unbiased estimators (BLUEs) of the fixed effects and best linear unbiased predictions (BLUPs) of the random effects were obtained from solutions to the mixed model equations (MMEs) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Henderson 1953</a>). Estimates of the variance parameters are then obtained through an average information algorithm (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gilmour <em>et al.</em>, 1995</a>) implemented through maximizing the residual maximum likelihood (REML) derived in <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Patterson and Thompson (1971)</a>. From these solutions the genomic best linear unbiased predictions (GBLUPs) of the additive genetic effects <strong>a</strong> can be written as<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-15.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-16.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-17.gif"/></noscript></span></span> These GBLUPs <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-18.gif"/></noscript></span></span> represent the relative genetic merit of the lines and are commonly called estimated breeding values.</p></div><div id="sec-5" class="subsection"><h4>Cross validation:</h4><p id="p-13">For each cross-validation scenario conducted, training data sets were created by setting the validation set records from the phenotypic data to missing and appropriately subsetting the genetic marker data to include training set lines only. A training set model was fitted using an adaptation of the linear mixed model defined in (1) with non-genetic parameters fixed at their estimates from the full model. Marker effects were then predicted using the methods described in <a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">Norman <em>et al.</em> (2017)</a>, namely<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-19.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-20.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-21.gif"/></noscript></span></span> were the genetic marker data and additive relationship matrix respectively for the training set of lines and <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-22.gif"/></noscript></span></span> were GBLUPs for training lines calculated using (3). Genomic predictions for lines in the validation set were then determined using<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-23.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-24.gif"/></noscript></span></span> is the genetic marker data for the validation set and <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-25.gif"/></noscript></span></span> is defined in (4).</p><p id="p-14">For cross-validation scenarios in section 2.6 where the number of markers is reduced below the number of lines used in the training set, the model (1) cannot be used due to rank deficiency in the relationship matrix. Consequently, an alternative formulation was adopted for the genetic effects defined in (2), namely<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-26.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-27.gif"/></noscript></span></span> is the genetic marker data with reduced numbers of markers for the training set, <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-28.gif"/></noscript></span></span> represents a vector of marker effects with assumed distribution <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-29.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-30.gif"/></noscript></span></span> are the residual genetic effects defined in (2). The iterative estimation algorithm proceeds similarly to the previous section and marker effect predictions for the training set were determined directly using<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-31.gif"/></noscript></span></span>GBLUPs of the additive effects for the validation lines were then immediately determined using an analogous equation to (5), namely <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/2889/embed/mml-math-32.gif"/></noscript></span></span>.</p></div><div id="sec-6" class="subsection"><h4>Computations:</h4><p id="p-15">All statistical analysis was carried out in the R Statistical Computing Environment (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">R Core Team 2017</a>). Linear mixed models were fitted using the flexible linear mixed modeling package ASReml-R (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Butler <em>et al.</em>, 2009</a>) available as an R package and downloadable from <em><a href="http://www.vsni.co.uk/software/asreml">www.vsni.co.uk/software/asreml</a></em>.</p></div></div><div id="sec-7" class="subsection"><h3>Impact of training set size on prediction accuracy</h3><p id="p-16">The effect of training set size on genomic prediction accuracy was assessed through an extended five-fold cross-validation analysis. First, the full panel was randomly divided into five folds each containing 2,075 lines. Four of these folds acted as a training set (8,300 lines) which was used to predict the remaining fold (validation set). The training set was then randomly sampled to sizes of 250, 500, 1,000, 2,000, 3,000, 4,000, 5,000, 6,000 and 7,000, where each acted as a training set to predict that fold’s validation set which remained at a fixed size of 2,075 lines. These subsets were sampled without replacement resulting in varying levels of replication for the different sizes. Within each fold there were 33 reps at 250, 16 of 500, 8 of 1,000, 4 of 2,000, 2 of 3,000, 2 of 4,000, and 1 of 5,000 and above. All training models were fitted according to (1) where marker effects were then calculated by (4), and used to form genomic predictions of lines in the validation set according to (5). All training sets within each fold were used to predict the same validation set. Relative prediction accuracies were calculated by correlating the genomic predictions to the corresponding additive GBLUP values from the full data set model. For the remainder of this paper, the term prediction accuracy is used to describe the capacity of the comprised training sets to predict line performance as described by the maximal model.</p></div><div id="sec-8" class="subsection"><h3>Impact of population structure on prediction accuracy</h3><p id="p-17">To investigate how genomic prediction training sets can be optimally designed, the panel was partitioned using two different approaches for the purposes of training and cross-validation. In the first method, K-means clustering was used to partition based on genetic similarity. This was used as a surrogate for assessing calibration within and between germplasm pool (breeding program). In the second method, the germplasm was partitioned by cross-year to examine the effect on prediction accuracy of including multiple historical ‘breeding cohorts’ (historical lines/data). Online Resource 1 details which lines belong to each cluster and breeding cohort.</p><div id="sec-9" class="subsection"><h4>Impact of underlying population structure:</h4><p id="p-18">K-means clustering was performed on a marker based genetic dissimilarity matrix using the K-means functionality inside the R statistical computing environment (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">R Core Team 2017</a>). The sum of squares within clusters was assessed when setting the number of clusters between 2 and 50, which showed the variance plateaus when there were more than five. The number of clusters was therefore set at five. In order to achieve clusters of equal size, 1,500 lines were randomly selected from each to be used in the cross-validation analysis. With these 7,500 lines, four cross-validation designs (detailed in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) were then used to achieve i) equal representation of all clusters in both the training and validation sets (‘all clusters’), ii) representing the same cluster in both the training and validation set (‘within cluster’), iii) representing one cluster in the training set and one different cluster in the validation set (‘between cluster - narrow training’), and iv) representing four clusters in the training set and the remaining one cluster in the validation set (‘between cluster - broad training’). Within each cluster, lines were randomly sampled without replacement to produce subsets. This allowed all training sets in each of the four designs to contain 1,000 lines, and all validation sets to contain 500 lines. In each design these subsets were rotated to all possible combinations in order to provide replication. All training models were fitted according to (1) where marker effects were then calculated by (4), and used to form genomic predictions of lines in the validation set according to (5). Prediction accuracies were calculated by correlating the genomic predictions to the corresponding additive GBLUP values from the full data set model.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2889/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Description of the four cross-validation designs used to assess the impact of underlying population structure. The partitions within each cluster were formed by randomly sampling without replacement. Replication was achieved by rotating partitions within each design to provide all combinations of partitions and clusters. All designs had consistent training and validation set sizes of 1,000 and 500 respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-478565654" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Description of the four cross-validation designs used to assess the impact of underlying population structure. The partitions within each cluster were formed by randomly sampling without replacement. Replication was achieved by rotating partitions within each design to provide all combinations of partitions and clusters. All designs had consistent training and validation set sizes of 1,000 and 500 respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2889/F1.medium.gif" width="440" height="345"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/9/2889/F1.medium.gif" width="440" height="345"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2889/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2889/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490812" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-19" class="first-child">Description of the four cross-validation designs used to assess the impact of underlying population structure. The partitions within each cluster were formed by randomly sampling without replacement. Replication was achieved by rotating partitions within each design to provide all combinations of partitions and clusters. All designs had consistent training and validation set sizes of 1,000 and 500 respectively.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h4>Impact of breeding cohort:</h4><p id="p-20">Here, lines from four different breeding cohorts were selected from the PYT-South subset of breeding lines. The cohorts were randomly selected from the second yield testing stage of the south breeding program from years 2010 to 2013, and each cohort contained 996 lines. Three cross-validation designs were used to assess i) one cohort year (training set) used to predict the following cohort year (validation set), ii) two cohort years (training set) used to predict the following cohort year (validation set), and iii) three cohort years (training set) used to predict the following cohort year (validation set). As in the K-means clustering method, lines were randomly sampled without replacement within each cohort year to produce subsets. This allowed all training and validation sets in each of the three designs to contain 996 lines. In design ii) the training sets were made up of one 498 line subset from each of the two cohort years, and in design iii) they consisted of one 332 line subset from each of the three cohort years. Cross-validations were performed according to the same methods used in the K-means clustering method.</p></div></div><div id="sec-11" class="subsection"><h3>Marker density analysis</h3><p id="p-21">Marker subsets of varying size (100, 500, 1,000, 3,000, 5,000, 10,000, 13,639 and 17,181) were selected in order to assess the effect of marker density on prediction accuracy, and its interaction with population structure. The 13,639 markers on the consensus map from <a id="xref-ref-37-6" class="xref-bibr" href="#ref-37">Norman <em>et al.</em> (2017)</a> were selected as the first subset, from which markers for the lower densities were selected with the criteria of being evenly distributed on the genome, as well as having high minor allele frequency (MAF). To achieve this, markers were first allocated into linkage map bins of varying size for each target density, and those with the highest MAF within each bin were selected. <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> summarizes each marker subset and genetic maps of each subset are plotted in Online Resource 2. Online Resource 3 details which markers belonged to each subset.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490806/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490806/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of the marker selections using the consensus map</span><div class="sb-div caption-clear"></div></div></div><div id="sec-12" class="subsection"><h4>Random five-fold cross validation:</h4><p id="p-24">The effect of marker density on prediction accuracy was assessed with random five-fold cross validation, where training sets consisted of 8,300 lines and validation sets 2,075 lines. The cross validation was repeated for each marker density. Training models for marker densities containing fewer markers than lines were fitted according to (6) where marker effects were determined directly. For densities containing more markers than lines, training models were fitted according to (1) and marker effects were then calculated through (4). Marker effects from either method were then used to formulate genomic predictions of lines in the training set according to (5), and prediction accuracies were calculated by correlating the predictions to additive GBLUP values from the full model.</p></div><div id="sec-13" class="subsection"><h4>K-means clustering:</h4><p id="p-25">The response of prediction accuracy to marker density was assessed in different population structures by repeating the K-means clustering method for each marker density. As in section 2.6.1, training models for densities containing fewer markers than lines were fitted according to (6), and those containing more markers than lines were fitted according to (1). Genomic predictions were calculated according to (5), and correlated to GBLUP values from the full model to determine prediction accuracy.</p></div></div><div id="sec-14" class="subsection"><h3>Data availability</h3><p id="p-26">File S1 specifies the breeding cohorts used for analysis. File S2 contains genetic map plots of each marker subset. File S3 specifies which markers were included in each subset, and the genetic map position of each marker. File S4 contains all genetic marker data, and file S5 contains all phenotype data. Supplemental material available at Figshare: <a href="https://figshare.com/s/287c2c7f1623008487a5">https://figshare.com/s/287c2c7f1623008487a5</a>.</p></div></div><div class="section results" id="sec-15"><h2 class="">Results</h2><div id="sec-16" class="subsection"><h3>Impact of training set size on prediction accuracy</h3><p id="p-27"><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> details the effect of training set size on genomic prediction accuracy for the four traits analyzed. A similar trend was observed at each trait with accuracy increasing substantially from training set size of 250 to 2,000. A correlation with the maximal model of 0.95 was achieved with training set sizes of between 3,950 and 7,650 (for traits glaucousness and relative maturity respectively). Glaucousness was the most accurate trait at all sizes, and maturity the least. The difference in accuracy between traits was more pronounced at smaller training set sizes (0.59 to 0.79 at size 250, 0.96 to 0.98 at size 8,300). Grain yield showed the most variation between replications of each training set size (indicated by the shading of upper and lower quartiles), and glaucousness the least.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2889/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic prediction accuracies from five-fold random cross-validation with varying training set sizes. Shading represents upper and lower quartiles. Prediction accuracy is defined as the correlation between genomic predictions of the validation set and their corresponding additive GBLUP values from the maximal model. TKW represents thousand kernel weight." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-478565654" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genomic prediction accuracies from five-fold random cross-validation with varying training set sizes. Shading represents upper and lower quartiles. Prediction accuracy is defined as the correlation between genomic predictions of the validation set and their corresponding additive GBLUP values from the maximal model. TKW represents thousand kernel weight.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2889/F2.medium.gif" width="440" height="321"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/9/2889/F2.medium.gif" width="440" height="321"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2889/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2889/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490798" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-28" class="first-child">Genomic prediction accuracies from five-fold random cross-validation with varying training set sizes. Shading represents upper and lower quartiles. Prediction accuracy is defined as the correlation between genomic predictions of the validation set and their corresponding additive GBLUP values from the maximal model. TKW represents thousand kernel weight.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Impact of population structure on prediction accuracy</h3><p id="p-29"><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> details the structure of lines included in each of the population structure analyses. Sub-plots <strong>A</strong> and <strong>B</strong> display components one and two, and one and three respectively from a PCA performed on the lines included in the K-means cluster analysis. Sub-plots <strong>C</strong> and <strong>D</strong> represent similar plots from a PCA performed on the lines included in the breeding cohort analysis, where lines are colored according to their cohort year. There is a clear distinction between the K-means clusters, while the genetic dissimilarity between the breeding cohorts is less pronounced.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2889/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pairwise plots of components from two principal component analyses (PCA). A First and second components of the PCA performed on lines included in the K-means clustering method, with lines colored according to which cluster they belonged. B First and third components of the PCA performed on lines included in the K-means clustering method, with lines colored according to which cluster they belonged. C First and second components of the PCA performed on lines included in the breeding cohort method, with lines colored according to which cohort they belonged. D First and third components of the PCA performed on lines included in the breeding cohort method, with lines colored according to which cohort they belonged." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-478565654" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Pairwise plots of components from two principal component analyses (PCA). A First and second components of the PCA performed on lines included in the K-means clustering method, with lines colored according to which cluster they belonged. B First and third components of the PCA performed on lines included in the K-means clustering method, with lines colored according to which cluster they belonged. C First and second components of the PCA performed on lines included in the breeding cohort method, with lines colored according to which cohort they belonged. D First and third components of the PCA performed on lines included in the breeding cohort method, with lines colored according to which cohort they belonged.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2889/F3.medium.gif" width="440" height="353"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/9/2889/F3.medium.gif" width="440" height="353"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2889/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2889/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490898" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-30" class="first-child">Pairwise plots of components from two principal component analyses (PCA). A First and second components of the PCA performed on lines included in the K-means clustering method, with lines colored according to which cluster they belonged. B First and third components of the PCA performed on lines included in the K-means clustering method, with lines colored according to which cluster they belonged. C First and second components of the PCA performed on lines included in the breeding cohort method, with lines colored according to which cohort they belonged. D First and third components of the PCA performed on lines included in the breeding cohort method, with lines colored according to which cohort they belonged.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-18" class="subsection"><h4>Impact of underlying population structure:</h4><p id="p-31"><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> summarizes prediction accuracies from the K-means clustering method of assessing population structure impacts on prediction accuracy. ‘All clusters’ and ‘within cluster’ accuracies were similarly high for glaucousness and grain size, whereas for grain yield ‘all clusters’ was slightly higher and for relative maturity slightly lower. For all traits, predicting between cluster with a broad training set was more accurate than predicting between cluster with a narrow training set, but both were significantly less accurate than ‘all clusters’ and ‘within cluster’.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2889/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Boxplots showing prediction accuracies from the K-means clustering method for each category of training and validation set combinations, detailed in section 2.5.1. Prediction accuracy was calculated by correlating predictions of the validation set to the corresponding additive GBLUP values from the full model with all lines included. TKW represents thousand kernel weight." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-478565654" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Boxplots showing prediction accuracies from the K-means clustering method for each category of training and validation set combinations, detailed in section 2.5.1. Prediction accuracy was calculated by correlating predictions of the validation set to the corresponding additive GBLUP values from the full model with all lines included. TKW represents thousand kernel weight.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2889/F4.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/9/2889/F4.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2889/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2889/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490745" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-32" class="first-child">Boxplots showing prediction accuracies from the K-means clustering method for each category of training and validation set combinations, detailed in section 2.5.1. Prediction accuracy was calculated by correlating predictions of the validation set to the corresponding additive GBLUP values from the full model with all lines included. TKW represents thousand kernel weight.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h4>Impact of breeding cohort:</h4><p id="p-33"><a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> presents prediction accuracies from the breeding cohort method of assessing the impact of population structure on prediction accuracy. This shows that as more cohort years were represented in the training set, prediction accuracy increased significantly for grain yield, and slightly for relative maturity. Glaucousness and TKW however, had relatively stable prediction accuracy regardless of how many cohort years were represented in the training set. Prediction accuracies were highest for TKW and glaucousness, with relatively maturity being slightly lower and grain yield lower again.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2889/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Boxplots summarizing the prediction accuracies from the breeding cohort method, as detailed in section 2.5.2. At each trait the first boxplot represents one cohort year used as a training set to predict the subsequent cohort year (validation set). The second represents two consecutive cohort years used as training to predict the subsequent cohort year, and third represents three consecutive cohort years used to predict the subsequent." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-478565654" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Boxplots summarizing the prediction accuracies from the breeding cohort method, as detailed in section 2.5.2. At each trait the first boxplot represents one cohort year used as a training set to predict the subsequent cohort year (validation set). The second represents two consecutive cohort years used as training to predict the subsequent cohort year, and third represents three consecutive cohort years used to predict the subsequent.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2889/F5.medium.gif" width="410" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/9/2889/F5.medium.gif" width="410" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2889/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2889/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490792" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-34" class="first-child">Boxplots summarizing the prediction accuracies from the breeding cohort method, as detailed in section 2.5.2. At each trait the first boxplot represents one cohort year used as a training set to predict the subsequent cohort year (validation set). The second represents two consecutive cohort years used as training to predict the subsequent cohort year, and third represents three consecutive cohort years used to predict the subsequent.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-20" class="subsection"><h3>Impact of marker density on prediction accuracy</h3><p id="p-35"><a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> summarizes each marker selection using the consensus map to calculate unique positions, markers per map position and mean interval. This shows only a slight increase in the number of map positions at selections containing more than 5,000 markers. Therefore, at the selections with more than 5,000 markers, the mean position interval plateaus off and markers per map position increases. The mean MAF of the markers at each selection starts very high at 0.49 for the 100 marker selection, and steadily decreases to 0.26 for the 13,639 selection.</p><p id="p-36">The effect of marker density on prediction accuracy was assessed in the first instance through random five-fold cross validation, the results of which are summarized in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>. All four traits showed a sharp increase in accuracy before reaching a plateau at approximately 5,000 markers, with only a marginal increase in prediction accuracy when increasing from 5,000 to 17,181 markers. All traits showed the highest prediction accuracy when all available markers were used. Glaucousness, relative maturity and grain yield all had similar response curves, but TKW had a more pronounced increase in accuracy with marker number, particularly when increasing from 1,000 to 3,000 markers.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2889/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot showing the effect of marker density on prediction accuracy for each trait. Prediction accuracy was assessed by performing random five-fold cross-validation for each selection of markers, and correlating predictions of the validation set to the corresponding additive GBLUP values from the full model with all lines included. Marker subsets were selected to be evenly distributed over the genome and to have high minor allele frequency." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-478565654" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Plot showing the effect of marker density on prediction accuracy for each trait. Prediction accuracy was assessed by performing random five-fold cross-validation for each selection of markers, and correlating predictions of the validation set to the corresponding additive GBLUP values from the full model with all lines included. Marker subsets were selected to be evenly distributed over the genome and to have high minor allele frequency.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2889/F6.medium.gif" width="440" height="322"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/9/2889/F6.medium.gif" width="440" height="322"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2889/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2889/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490803" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-37" class="first-child">Plot showing the effect of marker density on prediction accuracy for each trait. Prediction accuracy was assessed by performing random five-fold cross-validation for each selection of markers, and correlating predictions of the validation set to the corresponding additive GBLUP values from the full model with all lines included. Marker subsets were selected to be evenly distributed over the genome and to have high minor allele frequency.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-21" class="subsection"><h3>Effect of interaction Between marker density and population structure on prediction accuracy</h3><p id="p-38">The K-means clustering analysis was repeated for each marker density in order to investigate the interaction between population structure and marker density (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). Similar to the five-fold cross validation analysis, prediction accuracies increased sharply up to approximately 3,000 markers before plateauing. Similar responses were observed for ‘all clusters’ and ‘within cluster’ prediction structures across all traits. Between cluster prediction saw greater response to increased marker density, particularly with broad training when increasing from 100 to 1,000 markers. Relative maturity saw a slight decrease in prediction accuracy when marker number was increased beyond 5,000.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2889/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plots showing the interactive response of prediction accuracy to marker density and population structure. The K-means clustering method detailed in section 2.5.1 was repeated for each selection of markers. Marker subsets were selected to be evenly distributed over the genome and to have high minor allele frequency." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-478565654" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Plots showing the interactive response of prediction accuracy to marker density and population structure. The K-means clustering method detailed in section 2.5.1 was repeated for each selection of markers. Marker subsets were selected to be evenly distributed over the genome and to have high minor allele frequency.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2889/F7.medium.gif" width="440" height="355"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/8/9/2889/F7.medium.gif" width="440" height="355"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2889/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2889/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490741" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-39" class="first-child">Plots showing the interactive response of prediction accuracy to marker density and population structure. The K-means clustering method detailed in section 2.5.1 was repeated for each selection of markers. Marker subsets were selected to be evenly distributed over the genome and to have high minor allele frequency.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-22"><h2 class="">Discussion</h2><p id="p-40">If plant breeders are to effectively apply genomic selection in their breeding programs, they require a sound understanding of factors affecting prediction accuracy in large scale germplasm datasets. In the present study we utilized a panel of 10,375 lines sourced from an active breeding program to investigate the effect and interaction of training set size, population structure and marker density on prediction accuracy. The findings presented here will assist breeders in optimizing their programs, allowing them to make the most effective and efficient use of their resources when implementing genomic selection.</p><div id="sec-23" class="subsection"><h3>Effect of training set size on prediction accuracy</h3><p id="p-41">An important factor influencing genomic prediction accuracy is the size of the training set used to develop the prediction calibration (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Nakaya and Isobe 2012</a>). However, research questions pertaining to this have previously proven difficult to address, as the large number of lines required to locate the point of diminishing returns has an often prohibitively high cost of genotyping. The data set analyzed here provides a unique opportunity to investigate the effect of population size on prediction accuracy in bread wheat. Prediction accuracy increased substantially when the training set size was increased from 250 lines to approximately 2,000, after which the rate of increase slowed. While an acceptable prediction accuracy would be determined by the breeder on a case by case basis, if we look at an accuracy of 0.95 as an example, this is achieved at a training set size of 3,930 and 7,450 for glaucousness and relative maturity respectively. This result confirms previous findings from smaller populations (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Heffner <em>et al.</em>, 2011a</a>, <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">b</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Isidro <em>et al.</em>, 2015</a>), and extends the relationship to larger training sets showing there is a point at which accuracy begins to plateau in response to increased training set size. Plant breeders should take this result into account when weighing up the benefit of including additional lines in a training set. While there were differences between traits in the level of accuracy achieved, the trend in response to training set size was consistent for all traits despite their differences in genetic complexity. This suggests that response in prediction accuracy to training set size is not dependent on the complexity and genetic architecture of the trait.</p><p id="p-42">The difference in prediction accuracy between traits was more pronounced at smaller training set sizes. This was also driven by the genetic complexity of the trait, as more lines are needed to provide the high number of allelic observations required to accurately predict small effect QTL (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gilmour 2007</a>). Prediction accuracies in this analysis varied more within the smaller training set sizes than the large, particularly for grain yield. This indicates population structure was present and the variation in accuracy was likely caused by the presence or absence of highly related lines across training and validation sets. (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Poland <em>et al.</em>, 2012</a>). In the next section we investigate how the relatedness between training and validation sets affects the resultant accuracy.</p></div><div id="sec-24" class="subsection"><h3>Effect of population structure on prediction accuracy</h3><p id="p-43">K-means clustering produced five genetically distinct clusters, which is demonstrated in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>. Prediction accuracy within and between clusters was tested using structured cross-validation with training sets containing 1,000 lines and validation sets containing 500 lines. The breeding cohorts were less distinct as they were all sourced from the southern breeding program. The accuracy of predicting one cohort using a training set sourced from one, two or three prior cohorts was tested using training and validation sets of the same size as those in the K-means clustering method. This unique assessment is representative of how genomic prediction would be applied in a commercial breeding program.</p><p id="p-44">In the K-means cluster method, ‘all clusters’ and ‘within cluster’ prediction accuracies were similar for glaucousness and TKW. The training sets of both prediction structures directly represent the clusters in their respective validation set, the only difference being that ‘all clusters’ uses all five clusters whereas ‘within cluster’ uses just one. This result therefore suggests the broadness of the training and validation sets has little effect on prediction accuracy when the training set contains at least some lines that are highly representative of those being predicted. For relative maturity however, ‘within cluster’ prediction accuracy was slightly higher than ‘all clusters’, and the reverse was observed for grain yield. There are several large effect photoperiod and vernalisation genes that control maturity (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Snape <em>et al.</em>, 2001</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Cane <em>et al.</em>, 2013</a>), and the predominating genes differ between clusters (data not shown). The higher accuracy when predicting maturity within cluster was therefore likely to be caused by the key large effect genes having greater representation in the training set. For grain yield on the other hand, the increased diversity was beneficial as ‘all clusters’ showed higher prediction accuracy than ‘within cluster’. This is because there was more and comparable phenotypic diversity represented within both the training and validation sets for ‘all clusters’.</p><p id="p-45">For all traits, predicting a single cluster using a broad training set produced higher accuracies than predicting with narrow training, but was substantially less accurate than ‘within cluster’ and ‘all clusters’. This shows that prediction accuracy is significantly higher when the training set contains close relatives of lines in the validation set, but accuracy can also be increased by including more genetic diversity in the training set. Breeders should therefore design genetically diverse training sets that are highly related to the prediction set in order to maximize genetic response to genomic selection. This is corroborated by the results of the breeding cohort cross-validation, where prediction accuracy was improved for grain yield and relative maturity by including more cohort years (and therefore more diversity) in the training set. With increased genetic diversity and high SNP density, the training set can better capture short haplotype effects that are relevant to the validation set. This type of calibration is based on short haplotype effects and linkage disequilibrium information, and is suggested by <a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Hickey <em>et al.</em> (2014)</a> to be less susceptible to breaking down after multiple breeding cycles.</p><p id="p-46">The breeding cohort analysis is the most representative of how genomic selection would be applied in a breeding program, predicting the current cohort using previous cohorts. The increase in prediction accuracy with more cohorts in the training set was most pronounced for grain yield, and supports previous findings in rye (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Auinger <em>et al.</em>, 2016</a>). <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Muir (2007)</a> observed through simulation of animal breeding that continued selection over multiple generations eventually reduced prediction accuracy. The difference between that study and the present is the longer generation intervals of wheat breeding and consequently the fewer number of generations represented. The results presented here show that incorporating more breeding cohorts in the training set is beneficial in a conventional breeding program with a long generation interval. A recent study by <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gorjanc <em>et al.</em> (2018)</a> investigates the response in a rapid cycling program which uses genomic selection to quickly identify parents.</p><p id="p-47">While grain yield undergoes continual and intense selection within the breeding program, relative maturity and TKW are threshold traits and therefore change less over time, which results in them benefiting less from the inclusion of additional cohort years in the training set. Glaucousness undergoes no direct selection meaning genetic change will only occur through correlated response, and it therefore sees little benefit from adding more cohort years to the training set.</p></div><div id="sec-25" class="subsection"><h3>Marker density</h3><p id="p-48">The effect of marker density on prediction accuracy was assessed with a random five-fold cross validation analysis performed with various marker densities. All traits experienced a strong response to increases in marker density up to 5,000 markers, showing that this was sufficient for generating a relatively accurate prediction calibration within this panel. This number is significantly higher than the plateau point of previous studies in smaller populations (<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Heffner <em>et al.</em>, 2011b</a>), as high marker densities only facilitate finer resolution and more accurate estimates of QTL effects when combined with large population size and low linkage disequilibrium (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Huang <em>et al.</em>, 2012</a>). TKW benefited from increased marker density more than the other traits, which could be explained by its quantitative genetic nature requiring more markers to accurately estimate its many small QTL effects (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Zhang <em>et al.</em>, 2015</a>). However, grain yield is also a highly quantitative trait and it saw a similar response curve to the more qualitative traits glaucousness and relative maturity.</p><p id="p-49">The interactive effect of marker density and population structure on prediction accuracy was assessed by repeating the K-means cluster analysis with various marker densities. The density at which prediction accuracy plateaued was slightly lower than that observed in the random five-fold cross validation. This is consistent with previous studies using smaller data sets where additional markers benefited prediction accuracy more when larger training sets were used (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Heffner <em>et al.</em>, 2011a</a>,<a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">b</a>). Prediction accuracy responded more to increased marker density when predicting between clusters, particularly when there was more genetic diversity in the training set. This is consistent with the findings of <a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">Hickey <em>et al.</em> (2014)</a>, where in a simulated maize data set the required marker density was lower when closely related material was shared between training and validation sets. The study also showed there was greater response to increased marker density when the training set contained more diversity, which corroborates our findings. The slight decrease in prediction accuracy at high marker densities for relative maturity is likely due to excess markers overfitting the model (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Heslot <em>et al.</em>, 2012</a>). A similar result was seen in <a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">Heffner <em>et al.</em> (2011a)</a>, where higher marker densities resulted in lower prediction accuracy in bi-parental wheat populations.</p></div><div id="sec-26" class="subsection"><h3>Conclusions</h3><p id="p-50">Here we used a wheat panel of unprecedented size to investigate several key factors affecting genomic prediction accuracy that previously have not been explored at this scale. We showed there is a point at which prediction accuracy begins to plateau in response to training set size, and that this response is independent from the genetic complexity of the trait. The population structure analyses showed that relatedness between training and validation sets has a large effect on prediction accuracy, but importantly when relatedness is low, as is often the case when applying genomic selection, prediction accuracy can be increased by increasing diversity in the training set. We also found that traits under higher selection pressure can be more accurately predicted by including several previous breeding cohorts in the training set. This was shown for up to three previous cohorts, but further work should be done to explore how stable this trend is across different breeding programs and more cohorts. By assessing the interaction between marker density and population structure, we showed the response to increased marker density is larger when using a diverse training set and predicting from poorly related training sets. The work presented herein provides a framework for pragmatic plant breeders to optimally design their genomic selection training strategy to achieve high selection accuracy and subsequent rates of genetic gain.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-51">The authors thank Brian Cullis and Alison Smith for their guidance and substantial work in establishing the project, and Emi Tanaka for her work on the project. We also thank Australian Grain Technologies for their contribution in managing the Roseworthy field experiment and funding the research. We acknowledge funding from the Australian Research Council, and the Grains Research and Development Corporation.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-52">Supplemental material available at Figshare: <a href="https://figshare.com/s/287c2c7f1623008487a5">https://figshare.com/s/287c2c7f1623008487a5</a>.</p></li><li class="fn" id="fn-4"><p id="p-53"><em>Communicating editor: J.-L. Jannink</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-04-11"><span class="received-label">Received </span>April 11, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-06-29"><span class="accepted-label">Accepted </span>June 29, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Norman <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.1" data-doi="10.1007/s00122-016-2756-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Auinger</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schӧnleben</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehermeier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korzun</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Model training across multiple breeding cycles significantly improves genomic prediction accuracy in rye (<em>Secale cereale L</em>.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">2043</span>–<span class="cit-lpage">2053</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2756-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D2043%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2756-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2756-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.2" data-doi="10.1007/s00122-017-3011-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ben Hassen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartholomé</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orasen</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colombi</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Rice diversity panel provides accurate genomic predictions for complex traits in the progenies of biparental crosses involving members of the panel.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">417</span>–<span class="cit-lpage">435</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-017-3011-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D131%26rft.spage%253D417%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-017-3011-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-017-3011-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.3" data-doi="10.1007/s00122-011-1740-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Izanloo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuchel</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chalmers</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Identification of novel quantitative trait loci for days to ear emergence and flag leaf glaucousness in a bread wheat (<em>Triticum aestivum L</em>.) population adapted to southern Australian conditions.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">697</span>–<span class="cit-lpage">711</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-011-1740-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DBennett%26rft.auinit1%253DD.%26rft.volume%253D124%26rft.issue%253D4%26rft.spage%253D697%26rft.epage%253D711%26rft.atitle%253DIdentification%2Bof%2Bnovel%2Bquantitative%2Btrait%2Bloci%2Bfor%2Bdays%2Bto%2Bear%2Bemergence%2Band%2Bflag%2Bleaf%2Bglaucousness%2Bin%2Ba%2Bbread%2Bwheat%2B%2528Triticum%2Baestivum%2BL.%2529%2Bpopulation%2Badapted%2Bto%2Bsouthern%2BAustralian%2Bconditions.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1740-3%26rft_id%253Dinfo%253Apmid%252F22045047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1740-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22045047&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.4" data-doi="10.1007/s00122-012-1831-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Izanloo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuchel</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langridge</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Genetic dissection of grain yield and physical grain quality in bread wheat (<em>Triticum aestivum L</em>.) under water-limited environments.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">271</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-012-1831-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DBennett%26rft.auinit1%253DD.%26rft.volume%253D125%26rft.issue%253D2%26rft.spage%253D255%26rft.epage%253D271%26rft.atitle%253DGenetic%2Bdissection%2Bof%2Bgrain%2Byield%2Band%2Bphysical%2Bgrain%2Bquality%2Bin%2Bbread%2Bwheat%2B%2528Triticum%2Baestivum%2BL.%2529%2Bunder%2Bwater-limited%2Benvironments.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1831-9%26rft_id%253Dinfo%253Apmid%252F22374139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1831-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22374139&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.5" data-doi="10.1007/s00122-014-2403-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bentley</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scutari</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gosman</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faure</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bedford</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Applying association mapping and genomic selection to the dissection of key traits in elite European wheat.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">2619</span>–<span class="cit-lpage">2633</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-014-2403-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D2619%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2403-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2403-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-8.9.2889.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">ASReml-R reference manual</span>, <span class="cit-publ-name">Queensland Department of Primary Industries</span>, <span class="cit-publ-loc">Queensland, Australia</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.7" data-doi="10.1071/CP13086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cane</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eagles</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laurie</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trevaskis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vallance</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title"><em>Ppd-B1</em> and <em>Ppd-D1</em> and their effects in southern Australian wheat.</span> <abbr class="cit-jnl-abbrev">Crop Pasture Sci.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">100</span>–<span class="cit-lpage">114</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1071/CP13086</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BPasture%2BSci.%26rft.volume%253D64%26rft.spage%253D100%26rft_id%253Dinfo%253Adoi%252F10.1071%252FCP13086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/CP13086&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000320694100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franco</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction of gene bank wheat landraces. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1834</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D6%26rft.spage%253D1819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.9" data-doi="10.1534/genetics.112.147983"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic prediction in animals and plants: simulation of data, validation, reporting, and benchmarking.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">365</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.147983</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.147983%26rft_id%253Dinfo%253Apmid%252F23222650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzI4ODkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.10" data-doi="10.1007/s00122-014-2341-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bansal</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bariana</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayden</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic prediction for rust resistance in diverse wheat landraces.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">1795</span>–<span class="cit-lpage">1803</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-014-2341-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D1795%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2341-8%26rft_id%253Dinfo%253Apmid%252F24965887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2341-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24965887&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.11" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-genome regression and prediction methods applied to plant and animal breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.143313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzI4ODkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.12" data-doi="10.1016/j.tplants.2014.05.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Desta</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ortiz</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic selection: genome-wide prediction in plant improvement.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">592</span>–<span class="cit-lpage">601</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2014.05.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D19%26rft.spage%253D592%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2014.05.006%26rft_id%253Dinfo%253Apmid%252F24970707%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2014.05.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24970707&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.13" data-doi="10.1186/1297-9686-43-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forni</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Different genomic relationship matrices for single-step analysis using phenotypic, pedigree and genomic information.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-43-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DForni%26rft.auinit1%253DS.%26rft.volume%253D43%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DDifferent%2Bgenomic%2Brelationship%2Bmatrices%2Bfor%2Bsingle-step%2Banalysis%2Busing%2Bphenotypic%252C%2Bpedigree%2Band%2Bgenomic%2Binformation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-1%26rft_id%253Dinfo%253Apmid%252F21208445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21208445&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.14" data-doi="10.1016/j.csda.2006.12.031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mixed model regression mapping for QTL detection in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Comput. Stat. Data Anal.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">3749</span>–<span class="cit-lpage">3764</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.csda.2006.12.031</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BStat.%2BData%2BAnal.%26rft.volume%253D51%26rft.spage%253D3749%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.csda.2006.12.031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.csda.2006.12.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.15" data-doi="10.2307/2533274"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Average information REML: an efficient algorithm for variance parameter estimation in linear mixed models.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">1440</span>–<span class="cit-lpage">1450</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/2533274</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D51%26rft.spage%253D1440%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2533274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2533274&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995TQ15600022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.16" data-doi="10.2307/1400446"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Accounting for natural and extraneous variation in the analysis of field experiments.</span> <abbr class="cit-jnl-abbrev">J. Agric. Biol. Environ. Stat.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">269</span>–<span class="cit-lpage">293</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/1400446</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BBiol.%2BEnviron.%2BStat.%26rft.volume%253D2%26rft.spage%253D269%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1400446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1400446&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-8.9.2889.17" data-doi="10.1007/s00122&amp;ndash;018&amp;ndash;3125&amp;ndash;3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gorjanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaynor</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Optimal cross selection for long-term genetic gain in two-part programs with rapid recurrent genomic selection.</span> <span class="cit-source">Theor. Appl. Genet</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122–018–3125–3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122%2526ndash%253B018%2526ndash%253B3125%2526ndash%253B3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122&amp;ndash;018&amp;ndash;3125&amp;ndash;3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.18" data-doi="10.1007/s00122-016-2840-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korzun</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bothe</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebmeyer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome-wide mapping and prediction suggests presence of local epistasis in a vast elite winter wheat populations adapted to central europe.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">635</span>–<span class="cit-lpage">647</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2840-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D635%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2840-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2840-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.19" data-doi="10.2135/cropsci2008.08.0512"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection for crop improvement.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2008.08.0512</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHeffner%26rft.auinit1%253DE.%2BL.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D12%26rft.atitle%253DGenomic%2BSelection%2Bfor%2BCrop%2BImprovement%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.08.0512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.08.0512&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000263344300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.20" data-doi="10.2135/cropsci2011.05.0253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souza</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">a</span> <span class="cit-article-title">Genomic selection accuracy for grain quality traits in biparental wheat populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">2597</span>–<span class="cit-lpage">2606</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2011.05.0253</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D51%26rft.spage%253D2597%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.05.0253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.05.0253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000295839200031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.21" data-doi="10.3835/plantgenome2010.12.0029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">b</span> <span class="cit-article-title">Genomic selection accuracy using multifamily prediction models in a wheat breeding program.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">75</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2010.12.0029</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D65%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2010.12.0029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2010.12.0029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.22" data-doi="10.2307/3001853"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1953</span> <span class="cit-article-title">Estimation of variance and covariance components.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">226</span>–<span class="cit-lpage">252</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/3001853</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D9%26rft.spage%253D226%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3001853%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3001853&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1953UG26500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.23" data-doi="10.2135/cropsci2011.06.0297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in plant breeding: a comparison of models.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">146</span>–<span class="cit-lpage">160</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2011.06.0297</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0297&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.24" data-doi="10.2135/cropsci2013.03.0195"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hearne</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Evaluation of genomic selection training population designs and genotyping strategies in plant breeding programs using simulation.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1476</span>–<span class="cit-lpage">1488</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2013.03.0195</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D1476%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.03.0195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.03.0195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.25" data-doi="10.1111/j.1467-7652.2012.00702.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">George</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forrest</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kilian</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayden</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A multiparent advanced generation inter-cross population for genetic analysis in wheat.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">826</span>–<span class="cit-lpage">839</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1467-7652.2012.00702.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BBiotechnol%2BJ%26rft.aulast%253DHuang%26rft.auinit1%253DB.%2BE.%26rft.volume%253D10%26rft.issue%253D7%26rft.spage%253D826%26rft.epage%253D839%26rft.atitle%253DA%2Bmultiparent%2Badvanced%2Bgeneration%2Binter-cross%2Bpopulation%2Bfor%2Bgenetic%2Banalysis%2Bin%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-7652.2012.00702.x%26rft_id%253Dinfo%253Apmid%252F22594629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-7652.2012.00702.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22594629&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307051700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.26" data-doi="10.1007/s00122-006-0346-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cloutier</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lycar</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Radovanovic</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Humphreys</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Molecular detection of QTLs for agronomic and quality traits in a doubled haploid population derived from two Canadian wheats (<em>Triticum aestivum L</em>.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">753</span>–<span class="cit-lpage">766</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-006-0346-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DHuang%26rft.auinit1%253DX.%2BQ.%26rft.volume%253D113%26rft.issue%253D4%26rft.spage%253D753%26rft.epage%253D766%26rft.atitle%253DMolecular%2Bdetection%2Bof%2BQTLs%2Bfor%2Bagronomic%2Band%2Bquality%2Btraits%2Bin%2Ba%2Bdoubled%2Bhaploid%2Bpopulation%2Bderived%2Bfrom%2Btwo%2BCanadian%2Bwheats%2B%2528Triticum%2Baestivum%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0346-7%26rft_id%253Dinfo%253Apmid%252F16838135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0346-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16838135&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239301500018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.27" data-doi="10.1007/s00122-014-2418-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Isidro</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Training set optimization under population structure in genomic selection.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">145</span>–<span class="cit-lpage">158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-014-2418-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D145%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2418-4%26rft_id%253Dinfo%253Apmid%252F25367380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2418-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25367380&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.28" data-doi="10.1093/bfgp/elq001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic selection in plant breeding: from theory to practice.</span> <abbr class="cit-jnl-abbrev">Brief. Funct. Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">177</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bfgp/elq001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BFunct.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felq001%26rft_id%253Dinfo%253Apmid%252F20156985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bfgp/elq001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20156985&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276191200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.29" data-doi="10.1007/s00122-007-0629-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuchel</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langridge</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eagles</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jefferies</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic dissection of grain yield in bread wheat. I. QTL analysis.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">1029</span>–<span class="cit-lpage">1041</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-007-0629-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DKuchel%26rft.auinit1%253DH.%26rft.volume%253D115%26rft.issue%253D8%26rft.spage%253D1029%26rft.epage%253D1041%26rft.atitle%253DGenetic%2Bdissection%2Bof%2Bgrain%2Byield%2Bin%2Bbread%2Bwheat.%2BI.%2BQTL%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-007-0629-7%26rft_id%253Dinfo%253Apmid%252F17713755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-007-0629-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17713755&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251319400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.30" data-doi="10.1186/1297-9686-43-19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seefried</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinhardt</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rensing</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thaller</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Impacts of both reference population size and inclusion of a residual polygenic effect on the accuracy of genomic prediction.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">19</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-43-19</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DLiu%26rft.auinit1%253DZ.%26rft.volume%253D43%26rft.spage%253D19%26rft.epage%253D19%26rft.atitle%253DImpacts%2Bof%2Bboth%2Breference%2Bpopulation%2Bsize%2Band%2Binclusion%2Bof%2Ba%2Bresidual%2Bpolygenic%2Beffect%2Bon%2Bthe%2Baccuracy%2Bof%2Bgenomic%2Bprediction.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-19%26rft_id%253Dinfo%253Apmid%252F21586131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-19&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21586131&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-other" id="cit-8.9.2889.31"><div class="cit-metadata unstructured">Lund M, Van den Berg I, Ma P, Brøndum R, Su G (2016) How to improve genomic predictions in small dairy cattle populations. Animal 10(6):1042–1049</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.32" data-doi="10.1007/s00122-014-2322-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maphosa</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langridge</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parent</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emebiri</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic control of grain yield and grain physical characteristics in a bread wheat population grown under a range of environmental conditions.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">1607</span>–<span class="cit-lpage">1624</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-014-2322-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D1607%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2322-y%26rft_id%253Dinfo%253Apmid%252F24865506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2322-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24865506&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMjg4OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.34" data-doi="10.1007/s00122-016-2818-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ametz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gungor</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akgöl</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Epure</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic assisted selection for enhancing line breeding: merging genomic and phenotypic selection in winter wheat breeding programs with preliminary yield trials.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">363</span>–<span class="cit-lpage">376</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2818-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D363%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2818-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2818-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.35" data-doi="10.1111/j.1439-0388.2007.00700.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muir</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Comparison of genomic and traditional BLUP-estimated breeding value accuracy and selection response under alternative trait and genomic parameters.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">342</span>–<span class="cit-lpage">355</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0388.2007.00700.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DMuir%26rft.auinit1%253DW.%2BM.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D342%26rft.epage%253D355%26rft.atitle%253DComparison%2Bof%2Bgenomic%2Band%2Btraditional%2BBLUP-estimated%2Bbreeding%2Bvalue%2Baccuracy%2Band%2Bselection%2Bresponse%2Bunder%2Balternative%2Btrait%2Band%2Bgenomic%2Bparameters.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2007.00700.x%26rft_id%253Dinfo%253Apmid%252F18076471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2007.00700.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18076471&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251432800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.36" data-doi="10.1093/aob/mcs109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakaya</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isobe</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Will genomic selection be a practical method for plant breeding?</span> <abbr class="cit-jnl-abbrev">Ann. Bot. (Lond.)</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">1303</span>–<span class="cit-lpage">1316</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/aob/mcs109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%2B%2528Lond.%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcs109%26rft_id%253Dinfo%253Apmid%252F22645117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/aob/mcs109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22645117&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.37" data-doi="10.1007/s00122-017-2975-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Norman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Telfer</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Increased genomic prediction accuracy in wheat breeding using a large Australian panel.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">2543</span>–<span class="cit-lpage">2555</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-017-2975-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D2543%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-017-2975-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-017-2975-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.38" data-doi="10.1093/biomet/58.3.545"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">Recovery of inter-block information when block sizes are unequal.</span> <abbr class="cit-jnl-abbrev">Biometrika</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">545</span>–<span class="cit-lpage">554</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/biomet/58.3.545</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft.stitle%253DBiometrika%26rft.aulast%253DPATTERSON%26rft.auinit1%253DH.%2BD.%26rft.volume%253D58%26rft.issue%253D3%26rft.spage%253D545%26rft.epage%253D554%26rft.atitle%253DRecovery%2Bof%2Binter-block%2Binformation%2Bwhen%2Bblock%2Bsizes%2Bare%2Bunequal%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252F58.3.545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/biomet/58.3.545&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1971L054700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.39" data-doi="10.1371/journal.pgen.0020190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Population structure and eigenanalysis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e190</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.0020190</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPatterson%26rft.auinit1%253DN.%26rft.volume%253D2%26rft.issue%253D12%26rft.spage%253De190%26rft.epage%253De190%26rft.atitle%253DPopulation%2Bstructure%2Band%2Beigenanalysis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020190%26rft_id%253Dinfo%253Apmid%252F17194218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020190&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17194218&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.40" data-doi="10.3835/plantgenome2012.06.0006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in wheat breeding using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">113</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2012.06.0006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D5%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.06.0006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.06.0006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.41" data-doi="10.1038/nrg2813"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">New approaches to population stratification in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">459</span>–<span class="cit-lpage">463</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2813</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPrice%26rft.auinit1%253DA.%2BL.%26rft.volume%253D11%26rft.issue%253D7%26rft.spage%253D459%26rft.epage%253D463%26rft.atitle%253DNew%2Bapproaches%2Bto%2Bpopulation%2Bstratification%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2813%26rft_id%253Dinfo%253Apmid%252F20548291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2813&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20548291&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278998500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-web" id="cit-8.9.2889.42"><div class="cit-metadata unstructured">R Core Team, (2017) R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna, Austria, URL <a href="https://www.R-project.org/">https://www.R-project.org/</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.43" data-doi="10.1023/A:1017541505152"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Snape</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarma</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quarrie</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fish</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galiba</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Mapping genes for flowering time and frost tolerance in cereals using precise genetic stocks.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">309</span>–<span class="cit-lpage">315</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1023/A:1017541505152</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D120%26rft.spage%253D309%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1017541505152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1017541505152&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000170950800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.44" data-doi="10.2527/jas.2007-0010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Solberg</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonesson</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H.</span></span></li></ol><cite> <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genomic selection using different marker types and densities.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">2447</span>–<span class="cit-lpage">2454</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas.2007-0010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2007-0010%26rft_id%253Dinfo%253Apmid%252F18407980%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2007-0010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18407980&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259587000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.45" data-doi="10.1007/s10681-008-9794-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">QTL analysis of kernel shape and weight using recombinant inbred lines in wheat.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">615</span>–<span class="cit-lpage">624</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10681-008-9794-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D165%26rft.spage%253D615%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-008-9794-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-008-9794-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000262087300019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.46" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.47" data-doi="10.1111/j.1365-3180.1974.tb01084.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zadoks</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Konzak</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite> <span class="cit-pub-date">1974</span> <span class="cit-article-title">A decimal code for the growth stages of cereals.</span> <abbr class="cit-jnl-abbrev">Weed Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">415</span>–<span class="cit-lpage">421</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-3180.1974.tb01084.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWeed%2BRes.%26rft.volume%253D14%26rft.spage%253D415%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-3180.1974.tb01084.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-3180.1974.tb01084.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1974V546200012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2889.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erbe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ober</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Accuracy of whole-genome prediction using a genetic architecture-enhanced variance-covariance matrix. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">615</span>–<span class="cit-lpage">627</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D5%26rft.spage%253D615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/9/2889.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/9/2881" title="Targeted Gene Sequencing in Children with Crohn’s Disease and Their Parents: Implications for Missing Heritability" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/9/2901" title="Tripartite Chromatin Localization of Budding Yeast Shugoshin Involves Higher-Ordered Architecture of Mitotic Chromosomes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/9"><span>Volume 8 Issue 9, September 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/9.cover-source.jpg?itok=77yoxn8o" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/9/2889" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-nK-RmZAfu83DVVYF3HcO1C_6ub6UYWNCUv-8fsR_onk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/9/2889" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/9/2889" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/9/2889" class="active" data-icon-position="" data-hide-link-title="0">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/490724" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-0SWgHRc29y6-5wISVp_qZc6uaVroUr2E-UQnWjbI7Wg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/9/2889.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/9/2889" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-AUb4SRC-wVa_TNxwGJTI8H7ZkqCdz0cdV5kCmE3rvLY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/9/2889" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/9/2889.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724--2" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0794-4907" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Adam Norman</span>, <span class="highwire-citation-author" data-delta="1">Julian Taylor</span>, <span class="highwire-citation-author" data-delta="2">James Edwards</span> and <span class="highwire-citation-author" data-delta="3">Haydn Kuchel</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2889-2899;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200311 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490724--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Adam Norman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Agriculture, Food & Wine, University of Adelaide</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adam%2BNorman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Norman%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAdam%2BNorman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0794-4907" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Adam Norman</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Julian Taylor </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Agriculture, Food & Wine, University of Adelaide</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julian%2BTaylor%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Taylor%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJulian%2BTaylor%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">James Edwards </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Agriculture, Food & Wine, University of Adelaide</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BEdwards%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Edwards%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BEdwards%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Haydn Kuchel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Agriculture, Food & Wine, University of Adelaide</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haydn%2BKuchel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kuchel%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHaydn%2BKuchel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/9/2889" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/9/2889&amp;title=Optimising%20Genomic%20Selection%20in%20Wheat%3A%20Effect%20of%20Marker%20Density%2C%20Population%20Size%20and%20Population%20Structure%20on%20Prediction%20Accuracy" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/9/2889&amp;title=Optimising%20Genomic%20Selection%20in%20Wheat%3A%20Effect%20of%20Marker%20Density%2C%20Population%20Size%20and%20Population%20Structure%20on%20Prediction%20Accuracy" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/9/2889&amp;title=Optimising%20Genomic%20Selection%20in%20Wheat%3A%20Effect%20of%20Marker%20Density%2C%20Population%20Size%20and%20Population%20Structure%20on%20Prediction%20Accuracy" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/9/2889&amp;text=Optimising%20Genomic%20Selection%20in%20Wheat%3A%20Effect%20of%20Marker%20Density%2C%20Population%20Size%20and%20Population%20Structure%20on%20Prediction%20Accuracy" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/9/2889&amp;title=Optimising%20Genomic%20Selection%20in%20Wheat%3A%20Effect%20of%20Marker%20Density%2C%20Population%20Size%20and%20Population%20Structure%20on%20Prediction%20Accuracy" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/9/2889&amp;t=Optimising%20Genomic%20Selection%20in%20Wheat%3A%20Effect%20of%20Marker%20Density%2C%20Population%20Size%20and%20Population%20Structure%20on%20Prediction%20Accuracy" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/9/2889&amp;title=Optimising%20Genomic%20Selection%20in%20Wheat%3A%20Effect%20of%20Marker%20Density%2C%20Population%20Size%20and%20Population%20Structure%20on%20Prediction%20Accuracy" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/9/2889&amp;title=Optimising%20Genomic%20Selection%20in%20Wheat%3A%20Effect%20of%20Marker%20Density%2C%20Population%20Size%20and%20Population%20Structure%20on%20Prediction%20Accuracy" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/490724/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724--3" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0794-4907" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Adam Norman</span>, <span class="highwire-citation-author" data-delta="1">Julian Taylor</span>, <span class="highwire-citation-author" data-delta="2">James Edwards</span> and <span class="highwire-citation-author" data-delta="3">Haydn Kuchel</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2889-2899;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200311 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490724--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Adam Norman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Agriculture, Food & Wine, University of Adelaide</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adam%2BNorman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Norman%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAdam%2BNorman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0794-4907" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Adam Norman</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Julian Taylor </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Agriculture, Food & Wine, University of Adelaide</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julian%2BTaylor%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Taylor%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJulian%2BTaylor%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">James Edwards </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Agriculture, Food & Wine, University of Adelaide</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BEdwards%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Edwards%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BEdwards%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Haydn Kuchel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Agriculture, Food & Wine, University of Adelaide</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haydn%2BKuchel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kuchel%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHaydn%2BKuchel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2889&amp;rft.epage=2899&amp;rft.atitle=Optimising%20Genomic%20Selection%20in%20Wheat%3A%20Effect%20of%20Marker%20Density%2C%20Population%20Size%20and%20Population%20Structure%20on%20Prediction%20Accuracy&amp;rft.volume=8&amp;rft.issue=9&amp;rft.date=2018-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Norman%2C+Adam&amp;rft.au=Taylor%2C+Julian&amp;rft.au=Edwards%2C+James&amp;rft.au=Kuchel%2C+Haydn"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/490724/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/490724/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/490724/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/490724/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/490724/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/490724/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/490724/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/490724/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/490724/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/490724/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/490724/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/490724/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="last even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/9/2889" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-22" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/9/2889.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/9/2889.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/9/2889.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/9/2889.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
