<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/9/3039.full.txt" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/9/3039.full.pdf" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/9/3039/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/9/3039.atom" />
<meta name="HW.pisa" content="ggg;8/9/3039" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models" />
<meta name="DC.Identifier" content="10.1534/g3.118.200435" />
<meta name="DC.Date" content="2018-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Granato et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="One of the major issues in plant breeding is the occurrence of genotype × environment (GE) interaction. Several models have been created to understand this phenomenon and explore it. In the genomic era, several models were employed to improve selection by using markers and account for GE interaction simultaneously. Some of these models use special genetic covariance matrices. In addition, the scale of multi-environment trials is getting larger, and this increases the computational challenges. In this context, we propose an R package that, in general, allows building GE genomic covariance matrices and fitting linear mixed models, in particular, to a few genomic GE models. Here we propose two functions: one to prepare the genomic kernels accounting for the genomic GE and another to perform genomic prediction using a Bayesian linear mixed model. A specific treatment is given for sparse covariance matrices, in particular, to block diagonal matrices that are present in some GE models in order to decrease the computational demand. In empirical comparisons with Bayesian Genomic Linear Regression (BGLR), accuracies and the mean squared error were similar; however, the computational time was up to five times lower than when using the classic approach. Bayesian Genomic Genotype × Environment Interaction (BGGE) is a fast, efficient option for creating genomic GE kernels and making genomic predictions." />
<meta name="DC.Contributor" content="Italo Granato" />
<meta name="DC.Contributor" content="Jaime Cuevas" />
<meta name="DC.Contributor" content="Francisco Luna-Vázquez" />
<meta name="DC.Contributor" content="Jose Crossa" />
<meta name="DC.Contributor" content="Osval Montesinos-López" />
<meta name="DC.Contributor" content="Juan Burgueño" />
<meta name="DC.Contributor" content="Roberto Fritsche-Neto" />
<meta name="article:published_time" content="2018-09-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;One of the major issues in plant breeding is the occurrence of genotype × environment (GE) interaction. Several models have been created to understand this phenomenon and explore it. In the genomic era, several models were employed to improve selection by using markers and account for GE interaction simultaneously. Some of these models use special genetic covariance matrices. In addition, the scale of multi-environment trials is getting larger, and this increases the computational challenges. In this context, we propose an R package that, in general, allows building GE genomic covariance matrices and fitting linear mixed models, in particular, to a few genomic GE models. Here we propose two functions: one to prepare the genomic kernels accounting for the genomic GE and another to perform genomic prediction using a Bayesian linear mixed model. A specific treatment is given for sparse covariance matrices, in particular, to block diagonal matrices that are present in some GE models in order to decrease the computational demand. In empirical comparisons with Bayesian Genomic Linear Regression (BGLR), accuracies and the mean squared error were similar; however, the computational time was up to five times lower than when using the classic approach. Bayesian Genomic Genotype × Environment Interaction (BGGE) is a fast, efficient option for creating genomic GE kernels and making genomic predictions.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/09/01" />
<meta name="citation_mjid" content="ggg;8/9/3039" />
<meta name="citation_id" content="8/9/3039" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/9/3039" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/9/3039.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/9/3039.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/9/3039.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200435" />
<meta name="citation_pmid" content="30049744" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="3039" />
<meta name="citation_lastpage" content="3047" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Italo Granato" />
<meta name="citation_author_institution" content="Department of Genetics, “Luiz de Queiroz” College of Agriculture, University of São Paulo, Piracicaba, São Paulo, Brazil" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2093-6810" />
<meta name="citation_author" content="Jaime Cuevas" />
<meta name="citation_author_institution" content="Universidad de Quintana Roo, Chetumal, Quintana Roo, México" />
<meta name="citation_author_email" content="jaicueva@uqroo.edu.mx" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0685-2867" />
<meta name="citation_author" content="Francisco Luna-Vázquez" />
<meta name="citation_author_institution" content="Facultad de Telemática, Universidad de Colima, Colima, Colima, 28040, México," />
<meta name="citation_author" content="Jose Crossa" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, México DF, México" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9429-5855" />
<meta name="citation_author" content="Osval Montesinos-López" />
<meta name="citation_author_institution" content="Facultad de Telemática, Universidad de Colima, Colima, Colima, 28040, México," />
<meta name="citation_author" content="Juan Burgueño" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, México DF, México" />
<meta name="citation_author" content="Roberto Fritsche-Neto" />
<meta name="citation_author_institution" content="Department of Genetics, “Luiz de Queiroz” College of Agriculture, University of São Paulo, Piracicaba, São Paulo, Brazil" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4310-0047" />
<meta name="citation_reference" content="citation_journal_title=Stat. Sci.;citation_author=W. Astle;citation_author=D. Balding;citation_title=Population Structure and Cryptic Relatedness in Genetic Association Studies.;citation_pages=451-471;citation_volume=24;citation_year=2009;citation_doi=10.1214/09-STS307" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=C. F. Azevedo;citation_author=M. D. V. de Resende;citation_author=F. F. E. Silva;citation_author=J. M. S. Viana;citation_author=M. S. F. Valente;citation_title=Ridge, Lasso and Bayesian additive-dominance genomic models.;citation_pages=105;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12863-015-0264-2" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Burgueño;citation_author=G. de los Campos;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers.;citation_pages=707-719;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0299" />
<meta name="citation_reference" content="citation_journal_title=Inverse Probl.;citation_author=L. Cavalier;citation_title=Nonparametric statistical inverse problems.;citation_pages=034004;citation_volume=24;citation_year=2008;citation_doi=10.1088/0266-5611/24/3/034004" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=G. De Los Campos;citation_author=P. Pérez;citation_author=D. Gianola;citation_author=J. Burgueño;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.;citation_volume=186;citation_year=2010;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=J. Crossa;citation_author=P. Pérez-Rodríguez;citation_author=J. Cuevas;citation_author=O. Montesinos-López;citation_author=D. Jarquín;citation_title=Genomic Selection in Plant Breeding: Methods, Models, and Perspectives.;citation_pages=961-975;citation_volume=22;citation_year=2017;citation_doi=10.1016/j.tplants.2017.08.011" />
<meta name="citation_reference" content="citation_author=J. Cuevas;citation_author=J. Crossa;citation_author=O. Montesinos-Lopez;citation_author=J. Burgueno;citation_author=P. Pérez-Rodríguez;citation_volume=7;citation_year=2017;citation_pmid=27793970;citation_doi=10.1534/g3.116.035584" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Cuevas;citation_author=J. Crossa;citation_author=V. Soberanis;citation_author=S. Pérez-Elizalde;citation_author=P. Pérez-Rodríguez;citation_title=Genomic Prediction of Genotype × Environment Interaction Kernel Regression Models.;citation_pages=1-20;citation_volume=9;citation_year=2016;citation_doi=10.3835/plantgenome2016.03.0024" />
<meta name="citation_reference" content="citation_author=J. Cuevas;citation_author=I. Granato;citation_author=R. Fritsche-Neto;citation_author=O. A. Montesinos-Lopez;citation_author=J. Burgueño;citation_year=2018;citation_pmid=29476023;citation_doi=10.1534/g3.117.300454" />
<meta name="citation_reference" content="citation_author=J. Cuevas;citation_author=S. Pérez-Elizalde;citation_author=V. Soberanis;citation_author=P. Pérez-Rodríguez;citation_author=D. Gianola;citation_year=2014;citation_pmid=25155273;citation_doi=10.1534/g3.114.013094" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=G. de los Campos;citation_author=D. Gianola;citation_author=G. J. Rosa;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.;citation_pages=295-308;citation_volume=92;citation_year=2010;citation_issue=4;citation_pmid=20943010;citation_doi=10.1017/S0016672310000285" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=H. Naya;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=A. Legarra;citation_title=Predicting quantitative traits with regression models for dense molecular markers and pedigree.;citation_volume=182;citation_year=2009;citation_pmid=19293140;citation_doi=10.1534/genetics.109.101501" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. B. Endelman;citation_title=Ridge Regression and Other Kernels for Genomic Selection with R Package rrBLUP.;citation_pages=250-255;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_title=Priors in whole-genome regression: The Bayesian alphabet returns.;citation_volume=194;citation_year=2013;citation_pmid=23636739;citation_doi=10.1534/genetics.113.151753" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Gianola;citation_author=J. B. C. H. M. Van Kaam;citation_title=Reproducing Kernel Hilbert Spaces Regression Methods for Genomic Assisted Prediction of Quantitative Traits;citation_pages=2289-2303;citation_volume=178;citation_year=2008;citation_issue=4;citation_pmid=18430950;citation_doi=10.1534/genetics.107.084285" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. M. González-Camacho;citation_author=G. de los Campos;citation_author=P. Pérez;citation_author=D. Gianola;citation_author=J. E. Cairns;citation_title=Genome-enabled prediction of genetic values using radial basis function neural networks.;citation_pages=759-771;citation_volume=125;citation_year=2012;citation_issue=4;citation_pmid=22566067;citation_doi=10.1007/s00122-012-1868-9" />
<meta name="citation_reference" content="citation_author=A. M. Holand;citation_author=I. Steinsland;citation_author=S. Martino;citation_author=H. Jensen;citation_year=2013;citation_pmid=23708299;citation_doi=10.1534/g3.113.006700" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. Jarquín;citation_author=J. Crossa;citation_author=X. Lacaze;citation_author=P. Du Cheyron;citation_author=J. Daucourt;citation_title=A reaction norm model for genomic selection using high-dimensional genomic and environmental data.;citation_pages=595-607;citation_volume=127;citation_year=2014;citation_pmid=24337101;citation_doi=10.1007/s00122-013-2243-1" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=A. Legarra;citation_author=I. Aguilar;citation_author=I. Misztal;citation_title=A relationship matrix including full pedigree and genomic information;citation_pages=4656-4663;citation_volume=92;citation_year=2009;citation_issue=9;citation_pmid=19700729;citation_doi=10.3168/jds.2009-2061" />
<meta name="citation_reference" content="citation_author=M. Lopez-Cruz;citation_author=J. Crossa;citation_author=D. Bonnett;citation_author=S. Dreisigacker;citation_author=J. Poland;citation_year=2015;citation_pmid=25660166;citation_doi=10.1534/g3.114.016097" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=B. Mathew;citation_author=A. M. Holand;citation_author=P. Koistinen;citation_author=J. Léon;citation_author=M. J. Sillanpää;citation_title=Reparametrization-based estimation of genetic parameters in multi-trait animal model using Integrated Nested Laplace Approximation.;citation_pages=215-225;citation_volume=129;citation_year=2016;citation_doi=10.1007/s00122-015-2622-x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=T. H. E. Meuwissen;citation_author=U. G. Indahl;citation_author=J. Ødegård;citation_title=Variable selection models for genomic selection using whole-genome sequence data and singular value decomposition.;citation_pages=94;citation_volume=49;citation_year=2017;citation_doi=10.1186/s12711-017-0369-3" />
<meta name="citation_reference" content="citation_author=O. A. Montesinos-López;citation_author=A. Montesinos-López;citation_author=J. Crossa;citation_author=J. C. Montesinos-López;citation_author=F. J. Luna-Vázquez;citation_year=2017;citation_pmid=28391241;citation_doi=10.1534/g3.117.041202" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. Nishio;citation_author=M. Satoh;citation_title=Including dominance effects in the genomic BLUP method for genomic evaluation.;citation_pages=e85792;citation_volume=9;citation_year=2014;citation_pmid=24416447;citation_doi=10.1371/journal.pone.0085792" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Biol. Environ. Stat.;citation_author=S. Pérez-Elizalde;citation_author=J. Cuevas;citation_author=P. Pérez-Rodríguez;citation_author=J. Crossa;citation_title=Selection of the Bandwidth Parameter in a Bayesian Kernel Regression Model for Genomic-Enabled Prediction.;citation_pages=512-532;citation_volume=20;citation_year=2015;citation_doi=10.1007/s13253-015-0229-y" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Pérez;citation_author=G. de los Campos;citation_title=Genome-wide regression &amp; prediction with the BGLR statistical package.;citation_volume=198;citation_year=2014;citation_pmid=25009151;citation_doi=10.1534/genetics.114.164442" />
<meta name="citation_reference" content="citation_author=M. B. Sousa;citation_author=J. Cuevas;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=F. Technow;citation_author=C. Riedelsheimer;citation_author=T. A. Schrag;citation_author=A. E. Melchinger;citation_title=Genomic prediction of hybrid performance in maize with models incorporating dominance and population specific marker effects.;citation_pages=1181-1194;citation_volume=125;citation_year=2012;citation_issue=6;citation_pmid=22733443;citation_doi=10.1007/s00122-012-1905-8" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=L. Tusell;citation_author=P. Pérez-Rodríguez;citation_author=S. Forni;citation_author=D. Gianola;citation_title=Model averaging for genome-enabled prediction with reproducing kernel Hilbert spaces: A case study with pig litter size and wheat yield.;citation_pages=105-115;citation_volume=131;citation_year=2014;citation_pmid=24397267;citation_doi=10.1111/jbg.12070" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. Unterseer;citation_author=E. Bauer;citation_author=G. Haberer;citation_author=M. Seidel;citation_author=C. Knaak;citation_title=A powerful tool for genome analysis in maize: development and evaluation of the high density 600 k SNP genotyping array.;citation_pages=823;citation_volume=15;citation_year=2014;citation_pmid=25266061;citation_doi=10.1186/1471-2164-15-823" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. Wang;citation_author=Y. Da;citation_title=Quantitative genetics model as the unifying model for defining genomic relationship and inbreeding coefficient.;citation_pages=e114484;citation_volume=9;citation_year=2014;citation_doi=10.1371/journal.pone.0114484" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=V. Wimmer;citation_author=T. Albrecht;citation_author=H.-J. Auinger;citation_author=C.-C. Schön;citation_title=synbreed: a framework for the analysis of genomic prediction data using R.;citation_volume=28;citation_year=2012;citation_pmid=22689388;citation_doi=10.1093/bioinformatics/bts335" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yang;citation_author=B. Benyamin;citation_author=B. P. Mcevoy;citation_author=S. Gordon;citation_author=A. K. Henders;citation_title=Common SNPs explain a large proportion of the heritability for human height.;citation_pages=565-569;citation_volume=42;citation_year=2010;citation_issue=7;citation_pmid=20562875;citation_doi=10.1038/ng.608" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=X. Zheng;citation_author=D. Levine;citation_author=J. Shen;citation_author=S. M. Gogarten;citation_author=C. Laurie;citation_title=A high-performance computing toolset for relatedness and principal component analysis of SNP data.;citation_volume=28;citation_year=2012;citation_pmid=23060615;citation_doi=10.1093/bioinformatics/bts606" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9.cover-source.jpg" />
<meta name="twitter:description" content="One of the major issues in plant breeding is the occurrence of genotype × environment (GE) interaction. Several models have been created to understand this phenomenon and explore it. In the genomic era, several models were employed to improve selection by using markers and account for GE interaction simultaneously. Some of these models use special genetic covariance matrices. In addition, the scale of multi-environment trials is getting larger, and this increases the computational challenges. In this context, we propose an R package that, in general, allows building GE genomic covariance matrices and fitting linear mixed models, in particular, to a few genomic GE models. Here we propose two functions: one to prepare the genomic kernels accounting for the genomic GE and another to perform genomic prediction using a Bayesian linear mixed model. A specific treatment is given for sparse covariance matrices, in particular, to block diagonal matrices that are present in some GE models in order to decrease the computational demand. In empirical comparisons with Bayesian Genomic Linear Regression (BGLR), accuracies and the mean squared error were similar; however, the computational time was up to five times lower than when using the classic approach. Bayesian Genomic Genotype × Environment Interaction (BGGE) is a fast, efficient option for creating genomic GE kernels and making genomic predictions." />
<meta name="og-title" property="og:title" content="BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/9/3039" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="One of the major issues in plant breeding is the occurrence of genotype × environment (GE) interaction. Several models have been created to understand this phenomenon and explore it. In the genomic era, several models were employed to improve selection by using markers and account for GE interaction simultaneously. Some of these models use special genetic covariance matrices. In addition, the scale of multi-environment trials is getting larger, and this increases the computational challenges. In this context, we propose an R package that, in general, allows building GE genomic covariance matrices and fitting linear mixed models, in particular, to a few genomic GE models. Here we propose two functions: one to prepare the genomic kernels accounting for the genomic GE and another to perform genomic prediction using a Bayesian linear mixed model. A specific treatment is given for sparse covariance matrices, in particular, to block diagonal matrices that are present in some GE models in order to decrease the computational demand. In empirical comparisons with Bayesian Genomic Linear Regression (BGLR), accuracies and the mean squared error were similar; however, the computational time was up to five times lower than when using the classic approach. Bayesian Genomic Genotype × Environment Interaction (BGGE) is a fast, efficient option for creating genomic GE kernels and making genomic predictions." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9.cover-source.jpg" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.g3journal.org/content/8/9/3039" />
    <title>BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"6Ru80GkhKrh7c35r6jQvt2X8odIJs5GuRAIMpblqBzk","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490727","apath":"\/ggg\/8\/9\/3039.atom","pisa":"ggg;8\/9\/3039","ac":{"\/ggg\/8\/9\/3039.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/9\/3039.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/9\/3039"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/9\/3039"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/9\/3039":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022DWr3653KxmpmlQ9OKC1afnT7cvpgv4cHJnObE+FjKpQ5JUlKWowrPYlPtx8uPkNBJVddnGX4yjFcNLfZBLQxExKcCs+Dxd1LcJH89hp\\\\\\\/lYTZA4CgAifREnOcZHeX9FfrdoB7wI9VMzNthSfdXqDiAroUOTQ5ciBELB5WWPdg6l5tEziIFLZUpLP9u0\\\\\\\/Qtfl\\\\\\\/5aD527yBnZlrXA7MRzTjjaOSZxOr8y1\\\\\\\/BNKNvLYJD1hVy9\\\\\\\/9Jl4z76QMDOQvjQF9pC0Gmk+Rx5mLnWMeQxJs7dtIwgysOiB1vw\\\\\\\/xyFr1XCknqKpQAmnR7wCTO2P3RYi0u0Y7rRYGHRrqxWbTzEmmWEhB4XlqYz7M3FZoUCHWQQBaqnz2P+z4VccIxeerwGq2m6R+0otHDuAaRFGF5IksPGYeB9hNERRrBw64tNdH+NNF+4Ky44GUb1ThkabU2mXsSEDztzQOf0jSymimZ2OjMAcKOuhZ0jytS0thFRQS9NSTj1Glrn\\\\\\\/ijby6zcZAAqAO\\\\\\\/1Se4qvU0zHeKFtVHaYN1\\\\\\\/THt+t40AoCcq7JHl9md00UfSPeOwrcnhpvC8LTHUNX7TB1wIzA3\\\\\\\/sX8ejtGDaN7gZv9yV0DDcpnL3IqxajDRJlI34A6NSUJLRYTDbdgCY2veLq0HMxhO0PJ08zqe9L39pnECEozcHYKV2fwnTwaGWyS9FxMHJ583uO4UMeswMjWdq9ISWnBI3pC3px5j0nPbvyw8kiOELghTbXHr39DPAkjCU+8cIQbyGy8n64di5l5fPtkbOJO7Pps6AHsK2s2L1Aq4Dqh\\\\\\\/vw6KcQyYRSBXvzNZuLETClWOKLJKqjeLEUUc5v4jvWIs+9u7N+02NK+68lBTEAvdLZGyJQgVidWUksNlf647CmOi\\\\\\\/B8+Oylk9oHckZoERZcmRiXmCgWF5KUAFSePKR1xVmvmnQtr0IQAwnIXYkDSmpjYXv+gUxmx0P+YSYzbDpFnBi4hb5H1JuOjqemphDNXTKNrc8LEwE6DG83sfKFZ\\\\\\\/V3cuTm5DpFu4+vfdzNcCawvftuHgo9Ddf\\\\\\\/epyvOoo7zsEGKOQKkacH7eknhVMrFmu15QNBzn3TfdfIQ1xi8C78KiEmEN0Kpp2eC+HmJbaXNSK8v1vaUkYkYO+a84OhsHq4qOYbtZilUTgRGvuv1gRu4wGp1+SQwC8PzGUmaqhxShQNoMveObVQna8Fr7E8e6\\\\\\\/muH8qDOqEsmyDYoVM1IYziL8uVleNgEfNSCUzmlQ8lgeQUc2jBqbcu7d7801T61fN8UF7usoZ88U\\\u0022,\\\u0022iv\\\u0022:\\\u0022wTaU\\\\\\\/sfU5qmixOYShoeXkQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ef8a440fc063eaebd25847687ade0b442e2026e4f4a989d9d8618b0f2e5b1161\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022zCuGPUOQg6jma8ZHBPIoSKTKN8KtTxqXTnRxpQtA70MIJmQ6IdfHzjwGmvfuy5AZ9U7\\\\\\\/dkg3eAzYytc2zsdvmiyJ0rtkg2wprLWKse9jwLFNczmtybdm\\\\\\\/pLu49TXKr71DqlUkVU+6Jg9bSHvL6xSvqus6vsYikjA94z85Gy72AZCOuHhGkJ82s9d12\\\\\\\/8n1FR1\\\\\\\/+pAva4QkGj8i37ZW8Q59\\\\\\\/p2Df+yORTDTaEs7lUAHMKgPgaTRzN+0TOuBrmUCQufDcODNsybKVoWOwoAjq39Vz5pine6oYGk4kiSdNQBONU07XGH5YQtylDm4RWDyNjU8wopbjtdgpltv9leMsuLRgMR7OT\\\\\\\/xyKY358lKlgmMH3g0Si1QUxJNMePkd72rkpvTJClzBRj8IyVPUe9XC2nAec0mT7vz3MUGJpgp4aE9ZbiX0ix67oEqsuDk05B\\\\\\\/ZAADbx0VeutHRDA2rngu\\\\\\\/C0I\\\\\\\/kiqousdFy\\\\\\\/4cTxF7426TjkoxxdPHXoC+rrUwN6OcROHyiUd7MBNZzCNJTey7yJvfYtoNEWtIwrl0wJJKAd7zOdo8UvW\\\\\\\/jUCAeve78Q5mpzdK3pJdgLhykIlLYrrJnm9+cVxSsNuqyAPksFZKKnASuTziUjE6DWTqz2RbV0ufqj3hdUMElOZMLSgyYndHtW9DYsW2ZtgeXh7uq7FJH\\\\\\\/n+tx6V5uu\\\\\\\/JXPLGWZWleM\\\\\\\/3EgfeOc4u\\\\\\\/zK+s0vaGvyKLMEDp9O27UePA78nsc7g6nW9wO299mJxkFd8kLn8zIG8SGqTmeqtbL3R6jMcOScjMdgwJWRgMDK+4QWKteYFUgLPVRDxRYXqN6rqx8GBKm0PEKYJr0Py6NZprLm5VzQpD5BZU3sC5fI9jPnpwdh642ehHSiKiWI+o512PCJHdJ8Ex3UMtDLabw\\\\\\\/FFoiE3ne7pop2kLN5ArYsrRQKBo3pntnWGfY1a0Z1wu4\\\\\\\/LoCNbW2n4lMUa6teSSRNRP3TZDJVL9cvmPeEwoSGPeL462MT+r7nGWUR1BflLKfdZc3vJqu7OBUdy8bGWNkyE1\\\\\\\/qxnK+6uSHqzIwyl9eICMEqxYSKU9NTReQgfXkuXtoyAdFVWJzT\\\\\\\/OOOXNbHikzUFdQAkbFjPannweRvV6zMiiW2hU4ktFhIDHS0qMTbPxvHf9jmWtKRl0myQ1bhT1Ed\\\\\\\/eVmW4I4r8e1876dUBgKSJPUNk1cEKqWQotZvfOQ2gCMjMmoK4WlHB+MGHvosAKDwIN1ey\\\\\\\/bcXMh3yIE5DFoC1RwCen5iOf9LEr2kiMbLkSlcKTT+PTOJlkwmPbr6RGKXarYwXeVuO7BBgEzQx8\\\\\\\/9nICrXyv0Kp6S8Srue0HCAbZ7hk3b1gdRXWODD9TXOfKhW\\\\\\\/\\\\\\\/Oxo1L6AKaLu3rkzCNzkXVnC9E2zJ\\\\\\\/ByQgc9ck9PlYbDTf96wKj7Cq7wAOXHfI0BunRAJOZh8\\\\\\\/3yPxX+NidFgteH9HyWRJMfk2bn6KY0AVUqCB5j\\\\\\\/A5N1gn36S0NG1ejduY2GS\\\\\\\/pVWAIcF9nqVyYBP398pnKm0RsojZIl79OHt8SbQYnnGQjb0bfzr9fbwKm7IfdAqns\\\\\\\/1PMmrWjYm1XCGZlQpoOMgmkAiVNvsEaH9L4o9+YkXFFszFNWEd0RpPIXFnS0jUig\\\\\\\/eoUOOhsIzvoS\\\\\\\/2WqOyb3JY\\\\\\\/egCOGYRoe\\\\\\\/RIxGdzBIlmltxIbPAKZgl7eHoQPCluyniu5GqAdsl1kMFmwHq0Tplc6XvDeZkDOLzt0Cy18f\\\\\\\/1YVGpRQzPL7KtwRIcMcDtljEgdv3vltUy2YEJTnI5\\\\\\\/Y1RRP+\\\\\\\/M1j+2fGstCk\\\\\\\/1G1+SeRo8hwtwaI4ZBU6SxAtP4OhdgG38wtYXpHkJFz11IbP49lliqh6eQN+X8W2bKjxPui2Y1pAApF+FX1g0BBPfqKz\\\\\\\/pe+CszFVztydLg7iNqJLCqMbzKklXh+O2XHwAPGLX27lQNDoLA5hBm+ogOLgTiD9pUPQ0WW20E0lNYpvs+8cgQmndebljRmXlj6N9OqiJBaCxGKtXGQ6PmxLbApyxIB9ova7JDVrQtzZRUbq8w3uTFxjXdJvhC7SA\\\\\\\/tv+l+wcjZjhbim2yND5FzGRmR0GrL7S9HbbF+BmQLrIctNIfAi5L84exitRoWEm6bvnPGtKmDSmSAoK8ta\\\\\\\/gSFcuRdLuxEAOspifhXtZqLApdMqB6Ol5OowaV0SkapMC+QQ\\\\\\\/Xv4aTuFnAF0DGW483lfZO+KFQoVG5FuqeJO7LNWOLIn8UfrJMiqxRsbcYTdcaCLix4GmcuL\\\\\\\/Jpmh8uaejC54Xp\\\\\\\/C1z\\\\\\\/yLt12o\\\\\\\/aZuDMG300qqthbn1Lc1xzifsObzAGU68YAsVt36YSbtcc1AL42bRzB1cYo4xRoJC\\\\\\\/JoCXM83qM7frX\\\\\\\/IwyDZu2Orm8+SugLH6Y08p1GQRm0QgqT5ENaGb43fw70zgK4CziqcHQAhozWy07QT1RDqaWBvAT3eP\\\\\\\/olEpXnXUf\\\\\\\/WSrh1YeiI0puWKKDHLsL1RLaRW+DJ6gXPlZkP1H6pDrUJSW+0nitDilV7qnJIp0KFoTzh\\\\\\\/rls0foFEUSW2PYBZryZ4kGol2lg4mjMoCUnW+qJgudZz6yeqRGkaCjiwq4V7CljVR5cdD3g\\\\\\\/BqXvgXnBOWwl0nKOHW7HqJ69Nub0WurJcPqjNbbN3ECIeqCoYHOCZuNHl0SYwJE0PcMncexhXwOQIGpydimvEcTwCuBCtB6+43pRU542oHVw9UmaBoeVuykMi4KBzdONXJ4Ikr5+PRO+3NYfz6K\\\\\\\/eGVCOK66ra3bRZNg4laVzUq5PTMT4mmfJ2b2GudWkgXQypJweelSQAfr2l1yQyORSgtkx8tgOB5OBDv3T\\\\\\\/PCMSCHyzhMf9whifcQE8jxAI2HZLzHjkky5GchvndWfAnC089dW91bJVHxqYTe4yqvg5qBt7\\\\\\\/ScJTbwvtlIROhwuzcxvc7bnhlNYmZcbACyjQr4jpRsLvX6aBRkKe88sD+oryp\\\\\\\/3cb0WbFuZc5xYSOB8v4EOvQbjqZSeyN1Mb5Hh5YMODJkwcaWJQA1DG23lFgwiAlWGZmwUwRomBg5E5WBYfmseTdMIOqIwgLpVqxH9sOKI4Lv5h7YrLlL1uuF9v9hLWPhhPJzr+gVYkCtgUOK34nFtSnHpnv9AUcTm9JIUseFYWn9\\\\\\\/wgX3Q5Xvb0FGLqbG+8lJVyallEBxkSQk3osFKeCFgX4cVwIHX3rJmp13cdTgjbZy2NCinOiNo1j4XYcp4ozwS2hVynLq4YOj6IOmJGuuwC95fDItjZxtq8G7AzKUmOQgR3P6mK7K72ISiJ4TG028JTZCqo\\\\\\\/AgHY7wWgdjfJ\\\\\\\/7dpH\\\\\\\/TWQTKSK8XMLP2OnQ7QnSciLMflmE1A5Qer6fOWSP4odF3U3strEzxZ43HcF8s+Eq+3ofTPTJKeQIAur+VhvpN2Daw3B2RgH6Ksd\\\\\\\/4DabhTGVw5JYS2L5L7MWSgF3BNl\\\\\\\/A5h7irmF2KlsjWiYjuslz1vv9Ha08K2h1hmHG9aXvPm5ZykkOZBlVXiGec5Zi0w7YdA9KG1kF7fHkSbb4nvJPtl48nQKcE8JVwWW1JkRkyhV35vSTEHGkP2Y2kudp6l4XQsSic7TWUpJNHs8p+c2gSZLqLwPux9cznKwNxeuy44XQsVstTAz8bnaXI8EUyPkmUcxAS1ZdqcPEPwPiT2FfwU1UaGyckFM86zZGMQBxq73i5T0b3Ab\\\\\\\/rf2qbmsfKKXkEN\\\\\\\/dt2PHyymhNLvzlaSzlUPc8c08nwE27paGjIdaNDWb81gZlrhJIQFD3WftV+wHFkJ72FCR+qPlamHK+D0M6L5oUgemZt39ClEgOz8q8jGoR4w0Lgb7ocAuvqSKdCk\\\\\\\/H5TAGs4VWkknkg67YebdcTwqT4JFk72XLA6iNwOLGRsFuvkXVpsdUohKSeK89zO+JbzYrbL6PgD4PWE7D8Ezoqbju9QXk520bSLCokIM58mMQHI4eExtUSNhV1WxjtCJ\\\\\\\/9PM\\\\\\\/\\\\\\\/5d4C8LNBlWRKE3qX8nTVgYewVBtvePWi1wLHodBm7Er6P17qfYuvBTZF1fh2MQjPwro9rvsH7EWA3pQkv27Vv+lV0WnOYoV2KJfgocgxBZneWMc1yRKd5+Lg7JiLNDvqq9sE8xIt+lOSo1y46TsPSrAW\\\\\\\/lyR2jiANWK8TegVBB\\\\\\\/6urKFxfAnIJLqiZp+\\\\\\\/oC3sjDUx1Xq67OIBD2CBBAos25jgHTjjOxUzefIP6x1d1o00ABKcrzRnSDp31cXO5EYOMocSD61Hn5Pt0HA1HUCCCmr5\\\\\\\/VVkEcveu6YsLwBqeWxS2i+s9aUkXPcjOCTxHscUHUgxqip+uyHMtUz8hsJaZwBJE\\\\\\\/rgfh4usIdyOxIgebDhNHByE3dDhqMy381Uc1SN1uYatPaZsNyXAmFSgkuU69KWN1bsZx6vF0z+Zye5G1TN\\\\\\\/vR9czcTMetvFtr8RZk14SMV8qxILc3XoTlA\\\\\\\/gEX\\\\\\\/Yk6BxFISHdRBHwSVXFbYBlfHxICDT0BEMixDBIGiDrWOwydMfkwkdBKWz18X8qikcGeEnNohE80iVoQ6ld8vUdTvGAuYtusK5PXKiJi0gdTojbWdZl2gB+NnzPMhT5F8XGtJkBMt6u\\\\\\\/2ive22CZi3LJDI7W\\\\\\\/OlGZpbcI9w0IgO2TMzIiZrGJ7PhbVLUIwTvXoxI8rizO7Fur3x2X6iUiuZFUvDB2S7E6MholJ1NnV2xb5\\\\\\\/8WwvnQL02EIV1cnCl0ipxmXNeDmhO\\\\\\\/QygsBOuFMKjSOJUFQHTUaitUdSNd7dGJLma\\\\\\\/x6Z1W6B3n\\\\\\\/velVpm0bz2BKtwDaQqijbySxd3CGAjwaTQFPyjvhdegU22MlhDYSUcmYdRUDdRavEe03uxInNOvBEp4unWAovnXTxG4Cxk5sChHnWWZue2XdzzBE4CNvagORFoJrpZQuGUt0t7fUJNz0zg66JAUb\\\\\\\/VoFDU5L1eH7CJT\\\\\\\/iuilBCYVCw3coLIFAjijekrEPo0umg3u1ThKQD7zj1Ivl6Kon16\\\\\\\/1Sns9yydnx+1Vbj7r030IfJ2dsgRjMiXx9oJUgN0Co3iOQrCB4cpHrIr5qAovvRhasJ4\\\\\\\/aOc58ZwJ7aSZ1pL2E1yk+qDhHmcEyhsduySUR613LJAIL2qz9uZ7v8+BgGWrPs0qcArbg6mCywLRtaMATCwunL+tUfJ\\\\\\\/ZovrBGpFpuQ9pjKBkpxLKHShJf2TRauLNPMLWxTyPJQBZeApt7Iw+VJwgmuc+Xg\\\\\\\/eiP7o2s\\\\\\\/EK\\\\\\\/pZuAzRXfAQ7zqtSpwqwM4VQik1jsiWSoEv79AflE8Bv+knnVT9sbvFm19TZbRP62P\\\\\\\/3bpEEyr4UWwMgyFCLKGc\\\\\\\/MxfZT1vpyIlIQZKPSYpbekRFXFBKvyc1cs8v2BWqtzU04LmMMUWhTgJ6fVIb8ZBiGGdM+2IChRWwbjEcuZQ608e3lWg0MOaKpy+285Vh+MVqtqVR6zjpn53QD6jQxO5ABcrcs1ctPY0VXpbj+J1tJHoX1X2QaExlffPi5VFc+ZVWi4GlKZS68qMdpJ14at8wMd8foXhKiDIWwj9gmoJSphmb85QfllreTRrVCkloiSJcjV7QGpoAW6eE+UZLbfw94opnlPC7tus35g+javzPaPwebG8TYSPhfcAi4aSzPkwGqrAYln6pkgr8GPpz+iqUHlqX0LxwOa\\\\\\\/uRzNWIl\\\\\\\/P\\\\\\\/UYCrW6QUpV9i1PSMz1E31xnWaa1RPtUc1XZptVkbin\\\\\\\/NUvY4eZ1iE4yT6yAuL+lzzt\\\\\\\/wl1IrTT1A181V0UWsCBQNDuQsfHSPUWCjNhrjGyX95eNQ1dh5M20xnWe77hFvHWfOo1qFUr\\\\\\\/nV0TDV6k3586zrvlgS2ZDY\\\\\\\/nTa8RaHYj1ejDg5hlecHfGtD9l35TLIkWOWos0s\\\\\\\/rOriOP29iXNkCYfdAZowwPFfBf9Vh1m0gf49+oV+43hRpVPMlK2V69QSOdzkcFWKJoeTmj7tp02Lc4CAw\\\\\\\/B28lz3Lst2OBW1oDFiaPDhlOtEAf3FDuYAzVFL7uGMTzNc4TXZXLdOtLE+6wK2FLm9ZOcBcJYZXexG+99b04f68+32RGweky4428ukuMXhRNlB2m0aMwilMsMYYr5TMorQs\\\\\\\/6YCyuJzqkMTi0FDKzTNjUM7Fu00JOPsALdTdZLFGcaALlz2DHcPxvQkqjY2v75fEtfOYvtCYt0FUjjQnpLtnYMeHezhYidf9knSRHNZZ8sUImQcoksKsf3htGsNLox00\\\\\\\/A280oxD31JH7eOrGDGn4osykcl1MRvOxZsPfJ31oMnS6Ht6d\\\\\\\/mF54ErTCkHsEnW\\\\\\\/kT2EPYX58ltkbk0DHPYycIft6lSWTC43ueE9xVPeXq2+0H8l\\\\\\\/\\\\\\\/Q26ti0LzUY6QnvxIVjIvN9wTUJPuO5hKVyPZd0C++4rQHiYYmrGlX4H3vEk+RZzJLf3fbnJ1y7QbqpRoHPTonyMnnTh52dIh0\\\\\\\/pXjC3QguUKrLQ3J1D0C695ecuqxrgz5gpmN3WNBhn7kMIyC\\\\\\\/bZU6VW2hoD0UQ0F4tsigH7z4L0pNAJigj6Z8DAgCPcAFAOySmXxwUpzxmPPJh\\\\\\\/hKpzI+MQ06A26kc+SfRbfDL3weAo\\\\\\\/mO68BEB5UrBKTRBIhTOVZ5N4RWGjvN0Qywh5oO1TVE45gfVJiGONEZZP6nK1MYT7F3KoHasjd90dLzGXqEiSfB7BN8k9jpr5Co2Yf3\\\\\\\/o+6iy884o67IRA43PMUDt0PTej+bE\\\\\\\/3RiW2fEVUgRDPbQ4h\\\\\\\/HDcjJ8M4iEKBWWoAED\\\\\\\/\\\\\\\/9Jhekyfq998Py1v3BC072D5IzBHPaUdX5PVlFQ\\\\\\\/n5t0fVhobKg6kqBjYjOtO+sJ\\\\\\\/487wqnst2aopu7d7qy99D2+3yADuzkds60HBVjnNg99AoBQomSfP6KXro3qfWRGrIvHc7ydKB75KDzuq2U2ykSRAud3X08ntJTnPSuofqX\\\\\\\/fqQsRTjCmnoCsHjpyuBIj45reX005KdxpRLSQXVfVegAmtD\\\\\\\/DP+Dm9or2p86NfJAXtYBWjxXckSev\\\\\\\/G2UZQrP6Uy+aurWKOAPfhcsTzQxku\\\\\\\/L3iOdHZpxjHYH4kDNW2ym5VtxI4w26eNqcTsz5vH1T\\\\\\\/y3nyL7FzAkEztndvBfNj77XuXdaGYdLPnfahXNAIZkDzTUn5Q6+GiDAyb9cxnGvHHI+wI8ZPVKpJ8qbBg0+klcFr1UVc\\\\\\\/B+kMx7GU5Yg+EP2jZuMf1JchuR4o15p1s2GThs0ss7G0LBt7n6jowSB6IJ1BDlfX+OtfWL771MMp98kgwzRl4t+BA+A+8UYHpYmOuv4\\\\\\\/RhlCyv0Ygg7g\\\\\\\/1ggK8gcTk+frAbxOJOtMM9Hi1oPE+\\\\\\\/MG2DJq7wKBG5J2+i4JTIxXdkEb4\\\\\\\/fLFCkBBZDGlWUEA\\\\\\\/I2aqRObtttIVyoZ9m2MSXHeR+3YlAf5ltKtnOYCDtffdH0VQHzxCAZMXb9U6pCrg8KWLPckuOqSSGpYFyBXQOMVNFsYX5p3PBpg4nmC1beCnEQlUly2qUNAwEJupOF4OkXHHXVmptFVgAyXtKEQOhjvd\\\\\\\/kZLnj8tcsatdG1vAHK6SL98MyBBTCgByFr+iwb+Z+0WnU0dZVmQmoOdmRy7\\\\\\\/f8pgVw0XGpgHaDC+oJ6LfALDHuiPcFEpLa1Mv+XUW3uP9vqxwPVAL4RF23\\\\\\\/qkGVIH\\\\\\\/GQ7aPglFDsJgYpWH67ATPOLdk\\\\\\\/3v\\\\\\\/n6dcYlysuPH8gEmCmYYAts67CB7MTbJAo9RRMaS5CENyX6LSky2hxDIy\\\\\\\/r+AizgvI6rj64XJtG6rmgfBmyxIqzen+DDnYh6iRArNmn3d3pwL0E1EFWMy66R1zzneG4gTN74TzSzKnAIS1AZ\\\\\\\/pSomV8EINqmuKLS3uKFXyYinuJUls\\\\\\\/P0ThJEmQsEOGou1Lm8l7eUZbRo9Y8iPV2tn9iOfpJ421v30S\\\\\\\/qI92ZxwvoJtjNpbMig96lvqz3+yHeep0m3QxZRLTEQIPqTjB7d73+M7nKY1UX+JHpwaH6fHdl+oezzSxLhMTT+kzNg+kNhy6RBU3KHNH+oAwTftGgjjLXJnxFgy1fP8+AMZzqe55+6BQNQWtM+dUoHI1\\\\\\\/\\\\\\\/M04kTaSLT697aO+zIAODhlpRzT8zskWT3Am9BYqPWENuw1E0FcHylyFyUhTO4SPa4jWI1Wm2KHMcTjq06sdHjRBqFzoTHuQwrRjtCwKeYUmbWZtlE+Pi3iopfPjcCKOCaDXi1RyIgLOBskWy5+IUqL1GdQZNe8eTAIMuJ\\\\\\\/13W+8\\\\\\\/Ed4olsxcE0x0PGGzkpljDUwvW1uQZ60ebZpfkuwf9ZyN6D0MryWDUswZ8fiJSPwr0saFw5afwpQpA0O3m8ASTNC1q6c\\\\\\\/anh3d44p8UBq5Tv\\\\\\\/2Lv5cqUfmL7\\\\\\\/ZSKnBYjlS+OwjLkEAlHNIhy9LeqguWb1kuVz1ipuxNiNCVn+vKTGkIuDCvWUkGLAjBmtlKdLjr+Yw2+FUnDjTxWah1EfMv7r1IqyAUQKu5cwNQ6iH33qZwgIW7w6ljyFa473mkevuUMscwgYOlceyI1PPPOMa7e7fib+ySk\\\\\\\/z+6RF\\\\\\\/EpeEZcr0spujmvLpr3zHBLuppW25ueqmw6KQxd1MWMaKmxMPbzapOXJDzRNRuitdP4cB7uDSQnPUFzbkSX3tGzPeVxaXD2i3IxtKsAhQ53J7IsWcfjwhnbmlRgqx\\\\\\\/tLj\\\\\\\/E+5R\\\\\\\/wzquytoD\\\\\\\/E0ybBRXbXBpNKpbPaWQkk7lGi4B4czo9bi4ypYClaHcgBvY4luBjUn3Hyo6hkRlRi6EieUB1NqT0PgSttRSpM2YzWMEXuj\\\\\\\/akEliPaz+4xDEyH1GN9kuUPIU3p7I+TASqC\\\\\\\/A4ccjVHrSbIIBQjmp3BkJydwc6VOXkOoVQCqX\\\\\\\/UGTohlh7yB5ZjLgLDrVk8GeIznNbuhsxCf6Nb7yNRoyfJ3zuIP91kl+RZz1UF1Bg4D8zJydiXUmLj4fkDsfDPrhPoikIbbdyJS67oxWMd9Da+w0x\\\\\\\/H0JaNZMzG53zQo+OCSAhysaWjO2twlzCWDz7VOt2rEmOa0q44t7lLHJsIgj2fQUEscsLjWrEvNy22tRscFOORd9TMaDTJDeU04oT69RrvGtVaL9kt06K3ZjV527i6Fzcu110G7WzV82N+yZRJQI+0gbTIC36Y3TKunwlOqJ60qrZ8a6QujkqkhGP+9fAiaNx+cnL8i8xTAzbd6mKBSPSNkHx+V0b1E3VSlkmH5IvXZ2176z8v\\\\\\\/MNBDnfTnSnhfn5GfsNaMqOv+291eTSyLO0A4WGro3yCWKEaJ6O5G8u+bZ6kkzFxH8BdJKZFzuH7o2EjaAfmzR067PRd4WU\\\\\\\/fKLJbdltLExorx2MJrK4Kw0VJB4wv4x\\\\\\\/E8UN6+eN7eYeJuAO\\\\\\\/AIUtfgo0D40JkMIN4OiB2hubEcYLTgCkMOvIwa39u14dfjV6z2ZLy\\\\\\\/ktsbqVoiLQHhbw6epZ6T\\\\\\\/rOU4IbMSOxjnU\\\\\\\/zTpaYXhfl8It65CXsjY46wSYhZQlJRQ7PN+wZwtLimUl6ph5f5AG7ok0xNh7netvU9FpHXTVq0P84fbC\\\\\\\/TLCC5ENkke03UmWwB7J1QfV\\\\\\\/NyRhm2E\\\\\\\/xLl2LYCg2MEL2bxgmgETz9kQrdKEj8Jj7XV7Zplx1yhsDJcCwwAfna2shiQ4RjVGPMLmNzNMnS8YT4H9PAkMHiqeIwNzZ+SydZd\\\\\\\/o8sijwVf4IFPmJyQ1Y8InDuyUhTnb292DwYDO7llPbJxPuSDe4iCR0XB\\\\\\\/29I2InG\\\\\\\/oxLWG2bXQjJ4G4xAuttUJgm5Kx23YI5maC2TnSrHPNkioapJidn4KhKrv3+0TciGNgflGcDGdnUWWvefgJOyUKe8jMOxbjkmc8VHFTu4EcFiOEMm9cjdlynK\\\\\\\/ItrxHWkq7etb\\\\\\\/z9f7QkuESqFBd+KIuTdhCPIo\\\\\\\/UMZ2vllliiB1yG9cBs4Gqe2UtmFVOGenkkQZatHAsUK0xW\\\\\\\/g3IKDGM4VPe3ErR5dOd+B51HFXCUcfh8dgCnIZfQWj4OmCCXg5wmAGvlRzlkHqN1Cc+bUAIxJB6RgMTlxxVzj5IOBUeStg3Cn0bo5N7SBSAyD3dw\\\\\\\/\\\\\\\/edSRhkQZ5Bqslu1cvxoVxjbdm71\\\\\\\/1KeqcWOGMnw07mDL6e4FHI1+c1r2y++NFX061KMo4LmOMx+bYHlvz0jTSyyym+p3dQXYKUh0J8RNt+30yploBBOrpTCeX56ELBU\\\\\\\/RLJdEEv3C5CiKRXqqXeGm5Nf38TBS3X6MMSzwTlvpx2Gb4I6fO9WsjAiZ+43CemH0hN8UyLW5oWQi6wjwfz+qovudyHZsdDXMjyPAxcSpOwEiaaEWYm2YJ7J\\\\\\\/TwruvXXgOiknjCQlE\\\\\\\/wbr9\\\\\\\/\\\\\\\/YfMjjzO1u75rGZyk2p7E0dktfX1KEGdAhOIniBVzxZ7VwlTgN4XXF0LerOFzGNnDmbY1QubDwKasO8Zftl8wBwLlA7a+AS9wtlboSd86cToZgUVOCuHsfs0bz9VHFiEiILHL2GCA9t27QAwHdF2CxLm0SOzr0Q4R4hoVwSQq7lDZk+8KTsRRlhJHom82+N\\\\\\\/cQRqtyIL1jUAWB8Pckkayle8FK5TNUC0ms\\\\\\\/J9G8Jcsgrw6BRZIsObXBsG7rv8QhEh1aLLC5G+C1FMFuexyDe1FJ6exC2JNW2FBnQxNaTCBVjy6XFIS19CDJXze+EQwiN7ZdWWrDMjHz9IvLKyF7GNM3MSm6TBRZ9Barll2ViUdqYLwZsbnHIcyNxXkDlb8wypObCqNbBdK3JWyfEJlMWewp+0MEYYdAsdmwyd\\\\\\\/gpUajRcSP3+zXDtInYOvhJmezDIFlgGZHtUBfzYPKFIpmY+Q86omvDeqSqPJPV\\\\\\\/uVK8h0izwG291zx0MLptmzP5TrHxdGiyrF5SLMY0mpXdQ95iFQiLwqaKd3UDVdhoMrbaOAsxo8rtmxMiHU1uYIHyixUXE49hpbdZW34q2jcx2aZDyPbzg8lTs2cW\\\\\\\/niIPj1JugYkGL3s2HlP1sfEo+cI8zhB7FgCDF4Dtj6AtuDdxpLYeXPz81RSURbNDDwKnMLB4Lua9Bxvdswdfjr7\\\\\\\/W3U\\\\\\\/y2cAfLwvPN8A4o7cFVNHCpRwOQpCFMM34kCTj\\\\\\\/1jadovIstVSpRYYYVIrKvNgjbBTW+DptWq5GLDoPsFMW2+Sx\\\\\\\/J5JBEYlXttW+QI6hWCEupKETcfI042et2MwFNR6lQ7jMC20MzFSVZ10RMtdHRzUEGmsnt\\\\\\\/uHgQjULpOKio55A9d8YC\\\\\\\/chU4eQJ\\\\\\\/WSe7Ib1XnDdkXYadsxRJyNDBegsGXk7b2mfQcetDufUs0spz2ld1GGV9cpt+ia2q\\\\\\\/qb3ZEVJ0fgYKfVluWxnGP+bywx4nCZ6Xud97Dff\\\\\\\/YE5GlRLn5IkSDVE5gsHPTo06so4J1nhL8ViZLV03UU9dI1dpDurFeHoacOsbcp91wMRaxrZcl6KtE7yxDHQry4496bJ2Lv6wPLB0VaENF6b43PuNeKMZHdlKu2TCwScNYnmtS0K+v6rod4aHlGYLwpPu1mqs6b5RYIxESplOrNCSRyUbuC+sbkEN1Etrx+2xjn9ozgHCC4K6WRvz2Icx4coCDI3wkCJJ6FYbJpNO66Dqp15YAneOXHTiWAtgK8yK3fGPLWebQDrHAnAdunwWCO5zr7JUDlejVu8AB87omUjDCZDojp5gwpouiHYP5sh9AXDCP2hbWBAVVPgriGBteN0dkTCsWHb9byHOWaI32rS2Z0FLrhJbh\\\\\\\/H6ubHd786JvZbryHm1hq3h3Jvj+JBQT\\\\\\\/l0fgpx4a9NkPrIW3L0K+29nLHH00Mw2FluG3A6COlNHgwXd4lsuNYq2CDKOlOXlDHjQhQ2mIbkPVR9fTcBvZUuqNfReR5EyRYIKyW+gEWUQh3PfT+yU0YCXzcAJR7jOiA5SNsbh1eK\\\\\\\/zAuxwl6XGiFJpPK8DjhpYDENrQzCiLm7iGRgyS1WVZm4l2QZ+CLLMK4q9+v6JLxzwBpFqQwDjJxQklv9a0Y15mVq1nYDf8zp4tg0mpMwerklk1RFOyg0GeGrNwucGZXvCSrJWTegdv9a84LVNGidmgFz5UXVyLsLM32avbhslRrVRRL8TqeBl2WFLuKHiCXJUJEjMIa6474mqp+MchEt2f0rws3IPWKOQfD+J556OVxGejRHeFFbw\\\\\\\/bZawG+Db43nb\\\\\\\/hncCzcC8Y2J9qw0J\\\\\\\/oNjtkqA9oAcrOZmJY3V5DTpS6+ekhm6xDRzvj0FPXLP2cCNvTNIvdGSXK07+azk3MAZTZCVt\\\\\\\/670C1lBl4M48OP9BtsSNtED0Bzk8XFVplpQWIk\\\\\\\/EmRpe4xzIMbgb8cn4JTNcmZSKgyA5\\\\\\\/auAU1TVXOq4y2IRZ5Lj5Qtp9BWcGC3ocVT1lWs9CAKnBcUJMzmCRkA1bXemxPQKUKiBTSUHNGJkHHXijLHCZyvDH5qxZp01Y\\\\\\\/OyhrmDVeVZRWlH+SK+Ldj8O6r1+A2HAqGSUaaVcRWlxNiYT5MKpVQgftxIlZ+UsIOwJI82KAEiFcYfwoE41tGZqcnPgKFW72a0VhrAlLdjzLw9nR269KGtSmaq3MWnJfKeyyffmoKIOM8LU\\\\\\\/fUTboKsQCdMqsTGGOKjtIjrYTfTDcrnhAW5s9iCdzn6tvmVYPvncw7rBQ6SSP2OCVmEmWtj7G7P8XKjuFIXbPJOFIBP8R4Jexj+comwIigB6RSzPqwTdX0LiniI58pwePXF35TLytST6PJXmguno7IkVxR6GDFWAyaKmpBGhWFI5oRmR55BxeKQMMipgCT5Zm19807x3HlyamsABhfdpgiVZJIiIB\\\\\\\/GRIIVeCSpwRrFzXoia+U42wZT5zIXGh13VIpjZz4H7NbsEN5AIl4vS47Ya5dWmP7PGMs9i6CzWsQ9LD\\\\\\\/lU9qt2qRHXDEPYJvdn+dN8MFTug12YOwbHKZPeoKAGcGp5LBQLo8hjK9uJeYjRrjias816lfGX2iro4x9umL+htmmPf\\\\\\\/YQeVQFXmzHbnT3tqzeYZ+hPj2TenB8qHYRlOYbkcLP3\\\\\\\/xcg0aUbBI6yjfDJTPOM7Qc9qL93Te2HGfAPI2LH9TPL69zMhl8S25t+9Bcwz8K7nn+qf4Z7EtFE\\\\\\\/YDKp6T5VFO6U04fZGPcjJn4faWUl846qHA1v7Pw0UwcMzJv+ah5TSNn7xejmvmUjsLdHUxG+AICSW7NeBKjEr6\\\\\\\/tPF2ffg0XPCugi\\\\\\\/DwGMZYdm2FntMMWMflQ4z6OBKWkkaFO\\\\\\\/gQNQ4ISviMBM9OkAcynxPpcjudcpMoc9XV6SNpqXYfS90RAh3hRsLwCpZIkGYaGg9FCGlGHGkSbHPzZ9w1rHoDv8VOpjbfUrEt1GqKB7MVyN1nRcxSsdANTICFTCb\\\\\\\/RPhHXVVaE9v2LRE56TPXOZpjt+szLyiY3tkEGtFzrT8WyX7R6183x\\\\\\\/E68Lyo0qxbu01CkamllzMVD4yrE9U2rSEYPiFpaEW2\\\\\\\/5+OulJbsYlZHdtulfJRTdeMOB2EnXXSQjUFGFg\\\\\\\/DWdsXij6gwC4pX16MOhZ4kIWymf84vc2+\\\\\\\/Ax4k8z6etKlr6U34mkGttOPT7z\\\\\\\/mC9HawjFSb4k8hZNHTMTvUub1Hr6ioU\\\\\\\/\\\\\\\/EdpFd1XiEt7gk+1lwexMzpnDrFd\\\\\\\/SqMdDfUNHCYaDd4JZk+BOfj0HLpBvZZUfyByqm93A980m4CRJsArqYECrF0TpcAB6ZqLtnHZVlDit+A\\\\\\\/oVZ9nOMaZyzMs0FEC4kBPSznbWB\\\\\\\/mHFNT+vEn6tG4oFi94Gd8eEGF4Oi0qCVNJi5Ksu+7ZoKActkCkFoRMGdKBiW1Nus4PV8SyOlwHeKOqF+ySHFvJKKJ6jOQsklA0zn+86Eyswt4A\\\\\\\/a08culxuYxNw8WfnnZas\\\\\\\/xZ6vGqdcLw4HQZJwXzz9ay2NSoyW0cS7lAwqfGiHI6zJU+b9gGWzirqvp5v+nKzWsusO1QOOuL5jQGRue3Sque2r\\\\\\\/FRSxd32H0\\\\\\\/ianjjtQrWUff6kbYqDhEhM0v0tWlUhO2rcmQh0+CYhuq\\\\\\\/YXeffRdrwyQNPZOYSuRhah4jbJBnAoWrqRT80Qj\\\\\\\/HNTAp0bDsHyxzi4okoLxvYuzSUL4FhnM5o3jbsrwT6aqmHwGlkWQut8FGftj9LzXxszMuMV59L5ZXqEhACxN2MTZpC7L1+8DnBZR7MYxWRLD8xTqvd7Sp9CUioBC+9QAMQmePcGoddRgzpoJG3xcYX7A2kZaNllWJVTeXuzi33+FOxJkM1wqQ+QF8K7LpIGRi0HNUr6yPa0MRbjWI5IGFPN2EdvZfw8m6CXcUuLqD2j0pywPZEMEK3nVzxremSq54JGt9UNUvLucnQ3oDJ9\\\\\\\/epdmqBYHUwqH9fWr2AZu\\\\\\\/pd9t12sOhCx7VGlU0Wk2mfbABzv918mkAWb9AMJpC8vQYddd0PKE3sgtxmd3kB89KH4nFqypMUsIyx5pEh5qjRH3o2FJZrJ8Zvkhr69uOgZkChrX1zhOnVdYuTPymbjdTT+2ASnbiVsP9TJabbJ5XkjIQtU2UeAoKUjbvvuEfZqvoCkiQzB2Bvw2VWfBYgYjYRuNBxQU7GNFmjFbUW4GTooh+Vr9iQ\\\\\\\/D2D7ZhNE7\\\\\\\/x5TeOgc15BBxWoORnHdRbcrodMHJrDNZkCOurCFeq1Y0pN1jqwCsjY3JxlZANSA2A6feYQAPdtB6ZT8K1\\\\\\\/0sGGQu5WPE8xI5rxRWPPUaUFFXKmXrEFXyIazFhWHfmlqcjsH+ilNE91i\\\\\\\/mUjCUaN+K0TCL2zqvJ4KwDUFh5kzTbfaLF4LwTXppVHMoBRXhfT6lOWiHVjyI0d5MkULiCzIK3oMiTaX664URoktHwje88tluhL+09mE\\\\\\\/fBrCuUzdOWfOlLA42ZYuzx8QxtcM7MK171\\\\\\\/HSPm1VnChACm7shCDif06IF1yWYZyp9u0gRSxwwHDajUKMjLtmrUgcsVUWsUGPwPQWIo2Og1\\\\\\\/1Mih9wfJFQJ5IroohiEblY2aW9bTAv0KFlyUS3zK8WXdtELAeHIev7Irpm9odcSiyhJjsta9gDM72qvED6wwmQkV2Yks67eB9CDF23aAdfPDgKVTnsazv5q\\\\\\\/rK3oIiXn6huBkdvkGvyZqNTB1TqS2KpcYmwS+q2TNurTKK6WYshggraOd\\\\\\\/AJ424jJtdRcp75uU22CtpaDQxsatVfnZbVNSu0zKKgiAKirJm6p82Mxo37zPS9+Ef\\\\\\\/O9g7OlRkSpsuunDF31WtvQ2qtgen4vE4QzS1HHLtgYkc0LjjCUr\\\\\\\/Ep8b3hdiPNgpl1C1ZjGAwJi8lUgfBHxNrcrJFG3KdtQ2DxDwtyVv+BnBogh78iJ4l4veeU+dYlykmcaTrmcX7Iawxbxh99D82fk+9uwNVr8B3xQGmTf193rStTRMgtGdlCFumg5\\\\\\\/vjZluyt8pHWCGviqAhesV3h7+f2c72U2RaHT7O1iXX\\\\\\\/40wo1EKEqa7251B0A6OlOF7nA0PrJaqkPUnB0z5RX9BqZ3ob15VI0CcnhXG12XXAhzdJulIoeVBgPOLCqNmSr78GZlVDfdtVU1MfgyPlorXr6SJBaz+\\\\\\\/TMQGw1YbRAPzySwPlSmLJIt7SmLCcWJfxw0JTTzpopfpnAQ0QfAAeFJt1I\\\\\\\/pfMfqq9E\\\\\\\/nkBfHWoDddODVvPFpnyQAdRIG9qec7yJiQRVP8zSjbD+TC\\\\\\\/nfMOXHWo63ywSQpvqkG0\\\\\\\/mDbFTz7\\\\\\\/TPWGuRLJmf0l\\\\\\\/Bn45fEDf\\\\\\\/uAaKX\\\\\\\/z9yPE+WmHXzn9XX3e7xNpBG5MK985gMQj2Lia1UN0e0a0Ps2maA9H\\\\\\\/r284+3UV1Ss5P2Qo8pX5kv+Tohyg5FZCuDUdEPsH3VwQibp6oTChVy7r0qq8W8g\\\\\\\/R81zHCeJ4v+LkQVA3wEB7tT70uaLobVSK7Qa4durbXzp33ei416xFfhJxlrKFpfdGQtbIKSNTGVez3a1i0ClNYhfLDu6WZm+wGNcC1t2d5kkLw0XZGYa2WkxVk63ZrVDLUAJtkn4iFh8oGy2+OZEMq3Ccgg4tGfTVXbN\\\\\\\/ras1O78E+dO49za\\\\\\\/oEGt47YwwJUzbP4zTGx7usre2myEN5\\\\\\\/91f79ODHg+Om2w7lWVPQvQ2rxdgo+moFaW4j\\\\\\\/O6ndQrY+q+VIybtHMGzjPY27QbXGQqfxtVahGdDyZuyYI1r491OwksqFhBpBn+l\\\\\\\/KFYCY\\\\\\\/d7KTvKlMQWGTaXeKu0eDIs8uYsiNaE097JyIsnSoxaQVKz46MDuUw3qIaonrB6Ro\\\\\\\/u1gv7xnPGyRCrjtb7WluRCUt+XO0BL2NAXDc894PMC+GNC+hBOXy6V2e896ir8olfX3DXjNnnK1oibhDEQFZAsHEME0nZB00skwrYqeoRU4mNJsqVqvd9DyklIAIjM4wBiU0prX+WXF36vLbEbAdz5YmtqZe7bjNsXeS+\\\\\\\/GJ2LXZqEpxn+w\\\\\\\/mO2XRjO90IbkoC8jlzPDTlDTI9pQsLDX4jwUM1MCjxRWrnhBSnM6UQgKeRPQXxjCgSeZ7Nnx8M4jtxosvHZAC7eVRzxi7PlnOcrYkHTsqSm6DSQAA\\\\\\\/3j4t6upjiVAKfga8nmWCHtanNcauAszO0JVHR6Cdw2eEVZ7dHPt9m8TQObaEmwlqalezQ\\\\\\\/AOAchHYKqgPXO+f3slqskEOQcj7HTp2s+RRPkmeyeUq63fd3UBHBjQhZ2rzHI3PHaEIPqHTSDqRyohB\\\\\\\/5n3gsGqgG+HavO\\\\\\\/bkPeWTf\\\\\\\/oZ6zk2Tky6tqGSACC1+Tmjo7tJjYF39AXJ6vb9j4kg6L6KWn9\\\\\\\/iiLYqp+Llt+hggt7eGbICLgSxUwMWnqUOYNEttvnUSu3lGDwWfAplnHla6vnruV6oWsIYqaFYdxCcTADtQsO+yLTlOAMZT2iINKluXYJJkQW79ujNhqj1aej8EU8uE135IWOwWanemKRIuoUTRgUT5JvZd3S8plbM79Qknk50nU1ZnAx7VYW+c6TuALb9tZLe5jc1aZtnVCA9qEmkbHiw2mOMBPBtRDRe9LvXwBOsnx6YsYS62WA2mQna5foYFsfsD6Qgik16+7KOvy7aNrlU+Lf64E1BRebonWqq1z6HSXNLLt3dtv6p74Sa9I\\\\\\\/30h2NKR3CUJjACwPRUc23UDbqactSZiZ1Itb41iCgbqLzTp8RvPl1cCuFhq2RRx5albJvKcnCU6wzIvk1pyItmBsTIBu+QJOaJ3KxEBoblQ2ywG8ZXfhs2YFuZ6FAnWii5eFTMNK1QQWUf6KrtJ0YkKF4Gfd9TG9v9\\\\\\\/hEFmuMullu33KNm3Ye8EWnb6Ub5Rg9PFzRiAbmMFTxMkP0Y4GY6k6od8I3kXkBKdJ4MOfycIqFu+NGJE9rM5HrCd0NpmlmCbNWJrqo2Hfwpj3IbGJ\\\\\\\/8XQnRq0v5D3X3QpXfhmLUVvVnAEU+buTb4mLVxNfPFsC+7XcA48xfNHMIygyJV2I7x5bxOUwILDIvXe9SXfGP8mLWJAvZOpTw0FcUiabN4maJcceiyYAWjxFJqiEeGUZ\\\\\\\/Wk\\\\\\\/PMHG8+AyDBj1Qit9yMEZtcp3ZYlr4QoVFSzFOQGw5El7nS7zphtrO7+aYUilwubsN9\\\\\\\/5MnOYuEk+Kc8d5+6DF6hA3FJQrec0AGNVG2Z4gE4xAGcne8DpM93tJ\\\\\\\/97+oke6Z433FX7LOiju67etGB21VnaBu5Mh5z5X8X2vSNpgsQKEs\\\\\\\/pA2guwDCAj8seJUeKgbo4W78hAj8oPVFd5JwGTyxww649VAT\\\\\\\/\\\\\\\/7ubTHiLk+e9GvNcFHinngrZwHNXSFfMzD+TOnFVM6l2gnFFuC61e05z19wimg2e6\\\\\\\/IDc5lMZEWHQwy2rcGZrbj4QZRHMkyntc5+lmeYu8qhp5Eh\\\\\\\/x7XIdBtvTWhWJK777pQt9G\\\\\\\/+HtwVscZbj3lJLl9S3pnCutU8HtgPrq1kCMTAhYpz2soFu2KRs5S4RyVAWBdzWr+rNYoQCRHXRtcO0l7AW+j8bD8cz7v02eAwb5M3hsJZpOAFsEOxk5UIi+8TbK6+cXXPszLFBHHCmWwla6nB7Qklo67EGVnu7IfozD97lAxslexXGeou\\\\\\\/jl4Z1OSmmINcNk\\\\\\\/k90SNl1p16PQhGj6RJFrC3euz+ylDiWEN7ZTzjdkawjDzuscsvCagRniy3vznE3etraXgTDl8Wm209EgfpQ3uHbPR62EqZG6heie29CtC6epldQede6jqKF\\\\\\\/8UxSP2IRca2aeTGZEyyuPf3cgNU\\\\\\\/ETN2pt7GG21336Fj\\\\\\\/KgEend+0nSqd4u1uJWcLaN1HTsN5k6t1sERcBVi0H\\\\\\\/0uvsyoMgNfjIadVZDIurnh7o5q9J7W0HapKzsjMRGC5WCmnxuDV96Lbw5nDllTCfQABiirts3gKBz1oEWV975o0uxFY6l5UEVc2zp\\\\\\\/D\\\\\\\/+yT4LXvFiT7uPo1fxtzDldgce\\\\\\\/092ZQ8Isr2qzbcvYk8AuHO9COf9eWmKB+fmMYQNDdENiX89g4mKy3hMot6GDgbzQevZTr8jF22Y1LrpLe3H4\\\\\\\/0YuXD4\\\\\\\/4XfM7aBn\\\\\\\/hddpRzDEzx+NT3+sZRxkAwhDCwsY9imknIFi4Wv\\\\\\\/4J8bw3KJmqyUy65Ftqcfl3JLZlbW4p\\\\\\\/i2waMztYaXoaYQDONhAN9WXlMVaAnVQ8Eu3FZzwMBaNN4bR6jWvXDe75CHkQZqvVtj59jTzlYHqaYCXARIHb6R6Zi7XymQAN\\\\\\\/okp3EJlgqBInL9soBDEkgihd1gQbAqGBXjqtQ2ucJqedTqFpEVnfdWuJ\\\\\\\/7wEE9FyNCHyySC4E8t38fnU4XVey4BkTDzh\\\\\\\/OkrqeAAWy0C14nac1Zqk4QCrKg1uRTXyUo0X07BOzyiisvDpzepS7PK+\\\\\\\/ONf6ajrYgL4zk\\\\\\\/rHn7yuLBAITkwAZekjP8II6H1FwjvzYqki40lwhWlez6AAM35Lma3dl0QrdsH1j6IJZ5AdILs1y+88YaraseH\\\\\\\/GvAWNRuwTMZVAKAGfYr9bFiJH55slWujCZ7V9DaRz0dKt90pBdzqihJeeecTZv1BSBkO64WeVcFUSeiOfXHBx7LRtXrFVyt67PROhq83hpneR2LjoZR5r7ZBIcSgfNJRQbdcKlBWTP8QIkFGzvo\\\\\\\/f2uv95V9GFadGXuKEC9GkcyNyF1aQdHgDQZVx3VGvCLuzTYnkHRSqe98eZriEw9rpZ\\\\\\\/xh1+nYwzHaHOYHCddffRyVTRlIWfb5equ4OjNH6lNKnnZLsoT8P+hcRMz80oVn2PP2zOfmHxe\\\\\\\/kBcK6aLYljIPRpbIl3mf6r0bgYuFhdK7Yy1CNb3\\\\\\\/f9BfxNHrYrOzASg38GgQYeulBRiyctmyit\\\\\\\/d3lyc33xMkPR+xzZoecoG4ssMm813hFfiUOitKQFDNTIPelsXoVWqiPweqlN7G8YjaHyc+m2mVggUiKMx82ooVLh7E1IuuNBMy2fhAJUrnohfCuHANmniFS0U5vp8XUSBFEBZvqHX3XE3IbN0gWzqr5tP7F\\\\\\\/rF7VLHL77T3UC\\\\\\\/oQg4LbSeKaV0TMQ5vHswVAUY3\\\\\\\/WtTldesgVFfmnBxSw8UZL8i8AdCv1ovMmU+YNmMse6II9JuKTOBjR9XTsj9nrqgJbsX4r7we9yuV495+3pssVhUrOzWTNGDz9cLzrABb2l2QGo5JsI0LalRhHXCK8Y53RL42t9HK6ZqPoAf6422CTIowtwzQyLNlp2pK\\\\\\\/8vzzJKhiErQXwBO8IU66WuKFHuRzMB2WJPhaPbI5fSARNOwe4Vrwiht6Yy4RRBnv5C0E\\\\\\\/9UBL4PqkEBNvIZNGbRCKxYfX0Vps\\\\\\\/UBXyXeY5icqo+AUpkQehSHbsOkxTywM1ZX8Diqcu\\\\\\\/eWdIyAOouR9inJN8wk\\\\\\\/ONft4Fe72CEzUOrUahzh1ZxtNzmYHlr0xMyW4NznC3wk938UuCRFglWXNvZpuCVBLYA0rcz+5jBVLy5x6m8oLpuNwsQYL4bsTztFDRrlJdrZs78E5zQpJi1SogXILCzPlTIq7VtMWNWFLCIX4oeKIjX2lfTBOGnVH6iOSRGlgn+RusOC3Z6vw72QqxzA4WOZpYp8lXrZcCak5dYrl3X252QGRXBGZCGY5SyUqtzF6Zg1S4BGtyD8ZhUH2u\\\\\\\/EeKHIcqmu8MlU5n+53W4uBtB0BvL7LTInTsNo2C1WpZGMFTEA5qAYQH9QSyZ3tp6r9yRKybi0O7u1yeeYVSB9HTcb52daImM40rouR2\\\\\\\/IMXKQaGn4l0jQc4brjHuM9niLCXy5pSNd77rwCa1q5tpHgKcrUVtJC4CwabsyiZ3ywheoseGjBMbokF+lDC8jCFSeDd4ibep4TJ7kS4MtXxcT0cQ\\\\\\\/541VssnEd7vg8h\\\\\\\/RoTcyXSpm2rB5MpP9VKPStL5dRfq9VfFq2aKjLT2\\\\\\\/EIGbzgo2ykNcLJ+a\\\\\\\/eTBTsxU6j\\\\\\\/ShR047OKEYODPVagYsaHr6skSM9c9mYKGdXlFEZBRxoNp4bcMbs\\\\\\\/Aa\\\\\\\/z\\\\\\\/IgtzZLhYsDceN58VK2LICnEEgXjKWqarK2jg80grmRi3\\\\\\\/Ea55bxApHI7hGOHW+onjisAsjfw6OmJ53dkIVuVU4KZVaDK8T98vgGtozivCQiFk\\\\\\\/1tvY\\\\\\\/T8bd1FFg5Z81n0ulKGvHJbS\\\\\\\/\\\\\\\/mCL4dMzULq\\\\\\\/npGGPRRPVgh8+XiN4WDcPU9JGzyvhrHvrdHej1yTweq42rKYTO\\\\\\\/HxHlFgOw9WwrPdt4bjHRTou9xJb++NqKMCIfZDJTxDJIe44tK1g7lbTNGxYvZ+DHF1tIohTJTKD0RF1VD2lYOPv90adKXaSTEfvnSGlJP\\\\\\\/R6d52pWK\\\\\\\/DCGPw6r5Ug4Z8tRQLpsBLb3nRrFys3STCmna+1dBhTAMUdk63Ex6TmlNSkSatkhpA2\\\\\\\/qwiEZT2qqw8fOHHfWBSMye31nb6ogKKd7d9xMQzdkYVqY6zOLSyVcX4RaGuE20MCqtSdzFFcHSqeakYKf75aYBEPcxykTgt5BdeEMd5Wj1T5XZg+tK2VQUM85GFW4m9SuL7auAFAcX1hZZhhBi2Qcgnup1u5n1e473PGJOQvMlDbVfmv4I6EoUPokPklwNtGofXH4K8OEumPjI52vr3aB062qvVzUEeQu7caLxegZu0qydJRveqfpK\\\\\\\/n0ZXGi6Ds1KxYW4q8pBk8b7FmHTw0bhA7jzu83fC+xz5YnuoAvshSBnfdqdEuYonAnSmFWgx3fzEBIBJfvvlYJIGSsp3qScxb7WVs9NvZ8GSsFJgRK+F5KLdV1NBJ8PbclDOm73IeBS0hhK\\\\\\\/bFG1R96SbhVk65L6sII0svzOqqRsrIcy+9bSl4v+lxs+NLkZix8ZLpq5OY\\\\\\\/tbseICCLSGb2MlxB0jgPGa9tq1jjCjwfJjtMotoqJ2ppVztn6r8ruLg2ufJ3lJbCaV\\\\\\\/FthVqWFG1vyjk0EC1oHw8MQEVhwthoeLQ8ueJRZDcFT8oSHbG3+0l5q\\\\\\\/Rfbr4zQAM\\\\\\\/YUb0WQFZlG1icjLq6sTDhcexgzeov2HZatrOULAN4r7HoDeOK+3m7lS6FmATNee6nAtXewRyNWCQvAC\\\\\\\/8jv5QlmJnQp3B5bxPdSog\\\\\\\/Gj25RK7KK6CgY98RhVLWi2HB4hSTCVOrQI326JxXrMFn6mQjoLJQHScGtgHFa3TorqhpxZ7kNayQvra3MC3ZGpcVSjZ2O7dTOF6SmG3r\\\\\\\/JZ6Agw7FxF5uLHs7Zk+x\\\\\\\/4gb5030YYUNsba5j+PgoiIi1XNTf5hLZZgdKJl3nq\\\\\\\/C973aVpLKVq3AepCx8UJdvl1SFOXR2ZItAaG3zK1yhBK6\\\\\\\/GpAQypzpjNLHyypL+XQUD8QGlNjOsPhh\\\\\\\/Z35igKH3Y7el8kxpqjvmsk9WG3VUUlivES1dGKt\\\\\\\/rxreY09Ldfloz+8pmO9VHyLPX0lqbHKtgBM8xe3XJKtx7\\\\\\\/GNSbZesChnlrXKwRUhZlGi+UnOzGeTk+mCuFQ0FrnOrqmylkHmD36YmVOgr8yHipiRPpoVTLfR+hupBYUMx5xoal7KRRBkjZr1aRi9W2JJ62DNtQ9lb8yPg95beSfyyj48YXViFG+yEjOy0ylrIHm4km6clZ9UOWgtrR0jcAE83VS54ksHlwkKoVS7mdD5CZSYyIEGE1WqIo7Ux\\\\\\\/Q3ksVpQHYDN5Jhn8DBYRetr53bp4zJCc850LFktLvSrm46t1QDVC71oFlpSusj2OwmSNCkKpXFxgwfniP8djFvpM10O29tAtKQ4YHn9foceqKbNWMsAEBdQk8357PToEfGpzPXEAfRCkBhziVwoewhF5GXBSdvpYIqQabSANsenB8tJAkekGwq4Cc8eaTNwv3i+69P2nqFO6+3MTAflJRnJpuXkvXNK\\\\\\\/SDWy6QsiCiqG0TM5bi9OQUreGciDwjt0UVkLGwZC3T7ZRzG7N92aC3Lw3CKbB4QDYt4\\\\\\\/0rD7OlVg14MANFvjYuFdHD8BLtAMk4HV\\\\\\\/pRv1MJQhXTc5WDYpUAo2P7hFUt8yNzei1fopCi8EmU3b0gN5Xtf+ZmWU5dUxp8yFBBRTbrbWtwcthPFXHNknBJdtiLDLSk7RAueG0GSWmhAuRYfR2WCL+RMpfbvy\\\\\\\/gHN1v+oS88BaAT2M5qMR7rgsKVtNc5Vu7Tu4Dx4qtTZTKwgwQ+b5INQwxye30fujEholhxWHl1DU2Mfs2q3xa8MOH5YK2FKuatIz0O3z8y3dHgdM\\\\\\\/446K2BB\\\\\\\/tFO82488TXGo52nZ+u\\\\\\\/x+TFk9skyvxWEaj1J6aHWPLv7cG8XK85Dtsh04lFw3wmzu1OmXnIVW3ryfNUMmtRuZNPPzfQO42YpehSv57h5nCDKN\\\\\\\/ekaHY4Ia1GdHTQpJ9nxe3\\\\\\\/v3wOo3mbGR+cRjIBlwkqDyXRSzJPXPZ4liFpoZEUKQ73Ulh+AARcRiF3ossrYhAug+Rkg6nu27R\\\\\\\/jgvTZBSEcHh17S2TnYalauULHFTGam0rRI\\\\\\\/hmnmHJ3+vgFAkmnSE5RHhdUgPGJh4mpAIdGK47pcHlSS6VXw+cuONRkiUMuyh5eXKRiEOoASaXInenOdSz6YwBmWzc7Awh1uToLXGH0znB88qlNkD8iGPGcOyzNaMjfUoweHv503JIUw5Y4tzZ219iCnPObmjWJwVD+pzubneVuI1jAofDslJMJ30riIGemdhLXQKc2zgKnBwJqO\\\\\\\/q3H+ZIgsZkgLudsrufNLzkZ2WG2OrXTZiMYWQ+jBktJ5zWbZnWXxpaNXjBqIInyJDlC\\\\\\\/WzOh7NKP6339NB6mYigSkMXapXLkddp\\\\\\\/QYjiS1TSE+wIBi+AU+1Ag57wBQfgLF4pA22axAHozxR7wxlgh0K3DLacvrCsU4gIrRCM78waZ3Hrpk55epDkbg7R0M08P\\\\\\\/UEI9am6XOBKY4n8sa3Lwi8A3KTXQIr+VWZ7tub7m\\\\\\\/r38KfVI4YAoXu8WCM6FByNcxYQscZ8i4hPIXWPJMMXdU5cFzDDDEuwD5FPGEmIN1rzH1IrFwkkLqZNJtmNrLzxMT7HjyHwYO5I7h2mDIGpClrzGnhfVe8LTrt6FljsP0tU7DfrX9r35jCDcHES+3fxwHIsO3hmTgdzJtrzix8ourDKFbbjKOpX2QmgSCP7q2aNbNT5TZNmDaw8XxUUorZMWNIjVDA9YtS4rFv+g\\\\\\\/RxVOlnYw5c5thXyjoiZ0UH6JqnujCFNyrNuIhLwsp2QbsQGozo2Hvwse9vSIrXIqZf7tPC5IvT3DP8WxULCiz1YhiqkGZWov6XqCIYFfuSUNv9af0hG7WjkDJ8naE44nSngyOc9MTqW8PfvXuR0uYRXBbmPjiaPR9lgA6FxwsjF7KNs+6\\\\\\\/v7+Zyqo\\\\\\\/LFQ\\\\\\\/x3pYZ4wFESD0mIz5D5GxW0Yo80esnZ2rCJ0DXJbTCgN\\\\\\\/QFys6ZlY7oOWAfZCxgoxnQ29iE494EKZ4Bjon4qqd\\\\\\\/iHaJnSrpIGnbTycGxX\\\\\\\/1ybe97bti7ogvoa+4kVpkXLgTwoxdTccLvMPjGBTUZsuaNCANYImfHRulO8kO0iqZTX+eINgxffn1rNoqxAypL3D65KNgu26Y0wOOo9KmmcAEBewpSJ+nuVVaYkPv6hfG3CIkG5MTNkBzEnugyH0B5PN9iKNIeN+bKJNqwqGF3ssAvQrVvNHVmPhqxf6vMmEN+ArWxRjQOxbq2G45Y13Bep5Dpwbf9WvPx1Hs92Bgu4jFoDle\\\\\\\/RedXpd9FAgrxV+6nCHir1TZvMy4WRk4RTiT7wde9N0FDukGWNkAQti96KAuT3J7w5ToLwJSHY6NvrPUejY1RiQn0T2ZSz35cOLi8XLFEocnDjQOCuv\\\\\\\/phBpnz\\\\\\\/bu46foRYdHYScv\\\\\\\/ZL6kgzqqquu9YJ8iB5FpPjTKA0Deu4WtPM1sQiNGsmRVZtghoFE0LuebsM3EYyrfqX06Dvt1hV+WfYg1ZfW\\\\\\\/EdK7zzOxr7bLiJNfKVrm6+0pPulIoSV5+0bFS1hGcGigZNiwVlhUAAJuziT8w6pGMLRpMgmQipWM9Nsj\\\\\\\/I1AWZkiu3wOg2sRkMIJb8lRxnGTwZ\\\\\\\/f+zc86gyocgf2XVS55Qs39j4iryjA4hGFkvOzShjPDyQ9oNkk49MRos3w4K2ykYUv1AHTfC5hSTEa\\\\\\\/21xTkZZi1y1v8KLhj9+xyVtT\\\\\\\/31+7W6nVXg7NafhW6S99G87Gy\\\\\\\/brdKVpKIdhAqo66Cg4bIMsXn2EHU+mmd0EfNPkBzHJY4vvt7jZ+XdYeLRroOr53lrgZBjQFxYYRavi8xMWR\\\\\\\/7tTLfRr26WaprV1fe1VWt1BksmgXdE0Dkhhwy2iSIkJcaJbfiAgqCpp4+wGOH0kSS58W1jQZa02\\\\\\\/N+nedIDpC9uhpsDiSnBcx7Gt9w9x+sNYBbDFq64BY\\\\\\\/ayrnGjGkfER4TdTM3JVYrXlOgDN5gqHJG\\\\\\\/8HaYAOd4b9aR\\\\\\\/2y7cGUMJxS6sE+3+nM2AP0hHtgFtCzw\\\\\\\/tAXVJsFkRymIPZKbK0PLEVt9+bzuWIl3r+pRI+MO2O7FWTJMcO9y13yG5avHtZka2lYZHvQKLjEMJHzUuZZ0v3SLnhBGa2i4YTvte55VgUTCOgCMW3+ksHLNHxiGyRv3HA7jRQxtz7tAf7icf5YOLWPseWO6+wkIwlDUrkvTO1liTqOGvQMkYf6T+2nGOMKeGWbGaaH6JyT04tmvTZTUtybM6Mz\\\\\\\/Ka671RdWr0aWBziVWuL8NHq8tnS0L0hobu\\\\\\\/kwbmouUhzDBop3fvvo3\\\\\\\/qz+CAFvPY4\\\\\\\/XPh76pcqm4FdMAVMca6Jx91NYE1TfaZTYlPX0CHqMLUZgtxRiWCYrED0+n8JOEJXpPug79iW3n7P9c4EBUls\\\\\\\/vy4CNtGRBnRYCq9873W84ZiyLlPF\\\\\\\/iZHe9BkBe9IDHDXJUV0jGtdpiDHmAtplKImWAP66Pa37ZvOxVS2JZuLHv0qzT9BWMt22qbuYD1c34UEH5mdEKxqF+XCsbeI6mI5D15\\\\\\\/V75sNbmXTNvmQmCp7Cxdq10EumIZ\\\\\\\/tcFIKq6sA9RZM\\\\\\\/7tEmsk0dVgoLWjLnzre3lT0\\\\\\\/LxlTzB\\\\\\\/UTQInknHy3+HZOZImN35riZMyCZTi42eFRvb\\\\\\\/9mpa1xJpqlXfYB0sZ\\\\\\\/HV80IQQK8a6th5HbwIRmH68GN0oNM5hvWa0r2w8cYpF+Wct++Kr2UoktM7mDnAN5BHqne086bY2JZ8400fKdsqNiPGVwvVGy+8A255e0sUZ7mh11FU7XxYr\\\\\\\/j5gMghQPr2sYEWx0CChunR9INH3B2v8iZkkiIJBScSdIpbDkmkz\\\\\\\/avHEoKZkKx1zVGU6VwRI4O7UuwC0yyQEvcsE7vzMqatQwKPrH4U8LYAWt+WHw6bc4LZUNFj2ASLFZW4SX7jP\\\\\\\/RSX4\\\\\\\/uDOabqc16AKidLSDms3l39BprzCZM5rmdjPig81TD4O44fXNZkCNA\\\\\\\/3PlwKkv1f+cOW1y2+c8y4M3LCULk0aO43KT5iOJCI8t2n1Oz0QZ1O2qexFOKS7HlxPGxhTXCpGI3zOOiwMDMffaiBUysfWz+CL0+gqfixdatzCC8X+279ZxTDwixNeB+WjjdjE0ISutyAjwBjTsMLpg+kLKtR+2WvHha4qAFKonIpR4FevJK7D+IvKr7VSsLuiVQojdM+jCBEkkRfdUG4JS3LBX5MpBCjwjI1NPIx0gwBVzd51yeNvZOjLIvR2Y1Ea2fXahuLgooVkPJjYScYdasNCGbLGzftKW3e+vJ5cb3HgS\\\\\\\/SdCUvWUCIrHEa9pCZDnibSetkuLyEx9G1QA4Uq\\\\\\\/iSlME1Euj8WP8EEL1PutzLhUa0Nuo\\\\\\\/wKNTRyUnR0SvAWoRcHI2UzUD\\\\\\\/gxeFFNfL\\\\\\\/ScVJKmC717LP\\\\\\\/52UFz8ISwnZn10DbC6yjW6XjfvKcqJ+kNE2F9cNvi7bdFlq6mC6bCHEOixjxGY1gqnC2Esbjdg7nBOWtzNyD3PPOlH\\\\\\\/6uzXFnsWuxDvwzRmtwSvOTTQ9EFFra55CVvlpCc\\\\\\\/eGAeOzm1l0\\\\\\\/OPZcJWt7vGAYtyGblRRrkvl89RIe2vq2NsAlrjRxQjZrH1wxCjMCctPjet40oFJwJfbmSzU5Zsl18\\\\\\\/l2x0ahAvSeEwwg3NKOHfOyloQ3IKwWwYwkYi2Bgaa\\\\\\\/kMz6xaFIUZQK+7\\\\\\\/4DbW+xZ5wBz2lzGNlUfhMzr7LJEKVR1nLG48XEjL1Kf8RDY844MPmb4mgMCUNVutpFJ7Xn033LrwZHA7uMHeatcvtmtzHGp9VOEmsEyxEc9PZ3maU60GCy+DSAX\\\\\\\/z8DZ2LcXDTJE5WRLnc5X+sq8idd3sL14zMRDxAZvQ8DYIQ+NStXLZS3PzrpFpCJScvDrozahsFK8U8ql4OJQrgMqx2MPe0v2+P4eLsuEbbhRZaFgq5Z09v0mfWG2OBLPXCa8gHB61XoWQMJN7CR8yyCgYKcuY\\\\\\\/K2W2VNk77sopQZCvqy+TWiGusxxg+iILBzlI8yklFDoLGp2jsHR7dmr0rCKGrQ+eGkNOBlldN1QuFKq2BANH5QJqGCVZADWR\\\\\\\/H2TgjViEfIL8mCOnsn8ZHAi4UJU+cpzvA7e5bacCXf5ib4bEHzMUERfTlgc9O+QQDaDqSXzU7Kfmo\\\\\\\/psQClPeQZHbFmkqRH8IusQuY5c4s2pKTzZiTu2bP3pzdbEZnuu3gZw4dwVMjOMaYe1gTZpghjoNrP7a\\\\\\\/+Dxoe8FHSPfaZ+j3X6ePxHMuYbt0PHadacpVTLEw\\\\\\\/\\\\\\\/Zqur1rjQHXBC7Yn8pHkEOUev3gCiKUnBhrpjVZh\\\\\\\/+3dHqNv7GFnrQXvO5EMhCsgY9vmqyIvYEmzuMf9Mx3UY9fM7b+e2iHps+EPNqRVNjXcqSSEyVD1h6RmeMzSyEfyYCxkk\\\\\\\/qhSyh\\\\\\\/csVfHaiMr5QBXvmDge6SA1cc2dh9D90j6Kkwh4mew5N20nUd0cSe963VW8lh\\\\\\\/\\\\\\\/iB3r4b1MaY+YnM6ICQcxFSXRe6CqGE2HU6gghtg7gG4P0iLXMySwyQIKSO5IYfuu2gLis+hHye9RU7ggVazh+KIC4NDbuvhGlZrmTXNtzpyKh739k2HsFar7Li\\\\\\\/VSio3a1suXisyQBZR2xnkGuE6nz2HEBDiDAvZUshy9BrffOBtCkF+Qb+OIS78rbdr68kkMdJ3eCTmdChShudcnNsAPsd6S97zK\\\\\\\/x1R+HSQ6t70ZFaDN3tFhH6247Xfr7qIUKrcesNlc1ZogGi\\\\\\\/Lt1hp3qgKD8HsBxDo5fXnjs7WXQylXsriJDvuOMfSnTTAwaK\\\\\\\/yxrhy0dB8pjzuBknVJmISB++ZvVpLuswxiDxW1sAqgiVNQ4YXixrviUinTgRFDBHpLi8mVVqY8ojuwQZjNEruCt5bPMGni0jkIXOK9kbsO+icYtsmmis4hxc9eBe260NfGl7MSd6TbWY1c6fzKa7jcmJ\\\\\\\/bWr3xlAhvyqa9Abm6G+Fts6a1neXzACrtbjWupKWBiS08nJhR9qYxKTaIjof\\\\\\\/CS4n0T6rxJUXmgprRzmDZkC4SgZe3klIfMhnsLbgadwzG1EaOzI79EPMhZFD31LFPNFk+dFe47NtG+EZySHXlv62TyIJ4sg5DQ4O4g44jYuZqaLBz\\\\\\\/SkSJXPOcmF9r8l9VQn6Qh0Dtl9Al82fxUu8E7oaN7HMBrPrto1PqT33El3thBp5IoZIhOyEWrJkWzwD0fspF3yruXR9w1tQvucHYAXDxYagSJ6YmH91GVND8t5oj1qoykPFJmcQxDxqtGvzBPGreVIksqAg+oKHMHs0Qa5IVkkSSn\\\\\\\/5dX5iTZug2fhLZIaRqmTQ+eSE\\\\\\\/rdJie170tUPgtSDJrb6j2QC8pJjBFYylQvgX++PHCs5X9J0bFArrzUVAobBHuyUjA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022yCqgNjSk4Vj5EfvrTtj1kA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c5b5e216bda610b06f3193b7e2e3c45bd2313b19f7ea9b64325e405ab0fc81cb\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022QLx898G7hFx4O\\\\\\\/RVDDW8MMsFew5eV0Id3z4NLmHS0fKAPJKjvM4ZGT8VvT3iLAriYrTNUvOQ8Db1SHrut2JgKQb4nuc2dUZuhdNZD6rI\\\\\\\/3fQEt\\\\\\\/tzAUxH2Bx+LPGFpEtd5WLsWw5k+oOcd3nR7mq086LnPHHSmKW6gNm9oAo8Y5IP6Uyup4oFDYiYK15\\\\\\\/PJTMBQwjqb9ZblG0i0+ztOje1JpO7BMkw5MGXyDmRMAbn0yn+f+1XlSCXws1vzThp6jX2EeP4jBxD\\\\\\\/s+uPnohlwALTqpcBUFh7UV3d4WLes1DiLrANN3FCyLxL9Y8CdUcEEcMxt+2HQkdO+fHghjQRuKQEG18B3mJGOnevm9ynVhjnj65tI0UlUMSGBmjvsRpSZWPI+rRnhsdLgqMyoQwKasoboj+SnzM1u\\\\\\\/sNf4njXYDE1Zgn+WvOb0dv5zawlCzJVZFLVTwEUCVMPa7aNc5GaTcS56BdldColwUw22rcib+9ph7Duxx0Tw2BYW06ZXVa54a1uCYLpAcrBU3tDaIwfyt9kXKDqeQyAiBUKlVmWsQGPTV5G\\\\\\\/zXfQdieMFj59cF\\\\\\\/Q\\\\\\\/74At+t08snU+x7GxFZfsH5yNfRQqwkfD4mopDA12OKbahPsGHUSS0b1iPETBGzpu97Dac1YzwqXCEr+z6f5jXmmHnOSyNeJf8wH8RS\\\\\\\/QTn8bG9Kx70isTAUn5fBAtQkNQmw0RxgVa7Ryje3W7\\\\\\\/DK2PtjJhYzANOpdi\\\\\\\/LWXlcTfMDIFAPHsIPRgNKL4UokiN0z5zpG3+qhytelGCtufVk9ToNAwdVqvrL8wN3T43FStsLnU7splCzepH8pXhL8zvreUOWZkRbBoCx\\\\\\\/JOocpm2juefzijwG9R12zrWoI2lkCCdaay2TVeJ2Bn0\\\\\\\/RUqfFYViuwgomcXSbJm6uMZghA33QKxnxC9uQHwN8yDmo6AB4ZAhoubd\\\\\\\/3ybZYeDXC2TizEeeF79OoILdaDMSseh2wp\\\\\\\/I9yJYeNu71sWZSmLNAlTuAUbWFfBalYU1ZHVLsH4JDAHq0+RAhbq6puWdT+j8WWdYMXs+84Vd58SAitf1wAUY7\\\\\\\/dtmqfQvElqwM55PWtwTvw2mxHuWC6PR5Gj7Bw9PEDjxcolpS\\\\\\\/S4uwydCYEVQHPIDLkZiRm5ISS2uSxZ0Gu1sj0IrPQuHAfZpuVQ\\\\\\\/LW2b7EZa2ckA8w+sQsTGr\\\\\\\/cXorcQRJ9PNSs3gNLbZ5dCXXtbgWSy\\\\\\\/qEQyNn6fPmN6GYa8a9MPIHmusTKPj4w2du2faReVUt7\\\\\\\/BUSrl\\\\\\\/JIP3aj3LNFGU5LPsXsf3\\\\\\\/CuZicU3BvxeU2J1QSknGmkIGsXA8UPXwznBVg0Sx02R9vN7zPucqS\\\\\\\/EIIrdXtrh9Q0T4uOkKuE4JlS7ZFCSqGIp6zsdJ8cPIOiGvpQJPMJrcso1Jio\\\\\\\/qw+B6CauOL9WPUKuHy+NPceg\\\\\\\/IIRK+57Pt\\\\\\\/XR\\\\\\\/zH1MM9sFt2pYhTn8EDhQRlRaU+L9ofsuoKo+NDeEd8cppgEheRlPfDtVQTjUHNinFUcGGFWF3HU19hVh5abUGrrkvLUrcejUs5ZyfVxPUac\\\\\\\/+QT1eM3UYwZuB3ztknWicHWoyzRtJsJrOzJxVW1e1lv3SyHeihCsLQVEU\\\\\\\/KPE1LTBU81KKvOxhbxKw7BofPJFytORrIvRNMHc473d6EUrhO2Oqg6ZoOYWWduOMnOufPK5nD8VWsEUEpusTwAVNEtICKHUhsY84RIaQXiOFGYpC4YOTot\\\\\\\/KQ6Sa8vq+NWlocFwUyI9V39IKHpQFzHtqo7Mog2w9kCyo0reYuyfifZAmt34ojAg0fgW7jNwNEb3W9ta\\\\\\\/bOWU5WSglSlUSU1Y+zgWBVb3xERauA4F7pdPFGlC3GzmKDAiTIR\\\\\\\/A\\\\\\\/SyOd8ce6mfnAVxOX\\\\\\\/UiHQHXDI4hzsJ0nGsjvwR0dbstfBVC9EsAysg1nsOdOwMV4KuOVab\\\\\\\/BW+YWiQjF10hVSx4Ino0tcn\\\\\\\/jdBMl2VHq1MKR0uEYD2x6GS9gOnlyYFNlp7TehvX57BA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022f+eXhq7Lig3vFtd0dLGoqg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229bdef4fbc338c88a5a5e0784508659c75b4cd2d5dea17e082785928b9ec11c2b\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-490727 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/9/3039" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Zrcb1IPrl4qLkwuyp3pUSKT-ft90bI0Jep3gNRyS92E" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/9/3039" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-2NMbfu4axRZgIVmo5cx07_KsQlaP_hib4SDKlBWBYfE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/9/3019" title="A Bayesian Decision Theory Approach for Genomic Selection" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/9/3049" title="Tandem Duplicate Genes in Maize Are Abundant and Date to Two Distinct Periods of Time" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-2093-6810" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Italo Granato</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-0685-2867" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jaime Cuevas</span>, <span class="highwire-citation-author" data-delta="2">Francisco Luna-Vázquez</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jose Crossa</span>, <span class="highwire-citation-author" data-delta="4">Osval Montesinos-López</span>, <span class="highwire-citation-author" data-delta="5">Juan Burgueño</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-4310-0047" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Roberto Fritsche-Neto</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3039-3047;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200435 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490727" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Italo Granato </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, “Luiz de Queiroz” College of Agriculture, University of São Paulo, Piracicaba, São Paulo, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Italo%2BGranato%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Granato%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AItalo%2BGranato%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2093-6810" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Italo Granato</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jaime Cuevas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Universidad de Quintana Roo, Chetumal, Quintana Roo, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaime%2BCuevas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cuevas%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJaime%2BCuevas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0685-2867" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jaime Cuevas</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jaicueva@uqroo.edu.mx" class="" data-icon-position="" data-hide-link-title="0">jaicueva@uqroo.edu.mx</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Francisco Luna-Vázquez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Facultad de Telemática, Universidad de Colima, Colima, Colima, 28040, México,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Francisco%2BLuna-V%C3%A1zquez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Luna-V%C3%A1zquez%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFrancisco%2BLuna-V%25C3%25A1zquez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jose Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, México DF, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jose%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJose%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jose Crossa</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Osval Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Facultad de Telemática, Universidad de Colima, Colima, Colima, 28040, México,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, México DF, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Roberto Fritsche-Neto </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, “Luiz de Queiroz” College of Agriculture, University of São Paulo, Piracicaba, São Paulo, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roberto%2BFritsche-Neto%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fritsche-Neto%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoberto%2BFritsche-Neto%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4310-0047" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Roberto Fritsche-Neto</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/9/3039" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490727" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:490727/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/9/3039.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490727" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:490727/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/9/3039.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490727" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:490727/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/9/3039.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490727" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:490727/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">One of the major issues in plant breeding is the occurrence of genotype × environment (GE) interaction. Several models have been created to understand this phenomenon and explore it. In the genomic era, several models were employed to improve selection by using markers and account for GE interaction simultaneously. Some of these models use special genetic covariance matrices. In addition, the scale of multi-environment trials is getting larger, and this increases the computational challenges. In this context, we propose an R package that, in general, allows building GE genomic covariance matrices and fitting linear mixed models, in particular, to a few genomic GE models. Here we propose two functions: one to prepare the genomic kernels accounting for the genomic GE and another to perform genomic prediction using a Bayesian linear mixed model. A specific treatment is given for sparse covariance matrices, in particular, to block diagonal matrices that are present in some GE models in order to decrease the computational demand. In empirical comparisons with Bayesian Genomic Linear Regression (BGLR), accuracies and the mean squared error were similar; however, the computational time was up to five times lower than when using the classic approach. Bayesian Genomic Genotype × Environment Interaction (BGGE) is a fast, efficient option for creating genomic GE kernels and making genomic predictions.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/ge-genotype-%C3%97-environment-ge" class="hw-term hw-article-keyword hw-article-keyword-ge-genotype-&#xD7;-environment-ge" rel="nofollow">GE: genotype × environment (GE)</a></li><li class="kwd"><a href="/keyword/bgge-bayesian-genomic-genotype-%C3%97-environment-interaction" class="hw-term hw-article-keyword hw-article-keyword-bgge-bayesian-genomic-genotype-&#xD7;-environment-interaction" rel="nofollow">BGGE: Bayesian Genomic Genotype × Environment Interaction</a></li><li class="kwd"><a href="/keyword/gs-genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-gs-genomic-selection" rel="nofollow">GS: Genomic Selection</a></li><li class="kwd"><a href="/keyword/bglr-bayesian-genomic-linear-regression" class="hw-term hw-article-keyword hw-article-keyword-bglr-bayesian-genomic-linear-regression" rel="nofollow">BGLR: Bayesian Genomic Linear Regression</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">Genomic selection has the advantage of saving time and resources when selecting genotypes by employing genomic-enabled prediction methods for complex traits, along with pedigree information, molecular markers and/or environmental covariates (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Crossa <em>et al.</em> 2017</a>). In the genomic selection method proposed by <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Meuwissen <em>et al.</em> (2001)</a>, Bayesian models were introduced in the context of whole-genome regression; they have become common in genomic prediction (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gianola 2013</a>). Within this framework, appropriate prior distributions and simulations via Markov Chain Monte Carlo (MCMC) allow convergence for predictive posterior distributions that cannot be solved analytically. However, these methods require thousands of iterations to ensure convergence, so that if the model is complex, the sampling process can increase the computational time. In this context, attempts have been made to reduce the computational time of Bayesian models with approaches that do not use MCMC, such as variational Bayesian methods (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Montesinos-López <em>et al.</em> 2017</a>) and Integrated Nested Laplace Approximation (INLA) (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Holand <em>et al.</em> 2013</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Mathew <em>et al.</em> 2016</a>). These methods are faster; however, they have constraints that may lead to lower prediction accuracy, which is undesired.</p><p id="p-4">Using molecular markers (<em>p</em>) in classic parametric regression with n individuals can lead to the problem of <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-1.gif"/></noscript></span></span> setting, which can be reduced by using semi-parametric regression such as Reproducing Kernel Hilbert Spaces (RKHS) (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gianola and Van Kaam 2008</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">de los Campos <em>et al.</em> 2010</a>). These approaches assume the contribution of molecular markers as a random variable in some distributions with a covariance matrix that consists of a scalar variance component and a known covariance kernel obtained by markers. This covariance kernel can model genetic effects as additive, dominance and epistasis, as a mixture of these effects or even as genetic and non-genetic remaining effects (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Crossa <em>et al.</em> 2010</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Technow <em>et al.</em> 2012</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Azevedo <em>et al.</em> 2015</a>). Genomic-enabled predictions are usually done using models that do not take into account genotype × environment interaction (GE). Nevertheless, the advantage of genomic models that take into account information from multi-environment trials simultaneously has been proved (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Burgueño <em>et al.</em> 2012</a>). Hence, a family of genomic models was developed to account for GE interaction; these models also allow incorporating fixed effects of environments and several genetic and environmental effects into a variety of linear mixed models (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Jarquín <em>et al.</em> 2014</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lopez-Cruz <em>et al.</em> 2015</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Sousa <em>et al.</em> 2017</a>).</p><p id="p-5">In this paper, we describe the Bayesian Genomic Genotype × Environment (BGGE) R package that fits genomic linear mixed models to single environments and multi-environments with GE models. The increase in speed is achieved by reparametrization through orthogonal rotation of the random vectors, allowing the use of univariate distributions in the sampling process (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cavalier 2008</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cuevas <em>et al.</em> 2014</a>). Also, some special treatments are given for structured dispersed covariance matrices, in particular, those structured as a block diagonal, prevalent in some GE models (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Sousa <em>et al.</em> 2017</a>). We present statistical models and algorithms with a generic linear mixed model and its Bayesian counterpart, which is the base of the BGGE package, as well as the most representative part of the prediction process and kernel construction for genomic models. In addition we describe the <em>getK</em> and <em>BGGE</em> functions, which offer the possibility of fitting six different multi-environment genomic models with GE based on models proposed by <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Jarquín <em>et al.</em> (2014)</a> and <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Lopez-Cruz <em>et al.</em> (2015)</a>,; we also give some examples of their use, and compare them to other packages that use Bayesian approaches.</p><p id="p-6">We note that although the <em>getK</em> function is an auxiliary function, it allows fitting not only the six genomic multi-environment models with GE, but can also model several other situations. However, the potential of the package is given by the <em>BGGE</em> function that provides the versatility needed to fit a great number of different genomic data sets.</p><div class="section other1" id="sec-1"><h2 class="">Statistical Models</h2><div id="sec-2" class="subsection"><h3>Linear mixed model</h3><p id="p-7">Consider the following basic linear mixed models that cover the diversity of models that can be applied to single or multi-environment trials. Assume q vectors of random effects:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-2.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where y is the vector combining the genotypic means of observations. The scalar μ is the common intercept or the mean. Matrix <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-3.gif"/></noscript></span></span>represents the design matrix associated with the vector of fixed effects β. Random vectors <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-4.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-5.gif"/></noscript></span></span> are assumed to be independent of other random effects. We expected that <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-6.gif"/></noscript></span></span> would follow a normal distribution with zero mean and a covariance matrix of the form <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-7.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-8.gif"/></noscript></span></span> is a scalar representing the unknown variance parameter to be estimated from <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-9.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-10.gif"/></noscript></span></span> is a known symmetric positive semi-definite covariance matrix. Model (1) is very general and it can be used to model different problems in biology or other areas, particularly genomic selection areas. It should be pointed out that in this first version of the BGGE package, the design matrix <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-11.gif"/></noscript></span></span> is limited to being a full rank matrix with the vectors <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-12.gif"/></noscript></span></span> being of the same size as y, and representing in the most common case, a reparameterization equivalent to <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-13.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-14.gif"/></noscript></span></span> used in mixed models for genomic selection (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Crossa <em>et al.</em> 2010</a>, <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Jarquín <em>et al.</em> 2014</a>), where <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-15.gif"/></noscript></span></span>or <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-16.gif"/></noscript></span></span> are known incidence matrices that relate the genotypes to the observations, and g or <strong>u</strong> are the random genetic effects of the genotypes (a known matrix multiplied by a random vector results in a random vector of the same size as the response variable vector <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-17.gif"/></noscript></span></span>.</p><p id="p-8">Finally, random error vector<span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-18.gif"/></noscript></span></span>, of the same length as y, follows a normal distribution with zero mean and form <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-19.gif"/></noscript></span></span> where Σ is a covariance matrix <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-20.gif"/></noscript></span></span>, and I is an identity matrix. The previous assumptions allow the BGGE models to be used only with continuous data assumed to have a multivariate normal distribution with observations (not independent) that depend on the variance-covariance structure of the genotypes. The main objective of the BGGE is to focus on the covariance structure more than on the possible heteroscedasticity/homoscedasticity of the error.</p></div><div id="sec-3" class="subsection"><h3>Linear mixed model parametrization</h3><p id="p-9">The main objective of the reparameterization of model (1) is to rotate the dependent observations in the response vector y that follows a multivariate normal distribution to an orthogonal space that ensures independence. This rotation allows overcoming matrix problems (<em>e.g.</em>, not full rank matrices), thus vectorising matrices that result in much faster computation and estimation of the model’s parameters. This rotation is achieved with the decomposition or factorization of matrices such as singular value decomposition (SVD) or eigen-decomposition that are commonly used in parametric regression models like principal component regression or in genomic-enabled prediction (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Cuevas <em>et al.</em> 2014</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Meuwissen <em>et al.</em> 2017</a>).</p><p id="p-10">In linear mixed models, the covariance matrix is symmetric and positive semidefinite and can be factorized by using the eigen-decomposition of K of order n × <em>n</em> (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">de los Campos <em>et al.</em> 2010</a>). Hence, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-21.gif"/></noscript></span></span>, where S is a diagonal matrix with n non-zero eigenvalues and U is an orthogonal matrix with eigenvectors associated with n eigenvalues. To facilitate reading, we use a single kernel model, considering that <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-22.gif"/></noscript></span></span>. <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Cuevas <em>et al.</em> (2014)</a> proposed an orthogonal transformation by multiplying both sides of (1) by U':<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-23.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-24.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>such that model (2) becomes:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-25.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-26.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-27.gif"/></noscript></span></span> and<span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-28.gif"/></noscript></span></span>. The model assumes that b comes from a normal distribution such that <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-29.gif"/></noscript></span></span>, considering that <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-30.gif"/></noscript></span></span>. Similarly, model (2) assumes that e comes from a normal distribution such that <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-31.gif"/></noscript></span></span>. The rotation causes the elements of b to be independent with univariate normal distributions. It is also worth noting that eigenvalues that are very close to zero (less than <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-32.gif"/></noscript></span></span>) reflect the noise (and numerical errors) and their associated eigenvectors can be eliminated, thereby reducing the dimension of matrices U and S. Note that the proposed matrices K were previously scaled in order to reduce their magnitude. In addition, the BGGE package offers an argument (tol = tolerance) to change the default value of (<span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-33.gif"/></noscript></span></span>) for the eigenvalues considered equal to zero.</p></div><div id="sec-4" class="subsection"><h3>Bayesian linear mixed models</h3><p id="p-11">The BGGE solves the linear mixed models through Bayesian hierarchical modeling. The distribution of the transformed data d, given b and <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-34.gif"/></noscript></span></span>, is:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-35.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>The Bayesian linear mixed model assumes that <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-36.gif"/></noscript></span></span>; then the conditional distribution of <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-37.gif"/></noscript></span></span> is as follows:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-38.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-39.gif"/></noscript></span></span> are the eigenvalues and <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-40.gif"/></noscript></span></span>is the unknown scale. This reparameterization allows sampling from univariate normal distributions, making the convergence process simpler and faster.</p><p id="p-12">The proposed conjugate prior distribution for <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-41.gif"/></noscript></span></span> is a scaled inverse chi squared, p(<span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-42.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-43.gif"/></noscript></span></span> denotes the degree of freedom and <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-44.gif"/></noscript></span></span> represents the scale factor. In the BGGE package, the degrees of freedom are set to a value of 3 with the idea of ​​not generating infinite values in the samples of <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-45.gif"/></noscript></span></span>. On the other hand, the prior distribution used for <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-46.gif"/></noscript></span></span> was previously computed from the data, as suggested by <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Pérez and de los Campos (2014)</a> (see details in the Appendix). The conjugate prior distribution used for <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-47.gif"/></noscript></span></span> is a scaled inverse chi squared, p(<span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-48.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-49.gif"/></noscript></span></span> represents the degrees of freedom and <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-50.gif"/></noscript></span></span> denotes the scale factor.</p><p id="p-13">Hence, the joint posterior distribution of <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-51.gif"/></noscript></span></span> given d<span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-52.gif"/></noscript></span></span>, is:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-53.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>From equations (5) and (6), conditional distributions can be constructed to generate the MCMCs through a Gibbs sampler. Note that u genetic values can be recovered from <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-54.gif"/></noscript></span></span>. Details are presented in the Appendix.</p></div><div id="sec-5" class="subsection"><h3>Sparse matrices</h3><p id="p-14">In an attempt to speed up the prediction algorithm, several special treatments are given for sparse matrices. In several GE models (<a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Jarquín <em>et al.</em> 2014</a>; <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Lopez-Cruz <em>et al.</em> 2015</a>), some random u effects have an associated covariance matrix that can be considered sparse with submatrices in a known structure. Thus, instead of applying eigen-decomposition in the complete matrix, we identify, individualize and apply eigen-decomposition in the submatrices that compose the block diagonal; this speeds up eigen-decomposition and makes the multiplication of matrices and vectors faster, thus reducing the iteration time.</p></div><div id="sec-6" class="subsection"><h3>Obtaining multi-environment kernels</h3><p id="p-15">Different multi-environment models are defined based on the construction of the kernel matrices, using information available on genotypes, molecular markers and the environment (<a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">Jarquín <em>et al.</em> 2014</a>; <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Sousa <em>et al.</em> 2017</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cuevas <em>et al.</em> 2018</a>). The construction of multi-environment kernels depends on two primary processes: the choice of covariance function and the multi-environment model.</p></div><div id="sec-7" class="subsection"><h3>Choice of covariance function</h3><p id="p-16">Two covariance or kernel functions are generated internally; to facilitate the reading we will use the same names of the methods (GB and GK) used in <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cuevas <em>et al.</em> (2016</a>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">2017)</a>. The Genomic Best Linear Unbiased Predictor (GBLUP or <strong>GB</strong>) is the standard linear kernel from the properties of a multivariate normal distribution in linear mixed models and is usually referred to as the genomic relationship matrix. Thus, <strong>GB</strong> is obtained as follows:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-55.gif"/></noscript></span></span>where X is the marker matrix and p is the number of markers. This matrix was proposed by VanRaden in 2008, and since then, it has been used successfully in genomic prediction (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">de los Campos <em>et al.</em> 2009</a>).</p><p id="p-17">Another covariance function is the Gaussian kernel (<strong>GK</strong>). The <strong>GK</strong> appeared as a reproducing kernel (RK) in the semi-parametric model Reproducing Kernel Hilbert Spaces (RKHS) (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">González-Camacho <em>et al.</em> 2012</a>) and is defined as follows:<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-56.gif"/></noscript></span></span>where h is the bandwidth parameter that controls the rate of decay of the covariance between genotypes, and q is the percentile of the square of the Euclidean distance <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-57.gif"/></noscript></span></span>, which is a measure of the genetic distance between individuals. Results have shown that <strong>GK</strong> performs better than <strong>GB</strong> (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Cuevas <em>et al.</em> 2016</a>; <a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Sousa <em>et al.</em> 2017</a>). Note that the BGGE package is not limited to using the above matrices; other matrices can be used as long as they are symmetric and positive semidefinite.</p></div><div id="sec-8" class="subsection"><h3>Uses of the BGGE</h3><p id="p-18">The BGGE package is generic and can be used to fit a great number of mixed models. For example, in genomic-enabled prediction it can be used to fit a single environment and/or multi-environments with GE including pedigree, genomic and environmental information. The conditions needed to use this first version of BGGE are: (1) must have continuous observations with multivariate normal distribution; (2) must include as many random effects as necessary, assuming they have multivariate normal distribution with variance-covariance matrices that are symmetric and positive semidefinite; (3) random errors are assumed to be homoscedastic. The main objective of this article is to describe and explain the use of the BGGE package in the context of genomic-enabled predictions. In addition, the article explains functions to generate variance-covariances of six GE models. The function used to fit these six models is considered auxiliary (because it is not the principal function).</p><p id="p-19">The models considered in genomic GE were developed by <a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">Jarquín <em>et al.</em> (2014)</a>, <a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Lopez-Cruz <em>et al.</em> (2015)</a> and <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Cuevas <em>et al.</em> (2018)</a>. The six models considered in this study had a general mean <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-58.gif"/></noscript></span></span>and fixed effects <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-59.gif"/></noscript></span></span> (for example, this could refer to the fixed effects of environments). The first multi-environment model added to <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-60.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-61.gif"/></noscript></span></span> a random vector of main genotypic effects (MM) (<a id="xref-ref-18-7" class="xref-bibr" href="#ref-18">Jarquín <em>et al.</em> 2014</a>), assuming these genetic effects across environments are constant, with a variance-covariance structure of <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-62.gif"/></noscript></span></span> (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), where <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-63.gif"/></noscript></span></span> is a known incidence matrix that relates the genotypes to the observations in the environments (<a id="xref-ref-18-8" class="xref-bibr" href="#ref-18">Jarquín <em>et al.</em> 2014</a>). The second model MM<strong><em>l</em></strong> adds to the MM model a random intercept <strong><em>l</em></strong> (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) with variance-covariance structure <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-64.gif"/></noscript></span></span> (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Cuevas <em>et al.</em> 2018</a>). The third model is the multi-environment, single variance genotype × environment deviation model (MDs), which is an extension of the main genetic effect model (MM), but incorporates a random deviation effect of GE. <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> shows that this component has a variance-covariance structure <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-65.gif"/></noscript></span></span>, where ° is the Hadamard product and <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-66.gif"/></noscript></span></span> is a known matrix of environmental covariables (<a id="xref-ref-18-9" class="xref-bibr" href="#ref-18">Jarquín <em>et al.</em> 2014</a>; <a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">Sousa <em>et al.</em> 2017</a>). When a random intercept is added to model MDs, the fourth model is MDs<strong><em>l</em></strong> (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). An alternative model is the multi-environment, environment-specific, variance genotype × environment deviation model (MDe) proposed by <a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">Lopez-Cruz <em>et al.</em> (2015)</a>. In MDe, a vector of specific environment effects is added with a known variance-covariance structure such that the blocks that correspond to the columns and rows of environment <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-67.gif"/></noscript></span></span> are a matrix <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-68.gif"/></noscript></span></span>with the other elements equal to zero (<a id="xref-ref-28-6" class="xref-bibr" href="#ref-28">Sousa <em>et al.</em> 2017</a>). Again, when a random intercept component is added, a new model is generated, the MDe <strong><em>l</em></strong> (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Cuevas <em>et al.</em> 2018</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490914/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490914/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">- Known variance-covariance matrices for six models of function <em>getGK</em></span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section methods" id="sec-9"><h2 class="">Experimental Data Set</h2><p id="p-20">To show how to use the package, a maize data set is available that includes phenotypes, SNP markers and two kernels. The data set consists of 614 maize hybrids evaluated at Piracicaba and Anhumas, São Paulo, Brazil, in 2017. Field trials were carried out using an augmented block design, with two commercial hybrids as checks. At each site, two levels of nitrogen (N) fertilization, Ideal N (IN) and Low N (LN) were applied. The combination site and the N level formed the four environments (P-IN, P-LN, A-IN, and A-IN). The field trials carried out under ideal N conditions received 100 kg ha<sup>-1</sup> of N (30 kg ha<sup>-1</sup> at sowing and 70 kg ha<sup>-1</sup> in a coverage application) at the V8 plant stage. The experiments carried out under low N received only 30 kg/ha of N at sowing. For each field trial, we adjusted phenotypic values by the experimental design (incomplete block). We fitted a mixed model with the random effect of the genotypes (including treatments and checks) and the random effect of the incomplete block to recover the inter block information.</p><p id="p-21">The 49 parental lines were genotyped with the Affymetrix Axiom Maize Genotyping Array of 616 K SNPs (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Unterseer <em>et al.</em> 2014</a>). Quality control for call rate and missing marker imputation was applied in the parental lines. Markers with call rates lower than 0.9 and with at least one heterozygous locus were removed. Hybrid genotypes were composed by combining their respective parental lines. A second quality control was performed after a hybrid matrix was constructed, in which markers with minor allele frequency (MAF) lower than 0.05 were removed. After that, we pruned the hybrids’ SNP matrix by removing markers with a pairwise linkage disequilibrium (LD) greater than 0.9. Quality control was performed using the R package synbreed (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Wimmer <em>et al.</em> 2012</a>) and LD pruning was carried out using the SNPRelate R package (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Zheng <em>et al.</em> 2012</a>). After pre-processing the data set, 34,571 high-quality SNPs were available.</p><div id="sec-10" class="subsection"><h3>Data and Software Availability</h3><p id="p-22">The BGGE R package is available at <a href="http://cran.r-project.org">CRAN</a> (<a href="https://cran.r-roject.org/web/packages/BGGE/BGGE.pdf">https://cran.r-roject.org/web/packages/BGGE/BGGE.pdf</a>). The following link hdl:11529/10548107 contains the maize data set comprising 614 maize lines under maizefiles.RData (from maizefiles.tab); ‘geno’ is the matrix of markers, ‘pheno_geno’ is the data.frame with the first column indicating the factor environment, another column corresponding to the entry’s unique ID (GID); GK denotes the Gaussian kernel matrix and GB represents the GBLUP matrix.</p></div></div><div class="section other2" id="sec-11"><h2 class="">Description and Application of the Bgge Package</h2><p id="p-23">This section shows how to use the BGGE R package, first by describing the two main principal functions in detail and then illustrating its use with a real data set. <em>We then show how to fit models</em> MM, MDs, and MDe with various kernels including GB, GK, as well as Kernel Averaging (KA).</p><div id="sec-12" class="subsection"><h3>Describing functions</h3><p id="p-24">In what follows, we present the use and describe the main aspects of the two functions: <em>getK</em> and <em>BGGE</em>. The <em>getK</em> function creates multi-environment kernels or known covariance matrices for the MM, MDs, and MDe models (<a id="xref-ref-28-7" class="xref-bibr" href="#ref-28">Sousa <em>et al.</em> 2017</a>) with or without random intercepts MM<strong><em>l</em></strong>, MDs<strong><em>l</em></strong>, MDe<strong><em>l</em></strong> (<a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Cuevas <em>et al.</em> 2018</a>). The objective is to help the user construct these matrices (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>), which will be used as entries in the <em>BGGE</em> function to be able to fit the model. Note that the use of the <em>BGGE</em> function does not depend on <em>getK</em>.</p><div class="boxed-text" id="boxed-text-1"><span class="boxed-text-label">Box 1</span> <p id="p-25">getK(Y, X, kernel = c(“GK”, “GB”), setKernel = NULL, bandwidth = 1, model = c(“SM”, “MM”, “MDs”, “MDe”), intercept.random = FALSE, quantil = 0.5)</p></div><p id="p-26">The <em>getK</em> function is an auxiliary function for constructing variance-covariance matrices like those shown in <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a> using the GB (GBLUP) or Gaussian kernel (GK) methods. Box 1 (above) contains the main arguments of the <em>getK</em> function. <strong>Y</strong> is a data.frame phenotypic data set with three columns; the first column is a factor for environments, the second column is a factor identifying genotypes, and the third column contains the trait of interest. <strong>X</strong> is the marker matrix in which individuals are in rows and markers in columns, and missing markers are not allowed; the <strong>kernel</strong> argument is the method used to construct the GK or GB kernels. In the case of the Gaussian kernel (GK), the <strong>bandwidth</strong> (default is 1) and <strong>quantile</strong> (default is 0.5) arguments are equivalent to the bandwidth parameter and the quantile, as previously defined. The bandwidth parameter can be estimated using a Bayesian approach, as presented in <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Pérez-Elizalde <em>et al.</em> (2015)</a>.</p><p id="p-27">When choosing a covariance matrix other than GB and GK (for example, the pedigree relationship - matrix <strong><em>A</em></strong>), these kernels are passed by the <strong>setKernel</strong> argument (default is NULL). The argument <strong>model</strong> allows us to choose models MM, MDs, and MDe. Additionally, a univariate single model (SM) can be chosen. The argument <strong>intercept.random</strong> (default is FALSE) is an option for adding the random intercept of the genetic component (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>). The output of BGGE is a two-level list indicating the covariance matrix of the selected model and the type of matrix, where “D” stands for dense, and “BD” stands for block diagonal.</p><p id="p-28">The main function of the package is the <em>BGGE</em> function that aims to perform genomic prediction through a linear mixed model for continuous variables.</p><div class="boxed-text" id="boxed-text-2"><span class="boxed-text-label">Box 2</span> <p id="p-29">BGGE(y, K, XF = NULL, ne, ite = 1000, burn = 200, thin = 3)</p></div><p id="p-30">Box 2 presents the arguments for the BGGE function. <strong>y</strong> is the response variable (allowing missing values). <strong>K</strong> is a two-level list containing the matrix (<em>i.e.</em>, <strong>K = </strong>list(list(Kernel = GK,Type = ”D”))) associated with each random effect vector in the model and the type of matrix (D = Dense, BD = Block diagonal). <strong>XF</strong> is the design matrix used to fit fixed effects, <strong>ne</strong> is a vector defining the number of genotypes in each environment, and <strong>ite</strong>, <strong>burn</strong>, and <strong>thin</strong> define the number of iterations of the sampler, the number of samples to be discarded, and the thinning used to compute posterior means, respectively. Further details on K and ne are given in the examples below.</p></div><div id="sec-13" class="subsection"><h3>Example 1: fitting THE MM model</h3><p id="p-31">In this example, we show how to fit the main effects genotypic model (MM) (<a id="xref-ref-18-10" class="xref-bibr" href="#ref-18">Jarquín <em>et al.</em> 2014</a>; <a id="xref-ref-28-8" class="xref-bibr" href="#ref-28">Sousa <em>et al.</em> 2017</a>) along with the linear kernel GBLUP. First, we obtain the kernel through <em>getK</em>.</p><div class="boxed-text" id="boxed-text-3"><span class="boxed-text-label">Box 3</span> <p id="p-32">rm(list = ls())</p><p id="p-33">library(BGGE)</p><p id="p-34">### Load the maize dataset from supplementary material</p><p id="p-35">load(“maizefiles.Rdata”)</p><p id="p-36">head (geno) # the marker matrix</p><p id="p-37">head(pheno_geno) # the phenotypic data</p><p id="p-38">K1 &lt;- getK(Y=pheno_geno, X=geno, kernel=“GB”, model = “MM”)</p></div><p id="p-39">The phenotypic file must be provided as a data frame with three columns that identify the environments, the individuals or genotypes, and the phenotypic observations. When in the presence of the marker matrix, it is necessary to choose the covariance function to create the kernel. The <em>getK</em> returns a two-level list with the kernels for the respective model and a definition of the type of matrix. The MM model produces only one covariance matrix (K1) considered as dense.</p><div class="boxed-text" id="boxed-text-4"><span class="boxed-text-label">Box 4</span> <p id="p-40">##Continue from Box 3</p><p id="p-41">ne &lt;- as.vector(table(pheno_geno$env))</p><p id="p-42">fit &lt;- BGGE(y = pheno_geno$GY, K = K1, ne = ne, verbose = T)</p><p id="p-43">## K1 from Box 3</p><p id="p-44">fit$yHat[pheno_geno$env == “AN_IN”] #predicted values for ##environment 1</p><p id="p-45">fit$K$G$varu #genetic variance</p><p id="p-46">fit$varE #residual variance</p><p id="p-47">plot(fit$yHat, pheno_geno$GY)</p></div><p id="p-48">Box 4 presents the basic syntax for the <em>BGGE</em> function. The input for K is the two-level list returned by the <em>getK</em> function. The <em>BGGE</em> function fits a multi-environment main genotypic model (MM), with a total of 1000 cycles of a Gibbs sampler (the default value for the number of iterations), and the first 200 samples are discarded (the default burn-in value). Also, samples are collected at a thinning interval of three. The <em>BGGE</em> function returns a list with estimated posterior means for each random term in the linear model and the predicted genetic values. To assess convergence and estimate the Monte Carlo error, samples of the intercept and random effect variances are stored and returned in the same output list.</p></div><div id="sec-14" class="subsection"><h3>Example 2: fitting THE MDe model</h3><p id="p-49">In this example, we show how to fit the environment-specific variance genotype × environment deviation model (MDe) (<a id="xref-ref-20-6" class="xref-bibr" href="#ref-20">Lopez-Cruz <em>et al.</em> 2015</a>; <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Cuevas <em>et al.</em> 2016</a>) along with the non-linear Gaussian kernel (GK).</p><div class="boxed-text" id="boxed-text-5"><span class="boxed-text-label">Box 5</span> <p id="p-50">rm(list = ls())</p><p id="p-51">library(BGGE)</p><p id="p-52">### Load the maize dataset from supplementary material</p><p id="p-53">load(“maizefiles.Rdata”)</p><p id="p-54">ne &lt;- as.vector(table(pheno_geno$env))</p><p id="p-55">K2 &lt;- getK(Y = pheno_geno, X=geno, kernel = “GK”, bandwidth = 1, model = “MDe”)</p><p id="p-56">fit &lt;- BGGE(y = pheno_geno$GY, K = K2, ne = ne)</p><p id="p-57">fit$yHat[pheno_geno$env == “AN_LN”] #predicted values for environment 2</p><p id="p-58">fit$K$G$varu #main genetic variance</p><p id="p-59">fit$varE #residual variance</p><p id="p-60">fit$K$AN_LN$varu #specific genetic variance</p><p id="p-61">fit$varE #residual variance</p><p id="p-62">plot(fit$yHat, pheno_geno$GY)</p></div><p id="p-63">In Box 5, the <em>getK</em> function uses the Gaussian kernel and a bandwidth parameter of one and a quantile of 0.5 (default value). However, this can be modified by the bandwidth and quantile arguments. In the MDe model, the <em>getK</em> function returns, in the K2 list, the variance-covariance matrix for the main genotypic effect (<span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-84.gif"/></noscript></span></span> (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>) and the kernel <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-85.gif"/></noscript></span></span> for each environment. This model is characterized by structured matrices for specific environments.</p><p id="p-64">The MDe model uses the ne argument to extract the sub-matrices for each environment instead of decomposing the big sparse matrix into singular values. The BGGE returns the predicted posterior mean of genetic effects (main effect + environment-specific effects) and the estimated compound variances. Box 5 shows some elements of the output list ‘fit’, such as predictive values y in environment 2, the variance component of the main effects and the variance component specific to environment 1.</p></div><div id="sec-15" class="subsection"><h3>Example 3: fitting multi-kernel multi-environment models</h3><p id="p-65">When using the Gaussian kernel (GK), the problem of selecting the best bandwidth parameter arises. As pointed by <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">de los Campos <em>et al.</em> (2010)</a>, with extreme bandwidth values, the information of the markers is practically lost, making it necessary to optimize the best parameter. <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Endelman (2011)</a> and <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Pérez-Elizalde <em>et al.</em> (2015)</a> proposed two different approaches for optimizing this parameter via REML and the Bayesian framework, respectively. However, <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">de los Campos <em>et al.</em> (2010)</a> addressed the problem by proposing a multi-kernel average approach in which a sequence of kernels is obtained from a grid of bandwidth parameters, called kernel averaging (KA).</p><div class="boxed-text" id="boxed-text-6"><span class="boxed-text-label">Box 6</span> <p id="p-66">rm(list = ls())</p><p id="p-67">library(BGGE)</p><p id="p-68">### Load the maize dataset from supplementary material</p><p id="p-69">load(“maizefiles.Rdata”)</p><p id="p-70">ne &lt;- as.vector(table(pheno_geno$env))</p><p id="p-71">K3 &lt;- getK(Y = pheno_geno, X = X, kernel = “GK”, bandwidth = c(0.25,1,2.5), model = “MDs”)</p><p id="p-72">fit &lt;- BGGE(y = pheno_geno$GY, K = K3, ne = ne)</p><p id="p-73">fit$yHat #predicted values</p><p id="p-74">fit$K$G_1$varu # main genetic variance for kernel 1 (bandwidth = 0.25)</p><p id="p-75">fit$K$GE_1$varu #G x E variance for kernel 1 (bandwidth = 0.25)</p><p id="p-76">fit$varE #residual variance</p><p id="p-77">plot(fit$yHat, pheno_geno$GY)</p></div><p id="p-78">We use the MDs model as an example. Since the bandwidth argument accepts a vector as input, it can be used as a solution to create multi-kernels using a range of bandwidth values. For the present models, <em>getK</em> will create <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-86.gif"/></noscript></span></span> kernels, in which n is the number of basic kernels for each model and v is the number of bandwidth parameters.</p></div><div id="sec-16" class="subsection"><h3>Example 4: fitting additive + dominance models</h3><p id="p-79">Several kernels were proposed as <em>t</em> (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Tusell <em>et al.</em> 2014</a>) and exponential (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Endelman 2011</a>), as well as other estimators of the genomic relationship between subjects (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Astle and Balding 2009</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Yang <em>et al.</em> 2010</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Wang and Da 2014</a>) and the combination of non-additive kernels (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Nishio and Satoh 2014</a>) in an attempt to improve prediction. Hence, it is possible to use kernels other than GB and GK, as well as to combine them to create multi-environment kernels. In this example, we show how to apply external kernels to fit genome prediction to model MDs (<a id="xref-ref-18-11" class="xref-bibr" href="#ref-18">Jarquín <em>et al.</em> 2014</a>). For instance, using an SNP matrix, it is possible to compute additive and dominance relationship matrices (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Azevedo <em>et al.</em> 2015</a>) and combine them to build multi-environment kernels.</p><div class="boxed-text" id="boxed-text-7"><span class="boxed-text-label">Box 7</span> <p id="p-80">rm(list = ls())</p><p id="p-81">library(BGGE)</p><p id="p-82">### Load the maize dataset from supplementary material</p><p id="p-83">load(“My directory/maizefiles.Rdata”)</p><p id="p-84">ne &lt;- as.vector(table(pheno_geno$env))</p><p id="p-85">ne &lt;- as.vector(table(pheno_geno$env))</p><p id="p-86">Xd &lt;- geno</p><p id="p-87">Xd[Xd == 2] &lt;- 0</p><p id="p-88">W &lt;- (geno) #SNP matrix geno coded as 0, 1 and 2</p><p id="p-89">S&lt;-(Xd) #SNP matrix Xd coded as 0(homozygous) and 1 (heterozygous)</p><p id="p-90">GBa &lt;-tcrossprod(W)/ncol(W)#Kernel GBLUP for additive</p><p id="p-91">GBd &lt;-tcrossprod(S)/ncol(S)#Kernel GBLUP for dominance</p><p id="p-92">Ker &lt;- list(Ga = GBa, Gd = GBd)</p><p id="p-93">K5 &lt;- getK(Y = pheno_geno, setKernel = Ker, model = “MDs”)</p><p id="p-94">fit &lt;- BGGE(y = pheno_geno$GY, K = K5, ne = ne)</p><p id="p-95">fit$yHat # predicted values</p><p id="p-96">fit$K$G_Ga$varu #main genetic additive variance</p><p id="p-97">fit$K$G_Gd$varu #main genetic dominance variance</p><p id="p-98">fit$varE #residual variance</p><p id="p-99">plot(fit$yHat, pheno_geno$GY)</p></div><p id="p-100">In the initial call for <em>getK</em>, we introduce the setKernel argument that allows passing a list of kernels other than those computed internally. Thus, it creates <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-87.gif"/></noscript></span></span> kernels, where n is the number of basic kernels for each model and k is the number of kernels introduced by the user.</p></div><div id="sec-17" class="subsection"><h3>Example 5: fitting GENOMIC + PEDIGREE models</h3><p id="p-101">Genomic predictions can be improved by combining genomic relationship matrices and pedigree information. <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Legarra <em>et al.</em> (2009)</a> proposed combining the <strong>G</strong> matrix and the pedigree into the <strong>H</strong> matrix. In contrast, <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Crossa <em>et al.</em> (2010)</a> proposed that the genomic relationship and the pedigree be modeled as the sum of the two components. Hence, in this example, we show how to make predictions using genomic relationships along with pedigree information. We used the wheat data set available in BGLR (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Pérez and de los Campos 2014</a>).</p><div class="boxed-text" id="boxed-text-8"><span class="boxed-text-label">Box 8</span> <p id="p-102">rm(list = ls())</p><p id="p-103">library(BGLR)</p><p id="p-104">data(wheat)</p><p id="p-105">wheat.X &lt;- scale(wheat.X)</p><p id="p-106">env &lt;- ncol(wheat.Y)</p><p id="p-107">gen &lt;- nrow(wheat.Y)</p><p id="p-108">rownames(wheat.X) &lt;- 1:gen</p><p id="p-109">whe.Y &lt;- data.frame(env = gl(n = env, k = gen),</p><p id="p-110">GID = gl(n = gen, k = 1, length = gen*env),</p><p id="p-111">Y = as.vector(wheat.Y))</p><p id="p-112">GB &lt;- tcrossprod(wheat.X)/ncol(wheat.X) #genomic relationship</p><p id="p-113">Kga &lt;- list(G = list(Kernel = GB, Type = “D”),</p><p id="p-114">A = list(Kernel = wheat.A, Type = ”D”))</p><p id="p-115">y &lt;- whe.Y[whe.Y$env == 1, 3]</p><p id="p-116">fit &lt;- BGGE(y = y, K = Kga, ne = 599)</p><p id="p-117">fit$yHat # predicted values</p><p id="p-118">fit$K$G$varu #genetic additive variance (markers)</p><p id="p-119">fit$K$A$varu #genetic additive variance (pedigree)</p><p id="p-120">fit$varE #residual variance</p><p id="p-121">plot(fit$yHat, y)</p></div><p id="p-122">In Box 8, we fit the genomic + pedigree model for environment 1. To do this, we combined the genomic matrix and the pedigree in a list. In the list used as input for BGGE, the type of matrices is assigned as dense. For the BGGE function, since there is only one environment, ne is the number of genotypes evaluated in environment 1.</p></div><div id="sec-18" class="subsection"><h3>Empirical comparisons</h3><p id="p-123">The method applied in BGGE using different features was compared to the standard Bayesian kernel regression proposed by <a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">de los Campos <em>et al.</em> (2010)</a> (BGLR). The comparison of the performance of methods BGGE and BGLR was based on: (i) comparing their variance components, and (ii) comparing the computing time to the time it takes to fit three different genotype × environment models. The posterior variance components were estimated using full data. The computational time was also included in the comparison. The genomic GE models were fitted using BGLR (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Pérez and de los Campos 2014</a>) through the RKHS model and BGGE packages, using a Gibbs sampler with 60,000 iterations, a burn-in of 10,000 and a thinning interval of 10, with 5,000 samples for inference at the end. Kernels for GE models were built into the <em>getK</em> function.</p><p id="p-124">The approach used for prediction includes an orthogonal transformation of the model. Despite the expected theoretical difference between these two approaches, the observed difference was not significant. For the two data sets, the residual variance was slightly lower when using the BGGE approach (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). In contrast, the genetic variance components were high for BGGE. Despite this, there is no clear advantage in using one package instead of the other. However, computational time of the BGGE was up to five times faster than that of the BGLR approach (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). The BGLR package uses approaches to fit generalized linear models and thus fits a wide range of Bayesian regression models like Bayesian LASSO, Bayes A, and Bayes B, among others. The BGGE package specializes in linear mixed models with some features for GE kernels.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490856/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490856/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">- HEL data set. Estimates of variance components obtained by the BGGE and BGLR functions for the multi-environment models, main genotypic effect model (MM), single variance G×E deviation model (MDs) and the environment-specific variance G×E deviation model (MDe) with a G-BLUP kernel</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490802/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490802/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">- Total time (in seconds) to execute the BGGE and BGLR functions for the multi-environment models, main genotypic effect model (MM), single variance G×E deviation model (MDs) and environment-specific variance G×E deviation model (MDe) with the G-BLUP kernel</span><div class="sb-div caption-clear"></div></div></div><p id="p-125">The main mechanisms that increase the speed of the process for fitting the models are: the reparameterization of the model and the way sparse block diagonal matrices are handled. In the context of genomic parametric regression, <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Cuevas <em>et al.</em> (2014)</a> showed that the new parameterization allows reducing the dimensionality; moreover, it gives a computational advantage because it allows simulations with univariate distributions for a smaller number of parameters. The extra features of the sparse structure matrix assumed in the BGGE algorithm reduce dimensionality by decreasing the computational time.</p></div><div id="sec-19" class="subsection"><h3>Conclusions</h3><p id="p-126">The proposed package was built to make genomic predictions for continuous variables focused on genomic GE models. Using information from multi-environment trials can improve prediction, and several models have been created (<a id="xref-ref-28-9" class="xref-bibr" href="#ref-28">Sousa <em>et al.</em> 2017</a>; <a id="xref-ref-9-6" class="xref-bibr" href="#ref-9">Cuevas <em>et al.</em> 2018</a>). However, each GE model has its own properties and, therefore, specific kernels must be created in the BGGE.</p><p id="p-127">The purpose of the <em>getK</em> is to generate kernels for six genomic GE models. Hence, multi-environment kernels are produced using covariance functions created internally (GB or GK). Also, there is an extra argument that allows other kernels to be passed, which opens the possibility of combining different kernels, such as additive with dominance or pedigree, for multi-environment models. For the Gaussian kernel, different values of bandwidth parameters can be introduced to create several kernels, as defined in kernel averaging (<a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">de los Campos <em>et al.</em> 2010</a>). The output produced by <em>getK</em> is in the proper format to be used in the BGGE prediction function.</p><p id="p-128">The BGGE function uses a reparametrization (<a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Cuevas <em>et al.</em> 2014</a>) of the linear mixed model regression in the Bayesian context. These features allow simulations with univariate distributions. We also explored the properties of structured sparsity in some GE kernels to decrease the computational time. Therefore, the package is a fast and efficient option for predicting genetic values. The BGGE was programmed entirely in R and does not have dependencies.</p></div></div><div class="section app" id="app-1"><h2 class="">Appendix</h2><p id="p-129">The conditional posterior distribution of b:<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-95.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-96.gif"/></noscript></span></span>. Assuming that <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-97.gif"/></noscript></span></span>, the genetic effects u can be recovered by <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-98.gif"/></noscript></span></span>.</p><p id="p-130">The conditional distributions of <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-99.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-100.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-101.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-102.gif"/></noscript></span></span> are:<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-103.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-104.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-105.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-106.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9/3039/embed/mml-math-107.gif"/></noscript></span></span> is the proportion of variance that one expects, a priori, to be explained by the regression, with a default value of 0.5.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-131"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-05-17"><span class="received-label">Received </span>May 17, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-07-23"><span class="accepted-label">Accepted </span>July 23, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Granato <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.1" data-doi="10.1214/09-STS307"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Astle</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balding</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Population Structure and Cryptic Relatedness in Genetic Association Studies.</span> <abbr class="cit-jnl-abbrev">Stat. Sci.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">471</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1214/09-STS307</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BSci.%26rft.volume%253D24%26rft.spage%253D451%26rft_id%253Dinfo%253Adoi%252F10.1214%252F09-STS307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/09-STS307&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.2" data-doi="10.1186/s12863-015-0264-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Azevedo</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Resende</span>  <span class="cit-name-given-names">M. D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">F. F. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viana</span>  <span class="cit-name-given-names">J. M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valente</span>  <span class="cit-name-given-names">M. S. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Ridge, Lasso and Bayesian additive-dominance genomic models.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">105</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12863-015-0264-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D16%26rft.spage%253D105%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12863-015-0264-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12863-015-0264-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.3" data-doi="10.2135/cropsci2011.06.0299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">719</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2011.06.0299</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D707%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.4" data-doi="10.1088/0266-5611/24/3/034004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cavalier</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Nonparametric statistical inverse problems.</span> <abbr class="cit-jnl-abbrev">Inverse Probl.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">034004</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1088/0266-5611/24/3/034004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInverse%2BProbl.%26rft.volume%253D24%26rft.spage%253D034004%26rft_id%253Dinfo%253Adoi%252F10.1088%252F0266-5611%252F24%252F3%252F034004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1088/0266-5611/24/3/034004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.5" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.118521</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNzEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzMwMzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.6" data-doi="10.1016/j.tplants.2017.08.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic Selection in Plant Breeding: Methods, Models, and Perspectives.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">961</span>–<span class="cit-lpage">975</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2017.08.011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D22%26rft.spage%253D961%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2017.08.011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2017.08.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.7" data-doi="10.1534/g3.116.035584"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-Lopez</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueno</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> Bayesian Genomic Prediction with Genotype × Environment Interaction Kernel Models. G3: Genes|Genomes|Genetics <span class="cit-vol">7</span>:<span class="cit-fpage">41</span>–<span class="cit-lpage">53</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.035584</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.035584%26rft_id%253Dinfo%253Apmid%252F27793970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjY6IjcvMS80MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOS8zMDM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.8" data-doi="10.3835/plantgenome2016.03.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soberanis</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Elizalde</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic Prediction of Genotype × Environment Interaction Kernel Regression Models.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span> Pant Genome 9: <span class="cit-fpage">1</span>–<span class="cit-lpage">20</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.03.0024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D9%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.03.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.03.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.9" data-doi="10.1534/g3.117.300454"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Granato</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritsche-Neto</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-Lopez</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> Genomic-enabled prediction Kernel models with random intercepts for multi-environment trials. G3-Genes Genom Genet 8: <span class="cit-fpage">1347</span>–<span class="cit-lpage">1365</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.300454</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.300454%26rft_id%253Dinfo%253Apmid%252F29476023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjgvNC8xMzQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzMwMzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.10" data-doi="10.1534/g3.114.013094"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Elizalde</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soberanis</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> Bayesian Genomic-Enabled Prediction as an Inverse Problem. G3:Genes|Genomes|Genetics 4: <span class="cit-fpage">1991</span>–<span class="cit-lpage">2001</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.114.013094</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.013094%26rft_id%253Dinfo%253Apmid%252F25155273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjQvMTAvMTk5MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOS8zMDM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.11" data-doi="10.1017/S0016672310000285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">308</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672310000285</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DDE%2BLos%2BCampos%26rft.auinit1%253DG.%26rft.volume%253D92%26rft.issue%253D4%26rft.spage%253D295%26rft.epage%253D308%26rft.atitle%253DSemi-parametric%2Bgenomic-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Breproducing%2Bkernel%2BHilbert%2Bspaces%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000285%26rft_id%253Dinfo%253Apmid%252F20943010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20943010&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.12" data-doi="10.1534/genetics.109.101501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting quantitative traits with regression models for dense molecular markers and pedigree.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">385</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.101501</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101501%26rft_id%253Dinfo%253Apmid%252F19293140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzMwMzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.13" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge Regression and Other Kernels for Genomic Selection with R Package rrBLUP.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2011.08.0024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.14" data-doi="10.1534/genetics.113.151753"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Priors in whole-genome regression: The Bayesian alphabet returns.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">596</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.151753</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.151753%26rft_id%253Dinfo%253Apmid%252F23636739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzMwMzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.15" data-doi="10.1534/genetics.107.084285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Kaam</span>  <span class="cit-name-given-names">J. B. C. H. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Reproducing kernel Hilbert spaces regression methods for genomic assisted prediction of quantitative traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">2289</span>–<span class="cit-lpage">2303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.084285</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGianola%26rft.auinit1%253DD.%26rft.volume%253D178%26rft.issue%253D4%26rft.spage%253D2289%26rft.epage%253D2303%26rft.atitle%253DReproducing%2BKernel%2BHilbert%2BSpaces%2BRegression%2BMethods%2Bfor%2BGenomic%2BAssisted%2BPrediction%2Bof%2BQuantitative%2BTraits%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.084285%26rft_id%253Dinfo%253Apmid%252F18430950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC80LzIyODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMzAzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.16" data-doi="10.1007/s00122-012-1868-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">González-Camacho</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cairns</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-enabled prediction of genetic values using radial basis function neural networks.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">759</span>–<span class="cit-lpage">771</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-012-1868-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DGonzalez-Camacho%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D125%26rft.issue%253D4%26rft.spage%253D759%26rft.epage%253D771%26rft.atitle%253DGenome-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Bradial%2Bbasis%2Bfunction%2Bneural%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1868-9%26rft_id%253Dinfo%253Apmid%252F22566067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1868-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22566067&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.17" data-doi="10.1534/g3.113.006700"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holand</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinsland</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martino</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> Animal Models and Integrated Nested Laplace Approximations. G3:Genes|Genomes|Genetics 3: <span class="cit-fpage">1241</span>–<span class="cit-lpage">1251</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.113.006700</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.006700%26rft_id%253Dinfo%253Apmid%252F23708299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjMvOC8xMjQxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzMwMzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.18" data-doi="10.1007/s00122-013-2243-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacaze</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du Cheyron</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daucourt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A reaction norm model for genomic selection using high-dimensional genomic and environmental data.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">607</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-013-2243-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D595%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2243-1%26rft_id%253Dinfo%253Apmid%252F24337101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2243-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24337101&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.19" data-doi="10.3168/jds.2009-2061"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A relationship matrix including full pedigree and genomic information.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">4656</span>–<span class="cit-lpage">4663</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2009-2061</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DLegarra%26rft.auinit1%253DA.%26rft.volume%253D92%26rft.issue%253D9%26rft.spage%253D4656%26rft.epage%253D4663%26rft.atitle%253DA%2Brelationship%2Bmatrix%2Bincluding%2Bfull%2Bpedigree%2Band%2Bgenomic%2Binformation%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2009-2061%26rft_id%253Dinfo%253Apmid%252F19700729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2009-2061&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19700729&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269156600059&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.20" data-doi="10.1534/g3.114.016097"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lopez-Cruz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonnett</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> Increased Prediction Accuracy in Wheat Breeding Trials Using a Marker × Environment Interaction Genomic Selection G3:Genes|Genomes|Genetics 5: <span class="cit-fpage">569</span>–<span class="cit-lpage">82</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.114.016097</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.016097%26rft_id%253Dinfo%253Apmid%252F25660166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjUvNC81NjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMzAzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.21" data-doi="10.1007/s00122-015-2622-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mathew</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holand</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koistinen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Léon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sillanpää</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Reparametrization-based estimation of genetic parameters in multi-trait animal model using Integrated Nested Laplace Approximation.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">215</span>–<span class="cit-lpage">225</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-015-2622-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D215%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-015-2622-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-015-2622-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMzAzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.23" data-doi="10.1186/s12711-017-0369-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Indahl</span>  <span class="cit-name-given-names">U. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ødegård</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Variable selection models for genomic selection using whole-genome sequence data and singular value decomposition.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">94</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-017-0369-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D49%26rft.spage%253D94%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-017-0369-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-017-0369-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.24" data-doi="10.1534/g3.117.041202"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luna-Vázquez</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-comment">A Variational Bayes Genomic-Enabled Prediction Model with Genotype × Environment Interaction. G3:Genes|Genomes|Genetics 7: g3.117.041202</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.041202</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.041202%26rft_id%253Dinfo%253Apmid%252F28391241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvNi8xODMzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzMwMzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.25" data-doi="10.1371/journal.pone.0085792"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nishio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Satoh</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Including dominance effects in the genomic BLUP method for genomic evaluation.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e85792</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0085792</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De85792%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0085792%26rft_id%253Dinfo%253Apmid%252F24416447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0085792&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24416447&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.26" data-doi="10.1007/s13253-015-0229-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Elizalde</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Selection of the Bandwidth Parameter in a Bayesian Kernel Regression Model for Genomic-Enabled Prediction.</span> <abbr class="cit-jnl-abbrev">J. Agric. Biol. Environ. Stat.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">512</span>–<span class="cit-lpage">532</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s13253-015-0229-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BBiol.%2BEnviron.%2BStat.%26rft.volume%253D20%26rft.spage%253D512%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs13253-015-0229-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s13253-015-0229-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.27" data-doi="10.1534/genetics.114.164442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide regression &amp; prediction with the BGLR statistical package.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">495</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.164442</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164442%26rft_id%253Dinfo%253Apmid%252F25009151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzMwMzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sousa</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, E. G. de O. Couto, P. Pérez-Rodríguez, D. Jarquín, R. Fritsche-Neto <em>et al</em>., <span class="cit-pub-date">2017</span> Genomic-Enabled Prediction in Maize Using Kernel Models with Genotype × Environment Interaction. G3:Genes|Genomes|Genetics doi: <a href="http://doi.org/10.1534/g3.117.042341">10.1534/g3.117.042341</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.29" data-doi="10.1007/s00122-012-1905-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrag</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of hybrid performance in maize with models incorporating dominance and population specific marker effects.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">1181</span>–<span class="cit-lpage">1194</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-012-1905-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DTechnow%26rft.auinit1%253DF.%26rft.volume%253D125%26rft.issue%253D6%26rft.spage%253D1181%26rft.epage%253D1194%26rft.atitle%253DGenomic%2Bprediction%2Bof%2Bhybrid%2Bperformance%2Bin%2Bmaize%2Bwith%2Bmodels%2Bincorporating%2Bdominance%2Band%2Bpopulation%2Bspecific%2Bmarker%2Beffects.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1905-8%26rft_id%253Dinfo%253Apmid%252F22733443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1905-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22733443&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.30" data-doi="10.1111/jbg.12070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tusell</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forni</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Model averaging for genome-enabled prediction with reproducing kernel Hilbert spaces: A case study with pig litter size and wheat yield.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">115</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jbg.12070</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D131%26rft.spage%253D105%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjbg.12070%26rft_id%253Dinfo%253Apmid%252F24397267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jbg.12070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24397267&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.31" data-doi="10.1186/1471-2164-15-823"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Unterseer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haberer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seidel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knaak</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A powerful tool for genome analysis in maize: development and evaluation of the high density 600 k SNP genotyping array.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">823</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-823</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D823%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-823%26rft_id%253Dinfo%253Apmid%252F25266061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-823&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25266061&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.9.3039.32" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.33" data-doi="10.1371/journal.pone.0114484"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Da</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Quantitative genetics model as the unifying model for defining genomic relationship and inbreeding coefficient.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e114484</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0114484</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De114484%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0114484%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0114484&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.34" data-doi="10.1093/bioinformatics/bts335"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrecht</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auinger</span>  <span class="cit-name-given-names">H.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schön</span>  <span class="cit-name-given-names">C.-C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">synbreed: a framework for the analysis of genomic prediction data using R.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2086</span>–<span class="cit-lpage">2087</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts335</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts335%26rft_id%253Dinfo%253Apmid%252F22689388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts335&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22689388&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.35" data-doi="10.1038/ng.608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benyamin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mcevoy</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henders</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Common SNPs explain a large proportion of the heritability for human height.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">565</span>–<span class="cit-lpage">569</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.608</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D42%26rft.issue%253D7%26rft.spage%253D565%26rft.epage%253D569%26rft.atitle%253DCommon%2BSNPs%2Bexplain%2Ba%2Blarge%2Bproportion%2Bof%2Bthe%2Bheritability%2Bfor%2Bhuman%2Bheight.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.608%26rft_id%253Dinfo%253Apmid%252F20562875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.608&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20562875&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279242400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.3039.36" data-doi="10.1093/bioinformatics/bts606"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levine</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogarten</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laurie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A high-performance computing toolset for relatedness and principal component analysis of SNP data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">3326</span>–<span class="cit-lpage">3328</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts606</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts606%26rft_id%253Dinfo%253Apmid%252F23060615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts606&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23060615&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F3039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312105300026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/9/3039.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/9/3019" title="A Bayesian Decision Theory Approach for Genomic Selection" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/9/3049" title="Tandem Duplicate Genes in Maize Are Abundant and Date to Two Distinct Periods of Time" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/9"><span>Volume 8 Issue 9, September 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/9.cover-source.jpg?itok=77yoxn8o" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/9/3039" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-XWJy8vUrt7JaQme8mprG9VXdeDf4t8NDI9oyu6ARvho" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/9/3039" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/9/3039" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/9/3039" class="active" data-icon-position="" data-hide-link-title="0">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/490727" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ZYg4DZ04A3wlOk8A9Ymw-3MgykaUuEqDYrZRjV1xZWs" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/9/3039.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/9/3039" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-OXc8Fo9tOjEAiN8QEiFyTBoGCE3KnzYmJDSEcqoSAaY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/9/3039" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/9/3039.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727--2" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-2093-6810" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Italo Granato</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-0685-2867" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jaime Cuevas</span>, <span class="highwire-citation-author" data-delta="2">Francisco Luna-Vázquez</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jose Crossa</span>, <span class="highwire-citation-author" data-delta="4">Osval Montesinos-López</span>, <span class="highwire-citation-author" data-delta="5">Juan Burgueño</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-4310-0047" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Roberto Fritsche-Neto</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3039-3047;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200435 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490727--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Italo Granato </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, “Luiz de Queiroz” College of Agriculture, University of São Paulo, Piracicaba, São Paulo, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Italo%2BGranato%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Granato%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AItalo%2BGranato%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2093-6810" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Italo Granato</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jaime Cuevas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Universidad de Quintana Roo, Chetumal, Quintana Roo, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaime%2BCuevas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cuevas%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJaime%2BCuevas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0685-2867" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jaime Cuevas</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jaicueva@uqroo.edu.mx" class="" data-icon-position="" data-hide-link-title="0">jaicueva@uqroo.edu.mx</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Francisco Luna-Vázquez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Facultad de Telemática, Universidad de Colima, Colima, Colima, 28040, México,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Francisco%2BLuna-V%C3%A1zquez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Luna-V%C3%A1zquez%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFrancisco%2BLuna-V%25C3%25A1zquez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jose Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, México DF, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jose%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJose%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jose Crossa</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Osval Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Facultad de Telemática, Universidad de Colima, Colima, Colima, 28040, México,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, México DF, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Roberto Fritsche-Neto </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, “Luiz de Queiroz” College of Agriculture, University of São Paulo, Piracicaba, São Paulo, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roberto%2BFritsche-Neto%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fritsche-Neto%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoberto%2BFritsche-Neto%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4310-0047" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Roberto Fritsche-Neto</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/9/3039" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/9/3039&amp;title=BGGE%3A%20A%20New%20Package%20for%20Genomic-Enabled%20Prediction%20Incorporating%20Genotype%20%C3%97%20Environment%20Interaction%20Models" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/9/3039&amp;title=BGGE%3A%20A%20New%20Package%20for%20Genomic-Enabled%20Prediction%20Incorporating%20Genotype%20%C3%97%20Environment%20Interaction%20Models" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/9/3039&amp;title=BGGE%3A%20A%20New%20Package%20for%20Genomic-Enabled%20Prediction%20Incorporating%20Genotype%20%C3%97%20Environment%20Interaction%20Models" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/9/3039&amp;text=BGGE%3A%20A%20New%20Package%20for%20Genomic-Enabled%20Prediction%20Incorporating%20Genotype%20%C3%97%20Environment%20Interaction%20Models" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/9/3039&amp;title=BGGE%3A%20A%20New%20Package%20for%20Genomic-Enabled%20Prediction%20Incorporating%20Genotype%20%C3%97%20Environment%20Interaction%20Models" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/9/3039&amp;t=BGGE%3A%20A%20New%20Package%20for%20Genomic-Enabled%20Prediction%20Incorporating%20Genotype%20%C3%97%20Environment%20Interaction%20Models" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/9/3039&amp;title=BGGE%3A%20A%20New%20Package%20for%20Genomic-Enabled%20Prediction%20Incorporating%20Genotype%20%C3%97%20Environment%20Interaction%20Models" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/9/3039&amp;title=BGGE%3A%20A%20New%20Package%20for%20Genomic-Enabled%20Prediction%20Incorporating%20Genotype%20%C3%97%20Environment%20Interaction%20Models" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/490727/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727--3" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-2093-6810" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Italo Granato</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-0685-2867" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jaime Cuevas</span>, <span class="highwire-citation-author" data-delta="2">Francisco Luna-Vázquez</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jose Crossa</span>, <span class="highwire-citation-author" data-delta="4">Osval Montesinos-López</span>, <span class="highwire-citation-author" data-delta="5">Juan Burgueño</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-4310-0047" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Roberto Fritsche-Neto</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3039-3047;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200435 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490727--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Italo Granato </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, “Luiz de Queiroz” College of Agriculture, University of São Paulo, Piracicaba, São Paulo, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Italo%2BGranato%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Granato%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AItalo%2BGranato%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2093-6810" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Italo Granato</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jaime Cuevas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Universidad de Quintana Roo, Chetumal, Quintana Roo, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaime%2BCuevas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cuevas%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJaime%2BCuevas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0685-2867" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jaime Cuevas</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jaicueva@uqroo.edu.mx" class="" data-icon-position="" data-hide-link-title="0">jaicueva@uqroo.edu.mx</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Francisco Luna-Vázquez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Facultad de Telemática, Universidad de Colima, Colima, Colima, 28040, México,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Francisco%2BLuna-V%C3%A1zquez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Luna-V%C3%A1zquez%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFrancisco%2BLuna-V%25C3%25A1zquez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jose Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, México DF, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jose%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJose%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jose Crossa</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Osval Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Facultad de Telemática, Universidad de Colima, Colima, Colima, 28040, México,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, México DF, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Roberto Fritsche-Neto </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, “Luiz de Queiroz” College of Agriculture, University of São Paulo, Piracicaba, São Paulo, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roberto%2BFritsche-Neto%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fritsche-Neto%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoberto%2BFritsche-Neto%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4310-0047" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Roberto Fritsche-Neto</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3039&amp;rft.epage=3047&amp;rft.atitle=BGGE%3A%20A%20New%20Package%20for%20Genomic-Enabled%20Prediction%20Incorporating%20Genotype%20%C3%97%20Environment%20Interaction%20Models&amp;rft.volume=8&amp;rft.issue=9&amp;rft.date=2018-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Granato%2C+Italo&amp;rft.au=Cuevas%2C+Jaime&amp;rft.au=Luna-V%C3%A1zquez%2C+Francisco&amp;rft.au=Crossa%2C+Jose&amp;rft.au=Montesinos-L%C3%B3pez%2C+Osval&amp;rft.au=Burgue%C3%B1o%2C+Juan&amp;rft.au=Fritsche-Neto%2C+Roberto"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/490727/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/490727/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/490727/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/490727/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/490727/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/490727/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/490727/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/490727/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/490727/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/490727/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/490727/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/490727/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="last even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/9/3039" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Statistical Models</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Experimental Data Set</a></li><li class="even"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Description and Application of the Bgge Package</a></li><li class="odd"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/9/3039.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/9/3039.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/8/9/3039.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__CK8qNg7zHnCPfipcSU0qjCyIiw2pe-WKGx2V0TgizGw__wprDSLLH9CuacxJkxX2DuWt0EaauZA-f_LGjovfY3aw__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
