<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/9/2953" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/9/2953.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/9/2953.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/9/2953.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/9/2953.atom" />
<meta name="HW.pisa" content="ggg;8/9/2953" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Complex Haplotypes of GSTM1 Gene Deletions Harbor Signatures of a Selective Sweep in East Asian Populations" />
<meta name="DC.Identifier" content="10.1534/g3.118.200462" />
<meta name="DC.Date" content="2018-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 SAITOU et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.6384782" />
<meta name="DC.Description" content="The deletion of the metabolizing Glutathione S-transferase Mu 1 ( GSTM1 ) gene has been associated with multiple cancers, metabolic and autoimmune disorders, as well as drug response. It is unusually common, with allele frequency reaching up to 75% in some human populations. Such high allele frequency of a derived allele with apparent impact on an otherwise conserved gene is a rare phenomenon. To investigate the evolutionary history of this locus, we analyzed 310 genomes using population genetics tools. Our analysis revealed a surprising lack of linkage disequilibrium between the deletion and the flanking single nucleotide variants in this locus. Tests that measure extended homozygosity and rapid change in allele frequency revealed signatures of an incomplete sweep in the locus. Using empirical approaches, we identified the Tanuki haplogroup, which carries the GSTM1 deletion and is found in approximately 70% of East Asian chromosomes. This haplogroup has rapidly increased in frequency in East Asian populations, contributing to a high population differentiation among continental human groups. We showed that extended homozygosity and population differentiation for this haplogroup is incompatible with simulated neutral expectations in East Asian populations. In parallel, we found that the Tanuki haplogroup is significantly associated with the expression levels of other GSTM genes. Collectively, our results suggest that standing variation in this locus has likely undergone an incomplete sweep in East Asia with regulatory impact on multiple GSTM genes. Our study provides the necessary framework for further studies to elucidate the evolutionary reasons that maintain disease-susceptibility variants in the GSTM1 locus." />
<meta name="DC.Contributor" content="M. Saitou" />
<meta name="DC.Contributor" content="Y. Satta" />
<meta name="DC.Contributor" content="O. Gokcumen" />
<meta name="article:published_time" content="2018-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Complex Haplotypes of GSTM1 Gene Deletions Harbor Signatures of a Selective Sweep in East Asian Populations" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The deletion of the metabolizing Glutathione S-transferase Mu 1 (&lt;i&gt;GSTM1&lt;/i&gt;) gene has been associated with multiple cancers, metabolic and autoimmune disorders, as well as drug response. It is unusually common, with allele frequency reaching up to 75% in some human populations. Such high allele frequency of a derived allele with apparent impact on an otherwise conserved gene is a rare phenomenon. To investigate the evolutionary history of this locus, we analyzed 310 genomes using population genetics tools. Our analysis revealed a surprising lack of linkage disequilibrium between the deletion and the flanking single nucleotide variants in this locus. Tests that measure extended homozygosity and rapid change in allele frequency revealed signatures of an incomplete sweep in the locus. Using empirical approaches, we identified the &lt;i&gt;Tanuki&lt;/i&gt; haplogroup, which carries the &lt;i&gt;GSTM1&lt;/i&gt; deletion and is found in approximately 70% of East Asian chromosomes. This haplogroup has rapidly increased in frequency in East Asian populations, contributing to a high population differentiation among continental human groups. We showed that extended homozygosity and population differentiation for this haplogroup is incompatible with simulated neutral expectations in East Asian populations. In parallel, we found that the &lt;i&gt;Tanuki&lt;/i&gt; haplogroup is significantly associated with the expression levels of other &lt;i&gt;GSTM&lt;/i&gt; genes. Collectively, our results suggest that standing variation in this locus has likely undergone an incomplete sweep in East Asia with regulatory impact on multiple &lt;i&gt;GSTM&lt;/i&gt; genes. Our study provides the necessary framework for further studies to elucidate the evolutionary reasons that maintain disease-susceptibility variants in the &lt;i&gt;GSTM1&lt;/i&gt; locus.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/09/01" />
<meta name="citation_mjid" content="ggg;8/9/2953" />
<meta name="citation_id" content="8/9/2953" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/9/2953" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/9/2953.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/9/2953.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/9/2953.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200462" />
<meta name="citation_pmid" content="30061374" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2953" />
<meta name="citation_lastpage" content="2966" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="M. Saitou" />
<meta name="citation_author_institution" content="Dept. of Biological Sciences, SUNY at Buffalo, NY, 14260-1300" />
<meta name="citation_author_institution" content="Dept. of Biological Sciences, The University of Tokyo, Tokyo, Japan, 113-0033" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8794-3217" />
<meta name="citation_author" content="Y. Satta" />
<meta name="citation_author_institution" content="Dept. of Evolutionary Studies of Biosystems, SOKENDAI (Graduate University for Advanced Studies), Kanagawa, Japan 240-0193" />
<meta name="citation_author" content="O. Gokcumen" />
<meta name="citation_author_institution" content="Dept. of Biological Sciences, SUNY at Buffalo, NY, 14260-1300" />
<meta name="citation_author_email" content="omergokc@buffalo.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4371-679X" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. R. Abecasis;citation_author=D. Altshuler;citation_author=A. Auton;citation_author=L. D. Brooks;citation_title=A map of human genome variation from population-scale sequencing.;citation_pages=1061-1073;citation_volume=467;citation_year=2010;citation_issue=7319;citation_pmid=20981092;citation_doi=10.1038/nature09534" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. R. Abecasis;citation_author=A. Auton;citation_author=L. D. Brooks;citation_title=An integrated map of genetic variation from 1,092 human genomes.;citation_pages=56-65;citation_volume=491;citation_year=2012;citation_issue=7422;citation_pmid=23128226;citation_doi=10.1038/nature11632" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=P. Bhattacharjee;citation_author=S. Paul;citation_author=M. Banerjee;citation_author=D. Patra;citation_author=P. Banerjee;citation_title=Functional compensation of glutathione S-transferase M1 (GSTM1) null by another GST superfamily member, GSTM2.;citation_pages=2704;citation_volume=3;citation_year=2013;citation_pmid=24048194;citation_doi=10.1038/srep02704" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=L. M. Boettger;citation_author=R. M. Salem;citation_author=R. E. Handsaker;citation_author=G. M. Peloso;citation_author=S. Kathiresan;citation_title=Recurring exon deletions in the HP (haptoglobin) gene contribute to lower blood cholesterol levels.;citation_pages=359-366;citation_volume=48;citation_year=2016;citation_doi=10.1038/ng.3510" />
<meta name="citation_reference" content="citation_journal_title=J. Mol. Diagn.;citation_author=A. Buchard;citation_author=J. J. Sanchez;citation_author=K. Dalhoff;citation_author=N. Morling;citation_title=Multiplex PCR detection of GSTM1, GSTT1, and GSTP1 gene variants: simultaneously detecting GSTM1 and GSTT1 gene copy number and the allelic status of the GSTP1 Ile105Val genetic variant.;citation_volume=9;citation_year=2007;citation_pmid=17916600;citation_doi=10.2353/jmoldx.2007.070030" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=H. Chen;citation_author=N. Patterson;citation_author=D. Reich;citation_title=Population differentiation as a test for selective sweeps.;citation_volume=20;citation_year=2010;citation_pmid=20086244;citation_doi=10.1101/gr.100545.109" />
<meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.;citation_author=M. Claussnitzer;citation_author=S. N. Dankel;citation_author=K.-H. Kim;citation_author=G. Quon;citation_author=W. Meuleman;citation_title=FTO Obesity Variant Circuitry and Adipocyte Browning in Humans.;citation_pages=895-907;citation_volume=373;citation_year=2015;citation_pmid=26287746;citation_doi=10.1056/NEJMoa1502214" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. F. Conrad;citation_author=D. Pinto;citation_author=R. Redon;citation_author=L. Feuk;citation_author=O. Gokcumen;citation_title=Origins and functional impact of copy number variation in the human genome.;citation_pages=704-712;citation_volume=464;citation_year=2010;citation_issue=7289;citation_pmid=19812545;citation_doi=10.1038/nature08516" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=J. L. Crisci;citation_author=Y.-P. Poh;citation_author=S. Mahajan;citation_author=J. D. Jensen;citation_title=The impact of equilibrium assumptions on tests of selection.;citation_pages=235;citation_volume=4;citation_year=2013;citation_pmid=24273554;citation_doi=10.3389/fgene.2013.00235" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Arteriosclerosis, Thrombosis, and Vascular Biology;citation_journal_abbrev=Arterioscler. Thromb. Vasc. Bio.;citation_author=E. Ricciotti;citation_author=G. A. FitzGerald;citation_title=Prostaglandins and Inflammation;citation_pages=986-1000;citation_volume=31;citation_year=2011;citation_issue=5;citation_pmid=21508345;citation_doi=10.1161/ATVBAHA.110.207449" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=L. Feuk;citation_author=A. R. Carson;citation_author=S. W. Scherer;citation_title=Structural variation in the human genome.;citation_pages=85-97;citation_volume=7;citation_year=2006;citation_issue=2;citation_pmid=16418744;citation_doi=10.1038/nrg1767" />
<meta name="citation_reference" content="citation_journal_title=Leg. Med. (Tokyo);citation_author=J. Fujihara;citation_author=T. Yasuda;citation_author=R. Iida;citation_author=H. Takatsuka;citation_author=Y. Fujii;citation_title=Cytochrome P450 1A1, glutathione S-transferases M1 and T1 polymorphisms in Ovambos and Mongolians.;citation_pages=S408-S410;citation_volume=11;citation_year=2009;citation_doi=10.1016/j.legalmed.2009.01.073" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Fumagalli;citation_author=I. Moltke;citation_author=N. Grarup;citation_author=F. Racimo;citation_author=P. Bjerregaard;citation_title=Greenlandic Inuit show genetic signatures of diet and climate adaptation;citation_pages=1343-1347;citation_volume=349;citation_year=2015;citation_issue=6254;citation_pmid=26383953;citation_doi=10.1126/science.aab2319" />
<meta name="citation_reference" content="citation_journal_title=Lancet;citation_journal_abbrev=Lancet;citation_author=M. García-Closas;citation_author=N. Malats;citation_author=D. Silverman;citation_author=M. Dosemeci;citation_author=M. Kogevinas;citation_title=NAT2 slow acetylation, GSTM1 null genotype, and risk of bladder cancer: results from the Spanish Bladder Cancer Study and meta-analyses.;citation_pages=649-659;citation_volume=366;citation_year=2005;citation_issue=9486;citation_pmid=16112301;citation_doi=10.1016/S0140-6736(05)67137-1" />
<meta name="citation_reference" content="citation_journal_title=Cancer Epidemiology Biomarkers &amp; Prevention;citation_journal_abbrev=Cancer Epidemiol. Biomarkers Prev.;citation_author=S. Garte;citation_author=L. Gaspari;citation_author=A.-K. Alexandrie;citation_author=C. Ambrosone;citation_author=H. Autrup;citation_title=Metabolic Gene Polymorphism Frequencies in Control Populations;citation_pages=1239-1248;citation_volume=10;citation_year=2001;citation_issue=12;citation_pmid=11751440" />
<meta name="citation_reference" content="citation_journal_title=American Journal of Physical Anthropology;citation_journal_abbrev=American Journal of Physical Anthropology;citation_author=P. Gaspar;citation_author=M. H. Hutz;citation_author=F. M. Salzano;citation_author=K. Hill;citation_author=M. Hurtado;citation_title=Polymorphisms of CYP1a1, CYP2e1, GSTM1, GSTT1, and TP53 genes in Amerindians.;citation_pages=249-256;citation_volume=119;citation_year=2002;citation_issue=3;citation_pmid=12365037;citation_doi=10.1002/ajpa.10128" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=S. Girirajan;citation_author=C. D. Campbell;citation_author=E. E. Eichler;citation_title=Human copy number variation and complex genetic disease.;citation_pages=203-226;citation_volume=45;citation_year=2011;citation_pmid=21854229;citation_doi=10.1146/annurev-genet-102209-163544" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=P. J. Hastings;citation_author=J. R. Lupski;citation_author=S. M. Rosenberg;citation_author=G. Ira;citation_title=Mechanisms of change in gene copy number.;citation_pages=551-564;citation_volume=10;citation_year=2009;citation_issue=8;citation_pmid=19597530;citation_doi=10.1038/nrg2593" />
<meta name="citation_reference" content="citation_journal_title=Annual review of pharmacology;citation_journal_abbrev=Annu Rev Pharmacol;citation_author=J. D. Hayes;citation_author=J. U. Flanagan;citation_author=I. R. Jowsey;citation_title=Glutathione transferases.;citation_pages=51-88;citation_volume=45;citation_year=2005;citation_pmid=15822171;citation_doi=10.1146/annurev.pharmtox.45.120403.095857" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. D. Hernandez;citation_author=J. L. Kelley;citation_author=E. Elyashiv;citation_author=S. C. Melton;citation_author=A. Auton;citation_title=Classic Selective Sweeps Were Rare in Recent Human Evolution;citation_pages=920-924;citation_volume=331;citation_year=2011;citation_issue=6019;citation_pmid=21330547;citation_doi=10.1126/science.1198878" />
<meta name="citation_reference" content="citation_journal_title=Genomics Inform.;citation_author=E. P. Hong;citation_author=J. W. Park;citation_title=Sample size and statistical power calculation in genetic association studies.;citation_pages=117-122;citation_volume=10;citation_year=2012;citation_pmid=23105939;citation_doi=10.5808/GI.2012.10.2.117" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=R. R. Hudson;citation_title=Generating samples under a Wright-Fisher neutral model of  genetic variation;citation_pages=337-338;citation_volume=18;citation_year=2002;citation_issue=2;citation_pmid=11847089;citation_doi=10.1093/bioinformatics/18.2.337" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. H. Huson;citation_author=D. C. Richter;citation_author=C. Rausch;citation_author=T. Dezulian;citation_author=M. Franz;citation_title=Dendroscope: An interactive viewer for large phylogenetic trees.;citation_pages=460-460;citation_volume=8;citation_year=2007;citation_pmid=18034891;citation_doi=10.1186/1471-2105-8-460" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. A. Frazer;citation_author=D. G. Ballinger;citation_author=D. R. Cox;citation_author=D. A. Hinds;citation_title=A second generation human haplotype map of over 3.1 million SNPs.;citation_pages=851-861;citation_volume=449;citation_year=2007;citation_issue=7164;citation_pmid=17943122;citation_doi=10.1038/nature06258" />
<meta name="citation_reference" content="citation_journal_title=Genomics Proteomics Bioinformatics;citation_author=L. Jin;citation_author=X.-Y. Zuo;citation_author=W.-Y. Su;citation_author=X.-L. Zhao;citation_author=M.-Q. Yuan;citation_title=Pathway-based analysis tools for complex diseases: a review.;citation_pages=210-220;citation_volume=12;citation_year=2014;citation_pmid=25462153;citation_doi=10.1016/j.gpb.2014.10.002" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cytogenet.;citation_author=A. V. Khrunin;citation_author=I. N. Filippova;citation_author=A. M. Aliev;citation_author=T. V. Tupitsina;citation_author=P. A. Slominsky;citation_title=GSTM1 copy number variation in the context of single nucleotide polymorphisms in the human GSTM cluster.;citation_pages=30;citation_volume=9;citation_year=2016;citation_doi=10.1186/s13039-016-0241-0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. L. Kim;citation_author=Y. Satta;citation_title=Population genetic analysis of the N-acylsphingosine amidohydrolase gene associated with mental activity in humans.;citation_volume=178;citation_year=2008;citation_pmid=18245333;citation_doi=10.1534/genetics.107.083691" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Kimura;citation_author=T. Ohta;citation_title=THE AGE OF A NEUTRAL MUTANT PERSISTING IN A FINITE POPULATION;citation_pages=199-212;citation_volume=75;citation_year=1973;citation_issue=1;citation_pmid=4762875" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=M. Kircher;citation_author=D. M. Witten;citation_author=P. Jain;citation_author=B. J. O’Roak;citation_author=G. M. Cooper;citation_title=A general framework for estimating the relative pathogenicity of human genetic variants.;citation_pages=310-315;citation_volume=46;citation_year=2014;citation_pmid=24487276;citation_doi=10.1038/ng.2892" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Kong;citation_author=G. Thorleifsson;citation_author=D. F. Gudbjartsson;citation_author=G. Masson;citation_author=A. Sigurdsson;citation_title=Fine-scale recombination rate differences between sexes, populations and individuals.;citation_pages=1099-1103;citation_volume=467;citation_year=2010;citation_issue=7319;citation_pmid=20981099;citation_doi=10.1038/nature09525" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=E. M. Leffler;citation_author=G. Band;citation_author=G. B. J. Busby;citation_author=K. Kivinen;citation_author=Q. S. Le;citation_title=Resistance to malaria through structural variation of red blood cell invasion receptors.;citation_volume=356;citation_year=2017;citation_pmid=28522690;citation_doi=10.1126/science.aam6393" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=M. Lek;citation_author=K. J. Karczewski;citation_author=E. V. Minikel;citation_author=K. E. Samocha;citation_author=E. Banks;citation_title=Analysis of protein-coding genetic variation in 60,706 humans.;citation_pages=285-291;citation_volume=536;citation_year=2016;citation_pmid=27535533;citation_doi=10.1038/nature19057" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Librado;citation_author=J. Rozas;citation_title=DnaSP v5: A software for comprehensive analysis of DNA polymorphism data.;citation_volume=25;citation_year=2009;citation_pmid=19346325;citation_doi=10.1093/bioinformatics/btp187" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Y. L. Lin;citation_author=P. Pavlidis;citation_author=E. Karakoc;citation_author=J. Ajay;citation_author=O. Gokcumen;citation_title=The evolution and functional impact of human deletion variants shared with archaic hominin genomes.;citation_volume=32;citation_year=2015;citation_pmid=25556237;citation_doi=10.1093/molbev/msu405" />
<meta name="citation_reference" content="citation_journal_title=Journal of Medical Genetics;citation_journal_abbrev=J. Med. Genet.;citation_author=P.-Y. Liu;citation_author=Y.-Y. Zhang;citation_author=Y. Lu;citation_author=J.-R. Long;citation_author=H. Shen;citation_title=A survey of haplotype variants at several disease candidate genes: the importance of rare variants for complex diseases;citation_pages=221-227;citation_volume=42;citation_year=2005;citation_issue=3;citation_pmid=15744035;citation_doi=10.1136/jmg.2004.024752" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=J. Lonsdale;citation_author=J. Thomas;citation_author=M. Salvatore;citation_author=R. Phillips;citation_author=E. Lo;citation_title=The Genotype-Tissue Expression (GTEx) project.;citation_pages=580-585;citation_volume=45;citation_year=2013;citation_pmid=23715323;citation_doi=10.1038/ng.2653" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. MacArthur;citation_author=E. Bowler;citation_author=M. Cerezo;citation_author=L. Gil;citation_author=P. Hall;citation_title=The new NHGRI-EBI Catalog of published genome-wide association studies (GWAS Catalog).;citation_volume=45;citation_year=2017;citation_pmid=27899670;citation_doi=10.1093/nar/gkw1133" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. Marques-Bonet;citation_author=J. M. Kidd;citation_author=M. Ventura;citation_author=T. A. Graves;citation_author=Z. Cheng;citation_title=A burst of segmental duplications in the genome of the African great ape ancestor.;citation_pages=877-881;citation_volume=457;citation_year=2009;citation_issue=7231;citation_pmid=19212409;citation_doi=10.1038/nature07744" />
<meta name="citation_reference" content="citation_journal_title=Oncogene;citation_journal_abbrev=Oncogene;citation_author=C. C. McIlwain;citation_author=D. M. Townsend;citation_author=K. D. Tew;citation_title=Glutathione S-transferase polymorphisms: cancer incidence and therapy.;citation_pages=1639-1648;citation_volume=25;citation_year=2006;citation_issue=11;citation_pmid=16550164;citation_doi=10.1038/sj.onc.1209373" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. R. Abecasis;citation_author=A. Auton;citation_author=L. D. Brooks;citation_author=M. A. DePristo;citation_title=An integrated map of genetic variation from 1,092 human genomes.;citation_pages=56-65;citation_volume=491;citation_year=2012;citation_issue=7422;citation_pmid=23128226;citation_doi=10.1038/nature11632" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=G. A. T. Mc Vean;citation_author=S. R. Myers;citation_author=S. Hunt;citation_author=P. Deloukas;citation_author=D. R. Bentley;citation_title=The Fine-Scale Structure of Recombination Rate Variation in the Human Genome;citation_pages=581-584;citation_volume=304;citation_year=2004;citation_issue=5670;citation_pmid=15105499;citation_doi=10.1126/science.1092500" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. Moorjani;citation_author=C. E. G. Amorim;citation_author=P. F. Arndt;citation_author=M. Przeworski;citation_title=Variation in the molecular clock of primates.;citation_volume=113;citation_year=2016;citation_pmid=27601674;citation_doi=10.1073/pnas.1600374113" />
<meta name="citation_reference" content="citation_journal_title=Pharmacogenomics;citation_journal_abbrev=Pharmacogenomics;citation_author=H. M. Mortensen;citation_author=A. Froment;citation_author=G. Lema;citation_author=J.-M. Bodo;citation_author=M. Ibrahim;citation_title=Characterization of genetic variation and natural selection at the arylamine N-acetyltransferase genes in global human populations.;citation_pages=1545-1558;citation_volume=12;citation_year=2011;citation_issue=11;citation_pmid=21995608;citation_doi=10.2217/pgs.11.88" />
<meta name="citation_reference" content="citation_journal_title=Clinical Cancer Research;citation_journal_abbrev=Clin. Cancer Res.;citation_author=A. M. Moyer;citation_author=O. E. Salavaggione;citation_author=S. J. Hebbring;citation_author=I. Moon;citation_author=M. T. Hildebrandt;citation_title=Glutathione S-Transferase T1 and M1: Gene Sequence Variation and Functional Genomics;citation_pages=7207-7216;citation_volume=13;citation_year=2007;citation_issue=23;citation_pmid=18056202;citation_doi=10.1158/1078-0432.CCR-07-0635" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=V. M. Narasimhan;citation_author=R. Rahbari;citation_author=A. Scally;citation_author=A. Wuster;citation_author=D. Mason;citation_title=Estimating the human mutation rate from autozygous segments reveals population differences in human mutational processes.;citation_pages=303;citation_volume=8;citation_year=2017;citation_doi=10.1038/s41467-017-00323-y" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. Nei;citation_author=W. H. Li;citation_title=Mathematical model for studying genetic variation in terms of restriction endonucleases.;citation_pages=5269-5273;citation_volume=76;citation_year=1979;citation_issue=10;citation_pmid=291943;citation_doi=10.1073/pnas.76.10.5269" />
<meta name="citation_reference" content="citation_journal_title=BMC Evol. Biol.;citation_author=P. Pajic;citation_author=Y.-L. Lin;citation_author=D. Xu;citation_author=O. Gokcumen;citation_title=The psoriasis associated deletion of late cornified envelope genes and has been maintained under balancing selection since Human Denisovan divergence.;citation_pages=1-7;citation_volume=16;citation_year=2016;citation_pmid=26727998;citation_doi=10.1186/s12862-015-0575-y" />
<meta name="citation_reference" content="citation_journal_title=Cancer letters;citation_journal_abbrev=Cancer Lett;citation_author=F. F. Parl;citation_title=Glutathione S-transferase genotypes and cancer risk.;citation_pages=123-129;citation_volume=221;citation_year=2005;citation_issue=2;citation_pmid=15808397;citation_doi=10.1016/j.canlet.2004.06.016" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=G. H. Perry;citation_author=N. J. Dominy;citation_author=K. G. Claw;citation_author=A. S. Lee;citation_author=H. Fiegler;citation_title=Diet and the evolution of human amylase gene copy number variation.;citation_pages=1256-1260;citation_volume=39;citation_year=2007;citation_issue=10;citation_pmid=17828263;citation_doi=10.1038/ng2123" />
<meta name="citation_reference" content="citation_journal_title=Molecular biology reports;citation_journal_abbrev=Mol Biol Rep;citation_author=S. Piacentini;citation_author=R. Polimanti;citation_author=F. Porreca;citation_author=C. Martínez-Labarga;citation_author=G. F. De Stefano;citation_title=GSTT1 and GSTM1 gene polymorphisms in European and African populations.;citation_pages=1225-1230;citation_volume=38;citation_year=2011;citation_issue=2;citation_pmid=20563854;citation_doi=10.1007/s11033-010-0221-0" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. Prüfer;citation_author=F. Racimo;citation_author=N. Patterson;citation_author=F. Jay;citation_author=S. Sankararaman;citation_title=The complete genome sequence of a Neanderthal from the Altai Mountains;citation_pages=43-49;citation_volume=505;citation_year=2014;citation_issue=7481;citation_pmid=24352235;citation_doi=10.1038/nature12886;citation_georef=2014009693" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Pybus;citation_author=G. M. Dall’Olio;citation_author=P. Luisi;citation_author=M. Uzkudun;citation_author=A. Carreño-Torres;citation_title=1000 Genomes Selection Browser 1.0: a genome browser dedicated to signatures of natural selection in modern humans.;citation_volume=42;citation_year=2014;citation_pmid=24275494;citation_doi=10.1093/nar/gkt1188" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=F. Racimo;citation_author=S. Sankararaman;citation_author=R. Nielsen;citation_author=E. Huerta-Sánchez;citation_title=Evidence for archaic adaptive introgression in humans.;citation_pages=359-371;citation_volume=16;citation_year=2015;citation_pmid=25963373;citation_doi=10.1038/nrg3936" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. Redon;citation_author=S. Ishikawa;citation_author=K. R. Fitch;citation_author=L. Feuk;citation_author=G. H. Perry;citation_title=Global variation in copy number in the human genome.;citation_pages=444-454;citation_volume=444;citation_year=2006;citation_issue=7118;citation_pmid=17122850;citation_doi=10.1038/nature05329" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. Reich;citation_author=R. E. Green;citation_author=M. Kircher;citation_author=J. Krause;citation_author=N. Patterson;citation_title=Genetic history of an archaic hominin group from Denisova Cave in Siberia.;citation_pages=1053-1060;citation_volume=468;citation_year=2010;citation_issue=7327;citation_pmid=21179161;citation_doi=10.1038/nature09710;citation_georef=2011015829" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=N. A. Rosenberg;citation_author=L. Huang;citation_author=E. M. Jewett;citation_author=Z. A. Szpiech;citation_author=I. Jankovic;citation_title=Genome-wide association studies in diverse populations.;citation_pages=356-366;citation_volume=11;citation_year=2010;citation_issue=5;citation_pmid=20395969;citation_doi=10.1038/nrg2760" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=N. Rothman;citation_author=M. Garcia-Closas;citation_author=N. Chatterjee;citation_author=N. Malats;citation_author=X. Wu;citation_title=A multi-stage genome-wide association study of bladder cancer identifies multiple susceptibility loci.;citation_pages=978-984;citation_volume=42;citation_year=2010;citation_issue=11;citation_pmid=20972438;citation_doi=10.1038/ng.687" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. Ruwende;citation_author=S. C. Khoo;citation_author=R. W. Snow;citation_author=S. N. Yates;citation_author=D. Kwiatkowski;citation_title=Natural selection of hemi- and heterozygotes for G6PD deficiency in Africa by resistance to severe malaria.;citation_pages=246-249;citation_volume=376;citation_year=1995;citation_issue=6537;citation_pmid=7617034;citation_doi=10.1038/376246a0" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Cancer Res.;citation_author=M. Saadat;citation_title=GSTM1 null genotype associated with age-standardized cancer mortality rate in 45 countries from five continents: an ecologic study.;citation_pages=74-91;citation_volume=3;citation_year=2007;citation_doi=10.3923/ijcr.2007.74.91" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. C. Sabeti;citation_author=P. Varilly;citation_author=B. Fry;citation_author=J. Lohmueller;citation_author=E. Hostetter;citation_title=Genome-wide detection and characterization of positive selection in human populations.;citation_pages=913-918;citation_volume=449;citation_year=2007;citation_issue=7164;citation_pmid=17943131;citation_doi=10.1038/nature06250" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=M. Saitou;citation_author=Y. Satta;citation_author=O. Gokcumen;citation_author=T. Ishida;citation_title=Complex evolution of the GSTM gene family involves sharing of GSTM1 deletion polymorphism in humans and chimpanzees.;citation_pages=293;citation_volume=19;citation_year=2018;citation_doi=10.1186/s12864-018-4676-z" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=J. Salojärvi;citation_author=O.-P. Smolander;citation_author=K. Nieminen;citation_author=S. Rajaraman;citation_author=O. Safronov;citation_title=Genome sequencing and population genomic analyses provide insights into the adaptive landscape of silver birch.;citation_pages=904-912;citation_volume=49;citation_year=2017;citation_doi=10.1038/ng.3862" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=S. F. Schaffner;citation_author=C. Foo;citation_author=S. Gabriel;citation_author=D. Reich;citation_author=M. J. Daly;citation_title=Calibrating a coalescent simulation of human genome sequence variation;citation_pages=1576-1583;citation_volume=15;citation_year=2005;citation_issue=11;citation_pmid=16251467;citation_doi=10.1101/gr.3709305" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=C. M. Schlebusch;citation_author=L. M. Gattepaille;citation_author=K. Engström;citation_author=M. Vahter;citation_author=M. Jakobsson;citation_title=Human Adaptation to Arsenic-Rich Environments.;citation_volume=32;citation_year=2015;citation_pmid=25739736;citation_doi=10.1093/molbev/msv046" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=D. R. Schrider;citation_author=A. D. Kern;citation_title=Soft Sweeps Are the Dominant Mode of Adaptation in the Human Genome.;citation_pages=1863-1877;citation_volume=34;citation_year=2017;citation_doi=10.1093/molbev/msx154" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=A. Sekar;citation_author=A. R. Bialas;citation_author=H. de Rivera;citation_author=A. Davis;citation_author=T. R. Hammond;citation_title=Schizophrenia risk from complex variation of complement component 4.;citation_pages=177-183;citation_volume=530;citation_year=2016;citation_pmid=26814963;citation_doi=10.1038/nature16549" />
<meta name="citation_reference" content="citation_journal_title=Fly (Austin);citation_author=S. L. Sheeley;citation_author=B. F. McAllister;citation_title=Patterns of natural selection at the Alcohol dehydrogenase gene of Drosophila americana.;citation_pages=243-246;citation_volume=2;citation_year=2008;citation_doi=10.4161/fly.6875" />
<meta name="citation_reference" content="citation_journal_title=R J.;citation_author=A. South;citation_title=rworldmap: A New R package for Mapping Global Data.;citation_pages=35-43;citation_volume=3;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=P. H. Sudmant;citation_author=S. Mallick;citation_author=B. J. Nelson;citation_author=N. Krumm;citation_author=J. Huddleston;citation_title=Global diversity, population stratification, and selection of human copy number variation.;citation_volume=349;citation_year=2015;citation_pmid=26249230;citation_doi=10.1126/science.aab3761" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=P. H. Sudmant;citation_author=T. Rausch;citation_author=E. J. Gardner;citation_author=R. E. Handsaker;citation_author=A. Abyzov;citation_title=An integrated map of structural variation in 2,504 human genomes.;citation_pages=75-81;citation_volume=526;citation_year=2015;citation_pmid=26432246;citation_doi=10.1038/nature15394" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Tajima;citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.;citation_pages=585-595;citation_volume=123;citation_year=1989;citation_issue=3;citation_pmid=2513255" />
<meta name="citation_reference" content="citation_author=N. Tatewaki;citation_author=K. Maekawa;citation_author=N. K. Atori;citation_author=K. K. Urose;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=Initial sequence of the chimpanzee genome and comparison with the human genome.;citation_pages=69-87;citation_volume=437;citation_year=2005;citation_issue=7055;citation_pmid=16136131;citation_doi=10.1038/nature04072" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=C. L. Usher;citation_author=R. E. Handsaker;citation_author=T. Esko;citation_author=M. A. Tuke;citation_author=M. N. Weedon;citation_title=Structural forms of the human amylase locus and their relationships to SNPs, haplotypes and obesity.;citation_pages=921-925;citation_volume=47;citation_year=2015;citation_pmid=26098870;citation_doi=10.1038/ng.3340" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=A. Varki;citation_author=T. K. Altheide;citation_title=Comparing the human and chimpanzee genomes: Searching for needles in a haystack;citation_pages=1746-1758;citation_volume=15;citation_year=2005;citation_issue=12;citation_pmid=16339373;citation_doi=10.1101/gr.3737405" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=P. M. Visscher;citation_author=M. A. Brown;citation_author=M. I. McCarthy;citation_author=J. Yang;citation_title=Five years of GWAS discovery.;citation_pages=7-24;citation_volume=90;citation_year=2012;citation_issue=1;citation_pmid=22243964;citation_doi=10.1016/j.ajhg.2011.11.029" />
<meta name="citation_reference" content="Voight, B. F., S. Kudaravalli, X. Wen, and J. K. Pritchard, 2006 A map of recent positive selection in the human genome. PLoS Biol. 4: 0446–0458." />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=W. Wang;citation_author=Z. Z. Xu;citation_author=M. Costanzo;citation_author=C. Boone;citation_author=C. A. Lange;citation_title=Pathway-based discovery of genetic interactions in breast cancer.;citation_pages=e1006973;citation_volume=13;citation_year=2017;citation_doi=10.1371/journal.pgen.1006973" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=B. S. Weir;citation_author=C. C. Cockerham;citation_title=Estimating F-Statistics for the Analysis of Population Structure.;citation_pages=1358-1370;citation_volume=38;citation_year=1984;citation_doi=10.1111/j.1558-5646.1984.tb05657.x" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=J. Weischenfeldt;citation_author=O. Symmons;citation_author=F. Spitz;citation_author=J. O. Korbel;citation_title=Phenotypic impact of genomic structural variation: insights from and for human disease.;citation_pages=125-138;citation_volume=14;citation_year=2013;citation_pmid=23329113;citation_doi=10.1038/nrg3373" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=N. Craddock;citation_author=M. E. Hurles;citation_author=N. Cardin;citation_author=R. D. Pearson;citation_title=Genome-wide association study of CNVs in 16,000 cases of eight common diseases and 3,000 shared controls.;citation_pages=713-720;citation_volume=464;citation_year=2010;citation_issue=7289;citation_pmid=20360734;citation_doi=10.1038/nature08979" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=H. E. Wheeler;citation_author=K. P. Shah;citation_author=J. Brenner;citation_author=T. Garcia;citation_author=K. Aquino-Michaels;citation_title=Survey of the Heritability and Sparse Architecture of Gene Expression Traits across Human Tissues.;citation_pages=e1006423;citation_volume=12;citation_year=2016;citation_doi=10.1371/journal.pgen.1006423" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=G. Wojcik;citation_author=M. Graff;citation_author=K. K. Nishimura;citation_author=R. Tao;citation_author=J. Haessler;citation_title=Genetic Diversity Turns a New PAGE in Our Understanding of Complex Traits.;citation_year=2017;citation_doi=10.1101/188094" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=Y. Xue;citation_author=D. Sun;citation_author=A. Daly;citation_author=F. Yang;citation_author=X. Zhou;citation_title=Adaptive evolution of UGT2B17 copy-number variation.;citation_pages=337-346;citation_volume=83;citation_year=2008;citation_issue=3;citation_pmid=18760392;citation_doi=10.1016/j.ajhg.2008.08.004" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=D. Xu;citation_author=Y. Jaber;citation_author=P. Pavlidis;citation_author=O. Gokcumen;citation_title=VCFtoTree: a user-friendly tool to construct locus-specific alignments and phylogenies from thousands of anthropologically relevant genome sequences.;citation_pages=426;citation_volume=18;citation_year=2017;citation_doi=10.1186/s12859-017-1844-0" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=D. Xu;citation_author=P. Pavlidis;citation_author=R. O. Taskent;citation_author=N. Alachiotis;citation_author=C. Flanagan;citation_title=Archaic hominin introgression in Africa contributes to functional salivary MUC7 genetic variation.;citation_pages=2704-2715;citation_volume=34;citation_year=2017;citation_doi=10.1093/molbev/msx206" />
<meta name="citation_reference" content="Xu, S.-J., Y.-P. Wang, B. Roe, W. R. Pearson, and G. G. Deletion, 1998 Characterization of the Human Class Mu Glutathione S -Transferase Gene Cluster and the GSTM1 Deletion *. 273: 3517–3527." />
<meta name="citation_reference" content="citation_journal_title=Annual review of genomics and human genetics;citation_journal_abbrev=Annu Rev Genomics Hum Genet;citation_author=F. Zhang;citation_author=W. Gu;citation_author=M. E. Hurles;citation_author=J. R. Lupski;citation_title=Copy number variation in human health, disease, and evolution.;citation_pages=451-481;citation_volume=10;citation_year=2009;citation_pmid=19715442;citation_doi=10.1146/annurev.genom.9.081307.164217" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Complex Haplotypes of GSTM1 Gene Deletions Harbor Signatures of a Selective Sweep in East Asian Populations" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9.cover-source.jpg" />
<meta name="twitter:description" content="The deletion of the metabolizing Glutathione S-transferase Mu 1 ( GSTM1 ) gene has been associated with multiple cancers, metabolic and autoimmune disorders, as well as drug response. It is unusually common, with allele frequency reaching up to 75% in some human populations. Such high allele frequency of a derived allele with apparent impact on an otherwise conserved gene is a rare phenomenon. To investigate the evolutionary history of this locus, we analyzed 310 genomes using population genetics tools. Our analysis revealed a surprising lack of linkage disequilibrium between the deletion and the flanking single nucleotide variants in this locus. Tests that measure extended homozygosity and rapid change in allele frequency revealed signatures of an incomplete sweep in the locus. Using empirical approaches, we identified the Tanuki haplogroup, which carries the GSTM1 deletion and is found in approximately 70% of East Asian chromosomes. This haplogroup has rapidly increased in frequency in East Asian populations, contributing to a high population differentiation among continental human groups. We showed that extended homozygosity and population differentiation for this haplogroup is incompatible with simulated neutral expectations in East Asian populations. In parallel, we found that the Tanuki haplogroup is significantly associated with the expression levels of other GSTM genes. Collectively, our results suggest that standing variation in this locus has likely undergone an incomplete sweep in East Asia with regulatory impact on multiple GSTM genes. Our study provides the necessary framework for further studies to elucidate the evolutionary reasons that maintain disease-susceptibility variants in the GSTM1 locus." />
<meta name="og-title" property="og:title" content="Complex Haplotypes of GSTM1 Gene Deletions Harbor Signatures of a Selective Sweep in East Asian Populations" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/9/2953" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The deletion of the metabolizing Glutathione S-transferase Mu 1 ( GSTM1 ) gene has been associated with multiple cancers, metabolic and autoimmune disorders, as well as drug response. It is unusually common, with allele frequency reaching up to 75% in some human populations. Such high allele frequency of a derived allele with apparent impact on an otherwise conserved gene is a rare phenomenon. To investigate the evolutionary history of this locus, we analyzed 310 genomes using population genetics tools. Our analysis revealed a surprising lack of linkage disequilibrium between the deletion and the flanking single nucleotide variants in this locus. Tests that measure extended homozygosity and rapid change in allele frequency revealed signatures of an incomplete sweep in the locus. Using empirical approaches, we identified the Tanuki haplogroup, which carries the GSTM1 deletion and is found in approximately 70% of East Asian chromosomes. This haplogroup has rapidly increased in frequency in East Asian populations, contributing to a high population differentiation among continental human groups. We showed that extended homozygosity and population differentiation for this haplogroup is incompatible with simulated neutral expectations in East Asian populations. In parallel, we found that the Tanuki haplogroup is significantly associated with the expression levels of other GSTM genes. Collectively, our results suggest that standing variation in this locus has likely undergone an incomplete sweep in East Asia with regulatory impact on multiple GSTM genes. Our study provides the necessary framework for further studies to elucidate the evolutionary reasons that maintain disease-susceptibility variants in the GSTM1 locus." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/9.cover-source.jpg" />
    <title>Complex Haplotypes of GSTM1 Gene Deletions Harbor Signatures of a Selective Sweep in East Asian Populations | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"eFJYrSmhT_OC90insRSRyRIjB17IU0K9eSK6Q6Clb9c","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490722","apath":"\/ggg\/8\/9\/2953.atom","pisa":"ggg;8\/9\/2953","ac":{"\/ggg\/8\/9\/2953.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/9\/2953.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/9\/2953"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/9\/2953"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/9\/2953.supplemental":true,"\/content\/8\/9\/2953":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022CQVnpEc4NR27eQjKpHVWG72A+jFLYQHfyKKYhMxHHFrMXEJ3Cp06zzRROgYQL1FTkm2g0IORBl+JEFYNcBseb3QrjOypClhb3kdZrgcbQdCVqSq+yfweLRrIhhZWGMJDJJNN7\\\\\\\/6691ildwp2QPMwI15AhJw50\\\\\\\/Anv1iIYhOKetPeiT7i9PtAeJ8+sz7FcYUQEU0FgBNVPSI0grD09RRLlDyg4ZsJstQEm8M5LZWReA0oApNsQ5kS7QprDoj2QPNVnCq8TUNez4dx8VdrBMAqOxLFVsSWnIn2oVSz1rPy03vpj47qLROgda60FUpZRihi4s0OgGwB\\\\\\\/EjmrfQHUPnzKwHvUIjrOeOgrk0m67RrKNAE71jyCpT21FuDvq5UpkxoL8YWJFRLDDzWvfXLpaGhe3pcrAHpyfW73ts+cFXdJPkgagjYtMhgsrO7oG0kicr9wSR58hG48SuxzcsI8GQ2XE9okJI7Q8GkARI5CHrlzqRQwNi6R4DNziRVxtufyUDADbhnfBB1EyRMPGVVexEFKufmFRk6vogYyP375m0z93e9vEZkhvjjL+xfTFVkGt0uPp+dMKaoMyuDL5XZg1gAcfm1sfxYZRgFOwwvjfQzRgZtLDUApFeWcVt612sYwjZPOJpyNtlexGlEgxvsheCjFeSpdlbyY9ef3U8wh\\\\\\\/bgKQjMeXracgYXINj877A45sXnWgTeF0EaUvs1qOGjVQhhf7tl9jj8dh2IR2MJzT2gQG0nxMHFaBdrQm68lHuOD\\\\\\\/lF0tMPd2nRDa\\\\\\\/BVCJ1eRZ1f2WPhCyPtNYQBSaUV94FmvgVUXIFl1Yj0B9Dy4xw7N5lMVTHS4meVIrZb5kUJD+G17KqKPvj6XG4nTlVsBEiZbP7CL8Xl+feafh0YhHYOKdPTYIYaDdecUifl44VxXpQGjO1eC2v2sZBy1FMXRTwR2aGxJFYVnDhVWPtoWpRYQfIrOl3+MQ5fJgscw0nuRzTatkQa4199Plkf3f5lDIJVDkAVFqzV7oeO9weeeZ+3Fs69ybPefBivUc31epLJbqr8xIGX0z5kI6jwvuPepqHLF\\\\\\\/tr0VFCMlFHTIfe0oMbmKzo+1zrnrGXh7JpiDb4GntasTNMYWjOQ\\\\\\\/QLe1J4RHzKxBQHpx6XXaEeRBX5luedk3QY1g7qd6A3Mp\\\\\\\/yVkh4BF8U4JtGxvrOVWwrU11PeyZkLmVC6Fqkv2Cx1L9v39cxaKB\\\\\\\/vV\\\\\\\/pA71vk4vDcqFUVnb9pkkBajhorwUAnvqIbUdK9WL6X84\\\\\\\/bAf+Q0OK2mAqdW8\\\u0022,\\\u0022iv\\\u0022:\\\u0022FJ4wVKgrvbzMmXSmOaWhSw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e8fdca801fa6cba84f7f2a051d13412cd742b43089f8d701ba0582deb4666dc1\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022HepXyE+I40HpvN3QIj2YhWtQJpTfplEeYAxS4RoJca2zFjU0gu17V1rB+i8WLng+5vz87gP+nkzwiFW0Ws9te12sHyQULEcuy2MhkFTOjZOZTi4zFJvsiacUt5il5\\\\\\\/6u3KFgyYoCh9vQacotwr2j9jc4IOIGkRclNkjvc351vwdsimn\\\\\\\/f4bA6CocB5Fdp9sdSfXeoy2afo5mnbDJQFD5mzT2dmb9A0xmEnaebB1nzBux\\\\\\\/1fAho4D37foBq4IFOwxpzALlmjZYbsG8wObQ671E0Vguz2+jV7MJvKnYiHFoku5tYA0\\\\\\\/ZxaOoIiN5WjiMG4hT6dauYwld4LhTgmOasRfpkqInnpNQqwNi12zRtLMZb75Cwvz+yzuwdGZIKUVJRRp9sufjtCOR5U1Vf+2sXgOOTVEBAp2lHAlf0xCZ9t9GBy3fkZHSN19LI\\\\\\\/N8iOs5hAxyVTSyeFSN5QN4\\\\\\\/D1ZQCqnfdVxuv7TPWUfinZx7lvKIxjKefbrS\\\\\\\/4xh\\\\\\\/agL\\\\\\\/+Hg6R1UwzbU1Q\\\\\\\/RhrLJ15oawz0jiamJBfedSWSdZWG3SkLR076T4feTxxvykJQey42S\\\\\\\/1bMvQ7sDCIaIPndtmnk5SYUdPBGM5fonaIVt\\\\\\\/uRI5sijBe05TJZMNJsw8JWjU04gmvv5QFKfw5ovziLH\\\\\\\/c2t399z8\\\\\\\/I+GO0fHEJ4zL4mgP1dfu4mWN1kB9hNxwA\\\\\\\/xjqTam26fiaP2ppZSuhVSNcLkl+KdjbFnnSH7o9g2vQlGPdDxcsOkfVpRyosLbHONPbEd6wrPq1NKuXAsmoSN97ucFFSVBAYXTE6ZnP+89AjCaSNOEarohPMILFV5osVbC0qphRfc94yWqRvYUYbr2Br8Fc4aHPbd+XRw7pQGntxUJPUZKNk0pJNm490tebpCt4tEpnRBtrD0C\\\\\\\/UBs8Go3qZHOJcWp8F9WYjpLzgOs2tSvUegaC1xiUCaMiJGrpzK+0LZAyCETEZdvDmS0uA9jwLib8AD1hJ+X6xCCXFLnLUdTIbU38T0OOOT66ZzDBICSFsUVZwY9HzZ48XNdFZ91tYTuid6LIrdWQwP9\\\\\\\/hQVE\\\\\\\/ayCBnav5lQtcSEs7hoIOutw7LP0VPVtA2EQFl2fTJg+y0gjBQBTI1y7bdwnqwb+ijgqixhiGTIeGw+7ZOF1QJX33OtO5pMt3AWVOPP1gbLyuJWUx8eO1js81C0qI3uQ+vI75U6cP+us4HVlt837JdG0MqtHNyup7XKJRvmSiBDhfb5uf3QnHozn78tqFVO76x11QpkeDdjsHJp\\\\\\\/JsubLEJyMCs442TQpEHGADAslZuTrG+\\\\\\\/VVxyGJpBPR\\\\\\\/6zfP4ZAy\\\\\\\/oIW01cJJiEr2EHbn7p5xlTy2TIwbmRfLoA+bRneCwlsR3+kp+QkMnk1fdjGyf4HEIDD0uwYqpE9nV9DoFZM7ub8s15lZ0t2y1p8tSnC8DDmD+7kddlP6DSl1wzXt6e9oINYuN2OPfmXuWiYTw18DnMBYkQh5HwpizS7MC5utYifIjYdsUy3\\\\\\\/Ud6EBmUIVc6lr1gVFtm3KAWNS8UAJORNpsNa3nNJaFEZR5Y\\\\\\\/R2gRaJfooCTmEfm17WoCe1CMZ2t+zLQPyWEc7FEpk+BMo+ji+UwDUTv0CxgrQAAcpIYk7\\\\\\\/uWRRBeVB2wDWwMo+TynybPGnVBzFkEgFq2UbgTtsZ0cH1X0z8YDd5gi+l+3+CnpYKfmNQfUVTLgbjtXUXg51dW16mBVL\\\\\\\/x51bu0rxUTvfu5zKFfKaMvuySDyP0SlVBVfv6UiHoUL3xlou6UUgZnHuqSk\\\\\\\/EtiwM7qRi1AUw2TJ9\\\\\\\/IknTlPK3jCzUoZm6FwfTjLnViZWk38yyX0eNFx4ApTUTe4QVuPQ7twskdyOK9bh5o17V1QFWzSs3+CI76mFNwE++jq44cqYeAh2HMr2YWEN12Lbi7dt3\\\\\\\/IFjzK5bnYArvdPyFARrgvVvsMdaQpn4mwYHKcbXP04VHZcsliKaXtXmTrIn5nOPm6OJQD4di89R53iey0F7xlpaelcdAgfKtUAkWwocbNJqFI8SJMRf+ni+2OKspLBy5Wrr9+q5KgSHE8fedKSmneklbAwHVZUvpWzxAOqIF2zhhhOqTrUsAvPghN30mcUKmF5DAbVQM7tdToQ5xjvuzEmxe15wg3THI2H4Tf3yWZA6bDeq1HsPm7Bg14bR\\\\\\\/cn6FMzImwcCnMev\\\\\\\/aiY7aMQ20jwLebA3yPOhByPSuJllCj1tnrKT+lmaMr+nWCXF7Kx8TmSx1eH\\\\\\\/oyNdg9lK2QFaIofNQ89IfJRy1hmEFroxgDUtTkg7MROrZiLYXNvX1gYJLe+6rce2FyyoWArse3CNFU+BnlnHHbMRcz1J+mafaR7N4TqEhKDwT+jbt8XwUfXBKOEvFXzrDcNZcVJhfAzPqRziVu2jYqpfh2OftO93VmlNGm0ZX7WgxCxcJPw6ZNZLUQRVylY1hqT51nGPmU1kxVcTAh4hxxGmAZcX9cC8IVbYUB66bdS3obUf5LbQ+C32i7\\\\\\\/fsTpHWQYSduzMdVWX39mLsSxwAFGV9XmXM3Vx12JHvT1X7h90e4mNut1LzwMkZdfqqyne0m4BBN17+p9CEb5bqUUFE0a+jexPz3PMIqUJRUVPG\\\\\\\/N143jgxkH48k8\\\\\\\/4ZVAaXw71gETIUhN1II6WCq61yrO5obhdrdZnMF9ZL1RfQVEDViAosaezY46D+2353mfsPkHSGjHuZRnFvVRXUf2Q2jlF8RCHRSbIwV6T2SF2i1PHyNo3DKqCk2tWAbewYDL\\\\\\\/asD2Ic2fd+d\\\\\\\/Rb7th6\\\\\\\/ry+qH054Jyc5UaW2xENBJCqEQEkRZ+wcLAqh0NbGgrRESBu2E3hK5qELZeqI98I89pNtUITT6Cv5E8TYtJUDBsp8so14nMib1L3jr8Rzwm4WbAtU9RFLJIlq74X7uB869vs9OfPmmYLD+EgbXEzfq3u1pKB8n+jiscO+YOZ6W1Av0nCvEJPbU58dHKIsDcz8JtKvHZ3b4eZB+KpJHdu2dg8+ZQ0jSh7dy0+HQ4PfiP1n7xrzO+YGS\\\\\\\/XYfW2L3OEwqnZsJ+3qQZrHG7uqQrkh6HxCYzFY6gWKE5B4jbdXAfs1MUC5ZLKRQyMLv2efqhO5jIbupThIBQJY5hzJ03fdQfX3Bhui3EFBk7gt\\\\\\\/ME96NElxIjivgOODo3wq+LPzcF0FI6ivyolB80Gy5YVOIq9W79wLJrq\\\\\\\/KW\\\\\\\/TT883+yCZp\\\\\\\/+ZRiKCGuaM\\\\\\\/Hs+NlwZsa7EL8bU10yjYJ8gpXvEN7XJC2nLfZMHj3KavryaNhKrncbkGTJxnnB8J++sfB7sZW8Tq1Do2b3UZBs5djdPlcOft0EdIt\\\\\\\/hrFvzROHJiZKKGInu7ElHc9kZOBSvhs5Nsl8ZgJNrsPvTV0sJ5me35LhOt6ZB9UNJ2t1f2H0VGvlmaqGDTbQTHkbMjQ3IAeWCGDSpvkGmDyQzWN1yszTruM0cA\\\\\\\/WsryKxTF1n7JFk3Z52D\\\\\\\/BWwFFVBD98OlYnAgutZa15zTLc\\\\\\\/N6MCWaI5DPk4poaUBmr9eh2Xsm9GyoAewCirbCahnvc+DuEYNDqhTjn1fmJOifLoVF8oq91lFCnnARchQiF\\\\\\\/vKn+qWUUDyzdzFEfH\\\\\\\/CrD7AuQYzSLVzSGphfpY31kM0GDEraLf2wdqFariISaL3qj3GdW8Vwad5nXdxz+O4GZL158sXaVPxm\\\\\\\/6GEETqfyehEocMlV7oamPdFgGytzlK+BnXdCFCObrYALAwPmYVbQiHUvn4eNZb9JZ86JFxwGP5KC7uqFeNizJKanQB5QfVXQzxlUiUTuMkxd\\\\\\\/yie2EOL0YT1cWtfNUePROvVmQwmuWVOCNaTQHocEzbYCc+GAFkeUX3HWVMHDraOXePUDVtzkNAgJKg1SGeulAhUoQWwjRIhc7xcv5Q3kcwdSnhK21EsN9bUFPW5W\\\\\\\/\\\\\\\/myvZRwM9xA7eU5SPbdswZAqUOg\\\\\\\/MrG0a0qoSOFCzBXRTQ4HtjIBJ69j5WEy2t\\\\\\\/qAgTUQqU1GGnJIPU1D9iH9DpeBdv1KwWkX6kMw3oJCI+4Qtg+\\\\\\\/JSHgfY2uADSC7jo9nR4rhurkp6iuatw2BnbXjBNK0dD7eYfq64Li1Bm\\\\\\\/oaznyHR950obnhKMcW\\\\\\\/cICHviK6UZgNH8OHndn2tFl0g4u\\\\\\\/YClI7T2kmeVvzTOL0KSnvz0xEoG0YbZvItA2vyVo\\\\\\\/mnXW0Yq88YPxN0dz465l0k\\\\\\\/WMRe\\\\\\\/GBXKyzkjexkrK+jdvgb8PxsryWXzXQHSZh08ZRi7BDRHrEJByqW4rUflu\\\\\\\/YKngbQanI17WGEPw\\\\\\\/khPylSAgCD912zTnOdJVVBzkKkXkUG8KKH7VUcQRfZ56z+4dhqiO6VkN2F3LWkI6rdsK2UksX7gNl89ueWvODJCDCcSidfnpv2NOCuCrdH3MxPpsVh4mIM7L9z5a\\\\\\\/\\\\\\\/B4a2II6+Lkg0VyxpU\\\\\\\/FK+PtGtbd9ic2h1K+n0lX1nGVbGmKcFPmsZRNdr5QbUTliz4TSDhzZwg4SnyT85jkO632YgcphiBVcT6IIdAm4OYharnjJudP1JJw5muixXBvfG2gpS8fdG\\\\\\\/SkEKuGWqxi8j\\\\\\\/eemSWH7bnnJYhuXyr2zO05Dzh42W2UCBWzyAAMViolUFDvhd6nX5GoTVWgpLi7lh+v+90ffpOO7Tkxlcn7l22CHSszSMkG0UD1ForkVBFDe5utbKyAf00kmGE2npb\\\\\\\/zJeN1c1Hm7dqoKt58DnXOSuU7HyzoF7\\\\\\\/5B9xCm2MhFOYkJK9UMyMv96Q4ZJFpZWb4Uq4OrVd3HxRtd\\\\\\\/dcd7sabCoY7dhEFpnMtwdsvw0a3tjbpceCmj9jtLQAETEQExCHXbINb4liGX5fT4ZoHw\\\\\\\/Pf+H0jqxJdMgQLQihPIGTx7sfW3LOZ2TOVs5RFcPTuXYJIc\\\\\\\/szzt23H8L21pv5\\\\\\\/cUfpRuk+hBhEtFPQRWUUolhmMB7LNTN6viRaUp9Ep0d\\\\\\\/oTQnk3QTprQgK0bXdPkV0lnYC7DJwFLPOdsi8MVISa2GhaGy6rB\\\\\\\/gnTSKczb0wkNdX6TZk\\\\\\\/BcDHSxoiEYH2eJOSjwTrw6P\\\\\\\/4lKnQIVIETJ+fmaUIhMFbvC31FAsF9amUILcgwGVGA+dGMxSeHPKUKcPlZnXaIUppbAO+d2vdguUWoFoWehhOlrjRck4eBpMXxB4SXhqowgOwhVLWSzl\\\\\\\/TwclSsQkYFtTF22lDPyjzOUrEqf25jCOmMFiTwLsP5VoMwR4dg9TM1J98jr7U87S5m3K7Bri2IqklO6hDpzounNVanHdEHqfOJnGmn+M7MNFNnFrOU0QVapWY2mcLdcRdIjbKS\\\\\\\/W29mzDfqZaV4kWG1eTiPHT7RjSw5QZ4eEQYF16RAnEm2Ga4aDGLPNExb4CC9NMb9\\\\\\\/K+Nk7fGvrerBogWqt2D2ysqRGTvUhiBzDhcbrymY2G99eEHy+O6VgXQWuXt8DOOj1dtnQ5eiadnFSouQ8RJA0KJdqvpCQ7kOYnCwk2J40JtLaHBJmN0vUlKzttTo62UsGacZWYNx+pViHN49QLX2iqEONEoNRDOQvTe001Y5vm9O42RYC\\\\\\\/yMsEquysRNeBx1Av2NCMvPehHkLjUhpqcuTe8A0FwQZcVcqfHU2KhbmflhrGoIGfosR9Iim2VkZcw3jFrm+PjgLl7kE6GVOHEKr9+Apz3EsTAhRn2aUPuMlNjV0ak6GWzFXlsyzZlxJDHSD9mMqd2J6VgxhSHHnibnwt3mo9puvfkiXlAcmd0AT0kQ4rlpb88+y6er5ApSbBtKeUysFlyLcS0hk+blTz+yovzvVgQn7SB1FB0mr0mlnCQYe0xDhYT8D4sN9MABMEBElhKkxdGDDmHw2oKRHjYdvm1xANAlyg8IncKWjJXsrFScOv\\\\\\\/SLVt4f9fIp5I0KLadgYBtFxKbYOIB0o9010rEQjZ6GBQ0DOOWQCHThlcfmCvyWCdSYJZ1T1Ik35uDft\\\\\\\/ZTVI+q1YpdFqdHyLC3iVWJXR50ydMaGPijqYXHVP7fX\\\\\\\/ENiQ4gYpuJfNho9vPiMZbwSyZ5BTnfWa0SjVc2zKXQzcVE\\\\\\\/7gs+j1a8mJFJnkWjBHYV\\\\\\\/xlGs+XNMkoEQEuKMVp+3PIGgn4m5mhi7LJ73Mgf2Cj8ZxbE6xBJiKw\\\\\\\/7Ql8crmA\\\\\\\/NkzKjYUjesSzfypd7f23xdn5k5S3czxJRMv3IW9LuVcFlNxjhCb2kSgdtOrdw+vsrh3oxHLwQdyscFsxeZtIYk38dwHlKz\\\\\\\/+5\\\\\\\/V8Mo2Bazl905PuFsVdgELPxJc91pn2kohivgA5Jqg0sicMino3zmdUKGeLQyxaIjM4jQFAnZuWib\\\\\\\/fZEJoUr5D23VmKxXvw+rZRj3m+4AEayg\\\\\\\/hTe1\\\\\\\/46GUFNsH40ZYnbaebySRT82JIkOpG4Wq4K6EH6pOmBDnxv0+UVVqCB5QRfl\\\\\\\/QazW40MCj+xQNtCvIN8skAniwZtRO92vDpREtV5E9PyjmV267qEucmL+guIZ7GnkUC1jr\\\\\\\/tlY7Sc5\\\\\\\/MidYdADWJ8TPSamNfde7M9GL7voHOn780O3coeP6TGUXeUJVX\\\\\\\/w+aTjIWZSD2qnforSqeo33LvxKwdHe1tHWh5vc0fBXm7oGiGIiFHgTwJxQinVydaeIVz3a3FCamlMk8zQSRqwzRDMUuVkDjkqLPxhWqjWgF3A3LLM9dUlMi6dQy8mOmpbwFULeHBS+MTC5G5JYmxkJ8QJwWfjj5svMtT7V2xSzk9Ag00okKUS00mzA0WxJZrNHS0b8ugGpWkIcUqrZnoCUOM1FhfMpEOP5pwREkVoZKPau4u+8RW19B\\\\\\\/ALG5Lvle2OuLu7EVocUzW3kV0yL+S5I04KJargHYgaIOk8vMW9+ei7lnh5ACpbDAf605fsDV2YJIHKiV7WhWB5AnbQSC2zK50WDSzHFmOGBrH6K9p+LYBw6L+Sz+\\\\\\\/50RO0MgO2c16zxjS2orCiErQ5+wyCifiTbZZKouTh0VywxrzrjKEfHgwn0\\\\\\\/biXiwsi3zAVxf75QoYZUZHfzxcgWLcOXO+uHaYckUIPHXPVomwuCfPqFk0lZ19qT62+cwZeN76ZBmrYGkhIRbXUQSoJ7MRJLHzzKXukHDSim0JreWSB\\\\\\\/iJ6UaZ4HNKwJagiJbHdhtx8Mm4MmrO2bRI3dIy548Tt\\\\\\\/WV5HYZaOoukwtS0ChyIl1oy+NdCu6OH1K2DsExWFV1bQN2mJd52wMgWjq5MULy7iblhte2tyI8v2zptqMpu+COWWy6EUzcBztOqE9KvuSzr3DMvWffFmwowYWt19zUFcH7qSbCLwS74X\\\\\\\/nmm5GglhGqns+xTukwAfzmzlz2xkYbm7SMhVwgb05HxXg9wbFnDEiVmzuZ9Ld0BJEhCmGZIF+7g5Rj5UGwJFISQACTDlx5cIgnthEJHi++7HkDGXiVjr9uI1ILJ7q9idexOYMvT8xumpKcKpRr+NsNU4A3TwlmS1gy7iUIa536FtpME1OAQEFzNMWZ5giVYNQIaN625GxLmvocza0Cunf2syARZimGtfbbPsqh8h3UD8dXFOVjQcFJbgVmVLDJmfxiym2ztdfMXZQz1TdTjJ081ZBut05CNvt9qdKYt1lUt3TKO8sYx47On9kGX2X4Cky9CTxkp6CKkYwoQ7am3Nffk\\\\\\\/4i6qP5BPt\\\\\\\/+Q3XgOmc+2TQooUl0yoS2Wi80JvHvUA4a9s4N8Z5GUXnR9ExwSDFIp8FOe7l5ubL++GqAdL4pf5jtrnr0hWVpHuO4nhwu74leeezORFiY2ko6HI6KjkVA\\\\\\\/m+2j58OQcx5gXxSC01MYaTNGB08YNIfFr37kAbF4J5lC\\\\\\\/NnQmGo9l7q+VG6N397b\\\\\\\/0r1W+JxIPVDAuoP419KMsE2k3yb70fiAKUjkRuPUMNZbah9qM0DyV+++RmuqeoOOtkyaEevIEaguJZi183l2DAGTF1DthBovjfPp9lOkZWvOS\\\\\\\/o8s+cRSRZwge+Iu5SB3X1lAnUOvGlzRmwX0t9WCTkJ46hSvxNWi5zQrLxYacskyC7d9igDykaRs6MioUaKtFYYOWf\\\\\\\/ZnulUcYh1U+G6ZJUMq73Zo9pepPG6Qd1awcmVDfMn0BAjbEJd24IMenZKpqZ\\\\\\\/czlTyYMy1jsCdzHHerIrtf5y4wm1nEJ4lTCiwkqVwuvso8J7Z+A2CBo53kypr9biYBp8hjwkEkmikwlzH1BXCIabLkKC5hNFGctKhCJX3MNhmx0do5C02v4oLIfRVATEGmwfYeq8uUHUoaE8Kqj3lqlMuyuvxoSS2Q\\\\\\\/c+sggFYC2T0bnihPzpAoce61lQYQZU+0lTR5Pi\\\\\\\/xkzELYRh7dE8LIRiCq2g2JUOIVOteDkzMGV+wEnwt2xfKpyOna6rYQEiMTGB9yx4Wv\\\\\\\/pfcSAgHwX4TmCJ0OnlO+1m5VUMOWIbWpfwmzjNcq5nJRMvxOunX6fp+csnMG2Ub8xvgKXYSVyd7nQhJ7LAeNT4qF5ahCypFchvwyHuRW\\\\\\\/w93WJX4yJxZHPHCLGUQ4tX6+ZisR1j2ZE8CihMZVwtawvX09irEQk4a\\\\\\\/E28DkWCq8z5HWvotb\\\\\\\/SvK4RdlkfAdOQnnHsembmKngMgLufuXHHI0eeZEhYkB1oZIdAmEH1l9OTufIFA8E8fruVqjY2tQG2eZrY455IgmlZRyEFmRbMEckB4GuneI3yTLrQ1EctGcwWv0j7liDuoRp+hOp7e8YTrT0Y65RtjwZx4EEZ9VImtUAebjeghX4Bmj6mZOMHax2jBEe4\\\\\\\/UmXNa5syvBD5\\\\\\\/sld4wYNmqhYXCsSgEh\\\\\\\/2vHHKPOLR6P7VT9cSwDIzRfpZR5S0wnRASJRh9YTylxCQuBWEOL87c26L7z3mwCKqi8sUFwABt+NCK\\\\\\\/121gOXnt67P59pIQzbG4yRsgYOT6FCfqIct1nGwfovJUYXjfa6P+v2Z+5kBJGmPa\\\\\\\/gj18ODBRbjUxEwgEkG4qnN0Uh0XNg4j6x7zkl8yn6vZ\\\\\\\/Urgbl1U5lOWMK8SCRBa+odx\\\\\\\/CClVHVSlcPprERkTexOZ8rTS\\\\\\\/pkX8837SiHCsh89BXNkN+xbPLu+6zQ3YyIJM39PrH+BGSQ8ygDRR7Jmycj5sU3ej4XYMsH3tl7QFY0L5qgb4LSWNEClktCe4pAceVbUaDbe3LmoCMSp5OtexuMRZqtmbYMuwKLLQgGfmkMDM5NGMdXPZupagndz18H97o+oGIr\\\\\\\/QtDhOOibWFthNFUoUf90qn5\\\\\\\/6dZwdA4SoqZY\\\\\\\/YnWJXKuGPrh822yVdSyZ97NJoudANgFLMuSlWHesf4QGP700j40xyiGuhhK0IaAO5ZXNnEstr4JbAjQvIPCTuvqpr5a0NqflKQXD6Z+dv5xG4dujdD60K0EodpQ\\\\\\\/Sh33Ylm\\\\\\\/Zkw5QFsZ3OAIxb2torWpAgb94C5jbbD2ZIi3f+\\\\\\\/jdY2nUsbwv8mmTTMcOF7I3yJupEgyrbC1YSbZKNZWGjCAy\\\\\\\/wExCnTVgYMR99pOjSGWc41zXE6PczVzSAoDDWrh8bQFHhilrtSKwbgnk5lDdUC0hZIjc0uI5l1pXMPmlWtLhW2dh1napeNmyOq5qEIvAwnAr6HoMuGWjrlmnYwhPqBbV3QNrWsA\\\\\\\/pUo3e1gcT8qie6YM7fPJLvY21SQrWx5S6gmN4TORvRwCAE4NzWtekMKVOd0bT4YcckF+i2yZg2yXOTcarMe956s2CVKv1MUFbllR7mQJJfDJRgUKbQBMHGtBT7zlkXfYjYKL4gvHj5gWCmCMnenJ3tfxPDi1R6puGSiBtkb+yOwH3HMp0Vmap64WB5ybkvNMT7qJZR3C9dBucay\\\\\\\/EgmWfs0VLQxVK2k6\\\\\\\/\\\\\\\/Wh0uHvFg6qGlG\\\\\\\/s\\\\\\\/n682jUp3OVtBBbgDzxZdG9mrnoy4wI3biNdFwQWkE+pmuRfB6HU2tUlMAKKxG3zUdNoQFjEsg\\\\\\\/fOke+rPTyZZ9ciRr2hHyMSA\\\\\\\/6TpYHk+ng2DQc6I1aRYGXF4y+IGeJfT\\\\\\\/yy5XLR8A0lHss9MyXDzibhG0XlEuYphzis7Mpr9J75tVXLPYVthO+35alVtrp13y+BiuAGNQp7R7odHOl42neIvgb968kb6IwUo4vZoRRl9rOzi5wDaPP4rQ\\\\\\\/yCGvimFRgpNawgK\\\\\\\/2yRVjdMXtVITZ663ZNoRsTgb+2Bc7oDMnguTdpYsPIbFP6w1et\\\\\\\/ybsdbPkIl347oYp67K+H9Nb6KQ0xhsJOUyivkv6A22JJvpe9XBsLR9w72SRde5d7BzykoLo9VCouyg73zHHAQtq3cSBxTxFYn8iuDl8fW4hi6G7AYTIUWk7aeAPt8YkbymL4ufn9S0DtmVmo75R6hqd7DqwqK20m8y\\\\\\\/KDheI2OvypnWTA4rhxN0M6G3ZqxZeIL2AwAQPRikZLlvq1+mIKnj4z79c14V+hTX0mzA4RknSZkq\\\\\\\/L3diuzU2zEPgDnpt+XEt8iLSV35O9AiF8DVReHkn3ztOy7DdRBBcYPAeWgw6dSTOXEpwWwdpvDdu0fC59RvolCdaaTzwbvz2s\\\\\\\/NI+8D\\\\\\\/mMaBp9KeLNY5N\\\\\\\/OGmojPI4MxTfYSYrNeh7GVO2T1DRDjZR3d0rvDaQW8Ol69CYNFENTBnlZHYbcfCo55XXnHx5\\\\\\\/VQyp\\\\\\\/7n99Ihp+X1p8gRB+9aqm3PyDnJMCI80rnSwnnsKrAzoGlq9QCM\\\\\\\/4arAsxBpKstgaeeHifUJAgpauytPw\\\\\\\/jzoehVS95qolFblZndoi7lp\\\\\\\/b3u37Z7sDa7WgcYxXcRMmvZpZc7j1zPHJglCnqvjgtk8VDjFIEjMBRY+vdkLNT6LypZkqkeJSAuc4jhQ3oUge6Pez4SQByEys4n7PVSXrxdYHFPmCKl6SHcUKHucM0fp6WYTZb+5rkuqZdT39m9ZJYzcsbkzYiTOs3v8AuFIDZmJjD0dyGa+FF38k+B7Bq\\\\\\\/QbeogI49qVPy0YymmhaWOzfVz1lcL1JZd+NjRUkDD3W3TC7+giTAwPGoV3\\\\\\\/aiFBCLQ\\\\\\\/vsviVBv+2W+8YXVcA8WWsi9lC+wmCskCwdN\\\\\\\/LxA0S4g3V8LGJtnn1doByuCjxv6dh8VMDFcs9K9BDURyzUPz8hLA1Xntvz9dl+x1RZvakusM+Hk\\\\\\\/Holw1YklmwXRKvkG2THKcwdBV\\\\\\\/DY9ZrEV7pkXjV9BM7qJYbzFx1yki4Sat5iVnKmXIzQgD726x88H60hgLRXEOI76AIPy5jimBMWjX1zKmaxgCzxA9LqtMj\\\\\\\/fDhM4WP\\\\\\\/mPyDJpOo9uNWBi6InWm52y3xwI4GignFZ1jgXhHt1HoEdYL+IN8nb5MjY25vn2Gtj\\\\\\\/2QrbTgmwBGiPEzXK6VKZKWpaosTk5\\\\\\\/\\\\\\\/7E7MNWFzD0XnD78zMfPoPClXGI1REWncAFF6P9MAk2v506\\\\\\\/bw\\\\\\\/evXGC0Mbm9HnhfD41EP2+KALyiwLWyObFdXisk2mYMTLbQ9Z2AA3zYoeBAGcJqdSyOvIUfcz3J7BVdUzEDNt5ILUvW5afCchd8++uIYMS\\\\\\\/tzimUL9kV8iZSkJS+CCxgjwVoTaz3kL9SI1KAmpEK50ruB34QdBKb4bS0u+UVJxLwWZPG6hu9QPAZeF8tTQzKcFEso5W\\\\\\\/fPsDDpUEg4mEw35bvASAHVYuEYJMjOO6KM65zbD6j0+1JHjHfCQWoIMHPLBChJ0zmfyzO75J4X1jibRsPswOuPUr7Zaeay5ojjmFumt5SaCxY2g31nwkNqyXkGSMTvkfaUH0zN6yNXR8ROky780pYcKqSjSxEHezF0fUhw\\\\\\\/nDAfOJJwM\\\\\\\/05kA4t0SuDuJ+31ursjPDUT8M\\\\\\\/ZUJliZI6VddNNVccHEH+6IbdeMN2a4T4b9d5C0BuFhEQn5+U\\\\\\\/qoHj4\\\\\\\/eNTLs1QsgywM5BfzjUNldJS\\\\\\\/je0RFHT043V1Jl\\\\\\\/8j8EFgM2VYzWC+G\\\\\\\/b4bD9BYRjL\\\\\\\/5gK9ZnGHcZcPjEVoJm\\\\\\\/uk\\\\\\\/rL08pXzYnEk14DaJQRsj4sd+Kb0fsODvdzzmWdS+LXLIZn6wBIsrxkM3bamf9YpjerbnaWIgTF9ob7l46GNj5z2w+ks6SZbKEv\\\\\\\/5mADwd+FAH2mvSqFdEa0BhpgW91SsGJyqB5rO4FGnjqaMIJlnGtz2FApWvVFDiwJ56vbkjYyplRhzpOM5rWT1mMOUH6LVYY2+vG8KsBNRoLRhR5ZYWi0IFrIZSUT3SzpXDfphwtzcP42U+fkpEqSWL3I8lgdGtaEC2JR638\\\\\\\/4uerlLAinDN7GoRHLV\\\\\\\/sqI7\\\\\\\/+IgU4EBFexqUeJ4h6pk4BaHIj9cXxyS3boCvoe2xrKAJEtyZ7T99j5aJoUnlLy+cexJEMFvWR5GNIJqiG2vxnAEzvMN95Vu+w4l95uk50OvZjfL3\\\\\\\/QaRDx\\\\\\\/f7nBR0vuAvDotbshLQnUNkWiQpCNhctQRLbT2DzsBKCf5fbWntOW5eXHUoZ7uQrYATbaYllXs4o+K95wgm8RSss1KgL8fccRBy0XnT3uk6j7+Kjp+pog\\\\\\\/9M44U8LvgOMHQJS\\\\\\\/\\\\\\\/7BeMWoKtM59iBliXWqxZbPqpLxUB3zKTtSByYxQuiI6jz38YZ8eBP+1Ko7KkuCoDibwwJ3xvJqbW5mbcYUGgUGZQmw1+TFBgevB7Y7DzB5NWGY+nAixuJt1ObWQYWv3AAy9Ge9ZhbJhf4xJNdWEomzWWNrP\\\\\\\/CNWil6hs2nrkOYblXdAGITc9sqmmMntjvqmQpFjGD34Ngvos9ReTHMStkaGKQpjaMjfvCxS1vKxBrJRoV1R17D9ThpFX2uaCyQ2UNjyNjGLsAK\\\\\\\/CH62WJJRkvPo84SsfAZC2lnCE47a01yWYX7BHK5orlqXPD\\\\\\\/PP2c8v1twDszyCY7nGB\\\\\\\/UD820eKrN2HG8H4ok6Ve3kM4zox8kIRJCWnd3XsxhEyvmzGfp\\\\\\\/1WIq1qPLAWvo7pDK1gRFIggZSUoMCSNslr4WpUtSckMBIwxs+CwZiI7RD4rqphrDC9Ew3KKSZpDJXcqOc9nYkJJFmDHchm5xXjdI2cvQIfVHNaRzfT63bG1sgFNg5wbSezMmz5RcFZTDL13MmcRolNG9yh547vg4wSrwqmmngWDme5zQhbXtwTrzlyvSuFx+UWokkIYv9n7jOE2SoogntYbv4P0D3mN7oaVkJiV626u6GCgsQnC5+cRHXOMWLONqtnmL4p+zVQ3IOOcrorGV6Mm4lOHbcfKefE+OXM3D0xWvFDfyjkNkk\\\\\\\/+Bt69ArbflhcCIwqkGfluPSwO9WJdjxPpcXq9nAatCHIetM2ydXcaZTpe2MpHa5V6wBGXqoL3H8k5mrkw1990FPIIvm1RcpivwjuzWojb71Z8jNAOPu49evAK\\\\\\\/f5irQonVBc8q\\\\\\\/euXPMBqCXwxy\\\\\\\/NQvVV9fTbRbEYSDPV2Uoa5nl2hZf1dXJ+U5gTikPliW22wfT+oBiBUVGTiKf829S7npaXBf3yv9xsB8UUNMInPJu0LfiOWsaU7lieCiES0gNDAfeTGtwqbC8y8FvJqKI7fqtZjVrHfXKqq\\\\\\\/Z1kMpby2SDk5ZQhMjBjN8z3uY6\\\\\\\/9Uk+bNEo95rbHSbZUn52DRa+pLQZmN1ySGhFlEhQk6LYXBSCGM4DaYGFRamuOzNXrwj67iV\\\\\\\/32Fqq2GNIBjGhFbrhPoJNjfORhpXpTOpSzcQK9YyAbbSmQ3XlYxGzIV0lx42cDz+DwqdsBPMplTooviSgLhc\\\\\\\/Who36a9d+KQPHefMtiYzfIP19KnGTLoZPZumaah4RCPrT3arzaCjn4+EzobdwNniHEx3kRubQAMehu4YihrZ3NCO9isWNded6Ov2zSEAyl72n9Uyb8rQC1IZsfMGe4RK+jSxXqd3MZP0ZdM+I8l6G0NV57cja2cW+vwZmB1wRxeWqi8\\\\\\\/zP6P6ynhNI7z\\\\\\\/H6hkkstHctAwyfWsUilgT93ki10SsHDRNFpgbABHyqWvblu0G3TnA\\\\\\\/TJgkXs0Rg5v5v8kI4y4ibLxOQl5MXwu3nEooatw\\\\\\\/KeHEeFOC8afL8FCdBEz6KrlDF8IExzEOtjuEsB9xKuDOpEB\\\\\\\/MzYBZT08CXeCaMreF+uuhipExokhchUFCamw8leBmFd4MrLu8aug2byq9okMrajj105Q+SLjqhoKSWJOJ1Bf\\\\\\\/QdeVzqZ+\\\\\\\/F2l+nfuHA9rgi2v9rUR2Ky+8sWy1b\\\\\\\/BIHyW0eM\\\\\\\/TsaDouCikbtAqryjdsJfLUiU\\\\\\\/3xxpDJN4y+OisDfpzFd8io2odF9xZEkqWkIr0MCVfVxN3Xq\\\\\\\/cq9fxl7ak1yI43HjgooAEIKxL0JGmuznQhJ6fnJTjS8CoGBEkgoSeM9Xlix7KuwIcdaz0i3Gt5gZhISV\\\\\\\/zHU1OUZiLB+PV7LvbyqwQtrZryxSLQrskkHPgnUSi51Flc8VX0QxdNsJuw8DXc2vkB+beKpBkiWlNOxNLiMApc+EN3WKKuaDFvuQWhwD1L8NaV\\\\\\\/KuLZiUunEV+daD6ZjEb+jIRDYdJXBB8OA6ZIKICrZ0GDYLA6W2wY8ROnkwPaCZSbOGPZoJJ61\\\\\\\/y176mQFcaB+Q1ITzfH\\\\\\\/TZGma0JMBxjiIncknVduw1yT\\\\\\\/IIrSlDeIOnSEPqflglm81twNqXaYL3SSKyOB4AtD5YJblISVdA4IDl7KaY+tNEFWyheHOjIzKfkWqZteNGClJr\\\\\\\/cwaS11WrLS+5Bxg4+LRbBnYBjRMBjjNhn4vzVcPpUyqQV86\\\\\\\/EjjyDjU9bVtgrlRxW2AWLfqyPHX983zOvCQ1h27zrYo18hU2zEWb4djV30b1kacUOA4d0tz9wFqmkUcRw4NpsS55oWRGVtstviyYhQ+m2kr2Ee2hyTpRjGfvPZJivkvGvxPHggtZvnejm169+q4hCe6s6tnjdwRLaIiDZPQ7JSkbCWf9bDmsXdTpHlDv2Zi7yfzGlxW0VoKizkhrhLCzjao+fuE6l+HLdqIY7OJ\\\\\\\/GcjcrDLkVpAvIAgq4+WzDcheh9TlnAI6tuFDFRwbBgVmem+LUYWH5ezss9fO68+p6RRpVtv3\\\\\\\/kgz76y7FRW3q57tCqru1mNgQnbG20S3OSul8Et9mc6kMcuw8oZjVcSJMVlKS1d110cf\\\\\\\/6NOJAysLh\\\\\\\/coIkn3cD5vBRCXyugSuHUN0sYZCkIF1wY5jgyDx5iLrX5xnifSaIIBIIASljbmi4dZ0g4XeUK\\\\\\\/q22J5HtNj5UZ57ue13qVeZhUr1S2u5pgLRNf1va0\\\\\\\/l7KZTjaFSqzTBut9G6IpWKZ\\\\\\\/6imrvNeWf19A6kqY1BD4EoRYs6FeO8Lp3iyUJXccUX1ByNVDSGl+zvQWx2Z7x8yewtCAMsruBZLwZzZGGpbGZmPwYLCjf1iwaT9y+mIKiEpm\\\\\\\/iFhbvrnN0BYNCQerHk23NYQUV5BZughmCODwuRzwDnB97XSzv\\\\\\\/u42hPlv2Q5lvibkcAaMtE8GefRE4csemKdyCOJI4O1ex+Z7W8fd8+eAlcGQeA\\\\\\\/3GsMQJFmQFuXbdQBxpJf6sGEvKdJWoOFMCwEj2jXUpy2b\\\\\\\/qKceBvy8y5lLQRNuVc9dRzusDkETiAw3CwGvxw9G9K2i6ym5o1EkVFTXxZ5odX5GZ+cUw2WUdqnAysYmyZtCrBKnqsEtakeYYFofebuYEmY7ogNFuFCWYiAxXp3BCefIBAeeY6EvcyAzfBaHBZczNXKcvXRu59j\\\\\\\/CcxcJ8j4Qdu2rvx9EbLRYWzVu3X8wHNL3jRfHhS5TpbQm3YvQxXWNzEuFi8OF3AFshkseT+svTzd\\\\\\\/51dZ24pJRPbKlxrrKTzutjgORX6x1yzbXEgmtsOUJwwkzm11hGB+e+O+qEd8yGPDR\\\\\\\/FCfcS5jQq56QtbVn35TNv8a93jup+35dWVzxH0sMO+E1\\\\\\\/cSXWMnHIgXBkN\\\\\\\/01dNKwUZ+l7FLN3IYAhICw72MGZ+FRSv7mhRn6\\\\\\\/4HcBYz2dcvac\\\\\\\/zLXsW78cQCSjEYuqnCfCNYV75dWX3Bi748fxhNQBKEPZo6FfwiCgHyeoNpk9imXJ2BlT9O6VQvtEy9z3oinavPI1NV279m5\\\\\\\/ZAhzwtaLkhZ6iVtc2RTa88OIQdWkZEYZQ+I37XYeBvuGbmHJ0dqn76EwDHiGG+ZXDNsgjb7iEIEWlgsvp2DZO0N4fo6rxrr1Ohj4TV+dHQrydxYym1VIRE2w1J8bPAz5e4Lu7NYNX1UswWFVyTjVgIstf\\\\\\\/fRu7HZpei9Oey102J2yxowdu+mfLgIyarm\\\\\\\/00eOLBYM1CSh\\\\\\\/ZuaNof84m5LwuOg9drZat2OIT00rRZE5lt9RMIgupTnz\\\\\\\/5e\\\\\\\/ZTRa\\\\\\\/1Deb2UoY\\\\\\\/7UN2mFJXSCwHBny0Rc5nDl6nMQ+1qxrtBwdboTKdeeN9TgdYHUvaMYM0lsfT9VPsycE0rjTn0zV4n4nfUpSR2YtJkiAmJNmqdOk\\\\\\\/iXFZx2nbkVCOg7o\\\\\\\/XpYjuLGdendOYCpKdnHoV8H5MbOYj7OIVhgiwvqINh35f0WV6Zi5e8P9bU4KaQtOWlfLPooQ0slQCodf5NiuYEA4101wwjxKn8yuRnEAKQ2XJaqQ3NcYQQfStZRGv4TTwqADy\\\\\\\/TuY51bbeljIj\\\\\\\/fbLOdwqrhDoyTnyIkEgChHCHK+Gwba6vZijm35BnIWfMmK3Nm09bjCuPtmhaCWxEIX7lf+n9XQjCkOofFPsw\\\\\\\/TNwFGuxmVH\\\\\\\/d1BrhM76bvOU5LuF74XK+\\\\\\\/zEObCScIhBeKqEh9C5VmXjD+ikcN37TJRa\\\\\\\/qnU2PUGxb8MxMooj+O\\\\\\\/DcZUz1RoPOQnRS0mwAYpBvNukWcq9cwZOEUb1sQEKQ2AoLaX1id3iAH+rsmLx+KCna3jkzlnSbmTWGlUiiBk1yWDL5fiGrhDlfm\\\\\\\/k3eSnFQaeVbd8ET2LP\\\\\\\/NiLUC0xi0jkJGA0\\\\\\\/+zEeoBViy+XB22kXMgxwLiIzFMgE3EF6iLtsW26ywyAm+LS69d+EF1DZMIH2varje+uMoo2PcIMDGVDJ5I1ngLs4eqs8N2ms3V1KzDGo43Sv1vSKafXNA5faj6y363BgDfrh4sny6\\\\\\\/qGR+T3z0cXlbMObsASD3X\\\\\\\/v4DL1mz+Rv+yy+UjhMP8ea+Mr2I9jdUs1Z6vDttoF2ZsMQtroq01Ze556JJLQjA72HeZlufNC6ucxst31bYGNuBZTQHmgbtR\\\\\\\/qBztT7xNYXwuQvPFy9DtGwzAg5RTzCMlFo0TjdjP4nOrRGNLIRWyysfoqQrfW+SUG+78Pc28RbH+7QJIXY2n+Ef8bMX5xV04tgbW37yaQYPX9axJOIWF3mlW36mQlynEspbVCnqoQYW0gIXCH5RCBJfR5jbqgLJ+zaCVNhUjvI3a4gsK4JsibLx+RJsZPNqgVQkOppqU6KYunllkqMz+ISQoCNFJNhtelmQDdx7NpZJdf1xD8oMcg9UL2PeznNIjr0QhMr2JBYkypdR9ra\\\\\\\/S4lrynWeg0YmliQXIVf+ECNZJLyCKvB3WagKOmhkqKNa0HBRcgHVZ+cEYWBhujG5OtvVpzRd8wlCNdyqUzqTCgac9raIGH8mbDsZDjhFqCFTxaR8\\\\\\\/A8ZD0vjENlFZCvIpWwLq0hiBlzcXZkvIaW9U\\\\\\\/XwavIDxifC6243JYQRt6uOAV8BTgpAytPOryxyJaXzbiI7\\\\\\\/OIv7s19RREEl8t9lLWTfGoYbPc0GdZlm9EvQCMJ7bbjfJecCgjze0ElfRzzu4puwizW6KTBqLaBhURmLOlK0B\\\\\\\/Q+06xLrSekMb12YGN\\\\\\\/ZJsZm1YIWPW2OneT5Ejzkc\\\\\\\/tc2dIeOgzl9aGFjcr1\\\\\\\/59FpKDq7v0HbDqzVdBetFV94pUVTMhSn8lqzWLYgHQGPXtV1UfOZMyinAA2xVOIDcXhTuaRCmlb7obyrds2HgWA1CJPC9OF6j+zVM9vTiyYV5YzL\\\\\\\/OnIwOEIn\\\\\\\/IrW6doj4eTZ45YeviPXXH63iF3mY3UvO58Nnb\\\\\\\/EAFaNjAMMBepbHmYJAxKonvB62l10FjWs88Kv9rsN+gDpnpoSQvgQcWbvLpD5DBQNUr\\\\\\\/PCvfj29BJ8ZnLFB+jlM+0tov2wPsUn1DDLKduCkInatPkm\\\\\\\/PdCwbT7TsSxaO+LSIySExQrNnLaEbfT0sTvMUj0SqbKGhKAZHpl44q6aXW+L7vFRiTyBPeSnTTNAoGRRTxNAOgZdF0WxitLMr4g4\\\\\\\/9uTMrlYInCj\\\\\\\/J4Jqps3o1WZOTIoU7IYXGO7LQUOI0ZGLYFZgbh3IwAax1mdAkU\\\\\\\/QRobp846IMKvGvc\\\\\\\/YmCQ48H8DyVPuxqagfdTSj9Zw8WYU+Etf37uxC+ae1zofuL6IfQGnltNTmHsyrSGtuR9+mh8p\\\\\\\/qImhAbep6LVWxaKbZgjMgT5fW53mBm7R7+JJIvrQQUt\\\\\\\/SHvn5RhCWR72uNYSu8\\\\\\\/vsVuGeJWDwSQnPcXSHivQVgbhr05mIyL1Zr3ziMwBm5mC\\\\\\\/ONYBJ1pRIOvwzXxB810wJ9Bv60wtvzDC89Kvc4LNZ0BH9zuSginZI9ECjF\\\\\\\/N0+K\\\\\\\/5ykiaJuZd6TiHkx9nJCCslHT+NqwMBiqCqk33MTZQnUGxYPX75tq3HUvtkBAvqvoBDkPrKRcA0fe8NuF0fmXFhK\\\\\\\/LfzWeL7FRSIMmvk12VzqY02W0Q3d6xaazj+FuedXiQchTZY5cSt8vzbj44s\\\\\\\/feN9w4+nX1BRUxsB1aAUwqGhw3dkKtQN9Dy7PXhnWh\\\\\\\/O1m0ubaIeHpYK06pswYLFU6\\\\\\\/6+l5C2r5IL0e4qa66A5K7iVM2ts2DXYjYBM3m2R03f3wIpKKpf6Z9D4RTNC26o+38q8ycQ3uEaARHDnIFeTbp\\\\\\\/Oh3\\\\\\\/QugV\\\\\\\/75X0JeL9tdUcBz7E2Zkw8t0QgS5vm5PfRdz\\\\\\\/ZBIOaMGVY39dFQsxFZSNxQ4+NPESam4b673YGub98ddNp6cPksInLw+W4eCf5YjJyRCTNnBuTrMcaKI2Hkhy\\\\\\\/XizC96QCzXsM\\\\\\\/OO4SM4m3NrEF3ibvC3UEnBG1DJtUAK4Uy4cwijKdb+42hCvD5zJqr\\\\\\\/fjhbUfj9ioRaBP8m+P8Ml7rJHdYX484\\\\\\\/iP3h\\\\\\\/9E+t+0ZNVIkkOs9LpTLF\\\\\\\/71ILypnVHtRzr5ZvPkAz24U+ArVFq1M4\\\\\\\/ot7eaTRRgOH2s5hVOhW2ymykwT8bNmfv5uVDPqItzCIeD8CaUgKdRbmX8S\\\\\\\/g8hizxG9wswDo\\\\\\\/4ZFMAnArN\\\\\\\/XDCku6gKwvvvv+va8+ry6\\\\\\\/RmVNZNAgoK58zg4os7UnPigkcrtevltPSGSbBz4lQhxzuIixrn26GSyNrcxUfR60S56YTLy5\\\\\\\/S54SrWv+mNP7vAfng3ZnX1qBUYXfSGlP9BPU4v88UXwxk+IJcGTO9vQd5quAzMNIXRcmixJbgjfDmwTYv7epbBitsJbgM6o\\\\\\\/4Aea4YUf5sdD0IbHfOZdpKVPxgIZzQM\\\\\\\/ynpNKLo3evXkdtX5uFcw7j\\\\\\\/r3tchbYHndf4YcwA7KlZsfGLGInnHu0RKIhNYj23bf7e1WDBA88noRGtASHEfUWGtPFqobul+e44foXn\\\\\\\/ihBglMD3IEUXzqmSOWGoarPwSnhn4vFPXK1gcTENdmV95kHUPghg1hObjXu5pTycI1qYY+PxdiKCpljDD5AMoHlEck9YhXKOtJRauxB+zmyTpL19PuhdFwIRIYuRhVbJSx7XGVUkm\\\\\\\/5kuGZsVFZQOmsy7Ap3PZ+lD3rKRykJ2Ii6xBCzL5A2XwrvwWWCC5u89QMaHmdMDUYIzmC2vdVl7i\\\\\\\/neIEnpaI73MsMaNg4EzTUBAHGlQSt862FXdrcR4UiJWN2nC2t8L6\\\\\\\/CwekDVnhBCaFzn7yTI9VbyCn48rXjVbyb+SFRessCT8XOdcWqbM+fq\\\\\\\/A3lkmLYL+E1FsRFOkJfsnqkQsk3ga2FoGcWUxyYLA9EBJp7tyTTcJPvXuhk50KPcfFy0CeaFVE0EgJmoh7PmKbeV20lSfj5BP2meqvnDkSKEba2AhhVRl7xqWedzAmlTf5GIHr5iqPmMSUuzY68iARqpsc0HYhKdgkNcc\\\\\\\/bRmJJRiD3bb2ZnDLoh9pEMvUpu3UtFvt9rjkC3IcyXHBcQqu0+7cgtFH5l0GYaqEBXzTPbCYF0aSWzvR13HvlblI59Xnl0qQQzuHFkxwAuYXgATj1hjTYPp535l13\\\\\\\/qpub8K7BNmDXfNlGazVAg9\\\\\\\/gXDcm7jQi1YNkgqK4BquZiEZpUi49o9ZxP2elBa2SSPBgoddlZ1Nf+30kB0m3r8ACTZ\\\\\\\/lSrC\\\\\\\/eMQn4ulemU5X4AklRcyEaxx6HMJ3i7E2dIe3acSxsm1GOASh3kdFZ4SfkoZlbngTq8o\\\\\\\/i9QzxsGDNyvC9JaROXmOTHpmcIq9G3tepNZLTr\\\\\\\/cM4bc\\\\\\\/5ZfpSXF2r12BYIFYnLsqJguMAlCQzTHfGy1QEw80fDonkhSrE8lwVIOwfatR7F6cR30WbcAbD84AkSSWjqfeC+zqJiDrWKo8PD4i+KnEv0\\\\\\\/+1+unUS1k4Eqw1JUM9xzhwDnkU7WyaIttUkNFY79U66cGe\\\\\\\/lybOGk69yoqoHCfy4ugPy7sejF3U8c69akylPuTJCzItuwupaTq7Dg3NdlCgnCDUlm9mGTckzVgFMDkr0QLzihxltIvPyPZy6AW2UIXHLUZq3H7l4DhL7k32HrUldlsbW5LLBgGtT6kk7wq1prdke5bcAqxy1MOBASMM8P3HgmjafGOxKD2BfmtKjTOvaCWzngxvIMBhfxONFrlJveEZfQZfA5P5XjZdO\\\\\\\/XoGnMAf0bt2vqy1ZwaJFO2OSMp5H5XScTW\\\\\\\/orsLGXAv7\\\\\\\/30tultFrm5NRE+wnmslbdaC5exmN1d6ejzIgXCm7GHJyCT3vc\\\\\\\/RDrc5m35+aTD3xNK3XGa1gG2WvIHRcj7UpprGEQE5644vFQsgKBqEE7Jm0J848h+EEwAaPBkcBjDopbB+1uLygaIswv+tX\\\\\\\/lppQXNFFjky9xTCc3NsqGL2lXmGSbMvEfESKUGJ00MyckHb9vN\\\\\\\/A5IJJDpSu69ba1M72JXnqX2KdVyR91H0VmyX6LpsMHtVCFVhQqn8k3r0TXVsIMn3NvRpHB9fgGKVqtd99lWU8c9tga26+ybYYE5LqBOVtQ4STuum0Jwoe9AhpIdNdGH8Sxftb8WtM43woleHpls\\\\\\\/TKo\\\\\\\/4qtvC\\\\\\\/6aIOcGvD3JcNuzH5MtXYuiBOuSLJujVh\\\\\\\/ZOXqBdN4HqUD8p9Yp67TU96V\\\\\\\/A\\\\\\\/k9qcFB3oBYqtUADg5N20vnXHF84hjZIu0FZLLtAi4hpXBpELpfM8rcgogT7oGnnkFxCSGlfUV9B\\\\\\\/JLzI1bdbUk\\\\\\\/\\\\\\\/L+2nI4dGyhPbmY+r\\\\\\\/A7jBKoEfwel2nXrHwhTVLJmI0j4gPN1Vyp+vU5sGhmBBg5hSFtnG58+1aUwyfe1dRbbogUu9PHO\\\\\\\/0sqA50ugGKHNukgifrxDutnb2fvbeCzNM5mGR5+BcJ7D3hw+SYNb8+Iib5JSbdgvBR21CXkARSOFJKJfXtFExflmUwNyLvcCODyQ5sZbwvOCsD7A9GxqXZNRfduI9aUkNwRZOWoHMWpEL3N9iC7hf8GSOL31QhNAdb00yD5gdpnit2ZieYGLdtVJIYaGLe5lbQIikOwWrYwv5QweVcWYeQOZcJjsyA+rhY9uHzJdI7ME4XLSHfVq2AU\\\\\\\/WHZzlbhrVSJFslUia6KtgCmgaXdPZTn4MsVRf+ANPRjUGs3Ge4YXp2lUdJUAbotinlzeUvCnq6RZ0Y1om7XLoE8bkNq8\\\\\\\/2SYC90qIZ8w23fS\\\\\\\/Xnquzs7NlquAPgSZstuEpRt2feo3giFoom9uAbj9A+BpSgwj1VhlMLNCGu6xoVabeEOHYD3VKUYg625MQ4dNTzMa6hWrsiSVjquO+lyDWgaWnoUjhqVN4cC6D1L8OTzDWs1WtVzcJD3T6SfnIFUfKjpCaNTHaPzCrVnvQuOliqSBAlAlHPviBty9gXIfy61sbbmPcsypHoXGu\\\\\\\/LSuyMm\\\\\\\/pH0XQY287XCZKSU6ZP5Ak7pirQ0zErvWIgM7bK6jzBv9HFQQL\\\\\\\/hTu0+eV51LWEJ2f5y8ww+2kdS40cfXh0xg+9x9SCyle0Trmkm0pJO0ksjk4hbErBxWvN2PZof0N02QhRZJeaQmt1k3fZN79QaIoolPKhK6Tva0kxACKJGM5zLsMW3LhULD3WRHMFl3ZZWG\\\\\\\/FOXweYDiwnRInRYU8f5ygu7Tj6aq7Fb+yiuoYVdpqGuzf7ANF8jiVdTyru\\\\\\\/427NeMr8DK2kTfw90\\\\\\\/92AHauqJlv9uhKGNIuaNUTJG4feUP3Syr2UKKQ1gwrPNdHVCaP6uGi6YC9BF5IkdAsGsa\\\\\\\/BBfojt6PyERzeUMj\\\\\\\/\\\\\\\/gxOC2+tPQ9SpjjYTOfe8I0zJmdHLtnlCzQimPj64IikvZlwjs7K+tfr3GkIfU9DuasdrlUCCWw+xqxshD3xehBrNdhc1UMY\\\\\\\/ze\\\\\\\/rPIrdKVTC4C5BeynEtrH+v0LgpsQ9rF1\\\\\\\/tA+j+cQ4Fr8oQuk\\\\\\\/suU75IOwlGCeoByiup8ckATbXvzv6YyN\\\\\\\/N00\\\\\\\/W4qlHx0GjlIP9q558OlJzgWw3LGA3f8RdckzVKjKYaqdGbaywLnvWUgMQBxOuo+WpfynTeg2o6if\\\\\\\/uI5EUXQt\\\\\\\/K\\\\\\\/8gHXk1dzHUWQg4Ng1NOzmWOSla\\\\\\\/MOr\\\\\\\/r8zRVdjl9OC2rDi\\\\\\\/R3HPIAUPXJZWgS\\\\\\\/j\\\\\\\/uD9d+BaDmAml1y+Tq+y1os8SZ0JPnXKewdrqlg8GBDsTlGUPocAUVzBVfV3ve+NIpw+ialr6rocvBURvZwHjZW9aozXFlaNVzNhi4RMMawiXqou+vsOex5sSRLt1fKpTL4TzFJj9R2mMawKiOULHak0zbRuk9HhLRkMVqicOmNtjRDJODZgEreuLwEtfKJCfQbIpG6OdZZ4hJazMpKh98FdKQ5k7abcjqZzw46HfbnJY6Ir+Gno5tckxAjTyW0h3vre6CoyfONBDhBlXz1rioXfwYUvUsmESZdAjrDbFRF\\\\\\\/NOxe5lx1NuIpKyA4XgjyD7aAhBbE3Kc7f5oKmuGgowB+OPo71yHrF\\\\\\\/Aze9LJ\\\\\\\/5BX+URuM\\\\\\\/o9ZEL1iyVFDD9oh53Q0Ewra\\\\\\\/YSvwaKh4LEe\\\\\\\/xt79z0it03DEf6DbpuSIEezMyC\\\\\\\/GWl+Cp4GKD7tQZyW\\\\\\\/e\\\\\\\/cF8CzyQRyakEJfs0h1QUuWfAnTxRciBDY2Ry0bimAIxniLP6d1tSE5SL\\\\\\\/hV+dWgake4i0DND7qedlOgrWN+\\\\\\\/LHU6Q6AU82\\\\\\\/IpPG+uGVJAv\\\\\\\/I\\\\\\\/vZ4gmkr1oZECEotMOuJzUNM\\\\\\\/o9jYQXw2YRBKoV3gdMDt9Uh9YX1XkBPpbh0f11mNCu8vTkWKD4Pnyo7RvYvqB4PAP\\\\\\\/ztbbfZTnXsAEnVYltJs5yVz3PiXgP3FLO5gj+0yDfzYRy4zPQzbJty2mwLAVoJcObCrx\\\\\\\/\\\\\\\/3Zd+0xw9zY3kjl6ZRL9mLd7Yd37f+oohva\\\\\\\/L7T1ZReQcrX6fPnLJKGea4MsdUL4t00Pfq+dkTi1x+7in88nXXNFF6JoLd480BLvPtqYa2VeDF7DJfMW6EWVEmM\\\\\\\/ZQ0b3SL1V8sV7lsD5KG9\\\\\\\/Bm2VZjo0r4cuFY90nXmCXuqCEKOpdL+wcdCKWunjAOiTw1IRBaSAkhhQvmvPlx3zmkdbbfh3s22ukDSZY3fJBLGSfb6III7BxGVugxzv43uqyqE6l6bqSJazKcGU+KRw5i3SmrBuXcYaxUNaBHKs71FVHGlm2fD3bJ0\\\\\\\/9r+lfj38+veIKMHw7V0sRDJ9qWS064FCVrKOR1JXa9klqN9oWOc5pXcGAy74hR6gS0X66BR6o+jqi4Aj+M1ieDDktu9n9fgOKgKX06PNQKS25nTGj7zPR8oIMDGgp5XSUrh5xJ1aXNDsxmRUAGP0fM9C2CNG5DPUixGHwa5Nhtv8bNtB61hJdiBUMRz6Py7QXQlgiNcW3i\\\\\\\/2L+nUd6477F0PqYYKYzTcRntro620iGg\\\\\\\/BGWUPjtjrkvy2FpQxXbgmxJDpLdo56hvxFkeqGIIxS8pOXjwJIg9V8YkWjlQ\\\\\\\/4vXikWem8DqtBU\\\\\\\/mjAlI72VJkDA+ZC2bc1jxJYNcE6J3ab9VX\\\\\\\/cKbSAA5gZ2+BUtK0oYC1qIaa2qsQUYSkSjEEhOhTEqeUNNDw3gBzQnbf84W+hOPOZvCz21qjRWc0JOIQqTnWEWvpRtakcAuqd8bgUFgxuxlG+zYPNg5u9XKPlDbgaCIzXabfbrOkJdqCyGB7YMs6vmjhXmAlty9yalIkgCB11KvL3JaXDVM\\\\\\\/uA59RMOh60HQi27CLdF\\\\\\\/tuymr+ki4nLPNZLZ2nAJAG9Lu8LwlryLnZascz0UQDnH6da0RoMyVuZ7Xk6+doEWD8TgvwJSVSAJKLo3W4BmnYpJS6zaw9OMZXQ1K5F8BYwEIldpPVpmqsBBgkOAffKXXqftdw8UcElo8WhmjWsfDkxF5A\\\\\\\/9oY3OkDm2t1WhvEy\\\\\\\/q\\\\\\\/54\\\\\\\/7f7g1ELl6W+4irkQCe5SkxdFtJocykdqPSbBjRM9DJ7xiX9J+up3tR2m3bJZQdgzHcE4gy9\\\\\\\/NqXa7aK2KVc2aRywOA+EhRa0MBP5YDHOUa20MEKRqRTzDX4ypM9LOhHKNwBOvdp5RDIqbr5dDGRTDjPh3IA9Ro3XQieEJuXyfnU+LfEuLqqZmDeXSM3gjzKgfp+aYUEaisHPNBwc7tD0T8TWeI6XD04mAHAaRa1GSH+gkOCIoSymRHPckhJKP5GBXsJIkK+mDDMsn1BuF39zFkMHkXlx+mEH63QJrOr959AiJvaTecZPAK2EMXA+ZQ5QqDNsrF7Vvvsk4tVHGde3YYx1iJs0bfkswvyv2f8Ul4Tsk5s8+SqgZ3\\\\\\\/qrtuaToKmaNWHloVLjYq4ReT0d4A5EbXzAeHqC3FLW7AZ9h4VORKjsy7Bqe0dKZSgz7y54YEV+dahlLX\\\\\\\/p8w8wjcD35uFE7MAs0415glpPIxbcIx9+JL59L4cp53AFlP\\\\\\\/r\\\\\\\/CGw\\\\\\\/IQ6mkhqqNUxST32WjfD6LMoF45Ehkd6GnvMAbiNY7YNM\\\\\\\/+LAc8BDo2BhvKlGizyGMntqTvV\\\\\\\/AFRrwCzZey7DFIVcKhQ\\\\\\\/zccsWA\\\\\\\/qFkOTLgnVwWyMwgvX8pIhy0P+JT0\\\\\\\/Pf2A0vMUpueSQgeLaWfcZeprAbETFrvxWCGsdCq98HonbUsJKdRsxZjswD9LVbE01bpxFJKYLwKqczy2ue4Z7lZrJ00o\\\\\\\/Mcl+XwRdRX6my9Cq9tKDlaYjm6fjmod4eakufgZRla\\\\\\\/2vocmgt4Teu3DqEp5tZG+gxHisXjOx2p1uAJeAzvuEZjBZ64gZbN1WgNibgglyBp21cqM\\\\\\\/2lSoAbiIA2RzOSdbi3RYUfAngCfDwaJlTywQB9hCPV2jamhmJxdGRNVadB5FE\\\\\\\/pJvLMSr\\\\\\\/+T1vWHxqQ03aBf3MU+y\\\\\\\/FEwXvnCmENGCvNsJi374V3OCVWfvkS+QjwvJd7PbWgD6X1vRhMhkA14AZ\\\\\\\/Oi+RZ2s4Ki+Tk2pw9Vr68Ig8eUrece1A5+w1sKSuSEs3Awbs8iuTTMjZu8jFc8xBH3N3RcelxWkKUCqnpQdmSSHJwQrHiyAwBS85xs\\\\\\\/LVgA6+5ldDQLuO3WGrHR3SqpEmV1gBwJMs0xDWa9KVtyvxtYzIizkp4mihk1yph20WP9yj3jMtDtM8QPcQsXtPdJ6QgLrALv9gYhvfCJ1yASTkDkgU2FfWUbS8gVlmmKf2+Cwi9o77R7VcTXZTiv8qICssGlR200ewjke5iHt3WrZNpZMH5sspPnGvpBop5MZr\\\\\\\/gbTeW0KnWLvxwrWIB3GE6c430X15OUuddafs7uW1x4qUbZNxDVuZLhNCjMsND+UIVuO3pO0xmTmDA93M7DYS\\\\\\\/7ApwVF\\\\\\\/uIRpxm4WuLY+1I6xl6Ia0PedND0TGsvGlh4ux5\\\\\\\/SkDoSz6YLW5EegU+R39fGqvEzQIsWQ0297t6yvyV6xZDQbl8HiYcUepJcgzIRruITw2Ey9vLLNFia7clrsK2OQ5d8RX+wVs5Dom3LsqyJrBmV1G3zI9uYZ8q4pvFXj9buAE2NQz33d1Kw06oGo2AlFfPCtuFsDXj4wPX6HnnDlXlif5jtzNKBtJ38qTBJpmy7TRv0jx2i9tLYDf5xTOP18vNBo8b8xO2hdtUi2Q3kF+ER543s1BJMeGnEgExExKymjQKKmvTZ0mWwJzZq9+TzFaAHVNgrOpX+4nRALjiers5B2B3SFElPGPBOPM22p7XhwjQoXEM33wywsv6zlJ7VWm3S07vdUL4nNAxdVSDcBUg8xwKzDsyB8p8JBepkGRdPft3KLSkW0EHXlWlWomNybqPUCEV31RMsuxMBGmt1vyx2lnRXdN0mzOxTORfifLKWRrkT9bOpn6EC6JUzm5H+3pOI6S3L9okHH\\\\\\\/BJyp0zYpjyOldv1hRFVFgTlp\\\\\\\/xQsHrAADQ6ijJb2KMeHgkUbDEM63IDOU\\\\\\\/hhVBUItYHr5AmqL1pOR0kGj\\\\\\\/d0nprisJ1LJpM61a8olwMAnve5LK7x31zHjkF4SQdZfLkXAdJWulky5bmrg5e\\\\\\\/+AIegzcHgLmxr\\\\\\\/C1yUr8cde4oTfcEr4mDCQlhKt0Vwkb7\\\\\\\/SlTWo7ZMgbBBk\\\\\\\/JgKYurApcwfQqcsecIeUUYcOTO8nrc5Qqol4qru2i+U9L3mslW3V2jNl3E6dQ\\\\\\\/NFdoium2TRFSWMykHg2zbuxEAs9OqJvfzAX5vPtZ+bxoTinkD3oBFMb2mGv3kuRR07dILbYxta9fpl0a1IKbSNp74gGwGMQIpStA\\\\\\\/6kQXBrVTqVPro6f+Z0hjZ0GJciRuYu+pnuddygW7+MGY+Wc5tmV9OMAILOparb2VDFv4eqLvR1untwoPe3kuFMEFrfXrAXECBXYg9oHmlnhAUWvnmVur7FtfseCJjWmn+ERyPa4xWyivyc\\\\\\\/b\\\\\\\/DiGVWKiLwu2b4vHHW\\\\\\\/ii7SNxbZlwj08YDLDiUQ7ihCejFddrqqKuJWYuH2LW8IOSa1V07cMepe8lNXMBz+j3g7ht+QN+Nu75i3xZdC8tG9LfrhJKhD42JgW91w+q1orhcX0nf5RP3D\\\\\\\/5QqRzAt2GmTqY01t1NKOD8jQ2xHl3Wrt8e1C6qlaRhe\\\\\\\/UcYAxyzMb0622ygS7GwaAR2f92OW7sLyprvSYNDU5iOAjkW0\\\\\\\/ax0BH19cYLu7Hqe6KJxawwrgT2ITgC1kqPcmNKmxZzYQ9gZhTn5oUiNSQCtOk7SysURz1XMHQk4jMFaypWNaT0\\\\\\\/N21bw6HTX0nPLzlKeBdXNazm3+MqGKP+g1GbF\\\\\\\/xUzpZCa+fRj66X8Fwrb0i2sRgPVj6m6amLHPaRxhsDbbKZK5+HYPaAzqVYesedwDoAbKT5mpvE83aye6bFPSqpqstcdzOC1C6UPNmLK0fzv252998C6hJrLlyR2wWadcf0MyENBupajkuOLeMiUG8oLi9xrfN9cWvJ3e0BFmsO\\\\\\\/dsAy7mQZOSvSCziOLddqz3nFqj8kakYmjHqE5qeDgHJSUZfCIJxDtEXyxZ0ZUai4IDeFS84wkzmfQ4DSZJYuLYJIydfxQibIAMJt2qIPMHfYXcyCcDF40b1My7r6iqXsZbhUc9FJVEghezMiKic\\\\\\\/nnJz4wYGyK8Y22x7oSoc+Jn\\\\\\\/BrjHGSjPAPOKJaO9siqHnWUamabEm5p5+t5XMq7MGO3VJy3mXZctOxnRtGP+nBPAe4oNHr+Uy8wohN0pEPJuuvnzQIipTspfyB92D1POEktGQyWNT\\\\\\\/kH1hu9opGRzxl6dSfDz78NBIfgG\\\\\\\/JL4z0cKgatI9cG4mdX1teH2i9JLbBG1PFpCB\\\\\\\/EjinY\\\\\\\/6UFP0a0Z8DSYw46\\\\\\\/od9Wfh9wlSlqxBd6WqSK8d7o5mGXOKCkJjt0G5yfx7wzKDLx\\\\\\\/FX8DI5Du3nPtCknlSKn49SNBtN00sNX\\\\\\\/1b9fSCfHwQsIf9BhAv8Cxoov3ls9+sQ4Y88tcqpstrBH9tGmIU+jLYgkTMg+Ypocex5gu5I5l3hysUtfGADAvLsqszo17xOIbY14Gih56opisR\\\\\\\/\\\\\\\/9WgUDtOfNqk\\\\\\\/\\\\\\\/cxklcpBtnRIymFnhhfCuxPyw3xVWL\\\\\\\/xIFvf\\\\\\\/6Z1YNYYD7J\\\\\\\/\\\\\\\/0MB2YCi8gUeBTG5xUsL9aYxD8PE09cX81l7WZiG2LDECOu5VF7nNYt461nHugVqDUROxwrX1yhEkfs0Kp7X34dQ2+rfzh88qOPM84Cx6zWvZdU9CtnXKTPwIg83pWAMYUinjUYXX1AGpBf4nSDPgR7RvNHkj7+9YB4veMko2MlvWHV95QNWtwSdWjYzIXMm1dQDF+XfqqmhQlxjnIamZBmO2KyWGjdW8ook2eVLBHtyau2ZWl3q3FVaLA\\\\\\\/XWLkn8E1JBwWfwDOOuTapLCOwxHxwU4ws8mF\\\\\\\/uIogkrVjlXSm4C58cxr8fJdoea8EREMYRAUB4bWem0qSXQkDrWmcr0jsqJlor6haIKVhr4ytbka5PbMVVVgTkeW5XfpLpfdjUxXc8\\\\\\\/cJF6D24033z\\\\\\\/Iz9ntQ+H06Q=\\\u0022,\\\u0022iv\\\u0022:\\\u0022FzgNvusHbh4wkdAe8DyKsQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002254f53179f9b159d2567496b7f7d0226fbeb496a1fbc9173b0925621c0d829ca7\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022XjGgER58WEiupLcZs+SSoxDNsizZIWb78hslS0knA0TN7OjsiP6bh9AXyFuEa1sx74EVdVlU4SjDZmAiuZpMh\\\\\\\/GA3SiUc7mtwkYVmC7V16okcjwH6hGMCuZBwR8gTLzdBBlmj8UzHv3bPVG4eqaCOwH4PJSLdPMFEBiBiUfePEQ2Sg\\\\\\\/aCtwtws9+DiJwQHZc9I5Okm5S\\\\\\\/5aV6RaYaJ8QP+RFL58Xwq4HHJSFCBManHXCqXb0l3UqEWkGkxKlT3Ci+Sh1B\\\\\\\/5krcGHQX10b+qH5qUEXbmYxc\\\\\\\/dYQpEBbnENWNdeyWtNgnAW2iJo\\\\\\\/lyvb\\\\\\\/7nLskR7wV69K3Y0vsWUWrNAUDTfAqp4v5vd5p8Lx2EjeCdvyeNJBLSGlOLWKmqiwBs8eYcVVpWtPCnSeZTD9M6eYFBGMKqNfDe+iwlQfbXZrjbuDDj9oh12v+r3IHvvwNxElhzpZJmc8lrPWqEwYiOouVr5DODi2LVC1Cdj6u1bku5cnTTRsB+rDEl6iO+jUn1hPoIii6Hpyne4WD01e606iMHZJgCe+2T1zjbdstXKV59QqPAZUzvwJfKkJTLBz2aYKpMdtD1wOuLinHawtWLvSrUJLdtPBR3r3WqULv44JgIuNzEYXMPM1Ha+eoWoufmpQVpHEdVsUHKTEfGPEHC2N188Kx7j8HxYqaAJb5ko1WPyjpNhA8qT\\\\\\\/YKx2S4DpzKnJ2nvv1OcnRr4GznSRij2A3eo4XwrblOz1fcyTJ1t9qCHxQzoEIL6\\\\\\\/bg9efBvRrcW8HKlpnH\\\\\\\/PR+rzM\\\\\\\/N1G9xNkX6zv62wh6bEAcAMn7URBj7EGRkZPeD4t56Qc5Ju3OiygA9iWTpJ1HHbJnvBNwkBrCTY967t63JU3jYy4bYjQF\\\\\\\/gUteER4JV8k4omf5WYGhhTS1dN1dfYGIshVeiFslvzS4SbsccGKzP\\\\\\\/ZAwQwGaNj0mDP+y8d80W24X5aHGEs+aQkfN8AlxXrsSBhtbIU9kX+j7vOFEow8CJaoGMTSxnlxCVAO2ksvMf+X5xCWFBG58TjNrdmc55f3vaLqXytJPs8JJMeq9QyXz8LzpcM6gbFFh7CyBrHdK0y\\\\\\\/iQ+QSX0KYyBQvW1iFiCjM20cTJG5Tjrcx+b5CFHZbNKTFhK6ldUiQxCv7Y4f3lOaMu4d2LJ5Co2Qb2zxYXWRw1NKpxqRk76iwPRmIUxFeD42iu9oLrFsMeOeFZo+5BIcqLdPqPvstU2hoOlT+2KMXG2r8XJ9wTQSfB6H8WMSL\\\\\\\/WV\\\\\\\/F2WydPYTRhEfsAtXDCHlQnmZ8kx4H2fWdgv674\\\\\\\/XcBr7agXyNK1lhmUdbBngNuxhKmYluqwDIdF\\\\\\\/T3tSjXrOBd9eDl7TYirriSCrsEH1dsyT+eU+XDcoJ9JPOI35WtSKeNtj5JA5SGLq6TMICLgH6o38ugjX4bfDuEVBJlTDb5JqkugTkN6EAak7GMqtk7raerjL\\\\\\\/ApUBk9fQuIiz9p1FNMZrLLJ3V0RRlLafSRiRqc2DqRfqAoQfmUNqUobwk2UFZBz5h3Aymfx+zncnZ45yz88adHsB6ejPl\\\\\\\/qqzouWRqx5Bn6g1Htc8R5xKJzwBcT+6C9moiHDFFIUsJ5gBdOv3wX2E1T6BeuLVCLiewtwO9kseF8q8gySQh2OFNzDShnLtQkcFCBOfHUdMLz\\\\\\\/vXQl\\\\\\\/K9klYoAEHsRVvHDsbwe4gAC5Tgcz8lka2oqQUcKofxr0mdpuq0ungPejjjH7GTgA9oGVrUVjh6B9iqo1jhNwdoiDiL4qyiZn207hHPDMgcjxhDQu8OkT8\\\\\\\/QXnCdn9fvx+3CwgCT0f6z6QqNRL+80pDvUjaMMHqdsaJVwynYH9hcPHkvJC8MvUwdZ5yN+9xK6dNIM6XH3u63z0Ar911aFvC4uhVY97YddM13tfnpmJJXEokZt6ttrM8ccI3b6CFjPSmmqjQ528V1WLNNNtFEyt\\\\\\\/qdOIZzVToNRUeDMRr0khBBwhmY4vj\\\\\\\/z6Mb36RhyHAjjFdOF8hxcGJCGT\\\\\\\/FsrfMscl7VOU1fvmn76RGNWJqK07O9mGyqkfE9NfT\\\\\\\/gODBapBaAY3EXechgVTZohSRY51+ctZAeqFHS\\\\\\\/nYqedvV\\\\\\\/KAgAOBioEhyqgby\\\\\\\/DikcOHH7UdQ7ZrGykdlo2eTloGTmxZhOlQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022xyE7ldAGCWG+ZY+FK1mZtw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e67334b14dd4d30c0de2eb11b2af1843e862bad39a52d0bcf21423c23e4fa7c1\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-490722 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/9/2953" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-37kuTA5bLpTcrCWBrSUl2_DeXiV49zUEPVt1dWyTtc4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/9/2953" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-IbWsvaSEQPzWk4XHkzHSxPosZdAs3dGGPjJtUqou3xU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/9/2941" title="Acute Effects of Drugs on Caenorhabditis elegans Movement Reveal Complex Responses and Plasticity" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/9/2967" title="Whole Genome Sequencing, de Novo Assembly and Phenotypic Profiling for the New Budding Yeast Species Saccharomyces jurei" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490722" id="node490722" data-pisa="ggg;8/9/2953" data-pisa-master="ggg;g3.118.200462" data-apath="/ggg/8/9/2953.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Complex Haplotypes of <em>GSTM1</em> Gene Deletions Harbor Signatures of a Selective Sweep in East Asian Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8794-3217" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>M. Saitou</span>, <span class="highwire-citation-author" data-delta="1">Y. Satta</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-4371-679X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>O. Gokcumen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2953-2966;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200462 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490722" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">M. Saitou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Dept. of Biological Sciences, SUNY at Buffalo, NY, 14260-1300</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Dept. of Biological Sciences, The University of Tokyo, Tokyo, Japan, 113-0033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BSaitou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Saitou%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AM.%2BSaitou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8794-3217" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for M. Saitou</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Y. Satta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Dept. of Evolutionary Studies of Biosystems, SOKENDAI (Graduate University for Advanced Studies), Kanagawa, Japan 240-0193</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Y.%2BSatta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Satta%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AY.%2BSatta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">O. Gokcumen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Dept. of Biological Sciences, SUNY at Buffalo, NY, 14260-1300</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=O.%2BGokcumen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gokcumen%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AO.%2BGokcumen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4371-679X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for O. Gokcumen</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:omergokc@buffalo.edu" class="" data-icon-position="" data-hide-link-title="0">omergokc@buffalo.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/9/2953" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490722" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:490722/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/9/2953.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490722" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:490722/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/9/2953.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:490722" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:490722/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/9/2953.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490722" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:490722/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/9/2953.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490722" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:490722/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The deletion of the metabolizing Glutathione S-transferase Mu 1 (<em>GSTM1</em>) gene has been associated with multiple cancers, metabolic and autoimmune disorders, as well as drug response. It is unusually common, with allele frequency reaching up to 75% in some human populations. Such high allele frequency of a derived allele with apparent impact on an otherwise conserved gene is a rare phenomenon. To investigate the evolutionary history of this locus, we analyzed 310 genomes using population genetics tools. Our analysis revealed a surprising lack of linkage disequilibrium between the deletion and the flanking single nucleotide variants in this locus. Tests that measure extended homozygosity and rapid change in allele frequency revealed signatures of an incomplete sweep in the locus. Using empirical approaches, we identified the <em>Tanuki</em> haplogroup, which carries the <em>GSTM1</em> deletion and is found in approximately 70% of East Asian chromosomes. This haplogroup has rapidly increased in frequency in East Asian populations, contributing to a high population differentiation among continental human groups. We showed that extended homozygosity and population differentiation for this haplogroup is incompatible with simulated neutral expectations in East Asian populations. In parallel, we found that the <em>Tanuki</em> haplogroup is significantly associated with the expression levels of other <em>GSTM</em> genes. Collectively, our results suggest that standing variation in this locus has likely undergone an incomplete sweep in East Asia with regulatory impact on multiple <em>GSTM</em> genes. Our study provides the necessary framework for further studies to elucidate the evolutionary reasons that maintain disease-susceptibility variants in the <em>GSTM1</em> locus.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/copy-number-variants" class="hw-term hw-article-keyword hw-article-keyword-copy-number-variants" rel="nofollow">copy number variants</a></li><li class="kwd"><a href="/keyword/structural-variation" class="hw-term hw-article-keyword hw-article-keyword-structural-variation" rel="nofollow">structural variation</a></li><li class="kwd"><a href="/keyword/cellular-detoxification" class="hw-term hw-article-keyword hw-article-keyword-cellular-detoxification" rel="nofollow">cellular detoxification</a></li><li class="kwd"><a href="/keyword/natural-selection" class="hw-term hw-article-keyword hw-article-keyword-natural-selection" rel="nofollow">natural selection</a></li><li class="kwd"><a href="/keyword/soft-sweep" class="hw-term hw-article-keyword hw-article-keyword-soft-sweep" rel="nofollow">soft sweep</a></li></ul><div class="section other1" id="sec-1"><p id="p-3">Here, we describe the evolutionary forces that shape the variation in the <em>GSTM1</em> gene, which has been shown to be an important metabolism gene and associated with bladder cancer, autoimmune disorders and response to different drugs. Our results reveal a genetic type common in East Asian populations that may have important evolutionary and biomedical implications.</p><p id="p-4">Thousands of structural variants (SVs, <em>i.e.</em>, deletions, duplications, inversions and translocations of large segments of DNA) comprise a major part of genetic variation among humans (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Redon <em>et al.</em> 2006</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Sudmant <em>et al.</em> 2015b</a>). Several studies have shown that common SVs can have important functional effects, contributing to both normal and pathogenic phenotypic variation (<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Zhang <em>et al.</em> 2009</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Weischenfeldt <em>et al.</em> 2013</a>). In addition, there were multiple recent studies showing adaptive phenotypes that are underlain by SVs in humans (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Perry <em>et al.</em> 2007</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Girirajan <em>et al.</em> 2011</a>). However, there is still a major gap in our understanding of the functional and evolutionary impact of SVs due to three interrelated factors. First, multiple mutational mechanisms lead to SV formation, affecting the size and sequence content of the resulting variants, as well as the rate of their formation (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hastings <em>et al.</em> 2009</a>). Second, most of the functionally relevant SVs are in complex, segmental duplication regions (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Feuk <em>et al.</em> 2006</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Marques-Bonet <em>et al.</em> 2009</a>). This genomic context complicates discovery and genotyping of SVs themselves and also leads to higher than usual false negative and false positive single nucleotide variation calls in those regions. Third, the segmental duplication regions also lead to frequent non-allelic homologous recombination events, leading to the formation of new SVs. Linkage disequilibrium-based imputation is essential for many evolutionary genetic analyses (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Crisci <em>et al.</em> 2013</a>) and genome-wide association studies (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Visscher <em>et al.</em> 2012</a>). Recurrent SVs and sequence exchange between homologous sequences along with recombination break the linkage disequilibrium in segmental duplication-rich loci (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Usher <em>et al.</em> 2015</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Boettger <em>et al.</em> 2016</a>). As a result, the majority of SVs cannot be imputed accurately using “tag” single nucleotide polymorphisms (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Wellcome Trust Case Control Consortium <em>et al.</em> 2010</a>) and the standard, haplotype-based neutrality tests cannot be directly conducted.</p><p id="p-5">To overcome these issues, recent studies have focused on resolving complex haplotype architectures harboring SVs in a locus-specific manner. For example, the reassessment of SVs in the haptoglobin locus revealed recurrent exonic deletions that are associated with blood cholesterol levels (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Boettger <em>et al.</em> 2016</a>). Similarly, the reconstruction of the haplotype-level variation in the <em>GYPA</em> and <em>GYPB</em> (Glycophorin A and Glycophorin B) locus has revealed that a specific SV leads to resistance to malaria in African populations (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Leffler <em>et al.</em> 2017</a>). To highlight some of the negative results, careful reconstruction of the haplotype-level variation in salivary <em>Amylase</em> (<a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Usher <em>et al.</em> 2015</a>) showed that previous associations in locus with obesity may be false. Similarly, a haplotype level analysis of salivary <em>MUC7</em> gene showed that previous association of this locus to asthma susceptibility was likely spurious but found surprising evidence of archaic hominin introgression in Africa affecting the structural variation in this locus (<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Xu <em>et al.</em> 2017b</a>). Overall, with the availability of population-level genome variation datasets, it is possible to scrutinize the evolutionary and functional impact of SVs residing in complex regions of the genome.</p><p id="p-6">Within this context, we focused on resolving the mechanisms and evolutionary forces that lead to the common haplotypes that carry the deletion of the glutathione S-transferase mu 1 (<em>GSTM1</em>) gene in humans (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). <em>GSTM1</em> is a member of the <em>GST</em> gene family, which code for cellular detoxifying enzymes and studied intensively within cancer biology and pharmacogenomics context (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Parl 2005</a>). However, these enzymes are functionally relevant to human evolution as well. It has already been shown that they contribute to cellular detoxification of naturally existing products (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hayes <em>et al.</em> 2005</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">McIlwain <em>et al.</em> 2006</a>). For example, GST enzymes metabolize sulforaphane, which is an abundant product in cruciferous vegetables with unknown metabolic effects (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Hayes <em>et al.</em> 2005</a>). Similarly, they metabolize prostaglandin, an important naturally existing metabolite and a target of aspirin (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Ricciotti and FitzGerald 2011</a>). Prostaglandin affects multiple evolutionarily relevant processes, including constriction of the blood vessels and platelet aggregation. As such, it is plausible that variation affecting <em>GST</em> gene family may have adaptive effects. In fact, such adaptive evolution in humans has been documented for variation affecting other metabolizing genes. These examples include but not limited to the deletion of the UDP Glucuronosyltransferase Family 2 Member B17 (<em>UGT2B17)</em> (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Xue <em>et al.</em> 2008</a>), and single nucleotide variation affecting other metabolizing genes, such as N-acetyltransferase 2 <em>(NAT2)</em> (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Mortensen <em>et al.</em> 2011</a>), alcohol dehydrogenase (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Sheeley and McAllister 2008</a>), and Glucose-6-phosphate dehydrogenase (<em>G6PD)</em> (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Ruwende <em>et al.</em> 1995</a>). Therefore, it is not a stretch to imagine that the members of the metabolizing functions of <em>GST</em> gene family are adaptively relevant. Within that context, the deletion of <em>GSTM1</em> and other variants in this locus may have a broader effect on the neighboring <em>GSTM</em> gene family, members of which have similar metabolizing functions (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Hayes <em>et al.</em> 2005</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2953/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The formation of the GSTM1 deleted allele and its geographical distribution. Two highly-similar segmental duplications fuse each other and form the deleted allele (upper). The geographical distribution of the GSTM1 deletion allele (red) is shown in the lower figure. The frequency data are from the 1000 Genomes Project phase 3 database (Sudmant et al. 2015a)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1769612826" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The formation of the &lt;em&gt;GSTM1&lt;/em&gt; deleted allele and its geographical distribution. Two highly-similar segmental duplications fuse each other and form the deleted allele (upper). The geographical distribution of the &lt;em&gt;GSTM1&lt;/em&gt; deletion allele (red) is shown in the lower figure. The frequency data are from the 1000 Genomes Project phase 3 database (Sudmant &lt;em&gt;et al.&lt;/em&gt; 2015a).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2953/F1.medium.gif" width="440" height="355"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/9/2953/F1.medium.gif" width="440" height="355"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2953/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2953/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490746" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-7" class="first-child">The formation of the <em>GSTM1</em> deleted allele and its geographical distribution. Two highly-similar segmental duplications fuse each other and form the deleted allele (upper). The geographical distribution of the <em>GSTM1</em> deletion allele (red) is shown in the lower figure. The frequency data are from the 1000 Genomes Project phase 3 database (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Sudmant <em>et al.</em> 2015a</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-8">In addition to its functional relevance, the <em>GSTM1</em> gene deletion is one of the most common whole gene deletions observed among human populations (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>) (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Garte <em>et al.</em> 2001</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gaspar <em>et al.</em> 2002</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Saadat 2007</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Buchard <em>et al.</em> 2007</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fujihara <em>et al.</em> 2009</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Piacentini <em>et al.</em> 2011</a>). A whole gene deletion leads to the lack of the corresponding protein and thus it is thought to be deleterious in most cases. Therefore, it is rare for whole gene deletions to be at high frequencies (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Conrad <em>et al.</em> 2010</a>; <a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Sudmant <em>et al.</em> 2015b</a>). Nevertheless, the average global frequency of Eurasians with homozygous <em>GSTM1</em> deletion is higher than 50% (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Sudmant <em>et al.</em> 2015a</a>), which puts it among the most frequent whole gene deletions observed in Eurasians according to 1000 Genomes Project database, the other being deletions affecting the <em>LCE3BC</em> (Late Cornified Envelope 3B and C) and the <em>UGT2B17</em> (uridine diphosphoglucuronosyltransferase), both of which have been shown to be evolving under non-neutral conditions (<a id="xref-ref-85-2" class="xref-bibr" href="#ref-85">Xue <em>et al.</em> 2008</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Pajic <em>et al.</em> 2016</a>).</p><p id="p-9">Despite the functional and potential evolutionary importance of the <em>GSTM</em> gene family, a clear understanding of the mechanism through which the <em>GSTM1</em> deletion and other variations in this locus actually affect phenotype remains elusive. We believe that understanding the underlying evolutionary forces that shape the variation in this locus will give us a framework to better target the functionally relevant haplotypes. Such an evolutionary analysis has been difficult since the <em>GSTM1</em> gene resides in a complex genomic region with two segmental duplications (SDs) flanking both sides of the gene (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>). This particular genomic architecture predisposes <em>GSTM1</em> to non-allelic homologous recombination events, which can lead to formation of new SVs. In fact, the polymorphic deletion observed in humans was almost certainly a result of such a mechanism (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Xu <em>et al.</em> 1998</a>). Moreover, we recently reported that recurrent deletions of the <em>GSTM1</em> gene have happened independently in the human and chimpanzee lineages and that multiple rearrangements have defined genetic variation in this locus in primates (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Saitou <em>et al.</em> 2018</a>). These complicate the phylogenetic and broader evolutionary analysis immensely. This complication may also explain the paucity of evolutionary analysis in the <em>GSTM1</em> locus, which is otherwise the target of more than 500 publications in the last 20 years, most of which are single-locus association studies implicating variation in this locus in susceptibility to multiple cancers (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Parl 2005</a>).</p><p id="p-10">Collectively, the functional relevance of the <em>GSTM1</em> deletion, its mechanistic complexity, and its high allele frequency in human populations make it suitable as a model for studying the evolution of metabolizing gene families. Therefore, in this paper, we analyzed hundreds of genomes to resolve the haplotypic variation that defines the <em>GSTM1</em> deletion.</p></div><div class="section materials-methods" id="sec-2"><h2 class="">Materials and Methods</h2><div id="sec-3" class="subsection"><h3>Materials and Methods</h3><div id="sec-4" class="subsection"><h4>Study Populations:</h4><p id="p-11">We primarily used data from 3 populations, YRI (Yoruba in Ibadan), CEU (Utah residents with Northern and Western European ancestry), and CHB (Han Chinese in Beijing) available from the 1000 Genome Project (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">1000 Genomes Project Consortium <em>et al.</em> 2012</a>; <a id="xref-ref-71-3" class="xref-bibr" href="#ref-71">Sudmant <em>et al.</em> 2015b</a>). For the phylogenetic analysis, we also analyzed haplotypes of the chimpanzee reference genome (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">The Chimpanzee Sequencing Consortium 2005</a>), the Neanderthal genome (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Prüfer <em>et al.</em> 2014</a>) and Denisovan genome (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Reich <em>et al.</em> 2010</a>).</p></div><div id="sec-5" class="subsection"><h4>Confirmation of the genotyping and phasing in our dataset:</h4><p id="p-12">To validate the accuracy of the genotyping of the <em>GSTM1</em> deletion in 1KGP dataset, we used digital droplet polymerase chain reaction experiments to genotype the <em>GSTM1</em> deletion in 17 humans from 1KGP (samples Available through Coriell Institute for Medical Research) (forward primer: TCGAGGGTGCCATTACATTC; reverse ACTTCTGTCCTGGGTCATTC; probe: /56-FAM/TAGGAGCAG /ZEN/GCAGGTGATGTGAAC /3IABkFQ/). We followed standard protocol provided by Bio-Rad EIF2C1 probe assay (Table S1).</p><p id="p-13">To investigate the accuracy of the 1KGP genotyping of single nucleotide variants, we investigated 74 CHB individuals and 6 sites in the target regions (chr1:110211681-110223007 and chr1:110246810-110255596) reported both in 1KGP and HapMap (this is all the SNPs genotyped in HapMap in the region). Of all the 444 sites, 9 were “NN” in hapmap and 3 showed different results in the two databases. Overall, we found approximately %99.3 concordance between the two datasets.</p><p id="p-14">To validate the accuracy of the phasing in 1KGP project data that we used in this study, we compared haplotypes on 50kb on each side of the deletion in our dataset with phased HapMap genomes. Specifically, we were able to find 322 heterozygous sites in 30 phased HapMap CEU genomes. 320 (∼99.4%) of these are concordant with the 1KGP dataset. We also investigated Sanger sequencing data from seven deleted haplotypes for which we generated long-range PCR amplicons for an earlier study (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Saitou <em>et al.</em> 2018</a>). Since, the samples that were used for Sanger sequencing were not present in 1000 Genomes database, we asked whether common polymorphic sites in 1000 Genomes are also polymorphic in the sequenced haplotypes. Here, we defined common by 30% or higher allele frequency. For the 15 common single nucleotide variants that were reported in the 1000 Genomes project that overlap with the region (<strong>chr1:</strong>110216300-110225620 and <strong>chr1:</strong>110244324-110246143, the deletion was excluded), we found 14 variants among the 7 haplotypes (one example is represented in Figure S1). The only one that was not polymorphic among the seven haplotype dataset was rs61799140. This SNP has been detected by more than a dozen previous studies based on the dbSNP (<a href="https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=61799140">https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=61799140</a>). As such, we concluded that the absence of polymorphism in this location among the sequenced dataset is likely because it was not represented among the seven haplotypes, rather than it being a false-positive genotype in the 1000 Genomes dataset. Overall, we are confident that 1000 Genomes adequately captures the genetic variation in this locus and that the variant calling which our further analysis based on was accurate.</p></div><div id="sec-6" class="subsection"><h4>Linkage disequilibrium and haplotype analyses:</h4><p id="p-15">Vcftools (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Danecek <em>et al.</em> 2011</a>) was used to calculate R<sup>2</sup> values between the <em>GSTM1</em> deletion and flanking single nucleotide variations in Han Chinese in Bejing, China (CHB), Utah residents with Northern and Western European ancestry (CEU) and Yoruba in Ibadan, Nigeria (YRI) populations in 1KGP phase 3 datasets (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>, Figure S2). The Ensembl genome browser (<a href="http://asia.ensembl.org/index.html">http://asia.ensembl.org/index.html</a>) was used to calculate and visualize linkage disequilibrium (LD) between single nucleotide variations and that (or LD) between the deletion and single nucleotide variations. Based on the R<sup>2</sup> values, we used two target regions for the statistical neutrality tests. Two target regions were selected for the following analyses; target1, upstream the deletion (chr1:110211681-110223007 in the hg19) and target2, downstream the deletion (chr1:110246810-110255596 in the hg19), which showed relatively high R<sup>2</sup> with the <em>GSTM1</em> deletion in CHB (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). We constructed a phylogenetic tree of the combination of these two regions by VCFtoTree (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Xu <em>et al.</em> 2017a</a>) and visualized it in Dendroscope (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Huson <em>et al.</em> 2007</a>). For the clustering and the visualization of haplotypes and its cluster analysis, we used the pipeline described in <a id="xref-ref-87-2" class="xref-bibr" href="#ref-87">Xu <em>et al.</em>, (2017b)</a>. The clustering was conducted based only on the haplotypes themselves with neither <em>a priori</em> input of haplogroups nor the deletion status (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2953/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Linkage disequilibrium in the GSTM1 region in CHB. (A) R2 value between the GSTM1 deletion and flanking single nucleotide variants in CHB. Each dot represent each single nucleotide variants. X-axis indicates chromosomal location and Y-axis indicates the R2 value between the GSTM1 deletion and flanking single nucleotide variants. Target regions were chr1:110211681-110223007 (target1, upstream the deletion) and chr1:110246810-110255596 (target2, downstream the deletion). (B) Recombination rate of the GSTM1 flanking region from DeCode (orange) and 1000genomes (blue). The chromosomal average of recombination rate from decode (orange) and 1000genomes (blue) were also plotted in the graph by broken lines. (C) Linkage disequilibrium (LD, R2 value) between the single nucleotide variants around the GSTM1 gene in CHB. Color coded in a continuous R2 values between each single nucleotide variants from 0 (white) to 1 (red)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1769612826" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Linkage disequilibrium in the &lt;em&gt;GSTM1&lt;/em&gt; region in CHB. (A) R&lt;sup&gt;2&lt;/sup&gt; value between the &lt;em&gt;GSTM1&lt;/em&gt; deletion and flanking single nucleotide variants in CHB. Each dot represent each single nucleotide variants. X-axis indicates chromosomal location and Y-axis indicates the R&lt;sup&gt;2&lt;/sup&gt; value between the &lt;em&gt;GSTM1&lt;/em&gt; deletion and flanking single nucleotide variants. Target regions were chr1:110211681-110223007 (target1, upstream the deletion) and chr1:110246810-110255596 (target2, downstream the deletion). (B) Recombination rate of the &lt;em&gt;GSTM1&lt;/em&gt; flanking region from DeCode (orange) and 1000genomes (blue). The chromosomal average of recombination rate from decode (orange) and 1000genomes (blue) were also plotted in the graph by broken lines. (C) Linkage disequilibrium (LD, R&lt;sup&gt;2&lt;/sup&gt; value) between the single nucleotide variants around the &lt;em&gt;GSTM1&lt;/em&gt; gene in CHB. Color coded in a continuous R&lt;sup&gt;2&lt;/sup&gt; values between each single nucleotide variants from 0 (white) to 1 (red).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2953/F2.medium.gif" width="440" height="309"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/9/2953/F2.medium.gif" width="440" height="309"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2953/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2953/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490876" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-16" class="first-child">Linkage disequilibrium in the <em>GSTM1</em> region in CHB. (A) R<sup>2</sup> value between the <em>GSTM1</em> deletion and flanking single nucleotide variants in CHB. Each dot represent each single nucleotide variants. X-axis indicates chromosomal location and Y-axis indicates the R<sup>2</sup> value between the <em>GSTM1</em> deletion and flanking single nucleotide variants. Target regions were chr1:110211681-110223007 (target1, upstream the deletion) and chr1:110246810-110255596 (target2, downstream the deletion). (B) Recombination rate of the <em>GSTM1</em> flanking region from DeCode (orange) and 1000genomes (blue). The chromosomal average of recombination rate from decode (orange) and 1000genomes (blue) were also plotted in the graph by broken lines. (C) Linkage disequilibrium (LD, R<sup>2</sup> value) between the single nucleotide variants around the <em>GSTM1</em> gene in CHB. Color coded in a continuous R<sup>2</sup> values between each single nucleotide variants from 0 (white) to 1 (red).</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2953/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Haplotypes of the combined target1 and target2 regions. (A) The phylogenetic relationship of the downstream target 2 region of CEU, YRI, CHB, Neanderthal, Denisovan and Chimpanzee constructed by the maximum-likelihood method. The GSTM1-deleted haplotypes were marked by green and the GSTM1-non-deleted haplotypes were marked by red manually. (B) Clustered haplotypes of the upstream target 1 region and downstream target 2 regions of CHB and CEU. The clustering was conducted based only on the haplotypes themselves with neither a priori input of haplogroups nor the deletion status. The GSTM1-deleted haplotypes were marked by green and the GSTM1-non-deleted haplotypes were marked by red. The Tanuki haplogroup, which showed high population differentiation, was marked by blue and other haplotypes were marked by yellow." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1769612826" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Haplotypes of the combined target1 and target2 regions. (A) The phylogenetic relationship of the downstream target 2 region of CEU, YRI, CHB, Neanderthal, Denisovan and Chimpanzee constructed by the maximum-likelihood method. The &lt;em&gt;GSTM1&lt;/em&gt;-deleted haplotypes were marked by green and the &lt;em&gt;GSTM1&lt;/em&gt;-non-deleted haplotypes were marked by red manually. (B) Clustered haplotypes of the upstream target 1 region and downstream target 2 regions of CHB and CEU. The clustering was conducted based only on the haplotypes themselves with neither &lt;em&gt;a priori&lt;/em&gt; input of haplogroups nor the deletion status. The &lt;em&gt;GSTM1&lt;/em&gt;-deleted haplotypes were marked by green and the &lt;em&gt;GSTM1&lt;/em&gt;-non-deleted haplotypes were marked by red. The &lt;em&gt;Tanuki&lt;/em&gt; haplogroup, which showed high population differentiation, was marked by blue and other haplotypes were marked by yellow.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2953/F3.medium.gif" width="440" height="372"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/9/2953/F3.medium.gif" width="440" height="372"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2953/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2953/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490869" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-17" class="first-child">Haplotypes of the combined target1 and target2 regions. (A) The phylogenetic relationship of the downstream target 2 region of CEU, YRI, CHB, Neanderthal, Denisovan and Chimpanzee constructed by the maximum-likelihood method. The <em>GSTM1</em>-deleted haplotypes were marked by green and the <em>GSTM1</em>-non-deleted haplotypes were marked by red manually. (B) Clustered haplotypes of the upstream target 1 region and downstream target 2 regions of CHB and CEU. The clustering was conducted based only on the haplotypes themselves with neither <em>a priori</em> input of haplogroups nor the deletion status. The <em>GSTM1</em>-deleted haplotypes were marked by green and the <em>GSTM1</em>-non-deleted haplotypes were marked by red. The <em>Tanuki</em> haplogroup, which showed high population differentiation, was marked by blue and other haplotypes were marked by yellow.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h4>Tests of natural selection on the Target region:</h4><p id="p-18">We first constructed a null distribution by calculating the <em>F<sub>ST</sub></em> (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Weir and Cockerham 1984</a>), Tajima’s D (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Tajima 1989</a>), π (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Nei and Li 1979</a>) and iHS (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Voight <em>et al.</em> 2006</a>) values for 1000 randomly chosen regions on the same chromosome where the <em>GSTM1</em> is located (chromosome 1) and also match our target region by size (∼9kb). Then, we calculated these statistics observed in the downstream regions of the <em>GSTM1</em> deletion (chr1:110246810-110255596, downstream the deletion, also represented in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>) and compare the results with those from the random regions. We also replicated this with size-matched regions that overlap with all 618 segmental duplications on chromosome 1 to match the genomic features in the <em>GSTM1</em> region (Figure S3).</p><p id="p-19">We used genomic data of CEU, CHB and YRI populations in 1KGP. Tajima’s D value, iHS and π were calculated for 3kb intervals. XP-CLR were calculated for 2kb intervals. All these measurements were acquired through the 1000 Genomes Selection Browser (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Pybus <em>et al.</em> 2014</a>).</p><p id="p-20">To visualize the geographic distribution of the <em>Tanuki</em> haplogroup, which was linked with the deletion in East Asia, we used data from 15 populations in the 1000 Genome Project for haplotype distribution analysis: BEB, CDX, CHB, ESN, FIN, GBR, GWD, IBS, JPT, KHV, LWK, MSL, PJL, TSI, and YRI, which have not experienced recent population admixture or migration. We used the “rworldmap” package (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">South 2011</a>) to visualize the global distribution of the haplotypes.</p></div><div id="sec-8" class="subsection"><h4>Simulations for <em>F<sub>ST</sub></em> and π values:</h4><p id="p-21">For the simulation analyses, we used ms (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hudson 2002</a>) to conduct 1000 simulations matching the size (8,787bp) and the observed Watterson Estimator (θ=9.14) which was calculated by DnaSP (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Librado and Rozas 2009</a>) of the downstream target2 region (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). For each simulation, we generated 216 and 206 haplotypic sequences for YRI and CHB, respectively. We chose these populations and the number of sequences to match the empirical data from the 1000 Genome Project. To accurately model the effect of the demography in our simulations, we used the published parameters for East Asian and African populations (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Schaffner <em>et al.</em> 2005</a>) and we implemented the recombination rates of this region (ρ=5) reported by the HapMap Consortium (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">International HapMap Consortium <em>et al.</em> 2007</a>). Once the sequences are generated, we focused on constructing expected distributions for population differentiation (<em>F<sub>ST</sub></em>) and nucleotide diversity (π).</p><p id="p-22">The command line for ms simulator was ./ms 422 1000 -t 91.4 -I 2 216 206 -m 1 2 3.2 -n 2 0.077 -en 0.0005 1 0.25 -en 0.001 2 0.077 -ej 0.00875 2 1 -en 0.0425 1 0.125 -r 5 8787.</p><p id="p-23">We calculated <em>F<sub>ST</sub></em> values between YRI and CHB populations for each single nucleotide variant that was generated in the simulations. We then plotted the values with the matching allele frequencies (0.69-0.70, the range of standard deviation of the observed allele frequency of the variations in <em>Tanuki</em> haplogroup) for each of these variants. Next, we considered the distribution of the simulated π values. To simulate this, we first generated 1000 simulated datasets as described above. We then focused on the segregating sites with alternative allele frequencies at 0.69-0.70, the range of standard deviation of the observed allele frequency of the variations in <em>Tanuki</em> haplogroup. Based on this, we calculated π for each of these haplogroups which carry the alternative allele for each single nucleotide variation in the simulated CHB population (Figure. S4).</p><p id="p-24">We observed that the correlation fit between the simulated and empirical <em>F<sub>ST</sub></em> values are high (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>), but notably less than the correlation between the simulated and empirical π values (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). We believe that this is due to the fact that we used demographic parameters for two populations (East Asian and African populations (<a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Schaffner <em>et al.</em> 2005</a>) for the <em>F<sub>ST</sub></em> simulations, while assumed parameters for only the East Asian populations for the π calculations.</p></div><div id="sec-9" class="subsection"><h4>The Run of Homozygosity analysis:</h4><p id="p-25">We used the demographic model of (<a id="xref-ref-64-3" class="xref-bibr" href="#ref-64">Schaffner <em>et al.</em> 2005</a>) with theta = 0.00375 per site and Rec rate = 0.001 per site and 4<em>Ne</em>m = 9 (<em>Ne</em> = effective population size <em>and</em> m= migration rate) from the empirical observation. We ran more than 7000 simulations per each condition and obtained 70-120 core sites with the frequency of 0.69+-0.031 (Standard deviation calculated from Binomial Distribution) in East Asia and 0 in Africa under neutrality. We calculated both side of ROH for the 70-120 core sites from the simulation. We also calculated one side (downstream) of ROH of the 238 <em>Tanuki</em> homozygous individuals in East Asian populations for the <em>Tanuki</em> SNP as there was a recombination hotspot in the upstream of <em>Tanuki</em> region (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure. 2B</a>). We used three recombination rates (20, 200, 2000 recombinations per the 200kb region) to see the impact of recombination rates on ROH. r = 200 was the most realistic parameter from the observation.</p><p id="p-26">The command line for ms simulator was ./ms 422 1000 -t 750 -r 200 200000 -I 2 206 216 9 -en 0.0005 2 0.25 -en 0.001 1 0.077 -en 0.0425 2 0.125</p><p id="p-27">To test for potential sampling bias (<em>e.g.</em>, consanguinity) in our dataset, we conducted bootstrap resampling of individual phased chromosomes. There are 238 individuals in East Asian populations (CHB, JPT, CHS, KHV, CDX) that are homozygous for <em>Tanuki</em> haplotype. To construct a distribution to investigate whether there is a sampling bias, we randomly resampled from 694 phased <em>Tanuki</em> haplotypes to generate 500 datasets each with 238 homozygous constructs. This allowed us to test whether the observed value of ROH deviates from randomly resampled distribution. We observed no such deviation, suggesting no sampling bias in our analysis (Figure S5).</p></div><div id="sec-10" class="subsection"><h4>Age estimation of <em>Tanuki</em> haplogroup and selection on <em>Tanuki</em> haplogroup:</h4><p id="p-28">We conducted several age estimates in this paper. First, we used the simple formula detailed (15) in (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kimura and Ohta 1973</a>) and estimated the expected time for the <em>Tanuki</em> haplogroup to reach the current frequency in East Asia and Europe under neutrality.</p><p id="p-29">Second, we used the average sequence differences of the <em>Tanuki</em> haplogroup from a non-<em>Tanuki</em> haplotype (Figure S6, S7) to calculate a coalescent date for the divergence of these two haplogroups. Specifically, based on a more recent (5 million years (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Varki and Altheide 2005</a>)) and older (∼12 million years (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Moorjani <em>et al.</em> 2016</a>)) divergent estimates between humans and chimpanzees, we estimated the coalescent of <em>Tanuki</em> haplogroup (0.001048 distance from the root haplotype) to be between 300 to 720 thousand years before present.</p><p id="p-30">Third, we calculated the starting date of the putative selection on <em>Tanuki</em> haplogroup based on the run-of-homozygosity in this locus with method detailed by <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Racimo <em>et al.</em> (2015)</a>. Briefly, at a past time when r*L*t (r = recombination rate, L = the length of run of homozygosity, t = the time when the selection on the variant) reaches 1, the ROH shows rapid breaks because of recombination. So, we can assume that the t at that time is the start point of the selection. We used r= (1.45±0.05)× 10<sup>−8</sup>/generation (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Narasimhan <em>et al.</em> 2017</a>). In the paper, they noted this value and confidence interval from their two estimates for the mutation rate (1.51 and 1.41 × 10<sup>−8</sup> per bp per generation (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Narasimhan <em>et al.</em> 2017</a>), u = 3 × 10<sup>−8</sup>/site/generation, and observed averaged ROH<sub>aver</sub> of the <em>Tanuki</em> Haplogroup (∼16kb).</p><p id="p-31">It is important to note some caveats here. First, the coalescence dates depend on the local recombination and mutation rates, and even our best estimates may not be accurate with our current knowledge. Second, given that this locus has likely evolved under adaptive forces, it would be more appropriate to incorporate dating approaches that consider selection. However, we have little insight into the timing and the strength of the selection, and thus we focused on dating approaches that do not depend on selection coefficient as a parameter.</p></div></div></div><div class="section other2" id="sec-11"><h2 class="">Data availability</h2><p id="p-32">Table S1 contains genotyping of the <em>GSTM1</em> copy number by ddPCR of 17 individuals in YRI population in the 1KGP. Supplemental materials are available at Figshare: Saitou, Marie; Gokcumen, Omer; Satta, Yoko (2018): G3-Supp0626.pdf.; <a href="https://doi.org/10.25387/g3.6384782">https://doi.org/10.25387/g3.6384782</a>.</p></div><div class="section results" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>Imperfect linkage disequilibrium Between the <em>GSTM1</em> deletion and flanking single nucleotide variants in the human lineage</h3><p id="p-33">To understand the evolutionary mechanisms which maintain the <em>GSTM1</em> deletion common in humans, we first attempted to resolve the haplotype structure of the locus. We calculated the R<sup>2</sup> value between the <em>GSTM1</em> deletion and flanking single nucleotide variants in Han Chinese in Bejing, China (CHB), Utah residents with Northern and Western European ancestry (CEU) and Yoruba in Ibadan, Nigeria (YRI) populations in 1KGP phase 3 datasets (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2A</a>, Figure S1). Remarkably, we found almost no linkage disequilibrium between the <em>GSTM1</em> deletion flanking single nucleotide variants in CEU and YRI (Figure S1), and observable, but imperfect linkage disequilibrium in CHB only extending to &gt;2.5kb on each side of the deletion (R<sup>2</sup>=∼0.45,<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2A</a>). One way to explain this low level of linkage disequilibrium is by invoking a recombination hotspot in the region. However, we found that the reported recombination in both deCODE (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kong <em>et al.</em> 2010</a>) and 1000 genomes phase 3 datasets for the region (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1000 Genomes Project Consortium <em>et al.</em> 2010</a>) is high, but not exceptionally high to be categorized as a hotspot (3-17 cM/Mb, (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Mc Vean <em>et al.</em> 2004</a>)) (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2B</a>).</p><p id="p-34">One mechanism that can explain the imperfect linkage disequilibrium between the deletion and the neighboring variants can be recurrent formation of deletions. In other words, if two or more independent deletion events have occurred in different haplotypic backgrounds, then we expect (i) different deletion breakpoints, and (ii) LD to be intact between the variants on each side of the deletion as described by Boettger <em>et al.</em> (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Boettger <em>et al.</em> 2016</a>). In a previous study, we conducted an analysis of the breakpoints of this deletion, but, due to repetitive nature of this locus, could not identify the exact breakpoints of this deletion (<a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">Saitou <em>et al.</em> 2018</a>). In parallel, we found that there is relatively weak, but observable linkage disequilibrium (R<sup>2</sup>=∼0.5) between variants on each side of the deletion in CHB (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2C</a>). Overall, our results cannot rule out recurrent formation of the <em>GSTM1</em> deletion in the human lineage.</p><p id="p-35">To further understand the haplotypic variation in this locus, we constructed a phylogenetic tree using sequence variation data in the “target” region flanking the deletion for 620 phased haplotypes from YRI, CEU, CHB populations, as well as chimpanzee, Denisovan, Neanderthal genomes (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>). As expected from the low levels of linkage disequilibrium between single nucleotide variants and the deletion, we observed no clear separation between haplotypes with and without the deletion. Instead, we found multiple branches that are predominantly populated with deleted haplotypes and others with non-deleted haplotypes without notable population structuring. These observations are concordant with the recent study documenting the genetic variation in this locus in a Russian population (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Khrunin <em>et al.</em> 2016</a>). It is important to note here, however, that Neanderthal and Denisovan haplotypes, as well as the branching point for the chimpanzee reference haplotype, all cluster with the predominantly non-deleted haplotypes. This suggests that the deletion is a derived variant that likely evolved after the human-Neanderthal divergence (<em>i.e.</em>, in the last 1 Million years).</p><p id="p-36">Despite these general insights, the observable lack of linkage disequilibrium between the <em>GSTM1</em> deletion and the neighboring variants reduces the power of disease-association studies (<em>e.g.</em>, GWAS) that rely on imputation of the <em>GSTM1</em> deletion genotype using nearby single nucleotide variants. Even considering multiple single nucleotide variants within a single population, we were not able to impute the deletion accurately in any of the study populations (YRI, CHB, and CEU) (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). This is concordant with the previous finding reported the difficulty to predict the copy number of the <em>GSTM1</em> gene by the flanking haplotypes in CEU (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Khrunin <em>et al.</em> 2016</a>). As such, direct genotyping, rather than imputation, may be more robust approach to study <em>GSTM1</em> deletion.</p></div><div id="sec-14" class="subsection"><h3>A potential signature of an incomplete sweep in East Asian populations in the <em>GSTM1</em> locus</h3><p id="p-37">To reveal any potential signatures of adaptive evolution affecting variation in this locus, including the <em>GSTM1</em> deletion, we conducted statistical neutrality tests using single nucleotide variation data from the sequences flanking the <em>GSTM1</em> deletion. Specifically, we first constructed a null empirical distribution by calculating the <em>F<sub>ST</sub></em> (<a id="xref-ref-80-2" class="xref-bibr" href="#ref-80">Weir and Cockerham 1984</a>), Tajima’s D (<a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Tajima 1989</a>), π (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Nei and Li 1979</a>), iHS (<a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Voight <em>et al.</em> 2006</a>), XP-EHH and XP-CLR values for 1000 randomly chosen regions on the same chromosome where the <em>GSTM1</em> is located (chromosome 1) and also match our target region by size (∼9kb). Then, we calculated these statistics observed in the downstream regions of the <em>GSTM1</em> deletion and compared the results with those from the random regions (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>, Figure. S2). We also replicated this with randomly selected, size-matched regions that overlap with all of the 618 segmental duplications on chromosome 1 to match the genomic features in the <em>GSTM1</em> region (Figure. S2). We found that both XP-CLR and XP-EHH tests showed significant differences as compared to neutrality when <em>CHB</em> population is involved (<em>p</em> values &lt; 0.01, Wilcoxon rank sum test with continuity correction). These tests measures the change in allele frequency in one population has occurred more quickly than expected by drift alone and the difference in extended homozygosity between two populations (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>, <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). XP-EHH (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Sabeti <em>et al.</em> 2007</a>) is cross-population extended haplotype homozygosity and XP-CLR (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chen <em>et al.</em> 2010</a>) is multi-locus allele frequency differentiation between two populations. Collectively, the unusually high XP-CLR and XP-EHH values in CHB population suggest an incomplete sweep has shaped the distribution of <em>Tanuki</em> haplogroup in this population.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2953/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Neutrality tests on the target region (chr1:110246810-110255596, downstream the deletion, also represented in Figure. 2.), the 1000 randomly selected 9kb regions, and all the segmental duplications on chromosome one, which contains the GSTM1 in CEU, CHB and YRI populations in 1KGP. XP-CLR and XP-EHH were calculated for 2kb intervals. The stars show significant differences (P &lt; 0.01, Wilcoxon Rank Sum Test). (B) The geographical distribution of the Tanuki haplogroup calculated from the 1KGP phase3 data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1769612826" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Neutrality tests on the target region (chr1:110246810-110255596, downstream the deletion, also represented in Figure. 2.), the 1000 randomly selected 9kb regions, and all the segmental duplications on chromosome one, which contains the &lt;em&gt;GSTM1&lt;/em&gt; in CEU, CHB and YRI populations in 1KGP. XP-CLR and XP-EHH were calculated for 2kb intervals. The stars show significant differences (&lt;em&gt;P&lt;/em&gt; &lt; 0.01, Wilcoxon Rank Sum Test). (B) The geographical distribution of the &lt;em&gt;Tanuki&lt;/em&gt; haplogroup calculated from the 1KGP phase3 data.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2953/F4.medium.gif" width="440" height="413"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/9/2953/F4.medium.gif" width="440" height="413"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2953/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2953/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490904" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-38" class="first-child">(A) Neutrality tests on the target region (chr1:110246810-110255596, downstream the deletion, also represented in <a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure. 2</a>.), the 1000 randomly selected 9kb regions, and all the segmental duplications on chromosome one, which contains the <em>GSTM1</em> in CEU, CHB and YRI populations in 1KGP. XP-CLR and XP-EHH were calculated for 2kb intervals. The stars show significant differences (<em>P</em> &lt; 0.01, Wilcoxon Rank Sum Test). (B) The geographical distribution of the <em>Tanuki</em> haplogroup calculated from the 1KGP phase3 data.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490880/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490880/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">XP-EHH and XP-CLR between CHB-YRI, CEU-YRI and CHB-CEU. Significant values were shaded.</span><div class="sb-div caption-clear"></div></div></div><p id="p-39">To investigate the haplotypic background of this putative sweep, we focused on the single nucleotide variants that showed the highest population differentiation (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a>). We first showed that they are in high linkage disequilibrium with each other (R<sup>2</sup> &gt; 0.84), indicating that they represent a single haplotype group in East Asian populations (Figure S8, Table S2, called <em>Tanuki</em> haplogroup from hereon) in the target2 region (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2A</a>). We first investigated the global distribution of the haplogroup and found that it represents the major allele (&gt;70% allele frequency) in East Asian populations, but it is virtually absent in sub-Saharan African populations (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a>, Figure S9).</p><p id="p-40">To determine the age of this haplogroup, we constructed a phylogenetic tree (Figure S6) of all <em>Tanuki</em> haplotypes in CEU and CHB populations (no <em>Tanuki</em> haplotype was found among YRI genomes), along with a non-<em>Tanuki</em> haplotype and the chimpanzee reference genome haplotype as outgroups. We surmised that <em>Tanuki</em> haplotype diverged from other human haplotypes at least 300 thousand years ago (∼300K to 720K YBP). Even considering the relatively large deviation of this estimate, this date is far earlier than the out-of-Africa migrations. As such, we conclude that this haplogroup has originated in Africa, but has increased in frequency in Eurasia. It is plausible that some <em>Tanuki</em> haplotypes still remain in African populations, but at a low allele frequency.</p><p id="p-41">To further understand the haplotypic background of the significantly high XP-CLR and XP-EHH values in Asian populations as compared to European and African populations, we constructed a network of <em>Tanuki</em> haplotypes (Figure S7). We found that a vast majority of <em>Tanuki</em> haplotypes in CHB population has identical sequences, suggesting a rapid and recent increase in the allele frequency in the East Asian populations. This observation explains the unusually high XP-CLR and XP-EHH values observed for CHB population and further support the notion that a incomplete sweep has shaped the distribution of <em>Tanuki</em> haplogroup in this population.</p></div><div id="sec-15" class="subsection"><h3>Allele frequency of the <em>Tanuki</em> haplogroup is unusual in East Asia, but not in Europe</h3><p id="p-42">The high XP-CLR and XP-EHH values along with very high frequency of <em>Tanuki</em> haplogroup in CHB potentially point to the effect of an local sweep. The alternative, null hypothesis would be that the observed difference in allele frequency is due to the effect of drift alone. To further distinguish between these two scenarios, we conducted multiple empirical and simulation based analyses. First, we investigated whether the allele frequency of <em>Tanuki</em> haplogroup in East Asian and European populations can be explained by drift alone. To do this, we first assumed that the putative target for selection is an allele that reached to approximately 70% allele frequency in East Asia, but remain at 25% allele frequency in Europe (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4B</a>). Then, we used the simple formula detailed (15) in (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Kimura and Ohta 1973</a>) and estimated that it would take ∼580 and ∼160 thousand years under neutrality for that single haplotype to increase in allele frequency from 1/2<em>N<sub>e</sub></em> in ancestral African population to 70% and 25% observed in Asia and Europe, respectively. This results suggest that the allele frequency of <em>Tanuki</em> haplotypes in Europe can be explained by neutrality alone, but not in East Asia.</p><p id="p-43">To further interrogate whether the high allele frequency of <em>Tanuki</em> haplotypes can be explained by genetic drift alone, we conducted simulation-based analyses of the genetic variation we observed (for detailed conditions for simulations, see Methods). Specifically, we used ms (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Hudson 2002</a>) to generate 1000 simulated datasets comprising sequences matching the size (8,787bp) of the target2 region of <em>GSTM1</em> (<a id="xref-fig-2-12" class="xref-fig" href="#F2">Figure 2</a>). For these simulations, we used demographic parameters previously laid out for CHB and YRI populations (<a id="xref-ref-64-4" class="xref-bibr" href="#ref-64">Schaffner <em>et al.</em> 2005</a>) to accurately model the effect of drift on nucleotide diversity and allele frequency.</p><p id="p-44">Once the sequences were generated, we calculated <em>F<sub>ST</sub></em> values for each single nucleotide variant that was generated in the simulations between two simulated populations (YRI and CHB). To verify the accuracy of our simulations, we first compared our simulated sequences to empirical distribution of <em>F<sub>ST</sub></em> between YRI and CHB populations for randomly selected single nucleotide variants across the genome (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5A</a>). We then plotted the <em>F<sub>ST</sub></em> values with the matching allele frequencies (0.69-0.70, that represent the deviation of the observed allele frequency of the variations in <em>Tanuki</em> in Asian populations) for each of these variants observed in the simulation. We found that none of 603 frequency-matched simulated <em>F<sub>ST</sub></em> is higher than the observed <em>F<sub>ST</sub></em> for the <em>Tanuki</em> haplogroup (<strong>&gt;99<sup>th</sup> percentile</strong>, <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5A</a>). Overall, these results suggest that the unusually high allele frequency of <em>Tanuki</em> haplotypes in CHB population is unlikely to be explained by neutral forces alone.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2953/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) The average values of FST for Tanuki SNPs (dotted line), the 1000 randomly selected 9kb regions (red), 9kb regions on chromosome 1 that overlap with segmental duplications (green), simulated data under neutrality (blue), and frequency-matched (0.69 - 0.70) single nucleotide variants on chromosome 1 (purple). We found that none of 603 frequency-matched simulated FST is higher than the observed FST for the Tanuki haplogroup. (B) The &#x3C0; value for Tanuki haplotypes (dotted line), the 1000 randomly selected 9kb regions (red), 9kb regions on chromosome 1 that overlap with segmental duplications (green), simulated data under neutrality (blue). The results showed that &#x3C0; observed in the Tanuki haplogroup is lowest among 389 frequency-matched simulated values. (C) The Run of Homozygosity of the Tanuki SNP (purple) and simulated frequency-matched SNPs with 3 different recombination rates, 20 (red), 200 (green), 2000 (blue). We found that the ROH values observed for the Tanuki single nucleotide variants in the 1KGP East Asian populations is significantly higher than what is expected under neutrality based on frequency matched simulation results with r = 200, the most realistic parameter from the observation (Fig. 5C, P &lt; 2.5x10&#x2212;12, Wilcoxon-Rank-Sum Test)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1769612826" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) The average values of &lt;em&gt;F&lt;sub&gt;ST&lt;/sub&gt;&lt;/em&gt; for &lt;em&gt;Tanuki&lt;/em&gt; SNPs (dotted line), the 1000 randomly selected 9kb regions (red), 9kb regions on chromosome 1 that overlap with segmental duplications (green), simulated data under neutrality (blue), and frequency-matched (0.69 - 0.70) single nucleotide variants on chromosome 1 (purple). We found that none of 603 frequency-matched simulated &lt;em&gt;F&lt;sub&gt;ST&lt;/sub&gt;&lt;/em&gt; is higher than the observed &lt;em&gt;F&lt;sub&gt;ST&lt;/sub&gt;&lt;/em&gt; for the &lt;em&gt;Tanuki&lt;/em&gt; haplogroup. (B) The &#x3C0; value for &lt;em&gt;Tanuki&lt;/em&gt; haplotypes (dotted line), the 1000 randomly selected 9kb regions (red), 9kb regions on chromosome 1 that overlap with segmental duplications (green), simulated data under neutrality (blue). The results showed that &#x3C0; observed in the &lt;em&gt;Tanuki&lt;/em&gt; haplogroup is lowest among 389 frequency-matched simulated values. (C) The Run of Homozygosity of the &lt;em&gt;Tanuki&lt;/em&gt; SNP (purple) and simulated frequency-matched SNPs with 3 different recombination rates, 20 (red), 200 (green), 2000 (blue). We found that the ROH values observed for the &lt;em&gt;Tanuki&lt;/em&gt; single nucleotide variants in the 1KGP East Asian populations is significantly higher than what is expected under neutrality based on frequency matched simulation results with r = 200, the most realistic parameter from the observation (Fig. 5C, &lt;em&gt;P&lt;/em&gt; &lt; 2.5x10&lt;sup&gt;&#x2212;12&lt;/sup&gt;, Wilcoxon-Rank-Sum Test).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2953/F5.medium.gif" width="440" height="383"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/9/2953/F5.medium.gif" width="440" height="383"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2953/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2953/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490896" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-45" class="first-child">(A) The average values of <em>F<sub>ST</sub></em> for <em>Tanuki</em> SNPs (dotted line), the 1000 randomly selected 9kb regions (red), 9kb regions on chromosome 1 that overlap with segmental duplications (green), simulated data under neutrality (blue), and frequency-matched (0.69 - 0.70) single nucleotide variants on chromosome 1 (purple). We found that none of 603 frequency-matched simulated <em>F<sub>ST</sub></em> is higher than the observed <em>F<sub>ST</sub></em> for the <em>Tanuki</em> haplogroup. (B) The π value for <em>Tanuki</em> haplotypes (dotted line), the 1000 randomly selected 9kb regions (red), 9kb regions on chromosome 1 that overlap with segmental duplications (green), simulated data under neutrality (blue). The results showed that π observed in the <em>Tanuki</em> haplogroup is lowest among 389 frequency-matched simulated values. (C) The Run of Homozygosity of the <em>Tanuki</em> SNP (purple) and simulated frequency-matched SNPs with 3 different recombination rates, 20 (red), 200 (green), 2000 (blue). We found that the ROH values observed for the <em>Tanuki</em> single nucleotide variants in the 1KGP East Asian populations is significantly higher than what is expected under neutrality based on frequency matched simulation results with r = 200, the most realistic parameter from the observation (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Fig. 5C</a>, <em>P</em> &lt; 2.5x10<sup>−12</sup>, Wilcoxon-Rank-Sum Test).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Haplotype homozygosity provides further evidence for a selective sweep</h3><p id="p-46">In parallel to the allele frequency based analyses described above, we conducted tests based on haplotype similarity. We first followed the reasoning outlined by Kim and Satta (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kim and Satta 2008</a>). Briefly, if a recent sweep indeed increased the allele frequency of a particular existing haplotype group, we expect that the nucleotide diversity would decrease among the haplotypes that were sweeped, but not the others. Indeed, we found that the nucleotide diversity of haplotypes that belongs to the <em>Tanuki</em> haplogroup, which make up &gt;70% of haplotypes in East Asians, is approximately six times lower (π=0.00012) in East Asians as compared to the nucleotide diversity observed in haplotypes that do not belong to the <em>Tanuki</em> haplogroup in the same population (π=0.00076).</p><p id="p-47">To explore this issue, we have considered the π values for <em>Tanuki</em> and <em>nonTanuki</em> haplotypes across a larger genomic region surrounding the <em>GSTM1</em>, including sequences that are not overlapping segmental duplications in multiple East Asian populations (CHB, CHS, JPT and KHV) (Figure. S10). These results also corroborates our initial finding that <em>Tanuki</em> haplotypes show unusually low π as compared to simulated expectations (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure. 5B</a>).</p><p id="p-48">To quantify this observation, we used simulated sequences (as described above) to construct an expected, neutral distribution of nucleotide diversity (π) values. Specifically, we wanted to test whether π calculated for the haplotypes that belong to the <em>Tanuki</em> haplogroup is lower than expected distribution under neutrality. To simulate this, we first generated 1000 simulated datasets as described above. From the simulated dataset, we chose haplogroups within the simulated with alternative allele frequencies at 0.69-0.70 to match that of <em>Tanuki</em> haplogroup in frequency (please see Figure S4). Then, we plotted the π values for these simulated haplogroups. The results showed that π observed in the <em>Tanuki</em> haplogroup is lowest among 389 frequency-matched simulated values (<strong>&lt;1<sup>st</sup> percentile,</strong> <a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5B</a>).</p><p id="p-49">To test whether the low nucleotide diversity observed in <em>Tanuki</em> haplogroup is also translated into extended homozygosity, a hallmark of a selective sweep, we simulated expected Runs of Homozygosity (ROH) assuming demographic parameters outlined in (<a id="xref-ref-64-5" class="xref-bibr" href="#ref-64">Schaffner <em>et al.</em> 2005</a>) and different recombination rates (see methods for details). We found that the ROH values observed for the <em>Tanuki</em> single nucleotide variants in the 1KGP East Asian populations is significantly higher than what is expected under neutrality based on frequency matched simulation results with r = 200, the most realistic parameter from the observation (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5C</a>, <em>P</em> &lt; 2.5x10<sup>−12</sup>, Wilcoxon-Rank-Sum Test). The averaged ROH of the <em>Tanuki</em> haplogroup was 16Kb.</p><p id="p-50">This calculation also allowed us to calculate a potential date for the sweep using the approach outlined elsewhere (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Racimo <em>et al.</em> 2015</a>) (see methods). Briefly, this method uses the length of ROH to estimate the age of the beginning of a sweep under given recombination and mutation rates. Using this method, we estimated this date to be 41.7±0.5K years ago when the selection on <em>Tanuki</em> Haplogroup began (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). Combined, the observed values of population differentiation and haplotype diversity are inconsistent with the neutral evolution of this locus and are parsimonious with our hypothesis that the <em>Tanuki</em> haplogroup has increased its frequency under adaptive evolution in the East Asian populations.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/9/2953/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The evolutionary history of the Tanuki haplogroup. Based on our analyses, we argue that the Tanuki haplogroup originated before Out of Africa migrations (&#x223C;300K years before present) and spread out to Eurasia. We further argue that in Europe, the frequency of Tanuki haplogroup increased approximately to 0.25 under neutrality. In Asia, the selection on the Tanuki haplogroup started 41.7K years ago and pushed frequency of the Tanuki haplogroup to 0.7 under selection." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1769612826" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The evolutionary history of the &lt;em&gt;Tanuki&lt;/em&gt; haplogroup. Based on our analyses, we argue that the &lt;em&gt;Tanuki&lt;/em&gt; haplogroup originated before Out of Africa migrations (&#x223C;300K years before present) and spread out to Eurasia. We further argue that in Europe, the frequency of Tanuki haplogroup increased approximately to 0.25 under neutrality. In Asia, the selection on the &lt;em&gt;Tanuki&lt;/em&gt; haplogroup started 41.7K years ago and pushed frequency of the &lt;em&gt;Tanuki&lt;/em&gt; haplogroup to 0.7 under selection.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/9/2953/F6.medium.gif" width="440" height="364"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/9/2953/F6.medium.gif" width="440" height="364"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/9/2953/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/9/2953/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490855" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-51" class="first-child">The evolutionary history of the <em>Tanuki</em> haplogroup. Based on our analyses, we argue that the <em>Tanuki</em> haplogroup originated before Out of Africa migrations (∼300K years before present) and spread out to Eurasia. We further argue that in Europe, the frequency of Tanuki haplogroup increased approximately to 0.25 under neutrality. In Asia, the selection on the <em>Tanuki</em> haplogroup started 41.7K years ago and pushed frequency of the <em>Tanuki</em> haplogroup to 0.7 under selection.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Potential functional impact of <em>Tanuki</em> haplogroup</h3><p id="p-52">By carefully resolving the different haplotype groups in the <em>GSTM1</em> locus, we were able to detect a putatively adaptive haplogroup (<em>Tanuki</em> haplogroup). The exact underlying evolutionary reason and the functional impact of the increase in frequency of the <em>Tanuki</em> haplogroup remain excellent venues for future studies. One obvious question is whether the <em>GSTM1</em> deletion is the actual target of positive selection. Indeed, we discovered that <em>Tanuki</em> haplogroup is significantly, albeit imperfectly, linked with the <em>GSTM1</em> deletion in the CHB population (R<sup>2</sup>=∼0.47) (Table S2). It is important to note that despite the low linkage disequilibrium, ∼91% of the <em>Tanuki</em> haplotypes harbor the <em>GSTM1</em> deletion. We reasoned that if the <em>GSTM1</em> deletion was the target for the selective sweep that we observed, then other haplotypes carrying the <em>GSTM1</em> deletion also show evidence for selection. However, we did not observe any unusual increase in allele frequency of other haplotypes that carry the deletion. As such, it is highly plausible, as drastic an event as a whole gene deletion is, <em>GSTM1</em> deletion may not be the main target of the selective sweep that we observed.</p><p id="p-53">To further follow up this thread, we asked whether the <em>GSTM1</em> gene has no fitness consequences and hence accumulating loss of function mutations, such as the deletion. Under this scenario, the <em>GSTM1</em> deletion just swept along with the remainder of the <em>Tanuki</em> haplogroup with scant adaptive consequences. Contradicting this possibility, it was reported that the <em>GSTM1</em> gene sequence is conserved among great apes, as well as between humans and archaic humans. Specifically, we found only 4 nonsynomous variants between human and chimpanzee <em>GSTM1</em>, and found none among human, Neanderthal and Denisovan <em>GSTM1</em> genes (Table S3). In addition, only two commonly observed nonsynonymous mutations were reported for <em>GSTM1</em> gene in human population, both of which have little effect on the <em>GSTM1</em> function (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Moyer <em>et al.</em> 2007</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Tatewaki <em>et al.</em> 2009</a>). ExAC database allowed us to be more specific (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lek <em>et al.</em> 2016</a>). The expected number of loss-of-function variants at the <em>GSTM1</em> locus is 8.4 but observed number was 2 in the Exome analysis in 60,706 humans. In addition, the frequency of these loss-of-function variants are extremely low (&lt;0.00003133). It is important to note that the deletion of the entire gene in approximately half of human genomes is not currently reported in ExAC database. However, our point here is that if <em>GSTM1</em> has no fitness effect in humans, we expect to find relatively high frequencies of loss of function variants in addition to the deletion. Instead, virtually all the nondeleted <em>GSTM1</em> haplotypes carry an intact, open-reading frame, further supporting the notion that both deleted and nondeleted haplotypes are maintained in the population.</p><p id="p-54">These results suggest that it is likely that the <em>GSTM1</em> deletion has fitness effects, but it is independent of the selective sweep we observed for the <em>Tanuki</em> haplogroup. To explain these observations, we hypothesize that <em>Tanuki</em> haplogroup affect the function of multiple genes in the <em>GSTM</em> gene family and not just <em>GSTM1</em>. Consequently, the putative adaptive phenotype is a result of combination of these effects. Such a perspective has precedent where other studies described likely adaptive haplotypes that have effects on multiple metabolism gene families, such as <em>IRX</em> (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Claussnitzer <em>et al.</em> 2015</a>) and <em>FAD</em> (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fumagalli <em>et al.</em> 2015</a>).</p><p id="p-55">To test whether <em>Tanuki</em> haplogroup indeed affect other neighboring genes, we investigated functional variants that are linked with this haplogroup. Specifically, we identified variants with unusually high PHRED scores (&gt;90<sup>th</sup> percentile) (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kircher <em>et al.</em> 2014</a>), GWAS variants (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">MacArthur <em>et al.</em> 2017</a>), and nonsynomous variants (Figure S11). We found no nonsynomous or GWAS variants that is linked with the Tanuki haplogroup. Nevertheless, we found one variant that has a high PHrED score (rs61799140). It is thus plausible that <em>Tanuki</em> haplogroup may have regulatory effect on nearby genes. To further test this, we used gene expression data from Gtex portal (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lonsdale <em>et al.</em> 2013</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Wheeler <em>et al.</em> 2016</a>). Our results showed that this haplogroup is associated with significant decreases in expression of the <em>GSTM5</em> gene in most tissues, especially in the brain, but increases the expression of <em>GSTM3</em> in skeletal muscle (Figure S12). As such, one potential argument would be that the selected effect is not the functioning of a single gene, but the overall regulatory impact of the <em>Tanuki</em> haplogroup on the <em>GSTM</em> locus.</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><div id="sec-19" class="subsection"><h3>Working With <em>complex</em> genomic structural variation loci</h3><p id="p-56">Our results provide a case study for the evolutionary impact of common haplotypic variation in a complex locus, involving both functional single nucleotide and copy number variants. One particular challenge in this locus that we verified is the general lack of linkage disequilibrium between common, neighboring variants. Specifically, we found a lack of strong linkage disequilibrium between the <em>GSTM1</em> deletion and neighboring variants. This finding corroborates our previous work at a deeper evolutionary depth, where we found evidence for multiple gene conversion events differentiating human and chimpanzee <em>GSTM</em> locus (<a id="xref-ref-62-4" class="xref-bibr" href="#ref-62">Saitou <em>et al.</em> 2018</a>). As such, most genome-wide association studies would not have effectively interrogated the potential biomedical impact of the <em>GSTM1</em> deletion because they have only investigated the single nucleotide variants, which do not adequately tag the deletion variant. In fact, when we consider the best case scenario for such studies and focus on East Asia, where the linkage disequilibrium is relatively high (R<sup>2</sup> = 0.4), statistical power for a single nucleotide variant based genome-wide association study would still be low. For example, based on <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hong and Park (2012)</a>’s calculations, the statistical power to detect an association would be only 26.5% when assuming a relatively standard experimental setup (<em>e.g.</em>, the odds ratio of the phenotype 1.3, 1000 cases and 1000 controls, <em>etc</em>.).</p><p id="p-57">This is not an exceptional case since the majority of deletions reside in similar haplotypic architectures (<a id="xref-ref-71-4" class="xref-bibr" href="#ref-71">Sudmant <em>et al.</em> 2015b</a>), complicating both evolutionary analysis, as well as disease association studies that depend on imputation to interrogate structural variants. Almost one-third of the deletions reported in 1000 genomes have R<sup>2</sup> &lt; 0.6 with neighboring variants (<a id="xref-ref-71-5" class="xref-bibr" href="#ref-71">Sudmant <em>et al.</em> 2015b</a>). As such genome-wide association studies will lose significant amount of power when assessing the effect of such SVs on tested trait if they are using imputation-based genotyping. Consequently, it is important to conduct direct genotyping of the deletion for such analyses or carefully resolve the haplotype structure within the locus to accurately assess the biomedical impact of these variants. Such locus-specific analyses indeed led to important connections between several structural variants and human diseases (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Rothman <em>et al.</em> 2010</a>; <a id="xref-ref-75-3" class="xref-bibr" href="#ref-75">Usher <em>et al.</em> 2015</a>; <a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Boettger <em>et al.</em> 2016</a>).</p></div><div id="sec-20" class="subsection"><h3>Evolution of the <em>GSTM1</em> locus</h3><p id="p-58">Here, we report an incomplete sweep that increase the allele frequency increase of n existing haplogroup (<em>Tanuki</em>) in East Asian populations, putatively affecting the function of multiple <em>GSTM</em> genes (as summarized in <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). Recent studies have shown the importance of sweeps on standing variation (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Schrider and Kern 2017</a>), rather than hard sweeps on novel variants (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hernandez <em>et al.</em> 2011</a>), to be the dominating type of positive selection acting on human genome.</p><p id="p-59">These insights fit well with our observations for the <em>Tanuki</em> haplogroup as well as with the emerging, broader picture where genes involved in metabolism have been shown to evolve under complex adaptive forces. For example, the <em>GSTM1</em> deletion and multiple <em>NAT</em> (N-acetyltransferase) variants were often considered together as the leading candidates for explaining the genetic basis of bladder cancer susceptibility (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">García-Closas <em>et al.</em> 2005</a>). <em>NAT2</em> has been reported to be evolving under balancing selection (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Mortensen <em>et al.</em> 2011</a>). In addition, variations in other xenobiotic metabolism genes, such as members of the <em>SLC</em> (solute carrier) gene family, have also been shown to be evolving adaptively (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Sabeti <em>et al.</em> 2007</a>). Some of these variations involve gene deletions. For example, the deletion of <em>UGT2B17</em> (uridine diphosphoglucuronosyl-transferase) has been shown to have increased in frequency in East Asian populations under adaptive forces (<a id="xref-ref-85-3" class="xref-bibr" href="#ref-85">Xue <em>et al.</em> 2008</a>). Overall, functional variation affecting metabolizing genes may be maintained adaptively in the human populations due to varying environmental pressures.</p></div><div id="sec-21" class="subsection"><h3>Implications to evolutionary medicine</h3><p id="p-60">As mentioned above, the variation in the <em>GSTM</em> locus has been the subject of more than 500 studies within the context of multiple diseases (<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Parl 2005</a>). However, given that majority of these studies are correlations, they lack mechanistic or evolutionary insights as to why the genetic variation in this locus is relevant. The emerging picture is that of a gene family (<em>GSTM1-5</em>) that broadly metabolizes multiple carcinogenic substances, for example, 4-nitroquinoline-1-oxide (NQO) (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Hayes <em>et al.</em> 2005</a>). On top of that functional layer, it has been shown that this gene family is riddled with common genetic variation, including the unusually common deletion of the <em>GSTM1</em>. These genetic variants, as expected, were associated with multiple cancers (<a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">Parl 2005</a>). The mechanistic explanation would be that the reduced the <em>GSTM</em> function leads to higher susceptibility to carcinogenic substance exposure (<a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">Hayes <em>et al.</em> 2005</a>).</p><p id="p-61">However, there seems to be some functional redundancy among the <em>GSTM</em> gene family members. As a consequence, the association between single variants to traits may not completely capture the biomedical impact of the variation in this locus as a whole. For example, the <em>GSTM2</em> may compensate some of the lost function due to the <em>GSTM1</em> deletion (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bhattacharjee <em>et al.</em> 2013</a>). As such, the overall functional impact of a variant depends on its genomic context. Moreover, a recent pathway-level analysis revealed that glutathione conjugation pathway, for which the <em>GSTM</em> genes are central, is a significant player in determining breast cancer susceptibility (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Wang <em>et al.</em> 2017</a>). The implication being that rather than single variants, the combined effect of multiple variants affecting the function of the genes in this pathway eventually contributes to the overall disease susceptibility. This is not a new concept (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Jin <em>et al.</em> 2014</a>). Such insights into the collective but varied impact of variation in a given locus have been discussed in evolutionary context in multiple species (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lin <em>et al.</em> 2015</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Salojärvi <em>et al.</em> 2017</a>). We are now in a position to quantitatively address this issue, especially using genealogical approaches as we described for the <em>GSTM1</em> here.</p><p id="p-62">From an evolutionary medicine point of view, it is important to highlight two interrelated features of the <em>GSTM</em> locus. First, even when a single haplotype affects susceptibility to disease, this may be due to pleiotropic effects. Indeed, we described multiple putative functional effects of the <em>Tanuki</em> haplogroup in East Asian populations, including the loss of function due to the <em>GSTM1</em> deletion and the decrease of the <em>GSTM5</em> expression. This result exemplifies the benefits of a haplotype-level understanding of the genetic variation. We argue based on our results and those of others (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Liu <em>et al.</em> 2005</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Claussnitzer <em>et al.</em> 2015</a>) that the co-occurrence of multiple functionally and biomedically relevant variants in particular haplotypes should be treated as the norm, rather than the exception. As exemplified in this study, functional analysis of haplotypes that are under selection may provide crucial targets for future mechanistic studies.</p><p id="p-63">Second, our results add to the growing list of population-specific haplotypes that may contribute to disease susceptibility, further underlying the importance of conducting genetic epidemiology studies in ancestrally diverse populations (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Rosenberg <em>et al.</em> 2010</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Wojcik <em>et al.</em> 2017</a>). This is especially pertinent to the <em>GSTM</em> locus, given that environmental toxins and carcinogens, which may vary from one population to the other, are the primary target for this gene family. Such loci are also targets for local selection in humans, may be best exemplified by the recent study on Arsenic adaptation in Argentinian Andes population (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Schlebusch <em>et al.</em> 2015</a>).</p></div></div><div class="section conclusions" id="sec-22"><h2 class="">Conclusion</h2><p id="p-64">In this study, we report a particular haplogroup (<em>Tanuki</em> haplogroup) carrying the deletion allele that has likely evolved under non-neutral conditions and reached a high allele frequency in East Asian populations. This haplogroup has a broad regulatory effect on the metabolizing <em>GSTM</em> gene family. Overall, our study adds to the emerging notion that complex loci involving and structural variants may contribute to adaptive and biomedically relevant phenotypic variation (<a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Boettger <em>et al.</em> 2016</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Sekar <em>et al.</em> 2016</a>).</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-65">This study is supported by MS’s fund from Astellas Foundation for Research on Metabolic Disorders, Graduate Program for Leaders in Life Innovation, The University of Tokyo Life Innovation Leading Graduate School from MEXT, Japan and Grant-in-Aid for Japan Society for the Promotion of Science (JSPS) Fellows Grant Number 264456. This work is also supported by OG’s start-up funds from University at Buffalo Research Foundation.</p><p id="p-66">We would like to thank Skyler Resendez for careful reading of the earlier versions of this manuscript. We would like to thank Takafumi Ishida, Jun Ohashi, Naoko Fujito and Mehmet Somel for discussions with regards to population genetic analyses. This study constitutes a part of the doctoral thesis of Marie Saitou submitted to The University of Tokyo, Japan.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-67">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6384782">https://doi.org/10.25387/g3.6384782</a>.</p></li><li class="fn" id="fn-2"><p id="p-68"><em>Communicating editor: S. Tishkoff</em></p></li><li class="fn" id="fn-3"><p id="p-69">Figures: Saitou, Marie; Gokcumen, Omer; Satta, Yoko (2018): G3-Supp0626.pdf. figshare. Dataset.</p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-05-29"><span class="received-label">Received </span>May 29, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-07-09"><span class="accepted-label">Accepted </span>July 9, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 SAITOU <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.1" data-doi="10.1038/nature09534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">1000 Genomes Project Consortium,</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altshuler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A map of human genome variation from population-scale sequencing.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">1061</span>–<span class="cit-lpage">1073</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature09534</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDurbin%26rft.auinit1%253DR.%2BM.%26rft.volume%253D467%26rft.issue%253D7319%26rft.spage%253D1061%26rft.epage%253D1073%26rft.atitle%253DA%2Bmap%2Bof%2Bhuman%2Bgenome%2Bvariation%2Bfrom%2Bpopulation-scale%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09534%26rft_id%253Dinfo%253Apmid%252F20981092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20981092&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283548600039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.9.2953.2" data-doi="10.1038/nature11632"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">1000 Genomes Project Consortium,</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An integrated map of genetic variation from 1,092 human genomes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">491</span>: <span class="cit-fpage">56</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAbecasis%26rft.auinit1%253DG.%2BR.%26rft.volume%253D491%26rft.issue%253D7422%26rft.spage%253D56%26rft.epage%253D65%26rft.atitle%253DAn%2Bintegrated%2Bmap%2Bof%2Bgenetic%2Bvariation%2Bfrom%2B1%252C092%2Bhuman%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11632%26rft_id%253Dinfo%253Apmid%252F23128226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11632&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23128226&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310434500030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.3" data-doi="10.1038/srep02704"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bhattacharjee</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banerjee</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patra</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banerjee</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Functional compensation of glutathione S-transferase M1 (GSTM1) null by another GST superfamily member, GSTM2.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">2704</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep02704</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D3%26rft.spage%253D2704%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep02704%26rft_id%253Dinfo%253Apmid%252F24048194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep02704&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24048194&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.4" data-doi="10.1038/ng.3510"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boettger</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salem</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peloso</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kathiresan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Recurring exon deletions in the HP (haptoglobin) gene contribute to lower blood cholesterol levels.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">359</span>–<span class="cit-lpage">366</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3510</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D48%26rft.spage%253D359%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3510&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.5" data-doi="10.2353/jmoldx.2007.070030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buchard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dalhoff</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morling</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Multiplex PCR detection of GSTM1, GSTT1, and GSTP1 gene variants: simultaneously detecting GSTM1 and GSTT1 gene copy number and the allelic status of the GSTP1 Ile105Val genetic variant.</span> <abbr class="cit-jnl-abbrev">J. Mol. Diagn.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">612</span>–<span class="cit-lpage">617</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2353/jmoldx.2007.070030</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BDiagn.%26rft_id%253Dinfo%253Adoi%252F10.2353%252Fjmoldx.2007.070030%26rft_id%253Dinfo%253Apmid%252F17916600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2353/jmoldx.2007.070030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17916600&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.6" data-doi="10.1101/gr.100545.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Population differentiation as a test for selective sweeps.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">393</span>–<span class="cit-lpage">402</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.100545.109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.100545.109%26rft_id%253Dinfo%253Apmid%252F20086244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzMvMzkzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzI5NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.7" data-doi="10.1056/NEJMoa1502214"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Claussnitzer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dankel</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">K.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quon</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuleman</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">FTO Obesity Variant Circuitry and Adipocyte Browning in Humans.</span> <abbr class="cit-jnl-abbrev">N. Engl. J. Med.</abbr> <span class="cit-vol">373</span>: <span class="cit-fpage">895</span>–<span class="cit-lpage">907</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1056/NEJMoa1502214</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BEngl.%2BJ.%2BMed.%26rft.volume%253D373%26rft.spage%253D895%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1502214%26rft_id%253Dinfo%253Apmid%252F26287746%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa1502214&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26287746&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.8" data-doi="10.1038/nature08516"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conrad</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pinto</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Redon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feuk</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gokcumen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Origins and functional impact of copy number variation in the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span>: <span class="cit-fpage">704</span>–<span class="cit-lpage">712</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature08516</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DConrad%26rft.auinit1%253DD.%2BF.%26rft.volume%253D464%26rft.issue%253D7289%26rft.spage%253D704%26rft.epage%253D712%26rft.atitle%253DOrigins%2Band%2Bfunctional%2Bimpact%2Bof%2Bcopy%2Bnumber%2Bvariation%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08516%26rft_id%253Dinfo%253Apmid%252F19812545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08516&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19812545&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276205000035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.9" data-doi="10.3389/fgene.2013.00235"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crisci</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poh</span>  <span class="cit-name-given-names">Y.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahajan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The impact of equilibrium assumptions on tests of selection.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">235</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2013.00235</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D4%26rft.spage%253D235%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2013.00235%26rft_id%253Dinfo%253Apmid%252F24273554%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2013.00235&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24273554&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.10" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr330</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21653522&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292778700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.11" data-doi="10.1161/ATVBAHA.110.207449"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ricciotti</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">FitzGerald</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Prostaglandins and Inflammation.</span> <abbr class="cit-jnl-abbrev">Arterioscler. Thromb. Vasc. Biol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">986</span>–<span class="cit-lpage">1000</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1161/ATVBAHA.110.207449</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArteriosclerosis%252C%2BThrombosis%252C%2Band%2BVascular%2BBiology%26rft.stitle%253DArterioscler.%2BThromb.%2BVasc.%2BBio.%26rft.aulast%253DRicciotti%26rft.auinit1%253DE.%26rft.volume%253D31%26rft.issue%253D5%26rft.spage%253D986%26rft.epage%253D1000%26rft.atitle%253DProstaglandins%2Band%2BInflammation%26rft_id%253Dinfo%253Adoi%252F10.1161%252FATVBAHA.110.207449%26rft_id%253Dinfo%253Apmid%252F21508345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYXR2YmFoYSI7czo1OiJyZXNpZCI7czo4OiIzMS81Lzk4NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOS8yOTUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.12" data-doi="10.1038/nrg1767"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feuk</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carson</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scherer</span>  <span class="cit-name-given-names">S. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Structural variation in the human genome.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">97</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg1767</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DFeuk%26rft.auinit1%253DL.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253D85%26rft.epage%253D97%26rft.atitle%253DStructural%2Bvariation%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1767%26rft_id%253Dinfo%253Apmid%252F16418744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1767&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16418744&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234714000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.13" data-doi="10.1016/j.legalmed.2009.01.073"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fujihara</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yasuda</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iida</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takatsuka</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fujii</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Cytochrome P450 1A1, glutathione S-transferases M1 and T1 polymorphisms in Ovambos and Mongolians.</span> <abbr class="cit-jnl-abbrev">Leg. Med. (Tokyo)</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">S408</span>–<span class="cit-lpage">S410</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.legalmed.2009.01.073</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLeg.%2BMed.%2B%2528Tokyo%2529%26rft.volume%253D11%26rft.spage%253DS408%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.legalmed.2009.01.073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.legalmed.2009.01.073&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.14" data-doi="10.1126/science.aab2319"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fumagalli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moltke</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grarup</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Racimo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bjerregaard</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Greenlandic Inuit show genetic signatures of diet and climate adaptation.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">349</span>: <span class="cit-fpage">1343</span>–<span class="cit-lpage">1347</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.aab2319</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DFumagalli%26rft.auinit1%253DM.%26rft.volume%253D349%26rft.issue%253D6254%26rft.spage%253D1343%26rft.epage%253D1347%26rft.atitle%253DGreenlandic%2BInuit%2Bshow%2Bgenetic%2Bsignatures%2Bof%2Bdiet%2Band%2Bclimate%2Badaptation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aab2319%26rft_id%253Dinfo%253Apmid%252F26383953%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDkvNjI1NC8xMzQzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzI5NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.15" data-doi="10.1016/S0140-6736(05)67137-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">García-Closas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malats</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silverman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dosemeci</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kogevinas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">NAT2 slow acetylation, GSTM1 null genotype, and risk of bladder cancer: results from the Spanish Bladder Cancer Study and meta-analyses.</span> <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">366</span>: <span class="cit-fpage">649</span>–<span class="cit-lpage">659</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0140-6736(05)67137-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.stitle%253DLancet%26rft.aulast%253DGarc%25C3%25ADa-Closas%26rft.auinit1%253DM.%26rft.volume%253D366%26rft.issue%253D9486%26rft.spage%253D649%26rft.epage%253D659%26rft.atitle%253DNAT2%2Bslow%2Bacetylation%252C%2BGSTM1%2Bnull%2Bgenotype%252C%2Band%2Brisk%2Bof%2Bbladder%2Bcancer%253A%2Bresults%2Bfrom%2Bthe%2BSpanish%2BBladder%2BCancer%2BStudy%2Band%2Bmeta-analyses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252805%252967137-1%26rft_id%253Dinfo%253Apmid%252F16112301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(05)67137-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16112301&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231426400028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garte</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaspari</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alexandrie</span>  <span class="cit-name-given-names">A.-K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ambrosone</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Autrup</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Metabolic Gene Polymorphism Frequencies in Control Populations.</span> <abbr class="cit-jnl-abbrev">Cancer Epidemiol. Biomarkers Prev.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1239</span>–<span class="cit-lpage">1248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BEpidemiology%2BBiomarkers%2B%2526%2BPrevention%26rft.stitle%253DCancer%2BEpidemiol.%2BBiomarkers%2BPrev.%26rft.aulast%253DGarte%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.issue%253D12%26rft.spage%253D1239%26rft.epage%253D1248%26rft.atitle%253DMetabolic%2BGene%2BPolymorphism%2BFrequencies%2Bin%2BControl%2BPopulations%26rft_id%253Dinfo%253Apmid%252F11751440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiY2VicCI7czo1OiJyZXNpZCI7czoxMDoiMTAvMTIvMTIzOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOS8yOTUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.17" data-doi="10.1002/ajpa.10128"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaspar</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hutz</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzano</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurtado</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Polymorphisms of CYP1a1, CYP2e1, GSTM1, GSTT1, and TP53 genes in Amerindians.</span> <abbr class="cit-jnl-abbrev">Am. J. Phys. Anthropol.</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">256</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/ajpa.10128</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BPhysical%2BAnthropology%26rft.stitle%253DAmerican%2BJournal%2Bof%2BPhysical%2BAnthropology%26rft.aulast%253DGaspar%26rft.auinit1%253DP.%2BA.%26rft.volume%253D119%26rft.issue%253D3%26rft.spage%253D249%26rft.epage%253D256%26rft.atitle%253DPolymorphisms%2Bof%2BCYP1a1%252C%2BCYP2e1%252C%2BGSTM1%252C%2BGSTT1%252C%2Band%2BTP53%2Bgenes%2Bin%2BAmerindians.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fajpa.10128%26rft_id%253Dinfo%253Apmid%252F12365037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/ajpa.10128&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12365037&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.18" data-doi="10.1146/annurev-genet-102209-163544"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Girirajan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eichler</span>  <span class="cit-name-given-names">E. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Human copy number variation and complex genetic disease.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">226</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-genet-102209-163544</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DGirirajan%26rft.auinit1%253DS.%26rft.volume%253D45%26rft.spage%253D203%26rft.epage%253D226%26rft.atitle%253DHuman%2Bcopy%2Bnumber%2Bvariation%2Band%2Bcomplex%2Bgenetic%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genet-102209-163544%26rft_id%253Dinfo%253Apmid%252F21854229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genet-102209-163544&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21854229&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299299600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.19" data-doi="10.1038/nrg2593"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hastings</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lupski</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ira</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mechanisms of change in gene copy number.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">551</span>–<span class="cit-lpage">564</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2593</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DHastings%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D10%26rft.issue%253D8%26rft.spage%253D551%26rft.epage%253D564%26rft.atitle%253DMechanisms%2Bof%2Bchange%2Bin%2Bgene%2Bcopy%2Bnumber.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2593%26rft_id%253Dinfo%253Apmid%252F19597530%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2593&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19597530&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268038200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.20" data-doi="10.1146/annurev.pharmtox.45.120403.095857"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flanagan</span>  <span class="cit-name-given-names">J. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jowsey</span>  <span class="cit-name-given-names">I. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Glutathione transferases.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Pharmacol. Toxicol.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">88</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.pharmtox.45.120403.095857</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bpharmacology%26rft.stitle%253DAnnu%2BRev%2BPharmacol%26rft.volume%253D45%26rft.spage%253D51%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.pharmtox.45.120403.095857%26rft_id%253Dinfo%253Apmid%252F15822171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.pharmtox.45.120403.095857&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15822171&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227504000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.21" data-doi="10.1126/science.1198878"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elyashiv</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melton</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Classic selective sweeps were rare in recent human evolution.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">331</span>: <span class="cit-fpage">920</span>–<span class="cit-lpage">924</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1198878</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHernandez%26rft.auinit1%253DR.%2BD.%26rft.volume%253D331%26rft.issue%253D6019%26rft.spage%253D920%26rft.epage%253D924%26rft.atitle%253DClassic%2BSelective%2BSweeps%2BWere%2BRare%2Bin%2BRecent%2BHuman%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1198878%26rft_id%253Dinfo%253Apmid%252F21330547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzEvNjAxOS85MjAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMjk1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.22" data-doi="10.5808/GI.2012.10.2.117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Sample size and statistical power calculation in genetic association studies.</span> <abbr class="cit-jnl-abbrev">Genomics Inform.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">117</span>–<span class="cit-lpage">122</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.5808/GI.2012.10.2.117</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%2BInform.%26rft.volume%253D10%26rft.spage%253D117%26rft_id%253Dinfo%253Adoi%252F10.5808%252FGI.2012.10.2.117%26rft_id%253Dinfo%253Apmid%252F23105939%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5808/GI.2012.10.2.117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23105939&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.23" data-doi="10.1093/bioinformatics/18.2.337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Generating samples under a Wright-Fisher neutral model of genetic variation.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">338</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/18.2.337</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DHudson%26rft.auinit1%253DR.%2BR.%26rft.volume%253D18%26rft.issue%253D2%26rft.spage%253D337%26rft.epage%253D338%26rft.atitle%253DGenerating%2Bsamples%2Bunder%2Ba%2BWright-Fisher%2Bneutral%2Bmodel%2Bof%2B%2Bgenetic%2Bvariation%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F18.2.337%26rft_id%253Dinfo%253Apmid%252F11847089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/18.2.337&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11847089&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174028100021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.24" data-doi="10.1186/1471-2105-8-460"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huson</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richter</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rausch</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dezulian</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Dendroscope: An interactive viewer for large phylogenetic trees.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">460</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-8-460</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHuson%26rft.auinit1%253DD.%2BH.%26rft.volume%253D8%26rft.spage%253D460%26rft.epage%253D460%26rft.atitle%253DDendroscope%253A%2BAn%2Binteractive%2Bviewer%2Bfor%2Blarge%2Bphylogenetic%2Btrees.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-8-460%26rft_id%253Dinfo%253Apmid%252F18034891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-8-460&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18034891&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.25" data-doi="10.1038/nature06258"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">International HapMap Consortium,</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frazer</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ballinger</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinds</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A second generation human haplotype map of over 3.1 million SNPs.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">449</span>: <span class="cit-fpage">851</span>–<span class="cit-lpage">861</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature06258</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DFrazer%26rft.auinit1%253DK.%2BA.%26rft.volume%253D449%26rft.issue%253D7164%26rft.spage%253D851%26rft.epage%253D861%26rft.atitle%253DA%2Bsecond%2Bgeneration%2Bhuman%2Bhaplotype%2Bmap%2Bof%2Bover%2B3.1%2Bmillion%2BSNPs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06258%26rft_id%253Dinfo%253Apmid%252F17943122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06258&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17943122&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250230600036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.26" data-doi="10.1016/j.gpb.2014.10.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zuo</span>  <span class="cit-name-given-names">X.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">W.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">M.-Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Pathway-based analysis tools for complex diseases: a review.</span> <abbr class="cit-jnl-abbrev">Genomics Proteomics Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">210</span>–<span class="cit-lpage">220</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.gpb.2014.10.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%2BProteomics%2BBioinformatics%26rft.volume%253D12%26rft.spage%253D210%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gpb.2014.10.002%26rft_id%253Dinfo%253Apmid%252F25462153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gpb.2014.10.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25462153&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.27" data-doi="10.1186/s13039-016-0241-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khrunin</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Filippova</span>  <span class="cit-name-given-names">I. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aliev</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tupitsina</span>  <span class="cit-name-given-names">T. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slominsky</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">GSTM1 copy number variation in the context of single nucleotide polymorphisms in the human GSTM cluster.</span> <abbr class="cit-jnl-abbrev">Mol. Cytogenet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">30</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13039-016-0241-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCytogenet.%26rft.volume%253D9%26rft.spage%253D30%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13039-016-0241-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13039-016-0241-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.28" data-doi="10.1534/genetics.107.083691"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Satta</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Population genetic analysis of the N-acylsphingosine amidohydrolase gene associated with mental activity in humans.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">1505</span>–<span class="cit-lpage">1515</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.083691</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.083691%26rft_id%253Dinfo%253Apmid%252F18245333%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC8zLzE1MDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMjk1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohta</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">The age of a neutral mutant persisting in a finite population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">199</span>–<span class="cit-lpage">212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKimura%26rft.auinit1%253DM.%26rft.volume%253D75%26rft.issue%253D1%26rft.spage%253D199%26rft.epage%253D212%26rft.atitle%253DTHE%2BAGE%2BOF%2BA%2BNEUTRAL%2BMUTANT%2BPERSISTING%2BIN%2BA%2BFINITE%2BPOPULATION%26rft_id%253Dinfo%253Apmid%252F4762875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzUvMS8xOTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMjk1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.30" data-doi="10.1038/ng.2892"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kircher</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witten</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Roak</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A general framework for estimating the relative pathogenicity of human genetic variants.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">310</span>–<span class="cit-lpage">315</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2892</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D310%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2892%26rft_id%253Dinfo%253Apmid%252F24487276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2892&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24487276&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.31" data-doi="10.1038/nature09525"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorleifsson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gudbjartsson</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sigurdsson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fine-scale recombination rate differences between sexes, populations and individuals.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">1099</span>–<span class="cit-lpage">1103</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature09525</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKong%26rft.auinit1%253DA.%26rft.volume%253D467%26rft.issue%253D7319%26rft.spage%253D1099%26rft.epage%253D1103%26rft.atitle%253DFine-scale%2Brecombination%2Brate%2Bdifferences%2Bbetween%2Bsexes%252C%2Bpopulations%2Band%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09525%26rft_id%253Dinfo%253Apmid%252F20981099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09525&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20981099&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283548600046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.32" data-doi="10.1126/science.aam6393"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leffler</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Band</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Busby</span>  <span class="cit-name-given-names">G. B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kivinen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le</span>  <span class="cit-name-given-names">Q. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Resistance to malaria through structural variation of red blood cell invasion receptors.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">356</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.aam6393</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aam6393%26rft_id%253Dinfo%253Apmid%252F28522690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE3OiIzNTYvNjM0My9lYWFtNjM5MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOS8yOTUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.33" data-doi="10.1038/nature19057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karczewski</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minikel</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samocha</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Analysis of protein-coding genetic variation in 60,706 humans.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">536</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">291</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature19057</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D536%26rft.spage%253D285%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature19057%26rft_id%253Dinfo%253Apmid%252F27535533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature19057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27535533&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.34" data-doi="10.1093/bioinformatics/btp187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Librado</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rozas</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">DnaSP v5: A software for comprehensive analysis of DNA polymorphism data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1451</span>–<span class="cit-lpage">1452</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp187</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp187%26rft_id%253Dinfo%253Apmid%252F19346325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp187&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19346325&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266109500026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.35" data-doi="10.1093/molbev/msu405"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavlidis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karakoc</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ajay</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gokcumen</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The evolution and functional impact of human deletion variants shared with archaic hominin genomes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1008</span>–<span class="cit-lpage">1019</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msu405</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu405%26rft_id%253Dinfo%253Apmid%252F25556237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu405&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25556237&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.36" data-doi="10.1136/jmg.2004.024752"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">P.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">J.-R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A survey of haplotype variants at several disease candidate genes: the importance of rare variants for complex diseases.</span> <abbr class="cit-jnl-abbrev">J. Med. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">221</span>–<span class="cit-lpage">227</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1136/jmg.2004.024752</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMedical%2BGenetics%26rft.stitle%253DJ.%2BMed.%2BGenet.%26rft.aulast%253DLiu%26rft.auinit1%253DP-Y%26rft.volume%253D42%26rft.issue%253D3%26rft.spage%253D221%26rft.epage%253D227%26rft.atitle%253DA%2Bsurvey%2Bof%2Bhaplotype%2Bvariants%2Bat%2Bseveral%2Bdisease%2Bcandidate%2Bgenes%253A%2Bthe%2Bimportance%2Bof%2Brare%2Bvariants%2Bfor%2Bcomplex%2Bdiseases%26rft_id%253Dinfo%253Adoi%252F10.1136%252Fjmg.2004.024752%26rft_id%253Dinfo%253Apmid%252F15744035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToiam1lZGdlbmV0IjtzOjU6InJlc2lkIjtzOjg6IjQyLzMvMjIxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzI5NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.37" data-doi="10.1038/ng.2653"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lonsdale</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salvatore</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The Genotype-Tissue Expression (GTEx) project.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">580</span>–<span class="cit-lpage">585</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2653</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D580%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2653%26rft_id%253Dinfo%253Apmid%252F23715323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2653&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23715323&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.38" data-doi="10.1093/nar/gkw1133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">MacArthur</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowler</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cerezo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gil</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The new NHGRI-EBI Catalog of published genome-wide association studies (GWAS Catalog).</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">D896</span>–<span class="cit-lpage">D901</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkw1133</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw1133%26rft_id%253Dinfo%253Apmid%252F27899670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw1133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27899670&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.39" data-doi="10.1038/nature07744"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marques-Bonet</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kidd</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ventura</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graves</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A burst of segmental duplications in the genome of the African great ape ancestor.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">457</span>: <span class="cit-fpage">877</span>–<span class="cit-lpage">881</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature07744</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMarques-Bonet%26rft.auinit1%253DT.%26rft.volume%253D457%26rft.issue%253D7231%26rft.spage%253D877%26rft.epage%253D881%26rft.atitle%253DA%2Bburst%2Bof%2Bsegmental%2Bduplications%2Bin%2Bthe%2Bgenome%2Bof%2Bthe%2BAfrican%2Bgreat%2Bape%2Bancestor.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07744%26rft_id%253Dinfo%253Apmid%252F19212409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07744&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212409&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263266700044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.40" data-doi="10.1038/sj.onc.1209373"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McIlwain</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Townsend</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tew</span>  <span class="cit-name-given-names">K. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Glutathione S-transferase polymorphisms: cancer incidence and therapy.</span> <abbr class="cit-jnl-abbrev">Oncogene</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1639</span>–<span class="cit-lpage">1648</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/sj.onc.1209373</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOncogene%26rft.stitle%253DOncogene%26rft.aulast%253DMcIlwain%26rft.auinit1%253DC.%2BC.%26rft.volume%253D25%26rft.issue%253D11%26rft.spage%253D1639%26rft.epage%253D1648%26rft.atitle%253DGlutathione%2BS-transferase%2Bpolymorphisms%253A%2Bcancer%2Bincidence%2Band%2Btherapy.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.onc.1209373%26rft_id%253Dinfo%253Apmid%252F16550164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.onc.1209373&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16550164&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235904700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.41" data-doi="10.1038/nature11632"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">1000 Genomes Project Consortium,</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An integrated map of genetic variation from 1,092 human genomes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">491</span>: <span class="cit-fpage">56</span>–<span class="cit-lpage">65</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature11632</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAbecasis%26rft.auinit1%253DG.%2BR.%26rft.volume%253D491%26rft.issue%253D7422%26rft.spage%253D56%26rft.epage%253D65%26rft.atitle%253DAn%2Bintegrated%2Bmap%2Bof%2Bgenetic%2Bvariation%2Bfrom%2B1%252C092%2Bhuman%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11632%26rft_id%253Dinfo%253Apmid%252F23128226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11632&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23128226&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310434500030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.42" data-doi="10.1126/science.1092500"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mc Vean</span>  <span class="cit-name-given-names">G. A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deloukas</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bentley</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The Fine-Scale Structureof Recombination Rate Variationin the Human Genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">304</span>: <span class="cit-fpage">581</span>–<span class="cit-lpage">584</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMcVean%26rft.auinit1%253DG.%2BA.%2BT.%26rft.volume%253D304%26rft.issue%253D5670%26rft.spage%253D581%26rft.epage%253D584%26rft.atitle%253DThe%2BFine-Scale%2BStructure%2Bof%2BRecombination%2BRate%2BVariation%2Bin%2Bthe%2BHuman%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1092500%26rft_id%253Dinfo%253Apmid%252F15105499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDQvNTY3MC81ODEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMjk1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.43" data-doi="10.1073/pnas.1600374113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moorjani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amorim</span>  <span class="cit-name-given-names">C. E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arndt</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przeworski</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Variation in the molecular clock of primates.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">10607</span>–<span class="cit-lpage">10612</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1600374113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1600374113%26rft_id%253Dinfo%253Apmid%252F27601674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEzLzM4LzEwNjA3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzI5NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.44" data-doi="10.2217/pgs.11.88"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mortensen</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Froment</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lema</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bodo</span>  <span class="cit-name-given-names">J.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ibrahim</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Characterization of genetic variation and natural selection at the arylamine N-acetyltransferase genes in global human populations.</span> <abbr class="cit-jnl-abbrev">Pharmacogenomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1545</span>–<span class="cit-lpage">1558</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2217/pgs.11.88</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPharmacogenomics%26rft.stitle%253DPharmacogenomics%26rft.aulast%253DMortensen%26rft.auinit1%253DH.%2BM.%26rft.volume%253D12%26rft.issue%253D11%26rft.spage%253D1545%26rft.epage%253D1558%26rft.atitle%253DCharacterization%2Bof%2Bgenetic%2Bvariation%2Band%2Bnatural%2Bselection%2Bat%2Bthe%2Barylamine%2BN-acetyltransferase%2Bgenes%2Bin%2Bglobal%2Bhuman%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.2217%252Fpgs.11.88%26rft_id%253Dinfo%253Apmid%252F21995608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2217/pgs.11.88&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21995608&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000297461900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.45" data-doi="10.1158/1078-0432.CCR-07-0635"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moyer</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salavaggione</span>  <span class="cit-name-given-names">O. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hebbring</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moon</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hildebrandt</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Glutathione S-transferase T1 and M1: gene sequence variation and functional genomics.</span> <abbr class="cit-jnl-abbrev">Clin. Cancer Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">7207</span>–<span class="cit-lpage">7216</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1158/1078-0432.CCR-07-0635</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2BCancer%2BResearch%26rft.stitle%253DClin.%2BCancer%2BRes.%26rft.aulast%253DMoyer%26rft.auinit1%253DA.%2BM.%26rft.volume%253D13%26rft.issue%253D23%26rft.spage%253D7207%26rft.epage%253D7216%26rft.atitle%253DGlutathione%2BS-Transferase%2BT1%2Band%2BM1%253A%2BGene%2BSequence%2BVariation%2Band%2BFunctional%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1158%252F1078-0432.CCR-07-0635%26rft_id%253Dinfo%253Apmid%252F18056202%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6ImNsaW5jYW5yZXMiO3M6NToicmVzaWQiO3M6MTA6IjEzLzIzLzcyMDciO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzkvMjk1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.46" data-doi="10.1038/s41467-017-00323-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Narasimhan</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rahbari</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scally</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wuster</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mason</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Estimating the human mutation rate from autozygous segments reveals population differences in human mutational processes.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41467-017-00323-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D8%26rft.spage%253D303%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-017-00323-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-017-00323-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.47" data-doi="10.1073/pnas.76.10.5269"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Mathematical model for studying genetic variation in terms of restriction endonucleases.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">5269</span>–<span class="cit-lpage">5273</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.76.10.5269</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DNei%26rft.auinit1%253DM.%26rft.volume%253D76%26rft.issue%253D10%26rft.spage%253D5269%26rft.epage%253D5273%26rft.atitle%253DMathematical%2Bmodel%2Bfor%2Bstudying%2Bgenetic%2Bvariation%2Bin%2Bterms%2Bof%2Brestriction%2Bendonucleases.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.76.10.5269%26rft_id%253Dinfo%253Apmid%252F291943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiNzYvMTAvNTI2OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOS8yOTUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.48" data-doi="10.1186/s12862-015-0575-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pajic</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Y.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gokcumen</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The psoriasis associated deletion of late cornified envelope genes and has been maintained under balancing selection since Human Denisovan divergence.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol.%2BBiol.%26rft.volume%253D16%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12862-015-0575-y%26rft_id%253Dinfo%253Apmid%252F26727998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12862-015-0575-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26727998&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.49" data-doi="10.1016/j.canlet.2004.06.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parl</span>  <span class="cit-name-given-names">F. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Glutathione S-transferase genotypes and cancer risk.</span> <abbr class="cit-jnl-abbrev">Cancer Lett.</abbr> <span class="cit-vol">221</span>: <span class="cit-fpage">123</span>–<span class="cit-lpage">129</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.canlet.2004.06.016</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2Bletters%26rft.stitle%253DCancer%2BLett%26rft.aulast%253DParl%26rft.auinit1%253DF.%2BF.%26rft.volume%253D221%26rft.issue%253D2%26rft.spage%253D123%26rft.epage%253D129%26rft.atitle%253DGlutathione%2BS-transferase%2Bgenotypes%2Band%2Bcancer%2Brisk.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.canlet.2004.06.016%26rft_id%253Dinfo%253Apmid%252F15808397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.canlet.2004.06.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15808397&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228607000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.50" data-doi="10.1038/ng2123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perry</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dominy</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Claw</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fiegler</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Diet and the evolution of human amylase gene copy number variation.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1256</span>–<span class="cit-lpage">1260</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng2123</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DPerry%26rft.auinit1%253DG.%2BH.%26rft.volume%253D39%26rft.issue%253D10%26rft.spage%253D1256%26rft.epage%253D1260%26rft.atitle%253DDiet%2Band%2Bthe%2Bevolution%2Bof%2Bhuman%2Bamylase%2Bgene%2Bcopy%2Bnumber%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2123%26rft_id%253Dinfo%253Apmid%252F17828263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17828263&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249737400022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.51" data-doi="10.1007/s11033-010-0221-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piacentini</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Polimanti</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Porreca</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martínez-Labarga</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Stefano</span>  <span class="cit-name-given-names">G. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">GSTT1 and GSTM1 gene polymorphisms in European and African populations.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Rep.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1225</span>–<span class="cit-lpage">1230</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11033-010-0221-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiology%2Breports%26rft.stitle%253DMol%2BBiol%2BRep%26rft.aulast%253DPiacentini%26rft.auinit1%253DS.%26rft.volume%253D38%26rft.issue%253D2%26rft.spage%253D1225%26rft.epage%253D1230%26rft.atitle%253DGSTT1%2Band%2BGSTM1%2Bgene%2Bpolymorphisms%2Bin%2BEuropean%2Band%2BAfrican%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11033-010-0221-0%26rft_id%253Dinfo%253Apmid%252F20563854%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11033-010-0221-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20563854&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.52" data-doi="10.1038/nature12886"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prüfer</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Racimo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jay</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sankararaman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The complete genome sequence of a Neanderthal from the Altai Mountains.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">505</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">49</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature12886</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPruefer%26rft.auinit1%253DK.%26rft.volume%253D505%26rft.issue%253D7481%26rft.spage%253D43%26rft.epage%253D49%26rft.atitle%253DThe%2Bcomplete%2Bgenome%2Bsequence%2Bof%2Ba%2BNeanderthal%2Bfrom%2Bthe%2BAltai%2BMountains%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12886%26rft_id%253Dinfo%253Apmid%252F24352235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12886&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2014009693&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=24352235&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000329163300020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.53" data-doi="10.1093/nar/gkt1188"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pybus</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dall’Olio</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luisi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uzkudun</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carreño-Torres</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">1000 Genomes Selection Browser 1.0: a genome browser dedicated to signatures of natural selection in modern humans.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">D903</span>–<span class="cit-lpage">D909</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkt1188</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1188%26rft_id%253Dinfo%253Apmid%252F24275494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt1188&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24275494&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331139800132&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.54" data-doi="10.1038/nrg3936"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Racimo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sankararaman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huerta-Sánchez</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Evidence for archaic adaptive introgression in humans.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">359</span>–<span class="cit-lpage">371</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3936</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D16%26rft.spage%253D359%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3936%26rft_id%253Dinfo%253Apmid%252F25963373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3936&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25963373&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.55" data-doi="10.1038/nature05329"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Redon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishikawa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fitch</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feuk</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perry</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Global variation in copy number in the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">444</span>: <span class="cit-fpage">444</span>–<span class="cit-lpage">454</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature05329</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRedon%26rft.auinit1%253DR.%26rft.volume%253D444%26rft.issue%253D7118%26rft.spage%253D444%26rft.epage%253D454%26rft.atitle%253DGlobal%2Bvariation%2Bin%2Bcopy%2Bnumber%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05329%26rft_id%253Dinfo%253Apmid%252F17122850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05329&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17122850&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242215700038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.56" data-doi="10.1038/nature09710"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kircher</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krause</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic history of an archaic hominin group from Denisova Cave in Siberia.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">468</span>: <span class="cit-fpage">1053</span>–<span class="cit-lpage">1060</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature09710</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DReich%26rft.auinit1%253DD.%26rft.volume%253D468%26rft.issue%253D7327%26rft.spage%253D1053%26rft.epage%253D1060%26rft.atitle%253DGenetic%2Bhistory%2Bof%2Ban%2Barchaic%2Bhominin%2Bgroup%2Bfrom%2BDenisova%2BCave%2Bin%2BSiberia.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09710%26rft_id%253Dinfo%253Apmid%252F21179161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09710&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2011015829&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=21179161&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285553800049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.57" data-doi="10.1038/nrg2760"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jewett</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szpiech</span>  <span class="cit-name-given-names">Z. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jankovic</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide association studies in diverse populations.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">356</span>–<span class="cit-lpage">366</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2760</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DRosenberg%26rft.auinit1%253DN.%2BA.%26rft.volume%253D11%26rft.issue%253D5%26rft.spage%253D356%26rft.epage%253D366%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bin%2Bdiverse%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2760%26rft_id%253Dinfo%253Apmid%252F20395969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2760&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20395969&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276771400013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.58" data-doi="10.1038/ng.687"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rothman</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Closas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chatterjee</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malats</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A multi-stage genome-wide association study of bladder cancer identifies multiple susceptibility loci.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">978</span>–<span class="cit-lpage">984</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.687</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DRothman%26rft.auinit1%253DN.%26rft.volume%253D42%26rft.issue%253D11%26rft.spage%253D978%26rft.epage%253D984%26rft.atitle%253DA%2Bmulti-stage%2Bgenome-wide%2Bassociation%2Bstudy%2Bof%2Bbladder%2Bcancer%2Bidentifies%2Bmultiple%2Bsusceptibility%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.687%26rft_id%253Dinfo%253Apmid%252F20972438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.687&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20972438&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283540500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.59" data-doi="10.1038/376246a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruwende</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khoo</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snow</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yates</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwiatkowski</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Natural selection of hemi- and heterozygotes for G6PD deficiency in Africa by resistance to severe malaria.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">376</span>: <span class="cit-fpage">246</span>–<span class="cit-lpage">249</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/376246a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRuwende%26rft.auinit1%253DC.%26rft.volume%253D376%26rft.issue%253D6537%26rft.spage%253D246%26rft.epage%253D249%26rft.atitle%253DNatural%2Bselection%2Bof%2Bhemi-%2Band%2Bheterozygotes%2Bfor%2BG6PD%2Bdeficiency%2Bin%2BAfrica%2Bby%2Bresistance%2Bto%2Bsevere%2Bmalaria.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F376246a0%26rft_id%253Dinfo%253Apmid%252F7617034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/376246a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7617034&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RK33100044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.60" data-doi="10.3923/ijcr.2007.74.91"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saadat</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">GSTM1 null genotype associated with age-standardized cancer mortality rate in 45 countries from five continents: an ecologic study.</span> <abbr class="cit-jnl-abbrev">Int. J. Cancer Res.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">74</span>–<span class="cit-lpage">91</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3923/ijcr.2007.74.91</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BCancer%2BRes.%26rft.volume%253D3%26rft.spage%253D74%26rft_id%253Dinfo%253Adoi%252F10.3923%252Fijcr.2007.74.91%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3923/ijcr.2007.74.91&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.61" data-doi="10.1038/nature06250"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sabeti</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varilly</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fry</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohmueller</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hostetter</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide detection and characterization of positive selection in human populations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">449</span>: <span class="cit-fpage">913</span>–<span class="cit-lpage">918</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature06250</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSabeti%26rft.auinit1%253DP.%2BC.%26rft.volume%253D449%26rft.issue%253D7164%26rft.spage%253D913%26rft.epage%253D918%26rft.atitle%253DGenome-wide%2Bdetection%2Band%2Bcharacterization%2Bof%2Bpositive%2Bselection%2Bin%2Bhuman%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06250%26rft_id%253Dinfo%253Apmid%252F17943131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06250&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17943131&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250230600049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.62" data-doi="10.1186/s12864-018-4676-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saitou</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Satta</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gokcumen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishida</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Complex evolution of the GSTM gene family involves sharing of GSTM1 deletion polymorphism in humans and chimpanzees.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">293</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-018-4676-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D19%26rft.spage%253D293%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-018-4676-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-018-4676-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.63" data-doi="10.1038/ng.3862"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salojärvi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smolander</span>  <span class="cit-name-given-names">O.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nieminen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rajaraman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Safronov</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome sequencing and population genomic analyses provide insights into the adaptive landscape of silver birch.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">904</span>–<span class="cit-lpage">912</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3862</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D49%26rft.spage%253D904%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3862&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.64" data-doi="10.1101/gr.3709305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaffner</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foo</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabriel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Calibrating a coalescent simulation of human genome sequence variation.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1576</span>–<span class="cit-lpage">1583</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.3709305</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DSchaffner%26rft.auinit1%253DS.%2BF.%26rft.volume%253D15%26rft.issue%253D11%26rft.spage%253D1576%26rft.epage%253D1583%26rft.atitle%253DCalibrating%2Ba%2Bcoalescent%2Bsimulation%2Bof%2Bhuman%2Bgenome%2Bsequence%2Bvariation%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3709305%26rft_id%253Dinfo%253Apmid%252F16251467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xMS8xNTc2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzI5NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.65" data-doi="10.1093/molbev/msv046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schlebusch</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gattepaille</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Engström</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vahter</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Human Adaptation to Arsenic-Rich Environments.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1544</span>–<span class="cit-lpage">1555</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msv046</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsv046%26rft_id%253Dinfo%253Apmid%252F25739736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msv046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25739736&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.66" data-doi="10.1093/molbev/msx154"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kern</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Soft Sweeps Are the Dominant Mode of Adaptation in the Human Genome.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">1863</span>–<span class="cit-lpage">1877</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msx154</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft.volume%253D34%26rft.spage%253D1863%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsx154%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msx154&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.67" data-doi="10.1038/nature16549"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sekar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bialas</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Rivera</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hammond</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Schizophrenia risk from complex variation of complement component 4.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">530</span>: <span class="cit-fpage">177</span>–<span class="cit-lpage">183</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature16549</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D530%26rft.spage%253D177%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature16549%26rft_id%253Dinfo%253Apmid%252F26814963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature16549&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26814963&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.68" data-doi="10.4161/fly.6875"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sheeley</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McAllister</span>  <span class="cit-name-given-names">B. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Patterns of natural selection at the Alcohol dehydrogenase gene of Drosophila americana.</span> <abbr class="cit-jnl-abbrev">Fly (Austin)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">246</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4161/fly.6875</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFly%2B%2528Austin%2529%26rft.volume%253D2%26rft.spage%253D243%26rft_id%253Dinfo%253Adoi%252F10.4161%252Ffly.6875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/fly.6875&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">South</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">rworldmap: A New R package for Mapping Global Data.</span> <abbr class="cit-jnl-abbrev">R J.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">35</span>–<span class="cit-lpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR%2BJ.%26rft.volume%253D3%26rft.spage%253D35%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.70" data-doi="10.1126/science.aab3761"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sudmant</span>  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mallick</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krumm</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huddleston</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">a</span> <span class="cit-article-title">Global diversity, population stratification, and selection of human copy number variation.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">349</span>: <span class="cit-fpage">aab3761</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.aab3761</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aab3761%26rft_id%253Dinfo%253Apmid%252F26249230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNDkvNjI1My9hYWIzNzYxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzI5NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.71" data-doi="10.1038/nature15394"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sudmant</span>  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rausch</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abyzov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">b</span> <span class="cit-article-title">An integrated map of structural variation in 2,504 human genomes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">526</span>: <span class="cit-fpage">75</span>–<span class="cit-lpage">81</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature15394</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D526%26rft.spage%253D75%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature15394%26rft_id%253Dinfo%253Apmid%252F26432246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature15394&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26432246&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.72"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTajima%26rft.auinit1%253DF.%26rft.volume%253D123%26rft.issue%253D3%26rft.spage%253D585%26rft.epage%253D595%26rft.atitle%253DStatistical%2Bmethod%2Bfor%2Btesting%2Bthe%2Bneutral%2Bmutation%2Bhypothesis%2Bby%2BDNA%2Bpolymorphism.%26rft_id%253Dinfo%253Apmid%252F2513255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIzLzMvNTg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzI5NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.73"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tatewaki</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maekawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atori</span>  <span class="cit-name-given-names">N. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Urose</span>  <span class="cit-name-given-names">K. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> Genetic Variations and Haplotype Structures of the Glutathione S-transferase Genes, GSTT1 and GSTM1, in a Japanese Patient Population Naoko. 24: 118–126.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.74" data-doi="10.1038/nature04072"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The Chimpanzee Sequencing Consortium</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Initial sequence of the chimpanzee genome and comparison with the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">437</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">87</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature04072</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D437%26rft.issue%253D7055%26rft.spage%253D69%26rft.epage%253D87%26rft.atitle%253DInitial%2Bsequence%2Bof%2Bthe%2Bchimpanzee%2Bgenome%2Band%2Bcomparison%2Bwith%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04072%26rft_id%253Dinfo%253Apmid%252F16136131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04072&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16136131&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231560400045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.75" data-doi="10.1038/ng.3340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Usher</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esko</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuke</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weedon</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Structural forms of the human amylase locus and their relationships to SNPs, haplotypes and obesity.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">921</span>–<span class="cit-lpage">925</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3340</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D921%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3340%26rft_id%253Dinfo%253Apmid%252F26098870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26098870&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.76" data-doi="10.1101/gr.3737405"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Varki</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altheide</span>  <span class="cit-name-given-names">T. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Comparing the human and chimpanzee genomes: searching for needles in a haystack.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1746</span>–<span class="cit-lpage">1758</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.3737405</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DVarki%26rft.auinit1%253DA.%26rft.volume%253D15%26rft.issue%253D12%26rft.spage%253D1746%26rft.epage%253D1758%26rft.atitle%253DComparing%2Bthe%2Bhuman%2Band%2Bchimpanzee%2Bgenomes%253A%2BSearching%2Bfor%2Bneedles%2Bin%2Ba%2Bhaystack%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3737405%26rft_id%253Dinfo%253Apmid%252F16339373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xMi8xNzQ2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC85LzI5NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.77" data-doi="10.1016/j.ajhg.2011.11.029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Five years of GWAS discovery.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">7</span>–<span class="cit-lpage">24</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2011.11.029</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DVisscher%26rft.auinit1%253DP.%2BM.%26rft.volume%253D90%26rft.issue%253D1%26rft.spage%253D7%26rft.epage%253D24%26rft.atitle%253DFive%2Byears%2Bof%2BGWAS%2Bdiscovery.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2011.11.029%26rft_id%253Dinfo%253Apmid%252F22243964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2011.11.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22243964&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-other" id="cit-8.9.2953.78"><div class="cit-metadata unstructured">Voight, B. F., S. Kudaravalli, X. Wen, and J. K. Pritchard, 2006 A map of recent positive selection in the human genome. PLoS Biol. 4: 0446–0458.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.79" data-doi="10.1371/journal.pgen.1006973"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Z. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Pathway-based discovery of genetic interactions in breast cancer.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">e1006973</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1006973</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D13%26rft.spage%253De1006973%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006973&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.80" data-doi="10.1111/j.1558-5646.1984.tb05657.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cockerham</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Estimating F-Statistics for the Analysis of Population Structure.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1358</span>–<span class="cit-lpage">1370</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.1984.tb05657.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D38%26rft.spage%253D1358%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.1984.tb05657.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.1984.tb05657.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.81" data-doi="10.1038/nrg3373"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weischenfeldt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Symmons</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spitz</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korbel</span>  <span class="cit-name-given-names">J. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Phenotypic impact of genomic structural variation: insights from and for human disease.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">125</span>–<span class="cit-lpage">138</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3373</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D14%26rft.spage%253D125%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3373%26rft_id%253Dinfo%253Apmid%252F23329113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3373&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23329113&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.82" data-doi="10.1038/nature08979"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Wellcome Trust Case Control Consortium,</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Craddock</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurles</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardin</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pearson</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide association study of CNVs in 16,000 cases of eight common diseases and 3,000 shared controls.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">720</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature08979</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D464%26rft.issue%253D7289%26rft.spage%253D713%26rft.epage%253D720%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2BCNVs%2Bin%2B16%252C000%2Bcases%2Bof%2Beight%2Bcommon%2Bdiseases%2Band%2B3%252C000%2Bshared%2Bcontrols.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08979%26rft_id%253Dinfo%253Apmid%252F20360734%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08979&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20360734&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276205000036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.83" data-doi="10.1371/journal.pgen.1006423"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wheeler</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brenner</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aquino-Michaels</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Survey of the Heritability and Sparse Architecture of Gene Expression Traits across Human Tissues.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1006423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1006423</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1006423%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006423&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.84" data-doi="10.1101/188094"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wojcik</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graff</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishimura</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tao</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haessler</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genetic Diversity Turns a New PAGE in Our Understanding of Complex Traits.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-fpage">188094</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/188094</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F188094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiIxODgwOTR2MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvOS8yOTUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.85" data-doi="10.1016/j.ajhg.2008.08.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Adaptive Evolution of UGT2B17 Copy-Number Variation.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">346</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2008.08.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DXue%26rft.auinit1%253DY.%26rft.volume%253D83%26rft.issue%253D3%26rft.spage%253D337%26rft.epage%253D346%26rft.atitle%253DAdaptive%2Bevolution%2Bof%2BUGT2B17%2Bcopy-number%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2008.08.004%26rft_id%253Dinfo%253Apmid%252F18760392%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2008.08.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18760392&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259307200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.86" data-doi="10.1186/s12859-017-1844-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaber</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavlidis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gokcumen</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">a</span> <span class="cit-article-title">VCFtoTree: a user-friendly tool to construct locus-specific alignments and phylogenies from thousands of anthropologically relevant genome sequences.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">426</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-017-1844-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D18%26rft.spage%253D426%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-017-1844-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-017-1844-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.87" data-doi="10.1093/molbev/msx206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavlidis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taskent</span>  <span class="cit-name-given-names">R. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alachiotis</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flanagan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">b</span> <span class="cit-article-title">Archaic hominin introgression in Africa contributes to functional salivary MUC7 genetic variation.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">2704</span>–<span class="cit-lpage">2715</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msx206</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft.volume%253D34%26rft.spage%253D2704%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsx206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msx206&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-other" id="cit-8.9.2953.88"><div class="cit-metadata unstructured">Xu, S.-J., Y.-P. Wang, B. Roe, W. R. Pearson, and G. G. Deletion, 1998 Characterization of the Human Class Mu Glutathione S -Transferase Gene Cluster and the GSTM1 Deletion <sup>*</sup>. 273: 3517–3527.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-8.9.2953.89" data-doi="10.1146/annurev.genom.9.081307.164217"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurles</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lupski</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Copy number variation in human health, disease, and evolution.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">481</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.genom.9.081307.164217</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenomics%2BHum%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DF.%26rft.volume%253D10%26rft.spage%253D451%26rft.epage%253D481%26rft.atitle%253DCopy%2Bnumber%2Bvariation%2Bin%2Bhuman%2Bhealth%252C%2Bdisease%252C%2Band%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genom.9.081307.164217%26rft_id%253Dinfo%253Apmid%252F19715442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genom.9.081307.164217&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19715442&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F9%2F2953.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270313200021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/9/2953.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/9/2941" title="Acute Effects of Drugs on Caenorhabditis elegans Movement Reveal Complex Responses and Plasticity" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/9/2967" title="Whole Genome Sequencing, de Novo Assembly and Phenotypic Profiling for the New Budding Yeast Species Saccharomyces jurei" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/9"><span>Volume 8 Issue 9, September 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/9.cover-source.jpg?itok=77yoxn8o" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/9/2953.supplemental" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0OFppr_d_I7rNtNUGYmv9VhjGjgdkoAVFfLeqoJSAl0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/9/2953" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/9/2953" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/9/2953" class="active" data-icon-position="" data-hide-link-title="0">Complex Haplotypes of GSTM1 Gene Deletions Harbor Signatures of a Selective Sweep in East Asian Populations</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/490722" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-iZWl_J6vdsRXtcetWbIV4bSRtHnCmj-FTdceqNTA4zQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/9/2953.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/9/2953" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-dW2Fy700Zu1fowcgnm3tfZxOWO9tLe4ta1iMxJf4vho" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/9/2953" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/9/2953.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490722" id="node490722--2" data-pisa="ggg;8/9/2953" data-pisa-master="ggg;g3.118.200462" data-apath="/ggg/8/9/2953.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Complex Haplotypes of <em>GSTM1</em> Gene Deletions Harbor Signatures of a Selective Sweep in East Asian Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8794-3217" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>M. Saitou</span>, <span class="highwire-citation-author" data-delta="1">Y. Satta</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-4371-679X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>O. Gokcumen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2953-2966;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200462 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490722--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">M. Saitou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Dept. of Biological Sciences, SUNY at Buffalo, NY, 14260-1300</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Dept. of Biological Sciences, The University of Tokyo, Tokyo, Japan, 113-0033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BSaitou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Saitou%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AM.%2BSaitou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8794-3217" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for M. Saitou</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Y. Satta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Dept. of Evolutionary Studies of Biosystems, SOKENDAI (Graduate University for Advanced Studies), Kanagawa, Japan 240-0193</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Y.%2BSatta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Satta%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AY.%2BSatta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">O. Gokcumen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Dept. of Biological Sciences, SUNY at Buffalo, NY, 14260-1300</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=O.%2BGokcumen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gokcumen%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AO.%2BGokcumen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4371-679X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for O. Gokcumen</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:omergokc@buffalo.edu" class="" data-icon-position="" data-hide-link-title="0">omergokc@buffalo.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/9/2953" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/9/2953&amp;title=Complex%20Haplotypes%20of%20GSTM1%20Gene%20Deletions%20Harbor%20Signatures%20of%20a%20Selective%20Sweep%20in%20East%20Asian%20Populations" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/9/2953&amp;title=Complex%20Haplotypes%20of%20GSTM1%20Gene%20Deletions%20Harbor%20Signatures%20of%20a%20Selective%20Sweep%20in%20East%20Asian%20Populations" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/9/2953&amp;title=Complex%20Haplotypes%20of%20GSTM1%20Gene%20Deletions%20Harbor%20Signatures%20of%20a%20Selective%20Sweep%20in%20East%20Asian%20Populations" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/9/2953&amp;text=Complex%20Haplotypes%20of%20GSTM1%20Gene%20Deletions%20Harbor%20Signatures%20of%20a%20Selective%20Sweep%20in%20East%20Asian%20Populations" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/9/2953&amp;title=Complex%20Haplotypes%20of%20GSTM1%20Gene%20Deletions%20Harbor%20Signatures%20of%20a%20Selective%20Sweep%20in%20East%20Asian%20Populations" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/9/2953&amp;t=Complex%20Haplotypes%20of%20GSTM1%20Gene%20Deletions%20Harbor%20Signatures%20of%20a%20Selective%20Sweep%20in%20East%20Asian%20Populations" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/9/2953&amp;title=Complex%20Haplotypes%20of%20GSTM1%20Gene%20Deletions%20Harbor%20Signatures%20of%20a%20Selective%20Sweep%20in%20East%20Asian%20Populations" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/9/2953&amp;title=Complex%20Haplotypes%20of%20GSTM1%20Gene%20Deletions%20Harbor%20Signatures%20of%20a%20Selective%20Sweep%20in%20East%20Asian%20Populations" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/490722/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490722" id="node490722--3" data-pisa="ggg;8/9/2953" data-pisa-master="ggg;g3.118.200462" data-apath="/ggg/8/9/2953.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Complex Haplotypes of <em>GSTM1</em> Gene Deletions Harbor Signatures of a Selective Sweep in East Asian Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8794-3217" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>M. Saitou</span>, <span class="highwire-citation-author" data-delta="1">Y. Satta</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-4371-679X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>O. Gokcumen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2953-2966;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200462 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490722--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">M. Saitou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Dept. of Biological Sciences, SUNY at Buffalo, NY, 14260-1300</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Dept. of Biological Sciences, The University of Tokyo, Tokyo, Japan, 113-0033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BSaitou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Saitou%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AM.%2BSaitou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8794-3217" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for M. Saitou</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Y. Satta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Dept. of Evolutionary Studies of Biosystems, SOKENDAI (Graduate University for Advanced Studies), Kanagawa, Japan 240-0193</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Y.%2BSatta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Satta%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AY.%2BSatta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">O. Gokcumen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Dept. of Biological Sciences, SUNY at Buffalo, NY, 14260-1300</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=O.%2BGokcumen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gokcumen%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AO.%2BGokcumen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4371-679X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for O. Gokcumen</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:omergokc@buffalo.edu" class="" data-icon-position="" data-hide-link-title="0">omergokc@buffalo.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2953&amp;rft.epage=2966&amp;rft.atitle=Complex%20Haplotypes%20of%20GSTM1%20Gene%20Deletions%20Harbor%20Signatures%20of%20a%20Selective%20Sweep%20in%20East%20Asian%20Populations&amp;rft.volume=8&amp;rft.issue=9&amp;rft.date=2018-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Saitou%2C+M.&amp;rft.au=Satta%2C+Y.&amp;rft.au=Gokcumen%2C+O."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/490722/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/490722/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/490722/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/490722/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/490722/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/490722/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/490722/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/490722/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/490722/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/490722/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/490722/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/490722/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/9/2953" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Data availability</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="odd"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#sec-22" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/9/2953.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/9/2953.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/9/2953.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/9/2953.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
